<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Neuronal Cbl Controls Biosynthesis Insulin-Like Peptides in <br /> Drosophila melanogaster <br /> Yue Yu,a Ying Sun,a Shengqi He,a Cheng Yan,a Liangyou Rui,b Wenjun Li,a Yong Liua <br /> Key Laboratory Nutrition Metabolism, Institute Nutritional Sciences, Shanghai Institutes Biological Sciences, Chinese Academy Sciences, Graduate <br /> School Chinese Academy Sciences, Shanghai, China,a Department Molecular Integrative Physiology, University Michigan Medical School, Ann <br /> Arbor, Michigan, USAb <br />  <br /> The Cbl family proteins function E3 ubiquitin ligases adaptor proteins regulate various cellular signaling events, <br /> including insulin/insulin-like growth factor 1 (IGF1) epidermal growth factor (EGF) pathways. These pathways play es- <br /> sential roles growth, development, metabolism, <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span>. Here Drosophila melanogaster, Drosophila Cbl <br /> (dCbl) regulates longevity carbohydrate metabolism downregulating production Drosophila insulin-like pep- <br /> tides (dILPs) brain. We dCbl highly expressed brain knockdown expression dCbl specif- <br /> ically neurons <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>RNA interference</span> increased sensitivity oxidative stress starvation, decreased carbohydrate levels, and <br /> shortened life span. Insulin-producing neuron-specific knockdown dCbl resulted similar phenotypes. dCbl deficiency in <br /> either brain insulin-producing cells upregulated expression dilp genes, resulting elevated activation dILP <br /> pathway, including phosphorylation Drosophila Akt Drosophila extracellular signal-regulated kinase (dERK). Genetic <br /> interaction analyses revealed blocking Drosophila epidermal growth factor receptor (dEGFR)-dERK signaling pan-neu- <br /> rons insulin-producing cells overexpressing dominant-negative form dEGFR abolished effect dCbl deficiency on <br /> the upregulation dilp genes. Furthermore, knockdown c-Cbl INS-1 cells, rat &#226;?&#164;-cell line, increased insulin biosyn- <br /> thesis glucose-stimulated secretion ERK-dependent manner. Collectively, results suggest neuronal dCbl reg- <br /> ulates life span, stress responses, metabolism suppressing dILP production EGFR-ERK pathway mediates the <br /> dCbl action. Cbl suppression insulin biosynthesis evolutionarily conserved, raising possibility Cbl similarly <br /> exert physiological actions regulating insulin production &#226;?&#164; cells. <br />  <br />  <br />  <br /> U    pon ligand stimulation, activation receptor tyrosine ki- <br />      nases (RTKs) initiates downstream signaling responses <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> control physiological processes (50). Evolutionarily con- <br />                                                                                        interactions plethora proteins (51, 57). Interestingly, the <br />                                                                                        Cbl orthologue fruit fly, Drosophila melanogaster (dCbl), <br />                                                                                        exists long short isoforms result alternative splic- <br /> served invertebrates mammals, insulin/insulin-like                             ing (47). The long form dCbl domain structure identical to <br /> growth factor 1 (IGF1) epidermal growth factor (EGF) act                           mammalian c-Cbl Cbl-b, short version <br /> through RTK-mediated signaling cascades, play central                            contains solely TKB RING finger domains. Both isoforms <br /> roles regulation growth, development, metabolism,                        shown downregulate EGFR signaling (32, 41), and <br /> survival (3, 15, 29, 36, 56, 69). Sophisticated regulatory mecha-                      recent studies documented long isoform dCbl <br /> nisms work regulate duration intensity RTK                        regulates EGFR pathway, short preferentially <br /> signaling. The Cbl (Casitas B-lineage lymphoma) proteins, fam-                       controls notch signaling (62). <br /> ily E3 ubiquitin ligases adaptor proteins (60), key reg-                       The evolutionarily conserved insulin/IGF1 signaling through <br /> ulators RTK signaling, best exemplified neg-                     RTKs regulates multiple physiological processes, including <br /> ative control EGF pathway Cbl-mediated                                  metabolic homeostasis, stress resistance, longevity (15, 56). <br /> ubiquitylation endocytic destruction EGF receptor                           The insulin signaling pathway subject positive and <br /> (EGFR) (11, 26, 27, 51, 55, 67). However, functional evolution                     negative regulation (9, 53, 54, 58). Emerging evidence suggests an <br /> of Cbl&#226;&#8364;&#8482;s regulatory action respect physiological inter-                    unanticipated complexity respect functional effects of <br /> connection cooperation multiple RTK pathways remains                            mammalian Cbl proteins insulin actions. In 3T3-L1 adi- <br /> poorly understood.                                                                     pocytes, Cbl demonstrated act adaptor molecule and <br />     Cbl proteins known regulate diverse range cellular                     play positive regulatory insulin-controlled glucose trans- <br /> events promoting ubiquitylation-directed degradation                        port (30, 31, 49). On hand, reported c-Cbl <br /> target proteins acting adaptors signaling com-                        promote ubiquitylation insulin IGF1 recep- <br /> plexes (51). A growing body evidence established Cbl-                      tors (1, 52). Moreover, whole-body ablation c-Cbl mice led <br /> dependent downregulation EGFR pathway evolutionarily <br /> conserved Caenorhabditis elegans vertebrates (14, 17, 27, <br /> 64). In mammals, Cbl homologues, c-Cbl, Cbl-b,                            Received 5 June 2012 Accepted 29 June 2012 <br /> and Cbl-3, possess highly conserved TKB (tyrosine-kinase-                           Published ahead print 9 July 2012 <br /> binding) RING finger domains N-terminal regions,                             Address correspondence Yong Liu, liuy@sibs.ac.cn. <br /> allowing function E3 ubiquitin ligases. c-Cbl Cbl-b                        Copyright &#194;&#169; 2012, American Society Microbiology. All Rights Reserved. <br /> are ubiquitously expressed, contain proline-rich do-                             doi:10.1128/MCB.00592-12 <br /> mains extended C-terminal portions mediate <br />  <br />  <br /> 3610   mcb.asm.org                                Molecular Cellular Biology   p. 3610 &#226;&#8364;&#8220;3623                                 September 2012 Volume 32 Number 18 <br />                                                                                                                   Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br /> to reduced adiposity, presumably increased energy ex-                     Starvation paraquat sensitivity assays. Flies 3 days age (10 <br /> penditure, improving peripheral insulin sensitivity (37).               flies/vial) maintained starvation medium containing 1% agar or <br /> These observations indicate c-Cbl negatively regulates insulin          starvation medium supplemented 5% sucrose 30 mM para- <br /> signaling promoting insulin receptor ubiquitylation deg-              quat. Flies transferred daily fresh medium, number of <br /> radation (38). c-Cbl Cbl-b serve key modulators               dead flies recorded 12 h. <br />                                                                                   Body weight life span. The body weight individual 3-day-old <br /> immune responses (10, 19, 44), genetic deletion Cbl-b was <br />                                                                              flies (90 flies/batch) measured using precision balance. For life span <br /> shown enhance infiltration activation adipose tissue <br />                                                                              determination, newly eclosed flies placed vials (10 flies/vial). Flies <br /> macrophages, resulting peripheral insulin resistance mice              transferred fresh food day, number dead <br /> (18). Given different roles insulin signaling diverse tissues      flies monitored. <br /> or cell types, elucidated Cbl exerts cell type-          Glycogen trehalose measurements. Flies (8 10/group) were <br /> or tissue-specific functions control insulin action sys-        anesthetized using CO2 homogenized 250 &#226;?&#174;l 0.25 M Na2CO3 <br /> temic metabolism.                                                            buffer. After incubation 95&#194;&#176;C 2 h, 150 &#226;?&#174;l 1 M acetic acid 600 <br />     The Drosophila genome contains seven insulin-like peptide                &#226;?&#174;l 0.25 M sodium acetate (pH 5.2) added. The mixture cen- <br /> (dILP) genes (dilp1 dilp7) single insulin receptor                trifuged 20 min 12,000 rpm room temperature. To convert treh- <br /> (dInR) gene (4). dILP2, dILP3, dILP5, primary forms             alose glycogen glucose, aliquots (50 &#226;?&#174;l) supernatant were <br /> dILPs, expressed secreted median neurosecretory               incubated 0.05 ml porcine trehalase (T8778; Sigma) 37&#194;&#176;C over- <br /> insulin-producing cells (IPCs) brain (5, 20). IPCs                night 5 units/ml amyloglucosidase (A7420; Sigma) 55&#194;&#176;C 4 h. <br />                                                                              Total glucose measured glucose assay reagent (G3293; Sigma). For <br /> thought functionally analogous mammalian pancreatic <br />                                                                              normalization, protein content supernatant determined using a <br /> islet &#226;?&#164; cells (48). The expression dilp genes shown be <br />                                                                              Bio-Rad protein assay reagent. <br /> regulated independently nutritional signals developmental                  Microscopic imaging analysis. Fly brains fixed 4% parafor- <br /> stages (5, 20); however, underlying mechanisms remain largely            maldehyde (PFA) sodium phosphate buffer (pH 7.4). After rinsing in <br /> elusive.                                                                     sodium phosphate buffer, brains incubated 0.01 M phosphate- <br />     We investigated physiological functions dCbl using ge-            buffered saline (PBS; pH 7.4). Specimens analyzed using Olym- <br /> netic models Drosophila. Here report neuronal dCbl                pus FV1000 confocal microscope (Japan) &#226;&#171;&#187;60 water objective with <br /> participates biosynthetic regulation dILPs                 numerical aperture 1.2. Confocal microscopic images obtained <br /> downregulation EGFR signaling. Our findings indicate             optical section thickness 0.31 0.37 &#226;?&#174;m processed using <br /> Cbl-mediated modulation insulin production represent               Olympus FV1000 software. <br /> evolutionarily conserved, cell type-specific mode Cbl actions               Reverse transcription-PCR (RT-PCR) analysis. Total RNA pre- <br /> the coordinate control growth, life span, metabolism.                 pared 3-day-old flies INS-1 cells TRIzol reagent (Invitro- <br />                                                                              gen). First-strand cDNA synthesized using Moloney murine leukemia <br /> MATERIALS AND METHODS                                                        virus reverse transcriptase random hexamer primers (Invitrogen). <br /> Fly strains. The P{SUPor-P}dCblEY11427 line (designated dCblEY/&#226;&#171;&#185;     Regular PCR performed using Taq kits (TaKaRa), quantitative <br /> heterozygote) P{EPgy2}dCblKG03080 line (designated dCblKG/&#226;&#171;&#185;      PCR (qPCR) ABI 7500 fast real-time PCR using <br /> the heterozygote), P-element inserted 5= untranslated     SYBR green PCR master mix (ABI). Ribosomal protein L32 (RPL32), <br /> region (UTR) disruption dCbl gene, obtained         &#226;?&#163;-tubulin 84B (&#226;?&#163;-Tub84B), GAPDH (glyceraldehyde-3-phosphate de- <br /> Bloomington Drosophila Stock Center backcrossed w1118 back-       hydrogenase) used internal control normalization. The oli- <br /> ground generations. The green fluorescent protein (GFP)-express-     gonucleotide primers target genes analyzed follows: dCbl, <br /> ing line P{GAL4-Kr.C}DC2,P{UAS-GFP.S65T}DC (Bloomington Dro-                 sense primer 5=-TCAAGGGCACCGAACAAAT-3= antisense primer <br /> sophila Stock Center) used cross P{SUPor-P}dCblEY11427       5=-TGCTCCGTCGTCCCAAAC-3=; dCbl long form, sense primer 5=- <br /> or P{EPgy2}dCblKG03080 line generating GFP-labeled heterozygous          ATGAGGACATCGTTGAGGTGG-3= antisense primer 5=-CAGAGT <br /> dCblEY/&#226;&#171;&#185; dCblKG/&#226;&#171;&#185; lines, respectively. These heterozygous lines      TGCGTGTAATGGGAG-3=; dCbl short form, sense primer 5=-AACT <br /> were subsequently crossed produce identifiable homozygous             AACATCCACAGACCAAC-3= antisense primer 5=-GAAGAACGAA <br /> dCblEY/KG larvae studies.                                        CAACTAAACGA-3=; dilp2, sense primer 5=-ATCTGGACGCCCTCA <br />     The UAS-dCbl-RNAi line (where UAS upstream activating sequence        ATCC-3= antisense primer 5=-CCAAGATAGCTCCCAGGAAAGA- <br /> and RNAi RNA interference) (transformant accession number 22335)          3=; dilp3, sense primer 5=-AGAGAACTTTGGACCCCGTGAA-3= and <br /> was obtained Vienna Drosophila RNAi Center. For life span       antisense primer 5=-TGAACCGAACTATCACTCAACAGTCT-3=; for <br /> experiments, UAS-dCbl-RNAi flies used backcrossing       dilp5, sense primer 5=-CGCTCCGTGATCCCAGTT-3= antisense <br /> w1118 background generations. The elav-Gal4 dominant-neg-        primer 5=-GCTATCCAAATCCGCCAAG-3=; CG2162, sense primer <br /> ative (DN) UAS-EGFR-DN lines Bloomington Drosophila            5=-TGCCCAAAATGGACTATACGG-3= antisense primer 5=-GCTT <br /> Stock Center. UAS-EGFR-DN backcrossed w1118 background          CAACTTGACAAACGGAT-3=; rat Ins-1, sense primer 5=-ACACCCA <br /> for generations crossing UAS-dCbl-RNAi line. The        AGTCCCGTCGTGAAGTGG-3= antisense primer 5=-GGCGGGGA <br /> dilp2-Gal4 line generous gift Eric Rulifson (University Cal-   GTGGTGGACTCAGT-3=; rat Ins-2, sense primer 5=-TCATCCT <br /> ifornia San Francisco [UCSF]). Heterozygous UAS-dCbl-RNAi animals         CTGGGAGCCCCGC-3= antisense primer 5=-GGTCTGAAGGTCAC <br /> were crossed elav-Gal4 dilp2-Gal4 generate strains       CGGCCC-3=. <br /> neuronal IPC-specific dCbl knockdown making use cyo bal-             Antibodies chemicals. Polyclonal antibodies p-Akt <br /> ancer. For IPC imaging, UAS-srcGFP line (Bloomington Drosophila          (synthetic phosphopeptide corresponding residues surrounding <br /> Stock Center) crossed UAS-dCbl-RNAi UAS-EGFR-DN flies            Ser473 mouse Akt), total Akt (a synthetic peptide corresponding to <br /> before crossing dilp2-Gal4 drivers generate              carboxy-terminal sequence mouse Akt), p-ERK (a synthetic <br /> dilp2G4&#226;&#172;&#381;GFP, dCbl-RNAi dilp2G4&#226;&#172;&#381;GFP, EGFR-DN flies. IPC cells              phosphopeptide corresponding residues surrounding Thr202/ <br /> were visualized GFP fluorescence confocal microscopy. Flies     Tyr204 human p44/42 mitogen-activated protein [MAP] kinase), <br /> maintained 25&#194;&#176;C 12-h dark/12-h light cycles standard food         total ERK (a synthetic peptide corresponding sequence the <br /> medium (20 g inactivated yeast powder, 60 g corn flour, 10 g agar, 100 g     C terminus rat p44/42 MAP kinase) purchased Cell <br /> sucrose, 15 ml 10% Tego 75% ethanol liter).                       Signaling, &#226;?&#163;-tubulin antibody purchased Biomeda. <br />  <br />  <br /> September 2012 Volume 32 Number 18                                                                                                      mcb.asm.org 3611 <br />  Yu et al. <br />  <br />  <br />  <br />  <br /> FIG 1 Developmental tissue expression patterns dCbl mRNA. (A) Expression features dCbl mRNA different developmental stages. dCbl mRNA <br /> abundance wild-type w1118 flies (60 embryos, 20 30 larvae, 10 pupae, 10 adult flies 3 days age; n &#226;&#172;&#381; 3 independent experiments) assessed by <br /> real-time qRT-PCR using RPL32 internal control. L1, L2, L3 first-, second-, third-instar larvae, respectively. (B) Tissue distribution patterns dCbl <br /> mRNA expression w1118 third-instar larvae adult flies (40 larvae 20 adults, n &#226;&#172;&#381; 3 independent experiments). (C) The abundance mRNA <br /> transcripts encoding long (L) short (S) form dCbl determined qRT-PCR w1118 flies different developmental stages (60 embryos, 20 30 <br /> larvae, 10 pupae, 10 adult flies; n &#226;&#172;&#381; 3 independent experiments). (D) Tissue expression patterns mRNA transcripts long (L) short (S) forms <br /> of dCbl w1118 third-instar larvae adult flies (30 larvae 30 adults, n &#226;&#172;&#381; 3 independent experiments). <br />  <br />  <br /> ERK-specific kinase MEK inhibitor PD98059 (P215) phosphoino-                    following sequences: si-c-Cbl_#1, 5=-GCUUCCAAGGCUUCUUCUAdT <br /> sitide 3-kinase (PI3K)-specific kinase inhibitor LY294002 (L9908)                   dT-3=, directed coding region nucleotide (nt) 1552 <br /> were Sigma, EGF R&amp;D. PDX-1 antibody                      downstream start codon rat c-Cbl; si-c-Cbl_#2, 5=-CCUUGGA <br /> Upstate (07-696; Millipore).                                                        AUGGGAGAGAAUdTdT-3=, directed coding region nt <br />     Western immunoblotting. Protein extracts prepared flies               1819 downstream start codon rat c-Cbl (GenBank accession <br /> by homogenization radioimmunoprecipitation assay (RIPA) buffer                   number XM_576396); si-CON, 5=-GCAGUAAGCGAUACGCAAAd <br /> (150 mM NaCl, 1% NP-40, 0.5% sodium deoxycholate, 0.1% SDS, 50 mM                   TdT-3=, nonspecific scrambled control. INS-1 cells transfected <br /> Tris-HCl) using Polytron homogenizer. Lysates sonicated             siRNA duplexes 50 nM using Lipofectamine RNAiMAX reagent <br /> 30 s centrifuged 20 min 13,000 &#226;&#171;&#187; g remove debris. INS-1           (13778; Invitrogen), cells cultured 48 h anal- <br /> cells lysed RIPA buffer. Proteins (40 &#226;?&#174;g) separated            ysis. <br /> SDS-PAGE transferred polyvinylidene difluoride filter mem-                     Plasmid adenoviral c-Cbl overexpression. The pKF plasmid for <br /> brane (Amersham Biosciences). After incubation desired anti-               human c-Cbl expression generous gift Jian Zhang (University <br /> bodies, blots developed Amersham Biosciences ECL Plus              Chicago). The cDNA fragment c-Cbl subcloned pcDNA6.0 <br /> detection system.                                                                   (Invitrogen). The c-Cbl-C381A substitution mutant generated using <br />     ChIP assays. Chromatin immunoprecipitation (ChIP) assays                   Muta-direct site-directed mutagenesis kit (SBS, Shanghai, China). Re- <br /> conducted PDX-1 antibody following manufacturer&#226;&#8364;&#8482;s in-                  combinant adenoviruses overexpression c-Cbl proteins were <br /> structions (26156; Thermo). Quantitative PCR using ABI                  produced AdEasy adenoviral vector (Stratagene, La Jolla, <br /> 7500 Fast real-time PCR system. The primers used amplification           CA), following manufacturer&#226;&#8364;&#8482;s instructions. Briefly, control virus, <br /> Ins-1 promoter sequence 5=-CTGGGAAATGAGGTGGAAAA-3=                             coding sequence EGFP (pEGFP-N1; Clontech, Mountain View, <br /> and 5=-AGGAGGGGTAGGTAGGCAGA-3=.                                                     CA) subcloned pShuttle-cytomegalovirus (CMV) vector di- <br />     Cell culture, transfection, luciferase reporter assay. Rat insuli-          gested SalI NotI, cDNA fragments encoding c-Cbl pro- <br /> noma INS-1 cells cultured RPMI 1640 medium containing 2 mM                  teins likewise subcloned pShuttle-CMV. The shuttle plasmids <br /> L-glutamine supplemented 10% fetal bovine serum, 16.7 mM glu-                  pAdEasy-1 subsequently cotransformed BJ5183 bacteria to <br /> cose, 10 mM HEPES, 1 mM sodium pyruvate, 50 &#226;?&#174;M 2-mercaptoethanol,                   yield recombinant adenoviral DNA, used transfection <br /> 100 IU ml/liter penicillin, 100 &#226;?&#174;M streptomycin 5% CO2 37&#194;&#176;C.              HEK293A cells. Recombinant viral particles obtained cell <br /> INS-1 cells transfected rat insulin 1 promoter (RIP)-lu-              lysates. INS-1 cells 60% confluence infected multiplicity of <br /> ciferase (Luc) reporter construct (kindly provided M. German, UCSF)              infection (MOI) 20. <br /> using Lipofectamine 2000 reagents (Invitrogen). Luciferase activities              Measurement insulin secretion content. Insulin secretion <br /> analyzed using dual-luciferase assay kit (Promega), according              INS-1 cells determined described previously (43, 66). Briefly, <br /> manufacturer&#226;&#8364;&#8482;s instructions.                                                        confluent cells 24-well plates washed HEPES-balanced salt <br />     c-Cbl knockdown transient transfection. Duplex small interfering             solution (HBSS), followed incubation buffer 2 h. Insu- <br /> RNA (siRNA) oligonucleotides used silencing expression c-Cbl             lin secreted medium measured static incubation <br /> were purchased Genepharma Ltd. (Shanghai, China)                   2 h HBSS containing 2.8 mM 16.7 mM glucose. To determine the <br />  <br />  <br /> 3612    mcb.asm.org                                                                                                                    Molecular Cellular Biology <br />                                                                                                                          Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br />  <br /> FIG 2 dCbl essential development survival flies. (A) Two heterozygous dCbl mutant lines P-element insertion indicated dCbl loci were <br /> crossed generate dCblEY/KG line. Genomic organization dCbl gene containing P-element shown (black boxes, coding exons; open boxes, UTRs; <br /> triangles, P-elements). (B) dCbl mRNA expression levels analyzed qRT-PCR body 3-day-old adult flies indicated genotype (30 <br /> flies/group, n &#226;&#172;&#381; 3 independent experiments). RPL32 used internal control. (C) Disruption dCbl resulted developmental delay. The pupation rate <br /> of wild-type w1118 dCblEY/KG larvae determined recording number pupae third-instar larval stage (&#226;&#172;&#381;120 animals/group) 24 h. (D) <br /> Disruption dCbl reduced survival rate. The percentage w1118 dCblEY/KG flies survived larval stage (&#226;&#172;&#381;120 animals/group). Results are <br /> shown mean &#226;&#171;&#190; SEM; **, P &#226;&#172;? 0.01 versus w1118 Student&#226;&#8364;&#8482;s t test. (E) Disruption dCbl led decreased carbohydrate levels. Whole-body glycogen and <br /> trehalose contents measured 3-day-old male adult w1118 dCblEY/KG flies. Data presented mean &#226;&#171;&#190; SEM; *, P &#226;&#172;? 0.05 versus w1118 control <br /> by one-way ANOVA. (F) Sensitivity starvation. Male adult flies 3 days age deprived food, survival rates determined (126 w1118 flies and <br /> 122 dCblEY/KG flies). &#226;?&#185;2 &#226;&#171;&#189; 66.91; P &#226;&#172;? 0.001 log-rank test. (G) Life span. Survival curves male w1118 (n &#226;&#171;&#189; 225) dCblEY/KG (n &#226;&#171;&#189; 184) flies. &#226;?&#185;2 &#226;&#171;&#189; 405.1; P &#226;&#172;? <br /> 0.0001 log-rank test. <br />  <br />  <br />  <br /> intracellular insulin content, cells lysed incubation overnight         adult flies, expression dCbl abundant in <br /> 4&#194;&#176;C 1 ml mixture ethanol-water-concentrated HCl (750:235:15).           head body (Fig. 1B). When detected specific <br /> After centrifugation 5 min 12,000 &#226;&#171;&#187; g 4&#194;&#176;C, insulin measured          transcripts encoding dCbl isoform, short form was <br /> using enzyme-linked immunosorbent assay (ELISA) kit (EZRMI-13K; <br />                                                                                    efficiently expressed long form embryos, <br /> Millipore). <br />     Statistical analysis. All data presented mean &#226;&#171;&#190; standard            higher expression observed long form in <br /> error mean (SEM). Statistical analysis performed using               pupae adults (Fig. 1C). This implies distinct functional <br /> unpaired two-tailed t test one-way two-way analysis variance             roles dCbl isoforms development. Interest- <br /> (ANOVA), followed Bonferroni&#226;&#8364;&#8482;s posttest, GraphPad Prism soft-              ingly, long form expressed higher level the <br /> ware (version 5.0). Log-rank tests used analysis life span         head adult flies (Fig. 1D), suggesting predominant role for <br /> stress survival curves. P values &#226;&#172;?0.05 considered statistically sig-       isoform brain. <br /> nificant. <br />                                                                                       Next, investigated importance dCbl development <br /> RESULTS                                                                            survival using mutant lines flies (designated dCblEY and <br /> Characterization dCbl expression importance de-                      dCblKG; Bloomington Drosophila Stock Center) dCbl <br /> velopment survival. To gain insight functions                      gene disrupted insertion P-element 5= UTR exon <br /> dCbl, examined mRNA expression patterns dCbl                          (Fig. 2A). Due poor viability homozygous dCblEY/EY or <br /> w1118 flies quantitative RT-PCR (qRT-PCR). When assessed                        dCblKG/KG flies, crossed mutant lines obtained <br /> for transcripts encoding long short forms, dCbl                       viable heteroallelic dCblEY/KG flies markedly decreased <br /> was expressed high levels embryos adults low                      dCbl mRNA transcripts (Fig. 2B). Disruption dCbl expression <br /> levels larval stages (Fig. 1A). Moreover, larvae                    resulted &#226;&#172;&#402;24-h delay larval pupation (Fig. 2C) an <br />  <br />  <br /> September 2012 Volume 32 Number 18                                                                                                             mcb.asm.org 3613 <br />  Yu et al. <br />  <br />  <br />  <br />  <br /> FIG 3 Knockdown neuronal dCbl expression reduces oxidative stress resistance life span. (A) Knockdown neuronal dCbl expression analyzed by <br /> qRT-PCR head adult elavG4&#226;&#172;&#381;dCbl-Ri flies relative control elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies male female flies (90 flies/group, n &#226;&#172;&#381; 3 independent <br /> experiments). The mRNA expression CG2162, potential off-target gene, assessed. (B) The body weight adult flies indicated genotype 3 <br /> days age (90 flies/genotype). Error bars panels A B represent SEMs; *, P &#226;&#172;? 0.05 Student&#226;&#8364;&#8482;s t test. (C) Life span. Survival curves male female <br /> elavG4&#226;&#172;&#381;&#226;&#171;&#185; (300 male 249 female) elavG4&#226;&#172;&#381;dCbl-Ri (252 male 264 female) flies. &#226;?&#185;2 &#226;&#171;&#189; 220.2 males &#226;?&#185;2 &#226;&#171;&#189; 119.4 females; P &#226;&#172;? 0.0001 by <br /> log-rank test. (D) Sensitivity paraquat. Male female adult flies 3 days age treated 30 mM paraquat. Survival rates determined for <br /> elavG4&#226;&#172;&#381;&#226;&#171;&#185; (165 male 148 female) elavG4&#226;&#172;&#381;dCbl-Ri (200 male 208 female) flies. &#226;?&#185;2 &#226;&#171;&#189; 113.8 males &#226;?&#185;2 &#226;&#171;&#189; 74.84 females; P &#226;&#172;? 0.0001 by <br /> log-rank test. <br />  <br />  <br />  <br /> &#226;&#172;&#402;63% reduction adult survival rate male dCblEY/KG flies (Fig.              &#226;&#172;&#402;55% male elavG4&#226;&#172;&#381;dCbl-Ri flies &#226;&#172;&#402;48% female <br /> 2D). Notably, whole-body glycogen trehalose (an insect                   elavG4&#226;&#172;&#381;dCbl-Ri flies compared elavG4&#226;&#172;&#381;&#226;&#171;&#185; control <br /> disaccharide main form circulatory carbohydrates)                 flies (Fig. 3A). In contrast, changes mRNA levels of <br /> levels male dCblEY/KG flies decreased &#226;&#172;&#402;35% &#226;&#172;&#402;44%, re-                   CG2162 (Fig. 3A), potential off-target gene UAS-dCbl-RNAi <br /> spectively, compared w1118 control line (Fig. 2E).               flies, detected. Interestingly, knockdown neuronal dCbl <br /> Moreover, male dCblEY/KG flies showed increased sensitivity                   expression led &#226;&#172;&#402;12% increase body weight male <br /> starvation, exhibiting &#226;&#172;&#402;20% decrease median survival                   adult flies (Fig. 3B). Moreover, neuronal dCbl suppression sig- <br /> time food deprivation (Fig. 2F) (3.6 days dCblEY/KG flies               nificantly reduced life span male female <br /> and 4.5 days w1118 flies). dCbl deficiency led dramat-             elavG4&#226;&#172;&#381;dCbl-Ri flies (Fig. 3C), &#226;&#172;&#402;27% &#226;&#172;&#402;24% de- <br /> ically shortened life span, &#226;&#172;&#402;67% reduction median                 creases, respectively, observed median life span (53 days <br /> life span observed male dCblEY/KG flies (Fig. 2G; 23 days                 male elavG4&#226;&#172;&#381;dCbl-Ri flies, 73 days male elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies, <br /> dCblEY/KG flies 70 days w1118 control). These data suggest               68 days female elavG4&#226;&#172;&#381;dCbl-Ri flies, 90 days female <br /> that dCbl plays essential role Drosophila development                  elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies). Consistent shorter life span, male <br /> survival, agreement previous reports (41, 42, 63).                       female elavG4&#226;&#172;&#381;dCbl-Ri flies exhibited reduced tolerance <br />     Neuronal dCbl regulates Drosophila life span stress tol-                 oxidative stress (Fig. 3D), displaying &#226;&#172;&#402;25% &#226;&#172;&#402;26% de- <br /> erance. Given high expression dCbl heads flies (Fig.            creases median survival time, respectively, treatment <br /> 1B D), investigated physiological functions neuronal               paraquat (5.0 days male elavG4&#226;&#172;&#381;dCbl-Ri flies, 6.7 days <br /> dCbl using Gal4 UAS-RNA interference (RNAi)                    male elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies, 6.6 days female elavG4&#226;&#172;&#381;dCbl-Ri <br /> knock expression neuronal dCbl. We crossed UAS-                      flies, 8.9 days female elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies). These data thus <br /> dCbl-RNAi flies (Vienna Drosophila RNAi Center) pan-neu-                    revealed crucial role neuronal dCbl regulation lon- <br /> ron elav-GAL4 drivers generate elav-GAL4/UAS-dCbl-RNAi                        gevity stress tolerance Drosophila. <br /> (here denoted elavG4&#226;&#172;&#381;dCbl-Ri) flies. Under fed starved con-                       Neuronal dCbl deficiency associated upregulation of <br /> ditions, head dCbl mRNA abundance decreased                               insulin-like peptides. The insulin/IGF pathway known reg- <br />  <br />  <br /> 3614    mcb.asm.org                                                                                                                Molecular Cellular Biology <br />                                                                                                                           Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br />  <br /> FIG 4 Neuronal dCbl knockdown results upregulated dilp gene expression parallel increased phosphorylation dAkt dERK. (A) RNA was <br /> extracted head fed starved (Stv; 24 h) male adult elavG4&#226;&#172;&#381;&#226;&#171;&#185; elavG4&#226;&#172;&#381;dCbl-Ri flies (3 days age, 90 flies/group). The abundance dilp2, dilp3, <br /> or dilp5 mRNA measured qRT-PCR (n &#226;&#172;&#381; 3 independent experiments). After normalization levels RPL32, results shown means &#226;&#171;&#190; SEMs; <br /> *, P &#226;&#172;? 0.05 versus value control elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies one-way ANOVA. (B) Western immunoblot analysis dAkt dERK phosphorylation using the <br /> indicated antibodies. Head body protein extracts analyzed fed starved (24 h) male adult flies (5 days age, 90 flies/group). Representative results <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> independent experiments shown. Drosophila &#226;?&#163;-tubulin used internal control. (C D) Relative phosphorylation levels dAkt (C) and <br /> dERK (D) determined densitometric quantification immunoblots, presented mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3). *, P &#226;&#172;? 0.05 versus control elavG4&#226;&#172;&#381;&#226;&#171;&#185; <br /> flies; #, P &#226;&#172;? 0.05 fed starved conditions two-way ANOVA. <br />  <br />  <br /> ulate stress resistance longevity flies (8, 59). We asked                   To determine effects neuronal dCbl knockdown on <br /> neuronal dCbl regulates stress resistance longevity sup-                    dILP signaling, measured phosphorylation levels of <br /> pressing production dILPs brain. Indeed, male                     dAkt extracellular signal-related kinase dERK <br /> elavG4&#226;&#172;&#381;dCbl-Ri flies, knockdown neuronal dCbl increased                     (Fig. 4B). In male elavG4&#226;&#172;&#381;dCbl-Ri flies, phospho-dAkt levels <br /> mRNA expression levels dilp2, dilp3, dilp5 &#226;&#172;&#402;113%,                        head body increased &#226;&#172;&#402;51% &#226;&#172;&#402;41%, respec- <br /> &#226;&#172;&#402;44%, &#226;&#172;&#402;92%, respectively, fed state &#226;&#172;&#402;50%,                        tively, fed state &#226;&#172;&#402;50% &#226;&#172;&#402;42%, respectively, in <br /> &#226;&#172;&#402;120%, &#226;&#172;&#402;100%, respectively, starved condition                     starved state compared level elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies <br /> comparison elavG4&#226;&#172;&#381;&#226;&#171;&#185; control flies (Fig. 4A). Female                    (Fig. 4C). Phospho-dERK levels head body increased <br /> elavG4&#226;&#172;&#381;dCbl-Ri flies showed similarly upregulated expres-                     &#226;&#172;&#402;62% &#226;&#172;&#402;53%, respectively, fed conditions by <br /> sion dilp2 (by &#226;&#172;&#402;99%), dilp3 (by &#226;&#172;&#402;102%), dilp5 (by &#226;&#172;&#402;76%)                     &#226;&#172;&#402;46% &#226;&#172;&#402;51%, respectively, starved conditions (Fig. <br /> in fed state (data shown).                                                 4D). Therefore, knockdown dCbl expression neurons in- <br />  <br />  <br /> September 2012 Volume 32 Number 18                                                                                                              mcb.asm.org 3615 <br />  Yu et al. <br />  <br />  <br />  <br />  <br /> FIG 5 IPC-specific dCbl knockdown leads elevated dilp expression reduced stress tolerance longevity. (A) The mRNA abundance dilp2, <br /> dilp3, dilp5 assessed qRT-PCR heads male adult dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; dilp2G4&#226;&#172;&#381;dCbl-Ri flies (3 days old, 90/group, n &#226;&#172;&#381; 3 independent <br /> experiments) fed starved state. Normalization using RPL32 internal control. Data represented mean &#226;&#171;&#190; SEM; *, P &#226;&#172;? <br /> 0.05 versus dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; control flies one-way ANOVA. (B) Photo male adult dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; (left) dilp2G4&#226;&#172;&#381;dCbl-Ri (right) flies. The body weight <br /> was measured fed states (3 days age, 90 flies/group), shown mean &#226;&#171;&#190; SEM; *, P &#226;&#172;? 0.05 Student&#226;&#8364;&#8482;s t test. (C) Life span. Survival curves of <br /> male dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; (n &#226;&#171;&#189; 266) dilp2G4&#226;&#172;&#381;dCbl-Ri (n &#226;&#171;&#189; 293) flies. &#226;?&#185;2 &#226;&#171;&#189; 68.10; P &#226;&#172;? 0.0001 log-rank test. (D) Sensitivity paraquat. Male adult flies <br /> at 3 days age treated 30 mM paraquat, survival rates determined (264 dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; flies 265 dilp2G4&#226;&#172;&#381;dCbl-Ri flies). &#226;?&#185;2 &#226;&#171;&#189; 90.12; <br /> P &#226;&#172;? 0.0001 log-rank test. <br />  <br />  <br /> creased whole-body insulin/IGF signaling, likely                  pathway Drosophila controls glucose homeostasis response <br /> upregulation dILP production.                                              nutritional states. To determine dCbl neurons or <br />     dCbl IPCs regulates dilp expression, life span, stress             IPCs regulates carbohydrate metabolism, measured total <br /> resistance. dILPs expressed secreted mainly 7                 glycogen trehalose levels adult flies. Knockdown neuro- <br /> pairs neurons termed IPCs (4, 6). To investigate           nal expression dCbl decreased whole-body glycogen (by <br /> dCbl regulates production dILPs IPCs, crossed UAS-               &#226;&#172;&#402;28%) trehalose (by &#226;&#172;&#402;41%) male elavG4&#226;&#172;&#381;dCbl-Ri flies <br /> dCbl-RNAi flies dilp2-GAL4 drivers generate dilp2-GAL4/               fed conditions compared elavG4&#226;&#172;&#381;&#226;&#171;&#185; control <br /> UAS-dCbl-RNAi (here denoted dilp2G4&#226;&#172;&#381;dCbl-Ri) flies                   flies (Fig. 6A). IPC-specific knockdown dCbl expression simi- <br /> dCbl expression specifically knocked IPCs. In male                larly decreased glycogen (by &#226;&#172;&#402;24%) trehalose (by &#226;&#172;&#402;27%) in <br /> dilp2G4&#226;&#172;&#381;dCbl-Ri flies, mRNA levels dilp2, dilp3, dilp5             male dilp2G4&#226;&#172;&#381;dCbl-Ri flies compared dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; <br /> increased &#226;&#172;&#402;71%, &#226;&#172;&#402;43%, &#226;&#172;&#402;69%, respectively, fed state             control flies (Fig. 6B). Then, examined effect dCbl defi- <br /> and &#226;&#172;&#402;74%, &#226;&#172;&#402;66%, &#226;&#172;&#402;46%, respectively, starved                  ciency survival rates starvation. Both neuronal <br /> condition compared dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; control flies (Fig. 5A).           IPC-specific knockdown dCbl expression significantly re- <br /> Consistently, dilp2G4&#226;&#172;&#381;dCbl-Ri flies &#226;&#172;&#402;9% increase               duced tolerance food deprivation, leading &#226;&#172;&#402;15% <br /> body weight (Fig. 5B) exhibited significant shortening life      (elavG4&#226;&#172;&#381;dCbl-Ri) &#226;&#172;&#402;12% (dilp2G4&#226;&#172;&#381;dCbl-Ri) decreases in <br /> span (Fig. 5C), &#226;&#172;&#402;14% decrease median life span ob-             median survival times (Fig. 6C D; 7.3 days for <br /> served (71 days dilp2G4&#226;&#172;&#381;dCbl-Ri flies 83 days                     elavG4&#226;&#172;&#381;dCbl-Ri flies 8.6 days elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies; 8.8 days for <br /> dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; flies). Paraquat tolerance experiments showed in-                dilp2G4&#226;&#172;&#381;dCbl-Ri flies 10.0 days dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; flies). These <br /> creased sensitivity oxidative stress male dilp2G4&#226;&#172;&#381;dCbl-Ri flies         data indicate dCbl IPCs regulates carbohydrate metabo- <br /> (Fig. 5D), &#226;&#172;&#402;42% reduction median survival time ob-                 lism starvation resistance upregulation dILP pro- <br /> served (4.4 days dilp2G4&#226;&#172;&#381;dCbl-Ri flies 7.6 days                   duction. <br /> dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; flies). These results suggest dCbl cell auton-                 dCbl regulation dilp expression genetically coupled to <br /> omously regulate dILP production IPCs.                                     dEGFR signaling pathway. To investigate mechanism by <br />     dCbl deficiency neurons IPCs affects carbohydrate me-               dCbl suppresses dilp expression IPCs, examined the <br /> tabolism starvation resistance. The insulin/IGF signaling                 role dEGFR pathway overexpressing dominant-nega- <br />  <br />  <br /> 3616    mcb.asm.org                                                                                                           Molecular Cellular Biology <br />                                                                                                                     Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br />  <br /> FIG 6 Metabolic effects neuronal IPC-specific suppression dCbl expression. (A B) Whole-body glycogen trehalose levels measured male <br /> adult flies fed conditions (3 days age, 10 flies/group, n &#226;&#172;&#381; 3 independent experiments). Values elavG4&#226;&#172;&#381;&#226;&#171;&#185; versus elavG4&#226;&#172;&#381;dCbl-Ri flies (A) or <br /> dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; versus dilp2G4&#226;&#172;&#381;dCbl-Ri flies (B) normalized total protein levels. Data presented mean &#226;&#171;&#190; SEM; *, P &#226;&#172;? 0.05 versus &#226;&#172;&#381;&#226;&#171;&#185; control <br /> group one-way ANOVA. (C D) Sensitivity starvation. Survival curves starved male adult flies 3 days age determined elavG4&#226;&#172;&#381;&#226;&#171;&#185; flies <br /> (n &#226;&#171;&#189; 226) versus elavG4&#226;&#172;&#381;dCbl-Ri flies (n &#226;&#171;&#189; 205) (C) dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; flies (n &#226;&#171;&#189; 137) versus dilp2G4&#226;&#172;&#381;dCbl-Ri flies (n &#226;&#171;&#189; 133) (D). &#226;?&#185;2 &#226;&#171;&#189; 163.5 (C) &#226;?&#185;2 &#226;&#171;&#189; 29.66 <br /> (D); P &#226;&#172;? 0.0001 versus &#226;&#172;&#381;&#226;&#171;&#185; control group log-rank test. <br />  <br />  <br />  <br /> tive form EGFR (dEGFR-DN) (12) pan-neurons IPCs.                      regulation <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>cell survival</span> proliferation (28, 33, 63), first <br /> First, crossed UAS-dEGFR-DN flies UAS-dCbl-RNAi flies                  utilized UAS-srcGFP line produce dilp2G4&#226;&#172;&#381;GFP, dCbl- <br /> to generate UAS-dCbl-RNAi; dEGFR-DN flies. Subsequently,                    RNAi dilp2G4&#226;&#172;&#381;GFP, dEGFR-DN flies tested whether <br /> crossed UAS-dCbl-RNAi; dEGFR-DN flies elav-GAL4                           dCbl suppression blocking dEGFR signaling affects sur- <br /> drivers produce elavG4&#226;&#172;&#381;dCbl-RNAi; dEGFR-DN (here de-                        vival IPCs. GFP-facilitated visualization showed alterations in <br /> noted elavG4&#226;&#172;&#381;dRi, DN) flies dEGFR signaling spe-                  morphology number IPCs dilp2G4&#226;&#172;&#381;GFP, dCbl-RNAi <br /> cifically inhibited neurons dEGFR-DN overexpression                 dilp2G4&#226;&#172;&#381;GFP, dEGFR-DN flies compared dilp2G4&#226;&#172;&#381;GFP <br /> in context neuron-specific dCbl knockdown. In comparison                controls (Fig. 8A). Consistently, IPC-specific overexpression of <br /> with elavG4&#226;&#172;&#381;&#226;&#171;&#185; controls, male elavG4&#226;&#172;&#381;dEGFR-DN flies                      dEGFR-DN reduced body weight (Fig. 8B) ex- <br /> &#226;&#172;&#402;13% body weight decrease, neuronal dEGFR-DN overex-                       pression dilp2 dilp3 (Fig. 8C) male dilp2G4&#226;&#172;&#381;dEGFR-DN <br /> pression elavG4&#226;&#172;&#381;dRi, DN flies blunted dCbl deficiency-depen-                flies blunted dCbl deficiency-dependent increases body <br /> dent increases body weight (Fig. 7A). In accordance ERK                weight (Fig. 8B) upregulation dilp2 dilp3 (Fig. 8C) male <br /> phosphorylation activation key component EGFR sig-                 dilp2G4&#226;&#172;&#381;dRi, DN flies. <br /> naling pathway (50), elavG4&#226;&#172;&#381;dEGFR-DN flies exhibited marked                        Taken together, genetic interaction analyses demonstrate <br /> reductions phosphorylation dERK dAkt head;                dCbl suppresses dilp2 dilp3 expression modulating <br /> and neuronal dEGFR-DN overexpression largely reversed dCbl                     EGFR signaling IPCs affecting growth survival of <br /> deficiency-promoted increases phosphorylation levels                  IPCs. In addition, inability dEGFR-DN affect expres- <br /> (Fig. 7B). Importantly, male elavG4&#226;&#172;&#381;dEGFR-DN flies showed sig-                 sion dilp5 suggests additional pathways involved in <br /> nificant reductions mRNA levels dilp2 (by &#226;&#172;&#402;52%                   dCbl-mediated suppression dILP production. <br /> &#226;&#172;&#402;49% normalized level RPL32 &#226;?&#163;-Tub84B,                           Mammalian c-Cbl regulates insulin production secre- <br /> respectively) dilp3 (by &#226;&#172;&#402;39% &#226;&#172;&#402;53% normalized                   tion INS-1 &#226;?&#164; cells. To determine dCbl regulation of <br /> the level RPL32 &#226;?&#163;-Tub84B, respectively) dilp5                   dILP production evolutionarily conserved, examined the <br /> (Fig. 7C). Furthermore, dCbl deficiency-dependent elevations                effect suppression c-Cbl expression insulin production in <br /> the expression dilp2 dilp3 dilp5 significantly             rat insulinoma INS-1 cells. In accordance established role <br /> blocked neuronal dEGFR-DN overexpression male                            c-Cbl negatively regulating EGFR signaling, knockdown of <br /> elavG4&#226;&#172;&#381;dRi, DN flies (Fig. 7C).                                                c-Cbl siRNAs (si-c-Cbl) substantially increased level of <br />     To confirm dCbl regulation dilp expression in-             EGFR phosphorylation ERK INS-1 &#226;?&#164; cells (Fig. 9A). <br /> volves dEGFR pathway specifically IPCs, crossed UAS-dCbl-            Importantly, knockdown c-Cbl significantly increased the <br /> RNAi; dEGFR-DN flies dilp2-GAL4 drivers generate                       mRNA abundance Ins-1 Ins-2 (Fig. 9B) as <br /> dilp2G4&#226;&#172;&#381;dCbl-Ri; dEGFR-DN (here denoted dilp2G4&#226;&#172;&#381;dRi, DN)                       cellular insulin contents (by &#226;&#172;&#402;40%) (Fig. 9C). Furthermore, <br /> flies dEGFR-DN overexpressed face dCbl                  knockdown c-Cbl increased glucose-stimulated insulin secre- <br /> knockdown IPCs. As dCbl EGFR implicated                tion &#226;&#172;&#402;51% (from &#226;&#172;&#402;1.83- &#226;&#172;&#402;2.77-fold) (Fig. 9D). To test <br />  <br />  <br /> September 2012 Volume 32 Number 18                                                                                                        mcb.asm.org 3617 <br />  Yu et al. <br />  <br />  <br />  <br />  <br /> FIG 7 Blocking dEGFR signaling reverses effects neuronal dCbl deficiency. (A) Male adult elavG4&#226;&#172;&#381;&#226;&#171;&#185;, elavG4&#226;&#172;&#381;dCbl-Ri, elavG4&#226;&#172;&#381;dEGFR-DN, and <br /> elavG4&#226;&#172;&#381;dCbl-Ri, EGFR-DN (dRi,DN) flies 3 days age. The body weight measured fed states (90 flies/group). (B) Western immunoblot analysis <br /> of phosphorylation levels dAkt dERK head protein extracts fed male adult flies indicated genotype (90 flies/group). Drosophila <br /> &#226;?&#163;-tubulin used loading control. Representative results shown independent experiments. (C) The mRNA abundance dilp2, dilp3, dilp5 <br /> in head male adult flies determined qRT-PCR (90 flies/group, n &#226;&#171;&#189; 4 5 independent experiments). Normalization using RPL32 or <br /> &#226;?&#163;-Tub84B internal control. Data panels A C shown mean &#226;&#171;&#190; SEM. *, P &#226;&#172;? 0.05 versus elavG4&#226;&#172;&#381;&#226;&#171;&#185; control; #, P &#226;&#172;? 0.05 between <br /> elavG4&#226;&#172;&#381;dCbl-Ri elavG4&#226;&#172;&#381;dRi, DN one-way ANOVA. <br />  <br />  <br /> whether c-Cbl inhibits insulin transcription, cotransfected                  tory effect, c-Cbl knockdown substantially augmented ability <br /> INS-1 cells si-c-Cbl RNAs              EGF increase RIP transcriptional activity (Fig. 9F). <br /> plasmid expressing luciferase reporter control rat                Insulin transcription known upregulated tran- <br /> insulin 1 promoter (RIP). Suppression c-Cbl expression re-                   scription factors subject modulation ERK signaling <br /> sulted 3- 4-fold enhancement RIP transcriptional ac-               (23), including PDX-1. We performed ChIP experiments ex- <br /> tivity (Fig. 9E). While EGF showed significant stimula-                amine effect c-Cbl suppression binding PDX-1 to <br />  <br />  <br />  <br />  <br /> FIG 8 Inhibition dEGFR signaling reverses effects IPC-specific dCbl knockdown affecting IPCs. (A) The brain IPCs visualized using GFP <br /> in male adult dilp2G4&#226;&#172;&#381;GFP, dilp2G4&#226;&#172;&#381;GFP, dCbl-RNAi, dilp2G4&#226;&#172;&#381;GFP, dEGFR-DN flies 3 days age. Shown IPC bodies localized medially the <br /> pars intercerebralis. (B) The body weight male adult flies measured fed states (90 flies/group). (C) The mRNA abundance dilp2, dilp3, dilp5 <br /> in head male adult flies (90 flies/group, n &#226;&#172;&#381; 3 independent experiments) determined qRT-PCR using RPL32 &#226;?&#163;-Tub84B internal control <br /> for normalization. Data panels B C shown mean &#226;&#171;&#190; SEM; *, P &#226;&#172;? 0.05 versus dilp2G4&#226;&#172;&#381;&#226;&#171;&#185; control; #, P &#226;&#172;? 0.05 dilp2G4&#226;&#172;&#381;dCbl-RNAi and <br /> dilp2G4&#226;&#172;&#381;dRi, DN one-way ANOVA. <br />  <br />  <br />  <br /> 3618    mcb.asm.org                                                                                                              Molecular Cellular Biology <br />                                                                                                                          Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br />  <br /> FIG 9 Knockdown c-Cbl expression enhances insulin production INS-1 &#226;?&#164; cells. (A D) INS-1 cells maintained 16.7 mM glucose transfected <br /> for 48 h oligonucleotide control random sequence (small interfering control [si-Con]) oligonucleotides (si-c-Cbl_#1 si-c-Cbl_#2) were <br /> directed c-Cbl. (A) Protein abundance c-Cbl EGFR phosphorylation levels Akt ERK analyzed Western immunoblotting using <br /> the indicated antibodies. &#226;?&#163;-Tubulin used loading control. (B) The abundance Ins-1 Ins-2 mRNA determined qRT-PCR. GAPDH used <br /> as internal control normalization. (C) Intracellular insulin content measured ELISA. For panels B C, data shown mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; <br /> 3 independent experiments); *, P &#226;&#172;? 0.05 versus small interfering control Student&#226;&#8364;&#8482;s t test. (D) Secreted insulin measured ELISA. Transfected cells were <br /> precultured 2.8 mM glucose 2 h subjected stimulation 16.7 mM glucose. Data represent mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); <br /> *, P &#226;&#172;? 0.05 versus small interfering control; #, P &#226;&#172;? 0.05 versus values 2.8 mM glucose two-way ANOVA. (E F) INS-1 cells cultured 16.7 mM glucose <br /> were cotransfected 48 h small interfering control, si-c-Cbl_#1, si-c-Cbl_#2, plasmid expressing RIP-Luc reporter. (E) Luciferase <br /> activities directly measured cell extracts. (F) Luciferase activities determined cells preincubated medium containing 2.8 mM glucose <br /> and 1% fetal bovine serum cultured PBS 200 ng/ml EGF 6 h. Results shown mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); *, P <br /> &#226;&#172;? 0.05 versus small interfering control; #, P &#226;&#172;? 0.05 versus PBS control one-way ANOVA. (G) ChIP assays. INS-1 cells cultured 16.7 mM glucose were <br /> transfected 48 h small interfering control, si-c-Cbl_#1, si-c-Cbl_#2. Chromatin extracts immunoprecipitated anti-PDX-1 antibody IgG. <br /> The precipitated genomic DNA subjected qPCR analysis triplicate using primers Ins-1 promoter. Cellular chromatin extract used the <br /> input control normalization. Data shown mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); *, P &#226;&#172;? 0.05 versus small interfering control one-way <br /> ANOVA. <br />  <br />  <br />  <br /> the insulin promoter. Quantitative PCR assessment revealed sig-                   transcriptional activity (Fig. 11E). In contrast, blocking Akt phos- <br /> nificant increases PDX-1 abundance bound Ins-1 pro-                     phorylation PI3K inhibitor LY294002 (Fig. 11F) did not <br /> moter region result c-Cbl knockdown INS-1 cells                   influence effect c-Cbl knockdown RIP activity (Fig. <br /> were transfected si-c-Cbl RNAs (Fig. 9G).                                    11G). As ERKs reported regulate transcription of <br />     In contrast observations c-Cbl-knockdown cells,                   insulin genes (23, 68), data indicate c-Cbl controls in- <br /> transient overexpression wild-type (WT) c-Cbl                   sulin production downregulation ERK-mediated sig- <br /> E3 ligase-defective mutant (C381A) significantly reduced                      naling pancreatic &#226;?&#164; cells. <br /> transcriptional activity RIP INS-1 cells (Fig. 10A). Addition- <br /> ally, adenovirus-mediated overexpression c-Cbl-WT                      DISCUSSION <br /> c-Cbl-C381A decreased insulin contents INS-1 cells (Fig.                   In Drosophila, single dCbl gene encodes forms dCbl pro- <br /> 10B). Thus, results suggest c-Cbl negatively regu-                teins, evolved homologues, c-Cbl, Cbl-b, and <br /> lates insulin biosynthesis secretion downregulating                   Cbl-c, mammals (60). As multidomain adaptor proteins with <br /> EGFR signaling mammalian &#226;?&#164; cells, revealing regulatory ac-                   intrinsic E3 ubiquitin ligase activities, c-Cbl Cbl-b been <br /> tion Cbl evolutionarily conserved.                                     implicated diverse physiological processes, displaying charac- <br />     Activation ERK mediates effect c-Cbl insulin pro-                teristics functional redundancy certain circumstances. <br /> duction INS-1 cells. Activation ERKs constitutes key signal               However, understanding detailed mechanisms un- <br /> transduction step EGFR actions (65). To determine                      derlie physiological actions dCbl limited, largely <br /> ERK mediates c-Cbl action, examined effect                          myriad signaling pathways subject the <br /> PD98059, inhibitor ERK-specific kinase MEK, INS-1 cells.                 positive negative influences Cbl proteins. In present <br /> PD98059 blocked ERK phosphorylation, expected (Fig. 11A);                      study, utilization genetic models Drosophila enabled to <br /> importantly, PD98059 abolished c-Cbl knockdown-induced                       examine molecular evolution Cbl actions. Our findings re- <br /> increases insulin contents (Fig. 11B) glucose-stimulated                   vealed functional importance dCbl IPCs brains <br /> insulin secretion (Fig. 11C). Furthermore, inhibition ERK (Fig.                flies. dCbl cell autonomously inhibits production dILPs <br /> 11D) abrogated ability c-Cbl deficiency enhance RIP                 IPCs comprises additional layer coordination in- <br />  <br />  <br /> September 2012 Volume 32 Number 18                                                                                                             mcb.asm.org 3619 <br />  Yu et al. <br />  <br />  <br />  <br />                                                                                   downregulating RTK pathways, platelet-de- <br />                                                                                   rived growth factor (PDGF) receptor signaling pathway (7). <br />                                                                                   PDGF signaling reported influenced c-Cbl <br />                                                                                   (35). Additional mechanisms contribute ob- <br />                                                                                   served phenotypes flies dCbl deficiency neurons or <br />                                                                                   IPCs enhancement insulin secretion INS-1 <br />                                                                                   cells. <br />                                                                                       The molecular mechanisms dilp genes tran- <br />                                                                                   scriptionally controlled poorly understood Drosophila. <br />                                                                                   Whereas neuronal IPC-specific dCbl suppression affected <br />                                                                                   expression dilp genes examined, EGFR signaling <br />                                                                                   genetically coupled regulation dilp2 dilp3 but <br />                                                                                   dilp5. This supports notion multiple sig- <br />                                                                                   naling mechanisms involved mediating dCbl regula- <br />                                                                                   tion dILP production. On interesting note, distinct pat- <br /> FIG 10 Effects c-Cbl overexpression insulin production INS-1 &#226;?&#164; cells.    terns dilp gene expression reported recent <br /> (A) INS-1 cells maintained 16.7 mM glucose cotransfected 48 h         study implicated dERK signaling mediating SNF (short <br /> with vector (control [Con]) plasmids encoding WT E3        neuropeptide F) regulation dilp expression IPCs of <br /> ligase-deficient mutant (C381A) c-Cbl protein RIP-Luc re- <br /> porter. Cell lysates analyzed Western immunoblotting indi-       Drosophila (25). In mammalian &#226;?&#164; cells, number transcrip- <br /> cated antibodies. Luciferase activities measured, results shown   tion factors, PDX-1, MafA, NFAT, act regulate <br /> the mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); *, P &#226;&#172;? 0.05 versus            insulin expression response ERK activation (21, 24). <br /> control one-way ANOVA. (B) INS-1 cells maintained 16.7 mM glucose           While observed c-Cbl deficiency-elicited augmentation of <br /> were infected 48 h control adenovirus-GFP (Ad-GFP; MOI of <br /> 20) adenoviruses expressing indicated forms c-Cbl protein (at an <br />                                                                                   PDX-1 binding insulin promoter INS-1 cells, re- <br /> MOI 20). Cell lysates analyzed immunoblotting, intracellular       mains unclear factors implicated. In <br /> insulin content measured ELISA. Values shown mean &#226;&#171;&#190; SEM         addition, defined transcription factors <br /> (n &#226;&#171;&#189; 3 independent experiments); *, P &#226;&#172;? 0.05 compared adenovirus-          dEGFR pathway, e.g., PntP1 (13, 70), PDX-1 ana- <br /> GFP control one-way ANOVA. <br />                                                                                   logue act mediate dCbl suppression dILP production <br />                                                                                   Drosophila. In context, molecular evolution the <br />                                                                                   regulatory factors involved transcriptional control of <br /> sulin/IGF regulation growth, carbohydrate metabolism, stress <br />                                                                                   insulin/IGF1 represents intriguing question warrants <br /> resistance, longevity. <br />                                                                                   detailed investigations. <br />     We dCbl IPCs Drosophila controls the <br />                                                                                       Recent studies suggest common ancestral origin both <br /> biosynthesis dILPs. dILPs ligands insulin/IGF-like <br />                                                                                   mammalian hypothalamic neurosecretory cells IPCs of <br /> signaling pathway regulates growth, development, metabo- <br /> lism, life span (4, 5). We demonstrated disruption dCbl               flies (61). Global c-Cbl-<span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> mice (37) heterozygous <br /> in neurons IPCs resulted upregulation dILPs, increased                   mice expressing C379A knock-in <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> RING <br /> body growth, higher sensitivity oxidative stress starvation,                finger domain c-Cbl (38) higher energy expenditure <br /> decreased levels trehalose, shortened life span. The longev-               improved peripheral insulin sensitivity. Given central <br /> ity metabolic phenotypes explained increased dILP                   importance hypothalamic circuitries control of <br /> production signaling. We observed c-Cbl, mam-                     whole-body energy balance (22), tempting speculate that <br /> malian homologue, similarly inhibited insulin production se-                  c-Cbl Cbl-b exert metabolic effects modulating <br /> cretion rat pancreatic &#226;?&#164; cells. However, whole-body deletion                hypothalamic neuroendocrine networks. Besides EGFR, <br /> c-Cbl does affect blood insulin levels mice (60),                  neuronal Cbl target signaling molecules, as <br /> abrogation Cbl-b leads increase insulin levels presum-                Src homology 2B (SH2B), interact Cbl. We and <br /> ably insulin resistance resulting adipose inflamma-                   previously shown dSH2B, adaptor protein <br /> tion (18). Thus, c-Cbl Cbl-b likely suppress insulin              positively regulates insulin/IGF1 leptin signaling <br /> expression secretion, deficiency form func-                 pathways (40, 45, 46), plays crucial roles metabolism, stress <br /> tionally compensated form mice. Conditional                   resistance, longevity flies (54). SH2B1, SH2B family <br /> gene-targeting studies mouse models needed clarify                 member, required maintenance normal body <br /> issue.                                                                            weight glucose homeostasis humans mice (2, <br />     It established Cbl downregulates                       39). It deciphered dCbl exerts actions <br /> EGFR signaling pathway E3 ubiquitin ligase activity                    metabolism, stress tolerance, life span inter- <br /> both insects mammals (51, 55). EGFR signaling                        acting dSH2B regulate insulin/IGF1 pathway. <br /> documented play crucial role postnatal growth (34)                         In summary, knockdown dCbl neurons <br /> well expansion pancreatic &#226;?&#164;-cell mass response                    IPCs results increased dILP production and/or signaling <br /> feeding high-fat diet pregnancy (16). The Cbl                      activation, leading reduced life span stress tolerance. <br /> family members likely negatively regulate insulin biosynthesis                    dCbl suppresses dILP production, part, down- <br /> and secretion insects mammals downregulating                       regulating EGFR/ERK pathway E3 ligase activ- <br /> the EGFR/ERK pathway E3 ubiquitin ligase activity.                      ity. Moreover, Cbl suppression insulin biosynthesis evo- <br /> Cbl regulate growth proliferation &#226;?&#164; cells                  lutionarily conserved mammalian &#226;?&#164; cells, suggesting the <br />  <br />  <br /> 3620    mcb.asm.org                                                                                                       Molecular Cellular Biology <br />                                                                                                                       Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br />  <br /> FIG 11 c-Cbl affects insulin production ERK-dependent fashion &#226;?&#164; cells. (A C) INS-1 cells maintained 16.7 mM glucose transfected 48 h <br /> with small interfering control si-c-Cbl_#1 si-c-Cbl_#2. (A B) Transfected cells treated 2 h dimethyl sulfoxide (DMSO) 20 &#226;?&#174;M <br /> ERK-specific kinase MEK inhibitor PD98059. (A) Protein abundance c-Cbl EGFR phosphorylation levels Akt ERK analyzed by <br /> Western immunoblotting indicated antibodies. &#226;?&#163;-Tubulin used loading control. (B) Intracellular insulin content determined ELISA. <br /> Data shown mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); *, P &#226;&#172;? 0.05 versus small interfering control two-way ANOVA. (C) Secreted insulin was <br /> measured ELISA. Transfected cells precultured 2.8 mM glucose 2 h subjected stimulation 16.7 mM glucose presence of <br /> dimethyl sulfoxide 20 &#226;?&#174;M PD98059. Fold stimulation values shown mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); *, P &#226;&#172;? 0.05 versus small <br /> interfering control two-way ANOVA. (D G) INS-1 cells cultured 16.7 mM glucose cotransfected 48 h small interfering control, <br /> si-c-Cbl_#1, si-c-Cbl_#2, RIP-Luc reporter plasmid. Cells treated dimethyl sulfoxide, 20 &#226;?&#174;M PD98059 (D E), 60 &#226;?&#174;M PI3K <br /> inhibitor LY294002 (F G) 16 h. Western immunoblot analysis protein abundance c-Cbl phosphorylation levels ERK (D) Akt (F). <br /> Luciferase activities measured cell extracts (E G). Results shown mean &#226;&#171;&#190; SEM (n &#226;&#171;&#189; 3 independent experiments); *, P &#226;&#172;? 0.05 versus small <br /> interfering control one-way ANOVA. <br />  <br />  <br />  <br /> Cbl pathway critical insulin production secretion                     1-09, CAS/SAFEA International Partnership Program), the <br /> across animal species.                                                           Science Technology Commission Shanghai Municipality (grant <br />                                                                                  10XD1406400) Y.L. W.L. <br /> ACKNOWLEDGMENTS                                                                      Y.Y., W.L., Y.L. conceived designed experiments. Y.Y., <br />                                                                                  Y.S., S.H., C.Y. performed experiments. Y.Y., Y.S., S.H., L.R., <br /> We thank Bloomington Drosophila Stock Center Vienna Drosoph- <br />                                                                                  W.L., Y.L. analyzed data. Y.Y., L.R., W.L., Y.L. wrote the <br /> ila RNAi Center providing fly stocks information Eric <br />                                                                                  paper. <br /> Rulifson M. German (UCSF) Jian Zhang (University Chicago) <br /> for reagents. We thank Jiansheng Kang Wenxia Li (Institute          REFERENCES <br /> for Nutritional Sciences) Wei Song (Harvard Medical School) for <br />                                                                                   1. Ahmed Z, Smith BJ, Pillay TS. 2000. The APS adapter protein couples the <br /> technical assistance. <br />                                                                                      insulin receptor phosphorylation c-Cbl facilitates ligand- <br />     This work supported grants Ministry Science               stimulated ubiquitination insulin receptor. FEBS Lett. 475:31&#226;&#8364;&#8220;34. <br /> Technology (973 Program grants 2012CB524900 2011CB910900),                    2. Bachmann-Gagescu R, et al. 2010. Recurrent 200-kb deletions 16p11.2 <br /> the National Natural Science Foundation (grants 30970584, 81021002,                  include SH2B1 gene associated developmental delay and <br /> 30988002, 30830033), Chinese Academy Sciences (The Knowl-                 obesity. Genet. Med. 12:641&#226;&#8364;&#8220; 647. <br /> edge Innovation Programs, grants KSCX2-EW-R-09 KSCX2-EW-Q-                    3. Barbieri M, Bonafe M, Franceschi C, Paolisso G. 2003. Insulin/IGF-I- <br />  <br />  <br />  <br /> September 2012 Volume 32 Number 18                                                                                                          mcb.asm.org 3621 <br />  Yu et al. <br />  <br />  <br />  <br />       signaling pathway: evolutionarily conserved mechanism longevity                beta-elemene-induced cell apoptosis lung cancer cells. Mol. Med. Re- <br />       yeast humans. Am. J. Physiol. Endocrinol. Metab. 285:E1064 &#226;&#8364;&#8220;                 port 4:1243&#226;&#8364;&#8220;1246. <br />       E1071.                                                                         29.   Liu G, Rogers J, Murphy CT, Rongo C. 2011. EGF signalling activates the <br />  4.   Brogiolo W, et al. 2001. An evolutionarily conserved function                 ubiquitin proteasome modulate C. elegans lifespan. EMBO J. <br />       Drosophila insulin receptor insulin-like peptides growth control.             30:2990 &#226;&#8364;&#8220;3003. <br />       Curr. Biol. 11:213&#226;&#8364;&#8220;221.                                                        30.   Liu J, DeYoung SM, Hwang JB, O&#226;&#8364;&#8482;Leary EE, Saltiel AR. 2003. The roles <br />  5.   Broughton SJ, et al. 2005. Longer lifespan, altered metabolism, stress           Cbl-b c-Cbl insulin-stimulated glucose transport. J. Biol. Chem. <br />       resistance Drosophila ablation cells making insulin-like ligands.         278:36754 &#226;&#8364;&#8220;36762. <br />       Proc. Natl. Acad. Sci. U. S. A. 102:3105&#226;&#8364;&#8220;3110.                                 31.   Liu J, Kimura A, Baumann CA, Saltiel AR. 2002. APS facilitates c-Cbl <br />  6.   Cao C, Brown MR. 2001. Localization insulin-like peptide brains             tyrosine phosphorylation GLUT4 translocation response insulin <br />       flies. Cell Tissue Res. 304:317&#226;&#8364;&#8220;321.                                          3T3-L1 adipocytes. Mol. Cell. Biol. 22:3599 &#226;&#8364;&#8220;3609. <br />  7.   Chen H, et al. 2011. PDGF signalling controls age-dependent prolifera-         32.   Meisner H, et al. 1997. Interactions Drosophila Cbl epidermal <br />       tion pancreatic [bgr]-cells. Nature 478:349 &#226;&#8364;&#8220;355.                                 growth factor receptors role Cbl R7 photoreceptor cell develop- <br />  8.   Cheng CL, Gao TQ, Wang Z, Li DD. 2005. Role insulin/insulin-like                  ment. Mol. Cell. Biol. 17:2217&#226;&#8364;&#8220;2225. <br />       growth factor 1 signaling pathway longevity. World J. Gastroenterol.        33.   Miettinen P, Ormio P, Hakonen E, Banerjee M, Otonkoski T. 2008. EGF <br />       11:1891&#226;&#8364;&#8220;1895.                                                                        receptor pancreatic beta-cell mass regulation. Biochem. Soc. Trans. <br />  9.   Choi Y, et al. 2002. PTEN, SHIP SHIP2, suppresses                    36:280 &#226;&#8364;&#8220;285. <br />       PI3K/Akt pathway induces growth inhibition apoptosis my-            34.   Miettinen PJ, et al. 2006. Downregulation EGF receptor signaling in <br />       eloma cells. Oncogene 21:5289 &#226;&#8364;&#8220;5300.                                                 pancreatic islets causes diabetes impaired postnatal beta-cell <br /> 10.   Duan L, Reddi AL, Ghosh A, Dimri M, Band H. 2004. The Cbl family                     growth. Diabetes 55:3299 &#226;&#8364;&#8220;3308. <br />       ubiquitin ligases: destructive forces control antigen recep-   35.   Miyake S, Mullane-Robinson KP, Lill NL, Douillard P, Band H. 1999. <br />       tor signaling. Immunity 21:7&#226;&#8364;&#8220;17.                                                     Cbl-mediated negative regulation platelet-derived growth factor recep- <br /> 11.   Ettenberg SA, et al. 1999. cbl-b inhibits EGF-receptor-induced apoptosis             tor-dependent cell proliferation. A critical role Cbl tyrosine kinase- <br />       enhancing ubiquitination degradation activated receptors. Mol.             binding domain. J. Biol. Chem. 274:16619 &#226;&#8364;&#8220;16628. <br />       Cell Biol. Res. Commun. 2:111&#226;&#8364;&#8220;118.                                             36.   Moghal N, Sternberg PW. 1999. Multiple positive negative regulators <br /> 12.   Freeman M. 1996. Reiterative use EGF receptor triggers differenti-            signaling EGF-receptor. Curr. Opin. Cell Biol. 11:190 &#226;&#8364;&#8220;196. <br />       ation cell types Drosophila eye. Cell 87:651&#226;&#8364;&#8220; 660.               37.   Molero JC, et al. 2004. c-Cbl-deficient mice reduced adiposity, <br /> 13.   Gabay L, et al. 1996. EGF receptor signaling induces pointed P1 tran-                higher energy expenditure, improved peripheral insulin action. J. <br />       scription inactivates Yan protein Drosophila embryonic ventral            Clin. Invest. 114:1326 &#226;&#8364;&#8220;1333. <br />       ectoderm. Development 122:3355&#226;&#8364;&#8220;3362.                                           38.   Molero JC, et al. 2006. Genetic ablation c-Cbl ubiquitin ligase <br /> 14.   Galisteo ML, Dikic I, Batzer AG, Langdon WY, Schlessinger J. 1995.                   domain results increased energy expenditure improved insulin <br />       Tyrosine phosphorylation c-cbl proto-oncogene protein product                 action. Diabetes 55:3411&#226;&#8364;&#8220;3417. <br />       association epidermal growth factor (EGF) receptor EGF           39.   Morris DL, Cho KW, Rui L. 2010. Critical role Src homology 2 <br />       stimulation. J. Biol. Chem. 270:20242&#226;&#8364;&#8220;20245.                                         (SH2) domain neuronal SH2B1 regulation body weight and <br /> 15.   Giannakou ME, Partridge L. 2007. Role insulin-like signalling Dro-             glucose homeostasis mice. Endocrinology 151:3643&#226;&#8364;&#8220;3651. <br />       sophila lifespan. Trends Biochem. Sci. 32:180 &#226;&#8364;&#8220;188.                            40.   Morris DL, Cho KW, Zhou Y, Rui L. 2009. SH2B1 enhances insulin <br /> 16.   Hakonen E, et al. 2011. Epidermal growth factor (EGF)-receptor signal-               sensitivity stimulating insulin receptor inhibiting tyrosine <br />       ling needed murine beta cell mass expansion response high-fat           dephosphorylation insulin receptor substrate proteins. Diabetes 58: <br />       diet pregnancy pancreatic duct ligation. Diabetologia              2039 &#226;&#8364;&#8220;2047. <br />       54:1735&#226;&#8364;&#8220;1743.                                                                  41.   Pai LM, Barcelo G, Schupbach T. 2000. D-cbl, negative regulator the <br /> 17.   Hime GR, Dhungat MP, Ng A, Bowtell DD. 1997. D-Cbl, Drosophila                   Egfr pathway, required dorsoventral patterning Drosophila oo- <br />       homologue c-Cbl proto-oncogene, interacts Drosophila                 genesis. Cell 103:51&#226;&#8364;&#8220; 61. <br />       EGF receptor vivo, despite lacking C-terminal adaptor binding sites.        42.   Pai LM, et al. 2006. Differential effects Cbl isoforms Egfr signaling <br />       Oncogene 14:2709 &#226;&#8364;&#8220;2719.                                                              Drosophila. Mech. Dev. 123:450 &#226;&#8364;&#8220; 462. <br /> 18.   Hirasaka K, et al. 2007. Deficiency Cbl-b gene enhances infiltration    43.   Qiu Y, et al. 2010. A crucial role RACK1 regulation glucose- <br />       activation macrophages adipose tissue causes peripheral insulin            stimulated IRE1alpha activation pancreatic beta cells. Sci. Signal. 3:ra7. <br />       resistance mice. Diabetes 56:2511&#226;&#8364;&#8220;2522.                                           doi:10.1126/scisignal.2000514. <br /> 19.   Huang F, Gu H. 2008. Negative regulation lymphocyte development             44.   Rao N, Dodge I, Band H. 2002. The Cbl family ubiquitin ligases: <br />       function Cbl family proteins. Immunol. Rev. 224:229 &#226;&#8364;&#8220;238.              critical negative regulators tyrosine kinase signaling immune <br /> 20.   Ikeya T, Galic M, Belawat P, Nairz K, Hafen E. 2002. <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span>-                       system. J. Leukoc. Biol. 71:753&#226;&#8364;&#8220;763. <br />       dependent expression insulin-like peptides neuroendocrine cells        45.   Ren D, Li M, Duan C, Rui L. 2005. Identification SH2-B key <br />       CNS contributes growth regulation Drosophila. Curr. Biol.               regulator leptin sensitivity, energy balance, body weight mice. <br />       12:1293&#226;&#8364;&#8220;1300.                                                                        Cell Metab. 2:95&#226;&#8364;&#8220;104. <br /> 21.   Khoo S, et al. 2003. Regulation insulin gene transcription ERK1      46.   Ren D, et al. 2007. Neuronal SH2B1 essential controlling energy and <br />       ERK2 pancreatic beta cells. J. Biol. Chem. 278:32969 &#226;&#8364;&#8220;32977.                      glucose homeostasis. J. Clin. Invest. 117:397&#226;&#8364;&#8220; 406. <br /> 22.   Lam TK. 2010. Neuronal regulation homeostasis <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> sensing.          47.   Robertson H, Hime GR, Lada H, Bowtell DD. 2000. A Drosophila <br />       Nat. Med. 16:392&#226;&#8364;&#8220;395.                                                                analogue v-Cbl dominant-negative oncoprotein vivo. Oncogene <br /> 23.   Lawrence M, Shao C, Duan L, McGlynn K, Cobb MH. 2008. The protein                    19:3299 &#226;&#8364;&#8220;3308. <br />       kinases ERK1/2 roles pancreatic beta cells. Acta Physiol. 192:    48.   Rulifson EJ, Kim SK, Nusse R. 2002. Ablation insulin-producing <br />       11&#226;&#8364;&#8220;17.                                                                               neurons flies: growth diabetic phenotypes. Science 296:1118 &#226;&#8364;&#8220;1120. <br /> 24.   Lawrence MC, McGlynn K, Park BH, Cobb MH. 2005. ERK1/2-                        49.   Saltiel AR, Pessin JE. 2003. Insulin signaling microdomains the <br />       dependent activation transcription factors required acute                 plasma membrane. Traffic 4:711&#226;&#8364;&#8220;716. <br />       chronic effects glucose insulin gene promoter. J. Biol. Chem.        50.   Schlessinger J. 2000. Cell signaling receptor tyrosine kinases. Cell 103: <br />       280:26751&#226;&#8364;&#8220;26759.                                                                     211&#226;&#8364;&#8220;225. <br /> 25.   Lee KS, et al. 2008. Drosophila short neuropeptide F signalling regulates      51.   Schmidt MH, Dikic I. 2005. The Cbl interactome functions. Nat. <br />       growth ERK-mediated insulin signalling. Nat. Cell Biol. 10:468 &#226;&#8364;&#8220; 475.             Rev. Mol. Cell Biol. 6:907&#226;&#8364;&#8220;918. <br /> 26.   Levkowitz G, et al. 1999. Ubiquitin ligase activity tyrosine phosphor-     52.   Sehat B, Andersson S, Girnita L, Larsson O. 2008. Identification c-Cbl <br />       ylation underlie suppression growth factor signaling c-Cbl/Sli-1.              new ligase insulin-like growth factor-I receptor distinct roles <br />       Mol. Cell 4:1029 &#226;&#8364;&#8220;1040.                                                              Mdm2 receptor ubiquitination endocytosis. Cancer Res. 68: <br /> 27.   Levkowitz G, et al. 1998. c-Cbl/Sli-1 regulates endocytic sorting                5669 &#226;&#8364;&#8220;5677. <br />       ubiquitination epidermal growth factor receptor. Genes Dev. 12:         53.   Slack C, et al. 2010. Regulation lifespan, metabolism, stress re- <br />       3663&#226;&#8364;&#8220;3674.                                                                           sponses Drosophila SH2B protein, Lnk. PLoS Genet. 6:e1000881. <br /> 28.   Li L, et al. 2011. Cbl-regulated Akt ERK signals involved                 doi:10.1371/journal.pgen.1000881. <br />  <br />  <br />  <br /> 3622    mcb.asm.org                                                                                                                      Molecular Cellular Biology <br />                                                                                                                              Drosophila Cbl Controls Insulin Biosynthesis <br />  <br />  <br />  <br /> 54. Song W, et al. 2010. SH2B regulation growth, metabolism, lon-                   cell differentiation eye development. PLoS One 3:e1447. doi: <br />     gevity insects mammals. Cell Metab. 11:427&#226;&#8364;&#8220; 437.                           10.1371/journal.pone.0001447. <br /> 55. Swaminathan G, Tsygankov AY. 2006. The Cbl family proteins: ring                 64.   Waterman H, Levkowitz G, Alroy I, Yarden Y. 1999. The RING finger of <br />     leaders regulation cell signaling. J. Cell. Physiol. 209:21&#226;&#8364;&#8220; 43.                 c-Cbl mediates desensitization epidermal growth factor receptor. J. <br /> 56. Taguchi A, White MF. 2008. Insulin-like signaling, <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> homeostasis,               Biol. Chem. 274:22151&#226;&#8364;&#8220;22154. <br />     life span. Annu. Rev. Physiol. 70:191&#226;&#8364;&#8220;212.                                   65.   Wetzker R, Bohmer FD. 2003. Transactivation joins multiple tracks to <br /> 57. Tanaka S, Neff L, Baron R, Levy JB. 1995. Tyrosine phosphorylation                 ERK/MAPK cascade. Nat. Rev. Mol. Cell Biol. 4:651&#226;&#8364;&#8220; 657. <br />     translocation c-cbl protein activation tyrosine kinase signal-   66.   Yang L, et al. 2010. Deficiency RNA editing enzyme ADAR2 impairs <br />     ing pathways. J. Biol. Chem. 270:14347&#226;&#8364;&#8220;14351.                                          regulated exocytosis. FASEB J. 24:3720 &#226;&#8364;&#8220;3732. <br /> 58. Taniguchi CM, Emanuelli B, Kahn CR. 2006. Critical nodes signalling           67.   Yoon CH, Lee J, Jongeward GD, Sternberg PW. 1995. Similarity sli-1, <br />     pathways: insights insulin action. Nat. Rev. Mol. Cell Biol. 7:85&#226;&#8364;&#8220;96.             regulator vulval development C. elegans, mammalian proto- <br /> 59. Tatar M, Bartke A, Antebi A. 2003. The endocrine regulation aging                oncogene c-cbl. Science 269:1102&#226;&#8364;&#8220;1105. <br />     insulin-like signals. Science 299:1346 &#226;&#8364;&#8220;1351.                                    68.   Zhang SS, et al. 2009. Coordinated regulation Shp2 tyrosine phospha- <br /> 60. Thien CBF, Langdon WY. 2001. Cbl: adaptations regulate pro-                    tase signaling events controlling insulin biosynthesis pancreatic beta- <br />     tein tyrosine kinases. Nat. Rev. Mol. Cell Biol. 2:294 &#226;&#8364;&#8220;307.                           cells. Proc. Natl. Acad. Sci. U. S. A. 106:7531&#226;&#8364;&#8220;7536. <br /> 61. Wang S, Tulina N, Carlin DL, Rulifson EJ. 2007. The origin islet-like         69.   Zhang W, Thompson BJ, Hietakangas V, Cohen SM. 2011. MAPK/ERK <br />     cells Drosophila identifies parallels vertebrate endocrine axis.             signaling regulates insulin sensitivity control glucose metabolism Dro- <br />     Proc. Natl. Acad. Sci. U. S. A. 104:19873&#226;&#8364;&#8220;19878.                                       sophila. PLoS Genet. 7:e1002429. doi:10.1371/journal.pgen.1002429. <br /> 62. Wang Y, Chen Z, Bergmann A. 2010. Regulation EGFR Notch                   70.   Zhu S, Barshow S, Wildonger J, Jan LY, Jan YN. 2011. Ets transcription <br />     signaling distinct isoforms D-cbl Drosophila development.                 factor Pointed promotes generation intermediate neural progeni- <br />     Dev. Biol. 342:1&#226;&#8364;&#8220;10.                                                                   tors Drosophila larval brains. Proc. Natl. Acad. Sci. U. S. A.  <br /> </body></html>