<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Nat Cell Biol. 2013 May ; 15(5): 491&#226;&#8364;&#8220;501. doi:10.1038/ncb2720. <br />  <br />  <br />  <br />                            DAF-16/FOXO employs chromatin remodeller SWI/SNF <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />                            promote stress resistance longevity <br />                            Christian G. Riedel1,2,6, Robert H. Dowen1,2, Guinevere F. Lourenco1,2, Natalia V. <br />                            Kirienko1,2, Thomas Heimbucher3, Jason A. West1,2, Sarah K. Bowman1,2, Robert E. <br />                            Kingston1,2, Andrew Dillin3, John M. Asara4,5, Gary Ruvkun1,2,7 <br />                            1Department Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, <br />  <br />                            02114, USA <br />                            2Department       Genetics, Harvard Medical School, Boston, Massachusetts, 02114, USA <br />                            3Molecular  Cell Biology Laboratory, The Salk Institute Biological Studies, La Jolla, <br />                            California, 92037, USA <br />                            4Division <br />                                    Signal Transduction, Beth Israel Deaconess Medical Center, Boston, Massachusetts, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            02115, USA <br />                            5Department       Medicine, Harvard Medical School, Boston, Massachusetts, 02115, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  Organisms constantly challenged stresses privations require adaptive responses for <br />                                  <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>survival</span>. The transcription factor DAF-16/FOXO central nexus responses, but <br />                                  despite importance little known regulates target genes. Proteomic <br />                                  identification DAF-16/FOXO binding partners Caenorhabditis elegans subsequent <br />                                  functional evaluation <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) revealed candidate DAF-16/FOXO <br />                                  cofactors, notably chromatin remodeller SWI/SNF. DAF-16/FOXO SWI/SNF form a <br />                                  complex globally colocalize DAF-16/FOXO target promoters. We specifically <br />                                  gene-activation, DAF-16/FOXO depends SWI/SNF, facilitating SWI/SNF recruitment to <br />                                  target promoters, order activate transcription presumed remodelling local chromatin. <br />                                  For animal, translates essential role SWI/SNF DAF-16/FOXO-mediated <br />                                  processes, i.e. dauer formation, stress resistance, promotion longevity. Thus give <br />                                  insight mechanisms DAF-16/FOXO-mediated transcriptional regulation establish a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  critical link ATP-dependent chromatin remodelling lifespan regulation. <br />  <br />  <br />                            Keywords <br />                                  daf-16; FOXO; SWI/SNF; longevity; stress response; dauer formation; chromatin remodelling; <br />                                  proteomics <br />  <br />  <br />  <br />  <br />                            7 <br />                             Correspondence addressed to: G.R. ruvkun@molbio.mgh.harvard.edu. <br />                            6current address: European Research Institute Biology Ageing, University Medical Center Groningen, 9713 AV Groningen, <br />                            Netherlands. <br />                            Author Contributions: <br />                            C.G.R. G.R. conceived designed experiments. C.G.R., G.F.L., N.V.K., J.A.W., J.M.A. conducted experiments. <br />                            R.H.D. analysed mRNA-Seq ChIP-seq data. S.K.B., R.E.K., T.H., A.D. provided unpublished methods, materials, and <br />                            advice. C.G.R. G.R. wrote manuscript. <br />                            Competing Financial Interests: <br />                            The authors declare competing financial interests. <br />                             Riedel et al.                                                                                          Page 2 <br />  <br />  <br />                                Introduction <br />                                            The ability adapt stresses privations crucial survival the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            longevity species. Sophisticated mechanisms place perceive dire <br />                                            conditions relay appropriate responses, i.e. cytoprotective homeostatic <br />                                            measures reversible cessation development reproduction1. A <br />                                            core pathway responses animals environmentally responsive insulin-like <br />                                            signalling pathway conserved downstream component, forkhead transcription <br />                                            factor DAF-16/FOXO2. In presence ample food optimal conditions, high insulin- <br />                                            like signalling inactivates DAF-16/FOXO AKT kinase-mediated phosphorylation, <br />                                            causing cytoplasmic sequestration DAF-16/FOXO 14-3-3 proteins. Conversely, upon <br />                                            dire conditions, i.e. cues reduce insulin-like signalling allow reversal of <br />                                            DAF-16/FOXO phosphorylation, sequestration alleviated, enabling DAF-16/FOXO to <br />                                            translocate nucleus, engages transcriptional regulation. Several hundred <br />                                            DAF-16/FOXO target genes identified, concerted action that <br />                                            confers wide range beneficial effects organism &#226;&#8364;&#8220; notably stress resistance <br />                                            longevity, metabolic responses, stem cell maintenance, tumour <br />                                            suppression3,4. Although studies explored signalling pathways leading to <br />                                            DAF-16/FOXO activation, little known mechanisms cofactors which <br />                                            DAF-16/FOXO regulates target genes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Results <br />                                Identification candidate DAF-16/FOXO cofactors <br />                                            To address problem used nematode Caenorhabditis elegans, model that <br />                                            instrumental dissecting functions DAF-16/FOXO, its <br />                                            amenability genetic manipulation compatibility high-throughput screening <br />                                            approaches developmental arrest, stress resistance lifespan phenotypes. We <br />                                            biochemically characterized DAF-16/FOXO size exclusion chromatography C. <br />                                            elegans lysates, revealed significant DAF-16/FOXO partitions to <br />                                            high molecular weight fractions (Fig. 1a). Activation DAF-16/FOXO daf-2/insulin- <br />                                            like growth factor mutant background increased partitioning DAF-16/FOXO these <br />                                            fractions (Fig. 1a), suggesting DAF-16/FOXO activity involves maybe even <br />                                            requires binding proteins, i.e. cofactors. Hence conducted screen identify <br />                                            cofactors DAF-16/FOXO, combined proteomic identification DAF-16/ <br />                                            FOXO binding partners high-throughput functional assays using RNA interference <br />                                            (RNAi) (Fig. 1b). Epitope-tagged DAF-16/FOXO immunoprecipitated different <br />                                            states activation, using different C. elegans genetic backgrounds: wild-type <br />                                            (DAF-16 partially active), daf-2(e1370ts) restrictive temperature (insulin/IGF receptor <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutant; DAF-16 fully active), daf-18(mg198lf) (PTEN mutant, causing constitutively <br />                                            high PIP3 signalling turn constitutively activates AKT kinases that <br />                                            phosphorylate inactivate DAF-16/FOXO; DAF-16 inactive). Proteins specifically <br />                                            co-purified DAF-16/FOXO identified tandem mass spectrometry (LC-MS/MS; <br />                                            Fig. 1c). In addition previously known DAF-16/FOXO binding partners (i.e. the <br />                                            14-3-3 proteins FTT-2 PAR-55), identified 131 new binding partners DAF-16/ <br />                                            FOXO, majority enriched purifications active DAF-16/FOXO <br />                                            (Table S1). <br />  <br />                                            To identify potential DAF-16/FOXO cofactors binding partners, silenced <br />                                            expression RNAi examined range aging gene expression phenotypes <br />                                            expected altered DAF-16/FOXO activity: To identify cofactors required DAF-16/ <br />                                            FOXO activity, tested gene inactivation impaired DAF-16/FOXO-induced <br />                                            lifespan extension inability up-regulate DAF-16/FOXO-activated gene sod-3 in <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                           Page 3 <br />  <br />  <br />                                            daf-2 mutant animals (Tables 1, S2, S3). To identify interacting proteins antagonize <br />                                            DAF-16/FOXO activity, tested gene inactivation extension lifespan or <br />                                            inappropriate induction sod-3 wild-type animals (Tables S4&#226;&#8364;&#8220;S6). Out 72 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DAF-16/FOXO binding partners tested, inactivation genes encoding 32 them <br />                                            caused significant phenotypes assays. Thus considered <br />                                            candidate cofactors DAF-16/FOXO (Tables 1, S6). Comparison frequency of <br />                                            phenotypes Table S2 similar genome-wide survey6 indicated list of <br />                                            binding partners 20-fold enriched proteins involved DAF-16/FOXO <br />                                            function, underscoring utility co-purification approach. Many 32 candidate <br />                                            cofactors act regulation chromatin transcription (e.g. SWSN-1, SWSN-3, <br />                                            BAF-1, ELB-1, DCP-66, DPY-30, ZFP-1, MRG-1) protein folding/homeostasis (e.g. <br />                                            RPN-9, RPN-12, CCT-8, PFD-6, PFD-2) (Tables 1, S6), important for <br />                                            stress response lifespan regulation7,8. And candidate cofactors even <br />                                            emerged previous aging-related studies, e.g. BAF-1 controls age-dependent muscle <br />                                            integrity9 CCT-8 component lifespan-regulatory cytosolic chaperonin T <br />                                            complex10. <br />  <br />                                SWI/SNF binds colocalizes DAF-16/FOXO <br />                                            While follow-up candidate cofactors yield compelling insight into <br />                                            DAF-16/FOXO mechanism function, focused candidate particularly strong <br />                                            phenotypes both, regulation lifespan expression sod-3: SWSN-1 (ortholog of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            human BAF155/170), core subunit chromatin remodeller SWI/SNF. Two additional <br />                                            SWI/SNF subunits, SWSN-3 (ortholog human BAF57) SWSN-8 (ortholog human <br />                                            OSA/BAF250), 32 candidate cofactors emerged the <br />                                            proteomic analysis (Tables 1, S1). <br />  <br />                                            SWI/SNF essential 1&#226;&#8364;&#8220;2 MDa multi-subunit complex repositions, exchanges, or <br />                                            displaces nucleosomes ATP-dependent manner11,12 (Fig. S1a,b). The BRG1/BRM <br />                                            ortholog SWSN-4 provides complex&#226;&#8364;&#8482;s catalytic activity comprises, with <br />                                            SWSN-1/BAF155/170 SNFC-5/INI1, core subunits13. Additional accessory subunits <br />                                            direct specificity complex. In particular subclasses SWI/SNF, BAF and <br />                                            PBAF, differ presence accessory signature subunits (i.e. SWSN-8/OSA and <br />                                            PBRM-1/Polybromo (Fig. S1a,b)), distinct functions14,15. Although chromatin <br />                                            remodelling ubiquitous process, roles SWI/SNF surprisingly confined. For <br />                                            example, yeast 6% gene expression events depend SWI/SNF16, suggesting <br />                                            particular mechanisms direct SWI/SNF specific sites. <br />  <br />                                            In C. elegans, studies SWI/SNF remained &#226;&#8364;&#8220; focused role in <br />                                            asymmetric T-cell division gonad morphogenesis15,17. To explore relationship <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DAF-16/FOXO SWI/SNF, confirmed interaction DAF-16/ <br />                                            FOXO SWI/SNF subunits (including catalytic core SWSN-4/BRG1) excluded <br />                                            possibility DNA-mediated interaction co-immunoprecipitation (co-IP) the <br />                                            presence Benzonase, nuclease degrades DNA RNA (Fig. 1d,e). Next we <br />                                            determined tissues subcellular compartments DAF-16/FOXO SWI/SNF <br />                                            coexpressed interaction occur. Both expressed globally, <br />                                            including tissues important DAF-16/FOXO function (i.e. intestine neurons)17,18 <br />                                            (Fig. S1c). While inactive DAF-16/FOXO (e.g. daf-18(mg198lf)) sequestered the <br />                                            cytoplasm translocates nucleus activation (e.g. daf-2(e1370ts)), <br />                                            SWI/SNF subunits resided constitutively nucleus15,17,19 (Fig. S1c, data shown). <br />                                            Consistent observation, mass spectrometric comparison DAF-16/FOXO <br />                                            purifications wild-type, daf-2(e1370ts), daf-18(mg198lf) animals revealed a <br />                                            positive correlation activation DAF-16/FOXO binding SWI/SNF <br />                                            (comparison based spectral counts, Table S1). Thus appears DAF-16/FOXO and <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                           Page 4 <br />  <br />  <br />                                            SWI/SNF encounter cell types, interaction preferentially <br />                                            occurring DAF-16/FOXO activation translocation nucleus, i.e. low <br />                                            insulin-like signalling conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                SWI/SNF required DAF-16/FOXO-mediated transcriptional regulation <br />                                            Next evaluated importance SWI/SNF DAF-16/FOXO activity. Since DAF-16/ <br />                                            FOXO exerts function transcriptional regulation target genes, tested whether <br />                                            SWI/SNF required regulation occur. We used transcriptional reporters two <br />                                            DAF-16/FOXO-regulated genes, sod-3 ins-7, examine DAF-16/FOXO-dependent <br />                                            transcriptional activation repression, respectively6,20. Expression Psod-3::GFP <br />                                            reporter induced conditions low insulin-like signalling, e.g. daf-2 mutants18. <br />                                            Additional inactivation RNAi DAF-16/FOXO SWI/SNF core subunits <br />                                            SWSN-4/BRG1, SWSN-1/BAF155/170 BAF-subclass signature subunit SWSN-8/ <br />                                            OSA, suppressed induction Psod-3::GFP (Fig. 2a,b). Likewise, repression of <br />                                            Pins-7::ins-7::GFP suppressed RNAi SWI/SNF subunits, particular <br />                                            SWSN-4/BRG1 SWSN-1/BAF155/170 lesser extent SWSN-8/OSA, SWSN-3/ <br />                                            BAF57, SNFC-5/INI1 (Fig. 2c,d). It important note contrast BAF- <br />                                            subclass signature subunit SWSN-8/OSA, RNAi PBAF-subclass signature <br />                                            subunits PBRM-1/Polybromo, TAG-298/BRD7, SWSN-7/BAF200 (by validated RNAi <br />                                            conditions (Fig. S2)) yielded significant phenotypes assays, suggesting that <br />                                            specifically BAF-like subclass SWI/SNF complex mediates DAF-16/FOXO <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            functions. <br />  <br />                                            We tested SWI/SNF required regulation endogenous DAF-16/ <br />                                            FOXO target genes genome-wide. By high-throughput sequencing mRNA (mRNA-Seq), <br />                                            compared gene expression levels wild-type, daf-2(e1370ts), daf-2(e1370ts); <br />                                            daf-16(0), daf-2(e1370ts); swsn-1(os22ts) young adult animals restrictive <br />                                            temperature. First tested daf-16(0) swsn-1(os22ts) suppress the <br />                                            differential gene expression caused decreased insulin-like signalling daf-2(e1370ts), <br />                                            thought entirely mediated DAF-16/FOXO3. As indicated extensive <br />                                            reversal differential gene expression wild-type levels (78% genes activated and <br />                                            71% genes repressed daf-2(e1370ts) mutant), daf-16(0) largely suppressed gene <br />                                            expression changes induced daf-2(e1370ts) (Fig. 3a,b). Likewise swsn-1(os22ts) <br />                                            significantly reversed differential gene expression, lesser extent (42% of <br />                                            genes activated 32% genes repressed daf-2(e1370ts) mutant reverted; Fig. <br />                                            3a,b). This swsn-1(os22ts) non-null weaker allele than <br />                                            daf-16(0), SWSN-1 important, acting subset DAF-16/FOXO <br />                                            target genes. Next determined overlap DAF-16/FOXO- SWI/SNF- <br />                                            regulated genes. Importantly, DAF-16/FOXO SWI/SNF regulate a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            small fraction genome (5.9% 7.5%, respectively), dependent gene sets <br />                                            showed substantial significant overlap: Of 656 genes activated 356 genes <br />                                            repressed DAF-16/FOXO daf-2(e1370ts) animals, 32% (211) activated 15% <br />                                            (53) repressed genes co-regulated SWI/SNF (Fig. 3c), suggesting large <br />                                            fraction DAF-16/FOXO-mediated gene regulatory events require SWI/SNF. GO-term <br />                                            analysis co-dependent genes showed significant enrichment genes involved in <br />                                            aging, oxidative stress response, organismal defence (Table S7). Focusing DAF-16/ <br />                                            FOXO target genes, specifically genes activated DAF-16/FOXO observed a <br />                                            strong positive correlation extent DAF-16/FOXO-mediated activation and <br />                                            requirement SWI/SNF activation occur (as indicated declining <br />                                            trendline Figure 3d, right panel). No substantial correlation genes non- <br />                                            regulated repressed DAF-16/FOXO (Fig. 3d, left middle panels). Thus SWI/SNF <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                           Page 5 <br />  <br />  <br />                                            required regulation large subset DAF-16/FOXO target genes &#226;&#8364;&#8220; <br />                                            predominantly activated DAF-16/FOXO. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                DAF-16/FOXO SWI/SNF colocalize chromatin <br />                                            We sought mechanism SWI/SNF impacts DAF-16/FOXO-mediated gene <br />                                            regulation. Given SWI/SNF activated DAF-16/FOXO known associate <br />                                            DNA2,11,12, determined genome-wide positioning DAF-16/FOXO, SWI/ <br />                                            SNF catalytic subunit SWSN-4/BRG1, SWI/SNF core regulatory subunit SWSN-1/ <br />                                            BAF155/170 daf-2(e1370ts) animals restrictive temperature using chromatin <br />                                            immunoprecipitation followed high-throughput sequencing (ChIP-Seq). Using strains <br />                                            expressing DAF-16::GFP, SWSN-4::GFP, SWSN-1::GFP, identified 1952 <br />                                            binding sites DAF-16/FOXO, 3804 binding sites SWSN-4/BRG1, 5727 binding <br />                                            sites SWSN-1/BAF155/170. Despite use multi-copy transgenes express <br />                                            non-physiological levels, observed reassuring enrichment canonical DAF-16/ <br />                                            FOXO associated motif TRTTTAC3 DAF-16/FOXO binding sites. SWSN-4/BRG1 and <br />                                            SWSN-1/BAF155/170 binding sites likewise enriched motifs, although <br />                                            motifs defined roles (Table S8). Interestingly, SWI/SNF DAF-16/ <br />                                            FOXO binding sites shared enrichment motif known associate Trithorax-like, a <br />                                            protein Drosophila melanogaster functionally related SWI/SNF21 (Table S8). <br />                                            DAF-16/FOXO binding sites daf-2(e1370ts) correlated DAF-16/FOXO binding <br />                                            sites previously identified wild-type animals4 (Fig. S3a). And DAF-16/FOXO as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            SWI/SNF binding sites located promoter regions, consistent these <br />                                            proteins functioning transcriptional regulation (Fig. 4a). Genes immediately downstream <br />                                            DAF-16/FOXO binding sites strongly enriched DAF-16/FOXO-activated genes, <br />                                            depleted DAF-16/FOXO-repressed genes (Fig. 4b). 87% directly <br />                                            regulated genes experienced activation 13% repression DAF-16/FOXO, <br />                                            suggesting DAF-16/FOXO predominantly transcriptional activator. <br />  <br />                                            Given DAF-16/FOXO SWI/SNF form complex (Fig. 1d,e, Table S1), tested if <br />                                            colocalize chromatin. Indeed, SWSN-4/BRG1 SWSN-1/ <br />                                            BAF155/170 strongly enriched right summits DAF-16/FOXO binding sites <br />                                            (Fig. 4c,e), supporting model DAF-16/FOXO-SWI/SNF interaction. And fact that <br />                                            colocalization occurred majority DAF-16/FOXO binding sites <br />                                            (Fig. 4d,e), supports intimate connection DAF-16/FOXO this <br />                                            cofactor. <br />  <br />                                DAF-16/FOXO recruits SWI/SNF directly activated target genes <br />                                            There substantial precedence transcription factors employ chromatin remodellers as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cofactors12. In case SWI/SNF, transcription factors recruit target promoters <br />                                            induce local nucleosome repositioning alters accessibility cis-regulatory <br />                                            promoter elements regulate transcription22. In order test DAF-16/FOXO employs <br />                                            SWI/SNF similar manner, investigated binding DAF-16/FOXO <br />                                            target promoters depends other. Inactivation SWI/SNF use swsn-1(os22ts) <br />                                            restrictive temperature effect DAF-16/FOXO expression, nuclear localization, <br />                                            binding target promoters (Fig. 5a, S3b,c). Next looked SWI/SNF <br />                                            catalytic core subunit, SWSN-4/BRG1. Again, loss DAF-16/FOXO did alter <br />                                            SWSN-4/BRG1 expression levels (Fig. S3b), nuclear localization (Fig. S3c), did we <br />                                            observe significant changes abundance SWSN-4/BRG1 promoters were <br />                                            non-regulated repressed DAF-16/FOXO (Fig. 5b). However, specifically a <br />                                            large fraction promoters directly bound activated DAF-16/FOXO (e.g. <br />                                            promoters sod-3, ctl-3, hil-1) observed substantial loss SWSN-4/BRG1 binding <br />                                            absence DAF-16/FOXO (Fig. 5b,d, S4a). As additional control, investigated <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                         Page 6 <br />  <br />  <br />                                            promoters activated DAF-16/FOXO (as judged mRNA-Seq) were <br />                                            lacking DAF-16/FOXO binding site, assuming indirect targets of <br />                                            DAF-16/FOXO, e.g. regulated transcription factors events downstream of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DAF-16/FOXO. Here observed change SWSN-4/BRG1 binding loss of <br />                                            DAF-16/FOXO, suggesting SWSN-4/BRG1 recruitment directly DAF-16/FOXO- <br />                                            activated promoters controlled physical presence DAF-16/FOXO not <br />                                            mere events transcriptional activation (Fig. 5b). Findings SWSN-4/BRG1 were <br />                                            confirmed analysis SWSN-1/BAF155/170, yielding similar results (Fig. 5c, S3b, S4a, <br />                                            data shown). Results additionally confirmed replicated conventional ChIP- <br />                                            qPCR experiments (Fig. S4b). We conclude DAF-16/FOXO employ SWI/SNF a <br />                                            cofactor manner previously described transcription factors, we <br />                                            infer following model DAF-16/FOXO activate target genes: DAF-16/ <br />                                            FOXO binds promoters independent SWI/SNF, presumably directly accessing its <br />                                            target sequences, promoters DAF-16/FOXO directly activates, aids the <br />                                            recruitment SWI/SNF, inducing local chromatin remodelling. This remodelling <br />                                            enhances accessibility activatory cis-regulatory promoter elements binding by <br />                                            downstream components (e.g. transcription machinery), resulting in <br />                                            transcriptional activation (Fig. S5). <br />  <br />                                SWI/SNF required DAF-16/FOXO-mediated dauer formation <br />                                            Given collective evidence DAF-16/FOXO employs SWI/SNF gene-activation- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            specific cofactor, wondered importance cofactor various <br />                                            DAF-16/FOXO-mediated functions animal. In C. elegans, DAF-16/FOXO required <br />                                            entry dauer state, developmental arrest diapause state allows for <br />                                            survival adverse conditions23. We induced state inactivating insulin-like <br />                                            signalling tested RNAi SWI/SNF subunits able suppress it. <br />                                            Consistent SWI/SNF important cofactor DAF-16/FOXO, loss of <br />                                            daf-16 SWI/SNF subunits prevented formation SDS-resistant dauer <br />                                            larvae (Fig. 6a). Upon closer investigation, absence SWI/SNF animals <br />                                            attempted dauer entry, execution dauer program failed, leading instead a <br />                                            DAF-16/FOXO-dependent developmental arrest L3 stage (Fig. S6a). These <br />                                            arrested animals lacked typical longevity dauer larvae frequently void of <br />                                            dauer-specific anatomical features hypodermal alae pharyngeal plug (Fig. <br />                                            S6b&#226;&#8364;&#8220;d). Consistent gene expression data (Fig. 2b,d), defective dauer formation was <br />                                            specifically observed loss core SWI/SNF subunits SWSN-4/BRG1 and <br />                                            SWSN-1/BAF155/170 BAF signature subunit SWSN-8/OSA, loss PBAF <br />                                            signature subunits PBRM-1/Polybromo, TAG-298/BRD7, SWSN-7/BAF200 did not <br />                                            disrupt dauer formation. This implies specific role BAF-like subclass of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            SWI/SNF mediating DAF-16/FOXO functions. <br />  <br />                                            Given loss-of-function <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> SWI/SNF, tested overexpression <br />                                            SWI/SNF subunits SWSN-4/BRG1 SWSN-1/BAF155/170 promote dauer <br />                                            formation. Although gain-of-function phenotypes approach difficult to <br />                                            obtain interpret multi-subunit nature SWI/SNF complex its <br />                                            requirement targeted appropriate sites, observed daf-16-dependent <br />                                            moderate enhancement dauer formation daf-2(e1370ts) animals 22&#194;&#176;C (Fig. S6e). <br />  <br />                                SWI/SNF required DAF-16/FOXO-mediated longevity stress resistance <br />                                            Beyond role dauer formation, DAF-16/FOXO potent mediator lifespan <br />                                            extension, particular decreased insulin-like signalling. Inactivation SWI/ <br />                                            SNF core subunits SWSN-4/BRG1 SWSN-1/BAF155/170 RNAi fully suppressed this <br />                                            lifespan extension (Fig. 6b,d). Partial suppression seen RNAi non-core <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                            Page 7 <br />  <br />  <br />                                            subunit SWSN-3/BAF57 (Fig. S7a). Even post-developmental RNAi SWI/SNF <br />                                            subunits sufficient partially suppress DAF-16/FOXO-induced lifespan extension <br />                                            (Fig. 6e, Fig. S7b). All lifespan phenotypes induced SWI/SNF RNAi diminished in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            daf-16(0) mutant worms, consistent SWI/SNF functioning cofactor in <br />                                            pathway DAF-16/FOXO (Fig. 6c,e, Fig. S7b). We confirmed these <br />                                            phenotypes mutant analysis. Consistent RNAi results, hypomorphic alleles <br />                                            swsn-4(os13ts) swsn-1(os22ts) impaired DAF-16/FOXO-mediated lifespan <br />                                            extension (Fig. S7c,d). Unlike SWI/SNF alleles RNAi conditions tested, the <br />                                            particularly strong allele swsn-1(os22ts) shortened lifespan daf-16(0) animals, <br />                                            showing SWI/SNF lifespan effects independent DAF-16/FOXO. <br />                                            But consistent requirement SWI/SNF DAF-16/FOXO function two <br />                                            acting pathway, daf-16(0) failed shorten lifespan swsn-1(os22ts) <br />                                            animals (Fig. S7d). <br />  <br />                                            The lifespan extending capabilities DAF-16/FOXO highly correlated ability <br />                                            induce stress response pathways3,24. We tested SWI/SNF required DAF-16/ <br />                                            FOXO-mediated stress resistance insulin-like signalling mutants. Inactivation SWI/ <br />                                            SNF swsn-1(os22ts) swsn-3(RNAi) specifically blocked enhanced resistance of <br />                                            daf-2 mutants oxidative stress (tBuOOH, Fig. 6f, S7e). In addition, swsn-1(os22ts) was <br />                                            able block enhanced resistance daf-2 mutants heat stress (32&#194;&#176;C, Fig. S7f). <br />                                            Inactivation SWI/SNF showed lesser effect daf-16 mutant backgrounds (Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            6f, S7e,f), suggesting SWI/SNF functions pathway DAF-16/ <br />                                            FOXO. Thus consistent SWI/SNF important cofactor DAF-16/FOXO, is <br />                                            required DAF-16/FOXO-mediated gene regulation, actually required <br />                                            broad range DAF-16/FOXO-mediated functions animal, particular dauer <br />                                            formation, stress resistance, promotion longevity. <br />  <br />                                Discussion <br />                                            Despite extensive knowledge pathways leading DAF-16/FOXO activation, long <br />                                            remained elusive means help cofactors activated DAF-16/FOXO <br />                                            regulates transcription confers suite beneficial effects organism. Our <br />                                            study provided systematic identification DAF-16/FOXO cofactors, itself <br />                                            provides significant resource future studies, focusing prominent <br />                                            candidate, chromatin remodeller SWI/SNF, able illuminate transcription- <br />                                            regulatory events downstream DAF-16/FOXO activation. We showed DAF-16/ <br />                                            FOXO predominantly transcriptional activator provided mechanistic insight into <br />                                            activation achieved &#226;&#8364;&#8220; DAF-16/FOXO recruiting BAF-like <br />                                            subclass SWI/SNF target promoters. Extensive exploration SWI/SNF function in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            systems suggests recruitment induces local chromatin remodelling enable <br />                                            binding downstream transcriptional components activates transcription22. <br />                                            These findings jibe previous implications DAF-16/FOXO pioneer <br />                                            transcription factor25,26, suggesting specifically pro-longevity stimuli DAF-16/ <br />                                            FOXO activated autonomously binds wide range target promoters it <br />                                            nucleates transcriptional activation alteration chromatin states. <br />  <br />                                            Chromatin states alteration previously shown profound stress <br />                                            responsive lifespan-regulatory effects, described alterations mostly <br />                                            limited epigenetic changes (i.e. histone methylation27,28 acetylation29) their <br />                                            mechanistic link known stress responsive lifespan-regulatory pathways often <br />                                            remained unclear. We established DAF-16/FOXO-controlled stress-responsive and <br />                                            lifespan-regulatory roles type chromatin alteration fundamentally distinct <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                            epigenetic changes, ATP-dependent chromatin remodelling hands SWI/ <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                                          Page 8 <br />  <br />  <br />                                            SNF. This opens new dimension alteration chromatin states regulate <br />                                            stress response longevity. Also potential cross-talk ATP-dependent <br />                                            chromatin remodelling lifespan-regulatory epigenetic marks important to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            investigate future. For example, enrichment SWI/SNF DAF-16/FOXO binding <br />                                            sites substantially entirely dependent DAF-16/FOXO (Fig. 5b&#226;&#8364;&#8220;d, Fig. S4) and <br />                                            supported factors epigenetic marks, histone acetylation a <br />                                            strong candidate30. <br />  <br />                                            Finally, like note FOXO SWI/SNF evolutionarily conserved <br />                                            FOXO lifespan regulatory roles humans31,32. Hence described <br />                                            roles SWI/SNF conserved, exploration eventually benefit <br />                                            understanding aging age-related diseases humans. <br />  <br />                                Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                            We thank Hitoshi Sawa, Shohei Mitani, Marlene Hansen, Caenorhabditis Genetics Center strains. We <br />                                            thank Gabriel Hayes, Ulandt Kim, Mark Borowski, Ania Puczinska, Daniel Grau experimental support. We <br />                                            thank Iain Cheeseman, Karim Bouazoune, Matthew Simon, Behfar Ardehali, William Mair, Taiowa Montgomery, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Avruch lab helpful discussions. This work supported grants National Institutes of <br />                                            Health G.R. (AG014161 AG016636), J.M.A. (5P30CA006516 2P01CA120964), J.A.W. <br />                                            (F32GM093491), N.V.K. (F32AI100501-01), R.E.K. (GM048405). C.G.R. supported long-term <br />                                            fellowships Human Frontier Science Program European Molecular Biology Organization, R.H.D. <br />                                            American Cancer Society (122240-PF-12-078-01-RMC), N.V.K. Tosteson Postdoctoral Fellowship <br />                                            Award, S.K.B. Damon Runyon Cancer Research Foundation, T.H. Glenn Foundation Medical <br />                                            Research Austrian Science Fund (FWF). <br />  <br />  <br />                                References <br />                                            1. Kenyon C. The plasticity aging: insights long-lived mutants. Cell. 2005; 120:449&#226;&#8364;&#8220;60. <br />                                               [PubMed: 15734678] <br />                                            2. Calnan DR, Brunet A. The FoxO <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>. Oncogene. 2008; 27:2276&#226;&#8364;&#8220;88. [PubMed: 18391970] <br />                                            3. Murphy CT, et al. Genes act downstream DAF-16 influence lifespan Caenorhabditis <br />                                               elegans. Nature. 2003; 424:277&#226;&#8364;&#8220;83. [PubMed: 12845331] <br />                                            4. Gerstein MB, et al. Integrative analysis Caenorhabditis elegans genome modENCODE <br />                                               project. Science. 2010; 330:1775&#226;&#8364;&#8220;87. [PubMed: 21177976] <br />                                            5. Berdichevsky A, Viswanathan M, Horvitz HR, Guarente L. C elegans SIR-2.1 interacts 14-3-3 <br />                                               proteins activate DAF-16 extend life span. Cell. 2006; 125:1165&#226;&#8364;&#8220;77. [PubMed: 16777605] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            6. Samuelson AV, Carr CE, Ruvkun G. Gene activities mediate increased life span C elegans <br />                                               insulin-like signaling mutants. Genes Dev. 2007; 21:2976&#226;&#8364;&#8220;94. [PubMed: 18006689] <br />                                            7. Pollina EA, Brunet A. Epigenetic regulation aging stem cells. Oncogene. 2011; 30:3105&#226;&#8364;&#8220;26. <br />                                               [PubMed: 21441951] <br />                                            8. Hartl FU, Bracher A, Hayer-Hartl M. Molecular chaperones protein folding proteostasis. <br />                                               Nature. 2011; 475:324&#226;&#8364;&#8220;32. [PubMed: 21776078] <br />                                            9. Margalit A, et al. Barrier autointegration factor blocks premature cell fusion maintains adult <br />                                               muscle integrity C. elegans. J Cell Biol. 2007; 178:661&#226;&#8364;&#8220;73. [PubMed: 17698609] <br />                                            10. Curran SP, Ruvkun G. Lifespan regulation evolutionarily conserved genes essential for <br />                                                 viability. PLoS Genet. 2007; 3:e56. [PubMed: 17411345] <br />                                            11. Kwon CS, Wagner D. Unwinding chromatin development growth: genes time. <br />                                                 Trends Genet. 2007; 23:403&#226;&#8364;&#8220;12. [PubMed: 17566593] <br />                                            12. Hargreaves DC, Crabtree GR. ATP-dependent chromatin remodeling: genetics, genomics and <br />                                                 mechanisms. Cell Res. 2011; 21:396&#226;&#8364;&#8220;420. [PubMed: 21358755] <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                                    Page 9 <br />  <br />  <br />                                            13. Phelan ML, Sif S, Narlikar GJ, Kingston RE. Reconstitution core chromatin remodeling <br />                                                complex SWI/SNF subunits. Mol Cell. 1999; 3:247&#226;&#8364;&#8220;53. [PubMed: 10078207] <br />                                            14. Moshkin YM, Mohrmann L, Van Ijcken WFJ, Verrijzer CP. Functional differentiation SWI/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                SNF remodelers transcription cell cycle control. Mol Cell Biol. 2007; 27:651&#226;&#8364;&#8220;61. [PubMed: <br />                                                17101803] <br />                                            15. Shibata Y, Uchida M, Takeshita H, Nishiwaki K, Sawa H. Multiple functions PBRM-1/ <br />                                                Polybromo- LET-526/Osa-containing chromatin remodeling complexes C. elegans <br />                                                development. Dev Biol. 2012; 361:349&#226;&#8364;&#8220;57. [PubMed: 22119053] <br />                                            16. Holstege FC, et al. Dissecting regulatory circuitry eukaryotic genome. Cell. 1998; 95:717&#226;&#8364;&#8220; <br />                                                28. [PubMed: 9845373] <br />                                            17. Sawa H, Kouike H, Okano H. Components SWI/SNF complex required asymmetric <br />                                                cell division C. elegans. Mol Cell. 2000; 6:617&#226;&#8364;&#8220;24. [PubMed: 11030341] <br />                                            18. Libina N, Berman JR, Kenyon C. Tissue-specific activities C. elegans DAF-16 regulation <br />                                                lifespan. Cell. 2003; 115:489&#226;&#8364;&#8220;502. [PubMed: 14622602] <br />                                            19. Henderson ST, Johnson TE. daf-16 integrates developmental environmental inputs mediate <br />                                                aging nematode Caenorhabditis elegans. Curr Biol. 2001; 11:1975&#226;&#8364;&#8220;80. [PubMed: 11747825] <br />                                            20. Murphy CT, Lee SJ, Kenyon C. Tissue entrainment feedback regulation insulin gene <br />                                                expression endoderm Caenorhabditis elegans. Proc Natl Acad Sci. 2007; 104:19046&#226;&#8364;&#8220;50. <br />                                                [PubMed: 18025456] <br />                                            21. Farkas G, et al. The Trithorax-like gene encodes Drosophila GAGA factor. Nature. 1994; <br />                                                371:806&#226;&#8364;&#8220;8. [PubMed: 7935842] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            22. Peterson CL, Workman JL. Promoter targeting chromatin remodeling SWI/SNF <br />                                                complex. Curr Opin Genet Dev. 2000; 10:187&#226;&#8364;&#8220;92. [PubMed: 10753786] <br />                                            23. Fielenbach N, Antebi A. C elegans dauer formation molecular basis plasticity. Genes <br />                                                Dev. 2008; 22:2149&#226;&#8364;&#8220;2165. [PubMed: 18708575] <br />                                            24. Lee SS, Kennedy S, Tolonen AC, Ruvkun G. DAF-16 target genes control C. elegans life-span <br />                                                metabolism. Science. 2003; 300:644&#226;&#8364;&#8220;7. [PubMed: 12690206] <br />                                            25. Zaret KS, Carroll JS. Pioneer transcription factors: establishing competence gene expression. <br />                                                Genes Dev. 2011; 25:2227&#226;&#8364;&#8220;41. [PubMed: 22056668] <br />                                            26. Hatta M, Cirillo LA. Chromatin opening stable perturbation core histone:DNA contacts by <br />                                                FoxO1. J Biol Chem. 2007; 282:35583&#226;&#8364;&#8220;93. [PubMed: 17923482] <br />                                            27. Greer EL, et al. Transgenerational epigenetic inheritance longevity Caenorhabditis elegans. <br />                                                Nature. 2011; 479:365&#226;&#8364;&#8220;71. [PubMed: 22012258] <br />                                            28. Greer EL, et al. Members H3K4 trimethylation complex regulate lifespan germline- <br />                                                dependent manner C. elegans. Nature. 2010; 466:383&#226;&#8364;&#8220;387. [PubMed: 20555324] <br />                                            29. Longo VD, Kennedy BK. Sirtuins aging age-related disease. Cell. 2006; 126:257&#226;&#8364;&#8220;268. <br />                                                [PubMed: 16873059] <br />                                            30. Chatterjee N, et al. Histone H3 tail acetylation modulates ATP-dependent remodeling through <br />                                                multiple mechanisms. Nucleic Acids Res. 2011; 39:8378&#226;&#8364;&#8220;91. [PubMed: 21749977] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            31. Flachsbart F, et al. Association FOXO3A variation human longevity confirmed German <br />                                                centenarians. Proc Natl Acad Sci. 2009; 106:2700&#226;&#8364;&#8220;5. [PubMed: 19196970] <br />                                            32. Willcox BJ, et al. FOXO3A genotype strongly associated human longevity. Proc Natl Acad <br />                                                Sci. 2008; 105:13987&#226;&#8364;&#8220;92. [PubMed: 18765803] <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                        Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 1. <br />                                            DAF-16/FOXO binds chromatin remodeller SWI/SNF. (a) C. elegans strains yielding <br />                                            inactive cytoplasmic DAF-16/FOXO (daf-18(mg198lf)/PTEN) active nuclear DAF-16/ <br />                                            FOXO (daf-2(e1370ts)/insulin-IGF receptor) shifted 20 h restrictive temperature, <br />                                            lysed lysates separated Superose 6 size-exclusion column. Fractions analysed <br />                                            SDS-PAGE western blotting. (b) Schematics screen DAF-16/FOXO <br />                                            cofactors. (c) SDS-PAGE/silver stain analysis large-scale &#206;&#177;-GFP immunoprecipitations <br />                                            indicated strains. (d,e) Confirmatory co-IPs. DAF-16/FOXO::FLAG (d) SWSN-4/ <br />                                            BRG1::GFP (e) immunoprecipitated whole-worm lysates indicated strains. <br />                                            50 U/ml Benzonase added eliminate DNA- RNA-mediated interactions. Samples <br />                                            analysed SDS-PAGE western blotting. For inputs fractions were <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                              Page 11 <br />  <br />  <br />                                            loaded: 2% SWSN-1::GFP (d), 2% DAF-16::FLAG (e), 20% SWSN-4::GFP <br />                                            (e). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                     Page 12 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 2. <br />                                            A BAF-like subclass SWI/SNF required regulation DAF-16/FOXO target genes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Psod-3::GFP; daf-2(e1370ts) Pins-7::ins-7::GFP animals grown L1-stage <br />                                            indicated RNAi bacteria. daf-2(e1370ts) inactivated shift restrictive <br />                                            temperature L4-stage. GFP-fluorescence evaluated day 4 adulthood (n=50 <br />                                            animals). Representative images shown (scale bar: 100 &#206;&#188;m) (a,c). Common SWI/SNF <br />                                            subunits ones specific subclasses BAF PBAF indicated accordingly (b,d). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                           Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 3. <br />                                            SWI/SNF required regulation large fraction DAF-16/FOXO target genes &#226;&#8364;&#8220; in <br />                                            particular activated DAF-16/FOXO. Wild-type, daf-2(e1370ts), daf-2(e1370ts); <br />                                            daf-16(0), daf-2(e1370ts); swsn-1(os22ts) C. elegans grown L4-stage, then <br />                                            shifted restrictive temperature. After 20 h, genome-wide mRNA expression levels were <br />                                            determined mRNA-Seq. (a) Scatter-plots comparing gene expression wild-type that <br />                                            various mutant strains. Only genes significantly induced daf-2(e1370ts) (left <br />                                            panel) repressed daf-2(e1370ts) (right panel) shown. Colours indicate strains in <br />                                            gene expression analysed. As indicated reversion genes wild- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            type expression levels (shift genes plots&#226;&#8364;&#8482; indicated diagonals), daf-16(0) <br />                                            extensively swsn-1(os22ts) partially suppress gene expression changes of <br />                                            daf-2(e1370ts) animals. (b) Confirmation mRNA-Seq data quantitative RT-PCR, <br />                                            looking expression levels endogenous sod-3, mtl-1, hsp-12.6, ins-7. Expression <br />                                            levels shown relative wild-type levels (black dotted lines). Consistent the <br />                                            mRNA-Seq results, daf-16(0) tends fully mutants SWI/SNF tend partially <br />                                            suppress differential gene expression caused daf-2(e1370ts). (c) Significant overlap <br />                                            genes regulated DAF-16/FOXO SWSN-1/BAF155/170 (hypergeometric <br />                                            test). Genes downregulated daf-2(e1370ts); daf-16(0) daf-2(e1370ts); <br />                                            swsn-1(os22ts) compared daf-2(e1370ts) shown upper diagram, genes <br />                                            upregulated comparisons shown lower diagram. (d) Correlation between <br />                                            extent DAF-16/FOXO- SWI/SNF-mediated differential gene expression in <br />                                            daf-2(e1370ts) animals. Red lines represent trendlines, blue numbers fraction of <br />                                            genes up- downregulated swsn-1(os22ts), n denotes number of <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                    Page 14 <br />  <br />  <br />                                            DAF-16/FOXO-non-regulated, repressed, activated genes contributing the <br />                                            plots. Specifically genes activated DAF-16/FOXO daf-2(e1370ts), extent of <br />                                            DAF-16/FOXO-mediated activation correlates increasing dependence SWI/SNF, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            indicated declining trendline plot right. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                         Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 4. <br />                                            SWI/SNF extensively associates DAF-16/FOXO-bound promoter regions. C. elegans <br />                                            daf-2(e1370ts); DAF-16::GFP, daf-2(e1370ts); SWSN-4::GFP, daf-2(e1370ts); <br />                                            SWSN-1::GFP grown asynchronously e1370ts inactivated 20h shift to <br />                                            restrictive temperature. ChIP-Seq GFP-tagged proteins performed binding sites <br />                                            determined. (a) Each binding site associated closest transcriptional start <br />                                            site (TSS) distances plotted, revealing DAF-16/FOXO SWI/SNF are <br />                                            predominantly located 2.5 kb TSS promoter regions. (b) Genes <br />                                            vicinity DAF-16/FOXO binding site strongly enriched DAF-16/FOXO- <br />                                            activated depleted DAF-16/FOXO-repressed genes. This effect decays with <br />                                            distance binding site TSS. (c) DAF-16/FOXO bound promoter regions <br />                                            strongly enriched binding SWSN-4::GFP SWSN-1::GFP. Mean read <br />                                            distributions DAF-16/FOXO binding sites shown indicated strains. (d) <br />                                            Venn diagram showing overlap binding sites DAF-16/FOXO, SWSN-4/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            BRG1, SWSN-1/BAF155/170. (e) Heat-map representation data contributing to <br />                                            (c,d), supporting SWI/SNF binding occurs majority DAF-16/FOXO <br />                                            binding sites. Lines heat-map represent individual DAF-16/FOXO-bound regions <br />                                            sorted intensity DAF-16/FOXO binding. <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 5. <br />                                            DAF-16/FOXO recruits SWI/SNF specifically target promoters directly activated <br />                                            DAF-16/FOXO. C. elegans daf-2(e1370ts); DAF-16::GFP, daf-2(e1370ts); <br />                                            swsn-1(os22ts); DAF-16::GFP, daf-2(e1370ts); SWSN-4::GFP, daf-2(e1370ts); daf-16(0); <br />                                            SWSN-4::GFP, daf-2(e1370ts); SWSN-1::GFP, daf-2(e1370ts); daf-16(0); <br />                                            SWSN-1::GFP grown asynchronously e1370ts os22ts alleles inactivated <br />                                            20h shift restrictive temperature. ChIP-Seq GFP-tagged proteins performed. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (a) DAF-16/FOXO binding promoter regions affected absence SWI/SNF. <br />                                            Mean read distributions different subsets DAF-16/FOXO target promoters are <br />                                            shown indicated strains. n denotes number promoter regions used each <br />                                            analysis p-values denote significance change DAF-16/FOXO binding. <br />                                            Corresponding histograms illustrate hardly promoters underwent binding <br />                                            change larger 2-fold. (b) Loss DAF-16/FOXO impairs binding SWSN-4/BRG1 <br />                                            specifically promoters directly bound activated DAF-16/FOXO. Mean read <br />                                            distributions different subsets DAF-16/FOXO target promoters shown the <br />                                            indicated strains. n denotes number promoter regions used analysis p- <br />                                            values denote significance change SWSN-4/BRG1 binding. Corresponding <br />                                            histograms illustrate fraction promoters underwent larger 2-fold change in <br />                                            SWSN-4 binding (colored regions). (c) Histograms illustrating loss DAF-16/FOXO <br />                                            likewise impairs binding SWSN-1/BAF155/170 specifically promoters directly bound <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                  Page 17 <br />  <br />  <br />                                            activated DAF-16/FOXO. (d) Prominent example daf-16(0)-dependent changes in <br />                                            SWSN-4/BRG1 binding sod-3 promoter (a promoter directly bound activated by <br />                                            DAF-16/FOXO). ChIP-Seq data normalized, smoothed 50 bp bins, then <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            displayed UCSC genome browser. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                          Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 6. <br />                                            SWI/SNF required DAF-16/FOXO-mediated dauer formation, stress resistance, and <br />                                            longevity. (a) Dauer suppression assay. Eggs C. elegans activated DAF-16/FOXO <br />                                            (daf-2(e1370ts)) placed indicated RNAi bacteria grown restrictive <br />                                            temperature. Dauers identified 5 days based morphology resistance <br />                                            1 % SDS. Common SWI/SNF subunits ones specific subclasses BAF PBAF <br />                                            indicated accordingly. Marked RNAi clones (*,#) significantly different from <br />                                            control RNAi (t-Test; p&lt;0.05), (#) led non-dauer arrest L3 <br />                                            stage. (see Fig. S6a) (n=50 animals; error bars based S.D. 3 independent <br />                                            experiments) (b&#226;&#8364;&#8220;d) Lifespan phenotypes caused inactivation SWI/SNF. Indicated <br />                                            strains grown L1-stage indicated RNAi bacteria. Animals shifted to <br />                                            restrictive temperature L4-stage. (e) Also post-developmental inactivation SWI/ <br />                                            SNF impairs DAF-16/FOXO-mediated longevity. Indicated C. elegans strains grown <br />                                            L1-stage E. coli HT115. At L4-stage, animals shifted indicated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            RNAi bacteria e1370ts inactivated shift restrictive temperature. (f) Oxidative <br />                                            stress resistance assay. Indicated strains grown L4-stage, e1370ts and <br />                                            os22ts alleles inactivated shift restrictive temperature. 24 h later, animals were <br />                                            exposed 6 mM tert-Butylhydroperoxide (tBuOOH) survival monitored. All <br />                                            survival data panels (b&#226;&#8364;&#8220;f) obtained minimum 100 animals condition, <br />                                            mean survival times S.E.M. obtained Kaplan-Meier analysis, significant <br />                                            differences conditions determined log-rank test (for exact numbers of <br />                                            animals statistical data Table S9). <br />  <br />  <br />  <br />  <br />                                                Nat Cell Biol. Author manuscript; available PMC 2013 August 21. <br />                             Riedel et al.                                                                                               Page 19 <br />  <br />  <br />  <br />  <br />                                                                                                   Table 1 <br />                            Short-list candidate DAF-16/FOXO cofactors required DAF-16/FOXO function. Table sorted by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            intensity lifespan phenotypes. For list functional evaluation data Tab <br /> </body></html>