<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Nov. 1989, p. 5045-5054                                                                   Vol. 9, No. 11 <br /> 0270-7306/89/115045-10$02.00/0 <br /> Copyright &#194;&#169; 1989, American Society Microbiology <br />  <br />  <br /> Molecular Analysis SNF4 Gene Saccharomyces cerevisiae: <br />   Evidence Physical Association SNF4 Protein the <br />                        SNF1 Protein Kinase <br />                        JOHN L. CELENZA,t FRANCIS J. ENG,t AND MARIAN CARLSON* <br />    Department Genetics Development Institute Cancer Research, College Physicians Surgeons of <br />                                   Columbia University, New York, New York 10032 <br />                                             Received 16 June 1989/Accepted 31 July 1989 <br />  <br />             The SNF4 gene required expression glucose-repressible genes response glucose deprivation in <br />          Saccharomyces cerevisiae. Previous evidence suggested SNF4 functionally related SNFI, another <br />          essential gene global regulatory encodes protein kinase. Increased SNFI gene dosage <br />          partially compensates <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> SNF4, SNF4 function required maximal SNF1 protein <br />          kinase activity vitro. We cloned SNF4 identified 1.2-kilobase RNA, regulated by <br />          glucose repression. A 36-kilodalton SNF4 protein predicted nucleotide sequence. Disruption the <br />          chromosomal SNF4 locus revealed requirement SNF4 function stringent low temperature <br />          (23&#194;&#176;C). A bifunctional SNF4-lacZ gene fusion includes entire SNF4 coding sequence was <br />          constructed. The fusion protein shown immunofluorescence microscopy distributed throughout <br />          cell, partial localization nucleus. The SNF4-(-galactosidase protein coimmunoprecipitated with <br />          SNF1 protein kinase, providing evidence physical association proteins. <br />  <br />  <br />    The expression genes controlled glucose                  mutations SNFI: defects utilization carbon <br /> (carbon catabolite) repression procaryotic eu-                sources require expression glucose-repressible genes, <br /> caryotic cells. In Saccharomyces cerevisiae, important                 sucrose, raffinose, galactose, maltose, glycerol, and <br /> global regulatory controls expression genes               ethanol, failure homozygous diploids sporulate (4, <br /> in response glucose availability. Previously,                  28). Genetic analysis interactions snfl snf4 <br /> identified components regulatory isolating              mutations extragenic suppressors snfmutations, ssn6 <br /> mutants defective regulation SUC2 (inver-              ssn20 (spt6), indicated SNFJ SNF4 related <br /> tase) gene (4, 5, 28, 29). These studies identified SNFJ             functions distinct SNF genes <br /> gene essential expression glucose-                (28, 30). In accompanying paper (9), present addi- <br /> repressible genes response glucose limitation. SNFJ              tional genetic evidence functional interaction between <br /> the CCRI (11, 12; L. Neigeborn M. Carlson,                    genes: increased dosage SNFJ mutations in <br /> unpublished results) CAT] (13, 14, 39, 45),                SNFJ partially compensate deletion SNF4. We <br /> also affect glucose repression. Molecular analysis <br /> showed SNFJ encodes 72-kilodalton (kDa) protein-                   report biochemical evidence SNF4 function is <br /> serine/threonine kinase (8), accompanying paper                   required maximal SNF1 protein kinase activity vitro; <br /> presents genetic evidence protein kinase activity             immunoprecipitation kinase assays, autophosphoryla- <br /> crucial SNF1 function (9). The amounts SNFJ                    tion activity SNF1 kinase derived snf4 mutant <br /> RNA protein regulated glucose repression (7,               greatly reduced relative wild type. A modest <br /> 8), known SNF1 kinase activity              effect SNF4 level SNF1 protein noted. <br /> is regulated vivo. One key problems understand-                 These genetic biochemical studies suggest the <br /> ing biological role protein kinases identification          SNF4 product functions positive effector SNF1 <br /> of proteins interact kinase vivo,              protein kinase. SNF4 affect SNF1 function directly, <br /> targets kinase modulators regulators kinase            protein-protein interaction, indirectly, a <br /> activity. In yeast cells, genetic analysis offer important            cascade events. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> examine possibility that <br /> clues resolving problems.                                     SNF4 functions positive effector examine the <br />    Genetic studies strongly suggest close functional rela-              mechanism interaction, undertaken molec- <br /> tionship SNFJ SNF4, gene identified                ular analysis SNF4 gene product. In paper, <br /> our mutant search. SNF4 required expression                report cloning sequence analysis SNF4 <br /> many glucose-repressible genes response glucose dep-                gene. Disruption chromosomal locus led un- <br /> rivation. In case SUC2 gene, SNFJ SNF4                usual finding snf4 deletion mutations cause tempera- <br /> have shown act transcriptional level (3, 36).              ture-sensitive <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. We constructed bifunctional <br /> Mutations SNF4 cause phenotype similar caused                SNF4-lacZ fusion use analysis gene product and <br />                                                                           localization cell. We demonstrate coimmuno- <br />   * <br />     Corresponding author.                                                 precipitation SNF4-p-galactosidase fusion protein <br />   t Present address: Whitehead Institute, Cambridge, MA 02142.            SNF1 protein, providing evidence the <br />   t Present address: Department Biochemistry, Albert Einstein          physical association SNF4 protein SNF1 <br /> College Medicine, Bronx, NY 10461.                                     protein kinase vivo. <br />                                                                    5045 <br />  5046        CELENZA ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br />                          TABLE 1. S. cerevisiae strains                  deletion plasmid inserted yeast genome by <br />  Straina                                     Genotype                    integrative transformation MCY638 uracil indepen- <br />                                                                          dence; haploid strain used previous <br /> MCY501 .....              MATa ade2-101 SUC2                             isolation snf4-319 amber allele (28) suggested the <br /> MCY638 .....              MATa his4-539 lys2-801 ura3-52 SUC2            deletion lethal. Integration SNF4 locus <br /> MCY888 .....              MATa snf4-AJ lys2-801 ura3-52 SUC2 <br /> MCY1093 .....             MATa his4-539 lys2-801 ura3-52 SUC2            result duplication SNF4 gene sequences, and <br /> MCY1094 .....             MATa ade2-101 ura3-52 SUC2                     excision plasmid occur homologous <br /> MCY1209 .....             MATa snf4-319 his4-539 ura3-52 SUC2            recombination duplicated sequences, resulting <br /> MCY1217 .....             MATox snf4-319 his4-539 ura3-52 SUC2           loss plasmid-borne URA3 marker (38). Ura- <br /> MCY1218 .....             MATa snf4-319 his4-539 lys2-801 ura3-52 SUC2   segregants selected plating transformants me- <br /> MCY1595 .....             MATx snfl-A3 his4-539 lys2-801 ura3-52 his3-   dium containing 5-fluoro-orotic acid (1). A Ura- Snf- strain <br />                             A200 SUC2                                    identified, blot hybridization analysis genomic <br /> MCY1793 .....             MATa snfl-A3 snf4-Al lys2-801 ura3-52 ade2-    DNA strain confirmed presence deletion <br />                             101 leu2::HIS3 SUC2                          SNF4 locus. The snf4-AJ strain crossed a <br /> MCY1830 .....             MATa snf4-2 ade2-101 ura3-52 SUC2 <br /> MCY1853 .....             MATa snf4-2 ura3-52 his4-539 lys2-801 SUC2     wild-type SNF4 strain (MCY501), tetrad analysis the <br />                                                                          diploid showed 2:2 segregations Snf phenotype in <br />    All strains     isogenic     congenic S288C.              tetrads. <br />                                                                             Although deletion sequenced, further <br />                 MATERIALS AND METHODS                                    evidence suggests correct reading frame not <br />                                                                          restored 3&apos; deletion. Integration plasmid carrying <br />   Strains genetic methods. The S. cerevisiae strains used            SNF4(321)-lacZ fusion (described below) ge- <br /> are listed Table 1. Standard procedures followed             nome snf4-AJ mutant resulted transformants <br /> crossing, sporulation, tetrad analysis (41) trans-           carrying deletion fused lacZ, judged gel transfer <br /> formation (18, 19). Media methods scoring carbon                 hybridization (42). These transformants did express <br /> source utilization described previously (4).                   P-galactosidase, suggesting deletion shifted lacZ out <br />    Cloning SNF4 gene. Plasmid DNA YEp24                    frame. <br /> library (3) used transform (18) strain MCY1209 (snf4                 Construction snf4-2 allele. A 0.25-kb deletion was <br /> ura3) uracil independence. Transformants displaying                 constructed pFE27-12 described snf4-AJ dele- <br /> Snf+ phenotype selected ability utilize              tion. The plasmid used transform diploid <br /> raffinose anaerobic conditions methods described                MCY1093 x MCY1094, Ura- segregants selected <br /> previously (6). To recover complementing plasmid                using 5-fluoro-orotic acid (1). Tetrad analysis Ura- <br /> each transformant, total genomic DNA isolated                    diploid showed 2:2 segregations temperature-sensitive <br /> used transform Escherichia coli HB101 ampicillin                   Snf phenotype. Four spore clones tetrad ana- <br /> resistance.                                                              lyzed genomic blot hybridization confirm the <br />    To cloned DNA directs integration                  <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>deletion mutation</span> segregating 2:2. <br /> plasmid SNF4 locus, 2.7-kilobase-pair (kb) fragment                Construction SNF4-lacZ fusions. pFE27-2 DNA was <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> complementing plasmids subcloned                cleaved restriction sites 3&apos; SNF4 gene, treated with <br /> the integrating vector YIp5 (2). The plasmid used                 BAL 31 nuclease, filled Klenow fragment E. <br /> transform MCY638 (SNF4 ura3), Ura+ transfor-                     coli DNA polymerase I, digested HindIll. Frag- <br /> mants crossed MCY1217 (snf4 ura3). Tetrad analysis               ments desired size range purified agarose gel <br /> of resulting diploids showed Ura+ Snf&apos;                  electrophoresis ligated YEp356R DNA (27) had <br /> phenotypes cosegregated (2:2) tetrads                 digested HindIII SmaI. E. coli transformants <br /> diploid.                                                                 screened plasmids inserts correct size, <br />    Subclones. pFE27-2 subclone 4.6-kb HindIII                used transform MCY888 selection <br /> fragment pCGS40 (15), contains 2,um plasmid                 plasmid URA3 marker. The SNF4-lacZ fusion junc- <br /> origin URA3 gene. Plasmids pCP8, pCP17, pCP3,                    tions plasmids pSL184 pSL321 located by <br /> pCP2, pBG5 derived pFE27-2. pFE27-12                       sequence analysis codons 184 321, respectively, of <br /> contains 1.1-kb Clal fragment subcloned pLS1,                 SNF4. Both fusion proteins expected size (135 <br /> derivative YIp5 lacking EcoRI site (35). pRA5                 150 kDa, respectively), judged immunoblot anal- <br /> pRA20 0.65- 0.45-kb ClaI-EcoRI fragments,                    ysis. The gene fusions transferred vector <br /> respectively, subcloned pSP64 (24) AccI                 YEp366R (27), carries LEU2 selectable marker. <br /> and EcoRI sites.                                                            Enzyme assays. Glucose-repressed cultures grown to <br />    Preparation analysis DNA RNA. DNAs                    mid-log phase synthetic complete medium containing 2% <br /> prepared described previously (35). Manipulation                  glucose, derepressed cultures prepared shift <br /> analysis DNA carried standard methods (23).               medium containing 0.05% glucose 3 h. Selection for <br /> Preparation gel transfer hybridization analysis                   plasmids maintained omission uracil the <br /> poly(A)-containing RNAs carried described                    media. Secreted invertase assayed (16) cells as <br /> previously (37).                                                         described previously (6). P-Galactosidase assayed (26) <br />    DNA sequence analysis. Intact BAL 31-treated restric-             permeabilized cells (17). <br /> tion fragments cloned M13mpl8 M13mpl9                         Coimmunoprecipitation assay. The immunoprecipitation <br /> (31). The sequences strands determined               kinase assay performed essentially described the <br /> method Sanger et al. (34), using 17-nucleotide se-                accompanying paper (9) anti-SNF1 antiserum (2 <br /> quencing primer (Amersham Corp.).                                        ,ul) used affinity purification. After electropho- <br />    Construction snf4-A1 allele. A 0.15-kb deletion            resis, separated proteins analyzed immunoblot- <br /> constructed pFE27-12 treating EcoRI-digested                   ting (44) described accompanying paper (9). The <br /> DNA BAL 31 nuclease ligating. The resulting                     primary antibody detected using ProtoBlot immu- <br />  VOL. 9, 1989                                              ASSOCIATION OF SNF4 WITH SNF1 PROTEIN KINASE                                                  5047 <br />  <br /> noblotting (Promega Biotec). The rabbit anti-SNF1         pEsmi2                                                                    Conp6nmnkoion <br /> antibody (8) detected using goat anti-rabbit immuno-             H CA R          CB                   C       B                 H <br />                                                                 pfE2&gt;-2     L              a <br />                                                                                                              Il                                    +. <br /> globulin G (Fc)-alkaline phosphatase conjugate. Mouse <br /> monoclonal anti-f-galactosidase antibody (Promega Biotec) <br /> was detected using goat anti-mouse immunoglobulin G          PCPS <br /> (heavy light chain)-alkaline phosphatase conjugate. Be-     pCP17 <br /> fore reprobed different primary antibody,      pCP3 <br /> blot treated 5% nonfat dry milk buffer (20 mM       pOG5                                                                               + <br />  <br /> Tris hydrochloride [pH 7.5], 0.5 M NaCl, 0.05% Tween 20). <br /> The immunoblots exposed film detect 32p_        pCP2 <br /> labeled products. Inclusion 0.1 M NaCl, 0.5% deoxycho-       pFE27-12 <br /> late, 0.1% sodium dodecyl sulfate buffer used    pRA5                4 <br /> immunoprecipitation washing did affect coimmu-      pRA20            kvI <br /> noprecipitation SNFJ SNF4-lacZ gene products, nor <br /> did use affinity-purified (9) anti-SNF1 antibody.              FIG. 1. Restriction maps clones containing SNF4 sequences. <br />    Immunofluorescence microscopy. Cells grown min-      Plasmid constructions described Materials Methods. Only <br /> imal medium containing 2% glucose lacking uracil        cloned yeast DNA indicated. The ability episomal plasmids <br /> then shifted minimal medium containing 0.05%            complement snf4 mutant (MCY1218) growth raffinose is <br /> glucose (derepressing conditions) 3 h. Cells pre-      indicated. Symbols: , approximate position direction of <br /> pared described previously (10) stained mono-       transcription SNF4 RNA; _, 322-codon open reading <br /> clonal antibody P-galactosidase (Promega Biotec)         frame; - , *-, direction transcription single- <br /> fluorescein isothiocyanate-conjugated F(ab&apos;)2 fragment       stranded probes prepared plasmids pRA5 pRA20. Restric- <br /> sheep antibody mouse immunoglobulin G (Sigma Chemi-          tion sites: B, BgIII; C, ClaI; H, HindIII; R, EcoRI. An additional <br />                                                                 ClaI site position 448 SNF4 sequence (see Fig. 3) is <br /> cal Co.). Both incubations antibody carried       methylated bacteria. <br /> overnight 4&#194;&#176;C. Cells stained 4&apos;,6-dia- <br /> midino-2-phenylindole. Cells observed photo- <br /> graphed described previously (10).                           tional Biomedical Research Foundation using com- <br />                                                                 puter program FASTP (22) sequences in <br />                          RESULTS                                GenBank (release 58.0) translated reading frames <br />                                                                 using program TFASTA (32). No significant homol- <br />    Cloning SNF4 gene. Plasmids carrying cloned       ogy found. By examining sequence yeast gene <br /> SNF4 gene isolated library (3) selecting     CAT3 (40), however, determined SNF4 same <br /> complementation snf4 mutation yeast cells (see        gene. <br /> Materials Methods). Three plasmids complement             Disruption SNF4 chromosomal locus. The cloned <br /> snf4 growth raffinose contain overlapping DNA        DNA used disrupt SNF4 chromosomal <br /> segments recovered. We subcloned restriction frag-       locus. A deletion 150 <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> pairs constructed at <br /> ment integrating vector showed genetic           EcoRI site pFE27-12. This deletion introduced <br /> analysis cloned DNA directs integration         yeast genome transplacement (38) replace the <br /> plasmid chromosomal SNF4 locus (see Materials        wild-type SNF4 sequence (see Materials Methods). The <br /> Methods). The SNF4 gene localized cloned DNA         newly created allele, designated snf4-Al, conferred pheno- <br /> by testing subcloned segments ability complement     type indistinguishable snf4-319 amber mutant <br /> a snf4 mutation. A 4.6-kb HindIII fragment common        respect impaired growth sucrose, raffinose, galac- <br /> three SNF4 clones subcloned, resulting plasmid,     tose, glycerol inability derepress invertase (Table <br /> pFE27-2, complement snf4 (Fig. 1 Table 3).     2). The snf4-AJ allele recessive fails complement <br /> Derivatives pFE27-2 constructed tested     snf4-319 growth raffinose. These results confirm that <br /> complementation. The results localized SNF4 gene       cloned gene SNF4. <br /> 1.4-kb region (Fig. 1).                                            Temperature-dependent phenotype snf4 null mutant. In <br />    The SNF4 RNA identified Northern (RNA) blot           course testing phenotypes snf4 mutants, we <br /> analysis poly(A)-containing RNA (Fig. 2). A 1.2-kb RNA       noted striking effect temperature. At 30&#194;&#176;C, usual <br /> homologous SNF4 detected hybridization a <br /> probe prepared pFE27-12 (Fig. 1). This RNA was <br /> equally abundant glucose-repressed derepressed                        TABLE 2. Temperature dependence the <br /> cells, indicating expression regulated                            snf4 mutant phenotype <br /> glucose repression. The direction transcription (Fig. 1) <br /> was determined using single-stranded probes prepared                                        Invertase activity&apos;                              Growthye <br />                                                                                                                                               phenotype <br /> from pRA5 pRA20 (Fig. 2).                                   Relevant            230C                                 300C <br />    Nucleotide sequence SNF4. Figure 3 shows nucleo-      genotype <br /> tide sequence 1.4-kb complementing region the <br /> amino acid sequence predicted 36,398-Da SNF4 gene <br />                                                                             Repressed          prDesrsed   Repressed            prDesrse     23C        30C <br /> product. The open reading frame 322 amino acids     snf4-Al         1                 19             1                    1     + <br /> encoded 1.2-kb SNF4 RNA. No signature sequences          snf4-A2         1                   9            1                    1     + <br /> for functional domains identified, hydropathy        SNF4            1                186             1                246       +++         +++ <br /> plot (20) revealed hydrophobic region sufficient <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>     Expressed micromoles glucose released minute 100 mg (dry <br /> to span membrane. The amino acid sequence              weight) cells. Values snf4-AJ] averages assays different <br /> predicted protein compared sequences        mutants. Values wild type averages assays strains. <br /> Protein Identification Resource (release V19.0) Na-        b Growth sucrose raffinose. <br />  5048      CELENZA ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />                                F,         iR              R      D                          formed smaller colonies did wild type. The <br />                                                                                             levels expression secreted invertase activity 23 and <br />                                                                                             30&#194;&#176;C consistent growth phenotypes (Table 2). <br />                                                                                               This finding snf4 mutants temperature- <br />                                                                                             sensitive phenotype raised possibility the <br />                                                                                             amber mutation snf4-Al deletion truly null <br />                                                                                             mutation. We constructed another, 0.25-kb dele- <br />                                                                                             tion, designated snf4-A2, introduced genome <br />                                                                                             diploid strain (see Materials Methods). Tetrad <br />                 SNFI           _                                                            analysis diploid revealed 2:2 segregation temper- <br />                 UAA3      A   di                                                            ature-sensitive Snf phenotype indistinguishable of <br />                                                                                             previous snf4 alleles. We consider extremely <br />                                                                                             unlikely alleles encode temperature-sensitive <br />                 Pr.Obe pFE27-12     DRA5        :) RA 20                                    proteins. Thus, phenotype presumably corresponds to <br />   FIG. 2. Analysis SNF4 RNA. Poly(A)-containing RNAs                                 null phenotype. Apparently, requirement the <br /> were prepared glucose-repressed (R) derepressed (D) cells                          SNF4 protein stringent cells grown 23&#194;&#176;C. <br /> (MCY501), separated electrophoresis agarose gel contain-                              This temperature sensitivity distinguishes snf4 null <br /> ing formaldehyde (23), transferred nitrocellulose (43), hybrid-                      mutant phenotype snfl phenotype, was <br /> ized indicated probes. The 1.2-kb SNF4 RNA 0.9-kb                            equally mutant temperatures. We tested snfl snf4 <br /> URA3 RNA detected hybridization 32P-labeled pFE27-                             double mutants anaerobic growth sucrose raffi- <br /> 12 DNA prepared nick translation (33). Subsequent hybridization <br /> of filters SUC2 probe confirmed yeast cells                        nose 23&#194;&#176;C. The phenotype double mutant was <br /> properly glucose repressed derepressed; 1.8-kb SUC2                            identical snfl mutant. <br /> mRNA detected RNA repressed cells,                                Increased SNF4 gene dosage does restore invertase <br /> 1.8- 1.9-kb mRNAs present derepressed sample.                               derepression snfl mutant. Previous genetic evidence <br /> Single-stranded probes prepared (25) pRA5 pRA20,                              suggested SNF4 gene SNFJ protein kinase <br /> and hybridization performed described previously (46).                               gene play functionally related roles expression SUC2 <br /> Subsequent hybridization filter probed pRA20                           (28, 30). We accompanying paper (9) that <br /> confirmed presence intact RNA. <br />                                                                                             increased dosage SNFJ gene suppresses defect in <br />                                                                                             SUC2 derepression caused mutation SNF4. We <br /> temperature growth yeast cells laboratory,                                tested multiple copies SNF4 gene <br /> snf4 mutants showed anaerobic growth raffinose                                    restore invertase expression snfl deletion mutant. <br /> only weak growth sucrose. At 23&#194;&#176;C, however,                                     The multicopy plasmid pFE27-2 did restore invertase <br /> snf4-319 snf4-AJ mutants capable growing                                        expression snfl mutant affect regulation a <br /> anaerobically sucrose raffinose,                                  wild-type strain (Table 3). This finding consistent with <br /> grew slowly wild type (Table 2). The                                     biochemical evidence SNF4 positive effector the <br /> <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> observed respect aerobic growth                                      SNF1 protein kinase activity. <br /> glycerol. At 37&#194;&#176;C, mutants grew quite poorly glucose                                    Coimmunoprecipitation SNF1 protein kinase a <br /> and, streaked rich medium containing glucose,                                       SNF4-j-galactosidase fusion protein. The SNF4 protein <br />  <br />        -302 <br />        -240                                             AGOGAG <br />        -120 <br />               Hot L <br />               AI <br />                         Pro Tbr Gin Asp S&amp;t Gln Gu Lys Val Ser Ile Glu Gln Gln Leu Al Val GlU Ser Ile Ar <br />                       A COG AC CA C GAT TCA CM GAAb CTT TCT ATT GMA CAG CAG TTA OC T       m TiC  AThA <br />                                                                                                                                 Ph.e <br />                                                                                                                              La rrT    Leu n Ser Lys Thr <br />                                                                                                                                        TTG    CG <br />                                                                                                                                                             30 <br />  <br />             Per Tr s Val Leu Pro Val Ser Tyr Ara Lou 11e Val Lou Lsp Thr Ser Lou Leu Va1 Ls LY Ser Leu Asn Val Leu Leu Gln Asn                              60 <br />          91 TCT TiT GA CTC TTG OCT GTT TCT TAC CCT TTA ATT GTC TTG GAC ACC TCG TTG TTA CTC AG AM TCA CTG MT GTT CTT TTG CM MT <br />            Ser Ile Vol Ser Ala ProLou Trp Asp Ser Lys Thr Ser Ara Phe Ala Gly                   Lou Leu Thr Thr   Thr As    Phe Ile    Asa Val le Gin Tyr   90 <br />        181 MC ATT OTN TCT CC OCA  TTA TGC GAC TCC AAG ACT TCC AGO TTC OCT OGM                   CT CTA ACT ACT    ACA   ml  TTT ATT    MT GTC ATC CAG TAT <br />                Pw Per As. Pro As  La Pbe Glu Lou Val hs La Lou Gin Lou As                       GI Lou Las sp     Ile   Glu Ar Ala             p <br />                                                                                                                                   Lou C1 VoIlAsp <br />        271 TTr <br />                                                                                                                                                             120 <br />            T   TC TCC MT CCA GA   Al  TTC GM TTA CTA CAC AATTA CAG TTA GAT                        o TTA A GAT     ATA   GAG CI OCTCTC OCT CTT CAT     CG <br />            L Asp Thr Ala Sr Ile      Pro &amp;rAra Pro LuPb Glu Ala Cys Lou                         Lys Not Leu Glu   Ser   Arg Ser G Ar Ile Pro Leu      Ile   15U <br />        3L1 CIAaCAcA O TOA AlT     CAT CCT TCT AcO COC TTA TrT GAG 0c TOT cOTT                   AOC ATG TrA CM    TCA   AGA ACT OCT <br />                                                                                                                                   AGA ATA CCA CTG     ATC <br />            As GIs Asp Cl Giu bhr  His Ara Glu Ile Val Val S.r Val Lou Thr Gln                   Tyr Arm Ile Lou   Lr Phe Val Al Lou Asn Cs Ar         Glu   180 <br />        451 CU  CM  CGT GMA GAG MA CAT AC  GA ATT aTC CTT ACT r CTr MG CAM                       TAT AG&amp;tY         AAl TTC GTT OCT TTA MT TUC AGG      GM <br />            Thr nIs FM law Lys lie Pro Ile GCO GA <br />                                               Asp Lou Asa Ile lIe Thr CM <br />                                                                       Gin As                    Asn      La Sor <br />                                                                                                      Not AM       Cas CM <br />                                                                                                                       Gin Nt Thr Tbr Pro Val Ile      As    210 <br />        541 MA CAT  mT? cTA aw ATT CCA ATT         TTG AMc ATT ATT MC      GAT                   MC ATG      AOC   MTT     ATC CC ACT CCC GTC ATA      CT <br />            Vol Ile lsot            Lou Tbr GIsGl  Ars Va        Set Sr Val Pro    Ilelle  spGlu As Gl Tyr         Leu Ile Asn Vl Tyr Glu      Ala TyrA      240 <br />        631 GTC ATr CG ATC          OTT MCC CM OCT CGCryT        TT TCC CTC OCT    ATT ATT GAC GA MC 00C TAC       TTA ATC MC GTA TAT GC       OCA TAG CAT <br />            Vol Lou Gly Lou          1e L G GlyC    le Trr       AsLp Lu Ser       Leu Ser Val Cly Glu Ala Leu     Met Ara Ara Ser As As       Phe Glu GC    270 <br />        721 CTC CTA OOCTTC          AlA    MOal 0c ATOTAC        AAGC <br />                                                                 MC     CTO TCA    TTG AC  GTC oWA GM GCC CTT      ATG AGC AGA ACT Ml CAf      TTT GAG COT <br />            Vol Tyr Tbr Cys         Thr Ly Aa Asp L Leu          Ser Tbr Ile Nat   Asp An Ile Ar Lys Ala Ar        Vai1L. Ar Pbe Pbe Vol       Vl Asp As     300 <br />        611 CT TAT MA TOO           MT AL; MT GALTAATTA          TCT ACT ATT ATC   GT AMt ATC XC A     WCA C       GTG CAT AGC TTC TT CTA      GT GT CGT <br />        901 Vo   1 A 1 <br />               GTc        GT        Cr GI1 Val Lou Tbr Leu <br />                                    Volo   cTo   TTG   AcrcTTA <br />                                                                 Per Asp 11 Leu <br />                                                                   C cAT AT cTc <br />                                                                                   Lys Tyr ATC <br />                                                                                   AM TaT  Ile Lou Leu Gy Ser <br />                                                                                               CTT CTA <br />                                                                                                                As * <br />                                                                                                         OCT CC AC TAGAT GTGTACATA 322 <br />        1119 ACTGTCGAT <br />   FIG. 3. Nucleotide sequence SNF4 gene deduced amino acid sequence. Nucleotides numbered left, amino acids <br /> are numbered right. Symbols: *, in-frame termination codon; V, fusion sites SNF4-lacZ fusions amino acids 184 321. <br /> The EcoRI site used construction deletions snf4-AJ snf4-A2 underlined. The sizes SNF4-lacZ fusion proteins two <br /> bacterial proteins produced fusions N-terminal coding region E. coli trpE gene codons 21 169 SNF4 <br /> confirmed reading frame positions (data shown). <br />  VOL. 9, 1989                                                                          ASSOCIATION OF SNF4 WITH SNF1 PROTEIN KINASE                      5049 <br />  <br />        TABLE 3. Effects multicopy SNF4 plasmid                                        We test coprecipitation proteins in <br />                     invertase expression                                                   immune complexes. Figure 4 shows results two <br />                                    Secreted invertase activityb                            experiments. In experiment (Fig. 4A B), anti- <br />  Relevant genotypea                                                                        SNF1 serum used immunoprecipitation; the <br />                                          Repressed                       Derepressed       second experiment (Fig. 4C D), anti-i-galactosidase <br /> snf4-AJ (pCGS40)                          1                       3                        antibody used. In experiment, immunoprecip- <br /> snf4-Al (pFE27-2)                         1                     208                        itated proteins analyzed immunoblotting, the <br /> snfl-A3 (pCGS40)                          1                       1                        antibody preparations used sequentially probe <br /> snfl-A3 (pFE27-2)                       &lt;1                       &lt;1                        blot. <br /> Wild type (pCGS40)                      &lt;1                      153                           In experiment, antiserum specific SNF1 <br /> Wild type (pFE27-2)                     &lt;1                      170                        protein (8) used precipitate SNF1 protein an <br />   Strains used MCY888 (snf4-AJ), MCY1595 (snfl-A3), MCY1093                     extract strain carrying SNF4(321)-lacZ fusion, and <br /> (SNFI SNF4). pCGS40 (15) parent vector pFE27-2.                                 immunoprecipitate examined presence <br />   b Expressed micromoles glucose released minute 100 mg (dry                 SNF4(321)-lacZ fusion protein immunoblot analy- <br /> weight) cells. Values averages assays transformants.                    sis, using antibody ,B-galactosidase. The fusion protein <br />                                                                                            coimmunoprecipitated SNF1 protein (Fig. 4A, lane <br /> could affect activity SNF1 protein kinase                                  a). Recovery fusion protein depended precipitation <br /> direct mechanism, protein-protein interaction,                               SNF1 protein fusion protein recov- <br /> an indirect process, involving cascade events.                                ered immune complexes prepared snfl deletion <br /> To examine possibility SNF1 SNF4 proteins                                 mutant (Fig. 4A, lane b). In second experiment test for <br /> are physically associated, tagged SNF4 gene                                   coimmunoprecipitation, fusion protein precip- <br /> product constructing fusion codon 321                                  itated anti-,-galactosidase antibody, immuno- <br /> SNF4 gene E. coli lacZ gene (see Materials                                     precipitate subjected immunoblot analysis, using <br /> Methods). The plasmid-bome gene fusion, designated                                         anti-SNF1 antiserum. The SNF1 protein coimmunoprecipi- <br /> SNF4(321)-lacZ, complemented snf4 mutation raffinose                                 tated SNF4(321)-lacZ fusion product (Fig. 4C, lane <br /> utilization. The fusion produced P-galactosidase activity                             e). Precipitation SNF1 protein dependent the <br /> in yeast cells regulated glucose repression                                presence SNF4-p-galactosidase fusion protein the <br /> (30 75 U activity normalized cell density described                            extract (Fig. 4C, lane h). Rough quantitation immu- <br /> by Miller [26] glucose-repressed derepressed                                   noblots suggested substantial fraction immuno- <br /> cells). We constructed second fusion codon 184                                precipitable SNF1 protein coimmunoprecipitated the <br /> the SNF4 gene, SNF4(184)-lacZ, failed comple-                                     fusion protein vice versa. <br /> ment snf4 did express P-galactosidase activity. Both                                      To coimmunoprecipitation proteins <br /> fusions produced proteins expected sizes (150 135                               dependent SNF4-encoded portion fusion <br /> kDa). Consistent complementation data, SNF1                                       protein ,-galactosidase moiety, tested for <br /> protein kinase activity detected immunoprecipitates                                 coimmunoprecipitation SNF1 protein the <br /> derived snf4 mutants carrying SNF4(321)-lacZ                                      SNF4(184)-lacZ fusion protein, did exhibit SNF4 <br /> fusion SNF4(184)-lacZ fusion (data shown).                                 function. In test, SNF4(184)-lacZ fusion protein <br />                           A.          F:.N ;;Z, *&apos;\   B.          :N,\   --N                      C.               --N        D.       :N <br />                                   11&gt;eV -N   -:$. <br />                                               , <br />                                                   ,                     .N <br />                                                                  fl.IN e-V N   ck.&apos;                       -, <br />                                                                                                         &quot;I.),.- et.)rv k., <br />                                                                                                           r          &apos;N 4                       - <br />                                 .., <br />                                SS -Q\Qe7 Y,,.S <br />                                               C-;.r            5;, ..63 .. <br />                                                                          s-Q                        .;, C.,            63 <br />                                                                                                                                    4   v        N <br />                                                                                                                                                     ~ <br />                                                                                                         Q <br />                            -V,Y..                                       Y                                                                   - <br />                                                            v                                       y .- Y <br />                                                                  -F.zz -.,, <br />                                                                                                            . <br />  <br />  <br />                                         -F -P                                  -P                  -.    -F              -P <br />  <br />  <br />  <br />  <br />                                                                                                                                        . <br />                                                                                                                                        ..&apos; <br />  <br />  <br />                                                                                                        e <br />                                                                                                        f <br />                                                                                                        gh <br />                                                                                                         ef ;&quot; i-g..- <br />                                   b d  c    b c d                                f       h     e    f      gh              e <br />  <br />  <br />  <br />  <br />   FIG. 4. Coimmunoprecipitation SNF1 protein SNF4-p-galactosidase fusion proteins. (A) Proteins immunoprecipitated <br /> with anti-SNF1 antiserum (8), separated electrophoresis 6% sodium dodecyl sulfate-polyacrylamide (21), examined by <br /> immunoblot analysis (44), using anti-,3-galactosidase antibody described Materials Methods. (B) The blot shown panel A was <br /> reprobed anti-SNF1 serum detect immunoprecipitated SNF1 protein. The extra bands correspond rabbit serum proteins the <br /> immunoprecipitate reacted anti-rabbit immunoglobulin G (Fc)-alkaline phosphatase conjugate used detect anti-SNF1 <br /> antibody. (C) Proteins immunoprecipitated anti-,B-galactosidase antibody subjected immunoblot analysis, using anti-SNF1 <br /> antiserum. (D) The blot shown panel C reprobed anti-p-galactosidase antibody detect immunoprecipitated SNF4- <br />  3-galactosidase fusion proteins. The positions SNF1, SNF4(321)-lacZ, SNF4(184)-lacZ gene products indicated. Lanes <br /> contained proteins immunoprecipitated extracts prepared following strains: e, MCY888 (pSL321); b f, MCY1595 <br /> (pSL321); c g, MCY888 (pSL184); d h, MCY1093. Relevant genotypes indicated lanes. Plasmids pSL321 pSL184 <br /> carry SNF4(321)-lacZ SNF4(184)-lacZ fusions, respectively. <br />  5050     CELENZA ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />                     1-       (0-                                                 C. A.L    SJ     HE-.L                        _&gt;I1{ <br />  <br />                              cz    cz <br />                                                                                                             2   v30   3 <br />                                                                                                                            v223   3    3 <br />  <br />  <br />  <br />  <br />                                         - SNF41- <br />                                            13ga                                             A                             B3 <br />                                                                           FIG. 6. Immunoblot analysis SNF1 protein snf4 mutant and <br />                                                                         wild-type cells. Strains grown mid-log phase rich medium <br />                                                                         containing glucose indicated temperature. Each sample rep- <br />                                                                         resents protein derived 40 ,ug total protein. Samples <br />                                          JSNFH <br />                                                                         analyzed electrophoresis 7.5% sodium dodecyl sulfate- <br />                                           Protelsls <br />                                                                         polyacrylamide gel (21) immunoblotting (44). SNF1 protein was <br />                                                                         detected anti-SNF1 serum. (A) Wild-type (MCY1093) and <br />                                                                         snf4-A2 mutant (MCY1853) cultures grown 30&#194;&#176;C. Total protein <br />                                                                         (TOTAL) prepared fractionated centrifugation 13,000 <br />                                                                         x g described immunoprecipitation kinase assay. The <br />                                                                         supernatant (SUP) loaded gel directly, pellet <br />                                                                         (PELLET) washed immunoprecipitation buffer, <br />    FIG. 5. Coimmunoprecipitation C-terminally deleted SNF1           collected centrifugation 13,000 x g, suspended in <br /> products SNF4-p-galactosidase. Proteins immunoprecip-         sample buffer. (B) Wild-type (MCY1093) snf4-A2 mutant <br /> itated anti-p-galactosidase antibody, separated electropho-     (MCY1830) strains grown 23, 30, 37&#194;&#176;C, indicated. Total <br /> resis 7.5% sodium dodecyl sulfate-polyacrylamide,           protein analyzed immunoblotting panel A. <br /> examined immunoblot analysis, using anti-p-galactosidase <br /> antibody anti-SNF1 serum described Materials Meth- <br /> ods. The products snfl deletion alleles (9) SNF4(321)-   modest, reproducible. To confirm SNF4 <br /> lacZ fusion indicated. Proteins immunoprecipitated        affects the SNF1 protein present cell <br /> extracts prepared strain MCY1793 (snfl-A3 snf4-AJ) carrying        simply immunoprecipitation, compared <br /> the SNF4(321)-lacZ fusion vector YEp366R         SNF1 protein present wild-type snf4 mutant <br /> plasmids pCEsnfl-A4, pCEsnfl-A5, pCEsnfl-A6, pCEsnfl-A7.             extracts immunoblot analysis total protein (Fig. 6A). <br /> Strains grown selection maintenance plasmids.     The level SNF1 protein fewfold reduced snf4 <br />                                                                         mutant. In experiment, crudely fraction- <br /> did coprecipitate immunoprecipitated SNF1                  ated extracts centrifugation 13,000 x g, done <br /> protein (Fig. 4A, lane c). Recovery SNF1 protein             clear lysate immunoprecipitation. Immunoblot <br /> verified subsequent probing immunoblot              analysis showed SNF1 protein present in <br /> anti-SNF1 antiserum (Fig. 4B, lane c). In second test,          supernatant mutant extract reduced <br /> defective fusion protein immunoprecipitated first. The              relative wild-type level. <br /> SNF1 protein did coprecipitate SNF4(184)-lacZ                 The evidence physical association proteins <br /> protein immune complexes prepared anti-,-galac-                 suggests SNF4 affect stability SNF1 <br /> tosidase antibody (Fig. 4C, lane g); recovery                    protein. The temperature dependence snf4 mutant <br /> SNF4(184)-lacZ fusion protein verified reprobing             phenotype, described above, suggests effect SNF4 <br /> immunoblot anti-p-galactosidase antibody (Fig. 4D,                 stability SNF1. A stringent requirement for <br /> lane g).                                                                SNF4 higher temperatures reflect thermolability of <br />    Coimmunoprecipitation C-terminally deleted SNF1 prod-             SNF1 protein absence SNF4. To test idea, <br /> ucts SNF4-4-galactosidase. In accompanying paper               examined effect snf4 mutation the <br /> (9), C-terminal deletion mutations SNFI.                 level SNF1 protein varies temperature. The amounts <br /> Some evidence presented truncation C terminus            SNF1 protein present snf4 mutant wild-type strains <br /> reduces dependence residual SNF1 function             grown 23, 30, 37&#194;&#176;C compared immunoblot <br /> presence SNF4. We examined coimmuno-                   analysis (Fig. 6B). In wild-type strain, level SNF1 <br /> precipitation truncated SNF1 proteins bifunc-           protein temperatures. In contrast, snf4 <br /> tional SNF4-3-galactosidase protein (Fig. 5). The SNF4                  mutant cells grown 37&#194;&#176;C contained greatly reduced <br /> fusion protein immunoprecipitated anti-,-galac-                SNF1 protein. This finding supports idea that <br /> tosidase antibody, recovered proteins exam-                SNF4 functions stabilize SNF1 kinase that <br /> ined immunoblot analysis, using anti-p-ga-                      stabilizing influence critical higher tempera- <br /> lactosidase antibody anti-SNF1 serum. Each              tures. However, stabilization SNF1 does appear be <br /> truncated SNF1 proteins detected. The proportion                 major function SNF4. The SNF1 protein in <br /> the protein coimmunoprecipitated                  snf4 cells grown 23 30&#194;&#176;C markedly different <br /> estimated. This evidence suggests C-terminal                 despite marked difference phenotype two <br /> deletions eliminate association mutant                temperatures. Moreover, snf4 strains exhibited distinctly <br /> SNF1 proteins SNF4.                                                 mutant phenotype 23&#194;&#176;C, temperature the <br />    Effect SNF4 level SNF1 protein. In accom-           level SNF1 protein kinase fewfold reduced <br /> panying paper (9), noted SNF1 protein              relative wild-type level. Thus, SNF4 appears <br /> recovered immunoprecipitation snf4 mutant                  role stabilizing SNF1 protein, level of <br /> reduced fewfold relative wild-type level. This effect,          SNF1 protein does correlate growth pheno- <br />  VOL. 9, 1989                                                 ASSOCIATION OF SNF4 WITH SNF1 PROTEIN KINASE                   5051 <br />  <br /> type, suggesting stabilization major function      Disruption SNF4 led unexpected finding the <br /> SNF4. These results consistent finding           snf4 null phenotype temperature dependent. Two snf4 <br /> SNF4 profoundly affects vitro kinase activity SNF1       deletion mutants amber mutant exhibited tempera- <br /> (9).                                                               ture-sensitive phenotype: 30 37&#194;&#176;C these <br />    Localization SNF4 protein immunofluorescence.         strains showed clear Snf phenotype, 23&#194;&#176;C showed <br /> Indirect immunofluorescence microscopy used local-          partially mutant phenotype. These alleles are <br /> ize SNF4(321)-lacZ fusion product yeast cell.       unlikely encode temperature-sensitive SNF4 proteins, <br /> Cells carrying gene fusion multicopy plasmid         favor idea requirement for <br /> stained monoclonal antibody P-galactosidase. In            SNF4 stringent higher temperatures. The finding <br /> many cells, fusion protein partially localized      deletion SNF4 causes partial Snf- phenotype <br /> nucleus, substantial portion remained            23&#194;&#176;C provides genetic evidence SNF4 affects, <br /> cytoplasm (Fig. 7). The staining pattern appeared         absolutely required for, SNF1 function. <br /> in cells prepared conditions glucose repression           Genetic evidence argued regulatory role for <br /> derepression. We unable detect specific staining        SNF4 glucose repression increased SNFJ gene <br /> cells carrying fusion centromere-containing plas-         dosage restored glucose-repressible invertase expression in <br /> mid.                                                               snf4 null mutant (9). The findings presented are <br />    Previously, showed immunofluorescence micros-             consistent view. We expression <br /> copy SNF1 protein distributed           SNF4 RNA production bifunctional SNF4- <br /> cell, apparent nuclear localization (8). Additional        P-galactosidase fusion protein regulated glucose repres- <br /> studies SNF1 protein bifunctional SNF1-          sion distribution SNF4-,-galactosidase <br /> ,-galactosidase protein confirmed result. Thus,           fusion protein cell does change. <br /> SNF1 SNF4-,-galactosidase proteins appear                 The important finding regard mechanism <br /> extent differently distributed cell. Interpretation      SNF4 action bifunctional SNF4-,B-galactosidase <br /> these results subject reservation      fusion protein coimmunoprecipitates SNF1 protein <br /> certain staining patterns accurately reflect local-   kinase. This result indicates proteins phys- <br /> ization native proteins. It possible           ically associated. Presumably true native <br /> fraction SNF1 SNF4 protein physically             SNF4 protein. Although coimmunoprecipitation ex- <br /> associated the stoichiometry       periments rigorously quantitative, appeared that <br /> association different nucleus. Our coimmunoprecip-       substantial fraction of proteins was <br /> itation experiments sufficiently quantitative rule      associated other. The association represents rela- <br /> out possibility fraction protein failed       tively tight binding, complex survived immunoprecip- <br /> coimmunoprecipitate other. Another possibility         itation conditions moderate salt detergent <br /> that fraction protein accessible immuno-       concentrations. <br /> precipitation, note portion SNF1 protein         We note possibility fraction the <br /> was removed clearing lysate immunoprecipi-           SNF1 SNF4 protein physically associated <br /> tation (Fig. 6B).                                                  other. Immunofluorescent staining revealed some- <br />                                                                    different distributions SNF1 (8) SNF4- <br />                         DISCUSSION                                  3-galactosidase proteins cell, protein <br />    Previous genetic evidence suggested functional relation-      partially localized nucleus. The evidence of <br /> ship SNF4 gene SNFJ protein kinase             Schuller Entian consistent partial nuclear <br /> gene, required expression glucose-        localization: isolated yeast nuclei 70% of <br /> repressible genes response glucose deprivation. In       ,-galactosidase activity CAT3-lacZ fusion product <br /> accompanying paper (9), report increased dosage         present nuclear fraction (40). <br /> the SNFJ gene compensates mutation SNF4, restor-             What function SNF4 protein? The simple <br /> ing glucose-repressible invertase expression. We report       model consistent genetic biochemical evi- <br /> biochemical evidence SNF4 gene product re-             dence SNF4 protein positively affects function <br /> quired maximal activity SNF1 protein kinase          SNF1 protein kinase virtue presence a <br /> vitro. These findings suggested SNF4 acts positive       complex protein kinase. One possibility the <br /> effector SNF1 kinase. In study, carried        SNF4 protein stimulates SNF1 protein kinase direct <br /> out molecular analysis SNF4 gene product.         protein-protein interaction. Another possibility SNF4 <br /> We report evidence SNF4 protein SNF1              inhibits counteracts negative effector SNF1. The <br /> protein kinase physically associated. These findings           idea consistent evidence suggesting dele- <br /> provide molecular basis understanding effects       tion C terminus SNFJ results truncated protein <br /> of SNF4 function SNF1 kinase            residual function independent SNF4 (9). <br /> genetic properties SNFJ SNF4.                               According model, C-terminal domain would <br />    We cloned SNF4 gene complementation snf4            mediate interaction putative negative effector or <br /> mutation. Sequence analysis indicated SNF4 encodes          itself act negative effector. SNF4 counter- <br /> 322-amino-acid protein unusual features. Sequence          act negative effector need interact directly with <br /> comparison revealed SNF4 gene CAT3             C-terminal region. More complicated mechanisms of <br /> (40), originally identified gene affecting          SNF4 function imagined; example, cannot <br /> expression glucose-repressible enzymes (14). The        exclude possibility SNF4 acts protect phosphor- <br /> identity suspected earlier original cat3       ylated targets SNF1 kinase, present the <br /> mutants showed defect derepression invertase (14)         complex, attack phosphatase. <br /> or growth defects sucrose, raffinose, galactose (39).          SNF4 modest fewfold effect level SNF1 <br /> Disruption CAT3 recently reported cause             protein. Studies accompanying paper suggested that <br /> defects (40).                                                      SNF4 likely affect stability SNF1 protein <br />  5052   CELENZA ET AL.                            MOL. CELL. BIOL. <br />  <br />  <br />                              acu <br />                                CZ <br />  <br />  <br />  <br />  <br />                         A G8_._ <br />                         N    *D      -^o <br />  <br />  <br />  <br />                         &amp;z&lt;          z     e; <br />                         C )          Q : <br />  <br />  <br />  <br />                         - C) 02N <br />  <br />                         E ~0Q <br />                         00 0.3C_ <br />                         u o <br />                                  -Cu <br />                             =        .2 *Q <br />                         t-.D          &#194;&#176; <br />                                              , <br />                         00 <br />                          .Cu <br />                         8 c <br />  <br />                         8tC.) t= <br />                               u, <br />  <br />                         a)            aQ <br />  <br />  <br />  <br />  <br />                                  . <br />  <br />                         C            s 0 <br />                           00 E <br />                         t-O.. <br />                                      0o <br />  <br />  <br />                                      0 CuO <br />  <br />                                 Cu   0 . <br />  <br />                         a <br />                         Cu 2 0)            .Cu <br />                         4O <br />                         2- ,0 Cu <br />  <br />                          ) <br />                                            Co. <br />                         Cu   Cu-. <br />                         Cu toCu <br />                            .2 = <br />                              SL. <br />                         Z       8 O0.C <br />                                      4.4 <br />  <br />                         _e            _ <br />                         Na-)ocuC <br />                         =1    Y <br />  <br />                                 0 a. <br />                                      I-.. <br />  <br />  <br />                              0C oz <br />                                 Cud <br />                                      3U <br />                                      o Cu <br />                         ct <br />                         N- <br />  <br />  <br />  <br />  <br />                         2       Cub <br />                          -a                0.0 <br />  <br />  <br />  <br />                                 3: .(4 40 D0 <br />  VOL. 9, 1989                                                       ASSOCIATION OF SNF4 WITH SNF1 PROTEIN KINASE                           5053 <br />  <br /> rather expression: snf4 wild-type strains con-               13. Entian, K.-D. 1986. Glucose repression: complex regulatory <br /> tained SNFJ-lacZ fusion protein,                         yeast. Microbiol. Sci. 3:366-371. <br /> which subject transcriptional                   14. Entian, K.-D., F. K. Zimmermann. 1982. New genes <br /> translational controls SNF1 protein, multicopy                     involved carbon catabolite repression derepression the <br /> SNFJ plasmids produced                          yeast Saccharomyces cerevisiae. J. Bacteriol. 151:1123-1128. <br />                                                                           15. Goff, C. G., D. T. Moir, T. Kohno, T. C. Gravius, R. A. Smith, <br /> SNF1 regardless SNF4 genotype (9). The greatly                         E. Yamasaki, A. Taunton-Rigby. 1984. Expression calf <br /> reduced level SNF1 protein observed snf4 mutant                    prochymosin S. cerevisiae. Gene 27:35-46. <br /> high temperature suggests SNF1 stable            16. Goldstein, A., J. 0. Lampen. 1975. P-D-Fructofuranoside <br /> absence SNF4. In view evidence physical                         fructohydrolase yeast. Methods Enzymol. 42C:504-511. <br /> association proteins, idea SNF4 affects               17. Guarente, L. 1983. Yeast promoters lacZ fusions designed <br /> the stability SNF1 plausible attractive.                    study expression cloned genes yeast. Methods Enzymol. <br />    It important emphasize, however, modest                     101C:181-191. <br /> effect SNF4 level SNF1 protein kinase does               18. Hinnen, A., J. B. Hicks, G. R. Fink. 1978. Transformation of <br /> not account snf4 mutant phenotype. Elevation                       yeast. Proc. Natl. Acad. Sci. USA 75:1929-1933. <br />                                                                           19. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor- <br /> SNF1 levels snf4 mutant does completely restore                    mation intact yeast cells treated alkali cations. J. <br /> wild-type phenotype (9), snf4 mutants exhibit differ-                   Bacteriol. 153:163-168. <br /> ence phenotype 23 30&#194;&#176;C reflected             20. Kyte, J., R. F. Doolittle. 1982. A simple method for <br /> levels SNF1 protein. We observed substantial effect                   displaying hydrophobic character protein. J. Mol. Biol. <br /> SNF4 vitro kinase activity SNF1 (9),                      157:105-132. <br /> believe major role SNF4 affect function             21. Laemmli, U. K. 1970. Cleavage structural proteins the <br /> of SNF1 protein kinase.                                                   assembly head bacteriophage T4. Nature (London) <br />                                                                               227:680-685. <br />                                                                           22. Lipman, D. J., W. R. Pearson. 1985. Rapid sensitive <br />                     ACKNOWLEDGMENTS                                           protein similarity searches. Science 227:1435-1441. <br />   We thank Lenore Neigeborn isolating SNF4 clones             23. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br /> Linda Marshall-Carlson immunofluorescence microscopy.                     cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />   This work supported Public Health Service grant                      Cold Spring Harbor, N.Y. <br /> GM34095 National Institutes Health American         24. Mead, D. A., E. S. Skorupa, B. Kemper. 1985. Single <br /> Cancer Society Faculty Research Award M.C.                                 stranded DNA SP6 promoter plasmids engineering mutant <br />                                                                               RNAs proteins: synthesis &quot;stretched&quot; preproparathy- <br />                         LITERATURE CITED                                      roid hormone. Nucleic Acids Res. 13:1103-1118. <br />  1. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A positive           25. Melton, D. A., P. A. Krieg, M. R. Rebagliati, T. Maniatis, K. <br />     selection mutants lacking orotidine-5&apos;-phosphate decarbox-            Zinn, M. R. Green. 1984. Efficient vitro synthesis of <br />     ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen.       biologically active RNA DNA hybridization probes from <br />     Genet. 197:345-346.                                                       plasmids containing bacteriophage SP6 promoter. Nucleic <br />  2. Botstein, D., S. C. Falco, S. E. Stewart, M. Brennan, S. Scherer,         Acids Res. 12:7035-7056. <br />     D. T. Stinchcomb, K. Struhl, R. W. Davis. 1979. Sterile host      26. Miller, J. H. 1972. Experiments molecular genetics. Cold <br />     yeast (SHY): eukaryotic biological containment            Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />     recombinant DNA experiments. Gene 8:17-24.                            27. Myers, A. M., A. Tzagoloff, D. M. Kinney, C. J. Lusty. 1986. <br />  3. Carlson, M., D. Botstein. 1982. Two differentially regulated          Yeast shuttle integrative vectors multiple cloning sites <br />     mRNAs different 5&apos; ends encode secreted intracellular            suitable construction lacZ fusions. Gene 45:299-310. <br />     forms yeast invertase. Cell 28:145-154.                            28. Neigeborn, L., M. Carlson. 1984. Genes affecting the <br />  4. Carlson, M., B. C. Osmond, D. Botstein. 1981. Mutants              regulation SUC2 gene expression glucose repression in <br />     yeast defective sucrose utilization. Genetics 98:25-40.                Saccharomyces cerevisiae. Genetics 108:845-858. <br />  5. Carlson, M., B. C. Osmond, L. Neigeborn, D. Botstein. 1984.       29. Neigeborn, L., M. Carlson. 1987. Mutations causing consti- <br />     A suppressor snfl mutations causes constitutive high-level             tutive invertase synthesis yeast: genetic interactions snf <br />     invertase synthesis yeast. Genetics 107:19-32.                         mutations. Genetics 115:247-253. <br />  6. Celenza, J. L., M. Carlson. 1984. Cloning genetic             30. Neigeborn, L., K. Rubin, M. Carlson. 1986. Suppressors of <br />     mapping SNFI, gene required expression glucose-               snJ2 mutations restore invertase derepression cause tem- <br />     repressible genes Saccharomyces cerevisiae. Mol. Cell. Biol.           perature-sensitive lethality yeast. Genetics 112:741-753. <br />     4:49-53.                                                              31. Norrander, J., T. Kempe, J. Messing. 1983. Construction of <br />  7. Celenza, J. L., M. Carlson. 1984. Structure expression            improved M13 vectors using oligodeoxynucleotide-directed mu- <br />     SNFI gene Saccharomyces cerevisiae. Mol. Cell. Biol.            tagenesis. Gene 26:101-110. <br />     4:54-60.                                                              32. Pearson, W. R., D. J. Lipman. 1988. Improved tools for <br />  8. Celenza, J. L., M. Carlson. 1986. A yeast gene                biological sequence comparison. Proc. Natl. Acad. Sci. USA <br />     essential release glucose repression encodes protein           85:2444 2448. <br />     kinase. Science 233:1175-1180.                                        33. Rigby, P. W. J., M. Dieckmann, C. Rhodes, P. Berg. 1977. <br />  9. Celenza, J. L., M. Carlson. 1989. Mutational analysis          Labeling deoxyribonucleic acid high specific activity vitro <br />     Saccharomyces cerevisiae SNF1 protein kinase evidence                 nick translation DNA polymerase I. J. Mol. Biol. <br />     functional interaction SNF4 protein. Mol. Cell.              113:237-251. <br />     Biol. 9:5034-5044.                                                    34. Sanger, F., S. Nicklen, A. R. Coulson. 1977. DNA sequenc- <br /> 10. Celenza, J. L., L. Marshall-Carlson, M. Carlson. 1988. The            ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br />     yeast SNF3 gene encodes glucose transporter homologous               USA 74:5463-5467. <br />     mammalian protein. Proc. Natl. Acad. Sci. USA 85:2130-            35. Sarokin, L., M. Carlson. 1984. Upstream region required for <br />     2134.                                                                     regulated expression glucose-repressible SUC2 gene of <br /> 11. Ciriacy, M. 1977. Isolation characterization yeast mu-             Saccharomyces cerevisiae. Mol. Cell. Biol. 4:2750-2757. <br />     tants defective intermediary carbon metabolism carbon       36. Sarokin, L., M. Carlson. 1985. Upstream region the <br />     catabolite derepression. Mol. Gen. Genet. 154:213-220.                    SUC2 gene confers regulated expression heterologous gene <br /> 12. Denis, C. L. 1984. Identification new genes involved            Saccharomyces cerevisiae. Mol. Cell. Biol. 5:2521-2526. <br />     regulation yeast alcohol dehydrogenase II. Genetics 108:           37. Sarokin, L., M. Carlson. 1985. Comparison yeast <br />     833-844.                                                                  invertase genes: conservation upstream regulatory re- <br />  5054     CELENZA ET AL.                                                                                                 MOL. CELL. BIOL. <br />  <br />    gion. Nucleic Acids   Res. 13:6089-6103.                                DNA fragments separated gel electrophoresis. J. Mol. Biol. <br /> 38. Scherer, S., R. W. Davis. 1979. Replacement chromo-             98:503-517. <br />     segments altered DNA sequences constructed          43.   Thomas, P. S. 1980. Hybridization denatured RNA small <br />     vitro. Proc. Natl. Acad. Sci. USA 76:4951-4955.                        DNA fragments transferred nitrocellulose. Proc. Natl. Acad. <br /> 39. Schuller, H.-J., K.-D. Entian. 1987. Isolation expres-         Sci. USA 77:5201-5205. <br />     sion analysis yeast regulatory genes involved      44.   Towbin, H., T <br /> </body></html>