<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  Developmentally Essential Protein Flightless I Is a Nuclear Receptor <br />                Coactivator with Actin Binding Activity <br />  <br />  <br />             Hormone-activated nuclear receptors (NR) activate transcription by recruiting multiple coactivator com- <br />           plexes <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> the promoters of target genes. One important coactivator complex includes a p160 coactivator (e.g., <br />           GRIP1, SRC-1, or ACTR) that binds directly to activated NR, the histone acetyltransferase p300 or CBP, and <br />           the arginine-specific histone methyltransferase CARM1. We previously demonstrated that the coactivator <br />           function of CARM1 depends both on the methyltransferase activity and on additional unknown proteins that <br />           bind to CARM1. In this study a yeast two-hybrid screen for proteins that bind CARM1 identified the protein <br />           Flightless I (Fli-I), which has essential roles in Drosophila and mouse development. Fli-I bound to CARM1, <br />           GRIP1, and NRs and cooperated synergistically with CARM1 and GRIP1 to enhance NR function. Fli-I bound <br />           poorly to and did not cooperate with PRMT1, a CARM1-related protein arginine methyltransferase that also <br />           functions as an NR coactivator. The synergy between GRIP1, CARM1, and Fli-I required the methyltransferase <br />           activity of CARM1. The C-terminal AD1 (binding site for p300/CBP) and AD2 (binding site for CARM1) <br />           activation domains of GRIP1 contributed to the synergy but were less stringently required than the N-terminal <br />           region of GRIP1, which is the binding site for Fli-I. Endogenous Fli-I was recruited to the estrogen-regulated <br />           pS2 gene promoter of MCF-7 cells in response to the hormone, and reduction of endogenous Fli-I levels by <br />           small interfering RNA reduced hormone-stimulated gene expression by the endogenous estrogen receptor. A <br />           fragment of Fli-I that is related to the actin binding protein gelsolin enhanced estrogen receptor activity, and <br />           mutations that reduced actin binding also reduced the coactivator function of this Fli-I fragment. These data <br />           suggest that Fli-I may facilitate interaction of the p160 coactivator complex with other coactivators or <br />           coactivator complexes containing actin or actin-like proteins. <br />  <br />  <br />    The nuclear hormone receptors (NRs) are a family of hor-                  driven developmental processes in mice lacking functional <br /> mone-activated transcriptional activator proteins, which in-                 p160 genes (24, 67, 68). <br /> clude the receptors for steroid hormones, thyroid hormone,                      Coactivators generally have two types of functional domains, <br /> vitamin D, and retinoids (3, 44, 62). NRs bind directly or                   i.e., those that anchor the coactivator to the promoter and <br /> through other proteins to specific enhancer elements associ-                 so-called activation domains that transmit the activating signal <br /> ated with the promoters of target genes and thereby activate or              toward the transcription machinery. Activation domains may <br /> repress transcription. Transcriptional activation involves the               propagate the activating signal by catalyzing covalent modifi- <br /> recruitment of various coactivator proteins that cause local                 cations of histones and other proteins in the transcription <br /> remodeling of chromatin in the promoter region and help to                   machinery or by facilitating transcription initiation through <br /> recruit RNA polymerase II (25, 28, 46).                                      direct protein-protein interactions that anchor other coactiva- <br />    Many of the coactivators exist or function as complexes of                tors to the promoter or recruit or activate other components of <br /> multiple subunits that collaborate to accomplish a single step               the transcription machinery. The p160 coactivators are teth- <br /> in transcriptional activation or perform multiple functions that             ered to the promoter by one or more of three NR box motifs <br /> cooperate in a synergistic manner. For example, one relatively               with the consensus sequence LXXLL (where L represents <br /> well-characterized functional coactivator complex is organized               leucine and X represents any amino acid) (19, 27, 60, 63). <br /> around the p160 coactivators, which include SRC-1, GRIP1/                    These motifs form amphipathic alpha-helices that insert into a <br /> TIF2, and pCIP/ACTR/RAC3/AIB1/TRAM1. The functional                          conserved hydrophobic groove located in the AF-2 activation <br /> and physiological importance of p160 coactivators in NR-me-                  function of the ligand binding domains of all hormone-acti- <br /> diated transcriptional activation has been established through               vated NRs (15, 48). The NR Box motifs are located approxi- <br /> transient-transfection assays (25, 35, 49), in vitro transcription           mately in the middle of the 1,400-amino-acid polypeptide <br /> systems (40), loss of NR function in cells injected with p160                chain of p160 coactivators. To date two conserved activation <br /> antibodies (60), and specific deficiencies in nuclear receptor-              domains located near the C terminus of p160 coactivators have <br />                                                                              been identified as binding sites for additional coactivators: <br />                                                                              AD1 binds the histone acetyltransferase p300 or CBP (12, 60, <br />   * Corresponding author. Mailing address: Department of Pathology, <br /> HMR 301, University of Southern California, 2011 Zonal Ave., <br />                                                                              63), while AD2 binds the histone methyltransferase CARM1 <br /> Los Angeles, CA 90089-9092. Phone: (323) 442-1289. Fax: (323) 442-           (11). The recruitment and coactivator function of CARM1 and <br /> 3049. E-mail: stallcup@usc.edu.                                              p300 (or CBP) apparently depends on the presence of p160 <br />  <br />                                                                       2103 <br />  2104      LEE ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> coactivators (10, 11, 37); therefore, we refer to them as sec-                  GRIP1N (amino acids 5 to 765), GRIP1 M (amino acids 730 to 1121), GRIP1C <br /> ondary coactivators. In addition to AD1 and AD2, there may                      (amino acids 1122 to 1462) (11), GRIP1&#226;&#338;&#172;AD1 (lacking amino acids 1057 to <br />                                                                                 1109), GRIP1&#226;&#338;&#172;AD2 (lacking amino acids 1122 to 1462) (42), GRIP1&#226;&#338;&#172;N (lacking <br /> be other undefined activation domains of p160 coactivators;                     amino acids 1 to 562) (J. H. Kim, H. Li, and M. R. Stallcup, submitted for <br /> for example, the N-terminal region is very highly conserved                     publication), rat PRMT1 (32), and CARM1(E267Q) with Glu267 changed to <br /> among the three p160 coactivators (2) but presently has no                      Gln (35). The following mammalian cell expression vectors with cytomegalovirus <br /> known role in coactivator function.                                             promoters were used: pCMV-p300 encoding full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> p300 (51) and <br />    CARM1 belongs to the protein arginine methyltransferase                      pcDNA3.FliI containing a SalI-BamHI cDNA fragment encoding full-length <br />                                                                                 human Fli-I inserted into the XhoI and BamHI sites of pcDNA3. The leucine- <br /> family, which is characterized by a highly conserved arginine-                  rich repeat (LRR) region (amino acids 1 to 494) and gelsolin-like region (amino <br /> specific methyltransferase domain (71). Each of the six mem-                    acids 495 to 1268) of human Fli-I were amplified by PCR with flanking MfeI and <br /> bers of the mammalian family of proteins methylates a differ-                   SalI sites, digested with these enzymes, and were inserted into the EcoRI and <br /> ent set of protein substrates (23, 57). For example, CARM1                      XhoI sites of pSG5.Flag; pSG5.Flag was derived <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> pSG5.Flag-TEF4 (5). <br /> methylates histone H3 (11), p300/CBP (69), and the RNA                          Mammalian expression vectors for pSG5.flag-Gel.A (human Fli-I amino acids <br />                                                                                 495 to 827), pSG5.flag-Gel.A(G602S), and pSG5.flag-Gel.A(E585K) were also <br /> binding protein HuR (36), while PRMT1 methylates histone                        generated by PCR and site-directed mutagenesis of human Fli-I and were in- <br /> H4 and several RNA binding proteins distinct from HuR (57).                     serted into the EcoRI and XhoI sites. The expression vector for HA-tagged <br /> After its recruitment to the promoters of steroid hormone-                      BAF53 (50) was kindly provided by M. Cole (Princeton University). <br /> inducible genes, CARM1 methylates histone H3 in the local                          Mammalian expression vectors pHE0 encoding human estrogen receptor <br /> promoter region as part of the process of transcriptional acti-                 (ER) &#226;&#65533;&#163; and pCMX.hTR&#226;&#65533;&#164;1 encoding human thyroid hormone receptor (TR) &#226;&#65533;&#164;1 <br />                                                                                 were described previously, as were the reporter plasmids MMTV(ERE)-LUC for <br /> vation (41). In transient-transfection assays CARM1, but not                    ER, MMTV(TRE)-LUC for TR, and GK1 for Gal4 (11). Mammalian expression <br /> PRMT1, cooperates synergistically with p300 (or CBP) and                        vector pM.LRR encoding the Gal4 DNA-binding domain (DBD) fused to the <br /> p160 coactivators to enhance transcriptional activation by NRs                  LRR region of Fli-I (amino acids 1 to 494) was constructed by inserting a <br /> (35). The binding interface between CARM1 and p160 pro-                         PCR-amplified cDNA fragment flanked by EcoRI and SalI sites into the anal- <br /> teins involves an undefined surface on the CARM1 methyl-                        ogous sites of pM (Clontech). <br />                                                                                    Bacterial expression vectors encoding glutathione S-transferase (GST) fused <br /> transferase domain and the C-terminal AD2 domain of p160                        to GRIP1 fragments were described previously: GRIP1N (amino acids 5 to 765) <br /> coactivators (11, 58). Transmission of the activating signal to                 (30); GRIP1 M (amino acids 730 to 1121) (29); and GRIP1C (amino acids 1122 <br /> the transcription machinery by CARM1 involves the methyl-                       to 1462) (42). GST fused to ER was encoded by pGEX5X-1.HEG0 (65). Yeast <br /> transferase activity (11, 35, 41) and additional domains, such as               expression vector pGBT9.CARM1 encoding Gal4 DBD fused to full-length <br /> the unique C-terminal and N-terminal domains, which are not                     CARM1 (amino acids 3 to 608) contained an EcoRI insert encoding CARM1. <br />                                                                                    Protein-protein interaction assays. The following procedures were performed <br /> involved in the methyltransferase activity (58). This result sug-               as described previously: yeast two-hybrid screening (11); production of GST <br /> gests that CARM1 transmits its activating signal by two coop-                   fusion proteins in bacteria and binding of labeled proteins translated in vitro to <br /> erative mechanisms, i.e., protein methylation and protein-pro-                  GST fusion proteins bound to glutathione-agarose beads (32); and coimmuno- <br /> tein interactions.                                                              precipitation and immunoblot experiments (35) with anti-Flag (Sigma F3165), <br />    We thus propose that additional, presently undefined, pro-                   anti-ER and anti-TR (Santa Cruz), and anti-HA (Roche 3F10) antibodies (1 &#226;&#65533;&#174;g <br />                                                                                 of antibody for immunoprecipitation and 1:1,000 dilutions for immunoblots). <br /> teins are likely to participate in transcriptional activation                      Cell culture and transient-transfection assays. Reporter gene activation as- <br /> through protein-protein interactions with GRIP1 and CARM1.                      says were performed by transient transfection of CV-1 or MCF-7 cells in six-well <br /> We have identified Flightless I (Fli-I) in a yeast two-hybrid                   dishes as described previously (35). Where indicated, 20 nM estradiol (E2) or <br /> screen (22) for proteins which interact with CARM1 and may                      2,5,5&#226;&#172;&#732;-triiodo-L-thyronine (T3) was added to the transfected cultures during the <br /> therefore cooperate with it as coactivators. In Drosophila, par-                last 40 h of growth. <br />                                                                                    Peptide antibody production for Fli-I. A peptide representing amino acids <br /> tial loss-of-function mutations of fliI result in a loss of flight              1232 to 1251 in a unique portion of the gelsolin-like region of Flightless-I <br /> ability, while severer mutations lead to impaired cellularization               (human) was synthesized by Tufts University Microchemical Core, and the rabbit <br /> and gastrulation of the embryo (8). Fli-I is a highly conserved                 antiserum for this peptide (&#226;&#65533;&#163;-Fli-C) was generated by Zymed. <br /> protein among diverse vertebrate and invertebrate species (7,                      ChIP assay. Chromatin immunoprecipitation (ChIP) assays ware performed <br /> 8). The human gene FLII is mapped to a region of human                          according to previously described protocols (41, 43, 56) with minor modifica- <br />                                                                                 tions. MCF-7 cells were cultured for at least 4 days in phenol red-free Dulbecco&#226;&#8364;&#8482;s <br /> chromosome 17p, which is associated with Smith-Magenis syn-                     modified Eagle medium supplemented with 5% charcoal-dextran-stripped fetal <br /> drome, a genetic disease causing developmental and behav-                       bovine serum. Cells at approximately 90 to 95% confluency were treated with 100 <br /> ioral abnormalities (13). While it is not yet clear whether                     nM E2 or vehicle for 45 min before formaldehyde cross-linking and chromatin <br /> Smith-Magenis syndrome is due to haploinsufficiency of FLII,                    preparation. Immunoprecipitation of sonicated chromatin solutions was con- <br /> homozygous disruption of the orthologous Fliih gene in mice                     ducted by overnight incubation at 4&#194;&#176;C with the appropriate antibody. After the <br />                                                                                 cross-linking of the immunoprecipitated chromatin was reversed, DNA purifica- <br /> leads to very early failure of embryonic development (6). Fli-I                 tion was achieved by phenol-chloroform extraction and ethanol precipitation. <br /> is an actin binding protein that can be either associated with                  The purified DNA was dissolved in 50 &#226;&#65533;&#174;l of 10 mM Tris-HCl, pH 8.0&#226;&#8364;&#8220;1 mM <br /> the cytoskeleton or in the nucleus, but its exact physiological                 EDTA, and 1 &#226;&#65533;&#174;l was used for PCR analysis. For the ChIP and reimmunopre- <br /> functions have not been elucidated (16, 17, 26). Since we iden-                 cipitation (Re-IP) assay, the immunoprecipitated fractions were eluted by incu- <br /> tified Fli-I as a CARM1 binding protein, we investigated its                    bating for 30 min at 37&#194;&#176;C in IP elution buffer with 10 mM dithiothreitol (43). The <br />                                                                                 eluates were diluted 50-fold with dilution buffer (1% Triton X-100, 2 mM EDTA, <br /> ability to bind to and collaborate with CARM1 and other NR                      150 mM NaCl, and 20 mM Tris-HCl, pH 8.1) and were reprecipitated by specific <br /> coactivators in mammalian cells.                                                antibodies. Antibodies used in ChIP assays were obtained from Santa Cruz <br />                                                                                 Biotechnology (normal mouse immunoglobulin G [IgG], normal rabbit IgG; ER, <br />                                                                                 AIB1, p300, and &#226;&#65533;&#163;-Fli-N against the Fli-I LRR region) or Upstate Biotechnology <br />                        MATERIALS AND METHODS                                    (CARM1). The primers used for detecting portions of the pS2 gene by PCR are <br />    Plasmids. Mammalian cell expression vector pSG5.HA (11), which has simian    listed below from 5&#226;&#172;&#732; to 3&#226;&#172;&#732; with nucleotide numbers representing the distance <br /> virus 40 and T7 promoters, was used to express proteins with an N-terminal      from the transcription start site: for the human pS2 promoter region (&#226;&#171;&#186;31 to <br /> hemagglutinin (HA) epitope tag. Plasmids encoding the following proteins were   &#226;&#171;&#186;353), GGCAGGCTCTGTTTGCTTAAAGAGCG and GGCCATCTCTCAC <br /> previously constructed from pSG5.HA: mouse CARM1, mouse GRIP1,                  TATGAATCACTTCTGC; and for the pS2 coding region (&#226;&#171;&#185;871 to &#226;&#171;&#185; 1200), <br />  VOL. 24, 2004                                                                        NUCLEAR RECEPTOR COACTIVATOR Flightless I                        2105 <br />  <br /> TGCCAGCTGTGGGGAGCTGAATAACTT and CAGTTCGTTCTGTACA <br /> CCGAGGCCACT. <br />    siRNA. Small interfering RNA (siRNA) experiments were performed accord- <br /> ing to previously published methods (55) with minor modifications. Single- <br /> stranded RNAs were synthesized by the University of Southern California Norris <br /> Comprehensive Cancer Center Microchemical Core Laboratory. Equimolar con- <br /> centrations of complementary RNAs were incubated together in annealing buffer <br /> (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, and 2 mM magne- <br /> sium acetate) at 90&#194;&#176;C for 1 min and then at 37&#194;&#176;C for 1 h to form a duplex. siRNA <br /> duplexes were transfected into MCF-7 cells by using Lipofectamine 2000 or <br /> Oligofectamine (Invitrogen) according to the manufacturer&#226;&#8364;&#8482;s protocol. The <br /> siRNA sequences used (listed 5&#226;&#172;&#732; to 3&#226;&#172;&#732;) were as follows: si-FliA, GCUGGAAC <br /> ACUUGUCUGUGdTdT and CACAGACAAGUGUUCCAGCdTdT; si-FliB, <br /> CAACCUGACCACGCUUCAUdTdT and AUGAAGCGUGGUCAGGUUG <br /> dTdT; si-CARM1, GCAGUCCUUCAUCAUCACCdTdT and GGUGAUGAU <br /> GAAGGACUGCdTdT; si-AIB1 (55), AGACUCCUUAGGACCGCUUdTdT <br /> and AAGCGGUCCUAAGGAGUCUdTdT. <br />  <br />  <br />                                  RESULTS <br />    Identification of Fli-I as a CARM1 binding protein by yeast <br /> two-hybrid screening. Bait plasmid pGBT9-CARM1, encoding <br /> full-length CARM1 fused to Gal4 DBD, and a 17-day-old <br /> mouse embryo cDNA library in Gal4 activation domain fusion <br /> vector pGAD10 were sequentially transformed into yeast <br /> strain Hf7c, which contains the yeast his3 gene and a lacZ gene <br /> under the control of Gal4 response elements. Selection of 1.1 <br /> &#226;&#171;&#187; 106 transformed yeast cells on agar plates lacking histidine <br /> resulted in the growth of 126 colonies, of which 25 clones were                       FIG. 1. Binding of CARM1 to Fli-I in vitro and in vivo. (A) The <br /> subsequently confirmed as positive by testing for the expres-                      LRR region (vertical stripes) and two gelsolin-like motifs (black boxes) <br /> sion of &#226;&#65533;&#164;-galactosidase. Two of the 25 clones encoded                              of Fli-I protein are indicated. (B) In vitro binding assay. [35S]Fli-I <br />                                                                                    protein synthesized in vitro was incubated with GST or GST-CARM1 <br /> CARM1, which is consistent with previous structural and func-                      immobilized on glutathione-agarose beads; bound proteins were <br /> tional studies indicating that the protein arginine methyltrans-                   eluted and analyzed by sodium dodecyl sulfate-polyacrylamide gel <br /> ferases exist as homodimers (45, 71) and which indicates the                       electrophoresis and autofluorography. A portion of the in vitro trans- <br /> validity of the bait and the yeast two-hybrid screening condi-                     lated Fli-I protein before incubation with the beads is shown at left <br />                                                                                    (Input, 10%). The position of the 145-kDa full-length Fli-I protein is <br /> tions for identification of protein partners of CARM1.                             indicated. (C) Coimmunoprecipitation assay. pSG5.Flag-FliI(LRR) or <br />    Among the other positive cDNA clones, two encoded an                            pSG5.Flag-FliI(gelsolin) (2.5 &#226;&#65533;&#174;g) was transiently transfected along <br /> N-terminal fragment of the 1,268-amino acid mouse Fli-I pro-                       with pSG5.HA-CARM1 or pSG5.HA-PRMT1 (2.5 &#226;&#65533;&#174;g) into Cos-7 <br /> tein (extending to codon 480). We used a full-length human                         cells. Immunoprecipitation (IP) was performed on transfected-cell ex- <br /> cDNA clone (8) for all studies described below. The N-termi-                       tracts with antibodies against the Flag epitope, and the precipitated <br />                                                                                    proteins were analyzed by immunoblotting (W) with anti-HA antibod- <br /> nal region of Fli-I contains an LRR domain consisting of 16                        ies (upper panel). A sample of the transfected-cell extract before <br /> tandem copies of a 23-amino-acid leucine-rich motif (Fig. 1A)                      immunoprecipitation (2% of the volume used for the immunoprecipi- <br /> (8, 31). LRR domains have been implicated in protein-protein                       tation shown in lane 3) is shown in lane 1. The relative expression of <br /> interactions. The central and C-terminal regions of the Fli-I                      HA-CARM1 and HA-PRMT1 before immunoprecipitation is shown <br />                                                                                    in the lower panel. Results are representative of two independent <br /> polypeptide chain are homologous to the gelsolin family of                         experiments. LRR, LRR fragment of Fli-I; GEL, gelsolin-like frag- <br /> actin binding proteins (34). Fli-I binds actin and colocalizes                     ment of Fli-I; and IgG, position of the immunoglobulin heavy chain on <br /> with the actin cytoskeleton under certain conditions but can                       the immunoblot. <br /> also localize to the nucleus under other conditions (16, 17). <br /> These studies suggest that Fli-I may have a nuclear function <br /> and in addition may be involved in signal-regulated remodeling                     alone (lane 2), and full-length Fli-I was enriched over the <br /> of or coupling of other proteins to actin or the actin cytoskel-                   smaller bands when the input and CARM1-bound fractions <br /> eton. Since we identified Fli-I in our two-hybrid screen by its                    were compared. These data suggest that Fli-I binds directly to <br /> ability to bind the NR coactivator CARM1, we confirmed the                         CARM1. <br /> binding by other methods and investigated the ability of Fli-I to                     To achieve more efficient expression and to test the proper- <br /> cooperate with CARM1 and other NR coactivators in enhanc-                          ties of specific Fli-I subdomains, Fli-I was subsequently divided <br /> ing transcriptional activation by NRs.                                             into two separate fragments: amino acids 1 to 494, containing <br />    Specific binding of Fli-I to CARM1 and NRs in vitro and in                      the LRR region; and amino acids 495 to 1268, containing the <br /> vivo. Fli-I protein translated in vitro was tested for its ability to              gelsolin-like domains. To test interactions in vivo, the Fli-I <br /> bind GST-CARM1 immobilized on glutathione-agarose beads                            fragments fused to a Flag epitope were expressed by transient <br /> (Fig. 1B). As reported previously (17), production of full-                        transfection in Cos-7 cells along with HA epitope-tagged <br /> length (145-kDa) Fli-I in this system was relatively inefficient                   CARM1 and coimmunoprecipitation experiments were per- <br /> (lane 1). There was relatively weak but specific binding of                        formed on the transfected-cell extracts by using antibodies <br /> full-length Fli-I to GST-CARM1 (lane 3), compared with GST                         against the Flag epitope and subsequently analyzing the im- <br />  2106     LEE ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br /> munoprecipitates by immunoblot with antibodies against the <br /> HA epitope (Fig. 1C, upper panel). When CARM1 was ex- <br /> pressed without any Fli-I fragment, it was not detected in the <br /> immunoprecipitates (lane 3); however, CARM1 coimmuno- <br /> precipitated with the N-terminal LRR domain of Fli-I (lane 4) <br /> as well as the C-terminal gelsolin-like domain of Fli-I (lane 7), <br /> suggesting the presence of at least two direct or indirect <br /> CARM1 binding sites on the Fli-I protein. <br />    To test the specificity of the interaction between Fli-I and <br /> CARM1, another member of the protein arginine methyltrans- <br /> ferase family, PRMT1 (38), was substituted for CARM1. <br /> There is substantial homology between CARM1 and PRMT1 <br /> in their 320-amino-acid methyltransferase domains (71), and <br /> both proteins can function as NR coactivators in cooperation <br /> with p160 coactivators (32). PRMT1 specifically coprecipitated <br /> with both fragments of Fli-I (Fig. 1C, upper panel, lanes 5, 6, <br /> and 8) but much more weakly than CARM1 (lanes 4 and 7). <br /> Immunoblots performed with anti-HA antibodies on samples <br /> of the transfected-cell extracts taken before immunoprecipita- <br /> tion showed that CARM1 and PRMT1 were expressed at sim- <br /> ilar levels (Fig. 1C, lower panel). <br />    Since Fli-I binds to NR coactivator CARM1, we also tested <br /> whether Fli-I may bind to NRs. Unlike full-length Fli-I (Fig. <br /> 1B), 35S-labeled LRR and gelsolin-like domains of Fli-I were <br /> synthesized efficiently in vitro (Fig. 2A, lanes 1 and 5). The <br /> LRR and gelsolin-like fragments did not bind to GST (lanes 2 <br /> and 6) and bound very weakly to a GST-ER fusion protein in <br /> the absence of E2 (lanes 3 and 7). The interaction of both Fli-I <br /> fragments with GST-ER was strongly enhanced by E2 (lanes 4 <br /> and 8). Coimmunoprecipitation experiments were used to test <br /> for the same interactions in Cos-7 cells transiently expressing <br /> ER with each Flag-tagged Fli-I fragment. When the Fli-I frag- <br /> ments or ER was expressed alone, anti-Flag antibodies did not          FIG. 2. Binding of NRs to Fli-I in vitro and in vivo. (A) In vitro <br /> precipitate ER (Fig. 2B, upper panel, lanes 1 and 2). However,      binding with ER. LRR and gelsolin-like fragments of Fli-I were trans- <br />                                                                     lated in vitro and were tested for binding to bead-bound GST or <br /> when ER was expressed with the LRR fragment or the gelso-           GST-ER in the absence or presence of E2. (B) Coimmunoprecipita- <br /> lin-like fragment of Fli-I, ER was coprecipitated by anti-Flag      tion with ER. Flag-tagged LRR or gelsolin-like fragments of Fli-I (2.5 <br /> antibody (lanes 3 to 6). Moreover, in contrast to the binding in    &#226;&#65533;&#174;g of plasmid) were coexpressed with ER (2.5 &#226;&#65533;&#174;g of plasmid) in Cos-7 <br /> vitro (Fig. 2A), the association between ER and the Fli-I frag-     cells in the presence or absence of E2. Immunoprecipitation (IP) by <br /> ments in vivo occurred in the presence or absence of E2 (Fig.       anti-Flag antibody was followed by immunoblotting (W) with anti-ER <br />                                                                     antibody (upper panel). Expression of ER in the transfected-cell ex- <br /> 2B, upper panel, lanes 3 to 6). Since the binding of Fli-I          tracts before immunoprecipitation (input) was examined by immuno- <br /> fragments to ER was hormone dependent in vitro but not in           blotting with anti-ER antibodies (lower panel). (C) Coimmunoprecipi- <br /> vivo, additional cellular factors may stabilize the interaction     tation with TR and CARM1. Coimmunoprecipitation of the Fli-I <br /> between ER and Fli-I in vivo, especially in the absence of          fragments with TR (2.5 &#226;&#65533;&#174;g of plasmid) or HA-tagged CARM1 (2.5 &#226;&#65533;&#174;g <br />                                                                     of plasmid) was tested as done for panel B in the presence and absence <br /> hormone. Alternatively, the overexpression of the proteins in       of T3, except that anti-TR (lanes 1 to 6) or anti-HA (lanes 7 and 8) was <br /> transient transfections could cause the hormone-independent         used for immunoprecipitation, and Fli-I fragments were detected on <br /> association.                                                        the subsequent immunoblot (W) with anti-Flag antibodies. Expression <br />    Interaction of the Fli-I fragments with another NR, TR, was      of LRR and gelsolin-like fragments of Fli-I in the transfected-cell <br /> also tested by coimmunoprecipitation; in this case, antibodies      extracts was examined before immunoprecipitation (input) by immu- <br />                                                                     noblot with anti-Flag antibodies (lower panel). <br /> against TR were used for immunoprecipitation, and antibodies <br /> against the Flag epitope tag were used to visualize the copre- <br /> cipitated Fli-I fragments. As with ER, TR coprecipitated with <br /> both the LRR fragment and the gelsolin-like fragment of Fli-I       test whether endogenous Fli-I protein is recruited to an NR- <br /> in a hormone-independent manner (Fig. 2C, upper panel,              regulated promoter in response to the appropriate hormone. <br /> lanes 1 to 6). The association of the Fli-I fragments with          For this purpose, two antibodies against Fli-I, one commer- <br /> CARM1 was used as a positive control (lanes 7 and 8).               cially available mouse monoclonal against the LRR region and <br />    Recruitment of endogenous Fli-I protein to a steroid hor-        one custom-made rabbit peptide-directed antiserum against a <br /> mone-regulated promoter in response to hormone. The asso-           unique portion of the gelsolin-like region, were shown by im- <br /> ciation of Fli-I with NRs and several NR coactivators suggests      munoblot and immunoprecipitation assays to recognize specif- <br /> that Fli-I may participate in the NR-dependent transcriptional      ically the Fli-I protein (data not shown; also see Fig. 7B). <br /> regulation process. Therefore, ChIP assays were performed to        Endogenous Fli-I protein was detected in MCF-7 breast cancer <br />  VOL. 24, 2004                                                           NUCLEAR RECEPTOR COACTIVATOR Flightless I                      2107 <br />  <br />  <br /> cells (data not shown), which contain the estrogen-inducible <br /> pS2 gene. When MCF-7 cells were treated with 100 nM E2 or <br /> were left untreated for 45 min and were analyzed by ChIP <br /> analysis, ER protein was recruited to pS2 promoter in a hor- <br /> mone-dependent manner (Fig. 3A and B). Similarly, hormone- <br /> dependent recruitment of Fli-I protein to the pS2 promoter <br /> region was observed when 0.5 &#226;&#65533;&#174;g of the mouse monoclonal <br /> antibody (Fig. 3A) or 5 &#226;&#65533;&#174;l of the rabbit antiserum (Fig. 3B) <br /> against Fli-I was used; however, when twice as much of either <br /> anti-Fli-I antibody was used, Fli-I protein appeared to be con- <br /> stitutively associated with the pS2 promoter (Fig. 3A, last two <br /> lanes, and B, not shown). These results suggest that small <br /> amounts of Fli-I protein are associated with the pS2 promoter <br /> even in the absence of hormone but that promoter occupancy <br /> by Fli-I is increased by E2 treatment. Normal IgG served as a <br /> negative control for the immunoprecipitations to demonstrate <br /> the specificity of the Fli-I antibodies (Fig. 3A and B). When <br /> PCR primers recognizing a portion of the coding sequence <br /> (nucleotides 871 to 1200, relative to the transcription start site) <br /> of the pS2 gene were used to test the same chromatin fractions <br /> immunoprecipitated with the antibodies against ER or Fli-I, <br /> no amplification products were observed (Fig. 3B, bottom <br /> panel), indicating that the recruitment of both ER and Fli-I is <br /> specific for the pS2 promoter, not for the coding region of the <br /> pS2 gene. Thus, both ER and Fli-I appear in the transcription <br /> initiation complex but do not appear to travel along with RNA <br /> polymerase as it progresses through the coding region. The <br /> recruitment of Fli-I protein to the pS2 promoter could be <br /> through the binding of Fli-I to ER or to one of the other <br /> coactivators, such as GRIP1 or CARM1. <br />    While these ChIP data indicate that both ER and Fli-I are <br /> recruited to pS2 promoters in the cell culture in response to <br /> hormone, they do not indicate whether Fli-I and ER molecules <br /> are present simultaneously on any individual pS2 promoter. <br /> ChIP and Re-IP assays were used to answer this question: first, <br /> the ChIP protocol was performed with the anti-ER antibody; <br /> the precipitated ER-containing chromatin fractions were                  FIG. 3. Estrogen-dependent recruitment of endogenous Fli-I to <br /> eluted from the antibody-bead complex with dithiothreitol,            the endogenous pS2 promoter in MCF-7 cells. (A and B) ChIP assay. <br /> diluted, and reimmunoprecipitated by using the two different          MCF-7 cells were incubated with or without E2 hormone for 45 min. <br /> antibodies against Fli-I or antibodies against a known ER-            Extracted chromatin was incubated with antibodies against the indi- <br />                                                                       cated proteins, and DNA purified from the immunoprecipitated (IP) <br /> associated coactivator. DNA was recovered from the second             fractions was analyzed by PCR with primers for the pS2 promoter or <br /> immunoprecipitated pellet and was analyzed by PCR with                coding region. Mouse monoclonal antibody against the Fli-I LRR <br /> primers against the pS2 promoter as before (Fig. 3C). By this         region (&#226;&#65533;&#163;-Fli-N) was used at 0.5 or 1 &#226;&#65533;&#174;g per assay, rabbit antiserum <br /> analysis, ER was shown to be simultaneously localized on in-          against a peptide from the gelsolin-like region of Fli-I (&#226;&#65533;&#163;-Fli-C) was <br />                                                                       used at 5 &#226;&#65533;&#174;l per assay, and all other antibodies were used at 1 &#226;&#65533;&#174;g per <br /> dividual hormone-activated pS2 promoters with Fli-I as well as        assay, including normal mouse or rabbit IgG. The results presented <br /> with each of three known ER coactivators, AIB1, CARM1, and            here are representative of seven independent experiments. (C) Simul- <br /> p300. When the ChIP and Re-IP procedure was performed                 taneous binding of pS2 promoter by ER and several coactivators. <br /> with the anti-Fli-I antibody for the first immunoprecipitation,       Chromatin from hormone-treated or untreated MCF-7 cells was im- <br />                                                                       munoprecipitated first with anti-ER antibody; immunoprecipitates <br /> Fli-I was shown to coincide with ER, AIB1, and CARM1 on               were eluted from the beads and were immunoprecipitated again with <br /> individual hormone-activated pS2 promoters, and CARM1 co-             antibodies against the specified proteins. Amounts of antibodies used: <br /> localized with Fli-I even on hormone-independent promoters            0.5 &#226;&#65533;&#174;g of &#226;&#65533;&#163;-Fli-N, 5 &#226;&#65533;&#174;l of &#226;&#65533;&#163;-Fli-C, and 1 &#226;&#65533;&#174;g of all other antibodies. <br /> (Fig. 3D). Since high levels of the Fli-I antibodies were used in     DNA purified from the immunoprecipitated fractions was analyzed by <br />                                                                       PCR with primers for the pS2 promoter. (D) Simultaneous binding of <br /> this experiment (Fig. 3D), the hormone-independent binding            pS2 promoter by Fli-I and ER or other coactivators. MCF-7 cell <br /> of Fli-I and CARM1 to the pS2 promoter observed in this               chromatin (E2 treated or untreated) was immunoprecipitated first by <br /> experiment is consistent with the results observed with high          1 &#226;&#65533;&#174;g of &#226;&#65533;&#163;-Fli-N; immunoprecipitates were eluted from the beads and <br /> levels of Fli-I antibodies in standard ChIP assays (Fig. 3A).         were immunoprecipitated again with 1 &#226;&#65533;&#174;g of antibodies against the <br />                                                                       specified proteins. DNA purified from the immunoprecipitated frac- <br /> The ChIP and Re-IP results indicate that endogenous Fli-I is <br />                                                                       tions was analyzed by PCR with primers for the pS2 promoter. <br /> part of a large ER-associated coactivator complex assembled <br /> on the hormone-activated pS2 promoter. <br />  2108     LEE ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Fli-I is a secondary coactivator for NRs. (A) Dependence of Fli-I coactivator activity on GRIP1 and ER. CV-1 cells were transfected <br /> with MMTV(ERE)-LUC reporter plasmid (250 ng) and with expression vectors encoding ER (100 ng), GRIP1 (250 ng), CARM1 (500 ng), and <br /> Fli-I (500 ng) as indicated elsewhere. Transfected cells were grown with E2, and luciferase activities of the transfected-cell extracts were <br /> determined. (B) Increasing coactivator activity with increasing Fli-I. Transfections were performed as done for panel A except that 200 ng of <br /> CARM1 vector and 200 to 600 ng of Fli-I vector were used. (C) Partial coactivator function by the Fli-I LRR domain. Transfections were <br /> performed with MMTV(TRE)-LUC reporter plasmid (250 ng) and expression vectors encoding TR (100 ng), GRIP1 (250 ng), CARM1 (500 ng), <br /> and full-length Fli-I or the LRR or gelsolin-like fragment (500 ng). Transfected cells were grown with T3 before harvesting for luciferase assays. <br /> Similar results were obtained with ER and the MMTV(ERE)-LUC reporter plasmid (data not shown). (D) Autonomous activation function in the <br /> Fli-I LRR domain. CV-1 cells were transfected with GK1 reporter plasmid (250 ng) encoding luciferase controlled by Gal4 response elements and <br /> expression vectors encoding Gal4 DBD or Gal4 DBD fused to the LRR domain (250 ng) and ER (500 ng). Transfected cells were grown with E2 <br /> as indicated. Results portrayed in panels A to D are representative of three or more independent experiments. <br />  <br />  <br />  <br />  <br />    Fli-I is a secondary coactivator for NRs. Since Fli-I binds to          reporter gene activity. In contrast to these high-NR conditions, <br /> NRs and the NR coactivator CARM1 and since Fli-I is re-                    use of much lower levels of NR expression vectors in the <br /> cruited to the pS2 promoter in response to hormone activation,             transfection results in much more stringent coactivator re- <br /> we tested the ability of Fli-I to serve as a coactivator for NRs,          quirements. Under low-NR conditions, the NR alone effects <br /> i.e., to enhance the ability of NRs to activate expression of              little or no reporter gene expression; instead, specific combi- <br /> reporter genes containing NR-responsive enhancer elements.                 nations of three coactivators are required along with the NR <br /> We have previously defined two different sets of transient-                and its hormone for efficient reporter gene expression. <br /> transfection conditions for testing the function of NR coacti-                When CV-1 cells were transfected with high levels of ER <br /> vators (32, 35). When standard, relatively high levels of NR               expression vector (100 ng per well of a six-well dish), GRIP1 <br /> expression vector are transfected into CV-1 cells, the NR alone            enhanced the ability of ER to activate reporter gene expression <br /> exhibits hormone-dependent activation of a suitable reporter               in the presence of E2 (Fig. 4A, assays 1 to 3). Addition of <br /> gene. The effects of some individual coactivators and some                 CARM1 or Fli-I along with GRIP1 further enhanced reporter <br /> pairs of coactivators can also be observed, but addition of a              gene expression (assays 4 and 5). The ability of CARM1 and <br /> third coactivator fails to elicit any further enhancement of               Fli-I to function as coactivators depended on the presence of <br />  VOL. 24, 2004                                                         NUCLEAR RECEPTOR COACTIVATOR Flightless I                  2109 <br />  <br />  <br /> both ER and GRIP1 (assays 6 to 9). Thus, as previously shown        previously for these low-NR conditions (35), TR alone did not <br /> for CARM1 (10, 11), Fli-I functioned as a secondary coactiva-       activate the reporter gene and coexpression of GRIP1 with TR <br /> tor; i.e., its ability to enhance NR function depended on the       caused little or no activation (Fig. 5A, assays 1 to 3). Coex- <br /> coexpression of a p160 coactivator such as GRIP1. In the            pression of GRIP1 with one other coactivator (CARM1, p300, <br /> presence of ER and GRIP1, the degree of enhancement of              Fli-I, or Fli-I fragments) also caused little or no enhancement <br /> reporter gene activity by Fli-I was roughly proportional to the     of TR-mediated reporter gene expression (assays 4 to 8). How- <br /> amount of Fli-I expression vector transfected (Fig. 4B). Under      ever, the coexpression of GRIP1 and CARM1 with p300 or <br /> similar high-NR conditions, Fli-I also functioned as a second-      Fli-I resulted in a dramatic synergistic enhancement of TR <br /> ary coactivator for TR (Fig. 4C, assays 1 to 5 and 8 and 9).        function (assays 9 and 10). Both domains of Fli-I were required <br /> Since the separate LRR and gelsolin-like domains of Fli-I each      for the GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli-I coactivator synergy (assays 10 to <br /> bound to CARM1 and NRs, we also tested the ability of these         12). <br /> Fli-I fragments to replace full-length Fli-I as coactivators. The      While the GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli-I combination was synergis- <br /> LRR fragment exhibited a weak GRIP1-dependent coactivator           tic, substitution of p300 or PRMT1 for CARM1 failed to pro- <br /> activity, compared with full-length Fli-I, but the gelsolin-like    duce synergy (Fig. 5B, assays 8 to 10). The synergy between <br /> fragment had no coactivator activity (Fig. 4C, assays 6 and 7       CARM1 and Fli-I was completely dependent on the coexpres- <br /> and 10 and 11). It is worth noting that the two fragments of        sion of GRIP1 (assay 11), confirming the roles of CARM1 and <br /> Fli-I are expressed at much higher levels in transient transfec-    Fli-I as secondary coactivators. Thus, Fli-I specifically cooper- <br /> tions than full-length Fli-I (data not shown); when this is taken   ated with CARM1 as a coactivator and did not cooperate with <br /> into account, full-length Fli-I is a much more powerful coac-       p300 or PRMT1. The functional cooperation of Fli-I with <br /> tivator than the LRR fragment of Fli-I, and thus, the efficient     CARM1 but not PRMT1 correlates with the binding specificity <br /> coactivator function of Fli-I depends upon both the LRR and         of Fli-I with CARM1 but not PRMT1 (Fig. 1) and thus sug- <br /> gelsolin-like domains.                                              gests that the binding of CARM1 and Fli-I may be important <br />    Autonomous activation function in the LRR domain of Fli-I.       for their functional cooperation. Although p300 and PRMT1 <br /> After coactivators are recruited to the promoter, they can          failed to cooperate functionally with Fli-I (Fig. 5B), they were <br /> transmit an activating signal to the transcription machinery via    still capable of functioning synergistically with other combina- <br /> an enzymatic activity (e.g., various histone modifications) or      tions of coactivators: p300 cooperated synergistically with <br /> through protein-protein contacts with downstream compo-             CARM1 and GRIP1 (Fig. 5A, assay 9) (35) and PRMT1 also <br /> nents of the transcription machinery, such as other coactiva-       functioned synergistically with CARM1 and GRIP1 (Fig. 5C, <br /> tors, chromatin components, basal transcription factors, or         assay 11) (32). Thus, the low-NR transient-transfection system <br /> RNA polymerase II. In some but not all cases, such signal           revealed functional collaborations between specific combina- <br /> transduction domains of coactivators have an autonomous ac-         tions of coactivators: when coexpressed with GRIP1, CARM1 <br /> tivation function that can activate a reporter gene when the        cooperated with p300 (Fig. 5A) (35), with Fli-I (Fig. 5A to C), <br /> activation function is fused to Gal4 DBD and the fusion pro-        and with PRMT1 (Fig. 5C) (32), but Fli-I did not cooperate <br /> tein is expressed with a reporter gene controlled by Gal4 re-       with p300 (Fig. 5B) or PRMT1 (Fig. 5B), and PRMT1 did not <br /> sponse elements (10, 32, 58). When the LRR domain of Fli-I          cooperate with p300 (35). <br /> was fused to Gal4 DBD, it enhanced expression of a Gal4-               To test the role of CARM1&#226;&#8364;&#8482;s protein arginine methyl- <br /> responsive reporter gene (Fig. 4D, assays 1 and 2). Coexpres-       transferase activity in the GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli-I synergy, a <br /> sion of ER substantially enhanced this activity in a hormone-       CARM1(E267Q) mutant was substituted for CARM1 in the <br /> independent manner (assays 3 and 4), which corresponds to           low-NR transient-transfection system. CARM1(E267Q) lacks <br /> the hormone-independent binding of Fli-I to ER in coimmu-           protein methyltransferase activity but retains the ability to bind <br /> noprecipitation studies (Fig. 2B). These results suggest that the   GRIP1, and substitution of this mutant for wild-type CARM1 <br /> autonomous activation function of Fli-I may contribute to the       almost completely eliminated the GRIP1-CARM1-p300 coac- <br /> coactivator function of Fli-I through interaction with down-        tivator synergy (35). The E267Q <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span> also eliminated the <br /> stream components of the transcription machinery; they fur-         GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli-I synergy with ER (Fig. 5C, assays 8 and <br /> ther suggest the functional relevance of the binding of Fli-I to    9) and with TR (data not shown), indicating that the CARM1 <br /> NRs.                                                                methyltransferase activity is essential for the coactivator syn- <br />    Coactivator synergy between CARM1 and Fli-I. Since               ergy between CARM1 and Fli-I. The failure of PRMT1 and <br /> CARM1 and Fli-I each functions as a secondary NR coactiva-          the CARM1(E267Q) mutant to cooperate with Fli-I was not <br /> tor in cooperation with GRIP1 (Fig. 4) and since CARM1 and          due to lack of their expression or to improper folding of the <br /> Fli-I specifically bind to each other (Fig. 1), we tested whether   CARM1 mutant, because PRMT1 exhibited synergy with wild- <br /> CARM1 and Fli-I can cooperate as coactivators for NRs. We           type CARM1 and because the CARM1 mutant retained par- <br /> employed the transient-transfection assay system with low lev-      tial synergistic activity with PRMT1, even though the mutant <br /> els of NR expression vector, since we previously established        was inactive in combination with Fli-I (Fig. 5C, assays 9 to 12). <br /> that these low-NR conditions are characterized by stringent         Thus, the CARM1 methyltransferase activity was essential for <br /> requirements for multiple coactivators and facilitate the obser-    synergistic cooperation of CARM1 with Fli-I but was less im- <br /> vation of synergistic interactions among multiple coactivators      portant for synergistic cooperation of CARM1 with PRMT1. <br /> (35). In this series of experiments with TR, all transfected           Coactivator function of Fli-I depends on the N-terminal <br /> cultures were exposed to thyroid hormone, since we previously       domain of GRIP1. Since the coactivator function of Fli-I was <br /> demonstrated that all coactivator activities depend on the pres-    completely dependent on coexpression of GRIP1, we tested <br /> ence of NR and the appropriate hormone. Also as shown               whether Fli-I binds to GRIP1 and whether such an interaction <br />  2110     LEE ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                         might explain the functional coactivator relationship between <br />                                                                         GRIP1 and Fli-I. Three fragments of GRIP1 were prepared as <br />                                                                         GST fusion proteins: GRIP1.N, amino acids 5 to 730, contain- <br />                                                                         ing the basic helix-loop-helix (bHLH) and two Per-ARNT-SIM <br />                                                                         (PAS) domains as well as the first two (of three) LXXLL <br />                                                                         motifs; GRIP1.M, amino acids 730 to 1121, containing the <br />                                                                         third LXXLL motif and the AD1 domain, which binds CBP <br />                                                                         and p300; and GRIP1.C, amino acids 1122 to 1462, containing <br />                                                                         the AD2 domain, which binds CARM1. The Fli-I LRR frag- <br />                                                                         ment translated in vitro bound specifically to immobilized <br />                                                                         GST-GRIP1.N (Fig. 6A, upper panel, lane 3), but the gelsolin- <br />                                                                         like fragment bound very weakly to the GRIP1 N-terminal <br />                                                                         region (lane 8). Only weak background-level binding of both <br />                                                                         Fli-I fragments was observed with GST (lanes 2 and 7), GST- <br />                                                                         GRIP1.M (lanes 4 and 9), or GST-GRIP1.C (lanes 5 and 10). <br />                                                                         As a positive control, we demonstrated that CARM1 trans- <br />                                                                         lated in vitro bound to GST-GRIP1.C (Fig. 6A, lower right <br />                                                                         panel), as shown previously (11). These results suggest a direct <br />                                                                         interaction between the LRR domain of Fli-I and the N-ter- <br />                                                                         minal region of GRIP1. <br />                                                                            To test for Fli-I-GRIP1 interactions in vivo, HA-tagged <br />                                                                         GRIP1 fragments were expressed by transient transfection in <br />                                                                         Cos-7 cells along with Flag-tagged Fli-I fragments. Immuno- <br />                                                                         precipitations were performed with anti-Flag antibodies, and <br />                                                                         coprecipitated GRIP1 fragments were visualized by immuno- <br />                                                                         blot with antibodies against the HA epitiope. As in the in vitro <br />                                                                         binding studies (Fig. 6A), the coimmunoprecipitation experi- <br />                                                                         ments demonstrated that the GRIP1 N-terminal region bound <br />                                                                         to the Fli-I LRR domain but not to the Fli-I gelsolin-like <br />                                                                         domain (Fig. 6B, lanes 5 and 8). However, while the C-termi- <br />                                                                         nal region of GRIP1 did not bind to either Fli-I domain in vitro <br />                                                                         (Fig. 6A), it bound to both domains of Fli-I in vivo (Fig. 6B, <br />                                                                         lanes 7 and 10). The middle domain of GRIP1 did not bind to <br />                                                                         either Fli-I domain (lanes 6 and 9). The combined results from <br />                                                                         the in vitro and in vivo binding studies suggest that the Fli-I <br />                                                                         LRR domain binds directly to the N-terminal region of <br />                                                                         GRIP1, while both the LRR and gelsolin-like domains of Fli-I <br />                                                                         bind indirectly to the C-terminal region of GRIP1. Since <br />                                                                         CARM1 binds to the C-terminal region of GRIP1 and since <br />                                                                         both the LRR and gelsolin-like domains of Fli-I can bind to <br />                                                                         CARM1, we propose that the binding of Fli-I to the GRIP1 <br />                                                                         C-terminal region observed in vivo is mediated through endog- <br />                                                                         enous CARM1, which is present at relatively high levels in <br />                                                                         Cos-7 cells (data not shown). <br />                                                                            While Fli-I physically interacts with NRs, CARM1, and <br />                                                                         GRIP1, the functional dependence of Fli-I on GRIP1 (Fig. 4 <br />                                                                         and 5) led us to test whether the observed binding of Fli-I to <br />                                                                         the N-terminal region of GRIP1 (Fig. 6A and B) is important <br />                                                                         for the functional collaboration between these two coactiva- <br />                                                                         tors. We also tested whether other known functional domains <br />                                                                         of GRIP1 are required for GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli-I coactivator <br />                                                                         synergy. Deletion mutants of GRIP1 lacking the N-terminal <br />                                                                         domain (&#226;&#338;&#172;N, lacking amino acids 1 to 562), the AD1 domain <br />  <br />   FIG. 5. Synergistic coactivator function between Fli-I and CARM1. <br /> CV-1 cells were transfected with MMTV(TRE)-LUC (250 ng) (A and <br /> B) or MMTV(ERE)-LUC (250 ng) (C) and expression vectors encod- <br /> ing TR (1 ng) or ER (1 ng), GRIP1 (250 ng), CARM1 wild type or          not with PRMT1 or p300. (C) Methyltransferase activity of CARM1 <br /> E267Q mutant (500 ng), PRMT1 (500 ng), p300 (500 ng), and full-         required for synergy with Fli-I. Transfected cells were grown with T3 <br /> length Fli-I or the LRR or gelsolin-like fragment (500 ng). (A) Full-   (A and B) or E2 (C) before harvest. Results in each panel are repre- <br /> length Fli-I required for synergy. (B) Fli-I synergy with CARM1 but     sentative of three or more independent experiments. <br />  VOL. 24, 2004                                                            NUCLEAR RECEPTOR COACTIVATOR Flightless I                          2111 <br />  <br />  <br />                                                                        (&#226;&#338;&#172;AD1, lacking amino acids 1057 to 1109), or the AD2 domain <br />                                                                        (&#226;&#338;&#172;AD2, lacking amino acids 1122 to 1462) were coexpressed <br />                                                                        with CARM1, Fli-I, and low levels of TR in transient-transfec- <br />                                                                        tion assays. As shown above (Fig. 5), reporter gene activation <br />                                                                        by hormone-activated TR under these conditions was synergisti- <br />                                                                        cally enhanced by wild-type GRIP1, CARM1, and Fli-I, and all <br />                                                                        three coactivators were required for efficient reporter gene ex- <br />                                                                        pression (Fig. 6C, assays 1 to 15). In contrast, the GRIP1&#226;&#338;&#172;N <br />                                                                        mutant had no activity when coexpressed with CARM1 and Fli-I <br />                                                                        (assay 16); the GRIP1&#226;&#338;&#172;AD1 and GRIP1&#226;&#338;&#172;AD2 mutants exhibited <br />                                                                        weak partial activity in cooperation with CARM1 and Fli-I <br />                                                                        (assays 17 and 18). All three mutants were expressed at levels <br />                                                                        equal to or exceeding that of wild-type GRIP1 (data not <br />                                                                        shown) (42). Thus, the GRIP1 N-terminal domain, which is the <br />                                                                        direct binding site for Fli-I, was absolutely required for <br />                                                                        GRIP1-CARM1-Fli-I coactivator synergy. The strong require- <br />                                                                        ment for the N-terminal domain, which is not essential for the <br />                                                                        binding of GRIP1 to NR, p300, or CARM1 (11, 29, 42), sug- <br />                                                                        gests that the interaction between the GRIP1 N-terminal do- <br />                                                                        main and Fli-I is important for the recruitment of Fli-I to the <br />                                                                        coactivator complex and for the functional synergy between <br />                                                                        GRIP1 and Fli-I. The partial requirement for AD1 suggests <br />                                                                        that endogenous p300 or CBP, which binds to AD1, may con- <br />                                                                        tribute to the GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli-I synergy. Although the <br />                                                                        AD2 domain was important for achieving full GRIP1&#226;&#8364;&#8220; <br />                                                                        CARM1&#226;&#8364;&#8220;Fli-I synergy, the fact that it was not absolutely re- <br />                                                                        quired was somewhat surprising, since CARM1 binds to the <br />                                                                        AD2 region of GRIP1; the fact that CARM1 can bind to Fli-I <br />                                                                        may provide an alternative mechanism for recruiting CARM1 <br />                                                                        to the coactivator complex and thus for achieving the partial <br />                                                                        coactivator synergy observed with the GRIP1&#226;&#338;&#172;AD2 mutant. <br />                                                                        This interpretation is supported by the fact that the AD2 do- <br />                                                                        main is absolutely required for synergy between GRIP1, <br />                                                                        CARM1, and p300 (35); i.e., since Fli-I was not included in <br />                                                                        that case, there was no alternative binding site for CARM1. <br />                                                                           siRNAs for Fli-I suppress ER-mediated transcription. To <br />                                                                        further test the physiological relevance of Fli-I as a coactivator, <br />                                                                        siRNAs directed toward the LRR domain of Fli-I were de- <br />                                                                        signed and tested for effects on ER transcription. The speci- <br />                                                                        ficity of these siRNAs was tested by cotransfecting them into <br />                                                                        Cos-7 cells with separate vectors encoding the Flag-tagged <br />                                                                        LRR and gelsolin-like domains of Fli-I. In an immunoblot <br />                                                                        analysis of the transfected-cell extracts with anti-Flag antibody, <br />  <br />  <br />  <br />  <br />                                                                        fragments to GRIP1 fragments in vivo. Coimmunoprecipitations were <br />                                                                        performed as done for Fig. 1C. Cos-7 cells were transfected with <br />                                                                        pSG5.Flag-FliI(LRR) or pSG5.Flag-FliI(gelsolin) (3 &#226;&#65533;&#174;g) and pSG5.HA <br />                                                                        plasmid encoding GRIP1N, GRIP1 M, or GRIP1C (3 &#226;&#65533;&#174;g). Immunopre- <br />                                                                        cipitation (IP) was performed with anti-Flag antibodies, and immunoblot- <br />                                                                        ting (W) was performed with anti-HA antibodies. Immunoblotting of <br />   FIG. 6. Coactivator activity of Fli-I depends on the N-terminal      immunoprecipitated proteins is shown in the upper panel, and immuno- <br /> domain of GRIP1. (A) LRR domain of Fli-I binds to GRIP1 N-             blotting of the cell extracts before immunoprecipitation is shown in the <br /> terminal region in vitro. In vitro binding assays were performed as    lower panel. (C) Requirement for GRIP1 N-terminal region in coactiva- <br /> done for Fig. 1B. Upper panel tests the binding of 35S-labeled Fli-I   tor synergy. CV-1 cells were transiently transfected with MMTV(TRE)- <br /> LRR and gelsolin-like fragments to GST and GST fused to GRIP1          LUC (250 ng) and expression vectors encoding TR (1 ng), CARM1 (500 <br /> N-terminal (GRIP1.N, amino acids 5 to 765), middle (GRIP1.M,           ng), Fli-I (500 ng), and the wild type or indicated mutant of GRIP1 (250 <br /> amino acids 730 to 1121), and C-terminal (GRIP1.C, amino acids 1122    ng): GRIP1&#226;&#338;&#172;N lacked amino acids 1 to 562, &#226;&#338;&#172;AD1 lacked amino acids <br /> to 1462) fragments. Lower right panel shows binding of 35S-labeled     1057 to 1109, and &#226;&#338;&#172;AD2 lacked amino acids 1122 to 1462. Transfected <br /> CARM1 to GST-GRIP1C. Lower left panel shows Coomassie-stained          cells were grown with T3 before harvest. Results are representative of two <br /> gel of the GST-GRIP1 fusion proteins used. (B) Binding of Fli-I        independent experiments. <br />  2112     LEE ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> the two siRNAs for Fli-I completely or partially inhibited the <br /> expression of the LRR domain (Fig. 7A, lanes 2 to 4), whereas <br /> siRNAs for AIB1 and CARM1 did not influence the expres- <br /> sion of LRR (lanes 5 and 6). In contrast, the two siRNAs for <br /> Fli-I had no effect on the expression of the Fli-I gelsolin-like <br /> domain (lanes 8 and 9), confirming the specificity of the Fli-I <br /> siRNAs. <br />    Next, the effects of the same siRNAs on endogenous pro- <br /> teins were investigated by transfecting them into MCF-7 cells <br /> and by monitoring the expression levels of endogenous pro- <br /> teins by immunoblot. Endogenous Fli-I protein was specifically <br /> reduced by the two siRNAs for Fli-I but not by siRNAs for <br /> AIB1 or CARM1 (Fig. 7B, top panel). Endogenous CARM1 <br /> expression was inhibited by siRNA for CARM1 but not by the <br /> other siRNAs (middle panel). The endogenous actin level was <br /> not changed by any of the siRNAs. Thus, the siRNAs tested <br /> here are efficient and specific in reducing endogenous protein <br /> expression. <br />    The effect of these siRNAs on transcriptional activation by <br /> endogenous ER was investigated in MCF-7 cells. MCF-7 cells <br /> were transfected with an estrogen-responsive reporter plasmid, <br /> and the transcription activation by endogenous ER was deter- <br /> mined by reporter analysis. The estrogen-responsive reporter <br /> gene activation by endogenous ER was significantly enhanced <br /> by E2 (Fig. 7C, assays 1 and 6). Cotransfection of the reporter <br /> plasmid together with either Fli-I siRNA, the CARM1 siRNA, <br /> or the AIB1 siRNA resulted in loss of most of the estrogen- <br /> dependent reporter gene expression (assays 6 to 14), indicating <br /> that endogenous AIB1, CARM1, and Fli-I were all required <br /> for endogenous ER function. The siRNAs for Fli-I were re- <br /> producibly more efficient in inhibiting ER activity than were <br /> the CARM1 and AIB1 siRNAs (data not shown). siRNA di- <br /> rected against an irrelevant sequence had no effect on reporter <br /> gene activation by E2 and endogenous ER (data not shown). <br /> These data demonstrate the importance of Fli-I as a coactiva- <br /> tor for the action of endogenous ER. <br />    Role of actin binding activity of Fli-I gelsolin-like domains <br /> in coactivator function. Because the gelsolin-like domain of <br /> Fli-I is important for Fli-I coactivator function (Fig. 5A) and <br /> also can bind to actin (16, 17, 26, 39), we questioned whether <br /> the actin binding activity of Fli-I may be important for its <br /> coactivator function. In fact, some actin binding proteins, in- <br /> cluding supervillin and gelsolin itself, have already been shown <br /> to bind to and enhance transcriptional activation by NRs (28,            FIG. 7. Requirement for Fli-I in MCF-7 cells for ER-dependent gene <br />                                                                       expression. (A) Specificity of LRR-directed siRNAs. Cos-7 cells (in 12- <br /> 59). Actin and actin-like proteins are components of a number         well culture dishes with 1 ml of medium per well) were transfected with a <br /> of nuclear complexes involved in transcription and its regula-        pSG5.Flag vector encoding the LRR or gelsolin-like domain of Fli-I (1 <br /> tion, including the SWI/SNF ATP-dependent chromatin-re-               &#226;&#65533;&#174;g) along with 5 &#226;&#65533;&#174;l of 20 &#226;&#65533;&#174;M siRNAs. Forty hours after transfection, cell <br /> modeling complex (4, 52, 53) (see Discussion). Like the protein       extracts were analyzed by immunoblotting (W) with anti-Flag antibody. <br />                                                                       Lane 1 shows extract from cells transfected with empty vector and no <br /> gelsolin, the gelsolin-like domain of Fli-I consists of two gel-      siRNA. (B) Reduction of endogenous Fli-I protein level in MCF-7 cells by <br /> solin-like motifs, each of which contains three repeats of an         siRNAs for Fli-I. The indicated siRNAs were transfected as done for <br /> actin binding motif (8). In coimmunoprecipitation assays, we          panel A into MCF-7 cells, and endogenous protein expression was ana- <br /> found that the first of the two gelsolin-like motifs in Fli-I         lyzed by immunoblotting (W) with the antibody indicated on the right. <br /> (Gel.A, amino acids 495 to 827) can bind to Brg- or Brm-              (C) siRNAs for Fli-I suppress gene activation by endogenous ER and <br />                                                                       coactivators in MCF-7 cells. MMTV(ERE)-LUC reporter plasmid (1 &#226;&#65533;&#174;g) <br /> associated factor 53 (BAF53), an actin-like component of the          was transfected into MCF-7 cells in 12-well culture dishes containing 1 ml <br /> SWI/SNF complex (Fig. 8A, upper panel). We next tested                of medium along with the indicated siRNAs (5 or 10 &#226;&#65533;&#174;l of 20 &#226;&#65533;&#174;M siRNA). <br /> whether expression of the Gel.A fragment of Fli-I would either        Before transfection, MCF-7 cells were maintained in phenol red-free <br /> enhance (i.e., serve as a coactivator for) or inhibit (i.e., act as   Dulbecco&#226;&#8364;&#8482;s modified Eagle medium with 5% charcoal-dextran-treated <br />                                                                       fetal bovine serum for at least 4 days. After transfection of siRNAs, <br /> a dominant-negative mutant to interfere with the action of) ER        MCF-7 cells were treated with 20 nM E2 as indicated. Luciferase activity <br /> function. Coexpression of the Gel.A fragment of Fli-I with ER         was determined 48 h after transfection. The data shown are representative <br /> in transient-transfection assays enhanced estrogen-dependent          of 10 independent experiments. <br />  VOL. 24, 2004                                                                 NUCLEAR RECEPTOR COACTIVATOR Flightless I                  2113 <br />  <br />  <br />                                                                             activity in the coactivator function of the Gel.A fragment of <br />                                                                             Fli-I, two point mutations were generated in the first of the <br />                                                                             three putative actin binding repeats within Gel.A. The E585K <br />                                                                             mutant of Gel.A was designed according to a previously re- <br />                                                                             ported mutation in the gelsolin protein that is defective in actin <br />                                                                             binding (64); the G602S mutation of Gel.A corresponds to the <br />                                                                             Drosophila fliI3 mutation, which causes a partial defect in Dro- <br />                                                                             sophila muscle development and results in the flightless phe- <br />                                                                             notype (18). In coimmunoprecipitation assays, the G/S muta- <br />                                                                             tion had no effect on binding of Gel.A to BAF53, but the E/K <br />                                                                             mutation reduced Gel.A binding to BAF53 (Fig. 8A, upper <br />                                                                             panel). In contrast, both mutants retained the ability of the <br />                                                                             wild-type Gel.A fragment to bind ER (Fig. 8A, lower panel); <br />                                                                             the retention of ER-binding activity shows that both mutants <br />                                                                             are folded properly and that the surfaces of Gel.A that interact <br />                                                                             with actin and with ER are distinct. <br />                                                                                The Gel.A wild-type and mutant proteins were compared <br />                                                                             for their ability to serve as coactivators for ER. The G/S mu- <br />                                                                             tant was equivalent to the wild-type Gel.A fragment in its <br />                                                                             ability to enhance estrogen-dependent activation of a tran- <br />                                                                             siently transfected reporter gene by ER; however, the coacti- <br />                                                                             vator function of the E/K mutant was substantially reduced <br />                                                                             (Fig. 8B). The reduced activity of the E/K mutant was not due <br />                                                                             to improper folding or diminished expression or stability, as <br />                                                                             shown by the ability of this mutant to bind ER normally (Fig. <br />                                                                             8A). The specific, correlated loss of actin binding (64) and of <br />                                                                             binding to actin-like BAF53 with loss of coactivator function in <br />                                                                             Gel.A mutants suggests that binding of actin or actin-like pro- <br />                                                                             teins is important for the coactivator function of the Gel.A <br />                                                                             fragment and, by extension, the coactivator function of Fli-I. <br />  <br />  <br />                                                                                                      DISCUSSION <br />                                                                                Fli-I is a novel secondary coactivator for NRs. Fli-I can <br />                                                                             function in cooperation with specific combinations of other NR <br />    FIG. 8. Role of actin binding activity of Fli-I gelsolin-like domains    coactivators to enhance the ability of NRs to activate transcrip- <br /> in coactivator function. (A) Binding of Fli-I Gel.A domain (wild type       tion of NR-regulated reporter genes (Fig. 4 and 5). Fli-I was <br /> and mutants) to actin-like molecule BAF53 and to ER. Vectors en-            identified because of its binding to CARM1 (Fig. 1), but it also <br /> coding the human Fli-I gelsolin-like Gel.A fragment (amino acids 495 <br /> to 827) and two point mutants (E585K and G602S) (2.5 &#226;&#65533;&#174;g) were <br />                                                                             binds specifically in vitro and in vivo to GRIP1 (Fig. 6) and to <br /> transfected into Cos-7 cells along with vectors encoding the actin-like     at least two NRs (Fig. 2). Which of these protein-protein in- <br /> protein BAF53 (2.5 &#226;&#65533;&#174;g, upper panel) or ER (2.5 &#226;&#65533;&#174;g, lower panel). Cell       teractions is important for the coactivator function of Fli-I? <br /> lysates were prepared and immunoprecipitated (IP) with antibodies           Since Fli-I by itself did not enhance the activity of NRs but <br /> against HA or ER. The precipitated fractions were analyzed by immu-         required the presence of a p160 coactivator (Fig. 4 and 5), the <br /> noblotting (W) with anti-Flag antibody. (B) Coactivator activity of <br /> wild-type and mutant Fli-I Gel.A fragments. pSG5.Flag vectors encod-        binding of Fli-I to NRs is apparently not sufficient for Fli-I <br /> ing wild-type or mutant Fli-I Gel.A fragments (0.5 &#226;&#65533;&#174;g) were trans-          coactivator function. However, the ability of ER to enhance <br /> fected into CV1 cells with 0.1 &#226;&#65533;&#174;g of ER expression vector and 0.25 &#226;&#65533;&#174;g       the autonomous activation function of the Fli-I LRR domain <br /> of MMTV(ERE)-LUC reporter plasmid; E2 was added after transfec-             (Fig. 4D) and the ability of the gelsolin-like Gel.A fragment of <br /> tion, and luciferase activity was determined 40 h after transfection. The <br /> data shown are representative of seven independent experiments. <br />                                                                             Fli-I to enhance ER function (Fig. 8B) indicate the existence of <br />                                                                             a functional relevance to the ER-Fli-I interaction and thus <br />                                                                             suggest that the Fli-I-NR interaction may contribute to the <br />                                                                             ability of Fli-I to serve as an NR coactivator. <br /> expression of a transient reporter gene (Fig. 8B, assays 1 to 4                Enhancement of NR function by Fli-I was observed only <br /> and 13).                                                                    when GRIP1 was coexpressed with Fli-I, whether or not addi- <br />    In previous mutational studies with gelsolin, the first puta-            tional coactivators such as CARM1 and p300 were also coex- <br /> tive actin binding repeat (of three) within the first gelsolin-like         pressed (Fig. 4 and 5). The dependence of Fli-I coactivator <br /> motif (of two) was found to be more important for binding                   function on GRIP1 indicates that Fli-I is a secondary coacti- <br /> G-actin, while the second and third repeats in the first motif              vator and therefore that the binding of Fli-I to GRIP1 is <br /> were shown to be more important for binding F-actin (8). The                essential for the coactivator function of Fli-I. While Fli-I <br /> Gel.A fragment of Fli-I represents the first of the two gelsolin-           bound to both the N-terminal and C-terminal regions of <br /> like motifs in Fli-I. To test the importance of actin binding               GRIP1 in coimmunoprecipitation experiments (Fig. 6B), Fli-I <br />  2114     LEE ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> bound only to the N-terminal region of GRIP1 in vitro (Fig.          Fli-I did not function synergistically with the protein acetyl- <br /> 6A), suggesting a direct interaction with the N-terminal region      transferase p300 or with PRMT1 (Fig. 5B and C), another <br /> of GRIP1 and an indirect binding to the C-terminal region of         member of the protein arginine methyltransferase family that <br /> GRIP1. Furthermore, the cooperative coactivator activity of          methylates different protein substrates from CARM1 (57). The <br /> Fli-I with CARM1 and GRIP1 was absolutely dependent on               ability of Fli-I to bind strongly to CARM1 but only weakly to <br /> the N-terminal region of GRIP1 but was only partially depen-         PRMT1 suggests that the binding between CARM1 and Fli-I <br /> dent on the C-terminal AD1 and AD2 regions of GRIP1 (Fig.            contributes to the selective functional synergy observed be- <br /> 6C). Thus, the binding of Fli-I to the N-terminal region of          tween Fli-I and CARM1 but not PRMT1. The synergy be- <br /> GRIP1 appears to be the critical binding interaction that po-        tween CARM1 and Fli-I required the methyltransferase activ- <br /> tentiates Fli-I recruitment to the target gene promoter. While       ity of CARM1 (Fig. 5C). The target protein(s) that must be <br /> we have no definitive data at present as to whether the Fli-I-       methylated by CARM1 to achieve coactivator synergy could <br /> CARM1 binding is necessary for Fli-I coactivator function, the       include histone H3, which is known to be methylated by <br /> functional synergy between these two coactivators (Fig. 5) sug-      CARM1 during steroid hormone activation of stably inte- <br /> gests that this binding interaction may well contribute. The         grated genes (41), and/or another presently unknown target, <br /> contacts between Fli-I and multiple components in the tran-          such as a component of the transcription machinery or another <br /> scription complex (NRs, GRIP1, and CARM1) may result in a            coactivator. While both CARM1 and Fli-I are apparently re- <br /> more efficient recruitment of Fli-I to the promoter, a more          cruited to the promoter through their interaction with GRIP1, <br /> stable coactivator complex, or a more highly functional con-         CARM1 binds to the C-terminal AD2 domain of GRIP1 (10, <br /> formation of the coactivator complex.                                11), whereas Fli-I binds to the N-terminal region of GRIP1 <br />    Both the LRR and gelsolin-like domains of Fli-I were re-          (Fig. 6). This observation and the synergistic cooperation be- <br /> quired for its coactivator function. While the LRR domain            tween CARM1 and Fli-I indicate that these two coactivators <br /> alone had modest coactivator function in cooperation with            have distinct but complementary roles in the transcriptional <br /> GRIP1 at high levels of NRs (Fig. 4C), both Fli-I domains were       activation process. That is, they mediate the functions of two <br /> required absolutely under low-NR conditions (Fig. 5A). The           different regions of GRIP1 and also contribute to the tran- <br /> binding studies suggest that the LRR domain is primarily or          scriptional activation process by two different mechanisms. <br /> solely responsible for the critical Fli-I-GRIP1.N interaction        CARM1 methylates histones, p300/CBP, and possibly other <br /> (Fig. 6). It appears that the binding of Fli-I to CARM1 and          proteins in the transcription initiation complex (41, 69) and <br /> NRs also contributes to the cooperative coactivator function of      also may recruit additional coactivators (58). The complemen- <br /> Fli-I and that both the LRR and gelsolin-like domains partic-        tary role of Fli-I could involve a specific protein (presently <br /> ipate in these binding interactions. The autonomous transcrip-       unknown) that interacts with and is the target of the autono- <br /> tional activation activity in the LRR domain may explain why         mous transcriptional activation domain in the LRR region of <br /> this domain exhibited modest coactivator function with GRIP1         Fli-I; in addition, the ability of the Fli-I gelsolin-like domain to <br /> under high-NR conditions and also suggests that this activation      bind actin (16, 17) and actin-like proteins may be important for <br /> function may play a role in Fli-I coactivator activity, presum-      the coactivator function of Fli-I (Fig. 8; also see additional <br /> ably through protein-protein interactions with other compo-          discussion below). <br /> nents of the transcription machinery.                                   The cooperation of Fli-I with GRIP1 also provides addi- <br />    Two other proteins previously found to bind to the Fli-I          tional insight into the mechanism of coactivator function by <br /> LRR domain include FLAP1 (also called LRRFIP2) and                   GRIP1. The N-terminal 350 amino acids of GRIP1 and the <br /> FLAP2 (also called TRIP, LRRFIP1, and GCF2) (9). Evi-                other two members of the p160 coactivator family is the most <br /> dence to date suggests that FLAP2 can bind specific RNA              highly conserved region of the family (2), but the role of this <br /> sequences but can also function as a specific DNA-binding            region in p160 coactivator function is unclear. One basic helix- <br /> transcriptional repressor. Binding of Fli-I to the transcriptional   loop-helix (bHLH) and two Per-Arnt-Sim (PAS) motifs are <br /> regulator FLAP2 is interesting in light of, and also consistent      found in this region. Recently, several proteins that bind to this <br /> with, our conclusions that Fli-I can function as a transcriptional   region of p160 coactivators have been identified. Some of these <br /> coregulator for NRs. Separate studies found that the LRR             proteins, including TEF4 (5) and MEF2C (14), are DNA- <br /> domain of the Caenorhabditis elegans Fli-I homologue can bind        binding transcriptional activators, which apparently recruit <br /> to Ras but not to other small GTPases (9). While a similar           p160 coactivators by binding to the bHLH-PAS region; thus <br /> binding interaction in higher organisms has yet to be con-           p160 coactivators can function as coactivators for many differ- <br /> firmed, the interaction in C. elegans has interesting implica-       ent classes of transcriptional activator proteins in addition to <br /> tions for possible regulation of Fli-I coactivator activity by       NRs. Other coactivators, including BAF57 (4) and MMS19 <br /> signaling pathways involving specific small GTPases.                 (66), also bind to the bHLH-PAS region of p160 coactivators. <br />    Characterization of CARM1 and GRIP1 function by identi-           While these findings suggest that the N-terminal bHLH-PAS <br /> fication of interacting proteins. The mechanisms by which            region may contribute to p160 coactivator function by recruit- <br /> CARM1 and GRIP1 enhance NR function involve the ability              ing additional coactivators to the promoter, a clear demonstra- <br /> of CARM1 and GRIP1 to bind other proteins and transmit the           tion of this relationship has not previously been shown. The <br /> transcriptional activation signal to them or cooperate with          fact that Fli-I binds to the N-terminal region of GRIP1 and <br /> them as coactivators (10, 35, 42, 58). Thus, our data indicate       cooperates synergistically with GRIP1 as a coactivator demon- <br /> that the coactivator function of CARM1 may involve its ability       strates a functional relevance for the binding of Fli-I to GRIP1 <br /> to bind to and function synergistically with Fli-I. The synergis-    and indicates that one role of the N-terminal region of GRIP1 <br /> tic cooperation between CARM1 and Fli-I was specific, since          is to recruit Fli-I as a coactivator, just as the AD1 domain of <br />  VOL. 24, 2004                                                           NUCLEAR RECEPTOR COACTIVATOR Flightless I                          2115 <br />  <br />  <br /> p160 coactivators recruits p300 or CBP and just as the AD2 <br /> domain of p160 coactivators recruits CARM1. Thus, our re- <br /> sults demonstrate that the N-terminal region of p160 coacti- <br /> vators can serve as an additional activation or signal transmis- <br /> sion domain, along with AD1 and AD2. <br />    The p160 coactivators appear to serve as scaffold proteins <br /> that contact NRs, as well as other classes of DNA-binding <br /> transcriptional activator proteins, directly and appear to as- <br /> semble a complex of other coactivators on the promoter (25, <br /> 46). The specific complement of coactivators required may <br /> vary for different promoters and in different cell types. For           FIG. 9. Proposed model for cooperative linking of p160 and SWI/ <br /> example, our results show that different domains of GRIP1 are         SNF coactivator complexes by Fli-I protein. Fli-I protein can bind to <br /> important to potentiate synergy between different combina-            components of the p160 coactivator complex (p160 and CARM1), <br /> tions of coactivators. When GRIP1 acts synergistically with           which contains histone acetylating (CBP/p300) and methylating <br />                                                                       (CARM1) activities. Fli-I can also bind to actin and the actin-like <br /> CARM1 and p300, the AD2 domain is absolutely required,                protein BAF53, both of which are components of the ATP-dependent <br /> while deletion of the AD1 domain causes only partial loss of          nucleosome-remodeling complex SWI/SNF. Thus, one possible mech- <br /> synergy (35). In contrast, when GRIP1 acts synergistically with       anism for the contribution of Fli-I as a coactivator could be to facilitate <br /> CARM1 and Fli-I, the AD1 and AD2 domains both contribute              the recruitment of or somehow coordinate the complementary chro- <br />                                                                       matin-remodeling activities of the SWI/SNF complex and the p160 <br /> but are not absolutely required (Fig. 6C). Thus, the specific <br />                                                                       coactivator complex. H, hormone. <br /> domains of p160 coactivators required may vary depending on <br /> what complex of secondary coactivators is available and/or <br /> assembled on the promoter. In addition, since AD2 is the <br /> direct binding site for CARM1, the reduced requirement for            nisms, i.e., ATP-dependent nucleosome remodeling and his- <br /> the AD2 domain when Fli-I is present with CARM1 suggests              tone modifications, respectively (25, 46). Overexpression of <br /> that CARM1 can be recruited to the p160 coactivator indi-             BAF57 was shown to enhance NR function in a cell line with <br /> rectly by binding to Fli-I, which is bound to the N-terminal          low levels of endogenous BAF57 (4), but whether the p160- <br /> region of the p160 coactivator.                                       binding ability of BAF57 is important for the coactivator func- <br />    Implications of the actin binding properties of Fli-I for its      tions of BAF57 and the SWI/SNF complexes is not yet deter- <br /> coactivator function. The gelsolin-like motifs of Fli-I can bind      mined. <br /> to actin in both the G (globular) and F (polymerized) forms,             A role for unknown actin binding proteins in the regulated <br /> and Fli-I was observed by immunofluorescence to associate             recruitment of SWI/SNF complexes to chromatin was previ- <br /> with the actin cytoskeleton under certain cellular conditions         ously proposed (72). We have observed that Fli-I not only <br /> (16, 17). The association with actin and the subcellular local-       binds actin (17) but also binds the actin-like BAF53 in coim- <br /> ization of Fli-I appear to be regulated by pathways involving         munoprecipitation experiments (Fig. 8A), as well as p160 co- <br /> phosphatidylinositol 3-kinase and Ras-related small GTPases;          activators (Fig. 6). Thus, Fli-I, like BAF57, could help to se- <br /> for example, in cells grown in low serum concentrations Fli-I         cure the association of an SWI/SNF complex to a p160 <br /> was primarily nuclear, but at higher serum concentrations, Fli-I      coactivator complex and thus help to coordinate the comple- <br /> was found in the cytoplasmic compartment associated with the          mentary ATP-dependent nucleosome-remodeling activity of <br /> cytoskeleton (17). These observations are consistent with reg-        the SWI/SNF complex with the histone acetylating (e.g., from <br /> ulated dual roles for Fli-I as a transcriptional coactivator in the   CBP and p300) and methylating (e.g., from CARM1 and <br /> nucleus and a cytoplasmic modulator of the cytoskeleton; fur-         PRMT1) activities of the p160 coactivator complex (Fig. 9). <br /> thermore, they suggest that Fli-I may function as a communi-             Recently, another actin binding protein, supervillin, was re- <br /> cation link through which cytoskeleton structure may influence        ported as a coactivator for some NRs (59). The actin binding <br /> transcription.                                                        motif of supervillin, like that of Fli-I, is highly homologous to <br />    In addition to its role as a cytoskeleton structural protein,      gelsolin. In addition, our yeast two-hybrid screen for CARM1 <br /> previous studies suggested roles for actin in the transcription       binding proteins identified not only Fli-I but also profilin 1 <br /> process (21, 54). Actin and actin-like proteins are found in the      (data not shown), another actin binding protein (72). Although <br /> nucleus in association with various complexes associated with         a possible role for profilin 1 as a coactivator remains to be <br /> the transcription machinery (1). Both actin and the actin-like        tested, we speculate that Fli-I, supervillin, and profilin 1 may <br /> protein BAF53 were identified as integral and essential com-          represent a class of actin binding coactivators with similar <br /> ponents of SWI/SNF coactivator complexes, which occur in              mechanisms of enhancing transcriptional activation by associ- <br /> multiple forms and contribute to the chromatin-remodeling             ation with actin and actin-like proteins. The dual roles of Fli-I <br /> process via an ATP-dependent nucleosome-remodeling activ-             as an actin cytoskeleton-associated protein and transcriptional <br /> ity (53, 70, 72). Another component of the SWI/SNF com-               coactivator are not unprecedented. Another developmentally <br /> plexes, BAF57, can (like Fli-I) bind to both ER and the N-            important protein, &#226;&#65533;&#164;-catenin, participates in regulation of the <br /> terminal region of p160 coactivators (4); the interaction             cytoskeleton by association with cell adhesion molecules (47) <br /> between BAF57 and the ER-assembled transcription initiation           and also functions as a coactivator in the nucleus (20, 28, 61), <br /> complex suggests a possible mechanism for coordinating the            where it can bind to and function synergistically with CARM1 <br /> actions of the SWI/SNF and p160 coactivator complexes, which          as coactivators for NRs and other classes of DNA-binding <br /> remodel chromatin by different but complementary mecha-               transcriptional activator proteins (33). As is true for Fli-I, the <br />  <br /> localization of &#226;&#65533;&#164;-catenin between cytoplasm and nucleus is                       <br />                                                                                      <br /> regulated by specific signal transduction pathways. Thus, Fli-I                    <br /> may be part of another signal transduction pathway that links                    <br /> cytoskeleton structure with transcriptional regulation. Since                          <br />                                                                                     <br /> CARM1 and Fli-I have a functional synergy as transcriptional                          <br /> coactivators, it is also tempting to speculate that the CARM1-                       <br /> Fli-I interaction could influence the cytoplasmic function of                           <br />                                                                                      <br /> Fli-I as well as its nuclear activity.                                                 <br />    <br /> </body></html>