<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 1990, p. 887-897                                                                      Vol. 10, No. 3 <br /> 0270-7306/90/030887-11$02.00/0 <br /> Copyright C) 1990, American Society Microbiology <br />  <br />  <br />         A Yeast ARS-Binding Protein Activates Transcription <br />  Synergistically Combination Other Weak Activating Factors <br />                               ANDREW R. BUCHMAN*t AND ROGER D. KORNBERG <br />  Department Cell Biology, Beckman Laboratories, Stanford University School Medicine, Stanford, California 94305 <br />                                           Received 8 August 1989/Accepted 14 November 1989 <br />  <br />              ABFI (ARS-binding protein I) yeast protein binds specific DNA sequences associated several <br />           autonomously replicating sequences (ARSs). ABFI binds sequences located promoter regions some <br />           yeast genes, including DEDI, essential gene unknown function transcribed constitutively high <br />           level. ABFI purified specific binding DED) upstream activating sequence (UAS) found <br />           recognize related sequences promoters, ARS (ARSJ), transcriptional silencer <br />           (HMR E). All ABFI-binding sites, regardless origin, provided weak UAS function vivo examined in <br />           test plasmids. UAS function abolished point mutations reduced ABFI binding vitro. Analysis of <br />           DEDI promoter showed ABFI-binding sites combine synergistically adjacent T-rich <br />           sequence form strong constitutive activator. The DEDI T-rich element acted synergistically other <br />           ABFI-binding sites binding sites multifunctional yeast activators. An examination the <br />           properties sequences surrounding ARSI left open possibility ABFI enhances initiation DNA <br />           replication ARS) transcriptional activation. <br />  <br />    Promoters higher eucaryotes complex structures                  diverse genetic elements. Four members class have <br /> made separable components (for reviews,                     identified far: centromere-binding protein 1 (CP1), <br /> references 28 37). Deletion studies revealed                  factor Y, general regulatory factor I (GRFI), ARS- <br /> groups regulatory sequences needed activate transcrip-               binding factor I (ABFI). CP1 binds sequences conserved <br /> tion-enhancers, located kilobase pairs                 domain yeast centromeres, CDE-I, intergenic <br /> promoter, proximal elements positioned just upstream                   regions (5). Factor Y-binding sites UASs <br /> of TATA sequence. Typically, enhancers proximal                    contained telomere-associated DNA <br /> promoter elements contain binding sites                   elements X Y&apos; (18; A. R. Buchman N. F. Lue, <br /> different transcription factors. Binding sites factors           unpublished data). Binding sites GRFI, known as <br /> greatly stimulate transcription various               RAP1 (53) TUF (24, 25), serve UASs wide variety <br /> combinations little effect (14, 19, 22, 40,          genes involved transcription, translation, RNA proc- <br /> 49, 50). In contrast, studies promoters Saccharomyces                essing, glycolysis, cell growth, functions (8, 9). <br /> cerevisiae suggested streamlined organization                  Interestingly, GRFI-binding sites important of <br /> (for review, reference 61). Upstream activating se-                  transcriptional silencers inactive mating type loci, <br /> quences (UASs) S. cerevisiae usually clustered               HML HMR (6, 8, 9, 30, 53). Also, C1_3A repeats at <br /> several <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> pairs start transcription.                  yeast telomeres contain binding sites GRFI (8, 9). <br /> Furthermore, single binding site regulatory protein,               ABFI binds sequences associated ARSI ARS2, as <br /> such GAL4 (20, 34) GCN4 (23), suffices                      autonomously replicating sequences (ARSs) the <br /> activate transcription. Some investigators (14, 59)                   regulatory elements silent mating-type loci-HMR E, <br /> proposed rudimentary mechanism explain                     HMR I, HML I (8). However, fully functional <br /> the constitutive transcription yeast genes contain                 ARSs lack binding sites ABFI, ABFI-binding <br /> thymidine-rich UASs. This notion based binding                sites far removed ARS. The suggestive <br /> sequence-specific transcription factors                  situation discovery ABFI-binding sites the <br /> inability poly(dT)-poly(dA) form nucleosomes vitro                promoter region DED1 gene, essential gene of <br /> (31, 42). It suggested T-rich UASs simply prevent                 unknown function transcribed constitutively high <br /> the assembly nucleosomes promoter, making                    level. Previous investigators determined transcription of <br /> region accessible transcription apparatus.                     DEDI depends 39-base-pair (bp) T-rich sequence lo- <br />   <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> gain deeper understanding molecular mecha-                   cated ABFI-binding sites TATA ele- <br /> nisms governing gene expression chromatin structure                 ment, keeping nucleosome exclusion idea (59). <br /> S. cerevisiae, taken biochemical approach                 However, preliminary experiments the <br /> identifying transcription factors assaying functions             ABFI-binding region T-rich sequence play impor- <br /> in vitro. Our investigations uncovered class                     tant activating transcription (35). These observations <br /> multifunctional DNA-binding proteins distinguished                motivated careful study properties ABFI and <br /> by abundance-more 103 molecules                     possible function activator DEDI other <br /> haploid cell-and ability bind specific sequences               genes. Our results lead conclusion ABFI func- <br />                                                                            tions relatively weak activator compared with <br />   * <br />     Corresponding author.                                                  well-characterized yeast transcription factors. A high <br />   t Present address: Department Molecular Cell Biology, 310         level constitutive transcription achieved DEDI <br /> South Frear Laboratory, The Pennsylvania State University, Uni-            synergistic actions multiple weak activating ele- <br /> versity Park, PA 16802.                                                    ments, contain binding sites ABFI factors that <br />                                                                      887 <br />  888    BUCHMAN AND KORNBERG                                                                                  MOL. CELL. BIOL. <br />  <br /> bind T-rich region. Comparison properties          BamHI fragment pCZ-TEL, generating pCZ-EI. pCZ- <br /> ABFI GRFI factor Y suggests weak          SPT2, -SPT2mtA, -SPT2mtB, -mtIA, -mtIB, -TCM1, -PAB, <br /> multifunctional activators various combinations fre-          -SYM, -ARS1, -TGS2 constructed ligating each <br /> quently play role yeast gene expression.                    oligonucleotide 1-kbp HindIII-ClaI 7.9-kbp ClaI- <br />                                                                  BamHI fragments pCZ-EI. For construction pCZ- <br />                MATERIALS AND METHODS                             DEDSPT, oligonucleotide ligated 9-kbp <br />                                                                  XhoI-XbaI fragment pCZA. pCZ-DED48 de- <br />    Computer-assisted searches DNA sequences. Yeast DNA        scribed (35) contains oligonucleotide inserted <br /> sequences GenBank EMBL data bases                EcoRI XhoI sites polylinker pCZA. <br /> searched matches ABFI consensus sequence           Plasmids containing DED48 sequence another <br /> using programs obtained University          synthetic oligonucleotide (pCZ-oligonucleotide/DED48 plas- <br /> Wisconsin Computer Group (15). Matches PAB,               mids) constructed ligation 7.9-kbp EcoRI-ClaI <br /> SP12, TCMJ, tRNAser genes span nucleotides 460            fragment corresponding pCZ derivative, having the <br /> 477 (46), 214 228 (44), -232 -218 (51), 244 258     oligonucleotide alone, 1-kbp EcoRI-ClaI fragment of <br /> (1), respectively, published sequences.                   pCZ-DED48. pCZ-(DED48)2 generated creating a <br />    Synthetic oligonucleotides oligonucleotide-Sepharose      tandem duplication DED48 sequence. pCZ-DED48 <br /> resin. Oligonucleotides produced Applied            DNA digested separately XhoI EcoRI. The <br /> Biosystems DNA synthesizer purified an-             linearized DNAs treated Klenow fragment of <br /> nealed described previously (4). The oligonucleotides         DNA polymerase deoxynucleoside triphosphates and <br /> contain sequences derived yeast DNA,             digested ClaI endonuclease. The 1-kbp EcoRI (blunt)- <br /> sequences end allow ligation restriction   ClaI fragment 7.9-kbp ClaI-XhoI (blunt) fragment were <br /> fragments. (For sequences of oligonucleotides,       ligated form pCZ-(DED48)2. pCZ-dAT2 de- <br /> see Table 2.) The sequence TEL oligonucleotide        scribed previously (same pdAT-2; [35]) contains a <br /> be elsewhere, preliminary descriptions       116-bp AccI-XhoI fragment DEDI DNA (nucleotides 756 <br /> the binding properties El, mtIA, mtIB, SYM,           880 numbering scheme Struhl [60]) inserted <br /> DED1 oligonucleotides (8). One strand DOMA oligo-         SalI XhoI sites polylinker pCZA. <br /> nucleotide, contains ARS consensus match          For insertion ARSI domain B fragments pCZ <br /> ARS1, sequence 5&apos;-TCGAGTTGTTATTTTACAGA                   plasmids, ARSI DNA modified could <br /> TF&apos;TTTATG1TTAGATCTTTTATGCTITGC1TIT&apos;CAAAAGG                       placed directly upstream CYCI promoter. YRP14/ <br /> CCTGCA-3&apos;. The complementary strand annealed                CEN4/ARSJA108 A149 DNAs (54) digested <br /> bases 5 56. For DNA affinity chromatography, double-          BglII, treated Klenow fragment DNA <br /> stranded DED1 oligonucleotide coupled Sepharose           polymerase, ligated XhoI octanucleotide linkers (5&apos;- <br /> CL-2B modification previously described proce-         CCTCGAGG-3&apos;), digested XhoI BamHI. The <br /> dure (29). The oligonucleotide (500 ,ug) allowed react    respective 140- 99-bp XhoI-BamHI fragments were <br /> with 4 ml CNBr-activated Sepharose beads. The coupling        isolated gel electrophoresis, ligated 1-kbp <br /> efficiency, measured spectrophotometrically 260 nm,       XhoI-ClaI 7.9-kbp ClaI-BamHI fragments pCZ-TEL <br /> greater 60%.                                                produce pCZ-A108 pCZ-A149. A 117-bp PstI-XbaI <br />    Plasmid probe DNAs. The pCZ family plasmids (9         fragment pCZ-A108, containing ARSI domain <br /> kbp) shuttle vectors contain pUC18 replication        B, ligated 9-kbp XhoI-XbaI fragment pCZA <br /> origin P-lactamase gene propagation Escherichia       DOMA oligonucleotide produce pCZ- <br /> coli, URA3, SUP]], ARSI, CEN4 sequences           A108/DOMA. Probe DNAs restriction fragments de- <br /> for maintenance S. cerevisiae (plasmid maps      rived pCZ plasmids labeled [a-32P]dATP <br /> in references 9 35). To test UAS functions inserted   (Amersham Corp.) DNA polymerase Klenow frag- <br /> DNAs, pCZ plasmids contained yeast CYCI pro-             ment replacement synthesis (36). The specific radioactiv- <br /> moter (nucleotides -138 +5 published numbering         ity probes varied 5 x 103 2 x 104 cpm/fmol. <br /> scheme [38]) fused lacZ protein-coding sequences. The            Protein-DNA binding assays. Generally, nitrocellulose fil- <br /> CYCI UAS replaced polylinker sequence, having            ter-binding assays gel electrophoresis mobility shift <br /> the following configuration restriction sites, plasmid     assays performed described previously (8). Binding <br /> pCZ-TEL (9): HindIII-SphI-PstI-SalI-XbaI-BamHI-TEL               reactions carried 20 ,ul buffer A (25 mM <br /> oligonucleotide-EcoRI-SacI-KpnI-SmaI-SalI-XhoI-CYCl-             HEPES [N-2-hydroxyethylpiperazine-N&apos;-2-ethanesulfonic <br /> lacZ. Plasmid pCZA (35) contains inserted oligonucleo-        acid] [pH 7.5], 5 mM MgCl2, 0.1 mM EDTA, 0.5 mM <br /> tide lacks BamHI site listed above. Synthetic oligo-     dithiothreitol, 10% glycerol, 1 mM phenylmethylsulfonyl <br /> nucleotides inserted pCZ DNA different ways         fluoride, 2 puM pepstatin A, 0.6 p.M leupeptin, 5 ,ug of <br /> depending kinds ends possessed. pCZ-DED1,         antipain ml) containing 50 mM KCI, 0.01% Nonidet P-40 <br /> -HML35, -RAP1, -GAL2 produced ligating           (NP-40), 1 5 fmol 32P-labeled probe DNA, 2 ,ug of <br /> indicated oligonucleotide 1-kbp EcoRI-ClaI 7.9-kbp      bovine serum albumin, 1 ,ug poly(dI-dC), various <br /> ClaI-BamHI fragments pCZ-TEL. For construction         amounts yeast protein competitor DNA. After incu- <br /> pCZ-EI, HindIII site introduced EcoRI    bation room temperature, reaction mixtures either <br /> site polylinker. pCZ-TEL digested EcoRI,         filtered nitrocellulose electrophoresed 3% <br /> and ends blunt DNA polymerase             NuSieve agarose (FMC Corp.) gels. The relative binding <br /> Klenow fragment (Pharmacia LKB Biotechnology, Inc.).             affinities different sequences determined gel <br /> Synthetic HindIII linkers (5&apos;-CAAGCTTG-3&apos;; New England           electrophoresis mobility shift assays performed <br /> BioLabs, Inc.) ligated linearized DNA,         0.7 0.35 ng purified ABFI labeled probe. <br /> was subsequently digested HindlIl ClaI. A 1-kbp         The probe reaction mixture free <br /> HindIII-ClaI fragment purified gel electrophoresis        bound ABFI measured quantitating the <br /> and joined oligonucleotide El 7.9-kbp ClaI-           radioactivity bands dried gel AMBIS <br />  VOL. 10, 1990                                    TRANSCRIPTION ACTIVATION BY YEAST ARS-BINDING PROTEIN                      889 <br />  <br /> Radioanalytic Imaging System (Ambis Systems Inc., San                                      RESULTS <br /> Diego, Calif.). Equations used calculate relative binding <br /> affinities data discussed          Homology search ABFI-binding sites. To gain insight <br /> (33). For purpose estimates assumed      possible function ABFI gene activator, <br /> ABFI pure, 100% active, bound monomer (135           looked additional binding sites yeast DNA us- <br /> kilodaltons [kDa]). The specific radioactivities probes        ing homology search program. Data bases of <br /> were determined scintillation counting measuring           yeast sequences (GenBank EMBL) searched for <br /> DNA concentrations Hoechst 33258 dye fluorom-          matches ABFI consensus sequence, 5&apos;-RDCNYN <br /> eter (TKO 100; Hoefer Scientific Instruments).                    NNNNACGAD-3&apos; (R = A G; Y = C T; D = A, G, or <br />    Purification ABFI. Cells yeast strain BJ926 (ala     T; N = A, G, C, T), derived regions known bind <br /> trpll+ prcl-1261prcl-126 pep4-31pep4-3 prbl-I1221prblI 122        factor high affinity (8). A large number matches <br /> canllcanl) grown YPD medium (2% Bacto-Peptone             found, intergenic regions. Four matches <br /> [Difco Laboratories], 1% yeast extract, 2% glucose)     chosen study based location up- <br /> culture reached optical density 600 nm 6.0           stream genes interest. One match upstream <br /> harvested centrifugation. The pellet washed          region gene, PAB, encoding poly(A)-binding protein <br /> equal volume buffer A resuspended              (46). Another located SPT2 gene (44), which <br /> volume buffer A, subsequently adjusted 0.4        encodes protein regulates transcription trans- <br /> M KCl. Cells broken 4&#194;&#176;C glass bead disruption,         poson TY]. A match, located upstream ribo- <br /> using 10 30-s cycles bead beater (Biospec Products). The     somal protein gene TCMI (51), coincides sequence <br /> cell lysate centrifuged Sorval SS-34 rotor 12,000     previously shown function UAS (21). The fourth <br /> x g 10 min; supernatant decanted centri-          match adjacent sequence encoding 5&apos; end a <br /> fuged Beckman Ti6O rotor 250,000 x g 3 h 4&#194;&#176;C.      tRNA9er (1). To compare properties new se- <br />                                                                   quences characterized previously, <br /> The clarified extract adjusted 100 mM KCl dilution      synthetic oligonucleotides produced contained <br /> with buffer A (total volume, 560 ml) loaded       matches (see Table 2). Initially, exam- <br /> column DEAE-cellulose (5 12 cm; Whatman DE52)            ined ability new oligonucleotides bind ABFI by <br /> a rate 250 ml/h. The column washed 500 ml          testing competitors nitrocellulose filter-binding <br /> buffer A 0.05 M KCl, protein eluted       assays (8), carried whole-cell extract <br /> 1,150-ml gradient buffer A 0.05 0.5 M KCl.             32P-labeled probe containing ABFI-binding site at <br /> Fractions containing ABFI (0.22 M KCI peak) com-             HMR E (El oligonucleotide). The SPT2, PAB, TCM1, and <br /> bined, diluted 0.05 M KCI, loaded phospho-          TGS2 (tRNAs&#194;&#182;) oligonucleotides blocked specific protein <br /> cellulose column (2.5 18 cm; Whatman P11) 100 ml/h.         binding probe concentrations indicated that <br /> The column washed 180 ml buffer A 0.05 M         bound ABFI high affinity (data shown). <br /> KCI, protein eluted 450-ml linear gradient        Purification ABFI. Considering variety DNA <br /> of buffer A 0.05 0.5 M KCl. Peak fractions containing     elements bound crude ABFI vitro-ARSs, <br /> ABFI (0.24 M KCI) combined diluted 0.1 M              silencers, UASs-it possible factor <br /> KCl, NP-40 added 0.01%. The protein mixed          actually composed multiple proteins related <br /> with 1.7 mg salmon sperm DNA applied 2-ml             binding specificities. We sought clarify point through <br /> column DED1 oligonucleotide-Sepharose 13 ml/h. The          purification ABFI. Fractionation monitored by <br /> column washed sequentially 5 ml buffer A         using nitrocellulose filter-binding assays 32P-labeled <br /> 0.05 M KCI 0.01% NP-40, 7 ml buffer A 0.3 M           probes having different ABFI-binding sites, joined <br /> KCl 0.01% NP-40, 10 ml buffer A 1.0 M KCl         184 bp CYCI DNA. Probes containing ARS1, EI, <br /> and 0.01% NP-40. Peak fractions ABFI 1.0 M            DED1, TCM1 oligonucleotides (see Table 2) se- <br /> KCI step combined, adjusted 0.1 M KCl,                quences derived ARS, silencer (HMR E), two <br /> reapplied oligonucleotide-Sepharose column,          UASs, respectively. A yeast whole-cell extract fraction- <br /> was washed procedure. Fractions containing            ated chromatography DEAE-cellulose, phosphocellu- <br /> ABFI stored freezing liquid nitrogen. Protein          lose, DED1 oligonucleotide-Sepharose (see Materials <br /> concentrations determined method Bradford          Methods). A single peak binding activity found <br /> (3), using bovine serum albumin standard.                  chromatography resin, peak profile was <br />    Assays UAS function. Plasmid DNAs introduced           DNA probes. Overall, the <br /> into yeast strain 5C (a his3 A200 ura3-52) using         binding activity purified 5,160-fold 15% yield <br /> lithium acetate transformation procedure (26), selecting      (Table 1). The purified ABFI oligonucleotide- <br /> uracil prototrophy minimal medium (0.67% yeast nitrogen        Sepharose column analyzed sodium dodecyl sulfate- <br /> base, 0.5% ammonium sulfate, 2% glucose) supplemented             polyacrylamide gel electrophoresis, proteins were <br /> with histidine. Transformants picked grown            detected silver stain (Fig. 1A). Two major polypep- <br /> liquid minimal medium supplemented histidine       tides resolved masses 135 145 kDa. <br /> culture reached optical density 600 nm 1.0. Cells           The integrity purity ABFI judged using gel <br /> were harvested centrifugation, washed, suspended        electrophoresis mobility shift assays 32P-labeled probes <br /> an buffer (200 mM Tris hydrochloride [pH 7.5],          containing various oligonucleotides joined 184 bp CYCI <br /> 200 mM NaCl, 10 mM magnesium acetate, 5% glycerol, 10             DNA (Fig. 1B). The single specific complex formed with <br /> mM P-mercaptoethanol, 1 mM phenylmethylsulfonyl fluo-             purified ABFI EI (HMR E) probe same <br /> ride) equal volume cell pellet. Cells broken   mobility complex produced whole-cell extract. <br /> by glass bead disruption, extract clarified        Probes containing sequences recognized abundant <br /> centrifugation 10 min 14,000 x g microcentrifuge.     yeast DNA-binding factors tested. The probe with <br /> P-Galactosidase enzyme assays performed o- <br /> nitrophenyl-3-D-galactoside described previously (39). <br />                                                                   binding sites GRFI sequence (TEL oligonucleotide) <br />                                                                   derived C1_3A repeat yeast telomeres (8). Probes <br />  890     BUCHMAN AND KORNBERG                                                                                                                                         MOL. CELL. BIOL. <br />  <br />                                                                 TABLE 1. Purification ABFI <br />                                                                                                                                                                           Purification <br />           Fraction                        Protein (mg)                Vol (ml)                    Activity (U)                       Sp act (U/mg) <br />                                                                                                                                                                Yield (%)                 Fold <br /> Whole-cell extract                        1,980                        560                            1,160,000                             585                     100                  1 <br /> DEAE-cellulose                              205                        132                              654,000                           3,200                      56                  5.5 <br /> Phosphocellulose                             13                         80                              254,000                          19,500                      22                 33.4 <br /> Oligonucleotide-Sepharose                     0.058                      2.9                            175,200                       3,020,000                      15              5,160 <br />  One <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> DNA-binding activity required retain 1 fmol labeled probe nitrocellulose filter. <br />  <br />  <br /> capable binding CP1 factor Y contained sequences                                       ABFI. Thus, purified ABFI appeared largely intact <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> upstream GAL2 gene (5) RAPJ gene (53),                                    free DNA-binding activities. <br /> respectively. (Note RAP] gene encodes DNA-                                          Binding properties ABFI vitro. To correlate the <br /> binding protein GRFI, factor Y GRFI                                    binding purified ABFI vitro function vivo, we <br /> binds upstream region gene [A. R. Buchman,                                        characterized binding properties panel synthetic <br /> unpublished data].) Assays showed whole-cell ex-                                     oligonucleotides containing matches ABFI consensus <br /> tract contained comparable amounts ABFI, GRFI, factor                                      sequence. In addition matches discussed above, an <br /> Y, CP1, producing complexes varied mobility                                   oligonucleotide tested contained synthetic sym- <br /> each factor (Fig. 1B); however, GRFI, CP1, factor Y                                       metrized version ABFI consensus sequence, SYM (8), <br /> were purified ABFI. A previous study (35)                                    derived hypothetical half site. Each oligonucleotide, <br /> suggested ABFI-binding region DED1 gene                                       joined 184 bp CYCI DNA, used probe gel <br /> functions synergistically adjacent T-rich sequence                                 electrophoresis mobility shift assays. A single well-defined <br /> activate transcription vivo. Factors bind T-rich                                 complex formed probe, relative <br /> element stimulated transcription 30-fold                                      affinities whole-cell extract or <br /> yeast vitro (35). When probe containing                                       purified ABFI: SPT2 &gt; El &gt; TCM1 &gt; DED1, PAB &gt; SYM <br /> DEDI T-rich sequence (DED48) mixed whole-cell                                        &gt; ARS1 &gt; TGS2 (Fig. 2B C). Quantitative measure- <br /> extract, specific complexes elec-                                   ments relative binding affinities sequence (see <br /> trophoretic mobility shift assay (Fig. 1B C). The T-                                      Materials Methods) revealed sixfold difference be- <br /> rich-sequence-binding factors detected assay ap-                                      tween highest lowest affinities (Table 2). <br /> peared abundant bound lower affinity                                  We examined binding properties oligonucle- <br /> than ABFI, GRFI, factor Y, CP1. No T-rich-sequence-                                        otides contained single base changes ABFI <br /> binding factors detected highly fractionated                                      consensus region (Table 2). Oligonucleotides SPT2mtA and <br />  <br />                                                          ABF!          GRF!        Factor Y                OPI.        T-rich             BEnoi        T- rich <br />                                                                                                                                           Facto&apos;r <br />                                                            El          TEL             RAP1            GAL2            DED48              Probe           DED48 <br />                                                          I      -_i      r         -              1                                                            -I1 <br />  <br />                                                                 m <br />                                                                 c:            m <br />                                                                               M. <br />                                                                               c:             m <br />                                                                                              &lt;:                   m <br />                                                                                                                   &lt;r <br />                                                                                                                        ;,       c&lt;        Prote in <br />                                                                 &lt;             &lt; <br />  <br />  <br />  <br />                        200 - <br />  <br />                                       =   ABFI <br />                                                                  -I <br />                        116   - <br />                                                                                                                                        Complexes <br />                         97 -                                                                                                                  I      .. <br />                         66 - <br />                                                       d_              uIqI                            i-                    f        }-    Probes <br />  <br />                         43 - <br />                                  A                                                       B                                                                    C <br />   FIG. 1. Purification ABFI. (A) Purified ABFI (0.2 ,ug) 1.0 M KCl step DED1 oligonucleotide-Sepharose column <br /> procedure analyzed sodium dodecyl sulfate-7.5% polyacrylamide gel electrophoresis, proteins reveal silver stain (643). <br /> The positions sizes kilodaltons protein standards shown left: myosin (200 kDa), 3-galactosidase (116 kDa), phosphorylase <br /> b (97 kDa), bovine serum albumin (66 kDa), ovalbumin (43 kDa). (B) Gel electrophoresis mobility shift assays performed with <br /> 32P-labeled BamHI fragments (0.22 kbp) pCZ-TEL, -El, -RAP1, -GAL2 EcoRI-BamHI fragment (0.20 kbp) from <br /> pCZ-DED48. Reaction mixtures contained 2.7 ,ug whole-cell extract (WCE), 1 ng purified ABFI (El reaction), 10 ng ABFl <br /> (TEL, RAP1, GAL2, DED48 reactions). The dried gel analyzed autoradiography exposed 1 h. (C) The right lanes <br /> shown panel B exposed 17 h. <br />  VOL. 10, 1990                                                                                     TRANSCRIPTION ACTIVATION BY YEAST ARS-BINDING PROTEIN                      891 <br />  <br />         A                     ABFI                        WKE                                                   consensus sequence increased insertion A <br />                                                                                                                 residue center. The alteration mtIB reduced specific <br />                                                                                   Protein <br />  <br />                                                               m <br />                                                                                                                 binding 36-fold crude purified ABFI (Fig.1A). <br />                                    .,:                     &lt; <br />  <br />                     cu-             E                                             Probe <br />                                                                                                                 No binding detected, using whole-cell extract or <br />                                                                                                                 purified ABFI, probes containing mtIA, SPT2mtA, <br />                                                                                                                 SPT2mtB oligonucleotide (Fig. 1). Thus, these <br />                                                                                                                 base changes reduced binding affinity 500-fold more. <br />                                                                                                                    UAS function ABFI-binding sites. Our studies the <br />                                                                          Complex                           locations binding sites ABFI raised possibility that <br />                                                                                                                 ABFI act transcription factor situations <br />                                                                                                                 (8). Binding sites promoter regions the <br />                                                                                                                 DEDI    gene 1,950-base RNA 2p.m plasmid. <br />                                                                                                                 Recent results lend support notion ABFI <br />                                                                               Probe <br />                                                                                                                 acts transcriptional activator. In experiments described <br />                                                                           - <br />  <br />  <br />  <br />  <br />                                                                                                                 above, demonstrated binding site ABFI the <br />                                                                                                                 TCMI gene coincides 19-bp region shown others <br />                                                                                                                 (21) UAS. Also, analysis regulatory region of <br />         B                                              ABFI                                                     DEDI     promoter revealed composed two <br />                                                                                                                 segments activate transcription synergistically (35). One <br />                                                                                    &lt;      m <br />                                                                                                                 segment, just upstream TATA element, contains a <br />                 N <br />  <br />                 U)a- LUI <br />                           X <br />                               _ <br />  <br />                               C. <br />                                          a <br />                                          LU <br />                                                   mm              V)u <br />                                                                          N <br />  <br />                                                                         c)) <br />                                                                           0 <br />                                                                                                       Probe     39-bp region rich T residues coding strand. Imme- <br />                 (                        0 <br />                                               | <br />                                                   CL <br />                                                               i&lt;                   W I. n <br />                                                                                                                 diately adjacent T-rich segment 68-bp region <br />                                                                                                                 containing binding sites ABFI (see below). Either <br />                                                                                                                 region weak activator transcription, but <br />                                                                                                                 combined produced strong activating effect. <br />                                                                                                   - <br />                                                                                                       Complex   However, remained determined UAS <br />                             - - -                        - - - <br />                                                                                                                 functions sequences containing ABFI-binding sites <br />                                                                                                                 fact action ABFI factors that <br />                                                                                                                 bind regions. We reasoned ABFI can <br />                                                                                                                 function activator, oligonucleotides bind <br />                               _m                  si <br />                                                    m n - Probe                                                  ABFI vitro provide weak UAS function in <br />                                                                                                                 vivo examined test plasmids. Furthermore, all <br />                                                                                                                 ABFI-binding oligonucleotides produce strong ac- <br />                                                                                                                 tivating effect combined T-rich element the <br />         c <br />                                                                                                                 DEDI   gene. <br />                                                   VCE                                                             These predictions tested use yeast centro- <br />                                                                                   E       E <br />                                                                                                                 meric plasmids carrying CYCJ-lacZ (promoter-coding re- <br />             CM <br />                             &gt;,- n <br />                           - C,,               s u:cn tN <br />                                               m-            N <br />                                                                               E <br />                                                                                                       Probe     gion fusion) reporter gene URA3 selectable marker. <br />             a- <br />             I <br />                                 LL               CC&lt; n CL :                                                     The normal UAS       CYCI   promoter replaced in <br />                                                                                                                 plasmids polylinker region, receive <br />                                                                                                                 sequences tested UAS function. In set of <br />                                                                                                                 constructions,    ABFI    oligonucleotide inserted <br />                                                                                                                 position 56 bp upstream CYCJ TATA <br />                                                               __                              -   Complex <br />                            _4EI4                                                                                element (pCZ-oligonucleotide plasmids). In set of <br />                                                                                                                 constructions, 48-bp sequence containing DEDI T-rich <br />                                                                                                                 activator (DED48 oligonucleotide) inserted between <br />                                                                                                                 ABFI oligonucleotide    CYCI       TATA element <br />                                                                                                                 (pCZ-oligonucleotide/DED48 derivatives). Plasmids in- <br />                                                                                                       Probe     troduced yeast strain     5C his3-A200 ura3-52), <br />                                                                                                                                                        (a <br />                                                                                                                 selecting uracil prototrophy, UAS function was <br />   FIG. 2. Gel electrophoresis mobility shift assays carried <br />                                                                                                                 assayed measuring   ,-galactosidase    activity extracts of <br />                                                                                                                 transformants (Table 2). All constructs containing oligonu- <br /> out 32P-labeled BamHI fragments (0.22 kbp) pCZ-SPT2, <br /> -SPT2mtA, -SPT2mtB, -TCM1, -DED1, -PAB, -SYM, -ARS1, <br />                                                                                                                 cleotides bound  ABFI   vitro showed significant UAS <br />                                                                                                                 function vivo. Plasmids containing just ABFI oligonu- <br /> -TGS2, -El, -mtIA, -mtIB. Reaction mixtures contained                                                cleotides exhibited levels expression 3- 26-fold <br /> 2.7 ,ug whole-cell extract (WCE) 0.7 ng purified ABFI. <br /> Reaction mixtures whole-cell extract contained 20 ng RAP1                                               grater control plasmid insert (pCZA, 0.9 U). <br /> oligonucleotide order block formation complexes                                                 The constructions having ABFI-binding site the <br /> probes factor Y, binds low affinity sequence                                            T-rich element (DED48) exhibited higher levels expres- <br /> the SmaI-KpnI region polylinker (9).                                                                     sion, varying 6- 20-fold greater plasmid with <br />                                                                                                                 T-rich element (pCZ-DED48, 9 U). The combina- <br /> SPTF2mtB derivatives SPT2 contain alterations in <br />                                                                                                                 tion   DED1 DED48 oligonucleotides (pCZ-DED1/ <br />                                                                                                                 DED48, 180 U) appeared fully reconstitute activating <br /> positions 12 13, respectively, consensus se-                                                         effect normal upstream regulatory region DED) <br /> quence. mtIA mtIB mutant forms oligonucle-                                                       gene, 116-bp AccI-XhoI fragment (pCZ-dAT2, 170 U). In <br /> otide El. In mtIB, position 3 consensus sequence                                                     cases, ABFI T-rich elements acted a <br /> altered. In mtIA, spacing halves                                                          synergistic fashion (3.9 9.6 times sum individual <br />  892      BUCHMAN AND KORNBERG                                                                                                                MOL. CELL. BIOL. <br />  <br />                                       TABLE 2. Binding properties UAS function different oligonucleotides <br />                                                     UAS function <br /> Oligonucleotide           Relative <br />    inserta             affinityb           CZC <br />                                              PCZ&apos;            pCZ plus                                              Sequencee <br />                                                              DED48d <br />  Consensus                                                                                                                      RDCNYNNNNNACGAD <br />  SPT2                       1.0               4.5             130                                              GATCCAAATAATCATGTTAAACGAAAAAGTA <br />  SPT2mtA                  &lt;0.002              0.30                 3.0                                         GATCCAAATAATCATGTTAAAgGAAAAAGTA <br />  SPT2mtB                  &lt;0.002              0.30                 4.6                             GATCCAAATAATCATGTTAAACcAAAAAGTA <br />  DEDSPT                     1.0              50                                 CTAGACGTGTCATTCTGAACGAGGCGATAATCATGTTAAACGAAAAAC <br />  El                        0.73               7.0              63                                  GATCCCATACATCATAAAATACGAACGA <br />  mtIA                     &lt;0.002              0.33              3.4                                GATCCAATACATCATAAaAATACGAACGA <br />  mtIB                       0.02              0.54              6.0                                GATCCAATACATaATAAAATACGAACGA <br />  TCM1                       0.64              7.1              74                                  GATCCTGAAAAACGTACAAAACGATTACTAA <br />  DED1                       0.45             24               180                    GATCCAATGCATCATTCTATACGTGTCATTCTGAACGAGGCG <br />  PAB                        0.44              6.4              74                                  GATCCGATCAATCATGTTTAACGAAAACTGA <br />  SYM                        0.30              5.0              84                                  GATCCAATATATCGTTATTAACGATATATTA <br />  ARS1                       0.26              8.0              89                                  GATCCTTCTTAGCATTTTTGACGAAATTTGA <br />  TGS2                       0.17              2.8              57                                  GATCCATGATATCATGTAAAACGAGGCAACA <br />   A                                           0.90 <br />  dAT2                                       170 <br />  DED48                                        9.0                               AATTCTTTCCTTTTTTCTTTTTGCTTTTTCTTTTTTTTTCTCTTGAAC <br />  (DED48)2                                    70 <br />  HML35                                       11               110               GATCCAAAAATCAAAAACCCATTCATAAGATACTG <br />  RAP1                                         2.6              55               GATCCCAATGGCCGCTACCCGCCTCTTG <br />  GAL2                                         0.52              9.0             GATCCAAAATGGGTCACGTGATCTATATTG <br />   Oligonucleotides DNAs inserted polylinker region pCZ plasmids. <br />   b The relative binding affinity sequences purified ABFI measured gel electrophoresis mobility shift assays (see Materials Methods) is <br /> normalized SPT2. Estimates indicated apparent association constant ABFI SPT2 sequence 5 x 1010 M conditions (data <br /> not shown). <br />   &apos; UAS function oligonucleotide assessed based expression CYCI-IacZ fusions pCZ derivatives, determined measuring the <br /> specific activity ,B-galactosidase (in nanomoles minute milligram protein) extracts yeast cells containing plasmid. <br />   d UAS function oligonucleotide DED48 T-rich sequence determined based p-galactosidase activity extracts yeast cells <br /> containing pCZ-oligonucleotide/DED48 plasmid. <br />   e The sequence strand oligonucleotide listed 5&apos; 3&apos; left right. The complementary strand anneals fifth base left to <br /> the base shown 4-base 5&apos; protruding end 5&apos;-AATT-3&apos; DED1, HML35, RAP1, GAL2; 5&apos;-TCGA-3&apos; DEDSPT DED48; and <br /> 5&apos;-AGCT-3&apos; others. The ABFI consensus region oligonucleotide boldface. Second occurrences ABFI consensus sequence underlined <br /> for DED1 DEDSPT. Sequences thought recognized DNA-binding proteins underlined DED48 (T rich), HML35 (GRFI), RAP1 (factor <br /> Y), GAL2 (CP1). <br />  <br />  <br />  <br /> elements). Activation ABFI oligonucleotides vivo                              ABFI added bind half probe the <br /> depended ability bind ABFI vitro. The                                reaction mixture, major complex formed DED1 <br /> synthetic symmetrized sequence SYM activat-                            probe (&gt;95%) mobility complexes formed <br /> ing effect oligonucleotides natural yeast DNA                         ABFI-binding sites (Fig. 2B), indicating the <br /> sequences. Importantly, mutant oligonucleotides                            probe associated single molecule ABFI. A <br /> with diminished binding vitro (mtIA, mtIB, SPT2mtA,                          different result observed ABFI used (7- <br /> SPT2mtB) failed increase expression vivo,                        14-fold more, Fig. 3A). A single complex produced <br /> or combination T-rich element. The control                              El probe (HMR E), complexes formed <br /> plasmids pCZA pCZ-DED48 expressed slightly                              DED1 probe. The major DED1 complex was <br /> higher levels constructions containing mutant                          reduced mobility expected binding of <br /> oligonucleotides. This explained                          molecules ABFI. This doubly bound DED1 complex <br /> weak activator sequence control plasmids                             appeared relatively unstable, evidenced the <br /> disrupted insertion oligonucleotides.                                     smear radioactivity singly bound <br />    Interactions compound UASs. Our original analysis                         complex, presumably caused dissociation lower- <br /> of DEDJ promoter suggested tandem                           mobility complex electrophoresis. Such instability <br /> binding sites ABFI (8). In DNase I protection assays,                           arise partial steric interference prox- <br /> crude ABFI nuclear extracts protected larger region                          imity binding sites DED1 sequence (the <br /> at DEDI (35 bases) ABFI-binding sites (20 25                       consensus sequences overlap 1 bp, Table 2). With in <br /> bases). The region DEDI protected assays                                mind examined properties oligonucleotide, <br /> contained adjacent sequences homology                             DEDSPT, containing ABFI-binding sites (DEDJ and <br /> ABFI-binding sites (marked sequence Table 2). The                         SPT2) separated greater distance (6 bp). The DEDSPT <br /> UAS assays described indicated occurrence                            oligonucleotide, however, behaved the DED1 <br /> of ABFI-binding sites functional significance                         sequence gel electrophoresis mobility shift assays. Large <br /> vivo. The plasmids DED1 oligonucleotide, pCZ-                              amounts ABFI required produce doubly <br /> DED1 pCZ-DED1/DED48, expressed higher                                 bound complex (Fig. 3A; data shown), again <br /> level constructs ABFI-binding oligonucleo-                          appeared unstable. When examined test plasmid in <br /> tides. The physical basis enhanced activation                          vivo, DEDSPT oligonucleotide, like DED1, produced <br /> explored gel electrophoresis mobility shift assays. When                         elevated UAS function (pCZ-DEDSPT, 50 U, Table 2). <br />  VOL. 10, 1990                                                                           TRANSCRIPTION ACTIVATION BY YEAST ARS-BINDING PROTEIN                          893 <br />  <br />             A                                                                                            mobility shift assays (Fig. 3B) probes containing just an <br />                                       DED1                                      Probe <br />                                                                                                          ABFI-binding site (El) ABFI-binding site the <br />                                                                                                          T-rich element (EI/DED48). With El probe, addition of <br />                                                     DEDSPT <br />  <br />  <br />  <br />  <br />                              +   ++   +    ++       +           ++              ABFI                     whole-cell extract produced major complex ABFI <br />                                                                                                          minor complexes (&lt;1%), presumably derived <br />                                                                                                          degraded ABFI (lane 2). Complex formation the <br />                                                                                                          El probe completely blocked excess EI oligonucle- <br />                                                                                                          otide (20 ng, lane 6) DED48 oligonucleotide <br />                                                                                                          (20 ng, lane 4). With EI/DED48 probe additional <br />                                       -0                                                                 complexes resolved having lower mobilities the <br />                                            . <br />                                                                                                          ABFI-derived complexes produced EI probe (lane <br />                                                                                - Complexes               3). The lower-mobility complexes EI/DED48 were <br />                                                                                                          abolished addition El (lane 7) DED48 <br />                                                                                                          (lane 5) oligonucleotide, indicating complexes <br />                                                                                                          contained ABFI- T-rich-sequence-binding factors. <br />                                                                                                          Notably, binding ABFI did appear increase the <br />                                                                                                          association factors T-rich sequence. There was <br />                                                                                                          increase complexes blocked DED48 <br />                      I                                                     -   Probe                     oligonucleotide ABFI bound complexes <br />                                                                                                          formed DED48 sequence absence ABFI <br />                                                                                                          (compare Fig. 3B, lanes 3 7; bands Fig. 3B, lane 7; <br />                                                                                                          represent complexes formed gel <br />             E3                        DED48             E =l                    Competitor               shown Fig. 1C DED48 probe). <br />                                                 r <br />                                                                        l                                    The results experiments suggest combi- <br />                                  0 <br />                                  0 <br />                                            0 <br />                                            0                   0                                         nation ABFI T-rich elements produces strong UAS <br />                                  wLw                           ul <br />                                                                0                Probe                    activity mechanisms involving highly coopera- <br />                                                                                                          tive binding factors elements. This raises the <br />                          +       +    +    +        +           +               WCE                      issue various weak activating elements can <br />                  1 <br />                  l <br />                          2       3    4    5        6            7                                       generally combine create strong activators. We pursued <br />                                                                                                          question comparing activating properties other <br />                                                                                                          members class abundant yeast DNA-binding fac- <br />                                                                                                          tors. UAS test plasmids constructed contained <br />                                                                                                          binding sites factor Y (pCZ-RAP1), CP1 (pCZ-GAL2), <br />                                                                                                          GRFI (pCZ-HML35) alone, these <br />                                                                                                          binding sites joined T-rich element DEDI <br />                                                                                                          gene (DED48 derivatives). We showed previously GRFI <br />                                                                                                          potent transcriptional activator vivo vitro (9). In <br />                                                                Jk. ,           Complexes                 present study, GRFI-binding site used HML <br />                                                                                                          E DNA (HML35 oligonucleotide) relatively low <br />                                                                                                          binding affinity correspondingly low UAS activity (36- <br />                                                                                                          fold lower activity strongest GRFI-binding sites). <br />                                                                                                          ,-Galactosidase assays performed ex- <br />                                                                                }Probes <br />                                                                                                          tracts yeast strains containing different plasmid con- <br />                                                                                                          structions (Table 2). The weak GRFI-binding site (pCZ- <br />                                                                                                          HML35, 11 U) gave level expression comparable the <br />   FIG. 3. (A) Gel electrophoresis mobility shift assays car-                                        active ABFI-binding sites, expression greatly <br /> ried 32P-labeled XbaI-BamHI fragment pCZ-                                                enhanced combined T-rich element <br /> DEDSPT (198 bp) BamHI fragments pCZ-DED1 (225 bp)                                               (pCZ-HML35/DED48, 106 U). As ABFI-binding <br /> and pCZ-EI (218 bp). Reaction mixtures contained 5 ng (+)                                      sites, factor Y-binding site relatively weak UAS <br /> 10   ng   (+ +) purified ABFI. (B) Gel electrophoresis mobility shift                                 (pCZ-RAP1, 2.6 U), activity increased syner- <br /> assays       performed 32P-labeled BamHI fragments                                        gistically conjunction T-rich element (pCZ- <br /> pCZ-EI (218 bp) pCZ-EI/DED48 (233 bp). Reaction mixtures                                              RAP1/DED48, 55 U). We plasmid carrying <br /> contained 4 pLg whole-cell extract (WCE), contained 20                                       tandem duplication T-rich element [pCZ-(DED48)2] <br /> ng   indicated competitor oligonucleotide. <br />                                                                                                          expressed seven times (70 U) level of <br />                                                                                                          plasmid single T-rich sequence (pCZ-DED48, 9 <br /> Thus, highly cooperative binding ABFI                                                U). In contrast, binding site centromere-binding <br /> found vitro, addition weak secondary ABFI-                                                   protein CP1 did significantly increase expression, either <br /> binding site significantly increased activation vivo.                                                 (pCZ-GAL2, 0.5 U) T-rich sequence <br />   Along similar lines, explanation observed                                                  (pCZ-GAL2/DED48, 9 U). <br /> synergism vivo ABFI-binding sites T-rich                                                 Transcriptional activation ARSI DNA. The observation <br /> element ABFI recruits transcription                                                  ABFI participates activating transcription some <br /> factors T-rich region direct cooperative inter-                                           genes leads question related its <br /> actions. This mechanism plausible considering                                                 function ARSI, origin cellular DNA replication (7). <br /> the apparently low abundance affinity T-rich-                                                  The ABFI-binding site ARS1 lies within&apos; three <br /> sequence-binding factors (Fig. 1B C). We looked                                                  functional domains characterized deletion analysis (13, <br /> evidence factor recruitment using electrophoretic                                                  54, 56, 58). Domain A, required autonomous <br />  894           BUCHMAN AND KORNBERG                                                                                 MOL. CELL. BIOL. <br />  <br />                               B           A         c                  just ABFI site ARSJ (pCZ-ARS1, Table 2). <br />                                                                        This reduced level activation simply resulted <br />                TRP1                                                    increased distance ABFI site CYCI <br />                        ABFI              ARS <br />                                                                        TATA element (35). Removal ABFI-binding site from <br />                                                                        domain B reduced expression sixfold (pCZ-A149, 0.5 U). <br />        A108        A149           pstI   BgI11                         Adding ARS consensus sequence domain A domain <br />                                                                        B reduced did eliminate expression (pCZ-A108/ <br />                pCZ-A1 49 <br />                            -Y&quot;,c                                       DOMA, 0.6 U). In comparison, silencer fragment from <br />                               &quot;,, CYC1/lacZ                            HMR E completely blocks expression ,-galactosidase in <br />                                                                        similar constructions (9; &lt;0.02 U). <br /> pCZ-A108 <br />                                                  Xhol TATA <br />                                                                                                 DISCUSSION <br /> pCZ- <br /> AB10MA                                                                    Properties ABFI. We purified ABFI specific binding <br />   FIG. 4. Transcriptional activation ARSI sequences. A map       DEDI UAS DNA. The purified protein bound specific <br /> ARS1 shown, ABFI-binding site ARS consensus            sequences variety genetic elements-UASs, silenc- <br /> sequence depicted closed open boxes, respectively. The          ers, ARSs-and appeared composed two <br /> ARS1 domains A, B, C shown above, arrow                polypeptides 135 145 kDa. These results in <br /> indicates end TRPI coding region. The endpoints          excellent agreement reported groups that <br /> deletions A108 A149 (54) marked map. These           purified protein specifically binds ARSl DNA <br /> deletions remove sequences left endpoint. In A149, 11 <br /> bp ABFI consensus region removed, A149 DNA does         (16, 62). Sweder et al. (62) described doublet 135 kDa <br /> not bind ABFI vitro (data shown). The various ARSI              following sodium dodecyl sulfate-polyacrylamide gel elec- <br /> fragments inserted pCZ plasmids illustrated below. To test    trophoresis purified preparation, found <br /> for UAS function, fragments fused CYCI        renaturation bands doublet could <br /> promoter, depicted right. Restriction sites used construc-   specifically bind ARS1 DNA. Thus, two <br /> tions marked maps ARSI CYCI.                         bands slightly different forms polypeptide. <br />                                                                        Hamil et al. (21) described partially fractionated <br />                                                                        activity, designated TAF (TCMJ activation factor), that <br /> replication, contains match consensus sequence            binds specifically UAS ribosomal protein gene <br /> found ARS elements (for review, reference 10).                TCMI. Photochemical cross-linking experiments indicated <br /> Domains B C, flank domain A, absolutely              TAF contains 147-kDa polypeptide, reasonable <br /> required significantly affect stability minichro-        agreement reported ABFI. <br /> mosomes. For example, removal 41 bp domain B,                       Previous suppositions features recognized by <br /> including ABFI-binding site, reduces minichromosome                ABFI derived comparing binding sites revealed in <br /> stability fourfold certain growth conditions (54;            DNase I protection assays (8). We numerous <br /> difference A108 A149, Fig. 4). There prece-            matches consensus sequence 5&apos;-RDCNYNNNN <br /> dents transcription factors process transcrip-          NACGAD-3&apos;, varied considerably positions, <br /> tion playing role DNA replication. In E. coli, transcrip-         bound protein high affinity vitro. Binding was <br /> tion RNA polymerase apparently activates initiation              greatly reduced single base changes modified con- <br /> replication unwinding DNA duplex (2). In adenovirus             served positions consensus sequence changed <br /> genomes, binding sites cellular transcription factors          spacing halves binding site. We have <br /> an important origin DNA replication (27, 41,            found, however, matches consensus <br /> 45). We showed (see above) oligonucleotide contain-            sequence fail bind ABFI high affinity vitro (A. R. <br /> ing ABFI-binding site domain B ARS1                    Buchman, unpublished results). The consensus sequence we <br /> activate CYCI promoter vivo. However, domain B                    used optimally reflect residues recognized <br /> appears complex structure. Sequences             ABFI, regional features DNA sequence <br /> ABFI-binding site ARS consensus sequence               significantly affect binding. For example, ABFI- <br /> impact ARS function (16, 54, 56). Also,                binding site ARSI region DNA an <br /> additional DNA-binding factors, ABF-II (16) ABF-III                intrinsically bent structure (54). <br /> (62), recognize sequences domain B. As                    Transcriptional activation ABFI. Previous studies gen- <br /> DEDI promoter, elements ARSI com-                 erated confusing picture function ABFI because <br /> bine ABFI site greatly increase activating             diversity elements bound factor vitro. <br /> effect. On hand, possible activating          Results present experiments demonstrate convinc- <br /> ability ABFI counteracted ARSI. For instance,             ingly ABFI functions situations gene <br /> silencer HMR E contains binding sites activator             activator vivo. Binding sites associated the <br /> proteins, ABFI GRFl. A key sequence, ARS,                 promoter regions numerous genes supplying important <br /> converts silencer negative regulatory element (6).            functions. (See reference 17 description ABFI- <br /> Indeed, ARS1 itself appears weak silencing effect            binding sites located upstream genes involved mito- <br /> when inserted place HMR E (30). We investigated              chondrial functions.) Some binding sites correspond <br /> activating properties sequences surrounding ARSI                 UASs defined deletion analysis (21, 35). We showed <br /> inserting portions domains A B plasmid pCZA             ABFI-binding sites act weak UASs strong <br /> a position 22 bp upstream CYCI TATA element (Fig.               UASs conjunction T-rich element, regardless of <br /> 4). UAS function measured assaying ,B-galactosidase             sites normally located ARS, silencer, <br /> activity extracts yeast strains carrying plasmid.           promoter region. A completely synthetic symmetrical <br /> Domain B showed significant UAS activity (pCZ-                   binding site activating properties natural <br /> A108, 3.2 U), 2.5-fold construction            ABFI-binding sequences, point mutations abolished <br />  VOL. 10, 1990                                     TRANSCRIPTION ACTIVATION BY YEAST ARS-BINDING PROTEIN                               895 <br />  <br /> binding resulted loss UAS activity. A single binding site    ABFI-binding site binding site GRFI was <br /> for ABFI large effect gene expression. A           eliminated. GRFI, like ABFI, functions transcrip- <br /> 10-fold effect expression promoter             tional activator contexts. It known other <br /> containing ABFI-binding site (pCZ-SPT2 versus pCZ-             activator proteins substitute GRFI ABFI the <br /> SPT2mtA -SPT2mtB pCZ-EI versus pCZ-mtIA                silencer special features ABFI and <br /> -mtIB; Table 2), 40-fold effect               GRFI possessed activators required for <br /> observed conjunction T-rich element (pCZ-SPT2/           proper silencer function. <br /> DED48 versus pCZ-SPT2mtA/DED48, Table 2). It appears,                 Combinatorial effects activator proteins. In higher eu- <br /> however, ABFI intrinsically weaker well-        caryotes, promoters generally composed multiple <br /> characterized yeast regulatory factors. When examined           binding sites activator proteins combine synergisti- <br /> similar plasmid constructions, high-affinity binding sites     cally affect transcription. The view emerged from <br /> the GAL4 GRFI activator proteins 5- 30-fold            analysis DED1 regulatory region similar to <br /> stronger UASs single ABFI-binding sites (9, 35). It           situation higher eucaryotes contrasts directly with <br /> seems unlikely limited activity ABFI-binding sites     model proposed earlier, based solely nucle- <br /> was partial occupancy protein vivo,      osome exclusion T-rich sequences (59). ABFI-binding <br /> was correlation relative binding affinity        sites combined T-rich element a <br /> vitro UAS activity (Table 2). We examined        synergistic manner create strong constitutive activator <br /> capacity ABFI stimulate transcription vitro, using      DEDI. The T-rich sequence itself compound <br /> yeast nuclear extract templates containing mul-        element, appeared recognized specifically by <br /> tiple ABFI-binding sites (35; A. R. Buchman, unpublished           factors vitro. Unfortunately, little said <br /> data). Although T-rich sequence DEDI gene               mechanisms responsible high level of <br /> caused marked stimulation vitro (as 30-fold),      activation. Synergistic effects generated factors <br /> significant stimulation observed ABFI-binding sites       act entirely different mechanisms. Our <br /> alone stimulation seen ABFI-               results suggest highly specific interactions between <br /> binding sites combined T-rich sequence.              activators necessary synergism, combina- <br /> Supplementing nuclear extract purified ABFI          tions weak activators tested effect. Furthermore, <br /> failed increase transcription templates. There       cooperative binding ABFI- T-rich-sequence-binding <br /> are possible explanations lack activation    factors vitro. It remains possible, however, <br /> ABFI vitro. ABFI act mechanism operating         detect interactions assays that <br /> in vitro reaction, ABFI damaged          used. Studies higher eucaryotes implied that <br /> preparation nuclear whole-cell extracts. Alterna-           separate classes activator proteins based their <br /> tively, intrinsic activating ability ABFI        different abilities combine synergistically themselves <br /> weak detected conditions used.                     activators (19). We tested combi- <br />    Other functions ABFI. The functions ABFI provides         nations yeast activators described here. Conse- <br /> elements associated RNA polymerase II promoters           quently, clear combinations weak acti- <br /> remain mystery. It conceivable          vators invariably act synergistically S. cerevisiae (43) or <br /> activating capacity ABFI applies situations.      yeast activators grouped different <br /> Generally, RNA polymerase III promoters tRNA genes              mechanistic categories. Nevertheless, gained infor- <br /> are located transcribed region, 5&apos;-flanking    mation basic characteristics members the <br /> sequences, ABFI-binding site tRNASer            highly abundant class yeast DNA-binding proteins. Of <br /> gene (TGS2), positive negative effects              these, factor clearly analogous binding specificity <br /> expression (32, 52). Also, DNA-binding proteins ap-           transcription factor higher eucaryotes CP1, which <br /> pear act promoters transcribed RNA poly-           recognizes sequences mammalian activator <br /> meraseII III (11). Although yeast tRNA genes           USF (48) MLTF (12). Curiously, found <br /> contain matches ABFI consensus sequence             evidence CP1 functions activator S. <br /> position, yeast sigma elements contain        cerevisiae. GRFI fairly strong transcriptional activator <br /> weak ABFI-binding site (one-half one-fourth affinity     key component UASs genes. The <br /> TGS2; A. R. Buchman, unpublished data). Transposable               present studies ABFI detailed studies of <br /> sigma elements closely associated 5&apos; ends          factor Y (D.I. Chasman, N. F. Lue, Y. Lorch, A. R. <br /> many tRNA genes (within 20 bp [47]).                               Buchman, J. Lapointe, R. D. Kornberg, manuscript in <br />    It remains possibility ABFI enhances initiation      preparation) indicate proteins intrinsi- <br /> of DNA replication ARSI transcriptional activation.          cally weak activators combine various ways with <br /> Domain B ARS1, contains ABFI-binding site,            activators yeast genes. Considering abundance <br /> had weak UAS activity. Addition ARS consensus               proteins, likely general <br /> sequence reduced did eliminate UAS function. Other         organizational features DEDI promoter repeated <br /> investigators, however, detected stable transcripts       times yeast genome. <br /> that emanate normal ARSI region (57). There may <br /> be reasons carefully regulating transcription ARSs.                             ACKNOWLEDGMENTS <br /> High levels transcription ARS1 interfere         We thank Pam Patek producing synthetic oligonucleotides. <br /> function (55). Future studies special plasmid construc-           This work supported Public Health Service         grant   GM- <br /> tions (such derivatives pCZ-A108/DOMA, Fig. 4)               36659 National Institutes Health   R.D.K. <br /> should permit simultaneous analysis transcriptional                              LITERATURE CITED <br /> activation ARS function.                                        1. Baker, R. E., A.Eigel, D. Vogel, H. Feldman. 1982. <br />    Earlier studies demonstrated binding site ABFI           Nucleotide sequences yeast genes tRNAser, <br />                                                                                                                      tRN2rg and <br /> was significant constituent transcriptional silencer       tRNA Val: homology blocks occur vicinity different <br /> HMR E (6, 30). Silencer activity reduced <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>            tRNA genes. EMBO J. 1:291-295. <br />  896      BUCHMAN AND KORNBERG                                                                                              MOL. CELL. BIOL. <br />  <br />  2. Baker, T. A., A. Kornberg. 1988. Transcriptional activation      23. Hill, D. E., I. A. Hope, J. P. Macke, K. Struhl. 1986. <br />     initiation replication E. coli chromosomal origin:         Saturation mutagenesis yeast his3 regulatory site: require- <br />     RNA-DNA hybrid near oriC. Cell 55:113-123.                             ments transcriptional induction binding GCN4 <br />  3. Bradford, M. M. 1976. A rapid sensitive method                activator protein. Science 234:451-457. <br />     quantitation microgram quantities protein utilizing        24. Huet, J., P. Cottrell, M. Cool, M. L. Vignais, D. Thiele, C. <br />     principle protein-dye binding. Anal. Biochem. 72: 248-254.             Marck, J. M. Buhler, A. Sentenac, P. Fromageot. 1985. A <br />  4. Bram, R., R. Kornberg. 1985. Specific protein binding far          general upstream-binding factor genes yeast transla- <br />     upstream activating sequences polymerase II promoters.                 tion apparatus. EMBO J. 4:3539-3547. <br />     Proc. Natl. Acad. Sci. USA 82:43-47.                                 25. Huet, J., A. Sentenac. 1987. TUF, DNA-binding factor <br />  5. Bram, R. J., R. D. Kornberg. 1987. Isolation Saccha-             specific UAS,pg upstream activating sequences: identifica- <br />     romyces cerevisiae centromere DNA-binding protein, human              tion protein DNA-binding domain. Proc. Natl. <br />     homolog, possible role transcription factor. Mol.            Acad. Sci. USA 84:3648-3652. <br />     Cell. Biol. 7:403-409.                                               26. Ito, H., Y. Fukuda, K. Murata, A. Kinura. 1983. Transfor- <br />  6. Brand, A. H., G. Micklem, K. Nasmyth. 1987. A yeast                   mation intact yeast cells treated alkali cations. J. <br />     silencer contains sequences promote autonomous plas-             Bacteriol. 153:163-168. <br />     mid replication transcriptional activation. Cell 51:709-719.     27. Jones, K. A., J. T. Kadonaga, P. J. Rosenfeld, T. J. Kelly, R. <br />  7. Brewer, B. J., W. L. Fangman. 1987. The localization               Tjian. 1987. A cellular DNA-binding protein activates <br />     replication origins ARS plasmids S. cerevisiae. Cell                eukaryotic transcription DNA replication. Cell 48:79-89. <br />     51:463-471.                                                          28. Jones, N. C., P. W. J. Rigby, E. B. Ziff. 1988. Trans-acting <br />  8. Buchman, A. R., W. J. Kimmerly, J. Rine, R. D. Kornberg.              protein factors regulation eukaryotic transcription: <br />     1988. Two DNA-binding factors recognize specific sequences             lessons studies DNA tumor viruses. Genes Dev. <br />     silencers, upstream activating sequences, autonomously repli-             2:267-281. <br />     cating sequences, telomeres Saccharomyces cerevisiae.         29. Kadonaga, J. T., R. Tjian. 1986. Affinity purification of <br />     Mol. Cell. Biol. 8:210-225.                                               sequence-specific DNA-binding proteins. Proc. Natl. Acad. Sci. <br />  9. Buchman, A. R., N. F. Lue, R. D. Kornberg. 1988. Connec-              USA 83:5889-5893. <br />     tions transcriptional activators, silencers, telo-       30. Kimmerly, W. J., A. Buchman, R. Kornberg, J. Rine. 1988. <br />     meres revealed functional analysis yeast DNA-                  Roles DNA-binding proteins replication, segregation, <br />     binding protein. Mol. Cell. Biol. 8:5086-5099.                            transcriptional repression mediated yeast silencer. <br /> 10. Campbell, J. L. 1983. Yeast DNA replication vitro               EMBO J. 77:2241-2253. <br />     vivo, p. 109-155. In J. Setlow M. Hollander (ed.), Genetic       31. Kunkel, G. R., H. G. Martinson. 1981. Nucleosomes will <br />     engineering, vol. 5. Plenum Publishing Corp., New York.                   form double-stranded RNA poly(dA):poly(dT) in <br /> 11. Carbon, P., S. Murgo, J. P. Ebel, A. Krol, T. Graham, I. W.           recombinant DNA. Nucleic Acids Res. 12:7057-7070. <br />     Mattaj. 1987. A common octamer motif binding protein               32. Larson, D., J. Bradford-Wilcox, L. S. Young, K. Sprague. <br />     involved transcription U6 snRNA RNA polymerase                1983. A short 5&apos;-flanking region containing conserved sequences <br />     II U2 snRNA RNA polymerase II. Cell 51:71-79.                      required silkworm alanine tRNA gene activity. Proc. Natl. <br /> 12. Carthew, R. W., L. A. Chodosh, P. A. Sharp. 1985. An RNA              Acad. Sci. USA 80:3416-3420. <br />     polymerase II transcription factor binds upstream element       33. Letovsky, J., W. S. Dynan. 1989. Measurement the <br />     adenovirus major late promoter. Cell 43:439-448.                   binding transcription factor SP1 single GC box recogni- <br /> 13. Celniker, S. E., K. Sweder, F. Srienc, J. E. Bailey, J. L.            tion sequence. Nucleic Acids Res. 17:2639-2653. <br />     Campbell. 1984. Deletion mutations affecting autonomously             34. Lorch, Y., R. D. Kornberg. 1985. A region flanking the <br />     replicating sequence ARSI Saccharomyces cerevisiae. Mol.                GAL7 gene binding site GAL4 protein upstream <br />     Cell. Biol. 4:2455-2466.                                                   activating sequences yeast. J. Mol. Biol. 186:821-824. <br /> 14. Chen, W., S. Tabor, K. Struhl. 1987. Distinguishing be-           35. Lue, N. F., A. R. Buchman, R. D. Kornberg. 1989. Activa- <br />     tween mechanisms eukaryotic transcriptional activation             tion yeast RNA polymerase II transcription thymidine- <br />     bacteriophage T7 RNA polymerase. Cell 50:1047-1055.                        rich upstream element vitro. Proc. Natl. Acad. Sci. USA <br /> 15. Devereux, J., P. Haeberli, 0. Smithies. 1984. A comprehen-             86:486-490. <br />      sive set sequence analysis programs VAX. Nucleic          36. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />     Acids Res. 12:387-395.                                                     cloning: laboratory manual. Cold Spring Harbor Laboratory, <br /> 16. Diffley, J. F. X., B. Stillman. 1988. Purification yeast          Cold Spring Harbor, N.Y. <br />      protein binds origins DNA replication transcrip-    37. Maniatis, T., S. Goodbourn, J. A. Fischer. 1987. Regulation <br />      tional silencer. Proc. Natl. Acad. Sci. USA 85:2120-2124.                 inducible tissue-specific gene expression. Science 236: <br /> 17. Dorsman, J. C., W. C. van HeeswiJk, L. A. Grivell. 1988.               1237-1245. <br />      Identification factors bind upstream se-         38. McNeil, J. B., M. Smith. 1985. Transcription initiation the <br />      quences number nuclear genes coding mitochondrial             Saccharomyces cerevisiae Iso-1-cytochrome c gene: multiple, <br />      proteins genetic elements important cell division           independent T-A-T-A sequences. J. Mol. Biol. 187:363-378. <br />      yeast. Nucleic Acids Res. 16:7287-7301.                              39. Miller, J. H. 1972. Experiments molecular genetics. Cold <br /> 18. Fedor, M. J., N. F. Lue, R. D. Kornberg. 1988. Statistical             Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />      positioning nucleosomes specific protein-binding         40. Ondek, B., L. Gloss, W. Herr. 1988. The SV40 enhancer <br />      upstream activating sequence yeast. J. Mol. Biol. 204:                 contains distinct levels organization. Nature (London) <br />      109-127.                                                                333:40-45. <br /> 19. Fromental, C., M. Kanno, H. Nomiyama, P. Chambon.                41. Pruijn, G. J. M., W. van Driel, P. C. van der Viiet. 1986. <br />      1988. Cooperativity hierarchical levels functional orga-         Nuclear factor III, novel sequence-specific DNA-binding <br />      nization SV40 enhancer. Cell 54:943-953.                         protein HeLa cells stimulating adenovirus DNA replica- <br /> 20. Giniger, E., S. Varnum, M. Ptashne. 1985. Specific DNA               tion. Nature (London) 322:656-659. <br />      binding GAL4, positive regulatory protein yeast. Cell       42. Prunell, A. 1982. Nucleosome reconstitution plasmid in- <br />      40:767-774.                                                             serted poly(dA):(dT). EMBO J. 1:173-179. <br /> 21. Hamil, K. G., H. G. Nam, H. M. Fried. 1988. Constitutive         43. Ptashne, M. 1988. How eukaryotic transcriptional activators <br />      transcription yeast ribosomal protein gene TCMJ promoted          work. Nature (London) 335:683-689. <br />      uncommon cis- trans-acting elements. Mol. Cell. Biol.        44. Roeder, G. S., C. Beard, M. Smith, S. Keranen. 1985. <br />       8:4328-4341.                                                           Isolation characterization SPT2 gene, negative <br /> 22. Herr, W., J. Clarke. 1986. The SV40 enhancer composed             regulator TY-controlled yeast gene expression. Mol. Cell. <br />     multiple functional elements compensate              Biol. 5:1543-1553. <br />     another. Cell 45:461-470.                                            45. Rosenfeld, P. J., E. A. O&apos;Neil, R. J. Wides, T. J. Kelly. 1987. <br />  VOL. 10, 1990                                         TRANSCRIPTION ACTIVATION BY YEAST ARS-BINDING PROTEIN                              897 <br />  <br />       Sequence specific interactions cellular DNA-binding       54. Snyder, M., A. R. Buchman, R. W. Davis. 1986. Bent DNA <br />       proteins adenovirus origin replication. Mol. Cell.         yeast autonomously replicating sequence. Nature (London) <br />       Biol. 7:875-886.                                                      324:87-89. <br /> 46.   Sachs, A. B., M. W. Bond, R. D. Kornberg. 1986. A single      55. Snyder,  <br /> </body></html>