<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The C-Terminal Domain of the Novel Essential Protein <br /> Gcp Is Critical for Interaction with Another Essential <br /> Protein YeaZ of Staphylococcus aureus <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Previous studies have demonstrated that the novel protein Gcp is essential for the viability of various bacterial species <br />      including Staphylococcus aureus; however, the reason why it is required for bacterial growth remains unclear. In order <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />      explore the potential mechanisms of this essentiality, we performed RT-PCR analysis and revealed that the gcp gene <br />      (sa1854) was co-transcribed with sa1855, yeaZ (sa1856) and sa1857 genes, indicating these genes are located in the same <br />      operon. Furthermore, we demonstrated that Gcp interacts with YeaZ using a yeast two-hybrid (Y2H) system and in vitro pull <br />      down assays. To characterize the Gcp-YeaZ interaction, we performed alanine scanning mutagenesis on the residues of C- <br />      terminal segment of Gcp. We found that the mutations of the C-terminal Y317-F322 region abolished the interaction of Gcp <br />      and YeaZ, and the mutations of the D324-N329 and S332-Y336 regions alleviated Gcp binding to YeaZ. More importantly, <br />      we demonstrated that these key regions of Gcp are also necessary for the bacterial <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>survival</span> since these mutated Gcp could <br />      not complement the depletion of endogenous Gcp. Taken together, our data suggest that the interaction of Gcp and YeaZ <br />      may contribute to the essentiality of Gcp for S. aureus survival. Our findings provide new insights into the potential <br />      mechanisms and biological functions of this novel essential protein. <br />  <br />  <br /> Introduction                                                                          [10], Escherichia coli [11], Bacillus subtilis [12,13], Francisella novicida <br />                                                                                       [14], Pseudomonas aeruginosa [15], and Mycoplasm genetalium [16]. <br />    The prevalence of methicillin-resistant Staphylococcus aureus                      Additionally, the protein is important for eukaryotes, such as <br /> (MRSA) and vancomycin-resistant S. aureus (VRSA) has caused                           Saccharomyces cerevisiae [17] and Arabidopsis thaliana [18]. We have <br /> serious public health concerns worldwide [1,2]. The limited                           demonstrated that S. aureus Gcp plays an important role in the <br /> options of antibiotics for the treatment of infections associated with                process of bacterial autolysis, suggesting a potential role in the cell <br /> MRSA and/or VRSA highlight an urgent need for the                                     wall biosynthesis pathway [6], however, the reason why Gcp is <br /> development of novel potent antimicrobial agents. Bacterial                           required for bacterial viability remains elusive. <br /> essential proteins are potential targets for the development of                          Structural analysis of Gcp homologs shows that Gcp belongs to <br /> new classes of antibiotics [3,4]; however, the biological function of                 the ASKHA (acetate and sugar kinases, HSP70 and actin) <br /> many essential proteins is still unclear. The characterization and                    superfamily [18&#226;&#8364;&#8220;22]. A conserved metal binding motif HXEXH <br /> validation of functional unknown essential proteins are therefore of                  is inserted within the HSP70 (heat-shock protein70)-actin-like fold <br /> great importance to assess their suitability as targets for the                       (HALF), suggesting a metal binding ability and an ATP dependent <br /> development of the novel antibiotics.                                                 protease activity [18]. The crystal structure analysis of the Gcp <br />    Our previous studies have indicated that a novel essential                         Pyrococcus abyssi ortholog, Pa-Kae1, has revealed that Pa-Kae1is <br /> protein Gcp is a potential target for the development of new classes                  multifunctional, binding iron and ATP, as well as possessing DNA <br /> of antibacterial agent against MRSA and/or VRSA [5,6]. Gcp is                         binding and apurinic endonuclease activity [20], whereas the Gcp <br /> homologous to the Gcp first identified in Mannheimia haemolytica                      homolog in yeast, Kae1 (kinase -associated endopeptidase 1), is a <br /> (formerly Pasteurella haemolytica) and specifically cleaves O-sialogly-               component of the KEOPS/EKC (kinase, endopeptidase and other <br /> cosilated protein [7]. Gcp homologs are ubiquitous in all three                       proteins of small size/endopeptidase-like and kinase associated to <br /> kingdoms of life, with the exception of the endosymbiotic bacteria                    transcribed chromatin) complex that are required for telomere <br /> Carsonella ruddii and Sulcia muelleri, which possess highly reduced                   maintenance and transcription of essential eukaryotic genes <br /> genomes [8,9]; however, these Gcp homologs do not exhibit                             [21,23,24]. Collectively, the above data indicate that Gcp <br /> glycoprotease activity. It has been revealed that these Gcp                           homologs may possess different functions among different species. <br /> homologs are required for cell viability of many bacterial species                       Recently, it has been reported that in E. coli, an essential protein <br /> examined to date, including S. aureus [10], Streptococcus pneumonia                   YeaZ binds to YgjD (the Gcp homolog in E. coli) and YjeE, which <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                                   May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                          C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br /> are both essential for E. coli growth [25,26]. YeaZ is a bacterial <br /> specific member of the ASKHA superfamily. It is also essential for <br /> growth of a variety of bacterial species [3,10&#226;&#8364;&#8220;15], such as S. aureus <br /> [10] and E. coli [11]. Crystal structural analyses reveal that the E. <br /> coli [27], Samonella typhimurium and Thermotoga maritime YeaZs <br /> possess a HALF fold, but lack the metal binding motif and the <br /> ATP binding site [28,29]. This suggests that YeaZ may bind to a <br /> nucleotide through an interaction with a small molecule ligand or <br /> a partner protein to adopt an active conformation [28]. In <br /> addition, EcYeaZ can specifically cleave YgjD [26], but StYeaZ <br /> does not exhibit such an enzymatic activity [28]. <br />    Sequence alignment analysis revealed that S. aureus Gcp, YeaZ, <br /> and SA1857 proteins are the homologs of YgjD, YeaZ, and YjeE <br /> in E. coli, respectively [26], however, whether these proteins <br /> interact with each other in S. aureus is currently unknown. In this <br /> study, we utilized the genetic and biochemical approaches and <br /> demonstrated that the staphylococcal Gcp binds to YeaZ. <br /> Importantly, we have identified the key domains of Gcp that are <br /> important for Gcp-YeaZ interaction, as well as critical for Gcp&#226;&#8364;&#8482;s <br /> essentiality for S. aureus growth. <br />  <br /> Results <br /> Both gcp and yeaZ genes are located in the same operon <br />   Genomic DNA sequence alignment for S. aureus revealed that <br /> the sequences of sa1857, yeaZ (sa1856), sa1855, and gcp (sa1854) are <br /> sequentially overlapped (Fig. 1A). It was revealed that the first <br /> 47 bps of yeaZ overlap with the last 47 bps of sa1857; the first <br /> 28 bps of sa1855 overlap with the last 28 bps of yeaZ; and the first <br /> 8 bps of gcp overlap with the last 8 bps of sa1855. This data <br />                                                                              Figure 1. The co-transcription of the sa1857, sa1856 (yeaZ), <br /> suggests that the four genes might be co-transcribed <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> a                  sa1855, and sa1854 (gcp) genes. (A) Schematic representation of <br /> common promoter. To examine this possibility, we performed                   sa1857, sa1856 (yeaZ), sa1855, and sa1854 (gcp) genes in S. aureus. (B) <br /> RT-PCR using the forward primers P1for, P2for, and P3for                     RT-PCR using different primer pairs. RT (2), total RNA of S. aureus used <br /> specifically binding to sa1857, yeaZ and sa1855, respectively, and a         as the template; RT (+), cDNA used as the template; positive controls, <br /> common reverse primer P4rev specifically binding to gcp. The RT              the genomic DNA used as the template (lane 1, 4, 7); M, 1 kb DNA <br /> negative controls using total RNA as a template did not yielded a            ladder. <br />                                                                              doi:10.1371/journal.pone.0020163.g001 <br /> PCR product (Fig. 1B, lane 2, 5, 8). In contrast, when cDNA was <br /> used as a template, PCR products of 1.4 kb, 800 bp, and 400 bp <br /> were obtained, corresponding to the expected sizes (Fig. 1B),                   To further confirm the interaction between staphylococcal Gcp <br /> indicating that the genes are co-transcribed from at lease one               and YeaZ, we performed in vitro pull down assays using the <br /> common promoter.                                                             purified His-tagged Gcp and GST-YeaZ fusion proteins. The <br />                                                                              GST tag was used to immobilize YeaZ on glutathione Sepharose <br />                                                                              resin. Western blot analysis showed that the YeaZ bound resin was <br /> Staphylococcal Gcp interacts with YeaZ <br />                                                                              able to retain the refolded Gcp, whereas the control resin alone <br />    In E. coli the Gcp homolog, YgjD, interacts with YeaZ [25,26]; <br />                                                                              could not retain the Gcp (Fig. 2B), indicating that the recombinant <br /> in S. aureus, sa1857, yeaZ, sa1855, and gcp genes are localized to the <br />                                                                              staphylococcal Gcp specifically binds to the recombinant staph- <br /> same operon. These led us to hypothesize that in S. aureus Gcp may <br />                                                                              ylococcal YeaZ. <br /> interact with YeaZ, SA1855, and/or SA1857 and function <br />                                                                                 The overexpression of Gcp in E. coli resulted in the recombinant <br /> coordinately. To test this hypothesis, we utilized a yeast two- <br />                                                                              protein forming inclusion bodies, thus the recombinant protein <br /> hybrid system by fusing Gcp, YeaZ, SA1855 and SA1857 <br /> separately with the GAL4 activation domain (GAD) and the                     was purified under denaturing conditions, and refolded by a series <br /> GAL4 DNA binding domain (GBD), respectively. Two different                   of dialysis. Sequence analysis of Gcp revealed four cysteine <br /> fusion plasmids were co-transformed into the yeast PJ69-4A                   residues in the C-terminus, thus it is possible for the purified <br /> competent cells. The results showed that the negative controls               recombinant Gcp molecules to form intermolecular disulfide <br /> carrying pGAD-gcp/pGBD, or pGAD/pGBD empty vectors did                       bonds, which would result in malfunctioning Gcp polymers. To <br /> not grow in the media lacking His/Leu/Trp, and media lacking                 rule out this possibility, we examined the integrity of unfolded and <br /> Ade/Leu/Trp (Fig. 2A). In contrast, yeast cells carrying pGAD-               refolded recombinant Gcp proteins by SDS-PAGE under non- <br /> gcp/pGBD-yeaZ grew normally on the above selective media,                    reducing condition and did not reveal any oligomers of Gcp (data <br /> indicating a possible binding interaction between Gcp and YeaZ               not shown). <br /> (Fig. 2A). No other interactions were revealed among these four <br /> proteins (data not shown). In addition, yeast cells carrying either          C-terminal Y317&#226;&#8364;&#8220;Y336 segment is crucial for the <br /> pGAD-yeaZ/pGBD-gcp or pGAD/pGBD-gcp grew in the above                        staphylococcal Gcp binding to YeaZ <br /> selective media, suggesting that GBD-Gcp fusion may interact                    In order to identify which domains of staphylococcal Gcp are <br /> with the activation domain leading to an auto-activation of the              important for the Gcp-YeaZ interaction, we created nine different <br /> reporter (data not shown).                                                   truncated Gcp mutants by PCR (Fig. 3A). These truncated Gcp <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                               May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                           C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />  <br />  <br /> Figure 2. The determination of Gcp binding to YeaZ. (A) Yeast two hybrid analysis of interaction between Gcp and YeaZ on histidine, leucine, <br /> and tryptophan drop out synthetic complete (SC-His, -Leu, -Trp) plates with 3-amino-1,2,4,-triazole (3-AT) and SC plates lacking adenine, leucine, and <br /> tryptophan (SC-Ade, -Leu, -Trp). Gcp was fused with the activation domain and YeaZ was fused to binding domain. The minus signs indicate empty <br /> vector controls. (B) In vitro immunoprecipitation analysis of interaction between Gcp and YeaZ. His-tagged Gcp and GST-tagged YeaZ were purified <br /> from E. coli. Gcp and YeaZ were incubated with Glutathione Sepharose 4B beads together or separately. Western blotting was carried out with rabbit- <br /> anti-Gcp serum to detect Gcp. Lane 1: purified Gcp protein was loaded as a positive control. <br /> doi:10.1371/journal.pone.0020163.g002 <br />  <br /> segments were fused with the activation domain of pGAD,                       examined the complementary effect of gcp expression in trans by <br /> respectively, and the resulting recombinant plasmids were utilized            measuring the bacterial growth during the depletion of endoge- <br /> to conduct Y2H analyses with pGBD-yeaZ. The results showed                    nous Gcp. The results showed that the growth of the staphylo- <br /> that the yeast cells carrying Gcpseg1 mutant (E337&#226;&#8364;&#8220;E341                       coccal Gcp complementary strain was IPTG-independent, indi- <br /> truncated Gcp) and YeaZ grew on both SC-His, -Leu, -Trp and                   cating the gcp expression in trans complements the depletion of <br /> SC-Ade, Leu, -Trp selective plates. However, the yeast cells                  endogenous Gcp (Fig. 5B), whereas the growth of the parental <br /> carrying other truncated Gcp fusions and YeaZ did not grow on                 control was IPTG-dependent (Fig. 5C). These data further <br /> these selective plates (Fig. 3B).                                             demonstrated the requirement of Gcp for bacterial viability in <br />    Alanine scanning mutagenesis has been successfully and                     culture and the feasibility of the Gcp complementary system. <br /> extensively utilized in the determination of enzymatic and                       Next, we determined whether the identified segment and <br /> functional residues of proteins [30&#226;&#8364;&#8220;32]. Substitution of alanine              regions that are crucial for Gcp interaction with YeaZ are also <br /> removes all side chain atoms beyond b carbon and yet maintains                required for growth by examining the complementary effect of the <br /> the main chain structure [30]. To further identify which residue(s)           mutated Gcp genes. The results showed that the growth of the <br /> is required for Gcp binding to YeaZ, we utilized this approach to             Pspac-regulated gcp expression strain carrying complementary <br /> substitute potential hot residues of the C-terminal Y317&#226;&#8364;&#8220;Y336                 Gcpseg1 was independent of IPTG (Fig. 5D), indicating that the <br /> segment with alanine. The critical Y317&#226;&#8364;&#8220;Y336 segment was                      expression of Gcpseg1 in trans can complement Gcp and the C- <br /> divided into three regions, including Y317-F322, D324-N329, and               terminal region E337&#226;&#8364;&#8220;E341 is dispensable for Gcp&#226;&#8364;&#8482;s essentiality. In <br /> S332-Y336 regions as shown in Fig. 4A. Five to six amino acids in             contrast, the growth of the Pspac-regulated gcp expression strain <br /> these regions were replaced by alanine stretches, respectively. The           carrying complementary Gcpseg2 was still dependent on IPTG <br /> mutated gcp fragments were fused with the activation domain of                (Fig. 5E). To determine which domain is critical for growth, we <br /> pGAD, which was used for the Y2H assays. We found that the                    examined the complementary effect of Gcp317-322A, Gcp324- <br /> yeast cells carrying with Gcp317-322A and YeaZ did not grow on                329A and Gcp332-336A, and found that the Pspac-regulated gcp <br /> any SC dropout selective medium (Fig. 4B); the yeast cells carrying           expression strains carrying Gcp317-322A (Fig. 5F), Gcp324-329A <br /> Gcp324-329A or Gcp332-336A, and YeaZ appeared to have a                       (Fig. 5G), or Gcp332-336A (Fig. 5H) mutant exhibited IPTG <br /> dramatic growth defect on the selective medium (Fig. 4B). Taken               dependent growth, indicating that these mutated Gcp are unable <br /> together, the above data indicate the importance of the C-terminal            to complement the depletion of endogenous Gcp. These data <br /> region of Gcp for proper interaction with YeaZ, specifically the              suggest that the three domains, Y317-F322, D324-N329, and <br /> region of Y317-F322 and to a lesser extent the regions of D324-               S332-Y336, are likely critical for Gcp&#226;&#8364;&#8482;s essentiality. <br /> N329 and S332-Y336. <br />                                                                               No O-sialoglycoprotein endopeptidase activity was <br /> Staphylococcal Gcp alanine mutants cannot complement                          detected for the recombinant staphylococcal Gcp and <br /> wild type Gcp for bacterial growth                                            YeaZ proteins <br />    We have previously demonstrated that Gcp is essential for S.                  To determine whether the novel essential proteins Gcp and <br /> aureus growth [5,6], but the molecular mechanisms of Gcp&#226;&#8364;&#8482;s                    YeaZ possess an O-sialoglycoprotein endopeptidase activity, the <br /> essentiality remain elusive. The discovery that Gcp interacts with            purified recombinant Gcp and YeaZ were utilized to treat O- <br /> YeaZ led us to hypothesize that their interaction is important for            sialoglycoprotein, glycophorin A, according to the manufacturer&#226;&#8364;&#8482;s <br /> Gcp&#226;&#8364;&#8482;s essentiality. In order to examine this hypothesis, we first             instruction. The degradation of glycophorin A protein was <br /> created the Gcp complementary system (Fig. 5A). Then, we                      detected by SDS-PAGE and western blot using a monoclonal <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                               May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                              C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />                                                                                   findings may provide new insights into the molecular mechanisms <br />                                                                                   and biological function of the essential protein, Gcp. <br />                                                                                      The identification of the staphylococcal Gcp-YeaZ interaction is <br />                                                                                   consistent with the previous report that the Gcp homolog in E. coli, <br />                                                                                   YgjD, binds to YeaZ the homolog of SA1856 [25,26]. However, in <br />                                                                                   E. coli another essential protein, YjeE, also interacts with YeaZ, <br />                                                                                   suggesting that YjeE may function as a regulator to modulate <br />                                                                                   YeaZ-YgjD interaction, and that the YjeE-YeaZ-YgjD network <br />                                                                                   may be involved in an essential cellular process [26]. In contrast, in <br />                                                                                   S. aureus there is no report indicating that the homolog of YjeE, <br />                                                                                   SA1857, is indispensable for bacterial viability [10]. We are in the <br />                                                                                   process of examining the requirement of SA1857 and SA1855 for <br />                                                                                   the growth of S. aureus. Furthermore, in our Y2H studies we found <br />                                                                                   no evidence that SA1857 and SA1855 interacts with either Gcp or <br />                                                                                   YeaZ. Although the staphylococcal YeaZ and Gcp proteins have <br />                                                                                   approximately 26% and 42% identity compared to E. coli YeaZ <br />                                                                                   and YgjD sequences, respectively, our studies indicate that Gcp <br />                                                                                   and YeaZ likely function differently or interact with different <br />                                                                                   partner proteins compared to their homologs in E. coli. We <br />                                                                                   observed that in Y2H assays the growth of yeast cells on SC-His <br />                                                                                   plates seems to be faster than on SC-Ade plates. This is likely due <br />                                                                                   to the stringent growth difference between SC-Ade and SC-His <br />                                                                                   selective medium [33]. Although we unveiled that the <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>mutation</span> of <br />                                                                                   Y317-F322 region eliminated the ability of Gcp to bind YeaZ, and <br />                                                                                   the mutations of D324-N329 and S332-Y336 regions alleviated <br />                                                                                   the capacity of Gcp to bind YeaZ, the attenuated binding ability <br />                                                                                   may be attributable to the altered tertiary conformation of Gcp. <br />                                                                                   To determine the potential impact of topology for mutated Gcp <br />                                                                                   proteins is beyond scope of the present study. However, we are <br />                                                                                   currently working to further characterize and define the <br />                                                                                   interaction of Gcp and YeaZ and identify the critical residue(s) <br />                                                                                   for Gcp to bind YeaZ and vice versa. <br /> Figure 3. Diagrams represent different deletion mutations of                         In both yeast and archaea, Kae1p (Gcp homolog) directly <br /> Gcp (A); and interactions between S. aureus Gcp and YeaZ                          interacts with a Bud32p kinase, inhibiting the kinase activity, <br /> detected by yeast two-hybrid analyses (B). Yeast cells containing                 which is required both for transcription and the telomere <br /> the indicated plasmids were serially diluted, and 5 ml of diluted cells was       homeostatic function of the endopeptidase-like kinase chromatin- <br /> spotted on the plate with media lacking histidine, leucine, and <br /> tryptophan; or lacking adenine, leucine, and tryptophan to score for              associated (EKC)/kinase, endopeptidase, and other peptidases of <br /> interactions after incubation at 30uC.                                            small size (KEOPS) in yeast cells [21,23,24]. In eukaryotes, the <br /> doi:10.1371/journal.pone.0020163.g003                                             Gcp homolog is Qri7 in yeast and OSGEPL1 (O-sialoglycoprotein <br />                                                                                   endopeptidase) in worm and human. Qri7 and OSGEPL1 anchor <br /> antibody against glycophorin A. Significant degradation of                        to the mitochondrial inner membrane and are essential for the <br /> glycophorin A was detected after treatment with Gcp of M.                         maintenance of mitochondrial genome [18,34]. Qri7 is able to <br /> haemolytica A1 (Fig. 6A), whereas no degradation of glycophorin A                 complement the depletion of E. coli YgjD (homolog of Gcp), <br /> was detected after treatment with recombinant Gcp, YeaZ, or the                   whereas Kae1 fails to do so, suggesting the functional similarity <br /> combination of Gcp and YeaZ (Fig. 6A). In addition, we revealed                   between Qri7 and YgjD [34]. Although bacteria lack a Bud32 <br /> that the purified recombinant staphylococcal YeaZ did not cleave                  homolog, YeaZ may substitute for Bud32 to form a functional <br /> recombinant Gcp of S. aureus under the conditions in the presence                 complex with Gcp. Our complementation experiments established <br /> or absence of Mg2+, Mn2+, Ca2+, Zn2+ (Fig. 6B).                                   that the mutations of Gcp that disrupt the Gcp-YeaZ interaction <br />                                                                                   could not complement the depletion of endogenous Gcp. We also <br /> Discussion                                                                        observed that the deletion of the C-terminal residues, E337&#226;&#8364;&#8220;E341, <br />                                                                                   slightly impaired the ability of Gcp to interact with YeaZ (Fig. 3B); <br />    In the present study, we employed genetic and biochemical                      and consequently, this truncated Gcp (Gcpseg1) delayed its <br /> approaches and demonstrated that in the gcp operon of S. aureus,                  complementary effect on bacterial growth (longer lag-phase of <br /> the essential Gcp protein interacts with another essential protein                growth). These data suggest that the staphylococcal Gcp-YeaZ <br /> YeaZ. More importantly, we identified key domains of Gcp that                     interaction may play an important role in Gcp&#226;&#8364;&#8482;s essentiality; thus <br /> are not only required for Gcp binding to YeaZ, but also play                      the interruption of Gcp-YeaZ interaction may be utilized as a <br /> important roles in Gcp&#226;&#8364;&#8482;s essentiality for bacterial growth. Our                   novel mode of action for developing new classes antibacterial <br /> results suggest that the interaction of Gcp and YeaZ may                          agents, partially against MRSA or VRSA caused infections. It is <br /> contribute to the essentiality of Gcp. In addition, our results                   known that bacterial autolysis is important for bacterial cell <br /> showed that there was neither O-sialoglycoprotein endopeptidase                   division and growth. Previously, we have demonstrated that the <br /> activity nor Gcp specific protease activity detected for the purified             staphylococcal Gcp is a critical modulator of bacterial autolysis <br /> recombinant staphylococcal Gcp and/or YeaZ, suggesting that the                   [6]; however, the mechanism of regulation of autolysis by Gcp <br /> staphylococcal Gcp and YeaZ may function differently compared                     remains to be determined; it is necessary to explore the potential <br /> to their homologs in M. hemolytica and E. coli, respectively. These               role of the Gcp-YeaZ interaction in the process of autolysis. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                             4                              May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                           C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />  <br />  <br /> Figure 4. Diagrams represent different alanine substitutions of amino acids in Gcp (A); and interactions between S. aureus Gcp and <br /> YeaZ detected by yeast two-hybrid analyses (B). Yeast cells containing the indicated plasmids were serially diluted, and 5 ml of diluted cells was <br /> spotted on the plate with media lacking histidine, leucine, and tryptophan (SC-His, -Leu, -Trp); or lacking adenine, leucine, and tryptophan (SC-Ade, - <br /> Leu, -Trp) to score for interactions after incubation at 30uC. <br /> doi:10.1371/journal.pone.0020163.g004 <br />  <br />  <br />    Our purified recombinant staphylococcal Gcp and YeaZ did                   the potential mechanisms and biological function of the novel <br /> not appear to have an O-sialoglycoprotein endopeptidase activity              essential protein, Gcp, as well as potential novel targets for the <br /> against glycophorin A. We also employed an alternative                        development of new classes antibacterial agents. <br /> approach to examine the potential glycoprotease activity by <br /> measuring cell associated ligand, PSGL1 (P-selectin glycoprotein              Materials and Methods <br /> ligand 1), using FACS. Neither the purified recombinant <br /> staphylococcal Gcp, YeaZ, nor the concentrated supernatants                   Bacterial strain and culture medium <br /> of S. aureus culture appeared to have the glycoprotease activity                 The bacterial strains and plasmids used in the study are listed in <br /> (data not shown). Our results are in agreement with previous                  Table 1. E. coli strain, DH10B, was used as a host for recombinant <br /> reports that YgjD (homolog of Gcp in E. coli) and Kae1 (homolog               plasmid construction and amplification. Luria-Bertani (LB) liquid <br /> of Gcp in Pyrococcus abyssi) do not exhibit any endopeptidase                 medium and LB-agar plates were used for the growth and <br /> activity [20,26]. In addition, E. coli YeaZ is able to specifically           maintenance of E. coli. The E. coli strain BL21(DE3) was used as a <br /> cleave YgjD (the homolog of Gcp), whereas consistent with S.                  host strain for the expression of recombinant Gcp and YeaZ. <br /> typhimurium YeaZ [28] our purified soluble recombinant staphy-                Ampicillin was used at 100 mg/ml in LB media for the selection of <br /> lococcal YeaZ did not exhibit such a protease activity. The lack of           E. coli carrying the plasmid pGEX4T-1, pMY1107, or Yeast two- <br /> protease activity of purified recombinant Gcp and YeaZ may                    hybrid (Y2H) vectors pGAD-C1 and pGBD-C1. Kanamycin was <br /> result from potential modifications during expression and                     used at 50 mg/ml in LB media for the selection of E. coli carrying <br /> purification process. However, it is likely that the Gcp and YeaZ             plasmid pET24b. A Saccharomyces cerevisiae strain, PJ69-4A, was <br /> homologs may have different biological functions among different              used in Y2H studies [33]. The PJ69-4A strain has three reporter <br /> species, because sequence alignment analysis revealed that the                genes for a positive Y2H interaction: HIS3 driven by GAL1 <br /> staphylococcal Gcp does not possess any E. coli YeaZ cleavage                 promoter, ADE2 by the GAL2 promoter, and lacZ by the GAL7 <br /> sites, including K171- L172 and F195-V196 residues that were                  promoter. YEPD (1% yeast extract, 2% peptone, 2% dextrose, <br /> identified in YgjD [26]. In addition, we cannot exclude the                   pH 6.0) and synthetic complete (SC) liquid medium and plates <br /> possibility that the staphylococcal Gcp and YeaZ proteins may                 were used for S. cerevisiae growth. Where indicated, certain <br /> possess a specific proteolytic activity against a substrate in a              components were dropped out from the SC media. A Pspac- <br /> process that is critical for the viability of S. aureus.                      regulated gcp mutant of S. aureus was constructed as previously <br />    In conclusion, we demonstrated within the four novel proteins              described [5], which was used as a host cell for complementation <br /> encoded by the gcp operon, the essential Gcp interacts with                   experiments. S. aureus was grown in Tryptic Soy Broth (TSB) <br /> another essential protein YeaZ of S. aureus. Moreover, we                     containing erythromycin (5 mg/ml) and different concentrations of <br /> identified the C-terminal Y317-F322, D324-N329, and S332-                     isopropyl b-D-1-thiogalactopyranoside (IPTG) at 37uC with <br /> Y336 regions to be important for Gcp to bind YeaZ, as well as for             shaking. <br /> Gcp&#226;&#8364;&#8482;s essentiality, whereas the C-terminal E337&#226;&#8364;&#8220;E344 region is <br /> dispensable for the staphylococcal Gcp-YeaZ interaction and                   RNA purification and RT-PCR analysis <br /> Gcp&#226;&#8364;&#8482;s essentiality. These data suggest that the interaction of Gcp              Overnight cultures of S. aureus were used at 1% inoculum in <br /> and YeaZ may at least partially contribute to the essentiality of             TSB medium and grown to the medium-exponential phase of <br /> Gcp for S. aureus growth. Our findings provide new insights into              growth. Cells were harvested by centrifugation, and the total RNA <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                               May 2011 | Volume 6 | Issue 5 | e20163 <br />                                   C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org   6   May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                         C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />  <br /> Figure 5. The impact of the expression of wild type or mutated Gcp in trans on the growth of the Pspac-regulated gcp expression <br /> strain. (A) Construction of Gcp complementation system using the Pspac-regulated gcp expression strain and Western blot analysis of gcp expression <br /> in trans. Lane 1, purified recombinant Gcp; lane 2, negative control (JRN0210); lane 3, Gcp complementary strain (JNR0110). Without an inducer, ATc, <br /> Gcp is constitutively expressed due to the leakage of Pxyl/tetO promoter. (B&#226;&#8364;&#8220;H) Growth curves of Gcp complementary strains. The Pspac-regulated <br /> gcp expression S. aureus strain carrying the staphylococcal wild-type gcp complementary plasmid, pXL108 was used as positive control (B); carrying <br /> parental plasmid, pMY1107, as a negative control (C); carrying the gcpseg1 complementary plasmid, pLT109 (D); carrying the gcpseg2 complementary <br /> plasmid, pLT209 (E); carrying the gcp1-1 complementary plasmid, pLT309 (F); carrying the gcp1-2 complementary plasmid, pLT409 (G); carrying the <br /> gcp1-3 complementary plasmid, pLT509 (H). The above strains were incubated overnight in TSB in the presence of appropriate antibiotics and <br /> 0.2 mM IPTG. Bacteria were diluted and incubated in fresh TSB containing appropriate antibiotics and different concentration of IPTG at 37uC with <br /> shaking. The cell growth was monitored by measuring OD600 nm every 15 min for 18 h with 1 min mixing before each reading. <br /> doi:10.1371/journal.pone.0020163.g005 <br />  <br />  <br />  <br /> was isolated by using the RNAPrep kit (Promega, MI).                         and tryptophan at 30uC. The examination was performed either <br /> Contaminating DNA was removed with a TURBO DNA-free                          by streak plates or by spotting plates with serial dilutions of <br /> Kit (Ambion), and the RNA yield was determined spectrophoto-                 suspension on SC plates lacking leucine, tryptophan and histidine <br /> metrically at 260 nm. The cDNA was synthesized by employing                  with 2 mM 3-amino-1,2,4,-triazole (3-AT) (SC-His, -Leu, -Trp) <br /> RNA as a template using the Promega-reverse transcription kit.               and SC plates lacking leucine, tryptophan and adenine (SC-Ade, - <br /> Co-transcription of gcp operon was determined by RT-PCR using                Leu, -Trp). For the spots assays, yeast cells in exponential growth <br /> primers listed in Table 2.                                                   were harvested and resuspended at 46106 cells per milliliter. A <br />                                                                              volume of 5 ml of this cell suspension and its 5 fold serial dilutions <br /> Examination of protein interactions using the Y2H                            were spotted on plates with decreasing concentrations from left to <br />                                                                              right. Growth on plates was examined daily and documented after <br /> system <br />                                                                              4 days of incubation. <br />    To examine possible protein-protein interaction, we employed a <br /> yeast two-hybrid (Y2H) technology as described [33,35]. The <br /> staphylococcal gcp, sa1855, yeaZ and sa1857 genes were amplified             Cloning, expression and purification of recombinant Gcp- <br /> by PCR using the primers listed in Table 1. PCR products were                His tag fusion protein and GST-YeaZ fusion protiens <br /> purified, cloned into EcoRI and BglII sites of pGAD-C1 two-hybrid               The staphylococcal gcp and yeaZ genes were obtained by PCR <br /> vector containing GAL4 transcription activation domain (GA-                  using the primers listed in Table 2 and cloned into NdeI and XhoI <br /> L4AD) and BamHI and SalI sites of pGBD-C1 vector containing                  sites of pET24b and BamHI and SalI sites of pGEX4T-1, <br /> GAL4 DNA-binding domain (GAL4BD), respectively. The yeast                    respectively. The recombinant plasmids were transformed into <br /> strain PJ69-4A was co-transformed with the two fusion plasmids               E. coli strain BL21 (DE3), respectively. An approximately 500 ml <br /> using the high efficiency LiAc-PEG method [35]. The yeast                    culture of each strain in LB medium was incubated aerobically at <br /> transformants were grown on SC drop-out plates lacking leucine               37uC to an optical density (OD) at 600 nm of 0.6. The <br />                                                                              recombinant protein expression was subsequently induced by the <br />                                                                              addition of 1 mM IPTG. After 5 h of incubation at 37uC, cells <br />                                                                              were harvested. For the purification of Gcp-His fusion proteins, <br />                                                                              the cell pellet was collected and resuspended in the binding buffer <br />                                                                              (300 mM NaCl, 50 mM NaH2PO4, 8 M urea, 10 mM imidazole, <br />                                                                              pH 8.0) with the addition of 1 mg/ml lysozyme and 1 mM PMSF <br />                                                                              at room temperature for 1 h. The cells were lysed by sonication on <br />                                                                              ice; after centrifugation at 25,000 g for 20 min at 4uC, the <br />                                                                              supernatant was collected and applied onto Ni-NTA resin <br />                                                                              (Qiagen) under denaturing condition according to the manufac- <br />                                                                              turer&#226;&#8364;&#8482;s instruction. The purified recombinant Gcp proteins were <br />                                                                              refolded by dialysis using a series of dialysis buffers (300 mM <br />                                                                              NaCl, 50 mM NaH2PO4, 10% glycerol with 4 M, 2 M, 1 M, 0 M <br />                                                                              urea, final buffer 150 mM NaCl, 20 mM NaH2PO4, PH 7.3). For <br />                                                                              the purification of GST-YeaZ proteins, the cell pellet was <br />                                                                              resuspended in the binding buffer (150 mM NaCl, 20 mM <br />                                                                              NaH2PO4, pH 7.3) containing 1 mg/ml lysozyme and 1 mM <br />                                                                              PMSF at room temperature for 1 h. The cells were lysed by <br />                                                                              sonication on ice; after centrifugation at 25,000 g for 20 min at <br />                                                                              4uC, the supernatant was collected and mixed with Glutathione <br />                                                                              Sepharose 4B beads (GE Healthcare) according to the manufac- <br />                                                                              turer&#226;&#8364;&#8482;s instructions. The purified recombinant proteins were <br /> Figure 6. SDS-PAGE and Western blot analysis of Gcp and <br />                                                                              confirmed by SDS-polyacrylamide gel electrophoresis (PAGE) <br /> YeaZ&#226;&#8364;&#8482;s protease activity. (A) Western blot results showing hydrolysis        followed by Coomassie bright blue staining; the protein concen- <br /> of Glycophorin A. GPA2 and GPA represent dimer and monomer of                tration was measured using the BCATM Protein Assay kit (Thermo <br /> glycophorin A. 10 mg glycophorin A was incubated with PBS (lane 1),          Scientific). <br /> 5 mg O-sialoglycoprotein endopeptidase (Gcp) of P. haemolytica <br /> (Cedarlane Laboratories, lane 2), 5 mg Gcp (lane 3), 25 mg Gcp (lane <br /> 4), 50 mg Gcp (lane 5), 33 mg YeaZ (lane 6) and 5 mg Gcp and 3.3 mg          In vitro pull down assay <br /> YeaZ (lane 7). (B) SDS-PAGE analysis of the impact of the recombinant          Purified GST-YeaZ fusion protein was mixed with Glutathi- <br /> staphylococcal YeaZ on Gcp.                                                  one Sepharose 4B resin at room temperature and allowed to <br /> doi:10.1371/journal.pone.0020163.g006                                        bind for 30 min. A total of 15 ml of resin bound GST-YeaZ <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                              May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                                 C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />  <br />  Table 1. Strains and plasmids used in this study. <br />  <br />  <br />  Strain or plasmid                       Description                                                                              Reference <br />  <br />  Strains <br />  RN4220                                  Laboratory strain; rsbU2                                                                 [39] <br />  WCUH29                                  Clinical human isolate; rsbU+                                                            [40] <br />  JRN0105                                 RN4220::Pspac-gcp, spac promoter regulated gcp expression mutant                         [5] <br />  JRN0110                                 JRN0105 with plasmid pXL108                                                              This study <br />  JRN0210                                 JRN0105 with plasmid pMY1107                                                             This study <br />  JRN0310                                 JRN0105 with plasmid pLT109                                                              This study <br />  JRN0410                                 JRN0105 with plasmid pLT209                                                              This study <br />  JRN0510                                 JRN0105 with plasmid pLT309                                                              This study <br />  JRN0610                                 JRN0105 with plasmid pLT409                                                              This study <br />  JRN0710                                 JRN0105 with plasmid pLT509                                                              This study <br />  PJ69-4A                                 MATa trp1-901 leu2-3, 112 ura3-52 his3-200 gal4Dgal80DLYS2::GAL1-HIS3 GAL2-ADE2          [33] <br />                                          met2::GAL7-lacZ <br />  Plasmids <br />  pGEX4T-1                                Overexpression vector, can be used to add GST tag to the N terminal of protein of interestGE Healthcare <br />  pET-24b                                 Overexpression vector, can be used to add His tag to the N or C terminal of protein of Novagen <br />                                          interest <br />  pGAD-C1/pGBD-C1                         Y2H vector carrying GAL4 transcription activation/DNA binding domain                     [33] <br />  pGAD/gcp                                pGAD-C1 carrying gcp                                                                     This study <br />  pGAD/gcpseg1                            pGAD-C1 carrying gcpseg1 mutant                                                          This study <br />  pGAD/gcpseg2                            pGAD-C1 carrying gcpseg2 mutant                                                          This study <br />  pGAD/gcpseg3                            pGAD-C1 carrying gcpseg3 mutant                                                          This study <br />  pGAD/gcpseg4                            pGAD-C1 carrying gcpseg4 mutant                                                          This study <br />  pGAD/gcpseg5                            pGAD-C1 carrying gcpseg5 mutant                                                          This study <br />  pGAD/gcpseg6                            pGAD-C1 carrying gcpseg6 mutant                                                          This study <br />  pGAD/gcpseg7                            pGAD-C1 carrying gcpseg7 mutant                                                          This study <br />  pGAD/gcpseg8                            pGAD-C1 carrying gcpseg8 mutant                                                          This study <br />  pGAD/gcpseg9                            pGAD-C1 carrying gcpseg9 mutant                                                          This study <br />  pGAD/gcp1-1                             pGAD-C1 carrying gcp1-1 mutant                                                           This study <br />  pGAD/gcp1-2                             pGAD-C1 carrying gcp1-2 mutant                                                           This study <br />  pGAD/gcp1-3                             pGAD-C1 carrying gcp1-3 mutant                                                           This study <br />  pGAD/sa1855                             pGAD-C1 carrying sa1855                                                                  This study <br />  pGAD/yeaZ                               pGAD-C1 carrying yeaZ                                                                    This study <br />  pGAD/sa1857                             pGAD-C1 carrying sa1857                                                                  This study <br />  pGBD/gcp                                pGBD-C1 carrying gcp                                                                     This study <br />  pGBD/sa1855                             pGBD-C1 carrying sa1855                                                                  This study <br />  pGBD/yeaZ                               pGBD-C1 carrying yeaZ                                                                    This study <br />  pGBD/sa1857                             pGBD-C1 carrying sa1857                                                                  This study <br />  pMY1107                                 pFF40 inserted with TetR regulation region                                               This study <br />  pXL108                                  pMY1107 carrying gcp                                                                     This study <br />  pLT109                                  pMY1107 carrying gcpseg1 mutant                                                          This study <br />  pLT209                                  pMY1107 carrying gcpseg2 mutant                                                          This study <br />  pLT309                                  pMY1107 carrying gcp1-1 mutant                                                           This study <br />  pLT409                                  pMY1107 carrying gcp1-2 mutant                                                           This study <br />  pLT509                                  pMY1107 carrying gcp1-3 mutant                                                           This study <br />  <br />  doi:10.1371/journal.pone.0020163.t001 <br />  <br />  <br /> protein was incubated with the molar equivalents of purified                     mixing with 26 the bead bed volume of elution buffer <br /> Gcp-His tag protein for 2 h at room temperature, washed 6                        containing 10 mM reduced glutathione. The elution fraction <br /> times with phosphate buffer solution using the equivalent                        was subjected to SDS-PAGE and standard western blotting <br /> volume of 100 times the bead bed volume, and eluted by                           using rabbit anti-Gcp serum. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            8                                  May 2011 | Volume 6 | Issue 5 | e20163 <br />                                                                                                                            C-Terminal Domain of Essential Protein Gcp <br />  <br />  <br />  <br />  Table 2. Primers used in this study.                                                      cII and Gcp1-3revSacII, and GcprevAscI with Gcp1-1forSacII, <br />                                                                                            Gcp1-2forSacII and Gcp1-3forSacII (Table 2). Three different 59 <br />                                                                                            end segments and 39 end segments of gcp were amplified by PCR, <br />  Name                   Sequence                                                           ligated to SacII site, and reformed mutated gcp1-1, gcp1-2, and gcp1- <br />                                                                                            3, respectively. These mutated genes encode three different C- <br />  P1for                  CTGATGAAGATTAGGGTTTGATGA <br />                                                                                            terminal alanine stretch mutants, Gcp317-322A, Gcp324-329A, <br />  P2for                  TTACAAGATGAATTACAAGGTGAAGTGA                                       and Gcp332-336A, accordingly. <br />  P3for                  TTATCGATCAAGCTCAAATTACAACAG. <br />  P4rev                  ACTAGAGCCTCGTTATTTGTTGTTGTTA                                       Construction of complementary plasmid and <br />  gcpforEcoRI            TTGAATTCATGACTAAAGATATATTAATACTAGC                                 characterization of bacteria growth <br />  gcprevBglII            TTAGATCTTTCTGCAGAATACTCTTCTA                                          In order to examine whether the expression of wild type or the <br />  gcpseg1RevBglII        GCAGATCTTTATAAATCTATATTGCTGTGCC                                    constructed alanine mutants of Gcp in trans are able to <br />                                                                                            complement the depletion of endogenous Gcp, we created a <br />  gcpseg2RevBglII        TTAGATCTTTACAAAGAGTGGCCGGCA <br />                                                                                            complementary plasmid using a shuttle vector, pFF40, which <br />  gcpseg3RevBglII        TTAGATCTTTAATTGACTTTGCATTGATCC <br />                                                                                            expresses the lacI gene [37]. A TetR regulatory cassette, obtained <br />  gcpseg4RevBglII        TTAGATCTTTAAGCAACAATTAATCGCTGA                                     by PCR, was inserted into the SacII site of pFF40 and the new <br />  gcpseg5RevBglII        TTAGATCTTTATTTAAACGTAAGCACCTCTAC                                   plasmid was designated as pMY1107. The wild type and alanine <br />  gcpseg6RevBglII        TTAGATCTTTATTGATTGTGAAGTTGATTGAT                                   mutant gcp genes were obtained by PCR using the primers listed in <br />  gcpseg7RevBglII        TTAGATCTTTAAAAATCATAACTATCTTTATCCAA <br />                                                                                            Table 2 and cloned into the PmeI and AscI sites of pMY1107, <br />                                                                                            resulting in plasmids pXL108, pLT109, pLT209, pLT309, <br />  gcpseg8RevBglII        TTAGATCTTTAAGCAGCCAACCGATCAA <br />                                                                                            pLT409, and pLT509. The recombinant plasmids were electro- <br />  gcpseg9RevBglII        TTAGATCTTTAAATAAGTGCAATTAGCGGG <br />                                                                                            phorated into the Pspac-regulated gcp S. aureus mutant [5]. The <br />  gcpfor1                AAACTATGACTAAAGATATATTAATACTAG                                     growth curves of transformed S. arueus were obtained using an <br />  gcprevAscI             TTGGCGCGCCTTCTGCAGAATACTCTTCTA                                     automated microtiter plate format on SpectrMaxPlus384 spectro- <br />  gcpseg1revAscI         TTGGCGCGCCTTATAAATCTATATTGCTGTGCC                                  photometer (Molecular Devices). S. aureus strains were incubated in <br />  gcpseg2revAscI         TTGGCGCGCCTTACAAAGAGTGGCCGGCA <br />                                                                                            TSB with appropriate antibiotics and different concentrations of <br />                                                                                            IPTG (0, 5, 20, 50 and 100 mM) at 37uC. Cell growth was <br />  gcp1-1forSacII         GCTGCCGCGGCTGCTGCTGCTGATTTAGCATTAAAT <br />                                                                                            monitored by measuring OD600 nm every 15 min with 1 min <br />  gcp1-1revSacII         AGCAGCAGCCGCGGCAGCCAAAGAGTGGCCGGCAAC <br />                                                                                            mixing before each reading. <br />  gcp1-2forSacII         GCTGCCGCGGCTGCTGGGCACAGCAATATAGAT <br />  gcp1-2revSacII         AGCAGCCGCGGCAGCAGCAAATCGACCTTGCTG                                  Enzymatic activity analysis <br />  gcp1-3forSacII         GCTGCCGCGGCTGCTGAAGAGTATTCTGCAGAA                                     To determine whether the staphylococcal Gcp and YeaZ <br />  gcp1-3revSacII         AGCAGCCGCGGCAGCGTGCCCATTTAATGCTAA                                  possess any O-sialoglycoprotein endopeptidase activity or protease <br />  gcpmutrevAscI          TTGGCGCGCCTTATTCTGCAGAATACTCTTC <br />                                                                                            activity, purified recombinant Gcp and YeaZ proteins were <br />                                                                                            utilized to treat O-sialoglycoprotein glycophorin A (Sigma) as <br />  gcppETforNdeI          GGAATTCCATATGACTAAAGATATATTAATACTAG <br />                                                                                            described [38]. The degradation of glycophorin A protein was <br />  gcppETrev              CCGCTCGAGTTCTGCAGAATACTCTTCTA                                      detected by SDS-PAGE and western blotting using a monoclonal <br />  yeaZforBamHI           TCGCGGATCCATGAACAAATTAAGGAGGCAAT                                   antibody against glycophorin A. To examine the effect of different <br />  yeaZrevSalI            CTCCGTCGACTTAATTGTTCTTTTGACTGTTGA                                  ions, including magnesium, calcium, and zinc on the potential <br />  sa1857BamHI            TCGCGGATCCTTGATAAAGATAAATAATTTAGATG                                enzymatic activity, the purified recombinant Gcp and/or YeaZ <br />  sa1857SalI             CTCCGTCGACTCAATGAGCAGCGAATTCATG <br />                                                                                            was incubated with glycophorin A in individual reaction buffers <br />                                                                                            containing different concentrations of Mg2+, Mn2+, Ca2+, or Zn2+ <br />  sa1855BamHI            TCGCGGATCCTTGGATCAACAGTCAAAAGAAC <br />                                                                                            at 37uC for 3 h. <br />  sa1855SalI             CTCCGTCGACTTAGTCATTTAAATTCACCCAC <br />  <br />  doi:10.1371/journal.pone.0020163.t002                                            <br /> Alanine scanning mutagenesis                                                            <br />   To identify the domains of Gcp important for binding to YeaZ, <br /> we performed alanine scanning mutagenesis assays. Five to six                              <br /> amino acids of alanine per stretch replaced the native amino acids                        <br /> and were generated by PCR using the primers listed in Table 2 as                         <br /> previously described [36]. In brief, six pairs of primers were                       <br /> designed, including Gcpfor1 with Gcp1-1revSacII, Gcp1-2revSa- <br /> </body></html>