<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>          Inactivation of YME1, a Member of the ftsH-SEC18-PAS1- <br />           CDC48 Family of Putative ATPase-Encoding Genes, Causes <br />                Increased Escape of DNA <span id='am-5' about='protege:FROM' typeof='owl:Thing'>from</span> Mitochondria in <br />                            Saccharomyces cerevisiae <br />  <br />              The yeast nuclear gene YMEJ was one of six genes recently identified in a screen for mutations that elevate <br />            the rate at which DNA escapes from mitochondria and migrates <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> the nucleus. ymel mutations, including a <br />            deletion, cause four known recessive phenotypes: an elevation in the rate at which copies of TRPI and ARSI, <br />            integrated into the mitochondrial genome, escape to the nucleus; a heat-sensitive respiratory-growth defect; a <br />            cold-sensitive growth defect on rich glucose medium; and synthetic lethality in rho- (cytoplasmic petite) cells. <br />            The cloned YMEI gene complements all of these phenotypes. The gene product, Ymelp, is immunologically <br />            detectable as an 82-kDa protein present in mitochondria. Ymelp is a member of a family of homologous <br />            putative ATPases, including Secl8p, Paslp, Cdc48p, TBP-1, and the FtsH protein. Ymelp is most similar to <br />            the Escherichia coli FtsH protein, an essential protein involved in septum formation during cell division. This <br />            observation suggests the hypothesis that Ymelp may play a role in mitochondrial fusion and/or division. <br />  <br />    The intracellular movement of DNA from mitochondria to                  generating respiration-competent (at temperatures of 300 or <br /> the nucleus can be experimentally detected in strains of                   below) petite-negative strains of S. cerevisiae. We also <br /> Saccharomyces cerevisiae that contain nuclear genetic                      describe here the isolation and nucleotide sequence of YME1 <br /> markers in their mitochondrial genomes (43, 44). This phe-                 and the detection of its product, Ymelp, in mitochondria. <br /> nomenon is of interest since it has probably played a role in              Ymelp has significant homology to a family of putative <br /> the transfer of genetic information from mitochondrial to                  ATPases whose members are involved in processes ranging <br /> nuclear genomes that has occurred during eucaryotic evolu-                 from organelle biogenesis to gene expression to cell division. <br /> tion (10, 13, 19, 25, 31, 47). Equally if not more importantly,            Ymelp appears to be most related to the Escherichia coli <br /> an examination of the mechanism by which DNA escapes                       FtsH protein, an essential protein involved in septum for- <br /> from mitochondria should provide a novel avenue for iden-                  mation during cell division (46). <br /> tifying and studying factors involved in maintaining the <br /> integrity of mitochondria during the dynamic processes of                               MATERIALS AND METHODS <br /> cell division and fusion. <br />    To study genetically the mechanism by which DNA es-                     Strains, strain constructions, and genetic methods. The E. <br /> capes mitochondria, we devised a screen for mutations that              coli strain used for preparation and manipulation of DNA <br /> elevate the rate at which escape occurs (44). A fragment of             was DH5a [F- endA41 hsdR17(r - iK+) supE44 thi-l X recA <br /> yeast nuclear DNA bearing the TRPJ gene and its associated              gyrA96 relAI A(argF-lacZYA) U169 +80 lacZAM15]. <br /> origin of nuclear DNA replication, ARS1, was integrated                    The genotypes of the S. cerevisiae strains used in this <br /> into a fully functional mitochondrial chromosome. Strains               work are listed in Table 1. Standard genetic techniques were <br /> containing the mitochondrially located TRP1 gene and a                  used to construct and analyze the various yeast strains (38). <br /> nuclear trpl deletion were phenotypically tryptophan auxo-                 Media. E. coli containing plasmids was grown in LB (10 g <br /> trophs (Trp-). However, such strains produced Trp+ deriv-               of Bacto Tryptone, 10 g of NaCl, and 5 g of yeast extract per <br /> atives which contained the TRPJ-ARSJ fragment and por-                  liter) plus 125 ,ug of ampicillin per ml. Yeast strains were <br /> tions of mitochondrial DNA (mtDNA) replicating as                       grown in complete glucose medium (YPD medium), com- <br /> plasmids in their nuclei. To identify mutations affecting the           plete ethanol-and-glycerol medium (YPEG), or minimal glu- <br /> rate of escape, we screened for elevated production of Trp+             cose medium plus the indicated nutrients (SD medium) (44). <br /> derivatives. Twenty-one recessive nuclear mutations that                Ampicillin and nutrients were obtained from Sigma. <br /> fell into six complementation groups, designated YME1                      Isolation of ymel-complementing plasmids. The tempera- <br /> through YME6 (for yeast mitochondrial escape), were ob-                 ture-sensitive respiratory-growth <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> caused by the <br /> tained.                                                                 ymel-I <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> was used as the basis for cloning YMEL. <br />    The YMEJ gene is of particular interest since ymel muta-             PTY62 was transformed (18) with 70 ,ug of DNA prepared <br /> tions cause a heat-sensitive defect in respiratory growth and           from a YCp50-based S. cerevisiae genomic DNA bank (34). <br /> a cold-sensitive defect in growth on fermentable carbon                 Twelve thousand Ura+ colonies grew on minimal SD me- <br /> sources (44). In addition, as reported here, ymel mutations             dium after incubation for 4 days at 30&#194;&#176;C. The transformants <br /> cause lethality in rho- (cytoplasmic petite) cells, thereby             were prewarmed at 37&#194;&#176;C for 1 h and then replica plated onto <br />                                                                         YPEG plates and incubated at 37&#194;&#176;C. Three transformants <br />                                                                         were able to grow at the restrictive condition in a plasmid- <br />   * <br />       Corresponding author.                                             dependent fashion. Total yeast DNA was prepared from <br />                                                                     5418 <br />  VOL. 13, 1993                                       ymel MUTATIONS INCREASE DNA ESCAPE FROM MITOCHONDRIA                            5419 <br />  <br />                                                   TABLE 1. Yeast strains used in this study <br />     Strain                                                       Genotype                                            Source or reference <br /> PTY33                            MATa ade2 ura3-52 leu2-3,112 trpl-Al [rho&apos; TRP1]                                      44 <br /> PTY33 xPTY44                     MATaad2 LYS2 ura3-52 leu2-3,112 til-Al [rho&apos; TRP1]                                    This study <br />                                  M4TaADE2 lys2 ura3-52 leu2-3,112 trpl-Al <br /> PTY44                            MATa Iys2 ura3-52 leu2-3,112 trpl-Al [rho&apos; TRP1]                                      44 <br /> PTY51                            ALTa ade2 ura3-52 leu2-3,112 trpl-AJ YMEI::URA3 [rho&apos; TRP1]                           This study <br /> PTY52                            AMTa Iys2 ura3-52 leu2-3,112 trpl-Al ymel-Al::URA3 [rho&apos; TRPI]                        This study <br /> PTY62                            MATa, lys2 ura3-52 leu2-3,112 trpl-Al ymel-l [rho&apos; TRPI]                              44 <br /> PTH78                            AMTa ade2-101 ura3-52 petl23::URA3                                                    16 <br /> LSF236                           MATa ade2 ura3-52 trpl-Al pet494-41                                                   L. S. Folley <br /> TF197                            ATcTa, leu2-3,112 ura3-52ymel-l [rho&apos; TRPI]                                           This study <br /> A236-57B                         MATa, leu2-3 trpl met4 aro7 his3 lysll SUC2 MAL3 canl                                 12 <br /> X12-6B                           MATa, radl-l ade2-1 gal2                                                              YGSCa <br />  a YGSC, Yeast Genetic Stock Center, Berkeley, Calif. <br />  <br />  <br /> these three transformants and used to transform E. coli. Two            generate a chimeric gene encoding a glutathione S-trans- <br /> of the rescued plasmids were identical (pPT31 in Fig. 2), and           ferase-Ymelp fusion protein. The resulting plasmid, pPT44, <br /> the third contained a smaller but overlapping insert (pPT32             was transformed into E. coli DH5a, and the fusion protein <br /> in Fig. 2).                                                             was induced and isolated as described previously (41). The <br />    Nucleic acid techniques and plasmid constructions. Restric-          fusion protein was used to immunize rabbits as previously <br /> tion enzymes and T4 DNA ligase were purchased from                      described (5). <br /> Bethesda Research Laboratories and New England Biolabs.                   Total protein extracts were prepared from YPEG-grown <br /> Standard techniques for generating recombinant DNAs and                 yeast cells. Cells were collected and washed once in 1 M <br /> performing DNA blot hybridizations were used (26).                      sorbitol-100 mM EDTA, pH 7.5. Cells were resuspended in <br />    Plasmids pPT31AX and pPT31AB (see Fig. 2) were con-                  0.05 times the original volume of the same buffer with 160 ,ug <br /> structed by digestion of pPT31 with XbaI and BamHI,                     of Zymolyase 20T (ICN Immunobiologicals, Inc.) per ml and <br /> respectively, followed by gel purification of the large restric-        incubated for 3 h at 37&#194;&#176;C. The spheroplasted cells were <br /> tion fragment and subsequent recircularization with T4 li- <br /> gase. Both pPT31AX and pPT31AB are YCp50-based CEN                      pelleted in 10 pellet volumes of 50 mM Tris-20 mM <br /> plasmids. Plasmids pPT34 and pPT35 are, respectively, Sall              EDTA-1% sodium dodecyl sulfate (SDS), pH 7.4, and <br /> and BamHI fragments of pPT32 that were gel purified and                 heated at 65&#194;&#176;C for 30 min. The protein extracts were then <br /> ligated into their corresponding sites in the polylinker of the         frozen at -80&#194;&#176;C until needed. A crude cell extract, purified <br /> URA3 CEN vector pRS316 (40). Plasmid pPT49 was con-                     mitochondria, and postmitochondrial supernatant were pre- <br /> structed by digesting pPT32 with MluI and EcoRI and then                pared from YPEG-grown PTY44 yeast cells essentially as <br /> purifying the 3-kb insert fragment and ligating it to the vector        described by Yaffe (50). Briefly, cells were spheroplasted by <br /> fragment created by digesting pPT34 with AMluI and EcoRI.               treatment with Zymolyase and broken in a Dounce homog- <br /> Five kilobase pairs of yeast DNA plus 378 bp of vector                  enizer. Unbroken cells and debris were removed by centrif- <br /> sequence contained in this plasmid was excised by digestion             ugation at 3,000 x g. A crude mitochondrial pellet was <br /> with XhoI and EcoRI and ligated into the same sites of                  generated by centrifuging the extract at 9,500 x g and <br /> pRS315, a LEU2 CEN vector (40), to create pPT49.                        collecting the pellet. The supernatant from this spin consti- <br />    The ymel-Al::URA3 mutation was constructed as fol-                   tuted the postmitochondrial supernatant. Mitochondria were <br /> lows. pPT42 DNA was digested with HpaI, removing 1,370                  washed by twice resuspending the mitochondrial pellet and <br /> nucleotides of internal YME1 sequence. pPT42 has the same               pelleting the mitochondria again at 9,500 x g. Mitochondria <br /> insert DNA as pPT49 in the corresponding sites in the                   were purified further by banding them in a Percoll density <br /> polylinker of pBluescript KS&apos;. In place of these nucleotides            gradient. <br /> was ligated a 1.5-kb SmaI fragment containing the URA3                     Protein fractions were subjected to SDS-polyacrylamide <br /> gene, generating pPT45 (see Fig. 2). Sixteen micrograms of              gel electrophoresis as described elsewhere (23). Protein- <br /> pPT45 DNA was digested with BglII and HindIII, and this                 containing fractions were separated on a 10% gel and elec- <br /> was used to transform the yeast diploid PTY33 xPTY44 to <br /> uracil prototrophy. A Ura+ transformant was sporulated,                 troblotted at 15 V for 45 min to 0.2-,um nitrocellulose <br /> and tetrads were dissected. A spore from one of those                   (Bio-Rad) by using a Bio-Rad Trans-Blot SD cell. The <br /> tetrads is PTY52.                                                       transfer buffer was 48 mM Tris-39 mM glycine, pH 9.2. <br />    The DNA sequence was determined on double-stranded                   After transfer, the filter was stained with Ponceau S in 5% <br /> templates by the nucleotide chain termination method (37).              acetic acid and subsequently destained with distilled water. <br /> Templates were generated from pPT34 and pPT35 by sub-                   The filter was blocked with 20 mM Tris-500 mM NaCl-3% <br /> cloning with restriction enzymes and exonuclease III-gener-             gelatin, pH 7.5. The filter was then washed in 20 mM <br /> ated deletions. Several gaps in the sequence were closed by             Tris-500 mM NaCl-0.05% Tween 20, pH 7.5. The rabbit <br /> having oligonucleotides synthesized and using them as prim-             antisera were diluted 1:200 in the wash solution containing <br /> ers.                                                                    1% gelatin and incubated with the filter for 1 h at 37&#194;&#176;C with <br />    Detection of Ymelp and mitochondrial fractionation. A                shaking. Immune complexes were identified by reaction with <br /> 1.5-kb HpaI-EcoRI fragment of DNA, encoding the car-                    a secondary antibody, purified goat anti-rabbit immunoglob- <br /> boxy-terminal 105 residues of Ymelp, was ligated into the               ulin G heavy-plus-light-chain gold conjugate (Bio-Rad), for <br /> glutathione S-transferase vector pGEX-3X (Pharmacia) to                 16 h at 37&#194;&#176;C with shaking. <br />  5420     THORSNESS ET AL.                                                                                               MOL. CELL. BIOL. <br />  <br />   Nucleotide sequence accession number. The sequence for            A      Vme I YM <br />                                                                                  -A I::IIRA i <br /> YME1 has been assigned <span id='am-3' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span>                                      l .1  I <br /> L14616. <br />                                                                   ymc 1 -Al::URA3                                                  YMF1 <br />                            RESULTS <br />                                                                          lVec^torl|                                                I YMII <br />    Phenotypes associated with YME] mutations. Among the <br />  six genes identified in our previous screen for mutations <br />  affecting the rate at which mtDNA escapes to the nucleus, <br /> YME1 was of particular interest since ymel mutations                                  ymc I-I                Evmc-i ] <br /> caused several phenotypes. In addition to having an in-                               I YMI 1                      IVectorl <br /> creased rate of mtDNA escape (Fig. 1A), ymel strains were <br />  also unable to grow on nonfermentable carbon sources at <br /> 37&#194;&#176;C and showed significantly decreased growth rates on <br /> rich medium containing glucose or galactose at 14&#194;&#176;C (Fig.            B                                  S&apos;&apos;ITAI N <br />  1B).                                                                     PLASMID)                YM <br />                                                                                                    N v1; <br />                                                                                                       I  n I <br />                                                                                                          mv      yrnC I --A I <br />    During the preliminary phenotypic characterization of                                                                :(1RA ,3 <br /> ymel mutants (44), we found that there was no elevation in                 Ve(tor <br /> the frequency of cytoplasmic petite, or rho-, mutants. <br /> Indeed, a more careful examination suggested that the <br /> ymel-1 mutation might actually prevent accumulation of                      YMFI <br /> rho- mutants: among 14,000 clones of a ymel-1 mutant <br /> strain, no rho- colonies were found, in contrast to an <br /> isogenic wild-type strain, in which approximately 1% of the                Vector <br /> cells were rho-. <br />    The incompatibility of the ymel-1 mutation with rho- <br /> mutations was confirmed in two ways. First, ymel-1 strains                  YMI I <br /> could not grow on medium containing a 25-,ug/ml concentra- <br /> tion of ethidium bromide (Fig. 1C), a drug that very effi- <br /> ciently induces rho- mutations (14). Second, the ymel-1 <br /> allele caused inviability in strains that also carried a null <br /> petl23 mutation, which is known to destabilize rho&apos; mtDNA           C ymel-AI::11l/A43 <br /> (16). The ymel mutant strain PTY62 was crossed with the                                                                     YMUIJ <br /> petl23::URA3 strain PTH78, and the resulting diploids were                       IYMliII         Ad                        Vec tor <br /> sporulated. Twenty tetrads were dissected on YPD medium <br /> and scored for growth, respiratory ability, and uracil pro- <br /> totrophy (to follow segregation of petl23::URA3). One- <br /> quarter of the spores did not yield viable haploids, although     ymclI-Al::1IRA.4 <br /> they germinated and formed microcolonies. In all cases the               lVectorl                                           I YMFI <br /> viable spores either were wild type or contained the ymel-1 <br /> orpetl23::URA3 mutation, but not both. <br />    The ymel-I mutation was not simply incompatible with <br /> unconditional respiratory defects. This was demonstrated by                           IVic/- I                     PI <br />  <br />                                                                                                                    viln I - 1 <br /> crossing a ymel-1 strain (PTY62) and a strain (LSF236)                                 YmJ]:-I                      Ves tI orI <br /> carrying a pet494 mutation, which prevents respiration but         FIG. 1. Complementation of ymel phenotypes with cloned yeast <br /> does not destabilize rho&apos; mtDNA (8, 29). The viability of       genomic DNA. A YMEI strain (PTY44), a ymel-l mutant (PTY62), <br /> spores from this cross was normal, and one-quarter of the       and a ymel-Al ::URA3 mutant (PTY52) were transformed with either <br /> progeny were double mutants. Thus, ymel-1 appears to be         the YMEl-carying plasmid pPT49 or the vector pRS315. The trans- <br /> specifically incompatible with rho- (cytoplasmic petite) mu-    formants, indicated with the plasmid in brackets when appropriate, <br /> tations and effectively produces respiration-competent (at      were analyzed as follows. (A) Detection of DNA escape from mito- <br /> temperatures of 30&#194;&#176;C or below) petite-negative strains of S.    chondria to the nucleus. Confluent sectors were grown on SD medium <br /> cerevisiae.                                                     supplemented with adenine, uracil, lysine, and tryptophan and replica <br />    Isolation of yeast genomic DNA that complements ymel         plated to SD medium supplemented with adenine, uracil, and lysine <br />                                                                 (the plate shown) to detect Trp+ colonies resulting from the escape of <br /> mutations. The yeast strain PTY62, bearing the ymel-I and       TRP1 from mitochondria. The plate was photographed after 5 days of <br /> ura3-52 mutations, was transformed with the YCp5O-based         incubation at 30&#194;&#176;C. (B) Respiratory growth at 37&#194;&#176;C and growth on rich <br /> genomic library of Rose et al. (34). About 12,000 transfor-     glucose medium at 14&#194;&#176;C. Cells patched onto YPD medium were printed <br /> mants were obtained, replica plated to YPEG, and incubated      to YPEG and incubated for 2 days at 37C (upper panel) and printed to <br /> at 37&#194;&#176;C. Three transformants were capable of growth on          YPD medium and incubated for 7 days at 14&#194;&#176;C (lower panel). (C) <br /> YPEG at 37&#194;&#176;C. Total DNA was prepared from these trans-          Growth on glucose in the presence of ethidium bromide. The cells were <br /> formants, and their plasmids were recovered by transforma-      streaked to SD medium containing 25 Fg of ethidium bromide per ml, <br /> tion of E. coli. Reintroduction of these plasmids into PTY62    adenine, uracil, lysine, leucine, and tryptophan and incubated at 30&apos;C <br /> demonstrated that they complemented the heat-sensitive          for 3 days. Cells were picked from isolated colonies (for strains <br />                                                                 containing YMEJ) or from the area of heaviest growth (for ymel <br /> respiration phenotype of ymel-1. Restriction analysis of the    strains), streaked to the same medium, and incubated at 30&#194;&#176;C for 4 days <br /> plasmids revealed that two of the recovered plasmids were       to give the growth shown. We have found it necessary to reculture <br /> identical (pPT31 in Fig. 2). The third plasmid, pPT32,          strains several times in the presence of ethidium bromide to ensure the <br /> contained a somewhat smaller insert from the same region of     generation of rho- or rhoo mitochondrial genomes. <br />                         <br />    FIG. 2. Restriction map, complementation analysis, and gene disruption of the YMEI region. The thick lines represent vector sequences, <br /> and the thinner lines represent yeast genomic sequences. The genomic sequences in the various subclones are aligned. The position and <br /> orientation of the YME1 open reading frame are indicated on plasmid pPT31 by the box and arrow, respectively. Plasmids pPT31, pPT32, <br /> pPT31AX, and pPT31AB are YCp5O-based plasmids. Plasmids pPT34 and pPT35 are pRS316 based, and pPT49 is pRS315 based. The insert <br /> in plasmid pPT45 is in pBluescript KS+. pPT45 was used in a one-step gene replacement at the YMEI locus. When the resulting locus was <br /> placed in trans to ymel-l in a diploid, it did not complement, so this plasmid has been indicated as noncomplementing. Restriction <br /> endonuclease cleavage sites: B, BamHI; C, ClaI; E, EcoRI; G, BglII; H, HindIII; Hp, HpaI; M, MluI; S, SatI; X, XbaI; Xh, XhoI. <br />  <br />  <br /> the genome. Subsequent subcloning and transformation of                    chain termination reactions were used to determine the <br /> PTY62 defined the complementing region of DNA more                         sequence of 2,839 nucleotides on both strands (Fig. 3). This <br /> accurately (Fig. 2). The subcloned segment of DNA in                       sequence has a single open reading frame that encodes a <br /> plasmid pPT49 complemented all four phenotypes associ-                     protein of 747 amino acids with a predicted relative molec- <br /> ated with the ymel-I mutation (Fig. 1).                                    ular weight of 81,679. The sequence of the predicted protein, <br />    Genetic mapping of the YMEI locus. A 3.5-kb Clal frag-                  Ymelp, was used to search sequence data bases. We found <br /> ment from pPT32, containing much of the YMEJ gene and                      significant sequence homology to a family of proteins that <br /> some 3&apos; sequence, was introduced into an integrating URA3 <br /> vector. That plasmid, pPT37, was linearized with the restric- <br /> tion endonuclease MluI and used to transform the wild-type <br /> strain PTY33. The Ura+ transformant, PTY51, was mated to                     TABLE 2. Tetrad analysis of linkage between markers in the <br /> the ymel-1 strain TF197, and the resulting diploid was                                  YMEI region of chromosome 16a <br /> sporulated. There were no recombinants among 18 tetrads                                                  Ascus type                 Segregation <br /> scored for uracil prototrophy and heat-sensitive respiratory                                             (no. of asci)              (no. of asci)&apos;        Distance <br /> growth, indicating that the cloned chromosomal DNA cor-                          Interval                                                                  (cM) <br /> responded to the YMEI locus.                                                                        PD      NPD            T        FD           SD <br />    The plasmid pPT35 was used to probe a chromosome blot                   ymel-aroT                33        2            45                               35 <br /> of S. cerevisiae DNA. The probe was found to hybridize to                  ymel-CEN16C                                              53           24         16 <br /> the filter only at the position of chromosome 16 (data not                 ymel-radld            21       5       37                                        53 <br /> shown). Meiotic-mapping data for YME1, from tetrad anal-                     a PD, parental ditype; NPD, nonparental ditype; T, tetratype; FD, first <br /> ysis, is summarized in Table 2. YMEJ is located on the right               division; SD, second division; cM, centimorgans. <br /> arm of chromosome 16 about 16 centimorgans from the                          b Scored relative to the centromere-linked marker trpl. <br /> centromere and about 35 centimorgans from aro7.                              c From the cross TF197 x A236-57B. <br />    Primary sequence analysis of YMEI. Dideoxynucleotide                      d   From the cross TF197 x X12-6B. <br />  5422    THORSNESS ET AL.                                                                                      MOL. CELL. BIOL. <br />  <br />          TGGTACCTCTTTCTTCGAGGAGTGG TAGCTCCTGTACGCTGTAGCGGTAG TGAAAAGGCAATATATCGGAGTTGG TGAAGTGAGTTAAATCTCTACCTTG   100 <br />           CGITITTGATAAGTAAAAAGCAAGC ACAGCTTAAAGGATAAACAGGATTT TATTGAAGCAATITMAATTATAAT ACATTGTGGATAGAACGAAAACAGA   200 <br />           GACGTGATAGATGAACGTlICAAAA ATACTTGTGTCGCCCACGGTCACGA CAAATGTIITACGCATATTTGCTCC CAGGCTACCTCAAATCGGTGCIICT  300 <br />                      M N V S K I L V S P T V T T N V L R I F A P R L P Q I G A S <br />          TTGTTAGTTCAAAAAAAATGGGCCT TAAGATCAAAGAAGITCTACCGTTT TTATTCTGAAAAGAATAGCGGTGAA ATGCCTCCTAAGAAGGAAGCTGATA   400 <br />           L L V Q K K W A L R S K K F Y R F Y S E K N S G E M P P K K E A D S <br />           GCTCTGGAAAGGCATCCAATAAATC CACGATATCTTCAATCGACAATTCG CAACCACCACCTCCATCGAACACTA ATGATAAAACCAAACAAGCGAACGT  500 <br />             S G K A S N K S T I S S I D N S Q P P P P S N T N D K T K Q A N V <br />          AGCTGTGTCACATGCTATGCTAGCA ACTAGAGAACAAGAGGCCAATAAAG ACTTAACGAGTCCTGATGCACAAGC AGCCTTTTACAAAC1TCTCTTACAA   600 <br />            A V S H A M L A T R E Q E A N K D L T S P D A Q A A F Y K L L L Q <br />           TCAAACTACCCGCAATACGTGGTCT CTAGGTTTGAGACCCCCGGTATTGC GTCATCGCCTGAATGCATGGAACTG TACATGGAGGCCCTGCAGAGGATAG  700 <br />           S N Y P Q Y V V S R F E T P G I A S S P E C M E L Y M E A L Q R I G <br />           GCAGACACTCGGAGGCTGATGCCGT TAGACAAAACCTACTGACAGCCAGC TCTGCTGGCGCAGTTAACCCATCAT TGGCGTCATCTTCATCAAACCAGTC  800 <br />             R H S E A D A V R Q N L L T A S S A G A V N P S L A S S S S N Q S <br />          AGGTTATCATGGTAACTTTCCATCG ATGTATTCTCCGCTTTATGGATCTC GCAAAGAGCCACTACATGTTGTCGT ATCTGAATCAACTIIIACTGTTGTA   900 <br />            G Y H G N F P S M Y S P L Y G S R K E P L H V V V S E S T F T V V <br />           TCGAGATGGGTAAAGTGGCTGCTTG TTTTCGGTATCTTAACCTACTCTTT TTCTGAAGGTTITAAATACATCACA GAAAATACAACGCTACTAAAGTCGT 1000 <br />           S R W V K W L L V F G I L T Y S F S E G F K Y I T E N T T L L K S S <br />           CAGAAGTAGCCGACAAATCAGTTGA TGTAGCTAAGACAAATGTTAAATTT GATGATGTCTGCGGTTGTGATGAGG CCCGTGCGGAATTGGAAGAAATTGT 1100 <br />             E V A D K S V D V A K T N V K F D D V C G C D E A R A E L E E I V <br />           TGATTTCCTGAAAGATCCAACTAAG TACGAGTCCIIGGGTGGTAAACTAC CAAAGGGTGTACTGTTGACTGGACC TCCTGGTACAGGTAAAACTTTGTTG 1200 <br />            D F L K D P T K Y E S L G G K L P K G V L L T G P P G T G K T L L <br />           GCTAGGGCCACTGCCGGAGAGGCTG GTGTAGATFITTTCTTTATGTCAGG TTCTGAATTTGATGAAGTTTACGTC GGTGTTGGTGCTAAACGTATCCGTG 1300 <br />          A R A T A G E A G V D F F F M S G S E F D E V Y V G V G A K R I R D <br />          ATTTGTTTGCTCAAGCGCGTTCTCG TGCACCAGCTATTAIITFMATTGAT GAATTAGATGCCATCGGTGGTAAGC GTAATCCAAAGGACCAAGCTTACGC 1400 <br />             L F A Q A R S R A P A I I F I D E L D A I G G K R N P K D Q A Y A <br />           CAAACAAACGTTGAATCAGTTATTG GTCGAATTAGATGGTTTCTCACAAA CAAGTGGAATTATTATTATTGGTGC CACAAATTTCCCTGAGGCTTTAGAT 1500 <br />            K Q T L N Q L L V E L D G F S Q T S G I I I I G A T N F P E A L D <br />           AAGGCGTTAACTAGACCAGGTAGAT TTGATAAGGTTGTGAATGTGGAITT GCCAGATGTTCGTGGTCGTGCTGAT ATTITAAAGCATCACATGAAGAAGA 1600 <br />           K A L T R P G R F D K V V N V D L P D V R G R A D I L K H H M K K I <br />          TTACATTGGCAGACAATGTGGATCC AACTATTATTGCGCGTGGTACCCCC GGTTTATCAGGCGCTGAGCTGGCAA ATTTAGTCAACCAAGCAGCAGTTTA 1700 <br />             T L A D N V D P T I I A R G T P G L S G A E L A N L V N Q A A V Y <br />           CGCGTGTCAAAAAAATGCTGITTCC GTTGATATGTCCCACTTCGAGTGGG CTAAGGATAAGATATTGATGGGTGC TGAGAGAAAGACTATGGTCTTAACA 1800 <br />            A C Q K N A V S V D M S H F E W A K D K I L M G A E R K T M V L T <br />           GATGCAGCCAGAAAGGCCACTGCTT TCCACGAGGCTGGACATGCCATTAT GGCCAAATACACCAATGGTGCTACC CCGCTATACAAGGCCACGATATTGC 1900 <br />           D A A R K A T A F H E A G H A I M A K Y T N G A T P L Y K A T I L P <br />           CTAGAGGTAGGGCATTGGGTATTAC TTTTCAATTGCCAGAAATGGATAAG GTCGACATCACCAAAAGGGAGTGTC AAGCCAGACTGGACGTGTGCATGGG 2000 <br />             R G R A L G I T F Q L P E M D K V D I T K R E C Q A R L D V C M G <br />           GGGCAAAATTGCAGAAGAATTAATT TATGGTAAAGATAACACCACAAGTG GTTGTGGGTCTGACTTGCAAAGCGC CACCGGCACAGCAAGGGCTATGGTT 2100 <br />            G K I A E E L I Y G K D N T T S G C G S D L Q S A T G T A R A M V <br />           ACTCAATATGGTATGAGTGATGATG TAGGTCCCGTTAACTTGTCAGAAAA TTGGGAATCTTGGTCTAATAAGATT CGCGATATTGCTGATAATGAAGTGA 2200 <br />           T Q Y G M S D D V G P V N L S E N W E S W S N K I R D I A D N E V I <br />           TTGAACTTITGAAGGACTCCGAGGA AAGAGCAAGAAGACTATTAACTAAG AAAAATGTTGAGCTACATAGACTTG CGCAAGGTCIIATTGAATATGAAAC 2300 <br />             E L L K D S E E R A R R L L T K K N V E L H R L A Q G L I E Y E T <br />           TCTAGATGCCCACGAAATCGAACAA GTTTGTAAAGGTGAAAAACTGGACA AACTGAAAACITCCACCAATACAGT CGTAGAAGGACCAGACAGTGATGAA 2400 <br />            L D A H E I E Q V C K G E K L D K L K T S T N T V V E G P D S D E <br />           CGTAAAGATATAGGCGATGATAAAC CCAAAATTCCTACAATGTTAAATGC ATGA1TTTATTCTAAGAAG1IAAAC GTATGAAGGAACCTACCTCAAGACC 2500 <br />             R K D I G D D K P K I P T M L N A <br />           GAGAATGCTTTGTTCTT17CTGGTC GGTTTCGCTTTTGCTTCATAACACC AGCAATGAAGCATGATCGCGGGAAA CITTCAGAATCTACATCTTCTCCTG 2600 <br />           GTCAGCTTGAAACTTCAGATTATTC GTAAGGAGAAATTATAATATTATTT CTGTGATAAGAAAAGAGGCATTAAG CTTGGGTATTTGCGACAAAATACTT 2700 <br />           CAGGATAACCGATGATCGCAAGTGG GTTATTAATTTCTTTCATCCCTTAA CCAACCGTCGCTT7CCCCCAGTGAC AAGCCTTAAAAAAGTCACACCATGG 2800 <br />           AACGGAGTATCGTTGATGATTAAAC CAATCATTAATCTTTCATAAAAGTT AAGATAAAAGCACTTGTTGATATAT ATAGGTACATGTGTATCAATACGCC 2900 <br />           ATACATTGTAAATAGAATATTCAAT GGATAAACGTGTGTAGCAAAAATAA TCACATTGTAGCGCTAGTTAG                               2971 <br />   FIG. 3. Nucleotide sequence of YME1 and predicted amino acid sequence of Ymelp. The sequence of the 2,971 bp was determined by <br /> sequencing both the coding and anticoding strands. <br />  <br />  <br /> are putative ATPases, one of which (32) has been biochem-         shorter proteins: Suglp (42) and Afglp (24) from yeast cells <br /> ically demonstrated to have ATPase activity. Ymelp is most        and the HIV-Tat-associated proteins TBP-1 (30) and Mssl <br /> closely related to the FtsH protein from E. coli (46). The        (39) from mammalian cells. <br /> FtsH protein is apparently involved in the posttranslational        A comparison of the sequences of Ymelp and the FtsH <br /> processing of PBP 3, a protein necessary for septation during     protein is presented in Fig. 4. Over their full lengths, Ymelp <br /> cell division in E. coli (46). Ymelp is also related to the       and the FtsH protein share 30% amino acid identity, and an <br /> 84-kDa yeast protein Secl8p, which is required for several        additional 3% of the residues are conserved substitutions. <br /> membrane fusion events involving the endoplasmic reticu-          The amino-terminal portions of these proteins have no <br /> lum, vesicles, and the Golgi apparatus in the secretory           significant homology. The amino terminus of the FtsH <br /> pathway (7). The mammalian homolog of Secl8p is NSF, the          protein contains two hydrophobic stretches that span the <br /> N-ethylmaleimide-sensitive fusion protein (49). Two other         cytoplasmic membrane twice, leaving the bulk of the protein <br /> large yeast proteins in this family are Paslp, which is           in the cytoplasm (45). There are no obvious sequence <br /> involved in peroxisome assembly (9), and Cdc48p, which            elements in Ymelp that would suggest association of the <br /> appears to be involved in nuclear division (11). Two Cdc48p       protein with membranes. From residue 272 of Ymelp and <br /> homologs, VCP (21) and p97-ATPase (32), have been found           residue 145 of the FtsH protein to their respective C termini, <br /> in higher eucaryotes. Ymelp is also homologous to several         these proteins are 46% identical. These sequences include <br />  VOL. 13, 1993                                        ymel MUTATIONS INCREASE DNA ESCAPE FROM MITOCHONDRIA                             5423 <br />  <br />   Ymelp      MNVSRILVSP TVTTNVLRIF APRLPQIGAS LLVQKRWALR SRRFYRFYSE RNSGEMPPRK                             EADSSGKASN RSTIS          75 <br />  <br />   Ymelp      SIDNSQPPPP SNTNDRTRQA NVAVSHAMLA TREQEANKDL TSPDAQAAFY RLLLQSNYPQ YVVSRFETPG IASSP 150 <br />   FtsHp                                                                                      MALILVIA                SV FSFG         23 <br />  <br />   Ymelp      ECMELYMEAL QRIGRHSEAD AVRQNLLTAS SAGAVNPSLA SSSSNQSGYH GNFPSMYSPL YGSRREPLHV VVSES 225 <br />   FtsHp      PSESNGRKVD YSTFLQEVNN DQVREARING REINVTRRDS NRYTTYIPVQ DPRLLDNLLT RNVRVVGEPP        98                       EBEU <br />   Ymelp      TFTVVSRWVK WLLVFGILTY SFSEGFRYIT ENTTLLKSSE VADKS                                             CDE                     300 <br />   FtsHp      LASIFISWFP MLLLIGVWIF FMRQNQGGGG RGAMSFGKSR ARMLT                                             CD                      171 <br />  <br />   Ymelp      K      KYE G G        K        T    P        L                                                               S        375 <br />   FtsHp      R eRFQ1G GP                             TE                     F           F          GV                              246 <br />  <br />   Ymelp         t    I                P---R DQAY QTLN         LV           QT I447 <br />   FtsHp             ID                AGLGG GHDE QTLN          V           GN GI I 321 <br />  <br />   Ymelp         DVHRI   IAR GTP SGDKN5V <br />                                   Y Q V F D22 <br />   FtsHp         |DVRG!D    D <br />                            V     DNR <br />                                   F     F DK 396 <br />  <br />   Ymelp                                      GHA YTN GAR L                                              VD ITRRECQARL DVCi597 <br />   FtsHp                        KE t         EAGA RVPH   PRGRAL                                 P         IS ASRQKLESQI STL          471 <br />  <br />   Ymelp                  IYG     DNTT*4GS FQ                               D        LI ----WES                  ERD----- -i         660 <br />   FtSHp                  IYG     EHVS SN HR                                 I       LY3 EGEVFLG             jKHMSDET                546 <br />  <br />    Ymelp         KEI4RDS E              LRNVELAQ                   YE            QV--CR     GEKLD---RL RTS   EG PDSDE 730 <br />    FtsHp                 ERN Y          LNM   RD                   YE            DLMARR     DVRPPAGWEE PGA tSGDN GSPKA 621 <br />  <br />    Ymelp     RRDIGDDRPRK                -       NA <br />                                                  N                                                                                  747 <br />    FtsHp        PRPVDEPRTP     143E1TEDR                                                                                            644 <br />   FIG. 4. Amino acid sequence comparison of Ymelp and the FtsH protein (FtsHp). The sequences were aligned with the aid of the <br /> alignment program of GeneWorks 2.2 from IntelliGenetics. Identical amino acids are boxed, the nucleotide-binding motifs (3) are indicated <br /> by lines above the sequence elements, and the membrane-spanning sequences of the FtsH protein (45, 46) are underlined. No significant level <br /> of homology between these two proteins is evident in the amino-terminal portions. Consequently, the boxed identities and gaps introduced <br /> by the alignment program prior to residue 272 for Ymelp and residue 145 for the FtsH protein have been eliminated in this alignment. <br />  <br />  <br /> the region of homology shared among all members of this                 analysis (not shown) to verify that the Ura+ spores con- <br /> family of putative ATPases. As has been previously noted,               tained the expected mutation, termed ymel-Al::URA3. In <br /> this region contains two elements believed to be important              each case, the Ura+ spores exhibited the same phenotypes <br /> for ATP binding (underlined sequences in Fig. 4) (3), but it            caused by the ymel-I mutation (Fig. 1): a high rate of DNA <br /> also contains homologous sequence elements in addition to               escape from mitochondria to the nucleus, temperature- <br /> those assigned to ATP-binding or ATPase activity. The                   sensitive respiratory growth at 37&#194;&#176;C, slow growth on rich <br /> duplication of the conserved nucleotide-binding region noted            glucose medium at 14&#194;&#176;C, and lethality when combined with <br /> for Paslp, Secl8p, and Cdc48p (9, 11) is not evident in either          rho- mtDNA. <br /> Ymelp or the FtsH protein.                                                The ymel-Al::URA3 mutation deleted over 60% of the <br />    Generation and phenotypic consequences of a ymel null                structural gene, including all of the highly conserved se- <br /> mutation. To destroy the function of YMEI, 1,370 nucle-                 quence associated with ATP binding and hydrolysis, sug- <br /> otides internal to the reading frame were deleted and re-               gesting that it is a null mutation. If this is the case, then <br /> placed by the URA3 gene. The resulting DNA fragment                     Ymelp is dispensable for growth and respiration at 30&#194;&#176;C, <br /> (pPT45; Fig. 2) carried URA3 flanked by approximately 2 kb              although its absence results in high rates of DNA escape <br /> of 5&apos; YMEI sequence, including the presumptive YMEJ                     from mitochondria. <br /> promoter and the first 185 codons, and approximately 1.5 kb                Six independent alleles of ymel were isolated in the <br /> of 3&apos; YMEJ sequence which included the C-terminal 105                   original mutant screen (44), and the phenotypes caused by all <br /> codons (Fig. 2). This DNA, in linear form, was used to                  are essentially the same as those caused by the ymel -Al:: <br /> transform the diploid strain PTY33xPTY44 to Ura+, delet-                URA3 mutation. The ymel-l mutation appears to be a <br /> ing one copy of YMEJ (36). A Ura+ transformant was                      <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>missense mutation</span> since full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> Ymelp can be detected <br /> purified and sporulated. The resulting tetrads each contained           in aymel-l strain by antibodies (see below) directed against <br /> four viable spores, two of which were Ura+ and two of                   Ymelp. It is interesting thatymel-l exhibits weak intragenic <br /> which were Ura-. Total DNA was prepared from each spore                 complementation with the ymel-3 allele, suggesting that <br /> of one tetrad and subjected to DNA blot hybridization                   Ymelp may function in a multimeric complex. <br />  <br />    FIG. 5. Immunological detection of Ymelp in yeast extracts with <br /> anti-Ymelp antisera. Lanes: P, preimmune serum probing of wild- <br /> type whole-cell extract; 1, PTY52 (ymel-A1::URA3) whole-cell <br /> extract; 2, PTY44 (wild type) whole-cell extract; 3, PTY52/pRS315 <br /> (vector) whole-cell extract; 4, PTY52/pPT49 (YME1 plasmid) whole- <br /> cell extract; 5, PTY44/pRS202 (multicopy vector) whole-cell ex- <br />  <br />                                                                                                      mitochondria. <br />                                                                                                                                                  MOL. CELL. BIOL. <br />  <br />                                                                                                      (44). Here, we have shown that this phenotype, as well as <br />                                                                                                      others associated withymel mutations, is caused by the loss <br />                                                                                                      of the protein product, Ymelp, which is normally located in <br />                                                                                                         In addition to increasing the rate of DNA escape from <br />                                                                                                      mitochondria, Ymelp deficiency causes two other pheno- <br />                                                                                                      types that are clearly related to mitochondrial function. <br />                                                                                                      First, ymel mutants are unable to grow on nonfermentable <br />                                                                                                      carbon sources at 37&#194;&#176;C. The second phenotype is lethality of <br />                                                                                                      cells carrying both a ymel nuclear mutation and rho- <br />                                                                                                      (cytoplasmic petite) deletion mutations affecting the mito- <br />                                                                                                      chondrial genome. All three of these phenotypes, caused by <br />                                                                                                      the loss of Ymelp, could be the result of decreased integrity <br />                                                                                                      (increased leakiness) of the mitochondrial compartment. <br />                                                                                                         rho- cells cannot carry out any mitochondrial protein <br />                                                                                                      synthesis. Therefore, their mitochondrial inner membranes <br />                                                                                                      lack several major protein components (reviewed in refer- <br />                                                                                                      ences 6 and 33), a condition likely to result in decreased <br /> tract; 6, PTY44/pPT48 (YME1 in multicopy vector) whole-cell                                          mitochondrial integrity. Indeed, we have observed that <br /> extract; 7, crude cell extract; 8, postmitochondrial supernatant; 9,                                 DNA escapes from rho- mitochondria at a 30-fold-higher <br /> purified mitochondrial pellet; 10, Percoll gradient-purified mitochon-                               rate than from rho&apos; organelles (44). Intact mitochondrial <br /> dria. The crude cell extract in lane 7, the postmitochondrial super-                                 compartments are required for viability in S. cerevisiae (1). <br /> natant in lane 8, and the mitochondrial pellet in lane 9 were prepared                               Thus, one interpretation of the synthetic lethality of rho- <br /> such that the concentrations of both soluble protein and protein                                     ymel double mutants is that the combined absence of all <br /> associated with mitochondria remained constant, and equal volumes                                    mitochondrial translation products as well as Ymelp reduces <br /> were loaded on the polyacrylamide gel.                                                               mitochondrial integrity below a threshold necessary for <br />                                                                                                      viability. <br />                                                                                                         Ymelp exhibits significant homology to a family of pro- <br />   Ymelp is associated with mitochondria. To generate an                                              teins that contain sequence elements implicated in nucle- <br /> antibody against Ymelp, DNA encoding the C-terminal 105                                              otide binding. One member of this protein family, mamma- <br /> amino acids of Ymelp was fused in frame to the 3&apos; end of the                                         lian p97-ATPase, has been shown biochemically to have <br /> glutathione S-transferase-encoding gene (41), and the result-                                        ATPase activity (32). Ymelp is most homologous to a <br /> ing fusion protein was isolated and injected into rabbits. The                                       protein from E. coli, encoded by the ftsH gene, which is <br /> antisera were first used to probe immunoblots of total                                               involved in formation of the septum during cell division (46). <br /> protein extracts from the wild-type yeast strain PTY44 and                                           The next-most-homologous protein to Ymelp is the yeast <br /> from the ymel-Al::URA3 strain PTY52 that had been elec-                                              protein Secl8p (19% identical to Ymelp). Secl8p and the <br /> trophoresed on a 10% polyacrylamide-SDS gel. A protein of                                            corresponding mammalian protein, NSF, are required for <br /> the expected size of 82 kDa was detected by immunogold                                               membrane fusion events involving the endoplasmic reticu- <br /> labeling in the wild-type extract but was absent in the                                              lum, vesicles, and the Golgi apparatus in the secretory <br /> ymel-Al::URA3 strain (Fig. 5). The 82-kDa protein was                                                pathway (7, 49). NSF is assembled into a membrane-associ- <br /> restored in the ymel-Al::URA3 strain by introducing a                                                ated &quot;fusion machine&quot; containing several other components <br /> plasmid expressing the YME1 gene. Thus, the antisera raised                                          (4; reviewed in references 35 and 48). <br /> against the fusion protein detect Ymelp in crude extracts of                                            The two other large yeast proteins in this family, Paslp <br /> S. cerevisiae. It is interesting that when the YMEI gene is                                          and Cdc48p, are less closely related to Ymelp. While their <br /> carried by a multicopy plasmid, there is apparently no                                               functions are less well defined than those of Secl8p, they <br /> increase in the amount of Ymelp present in the cell.                                                 also appear to be involved in organellar assembly and/or <br />    Because ymel mutations cause phenotypes associated                                                rearrangement. Paslp is required for the assembly of perox- <br /> with mitochondria, it seemed likely that Ymelp might be a                                            isomes (9). Cold-sensitive mutations affecting Cdc48p block <br /> mitochondrial protein. To test this idea, crude extracts of                                          spindle pole body duplication and nuclear division (11). <br /> wild-type yeast cells were fractionated into mitochondrial                                              The homology between Ymelp and the FtsH protein, and <br /> pellets and postmitochondrial supernatants. These fractions                                          to a lesser extent Secl8p, suggests the hypothesis that <br /> were electrophoresed and blotted as before and then probed                                           Ymelp could be involved in promoting efficient mitochon- <br /> with the antisera. Ymelp was found to be depleted from the                                           drial fusion and/or division. This hypothesis could easily <br /> postmitochondrial fraction and enriched in the mitochondrial                                         account for the three mitochondrion-related phenotypes of <br /> fraction (Fig. 5). To confirm the mitochondrial location of                                          ymel mutants described above if, in the absence of Ymelp, <br /> Ymelp, the mitochondrial pellet was resuspended and sub-                                             organellar fusion and/or division structures were not tightly <br /> jected to buoyant density centrifugation in a Percoll gradi-                                         sealed. For example, breaches in mitochondrial membranes <br /> ent. These gradient-purified mitochondria still contained                                            during aberrant fusion or division events could allow DNA <br /> Ymelp (Fig. 5).                                                                                      to leak into the cytoplasm. The fourth known phenotype of <br />                                                                                                      ymel mutants is cold-sensitive growth on rich glucose- <br />                                              DISCUSSION                                              containing medium (Fig. 1) but not on medium containing <br />                                                                                                      nonfermentable carbon sources (44). This too could be due <br />   Mutations in the gene YME1 were isolated by screening                                              to a defect in mitochondrial division if one assumed that at <br /> for respiration-competent cells at 30&#194;&#176;C that exhibited an                                            low temperatures division of mitochondria lacking Ymelp <br /> increased rate (25 times higher than the wild-type rate for                                          was too slow to keep pace with rapid cell growth on glucose <br /> ymel-1) of DNA escape from mitochondria to the nucleus                                               and that no checkpoint function held up cell division for <br />  <br />  <br /> completion of mitochondrial division (17). A total block of                <br /> mitochondrial division would be expected to produce a lethal               <br /> phenotype (27, 28, 51).                                                    <br />    Whatever role Ymelp may play in mitochondrial function,         <br /> the polypeptide appears to be part of a multimeric complex.        <br />                                                                     <br /> We observed intragenic complementation between the ap-               <br /> parent missense allele ymel-1 and the uncharacterized               <br /> ymel-3 mutation, suggesting that the functional <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> contains              <br /> more than one copy of Ymelp. In addition, an elevated gene         <br /> dosage of YME1 did not cause overaccumulation of the               <br /> protein, suggesting that the level of a complex containing       <br /> Ymelp is limited by the level of some other component and               <br /> that unassembled Ymelp may be degraded. Our previous                   <br /> observation that aymel yme2 double mutant had a synthetic          <br /> nonrespiratory phenotype (44) suggests the possibility that               <br /> the YME2 product could be part of such a complex.                           <br />    A novel consequence of ymel mutations is that they                       <br /> convert the petite-positive yeast S. cerevisiae into respiring          <br /> (at permissive temperatures) petite-negative strains by pre-             <br /> venting the growth of rho- cells. This property will be useful              <br /> for many genetic screens and selections in which the very             <br /> frequent rho- mutations represent background that can now                   <br /> be easily eliminated. The opi (pet9) mutation also kills rho-            <br /> cells. However, this mutation, which inactivates a major               <br /> mitochondrial ADP-ATP translocator, also prevents respira-                <br /> tory growth (20, 22). Interestingly, other recent evidence                  <br /> also indicates that the differences between petite-positive                 <br /> and petite-negative yeasts may not be very great. Nuclear                <br /> mutations have been selected in the petite-negative yeasts                  <br /> Schizosaccharomyces pombe (15) and Kluyveromyces lactis                     <br /> (2) that allow them to tolerate the loss of mtDNA.                    <br />   <br /> </body></html>