<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Chavali et al. BMC Systems Biology 2012, 6:27 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                                Open Access <br />  <br /> Metabolic network analysis predicts efficacy of <br /> FDA-approved drugs targeting causative <br /> agent neglected tropical disease <br /> Arvind K Chavali1, Anna S Blazier1, Jose L Tlaxca1, Paul A Jensen1, Richard D Pearson2,3 Jason A Papin1,4* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Systems biology holds promise new approach drug target identification drug discovery <br />   neglected tropical diseases. Genome-scale metabolic reconstructions, assembled annotated genomes <br />   vast array bioinformatics/biochemical resources, provide framework interrogation human <br />   pathogens serve platform generation future experimental hypotheses. In article, the <br />   application selection criteria Leishmania major targets (e.g. silico gene lethality) drugs (e.g. <br />   toxicity), method (MetDP) rationally focus subset low-toxic Food Drug Administration (FDA)- <br />   approved drugs introduced. <br />   Results: This metabolic network-driven approach identified 15 L. major genes high-priority targets, 8 high- <br />   priority synthetic lethal targets, 254 FDA-approved drugs. Results compared previous literature findings <br />   existing high-throughput screens. Halofantrine, antimalarial agent prioritized using MetDP, showed <br />   noticeable antileishmanial activity experimentally evaluated vitro L. major promastigotes. <br />   Furthermore, synthetic lethality predictions aided prediction superadditive drug combinations. For <br />   proof-of-concept, double-drug combinations evaluated vitro L. major combinations <br />   involving drug disulfiram showed superadditivity presented. <br />   Conclusions: A direct metabolic network-driven method incorporates single gene essentiality synthetic <br />   lethality predictions proposed generates set high-priority L. major targets, turn associated <br />   select number FDA-approved drugs candidate antileishmanials. Additionally, selection high- <br />   priority double-drug combinations provide attractive alternative avenue drug discovery <br />   leishmaniasis. <br />  <br />  <br /> Background                                                                             Caused Leishmania species, disease manifest <br /> Over billion people infected                                    itself varying clinical pathologies including visceral, cuta- <br /> neglected tropical diseases (NTDs) [1,2]. These diseases                               neous mucocutaneous forms [3,4]. These parasites tran- <br /> comprise group parasitic bacterial infections                            sition different morphological stages - <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> affect poorest marginalized popula-                               flagellated promastigotes female phlebotomine <br /> tions world, live 1.25                                sandfly (vector) gut non-flagellated amastigotes host <br /> USD day [2]. Leishmaniasis NTD ende-                             macrophages [5]. Leishmaniasis NTDs tend be <br /> mic 88 countries total 350 million people                              overshadowed research focused malaria, <br /> risk. The disease associated global prevalence                            tuberculosis HIV/AIDS [6]. There lack of <br /> 12 million cases, yearly incidence 1.5 2 million cases,                        pharmaceutical companies pursuing drug <br /> and annual mortality rate 59,000 deaths [3].                                development NTDs complicated profit <br />                                                                                        motive [7]. Issues high toxicity, life-threatening side- <br /> * Correspondence: papin@virginia.edu                                                   effects, cost, parenteral administration emergence of <br /> 1 <br />  Department Biomedical Engineering, University Virginia, Charlottesville,        resistance drawbacks common existing antileishma- <br /> VA, USA <br /> Full list author information available end article <br />                                                                                        nial drugs sodium stibogluconate, meglumine <br />  <br />                                          &#194;&#169; 2012 Chavali et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                          Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                          medium, provided original work properly cited. <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                      Page 2 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> antimoniate, pentamidine amphotericin B [4,8]. There-         Subsequently, set criteria imposed select for <br /> fore, limited viable treatment options alterna-    approved drugs demonstrate effective <br /> tives pipeline, exists obvious need          antileishmanial activity. Constraints applied both <br /> develop radically new strategies conceptual frameworks      L. major genes (<span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> select high-priority targets) and <br /> to identify compounds novel antileishmanial activity.      drugs (to select high tolerance terms of <br />   Computational metabolic networks recon-             toxicity). To arrive suitable cutoff values, sensitivity <br /> structed pathogenic organisms [9,10], including     associated various cutoffs used MetDP was <br /> Leishmania major, agent cutaneous leishmaniasis           explored (see Additional file 1: Figures S1 S2). In the <br /> [11]. Implementing constraint-based modeling techniques,        selection high-priority L. major targets, metric <br /> such flux balance analysis (FBA), network recon-          applied druggability score L. major genes ran- <br /> structions yielded valuable insight gene essential-    ging 0 (not druggable) 1 (highly druggable). <br /> ity enzyme robustness varying environmental           Druggability indices obtained TDR Targets <br /> conditions [11,12]. Accordingly, analysis pathogen intra-    database. The imposition moderate druggability con- <br /> cellular processes used predict critical protein      straint 0.6 selected 88 L. major genes associated <br /> targets perturbed singly combination       638 drugs (see Figure 2A). Next, computational pre- <br /> adverse effects virulence and/or growth. Here, direct      dictions abnormal growth <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>phenotype</span> (lethal or <br /> metabolic network-driven method incorporates single        growth-reducing silico gene deletions) perform- <br /> gene essentiality synthetic lethality predictions pro-   ing FBA metabolic reconstruction also <br /> posed generates set high-priority L. major targets,   applied selection priority L. major genes. <br /> which turn associated select number Food       Following sensitivity analysis (see Additional file 1: Figure <br /> and Drug Administration (FDA)-approved drugs           S1), genes knocked the <br /> candidate antileishmanials. Subsequently, candidate drugs,      metabolic model yielded greater 30% growth defect <br /> which clinical use indications,    selected. At cutoff, 22 L. major genes linked to <br /> begin investigated clinical use leishmania-   372 drugs remained (see Figure 2A). Further, flux varia- <br /> sis. In addition, selection high-priority double-drug    bility analysis (FVA) performed metabolic <br /> combinations demonstrate superadditivity pro-        network L. major determine flux ranges gene- <br /> vide attractive alternative avenue drug dis-     associated (or enzyme-catalyzed) reactions. The motiva- <br /> covery leishmaniasis. By integrating publicly           tion use FVA score genes based an <br /> available resources using direct efficient method,    assumption genes linked reactions little <br /> approach presented offers significant implications      variability flux good targets disrupt the <br /> future drug discovery drug repurposing strategies           metabolic network. To elaborate, assumed that <br /> against leishmaniasis NTDs.                           enzymatic reactions inflexible wide range of <br />                                                                 flux values necessarily robust network per- <br /> Results                                                         turbations. Hence, FVA score genes computed <br /> Conceptual platform MetDP: A pipeline                    (see Methods) used metric alongside drugg- <br /> prioritization targets FDA-approved drugs                ability index silico gene essentiality analysis to <br /> The flowchart conceptual framework prioriti-        prioritize L. major targets. With FVA cutoff 1 (see <br /> zation drug drug targets L. major (hence-        Additional file 1: Figure S1 sensitivity associated with <br /> forth titled MetDP Metabolic network guided Drug            various FVA cutoffs), number L. major genes <br /> Pipeline) presented Figure 1. The specific datasets,      reduced 15 332 FDA-approved drugs passing <br /> tools cutoffs used process prioritizing effec-    cutoff (see Table 1 list targets). <br /> tive drugs drug targets L. major depicted       In parallel, high-priority synthetic lethal targets also <br /> in diagram presented Figure 2. The previously pub-       selected MetDP framework. The motivation <br /> lished metabolic reconstruction L. major accounted       investigating synthetic lethal targets there <br /> the function 560 genes [11]. Using bioinformatics tools,     drugs act multiple L. major targets <br /> a mapping created 560 genes drugs         simultaneous perturbation results growth inhibi- <br /> in publicly available DrugBank STITCH databases.            tion parasite. Using rationale, synthetic lethal- <br /> Using BLAST-based sequence similarity, association           ity constraint introduced MetDP order to <br /> was L. major genes known targets           account FDA-approved drugs target genes <br /> then transitively drugs. Deliberately implementing E-     involved &#226;&#8364;&#732;non-trivial lethal&#226;&#8364;&#8482; double combination (i.e. <br /> value cutoff STITCH confidence metric          genes combination individually non-essential <br /> selecting approved drugs resulted 538 L. major      combination essential). Using meta- <br /> genes linked 926 FDA-approved drugs (see               bolic network reconstruction L. major, 56 double gene <br /> Methods).                                                       deletions (out total 156,520 combinations) were <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                           Page 3 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br />                        STEP 1                                          STEP 2                                        STEP 3 <br />     Aim <br />  <br />  <br />                   Association L.                               Prioritizing L. major                           Removing toxic <br />                  major genes drugs                                    targets                                   compounds <br />  <br />  <br />  <br />  <br />                 DrugBank STITCH                                Metabolic network                               DrugBank; <br />     Datasets <br />  <br />  <br />  <br />  <br />                     databases                                  reconstruction L.                          ChemIDplus (NIH); <br />                     targets/drugs                                       major                                     MSDS (Sigma) <br />  <br />  <br />                                                                       TDR Targets <br />                                                                     druggability score <br />  <br />  <br />                                                                 Druggability score L. <br />                                                                     major genes in <br />                                                                     reconstruction <br />     Analysis <br />  <br />  <br />  <br />  <br />                 Sequence similarity of <br />                                                                 Flux Balance Analysis                            Toxicity FDA <br />                 L. major genes known <br />                                                                   predicted abnormal                          approved drugs (LD50) <br />                          targets <br />                                                                   growth phenotype <br />  <br />                                                                     Flux variability <br />                                                                 Analysis enzymes for <br />                                                                  assessing robustness <br />  <br />  <br />  <br />                    E-value cutoff 1e-3                             Druggability &gt;= 0.6                        Remove metabolites, <br />                                                                                                                illicit substances; <br />     Cutoffs <br />  <br />  <br />  <br />  <br />                   Only FDA-approved                                 Lethal growth- <br />                         drugs                                        reducing genes                          Tolerance toxicity <br />                                                                                                               rating &gt; 3 (Hodge and <br />                    STITCH conf. &gt; 0.4                             FVA gene score = 1                              Sterner scale) <br />  <br />  <br />  <br />  <br />                                                                                                                      Potential <br />                                                                                                               antileishmanial drugs <br />                                                                                                               associated high- <br />                                                                                                                  priority targets <br />  <br />  Figure 1 MetDP concept. The flowchart illustrates conceptual framework (MetDP) prioritization drug drug targets L. <br />  major. The step deals association L. major genes drugs. The second step includes prioritization L. major targets and <br />  drugs using constraints druggability, gene essentiality flux variability. The final step allows removal toxic compounds from <br />  prioritized list drugs yield set low-toxic candidate antileishmanials. <br />  <br />  <br /> previously identified non-trivial lethal [11]. Once again,               druggability index greater equal 0.5. In addition, <br /> a moderate druggability constraint 0.3 0.6                    genes combination needed moderate <br /> range considered: gene combinations average                     druggability indices 0.5 higher (see Figure 2B). A <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                                                                                                                                       Page 4 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br />  A                                    MetDP &#226;&#8364;&#8220; single gene deletions                                                                            B                             MetDP &#226;&#8364;&#8220; double gene deletions <br />  <br />  <br />                                            Met. Recon.                                                                                                                          Met. Recon.       TDR Targets <br />                                                                TDR Targets <br />                                                                                         STITCH                                                                                                                             STITCH <br />                                              560 genes                                                                                                                          560 genes <br />                                                              Druggability indices                                                                                                               Druggability indices <br />                                            1112 reactions                           74000+ compounds                                                                          1112 reactions                           74000+ compounds <br />                           DrugBank        1101 metabolites                            2.5M proteins <br />                                                                                                                                                               DrugBank       1101 metabolites                            2.5M proteins <br />                                                                                       630 organisms                                                                                                                      630 organisms <br />                             4774 drugs                                                                                                                         4774 drugs <br />                            4554 targets                                                          ChemIDplus;                                                  4554 targets                                                          ChemIDplus; <br />                           796 organisms                                                                                                                      796 organisms                                                             MSDS <br />                                                                                                     MSDS <br />                             LD50 data                                                                                                                          LD50 data <br />                                                                                                    LD50 data                                                                                                                          LD50 data <br />  <br />  <br />  <br />  <br />                                                                                                        1st cutoff: E-value &lt; 0.001                                                                                                        1st cutoff: E-value &lt; 0.001 <br />           538 L. major genes                                                                                                                   538 L. major genes <br />   BLAST                                                                                                2nd cutoff: Approved drugs      BLAST                                                                                              2nd cutoff: Approved drugs <br />               926 Drugs                                                                                                                            926 Drugs <br />                                                                                                        3rd cutoff: STITCH conf &gt; 0.4                                                                                                      3rd cutoff: STITCH conf &gt; 0.4 <br />               88 L. major genes                                                                                                                      42 synthetic lethals                                                           4th cutoff: Synthetic lethality <br />                                                                                                   4th cutoff: Drugg. &gt;= 0.6                    FBA <br />                   638 Drugs                                                                                                                               330 Drugs <br />  <br />                     22 L. major genes                                                        5th cutoff: Abnormal growth                                  8 synthetic lethals                                                   5th cutoff: Avg. drugg. &gt;= 0.5 <br />               FBA <br />                         372 Drugs                                                            phenotype (&gt;30% defect)                                          148 Drugs <br />                           15 L. major genes                                                                                                                  8 synthetic lethals                                            6th cutoff: Removing metabolites; <br />                     FVA                                                                  6th cutoff: Flux var. = 1 <br />                               332 Drugs                                                                                                                          107 Drugs                                                  illicit drugs; Tox. rating &gt;= 3 <br />  <br />                                15 L. major genes                                     7th cutoff: Removing                                                         8 synthetic lethals                                   7th cutoff: Targeting genes <br />                                    285 Drugs                                         metabolites; illicit drugs                                                        37 Drugs                                         combination <br />  <br />                                    15 L. major genes                            8th cutoff: Tox. rating &gt;= 3                                                          8 synthetic lethals                          8th cutoff: Unique drugs to <br />                                        240 Drugs                                                                                                                           14 Drugs                                list 240 <br />  <br />  <br />  <br />  <br />                                                     FDA-approved drugs                                                                                                             FDA-approved drugs with <br />                                                     potential antileishmanial                                                                                                           potential antileishmanial <br />                                                              activity                                                                                                                            activity <br />  <br />  <br />  Figure 2 Pipelines prioritization targets drugs. The funnel diagrams panels A B depict specific datasets, tools and <br />  cutoffs used process prioritization. Panel A presents prioritization scheme includes single gene deletions constraint, while <br />  Panel B presents prioritization scheme includes double gene deletions (or synthetic lethality) constraint. <br />  <br />  <br /> total 8 non-trivial lethal double gene combinations                                                                                     Consequently, implementation MetDP by <br /> satisfied criterion (see Table 2 list synthetic                                                                             means iterating pipeline (presented in <br /> lethal targets). With synthetic lethality druggability                                                                          Figures 2A 2B), prioritized list single L. major <br /> constraints yielding 8 double gene deletions,                                                                             targets included 15 metabolic genes (out 560 genes in <br /> an FVA constraint applied search high-                                                                               metabolic reconstruction; 2.7%) prioritized <br /> priority synthetic lethal targets.                                                                                                      list synthetic lethal targets included 8 double-gene <br />   The final step MetDP involved considerations                                                                                    combinations (out 156,520; 0.005%). Collectively, these <br /> tolerance toxicity order place emphasis                                                                                    lists high-priority targets associated 254 <br /> drugs potential clinical relevance leish-                                                                             FDA-approved drugs (out 4329 drugs DrugBank; <br /> maniasis. Drugs known common metabolites                                                                                    5.9%). The list prioritized L. major targets presented <br /> chemical elements (e.g. NADH, adenine, ATP, amino                                                                                       Tables 1 2, list 254 FDA-approved drugs <br /> acids, ethanol, iron zinc) present meta-                                                                                 (henceforth referred &#226;&#8364;&#732;Lm254&#226;&#8364;&#8482;) presented Addi- <br /> bolic reconstruction removed analysis.                                                                                tional file 1: Table S2, list drugs associated <br /> Also, illicit drugs controlled substances (Sche-                                                                                8 synthetic lethal targets presented Addi- <br /> dule I V) removed. Second, lethal                                                                                     tional file 1: Table S3. <br /> dose 50 (LD50) data culled DrugBank data-                                                                                        In summary, MetDP direct rational method of <br /> <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> material safety data sheets available online,                                                                              prioritizing drugs drug targets metabolic net- <br /> a toxicity rating based Hodge Sterner scale                                                                                  work underlying framework providing <br /> (see Additional file 1: Table S1) applied. Following                                                                                possible mechanism action drugs <br /> tolerance toxicity analysis, 15 single drug targets                                                                             selected (see Figure 3). <br /> were associated 240 drugs toxicity rating <br /> greater equal 3 (&#226;&#8364;&#8482;moderately toxic&#226;&#8364;&#8482;                                                                                   Target validation: Comparison target predictions with <br /> Hodge Sterner scale). Meanwhile, total 107                                                                                     observations previous literature <br /> drugs associated 8 non-trivial lethal gene                                                                                As initial validation, genes Table 1 <br /> combinations. Of these, 37 drugs mapped                                                                                    identified proven drug targets literature <br /> genes involved combination 14                                                                                    previously identified inhibitors Leish- <br /> were unique compared list 240 drugs                                                                                      mania related species. A particularly interesting <br /> (see Figure 2B).                                                                                                                        target LmjF05.0350 encoding trypanothione <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                                 Page 5 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> Table 1 The list prioritized L. major targets <br /> L. major              Enzyme                   E.C.           Metabolic pathway(s)               Sub-cellular     Druggability    Growth       No. of <br /> targets                                                                                             localization(s)&#226;&#8364;&#160;                   defect*      drugs <br /> LmjF04.0580         spermidine synthase            2.5.1.16        methionine metabolism                cytosol             0.8        100% (L)          3 <br /> LmjF05.0350       trypanothione reductase          1.8.1.12     trypanothione metabolism               glycosome            0.9        100% (L)         45 <br /> LmjF05.0830        methylthioadenosine             2.4.2.28        methionine metabolism                cytosol             0.8        100% (L)         15 <br />                      phosphorylase <br /> LmjF06.0650         lanosterol synthase            5.4.99.7         steroid biosynthesis                   ER               0.6        100% (L)          3 <br /> LmjF11.1100        sterol 14-demethylase          1.14.13.70        steroid biosynthesis                   ER               0.8        100% (L)         133 <br /> LmjF12.0280       ornithine decarboxylase          4.1.1.17    urea cycle metabolism             cytosol             1          100% (L)         40 <br />                                                                        amino groups <br /> LmjF13.1620      squalene monooxygenase           1.14.99.7         steroid biosynthesis                   ER               0.8        100% (L)         14 <br /> LmjF18.0020        diphosphomevalonate             4.1.1.33         steroid biosynthesis           glycosome/cytosol        0.6        100% (L)          5 <br />                        decarboxylase <br /> LmjF22.1360     dimethylallyltranstransferase       2.5.1.1         steroid biosynthesis           glycosome/cytosol        0.8        100% (L)         11 <br />                   geranyltranstransferase          2.5.1.10 <br /> LmjF25.1120      aldehyde dehydrogenase            1.2.1.3              pathways                mitochondria           1         &gt; 30% (GR)        28 <br /> LmjF30.3190     hydroxymethylglutaryl CoA          1.1.1.34         steroid biosynthesis             glycosome/             0.8        100% (L)         29 <br />                        reductase                                                                     mitochondria <br /> LmjF31.2940          squalene synthase             2.5.1.21         steroid biosynthesis           glycosome/cytosol        0.8        100% (L)          9 <br /> LmjF32.1580 phosphomannose isomerase               5.3.1.8         fructose mannose                 cytosol             0.6        100% (L)          5 <br />                                                                          metabolism <br /> LmjF33.2720      beta-ketoacyl-acyl-carrier-       2.3.1.41        fatty acid biosynthesis           mitochondria           0.7        100% (L)          5 <br />                     protein synthase I <br /> LmjF35.3340          phosphogluconate              1.1.1.44    pentose phosphate pathway                cytosol             0.8        100% (L)         13 <br />                       dehydrogenase <br /> * (L) Lethal, (GR) Growth-reducing; &#226;&#8364;&#160; (ER) Endoplasmic Reticulum <br />  <br />  <br /> reductase. In trypanosomatids, trypanothione reductase                             exclusive substrate specificities conferring advantage <br /> takes place glutathione reductase mainte-                           purposes drug targeting parasite <br /> nance intracellular redox balance [13]. Trypanothione                           enzyme [13]. Many inhibitors identified for <br /> reductase required <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>survival</span> Leishmania sp.                           trypanothione reductase, including trivalent antimony <br /> and important physiological role enzyme                             (Sb (III)) ions recently demonstrated [14]. Also, two <br /> defense oxidative damage parasites resi-                          nitrofuran derivative compounds experimentally <br /> dent inside macrophages [13]. Host glutathione reductase                           shown effective non-competitive inhibitors try- <br /> and parasite trypanothione reductase mutually                                 panothione reductase Trypanosoma cruzi, a <br />  <br />  <br /> Table 2 The list prioritized synthetic lethal targets <br /> Gene#1               Enzyme                Druggability No.         Gene#2              Enzyme            Druggability No.      Avg. <br />                                                              drugs                                                                  drugs    druggability <br /> LmjF27.2050         ribonucleoside-                  1             13     LmjF22.1290           ribonucleoside-             1          13           1 <br />                  diphosphate reductase                                                       diphosphate reductase <br />                        small chain                                                                 small chain <br /> LmjF06.0860     dihydrofolate reductase-             1             76     LmjF21.1210          thymidine kinase            0.8         10          0.9 <br />                   thymidylate synthase <br /> LmjF20.0100     phosphoglycerate kinase              0.8           7      LmjF24.0850 triose-phosphate isomerase           0.8         7           0.8 <br /> LmjF24.0850 triose-phosphate isomerase               0.8           7      LmjF30.3380      phosphoglycerate kinase         0.5         7           0.65 <br /> LmjF04.0960         adenylate kinase                 0.5           5      LmjF34.0110           adenylate kinase           0.8         4           0.65 <br /> LmjF30.3520      S-adenosylmethionine                0.6           17     LmjF30.3500        S-adenosylmethionine          0.6         17          0.6 <br />                       synthetase                                                                  synthetase <br /> LmjF05.0510      F-type H+-transporting              0.5           2      LmjF05.0500        F-type H+-transporting        0.5         2           0.5 <br />                    ATPase alpha chain                                                          ATPase alpha chain <br /> LmjF25.1180      F-type H+-transporting              0.5           3      LmjF25.1170        F-type H+-transporting        0.5         3           0.5 <br />                    ATPase beta chain                                                            ATPase beta chain <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                         Page 6 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> trypanosomatid organism causative agent Chagas                      Drug validation: Comparison drug predictions to <br /> disease [15]. MetDP predicted efficacy nitro-                    literature findings existing high-throughput screens <br /> furans associated LmjF05.0350 namely, nitrofura-                       As additional preliminary validation, literature was <br /> zone nitrofurantoin (see Figure 3). Other interesting                   mined consider drugs evaluated clini- <br /> targets prioritized list include LmjF12.0280                     cally leishmaniasis (see Additional file 1: Table S4). <br /> LmjF04.0580 encoding ornithine decarboxylase                        There positives Lm254: amphotericin B <br /> spermidine synthase, respectively. Similarly trypa-                      [17-20], ketoconazole [17,21], fluconazole [17,22], clotri- <br /> nothione reductase, genes essential Leishma-                    mazole [23], itraconazole [19,24], miconazole [23], terbina- <br /> nia sp. model predictions consistent                           fine [25], metronidazole [19,26] allopurinol [17-19]. <br /> experimental observations [11]. Ornithine decarboxylase                     Note drugs, literature contro- <br /> is important target Trypanosoma brucei,                          versial regards efficacy treating leishmania- <br /> another trypanosomatid organism causative agent                         sis. From computational network analysis, seven the <br /> for human African trypanosomiasis, well-estab-                       positive candidates predicted exclusively target <br /> lished inhibitor (eflornithine) [16]. See discussion                 L. major enzymes involved steroid biosynthesis, with <br /> on false negative target predictions Additional file 1.                  exceptions ketoconazole allopurinol. The anti- <br />  <br />  <br />  <br />  <br />                                              Leishmania major metabolic network <br />  <br />  <br />  <br />  <br />                                                                                                                                  Ketoconazole <br />  <br />  <br />  <br />  <br />                    Nitrofurazone <br />  <br />                                                                                            Disulfiram <br />                Nitrofurantoin <br />  <br />                                                              Halofantrine <br />  <br />  <br />  <br />  <br />                                                                                                                    Amoxicillin <br />  <br />  <br />  <br />  <br />  Figure 3 Metabolic network analysis prediction high-priority targets drugs. The pipeline prioritization drug and <br />  drug targets L. major (MetDP) based underlying metabolic network. This framework enables hypothesized description a <br />  mechanism action drugs identified. The figure highlights specific prioritized targets (singly lethal lethal combination) and <br />  associated drugs. Compound structures obtained PubChem database. <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                     Page 7 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> fungal ketoconazole predicted associated    drugs, bioactive compounds natural products pro- <br /> L. major enzymes: involved steroid biosynthesis     duced 35 hits assayed concentration 1 &#206;&#188;M <br /> and participating fructose mannose metabo-          [38]. One drug, paclitaxel, included list <br /> lism. And, allopurinol associated enzyme            35 hits present Lm254. Paclitaxel not <br /> involved methionine metabolism (see Additional file 1:      included set 68 drugs evaluated Sharlow et <br /> Table S4; note &#226;&#8364;&#732;Dependency confidence      al. study. In previous literature, paclitaxel IFNg <br /> interactions DrugBank STITCH&#226;&#8364;&#8482; Additional file        shown induce killing L. major infected murine <br /> 1). Previous research identified azole drugs ketoco-   macrophages [39]. In summary, comparison HTS <br /> nazole, itraconazole, miconazole fluconazole         data, 10 71 (14.1%) drugs Lm254 were <br /> sterol 14-demethylase inhibitors evaluated      experimentally evaluated HTS classified be <br /> in vitro T. cruzi parasites mouse models      potential antileishmanial hits. <br /> T. cruzi infection [27]. Additionally, fungal species      Taken together, inclusion clinically relevant <br /> Candida albicans, clotrimazole terbinafine       candidates existing HTS data yields total 18 <br /> shown inhibitors sterol 14-demethylase squa-      drugs (7.1% Lm254) having potential antileish- <br /> lene monooxygenase, respectively [28,29]. These refer-         manial activity (see Additional file 1: Table S2 list <br /> ences organisms serve check        18 drugs). The HTS data data drugs have <br /> the computationally predicted links MetDP           used clinically leishmaniasis account for <br /> L. major genes drugs.                                      83 254 drugs. Any drugs Lm254 pre- <br />    There false negative results: imiquimod      viously evaluated L. major parasites (in vitro and/ <br /> [20], paromomycin [17,19,20], pentamidine [17-20]          vivo) serve prime candidates future investiga- <br /> sodium stibogluconate [17-21]. Imiquimod modulator        tion discovery antileishmanial compounds. <br /> of innate immune response [30], paromomycin is <br /> an aminoglycoside antibiotic binds aminoacyl       In vitro experimental evaluation halofantrine <br /> decoding site ribosomal 16S RNA [31,32]. Since          The step involved vitro evaluation candidate <br /> both imiquimod paromomycin primarily target         antileishmanials prioritized using MetDP. Halofantrine, <br /> L. major proteins, exclusion Lm254              antimalarial agent, 14 drugs was <br /> obvious. However, reasons exclusion        selected synthetic lethality analysis (see Figure 4A), and <br /> drugs Lm254 examined (see          experimentally evaluated antileishmanial activity <br /> Additional file 1). Hence, thirteen (69.2%)        L. major promastigotes. This drug included <br /> clinically relevant drugs leishmaniasis present    HTS data used validate selection of <br /> Lm254. Three drugs used clinically leishma-      high-priority drugs. Halofantrine showed noticeable antil- <br /> niasis (meglumine antimoniate [17-20], miltefosine [20,33]     eishmanial activity concentration response assay (see <br /> and sitamaquine [19,20,34]) considered         Figure 4B). When compared &#226;&#8364;&#732;No Drug&#226;&#8364;&#8482; control, the <br /> analysis present DrugBank.                 effect halofantrine 3 &#206;&#188;M higher concentrations <br />    Additionally, high-throughput drugs screens         statistically significant (p &lt; 0.05). The IC50 halofan- <br /> against vitro growth Leishmania Trypanosoma          trine L. major promastigotes calculated be <br /> species recently published [35-38]. In particular,   approximately 9.5 &#206;&#188;M (Figure 4C). <br /> one study Sharlow et al. screened 196,146 compounds           Originally, DrugBank database, halofantrine <br /> at 10 &#206;&#188;M L. major promastigotes [37]. A total       associated vacuolar ATP synthase catalytic <br /> 187 FDA-approved drugs DrugBank              subunit A Plasmodium falciparum. Upon imple- <br /> database overlapped set 196,146 compounds;         mentation MetDP pipeline, halofantrine asso- <br /> 68 187 present Lm254. Of 68 drugs,        ciated following synthetic lethal pairs: <br /> seven (pimozide, furazolidone, perphenazine, bifonazole,       LmjF05.0500 &amp;LmjF05.0510, genes encoding F-type <br /> disulfiram, clotrimazole floxuridine) active       H+-transporting ATPase alpha chain, LmjF25.1170 <br /> primary hits high-throughput screen,            &amp;LmjF25.1180, genes encoding F-type H+-transporting <br /> remaining 61 classified inactive did       ATPase beta chain. While exact mechanism of <br /> meet 50% inhibition threshold evaluated 10         action halofantrine L. major requires further <br /> &#206;&#188;M. Furthermore, assay optimization valida-         investigation, model predictions halofantrine <br /> tion screening study, additional drugs     represent testable hypotheses. Towards end, an <br /> included list 68, tamoxifen mycophenolic         ATP bioluminescence assay performed further <br /> acid, deemed primary hits [37]. These      investigate effect halofantrine ATP levels L. <br /> drugs included Lm254. A second high-              major promastigotes. <br /> throughput screening (HTS) study inhibitors cytotoxic        As Figure 4D demonstrates, parasites incubated <br /> to bloodstream form T. brucei tested 2,160 FDA-approved        10 &#206;&#188;M halofantrine various <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                                                                                                                                                                                      Page 8 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br />                   A                                                                  Non-essential genes                           Synthetic lethal targets <br />                                                                                                                                                                        B                                                                                                        ** p &lt; 0.05 <br />                                                                                                                                                                                                  140                                                                            ** p &lt; 0.01 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        AlamarBlue fluorescence <br />                  Agent                                                          Drug                     Drug                               Drug                                                 120 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                             (normalized) <br />                                                                                                                                                                                                  100 <br />                                                                                                                                                                                                                                       *       * <br />                                                                                                                                                                                                    80                                                 ** <br />                                                                                                                                                                                                    60                                                          ** ** <br />             Target                                                             Gene X                  Gene Y                      Gene X             Gene Y                                                                                                         ** ** <br />                                                                                                                                                                                                    40                                                                                ** <br />                                                                                                                                                &amp;                                                                                                                                              ** <br />                                                                                                                                                                                                    20 <br />                                                                                                                                                                                                     0 <br />  <br />  <br />                 Effect                                                                                                                     NO                                                                                    Halofantrine concentration (&#206;&#188;M) <br />                                                                                GROWTH                GROWTH <br />                                                                                                                                          GROWTH <br />  <br />  <br />  <br />                    C                                                                                                                                                   D <br />                                                                          120 <br />                                   AlamarBlue fluorescence (normalized) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                   Conditions <br />                                                                                                                                                                                                                                                   Inhibition of <br />                                                                          100                                                                                                                                                                        F1-ATPase <br />                                                                                                                                                                                                                                      No                         Inhibition Inhibtion of <br />                                                                                                                                                                                                                     Media                              and <br />                                                                                                                                                                                                                                  inhibition                      glycolysis       both <br />                                                                          80                                                                                                                                                                        cytochrome <br />                                                                                                                                                                                                                                                     c oxidase <br />  <br />                                                                          60                                                   IC50 = 9.5&#206;&#188;M <br />                                                                                                                                                                                                           Media      + <br />                                                                                                                                                                                                  HBS glucose                     +                    + <br />                                                                                                                                                                        Reagents <br />  <br />  <br />                                                                          40                                                                                                         HBS glucose                                                                   +               + <br />                                                                                                                                                                                                     Sodium Azide                                           +                              + <br />                                                                          20 <br />                                                                                                                                                                                                  2-deoxy glucose                                                          +               + <br />                                                                           0                                                                                                                      Sodium Pyruvate                                                          +               + <br />                                                                                0.1           1                10              100           1000 <br />                                                                                              Concentration (&#206;&#188;M) <br />  <br />  <br />  <br />                      E                                                                                                                                                 F <br />                                                  600000                                *                                Parasites only <br />                                                                                                                                                                                                                                                                                 ** p &lt; 0.05 <br />                                                  500000                                                                 Parasites + Halofantrine (10 &#206;&#188;M) <br />                                                                                                                                                                                                  0.14                                                                           ** p &lt; 0.01 <br />                                                  400000                                            ***                                                *** p &lt; 0.05                                                                    Parasites only <br />                                                                                                                                                                                                  0.12 <br />             ATP Bioluminescence <br />  <br />  <br />  <br />  <br />                                                                                                                                                       *** p &lt; 0.01 <br />                                                  300000                                                                                               *** p &lt; 0.001                               0.1                                 Parasites + Halofantrine (10 &#206;&#188;M) <br />                                                                                                                                                                              Absorbance <br />  <br />  <br />  <br />  <br />                                                                  60000                                                                   ***                                                     0.08 <br />                                                                                                                                                                                                                                                   *                 ** <br />                                                                                                                                                                                                  0.06 <br />                                                                  50000                                                  *** <br />                                                                                                                                                                                                  0.04 <br />                                                                  40000 <br />                                                                                                                                                                                                  0.02 <br />                                                                  30000 <br />                                                                                                                                                                                                     0 <br />                                                                  20000                                                                                                                                      Media           No            Inhibition         Inhibition   Parasites <br />                                                                                                                                                                                                                          inhibition        F1-ATPase              glycolysis       with <br />                                                                  10000                                                                                                                                                                                                      inhibition <br />                                                                                                                                                                                                                                           cytochrome                             both <br />                                                                            0                                                                                                                                                                c oxidase <br />                                                                                      Media          No             Inhibition      Inhibition       Parasites <br />                                                                                                  inhibition         F1-ATPase           glycolysis           with <br />                                                                                                                                                       inhibition <br />                                                                                                                    cytochrome                              both <br />                                                                                                                      c oxidase <br />  <br />  Figure 4 Synthetic lethality experimental evaluation halofantrine L. major promastigotes. Synthetic lethality analysis <br />  highlights interaction genes combination; simultaneous <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>knockout</span> genes combination causes perturbation the <br />  functioning associated reactions network, leading adverse effect biomass production. A similar effect biomass <br />  occur genes combination knocked individually. Panel A illustrates concept drug acting multiple <br />  targets (that synthetically lethal) order inhibit growth L. major. The effect halofantrine evaluated different concentrations against <br />  L. major promastigotes presented Panel B. The y-axis indicates alamarBlue fluorescence normalized &#226;&#8364;&#732;No Drug&#226;&#8364;&#8482; control. AMP-B refers to <br />  Amphotericin B, used positive control assay. In Panel C, four-parameter log-logistic regression performed the <br />  concentration response data compute IC50 halofantrine L. major promastigotes. Panel D provides setup experimental <br />  conditions corresponding reagents used ATP bioluminescence assay determine effects halofantrine ATP levels L. <br />  major. The results ATP bioluminescence assay presented Panel E. Parasites incubated halofantrine 10 &#206;&#188;M <br />  presence mitochondrial and/or glycolytic ATP blocks 2 hours. The absorbance monitored 18 hours results are <br />  shown Panel F. All error bars indicate standard error. Statistical significance panels B E determined using one-tailed Student&#226;&#8364;&#8482;s t- <br />  test, statistical significance panel F determined using two-tailed Student&#226;&#8364;&#8482;s t-test. As additional note, absorbance <br />  measurements panel F parasites incubated halofantrine significant conditions. However, this <br />  significance attributable precision microplate reader meaningful biological implications regarding variations in <br />  absorbance. In Panels E F, condition parasites incubated mitochondrial glycolytic ATP blocks displayed with <br />  dashed bar serves reference. In particular case, halofantrine added. <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                    Page 9 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> conditions follows: (a) media, (b) HEPES-buffered sal-       combinations power network analysis can <br /> ine (HBS) &#226;&#8364;&#732;No inhibition&#226;&#8364;&#8482;, (c) HBS sodium azide        augment single compound discovery strategies. <br /> for &#226;&#8364;&#732;Inhibition F1-ATPase cytochrome c oxidase&#226;&#8364;&#8482;,            Since disulfiram previously shown sub- <br /> (d) HBS glucose, supplemented 2-               micromolar IC50 [37] (see dose response data dis- <br /> deoxy-D-glucose sodium pyruvate &#226;&#8364;&#732;Inhibition          ulfiram Additional file 1: Figure S7), goal to <br /> glycolysis&#226;&#8364;&#8482;, (e) HBS glucose, supplemen-        identify superadditive drug combinations involve dis- <br /> ted sodium azide, 2-deoxy-D-glucose sodium             ulfiram. Marketed Antabuse, disulfiram LD50 of <br /> pyruvate &#226;&#8364;&#732;Inhibition both&#226;&#8364;&#8482;. In assay, sodium         8600 mg/kg (rat; oral) used treatment alco- <br /> azide used inhibit mitochondrial oxidative ATP           holism. At moderate STITCH confidence greater than <br /> generation [40]. Further, 2-deoxy-D-glucose (a glucose          0.4, 8 high-priority synthetic lethal gene pre- <br /> analog competing substrate hexokinase)              dictions associated disulfiram. Therefore, the <br /> used sodium pyruvate glucose-free buffer          search criterion expanded include gene-drug inter- <br /> to inhibit glycolytic ATP synthesis [40]. In Figure 4E,         actions lower STITCH confidence (greater 0.15; <br /> when parasites subject &#226;&#8364;&#732;No inhibition,&#226;&#8364;&#8482; halo-       Methods mapping L. major genes and <br /> fantrine caused significant reduction ATP levels.          drugs; Additional file 2 list initial gene- <br /> Figure 4E shows ATP levels parasites         drug associations). <br /> &#226;&#8364;&#732;Inhibition F1-ATPase cytochrome c oxidase&#226;&#8364;&#8482;                 With relaxation STITCH confidence con- <br /> dropped significantly halofantrine added.              straint, pair high-priority synthetic lethals was <br /> Likewise, ATP levels parasites &#226;&#8364;&#732;Inhibition          associated disulfiram: LmjF25.1170 &amp;LmjF25.1180, <br /> glycolysis&#226;&#8364;&#8482; dropped significantly halofantrine         genes encoding F-type H+-transporting ATPase beta <br /> added. In instances, ATP levels dropped         chain. After tolerance toxicity analysis, genes <br /> level comparable parasites subject simulta-           associated 16 drugs (one them <br /> neous inhibition glycolysis F1-ATPase/cyto-              disulfiram; Additional file 1: Table S5). Since <br /> chrome c oxidase (&#226;&#8364;&#8482;Inhibition both&#226;&#8364;&#8482;). Hence, ATP         16 drugs predicted act genes syn- <br /> bioluminescence assay suggests halofantrine         thetic lethal pair, consequently predicted be <br /> effect glycolytic mitochon-        effective individually well. All possible drug combina- <br /> drial oxidative ATP generation mechanisms. Future               tions involving disulfiram (a total 15) then <br /> experimental efforts directed precise            selected. Combinations disulfiram drugs <br /> characterization underlying mechanism action          (kanamycin, clozapine, amoxicillin, chlorpromazine, dox- <br /> of drug elucidating potential metabolic           ycycline isoniazid) experimentally evaluated <br /> targets halofantrine L. major. Correspondingly,           L. major promastigotes using alamarBlue <br /> absorbance monitored make certain               assay. Figure 5 demonstrates superadditive nature of <br /> effects seen regard ATP levels           experimental combinations disulfiram kana- <br /> not function variations cell count var-      mycin, clozapine, amoxicillin, chlorpromazine. In all <br /> ious conditions (Figure 4F). Finally, halofan-         cases, experimental combinations signifi- <br /> trine known associated cardiotoxicity [41],       cantly greater inhibitory effect parasite growth as <br /> its potential antileishmanial agent inves-      compared calculated additive results (see Methods <br /> tigated detail.                                         calculations additivity; representative <br />                                                                 concentration profile displayed Figure 5; also <br /> Model-guided drug combinations alternative                Additional file 1: Figures S8, S9, S10 S11 data on <br /> strategy leishmaniasis                                  concentrations tested). Combinations disulfiram <br /> With ability computationally simulate synthetically      doxycycline isoniazid did statistically <br /> lethal gene deletions, exists opportunity predict   significant superadditivity (data shown). <br /> and prioritize multiple combination drug therapies            Two combinations demonstrated superaddi- <br /> may superadditive/synergistic. A potential advantage      tivity involved antibiotics - kanamycin aminoglycoside <br /> superadditive drug combinations overcoming toxicity       antibiotic amoxicillin common b-lactam antibiotic. <br /> side-effects linked high doses individual drugs needed    The combinations involved antipsychotics - clo- <br /> to establish inhibitory effect combination      zapine chlorpromazine used treatment of <br /> [42]. One strategy prioritizing clinically-relevant      symptoms associated schizophrenia. Of com- <br /> drug combinations selectively focus drug          binations, disulfiram + amoxicillin best overall <br /> that demonstrated excellent antileishmanial activity         toxicity rating. Moreover, clozapine chlorproma- <br /> vitro associated low-toxicity. All combina-        zine demonstrated inhibitory effects L. major promasti- <br /> tions involving particular drug prioritized   gotes tested individually (see Additional file 1: <br /> experimental analysis. Selection high-priority drug     Figures S9 S11). Kanamycin amoxicillin did not <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                                                                    Page 10 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br />  A               Disulfiram + Kanamycin                                                       B                            Disulfiram + Clozapine <br />                                                                                                                                                                           ** <br />                                                                                                                          1 <br />                    1                                                 ** <br />                                                                                                                        0.9 <br />                  0.8 <br />                                                                                                                        0.8 <br />                  0.6                                                                                                   0.7 <br />  <br />  <br />  <br />  <br />                                                                                                           Inhibition <br />     Inhibition <br />  <br />  <br />  <br />  <br />                                                                                                                        0.6 <br />                  0.4 <br />                                                                                                                        0.5 <br />                  0.2                                                                                                   0.4 <br />                    0                                                                                                   0.3 <br />                                                                                                                        0.2 <br />                  -0.2 <br />                                                                                                                        0.1 <br />                  -0.4   Disulfiram   Kanamycin     Additive result        Experimental                                   0 <br />                          (0.2 &#206;&#188;M)     (10 &#206;&#188;M)          (bliss)            combination                                           Disulfiram      Clozapine    Additive result    Experimental <br />                                                                                                                                 (0.15 &#206;&#188;M)        (10 &#206;&#188;M)         (bliss)        combination <br />  <br />  <br />                                                                              *** p &lt; 0.01                                                                                          *** p &lt; 0.01 <br />  <br />  <br />  C               Disulfiram + Amoxicillin                                                        D                         Disulfiram + Chlorpromazine <br />                                                                 ***                                                                                                        ** <br />                    1                                                                                                   1 <br />  <br />                  0.8                                                                                               0.8 <br />  <br />                  0.6                                                                                               0.6 <br />                                                                                              Inhibition <br />     Inhibition <br />  <br />  <br />  <br />  <br />                  0.4                                                                                               0.4 <br />  <br />                  0.2                                                                                               0.2 <br />  <br />                    0                                                                                                   0 <br />  <br />                  -0.2                                                                                           -0.2          Disulfiram     Chlorpromazine Additive result     Experimental <br />                         Disulfiram   Amoxicillin   Additive result    Experimental                                             (0.1 &#206;&#188;M)          (4 &#206;&#188;M)         (bliss)         combination <br />                          (0.3 &#206;&#188;M)     (100 &#206;&#188;M)         (bliss)        combination <br />  <br />  <br />                                                                            **** p &lt; 0.001                                                                                          *** p &lt; 0.01 <br />  <br />  Figure 5 Drug combinations involving disulfiram. Disulfiram kanamycin, clozapine, amoxicillin chlorpromazine all <br />  predicted individually effective antileishmanial agents act genes synthetic lethal pair. However, also <br />  potential act synergistically produce growth inhibition L. major. Therefore, predictions model-guided superadditivity, is <br />  assumed interactive relationship present genes synthetic lethal deletion translate directly associated drug <br />  combinations. In panels A, B, C D, results concentration profile disulfiram + kanamycin, disulfiram + clozapine, disulfiram + <br />  amoxicillin disulfiram + chlorpromazine evaluated L. major promastigotes presented, respectively. The theoretical additivity bar <br />  computed using bliss additivity metric comparing effects individual drugs. A two-sample t-test comparing means <br />  used determine statistical significance theoretical experimental combinations. Concentrations provided in <br />  parentheses. Error bars signify standard error. The y-axis indicates fractional experimental effect inhibition growth relative &#226;&#8364;&#339;No Drug&#226;&#8364;? <br />  control (0 equals inhibition, 1 equals max inhibition). All data generated 48 hours post addition alamarBlue dye. <br />  <br />  <br /> demonstrate inhibitory effect concentrations                                     drug combinations individual drugs <br /> tested (see Additional file 1: Figures S8 S10).                                         involved independently effective antileishmanial <br />   Interestingly, disulfiram, kanamycin                                       agents, limitation potentially overcome. If the <br /> chlorpromazine present Lm254. In future,                                    organism resistant drugs during <br /> evaluating two-drug combinations drugs                                       infection, potential compensatory mechanism <br /> predicted effective individually acting single                                  place drug inhibit growth. This redun- <br /> lethal growth-reducing targets certain                                          dancy serve prevent delay onset resis- <br /> advantages. Based solely synthetic lethality analysis,                                tance [43]. <br /> the event parasite develops resistance of <br /> the drugs (in combination) mutations                                         Discussion <br /> enzyme/gene product encoded single gene syn-                                      Recent literature described applicability meta- <br /> thetic lethal deletion, overall drug combination                                  bolic network analysis target identification and <br /> be predicted ineffective. However, considering                                     drug discovery general. One study (targetTB) used a <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                               Page 11 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> variety network analyses bioinformatics-based         bring novel drug market [47]. Besides, FDA- <br /> sequence/structural assessments predict list tar-    approved drug demonstrated antileishmanial activ- <br /> gets Mycobacterium tuberculosis [44]. By priori-     ity prescribed off-<span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span>, making acces- <br /> tizing targets using layered approach, proteins did   sible patients urgently need alternative <br /> not pass sequential cut-offs filtered [44].         treatment option. <br /> Another study implemented network analysis mole- <br /> cular docking simulations identify small molecule         Conclusions <br /> inhibitors type II fatty-acid biosynthesis enzymes     Ultimately, metabolic reconstructions provide frame- <br /> Escherichia coli Staphylococcus aureus experi-       work interrogation human pathogens and <br /> mentally evaluated computational predictions [45].       serve platform generation future experimental <br /> FBA used simulate drug synergy effects        hypotheses. This metabolic network-driven approach <br /> assigning synergy scores combinations enzymes          identified 15 L. major genes high-priority targets, 8 <br /> that inhibit growth E. coli various environ-     high-priority synthetic lethal targets, 254 FDA- <br /> mental conditions [42]. Network reconstructions P.        approved drugs potential antileishmanial agents. As <br /> falciparum Vibrio vulnificus guided        experimental validation, antimalarial drug halofan- <br /> prediction drug targets, investigational com-         trine shown noticeable antileishmanial <br /> pounds experimentally evaluated           activity. Moreover, ATP bioluminescence <br /> pathogens [9,46].                                            assay, halofantrine affected glycoly- <br />    Here, novel pipeline prioritization drugs    tic mitochondrial ATP generation mechanisms. <br /> and drug targets presented ways dis-      Additionally, synthetic lethal predictions meta- <br /> tinct approaches previously developed. Instead       bolic network aided selection drug combina- <br /> restricting search compounds tar-       tions potential superadditivity. For proof-of- <br /> get non-human proteins computed Tanimoto             concept, double-drug combinations evaluated in <br /> similarity important metabolites, MetDP undertook       vitro L. major novel superadditive com- <br /> more expansive strategy making use protein-com-        binations involving drug disulfiram discovered. <br /> pound interactions present DrugBank              Selection high-priority double-drug combina- <br /> STITCH resources. Importantly, clear majority       tions guided metabolic network analysis pro- <br /> approved drugs target human proteins opposed           vide attractive alternative avenue drug <br /> proteins single organism (e.g. DrugBank).    discovery infectious diseases like leishmaniasis. <br /> Therefore, avoiding selection patho-        A rational method prioritizing drugs drug tar- <br /> gen targets similar human proteins, MetDP        gets metabolic network underlying frame- <br /> explicitly sought targets order create      work provides possible mechanism action <br /> link L. major genes drugs       drugs selected (e.g., network model <br /> approved. The side-effects caused drugs     enables analysis biosynthetic pathways are <br /> the prioritized list act related human targets   inhibited particular enzyme&#226;&#8364;&#8482;s function phar- <br /> relatively known.                                       macologically inhibited). Surely mechanism action <br />    In addition, restricting search FDA-       predicted needs validated follow- <br /> approved drugs eliminating investigational drugs         experimentation. However, given drug poten- <br /> from selection process, major regulatory hurdles         tial target, starting experimental hypothesis pro- <br /> that novel compounds subject approval      vided. A network-driven approach guiding selection <br /> widespread clinical use bypassed. For instance,       prioritized drugs immensely advantageous in <br /> average, 15 years 800         terms cost efficiency beginning phases of <br /> million USD bring single novel drug market [47].     drug discovery offer significant implications to <br /> Moreover, 20 30 new drugs approved        future drug repurposing strategies variety of <br /> FDA annually [47]. Instead, drugs approved       NTDs. <br /> clinical use known pharmacokinetics toxicity/ <br /> safety profiles (many approved drugs met superior       Methods <br /> safety standards phase IV post-market safety sur-       Datasets used <br /> veillance) [47]. Therefore, demonstrating novel antil-    L. major reconstruction <br /> eishmanial activity drug clinical use   The previously published metabolic reconstruction L. <br /> another indication, candidate drug combi-       major kind protozoan. It <br /> nations drugs begin effectively repurposed      accounted 560 genes, 1112 reactions, 1101 <br /> by evaluated phase II clinical trials bypass-   metabolites spanning unique sub-cellular localiza- <br /> ing approximately 40% overall cost takes      tions [11]. <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                 Page 12 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> DrugBank                                                       extensively implemented metabolic networks various <br /> The DrugBank knowledgebase (version 2.5; http://www.           organisms ranging prokaryotes eukaryotes [12]. <br /> drugbank.ca/) included 4774 drugs classified <br /> into categories namely, (a) small molecule, (b) bio-   Mapping L. major genes drugs DrugBank <br /> tech, (c) approved, (d) experimental, (e) nutraceutical, (f)   STITCH <br /> illicit, (g) withdrawn. These drugs linked 4554    First, BLASTP search performed 560 L. major <br /> peptide/protein non-peptide/non-protein drug targets.      genes metabolic reconstruction (query <br /> STITCH                                                         sequences) 4538 target proteins DrugBank (tar- <br /> The STITCH database (version 2.0; http://stitch.embl.de/)      sequences). An E-value cutoff set 0.001, and <br /> comprised 74000 small molecules 2.5 million+          repetitious links L. major genes DrugBank <br /> proteins 630 organisms. The STITCH database includes        target proteins removed. At cutoff, 440 <br /> many potentially important off-targets particular com-      L. major genes linked 1116 DrugBank target pro- <br /> pounds (derived experimental data, curated          teins transitively 1129 drugs. The list drugs was <br /> databases text mining) addition primary           refined focus FDA-approved drugs removing <br /> known targets, protein-compound interactions        experimental withdrawn drugs, resulted 313 <br /> STITCH associated confidence metric. It         L. major genes linked 257 FDA-approved drugs. <br /> be noted L. major present list organ-     In parallel, similar approach variations was <br /> isms targets explicitly included STITCH.          undertaken STITCH database. In order as <br /> TDR Targets                                                    expansive possible making initial associations <br /> Druggability indices L. major genes downloaded        L. major genes drugs, DrugBank and <br /> from TDR Targets database (version 3; http://tdrtar-       STITCH resources used. A BLASTP search per- <br /> gets.org/). The druggability index calculated using      formed 560 L. major genes 2.5 million+ <br /> combination approaches including sequence similarity        proteins (2,590,259) associate L. major genes the <br /> to known biological targets associated FDA-               proteins STITCH. Protein sequences derived from <br /> approved drugs curated databases (e.g. DrugStore          related <span id='am-2' about='xsd:string' typeof='owl:Thing'>STRING</span> database (http://string-db.org/). Again, <br /> and StARLITe), sequence-based Bayesian learning          E-value cutoff 0.001 chosen repetitious asso- <br /> algorithm, methods [48]. The index ranges          ciations L. major genes STITCH proteins <br /> between 0 (not druggable) 1 (highly druggable). In         removed. At cutoff, 560 genes associated <br /> all, 1598 L. major genes corresponding druggabil-       131,250 protein targets STITCH. <br /> ity index TDR Targets database. Of these, 261         Subsequently, STITCH compound-protein interac- <br /> accounted metabolic reconstruction.                 tion dataset refined consider compounds that <br /> DEA controlled substances                                      FDA-approved drugs (as classified DrugBank <br /> The list controlled substances Schedule down-        database). PubChem compound identifiers used to <br /> loaded Drug Enforcement Administration                map drugs DrugBank compounds STITCH. <br /> (DEA) Office Diversion Control database (http://            Only 1337 1466 approved drugs DrugBank had <br /> www.deadiversion.usdoj.gov/schedules).                         readily available PubChem compound identifiers (and a <br />                                                                similar PubChem IDs). In all, 1330 drugs with <br /> Tools programs used                                        unique DrugBank IDs mapped 1305 unique compounds <br /> BLAST                                                          STITCH dataset. Next, STITCH database was <br /> A BLAST+ v.2.2.24 executable downloaded           reduced consider protein targets strictly present <br /> NCBI FTP site (ftp://ftp.ncbi.nih.gov/blast/executables/       list 131,250 unique protein targets the <br /> blast+/). The program makeblastdb implemented           BLAST cut. Finally, STITCH dataset con- <br /> both DrugBank STITCH target protein sequences.             strained include medium confidence higher <br /> The amino acid sequences 560 L. major genes       interactions (STITCH scores greater 0.4). The newly <br /> the metabolic reconstruction obtained GeneDB         constrained STITCH dataset BLAST results <br /> (version 2.1; http://www.genedb.org/genedb/leish). Subse-      used link 538 L. major genes 905 FDA- <br /> quently, program blastp used query 560              approved drugs. At stage, results DrugBank and <br /> L. major sequences DrugBank STITCH            STITCH merged refinement analysis. <br /> target databases.                                              The merged list linked 538 L. major genes 926 FDA- <br /> FBA/FVA/COBRA toolbox                                          approved drugs (see Additional file 2). <br /> Flux Balance Analysis (FBA) Flux Variability Analysis <br /> (FVA) implemented using COBRA toolbox (ver-           Generating FVA score genes <br /> sion 1.3.3; http://opencobra.sourceforge.net/openCOBRA/        Flux variability analysis (FVA) allows ascertaining the <br /> Welcome.html) MATLAB. FBA FVA                 complete range numerical values flux the <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                  Page 13 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> biochemical network given optimal value                  gene absent, GPR evaluated to <br /> objective [49]. FBA computes possible optimal             TRUE (e.g. presence isozymes). &#226;&#8364;&#732;PFi&#226;&#8364;&#8482; <br /> reaction network state. Therefore, flux distribution           refers number gene states particu- <br /> resulting FBA (i.e. enumeration possible           lar gene present, GPR evaluated be <br /> fluxes network) feasible solu-          FALSE (e.g. subunits larger protein complex <br /> tions exist optimal value             need present together). Finally, &#226;&#8364;&#732;AFi&#226;&#8364;&#8482; refers <br /> cellular objective. Enzymatic reactions associated            number gene states particular gene <br /> small ranges flux values necessarily robust          absent, GPR evaluated FALSE. <br /> to various kinds network perturbations. Therefore,           The weights (c g,r ) calculated totaling up <br /> FVA score genes developed based                      matches (counted positively) mis-matches <br /> assumption genes linked reactions little flux         (counted negatively) dividing number of <br /> variability good targets disrupt meta-          possible states given GPR. Hence, gene-reac- <br /> bolic network.                                                     tion matrix (C) constructed elements of <br />   Step 1: Conferring weights reactions based FVA             matrix representing weights genes linked to <br />                                                                    corresponding reactions. <br />     From FVA output, computed flux ranges <br />     [MaxFlux - MinFlux] reaction net-            Step 3: Conferring weights genes based FVA <br />     work used generate FVA reaction score <br />     (r) 0 (low) 1 (high).                              Next, FVA reaction scores (ri) multiplied in <br />                                                                    gene weights GPR (cg,r) yield <br />                       MaxFluxi &#226;&#710;&#8217; MinFluxi                          FVA-gene matrix dimensions C. <br />     ri = 1 &#226;&#710;&#8217;                                                       The maximum value genes reactions <br />                  2 &#226;&#710;&#8212; max(|MaxFluxi | , |MinFluxi |) <br />                                                                    considered FVA score gene (value <br />                                                                    resided 0 1). <br />     This scoring method gives high score reactions <br />     little variability flux. For example, reactions <br />     spanned entire flux range [-999999, 999999] <br />                                                                 Calculations additive results <br />     receive score 0, reactions no <br />                                                                 The additive results Figure 5 (i.e. non-synergistic <br />     change flux receive score 1. <br />                                                                 expectation experimental results individual <br />                                                                 drugs) computed based bliss additivity <br />   Step 2: Evaluating contributions genes gene- <br />                                                                 metric [50]: <br /> protein-reaction (GPR) relationships <br />                                                                    E(x, y) = E(x) + E(y) &#226;&#710;&#8217; E(x)E(y) <br />     Separately, GPR relationships, formulated <span id='am-5' about='xsd:boolean' typeof='owl:Thing'>Boolean</span> <br />     logic statements published reconstruction           In equation above, &#226;&#8364;&#732;E&#226;&#8364;&#8482; refers fractional effect <br />     L. major [11], used confer weights genes         0 1, &#226;&#8364;&#732;x&#226;&#8364;&#8482; &#226;&#8364;&#732;y&#226;&#8364;&#8482; concentrations <br />     accounted network. The weights cal-         drugs experimental combination. <br />     culated follows:                                         Additionally, error propagated using formula <br />                                                                 [50]: <br />                 PTik + AFik &#226;&#710;&#8217; PFik &#226;&#710;&#8217; ATik                                                                  <br />     cgi ,rk =                                                      &#206;&#180;E1,2 = &#206;&#180;E1 + &#206;&#180;E2 + &#206;&#180;E1 /E1 + &#206;&#180;E2 /E2 &#226;&#710;&#8212; E1 (x) &#226;&#710;&#8212; E2 (y) <br />                          Nstates <br />                                                                   Here, E1&#194;&#177;&#206;&#180;E1 refers effect drug 1 E2&#194;&#177;&#206;&#180;E2 <br />     Nstates refers number possible states       refers effect drug 2. <br />     given GPR, calculated 2 n n            Subsequently, two-sample t-test performed to <br />     number genes GPR. The GPR evalu-            compare additive result experimental com- <br />     ated TRUE FALSE state. For             bination. A t-score test statistic generated using the <br />     given reaction k GPR, presence (P)            formula: <br />     absence (A) particular gene TRUE (T) <br />                                                                       [&#194;&#175;x1 &#226;&#710;&#8217; x&#194;&#175; 2 ] <br />     FALSE (F) outcome GPR tabulated              t=  <br />     matches (PT AF ) mis-matches (PF                  s 1 2 s2 2 <br />     ATi). Explicitly, given reaction k, &#226;&#8364;&#732;PTi&#226;&#8364;&#8482; refers                + <br />                                                                         n1      n2 <br />     number gene states (out 2n) parti- <br />     cular gene present, GPR evaluated          Here, x&#194;&#175; 1 x&#194;&#175; 2 refer means samples, s1 <br />     TRUE. &#226;&#8364;&#732;ATi&#226;&#8364;&#8482; refers number gene states          s2 refer standard deviations samples, <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                               Page 14 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> and n1 n2 refer number samples. One-tail       Bioluminescence assay <br /> p-value subsequently determined referring t-     The protocol bioluminescence assay modi- <br /> table (degrees freedom computed follows: n1 + n2 -     fied [40]. Parasites 8 &#195;&#8212; 106 cells/mL incu- <br /> 2).                                                          bated culture medium various buffers 2 hours <br />                                                              26&#194;&#176;C presence 10 &#206;&#188;M halo- <br /> Experimental methods                                         fantrine. Mitochondrial oxidative ATP generation was <br /> Materials                                                    inhibited incubating parasites HBS buffer with <br /> Black flat-bottom 96-well microtiter plates purchased   glucose plus 20 mM sodium azide, inhibitor F1- <br /> from Fisher Scientific (http://www.fishersci.com) used   ATPase cytochrome c oxidase complex IV <br /> in alamarBlue experiments. White flat-bottom 96-well     [40]. Glycolytic ATP generation inhibited incu- <br /> microtiter plates purchased Fisher Scientific      bating parasites glucose-free HBS buffer plus 5 <br /> were used bioluminescence experiments. alamarBlue     mM 2-deoxy-D-glucose, competitor glucose for <br /> was purchased Invitrogen (http://www.invitrogen.        hexokinase binding, 5 mM sodium pyruvate [40]. In <br /> com). CellTiter-Glo purchased Promega (http://      white opaque flat-bottom 96-well microtiter plate, 25 <br /> www.promega.com). All compounds used study           &#206;&#188;L parasite samples condition seeded <br /> were purchased Sigma-Aldrich (http://www.sigmaal-       triplicate. Heat-killed parasite samples (incubated at <br /> drich.com/). Compounds solubilized dimethyl          60&#194;&#176;C 20 minutes) prepared 8 &#195;&#8212; 106 cells/ <br /> sulfoxide (DMSO) water.                                   mL seeded triplicate. Additionally, three <br /> Parasite cultures                                            wells seeded 25 &#206;&#188;L media alone. Subse- <br /> Previously published protocols culturing L. major [37]    quently, 25 &#206;&#188;L CellTiter-Glo added control <br /> were adhered study. L. major promastigotes        experimental wells. The plate incubated the <br /> and protocol preparing media kindly provided        dark 26&#194;&#176;C 10 minutes. Luminescence moni- <br /> by Mary E. Wilson Melissa A. Miller, University       tored using FLUOstar Optima plate reader (BMG Lab- <br /> Iowa. Parasites complete HOMEM (see Additional file       tech). For absorbance measurements, 100 &#206;&#188;L control <br /> 1) cultured 25 cm2 plastic tissue culture flasks     experimental samples seeded triplicate at <br /> with sealed vented caps maintained 26&#194;&#176;C.           18 hour time point. The plate immediately <br /> alamarBlue assay                                             transferred Tecan infinite200 Pro microplate reader, <br /> The assay conducted accordance previously        absorbance monitored 600 nm. Calibration <br /> established protocols [51-53]. Briefly, promastigotes        data bioluminescence assay provided Addi- <br /> were diluted 1 &#195;&#8212; 106 cells/mL, black flat-bot-   tional file 1: Figures S12 S13. <br /> tom 96-well microtiter plate, 180 &#206;&#188;L suspension was <br /> incubated varying concentrations drugs (singly       Additional material <br /> or combination) triplicate. Specifically, 160 &#206;&#188;L of <br /> parasite samples seeded triplicate. Next,       Additional file 1: In supplement, additional experimental data, <br /> sample wells topped 20 &#206;&#188;L media +            analysis network characteristics presented not <br />                                                               described main article [11,17-22,54]. <br /> drug(s) (ratio altered achieve specific concentrations <br />                                                               Additional file 2: In supplement, initial gene-drug associations, <br /> of drug(s)) total volume equaled 180 &#206;&#188;L.        various metric scores L. major genes, synthetic lethal <br /> Heat-killed parasite samples (incubated 60&#194;&#176;C 20        predictions, toxicity ratings drugs, list drugs removed <br /> minutes) prepared 1 &#195;&#8212; 106 cells/mL seeded        MetDP analysis presented. <br /> in triplicate (160 &#206;&#188;L sample + 20 &#206;&#188;L media) to <br /> serve positive control. Amphotericin B 1 &#206;&#188;M also <br /> served positive control. If DMSO used      Acknowledgements <br /> solubilize drug(s), wells highest rele-   AKC gratefully acknowledges expresses appreciation Kevin M. D&#226;&#8364;&#8482;Auria <br />                                                              insightful invaluable discussions course the <br /> vant concentration DMSO included plate        project. In addition, authors wish thank Erwin P. Gianchandani, Joseph <br /> as negative control. Additionally, wells        Klembczyk Matthew Oberhardt respective contributions. The <br /> seeded 180 &#206;&#188;L media alone. The plate incu-       authors acknowledge financial support National Science <br />                                                              Foundation (CAREER program grant no. 0643548 JP GRFP PAJ), the <br /> bated 26&#194;&#176;C 24 hours time point 20 &#206;&#188;L      National Institutes Health (grant no. GM088244 JP), Double Hoo <br /> alamarBlue dye added control experimen-       research grant University Virginia (awarded AKC Joseph <br /> tal wells. Using Gemini EM Microplate Spectrofluo-         Klembczyk). <br /> rometer, fluorescence monitored excitation/           Author details <br /> emission wavelengths 544 nm/590 nm 24 48           1 <br />                                                               Department Biomedical Engineering, University Virginia, Charlottesville, <br /> hours post addition dye wells. Calibration data    VA, USA. 2Department Medicine, Division Infectious Diseases and <br />                                                              International Health, University Virginia, Charlottesville, VA, USA. <br /> alamarBlue assay provided Additional file 1: Figures   3 <br />                                                               Department Pathology, University Virginia, Charlottesville, VA, USA. <br /> S4, S5 S6. <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                                     Page 15 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> 4 <br />  Mail: Box 800759, Health System, University Virginia, Charlottesville, VA,     19. Singh S, Sivakumar R: Challenges new discoveries treatment <br /> 22908, USA.                                                                            leishmaniasis. J Infect Chemother 2004, 10(6):307-315. <br />                                                                                    20. Mishra J, Saxena A, Singh S: Chemotherapy leishmaniasis: past, present <br /> Authors&#226;&#8364;&#8482; contributions                                                                 future. Curr Med Chem 2007, 14(10):1153-1169. <br /> AKC performed computational experimental analysis. AKC, ASB            21. Navin TR, Arana BA, Arana FE, Berman JD, Chajon JF: Placebo-controlled <br /> JLT performed experiments. PAJ helped computational analysis.             clinical trial sodium stibogluconate (Pentostam) versus ketoconazole <br /> RDP assisted interpretation experimental data. AKC JP              treating cutaneous leishmaniasis Guatemala. J Infect Dis 1992, <br /> conceived designed study. All authors read approved final              165(3):528-534. <br /> manuscript.                                                                        22. Alrajhi AA, Ibrahim EA, De Vol EB, Khairat M, Faris RM, Maguire JH: <br />                                                                                        Fluconazole treatment cutaneous leishmaniasis caused by <br /> Competing interests                                                                    Leishmania major. N Engl J Med 2002, 346(12):891-895. <br /> The authors declare competing interests.                         23. Larbi EB, al-Khawajah A, al-Gindan Y, Jain S, Abahusain A, al-Zayer A: A <br />                                                                                        randomized, double-blind, clinical trial topical clotrimazole versus <br /> Received: 16 January 2012 Accepted: 27 April 2012                                      miconazole treatment cutaneous leishmaniasis eastern <br /> Published: 27 April 2012                                                               province Saudi Arabia. AmJTrop Med Hyg 1995, 52(2):166-168. <br />                                                                                    24. Dogra J, Saxena VN: Itraconazole leishmaniasis: randomised <br /> References                                                                             double-blind trial cutaneous disease. Int J Parasitol 1996, <br /> 1. Hotez PJ, Fenwick A, Savioli L, Molyneux DH: Rescuing billion            26(12):1413-1415. <br />     control neglected tropical diseases. Lancet 2009,                   25. Bahamdan KA, Tallab TM, Johargi H, Nourad MM, Ibrahim K, el Sherbini AH, <br />     373(9674):1570-1575.                                                               Karkashan E, Khare AK, Nauri MM: Terbinafine treatment of <br /> 2. Hotez PJ: Mass drug administration integrated control                   cutaneous leishmaniasis: pilot study. Int J Dermatol 1997, 36(1):59-60. <br />     world&#226;&#8364;&#8482;s high-prevalence neglected tropical diseases. Clin Pharmacol Ther       26. Mishra M, Thakur BD, Choudhary M: Metronidazole Indian kala-azar: <br />     2009, 85(6):659-664.                                                               results clinical trial. Br Med J (Clin Res Ed) 1985, 291(6509):1611. <br /> 3. Desjeux P: Leishmaniasis: current situation new perspectives. Comp          27. Buckner FS: Sterol 14-demethylase inhibitors Trypanosoma cruzi <br />     Immunol Microbiol Infect Dis 2004, 27(5):305-318.                                  infections. Adv Exp Med Biol 2008, 625:61-80. <br /> 4. Croft SL, Coombs GH: Leishmaniasis-current chemotherapy recent              28. Ryder NS: Terbinafine: mode action properties squalene <br />     advances search novel drugs. Trends Parasitol 2003,                     epoxidase inhibition. Br J Dermatol 1992, 126(Suppl 39):2-7. <br />     19(11):502-508.                                                                29. Marriott MS: Inhibition sterol biosynthesis Candida albicans by <br /> 5. Molyneux D, Killick-Kendrick R: Morphology, ultrastructure life cycles.         imidazole-containing antifungals. J Gen Microbiol 1980, 117(1):253-255. <br />     In The Leishmaniases Biology Medicine. Volume 1. Edited by: Peters W,   30. Stanley MA: Imiquimod imidazoquinolones: mechanism of <br />     Killick-Kendrick R. London: Academic Press Inc; 1987:121-176.                      action therapeutic potential. Clin Exp Dermatol 2002, 27(7):571-577. <br /> 6. Hotez PJ, Molyneux DH, Fenwick A, Savioli L, Takeuchi T: A Global Fund       31. Vicens Q, Westhof E: Crystal structure paromomycin docked the <br />     Fight Neglected Tropical Diseases: G8 Hokkaido Toyako 2008                  eubacterial ribosomal decoding A site. Structure 2001, 9(8):647-658. <br />     Summit ready? PLoS Negl Trop Dis 2008, 2(3):e220.                              32. Fernandez MM, Malchiodi EL, Algranati ID: Differential effects of <br /> 7. Caffrey CR, Steverding D: Recent initiatives strategies developing           paromomycin ribosomes Leishmania mexicana mammalian <br />     new drugs tropical parasitic diseases. Expert Opin Drug Discov 2008,        cells. Antimicrob Agents Chemother 2011, 55(1):86-93. <br />     3(2):173-186.                                                                  33. Garnier T, Croft SL: Topical treatment cutaneous leishmaniasis. Curr <br /> 8. Guerin PJ, Olliaro P, Sundar S, Boelaert M, Croft SL, Desjeux P, Wasunna MK,        Opin Investig Drugs 2002, 3(4):538-544. <br />     Bryceson AD: Visceral leishmaniasis: current status control, diagnosis,     34. Wasunna MK, Rashid JR, Mbui J, Kirigi G, Kinoti D, Lodenyo H, Felton JM, <br />     treatment, proposed research development agenda.                     Sabin AJ, Albert MJ, Horton J: A phase II dose-increasing study of <br />     Lancet Infect Dis 2002, 2(8):494-501.                                              sitamaquine treatment visceral leishmaniasis Kenya. <br /> 9. Plata G, Hsiao TL, Olszewski KL, Llinas M, Vitkup D: Reconstruction             AmJTrop Med Hyg 2005, 73(5):871-876. <br />     flux-balance analysis Plasmodium falciparum metabolic network.          35. Siqueira-Neto JL, Song OR, Oh H, Sohn JH, Yang G, Nam J, Jang J, <br />     Mol Syst Biol 2010, 6:408.                                                         Cechetto J, Lee CB, Moon S, et al: Antileishmanial high-throughput drug <br /> 10. Jamshidi N, Palsson BO: Investigating metabolic capabilities                screening reveals drug candidates new scaffolds. PLoS Negl Trop Dis <br />     Mycobacterium tuberculosis H37Rv using silico strain iNJ661             2010, 4(5):e675. <br />     proposing alternative drug targets. BMC Syst Biol 2007, 1:26.                  36. St George S, Bishop JV, Titus RG, Selitrennikoff CP: Novel compounds <br /> 11. Chavali AK, Whittemore JD, Eddy JA, Williams KT, Papin JA: Systems                 active Leishmania major. Antimicrob Agents Chemother 2006, <br />     analysis metabolism pathogenic trypanosomatid Leishmania                 50(2):474-479. <br />     major. Mol Syst Biol 2008, 4:177.                                              37. Sharlow ER, Close D, Shun T, Leimgruber S, Reed R, Mustata G, Wipf P, <br /> 12. Oberhardt MA, Palsson BO, Papin JA: Applications genome-scale                   Johnson J, O&#226;&#8364;&#8482;Neil M, Grogl M, et al: Identification potent chemotypes <br />     metabolic reconstructions. Mol Syst Biol 2009, 5:320.                              targeting Leishmania major using high-throughput, low-stringency, <br /> 13. Dumas C, Ouellette M, Tovar J, Cunningham ML, Fairlamb AH, Tamar S,                computationally enhanced, small molecule screen. PLoS Negl Trop Dis <br />     Olivier M, Papadopoulou B: Disruption trypanothione reductase               2009, 3(11):e540. <br />     gene Leishmania decreases ability survive oxidative stress        38. Mackey ZB, Baca AM, Mallari JP, Apsel B, Shelat A, Hansell EJ, Chiang PK, <br />     macrophages. EMBO J 1997, 16(10):2590-2598.                                        Wolff B, Guy KR, Williams J, et al: Discovery trypanocidal compounds by <br /> 14. Baiocco P, Colotti G, Franceschini S, Ilari A: Molecular basis antimony         cell HTS Trypanosoma brucei. Chem Biol Drug Des 2006, <br />     treatment leishmaniasis. J Med Chem 2009, 52(8):2603-2612.                      67(5):355-363. <br /> 15. Jockers-Scherubl MC, Schirmer RH, Krauth-Siegel RL: Trypanothione              39. Doherty TM, Sher A, Vogel SN: Paclitaxel (Taxol)-induced killing of <br />     reductase Trypanosoma cruzi. Catalytic properties enzyme               Leishmania major murine macrophages. Infect Immun 1998, <br />     inhibition studies trypanocidal compounds. Eur J Biochem 1989,            66(9):4553-4556. <br />     180(2):267-272.                                                                40. Manzano JI, Carvalho L, Perez-Victoria JM, Castanys S, Gamarro F: Increased <br /> 16. Heby O, Persson L, Rentala M: Targeting polyamine biosynthetic                 glycolytic ATP synthesis associated tafenoquine resistance in <br />     enzymes: promising approach therapy African sleeping sickness,             Leishmania major. Antimicrob Agents Chemother 2011, 55(3):1045-1052. <br />     Chagas&#226;&#8364;&#8482; disease, leishmaniasis. Amino Acids 2007, 33(2):359-366.           41. Bouchaud O, Imbert P, Touze JE, Dodoo AN, Danis M, Legros F: Fatal <br /> 17. Blum J, Desjeux P, Schwartz E, Beck B, Hatz C: Treatment cutaneous              cardiotoxicity related halofantrine: review based worldwide <br />     leishmaniasis travellers. J Antimicrob Chemother 2004,                       safety data base. Malar J 2009, 8:289. <br />     53(2):158-166.                                                                 42. Lehar J, Krueger AS, Avery W, Heilbut AM, Johansen LM, Price ER, Rickles RJ, <br /> 18. Olliaro PL, Bryceson AD: Practical progress new drugs changing             Short GF, Staunton JE, Jin X, et al: Synergistic drug combinations tend to <br />     patterns leishmaniasis. Parasitol Today 1993, 9(9):323-328.                     improve therapeutically relevant selectivity. Nat Biotechnol 2009, <br />                                                                                        27(7):659-666. <br />  Chavali et al. BMC Systems Biology 2012, 6:27                                                                                                  Page 16 16 <br /> http://www.biomedcentral.com/1752-0509/6/27 <br />  <br />  <br />  <br />  <br /> 43. WHO: Guidelines treatment malaria (2nd ed.). 2 edition. Geneva, <br />     Switzerland: WHO Press, World Health Organization; 2010. <br /> 44. Raman K, Yeturu K, Chandra N: TargetTB: target identification pipeline <br />     Mycobacterium tuberculosis interactome, reactome and <br />     genome-scale structural analysis. BMC Syst Biol 2008, 2:109. <br /> 45. Shen Y, Liu J, Estiu G, Isin B, Ahn YY, Lee DS, Barabasi AL, Kapatral V, <br />     Wiest O, Oltvai ZN: Blueprint antimicrobial hit discovery targeting <br />     metabolic networks. Proc Natl Acad Sci USA 2010, 107(3):1082-1087. <br /> 46. Kim HU, Kim SY, Jeong H, Kim TY, Kim JJ, Choy HE, Yi KY, Rhee JH, Lee SY: <br />     Integrative genome-scale metabolic analysis Vibrio vulnificus drug <br />     targeting discovery. Mol Syst Biol 2011, 7:460. <br /> 47. Chong CR, Sullivan DJ Jr: New uses old drugs. Nature 2007, <br />     448(7154):645-646. <br /> 48. Aguero F, Al-Lazikani B, Aslett M, Berriman M, Buckner FS, Campbell RK, <br />     Carmona S, Carruthers IM, Chan AW, Chen F, et al: Genomic-scale <br />     prioritization drug targets: TDR Targets database. Nat Rev Drug <br />     Discov 2008, 7(11):900-907. <br /> 49. Mahadevan R, Schilling CH: The effects alternate optimal solutions in <br />     constraint-based genome-scale metabolic models. Metab Eng 2003, <br />     5(4):264-276. <br /> 50. Goldoni M, Johansson C: A mathematical approach study combined <br />     effects toxicants vitro: evaluation Bliss independence <br />     criterion Loewe additivity model. Toxicol In Vitro 2007, <br />     21(5):759-769. <br /> 51. Oliveira-Silva F, Morais-Teixeira E, Rabello A: Antileishmanial activity <br />     azithromycin Leishmania (Leishmania) amazonensis, <br />     Leishmania (Viannia) braziliensis, Leishmania (Leishmania) chagasi. <br />     AmJTrop Med Hyg 2008, 78(5):745-749. <br /> 52. Mikus J, Steverding D: A simple colorimetric method screen drug <br />     cytotoxicity Leishmania using dye Alamar Blue. Parasitol Int <br />     2000, 48(3):265-269. <br /> 53. Shimony O, Jaffe CL: Rapid fluorescent assay screening drugs on <br />     Leishmania amastigotes. J Microbiol Methods 2008, 75(2):196-200. <br /> 54. Urbina JA, Concepcion JL, Rangel S, Visbal G, Lira R: Squalene synthase as <br />     chemotherapeutic target Trypanosoma cruzi Leishmania <br />     mexicana. Mol Biochem Parasitol 2002, 125(1-2):35-45. <br />  <br />  doi:10.1186/1752-0509-6-27 <br />  Cite article as: Chavali et al.: Metabolic network analysis predicts <br />  efficacy FDA-approved drugs targeting causative agent a <br />  neglected tropical disease. BMC Systems Biology 2012 6:27. <br />  <br />  <br />  <br />  <br />                                                                                      Submit manuscript BioMed Central <br />                                                                                      advantage of: <br />  <br />                                                                                      &#226;&#8364;&#162; Convenient online submission <br />                                                                                      &#226;&#8364;&#162; Thorough peer review <br />                                                                                      &#226;&#8364;&#162; No space constraints color figure charges <br />                                                                                      &#226;&#8364;&#162; Immediate publication acceptance <br />                                                                                      &#226;&#8364;&#162; Inclusion PubMed, CAS, Scopus Google Scholar <br />                                                                                      &#226;&#8364;&#162; R <br /> </body></html>