<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                  INVESTIGATION <br />  <br />  <br />  <br />  <br />                                 The Importance Conserved Features Yeast <br />                                Actin-Binding Protein 1 (Abp1p): The Conditional <br />                                                           Nature Essentiality <br />                                                                                         Bianca Garcia,* Elliott J. Stollar,&#226;&#8364;&#160;,1 Alan R. Davidson*,&#226;&#8364;&#160;,2 <br />                         *Department Molecular Genetics &#226;&#8364;&#160;Department Biochemistry, University Toronto, Toronto, Ontario, Canada M5S 1A8 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> Saccharomyces cerevisiae Actin-Binding Protein 1 (Abp1p) member Abp1 family proteins, diverse <br /> organisms including fungi, nematodes, &#239;&#172;&#8218;ies, mammals. All proteins family possess N-terminal Actin Depolymerizing <br /> Factor Homology (ADF-H) domain, central Proline-Rich Region (PRR), C-terminal SH3 domain. In study, employed <br /> sequence analysis identify additional conserved features family, including sequences rich proline, glutamic acid, serine, and <br /> threonine amino acids (PEST), family members examined, motifs, Conserved Fungal Motifs 1 2 <br /> (CFM1 CFM2), conserved fungi. We discovered that, similar mammalian homologs, Abp1p phosphorylated <br /> in PRR. This phosphorylation mediated Cdc28p Pho85p kinases, protects Abp1p proteolysis mediated by <br /> the conserved PEST sequences. We provide evidence intramolecular interaction PRR region SH3 domain that <br /> may affected phosphorylation. Although deletion CFM1 caused detectable <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> genetic backgrounds <br /> or conditions tested, deletion motif resulted signi&#239;&#172;?cant reduction growth combined deletion the <br /> ADF-H domain. Importantly, result demonstrates deletion highly conserved domains produce phenotype <br /> unless domains assayed conjunction deletions functionally important elements protein. <br /> Detection type intragenic synthetic lethality provides important approach understanding function individual <br /> protein domains motifs. <br />  <br />  <br />  <br /> S   ACCHAROMYCES cerevisiae Acting-Binding Protein 1 <br />     (Abp1p) &#239;&#172;?rst described member highly con- <br /> served family actin-binding proteins (Drubin et al. 1988) <br />                                                                                         sequences (Stollar et al. 2009). Given high conservation <br />                                                                                         ubiquitous occurrence Abp1 family members, these <br />                                                                                         proteins undoubtedly ful&#239;&#172;?ll critical function, investi- <br /> found diverse organisms including fungi, worms, &#239;&#172;&#8218;ies,                                gating functions important objective. In this <br /> and humans. The common features proteins                                work, used yeast Abp1p model gain further <br /> N-terminal Actin Depolymerizing Factor Homology (ADF-H)                                 insight family. <br /> domain (Lappalainen et al. 1998), followed large, mainly                              Abp1p originally identi&#239;&#172;?ed actin-binding pro- <br /> unstructured central region including Pro-Rich Region                                 tein actin-af&#239;&#172;?nity chromatography (Drubin et al. 1988), <br /> (PRR) C-terminal SH3 domain (Figure 1). The con-                                  shown localize cortical actin patches. <br /> servation SH3 domains proteins par-                               Abp1p plays important roles actin organization en- <br /> ticularly high (e.g., human yeast domains 45%                               docytosis. It binds actin &#239;&#172;?laments, actin mono- <br /> identical), recognize similar target peptide                              mers, mainly ADF-H domain (Lappalainen et al. <br />                                                                                         1998, Goode et al. 2001), possesses acidic motifs <br />                                                                                         required binding activation Arp2/3 <br /> Copyright &#194;&#169; 2012 Genetics Society America <br /> doi: 10.1534/genetics.112.141739 <br />                                                                                         complex (Goode et al. 2001). The SH3 domain mediates bi- <br /> Manuscript received May 4, 2012; accepted publication May 24, 2012                  ologically relevant interactions proteins <br /> Supporting information available online http://www.genetics.org/content/          involved endocytosis, Ark1p, Scp1p, Sjl2p (Lila <br /> suppl/2012/06/01/genetics.112.141739.DC1 <br /> 1 <br />   Present address: Department Physical Sciences, Eastern New Mexico University,      Drubin 1997; Fazi et al. 2002; Stefan et al. 2005; Haynes <br /> 2 <br />   Station #33, Portales, NM 88130.                                                      et al. 2007; Stollar et al. 2009). The mammalian homolog <br />  Corresponding author: Department Molecular Genetics Microbiology, <br />  University Toronto, 1 King&#226;&#8364;&#8482;s College Circle, Toronto, ON, Canada M5S 1A8. E-mail: <br />                                                                                         Abp1p (mAbp1), similar yeast Abp1p, binds <br />  alan.davidson@utoronto.ca.                                                             F-actin N-terminal actin-binding domain is <br />  <br />  <br />  <br />                                                                                                       Genetics, Vol. 191, 1199&#226;&#8364;&#8220;1211 August 2012    1199 <br />  Figure 1 Conserved features Abp1 family members. (A) Analysis domain structure Saccharomyces cerevisiae Abp1p (YEAST) other <br /> Abp1p homologs different species: Candida albicans (CANAL), Neurospora crassa (NEUCR), Caenorhabditis elegans (CAELE), Drosophila mela- <br /> nogaster (DROME), Mus musculus (MOUSE). Regions predicted high probability PEST sequences helical indicated gray <br /> brackets green horizontal bars, respectively. Con&#239;&#172;?rmed phosphorylation sites DROME MOUSE Abp1 homologs indicated. PRRs and <br /> the boundaries ADFH SH3 domains de&#239;&#172;?ned using Scanprosite (http://prosite.expasy.org/scanprosite). (B) Alignment Conserved Fungal <br /> Motifs (CFM1 CFM2) Abp1p homologs diverse fungal species. The following groups residues shaded colors: (C, I, <br /> L, V, M), (F, Y, W, H), (D, E), (N, Q), (R, K), (G, A, S, T), P. Numbering according S. cerevisiae (YEAST) Abp1p. Other species included the <br /> analysis Kluyveromyces lactis (KLULA), Candida albicans (CANAL), Debaryomyces hansenii (DEBHA), Yarrowia lipolytica (YARLI), Sclerotinia sclero- <br /> tiorum (SCLSC), Phaeosphaeria nodorum (PHANO), Neurospora crassa (NEUCR), Uncinocarpus reesii (UNCRE), Aspergillus fumigatus (ASPFU), Copri- <br /> nopsis cinerea (COPCI), Ustilago maydis (USTMA), Malassezia globosa (MALGL). A sequence logo, quanti&#239;&#172;?es conservation position in <br /> the alignment, shown. <br />  <br />  <br /> involved receptor-mediated endocytosis (Kessels et al.                     et al. 1999). An interesting aspect Abp1p function that <br /> 2001; Mise-Omata et al. 2003). The SH3 domain mediates                        vivo requirements domains differ depending on <br /> protein&#226;&#8364;&#8220;protein interactions proteins involved syn-                   genetic background assay carried out. For <br /> aptogenesis, endocytosis, cell motility (Kessels et al.                   example, SH3 domain required known <br /> 2001; Fenster et al. 2003; Han et al. 2003; Cortesio et al.                   ABP1-dependent genetic backgrounds (Lila Drubin <br /> 2010). mAbp1p recruited dynamic actin structures                        1997; Haynes et al. 2007), certain amino acid substitutions <br /> (Kessels et al. 2000), localization reminiscent                partially decrease af&#239;&#172;?nity domain its <br /> the localization yeast protein, cor-                 targets cause marked reduction viability the <br /> tical actin patches accumulating yeast bud                  abp1&#206;&#8221;sac6&#206;&#8221; abp1&#206;&#8221;sla2&#206;&#8221; backgrounds (Haynes et al. <br /> actin cables (Drubin et al. 1988).                                            2007). Surprisingly, deletion conserved ADF-H domain <br />    Although deletion yeast ABP1 gene does result                   resulted loss viability backgrounds, <br /> in slower growth, deletion synthetically lethal                  abp1&#206;&#8221;sla1&#206;&#8221; background (Quintero-Monzon <br /> deletions SAC6, SLA1, SLA2, encode actin-                    et al. 2005), functional roles regions of <br /> associated components involved endocytosis (Holtzman                       Abp1p varying conditions remain unknown. Another <br /> et al. 1993). In addition, combined deletion ABP1                      uninvestigated aspect Abp1p regulation through <br /> PRK1, encodes actin patch-associated protein ki-                     post-translation modi&#239;&#172;?cation. Since mAbp1 known be <br /> nase, results temperature-sensitive phenotype (Cope                      phosphorylated Src- Syk-family kinases (Larbolette <br />  <br />  <br />  <br /> 1200      B. Garcia, E. J. Stollar, A. R. Davidson <br />  Table 1 Yeast strains used study <br />  <br /> Strain                                                   Genotype                                                                 Source <br /> Y263                    MATa trp1D63 ura3-52 lys2-801 ade2-107 his3D200 leu2-D1                             Measday et al. (1994) <br /> BY1009a                 BY263 abp1D::Kan                                                                    Haynes et al. (2007) <br /> BY3002b                 a/a abp1D::Nat sla1D::Kan                                                           B. Andrew lab <br /> abp1Dprk1D              MATa abp1D::Nat prk1D::Kan                                                          Cross BY1689 BY2985 Haynes et al. (2007) <br /> RH3395                  MATa lys2 leu2 ura3 his3 bar1 sla2/end4::HIS3 end4D376-501::TRP1                    Wesp et al. (1997) <br /> abp1Dsla2Dcoil1         MATa lys2 leu2 ura3 his3 bar1 sla2/end4::HIS3                                       This study <br />                           end4D376-501::TRP1 abp1::Nat <br /> BY4068b                 MATa cdc28-as::Nat ura3 leu2 his3 met15                                             B. Andrew lab <br /> BY4131b                 MATa pho85-as::Hph ura3 leu2 his3 met15                                             B. Andrew lab <br /> BY4129b                 MATa cdc28-as::Nat pho85-as::Hph ura3 leu2 his3 met15 CAN1+LYP1+                    B. Andrew lab <br /> a <br />     Strains isogenic parent strain, BY263, S288C derivative. <br /> b <br />     Strains deletion consortium isogenic parent strain, BY4741 (MATa ura3D0 leu2D0 his3D1 met15D0), derived S288C (Brachmann <br />     et al. 1998). <br />  <br />  <br />  <br /> et al. 1999; Han et al. 2003; Larive et al. 2009; Boateng et al.                     previous work (Haynes et al. 2007), double <br /> 2012) phosphorylated peptides Abp1p                               mutants isolated heterozygous diploid <br /> identi&#239;&#172;?ed global studies yeast protein phosphor-                       abp1Dsla1D strain tetrad dissection allowed grow <br /> ylation (Ficarro et al. 2002; Ubersax et al. 2003; Chi et al.                        room temperature. In cases abp1Dprk1D and <br /> 2007; Holt et al. 2009), role phosphorylation                           abp1Dsla2Dcoil1 strains, double-mutant haploid strains <br /> function Abp1p expected.                                                 transformed directly different genetic con- <br />     In study, analyzed sequences diverse                         structs carrying mutant forms Abp1p. <br /> Abp1 family members identi&#239;&#172;?ed previously unrecog- <br />                                                                                      Recombinant DNA manipulations <br /> nized conserved motifs. By analyzing effects variety <br /> of ABP1 deletion mutants assayed different ge-                            Plasmids used study listed Table 2. Plasmid <br /> netic backgrounds, revealed functional roles                             p6-1 contains 3.5-kb EcoRI fragment carrying ABP1 <br /> these conserved elements Abp1p. We discov-                              gene 1.5 kb upstream initiator ATG codon <br /> ered Abp1p phosphorylated region                          180 bp downstream stop codon subcloned into <br /> mammalian counterparts. Our results highlight impor-                             pRS316DSalI (Haynes et al. 2007). This plasmid used <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> tance conservation analysis assessment phenotypes                          generate yeast expression plasmids wild-type (WT) and <br /> under variety conditions elucidation protein                         mutant versions Abp1p using standard cloning techni- <br /> function.                                                                            ques. All generated plasmids sequenced verify the <br />                                                                                      integrity constructs. <br /> Materials Methods <br />                                                                                      Protein immunoprecipitation Western blot analysis <br /> Yeast strains growth conditions <br />                                                                                      Yeast-cell protein extracts prepared exponentially <br /> Yeast strains used study listed Table 1. ABP1                         growing cells expressing WT mutant Abp1p lysing cells <br /> gene disruption create abp1Dsla2Dcoil1 strain                             glass beads (Lee et al. 1998). Protein concentrations <br /> carried homologous recombination chromo-                               measured using BCA Protein Assay Reagent kit <br /> somal locus standard PCR-based methods (Longtine                                  (Pierce Chemical, Rockford, IL). Protein extracts sepa- <br /> et al. 1998). Yeast cells grown liquid YPD                            rated denaturing 10% SDS-PAGE using Mini-Protean <br /> complete medium (1% yeast extract, 2% bactopeptone, sup-                             III (BioRad). After electrophoresis, proteins were <br /> plemented 2% glucose) yeast nitrogen <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> (YNB)                           transferred nitrocellulose &#239;&#172;?lters detected Western <br /> minimal medium supplemented 2% glucose (SD), un-                                blotting polyclonal chicken anti-Abp1p antibody <br /> less stated. Tetrads dissected using Singer                         (supplied B. Goode) using enhanced chemiluminescence. <br /> Instruments MSM manual micromanipulator. Yeast cells                                     For Abp1p immunoprecipitation, 1 mg total protein ex- <br /> were transformed using lithium-acetate/SS carrier/                            tracted BY263 yeast incubated 1 ml of <br /> PEG method (Gietz Woods 2002). Yeast cells expressing                            Abp1SH3 mouse monoclonal antibody (18-A, produced by <br /> the analog-sensitive (as) alleles grown YPD media                         University Toronto monoclonal antibody facility) in <br /> 30&#194;&#176; exponential growth phase (0.5 OD600) split                           total volume 100 ml lysis buffer [50 mM Tris&#226;&#8364;&#8220;HCl (pH <br /> into equal volumes, one-half allowed grow                        7.5), 250 mM NaCl, 5 mM EDTA, 1 mM DTT, 0.1% NP-40, <br /> without treatment second half treated                           50 mM NaF, protease inhibitor cocktail (Sigma)]. After <br /> with CDK inhibitor 1NM-PP1 25 mM &#239;&#172;?nal concentration.                              incubation 4&#194;&#176; 1 hr, 15 ml 50% (v/v) Protein <br />    To recover slow-growing double mutants expressed                             A-Sepharose slurry lysis buffer added further <br /> mutant versions Abp1p combination sla1 dele-                              incubated overnight 4&#194;&#176;. The resin washed times <br /> tions, used similar strategy described                            1 ml wash buffer containing 50 mM Tris&#226;&#8364;&#8220;HCl (pH <br />  <br />  <br />  <br />                                                                                                                  Conserved Features Yeast Abp1p           1201 <br />  Table 2 Plasmids used study <br />  <br /> Plasmid                                                            Construction                                                   Source <br /> ABP1WT                        p6-1 contains 3.5-kb EcoRI fragment carrying ABP1 gene 1.5 kb                       Haynes et al. (2007) <br />                                 upstream initiator ATG codon 180 bp downstream stop <br />                                 codon subcloned pRS316DSalI <br /> abp1DADH-F                    p6-1 derivative containing ABP1 ADF-H domain deletion (47&#226;&#8364;&#8220;200 amino acids)            This study <br /> abp1D200-440                  p6-1 derivative containing ABP1 200- 440-amino-acid deletion                         This study <br /> abp1DPR\R                     p6-1 derivative containing ABP1 PRR deletion (440&#226;&#8364;&#8220;530 amino acids)                    This study <br /> abp1DSH3                      p6-1 derivative containing ABP1 SH3 domain deletion                                   Haynes et al. (2007) <br /> abp1DCFM1                     p6-1 derivative containing ABP1 217- 232-amino-acid deletion                         This study <br /> abp1DCFM2                     p6-1 derivative containing ABP1 309- 324-amino-acid deletion                         This study <br /> abp1DCFM1DADH-F               p6-1 derivative containing ABP1 deletion ADF-H domain region           This study <br />                                 comprising amino acids 217&#226;&#8364;&#8220;232 <br /> abp1DCFM2DADH-F               p6-1 derivative containing ABP1 deletion ADF-H domain region           This study <br />                               comprising amino acids 309&#226;&#8364;&#8220;324 <br /> Abp1S*T*Pro                   p6-1 derivative containing ABP1 substitutions Ser Thr residues          This study <br />                                 PRR Ala <br /> Abp1T526Pro                   p6-1 derivative containing ABP1 Thr526-to-Ala substitution                            This study <br /> Abp1S*Pro                     p6-1 derivative containing ABP1 substitutions Ser residues PRR Ala   This study <br /> pRS426-ABP1                   ABP1 overexpression plasmid, contains 3.5-kb EcoRI fragment p6-1 subcloned          This study <br />                                 pRS426 <br /> EP1-E11 (pCDC28)              CDC28 promoter terminator subcloned MAGIC plasmid                        C. Boone lab <br />  <br />  <br />  <br /> 7.5), 250 mM NaCl, 5 mM EDTA, 1 mM DTT, 0.1% NP-                         10 mM EDTA, 4 mM sodium orthovanadate, 2 mM AEBSF) <br /> 40 resuspended SDS-gel loading buffer phos-                   added. The phosphatase reaction stopped addition <br /> phatase reaction buffer [10 mM Tris&#226;&#8364;&#8220;HCl (pH 7.5), 1 M NaCl,                  SDS sample buffer, lysates analyzed Western <br /> 10 mM MgCl2, 1 mM DTT, 0.1 mM 4-(2-Aminoethyl)                           blotting. <br /> benzenesulfonyl &#239;&#172;&#8218;uoride hydrochloride (AEBSF) (Sigma)]. <br />                                                                              Spot dilution growth assays <br /> Immune complexes visualized Western blotting <br /> with anti-Abp1p antibody. In experiments, membranes                     To assay growth solid media, yeast cells grown <br /> were stripped anti-Abp1p reprobed anti-tubulin                   overnight saturation selective media supplemented <br /> antibody (Sigma).                                                            2% dextrose appropriate amino acids. Five <br />                                                                              microliters 10-fold serial dilutions equivalent cell <br /> In vivo labeling experiments <br />                                                                              concentrations (0.1 OD600nm) strain expressing <br /> For overexpression Abp1p, plasmid pRS426-ABP1                         WT mutant Abp1p variants spotted minimal <br /> transformed strain BY263. Transformants grown                      media lacking uracil (SD-URA). Growth differences were <br /> to mid-log phase SD-URA, transferred YPD gen-                  recorded following incubation plates 48 hr at <br /> erations, shifted YPD minus Pi allowed                    indicated temperature. In cases, 5 mM caffeine <br /> duplicate once. The cells pelleted concentrated 20-                 (Sigma) added emphasize growth defects. All growth <br /> fold &#239;&#172;?nal volume 5 ml YPD-Pi, 1 mCi [32P]                  assays repeated times, represen- <br /> orthophosphate (Dupont/NEN) added. After 1 hr                    tative experiment shown Figure 2. <br /> incubation shaking, cells collected centrifuga- <br />                                                                              Protein stability assays using cycloheximide <br /> tion, washed 50 mM NaF, &#239;&#172;&#8218;ash-frozen. 32P-labeled <br /> Abp1p immunoprecipitated, subjected SDS-PAGE,                         For analyses protein stability, yeast cultures grown in <br /> transferred nitrocellulose membrane, detected                    SD-URA exponential growth phase (0.5 OD600nm) di- <br /> autoradiography. The nitrocellulose membrane                     vided halves. One-half allowed continue growth, <br /> analyzed Western blotting.                                                35 mg/ml cycloheximide (Sigma) added the <br />                                                                              other. Subsequently, equal numbers cells collected after <br /> Phosphatase treatment <br />                                                                              4 hr centrifugation immediately frozen; degradation <br /> Whole-cell extracts obtained following procedure                    stopped addition 1 mM sodium azide. Proteins <br /> described using modi&#239;&#172;?ed lysis buffer [100 mM Tris                    extracted whole-cell lysates, described earlier, <br /> (pH 8.0), 100 mM NaCl, 2 mM MnCl2, 10% glycerol, 1 mM                        analyzed Western blotting using anti-Abp1p anti- <br /> DTT, protease inhibitor cocktail (Sigma)]. For phospha-                  tubulin probes. Three independent experiments carried <br /> tase treatment (Ho et al. 1999), 100 mg total proteins                    out, representative experiment shown Figure 4D. <br /> were used incubated 30 min 30&#194;&#176; total volume <br />                                                                              NMR spectroscopy <br /> of 20 ml lysis buffer presence 2 mM MnCl2 and <br /> 1600 U l-phosphatase (New England Biolabs). Where                         The extended Abp1p SH3 domain construct, included <br /> indicated, protease/phosphatase inhibitor (250 mM NaF,                   SH3 domain 28 residues lying N-terminal the <br />  <br />  <br />  <br /> 1202      B. Garcia, E. J. Stollar, A. R. Davidson <br />  Figure 2 The novel fungal conserved motif CFM1 required absence ADF-H domain. (A) Domain structure Abp1p showing CFM1 and <br /> CFM2 predicted helical regions indicated green horizontal bar. Relevant amino acid coordinates Abp1p shown beneath diagram. <br /> (B) Yeast strains bearing indicated double- gene deletions complemented plasmids expressing WT Abp1p Abp1p mutants the <br /> indicated deleted regions left. Cultures spotted serial 10-fold dilutions incubated indicated temperatures 48 hr. Caffeine <br /> was added cases emphasize growth defects. (C) Yeast cells bearing plasmids expressing mutant forms Abp1p, indicated top, were <br /> analyzed Western blotting using anti-Abp1p antibody. <br />  <br /> domain, expressed thioredoxin fusion tobacco                  Results <br /> etch virus protease cleavage site puri&#239;&#172;?ed described <br />                                                                               Conserved sequence features the <br /> previously (Stollar et al. 2009). The 1H-15N HSQC spectrum                 Abp1 family <br /> the extended construct collected using standard methods <br /> (Kay et al. 1995) 30&#194;&#176; Varian 500 MHz spectrometer                     To gain insight functionally important regions <br /> equipped pulsed &#239;&#172;?eld gradients triple resonance                    Abp1 protein family, compared sequence fea- <br /> probe actively shielded z-gradients. Data processed                 tures proteins diverse fungal higher eukary- <br /> and analyzed using NMRPipe/NMRDraw (Delaglio et al.                           otic species. As seen Figure 1A, members family <br /> 1995) NMRView (Johnson Blevins 1994). Spectra                         possess N-terminal ADF-H domain one <br /> were recorded 50 mM sodium phosphate, 100 mM NaCl,                         C-terminal SH3 domain. In addition, PRR containing least <br /> and 1 mM EDTA (pH 7.0). Using established assignments                     10% Pro upstream SH3 domains cases. <br /> of Ark1p peptide-bound free Abp1p SH3 domain (E. J.                       Using algorithm Epest&#239;&#172;?nd (http://bioweb2.pasteur.fr/ <br /> Stollar H. Lin, A. R. Davidson, J. D. Forman-Kay, unpub-                  docs/EMBOSS/epest&#239;&#172;?nd.html), able detect <br /> lished results), peptide titrations used monitor                  sequences predicted high PEST scores PRR <br /> movement peaks assign &#239;&#172;?nal position Abp1p                   regions Abp1p homologs examined. The prevalence of <br /> SH3 domain:PRR peptide complex resonances. Almost                      PEST sequences, shown mediate protein <br /> the peptide complexes exchanging fast timescale                   degradation (Rogers et al. 1986; Rechsteiner Rogers <br /> regime facilitating method assignment. The 1H-15N                     1996), Abp1 family previ- <br /> HSQC spectrum extended PRR-SH3 domain construct                        ously reported. <br /> was essentially superimposable SH3 domain:                        Since regions C-terminal ADF-H domain mouse <br /> PRR peptide complex; thus, experiments assign                  Abp1 predicted high con&#239;&#172;?dence form helices, <br /> peaks spectrum unnecessary. Amide hydro-                     searched predicted helices homologs using <br /> gen chemical shift differences calculated previously                  Jpred3 algorithm (Cole et al. 2008). We all <br /> described (Stollar et al. 2009).                                              members Abp1 family displayed predicted helices <br />  <br />  <br />  <br />                                                                                                          Conserved Features Yeast Abp1p        1203 <br />  within region (Figure 1A). Furthermore, examina-         conditions. Since second actin-binding activity <br /> tion predicted helical regions fungal homologs      identi&#239;&#172;?ed putative helical region mouse Abp1, <br /> revealed highly conserved sequence motifs, Conserved          investigated possibility CFM1 CFM2 <br /> Fungal Motifs 1 2 (CFM1 CFM2) (Figure 1B). CFM1           motifs (Figure 1), region yeast <br /> lies just amino terminal &#239;&#172;?rst conserved predicted he-      Abp1p, play functional role ADF-H do- <br /> lix, CFM2 lies predicted conserved helix (Figure     main deleted. As shown Figure 2B, combining deletion <br /> 2A). These motifs conspicuous lie           CFM1 (D217&#226;&#8364;&#8220;232) deletion fragment includ- <br /> within area proteins generally highly di-     ing ADF-H (&#206;&#8221;47&#226;&#8364;&#8220;200) domain resulted much <br /> verse sequence despite conserved presence predicted     greater reduction growth deletion region <br /> helices. The conservation CFM1 (Figure 1B) particularly     conditions ADF-H domain was <br /> striking distantly related Abp1p   essential growth. Importantly, expression levels <br /> orthologs Basidiomycete species (Ustilago, Coprinopsis,      Abp1p deletion mutants similar <br /> and Malassezia), diverged S. cerevisiae       (Figure 2C). It noted different stress condi- <br /> 500 million years ago (Taylor Berbee 2006).                   tions used assays compared de- <br />                                                                   scribed conditions best <br /> Deletion ADF-H domain reveals functional role <br />                                                                   emphasize effect CFM1 deletion. We did ob- <br /> for CFM1 <br />                                                                   serve reduction viability deletion CFM2 <br /> To assess biological roles conserved features          motif (D309&#226;&#8364;&#8220;324) combined ADF-H domain <br /> identi&#239;&#172;?ed Abp1p, constructed plasmids expressing            deletion, suggesting region possesses function <br /> mutant versions Abp1p various regions deleted.            distinct CFM1 region. <br /> The biological activity plasmid-expressed proteins <br />                                                                   Yeast Abp1p phosphorylated PRR <br /> was tested yeast genetic backgrounds which <br /> Abp1p required viability conditions. These      Since mAbp1 phosphorylated (Larbolette et al. 1999; Han <br /> backgrounds included abp1Dsla1D, unable grow          et al. 2003; Larive et al. 2009; Boateng et al. 2012) and <br /> at 30&#194;&#176;, propagated room temperature (Haynes         phosphorylated peptides yeast Abp1p have <br /> et al. 2007), abp1Dprk1D, unable grow          identi&#239;&#172;?ed large-scale studies (Ficarro et al. 2002; <br /> 37&#194;&#176; (Cope et al. 1999). We utilized strain combining       Ubersax et al. 2003; Holt et al. 2009), sought further <br /> abp1&#206;&#8221; partial deletion SLA2 (sla2&#206;&#8221;coil1),         investigate possible phosphorylation Abp1p. Interest- <br /> displays impaired growth 37&#194;&#176; (Wesp et al. 1997). Trans-        ingly, Abp1p migrates double-protein band SDS- <br /> formation mutant strains plasmid express-       PAGE gels (Drubin et al. 1988). To determine the <br /> ing wild-type ABP1 resulted robust growth cells          existence double band phosphorylation, <br /> containing vector did grow restric-        whole-yeast extracts treated l-phosphatase, and <br /> tive conditions (Figure 2B), con&#239;&#172;?rming requirement        electrophoretic mobility treated protein eval- <br /> Abp1p strain backgrounds growth conditions.          uated. As seen Figure 3A, phosphatase treatment caused <br />    As previously observed (Haynes et al. 2007), ex-      disappearance band faster mobility, implying <br /> pression Abp1p lacking SH3 domain resulted growth       band represents phosphorylated form Abp1p. <br /> that better vector conditions     When phosphatase phosphatase inhibitor added <br /> genetic backgrounds Abp1p required. By contrast,     reactions, faster band remained. To con&#239;&#172;?rm that <br /> deletion central region (residues 200&#226;&#8364;&#8220;440) PRR          band contained phosphate, cells grown 32P- <br /> region (residues 440&#226;&#8364;&#8220;530) caused growth defects com-        orthophosphate, immunoprecipitated Abp1p had <br /> pared WT conditions. Deletion residues            incorporated 32P visualized SDS-PAGE gels expo- <br /> 47&#226;&#8364;&#8220;200 (abp1DADF-H), includes conserved ADF-H           sure &#239;&#172;?lm. It seen labeled band observed <br /> domain (residues 1&#226;&#8364;&#8220;144), caused partial reduction       WT extract, extract abp1&#206;&#8221; strain <br /> viability abp1Dsla1D abp1Dprk1D backgrounds,       (Figure 3B). Furthermore, labeled band intense <br /> no growth observed abp1Dsla2Dcoil1 37&#194;&#176;. These           extracts strain carrying plasmid overexpressed <br /> results consistent study ADF-H      Abp1p. Western blotting gels demonstrated that <br /> domain essential viability abp1Dsla2D   32P-labeled bands corresponded faster-migrating <br /> strain, abp1Dsla1D strain (Quintero-Monzon et al.   band Abp1p, con&#239;&#172;?rming band phosphorylated. <br /> 2005); however, abp1Dprk1D abp1Dsla2Dcoil1              To identify region Abp1p phosphorylated, <br /> not tested previous study. We tested deletion   Western blots carried extracts abp1&#206;&#8221; strain <br /> which ADF-H domain central region deleted       carrying ABP1 mutant plasmids described above. Bands of <br /> (&#206;&#8221;47&#226;&#8364;&#8220;440), mutant poorly expressed          similar intensity detected extracts, in- <br /> meaningful growth data obtained.                     dicating various deletions did large effect <br />    The presence ADF-H domain, binds actin,         Abp1p expression stability. In addition, double band <br /> is conserved feature Abp1p homologs; thus,        detected extracts PRR deleted <br /> surprising domain fully required growth     (Figure 3C). The absence phosphorylation mutant <br />  <br />  <br />  <br /> 1204    B. Garcia, E. J. Stollar, A. R. Davidson <br />  Figure 3 Yeast Abp1p phosphorylated vivo. (A) Wild-type yeast extracts treated l-phosphatase presence absence of <br /> phosphatase inhibitor indicated. Abp1p detected Western blot analysis using polyclonal anti-Abp1p antibody. (B, left) 32P-labeled Abp1p <br /> was detected vivo labeling immunoprecipitation Abp1p WT yeast cells WT cells bearing plasmid overexpressing ABP1 (+ABP1). <br /> The abp1D strain included negative control. Proteins separated SDS-PAGE gels transferred nitrocellulose membrane analyzed <br /> by autoradiography. (Right) Western blot membrane shown left panel probed anti-Abp1p antibody. The positions of <br /> phosphorylated unphosphorylated Abp1p indicated. (C) Yeast cells bearing plasmids expressing mutant forms Abp1p, indicated, were <br /> analyzed Western blotting using anti-Abp1p antibody. This &#239;&#172;?gure composed lanes extracted exposure gel. However, <br /> lanes containing extraneous data excised. Lanes place boxes indicate lanes adjacent original gel. <br /> (D) Phosphorylation PRR mutants. The sequence Abp1p PRR shown (residues 448&#226;&#8364;&#8220;533) potential sites phosphorylation (Ser Thr <br /> residues) boldface type. Thr residue position 526 indicated. Yeast protein extracts cells bearing plasmids expressing wild-type Abp1p (WT), <br /> Abp1p lacking PRR (DPRR), Ala substitutions potential phosphosites PRR (S*T*PRR), Ser residues mutated Ala (S*PRR), or <br /> Thr526Ala substitution (T526A) treated (+) (2) l-phosphatase analyzed Western blotting using anti-Abp1p antibody. <br />  <br />  <br /> was con&#239;&#172;?rmed phosphatase treatment (Figure 3D, bot-                        phosphorylation. Thus, conclude phosphorylation of <br /> tom). Since result indicated phosphorylation site               Thr526 causes change mobility Abp1p observed in <br /> was located PRR, mutagenized Ser Thr                    SDS-PAGE. Although site clearly phosphorylated, the <br /> residues region (no Tyr residues                Thr526Ala substitution did cause detectable growth <br /> PRR) means identify speci&#239;&#172;?c site phosphoryla-                  phenotype ABP1-dependent strain backgrounds <br /> tion. As expected, simultaneous replacement Ser                   described (data shown). In addition, testing of <br /> and Thr residues PRR Ala (S*T*PRR) abro-                      phosphomimetic Thr526Glu substitution abp1&#206;&#8221;/ <br /> gated phosphorylation detected absence                       prk1&#206;&#8221; abp1&#206;&#8221;/sla2&#206;&#8221;coil1 backgrounds did reveal <br /> faster-migrating Abp1p band. The remaining band,                        growth phenotype (data shown). <br /> comigrated upper band WT Abp1p extract, <br />                                                                               Cdc28p Pho85p phosphorylate Abp1p <br /> was affected addition phosphatase. By substituting <br /> each Ser Thr residue PRR individually,                    The sequence surrounding Thr526 residue (TPEK) exactly <br /> that substitution Thr526 (T526A) resulted loss                matches CDK consensus phosphorylation site [(S/T)-P-X- <br />  <br />  <br />  <br />                                                                                                          Conserved Features Yeast Abp1p         1205 <br />  (K/R)], region predicted high proba- <br /> bility phosphorylated yeast CDKs, Cdc28p, and <br /> Pho85p using Predikin software (Saunders et al. 2008; <br /> Saunders Kobe 2008). To determine Abp1p is <br /> a target CDKs, assessed level Abp1p phos- <br /> phorylation yeast strains carrying inhibitor-sensitive alleles <br /> of CDC28 (cdc28-as) PHO85 (pho85-as). In strains, <br /> kinase activity speci&#239;&#172;?cally inhibited addition of <br /> speci&#239;&#172;?c nucleotide analogs (Carroll et al. 2001; Ubersax et al. <br /> 2003), 1NM-PP1, compound we <br /> used here. As seen Figure 4A, phosphorylation could <br /> still detected strains carrying cdc28-as pho85-as <br /> alleles own, cdc28-as/pho85-as double- <br /> mutant strain displayed detectable phosphoylated Abp1p. <br /> This result implies CDKs able phos- <br /> phorylate Abp1p Abp1p phosphorylation persists in <br /> the single-mutant strains. It somewhat surprising that <br /> the cdc28-as/pho85-as double-mutant strain displayed no <br /> Abp1p phosphorylation absence inhibitor. Since <br /> the cdc28-as mutant reduced sixfold overall vitro <br /> activity absence inhibitor (Bishop et al. 2000), we <br /> surmise loss Abp1p phosphorylation double- <br /> mutant strain result simultaneous partial reduction in <br /> the activity kinases. We pho85-as <br /> mutant de&#239;&#172;?nitely inhibited addition 1NM-PP1 <br /> (Supporting Information, Figure S1), speci&#239;&#172;?c inhibi- <br /> tion cdc28-as mutant compound pre-          Figure 4 Investigation role Cdc28p Pho85p phos- <br />                                                                     phorylation Abp1p using analog sensitive (as) mutants. (A) Wild-type <br /> viously demonstrated (Bishop et al. 2000; Zimmermann et al.         (WT), cdc28-as, pho85-as, cdc28-as/pho85-as yeast strains were <br /> 2011).                                                              grown mid-log phase (log), inhibitor suspended DMSO <br />     To directly test ability Cdc28p phosphorylate         [1NM-PP1 (+)] just DMSO (2) added. In A&#226;&#8364;&#8220;D, cell extracts were <br /> Abp1p, plasmid producing wild-type Cdc28p intro-              analyzed Western blotting using anti-Abp1p antibodies. Loading con- <br /> duced cdc28-as/pho85-as strain. In case, phos-        trols performed using antitubulin antibody. (B) Extracts cdc28- <br />                                                                     as, pho85-as, cdc28-aspho85-as yeast cells bearing vector or <br /> phorylation Abp1p partially restored (Figure 4B),            plasmid expressing wild-type Cdc28p (pCDC28) promoter <br /> proving presence Cdc28p mediate phos-               subjected Western blot analysis. (C) Yeast cells stationary phase <br /> phorylation Abp1p. The incomplete restoration Abp1p           diluted fresh YPD broth (time 0), cells grown <br /> phosphorylation strain background             30&#194;&#176;. Samples removed indicated times, cell lysates <br /> perturbed regulation plasmid-borne CDC28 gene                analyzed Western blotting anti-Abp1p antibody. (D) Yeast <br />                                                                     cells bearing plasmids expressing wild-type Abp1p (WT), Abp1p with <br /> or factors speci&#239;&#172;?c double-mutant strain. Since        Thr526-to-Ala substitution (T526A), mutant (DPRR/&#206;&#8221;PEST)&#226;&#8364;&#8221;in <br /> many Cdc28p Pho85p substrates phosphorylated                PRR, including Thr526 PEST sequences, de- <br /> in cell-cycle-dependent manner (Huang et al. 2007;                leted&#226;&#8364;&#8221;were grown absence (2) presence (+) cycloheximide <br /> Enserink Kolodner 2010), assessed cell cycle             4 hr. The images assembled gel extraneous lanes <br /> dependence Abp1p phosphorylation a-factor growth              removed. In C D, lanes extracted exposure the <br />                                                                     gel. However, lanes containing extraneous data excised. Lanes <br /> synchronization experiments. Although levels Abp1p           placed boxes indicate lanes adjacent to <br /> phosphorylation varied time release a-factor          original gel. <br /> arrest, changes did correlate cell cycle pro- <br /> gression monitored changes cellular DNA con- <br />                                                                     Novel role PRR phosphorylation <br /> tent (Figure S2). <br />                                                                     Abp1p stabilization <br />     To determine yeast protein kinases are <br /> involved Abp1p phosphorylation, strains bearing single           Since substitution Thr526 phosphorylation site Ala <br /> deletions known nonessential kinase-encoding genes               did cause detectable growth phenotype, sought to <br /> were examined. We 80 strains               uncover possible biological roles phosphoryla- <br /> displayed alteration Abp1p phosphorylation                tion. Monitoring relative levels phosphorylated and <br /> assessed Western blotting lysates exponentially            unphosphorylated Abp1p exponential growth showed <br /> growing cells. Similarly, overexpression 27 different es-        levels changed relatively greater amount <br /> sential kinases caused change degree Abp1p             unphosphorylated form early stages growth <br /> phosphorylation (Figure S3).                                        followed accumulation phosphorylated form <br />  <br />  <br />  <br /> 1206    B. Garcia, E. J. Stollar, A. R. Davidson <br />  when stationary phase reached (Figure 4C). These data           SH3 domain manner similar bound target peptide. <br /> suggested phosphorylated form stable.          We synthesized Pro-rich peptide derived <br /> To directly test idea, monitored degradation         Abp1p PRR able bind Abp1p SH3 <br /> Abp1p time inhibition protein translation      domain intermolecularly added NMR sample <br /> addition cycloheximide. Strikingly, unphosphorylated         (Figure 5A, middle). The dissociation constant inter- <br /> form Abp1p completely degraded 4 hr             action measured isothermal titration calorimetry was <br /> of incubation cycloheximide little change level     64.3 mM (E. J. Stollar, H. Lin, A. R. Davidson, J. D. <br /> of phosphorylated form seen (Figure 4D, left). Con-         Forman-Kay, unpublished results). Comparison NMR <br /> sistent result, Thr526Ala mutant,        spectrum intermolecular complex spectrum <br /> be phosphorylated, degraded 4 hr           extended SH3 domain construct revealed additional <br /> cycloheximide (Figure 4D, center). These data demonstrate           residues extended construct signi&#239;&#172;?cant chemical <br /> that phosphorylation Thr526 leads stabilization           shift changes (Figure 5A, bottom). A plot residues <br /> of Abp1p.                                                           structure Abp1p SH3 domain:peptide complex <br />     As described above, presence PEST sequences,             shows residues lie surface domain <br /> which mediate protein degradation (Rogers et al. 1986;              (Figure 5B) likely positioned path the <br /> Rechsteiner Rogers 1996), conserved feature             PRR region follow engage intramo- <br /> the PRRs fungal higher eukaryotic Abp1p ortho-          lecular interaction peptide-binding surface the <br /> logs. To determine PEST regions required          domain. <br /> for rapid degradation unphosphorylated Abp1p, the <br /> effect deletion PRR (DPRR) Abp1p stability after <br />                                                                     Discussion <br /> inhibition protein synthesis analyzed. As shown in <br /> Figure 4D (right), &#206;&#8221;PRR mutant resistant proteol-         Members Abp1 family proteins in <br /> ysis addition cycloheximide, supporting impor-         diverse fungal metazoan organisms ranging yeast <br /> tance PEST sequences mediating degradation.          humans. The high degree conservation seen among <br /> It noted Abp1p phosphorylation site           proteins indicates crucial function these <br /> within region deleted &#206;&#8221;PRR mutant; thus,             organisms. It surprising, therefore, deletion the <br /> increased stability mutant occurs spite     ABP1 gene yeast causes detectable phenotype that <br /> unphosphorylated. In summary, data indicate pro-         measurable effects ABP1 mutations seen when <br /> tein degradation mediated PEST sequences              combined gene deletions (Holtzman <br /> PRR inhibited phosphorylation Abp1p Thr526.             et al. 1993; Lila Drubin 1997; Cope et al. 1999; Haynes <br />                                                                     et al. 2007). These &#239;&#172;?ndings emphasize deletion func- <br /> Abp1p SH3 domain potentially form <br />                                                                     tionally important proteins regions proteins not <br /> an intramolecular interaction PRR <br />                                                                     cause detectable phenotype tested under <br /> Since Abp1p SH3 domain lies immediately adjacent             laboratory conditions. Thus, uncovering roles these <br /> the PRR region, hypothesized intramolecular in-          conserved regions require phenotypic assays car- <br /> teraction regions play role            ried wide variety growth conditions ge- <br /> regulating Abp1p activity. To test idea, expressed          netic backgrounds. The results presented work clearly <br /> and puri&#239;&#172;?ed version Abp1p SH3 domain in-              demonstrate dif&#239;&#172;?culties associated assigning func- <br /> cluded 28 residues N-terminal start SH3 do-           tions conserved features yeast Abp1p, show <br /> main. This region includes PXXP motifs              thorough investigation reveal requirements for <br /> phosphorylation site Thr526. To assess ability            regions &#239;&#172;?rst appear dispensable. <br /> this region interact SH3 domain, collected               Through sequence alignment analysis, discovered two <br /> an 1H-15N HSQC NMR spectrum construct com-             highly conserved motifs, CFM1 CFM2, fungal <br /> pared spectra previously collected free peptide-      Abp1p orthologs. Given high conservation these <br /> bound forms Abp1p SH3 domain (Stollar et al. 2009).          motifs, striking deletion central <br /> In type spectrum, cross-peak observed cor-         portion Abp1p (&#206;&#8221;200&#226;&#8364;&#8220;440), includes motifs, <br /> responding amide group protein. The posi-            resulted absolutely reduction viability strain <br /> tions cross-peaks (i.e., chemical shifts)   backgrounds growth conditions tested (Figure 2). We <br /> sensitive chemical environment; thus, amide groups           were, however, able uncover function CFM1 de- <br /> in residues close peptide-binding surface display sig-       leting motif context Abp1p lacking ADF- <br /> ni&#239;&#172;?cant chemical shift changes peptide binding. It         H domain. This discovery implies CFM1 able ful&#239;&#172;?ll <br /> be seen Figure 5A similar changes chemical shifts        function ADF-H domain. One possibility is <br /> are seen comparing free Abp1p SH3 domain            CFM1 binds actin, similarly charged helical sequence <br /> Ark1p peptide-bound domain (top) extended                 mAbp1 (Larbolette et al. 1999; Kessels et al. 2000). <br /> Abp1p SH3 domain construct (bottom). These data indicate            It interact protein associated <br /> that PRR extended construct interacts           actin. We expect potential direct indirect <br />  <br />  <br />  <br />                                                                                           Conserved Features Yeast Abp1p   1207 <br />                                                                                                      Figure 5 The PRR binds the <br />                                                                                                     Abp1p SH3 domain. (A) The back- <br />                                                                                                     bone amide chemical shift differen- <br />                                                                                                     ces unbound Abp1p <br />                                                                                                     SH3 domain Abp1p SH3 <br />                                                                                                     domain bound high-af&#239;&#172;?nity tar- <br />                                                                                                     peptide derived Ark1p <br />                                                                                                     (top) (Stollar et al. 2009) the <br />                                                                                                     PRR-derived peptide (middle) are <br />                                                                                                     shown. The sequence PRR <br />                                                                                                     peptide used shown middle. <br />                                                                                                     The shows chemical shift <br />                                                                                                     differences unbound <br />                                                                                                     Abp1p SH3 domain ex- <br />                                                                                                     tended SH3 domain construct. The <br />                                                                                                     sequence PRR region attached <br />                                                                                                     SH3 domain extended <br />                                                                                                     construct shown. The residues <br />                                                                                                     highlighted blue that <br />                                                                                                     signi&#239;&#172;?cantly differ ex- <br />                                                                                                     tended construct Abp1p <br />                                                                                                     SH3 domain:PRR peptide intermo- <br />                                                                                                     lecular complex. (B) Amide groups <br />                                                                                                     signi&#239;&#172;?cant chemical shift <br />                                                                                                     differences PRR pep- <br />                                                                                                     tide-bound Abp1p SH3 domain in- <br />                                                                                                     termolecular complex the <br />                                                                                                     extended SH3 domain indicated <br />                                                                                                     blue spheres structure of <br />                                                                                                     Abp1p SH3 domain:Ark1p pep- <br />                                                                                                     tide complex. The dotted line shows <br />                                                                                                     possible path connecting do- <br />                                                                                                     main peptide be <br />                                                                                                     intramolecular complex <br />                                                                                                     indicates Thr526 lie <br />                                                                                                     region. The structure on <br />                                                                                                     right rotated 90&#194;&#176; rela- <br />                                                                                                     tive the left. The <br />                                                                                                     Abp1p SH3 domain shown gray <br />                                                                                                     Ark1p peptide green. <br />  <br />  <br /> association CFM1 actin weak           conditions (Nishizawa et al. 1998; Ptacek et al. <br /> deletion ADF-H domain observed abrogate lo-          2005; Huang et al. 2007, 2009; Sopko et al. 2007; Enserink <br /> calization Abp1p cortical actin patches assessed       Kolodner 2010). While sites Cdc28p-mediated <br /> GFP localization experiments (Quintero-Monzon et al. 2005).        phosphorylation detected Abp1p through <br /> An important conclusion studies CFM1 that,          large-scale phosphoproteomic studies (Ficarro et al. 2002; <br /> even motif highly conserved, function        Ubersax et al. 2003; Holt et al. 2009), Thr526 &#239;&#172;?rst <br /> become apparent unless protein sensitized          phosphorylation site Abp1p experimentally ver- <br /> perturbations, deletion ADF-H domain        i&#239;&#172;?ed mutagenesis. We exclude possibil- <br /> case. This type &#226;&#8364;&#339;intragenic synthetic lethality&#226;&#8364;? re-   ity residues Abp1p phosphorylated <br /> cognized handful studies (Teixeira et al. 1997;      phosphorylation events insensitive <br /> Kou Pugh 2004; Murphy et al. 2004; Dias et al. 2008;           phosphatase affect electrophoretic mobility. <br /> Garrey Mackie 2011), systematic searches intra-        Our studies Abp1p proteolysis indicate role Abp1p <br /> genic synthetic lethality carried out. Such          phosphorylation phosphorylated form Abp1p is <br /> experiments provide generally applicable approach          considerably resistant proteolysis unphos- <br /> for uncovering roles conserved regions multi-        phorylated form (Figure 4D). Consistent increased <br /> domain proteins.                                                   stability phosphorylated Abp1p, observed the <br />    Our work illuminated potentially impor-          relative level unphosphorylated Abp1p peaks mid-log <br /> tant features Abp1p. We demonstrated Abp1p            phase novo synthesis highest (Figure 4C). At later <br /> is phosphorylated Thr526 Cdc28p Pho85p               growth stages, Abp1p phosphorylated degraded, <br /> kinases (Figure 4), recognize consensus             Abp1p present cell stationary phase is <br /> phosphorylation sites functionally interchangeable         phosphorylated. We showed Abp1p proteolysis <br />  <br />  <br />  <br /> 1208    B. Garcia, E. J. Stollar, A. R. Davidson <br />  results presence PEST sequences PRR        taining deletions conserved regions pro- <br /> are located near site phosphorylation (Figure 1).           tein provide means achieve goal. <br /> Thus, phosphorylation Thr526 appears provide means <br /> of regulating proteolysis mediated nearby PEST se- <br />                                                                    Acknowledgments <br /> quences. Since PEST sequences conserved feature of <br /> Abp1p homologs fungal mammalian species                 We thank Brenda Andrews Charlie Boone plasmids and <br /> phosphorylation PRR seen mouse (Larbolette        yeast strains; Bruce Goode anti-Abp1p antibody; Karen <br /> et al. 1999; Han et al. 2003; Zanivan et al. 2008; Larive et al.   Maxwell critical reading manuscript. This work was <br /> 2009; Boateng et al. 2012) human Abp1 (Olsen et al.            supported operating grant Canadian Institutes <br /> 2006; Molina et al. 2007), propose regulation      Health Research (CIHR ARD grant MOP-13609 A.R.D.). <br /> Abp1p stability conserved feature family. It      E.J.S. supported grants National Center for <br /> is possible putative intramolecular interaction be-       Research Resources (5P20RR016480-12) National <br /> tween SH3 domain PRR, supported            Institute General Medical Sciences (8P20GM103451-12) of <br /> our NMR data, affected phosphorylation Thr526             National Institutes Health. <br /> since site lies SH3 domain PRR <br /> (Figure 5B). This intramolecular interaction play role <br /> in regulating Abp1p stability modulating acces-        Literature Cited <br /> sibility PRR binding partners. In mammalian <br />                                                                    Bishop, A. C., J. A. Ubersax, D. T. Petsch, D. P. Matheos, N. S. Gray <br /> cortactin, protein similar domain layout function        et al., 2000 A chemical switch inhibitor-sensitive alleles of <br /> as Abp1p (Olazabal Machesky 2001), phosphorylation                protein kinase. Nature 407: 395&#226;&#8364;&#8220;401. <br /> has suggested modulate intramolecular in-         Boateng, L. R., C. L. Cortesio, A. Huttenlocher, 2012 Src-me- <br /> teraction C-terminal SH3 domain upstream             diated phosphorylation mammalian Abp1 (DBNL) regulates <br /> PRR (Lua Low 2005).                                               podosome rosette formation transformed &#239;&#172;?broblasts. J. Cell <br />                                                                       Sci. 125: 1329&#226;&#8364;&#8220;1341. <br />     The investigation Abp1p provides instructive exam-       Brachmann, C. B., A. Davies, G. J. Cost, E. Caputo, J. Li et al., <br /> ple protein highly conserved evolution,           1998 Designer deletion strains derived Saccharomyces <br /> deletion gene encoding yeast results de-           cerevisiae S288C: useful set strains plasmids PCR- <br /> tectable growth defects laboratory conditions. The              mediated gene disruption applications. Yeast 14: <br /> presence Abp1p orthologs divergent species implies              115&#226;&#8364;&#8220;132. <br />                                                                    Carroll, A. S., A. C. Bishop, J. L. DeRisi, K. M. Shokat, E. K. <br /> that organisms lacking protein certainly display           O&#226;&#8364;&#8482;Shea, 2001 Chemical inhibition Pho85 cyclin-depen- <br /> a decreased evolutionary &#239;&#172;?tness. Clearly, requirements            dent kinase reveals role environmental stress response. <br /> for <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>survival</span> natural environment hundreds              Proc. Natl. Acad. Sci. USA 98: 12578&#226;&#8364;&#8220;12583. <br /> millions years stringent         Chi, A., C. Huttenhower, L. Y. Geer, J. J. Coon, J. E. Syka et al., <br /> in laboratory. For reason, detection phenotypes           2007 Analysis phosphorylation sites proteins Sac- <br />                                                                       charomyces cerevisiae electron transfer dissociation (ETD) <br /> for ABP1 deletions laboratory requires additional ma-          mass spectrometry. Proc. Natl. Acad. Sci. USA 104: 2193&#226;&#8364;&#8220;2198. <br /> nipulation genetic backgrounds growth conditions            Cole, C., J. D. Barber, G. J. Barton, 2008 The Jpred 3 sec- <br /> (Lila Drubin 1997; Fazi et al. 2002; Stefan et al. 2005;          ondary structure prediction server. Nucleic Acids Res. 36: <br /> Haynes et al. 2007; Stollar et al. 2009). In work,            W197&#226;&#8364;&#8220;W201. <br /> show deletions conserved domains motifs             Cope, M. J., S. Yang, C. Shang, D. G. Drubin, 1999 Novel <br />                                                                       protein kinases Ark1p Prk1p associate regulate the <br /> Abp1p, functionally important,              cortical actin cytoskeleton budding yeast. J. Cell Biol. 144: <br /> always lead detectable growth phenotypes,              1203&#226;&#8364;&#8220;1218. <br /> manipulation conditions reveal phenotypes. In         Cortesio, C. L., B. J. Perrin, D. A. Bennin, A. Huttenlocher, <br /> some cases, multiple conserved regions protein           2010 Actin-binding protein-1 interacts WASp-interacting <br /> have deleted phenotype emerges ob-               protein regulate growth factor-induced dorsal ruf&#239;&#172;&#8218;e forma- <br />                                                                       tion. Mol. Biol. Cell 21: 186&#226;&#8364;&#8220;197. <br /> served case CFM1 motif. Detection       Delaglio, F., S. Grzesiek, G. W. Vuister, G. Zhu, J. Pfeifer et al., <br /> type intragenic synthetic lethality important          1995 NMRPipe: multidimensional spectral processing sys- <br /> new addition repertoire systematic approaches          tem based UNIX pipes. J. Biomol. NMR 6: 277&#226;&#8364;&#8220;293. <br /> are generally employed uncovering protein functions.           Dias, C. A., V. S. Cano, S. M. Rangel, L. H. Apponi, M. C. Frigieri <br /> Due frequent dif&#239;&#172;?culty establishing conditions           et al., 2008 Structural modeling mutational analysis of <br />                                                                       yeast eukaryotic translation initiation factor 5A reveal new crit- <br /> the laboratory reveal phenotypes highly conserved        ical residues reinforce involvement protein synthesis. <br /> protein regions, suggest detection conserva-           FEBS J. 275: 1874&#226;&#8364;&#8220;1888. <br /> tion reliable approach identifying functionally      Drubin, D. G., K. G. Miller, D. Botstein, 1988 Yeast actin- <br /> important regions protein identi&#239;&#172;?cation           binding proteins: evidence role morphogenesis. J. Cell <br /> phenotypes resulting mutations. The future challenge             Biol. 107: 2551&#226;&#8364;&#8220;2561. <br />                                                                    Enserink, J. M., R. D. Kolodner, 2010 An overview Cdk1- <br /> is detect phenotypes associated protein regions          controlled targets processes. Cell Div. 5: 11. <br /> are strongly indicated important conservation        Fazi, B., M. J. Cope, A. Douangamath, S. Ferracuti, K. Schirwitz <br /> analysis. High-throughput genetics studies strains con-            et al., 2002 Unusual binding properties SH3 domain <br />  <br />  <br />  <br />                                                                                             Conserved Features Yeast Abp1p      1209 <br />     yeast actin-binding protein Abp1: structural func-         Larbolette, O., B. Wollscheid, J. Schweikert, P. J. Nielsen, J. <br />    tional analysis. J. Biol. Chem. 277: 5290&#226;&#8364;&#8220;5298.                           Wienands, 1999 SH3P7 cytoskeleton adapter protein and <br /> Fenster, S. D., M. M. Kessels, B. Qualmann, W. J. Chung, J. Nash             coupled signal transduction lymphocyte antigen re- <br />    et al., 2003 Interactions Piccolo actin/dyna-             ceptors. Mol. Cell. Biol. 19: 1539&#226;&#8364;&#8220;1546. <br />    min-binding protein Abp1 link vesicle endocytosis presynaptic      Larive, R. M., S. Urbach, J. Poncet, P. Jouin, G. Mascre et al., <br />    active zones. J. Biol. Chem. 278: 20268&#226;&#8364;&#8220;20277.                            2009 Phosphoproteomic analysis Syk kinase signaling in <br /> Ficarro, S. B., M. L. McCleland, P. T. Stukenberg, D. J. Burke, M. M.        human cancer cells reveals role cell-cell adhesion. Onco- <br />    Ross et al., 2002 Phosphoproteome analysis mass spectrom-              gene 28: 2337&#226;&#8364;&#8220;2347. <br />    etry application Saccharomyces cerevisiae. Nat. Bio-       Lee, J., K. Colwill, V. Aneliunas, C. Tennyson, L. Moore et al., <br />    technol. 20: 301&#226;&#8364;&#8220;305.                                                     1998 Interaction yeast Rvs167 Pho85 cyclin-dependent <br /> Garrey, S. M., G. A. Mackie, 2011 Roles 59-phosphate              kinase complexes link cell cycle actin cytoskele- <br />    sensor domain RNase E. Mol. Microbiol. 80: 1613&#226;&#8364;&#8220;1624.                  ton. Curr. Biol. 8: 1310&#226;&#8364;&#8220;1321. <br /> Gietz, R. D., R. A. Woods, 2002 Transformation yeast           Lila, T., D. G. Drubin, 1997 Evidence physical func- <br />    lithium acetate/single-stranded carrier DNA/polyethylene gly-             tional interactions Saccharomyces cerevisiae SH3 <br />    col method. Methods Enzymol. 350: 87&#226;&#8364;&#8220;96.                                  domain proteins, adenylyl cyclase-associated protein and <br /> Goode, B. L., A. A. Rodal, G. Barnes, D. G. Drubin, 2001 Acti-           actin cytoskeleton. Mol. Biol. Cell 8: 367&#226;&#8364;&#8220;385. <br />    vation Arp2/3 complex actin &#239;&#172;?lament binding pro-        Longtine, M. S., A. McKenzie III. D. J. Demarini, N. G. Shah, A. <br />    tein Abp1p. J. Cell Biol. 153: 627&#226;&#8364;&#8220;634.                                   Wach et al., 1998 Additional modules versatile eco- <br /> Han, J., R. Kori, J. W. Shui, Y. R. Chen, Z. Yao et al., 2003 The SH3        nomical PCR-based gene deletion modi&#239;&#172;?cation Saccha- <br />    domain-containing adaptor HIP-55 mediates c-Jun N-terminal ki-            romyces cerevisiae. Yeast 14: 953&#226;&#8364;&#8220;961. <br />    nase activation T cell receptor signaling. J. Biol. Chem. 278:     Lua, B. L., B. C. Low, 2005 Cortactin phosphorylation as <br />    52195&#226;&#8364;&#8220;52202.                                                              switch actin cytoskeletal network cell dynamics con- <br /> Haynes, J., B. Garcia, E. J. Stollar, A. Rath, B. J. Andrews et al.,         trol. FEBS Lett. 579: 577&#226;&#8364;&#8220;585. <br />    2007 The biologically relevant targets binding af&#239;&#172;?nity re-        Measday, V., L. Moore, J. Ogas, M. Tyers, B. Andrews, <br />    quirements function yeast actin-binding protein 1          1994 The PCL2 (ORFD)-PHO85 cyclin-dependent kinase com- <br />    Src-homology 3 domain vary genetic context. Genetics                 plex: cell cycle regulator yeast. Science 266: 1391&#226;&#8364;&#8220;1395. <br />    176: 193&#226;&#8364;&#8220;208.                                                         Mise-Omata, S., B. Montagne, M. Deckert, J. Wienands, O. <br /> Ho, Y., M. Costanzo, L. Moore, R. Kobayashi, B. J. Andrews,              Acuto, 2003 Mammalian actin binding protein 1 essential <br />    1999 Regulation transcription Saccharomyces cerevi-             endocytosis lamellipodia formation: functional anal- <br />    siae start transition Stb1, Swi6-binding protein. Mol. Cell.         ysis <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>RNA interference</span>. Biochem. Biophys. Res. Commun. 301: <br />    Biol. 19: 5267&#226;&#8364;&#8220;5278.                                                      704&#226;&#8364;&#8220;710. <br /> Holt, L. J., B. B. Tuch, J. Villen, A. D. Johnson, S. P. Gygi et al.,    Molina, H., D. M. Horn, N. Tang, S. Mathivanan, A. Pandey, <br />    2009 Global analysis Cdk1 substrate phosphorylation sites              2007 Global proteomic pro&#239;&#172;?ling phosphopeptides using <br />    provides insights evolution. Science 325: 1682&#226;&#8364;&#8220;1686.                 electron transfer dissociation tandem mass spectrometry. Proc. <br /> Holtzman, D. A., S. Yang, D. G. Drubin, 1993 Synthetic-lethal            Natl. Acad. Sci. USA 104: 2199&#226;&#8364;&#8220;2204. <br />    interactions identify novel genes, SLA1 SLA2, con-       Murphy, M. W., B. L. Olson, P. G. Siliciano, 2004 The yeast <br />    trol membrane cytoskeleton assembly Saccharomyces cerevi-              splicing factor Prp40p contains functional leucine-rich nuclear <br />    siae. J. Cell Biol. 122: 635&#226;&#8364;&#8220;644.                                         export signals essential splicing. Genetics 166: 53&#226;&#8364;&#8220;65. <br /> Huang, D., H. Friesen, B. Andrews, 2007 Pho85, multifunc-          Nishizawa, M., M. Kawasumi, M. Fujino, A. Toh-e, <br />    tional cyclin-dependent protein kinase budding yeast. Mol.             1998 Phosphorylation sic1, cyclin-dependent kinase <br />    Microbiol. 66: 303&#226;&#8364;&#8220;314.                                                   (Cdk) inhibitor, Cdk including Pho85 kinase required for <br /> Huang, D., S. Kaluarachchi, D. van Dyk, H. Friesen, R. Sopko et al.,         prompt degradation. Mol. Biol. Cell 9: 2393&#226;&#8364;&#8220;2405. <br />    2009 Dual regulation pairs cyclin-dependent protein             Olazabal, I. M., L. M. Machesky, 2001 Abp1p cortactin, <br />    kinases histone deacetylases controls G1 transcription             new &#226;&#8364;&#339;hand-holds&#226;&#8364;? actin. J. Cell Biol. 154: 679&#226;&#8364;&#8220;682. <br />    budding yeast. PLoS Biol. 7: e1000188.                                Olsen, J. V., B. Blagoev, F. Gnad, B. Macek, C. Kumar et al., <br /> Johnson, B. A., R. A. Blevins, 1994 NMRView:                  2006 Global, vivo, site-speci&#239;&#172;?c phosphorylation dynam- <br />    program visualization analysis NMR data. J. Bio-           ics signaling networks. Cell 127: 635&#226;&#8364;&#8220;648. <br />    mol. NMR 4: 603&#226;&#8364;&#8220;614.                                                  Ptacek, J., G. Devgan, G. Michaud, H. Zhu, X. Zhu et al., <br /> Kay, L. E., P. Keifer, T. Saarinen, 1995 Pure absorption gra-            2005 Global analysis protein phosphorylation yeast. Na- <br />    dient enhanced hetero-nuclear single quantum correlation                  ture 438: 679&#226;&#8364;&#8220;684. <br />    spectroscopy improved sensitivity. J. Am. Chem. Soc. 114:        Quintero-Monzon, O., A. A. Rodal, B. Strokopytov, S. C. Almo, and <br />    10663&#226;&#8364;&#8220;10665.                                                              B. L. Goode, 2005 Structural functional dissection the <br /> Kessels, M. M., A. E. Engqvist-Goldstein, D. G. Drubin,                  Abp1 ADFH actin-binding domain reveals versatile vivo <br />    2000 Association mouse actin-binding protein 1 (mAbp1/                 adapter functions. Mol. Biol. Cell 16: 3128&#226;&#8364;&#8220;3139. <br />    SH3P7), Src kinase target, dynamic regions cor-        Rechsteiner, M., S. W. Rogers, 1996 PEST sequences and <br />    tical actin cytoskeleton response Rac1 activation. Mol. Biol.       regulation proteolysis. Trends Biochem. Sci. 21: 267&#226;&#8364;&#8220;271. <br />    Cell 11: 393&#226;&#8364;&#8220;412.                                                     Rogers, S., R. Wells, M. Rechsteiner, 1986 Amino acid se- <br /> Kessels, M. M., A. E. Y. Engqvist-Goldstein, D. G. Drubin, B.            quences common rapidly degraded proteins: PEST hy- <br />    Qualmann, 2001 Mammalian Abp1, signal-responsive F-ac-                  pothesis. Science 234: 364&#226;&#8364;&#8220;368. <br />    tin-binding protein, links actin cytoskeleton endocytosis      Saunders, N. F., B. Kobe, 2008 The Predikin webserver: im- <br />    GTPase dynamin. J. Cell Biol. 153: 351&#226;&#8364;&#8220;366.                       proved prediction protein kinase peptide speci&#239;&#172;?city using <br /> Kou, H., B. F. Pugh, 2004 Engineering dimer-stabilizing mu-              structural information. Nucleic Acids Res. 36: W286&#226;&#8364;&#8220;W290. <br />    tations TATA-binding protein. J. Biol. Chem. 279: 20966&#226;&#8364;&#8220;       Saunders, N. F., R. I. Brinkworth, T. Huber, B. E. Kemp, B. <br />    20973.                                                                    Kobe, 2008 Predikin PredikinDB: computational frame- <br /> Lappalainen, P., M. M. Kessels, M. J. Cope, D. G. Drubin,                work prediction protein kinase peptide speci&#239;&#172;?city and <br />    1998 The ADF homology (ADF-H) domain: highly exploited                  associated database phosphorylation sites. BMC Bioinfor- <br />    actin-binding module. Mol. Biol. Cell 9: 1951&#226;&#8364;&#8220;1959.                       matics 9: 245. <br />  <br />  <br />  <br /> 1210     B. Garcia, E. J. Stollar, A. R. Davidson <br />  Sopko, R., D. Huang, J. C. Smith, D. Figeys, B. J. Andrews,          Ubersax, J. A., E. L. Woodbury, P. N. Quang, M. Paraz, J. D. Bleth- <br />    2007 Activation Cdc42p GTPase cyclin-dependent                 row et al., 2003 Targets cyclin-dependent kinase Cdk1. <br />    protein kinases budding yeast. EMBO J. 26: 4487&#226;&#8364;&#8220;4500.                 Nature 425: 859&#226;&#8364;&#8220;864. <br /> Stefan, C. J., S. M. Padilla, A. Audhya, S. D. Emr, 2005 The         Wesp, A., L. Hicke, J. Palecek, R. Lombardi, T. Aust et al., <br />    phosphoinositide phosphatase Sjl2 recruited cortical actin         1997 End4p/Sla2p interacts actin-associated proteins <br />    patches control vesicle formation &#239;&#172;?ssion            endocytosis Saccharomyces cerevisiae. Mol. Biol. Cell 8: <br />    endocytosis. Mol. Cell. Biol. 25: 2910&#226;&#8364;&#8220;2923.                             2291&#226;&#8364;&#8220;2306. <br /> Stollar, E. J., B. Garcia, P. A. Chong, A. Rath, H. Lin et al.,          Zanivan, S., F. Gnad, S. A. Wickstrom, T. Geiger, B. Macek et al., <br />    2009 Structural, functional, bioinformatic studies demon-            2008 Solid tumor proteome phosphoproteome analysis <br />    strate crucial role extended peptide binding site      high resolution mass spectrometry. J. Proteome Res. 7: <br />    SH3 domain yeast Abp1p. J. Biol. Chem. 284: 26918&#226;&#8364;&#8220;26927.              5314&#226;&#8364;&#8220;5326. <br /> Taylor, J. W., M. L. Berbee, 2006 Dating divergences          Zimmermann, C., P. Chymkowitch, V. Eldholm, C. D. Putnam, J. M. <br />    Fungal Tree Life: review new analyses. Mycologia 98:              Lindvall et al., 2011 A chemical-genetic screen unravel the <br />    838&#226;&#8364;&#8220;849.                                                                 genetic network CDC28/CDK1 links ubiquitin Rad6-Bre1 <br /> Teixeira, M. T., S. Siniossoglou, S. Podtelejnikov, J. C. Benichou, M.      cell cycle progression. Proc. Natl. Acad. Sci. USA 108: 18748&#226;&#8364;&#8220; <br />    Mann et al., 1997 Two functionally distinct domains gener-               18753. <br />    ated vivo cleavage Nup145p: novel biogenesis path- <br />    way nucleoporins. EMBO J. 16: 5086&#226;&#8364;&#8220;5097.                                                               Communicating editor: D. Lew <br />  <br />  <br />  <br />  <br />                                                                                                  Conserved Features Yeast Abp1p      1211 <br />               GENETICS <br />                                                               Supporting Information <br />              http://www.genetics.org/content/suppl/2012/06/01/genetics.112.141739.DC1 <br />  <br />  <br />  <br />  <br />   The Importance Conserved Features Yeast <br /> Actin-Binding Protein 1 (Abp1p): The Conditional <br />                            Nature Essentiality <br />                                   Bianca Garcia, Elliott J. Stollar, Alan R. Davidson <br />  <br />  <br />  <br />  <br />                                                 Copyright &#194;&#169; 2012 Genetics Society America <br />                                                                 DOI: 10.1534/genetics.112.141739 <br />  	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160;                                               phase*contrast#images##                                    Pho4p*GFP# <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160;                            oh# <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160;                      DMSO# <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160;              1NM*PP1# <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> Figure	 <br />  &#194;&#160;S1	 <br />  &#194;&#160;	 <br />  &#194;&#160;	 <br />  &#194;&#160;The	 <br />  &#194;&#160;direct	 <br />  &#194;&#160;Pho85	 <br />  &#194;&#160;target	 <br />  &#194;&#160;,	 <br />  &#194;&#160;Pho4,	 <br />  &#194;&#160;is	 <br />  &#194;&#160;desphosphorylated	 <br />  &#194;&#160;upon	 <br />  &#194;&#160;Pho85	 <br />  &#194;&#160;inhibition	 <br />  &#194;&#160;with	 <br />  &#194;&#160;1NM-&#194;&#173;&#226;&#8364;?PP1	 <br />  &#194;&#160;and	 <br />  &#194;&#160;shuttled	 <br />  &#194;&#160; <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>	 <br />  &#194;&#160;the	 <br />  &#194;&#160;cytoplasm	 <br />  &#194;&#160;to	 <br />  &#194;&#160;the	 <br />  &#194;&#160;nucleus.	 <br />  &#194;&#160;pho85-&#194;&#173;&#226;&#8364;?as/cdc28as	 <br />  &#194;&#160;strain	 <br />  &#194;&#160;was	 <br />  &#194;&#160;transformed	 <br />  &#194;&#160;with	 <br />  &#194;&#160;an	 <br />  &#194;&#160;expression	 <br />  &#194;&#160;plasmid	 <br />  &#194;&#160;containing	 <br />  &#194;&#160;a	 <br />  &#194;&#160; <br /> Pho4p-&#194;&#173;&#226;&#8364;?GFP	 <br />  &#194;&#160;fusion	 <br />  &#194;&#160;construct	 <br />  &#194;&#160;under	 <br />  &#194;&#160;the	 <br />  &#194;&#160;control	 <br />  &#194;&#160;of	 <br />  &#194;&#160;the	 <br />  &#194;&#160;ADH1	 <br />  &#194;&#160;promoter	 <br />  &#194;&#160;(BA1828,	 <br />  &#194;&#160;B.	 <br />  &#194;&#160;Andrews	 <br />  &#194;&#160;lab).	 <br />  &#194;&#160;Cells	 <br />  &#194;&#160;were	 <br />  &#194;&#160;grown	 <br />  &#194;&#160;to	 <br />  &#194;&#160; <br /> an	 <br />  &#194;&#160;OD600	 <br />  &#194;&#160;of	 <br />  &#194;&#160;0.5	 <br />  &#194;&#160;(0h)	 <br />  &#194;&#160;and	 <br />  &#194;&#160;treated	 <br />  &#194;&#160;with	 <br />  &#194;&#160;DMSO	 <br />  &#194;&#160;or	 <br />  &#194;&#160;25	 <br />  &#194;&#160;&#206;&#188;M	 <br />  &#194;&#160;1NM-&#194;&#173;&#226;&#8364;?PP1.	 <br />  &#194;&#160;Cellular	 <br />  &#194;&#160;localization	 <br />  &#194;&#160;of	 <br />  &#194;&#160;Pho4p	 <br />  &#194;&#160;was	 <br />  &#194;&#160;analyzed	 <br />  &#194;&#160;using	 <br />  &#194;&#160; <br /> fluorescence	 <br />  &#194;&#160;microscopy	 <br />  &#194;&#160;at	 <br />  &#194;&#160;15	 <br />  &#194;&#160;min	 <br />  &#194;&#160;after	 <br />  &#194;&#160;treatment.	 <br />  &#194;&#160;Representative	 <br />  &#194;&#160;fields	 <br />  &#194;&#160;are	 <br />  &#194;&#160;shown.	 <br />  &#194;&#160;	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160; <br /> 	 <br />  &#194;&#160;                                                               	 <br />  &#194;&#160; <br />  <br />  <br />  <br />  <br /> 2	 <br />  &#194;&#160;SI	 <br />  &#194;&#160;                                               B.	 <br />  &#194;&#160;Garcia,	 <br />  &#194;&#160;E.	 <br />  &#194;&#160;J.	 <br />  &#194;&#160;Stollar,	 <br />  &#194;&#160;and	 <br />  &#194;&#160;A.	 <br />  &#194;&#160;Davidson	 <br />  &#194;&#160; <br />                  	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br /> A#              	 <br />  &#194;&#160;                                                              B# <br />                 	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />  <br />  <br />  <br />  <br />                                                                                     arrest# <br />                                                                                                                                      &#206;&#177;F#release#(min)# <br />  <br />  <br />  <br />  <br />                                                                                       log# <br />                 	 <br />  &#194;&#160; <br />      t#(min)#   	 <br />  &#194;&#160;                                                                                     0#     15#     30#    45# 60# 75# 90# 105# 120# 135# 150#165# 180# <br />                 	 <br />  &#194;&#160; <br />      180#       	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160;                                                                                                                                                                             unphosphorylated#Abp1# <br />      165#       	 <br />  &#194;&#160;                                                                                                                                                                             phosphorylated#Abp1# <br />                 	 <br />  &#194;&#160; <br />      150#       	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />      135#                                                                                        Abp1p#probe# <br />                 	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />      120# <br />                 	 <br />  &#194;&#160; <br />      105#       	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />      90#        	 <br />  &#194;&#160; <br />                                                                                                                                                                                                    Coronin# <br />                 	 <br />  &#194;&#160; <br />      75#        	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />      60#        	 <br />  &#194;&#160;                                                                             coronin#probe# <br />                 	 <br />  &#194;&#160; <br />      45#        	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />      30# <br />                 	 <br />  &#194;&#160; <br />      15#        	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />      0#         	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; 1N#                     2N# <br />                 	 <br />  &#194;&#160; <br />                 	 <br />  &#194;&#160; <br />                 Figure	 <br />  &#194;&#160;S2	 <br />  &#194;&#160;	 <br />  &#194;&#160;	 <br />  &#194;&#160;	 <br />  &#194;&#160;Abp1p	 <br />  &#194;&#160;expression	 <br />  &#194;&#160;is	 <br />  &#194;&#160;not	 <br />  &#194;&#160;cell	 <br />  &#194;&#160;cycle	 <br />  &#194;&#160;regulated.	 <br />  &#194;&#160;(A)	 <br />  &#194;&#160;Flow	 <br />  &#194;&#160;cytometr <br /> </body></html>