<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mitochondrial Changes Ageing Caenorhabditis <br /> elegans &#226;&#8364;&#8220; What Do We Learn Superoxide Dismutase <br /> Knockouts? <br /> Jan Gruber1, Li Fang Ng1, Sheng Fong1, Yee Ting Wong1, Soon Ann Koh1, Ce-Belle Chen1, Guanghou <br /> Shui2, Wei Fun Cheong2, Sebastian Schaffer1, Markus R. Wenk1,2, Barry Halliwell1* <br /> 1 Department Biochemistry, Yong Loo Lin School Medicine, National University Singapore, Singapore, Singapore, 2 Department Biological Sciences, Faculty of <br /> Science, National University Singapore, Singapore, Singapore <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      One popular damage accumulation theories ageing mitochondrial free radical theory ageing <br />      (mFRTA). The mFRTA proposes ageing accumulation unrepaired oxidative damage, particular <br />      damage mitochondrial DNA (mtDNA). Within mFRTA, &#226;&#8364;&#732;&#226;&#8364;&#732;vicious cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; theory proposes reactive oxygen <br />      species (ROS) promote mtDNA mutations, lead increase ROS production. Recently, data been <br />      published Caenorhabditis elegans mutants deficient forms mitochondrial superoxide dismutase (SOD). <br />      Surprisingly, double mutants, lacking mitochondrial forms SOD, reduction lifespan. This been <br />      interpreted evidence mFRTA assumed mutants suffer significantly elevated <br />      oxidative damage mitochondria. Here, using novel mtDNA damage assay conjunction related, well <br />      established damage metabolic markers, investigate age-dependent mitochondrial decline cohort of <br />      ageing wild-type nematodes, particular testing plausibility &#226;&#8364;&#732;&#226;&#8364;&#732;vicious cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; theory. We apply methods <br />      insights gained investigation mutant strain C. elegans lacks forms mitochondrial SOD. <br />      While clear age-dependent, linear increase oxidative damage WT nematodes, evidence for <br />      autocatalytic damage amplification proposed &#226;&#8364;&#732;&#226;&#8364;&#732;vicious cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; theory. Comparing SOD mutants wild-type <br />      animals, oxidative damage levels mtDNA SOD mutants significantly different <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />      wild-type animals, i.e. total loss mitochondrial SOD did significantly increase oxidative damage <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />      mtDNA. Possible reasons unexpected result implications mFRTA discussed. <br />  <br />   Citation: Gruber J, Ng LF, Fong S, Wong YT, Koh SA, et al. (2011) Mitochondrial Changes Ageing Caenorhabditis elegans &#226;&#8364;&#8220; What Do We Learn Superoxide <br />   Dismutase Knockouts? PLoS ONE 6(5): e19444. doi:10.1371/journal.pone.0019444 <br />   Editor: Vladimir N. Uversky, University South Florida College Medicine, United States America <br />   Received March 3, 2011; Accepted March 29, 2011; Published May 18, 2011 <br />   Copyright: &#195;&#376; 2011 Gruber et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The authors like gratefully acknowledge Biomedical Research Council Singapore (Grant number: BMRC 07/1/21/19/524) for <br />   support study (http://www.a-star.edu.sg/AboutASTAR/BiomedicalResearchCouncil/tabid/64/Default.aspx). Work authors&#226;&#8364;&#8482; laboratory supported by <br />   grants Singapore National Research Foundation CRP Award No. 2007-04. The funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: jan_gruber@nus.edu.sg <br />  <br />  <br />  <br /> Introduction                                                                          increase risk dying. In context mFRTA, such <br />                                                                                       autocatalytic mechanism proposed. The &#226;&#8364;&#732;&#226;&#8364;&#732;vicious <br />    It suggested ageing consequence damage                       cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; variant mFRTA proposes reactive oxygen spe- <br /> accumulation [1,2]. However, unclear exactly                          cies (ROS) cause mtDNA mutations which, destabilising the <br /> damage accumulation lead time-dependent expo-                            mitochondrial electron transport chain (ETC), increase <br /> nential increase risk dying defining characteristic               ROS production [5,9,10]. <br /> of ageing process [3]. While various types damage                       However, oxidative damage DNA, proteins lipids <br /> accumulate age, types damage affecting                             occurs commonly vivo, damage efficiently <br /> biomolecular targets, any, determine rate ageing remains                    repaired and, consequently, steady state levels damage in <br /> unclear.                                                                              healthy individuals usually low [11]. Evidence mFRTA <br />    One popular damage accumulation theories                            general best equivocal far clear whether <br /> ageing mitochondrial free radical theory ageing                       oxidative damage plays causative role ageing [12&#226;&#8364;&#8220;14]. In <br /> (mFRTA), proposing ageing accumulation                          particular, data regarding damage <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> burden in <br /> partially unrepaired oxidative damage, particular mitochon-                     mtDNA sparse, technical limitations [11&#226;&#8364;&#8220;13,15]. <br /> drial DNA (mtDNA), resulting mtDNA mutations [4&#226;&#8364;&#8220;7]. If                                Due relative simplicity, ease genetic manipulation and <br /> damage accumulation explain exponential mortality                             short lifespan, nematode worm Caenorhabditis elegans (C. elegans) <br /> dynamics, (but to, see: [8]) expect damage                       attractive model explore open questions <br /> to autocatalytic, imply exponential damage                             regarding role damage accumulation ageing [16]. Since <br /> accumulation, turn explaining observed exponential                             single-gene ageing mutation (age-1) described C. <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                1                                  May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                  SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> elegans [17], large data generated regarding             commonly associated extended lifespan, the <br /> ageing genetic determinants nematodes [14,18].                     mechanism extension remains elucidated [14,39]. <br />     Ageing cohort studies provided evidence early                   It dangerous assume SOD mutants suffer from <br /> rapid decline parameters energy metabolism                   elevated steady state levels oxidative damage mitochondria and <br /> structural deterioration mitochondria age C. elegans               mtDNA strong experimental support. Interestingly, is <br /> [14,19&#226;&#8364;&#8220;21]. Furthermore, markers global oxidative damage,                  report unchanged levels aconitase activity the <br /> including lipofuscin total protein carbonyl content, marker             mitochondria sod-2 sod-3 mutants, arguing dramat- <br /> of oxidatively damaged protein, accumulate age;                     ically increased levels oxidative stress animals [28]. To <br /> accumulation rapid short-lived slower long-lived            really answer question, direct measurements oxidative <br /> mutant strains [19,22&#226;&#8364;&#8220;25]. While data support notion                damage mitochondrial macromolecules required. However, <br /> that oxidative damage correlated metabolic functional             measuring ROS-mediated mitochondrial damage challenging, <br /> decline ageing, hard infer direct association let       especially given limited biological material available from <br /> causality observations.                                             nematodes. To date methods determination mtDNA <br />     To overcome limitations inherent correlative               specific damage C. elegans data regarding age-dependent <br /> ageing cohort studies, mutant strains generated                     mtDNA damage reported. This major challenge in <br /> by perturbing antioxidant systems aim modulating                  making sense longevity patterns observed ageing mutants <br /> oxidative damage. Superoxide dismutase (SOD) important                 [12]. We address questions quantifying age- <br /> antioxidant enzyme, responsible conversion (dismutation)              dependent damage accumulation, including damage mtDNA in <br /> of superoxide hydrogen peroxide oxygen, step            ageing WT nematodes comparing damage levels between <br /> superoxide detoxification [11,26]. C. elegans isoforms             WT sod-2/sod-3 double mutants. <br /> mitochondrial SOD (encoded sod-2 sod-3), cytosolic <br /> SODs (sod-1 sod-5) extracellular SOD (sod-4) [27].                Results <br /> Recently, data published mutants deficient                The purpose study follow single <br /> both forms mitochondrial SOD. Surprisingly, contrast                 cohort ageing nematodes simultaneously determining and <br /> observations yeast, flies mice, appears C. elegans          correlating indicators functional, metabolic mitochondrial <br /> neither deletion sod-2 sod-3 decreases <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>survival</span>. Even sod-2/           decline oxidative stress damage. For purpose, <br /> sod-3 double mutants, entirely deficient mitochondrial SOD,                developed novel mitochondrial assays: single <br /> show reduction lifespan. In fact, authors reported            worm mtDNA copy number determination, second for <br /> a significant extension lifespan sod-2 sod-2/sod-3 double          sequence-specific mtDNA damage finally lipidomics method <br /> mutants [28&#226;&#8364;&#8220;31]. This lack detrimental effects mitochondrial            characterisation mitochondrial cardiolipin species (CL). <br /> SOD depletion lifespan observed despite clearly increased              While parameters reported assayed in <br /> susceptibility exogenous ROS increased protein oxidation,              ageing C. elegans previously, determining multiple closely related <br /> as judged elevated protein carbonyl levels [28&#226;&#8364;&#8220;30]. These                  parameters parallel cohort allows test the <br /> observations interpreted strong evidence               plausibility various assumptions predictions mFRTA <br /> central role ROS determining lifespan ageing rate,               &#226;&#8364;&#732;&#226;&#8364;&#732;vicious cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; theory. In second study, <br /> least nematodes [12,24,28,32,33]. This sentiment               applied assays insights gained ageing cohort <br /> supported fascinating evidence suggesting elevated                    study obtain new insights metabolic state a <br /> mitochondrial superoxide levels actually required                  mitochondrial sod-2/sod-3 double <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span>knockout</span> mutant strain <br /> lifespan extension ETC mutant strains [34].                           (GA480). Together simple model, aimed at <br />     However, worth remembering supposedly simple              explaining consequences SOD modulation mito- <br /> model organisms immensely complex biological regulatory             chondrial compartment, use insights further <br /> systems respond perturbations way                  explore question lack lifespan reduction in <br /> expected. Since mitochondrial SOD entirely absent sod-2/            GA480 really represents strong challenge notion ROS <br /> sod-3 mutants life, including development,            contribute determination lifespan C. elegans. <br /> possible fundamentally different wild-type (WT) <br /> animals terms mitochondrial function ROS production.                Functional decline ageing cohort <br />     To test possibility compensation, changes gene                  The data presented single large cohort of <br /> expression patterns examined functions overex-                JK1107 C. elegans observations representative at <br /> pression deletion various SOD genes. Some compensatory                 independent cohorts. To judge functional state at <br /> upregulations antioxidant stress response genes, including             ageing time point means quality control, we <br /> isoforms glutathione S-transferase, reported           determined survival, pharyngeal pumping motility <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> response deletion sod-2 sod-3 [27,30]. However,             (a measure health span [40,41]) cohorts. Figure 1 <br /> homeodynamic responses living systems mediated                      illustrates impact ageing indicators functional health. <br /> changes gene expression levels. For instance, overexpression               Mean lifespan 12.3 days maximum lifespan, defined as <br /> of antioxidant enzymes lead superficially surprising               average longest surviving 10% population, 19 <br /> result actually elevating ROS-mediated damage vivo,           days (Figure 1A). These values typical JK1107 strain at <br /> case SOD overexpression mammalian cells, flies mice                 restrictive temperature 25.5uC [42,43]. Pharyngeal pumping <br /> (reviewed in: [35,36]). The underlying mechanisms                    rates (Figure 1B) motility phenotype (Figure 1C) indicators <br /> observation complex, likely involving interactions                healthy functioning muscular nervous systems. Both <br /> elements antioxidant systems ETC,                   showed rapid age-dependent decline individuals able to <br /> necessarily require changes gene expression [35&#226;&#8364;&#8220;38]. This link             maintain rhythmic pharyngeal pumping day 12 (Figure 1B) <br /> between ROS defence systems ETC particularly                       60% animals retaining highest score (&#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482;, most <br /> relevant because, C. elegans reduction ETC activity             healthy) day 12 (Figure 1C). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                              May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                   SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br />  <br /> Figure 1. Age-dependent survival functional decline typical JK1107 (glp-1) C. elegans cohort. Nematodes maintained the <br /> restrictive temperature 25.5uC. (A) Survival curve. Mean lifespan 12.3 days (n = 134). (B) Functional decline judged reduction pharyngeal <br /> pumping rate (mean 6 SEM) age. Worms exhibiting irregular sporadic pumping excluded. (C) Motility phenotype surviving worms at <br /> different ages. Scoring carried according [40]. Briefly, class &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482; worms vigorously, leaving sinusoidal tracks. Class &#226;&#8364;&#732;&#226;&#8364;&#732;B&#226;&#8364;&#8482;&#226;&#8364;&#8482; worms move <br /> only response touch appear uncoordinated locomotion. Class &#226;&#8364;&#732;&#226;&#8364;&#732;C&#226;&#8364;&#8482;&#226;&#8364;&#8482; worms body exhibit head and/or tail movements <br /> in response touch. <br /> doi:10.1371/journal.pone.0019444.g001 <br />  <br />  <br /> Age-dependent decline energy metabolism,                                    ATP reflected worm metabolism, determined oxygen <br /> mitochondrial content function                                             consumption (Figure 2B). While oxygen consumption day 14 <br />    In parallel observations age-dependent functional               reliably determined low number of <br /> decline, determined markers metabolic mitochondrial                  surviving worms, oxygen consumption declined <br /> function JK1107 C. elegans. Figure 2A illustrates                  significantly age days 3 12 (Figure 2B). <br /> observed functional decline associated significant, linear          Consistent [19,20,44,45] [21,46] previous <br /> decline adenosine triphosphate (ATP) levels age. ATP                   studies, decline better fitted exponential (R <br /> levels decline 5.7% (of reference levels day 3) day               squared = 0.98) linear decline (R squared = 0.79). However, <br /> 62% total days 3 14. This age-dependent decrease                time-points, difficult judge significance <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          3                               May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                   SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> of observation. To determine age-dependent decline              moderate significant decline mtDNA copy number was <br /> metabolic function associated loss mitochondrial              accompanied statistically significant decline total worm <br /> number, determined mtDNA copy number individual                          content diphosphatidylglycerol lipid cardiolipin (CL). CL is <br /> worms function age. In contrast previous study [46],            mainly inner mitochondrial membrane it <br /> we statistically significant linear decline mtDNA                   regulates mitochondrial function. CL highly sensitive damage <br /> between days 3 14 (Figure 2C). However, decline                   ROS high content polyunsaturated fatty acids and <br /> relatively slight copy numbers old worms (day 14)                   location near sites ROS production inner <br /> average 27% lower compared young worms (day 3). This                  mitochondrial membrane. We total CL content was <br />  <br />  <br />  <br />  <br /> Figure 2. Metabolic parameters decline linearly age typical JK1107 cohort. (A) Age-dependent decline ATP content linear <br /> regression. Best linear fit (R squared 0.86) 95% confidence band. Slope statistically significantly different zero (P,0.0001). (B) Age- <br /> dependent decline oxygen consumption, normalised worm protein. Decline oxygen consumption statistically significant (P = 0.003, <br /> ANOVA Tukey&#226;&#8364;&#8482;s post-test). (C) Age-dependent decline mtDNA copy number worm linear regression. Best linear fit (R squared 0.98) <br /> and 95% confidence bands. Slope statistically significantly different zero (P,0.01). (D) Age-dependent loss mitochondrial membrane lipid <br /> cardiolipin C. elegans total lipid extract (n = 3 independent extractions time point). Total CL content clearly declines age (P,0.005). <br /> Lipid content expressed relative abundance phosphatidylcholines (PC), phosphatidylethanolamines (PE) Cardiolipin (CL), normalised PC <br /> content young worms. <br /> doi:10.1371/journal.pone.0019444.g002 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          4                               May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                    SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> approximately 50% lower older (day 10) animals compared                    Dichlorofluoresceine-diacetate (DCF-DA) popular fluores- <br /> young (day 3) animals (Figure 2D). Together, data                cence-based probe, commonly utilised detect generalised ROS <br /> a significant age-dependent decline energy metabolism                      production. DCF-DA deacetylated endogenous esterases <br /> ageing C. elegans related significant decline                     dichlorofluorescein (DCFH), react ROS <br /> mitochondrial number function.                                               form fluorophore DCF (reviewed in: [11]). While is <br />                                                                                  significant controversy regarding actual ROS responsible for <br />                                                                                  DCF-DA fluorescence, used C. elegans, a <br /> Successful ageing associated preserved mtDNA                             useful indicator global ROS flux intact animals [47,48]. <br /> copy number                                                                      Using average rate DCF-DA fluorescence worms <br />    Given evidence significant modest decline                    proxy ROS flux, statistically significant, linear <br /> mtDNA copy number age, sought way answer                        increase ROS production age (Figure 4A). However, the <br /> question mtDNA copy number matters functionally                       increase ROS production rate small, reaching <br /> healthy ageing. To address issue, took advantage fact             levels twice high baseline. <br /> that mtDNA copy number assay performed individual                         We determined different indicators oxidative <br /> animals. This allows ask question worms age               damage. Total protein carbonyl content common marker for <br /> successfully, judged motility phenotype,                     global oxidative protein damage [11,49]. In agreement with <br /> average higher (more preserved) mtDNA copy number                           previous work [19], significant increase total pro- <br /> worms age well. For experiment selected                        tein carbonyl content age maximal levels (day 12) <br /> approximately 50 animals class &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482; class &#226;&#8364;&#732;&#226;&#8364;&#732;B&#226;&#8364;&#8482;&#226;&#8364;&#8482;                  approximately double young animals (Figure 4B). This <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;C&#226;&#8364;&#8482;&#226;&#8364;&#8482; population 12-day-old animals. We in-                     increase appeared linear exponentially <br /> dividually determined mtDNA copy number. While                       accelerating, variability data high. Protein <br /> was significant overlap groups,                    carbonyl content worm lysate combines contributions <br /> worms aged successfully (remained motility class &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482;)             originating cytoplasmic, extracellular mitochondrial <br /> had statistically significantly higher average mtDNA copy number                 compartments. To investigate specifically oxidative <br /> than animals scored class &#226;&#8364;&#732;&#226;&#8364;&#732;B&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;C&#226;&#8364;&#8482;&#226;&#8364;&#8482; day 12 life          damage burden mitochondria changes age, developed a <br /> (Figure 3). Assuming groups started average                novel sequence-specific mtDNA damage assay (see Methods S1 <br /> copy number young animals (290,000 mtDNA copies                          Figure S1 details). Using assay, able detect <br /> worm), mtDNA copy number successfully ageing                    statistically significant, age-dependent increase oxidative <br /> individuals cohort declined average 28% 41%,                   mtDNA damage age. Although moderate fit, linear <br /> respectively. This illustrates that, mtDNA copy number                     model (R squared value 0.39, slope statistically different from <br /> declined groups, successful ageing appears                       zero (P = 0.029)) consistent data any <br /> correlate slower decline mtDNA copy number.                            exponential model (Figure 4c). Moreover, total damage burden <br />                                                                                  remained low maximum (at day 14) 3.5 lesions per <br /> Age-dependent increase ROS oxidative damage                               10,000 <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span> pairs, approximately 84% increase compared to <br />   According free radical theory ageing, age-dependent                  young animals (day 3). Finally, employed detailed <br /> functional decline associated significant increase              lipidomics analysis different CL species evaluate age- <br /> in oxidative damage, particular mtDNA. To test                        dependent changes mitochondrial matrix lipid. Overall, we <br /> prediction, determined age-dependent changes ROS                           highly polyunsaturated forms CL declined more <br /> production damage accumulation using different assays.                  rapidly age relatively unsaturated forms (Figure 4D). <br />                                                                                  For instance, major CL species (CL 80:13&#226;&#8364;&#8220;16), we <br />                                                                                  significant decrease highly unsaturated CL, CL <br />                                                                                  (20:4) species declining 50% day 3 day 10 <br />                                                                                  (P,0.005). By comparison, saturated CL (20:3) showed no <br />                                                                                  significant decline major CL species (CL 80:13&#226;&#8364;&#8220;16). <br />                                                                                  Within overall abundant (CL 78:12&#226;&#8364;&#8220;15) species, CL <br />                                                                                  (20:4) CL (20:3) marked decline age, this <br />                                                                                  decline pronounced saturated CL (20:4) <br />                                                                                  species. <br />                                                                                     Together data showed statistically <br />                                                                                  significant consistent increase ROS production and <br />                                                                                  associated molecular damage ageing C. elegans. However, this <br />                                                                                  increase, mitochondria, appeared quite small linear (at <br />                                                                                  twice levels young worms) ages there <br />                                                                                  obvious functional decline (Figure 1). <br /> Figure 3. Successfully ageing (class &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482;) old worms have <br /> higher mtDNA copy number animals showing motility <br /> defects. Comparison mtDNA copy number JK1107 worms                    The GA480 double mitochondrial SOD knockout mutant <br /> the cohort (identical genotype environment) scored                   A number mitochondrial SOD mutants described <br /> same age (day 12) different motility classes. Graph           C. elegans [28&#226;&#8364;&#8220;30]. One major results mutants, <br /> shows scatter median individual worm copy number                  respect mFRTA, surprising lack reduction in <br /> data normally distributed. Class &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482; worms statistically          lifespan response single multiple deletion SOD <br /> significantly higher mtDNA copy number (median 210,000 vs. 170,000 <br />                                                                                  genes [28&#226;&#8364;&#8220;30]. However, intriguing observation about <br /> copies) class &#226;&#8364;&#732;&#226;&#8364;&#732;B&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;C&#226;&#8364;&#8482;&#226;&#8364;&#8482; animals (P,0.001, Mann-Whitney test). <br /> The n-numbers class &#226;&#8364;&#732;&#226;&#8364;&#732;A&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;B&#226;&#8364;&#8482;&#226;&#8364;&#8482;/&#226;&#8364;&#8482;&#226;&#8364;&#8482;C&#226;&#8364;&#8482;&#226;&#8364;&#8482; 55 52 respectively,       SOD mutants consistent reports suggest delayed <br /> although typically class &#226;&#8364;&#732;&#226;&#8364;&#732;C&#226;&#8364;&#8482;&#226;&#8364;&#8482; worms day 12.                development reduced brood size [29,30]. To gain further <br /> doi:10.1371/journal.pone.0019444.g003                                            insight possible reasons phenotypes, applied our <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            5                             May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                   SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br />  <br /> Figure 4. Age-dependent changes parameters ROS production oxidative damage. (A) Generalised ROS production evaluated <br /> using worm DCF-DA assay intact animals. ROS production increases linearly age linear regression n = 9 replicates (R squared <br /> 0.91) positive slope, significantly different zero (P,0.05). (B) Age-dependent protein carbonyl content (PCC), normalised worm protein, <br /> slot blot linear regression. Linear regression (R squared 0.66) used reject zero slope (P = 0.19). However, protein carbonyl content in <br /> old animals significantly higher young animals (P = 0.0003, ANOVA Tukey&#226;&#8364;&#8482;s post-test). (C) Age-dependent increase oxidative damage <br /> to mtDNA measured qRT-PCR assay expressed lesions bp mtDNA. Oxidative mtDNA damage increases linearly age <br /> (P = 0.006, ANOVA Tukey&#226;&#8364;&#8482;s post-test). (D) Age-dependent change degree saturation mitochondrial specific lipid cardiolipin (CL). <br /> Note CL species 80:13 combination fatty acids 3620:3 1620:4; CL 80:14 = 2620:3 2620:4; CL 80:15 = 1620:3 and <br /> 3620:4; CL 80:16 = 4620:4. While abundance species CL declines age, unsaturated species (with higher number of <br /> double bonds) decline dramatically age corresponding, saturated forms (comparison day 3 vs. day 10. ns: significant, <br /> *: P,0.05, **: P,0.01, ***: P,0.01). <br /> doi:10.1371/journal.pone.0019444.g004 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          6                               May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> assays sod-2/sod-3 double mutant (GA480), lacks             78% WT day 3 day 7, respectively (Figure 5B). <br /> forms mitochondrial SOD.                                                 Together data suggested GA480 suffer a <br />    First, characterised extent developmental fecundity        decrease usable metabolic energy. To investigate this <br /> deficits GA480 determining schedule egg production,            possibility, determined ATP levels, oxygen consumption and <br /> number progeny day adulthood adult size           mtDNA copy number age-matched GA480 WT nematodes. <br /> growth rate GA480 (Figure 5A 5B). We GA480                While trend slightly lower mtDNA copy <br /> produced approximately 50% progeny WT overall                 number GA480 relative WT, ATP levels were <br /> also produced significantly fewer eggs peak days (days 5         significantly different mutant WT (Figure 5C 5D). <br /> 6) egg production (P,0.0001). Consistently, adult GA480             While trend reduction oxygen consumption, <br /> also significantly smaller, <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> GA480 55%          statistically significant (Figure 6D). Surprisingly, we <br />  <br />  <br />  <br />  <br /> Figure 5. Phenotypical fitness comparison GA480 sod-2/sod-3 double mutant strain (sod-2(gk257) I, sod-3(tm760)) WT N2 C. <br /> elegans. (A) Egg laying study comparing number eggs laid daily GA480 WT worms (n = 10 each). GA480 produce significantly eggs <br /> (total 222 vs. 107 eggs adult average, (P,0.0001, ANOVA Tukey&#226;&#8364;&#8482;s post-test)). (B) GA480 worms grow slowly significantly smaller <br /> than WT day 3 day 7 (both P,0.0001, Student&#226;&#8364;&#8482;s t-test). (C) Steady state levels ATP worm day 4 significantly different <br /> between GA480 WT worms (P = 0.18, Student&#226;&#8364;&#8482;s t-test). (D) No difference mtDNA copy number individual worms GA480 WT. <br /> Copy number assessed day 2 post bleach (n = 28 animals each), minimise confounding effects oocytes possible. The median <br /> is significantly different GA480 WT (P = 0.44, Mann-Whitney test). <br /> doi:10.1371/journal.pone.0019444.g005 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       7                               May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                   SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> found ROS production, determined animal DCF-                 mtDNA damage elevated dramatically GA480 as <br /> DA signal, significantly lower GA480 WT controls               expected complete mitochondrial SOD <br /> (Figure 6A B). Furthermore, steady-state mtDNA              knockout. In fact, mean mtDNA damage levels 4-day-old GA480 <br /> damage burden significantly higher GA480 WT                comparable 8-day-old WT animals ageing <br /> animals (Figure 6C). While appeared slight (non                 cohort study certainly lower highest levels detected <br /> significant) trend elevated mtDNA damage burden,                      oldest animals (day 14). <br />  <br />  <br />  <br />  <br /> Figure 6. MnSOD mutants suffer elevated oxidative damage. Comparison ROS related parameters MnSOD double <br /> mutant strain GA480 WT N2 C. elegans. (A) Averaged fluorescence signal worm DCF-DA generalised ROS assay (n = 7 repeats) day 4 <br /> WT GA480 animals. Despite lacking forms mitochondrial SOD, GA480 animals produce ROS associated fluorescence signal WT <br /> animals. (B) Analysis data (A); GA480 produce statistically significantly ROS measured DCF-DA assay (P = 0.024, Student&#226;&#8364;&#8482;s t- <br /> test). (C) Oxidative mtDNA lesions mtDNA samples (n = 4, each) day 4 GA480 WT worms. GA480 does suffer significantly elevated <br /> oxidative damage judged qRT-PCR assay oxidative mtDNA damage (P = 0.43, Student&#226;&#8364;&#8482;s t-test). (D) Oxygen consumption GA480 vs. WT <br /> animals day 4 life. The median oxygen consumption significantly different GA480 WT (P = 0.31, Mann-Whitney test). <br /> doi:10.1371/journal.pone.0019444.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         8                               May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                    SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br /> Making sense SOD knockout mutants silico                                 multiple SOD-independent pathways superoxide consumption <br />    In attempt explain experimental results obtained              classified different stoichiometry respect <br /> the GA480 SOD double mutant, utilised simple, analytically                hydrogen peroxide production. These different pathways are <br /> solvable model based Gardner et al. [37,50]. This model             modeled collectively pseudo-first-order reactions aggregate <br /> was originally designed minimal model explain                      rate constants (Figure 7 Table 1). Flux these <br /> paradoxical elevation ROS-mediated damage                         alternative pathways change levels SOD modulated <br /> observed SOD overexpressed (Figure 7, Table 1). While                  and, certain conditions, lead counterintuitive <br /> the model deliberately minimal, does reflect             results, instance terms hydrogen peroxide production <br /> important mechanisms previously shown                     [35,37]. Second, mechanisms high superoxide <br /> relevant understanding SOD modulation [35&#226;&#8364;&#8220;37,50].                   levels inhibit mitochondrial function, indirectly <br />    First, addition SOD-mediated dismutation, yields                modulating superoxide production rate [36,51&#226;&#8364;&#8220;55]. Here, this <br /> one molecule hydrogen peroxide superoxides,               feedback inhibition superoxide production is <br />  <br />  <br />  <br />  <br /> Figure 7. Minimal model mitochondrial superoxide production consumption hydrogen peroxide production. Model <br /> based Gardner et al. [37,50], modified mitochondrial compartment. A) Three classes superoxide consumption reactions with <br /> different hydrogen peroxide yields considered. Each class represents large number actual reactions yielding 1, 1/2 hydrogen <br /> peroxide superoxide consumed. The model includes simple negative feedback mechanism superoxide production rate. <br /> B) Equations production consumption superoxide hydrogen peroxide according model. Hydrogen peroxide, not <br />                                                                                              N <br /> superoxide, exit mitochondria, therefore, equation (1) steady state (d[O2 2]/dt = 0). The actual rate superoxide production v4/ <br />       N <br /> (1+[O2 2]/Ki)n, denominator models feedback inhibition superoxide negatively modulating rate production. The strength <br /> of feedback controlled exponent n, threshold feedback significant controlled inhibition constant <br /> Ki. For n = 0, feedback inhibition level superoxide while, n = 1, superoxide production K maximal superoxide levels are <br /> equal Ki. <br /> doi:10.1371/journal.pone.0019444.g007 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          9                                May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                              SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br />  Table 1. Model parameters. <br />  <br />  <br />  Name         Value                      Notes <br />                        27        21 <br />  v4           6.6610        Ms           Rate superoxide generation [35] <br />  k1           2.36109 M21s21             Rate constant MnSOD superoxide [77] <br />  k2           56102 s21                  Aggregate rate constant reactions yielding single molecule hydrogen peroxide superoxide consumed. The dominant <br />                                          processes considered reactions superoxide susceptible dehydratases, including aconitase (cellular <br />                                          concentration 1024 M [56], likely higher mitochondria mitochondrial localisation of, instance, <br />                                          aconitase). Typical second order rate constants 106&#226;&#8364;&#8220;107 M21s21 ([11] p101). Reactions ascorbate included here. <br />  k3           16103 s21                  Aggregate rate constant reactions yielding hydrogen peroxide. The representative process considered oxidation of <br />                                          ferricytochrome (mitochondrial concentration 5 mM [78,79]) superoxide. The rate constant process is <br />                                          2.66105 M21s21. Reactions NO likely negligible low NO concentrations [80] reactions ubiquinol are <br />                                          thermodynamically unfavorable [50]. <br />  Ki           7.8610211 M                Inhibition constant feedback inhibition superoxide. This chosen identical level superoxide calculated <br />                                          WT SOD case (SOD = 1) feedback inhibition (n = 0). This choice Ki means WT levels SOD, &#226;&#8364;&#732;&#226;&#8364;&#732;switching <br />                                          on&#226;&#8364;&#8482;&#226;&#8364;&#8482; inhibition results approximately 40% inhibition (Inhibition constant = 0.6, Figure 8). This consistent the <br />                                          significant increase (almost 50%) ATP levels observed following deletion uncoupling protein UCP4 WT <br />                                          C. elegans larvae, estimates approximately 25&#226;&#8364;&#8220;30% baseline proton leak [52,53]. <br />  [SOD]        361026 M                   WT concentration MnSOD mitochondria [77]. <br />  n            0 1                     Order feedback term: n = 0/no feedback inhibition, n = 1/linear feedback [50]. <br />  <br />  doi:10.1371/journal.pone.0019444.t001 <br />  <br />  <br /> modeled following approach Gardner et al. equation 3                        interpreted drop total usable metabolic energy flux <br /> (Figure 7B). Parameter values used (Table 1) based                    approximately 50% respect WT (from 0.62 0.29, <br /> [50] [35] modifications adapt model                            factor 1.0 corresponds theoretical maximum <br /> mitochondrial compartment. In particular, assume that,                        obtained feedback inhibition (n = 0)). <br /> in mitochondrial compartment, reaction superoxide <br /> with ferricytochrome likely contributes significantly k3                 Discussion <br /> major contributors k2 4Fe-4S clusters aconitase and <br /> other susceptible dehydratases [11,37,56]. Fortunately,                               The result WT ageing cohort study is <br /> qualitative behaviour model generally does depend                          little support &#226;&#8364;&#732;&#226;&#8364;&#732;vicious cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; theory C. elegans. The age- <br /> sensitively exact choice parameters.                                  dependent oxidative damage accumulation increase ROS <br />                                                                                       production linear limited (Figure 4). However, notion <br />                                                                                       mitochondria mitochondrial ROS matter ageing is <br /> Modeling results                                                                      supported results regarding rapid age-dependent <br />    Solving model assumptions outlined re-                         decline markers energy metabolism (Figure 2) the <br /> vealed superoxide does increase mitochondrial                        better preservation mtDNA copy number successfully <br /> SOD decreases (Figure 8). However, superoxide consump-                         ageing worms (Figure 3). <br /> tion alternative pathways (via k2 k3), increase                      While ROS related parameters change linearly age, <br /> remaines order magnitude (5&#226;&#8364;&#8220;6 fold                         metabolic decline, especially post reproduction, rapid C. <br /> parameters chosen here), feedback inhibition ignored                      elegans (Figure 2) [19,20,44,45]. One major challenges the <br /> (n = 0) (Figure 8 top). When inhibition considered (n = 1),                       damage accumulation theories ageing explain a <br /> the increase superoxide attenuated,                          moderate, linear accumulation damage age cause <br /> superoxide levels, complete knockout mutants                      rapid decline systems performance exponential <br /> other (for instance genetic) compensation, rose approxi-                      increases morbidity mortality seen ageing [8,57]. <br /> mately 3 fold (Figure 8 bottom). Importantly,                        Given role mitochondrial function, significant age- <br /> unrealistic case complete knockout compensation                        dependent change content composition CL age <br /> ignoring feedback inhibition, absolute steady state superoxide                        (Figure 2D 4D) mechanism warrants further <br /> levels remained low, rising 0.5 nM.                                       investigation context. <br />    Hydrogen peroxide production rates, contrast,                           Below, contrast data ageing cohort studies with <br /> order 10&#226;&#8364;&#8220;100 nmolar s21 decreased mitochondrial                           observations GA480 mutant strain possible <br /> SOD content reduced (Figure 8). This consequence                      link moderate age-dependent increases ROS <br /> negative feedback mechanism key feature model,                          significant decline energy production age. The lack <br /> explaining paradoxical increase oxidative stress SOD                        lifespan reduction mitochondrial SOD mutants been <br /> overexpression experiments [37,50]. For mitochondrial SOD                         interpreted major challenge mFRTA [12,24,28,32,33]. <br /> knockout mutants, hydrogen peroxide production                          This interpretation based assumption complete <br /> rates predicted model feedback                            knockout mitochondrial SOD associated a <br /> inhibition 39% 71% WT, respectively.                                     dramatic increase oxidative damage, particular mitochon- <br />    With parameters chosen here, feedback inhibition factor                    dria, lack lifespan effects indicates that <br /> (Ki) 0.62 SOD WT (SOD = 1) 0.29 complete SOD                          ROS mediated damage mitochondria does determine the <br /> knockout mutants (SOD = 0) (Figure 8 bottom). Since                              rate ageing C. elegans. Taken whole, data reveal that <br /> inhibition factor models processes mitochondrial uncou-                       situation complicated. Although mitochondrial <br /> pling and/or inhibition ETC components superoxide,                         SOD mutants increased susceptibility artificially <br />  <br />  <br />          PLoS ONE | www.plosone.org                                              10                                    May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                 SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br />  <br /> Figure 8. Results SOD knockdown/deletion overexpression according model. Model results showing dependency of <br /> mitochondrial superoxide level, hydrogen peroxide production rate feedback inhibition factor (decrease energy production relative n = 0 <br /> case) mitochondrial SOD feedback (top, n = 0) feedback inhibition (bottom, n = 1). WT levels mitochondrial SOD are <br /> normalised 1 indicated vertical dashed line. The area right WT levels reflects SOD overexpression area left <br /> reflects SOD depletion, complete knockout corresponding zero SOD. Note hydrogen peroxide production rate increases monotonically <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       11                              May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                   SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br /> with SOD. This result independent exact choice parameters feedback inhibition considered (n = 1) previously observed [50]. The <br /> inhibition factor axis number 1.0 0.0 indicates metabolic depression feedback inhibition to <br /> mitochondrial uncoupling inhibition ETC components superoxide (relative n = 0 case). When feedback inhibition factor close to <br /> 1.0, metabolism ROS production maximal. In particular, feedback ignored (n = 0) factor 1.0 <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>. Note that, WT <br /> SOD levels, feedback factor case n = 1 approximately 0.62. This interpreted effect residual mitochondrial uncoupling, <br /> even WT levels SOD, consistent expected basal proton leak UCP4 [53]. <br /> doi:10.1371/journal.pone.0019444.g008 <br />  <br /> elevated oxidative stress [28&#226;&#8364;&#8220;30], does prove               hydrogen peroxide production moderate increase in <br /> suffer dramatically elevated oxidative damage normal                mitochondrial superoxide levels. Given results, perhaps <br /> conditions. Using sequence-specific mtDNA damage assay,                 surprising effect mitochondrial SOD knockout in <br /> failed detect significant increase oxidative damage                 nematodes delayed development decreased fecundity, <br /> mtDNA GA480 mitochondrial sod-2/sod-3 double mutant.                    dramatically increased oxidative damage mito- <br /> While data rule slight increase mtDNA                      chondria shortened lifespan. The situation higher <br /> oxidative damage, comparison ageing cohort study clearly              organisms likely complicated fact able <br /> shows mtDNA oxidative damage levels young SOD                          respond reduced availability metabolic energy, explaining <br /> mutants lower seen old WT animals (Figure 4C                 severe phenotypes mitochondrial SOD knockout in <br /> and 6C).                                                                       species [63,64]. Of course, hydrogen peroxide and <br />    The notion oxidative damage is, most, mildly increased              superoxide likely affect signaling, including through <br /> in mitochondrial SOD mutants consistent limited               insulin-like growth factor pathway, changing relative <br /> biomarker data available literature. There evidence             abundance signaling effects contribute the <br /> for slight increase lipid oxidation product trans-4-hydroxy-          observed fitness effects explaining longevity <br /> 2-nonenal mitochondria sod-2 mutants [58]. In addition,              effects [29]. <br /> Hekimi lab reported small (30%&#226;&#8364;&#8220;40%) increase oxidative                        Some additional insights plausibility model can <br /> protein damage response RNAi knockdown sod-1 sod-2                derived considering ROS production SOD mutant strains <br /> in WT C. elegans approximately 100% increase protein                 measured different methodologies. Using worm <br /> carbonyl content sod-2 mutants [30,31]. By comparison,              DCF-DA dye assay, DCF-DA associated fluores- <br /> detected approximately 100% age-dependent increase                       cence animals (,50%) lower GA480 WT. <br /> protein carbonyl content ageing cohort study                   While questions regarding localisation identity of <br /> magnitude consistent age-dependent increases protein                chemical species measured DCF-DA [65,66], clear that <br /> carbonyl content observed C. elegans [19].                        DCF-DA does directly detect superoxide [11]. We therefore <br />    How possible complete loss mitochondrial SOD             interpret result general reduction total ROS flux. Since <br /> does cause oxidative damage severe indicated                  hydrogen peroxide, superoxide, exit mitochondria, the <br /> these results? One hint likely answer comes reports             observed 50% drop DCF-DA signal consistent the <br /> that SOD mutants small, developmentally delayed                    predicted ,50% decrease metabolism hydrogen peroxide <br /> have reduced progeny. It immediately obvious deletion               production rate SOD knockout mutants (Figure 8 bottom). <br /> of class antioxidant genes result fitness penalties        Others, using mitochondrial superoxide detector dye Mito- <br /> instead increased mortality reduced lifespan. However,              SOX red [58,67], slight (at 29%) increase in <br /> obvious way overcome reduced ROS detoxification capacity                 MitoSOX fluorescence single mitochondrial SOD mutants <br /> to produce ROS place [52,59]. Mitochondrial                  [58]. The authors suggest increase elevated <br /> uncoupling modulate ROS production conserved                   mitochondrial superoxide content, argue actual <br /> uncoupling protein UCP4 C. elegans potentially                 increase likely somewhat higher (as two-fold). <br /> result dramatic reduction ROS production,                  These vivo data consistent results extracted <br /> significant cost terms energy production [14,52,53,60,61]. In            mitochondria [68]. If data accepted, experimental <br /> this context, reported increase trans-4-hydroxy-2-nonenal            results indicate moderate increase intramitochondrial <br /> sod-2 mutants interest, lipid oxidation product              superoxide levels decrease overall ROS flux response to <br /> previously shown activate mitochondrial uncoupling                     loss mitochondrial SOD, result entirely consistent our <br /> proteins [58,62].                                                              model predictions (Figure 8 bottom). <br />    Similarly, evidence preferential effects oxidative              This interpretation suggests increased uncoupling, <br /> modification specific mitochondrial proteins, including acon-               reduced mitochondrial membrane potential and/or reduced <br /> itase, involved energy production [54,55]. Therefore,                ETC activity major consequences mitochondrial SOD <br /> moderate increase total protein carbonyls, detected               deletion. Specifically, model predicts available <br /> SOD mutants, associated significant depression              metabolic energy mitochondrial knockout mutants should <br /> ETC function. This explanation consistent evidence             ,50% lower WT. This broadly consistent with <br /> that treatment N-acetyl-cysteine (NAC), presumably reducing               observations terms magnitude reduction progeny <br /> mitochondrial ROS, leads increased oxygen consumption                   reduction eggs laid peak egg-laying days <br /> elevated ATP levels WT mitochondrial mutants                  (Figure 5A). Steady-state ATP levels, different <br /> [34]. This evidence supports notion mitochondrial                     GA480 compared WT, function energy <br /> superoxide important modulator mitochondrial function,                production consumption rates allow <br /> with slightly elevated superoxide levels suppressing metab-               conclusions regarding ATP flux. In contrast report the <br /> olism extending lifespan vice versa.                                   literature [30], did detect significant decrease oxygen <br />    Using simple mathematical model processes,               consumption GA480, trend lower <br /> shown SOD knockout mechanism result                       oxygen consumption (Figure 6D). However, given ETC <br /> significantly decreased metabolism associated decrease                  inhibition uncoupling expected contribute feedback <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         12                              May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                    SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> inhibition, model does easily allow prediction oxygen                Cohort studies <br /> consumption partially uncoupling mitochondria                        All age-dependent patterns reported observed consis- <br /> increase oxygen consumption ETC inhibition                          tently independent ageing cohort studies. <br /> decrease it. In context, interesting evaluate               Assays worm population performed in <br /> the effects mitochondrial SOD mutation UCP4 knockout                    triplicate. <br /> mutant background [53]. In summary, expectation, we <br /> found mitochondrial SOD double mutant strain GA480                     Nematode size <br /> showed evidence reduced ROS release did suffer                    Worms photographed egg laying study using <br /> significantly elevated oxidative mtDNA damage. Instead,                   calibrated Leica MZ10F microscope (Leica, Singapore). Worm <br /> mutants experience significant (up 50%) decrease                size (in mm) determined free curve tool provided by <br /> available metabolic energy, judged growth total, well <br />                                                                                 Leica Application Suite software (v2.6.0 R1). <br /> as peak fecundity. <br />     Given results suggest normal extended <br /> survival possible presence elevated mitochondrial <br />                                                                                 Pharyngeal pumping <br /> superoxide levels, mitochondrial SOD mutants                      Videos worms recorded days 4, 8, 12 14 806 <br /> mitochondrial ROS matter ageing C. elegans? Not                   magnification using dissecting microscope (Leica MZ16) with <br /> necessarily; model suggests keeping mitochondrial                      attached camera (DFC 300 FX). At 3 minutes video were <br /> superoxide low (sub nM) threshold achieved                   recorded animal (n = 10) viewed slow motion for <br /> the mitochondrial regulatory system, doing comes                 accurate quantification. <br /> with significant penalties terms energy ultimately fitness. <br /> That conserved mechanism governing tradeoff appears                 Motility phenotype <br /> have evolved actually supports notion mitochondrial ROS                   Age-synchronised worms assessed spontaneous <br /> are important determinants survival life history choices. It          locomotion response prodding platinum <br /> interesting note markers oxidative damage ROS                wormpick described [40]. <br /> production measured SOD mutants remain <br /> below levels seen old C. elegans. It possible, therefore,        Lifespan studies <br /> oxidative damage ageing animals mutants remain                     Worms counted 1 3 days adulthood, depending <br /> low reason, namely, levels fixed, tightly             prevalent mortality. Those failed respond mechanical <br /> controlled threshold trigger ETC inhibition mitochondrial              touch scored dead removed plates. Worms <br /> uncoupling. In scenario, main purpose mitochondrial                 crawled sides plate died away from <br /> SOD limit global oxidative damage maintain                     agar censored. <br /> mitochondrial function preventing superoxide &#226;&#8364;&#732;&#226;&#8364;&#732;rheostat&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> [51] engaging. Attempts modulating mitochondrial ROS                    Egg laying assay <br /> via antioxidant certain predomi-                     Mutants N2 WT nematodes (n = 10, each) transferred <br /> nantly modulate energy production ROS mediated                      hatching plates individual egg-laying plates day 1 of <br /> damage. This makes &#226;&#8364;&#732;&#226;&#8364;&#732;vicious cycle&#226;&#8364;&#8482;&#226;&#8364;&#8482; difficult envisage             adulthood. Animals transferred 24 h fresh <br /> because, ROS levels kept threshold                 egg-laying plates egg laying ceased. Eggs laid allowed to <br /> longer old animals, energy metabolism collapse point             hatch room temperature progeny counted 2 <br /> where inconsistent tissue function organismal                    days. <br /> survival. Finally, observation modulation mitochondrial <br /> function mitochondrial superoxide clearly closely linked to <br />                                                                                 ATP assay <br /> lifespan regulation raises question mechanism <br />                                                                                    The adenosine triphosphate (ATP) assay performed as <br /> reduction ETC flux causes lifespan extension. The assays <br />                                                                                 described [41] minor modifications. Briefly, flash frozen <br /> described complementary methods recently <br />                                                                                 nematodes re-suspended trichloroacetic acid, lysed and <br /> described [69,70] open exciting future opportunities <br />                                                                                 incubated ice, followed centrifugation. Supernatants were <br /> for elucidation possible role mitochondria and <br /> mtDNA context.                                                          pooled, aliquots ATP standards sample supernatant were <br />                                                                                 pipetted white 96-well plate, followed addition arsenite <br />                                                                                 ATP buffer. ATP levels measured using luminometer <br /> Materials Methods                                                           (Infinite 200, Tecan, Ma&#194;&#168;nnedorf, Switzerland) preprogrammed to <br /> Nematode strains maintenance                                                inject firefly lantern extract. <br />    The Bristol N2 (WT), JK1107 (glp-1) GA480 (sod-2/sod-3) C. <br /> elegans strains used. Worms maintained 20uC                     Oxygen consumption <br /> nematode growth medium (NGM) agar plates exception                        The metabolic rate C. elegans determined using a <br /> that JK1107 worms cultivated 25.5uC prevent progeny.                 Clark-type oxygen electrode (Hansatech, UK). Worms were <br /> Preparation NGM previously described [71]                    washed, re-suspended 1 ml M9 buffer (for 1 l: 3 g KH2PO4, <br /> addition 200 mg/ml final concentration Streptomycin.                      6 g Na2HPO4, 5 g NaCl, 1 ml MgSO4 [1 M]), placed in <br /> Streptomycin resistant Escherichia coli strain OP50-1 added                 electrode chamber, earlier stabilised for <br /> (50 ml 1010 cells/ml E. coli stock 35 mm petri dish                  30 min 1 ml air-saturated M9 buffer. Changes oxygen <br /> 500 ml E. coli 94 mm petri dish) plate. Worms used               concentration monitored polarographically 2&#226;&#8364;&#8220;15 min to <br /> in lipidomics study grown liquid culture [71]                   obtain oxygen consumption rates later corrected against <br /> synchronised day grown plates. This                 bacterial controls. Final oxygen consumption rates normal- <br /> necessary obtain sufficient numbers lipid extraction                 ised protein concentration using Dc-Protein Assay kit (Bio- <br /> protocol.                                                                       Rad, Hercules, USA). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          13                              May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                 SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br />  <br /> Measurement general reactive oxygen species (ROS)                         various lipids carried described previously [74,75]. <br />    100 worms transferred black 96-well              Electrospray ionisation mass spectrometry (ESI/MS) per- <br /> plate containing 100 ml M9 buffer. Dichlorofluoresceine-diacetate            formed Waters Micromass Q-Tof micro mass spectrometer <br /> (DCF-DA) (50 mM M9 buffer, 100 ml, Invitrogen, Carlsbad,                  upfront Waters CapLC inlet (Waters Corp., Milford, MA). <br /> USA) added well. ROS associated fluorescence                A Waters XTerra column (1 mm6150 mm) utilised for <br /> levels measured kinetically using fluorescence plate reader           separation lipids. Chloroform:methanol 1:1 (v/v) 5% of <br /> (SpectraMax Gemini EM, Molecular Devices, Sunnyvale, USA)                 300 mM piperidine (final 15 mM) used mobile phase <br /> excitation 485 nm emission 520 nm, room temperature,               isocratic elution. The mass spectrum acquired from <br /> 2 min 14 h. Data normalised bacterial controls,                  m/z 400&#226;&#8364;&#8220;1650 frequency 1 scan/s. An Agilent high <br /> composed 100 ml OP50-1-containing M9 buffer separate                   performance liquid chromatography (HPLC) 1200 coupled <br /> wells plate.                                                     Applied Biosystem Triple Quadrupole/Ion Trap mass <br />                                                                              spectrometer (3200 Qtrap) used quantification of <br /> Protein carbonyl determination                                               individual phospholipids sphingomyelin species [75,76]. <br />   Protein carbonyls determined according [41]. Briefly,              Separation individual classes polar lipids carried out <br /> worms collected, washed, re-suspended PBST (0.1%                     using Luna 3 mm silica column (i.d. 150 mm62.0 mm). HPLC <br /> Tween-20 PBS, 1st BASE, Singapore) containing 1 mM                        conditions were: mobile phase A (chloroform: methanol: ammo- <br /> phenylmethylsulfonyl fluoride, sonicated ice. Protein                 nium hydroxide, 89.5:10:0.5), B (chloroform: methanol: ammoni- <br /> concentrations lysates determined Dc-Protein              um hydroxide: water, 55:39:0.5:5.5); flow rate 300 ml/min; 5% B <br /> Assay (Bio-Rad, Hercules, USA). Samples derivatised, in-                3 min, linearly changed 30% B 24 min and <br /> cubated room temperature, neutralisation solution added.              maintained 5 min, linearly changed 70% B in <br /> Sample lysates loaded slot blot apparatus (Bio-Rad,              5 min maintained 7 min. Then, eluents changed <br /> Hercules, USA) transferred nitrocellulose                    original ratio 5 min maintained 6 min. Multiple <br /> membrane (Bio-Rad, Hercules, USA). After blocking non-                   reaction monitoring (MRM) transitions set quantita- <br /> specific binding, membrane probed anti-2,4-                     tive analysis various polar lipids. Levels individual lipids were <br /> dinitrophenylhydrazine antibody (Chemicon International, Teme-               quantified using spiked internal standards. PC-14:0/14:0, PE- <br />                                                                              14:0/14:0, CL-15:0(3)/16:1 C12-SM standards obtained <br /> cula, USA), followed secondary detection HRP conjugated <br />                                                                              Avanti Polar Lipids (Alabaster, AL, USA). <br /> anti-rabbit IgG antibody. <br />  <br /> Mitochondrial copy number quantification                                     Statistical analysis <br />                                                                                 For statistical analysis GraphPad Prism version 4.00 Apple <br />    Mitochondrial copy number quantified according [41]. <br />                                                                              Macintosh, GraphPad Software, San Diego (USA) used. Data <br /> Briefly, individual worms picked PCR tubes, lysed, and <br />                                                                              reported plotted mean 6 SEM unless stated. <br /> mtDNA copy number individual nematodes determined by <br />                                                                              Group differences assessed using unpaired t-test (single <br /> quantitative real-time PCR (qRT-PCR,) using reference sample <br />                                                                              comparison) ANOVA/linear regression (multiple comparisons/ <br /> of known copy number (quantified serial dilution). <br />                                                                              change time) exception individual worm mtDNA <br />                                                                              copy number, failed normality test and, therefore, median <br /> Sequence-specific mtDNA damage                                               differences analysed using non-parametric test (Mann- <br />    Sequence-specific burden oxidative DNA lesions                     Whitney test). Differences P,0.05 considered as <br /> determined based digest mtDNA formamidopyr-                   statistically significant. <br /> imidine-DNA glycolase (fpg), followed quantitative real time <br /> PCR (qRT-PCR) amplification. Fpg recognises excises a <br />                                                                              Modeling <br /> range modified DNA bases, including 8-oxoguanine, thereby <br />                                                                                 The mathematical model implemented, solved (numerical <br /> rendering templates containing lesion <br />                                                                              integration, zero point determination) plotted using GNU <br /> resistant PCR amplification; resulting drop amplifiable <br />                                                                              Octave language (GNU Linux) numerical compu- <br /> template concentration converted lesions base pair <br />                                                                              tation (http://www.octave.org). <br /> (bp). Since sensitivity protocol depends length the <br /> amplified template, suitable long extension qRT-PCR protocol <br /> was developed based [72]. See Methods S1 details              Supporting Information <br /> this protocol data analysis.                                             Figure S1 Example qRT-PCR curves young (left <br />                                                                              panel) old (right panel) worms. Template concentration <br /> Analysis lipids using high performance liquid                             post fpg digest lower oxidative lesions digested, <br /> chromatography/mass spectrometry (LC/MS)                                     converting lesions sequence gaps. Fpg digested samples <br />    Lipids extracted using modified Bligh Dyer double              higher Cts mock digested samples. The <br /> chloroform extraction method [73]. Approximately 10,000 worms                difference Ct fpg mock digested samples defined <br /> were homogenised 0.9 ml ice-cold organic solvent mixture                  DCt = Ctmock2Ctdigest. Note convention DCt,0, <br /> (chloroform: methanol, 1/2 v/v) incubated ice                    usually. <br /> agitation 2 h vacuum container 4uC dark room.                  (TIF) <br /> Subsequently, 0.3 ml ice-cold chloroform 0.45 ml ice- <br />                                                                              Methods S1 <br /> cold water added homogenate. The mixture was <br />                                                                              (DOC) <br /> vortexed 30 sec centrifuged 8600 g 5 min, lower <br /> organic layer carefully transferred microcentifuge tube <br /> and second extraction performed using 0.5 ml ice-              Acknowledgments <br /> cold chloroform. The organic extracts pooled, dried             The authors like thank Prof. Takao Inoue helpful <br /> N2 stream stored 280uC lipid analysis. Profiling             comments writing manuscript Dr. Alfredo Amiel <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        14                              May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                                         SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> Leonardia help support design troubleshooting                        Author Contributions <br /> of PCR assays. We thank Caenorhabditis Genetics Centre, is <br /> funded National Institutes Health National Centre Research                        Conceived designed experiments: JG BH. Performed the <br /> Resources, provision worm strains.                                                   experiments: LFN SF YTW JG SAK C-BC GS WFC. Analyzed data: <br />                                                                                                 JG BH SS GH. Contributed reagents/materials/analysis tools: MRW. <br />                                                                                                 Wrote paper: JG BH C-BC SS MRW. <br />  <br /> References <br />  1. Hayflick L (2007) Biological aging longer unsolved problem.                        29. Honda Y, Tanaka M, Honda S (2008) Modulation longevity diapause by <br />     Ann N Y Acad Sci 1100: 1&#226;&#8364;&#8220;13.                                                                    redox regulation mechanisms insulin-like signaling control in <br />  2. Holliday R (2006) Aging longer unsolved problem biology.                            Caenorhabditis elegans. Exp Gerontol 43: 520&#226;&#8364;&#8220;529. <br />     Ann N Y Acad Sci 1067: 1&#226;&#8364;&#8220;9.                                                                 30. Van Raamsdonk JM, Hekimi S (2009) Deletion mitochondrial superoxide <br />  3. Finch CE, Pike MC, Witten M (1990) Slow mortality rate accelerations                     dismutase sod-2 extends lifespan Caenorhabditis elegans. PLoS Genet 5: <br />     aging animals approximate humans. Science 249: 902&#226;&#8364;&#8220;905.                         e1000361. <br />  4. Miquel J, Economos AC, Fleming J, Johnson JE, Jr. (1980) Mitochondrial role              31. Yang W, Li J, Hekimi S (2007) A Measurable increase oxidative damage due <br />     cell aging. Exp Gerontol 15: 575&#226;&#8364;&#8220;591.                                                           reduction superoxide detoxification fails shorten life span long- <br />  5. Linnane AW, Marzuki S, Ozawa T, Tanaka M (1989) Mitochondrial DNA                               lived mitochondrial mutants Caenorhabditis elegans. Genetics 177: <br />     mutations important contributor ageing degenerative diseases.                      2063&#226;&#8364;&#8220;2074. <br />     Lancet 1: 642&#226;&#8364;&#8220;645.                                                                          32. Salmon AB, Richardson A, Perez VI (2010) Update oxidative stress <br />  6. Harman D (1972) The biologic clock: mitochondria? J Am Geriatr Soc 20:                      theory aging: does oxidative stress play role aging healthy aging? Free <br />     145&#226;&#8364;&#8220;147.                                                                                        Radic Biol Med 48: 642&#226;&#8364;&#8220;655. <br />  7. Harman D (1956) Aging: theory based free radical radiation chemistry.              33. Lapointe J, Hekimi S (2009) When theory aging ages badly. Cell Mol Life <br />                                                                                                     Sci 67: 1&#226;&#8364;&#8220;8. <br />     J Gerontol 11: 298&#226;&#8364;&#8220;300. <br />                                                                                                 34. Yang W, Hekimi S (2010) A mitochondrial superoxide signal triggers increased <br />  8. Gavrilov LA, Gavrilova NS (2001) The reliability theory aging longevity. <br />                                                                                                     longevity Caenorhabditis elegans. PLoS Biol 8: e1000556. <br />     J Theor Biol 213: 527&#226;&#8364;&#8220;545. <br />                                                                                                 35. Kowald A, Lehrach H, Klipp E (2006) Alternative pathways mechanism for <br />  9. Kowald A, Kirkwood TB (1996) A network theory ageing: interactions of <br />                                                                                                     negative effects associated overexpression superoxide dismutase. <br />     defective mitochondria, aberrant proteins, free radicals scavengers the <br />                                                                                                     J Theor Biol 238: 828&#226;&#8364;&#8220;840. <br />     ageing process. Mutat Res 316: 209&#226;&#8364;&#8220;236.                                                     36. Liochev SI, Fridovich I (2007) The effects superoxide dismutase H2O2 <br /> 10. Bandy B, Davison AJ (1990) Mitochondrial mutations increase oxidative                       formation. Free Radic Biol Med 42: 1465&#226;&#8364;&#8220;1469. <br />     stress: implications carcinogenesis aging? Free Radic Biol Med 8:                   37. Gardner R, Salvador A, Moradas-Ferreira P (2002) Why does SOD <br />     523&#226;&#8364;&#8220;539.                                                                                        overexpression enhance, decrease, hydrogen peroxide <br /> 11. Halliwell B, Gutteridge JMC (2007) Free Radicals Biology Medicine.                       production? A minimalist explanation. Free Radic Biol Med 32: 1351&#226;&#8364;&#8220;1357. <br />     Oxford: Oxford Science Publications. 704 p.                                                 38. Liochev SI, Fridovich I (2002) Superoxide nitric oxide: consequences of <br /> 12. Van Raamsdonk JM, Hekimi S (2010) Reactive Oxygen Species Aging                          varying rates production consumption: theoretical treatment. Free <br />     Caenorhabditis elegans: Causal Casual Relationship? Antioxid Redox Signal                    Radic Biol Med 33: 137&#226;&#8364;&#8220;141. <br />     13: 1911&#226;&#8364;&#8220;1953.                                                                              39. Rea SL (2005) Metabolism Caenorhabditis elegans Mit mutants. Exp <br /> 13. Gruber J, Schaffer S, Halliwell B (2008) The mitochondrial free radical theory               Gerontol 40: 841&#226;&#8364;&#8220;849. <br />     ageing&#226;&#8364;&#8220;where stand? Front Biosci 13: 6554&#226;&#8364;&#8220;6579.                                       40. Herndon LA, Schmeissner PJ, Dudaronek JM, Brown PA, Listner KM, et al. <br /> 14. Gems D (2009) Ageing oxidants nematode Caenorhabditis elegans.                       (2002) Stochastic genetic factors influence tissue-specific decline ageing C. <br />     SEB Exp Biol Ser 62: 31&#226;&#8364;&#8220;56.                                                                     elegans. Nature 419: 808&#226;&#8364;&#8220;814. <br /> 15. Bielas JH, Loeb LA (2005) Quantification random genomic mutations. Nat                   41. Schaffer S, Gruber J, Ng LF, Fong S, Wong YT, et al. (2010) The effect of <br />     Methods 2: 285&#226;&#8364;&#8220;290.                                                                             dichloroacetate health- lifespan C. elegans. Biogerontology: [Epub <br /> 16. Gruber J, Ng LF, Poovathingal SK, Halliwell B (2009) Deceptively simple                     ahead print]. <br />     simply deceptive&#226;&#8364;&#8220;Caenorhabditis elegans lifespan studies: considerations                42. Alper S, McElwee MK, Apfeld J, Lackford B, Freedman JH, et al. (2010) The <br />     aging antioxidant effects. FEBS Lett 583: 3377&#226;&#8364;&#8220;3387.                                        Caenorhabditis elegans germ line regulates distinct signaling pathways control <br /> 17. Friedman DB, Johnson TE (1988) Three mutants extend mean                          lifespan innate immunity. J Biol Chem 285: 1822&#226;&#8364;&#8220;1828. <br />     maximum life span nematode, Caenorhabditis elegans, define age-1                 43. TeKippe M, Aballay A (2010) C. elegans germline-deficient mutants respond to <br />     gene. J Gerontol 43: B102&#226;&#8364;&#8220;109.                                                                  pathogen infection using shared distinct mechanisms. PLoS One 5: e11777. <br /> 18. Kenyon C (2005) The plasticity aging: insights long-lived mutants. Cell             44. Shoyama T, Ozaki T, Ishii N, Yokota S, Suda H (2007) Basic principle the <br />     120: 449&#226;&#8364;&#8220;460.                                                                                   lifespan nematode C. elegans. Mech Ageing Dev 128: 529&#226;&#8364;&#8220;537. <br /> 19. Yasuda K, Ishii T, Suda H, Akatsuka A, Hartman PS, et al. (2006) Age-related                45. Shoyama T, Shimizu Y, Suda H (2009) Decline oxygen consumption <br />     changes mitochondrial structure function Caenorhabditis elegans.                      correlates lifespan long-lived short-lived mutants Caenorhabditis <br />     Mech Ageing Dev 127: 763&#226;&#8364;&#8220;770.                                                                   elegans. Exp Gerontol 44: 784&#226;&#8364;&#8220;791. <br /> 20. Suda H, Shouyama T, Yasuda K, Ishii N (2005) Direct measurement oxygen                   46. Brys K, Castelein N, Matthijssens F, Vanfleteren JR, Braeckman BP (2010) <br />     consumption rate nematode Caenorhabditis elegans using optical                     Disruption insulin signalling preserves bioenergetic competence mitochon- <br />     technique. Biochem Biophys Res Commun 330: 839&#226;&#8364;&#8220;843.                                             dria ageing Caenorhabditis elegans. BMC Biol 8: 91. <br /> 21. Braeckman BP, Houthoofd K, De Vreese A, Vanfleteren JR (2002) Assaying                      47. Artal-Sanz M, Tavernarakis N (2009) Prohibitin couples diapause signalling to <br />     metabolic activity ageing Caenorhabditis elegans. Mech Ageing Dev 123:                       mitochondrial metabolism ageing C. elegans. Nature 461: 793&#226;&#8364;&#8220;797. <br />     105&#226;&#8364;&#8220;119.                                                                                    48. Kampkotter A, Gombitang Nkwonkam C, Zurawski RF, Timpel C, Chovolou Y, <br />                                                                                                     et al. (2007) Effects flavonoids kaempferol fisetin thermotolerance, <br /> 22. Hosokawa H, Ishii N, Ishida H, Ichimori K, Nakazawa H, et al. (1994) Rapid <br />                                                                                                     oxidative stress FoxO transcription factor DAF-16 model organism <br />     accumulation fluorescent material aging oxygen-sensitive mutant <br />                                                                                                     Caenorhabditis elegans. Arch Toxicol 81: 849&#226;&#8364;&#8220;858. <br />     mev-1 Caenorhabditis elegans. Mech Ageing Dev 74: 161&#226;&#8364;&#8220;170. <br />                                                                                                 49. Headlam HA, Davies MJ (2004) Markers protein oxidation: different oxidants <br /> 23. Gerstbrein B, Stamatas G, Kollias N, Driscoll M (2005) In vivo spectro- <br />                                                                                                     rise variable yields bound released carbonyl products. Free Radic <br />     fluorimetry reveals endogenous biomarkers report healthspan dietary                    Biol Med 36: 1175&#226;&#8364;&#8220;1184. <br />     restriction Caenorhabditis elegans. Aging Cell 4: 127&#226;&#8364;&#8220;137.                               50. Gardner R, Moradas-Ferreira P, Salvador A (2006) Why does superoxide <br /> 24. Gems D, Doonan R (2009) Antioxidant defense aging C. elegans:                     dismutase overexpression increase hydrogen peroxide concentrations? An <br />     oxidative damage theory aging wrong? Cell Cycle 8: 1681&#226;&#8364;&#8220;1687.                                alternative explanation. J Theor Biol 242: 798&#226;&#8364;&#8220;800. <br /> 25. Adachi H, Fujiwara Y, Ishii N (1998) Effects oxygen protein carbonyl              51. Armstrong JS, Whiteman M, Yang H, Jones DP (2004) The redox regulation of <br />     aging Caenorhabditis elegans mutants long (age-1) short (mev-1) life                intermediary metabolism superoxide-aconitase rheostat. Bioessays 26: <br />     spans. J Gerontol A Biol Sci Med Sci 53: B240&#226;&#8364;&#8220;244.                                              894&#226;&#8364;&#8220;900. <br /> 26. McCord JM, Fridovich I (1968) The reduction cytochrome c milk xanthine                52. Brand MD (2000) Uncoupling survive? The role mitochondrial inefficiency <br />     oxidase. J Biol Chem 243: 5753&#226;&#8364;&#8220;5760.                                                            ageing. Exp Gerontol 35: 811&#226;&#8364;&#8220;820. <br /> 27. Back P, Matthijssens F, Vlaeminck C, Braeckman BP, Vanfleteren JR (2010)                    53. Iser WB, Kim D, Bachman E, Wolkow C (2005) Examination the <br />     Effects sod gene overexpression <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>deletion mutation</span> expression                      requirement ucp-4, putative homolog mammalian uncoupling proteins, <br />     profiles reporter genes major detoxification pathways Caenorhabditis                   stress tolerance longevity C. elegans. Mech Ageing Dev 126: <br />     elegans. Exp Gerontol 45: 603&#226;&#8364;&#8220;610.                                                              1090&#226;&#8364;&#8220;1096. <br /> 28. Doonan R, McElwee JJ, Matthijssens F, Walker GA, Houthoofd K, et al. (2008)                 54. Yan LJ, Levine RL, Sohal RS (1997) Oxidative damage aging targets <br />     Against oxidative damage theory aging: superoxide dismutases protect                     mitochondrial aconitase. Proc Natl Acad Sci U S A 94: 11168&#226;&#8364;&#8220;11172. <br />     oxidative stress little effect life span Caenorhabditis        55. Yan LJ, Sohal RS (1998) Mitochondrial adenine nucleotide translocase is <br />     elegans. Genes Dev 22: 3236&#226;&#8364;&#8220;3241.                                                               modified oxidatively aging. Proc Natl Acad Sci U S A 95: 12896&#226;&#8364;&#8220;12901. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                         15                                     May 2011 | Volume 6 | Issue 5 | e19444 <br />                                                                                                                      SOD Mutants Mitochondrial Aging C. elegans <br />  <br />  <br /> 56. Keyer K, Imlay JA (1997) Inactivation dehydratase [4Fe-4S] clusters                     species molecular compensation. J Gerontol A Biol Sci Med Sci 64: <br />     disruption iron homeostasis cell exposure peroxynitrite. J Biol Chem                530&#226;&#8364;&#8220;539. <br />     272: 27652&#226;&#8364;&#8220;27659.                                                                        69.   Yasuda K, Hartman PS, Ishii T, Suda H, Akatsuka A, et al. (2011) <br /> 57. Shahrestani P, Mueller LD, Rose MR (2009) Does aging stop? Curr Aging Sci 2:                   Interrelationships mitochondrial fusion, energy metabolism and <br />     3&#226;&#8364;&#8220;11.                                                                                          oxidative stress development Caenorhabditis elegans. Biochem <br /> 58. Dingley S, Polyak E, Lightfoot R, Ostrovsky J, Rao M, et al. (2010)                            Biophys Res Commun 404: 751&#226;&#8364;&#8220;755. <br />     Mitochondrial respiratory chain dysfunction variably increases oxidant stress            70.   Bratic I, Hench J, Trifunovic A (2010) Caenorhabditis elegans model system <br />     Caenorhabditis elegans. Mitochondrion 10: 125&#226;&#8364;&#8220;136.                                          mtDNA replication defects. Methods 51: 437&#226;&#8364;&#8220;443. <br /> 59. Kowaltowski AJ, Souza-Pinto NC, Castilho RF, Vercesi AE (2009)                        71.   Stiernagle T (2006) Maintenance C. elegans. Worm Book:1&#226;&#8364;&#8220;11. <br />     Mitochondria reactive oxygen species. Free Radic Biol Med 47: 333&#226;&#8364;&#8220;343.               72.   Melov S, Lithgow GJ, Fischer DR, Tedesco PM, Johnson TE (1995) Increased <br /> 60. Lemire BD, Behrendt M, DeCorby A, Gaskova D (2009) C. elegans longevity                        frequency deletions mitochondrial genome age Caenorhabditis <br />     pathways converge decrease mitochondrial membrane potential. Mech                           elegans. Nucleic Acids Res 23: 1419&#226;&#8364;&#8220;1425. <br />     Ageing Dev 130: 461&#226;&#8364;&#8220;465. <br />                                                                                              73.   Bligh EG, Dyer WJ (1959) A rapid method total lipid extraction and <br /> 61. Hanak P, Jezek P (2001) Mitochondrial uncoupling proteins phylogenesis&#226;&#8364;&#8220; <br />                                                                                                    purification. Can J Biochem Physiol 37: 911&#226;&#8364;&#8220;917. <br />     UCP4 ancestral uncoupling protein. FEBS Lett 495: 137&#226;&#8364;&#8220;141. <br />                                                                                              74.   Shui G, Bendt AK, Pethe K, Dick T, Wenk MR (2007) Sensitive profiling of <br /> 62. Brand MD, Affourtit C, Esteves TC, Green K, Lambert AJ, et al. (2004) <br />                                                                                                    chemically diverse bioactive lipids. J Lipid Res 48: 1976&#226;&#8364;&#8220;1984. <br />     Mitochondrial superoxide: production, biological effects activation of <br />     uncoupling proteins. Free Radic Biol Med 37: 755&#226;&#8364;&#8220;767.                                    75.   Shui G, Guan XL, Gopalakrishnan P, Xue Y, Goh JS, et al. (2010) <br /> 63. Kokoszka JE, Coskun P, Esposito LA, Wallace DC (2001) Increased                                Characterization substrate preference Slc1p Cst26p Saccharomyces <br />     mitochondrial oxidative stress Sod2 (+/2) mouse results age-                     cerevisiae using lipidomic approaches LPAAT activity assay. PLoS One <br />     r <br /> </body></html>