<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Transcript Cleavage Factor Mycobacterium <br /> tuberculosis Important Its <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>Survival</span> <br /> Arnab China1, Sonakshi Mishra1, Valakunja Nagaraja1,2* <br /> 1 Department Microbiology Cell Biology, Indian Institute Science, Bangalore, India, 2 Jawaharlal Nehru Centre Advanced Scientific Research, Bangalore, India <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      After initiation transcription, number proteins participate elongation termination modifying the <br />      properties RNA polymerase (RNAP). Gre factors group conserved bacteria. They regulate <br />      transcription projecting N-terminal coiled-coil domain active center RNAP secondary <br />      channel stimulating hydrolysis newly synthesized RNA backtracked elongation complexes. Rv1080c a <br />      putative gre factor (MtbGre) genome Mycobacterium tuberculosis. The protein enhanced efficiency promoter <br />      clearance lowering abortive transcription rescued arrested paused elongation complexes GC rich <br />      mycobacterial template. Although MtbGre similar domain organization shares key residues catalysis RNAP <br />      interaction Gre factors Escherichia coli, complement E. coli gre deficient strain. Moreover, MtbGre <br />      failed rescue E. coli RNAP stalled elongation complexes, indicating importance specific protein-protein interactions <br />      transcript cleavage. Decrease level MtbGre reduced bacterial survival fold indicating essential <br />      role mycobacteria. Another Gre homolog, Rv3788 functional transcript cleavage activity indicating single <br />      Gre sufficient efficient transcription M. tuberculosis genome. <br />  <br />   Citation: China A, Mishra S, Nagaraja V (2011) A Transcript Cleavage Factor Mycobacterium tuberculosis Important Its Survival. PLoS ONE 6(7): e21941. <br />   doi:10.1371/journal.pone.0021941 <br />   Editor: Tanya Parish, Queen Mary University London, United Kingdom <br />   Received April 15, 2011; Accepted June 13, 2011; Published July 8, 2011 <br />   Copyright: &#195;&#376; 2011 China et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The Phosphorimager Facility IISc supported Department Biotechnology, Government India, acknowledged. AC recipient <br />   Senior Research Fellowship Council Scientific Industrial Research, Government India. VN recipient J.C. Bose Fellowship the <br />   Department Science Technology Center Excellence Tuberculosis Research grant Department Biotechnology, Government India. The <br />   funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: vraj@mcbl.iisc.ernet.in <br />  <br />  <br />  <br /> Introduction                                                                            No information properties transcript cleavage factors <br />                                                                                         available genus mycobacteria harbors several <br />    Once process transcription initiated RNAP,                        pathogenic species. In manuscript character- <br /> important enzyme carry elongation termination                        istics M. tuberculosis Gre factor. <br /> <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> ensure full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> RNA synthesis. However, movement                              The genome M. tuberculosis harbors gre factor - Rv1080c [18], <br /> of RNAP template transcription                                 sharing 32% 26% identity (48% 43% similarity) the <br /> elongation uniform gets interrupted accidentally                      E. coli GreA GreB respectively. Other ORFs low <br /> or regulatory mechanisms [1]. Inadvertent disruption                      degree similarity E. coli Gre factors genome are <br /> elongation complex lead accumulation non-                               Rv3788 shares 16% identity 33% similarity the <br /> functional RNA potentially deleterious cell                         E. coli GreA (Figure S1A) Rv2103 &#226;&#8364;&#8220; hypothetical protein, <br /> [2]. To overcome interruptions, number transcription                         having lower similarity (9% identity 21% similarity with <br /> factors act elongation termination modifying                          E. coli GreA). The Gre like domain organization while <br /> properties RNAP [1,3,4]. These factors deal accidental                      lacks key acidic amino acids domains required <br /> disruption elongation process affect transcription                           Gre like activity. <br /> processivity fidelity modulating pausing, arrest, termination                        A number molecular processes significant differences in <br /> or anti-termination enzyme [1,5]. Prokaryotic transcript                         mycobacteria compared well-studied bacterial systems <br /> cleavage factors GreA GreB [6,7] eukaryotic analog,                       [19]. Presence large number sigma factors recognizing <br /> elongation factor TFIIS [8], stimulate intrinsic transcript cleavage                    unique sequences promoters GC rich genomes [20], <br /> activity RNAP [9,10] removal aberrant RNA 39 ends                         slow rates transcription macromolecular synthesis [21,22] <br /> so polymerization activity restored end                       occurrence novel transcription activators [18] etc. point <br /> cleaved RNA. They suppress RNAP pausing rescue arrested                          differences transcription process. The GC rich <br /> [7,11] road-blocked [12] transcription complexes, providing                          genome M. tuberculosis (65.6% G+C) pose additional <br /> RNAP second chance resume elongation [13] directly                              challenges transcribing RNAP role Gre <br /> accessing RNAP active center secondary channel                          factor critical high fidelity transcription. We <br /> [10,14]. Although homologs Gre factors                         demonstrate Rv1080c, primary Gre factor genome <br /> bacteria, characterized species viz.                      essential <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> unlike Gre factors characterized <br /> E. coli [6,7], Thermus thermophilus Thermus aquaticus [15,16,17].                   eubacteria. The protein needed efficient <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                  1                                   July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                         Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br /> promoter escape reducing abortive initiation anti-arrest          mirroring high degree similarity (Figure S5A). However, its <br /> action transcription elongation. Although properties              transcript cleavage activity appears higher compared the <br /> resemble E. coli GreA respects, does appear                MtbGre. In E. coli, GreA - induced hydrolysis generates mostly <br /> collaborate E. coli RNAP elongation process             shorter di- tri-nucleotides (type I cleavage), GreB - <br /> of properties tailored mycobacterial                  induced hydrolysis generates variable length fragments to <br /> transcription.                                                               18 nt length (type II cleavage) depending extent of <br />                                                                              RNAP backtracking [6,7,28]. The <span id='am-399' about='xsp:pattern' typeof='owl:Thing'>pattern</span> shown Figure 2A and <br /> Results                                                                      2B Figure S5A indicate mycobacterial Gre factor follows <br />                                                                              type I cleavage. <br /> Rv1080c Gre factor like domain organization                                 The MtbGre homolog &#226;&#8364;&#8220; Rv3788 protein 161 amino acids <br />    Rv1080c encodes 164 amino-acid protein having sequence              predicted coiled coil N-terminal domain C-terminal <br /> similarity E. coli transcript cleavage factors GreA             globular domain (Figure S1A S1B). The key acidic residues <br /> GreB (Figure S1). A homology model protein generated              required transcript cleavage activity Gre factors the <br /> by using crystal structure E. coli GreA (PDB <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span>:1GRJ)               hydrophobic residues C-terminal RNAP interaction region <br /> [23] template (Figure S1B). GreA GreB E. coli           conserved Rv3788. However, transcript cleavage assays <br /> distinct domains: N-terminal coiled-coil (Gre-NTD) C-               presented Figure 2C Rv3788 lacks cleavage <br /> terminal globular domain (Gre-CTD) [14,24,25]. NTD                        stimulatory activity stalled elongation complexes assay <br /> responsible stimulation specific nucleolytic anti-            conditions used canonical Gre factor not <br /> arrest activities, residues Gre-CTD interact             investigated further. <br /> RNAP-b9 subunit coiled-coil domain [26,27]. <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> model, it <br /> is evident Rv1080c similar E. coli GreA             Gre factor knock-down results growth retardation in <br /> GreB surface charge distribution (Figure S1C). The                    mycobacteria <br /> homology model Rv3788, Gre homolog M.                       To check importance gre factor cell growth, anti- <br /> tuberculosis genome, shows features Gre              sense construct generated cloning M. tuberculosis gre gene <br /> factor conserved ORF (Figure S1A, S1B S1C). The               reverse orientation control constitutive hsp60 <br /> M. smegmatis Gre (MsGre) 97% similarity M. tuberculosis         promoter pMV261 (Figure S5B). This strategy been <br /> protein amino acid sequence shares similar domain                 successfully employed assess physiological importance of <br /> architecture. To understand function nature                   mycobacterial genes [29,30,31]. The expression of <br /> transcript cleavage stimulatory activity mycobacterial Gre factor         M. tuberculosis gre anti-sense reduced viability M. tuberculosis <br /> and Gre factor homolog Rv3788, genes cloned                  (Figure 3A) folds compared control cells <br /> pET20b over-expression ,18 kDa proteins E. coli                transformed pMV261 vector. M. smegmatis cells <br /> (Figure S2A, S2B S2C). The identities expressed                   transformed MtbGre anti-sense construct showed <br /> proteins confirmed peptide-mass-fingerprinting using                 reduced viability (Figure 3A) compromised growth <br /> MALDI-TOF (data shown).                                                  compared cells transformed vector MtbGre <br />                                                                              over-expressing construct (Figure 3B). Western blots cell <br /> MtbGre stimulates intrinsic cleavage activity                         lysates probed anti-Gre antibody showed highly reduced level <br /> mycobacterial RNAP                                                           Gre protein cells anti-sense construct, suggesting that <br />    A stalled elongation complex comprising 20 nt RNA                  decreased survival reduction Gre <br /> generated T7A1 promoter (T7A1-TEC) studying                     concentration cells (Figure 3C). The M. smegmatis cells over- <br /> transcript cleavage elongation complexes (Figure S3A).                expressing MtbGre factor showed elongated <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> RNAP M. smegmatis (MsRNAP) M. tuberculosis                     (Figure S5C). <br /> (MtbRNAP) proficient carrying transcription               From data, apparent decrease in <br /> template (Figure S3B). Transcript cleavage intrinsic property          intracellular Gre levels caused growth defects in <br /> of catalytic center RNAP [9] slow                 organisms. This mean balanced pool of <br /> requires prolonged incubation. First, intrinsic cleavage activity       Gre required sustain cell viability. To measure the <br /> of enzymes E. coli, M. smegmatis M. tuberculosis           endogenous levels protein, semi-quantitative western blot <br /> compared. In enzyme systems, RNA fragments                  analysis carried different stages cell growth. The <br /> varied length generated incubation hrs. Varied          expression level endogenous Gre highest mid- <br /> amount short RNA fragments generated 39 end               exponential phase, M. smegmatis M. tuberculosis <br /> stalled TEC detected the gels (Figure 1A).         (Figure S6A). The Gre concentration M. smegmatis was <br /> Both MtbRNAP MsRNAP lower intrinsic cleavage                         ,82 fmoles/mg total protein early exponential stage cells and <br /> compared E. coli RNAP (EcRNAP) (Figure 1A),                       remained level late exponential phase, <br /> cleavage activity stimulated alkaline pH similar E. coli       declined slightly 66 fmols/mg total protein the <br /> enzyme (Figure 1B) indicating conservation mechanism              stationary phase (Figure S6B). Gre levels exponentially growing <br /> across different bacterial species. However, cleavage             M. tuberculosis cells comparable levels seen M. <br /> TEC complete mycobacterial RNAPs                     smegmatis cells (Figure S6A). Interestingly, combined of <br /> alkaline pH. The slower nuclease activity seen inherent            GreA (,53 fmol/mg total protein) GreB (,13 fmol/mg of <br /> to mycobacterial polymerases co-                      total protein) [32] exponentially growing E. coli cells is <br /> purification endogenous Gre factor (Figure S4).                           comparable level single Gre protein in <br />    MtbGre factor stimulated cleavage short fragments (2&#226;&#8364;&#8220;              mycobacteria. The RNAP concentration be <br /> 3 nt) 39 end nascent RNA 20-mer T7A1-TEC,                 comparable species (Gupta Nagaraja, <br /> and 50% cleavage achieved 12 minutes            unpublished results). Next, expression Gre response to <br /> (Figure 2A 2B) indicating Rv1080c functions like           different cellular stress conditions M. smegmatis determined <br /> Gre factor. The pattern seen MsGre nearly identical                 measuring protein content, mostly <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                              July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                       Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Intrinsic transcript cleavage property RNAP. Stalled elongation complexes bearing 20 mer transcript generated with <br /> M. smegmatis (Ms), M. tuberculosis (Mtb) E. coli (Ec) RNAP respectively. The complexes incubated prolonged time (1&#226;&#8364;&#8220;4 hrs) in <br /> transcription buffer (pH 7.5), followed resolving cleavage products 20% urea-PAGE. (B) pH-induced transcript-cleavage activity RNAP. <br /> The gels cleaved RNA generated 20 mer ternary complexes formed Ms, Mtb Ec RNAP buffers pH 6.0 10.0. <br /> doi:10.1371/journal.pone.0021941.g001 <br />  <br />  <br /> unperturbed (Figure S6C). RT-PCR experiments various                 cleavage transcript paused elongation complex, the <br /> conditions did significant alterations gre            trimmed TEC capable restarting transcription in <br /> mRNA levels (data shown). Together, results indicate             presence NTPs T7A1 promoter mycobac- <br /> that constant level Gre retained irrespective growth            terial PrrnB promoter templates (Figure 5A, 5B). However, the <br /> phases environmental conditions. Above findings contrast         minor differences patterns Figures 5A 5B be <br /> to observations organisms                 template specific effect. It possible stalled <br /> different stress conditions GreA level altered             elongation complexes generated T7A1 template entered <br /> [33,34]. Thus results presented Figure 3A 3C (gre       inactive arrested state elongated further. <br /> knock-down) Figure S6A S6C, surmise                Taken together, data experiments indicate MtbGre <br /> amount Gre mycobacteria comparable E.              factor function pre-formed stalled elongation complexes <br /> coli, maintaining level critical cell survival.                 induce transcript cleavage-restart activity. <br />  <br /> Reduction abortive transcription initiation, anti-                  Structural features Gre factors conserved MtbGre <br /> arrest activity MtbGre                                                      Alignment MtbGre E. coli counterparts revealed <br />    To determine activity MtbGre, vitro transcriptions       following conserved features (Figure 6A). (i) Acidic amino acids <br /> carried using M. smegmatis PrrnPCL1 template. The efficient       tip predicted coiled-coil domain N- <br /> open complex (RPO) formation effectively transmitted         terminus protein. In E. coli Gre factors, residues are <br /> synthesis length transcripts promoter high          involved Mg2+ co-ordination RNAP active center [10]. <br /> abortive RNA synthesis [35]. One properties E. coli          (ii) A short basic patch residues helix, which <br /> Gre factors reduce abortive RNA synthesis enhance                interacts 39 end RNA E. coli [38]. (iii) A globular <br /> promoter clearance [36,37]. MtbGre enhanced full-length                domain C-terminus protein. Residues domain <br /> transcript synthesis PrrnPCL1 overcoming abortive              E. coli GreB interact carboxyl-terminal coiled-coil <br /> transcripts (Figure 4A). Notably, intermittent pauses seen             domain RNAP b9 subunit [27]. The D43, E46 acidic tip <br /> above abortive transcripts transcription PrrnPCL1          coiled-coil domain (equivalent D36 E39 E. coli <br /> were reduced presence MtbGre (Figure 4B). After         GreA) S127 C-terminal globular domain MtbGre <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      3                              July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                         Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 2. MtbGre factor stimulates cleavage 20 mer transcript. (A) Determination <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> activity MtbGre. Stalled TEC generated <br /> with M. smegmatis RNAP incubated different concentrations MtbGre (10 nM 5 mM) 30 mins. Reactions terminated and <br /> resolved 20% urea PAGE. (B) Time-course MtbGre activity. Stalled TECs incubated 37uC 1 mM MtbGre aliquots removed <br /> at different time points quenched urea gel loading dye followed resolving 20% urea PAGE. The time required 50% cleavage of <br /> the TEC calculated plot. (C) The MtbGre homolog-Rv3788 does induce transcript cleavage. MtbGre induce cleavage of <br /> +20 nt stalled elongation complex T7A1 template (lanes 2 3). Rv3788 does detectable transcript cleavage stimulatory activity (lanes 4 <br /> and 5). <br /> doi:10.1371/journal.pone.0021941.g002 <br />  <br />  <br /> factor (equivalent E. coli GreA S119) (Figure 6A) mutated            cleavage stimulation activity (Figure 6B). These results indicate <br /> to D43N, E46R, S127E address function MtbGre.                acidic residues tip N-terminal <br /> The D43N S127E mutations completely abolished activity               predicted coiled-coil domain, D43 essential transcript <br /> of MtbGre factor. On hand, E46R mutant retained                cleavage activity. The loss activity S127E mutant was <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        4                               July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                           Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 3. Knock-down gre mycobacteria deleterious cell growth. (A) The pMV261-vector control, MtbGre anti-sense <br /> (pMVgreAS) MtbGre over-expressing (pMVgreOE) M. smegmatis M. tuberculosis H37Ra cells grown 18 hrs 7 days respectively in <br /> liquid cultures 37uC shaking condition, serially diluted (1027 vector control over-expression 1025 anti-sense) plated to <br /> determine cell viability. (B) Growth curve M. smegmatis cells vector control, MtbGre anti-sense MtbGre over-expression constructs. <br /> Cultures diluted Middlebrook 7H9 broth initial OD 600 0.02 0.04 incubated 36 hours. The growth curves plotted <br /> by measuring cell viability dilution plating different time points. (C) Western blot M. smegmatis cell lysates vector control (lane 1) <br /> and anti-sense MtbGre construct (lane 2) using polyclonal antibody MtbGre. Purified MtbGre used positive control (lane 3). <br /> doi:10.1371/journal.pone.0021941.g003 <br />  <br />  <br /> probably loss interaction RNAP. Ni-NTA                  increased transcription efficiency initiation and <br /> pull assays carried assess direct interaction             elongation phase process. During initiation, reduced the <br /> between purified MtbRNAP histidine tagged MtbGre                    abortive transcripts enhanced promoter clearance. At <br /> S127E variant. The MtbGre factor bound MtbRNAP (Lane 4                      elongation phase, protein rescued RNAP transcrip- <br /> Figure 6C), predicted S127E mutant did interact                tion pauses inducing transcript cleavage. Knocking of <br /> the RNAP (Lane 6 Figure 6C).                                                gene resulted growth retardation cell death indicating <br />                                                                                essentiality cell survival. <br /> MtbGre factor specific mycobacterial RNAP                               In organisms Gre factors analyzed far, they <br />    The MtbGre factor shares similar structural features (Figure 7A)            remarkably similar structural features. Functional charac- <br /> with E. coli GreA rescue halted elongation complexes.                terization Gre factors E. coli [6,7], T. aquaticus T. <br /> Therefore, ability MtbGre functionally complement E.             thermophilus [15,17,40] revealed conserved nature the <br /> coli Gre factors tested using E. coli DgreA/DgreB double             transcript cleavage stimulation activity required efficient <br /> <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>knock-out</span> strain [39], shows cold-sensitive phenotype.                 transcription process. However, gre genes be <br /> MtbGre factor expressed pTrc construct                        dispensable E. coli; DgreA - DgreB double knock-out strain showed <br /> complement E. coli DgreA/DgreB grown 28uC (Figure 7B)                       mild cold-sensitive phenotype [39]. In contrast, M. <br /> although protein expressed E. coli (Figure S7A). The                tuberculosis, protein appears pronounced and <br /> failure complement lack interaction                  indispensable role. In glance results appear be <br /> between E. coli RNAP MtbGre (Figure 7C). In support this,               contradicting earlier transposon mutagenesis studies led <br /> in vitro assays showed MtbGre factor functions                    isolation insertional <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>mutation</span> M. tuberculosis gre (http:// <br /> mycobacterial, i.e., M. smegmatis M. tuberculosis TECs                     mylims2.cvmbs.colostate.edu/tnlist/). We noticed the <br /> (Figure 7D). It did stimulate transcript cleavage E. coli               point insertion transposon 493rd position of <br /> RNAP containing TEC high concentration                          495 bases Rv1080c. Thus likely that, gene was <br /> (.10 mM). Similarly, E. coli GreA functional               inactivated mutant strain. Also, decrease in <br /> mycobacterial elongation complexes (Figure S7B).                               intracellular Gre levels, cell survival affected. Notably, <br />                                                                                significant protein present growth phases <br /> Discussion                                                                     indicating house-keeping function. Further, Gre protein <br />                                                                                level altered great extent different stress <br />   In study, characterization Rv1080c -             conditions, indicating optimum level protein be <br /> Gre factor present M. tuberculosis genome. The MtbGre                   required cell survival. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          5                               July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                        Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 4. Effect MtbGre factor promoter clearance abortive transcription. (A) Promoter clearance assays carried the <br /> absence (-N-) presence (-D-) 2 mM MtbGre. Transcripts resolved 8% urea-PAGE 109 nt long run-off transcripts quantified <br /> using Image Guage (Fuji Film) plotted (lower panel). The intensity bands normalized run-off transcript <br /> produced 24 mins presence MtbGre. (B) MtbGre reduces abortive transcript level. In assay, transcription reactions carried out <br /> in absence presence 1 mM 2 mM MtbGre. The reactions resolved 20% urea-PAGE visualize abortive transcripts (marked <br /> by &#226;&#8364;&#732;*&#226;&#8364;&#8482;). <br /> doi:10.1371/journal.pone.0021941.g004 <br />  <br />  <br />  <br />    MtbGre rescue pre-formed halted elongation complex              C-terminal globular domain Gre factors (GreA, GreB E. <br /> exert anti-arrest activity similar E. coli GreA ensure           coli MtbGre), interacts RNAP, shows <br /> efficient transcription elongation. The transcript cleavage pattern         considerable variation, certain specific residues the <br /> of MtbGre showed type I cleavage products i.e. predominantly 2&#226;&#8364;&#8220;             hydrophobic patch conserved proteins. Importance <br /> 3 nt fragments similar activity E. coli GreA. The longer          specific interactions RNAP Gre suggested from <br /> transcript cleavage pattern (2&#226;&#8364;&#8220;18 nt, type II) seen E. coli GreB       studies T. aquaticus. GreA T. aquaticus failed induce <br /> is mediated large stretch positively charged residues        transcript cleavage EcRNAP elongation complexes [15] similar <br /> N-terminal domain [38]. MtbGre does large                   present observation MtbGre. Thus appears the <br /> stretch basic amino acids surface charge distribution         transcript cleavage activity requires species-specific interactions, <br /> similar E. coli GreA (Figure S1C). In organisms having           partners viz RNAP Gre conserved <br /> GreB, RNAP backtrack farther larger RNA 39 end              characteristics species. Gre important <br /> fragment processed. Indeed, conditions, high affinity         function mycobacteria compensate low intrinsic <br /> interaction RNAP GreB results transcript cleavage            cleavage activity mycobacterial RNAP compared E. coli <br /> activity [16,27]. Earlier studies revealed lower transcription         themophilic counterparts. This deficiency affect the <br /> elongation rates mycobacteria [41,42]. Organisms E. coli         recovery arrest backtracked MtbRNAP absence of <br /> with faster transcription rates require Gre factors          MtbGre. The similar mechanism recently proposed to <br /> process shorter longer RNA.                                             explain growth inhibition yeast strains expressing the <br />    The action MtbGre restricted                       cleavage deficient mutant eukaryotic Gre homolog, TFIIS <br /> mycobacterial transcription machinery did rescue                [43]. The results presented data emerged till date <br /> halted elongation complex E. coli RNAP. Lack interaction              number studies Gre factors diverse group of <br /> between heterologous partners account                   organisms emphasize biological importance secondary <br /> observation. The interaction surface E. coli RNAP E. coli            channel binding proteins. The deletion greA led hypersen- <br /> GreB mapped conserved hydrophobic loop coiled-              sitivity phenotype various stress conditions E. coli [39], <br /> coil domain C-terminus b9 subunit [27]. The region         Sinorhizobium meliloti [44] Rhizobium tropici [45] implicating the <br /> also conserved mycobacterial RNAP (Figure 7E) indicating             importance Gre factors survival organism the <br /> the conserved architecture transcription machinery. However,             restrictive environment. In contrast, decrease Gre levels <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       6                               July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                           Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 5. RNAP restart transcription elongation transcript cleavage MtbGre. The starting materials experiments <br /> were purified stalled complexes radiolabeled RNA. (A) Cleavage-restart assay T7A1-TEC. The scheme left depicts reaction process. <br /> The TEC incubated (lane 1) MtbGre (lane 2 3), NTPs (lane 3 4), indicated. (B) Similar assay carried on <br /> M. smegmatis PrnB promoter template (the RNAP stalls +39 position absence UTP reaction mix). Transcripts analyzed by <br /> resolving 20% urea PAGE. <br /> doi:10.1371/journal.pone.0021941.g005 <br />  <br /> under normal cellular growth conditions itself reduced viability           Methods <br /> of M. tuberculosis. The indispensability Gre factor M. <br /> tuberculosis E. coli [39] T. thermophilus [17] indicates         Bacterial strains, plasmids growth conditions <br /> that intracellular role factor likely varied          M. smegmatis mc2155 [47] M. smegmatis SM07sigA [48,49] <br /> different species bacteria.                                                 cultured Middlebrook 7H9 medium (Difco) containing <br />    MtbGre transcription elongation factor              0.05% Tween-80 (Sigma) 0.4% glucose (Sigma) shaking <br /> the genome possessing cleavage activity ORF -                     conditions 37uC. M. tuberculosis H37Ra [50] cells cultured <br /> Rv3788 genome lower degree relatedness                 Middlebrook 7H9 medium supplemented ADC consisting <br /> not appear participate process. The lack transcript               0.2% glycerol (Sigma) 0.05% Tween-80 37uC. To check <br /> cleavage stimulatory activity Rv3788 attributed               expression pattern Gre different growth phases M. <br /> absence key residues N-terminus              smegmatis M. tuberculosis, cells grown 12, 18, 24, 30, 36 <br /> in Gre factors different organisms. Although acidic             48 hrs (for M. smegmatis) 3, 5, 7, 12 20 days (for M. <br /> residues needed Mg2+ co-ordination conserved Rv3788                 tuberculosis), pelleted centrifugation, lysed sonication <br /> (Figure S1A), Asn47 Tyr50 (present MtbGre), required                cell extracts prepared. <br /> binding backtracked protruding nascent RNA absent.                     Knock-down gre expression M. smegmatis mc2155 M. <br /> Nevertheless, Rv3788, features similar RNAP                 tuberculosis H37Ra carried generating plasmid <br /> secondary channel binding proteins                     pMVgreAS (Mtbgre anti-sense orientation) pMV261 [51]. The <br /> other intracellular role. It apparent RNAP                    coding sequence amplified using primers BamH1 site <br /> secondary channel binding proteins emerging key                  (Table 1) cloned downstream hsp60 promoter a <br /> regulators different cellular functions apart transcript           BamH1 site vector pMV261 generate plasmid pMVgreOE <br /> cleavage stimulatory functions [5].                                            (Table 1) over-expression MtbGre M. smegmatis and <br />    In conclusion, Rv1080c functions like bona fide Gre factor                M. tuberculosis. Comparison growth rates different strains <br /> with transcript cleavage stimulatory activity M. tuberculosis. Gre          carried inoculating (1% inoculum) 30 ml of <br /> function required optimal growth mycobacteria             Middlebrook 7H9 medium 25 mg ml21 kanamycin obtain <br /> contrast dispensability E. coli. GC rich templates               initial OD600 0.02 0.04. Growth strains was <br /> known impose blockage transcription                       monitored dilution - plating 8 day culture M. tuberculosis <br /> formation stable RNA-DNA hybrids [46]. Such strong barriers                 20 hrs cultures M. smegmatis grown 37uC shaking <br /> have overcome ensure high fidelity RNA synthesis. Slower              conditions. The cells diluted fresh media plated into <br /> transcription rates mycobacteria lead intermittent pauses            middlebrook 7H10 agar plates determine cell viability <br /> and stalling specific signals. Under circumstances RNAP               counting cfu. <br /> has ensure completing elongation process. Transcription <br /> factors like Gre, maintain efficiency preventing                  Western blots <br /> premature pauses, appear profound role                          To detect protein level different growth phases, cell <br /> maintaining genomic integrity M. tuberculosis.                          lysates probed Gre factor polyclonal antibody <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          7                              July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                       Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 6. Conserved residues MtbGre factor important Mg++ co-ordination RNAP binding. (A) Multiple sequence <br /> alignment MtbGren (164 aa) E. coli GreA (158 aa) GreB (158 aa). N-terminal coiled-coil domain marked blue basic patch in <br /> black. The C-terminus RNAP interaction domain marked red alignment. Conserved acidic residues tip N- terminus coiled coil <br /> domain S127 C-terminus, subjected site directed mutagenesis marked &#226;&#8364;&#732;*&#226;&#8364;&#8482;.(B) Comparative activity Wt D43N, E46R and <br /> S127E mutants T7A1 TEC. (C) Ni-NTA pull his-tagged Wt S127E mutant MtbRNAP. Lane-1 2 represent supernatant and <br /> pellet fraction control reaction having MtbRNAP. Lanes 3,4 represent supernatant pellet fraction WtGre respectively and <br /> lanes 5 6 represents mutant S127E MtbRNAP respectively. <br /> doi:10.1371/journal.pone.0021941.g006 <br />  <br />  <br />  <br /> raised mice anti-SigA antibody rabbit. The primary                different growth phase cultures subsequently probed with <br /> antibodies probed secondary antibody coupled             anti-Gre antibody. <br /> HRP blots developed using chemiluminescence <br /> substrate (GE Health Care). Expression Gre factor                 Microscopy <br /> different stress conditions checked growing M.                    M. smegmatis cells harboring pMV261 pMVgreAS or <br /> smegmatis cells till mid-log phase subjecting varied            pMVgreOE constructs grown Middlebrook 7H9 medium <br /> stresses described [52]. The Gre protein present            37uC mid-exponential phase. Cells pre-fixed PBS, <br /> the M. smegmatis cells determined western blot. Varying              1% (v/v) Triton X-100 (Sigma) 2% (v/v) toluene (Merck) <br /> concentrations purified M. smegmatis Gre loaded          solution incubated overnight 4uC. Cells stained with <br /> same gel standards 120 mg cell extracts               DAPI solution (49,6-diamidino-2-phenylindole), binds <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       8                              July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                         Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  <br /> Figure 7. MtbGre factor specific mycobacterial TEC. (A) Homology modeling MtbGre using E. coli GreA crystal structure (PDB <span id='am-398' about='Thesaurus:code' typeof='owl:Thing'>code</span>: <br /> 1GRJ) template. (B) Complementation E. coli DgreA/DgreB strain (TK1021) Mtb gre. The gene cloned pTrc99c E. coli cells <br /> harboring different plasmids grown till OD600 0.6 spotted IPTG containing plates incubated 28uC. (C) Interaction histidine <br /> tagged MtbGre Ec Mtb RNAPs Ni-NTA pull-down. Lanes1 2 - supernatant pellet EcRNAP lanes 5 6 MtbRNAP <br /> from control reactions. Lane 4 8 represent pellet fractions reactions Ec Mtb RNAP MtbGre respectively (D) TECs <br /> prepared Ms, Mtb Ec RNAP incubated MtbGre resulting products resolved 20% urea PAGE. (E) Comparison of <br /> coiled-coil domain present C-terminus RNAP b9 subunit M. tuberculosis, M. smegmatis, T. thermophillus E. coli. Residues marked <br /> with &#226;&#8364;&#732;+&#226;&#8364;&#8482; charged polar amino acids present loop region mycobacterial b9 subunit absent two. <br /> doi:10.1371/journal.pone.0021941.g007 <br />  <br /> specifically DNA. Microscopic observations carried            listed Table 1). The purification MtbGre, mutants and <br /> using Carl Zeiss fluorescent microscope 10006 magnification.            MsGre carried follows. E. coli BL21 cells [53] with <br />                                                                              pET20bgre mutants pET20bmsgre grown till OD600 <br /> Expression purification MtbGre, MsGre Rv3788                      0.6 37uC induced 0.3 mM IPTG. Cells lysed by <br />    gre (Rv1080c) Rv3788 genes PCR amplified M.                 sonication centrifuged 100,000 g 2 hrs. The superna- <br /> tuberculosis genomic DNA specific primers (Table 1)                 tants subjected 0&#226;&#8364;&#8220;65% ammonium sulfate precipitation <br /> cloned NdeI HindIII site pET20b (pET20bgre                re-suspended 3 ml TGE buffer [10 mM Tris-HCl, <br /> and pET20brv3788). The M. smegmatis gre (MSMEG_5263) gene                    pH 8.0, 5% glycerol, 0.1 mM EDTA] 50 mM NaCl and <br /> was PCR amplified M. smegmatis mc2155 genomic DNA                   subsequently resolved 120 ml Sephacryl S-100 gel filtration <br /> cloned pET20b (between NdeI HindIII site). Site directed              column. The fractions having Gre protein purified <br /> mutants Mtbgre generated using mega-primer inverse               DEAE - Sephacel chromatography eluting a <br /> PCR method pET20bgre template (primer sequences                linear NaCl gradient 50 mM 400 mM. The Rv3788 protein <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                               July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                                   Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br />  <br />  Table 1. Oligonucleotides, strains plasmids used study. <br />  <br />  <br />  Name                                Description                                                                                      Reference <br />                                          R <br />  M. smegmatis SM07sigA               (Hyg , his-rpoC, pJAM2mysA)                                                                      [46,47] <br />  M. smegmatis mc2 155                (A high efficiency transformation strain M. smegmatis)                                        [45] <br />  M. tuberculosis H37Ra               (An attenuated strain M. tuberculosis H37Ra)                                                  [48] <br />  E. coli BL21                        (hsdS gal (lcIts857 ind1 Sam7 nin5 lacUV5-T7 gene 1)                                             [51] <br />  E. coli TK1001                      MC1061 zgj-203::Tn10                                                                             [39] <br />  E. coli TK1021                      MC1061 greA::kan, greB::cat, zgj-203::Tn10                                                       [39] <br />  pMS002                              Derivative pBR322 containing Mtb greA gene                                                    [39] <br />  pMV261                              E. coli-mycobacteria shuttle vector hsp60 promoter                                        [49] <br />  pET20bgre                           M. tuberculosis gre cloned NdeI HincII pET20b                                     This study <br />  pET20brv3788                        M. tuberculosis Rv3788 cloned NdeI HindIII pET20b                                 This study <br />  pET20bmsgre                         M. smegmatis gre cloned NdeI HindIII pET20b                                       This study <br />  pET20bgre-his                       M. tuberculosis gre cloned pET20bwith C-terminus his-tag                                      This study <br />  pET20bgre-his                       M. tuberculosis gre S127E cloned pET20b C-terminus his-tag                               This study <br />  pET20bEcgreA-his                    E. coli greA cloned pET20b C-terminus his-tag                                            This study <br />  pMVgreAS                            M. tuberculosis gre cloned anti-sense orientation hsp60 promoter                        This study <br />  pMVgreOE                            M. tuberculosis gre coading sequence cloned hsp60 promoter                                 This study <br />  pTrc99gre                           M. tuberculosis gre cloned trc promoter                                                    This study <br />  Gre D43N Mut                        59 GAAGAAGGCAACCTGCGCGAGAAC 39                                                                   This study <br />  Gre E46R Mut                        59 GAAGGCGACCTGCGCCGTAACGGCGGATACCAC 39                                                          This study <br />  Gre S127E Mut                       59 TACTCGCCGAATGAACCGCTCGGTGGG 39                                                                This study <br />  greBamH1For                         59 ACGGATCCCGACCATATGACGGATACTCAAGTC 39                                                          This study <br />  greBamH1Rev                         59 ACGGATCCCGACCTGCTCGGAGATCTCGAACAG 39                                                          This study <br />  greNdeIFor                          59 CGACCATATGACGGATACTCAAGTC 39                                                                  This study <br />  greHindIIIRev                       59 ATAAGCTTCGACCTGCTCGGAGATCTCGAACAG 39                                                          This study <br />  rv3788NdeIFor                       59 ATGCGACATATGAGCGAGAAAGTCGAGTC 39                                                              This study <br />  rv3788HindIIIRev                    59 ATAAGCTTTTCTGAGGGCAGCTTGACAG 39                                                               This study <br />  MsgreNdeIFor                        59 ATGCGACATATGACCGATACCCAGGTCACC 39                                                             This study <br />  MsgreHindIIIRev                     59 ATAAGCTTTCCGCCTTGATACGGCTCAGC 39                                                              This study <br />  <br />  doi:10.1371/journal.pone.0021941.t001 <br />  <br />  <br /> was purified E. coli BL21 cells harboring pET20brv3788.                       glycerol, 50 mg ml21 BSA 100 mM KCl] 15 mins 37uC <br /> The purification involved 45&#226;&#8364;&#8220;60% ammonium sulfate precipi-                           form open complex. Subsequently 50 mg ml21 heparin was <br /> tation cell lysate followed DEAE - Sephacel                                  added reactions incubated 1 min. Transcription was <br /> chromatography. All proteins purified approximately                           initiated addition 100 mM NTPs 2 mCi of <br /> 95% pure judged SDS-PAGE (Figure S2C). From 2 liters                             a-32P[UTP]. Aliquots withdrawn different time intervals <br /> each cultures overexpressing proteins (MtbGre, MsGre                        reactions incubated indicated time. Reactions were <br /> and Rv3788), 5 mg of protein obtained. E.                       analyzed 22% urea-PAGE resolve abortive products. <br /> coli greA cloned C-terminal His-tag pET20b the <br /> protein purified E. coli BL21 cells [53] over-expressing                      Stalled TEC preparation <br /> the protein using Ni-NTA column. M. smegmatis RNAP                                  Transcription assays carried using T7A1 promoter and <br /> purified following method described earlier [49]. M.                            RNAPs E. coli, M. smegmatis M. tuberculosis. Ternary <br /> tuberculosis RNAP purified 2 liters M. tuberculosis                        elongation complexes generated 59 biotinylated T7A1 <br /> H37Ra cells grown 8 days 37uC MB7H9 medium                              promoter-containing DNA template (Figure S3A). The TECs for <br /> ADC supplement (Difco). The purification involved gel filtration                       E. coli mycobacterial RNAPs prepared following the <br /> on Superdex S-200 matrix subsequent heparin - Sepharose                            methods described E. coli T. thermophillus enzymes [15,26]. <br /> chromatography following method described native M.                            RPO formed incubating 100 nM M. smegmatis, M. <br /> smegmatis RNAP purification [49].                                                      tuberculosis E. coli RNAP 15 nM T7A1 promoter containing <br />                                                                                        template DNA 37uC 15 min transcription buffer. For <br /> Promoter clearance abortive transcription                                          multiple round transcription assays, 100 mM NTPs added <br />   100 nM RNAP 20 nM M. smegmatis PrrnPCL1 promoter                           reaction mix incubated 15 mins 37uC. <br /> containing template incubated transcription buffer                             Reactions stopped formamide dye analyzed Urea <br /> [50 mM Tris-HCl pH 7.5, 10 mM MgCl2, 100 mM DTT, 5%                                    PAGE. For stalled complex formation assays, RPO <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 10                              July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                             Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br /> formation, 100 mM ATP, 100 mM GTP 2 mCi [a-32P] ATP                           suspended 50 ml buffer mixed SDS-gel loading buffer, <br /> (300 Ci mmol21, Perkin Elmer) added. Reactions                          boiled loaded 11% SDS-PAGE the <br /> carried absence CTP UTP generate stalled                     supernatant fractions followed silver staining gel. <br /> elongation complexes containing 20 mer transcript. M. smegmatis <br /> PrrnB promoter template used preparation                           Complementation E. coli DgreA/DgreB strain with <br /> mycobacterial TEC. RNAP stalls +39 position absence                     M. tuberculosis gre <br /> of UTP template. TECs purified mixing                       The M. tuberculosis gre gene cloned pTrc99c vector to <br /> 5 ml Streptavidin-Sepharose beads (GE Healthcare)                      obtain pTrc99gre construct used complementing the <br /> reaction precipitated centrifugation. Pellets containing               E. coli TK1021 strain (Table 1). The parental strain TK1001 was <br /> elongation complexes washed thrice transcription buffer                 used wild type E. coli control. E. coli greA expressing plasmid <br /> supplemented 200 mM KCl 100 mg ml21 heparin                              pMS002 used positive control experiments [39]. <br /> followed washing twice transcription buffer. Indicated               The cells grown liquid culture different dilutions were <br /> amounts MtbGre Rv3788 added beads re-                           spotted LB plates containing 0.3 mM IPTG appropriate <br /> suspended transcription buffer followed incubation                   antibiotics (Table 1). <br /> 37uC 30 min. 150 nM MtbGre optimum for <br /> cleavage 50% T7A1 TECs defined the <br /> <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> activity. Reactions terminated addition                     Supporting Information <br /> formamide dye [0.025% (w/v) bromophenol blue, 0.025% (w/v)                        Figure S1 Sequence alignments homology modeling <br /> xylene cyanol FF, 0.08% amaranth (w/v), 10 mM EDTA, 0.025%                        Gre. (A) Multiple sequence alignment E. coli GreA and <br /> SDS 80% deionized formamide] RNA cleavage products                        GreB MtbGre Rv3788 using ClustalW program <br /> were analyzed electrophoresis 20% denaturing PAGE.                        (http://www.ebi.ac.uk/Tools/msa/clustalw2). The alignment <br /> Amaranth dye included formamide stop mix served size                  figure created using GenDoc &#226;&#8364;&#8220; multiple sequence alignment <br /> marker. In 22% urea PAGE dye moves position                            editor (http://www.psc.edu/biomed/genedoc). The conserved <br /> corresponding 2&#226;&#8364;&#8220;3 nt short RNA fragments [54].                                 amino acids shaded black substitutions similar <br />                                                                                   amino acids grey. The conserved acidic residues N- <br /> Intrinsic cleavage activity RNAP                                               terminus labeled &#226;&#8364;&#732;&#226;&#8364;&#732;&#194;&#164;&#226;&#8364;&#8482;&#226;&#8364;&#8482; hydrophobic residues the <br />    Intrinsic cleavage activity M. smegmatis, M. tuberculosis E.        C-terminus &#226;&#8364;&#732;&#226;&#8364;&#732;N&#226;&#8364;&#8482;&#226;&#8364;&#8482;. (B) Homology model MtbGre and <br /> coli RNAPs detected prolonged incubation (up 4 hrs)              Rv3788 (using E. coli GreA crystal structure &#226;&#8364;&#8220; 1GRJ as <br /> TECs (prepared 15 nM template 100 nM RNAP)                            template). Models generated using comparative <br /> transcription buffer (pH 7.5) 37uC followed resolving 20%              protein structure modeling program Modeller ver. 9.3. <br /> urea - acrylamide gel. pH - induced transcript cleavage reactions            [Eswar, N et al. Comparative Protein Structure Modeling <br /> carried different buffer systems 37uC 30 mins. (i)            With MODELLER. Current Protocols Bioinformatics, <br /> 40 mM PIPES adjusted pH 6.0 addition 1 M NaOH; (ii)                      John Wiley &amp; Sons, Inc., Supplement 15, 5.6.1&#226;&#8364;&#8220;5.6.30, <br /> 40 mM Tris adjusted pH 7.0, 8.0, 9.0 addition 1 M                2006]. (C) Surface charge distribution MtbGre and <br /> HCI; (iii) 40 mM CAPS adjusted pH 10.0 addition 1 M                  Rv3788. Positively charged surface shown blue and <br /> NaOH. All buffers contained 0.1 M KCl 10 mM MgCl2.                            negatively charged region red. E. coli GreA structure <br />                                                                                   (Stebbins et al. [23]) shown left GreB (Vassylyeva <br /> Cleavage-restart activity MtbGre                                               et al. [27]) second left compared the <br />    The 20 mer T7A1 TEC 39 mer M. smegmatis PrrnB TECs                      MtbGre (second right) Rv3788 (rightmost). <br /> were prepared using 15 nM biotinylated template 100 nM                     Positively charged region surface coiled-coil <br /> RNAP. The RNAP stalled T7A1 template using                         domain shown box. <br /> 100 mM ATP, GTP 2 mCi [a-32P] ATP (300 Cimmol21,                        (TIF) <br /> Perkin Elmer) 10 ml reaction volume. For generating <br />                                                                                   Figure S2 Over-expression Gre Rv3788. (A) Gre <br /> +39 stalled elongation complex PrrnB promoter, 100 mM ATP, <br />                                                                                   factors M. tuberculosis M. smegmatis over-expressed <br /> GTP CTP used 2 mCi [a-32P] ATP. To <br />                                                                                   purified E. coli BL21 cells. UN: induced cell lysates <br /> detect cleavage-restart activity MtbGre, TECs were <br />                                                                                   IN: induced cell lysate MtbGre MsGre over-expressing <br /> incubated MtbGre factor presence absence the <br />                                                                                   cells respectively. (B) Over-expression M. tuberculosis Rv3788 in <br /> four NTPs. Initially complexes incubated 2 mM of <br />                                                                                   E. coli BL21 cells. Both induced IPTG induced samples of <br /> MtbGre 30 min followed addition NTPs and <br />                                                                                   Rv3788 expressing cells robust hyper-expression. (C) <br /> incubation continued 10 min followed by <br />                                                                                   Purified proteins: MtbGre (17.8 kDa), Rv3788 (17.4 kDa) and <br /> resolving 20% urea PAGE. <br />                                                                                   MsGre (18 kDa). The yield proteins ,5 mg <br />                                                                                   2 liters culture. The proteins .95% pure. <br /> MtbGre-RNAP interaction                                                           (TIF) <br />    C-terminal his-tagged MtbGre S127E mutant were <br /> cloned pET20b purified using Ni-NTA column. 5 mg                      Figure S3 Transcription assays T7A1 promoter <br /> both RNAP (Ec Mtb) Gre protein used analyzing                     templates using mycobacterial RNAPs. (A) A modified <br /> direct interactions. Proteins incubated 15 mins                 T7A1 promoter used generating stalled complexes. <br /> in 50 ml volume incubation buffer containing 50 mM tris - HCl                  Residues underlined ones replaced original <br /> (pH 8.0), 100 mM potassium glutamate, 5% glycerol, 20 mM                      residues showed above. Biotin tag present 59 end the <br /> imidazole room temperature. 20 ml Ni-NTA pre-equilibrated                   template. In presence ATP GTP, RNAP forms stalled <br /> with incubation buffer added protein mixture                  TEC 20mer RNA. (B) Multiple-round vitro transcription <br /> incubated additional 30 mins rotary mixer. The                        E. coli, M. smegmatis M. tuberculosis RNAP T7A1 <br /> supernatant separated pellet washed thrice                   promoter. Abortive transcripts indicated lower panel. <br /> 400 ml incubation buffer. Finally, pellet re-                      (TIF) <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            11                              July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                                                Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br /> Figure S4   Ms Mtb RNAP free endogenous                                           probed anti-Gre antibody estimate Gre protein <br /> Gre factor contamination. 20 mg Ms Mtb RNAP                                        level cells different growth phases. (C) Western blot analysis <br /> were probed anti-Gre antibody. 100 ng purified MtbGre                                  Gre cell lysate M. smegmatis exposed different stresses. <br /> was used control.                                                                         (TIF) <br /> (TIF) <br />                                                                                                Figure S7 Expression Gre pTrcgre construct in <br /> Figure S5 Activity MsGre phenotypic effects                                         E. coli TK1021. (A) The gel shows expression MtbGre from <br /> of gre overexpression. (A) Transcript cleavage stimulatory                                     pTrcgre construct, induced 0.3 mM IPTG. (B) Transcript <br /> activity MsGre. (B) SDS-PAGE analysis cell lysates                              cleavage assays using MtbGre E. coli GreA. Only MtbGre <br /> M. smegmatis pMV261, over-expression construct                                   shows cleavage mycobacterial RNAP elongation complex. <br /> (pMVgreOE) antisense (pMVgreAS) mediated knock-down                                        (TIF) <br /> construct. (C) Morphology M. smegmatis cells over-expressing <br /> MtbGre. Comparison cellular morphology (left panel)                                     Acknowledgments <br /> nucleoid (right panel) M. smegmatis mc2155 cells harboring either <br /> the pMV261 vector pMV-greAS pMV-greOE constructs. Left                                   We thank Nobuo Shimamoto, Kyoto Sangyo University, strains <br /> panels- bright-field images; right panels-fluorescent images                                   TK1000, TK1021 plasmid pMS002, O. Krishnadev MBU, IISc, <br />                                                                                                generating homology model MtbGreA, G. Swapna E. coli <br /> showing DAPI-stained nucleoid. <br />                                                                                                RNAP AstraZeneca, Bangalore, M. tuberculosis sA antibodies. <br /> (TIF)                                                                                          Sergei Borukhov, University Medicine Dentistry, New Jersey, <br /> Figure S6 Determination expression pattern                                           USA, acknowledged helpful suggestions. <br /> Gre cells. (A) Expression gre in <br /> response different stresses M. smegmatis M. tuberculosis                             Author Contributions <br /> determined western blot. (B) Estimation level Gre                                 Conceived designed experiments: AC VN. Performed the <br /> protein M. smegmatis cells. Purified M. smegmatis Gre protein                           experiments: AC SM. Analyzed data: AC SM VN. Contributed <br /> was used standard. 120 mg total cell lysate proteins                              reagents/materials/analysis tools: VN. Wrote paper: AC VN. <br />  <br /> References <br />  1. Roberts JW, Shankar S, Filter JJ (2008) RNA polymerase elongation factors.                 20. Rodrigue S, Provvedi R, Jacques PE, Gaudreau L, Manganelli R (2006) The <br />     Annu Rev Microbiol 62: 211&#226;&#8364;&#8220;233.                                                                sigma factors Mycobacterium tuberculosis. FEMS Microbiol Rev 30: 926&#226;&#8364;&#8220;941. <br />  2. Deutscher MP (2006) Degradation RNA bacteria: comparison mRNA                     21. Harshey RM, Ramakrishnan T (1977) Rate ribonucleic acid chain growth in <br />     stable RNA. Nucleic Acids Res 34: 659&#226;&#8364;&#8220;666.                                                 Mycobacterium tuberculosis H37Rv. J Bacteriol 129: 616&#226;&#8364;&#8220;622. <br />  3. Borukhov S, Lee J, Laptenko O (2005) Bacterial transcription elongation factors:           22. Garcia MJ, Nunez MC, Cox RA (2010) Measurement rates synthesis of <br />     new insights molecular mechanism action. Mol Microbiol 55: 1315&#226;&#8364;&#8220;1324.                  components ribosomes Mycobacterium fortuitum: theoretical <br />  4. Vassylyev DG (2009) Elongation RNA polymerase: race                               approach qRT-PCR experimentation. PLoS One 5: e11575. <br />     roadblocks. Curr Opin Struct Biolol 19: 691&#226;&#8364;&#8220;700.                                           23. Stebbins CE, Borukhov S, Orlova M, Polyakov A, Goldfarb A, et al. (1995) <br />  5. Haugen SP, Ross W, Gourse RL (2008) Advances bacterial promoter                             Crystal structure GreA transcript cleavage factor Escherichia coli. <br />     recognition control factors bind DNA. Nat Rev Microbiol                 Nature 373: 636&#226;&#8364;&#8220;640. <br />     6: 507&#226;&#8364;&#8220;519.                                                                                24. Koulich D, Orlova M, Malhotra A, Sali A, Darst SA, et al. (1997) Domain <br />  6. Borukhov S, Polyakov A, Nikiforov V, Goldfarb A (1992) GreA protein:                         organization Escherichia coli transcript cleavage factors GreA GreB. J Biol <br />     transcription elongation factor Escherichia coli. Proc Natl Acad Sci U S A 89:            Chem 272: 7201&#226;&#8364;&#8220;7210. <br />     8899&#226;&#8364;&#8220;8902.                                                                                 25. Koulich D, Nikiforov V, Borukhov S (1998) Distinct functions N C- <br />  7. Borukhov S, Sagitov V, Goldfarb A (1993) Transcript cleavage factors E.                   terminal domains GreA, Escherichia coli transcript cleavage factor. J Mol <br />     coli. Cell 72: 459&#226;&#8364;&#8220;466.                                                                        Biol 276: 379&#226;&#8364;&#8220;389. <br />  8. Reines D, Chamberlin MJ, Kane CM (1989) Transcription elongation factor SII                26. Loizos N, Darst SA (1999) Mapping interactions Escherichia coli GreB with <br />     (TFIIS) enables RNA polymerase II elongate block transcription                 RNA polymerase ternary elongation complexes. J Biol Chem 274: <br />     human gene vitro. J Biol Chem 264: 10799&#226;&#8364;&#8220;10809.                                        23378&#226;&#8364;&#8220;23386. <br />  9. Orlova M, Newlands J, Das A, Goldfarb A, Borukhov S (1995) Intrinsic                       27. Vassylyeva MN, Svetlov V, Dearborn AD, Klyuyev S, Artsimovitch I, et al. <br />     transcript cleavage activity RNA polymerase. Proc Natl Acad Sci U S A 92:                   (2007) The carboxy-terminal coiled-coil RNA polymerase beta9-subunit is <br />     4596&#226;&#8364;&#8220;4600.                                                                                     main binding site Gre factors. EMBO Rep 8: 1038&#226;&#8364;&#8220;1043. <br /> 10. Laptenko O, Lee J, Lomakin I, Borukhov S (2003) Transcript cleavage factors                28. Borukhov S, Laptenko O, Lee J (2001) Escherichia coli transcript cleavage factors <br />     GreA GreB act transient catalytic components RNA polymerase.                         GreA GreB: functions mechanisms action. Methods Enzymol 342: <br />     EMBO J 22: 6322&#226;&#8364;&#8220;6334.                                                                          64&#226;&#8364;&#8220;76. <br /> 11. Marr MT, Roberts JW (2000) Function Transcription Cleavage Factors GreA                 29. Wu S, Howard ST, Lakey DL, Kipnis A, Samten B, et al. (2004) The principal <br />     GreB Regulatory Pause Site. Mol Cell 6: 1275&#226;&#8364;&#8220;1285.                                    sigma factor sigA mediates enhanced growth Mycobacterium tuberculosis vivo. <br /> 12. Kireeva ML, Hancock B, Cremona GH, Walter W, Studitsky VM, et al. (2005)                       Mol Microbiol 51: 1551&#226;&#8364;&#8220;1562. <br />     Nature nucleosomal barrier RNA polymerase II. Mol Cell 18: 97&#226;&#8364;&#8220;108.               30. Sharbati S, Schramm K, Rempel S, Wang H, Andrich R, et al. (2009) <br /> 13. Komissarova N, Kashlev M (1997) RNA polymerase switches                                Characterisation porin genes Mycobacterium fortuitum impact on <br />     inactivated activated states By translocating forth                     growth. BMC Microbiol 9: 31. <br />     DNA RNA. J Biol Chem 272: 15329&#226;&#8364;&#8220;15338.                                             31. Sun J, Wang X, Lau A, Liao TY, Bucci C, et al. (2010) Mycobacterial nucleoside <br /> 14. Opalka N, Chlenov M, Chacon P, Rice WJ, Wriggers W, et al. (2003) Structure                    diphosphate kinase blocks phagosome maturation murine RAW 264.7 <br />     function transcription elongation factor GreB bound bacterial                    macrophages. PLoS One 5: e8769. <br />     RNA polymerase. Cell 114: 335&#226;&#8364;&#8220;345.                                                         32. Rutherford ST, Lemke JJ, Vrentas CE, Gaal T, Ross W, et al. (2007) Effects of <br /> 15. Hogan BP, Hartsch T, Erie DA (2002) Transcript cleavage Thermus                             DksA, GreA, GreB transcription initiation: insights mechanisms <br />     thermophilus RNA polymerase. Effects GreA anti-GreA factors. J Biol                     factors bind secondary channel RNA polymerase. J Mol Biol <br />     Chem 277: 967&#226;&#8364;&#8220;975.                                                                             366: 1243&#226;&#8364;&#8220;1257. <br /> 16. Laptenko O, Borukhov S (2003) Biochemical assays Gre factors Thermus                 33. Singh VK, Jayaswal RK, Wilkinson BJ (2001) Cell wall-active antibiotic induced <br />     thermophilus. Methods Enzymol 371: 219&#226;&#8364;&#8220;232.                                                    proteins Staphylococcus aureus identified using proteomic approach. FEMS <br /> 17. Laptenko O, Kim SS, Lee J, Starodubtseva M, Cava F, et al. (2006) pH-                          Microbiol Lett 199: 79&#226;&#8364;&#8220;84. <br />     dependent conformational switch activates inhibitor transcription                   34. Len ACL, Harty DWS, Jacques NA (2004) Stress-responsive proteins are <br />     elongation. EMBO J 25: 2131&#226;&#8364;&#8220;2141.                                                              upregulated Streptococcus mutans acid tolerance. Microbiology 150: <br /> 18. Cole ST, Brosch R, Parkhill J, Garnier T, Churcher C, et al. (1998) Deciphering                1339&#226;&#8364;&#8220;1351. <br />     biology Mycobacterium tuberculosis complete genome sequence.               35. China A, Tare P, Nagaraja V (2010) Comparison promoter-specific events <br />     Nature 393: 537&#226;&#8364;&#8220;544.                                                                           transcription initiation mycobacteria. Microbiology 156: 1942&#226;&#8364;&#8220;1952. <br /> 19. Smith I, Bishai WR, Nagaraja V (2005) Control mycobacterial transcription.              36. Hsu LM, Vo NV, Chamberlin MJ (1995) Escherichia coli transcript cleavage <br />     In: Cole ST, ed. Tuberculosis Tubercle Bacillus. Washington, DC: ASM                   factors GreA GreB stimulate promoter escape gene expression vivo <br />     Press. pp 219&#226;&#8364;&#8220;231.                                                                             vitro. Proc Natl Acad Sci U S A 92: 11588&#226;&#8364;&#8220;11592. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                        12                                     July 2011 | Volume 6 | Issue 7 | e21941 <br />                                                                                                                                   Transcript Cleavage Factor M. tuberculosis <br />  <br />  <br /> 37. Sen R, Nagai H, Shimamoto N (2001) Conformational switching Escherichia                        Establishment Efficient Nitrogen-Fixing Symbiosis Phaseolus <br />     coli RNA polymerase-promoter binary complex facilitated elongation factor                   vulgaris. Molecular Plant-Microbe Interactions 15: 225&#226;&#8364;&#8220;232. <br />     GreA GreB. Genes Cells 6: 389&#226;&#8364;&#8220;401.                                                      46.   Belotserkovskii BP, Liu R, Tornaletti S, Krasilnikova MM, Mirkin SM, et al. <br /> 38. Kulish D, Lee J, Lomakin I, Nowicka B, Das A, et al. (2000) The functional role                   (2010) Mechanisms implications transcription blockage guanine-rich <br />     basic patch, structural element Escherichia coli transcript cleavage factors              DNA sequences. Proc Natl Acad Sci U S A 107: 12816&#226;&#8364;&#8220;12821. <br />     GreA GreB. J Biol Chem 275: 12789&#226;&#8364;&#8220;12798.                                                47.   Snapper SB, Melton RE, Mustafa S, Kieser T, Jacobs WR, Jr. (1990) Isolation <br /> 39. Susa M, Kubori T, Shimamoto N (2006) A pathway branching transcription                         characterization efficient plasmid transformation mutants Mycobacterium <br />     initiation Escherichia coli. Mol Microbiol 59: 1807&#226;&#8364;&#8220;1817.                                      smegmatis. Mol Microbiol 4: 1911&#226;&#8364;&#8220;1919. <br /> 40. Lamour V, Hogan BP, Erie DA, Darst SA (2006) Crystal structure Thermus                   48.   Mukherjee R, Chatterji D (2008) Stationary phase induced alterations in <br />     aquaticus Gfh1, Gre-factor paralog inhibits stimulates transcript              mycobacterial RNA polymerase assembly: A cue phenotypic resistance <br />     cleavage. J Mol Biol 356: 179&#226;&#8364;&#8220;188.                                                                rifampicin. Biochem Biophys Res Commun 369: 899&#226;&#8364;&#8220;904. <br /> 41. Harshey RM, Ramakrishnan T (1977) Rate ribonucleic acid chain growth                  49.   China A, Nagaraja V (2010) Purification RNA polymerase mycobacteria <br />     Mycobacterium tuberculosis H37Rv. J Bacteriol 129: 616&#226;&#8364;&#8220;622.                                       optimized promoter-polymerase interactions. Protein Expr Purif 69: <br />                                                                                                       235&#226;&#8364;&#8220;242. <br /> 42. Garcia MJ, Nunez MC, Cox RA (2010) Measurement rates synthesis of <br />                                                                                                 50.   Zheng H, Lu L, Wang B, Pu S, Zhang X, et al. (2008) Gene <br /> </body></html>