<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Research article <br />  <br />               Phenothiazines induce PP2A-mediated <br />                     apoptosis T cell acute <br />                      lymphoblastic leukemia <br />      Alejandro Gutierrez,1,2 Li Pan,3 Richard W.J. Groen,4,5 Frederic Baleydier,3,6 Alex Kentsis,1,2 <br />     Jason Marineau,4 Ruta Grebliunaite,1 Elena Kozakewich,1 Casie Reed,1 Francoise Pflumio,7,8 <br />   Sandrine Poglio,7,8 Benjamin Uzan,7,8 Paul Clemons,9 Lynn VerPlank,9 Frank An,9 Jason Burbank,9 <br />         Stephanie Norton,9 Nicola Tolliday,9 Hanno Steen,10 Andrew P. Weng,11 Huipin Yuan,12 <br />           James E. Bradner,4 Constantine Mitsiades,4 A. Thomas Look,1,2 Jon C. Aster3 <br />                   1Department Pediatric Oncology, Dana-Farber Cancer Institute Harvard Medical School, Boston, Massachusetts, USA. <br />        2Division <br />                Hematology/Oncology, Boston Children&#226;&#8364;&#8482;s Hospital Harvard Medical School, Boston, Massachusetts, USA. 3Department Pathology, <br />                  Brigham Women&#226;&#8364;&#8482;s Hospital Harvard Medical School, Boston, Massachusetts, USA. 4Department Medical Oncology, <br />    Dana-Farber Cancer Institute Harvard Medical School, Boston, Massachusetts, USA. 5Department Cell Biology, University Medical Center Utrecht, <br />                             Utrecht, Netherlands. 6Department Pediatric Hematology, Hospices Civils Lyon, Lyon, France. <br />            7Laboratoire Recherche sur les Cellules Souches H&#195;&#169;matopo&#195;&#175;&#195;&#169;tiques et Leuc&#195;&#169;miques, Institut Radiobiologie Cellulaire et Mol&#195;&#169;culaire, <br />  <br />   Direction des Sciences du Vivant, Commissariat &#195;&#160; l&#226;&#8364;&#8482;Energie Atomique et aux Energies Alternatives, Institut National la Sant&#195;&#169; et la Recherche M&#195;&#169;dicale, <br />              Fontenay-aux-Roses Universit&#195;&#169; Paris-Sud, Fontenay-aux-Roses, France. 8Universit&#195;&#169; Paris-Diderot, Paris, France. 9Broad Institute, <br />          Cambridge, Massachusetts, USA. 10Proteomics Center Department Pathology, Boston Children&#226;&#8364;&#8482;s Hospital Harvard Medical School, <br />           Boston, Massachusetts, USA. 11British Columbia Cancer Agency, Vancouver, Canada. 12Xpand Biotechnology BV, Bilthoven, Netherlands. <br />  <br />  <br />  <br />     T cell acute lymphoblastic leukemia (T-ALL) aggressive cancer frequently associated activating <br />     mutations NOTCH1 dysregulation MYC. Here, performed 2 complementary screens identify <br />     FDA-approved drugs drug-like small molecules activity T-ALL. We developed zebrafish <br />     screen small molecules toxic activity MYC-overexpressing thymocytes used human <br />     T-ALL cell line screen small molecules synergize Notch inhibitors. We identified antipsy- <br />     chotic drug perphenazine screens ability induce apoptosis fish, mouse, human <br />     T-ALL cells. Using ligand-affinity chromatography coupled mass spectrometry, identified protein <br />     phosphatase 2A (PP2A) perphenazine target. T-ALL cell lines treated perphenazine exhibited rapid <br />     dephosphorylation multiple PP2A substrates subsequent apoptosis. Moreover, shRNA knockdown <br />     specific PP2A subunits attenuated perphenazine activity, indicating PP2A mediates drug&#226;&#8364;&#8482;s anti- <br />     leukemic activity. Finally, human T-ALLs treated perphenazine exhibited suppressed cell growth and <br />     dephosphorylation PP2A targets vitro vivo. Our findings provide mechanistic explanation for <br />     recurring identification phenothiazines class drugs anticancer effects. Furthermore, these <br />     data suggest pharmacologic PP2A activation T-ALL cancers driven hyperphosphorylated <br />     PP2A substrates therapeutic potential. <br />  <br /> Introduction                                                                            mutations NOTCH1 occur T-ALL variously stimulate <br /> T cell acute lymphoblastic leukemia (T-ALL) aggressive can-                       NOTCH1 proteolysis generation ICN1 decrease ICN1 <br /> cer affecting mainly adolescents young adults. Intensified                          turnover, enhancing expression ICN1 target genes. One <br /> treatment regimens improved outcomes, patients                             important direct targets ICN1 context of <br /> fail conventional therapy dismal prognosis, T-ALL                            T-ALL cells MYC (6&#226;&#8364;&#8220;8); indeed, enforced expression MYC can <br /> remains fatal 20% children 50% adults                            rescue human T-ALL cell lines NOTCH1 inhibition (6, 9) <br /> (1&#226;&#8364;&#8220;3). New therapies desperately needed patients                       MYC transgenes drive T-ALL development mouse (10) <br /> poor-prognosis groups.                                                                  zebrafish (11, 12) models. However, NOTCH1 activation and <br />   One common genetic aberrations T-ALL acti-                          MYC overexpression sufficient T-ALL development, indi- <br /> vating mutations NOTCH1 (4), transmembrane receptor                                cating establishment maintenance T-ALL depends on <br /> that converted transcriptional activator series                    additional cooperative genetic epigenetic events dysregulate <br /> proteolytic cleavage events, carried                    signaling pathways. Indeed, activating NOTCH1 mutations <br /> intramembranous protease &#206;&#179;-secretase (5). &#206;&#179;-Secretase cleavage                          diverse collection recurrent muta- <br /> allows intracellular domain NOTCH1 (ICN1) translocate                         tions human T-ALL (13). Among frequent muta- <br /> <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> nucleus form transcriptional activation complex. The                       tions activate PI3K/AKT signaling pathway (14), has <br />                                                                                         complex interrelationship NOTCH signaling T-ALL cells <br /> Authorship note: Alejandro Gutierrez, Li Pan, Richard W.J. Groen, Frederic Baleydier,   (15) promising therapeutic target disease (16). <br /> and Alex Kentsis contributed equally work.                                        Repurposing FDA-approved drugs attractive approach <br /> Conflict interest: The authors declared conflict exists.    drug discovery, principle enable rapid translation to <br /> Citation article: J Clin Invest. 2014;124(2):644&#226;&#8364;&#8220;655. doi:10.1172/JCI65093.    clinic. Using 2 complementary screens, identified phenothi- <br />  <br /> 644	                                 The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014 <br />                                                                                                                                       research article <br />  <br />  <br />  <br />  <br /> Figure 1 <br /> Zebrafish screen small molecules toxic MYC-overexpressing thymocytes. (A) Primary screen design. (B) Hits primary <br /> screen. Arrow denotes result obtained PPZ (C). Representative images DMSO (control) PPZ-treated zebrafish larvae. (D) Dose- <br /> response curve secondary screen PPZ, 6 zebrafish larvae treated concentration. Drug doses higher 10 &#206;&#188;M induced general <br /> toxicity (not shown). Error bars = SD. <br />  <br />  <br /> azines class drugs NOTCH-independent anti&#226;&#8364;&#8220;T-ALL                     logically gene expression (11, 12, 23), wanted to <br /> activity. Phenothiazines used 50 years neu-                exploit model vivo drug discovery. We developed <br /> roleptic-type antipsychotic medications. The antipsychotic effects               fluorescence-based screen designed identify small <br /> of phenothiazines correlate ability block dopamine                 molecules selectively cytotoxic MYC-overexpressing <br /> receptors, broad array activities described,            thymocytes. Heterozygous zebrafish carrying tandem rag2:MYC- <br /> including antitumor effects. The basis antiproliferative                 ER;mitfa transgene (created cointegration rag2:MYC-ER and <br /> activities phenothiazines uncertain variously               wild-type mitfa transgenes genomic locus) mated <br /> attributed number mechanisms, including inhibition                    rag2:dsRed2 homozygous animals, generating rag2:MYC-ER&#226;&#8364;&#8220;pos- <br /> PKC (17), calmodulin (18), PI3K/AKT signaling (18&#226;&#8364;&#8220;21), can-                  itive &#226;&#8364;&#8220;negative embryos express rag2:dsRed2. All crosses <br /> cer stem cell activity (22).                                                     performed nacre pigment&#226;&#8364;&#8220;mutant background which <br />   Using quantitative mass spectrometry analyze drug-protein                   melanophores absent homozygous mitfa-inactivating <br /> binding proteome-wide, identified tumor suppressive ser-                  mutations (24). The rescue pigmentation defect fish car- <br /> ine/threonine phosphatase protein phosphatase 2A (PP2A)                     rying rag2:MYC-ER;mitfa transgene allowed distinguish <br /> new phenothiazine target (Alex Kentsis James E. Bradner, per-                MYC-ER&#226;&#8364;&#8220;positive &#226;&#8364;&#8220;negative zebrafish 3 days post-fertiliza- <br /> sonal correspondence). Phenothiazines stimulate rapid dephos-                    tion (dpf) (Figure 1A). At 3 dpf, zebrafish larvae arrayed into <br /> phorylation multiple PP2A targets implicated tumor cell                    96-well plates (3 larvae/well) incubated 12.5 &#206;&#188;M drug and <br /> growth <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span> cells, antitumor effects T-ALL                4-hydroxytamoxifen activate MYC-ER fusion protein. Four <br /> cells vitro vivo. Our findings provide likely explanation            days later, microscopy performed assay thymic fluorescence, <br /> for recurrent discovery phenothiazines screens com-                scored 3 (normal), 2 (intermediate), 1 (weak), 0 <br /> pounds anticancer effects, point new rational drug combi-                (no fluorescence) (Supplemental Figure 1; supplemental material <br /> nations consideration treatment NOTCH-driven cancers                   available online article; doi:10.1172/JCI65093DS1). Any <br /> such T-ALL, provide additional impetus development                    compound induced complete loss fluorescence 1 fish or <br /> and testing PP2A activators wide variety cancers.                     weak fluorescence 2 3 larvae well, considered hit <br />                                                                                  primary screen. Compounds induced death of <br /> Results                                                                          larvae generalized developmental abnormalities excluded <br /> Zebrafish screen small molecules activity MYC-overexpress-      analysis. Four libraries bioactive small molecules <br /> ing thymocytes. We previously developed zebrafish model MYC-                comprising 4,880 FDA-approved drugs, drug-like small molecules, <br /> induced T-ALL closely resembles human disease morpho-                   natural products screened fashion. <br />  <br /> 	                              The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014	                645 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Results screen small molecule inhibitors synergize &#206;&#179;-secretase inhibitor compound E suppress growth human <br /> T-ALL cell line KOPT-K1. KOPT-K1 cells incubated 384-well format small molecules Broad Institute bioactives collection <br /> in presence (A) absence (B) GSI (compound E, 0.1 &#206;&#188;M) duplicate. The effects drugs cell growth assessed 2 days <br /> by incubating CellTiter-Blue (Promega)for additional 4 hours, replicate compound measurement converted Z score <br /> relative corresponding DMSO-control distribution, described ChemBank (70). (C) ChemBank composite Z scores computed <br /> to combine replicates condition (+GSI &#226;&#8364;&#8220;GSI), allowing assays considered single plot; here, sought compounds <br /> that synergistic GSI, i.e., score negative composite Z presence GSI effect absence GSI. To <br /> express preferences quantitatively, devised 2-assay score (D) scales +GSI composite Z extent points in <br /> the negative direction extent exceeds &#226;&#8364;&#8220;GSI composite Z (see Methods); 330 compounds tested met initial criterion <br /> (scale factor &gt; 0.5), 43 compounds, including PPZ (inset), statistically distinguishable DMSO-control distribution. See <br /> Methods Supplemental Figure 4 additional details; screen data available ChemBank (see Methods). <br />  <br />  <br />  <br />   Although did identify compounds selectively                 tion) confirmed PPZ active MYC-overexpressing <br /> toxic MYC-overexpressing thymocytes, multiple compounds                     thymocytes dose-dependent manner (Figure 1D). <br /> with activity MYC-positive -negative thymocytes                 Human T-ALL screen small molecules synergize &#206;&#179;-secretase <br /> were identified. Among hits numerous glucocorti-                    inhibitors. In parallel, conducted cell-based screen the <br /> coids, known inducers thymocyte apoptosis,                Broad Institute bioactives chemical library collection (3,194 com- <br /> well conventional chemotherapeutics induce DNA dam-                    pounds) identify drugs synergistically suppress growth <br /> age (alkylating agents, topoisomerase inhibitors) (Figure 1B               T-ALL cells combination gamma-secretase inhibitors <br /> Supplemental Table 1). Many compounds mainstays                   (GSI). Roughly two-thirds small molecules collection <br /> of contemporary T-ALL therapy, validating utility              (n = 2108) overlap compounds screened zebrafish. The <br /> approach. A number novel compounds identified,                    screen conducted KOPT-K1 cells, NOTCH1-dependent <br /> including perphenazine (PPZ), FDA-approved phenothiazine                    T-ALL cell line overexpresses transcription factor TAL1, a <br /> antipsychotic (Figure 1, B C, Supplemental Table 1). Sec-              feature associated relatively poor prognosis disease. <br /> ondary screening zebrafish larvae (n = 6 larvae concentra-              We used combination GSI dexamethasone, known <br />  <br /> 646	                         The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014 <br />                                                                                                                                      research article <br />  <br />                                                                                 Figure 3 <br />                                                                                 PPZ subset related phenothiazines active estab- <br />                                                                                 lished zebrafish T-ALL. (A) Representative images zebrafish with <br />                                                                                 established MYC-induced T-ALL treated DMSO PPZ (1 &#206;&#188;M). <br />                                                                                 (B) Rate T-ALL regression zebrafish established MYC- <br />                                                                                 induced T-ALL, treated 1 &#206;&#188;M indicated compound. A mini- <br />                                                                                 mum 8 zebrafish analyzed condition. <br />  <br />  <br />                                                                                 fore elected study effects T-ALL cells greater detail. We <br />                                                                                 tested PPZ zebrafish established tumors found <br />                                                                                 PPZ active established MYC-induced T-ALL aris- <br />                                                                                 ing nonconditional rag2:EGFP-MYC transgenic zebrafish line <br />                                                                                 (Figure 3, A B), indicating activity independent of <br />                                                                                 4-hydroxytamoxifen conditional MYC-ER fusion transgene <br />                                                                                 used zebrafish-based screen. Activating Notch mutations <br />                                                                                 absent tumors arising model (26); thus, drugs that <br />                                                                                 score model Notch-independent mecha- <br />                                                                                 nisms. We tested activity series structurally related <br />                                                                                 phenothiazines 2 phenothiazines, prometha- <br />                                                                                 zine chlorpromazine, antileukemic activity zebrafish <br />                                                                                 1 &#206;&#188;M concentration, chlorprothixene pimethixene <br />                                                                                 lacked activity established T-ALL model (Figure 3B). <br />                                                                                   We extended studies PPZ phenothiazines to <br />                                                                                 mammalian T-ALL cell lines. PPZ single agent suppressed the <br />                                                                                 growth KOPT-K1 cells, cell line used human T-ALL <br />                                                                                 screen (Figure 4A), combination PPZ GSI caused <br />                                                                                 moderately synergistic growth suppression KOPT-K1 cells and <br /> synergistic combination (25), positive control. We initially               second human NOTCH1-dependent T-ALL cell line, DND41 <br /> performed isobologram analysis select concentrations GSI                  (Figure 4B), judged isobologram analysis (Supplemental <br /> (compound E, 100 nM) dexamethasone (200 nM)                       Table 2). Similar synergistic interactions PPZ GSI were <br /> a synergistic growth inhibitory effect KOPT-K1 cells (Supple-                observed notch1-dependent murine T-ALL cell lines 142 <br /> mental Figure 2) used concentrations drugs                    (Figure 4C Supplemental Table 2) 144 (Supplemental Table <br /> in screen optimization pilot studies. Under final conditions                2), harbor transgene driving expression constitu- <br /> selected, GSI versus GSI plus dexamethasone produced Z score                  tively active kras acquired activating mutations notch1 <br /> of 0.74 (Supplemental Figure 3); point reference, screens                (27). PPZ suppressed growth SUPT13 (Figure 4D), <br /> with Z scores greater 0.5 taken having high probabil-             GSI-insensitive human T-ALL cell line lacks NOTCH1 <br /> ity detecting true positives.                                                mutations, PF382, GSI-insensitive human T-ALL cell line <br />   We screened Broad Institute bioactives library KOPT-K1            expresses constitutively active mutated form NOTCH1 <br /> cells according strategy shown Supplemental Figure 4. In              (Supplemental Table 2). In GSI-insensitive cell lines, little <br /> brief, GSI/bioactive-compound interaction assessed inde-                 interaction PPZ GSI observed terms of <br /> pendently scoring effects GSI plus bioactive compound               cell numbers (Figure 4D) isobologram analysis (Supplemental <br /> (Figure 2A) effects bioactive compound                    Table 2), suggesting (as zebrafish T-ALL) phenothiazines <br /> (Figure 2B). To identify compounds synergize GSI,                  antileukemic effects mammalian T-ALL cell lines are <br /> looked data using 2 different approaches. First, com-                 Notch-independent. <br /> bined replicate data sets plotted composite Z scores (a               The mechanism T-ALL growth suppression PPZ inves- <br /> measure difference control state) presence               tigated human cell lines zebrafish (Figure 5). PPZ treat- <br /> absence GSI (Figure 2C). We scaled composite Z score                ment human murine T-ALL cell lines induced apoptosis, as <br /> in presence GSI fashion assessed screened mol-            assessed appearance sub-G1 DNA fraction (Figure 5, <br /> ecule extent suppressed T-ALL cell growth              A B), increased caspase activity (Figure 5, C D), and <br /> in presence GSI (the +GSI composite Z score) extent              increased annexin V staining (Supplemental Figure 5A), effects <br /> to suppression exceeded observed absence              enhanced cotreatment GSI (Figure 5, A, B, D, <br /> GSI (the so-called 2-assay score; Figure 2D). Using metrics,          Supplemental Figure 5B). We tested activity PPZ in <br /> reproducible set synergistic &#226;&#8364;&#339;hits&#226;&#8364;? totaled 43 3,194 compounds            established T-ALL tumors induced rag2:EGFP-MYC;rag2:EGFP- <br /> screened, 1.3% compounds (for structure identity             BCL2 double-transgenic zebrafish (Figure 5E), that <br /> compounds, ChemBank [http://chembank.broadinstitute.org/                    BCL2 overexpression completely blocked activity PPZ all <br /> chemistry/search/execute.htm?id=5685551]). Notably, PPZ                    zebrafish tested (n = 9), confirming PPZ acts induction of <br /> scored hits KOPT-K1 cell line screen.                  mitochondrial apoptosis. <br />   Verification antileukemic activity PPZ. Taken together, 2         PPZ binds activates PP2A tumor suppressor. Phenothiazines are <br /> screens suggested PPZ targets conserved regulator benign              best known inhibitors dopaminergic signaling. Multiple dopa- <br /> and malignant T cell progenitor growth viability, there-              receptors expressed T lineage cells proposed <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014	                  647 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 4 <br /> Activity PPZ related phenothiazines mammalian T-ALL cell lines. (A) Growth suppression KOPT-K1 cells indicated phe- <br /> nothiazines, expressed IC50. (B&#226;&#8364;&#8220;D) Effects compound E (GSI) PPZ growth Notch1-dependent human KOPT-K1 DND41 <br /> T-ALL cells (B), Notch1-dependent murine 142 T-ALL cells (C), NOTCH1-independent human SUP-T13 T-ALL cells (D). In experiment, <br /> each condition assayed triplicate, experiments repeated twice. In B&#226;&#8364;&#8220;D, PPZ compound E used 10 &#206;&#188;M <br /> and 0.1 &#206;&#188;M, respectively, (C) PPZ compound E used 3 &#206;&#188;M 6 nM, respectively. <br />  <br />  <br />  <br /> to regulate immune function (28). However, structurally unre-             protein targets KOPT-K1 T-ALL cell lysates labeled <br /> lated dopamine inhibitors haloperidol, domperidone, clozapine             stable isotopes (SILAC) (31). These protein pull-downs were <br /> were toxic Myc-overexpressing thymocytes, effect             presence absence competitor phenothiazines <br /> T-ALL progression zebrafish, did affect human KOPT-K1              varying IC50 KOPT-K1 cell growth inhibition, dis- <br /> T-ALL cell growth (data shown). Conversely, promethazine,               placement individual binding proteins fluorous PPZ <br /> phenothiazine does inhibit dopaminergic signaling (29),              matrix assessed quantitative mass spectrometry. We then <br /> decreased growth zebrafish T-ALLs human KOPT-K1                    rank ordered binding proteins based correlation between <br /> T-ALL cells (Figure 3B Figure 4A). Taken together, data             displacement drug drug&#226;&#8364;&#8482;s IC50 KOPT-K1 cells. This <br /> strongly suggest inhibition dopaminergic signaling unre-           approach identified PPP2R1A protein binding <br /> lated antileukemic activity compounds.                        affinity correlated closely biologic activities the <br />   Another reported activity phenothiazines abrogate             phenothiazines tested (Figure 6, A B). PPP2R1A A&#206;&#177; <br /> mitogenic signaling inhibition calmodulin (30),               scaffolding subunit protein phosphatase PP2A, sub- <br /> broad roles calcium-dependent intracellular signaling. However,            sequent work confirmed PPZ directly binds purified PP2A at <br /> we noted W13, relatively selective calmodulin inhibitor,            low micromolar concentrations (Alex Kentsis James E. Brad- <br /> highly toxic zebrafish larvae low nanomolar concentrations              ner, personal communication). <br /> (data shown), presumably calmodulin inhibition                   PP2A conserved heterotrimeric phosphatase many <br /> not compatible survival. Thus, difficult exclude          different isoforms, comprising scaffolding A subunit, vari- <br /> some contribution calmodulin inhibition observed                    able regulatory B subunit, catalytic C subunit (32, 33). Best <br /> effects phenothiazines, calmodulin inhibition explain               known tumor suppressor targeted SV40 small t anti- <br /> the activity phenothiazines zebrafish screen.                       gen, PP2A target oncogenic microRNA miR-19 (34), <br />   We left challenge identifying                collaborates activated Notch1 promote T-ALL devel- <br /> molecular target responsible activity phenothi-           opment. The tumor-suppressive activity PP2A believed be <br /> azines screens. To address issue, devised quantita-          mediated ability dephosphorylate diverse substrates <br /> tive mass spectrometry approach, termed activity correlation pro-             implicated oncogenic signaling, including AKT (35), p70S6K <br /> teomics, offers improve identification biologically            (36), MYC (37&#226;&#8364;&#8220;39), ERK (40, 41), BAD (42, 43). Thus, the <br /> relevant targets pleiotropic drugs. Briefly, used fluorous-             antileukemic effects PPZ phenothiazines T-ALL <br /> tagged PPZ fluorous ligand-affinity chromatography pull-               mediated PP2A, drugs stimulate <br />  <br /> 648	                        The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014 <br />                                                                                                                                      research article <br />  <br />                                                                                                                          Figure 5 <br />                                                                                                                          PPZ induces T-ALL cell apoptosis. <br />                                                                                                                          (A B) DNA histograms human <br />                                                                                                                          KOPT-K1 mouse 142 T-ALL <br />                                                                                                                          cells, respectively, treated with <br />                                                                                                                          compound E (GSI) PPZ alone <br />                                                                                                                          combination 96 hours. <br />                                                                                                                          (C D) PPZ increases caspase <br />                                                                                                                          3/7 activity KOPT-K1 cells alone <br />                                                                                                                          combination GSI (com- <br />                                                                                                                          pound E), used indicated <br />                                                                                                                          concentrations 48 hours. (E) A <br />                                                                                                                          representative rag2:EGFP-MYC; <br />                                                                                                                          rag2:EGFP-BCL2 double-trans- <br />                                                                                                                          genic zebrafish treated 1 &#206;&#188;M <br />                                                                                                                          PPZ shown. PPZ activity <br />                                                                                                                          bcl2-transgenic zebraf- <br />                                                                                                                          ish tested (n = 8). <br />  <br />  <br />  <br />  <br /> PP2A activity. As predicted, incubation KOPT-K1 cells PPZ               expanded passage NSG mice sensitivity PPZ in <br /> led rapid dephosphorylation PP2A substrates AKT, ERK,                 newly described serum-free culture T-ALL cells that <br /> p70S6K, MYC, BAD (Figure 6, C&#226;&#8364;&#8220;E), effects abrogated               relies completely defined medium feeder cells expressing <br /> by addition phosphatase inhibitor okadaic acid (Figure 6D),              Notch ligand DLL-1 (46). We noted low micromolar levels <br /> an inhibitor PP2A. Total AKT, ERK, p70S6K, BAD protein                   PPZ resulted dephosphorylation multiple PP2A targets <br /> levels unaffected PPZ, levels MYC protein fell               (AKT, p70S6K, BAD) impaired growth 4 T-ALLs test- <br /> (Figure 6E), line prior reports showing PP2A-mediated              ed conditions (Figure 8). Variable sensitivity GSI and <br /> dephosphorylation MYC residue S62 enhances MYC degrada-                      variable interaction GSI PPZ noted (Figure 8). <br /> tion (37). In addition, noted effects PPZ PP2A                We tested independent series primary T-ALLs sec- <br /> substrates growth T-ALL cell lines KOPT-K1                  ond culture relies high levels human serum and <br /> phenocopied FTY720 (Fingolimod) (Supplemental Figure 6, A                    observed similar results, lines growth inhibited low <br /> and B), FDA-approved immunomodulatory drug previously                        micromolar concentrations PPZ (Supplemental Figure 7). <br /> shown activate PP2A low micromolar concentrations (44, 45).                 One primary human T-ALL lines, hTALL2, grew suffi- <br /> In contrast, calmodulin inhibitor W13 effect PP2A                 ciently serum-free culture permit luciferiza- <br /> targets p70S6K (Supplemental Figure 6C).                                tion retrovirus, facilitated drug testing vivo. This <br />   To determine PP2A required activity PPZ,             carried mice bearing calcium phosphate scaffolds <br /> first used shRNA knock PP2A A&#206;&#177; scaffolding subunit.                 seeded human marrow&#226;&#8364;&#8220;derived mesenchymal stem cells, <br /> PP2A A&#206;&#177; knockdown impaired PPZ-induced dephosphoryla-                       approach allows drug testing &#226;&#8364;&#339;humanized&#226;&#8364;? <br /> tion PP2A targets (Figure 7A) desensitized KOPT-K1                  microenvironment. Treatment mice PPZ (10 mg/kg <br /> cells growth-suppressive effects PPZ (Figure 8B).                     intraperitoneally daily) caused significant decrease growth of <br /> shRNA knock-down catalytic PP2A C&#206;&#177; subunit similar                     hTALL2 cells scaffolds (Figure 9A) murine spleen <br /> effects (Figure 7, C D). Taken together, data strongly                (Figure 9B). Importantly, single dose PPZ (10 mg/kg) led <br /> suggest growth-suppressive effects PPZ T-ALL cells               dephosphorylation PP2A targets AKT, p70S6K, and <br /> mediated stimulation PP2A activity.                                  BAD hTALL2 blasts vivo (Figure 9C). These results that <br />   PPZ activates PP2A antileukemic effects primary human              PPZ produce antileukemic effects stimulate PP2A activity <br /> T-ALL cells vitro vivo. With results hand, remained       primary T-ALL lymphoblasts doses tolerated mice. <br /> to determined PPZ inhibit growth primary <br /> T-ALL cells vitro vivo, so, antileukemic               Discussion <br /> effects associated dephosphorylation PP2A targets.                 NOTCH1 compelling therapeutic target T-ALL, the <br /> To address issue vitro, tested series primary human             activity GSIs single agents T-ALL preclinical models (47) <br /> T-ALLs (summarized Supplemental Table 3)                       clinical trials (48) modest, suggesting need to <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014	                         649 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 6 <br /> PPZ binds PP2A leads rapid dephosphorylation PP2A targets KOPT-K1 T-ALL cells. (A) Activity correlation proteomics revealed <br /> that biologic activity phenothiazines correlates closely binding affinity PPP2R1A, scaffolding A&#206;&#177; subunit PP2A <br /> phosphatase, described (Alex Kentsis James E. Bradner, personal correspondence). (B) Structures phenothiazines tested. (C) <br /> Effects PPZ phosphorylation AKT, ERK, p70S6K, BAD. Lysates prepared 15 minutes treatment. (D) The PP2A inhibitor <br /> okadaic acid (OA) antagonizes PPZ-mediated dephosphorylation PP2A substrates. Lysates prepared 15 minutes PPZ treatment. <br /> (E) Effects PPZ MYC phosphorylation protein levels. Lysates prepared 30 minutes 6 hours PPZ treatment. In D, cells <br /> were pretreated okadaic acid 30 minutes prior addition PPZ. <br />  <br />  <br />  <br /> combine Notch pathway inhibitors compounds                     GSI combination TOR inhibitors (9, 47), PI3K inhibi- <br /> activity disease. We used complementary screens discov-             tors (58), MAPK inhibitors (58) increased activity over <br /> er small molecules Notch-independent anti&#226;&#8364;&#8220;T-ALL activity.                 GSI T-ALL. PP2A blunts signaling multiple <br /> Most small molecules scoring screens glucocorti-                  &#226;&#8364;&#339;nodes&#226;&#8364;? downstream surface receptors RAS dephos- <br /> coids, providing support use GSIs glucocorti-               phorylation signaling intermediates ERK (40, 41), <br /> coids combination relapsed/refractory T-ALL (25). A previ-               AKT (35), p70S6K (36) inhibit growth down- <br /> ously unidentified hit emerging screens PPZ,                 regulating MYC (37&#226;&#8364;&#8220;39) promote apoptosis dephosphory- <br /> which shown suppress growth piscine, murine,                 lating BAD (42, 43). In support PPZ acting PP2A activa- <br /> human T-ALLs induction apoptosis. These effects                  tor, data PPZ treatment affects phosphorylation <br /> PPZ linked biochemical, cell biological, genetic studies           of PP2A substrates, shRNA knockdown of <br /> to activation PP2A, tumor-suppressive protein phosphatase.                PP2A catalytic scaffolding subunits impaired antileukemic <br /> The ability PPZ activate PP2A represents likely                 activity PPZ. Withdrawal Notch signaling treatment with <br /> mechanism previously unexplained ability phenothi-                  GSI inhibitors interferes growth T-ALL cells <br /> azines downregulate PI3K/AKT signaling explain                  number mechanisms, including downregulation of <br /> recurrent identification compounds unbiased screens                MYC (6, 8, 59), IL-7R (55, 60), IGF-R1 (57), suppression <br /> for agents antineoplastic activity (18&#226;&#8364;&#8220;21, 49&#226;&#8364;&#8220;53).                        PI3K/AKT signaling (61). The partially overlapping effects of <br />   Given signaling mutated KRAS (54) receptors                PP2A stimulation Notch withdrawal key mitogenic signal- <br /> as IL-7R (55, 56) IGF-R1 (57) synergize Notch signaling            ing pathways predict drug combinations hit both <br /> induce maintain T-ALL growth, combinations agents                  targets additive growth-suppressive effects on <br /> antagonize downstream effectors pathways attractive               Notch-dependent T-ALL cells, consistent findings here <br /> therapeutic options disease. Indeed, prior work shown              GSI phenothiazines. <br />  <br /> 650	                         The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014 <br />                                                                                                                                      research article <br />  <br />                                                                                                                        Figure 7 <br />                                                                                                                        PP2A A&#206;&#177; (structural) C&#206;&#177; (cata- <br />                                                                                                                        lytic) subunits required the <br />                                                                                                                        growth-suppressive activity PPZ in <br />                                                                                                                        T-ALL cells. (A) Western blot analysis <br />                                                                                                                        KOPT-K1 T-ALL cells infected with <br />                                                                                                                        shRNA specific GFP (control) or <br />                                                                                                                        PP2A A&#206;&#177; treated doses <br />                                                                                                                        PPZ shown. (B) Effect same <br />                                                                                                                        shRNAs KOPT-K1 T-ALL cell <br />                                                                                                                        viability day 5 treatment with <br />                                                                                                                        control 5 &#206;&#188;M PPZ. (C) Western <br />                                                                                                                        blot analysis KOPT-K1 T-ALL cells <br />                                                                                                                        infected shRNA specific GFP <br />                                                                                                                        (control) PP2A C&#206;&#177; treated <br />                                                                                                                        doses PPZ shown. (D) <br />                                                                                                                        Effect shRNAs KOPT- <br />                                                                                                                        K1 cell viability treatment with <br />                                                                                                                        control (DMSO) 7.5 &#206;&#188;M PPZ. In A <br />                                                                                                                        C, cells treated PPZ <br />                                                                                                                        30 minutes. <br />  <br />  <br />  <br />  <br />    Our data suggest combination GSI PP2A activator               Zebrafish small molecule screen. At 3 dpf, transgenic zebrafish larvae <br /> is attractive strategy relapsed/refractory T-ALL,          expressing rag2:Myc-ER rag2:dsRed2 transgenes (12) arrayed into <br /> also merit consideration Notch-associated tumors,              96-well plates (3 larvae well) exposed individual small molecules <br /> mantle cell lymphoma (62), chronic lymphocytic leukemia (63&#226;&#8364;&#8220;65),                doses 12.5 &#206;&#188;M small molecules 1.25 &#206;&#188;M bioactive lipids, <br /> and breast cancer (66), failed conventional therapies. A limi-        50 &#206;&#188;g/l (129 nM) 4-hydroxytamoxifen activate Myc-ER <br /> tation PPZ phenothiazines marked extra-             transgene, antibiotics reduce bacterial overgrowth (25 units/ml <br /> pyramidal effects, limitation considering            penicillin, 25 mg/ml streptomycin, 5 mg/ml meropenem). Small <br /> drugs therapeutic agents cancer, activities antipsy-          molecules following small molecule libraries: Lopac 1280 <br /> chotic drugs low nanomolar concentrations, substantially               (n = 1280 compounds; Sigma-Aldrich); ICCB known bioactives (n = 480 <br /> the levels needed activate PP2A. It possible allevi-      compounds; Enzo Life Sciences); Prestwick Chemical Library (n = 1200 <br /> ate drawbacks engineering phenothiazine derivatives           compounds; Prestwick Chemical); Spectrum Collection (n = 2000 <br /> more potent PP2A activators lack activity dopamine recep-            compounds; MicroSource Discovery Systems). After 4 days treatment, <br /> tors; promethazine example phenothiazine antileuke-             thymic fluorescence individual larvae assessed fluorescence <br /> mia activity does avidly bind dopamine receptors, suggesting           microscopy. Small molecules induced general toxicity zebrafish <br /> that 2 activities separated. An alternative approach         larvae discarded. Eight wells 96-well plate dedicated to <br /> use PP2A activators favorable toxicity profiles            positive (dexamethasone 12.5 &#206;&#188;M) negative (DMSO) controls. <br /> are FDA approved, FTY720 (44). FTY720 potent                  KOPT-K1 cell line screen. The screening method used human T-ALL <br /> inhibitor sphingosine-1-phosphate receptor activity          cells shown schematically Supplemental Figure 4. Briefly, KOPT-K1 <br /> preclinical models variety cancers, including chronic lympho-           cells (10,000 well) seeded using MATRIX WellMate 384-well <br /> cytic leukemia (67), chronic myelogenous leukemia blast crisis (45),            plates RPMI supplemented 10% fetal calf serum, glutamine, pen- <br /> B cell acute lymphoblastic leukemia (45), mantle cell lymphoma              icillin/streptomycin 37&#194;&#176;C 5% CO2. To assess interactions between <br /> (68). However, phenothiazines, levels FTY-720           GSI (compound E) bioactive compounds, wells sequen- <br /> required PP2A stimulation low micromolar range, mak-             tially &#226;&#8364;&#339;pinned&#226;&#8364;? CyBi-Well duplicate compound E (100 nM; <br /> ing ideal clinical lead. It likely active       +GSI) DMSO (&#226;&#8364;&#8220;GSI) followed small molecule Broad Insti- <br /> selective PP2A-stimulating compounds need developed               tute bioactives collection. Bioactive library compounds delivered a <br /> they robust clinical impact anticancer drugs. Iden-            volume 25 nl produce final concentration ranged 5 &#206;&#188;M <br /> tification PP2A-stimulating effect phenothiazines provides            10 &#206;&#188;M. Positive controls consisted dexamethasone (200 nM) or <br /> a new path forward development small molecules                  dexamethasone (200 nM) plus GSI (100 nM compound E), negative <br /> improved on-PP2A diminished off-target activities.                          controls consisted vehicle (DMSO) GSI (100 nM compound E) <br />                                                                                 alone. After 48 hours treatment, CellTiter-Blue added 4 hours <br /> Methods                                                                         viable cell numbers determined measuring fluorescence 595 nm <br /> Transgenic zebrafish lines. The rag2:MYC-ER;mitf, rag2:dsRed2, rag2:EGFP-       plate reader. High-throughput screening data processed sepa- <br /> MYC, rag2:EGFP-BCL2 transgenic lines previously                   rately 2 assays (Figure 2A: +GSI; Figure 2B: &#226;&#8364;&#8220;GSI) deposition in <br /> described (12, 23, 69).                                                         ChemBank described (70). Complete raw processed data these <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014	                        651 <br />  research article <br />  <br />                                                                                                            &#226;&#8364;&#8220;(Z+GSI) &#195;&#8212; F, clearly discriminates com- <br />                                                                                                            pounds plotted Z+GSI <br />                                                                                                            (Figure 2D). Using initial cutoff F &gt; 0.5, <br />                                                                                                            identified 330 compounds, 43 <br />                                                                                                            compounds statistically distinct <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                                            2-assay DMSO distribution, modeled a <br />                                                                                                            multivariate normal distribution, sub- <br />                                                                                                            jected Holm-Bonferroni multiple hypothesis <br />                                                                                                            test correction. <br />                                                                                                               Cell culture. All cells cultured 37&#194;&#176;C <br />                                                                                                            5% CO2. Established cell lines grown <br />                                                                                                            RPMI supplemented 10% fetal calf <br />                                                                                                            serum, glutamine, penicillin/streptomycin. <br />                                                                                                            Primary human T-ALLs grown short- <br />                                                                                                            term cultures MS5-DLL1 stromal cells in <br />                                                                                                            high-serum (71) serum-free defined <br />                                                                                                            conditions (46) described. <br />                                                                                                               Cell viability assays. Cell viability measured <br />                                                                                                            using CellTiter-Blue cell viability assay kit <br />                                                                                                            (Promega). Cells plated black-walled <br />                                                                                                            96-well plates treated chemical com- <br />                                                                                                            pounds. Then 20 &#206;&#188;l CellTiter-Blue reagent <br />                                                                                                            added wells, contained <br />                                                                                                            100 &#206;&#188;l cells. After 4 hours 37&#194;&#176;C, fluores- <br />                                                                                                            cence recorded SpectraMax M3 micro- <br />                                                                                                            plate reader (Molecular Devices) measure <br />                                                                                                            resazurin reduction. <br />                                                                                                               Isobologram analysis. To determine whether <br />                                                                                                            drugs synergistic antileukemia cell growth <br />                                                                                                            effects, cells incubated various concen- <br />                                                                                                            trations drugs 48 hours incubat- <br />                                                                                                            ed CellTiter-Blue addition 4 hours. <br />                                                                                                            Cell numbers determined measuring <br />                                                                                                            fluorescence 595 nm plate reader. Data <br />                                                                                                            processed subjected isobologram <br />                                                                                                            analysis (72) using CalcuSyn determine the <br />                                                                                                            combination index (CI) ED50. A CI less <br />                                                                                                            0.7 ED50 taken indicative a <br />                                                                                                            synergistic interaction. <br />                                                                                                               Cell-cycle analysis. Cells washed with <br />                                                                                                            cold PBS fixed 90% ethanol overnight <br />                                                                                                            &#226;&#8364;&#8220;20&#194;&#176;C. Cells washed with <br /> Figure 8                                                                                                   cold PBS resuspended propidium iodide <br /> PPZ induces rapid dephosphorylation PP2A targets inhibits growth cultured primary (PI) staining solution (20 &#206;&#188;g/ml PI, 200 &#206;&#188;g/ml <br /> human T-ALL cells. (A&#226;&#8364;&#8220;D) Western blots showing effects incubation 15 minutes the <br />                                                                                                            DNAse-free RNAse A, 0.1% [v/v] Triton <br /> indicated doses PPZ. (E&#226;&#8364;&#8220;H) Effects GSI (compound E, 1 &#206;&#188;M) PPZ (5 &#206;&#188;M) in <br /> combination cell growth relative vehicle-treated control cells. Cells grown defined X-100 PBS). Following incubation 37&#194;&#176;C <br /> medium MS5-DLL-1 feeder cells. Cell counts determined 3 days treatment.                   15 minutes, samples analyzed a <br />                                                                                                            FACSCalibur flow cytometer (BD Biosciences) <br />                                                                                                            using Cell Quest software. <br /> screening experiments available ChemBank (http://chembank.             Apoptosis assays. Caspase 3 7 activities measured using Caspase- <br /> broadinstitute.org/assays/view-project.htm?id=1000654). To combine Glo 3/7 assay (Promega). Following treatment KOPT-K1 <br /> results 2 assays quantitatively, defined trigonometric scaling cells DMSO, GSI, PPZ, GSI plus PPZ, 5,000 cells sam- <br /> function express preferences (a) +GSI composite Z scores ple plated white-walled 96-well plates final volume of <br /> negative, (b) +GSI composite Z score amplitudes exceed &#226;&#8364;&#8220;GSI compos- 100 &#206;&#188;l. An equal volume Caspase-Glo 3/7 reagent added to <br /> ite Z score amplitudes; compounds fall left DMSO-con- well. After 1 hour room temperature, luminescence mea- <br /> trol distribution, main diagonal plot &#226;&#8364;&#8220;GSI compos- sured SpectraMax M3 microplate reader (Molecular Devices). As <br /> ite Z versus +GSI composite Z (Figure 2C). Using ratio ChemBank second measure apoptosis, cells stained annexin V PI <br /> composite Z scores (&#226;&#8364;&#8220;GSI/+GSI), defined &#206;&#184; = arctan(Z&#226;&#8364;&#8220;GSI/Z+GSI) using Apoptosis Detection Kit I (BD Biosciences) manufacturer&#226;&#8364;&#8482;s <br /> expressed scale factor F = cos(&#226;&#8364;&#8220;&#206;&#184;)cos(2&#206;&#184;), 2 factors respec- instructions, analyzed FACSalibur flow cytometer (BD Bio- <br /> tively expressing 2 preferences. As 2-assay composite score, sciences) using Cell Quest software. <br />  <br /> 652	                            The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014 <br />                                                                                                                                        research article <br />  <br />  <br />  <br />  <br /> Figure 9 <br /> PPZ inhibits growth dephosphorylates PP2A targets primary human T-ALL cells vivo. (A B) Effects treatment PPZ <br /> (10 mg/kg/d) growth human hTALL2 cells NSG mice. (A) Effects tumor cell growth assessed measuring biolumines- <br /> cence luciferized hTALL2 cells subcutaneous calcium phosphate scaffolds seeded human mesenchymal stem cells. Arrow indi- <br /> cates time initiation PPZ treatment. *P &lt; 0.0182. (B) Spleen weights treatment control mice times necropsy. *P &lt; 0.05. <br /> (C) Western blots prepared hTALL2 cells harvested engrafted mice 3 hours single dose PPZ (10 mg/kg). Each lane <br /> corresponds different animal. The blot stained phospho-p70S6K total p70S6K prepared second gel loaded <br /> with samples run parallel. <br />  <br />  <br />  <br />    Murine xenograft assays. Retrovirus transfecting MLCP,          pLKO.1-puro shPP2A C&#206;&#177; obtained Dana Farber Cancer Institute <br /> MSCV-based retrovector encoding firefly luciferase, mCherry, puro-            RNAi core facility. The targeting sequence inserted pLKO.1-puro <br /> mycin phosphotransferase, Phoenix GP/293T cells (provided                 shPP2A 5&#226;&#8364;&#178;-CCGGCCCATGTTGTTCTTTGTTATTCTCGAGAATAA- <br /> David Williams, Boston Children&#226;&#8364;&#8482;s Hospital). Transduced human hTALL2              CAAAGAACAACATGGGTTTTT-3&#226;&#8364;&#178;. pLKO.1-puro-GFP used an <br /> T-ALL cells isolated based mCherry expression using FACSAria            shRNA control vector. <br /> II SORP UV cell sorter (BD Biosciences). Luciferized hTALL2 cells               Chemicals. PPZ, chlorpromazine, chlorprothixene, pimethixene, pro- <br /> injected intrascaffold NOD/SCID &#206;&#179;-chain <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> (NSG) mice                  methazine, domeridone purchased Sigma-Aldrich. Com- <br /> bearing subcutaneous biphasic calcium phosphate scaffolds seeded             pound E FTY720 purchased Enzo Life Sciences. Okadaic <br /> mesenchymal stem cells followed tumor development using bio-              acid purchased LC Laboratories. <br /> luminescence imaging described (73). To assess antileukemic effects           Statistics. Statistical significance isobologram analyses determined <br /> of PPZ, mice treated daily 10 mg/kg PPZ intraperitoneal           using Calcusyn, differences spleen weights determined by <br /> injection T-ALL growth monitored bioluminescence. To assess            t test, performed Prism software. Statistical analyses KOPT-K1 cell <br /> the effect PPZ phosphorylation PP2A substrates T-ALL cells         line&#226;&#8364;&#8220;screening data performed described Figure 2 legend and <br /> vivo, mice bearing bone scaffolds loaded hTALL2 cells treated           Methods. <br /> with vehicle single dose PPZ (10 mg/kg) intraperitoneally. Three             Study approval. All animal studies approved Dana Farber Can- <br /> hours later, cells harvested scaffold 2 &#195;&#8212; 107 cells       cer Institute Animal Care Use Committee. <br /> incubated human CD45 MicroBeads (Miltenyi Biotec). Beads were <br /> subsequently washed cold PBS, pH 7.4, containing 0.5% BSA               Acknowledgments <br /> and 2 mM EDTA, resuspended 500 &#206;&#188;l PBS. Bound cells                 PP2A A&#206;&#177; shRNA gift W.C. Hahn. We wish thank <br /> separated beads MS column using MACS Separator                   P. Ballerini, J. Landman-Parker, T. Leblanc, A. Baruchel from <br /> immediately lysed Western blot analyses.                                      H&#195;&#180;pital Trousseau H&#195;&#180;pital R Debr&#195;&#169; help collection of <br />    Western blotting antibodies. Cells washed cold PBS resus-    patient samples S. Blacklow advice technical con- <br /> pended lysis buffer (50 mM Tris, pH 8.0, 5 mM EDTA, 100 mM NaF,              ceptual aspects studies. This work supported NIH <br /> 30 mM pyrophosphate, 2 mM molybdate, 2 mM vanadate, 1% NP-40) sup-                grants 1K08CA133103 (to A. Gutierrez) 5P01CA109901 (to A.T. <br /> plemented protease inhibitors (Sigma-Aldrich). Cell lysates incu-       Look); Specialized Center Research grant Leukemia <br /> bated ice 15 minutes centrifuged 15 minutes 12,000 g       Lymphoma Society (to J.C. Aster J.E. Bradner); grants from <br /> at 4&#194;&#176;C. Protein concentration measured Bradford assay (Bio-Rad),           William Lawrence Blanche Hughes Foundation (to A. Gutierrez, <br /> and equal amounts protein loaded SDS-polyacrylamide gels               J.C. Aster, A.T. Look, J.E. Bradner); grant Bear Neces- <br /> and resolved electrophoresis. Proteins stained following transfer      sities Pediatric Cancer Foundation (to A. Gutierrez); grants <br /> PVDF membranes following specific antibodies: anti-pAKT(S473),           Ligue Nationale contre le Cancer, Association Laurette Fugain, <br /> anti-AKT, anti-pERK(T202/Y204), anti-ERK, anti-pP70S6K(S389), anti-               Institut National du Cancer (INCA), Universit&#195;&#169;s Paris Diderot and <br /> P70S6K, anti-pBAD(S112), anti-BAD, anti-MYC, anti-PP2A subunit C              Paris Sud, INSERM, CEA Canceropole Ile France (StemPole) <br /> (all Cell Signaling Technology); pMYC(S62) (Abcam); anti&#226;&#8364;&#8220;                (to F. Pflumio, S. Poglio, B. Uzan). R. Groen received fund- <br /> &#206;&#178;-actin (Sigma-Aldrich). Staining developed Super Signal West            ing People Programme (Marie Curie Actions) Euro- <br /> Pico Chemiluminescent Substrate Super Signal West Dura Extended                pean Union&#226;&#8364;&#8482;s Seventh Framework Programme FP7/2007-2013/ <br /> Duration Substrate (Thermo Scientific) 2 minutes room tempera-             REA grant agreement no. 302428. A. Gutierrez research <br /> ture documented exposure x-ray film (Kodak).                            fellow Gabrielle&#226;&#8364;&#8482;s Angel Foundation Cancer Research and <br />    PP2A shRNA knockdown. Knockdown PP2A A&#206;&#177; (encoded                         previous scholar American Society Hematology Harold <br /> PPP2R1A) performed using pMKO.1-shAa vector described                  Amos Medical Faculty Development Program. The T-ALL cell line <br /> (74). Knockdown PP2A C&#206;&#177; (encoded PPP2CA) performed                 screen funded federal funds NCI Initia- <br />  <br /> 	                               The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014	                         653 <br />  research article <br />  <br /> tive Chemical Genetics contract no. N01-CO-12400. The                               Address correspondence to: A. Thomas Look, Department Pedi- <br /> content publication does necessarily reflect views                         atric Oncology, Dana-Farber Cancer Institute, 44 Binney St., Mayer <br /> policies Department Health Human Services, does                             630, Boston, Massachusetts 02115, USA. Phone: 617.632.5826; <br /> mention trade names, commercial products, organizations                                 Fax: 617.632.6989; E-mail: thomas_look@dfci.harvard.edu. Or <br /> imply endorsement US Government.                                                       to: Jon C. Aster, Department Pathology, Brigham Wom- <br />                                                                                               en&#226;&#8364;&#8482;s Hospital, 75 Francis St., Boston, Massachusetts 02115, USA. <br /> Received publication October 1, 2013, accepted October                                Phone: 617.525.4370; Fax: 617.525.4422; E-mail: jaster@rics.bwh. <br /> 30, 2013.                                                                                     harvard.edu. <br /> 	 1.	Marks DI, et al. T-cell acute lymphoblastic leuke-             proliferation survival. Ann N Y Acad Sci. 2008;             regulatory subunit B56alpha associates c-myc <br />       mia adults: clinical features, immunopheno-                1138:393&#226;&#8364;&#8220;403.                                                   negatively regulates c-myc accumulation. Mol <br />       type, cytogenetics outcome large           	 20.	Koren J, et al. Facilitating Akt clearance manipu-          Cell Biol. 2006;26(7):2832&#226;&#8364;&#8220;2844. <br />       randomised prospective trial (UKALL XII/ECOG                  lation Hsp70 activity levels. J Biol Chem. 2010;   	 39.	Junttila MR, et al. CIP2A inhibits PP2A human <br />       2993). Blood. 2009;114(25):5136&#226;&#8364;&#8220;5145.                         285(4):2498&#226;&#8364;&#8220;2505.                                               malignancies. Cell. 2007;130(1):51&#226;&#8364;&#8220;62. <br /> 	 2.	Goldberg JM, et al. Childhood T-cell acute lympho-       	 21.	Rho SB, Kim BR, Kang S. A gene signature-based            	40.	Chen Z, et al. MAP kinases. Chem Rev. 2001; <br />       blastic leukemia: Dana-Farber Cancer Institute            approach identifies thioridazine inhibitor             101(8):2449&#226;&#8364;&#8220;2476. <br />       acute lymphoblastic leukemia consortium experi-               phosphatidylinositol-3&#226;&#8364;&#178;-kinase (PI3K)/AKT path-           	 41.	Ugi S, Imamura T, Ricketts W, Olefsky JM. Protein <br />       ence. J Clin Oncol. 2003;21(19):3616&#226;&#8364;&#8220;3622.                    way ovarian cancer cells. Gynecol Oncol. 2011;               phosphatase 2A forms molecular complex with <br /> 	 3.	Ko RH, et al. Outcome patients treated                  120(1):121&#226;&#8364;&#8220;127.                                                 Shc regulates Shc tyrosine phosphorylation <br />       relapsed refractory acute lymphoblastic leu-         	 22.	Sachlos E, et al. Identification drugs including           downstream mitogenic signaling. Mol Cell Biol. <br />       kemia: Therapeutic Advances Childhood                    dopamine receptor antagonist selectively tar-              2002;22(7):2375&#226;&#8364;&#8220;2387. <br />       Leukemia Consortium study. J Clin Oncol. 2010;                cancer stem cells. Cell. 2012;149(6):1284&#226;&#8364;&#8220;1297.       	42.	Chiang CW, et al. Protein phosphatase 2A acti- <br />       28(4):648&#226;&#8364;&#8220;654.                                          	23.	Langenau DM, Feng H, Berghmans S, Kanki JP,                      vates proapoptotic function BAD inter- <br /> 	 4.	Koch U, Radtke F. Notch T-ALL: new players             Kutok JL, Look AT. Cre/lox-regulated transgenic                 leukin- 3-dependent lymphoid cells mecha- <br />       complex disease. Trends Immunol. 2011;32(9):434&#226;&#8364;&#8220;442.          zebrafish model conditional myc-induced T                  nism requiring 14-3-3 dissociation. Blood. 2001; <br /> 	 5.	Kopan R, Ilagan MX. The canonical Notch signal-                cell acute lymphoblastic leukemia. Proc Natl Acad               97(5):1289&#226;&#8364;&#8220;1297. <br />       ing pathway: unfolding activation mechanism.              Sci U S A. 2005;102(17):6068&#226;&#8364;&#8220;6073.                        	 43.	Chiang CW, et al. Protein phosphatase 2A dephos- <br />       Cell. 2009;137(2):216&#226;&#8364;&#8220;233.                              	24.	Lister JA, Robertson CP, Lepage T, Johnson SL,                   phorylation phosphoserine 112 plays gate- <br /> 	 6.	Weng AP, et al. c-Myc important direct target            Raible DW. nacre encodes zebrafish microph-                   keeper role BAD-mediated apoptosis. Mol Cell <br />       Notch1 T-cell acute lymphoblastic leukemia/             thalmia-related protein regulates neural-                  Biol. 2003;23(18):6350&#226;&#8364;&#8220;6362.	 <br />       lymphoma. Genes Dev. 2006;20(15):2096&#226;&#8364;&#8220;2109.                   crest-derived pigment cell fate. Development. 1999;       	44.	Matsuoka Y, Nagahara Y, Ikekita M, Shinomiya <br /> 	 7.	Sharma VM, et al. Notch1 contributes mouse                  126(17):3757&#226;&#8364;&#8220;3767.                                              T. A novel immunosuppressive agent FTY720 <br />       T-cell leukemia directly inducing expression     	 25.	Real PJ, et al. Gamma-secretase inhibitors reverse              induced Akt dephosphorylation leukemia cells. <br />       c-myc. Mol Cell Biol. 2006;26(21):8022&#226;&#8364;&#8220;8031.               glucocorticoid resistance T cell acute lympho-               Br J Pharmacol. 2003;138(7):1303&#226;&#8364;&#8220;1312. <br /> 	 8.	Palomero T, et al. NOTCH1 directly regulates                   blastic leukemia. Nat Med. 2009;15(1):50&#226;&#8364;&#8220;58.              	 45.	Neviani P, et al. FTY720, new alternative treat- <br />       c-MYC activates feed-forward-loop transcrip-      	 26.	Blackburn JS, et al. Notch signaling expands pre-             ing blast crisis chronic myelogenous leukemia and <br />       tional network promoting leukemic cell growth.                malignant pool T-cell acute lymphoblastic leuke-             Philadelphia chromosome-positive acute lympho- <br />       Proc Natl Acad Sci U S A. 2006;103(48):18261&#226;&#8364;&#8220;18266.           mia clones affecting leukemia-propagating               cytic leukemia. J Clin Invest. 2007;117(9):2408&#226;&#8364;&#8220;2421. <br /> 	 9.	Chan SM, Weng AP, Tibshirani R, Aster JC, Utz                  cell frequency. Leukemia. 2012;26(9):2069&#226;&#8364;&#8220;2078.           	46.	Yost AJ, et al. Defined, serum-free conditions for <br />       PJ. Notch signals positively regulate activity   	 27.	Ashworth TD, et al. Deletion-based mechanisms                   vitro culture primary human T-ALL blasts. <br />       mTOR pathway T-cell acute lymphoblastic leu-               Notch1 activation T-ALL: key roles RAG                Leukemia. 2013;27(6):1437&#226;&#8364;&#8220;1440. <br />       kemia. Blood. 2007;110(1):278&#226;&#8364;&#8220;286.                            recombinase conserved internal translational        	 47.	Cullion K, et al. Targeting Notch1 mTOR <br /> 	 10.	Felsher DW, Bishop JM. Reversible tumorigenesis               start site Notch1. Blood. 2010;116(25):5455&#226;&#8364;&#8220;5464.            pathways mouse T-ALL model. Blood. 2009; <br />       MYC hematopoietic lineages. Mol Cell. 1999;       	28.	Pacheco R, Prado CE, Barrientos MJ, Bernales S.                  113(24):6172&#226;&#8364;&#8220;6181. <br />       4(2):199&#226;&#8364;&#8220;207.                                                 Role dopamine physiology T-cells         	48.	DeAngelo DJ, Stone RM, Silverman LB, Aster <br /> 	11.	Langenau DM, et al. Myc-induced T cell leu-                    dendritic cells. J Neuroimmunol. 2009;216(1&#226;&#8364;&#8220;2):8&#226;&#8364;&#8220;19.            JC. A phase I clinical trial Notch inhibitor <br />       kemia transgenic zebrafish. Science. 2003;           	 29.	Horn AS, Post ML, Kennard O. Dopamine receptor                  MK-0752 patients T-cell acute lympho- <br />       299(5608):887&#226;&#8364;&#8220;890.                                            blockade neuroleptics, crystallographic               blastic leukemia/lymphoma (T-ALL) other <br /> 	12.	Gutierrez A, et al. Pten mediates Myc oncogene                 study. J Pharm Pharmacol. 1975;27(8):553&#226;&#8364;&#8220;563.                   leukemias. J Clin Oncol. 2006;24:6585. <br />       dependence conditional zebrafish model T        	 30.	Jaszczyszyn A, et al. Chemical structure pheno-        	 49.	Sachlos E, et al. Identification drugs including a <br />       cell acute lymphoblastic leukemia. J Exp Med. 2011;           thiazines biological activity. Pharmacol              dopamine receptor antagonist selectively tar- <br />       208(8):1595&#226;&#8364;&#8220;1603.                                             Rep. 2012;64(1):16&#226;&#8364;&#8220;23.                                          cancer stem cells. Cell. 2012;149(6):1284&#226;&#8364;&#8220;1297. <br /> 	 13.	Weng AP, et al. Activating mutations NOTCH1          	31.	Ong SE, et al. Identifying proteins           	50.	Zhelev Z, et al. Phenothiazines suppress prolif- <br />       human T cell acute lymphoblastic leukemia.                 small-molecule probes drugs bind cells. Proc             eration induce apoptosis cultured leukemic <br />       Science. 2004;306(5694):269&#226;&#8364;&#8220;271.                              Natl Acad Sci U S A. 2009;106(12):4617&#226;&#8364;&#8220;4622.                    cells influence viability nor- <br /> 	 14.	Gutierrez A, et al. High frequency PTEN, PI3K,       	 32.	Shi Y. Serine/threonine phosphatases: mechanism                 mal lymphocytes. Phenothiazines leukemia. <br />       AKT abnormalities T-cell acute lymphoblas-             structure. Cell. 2009;139(3):468&#226;&#8364;&#8220;484.                   Cancer Chemother Pharmacol. 2004;53(3):267&#226;&#8364;&#8220;275. <br />       tic leukemia. Blood. 2009;114(3):647&#226;&#8364;&#8220;650.               	 33.	Virshup DM, Shenolikar S. From promiscuity             	 51.	Gil-Ad I, Shtaif B, Levkovitz Y, Dayag M, Zeldich <br /> 	 15.	Palomero T, Ferrando A. Oncogenic NOTCH1 con-                 precision: protein phosphatases makeover.                 E, Weizman A. Characterization phenothiazine- <br />       trol MYC PI3K: challenges opportuni-               Mol Cell. 2009;33(5):537&#226;&#8364;&#8220;545.                                   induced apoptosis neuroblastoma glioma <br />       ties anti-NOTCH1 therapy T-cell acute lym-       	34.	Mavrakis KJ, et al. Genome-wide RNA-mediated                     cell lines: clinical relevance possible applica- <br />       phoblastic leukemias lymphomas. Clin Cancer               interference screen identifies miR-19 targets                tion brain-derived tumors. J Mol Neurosci. 2004; <br />       Res. 2008;14(17):5314&#226;&#8364;&#8220;5317.                                   Notch-induced T-cell acute lymphoblastic leukae-                22(3):189&#226;&#8364;&#8220;198. <br /> 	16.	Subramaniam PS, et al. Targeting nonclassical                  mia. Nat Cell Biol. 2010;12(4):372&#226;&#8364;&#8220;379.                   	52.	Gil-Ad I, et al. Phenothiazines induce apoptosis <br />       oncogenes therapy T-ALL. Cancer Cell. 2012;      	35.	Sablina AA, Hector M, Colpaert N, Hahn WC.                       B16 mouse melanoma cell line attenuate <br />       21(4):459&#226;&#8364;&#8220;472.                                                Identification PP2A complexes pathways                   vivo melanoma tumor growth. Oncol Rep. 2006; <br /> 	 17.	Zhu HG, Tayeh I, Israel L, Castagna M. Different              involved cell transformation. Cancer Res. 2010;              15(1):107&#226;&#8364;&#8220;112. <br />       susceptibility lung cell lines inhibitors            70(24):10474&#226;&#8364;&#8220;10484.                                       	 53.	Sangodkar J, et al. Targeting FOXO1/KLF6 axis <br />       tumor promotion inducers differentiation.        	 36.	Hahn K, Miranda M, Francis VA, Vendrell J, Zorzano              regulates EGFR signaling treatment response. <br />       J Biol Regul Homeost Agents. 1991;5(2):52&#226;&#8364;&#8220;58.                 A, Teleman AA. PP2A regulatory subunit PP2A-B&#226;&#8364;&#178;                  J Clin Invest. 2012;122(7):2637&#226;&#8364;&#8220;2651. <br /> 	 18.	Kau TR, et al. A chemical genetic screen identifies           counteracts S6K phosphorylation. Cell Metab. 2010;        	 54.	Chiang MY, et al. Leukemia-associated NOTCH1 <br />       inhibitors regulated nuclear export Fork-             11(5):438&#226;&#8364;&#8220;444.                                                  alleles weak tumor initiators acceler- <br />       head transcription factor PTEN-deficient tumor       	 37.	Yeh E, et al. A signalling pathway controlling c-Myc            ate K-ras-initiated leukemia. J Clin Invest. 2008; <br />       cells. Cancer Cell. 2003;4(6):463&#226;&#8364;&#8220;476.                        degradation impacts oncogenic transformation               118(9):3181&#226;&#8364;&#8220;3194. <br /> 	19.	Choi JH, et al. Potential inhibition PDK1/Akt               human cells. Nat Cell Biol. 2004;6(4):308&#226;&#8364;&#8220;318.         	55.	Gonzalez-Garcia S, et al. CSL-MAML-dependent <br />       signaling phenothiazines suppresses cancer cell      	38.	Arnold HK, Sears RC. Protein phosphatase 2A                      Notch1 signaling controls T lineage-specific <br />  <br />  <br /> 654	                                 The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 124&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 2&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;February 2014 <br />                                                                                                                                                        research article <br />  <br />       IL-7R{alpha} gene expression early human             	 61.	Palomero T, et al. Mutational loss PTEN induces     	 68.	Liu Q, et al. FTY720 shows promising vitro and <br />       thymopoiesis leukemia. J Exp Med. 2009;                   resistance NOTCH1 inhibition T-cell leuke-             vivo preclinical activity downmodulating <br />       206(4):779&#226;&#8364;&#8220;791.                                               mia. Nat Med. 2007;13(10):1203&#226;&#8364;&#8220;1210.                         Cyclin D1 phospho-Akt mantle cell lym- <br /> 	 56.	Zenatti PP, et al. Oncogenic IL7R gain-of-function      	62.	Kridel R, et al. Whole transcriptome sequencing               phoma. Clin Cancer Res. 2010;16(12):3182&#226;&#8364;&#8220;3192. <br />       mutations childhood T-cell acute lymphoblastic             reveals recurrent NOTCH1 mutations mantle           	 69.	Langenau DM, et al. Suppression apoptosis by <br />       leukemia. Nat Genet. 2011;43(10):932&#226;&#8364;&#8220;939.                     cell lymphoma. Blood. 2012;119(9):1963&#226;&#8364;&#8220;1971.                 bcl-2 overexpression lymphoid cells trans- <br /> 	 57.	Medyouf H, et al. High-level IGF1R expression        	 63.	Di Ianni M, et al. A new genetic lesion B-CLL:          genic zebrafish. Blood. 2005;105(8):3278&#226;&#8364;&#8220;3285. <br />       required leukemia-initiating cell activity             NOTCH1 PEST domain <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span>. Br J Haematol.            	 70.	Seiler KP, et al. ChemBank: small-molecule screen- <br />       T-ALL supported Notch signaling. J Exp              2009;146(6):689&#226;&#8364;&#8220;691.                                         ing cheminformatics resource database. Nucleic <br />       Med. 2011;208(9):1809&#226;&#8364;&#8220;1822.                             	 64.	Fabbri G, et al. Analysis chronic lymphocytic         Acids Res. 2008;36(Database issue):D351&#226;&#8364;&#8220;359. <br /> 	 58.	Dail M, et al. Mutant Ikzf1, KrasG12D, Notch1             leukemia coding genome: role NOTCH1 muta-           	 71.	Armstrong F, et al. NOTCH key regulator of <br />       cooperate T lineage leukemogenesis modu-               tional activation. J Exp Med. 2011;208(7):1389&#226;&#8364;&#8220;1401.         human T-cell acute leukemia initiating cell activity. <br />       late responses targeted agents. Proc Natl Acad Sci   	 65.	Puente XS, et al. Whole-genome sequencing iden-              Blood. 2009;113(8):1730&#226;&#8364;&#8220;1740. <br />       U S A. 2010;107(11):5106&#226;&#8364;&#8220;5111.                                tifies recurrent mutations chronic lymphocytic      	72.	Tallarida RJ. Drug synergism: detection <br /> 	 59.	Subramanyam D, Krishna S. c-Myc substitutes               leukaemia. Nature. 2011;475(7354):101&#226;&#8364;&#8220;105.                   applications. J Pharmacol Exp Ther. 2001; <br />       Notch1-CBF1 functions cooperative transfor-          	66.	Robinson DR, et al. Functionally recurrent rear-              298(3):865&#226;&#8364;&#8220;872. <br />       mation papillomavirus oncogenes. Virology.               rangements MAST kinase Notch                	 73.	Groen RW, et al. Reconstructing human hema- <br />       2006;347(1):191&#226;&#8364;&#8220;198.                                          gene families breast cancer. Nat Med. 2011;               topoietic niche immunodeficient mice: oppor- <br /> 	 60.	Wang H, et al. Genome-wide analysis reveals con-              17(12):1646&#226;&#8364;&#8220;1651.                                            tunities studying primary multiple myeloma. <br />       served divergent features Notch1/RBPJ            	 67.	Liu Q, et al. FTY720 demonstrates promising pre-             Blood. 2012;120(3):e9&#226;&#8364;&#8220;e16. <br />       binding human murine T lymphoblas-                     clinical activity chronic lymphocytic leukemia     	74.	Sablina AA, et al. The tumor suppressor PP2A <br />       tic leukemia cells. Proc Natl Acad Sci U S A. 2011;  <br /> </body></html>