<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 17 January 2014                                               Nucleic Acids Research, 2014, Vol. 42, No. 6 3623&#226;&#8364;&#8220;3637 <br />                                                                                                                 doi:10.1093/nar/gkt1386 <br />  <br />  <br /> Comparative ribosome profiling reveals extensive <br /> translational complexity different Trypanosoma <br /> brucei life cycle stages <br /> Juan-Jose&#194;&#180; Vasquez1, Chung-Chau Hon2,3, Jens T. Vanselow4, Andreas Schlosser4 and <br /> T. Nicolai Siegel1,* <br /> 1 <br />  Research Center Infectious Diseases, University Wuerzburg, Wuerzburg 97080, Germany, <br /> 2 <br />  De&#194;&#180;partement Biologie cellulaire et infection, Institut Pasteur, Unite&#194;&#180; Biologie Cellulaire du Parasitisme, <br /> Paris 75015, France, 3INSERM U786, Paris 75015, France 4Rudolf Virchow Center, University of <br /> Wuerzburg, Wuerzburg 97080, Germany <br />  <br /> Received November 7, 2013; Revised December 17, 2013; Accepted December 18, 2013 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        INTRODUCTION <br /> While gene expression fundamental tightly                              Eukaryotic gene expression regulated multiple levels, <br /> controlled cellular process regulated                                including extensive elaborate post-transcriptional <br /> multiple steps, exact contribution step                             regulation affects RNA stability, protein translation <br /> remains unknown organism. The absence                                 protein turnover (1). How individual <br /> transcription initiation regulation RNA polymer-                            levels contribute &#239;&#172;?nal outcome, steady state <br />                                                                                 level protein, known organism. The <br /> ase II protozoan parasite Trypanosoma brucei <br />                                                                                 need regulation level transcription and <br /> greatly simplifies task elucidating contri- <br />                                                                                 RNA stability, example level translation or <br /> bution translation global gene expression.                                protein stability, evident transcriptome <br /> Therefore, sequenced ribosome-protected                                 proteome studies revealed weak <br /> mRNA fragments T. brucei, permitting                                     correlations mRNA protein levels in <br /> genome-wide analysis RNA translation trans-                              various organisms, including yeast, humans proto- <br /> lational efficiency. We varies                             zoan parasite Leishmania major (2&#226;&#8364;&#8220;6). Furthermore, <br /> greatly life cycle stages parasite                               mRNA translation recently the <br /> and  100-fold genes, contributing                               extensively regulated step mammals (7). <br /> gene expression similar extent RNA stability.                              L. major related kinetoplastids Trypanosoma <br /> The ability map ribosome positions sub-codon                              cruzi Trypanosoma brucei causative agents of <br /> resolution revealed extensive translation                                  leishmaniasis, Chagas disease sleeping sickness, which <br /> upstream open reading frames located 50                                  signi&#239;&#172;?cant human diseases leading death of <br /> UTRs enabled identification hundreds                              tens thousands people worldwide year (8,9). <br />                                                                                 Interestingly, genome organization kinetoplastids <br /> previously un-annotated putative coding sequences <br />                                                                                 differs eukaryotes most <br /> (CDSs). Evaluation existing proteomics and <br />                                                                                 protein-coding genes transcribed long <br /> genome-wide RNAi data confirmed translation                                 polycistronic transcription units (PTUs). These PTUs <br /> of previously un-annotated CDSs suggested                                   encompass &gt;100 functionally unrelated genes <br /> an important role &gt;200 CDSs                                     (10&#226;&#8364;&#8220;13). Maturation polycistronic RNA precursors <br /> parasite <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span>, especially form                               functional mRNA transcripts involves trans-splicing <br /> infective mammals. Overall data                                39 nt leader sequence 50 end polyadenyla- <br /> translational control plays prevalent important                           tion 30 end (14&#226;&#8364;&#8220;16). The absence promoter <br /> role different parasite life cycle stages                                 sequence motifs, organization genes PTUs and <br /> T. brucei.                                                                      apparently conserved open chromatin structure <br />  <br />  <br />  <br />  <br /> *<span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed. Tel: +49 931 31 88 499; Fax: +49 931 318 25 78; Email: nicolai.siegel@uni-wuerzburg.de <br />  <br /> &#195;&#376; The Author(s) 2014. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which <br /> permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />  3624 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br /> surrounding RNA polymerase II (pol II) transcription            While approach successfully used demon- <br /> start sites (12,17) strongly indicate lack regulation      strate differential translation RNA T. brucei (43,44), <br /> of gene expression level pol II transcription         adapted higher resolution approach termed <br /> initiation. This lack transcriptional regulation      ribosome pro&#239;&#172;?ling recently developed (45). This <br /> greatly facilitate effort quantify extent         approach based high-throughput sequencing of <br /> which different post-transcriptional mechanisms contrib-        ribosome-protected RNA fragments, so-called ribosome <br /> ute gene expression.                                         &#226;&#8364;&#732;footprints&#226;&#8364;&#8482;. RNA protected ribosomes is <br />    Despite lack transcriptional control, L. major,       removed nuclease digestion; ribosome footprints are <br /> T. cruzi T. brucei capable tightly regulating        converted libraries DNA molecules suitable for <br /> gene expression complex life cycles,           high-throughput sequencing, abundance indi- <br /> which requires adaptation parasites           vidual footprints determined deep sequencing. In <br /> large environmental differences insect        yeast, mice Escherichia coli, measurements the <br /> vectors mammalian hosts (18,19). In T. brucei,        average ribosome occupancy (ribosome density pro&#239;&#172;?les) <br /> differences include, others, change tempera-         successfully used estimate rates trans- <br /> ture 37  C mammalian host 27  C           lation (rates protein synthesis) (45&#226;&#8364;&#8220;47). Ribosome <br /> insect vector change availability glucose,      density, perfect, shown much <br /> the preferred energy source parasite (20,21).            better predictor protein levels measurements of <br /> Consequently, signi&#239;&#172;?cant effort invested            mRNA levels (45). <br /> elucidating post-transcriptional mechanisms gene            Importantly, approach provides quantita- <br /> control parasites. Work laboratories      tive information number RNA molecules <br /> has led identi&#239;&#172;?cation sequence motifs capable      associated ribosomes, yields position-speci&#239;&#172;?c <br /> modulating RNA stability translational ef&#239;&#172;?ciency        information regarding location ribosomes on <br /> life cycle-speci&#239;&#172;?c manner (22&#226;&#8364;&#8220;27), bias codon usage        mRNA transcripts. This information critical, associ- <br /> suggested affect translational ef&#239;&#172;?ciency (28),         ation RNA transcript ribosome does not <br /> genome-wide analysis mRNA stability half-          necessarily mean transcript translated. <br /> lives transcripts range orders mag-        The ribosome associated 50 UTR it <br /> nitude T. brucei (29).                                       stalled, failing produce functional protein. In <br />    A comparable study evaluate degree regulation      addition, ribosome positions used accurately map <br /> at level protein translation performed      correct CDSs help identify novel CDSs have <br /> even multiple observations suggest translational         missed previous genome annotations (48). <br /> control important regulator gene expression.              Here report adaptation ribosome pro&#239;&#172;?ling <br /> For example, kinetoplastid genomes encode                  approach T. brucei. Comparative ribosome pro&#239;&#172;?ling <br /> proteins involved translation initiation control        analyses bloodstream procyclic parasite forms <br /> those yeast metazoans, including              allowed generate genome-wide picture transla- <br /> three homologues poly(A) binding proteins        tion. We observed 100-fold range translational <br /> four isoforms translation initiation factor (eIF4E)      ef&#239;&#172;?ciency genes life cycle-speci&#239;&#172;?c differences <br /> and &#239;&#172;?ve eIF4G isoforms (30&#226;&#8364;&#8220;34). Moreover, RNA-                  translational ef&#239;&#172;?ciency large number genes. In <br /> sequencing analyses performed T. brucei revealed wide-       addition, ribosome pro&#239;&#172;?ling data enabled identi- <br /> spread alternative trans-splicing, resulting multiple        &#239;&#172;?cation hundreds previously un-annotated CDSs <br /> transcripts, varying 50 untranslated region (UTR)          incorrectly annotated translation initiation sites, and <br /> lengths particular gene (13,35,36). These &#239;&#172;?ndings         suggest regulatory role uORFs. <br /> suggest variations 50 UTR lengths lead to <br /> the in- exclusion regulatory elements in&#239;&#172;&#8218;uence <br /> translational ef&#239;&#172;?ciency (37). Such regulatory elements          MATERIALS AND METHODS <br /> could small, so-called upstream open reading frames <br />                                                                 T. brucei culture cell harvest <br /> (uORFs) micro ORFs, ORFs located the <br /> 50 UTR mRNA. In eukaryotes, translation initiation           The procyclic form (PF) T. brucei strain Lister 427 were <br /> involves recruitment pre-initiation complex         cultured 27  C SDM-79 supplemented 10% fetal <br /> the 50 end mRNA, followed scanning UTR             bovine serum hemin (7.5 mg/l) medium (21) a <br /> to locate &#239;&#172;?rst AUG start codon (38). uORFs affect       density 107 cells/ml. Wild-type bloodstream form (BF) <br /> the scanning process shown in&#239;&#172;&#8218;uence            Lister 427 (MITat 1.2 clone 221) cultured 37  C <br /> translation downstream coding sequence (CDS)             HMI-11 cell density 1.5   106 cells/ml. Two <br /> by affecting rate site translation initi-   minutes harvest, cycloheximide added &#239;&#172;?nal <br /> ation (39,40). Based RNA-sequencing data, T. brucei       concentration 100 mg/ml. Cells collected centri- <br />  20% 50 UTRs contain uORF (35,36). In           fugation 3000   g, 4  C 5 min, washed polysome <br /> addition, shown luciferase reporter           lysis buffer (10 mM Tris-HCl pH 7.4, 300 mM KCl and <br /> that removal upstream start-codon lead         10 mM MgCl2), transferred 1.5-ml microcentrifuge <br /> a 7-fold increase protein levels (41).                       tube pelleted. To lyse cells, 360 ml polysome lysis <br />    In past, approaches identify quantify             buffer, 40 ml 10% n-octylglycoside 20 units of <br /> translated RNA transcripts focused isolation        TURBO DNaseI (Ambion) added 109 cells and <br /> of RNA transcripts polyribosome fractions (42).            incubated 30 min ice. The lysate centrifuged at <br />                                                                          Nucleic Acids Research, 2014, Vol. 42, No. 6 3625 <br />  <br />  <br /> 16 000  g, 4  C 10 min, supernatant transferred    RNA library life cycle stage. To investigate <br /> new microcentrifuge tube OD260 determined           distinct 3-nt periodicity ribosome pro&#239;&#172;?ling <br /> using Nanodrop 2000.                                          libraries, 50 ends mapped reads piled and <br />                                                                 piled-up coverage position gene <br /> Preparation RNA footprint mRNA sequencing                (using start stop codon reference point) <br /> libraries                                                       summed meta-gene analyses. Genes less <br /> For BF PF, 200 ml aliquots lysate               10 reads plotted region excluded. <br /> (OD260 = 40) digested RNase I (Ambion) at <br />                                                                 De&#239;&#172;?nition upstream ORFs novel CDSs <br /> RT (1200 units) ice (1600 units). After 1 h, diges- <br /> tions stopped adding 100 units SUPERase In           A uORF de&#239;&#172;?ned ORF  9 nt located <br /> RNase inhibitor (Ambion) RNase-treated samples.          annotated 50 UTR strand <br /> In parallel, 100 units SUPERase In RNase inhibitor           annotated ORF. Novel CDSs de&#239;&#172;?ned any <br /> were added 200 ml aliquot lysate containing         translated ORF   30 nt located 20 nt away <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> RNase I (undigested control). Monosomes puri&#239;&#172;?ed            existing annotations. In cases multiple <br /> using sucrose gradients described previously (45).           overlapping novel ORFs different frames <br />    Total RNA undigested lysates footprints         strand, longest retained. An <br /> collected monosome fractions puri&#239;&#172;?ed using          ORF de&#239;&#172;?ned translated   70% CDS is <br /> hot acid Phenol&#226;&#8364;&#8220;Chloroform&#226;&#8364;&#8220;Isoamyl alcohol (v/v/v               covered pooled ribosome pro&#239;&#172;?ling reads (i.e. <br /> 25:24:1) 65  C (49). To generate mRNA sequencing             libraries)  2 reads nucleotide. <br /> libraries, undigested RNA polyA-enriched using a <br /> Dynabeads&#195;&#8226; mRNA Puri&#239;&#172;?cation Kit (Ambion) according              Calculation ribosome release score <br /> the manufacturer&#226;&#8364;&#8482;s instructions. The polyA-enriched             Ribosome release scores (RRSs) ORFs, including <br /> RNA fragmented incubation RNA                    annotated CDSs, putative CDSs uORFs, were <br /> Fragmentation Reagent (Ambion) 70  C 30 min.             calculated described slight modi&#239;&#172;?cations (53). <br /> Successful fragmentation monitored 15%                 Brie&#239;&#172;&#8218;y, pseudo 30 UTR region (pUTR3) &#239;&#172;?rst <br /> denaturing-PAGE gel. Both, ribosome footprints              assigned ORFs. pUTR3 ORF de&#239;&#172;?ned as <br /> fragmented mRNA (26&#226;&#8364;&#8220;34 nt) size-selected elec-          region stop codon downstream start <br /> trophoresis using 15% denaturing-PAGE gel             codon frames strand. <br /> custom-made (IDT) synthetic RNA markers [50 -AUGUA              Reads mapped CDSs pUTR3 per <br /> CACGGAGUCGAGCUCAACCCGCAACGCGA-                                  kilobase (i.e. CDSrpk pUTR3rpk) ribosome <br /> (Phos)-30 ] [50 -AUGUACACGGAGUCGACCCAAC                     pro&#239;&#172;?ling RNA-sequencing control libraries were <br /> GCGA-(Phos)-30 ].                                               calculated ORF. The ratio CDSrpk to <br />    Sequencing libraries prepared described (50)         pUTR3rpk ribosome pro&#239;&#172;?ling RNA- <br /> except omission rRNA depletion step               sequencing control libraries (i.e. riboratio RNAratio) <br /> during footprint library preparation use         calculated. RRS ORF de&#239;&#172;?ned as <br /> KAPA HiFi polymerase (Kapa Biosystems) &#239;&#172;?nal             ratio riboratio RNAratio. <br /> ampli&#239;&#172;?cation step. <br />                                                                 Identi&#239;&#172;?cation novel putative CDSs important for <br /> Pre-processing mapping read                              parasite&#226;&#8364;&#8482;s &#239;&#172;?tness <br /> Reads libraries processed mapped using        We reanalysed data published genome-wide RNA <br /> the parameters. Adapter sequence (i.e. CTGTAGGC            interference &#239;&#172;?tness&#226;&#8364;&#8220;costs association study (54) our <br /> ACCATCAAT) trimmed 30 end reads                 newly de&#239;&#172;?ned CDSs. Brie&#239;&#172;&#8218;y, short read data down- <br /> using cutadapt (http://<span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>.google.com/p/cutadapt/)         loaded European Nucleotide Archive acces- <br /> reads shorter 20 nt trimming removed.           sion number ERP000431. Reads mapped to <br /> Trimmed reads mapped reference genome               genome using bowtie2 default &#226;&#8364;&#732;local-sensitive&#226;&#8364;&#8482; <br /> using bowtie-2 default &#226;&#8364;&#732;local-sensitive&#226;&#8364;&#8482; mode (51).        mode. The number reads mapping annotated <br /> Genome, gene annotations, strain 927              putative CDSs counted. To identify those <br /> version 4.2 downloaded EuPathDB (52)              putative CDSs signi&#239;&#172;?cantly reads mapped <br /> used reference analyses.                          <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>RNA interference</span> libraries, potentially con- <br />                                                                 tribute parasite&#226;&#8364;&#8482;s &#239;&#172;?tness, performed differential <br /> Calculation abundance, translational ef&#239;&#172;?ciency read <br />                                                                 read count analyses pair-wise library comparisons <br /> 50 -end periodicity <br />                                                                 (listed below) using softwares, including DESeq1 <br /> A read considered map region midpoint       (55), DESeq2 (http://www.bioconductor.org/packages/ <br /> falls annotated range. Abundance region      release/bioc/html/DESeq2.html) EdgeR (56), with <br /> a library de&#239;&#172;?ned reads mapped kilobase           default settings. CDSs identi&#239;&#172;?ed signi&#239;&#172;?- <br /> million non-structural RNA reads (rpkm). Non-structural         cantly reads RNA interference libraries all <br /> RNA refers reads mapped genome excluding          algorithms (P &lt; 0.05 case) linear fold change <br /> annotated rRNA tRNA regions. Translational ef&#239;&#172;?-               5 considered potentially related loss of <br /> ciency region de&#239;&#172;?ned ratio abundance         &#239;&#172;?tness. The pair-wise library comparisons include BF <br /> in ribosome pro&#239;&#172;?ling library control         uninduced versus BF induced 6 days, BF uninduced <br />  3626 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br /> versus differentiated cells induced BF uninduced         prepared different conditions demonstrated ro- <br /> versus PF induced.                                           bustness ribosome pro&#239;&#172;?ling approach despite the <br />                                                              elaborate library preparation protocol (Supplementary <br /> Analysis proteomics data                                  Figure S2, BF digestion 4  C versus digestion room <br /> All mass spectrometric raw data &#239;&#172;?les Butter et al.      temperature, R2 = 0.9755). <br /> (57) processed described data      To ensure nuclease-resistant RNA fragments rep- <br /> searched additionally previously un-annotated       resent ribosome footprints short RNA fragments <br /> CDSs listed Supplementary Tables S4 S5. Brie&#239;&#172;&#8218;y,       protected RNA-binding proteins, compared <br /> MaxQuant 1.4.1.2 including Andromeda search engine       genome-wide distribution sequence reads obtained <br /> (58,59) used processing raw data database        ribosome pro&#239;&#172;?ling RNA-sequencing libraries. <br /> searching. The experimental design template used      Alignment ribosome pro&#239;&#172;?ling reads revealed the <br /> described (57). The search performed    nuclease-protected reads predominantly mapped along <br /> combination databases: T. brucei database con-    CDSs reads aligning intergenic regions <br /> taining 19 103 annotated protein entries strains 427     (Figure 1A). The 50 nucleotide ribosome pro&#239;&#172;?ling <br /> and 927 (version 5.0, Tb427 Tb927, downloaded            reads aligned 12 nt upstream ATG codon to <br /> from http://tritrypdb.org), list uORFs putative      18 nt upstream stop codon (Figure 1B). Given a <br /> CDSs identi&#239;&#172;?ed study (Supplementary Tables S4        footprint <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> 28&#226;&#8364;&#8220;30 nt, ribosome protects mRNA <br /> and S5) database containing typical contaminants.      12 nt upstream start codon 9&#226;&#8364;&#8220;11 nt down- <br /> Enzyme search speci&#239;&#172;?city Trypsin/P            stream stop codon. In contrast ribosome foot- <br /> miscleavages tryptic digests LysC           print reads, RNA-sequencing reads aligned the <br /> miscleavages digests lysyl endopeptidase.           CDSs UTRs, expected (Figure 1A). In addition <br /> Carbamidomethylation cysteines set &#239;&#172;?xed modi-      enrichment CDSs, alignment of <br /> &#239;&#172;?cation; methionine oxidation protein N-terminal         ribosome pro&#239;&#172;?ling reads revealed distinct 3-nt period- <br /> acetylation considered variable modi&#239;&#172;?cations.        icity, 68% BF ribosome pro&#239;&#172;?ling reads starting at <br /> Mass accuracy tolerances (after recalibration)          &#239;&#172;?rst nucleotide codon (Figure 1B C and <br /> 6 ppm precursor ions, 0.5 Da CID MS/MS           Supplementary Figure S3). No 3-nt periodicity was <br /> and 20 ppm HCD MS/MS spectra, respectively.              observed mRNA sequence reads. <br /> False discovery rate &#239;&#172;?xed 1% peptide              Only T. brucei genes introns, encoding a <br /> protein level, posterior error probability (PEP)     poly(A) polymerase encoding DNA/RNA <br /> sorting criterion. Identi&#239;&#172;?cations matched       helicase (61,62). Surprisingly, decrease <br /> runs 2-min window. Protein groups identi&#239;&#172;?ed      sequence reads detectable RNA-sequencing across <br /> the potential CDS database unique           introns, ribosome pro&#239;&#172;?ling data clearly show <br /> peptide PEP &lt; 0.0005.                                  intronic RNA translated (Figure 2A). These <br />                                                              data indicate process cis-splicing un- <br />                                                              usually inef&#239;&#172;?cient trypanosomes, tight <br /> RESULTS                                                      control mechanism ensures mature mRNAs <br /> Ribosome footprints reveal ribosome position sub-codon    enter translational machinery. <br /> resolution                                                      The strong enrichment ribosome pro&#239;&#172;?ling reads <br />                                                              CDSs, lack ribosome pro&#239;&#172;?ling reads across <br /> To obtain genome-wide information protein synthesis,      introns observed 3-nt periodicity characteristic <br /> we adapted ribosome pro&#239;&#172;?ling protocol published         ribosome-protected RNA footprints argue against <br /> yeast mice (45,46) use T. brucei. We sequenced    unspeci&#239;&#172;?c RNA protection (45). Thus, ribosome pro&#239;&#172;?ling <br /> ribosome footprints fragmented mRNA         reads represent ribosome footprints provide <br /> trypanosome stages adapted vitro        means measure protein synthesis and, addition, <br /> culture, PF tsetse &#239;&#172;&#8218;y midgut              identify previously unannotated CDSs as <br /> proliferating BF lives blood mammalian    non-translated transcripts previously annotated coding <br /> host. Cultured cells treated cycloheximide      (for example Figure 1A, Tb927.10.2360). The may <br /> arrest translating ribosomes harvested. Because          incorrectly annotated translated the <br /> cycloheximide shown stabilize RNA tran-          life cycle stages examined. <br /> scripts T. brucei (60), treatment limited to <br /> 2 min applied both, cells used ribosome <br />                                                              Identi&#239;&#172;?cation translation initiation sites <br /> pro&#239;&#172;?lilng cells used RNA-sequencing analyses. <br /> To maximize accuracy ribosome footprints,         Genome-wide mapping spliced leader acceptor sites <br /> we optimized digestion ef&#239;&#172;?ciency testing different    (SASs), i.e. 50 end 50 UTR the <br /> nuclease concentrations reaction temperatures            spliced leader RNA trans-spliced, revealed hundreds <br /> (&#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section Supplementary           instances SASs mapped annotated <br /> Figure S1). Next, map translated regions RNA       CDSs far upstream annotated CDSs (13,35,36). <br /> to exclude RNase-protected regions scanned       These &#239;&#172;?ndings indicated genes true <br /> the 43S pre-initiation complex, used sucrose gradients    CDS shorter annotated CDS while, for <br /> to speci&#239;&#172;?cally enrich monosomes (Supplementary           others, allowed possibility translation initiation <br /> Figure S1). The high reproducibility libraries       AUG upstream annotated translation <br />                                                                                                                                                  Nucleic Acids Research, 2014, Vol. 42, No. 6 3627 <br />  <br />  <br /> A                                                                                                         A <br />  <br />  <br />  <br />  <br />                                                                                                           footprint (RPM) mRNA (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                  footprint (RPM) mRNA (RPM) <br />                                 mRNA density PF                                                                                               Tb927.3.3160                                                          Tb927.8.1510 <br /> mRNA (RPM) <br />  <br />  <br />  <br />                       4                                                                                                                2.0 <br />                                                                                                                                                                                                                4 <br />                       2 <br />                                                                                                                                        1.0                                                                     2 <br />                                                                                                                                        0.0                                                                     0 <br />                       0 <br /> footprint (RPM) <br />  <br />  <br />  <br />  <br />                       6         ribosome footprint density PF                                                                       0.3 <br />                                                                                                                                                                                                               2.0 <br />                       4                                                                                                                0.2 <br />                                                                                                                                        0.1                                                                    1.0 <br />                       2 <br />                                                                                                                                        0.0                                                                    0.0 <br />                       0 <br />                                                                                                           Chr III                                                                                  Chr VIII 00 <br />                                                                                                                                                 00      00      00      00                                          00      00 <br />                                 mRNA denstiy BF                                                                                           3,0     4,0     5,0     6,0                                   0,0   2,0     4,0 <br /> mRNA (RPM) <br />  <br />  <br />  <br />  <br />                       4                                                                                                                    81      81      81      81                                   49     49      49 <br />  <br />                       2 <br />                                                                                                           B <br />                       0 <br />  <br />  <br />  <br />  <br />                                                                                                           footprint (RPM) mRNA (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              footprint (RPM) mRNA (RPM) <br />                                                                                                                                                 Tb927.5.1990                                                           Tb927.10.2720 <br />                                                                                                                                        4 <br /> footprint (RPM) <br />  <br />  <br />  <br />  <br />                       3         ribosome footprint density BF                                                                                                                                              2.0 <br />                       2                                                                                                                2                                                                      1.0 <br />                       1                                                                                                                0                                                                      0.0 <br />                       0 <br />                                                                                                                                        4                                                                      0.4 <br />                                      20        30          40          50        60          70 <br />                                   .23       .23         .23         .23       .23         .23 <br />                               7.10      7.10        7.10        7.10      7.10        7.10                                             2                                                                      0.2 <br />                             92        92          92          92        92          92 <br />                           Tb       Tb           Tb          Tb        Tb          Tb                                                   0                                                                      0.0 <br />      Chr X 600,000                           602,000          604,000     606,000         608,000 <br />                                                                                                           Chr V    00   00   00   00                                                                           Chr X     00        00        00 <br />                                                                                                                 5,5 16,0 16,5 17,0                                                                                    9,0       0,0       1,0 <br />                                                                                                               61     6    6    6                                                                                    69        70        70 <br /> B                                          28 nt - ribosome footprint <br />                                                                                                           Figure 2. Ribosome footprints reveal translated regions. (A) Ribosome <br /> relative enrichment <br />  <br />  <br />  <br />  <br />                                                    AUG                                        UAA <br />   read 5&#194;&#180; ends <br />  <br />  <br />  <br />  <br />                                                                                                           pro&#239;&#172;?les intron-containing genes. Introns represented as <br />                                                      P site <br />  <br />  <br />  <br />  <br />                                                                                           P site <br />                                                      A site <br />  <br />  <br />  <br />  <br />                                                                                           A site <br />  <br />  <br />  <br />  <br />                           0.4                                                                             dashed line. (B) Ribosome pro&#239;&#172;?les possibly mis-annotated <br />                                                                                                           CDSs. Black bars mark annotated CDSs. Grey bars mark CDSs pre- <br />                           0.2                                                                             dicted based ribosome pro&#239;&#172;?les. Green boxes represent ATG codons <br />                                                                                                           red boxes represent stop codons. <br />                            0 <br />                                   -24 -18 -12 -6   0     6 12 18 24        -24 -18 -12 -6     0    6 12 <br />                                       CDS position (nt start)          CDS position (nt stop) <br />  <br /> C                                                                                                            In contrast RNA-sequencing, ribosome pro&#239;&#172;?ling <br />                                                                                                           allows determination translational landscape. <br /> percentage reads <br />  <br />  <br />  <br />  <br />                            80 <br />                                          ribosome footprint                                               Thus, allows identi&#239;&#172;?cation CDS mis-annotations <br />                                          mRNA <br />                            60                                                                             determination probable translation <br />                            40 <br />                                                                                                           initiation sites. Exemplary numerous genes, for <br />                                                                                                           Tb927.5.1990 detected translation the <br />                            20                                                                             &#239;&#172;?rst second AUG (green bars) indicating that <br />                             0                                                                             region translated BF PF that <br />                                  012 012                                                                  translation initiates second AUG (Figure 2B, <br />                                    frame <br />                                                                                                           left panel). For Tb927.10.2720 observed strong trans- <br /> Figure 1. Ribosome footprints reveal coding sequences sub-codon                                        lation upstream annotated CDS. The fact that <br /> resolution. (A) Ribosome footprints strongly enriched                                          translation starts upstream AUG, in <br /> CDSs. mRNA densities ribosome densities shown reads <br /> per nucleotide million reads (RPM) normalize differences                                       frame annotated translation initiation site, <br /> in library size. (B) Alignment 50 nucleotide ribosome foot-                                   suggests CDS Tb927.10.2720 longer than <br /> print reads map close translation start translation termin-                                    annotated (Figure 2B, right panel). These examples <br /> ation sites. Blue boxes mark approximate size ribosome                                         ribosome pro&#239;&#172;?ling data used the <br /> footprint. (C) Percentage position sequence reads relative to <br /> reading frame. <br />                                                                                                           re-annotation incorrectly annotated CDSs. <br />  <br />                                                                                                           Ribosome pro&#239;&#172;?ling reveals extensive translational control <br />                                                                                                           While role RNA stability investigated a <br /> initiation site. However, RNA-sequencing allowed                                                    genome-wide scale T. brucei (29), global regulation at <br /> the unambiguous identi&#239;&#172;?cation mis-annotation                                                       levels protein translation analysed. <br /> many genes, translation initiation does occur                                                  Therefore, decided apply ribosome pro&#239;&#172;?ling <br /> at &#239;&#172;?rst AUG downstream SAS. Thus, use-                                                     approach evaluate degree regulation occurring <br /> fulness RNA-sequencing identify true translation                                                level translation trypanosomes. <br /> initiation sites limited depends genomic                                                      To estimate rate protein synthesis trypano- <br /> context.                                                                                                  somes, determined ribosome footprint density <br />  3628 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br />                                                                A <br /> across CDSs [measured reads million reads kb                                                         BF                                                                   PF <br />  <br />  <br />  <br />  <br />                                                                number genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                           number genes <br />                                                                                                      3000                             mRNA                                  3000                              mRNA <br /> (rpkm), &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section]. Of 77 million                                                                           footprints                                                              footprints <br /> PF 44 million BF ribosome footprint sequence reads,                                              2000                                                                   2000 <br /> 28% (PF) 14% (BF) aligned annotated <br />                                                                                                      1000                                                                   1000 <br /> CDSs, 70% (PF) 85% (BF) mapped known <br /> structural non-coding RNA (ncRNA). The high percent-                                                      0                                                                     0 <br /> age structural ncRNA footprint fraction been <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                            15 <br />                                                                                                               -4 <br />                                                                                                               -2 <br />                                                                                                                0 <br />                                                                                                                2 <br />                                                                                                                4 <br />                                                                                                                6 <br />                                                                                                                8 <br />                                                                                                               10 <br />                                                                                                               12 <br />                                                                                                               14 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     -6 <br />                                                                                                                                                                                          -3 <br />                                                                                                                                                                                               0 <br />                                                                                                                                                                                                    3 <br />                                                                                                                                                                                                        6 <br />                                                                                                                                                                                                             9 <br />                                                                                                                                                                                                            12 <br /> reported previously (45) results large                                                                    log2 rpkm                                                           log2 rpkm <br /> of rRNA monosome fraction. The lower percentage <br />                                                                B <br /> of CDS reads BF sample compared PF                                                   1500       BF                                                          1500     PF <br />  <br />  <br />  <br />  <br />                                                                number genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                           number genes <br /> sample libraries generated the <br /> wild-type 427 strain reads aligned better-                                         1000                                                                   1000 <br /> annotated genome TREU 927 strain. In BF, <br />                                                                                                       500                                                                    500 <br /> some highly expressed genes encoding for <br /> variant surface glycoproteins differ sequence                                                  0                                                                     0 <br /> the 427 927 strains, footprints genes <br />  <br />  <br />  <br />  <br />                                                                                                               .0 <br />                                                                                                                     .0 <br />                                                                                                                           .0 <br />                                                                                                                                  .0 <br />                                                                                                                                   0 <br />                                                                                                                                   0 <br />                                                                                                                                   0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      .0 <br />                                                                                                                                                                                            .0 <br />                                                                                                                                                                                             .0 <br />                                                                                                                                                                                             .0 <br />                                                                                                                                                                                              0 <br />                                                                                                                                                                                              0 <br />                                                                                                                                                                                              0 <br />                                                                                                                                 0. <br />                                                                                                                                 2. <br />                                                                                                                                 4. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           0. <br />                                                                                                                                                                                           2. <br />                                                                                                                                                                                           4. <br />                                                                                                           -8 <br />                                                                                                                    -6 <br />                                                                                                                          -4 <br />                                                                                                                                -2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     -8 <br />                                                                                                                                                                                          -6 <br />                                                                                                                                                                                          -4 <br />                                                                                                                                                                                          -2 <br /> were aligned analysis.                                                                        translational efficiency (log2)                                        translational efficiency (log2) <br />    Nevertheless, 8072 genes (82% annotated CDSs), <br /> we able detect   10 ribosome footprint reads,        C <br />  <br />  <br />  <br />                                                                BF rank translational efficiency <br /> threshold de&#239;&#172;?ned translation (Supplementary <br /> Table S1). We rate translation differ                                                                                              genes, N = 7782 <br />                                                                                                       low <br /> greatly proteins. In PF, observed                                                                                                   PUF (Pumillio), N = 10 <br /> 11 448-fold difference ribosome density (3548-fold dif-                                                                                         cytochrome oxidases, N = 8 <br /> ference mRNA levels) 1% highly                                                                                             enzymes required <br /> the 1% weakly translated proteins, BF                                                                                            glycolysis, N = 10 <br /> we observed 1623-fold difference ribosome density                                                                                             alternative oxidase <br /> and 262-fold difference mRNA levels (Figure 3A). <br />                                                                                                       high                                low <br /> These &#239;&#172;?ndings suggest translational ef&#239;&#172;?ciency signi&#239;&#172;?- <br />                                                                                                        PF rank translational efficiency <br /> cantly contribute regulation gene expression. <br />    To determine translational ef&#239;&#172;?ciency individual <br />                                                                D <br /> genes, normalized differences mRNA abundance,                                                       BF mRNA density                                                   PF mRNA density <br />                                                                mRNA (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                                                                    mRNA (RPM) <br /> i.e. divided ribosome footprint density levels                                          2                                                                     2 <br /> of mRNA abundance. Our ribosome pro&#239;&#172;?ling data                                                         1                                                                     1 <br /> indicate relatively unbiased distribution ribosome                                               0                                                                     0 <br /> density CDSs BF minor increase <br />                                                                                                               BF footprint density                                              PF footprint density <br /> in density 50 end CDSs PF <br />                                                                footprint (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                       4                                                                     4 <br />                                                                                                                                                    footprint (RPM) <br />  <br />  <br />  <br />  <br /> (Supplementary Figure S3). Nevertheless,                                                     2                                                                     2 <br /> determining translational ef&#239;&#172;?ciency excluded <br /> mRNA footprint reads mapped &#239;&#172;?rst 40 nt                                                0                                                                     0 <br /> of CDS avoid possible artifacts ribosome stalling                                                         Tb09.211.1070                                                    Tb09.211.1070 <br /> at translation initiation sites. Between 1% ef&#239;&#172;?-          Chr IX                                     0                  0            0                         Chr IX          0         0         0 <br />                                                                                                            ,00                ,00          ,00                                       ,00       ,00       ,00 <br /> ciently 1% ef&#239;&#172;?ciently translated proteins,                                              59                 61           63                                        59        61        63 <br />                                                                                                       1,4                1,4          1,4                                       1,4       1,4       1,4 <br /> observed PF 117-fold BF 64-fold range <br />                                                                Figure 3. Translational ef&#239;&#172;?ciency regulated. (A) Histograms of <br /> in translational ef&#239;&#172;?ciency, i.e. proteins     mRNA abundance ribosome density (rate protein synthesis) <br /> produced transcript (Figure 3B Supplementary           BF (left panel) PF parasites (right panel). (B) Histogram of <br /> Table S2). This range translational ef&#239;&#172;?ciency similar    translational ef&#239;&#172;?ciency (ratio ribosome footprint density mRNA <br /> to observed yeast roughly 10-fold         abundance) BF (left panel) PF parasites (right panel). (C) Pair- <br />                                                                wise comparisons translational ef&#239;&#172;?ciency PF BF. CDSs were <br /> higher described mice (45,46).          ranked based translational ef&#239;&#172;?ciency (1 = highest translational ef&#239;&#172;?- <br /> Interestingly, observed correlation RNA          ciency, 7782 = lowest translational ef&#239;&#172;?ciency) BF PF. Ranks <br /> abundance translational ef&#239;&#172;?ciency (R2 = 0.03), sug-        life cycle stages correlation R = 0.7428. Gene <br /> gesting translational ef&#239;&#172;?ciency regulated independ-    families developmentally regulated translational ef&#239;&#172;?ciencies are <br /> ently RNA stability.                                        colour coded: Pumillio genes (red), cytochromes oxidase (blue) genes <br />                                                                required glycolysis (63) (green) alternative oxidase <br />                                                                (Tb927.10.7090, black). (D) Ribosome footprint pro&#239;&#172;?le gene with <br /> Life cycle-speci&#239;&#172;?c regulation translational ef&#239;&#172;?ciency       developmentally regulated translational ef&#239;&#172;?ciency. Green arrow indi- <br />                                                                cates direction transcription. <br /> Next addressed question translational ef&#239;&#172;?ciency <br /> is regulated life cycle-speci&#239;&#172;?c manner. Ribosome <br /> density measurements allow meaningful comparisons         cells (46,64); however, speed translation very <br /> of rate protein synthesis speed transla-    identical life cycle stages live at <br /> tion assumed remain constant. This assumption         different temperatures. In addition, current protocol <br /> supported measurements mouse embryonic stem              does allow absolute quanti&#239;&#172;?cation mRNA and <br />                                                                                       Nucleic Acids Research, 2014, Vol. 42, No. 6 3629 <br />  <br />  <br /> ribosome footprint levels different life cycle                   changes described glucose metabolism <br /> stages. Therefore, decided compare overall                        (Figure 3C). Whereas BF entirely dependent the <br /> changes translational ef&#239;&#172;?ciency life cycle                   glycolytic pathway generate energy, PF living the <br /> stage. To end ranked genes life cycle                   insect midgut, glucose availability uncommon or <br /> stage based relative translational ef&#239;&#172;?ciency                   absent, shown contain elaborate <br /> compared rank life cycle stages.                      energy metabolism (66). In contrast, ef&#239;&#172;?cient translation <br /> A pair-wise comparison indicated generally positive cor-                  group PUF (Pumilio FBF) proteins PF was <br /> relation translational ef&#239;&#172;?ciency stages                  unexpected (Figure 3C). PUF proteins regulate translation <br /> (Figure 3C, Pearson&#226;&#8364;&#8482;s correlation coef&#239;&#172;?cient = 0.7428,                      mRNA stability binding sequences target <br /> P &lt; 0.0001, 95% CI 0.7327&#226;&#8364;&#8220;0.7526), revealed                     RNAs shown affect gene expression in <br /> distinct life cycle-speci&#239;&#172;?c differences subsets genes.               organisms divergent Plasmodium falciparum, <br /> For example, 58 genes lowest 25% terms                   T. brucei, humans yeast (67&#226;&#8364;&#8220;69). Interestingly, we <br /> translational ef&#239;&#172;?ciency PF 25%                    poly(A) binding proteins, isoforms of <br /> most ef&#239;&#172;?ciently translated proteins BF. These                        eIF4Es &#239;&#172;?ve eIF4G isoforms, proteins <br /> contain various RNA binding proteins, numerous phos-                        involved control translation initiation, ef&#239;&#172;?- <br /> phatases expression site-associated genes (Table 1                  ciently translated PF BF (Supplementary <br /> Supplementary Table S3). For example gene                           Table S2). <br /> more ef&#239;&#172;?ciently translated BF PF                        Given large dynamic range life cycle-speci&#239;&#172;?c <br /> Figure 3D.                                                                  regulation, propose differences translational <br />    Based mRNA protein level measurements, differ-                    ef&#239;&#172;?ciency contribute substantially control gene <br /> ential translational ef&#239;&#172;?ciency predicted                       expression T. brucei. <br /> aconitase, cytochrome oxidases V VI procyclins <br /> (22,25,65). For genes, data indicate an <br />                                                                             uORFs frequently translated regulate <br /> increased translational ef&#239;&#172;?ciency PF (Table 1), thus <br />                                                                             translational ef&#239;&#172;?ciency <br /> con&#239;&#172;?rming previous observations validating meas- <br /> urements. For selected groups proteins, e.g. annotated                The broad  100-fold range translational ef&#239;&#172;?ciency <br /> cytochrome oxidase required glycolysis,                     genes differences translational <br /> noticed translational ef&#239;&#172;?ciency correspond                    ef&#239;&#172;?ciencies life cycle stages raise question of <br />  <br />  <br /> Table 1. Developmentally regulated translation <br />  <br /> Gene ID                    Description                                                            BF rank         PF rank    Change in <br />                                                                                                                              rank (PF&#226;&#8364;&#8220;BF) <br />  <br /> Translation up-regulated   BF <br />   Tb927.7.3250             Expression site-associated gene 6 (ESAG6) protein, putative            1372            7405         6033 <br />   Tb927.4.3980             Chaperone protein DNAj, putative                                       1301            7248         5947 <br />   Tb927.10.4780            GPI inositol deacylase precursor (GPIdeAc)                              390            5994         5604 <br />   Tb927.8.8140             Small GTP-binding rab protein, putative                                1217            6804         5587 <br />   Tb927.6.3480             RNA-binding protein, putative (DRBD5)                                  1856            7262         5406 <br />   Tb11.01.4701             Membrane-bound acid phosphatase 1 precursor (MBAP1)                     689            6065         5376 <br />   Tb927.1.4650             Cyclin-like F-box protein (CFB2)                                         10            5289         5279 <br />   Tb927.3.5660             UDP-Gal UDP-GlcNAc-dependent glycosyltransferase, putative           797            6041         5244 <br />   Tb927.2.6000             Glycosylphosphatidylinositol-speci&#239;&#172;?c phospholipase C (GPI-PLC)         1634            6793         5159 <br />   Tb927.4.5310             Serine/threonine-protein kinase a, putative                            1656            6776         5120 <br /> Translation up-regulated   PF <br />   Tb927.5.440              trans-Sialidase, putative                                              6785             418        6367 <br />   Tb927.7.6850             trans-Sialidase (TbTS)                                                 7513            1445        6068 <br />   Tb927.7.7470             Receptor-type adenylate cyclase GRESAG 4, putative                     7001            1268        5733 <br />   Tb927.1.2120             Calpain, putative                                                      6258             739        5519 <br />   Tb927.4.360              1,2-Dihydroxy-3-keto-5-methylthiopentene dioxygenase, putative         6587            1089        5498 <br />   Tb09.160.5550            Calpain-like cysteine peptidase, putative                              6936            1868        5068 <br />   Tb927.8.7690             Amino acid transporter (pseudogene), putative                          6693            1731        4962 <br />   Tb927.8.1610             MSP-B, putative                                                        7199            2272        4927 <br />   Tb11.01.6650             Serine/threonine-protein kinase, putative                              5280             363        4917 <br />   Tb927.7.7110             Leucine-rich repeat protein (LRRP), putative                           6501            1601        4900 <br /> Genes translational up-regulation PF previously shown <br />  Tb927.10.280            Cytochrome oxidase subunit VI (COXVI)                                    2820             810        2010 <br />  Tb09.160.1820           Cytochrome oxidase subunit V (COXV)                                       935             291         644 <br />  Tb927.10.14000          Aconitase (ACO)                                                           657              31         626 <br />  Tb927.6.510             GPEET2 procyclin precursor                                               3848             365        3483 <br />  Tb927.10.10260          EP1 procyclin (EP1)                                                      4682              34        4648 <br />  Tb927.10.10220          Procyclin-associated gene 2 (PAG2) protein (PAG2)                        6903            5375        1528 <br />  Tb927.5.330             Receptor-type adenylate cyclase GRESAG 4, putative                       3779             809        2970 <br />  <br /> List genes highest change translational ef&#239;&#172;?ciency PF BF. List does include hypothetical genes. <br />  3630 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br />                                                                  A                                                                               B <br /> how protein translation regulated. For small number                               uORF <br />  <br />  <br />  <br />  <br />                                                                                                                                                  percentage reads <br />                                                                                                                                                                        80 <br />  <br />  <br />  <br />  <br />                                                                  relative enrichment <br /> of genes T. brucei, sequence motifs 30 UTR                                                AUG                       UAA <br />  <br />  <br />  <br />  <br />                                                                    read 5&#194;&#180; ends <br /> been linked developmentally regulated changes trans-                                                                                                             60 <br />                                                                                                                                                                                 uORF <br /> lational ef&#239;&#172;?ciency, large majority genes                                                                                                             40 <br /> such motifs identi&#239;&#172;?ed (70). Numerous reports <br /> from eukaryotes demonstrated uORFs                                                                                                                      20 <br />                                                                                         -24 -18 -12 -6 0 6 -24 -18 -12 -6 0 6 12 <br /> function important regulators gene expression (71).                                                                                                              0 <br />                                                                                       CDS position (nt start) CDS position (nt stop) <br /> For example, analysis mRNA protein levels                                                                                                     1 <br />                                                                                                                                                         frame <br />                                                                                                                                                                2            0 <br /> &gt;10 000 mammalian genes revealed presence                                                                                          ribosome footprint <br /> an uORF correlates signi&#239;&#172;?cant reduction expres-                                                                                      mRNA <br /> sion downstream CDS. Furthermore, using                    C <br />  <br />  <br />  <br />  <br />                                                                     footprint (RPM) <br /> reporter constructs, estimated presence                            0.8 BF                                      BF <br />                                                                                                                           0.8 <br /> an uORF results decrease gene expression 30&#226;&#8364;&#8220;                               0.4                                 0.4 <br /> 80% minor reduction mRNA levels (72). <br /> No analyses performed T. brucei,                            0.0                                 0.0 <br /> has shown removal uATG leads 7-                                     PF                                    PF <br />  <br />  <br />  <br />  <br />                                                                     footprint (RPM) <br /> fold increase protein levels luciferase reporter (41).                        0.4                                   4 <br /> In manuscript, use term ORF uORF refer                              0.2                                   2 <br /> to regions DNA sequences beginning ATG and <br /> ending termination codon                                   0.0                                   0 <br /> encode proteins. The term coding sequence (CDS)                       Chr I <br />                                                                                                         Tb927.1.1610 <br />                                                                                                                           Chr I <br />                                                                                                                                                Tb927.1.1670 <br /> used refer ORFs encode proteins. Minimally,                                      411,000   411,500 412,000                  420,000 420,200 420,400 <br /> uORFs consist 9 nt, containing upstream start codon        Figure 4. Ribosome footprints reveal translation uORFs. <br /> (uATG), additional sense codon termination              (A) Alignment 50 nucleotides ribosome footprint reads <br /> codon, termination codon located              map close translation start translation termination sites of <br />                                                                  uORFs. (B) Percentage position sequence reads relative reading <br /> downstream start codon main CDS (73).              frame. (C) Ribosome pro&#239;&#172;?les genes uORF (left panel) and <br />    Even 50 UTRs assigned 4909          uORF (right panel). Narrow grey boxes represent 50 UTRs, <br /> genes (927 version 4.2), applying criteria listed above,     green box represents AUG-codon, red box represents termination <br /> we identi&#239;&#172;?ed 8310 uORFs 1092 (22%) 50               codon. <br /> UTRs contain uORF (Supplementary <br /> Table S4). Thus, uORF common in <br /> T. brucei mammals 40&#226;&#8364;&#8220;50% genes                  observed higher mRNA level genes <br /> contain uORFs (74,75). Nevertheless, uORFs                 uORFs compared genes uORFs <br /> cannot explain translational regulation tran-        (P &lt; 0.0001), differences mRNA levels lower <br /> scripts, represent factors import-       differences ribosome density (Table 2). Thus, <br /> ant regulation gene expression T. brucei.          data indicate median translational ef&#239;&#172;?ciency <br />    To determine degree uORFs translated,         higher genes uORF genes with <br /> a prerequisite affecting translational ef&#239;&#172;?ciency      uORF, suggesting presence uORFs nega- <br /> downstream CDS, calculated ribosome density               tively impact translation downstream CDSs. <br /> across uORFs identi&#239;&#172;?ed 1834 uORFs (22%)                    Noteworthy, median ribosome density CDSs <br />   2  read-coverage  70% ORF covered.                  higher BF PF (Table 2), observed <br /> The average length uORFs 22 aa. Just like           opposite 50 UTRs (P &lt; 0.0001). For 50 UTRs we <br /> for annotated CDSs, alignment ribosome pro&#239;&#172;?ling           higher ribosome density PF (median rpkm: <br /> reads uORFs revealed distinct 3-nt period-        7.73) BF (median rpkm: 2.89, Supplementary <br /> icity, 63% BF ribosome pro&#239;&#172;?ling reads starting        Table S4, example Figure 4C). In addition, we <br /> the &#239;&#172;?rst nucleotide codon (Figure 4A B). While          BF 50 UTRs uORF (4.08 rpkm) <br /> the periodicity ribosome footprints uORFs          higher ribosome density 50 UTRs without <br /> slightly pronounced annotated CDS,             uORFs (2.54 rpkm; P &lt; 0.0001). Unexpectedly, the <br /> data suggest large proportion uORFs                 contrary true PF. In PF 50 UTRs <br /> translated.                                                      uORF (8.34 rpkm) higher ribosome <br />    To evaluate regulatory potential uORFs,             density 50 UTRs uORF (5.01 rpkm; <br /> compared ribosome densities transcripts con-         P &lt; 0.0001). For example non-AUG uORF see <br /> taining uORF (N = 1092)           Figure 4C, right panel. Many non-AUG uORFs <br /> uORFs (N = 3817). In BF, median ribosome density             observed yeast mice embryonic stem cells <br /> across CDSs transcripts uORFs                    (mESCs). Interestingly, increase 50 UTR translation, <br /> 30.31 rpkm compared 44.86 rpkm genes            similar observed PF compared the <br /> uORFs (Table 2). In PF, median ribosome densities            BF, observed yeast starvation decrease <br /> were 20.01 rpkm (genes uORF) 36.50 rpkm                 50 UTR translation observed differentiation of <br /> (genes uORF). Thus, life cycle stages         pluripotent mESCs embryoid bodies (45,46). <br /> observed higher ribosome density genes                Genome-wide analyses prove disprove direct <br /> uORFs genes uORFs (P &lt; 0.0001). While              correlation translation uORFs and <br />                                                                                      Nucleic Acids Research, 2014, Vol. 42, No. 6 3631 <br />  <br />  <br /> Table 2. Translational ef&#239;&#172;?ciency genes uORF <br />  <br />                                                                    Transcripts            Transcripts          Mann&#226;&#8364;&#8220;Whitney test <br />                                                                    uORF (N = 3817)             uORF (N = 1092) <br />  <br /> Bloodstream form <br />   Ribosome density (median rpkm)                                   30.31                       44.66                        P &lt; 0.0001 <br />   mRNA levels (median rpkm)                                        30.83                       36.13                        P &lt; 0.0001 <br />   Translational ef&#239;&#172;?ciency (ribosome density/mRNA levels)            1.00                        1.27                        P &lt; 0.0001 <br /> Procyclic form <br />   Ribosome density (median rpkm)                                   20.01                       36.50                        P &lt; 0.0001 <br />   mRNA levels (median rpkm)                                        24.04                       30.38                        P &lt; 0.0001 <br />   Translational ef&#239;&#172;?ciency (ribosome density/mRNA levels)            0.87                        1.26                        P &lt; 0.0001 <br />  <br /> Reads mapping &#239;&#172;?rst 40 nt CDS considered measurements translational ef&#239;&#172;?ciency. Number genes annotated <br /> 50 UTR, N = 4909. <br />  <br />  <br />  <br />  <br /> translational ef&#239;&#172;?ciency complicated existence                 chromosome 10 assembly genomic coordinates <br /> multiple 50 UTR isoforms, resulting widespread                        incompatible.). <br /> differential trans-splicing (35,36,76). At time                      To determine set candidate CDSs is <br /> the heterogeneity UTR lengths raises intriguing                     translated, analysed published proteomics datasets <br /> possibility inclusion exclusion uORF                         (57) search protein products. For 24 2021 <br /> during alternative trans-splicing represent regula-                  candidate CDSs (average size 117 aa, 13 kDa), protein <br /> tory mechanism modulate translational ef&#239;&#172;?ciency.                        products identi&#239;&#172;?ed; however, these <br />                                                                            identi&#239;&#172;?ed peptides matched annotated CDSs. In <br /> Ribosome footprints reveal presence hundreds                     addition, identi&#239;&#172;?ed protein products 31 uORFs <br /> previously un-annotated CDSs                                               (Supplementary Table S4). We suspect low <br />                                                                            number identi&#239;&#172;?ed protein products relative the <br /> Even small proteins (&lt;200 aa) shown                    large number candidate CDSs caused, (i) by <br /> play major roles plant animal development (77,78),                  experimental set-up original proteomics <br /> the complexity short proteome remains largely                       analysis proteins separated 1D-SDS- <br /> unexplored algorithms reliably predict short                    PAGE prior in-gel protease digest mass spectro- <br /> CDSs lacking (79). We observed                       metric analysis proteins size smaller than <br /> all ribosome footprints aligned annotated features,                  5&#226;&#8364;&#8220;10 kDa typically run gel, (ii) limited <br /> the BF 101 197 reads (0.23%) PF 1 116 030                       sensitivity mass spectrometric analyses compared with <br /> reads (1.44%) did not. It noteworthy that, just                   DNA sequencing-based techniques (iii) fact <br /> observed 50 UTRs, percentage reads aligning                 ribosome-protected RNAs translated. <br /> to CDSs higher PF BF. We suspect                        The &#239;&#172;?rst point supported fact 67 <br /> that reads originate un-annotated                       (3%) candidate CDSs  90 aa ( 10 kDa) but <br /> UTRs stem previously un-annotated                        28% large candidate CDSs identify <br /> CDSs. A previous RNA-sequencing-based analysis                             peptides. Evidence supporting possibility all <br /> detected 1114 new, un-annotated transcripts PF,                     ribosome-protected RNAs translated exists mice <br /> 1011 potential encode                      known long ncRNAs be <br /> peptides (  25 aa). Using available proteomics data,                   associated ribosomes (46). To evaluate coding <br /> authors able con&#239;&#172;?rm translation 19 1114                 potential transcript separate long ncRNA <br /> transcripts (13). Given ability determine ribosome                  small coding genes, new metric established, <br /> positions nucleotide resolution, looked new, pre-                so-called RRS (53). The RRS takes advantage the <br /> viously un-annotated CDSs.                                                 fact translating ribosomes released encounter- <br />    We mapped ribosome footprint reads BF                      ing stop codon. This release results sharp decrease in <br /> PF determined ribosome footprint density                   ribosome occupancy protein-coding regions and <br /> potential CDSs 10 aa length. To avoid overlap                  subsequent 30 UTR. Consequently, RRS was <br /> with annotated CDSs, considered ORFs located                       de&#239;&#172;?ned ratio footprint reads putative <br /> at 20 nt away annotated features. This                          CDS footprint reads corresponding 30 UTR <br /> approach enabled identify set 2021 candidate                    divided ratio RNA reads putative CDS <br /> CDSs  2  read-coverage  70% ORF                            RNA reads corresponding 30 UTR (see <br /> covered. The candidate CDSs ranged length 10                       &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section). <br /> to 378 aa (average 30 aa, Supplementary Table S5). For                        Similarly annotated CDSs uORFs, averaged <br /> 797 1114 previously identi&#239;&#172;?ed transcripts,                 ribosome footprint data indicated well-de&#239;&#172;?ned 3-nt peri- <br /> the average ribosome footprint read-coverage  2                      odicity sequence reads occurring near translation <br /> (Supplementary Table S6. Previously identi&#239;&#172;?ed transcripts                  initiation sites. However, observed well-de&#239;&#172;?ned <br /> from chromosome 10 considered                         drop ribosome densities translation termination <br /> sequencing reads mapped newer version                     sites (Figure 5A B), indicate lack of <br />  3632 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br /> A                                                                                  B <br />                       putative CDS <br />                                                                                                          80 <br />                                                                                                                             1445 genes annotated T. brucei (Supplementary <br /> relative enrichment <br />  <br />  <br />  <br />  <br />                                         AUG                            UAA <br />  <br />  <br />  <br />  <br />                                                                                    percentage reads <br />                                                                                                                             Table S7). For 1445 genes median RRS was <br />   read 5&#194;&#180; ends <br />  <br />  <br />  <br />  <br />                                                                                                          60                 83.9 compared RRS 0.19 subset of <br />                                                                                                          40 <br />                                                                                                                             genes annotated hypothetical unlikely (N = 90). Thus, <br />                                                                                                                             similar observation mice, RRS appears be <br />                                                                                                          20                 good predictor productive translation. Next we <br />                       -24 -18 -12 -6   0    6 12      -24 -18 -12 -6   0    6 <br />                                                                                                          0                  determined RRS previously un-annotated can- <br />                       CDS position (nt start) CDS position (nt stop)                                0     1 <br />                                                                                                                   frame <br />                                                                                                                         2   didate CDSs ful&#239;&#172;?lled criteria RRS calculation <br />                                                                                                                             (N = 265). The median RRS candidate CDSs was <br />                                                                                 ribosome footprint <br />                                                                                 mRNA                                        3.99 lower median RRS annotated <br /> C                                                                                                                           genes (83.9). Thirteen candidate CDSs RRS larger <br />  (RPM) <br />                                                                                                                             83.9 98 candidate CDSs RRS higher than <br />                           BF mRNA density                        BF mRNA density                                            well-described intron-containing poly(A) polymerase <br />                       4 <br />                       3                                      8                                                              (RRS = 9.63). The RRS calculated for <br />   mRNA <br />  <br />  <br />  <br />  <br />                       2                                      4                                                              candidate CDSs identi&#239;&#172;?ed <br />                       1                                                                                                     protein products (RRS = 57.44). <br />                       0                                      0 <br />                                                                                                                                Finally, attempt learn function the <br />                2.0 BF footprint density                      8   BF footprint density                                       putative CDSs, searched known protein motifs <br />   footprint <br />  <br />  <br />  <br />  <br />                1.5                                           6 <br />                1.0                                           4 <br />                                                                                                                             candidate CDSs identi&#239;&#172;?ed <br />                0.5                                           2                                                              protein products RRS 10. <br />                0.0                                           0                                                              However, candidate CDS located within <br />                20 BF RNAi induction                       8   BF RNAi induction                                       histone H2B gene array contained an <br />   RNAi reads <br />  <br />  <br />  <br />  <br />                15                                            6                                                              H2B signature motif (Supplementary Table S5), no <br />                10                                            4                                                              motifs identi&#239;&#172;?ed. <br />                 5                                            2 <br />                 0                                            0 <br />                                                                                                                                Taken together, proteomics data well-de&#239;&#172;?ned <br />                                                                                                                             ribosome release (high RRS) suggest extensive translation <br />                           BF 6 days RNAi induction        4   BF 6 days RNAi induction                                annotated CDSs T. brucei. <br />   RNAi reads <br />  <br />  <br />  <br />  <br />                       8                                      3 <br />                       4                                      2 <br />                                                              1                                                              Newly identi&#239;&#172;?ed CDSs important parasite &#239;&#172;?tness <br />                       0                                      0 <br />                                                                                                                             The combination ribosome pro&#239;&#172;?ling data previ- <br />                                        putative CDS                        putative CDS <br />                  Chr III                                     Chr X                  0           0                           ously published mass spectrometric data allowed to <br />                                 00          00          00                       ,80         ,20                            identify large number new CDSs. Nonetheless, bio- <br />                              2,0         2,4         2,8                      77           78 <br />                            62          62          62                      2,5          2,5 <br />                                                                                                                             logical signi&#239;&#172;?cance CDSs remains determined. <br /> Figure 5. Ribosome footprints reveal previously un-annotated CDSs.                                                          In addition, ORFs high ribosome footprint <br /> (A) Alignment 50 nucleotides ribosome footprint reads that <br /> map close translation start translation termination sites candi- <br />                                                                                                                             density, low RRS mass spectrometric <br /> date CDSs. (B) Percentage position sequence reads relative                                                         evidence translated functional proteins, <br /> reading frame. (C) Ribosome, mRNA RIT-seq (RNAi target                                                                  may, nevertheless, play important regulatory roles. <br /> sequencing) pro&#239;&#172;?les previously un-annotated putative CDSs.                                                              Therefore, decided evaluate importance our <br />                                                                                                                             candidate CDSs, independent RRS mass spectro- <br />                                                                                                                             metric evidence, parasite survival. A previously pub- <br /> productive translation. Because observed abrupt                                                                       lished high-throughput phenotyping approach, termed <br /> drop footprint density downstream stop codon                                                                      RIT-seq, measured &#239;&#172;?tness&#226;&#8364;&#8220;cost associations using <br /> of annotated genes T. brucei (Figure 1A B),                                                                       RNA interference (RNAi). The RIT-seq data revealed a <br /> decided apply parameters evaluate like-                                                                  signi&#239;&#172;?cant loss &#239;&#172;?tness RNAi induction 2724 CDSs <br /> lihood ORFs high footprint density                                                                     BF, 1972 CDSs PF 2677 CDSs induced <br /> being translated functional proteins. Again, candidate                                                                 differentiated parasites (54). The approach based on <br /> CDSs de&#239;&#172;?ned ORFs minimum length                                                                           integration RNAi libraries trypanosomes a <br /> 10 aa average ribosome footprint coverage 2                                                                       comparison recovery RNAi targets tryp- <br /> over 70% ORF. To de&#239;&#172;?ne correspond-                                                                      anosome populations RNAi induction. <br /> ing putative 30 UTR, applied parameters used                                                                    The RNAi library generated using genomic DNA, <br /> in mice (53), i.e. 30 UTR de&#239;&#172;?ned region                                                                      effect parasite &#239;&#172;?tness determined for <br /> beginning immediately downstream ORF                                                                             annotated genes. Re-analysing datasets, &#239;&#172;?nd <br /> ending &#239;&#172;?rst subsequent start codon (in                                                                           signi&#239;&#172;?cant loss &#239;&#172;?tness RNAi induction 214 <br /> reading frame). A limitation RRS                                                                      putative CDSs BF (6 days RNAi induction), <br /> only calculated ORF entire foot-                                                                  16 CDSs PF 227 CDSs differentiation <br /> print read RNA read map putative CDS                                                                         (Supplementary Table S8). For examples candidate <br /> and corresponding 30 UTR. Because 30 UTRs                                                                          CDSs potentially essential viability BF see <br /> were short ful&#239;&#172;?ll criterion simply did                                                                   Figure 5C. It important note RNAi library <br /> contain footprint reads, sign ef&#239;&#172;?cient translation                                                                used RIT-seq experiments contains median insert <br /> termination, able determine RRS                                                                     size 600 bp (average: 1 kb), somewhat limiting the <br />                                                                            Nucleic Acids Research, 2014, Vol. 42, No. 6 3633 <br />  <br />  <br /> resolution RIT-seq data (80,81). Nevertheless,       regulation translation. Importantly, regulation we <br /> &#239;&#172;?ndings suggest important biological role subset         observe agrees reported previous studies <br /> of genes missed previous genome anno-           small subset genes includes proteins <br /> tations. Intriguingly, RNAi data reveal large develop-        previously unknown developmentally regulated. In <br /> mental differences &#239;&#172;?tness costs associated        addition, ef&#239;&#172;?cient translation proteins annotated as <br /> down-regulation putative CDSs, suggesting set         cytochrome oxidases PF enzymes important <br /> of candidate CDSs important                glycolysis BF consistent differences in <br /> mammalian stage.                                                  energy metabolism BF entirely dependent gly- <br />                                                                   colysis generate energy. <br />                                                                      How translational control achieved T. brucei <br /> DISCUSSION                                                        remains seen, analysis suggests uORF <br />                                                                   important contributor. Work organ- <br /> In study, report &#239;&#172;?rst genome-wide analysis         isms shown generally presence uORFs cor- <br /> protein synthesis strand-speci&#239;&#172;?c analysis RNA            relates reduced gene expression (71). However, for <br /> transcript levels T. brucei. It &#239;&#172;?rst analysis     genes, like transcription factor GCN4 yeast <br /> for eukaryotic pathogen organism tran-           activating transcription factor ATF4 mammals, <br /> scriptional control. Our data reveal large differences         correlation reversed stress (83,84). Thus, uORF <br /> translational ef&#239;&#172;?ciency transcripts life        exert positive negative effects translation of <br /> cycle stage transcript different          downstream CDSs. Based previous 50 UTR annota- <br /> stages. In addition, sequencing ribosome-protected         tions, identi&#239;&#172;?ed uORFs 22% T. brucei genes, <br /> RNA enabled identify transcripts likely         interesting learn regulatory effects <br /> be translated. Hundreds putative, previously unidenti-         uORFs supplemented translation non- <br /> &#239;&#172;?ed CDSs appear essential parasite &#239;&#172;?tness,          AUG uORFs. Our ribosome pro&#239;&#172;?ling data indicate an <br /> the analysis available proteomics data con&#239;&#172;?rmed            increase translation 50 UTRs ubiqui- <br /> existence 20 previously unknown CDSs.        tous translation initiation non-AUG sites the <br />    Eukaryotic gene expression regulated multiple            PF compared BF. Similarly, increase the <br /> levels, translation mRNA proteins              translation 50 UTRs observed yeast on <br /> one important (1,7). The ability determine         starvation (45). One highly conserved stress- <br /> both translatome transcriptome enabled                  induced mechanisms regulate translation involves the <br /> evaluate ef&#239;&#172;?ciency individual transcripts          inactivation translation initiation factor eIF2a <br /> are translated proteins suggests translational           phosphorylation conserved serine (85,86). <br /> control important regulatory mechanism                Phosphorylation T. brucei L. major orthologs <br /> T. brucei. Under single condition, observed transla-         occurs corresponding Thr residues (6,87), the <br /> tional ef&#239;&#172;?ciency vary orders magnitude;            generation T. brucei cell line exclusively expressing a <br /> thus, importance equals RNA stability             mutant form eIF2a demonstrated phosphorylation <br /> which similar range measured (82). However,            eIF2a required heat-induced translational <br /> we observed correlation RNA abundance              arrest formation heat shock stress granules <br /> translational ef&#239;&#172;?ciency, suggesting translational ef&#239;&#172;?-       T. brucei (88). Given &#239;&#172;?nding eIF2a be <br /> ciency regulated independently RNA stability.               important determining stringency initiator codon <br />    To use ribosome density estimate rate protein        selection (89), interesting eIF2a <br /> synthesis, speed translation constant. This        phosphorylation affects selection initiator codons <br /> assumption supported measurements mouse em-              plays role &#239;&#172;?ne-tuning gene <br /> bryonic stem cells demonstrate rate translation       expression. <br /> to consistent different classes mRNA,              Besides valuable tool study translational <br /> kinetics elongation independent length            regulation, ribosome pro&#239;&#172;?ling invaluable for <br /> protein abundance speed translation in-          discovery small peptides. While  2% the <br /> dependent codon usage (46,64). Nevertheless, speed         human genome contains annotated CDSs (90), transcrip- <br /> of translation different         tome analyses revealed genome-wide tran- <br /> life cycle stages parasite live 37  C 27  C.   scription RNA, considered non- <br /> Furthermore, approach does permit measurement             protein-coding. In lower organisms percentage the <br /> of absolute mRNA footprint abundance              genome encoding proteins higher (62,91,92), but <br /> individual life cycle stages, making direct comparisons        pervasive transcription non-protein coding regions <br /> translation ef&#239;&#172;?ciencies individual transcripts unreli-        essentially organisms (93). The vast <br /> able. Therefore, did compare translational ef&#239;&#172;?-            ncRNA triggered great in <br /> ciency directly, determined &#226;&#8364;&#732;rank&#226;&#8364;&#8482;              elucidating biological signi&#239;&#172;?cance transcripts <br /> translational ef&#239;&#172;?ciency genes life cycle          led identi&#239;&#172;?cation numerous regulatory <br /> stage, assuming &#226;&#8364;&#732;change rank&#226;&#8364;&#8482; indicated life            mechanisms controlled short long ncRNA. <br /> cycle-dependent translational control. While             Interestingly, majority transcription occurs as <br /> general positive correlation translational ef&#239;&#172;?ciency           long ncRNA humans (94), contain <br /> between life cycle stages (Pearson r = 0.7428),           ORFs making dif&#239;&#172;?cult unambiguously classify tran- <br /> for large number genes, observed developmental            script non-protein-coding. To distinguish long ncRNA <br />  3634 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br /> from mRNA, ORF length served                  SUPPLEMENTARY DATA <br /> commonly applied criterion, select-      Supplementary Data available NAR Online. <br /> ive pressure, short ORFs occur frequently chance <br /> while probability long ORFs occur chance are <br /> low. Nevertheless, recent &#239;&#172;?ndings reveal eukaryotic      ACKNOWLEDGEMENTS <br /> genomes contain large number small CDSs (95&#226;&#8364;&#8220;97) <br /> and small peptides play important biological roles       We thank Stan Gorski, Susanne Kramer, Christian Janzen <br /> (77,78,98,99). In addition, numerous long ncRNAs con-         Jose-Juan Lopez-Rubio valuable discussions and <br /> taining ORFs longer 100 aa as-        critical reading manuscript Yanjie Chao and <br /> sociate ribosomes (46,53). While &#239;&#172;?ndings           Jan Medenbach appreciated advice on <br /> complicate annotation CDSs demand new          generating ribosome pro&#239;&#172;?ling libraries. <br /> approaches annotate genes, raise exciting pos- <br /> sibility large number unknown small CDSs exists     FUNDING <br /> and awaits discovery. <br />    For study, RNA obtained 427 strain,        Young Investigator Program Research Center of <br /> the common laboratory strain, sequence reads         Infectious Diseases (ZINF) University of <br /> were aligned complete genome 927           Wuerzburg, Germany; German Research Foundation <br /> strain; nevertheless, RNA-sequencing analysis indicates       DFG [SI 1610/2-1]; Human Frontier Science Program <br /> widespread transcription. Thus, seen organ-       (to T.N.S.); French National Research Agency [ANR- <br /> isms, active transcription necessarily good pre-     2010-GENM-011-01, GENAMIBE C.C.H.]. Funding <br /> dictor CDSs. In contrast, ribosome                open access charge: German Research Foundation <br /> footprint density highly enriched annotated      (DFG) University Wuerzburg funding <br /> CDSs, excluding introns. It serve reliable   programme Open Access Publishing. <br /> tool identify novel genes. To search new CDSs and <br />                                                               Con&#239;&#172;&#8218;ict statement. None declared. <br /> to evaluate biological function, combined <br /> ribosome pro&#239;&#172;?ling, proteomics genome-wide RNAi <br /> data. While proteomics data allowed identify            REFERENCES <br /> protein products 20 relatively large candidate <br /> CDSs, experimental set-up proteomic analysis        1. Moore,M.J. (2005) From birth death: complex lives of <br />                                                                   eukaryotic mRNAs. Science, 309, 1514&#226;&#8364;&#8220;1518. <br /> that involved removal small peptides            2. Nagaraj,N., Wisniewski,J.R., Geiger,T., Cox,J., Kircher,M., <br /> impossible verify existence small CDSs.            Kelso,J., Paabo,S. Mann,M. (2011) Deep proteome and <br /> Thus, important veri&#239;&#172;?cation putative             transcriptome mapping human cancer cell line. Mol. Syst. <br /> CDSs proteomics data &#239;&#172;?nding                  Biol., 7, 548. <br /> &gt;200 putative CDSs appear essential           3. Lu,P., Vogel,C., Wang,R., Yao,X. Marcotte,E.M. (2007) <br />                                                                   Absolute protein expression pro&#239;&#172;?ling estimates relative <br /> parasite &#239;&#172;?tness. Furthermore, RIT-seq data suggest           contributions transcriptional translational regulation. <br /> candidate CDSs 13-fold likely essential            Nat. Biotechnol., 25, 117&#226;&#8364;&#8220;124. <br /> for viability PF BF differen-     4. Godoy,L.M., Olsen,J.V., Cox,J., Nielsen,M.L., Hubner,N.C., <br /> tiation cells BF PF. These &#239;&#172;?ndings point          Frohlich,F., Walther,T.C. Mann,M. (2008) Comprehensive <br />                                                                   mass-spectrometry-based proteome quanti&#239;&#172;?cation haploid <br /> possible role small peptides adaptation tryp-        versus diploid yeast. Nature, 455, 1251&#226;&#8364;&#8220;1254. <br /> anosomes life mammalian host. Generally,         5. McNicoll,F., Drummelsmith,J., Muller,M., Madore,E., Boilard,N., <br /> data suggest existence large explored          Ouellette,M. Papadopoulou,B. (2006) A combined <br /> small proteome, characterizations              proteomic transcriptomic approach study of <br />                                                                   stage differentiation Leishmania infantum. Proteomics, 6, <br /> small proteome include targeted               3567&#226;&#8364;&#8220;3581. <br /> proteomics analysis.                                           6. Lahav,T., Sivam,D., Volpin,H., Ronen,M., Tsigankov,P., <br />    In analysis, focused measuring translational        Green,A., Holland,N., Kuzyk,M., Borchers,C., Zilberstein,D. <br /> control identi&#239;&#172;?cation new CDSs. However,               et al. (2011) Multiple levels gene regulation mediate <br /> ribosome pro&#239;&#172;?ling data aid identi&#239;&#172;?ca-          differentiation intracellular pathogen Leishmania. <br />                                                                   FASEB J., 25, 515&#226;&#8364;&#8220;525. <br /> tion bona &#239;&#172;?de ncRNAs. For annotated CDSs            7. Schwanhausser,B., Busse,D., Li,N., Dittmar,G., Schuchhardt,J., <br /> observed RNA transcripts ribosome footprints,             Wolf,J., Chen,W. Selbach,M. (2011) Global quanti&#239;&#172;?cation <br /> suggesting presence ncRNA. Though, given              mammalian gene expression control. Nature, 473, 337&#226;&#8364;&#8220;342. <br /> we analysed life cycle stages,             8. Fevre,E.M., Wissmann,B.V., Welburn,S.C. Lutumba,P. (2008) <br />                                                                   The burden human African trypanosomiasis. PLoS Negl. Trop. <br /> putative ncRNAs translated stages             Dis., 2, e333. <br /> of parasite&#226;&#8364;&#8482;s life cycle. Thus, comprehensive search     9. Lozano,R., Naghavi,M., Foreman,K., Lim,S., Shibuya,K., <br /> for ncRNA include ribosome pro&#239;&#172;?ling analyses            Aboyans,V., Abraham,J., Adair,T., Aggarwal,R., Ahn,S.Y. et al. <br /> all life cycle stages.                                            (2012) Global regional mortality 235 causes of <br />                                                                   death 20 age groups 1990 2010: systematic analysis <br />                                                                   Global Burden Disease Study 2010. Lancet, 380, <br />                                                                   2095&#226;&#8364;&#8220;2128. <br /> ACCESSION NUMBERS                                             10. Martinez-Calvillo,S., Yan,S., Nguyen,D., Fox,M., Stuart,K. and <br />                                                                   Myler,P.J. (2003) Transcription Leishmania major Friedlin <br /> All sequencing data deposited European           chromosome 1 initiates directions single region. <br /> Nucleotide Archive: PRJEB4801.                                    Mol. Cell., 11, 1291&#226;&#8364;&#8220;1299. <br />                                                                                     Nucleic Acids Research, 2014, Vol. 42, No. 6 3635 <br />  <br />  <br /> 11. Martinez-Calvillo,S., Nguyen,D., Stuart,K. Myler,P.J. (2004)     31. Yoffe,Y., Zuberek,J., Lerer,A., Lewdorowicz,M., Stepinski,J., <br />     Transcription initiation termination Leishmania major             Altmann,M., Darzynkiewicz,E. Shapira,M. (2006) Binding <br />     chromosome 3. Eukaryot. Cell, 3, 506&#226;&#8364;&#8220;517.                                speci&#239;&#172;?cities potential roles isoforms eukaryotic initiation <br /> 12. Siegel,T.N., Hekstra,D.R., Kemp,L.E., Figueiredo,L.M.,                   factor 4E Leishmania. Eukaryot. Cell, 5, 1969&#226;&#8364;&#8220;1979. <br />     Lowell,J.E., Fenyo,D., Wang,X., Dewell,S. Cross,G.A.M.           32. De Gaudenzi,J., Frasch,A.C. Clayton,C. (2005) RNA-binding <br />     (2009) Four histone variants mark boundaries polycistronic        domain proteins Kinetoplastids: comparative analysis. <br />     transcription units Trypanosoma brucei. Genes Dev., 23,               Eukaryot. Cell, 4, 2106&#226;&#8364;&#8220;2114. <br />     1063&#226;&#8364;&#8220;1076.                                                           33. Dhalia,R., Reis,C.R., Freire,E.R., Rocha,P.O., Katz,R., <br /> 13. Kolev,N.G., Franklin,J.B., Carmi,S., Shi,H., Michaeli,S.             Muniz,J.R., Standart,N. Melo Neto,O.P. (2005) <br />     Tschudi,C. (2010) The transcriptome human pathogen                Translation initiation Leishmania major: characterisation of <br />     Trypanosoma brucei single-nucleotide resolution.                      multiple eIF4F subunit homologues. Mol. Biochem. Parasitol., <br />     PLoS Pathog., 6, e1001090.                                               140, 23&#226;&#8364;&#8220;41. <br /> 14. LeBowitz,J.H., Smith,H.Q., Rusche,L. Beverley,S.M. (1993)        34. Zinoviev,A. Shapira,M. (2012) Evolutionary conservation and <br />     Coupling poly(A) site selection trans-splicing                 diversi&#239;&#172;?cation translation initiation apparatus in <br />     Leishmania. Genes Dev., 7, 996&#226;&#8364;&#8220;1007.                                     trypanosomatids. Comp. Funct. Genomics, 2012, 813718. <br /> 15. Ullu,E., Matthews,K.R. Tschudi,C. (1993) Temporal order       35. Siegel,T.N., Hekstra,D.R., Wang,X., Dewell,S. Cross,G.A.M. <br />     RNA-processing reactions trypanosomes: rapid trans splicing           (2010) Genome-wide analysis mRNA abundance life- <br />     precedes polyadenylation newly synthesized tubulin transcripts.       cycle stages Trypanosoma brucei identi&#239;&#172;?cation splicing <br />     Mol. Cell. Biol., 13, 720&#226;&#8364;&#8220;725.                                           polyadenylation sites. Nucleic Acids Res., 38, 4946&#226;&#8364;&#8220;4957. <br /> 16. Matthews,K.R., Tschudi,C. Ullu,E. (1994) A common                36. Nilsson,D., Gunasekera,K., Mani,J., Osteras,M., Farinelli,L., <br />     pyrimidine-rich motif governs trans-splicing polyadenylation         Baerlocher,L., Roditi,I. Ochsenreiter,T. (2010) Spliced leader <br />     tubulin polycistronic pre-mRNA trypanosomes. Genes Dev.,           trapping reveals widespread alternative splicing patterns the <br />     8, 491&#226;&#8364;&#8220;501.                                                              highly dynamic transcriptome Trypanosoma brucei. PLoS <br /> 17. Wright,J.R., Siegel,T.N. Cross,G.A.M. (2010) Histone H3              Pathog., 6, e1001037. <br />     trimethylated lysine 4 enriched probable transcription      37. Siegel,T.N., Gunasekera,K., Cross,G.A. Ochsenreiter,T. <br />     start sites Trypanosoma brucei. Mol. Biochem. Parasitol., 136,        (2011) Gene expression Trypanosoma brucei: lessons from <br />     434&#226;&#8364;&#8220;450.                                                                 high-throughput RNA sequencing. Trends Parasitol., 27, 434&#226;&#8364;&#8220;441. <br /> 18. Clayton,C. Shapira,M. (2007) Post-transcriptional regulation     38. Jackson,R.J., Hellen,C.U. Pestova,T.V. (2010) The <br />     gene expression trypanosomes leishmanias. Mol.                 mechanism eukaryotic translation initiation principles of <br />     Biochem. Parasitol., 156, 93&#226;&#8364;&#8220;101.                                        regulation. Nat. Rev. Mol. Cell Biol., 11, 113&#226;&#8364;&#8220;127. <br /> 19. Matthews,K.R. (2011) Controlling coordinating development        39. Kozak,M. (1984) Selection initiation sites eucaryotic <br />     vector-transmitted parasites. Science, 331, 1149&#226;&#8364;&#8220;1153.                ribosomes: effect inserting AUG triplets upstream the <br /> 20. Cross,G.A., Klein,R.A. Linstead,D.J. (1975) Utilization           coding sequence preproinsulin. Nucleic Acids Res., 12, <br />     amino acids Trypanosoma brucei culture: L-threonine           3873&#226;&#8364;&#8220;3893. <br />     precursor acetate. Parasitology, 71, 311&#226;&#8364;&#8220;326.                    40. Somers,J., Poyry,T. Willis,A.E. (2013) A perspective on <br /> 21. Brun,R. Schonenberger,M. (1979) Cultivation vitro             mammalian upstream open reading frame function. Int. J. <br />     cloning procyclic culture forms Trypanosoma brucei            Biochem. Cell Biol., 45, 1690&#226;&#8364;&#8220;1700. <br />     semi-de&#239;&#172;?ned medium. Acta Trop., 36, 289&#226;&#8364;&#8220;292.                         41. Siegel,T.N., Tan,K.S. Cross,G.A.M. (2005) Systematic study <br /> 22. Furger,A., Schurch,N., Kurath,U. Roditi,I. (1997) Elements           sequence motifs RNA trans splicing Trypanosoma <br />     30 untranslated region procyclin mRNA regulate                 brucei. Mol. Cell. Biol., 25, 9586&#226;&#8364;&#8220;9594. <br />     expression insect forms Trypanosoma brucei                  42. Arava,Y., Wang,Y., Storey,J.D., Liu,C.L., Brown,P.O. and <br />     modulating RNA stability translation. Mol. Cell. Biol., 17,          Herschlag,D. (2003) Genome-wide analysis mRNA translation <br />     4372&#226;&#8364;&#8220;4380.                                                               pro&#239;&#172;?les Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA, <br /> 23. Hehl,A., Vassella,E., Braun,R. Roditi,I. (1994) A conserved          100, 3889&#226;&#8364;&#8220;3894. <br />     stem-loop structure 30 untranslated region procyclin       43. Capewell,P., Monk,S., Ivens,A., Macgregor,P., Fenn,K., <br />     mRNAs regulates expression Trypanosoma brucei. Proc. Natl             Walrad,P., Bringaud,F., Smith,T.K. Matthews,K.R. (2013) <br />     Acad. Sci. USA, 91, 370&#226;&#8364;&#8220;374.                                             Regulation trypanosoma brucei total polysomal mRNA <br /> 24. Hotz,H.R., Hartmann,C., Huober,K., Hug,M. Clayton,C.                 development mammalian host. PLoS One, 8, <br />     (1997) Mechanisms developmental regulation Trypanosoma             e67069. <br />     brucei: polypyrimidine tract 30 -untranslated region   44. Brecht,M. Parsons,M. (1998) Changes polysome pro&#239;&#172;?les <br />     surface protein mRNA affects RNA abundance translation.              accompany trypanosome development. Mol. Biochem. Parasitol., <br />     Nucleic Acids Res., 25, 3017&#226;&#8364;&#8220;3026.                                       97, 189&#226;&#8364;&#8220;198. <br /> 25. Mayho,M., Fenn,K., Craddy,P., Crosthwaite,S. Matthews,K.         45. Ingolia,N.T., Ghaemmaghami,S., Newman,J.R. Weissman,J.S. <br />     (2006) Post-transcriptional control nuclear-encoded cytochrome        (2009) Genome-wide analysis vivo translation with <br />     oxidase subunits Trypanosoma brucei: evidence genome-             nucleotide resolution using ribosome pro&#239;&#172;?ling. Science, 324, <br />     wide conservation life-cycle stage-speci&#239;&#172;?c regulatory elements.       218&#226;&#8364;&#8220;223. <br />     Nucleic Acids Res., 34, 5312&#226;&#8364;&#8220;5324.                                   46. Ingolia,N.T., Lareau,L.F. Weissman,J.S. (2011) Ribosome <br /> 26. Walrad,P., Paterou,A., Acosta-Serrano,A. Matthews,K.R.               pro&#239;&#172;?ling mouse embryonic stem cells reveals complexity <br />     (2009) Differential trypanosome surface coat regulation             dynamics mammalian proteomes. Cell, 147, 789&#226;&#8364;&#8220;802. <br />     CCCH protein co-associates procyclin mRNA cis-             47. Oh,E., Becker,A.H., Sandikci,A., Huber,D., Chaba,R., Gloge,F., <br />     elements. PLoS Pathog., 5, e1000317.                                     Nichols,R.J., Typas,A., Gross,C.A., Kramer,G. et al. (2011) <br /> 27. Helm,J.R., Wilson,M.E. Donelson,J.E. (2009) Differential             Selective ribosome pro&#239;&#172;?ling reveals cotranslational chaperone <br />     expression protease gene family African trypanosomes.            action trigger factor vivo. Cell, 147, 1295&#226;&#8364;&#8220;1308. <br />     Mol. Biochem. Parasitol., 163, 8&#226;&#8364;&#8220;18.                                 48. Michel,A.M. Baranov,P.V. (2013) Ribosome pro&#239;&#172;?ling: a <br /> 28. Horn,D. (2008) Codon usage suggests translational selection         Hi-Def monitor protein synthesis genome-wide scale. <br />     major impact protein expression trypanosomatids.             Wiley Interdiscip. Rev. RNA, 349, 4184&#226;&#8364;&#8220;4188. <br />     BMC Genomics, 9, 2.                                                  49. Ingolia,N.T. (2010) Genome-wide translational pro&#239;&#172;?ling by <br /> 29. Manful,T., Fadda,A. Clayton,C. (2011) The role 50 -30         ribosome footprinting. Methods Enzymol., 470, 119&#226;&#8364;&#8220;142. <br />     exoribonuclease XRNA transcriptome-wide mRNA                      50. Ingolia,N.T., Brar,G.A., Rouskin,S., McGeachy,A.M. and <br />     degradation. RNA, 17, 2039&#226;&#8364;&#8220;2047.                                         Weissman,J.S. (2012) The ribosome pro&#239;&#172;?ling strategy for <br /> 30. Yoffe,Y., Zuberek,J., Lewdorowicz,M., Zeira,Z., Keasar,C.,               monitoring translation vivo deep sequencing of <br />     Orr-Dahan,I., Jankowska-Anyszka,M., Stepinski,J.,                        ribosome-protected mRNA fragments. Nat. Protoc., 7, 1534&#226;&#8364;&#8220;1550. <br />     Darzynkiewicz,E. Shapira,M. (2004) Cap-binding activity       51. Langmead,B. Salzberg,S.L. (2012) Fast gapped-read <br />     eIF4E homolog Leishmania. RNA, 10, 1764&#226;&#8364;&#8220;1775.                    alignment Bowtie 2. Nat. Methods, 9, 357&#226;&#8364;&#8220;359. <br />  3636 Nucleic Acids Research, 2014, Vol. 42, No. 6 <br />  <br />  <br /> 52. Aurrecoechea,C., Barreto,A., Brestelli,J., Brunk,B.P., Cade,S.,            database eukaryotic uORF biology. Nucleic Acids Res.,, 42, <br />     Doherty,R., Fischer,S., Gajria,B., Gao,X., Gingle,A. et al., (2013)        D60&#226;&#8364;&#8220;D67. <br />     EuPathDB: eukaryotic pathogen database. Nucleic Acids Res.,        72. Calvo,S.E., Pagliarini,D.J. Mootha,V.K. (2009) Upstream <br />     41, D684&#226;&#8364;&#8220;D691.                                                             open reading frames cause widespread reduction protein <br /> 53. Guttman,M., Russell,P., Ingolia,N.T., Weissman,J.S.                    expression polymorphic humans. Proc. Natl Acad. <br />     Lander,E.S. (2013) Ribosome Pro&#239;&#172;?ling Provides Evidence                Sci. USA, 106, 7507&#226;&#8364;&#8220;7512. <br />     Large Noncoding RNAs Do Not Encode Proteins. Cell, 154,                73. Hood,H.M., Neafsey,D.E., Galagan,J. Sachs,M.S. (2009) <br />     240&#226;&#8364;&#8220;251.                                                                   Evolutionary roles upstream open reading frames mediating <br /> 54. Alsford,S. Horn,D. (2008) Single-locus targeting constructs            gene regulation fungi. Annu. Rev. Microbiol., 63, 385&#226;&#8364;&#8220;409. <br />     reliable regulated RNAi transgene expression                74. Matsui,M., Yachie,N., Okada,Y., Saito,R. Tomita,M. (2007) <br />     Trypanosoma brucei. Mol. Biochem. Parasitol., 161, 76&#226;&#8364;&#8220;79.                  Bioinformatic analysis post-transcriptional regulation uORF <br /> 55. Anders,S. Huber,W. (2010) Differential expression analysis             human mouse. FEBS Lett., 581, 4184&#226;&#8364;&#8220;4188. <br />     sequence count data. Genome Biol., 11, R106.                       75. Iacono,M., Mignone,F. Pesole,G. (2005) uAUG uORFs <br /> 56. Robinson,M.D., McCarthy,D.J. Smyth,G.K. (2010) edgeR:                human rodent 50 untranslated mRNAs. Gene, 349, 97&#226;&#8364;&#8220;105. <br />     Bioconductor package differential expression analysis           76. Helm,J.R., Wilson,M.E. Donelson,J.E. (2008) Different trans <br />     digital gene expression data. Bioinformatics, 26, 139&#226;&#8364;&#8220;140.                 RNA splicing events bloodstream procyclic Trypanosoma <br /> 57. Butter,F., Bucerius,F., Michel,M., Cicova,Z., Mann,M.                  brucei. Mol. Biochem. Parasitol., 159, 134&#226;&#8364;&#8220;137. <br />     Janzen,C.J. (2013) Comparative proteomics life cycle            77. Camby,I., Le Mercier,M., Lefranc,F. Kiss,R. (2006) <br />     stages stable isotope-labeled Trypanosoma brucei reveals novel          Galectin-1: small protein major functions. Glycobiology, <br />     components parasite&#226;&#8364;&#8482;s host adaptation machinery. Mol.               16, 137R&#226;&#8364;&#8220;157R. <br />     Cell. Proteomics, 12, 172&#226;&#8364;&#8220;179.                                         78. Fletcher,J.C., Brand,U., Running,M.P., Simon,R. and <br /> 58. Cox,J., Neuhauser,N., Michalski,A., Scheltema,R.A., Olsen,J.V.             Meyerowitz,E.M. (1999) Signaling cell fate decisions by <br />     Mann,M. (2011) Andromeda: peptide search engine                      CLAVATA3 Arabidopsis shoot meristems. Science, 283, <br />     integrated MaxQuant environment. J. Proteome Res., 10,            1911&#226;&#8364;&#8220;1914. <br />     1794&#226;&#8364;&#8220;1805.                                                             79. Dinger,M.E., Pang,K.C., Mercer,T.R. Mattick,J.S. (2008) <br /> 59. Cox,J. Mann,M. (2008) MaxQuant enables high peptide                    Differentiating protein-coding noncoding RNA: challenges <br />     identi&#239;&#172;?cation rates, individualized p.p.b.-range mass accuracies           ambiguities. PLoS Comput. Biol., 4, e1000176. <br />     proteome-wide protein quanti&#239;&#172;?cation. Nat. Biotechnol., 26,         80. Morris,J.C., Wang,Z., Drew,M.E. Englund,P.T. (2002) <br />     1367&#226;&#8364;&#8220;1372.                                                                 Glycolysis modulates trypanosome glycoprotein expression as <br /> 60. Webb,H., Burns,R., Ellis,L., Kimblin,N. Carrington,M.                  revealed RNAi library. EMBO J., 21, 4429&#226;&#8364;&#8220;4438. <br />     (2005) Developmentally regulated instability GPI-PLC            81. Alsford,S., Turner,D.J., Obado,S.O., Sanchez-Flores,A., <br />     mRNA dependent short-lived protein factor. Nucleic Acids           Glover,L., Berriman,M., Hertz-Fowler,C. Horn,D. (2011) <br />     Res., 33, 1503&#226;&#8364;&#8220;1512.                                                       High-throughput phenotyping using parallel sequencing RNA <br /> 61. Mair,G., Shi,H., Li,H., Djikeng,A., Aviles,H.O., Bishop,J.R.,              interference targets African trypanosome. Genome Res., 21, <br />     Falcone,F.H., Gavrilescu,C., Montgomery,J.L., Santori,M.I. et al.          915&#226;&#8364;&#8220;924. <br />     (2000) A new twist trypanosome RNA metabolism: cis-splicing         82. Manful,T., Cristodero,M. Clayton,C. (2009) DRBD1 the <br />     pre-mRNA. RNA, 6, 163&#226;&#8364;&#8220;169.                                              Trypanosoma brucei homologue spliceosome-associated <br /> 62. Berriman,M., Ghedin,E., Hertz-Fowler,C., Blandin,G.,                       protein 49. Mol. Biochem. Parasitol., 166, 186&#226;&#8364;&#8220;189. <br />     Renauld,H., Bartholomeu,D.C., Lennard,N.J., Caler,E.,                  83. Vattem,K.M. Wek,R.C. (2004) Reinitiation involving <br />     Hamlin,N.E., Haas,B. et al. (2005) The genome African               upstream ORFs regulates ATF4 mRNA translation in <br />     trypanosome Trypanosoma brucei. Science, 309, 416&#226;&#8364;&#8220;422.                     mammalian cells. Proc. Natl Acad. Sci. USA, 101, 11269&#226;&#8364;&#8220;11274. <br /> 63. Bakker,B.M., Michels,P.A., Opperdoes,F.R. Westerhoff,H.V.          84. Hinnebusch,A.G. (1993) Gene-speci&#239;&#172;?c translational control the <br />     (1997) Glycolysis bloodstream form Trypanosoma brucei               yeast GCN4 gene phosphorylation eukaryotic initiation <br />     understood terms kinetics glycolytic enzymes.          factor 2. Mol. Microbiol., 10, 215&#226;&#8364;&#8220;223. <br />     J. Biol. Chem., 272, 3207&#226;&#8364;&#8220;3215.                                        85. Dever,T.E., Feng,L., Wek,R.C., Cigan,A.M., Donahue,T.F. and <br /> 64. Dana,A. Tuller,T. (2012) Determinants translation                   Hinnebusch,A.G. (1992) Phosphorylation initiation factor <br />     elongation speed ribosomal pro&#239;&#172;?ling biases mouse                    2 alpha protein kinase GCN2 mediates gene-speci&#239;&#172;?c <br />     embryonic stem cells. PLoS Comput. Biol., 8, e1002755.                     translational control GCN4 yeast. Cell, 68, 585&#226;&#8364;&#8220;596. <br /> 65. Saas,J., Ziegelbauer,K., von Haeseler,A., Fast,B. Boshart,M.       86. Kong,J. Lasko,P. (2012) Trans <br /> </body></html>