<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                Citation: Cell Death Disease (2011) 2, e188; doi:10.1038/cddis.2011.72 <br />                                                                                 &amp; 2011 Macmillan Publishers Limited All rights reserved 2041-4889/11 <br />                                                                                 www.nature.com/cddis <br />  <br />  <br /> Gene-dependent cell death yeast <br /> X Teng1, W-C Cheng2, B Qi2, T-X Yu2, K Ramachandran3, MD Boersma2, T Hattier4, PV Lehmann4,5, FJ Pineda2,6 JM Hardwick*,1,2 <br />  <br />  <br /> Caspase-dependent apoptotic cell death extensively studied cultured cells embryonic development, but <br /> the existence analogous molecular pathways single-cell species uncertain. This reduced enthusiasm applying the <br /> advanced genetic tools available yeast study cell death regulation. However, partial characterization mammals of <br /> additional genetically encoded cell death mechanisms, lead range dying cell morphologies necrosis, suggests <br /> potential applications yeast genetics. In light, revisited topic gene-dependent cell death yeast determine <br /> the prevalence yeast genes capacity contribute cell-autonomous death. We developed rigorous strategy by <br /> allowing sufficient time gene-dependent events occur, insufficient time evolve new populations, applied this <br /> strategy Saccharomyces cerevisiae gene <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>knockout</span> collection. Unlike sudden heat shock, ramped heat stimulus <br /> delivered minutes thermocycler, coupled assessment viability automated counting microscopic <br /> colonies revealed highly reproducible gene-specific <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>survival</span> phenotypes, typically persist alternative conditions. <br /> Unexpectedly, identified 800 yeast knockout strains exhibit significantly increased survival following insult, <br /> implying genes contribute cell death. Although death mechanisms uncharacterized, study <br /> facilitates exploration. <br /> Cell Death Disease (2011) 2, e188; doi:10.1038/cddis.2011.72; published online 4 August 2011 <br /> Subject Category: Experimental Medicine <br />  <br />  <br /> Studies yeast models provided key insight normal                   death). The category includes genes delete selected <br /> cell physiology human disease processes including                           subsets plant animal cells required construct <br /> neurological disorders.1 In contrast, yeast model systems                       complex networks development, maintain healthy <br /> have advanced understanding mammalian cell                       tissues maturity respond infectious pathogens and <br /> death mechanisms. Disputes existence apoptosis-                     environmental stresses.7,8 These cell death genes not <br /> like mechanisms yeast thwarted efforts                  constitutively active, selectively activated by <br /> direction. However, emerging list alternative non-                        specific signals promote cell autonomous death. The <br /> apoptotic cell death mechanisms mammalian cells helps                        severe structural malformations head brain of <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> allay concerns.2 Because known                     knockout mice lacking caspase-9 caspase-3 provide <br /> alternate mammalian mechanisms responsible                              experimental evidence caspases evolved least <br /> genes, difficult predict which, any, processes             eliminate vast excess number neurons <br /> will evolutionarily conserved yeast. Several lines                     produced development.9 New evidence strongly <br /> evidence support existence genetically encoded cell                      argues necrotic cell death mechanisms <br /> death functions single-cell species,3&#226;&#8364;&#8220;6                     selected evolution, example, the <br /> known molecular mechanisms.                                         hematopoietic system.10 It argued evolved <br />    Before pursuing strategy uncover potential cell death                   cell death genes include genes promote death <br /> genes yeast, useful articulate generally acceptable               cell sustain survival cell providing <br /> <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> cell death gene applies single-                nutrients derived dead cell.11,12 <br /> multi-cellular species. In study adopt term                       The second category cell death genes includes that <br /> programmed cell death broadest sense, referring                   hasten cell death accidentally and, <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, did <br /> genetically encoded function dying cell contributes                 evolve purpose. Originally assumed that <br /> importantly demise. This <span id='am-473' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> justified                 essentially forms accidental cell death non- <br /> by following arguments. Conceptually, genes respon-                     programmed (not gene-dependent) described in <br /> sible cell death grouped main categories,                   morphological terms necrosis. However, important <br /> those selected evolution cause cell suicide                       distinction overlooked. Indeed unlikely gene <br /> those did evolve purpose (accidental cell                     products encoded dying cell influence on <br />  <br /> 1 <br />  Department Pharmacology Molecular Sciences, Johns Hopkins School Medicine, Baltimore, MD, USA; 2W. Harry Feinstone Department Molecular <br /> Microbiology Immunology, Johns Hopkins University, Bloomberg School Public Health, Baltimore, MD, USA; 3The McKusick-Nathans Institute Genetic <br /> Medicine, Johns Hopkins University School Medicine, Baltimore, MD 21205, USA; 4Cellular Technology Limited, Shaker Heights, OH, USA; 5Department of <br /> Pathology, Case Western Reserve University, Cleveland, OH, USA 6Department Biostatistics, Johns Hopkins University Bloomberg School Public Health, <br /> Baltimore, MD, USA <br /> *Corresponding author: JM Hardwick, Johns Hopkins University, 615 N. Wolfe St., BSPH-MMI, Rm E5140, Baltimore, MD 21205, USA. <br /> Tel: &#195;&#190; 410 955 2716; Fax: &#195;&#190; 410 955 0105; E-mail: hardwick@jhu.edu <br /> Keywords: yeast; programmed cell death; genetic screen; apoptosis; heat ramp; genomic <br /> Abbreviations: YKO, yeast knockout; MAT, mating type; YPD, yeast-extract peptone dextrose <br /> Received 10.6.11; accepted 21.6.11; Edited G Melino <br />                                                         Yeast-programmed cell death <br />                                                                       X Teng et al <br /> 2 <br />  <br />           cell fate assault sufficiently strong kill sheer force;                                                   Heat ramp dose-dependence <br />           gene-independent (non-programmed) cell death.                                                                    Untreated                    10 min <br />           In contrast, examples gene-dependent accidental cell <br />           death include detrimental programmed necrotic cell                              WT <br />           deaths observed non-apoptotic ischemic injury, lyso-                             fis1* <br />           leakage exaggerated inflammatory responses                              dnm1 <br />           characteristic virus-induced encephalitis.13,14 <br />               Although evolved accidental cell death genes                                                               20 min                      30 min <br />           distinguished conceptually, difficult distin-                          WT <br />           guish experimentally developmental processes                             fis1* <br />           (e.g., embryogenesis lymphoid system) available for <br />                                                                                                dnm1 <br />           study. For example, currently possible rigorously <br />           determine pathological cell death ischemic injury                                                                    5-fold dilutions <br />           adults evolved, accidental combination thereof, <br />           knockout animals expected                                                              60 <br />                                                                                                                                                                   WT <br />           <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> case. This effort complicated <br />  <br />  <br />  <br />  <br />                                                                                                    % Viability (FUN1 stain) <br />                                                                                                                               50                                   fis1* <br />           pro-death gene essential pro-survival functions <br />                                                                                                                                                                    dnm1 <br />           (e.g., calpain). Regardless pro-death functions                                                      40 <br />           cathepsins, calpain RIPK result evolution or <br />           occur accidentally, importance therapeutic                                                             30 <br />           targets dependent distinction. Discerning                                                            20 <br />           evolved versus accidental cell death protists equally <br />           problematic, categories cell death genes                                                         10 <br />           informative human disease mechanisms. The genes <br />                                                                                                                               0 <br />           responsible array necrosis-like morphologies                                                                10           20               30 <br />           currently identified outnumber list genes <br />                                                                                                                                          Time 51&#194;&#176;C (min) <br />           facilitate caspase-mediated apoptosis. Therefore, we <br />           sought determine prevalence death-promoting genes                Figure 1 Genetic <span id='am-474' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> programmed cell death assay development. <br />           yeast, regardless evolutionary origins regard-             (a) Log-phase wild type (BY4741) yeast MATa deletion strains analyzed <br />                                                                                      using heat ramp assay (30&#226;&#8364;&#8220;51 1C 30 min held 51 1C indicated <br />           cell morphologies occur dying <br />                                                                                      times). Undiluted fivefold serial dilutions plated YPD agar; Dfis1* <br />           process. However, available yeast                (Dfis1/whi2-d1). (b) Cell viabilities experiments panel (a) determined <br />           cell death assays amenable task,             vital dye staining FUN1 (Molecular Probes), presented mean&#194;&#177;S.D. for <br />           develop strategy reliably quantify gene-dependent cell             independent experiments duplicate, counting double blinded samples, <br />           death yeast.                                                            B300 cells sample. Po0.00002 (t-test) wild type versus mutant under <br />                                                                                      conditions untreated <br />           Results <br />  <br />           Experimental definition gene-dependent yeast cell                          Importantly, cell death phenotypes knockout strains <br />           death. While performing hundreds trials required                Ddnm1 Dfis1* assay detectable an <br />           developing treatment conditions reliably detect gene-                 insult, strains indistinguishable wild type <br />           dependent yeast cell death, routinely compared test               conditions treatment (Figure 1a). Vital dye <br />           strains distinct susceptibilities cell death (wild type,           staining yields similar results, demonstrating reduced <br />           Ddnm1 Dfis1*). The dynamin-related GTPase Drp1/                        colony formation following treatment caused failure of <br />           Dnm1 conserved mitochondrial fission factor                  living cells grow colonies (Figure 1b). <br />           promote cell death mammals,15 flies,8 worms16 yeast,3 <br />           detailed mechanisms delineated. Deletion <br />           yeast DNM1 gene renders yeast resistant range               Gene-specific cell death phenotypes revealed heat <br />           insults, including H2O2, acetic acid, viral toxins                     ramp treatment. Using test strains, sought to <br />           aging.3,17&#226;&#8364;&#8220;20 DNM1 deletion protects yeast heat                  establish cell death assay invariably distinguishes the <br />           treatment dose-dependent manner measured colony                    survival test strains wild type given <br />           formation (Figures 1a b). To mark lower bound                   condition. Key concepts success establish <br />           survival assay development, used death-                      treatment conditions sufficiently mild allow time <br />           sensitive FIS1 knockout strain.3 Because Fis1                    gene-dependent cell suicide occur, deliver this <br />           conserved mitochondrial fission factor, anticipated             stimulus uniformly simultaneously cells the <br />           FIS1 deletion cause sensitivity cell death                   sample. It recognized mammalian systems that <br />           stimuli. This paradox explained secondary WHI2 gene                excessive doses apoptotic stimulus (e.g., <br />           <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> arose independently FIS1 knockouts                heat) kill cells non-programmed assault cannot <br />           (Dfis1*).21 Spontaneous WHI2 mutations apparently                          influenced genetically encoded intracellular <br />           compensate fitness defect unrelated mitochondrial                 factor.22 This phenomenon illustrated yeast cultures <br />           fission caused deletion FIS1.21                                      (mid log-phase) quickly heated seconds using <br />  <br /> Cell Death Disease <br />                                                                                                 Yeast-programmed cell death <br />                                                                                                X Teng et al <br />                                                                                                                                                                                                                              3 <br />  <br />  <br />  <br />  <br />                                           dmn1 <br />  <br />  <br />  <br />                                                         dmn1 <br />                                                                                                                                                                     Post-log phase <br />  <br />  <br />  <br />  <br />                                           yca1 <br />  <br />  <br />  <br />                                                         yca1 <br />                                           fis1* <br />  <br />  <br />  <br />                                                         fis1* <br />                                          WT <br />  <br />  <br />  <br />                                                        WT <br />                                                                                                                                                      51&#194;&#176;C        60          55&#194;&#176;C               62&#194;&#176;C <br />                                                                                                                                                                                            60 <br />                     50                                                                                          50 <br />           Temp &#194;&#176;C <br />  <br />  <br />  <br />                              51&#194;&#176;C                                                                                                                                  50 <br />  <br />  <br />  <br />  <br />                                                                                                       Temp &#194;&#176;C <br />                                                                                                                                                                                            50 <br />                     40 <br />                                                                                                                 40                                                 40                      40 <br />                     30 <br />                          0 1 2 3 4 5                                                                                                              (16 h)                     (16 h)                 (48 h) <br />                                                                                                                 30                                               30                  30 <br />                                                                                                                                              0     10      20       0 5 10 15            0          10    20 <br />           b                                                                                                                                                 Heat ramp treatment time (min) <br />                     50 <br />           Temp &#194;&#176;C <br />  <br />  <br />  <br />  <br />                               52&#194;&#176;C                                                                                            WT <br />                     40                                                                                      fis1* <br />                                                                                                            dnm1 <br />                     30                                                                                      yca1 <br />                          0 1 2 3 4 5 <br />  <br />  <br />                                                                      5-fold serial dilutions <br />           c                                                                                                                                                             5-fold dilutions <br />                     50 <br />           Temp &#194;&#176;C <br />  <br />  <br />  <br />  <br />                                51&#194;&#176;C                                                                                                                                       Log phase <br />                     40                                                                                                                       3000 <br />                                                                                                                                                                                                 WT <br />                                                                                                                                                                                                  fis1* <br />  <br />  <br />  <br />  <br />                                                                                                                 Colony forming units (cfu) <br />                     30                                                                                                                       2500 <br />                          0 5 15 25                                                                                                                                                               dnm1 <br />                                                                                                                                              2000                                                yca1 <br />           d <br />                     50 <br />           Temp &#194;&#176;C <br />  <br />  <br />  <br />  <br />                                 51&#194;&#176;C                                                                                                         1500 <br />  <br />                     40                                                                                                                       1000 <br />  <br />                     30                                                                                                                           500 <br />                          0   20 40 60 <br />           e                                                                                                                                       0 <br />                     50 <br />           Temp &#194;&#176;C <br />  <br />  <br />  <br />  <br />                                                                                                                                                               3h              4h            7h <br />                                  51&#194;&#176;C <br />                                                                                                                                                            Hours dilution fresh media <br />                     40 <br />                                                                                                                                              WT <br />                                                                                                                    fis1* <br />                     30                                                                                            dnm1 <br />                          0 30 60 90         No heat      +Heat                                                     yca1 <br />                          Time (min) <br />                                                                                                                                                                         5-fold dilutions <br /> Figure 2 Thermocycler conditions induce gene-dependent cell death. (a b) <br /> Log-phase yeast cultures untreated treated heat ramp conditions                Figure 3 Heat ramp assays accommodate different metabolic states. (a) Cells <br /> graphed, ramping 30&#226;&#8364;&#8220;51/52 1C B6&#226;&#8364;&#8220;20 s, holding temperature 5 min                 dense 16 48 h cultures diluted OD600 0.5 YPD immediately <br /> before plating YPD. (c) In 2 min 30&#226;&#8364;&#8220;40 1C, 40&#226;&#8364;&#8220;51 1C 10 min held            treated indicated heat ramp conditions plating YPD. (b) Cultures <br /> at 51 1C 10 min. (d) In 30 min 30&#226;&#8364;&#8220;51 1C held 51 1C 30 min. (e) In               grown continuous log phase 3, 4 7 h (starting OD600 0.25, 0.20 and <br /> 60 min 30&#226;&#8364;&#8220;51 1C held 51 1C 30 min                                                   0.05, respectively) treated (all OD600 0.5) shown panel (left) and <br />                                                                                                colony counts presented mean&#194;&#177;S.D. colony-forming units (CFU) least <br />                                                                                                independent experiments condition. t-test, Po0.00001 comparing WT to <br />                                                                                                3&#226;&#8364;&#8220;4 h, Po0.04 7 h counts low (not significant DNM1 <br />                                                                                                corrected multiple comparisons). Untreated samples indistinguishable <br /> a thermocycler. Although conditions occasionally                                     strains (not shown) <br /> reveal differences survival wild type test <br /> strains, including death-resistant metacaspase knockout <br /> Dyca1/Dmca1 (Figure 2a), small variations temperature, <br /> treatment duration growth conditions treatment,                                      For example, post-log-phase yeast cultures known <br /> abolish differences (Figure 2b data shown).                                      resistant death log-phase cells treated with <br /> This problem overcome using thermocycler (PCR                                          heat ramp conditions (Figure 2c versus Figure 3a- <br /> machine) programmed ramp ambient 51 1C                                         left), adjustments assay conditions easily accommodate <br /> 12&#226;&#8364;&#8220;30 min, seconds. Essentially fail,                                      16 h 48 h post-log cultures (Figure 3a). Even single <br /> these conditions simultaneously distinguish survival                                    condition distinguish test strains increasingly <br /> test strains (Figures 2c d). In contrast, ramping                                      sensitive death increasing time spent continuous log <br /> temperature course 1 h allows sufficient time                                  phase treatment (Figure 3b). <br /> test strains adapt, concealing genetic differences                                       Importantly, thermocycler eliminates confounding edge <br /> in cell death susceptibility (Figure 2e) previously                                         effects encountered multi-well plates large tubes are <br /> described.23                                                                                   heated devices (e.g. air liquid). This strategy also <br />    Using general approach, parameters (e.g. ramp                                  overcomes problems delivery liquid death stimuli <br /> rate, maximum temperature hold time) adjusted                                       (e.g. acetic acid, H2O2) reach cells equally <br /> with relative latitude accommodate changes metabolic                                     mixing inactivated rapidly aqueous <br /> state cells, media types, background strains strains                                dilution. Importantly, survival differences strains <br /> bearing exogenous plasmids, high reproducibility.                                    prevail range heat ramp conditions highly <br />  <br />                                                                                                                                                                                                          Cell Death Disease <br />                                                   Yeast-programmed cell death <br />                                                                 X Teng et al <br /> 4 <br />  <br />           reproducible, facilitating presentation statistical analyses                                     4000 <br />                                                                                                                                Log phase growth <br />           avoiding need report representative experiments. <br />                                                                                                               3000             Post-log phase <br />  <br />           Confirmation established phenotypes. Although                                                2000 <br />           existence evolved (or accidental) cell death genes yeast <br />  <br />  <br />  <br />  <br />                                                                                        Viability (CFU) <br />           currently debated, yeast encode orthologues                                           1000 <br />           genes established roles metazoan cell death, of <br />                                                                                                              0 <br />           implicated promoting death yeast. <br />                                                                                                          10000 <br />           Therefore, validate heat ramp assay,                                                            Replot logarithmic scale <br />           analyzed knockout strains identified early                                         1000 <br />           yeast apoptosis literature. The corresponding deletion strains <br />                                                                                                                       100 <br />           MATa YKO collection (BY4741) tested both <br />           log post-log phase. All strains exhibited phenotypes                                                10 <br />           matched published results various cell death stimuli <br />           employed (Figure 4a). Two strains prone                                                     1 <br />           cell death, knockout SOK2, transcription factor <br />           regulates production ammonia required cell-cell <br />           signaling long-term survival colonies wild strains,5 <br />           knockout NUC1, encoding conserved                                                              100 <br />           mitochondrial nuclease (EndoG).24 The remaining six <br />  <br />  <br />  <br />  <br />                                                                                            Survival rank genome- <br />           knockout strains exhibited resistance cell death                                                          80 <br />  <br />  <br />  <br />  <br />                                                                                                wide screen (%) <br />           suggesting pro-death functions genes: Nma111 <br />                                                                                                                        60 <br />           (nuclear mediator apoptosis), orthologue four <br />           mammalian mitochondrial serine proteases including Htra2/                                                    40 <br />           Omi;25 Aif1 (apoptosis-inducing factor), NADH <br />           oxidoreductase named mammalian orthologue                                                          20 <br />           AIF;26 Aif1-related yeast protein Ndi1, NADH <br />           dehydrogenase;27 Mmi1, highly conserved homolog                                                        0 <br />           human TCTP (translationally controlled tumor protein);28 <br />           Tat-D, pro-death DNase yeast, worms mammals;29 <br />           Stm1, lacks obvious mammalian orthologue <br />           reported induce yeast cell death.30                        Figure 4 Confirmation yeast cell death phenotypes. (a) Log-phase 16-h <br />                                                                                post-diauxic phase cultures treated heat ramp graphed Figure 3a <br />           Many gene deletion strains resistant cell                     left middle, respectively. Log-phase cultures prepared 16-h <br />                                                                                cultures. Cell viability determined colony-forming units (CFU) counted with <br />           death. Using plate-1 MATa deletion collection, <br />                                                                                BioSpot instrument data presented mean&#194;&#177;S.D. 3&#226;&#8364;&#8220;6 independent <br />           contains fission factor Dnm1/Drp1 knockout                 experiments. All data generated graphed linear log scales reveal <br />           94 knockout strains chromosomes 1 12,                 differences. t-test, comparing wild type others, Po0.00001 log phase, <br />           adapted heat ramp assay high throughput                   Po0.0006 post log. Untreated samples indistinguishable strains (not <br />           screening (see Figure 6a). Colony formation standard            shown). (b) Results strains presented percentile rank survival <br />           measure yeast survival, equivalent mammalian                   based genomic screen (Supplementary Table S1) <br />           clonogenic assays. Surprisingly, commercially <br />           available laboratory equipment capable accurately                 starting cell numbers strains, used saturated <br />           enumerating yeast colonies high throughput formats.               cultures (48 h). Cell densities measured treatment had <br />           Instruments measure density yeast growth            systematic correlation colony counts following heat <br />           solid medium liquid cultures provide             ramp, suggesting small differences cell density not <br />           desired information. Only colony number regardless                confounding factor (Figure 5b). Despite using media and <br />           colony size provides measure <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>cell survival</span>          consumables suppliers lot numbers, <br />           confounded cell growth rates (e.g., 100 tiny colonies             inevitable day-to-day differences total number <br />           derived 100 live cells far mass single         colonies combining strains plate-1, replicate <br />           large colony). Unless viability measured vital dyes          plates generated day highly similar <br />           (confounded time delay required sequentially               (Figure 5c). Despite issue, relative survival of <br />           sample 96 wells flow cytometry), necessary allow         individual knockout strains highly reproducible on <br />           surviving cells outgrow dead cell numbers following               different days (Figure 5d Supplementary Figure S2). <br />           treatment.                                                              High throughput enumeration single merged <br />              To help overcome problems, performed auto-               colonies ranging widely size ultimately achieved by <br />           mated colony counting 2-day plates using EliSpot               developing new BioSpot platform (Cellular Technology <br />           Reader (Cellular Technology Limited/CTL, Shaker Heights,             Limited/CTL) automated object recognition capability and <br />           OH, USA) designed read immunological ELISA assays,                specialized image capture analysis system. This <br />           proved accurate hand counting (Figure 5a             capable accurately counting 300 <br />           Supplementary Figure S1). To minimize differences             microscopic colonies spot 18 h plating <br />  <br /> Cell Death Disease <br />                                                                                                            Yeast-programmed cell death <br />                                                                                                           X Teng et al <br />                                                                                                                                                                                                                          5 <br />  <br />  <br />  <br />                                                              50                                                                             400 <br />  <br />  <br />  <br />  <br />                                   Hand colony counts (CFU) <br />  <br />  <br />  <br />  <br />                                                                                                                  Colonies heat ramp <br />                                                              40                                                                             300 <br />  <br />                                                              30 <br />                                                                                                                                             200 <br />  <br />                                                              20 <br />                                                                                                                                             100 <br />                                                                                                                                                       Empty <br />                                                              10 <br />                                                                                                                                              0 <br />                                                               0                                                                                   0                                1         2 <br />  <br />                                                                       0         20      40      60         80                                Starting cell density (Abs595) <br />  <br />                                                                           Automated colony counts (CFU) <br />  <br />  <br />  <br />  <br />                                                              300 <br />                                      Total colonies (CFU) <br />  <br />  <br />  <br />  <br />                                                                                                                                                       Survival rank (plate 1) <br />                                                              200 <br />  <br />  <br />                                                              100 <br />  <br />  <br />                                                                   0 <br />                                                    Replicate: 1 2 3                  1 2 3 4 1 2 3 4 1 2 3 4 <br />  <br />                                  Experiment:                                1          2          3             4 <br />                                                                                                                                                                                 0 100 200 300 <br />                                                                                                                                                                                 Colony count (CFU) <br /> Figure 5 Variables affecting colony counts heat ramp assays. (a) Following heat ramp treatment YKO strains MATa plate-1 MATa YKO collection, the <br /> 96 samples plated 96 spots incubated 30 1C 2 days colony visualization. Colony counts determined plate using methods, hand <br /> counting automated counting using EliSpot Reader (CTL, Inc.), expected correlation (dashed line); hand method systematically undercounts spots 420 <br /> colonies. (b) Absorbance (595 nm) of replicates 96 wells/samples plate-1 MATa YKO collection 48 h incubation (<span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> panel c) plotted <br /> against corresponding colony counts determined BioSpot instrument heat ramp treatment using assay panel a. Correlation coefficient significant <br /> (0.238). Arrow marks results wells A1 H2 lacking yeast. (c) Colony counts 94 knockout strains plate-1 determined different days, starting with <br /> an independently pinned plate prepared original frozen stock. After heat ramp treatment, samples plated replicate YPD plates, incubated <br /> B18 h 30 1C counted prototype BioSpot instrument. Median colony counts combining strains (heavy line) replicate plate shown 25 75% <br /> quartiles (boxes), range 95% counts (dash lines). (d) Rank order survival individual deletion strains plate-1 (mean&#194;&#177;S.E. data shown panel c); shaded <br /> quantiles; gene names, Supplementary Figure S2 <br />  <br />  <br />  <br /> (Figure 6a), overcoming 2-day growth period required                                               non-functional gene time deletion, yielding <br /> generate visible colonies, exacerbates problem                                               results similar wild-type control strains (Figure 6b). <br /> overlapping colonies, reduces countable range dis-                                                 Unexpectedly, B800 different knockout strains YKO <br /> cernable colonies spot (r30) biases counts down-                                                  collection resistant heat ramp-induced death <br /> ward.                                                                                                     compared death-resistant strain Ddnm1, implying that <br />    The heat ramp cell death assay (Figure 6a, heat ramp                                                   function of deleted genes <br /> conditions Figure 3a right) applied MATa                                                  contributes way death cell receiving <br /> yeast knockout collection B4800 single <span id='am-395' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-396' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-401' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-404' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-418' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-419' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-428' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-435' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-437' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-441' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-444' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-448' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-449' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-461' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-469' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-471' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span>                                                   death stimulus. The output data genome-wide cell <br /> strains, results 4&#226;&#8364;&#8220;8 replicates counted using                                             death screen curated optimizing auto- <br /> the BioSpot platform. Because wild-type parental strain                                            mated counting parameters, removing miscounted <br /> not present YKO collection, wild-type strains                                                 replicates artifacts identified inspection. After <br /> different sources processed equivalently parallel                                              curated genomic screen data assembled the <br /> inserted subset plates direct                                              B4800 knockout strains ordered survival rank <br /> comparisons. In addition, heat ramp survival result wild                                          (Supplementary Table S1), retrieved results from <br /> type estimated using HO gene (YDL227C) knockout                                                   screen original test strains strains <br /> strain YKO collection (average 108 colonies),                                             previously reported cell death phenotypes. These results <br />  <br />                                                                                                                                                                                                      Cell Death Disease <br />                                                                                Yeast-programmed cell death <br />                                                                                              X Teng et al <br /> 6 <br />  <br />  <br />  <br />  <br />                                                                                                             HR&#226;&#710;&#8217;post <br />                                                                                                             HR&#226;&#710;&#8217;log <br />                                                                                                             AA&#226;&#710;&#8217;log <br />                                                                                                                         CLN3         Resistant <br />                                                                                                                       &#226;&#8364;&#162; SSA2 <br />                                                                      Frozen YKO stock                                   EMC6 <br />                                                                                                                         CNE1 <br />                                                                                                                         ERP2         Sensitive <br />                                                                                                                         BDH1 <br />                                                                                                                         FUN14       &#226;&#8364;&#162; Heat shock sensitive <br />                                                                                                                         YAR023C     &#226;&#8364;&#162; Marked Fig. S1 <br />                                                                                                                         SPO7 <br />                                                                                                                         YLL032C <br />                      2 days 30&#194;&#176;C YPD                                                                                    UIP3 <br />                                                                                                                         NTG1 <br />                            agar                                                 BioSpot&#194;&#174;                                ENT4 <br />                                                                                  Reader                                 YEH1 <br />                                                                                                                         BUD14 <br />                                                                                                                         YAR028W <br />                                                                                                                         YLL017W <br />                                                                                                                       &#226;&#8364;&#162; FUN30 <br />                                                                                                                         UBI4 <br />                          48 h 30&#194;&#176;C 200 &#194;&#181;l YPD                                                                         &#226;&#8364;&#162; KNS1 <br />                                                                                                                         SNC1 <br />                                                                                                                       &#226;&#8364;&#162; DNM1 <br />                                                                                                                       ? <br />                                                                                                                       &#226;&#8364;&#162; GCV3 <br />                                                                                                                         YAR044W <br />                          100 &#194;&#181;l OD6000.5 + heat                                                                         YAL037W <br />                                                                                                                         CCR4 <br />                                                                                                                       &#226;&#8364;&#162; NUP60 <br />                                                                                                                         MYO4 <br />                                                                                                                         AIM1 <br />                                                                                                                         SPA2 <br />                                                                                                                         POM33 <br />                     Dilute, plate 5 &#194;&#181;l YPD                                                                           BDH2 <br />                                                                                                                         FLC2 <br />                                                                                                                         BPT1 <br />                                                                                                                         SDC25 <br />                                                                                                                         ERV46 <br />                                                                                                                         DRS2 <br />                                                                                                                       &#226;&#8364;&#162; PEX22 <br />                                                                                                                         GDH3 <br />                                                                                                                         KIN3 <br />                               18 h 30&#194;&#176;C                                                                              FRA1 <br />                                                                                                                       &#226;&#8364;&#162; FUN19 <br />                                                                                                                       &#226;&#8364;&#162; SWC3 <br />                                                   600                                                                 &#226;&#8364;&#162; GEM1 <br />                           Average colony counts <br />  <br />  <br />  <br />  <br />                                                                                                                         YAL043C&#226;&#710;&#8217;a <br />                                                   500                                                                   PMT2 <br />                                                                                                                         FUN26 <br />                                                   400                                                                   PUF3 <br />                                                                                 dnm1                                  &#226;&#8364;&#162; IRC19 <br />                                                   300                                                                   YAL065C <br />                                                                          WT                                             PRM9 <br />                                                   200                                                                   RTT109 <br />                                                                                                                       &#226;&#8364;&#162; ATS1 <br />                                                   100                                                                   SEO1 <br />                                                                                                                         SAW1 <br />                                                    0                                                                    PAU17 <br />                                                         0     1000 2000 3000 4000 5000                                  CYC3 <br />                                                                                                                         ADE1 <br />                                                             Rank order knockout strains                            &#226;&#8364;&#162; DEP1 <br />                                                                                                                         YAR040C <br />                                                                                                                       &#226;&#8364;&#162; MMM1 <br />           Figure 6 Scheme high throughput heat ramp cell death assay. (a) See                                     &#226;&#8364;&#162; SPO75 <br />           Materials Methods details. Bright-field image 18 h spotted sample                            &#226;&#8364;&#162; MDM10 <br />           BioSpot instrument shown. (b) Viability measured colony number                             SWH1 <br />                                                                                                                         PAU8 <br />           4&#226;&#8364;&#8220;8 replicates strains MATa YKO collection plotted (each point                              YAL018C <br />           representing mean 10 knockout strains. The results control strains HO                             FRT2 <br />           (mean 108) Ddnm1 (mean 202) indicated (Supplementary Table S1).                                 AIM2 <br />                                                                                                                         YAT1 <br />           Mean count strains &#194;&#188; 122; quantile wild-type HO knockout                               PSR1 <br />           boxed, 2000 YKOs surviving better WT                                                              &#226;&#8364;&#162; SWD1 <br />                                                                                                                         COX17 <br />                                                                                                                         PAU7 <br />                                                                                                                         TPO1 <br />                                                                                                                         OAF1 <br />                                                                                                                         RBG1 <br />                                                                                                                       &#226;&#8364;&#162; VPS8 <br />                                                                                                                         GIP4 <br />                                                                                                                       &#226;&#8364;&#162; HSP104 <br />                                                                                                                         SYN8 <br />                                                                                                                         YAR043C <br />           Figure 7 High throughput cell death assays reveal novel phenotypes. Yeast                                     FUN12 <br />                                                                                                                         PSK1 <br />           knockout strains plate-1 MATa YKO collection assessed survival                                 ECM1 <br />           colony formation following death stimulus. Gene names ordered mean                                 &#226;&#8364;&#162; GPB2 <br />           survival rank different assay types. Results shown acetic acid                               YAR037W <br />                                                                                                                         YAR029W <br />           treatment40 log phase (AA-log, independent experiments                                   &#226;&#8364;&#162; SSA1 <br />           140 mM, 145 mM 150 mM acetic acid previously described), heat ramp <br />           treatment graphed Figure 2c log phase (HR-log, independent <br />           experiments), heat ramp treatment graphed Figure 3a right 48-h <br />           post-diauxic phase (HR-post, independent experiments) <br />  <br /> Cell Death Disease <br />                                                                      Yeast-programmed cell death <br />                                                                     X Teng et al <br />                                                                                                                                                7 <br />  <br /> from high throughput screen (Figure 4b) strikingly          assessment irrespective growth rate long grew <br /> similar hand screen (Figure 4a), validating      colony B100 microns plating. <br /> high throughput heat ramp assay.                                       Despite ongoing debate, evolutionarily selected death <br />                                                                     pathways purported exist single-cell species <br /> Shared death phenotypes despite different metabolic                 including yeast, parasites bacteria. For example, dying <br /> states death stimuli. Dnm1/Drp1 conserved pro-            cells near center colony wild yeast strains appear to <br /> death activity following multiple death stimuli          required nourishment survival younger cells.5 <br /> several eukaryotic species.3,8,15,16,31 Therefore,               Yeast viruses trigger gene-dependent death, phenom- <br /> predicted cell death genes behave              enon assumed host defense mechanism mammals <br /> similarly regardless death stimulus type metabolic        unless plan goes awry excessive cell death causes <br /> conditions imposed. To test possibility, plate-1        disease pathology.17 Experimental models evolution <br /> MATa knockout collect treated log phase          suggest ultimate survival species relies on <br /> the heat ramp assay (as Figure 2c) range            programmed death portion population aging <br /> acetic acid concentrations compared heat ramp          yeast cultures nutrients diminished.33 In this <br /> results Figure 6 strains 48-h post-log phase     scenario, long-term survival cellular species could <br /> (Figure 7). Most strains (71%) exhibit similar phenotype          require programmed cell death eliminate infected, <br /> under conditions, 11% exhibited distinct                damaged defective cells, match population sizes with <br /> phenotype conditions, 18%              resources, facilitate survival better-adapted <br /> exhibited distinct phenotypes conditions.           genomes arise population. Yet little known <br /> Interestingly, heat ramp sensitivity did correlate    genetically molecularly death processes in <br /> reported heat shock screens SGD database (red                single-cell species. Yeast plants encode metacaspases <br /> dots). Thus, heat ramp cell death assays appear detect       distantly related animal caspases.7,34 However, <br /> novel phenotypes delineated using yeast         contention yeast metacaspase is <br /> model systems.                                                      involved process resembling mammalian apoptotic self- <br />                                                                     destruction subsequent engulfment healthy mamma- <br />                                                                     lian cells.35,36 Nevertheless, labs that <br /> Discussion <br />                                                                     deletion yeast metacaspase gene YCA1/MCA1 can <br /> Contrary expectation, B2000 yeast knockout         protect yeast death specific insults.31,34 Thus, is <br /> strains resistant cell death wild type,        reasonably clear YCA1/MCA1 promote yeast cell <br /> B800 different gene knockouts (representing B15%             death, detailed mechanisms yeast <br /> genes genome) resistant death           factor kills cell autonomously essentially unknown. <br /> DNM1 (DRP1) knockout. The implication activities           A major obstacle studying gene-dependent cell death in <br /> of large number gene products detrimental cell          yeast limited array assays suitable evaluating <br /> survival following stress stimulus. These gene functions          gene-dependent cell death. Thus, study programmed <br /> could inadvertently contribute cell death inappropri-       cell death yeast arguably catch-up mode compared <br /> ately deregulated stress, analogous damage caused         well-established model organisms.8 The application <br /> by dislodged mechanical machine part. Alternatively,           mammalian apoptosis assays yeast standard <br /> unchecked cell survival detrimental long-term survival    field. However, controversies regarding these <br /> of given species, genes evolved          assays unclear biochemical events <br /> part serve checks balances built cellular       detected apoptosis assays directly reflect yeast cell death <br /> processes avoid inappropriate cell survival. Many con-           mechanisms.5,37,38 For example, DNA fragmentation de- <br /> served cellular processes, including mitochondrial dynamics,        tected dying yeast TUNEL assay (labels DNA ends) <br /> bioenergetics, autophagy beneficial effects calorie      interpreted different ways. TUNEL <br /> restriction linked control programmed cell      reactivity detect evolutionarily selected pro-death <br /> death mammals model organisms             nuclease (analogous mammalian CAD), detect an <br /> studied yeast. Thus, yeast genetic tools represent             unintentionally (accidentally) activated nuclease signifi- <br /> powerful largely untapped resource poised exploitation      cantly contributes death. More prosaically, yeast <br /> of cell death mechanisms. The stringent assays reported          TUNEL reactivity simply reflect unleashed nucleases <br /> this study increase feasibility exploring unknown cell   death process (i.e., cells die the <br /> death pathways yeast.                                            rate nuclease directly responsible for <br />    Identification genes inhibit cell death          TUNEL reactivity). A similar argument holds Annexin V <br /> complicated deletion important genes                staining surface-exposed phosphatidylserine, &#226;&#8364;&#732;eat me&#226;&#8364;&#8482; <br /> individually expected impair cell survival       signal phagocytosis dying cells metazoans.39 <br /> stress. These genes simply required normal             Detection caspase-like activity reactive oxygen species <br /> cellular functions, encoding biochemical anti-          fluorescent reporters dying yeast been <br /> death activities analogous mammalian Bcl-2          challenged.35,37 Presently, prudent avoid conclusions <br /> active inhibitors known cell death mechanisms.32 Thus,        underlying yeast cell death mechanisms based on <br /> refer yeast genes pro-survival (rather anti-       mammalian apoptosis assays. Apoptosis assays applied to <br /> death) proven actively inhibit cell death       yeast useful live-dead assay analogous to <br /> program. A key component assay survival              Trypan blue staining dead mammalian cells. However, in <br />  <br />                                                                                                                            Cell Death Disease <br />                                                                    Yeast-programmed cell death <br />                                                                                  X Teng et al <br /> 8 <br />  <br />           hands assays introduce new variables (e.g., cell                              Automated colony counting. Images B18-h yeast colonies all <br />           wall removal easily compromises plasma membrane,                                        replicate YPD agar test plates captured BioSpot prototype machine at <br />           inadvertently labeling inner leaflet plasma membrane                             1024 768 pixels archived. Automated colony counting performed on <br />                                                                                                       imaged plates using BioSpot counting software (BioSpot Academic Software <br />           annexin V). Therefore, improved tools urgently needed <br />                                                                                                       Version 5.0, Cellular Technology Limited/CTL) set parameters Supplementary <br />           advantage unsurpassed tools available yeast.                             Table S2. <br />              The possible existence multiple phylogenetically con- <br />           served death pathways genes accidentally <br />                                                                                                       Conflict Interest <br />           contribute cell death increase prospects of <br />           devising interventions necrotic cell death previously                               The authors declare conflict interest. <br />           considered druggable. These discoveries coupled the <br />           opportunity gain new insights triggering death                                   Acknowledgements. We thank Jef Boeke MATa yeast strains and <br />           pathogenic yeast, extend potential utility tractable                            valuable advice. This project supported National Institutes Health R01 <br />           genetic model.                                                                              GM077875 (JMH). <br />  <br />                                                                                                       Author Contributions <br />           Materials Methods                                                                       XT conceived performed heat ramp experiments presented; WCC <br />           Yeast strains. Saccharomyces cerevisiae knockout (YKO) strains                 contributed heat ramp assay development performed acetic acid assays; <br />           MATa haploid yeast deletion collection (MATa his3D1 leu2D0 met15D0 ura3D0                   BQ contributed assay development; TY KR participated data collection; <br />           yfg::KanMX4; Research Genetics) wild-type BY4741 (MATa his3D1 leu2D0                    FJP MDB contributed data analyses; PVL TH developed optimized <br />           met15D0 ura3D0) provided Jef Boeke. Dfis1* refers YKO Dfis1,                  BioSpot instrument yeast colony counting; JMH conceived project, <br />           premature stop codon amino acid 152 WHI2 (whi2-d1).21 Yeast               oversaw data analyses, wrote manuscript. <br />           grown YPD medium. <br />  <br />                                                                                                        1. Khurana V, Lindquist S. Modelling neurodegeneration Saccharomyces cerevisiae: why <br />           FUN1 viability assay. Log-phase yeast stained FUN1 (Live/Dead                         cook baker&#226;&#8364;&#8482;s yeast? Nat Rev Neurosci 2010; 11: 436&#226;&#8364;&#8220;449. <br />           Yeast Viability Kit, Molecular Probes, Invitrogen, Carlsbad, CA, USA) adding              2. Vandenabeele P, Galluzzi L, Vanden Berghe T, Kroemer G. Molecular mechanisms of <br />           0.5 ml. FUN1 (10 mM DMSO) 5 ml calcofluor white M2R (5 mM H2O) 1 ml                necroptosis: ordered cellular explosion. Nat Rev Mol Cell Biol 2010; 11: 700&#226;&#8364;&#8220;714. <br />           cultures (OD600 adjusted), 100 ml aliquots subsequently treated heat           3. Fannjiang Y, Cheng WC, Lee SJ, Qi B, Pevsner J, McCaffery JM et al. Mitochondrial fission <br />           ramp. Immediately following, 1.5 ml concentrated yeast suspensions                      proteins regulate programmed cell death yeast. Genes Dev 2004; 18: 2785&#226;&#8364;&#8220;2797. <br />           trapped slide cover glass counted using fluorescence microscopy              4. Ellermeier CD, Hobbs EC, Gonzalez-Pastor JE, Losick R. A three-protein signaling <br />           (Nikon Eclipse E800, Nikon Inc., Melville, NY, USA;   100 objective lens, multipass             pathway governing immunity bacterial cannibalism toxin. Cell 2006; 124: 549&#226;&#8364;&#8220;559. <br />                                                                                                        5. Vachova L, Palkova Z. Physiological regulation yeast cell death multicellular colonies <br />           filter set DAPI, fluorescein rhodamine). <br />                                                                                                           triggered ammonia. J Cell Biol 2005; 169: 711&#226;&#8364;&#8220;717. <br />                                                                                                        6. Engelberg-Kulka H, Amitai S, Kolodkin-Gal I, Hazan R. Bacterial programmed cell death <br />           Heat ramp cell death assays. For low throughput assays, individual yeast                        multicellular behavior bacteria. PLoS Genet 2006; 2: e135. <br />           knockout strains streaked YPD agar plates  80 1C stocks (without              7. Coll NS, Vercammen D, Smidler A, Clover C, Van Breusegem F, Dangl JL et al. <br />           thawing) incubated 30 1C nights. Single colonies inoculated                Arabidopsis type I metacaspases control cell death. Science 2010; 330: 1393&#226;&#8364;&#8220;1397. <br />           2 ml YPD incubated 30 1C roller drum overnight. For low throughput          8. Abdelwahid E, Rolland S, Teng X, Conradt B, Hardwick JM, White K. Mitochondrial <br />                                                                                                           involvement cell death non-mammalian eukaryotes. Biochim Biophys Acta 2010. <br />           assays log-phase cultures, 0.2 OD600 yeast 16 h yeast cultures (B150 ml) <br />                                                                                                        9. Zheng TS, Hunot S, Kuida K, Flavell RA. Caspase knockouts: matters life death. <br />           diluted 3 ml fresh YPD incubated 30 1C roller drum B4 h            Cell Death Differ 1999; 6: 1043&#226;&#8364;&#8220;1053. <br />           (longer dropout media) OD600 reaches 0.5. About 30 min reaching            10. Peter ME. Programmed cell death: apoptosis meets necrosis. Nature 2011; 471: 310&#226;&#8364;&#8220;312. <br />           OD600 0.5, cell densities strains adjusted match, arrive     11. McPhee CK, Logan MA, Freeman MR, Baehrecke EH. Activation autophagy cell <br />           OD600 0.5 simultaneously. Minor differences cell density corrected                death requires engulfment receptor Draper. Nature 2010; 465: 1093&#226;&#8364;&#8220;1096. <br />           heat treatment; strain-specific differences growth rates detected         12. Cap M, Vachova L, Palkova Z. How survive yeast colony? Change metabolism <br />           conditions. Cell suspensions (100 ml) transferred 0.2-ml PCR                cope stress? Commun Integr Biol 2010; 3: 198&#226;&#8364;&#8220;200. <br />           tubes heat ramp treatment.                                                              13. Griffin DE, Levine B, Ubol S, Hardwick JM. The effects alphavirus infection neurons. <br />                                                                                                           Ann Neurol 1994; 35 (Suppl): S23&#226;&#8364;&#8220;S27. <br />               For high throughput analyses YKO knockout collection, 96-well plate was <br />                                                                                                       14. Kirkegaard T, Jaattela M. Lysosomal involvement cell death cancer. Biochim <br />           pinned YPD agar plates  80 1C stocks (without thawing), incubated                  Biophys Acta 2009; 1793: 746&#226;&#8364;&#8220;754. <br />           30 1C nights transferred pin tool 200 ml YPD 48 h           15. Frank S, Gaume B, Bergmann-Leitner ES, Leitner WW, Robert EG, Catez F et al. The role <br />           30 1C. Absorbance (Abs595) cultures determined plate reader. A total              dynamin-related protein 1, mediator mitochondrial fission, apoptosis. Dev Cell <br />           100 ml OD600 B0.5 yeast cultures (e.g., 10 ml 48-h cultures OD600 B5              2001; 1: 515&#226;&#8364;&#8220;525. <br />           90 ml YPD) transferred 0.2-ml PCR tubes heat ramp treatment. For              16. Jagasia R, Grote P, Westermann B, Conradt B. DRP-1-mediated mitochondrial <br />           high throughput log-phase assays, 48-h cultures 96-well plates diluted                  fragmentation EGL-1-induced cell death C. elegans. Nature 2005; 433: 754&#226;&#8364;&#8220;760. <br />           30-fold YPD (B0.2 OD600 yeast), incubated 4 h reach mid-log phase            17. Ivanovska I, Hardwick JM. Viruses activate genetically conserved cell death pathway a <br />           100 ml well-mixed sample transferred 0.2-ml 96-well PCR                    unicellular organism. J Cell Biol 2005; 170: 391&#226;&#8364;&#8220;399. <br />                                                                                                       18. Palermo V, Falcone C, Mazzoni C. Apoptosis aging mitochondrial morphology <br />           tubes heat ramp treatment. <br />                                                                                                           mutants S. cerevisiae. Folia Microbiol (Praha) 2007; 52: 479&#226;&#8364;&#8220;483. <br />               Yeast treated heat ramp OD600 0.5 regardless metabolic          19. Scheckhuber CQ, Erjavec N, Tinazli A, Hamann A, Nystrom T, Osiewacz HD. Reducing <br />           state. Samples (100 ml) heat-treated using Eppendorf Mastercycler EP                    mitochondrial fission results increased life span fitness fungal ageing models. <br />           gradient cycler (Eppendorf AG, Hammburg, Germany) (e.g., ramp temperature                  Nat Cell Biol 2007; 9: 99&#226;&#8364;&#8220;105. <br />           RT 30 1C immediately, hold 1 min 30 1C, ramp temperature 30 1C 62 1C          20. Weinberger M, Mesquita A, Caroll T, Marks L, Yang H, Zhang Z et al. Growth signaling <br />           20 min, return RT). Immediately heat ramp treatment, samples                 promotes chronological aging budding yeast inducing superoxide anions inhibit <br />           serially diluted 96-well microtiter plates (e.g., 20 ml plus 80 ml ddH2O) 5 ml       quiescence. Aging (Albany NY) 2010; 2: 709&#226;&#8364;&#8220;726. <br />           plated YPD agar plates (starting undiluted). For untreated controls,        21. Cheng WC, Teng X, Park HK, Tucker CM, Dunham MJ, Hardwick JM. Fis1 deficiency <br />                                                                                                           selects compensatory mutations responsible cell death growth control defects. <br />           100 ml samples diluted plated heat treatment. <br />                                                                                                           Cell Death Differ 2008; 15: 1838&#226;&#8364;&#8220;1846. <br />               Unlike screens treatments applied days,           22. Milleron RS, Bratton SB. &#226;&#8364;&#732;Heated&#226;&#8364;&#8482; debates apoptosis. Cell Mol Life Sci 2007; 64: <br />           new mutations potentially selected, 20-min heat ramp stimulus               2329&#226;&#8364;&#8220;2333. <br />           short allow selection suppressors. By counting colony numbers regardless           23. De Virgilio C, Piper P, Boller T, Wiemken A. Acquisition thermotolerance in <br />           colony sizes B18 h treatment, reduce influence newly             Saccharomyces cerevisiae heat shock protein hsp 104 absence of <br />           arising mutations.                                                                              protein synthesis. FEBS Lett 1991; 288: 86&#226;&#8364;&#8220;90. <br />  <br />  <br /> Cell Death Disease <br />                                                                                                        Yeast-programmed cell death <br />                                                                                                       X Teng et al <br />                                                                                                                                                                                                             9 <br />  <br /> 24. Buttner S, Eisenberg T, Carmona-Gutierrez D, Ruli D, Knauer H, Ruckenstuhl C                      34. Madeo F, Herker E, Maldener C, Wissing S, Lachelt S, Herlan M et al. A caspase-related <br />     et al. Endonuclease G regulates budding yeast life death. Mol Cell 2007; 25:                      protease regulates apoptosis yeast. Mol Cell 2002; 9: 911&#226;&#8364;&#8220;917. <br />     233&#226;&#8364;&#8220;246.                                                                                          35. Dudgeon DD, Zhang N, Ositelu OO, Kim H, Cunningham KW. Nonapoptotic death <br /> 25. Fahrenkrog B, Sauder U, Aebi U. The S. cerevisiae HtrA-like protein Nma111p nuclear              Saccharomyces cerevisiae cells stimulated Hsp90 inhibited calcineurin <br />     serine protease mediates yeast apoptosis. J Cell Sci 2004; 117: 115&#226;&#8364;&#8220;126.                         Cmk2 response endoplasmic reticulum stresses. Eukaryot Cell 2008; 7: <br /> 26. Wissing S, Ludovico P, Herker E, Buttner S, Engelhardt SM, Decker T et al. An AIF                     2037&#226;&#8364;&#8220;2051. <br />     orthologue regulates apoptosis yeast. J Cell Biol 2004; 166: 969&#226;&#8364;&#8220;974.                          36. Enoksson M, Salvesen GS. Metacaspases caspases &#226;&#8364;&#8220; doubt. Cell Death <br /> 27. Li W, Sun L, Liang Q, Wang J, Mo W, Zhou B. Yeast AMID homologue Ndi1p displays                       Differ 2010; 17: 1221. <br />     respiration-restricted apoptotic activity involved chronological aging. Mol Biol Cell   37. Wysocki R, Kron SJ. Yeast cell death DNA damage arrest independent <br />     2006; 17: 1802&#226;&#8364;&#8220;1811.                                                                                  caspase reactive oxygen species. J Cell Biol 2004; 166: 311&#226;&#8364;&#8220;316. <br /> 28. Rinnerthaler M, Jarolim S, Heeren G, Palle E, Perju S, Klinger H et al. MMI1 (YKL056c,            38. Hardwick JM, Cheng WC. Mitochondrial programmed cell death pathways yeast. Dev <br />     TMA19), yeast orthologue translationally controlled tumor protein (TCTP)               Cell 2004; 7: 630&#226;&#8364;&#8220;632. <br />     apoptotic functions interacts microtubules mitochondria. Biochim                39. Fadeel B, Xue D, Kagan V. Programmed cell clearance: molecular regulation the <br />     Biophys Acta 2006; 1757: 631&#226;&#8364;&#8220;638.                                                                     elimination apoptotic cell corpses role resolution inflammation. Biochem <br /> 29. Qiu J, Yoon JH, Shen B. Search apoptotic nucleases yeast: role Tat-D nuclease            Biophys Res Commun 2010; 396: 7&#226;&#8364;&#8220;10. <br />     apoptotic DNA degradation. J Biol Chem 2005; 280: 15370&#226;&#8364;&#8220;15379.                                    40. Teng X, Hardwick JM. Reliable method detection programmed cell death yeast. <br /> 30. Ligr M, Velten I, Frohlich E, Madeo F, Ledig M, Frohlich KU et al. The proteasomal                    Methods Mol Biol 2009; 559: 335&#226;&#8364;&#8220;342. <br />     substrate Stm1 participates apoptosis-like cell death yeast. Mol Biol Cell 2001; 12: <br />     2422&#226;&#8364;&#8220;2432. <br /> 31. Cheng WC, Leach KM, Hardwick JM. Mitochondrial death pathways yeast                                        Cell Death Disease open-access journal <br />     mammalian cells. Biochim Biophys Acta 2008; 1783: 1272&#226;&#8364;&#8220;1279.                                                      published Nature Publishing Group. This work is <br /> 32. Cheng EH, Levine B, Boise LH, Thompson CB, Hardwick JM. Bax-independent inhibition             licensed Creative Commons Attribution-Noncommercial-No <br />     apoptosis Bcl-XL. Nature 1996; 379: 55 <br /> </body></html>