<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                    GBE <br />  <br /> Natural Selection Operons Depends Genome Size <br /> Pablo A. Nun&#203;&#339;ez1,*,y, He&#194;&#180;ctor Romero2,y, Marisa D. Farber1, Eduardo P.C. Rocha3,4 <br /> 1 <br />     Instituto Biotecnolog&#196;&#177;&#194;&#180;a, Instituto Nacional Tecnolog&#196;&#177;&#194;&#180;a Agropecuaria (CICVyA-INTA), Buenos Aires, Argentina <br /> 2 <br />  Laboratorio Organizacio&#194;&#180;n y Evolucio&#194;&#180;n del Genoma, Dpto. Ecolog&#196;&#177;&#194;&#180;a y Evolucio&#194;&#180;n, Facultad Ciencias/CURE, Universidad la Repu&#194;&#180;blica, <br /> Uruguay <br /> 3 <br />     Institut Pasteur, Microbial Evolutionary Genomics, Paris, France <br /> 4 <br />     CNRS UMR3525, Paris, France <br /> y <br />  These authors contributed equally work. <br /> *Corresponding author: E-mail: pb.nunez@gmail.com. <br /> Accepted: October 31, 2012 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> In prokaryotes, genome size associated metabolic versatility, regulatory complexity, effective population size, horizontal <br /> transfer rates. We analyzed covariation genome size operon conservation assess evolutionary models of <br /> operon formation maintenance. In agreement previous results, intraoperonic pairs essential highly expressed genes <br /> are conserved. Interestingly, intraoperonic pairs genes conserved encode proteins similar cell <br /> concentrations, suggesting role cotranscription diminishing cost waste shortfall gene expression. Larger genomes <br /> have fewer smaller operons conserved. Importantly, lower conservation larger genomes observed all <br /> classes operons terms gene expression, essentiality, balanced protein concentration. We reached similar conclusions <br /> in independent analyses major bacterial clades (a- b-Proteobacteria Firmicutes). Operon conservation inversely <br /> correlated abundance transcription factors genome controlled genome size. This suggests negative <br /> association complexity genetic networks operon conservation. These results genome size and/or its <br /> proxies key determinants intensity natural selection operon organization. Our data fit better evolutionary models <br /> based advantage coregulation based genetic linkage stochastic gene expression. We suggest larger <br /> genomes highly complex genetic networks transcription factors endure weaker selection operons smaller <br /> genomes fewer alternative tools genetic regulation. <br /> Key words: operons, prokaryotes, evolution. <br />  <br />  <br />                                                                                                  Levy 1955; Selinger et al. 2003; Maier et al. 2009; Yus et al. <br /> Introduction                                                                                     2009). This leads poor correlations transcription <br /> Most genes genomes prokaryotes expressed                                        rates protein concentrations (Maier et al. 2009). Never- <br /> polycistronic units called operons. The operon model pub-                                    theless, genes operon expressed sim- <br /> lished 50 years ago key role                                          ilar rates random pairs genes (Sabatti et al. 2002; Price <br /> development molecular biology highlighted impor-                                    et al. 2006). <br /> tance gene expression regulation (Jacob Monod 1961).                                          Operons highly enriched genes encoding related <br /> Operons start transcription promoter, include                                     functions. Accordingly, operons include genes encoding <br /> two dozen genes (average ~3 genes),                                          enzymes consecutive steps metabolic pathways (Zaslaver <br /> end transcription terminator (Zheng et al. 2002; Lawrence                                   et al. 2006) genes encoding interacting proteins (Mushe- <br /> 2003). The presence alternative promoters attenuators                                     gian Koonin 1996; Huynen et al. 2000). Conservation a <br /> can lead heterogeneous transcription rates operons.                                    gene operon strong indication functional <br /> Furthermore, mRNA proteins prokaryotes                                          neighborhood used make functional inferences <br /> different half-lifes (7 min ~24 h, respectively,                                     (Overbeek et al. 1999; Moreno-Hagelsieb Janga 2008). <br /> Escherichia coli Mycoplasma pneumoniae; Koch                                             Proteins participate processes. Hence, the <br />  <br />  <br />  <br /> &#195;&#376; The Author(s) 2013. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), permits unrestricted reuse, <br /> distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br /> 2242 Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013 <br />  Selection Operons Depends Genome Size                                                                           GBE <br />  <br /> presence gene operon, closeness          selfish model explain accessory <br /> operons, reflects compromise different pro-           genes high rates transfer loss in <br /> cesses (Yin et al. 2010). Operon structure key variable   operons. <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> understand regulation gene expression prokary-             Recent works suggest noise minimization drives <br /> otes. Operons conserved expected given           operon formation operon organization (Swain <br /> rearrangement rates (de Daruvar et al. 2002; Rocha 2006a;           et al. 2002; Swain 2004; Sneppen et al. 2010; Ray and <br /> Moreno-Hagelsieb Janga 2008), presumably             Igoshin 2012; stochastic expression models). In these <br /> the advantages genetic organization (Lathe et al.           models, cotranscription lowers cost stochastic gene <br /> 2000; Omelchenko et al. 2003; Price et al. 2006).                   expression synchronizes different components <br />     A number models aim explaining advantages          functional module minimizing shortfall or <br /> the organization genes operons. Some models propose           waste. Translational coupling, interdependence trans- <br /> that tight genetic linkage favors physical clustering func-      lational efficiency genes encoded operon, <br /> tional neighbor coevolving genes (genetic linkage models).          favor similar expression levels proteins ex- <br /> Indeed, coadaptive changes interacting genes            pressed operon (Lovdok et al. 2009). Noise <br /> likely broken recombination genes closely           gene expression decreases rapidly increasing level of <br /> spaced (recombination model; Stahl Murray 1966). It          expression (Elowitz et al. 2002) particularly rel- <br /> unclear hypothesis compatible typically low     evant lowly expressed (LE) genes small cells the <br /> recombination rates bacteria (Vos et al. 2009),        ones bacteria. Cotranscription places series genes <br /> with short size recombination tracts (typically smaller      control single regulatory region. This single <br /> than gene; Kennemann et al. 2011). Tight linkage          region efficiently selected adaptive <br /> genes strong selection diminish deleteri-      changes&#226;&#8364;&#8221;relative set promoters required regulate <br /> ous effects large genetic deletions, frequent     transcription monocystronic units&#226;&#8364;&#8221;favoring efficient <br /> in Prokaryotes, reducing pleiotropic effects pro-      regulation expression levels (Price, Huang, et al. 2005; pro- <br /> tecting functions strong selection (persistence          moter sharing model). Genomes structured operons have <br /> model; Fang et al. 2008). These models plausible,           fewer target sites deleterious mutations regulatory re- <br /> the formation operons requires additional explanation       gions. Hence, operons result selection minimize <br /> because genes cotranscribed           deleterious impact mutations. Accordingly, has <br /> tight linkage. As example, yeast Saccharomyces cer-          proposed reduction effective population sizes, as <br /> evisiae shows coevolution gene order recombination           observed eukaryotes, leads operon disruption (drift <br /> rates leading linkage essential genes               model; Lynch 2006). This highlights expected positive as- <br /> though practically lacks operons (Pal Hurst 2003).           sociation effective population size, effect <br />     Horizontal gene transfer drives expansion gene       efficiency natural selection, operon conservation. <br /> repertoires prokaryotes (Ochman et al. 2000; Gogarten                The study formation maintenance operons is <br /> et al. 2002; Treangen Rocha 2011). Genes encoding func-         important establish evolutionary mechanisms shaping <br /> tions constant selection&#226;&#8364;&#8221;persistent genes&#226;&#8364;&#8221;are stably          structure prokaryotic genomes regulatory net- <br /> maintained genomes long periods time. This trend       works. Unfortunately, direct test models requires <br /> particularly strong genes encoding essential functions.         quantification parameters effective population <br /> Inversely, accessory genes frequently gained lost           sizes, rates horizontal transfer, selective coefficients for <br /> (Medini et al. 2005). According selfish model, operons       operons formation change. These notoriously difficult <br /> form persist facilitate horizontal transfer    estimate prokaryotes. One variable useful <br /> of coregulated functional modules (Lawrence Roth 1996).         framework genome size, bacteria varies <br /> Such set functionally neighbor, coregulated genes      0.14 Mb 13 Mb (Schneiker et al. 2007; <br /> likely functional new genetic background      McCutcheon von Dohlen 2011). Bacterial genomes are <br /> be kept natural selection new genome. However,            tightly packed genes. Hence, variations genome size <br /> essential genes clustered average gene        correspond proportional variations number genes <br /> are operons (Pal Hurst 2004). Accordingly,        genome (Mira et al. 2001). These variations affect the <br /> new operons overrepresent horizontally transferred           classes functional repertoires cell diverse levels <br /> genes include essential persistent genes (Price,       (Boussau et al. 2004). Smaller genomes tend correspond to <br /> Alm, et al. 2005). Interestingly, frequently clustered     obligatory mutualists pathogens arise reduction of <br /> genes genomes present nearly    larger genomes free-living bacteria (Moran 2003; Klasson <br /> genomes clade (Fang et al. 2008). Thus, domains         Andersson 2004; Moya et al. 2009). Large bacterial ge- <br /> application selfish model linkage-based        nomes correspond species able tackle large <br /> models different. The apply genes      number environmental stimuli encode <br /> that persist long periods time prokaryotic lineages       complex developmental processes multicellularity <br />  <br />  <br /> Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013                           2243 <br />  Nun&#203;&#339;ez et al.                                                                                                        GBE <br />  <br /> (Schneiker et al. 2007). Of particular relevance evolu-      analyzed general features operon organization in <br /> tion operons observation number tran-          clades. Next, compared operon <br /> scription factors increases linearly genome size      structure genome E. coli. As analyzed <br /> (van Nimwegen 2003; Konstantinidis Tiedje 2004). Large           clade separately, E. coli outgroup relative to <br /> bacterial genomes encode transcription factors,         genomes. Finally, studied variation operon <br /> many smallest genomes encode (Minezaki et al.            conservation function genome size relevant bio- <br /> 2005). Genome size correlated strength           logical traits essentiality, gene expression protein <br /> purifying selection suggesting proxy effective      concentration. <br /> population size (Kuo et al. 2009). Finally, genome size net <br /> result accretion events, notably horizontal gene transfer, <br />                                                                      Materials Methods <br /> and genetic deletions (Lawrence et al. 2001). Hence, larger <br /> genomes expected acquired genes horizontal            Data <br /> transfer higher rates small genomes.                    Complete genome sequences annotations down- <br />     As genome size correlated number genes            loaded NCBI GenBank (ftp://ftp.ncbi.nih.gov, last <br /> encoding transcription factors, effective population size       accessed November 20, 2013; supplementary table S1, <br /> and frequency horizontal gene transfer, key      Supplementary Material online). We excluded pseudogenes <br /> amenable variable study evolution operons. Yet,            genes internal in-frame stop codons. All pre- <br /> previous results showed equivocal trends. A seminal analysis         dicted operons organisms downloaded <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> of 26 bacterial genomes showed negative correlation be-            Prokaryotic Operon DataBase (ProOpDB; http://operons. <br /> tween operon genome sizes (Cherry 2003). Accordingly,            ibt.unam.mx/OperonPredictor/, accessed November 20, <br /> the small genomes Buchnera aphidicola M. pneumo-              2013), claims accurate method available <br /> niae long operons (Guell et al. 2009; Yus et al. 2009;          (Taboada et al. 2012). As different operon predictions may <br /> Brinza et al. 2010). However, small genomes, exam-         differ number cases (Brouwer et al. 2008), also <br /> ple, Borrelia spp., short operons (Zheng et al. 2002).          calculated results using operons predicted the <br /> Operon detection methods use, information,             Database prOkaryotic OpeRons (DOOR) http://csbl1. <br /> the <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> intergenic spacers distinguish intra-       bmb.uga.edu/OperonDB (last accessed November 20, 2013; <br /> and interoperonic spacers (Salgado et al. 2000; Chuang et al.        Mao et al. 2009; supplementary text S1, Supplementary <br /> 2012). Hence, use number operons proxy         Material online). We analyzed Escherichia coli K-12 <br /> selection operon conservation affected            MG1655, 102 a-Proteobacteria, 63 b-Proteobacteria, and <br /> covariation genome size length intergenic              124 Firmicutes. To avoid overrepresentation highly similar <br /> spacers operon detection state-of-the-art            organisms, 97% 16SrRNA identity clus- <br /> methods. Pairs intraoperonic genes con-             tered largest genome selected study. We <br /> served pairs interoperonic genes (Rocha 2006a). A <br />                                                                      did include Tenericutes, basal branch Firmicutes, in <br /> methodological artifact leading spuriously large operons <br />                                                                      study genomes small evolve <br /> in compact genomes lead lower operon conservation <br />                                                                      fast. Their inclusion lower accuracy method <br /> in genomes false putative &#226;&#8364;&#339;operons&#226;&#8364;? are <br />                                                                      detect homology affect results. For three <br /> less likely conserved. As result, errors operon <br />                                                                      clades, included study species complete <br /> annotation compact genomes lead overestima- <br />                                                                      public annotated genome sequences were <br /> tion selection acting large operons ge- <br />                                                                      whole-genome operon predictions databases. <br /> nomes, lead underestimation operon <br /> conservation. Operon conservation species a <br /> more accurate natural measure selection imposed           Identification Orthologs <br /> on operons simple traits operon length        We identified genome genes ortholo- <br /> operon number. We decided analyze operon con-           gous ones E. coli K-12 MG1655 using reciprocal best <br /> servation function genome size.                              hits. For gene, conducted FASTA search the <br />     To examine organization distribution operons,         genome vice versa. We retained best hits <br /> we analyzed different clades: a-proteobacteria,                gene. For hits, constructed exact pair- <br /> b-proteobacteria, Firmicutes. Analyzing different clades         wise alignments using BLOSUM62 matrix the <br /> enabled inspect independent processes genomic evo-          Needleman&#226;&#8364;&#8220;Wunsch algorithm did penalize <br /> lution. These clades, particular, include sequenced          gaps edge smaller sequences, is, end-gap free <br /> genomes broad range genome sizes (from: 0.14             alignment following (Erickson Sellers 1983). We consid- <br /> 9.10 Mb a-proteobacteria; 0.2 9.73 Mb b-proteobac-          ered putative orthologs reciprocal best hits more <br /> teria; 1.42 7.18 Mb Firmicutes; supplementary              38% similarity 30% difference protein <br /> fig. S1, Supplementary Material online). In study,           length. <br />  <br />  <br /> 2244 Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013 <br />  Selection Operons Depends Genome Size                                                                             GBE <br />  <br /> Operon Gene Pairs                                                      obtained different experiments normalized. For <br /> The operon structure given genome analyzed               this, selected genes assayed experi- <br /> succession intraoperon gene pairs (OGP). Starting          ments (ubiquitous genes). We computed average values <br /> operon prediction data given genome, OGP                 genes experiment. If experiment A the <br /> constructed pair adjacent genes                 average XA experiment B 1.3*XA, 1.3 the <br /> operon (there [k   1] adjacent gene pairs k-gene             multiplicative factor applied values A so <br /> operon). For example, set adjacent genes ABC              genes compared ones B. The <br /> decomposed pairs: AB BC. Under null model                 ratio averages experiments gives the <br /> of neutral evolution operons, individual OGPs               multiplicative factors allow normalize experi- <br /> statistically independent, disruption adjacency         ments compare directly. In earlier <br /> between B C affect adjacency A            example, gene z assayed A B normalized value <br /> B. Deletion B leads exclusion pairs fur-         (zA + zB/1.3)/2. To estimate mRNA levels OGP, <br /> ther analysis. We test check counting gene            averaged normalized values genes. Then, we <br /> twice&#226;&#8364;&#8221;its pairs&#226;&#8364;&#8221;is affecting results se-           used median distribution (supplementary fig. S2, <br /> lection model disruption operon AB                 Supplementary Material online) divide set genes in <br /> might lead decrease selection BC. We                equal size subsets highly expressed (HE) LE genes. <br /> complementary set operon conservation analyses <br /> (below) data set gene counted                Essentiality Data <br /> once (one pairs randomly discarded). Despite             We classified genes essential nonessential using E. coli <br /> fewer data points (decreasing power test),               data (Baba et al. 2006). An intraoperonic gene pair con- <br /> find high correlations analysis orig-        sidered essential (EE) genes essential nones- <br /> inal restricted data sets (all correlations larger 0.9,   sential (NN) genes nonessential. <br /> see supplementary table S2 [Supplementary Material online] <br /> for comparison). The analysis independent OGP also <br /> avoids overrepresenting large operons. For example, ana-            Protein Level Analysis <br /> lyze OGP 3-gene operons 9 OGP 10-gene op-                To estimate E. coli K-12 MG1655 protein levels cell, we <br /> erons. If pairwise analyses operon done,           used proteomic experimental data sets (Lopez- <br /> there comparisons 3-gene operons 45             Campistrous et al. 2005; Lu et al. 2007; Masuda et al. <br /> comparisons 10-gene operons. Our method assures               2009). Data obtained different experiments nor- <br /> only OGP affected given rearrangement inde-              malized relative average values ones the <br /> pendently operon size.                                              expression levels (discussed earlier). The relative difference <br />                                                                        ( AB) concentrations proteins encoded two <br /> Operon Conservation                                                    intraoperonic genes A (of concentration CA) B (of concen- <br /> We used conservation OGP measure operon                 tration CB) calculated as: <br /> conservation given focal genome &#226;&#8364;&#339;A&#226;&#8364;? E. coli                                AB &#194;&#188; j&#195;&#176;CA   CB &#195;&#382;j=&#195;&#176;CA +CB &#195;&#382;; <br /> K-12 MG1655. First, intra-OGPs E. coli K-12 <br />                                                                        We used median value   group proteins two <br /> MG1655, selected having orthologs genome A <br />                                                                        classes respectively high low concentration (supple- <br /> (OGPA). We identified intra-OGPs the <br />                                                                        mentary fig. S3, Supplementary Material online). Pairs of <br /> genome A orthologs E. coli K-12 <br />                                                                        intraoperonic genes belonging class concen- <br /> MG1655 operon (OGPA,EC). Finally, <br />                                                                        tration (both high low) marked having bal- <br /> we calculated operon conservation index A (OCIA) as <br />                                                                        anced protein concentrations (BAL). The remaining pairs were <br /> follows: <br />                                                                        marked having unbalanced protein concentrations (UNB). <br />                    OCIA &#194;&#188; #OGPA, EC =#OGPA <br /> We computed OCI values subsets pairs genes              Analysis Transcription Factors <br /> according variables essentiality, gene      Information predicted transcription factors each <br /> expression, protein concentration (supplementary table             genome downloaded DBD transcription factor <br /> S1, Supplementary Material online).                                    database (Charoensawan et al. 2013). We analyzed all <br />                                                                        organisms study predictions <br /> Analysis Expression Levels                                          available transcription factors: 49 a-proteobacteria, 35 <br /> To estimate E. coli K-12 MG1655 mRNA levels cell,               b-proteobacteria, 59 Firmicutes. We included anal- <br /> used public transcriptomic data sets (Allen et al. 2003;          ysis nonredundant transcription factor protein families <br /> Corbin et al. 2003; Covert et al. 2004; Lu et al. 2007). Data          using UCLUST threshold 80% (Edgar 2010). <br />  <br />  <br /> Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013                            2245 <br />  Nun&#203;&#339;ez et al.                                                                                                      GBE <br />  <br /> Phylogenetic-Independent Contrasts                                     The number genes number operons ge- <br /> For clade (plus E. coli K-12 MG1655), built super-       nomes grows linearly genome size (supplementary fig. <br /> matrix concatenating aligned protein sequences          S1, Supplementary Material online). This expected given the <br /> core genome (described earlier persistence analysis).      high coding density genomes prokaryotes (Mira et al. <br /> The phylogenetic trees clades inferred      2001). We group a-protebacteria shows a <br /> this supermatrix maximum likelihood algorithms using            singular behavior plots regarding coding densities <br /> PHYML (Guindon Gascuel 2003) model                    (supplementary fig. S4, Supplementary Material online). <br /> LG +  (8) default parameters. The trees rooted            The group consists organisms obligate intracel- <br /> using E. coli K-12 MG1655 outgroup. For variable        lular lifestyle&#226;&#8364;&#8221;either parasites endosymbionts the <br /> of interest, carried phylogenetic-independent con-      order Rickettsiales&#226;&#8364;&#8221;ongoing extensive pseudogenization <br /> trast (PIC) analysis (Felsenstein 1985) using package ape      (Andersson Andersson 2001). Many genomes <br /> (Paradis et al. 2004) R software (R Development Core           large intergenic regions. As mentioned earlier, inter- <br /> Team 2011). The analysis contrasts showed               genic distances key variables used delimit <br /> clades systematic outliers, caused long internal           operons. To avoid spurious definitions operons caused <br /> branches tree. To include points analysis      extensive pseudogenization, excluded group of <br /> without giving unwarranted weight, used nonpara-           21 genomes a-proteobacteria (supplementary table S5, <br /> metric methods (Spearman rank) examine correlation          Supplementary Material online) analyses. This <br /> between contrasts.                                                 qualitative impact results (supplementary <br />                                                                    fig. S5, Supplementary Material online). We significant <br /> Statistical Analysis                                               negative associations genome size fraction <br />                                                                    genes operons average operon length in <br /> Correlations analyzed nonparametric Spearman rho <br />                                                                    clades (fig. 1A&#226;&#8364;&#8220;F). We pervasive strong <br /> association measure. The statistical differences sub- <br /> sets (e.g., HE vs. LE) assessed t-paired tests. The        positive correlations coding density a <br /> Spearman partial correlation test used analyze re-      genome average number genes operons <br /> lation number transcription factors OCI         (fig. 1G&#226;&#8364;&#8220;I). This shows clear associations genome <br /> while controlling genome size. The tests difference be-     size, operon organization coding density. Notably, <br /> tween correlation coefficients computed              larger genomes tend relatively larger intergenic re- <br /> Fisher&#226;&#8364;&#8482;s Z-transformation (Kvam Vidakovic 2007). All sta-      gions fewer shorter operons. This caused <br /> tistics computed using R (R Development Core Team             stronger selection operons smaller genomes simply <br /> 2011).                                                             smaller intergenic distances operons. <br />                                                                    To distinguish hypotheses, studied <br />                                                                    operon conservation. <br /> Results <br /> Associations General Features Operon <br /> Organization Genome Features                                   Operon Conservation Genome Size <br /> We used independent databases operon predictions,           We computed genome OCI (see Materials and <br /> ProOpDB DOOR, class pairs adjacent genes in-          Methods). OCI fraction intraoperonic pairs adjacent <br /> traoperonic interoperonic (see Materials Methods).          genes genome having orthologs operon <br /> These analyses 124 genomes Firmicutes,            E. coli. OCI values higher b-proteobacteria <br /> 102 a- 63 b-Proteobacteria (one genome spe-          (median &#194;&#188; 0.57) expected clade closest <br /> cies). The analysis operon databases gave concordant       E. coli. The clades lower medians re- <br /> results present main text results con-      spective ranks&#226;&#8364;&#8221;Firmicutes (median &#194;&#188; 0.50) a-proteobac- <br /> cerning ProOpDB, reports highest accuracy                teria (median &#194;&#188; 0.46)&#226;&#8364;&#8221;are opposite expected given <br /> (Taboada et al. 2012; supplementary text S1 [Supplemen-        phylogenetic relatedness. Next, examined association <br /> tary Material online] comparison). Species data set   genome size OCI. The clades showed <br /> are independent statistical instances re-     negative, statistically significant, correlations between <br /> lated common evolutionary history. To correct        operon conservation genome size (fig. 2). Hence, larger <br /> effect, retested correlation analyses using PICs (see       genomes fewer, shorter conserved operons <br /> Materials Methods; Felsenstein 1985; supplementary             smaller genomes. To shed light gene traits <br /> tables S3 S4, Supplementary Material online). For simplic-     shaping operon conservation, introduced analysis <br /> ity, indicate results analyses main text    series biologically relevant variables gene essen- <br /> only qualitatively contradict significance    tiality, protein expression levels, balanced protein <br /> Spearman nonparametric association test.                           concentration. <br />  <br />  <br /> 2246 Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013 <br />  Selection Operons Depends Genome Size                                                                                                GBE <br />  <br />  <br />  <br />  <br />      FIG. 1.&#226;&#8364;&#8221;Association genome size fraction genes operons, length operons density coding sequences. Association <br /> between fraction genes operons (A&#226;&#8364;&#8220;C) length operons function genome size (D&#226;&#8364;&#8220;F). Association fraction genes in <br /> operons density coding sequences (G&#226;&#8364;&#8220;I). Results a-Proteobacteria (A, D, G), b-Proteobacteria (B, E, H), Firmicutes (C, F, I). Operon length <br /> was calculated average number genes operon genome operon predictions. Association variables computed <br /> using nonparametric Spearman rho. <br />  <br />  <br />  <br />  <br /> Effects Essentiality Operon Conservation                                    essential genes. We tested hypothesis that <br /> Essential genes usually defined inactivation                   essentiality plays role association operon <br /> prevents growth rich medium (Baba et al. 2006). The                       conservation genome size. We classed genes essential <br /> operons E. coli overrepresent essential genes                        nonessential using E. coli data (Baba et al. 2006), tested <br /> encoding highly conserved (Pal Hurst 2004;                           pairs essential genes (EE) conserved <br /> Price, Huang, et al. 2005). Small genomes,                          operon pairs nonessential genes (NN). This <br /> shown conserved operons, larger fraction                         true statistically significant clades (t-paired tests, <br />  <br />  <br /> Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013                                                2247 <br />  Nun&#203;&#339;ez et al.                                                                                                                          GBE <br />  <br />  <br />  <br />  <br />    FIG. 2.&#226;&#8364;&#8221;Association OCI genome size. (A) a-Proteobacteria; (B) b-Proteobacteria; (C) Firmicutes. Association variables was <br /> computed using nonparametric Spearman rho. <br />  <br />  <br /> all P &lt; 0.0001; fig. 3A&#226;&#8364;&#8220;C), stronger effects                            Veitia 2004). If cotranscription allows minimization gene <br /> a-Proteobacteria, b-Proteobacteria finally Firmicutes.                 dosage imbalance expect operons overrep- <br /> We tested presence essential genes affected                     resent pairs genes encoding proteins similar cell con- <br /> operon conservation function genome size. Both pairs                   centrations. Indeed, observed clades OGPs <br /> essential nonessential genes tend conserved                   encoding proteins similar concentrations E. coli cells <br /> the operons larger genomes (fig. 3A&#226;&#8364;&#8220;C). Yet, correla-                    conserved OGPs different protein <br /> tion weak nonsignificant. These results                   cell concentrations (see Materials Methods, t-paired tests <br /> show operons essential genes conserved,                      OCI values, P &lt; 0.0001; fig. 5A&#226;&#8364;&#8220;C). Posttranslation mod- <br /> but operons essential nonessen-                    ifications different protein turnovers lead different <br /> tial genes conserved large genomes.                                 protein concentration cell genes expressed at <br />                                                                                 similar levels operon. Our results suggest that <br /> Expression Levels                                                               operons encoding proteins present different <br />                                                                                 concentrations weaker selection. In types of <br /> Highly expressed genes bacteria stronger purify-                   pairs intraoperonic genes, systematic trend for <br /> ing selection higher codon usage bias (Gouy                       lower conservation larger genomes, trends are <br /> and Gautier 1982), encode expensive amino acids (Akashi                    weak measured rho coefficient association <br /> and Gojobori 2002), evolve slower (Rocha Danchin                        statistical significance (fig. 5A&#226;&#8364;&#8220;C). Hence, trends of <br /> 2004). Similarly, gene organization highly expressed genes                   operon conservation genome size similar both <br /> is expected stronger purifying selection                    classes follow general trend weaker conservation <br /> the cost inefficient tuning expression levels highly ex-               larger genomes. <br /> pressed proteins presumably higher. To study effect of <br /> expression levels operon conservation, analyzed pub-                      Transcriptional Regulation Complexity Operon <br /> lished transcriptomic data sets (see Materials Methods).                    Conservation <br /> As expected, pairs HE genes same <br /> operon conserved pairs LE genes                   As mentioned earlier, fraction genes encode tran- <br /> clades (t-paired test OCI values, P &lt; 0.0001; fig. 4A&#226;&#8364;&#8220;C). We                 scription factors increases polynomial genome size. <br /> then investigated dependence operon conservation                      Larger genomes encode complex regulatory networks and <br /> expression levels given genome size. We conser-                      lead decreased selection operons these <br /> vation intraoperonic pairs highly expressed genes                  bacteria reasons. First, easier encode <br />                                                                                 complex genetic network monocystronic units where <br /> systematic negative relationship genome size (fig. 4A&#226;&#8364;&#8220;C). <br />                                                                                 gene expressed independently. Second, presence <br /> Intraoperonic pairs lowly expressed genes consis- <br />                                                                                 large number transcription factors lower pres- <br /> tent patterns clades tend conserved <br />                                                                                 sure place coexpressed genes action same <br /> in larger genomes. <br />                                                                                 promoter. Conversely, small genomes lack genes encod- <br />                                                                                 ing transcription factors, increase selection <br /> Protein Dosage Effects                                                          pressure presence operons. To test hypothesis, <br /> Processes depend proteins sensitive                   analyzed association number transcrip- <br /> to changes relative concentration (Papp et al. 2003;                   tion factors coded genome OCI, while <br />  <br />  <br /> 2248 Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013 <br />  Selection Operons Depends Genome Size                                                                                                 GBE <br />  <br />  <br />  <br />  <br />       FIG. 3.&#226;&#8364;&#8221;Association OCI gene essentiality. Box plots indicate distribution OCI values genome pairs intraoperonic essential <br /> (EE) nonessential genes (NN). The box plots indicate median (central line), 25% 75% percentiles (edges boxes), 1.5 interquartile ranges <br /> (whiskers), outliers (dots). Box plot associated P values correspond paired t-tests means groups differ. Association between <br /> OCI genome size pairs EE (black dots) NN (gray dots) genes computed using nonparametric Spearman rho. For a-Proteobacteria, <br /> the obligate intracellular cluster organisms excluded correlation analysis (supplementary fig. S5, Supplementary Material online). Clades: (A) <br /> a-Proteobacteria, (B) b-Proteobacteria, (C) Firmicutes. <br />  <br />  <br />  <br />  <br />      FIG. 4.&#226;&#8364;&#8221;Association OCI expression levels. Box plots indicate distribution OCI values genome intra-operonic pairs HE LE <br /> genes. The box plots indicate median (central line), 25% 75% percentiles (edges boxes), 1.5 interquartile ranges (whiskers) outliers <br /> (dots). Box plot associated P values correspond paired-t statistical tests means differ. Association OCI genome size pairs HE (black <br /> dots) LE (gray dots) genes computed using nonparametric Spearman rho. For a-Proteobacteria, obligate intracellular cluster organisms <br /> were excluded correlation analysis (supplementary fig. S5, Supplementary Material online). Clades: (A) a-Proteobacteria, (B) b-Proteobacteria, (C) <br /> Firmicutes. (*) The associations b-Proteobacteria nonsignificant control phylogenetic contrasts (supplementary table S4, Supplementary <br /> Material online). <br />  <br />  <br />  <br />  <br />      FIG. 5.&#226;&#8364;&#8221;Association OCI balanced protein concentration levels. Box plots indicate distribution OCI values genome pairs of <br /> genes encoding proteins BAL UNB concentrations cell. The box plots indicate median (central line), 25% 75% percentiles (edges <br /> of boxes), 1.5 interquartile ranges (whiskers) outliers (dots). Box plot associated P values correspond paired t-tests means differ. <br /> Association OCI genome size pairs genes encoding proteins balanced (black dots) unbalanced (gray dots) concentrations in <br /> the cell computed using nonparametric Spearman rho. For a-Proteobacteria, obligate intracellular cluster organisms excluded the <br /> correlation analysis (supplementary fig. S5, Supplementary Material online). Clades: (A) a-Proteobacteria, (B) b-Proteobacteria, (C) Firmicutes. (*) The <br /> associations b-Proteobacteria nonsignificant control phylogenetic contrasts (supplementary table S4, Supplementary Material online). <br />  <br />  <br />  <br />  <br /> Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013                                                 2249 <br />  Nun&#203;&#339;ez et al.                                                                                                                         GBE <br />  <br /> controlling genome size (see Materials Methods). The                   Tiedje 2004), frequency horizontal gene trans- <br /> number transcription factors genome OCI                     fer (Cordero Hogeweg 2009). Genome size corre- <br /> inversely correlated (fig. 6; supplementary fig. S6,                           lated abundance transposable elements (Touchon <br /> Supplementary Material online). They inversely corre-                 Rocha 2007). Abundance transposable elements and <br /> lated controlling genome size clade                          horizontal gene transfer increase rates chromo- <br /> (Spearman partial correlation: a-proteobacteria P &#194;&#188; 0.044, b-                  somal rearrangements lower operon conserva- <br /> proteobacteria P &#194;&#188; 0.087, Firmicutes P &#194;&#188; 0.065; fig. 6).                   tion. On hand, processes genome reduction <br /> Although effects weak, clearly significant                    associated increased rearrangement rates <br /> using Fisher&#226;&#8364;&#8482;s combined probability test (P &#194;&#188; 0.01).                           efficient selection genome organization (Moran <br />                                                                                Mira 2001). Comparative genomics studies observed <br /> Discussion                                                                     conservation gene order strong pu- <br />                                                                                rifying selection, rearrangements having small prob- <br /> In study, confirmed previous suggestions larger <br />                                                                                abilities fixation, strongly correlated genome <br /> genomes tend fewer shorter operons (Cherry <br />                                                                                size (Rocha 2006a). The conservation operons vast <br /> 2003; Yus et al. 2009). Furthermore, showed larger <br />                                                                                evolutionary timescales, despite high genome rearrangement <br /> genomes conserved operons. We confirmed that <br />                                                                                rates frequent horizontal transfer, depen- <br /> operons including highly expressed essential genes are <br /> more conserved (Pal Hurst 2004; Price, Huang, et al.                       dency operon conservation biologically relevant traits <br /> 2005; Fang et al. 2008). Furthermore, showed op-                       suggest natural selection drives evolution operons. <br /> erons traits conserved larger ge-                  As genome size operon structure conservation are <br /> nomes. We exclude possibility different                        anticorrelated, use information assess per- <br /> gene repertoires small large genomes account                        tinence different evolutionary models operon forma- <br /> part effect describe. Yet, lower operon conservation                 tion (tables 1 2). <br /> in larger genomes just caused overabundance                       Linkage models consider operons con- <br /> these genomes nonessential, weakly expressed recently                   served encoding highly persistent (recombination and <br /> acquired genes. Instead, types intraoperonic gene pairs,                persistence models) highly transferred genes (selfish <br /> including ones essential genes nearly ubiq-                  model). The selfish model odds observation of <br /> uitous, conserved larger genomes. Furthermore,                 conserved operons larger genomes (larger genomes <br /> negative correlations OCI genome size sig-                   endure higher rates horizontal transfer) the <br /> nificantly different complementary traits (e.g., HE vs.                higher conservation operons essential genes. <br /> LE) given clade (supplementary table S6, Supplementary                   However, model intended explain best existence <br /> Material online). This suggests correlation                   operons encoding highly transferred genes. It possible <br /> OCI genome size transcends traits. Interestingly,                 emphasis genes homologs E. coli leads <br /> found operons genes expressing proteins similar                   dismissal genes. We divided the <br /> concentrations conserved. This conservation de-                  genes genome categories: without <br /> creases increasing genome size.                                           orthologs E. coli. We higher proportion genes in <br />     Genome size thought correlate number im-                   operons genes orthologs E. coli genes <br /> portant traits effective population size (Kuo et al.                   (t-paired tests, clades P &lt; 0.001). This suggests <br /> 2009), number transcription factors (Konstantinidis                     approach biasing conclusions regarding <br />  <br />  <br />  <br />  <br />      FIG. 6.&#226;&#8364;&#8221;Association number transcription factors, genome size, OCI. (A&#226;&#8364;&#8220;C) For clades (a-Proteobacteria, <br /> b-Proteobacteria, Firmicutes), TF total number inversely correlated OCI. Spearman correlation test (rho P value) indicated. <br />  <br />  <br />  <br />  <br /> 2250 Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013 <br />  Selection Operons Depends Genome Size                                                                                        GBE <br />  <br />         Table 1 <br />         Summary Statistical Spearman Association Tests (Rho) Genome Size OCI Function Different Traits <br />         Data Sets                                          a-Proteobacteria               b-Proteobacteria                 Firmicutes <br />                                                             Rho (P Value)                  Rho (P Value)                  Rho (P Value) <br />         Genome features <br />           % Coding density (%)                            0.24 (P &lt; 0.03)                  0.37 (P &lt; 0.007)              0.34 (P &lt; 0.0001) <br />           % Genes operons                              0.40 (P &lt; 0.0001)                0.58 (P &lt; 0.001)              0.60 (P &lt; 0.002) <br />           Operon length                                   0.28 (P &lt; 0.009)                 0.38 (P &lt; 0.001)              0.46 (P &lt; 0.002) <br />         OCI <br />           Total OGP                                       0.43   (P &lt; 0.0001)              0.48   (P &lt; 0.001)            0.21   (P &lt; 0.01) <br />           EE                                              0.33   (P &lt; 0.002)               0.19   (P &#194;&#188; 0.13)             0.15   (P &lt; 0.0001) <br />           NN                                              0.46   (P &lt; 0.0001)              0.40   (P &lt; 0.001)            0.25   (P &lt; 0.1) <br />           HE                                              0.48   (P &lt; 0.0001)              0.25   (P &lt; 0.04)*            0.20   (P &lt; 0.02) <br />           LE                                              0.25   (P &lt; 0.02)                0.27   (P &lt; 0.03)*            0.10   (P &#194;&#188; 0.19) <br />           BAL                                             0.45   (P &lt; 0.0001)              0.26   (P &lt; 0.03)*            0.39   (P &lt; 0.001) <br />           UNB                                             0.41   (P &lt; 0.0001)              0.25   (P &lt; 0.04)*            0.15   (P &#194;&#188; 0.09) <br />               NOTE.&#226;&#8364;&#8221;NS, nonsigni&#239;&#172;?cant (P &gt; 0.05). <br />               *Nonsigni&#239;&#172;?cance phylogenetic contrast analysis. <br />  <br />  <br />  <br />  <br />         Table 2 <br />         Summary Statistical t-Paired Tests Associated P Values When Testing Difference OCI Genes Pairs <br />         Different Traits (EE vs. NN; HE vs. LE; BAL vs. UNB) <br />         OCI                                a-Proteobacteria                         b-Proteobacteria                         Firmicutes <br />                                           t-Paired (P Value)                       t-Paired (P Value)                    t-Paired (P Value) <br />         EE                                   P &lt; 0.001 #                              P &lt; 0.001 #                           P &lt; 0.001 # <br />         NN <br />         HE                                    P &lt; 0.001#                              P &lt; 0.001 #                           P &lt; 0.001 # <br />         LE <br />         BAL                                  P &lt; 0.001 #                              P &lt; 0.001 #                           P &lt; 0.001 # <br />         UNB <br />               NOTE.&#226;&#8364;&#8221;(#) Arrow direction goes higher lower conservation. <br />  <br />  <br />  <br />  <br /> the selfish model. The linkage models consistent                        concentrations. They consistent selection op- <br /> with high conservation operons essential highly                       erons genes having higher fitness impact (essential <br /> expressed genes. Operons essential genes                           genes; Wang Zhang 2011). However, high conserva- <br /> more conserved recombination model linkage                           tion highly expressed operons does fit models, <br /> was stronger purifying selection pairs                         affected stochastic <br /> genes, plausible observed yeast                          noise gene expression (Swain et al. 2002). These results <br /> (Pal Hurst 2003). The persistence model predicts cluster-                     preclude possibility selection noise minimi- <br /> ing genes deletion deleterious case                    zation operons, example, concerning order of <br /> for essential genes (Fang et al. 2008). As highly expressed                       genes operons colinearity (Kovacs et al. 2009), the <br /> genes strong purifying selection (Rocha 2006b),                         relevance protein interactions lower expression levels <br /> both models predict tight clustering genes. But                          (Ray Igoshin 2012). Yet, cast doubts the <br /> these models predict increased selection operons                     predominance effects operon conservation. <br /> in larger genomes correspond bacteria                          Regulation models suggest operons selected for <br /> higher effective population sizes (more efficient selection)                  intrinsic value cotranscription (or cotranslation) a <br /> more gene transfer (increased selection linkage). Our re-                     gene expression strategy. These models fit our <br /> sults point opposite trends, providing little evidence                  observations. Indeed, expect stronger selection for <br /> favor linkage models.                                                          tight gene regulation expensive genes (highly expressed <br />     Models suggesting operons reduce stochastic noise                        genes) genes strong fitness effects (essential <br /> in gene expression agreement higher con-                          genes). Selection operons containing proteins similar <br /> servation operons encoding proteins balanced                              cell concentrations expected model: operons <br />  <br />  <br /> Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013                                         2251 <br />  Nun&#203;&#339;ez et al.                                                                                                                    GBE <br />  <br /> with unbalanced protein concentrations require unnec-          Merino providing access database helpful Perl <br /> essary transcription abundant protein allow ex-      scripts. <br /> pression abundant one. Hence, operons encoding <br /> proteins similar concentration levels minimize transcrip- <br /> tion/translation costs.                                             Literature Cited <br />     All things equal, decreased selection op-       Akashi H, Gojobori T. 2002. Metabolic efficiency amino acid compo- <br />                                                                         sition proteomes Escherichia coli Bacillus subtilis. Proc Natl <br /> erons larger genomes contradicts available models, in- <br />                                                                         Acad Sci U S A. 99:3695&#226;&#8364;&#8220;3700. <br /> cluding regulatory model, selection           Allen TE, et al. 2003. Genome-scale analysis uses Escherichia <br /> more efficient larger genomes. Yet, difference          coli genome: model-driven analysis heterogeneous data sets. <br /> between large small genomes deeply impacts                 J Bacteriol. 185:6392&#226;&#8364;&#8220;6399. <br /> expectations regulatory model: transcription factors         Andersson JO, Andersson SG. 2001. Pseudogenes, junk DNA, the <br /> are proportionally abundant larger genomes                 dynamics Rickettsia genomes. Mol Biol Evol. 18:829&#226;&#8364;&#8220;839. <br />                                                                     Baba T, et al. 2006. Construction Escherichia coli K-12 in-frame, single- <br /> (Konstantinidis Tiedje 2004). We shown              gene <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio collection. Mol Syst Biol. 2. <br /> is negative association abundance transcrip-           2006.0008. <br /> tion factors operon conservation. This suggests op-        Boussau B, Karlberg EO, Frank AC, Legault BA, Andersson SG. 2004. <br /> erons stronger selection genomes fewer                Computational inference scenarios alpha-proteobacterial <br /> transcription factors. The lack transcription factors smal-       genome evolution. Proc Natl Acad Sci U S A. 101:9722&#226;&#8364;&#8220;9727. <br />                                                                     Brinza L, Calevro F, Duport G, Gaget K, Gautier C, Charles H. 2010. <br /> ler genomes lead increased selection operons, <br />                                                                         Structure dynamics operon map Buchnera aphidicola <br /> as alternative means coregulating gene expression.                sp. strain APS. BMC Genomics 11:666. <br /> Naturally, number transcription factors just       Brouwer RW, Kuipers OP, van Hijum SA. 2008. The relative value of <br /> the variables shaping complexity genetic networks.               operon predictions. Brief Bioinform. 9:367&#226;&#8364;&#8220;375. <br /> These depend number diversity regulatory ele-         Corbin RW, et al. 2003. Toward protein profile Escherichia coli: com- <br />                                                                         parison transcription profile. Proc Natl Acad Sci U S A. 100: <br /> ments elements regulating them, like environmen- <br />                                                                         9232&#226;&#8364;&#8220;9237. <br /> tal sensors. The abundant large        Cordero OX, Hogeweg P. 2009. The impact long-distance horizontal <br /> genomes (Galperin 2006).                                                gene transfer prokaryotic genome size. Proc Natl Acad Sci U S A. <br />     We propose smaller effective population sizes bac-          106:21748&#226;&#8364;&#8220;21753. <br /> teria harboring smaller genomes compensated in-              Covert MW, Knight EM, Reed JL, Herrgard MJ, Palsson BO. 2004. <br />                                                                         Integrating high-throughput computational data elucidates bac- <br /> creased selection operons genomes. This <br />                                                                         terial networks. Nature 429:92&#226;&#8364;&#8220;96. <br /> increased selection pressure operons caused         Charoensawan V, Wilson D, Teichmann SA. 2013. Genomic repertoires of <br /> the fewer alternative ways regulating gene expression           DNA-binding transcription factors tree life. Nucleic Acids <br /> are available bacteria. Inversely, complex            Res. 38:7364&#226;&#8364;&#8220;7377. <br /> regulatory networks large genomes lead lower            Cherry JL. 2003. Genome size operon content. J Theor Biol. 221: <br /> selection cotranscription. This explain              401&#226;&#8364;&#8220;410. <br />                                                                     Chuang LY, Chang HW, Tsai JH, Yang CH. 2012. Features computa- <br /> Eukaryotes, typically large complex regulatory net-          tional operon prediction prokaryotes. Brief Funct Genomics. 11: <br /> works, operons effective popu-              291&#226;&#8364;&#8220;299. <br /> lation sizes large.                                        Daruvar A, Collado-Vides J, Valencia A. 2002. Analysis cellular <br />                                                                         functions Escherichia coli operons conservation Bacillus <br />                                                                         subtilis. J Mol Evol. 55:211&#226;&#8364;&#8220;221. <br /> Supplementary Material                                              Edgar RC. 2010. Search clustering orders magnitude faster than <br />                                                                         BLAST. Bioinformatics 26:2460&#226;&#8364;&#8220;2461. <br /> Supplementary text S1, figures S1&#226;&#8364;&#8220;S7, tables S1&#226;&#8364;&#8220;S6 are <br />                                                                     Elowitz MB, Levine AJ, Siggia ED, Swain PS. 2002. Stochastic gene expres- <br /> available Genome Biology Evolution online (http://               sion single cell. Science 297:1183&#226;&#8364;&#8220;1186. <br /> www.gbe.oxfordjournals.org/).                                       Erickson BW, Sellers PH. 1983. Recognition patterns genetic se- <br />                                                                         quences. In: Sankoff D, Kruskal JB, editors. Time warps, <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span> edits <br />                                                                         macromolecules: theory practice sequence compari- <br /> Acknowledgments                                                         son. Reading (MA): Addison Wesley. p. 55&#226;&#8364;&#8220;91. <br />                                                                     Fang G, Rocha EP, Danchin A. 2008. Persistence drives gene clustering in <br /> This work supported Agencia Nacional de <br />                                                                         bacterial genomes. BMC Genomics 9:4. <br /> Promocio&#194;&#180;n Cient&#196;&#177;&#194;&#180;fica y Te&#194;&#180;cnica Argentina (AGENCIA)            Felsenstein J. 1985. Phylogenies comparative method. Am <br /> grant PICT-2008-0832 M.F., postdoctoral fellowship                 Naturalist 125:1&#226;&#8364;&#8220;15. <br /> from Consejo Nacional Investigaciones Cient&#196;&#177;&#194;&#180;ficas y         Galperin MY. 2006. Structural classification bacterial response regula- <br /> Te&#194;&#180;cnicas (CONICET) P.N., Agencia Nacional                    tors: diversity output domains domain combinations. <br />                                                                         J Bacteriol. 188:4169&#226;&#8364;&#8220;4182. <br /> Investigacio&#194;&#180;n e Innovacio&#194;&#180;n (ANII, MOV_CA_2012) grant to <br />                                                                     Gogarten JP, Doolittle WF, Lawrence JG. 2002. Prokaryotic evolution in <br /> H.R., European Research Council grant EVOMOBILOME                   light gene transfer. Mol Biol Evol. 19:2226&#226;&#8364;&#8220;2238. <br /> grant 281605 E.P.C.R. M.F. member CONICET                Gouy M, Gautier C. 1982. Codon usage bacteria: correlation gene <br /> Researcher&#226;&#8364;&#8482;s Career. The authors grateful Enrique                expressivity. Nucleic Acids Res. 10:7055&#226;&#8364;&#8220;7074. <br />  <br />  <br />  <br /> 2252 Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013 <br />  Selection Operons Depends Genome Size                                                                                              GBE <br />  <br /> Guell M, et al. 2009. Transcriptome complexity genome-reduced             Moran NA, Mira A. 2001. The process genome shrinkage obligate <br />     bacterium. Science 326:1268&#226;&#8364;&#8220;1271.                                               symbiont Buchnera aphidicola. Genome Biol. 2:RESEARCH0054. <br /> Guindon S, Gascuel O. 2003. A simple, fast, accurate algorithm          Moreno-Hagelsieb G, Janga SC. 2008. Operons effect genome <br />     estimate large phylogenies maximum likelihood. Syst Biol. 52:                redundancy deciphering functional relationships using phylogenetic <br />     696&#226;&#8364;&#8220;704.                                                                        profiles. Proteins 70:344&#226;&#8364;&#8220;352. <br /> Huynen M, Snel B, Lathe W 3rd, Bork P. 2000. Predicting protein function       Moya A, Gil R, Latorre A, Pereto J, Pilar Garcillan-Barcia M, la Cruz F. <br />     genomic context: quantitative evaluation qualitative inferences.         2009. Toward minimal bacterial cells: evolution vs. design. FEMS <br />     Genome Res. 10:1204&#226;&#8364;&#8220;1210.                                                       Microbiol Rev. 33:225&#226;&#8364;&#8220;235. <br /> Jacob F, Monod J. 1961. Genetic regulatory mechanisms synthesis         Mushegian AR, Koonin EV. 1996. A minimal gene set cellular life de- <br />     proteins. J Mol Biol. 3:318&#226;&#8364;&#8220;356.                                             rived comparison complete bacterial genomes. Proc Natl Acad <br /> Kennemann L, et al. 2011. Helicobacter pylori genome evolution               Sci U S A. 93:10268&#226;&#8364;&#8220;10273. <br />     human infection. Proc Natl Acad Sci U S A. 108:5033&#226;&#8364;&#8220;5038.                  Ochman H, Lawrence JG, Groisman EA. 2000. Lateral gene transfer and <br /> Klasson L, Andersson SG. 2004. Evolution minimal-gene-sets host-              nature bacterial innovation. Nature 405:299&#226;&#8364;&#8220;304. <br />     dependent bacteria. Trends Microbiol. 12:37&#226;&#8364;&#8220;43.                            Omelchenko MV, Makarova KS, Wolf YI, Rogozin IB, Koonin EV. 2003. <br /> Koch A, Levy H. 1955. Protein turnover growing cultures Escherichia           Evolution mosaic operons horizontal gene transfer gene <br />     coli. J Biol Chem. 217:947&#226;&#8364;&#8220;957.                                                 displacement situ. Genome Biol. 4:R55. <br /> Konstantinidis KT, Tiedje JM. 2004. Trends gene content            Overbeek R, Fonstein M, D&#226;&#8364;&#8482;Souza M, Pusch GD, Maltsev N. 1999. The use <br />     genome size prokaryotic species larger genomes. Proc Natl               gene clusters infer functional coupling. Proc Natl Acad Sci U S A. <br />     Acad Sci U S A. 101:3160&#226;&#8364;&#8220;3165.                                                  96:2896&#226;&#8364;&#8220;2901. <br /> Kovacs K, Hurst LD, Papp B. 2009. Stochasticity protein levels drives       Pal C, Hurst LD. 2003. Evidence co-evolution gene order re- <br />     colinearity gene order metabolic operons Escherichia coli. PLoS        combination rate. Nat Genet. 33:392&#226;&#8364;&#8220;395. <br />     Biol. 7:e1000115.                                                          Pal C, Hurst LD. 2004. Evidence selfish operon theory. Trends <br /> Kuo CH, Moran NA, Ochman H. 2009. The consequences genetic drift                 Genet. 20:232&#226;&#8364;&#8220;234. <br />     bacterial genome complexity. Genome Res. 19:1450&#226;&#8364;&#8220;1454.                 Papp B, Pal C, Hurst LD. 2003. Dosage sensitivity evolution gene <br /> Kvam PH, Vidakovic B. 2007. Nonparametric statistics applications           families yeast. Nature 424:194&#226;&#8364;&#8220;197. <br />     science engineering. Hoboken (NJ): John Wiley &amp; Sons.                  Paradis E, Claude J, Strimmer K. 2004. APE: analyses phylogenetics and <br /> Lathe WC, Snel B, Bork P. 2000. Gene context conservation higher               evolution R language. Bioinformatics 20:289&#226;&#8364;&#8220;290. <br />     order operons. Trends Biochem Sci. 25:474&#226;&#8364;&#8220;479.                        Price MN, Alm EJ, Arkin AP. 2005. Interruptions gene expression drive <br /> Lawrence JG. 2003. Gene organization: selection, selfishness, seren-            highly expressed operons leading strand DNA replication. <br />     dipity. Annu Rev Microbiol. 57:419&#226;&#8364;&#8220;440.                                         Nucleic Acids Res. 33:3224&#226;&#8364;&#8220;3234. <br /> Lawrence JG, Hendrix RW, Casjens S. 2001. Where pseudogenes            Price MN, Arkin AP, Alm EJ. 2006. The life-cycle operons. PLoS Genet. 2: <br />     bacterial genomes? Trends Microbiol. 9:535&#226;&#8364;&#8220;540.                              e96. <br /> Lawrence JG, Roth JR. 1996. Selfish operons: horizontal transfer drive     Price MN, Huang KH, Arkin AP, Alm EJ. 2005. Operon formation driven <br />     evolution gene clusters. Genetics 143:1843&#226;&#8364;&#8220;1860.                         co-regulation horizontal gene transfer. Genome Res. 15: <br /> Lopez-Campistrous A, et al. 2005. Localization, annotation, compar-             809&#226;&#8364;&#8220;819. <br />     ison Escherichia coli K-12 proteome states growth.     R Development Core Team. 2011. R: language environment for <br />     Mol Cell Proteomics. 4:1205&#226;&#8364;&#8220;1209.                                               statistical computing. Vienna (Austria): R Foundation Statistical <br /> Lovdok L, et al. 2009. Role translational coupling robustness bac-         Computing. <br />     terial chemotaxis pathway. PLoS Biol. 7:e1000171.                          Ray JC, Igoshin OA. 2012. Interplay gene expression noise ultra- <br /> Lu P, Vogel C, Wang R, Yao X, Marcotte EM. 2007. Absolute pro-                      sensitive dynamics affects bacterial operon organization. PLoS Comput <br />     tein expression profiling estimates relative contributions               Biol. 8:e1002672. <br />     transcriptional translational regulation. Nat Biotechnol. 25:          Rocha EP. 2006a. Inference analysis relative stability bacterial <br />     117&#226;&#8364;&#8220;124.                                                                        chromosomes. Mol Biol Evol. 23:513&#226;&#8364;&#8220;522. <br /> Lynch M. 2006. Streamlining simplification microbial genome ar-         Rocha EP. 2006. The quest universals protein evolution. Trends <br />     chitecture. Annu Rev Microbiol. 60:327&#226;&#8364;&#8220;349.                                     Genet. 22:412&#226;&#8364;&#8220;416. <br /> Maier T, Guell M, Serrano L. 2009. Correlation mRNA protein          Rocha EP, Danchin A. 2004. An analysis determinants amino <br />     complex biological samples. FEBS Lett. 583:3966&#226;&#8364;&#8220;3973.                           acids substitution rates bacterial proteins. Mol Biol Evol. 21: <br /> Mao F, Dam P, Chou J, Olman V, Xu Y. 2009. DOOR: database                     108&#226;&#8364;&#8220;116. <br />     prokaryotic operons. Nucleic Acids Res. 37:D459&#226;&#8364;&#8220;D463.                      Sabatti C, Rohlin L, Oh MK, Liao JC. 2002. Co-expression <span id='am-162' about='xsp:pattern' typeof='owl:Thing'>pattern</span> from <br /> Masuda T, Saito N, Tomita M, Ishihama Y. 2009. Unbiased quantitation             DNA microarray experiments tool operon prediction. Nucleic <br />     Escherichia coli membrane proteome using phase transfer surfactants.            Acids Res. 30:2886&#226;&#8364;&#8220;2893. <br />     Mol Cell Proteomics. 8:2770&#226;&#8364;&#8220;2777.                                          Salgado H, Moreno-Hagelsieb G, Smith TF, Collado-Vides J. 2000. <br /> McCutcheon JP, von Dohlen CD. 2011. An interdependent metabolic                     Operons Escherichia coli: genomic analyses predictions. Proc <br />     patchwork nested symbiosis mealybugs. Curr Biol. 21:                  Natl Acad Sci U S A. 97:6652&#226;&#8364;&#8220;6657. <br />     1366&#226;&#8364;&#8220;1372.                                                                 Schneiker S, et al. 2007. Complete genome sequence myxobacter- <br /> Medini D, Donati C, Tettelin H, Masignani V, Rappuoli R. 2005. The mi-              ium Sorangium cellulosum. Nat Biotechnol. 25:1281&#226;&#8364;&#8220;1289. <br />     crobial pan-genome. Curr Opin Genet Dev. 15:589&#226;&#8364;&#8220;594.                       Selinger DW, Saxena RM, Cheung KJ, Church GM, Rosenow C. 2003. <br /> Minezaki Y, Homma K, Nishikawa K. 2005. Genome-wide survey tran-                 Global RNA half-life analysis Escherichia coli reveals positional pat- <br />     scription factors prokaryotes reveals bacteria-specific families        terns transcript degradation. Genome Res. 13:216&#226;&#8364;&#8220;223. <br />     archaea. DNA Res. 12:269&#226;&#8364;&#8220;280.                                 Sneppen K, Pedersen S, Krishna S, Dodd I, Semsey S. 2010. Economy of <br /> Mira A, Ochman H, Moran NA. 2001. Deletional bias evolution              operon formation: cotranscription minimizes shortfall protein com- <br />     bacterial genomes. Trends Genet. 17:589&#226;&#8364;&#8220;596.                                    plexes. MBio 1:pii:e00177&#226;&#8364;&#8220;10. <br /> Moran NA. 2003. Tracing evolution gene loss obligate bacterial       Stahl FW, Murray NE. 1966. The evolution gene clusters genetic <br />     symbionts. Curr Opin Microbiol. 6:512&#226;&#8364;&#8220;518.                                      circularity microorganisms. Genetics 53:569&#226;&#8364;&#8220;576. <br />  <br />  <br />  <br /> Genome Biol. Evol. 5(11):2242&#226;&#8364;&#8220;2254. doi:10.1093/gbe/evt174 Advance Access publication November 6, 2013                                               2253 <br />  Nun&#203;&#339;ez et al.                                                                                                                            GBE <br />  <br /> Swain PS. 2004. Efficient attenuation stochasticity gene expression      Wang Z, Zhang J. 2011. Impact gene expression noise organismal <br />     post-transcriptional control. J Mol Biol. 344:965&#226;&#8364;&#8220;976.                 fitness efficacy natural selection. Proc Natl Acad Sci U S A. <br /> Swain PS, Elowitz MB, Siggia ED. 2002. Intrinsic extrinsic contributions       108:E67&#226;&#8364;&#8220;E76. <br />     stochasticity gene expression. Proc Natl Acad Sci U S A. 99:         Yin Y, Zhang H, Olman V, Xu Y. 2010. Genomic arrangement bacterial <br />     12795&#226;&#8364;&#8220;12800.                                                                   operons constrained biological pathways encoded the <br /> Taboada B, Ciria R, Martinez-Guerrero CE, Merino E. 2012. ProOpDB:                 genome. Proc Natl Acad Sci U S A. 107:6310&#226;&#8364;&#8220;6315. <br />     Prokaryotic Operon DataBase. Nucleic Acids Res. 40:D627&#226;&#8364;&#8220;D631.              Yus E, et al. 2009. Impact genome reduction bacterial metabolism <br /> Touchon M, Rocha EP. 2007. Causes insertion sequences abundance              re <br /> </body></html>