<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization Phlebotomus papatasi Peritrophins, <br /> and Role PpPer1 Leishmania major <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>Survival</span> in <br /> its Natural Vector <br /> Iliano V. Coutinho-Abreu, Narinder K. Sharma, Maricela Robles-Murguia, Marcelo Ramalho-Ortigao* <br /> Department Entomology, Kansas State University, Manhattan, Kansas, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The peritrophic matrix (PM) plays key role compartmentalization blood meal barrier pathogens many <br />      disease vectors. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> establish infection sand flies, Leishmania escape endoperitrophic space prevent <br />      excretion remnants blood meal digestion. In spite role played regarding Leishmania survival, little is <br />      known sand fly PM molecular components structural organization. We characterized peritrophins (PpPer1, <br />      PpPer2, PpPer3) Phlebotomus papatasi. PpPer1 PpPer2 display, respectively, chitin-binding <br />      domains (CBDs). PpPer3 hand CBDs, mucin-like domain, putative domain hallmarks of <br />      CBD, changes key amino acids. Temporal spatial expression analyses PpPer1 expressed <br />      specifically female midgut blood feeding. PpPer2 PpPer3 mRNAs constitutively expressed midgut <br />      hindgut, PpPer3 expressed Malpighian tubules. PpPer2 gene expressed in <br />      developmental stages. Interestingly, PpPer1 PpPer3 expression regulated Le. major infection. Recombinant <br />      PpPer1, PpPer2 PpPer3 obtained shown display similar biochemical profiles native; show <br />      PpPer1 PpPer2 able bind chitin. Knockdown PpPer1 led 44% reduction protein, spite of <br />      producing effect percentage infected sand flies, resulted 39% increase parasite load 48 h. Our data <br />      suggest PpPer1 component P. papatasi PM likely involved PM role barrier Le. major <br />      infection. <br />  <br />   Citation: Coutinho-Abreu IV, Sharma NK, Robles-Murguia M, Ramalho-Ortigao M (2013) Characterization Phlebotomus papatasi Peritrophins, Role of <br />   PpPer1 Leishmania major Survival Natural Vector. PLoS Negl Trop Dis 7(3): e2132. doi:10.1371/journal.pntd.0002132 <br />   Editor: Paulo Filemon Pimenta, Fundac&#194;&#184;ao Oswaldo Cruz, Brazil <br />   Received September 12, 2012; Accepted February 9, 2013; Published March 14, 2013 <br />   Copyright: &#195;&#376; 2013 Coutinho-Abreu et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The study described supported Award Number R01AI074691 National Institute Allergy Infectious Diseases. The content solely <br />   responsibility authors does necessarily represent official views National Institute Allergy Infectious Diseases National <br />   Institutes Health. Publication article funded Kansas State University Open Access Publishing Fund. The funders role study <br />   design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: mortigao@ksu.edu <br />  <br />  <br />  <br /> Introduction                                                                            digestion blood meal. It shown an <br />                                                                                         anterior plug prevents premature migration nectomonads to <br />    Leishmaniasis neglected vector-borne disease caused                          anterior midgut [10]. As parasites develop metacyclic <br /> several different species Leishmania [1,2]. Current estimates                     promastigotes, detach midgut migrate <br /> the distribution leishmaniasis worldwide [3] indicate                           foregut cardia (or stomodeal valve area). At <br /> endemic transmission occurs 98 countries, approximate                        cardia, shown Leishmania-secreted chitinase <br /> incidence 500,000 new human cases diagnosed annually,                            damages stomodeal valve preventing normal function, and <br /> and 350 million people risk infected [4]. The                        forcing sand fly regurgitate contents gut it <br /> DALY (or disability adjusted life years) burden leishmaniasis                    attempts blood feed [14,15]. It widely accepted that <br /> 2 million [4].                                                                          regurgitation carries Leishmania skin vertebrate <br />    Leishmania digenetic parasites, developing suitable                         host, principal mechanism parasite transmission. <br /> mammalian host sand fly vector [5]. To date,                           Regarding sand fly PM, earlier findings suggested it <br /> 90 species phlebotomine sand flies proven                               serves barrier Leishmania development [16,17]. These <br /> incriminated vectors Leishmania [6].                                              results supported feeding chitinase inhibitor <br />    In order survive successfully establish infection                   allosamidin P. papatasi showing Leishmania major remained <br /> sand fly, Leishmania overcome barriers (reviewed                           trapped inside thicker PM [7]. These studies revealed a <br /> [2]). First, following ingestion blood meal, transitional                  dual role sand fly PM protecting serving as <br /> stage Leishmania amastigotes survive proteolytic attack                       barrier Leishmania. Altogether, findings demonstrate the <br /> digestive enzymes [7&#226;&#8364;&#8220;10]. Upon developing promastigote                         PM important component sand fly vector competence. <br /> stage, parasites (nectomonads) escape endoperitrophic                             Despite importance, little known molecular <br /> space PM breakdown [7,11] attach midgut                                components sand fly PM [18,19], roles during <br /> epithelia [12,13], cases prevent excretion following                     infection Leishmania. <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                                 1                                   March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                   Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />   Author Summary                                                             org/tools/scanprosite/), Pfam (http://pfam.sanger.ac.uk/search), <br />                                                                              CDD (http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd. <br />   For successful development midgut                      shtml) domain databases. Chitin binding domain (CBD) classifica- <br />   sand fly vector, Leishmania overcome barriers                 tion type-A (CX13&#226;&#8364;&#8220;20CX5&#226;&#8364;&#8220;6CX9&#226;&#8364;&#8220;19CX10&#226;&#8364;&#8220;14CX4&#226;&#8364;&#8220;14C), type-B <br />   imposed vector include digestive proteases                 (CX12&#226;&#8364;&#8220;13CX20&#226;&#8364;&#8220;21CX10CX12CX2CX8CX7&#226;&#8364;&#8220;12C), type-C (CX8&#226;&#8364;&#8220; <br />   secreted midgut following blood meal                   9CX17&#226;&#8364;&#8220;21CX10&#226;&#8364;&#8220;11CX12&#226;&#8364;&#8220;13CX11C) performed visually, follow- <br />   insect, need escape endoperitrophic space,                 ing Consensus sequences described Tellam [23]. The mucin- <br />   attachment midgut epithelia prevent                          like domain amino acid composition assessed using the <br />   excretion remnants blood meal. The sand                    GeneRunner software (http://www.generunner.net/). Predicted <br />   fly peritrophic matrix (PM) constitutes important barrier               heme-regulatory motifs (HRM) visually identified cyste- <br />   establishment Leishmania sand fly <br />                                                                              ine-proline dipeptide [24]. <br />   trapped PM parasites passed <br />   remnants blood meal. Despite                        Multiple sequence alignment peritrophin CBDs per- <br />   role sand fly PM Leishmania development, charac-                     formed ClustalW tool BioEdit package [25]. <br />   terization molecular components assessment                   Alignment adjusted manually remove gaps. The <br />   roles Leishmania lacking. Thereby,                    CBDs P. papatasi peritrophins aligned CBD sequences <br />   performed molecular characterization P.                       identified peritrophins Lutzomyia longipalpis [19,26]. <br />   papatasi peritrophins named PpPer1, PpPer2, PpPer3.                    Alignment performed CBD sequence located <br />   Overall, demonstrated that: (1) PpPer3 displays                       sixth conserved cysteine residues. The L. <br />   putative CBD domain undertaken neo-                        longipalpis peritrophin cDNA sequence identified body <br />   functionalization, (2) PpPer1 PpPer3 genes display                     libraries (NSFM-72d06.q1k; [26]), referred LlPer3 an <br />   differential gene expression Le. major infection;                 ortholog P. papatasi PpPer3. A putative CBD identified <br />   (3) PpPer1 important component                      PpPer3 N-terminal sequence visual inspection and <br />   function P. papatasi PM barrier Le. major                  named Pp3put. Ll3put similarly identified L. <br />   infection.                                                                 longipalpis LlPer3. Peritrophin sequences displaying similarities to <br />                                                                              Pp3put Ll3put CBDs retrieved GenBank and <br />                                                                              aligned sand fly CBDs. <br />    Here, characterized peritrophins, PpPer1, PpPer2,               Phylogenetic analysis performed using Maximum <br /> PpPer3, previously identified midgut P. papatasi [18].             Likelihood method based Whelan Goldman model <br /> PpPer1 PpPer2 likely involved formation PM             [27]. The branch robustness inferred 500 bootstrap <br /> scaffold, suggested expression profiles ability       pseudo-replicates [28]. These analyses carried the <br /> respective recombinant proteins bind exogenous chitin. PpPer3             MEGA5 software [29]. <br /> on hand involved mechanisms related to <br /> protection epithelia, peritrophin displays mucin <br /> domain expressed gut tissues Malpighian                   Sand fly samples, RNA isolation cDNA synthesis <br /> tubules. We investigated role sand fly PM                  All sand flies used P. papatasi (PPIS strain) reared at <br /> barrier Leishmania development. Our results indicate                Department Entomology, Kansas State University, previ- <br /> reduction PpPer1 expression levels leads increase Le.            ously described [30]. For adult flies 3-to-5 day-old insects were <br /> major load P. papatasi. Altogether, results suggest PpPer1       used experiments described below. <br /> plays significant role Le. major development vector             Blood feeding sand flies performed using methods: 1) <br /> midgut.                                                                      direct feeding anesthetized (100 mg/kg ketamine; and <br />                                                                              3 mg/kg xylazine) BALB/c mice; 2) using glass feeders filled <br /> Materials Methods                                                        heat-inactivated mouse blood heat-inactivated mouse <br />                                                                              blood mixed 56106 Le. major amastigotes/ml [30]. Flies that <br /> Ethics statement                                                             fed directly anesthetized mouse used RT- <br />    This study carried strict accordance                  PCR assays. The feeding flies using glass feeders flies <br /> recommendations Guide Care Use                         used real time PCR analyses, flies injected with <br /> Laboratory Animals National Institutes Health. The                 dsRNA. Only fully engorged sand flies used. <br /> use animals study reviewed approved                   Phlebotomus papatasi dissections, RNA isolations, cDNA <br /> Committee Institutional Animal Care Use Kansas                 syntheses performed according [30]. <br /> State University (KSU-IACUC) (Permit Numbers 2747, 2748                     Total RNA obtained various tissues adult females <br /> 2749). All sand fly feedings animals bleeds                  dissected pooled follows. For midguts, tissues <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> performed animals anesthesia, efforts             sugar fed (0 h) dissected combined. Likewise, blood <br /> to minimize suffering.                                                       fed midguts dissected combined the <br />                                                                              following time points: 6 h, 12 h, 24 h, 36 h, 48 h, 72 h, 96 h, <br /> Bioinformatics analyses                                                      120 h, 144 h post blood meal (PBM). Pools adult carcasses, <br />    The cDNA sequences PpPer1, PpPer2, PpPer3                     hindguts, heads plus salivary glands, ovaries, Malpighian <br /> previously identified [18]. Predicted isoelectric points              tubules tissues obtained single sand flies <br /> molecular weights mature proteins obtained using                 dissected 0 h, 6 h, 12 h, 24 h, 36 h, 48 h, 72 h, 96 h, 120 h, <br /> Compute pI/Mw tool [20]. Putative secretory signal peptides             144 h PBM. The pool fat bodies single flies <br /> determined using SignalP 3.0 [21]. Prediction O-linked                    dissected 6 h, 12 h, 24 h, 36 h PBM. RNAs from <br /> glycosylated amino acids carried NetOGlyc 3.1 [22]              developmental stages obtained pools 20 eggs, 10 <br /> while N-linked glycosylation site prediction performed using             L1 larvae, stages L2, L3, L4, pupae. <br /> NetNGlyc 1.0 (http://www.cbs.dtu.dk /services/NetNGlyc/). Pro-               cDNAs obtained RNA samples pooled tissues were <br /> tein domains identified searching Prosite (http://expasy.            used RT-PCR reactions described below. <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                      2                             March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                            Phlebotomus papatasi Peritrophins <br />  <br />  <br />    For real-time PCR (qRT-PCR) assays, P. papatasi midguts                    dissected non blood fed blood fed P. papatasi 24, <br /> were dissected flies fed blood blood plus Le.              48, 72, 96 h PBM homogenized 50 ml PBS. Each midgut <br /> major (glass feeders) 24 h, 48 h, 72 h PBM. cDNAs obtained                 lysate boiled SDS lysis (Invitrogen) buffer 5 min one <br /> from RNA samples representing individual midguts                  midgut equivalent midgut pool separated under <br /> of time points used real time PCR reactions below.             non reducing conditions 4&#226;&#8364;&#8220;12% Bis-Tris NuPAGE gels <br />                                                                                   (Invitrogen). Proteins transferred nitrocellulose and <br /> RT-PCR real time qRT-PCR                                                      incubated 16 h 4uC anti-PpPer1 anti-PpPer3 <br />    PpPer1, PpPer2, PpPer3, b-tubulin cDNAs amplified using               antisera diluted 1:100 TBS-T. Blots washed incubated <br /> primer pairs described Table 1. The expression profiles                anti-mouse conjugated alkaline phosphatase (1:10,000) for <br /> genes obtained 23 amplification cycles PpPer2, 25                  1 h. Blots developed using Western Blue (Promega). <br /> cycles PpPer1 PpPer3, 28 cycles b-tubulin. Reactions <br /> were performed 25 ml total volume, containing 12.5 ml GoTaq                    Recombinant peritrophin expression <br /> Master Mix (Promega, Madison, WI), 1 ml cDNA, 0.5 pmoles                       Mature cDNAs sequences (without signal peptide) corre- <br /> primer, 10.5 ml molecular grade water. Amplification reactions                sponding peritrophins PpPer1, PpPer2, and <br /> were follows: 94uC 3 minutes (min); 23&#226;&#8364;&#8220;28 cycles                   PpPer3 PCR amplified P. papatasi midgut using specific <br /> 94uC 30 seconds (sec), 57&#226;&#8364;&#8220;58uC 1 min, 72uC 30 sec;                primer pair combinations (PpPer1Mat_717F/PpPer1R-His, <br /> a final amplification step 72uC 10 min.                                    PpPer2Mat_219F/PpPer2R-His,              PpPer3Mat_867F/ <br />    Real time quantitative PCR reactions performed                     PpPer3R-His, Table 1), reverse primers <br /> MasterCycler Realplex4 Eppendorf Real-Time PCR (Hamburg,                          containing 66-His tag (Table 1). PCR amplifications were <br /> Germany) using BioRad SyBR green (BioRad, Hercules, CA).                          performed follows: cycles 95uC 3 min, 94uC for <br /> Reactions set described [30]. Amplification conditions                 1 min, 68uC 1 min; cycles 94uC 1 min and <br /> and primer pairs used (Table 1) used RT-PCR                      62uC 1 min; 25 cycles 94uC 1 min, 60uC 1 min, <br /> reactions, total 40 amplification steps                     72uC 1 min. Each PCR product subsequently cloned <br /> performed. As housekeeping control, cDNA corresponding                       purified described [30&#226;&#8364;&#8220;32]. <br /> the S3 protein 40S ribosomal subunit amplified (Table 1).                Recombinant proteins expressed using cells (FreeStyle <br />                                                                                   CHO-S) reagents obtained Invitrogen, according to <br /> Temporal expression native peritrophins P.                                manufacturer protocols. Transfected cells incubated at <br /> papatasi                                                                          37uC (with 8% CO2) gentle shaking (125 rpm). Culture <br />   We assessed expression PpPer1, PpPer2 PpPer3                         supernatants collected 72 h concentrated using <br /> proteins present midgut lysates. Pools containing midguts                 Centricon filters (Millipore) 3 kDa (rPpPer2) 10 kDa <br />  <br />  Table 1. Primers list. <br />  <br />  <br />  Primer Name                              Nucleotide Sequence (59 39)                                                      Annealing Temperature <br />  <br />  PpPER1T7i_2_F                            TAATACGACTCACTATAGGGAGAATGAAGAACGTTGCAGTGAT                                         55uC/65uC <br />  PpPER1T7i_2_R                            TAATACGACTCACTATAGGGAGATTAGTGGTCGAAGCAGACTG                                         55uC/65uC <br />  PpPer1_122F                              CTCATGAAGAGTTTTGCATG                                                                57uC <br />  PpPer1_122R                              GAAACCGTCTTCACAGCTC                                                                 57uC <br />  PpPer2_168F                              5TGCCTGGTTTTCCTGTTC                                                                 58uC <br />  PpPer2_168R                              TCCTTGCACGAAAGTTCCC                                                                 58uC <br />  PpPer3_121F                              ATCTGCCCAGGACCATTAC                                                                 58uC <br />  PpPer3_121R                              AGTCGACTGTAGCGCAATC                                                                 58uC <br />  PpTub_148F                               GCGATGACTCCTTCAACAC                                                                 57uC <br />  PpTub_148R                               GTGATCAATTGTTCGGGATG                                                                57uC <br />  Pp40S_S3_136F                            GGACAGAAATCATCATCATG                                                                57uC <br />  Pp40S_S3_136R                            CCTTTTCAGCGTACAGCTC                                                                 57uC <br />  PpPer1Mat_717F                           GCTCATGAAGAGTTTTGCATG                                                               56uC <br />  PpPer1Mat_717R                           TTAGTGGTCGAAGCAGACTG                                                                56uC <br />  PpPer2Mat_219F                           GCCAATGTTTCTTGCCCACC                                                                56uC <br />  PpPer2Mat_219R                           CTATTTTTGTCCGCCTGGAG                                                                56uC <br />  PpPer3Mat_863F                           GAAGAAGTAGTACCAGGAATTC                                                              56uC <br />  PpPer3Mat_863R                           TCATTTTCCTTGGGAAGATTG                                                               56uC <br />  PpPer1R-His                              TTAGTGGTGATGGTGATGGTGGTCGAAGCAGACTG                                                 * <br />  PpPer2R-His                              CTAGTGGTGATGGTGATGATGTTTTTGTCCGCCTGG                                                * <br />  PpPer3R-His                              TCAGTGGTGATGGTGATGATGTTTTCCTTGGGAAGATTG                                             * <br />  <br />  Primer name, sequence, annealing temperatures primer pairs used dsRNA synthesis real time PCR analyses. <br />  *Described Materials Methods. <br />  doi:10.1371/journal.pntd.0002132.t001 <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                           3                                March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                Phlebotomus papatasi Peritrophins <br />  <br />  <br /> (rPpPer1 rPpPer3) cutoffs. Recombinant rPpPer1                    proteins separated 4&#226;&#8364;&#8220;12% pre-cast Bis-Tris NuPAGE gel <br /> rPpPer2 produced using HEK293 cells purified             (Invitrogen), transferred nitrocellulose, analyzed western <br /> described [33].                                                 blot. <br />  <br /> Purification expressed protein                                         Chitin binding assays <br />     The concentrated supernatants rPpPer1 rPpPer3                    We assessed ability purified 66 His-tagged <br /> recombinant proteins purified Ni-NTA HisTrap              recombinant proteins independently bind colloidal chitin <br /> column (GE Healthcare, Piscataway, NJ). Supernatants                 according [34]. Briefly, micrograms recombinant <br /> filtered 0.22 mm Millex syringe filter (Millipore),         protein mixed colloidal chitin suspended 100 ml <br /> dialyzed overnight PBS 4uC. After dialysis supernatant         10 mM sodium phosphate buffer, pH 8.0, incubated room <br /> was manually injected HisTrap column, previously                   temperature 1 h followed centrifugation 10,0006g. The <br /> equilibrated binding buffer A (20 mM sodium phosphate                supernatant saved unbound protein chitin pellet <br /> buffer, 500 mM sodium chloride, 10 mM imidazole, pH 7.4),             washed 50 ml buffer centrifuged as <br /> fitted HP1100 series HPLC system. Washes elution              indicated (wash w1). Additional washes were <br /> recombinant proteins carried according [33] Different         performed 10 mM sodium phosphate buffer containing 1 <br /> fractions corresponding wash interval exception       M sodium chloride, pH 8.0 (w2) 0.1 M acetic acid (w3), <br /> first 35 minutes collected analyzed Comassie-stained          respectively. The final pellet boiled 50 ml SDS-PAGE <br /> SDS-PAGE Western blot.                                                sample buffer (Invitrogen), centrifuged, supernatant <br />     For purification rPpPer2, gravity flow column used.          collected. Unbound protein, washes, SDS eluted fractions <br /> The concentrated supernatant obtained FreeStyle CHO-S                separated reducing condition 2% betamercaptoetha- <br /> cells filtered washed 5 times PBS using 3 kDa cutoff         nol (b-ME) pre-cast 4&#226;&#8364;&#8220;12% Bis-Tris NuPAGE (Invitrogen). <br /> Centricon filter (Millipore), loaded 1 ml Ni-NTA column          Proteins transferred nitrocellulose analyzed by <br /> in 5 ml syringe (BD Biosciences). The column washed            Western blot. <br /> 15 ml 20 mM sodium phosphate buffer-300 mM sodium chloride- <br /> 20 mM imidazole, eluted 5 ml 20 mM sodium                        Antisera production Western blot <br /> phosphate buffer-300 mM sodium chloride-300 mM imidazole.                    Antisera production performed described previously <br /> The eluted rPpPer2 concentrated (1.5 mg/ml) analyzed              [31,32,35]. The mature sequences PpPer1, PpPer2, and <br /> with Commassie-stained SDS-PAGE Western blot.                         PpPer3 amplified P. papatasi midguts (12 h PBM), using <br />                                                                           primer pairs PpPer1Mat_717F/PpPer1Mat_717R, PpPer2- <br /> Gel filtration chromatography                                             Mat_219F/PpPer2Mat_219R, PpPer3Mat_867F/PpPer3- <br />    Native (from P. papatasi midgut extract) recombinant               Mat_867R (Table 1), respectively. Amplifications follows: <br /> proteins subjected HPLC gel filtration chromatography.            94uC 3 min; 35 cycles 94uC 1 min, 56uC 1 min, and <br /> The P. papatasi midgut extract prepared 10 midguts               72uC 1 min; final extension 72uC 10 min. <br /> dissected 48 h PBM, pooled homogenized 50 ml PBS                   To determine effects dsRNA injection expression <br /> pH 7.4 0.01% TritonX-100 followed centrifugation               PpPer1 P. papatasi midguts, polyclonal anti-PpPer1 specific <br /> 14,0006g. The supernatant collected pellet                antisera (1:50 dilution), Western blot assays were <br /> extracted twice sonication 50 ml PBS pH 6.8, followed         performed according [30]. Densitometry analysis per- <br /> centrifugation above. The resulting supernatants            formed using TotalLab TL100 software (Nonlinear Dynamics, <br /> combined. For recombinant proteins, Ni-NTA purified                   Durham, NC). <br /> rPpPer1, rPpPer2, rPpPer3 used.                                     Western blots performed analyses the <br />    A 7.86300 mm Bio-Sil SEC-250 column (Biorad) fitted                66His-tagged recombinant proteins rPpPer1, rPpPer2, and <br /> onto HP Agilent 1100 series HPLC (Hewlett Packard, Santa                rPpPer3. The concentration recombinant protein was <br /> Clara, CA), protein separately loaded               determined using BCA Protein Assay Kit (Thermo Scientific, <br /> column. The HPLC performed using PBS pH 6.8 isocratic              Rockford, IL) measuring OD 280 nm. Recombinant <br /> flow rate 1 ml minute. Absorbance eluted measured           proteins (purified concentrated FreeStyle CHO-S supernatant) <br /> at 280 nm using HP1100 series variable wavelength detector.           fractionated 4&#226;&#8364;&#8220;12% reducing Bis-Tris NuPAGE pre-cast <br /> One ml fractions HPLC elute (both midgut lysate            gels (Invitrogen). Proteins transferred nitrocellulose filter <br /> recombinant proteins) collected 1.5 ml tubes 10 ml         (Whatman, Dassel, Germany), incubated overnight 4uC with <br /> each fraction blotted nitrocellulose membrane (dot           anti-His antibody (Santa Cruz, Santa Cruz, CA) diluted 1:2,000 or <br /> blot). Dot blot filters incubated corresponding anti-sera         different antisera, followed washes 10 minutes <br /> and anti-His antibodies indicated (Western blot).             TBS-T (TBS buffer 0.1% tween-20). Each blot was <br /> Fractionation dot blots repeated twice                  incubated anti-mouse antibody conjugated alkaline <br /> recombinant protein midgut lysate. Recombinants               phosphatase (Promega) diluted 1:10,000 TBS-T 1 h at <br /> rPpPer1 rPpPer2 obtained HEK-293 cells                 room temperature washed TBS-T indicated above. The <br /> fractionated using HP1100 series HPLC, collected                  protein bands visualized using Western Blue (Promega). <br /> fractions applied dot blots incubated anti-His              Alternatively, Western blots incubated anti-mouse- <br /> or specific antisera, described above.                            HRP conjugated secondary antibody (Promega) diluted 1:10,000, <br />                                                                           detected SuperSignal West Pico Chemiluminescence <br /> N-linked deglycosylation rPpPer2                                       Substrate (Thermo Scientific) chemiluminescence assays. <br />    N-linked deglycosylation carried PNGase F (New <br /> England Biolabs Inc., Ipswich, MA). Briefly, 200 ng purified           Knockdown PpPer1 <br /> rPpPer2 incubated separately 1000, 2000 3000 units             PpPer1 selected studies light mRNA <br /> of PNGase F overnight 37uC 20 ml reaction. The reaction             expression profile (midgut-specific regulated blood feeding). <br /> was stopped addition SDS gel loading buffer (Invitrogen),       Double-strand RNAs synthesized using Megascript RNAi <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                   4                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                     Phlebotomus papatasi Peritrophins <br />  <br />  <br /> kit (Ambion). Synthesis purification dsRNA                  Per3 (EU124607). LlPer3 (AM093395). C. felis. PL1 (AAM21354). <br /> injections sand flies performed according [30]. The                C. quinquefasciatus. conserved hypothetical protein (XP_001864216). <br /> dsRNA targeting PpPer1 (dsPpPer1) PCR amplified                      A. aegypti AaeL_AAEL012651 &#226;&#8364;&#8220; Ae51put (XP_001662775), Aae- <br /> primers PpPER1T7i_2 forward reverse (Table 1). dsGFP                  L_AAEL012645 &#226;&#8364;&#8220; Ae45put (XP_001662776), AeputAae- <br /> used control, described [36]. The effects dsRNA induced              L_AAEL012652 &#226;&#8364;&#8220; Ae52put (XP_001662772). <br /> knockdown assessed real time PCR analyses Western <br /> blot.                                                                         Results <br />    Next, assessed effects PpPer1 knockdown Le. major <br /> development P. papatasi midgut. In case, 80.5 ng           Peritrophins cDNA sequences predicted protein <br /> dsRNA injected intra-thoracically sand fly [30]. After                organization <br /> feeding infectious blood meal, midguts individually                   The complete cDNA sequences P. papatasi PpPer1, PpPer2, and <br /> dissected 48 h 96 h PBM homogenized 30 ml PBS                   PpPer3 previously identified published [18]. <br /> (pH 7.4). Live parasites counted using hemocytometer.                     PpPer1 open read frame (ORF) 792 bp long, encoding a <br />                                                                               protein 263 amino acids, predicted molecular weight of <br /> Statistical analyses                                                          28 kDa mature protein, acidic pI (4.84). Putative N- <br />    Unpaired t-test Mann-Whitney U test used assess                O-linked glycosylation residues N29 T211, respectively, <br /> for statistically significant differences expression profiles          expected add molecular weight secreted <br /> parasite counts data followed normal distribution,              protein. PpPer1 displays predicted signal peptide (amino acid <br /> respectively. Assessment distribution <span id='am-83' about='xsp:pattern' typeof='owl:Thing'>pattern</span> carried           residues 1&#226;&#8364;&#8220;18), suggesting protein secreted midgut <br /> D&#226;&#8364;&#8482;Agostino &amp; Pearson omnibus normality test. Differences                 lumen. Four type-A CBDs (PpPer1 CBD consensus sequence: <br /> considered statistically significant p,0.05. All statistical           CX13&#226;&#8364;&#8220;19CX5CX9&#226;&#8364;&#8220;10CX12CX7C) present mature <br /> assessments performed using GraphPad Prism v.5.0                         protein. In addition, putative HRM identified amino <br /> (GraphPad Software, Inc).                                                     acid residues 182&#226;&#8364;&#8220;183 CBD (Pp1CBD3 residues 44 <br />                                                                               45 Figure 1), residues 209&#226;&#8364;&#8220;210 fourth CBD <br /> Accession numbers                                                             (Pp1CBD4 residues 1 2 Figure 1). <br />   Sequence accession numbers: Sand flies. PpPer1 (Gen Bank                       PpPer2 ORF 270 bp coding predicted 7.8 kDa mature <br /> accession number: EU031912). PpPer2 (EU047543). PpPer3                        protein single type-A CBD (consensus sequence: <br /> (EU045354). LuloPer1 (EU124588). LuloPer2 (EU124602). Lulo-                   CX18CX5CX9CX12CX7C). Predicted N-glycosylation amino <br />  <br />  <br />  <br />  <br /> Figure 1. Multiple sequence alignment. Multiple sequence alignment performed individual CBD domains identified peritrophins <br /> from P. papatasi L. longipalpis. Pp1CBD1, Pp1CBD2, Pp1CBD3, Pp1CBD4 PpPer1 CBDs. Pp2CBD1 single CBD PpPer2. Pp3CBD1 <br /> and Pp3CBD2 CBDs identified PpPer3. Lulo1CBD1, Lulo1CBD2, Lulo1CBD3, Lulo1CBD4 CBDs LuloPer1. Lulo2CBD1 <br /> and Lulo3CBD1 single CBDs LuloPer2 LuloPer3, respectively. Ll3CBD1 Ll3CBD2 CBDs LlPer3. Pp3put Ll3put are <br /> putative domains similar CBDs identified P. papatasi PpPer3 L. longipalpis Ll3Per3. Such putative domain sequences identified in <br /> the N-terminal region C. felis PL1 - CfPl1put; C. quinquefasciatus conserved hypothetical protein - Cq16put; A. aegypti Ae51put, Ae45put, and <br /> Ae52put. The conserved cysteines highlighted grey conserved aromatic amino acids predicted bind chitin shown white with <br /> black highlight; HRM motifs underlined. Conserved amino acid residues displayed exclusively putative CBD domain sequences shown <br /> in red, additional cysteine residues indicated asterisk (*). <br /> doi:10.1371/journal.pntd.0002132.g001 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                       5                               March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                      Phlebotomus papatasi Peritrophins <br />  <br />  <br /> acid residues N19 N77 expected increase                     PpPer2 constitutively expressed sugar (0 h) blood fed guts <br /> molecular weight protein. PpPer2 acidic protein (pI                (Figure 3A). PpPer3 expressed midgut (Figure 3A) in <br /> 4.25), single putative HRM residues 22 23                  hindgut Malpighian tubules (Figure 3B). In spite being <br /> predicted CBD (corresponding residues 1 2 Pp2CBD1                  expressed sugar (0 h) blood fed midguts, PpPer3 mRNA <br /> Figure 1). The presence signal peptide cleavage site                  expression up-regulated 12 h 48 h PBM, <br /> between amino acids A17 A18 suggests protein secreted                somewhat similar PpPer1 expression profile. Among three <br /> into midgut lumen.                                                          P. papatasi peritrophins, PpPer2 expressed larval stages <br />    PpPer3 313 bp CBDs. Unlike PpPer1 PpPer2,                    (Figure 3C) and, comparatively, appeared highest <br /> PpPer3 mucin-like domain rich serine (18.2%), threonine                expression levels peritrophins, according Figure 3A. <br /> (36.4%), proline (12.1%), glutamine (12.1%) residues in <br /> addition type-A CBDs (PpPer3 CBD consensus sequence:                     Temporal expression native PpPer1 PpPer3 P. <br /> CX11CX5CX11&#226;&#8364;&#8220;13CX12CX4&#226;&#8364;&#8220;8C). The predicted molecular weight                       papatasi <br /> of mature PpPer3 32 kDa. Moreover, peritrophin                   PpPer1 PpPer3 secreted P. papatasi midgut following a <br /> neutral-to-basic pI (7.75). As large number residues (281T,                blood meal, easily detected Western blot 48 h PBM <br /> 285T, 286T, 292S, 293T, 294T, 295T, 296T, 300S, 301S, 302S,                     (Figure 4). PpPer3 clearly present midgut lysates dissected <br /> 303T, 304T, 305T, 306T, 307T, 309S, 310S) mucin-                     72 h 96 h PBM. Unfortunately, unable resolve <br /> like domain predicted O-linked glycosylated, PpPer3                   smear visible lysates prepared midguts dissected at <br /> molecular mass expected significantly greater. Two                     24 h PBM non reducing conditions, specific antisera <br /> additional features PpPer3 presence 57-residue long <br />                                                                                 anti-PpPer1 anti-PpPer3 did bind native proteins <br /> linker (Pp3CBD1) second (Pp3CBD2) CBDs, <br />                                                                                 reducing conditions. The smear present 24 h samples is <br /> and N-terminal sequence containing cysteine residues <br />                                                                                 likely cross reaction antisera mouse blood, in <br /> (Pp3put, Figure 1). Although Pp3put sequence displays a <br />                                                                                 view irrelevant findings. Nevertheless, native <br /> type-A CBD signature (CX10CX5CX11CX14CX10C) similar to <br />                                                                                 PpPer1, expression profile accordance mRNA <br /> other CBDs P. papatasi peritrophins, recognized a <br />                                                                                 expression profile observed Figure 3A. <br /> bona fide CBD standard bioinformatics&#226;&#8364;&#8482; tools. Two predicted <br /> HRM identified residues 138&#226;&#8364;&#8220;139 (corresponding to <br /> residues 29 30 Pp3CBD1, Figure 1) 150&#226;&#8364;&#8220;151 (residues                  Expression PpPer1 PpPer3 modulated Le. <br /> 44 45 Pp3CBD1, Figure 1) PpPer3 CBD                 major <br /> single HRM predicted residues 262&#226;&#8364;&#8220;263 (residues 44                      We evaluated effects Le. major infection expression P. <br /> 45 Pp3CBD2) second PpPer3 CBD sequence (Figure 1).                    papatasi peritrophin mRNAs (Figure 5). PpPer1 expression dis- <br />    According multiple sequence alignment P.                      played statistically significant up-regulation (20%) 24 h post- <br /> papatasi L. longipalpis peritrophin CBDs (Figure 1),                infection (Figure 5A). However, statistical difference was <br /> conserved cysteine residues characteristic type-A CBDs                   observed PpPer1 midgut expression later time points (48 h <br /> present. Interestingly, numbers amino acid residues              72 h) following Le. major infection. No difference observed <br /> the second cysteines, fourth fifth                PpPer2 time points assessed (Figure 5B). For <br /> cysteines variable. In addition, aromatic residues Y             PpPer3, mucin-like peritrophin, midgut mRNA levels were <br /> and F corresponding positions 25 26 second                   reduced 25% 24 h 28% 48 h Le. major infection <br /> and cysteines, position 48 fourth fifth               (Figure 5C). No differential PpPer3 expression observed Le. <br /> cysteines detected. Regarding HRM sites, co-                  major infected midguts 72 h post-infection. <br /> localized fourth cysteine residues. <br />    Pp3put, putative CBD domain present N-terminal                      Recombinant proteins <br /> portion P. papatasi PpPer3 peritrophin, displays extra                      Recombinant, 66His-tagged, rPpPer1, rPpPer2, rPpPer3 <br /> cysteine residues positions 18 53, residue insertion             successfully obtained using FreeStyle CHO-S cells <br /> (PY) fourth fifth conserved cysteines (Figure 1).               (Figure 6A). Posttranslational modifications (e.g., glycosylation) <br /> This putative CBD domain displays HRM motifs                        likely responsible increased molecular weight detected <br /> aromatic residues positions 25, 26, 48. Interestingly,             recombinant proteins. Interestingly, mass spec analysis of <br /> insect peritrophins features similar Pp3put                   rPpPer2 indicated major peak 9.2 kDa, minor peak at <br /> identified searching GenBank database putative              18.4 kDa, suggestive dimerization protein (not shown). In <br /> CBD domain P. papatasi.                                                    contrast, Western blot showed presence bands at <br />    A phylogenetic analysis suggests single clade Pp3put           approximately 16 kDa 20-to-24 kDa rPpPer2 (Figure 6A). <br /> Ll3put domains sand flies (P. papatasi L. longipalpis),            The higher molecular bands longer detected following <br /> Ae45put, Ae52put, Ae51put Ae. aegypti, Cq16put                digestion PNGase F indicative N-linked glycosylation <br /> C. quinquefasciatus, CfPL1put C. felis (Figure 2, blue box).       (Figure 6B). Differences predicted (7.8 kDa) the <br /> The phylogenetic analysis highlights elevated conservancy              estimated sizes rPpPer2 using mass spec Western analyses <br /> that exists CBD domains P. papatasi L. longipalpis           presence 66His tag interfering <br /> orthologous peritrophins, reported previously [18]).                         gel migration, non-predicted O-ring glycosylation(s), to <br />                                                                                 incomplete digestion N-linked residues. Similar SDS-PAGE <br /> Peritrophin mRNA expression profiles                                            migration discrepancies observed peritrophin 15 of <br />    The expression profiles PpPer1, PpPer2, PpPer3                   screwworm fly Chrysomya bezziana [37]. <br /> assessed semi-quantitative RT-PCR (Figure 3). PpPer1 mRNA                       The recombinant proteins analyzed gel filtration <br /> expression adult midgut-specific blood-induced (Figure 3A               column. The retention time rPpPer1 matched detected for <br /> and 3B); transcripts detected 12 h 72 h PBM,                   native molecule present P. papatasi midgut lysate. For <br /> with highest levels 48 h PBM. PpPer2 transcripts                    PpPer1, elution midgut lysate occurred fractions <br /> expressed midgut hindgut (Figure 3A 3B).                  6 7 (dot blot 1 Supporting Figure S1A), rPpPer1 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                         6                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                  Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />  <br /> Figure 2. Phylogenetic comparison. Condensed tree depicts putative CBD domains single branch (blue shadow box), displaying <br /> strong bootstrap support (84%). All branches CBDs orthologs sand fly peritrophins. Filled circles, filled triangles, open <br /> square indicate sand fly, mosquito, flea peritrophins, respectively. <br /> doi:10.1371/journal.pntd.0002132.g002 <br />  <br /> was eluted fractions 6&#226;&#8364;&#8220;8 (Supp. Figure S1B, dot blot). For             (washes 1&#226;&#8364;&#8220;3). Accordingly, rPpPer1 (Figure 7A) rPpPer2 <br /> rPpPer2, protein eluted mainly fractions 11&#226;&#8364;&#8220;13 (Supp.            (Figure 7B) detected boiling presence of <br /> Figure S1C, dot blot). We unable generate polyclonal anti-          SDS, demonstrating ability rPpPer1 rPpPer2 <br /> sera PpPer2 efficiently detect fraction midgut lysate.       bind chitin. In contrast, unable demonstrate binding <br /> Nevertheless, data clearly demonstrates rPpPer2            rPpPer3 chitin. <br /> produced, determined hybridization anti-His <br /> antibody (Supp. Figure S1C). For PpPer3, recombinant              PpPer1 knockdown affects Le. major load P. <br /> protein wide trailing, eluted fractions 6&#226;&#8364;&#8220;13 (Supp.        papatasi <br /> Figure S1D) native protein eluted fractions 6 7              PpPer1 selected knockdown experiments following <br /> (dot blot 2 Supp. Figure S1A).                                           assessments expression profile (Figure 3) according to <br />                                                                             data chitin binding assays. As PpPer1 expressed <br /> Chitin binding assays                                                       exclusively midgut blood feeding binds chitin, we <br />   Figure 7 shows results binding recombinant              reasoned involved PM formation. <br /> proteins colloidal chitin. No recombinant protein detected           Intra thoracic injections P. papatasi females 80.5 ng of <br /> the wash fractions obtained colloidal chitin binding assays        double-strand RNA specific PpPer1 (dsPpPer1) performed <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                     7                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                    Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org   8   March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                      Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br /> Figure 3. Peritrophin mRNA expression profiles. Expression PpPer1, PpPer2, PpPer3 mRNAs assessed RT-PCR (23&#226;&#8364;&#8220;25 cycles) P. <br /> papatasi midguts dissected adult females different time points blood feeding (0&#226;&#8364;&#8220;144 hours PBM) (A); pools tissues <br /> other midgut (B); eggs (pool 20 eggs) body larvae pupae (C). b-Tub used housekeeping control gene. The <br /> size cDNA fragments amplified 122 bp (PpPer1), 168 bp (PpPer2), 121 bp (PpPer3), 148 bp (b-Tub). CC: Carcass. HG: Hindgut. FB: Fat <br /> Body. HS: Head salivary glands. OV: Ovaries. MT: Malpighian Tubules. E: Eggs. L1: Larval stage 1. L2: Larval stage 2. L3: Larval stage 3. L4: <br /> Larval stage 4. P: Pupa. (-): Negative control. <br /> doi:10.1371/journal.pntd.0002132.g003 <br />  <br /> to assess role, any, PpPer1 protein Le. major                     serine, threonine, glutamine, proline residues. Mucin-like <br /> development. First determined injection dsPer1                   domains predicted heavily O-linked glycosylated that <br /> was able reduce mRNA protein levels. As shown                        contributes gel-like consistency PM, critical their <br /> Figure 8, injection dsPpPer1 led 45% 30% reduction               role protecting midgut epithelia abrasion, hydrolytic <br /> in mRNA expression levels 24 h 48 h PBM, respectively                   enzymes, heavy metals, pathogens [38,39]. <br /> (Figure 8A), corresponding reduction 44% protein                   In addition role PM formation, peritrophins also <br /> levels 24 h PBM (Figure 8B C).                                          known participate detoxification [40]. In mosquito A. <br />    In spite lack noticeable differences PM structure              aegypti, mucin-like peritrophin AeIMUC shown bind <br /> between dsPpPer1 RNA-injected versus control-injected sand flies               heme vitro heme-regulatory motifs (HRM), also <br /> following midgut dissection (not shown), knockdown PpPer1 led            bound chitin [40,41]. HRMs predicted P. papatasi <br /> an increase Le. major load P. papatasi midguts 39% 48 h        L. longipalpis peritrophins, suggesting role these <br /> and 22% 96 h post-infection (as shown Figures 9A 9B).                proteins heme binding detoxification sand flies. <br />                                                                                   We identified CBDs present peritrophins from <br /> Discussion                                                                     P. papatasi identified L. longipalpis sequence databases <br />                                                                                type-A CBDs displaying molecular hallmarks required for <br />    Here, characterized peritrophins sand fly P.                chitin binding [23,42,43]. Chitin binding hallmarks include six <br /> papatasi thought involved formation PM adult            conserved cysteine residues (with conserved number residues <br /> females. In addition, PM important component                   conserved cysteines matching consensus <br /> vector competence sand flies [7,16,17,30], assessed role             sequence type-A CBDs) [23], conserved aromatic amino <br /> of PpPer1 molecular barrier Le. major development.                acids residues predicted interact chitin fibrils [23,43]. <br />    Different PpPer1 PpPer2, PpPer3 mucin-like                       Although putative CBDs Pp3put Ll3put display the <br /> peritrophin CBDs mucin-like domain rich                      conserved cysteines interspaced characteristic <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br />  <br />  <br />  <br />  <br /> Figure 4. Expression PpPer1 PpPer3 P. papatasi midgut lysates. Expression native PpPer1 PpPer3 P. papatasi midgut <br /> lysates assessed using midgut equivalent (from pools guts time point) non blood fed blood fed P. papatasi lane. <br /> Lanes: M, size marker; 1, non blood fed midgut; 2, blood fed midgut dissected 24 h PBM; 3, blood fed midgut dissected 48 h PBM; 4, blood fed <br /> midgut dissected 72 h PBM; 5, blood fed midgut dissected 96 h PBM. Proteins transferred nitrocellulose blots incubated anti- <br /> PpPer1 (A) anti-PpPer3 (B) specific antisera. Arrows point native proteins PpPer1 (A) PpPer3 (B). <br /> doi:10.1371/journal.pntd.0002132.g004 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        9                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                     Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org   10   March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                    Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br /> Figure 5. Modulation peritrophin mRNA expression Le. major infection. qRT-PCR assays depicting differences PpPer1 (A), PpPer2 <br /> (B), PpPer3 (C) mRNA levels non-infected Le. major infected midguts dissected 24 h, 48 h, 72 h PBM. Each dot (symbol) <br /> represents mRNA expression levels single midgut horizontal bars indicate mean expression levels. The cDNA encoding S3 <br /> protein 40S ribosomal subunit used housekeeping control gene. The mean expression non-infected midguts used a <br /> standard (100%) comparisons percentage mRNA expression Le. major infected midguts time point. NI: Non-infected. INF: Le. <br /> major infected. NS: Not significant. *: Statistically significant, p,0.05. <br /> doi:10.1371/journal.pntd.0002132.g005 <br />  <br /> expected type-A CBDs [23], amino acids aromatics               putative domains, P. papatasi L. longipalpis, shed <br /> are putative domains. In addition, Pp3put              light functions. <br /> Ll3put CBDs display extra cysteines residues 20 54                We obtained recombinant rPpPer1, rPpPer2, rPpPer3 <br /> have unusual two-peptide insertion fourth fifth            confirmed ability rPpPer1 rPpPer2 bind colloidal <br /> conserved cysteines. Such features contrast         chitin likely involved formation PM1 P. <br /> bona fide sand fly type-A CBDs display conserved number             papatasi. <br /> amino acids fourth fifth cysteines (12 residues),                Our assessment temporal spatial expression P. <br /> and lack extra cysteines present putative domains. Thus,           papatasi peritrophins demonstrated PpPer1 expression is <br /> it tempting speculate Pp3put Ll3put CBD domains                midgut-specific blood-induced, resembling transcriptional <br /> underwent type neo-functionalization [44],              profile PpChit1, midgut specific P. papatasi chitinase [32]. The <br /> change occurred peritrophins insects,             PpPer1 protein appears secreted midgut earlier <br /> such cat flea mosquitoes. This hypothesis               time point (24 h PBM) following blood meal PpChit1, <br /> extent supported findings PpPer3 (which                   activity peaks 48 h 72 h PBM [30]. <br /> Pp3put domain) expressed Malpighian tubules,               Nonetheless, patterns protein expression consistent <br /> by lack binding rPpPer3 chitin (see below). Although             functional roles PpPer1 PpChit1 PM formation <br /> the factors, expression tissues             degradation, respectively. <br /> other sand fly gut suggestive additional function              In contrast results observed PpPer1, PpPer2 PpPer3 <br /> for protein. Future functional characterization                 mRNAs expressed prior blood feeding (constitutively), and <br />  <br />  <br />  <br />  <br /> Figure 6. Recombinant proteins. Western blots carried 66His-tagged rPpPer1, rPpPer2, rPpPer3 (250 ng protein) <br /> obtained FreeStyle CHO-S cells. Proteins separated 4&#226;&#8364;&#8220;12% reducing NuPAGE gels. (A) Proteins transferred nitrocellulose and <br /> incubated anti-His antibody (1:2,000), followed anti-mouse AP-conjugated (1:10,000). Lanes: M, molecular weight marker; 1, PpPer1; 2, <br /> PpPer2; 3, PpPer3. (B) rPpPer2 (200 ng) N-deglycosylated 1,000 (lane 2), 2,000 (lane 3) 3,000 (lane 4) units PNGase F 16 h 37uC <br /> (untreated rPpPer2 control lane 1). Separated proteins transferred nitrocellulose incubated anti-His followed anti-mouse AP- <br /> conjugated. M: molecular weight marker. Arrows indicate 16 kDa 20-to-24 kDa bands seen rPpPer2 preparations (A B). After PNGase <br /> F treatment 20-to-24 kDa bands longer detected. <br /> doi:10.1371/journal.pntd.0002132.g006 <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                      11                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                       Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />  <br /> Figure 7. Chitin binding assay. Recombinant rPpPer1 rPpPer2 assayed capacity bind colloidal chitin. Wash elute fractions <br /> corresponding PpPer1 (A) rPpPer2 (B) loaded reducing 4&#226;&#8364;&#8220;12% NuPAGE gels. Lanes: M, molecular weight marker; 1, unbound <br /> fraction; 2, wash 1 (10 mM sodium phosphate buffer, pH 8.0); 3, wash 2 (10 mM sodium phosphate buffer, pH 8.0 - 1M sodium chloride); 4, wash 3 <br /> (0.1 M acetic acid); 5, elute (SDS lysis buffer); 6, purified protein. Proteins transferred blots incubated anti-His followed anti- <br /> mouse AP-conjugated. <br /> doi:10.1371/journal.pntd.0002132.g007 <br />  <br /> their expression limited midgut: PpPer2 expressed           phins suggested previous transcriptome analyses studies of <br /> the midgut hindgut; PpPer3 expressed midgut,                P. papatasi L. longipalpis [18,19,47]. The expression of <br /> hindgut, Malpighian tubules. Although peritrophin expression                PpPer1 up-regulated 24 h post-infection PpPer3 <br /> in hindguts and/or Malpighian tubules detected                mRNA levels reduced 24 h 48 h post-infection. Up- <br /> the cat flea Ctenocephalides felis [45] fruit fly Drosophila            regulation PpPer1 Le. major assist protecting the <br /> melanogaster [46], physiological roles peritrophins             parasite proteolytic enzymes (parasite advantage), may <br /> tissues determined. Regarding constitutive                    response sand fly order possibly reduce <br /> expression patterns PpPer2 PpPer3, corresponding                     permeability PM (disadvantageous parasite). Whether <br /> proteins translated fashion. In Aedes                  multiple signals secreted parasite present the <br /> aegypti, peritrophin AeIMUCI constitutively expressed,               infected blood involved regulation needs be <br /> yet protein detected blood fed midguts                 determined. Similarly, regarding PpPer3, differential gene <br /> 24 h PBM [40]. PpPer2 expressed larval stages, similar            expression infected midguts parasite-mediated, vector- <br /> AeIMUCI [41]. However, following assessment midgut                       induced defensive response infection, needs further <br /> lysates native peritrophin expression, evident            investigated. <br /> least PpPer1, protein expression midgut correlates                 The role P. papatasi peritrophin PpPer1 Le. major <br /> mRNA expression. Native PpPer1 readily detected 48 h                      development P. papatasi midgut assessed RNA <br /> PBM, matching peak mRNA expression. For native PpPer3,                   interference (RNAi). PpPer1 chosen RNAi experiments <br /> we observed significant levels protein 48 h, lower                    shown expressed exclusively midgut, <br /> amounts 72 h 96 h PBM. This profile correlates                  blood feeding. <br /> partially mRNA expression profiles observed.                              The sand fly PM thought fulfill apparently opposing <br />    To assess Le. major infection capable                   roles (protection barrier) comes Leishmania infection <br /> modulating P. papatasi peritrophin gene expression, compared                 [2]. That sand fly PM protects Le. major digestive enzymes <br /> PpPer1, PpPer2, PpPer3 mRNA levels Le. major infected               early infection [7] initially suggested potential <br /> and blood fed midguts. Although Le. major infection able             alteration PM scaffold, increasing permeability the <br /> modulate PpPer2 expression profile, PpPer1 PpPer3 expression                knockdown removal peritrophins, lead to <br /> levels changed significantly infection. Regulation peritro-             killing parasites. The injection dsPpPer1 P. papatasi <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                        12                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                    Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />  <br /> Figure 8. PpPer1 knockdown mRNA protein levels. (A) real time qRT-PCR showing PpPer1 mRNA level reduction dsRNA-injected <br /> (dsPer1) P. papatasi compared control injected (GFP dsRNA). Knockdown effects PpPer1 mRNA expression assessed 24 h 48 h PBM <br /> that corresponded 72 h 96 h post dsRNA injection. Each symbol represents mRNA expression levels single midgut. Horizontal bars <br /> indicate mean expression levels. The S3 gene used housekeeping control gene. The mean expression PpPer1 dsGFP-injected flies was <br /> used 100% standard. NS: Not significant. *: Statistically significant, p,0.05 (Unpaired t-test). (B) Western blot assay showing reduction PpPer1 <br /> protein levels 24 h PBM (72 h dsRNA injection) dsPer1-injected flies compared dsGFP-injected (chemiluminescence development). Nine <br /> and half micrograms protein loaded 10% NuPAGE gel. (C) Densitometry showing 44% reduction intensity PpPer1 protein <br /> band obtained chemiluminescence development compared PpPer1 band intensity dsGFP-injected flies. For PpPer1 knockdown <br /> assays, 80 ng dsRNA injected intrathoracically 3-to-5 day old P. papatasi females fed 30% sucrose solution ad libitum. <br /> doi:10.1371/journal.pntd.0002132.g008 <br />  <br /> thorax reduced PpPer1 mRNA PpPer1 protein levels 45%.                  knockdown PpPer1 concomitant increase parasite load <br /> Although difference prevalence infected sand flies           48 h post infection exactly contradict Pimenta et <br /> observed 48 h post-infection (not shown), PpPer1 knockdown                 al. [7]. In case, killing Leishmania induced the <br /> led significant increase (39%) Le. major load P. papatasi          complete lack PM addition exogenous chitinases the <br /> midguts. One possibility explain results PM           blood meal affected parasite&#226;&#8364;&#8482;s own <br /> increased permeability allow greater influx digestive                chitinase. In contrast, here, specifically knocked single <br /> enzymes endoperitrophic space, turning blood meal                      peritrophin but, overall, PM present likely leading to <br /> digestion faster making nutrients readily available              distinct outcome studies: killing Leishmania <br /> Le. major multiplication. Although increased Le. major load PpPer1         versus increased load. Future studies assess changes PM <br /> knockdown noted 96 h post-infection (22%),                   porosity caused knockdown PpPer1 such <br /> increase statistically significant. Our results showing          changes affect flow digestive proteases nutrients in <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                      13                              March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                  Phlebotomus papatasi Peritrophins <br />  <br />  <br />  <br />  <br /> Figure 9. Effects PpPer1 knockdown Le. major infection. Knockdown PpPer1 leads greater Le. major load midgut P. <br /> papatasi. At 48 h post-infection (A), dsPpPer1 (dsPer1) injection caused increased (39%; p,0.05 Mann-Whitney U test) Le. major load compared <br /> to dsGFP-injected P. papatasi. Results shown combined data independent experiments. (B) Although statistically significant, <br /> PpPer1 knockdown led 22% increase Le. major load P. papatasi midguts 96 h post-infection compared dsGFP-injected. Each dot <br /> (filled circle square) represents number Le. major single midgut horizontal bars indicate mean parasite number. P. papatasi were <br /> infected 56106 Le. major amastigotes/ml heat-inactivated mouse blood. N: Number P. papatasi midguts dissected. NS: Not significant. *: <br /> Statistically significant, p,0.05. <br /> doi:10.1371/journal.pntd.0002132.g009 <br />  <br /> and endoperitrophic space help clarify issues.         permeability PM interfere development of <br /> Recently, Araujo et al [48] showed adding chitinase              Leishmania fitness sand flies. <br /> blood meal led accelerated egg-laying sand flies a <br /> reduction total eggs laid females, likely              Supporting Information <br /> associated speedier acquisition nutrients lack <br /> of PM. Conversely, current data laboratory (not                 Figure S1 Fractionation gel filtration chromatography P. <br /> shown) indicate feeding flies antisera targeting sand          papatasi midgut lysate, rPpPer1, rPpPer2, rPpPer3. <br /> fly midgut chitinase PpChit1 [32] leads delay egg laying             Lysate 10 P. papatasi midguts dissected 48 h PBM (native <br /> after blood feeding increase number eggs laid              proteins) (A), rPpPer1 (B), rPpPer2 (C) rPpPer3 (D) were <br /> female. Taken results suggest integrity          fractionated using HPLC. The A280 absorption spectra the <br />  <br />  <br /> PLOS Neglected Tropical Diseases | www.plosntds.org                     14                             March 2013 | Volume 7 | Issue 3 | e2132 <br />                                                                                                                                                 Phlebotomus papatasi Peritrophins <br />  <br />  <br /> eluted samples following HPLC fractionation obtained                                     Acknowledgments <br /> 10 ml fraction placed nitrocellulose membranes <br />                                                                                                   We grateful Dr. JG Valenzuela (LVMB/NIAID/NIH) providing <br /> and incubated specific antisera (A D) anti-His <br />                                                                                                   batch rPpPer1 PpPer2 produced HEK293F cells. We also <br /> antibodies (B C). Numbers A280 spectra refer                                        thankful Dr. Y. Arakane, Department Biochemistry, Kansas State <br /> retention time peak. (A) Dot blots containing midgut                                     University, providing colloidal chitin. <br /> lysate fractions incubated anti-PpPer1 (1) anti- <br /> PpPer3 (2) antisera, respectively. (D) Dot blot incubated                                Author Contributions <br /> anti-PpPer3 antisera. Numbers dot blots refer each <br />                                                                                                   Conceived designed experiments: IVCA MRO. Performed the <br /> fraction collected (fractions blotted vertically).                                           experiments: IVCA NKS MRM. Analyzed data: IVCA MRO. <br /> (TIFF)                                                                                            Contributed reagents/materials/analysis tools: IVCA NKS MRM MRO. <br />                                                                                                   Wrote paper: IVCA MRO. <br />  <br /> References <br />  1. Herwaldt BL (1999) Leishmaniasis. Lancet 354: 1191&#226;&#8364;&#8220;1199.                                      25. Hall TA (1999) BioEdit: user-friendly biological sequence alignment editor and <br />  2. Ramalho-Ortigao M, Saraiva EM, Traub-Cseko YM (2010) Sand Fly-                                    analysis program Windows 95/98/NT. Nucleic Acids Symposium Series 41: <br />     Leishmania Interactions: Long Relationships Not Necessarily Easy. The                         95&#226;&#8364;&#8220;98. <br />     Open Parasitology Journal 4: 195&#226;&#8364;&#8220;204.                                                         26. Dillon RJ, Ivens AC, Churcher C, Holroyd N, Quail MA, et al. (2006) Analysis <br />  3. Alvar J, Ve&#194;&#180;lez ID, Bern C, Herrero M, Desjeux P, et al. (2012) Leishmaniasis                     ESTs Lutzomyia longipalpis sand flies contribution toward <br />     worldwide global estimates incidence. PLoS One 7: e35671.                              understanding insect-parasite relationship. Genomics 88: 831&#226;&#8364;&#8220;840. <br />  4. TDR/WHO (2002) TDR strategic directions: Leishmaniasis. Geneva. pp. 1&#226;&#8364;&#8220;5.                      27. Whelan S, Goldman N (2001) A general empirical model protein evolution <br />  5. Kamhawi S (2006) Phlebotomine sand flies Leishmania parasites: friends                     derived multiple protein families using maximum-likelihood approach. <br />     foes? Trends Parasitol 22: 439&#226;&#8364;&#8220;445.                                                               Mol Biol Evol 18: 691&#226;&#8364;&#8220;699. <br />  6. WHO (2010) World Health Organization Technical Report Series no. 949:                         28. Felsenstein J (1985) Confidence limits phylogenies: An approach using the <br />     Control leishmaniases. Report meeting WHO Expert                               bootstrap. Evolution 39: 783&#226;&#8364;&#8220;791. <br />     Committee Control Leishmaniases, Geneva, 22&#226;&#8364;&#8220;26 March 2010.                          29. Tamura K, Dudley J, Nei M, Kumar S (2007) MEGA4: Molecular Evolutionary <br />  7. Pimenta PF, Modi GB, Pereira ST, Shahabuddin M, Sacks DL (1997) A novel                           Genetics Analysis (MEGA) software version 4.0. Mol Biol Evol 24: 1596&#226;&#8364;&#8220;1599. <br />     role peritrophic matrix protecting Leishmania hydrolytic                  30. Coutinho-Abreu IV, Sharma NK, Robles-Murguia M, Ramalho-Ortigao M <br />     activities sand fly midgut. Parasitology 115 (Pt 4): 359&#226;&#8364;&#8220;369.                              (2010) Targeting midgut secreted PpChit1 reduces Leishmania major <br />  8. Schlein Y, Romano H (1986) Leishmania major L. donovani: effects                           development natural vector, sand fly Phlebotomus papatasi. PLoS Negl <br />     proteolytic enzymes Phlebotomus papatasi (Diptera, Psychodidae). Exp Parasitol 62:             Trop Dis 4: e901. <br />     376&#226;&#8364;&#8220;380.                                                                                      31. Oliveira F, Kamhawi S, Seitz AE, Pham VM, Guigal PM, et al. (2006) From <br />  9. Borovsky D, Schlein Y (1987) Trypsin chymotrypsin-like enzymes                         transcriptome immunome: identification DTH inducing proteins a <br />     sandfly Phlebotomus papatasi infected Leishmania possible role                  Phlebotomus ariasi salivary gland cDNA library. Vaccine 24: 374&#226;&#8364;&#8220;390. <br />     vector competence. Med Vet Entomol 1: 235&#226;&#8364;&#8220;242.                                                32. Ramalho-Ortigao JM, Kamhawi S, Joshi MB, Reynoso D, Lawyer PG, et al. <br /> 10. Secundino N, Kimblin N, Peters NC, Lawyer P, Capul AA, et al. (2010)                              (2005) Characterization blood activated chitinolytic midgut of <br />     Proteophosphoglycan confers resistance Leishmania major midgut digestive                    sand fly vectors Lutzomyia longipalpis Phlebotomus papatasi. Insect Mol Biol <br />     enzymes induced blood feeding vector sand flies. Cell Microbiol 12: 906&#226;&#8364;&#8220;                    14: 703&#226;&#8364;&#8220;712. <br />     918.                                                                                          33. Teixeira C, Gomes R, Collin N, Reynoso D, Jochim R, et al. (2010) Discovery of <br /> 1 <br /> </body></html>