<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MERTK Interactions SH2-Domain Proteins the <br /> Retinal Pigment Epithelium <br /> Shameka J. Shelby1, Karen Colwill2, Sirano Dhe-Paganon3,4, Tony Pawson2,5, Debra A. Thompson1,6* <br /> 1 Department Biological Chemistry, University Michigan Medical School, Ann Arbor, Michigan, United States America, 2 Samuel Lunenfeld Research Institute, <br /> Mount Sinai Hospital, Toronto, Ontario, Canada, 3 Structural Genomics Consortium, University Toronto, Toronto, Ontario, Canada, 4 Department Physiology, <br /> University Toronto, Toronto, Ontario, Canada, 5 Department Medical Genetics, University Toronto, Toronto, Ontario, Canada, 6 Department Ophthalmology and <br /> Visual Sciences, University Michigan Medical School, Ann Arbor, Michigan, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The receptor tyrosine kinase MERTK plays essential role phagocytic uptake shed photoreceptor membranes by <br />      retinal pigment epithelium (RPE). A fundamental aspect signal transduction receptor tyrosine kinases involves <br />      autophosphorylation tyrosine residues recruit Src-homology 2 (SH2)-domain proteins receptor intracellular <br />      domain. The goal current study evaluate interactions human MERTK SH2-domain proteins present <br />      RPE. The MERTK intracellular domain expressed 6xHis-fusion protein (6xHis-rMERTK571&#226;&#8364;&#8220;999), purified and <br />      phosphorylated. Ni2+-NTA pull downs performed using 6xHis-rMERTK571&#226;&#8364;&#8220;999 incubations recombinant <br />      phosphotyrosine-recognition sequences expressed GST-fusion proteins. In addition, pull downs native SH2-domain <br />      proteins performed using 6xHis-rMERTK571&#226;&#8364;&#8220;999 protein homogenates rat RPE/choroid. For recombinant <br />      native proteins, western analysis detected MERTK interactions GRB2, PIK3R1 (P85a), VAV3, SRC. <br />      Immunohistochemical analysis localized protein mouse RPE. In cultured RPE-J cells incubated rod outer <br />      segments (OS), siRNA knockdown Grb2 effect OS binding, significantly reduced OS uptake. Pik3r1 localized <br />      early phagosomes Rab5 Eea1. Phosphorylation activation Src detected downstream of <br />      phagocytosis Mertk activation. These findings suggest MERTK signaling RPE involves cohort SH2-domain <br />      proteins potential regulate cytoskeletal rearrangement membrane movement. Identification the <br />      SH2-domain signaling partners MERTK important step defining mechanism RPE phagocytosis <br />      central function <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>survival</span> retina. <br />  <br />   Citation: Shelby SJ, Colwill K, Dhe-Paganon S, Pawson T, Thompson DA (2013) MERTK Interactions SH2-Domain Proteins Retinal Pigment <br />   Epithelium. PLoS ONE 8(2): e53964. doi:10.1371/journal.pone.0053964 <br />   Editor: Mike O. Karl, Center Regenerative Therapies Dresden, Germany <br />   Received June 13, 2012; Accepted December 5, 2012; Published February 4, 2013 <br />   Copyright: &#195;&#376; 2013 Shelby et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research supported grants National Institutes Health (P30-EY007003, P60-DK-020572, T32-EY13934), The Midwest Eye Bank &amp; <br />   Transplantation Center. The Structural Genomics Consortium registered charity (number 1097737) receives funds Canadian Institutes Health <br />   Research, Canadian Foundation Innovation, Genome Canada Ontario Genomics Institute, GlaxoSmithKline, Karolinska Institutet, Knut and <br />   Alice Wallenberg Foundation, Ontario Innovation Trust, Ontario Ministry Research Innovation, Merck &amp; Co., Novartis Research Foundation, the <br />   Swedish Agency Innovation Systems, Swedish Foundation Strategic Research, Wellcome Trust. DAT recipient Senior Scientific <br />   Investigator Award Research Prevent Blindness. SJS recipient University Michigan Rackham School Graduate Studies Merit Fellowship, and <br />   King-Chavez-Parks Future Faculty Program Fellowship. The funders role study design, data collection analysis, decision publish, preparation <br />   manuscript. <br />   Competing Interests: Funding GlaxoSmithKline Merck &amp; Co. received by, contributed <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>, general operating costs Structural <br />   Genomics Consortium University Toronto. This does alter authors&#226;&#8364;&#8482; adherence PLOS ONE policies sharing data materials. <br />   * E-mail: dathom@umich.edu <br />  <br />  <br />  <br /> Introduction                                                                          gene encoding Mertk [5]. As consequence loss-of- <br />                                                                                       function <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>mutation</span>, shed OS membranes bound not <br />    Vertebrate photoreceptor cells function conditions                        taken RPE [6]. Subsequent studies identified disease- <br /> high metabolic demand light exposure result chronic                       associated mutations gene encoding human MERTK in <br /> damage lipid protein components specialized outer                     individuals affected relatively rare form juvenile-onset <br /> segment (OS) membranes [1]. As result, photoreceptor cells                          retinitis pigmentosa [7,8,9,10,11]. <br /> undergo process continual renewal involving addition                         MERTK receptor tyrosine kinase belonging TAM <br /> new membranes <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span> OS just connecting                         receptor family (Tyro-3, Axl, Mer), ligands include the <br /> cilium, shedding spent membranes OS tips [2]. The                        vitamin K-dependent proteins growth arrest-specific protein 6 <br /> retinal pigment epithelium (RPE) removes shed debris                         (gas6) protein S [12], retinal dystrophy-gene <br /> the subretinal space phagocytic uptake [3], process                      products Tulp1 Tubby [13]. MERTK expressed in <br /> critical photoreceptor <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>cell survival</span>, cells depend                    monocytes tissues epithelial reproductive origin [14] <br /> close association RPE oxygen nutrients, for <br />                                                                                       contributes number cellular processes including <br /> ionic homeostasis subretinal space [4]. A key insight the <br />                                                                                       cell proliferation survival, down-modulation pro-inflam- <br /> molecular mechanism phagocytic uptake RPE resulted <br />                                                                                       matory signals, clearance apoptotic cells [15]. Studies the <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> discovery inherited retinal degeneration Royal <br />                                                                                       mechanism apoptotic cell clearance transfected HEK-293T <br /> College Surgeons (RCS) dystrophic rats results deletion <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                              February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                     MERTK Interactions SH2-Domain Proteins <br />  <br />  <br /> cells suggest MERTK signaling involves cross-talk avb5           (GRB), phosphatidylinositol 3-kinase regulatory subunit alpha <br /> integrin, resulting activation focal adhesion kinase (FAK)       (PIK3R1 P85a), vav proto-oncogenes (VAVs), SRC- <br /> SRC family non-receptor tyrosine kinases (SFKs) [16]. In              family kinases (SFKs), described below. <br /> macrophages RPE, proteolytic cleavage MERTK has <br /> been proposed act negative feedback loop limit                  GRB2 <br /> phagocytic particle binding avb5 integrin [16,17]. In addition,            A key role GRB proteins ability function as <br /> MERTK shown drive redistribution myosin II              adapters GEF proteins involved RAS activation of <br /> that essential normal phagocytic function RPE,           downstream kinases regulate multiple signaling processes <br /> potentially regulating formation closure phagocytic       biological activities, including NF-kB control inflammation <br /> cup [18].                                                                  regulation motor proteins involved cellular movement <br />    A central step MERTK signaling mechanism                  [25,26]. Previous studies hematopoietic cells identified MERTK <br /> activation receptor tyrosine kinase activity resulting trans-        interactions GRB2 [21], pointing importance of <br /> autophosphorylation. Three tyrosine residues, Y-749, Y-753,            defining role GRB proteins RPE phagocytosis. In the <br /> Y-754 present activation loop (human) MERTK-             current study, RT-PCR used evaluate expression mouse <br /> kinase domain identified sites autophosphorylation         RPE/choroid, retina, brain, liver. Transcripts encoding Grb2 <br /> [19]. Receptor tyrosine phosphorylation serves generate                 seen relatively high levels RPE/choroid compared <br /> docking sites signaling molecules, including Src-homology 2            levels present retina, brain, liver (Figure 1A). In contrast, <br /> (SH2) domain proteins function enzymes adapter                 transcripts encoding Grb7 isoform low RPE/choroid <br /> proteins [20]. Previous studies MERTK-associated proteins            high liver, transcripts encoding Grb10 isoform <br /> myeloid cells identified interactions multiple SH2-domain             high RPE/choroid retina, low liver. Assays of <br /> proteins [21,22,23]. The fundamental role SH2-domain                    potential MERTK interactions using Ni2+-NTA pull downs of <br /> proteins MERTK-downstream signaling suggests play             6xHis-rMERTK571&#226;&#8364;&#8220;999 incubated rSH2-domain fusion <br /> an essential role mechanism RPE phagocytosis. In             proteins showed strong recovery recombinant GRB2 <br /> present study, analysis expression, protein interactions,           protein (Figure 1B), exceeded recombinant GRB7 <br /> functional assays used identify SH2-domain proteins           GRB10, suggesting specificity interactions MERTK <br /> in RPE potential signal downstream MERTK                GRB isoforms. Analysis endogenous protein expression on <br /> and upstream cellular remodeling. The findings suggest             western blots showed strong Grb2 immunoreactivity RPE/ <br /> MERTK interacts multiple signaling partners RPE,               choroid RCS congenic dystrophic rats [27,28], less <br /> including SH2-domain proteins regulate cytoskeletal rearran-          seen rat RPE-J cell line [29] (Figure 1C). Pull downs of <br /> gement membrane movement professional phago-                  endogenous proteins mouse RPE/choroid homogenates <br /> cytes.                                                                     using 6xHis-rMERTK571&#226;&#8364;&#8220;999 resulted specific recovery Grb2 <br />                                                                            seen western blots (Figure 1D), obtained using <br /> Results                                                                    6xHis-rMERTK571&#226;&#8364;&#8220;864 missing C-terminal sequence (data not <br />                                                                            shown). Immunohistochemical analysis retina/RPE/choroid <br /> Development Study Tools Focus                                       cryosections BALB/c mice confirmed presence sub- <br />    To identify potential MERTK-signaling partners RPE,              stantial Grb2 expression RPE neural retina, with <br /> expression analysis coupled screening strategy             marked immunoreactivity present interneuron ganglion <br /> focused candidate SH2-domain proteins selected                  cell layers (Figure 1E), reflecting fundamental role Grb2 in <br /> extensive cDNA library encoding corresponding phospho-                 wide range signaling processes. Taken together, findings <br /> tyrosine-recognition sequences GST-fusion proteins [20]. The            demonstrate ability current approach capture <br /> recombinant human SH2-domains (rSH2-domains) ex-                      MERTK interactions endogenous proteins, consis- <br /> pressed E. coli purified GSH-affinity size-                  tent previous studies showing GRB2 binding MERTK- <br /> exclusion chromatography. Constructs encoding full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>              Y867 [21]. <br /> human MERTK cytoplasmic domain (rMERTK571&#226;&#8364;&#8220;999), well <br /> as truncated construct (rMERTK571&#226;&#8364;&#8220;864) [24], expressed              Grb2 RPE Phagocytosis <br /> in E. coli 6xHis fusion proteins purified using Ni2+-NTA               To evaluate effect Grb2 loss-of-function RPE <br /> affinity size-exclusion chromatography. To confirm autop-              phagocytosis, siRNAs used deplete Grb2 transcripts in <br /> hosphorylation tyrosine residues kinase domain,                  sub-confluent rat RPE-J cells, followed quantitative assays of <br /> rMERTK571&#226;&#8364;&#8220;864 subjected tryptic peptide analysis using              OS binding uptake. Transient transfection RPE-J cells with <br /> MALDI mass spectrometry (MALDI-MS). Ion fragmentation                      pool Grb2 targeting siRNAs resulted efficient <br /> patterns peptides designated P1, P2, P3 identified           knockdown transcript protein level (Figures 2A, <br /> three sites tyrosine phosphorylation catalytic domain,           2B). In contrast, Grb2 expression retained cells transfected <br /> Y749, Y753, Y754, agreement previously published               control non-targeting siRNA. Equivalent levels Mertk <br /> data [19] (Figure S1). To evaluate protein-protein interactions,           b-actin present treated control cells. <br /> Ni2+-NTA pull downs performed using rMERTK571&#226;&#8364;&#8220;999                     Furthermore, phalloidin staining actin cytoskeleton, and <br /> (corresponding full-length cytoplasmic domain)                 immunostaining antibodies ZO-1 protein present <br /> purified rSH2-domain fusion proteins, rat RPE/               tight junctions showed little disruption cellular integrity <br /> choroid homogenates. Candidates analysis included SH2-                 (Figure S2A). In assays phagocytic uptake using bovine OS <br /> domain proteins previously implicated MERTK downstream                  labeled AlexaFluor 555, RPE-J cells treated Grb2 <br /> signaling. Expression RPE evaluated transcript           targeting siRNAs exhibited significantly activity (,60% the <br /> and protein level cultured RPE-J cells, RCS congenic             6 h time point) cells treated control non-targeting <br /> and dystrophic rats. The combined results led focus                siRNA (p,0.0005) (Figures 2C, 2D). In cases, OS binding <br /> protein families previously implicated mechanisms phago-             unaffected (p.0.05). Taken together, findings support <br /> cytic uptake, including growth factor receptor-bound proteins              view interaction endogenous Grb2 MERTK <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             2                          February 2013 | Volume 8 | Issue 2 | e53964 <br />                                    MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   3   February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                              MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br /> Figure 1. Grb2 expressed interacts MERTK RPE. (A) Grb Hprt transcripts amplified RT-PCR mouse RPE/choroid, <br /> retina, brain, liver total RNA. Hprt primers served control. PE, RPE/choroid; RE, retina; B, brain; L, liver; (-), cDNA. (B) 6xHis-rMERTK571&#226;&#8364;&#8220;999 and <br /> GRB GST-SH2-domains purified E. coli interactions evaluated using Ni2+-NTA pull downs. Coomassie blue-stained proteins SDS <br /> gels shown. SH2, GRB GST-SH2 domain loading control; Ctrl, negative control omitting rMERTK; PD, pull GRB SH2-domain and <br /> rMERTK571&#226;&#8364;&#8220;999; MER, rMERTK571&#226;&#8364;&#8220;999 loading control. (C) Grb protein immunoreactivity western blots RPE/choroid protein homogenates from <br /> 4 wk old RCS congenic dystrophic rats 2 h post light-onset, RPE-J cell homogenates, probed antibodies recognizing Grb2. WT, <br /> congenic RCS-rat RPE/choroid; Dyst, dystrophic RCS-rat RPE/choroid; RpeJ, RPE-J cells. (D) Ni2+-NTA pull downs endogenous proteins RPE/ <br /> choroid homogenates RCS congenic dystrophic rats incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999. Western blots recovered <br /> proteins probed antibodies recognizing Grb2. Load, input RPE/choroid homogenate; Ctrl, Ni2+-NTA resin homogenate alone; PD, pull <br /> down including rMERTK571&#226;&#8364;&#8220;999 homogenate. (E) Grb2 localization retina/RPE/choroid cryosections 4 wk old BALB/c mice imaged using <br /> indirect immunofluorescence confocal microscopy. A selected area RPE enlarged specific localization cell layer. RPE, retinal <br /> pigment epithelium; OS, outer segments; IS, inner segments; ONL, outer nuclear layer; OPL, outer plexiform layer; INL, inner nuclear layer; IPL, inner <br /> plexiform layer; GCL, ganglion cell layer. <br /> doi:10.1371/journal.pone.0053964.g001 <br />  <br /> contributes, directly indirectly, phagocytic activity            consistent localization Pik3r1 OS-containing phago- <br /> RPE.                                                                              somes RPE, place proximity Rab5, an <br />                                                                                   established participant formation endosomes and <br /> PIK3R1 (P85a)                                                                     phagosomes [33]. Taken together, ability PIK3R1 undergo <br />    Stimulation phosphoinositide-3-kinase (PI3K) activity                    direct interaction MERTK, coupled colocalization <br /> activated receptor tyrosine kinases plays central role                       early endosome markers EEA1 RAB5, its <br /> downstream signaling mechanisms, including involved                      potential act effector RAB5 [33], suggest that <br /> regulating inflammatory responses [30,31]. Analysis                        PIK3R1 activation downstream MERTK signaling may <br /> expression PI3K regulatory subunit Pik3r1 mouse tissues                 represent novel mechanism contributing phagosome <br /> showed transcript levels relatively high RPE/choroid                 formation RPE. <br /> compared levels retina brain (Figure 3A). Ni2+-NTA pull <br /> downs 6xHis-rMERTK571&#226;&#8364;&#8220;999 showed specific recovery                        VAV Proteins <br /> a PIK3R1-fusion protein corresponding SH2-domain                              The VAV family proteins, VAV1, VAV2, VAV3 serve as <br /> present amino-terminal half regulatory subunit                      guanine nucleotide exchange factors (GEFs) RAC1 GTPase <br /> (PIK3R1-N), seen fusion protein                              contributes cytoskeletal rearrangement effects on <br /> corresponding SH2-domain present carboxy-terminal                   actin polymerization [34]. Previous studies reported VAV1 <br /> half (PIK3R1-C) (Figure 3B). Western analysis showed equivalent                   undergoes interaction MERTK independent receptor <br /> levels Pik3r1 protein RPE/choroid congenic                         phosphorylation status [23]. In current study, analysis of <br /> dystrophic rats, higher levels RPE-J cells (Figure 3C). Pull               expression mouse RPE/choroid showed transcripts encoding all <br /> downs 6xHis-rMERTK571&#226;&#8364;&#8220;999 incubated RPE/choroid                           VAV isoforms (Figure 5A). Pull-down assays 6xHis- <br /> homogenates showed specific recovery endogenous Pik3r1                     rMERTK571&#226;&#8364;&#8220;999 showed interaction rSH2-domain fusion <br /> protein congenic dystrophic samples (Figure 3D).                    proteins corresponding VAV1 VAV3, VAV2 <br /> Immunohistochemistry mouse retina/RPE/choroid cryosec-                         (Figure 5B). Western analysis showed Vav1 Vav3 were <br /> tions showed strong Pik3r1 immunoreactivity RPE                        present protein homogenates RPE/choroid congenic <br /> photoreceptor inner segments, intense labeling present                dystrophic rats, RPE-J cells (Figure 5C). Pull downs <br /> the inner retina (Figure 3E). Taken together, results               RPE/choroid homogenates showed interaction 6xHis- <br /> first demonstrate direct interaction MERTK PI3K                      rMERTK571&#226;&#8364;&#8220;999 endogenous Vav3, Vav1 <br /> in vitro native RPE protein.                                             (Figure 5D). Immunohistochemical analysis retina/RPE/ <br />                                                                                   choroid cryosections showed punctate localization Vav3 the <br /> Pik3r1 Early Phagosome                                                     RPE, diffuse labeling inner retina (Figure 5E). Viewed <br />    Previous studies shown PI3K participates early                    together, findings suggest association VAV3 <br /> events necessary phagosome formation maturation [32].                     comprises predominant MERTK interaction occurring with <br /> To investigate role Pik3r1 relative phagosome                           VAV family members RPE. <br /> formation RPE, cultured RPE-J cells incubated <br /> with bovine rod OS, indirect immunofluores-                        SRC <br /> cence microscopy performed using antibodies                              SRC-family kinases (SFKs) intracellular tyrosine kinases that <br /> Pik3r1, markers phagosome formation: early                     act downstream receptor activation powerfully impact <br /> endosomal antigen 1 (Eea1) Rab5 GTPase [31]. In RPE-J                         signaling. SFKs shown play role apoptotic cell <br /> cells incubated OS, proteins                        clearance dendritic cells [22] contribute immunor- <br /> found co-localize internalized rhodopsin-containing                       eceptor signaling activated particle uptake [35]. In the <br /> structures likely corresponding early phagosomes (Figure                       current study, analysis expression mouse RPE/choroid <br /> S3). In cells incubated added OS, Pik3r1 labeling                         identified transcripts encoding nearly entire SRC family, <br /> appeared uniform diffuse (Figure 4A), Eea1                            including Src, Fyn, Fgr, Yes, Hck, Lyn, Lck (Figure 6A). <br /> labeling associated small discrete vesicles present                      Transcripts encoding Hck, Fyn, Yes appeared relatively <br /> throughout cytoplasm (Figure 4A), Rab5 labeling                       abundant RPE/choroid versus retina. Pull-down assays <br /> widespread punctate (Figure 4B). In contrast, RPE-J cells                  6xHis-rMERTK571&#226;&#8364;&#8220;999 showed specific interaction the <br /> incubated rod OS, Pik3r1 Eea1 labeling seen                       rSH2-domain fusion proteins corresponding SRC HCK <br /> co-localize large vesicular structures (Figure 4A). These                      (Figure 6B). Western analysis showed Src present the <br /> structures similar appearance exhibiting co-                     RPE/choroid congenic dystrophic rats, RPE-J cells, <br /> localization Eea1 Rab5 (Figure 4B). These findings                     Hck protein levels significantly lower case <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                    4                           February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                         MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />  <br /> Figure 2. Grb2 silencing decreases phagocytic uptake RPE cells. RPE-J cells transfected pool Grb2 targeting siRNAs non- <br /> targeting siRNA control, expression evaluated 5 days later. (A) Transcript levels Grb2, Mertk, b-actin evaluated using RT-PCR. (B) <br /> Protein levels Grb2, Mertk, b-actin evaluated western blot analysis RPE-J cell homogenates. (C-D) Phagocytic activity assays. (C) Confocal <br /> images representative fields RPE-J cells showing ingested OS labeled red, DAPI-stained nuclei shown blue. Cells incubated with <br /> AlexaFluor 555-labeled bovine rod OS 4 h quenched addition trypan blue. (D) OS ingestion binding quantified three <br /> independent assays plotted percentage control total OS 4 h set 100%. Error bars represent mean 6 SEM, n = 3. P values <br /> were calculated using Student&#226;&#8364;&#8482;s t test, shown: p**,0.05, p****,0.00005. <br /> doi:10.1371/journal.pone.0053964.g002 <br />  <br /> (Figure 6C). However, rMERTK571&#226;&#8364;&#8220;999 pull downs RPE/                      apical microvilli. These results suggest SRC is <br /> choroid homogenates congenic dystrophic rats showed                  candidate direct interaction MERTK regulation <br /> interaction endogenous Src, interaction Hck             downstream effects RPE function. <br /> detected (Figure 6D). Immunohistochemical analysis Src <br /> showed strong labeling RPE, inner <br /> retinal layer (Figure 6E). In RPE, significant labeling extended <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                5                           February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />  <br /> Figure 3. Pik3r1 expressed interacts MERTK RPE. (A) Pik3r1 Hprt transcripts amplified mouse tissues RT-PCR. <br /> (B) Ni2+-NTA pull downs recombinant PIK3R1 GST-SH2 domains incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999 evaluated SDS-gels. (C) <br /> Pik3r1 immunoreactivity western blots rat RPE/choroid RPE-J cell homogenates. (D) Ni2+-NTA pull downs RPE/choroid protein <br /> homogenates RCS congenic dystrophic rats incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999, evaluated western analysis with <br /> antibodies recognizing Pik3r1. (E) Pik3r1 localization cryosections mouse retina/RPE/choroid. Details Figure 1. <br /> doi:10.1371/journal.pone.0053964.g003 <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                          6                        February 2013 | Volume 8 | Issue 2 | e53964 <br />                                    MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   7   February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                    MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br /> Figure 4. Pik3r1 co-localizes early phagosomes Eea1 Rab5 OS uptake. RPE-J cells incubated with, without, <br /> isolated bovine OS 4 h. Cells fixed stained antibodies recognizing Pik3r1 Eea1 AlexaFluor 488 secondary (A), or <br /> recognizing Rab5 Eea1 AlexaFluor 555 secondary (B), viewed using fluorescence confocal microscopy. Areas showing co-localization <br /> appear yellow merged images marked white arrowheads. <br /> doi:10.1371/journal.pone.0053964.g004 <br />  <br />  <br /> SRC Activation Downstream MERTK                                         MERTK essential uptake phase phagocytosis, <br />    Previous studies apoptotic cell clearance proposed         likely fulfills signaling role downstream MERTK <br /> SFKs facilitate crosstalk MERTK avb5 integrin               activation RPE. <br /> downstream signaling RAC1 involved cytoskeletal reorgani-               Recent studies shown RAC1 activation occurs <br /> zation [16,29,36], however, mechanistic details                downstream avb5 integrin, does require MERTK, <br /> specific SFK(s) involved elucidated. In current          suggesting integrin signaling CRKII/p130CAS/ <br /> study, activation status SRC evaluated using                 DOCK180/ELMO complex primary mechanism involved <br /> antibody recognizes active form protein            activating RAC1 RPE [36]. A model MERTK <br /> phosphorylated tyrosine residue 416 (pY416). This analysis              mediated phagocytosis based studies invokes in- <br /> showed formation pY416 SRC HEK-293T cells transfected                volvement SFKs crosstalk MERTK avb5 <br /> with active full-length MERTK, kinase-dead                    integrin regulating downstream signaling RAC1 <br /> MERTK (Figure 7A). In addition, formation pY416 Src                 [16,29,36,39]. Crosstalk integrins, RHO-family GTPases, <br /> seen RPE/choroid congenic RCS rats sacrificed             SFKs documented signaling involved cellular <br /> peak phagocytic uptake, seen animals sacrificed             adhesion [40]. The current studies establish MERTK <br /> before light onset, dystrophic RCS rats sacrificed         interacts recombinant SRC, endogenous Src <br /> time (Figure 7B). Formation pY416 Src seen RPE-J            rat RPE. In addition, activation MERTK promotes Src <br /> cells incubated rod OS, appearing highest 30 45 min            activation evidenced phosphorylation Y416. Transcripts <br /> post feeding (Figure 7C). Viewed together, findings suggest          encoding known mammalian SFKs, exception Blk, <br /> that SRC activation RPE result direct interaction          detected mouse RPE/choroid. This suggests are <br /> with activated MERTK, leave open possibility                  multiple opportunities SFKs directly participate MERTK <br /> additional SFKs function downstream MERTK signaling                 signaling RPE, crosstalk integrins involved <br /> in phagocytosis.                                                           cytoskeletal reorganization. Such mechanisms the <br />                                                                            potential make significant contributions cellular motility, in <br /> Discussion                                                                 SFKs important regulating RHO family <br />                                                                            GTPases, including RHOA functions induce actomyosin <br />    Phagocytic uptake RPE requires multiple activities          contractility [41]. <br /> regulate cellular plasticity, including cytoskeletal reorganization           Myosin motor proteins responsible actin motility an <br /> and membrane remodeling, contribute down-                      important driving force phagosome formation [42]. In RPE, <br /> modulating pro-inflammatory responses. Although required               myosin II shown interact MERTK and <br /> for binding effete membranes, MERTK signaling essential          contribute phagosome formation [18], myosin VIIa <br /> initiating phagocytic uptake. The current studies                implicated phagosome trafficking [43]. Previous <br /> MERTK interactions RPE/choroid phagocytic                 studies MERTK-associated proteins hematopoietic cells <br /> cell types involve overlapping set core SH2-domain proteins          identified adapter protein GRB2 [21] established this <br /> that regulated response MERTK loss-of-                  interaction prerequisite phagocytic clearance apoptotic <br /> function RCS rat. These findings lead new appreciation         cells [25]. In addition, GRB2 signaling myosin dynamin via <br /> of phagocytic mechanism adapted highly-specialized              RAS pathway shown induce membrane ruffling <br /> function RPE.                                                       cell migration [44]. The current studies MERTK <br />    Changes actin polymerization needed cytoskeletal re-             interacts endogenous Grb2 present RPE. Consistent <br /> organization regulated members RHO family                 previous findings Y867 required interaction of <br /> GTPases, including small G-proteins RAC1, CDC42,                   recombinant MERTK GRB2 [21], native Grb2 to <br /> RHO A [37]. Activation RHO family GTPases requires                      interact full-length rMERTK571&#226;&#8364;&#8220;999 intracellular domain, <br /> interaction guanine nucleotide exchange factors (GEFs)                rMERTK571&#226;&#8364;&#8220;864. In addition, knockdown Grb2 <br /> necessary GDP/GTP exchange [34]. These include                     inhibit phagocytic uptake ROS RPE-J cells <br /> VAV family SH2-domain proteins act GEFs                     culture, established to <br /> RAC1 involved regulating F-actin recruitment. Previous studies          direct effect phagocytic mechanism aspects cell <br /> in macrophages showed VAV1 interacts MERTK                       function viability. Taken together, findings point to <br /> independent phosphorylation status [23]. In current             conserved role GRB2 apoptotic cell clearance, endocytosis, <br /> studies, VAV1 VAV3 shown interact                    RPE phagocytosis. <br /> MERTK recombinant proteins, native Vav3                       In phagocytic uptake macrophages, formation the <br /> found interact assays endogenous rat RPE proteins. Vav3           phagocytic cup phagosome maturation require the <br /> was major isoform seen mouse RPE using                     involvement class I III PI3Ks [32]. In Fc-receptor <br /> immunohistochemical analysis. In contrast, expression VAV               signaling, role PI3K generate phosphoinositides <br /> family proteins detected cultured RPE-J cells retain       recruit RAB5 GTPase early-endosomal markers (in- <br /> the capacity perform MERTK-mediated phagocytic uptake. In               cluding EEA1) phagosome [30,31,33]. Previous studies <br /> addition, previous studies Vav2/3 <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice shown              using PI3K inhibitors wortmannin LY294002 shown <br /> that Vav loss-of-function results glaucoma <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span>,         loss PI3K activity inhibits RPE phagocytosis [45]. In <br /> retinal degenerative disease [38]. Taken together, findings          hematopoetic cells, PIK3R1 regulatory subunit class I PI3K <br /> are consistent view direct activation VAV proteins        shown interact indirectly GRB2 FMS- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             8                         February 2013 | Volume 8 | Issue 2 | e53964 <br />                                    MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   9   February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                   MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br /> Figure 5. Vav3 expressed interacts MERTK RPE. (A) Vav Hprt transcripts amplified mouse RPE/choroid RT-PCR. <br /> (B) Ni2+-NTA pull downs recombinant VAV GST-SH2 domains incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999 evaluated SDS-gels. (C) Vav <br /> protein immunoreactivity western blots rat RPE/choroid RPE-J cell homogenates. (D) Ni2+-NTA pull downs RPE/choroid protein <br /> homogenates RCS congenic dystrophic rats incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999, evaluated western analysis with <br /> antibodies recognizing Vav1 Vav3. (E) Vav3 localization cryosections mouse retina/RPE/choroid. Details Figure 1. <br /> doi:10.1371/journal.pone.0053964.g005 <br />  <br /> MERTK chimera [21]. In current studies, recombinant                   SRC, EEA1, Cell Signaling Technology; GAPDH, Ambion; <br /> MERTK shown interact endogenous Pik3r1,               b-actin RAB5, Abcam. AlexaFluor 555, AlexaFluor 488- <br /> possibility endogenous Grb2 contributed interaction          conjugated anti-rabbit IgG, AlexaFluor 555-conjugated anti- <br /> cannot excluded. However, studies using purified recombinant           mouse IgG, Sybr safe Invitrogen. Complete <br /> proteins showed PIK3R1 interacts                       protease inhibitors, PhosSTOP phosphatase inhibitors, Fugene <br /> rMERTK571&#226;&#8364;&#8220;999 intracellular domain. The site interaction,         transfection reagent, AmpliTaq Gold polymerase from <br /> as VAV3 SRC, predicted using various         Roche Diagnostics. Ni2+-NTA resin, RNeasy kit, Superscript II, <br /> algorithms [46] remain experimentally determined.               oligo-dT Qiagen. RPE-J HEK-293T cells were <br /> Taken together, findings suggest PI3K undergo              ATCC. Other chemicals reagents Sigma. The <br /> both direct indirect interactions MERTK. The finding             pcDNA 3.1+ expression vectors encoding full-length MERTK and <br /> that Pik3r1 colocalizes Eea1 Rab5 GTPase                 kinase-dead R844C-MERTK previously described [8]. <br /> phagosomes formed uptake OS RPE-J cells suggests <br /> that mechanism conserved professional phagocytes.           RT-PCR Analysis SH2-domain Protein Expression <br />    In immune cells, important role MERTK dampen                  Total RNA isolated freshly-dissected RPE/choroid, <br /> pro-inflammatory responses resulting apoptotic cell binding          retina, liver, brain C57BL/6 mice using RNAeasy kits, <br /> [47]. This accomplished GRB2 PI3K signaling             first-strand cDNAs generated using Superscript II and <br /> downstream autophosphorylation MERTK-Y867                      oligo-dT. Sequences encoding SH2-domain proteins interest <br /> contributes regulation NF-kB [21,25]. Studies               amplified gene-specific primers flanking one <br /> dendritic cells shown SRC signaling downstream          intron genomic sequence (Table S1) using AmpliTaq Gold <br /> MERTK plays important role immunomodulation [22]. The               polymerase following cycling conditions: 1 cycle 95uC <br /> finding GRB2, PI3K, SRC likely contribute MERTK               10 min followed 28 cycles 95uC 2 min, 60uC for <br /> signaling RPE raises important possibility           45 sec, 72uC 2 min. Primers hypoxanthine-guanine <br /> mechanism plays critical role controlling inflammatory               phosphoribosyltransferase (Hprt) served control. PCR <br /> responses elicited OS uptake, associated         products analyzed electrophoresis agarose gels stained <br /> aging disease.                                                        Sybr safe. Comparisons relative levels different gene <br />    Establishing identities MERTK-interacting SH2-domain            products using RT reaction amplifications <br /> proteins RPE provides new insight downstream                  containing single pair primers. <br /> signaling potentially linked various aspects phagocytic <br /> mechanism. The role MERTK loss-of-function inherited                Western Analysis SH2-domain Protein Expression <br /> retinal degeneration suggests key signaling partners             Dissected samples RPE/choroid dystrophic and <br /> candidates involvement group diseases. Thus,               congenic RCS rats harvested times corresponding peak <br /> increased knowledge phagocytic mechanism                   phagocytic uptake [48], confluent cultures rat RPE-J <br /> potential advance understanding causes RPE               cell line retains Mertk expression [29] harvested 3 <br /> dysfunction occurring aging disease,            days culture. Tissues cells homogenized 20 mM <br /> efforts develop targeted therapeutic strategies improve RPE         MOPS, 2 mM EGTA, 5 mM EDTA, 1% Triton X-100, 1 mM <br /> function.                                                                 DTT, protease phosphatase inhibitors; cellular debris was <br />                                                                           removed low speed centrifugation; protein concentrations <br /> Materials Methods                                                     supernates determined using modification the <br />                                                                           Lowry method [49]. Protein samples (7.5 mg) dissociated in <br /> Animals                                                                   16 SDS sample buffer, electrophoresed 10% polyacrylamide <br />    Experimental procedures involving animals performed            gels, transferred nitrocellulose. Blots blocked and <br /> accordance guidelines approval              incubated primary antibodies specific SH2-domain <br /> University Committee Use Care Animals                    proteins interest, alkaline phosphatase-conjugated <br /> University Michigan, compliance                   secondary antibodies, developed using 5-bromo-4-chloro-39- <br /> statement ethical care use animals Association          indolyphosphate p-toluidine nitro-blue tetrazolium chloride. <br /> for Research Vision Ophthalmology (ARVO). C57BL/6J <br /> and BALB/C mice bred animals obtained                  Immunohistochemical Analysis <br /> Jackson Laboratories. Pigmented dystrophic (RCS-p+) non-                 Eyes BALB/c mice perfused 4% paraformaldehyde <br /> dystrophic (RCS-rdy+p+) rats bred strains described         washed PBS, transitioned sucrose/OCT, flash <br /> in [27,28]. Mice rats housed 12-h/12-h light-dark           frozen. Retinal cross sections (10 mm) washed PBS and <br /> cycle (,300 lux room light) euthanized CO2                    permeabilized PBS-T (0.125% Triton X-100 ); blocked with <br /> inhalation approximately 2 h light onset, time levels        1% bovine serum albumin, 10% normal goat serum, 0.125% <br /> of RPE phagocytic uptake near maximal [48].                           Triton X-100; incubated primary antibodies 2 h, <br />                                                                           fluorophore-conjugated secondary antibodies 1 h. <br /> Materials                                                                 Species dilutions primary antibodies follows: Rabbit <br />   Primary antibodies: GRB2, Santa Cruz Biotechnology;                     anti-GRB2 (1:300), anti-PIK3R1 (1:300), anti-VAV3 (1:300), anti- <br /> PIK3R1, VAV1, VAV3, HCK, Millipore; c-SRC, pY416                      SRC (1:300), anti-RAB5 (1:300); Mouse anti-EEA1 (1:300); <br />  <br />  <br /> PLOS ONE | www.plosone.org                                           10                         February 2013 | Volume 8 | Issue 2 | e53964 <br />                                     MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org   11   February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                     MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br /> Figure 6. Src expressed interacts MERTK RPE. (A) Src family kinase Hprt transcripts amplified mouse RPE/choroid <br /> by RT-PCR. (B) Ni2+-NTA pull downs recombinant SRC HCK GST-SH2 domains incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999 evaluated on <br /> SDS-gels. (C) Src Hck immunoreactivity western blots rat RPE/choroid RPE-J cell homogenates. (D) Ni2+-NTA pull downs RPE/ <br /> choroid protein homogenates RCS congenic dystrophic rats incubated 6xHis-rMERTK571&#226;&#8364;&#8220;999, evaluated western <br /> analysis antibodies recognizing Src Hck. (E) Src localization cryosections mouse retina/RPE/choroid. Details Figure 1. <br /> doi:10.1371/journal.pone.0053964.g006 <br />  <br /> AlexaFluor 488 anti-rabbit IgG (1:500); AlexaFluor 555 anti-                target plates peptides separated liquid chromatogra- <br /> mouse IgG. Images obtained confocal fluorescence                    phy. Phosphopeptide analysis separated peptides was <br /> microscopy (Leica SP5). For immunohistochemistry RPE-J                 performed using 4700 MALDI TOF/TOF mass spectrometer <br /> cells, cultures challenged isolated bovine rod OS             (Applied Biosystems) peptide mass analysis using UniProt by <br /> [50], washed 3 times PBS, fixed 4% para-                      University Michigan Protein Core Facility. <br /> formaldehyde 30 min room temperature. The cells were <br /> then processed using methods retinal cross sections.        rMERTK Pull Downs rSH2-domains Native SH2- <br />                                                                             domain Proteins <br /> rSH2-domain Protein Expression Purification                                Purified GST-tagged-rSH2-domain proteins (10 mg) in- <br />    GST-tagged constructs pGEX2T vectors encoding                     cubated 6xHis-rMERTK571&#226;&#8364;&#8220;999 (10 mg) 50 mM <br /> phosphotyrosine-recognition sequences SH2-domain proteins                NaH2PO4, 300 mM NaCl, 10 mM imidazole, 0.1 mM PMSF <br /> were previously generated library representing nearly            1 h 4uC. 50 mL Ni2+-NTA resin slurry added the <br /> complete set known SH2-domain proteins [51]. The constructs              incubation continued additional 0.5 h. The beads were <br /> were transformed BL21 DE3 Gold bacteria large scale                collected brief centrifugation, washed times binding <br /> cultures grown Terrific Broth glycerol plus ampicillin         buffer including 50 mM imidazole, eluted binding buffer <br /> at 37uC OD600 0.8. Isopropyl b-D-1-thiogalactopyrano-              containing 500 mM imidazole. Negative controls omitted the <br /> side (final concentration 0.1 mM) added cells              6xHis-tagged-rMERTK571&#226;&#8364;&#8220;999. For native tissue, protein homo- <br /> incubated 15uC overnight. The cells pelleted                    genates RPE/choroid obtained described western <br /> resuspended phosphate buffered saline lysed French                analysis. Ni2+-NTA pull downs protein homogenates with <br /> press. Glutathione-agarose beads incubated cleared                6xHis-tagged-rMERTK571&#226;&#8364;&#8220;999 (10 mg) performed de- <br /> lysates 1 h 4uC, washed 10 volumes PBS- 1%                   scribed above. <br /> TritonX100, eluted buffer containing 8 mM glutathione, <br /> 50 mM Tris-HCl, pH 9.5. Fractions collected analyzed               Cell Culture Transfections <br /> on SDS gels, fractions containing rSH2-domains pooled,                HEK-293T cells maintained DMEM supplemented <br /> concentrated 1 mL, loaded Sephacryl S-200 HR                    10% FBS, 1 mM sodium pyruvate, 1 mM penicillin/ <br /> column (2161 in). Fractions collected rate 0.5 mL/             streptomycin 37uC 5% CO2. HEK-293T cells were <br /> min, analyzed SDS gels, containing purified rSH2-              transiently transfected full-length MERTK kinase-dead <br /> domains pooled concentrated.                                       R844C-MERTK using FuGENE recommended (Roche). Rat <br />                                                                             RPE-J cells maintained Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s <br /> rMERTK Expression Purification                                          medium (DMEM) supplemented 4% fetal bovine serum <br />    Two His-tagged expression constructs encoding human                  (FBS), 1 mM non-essential amino acids 33uC 5% CO2. <br /> MERTK cytoplasmic domain, amino acid residues 571 864                    Rat Grb2 siRNAs obtained Smartpool (Thermo <br /> (6xHis-rMERTK571&#226;&#8364;&#8220;864) [23] 571 999 (6xHis-                           Scientific) containing mixtures different duplexes to <br /> rMERTK571&#226;&#8364;&#8220;999), pET28a-LIC vector amplified                  minimize silencing unintended targets. ON-TARGET plus <br /> bacterial cells described rSH2-domains,                   non-targeting siRNA (at concentration total pool <br /> kanamycin replacing ampicillin cultures. Cells pelleted         targeting siRNAs) served negative control. RPE-J cells <br /> and resuspended lysis buffer containing 50 mM Tris-HCl,                  (32,000 cells well) passaged eight-well chamber <br /> 500 mM NaCl, 5% glycerol, 1 mM b-mercaptoethanol, 2 mM                      slides, 24 h later transfected 0.5 mg the <br /> imidazole, 200 mM phenylmethylsulfonyl fluoride (PMSF)               siRNAs plus 3.75 mL DharmaFect 3 transfection reagent as <br /> pH 8, lysed French press. Ni2+-NTA resin incubated               recommended (Dharmacon). The cells incubated the <br /> with cleared supernatants shaking 1 h 4uC, washed               siRNAs 48 h, medium changed, 24 h later the <br /> with 10 volumes 10 mM imidazole lysis buffer, eluted              cells transfected second time incubated an <br /> with 200 mM imidazole lysis buffer. The eluate concen-               additional 24 h. Cell viability assessed trypan blue staining, <br /> trated 1 mL, chromatographed Sephacryl S-200 HR                    equivalent cultures treated targeting non- <br /> described above, evaluated SDS gels, pooled, concentrat-             targeting siRNAs. Phagocytosis assays performed 5 days after <br /> ed. Recombinant MERTK autophosphorylated incubating                  siRNA transfection. <br /> with 10 mM ATP, 10 mM MgCl2 gel filtration buffer room <br /> temperature 3 h stored 280uC.                                Phagocytosis Assays <br />                                                                                Rod OS isolated bovine eyes [50] covalently <br /> Phosphotyrosine Analysis MALDI-MS                                        labeled AlexaFluor 555 [52]. RPE-J cells cultured 6 <br />    Purified, phosphorylated 6xHis-rMERTK571&#226;&#8364;&#8220;864 digested                days eight-well chamber slides, incubated 10 OS <br /> by addition porcine trypsin 50 mM ammonium bicarbonate,               cell 4 h 33uC. Unbound OS removed washing <br /> 0.05% SDS, incubated overnight 37uC. The digested                    cells 3 times PBS containing 0.2 mM CaCl2 1 mM <br /> peptides subjected TiO2 selection enrich phosphor-           MgCl2, cells fixed 4% paraformaldehyde. To <br /> ylated peptides evaporated dryness SpeedVac. The                distinguish total bound OS, duplicate samples incubated <br /> sample dissolved 5 mL 60% Acetonitrile 0.1%                      fixation 0.2% trypan blue quench fluorescence [53], <br /> Trifluoroacetic acid. 1 mL sample spotted MALDI                   shown Figure S2B. Slides mounted using Prolong Gold <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             12                         February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                           MERTK Interactions SH2-Domain Proteins <br />  <br />  <br />  <br />                                                                                  times indicated, followed western blot analysis. Results for <br />                                                                                  pY416 Src plotted mean fold change band intensity. Error <br />                                                                                  bars represent mean 6 SEM, n = 3. <br />                                                                                  doi:10.1371/journal.pone.0053964.g007 <br />                                                                                  containing DAPI, visualized fluorescence microscopy <br />                                                                                  (Eclipse E800; Nikon). Images taken 10&#226;&#8364;&#8220;12 random <br />                                                                                  fields (0.100 mm2 field) group. OS diameter of <br />                                                                                  2 mm quantified manually viewing field. Counts <br />                                                                                  ingested AlexaFluor 555-labeled OS obtained trypan <br />                                                                                  blue-treated samples, total OS counts (bound plus ingested) <br />                                                                                  obtained duplicate untreated samples. For each <br />                                                                                  condition, assays repeated times results represented <br />                                                                                  mean 6 standard error. <br />  <br />                                                                                  Supporting Information <br />                                                                                  Figure S1 Ion fragmentation patterns MALDI-MS <br />                                                                                  phosphopeptide analysis rMERTK571&#226;&#8364;&#8220;864. Purified <br />                                                                                  6xHis-rMERTK571&#226;&#8364;&#8220;864 autophosphorylated addition of <br />                                                                                  ATP digested addition porcine trypsin. Phosphopeptides <br />                                                                                  selected TiO2 enrichment separated LC and <br />                                                                                  subjected MALDI-MS analysis. (A) Summary peptides <br />                                                                                  identified MALDI-MS analysis. (B) Ion fragmentation data for <br />                                                                                  fragments P1, P2, P3 shown. Peaks labeled letter <br />                                                                                  &#226;&#8364;&#732;&#226;&#8364;&#732;b&#226;&#8364;&#8482;&#226;&#8364;&#8482; followed numbers correspond fragments masses <br />                                                                                  matches UniProt database. Autophosphorylation <br />                                                                                  detected sites catalytic domain human <br />                                                                                  MERTK (Y749, Y753, Y754) agreement Ling et al. <br />                                                                                  [19]. <br />                                                                                  (TIF) <br />                                                                                  Figure S2 Morphology siRNA transfected RPE-J cells <br />                                                                                  quenching AlexaFluor 555-OS fluorescence RPE <br />                                                                                  phagocytosis assays. RPE-J cells transfected pool <br />                                                                                  Grb2 targeting siRNAs non-targeting siRNA control. (A) <br />                                                                                  The morphology transfected cells evaluated immunos- <br />                                                                                  taining ZO-1 phalloidin staining actin using fluorescence <br />                                                                                  confocal microscopy. (B) Trypan blue quenching AlexaFluor <br />                                                                                  555-fluorescence surface RPE-J cells transfected with <br />                                                                                  non-targeting siRNA fed OS fed 2 hours, <br />                                                                                  quenched, fixed. Confocal images representative fields <br />                                                                                  OS labeled red, DAPI-stained nuclei blue. <br />                                                                                  (TIF) <br />                                                                                  Figure S3 Rhodopsin colocalization Pik3r1, Eea1, <br />                                                                                  Rab5 OS fed RPE-J cells. RPE-J cells incubated <br />                                                                                  isolated bovine OS 4 h. Cells fixed stained with <br />                                                                                  anti-rhodopsin anti-Eea1 (A), anti-Pik3r1 anti-Eea1(B), or <br />                                                                                  anti-Rab5 anti-Eea1 (C), using AlexaFluor 488 555 <br />                                                                                  secondary antibodies, visualization using fluorescence confo- <br />                                                                                  cal microscopy. Areas showing co-localization appear yellow in <br />                                                                                  merged images marked white arrowheads. <br />                                                                                  (TIF) <br />                                                                                  Table S1 PCR primer sequences used amplify <br />                                                                                  transcripts encoding SH2-domain proteins. <br />                                                                                  (DOCX) <br /> Figure 7. Src phosphorylated Y416 response MERTK <br /> activation RPE. Immunoreactivity antibodies recognizing                Acknowledgments <br /> pY416 Src, Src, Mertk (C-terminal antibody), Gapdh western <br /> blots cell tissue homogenates. (A) HEK-293T cells transfected             The authors like thank Xudong Huang (Structural Genomics <br /> with length MERTK (Mer) kinase-dead R844C-MERTK (MerKD). (B)             Consortium) providing rMERTK571&#226;&#8364;&#8220;864 protein MERTK- <br /> RPE/choroid homogenates 4 wk old RCS congenic dystrophic                expression constructs; Douglas Vollrath (Stanford University) anti- <br /> rats harvested 1 h prior light onset, 1.5 h post light onset           bodies MERTK carboxy-terminal domain; Ron Bush (National <br /> (peak phagocytic uptake). CD, congenic rat RPE/choroid dark;           Eye Institute) providing RCS-p+ congenic RCS-rdy+ p+ rats; the <br /> DD, dystrophic dark; CL, congenic peak activity; DL, dystrophic        University Michigan Protein Core Facility MALDI-MS analysis; and <br /> taken peak activity. (C) RPE-J cells incubated OS lysed         Kecia L. Feathers, Mitchell Gillett, Lin Jia, Steve Lentz, Austra Liepa <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  13                          February 2013 | Volume 8 | Issue 2 | e53964 <br />                                                                                                                               MERTK Interactions SH2-Domain Proteins <br />  <br />  <br /> (University Michigan, Kellogg Eye Center) Anna Dai (Samuel                             Author Contributions <br /> Lunenfeld Research Institute) technical assistance. <br />                                                                                               Conceived designed experiments: SJS DAT. Performed the <br />                                                                                               experiments: SJS. Analyzed data: SJS DAT. Contributed reagents/ <br />                                                                                               materials/analysis tools: DAT KC SD TP. Wrote paper: SJS DAT. <br />  <br /> References <br />  1. Strauss O (2005) The retinal pigment epithelium visual function. Physiol Rev                 apoptotic cells down-modulation lipopolysaccharide-inducible NF- <br />     85: 845&#226;&#8364;&#8220;881.                                                                                    kappaB transcriptional activation. J Biol Chem 283: 3618&#226;&#8364;&#8220;3627. <br />  2. Young RW (1967) The renewal photoreceptor cell outer segments. J Cell Biol             26.   Giubellino A, Burke TR, Jr., Bottaro DP (2008) Grb2 signaling cell motility <br />     33: 61&#226;&#8364;&#8220;72.                                                                                      cancer. Expert Opin Ther Targets 12: 1021&#226;&#8364;&#8220;1033. <br />  3. Young RW, Bok D (1969) Participation retinal pigment epithelium             27.   LaVail MM, Sidman RL, Gerhardt CO (1975) Congenic strains RCS rats <br />     rod outer segment renewal process. J Cell Biol 42: 392&#226;&#8364;&#8220;403.                                     inherited retinal dystrophy. J Hered 66: 242&#226;&#8364;&#8220;244. <br />  4. LaVail MM (2001) Legacy RCS rat: impact seminal study retinal              28.   LaVail MM (1981) Photoreceptor characteristics congenic strains RCS rats. <br />     cell biology retinal degenerative diseases. Prog Brain Res 131: 617&#226;&#8364;&#8220;627.                    Invest Ophthalmol Vis Sci 20: 671&#226;&#8364;&#8220;675. <br />  5. D&#226;&#8364;&#8482;Cruz PM, Yasumura D, Weir J, Matthes MT, Abderrahim H, et al. (2000)                    29.   Finnemann SC (2003) Focal adhesion kinase signaling promotes phagocytosis of <br />     Mutation receptor tyrosine kinase gene Mertk retinal dystrophic                   integrin-bound photoreceptors. Embo J 22: 4143&#226;&#8364;&#8220;4154. <br />     RCS rat. Hum Mol Genet 9: 645&#226;&#8364;&#8220;651.                                                        30.   Zoncu R, Efeyan A, Sabatini DM (2011) mTOR: growth signal integration <br />  6. Feng W, Yasumura D, Matthes MT, LaVail MM, Vollrath D (2002) Mertk                              cancer, diabetes ageing. Nat Rev Mol Cell Biol 12: 21&#226;&#8364;&#8220;35. <br />     triggers uptake photoreceptor outer segments phagocytosis cultured           31.   Simonsen A, Lippe R, Christoforidis S, Gaullier JM, Brech A, et al. (1998) EEA1 <br />     retinal pigment epithelial cells. J Biol Chem 277: 17016&#226;&#8364;&#8220;17022.                                 links PI(3)K function Rab5 regulation endosome fusion. Nature 394: 494&#226;&#8364;&#8220; <br />  7. Gal A, Li Y, Thompson DA, Weir J, Orth U, et al. (2000) Mutations                            498. <br />     MERTK, human orthologue RCS rat retinal dystrophy gene, cause                  32.   Vieira OV, Botelho RJ, Rameh L, Brachmann SM, Matsuo T, et al. (2001) <br />     retinitis pigmentosa. Nat Genet 26: 270&#226;&#8364;&#8220;271.                                                    Distinct roles class I class III phosphatidylinositol 3-kinases phagosome <br />  8. McHenry CL, Liu Y, Feng W, Nair AR, Feathers KL, et al. (2004) MERTK                            formation maturation. J Cell Biol 155: 19&#226;&#8364;&#8220;25. <br />     arginine-844-cysteine patient severe rod-cone dystrophy: loss mutant         33.   Christoforidis S, Miaczynska M, Ashman K, Wilm M, Zhao L, et al. (1999) <br />     protein function transfected cells. Invest Ophthalmol Vis Sci 45: 1456&#226;&#8364;&#8220;1463.                 Phosphatidylinositol-3-OH kinases Rab5 effectors. Nat Cell Biol 1: 249&#226;&#8364;&#8220;252. <br />  9. Tschernutter M, Jenkins SA, Waseem NH, Saihan Z, Holder GE, et al. (2006)                 34.   Bustelo XR (2000) Regulatory signaling properties Vav family. Mol <br />     Clinical characterisation family retinal dystrophy caused mutation              Cell Biol 20: 1461&#226;&#8364;&#8220;1477. <br />     Mertk gene. Br J Ophthalmol 90: 718&#226;&#8364;&#8220;723.                                              35.   Majeed M, Caveggion E, Lowell CA, Berton G (2001) Role Src kinases and <br /> 10. Brea-Fernandez AJ, Pomares E, Brion MJ, Marfany G, Blanco MJ, et al. (2008)                     Syk Fcgamma receptor-mediated phagocytosis phagosome-lysosome <br />     Novel splice donor site mutation MERTK gene associated retinitis                        fusion. J Leukoc Biol 70: 801&#226;&#8364;&#8220;811. <br />     pigmentosa. Br J Ophthalmol 92: 1419&#226;&#8364;&#8220;1423.                                                36.   Mao Y, Finnemann SC (2012) Essential diurnal Rac1 activation retinal <br /> 11. Mackay DS, Henderson RH, Sergouniotis PI, Li Z, Moradi P, et al. (2010)                         phagocytosis requires alphavbeta5 integrin tyrosine kinases focal <br />     Novel mutations MERTK associated childhood onset rod-cone                               adhesion kinase Mer tyrosine kinase. Mol Biol Cell 23: 1104&#226;&#8364;&#8220;1114. <br />     dystrophy. Mol Vis 16: 369&#226;&#8364;&#8220;377. <br />                                                                                               37.   May RC, Machesky LM (2001) Phagocytosis actin cytoskeleton. J Cell <br /> 12. Stitt TN, Conn G, Gore M, Lai C, Bruno J, et al. (1995) The anticoagulation                     Sci 114: 1061&#226;&#8364;&#8220;1077. <br />     factor protein S relative, Gas6, ligands Tyro 3/Axl family of <br />                                                                                               38.   Fujikawa K, Iwata T, Inoue K, Akahori M, Kadotani H, et al. (2010) VAV2 and <br />     receptor tyrosine kinases. Cell 80: 661&#226;&#8364;&#8220;670. <br />                                                                                                     VAV3 candidate disease genes spontaneous glaucoma mice and <br /> 13. Caberoy NB, Zhou Y, Li W (2010) Tubby tubby-like protein 1 new <br />                                                                                                     humans. PLoS One 5: e9050. <br />     MerTK ligands phagocytosis. EMBO J 29: 3898&#226;&#8364;&#8220;3910. <br />                                                                                               39.   Qin S, Rodrigues GA (2012) Roles alphavbeta5, FAK MerTK in <br /> 14. Graham DK, Dawson TL, Mullaney DL, Snodgrass HR, Earp HS (1994) <br />                                                                                                     oxidative stress inhibition RPE cell phagocytosis. Exp Eye Res 94: 63&#226;&#8364;&#8220;70. <br />     Cloning mRNA expression analysis novel human protooncogene, c- <br />                                                                                               40.   Zhang Y, Wang H (2012) Integrin signalling function immune cells. <br />     mer. Cell Growth Differ 5: 647&#226;&#8364;&#8220;657. <br />                                                                                                     Immunology 135: 268&#226;&#8364;&#8220;275. <br /> 15. Cohen PL, Caricchio R, Abraham V, Camenisch TD, Jennette JC, et al. (2002) <br />     Delayed apoptotic cell clearance lupus-like autoimmunity mice lacking              41.   Huveneers S, Danen EH (2009) Adhesion signaling - crosstalk integrins, <br />     c-mer membrane tyrosine kinase. J Exp Med 196: 135&#226;&#8364;&#8220;140.                                     Src Rho. J Cell Sci 122: 1059&#226;&#8364;&#8220;1069. <br /> 16. Wu Y, Singh S, Georgescu MM, Birge RB (2005) A role Mer tyrosine kinase               42.   Groves E, Dart AE, Covarelli V, Caron E (2008) Molecular mechanisms of <br />     alphavbeta5 integrin-mediated phagocytosis apoptotic cells. J Cell Sci 118:               phagocytic uptake mammalian cells. Cell Mol Life Sci 65: 1957&#226;&#8364;&#8220;1976. <br />     539&#226;&#8364;&#8220;553.                                                                                  43.   Gibbs D, Kitamoto J, Williams DS (2003) Abnormal phagocytosis retinal <br /> 17. Nandrot EF, Silva KE, Scelfo C, Finnemann SC (2012) Retinal pigment                             pigmented epithelium lacks myosin VIIa, Usher syndrome 1B protein. <br />     epithelial cells use MerTK-dependent mechanism limit phagocytic                        Proc Natl Acad Sci U S A 100: 6481&#226;&#8364;&#8220;6486. <br />     particle binding activity alphavbeta5 integrin. Biol Cell 104: 326&#226;&#8364;&#8220;341.                44.   Jeong SJ, Kim SG, Yoo J, Han MY, Park JC, et al. (2006) Increased association <br /> 18. Strick DJ, Feng W, Vollrath D (2009) Mertk drives myosin II redistribution                      dynamin II myosin II ras transformed NIH3T3 cells. Acta Biochim <br />     retinal pigment epithelial phagocytosis. Invest Ophthalmol Vis Sci 50:                   Biophys Sin (Shanghai) 38: 556&#226;&#8364;&#8220;562. <br />     2427&#226;&#8364;&#8220;2435.                                                                                45.   Hall MO, Agnew BJ, Abrams TA, Burgess BL (2003) The phagocytosis os is <br /> 19. Ling L, Templeton D, Kung HJ (1996) Identification major autopho-                        mediated PI3-kinase linked tyrosine kinase receptor, mer, is <br />     sphorylation sites Nyk/Mer, NCAM-related receptor tyrosine kinase. J Biol                 stimulated GAS6. Adv Exp Med Biol 533: 331&#226;&#8364;&#8220;336. <br />     Chem 271: 18355&#226;&#8364;&#8220;18362.                                                                    46.   Liu BA, Jablonowski K, Shah EE, Engelmann BW, Jones RB, et al. (2010) SH2 <br /> 20. Pawson T (2004) Specificity signal transduction: phosphotyrosine-SH2                    domains recognize contextual peptide sequence information determine <br />     domain interactions complex cellular systems. Cell 116: 191&#226;&#8364;&#8220;203.                             selectivity. Mol Cell Proteomics 9: 2391&#226;&#8364;&#8220;2404. <br /> 21. Georgescu MM, Kirsch KH, Shishido T, Zong C, Hanafusa H (1999) Biological                 47.   Sen P, Wallet MA, Yi Z, Huang Y, Henderson M, et al. (2007) Apoptotic cells <br />     effects c-Mer receptor tyrosine kinase hematopoietic cells depend                  induce Mer tyrosine kinase-dependent blockade NF-kappaB activation in <br />     Grb2 binding site receptor activation NF-kappaB. Mol Cell Biol 19:                dendritic cells. Blood 109: 653&#226;&#8364;&#8220;660. <br />     1171&#226;&#8364;&#8220;1181.                                                                                48.   LaVail MM (1976) Rod outer segment disk shedding rat retina: relationship to <br /> 22. Yi Z, Li L, Matsushima GK, Earp HS, Wang B, et al. (2009) A novel role c-                   cyclic lighting. Science 194: 1071&#226;&#8364;&#8220;1074. <br />     Src STAT3 apoptotic cell-mediated MerTK-dependent immunoregula-                    49.   Peterson GL (1977) A s <br /> </body></html>