<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 5, 899-905, August 1994 <br />  <br />  <br /> Mitochondrial Morphological Functional Defects <br /> in Yeast Caused ymel Are Suppressed <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>Mutation</span> <br /> of 26S Protease Subunit Homologue <br /> Corey L. Campbell, Noriko Tanaka, Karen H. White, <br /> and Peter E. Thorsness <br /> Department Molecular Biology, University Wyoming, Laramie, Wyoming 82071-3944 <br /> Submitted May 25, 1994; Accepted June 30, 1994 <br /> Monitoring Editor: Gottfried Schatz <br />  <br />  <br />          The absence functional Ymelp, putative ATP zinc-dependent protease localized <br />          mitochondria yeast, results abnormal mitochondrial function morphology. <br />          Yeast lacking Ymelp lose DNA mitochondria accelerated rate, fail grow on <br />          nonfermentable carbon sources 37&#194;&#176;C, severely deficient growth mitochondrial <br />          DNA suffers large deletions completely lost. In place normal reticulated mi- <br />          tochondrial network, strains lacking Ymelp punctate mitochondria grossly <br />          swollen compartments. The growth phenotypes morphological alterations evident in <br />          mutant yeast compensated mutation YNT1, essential gene yeast. <br />          The sequence YNT1 gene product indicates number related <br />          regulatory subunits 26S protease. This proteolytic activity necessary progression <br />          cell cycle implicated regulation transcription. Yntlp is <br />          distantly related Ymelp. <br />  <br />  <br /> INTRODUCTION                                                   phenotypes yeast: increased rate DNA escape <br />                                                                mitochondria subsequent migration the <br /> In budding yeast Saccharomyces cerevisiae, mito-           nucleus, inability utilize nonfermentable carbon <br /> chondrial morphology function vary according               sources ethanol glycerol 37&#194;&#176;C, slow growth <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> carbon source cell-cycle stage (Stevens, 1977,          complete glucose media 14&#194;&#176;C, severely com- <br /> 1981). Using genetic analyses, number genes in-           promised growth mitochondrial DNA deleted <br /> volved dynamic processes isolated           lost entirely (Thorsness Fox, 1993; Thorsness et <br /> (McConnell et al., 1990; McConnell Yaffe, 1992;            al., 1993) (Figure 1). <br /> Thorsness Fox, 1993). One gene YME1,                  To characterize Ymelp, used suppressor <br /> which encodes protein necessary maintaining              analysis isolate possible interacting gene products. <br /> the integrity mitochondrial compartment                 Suppressing mutations bypassed requirement <br /> (Thorsness et al., 1993).                                      Ymelp isolated ymel-Al:: URA3 strain by <br />   The nuclear-encoded YME1 gene encodes mito-                selecting revertants grew nonfermentable <br /> chondrially localized protein predicted size 82      carbon sources 37&#194;&#176;C. One reverting mutation, <br /> kDa (Thorsness et al., 1993). According sequence            yntl-1, suppressed phenotypes associated with <br /> analysis, Ymelp contains sequence elements suggestive          ymel-Al::URA3. In addition, mitochondrial mor- <br /> of ATPase zinc protease activities. The gene en-           phological abnormalities ymel strain partially <br /> coding Ymelp originally isolated genetic screen       reversed mutation. Here, iso- <br /> that identified mutations cause increased rate         lation characterization YNT1 yeast. YNT1 was <br /> of DNA escape mitochondria nucleus             identified virtue mutant allele yntl-1, reces- <br /> in yeast (Thorsness Fox, 1993).                            sive bypass suppressor intrinsic cold-sensitive <br />    The absence functional Ymelp,            growth <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> nonfermentable carbon sources. <br /> of original mutant allele, ymel-l deletion allele   Cloning sequencing revealed YNT1 identical <br /> ymel-A1 ::URA3, (Thorsness et al., 1993), results      previously sequenced gene, YTA2, exhibits se- <br />  <br /> &#194;&#169;) 1994 The American Society Cell Biology                                                                     899 <br />  C.L. Campbell et al. <br />  <br /> quence homology genes involved cell-                   ethanol/glycerol 30&#194;&#176;C fixed according methods described <br /> cycle regulation.                                                        Byers Goetch (1990). <br />  <br /> MATERIALS AND METHODS                                                    Identification Cloning YNT1 <br />                                                                          Approximately 108 cells yeast strain PTY52 (ymel-M ::URA3) <br /> Strains                                                                  plated YPEG cultured 37&#194;&#176;C. Revertant cells capable <br /> The Escherichia coli strain used preparation manipulation     respiratory growth 37&#194;&#176;C picked 1 wk incubation <br /> DNA DH5a [F-endAl hsdRl 7(rk - mk+) supE44 thi-1 X recA gyrA96       recultured restrictive conditions. Multiple rounds of <br /> relAl A(argF-laczya) U169 +80 lac ZAM15]. The genotypes S.       subculturing eventually produced independent revertants ca- <br /> cerevisiae strains used work listed Table 1. Standard     pable growth nonfermentable carbon sources restrictive <br /> genetic techniques used construct analyze various        temperature, which, NTY1, characterized work. NTY1 <br /> yeast strains (Sherman et al., 1986).                                    backcrossed PTY33 (wild-type) PTY60 (ymel-Al :: URA3), <br />                                                                          diploids sporulated, tetrads dissected. Each colony arising <br />                                                                          spore scored nutritional markers, mating type, and <br /> Media                                                                    ability grow nonfermentable carbon sources 14, 30, <br /> E. coli-containing plasmids grown LB (10 g bactotryptone,        37&#194;&#176;C. <br /> 10 g NaCl, 5 g yeast extract/l) plus 125 ,ug/ml ampicillin. Yeast      Yeast strains bearing yntl-l allele grow poorly nonfer- <br /> grown YPD (20 g glucose, 20 g bactopeptone, 10 g yeast extract,       mentable carbon sources 14&#194;&#176;C. This phenotype used basis <br /> and 40 mg tryptophan), YPEG (30 ml glycerol, 30 ml ethanol,        cloning YNT1. NTY4 (ura3-52 yntl-1) transformed 100 <br /> 20 g bactopeptone, 10 g yeast extract, 40 mg tryptophan),   jig DNA prepared multicopy, 2 ti-based S. cerevisiae genomic <br /> or SD + nutrients (6.7 g yeast nitrogen <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span> amino acids,     DNA bank (a gift Jasper Rine). Approximately 24 000 transfor- <br /> 20 g glucose, appropriate nutrients). Nutrients included      mants screened complementation cold-sensitive re- <br /> uracil 40 mg/l, adenine 40 mg/l, tryptophan 40 mg/l, lysine     spiratory deficient (Pet-cs) phenotype. Nine transformants comple- <br /> at 60 mg/l, leucine 100 mg/l. For agar plates, bactoagar      mented Pet-cs phenotype. Plasmid DNA prepared these <br /> added 20 g/l. Ethidium bromide containing medium SD + nu-          transformants used transform E. coli. Restriction mapping <br /> trients supplemented 25 Ag/ml ethidium bromide. One liter        rescued plasmids showed plasmids were <br /> sporulation media contained 10 g potassium acetate, 1 g yeast extract,   identical. The insert DNA multicopy plasmids <br /> 0.5 g glucose, 20 g bactoagar. Bactoagar, bactopeptone, bacto-       (pNT18) subcloned CEN vector (pRS316) (Sikorski and <br /> tryptone, yeast extract, yeast nitrogen base amino acids     Hieter 1989), ability low copy plasmid, pNT30, to <br /> were obtained Difco (Eietroit, MI). Ampicillin nutrients   rescue Pet-cs phenotype assayed. <br /> obtained Sigma (St. Louis, MO). <br />                                                                          Nucleic Acid Manipulations <br /> Growth Assays                                                            Restriction enzymes T4 DNA ligase purchased Bethesda <br /> The escape DNA mitochondria assayed described             Research Labs (Gaithersburg, MD) New England Biolabs (Beverly, <br /> (Thorsness Fox 1993). Briefly, yeast containing copy TRPI       MA). Standard techniques generating recombinant DNAs and <br /> integrated mitochondrial genome grown rich media         performing DNA blot hybridizations used (Maniatis et al., 1982). <br /> (YPD YPEG) agar plates replica plated minimal media       DNA sequence determined double-stranded templates using <br /> (SD + nutrients) lacking tryptophan. Those cells DNA-con-       nucleotide chain termination method (Sanger et al., 1977). <br /> taining TRP1 sequences escaped mitochondria migrated            Plasmid pNT30 subcloned, minimal complementing insert <br /> the nucleus rise colonies virtue complementation    low-copy plasmid determined. pNT53 contains 2 kilobase <br /> trpl-A1 allele nucleus. Strains DNA escapes mito-        (kb) Stu I/Xba I insert yeast genomic DNA pRS315, CEN plasmid <br /> chondria higher rate wild-type yeast increased      (Sikorski Hieter 1989), able rescue pet-cs phenotype <br /> amount papillation tryptophan prototrophy minimal media     yntl-1 mutant yeast strains. An integrating plasmid containing the <br /> plates incubating 72 h 30&#194;&#176;C.                                minimal complementing region constructed cloning ge- <br />   The ability yeast strains grow absence mitochondrial   nomic fragment pNT53 pRS306, creating pNT54. pNT54 <br /> DNA assayed culturing cells minimal media agar plates      targeted integrate chromosomal locus corresponding to <br /> containing 25 ug/ml ethidium bromide. Cells streaked single    insert digestion Sph I, cuts unique site just 5&apos; <br /> colonies plates incubated 72 h 30&#194;&#176;C. Then single     YNT1 open reading frame. <br /> colony restreaked second time media. These plates          A yntl-A1::URA3 mutation constructed follows. The 2-kb <br /> are scored second 72-h incubation 30&#194;&#176;C. We          Stu I/Xba I fragment containing YNT1 pNT53 inserted into <br /> found single colonies arising ethidium bromide-con-        Sma I/Xho I sites pBluescript KS&apos; (Stratagene, La Jolla, CA). <br /> taining plates lack mitochondrial DNA.                                   This construct digested HincII, removing 60% the <br />   The cold-sensitive phenotype yeast strains scored in-     YNT1 coding region (only 30 codons YNT1 remain 5&apos; point <br /> cubation cells 140 10-14 d. The temperature-sensitive phe-     insertion), 1.5-kb Sma I fragment containing URA3 in- <br /> notype yeast strains scored incubation cells 37&#194;&#176;C     serted. This plasmid, pNT55, containing yntl-A1l::URA3 allele <br /> for 48 h.                                                                digested HindIII Sph I. This linear DNA used to <br />                                                                          transform yeast diploid PTY33 X PTY44 uracil prototrophy. A <br />                                                                          Ura+ transformant sporulated dissected. <br /> Microscopy <br /> For confocal microscopy, strains cultured YPEG enriched <br /> galactose 30 37&#194;&#176;C harvested log phase. DiOC6 (3,3&apos;-         RESULTS <br /> dihexyloxacarbocyanine iodide) used 1 ,g/ml concentrations, <br /> a mitochondrially specific concentration. Methods equipment     Identification Isolation YNT1, Bypass <br /> used Koning et al. (1993). Strains observed using       Suppressor ymel <br /> DAPSMI, 2-(4-dimethylaminostyryl)-N-methylpyridinium iodide, on <br /> a Nikon epifluorescence microscope (Garden City, NY) similar        YME1 encodes mitochondrial protein putative <br /> results. For transmission electron microscopy strains cultured   ATPase protease activity (Figure 3A). Yeast cells <br />  <br /> 900                                                                                                              Molecular Biology Cell <br />                                                                                        . _. <br />                                                                                                         Suppression Mitochondrial Defects <br />  <br />  <br /> Table 1. Yeast strains <br />      Strain              Genotype                                                                              Source <br />      NTY1                MATa ura3-52 lys2 leu2-3,112 trpl-Ml ymel-A1::URA3 yntl-1 [p+, TRP1]                  This study <br />      NTY4                MATa ura3-52 ade2-101 leu2-3,112 trpl-A1 yntl-1 [p&apos;, TRP1]                            This study <br />      PTY33               MATa ura 3-52 ade2 leu2-3,112 trpl-Ml [p+, TRP1]                                      (Thorsness Fox, 1993) <br />      PTY44               MATa ura3-52 lys2 leu2-3,112 trpl-A1 [p+, TRP1]                                       (Thorsness Fox, 1993) <br />      PTY52               MATa ura3-52 lys2 leu2-3,112 trpl-Ml ymel-A1::URA3 [p+, TRP1]                         (Thorsness et al., 1993) <br />      PTY60               MATa ura3-52 ade2 leu2-3,112 trpl-Al ymel-A1l::URA3 [p+, TRPI]                        (Thorsness et al., 1993) <br />      PTY61               MATa ura3-52 ade2-101 leu2-3,112 trpl-A1 ymel-1 [p+, TRP1]                            (Thorsness Fox, 1993) <br />      PTY106              MATa ura 3-52 Iys2 leu2-3,112 trpl-Al ymel-1 yntl-l [p+, TRP1]                        This study <br />  <br />  <br />  <br /> lacking Ymelp number phenotypes in-                    linked ymel-Al::URA3. In crosses, Pet- <br /> dicate abnormal mitochondrial activity (Figures 1                 cs phenotype segregated 2cs:: 2wt. This inability to <br /> 2) (Thorsness et al., 1993). To learn                  utilize nonfermentable carbon sources 14&#194;&#176;C cose- <br /> function protein, genetic approach taken                gregated suppressor locus (Figure 1A). This <br /> to identify proteins processes com-               genetic locus designated YNT1 mutant <br /> pensate mutations YME1. Initially, attempts               allele yntl-1. <br /> made second-site revertants <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>point mutation</span> <br /> in YME1, tightly linked (presumably intragenic) <br /> revertants found. Therefore, spontaneous bypass                  A <br /> revertants ymel-Al::URA3 strain, PTY52,                         YPEG 370        YPD 14&#194;&#176;           YPEG 140 <br /> isolated culturing restrictive growth con- <br /> dition, 37&#194;&#176;C ethanol/glycerol medium (Thorsness                                                                          wildt gpe <br /> and Fox, 1993; Thorsness et al., 1993). One isolate, des-                             r                 1         1 <br /> ignated NTY1 containing nuclearly encoded                                     _                                     ymel <br /> suppressor yntl-l (see below), suppressed                                    X         ]       L_S                 ymeI, yntI <br /> growth phenotypes associated ymel (Figure 1). Al- <br /> though reverting mutation suppressed temper-                                                                        yntl <br /> ature-sensitive growth phenotype (Pet-ts), growth                                                                       ymel, yntl/pYNT1 <br /> still wild-type condition <br /> (Figure 1A). Also, cold sensitivity enriched media <br /> containing glucose galactose completely reversed.              B                      S D -Trp              S iD+EtBr <br /> In addition, suppressing mutation NTY1 delayed                                     wildt!ype                         Idtyjpe <br />                                                                                                                        lb <br /> onset mitochondrial DNA (mtDNA) escape (Figure <br /> 1B), resulting reduced papillations colonies <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Trp- Trp+. The suppressor reversed growth                                ymeJ                                      yme1 <br /> deficiency ymel (Figure 1C) presence a <br /> partially completely lost mitochondrial genome, as <br /> induced growth media containing ethidium                                ymel, yntl                                      ymel, yntl <br /> bromide. <br />    In backcrosses NTY1 parental strain op- <br /> posite mating type (PTY60, ymel-A1 ::URA3),                       Figure 1. yntl-1 suppresses functional defects mitochondria ymel <br /> suppression Pet-ts phenotype segregated 2ts::                  yeast strains. (A) Yeast indicated mutations inoculated <br /> 2wt tetrads dissected, indicating single nu-             equivalent concentrations YPEG YPD incubated the <br /> clear locus responsible suppressor activity.              indicated temperatures (Thorsness Fox, 1993). A plasmid bearing <br /> This suppressor determined recessive,                  YNT1 (pYNT1) restored growth ymel, yntl-I yeast strain YPEG <br /> as evidenced inability NTY1 X PTY60                     14&#194;&#176;C. (B) Escape DNA containing TRP1 gene mito- <br />                                                                       chondria subsequent migration nucleus allows comple- <br /> diploid grow nonfermentable carbon sources                   mentation nuclear trpl-A1 mutation. The increased rate DNA <br /> 37&#194;&#176;C. yntl-l suppressed phenotypes strains                escape mitochondria ymel strain suppressed yntl- <br /> bearing ymel-1, point mutation allele YME1                       1. (C) Complete loss mitochondrial DNA induced culturing <br /> (Thorsness et al., 1993). Backcrossing NTY1                     yeast presence 25 gg/ml ethidium bromide (Thorsness et <br /> PTY33 (wild-type) sporulation allowed recov-                  al., 1993). The severe growth deficiency ymel strains mi- <br />                                                                       tochondrial DNA suppressed yntl-1. Strains used figure <br /> ery Pet-ts spores (tetratype nonparental ditype                follows: wild-type PTY44, ymel PTY52, ymel yntl NTY1, yntl <br /> tetrads), indicated reverting mutation                  NTY4, ymel yntl/pYNT1 PTY106 containing pNT53. <br />  <br /> Vol. 5, August 1994                                                                                                                        901 <br />  C.L. Campbell et al. <br />  <br />  A <br />  <br />  <br />  <br />  <br />  &apos;&apos;&lt; <br />  .,\,, <br />       \st.*, ., . _s <br />       .w.&apos; <br />  <br />  <br />       _s.- <br />                        ;.\; <br />  <br />                                w <br />  <br />                               5S <br />                                X <br />                               _tAK <br />                                                  G <br />  <br />  <br />  <br />  <br />                                                                        Figure 2. Morphological defects mitochondria a <br />                                                                        ymel mutant yeast strain suppressed yntl-l <br />                                                                        mutation. Transmission electron <br />                                                                        focal fluorescence images wildtype (A-C), ymel (D- <br />                                                                        F),     ymel <br />                                                                                                               micrographs <br />  <br />                                                                                           yntl-l (G-I) yeast strains revealed altered <br />                                                                        mitochondrial profiles. For electron micrographs (A, <br />                                                                        D, G) yeast grown YPEG 30&#194;&#176;C. Mito- <br />                                                                        chondria indicated arrows. Bar, 1 ,um. Vacuoles <br />                                                                        containing osmium-dense material (D) occasionally <br />                                                                        detected wild-type mutant yeast. For the <br />                                                                        confocal fluorescence images (B, C, E, F, H, I), yeast <br />                                                                        grown YPEG 30&#194;&#176;C (B, E, H) on <br />                                                                        rich galactose media 37&#194;&#176;C (C, F, I), mito- <br />                                                                                                                               con- <br />  <br />  <br />  <br />  <br />                                                                        chondria stained DiOC6 (3,3&apos;-dihexyloxacar- <br />                                                                        bocyanine iodide). Confocal images constructed of <br />                                                                        optical sections (0.5 ,um each) Z-series <br />                                                                        scan. Bar, 5 Am. Strains used figure follows: <br />                                                                        wild-type PTY44, ymel PTY52, ymel yntl-l NTY1. <br />  <br />  <br /> Morphology Mitochondrial Compartments                     observed 12% (chi square test, p &lt; 0.001) of <br /> in Wild-Type, ymel, ymel/yntl-l Yeast                   PTY52 cells observed, compared &lt;3% (p &lt; 0.001) of <br /> Analysis mitochondrial compartment super-structure       revertant strain NTY1. yntl-1, present a <br /> using confocal fluorescence electron microscopy         wild-type YME1 background, apparent affect on <br /> revealed significant morphological alterations yeast     mitochondrial superstructure evidenced epi- <br /> lacking Ymelp (Figure 2). Under nonrepressing con-          fluorescence microscopy. <br /> ditions log phase, wild-type yeast complex <br /> reticulated network mitochondrial compartments           Cloning Sequencing YNT1 <br /> (Figure 2, A-C). In contrast, yeast bearing ymel-Al::       The cold-sensitive phenotype yntl-l strains was <br /> URA3 abbreviated globular mitochondria          used isolate wild-type gene. YNT1 origi- <br /> subpopulation cells containing grossly swollen       nally cloned 2-,u (multicopy) plasmid bank by <br /> compartments (Figure 2, D-F). Altered mitochondrial         complementing cold-sensitive growth nonfer- <br /> morphology necessarily consequence muta-        mentable carbon sources associated yntl a <br /> tions inhibiting oxidative phosphorylation,         plasmid-dependent fashion. However, gene also <br /> several strains containing unrelated mutations affecting    complemented phenotype low-copy <br /> this process retain normal mitochondrial profiles. Also,    CEN plasmid (Figure 1A). To confirm isolated <br /> this abnormal morphology merely characteristic       genomic DNA corresponded yntl-l locus, the <br /> of growth conditions requiring respiratory            minimal complementing genomic fragment cloned <br /> growth, punctate mitochondrial forms ev-        integrating URA3 plasmid. This plasmid was <br /> ident fermentable medium galactose        targeted integrate yeast genome digesting <br /> (Figure 2, C, F, I). The morphological abnormalities    restriction endonuclease Sph I, cuts <br /> in ymel -Al:: URA3 strain partially reversed   just 5&apos; YNT1 open reading frame (described be- <br /> yntl-l suppressor gene (Figure 2, G-I). However, re-        low). This linear DNA transformed PTY61, a <br /> vertants display shorter branching mitochondrial seg-       ymel-I strain. The resulting transformant then <br /> ments wild-type yeast.                              mated PTY106 (ymel-1, yntl-l). The resulting diploid <br />    The distended organelles caused ymel             sporulated, tetrads dissected. The inte- <br /> counted using DAPSMI staining epifluorescence           grated URA3 marker segregated opposition the <br /> microscopy. Grossly swollen mitochondrial profiles          suppressor activity 19 20 tetrads examined, <br />  <br />  <br /> 902                                                                                                   Molecular Biology Cell <br />                                                                                                                 Suppression Mitochondrial Defects <br />  <br />                                        A <br />                                        Ymelp   FrVaSKILVSP   TVTTN       LPQIGAS LI                 LRF(CFYR}SE KNSGNMPPKX 4 pDS1              75 <br />                                        Yntlp     -------     ----   2I3 ----VEKA --I                 K- ----j- --ASLL----                      41 <br />  <br />                                                              IT           A VAV8          I               TS <br />                                                                                                            ! i1 <br />                                        rmelp   SIDNSQPPPP            N <br />                                                                           _ <br /> Figure 3. Yntlp belongs ex- <br />                                        Yntlp   NVN --------                   KLlErKCE!       L*--- -&quot;~   IK--A&quot;-----                          93 <br />  <br /> tended putative ATPase family. (A) !melp ECKLr L QRIGRNSEAD IiQNLLTAS SAGr4P4                        SNQSG!HlNGNCPiGS1lGSRKNPLHV VVS4 225 <br /> Alignment Ymelp Yntlp Yntlp LVIGQFIJI DQ--NTG                           ----STT GXSIVR- 1$LDRELLK -- S!V&amp;RNSNALVDI LPP1              154 <br /> amino acid sequences. Identical <br /> amino acids boxed, nucleotide Ymelp TFTVVSRWVK WLLVFGILTY SFSEGFIIT NNTTLLKSSE                KS      NVKFDlG   CD    9 IVDFiq 300 <br /> binding motifs (Chin et al., 1988) Yntlp DSSI------ -----SVMGE NEKPDV- --------AD                 -        IR- --       -   ---- <br />                                                                                                                                             192 <br /> underlined, putative neutral Ymelp X <br /> zinc protease motif marked *.                  DPTUK <br />                                                        SAKA <br />                                                     TfDTPGG TAIt <br />                                                                                                                                             375 <br /> The Yntlp Ymelp sequences rntlp <br />                                              K                                                           SEP            K                   267 <br /> were aligned aid !melp jlIiP YAKQTLNQIS--GII4+ Tp                                                                IVVNV 445 <br /> computer program GeneWorks 2.2 Yntlp D1          D                                    3fR <br />                                                                                         _TINVKVF                           R          I3PP 341 <br /> from IntelliGenetics (Mountain <br /> View, CA); zinc protease motif Ymelp         MMNI,DT        I       T           TII ARGTP4SiAI ILVN             VS V D DKILM 520 <br /> was identified Blocks sequence Yntlp                      GTIA&quot; Z1F DSL IIRNDGjAiNQd G L                      !V IL         ATQVK 416 <br /> analysis program (Henikoff and <br />                                        Ymelp GABRKTMVLT DAARKATAFH EAGNAIKAK! TNGATPL!KA TILPRGRALG ITFQLPENDK VDITKRBCQA RLDVC 595 <br /> Henikoff, 1991). (B) Yntlp belongs Yntlp     TDNTVDKFDF YK <br /> to subfamily ATPase proteins <br /> that regulatory subunits Ymelp MGGRIA1ELI YGKDNTTSGC GSDLQSATGT ARANMTQYGN SDDVGPVNLS ENWKSWSNKI RDIADNEVIE LLKDS 674) <br /> the 26S protease. The degree sim- Ymelp EERARRLLTK KNVELHRLAQ GLIEYETLDA HEIEQVCKGK KLDKLKTSTN TVVEGPDSDE RKDIGDDKPK IPTML 745 <br /> ilarity displayed using grayscale Ymelp NA                                                                                             747 <br /> (Fr6lich, 1994). Percentage amino <br /> acid identity Yntlp calculated <br /> for subsets seven amino acids, <br /> each subset displaced the <br /> next residue, producing B                 0% 25% 50% 75% 100% identity Yntl p <br /> overlapping values percentage <br /> identity sequence. Values <br />                                            B~~                        . <br />  <br />  <br />  <br />  <br /> &lt;20% identity displayed                    0                         100                 200r-1         n   300                   400 aa <br /> white, values equal 100% Yntl p <br /> black, thresholds,               I I . 51 <br /> gray values equivalent pro- S4 <br /> portion    identity used. The   Mts2p       P. FIT,.1 , Am <br /> ATPase sequence elements in- Cim5p             I U.- ID&apos;       1. .     -,,If s ==..                                   H-M mm.. I M. <br /> dicated brackets.                   Sug1p             A Ai -v&apos;m              F K, NM                                    . f W. 71, <br />  <br />  <br /> which confirmed isolated DNA did fact contain                            pletion mitosis Schizosaccharomyces pombe (Gordon <br /> the YNT1 locus. Presumably, tetrad showed                           et al., 1993); Cim5p Suglp, 26S protease sub- <br /> 3ts:: lwt undergone gene conversion.                                         units necessary degradation ubiquitinated sub- <br />    Sequence analysis YNT1 revealed identical                       strates anaphase chromosome separation S. <br /> to previously identified yeast gene unassigned                              cerevisiae (Ghislain et al., 1993). <br /> function, YTA2, <span id='am-1' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> X73570 <br /> (Schnall et al., 1994). The YNT1 nucleic acid sequence                           YNT1 Is Essential Gene <br /> has assigned <span id='am-2' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>GenBank accession number</span> U06229. <br />    Yntlp homologous extended family pro-                             One chromosomal copy YNT1 deleted the <br /> teins, including Ymelp (Figure 3A), related                          diploid strain PTY33 X PTY44. The deleted locus was <br /> a 200-amino acid region containing consensus ATPase                            marked URA3 DNA (see MATERIALS AND <br /> element. Proteins family implicated                            METHODS) verified DNA-blot hybridization. <br /> in divergent processes, ranging vesicle fusion (Diaz                        This diploid sporulated dissected, only <br /> et al., 1989) transcription (Ohana et al., 1993). It re-                      viable spores recovered 20 <br /> mains unclear sequence homologies                                  tetrads dissected. None viable spores Ura+, <br /> these family members correspond functional simi-                              indicating functional copy YNT1 necessary <br /> larities.                                                                        cell viability. <br />    Yntlp high degree sequence homology to <br /> several genes identified regulatory                            DISCUSSION <br /> subunits 26S protease (Figure 3B): subunit 4 of <br /> the 26S protease humans (Dubiel et al., 1992);                              We demonstrated Ymelp necessary for <br /> Mts2p, 26S protease subunit necessary com-                             proper mitochondrial function structure yeast. <br />  <br /> Vol. 5, August 1994                                                                                                                            903 <br />  C.L. Campbell et al. <br />  <br /> Ymelp sequence elements suggest possible ATP-de-               1991). A role ubiquitination proteins organelle <br /> pendent neutral zinc protease activity (Thorsness et al.,      formation proposed based obser- <br />  1993) (Figure 3). Ymelp mitochondrial protein            vation PAS2 S. cerevisiae, gene necessary for <br /> (Thorsness, 1993, #595). More recent work lab-          peroxisome biogenesis proliferation, encodes a <br /> oratory demonstrated Ymelp tightly asso-           member ubiquitin-conjugating protein family <br /> ciated inner mitochondrial membrane (Weber            (Wiebel Kunau, 1992). The 26S protease known <br /> and Thorsness, unpublished data). A closely related            act ubiquinated nonubiquinated sub- <br /> protein bacteria containing sequence             strates (Hershko Ciechanover, 1992; Murakami et <br /> elements, FtsH protein, identified screen         al., 1992). If Yntlp, concert 26S protease com- <br /> for cell division mutants E. coli (Begg et al., 1992;       plex, acts subset mitochondrial outer membrane <br /> Tomoyasu et al., 1993). This homology, considered         proteins proteins destined interior mito- <br /> the mitochondrial morphology abnormalities de-             chondria, yntl-l mutation alters sub- <br /> ficient mitochondrial function ymel-Al ::URA3             strate specificity complex include substrates <br /> strain, suggests Ymelp plays role rearranging          normally acted Ymelp. If so, pleiotropic <br /> maintaining proper mitochondrial superstructure.           growth phenotypes associated ymel merely <br /> If Ymelp functions protease, phenotypic con-          demonstrate results lack maturation mul- <br /> sequences loss Ymelp activity result         tiple substrates. A mutated Yntlp partially com- <br /> from general reduction mitochondrial protein turn-        pensate range ymel phenotypes tar- <br /> over. The punctate distended mitochondrial com-            geting substrates lowered efficiency compared to <br /> partments ymel mutant strain result directly          Ymelp. Alternatively, mutant Yntlp may <br /> from excess abnormal mitochondrial proteins. Al-         recognize subset Ymelp substrates. <br /> ternatively, Ymelp maturase activity             This work provided intriguing genetic mi- <br /> subset mitochondrial proteins. Such unprocessed             croscopic evidence possible association the <br /> proteins lead poorly assembled protein com-             26S protease mitochondrial function mor- <br /> plexes, resulting structurally functionally im-         phology. Localization Yntlp yeast cell will <br /> paired mitochondrial compartments.                             required elucidation function. <br />    An altered form Yntlp partially compensates for <br /> the absence Ymelp. yntl-l mediate this <br /> suppression ymel-Al::URA3 different ways.            ACKNOWLEDGMENTS <br /> First, Yntlp altered way         We like thank Tom Fox, Teru Ogura, Michael Reith, Eric <br /> gain Ymelp-like function. Typically gain-of-func-         Weber, Mary Thorsness helpful discussions. We thank <br /> tion mutations dominant; however, yntl-l displays          Robin Wright assistance microscopy. This work sup- <br /> a recessive phenotype. In second model, Ymelp            ported United States Public Health Service grant (GM-47390) <br />                                                                Junior Faculty Research Award American Cancer So- <br /> Yntlp function competing pathwa <br /> </body></html>