<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Decoupling catalytic activity biological function of <br /> the ATPase powers lipopolysaccharide transport <br /> David J. Shermana, Michael B. Lazarusa,b, Lea Murphyc, Charles Liua, Suzanne Walkerb, Natividad Ruizc,1, <br /> and Daniel Kahnea,d,1 <br /> a <br />  Department Chemistry Chemical Biology, Harvard University, Cambridge, MA 02138; Departments bMicrobiology Immunobiology and <br /> d <br />  Biological Chemistry Molecular Pharmacology, Harvard Medical School, Boston, MA 02115; cDepartment Microbiology, The Ohio State University, <br /> Columbus, OH 43210 <br />  <br /> Edited M. J. Osborn, University Connecticut Health Center, Farmington, CT, approved February 10, 2014 (received review December 17, 2013) <br />  <br /> The cell surface Gram-negative bacteria contains lipopolysac-             spans compartments cell, cytoplasm the <br /> charides (LPS), provide barrier entry            OM (15, 16) (Fig. 1). E. coli seven different Lpt proteins that <br /> antibiotics. LPS assembly involves multiprotein LPS transport              essential LPS transport cell viability (17, 18). Three <br /> (Lpt) complex spans cytoplasm outer mem-                Lpt proteins form ABC composed het- <br /> brane. In complex, unusual ATP-binding cassette trans-               eromeric TMD complex (LptF LptG) homodimeric <br /> porter thought power extraction LPS outer              NBD complex (LptB). LptB, complex with <br /> leaflet cytoplasmic membrane transport             LptF LptG, ATPase activity vitro (19&#226;&#8364;&#8220;21). In addition, <br /> cell envelope. We introduce changes nucleotide-binding              LptB2FG complex closely associated bitopic IM <br /> domain, LptB, inactivate transporter function vivo. We               protein LptC, binds LPS (20&#226;&#8364;&#8220;23) trans- <br /> characterize residues using biochemical experiments com-               envelope bridge (15, 16). Based findings, current <br /> bined high-resolution crystal structures LptB pre- post-         model LptB2FG extracts LPS outer leaflet of <br /> ATP hydrolysis suggest role active site residue              IM sole energy input responsible entire <br /> phosphate exit. We identify conserved residue             process transport assembly LPS cell surface <br /> required ATPase activity essential interaction           concentration gradient (22). Coupling ABC trans- <br /> the transmembrane components. Our studies establish essen-               porter Lpt periplasmic bridge OM translocon enables <br /> tiality ATP hydrolysis LptB power LPS transport cells        cytoplasmic ATP drive periplasmic transit. Although hetero- <br /> suggest strategies inhibit transporter function away             meric architecture separate NBDs TMDs resembles that <br /> LptB active site.                                                            bacterial importers, LptB2FG perform unique function <br />                                                                              places class distinct traditional importers ex- <br /> ABC transporter   | antibiotic target | membrane biogenesis                  porters; LptB, cytoplasmic ATPase, power extrac- <br />                                                                              tion glycolipid periplasmic face IM. <br />  <br /> A    TP-binding cassette (ABC) systems represent the <br />      largest protein superfamilies domains life (1, 2). <br /> Many systems transporters share common <br />                                                                                 <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> understand LptB2FG transporter works, changed <br />                                                                              residues implicated important activities LptB and <br />                                                                              showed essential vivo function. One residue is <br /> architecture transmembrane domains (TMDs)                     proposed essential active site glutamate, two <br /> cytoplasmic nucleotide-binding domains (NBDs), bind                    residues essential catalytic activity purified <br /> and hydrolyze ATP. They couple energy ATP binding                 LptB. We residues affects ATPase <br /> hydrolysis transport variety substrates against <br /> a concentration gradient. Although eukaryotic ABC transporters                    Significance <br /> involved human diseases received attention (3), the <br /> canonical ABC systems extensively studied                     Gram-negative bacteria contain unusual outer membrane <br /> are Gram-negative bacterial importers (2).                                        prevents entry currently available antibiotics. <br />    Gram-negative bacteria, Escherichia coli, unique                This membrane contains complex glycolipid, LPS, ex- <br /> double-membrane architecture allows colonize harsh                   terior. It understood large molecule, which <br /> environments. The inner membrane (IM) contains phospholip-                        contain hundreds sugars fatty acyl chains, is <br /> ids, outer membrane (OM) asymmetric lipid                       transported cell envelope site synthesis in <br /> bilayer composed phospholipids inner leaflet LPS                    cytoplasmic membrane cell surface. Using combi- <br /> in outer leaflet (4). These membranes separated                       nation genetics, biochemistry, structural biology, we <br /> by aqueous periplasmic compartment. LPS complex                           characterized residues protein powers LPS transport <br /> glycolipid composed hundreds sugars attached core                      gain mechanistic insight ATP hydrolysis coupled <br /> containing fatty acyl chains (Fig. 1). Millions LPS molecules                  biological function transporter. These tools help <br /> must properly assembled division cycle (4)                         understand design antibiotics targeting essen- <br /> on cell surface establish permeability barrier pre-                 tial pathway. <br /> vents entry hydrophobic molecules, including antibiotics <br />                                                                              Author contributions: D.J.S., M.B.L., N.R., D.K. designed research; D.J.S., M.B.L., L.M., <br /> (5). Therefore, understanding LPS assembled OM <br />                                                                              C.L., N.R. performed research; D.J.S., M.B.L., L.M., S.W., N.R., D.K. analyzed data; <br /> could lead development better strategies target                 D.J.S., S.W., N.R., D.K. wrote paper. <br /> Gram-negative infections. In 1972, established               The authors declare conflict interest. <br /> biosynthesis LPS completed outer leaflet IM.             This article PNAS Direct Submission. <br /> Because LPS passively aqueous peri- <br />                                                                              Data deposition: The atomic coordinates structure factors deposited the <br /> plasm OM, recognized               Protein Data Bank, www.pdb.org (PDB ID codes 4P31, 4P32, 4P33). <br /> machinery transport LPS cell envelope (6, 7).                  1 <br />                                                                                  To correspondence addressed. E-mail: ruiz.82@osu.edu kahne@ <br />    In past decade, components essential                   chemistry.harvard.edu. <br /> transport assembly LPS identified (8&#226;&#8364;&#8220;14).                   This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> These LPS transport (Lpt) proteins comprise complex                   1073/pnas.1323516111/-/DCSupplemental. <br />  <br />  <br />  <br /> 4982&#226;&#8364;&#8220;4987 | PNAS | April 1, 2014 | vol. 111 | no. 13                                                               www.pnas.org/cgi/doi/10.1073/pnas.1323516111 <br />                                                                                             source wild-type LptB encoded pRC7KanLptB, <br />                                                                   O-Antigen                unstable, segregation-defective plasmid rapidly lost in <br />                                                           Glu         Hep <br />                                                                                            population, causing cell death (details strain con- <br /> LPS                                                                                        struction provided SI Materials Methods Tables <br />                           D                               Glu                              S1&#226;&#8364;&#8220;S3). When plasmid encoding functional LptB LptB- <br />       OM <br />                                                                                            His8 variant introduced strain, cells lost <br />                                                           Glu         Gal <br />  PL <br />                                                                                    Core <br />                                                                                            pRC7KanLptB viable; contrast, plasmid encod- <br />                       E <br />                                                      P <br />                                                           Hep         Hep                  ing nonfunctional LptB LptB-His8 variant introduced <br />                                                                                            strain, cells lost pRC7KanLptB died. With this <br />                                                           Hep <br />                                                 PEtN                                       approach, unlike wild-type LptB-His8 LptB, <br />       Periplasm <br />                                A                          Kdo         Kdo                  E163Q, H195A, F90A variants nonfunctional <br />                                                           O <br />                                                                                            (Fig. 2). This loss function result reduced protein <br />                                                  O <br />                                             HO- P             O        O                   levels (Fig. S1A). We tested F90Y variant an <br />                                C               O <br />                                                      OO <br />                                                  O                NH HO O           O      aromatic residue typically position other <br />  PL                                                       O        O            NH P OH <br />                                            O <br />                                                 O <br />                                                           O     HO <br />                                                                             O     O   O-   NBDs. We LptB LptB-His8 F90Y variants <br />       IM          F       G                          O                     HO              functional vivo. <br />  PL                                                                              Lipid A      We introduction plasmid-encoded LptB- <br />       Cytoplasm                                                                            E163Q-His8 LptB-H195A-His8 variants wild-type <br />                   B       B                                                                strain resulted increased OM permeability (Fig. S1B). The <br />                                                                                            simplest explanation dominant-negative effect the <br />               ATP         ADP + Pi                                                         nonfunctional variants substitute wild-type LptB Lpt <br />                                                                                            IM complexes, reducing number viable trans- <br /> Fig. 1. LPS transported cell surface E. coli essential Lpt pro- <br />                                                                                            porters compromising LPS transport. In contrast, mer- <br /> teins. (Right) The E. coli LPS structure composed lipid A (endotoxin),               odiploid strain harboring LptB-F90A-His8 variant did not <br /> a core region, O-antigen. (Left) Lpt proteins form transenvelope                 display increased OM permeability (Fig. S1B), suggesting it <br /> complex transports LPS site synthesis (IM), aqueous            replace wild-type LptB IM complex. <br /> periplasm, cell surface. LPS transport assembly believed to <br /> be powered ABC transporter LptB2FG IM (yellow), be-                          Residue F90 Is Essential Proper Formation Lpt IM Complex. <br /> cause ATP periplasm. A, LptA; B, LptB; C, LptC; D, LptD; E,             We assessed LptB variants produced the <br /> LptE; EtN, ethanolamine; F, LptF; G, LptG; Gal, D-galactose; Glu, D-glucose;               merodiploid strains interact Lpt IM components. <br /> Hep, L-glycero-D-mannoheptose; Kdo, 3-deoxy-D-manno-oct-2-ulosonic acid;                   Wild-type LptB-His8 interacts Lpt IM complex cells <br /> P, phosphate; PL, phospholipids. <br />                                                                                            copurifies endogenous LptF LptC (Fig. 3A). A lack <br />                                                                                            LptG antiserum prevented monitoring LptG, is <br /> activity intact Lpt IM complex,                             likely LptF LptG required formation of <br /> essential site binding TMDs. Based high-resolution                              stable complex LptB LptC (20). The E163Q and <br /> crystal structures LptB pre- post-ATP hydrolysis,                                H195A variants pull Lpt components, the <br /> propose residue facilitates phosphate exit                            F90A variant does not. These results consistent our <br /> the active site. These studies highlight importance                                 interpretation dominant-negative phenotypes exerted <br /> combining structural studies genetics biochemistry                             E163Q H195A variants. They F90A <br /> full complexes, individual components, understand                            variant form stable complex Lpt IM components, <br /> how unique ABC transporter functions. <br />  <br /> Results                                                                                                   In vivo function LptB LptB-His8 variants <br /> Identification LptB Residues Required Cell Viability. Using <br />                                                                                                                               Complementation lptB <br /> sequence homology NBDs, identified conserved <br /> residues located different regions LptB. E163                                          Amino acid <br />                                                                                                                                   LptB         LptB-His8 <br /> end Walker B motif essential catalysis                                             change <br />                                                                                                                None                 +                + <br /> ABC transporters. It proposed general <span id='am-162' about='xsp:base' typeof='owl:Thing'>base</span> deprot- <br />                                                                                                               E163Q                 -                - <br /> onates nucleophilic water molecule attacks &#206;&#179;-phos-                                               H195A                 -                - <br /> phate ATP (2, 24, 25). H195 located conserved switch                                              F90A                 -                - <br /> region adjacent active site NBDs impli-                                                 F90Y                 +                + <br /> cated catalytic activity, specific function debated <br /> (2, 25&#226;&#8364;&#8220;27). Lastly, F90 located Q-loop, pro- <br />                                                                                                         WT E163Q H195A       -          WT   F90A        -   WT    F90Y <br /> posed link ATP-binding site structurally diverse                             LptB-His8 <br />                                                                                                                                                                           MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> region interacts coupling helices TMDs of <br /> ABC transporters (2, 25, 28, 29). LptB shown re-                                LptB <br /> quired LPS transport and, consequently, cell viability E. coli <br />                                                                                              OmpA <br /> (11, 12). Therefore, mutated lptB determine importance <br /> of residues LPS biogenesis. Because defects in <br /> LPS assembly result phenotypes range increased OM                                                    lptB+                      lptB+                lptB <br /> permeability cell death (10, 17, 30), assessed suscep- <br />                                                                                            Fig. 2. Genetic studies indicate residues essential LptB functionality. <br /> tibility OM-impermeant antibiotics cell viability. <br />                                                                                            Functionality untagged His8-tagged LptB variants alterations in <br />    Plasmid-encoded LptB-His8 (used purification, below)                             active site Q-loop determined ability complement <br /> untagged LptB (to assess effect His8 tag) variants                              chromosomal &#206;&#8221;lptB allele. Anti-LptB Western blots residue <br /> were classified nonfunctional support cell                            changes (shown lanes) alter LptB-His8 levels. WT refers to <br /> viability absence wild-type lptB allele. To assess                        wild-type LptB-His8. Chromosomal lptB alleles strains shown below <br /> type complementation, constructed strain                            LptB-His8 bands. Cross-reacting OmpA shown loading control. <br />  <br />  <br /> Sherman et al.                                                                                                      PNAS | April 1, 2014 | vol. 111 | no. 13 | 4983 <br />                                                                                  clear electron density intact ATP (Fig. 4C). Additionally, unlike <br />                                                                                 LptB-ADP structure, prehydrolysis form crystallized a <br />                                                                                 canonical nucleotide-sandwich dimer (Fig. S2D). Secondary <br />                                                                                 structure matching alignments (35) LptB-ATP sand- <br />                                                                                 wich dimer ATP-bound E. coli MalK, NBD <br />                                                                                 maltose/maltodextrin importer [Protein Data Bank (PDB) <br />                                                                                 ID <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> 1q12], Methanocaldococcus jannaschii MJ0796 (a <br />                                                                                 homolog LolD, NBD lipoprotein exporter, PDB ID <br />                                                                                 <span id='am-163' about='Thesaurus:code' typeof='owl:Thing'>code</span> 1l2t) rmsd values 1.75 &#195;&#8230;. In cases, ATP is <br />                                                                                 sandwiched Walker A motif subunit the <br />                                                                                 signature motif opposing <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (Fig. S2E). <br /> Fig. 3. Biochemical studies indicate residues LptB essential catalysis <br />                                                                                    LptB possesses overall fold resembling NBD <br /> and proper coupling Lpt components. (A) Western blots           structures (Fig. 4A). It contains canonical L-shaped archi- <br /> affinity purifications LptB variants expressed merodipoloid strains       tecture (36) composed RecA-like &#206;&#177;/&#206;&#178;-ATPase domain and <br /> harboring plasmids expressing LptB variants. All plasmids express LptB-His8,    structurally diverse &#206;&#177;-helical domain. The RecA-like domain <br /> except untagged variant (labeled &#226;&#8364;&#339;No tag&#226;&#8364;?). Levels LptF LptC     contains Walker A Walker B motifs present many <br /> that copurify LptB shown. Levels BamA, OM protein           nucleoside triphosphate-binding proteins (37) (Fig. 4A). This <br /> nonspecifically interacts purification resin, shown sample    domain furnishes Mg2+- nucleotide-binding motifs <br /> preparation loading control. (B) LptB-His8 variants overexpressed <br />                                                                                 specific ABC proteins, namely, Q-loop, links the <br /> and purified, ATPase activity measured. Data represent the <br /> average SD experiments. WT, amino acid change LptB-His8.     highly conserved &#206;&#177;/&#206;&#178;-ATPase domain &#206;&#177;-helical do- <br />                                                                                 main, switch region, contains conserved H195. <br />                                                                                 As observed NBD structures, LSSG(E/Q) signature <br /> and does confer dominant <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>phenotype</span> mero-                        motif helical domain (25, 38) (Fig. 4A). Addi- <br /> diploid strain.                                                                 tionally, LptB structures reveal interface we <br />   The His-tagged F90Y variant pulls lesser                     predict LptB interacts TMDs LptF/G, based compar- <br /> IM Lpt components compared wild-type, E163Q,                       isons structures NBDs ABC <br /> H195A variants (Fig. 3A), showing does associate                 transporters (Fig. S2F). This interface contains grooves (Fig. 5) <br /> well IM components wild-type variant. This                      accommodate coupling helices TMDs, are <br /> defective interaction functional consequences                     involved communication NBDs TMDs in <br /> haploid strain producing LptB-F90Y-His8 shows increased OM                      ABC transporters (39, 40). <br /> permeability compared wild-type strain (Fig. S1C). Taken <br /> together, results implicate F90 association LptB                ATP Hydrolysis Induces Conformational Changes. Comparison the <br /> with LptF/G/C. They essential role E163                   pre- posthydrolysis complexes LptB demonstrates con- <br /> and H195 LptB function unrelated Lpt complex formation.                siderable movement number regions associated with <br />                                                                                 binding and/or hydrolysis &#206;&#179;-phosphate (Fig. 4B). Confor- <br /> Nonfunctional LptB Variants Show Variable Catalytic Activity. We                mational changes switch region, containing H195, and <br /> next tested ATPase activity different LptB variants               region surrounding Walker B motif, containing E163, are <br /> gain insight nature functional defects. We               apparently driven reorganization active site following ATP <br /> overexpressed purified variants stable proteins. As                  hydrolysis. This reorganization coupled changes signa- <br /> expected (19), LptB-E163Q-His8 catalytically inactive (Fig.                  ture motif Q-loop, contains F90 (Fig. 4 A B). <br /> 3B). Because variant forms stable IM complex, inability                 A closer view LptB-ATP active site reveals key <br /> to support cell viability attributed defective cata-             residues orient stabilize &#206;&#179;-phosphate re- <br /> lytic activity. Therefore, ATP hydrolysis LptB required               organize catalytic cycle maintain contact ADP <br /> LPS transport cell. We LptB-F90A-His8                     following phosphate-bond cleavage (Fig. 4 C D). Residue <br /> LptB-F90Y-His8 wild-type levels ATPase activity. Even                   163 (Q163 LptB-ATP E163 native LptB-ADP <br /> though substitutions affect protein folding cat-                structure), located end Walker B motif, H195, <br /> alytic activity, affect LptB function vivo. Finally,                located switch region, make contacts bridging water <br /> LptB-H195A-His8 variant shows reduction &#226;&#710;&#188;60% ATPase                     molecule &#206;&#179;-phosphate, respectively, LptB-ATP <br /> activity does support cell viability. This activity          structure (Fig. 4E). Glutamine-85, start the <br /> is somewhat surprising changing conserved histidine                Q-loop (giving name), forms octahedral co- <br /> in ABC transporter systems results nearly complete loss                ordination sphere surrounding metal ion associated with <br /> of ATPase activity (26, 31&#226;&#8364;&#8220;34).                                                 nucleotide (Fig. 4E). <br />                                                                                    E163 essential catalysis (Fig. 3B). A closer look the <br /> Crystal Structures LptB Bound ADP ATP. To understand              carbonyl oxygen chain residue 163 reveals is <br /> roles F90 H195 better, obtained crystal structures                 &#226;&#710;&#188;2.2 &#195;&#8230; closer nucleotide posthydrolysis struc- <br /> LptB ATP hydrolysis. We overexpressed                      ture based structural alignment Walker A motifs <br /> purified functional LptB-His8 obtained crystals               (36-GPNGAGKT-43). Through bridging water molecule, this <br /> native protein selenomethionine derivative following in-                  glutamate contacts &#206;&#178;-phosphate nucleotide (Fig. 4E). <br /> cubation ATP/MgCl2 (Fig. S2A Tables S4 S5).                        In LptB-ATP, Q163 oriented slightly farther nucle- <br /> There unambiguous density ADP-Mg2+, indicating ATP                      otide separated bridging water molecule <br /> hydrolysis occurred crystallization (Fig. S2B). This                 &#206;&#179;-phosphate (Fig. 4E). Based structural alignments <br /> 1.55-&#195;&#8230; structure referred LptB-ADP.                     E. coli MalK bound transition-state mimics (ADP-vana- <br />    To obtain prehydrolysis LptB/ATP complex, purified                  date ADP-aluminum fluoride) (41), comparison with <br /> catalytically inactive variant LptB-E163Q (Fig. 3B) obtained                ATP-bound MJ0796 structure (42), believe water is <br /> crystals preincubation ATP. The resulting 1.65-&#195;&#8230; LptB-               positioned nucleophilic water hydrolysis <br /> E163Q-ATP structure referred LptB-ATP                   reaction. Consistent hypothesis, water molecule is <br /> (Fig. 4A Fig. S2C). Examination active site demonstrated             present LptB-ADP structure (Fig. 4E). <br />  <br /> 4984 | www.pnas.org/cgi/doi/10.1073/pnas.1323516111                                                                                     Sherman et al. <br />  Fig. 4. Conformational changes ATP hydrolysis reorganization active site causes changes region LptB believed interact <br /> with LptF/G. (A) Cartoon rendering LptB-ATP (prehydrolysis), conserved ATPase ABC motifs indicated (Walker A, yellow; Walker B, orange; <br /> signature motif, blue; Q-loop, cyan; switch region, red). We assigned active site metal LptB-ATP structure Na+ ion. The distances the <br /> metal coordinating water molecules correspond characteristic Na+ (&#226;&#710;&#188;2.4 &#195;&#8230;) characteristic Mg2+ (&#226;&#8240;&#164;2.1 &#195;&#8230;), were <br /> observed LptB-ADP structure. In addition, LptB-ATP crystallized magnesium-free buffer condition containing sodium chloride. (B) Structural <br /> overlay LptB-ATP (red) LptB-ADP (blue) structures shows conformational changes ATP hydrolysis. The product (ADP) shown active <br /> site. (C) Close-up view ATP-bound active site, chains conserved residues Walker B, switch, Q-loop regions shown. The Fo-Fc <br /> omit map contoured 3&#207;&#402; shows clear electron density &#206;&#179;-phosphate. (D) Structural overlay, B, pre- posthydrolysis structures, with <br /> arrows indicating movement chains hydrolysis. Coloring conserved regions A. (E) Rearrangement active site residues upon <br /> ATP hydrolysis stabilize ADP shown. The octahedral coordination sphere active site cation shown, putative hydrolytic water indicated <br /> in ATP-bound active site. <br />  <br />  <br />  <br />    Comparison pre- posthydrolysis structures                      However, C&#206;&#177; shifts &#226;&#710;&#188;4 &#195;&#8230; ADP-bound structure. Not <br /> shows H195 switch region undergoes major con-                       does switch region move, chain H195 <br /> formational change (Fig. 4D). The chain H195 makes                        flips. One important unanswered question inorganic <br /> direct contact &#206;&#179;-phosphate ATP-bound structure.                   phosphate (Pi) exits active sites NBDs. There clear <br />                                                                                   electron density Pi group posthydrolysis crystal <br />                                                                                   structure (Fig. S2B), consistent claims Pi leaves <br />                                                                                   active site ADP released (42&#226;&#8364;&#8220;44). Based an <br />                  Groove                               Groove                      electrostatic potential surface LptB, H195 chain flips <br />                                    ATP                                            face negatively charged protein (Fig. S3). It is <br />                                                                                   possible &#206;&#179;-phosphate remains bound H195 that <br />      F90 <br />        0 <br />                                                                                                                                                                    MICROBIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                   movement switch region forces Pi electro- <br />                                                                                   static repulsion negatively charged area. It possible <br />                                                    F90 <br />                                                    F                              dramatic movement switch region observed dur- <br />           Q163                                                                    ing ATP hydrolysis plays critical role communicating <br />                                              R91 <br />                                                                     ATP           changes active site changes TMDs. <br />                   H195                                                               Because H195 essential vivo LptB-ATP structure <br />                                                                                   shows imidazole chain directly interacts the <br />                                                     Top View <br />                                                                                   &#206;&#179;-phosphate ATP, suspected ATPase activity ob- <br />                                                                                   served isolated NBD (Fig. 3B) reported inaccurately <br /> Fig. 5. Structural observations LptB implicate key binding site for <br />                                                                                   activity complex (21). Therefore, overex- <br /> TMDs. Surface renderings monomer LptB-ATP (frontal facing         pressed purified LptB2FGC detergent measured the <br /> views) residues Q163 H195 near ATP, residue F90 facing in-           ATPase activity wild-type complex complexes con- <br /> terior groove, residue R91 facing away groove.                taining LptB-E163Q LptB-H195A (Fig. S4A). The complex <br />  <br /> Sherman et al.                                                                                              PNAS | April 1, 2014 | vol. 111 | no. 13 | 4985 <br />  containing LptB-E163Q catalytically inactive, complex       ATP hydrolysis induces conformational changes LptB. The <br /> containing LptB-H195A &#226;&#710;&#188;10% ATPase activity               dramatic conformational change structural snapshots <br /> the wild-type complex. These results confirm hypothesis       switch region. Not does switch loop move, but <br /> although H195 sole catalytic residue, positioning       imidazole chain conserved H195 flips away <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> is important step catalytic cycle, suggested        active site negatively charged channel. This move- <br /> by crystal structures.                                             ment, combined observation H195 coordinates the <br />                                                                        &#206;&#179;-phosphate prehydrolysis complex, leads speculate <br /> Groove Region LptB Is Essential Interaction IM Partners.   residue helps &#206;&#179;-phosphate active site <br /> Taken together, ATP hydrolysis initiates global movement            acidic channel, ejected electrostatic <br /> LptB couples changes regions surrounding active site       repulsion (Fig. S3). This shift conserved histidine near <br /> (the switch Walker B domains) changes struc-         negatively charged surface observed structure the <br /> turally diverse helical domain (the signature motif Q-loop).   NBD MJ0796 (42). Such role consistent low-level <br /> These changes near helical domain result shift         ATPase activity observed LptB-H195A complex the <br /> groove located adjacent Q-loop (Fig. S4B Fig. 5). This      Lpt IM components. <br /> coupling ATP hydrolysis movement groove region             Our crystal structures reveal groove LptB undergoes <br /> could important communicating TMDs, thereby <br />                                                                        movement ATP hydrolysis. We hypothesize groove is <br /> affecting LPS transport. Therefore, interested the <br />                                                                        involved interaction TMD partners, LptF/G, and <br /> positioning F90 Q-loop, interpreted in <br />                                                                        movement essential connecting ATP hydrolysis by <br /> vivo biochemical results described mean F90 is <br />                                                                        LptB LPS extraction LptF/G. Our genetic bio- <br /> important binding putative coupling helices TMDs <br />                                                                        chemical studies implicate groove residue F90 important <br /> of ABC system. <br />    The chain F90 Q-loop faces interior           binding site LptF/G. The crystal structures suggest this <br /> the groove opening ATP- ADP-bound struc-               residue critical, aromatic chain faces inside <br /> tures (Fig. 5), suggesting residue functionally im-      groove likely interacts coupling helices the <br /> portant interacting TMDs        TMDs. Structural alignments NBDs, isolation <br /> catalytic cycle. Indeed, role residue F90 explain   complex TMDs, reveal that, despite part <br /> the lptB-F90A allele nonfunctional. To test positions       structurally diverse region, aromatic residue <br /> with chains facing away groove interior im-       location F90 LptB. For example, F90 LptB <br /> portant residues pointing groove, assessed im-      aligns F429 Salmonella typhimurium MsbA (PDB ID <br /> portance LptB-R91A, chain facing away         code 3b60), Y87 MalK (PDB ID code 2r6g), with <br /> the interior groove crystallographic snapshots          Y94 Sulfolobus solfataricus GlcV (PDB ID code 1oxx). In <br /> (Fig. 5). We LptB-R91A variant functionally          structure Sav1866 bound nonhydrolyzable ATP ana- <br /> similar wild-type LptB (Fig. S1 A C).                           logue (PDB ID code 2onj), guanidinium group R206 in <br />    Taken together, results support hypothesis           TMD close proximity make &#207;&#8364;-cation in- <br /> residues Q-loop LptB face interior           teraction aromatic ring F427 NBD, which <br /> groove important assembly complex, func-        aligns F90 LptB (Fig. S4D), suggesting residue <br /> tion, both. The high conservation F90 LptB orthologs      critical interaction TMDs. For LptB, have <br /> (Fig. S4C), combined fact lptB-F90Y allele           established importance F90 genetic analysis affinity <br /> confers mild OM permeability defects haploid strains (Fig. S1C),    purifications: The F90A variant fails complement results <br /> suggests conservative change phenylalanine ty-      destabilization ABC transporter. The aromatic char- <br /> rosine tolerated, optimal, LptB evolved        acter residue critical mediating interaction of <br /> phenylalanine position. These experiments lead con-      LptB LptF/G, F90Y substitution results in <br /> clude F90 forms critical binding site LptF/G.   partial loss function vivo, suggesting interaction is <br />                                                                        specific. Further studies underway examine key <br /> Discussion                                                             interaction TMDs important linking ATP <br /> We crystal structures catalytically inactive         hydrolysis extraction LPS opposite the <br /> variant LptB bound ATP wild-type LptB bound ADP.          membrane conformational changes groove region. <br /> By combining biochemistry genetic analysis LptB var-            We comprehensive analysis selected residues in <br /> iants, identified residues LptB important      ATPase component Lpt IM complex combines <br /> ATP hydrolysis (E163), assembly LptF/G (F90),             genetic studies essentiality, biochemical studies catalytic ac- <br /> function complex (H195). Structural studies reveal         tivity protein&#226;&#8364;&#8220;protein interaction, crystallographic analysis <br /> residues involved binding ATP catalyzing &#206;&#179;-phosphate            pre- posthydrolysis complexes. Our results highlight the <br /> hydrolysis reorganize stabilize product, ADP. Our crys-         importance combining multiple techniques understand the <br /> tallographic snapshots suggest movement residues        roles individual residues process transporting LPS <br /> the LptB active site coupled movement regions          IM cell surface. Neither catalytic activity cell <br /> interface LptF/G essential LPS transport.       viability illuminate functions certain residues. <br /> In addition, using catalytically inactive LptB-E163Q variant,     Clearly, interfering catalytic activity coupling other <br /> have demonstrated ATP hydrolysis LptB essential for <br />                                                                        Lpt components viable strategies development new <br /> cell viability; contrast, ATP hydrolysis required the <br />                                                                        antibiotics targeting LPS biogenesis. <br /> formation Lpt IM complex. <br />    With respect mode catalysis, observations         Materials Methods <br /> consistent prevailing general base catalysis model,        In Vivo Experiments. Strains, growth conditions, construction plasmids <br /> which carboxylate chain end Walker B motif        &#206;&#8221;lptB allele described SI Materials Methods Tables S1&#226;&#8364;&#8220; <br /> (in residue E163) deprotonates water molecule         S3. Functionality mutant lptB alleles assessed complementary <br /> serve nucleophile hydrolysis reaction. The locations    methods described SI Materials Methods. When necessary, the <br /> the general base potential nucleophile consistent         chromosomal &#206;&#8221;lptB allele transferred P1 transduction (45). <br /> with structures transition state E. coli maltose im-        OM permeability tests performed using disk diffusion assays (46). <br /> porter (41). The pre- postcatalysis structures reveal         LptB protein levels monitored immunoblotting using anti-LptB <br />  <br />  <br /> 4986 | www.pnas.org/cgi/doi/10.1073/pnas.1323516111                                                                                   Sherman et al. <br />  antisera raised rabbit using LptB peptide DDLSAEQREDRANE                      Protein Crystallization, Data Collection, Structure Determination. A de- <br /> immunogen (ProSci, Inc.).                                                                     tailed description structure determination provided SI Materials <br />                                                                                               Methods. Briefly, protein crystals grown vapor diffusion in <br /> Affinity Purifications ATPase Assays. Modifications affinity purifications             hanging drops. Experimental phasing obtained multiple isomorphous <br /> (15) explained SI Materials Methods. ATPase assays con-                       replacement anomalous scattering using selenomethionine derivative <br /> ducted described 5 mM ATP/MgCl2 LptB LptB2FGC                            crystal tantalum derivative crystal. The X-ray diffraction data col- <br /> variants (21).                                                                                lected National Synchrotron Light Source beamline X29 Brook- <br />                                                                                               haven National Laboratory, selenomethionine dataset, which <br /> Protein Overexpression Purification. LptB-His8 catalytically inactive             collected 24-ID-E Advanced Photon Light Source Argonne <br /> E163Q variant overexpressed E. coli purified nickel-affinity                   National Laboratory. <br /> chromatography gel filtration chromatography described (19), with <br /> notable changes described SI Materials Methods. Overexpression                      ACKNOWLEDGMENTS. We thank Rebecca M. Davis strain construction <br /> a selenomethionine-containing derivative LptB (SeMet-LptB-His8) described               Dr. Seong-Joo Lee collecting selenomethionine dataset. This <br />                                                                                               work supported National Science Foundation Graduate Research <br /> in SI Materials Methods. <br />                                                                                               Fellowship (to D.J.S.); Harvard Blavatnik Biomedical Accelerator Fund; <br />    LptB2FGC complexes (wild-type, E163Q, H195A variants) containing                       National Institutes Health Grants AI081059 (to D.K.), GM066174 (to D.K.), <br /> His6-LptB overexpressed E. coli purified described SI                       GM094263 (to S.W.); startup funds The Ohio State University <br /> Materials Methods.                                                                        (to N.R.). <br />  <br />  <br />  1. Dassa E, Bouige P (2001) The ABC ABCS: A phylogenetic functional classifica-       24. Moody JE, Millen L, Binns D, Hunt JF, Thomas PJ (2002) Cooperative, ATP-dependent <br />     tion ABC systems living organisms. Res Microbiol 152(3-4):211&#226;&#8364;&#8220;229.                      association nucleotide binding cassettes catalytic cycle ATP- <br />  2. Davidson AL, Dassa E, Orelle C, Chen J (2008) Structure, function, evolution           binding cassette transporters. J Biol Chem 277(24):21111&#226;&#8364;&#8220;21114. <br />     bacterial ATP-binding cassette systems. Microbiol Mol Biol Rev 72(2):317&#226;&#8364;&#8220;364.             25. Oswald C, Holland IB, Schmitt L (2006) The motor domains ABC-transporters. What <br />  3. Dean M, Rzhetsky A, Allikmets R (2001) The human ATP-binding cassette (ABC)                   structures tell us? Naunyn Schmiedebergs Arch Pharmacol 372(6):385&#226;&#8364;&#8220;399. <br />     transporter superfamily. Genome Res 11(7):1156&#226;&#8364;&#8220;1166.                                      26. Zaitseva J, Jenewein S, Jumpertz T, Holland IB, Schmitt L (2005) H662 linchpin of <br />  4. Raetz CR, Whitfield C (2002) Lipopolysaccharide endotoxins. Annu Rev Biochem 71:              ATP hydrolysis nucleotide-binding domain ABC transporter HlyB. EMBO J <br />     635&#226;&#8364;&#8220;700.                                                                                      24(11):1901&#226;&#8364;&#8220;1910. <br />  5. Nikaido H (2003) Molecular basis bacterial outer membrane permeability revisited.      27. Ernst R, et al. (2008) A <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>mutation</span> H-loop selectively affects rhodamine trans- <br />     Microbiol Mol Biol Rev 67(4):593&#226;&#8364;&#8220;656.                                                         port yeast multidrug ABC transporter Pdr5. Proc Natl Acad Sci USA 105(13): <br />  6. Osborn MJ, Gander JE, Parisi E (1972) Mechanism assembly outer membrane             5069&#226;&#8364;&#8220;5074. <br />     Salmonella typhimurium. Site synthesis lipopolysaccharide. J Biol Chem           28. Karpowich N, et al. (2001) Crystal structures MJ1267 ATP binding cassette re- <br />     247(12):3973&#226;&#8364;&#8220;3986.                                                                            veal induced-fit effect ATPase active site ABC transporter. Structure <br />  7. Osborn MJ, Gander JE, Parisi E, Carson J (1972) Mechanism assembly outer            9(7):571&#226;&#8364;&#8220;586. <br />     membrane Salmonella typhimurium. Isolation characterization cytoplasmic         29. Schmitt L, Benabdelhak H, Blight MA, Holland IB, Stubbs MT (2003) Crystal structure <br />     outer membrane. J Biol Chem 247(12):3962&#226;&#8364;&#8220;3972.                                            nucleotide-binding domain ABC-transporter haemolysin B: Identifica- <br />  8. Bos MP, Tefsen B, Geurtsen J, Tommassen J (2004) Identification outer mem-              tion variable region ABC helical domains. J Mol Biol 330(2):333&#226;&#8364;&#8220;342. <br />     brane protein required transport lipopolysaccharide bacterial cell      30. Ruiz N, Wu T, Kahne D, Silhavy TJ (2006) Probing barrier function outer <br />     surface. Proc Natl Acad Sci USA 101(25):9417&#226;&#8364;&#8220;9422.                                            membrane chemical conditionality. ACS Chem Biol 1(6):385&#226;&#8364;&#8220;395. <br />  9. Braun M, Silhavy TJ (2002) Imp/OstA required cell envelope biogenesis Es-       31. Davidson AL, Sharma S (1997) Mutation single MalK subunit severely impairs <br />     cherichia coli. Mol Microbiol 45(5):1289&#226;&#8364;&#8220;1302.                                                maltose transport activity Escherichia coli. J Bacteriol 179(17):5458&#226;&#8364;&#8220;5464. <br /> 10. Ruiz N, Gronenberg LS, Kahne D, Silhavy TJ (2008) Identification inner-mem-        32. Nikaido K, Ames GF (1999) One intact ATP-binding subunit sufficient support ATP <br />     brane proteins required transport lipopolysaccharide outer mem-             hydrolysis translocation ABC transporter, histidine permease. J Biol <br />                                                                                                   Chem 274(38):26727&#226;&#8364;&#8220;26735. <br />     brane Escherichia coli. Proc Natl Acad Sci USA 105(14):5537&#226;&#8364;&#8220;5542. <br />                                                                                               33. Schneider E, Hunke S (1998) ATP-binding-cassette (ABC) transport systems: Functional <br /> 11. Sperandeo P, et al. (2007) Characterization lptA lptB, essential genes <br />                                                                                                   structural aspects ATP-hydrolyzing subunits/domains. FEMS Microbiol Rev <br />     implicated lipopolysaccharide transport outer membrane Escherichia coli. <br />                                                                                                   22(1):1&#226;&#8364;&#8220;20. <br />     J Bacteriol 189(1):244&#226;&#8364;&#8220;253. <br />                                                                                               34. Shyamala V, Baichwal V, Beall E, Ames GF (1991) Structure-function analysis the <br /> 12. Sperandeo P, et al. (2008) Functional analysis protein machinery required for <br />                                                                                                   histidine permease comparison cystic fibrosis mutations. J Biol Chem <br />     transport lipopolysaccharide outer membrane Escherichia coli. J Bacteriol <br />                                                                                                   266(28):18714&#226;&#8364;&#8220;18719. <br />     190(13):4460&#226;&#8364;&#8220;4469. <br />                                                                                               35. Krissinel E, Henrick K (2004) Secondary-structure matching (SSM), new tool fast <br /> 13. Sperandeo P, Pozzi C, Deh&#195;&#178; G, Polissi A (2006) Non-essential KDO biosynthesis and <br />                                                                                                   protein structure alignment dimensions. Acta Crystallogr D Biol Crystallogr <br />     new essential cell envelope biogenesis genes Escherichia coli yrbG-yhbG locus. <br />                                                                                                   60(Pt 12 Pt 1):2256&#226;&#8364;&#8220;2268. <br />     Res Microbiol 157(6):547&#226;&#8364;&#8220;558. <br />                                                                                               36. Hung LW, et al. (1998) Crystal structure ATP-binding subunit ABC <br /> 14. Wu T, et al. (2006) Identification protein complex assembles lipopolysac- <br />                                                                                                   transporter. Nature 396(6712):703&#226;&#8364;&#8220;707. <br />     charide outer membrane Escherichia coli. Proc Natl Acad Sci USA 103(31): <br />                                                                                               37. Walker JE, Saraste M, Runswick MJ, Gay NJ (1982) Distantly related sequences the <br />     11754&#226;&#8364;&#8220;11759. <br />                                                                                                   alpha- beta-subunits ATP synthase, myosin, kinases ATP-requiring <br /> 15. Chng SS, Gronenberg LS, Kahne D (2010) Proteins required lipopolysaccharide <br />                                                                                                   enzymes common nucleotide binding fold. EMBO J 1(8):945&#226;&#8364;&#8220;951. <br />     assembly Escherichia coli form transenvelope complex. Biochemistry 49(22):           38. Hopfner KP, et al. (2000) Structural biology Rad50 ATPase: ATP-driven conforma- <br />     4565&#226;&#8364;&#8220;4567.                                                                                    tional control DNA double-strand break repair ABC-ATPase superfamily. <br /> 16. Freinkman E, Okuda S, Ruiz N, Kahne D (2012) Regulated assembly trans-                 Cell 101(7):789&#226;&#8364;&#8220;800. <br />     envelope protein complex required lipopolysaccharide export. Biochemistry             39. Hollenstein K, Frei DC, Locher KP (2007) Structure ABC transporter complex <br />     51(24):4800&#226;&#8364;&#8220;4806.                                                                             binding protein. Nature 446(7132):213&#226;&#8364;&#8220;216. <br /> 17. Ruiz N, Kahne D, Silhavy TJ (2009) Transport lipopolysaccharide cell        40. Mourez M, Hofnung M, Dassa E (1997) Subunit interactions ABC transporters: A <br />     envelope: The long road discovery. Nat Rev Microbiol 7(9):677&#226;&#8364;&#8220;683.                         conserved sequence hydrophobic membrane proteins periplasmic permeases <br /> 18. Sperandeo P, Deh&#195;&#178; G, Polissi A (2009) The lipopolysaccharide transport              defines important site interaction ATPase subunits. EMBO J 16(11): <br />     Gram-negative bacteria. Biochim Biophys Acta 1791(7):594&#226;&#8364;&#8220;602.                                 3066&#226;&#8364;&#8220;3077. <br />                                                                                                                                                                    <br /> </body></html>