<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                     Open Access <br />  <br /> Prediction essential proteins based gene <br /> expression programming <br /> Jiancheng Zhong1,2, Jianxin Wang1*, Wei Peng1, Zhen Zhang1, Yi Pan3 <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>From</span> IEEE International Conference Bioinformatics Biomedicine 2012 <br /> Philadelphia, PA, USA. 4-7 October 2012 <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Essential proteins indispensable cell survive. Identifying essential proteins important for <br />   improving understanding way cell working. There various types features related the <br />   essentiality proteins. Many methods proposed combine predict essential <br />   proteins. However, big challenge designing effective method predict integrating <br />   different features, explaining selected features decide essentiality protein. Gene expression <br />   programming (GEP) learning algorithm learns specifically relationships variables <br />   sets data builds models explain relationships. <br />   Results: In work, propose GEP-based method predict essential protein combing biological <br />   features topological features. We carry experiments S. cerevisiae data. The experimental results show <br />   method achieves better prediction performance methods using individual features. <br />   Moreover, method outperforms machine learning methods performs method is <br />   obtained combining outputs machine learning methods. <br />   Conclusions: The accuracy predicting essential proteins improved using GEP method combine <br />   topological features biological features. <br />  <br />  <br /> Background                                                                              In recent years, computational approaches have <br /> Essential proteins indispensable support cellular                             presented identify essential proteins according <br /> life [1]. Identifying essential proteins help under-                          features. One group researchers focus on <br /> stand minimal requirements cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>survival</span>,                              detecting essential proteins based topological <br /> plays significant role emerging field synthetic                          features protein-protein interaction (PPI) networks, <br /> biology [2]. Since deleting, interrupting blocking                            previous studies shown removal of <br /> of essential proteins leads death organisms,                               proteins larger number neighbours in <br /> essential proteins serve candidates drug-targets                           PPI networks likely cause organism <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> for developing novel therapies diseases, can-                             die [5]. Therefore, centrality methods been <br /> cers infectious diseases [3]. Moreover, studies                              come Degree Centrality (DC) [6], <br /> have pointed essential proteins correlation                            Betweenness Centrality (BC) [7], Closeness Centrality <br /> with human disease genes [4]. However expensive                                (CC) [8], Subgraph Centrality (SC) [9], Eigenvector Cen- <br /> and time-consuming experimentally identify essential                              trality (EC) [10], Information Centrality (IC) [11], Edge <br /> proteins.                                                                            Clustering Coefficient Centrality (NC) [12] on. <br />                                                                                      However, centrality methods lim- <br />                                                                                      its. For example, highly depend accuracy of <br />                                                                                      PPI networks ignore useful biological features. <br /> * Correspondence: jxwang@mail.csu.edu.cn <br /> 1 <br />  School Information Science Engineering, Central South University,            Recently, new methods combine topological <br /> Changsha, Hunan 410083, PR China                                                     features biological ones developed. <br /> Full list author information available end article <br />  <br />                                        &#194;&#169; 2013 Zhong et al.; licensee BioMed Central Ltd. This open access article distributed terms Creative Commons <br />                                        Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                        medium, provided original work properly cited. <br />  Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7                                                                  Page 2 8 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br /> Hart fellows pointed essential-     separate classifiers (Naive Bayes classifier, logistical <br /> ity special property protein complex essential    regression model, C4.5 decision tree CN2 rule). In <br /> proteins highly clustered certain protein        [18], approaches detecting essential protein are <br /> complexes [13]. Based observation, Ren et al.         introduced discussed. <br /> [14] integrate PPI network topology protein com-            Now machine learning methods avail- <br /> plexes information predict essential proteins. Accord-     able software packages [23], easily <br /> ing fact proteins protein complexes tend       adapted predict essential proteins using input fea- <br /> to co-expressed, Li et al. [15] propose new predic-      tures. However, difficult explain how <br /> tion method called PeC Tang et al [16] propose            features used classification. Moreover, few <br /> another one, WDC, integrates network topology           methods able automatically select appropriate fea- <br /> with gene expression profiles. Considering fact      tures. Researchers tend analyse relationship <br /> essential proteins conserved non-essential      features essentiality proteins with <br /> ones [17] frequently connects          statistical methods. And decide features <br /> [13], Peng et al. [15] proposed iterative method      selected train classifiers [19,22]. Gene expression <br /> for prediction essential proteins based         programming (GEP) learning algorithm it <br /> orthology PPI networks. Their results       learns specifically relationships vari- <br /> accuracy predicting essential proteins              ables sets data, builds models describe <br /> improved combing biological features      relationships [24]. The features weak <br /> topological features. Although methods mentioned          positive correlation essentiality proteins not <br /> above combine features essential proteins effi-       selected GEP classifier. <br /> ciently explain features work         In work, propose GEP-based method pre- <br /> decide essentiality proteins, better methods needs     dict essential protein combining biological features, <br /> to developed integrate appropriate features.       subcellular location, topological features, <br /> Because different types features relates    DC, BC, CC, SC, EC, IC, NC, com- <br /> to protein essentiality, suggests multiple         posed features computed methods PeC, WDC <br /> aspects organisms contribute determining            ION using biological topological features. We <br /> essentiality proteins [18].                                carry experiments S. cerevisiae (Baker&#226;&#8364;&#8482;s yeast) <br />    Another group researchers use machine learning          data. The experimental results method <br /> algorithms, support vector machine (SVM) [19],        outperforms using features calculated by <br /> decision tree [20], Naive Bayes[21] on, detect      existing methods (DC, BC, CC, SC, EC, IC, NC, PeC, <br /> essential proteins. These methods train classifier          WDC ION). Moreover, terms area an <br /> according features known essential proteins         ROC curve (AUC), method achieves better results <br /> and non-essential ones. Then test classifier       machine learning methods (SVM, SMO, Nai- <br /> same organism organisms. For example,            veBayes, Bayes Network, RBF Network, J48, Random <br /> Gustafson et al. [21] select lot essentiality related    Tree, Random Forest, NaiveBayes Tree), performs <br /> features including topological features,     method uses multiple machine learning <br /> degree centrality (DC) PPI networks, biological        methods. <br /> features, open reading frame (ORF) <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>, Phy- <br /> letic retention (PHY), paralogs, codon adaptation index       Results Discussion <br /> (CAI) on. And use Naive Bayes clas-        In section, firstly analyze results 10-fold <br /> sifier make prediction. Hwang et al. [19] build SVM        cross-validation. Then compare prediction our <br /> classifier combines biological features,    method existing methods calculate <br /> ORF length, strand PHY, topological features       individual features. Moreover methods com- <br /> PPI network, DC, BC, CC on. Addition-          pared machine learning-based methods. <br /> ally, researchers combine classifiers pre-    Finally best classifier explain to <br /> dict essential proteins. Acencio et al. [20] learn 12         combine individual features decide essentiality of <br /> different network topological features (DC, BC, CC        protein. <br /> so on) integrated network biological <br /> features, cellular localization biological pro-   Comparison 10-fold cross-validation results <br /> cesses information. Then decision tree-based          There classifiers 10-fold cross-validation <br /> classifiers trained tested based features.   (see section Methods). The performance classi- <br /> A best essentiality classifier obtained combining       fier compared terms ROC curves the <br /> the outputs diverse classifiers. Deng et al. [22]    areas curves (AUC). As Figure 1 shown, the <br /> also train classifier combining results       AUC values classifiers range 0.6975 to <br />  Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7                                                                                       Page 3 8 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br />  Figure 1 ROC curves AUC values classifiers trained 10-fold cross-validation. Original data divided 10 equal <br />  datasets, nine-folds used train classifier remaining fold used testing. The process repeated times to <br />  generated classifiers, datasets used exactly testing data. The figure illustrates ROC curves and <br />  corresponding AUC values classifiers. <br />  <br />  <br />  <br />  <br /> 0.8213. The average AUC value 0.7730. The perfor-                      ones. According known essential protein, values <br /> mance fluctuation depends datasets. In work,                  sensitivity (SN), specificity (SP), positive predictive <br /> we select average AUC value                          value (PPV), FPR, negative predictive value (NPV), <br /> classifiers perform following analyses.                            F-Measure, accuracy (ACC) Matthews Correlation <br />                                                                           Coefficent (MCC) calculated method. Table <br /> Comparison existing methods                                    1 shows values SN, SP, PPV, NPV, F-Mea- <br /> Since classifier combines features calculated                  sure, ACC MCC GEP classifier 0.5467, <br /> other existing methods DC, BC, CC, EC, IC, SC,                    0.8653, 0.5467, 0.8653, 0.5467, 0.7927 0.4120 <br /> NC, PeC, WDC ION, compare predictability                       respectively, higher existing meth- <br /> with others. The proteins PPI network                      ods. On hand, FPR value GEP 0.1347, <br /> ranked descend order according scores                           lowest methods. <br /> assigned classifier existing meth- <br /> ods. After values true positive rate (TPR)                Comparison machine learning-based methods <br /> false positive rate (FPR) computed method                    To evaluate prediction performance our <br /> with different percentages proteins selected                    classifier, compare machine learning <br /> predicted essential proteins. The values TPR FPR                   methods, SVM, SMO, NaiveBayes, Bayes Net- <br /> are plotted ROC curves different cut-off values.                  work, RBF Network, J48, RandomTree, RandomForest, <br /> As Figure 2 shown, ROC curve classifier                     NaiveBayes Tree, Which wildly used previous pre- <br /> obviously existing methods. The                   diction methods [19,21]. Acencio et al [19] build their <br /> AUC value GEP classifier 0.7761                        classifier combining decision tree classifiers (REP <br /> 0.0237 higher ION best performance                     tree, naive bayes tree, random tree, random forest, J48, <br /> among existing methods, 0.113 higher                    best-first decision tree, logistic model tree alternating <br /> the average AUC value (= 0.6631) existing                        decision tree). We compare classifier their <br /> methods.                                                                  composited classifier (named Acencio). All these <br />   Specially, select 1167 proteins ranked                   machine learning based methods implemented by <br /> method predicted essential proteins. The rest 3926                  using WEKA software package default parameters <br /> (= 5093-1167) proteins regarded non-essential                      setting carried 10-fold cross-validation based on <br />  Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7                                                                                                         Page 4 8 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br />      Figure 2 ROC curves AUC values GEP classifier methods using individual feature. We select classifier has <br />      average prediction performance classifiers generated 10-fold cross-validation, test original data containing 5093 <br />      proteins available learning features. The figure illustrates ROC curves classifier methods use individual feature. <br />  <br />  <br /> topological features biological features mentioned                            combined decide essentiality protein. The <br /> section Method. Their average AUC values listed                               expression GEP classifier best prediction <br /> Table 2. As Table 2 shown, average AUC value                                  performance obtained as <br /> GEP achieves 0.773, outperforms machine <br /> learning methods. Compared Acencio classifier, GEP <br /> classifier possesses equal prediction performance <br /> in term average AUC values.                                                   As expression shown, ION predictive fea- <br />                                                                                      tures. Because relates evolutionary conservation <br /> Analysis GEP classifier                                                           proteins, essential proteins highly con- <br /> The GEP classifier produces expression different                               served organisms [15]. The proteins located in <br /> types input features, describes features                            endoplasmic reticulum nucleus tend possess <br />  <br /> Table 1 Comparison GEP methods using individual feature <br /> Methods              SN               SP               FPR               PPV               NPV               F-measure             ACC               MCC <br /> GEP                  0.5467           0.8653           0.1347            0.5467            0.8653            0.5467                0.7927            0.4120 <br /> DC                   0.4002           0.8217           0.1783            0.4002            0.8217            0.4002                0.7251            0.2219 <br /> BC                   0.3505           0.8069           0.1931            0.3505            0.8069            0.3505                0.7023            0.1574 <br /> CC                   0.3548           0.8082           0.1918            0.3548            0.8082            0.3548                0.7043            0.1630 <br /> SC                   0.3676           0.8120           0.1880            0.3676            0.8120            0.3676                0.7102            0.1796 <br /> EC                   0.3676           0.8120           0.1880            0.3676            0.8120            0.3676                0.7102            0.1796 <br /> IC                   0.4010           0.8220           0.1780            0.4010            0.8220            0.4010                0.7255            0.2230 <br /> NC                   0.4353           0.8321           0.1679            0.4353            0.8321            0.4353                0.7412            0.2674 <br /> PeC                  0.4036           0.8227           0.1773            0.4036            0.8227            0.4036                0.7267            0.2263 <br /> ION                  0.5124           0.8551           0.1449            0.5124            0.8551            0.5124                0.7766            0.3675 <br /> WDC                  0.4576           0.8390           0.1610            0.4580            0.8388            0.4578                0.7516            0.2967 <br /> The proteins PPI network ranked descend order according scores assigned classifier existing methods. select 1167 <br /> proteins ranked method candidate essential proteins. The rest 3926 (= 5093-1167) proteins regarded non-essential proteins. According to <br /> known essential protein, values sensitivity (SN), specificity (SP), positive predictive value (PPV), FPR, negative predictive value (NPV), F-Measure, accuracy <br /> (ACC) Matthews Correlation Coefficent (MCC) calculated method. The table lists results. <br />  Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7                                                                              Page 5 8 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br /> Table 2 Comparison average AUC GEP                         use individual features. Thus, method effec- <br /> other machine learning based methods.                                     tively combine range different features predict <br /> Methods                                                  AUC              essential proteins. <br /> GEP                                                      0.773 <br /> SVM                                                      0.577            Methods <br /> SMO                                                      0.608            Experimental datasets <br /> NaiveBayes                                               0.744            We implement experiments based data S. cerevi- <br /> Bayes Network                                            0.731            siae (Bakers&#226;&#8364;&#8482; Yeast) PPI gene essen- <br /> RBF Network                                              0.669            tiality data complete reliable among <br /> J48                                                      0.687            various species. The PPI data S. cerevisiae down- <br /> Random Tree                                              0.612            loaded DIP database [27] using version pub- <br /> Random Forest                                            0.721            lished Oct.10, 2010, self-interactions and <br /> NaiveBayes Tree                                          0.746            repeated interactions. There total 5093 proteins <br /> Acencio                                                  0.778            24743 edges. <br /> This table shows average AUC values GEP classifiers      The list essential proteins integrated fol- <br /> machine learning methods.                                                 lowing databases: MIPS [28], SGD [29], DEG [1] and <br />                                                                           SGDP [30], contains 1285 essential proteins. <br /> indispensable functions, consistent                     Among 1285 essential protein, 1167 proteins present <br /> observation Acencio et al. [25]. However, pro-                   PPI network. In study, 1167 proteins are <br /> teins located cytoplasm vacuole likely                  regarded essential proteins 3926(= 5093- <br /> be essential proteins. Note input features                      1167) proteins nonessential proteins. <br /> are present expression. Our GEP classifier                        The information orthologous proteins used in <br /> discards features effective                 method ION, download Version 7 In <br /> predicting essentiality replaced features. For                Paranoid database [31]. The gene expression data of <br /> example classifier selects localized topological fea-                 yeast retrieved Tu et al., 2005 [32], containing <br /> tures WDC SC instead global ones                      6,777 gene products (proteins) 36 samples total. <br /> as BC, CC IC. Because localized centrality measures                   Among 6777 proteins, 4858 proteins involved in <br /> can obtain better prediction performance glo-                    yeast PPI network. The subcellular information is <br /> bal ones [26]. Both WDC PeC depend fea-                        downloaded eSLDB database [33], cate- <br /> tures co-expression co-clustering essential                     gorizes 5093 proteins PPI network 16 differ- <br /> proteins, classifier uses terms                     ent subcellular localizations. <br /> their capability prediction. <br />                                                                           Features selection <br /> Conclusions                                                               Our GEP classifier constructed predict essential <br /> As different types features relates essentiality             proteins based topological biological features. <br /> proteins, big challenge designing effec-             The topological features include degree centrality, <br /> tive method predict integrating different fea-                 betweenness centrality, closeness centrality, subgraph <br /> tures explaining selected features decide                   centrality, eigenvector centrality, information centrality <br /> the essentiality protein. In work, propose                   edge clustering coefficient centrality PPI network, <br /> GEP-based prediction method combines topologi-                      calculated centrality methods DC, BC, <br /> cal biological features. Compared machine                    CC, SC, EC, IC NC, respectively. Some composited <br /> learning-based methods, able select predictive                   features calculated methods (PeC, WDC, ION) <br /> features automatically generates expression                   integrate topological features biological <br /> describes relationships them. We carry                    features used work. <br /> experiments S. cerevisiae (Baker&#226;&#8364;&#8482;s yeast) data.(1) Ten                    Additionally, biological features subcellu- <br /> classifiers obtained 10-fold cross-validation                    lar localization considered. Because subcellular loca- <br /> based input features. The average AUC value                     tion plays crucial role protein function proteins <br /> 0.7730. (2) In terms average AUC values, method                    perform functions certain subcellular compart- <br /> outperforms number machine learning methods                          ments. Acencio et al. [25] proteins located in <br /> and comparable performance method                        nuclear subcellular compartments tend essential, <br /> combines output results decision trees. (3)                  essential biological processes cell viabi- <br /> We evaluate classifier testing proteins               lity place nuclear. In contrast, membrane <br /> PPI network available learning features. The                     proteins functions transporters participate in <br /> results indicate classifier performs better                metabolism related processes likely be <br />  Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7                                                                       Page 6 8 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br /> nonessential. In work, proteins PPI network <br /> are involved 16 different localizations including <br /> Vacuole, Vesicles, Lysosome, Membrane, Mitochondrion, <br /> Peroxisome, Secretory pathway, Cell wall, Cytoskeleton, <br /> Endoplasmic reticulum, Golgi, Transmembrane, Cyto- <br /> plasm, Nucleus Endosome, Extracellular. <br />   For feature value ranges, nor- <br /> malize features dividing corresponding <br /> maximum values, ranges features value -1 <br /> to 1. The coefficient (0.1, 0.2, 0.5, 1.0) added to <br /> adjust contribution feature. <br />  <br /> Classifier design <br /> As kind Evolutionary Algorithms (EA&#226;&#8364;&#8482;s), <br /> Gene expression programming (GEP) genotype/phe- <br /> notype genetic algorithm combines merits of <br /> both genetic algorithms genetic programming [24]. <br /> Each chromosome GEP expressed using nonlinear <br /> entities represented fixed-length linear <br /> encoded <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span>, mathematical expressions, poly- <br /> nomial constructs, logical expression, forth. <br /> Chromosomes evolved new ones gener- <br /> ated genetic operations <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, transposi- <br /> tion recombination guided fitness function. <br /> GEP able global searches classification and <br /> performs well, seldom adopted solve clas- <br /> sification problem essential proteins. <br />    To build classifier predicting essential proteins <br /> using GEP following major steps needed: defin- <br /> ing chromosome using function terminal set, <br /> initializing population generating group chro- <br /> mosomes, defining fitness function evaluating <br /> chromosomes, selecting eugenic ones populations, <br /> reproducing group chromosomes gen- <br /> eration, deciding termination model. Fig- <br /> ure 3 illustrates flowchart building GEP classifier. <br />    In work, GEP developed predict essential <br /> proteins. First all, set functions terminals is <br /> chosen define chromosomes expressed as <br /> nonlinear entities. The set functions contains arith- <br /> metic operators logic operators, add, sub- <br /> tract, multiply, divide, min, max, equal, sqrt, log, exp, <br /> abs, set terminals contains variables repre- <br /> senting features proteins (for example, topological <br /> and subcellular localization properties) relevant <br /> coefficients. Then build chromosomal structure.         Figure 3 Flowchart building GEP classifier. This figure shows <br /> For chromosome length head             flowchart building GEP classifier. <br /> number genes given. <br />    The second step randomly generate popula- <br /> tion. The input parameter indicates size popula-       Thus, chose maximum, 12000, number of <br /> tion. We varied number populations 800         populations. <br /> 12000, kept track results model produces.           The step define fitness function evalu- <br /> According results, performance models           ate individual chromosome. To obtain optimal out- <br /> increases gradually number populations rises.       put, select SSPN fitness function is <br />  Zhong et al. BMC Genomics 2013, 14(Suppl 4):S7                                                                                                  Page 7 8 <br /> http://www.biomedcentral.com/1471-2164/14/S4/S7 <br />  <br />  <br />  <br />  <br /> described product sensitivity (SN), specificity                      divided 10 equal datasets, nine-folds used <br /> (SP), positive predictive value (PPV), negative pre-                       train classifier remaining fold used <br /> dictive value (NPV). The formula follow:                                 testing. Since ratio essential non-essential <br />                                                                                proteins original data 1:3.36 (essential pro- <br />                                                                          (1)   teins: non-essential proteins = 1167:3926), fold <br />   Where SN, SP, PPV, NPV calculated respectively                           data maintains ratio essential proteins and <br /> using following formulae chromosome                                 nonessential proteins original data. The cross-valida- <br />                                                                                tion process repeated times generated clas- <br />                                                                          (2)   sifiers, datasets used exactly once <br />                                                                                testing data. <br />  <br />  <br />                                                                          (3)   Competing interests <br />                                                                                The authors declare competing interests. <br />  <br />                                                                                Authors&#226;&#8364;&#8482; contributions <br />                                                                                JCZ WP obtained protein-protein interaction data, essential proteins, <br />                                                                          (4)   gene expression data Orthologous data. JCZ JXW designed the <br />                                                                                method. JCZ WP analyzed results. JCZ, WP, JXW discussed <br />                                                                                extensively study drafted manuscript together. ZZ YP <br />                                                                                participated revising draft. All authors read approved the <br />                                                                          (5)   manuscript <br />  <br />                                                                                Acknowledgements <br />    Where TPi, TNi, FPi, FNi indicate, respectively,                        This work supported National Natural Science Foundation <br /> the number true positives, true negatives, false posi-                      China Grant No. 61232001, No.61128006 No.61003124; the <br /> tives, false negatives. Given protein features, use                     Program New Century Excellent Talents University (NCET-12-0547); the <br />                                                                                U.S. National Science Foundation Grants CCF-0514750, CCF-0646102 <br /> the fitness function compute scores chromo-                         CNS-0831634; Scientific Research Fund Hunan Provincial Education <br /> somes population.                                                           Department NO.12B080. <br />    The forth step select 30% populations as <br />                                                                                Declarations <br /> eugenic ones. Then fifth step performing set                  The publication costs article funded corresponding <br /> genetic operations (including mutation, transposition                      author. <br /> crossover) eugenic ones reproduces chromosomes                           This article published BMC Genomics Volume 14 <br />                                                                                Supplement S4, 2013: Selected articles IEEE International <br /> the generation size one.                      Conference Bioinformatics Biomedicine 2012: Genomics. The full <br />    Finally, chose maximum, 500, number                        contents supplement available online http://www. <br /> generations decide termination model. In                         biomedcentral.com/bmcgenomics/supplements/14/S4. <br /> this study, parameters used GEP classifier                      Authors&#226;&#8364;&#8482; details <br /> listed Table 3. We develop program predicts                        1 <br />                                                                                 School Information Science Engineering, Central South University, <br /> essential protein based GEP C++ Language.                                Changsha, Hunan 410083, PR China. 2Department Electronic and <br />                                                                                Information Engineering, Hunan Normal University, Changsha, Hunan <br />                                                                                410083, PR China. 3Department Computer Science, Georgia State <br /> Training testing set preparation                                           University, Atlanta, GA 30302-4110, USA. <br /> We build evaluate classifiers terms 10-fold <br />                                                                                Published: 1 October 2013 <br /> cross-validation analysis, original data are <br />                                                                                References <br />                                                                                1. Zhang R, Lin Y: DEG 5.0, database essential genes both <br /> Table 3 Parameters used GEP method                                          prokaryotes eukaryotes. Nucleic acids research 2009, 37:D455-458. <br /> Parameter       Description parameter        Setting parameter           2. Glass JI, Hutchison CA, Smith HO, Venter JC: A systems biology tour de <br />                                                                                    force near-minimal bacterium. Molecular systems biology 2009, 5:330. <br /> P1              Number Population            12000                          3. Clatworthy AE, Pierson E, Hung DT: Targeting virulence: new paradigm <br /> P2              Length Gene                  1                                  antimicrobial therapy. Nature chemical biology 2007, 3:541-548. <br />                                                                                4. Furney S, Alba MM, Lopez-Bigas N: Differences evolutionary history <br /> P3              Length Chromosome            60 <br />                                                                                    disease genes affected dominant recessive mutations. BMC <br /> P4              Length Head                  20                                 Genomics 2006, 7:165. <br /> P5              Mutation rate%                  0.25                           5. Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality centrality in <br />                                                                                    protein networks. Nature 2001, 411:41-42. <br /> P6              Cross rate%                     0.1 <br />                                <br /> </body></html>