<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 23 October 2012                                                          Nucleic Acids Research, 2013, Vol. 41, No. 1 e33 <br />                                                                                                                      doi:10.1093/nar/gks972 <br />  <br />  <br /> Rationally designed bidirectional promoter <br /> improves evolutionary stability of <br /> synthetic genetic circuits <br /> Song Yang1,*, Sean C. Sleight2 Herbert M. Sauro2 <br /> 1 <br />  Genomics Division, Lawrence Berkeley National Laboratory, 1 Cyclotron Road, Mail Stop 84-363, Berkeley, <br /> CA 94720, USA 2Department Bioengineering, University Washington, Seattle, WA 98195, USA <br />  <br /> Received July 16, 2012; Revised September 24, 2012; Accepted September 25, 2012 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                          population faster growing non-functional <br />                                                                                   mutants inactivated expression function <br /> One problem synthetic genes genetically en-                               heterologous genes. It experimentally and <br /> gineered organisms foreign DNAs                                theoretically con&#239;&#172;?rmed genes selective <br /> eventually lose functions evolutionary                                 pressure eventually lose functions (11&#226;&#8364;&#8220;16), and <br /> time absence selective pressures. This                                      lack robustness remains problem synthetic <br /> general limitation restrain long-term study                               biology approaches. <br /> and industrial application synthetic genetic                                      A synthetic genetic circuit bacteria host typically <br /> circuits. Previous studies shown                                composed promoters, ribosomal RNA- <br /> of crucial regulatory functions, prokaryotic                                binding sites, terminators target genes interest <br /> promoters synthetic genetic circuits espe-                                 (14). Among various parts genetic circuits, pro- <br /> cially vulnerable mutations. In study, ra-                             moters vulnerable loss-of-function <br />                                                                                   mutations (e.g. insertions, deletions entire <br /> tionally designed robust bidirectional promoters <br />                                                                                   promoter, deletions repeated sequences pro- <br /> (BDPs), self-protected com-                                 moter point mutations) (17). Therefore, protecting the <br /> plementarity overlapping forward                                     promoter <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>mutation</span> crucial enhance evolu- <br /> backward promoter sequences DNA duplex.                                        tionary stability synthetic genetic circuits. In work, <br /> When transcription target non-essential                                  report novel design robust genetic circuit <br /> gene (e.g. green fluorescent protein) coupled                                 controlled bidirectional promoter (BDP). <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> transcription essential gene (e.g. antibi-                              Typically strand DNA encodes functional units, <br /> otic resistance gene) BDP, evolu-                                 coding regions regulatory regions; comple- <br /> tionary half-time gene increases                               mentary strand does necessarily contain informa- <br /> 4&#226;&#8364;&#8220;10 times, depending strain experimen-                                tion. If sequence promoter region designed <br /> tal conditions used. This design using BDPs                                 complementary strand carries function <br />                                                                                   essential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>survival</span> host, promoter <br /> increase mutational stability genetic circuits <br />                                                                                   coupled protected comple- <br /> can potentially applied synthetic biology appli-                            mentary sequence. When complementary strand a <br /> cations general.                                                               promoter designed encode promoter, engineered <br />                                                                                   bidirectional promoter regulate gene expression both <br />                                                                                   directions: target gene forward direction <br /> INTRODUCTION                                                                      essential gene, e.g. antibiotic gene, reverse <br /> With recent advances synthetic biology, scientists                         direction. In case, mutations occur BDP <br /> are able manipulate biological organisms perform                            region may, principle, affect (or turn off) expression <br /> speci&#239;&#172;?c functions exist nature                                  target gene essential gene concurrently. <br /> introducing engineered functional genetic circuits                           Thus, viability non-functional mutant popu- <br /> the host genome (1&#226;&#8364;&#8220;9). In general, engineered genetic                             lation decrease selected against. In words, <br /> circuits essential survival host cell,                      target gene protected essential gene <br /> and expression demands extra metabolic load                              shared BDP. <br /> onto (10). Therefore, positive se-                            Indeed, overlapping functional units encoded com- <br /> lective pressure, synthetic organisms outcompeted                         plementary strands exist nature (18). In genomes <br />  <br />  <br /> *To correspondence addressed. Tel: +1 510 486 7442; Fax: +1 510 486 5614; Email: songyang@lbl.gov <br />  <br /> &#195;&#376; The Author(s) 2012. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which <br /> permits non-commercial reuse, distribution, reproduction medium, provided original work properly cited. For commercial re-use, contact <br /> journals.permissions@oup.com. <br />  e33 Nucleic Acids Research, 2013, Vol. 41, No. 1                                                                PAGE 2 OF 7 <br />  <br />  <br /> of organisms, adjacent promoters opposite              These freezer stocks streaked LB+100 mg/ml <br /> strands close overlap other.       ampicillin plates grown overnight 37  C. Three <br /> In way, trans-regulatory element             colonies chosen transformant (nine total <br /> co-regulate expression multiple genes operons        colonies) inoculated 1.5 ml LB+100 mg/ml <br /> in opposite direction, usually       ampicillin media Eppendorf deep-well plates sealed <br /> pathway functionally related (18). The co-regulation       Thermo Scienti&#239;&#172;?c gas-permeable membrane to <br /> function natural BDPs achieved proximity       allow maximum oxygen diffusion. Where appropriate, <br /> the promoters different strands     medium supplemented 50 mg/ml kanamycin. <br /> controlled regulatory element, e.g. repressor. One    LacI-regulated circuits inducible expression in <br /> example shows <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>point mutation</span> natural           MG1655 Z1 cells supplemented 1   10 3 M <br /> BDP region cause reduced expression genes           IPTG. After cultures grown 24 h 37  C, <br /> on sides (19).                                            shaking 250 r.p.m., cell density (OD600) &#239;&#172;&#8218;uor- <br />    Previous studies tried use co-regulation        escence (excitation wavelength: 485/15, emission wave- <br /> function naturally occurring BDPs synthetic              <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>: 516/20) measured Biotek Synergy HT <br /> circuits, reviewed (20), attempt aimed        plate reader. Then, evolved populations were <br /> increase evolutionary stability genetic circuit       propagated serial dilution scheme using 1:1000 <br /> using BDP. In design, engineered gene detrimen-     (log2 1000 = 9.97) dilution, corresponds the <br /> tal host, function engineered BDP     growth E. coli  10 generations day. This pro- <br /> to prevent mutation. Therefore, proximity         cedure repeats cultures lost &#239;&#172;&#8218;uor- <br /> also overlap forward backward regulatory            escence &#239;&#172;&#8218;uorescence decreased arrived a <br /> regions required robust self-protective BDPs.       plateau. <br />  <br />                                                                Sequencing results mutant plasmids <br /> MATERIALS AND METHODS <br />                                                                To determine loss-of-function mutations replicate- <br /> Circuit engineering use strains                         evolved MG1655/MDS42 populations, plasmids were <br /> All circuits obtained Registry         extracted individual clones low green &#239;&#172;&#8218;uores- <br /> Standard Biological Parts engineered using              cent protein (GFP) &#239;&#172;&#8218;uorescence. Plasmids polymer- <br /> Clontech In-Fusion PCR Cloning Kit described             ase chain reaction ampli&#239;&#172;?ed using vector-speci&#239;&#172;?c <br /> detail previously (21). All circuits encoded        primers, VF2 VR, gene-speci&#239;&#172;?c primers, <br /> pSB1A2 plasmid, high-copy number plasmid (100&#226;&#8364;&#8220;300            GFP200R KAN200R, 25-bp primers are <br /> plasmids/cell), ampicillin resistance gene. Plasmids   reverse complements 200&#226;&#8364;&#8220;225 regions GFP and <br /> were transformed strains chemical transformation      kanamycin-resistant gene, respectively. For con- <br /> or electroporation. Escherichia coli MG1655, multiple          struct, plasmids duplicates sequenced. <br /> deletion strain (MDS42) &#226;&#8364;&#732;Clean Genome&#226;&#8364;&#8482; Scarab <br /> Genomics MG1655 Z1 (15) used inducible ex-          RESULTS <br /> pression LacI-regulated promoters, strain <br /> overexpress LacI chromosome.                          BPD design characterization <br />    The BDPs derived LacI-regulated                Prokaryotic promoters characterized consensus <br /> promoter (R0011) gradual point mutation. Obtained           sequence, conserved sequence motif promoter <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> Registry Standard Biological Parts (14,22),        region. Two hexamers,           35 region (T82T84G78 <br /> R0011 arti&#239;&#172;?cial promoter consisting 35            A65C54A45) 10 region (T80A95T45A60A50T96), <br /> and 10 regions phage lambda cI-binding           distance (16&#226;&#8364;&#8220;18 bp), the <br /> sites replaced identical 17-bp lacO1 operator         conserved characters (25) (the subscripts denote the <br /> sites, 35 10 region            percentage occurrence frequently bases <br /> upstream 35 region. This hybrid design makes            E. coli). Our design rationale identify promoter <br /> R0011 strong promoter, tightly repressed LacI,        forward 35 10 regions happen the <br /> inducible IPTG strains express LacI. It known    reverse complementary sequences backward 10 <br /> that genetic circuits regulated R0011 quickly lose          35 regions. In ideal case, forward and <br /> function (&lt;50 generations) evolutionary half-life      backward consensus sequences exactly overlap each <br /> of  20 generations, owing chromosome misalignment           other. Any mutation (point mutation, insertion or <br /> between homologous sequences replication (termed        deletion) promoter region DNA strand <br /> &#226;&#8364;&#732;replication slippage&#226;&#8364;&#8482;) (23), owing spontaneous direct   inevitably cause mutation reverse promoter. <br /> repeat deletions (24). Because fast well-de&#239;&#172;?ned     As result, theory, expression engineered <br /> mutation promoter, BDPs study         gene protected expression antibiotic resist- <br /> constructed R0011.                                        ance gene crucial genes encoded other <br />                                                                strand shared BDP. <br /> Evolutionary stability measurements <br />                                                                   Although <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> consensus sequence statis- <br /> Three individual transformants circuit tested    tically conserved, exact consensus sequence is <br /> were grown overnight 37  C, shaking 250 r.p.m.        rarely nature. Individual promoters usually <br /> LB+100 mg/ml ampicillin, stored glycerol stock.         differ consensus sequence more <br />  PAGE 3 OF 7                                                                           Nucleic Acids Research, 2013, Vol. 41, No. 1 e33 <br />  <br />  <br />  <br />  <br /> Figure 1. Design bidirectional promoter gradually modifying consensus sequence. (A) Four sequences shown. Sequence 1 35 <br /> and 10 regions consensus sequence; Sequence 2 reverse complement Sequence 1. Sequence 3 promoter sequence identical <br /> Sequence 1 opposite strand reading backwards. Sequence 4 partial overlapping promoter shifted bases. Bases <br /> in green represent identical bases backward (Sequences 3 4) reverse complement sequences (Sequence 2), red color indicates <br /> the positions introduce point mutations create BDP. (B) Illustration genetic constructs, regular promoter (R0011) and <br /> bidirectional promoter (BDP-01) biological parts strands. Curved arrows represent promoters, ovals represent ribosome <br /> binding sites, hexagons represent terminators horizontal arrows represent coding sequences. <br />  <br />  <br /> positions [up 5-bp mismatch (25)]. In                  sequences carry minimum total number mutations <br /> general, similar promoter sequence                     consensus sequence (six mutations) equally <br /> consensus sequence, higher transcription ef&#239;&#172;?ciency                 distribute mutations forward backward <br /> of promoter. The rationale &#239;&#172;?nd 35 10                     strands (three each). The sequences forward and <br /> sequence pairs sequence complementary                            backward promoters identical. There <br /> sequence close consensus sequence                         totally 14 promoter sequences meet criteria <br /> possible, ensuring promoter ef&#239;&#172;?ciency directions.                  (Supplementary Data). We tested BDP <br /> As total 12 bp position                     function constructing red &#239;&#172;&#8218;uorescent protein (RFP) up- <br /> chosen bases, number possible com-                    stream GFP downstream promoter. <br /> binations enormous (see Supplementary Information).                     Unfortunately, &#239;&#172;&#8218;uorescence measurement showed that <br /> Our design strategy start consensus                         constructs able transcribe bidirectionally <br /> sequence strand gradually make point muta-                      produce red green &#239;&#172;&#8218;uorescence <br /> tions promoter achieve BDP functionality.                        (Supplementary Data). <br />    Figure 1A shows design BDP, Sequence                      Besides ideal design complete overlapping of <br /> 1 represents forward 35 10 regions con-                     forward backward promoters, practical <br /> sensus sequence Sequence 2 reverse complement                   partial overlapping design tested. Less stringent <br /> of Sequence 1. To promoter sequence                     robust point mutations complete overlap- <br /> Sequence 1 opposite strand, consensus sequence                   ping design, easier implement, partial overlapp- <br /> reversed create Sequence 3. In words, Sequences 1                 ing design theory maintain promoter function while <br /> and 3 identical sequences, read opposite direc-                 minimizing mutation recombination. Sequence 4 illus- <br /> tions (TTGACA_17bp_TATAAT). However, obtain                             trates partial overlapping design shifting entire <br /> the BDP, Sequences 2 3 match other.                          backward sequence (Sequence 3) positions the <br /> Comparison Sequences 2 3 indicates                       left. In design, positions need mutated <br /> the 12 positions consensus sequence, 6                  (instead six). Other designs forward-shifted and <br /> identical (labeled green), point mutations                  reverse-shifted promoters described Supplementary <br /> are required make match completely (labeled                     Table S1 Supplementary Information. If sequence <br /> red). The variable sites allow 46 possible combinations,               shifted 2 bp, increase chance an <br /> still enormous search space.                                            inactivating mutation opposite strand. In partial <br />    In prove-of-principle work, impossible enumer-              overlapping design, BDP identi&#239;&#172;?ed experimen- <br /> ate possible sequence combinations, experiment                  tally con&#239;&#172;?rmed gradual point mutation proced- <br /> began candidates, designed                          ure. The sequence 10 35 regions BDP-01 is <br />  e33 Nucleic Acids Research, 2013, Vol. 41, No. 1                                                                                  PAGE 4 OF 7 <br />  <br />  <br />  <br />  <br /> Figure 2. Performance stability genetic circuits E. coli MG1655 strain (A) clean genome MDS strain (B). In MG1655 strain, three <br /> circuits studied: regular promoter R0011 control (a), circuit BDP-01 kanamycin resistance gene encoded backward <br /> direction BDP-01 culture kanamycin (b) kanamycin (c). Construct b negative control c, absence of <br /> kanamycin media puts evolutionary pressure KanR, bidirectional promoter free evolutionary changes acts a <br /> regular promoter. In MDS strain, addition circuits tested MG1655 strain, GFP&#226;&#8364;&#8220;kanR fusion protein (d,e, R0011 K+G) and <br /> polycistronic design (f,g, R0011 K-G) compared. <br />  <br />  <br />  <br /> as follows: atTTGACA_15bp_tgTCAAAT (lowercase                             stability dynamics circuits BDP (BDP-01) <br /> denotes 2 bp required backward promoter                     versus unidirectional promoter (R0011) initially <br /> the strand).                                                        measured compared. These circuits encoded on <br />                                                                           high-copy number plasmid conferring ampicillin <br />                                                                           resistance use GFP target gene, the <br /> Evolutionary stability dynamics loss-of-function                      circuit BDP-01 promoter controls expression of <br /> mutations genetic circuits E. coli MG1655                           kanamycin resistance gene reverse direction, <br /> To examine designed BDP does improve                      R0011 does not. R0011 contains identical <br /> mutational robustness genetic circuits, evolutionary               17-bp lac operator sites. Derived R0011, BDP01 <br />  PAGE 5 OF 7                                                                             Nucleic Acids Research, 2013, Vol. 41, No. 1 e33 <br />  <br />  <br /> and R0011 differ 35 10 region,                          original R0011 circuit BDP circuit propagated <br /> share double operator sites.                                            kanamycin, replication slippage took place all <br />    Figure 2A shows evolutionary half-life                   samples, resulting deletion lacO sequence the <br /> original R0011 circuit (a)  20 generations,                       35 region. In contrast, deletions the <br /> GFP &#239;&#172;&#8218;uorescence/OD level drops half initial                       BDP circuit propagated kanamycin. All mutations in <br /> value. When promoter replaced BDP-01,                        BDP kanamycin-evolved populations caused by <br /> with kanamycin growth media (b), circuit                     insertion sequence (IS) elements (26,27) insert into <br /> has evolutionary half-life  45 generations. As                        downstream BDP-01 region, disrupting expres- <br /> BDP-01 does protective function                      sion GFP leaving KanR expression intact. <br /> antibiotic, possible reasons                 Interestingly, 13 identi&#239;&#172;?ed IS insertions, 6 are <br /> improved life span. One reason lowered ex-                     IS2 elements insert right 2 bp the <br /> pression gene, average predicted                      forward 10 region, right upstream the <br /> improve evolutionary half-life (15). The second reason                   backward 35 region. IS2 elements insert AT- <br /> the 2-bp shorter repeated lacO regions BDP-01 relative                   rich region (AA-IS2-AT), hotspot IS2 <br /> to R0011, presumably lower replication slippage                   insertion. Therefore, insertions took place <br /> rate. When kanamycin added media (c),                        BDP region, disrupting forward promoter but <br /> half-life BDP-01 circuit increased  80 gen-                  reverse promoter, expression kanamycin <br /> erations. The comparison BDP-01 circuit                     resistance gene affected. <br /> without kanamycin shows rationally designed <br /> BDP-01 promoter does provide protective function                          Evolutionary stability dynamics loss-of-function <br /> that increases evolutionary stability genetic circuit.               mutations genetic circuits E. coli MDS42 (&#226;&#8364;&#732;Clean <br />    Sequence analysis loss-of-function plasmids                        Genome&#226;&#8364;&#8482; strain) <br /> evolved populations demonstrates reason pro-                   Because IS insertions major cause mutations in <br /> tective function (Table 1 Figure 3). In                        BDP-01 circuits, next, obtained cell lines devoid of <br />  <br />  <br />  <br />  <br /> Figure 3. Mutation map loss-of-function mutations evolved MG1655 populations shown In Table 1. (A) (B), deletion one <br /> Lac operator 35 region homologous recombination (replication slippage) original R0011 circuit BDP01 circuit <br /> without kanamycin growth media. (C), No replication slippage found. Instead, 13 recognizable insertions, 4 cases located the <br /> forward GFP coding region ribosomal biding site, 9 cases promoter region 6 cases located exactly boundary <br /> of start position reverse promoter 35 region. The BDP colored red (complete overlapping forward reverse strain) yellow <br /> (partial overlapping). <br />  e33 Nucleic Acids Research, 2013, Vol. 41, No. 1                                                                                          PAGE 6 OF 7 <br />  <br />  <br /> Table 1. The sequencing results loss-of-function mutations evolved MG1655 populations <br />  <br />      Promoter      Reverse gene     Forward gene       Strain       Media           1       2      3       4        5      6        7       8      9 <br />  <br /> a    R0011         N/A              GFP                MG1655       Amp <br /> b    BDP-1         KanR             GFP                MG1655       Amp <br /> c    BDP-1         KanR             GFP                MG1655       Amp+Kan         IS2     IS2    IS2     N/A      IS2    N/A      IS5     IS2    IS2 <br />  <br /> For replicate-evolved population, cells highlighted red indicate mutations deletion replication slippage; orange cells <br /> indicate mutations insertion IS elements. Cells highlight indicate sequencing results vague indiscernible. <br /> N/A = applicable. <br />  <br />  <br /> IS elements test strain increases evolution-                  circuits lower circuits. <br /> ary stability BDP-01 circuits. The &#226;&#8364;&#732;Clean Genome&#226;&#8364;&#8482;                      This generally lower expression differ- <br /> E. coli MDS42 strain (multiple deletion strain) (28)                       ences strain, improper folding GFP fusion <br /> Scarab Genomics (http://www.scarabgenomics.com/)                         protein circuit and/or differences translation rate <br /> synthetic strain 15% genome deleted                      GFP KanR polycistronic circuit. <br /> E. coli K12, including non-essential genes, recombinogenic                    Despite low expression level, circuits quickly <br /> or mobile DNA (including IS elements),and cryptic                             lose function, kanamycin, an <br /> virulent genes. Figure 2B shows evolutionary                              evolutionary half-life  50 generations. This result <br /> dynamics various circuits propagated MDS42.                             shows target gene (GFP) protected by <br /> Overall, expression level GFP lower                         directly connecting essential gene fusion <br /> strain relative MG1655. The R0011 circuit evo-                      polycistronic link, accordance previous work. <br /> lutionary half-life  18 generations, as <br /> when propagated MG1655. The BDP-01 circuit grown <br /> without kanamycin media evolutionary                            DISCUSSION <br /> half-life  105 generations, double                       In study, designed, constructed and <br /> in MG1655. The BDP-01 circuit grown kanamycin                            characterized BDP expresses GFP forward <br /> media evolutionary half-life  185 generations,                      direction kanamycin resistance gene reverse <br /> also double compared MG1655. The                               direction. The utility BDP-01 demonstrated the <br /> kanamycin-evolved BDP-01 circuit 10 times                           increased evolutionary stability circuit when <br /> the life span original R0011 circuit MDS42                      propagated media kanamycin. Furthermore, the <br /> strain.                                                                       loss-of-function mutations kanamycin-evolved <br />    The GFP &#239;&#172;&#8218;uorescence level BDP-01 circuits,                              populations occur GFP-expressing component <br /> propagated kanamycin, &#239;&#172;?rst dropped                           circuit, indicating BDP-01 promoter ex- <br /> to half maximum magnitude, rose                        pressing kanamycin resistance essential survival. <br /> highest level dropped near zero. These                           The design BDP sequence genetic circuit controlled <br /> dynamics observed independently evolved                       BDP independent target gene interest; this <br /> populations kanamycin. This phenom-                           design generally applied synthetic genetic <br /> enon probably occurs physiological                         circuit. <br /> adaptation MDS42 strain required maintain                              It clear BDP-01 circuit improved evo- <br /> this circuit high-copy number plasmid. As                           lutionary stability propagated kanamycin versus <br /> strain 15% smaller normal genome,                           kanamycin, compared control <br /> take time maximize growth rate, given                         KanR-expressing circuits difference <br /> extra metabolic load high-copy number                              stability. In previous study (15), that <br /> plasmid. Sequence analysis loss-of-function                            GFP&#226;&#8364;&#8220;KanR fusion protein polycistronic <br /> plasmids showed IS insertion observed                          circuits lost function rate, whether <br /> evolved populations, indicating use MDS                           propagated kanamycin not, consistent this <br /> strain prevents prevalent IS insertion MG1655                          study. Therefore, difference KanR- <br /> strains. However, various types insertion                        expressing control circuits BDP-01 circuit not <br /> recombination-based mutations happened. Besides,                              different promoters use, R0011 this <br /> despite rare MG1655, point mutations                       study, R0010 promoter (more mutationally robust <br /> found MDS42 strains, possibly indicating                      absence repeated operator sequences) in <br /> of truncated genome, replication repairing machin-                    previous study. The major difference design of <br /> ery clean genome ef&#239;&#172;?cient                     circuit. Although metabolic load BDP circuit <br /> complete genome (Supplementary Information).                                  control circuits similar, GFP&#226;&#8364;&#8220;KanR fusion <br />    In addition, control circuits tested                 protein polycistronic circuits produce long RNA <br /> MDS42 strain. One circuit expresses GFP&#226;&#8364;&#8220;KanR fusion                         products (and long proteins fusion version), <br /> protein glycine&#226;&#8364;&#8220;serine linker circuit                      affect transcription translation rate and <br /> expresses GFP polycistronically linked KanR. Both                        biological function &#239;&#172;?nal protein product. <br /> circuits originally engineered previous study (15).                 In contrast, BDP design keeps original RNA <br /> As shown Figure 2B, initial expression levels                       protein product target gene (GFP) the <br />  PAGE 7 OF 7                                                                         Nucleic Acids Research, 2013, Vol. 41, No. 1 e33 <br />  <br />  <br /> protection gene (kanamycin resistance), normal                   7. Weber,W. Fussenegger,M. (2009) The impact synthetic <br /> biological function maintained.                                           biology drug discovery. Drug Discov. Today, 14, 956&#226;&#8364;&#8220;963. <br />                                                                           8. Gibson,D.G., Glass,J.I., Lartigue,C., Noskov,V.N., Chuang,R.Y., <br />    Another interesting result GFP expression                     Algire,M.A., Benders,G.A., Montague,M.G., Ma,L., <br /> levels MDS42  40% lower MG1655                             Moodie,M.M. et al. (2010) Creation bacterial cell controlled <br /> average circuits. Previous work demonstrated                     chemically synthesized genome. Science, 329, 52&#226;&#8364;&#8220;56. <br /> that expression level evolutionary half-life              9. Wang,H.H., Isaacs,F.J., Carr,P.A., Sun,Z.Z., Xu,G., Forest,C.R. <br />                                                                              Church,G.M. (2009) Programming cells multiplex genome <br /> different genetic circuits negatively correlated;                     engineering accelerated evolution. Nature, 460, 894&#226;&#8364;&#8220;898. <br /> average, higher expression level, shorter                10. Glick,B.R. (1995) Metabolic load heterologous gene <br /> half-life (15). However, observation based                        expression. Biotechnol. Adv., 13, 247&#226;&#8364;&#8220;261. <br /> comparing different circuits strain. The                     11. Cooper,V.S. Lenski,R.E. (2000) The population genetics of <br /> increase 80 185 generations MG1655                        ecological specialization evolving Escherichia coli populations. <br />                                                                              Nature, 407, 736&#226;&#8364;&#8220;739. <br /> MDS42 strains, respectively, kanamycin-evolved                   12. You,L., Cox,R.S. 3rd, Weiss,R. Arnold,F.H. (2004) <br /> BDP-01 circuit, likely decrease                     Programmed population control cell-cell communication and <br /> expression level, evolutionary half-life                       regulated killing. Nature, 428, 868&#226;&#8364;&#8220;871. <br /> original R0011 circuit did change ( 20 generations                   13. Elena,S.F. Lenski,R.E. (2003) Evolution experiments with <br /> in MG1655 MDS42 strains).                                           microorganisms: dynamics genetic bases adaptation. <br />                                                                              Nat. Rev. Genet., 4, 457&#226;&#8364;&#8220;469. <br />    As learned clean genome evolution ex-                 14. Canton,B., Labno,A. Endy,D. (2008) Re&#239;&#172;?nement and <br /> periment, evolutionary half-life designed syn-                    standardization synthetic biological parts devices. Nat. <br /> thetic circuit increases IS elements eliminated,                    Biotechnol., 26, 787&#226;&#8364;&#8220;793. <br /> but eventually mutations, point mutations,                 15. Sleight,S.C., Bartley,B.A., Lieviant,J.A. Sauro,H.M. (2010) <br />                                                                              Designing engineering evolutionary robust genetic circuits. <br /> insertions/deletions recombination, happen,                     J. Biol. Eng., 4, 12. <br /> the designed circuit outcompeted. Step step,               16. Arkin,A.P. Fletcher,D.A. (2006) Fast, cheap somewhat <br /> try eliminate probable mutations, evolu-                     control. Genome Biol., 7, 114. <br /> tion pick feasible mutations eventually                17. Sleight,S.C., Orlic,C., Schneider,D. Lenski,R.E. (2008) Genetic <br /> take over. It everlasting procedure synthetic                      basis evolutionary adaptation Escherichia coli stressful <br />                                                                              cycles freezing, thawing growth. Genetics, 180, 431&#226;&#8364;&#8220;443. <br /> biology elongate life span synthetic circuits.                 18. Beck,C.F. Warren,R.A. (1988) Divergent promoters, a <br />                                                                              common form gene organization. Microbiol. Rev., 52, 318&#226;&#8364;&#8220;326. <br />                                                                          19. Yan,G., Fu,Y. Faustman,D.L. (1997) Reduced expression of <br /> SUPPLEMENTARY DATA                                                           Tap1 Lmp2 antigen-processing genes nonobese diabetic <br />                                                                              (NOD) mouse mutation shared bidirectional <br /> Supplementary Data available NAR Online:                              promoter. J. Immunol., 159, 3068&#226;&#8364;&#8220;3080. <br /> Supplementary Tables 1&#226;&#8364;&#8220;5.                                                20. Venter,M. (2007) Synthetic promoters: genetic control cis <br />                                                                              engineering. Trends Plant Sci., 12, 118&#226;&#8364;&#8220;124. <br />                                                                          21. Sleight,S.C., Bartley,B.A., Lieviant,J.A. Sauro,H.M. (2010) <br /> FUNDING                                                                      In-fusion biobrick assembly re-engineering. Nucleic Acids <br />                                                                              Res., 38, 2624&#226;&#8364;&#8220;2636. <br /> Funding open access charge: Waived Oxford                         22. Endy,D. (2005) Foundations engineering biology. Nature, 438, <br /> University Press.                                                            449&#226;&#8364;&#8220;453. <br />                                                                          23. Lovett,S.T. (2004) Encoded errors: mutations rearrangements <br /> Con&#239;&#172;&#8218;ict statement. None declared.                                mediated misalignment repetitive DNA sequences. Mol. <br />                                                                              Microb <br /> </body></html>