<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> Emergence P2 <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>Phenotype</span> Pseudomonas aeruginosa PAO1 <br />  Strains Involves Various Mutations mexT mexF <br />  Preston M. Luong,a Benjamin D. Shogan,a Alexander Zaborin,a Natalia Belogortseva,a Joshua D. Shrout,b Olga Zaborina,a <br />  John C. Alverdya <br /> &#226;&#8364;&#185;Department Surgery, University Chicago, Chicago, Illinois, USAa; Department Civil &amp; Environmental Engineering Earth Sciences, University Notre Dame, <br />  Notre Dame, Indiana, USAb <br />  <br />  We recently demonstrated Pseudomonas aeruginosa PAO1 undergoes pronounced phenotypic change introduced <br />  intestines rats surgical injury. Recovered strains displayed specific phenotype (termed P2 phenotype) <br />  characterized altered pyocyanin production, high collagenase activity, high swarming motility, low resistance chloram- <br />  phenicol, increased killing Caenorhabditis elegans compared inoculating strain (termed P1 phenotype). The <br />  aims study characterize differences P. aeruginosa P1 P2 phenotypes quorum sensing and <br />  competitiveness. We determined presence P2 phenotype PAO1 strains various laboratories. Results <br />  demonstrated P2 cells display accelerated growth early exponential phase early activation quorum-sensing <br />  systems overcome growth P1 cells mixed population. Among PAO1 strains obtained different labora- <br />  tories, exhibited P2 phenotype. Of 27 mutants analyzed P. aeruginosa MPAO1 transposon library, 25 displayed <br />  P2 phenotypes. The P2 phenotype cases correlated lack expression mexE mexF mutations mexT <br />  mexF genes. In summary, strains possessing P2 phenotype distributed PAO1 strains commonly used a <br />  variety research laboratories. Genetically, characterized various mutations mexT mexF. <br />  <br />  <br />  <br />  S    table genetic mutations change phenotypes arise as <br />       microbes adapt environment maximize propaga- <br />  tion. This ecologically dependent shift phenotype doc- <br />                                                                                         cules, account attenuated virulence observed <br />                                                                                         strains functional MexT protein (5, 16, 17). <br />                                                                                             In study, performed detailed analyses intra- and <br />  umented bacterial species diverse contexts (1, 2).                      extracellular levels QS signaling molecules&#226;&#8364;&#8221;C4-homo- <br />  Recently, demonstrated MPAO1 strain Pseudomo-                           serine lactone (C4-HSL), 3OC12-HSL, Pseudomonas quino- <br />  nas aeruginosa transformed (or selected for)                          lone signal (PQS)&#226;&#8364;&#8221;in strains MPAO1-P1 MPAO1-P2 during <br />  virulent phenotype present colon rats subjected                      early stationary phase prior visible pyocyanin production in <br />  preoperative radiation followed colon resection anastomo-                       MPAO1-P2. MPAO1-P2 demonstrated accumulation intra- <br />  sis (3). The strain recovered anastomotic tissue (termed                      extracellular C4-HSL, 3OC12-HSL, PQS significantly <br />  MPAO1-P2, i.e., having P2 phenotype) exhibited high colla-                         higher activation RhlI transition stationary phase. We <br />  genase activity, swarming ability, increased pyocyanin production                      observed MPAO1-P2 greater competitiveness in <br />  liquid medium, increased tissue-destroying capacity com-                        mixed cultures MPAO1-P1, suggesting strains mu- <br />  pared initial strain (termed MPAO1-P1, having P1 phe-                       tations genes affect functionality MexEF-OprN <br />  notype). Comparative genomic sequencing analysis revealed                         efflux pump possessing P2 phenotype predominate. We <br />  MPAO1-P2 harbored single-nucleotide polymorphism (SNP)                               analyzed strains P. aeruginosa PAO1 obtained <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />  mexT gene results truncated nonfunctional                         laboratories exhibited P1 or <br />  MexT protein. Transformation mexT-P1 MPAO1-P2 re-                              P2 phenotype. Strains P2 phenotype harbored mutations <br />  verted strain P1 phenotype. The phenotypes P.                       mexT mexF. Additionally, analyzed mutant strains from <br />  aeruginosa similar P1 P2 previously                          P. aeruginosa transposon library (18), con- <br />  described (4&#226;&#8364;&#8220;7). In cited studies, wild-type PAO1 strain                       structed parent strain MPAO1-P1, discovered that <br />  harbored 8-bp insertion mexT results nonfunctional                     strains harbor P2 phenotype. Our data demonstrate that <br />  MexT protein phenotype similar P2. Its derivative mutant                      broad distribution P2 phenotype various <br />  selected norfloxacin (referred nfxC-type mu- <br />  tant) functional MexT protein characterized by <br />                                                                                            Received 9 September 2013 Accepted 4 November 2013 <br />  attenuated pyocyanin production (5), high resistance chloram- <br />                                                                                            Published ahead print 15 November 2013 <br />  phenicol (6), absence swarming motility (8) similar the <br />                                                                                            Address correspondence John C. Alverdy, jalverdy@surgery.bsd.uchicago.edu, <br />  P1 strains study (3). mexT described muta-                           Olga Zaborina, ozaborin@surgery.bsd.uchicago.edu. <br />  tional &#226;&#8364;&#339;hot spot,&#226;&#8364;? mutations contribute global phe-                          J.C.A. O.Z. joint senior coauthors. <br />  notypic changes P. aeruginosa (6, 9&#226;&#8364;&#8220;11). MexT transcrip-                          Supplemental material article http://dx.doi.org/10.1128 <br />  tional regulator MexEF-OprN efflux pump,                            /JB.01050-13. <br />  extensively characterized context antimicrobial agents                          Copyright &#194;&#169; 2014, American Society Microbiology. All Rights Reserved. <br />  (12&#226;&#8364;&#8220;15). It suggested MexEF-OprN pump                           doi:10.1128/JB.01050-13 <br />  involved efflux quorum-sensing (QS) signaling mole- <br />  <br />  <br />  504    jb.asm.org                                      Journal Bacteriology   p. 504 &#226;&#8364;&#8220;513                                            January 2014 Volume 196 Number 2 <br />                                                                                                                                 Emergence P2 Phenotype PAO1 Strains <br />  <br />  <br />  <br /> TABLE 1 Mutants displaying P1 P2 phenotypesa <br />                                                                                                    Pyocyanin <br />                                                                                                    production <br /> Mutant       P. aeruginosa        Mutant                                                                                   Chloramphenicol          Collagenase        Swarming <br /> no.          IDb                  IDc              Function                                        Agar       Liquid       resistance               activity           motility <br /> 1            PA1003               ID13375          MvfR                                            &#226;&#171;&#186;          &#226;&#171;&#186;            &#226;&#171;&#185;                        &#226;&#171;&#186;                  &#226;&#171;&#186; <br /> 2            PA5361               ID30430          PhoR                                            &#226;&#171;&#185;          &#226;&#171;&#186;            &#226;&#171;&#185;                        &#226;&#171;&#186;                  &#226;&#171;&#186; <br /> 3            PA5394               PW10101          Cardiolipin synthase                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 4            PA0452               PW1826           Flotilin                                        &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 5            PA2389               ID32408          PvdR                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 6            PA2391               ID9422           OpmQ                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 7            PA0685               ID39358          Probable type II secretion protein              &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 8            PA0690               ID18158          Hypothetical protein                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 9            PA0691               ID4721           Hypothetical protein                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 10           PA0696               ID11082          Hypothetical protein                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 11           PA0051               ID39650          PhzH                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 12           PA0674               ID21628          VreA                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 13           PA0675               ID9556           ECF sigma factor - VreI                         &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 14           PA0676               ID16317          Sigma factor regulator - VreR                   &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 15           PA0678               ID42074          HxcU putative pseudopilin                       &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 16           PA0677               ID19639          HxcW putative pseudopilin                       &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 17           PA0678               ID54030          HxcU putative pseudopilin                       &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 18           PA0679               ID20782          Hypothetical protein                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 19           PA0680               ID35854          HxcV putative pseudopilin                       &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 20           PA0682               ID33753          HxcX atypical pseudopilin                       &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 21           PA5170               ID31482          ArcD                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 22           PA0688               ID12226          Probable binding protein component              &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br />                                                       ABC transporter <br /> 23           PA0681               ID5144           HxcT pseudopilin                                &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 24           PA4221               ID48243          FptA                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 25           PA2388               ID6650           FpvR                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 26           PA5360               ID40588          PhoB                                            &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> 27           PA2426               PW5085           Sigma factor, PvdS                              &#226;&#171;&#186;          &#226;&#171;&#185;            &#226;&#171;&#186;                        &#226;&#171;&#185;                  &#226;&#171;&#185; <br /> a <br />   Chloramphenicol resistance defined growth 500 ng/&#226;?&#174;l chloramphenicol (n &#226;&#171;&#189; 2). Collagenase activity defined production collagenase 4 h growth TSB at <br /> levels 5 times higher P1 (negatives lower collagenase P1) (n &#226;&#171;&#189; 2). Mutant 1 (MvfR) does produce pyocyanin lack quinolone <br /> production. <br /> b <br />   As reported Pseudomonas Genome Database http://www.pseudomonas.com/. <br /> c <br />   As reported P. aeruginosa PAO1 transposon mutant library http://www.gs.washington.edu/labs/manoil/libraryindex.htm. <br />  <br />  <br />  <br /> PAO1 strains mutations affecting MexEF-OprN                              struct pUCP24/RFP plasmid, amplified gene encoding DS-Red <br /> pump.                                                                                      Express 2 protein plasmid pQE60NA-Express 2 (19) using primers <br />                                                                                            XbaI-Ds-Red Ds-Red-HindIII (see Table S1 supplemental ma- <br /> MATERIALS AND METHODS                                                                      terial) cloned Escherichia coli-P. aeruginosa shuttle vector <br /> Bacterial strains. P. aeruginosa strain MPAO1 (referred MPAO1-P1)                    pUCP24 (20) XbaI-HindIII cleavage sites create plasmid <br /> and 27 mutants (Table 1) obtained University Wash-                    pUCP24/RFP. The plasmid electroporated competent cells of <br /> ington transposon mutant library (18). MPAO1-P2 transformed                         MPAO1 strain P1/RFP, selected Pseudomonas isola- <br /> MPAO1-P1 strain isolated leaking rat anastomoses previously                        tion agar plates containing 100 &#226;?&#174;g/ml gentamicin. The selected clones <br /> described (3). W-PAO1 obtained Klaus Winzer, D-PAO1                           verified production red fluorescence confocal micros- <br /> was obtained Steve Diggle, University Nottingham, United Kingdom.                  copy. <br /> Although came laboratory Paul Williams, strains                    Construction MPAO1-P2/GFP (P2/GFP) expressing green fluo- <br /> were obtained 10 years apart. H-PAO1 obtained Tong Chuan He,                      rescence. P. aeruginosa MPAO1-P2 chromosomally tagged using a <br /> University Chicago, Chicago, IL. R-PAO1 obtained Cornelia                      mini-Tn7 insertion constitutively expressed green fluorescent pro- <br /> Reimmann, University Lausanne, Switzerland. C-PAO1 obtained                         tein (GFP) (21). Uptake pBK-miniTn7-gfp2 plasmid DNA pres- <br /> from Pierre Cornelis, Vrije University, Brussels, Belgium. Ig-PAO1 ob-                 ence pRK600 mobilization plasmid pUX-BF13 conjugation <br /> tained Barbara Iglewski, University Rochester, Rochester, NY. L-                   helper plasmid MPAO1-P2 recipient achieved conjuga- <br /> PAO1 obtained Linda McCarter, University Iowa, Iowa City, IA.                  tional mating transformants selected 100 &#226;?&#174;g/ml gentami- <br /> Ma-PAO1 obtained Colin Manoil, University Washington, Se-                      cin (22, 23). <br /> attle, WA. All strains stored 10% glycerol stock &#226;&#171;&#186;80&#194;&#176;C. Only cells                  Collagenase assay. Collagenase activity assessed rate of <br /> freshly plated stock used experiments. Cells stock                  degradation fluorescein-conjugated gelatin using EnzChek gelatin- <br /> plated tryptic soy broth (TSB) plates grown overnight 37&#194;&#176;C                 ase/collagenase assay kit (Molecular Probes). <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> 96-well <br /> were used designed.                                                             black clear-bottom plate, 180 &#226;?&#174;l TSB, 10 &#226;?&#174;l overnight P. aeruginosa <br />     Construction MPAO1-P1/RFP (P1/RFP) expressing red fluores-                          culture TSB, 10 &#226;?&#174;l gelatin substrate (1.0 mg/ml; DQ gelatin <br /> cence. P. aeruginosa MPAO1 transformed constructed plas-                      [catalog no. D12054; Molecular Probes] pig skin, fluorescein conju- <br /> mid pUCP24/RFP create red fluorescent strain P1/RFP. To con-                        gate) added. The negative control consisted 190 &#226;?&#174;l TSB 10 &#226;?&#174;l <br />  <br />  <br /> January 2014 Volume 196 Number 2                                                                                                                                  jb.asm.org 505 <br />  Luong et al. <br />  <br />  <br />  <br /> fluorescein-labeled gelatin. Fluorescence gelatin degradation            5,000 rpm resuspended 500 &#226;?&#174;l phosphate-buffered saline (PBS). <br /> bacteria measured 3 h incubation 37&#194;&#176;C 160 rpm. Values          The cell suspension analyzed fluorescence measured Tecan <br /> from negative control subtracted values strain.              Safire 2 multidetection microplate reader 560-nm/580-nm excitation/ <br /> Fluorescence values normalized cell density 600 nm. Experi-             emission red fluorescence 344-nm/434-nm excitation/emis- <br /> ments performed triplicate.                                                sion green fluorescence. Background red fluorescence displayed by <br />     Swarming assay. Swarming medium consisted 20 mM NH4Cl, 12                   MPAO1-P2/GFP (P2/GFP) subtracted red fluorescent values of <br /> mM Na2HPO4, 22 mM KH2PO4, 8.6 mM NaCl, 1 mM MgSO4, 1 mM                            pure MPAO1-P1/RFP (P1/RFP) mixed cultures assigned measure <br /> CaCl2, 11 mM dextrose, 0.5% Casamino Acids, 0.5% Difco Bacto-                  P1/RFP. Similarly, background green fluorescence displayed P1/RFP <br /> agar described previously (24). Prepared plates dried overnight         subtracted green fluorescence values pure P2/GFP mixed <br /> room temperature inoculated P. aeruginosa colonies              cultures assigned measure P2/GFP. Lastly, fluorescence values were <br /> picked sterile pipette tip overnight Pseudomonas isolation             normalized average fluorescence pure cultures equaled <br /> agar (PIA) plates. Swarming plates incubated 30&#194;&#176;C overnight.               1,000 relative fluorescence units. Direct comparisons between <br />     Caenorhabditis elegans killing assay. Wild-type C. elegans N2 pro-             fluorescence mixed culture pure culture particular <br /> vided Caenorhabditis Genetic Center (CGC), University Min-               color fluorescence. <br /> nesota, used experiments. Culturing, cleaning, egg preparation              To measure P1/P2 competitiveness coincubation intesti- <br /> for synchronization, transferring performed accordingly ref-           nal epithelial cells, HT-29 monolayers prepared follows. <br /> erence 25. Nematodes prepared experiment previously                HT-29 cells passages 19 34 grown Dulbecco&#226;&#8364;&#8482;s mini- <br /> described (26). The killing assay performed previously described            mal essential medium (DMEM; Gibco, Long Island, NY) supplemented <br /> (26) small modifications. Briefly, prepared nematodes trans-             10% fetal bovine serum (Gibco), 2 mM L-glutamine (Gibco), and <br /> ferred kanamycin (Km)-containing plates 1.3 ml P. aeruginosa             antibiotics (100 U/ml penicillin, 100 &#226;?&#174;g/ml streptomycin). Cells were <br /> culture grown 5 h 0.1&#226;&#171;&#187; TY (1 g/liter tryptone, 0.5 g/liter yeast extract,   cultured water-saturated atmosphere 95% air/5% CO2 form <br /> and 0.1 mM potassium phosphate buffer [pH 6.0]). Worms incubated           monolayers. P1/RFP P2/GFP solutions prepared DMEM an <br /> room temperature shaking. Fifteen worms plate plates     OD600 0.03, mixed solution prepared DMEM an <br /> experiments used variant. Nematodes considered dead     OD600 0.06 equal amounts strain. The original medium of <br /> did respond touch platinum picker.                                 HT-29 monolayers replaced 1 ml pure mixed cultures. <br />     Complementation &#226;&#338;&#172;hxcT hxcT. Complementation &#226;&#338;&#172;hxcT                   The coculture HT-29 monolayers incubated 5 h, which <br /> with hxcT performed using plasmid pUCP24/hxcT harboring                DMEM aspirated centrifuged 5,000 rpm pellet plank- <br /> hxcT gene. To construct pUCP24/hxcT, hxcT gene amplified              tonic cells. The cells resuspended 500 &#226;?&#174;l PBS red and <br /> MPAO1 genome DNA using primers HxcT-XbaI HxcT-HindIII (see                     green fluorescence measured. To examine P. aeruginosa adhered <br /> Table S1 supplemental material) cloned pUCP24 (20)             HT-29 monolayer, 1 ml lysis buffer consisting 20 mM Tris-HCl <br /> restriction sites XbaI HindIII. The complemented clone selected            (pH 7.4), 150 mM NaCl, 1% Triton X-100 added cell dish. <br /> on Pseudomonas isolation agar plates containing 100 &#226;?&#174;g/ml gentamicin               Cells scraped used fluorescence analysis. Fluorescence <br /> and PCR verified intact region mutant                values normalized background fluorescence pure PBS <br /> transposon inserted. Primers used designed Primer3 software              lysis buffer. Additionally, background red fluorescence values from <br /> (see Table S1 supplemental material).                                       pure P2/GFP cultures background green fluorescence values from <br />     C4-HSL/3OC12-HSL/PQS assay. P. aeruginosa strains grown                   pure P1/RFP cultures subtracted normalized fluorescence val- <br /> overnight TSB, diluted optical density 600 nm (OD600) 0.03          ues mean fluorescence pure cultures adjusted to <br /> with fresh TSB, continued grow 37&#194;&#176;C 160 rpm bac-              1,000 relative fluorescence units described above. Direct comparisons <br /> terial cell culture collection 6 h. Bacterial cultures centrifuged      fluorescence mixed culture pure <br /> 5,000 rpm 5 min separate pellet culture supernatant. The su-            culture particular color fluorescence. <br /> pernatant tested extracellular quorum-sensing (QS) molecules.                  Measurement pyocyanin. Pyocyanin measured previously <br /> Briefly, supernatant filtered 0.22-&#226;?&#174;m filter used           described (26, 29) 1.8 ml bacterial culture using 500 &#226;?&#174;l chloro- <br /> medium luminescent reporter strains. Luminescence recorded                 form extraction pyocyanin conditioned medium followed <br /> after 2 h reporter strain incubation supernatants normalized           reextraction 250 &#226;?&#174;l 0.2 M HCl. A 100-&#226;?&#174;l portion aqueous <br /> to P. aeruginosa cell density time collection (6 h). The bacterial       layer containing pyocyanin used OD520 measurements a <br /> pellet tested intracellular QS molecules. Briefly, bacterial pellets       PowerWave XS microplate spectrophotometer (Bio-Tek). <br /> were washed twice 1 ml TSB wash, subsequently, 100 &#226;?&#174;l 0.1              Growth P1 P1/P2 conditioned medium. To prepare condi- <br /> mm zirconia beads 500 &#226;?&#174;l TSB added pellet. Cells              tioned medium, grew P1 P2 TSB 8 h 37&#194;&#176;C 160 rpm. <br /> lysed using vortex adaptor Genie 2 (Ambion). After cells           Cells removed centrifugation, conditioned medium was <br /> been lysed vortexing maximum speed 10 min, 700 &#226;?&#174;l fresh TSB              passed 0.22-&#226;?&#174;m filter. <br /> was added supernatant filtered 0.22-&#226;?&#174;m filter.                   Overnight P1 cells diluted 1:100 cell-free medium for <br /> Protein concentrations measured Qubit 2.0 fluorometer                total volume 200 &#226;?&#174;l 96-well plate, incubated 37&#194;&#176;C <br /> using Qubit protein assay kit (Invitrogen) measured value             160 rpm. Optical density 600 nm tracked dynamically from <br /> from fresh TSB subtracted, supernatants used media lu-             0 18 h. <br /> minescent reporter strains. Luminescence measured 2 h reporter                Quantitative reverse transcriptase PCR (qRT-PCR) analysis. Three <br /> strain incubation normalized protein concentration. The following           individual colonies P1 P2 picked freshly prepared plates, <br /> reporter strains used: C4-HSL, E. coli harboring reporter plasmid     inoculated 2 ml TSB, grown overnight. Overnight cultures were <br /> pSB536 (27); 3OC12-HSL, E. coli harboring reporter plasmid                 diluted 2 ml fresh TSB OD600 0.03 grown 3.5 6 h. Two <br /> pSB1705 (27); PQS, P. aeruginosa &#226;&#338;&#172;PqsA/pqsA::luxCDABE (28).                volumes RNA Protect bacterial reagent (Qiagen) added 2 ml of <br />     Competitiveness assay. To assess competitiveness, red fluorescent              culture isolation RNA using UltraClean microbial RNA iso- <br /> MPAO1-P1 (P1/RFP) green fluorescent MPAO1-P2 (P2/GFP)                     lation kit (Mo Bio). DNA removed using Turbo DNA-free kit <br /> incubated separately mixed cultures 2 ml TSB. Each pure            (Invitrogen). For cDNA synthesis, 10 ng RNA converted cDNA <br /> culture 0 h OD600 0.03, mixed cultures 0 h contained         using high-capacity RNA-to-cDNA kit (Applied Biosystems). For qP- <br /> equal amounts strain total OD600 0.06. The TSB cultures           CRs, total volume 10 &#226;?&#174;l used included 1 &#226;?&#174;l cDNA, 0.2 &#226;?&#174;l <br /> were incubated 6 h, cells pelleted centrifugation       10 mM forward reverse primer (see Table S1 supplemental <br />  <br />  <br /> 506   jb.asm.org                                                                                                                         Journal Bacteriology <br />                                                                                                              Emergence P2 Phenotype PAO1 Strains <br />  <br />  <br />  <br />  <br /> FIG 1 MPAO1-P2 demonstrates early activation quorum sensing accumulation C4-HSL 3OC12-HSL -P2 compared MPAO1-P1 growth <br /> in TSB. (A) Greenish pigmentation suggesting higher pyocyanin production P2. (B) Pyocyanin measured 6 h 10 h. (C) Growth curves MPAO1-P1 <br /> and MPAO1-P2. n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.001. (D E) Intracellular (D) extracellular (E) levels C4-HSL, 3OC12-HSL, PQS 6 h. n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.05. (F) <br /> Expression rhlI, lasI, pqsA 6 h. n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.01. Black bars represent MPAO1-P1 gray bars represent MPAO1-P2. <br />  <br />  <br />  <br /> material), 5 &#226;?&#174;l Platinum SYBR green qPCR SuperMix-UDG                  produce large amounts pyocyanin compared MPAO1-P1 <br /> ROX (Invitrogen). qPCR performed using 7900HT fast real-time            (P1), produces negligible pyocyanin over- <br /> PCR (Applied Biosystems). The program amplification           night growth TSB (3). Here, performed dynamic tracking of <br /> initial heating step 50&#194;&#176;C 2 min, followed denaturation step    pyocyanin-specific pigmentation P1 P2 that <br /> 95&#194;&#176;C 15 s 40 cycles 95&#194;&#176;C 15 s 60&#194;&#176;C 1 min. Each      green pigmentation P2 appears early 8 h (Fig. 1A). <br /> sample run triplicate. The RT-negative control run ensure <br /> complete DNA removal. Gene expression normalized expression of <br />                                                                                 OD520 measurements pyocyanin extracted culture super- <br /> the housekeeping gene PA4748 previously described (30). Primers used         natants confirmed increased pyocyanin levels P2 8 h and <br /> were designed Primer3 software.                                            (Fig. 1B). Since pyocyanin biosynthesis regulated quo- <br />     Sequencing mexT mexF. For sequencing mexT mexF,             rum sensing (31), production pyocyanin time point <br /> colony overnight TSB agar plate collected sterile           suggests earlier activation quorum sensing P2 strain. <br /> pipette tip placed 10 &#226;?&#174;l distilled water, 1 &#226;?&#174;l used    Therefore, measured accumulation QS signaling <br /> the DNA template PCR. Each PCR mixture (total volume, 20 &#226;?&#174;l)            molecules P1 P2 growth TSB. Specifically, we <br /> additionally contained 0.5 &#226;?&#174;l forward reverse 10 mM primer          examined accumulation C4-HSL, 3OC12-HSL, PQS <br /> (see Table S1 supplemental material) 18 &#226;?&#174;l Platinum PCR              intracellularly extracellularly transition station- <br /> high-fidelity SuperMix (Invitrogen). PCR products verified through <br />                                                                                 ary phase (6 h) just visible pigmentation P2. Cell-free <br /> gel electrophoresis 0.8% agar cleaned ExoSAP-IT (Af- <br /> fymetrix) submitted sequencing University of <br />                                                                                 conditioned media cell lysates analyzed QS molecule <br /> Chicago DNA Sequencing &amp; Genotyping Facility (Chicago, IL). Prim-               production using E. coli (27) P. aeruginosa (28) reporter <br /> ers used designed Primer3 software.                                   strains C4-HSL, 3OC12-HSL, PQS activity. The data were <br />     Statistical analysis. Statistical significance determined Stu-   normalized cell density (for extracellular measurements) <br /> dent t test (P &#226;&#172;? 0.05). Standard errors displayed figures.           protein concentration (for intracellular measurements). Re- <br />                                                                                 sults demonstrated 6 h growth, P2 strain showed <br /> RESULTS                                                                         significantly higher intracellular levels C4-HSL PQS and <br /> MPAO1-P2 demonstrates early activation quorum sensing.                       higher extracellular levels C4-HSL 3OC12-HSL (Fig. 1D <br /> We previously shown liquid cultures MPAO1-P2 (P2)                  E). Intracellular 3OC12-HSL extracellular PQS also <br />  <br />  <br /> January 2014 Volume 196 Number 2                                                                                                          jb.asm.org 507 <br />  Luong et al. <br />  <br />  <br />  <br />  <br /> FIG 2 Enhanced virulence P2 low-<span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> medium. (A) Blue pigmentation suggesting higher pyocyanin production P2. (B) Quantitative analysis of <br /> pyocyanin measured 14 h. n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.05. (C) Kaplan-Meier <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> curves C. elegans incubated P2 versus P1 grown 0.1&#226;&#171;&#187; TY. <br />  <br />  <br />  <br /> seemed higher P2, differences did reach              mixed culture. In Fig. 3A C, intensity red fluo- <br /> statistical significance (Fig. 1D E).                                      rescence exhibited red fluorescent MPAO1-P1 strain in <br />     We noticed P2 growth consistently 50 70% higher                pure MPAO1-P1 culture mixed MPAO1- <br /> during early exponential phase P1, strains           P1/P2 culture reflects growth MPAO1-P1 strain. Figure <br /> entered stationary phase cell density (Fig. 1C). The           3A C intensity green fluorescence exhibited by <br /> differences growth likely affected shorter tran-             green fluorescent MPAO1-P2 strain pure <br /> sitioning time P2 lag log phase (see Fig. S1 sup-            MPAO1-P2 culture mixed MPAO1-P1/P2 culture, re- <br /> plemental material). Accelerated growth P2 potentially                flecting growth MPAO1-P2 strain. We performed <br /> contribute faster accumulation QS molecules,                experiment TSB-grown cultures relative <br /> reach threshold concentrations earlier time points activate            intensity red fluorescence mixed culture 60 70% <br /> quorum sensing, evident 8 h.                                      pure MPAO1-P1 culture, suggesting the <br />     To confirm earlier activation QS P2 strains,           growth MPAO1-P1 inhibited mixed cultures. How- <br /> next performed transcriptional analysis genes involved               ever, intensity green fluorescence mixed culture was <br /> biosynthesis quorum-sensing molecules: rhlI (C4-HSL syn-                    similar pure MPAO1-P2 culture (Fig. 3A), suggest- <br /> thase), lasI (3OC12-HSL synthase), pqsA (biosynthesis PQS               ing MPAO1-P1 did suppress MPAO1-P2 growth that <br /> precursor). Results demonstrated significantly higher expression               MPAO1-P2 growth mixed culture equal growth <br /> of rhlI pqsA 6 h P2 (Fig. 1F). The expression lasI                MPAO1-P2 individual cultures. Because MPAO1 <br /> seemed higher P1 P2, statisti-             pure cultures mixed cultures similar cell densities at <br /> cally significant. <br />                                                                                time fluorescence analysis (data shown), plausible <br />     We used <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-limited medium 0.1&#226;&#171;&#187; TY, which <br />                                                                                MPAO1-P2 remains abundance cost MPAO1-P1 <br /> P1 P2 strains equally low growth rates (see Fig. S2 the <br />                                                                                growth inhibition coculture. We examined phe- <br /> supplemental material), barely reaching cell density 0.2 at <br />                                                                                nomenon MPAO1-P2 dominance MPAO1-P1 persists in <br /> OD600, similar previously published data MPAO1 <br />                                                                                context host factors. MPAO1-P1 MPAO1-P2 in- <br /> growth (26). MPAO1 (P1 phenotype) grown medium <br /> never reached point pyocyanin visible. In con-                cubated mixed culture intestinal epithelial mono- <br /> trast, observed P2 displayed visible pyocyanin production              layers HT-29 cells. Fluorescence planktonic adherent <br /> at 12 h (Fig. 2A), suggesting quorum sensing, vir-              bacterial cells examined 5 h growth. In mixed cultures <br /> ulence, activated P2 strain <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-poor con-            compared pure MPAO1-P1 cultures, levels MPAO1-P1 were <br /> ditions. OD520 measurements pyocyanin culture superna-                 &#226;&#172;&#402;80% lower planktonic (Fig. 3B) adherent cells (Fig. <br /> tant confirmed increased pyocyanin levels P2 14 h (Fig. 2B).             3C). The MPAO1-P2 mixed cultures did not <br /> To confirm virulent phenotype P2 strain, per-                 differ significantly pure MPAO1-P2 (Fig. 3B <br /> formed C. elegans mortality assays nematodes                     C). These data suggest MPAO1-P2 greater <br /> transferred liquid cultures P1 P2 grown 0.1&#226;&#171;&#187; TY.                competitiveness MPAO1-P1 present intestinal <br /> The mortality rate caused P2 2-fold higher P1               epithelium. We tested hypothesis MPAO1-P2, with <br /> (Fig. 2C).                                                                     earlier induction quorum sensing, secretes exoproducts that <br />     MPAO1-P2 outcompetes MPAO1-P1 mixed culture.                          suppress growth MPAO1-P1. First, obtained cell-free <br /> Early propagation activation quorum sensing confer                  conditioned media growing MPAO1-P1 MPAO1-P2 in <br /> superior competitiveness strains P2 phenotype. To test             pure cultures 8 h removing cells centrifuga- <br /> this hypothesis, transformed MPAO1-P1 plasmid                      tion filtration. Afterwards, overnight MPAO1-P1 cells were <br /> harboring red fluorescent protein MPAO1-P2 plasmid                  grown MPAO1-P1- MPAO1-P2-conditioned media, <br /> harboring green fluorescent protein measured red green                 cell growth tracked dynamically. The data demonstrated <br /> fluorescence determine relative abundance strain                growth MPAO1-P1 cells MPAO1-P2-conditioned me- <br />  <br />  <br /> 508   jb.asm.org                                                                                                                     Journal Bacteriology <br />                                                                                                                          Emergence P2 Phenotype PAO1 Strains <br />  <br />  <br />  <br />  <br /> FIG 3 MPAO1-P2 outcompetes MPAO1-P1 mixed cultures. (A) Relative fluorescence exhibited MPAO1-P1 (red relative fluorescence units [RFU]) and <br /> MPAO1-P2 (green RFU) 6 h growth TSB separate mixed populations. n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.001. (B C) Relative fluorescence MPAO1-P1 MPAO1-P2 <br /> grown intestinal epithelial monolayers HT-29 cells. (B) Nonadherent bacterial cells (n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.001); (C) adherent bacterial cells (n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.01). (D and <br /> E) MPAO1-P1 growth P1- versus P2-conditioned media. (D) All time periods 0 18 h (n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.05); (E) 4- 8-h time points (n &#226;&#171;&#189; 3; *, P &#226;&#172;? 0.05). <br />  <br />  <br />  <br />  <br /> dium attenuated compared growth MPAO1-P1-condi-                             phenotype strains (C-PAO1, Ig-PAO1, L-PAO1, Ma-PAO1) <br /> tioned media (Fig. 3D E).                                                         P2 phenotype strains (W-PAO1 D-PAO1) demon- <br />     PAO1 strains display P1 P2 phenotype. Increased com-                       strated active transcription mexE, remaining P2 <br /> petitiveness P2 allow emergent P2 cells replace P1                      strains (H-PAO1 R-PAO1) did (Fig. 5A). Sequencing of <br /> population. From experience working P. aeruginosa PAO1                           mexT revealed strains mexE transcription contained <br /> strains various laboratories, noticed pronounced differ-                      mexT sequence identical MPAO1-P1. H-PAO1 and <br /> ences strains terms altered pyocyanin production                    R-PAO1, did transcribe mexE, contained frameshift <br /> solid agar liquid media referred                    mutations mexT. H-PAO1 contained 1-bp deletion (Fig. 5C), <br /> P1- P2-like. As described previously (3), P2 phenotype                      R-PAO1 contained 8-bp insertion (Fig. 5C) identical to <br /> characterized enhanced pyocyanin liquid medium, attenu-                         P. aeruginosa PAO1-UW strain. <br /> ated pyocyanin agarized medium, swarming motility, high col-                           To elucidate controversy P2 phenotype in <br /> lagenase activity, low resistance chloramphenicol. We                          strains W-PAO1 D-PAO1 intact mexT express <br /> assessed 8 PAO1 strains features half                        mexE, decided examine expression mexF these <br /> (W-PAO1, H-PAO1, R-PAO1, D-PAO1) possessed P2                                 strains transcribed operon. <br /> phenotype, half (C-PAO1, Ig-PAO1, L-PAO1,                         qRT-PCR revealed W-PAO1 D-PAO1 expressed <br /> Ma-PAO1) possessed P1 phenotype (Fig. 4).                                         negligible amounts mexF transcript (Fig. 5B) signifi- <br />     PAO1 strains P2 phenotype mutations leading                          cantly lower levels mexE transcript. Sequencing of <br /> dysfunction MexT MexF. We tested hypothesis                       mexF revealed strains harbored identical C-to-G SNP <br /> P. aeruginosa PAO1 strains P2 phenotype loss                          nucleotide 978, causing shift tyrosine premature <br /> MexEF-OprN efflux pump functionality. Due known im-                            stop codon (Fig. 5D). These data demonstrate PAO1 strains <br /> plication MexT switch MPAO1-P1 MPAO1-P2                             displaying P2 phenotype various mutations leading to <br /> (3), measured expression mexE, known posi-                      dysfunction MexEF-OprN efflux pump. <br /> tively regulated MexT. The mexE expression analyzed                             P2 phenotype broadly distributed P. aeruginosa <br /> qRT-PCR cells collected 3.5 h growth TSB. All P1                       MPAO1 mutant strains transposon mutant library. From <br />  <br />  <br /> January 2014 Volume 196 Number 2                                                                                                                         jb.asm.org 509 <br />  Luong et al. <br />  <br />  <br />  <br />  <br /> FIG 4 PAO1 strains exhibit P1 P2 phenotype. (A B) Pyocyanin production TSB agar (A) liquid TSB (B). (C) Swarming motility. (D) <br /> Collagenase activity measured degradation fluorescein labeled gelatin. n &#226;&#171;&#189; 3; P &#226;&#172;? 0.01 P2 strains compared P1 strains. (E) Resistance to <br /> chloramphenicol. Cell density measured overnight cultures grown TSB containing 500 &#226;?&#174;g/ml chloramphenicol. n &#226;&#171;&#189; 3; P &#226;&#172;? 0.01 P2 strains <br /> compared P1 strains. Black bars, P1 strains; gray bars, P2 strains. <br />  <br />  <br /> our experience working P. aeruginosa MPAO1 trans-                    type (see Fig. S3 supplemental material), confirming that <br /> poson mutant library, noticed occasional appearance                 P2 switch phenotype effect dis- <br /> high-pyocyanin-producing mutants explained                  rupted gene. <br /> by disruption gene. Given current knowledge the <br /> emergence P2 phenotype MPAO1 populations, hy-                    DISCUSSION <br /> pothesized pyocyanin production mutants                  PAO1 widely used P. aeruginosa strains sci- <br /> due P2 phenotype. We randomly chose 66 mutants (verified               entific research laboratories globe (32). Here we <br /> by PCR transposon insertion expected gene)               emergence P2 phenotype PAO1 populations <br /> assessed swarming capacity, collagenase activity, resis-             correlates various mutations leading MexEF- <br /> tance chloramphenicol, pyocyanin production. Among                     OprN efflux pump dysfunction global changes PAO1 be- <br /> them, 27 mutants demonstrated P1 P2 phenotype                   havior. <br /> (Table 1) chosen mexE expression analysis (Fig. 6A).                We demonstrated higher intracellular levels C4-HSL and <br /> In analysis, P1 mutants showed transcription mexE,            PQS higher extracellular levels C4-HSL 3OC12-HSL in <br /> while 24 25 P2 mutants did not. The pvdS mutant (mutant                P2 P1. P2 shows increased activation Rhl quorum- <br /> 27) demonstrated active transcription mexE despite P2 phe-             sensing subsystem, suggest dysfunction the <br /> notype. We examined mexF transcription PvdS mutant                MexEF-OprN pump mainly affects Rhl/C4-HSL quorum- <br /> and negligible mexF (Fig. 6B). Sequencing               sensing pathway P. aeruginosa. Although MexEF-OprN ef- <br /> PvdS mutant mexF revealed 17-bp insertion leading frame-               flux pump extensively characterized context of <br /> shift <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span> downstream premature stop codon (Fig. 6C).               antibiotics, suggested main purpose the <br />     To confirm phenotype switch P2             pump actually antimicrobial extrusion (33) may <br /> transposon insertion mutant gene, randomly chose                metabolic function (34). For example, isolates P. <br /> mutant P2 phenotype, &#226;&#338;&#172;hxcT mutant (no. 23 Table               aeruginosa mouse model pneumonia express the <br /> 1), complemented mutant functional hxcT gene.                 MexEF-OprN pump exposure antimicrobials (35). <br /> The complemented &#226;&#338;&#172;hxcT/hxcT mutant retained P2 pheno-                     There increasing evidence MexEF-OprN pump func- <br />  <br />  <br /> 510   jb.asm.org                                                                                                                    Journal Bacteriology <br />                                                                                                         Emergence P2 Phenotype PAO1 Strains <br />  <br />  <br />  <br />  <br /> FIG 5 PAO1 strains P2 phenotype mutations leading dysfunction MexT MexF. (A B) Expression mexE (A) mexF (B) <br /> determined qRT-PCR. n &#226;&#171;&#189; 3/group. (C) Sequence mexT H-PAO1 R-PAO1, demonstrating frameshift mutations. (D) Sequence mexF W-PAO1 <br /> and D-PAO1, demonstrating SNP leading premature stop codon. <br />  <br />  <br />  <br /> tions extrude quorum-sensing molecules. For example,               ments remains clarified. The increased competitiveness P2 <br /> MexEF-OprN efflux pump export HHQ (4-hydroxy-2-hep-                   attributed earlier activation quorum sensing ac- <br /> tylquinoline) (17), precursor PQS, leads decreased          celerated growth early exponential phase. The finding that <br /> intracellular PQS synthesis. It suggested 3OC12-       P2 outcompete P1 led assess prevalence P2 <br /> HSL secreted MexEF-OprN pump cultures               phenotype PAO1 strains, including MPAO1 <br /> very high cell densities (optical density, &#226;&#172;&#381;4) (5). Intriguingly,     transposon library (parental strain MPAO1 P1 phenotype) <br /> same study strains expressing MexEF-OprN pump              (18). Our results demonstrated that, group, P2 strains <br /> have &#226;&#172;&#402;60% lower extracellular concentration C4-HSL. In              harbor various mutations mexT mexF, leading dysfunc- <br /> contrast, suggested MexEF-OprN            tion MexEF-OprN. <br /> be involved export C4-HSL cell (16). Consistent            Additionally, P2 phenotype strains quite stable over <br /> with others&#226;&#8364;&#8482; findings (5, 16), did observe intracellular accumu-       time. For example, W-PAO1 D-PAO1 obtained the <br /> lation QS molecules MPAO1-P2 involved quo-             laboratory gap 10 years. The fact that <br /> rum sensing, known regulate pyocyanin production.             contained SNP mexF demonstrates stability <br /> The transcriptional response demonstrated expression Rhl           switch P2 phenotype. We demonstrated <br /> subsystem P2, significant difference lasI      high stability P2 phenotype strain MPAO1-P2, was <br /> and pqsA expression. It speculated compet-         generated rat model anastomotic leakage (3). The stabil- <br /> itive binding C4-HSL 3OC12-HSL LasR,                ity P2 phenotype appearance multiple contexts <br /> may offset increase LasI overactivation predicted           suggest certain environmental conditions favor emer- <br /> intracellular accumulation 3OC12-HSL. RhlR shown              gence. <br /> to inhibit expression pqsA (36) account           The emergence P2 phenotype transposon <br /> discordance expected increase pqsA expression P2          mutant library, constructed parental MPAO1 <br /> the transition stationary phase. P2 cells shift to-      strain P1 phenotype, important implications. <br /> ward increased Rhl/C4-HSL subtype.                                     Other investigators reported unexpected increase pyo- <br />     Precisely P2 outcompetes P1 <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-rich environ-             cyanin production universal hypersusceptibility mutants <br />  <br />  <br /> January 2014 Volume 196 Number 2                                                                                                     jb.asm.org 511 <br />  Luong et al. <br />  <br />  <br />  <br />  <br /> FIG 6 MPAO1 mutant strains P2 phenotype dysfunction MexEF. (A B) Expression mexE (A) mexF (B) determined qRT-PCR. (C) <br /> Sequence mexF &#226;&#338;&#172;pvdS strain (no. 27 Table 1), demonstrating 17-<span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span>-pair insertion. <br />  <br />  <br />  <br /> to antimicrobials seen broad range functionally unre-                      aeruginosa enhances virulence: possible role anastomotic leak. <br /> lated mutants library (37). These reports did pro-                     PLoS One 7:e44326. http://dx.doi.org/10.1371/journal.pone.0044326. <br />                                                                                   4.   Cosson P, Zulianello L, Join-Lambert O, Faurisson F, Gebbie L, Beng- <br /> vide mechanism phenomenon suggested                          hezal M, Van Delden C, Curty LK, Kohler T. 2002. Pseudomonas <br /> mutants appeared result unknown gener-                         aeruginosa virulence analyzed Dictyostelium discoideum host system. <br /> alizable effect library. Because P2 phenotype dis-                      J. Bacteriol. 184:3027&#226;&#8364;&#8220;3033. http://dx.doi.org/10.1128/JB.184.11.3027 <br /> plays significant phenotypic changes compared P1                           -3033.2002. <br /> phenotype independent disrupted gene given mutant,                 5.   Kohler T, van Delden C, Curty LK, Hamzehpour MM, Pechere JC. <br />                                                                                        2001. Overexpression MexEF-OprN multidrug efflux affects <br /> could argued P2 phenotype mask effect                       cell-to-cell signaling Pseudomonas aeruginosa. J. Bacteriol. 183:5213&#226;&#8364;&#8220; <br /> known mutation. Finally finding strains P. aeruginosa                      5222. http://dx.doi.org/10.1128/JB.183.18.5213-5222.2001. <br /> PAO1 various laboratories express P1 P2                    6.   Maseda H, Saito K, Nakajima A, Nakae T. 2000. Variation mexT <br /> phenotype complicate comparisons studies. There-                           gene, regulator MexEF-oprN efflux pump expression wild-type <br /> fore, advisable future studies involving use                       strains Pseudomonas aeruginosa. FEMS Microbiol. Lett. 192:107&#226;&#8364;&#8220;112. <br />                                                                                        http://dx.doi.org/10.1111/j.1574-6968.2000.tb09367.x. <br /> PAO1 strains identified terms expression                 7.   Fukuda H, Hosaka M, Hirai K, Iyobe S. 1990. New norfloxacin resis- <br /> of P1 P2 phenotype.                                                             tance gene Pseudomonas aeruginosa PAO. Antimicrob. Agents Che- <br />                                                                                        mother. 34:1757&#226;&#8364;&#8220;1761. http://dx.doi.org/10.1128/AAC.34.9.1757. <br /> ACKNOWLEDGMENTS                                                                   8.   Tian ZX, Mac Aogain M, O&#226;&#8364;&#8482;Connor HF, Fargier E, Mooij MJ, Adams <br />                                                                                        C, Wang YP, O&#226;&#8364;&#8482;Gara F. 2009. MexT modulates virulence determinants in <br /> This study funded Biological Sciences Collegiate Division Re-               Pseudomonas aeruginosa independent MexEF-OprN efflux pump. <br /> search Endowments University Chicago (P.M.L.) NIH RO1                    Microb. Pathog. 47:237&#226;&#8364;&#8220;241. http://dx.doi.org/10.1016/j.micpath.2009 <br /> 5R01GMO62344 (J.C.A.).                                                                 .08.003. <br />     MPAO1-P1/RFP constructed collaboration laboratory             9.   Zaoui C, Overhage J, Lons D, Zimmermann A, Musken M, Bielecki P, <br /> of Deborah Nelson, Department Neurobiology, Pharmacology &amp; Phys-                    Pustelny C, Becker T, Nimtz M, Haussler S. 2012. An orphan sensor <br /> iology Lab, University Chicago.                                                     kinase controls quinolone signal production MexT Pseudomonas <br />                                                                                        aeruginosa. Mol. Microbiol. 83:536 &#226;&#8364;&#8220;547. http://dx.doi.org/10.1111/j <br />                                                                                        .1365-2958.2011.07947.x. <br /> REFERENCES                                                                       10.   Jin Y, Yang H, Qiao M, Jin S. 2011. MexT regulates type III secretion <br />  1. Ciofu O, Mandsberg LF, Bjarnsholt T, Wassermann T, Hoiby N. 2010.                  MexS PtrC Pseudomonas aeruginosa. J. Bacteriol. <br />     Genetic adaptation Pseudomonas aeruginosa chronic lung in-               193:399 &#226;&#8364;&#8220; 410. http://dx.doi.org/10.1128/JB.01079-10. <br />     fection patients cystic fibrosis: strong weak mutators      11.   Klockgether J, Munder A, Neugebauer J, Davenport CF, Stanke F, <br />     heterogeneous genetic backgrounds emerge mucA and/or lasR mu-                   Larbig KD, Heeb S, Schock U, Pohl TM, Wiehlmann L, Tummler B. <br />     tants. Microbiology 156:1108 &#226;&#8364;&#8220;1119. http://dx.doi.org/10.1099/mic.0                2010. Genome diversity Pseudomonas aeruginosa PAO1 laboratory <br />     .033993-0.                                                                         strains. J. Bacteriol. 192:1113&#226;&#8364;&#8220;1121. http://dx.doi.org/10.1128/JB <br />  2. Kalia A, Bessen DE. 2004. Natural selection evolution streptococ-           .01515-09. <br />     cal virulence genes involved tissue-specific adaptations. J. Bacteriol.   12.   Kohler T, Michea-Hamzehpour M, Henze U, Gotoh N, Curty LK, <br />     186:110 &#226;&#8364;&#8220;121. http://dx.doi.org/10.1128/JB.186.1.110-121.2004.                     Pechere JC. 1997. Characterization MexE-MexF-OprN, positively <br />  3. Olivas AD, Shogan BD, Valuckaite V, Zaborin A, Belogortseva N,                     regulated multidrug efflux Pseudomonas aeruginosa. Mol. <br />     Musch M, Meyer F, Trimble WL, An G, Gilbert J, Zaborina O, Alverdy                 Microbiol. 23:345&#226;&#8364;&#8220;354. http://dx.doi.org/10.1046/j.1365-2958.1997 <br />     JC. 2012. Intestinal tissues induce SNP mutation Pseudomonas                 .2281594.x. <br />  <br />  <br />  <br /> 512   jb.asm.org                                                                                                                           Journal Bacteriology <br />                                                                                                                  Emergence P2 Phenotype PAO1 Strains <br />  <br />  <br />  <br /> 13. Tin S, Sakharkar KR, Lim CS, Sakharkar MK. 2009. Activity chitosans       27. Winson MK, Swift S, Fish L, Throup JP, Jorgensen F, Chhabra SR, <br />     combination antibiotics Pseudomonas aeruginosa. Int. J. Biol.         Bycroft BW, Williams P, Stewart GS. 1998. Construction analysis of <br />     Sci. 5:153&#226;&#8364;&#8220;160. http://dx.doi.org/10.3923/ijb.2009.153.159.                      luxCDABE-based plasmid sensors investigating N-acyl homoserine <br /> 14. Llanes C, Kohler T, Patry I, Dehecq B, van Delden C, Plesiat P. 2011.            lactone-mediated quorum sensing. FEMS Microbiol. Lett. 163:185&#226;&#8364;&#8220;192. <br />     Role MexEF-OprN efflux low-level resistance Pseu-            http://dx.doi.org/10.1111/j.1574-6968.1998.tb13044.x. <br />     domonas aeruginosa ciprofloxacin. Antimicrob. Agents Chemother.           28. Diggle SP, Matthijs S, Wright VJ, Fletcher MP, Chhabra SR, Lamont IL, <br />     55:5676 &#226;&#8364;&#8220;5684. http://dx.doi.org/10.1128/AAC.00101-11.                           Kong X, Hider RC, Cornelis P, Camara M, Williams P. 2007. The <br /> 15. Maseda H, Yoneyama H, Nakae T. 2000. Assignment substrate-                Pseudomonas aeruginosa 4-quinolone signal molecules HHQ PQS <br />     selective subunits MexEF-OprN multidrug efflux pump Pseu-              play multifunctional roles quorum sensing iron entrapment. <br />     domonas aeruginosa. Antimicrob. Agents Chemother. 44:658 &#226;&#8364;&#8220; 664. http:            Chem. Biol. 14:87&#226;&#8364;&#8220;96. http://dx.doi.org/10.1016/j.chembiol.2006.11.014. <br />     //dx.doi.org/10.1128/AAC.44.3.658-664.2000.                                  29. Zaborina O, Lepine F, Xiao G, Valuckaite V, Chen Y, Li T, Ciancio M, <br /> 16. Maseda H, Sawada I, Saito K, Uchiyama H, Nakae T, Nomura N. 2004.                Zaborin A, Petrof EO, Turner JR, Rahme LG, Chang E, Alverdy JC. <br />     Enhancement mexAB-oprM efflux pump expression quorum- <br />                                                                                      2007. Dynorphin activates quorum sensing quinolone signaling Pseu- <br />     sensing autoinducer cancellation regulator, MexT, the <br />                                                                                      domonas aeruginosa. PLoS Pathog. 3:e35. http://dx.doi.org/10.1371 <br />     mexEF-oprN efflux pump operon Pseudomonas aeruginosa. Antimi- <br />                                                                                      /journal.ppat.0030035. <br />     crob. Agents Chemother. 48:1320 &#226;&#8364;&#8220;1328. http://dx.doi.org/10.1128/AAC <br />     .48.4.1320-1328.2004.                                                        30. Zaborin A, Romanowski K, Gerdes S, Holbrook C, Lepine F, Long J, <br /> 17. Lamarche MG, Deziel E. 2011. MexEF-OprN efflux pump exports                  Poroyko V, Diggle SP, Wilke A, Righetti K, Morozova I, Babrowski T, <br />     Pseudomonas quinolone signal (PQS) precursor HHQ (4-hydroxy-2-                   Liu DC, Zaborina O, Alverdy JC. 2009. Red death Caenorhabditis <br />     heptylquinoline). PLoS One 6:e24310. http://dx.doi.org/10.1371/journal           elegans caused Pseudomonas aeruginosa PAO1. Proc. Natl. Acad. Sci. <br />     .pone.0024310.                                                                   U. S. A. 106:6327&#226;&#8364;&#8220; 6332. http://dx.doi.org/10.1073/pnas.0813199106. <br /> 18. Jacobs MA, Alwood A, Thaipisuttikul I, Spencer D, Haugen E, Ernst S,         31. Latifi A, Winson MK, Foglino M, Bycroft BW, Stewart GS, Lazdunski <br />     Will O, Kaul R, Raymond C, Levy R, Chun-Rong L, Guenthner D,                     A, Williams P. 1995. Multiple homologues LuxR LuxI control <br />     Bovee D, Olson MV, Manoil C. 2003. Comprehensive transposon mu-                  expression virulence determinants secondary metabolites through <br />     tant library Pseudomonas aeruginosa. Proc. Natl. Acad. Sci. U. S. A.          quorum sensing Pseudomonas aeruginosa PAO1. Mol. Microbiol. 17: <br />     100:14339 &#226;&#8364;&#8220;14344. http://dx.doi.org/10.1073/pnas.2036282100.                     333&#226;&#8364;&#8220;343. http://dx.doi.org/10.1111/j.1365-2958.1995.mmi_17020333.x. <br /> 19. Strack RL, Strongin DE, Bhattacharyya D, Tao W, Berman A,                    32. Carilla-Latorre S, Calvo-Garrido J, Bloomfield G, Skelton J, Kay RR, <br />     Broxmeyer HE, Keenan RJ, Glick BS. 2008. A noncytotoxic DsRed                    Ivens A, Martinez JL, Escalante R. 2008. Dictyostelium transcriptional <br />     variant whole-cell labeling. Nat. Methods 5:955&#226;&#8364;&#8220;957. http://dx.doi.org       responses Pseudomonas aeruginosa: common specific effects from <br />     /10.1038/nmeth.1264.                                                             PAO1 PA14 strains. BMC Microbiol. 8:109. http://dx.doi.org/10.1186 <br /> 20. Olsen RH, DeBusscher G, McCombie WR. 1982. Development broad-                 /1471-2180-8-109. <br />     host-range vectors gene banks: self-cloning Pseudomonas           33. Fetar H, Gilmour C, Klinoski R, Daigle DM, Dean CR, Poole K. 2011. <br />     aeruginosa PAO chromosome. J. Bacteriol. 150:60 &#226;&#8364;&#8220; 69.                            mexEF-oprN multidrug efflux operon Pseudomonas aeruginosa: regu- <br /> 21. Shrout JD, Chopp DL, Just CL, Hentzer M, Givskov M, Parsek MR.                   lation MexT activator response nitrosative stress chlor- <br />     2006. The impact quorum sensing swarming motility Pseu-                amphenicol. Antimicrob. Agents Chemother. 55:508 &#226;&#8364;&#8220;514. http://dx.doi <br />     domonas aeruginosa biofilm formation nutritionally conditional. Mol.          .org/10.1128/AAC.00830-10. <br />     Microbiol. 62:1264 &#226;&#8364;&#8220;1277. http://dx.doi.org/10.1111/j.1365-2958.2006         34. Kumar A, Schweizer HP. 2011. Evidence MexT-independent overex- <br />     .05421.x.                                                                        pression MexEF-OprN multidrug efflux pump Pseudomonas aerugi- <br /> 22. Choi KH, Schweizer HP. 2006. mini-Tn7 insertion bacteria single          nosa presence metabolic stress. PLoS One 6:e26520. http://dx.doi.org <br />     attTn7 sites: example Pseudomonas aeruginosa. Nat. Protoc. 1:153&#226;&#8364;&#8220;161. <br />                                                                                      /10.1371/journal.pone.0026520. <br />     http://dx.doi.org/10.1038/nprot.2006.24. <br />                                                                                  35. Join-Lambert OF, Michea-Hamzehpour M, Kohler T, Chau F, Fauris- <br /> 23. Koch B, Jensen LE, Nybroe O. 2001. A panel Tn7-based vectors for <br />                                                                                      son F, Dautrey S, Vissuzaine C, Carbon C, Pechere J. 2001. Differential <br />     insertion gfp marker gene delivery cloned DNA Gram- <br />     negative bacteria neutral chromosomal site. J. Microbiol. Methods           selection multidrug efflux mutants trovafloxacin ciprofloxacin <br />     45:187&#226;&#8364;&#8220;195. http://dx.doi.org/10.1016/S0167-7012(01)00246-9.                     experimental model Pseudomonas aeruginosa acute pneumonia <br /> 24. Tremblay J, Deziel E. 2008. Improving reproducibility Pseudomo-           rats. Antimicrob. Agents Chemother. 45:571&#226;&#8364;&#8220;576. http://dx.doi.org/10 <br />     nas aeruginosa swarming motility assays. J. Basic Microbiol. 48:509 &#226;&#8364;&#8220;515.        .1128/AAC.45.2.571-576.2001. <br />     http://dx.doi.org/10.1002/jobm.200800030.                                    36. McGrath S, Wade DS, Pesci EC. 2004. Dueling quorum sensing systems <br /> 25. Stie <br /> </body></html>