<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>DAF-16 D9 Desaturase Genes Promote Cold <br /> Tolerance Long-Lived Caenorhabditis elegans age-1 <br /> Mutants <br /> Fiona R. Savory, Steven M. Sait, Ian A. Hope* <br /> Institute Integrative Comparative Biology, Faculty Biological Sciences, The University Leeds, Leeds, United Kingdom <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      In Caenorhabditis elegans, mutants conserved insulin/IGF-1 signalling (IIS) pathway long-lived stress resistant <br />      altered expression DAF-16 target genes involved cellular defence metabolism. The three <br />      D9 desaturase genes, fat-5, fat-6 fat-7, included DAF-16 targets, established D9 <br />      desaturase enzymes play important role <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span> low temperatures. However, assessment cold tolerance has <br />      previously reported IIS mutants. We demonstrate long-lived age-1(hx546) mutants remarkably resilient <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />      low temperature stress relative wild type worms, dependent daf-16. We cold <br />      tolerance following direct transfer low temperatures increased wild type worms facultative, daf-16 <br />      dependent, dauer stage. Although cold tolerant <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>phenotype</span> age-1(hx546) mutants predominantly D9 <br />      desaturase genes, additional transcriptional targets DAF-16 involved. Surprisingly, survival wild type adults <br />      following rapid temperature decline dependent functional daf-16, cellular distributions DAF-16::GFP <br />      fusion protein indicate DAF-16 activated low temperature stress. This suggests cold-induced <br />      physiological defences specifically regulated IIS pathway DAF-16, expression DAF-16 target genes <br />      IIS mutants dauers sufficient promote cross tolerance low temperatures addition forms stress. <br />  <br />   Citation: Savory FR, Sait SM, Hope IA (2011) DAF-16 D9 Desaturase Genes Promote Cold Tolerance Long-Lived Caenorhabditis elegans age-1 Mutants. PLoS <br />   ONE 6(9): e24550. doi:10.1371/journal.pone.0024550 <br />   Editor: Peter Csermely, Semmelweis University, Hungary <br />   Received June 21, 2011; Accepted August 12, 2011; Published September 8, 2011 <br />   Copyright: &#195;&#376; 2011 Savory et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported White Rose University Consortium studentship. The funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: i.a.hope@leeds.ac.uk <br />  <br />  <br />  <br /> Introduction                                                                          linositol 3-kinase (PI3K) catalytic subunit homologue, long- <br />                                                                                       lived display increased resistance range environmental <br />    Genetic modifications promote longevity associated                       challenges, including oxidative stress [4,15], heat shock [5], <br /> with increased resistance environmental stress model                            ultraviolet light [16], heavy metals [17], hypoxia [18], microbial <br /> organisms ranging yeast mice. These longevity                             infections [19], hypertonic stress [20]. <br /> stress-resistant phenotypes induced disrupting conserved                       Although increased activity DAF-16 IIS mutants <br /> signal transduction pathways respond changes <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>                     appears induce physiological response promoting cross- <br /> availability and/or environmental conditions (reviewed                       tolerance multiple forms stress, forms stress not <br /> [1]). Such responses mediated physiological shift                    associated IIS pathway. Artificial selection for <br /> protection repair somatic molecules cells. The                        increased longevity promotes cold tolerance D. melanogaster <br /> extensively characterised pathways conserved                          [21,22], resistance low temperatures, an <br /> insulin/IGF-1 signalling (IIS) pathway, plays important                      important survival trait, reported IIS mutant. <br /> role determination longevity stress resistance                       In fact, D. melanogaster, ablation neurosecretory cells, which <br /> Caenorhabditis elegans [2,3,4,5], Drosophila melanogaster [6,7] mice,             produce insulin-like ligands required activation IIS <br /> Mus musculus [8], influence lifespan humans [9,10].                   pathway, increases lifespan, delays cold shock recovery [23]. <br />    In C. elegans, IIS pathway modulates development,                              Although ablated animals display enhanced resistance to <br /> metabolism, stress resistance longevity, part,                     starvation oxidative stress, reduced insulin/IGF-1 signalling <br /> regulating activity FOXO transcription factor DAF-16.                      actually increase sensitivity low temperatures flies. <br /> The IIS pathway negatively regulates DAF-16 activity                                  Whilst molecular chaperones (heat-shock proteins) known to <br /> favourable growth conditions [11,12]. However, insulin/                          play key role preventing repairing damage induced by <br /> IGF-1 signalling disrupted low food availability exposure                    high temperatures [24], preservation cell membrane fluidity <br /> to certain forms stress, DAF-16 accumulates nuclei,                         thought important cold tolerance poikilotherms. At <br /> binding activating promoters genes involved                          physiological temperatures organisms adapted <br /> cellular defence metabolism [11,12,13,14]. Mutants                      acclimated, membrane lipids maintained fluid liquid- <br /> defective components IIS pathway DAF-2,                        crystalline phase. However, temperatures drop a <br /> insulin/IGF-1 receptor homologue, AGE-1, phosphatidy-                          threshold level, lipid structure changes ordered, rigid gel <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                1                           September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                  Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br /> phase impairs vital membrane functions [25]. To promote               considerable overlap biochemical activity, loss-of-function <br /> survival low temperatures, poikilotherms reduce average          mutations fat-6 fat-7 induce compensatory responses <br /> temperature transition occurs increasing               expression remaining D9 desaturase genes, and <br /> proportion unsaturated fatty acids membrane phospholipids             mutants defective single D9 desaturase genes display no <br /> [26]. This response partially mediated activity       obvious phenotype [41]. Therefore, assess extent which <br /> D9 desaturase enzymes [27,28]. D9 desaturase genes                D9 desaturase enzymes contribute cold tolerance age-1(hx546) <br /> implicated cold tolerance bacteria [29], plants [30],             mutants, expression multiple D9 desaturase genes was <br /> poikilothermic animals [31,32,33].                                          simultaneously eliminated suppressed using combination of <br />    At favourable growth temperatures, expression               loss-of-function mutations <span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi). <br /> C. elegans D9 desaturase genes fat-5, fat-6 fat-7 regulated          The fat-6(tm331); fat-7(wa36) double mutant dramatically <br /> the IIS pathway DAF-16 [14], fat-6 identified           reduced survival times 4uC wild type (z = 27.79, <br /> a direct target DAF-16 [34]. All genes up-regulated         p,0.001) age-1(hx546) mutant (z = 214.5, p,0.001) back- <br /> IIS mutants facultative, daf-16-dependent, dauer             grounds relative respective controls (Figure 2A). Whilst mean <br /> stage [14,35,36,37]. Consequently, membrane fluidity              survival times reduced 40% wild-type background, <br /> maintained lower temperatures non-dauer IIS mutants              mean survival reduced 58% age-1(hx546) <br /> in wild type higher proportion unsaturated fatty acids          mutant background, suggesting fat-6 fat-7 play an <br /> in membrane phospholipids. This enhance survival IIS                 important role increased cold tolerance long-lived <br /> mutants following rapid temperature decline. In study             mutants. The age-1(hx546); fat-6(tm331); fat-7(wa36) triple mutants <br /> compared survival long-lived age-1(hx546) mutant wild type           moderately sensitive low temperatures wild type <br /> C. elegans low temperatures, assessed contribution D9         worms (z = 22.53, p = 0.012), mean survival times reduced <br /> desaturase genes cold tolerance genotype.                        14%. However, fat-6(tm331); fat-7(wa36) individuals <br />                                                                             age-1 mutant background cold tolerant than <br /> Results                                                                     wild type background (z = 5.27, p,0.001), mean <br />                                                                             survival times remaining approximately 43% greater. This <br /> Reduced insulin/IGF-1 signalling promotes cold tolerance                    suggests changes induced age-1(hx546) mutant allele <br /> in daf-16 dependent manner                                                promote cold tolerance involve gene genes, in <br />    To establish insulin/IGF-1 signalling plays role cold            addition fat-6 fat-7. <br /> tolerance, survival times compared young wild type                Relative wild type controls, RNAi fat-5 fat-6(tm331) <br /> (N2) age-1(hx546) mutant adults following direct transfer          mutants reduced mean survival times 4uC (z = 23.48, p,0.001) <br /> 20uC 4uC60.5uC. This substantially thermal range            24%, RNAi fat-5 fat-7(wa36) mutants reduced <br /> for wild type growth activity 15uC&#226;&#8364;&#8220;25uC [38], lipid          mean survival times 4uC (z = 24.84, p,0.001) 30% <br /> phase transition temperature approximately 10uC [39]. Survival           (Figure 2B). Similarly, RNAi fat-5 reduced mean survival times <br /> was considerably prolonged age-1(hx546) mutants relative              age-1(hx546); fat-6(tm331) double mutants (z = 26.67, p,0.001) <br /> wild type adults conditions (z = 12.02, p,0.001),          approximately 33%, age-1(hx546); fat-7(wa36) double <br /> mean survival times increased approximately 85% (Figure 1A).             mutants (z = 27.81, p,0.001) 38%, relative age-1(hx546) <br /> This indicates long-lived worms display increased                mutant controls (Figure 2B). However, single fat-6(tm331) or <br /> resistance low temperatures addition forms stress.        fat-7(wa36) desaturase mutants age-1(hx546) mutant <br /> To assess discrepancy cold tolerance              background subjected fat-5 RNAi remained considerably more <br /> genotypes mediated differences activity DAF-16,            cold tolerant equivalent individuals wild type background <br /> survival times monitored daf-16(mu86) <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>             (fat-6(tm331) mutants: z = 7.12, p,0.001, fat-7(wa36) mutants: <br /> and age-1(hx546); daf-16(mu86) double mutants following direct           z = 7.49, p,0.001). In cases mean survival times remained <br /> transfer 20uC 4uC60.5uC. No significant differences              approximately 52% greater age-1(hx546) mutant back- <br /> survival observed wild type, daf-16(mu86) mutants                ground. <br /> and age-1(hx546); daf-16(mu86) double mutants                      Simultaneous mutations D9 desaturase genes cause <br /> conditions (Figure 1A). This suggests daf-16 required           embryonic lethality [41]. Therefore, obtain individuals which <br /> increased cold tolerance age-1(hx546) mutants, does              expression D9 desaturase genes reduced <br /> contribute survival wild type adults low temperatures.             eliminated, RNAi used different strategies. First, fat- <br />    Dauer formation partially regulated IIS pathway         5 expression reduced fat-6(tm331); fat-7(wa36) double <br /> dependent daf-16 [3]. As D9 desaturase genes up-                   mutants age-1(hx546); fat-6(tm331); fat-7(wa36) triple <br /> regulated wild type dauers, examined resistance low           mutants. Second, fat-6 fat-7 expression reduced fat- <br /> temperature stress stage. Over 90% dauers survived           5(tm420) mutants age-1(hx546); fat-5(tm420) double <br /> at 4uC60.5uC fed wild type age-1(hx546) mutant          mutants. As fat-6 fat-7 approximately 84% nucleotide <br /> adults died (Figure 1B). This indicates dauers                 homology, expression genes suppressed either <br /> remarkably resilient cold temperatures, stresses,            targeted RNAi [41]. <br /> relative wild type adults (z = 13.49, p,0.001), cold               When function D9 desaturase genes reduced <br /> tolerance age-1(hx546) mutant adults lower wild type          eliminated, survival times 4uC reduced dramatically in <br /> dauers conditions (z = 8.85 p,0.001).                           wild type age-1(hx546) mutant backgrounds relative <br />                                                                             controls (Figure 2C). Compared wild type controls, RNAi of <br /> Cold tolerance age-1(hx546) mutants facilitated                    fat-5 fat-6(tm331); fat-7(wa36) double mutants reduced mean <br /> D9 desaturases                                                              survival times (z = 210.60, p,0.001) 67%, RNAi fat-6 <br />   The C. elegans D9 desaturase enzymes exhibit slight differences        fat-7 fat-5(tm420) mutants reduced mean survival times <br /> substrate specificity, FAT-5 primarily converting palmitic             (z = 28.23, p,0.001) approximately 58%. Relative age-1 <br /> acid palmitoleic acid, whilst FAT-6 FAT-7 primarily                  mutant controls, RNAi fat-5 age-1(hx546); fat-6(tm331); fat- <br /> convert stearic acid oleic acid [40]. Despite this,               7(wa36) triple mutants reduced mean survival times (z = 211.40, <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                       September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                 Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br />  <br />  <br /> Figure 1. Survival curves following direct transfer 206C 46C. Survival curves 4uC A) wild type (N2), age-1(hx546) mutants, daf- <br /> 16(mu86) mutants age-1(hx546); daf-16(mu86) double mutants, B) wild type (N2) age-1(hx546) mutant adults wild type (N2) dauers <br /> (n = 90&#226;&#8364;&#8220;100 genotype stage). <br /> doi:10.1371/journal.pone.0024550.g001 <br />  <br /> p,0.001) 69%, RNAi fat-6 fat-7 age-                approximately 75%. Similarly, mean survival times remained <br /> 1(hx546); fat-5(tm420) double mutants reduced mean survival                greater age-1(hx546); fat-5(tm420) double mutants fat-6/fat-7 <br /> times (z = 29.58, p,0.001) 62%. In genetic backgrounds,            RNAi plates fat-5(tm420) mutants fat-6/fat-7 RNAi <br /> fat-6(tm331); fat-7(wa36) mutants subjected fat-5 RNAi appeared         plates (z = 4.92, p,0.001) 66%. These results suggest that <br /> more sensitive cold fat-5(tm420) mutants subjected fat-6/       D9 desaturase genes play important role cold tolerant <br /> fat-7 RNAi. However, differences statistically              phenotype age-1(hx546) mutants, indicate additional <br /> significant and, real, simply reflect reduced efficiency       genes involved. <br /> of RNAi relative loss-of-function mutations. Importantly, <br /> individuals age-1(hx546) mutant background remained                D9 desaturase enzymes promote rapid recovery <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> more cold tolerant equivalent individuals wild type            cold shock <br /> background expression D9 desaturase                &#226;&#8364;&#732;Cold coma&#226;&#8364;&#8482; recovery times used characterise <br /> genes knocked knocked down. When fat-5                     variability cold tolerance different D. melanogaster strains <br /> expression suppressed RNAi age-1(hx546); fat-6(tm331);           [23,42]. To determine approach used assess <br /> fat-7(wa36) triple mutants, mean survival times higher        differences cold sensitivity C. elegans genotypes, we <br /> fat-6(tm331); fat-7(wa36) double mutants (z = 5.35, p,0.001)            compared recovery times room temperature (,22uC), following <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       3                       September 2011 | Volume 6 | Issue 9 | e24550 <br />                                   Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org   4   September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                    Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br />  <br /> Figure 2. Contribution D9 desaturase genes survival following direct transfer 206C 46C. Survival curves 4uC A) wild <br /> type (N2), age-1(hx546) mutants, fat-6(tm331); fat-7(wa36) double mutants age-1(hx546); fat-6(tm331); fat-7(wa36) triple mutants (n = 90&#226;&#8364;&#8220;100 per <br /> genotype), B) wild type (N2) age-1(hx546) mutant controls vector HT115 bacteria (N2(e) age-1(e)), fat-6(tm331) mutants, fat- <br /> 7(wa36) mutants, age-1(hx546); fat-6(tm331) double mutants age-1(hx546); fat-7(wa36) double mutants fat-5 RNAi bacteria (n = 50&#226;&#8364;&#8220;60 per <br /> genotype vector HT115 bacteria, n = 90&#226;&#8364;&#8220;100 genotype fat-5 RNAi bacteria), C) wild type (N2) age-1(hx546) mutant controls on <br /> empty vector HT115 bacteria, fat-6(tm331); fat-7(wa36) double mutants age-1(hx546); fat-6(tm331); fat-7(wa36) triple mutants fat-5 RNAi <br /> bacteria, fat-5(tm420) mutants age-1(hx546); fat-5(tm420) double mutants fat-6/fat-7 RNAi bacteria (n = 50&#226;&#8364;&#8220;60 individuals genotype/ <br /> treatment). <br /> doi:10.1371/journal.pone.0024550.g002 <br />  <br />  <br />  <br /> 6 hours exposure 4uC60.5uC, representative set                 use cold coma recovery assays assess variation cold <br /> strains; wild type (N2), age-1(hx546) mutants, fat-6(tm331); fat-            tolerance C. elegans genotypes. <br /> 7(wa36) double mutants age-1(hx546); fat-6(tm331); fat-7(wa36) <br /> triple mutants. Considerable variation cold coma recovery times           DAF-16 activated exposure low <br /> was observed different genotypes (Figure 3). Although              temperatures <br /> age-1(hx546) mutants resumed locomotion rapidly wild                   As functional daf-16 does appear required cold <br /> type worms removal low temperature stress                     tolerance wild type adults, asked specifically DAF-16 <br /> (t = 7.18, p,0.001), mean recovery times reduced              transcription factor activated low temperatures. Cellular <br /> 27%, differences recovery times genotypes                 distributions DAF-16::GFP fusion protein categorised a <br /> were apparent differences survival following prolonged          scale ranging 1 (unlocalised), representing minimal DAF-16 <br /> periods 4uC. However, fat-6(tm331); fat-7(wa36) double                activity, 4 (fully nuclear localised), representing maximal DAF-16 <br /> mutant substantially delayed recovery times wild type             activity (Figure 4A). Distributions scored young adults <br /> (t = 212.37, p,0.001) age-1(hx546) mutant (t = 212.65,                   following 6 hours exposure 4uC, compared controls <br /> p,0.001) backgrounds relative respective controls. Mean                   maintained 20uC. The 6 hour stress period considered a <br /> recovery times prolonged 74% wild type                        sufficient duration 20-fold increase fat-7 expression been <br /> background 76% age-1(hx546) mutant                      reported wild type C. elegans 3 hours transfer 25uC to <br /> background. FAT-6 and/or FAT-7 promote recovery                      10uC [33]. Whilst cellular distribution DAF-16::GFP was <br /> locomotion genotypes following exposure low                       predominantly unlocalised age-1(+) adults 20uC, accumulation of <br /> temperatures. Although recovery delayed age-1(hx546);                 DAF-16::GFP observed nuclei age-1(hx546) mutants <br /> fat-6(tm331); fat-7(wa36) triple mutants relative wild type           (category 1&#226;&#8364;&#8220;2: t = 17.26; age-1(+) individuals displayed categories 3 <br /> strain (t = 26.07, p,0.001), worms recovered rapidly              4, p,0.001) (Figure 4B). Although slight increase the <br /> than fat-6(tm331); fat-7(wa36) double mutants background              intensity nuclear localisation age-1(hx546) mutant adults 4uC <br /> that wild type age-1 (t = 6.89, p,0.001), mean                  (category 2&#226;&#8364;&#8220;3: t = 2.79, p = 0.029), significant change the <br /> recovery times reduced approximately 26%. These results                   subcellular localisation DAF-16::GFP observed age-1(+) C. <br /> reinforce findings survival experiments (Figure 2A)             elegans temperature relative controls maintained 20uC <br /> concerning contribution fat-6, fat-7 genes               (Figure 4B). These observations indicate wild type C. elegans does <br /> tolerance cold stress age-1(hx546) mutants, exemplify          respond cold conditions specifically IIS pathway <br />  <br />  <br />  <br />  <br /> Figure 3. Cold coma recovery measure cold tolerance. Variation recovery times wild type (N2), age-1(hx546) mutants, fat- <br /> 6(tm331); fat-7(wa36) double mutants age-1(hx546); fat-6(tm331); fat-7(wa36) triple mutants following 6 hours cold shock 4uC (n = 95&#226;&#8364;&#8220;100 <br /> individuals genotype, observed 2 separate blocks). <br /> doi:10.1371/journal.pone.0024550.g003 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        5                        September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                        Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br />  <br />  <br /> Figure 4. Subcellular localization DAF-16::GFP response cold stress. A) Subcellular distributions DAF-16::GFP categorised <br /> with score 1 (left) nuclear localization 4 (right) complete nuclear localization. B) The mean proportion age-1(+) adults and <br /> age-1(hx546) mutant adults displayed categories 1&#226;&#8364;&#8220;4 DAF-16::GFP localisation 20uC (left) 4uC (right). Error bars represent standard <br /> errors means (n = 100&#226;&#8364;&#8220;120 genotype treatment observed 3 separate blocks). <br /> doi:10.1371/journal.pone.0024550.g004 <br />  <br /> and DAF-16. It unknown genes involved                     changes membrane lipid composition brought up- <br /> delivering cold tolerance regulated response acclimation          regulation D9 desaturase genes. Both cold tolerant phenotype <br /> cold conditions non-dauer wild type C. elegans. Nevertheless,          reduction prolongation cold coma recovery time arise when <br /> of targets IIS pathway DAF-16 up/down                 D9 desaturase expression experimentally disrupted age-1(hx546) <br /> regulated age-1(hx546) mutants clearly overlap mecha-              mutants. Tolerance 4uC appeared reduced greater extent when <br /> nism(s) required cold resistance.                                          fat-6 fat-7 D9 desaturase genes defective as <br />                                                                                compared fat-5 fat-6 fat-5 fat-7 expression was <br /> Discussion                                                                     eliminated suppressed. Consistent this, Brock et al. [31] <br />                                                                                demonstrated relationship sensitivity 10uC 15uC. <br />    It established mutations disrupt insulin/IGF-1           The fat-6(tm331); fat-7(wa36) double mutants displayed additional <br /> signalling promote longevity stress resistance diverse                  defects, including reduced fat storage, slow growth, reduced fecundity <br /> evolutionary lineages. However, assessment cold tolerance                high proportion embryonic lethality, despite 40- <br /> has previously reported IIS mutant species, despite         fold increase fat-5 expression [31]. Presumably observations <br /> this presumably important component life history                reflect close evolutionary relationship substrate specificity the <br /> many wild populations. We demonstrated age-1(hx546)                  FAT-6 FAT-7 D9 desaturases, distinct specificity the <br /> mutants increased cold tolerance relative wild type C. elegans         FAT-5 D9 desaturase [40]. When expression D9 <br /> when transferred directly optimal growth temperature                desaturase genes eliminated suppressed, low temperature <br /> 4uC early adulthood. As activity reproduction completely            resistance reduced dramatically age-1(hx546) mutant <br /> arrest temperature, survival 4uC does simply reflect            wild type background, age-1 <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> conferred <br /> longevity optimal conditions. We established           elevation cold tolerance. This suggests cold <br /> first time dauer larvae survive prolonged periods low          tolerance age-1(hx546) mutants principally D9 <br /> temperatures compared adults undergone develop-                  desaturase genes, additional genes contribute phenotype. <br /> ment favourable growth conditions, represent survival           Mutations age-1 cause changes expression genes <br /> strategy seasonal temperate environments.                                   encoding enzymes involved lipid metabolism affect <br />                                                                                membrane fluidity [43]. <br /> D9 desaturase genes promote cold tolerance age- <br /> 1(hx546) mutants                                                               Additional DAF-16 target genes contribute cold <br />    The cold tolerant phenotype age-1(hx546) mutants dauers              tolerance <br /> may arise worms physiologically prepared               IIS mutants dauers differ wild type adults other <br /> stress, regard threat membrane function,       aspects metabolism involved cold tolerance. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          6                         September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                     Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br /> For instance, genes involved synthesis trehalose up-            resistance ultraviolet light, hypertonic stress pathogen <br /> regulated daf-2(e1370) mutants dauers [36,37,44],                  infections dependent daf-16, functional daf-16 is <br /> trehalose levels approximately 2-fold higher age-1(hx546)              required wild type levels resistance conditions <br /> mutants wild type [20]. Trehalose sugars                [16,19,20]. Although different forms stress require distinct <br /> implicated cold acclimation and/or cold tolerance yeast [45],           responses, mediated different signalling pathways tran- <br /> nematodes [46] insects [47,48,49]. Glycerol,             scription factors, overlap expected stress response <br /> established role cryoprotectant implicated               proteins involved [64]. Consequently, daf-16 independent mech- <br /> rapid cold-hardening insect species [50],         anisms target effector genes expression down <br /> higher levels age-1(hx546) mutants wild type C. elegans            regulated IIS mutants dauers involved mediating <br /> [20]. Trehalose glycerol involved modifying            cold-induced stress response. Several transcription factors, in <br /> membrane lipid phase transition directly affecting membrane               addition DAF-16, known regulate D9 desaturase gene <br /> fluidity [51,52], act combination D9                     expression, including nuclear hormone receptors NHR-49 and <br /> desaturase enzymes enhance preservation membrane                    NHR-80, SBP-1, C. elegans homologue sterol-regulatory- <br /> function low temperatures age-1(hx546) mutants.                         element-binding-protein (SREBP) transcription factors, key regu- <br />    Murray et al. [33] demonstrated acquired cold tolerance            lators lipid homeostasis mammals [41,65,66,67]. It is <br /> C. elegans following period acclimation 10uC                   conceivable play important <br /> exclusively dependent membrane lipid composition,               role cold tolerance and/or cold acclimation C. elegans. One or <br /> additional mechanisms contribute cold acclimation                 transcription factors transcription <br /> cold tolerance poikilothermic animals. Heat-shock proteins                 factor contribute cold tolerance age-1(hx546) <br /> enhance cold tolerance S. cerevisiae [53] variety insect       mutants, including activation desaturase genes. <br /> species [54], antioxidant enzymes implicated                 However, increased cold tolerant phenotype age-1(hx546) <br /> resistance low temperatures insects [55] plants [56,57].            mutants entirely dependent functional daf-16, the <br /> Several genes encoding heat-shock proteins expressed                   additional resistance cold stress exhibited age-1(hx546) <br /> higher levels IIS mutants dauers wild type adults              mutants appears mediated DAF-16 activity. <br /> [14,35,37], certain antioxidants present higher levels in <br /> age-1(hx546) mutants dauers wild type adults [4].                 Evolutionary conservation longevity stress <br /> These make contribution to, fully explain, additional            resistance mechanisms <br /> cold tolerance phenotype provided desaturase                  We established time association <br /> genes age-1(hx546) mutants.                                            longevity resistance low temperatures exists C. <br />                                                                               elegans. Such relationship previously observed D. <br /> DAF-16 mediates general specific stress                       melanogaster lines selected increased longevity [21,22]. Although <br /> response                                                                      reduced insulin/IGF-1 signalling age-1(hx546) <br />    In contrast high temperature oxidative stress [58], daf-            mutants promotes cold tolerance, Broughton et al. [23] reported <br /> 16(mu86) mutants sensitive low temperatures wild type            increased longevity induced ablation neurosecretory <br /> adults. Additionally, activation DAF-16, revealed                 cells produce insulin-like peptides associated cold <br /> nuclear localisation DAF-16::GFP, observed                        sensitivity D. melanogaster. However, ablated flies displayed <br /> individuals wild type background exposed low                   reduced levels cryoprotectant trehalose hemolymph, <br /> temperatures. This suggests insulin/IGF-1 signalling                 trehalose levels up-regulated C. elegans IIS mutants and <br /> DAF-16 activation required expression genes             dauer larvae [20,36,37,44]. This discrepancy presumably reflects <br /> physiological defence cold-induced damage. RNAi fat-6              differences downstream targets IIS pathway in <br /> and fat-7 increase resistance high temperatures wild type          different species. <br /> C. elegans independently daf-16 [59], example stress             The higher cold tolerance dauers important for <br /> defense mechanism mediated lipid metabolism                       survival C. elegans populations experience rapid fluctuations <br /> without DAF-16 activity.                                                      temperature. In invertebrates diapause primary <br />    Using line transgenic daf-16::gfp fusion gene [12] different       mechanism over-winter survival, induced response to <br /> to used study [11], Wolf et al. [60] reported                 changes temperature and/or photoperiod (reviewed [68,69]). <br /> moderate activation DAF-16, revealed increased nuclear               In D. melanogaster mosquito Culex pipiens, appears be <br /> localization, following 2 hours exposure 1uC 8uC. However,              regulated IIS pathway orthologues DAF-16 <br /> they observed nuclear localisation DAF-16::GFP                     [7,70,71]. However, contrast high temperatures [72], there <br /> approximately 40% adults wild type background                evidence suggest dauer formation C. elegans be <br /> optimal growth temperature 20uC. Cold exposure                    induced exposure cold conditions. Dauers presumably <br /> shown modify insulin/IGF-1 signalling tissue specific                 exhibit cold tolerance overlap effector genes <br /> manner rats, signal transduction reduced                 targeted IIS pathway/DAF-16 and, cold <br /> hypothalamus, skeletal muscle white adipose tissue,                   induced stress response. Overlaps effector genes different <br /> enhanced brown adipose tissue [61,62]. Though remains                   stress conditions facilitate evolutionary change stress <br /> unclear transgenic C. elegans strain accurately portrays           responses, allowing adaptation changes spectrum stress <br /> changes subcellular distribution DAF-16 different                 conditions encountered. <br /> thermal conditions, cold tolerance wild type C. elegans              Associations lifespan stress resistance mutants <br /> been shown daf-16 independent.                                     reduced insulin/IGF-1 signalling observed a <br />    DAF-16::GFP translocates nucleus response high                variety model organisms. As insights mechanisms which <br /> temperatures, oxidative stress starvation [11], remains               underlie associations contribute understanding of <br /> unlocalized exposure ultraviolet light [11], hypertonic               longevity stress resistance, interesting establish if <br /> stress [20], Pseudomonas aeruginosa infection [63]. Like              relationship lifespan cold tolerance identified in <br /> findings presented here, long-lived IIS mutants display increased             C. elegans IIS mutants conserved. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         7                        September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                                    Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br />  <br /> Materials Methods                                                                      5 mM levamisole epifluorescence imaging per- <br />                                                                                            formed using Leica DMR HC confocal microscope. <br /> Strains culture conditions <br />    The following genotypes obtained Caenorhabdi-                             Statistical analyses <br /> tis Genetics Centre: N2 Bristol (wild type), TJ1052 age-1(hx546),                             All analyses performed R version 2.10.1 [74]. <br /> CF1038 daf-16(mu86), BX107 fat-5(tm420), BX106 fat-6(tm331),                                  For survival analysis, small number individuals had <br /> BX153 fat-7(wa36), BX160 fat-6(tm331); fat-5(tm420), BX110 fat-                            died causes exposure low temperatures (e.g. <br /> 7(wa36); fat-5(tm420), BX156 fat-6(tm331); fat-7(wa36) TJ356                           rupturing vulva) removed analysis. Each <br /> N2; zIs356 (daf-16p::daf-16a::gfp; rol-6[su1006]). Long-lived age-                         survival experiment analysed independently using acceler- <br /> 1(hx546) mutants utilized ability develop                         ated failure time parametric regression model Weibull error <br /> reproduce temperatures induce developmental arrest                             distribution. These models selected based visual <br /> other IIS mutants. Crosses age-1(hx546) mutants                                inspection plotted residuals following analyses using variety <br /> additional strains confirmed PCR using gene specific                               different probability distributions. They considered be <br /> primers listed Table S1. Except stated otherwise, strains                         robust non-parametric survival models latter <br /> were maintained nematode growth media (NGM) plates,                                     typically greatest power hazard ratios (relative risks of <br /> containing 10 mg/ml nystatin 50 mg/ml streptomycin,                                death) covariates proportional, the <br /> seeded E. coli strain HB101. Strains maintained 20uC                          case data. However, assess generality results <br /> until cold tolerance assays commenced.                                                     obtained accelerated failure time models, analysed <br />                                                                                            survival experiments using non-parametric weighted cox <br /> RNAi                                                                                       regression models, provide unbiased average hazard ratio <br />    RNA interference (RNAi) induced feeding using clones                             estimates assuming proportional hazards. We obtained <br /> from Ahringer library. RNAi experiments performed                              qualitatively similar results sets analyses, but <br /> NGM plates supplemented Ampicillin, Tetracycline                                  provide z-scores derived accelerated failure time <br /> IPTG previously described [73]. C. elegans fed HT115                                 models results section. <br /> bacteria containing RNAi plasmid vector (pL4440)                                Cold coma recovery times compared genotypes <br /> used negative controls.                                                                 using generalised linear mixed effects model gamma error <br />                                                                                            distribution. The model fitted using penalised quasi- <br /> Cold tolerance assays                                                                      likelihood (PQL) method [75], included random effects terms <br />    Age-synchronised adults, day reproduction,                         account variation NGM plates blocks. <br /> transferred directly 20uC 4uC60.5uC plates containing                              DAF-16::GFP intracellular localisation scores compared <br /> approximately 20 individuals each. Survival monitored                                  genotypes temperature treatments using an <br /> regularly, following 20&#226;&#8364;&#8220;30 minutes recovery room temperature,                           ordinal multinomial continuation-ratio logit model [76]. When <br /> until individuals died. If survival status                                 certain categories displayed genotypes <br /> immediately apparent, worms gently touched                                     following particular thermal treatment, possible to <br /> platinum wire stimulate response.                                                     obtain parameters observed category <br />    Dauer larvae transferred directly 20uC 4uC60.5uC                           transitions. These situations described parenthesis in <br /> approximately 24 hours populations starved.                               results. P-values obtained using likelihood ratio tests to <br /> Survival status monitored 100 individuals, divided 5                          compare models following sequential removal of <br /> NGM plates, non-dauer controls died.                                         explanatory variables interactions. <br />  <br /> Cold coma recovery times                                                                   Supporting Information <br />    Age-synchronised adults, day reproduction, were <br /> transferred 20uC 4uC60.5uC 6 hours NGM plates                               Table S1 Sequences oligonucleotide primer pairs <br /> containing 10 individuals each. After stress period, recovery                          used PCRs genomic DNA confirm age-1, fat-5, <br /> times monitored room temperature (approximately 22uC).                             fat-6, fat-7 daf-16 genotypes. <br /> Individuals considered recovered began                           (DOCX) <br /> move spontaneously NGM plates. Recovery times were <br /> recorded 100 individuals genotype 2 separate blocks.                           Acknowledgments <br />                                                                                            We like thank Caenorhabditis Genetics Centre (CGC) for <br /> DAF-16::GFP localisation                                                                   providing strains, Dr. S. Bamps members Hope lab for <br />    Cellular distributions DAF-16::GFP fusion protein                             helpful discussions, A. Reynolds J. Taylor technical assistance with <br /> compared fed young adults                                        recovery assays, Prof. T. Benton statistical advice, anonymous <br /> maintained control temperature (20uC) exposed                                 reviewers valuable suggestions. <br /> 4uC60.5uC 6 hours. DAF-16::GFP localisation catego- <br /> rised 1&#226;&#8364;&#8220;4, 1 represents unlocalised distribution                             Author Contributions <br /> throughout cells 2, 3 4 represent increasingly nuclear                         Conceived designed experiments: FRS SMS IAH. Performed the <br /> distributions, 100&#226;&#8364;&#8220;120 individuals genotype tempera-                            experiments: FRS. Analyzed data: FRS SMS IAH. Wrote paper: <br /> ture treatment 3 separate blocks. Worms anaesthetized                            FRS SMS IAH. <br />  <br /> References <br /> 1. Kenyon C (2010) The genetics ageing. Nature 464: 504&#226;&#8364;&#8220;512.                             3. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans <br /> 2. Friedman DB, Johnson TE (1988) Three mutants extend mean                      mutant lives twice long wild type. Nature 366: 461&#226;&#8364;&#8220;464. <br />    maximum life span nematode, Caenorhabditis elegans, define age-1 gene.        4. Larsen PL (1993) Aging resistance oxidative damage Caenorhabditis <br />    J Gerontol 43: 102&#226;&#8364;&#8220;109.                                                                     elegans. Proc Natl Acad Sci U S A 90: 8905&#226;&#8364;&#8220;8909. <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                      8                          September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                                             Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br />  5. Lithgow GJ, White TM, Hinerfeld DA, Johnson TE (1994) Thermotolerance                          mutants implicates detoxification longevity assurance. J Biol Chem <br />     long-lived mutant Caenorhabditis elegans. J Gerontol 49: B270&#226;&#8364;&#8220;B276.                          279: 44533&#226;&#8364;&#8220;44543. <br />  6. Clancy DJ, Gems D, Harshman LG, Oldham S, Stocker H, et al. (2001)                          36.   McElwee J, Schuster E, Blanc E, Thornton J, Gems D (2006) Diapause- <br />     Extension life-span loss CHICO, Drosophila insulin receptor substrate                  associated metabolic traits reiterated long-lived daf-2 mutants nematode <br />     protein. Science 292: 104&#226;&#8364;&#8220;106.                                                                    Caenorhabditis elegans. Mechanisms Ageing Development 127: 458&#226;&#8364;&#8220;472. <br />  7. Tatar M, Kopelman A, Epstein D, Tu MP, Yin CM, et al. (2001) A mutant                       37.   Wang J, Kim SK (2003) Global analysis dauer gene expression in <br />     Drosophila insulin receptor homolog extends life-span impairs                            Caenorhabditis elegans. Development 130: 1621&#226;&#8364;&#8220;1634. <br />     neuroendocrine function. Science 292: 107&#226;&#8364;&#8220;110.                                              38.   Dusenbery DB, Barr J (1980) Thermal limits chemotaxis mutants the <br />  8. Holzenberger M, Dupont J, Ducos B, Leneuve P, Geloen A, et al. (2003) IGF-1                       nematode Caenorhabditis elegans defective thermotaxis. J Comp Physiol 137: <br />     receptor regulates lifespan resistance oxidative stress mice. Nature 421:               353&#226;&#8364;&#8220;356. <br />     182&#226;&#8364;&#8220;187.                                                                                    39.   Lyons JM, Keith AD, Thomason IJ (1975) Temperature-induced phase <br />  9. Bonafe M, Barbieri M, Marchegiani F, Olivieri F, Ragno E, et al. (2003)                           transitions nematode lipids influence respiration. J Nematol 7: <br />     Polymorphic variants Insulin-Like Growth Factor I (IGF-I) receptor                         98&#226;&#8364;&#8220;104. <br />     phosphoinositide 3-kinase genes affect IGF-I plasma levels human longevity.             40.   Watts JL, Browse J (2000) A palmitoyl-CoA specific D9 fatty acid desaturase <br />     J Clin Endocrinol Metab 88: 3299&#226;&#8364;&#8220;3304.                                                            Caenorhabditis elegans. Biochem Biophys Res Commun 272: 263&#226;&#8364;&#8220;269. <br /> 10. van Heemst D, Beekman M, Mooijaart SP, Heijmans BT, Brandt BW, et al.                       41.   Brock TJ, Browse J, Watts JL (2006) Genetic regulation unsaturated fatty acid <br />     (2005) Reduced insulin/IGF-1 signalling human longevity. Aging Cell 4:                        composition C. elegans. PLoS Genet 2: 997&#226;&#8364;&#8220;1005. <br />     79&#226;&#8364;&#8220;85.                                                                                      42.   David RJ, Gibert P, Pla E, Petavy G, Karan D, et al. (1998) Cold stress tolerance <br /> 11. Henderson S, Johnson T (2001) daf-16 integrates developmental                                 Drosophila: analysis chill coma recovery D. melanogaster. J Therm Biol 23: <br />     environmental inputs mediate aging nematode Caenorhabditis elegans.                     291&#226;&#8364;&#8220;299. <br />     Current Biology 11: 1975&#226;&#8364;&#8220;1980.                                                              43.   Schmookler Reis RJS, Xu LL, Lee H, Chae M, Thaden JJ, et al. (2011) <br /> 12. Lin K, Hsin H, Libina N, Kenyon C (2001) Regulation Caenorhabditis                         Modulation lipid biosynthesis contributes stress resistance longevity of <br />     elegans longevity protein DAF-16 insulin/IGF-1 germline signaling.                         C. elegans mutants. Aging 3: 125&#226;&#8364;&#8220;147. <br />     Nature Genetics 28: 139&#226;&#8364;&#8220;145.                                                                44.   Honda Y, Tanaka M, Honda S (2010) Trehalose extends longevity the <br /> 13. Lee SS, Kennedy S, Tolonen AC, Ruvkun G (2003) DAF-16 target genes                           nematode Caenorhabditis elegans. Aging Cell 9: 558&#226;&#8364;&#8220;569. <br />     control C. elegans life-span metabolism. Science 300: 644&#226;&#8364;&#8220;647.                          45.   Schade B, Jansen G, Whiteway M, Entian KD, Thomas DY (2004) Cold <br /> 14. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003)                          adaptation budding yeast. Molec Biol Cell 15: 5492&#226;&#8364;&#8220;5502. <br />     Genes act downstream DAF-16 influence lifespan Caenorhabditis             46.   Jagdale GB, Grewal PS, Salminen SO (2005) Both heat-shock cold-shock <br />     elegans. Nature 424: 277&#226;&#8364;&#8220;283.                                                                     influence trehalose metabolism entomopathogenic nematode. J Parasitol <br /> 15. Vanfleteren JR (1993) Oxidative stress ageing Caenorhabditis elegans.                      91: 988&#226;&#8364;&#8220;994. <br />     J Biochem 292: 605&#226;&#8364;&#8220;608.                                                                     47.   Benoit JB, Lopez-Martinez G, Elnitsky MA, Lee RE, Denlinger DL (2009) <br /> 16. Murakami S, Johnson TE (1996) A genetic pathway conferring life extension                         Dehyration-induced cross tolerance Belgica antarctica larvae cold heat is <br />     resistance UV stress Caenorhabditis elegans. Genetics 143: 1207&#226;&#8364;&#8220;1218.                   facilitated trehalose accumulation. Comp Biochem Physiol A 152: 518&#226;&#8364;&#8220;523. <br /> 17. Barsyte D, Lovejoy DA, Lithgow GJ (2001) Longevity heavy metal                          48.   Khani A, Moharramipour S, Barzegar M (2007) Cold tolerance trehalose <br />     resistance daf-2 age-1 long-lived mutants Caenorhabditis elegans. The                   accumulation overwintering larvae codling moth, Cydia pomonella <br />     FASEB Journal 15: 627&#226;&#8364;&#8220;634.                                                                        (Lepidoptera: Tortricidae). Eur J Entomology 104: 385&#226;&#8364;&#8220;392. <br /> 18. Scott BA, Avidan MS, Crowder CM (2002) Regulation hypoxic death C.                    49.   Overgaard J, S&#195;&#184;rensen JG, S&#195;&#184;ren OP, Loeschcke V, Holmstrup M (2005) <br />     elegans insulin/IGF receptor homolog DAF-2. Science 296: 2388&#226;&#8364;&#8220;2391.                        Changes membrane lipid composition following rapid cold hardening in <br />                                                                                                       Drosophila melanogaster. J Insect Physiol 51: 1173&#226;&#8364;&#8220;1182. <br /> 19. Garsin DA, Villanueva JM, Begun J, Kim DH, Sifri CD, et al. (2003) Long-lived <br />                                                                                                 50.   Lee RE, Chen CP, Denlinger DL (1987) A rapid cold-hardening process in <br />     C. elegans daf-2 mutants resistant bacterial pathogens. Science 300: 1921. <br />                                                                                                       insects. Science 238: 1415&#226;&#8364;&#8220;1417. <br /> 20. Lamitina ST, Strange K (2005) Transcriptional targets DAF-16 insulin <br />                                                                                                 51.   Boggs JM, Rangaraj G (1985) Phase transitions fatty acid spin <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> behavior <br />     signaling pathway protect C. elegans extreme hypertonic stress. Am J Physiol <br />                                                                                                       interdigitated lipid phases induced glycerol. Biochim Biophys Acta 816: <br />     Cell Physiol 288: 467&#226;&#8364;&#8220;474. <br />                                                                                                       221&#226;&#8364;&#8220;233. <br /> 21. Luckinbill LS (1998) Selection longevity confers resistance low- <br />                                                                                                 52.   Crowe JH, Hoekstra FA, Crowe LM (1992) Anhydrobiosis. Ann Rev Physiol 54: <br />     temperature stress Drosophila melanogaster. J Gerontol A Bio Sci Med Sci 53: <br />                                                                                                       579&#226;&#8364;&#8220;599. <br />     B147&#226;&#8364;&#8220;153. <br />                                                                                                 53.   Pacheco A, Pereira C, Almeida MJ, Sousa MJ (2009) Small heat-shock protein <br /> 22. Norry FM, Loeschcke VR (2002) Longevity resistance cold stress cold- <br />                                                                                                       Hsp12 contributes yeast tolerance freezing stress. Microbiology 155: <br />     stress selected lines controls Drosophila melanogaster. J Evol Biol 15: <br />                                                                                                       2021&#226;&#8364;&#8220;2028. <br />     775&#226;&#8364;&#8220;783. <br />                                                                                                 54.   Rinehart JP, Li A, Yocum GD, Robich RM, Hayward SAL, et al. (2007) Up- <br /> 23. Broughton SJ, Piper MD, Ikeya T, Bass TM, Jacobson J, et al. (2005) Longer                        regulation heat shock proteins essential cold survival insect <br />     lifespan, altered metabolism, stress resistance Drosophila ablation                diapause. Proc Natl Acad Sci U S A 104: 11130&#226;&#8364;&#8220;11137. <br />     cells making insulin-like ligands. Proc Natl Acad Sci USA 102: 3105&#226;&#8364;&#8220;3110.                   55.   Joanisse DR, Storey KB (1996) Oxidative stress antioxidants in <br /> 24. Lindquist S (1986) The heat-shock response. Ann Rev Biochem 55: 1151&#226;&#8364;&#8220;1191.                        overwintering larvae cold-hardy goldenrod gall insects. J Exp Biol 199: <br /> 25. Hazel JR (1995) Thermal adaptation biological membranes: Is homeoviscous                       1483&#226;&#8364;&#8220;1491. <br />     adaptation explanation? Ann Rev Physiol 57: 19&#226;&#8364;&#8220;42.                                      56.   Dai F, Huang Y, Zhou M, Zhang G (2009) The influence cold acclimation on <br /> 26. Cossins AR (1994) Homeoviscous adaptation biological membranes                         antioxidative enzymes antioxidants sensitive tolerant barley cultivars. <br />     functional significance. In: Cossins A, ed. Temperature adaptation biological                  Biologia Plantarum 53: 257&#226;&#8364;&#8220;262. <br />     membranes Portland Press. pp 63&#226;&#8364;&#8220;76.                                                         57.   Thomashow MF (1999) Plant cold acclimation: Freezing tolerance genes and <br /> 27. Tiku PE, Gracey AY, Macartney AI, Beynon RJ, Cossins AR (1996) Cold-                              regulatory mechanisms. Ann Rev Plant Physiol Plant Mol Biol 50: 571&#226;&#8364;&#8220;599. <br />     induced expression D9 desaturase carp transcriptional post-                    58.   Henderson ST, Bonafe M, Johnson TE (2006) daf-16 protects nematode <br />     translational mechanisms. Science 271: 815&#226;&#8364;&#8220;818.                                                   Caenorhabditis elegans food deprivation. J Gerontol A Biol Sci Med Sci 61: <br /> 28. Uemura M, Joseph RA, Steponkus PL (1995) Cold acclimation Arabidopsis                          444&#226;&#8364;&#8220;60. <br />     thaliana: Effect plasma membrane lipid composition freeze-induced                    59.   Horikawa M, Sakamoto K (2009) Fatty-acid metabolism involved stress- <br />     lesions. Plant Physiol 109: 15&#226;&#8364;&#8220;30.                                                                resistance mechanisms Caenorhabditis elegans. Biochem Biophys Res Comm 390: <br /> 29. Wada H, Combos Z, Murata N (1990) Enhancement chilling tolerance                          1402&#226;&#8364;&#8220;1407. <br />     cyanobacterium genetic manipulation fatty acid desaturation. Nature 347:              60.   Wolf M, Nunes F, Henkel A, Heinick A, Paul RJ (2008) The MAP kinase JNK-1 <br />     200&#226;&#8364;&#8220;203.                                                                                          Caenorhabditis elegans: location, activation influences temperature- <br /> 30. Ishizaki-Nishizawa O, Fujii T, Azuma M, Sekiguchi K, Murata N, et al. (1996)                      dependent insulin-like signalling, stress responses fitness. J Cell Physiol 214: <br />     Low temperature resistance higher plants significantly enhanced non-                   721&#226;&#8364;&#8220;229. <br />     specific cyanobacterial desaturase. Nature Biotechnol 14: 1003&#226;&#8364;&#8220;1006.                        61.   Gasparetti AL, Souza CT, Pereira-da-Silva M, Oliviera RLGS, Saad MJA, <br /> 31. Brock T, Browse J, Watts J (2007) Fatty acid desaturation regulation                   et al. (2003) Cold exposure induces tissue-specific modulation insulin- <br />     adiposity Caenorhabditis elegans. Genetics 176: 865&#226;&#8364;&#8220;875.                                       signalling pathway Rattus norvegicus. J Physiol 552: 149&#226;&#8364;&#8220;162. <br /> 32. Gracey AY, Fraser EJ, Li W, Fang Y, Brass A, et al. (2004) Coping cold:             62.   Torsoni AM, Carvalheira JB, Pereira-Da-Silva M, Carvalho-Filho MA, <br />     integrative, multi-tissue analysis transcriptome poikilothermic                       Saad MJA, et al. (2003) Molecular functional resistance insulin in <br />     vertebrate. Proc Natl Acad Sci USA 101: 16970&#226;&#8364;&#8220;16975.                                              hypothalamus rats exposed cold. Am J Physiol Endo 285: E216&#226;&#8364;&#8220;E223. <br /> 33. Murray P, Hayward SAL, Govan GG, Gracey AY, Cossins AR (2007) An                            63.   Troemel ER, Chu SW, Reinke V, Lee SS, Ausubel FM, et al. (2006) p38 MAPK <br />     explicit test phospholipid saturation hypothesis acquired cold tolerance                regulates expression immune response genes contributes longevity C. <br />     Caenorhabditis elegans. Proc Natl Acad Sci U S A 104: 5489&#226;&#8364;&#8220;5494.                               elegans. PLoS Genet 2: e183. <br /> 34. Schuster E, McElwee JJ, Tullet JMA, Doonan R, Matthijssens F, et al. (2010)                 64.   Ku&#194;&#168;ltz D (2005) Molecular evolutionary basis cellular stress response. <br />     DamID C. elegans reveals longevity-associated targets DAF-16/FoxO.                          Ann Rev Physiol 67: 225&#226;&#8364;&#8220;257. <br />     Molecular Systems Biol 6: 399.                                                              65.   Ashrafi K, Chang FY, Watts JL, Fraser AG, Kamath RS, et al. (2003) Genome- <br /> 35. McElwee JJ, Schuster E, Blanc E, Thomas JH, Gems D (2004) Shared                                  wide RNAi analysis Caenorhabditis elegans fat regulatory genes. Nature 421: <br />     transcriptional signature Caenorhabditis elegans dauer larvae long-lived daf-2             268&#226;&#8364;&#8220;272. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                          9                              September 2011 | Volume 6 | Issue 9 | e24550 <br />                                                                                                                        Cold Tolerance Long-Lived C. elegans Mutants <br />  <br />  <br /> 66. van Gilst MR, Hadjivassiliou H, Jolly A, Yamamoto KR (2005) Nuclear                       72. Golden JW, Riddle DL (1984) The Caenorhabditis elegans dauer larva: <br />     hormone receptor NHR-49 controls fat consumption fatty acid composition                   developmental effects pheromone, food temperature. Dev Biol 102: <br />     C. elegans. PLoS Biol 3: 301&#226;&#8364;&#8220;312.                                                          368&#226;&#8364;&#8220;378. <br /> 67. Yang F, Vought BW, Satterlee JS, Walker AK, Sun ZYJ, et al. (2006) An ARC/                73. Kamath RS, Ahringer J (2003) Genome-wide RNAi screening Caenorhabditis <br />     Mediator subunit required SREBP control cholesterol lipid                          elegans. Methods 30: 313&#226;&#8364;&#8220;321. <br />     homeostasis. Nature 442: 700&#226;&#8364;&#8220;704.                                                         74. R Development Core Team (2010) R: A language environment for <br /> 68. Bale JS, Hayward SAL (2010) Insect overwintering changing climate. J Exp                 statistical computing. Available: http://www.R-project.org Accessed 2011 Jun <br />     Biol 213: 980&#226;&#8364;&#8220;994.                                                                            16. <br /> 69. Denlinger DL (2002) Regulation diapause. Ann Rev Entomol 47: 93&#226;&#8364;&#8220;122.                   75. Breslow NE, Clayton DG (1993) Approximate inference generalized linear <br /> 70. Sim C, Denlinger DL (2008) Insulin signalling FOXO regulate                           mixed models. J Am Stat Assoc 88: 9&#226;&#8364;&#8220;25. <br />     overwintering diapause mosquito Culex pipiens. Proc Natl Acad Sci U S A            76. Agresti A (2002) Categorical data analysis. New York: Wiley-Interscience <br />     105: 6777&#226;&#8364;&#8220;6781.                                                                               Publishers. 734 p. <br /> 71. Williams KD, Busto M, Suster ML, So AKC, Ben-Shahar Y, et al. (2006) <br />     Natural variation Drosophila melanogaster diapause insulin-regulated <br />     PI3-kinase. Proc Natl Acad Sci U S A 103: 15911&#226;&#8364;&#8220;15915. <br />  <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org               <br /> </body></html>