<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Sept. 2011, p. 6133&#226;&#8364;&#8220;6140                                                              Vol. 77, No. 17 <br /> 0099-2240/11/$12.00 doi:10.1128/AEM.00630-11 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />           Determining Extremes Cellular NAD(H) Level by <br />             Using Escherichia coli NAD&#226;&#171;&#185;-Auxotrophic Mutant&#228;&#338;&#164; <br />   Yongjin Zhou,1,3 Lei Wang,1,3 Fan Yang,1 Xinping Lin,1,3 Sufang Zhang,1 Zongbao K. Zhao1,2* <br /> Dalian Institute Chemical Physics, Chinese Academy Sciences, Dalian 116023, People&#226;&#8364;&#8482;s Republic China1; Dalian National <br />               Laboratory Clean Energy, Dalian 116023, People&#226;&#8364;&#8482;s Republic China2; Graduate University of <br />                            Chinese Academy Sciences, Beijing 100049, People&#226;&#8364;&#8482;s Republic China3 <br />                                                Received 21 March 2011/Accepted 28 June 2011 <br />  <br />              NAD (NAD&#226;&#180;&#8482;) reduced form (NADH) omnipresent cofactors biological systems. However, is <br />           difficult determine extremes cellular NAD(H) level live cells NAD&#226;&#180;&#8482; level tightly <br />           controlled biosynthesis regulation mechanism. Here, developed strategy determine extreme <br />           NAD(H) levels Escherichia coli cells genetically engineered NAD&#226;&#180;&#8482; auxotrophic. First, we <br />           expressed ntt4 gene encoding NAD(H) transporter E. coli mutant YJE001, deletion <br />           nadC gene responsible NAD&#226;&#180;&#8482; novo biosynthesis, showed NTT4 conferred mutant <br />           strain better growth presence exogenous NAD&#226;&#180;&#8482;. We constructed NAD&#226;&#180;&#8482;-auxotrophic mutant <br />           YJE003 disrupting essential gene nadE, responsible step NAD&#226;&#180;&#8482; biosynthesis in <br />           cells harboring ntt4 gene. The minimal NAD&#226;&#180;&#8482; level determined M9 medium proliferating YJE003 <br />           cells preloaded NAD&#226;&#180;&#8482;, maximal NAD(H) level determined exposing <br />           cells high concentrations exogenous NAD(H). Compared supplementation NADH, cells <br />           grew faster higher intracellular NAD(H) level NAD&#226;&#180;&#8482; fed. The intracellular NAD(H) <br />           level increased increase exogenous NAD&#226;&#180;&#8482; concentration, reached plateau. Thus, a <br />           minimal NAD(H) level 0.039 mM maximum 8.49 mM determined, 0.044&#226;&#180;&#8250; and <br />           9.6&#226;&#180;&#8250; wild-type cells, respectively. Finally, potential application strategy biotech- <br />           nology briefly discussed. <br />  <br />  <br />    NAD (NAD&#226;&#171;&#185;; reduced state NADH) plays                              inner membrane, NAD&#226;&#171;&#185; se im- <br /> essential role cellular metabolism function                 permeable bacterial cytoplasmic membrane (Fig. 1B) <br /> cofactor 300 redox reactions (10). Thus, NAD&#226;&#171;&#185;                      (21). The NAD&#226;&#171;&#185; transporter, NTT4, discovered in <br /> level NADH/NAD&#226;&#171;&#185; ratio determine metabolic                          prokaryote chlamydial endosymbiont Protochlamydia <br /> fluxes pathways transcriptional level                   amoebophila UWE25, transport intact NAD(H) <br /> of genes vivo (13, 14). For example, high NADH/                      counterexchange ADP (11). Recently, NAD&#226;&#171;&#185; <br /> NAD&#226;&#171;&#185; ratio triggers overflow metabolism (33), low                        carriers located chloroplasts mitochondria been <br /> NADH/NAD&#226;&#171;&#185; ratio enhances glycolytic flux (38).                             discovered eukaryotes Saccharomyces cerevisiae and <br /> Manipulating NADH/NAD&#226;&#171;&#185; ratio considered                          Arabidopsis thaliana (20, 28). These transporters import <br /> the effective strategies improving productivity                   NAD&#226;&#171;&#185;, counterexchange nucleotides (ADP, AMP, or <br /> of products (19, 23). There metabolic path-                       GMP), cells organelles lack NAD&#226;&#171;&#185; biosynthesis <br /> ways NAD&#226;&#171;&#185; biosynthesis Escherichia coli, novo                    capacity support biological process (Fig. 1B). <br /> pathway salvage pathway (3, 15). The novo path-                        The level intracellular NAD(H) tightly controlled by <br /> way starts oxidation aspartate iminosuccinic                    variety mechanisms, including biosynthesis sal- <br /> acid, catalyzed NadB turn reacts                       vage NadR regulation (10). When internal <br /> dihydroxyacetone phosphate quinolinic acid (QA),                       NAD(H) level low, NadR interacts integral mem- <br /> followed phosphoribosylation decarboxylation, re-                       brane protein PnuC, responsible transport <br /> sulting nicotinic acid mononucleotide (NAMN). The sal-                      NMN cytoplasmic membrane, promoting net <br /> vage pathway converts precursors (nicotinic acid                       NMN transportation cell. When NAD(H) level <br /> [NA], nicotinamide [NM], nicotinamide mononucle-                           increases, NadR binds operator regions target <br /> otide [NMN]) NAMN, converted                            genes (nadA, nadB, pncB) represses transcrip- <br /> NAD&#226;&#171;&#185; adenylation amidation catalyzed                            tion decrease biosynthesis. However, studies <br /> NadD NadE, respectively (Fig. 1A). Like                        succeeded enhancing NAD(H) pool manip- <br /> precursors, NA, NMN taken                           ulating biosynthetic pathway regulation <br /> medium (25); however, extracellular NAD&#226;&#171;&#185; degraded                        system. With overexpression pncB control <br /> NMN adenosine periplasm passing                              native operator promoter sequences, intra- <br />                                                                                cellular NAD(H) increased 5-fold cells cultured <br />                                                                                presence 0.1 mM exogenous nicotinic acid (34). <br />    * Corresponding author. Mailing address: Dalian Institute Chem- <br /> ical Physics, CAS, Dalian 116023, People&#226;&#8364;&#8482;s Republic China. Phone <br />                                                                                The pncB mutated operator region protect <br /> and fax: 86-411-84379211. E-mail: zhaozb@dicp.ac.cn.                           regulatory effect NadR overexpressed <br />    &#228;&#338;&#164; <br />      Published ahead print 8 July 2011.                                  lac promoter, resulting 2.2-fold increase of <br />  <br />                                                                         6133 <br />  6134     ZHOU ET AL.                                                                                              APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Overview NAD&#226;&#171;&#185; metabolism. (A) NAD&#226;&#171;&#185; biosynthetic salvage pathways E. coli. (B) Schematic representation linkage <br /> between NAD&#226;&#171;&#185; biosynthesis transport pathway. Dashed lines NAD&#226;&#171;&#185; transport model chloroplasts mitochondria, <br /> discovered eukaryotes S. cerevisiae A. thaliana; solid lines refer NAD&#226;&#171;&#185; metabolic pathway transport pathway prokaryotes E. <br /> coli Chlamydia, respectively. Abbreviations: ASP, asparate; DHAP, dihydroxyacetone phosphate; ImASP, imino asparate; QA, quinolinate; <br /> NA, nicotinate; NM, nicotinamide; NMN, nicotinamide mononucleotide; NmR, nicotinamide ribonucleoside; NaR, nicotinate ribonucleoside; <br /> NAMN, nicotinic acid mononucleotide; dNAD, deamino NAD&#226;&#171;&#185;; Ade, adenosine. Enzymes: NadB, L-aspartate oxidase; NadA, quinolinate <br /> synthetase; NadC, quinolinate phosphoribosyltransferase; NadD, NAMN adenyltransferase; NadE, NAD&#226;&#171;&#185; synthetase; UshA, NMN glycohydro- <br /> lase; DeoD, purine-nucleoside phosphorylase PncA, Nm deamidase; PncB, NA phosphoribosyltransferase; NudC, NAD&#226;&#171;&#185; pyrophosphatase; NadK, <br /> NAD&#226;&#171;&#185; kinase. ScNDT1 ScNDT2, NAD&#226;&#171;&#185; carriers NDT1 NDT2 S. cerevisiae; AtNDT1 AtNDT2, NAD&#226;&#171;&#185; carriers NDT1 and <br /> NDT2 A. thaliana. <br />  <br />  <br />  <br /> the NAD(H) level (22). Cooverexpression pncB                   In study, manipulated NAD(H) level feeding <br /> nadE gene led dramatic 7-fold increase                   NAD(H) NAD&#226;&#171;&#185;-auxotrophic E. coli mutant was <br /> NAD(H) level (12). The NAD(H) level reduced                      constructed disrupting essential gene nadE, is <br /> with nadD <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>mutation</span>, encoded NAMN adeny-                     responsible step NAD&#226;&#171;&#185; biosynthesis cells <br /> lyltransferase (25).                                                      NAD&#226;&#171;&#185; transporter NTT4 expression background. <br />    Although NAD(H) pool adjusted manipu-                    This led successful determination extremes the <br /> lating biosynthetic pathway, remains difficult inves-           intracellular NAD(H) level. Finally, discuss potential <br /> tigate effects extremes NAD(H) level cell                application results fields biotechnology re- <br /> growth corresponding self-regulation mechanism.                lated cofactor engineering. <br />  VOL. 77, 2011                                                                           EXTREMES OF THE CELLULAR NAD(H) LEVEL                                   6135 <br />  <br />  <br />                                                     TABLE 1. Strains plasmids used study <br />  Strain plasmid                                                 Genotype characteristic                                                Resource reference <br />  <br /> E. coli strains <br />   DH5&#226;?&#163;                         F&#226;&#171;&#186; &#226;?&#190;80dlacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;(lacZYA-argF)U169 deoR recA1 endA1 hsdR17(rk&#226;&#171;&#186; mk&#226;&#171;&#185;) phoA supE44                                     TaKaRa <br />                                   &#226;?&#173;&#226;&#171;&#186; thi-1 gyrA96 relA1 <br />   BW25113                      rrnB3 &#226;&#338;&#172;lacZ4787 hsdR514 &#226;&#338;&#172;(araBAD)567 &#226;&#338;&#172;(rhaBAD)568 rph-1                                                             8 <br />   JW0105                       BW25113/nadC::kan                                                                                                   2 <br />   YJE001                       BW25113/nadC::kan/pET15b-NTT4                                                                                       This study <br />   YJE002                       BW25113/pKD46/pET15K-NTT4                                                                                           This study <br />   YJE003                       BW25113/pET15K-NTT4/nadE::cat                                                                                       This study <br />  <br /> Plasmids <br />   pMD18-T                      lacZ pBR322 ori bla; cloning vector                                                                                 TaKaRa <br />   pET15b                       T7 promoter, lacI pBR322 ori bla                                                                                    Novagen <br />   pET15b-NTT4                  ntt4 inserted NdeI-BamHI site, NTT4 expression, bla                                                              This study <br />   pET15K-NTT4                  ntt4 inserted NdeI-BamHI site, NTT4 expression, kan                                                              This study <br />   pKD3                         oriR6K&#226;?&#165; bla cat rgnB(Ter)                                                                                           8 <br />   pKD4                         oriR6K&#226;?&#165; bla kan rgnB(Ter)                                                                                           8 <br />   pKD46                        araBp-gam-bet-exo bla repA101(Ts) oriR101                                                                           8 <br />  <br />  <br />  <br />                           MATERIALS AND METHODS                                     below. Briefly, reaction mixture consisted plasmid (200 ng), megaprimer <br />                                                                                     Kan (600 ng), 5&#226;&#171;&#187; PrimeSTAR buffer (10 &#226;?&#174;l), 2.5 mM deoxynucleoside triphos- <br />    Bacterial strains plasmids. Cloning plasmid propagation per- <br />                                                                                     phates (6 &#226;?&#174;l), PrimeSTAR HS DNA polymerase (1.25 U) total volume 50 <br /> formed E. coli DH5&#226;?&#163;. E. coli BW25113(pKD3, pKD4, pKD46) kindly <br />                                                                                     &#226;?&#174;l. The thermocycling conditions 95&#194;&#176;C 5 min, 35 cycles 95&#194;&#176;C 1 <br /> provided B. L. Wanner (Purdue University). E. coli JW0105 obtained <br />                                                                                     min, 55&#194;&#176;C 45 s, 68&#194;&#176;C 6 min, final 68&#194;&#176;C 20 min. The reaction <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> KEIO <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>knockout</span> collection E. coli BW25113 background <br /> (2). The strains plasmids used listed Table 1.                        mix treated DpnI 37&#194;&#176;C 4 h digest methylated parental <br />    Growth media conditions. E. coli routinely cultivated agitation     plasmid transformed E. coli DH5&#226;?&#163; competent cells. <br /> at 37&#194;&#176;C 30&#194;&#176;C (for gene disruption) 200 rpm LB broth (10 g             nadE 200-bp flanking regions amplified E. coli BW25113 <br /> tryptone [Oxoid], 5 g yeast extract [Oxoid], 10 g NaCl liter water), SOB        genomic DNA using primer pair nadE-F/nadE-R cloned into <br /> broth (20 g tryptone [Oxoid], 5 g yeast extract [Oxoid], 0.5 g NaCl, 2.5 mM         pMD18-T. Disruption cassettes nadE constructed replacing nadE <br /> KCl, 1 mM MgCl2, 1 mM MgSO4 liter water), M9 minimal medium (6               cat amplified pKD3 primer pair cat(E)-F/cat(E)-R, resulting <br /> g Na2HPO4, 3 g KH2PO4, 0.5 g NaCl, 1 g NH4Cl, 1 mM MgSO4, 0.1 mM                    pMD18T-nadE::CAT. <br /> CaCl2 liter water) containing 2% glucose, 1,000&#226;&#171;&#187; trace metal mix (27           Deletion nadE performed described previously (8) minor <br /> g FeCl3 &#228;&#161;&#160; 6H2O, 2 g ZnCl2 &#228;&#161;&#160; 4H2O, 2 g CaCl2 &#228;&#161;&#160; 2H2O, 2 g Na2MoO4 &#228;&#161;&#160; 2H2O,             modification. The nadE disruption cassette nadE::CAT PCR amplified from <br /> 1.9 g CuSO4 &#228;&#161;&#160; 5H2O, 0.5 g H3BO3 liter water). The strains inocu-           pMD18T-nadE::CAT prime pair nadE-F/nadE-R transformed <br /> lated overnight cultures 37&#194;&#176;C, washed 2 times M9 medium,           E. coli YJE002 electroporation. The mutants selected SOB <br /> transferred 50 ml M9 medium supplemented NAD(H)                  plates supplemented kanamycin sulfate (50 &#226;?&#174;g/ml), chloramphenicol (30 <br /> optical density 600 nm (OD600) 0.02. Samples regularly taken         &#226;?&#174;g/ml) NAD&#226;&#171;&#185; (100 &#226;?&#174;M). <br /> measurement OD600 NAD(H).                                                   Cofactor measurements. Cofactor recycling assay used determine in- <br />    Reagents. Pfu DNA polymerase supplied Tiangen Biotech (Beijing,           tracellular concentrations NAD(H) NADP(H) minor modifications <br /> China). All primers used study (Table 2) purchased Invitrogen     (7, 22). For assay, 5 &#226;&#171;&#187; 109 cells collected centrifugation at <br /> (Shanghai, China), DNA sequencing performed TaKaRa (Dalian,              8,000 &#226;&#171;&#187; g 2 min cell pellets washed twice ice-cold phos- <br /> China). A DNA gel purification kit plasmid extraction kit purchased        phate-buffered saline (PBS) buffer. Then, 300 &#226;?&#174;l 0.2 M NaOH [for NAD(P)H <br /> from Beyotime (Haimen, China). DpnI purchased New England Bio-             extraction] HCl [for NAD(P)&#226;&#171;&#185; extraction] added cell pellets, <br /> Labs (Beijing, China), PrimeSTAR HS DNA polymerase                heated 10 min 55&#194;&#176;C. The extracts neutralized by <br /> reagents purchased TaKaRa (Dalian, China). All chemicals             adding 300 &#226;?&#174;l 0.1 M HCl [for NAD(P)H extraction] NaOH [for <br /> purchased Sigma (Shanghai, China).                                             NAD(P)&#226;&#171;&#185; extraction]. The cellular debris removed centrifuging at <br />    DNA manipulation. The ntt4 gene (11) custom synthesized TaKaRa            12,000 &#226;&#171;&#187; g 5 min. Supernatants transferred new tubes stored <br /> (Dalian, China) cloned NdeI-BamHI site pET15b         &#226;&#171;&#186;20&#194;&#176;C 24 h. <br /> primer pair NTT4-F/NTT4-R, resulting NTT4 expression vector pET15b-             The cycling assay performed using reagent mixture consisting equal <br /> NTT4. pET15K-NTT4 constructed substituting bla gene kan             volumes 1.0 M Bicine buffer (pH 8.0), absolute ethanol [for NAD(H) <br /> amplified Kan-F/Kan-R, using restriction-free (RF) cloning strategy        assay] 30 mM glucose-6-phosphate [for NADP(H) assay], 40 mM EDTA <br /> (32). The RF cloning experiment according procedure described       (pH 8.0), 4.2 mM 3-(4,5-dimethyl-2-thiazolyl)-2,5-diphenyl-2H-tetrazolium bro- <br />  <br />  <br />  <br />                                                             TABLE 2. Primers used study <br />  Primer                                                             Sequence (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;)a                                                                 Function <br />  <br /> NTT4-F                GAGCATATGAGTAAAACAAACCAG                                                                                                   ntt4 cloning <br /> NTT4-R                TACGGATCCTTAGTGATGATGATGATGATGTTTTTTTATAAAAG <br /> Kan-F                 TTAAACATGCCAGTGATGCAAAGGTAGTGCAAGAGCTATGACATATACCTGCCGTTCAC                                                                kan cloning <br /> Kan-R                 TCAGGCGGTCAGTGTATCATCACTCATACTCTGCCCGACACCATGGTCATAGCTGTTTC <br /> nadE-F                GTTAATACCGCCGCTGCACTCAG                                                                                                    nadE cloning <br /> nadE-R                TCACCATGAAACAGATACACCCCTG <br /> cat(E)-F              TGTCTTTTCTGTCTGGAGGGGTTCAATGATGGGAATTAGCCATGGTCC                                                                           cat amplification <br /> cat(E)-R              CAAATTATTACTTTTTCCAGAAATCATCGTGTAGGCTGGAGCTGCTTC <br />   a <br />       The restriction sites underlined. <br />  6136       ZHOU ET AL.                                                                                                       APPL. ENVIRON. MICROBIOL. <br />  <br /> mide, twice volume 16.6 mM phenazine ethosulfate, 3 volumes of <br /> water, previously incubated 25&#194;&#176;C. The following volumes added 1-ml <br /> cuvettes: 50 &#226;?&#174;l neutralized extract, 0.9 ml reagent mixture. For NAD(H) <br /> assay, reaction started adding 50 &#226;?&#174;l yeast ADH II (500 U/ml in <br /> Bicine buffer). Fifty microliters yeast glucose-6-phosphate dehydrogenase (10 <br /> U/ml) added start NADP(H) assay. The absorbance 570 nm was <br /> recorded 2 min 25&#194;&#176;C. The volume cell 10&#226;&#171;&#186;12 ml, cell <br /> concentrations calculated based assumption OD600 1.0 <br /> corresponds 109 cells/ml (5); thus, cell density correlated cell <br /> volume, 1 ml/liter equivalent OD600 1. <br />    Determination minimal NAD(H) level. The colonies NAD&#226;&#171;&#185;-auxo- <br /> trophic mutant YJE003 picked LB medium containing 100 &#226;?&#174;M NAD&#226;&#171;&#185; <br /> and cultivated 12 h agitation 37&#194;&#176;C, 200 rpm. Then, cells were <br /> inoculated 100 ml M9 medium supplemented 100 &#226;?&#174;M NAD&#226;&#171;&#185; an <br /> OD600 0.02 cultivated early exponential phase (OD600, &#226;&#172;&#8225;0.8). The <br /> cells collected washed twice M9 medium inoculated into <br /> 50 ml M9 medium, initial OD600 0.1, 0.2, 0.3, respectively, <br /> and cultivated stationary phase, cofactor level minimal <br /> level E. coli growth. The cells collected washed ice-cold PBS <br /> buffer used intracellular NAD(H) measurements. <br />    Determination maximal NAD(H) level. The NAD&#226;&#171;&#185;-auxotrophic mu- <br /> tant YJE003 picked LB medium containing 100 &#226;?&#174;M NAD&#226;&#171;&#185; and <br /> cultivated 12 h agitation 37&#194;&#176;C, 200 rpm. The cells collected <br /> and washed twice M9 medium, inoculated 50 ml M9 medium <br /> supplemented different concentrations NAD&#226;&#171;&#185;, initial OD600 <br /> of 0.02 cultivated agitation 37&#194;&#176;C, 200 rpm stationary phase. The <br /> cells used intracellular NAD(H) measurements washing twice <br /> with ice-cold PBS buffer. <br />  <br />  <br />                                   RESULTS <br />                                                                                        FIG. 2. Verification function NTT4 &#226;&#338;&#172;nadC mutant. <br />    Verification NTT4 function &#226;&#338;&#172;nadC mutant.                                (A) NTT4 function verified expressing E. coli JW0105, <br /> NAD(H) se impermeable bacterial cytoplasmic                            lacks novo NAD&#226;&#171;&#185; biosynthesis pathway nadC <br />                                                                                      disruption. (B) NTT4-expressing strain YJE001 better growth than <br /> membrane (1, 21), usually intermediates,                         control strain cultivated M9 medium containing 100 &#226;?&#174;M <br /> as AMP, NMN, NA, directly assimilated. Recently,                          NAD&#226;&#171;&#185;. <br /> the NAD&#226;&#171;&#185; transporter, NTT4, discovered chlamyd- <br /> ial endosymbiont P. amoebophila UWE25 (11). The properties <br /> of transporter studied cloning ntt4 gene E.                    BW25113 obtained disruption nadE gene obtain <br /> coli analyzing transportation activity 32P-labeled                      NAD&#226;&#171;&#185;-auxotrophic mutant YJE003. The mutant strain <br /> nucleotides cells resting state. However, remains                      grew LB plates supplemented NAD&#226;&#171;&#185; or <br /> unknown transportation activity NTT4 suf-                         NADH failed grow plates NAD&#226;&#171;&#185; intermediates, <br /> ficient support cell growth exogenous NAD&#226;&#171;&#185;. We con-                            NA NMN (Fig. 3B). <br /> structed E. coli YJE001, ntt4 gene expressed                           Comparison cofactor levels exogenous <br /> E. coli JW0105, strain lacks novo NAD&#226;&#171;&#185; biosyn-                         NAD&#226;&#180;&#8482;- NADH-fed cells. A previous study showed that <br /> thesis pathway nadC disruption (Fig. 2A). It                        NTT4 transport NAD&#226;&#171;&#185; NADH the <br /> that YJE001 grew 20% faster exponential phase                        NADH uptake rate little lower resting cells (11), but <br /> the presence NAD&#226;&#171;&#185; medium (Fig. 2B),                              growing cells NTT4 transportation activity differences <br /> wild-type BW25113 cells insensitive exogenous NAD&#226;&#171;&#185;                           NAD&#226;&#171;&#185; NADH remained undetermined. There- <br /> (data shown), suggesting NTT4 protein func-                         fore, examined effects concentration exoge- <br /> tional NAD&#226;&#171;&#185; transporter E. coli. We surmised                       nous NAD&#226;&#171;&#185; NADH intracellular cofactor levels as <br /> activity NTT4 able rescue cells NAD&#226;&#171;&#185;                         cell growth profile. It turned cells grew <br /> biosynthesis completely blocked. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> test conjecture,                        faster fed NAD&#226;&#171;&#185;, instead NADH. In cells fed NAD&#226;&#171;&#185;, <br /> we decided disrupt essential gene nadE, in-                          total intracellular NAD(H) levels 4.82 mM 14 h and <br /> volved step NAD&#226;&#171;&#185; biosynthesis.                                        7.55 mM 26 h, higher levels those <br />    Construction NAD&#226;&#180;&#8482;-auxotrophic mutant nadE dis-                          fed NADH (Table 3). These results sug- <br /> ruption. In order manipulate intracellular NAD&#226;&#171;&#185; level,                        gested NTT4 prefers NAD&#226;&#171;&#185; NADH, con- <br /> we attempted remove NAD&#226;&#171;&#185; regulation effect dis-                            sistent results observed using resting cells (11). It was <br /> rupting essential gene nadE, responsible                        NAD&#226;&#171;&#185;/NADH ratio higher NAD&#226;&#171;&#185;- <br /> last step NAD&#226;&#171;&#185; biosynthesis. However, impractical                        fed cells NADH-fed cells, indicating NAD&#226;&#171;&#185;-fed <br /> if cells did access exogenous NAD&#226;&#171;&#185; (2). As                           cells higher oxidative state. <br /> &#226;&#338;&#172;nadC mutant carrying ntt4 gene showed advantageous                              Determination minimal cofactor level E. coli cell <br /> growth profile presence exogenous NAD&#226;&#171;&#185;,                                 proliferation. Many studies successfully perturbed the <br /> thought possible construct NAD&#226;&#171;&#185;-auxotro-                           intracellular cofactor level, especially increasing NAD&#226;&#171;&#185; <br /> phic mutant NTT4 expression background gener-                             level (34) manipulation NAD&#226;&#171;&#185;/NADH ratio (22), <br /> ated (Fig. 3A). Thus, expressed ntt4 gene E. coli                          difficult determine minimal cofactor level for <br />  VOL. 77, 2011                                                                   EXTREMES OF THE CELLULAR NAD(H) LEVEL                           6137 <br />  <br />  <br />  <br />  <br />    FIG. 3. Construction characterization NAD&#226;&#171;&#185;-auxotro- <br /> phic mutant YJE003. (A) NAD&#226;&#171;&#185;-auxotrophic mutants con- <br /> structed disrupting nadE gene NTT4 expression back- <br /> ground. (B) <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>Phenotype</span> NAD&#226;&#171;&#185;-auxotrophic mutant YJE003. <br /> About 109 overnight cultured cells suspended 1 ml double- <br /> distilled H2O (OD600, 0.1), 10-&#226;?&#174;l aliquots dilutions 10&#226;&#171;&#186;1        FIG. 4. Determination minimal cofactor level cell <br /> 10&#226;&#171;&#186;6 spotted corresponding plates (from left right).          growth, based NAD&#226;&#171;&#185; distribution strategy NAD&#226;&#171;&#185;- <br />                                                                              auxotrophic mutant YJE003. (A) Schematic NAD&#226;&#171;&#185; distribution <br />                                                                              strategy. The auxotrophic mutant YJE003 preloaded certain <br />                                                                              NAD&#226;&#171;&#185; cultivating M9 medium containing 100 &#226;?&#174;M <br />                                                                              NAD&#226;&#171;&#185; early log phase (OD600, &#226;&#172;&#8225;0.8), washed twice M9 me- <br /> cell proliferation. This NAD&#226;&#171;&#185; level regulated <br />                                                                              dium, transferred M9 medium NAD&#226;&#171;&#185;. The cells <br /> by NAD&#226;&#171;&#185; biosynthesis pathways, activated supply                 proliferated, cofactor distributed daughter cells until <br /> NAD&#226;&#171;&#185; NAD&#226;&#171;&#185; level low. With NAD&#226;&#171;&#185;-auxo-                         cellular cofactor limited cell division. (B) Growth curve of <br /> trophic mutant, minimal cofactor level cell growth                   YJE003 M9 medium NAD&#226;&#171;&#185; different inoculation <br /> could determined based NAD&#226;&#171;&#185; distribution strategy                  densities (0.1 0.3). (C) Intracellular cofactor concentrations were <br />                                                                              measured cells reached stationary phase. The data <br /> (Fig. 4A), inspired plasmid distribution                    represent averages &#226;&#171;&#190; standard deviations three <br /> model (31). Briefly, auxotrophic mutant YJE003 pre-                  independent samples. <br /> loaded certain NAD&#226;&#171;&#185; transferred <br /> into minimal medium NAD&#226;&#171;&#185;. The cells divided and <br /> the cofactor distributed daughter cells cel-              0.3, maximal OD600 1.7 (Fig. 4B), was <br /> lular cofactor level reached minimum limited cell                   lower OD600 2.7 obtained YJE003 cells <br /> division. It interesting matter initial              cultivated presence 100 &#226;?&#174;M NAD&#226;&#171;&#185; (Fig. 5A). <br /> OD600 0.1, 0.2, 0.3, YJE003 cells reached stationary              These observations suggested cofactor level in- <br /> phase cultivation 4 h, cell density                  deed limiting cell division. <br /> increased 4-fold. For example, initial OD600                    As cell density YJE003 increased 4-fold M9 me- <br />  <br />  <br />                             TABLE 3. Intracellular NAD(H) levels E. coli YJE003 cells fed 100 &#226;?&#174;M NAD&#226;&#171;&#185; NADH <br />                                                                            Intracellular levela (mM) <br />  Feed                   Duration (h) feeding                                                                                   NAD&#226;&#171;&#185;/NADH ratio <br />                                                           NAD&#226;&#171;&#185;                     NADH                    NAD(H) <br />  <br /> NAD&#226;&#171;&#185;                              14                   4.26 &#226;&#171;&#190; 0.55               0.56 &#226;&#171;&#190; 0.09             4.82 &#226;&#171;&#190; 0.62                     7.66 <br />                                   26                   6.23 &#226;&#171;&#190; 0.13               1.32 &#226;&#171;&#190; 0.10             7.55 &#226;&#171;&#190; 0.17                     4.72 <br />  <br /> NADH                              14                   2.21 &#226;&#171;&#190; 0.21               0.41 &#226;&#171;&#190; 0.12             2.62 &#226;&#171;&#190; 0.32                     5.33 <br />                                   26                   2.46 &#226;&#171;&#190; 0.06               0.68 &#226;&#171;&#190; 0.04             3.14 &#226;&#171;&#190; 0.09                     3.63 <br />   a <br />       Values means &#226;&#171;&#190; standard deviations. <br />  6138      ZHOU ET AL.                                                                                                      APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />                                                                              &#226;?&#174;M, respectively. The total NAD(H) level reached ex- <br />                                                                              tremely high value 8.49 mM initial feeding NAD&#226;&#171;&#185; <br />                                                                              concentration 1 mM (Fig. 5B), 9.6-fold of <br />                                                                              wild-type cells. <br />  <br />  <br />                                                                                                            DISCUSSION <br />                                                                                 In previous study, NTT4 chlamydial P. amoebophila <br />                                                                              UWE25 expressed E. coli characterized an <br />                                                                              NAD&#226;&#171;&#185; transporter incubating resting recombinant E. <br />                                                                              coli cell 32P-labeled nucleotides (11), means to <br />                                                                              determine effect transport NAD&#226;&#171;&#185; growing cells <br />                                                                              remained unavailable. In present study, confirmed <br />                                                                              function NTT4 E. coli disrupting nadC gene, <br />                                                                              responsible novo biosynthesis NAD&#226;&#171;&#185; <br />                                                                              (Fig. 2). We showed NAD&#226;&#171;&#185; transportation <br />                                                                              activity NTT4 support cell growth fully, depending on <br />                                                                              uptake extracellular NAD&#226;&#171;&#185; (Fig. 3). It known that <br />                                                                              disruption essential genes nadD nadE prohibitive <br />                                                                              (2, 25), successful used E. coli strain that <br />    FIG. 5. Manipulation intracellular NAD(H) level dif-          carried ntt4 gene (Fig. 3B). Therefore, strategy could <br /> ferent exogenous NAD&#226;&#171;&#185; feeding levels. (A) Growth curve YJE003             explored alternative identification NAD&#226;&#171;&#185; <br /> cultivated M9 medium containing different concentrations               transporter utilization radioactive nucleotides. It <br /> NAD&#226;&#171;&#185;. &#226;?&#174; growth rate (h&#226;&#171;&#186;1), calculated the <br /> linear slopes logarithmic plots growth curves. (B) Intracellular   used identify characterize NAD&#226;&#171;&#185; <br /> NAD(H) concentrations measured 24 h cells reached                biosynthetic pathways, recently discovered in <br /> stationary phase. The data represent averages &#226;&#171;&#190; standard devia-          Francisella tularensis (24). ATP essential energy cofactor <br /> tions independent samples.                                determines energy metabolism metabolic flux <br />                                                                              (17). Several ATP transporters identified het- <br />                                                                              erologously expressed E. coli (27, 29, 30). The extremes of <br /> dium NAD&#226;&#171;&#185; supply, estimated total cofactor                 cellular ATP levels determined using similar <br /> level reduced 20% parent                  strategy. The impacts extreme cofactor levels the <br /> cells. The results cofactor measurements consistent              metabolism, transcriptome, interactions cofactor <br /> with expectations. The total NAD(H) NADP(H) level                    enzyme (35) investigated future, these <br /> was 0.12 mM, 20% parent cells                    studies new insights microbial cell biology. <br /> (Fig. 4C) 10% wild-type BW25113 cells.                     The level NAD tightly controlled variety mech- <br /> The NAD(H) level 0.04 mM, 4.4%                      anisms, including biosynthesis salvage. Studies have <br /> that wild-type cells (Fig. 4C Table 4). The cellular              successfully manipulated NAD&#226;&#171;&#185; level overexpressing <br /> NADP(H) level 0.08 mM, 50% 20%                       pncB changing activity NAMN adenylyltransferase by <br /> that parent cells wild-type cells, respectively               constructing nadD mutant library (25, 26). However, the <br /> (Fig. 4C). It pointed NAD(H) levels                extremes NAD(H) level, i.e., minimal maximal <br /> the wild-type cells consistent previous report (9),              level, remain undetermined live E. coli cells. We con- <br /> indicating data accurate.                                      structed NAD&#226;&#171;&#185;-auxotrophic mutant NAD&#226;&#171;&#185; bio- <br />    Determination maximal cofactor level live E. coli               synthesis completely blocked, designed NAD&#226;&#171;&#185; <br /> cells. With NAD&#226;&#171;&#185;-auxotrophic mutant, minimal cofac-                  distribution strategy determine minimal NAD&#226;&#171;&#185; level for <br /> tor determined cell division strategy. However,                 cell growth (Fig. 4). With mutant strain YJE003, we <br /> does NAD(H) level change response concentra-                   showed minimal NAD(H) level 4.4% of <br /> tion exogenous NAD&#226;&#171;&#185;? Is maximal NAD(H) level                      wild-type cells. It demonstrated 20% total <br /> that live E. coli cells uphold? In order answer                 cellular NAD(H) wild-type cells freely accessible the <br /> questions, cultivated YJE003 cells presence of <br /> NAD&#226;&#171;&#185; different concentrations (10 &#226;?&#174;M 10 mM). The lag <br /> phase cells decreased increases exogenous                 TABLE 4. Intracellular NAD(H) levels NAD&#226;&#171;&#185;-auxotrophic <br /> NAD&#226;&#171;&#185; concentration, growth rate exponential                               mutant YJE003 wild-type E. coli <br /> phase remained (5.1 5.3 h&#226;&#171;&#186;1). This suggested that <br />                                                                                                                   NAD(H) level (mM)a <br /> the higher NAD&#226;&#171;&#185; transportation rate resulting higher                  Cofactor <br /> exogenous NAD&#226;&#171;&#185; concentration beneficial early                                        Maximal                  Minimal          Wild type <br /> stage cell growth advantages exponential                   NAD&#226;&#171;&#185;              7.50 &#226;&#171;&#190; 0.10           0.019 &#226;&#171;&#190; 0.002        0.64 &#226;&#171;&#190; 0.04 <br /> growth (Fig. 5A). The intracellular NAD(H) levels increased                  NADH              0.99 &#226;&#171;&#190; 0.15           0.020 &#226;&#171;&#190; 0.004        0.24 &#226;&#171;&#190; 0.01 <br /> with increase initial exogenous NAD&#226;&#171;&#185; concentration <br /> and reached maxima 7.50 mM NAD&#226;&#171;&#185; 1.51 mM NADH                          Total             8.49 &#226;&#171;&#190; 0.24           0.039 &#226;&#171;&#190; 0.002        0.88 &#226;&#171;&#190; 0.02 <br /> with initial feeding NAD&#226;&#171;&#185; concentration 1 mM 60                     a <br />                                                                                   Data means &#226;&#171;&#190; standard deviations. <br />  VOL. 77, 2011                                                             EXTREMES OF THE CELLULAR NAD(H) LEVEL                                     6139 <br />  <br />  <br /> majority (80%) bound enzymes (35). The fact             thank National BioResource Project (NIG, Japan) providing <br /> minimal NAD(H) level mutant strain far lower           &#226;&#338;&#172;nadC mutant JW0106 KEIO Collection Barry L. Wan- <br />                                                                        ner (Purdue University) providing E. coli BW25113(pKD3, pKD4, <br /> the freely accessible NAD(H) level wild-type cells suggests         pKD46). <br /> that binding affinity NAD(H) enzymes should <br /> be higher observed previously. Moreover, min-                                             REFERENCES <br /> imal cellular NADP(H) level 2-fold minimal              1. Adler, L. W., B. P. Rosen. 1977. Functional mosaicism membrane <br /> NAD(H) level, NADP(H) level lower                   proteins vesicles Escherichia coli. J. Bacteriol. 129:959&#226;&#8364;&#8220;966. <br />                                                                         2. Baba, T., et al. 2006. Construction Escherichia coli K-12 in-frame, single- <br /> NAD(H) level wild-type cells (Fig. 4C), indicating E.              <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants: Keio collection. Mol. Syst. Biol. 2:2006.0008. <br /> coli cells need NADP(H) biosynthesis biomass                3. Begley, T. P., C. Kinsland, R. A. Mehl, A. Osterman, P. Dorrestein. 2001. <br /> precursors cofactor starvation state, NADP(H)                The biosynthesis nicotinamide adenine dinucleotides bacteria. Vitam. <br />                                                                            Horm. 61:103&#226;&#8364;&#8220;119. <br /> the main electron carrier anabolism (4).                             4. Blank, L. M., B. E. Ebert, K. Buehler, B. Buhler. 2010. Redox bioca- <br />    In studies, cellular cofactor concentration             talysis metabolism: molecular mechanisms metabolic network anal- <br /> been increased manipulating NAD&#226;&#171;&#185; biosynthetic                  ysis. Antioxid. Redox Signal. 13:349&#226;&#8364;&#8220;394. <br />                                                                         5. Brumaghim, J. L., Y. Li, E. Henle, S. Linn. 2003. Effects hydrogen <br /> pathway (12, 16) feeding permeabilized cells (18, 36, 37).              peroxide nicotinamide nucleotide metabolism Escherichia coli: <br /> The NAD(H) level increased 6-fold previously (12).               changes enzyme levels nicotinamide nucleotide pools studies of <br />                                                                            oxidation NAD(P)H Fe(III). J. Biol. Chem. 278:42495&#226;&#8364;&#8220;42504. <br /> Yet, maximal NAD(H) level live cells               6. Buhler, B., J. B. Park, L. M. Blank, A. Schmid. 2008. NADH availability <br /> tolerate? To address fundamental question, increased               limits asymmetric biocatalytic epoxidation growing recombinant Esche- <br /> the intracellular NAD(H) level exposing NAD&#226;&#171;&#185;-auxo-                  richia coli strain. Appl. Environ. Microbiol. 74:1436&#226;&#8364;&#8220;1446. <br />                                                                         7. Chemler, J. A., Z. L. Fowler, K. P. McHugh, M. A. Koffas. 2009. <br /> trophic mutant YJE003 cells higher exogenous NAD&#226;&#171;&#185; con-                  Improving NADPH availability natural product biosynthesis Esche- <br /> centrations M9 medium. We observed 9.6-fold in-                   richia coli metabolic engineering. Metab. Eng. 12:96&#226;&#8364;&#8220;104. <br /> crease NAD(H) level compared wild-type            8. Datsenko, K. A., B. L. Wanner. 2000. One-step inactivation chromo- <br />                                                                            somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br /> BW25113 cells presence 1 mM exogenous NAD&#226;&#171;&#185;.                      Sci. U. S. A. 97:6640&#226;&#8364;&#8220;6645. <br /> The intracellular NAD(H) level did increase            9. Dhamdhere, G., H. I. Zgurskaya. 2010. Metabolic shutdown Esche- <br /> exogenous NAD&#226;&#171;&#185; concentrations 10 mM, indicating                 richia coli cells lacking outer membrane channel TolC. Mol. Microbiol. <br />                                                                            77:743&#226;&#8364;&#8220;754. <br /> the maximal NAD(H) level E. coli 8.49 mM (Fig.           10. Foster, J. W., Y. K. Park, T. Penfound, T. Fenger, M. P. Spector. 1990. <br /> 5B). In wild-type cells, NAD&#226;&#171;&#185; biosynthesis repressed            Regulation NAD metabolism Salmonella typhimurium: molecular se- <br />                                                                            quence analysis bifunctional nadR regulator nadA-pnuC <br /> salvage NAD&#226;&#171;&#185; degradation activated intracel-               operon. J. Bacteriol. 172:4187&#226;&#8364;&#8220;4196. <br /> lular NAD(H) level high. In contrast, NAD&#226;&#171;&#185; transportation           11. Haferkamp, I., et al. 2004. A candidate NAD&#226;&#171;&#185; transporter intracellular <br /> in mutant strain YJE003 likely independent                  bacterial symbiont related chlamydiae. Nature 432:622&#226;&#8364;&#8220;625. <br />                                                                        12. Heuser, F., K. Schroer, S. Lutz, S. Bringer-Meyer, H. Sahm. 2007. <br /> biosynthesis regulation mechanism, leading accumulation              Enhancement NAD(P)(H) pool Escherichia coli biotransforma- <br /> intracellular NAD(H) high level.                                 tion. Eng. Life Sci. 7:343&#226;&#8364;&#8220;353. <br />                                                                        13. Holm, A. K., et al. 2010. Metabolic transcriptional response cofactor <br />    Cellular cofactor concentration important whole-cell              perturbations Escherichia coli. J. Biol. Chem. 285:17498&#226;&#8364;&#8220;17506. <br /> redox biocatalysis (4, 6), determines biocat-        14. Hou, J., G. Scalcinati, M. Oldiges, G. N. Vemuri. 2010. Metabolic impact <br /> alyst operates effectively given condition. If cellular        increased NADH availability Saccharomyces cerevisiae. Appl. Environ. <br />                                                                            Microbiol. 76:851&#226;&#8364;&#8220;859. <br /> NAD(H) concentration lower Km, biocatalyst          15. Hove-Jensen, B. 1996. Phosphoribosyl diphosphate synthetase-independent <br /> not working optimal condition, based as-                  NAD&#226;&#171;&#185; novo synthesis Escherichia coli: new phenotype phosphate <br /> sumption apparent kinetics NAD(H) follows                  regulon mutants. J. Bacteriol. 178:714&#226;&#8364;&#8220;722. <br />                                                                        16. Knepper, A., M. Schleicher, M. Klauke, M. Weuster-Botz. 2008. En- <br /> classic rate equation V &#226;&#171;&#189; Vmax [NAD(H)]/(Km &#226;&#171;&#185; [NAD(H)]).                   hancement NAD(P)(H) pool Saccharomyces cerevisiae. Eng. Life <br /> Therefore, determination apparent kinetics NAD(H)                Sci. 8:381&#226;&#8364;&#8220;389. <br />                                                                        17. Koebmann, B. J., H. V. Westerhoff, J. L. Snoep, D. Nilsson, P. R. Jensen. <br /> is necessary optimizing designing biocatalysts. It             2002. The glycolytic flux Escherichia coli controlled demand for <br /> been difficult acquire apparent kinetics data using                ATP. J. Bacteriol. 184:3909&#226;&#8364;&#8220;3916. <br /> wild-type cells, self-regulation mechanism limits          18. Kratzer, R., et al. 2011. Enzyme identification development whole- <br />                                                                            cell biotransformation asymmetric reduction o-chloroacetophenone. <br /> the fluctuation cellular cofactor <br /> </body></html>