<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                    INVESTIGATION <br />  <br />  <br />  <br />  <br />                                                   Probing Boundaries Orthology: The <br />                                                            Unanticipated Rapid Evolution of <br />                                                                     Drosophila centrosomin <br />                                                                                                         Robert C. Eisman Thomas C. Kaufman1 <br />                                                                                        Department Biology, Indiana University, Bloomington, Indiana 47405 <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> The rapid evolution essential developmental genes protein products intriguing problematic. The <br /> rapid evolution gene products simple protein folds lack well-characterized functional domains typically result low <br /> discovery rate orthologous genes. Additionally, absence orthologs dif&#239;&#172;?cult study processes mechanisms <br /> underlying rapid evolution. In study, investigated rapid evolution centrosomin (cnn), essential gene encoding <br /> centrosomal protein isoforms required syncytial development Drosophila melanogaster. Until recently rapid divergence of <br /> cnn identi&#239;&#172;?cation orthologs dif&#239;&#172;?cult questionable Cnn violates assumptions underlying models for <br /> protein evolution. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> overcome limitations, identi&#239;&#172;?ed group insect orthologs present conserved features likely <br /> to required functions attributed cnn D. melanogaster. We rapid divergence Cnn isoforms is <br /> apparently frequent coding sequence indels accelerated rate intronic additions eliminations. These changes <br /> appear buffered multi-exon multi-reading frame maximum potential ORFs, simple protein folds, splicing <br /> machinery. These buffering features occur genes Drosophila help prevent potentially deleterious mutations <br /> due indels genes large coding exons exon-dense regions separated small introns. This work promises useful for <br /> future investigations cnn potentially rapidly evolving genes proteins. <br />  <br />  <br />  <br />  <br /> A     S genomic era advances, molecular biology has <br />       shifted single-gene studies global analyses of <br /> genomes broad phylogenetic range organisms. For <br />                                                                                        known genes model organisms, essential or- <br />                                                                                        thology assignments accurate (Hulsen et al. 2006). <br />                                                                                        The importance orthology evident the <br /> single-gene global studies, critical &#239;&#172;?rst step molec-                        number databases reporting orthologous gene sets by <br /> ular biologists accurate identi&#239;&#172;?cation orthologous                           number terms introduced relational sub- <br /> genes. The concept orthology introduced Fitch                             categories time Fitch initially introduced the <br /> differentiate orthologous genes, one-                             concept orthologous paralogous gene relationships. <br /> to-one relationship species, paralogous genes,                                 Since orthologous genes related descent and <br /> which result duplication event original ortholog                         orthology based sequence homology phylogeny, <br /> within species (Fitch 1970, 2000). Because orthologs                             process identi&#239;&#172;?cation relatively simple. This <br /> often closely related paralogs, generally as-                          certainly true genes, signi&#239;&#172;?cant subset <br /> sumed gene function likely conserved. Since                          genes problems confound process. One <br /> functional annotation uncharacterized genomes gener-                             method &#239;&#172;?nding orthologs use orthologous gene sets <br /> ally computationally based overall sequence identity                           public databases, gene sets variable <br />                                                                                        depending methods used determine orthology and <br />                                                                                        number genomes used create database <br /> Copyright &#194;&#169; 2013 Genetics Society America                                    (Altenhoff Dessimoz 2009). It shown that <br /> doi: 10.1534/genetics.113.152546 <br /> Manuscript received April 25, 2013; accepted publication May 28, 2013              gene prediction errors (Nagy et al. 2011b) confusing <br /> Supporting information available online http://www.genetics.org/lookup/suppl/    orthologs, paralogs, epaktologs, genes acquire <br /> doi:10.1534/genetics.113.152546/-/DC1. <br /> 1 <br />  Corresponding author: Indiana University, 1001 E. Third St., Bloomington, IN 47405. <br />                                                                                        conserved domain independently (Nagy et al. 2011a), all <br />  E-mail: kaufman@indiana.edu                                                           confound orthologous gene identi&#239;&#172;?cation. In Nagy <br />  <br />  <br />  <br />                                                                                                          Genetics, Vol. 194, 903&#226;&#8364;&#8220;926 August 2013        903 <br />  et al. studies errors common         characterized genes superfamily pro- <br /> enough result erroneous predictions global      teins. Several experimental studies genes do- <br /> analyses protein-domain architecture evolution. At        main fungi vertebrates gene <br /> single-gene level, problems com-        function diverges rapidly lineages, as <br /> plicated rapid evolution absence known cu-         paralogous vertebrate genes (see Discussion). <br /> rated functional domains. Although generally accepted      Additionally, D. melanogaster shown this <br /> that multi-domain proteins encoded orthologous genes          domain constitutive cnn exon long- and <br /> should share multiple domains domain architecture,           short-form families Cnn proteins (Eisman et al. 2009). <br /> and low-complexity regions used de-        However, localization short forms mitotic centro- <br /> termine orthology (Galperin Koonin 1998; Mushegian           somes mutants lacking long forms fails rescue mi- <br /> et al. 1998), criteria easily ignored molecular   totic defects (Eisman et al. 2009). This shows single <br /> data limited.                                                domain explain Cnn function domain <br />     One example issues complicate orthologous       determine orthology, but, mentioned <br /> gene identi&#239;&#172;?cation provided Drosophila melanogaster        above, cases has. <br /> centrosomin (cnn), gene function essential            While data suggest known functions D. <br /> formation functional centrosomes, organization         melanogaster cnn applicable insect <br /> actin microtubule cytoskeleton, chromosome segre-        species, claims orthology cnn vertebrate <br /> gation rapid cleavage divisions syncytial em-      genes CDK5RAP2 mmg infer function becoming <br /> bryos (Megraw et al. 1999; Vaizel-Ohayon Schejter            common literature. Additionally, Bloomington <br /> 1999; Eisman et al. 2009). Simple searches orthologous        Stock Center lists cnn ortholog CDK5RAP2 <br /> gene sets cnn public databases variable. Based      useful studying, Drosophila, human disease auto- <br /> on results OrthoDB EggNOG cnn orthologs        somal recessive primary microcephaly. Since cnn and <br /> limited insects, Homologene predicts insects     CDK5RAP2 centrosomal proteins necessary local- <br /> and vertebrate gene Cyclin Dependent Kinase5 Regula-         ization g-tubulin (Megraw et al. 2001; Fong et al. 2008), <br /> tory Activating Protein2 (CDK5RAP2), paralo-         possible experimental studies CDK5RAP2 could <br /> gous vertebrate gene myomegalin (mmg), Inparanoid            provide new insights directions future molecular <br /> has complex somewhat inconsistent mix putative          studies cnn. Alternatively, phenotypic similarities between <br /> orthologs. Sequence alignments various gene sets        mutational studies cnn CDK5RAP2 to <br /> suggest OrthoDB EggNOG best set          direct indirect effects associated loss essential <br /> functional inference D. melanogaster experimental           centrosome function. <br /> data, correct assumption?                              To differentiate possibilities, impor- <br />     There reasons explain variability   tant establish reliable cnn orthologous gene set likely to <br /> in cnn orthologous gene sets. Comparisons Cnn proteins        similar functions. As mentioned above, cnn transcrip- <br /> within genus Drosophila indicate gene           tionally complex mixed pool Cnn-encoded splice <br /> encoded proteins evolving rapidly approxi-          variants present embryonic centrosomes (Eisman et al. <br /> mately half <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Cnn proteins form simple           2009). The identi&#239;&#172;?cation splice variants dif&#239;&#172;?cult at <br /> coiled-coil domains, reliable determining      best gene models species predict single protein <br /> orthology (Tatusov et al. 1997). Additionally, D. mela-          rapid evolution cnn make ab initio exon and <br /> nogaster cnn transcriptionally complex, encoding     splicing predictions unreliable. However, D. mela- <br /> a dozen splice variants form protein families (Eis-     nogaster Cnn-PA isoform spans entire gene, iso- <br /> man et al. 2009). Possibly complexity    form suf&#239;&#172;?cient rescue mutant <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> provided <br /> rapidly evolving sequence, cnn gene models       short isoforms present (Eisman et al. 2009). Moreover, <br /> insects variable, containing possible exon   orthologous genes encoding Cnn-PA-like isoforms have <br /> the gene lacking exons. Finally,         sequence information future transcriptional studies and <br /> exception small domain (Flory et al. 2002; Sawin          identify majority protein domains necessary for <br /> et al. 2004; Zhang Megraw 2007; Fong et al. 2008),           cnn function. <br /> cnn encodes identi&#239;&#172;?ed functional protein domains.           One possible explanation fact gene model <br />     The conserved domain cnn used infer orthology      predictions cnn insects incorrect may <br /> across broad phylogenetic distances protein        nature sequence spanning coding exons <br /> superfamily predicted associated micro-        intervening introns. The general assumption the <br /> tubules (Flory et al. 2002; Venkatram et al. 2004). This         reading frame coding exon encounter stop codon <br /> domain single gene fungal insect species    adjacent intron. A recent study fre- <br /> and genes vertebrate species. Although conser-     quency in-frame stop codons &#239;&#172;?rst 40&#226;&#8364;&#8220;70 bases of <br /> vation domain predicts genes superfamily      adjacent downstream intron 40&#226;&#8364;&#8220;70%, respectively <br /> are likely functional similarities, does     (Zhang et al. 2003). However, study insect cnn <br /> necessarily itself support strong functional inference   orthologs, reading frame exon <br />  <br />  <br />  <br /> 904    R. C. Eisman T. C. Kaufman <br />  frequently extends adjacent introns,           framework functional annotation orthologs, <br /> some cases single reading frame include multiple            paralogs, epaktologs share degree homol- <br /> coding exons introns. For discussion purposes, pres-       ogy Cnn. <br /> ent concept maximum potential open reading frames <br /> (MPOs) associated coding exons. While MPO            Materials Methods <br /> most coding exons likely similar actual spliced <br /> exon, subset genes Drosophila          Identi&#239;&#172;?cation centrosomin orthologs <br /> contain longer MPOs multi-exon MPOs. These               The genus Drosophila orthologs identi&#239;&#172;?ed GBrowse <br /> extensive MPOs associated exon-dense regions con-        BLAST downloaded FlyBase (http://&#239;&#172;&#8218;ybase. <br /> taining small introns.                                            org/) taken frozen CAF1 assemblies and <br />     We type MPO associated         Drosophila pseudoobscura Release 2.0. We retrieved <br /> frequent nucleotide changes rapid accumulation         genomic sequences RNA-Seq data additional <br /> random small insertions deletions (indels). Since indels      sequenced Drosophila genomes Baylor sequenc- <br /> in protein-coding sequences considered rare             ing center modENCODE project (http://www.hgsc. <br /> genomic changes (Rokas Holland 2000) indels               bcm.tmc.edu/collaborations/insects/dros_modencode/ <br /> ranging single amino acid 20 amino acids           RNAseq_data/). These manually annotated by <br /> been used determine phylogenies (Gupta 1998; Keeling           exon-to-exon comparisons translation based on <br /> et al. 2000), apparently common random           known D. melanogaster complementary DNA (cDNA) <br /> in Cnn? The underlying assumption indels        clones available FlyBase. All non-Drosophila sequences <br /> due codon loss, rapid nucleotide sequence diver-       identi&#239;&#172;?ed orthology links FlyBase) and <br /> gence adjacent Cnn indels suggests            downloaded ENSEMBLE Bombyx mori, which <br /> nontriplet indels change reading frame. In          based full-length cDNA obtained silk <br /> study, present evidence nontriplet indels          moth genome project. Sequences annotated and <br /> buffered extended MPOs, conjunction            mapped using MacVector 12.5 (Accelrys Inc.). Electronic <br /> an unknown relaxed splicing mechanism simple protein          versions annotated sequences available on <br /> folds encoded exons. These features appear        request. <br /> make possible retain protein function normally <br />                                                                   Drosophila stocks <br /> deleterious mutations occur. <br />     This work presents detailed analysis cnn ortholo-      All &#239;&#172;&#8218;ies used study grown standard <br /> gous gene set insects. In addition domain         cornmeal&#226;&#8364;&#8220;agar medium 25&#194;&#176;. To express D. mela- <br /> mentioned above, identify additional highly con-         nogaster GFP::Cnn-PA fusion transgene embryogen- <br /> served motifs unique insects overall         esis, UASpBacNPF::GFP::Cnn-PA homozygous transgenic <br /> structure Cnn-PA conserved insect            males D. melanogaster, Drosophila simulans, Drosophila <br /> orders. While protein motifs structure conserved,         yakuba, D. pseudoobscura crossed appropriate <br /> the gene protein diverge considerably species         females homozygous pBac(3xP3-EGFPafm)::nos- <br /> and orders. The intron&#226;&#8364;&#8220;exon structure cnn similar           Gal4::(pW8-) transgene previously described (Holtzman <br /> within order signi&#239;&#172;?cantly different orders       et al. 2010). Embryos collected pBac(3xP3-EGFPafm):: <br /> investigated, changes correlated           nos-Gal4::(pW8-); w; P{w+mC=pUASP- GFP::Cnn-PA} moth- <br /> rapid evolution Cnn proteins. In addition gene archi-       ers stained Cnn, a-tubulin, DNA determine <br /> tecture changes, nucleotide substitution         effects D. melanogaster Cnn-PA chimeric protein on <br /> frequent indels associated rapid evolution        native Cnn function divergent species. <br /> Cnn-PA changes appear buffered by <br />                                                                   Cloning transformation <br /> the inherent nature sequence, simple protein folds, <br /> and potentially relaxed splicing mechanisms. While          To construct GFP::Cnn-PA fusion protein, PCR- <br /> changes buffered species,           ampli&#239;&#172;?ed EGFP (Living Colors Invitrogen) the <br /> divergence Cnn-PA dominant-negative affect            following primers: SpeGfp-59 (ACT AGT ATG GTG AGC <br /> protein function species. Finally, present         AAG GGC GAG) GfpMlu-39 (ACG CGT CTT GTA CAG <br /> concept MPOs associated coding exons. The ex-             CTC GTC CAT G). We ampli&#239;&#172;?ed Cnn-PA transcript <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> tended MPOs present cnn appear common rapidly         previously described Cnn-PA cDNA (Heuer et al. 1995) <br /> evolving genes Drosophila buffer poten-        added MluI EcoRI sites oligos using standard <br /> tially deleterious genomic changes. Additionally,         PCR techniques. PCR products cloned Topo-TA <br /> that comparative studies changes homologous            pCR2.1 vector (Invitrogen) manufacturer&#226;&#8364;&#8482;s pro- <br /> MPOs provide useful tool identi&#239;&#172;?cation relatively    tocol veri&#239;&#172;?ed sequencing. All cloning done <br /> small signi&#239;&#172;?cant changes coding regions broad     using in-gel ligation techniques (Kalvakolanu Livingston <br /> phylogenetic range. It hope provides com-      1991). PCR products digested appropriate <br /> prehensive set orthologs molecular studies cnn      enzymes subcloned pBlueskript vector. Following <br />  <br />  <br />  <br />                                                                                                    Rapid Evolution Cnn   905 <br />  digestion NotI, GFP::Cnn-PA fusion shuttled       formation parthenogenetic development in <br /> into UASpBacNPF vector, plasmid transformation         Drosophila mercatorum (Eisman Kaufman 2007). Addi- <br /> was described (Holtzman et al. 2010).                  tionally, shown long- short-splice <br />                                                                variants isoforms present development and <br /> Fixation immunostaining <br />                                                                protein families differ signi&#239;&#172;?cantly pro- <br /> Embryos collected &#239;&#172;?xed 50% heptane:50%             tein sequence level (Eisman et al. 2009). However, two <br /> Methanol/EGTA previously described (Eisman et al.           pieces evidence suggest gene protein prod- <br /> 2006). Embryos immunostained previously de-            ucts evolving rapid rate genus. First, <br /> scribed (Gorman et al. 1991). Specimens mounted           hybridization ef&#239;&#172;?ciency Cnn-PA coding sequence to <br /> on glass slides 90% glycerol 10% PBS 0.2 mM        genomic DNA Southern blots decreases rapidly within <br /> n-propyl gallate (Sigma). The following antibodies        subgenus Sophophora detectable outside the <br /> used: guinea pig (whole sera) anti-Cnn.Ex1a2 1:500          subgenus Drosophila (Figure 1, top). Second, reactivity <br /> (Megraw et al. 1999) anti-a-tubulin 84B (Matthews          original Cnn antibody, covers 67% Cnn- <br /> et al. 1989). DNA stained TOTO3 (Molecular            PA protein (Heuer et al. 1995), decreases signi&#239;&#172;?cantly <br /> Probes) &#239;&#172;?nal concentration 1:1000. All &#239;&#172;&#8218;uorescent      genus (Figure 1, bottom). <br /> secondary antibodies used 1:200 (Jackson Immu-            One possible explanation single relatively small <br /> noResearch Labs).                                              motif (discussed below) protein responsible <br /> Microscopy imaging                                         Cnn function remainder protein is <br />                                                                important diverge. Alternatively, Cnn <br /> All images captured Leica TCS confocal micro-        apparently essential syncytial development <br /> scope 63X HCX Plan Apo oil immersion objective,         spermatogenesis, role gene environ- <br /> using TCSNT software. Images Z-series range       ment relatively novel, functionally orthologous <br /> from 2.5 6 mm thick, composed 0.5- 0.9-mm-thick       genes present limited phylogenetic <br /> sections. Projected images processed assembled        range. To differentiate possibilities, we <br /> into &#239;&#172;?gures Adobe Photoshop version 7.0 software.         searched analyzed putative orthologs from <br /> MPO search, genomic sequences, gene annotation             Drosophila species insects suf&#239;&#172;?- <br />                                                                cient genomic sequence. We &#239;&#172;?rst report results our <br /> To search D. melanogaster genome MPOs, origi-       analysis genus Drosophila. <br /> nally determined far reading frame annotated       The D. melanogaster cnn-RA transcript used de- <br /> exons extended designated 59 39 ends using      termine orthology transcript spans entire gene, <br /> Release 4.2. These annotations updated con-      contains constitutive D. melanogaster cnn exons, and <br /> &#239;&#172;?rmed using recent FlyBase release (5.4). From          rescues cnn mutant phenotype species provided <br /> these results selected genes MPO          Cnn short isoforms present (Eisman et al. 2009). This <br /> extending .3000 nucleotides annotated end.           suggests cnn-RA splice variant contains critical <br /> From set selected seven genes manually anno-       domain domains necessary Cnn long-isoform function. <br /> tated sequence gene based protein isoforms      Since transcript spans entire gene, provides <br /> available FlyBase. The algorithm available request.   landmarks investigate conservation, lack thereof, of <br /> Genomic sequences FlyBase frozen CAF1        complex splicing previously described D. mela- <br /> assemblies D. pseudoobscura Release 2.0. Sequences     nogaster cnn gene (Eisman et al. 2009). Based D. <br /> were manually annotated described above. The cnn            melanogaster Cnn-PA protein identi&#239;&#172;?ed putative <br /> sequences North Carolina population D. mela-         cnn orthologs 13 Drosophila species. <br /> nogaster (Mackay et al. 2012) recovered http:// <br /> dgrp.gnets.ncsu.edu/. These annotated described     Conserved motifs identify cnn orthologs <br /> the results. Electronic versions annotated sequences    Prior identi&#239;&#172;?cation cnn orthologs reported here, <br /> are available request.                                      knew D. melanogaster Cnn-PA putative <br />                                                                leucine zippers coiled-coil regions (Heuer et al. <br /> Results                                                        1995) protein phosphorylated syn- <br />                                                                cytial development (Li Kaufman 1996). To in- <br /> The coding regions centrosomin diverge rapidly <br />                                                                vestigate rapid divergence Cnn-PA, &#239;&#172;?rst wanted to <br /> Genetic molecular analyses cnn D. melanogaster       identify conserved features protein. This done <br /> suggest multiple motifs domains con-        using identi&#239;&#172;?ed putative orthologs predicted <br /> served orthologs. Cnn proteins required     encoded proteins, compared conserved <br /> initiation PeriCentriolar Matrix (PCM) biosynthesis D.   sequences identi&#239;&#172;?ed. It noted used only <br /> melanogaster (Dobbelaere et al. 2008; Conduit Raff         orthologs high-quality genomic sequence lacked <br /> 2010), function appears conserved        large stretches ambiguous <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> calls, poly-&#226;&#8364;&#339;N&#226;&#8364;? regions <br /> the genus Drosophila Cnn required novo centro-    cnn gene, unfortunately removed closely <br />  <br />  <br />  <br /> 906    R. C. Eisman T. C. Kaufman <br />                                                                                               Figure 1 The rapid divergence cnn in <br />                                                                                              Drosophila. (Top) The coding sequence <br />                                                                                              centrosomin gene evolves rapidly <br />                                                                                              genus Drosophila shown <br />                                                                                              weak hybridization labeled <br />                                                                                              cnn-RA transcripts genomic DNA <br />                                                                                              Southern blots. The &#239;&#172;?rst seven <br />                                                                                              lanes species subgenus Soph- <br />                                                                                              ophora, strongest signals are <br />                                                                                              members melanogaster group. <br />                                                                                              The lanes species sub- <br />                                                                                              genus Drosophila, representing ex- <br />                                                                                              tent phylogenetic range <br />                                                                                              producing detectable signal. (Bottom) <br />                                                                                              The Cnn-PA protein evolves rapidly, <br />                                                                                              evidenced decreased reactivity of <br />                                                                                              anti-Cnn antibody immunostained <br />                                                                                              embryos. Cnn staining (green) strong <br />                                                                                              Dmel, Dsim, Dere, Dyak, mem- <br />                                                                                              bers melanogaster subgroup, but <br />                                                                                              weak Dpse undetect- <br />                                                                                              able Dvir using antibody <br />                                                                                              concentrations. The staining both <br />                                                                                              Dpse Dvir improved in- <br />                                                                                              creasing concentration Cnn <br />                                                                                              antibody. Microtubules shown in <br />                                                                                              red DNA blue. Dmel: D. mela- <br />                                                                                              nogaster; Dsim: D. simulans; Dere: D. <br />                                                                                              erecta; Damb: D. ambigua; Dwil: D. wil- <br />                                                                                              listoni; Dsal: D. saltans; Dana: D. ananas- <br />                                                                                              sae; Dmer: D. mercatorum; Dvir: D. <br />                                                                                              virilis; Dyak: D. yakuba; Dpse: D. <br />                                                                                              pseudoobscura. <br />  <br />  <br />  <br />  <br /> related D. simulans Drosophila sechellia species    inclusion &#226;&#8364;&#339;Centrosomin&#226;&#8364;? previous infers <br /> analyses. Any putative protein orthologs possess          Cnn function described D. melanogaster be <br /> a similar structural architecture shared motifs. While       retained proteins motif. Second, is <br /> we identi&#239;&#172;?ed features promise inter-        sequence based implies general function the <br /> esting candidates future molecular experiments,      motif consistent experimental studies. <br /> study present features likely conserved          The conserved motifs present Drosoph- <br /> in cnn orthologs.                                            ila Cnn potential orthologs. The &#239;&#172;?rst centrally located and, <br />    The &#239;&#172;?rst conserved Cnn motif D. mela-         based sequence, named SESAW motif <br /> nogaster cnn splice variants highly conserved      (Figure 2). This motif known phosphorylated at <br /> Drosophila species investigated (Figure 2). This motif   protein kinase A site (http://www.phosphopep.org/) <br /> a conserved domain superfamily proteins        and, based silico analyses, casein kinase <br /> from fungi animals amino terminus           II glycogen synthase site shaggy kinase site with <br /> these proteins. Experimental evidence shows         high probability scores phosphorylated. The sec- <br /> of proteins associated microtubule cyto-      ond Drosophila-conserved sequence named SPD <br /> skeletal and/or endomembrane system, overall       motif (Figure 3). It located near carboxy terminus of <br /> functions considerably different (see Discussion). The       protein shown phosphorylated at <br /> motif previously named Centrosomin Motif 1              putative protein kinase C site. Both motifs fall in <br /> (Zhang Megraw 2007), prefer Ken-             regions predicted helical character capable <br /> Flee Cytoskeleton motif (KFC) motif reasons. First,      forming coiled-coil. Nonetheless, coiled-coil domains <br />  <br />  <br />  <br />                                                                                                      Rapid Evolution Cnn        907 <br />  Figure 2 Conserved motifs structure Cnn-PA Drosophila. Protein alignments KFC, SESAW, SPD motifs carboxy terminus are <br /> nearly invariant genus Drosophila present long-form splice variants Cnn. Their position Cnn-PA shown aligned the <br /> secondary structure protein. Coiled-coil domains represented spirals noncoiled a-helical regions solid. Dbip: D. biarmipes; Dtak: D. <br /> takahashii; D&#239;&#172;?c: D. &#239;&#172;?cusphila; Dele: D. elegans; Dbip: D. bipectinata; Dmoj: D. mojavensis; Dgri: D. grimshawi. <br />  <br />  <br /> bound both. Based BLAST analyses motifs pres-                      tions attributable Cnn, certain will <br /> ent arthropods. The actual function                   functional orthologs. <br /> motifs subject future studies, seems <br />                                                                                Rapid changes intron&#226;&#8364;&#8220;exon structure cnn <br /> likely involved regulating Cnn function dur- <br />                                                                                orthologs MPOs <br /> ing cell cycle. <br />    The fourth conserved Cnn motif carboxy                            While conservation motifs invaluable the <br /> terminus (Figure 2) nearly invariant genus                       identi&#239;&#172;?cation potential cnn orthologs, motifs provide <br /> Drosophila. We previously shown deletion                     insights mechanism(s) underlying rapid evo- <br /> COOH motif SPD D. melanogaster results mis-                      lution gene. Since non-Drosophila proteins <br /> localization truncated Cnn region chromo-                     (discussed below) based ab initio gene models, we <br /> somes dividing nuclei centrosomes                           wanted know apparent changes gene <br /> (Eisman et al. 2009). Thus potential function                     architecture exon&#226;&#8364;&#8220;intron structure relative D. mel- <br /> both motifs restrict Cnn centrosome.                     anogaster cDNA-based model transcripts. <br />    In addition conserved motifs, Drosophila                     When initiated work Drosophila genomes <br /> Cnn-PA proteins  50% coiled-coil, arrangement                      assembled, GLEAN consensus mod- <br /> of coiled-coil regions similar D. melanogaster (Fig-                 els available individual models highly <br /> ure 2). The D. melanogaster Cnn-PA leucine zippers                       variable, manually annotated genes. To identify <br /> are conserved genus Drosophila, &#239;&#172;?rst                    individual exons, &#239;&#172;?rst identi&#239;&#172;?ed short coding &#226;&#8364;&#339;word&#226;&#8364;? <br /> third zippers absent proteins described                     based conserved motifs presumed exon <br /> here. The data predict true cnn orthologs                    sequences extended reading frame containing <br /> encode coiled-coil protein containing KFC, SESAW,                        word upstream downstream &#239;&#172;?rst stop codon. <br /> and SPD motifs, carboxy terminus similar                       We designate extended reading frames MPO of <br /> the Drosophila tail. It likely proteins                  coding exon. Each MPO aligned D. <br /> an important component centrosome, predic-                     melanogaster cnn-RA coding exons determine best esti- <br /> tion shown experimentally outside Drosophila.                    mate splicing. This method necessary since, based <br /> Proteins subset motifs share func-                    silico analysis, cnn paucity strong canonical <br />  <br />  <br />  <br /> 908     R. C. Eisman T. C. Kaufman <br />  Figure 3 Drosophila MPOs confound gene-modeling programs. Splicing predictions cnn transcripts dif&#239;&#172;?cult lack strong splicing <br /> signals, multi-exon MPOs, MPOs extend ends known splice sites. A comparison MPOs Drosophila species this <br /> study (boxed area) showing MPOs cnn reveals MPOs variable species. At MPOs D. melanogaster cnn; <br /> the multi-colored boxes exons encoding Cnn-PA (below grey boxes), gray black boxes coding exons present cnn <br /> splice variants. The MPO maps transcriptional complexity D. melanogaster cnn probably conserved genus. <br />  <br />  <br /> splicing sequences, partially explain variabil-                  To produce reasonable gene models rely <br /> ity seen individual ab initio models, appeared                 solely ab initio predictions, necessary some <br /> since started analysis. This fact combined                 independent assessment actual splicing <span id='am-84' about='xsp:pattern' typeof='owl:Thing'>pattern</span> used <br /> observation MPO domains D. melanogaster                     different species. The robust model seen D. mel- <br /> all members genus examined over-                     anogaster historically relied collection cDNAs, <br /> lapping cases contain multiple exons, extend-                   recently largely substantiated by <br /> ing regions known intronic D. melanogaster,                RNA-Seq data (McQuilton et al. 2012). Unfortunately, with <br /> made construction cogent gene model spe-                   exception single EST D. pseudoobscura, no <br /> cies dif&#239;&#172;?cult. A summary positions extent                 cDNA evidence available 13 species. Fortu- <br /> MPOs D. melanogaster 13 species shown                   nately, however, RNA-Seq data 5 13 <br /> in Figure 3.                                                                species (https://www.hgsc.bcm.edu/content/drosophila- <br />  <br />  <br />  <br />                                                                                                                     Rapid Evolution Cnn      909 <br />  Figure 4 Intronic indels common random Drosophila cnn. Cnn-PA coding exons (color coded Fig. 3) span entire cnn gene. When <br /> maps scale aligned conserved carboxy terminus, intronic indels obvious. Because frequent random indels, exon positions <br /> are variable size cnn genus. Gene sizes shown left. <br />  <br />  <br />  <br /> modencode-project). Using data, possible                     splicing cnn transcripts, available RNA-Seq data <br /> construct substantiated gene models exons encod-                     indicate species splice small intron out <br /> ing ORF corresponding Cnn-PA. With models                       mature transcript. <br /> in hand, possible infer precisely mod- <br />                                                                              Annotation cnn insects <br /> els orthologs Cnn-PA 8 species. <br />    The results analysis, shown Figure 4, reveal              We identi&#239;&#172;?ed potential cnn orthologs other <br /> within genus Drosophila overall gene architecture                 insects initially based BLAST analyses available data- <br /> apparently conserved putative coding exons                      bases. Our initial probes genomes derived <br /> similar size, introns exons accumulated                      conserved domains genus Drosophila. <br /> numerous indels resulting gene size variability (Figure                   We able unambiguously identify KFC, SESAW, and <br /> 4). Unlike RNA-Seq-substantiated deduced coding                      SPD mosquitos, hymenopterans, lepidop- <br /> exons, MPOs variable size, generally larger                  terans, Tribolium (Figure 5). <br /> the spliced exons, observed changes correlate                    It possible identify COOH terminal <br /> with phylogenetic order, suggesting sequence                        conserved domain mosquitos, Hymenoptera, and <br /> changes relatively common undirected. Much                        Lepidoptera but, domain conserved within <br /> the variability MPOs different species                     orders, peptides divergent orders. <br /> accounted small internal indels con-                      Although terminus diverges, recog- <br /> served motifs, changes outside coding                  nized BLAST Clustal analyses always <br /> regions, indels account protein size variability               contained coding exon annotated genes. <br /> (Figure 4). Additionally, exons 6 7 D. mel-                     Interestingly, COILS (Lupas et al. 1991) analyses show <br /> anogaster model reside single MPO seven                       Cnn-PA orthologs investigated similar ar- <br /> Drosophila species investigated. Thus potential exists                   rangement coiled-coil domains. Additionally, Cnn- <br /> to alternatively splice exons larger single exon              PA orthologs vary  720 amino acids Lepidoptera <br /> retaining intron exons 6 7 (Figure 3                     1400 amino acids Hymenoptera, spacing of <br /> Figure 4). However, extensive cDNA data D. melanogaster                   protein motifs proportionally constant species. The <br /> show evidence 647 intron retained                    second centrally located zipper present insect species <br />  <br />  <br />  <br /> 910     R. C. Eisman T. C. Kaufman <br />  Figure 5 Cnn-PA motifs structure conserved insects. Similar Figure 2, conserved motifs carboxy terminus aligned to <br /> the structure Cnn-PA A. mellifera. Because databases literature assert vertebrate genes CDK5RAP2 mmg are <br /> orthologous cnn, included human co-orthologs alignments. The human genes signi&#239;&#172;?cantly different structure (not shown), the <br /> carboxy terminus signi&#239;&#172;?cant homology insects, terminus divergent human paralogs insects <br /> (bottom). The SESAW SPD motifs detectable vertebrate gene. The Cnn motifs diverged orders highly conserved <br /> within orders. The carboxy terminus (CTERM) divergent conserved motifs shown alignments Hymenoptera (top) and <br /> D. melanogaster (top, line), highly conserved Hymenoptera Lepidoptera (bottom). Agam: A. gambiae; Aegy: A. egypti; Amel: <br /> A. mellifera; Bter: B. terrestris; Bimp: B. impatiens; Nvit: N. vitripennis; C&#239;&#172;&#8218;o: C. &#239;&#172;&#8218;oridanus; Aech: A. echinatior; Sinv: S. invicta; Hsal: H. salator; Bmor: B. <br /> mori; Dple: D. plexippus; Hsap; Homo sapiens. <br />  <br />  <br />  <br /> examined honeybee Apis mellifera. Thus cen-                           With exception mosquitos, Cnn-PA appar- <br /> trally located leucine zipper functionally important                    ently encoded exons relative genus Drosoph- <br /> some species, good predictor orthology.                        ila insects investigated. Although fragmented, <br />    Using conserved motifs, knowledge gene                        hymenopteran coding exons islands length <br /> structure &#239;&#172;&#8218;ies, cDNA B. mori, available gene                        gene, intronic deletions result Dro- <br /> models species annotated cnn gene                         sophila-like maps. In general, overall gene architecture is <br /> these 13 additional insects. For ant species Harpegnathos                     conserved orders considerably different among <br /> salator Solenopsis invicta altered predicted gene                      orders. The exception pattern Aedes aegypti, which <br /> models models missing &#239;&#172;?rst cod-                         additional intron relative Anopheles gambiae and <br /> ing exons present Hymenoptera models                             Drosophila species (Figure 6). Of particular are <br /> these exons clearly present genomic scaffold                          coding sequences encompassing D. melanogaster exons 6 <br /> used generate original models. In lepidopter-                      7, region contains orthologous motifs <br /> ans B. mori Danaus plexippus, region similar D.                          described exon 6 largest cnn exon D. <br /> melanogaster exon 4 apparently lost. In beetle                       melanogaster. In non-dipteran species examined, exon 7 <br /> Tribolium castaneum, exons span                          single terminal exon cases, homologous <br /> region correlated D. melanogaster exons 4 5,                         exon 6 region appears split three, four, &#239;&#172;?ve <br /> the peptide sequences exons similar                      introns, depending species (Figure 6). Unlike the <br /> of species study. The results analysis                  extensive MPOs Drosophila (Figure 2) as <br /> are shown Figure 6.                                                            mosquito A. egypti (data shown), MPOs the <br />  <br />  <br />  <br />                                                                                                                               Rapid Evolution Cnn        911 <br />  Figure 6 cnn intron&#226;&#8364;&#8220;exon structure changes signi&#239;&#172;?cantly insect orders. A comparison Cnn-PA coding exons relative D. melanogaster in <br /> the Diptera (A), Hymenoptera (B), Lepidoptera coleopteran (C) shows multiple exon fusions splitting events occurred cnn <br /> orthologs. Coding exons color-coded Figure 3 indicate sequence homology. The gray boxes T. castaneum homology other <br /> Cnn proteins, motifs structure conserved, lepidopterans exons homologous D. melanogaster exon 4. Although <br /> homologous exons split hymenopterans D. plexippus, overall arrangement coding exons similar Drosophila. As Drosophila, <br /> gene size (left) variable. <br />  <br />  <br /> species shown Figure 6, exception Hyme-                  Sequence divergence Cnn genus Drosophila <br /> noptera (discussed below), approximately co-extensive                   insects <br /> with coding exons. The presence multi-exon MPOs                   As noted above, highly conserved domains <br /> Drosophila sp. A. gambiae, species                  amino acid sequence Cnn-PA genus <br /> larger genomes, suggests multi-exon MPOs as-                    Drosophila. This stands contrast fact pro- <br /> sociated reduced genome size.                                          tein evolving rapidly. To determine nature <br />    Although ancestral intron&#226;&#8364;&#8220;exon structure cnn               rapid change, compared sequences Cnn-PA <br /> known, likely multiple exon                   dipterans represented sample (Figure 7). As <br /> fusions dipteran lineages investigated. This consis-              seen, percentage identity amino acid se- <br /> tent &#239;&#172;?nding reduction genome size                   quence diverges sharply genus Drosophila and <br /> Drosophila (Petrov Hartl 1997, 1998; Petrov et al. 2000;                dissimilar comparing Drosophila spe- <br /> Petrov 2002) A. gambiae (Nene et al. 2007)                    cies mosquitos. Members melanogaster <br /> deletions intronic intergenic regions. Since imme-               subgroup (Drosophila erecta D. yakuba) 92% iden- <br /> diate gain complete intron likely deleterious               tity D. melanogaster 95% other. Addi- <br /> effects protein function introduce premature stop                   tional evidence similarity gapping produce <br /> codon, multi-exon MPOs represent intermediate                    alignment minimal. As moves D. mel- <br /> step process exon fusion.                                     anogaster (D. biarmipis, D. takahashii, D. &#239;&#172;?cusphila, and <br />  <br />  <br />  <br /> 912     R. C. Eisman T. C. Kaufman <br />  D. elegans), identity D. melanogaster drops aver-      mellifera group ants, gaps needed to <br /> age 85% concomitant increase gapping              achieve alignment moderately high. The Hy- <br /> achieve alignment. The members ananassae               menoptera Nasonia viripenis, parasitic wasp. It the <br /> group (Drosophila ananassae Drosophila bipectinata)           divergent group 50&#226;&#8364;&#8220;60% sequence identity <br /> are diverged, showing 78% 79% identity           compared bees ants. Gapping mod- <br /> to D. melanogaster increase gapping. This trend         erately high. The sequence divergence seen Hy- <br /> continues obscura group (D. pseudoobscura) and,      menoptera magnitude seen the <br /> Drosophila willistoni, 67% 65% identity D. mela-           comparison Drosophila species belonging two <br /> nogaster, respectively, additional increment gap-      subgenera (Drosophila Sophophora). The lepidop- <br /> ping. It interesting note comparisons      teran species (B. mori D. plexippus) diverged at <br /> and aforementioned groups          similar level, showing 67% identity. When are <br /> subgenus Sophophora similar levels divergence          compared hymenopteran species, percentage of <br /> observed comparison D. melanogaster. The        identity 11% signi&#239;&#172;?cant number gaps needed <br /> members subgenus Drosophila (Drosophila mojavensis,        achieve alignment signi&#239;&#172;?cant difference in <br /> Drosophila virilis, Drosophila grimshawi) roughly     protein size. This level divergence seen compar- <br /> equivalently diverged D. melanogaster          ison single coleopteran (T. castaneum) D. mela- <br /> other range 54&#226;&#8364;&#8220;68% identity signi&#239;&#172;?cant level      nogaster. This level divergence compares species <br /> of gapping achieve alignment. Finally, comparison     different insect orders points reason dif&#239;&#172;?culty <br /> two mosquitos (A. gambiae A. aegypti) shows amino        discovering orthologs cnn reason molecular <br /> acid identity drops 25&#226;&#8364;&#8220;30% range large num-         probes designed D. melanogaster sequence rapidly loose <br /> ber gaps required &#239;&#172;?nd level alignment.         potency phylogenetic distance. Despite rapid <br /> Interestingly, neighbor-joining tree derived solely    divergence Cnn protein sequence, neighbor-joining <br /> Cnn sequence precisely tracks consensus phylogeny         tree derived sequences, case the <br /> the genus Drosophila relationship mosquitos.       Diptera, closely tracks consensus phylogeny species <br />    Since Cnn seen diverge rapidly           used analysis (Figure 9). <br /> within genus Drosophila highly            Pair-wise comparisons structure Cnn protein <br /> conserved motifs, wanted positions           models hymenopterans lepidopterans, similar to <br /> domains conservation relative diverg-       dipterans, performed, selected results <br /> ing. As noted above, Cnn-PA orthologs similar coiled-      shown Figure 10. As dipterans, coiled-coil <br /> coil domain arrangement similar spacing KFC,           structure relative positioning KFC, SESAW, SPD, <br /> SESAW, SPD, C-terminal motifs allowing pairwise           C-terminal motifs conserved A. mellifera and <br /> comparisons D. melanogaster/D. virilis, A. gambiae/       N. vitripenis lepidopterans, B. mori D. plex- <br /> A. aegypti, D. melanogaster/A. gambiae (Figure 8). The        ippus. This conservation structure independent overall <br /> KFC motif near amino terminus              protein size lepidopteran proteins half <br /> a coiled-coil domain SESAW SPD motifs           length dipteran hymenopteran examples. <br /> in helical domains &#239;&#172;&#8218;ank largely coiled-coil domain.           As Diptera, KFC motif near amino <br /> The C-terminal motif separated SPD motif           terminus coiled-coil domain, SESAW SPD are <br /> a short domain mixed coiled-coil helical character.      located COOH terminus helical domains that <br /> The regions conserved motifs            &#239;&#172;&#8218;ank coiled-coil region, C terminus separated <br /> conserved, percentage identity mirrors seen     SPD short span combined coiled-coil/helical <br /> in comparison Cnn-PA proteins. It case     character. The regions conserved motifs less <br /> that essentially gaps needed alignment      conserved level divergence and <br /> found regions conserved motifs.                gapping seen entire protein. These results clearly <br />    A similar comparison insects        essentially divergence occurs regions of <br /> the results comparison shown Figure 9. Since 8     simple coiled-coil domains helical protein folds. Finally, <br /> of 11 species considered hymenopterans, based         comparison D. melanogaster A. mellifera B. <br /> our comparisons honeybee A. mellifera. The           mori demonstrates that, despite signi&#239;&#172;?cant differences in <br /> species bee (A. mellifera, Bombus terrestris, Bombus       gene structure (exon&#226;&#8364;&#8220;intron positions), protein size, and <br /> impatiens) relatively high levels sequence identity    different interspersion helical coiled-coil domains, <br /> A. mellifera 85%. The alignment does require modest          relative disposition conserved motifs the <br /> number gaps, however. The bees genus Bombus         structure protein maintained. It shown <br /> show high level sequence identity (98%)        domain architecture conserved orthologs <br /> and require gaps achieve alignment. A comparison            high degree conservation likely be <br /> with ant species (Camponotus &#239;&#172;&#8218;oridanus, Acromyrmex           necessary functional conservation (Forslund et al. <br /> echinatior, S. invicta, H. salator) yields sequence identi-   2011). Although motifs used study are <br /> ties 60&#226;&#8364;&#8220;80% range comparison A.            curated domains, results suggest that <br />  <br />  <br />  <br />                                                                                                       Rapid Evolution Cnn   913 <br />  Figure 7 Sequence divergence frequent indels associated rapid evolution Cnn-PA Diptera. Pair-wise comparisons Cnn-PA <br /> proteins Drosophila mosquitoes (gray boxes) percentage identity protein decreases (section diagonal solid boxes) <br /> and indels accumulate (section diagonal solid boxes) relatively short periods time. A neighbor-joining tree Cnn-PA proteins <br /> showing distance based number differences gaps distributed proportionally shows molecular phylogeny cnn consistent <br /> with accepted organismal phylogeny species. Cnn-PA protein size shown species (top row) haploid genome sizes in <br /> parentheses (left column). <br />  <br />  <br /> the domain architecture cnn orthologs similar                 species. Based pairwise comparisons closely <br /> that motif required functions at-                  related species study, likely would <br /> tributed Cnn D. melanogaster.                                          minimal allelic variation cnn species. To assess <br />                                                                              evolution Cnn species, used available <br /> Sequence divergence Cnn population of <br />                                                                              sequences 162 lines single North Carolina pop- <br /> D. melanogaster <br />                                                                              ulation D. melanogaster (http://www.dpgp.org/; http:// <br /> Clearly, rapid divergence Cnn characterized                     service004.hpc.ncsu.edu/mackay/Good_Mackay_site/DBRP. <br /> sequence changes multiple insertions deletions                       html; data shown). These lines representative the <br />  <br />  <br />  <br /> 914     R. C. Eisman T. C. Kaufman <br />  Figure 8 The divergence Cnn- <br /> PA coiled-coil domains dipter- <br /> ans. A schematic representation <br /> of divergence dipteran <br /> Cnn-PA conserved motifs (black <br /> bars) coiled-coil domains <br /> (shaded bars) shows average <br /> percentage identity the <br /> number gaps needed for <br /> alignment region.       39 <br /> The least-conserved motif the <br /> carboxy terminus. The divergence <br /> graph structural models <br /> for (top pair) D. melanogaster <br /> and D. virilis, (middle pair) A. <br /> gambiae A. aegypti, D. <br /> melanogaster A. gambiae. <br /> Arrowheads indicate exon splice <br /> sites species. <br />  <br />  <br />  <br />  <br />  Rapid Evolution Cnn      915 <br />  Figure 9 A lower limit divergence number indels Cnn-PA orthologs. A table similar Figure 7 showing divergence Cnn- <br /> PA insects study reveals similar trend dipterans. Comparisons orders suggest lower limit sequence <br /> divergence upper limit number indels tolerated Cnn-PA. <br />  <br />  <br /> haploid genomes present single population                     future experimental studies required that <br /> time point (Mackay et al. 2012). Sixty lines                       true. The amino acid changes single- <br /> removed analysis ambiguous base calls                      nucleotide substitutions explain rapid <br /> or heterogeneity individual bases coding                  sequence divergence exons, as <br /> sequence Cnn-PA.                                                            Cnn-PA exon 4 region. This divergent exon <br />    We compared transcript protein sequences                     (s) orthologs; lost lepidopterans and <br /> reference, wild-type Cnn-PA sequence, FlyBase                      produces multiple gaps alignments species in <br /> found 83 variable nucleotide positions including 23 non-                       study. To investigate effect potential deletions <br /> synonymous substitutions. These changes translate 29                      North Carolina population, took sequences <br /> protein variants plus wild-type protein                     single doublet ambiguous base calls (&#226;&#8364;&#339;n&#226;&#8364;?), removed <br /> four amino acid changes Cnn-PA (Table 1). Approxi-                          sequence, translated sequence to <br /> mately two-thirds substitutions coiled-coil                      analyze resultant peptides. Surprisingly, the <br /> regions, substitutions nonsimilar amino acid                      translated peptides approximately length as <br /> residues (Figure 11). While variants pres-                    D. melanogaster reference sequence diverged <br /> ent just genomes, frequent,                    nearly 100% point deletion. We pre- <br /> and variant changes frequent                      viously reported use overlapping reading <br /> designated wild-type sequence (14 lines each).                                 frames used produce unique splice variants same <br />    These results signi&#239;&#172;?cant protein                   genomic sequence (Eisman et al. 2009). The result the <br /> variation Cnn-PA population, assume                         North Carolina sequences demonstrates exons in <br /> that alleles functionally compatible,                       cnn multiple reading frames spanning large portions of <br />  <br />  <br />  <br /> 916     R. C. Eisman T. C. Kaufman <br />  Figure 10 The divergence Cnn-PA consistent insects. A schematic representation divergence Cnn-PA, similar Figure 8, <br /> showing comparisons (top left) A. mellifera (bottom) N. vitripennis (bottom); (top right) B. mori (top) D. plexippus (bottom); (bottom <br /> left) D. melanogaster (top) A. mellifera (bottom); (bottom right) D. melanogaster (top) B. mori (bottom). The rate change the <br /> Hymenoptera Lepidoptera similar rate Diptera. The comparisons D. melanogaster lower limit divergence Cnn-PA <br /> across phylogenetic range. Arrowheads indicate exon splice sites species. <br />  <br />  <br /> the coding exon, potentially buffering nontriplet                    loss Drosophila precise imprecise genomic deletion <br /> insertions deletions coding sequence. Obviously, pro-               nonhomologous end joining (NHEJ) DNA break <br /> duce full-length transcript, splicing re-                 repair (Llopart et al. 2002; Yenerall et al. 2011). A third <br /> laxed accommodate nontriplet insertion deletion                   mechanism intron loss D. melanogaster genomic dele- <br /> an internal coding exon. The lack strong canonical splic-                 tions remove 59 intronic splice sites shorten in- <br /> ing signals cnn allow happen, ex-                    tron size capable spliced primary <br /> tensive sequencing transcripts lines                   transcript, resulting retention intron (Talerico <br /> required &#239;&#172;?rm conclusions.                                                Berget 1994). To differentiate mecha- <br />                                                                              nisms, analyzed exon 6/7 boundary Dro- <br /> A potential buffering function MPOs in <br />                                                                              sophila species known splice sites based cDNA or <br /> Drosophila cnn <br />                                                                              RNA-Seq data. <br /> The analyses consistent conclusion                      The exon 6/7 splice boundary present species in <br /> cnn diverging rapidly nucleotide substitution                  study, suggesting intron conserved, but <br /> frequent insertions deletions coding sequence. Addi-                  changes MPOs genus Drosophila (Figure 3) <br /> tionally, intron&#226;&#8364;&#8220;exon structure gene changed                  sequence spanning area divergent. <br /> considerably insect orders intron gain loss                 Consistent sequence divergence, protein sequences <br /> events. The divergent regions Cnn-PA orthologs                       &#239;&#172;&#8218;anking splice junction divergent Drosophila spe- <br /> have multiple exon fusion intron loss events Dro-                  cies. To investigate changes intron &#239;&#172;&#8218;anking cod- <br /> sophila relative orders (arrowheads, Figure 10).                    ing sequences, aligned genomic sequences the <br /> Protein alignments species study                     upstream SPD motif start downstream con- <br /> suggest short sequences &#239;&#172;&#8218;anking fusion points                   served COOH tail D. melanogaster, Drosophila biar- <br /> divergent orders continue evolve rapidly                      mipes, D. pseudoobscura. The splicing based on <br /> after fusion event orders. Global analyses Dro-                cDNA, RNA-Seq, EST data, respectively. <br /> sophila shown introns lost reverse                      The intron 61 nucleotides D. melanogaster D. <br /> transcriptase mechanism, resulting perfect excision                pseudoobscura 57 nucleotides D. biarmipes. Relative <br /> the intron (Coulombe-Huntington Majewski 2007;                           D. melanogaster, combination triplet nontriplet <br /> Yenerall et al. 2011). A rare second mechanism intron                     deletions introduce in-frame stop codon D. biarmipes, <br />  <br />  <br />  <br />                                                                                                                      Rapid Evolution Cnn       917 <br />           Table 1 Cnn-PA allelic variation population <br />  <br />          North Carolina lines             Variable nucleotide positions                Nonsynonymous positions                 Alleles protein variants <br />                    105                                     83                                        23                              29 + wild type <br />  <br />                                                   No. amino acid changes (corresponding no. alleles) <br />                  1 aa (9)                              2 aa (6)                          3 aa (11)                                       4 aa (3) <br />          The characterization allelic variation 105 unique strains single population (top row) shows total number nucleotide <br />          substitutions, number nonsynonymous substitutions, number protein variants. The number amino acid differences (one four) <br />          number allelic variants carrying differences shown the table. The difference nonsynonymous <br />          substitutions protein variants unique combinations nonsynonymous substitutions encoding total 29 protein variants. <br />  <br />  <br />  <br /> changing MPO structure (Figure 12A). Surprisingly,                               deletion (Figure 12B) MPOs long <br /> while D. melanogaster D. pseudoobscura introns                               secondary reading frames overlapping actual coding se- <br /> are equal lengths, offset 22 nucleotides                            quence. While somewhat surprising, previously <br /> respect coding sequence spliced different phases                           shown overlapping reading frames differen- <br /> (Figure 12A). Protein alignments reveal rapid divergence                             tially used generate Cnn splice variants (Eisman et al. <br /> and frequent indels &#239;&#172;&#8218;anking known splice junction be-                            2009). These results nontriplet insertion de- <br /> tween species (arrowheads, Figure 12A). The                              letion coding sequence near splice junction result <br /> rapid divergence accelerated rate non-                           rapid codon change buffered cou- <br /> synonymous substitutions sequence &#239;&#172;&#8218;anking intron,                             pled relaxed splicing mechanisms. <br /> nontriplet insertions deletions change multiple                                 The type MPO discussed appears have <br /> codons single event, combination mech-                           limited phylogenetic range restricted the <br /> anisms. While intron appears conserved                             genus Drosophila. However, did identify single multi- <br /> a broad phylogenetic range, sequence splice sites                            exon MPO parasitic wasp N. vitripennis absent <br /> evolve rapidly.                                                                      Hymenoptera. The multi-exon MPO spans the <br />    The results splicing Drosophila                            exons homologous exon 4 D. mela- <br /> evolve rapidly and, cases, relaxed compared                           nogaster Cnn-PA intron exon 4 and <br /> to canonical splicing mechanisms. Additionally,                             exon 5 (Figure 12C). In Hymenoptera, &#239;&#172;?rst two <br /> parsimonious explanation rapid protein divergence                             exons homologous exon 4 homologous exon 5 are <br /> that nontriplet deletions caused frameshifts                            conserved. The fragment exon 4 homology, exon <br /> divergent peptides actually encoded unique sets                            4c, length A. mellifera N. vitripennis, yet <br /> codons. Typically, nontriplet insertions deletion coding                       encoded peptide diverged signi&#239;&#172;?cantly (bottom, Fig- <br /> sequence predicted generate deleterious nonsense                              ure 12C). The coding sequence nontriplet inser- <br /> mutations. One possibility extended reading                              tions deletions gaps allowed (data shown) <br /> frames associated Drosophila MPOs buffer                          or, exons length, accelerated <br /> events. We analyzed Drosophila MPOs                              substitution rate (boxed, Figure 12C). Similar above <br /> that paucity stop codon dinucleotides                          example Drosophila, parsimonious explana- <br /> exonic intronic regions. A paucity stop codon dinu-                           tion rate change nontriplet insertions or <br /> cleotides small introns insect                     deletions create multiple new codons. In addition coding <br /> species. Perhaps feature relaxed splicing mecha-                            changes exon 4c, N. vitripennis intron exons <br /> nisms act buffers premature stop codons                            4c 5 large deletion, including 59 splice site <br /> intron size reduction prior fusion exons.                              (Figure 12C), suggesting splicing evolve rapidly as <br />    To investigate potential buffering capacity                        does Drosophila. <br /> MPOs, looked exon 6/7 intron North                                      Taken together, data suggest multi-exon <br /> Carolina D. melanogaster population sequences. The major-                            MPOs, conjunction relaxed splicing mechanisms, <br /> ity lines wild-type sequence, &#239;&#172;?ve lines                           buffer small potentially deleterious insertions and <br /> a 2-nt deletion, &#239;&#172;?ve lines deletion single-                           deletions, especially genes regions genes small <br /> nucleotide deletion, line deletions                             introns separating coding exons. These results show <br /> an additional single-nucleotide deletion. These genomic                              comparisons MPOs species power- <br /> sequences assembled reference D. melanogaster                            ful tool detection small, signi&#239;&#172;?cant, changes <br /> genomic sequence deletions represented &#226;&#8364;&#339;n&#226;&#8364;?                          genes. <br /> assemblies. While sequencing required to <br />                                                                                      The dominant-negative effect divergent Cnn-PA <br /> verify real deletions, fact occur <br />                                                                                      species <br /> in multiple lines suggests random sequenc- <br /> ing errors. In lines nontriplet deletions,                          The data identify Cnn-PA diverges rapidly and <br /> new reading frames extend point                               provide possible explanation processes that <br />  <br />  <br />  <br /> 918    R. C. Eisman T. C. Kaufman <br />  Figure 11 Cnn-PA allelic variation D. melanogaster population. The 23 amino acid substitutions present single population D. <br /> melanogaster mapped structural model Cnn-PA, showing amino acid substitution position model. Ten 23 <br /> amino acid substitutions chemically similar residues. In addition wild-type protein, unique combinations these <br /> substitutions produce 29 allelic variants Cnn-PA population. Brackets model positions conserved Cnn motifs, <br /> and positions exon splice junctions indicated solid triangles model. <br />  <br />  <br /> underlie changes, does account possible              function cleavage. The obvious experiments to <br /> effects protein function. The &#239;&#172;?nding rapid diver-                 conduct express cnn orthologs D. melanogaster and <br /> gence associated coiled-coil regions suggests                test rescue ability divergent alleles D. mel- <br /> these protein folds buffer changes coiled-              anogaster native protein absent, scope <br /> coil protein folds composed simple heptad repeats              study. <br /> have minimal structural constraints (Subbiah 1989). How- <br />                                                                           Multi-exon MPOs correlated rapid divergence <br /> ever, if, example, protein forms homodimers, sequence <br />                                                                           alternative splicing <br /> divergence small indels negative effect on <br /> protein function. Divergent proteins repel other,                Although coding exon MPO, prediction is <br /> preventing dimer formation, numerous indels                     MPOs co-extensive actual coding <br /> shift functional motifs register other,             sequence. Since clearly case cnn Dro- <br /> making coexpression heteromorphic peptides                     sophila, wanted know unique phenome- <br /> possibly antagonistic, resulting mutant phenotype.                   non genes D. melanogaster contained long <br />    To test possibility, expressed D. mela-                 MPOs relative known coding sequences. To identify can- <br /> nogaster GFP::Cnn-PA fusion protein syncytial devel-               didate genes, queried annotated exon asked <br /> opment D. melanogaster, D. simulans, D. yakuba, D.                 longer MPO extended 59 39 <br /> pseudoobscura. In live embryos, GFP localizes centro-              ends annotated exon. We MPOs D. <br /> some normally D. melanogaster D. simulans,                    melanogaster varied essentially size <br /> accumulates relatively high levels D. yakuba                 annotated exon extending 14,000 nucleotides <br /> an abnormal pattern, low variable accumula-                   59 39 direction annotated end exon. <br /> tion levels D. pseudoobscura (Figure 13, A-D).                         We identi&#239;&#172;?ed 398 genes containing MPO <br />    In &#239;&#172;?xed embryos immunostained detect Cnn                   extending 1000 nucleotides end of <br /> fusion protein native Cnn, Cnn staining similar            annotated exon, representing  3% genes D. <br /> live images prophase (Figure 13, &#226;&#8364;&#339;D. yakuba&#226;&#8364;? col-                  melanogaster. Since wanted manually annotate several <br /> umn). During metaphase, defects associated accu-                 genes range Drosophila species identify simi- <br /> mulation fusion protein apparent. In D.                     larities evolution cnn, restricted list to <br /> simulans D. yakuba, centrosomes detach spin-                 genes MPO extending .3000 nucleotides beyond <br /> dle and, D. yakuba, centrosome replication precocious               annotated exon. There 39 genes category <br /> at spindles (Figure 13, Q-T). Metaphase spindles D.               chose 7 traditional Drosophila names, <br /> pseudoobscura defective, having multiple poles               genes likely essential developmental <br /> and fusion spindles (Figure 13, T), similar                genes characterized genetic molecular level. <br /> cnn mutant phenotypes described D. melanogaster                        The extent analysis changes our <br /> (Megraw et al. 1999; Eisman et al. 2009). Consistent                 candidate genes, study report high- <br /> this phenotype, development fails embryos.               lights related target MPOs identi&#239;&#172;?ed screen <br /> The control embryos experiments normal                       (Table 2; Supporting Information: Figure S1, Figure S2, Fig- <br /> throughout cell cycle (Figure 13, E-H M-P) de-                ure S3, Figure S4, Figure S5. <br /> velopment proceeds normally.                                                 Of seven genes investigated, reported be <br />    These data demonstrate coexpression divergent                  rapidly evolving Drosophila. Only shortstop (shot) is <br /> alleles cnn dominant-negative effect protein                  conserved, target MPO, evolving <br />  <br />  <br />  <br />                                                                                                                  Rapid Evolution Cnn     919 <br />  Figure 12 Splicing changes nontriplet indels create new codons Cnn-PA orthologs. (A) Rapid changes splicing intron exon 6 <br /> (green) exon 7 (red) D. melanogaster, D. biarmipes, D. pseudoobscura indels shortened intron D. biarmipes moved the <br /> intron D. pseudoobscura. The exon MPO (light red) maps (top) aligned SPD motif (left) start carboxy terminus (right) <br /> showing effect changes exon position. A single substitution D. biarmipes introduced stop codon (boxed, middle), changing the <br /> MPO. The aligned protein sequences (bottom) showing splice sites (arrowheads) rapid divergence associated changes, <br /> which typical existing splice sites orthologs sites exon fusion. The evidence suggests changes nontriplet indels <br /> and new codon usage coding sequence relaxed splicing. (B) An alignment region North Carolina lines shows multiple <br /> lines begun accumulate nontriplet indels (top). These changes change splicing, reveal buffering capacity MPOs (light <br />  <br />  <br />  <br />  <br /> 920     R. C. Eisman T. C. Kaufman <br />  rapidly, demonstrating divergent MPOs exist other-               &#239;&#172;?cation additional orthologs, accurate functional annota- <br /> wise conserved genes. Target MPOs dumpy (dp), prospero                    tion, future investigations processes and <br /> (pros), short shot, mushroom body defective (mud)                    compensatory mechanisms potentially underlie the <br /> known alternatively spliced intron retention/exclu-              rapid evolution cnn. <br /> sion (IRE) mechanism uncharacterized splice                    Based analyses, cnn orthologs appear re- <br /> variants dp. The target MPOs rhinocerous (rno), lava lamp              stricted insects, agreement OrthoDB Egg- <br /> (lva), Futsch, mud associated signi&#239;&#172;?cant vari-              NOG orthologous gene sets, additional genomes <br /> ation gene models genus Drosophila,                   expand range groups. Orthologous <br /> manual annotation MPOs &#239;&#172;?nds reason                      Cnn proteins highly conserved KFC, SESAW, <br /> considerable variation (Figure S1, Figure S2, Figure S3, Figure              SPD motifs, divergent carboxy ter- <br /> S4, Figure S5, File S1, Figures 2&#226;&#8364;&#8220;5). However, &#239;&#172;?nd                 minus. The arrangement motifs coiled-coil <br /> strong evidence genes studied loss              domains conserved proteins vary size from <br /> gain introns outside target MPOs, require               707 amino acids D. plexippus 1424 amino acids T. <br /> future sequencing verify changes. Finally, target               castaneum. This conservation structure suggests func- <br /> MPOs encode coiled-coil simple repetitive protein              tion conserved (Forslund et al. 2011) provides <br /> domains, similar MPOs cnn.                                         schematic Cnn domain architecture direct future <br />    In general, regions high conservation separated                     studies. <br /> regions high divergence characterize peptides                         The somewhat limited phylogenetic range Cnn ortho- <br /> encoded target MPOs discussed above. The divergent                    logs provides new insight functional evolution <br /> regions introduce multiple small gaps aligned sequences                   Cnn. In D. melanogaster, Cnn required males during <br /> and result signi&#239;&#172;?cant variation protein length                   spermatogenesis (Li et al. 1998) maternal-effect <br /> species similar cnn orthologs. Our target                embryonic lethal gene females, required centro- <br /> MPOs appear confound gene model algorithms                        syncytial development cellularization <br /> be associated changes intron&#226;&#8364;&#8220;exon structure                   (Megraw et al. 1999; Vaizel-Ohayon Schejter 1999; <br /> genes. While mechanisms clear, presence                   Eisman et al. 2009). Other stages develop- <br /> reading frames extending known end                        ment, maternal supply Cnn+ oocytes suf&#239;&#172;?cient for <br /> coding exons apparently indicative rapid evolution.                    development morphologically normal sterile adult <br />                                                                              &#239;&#172;&#8218;ies. Since type development common species <br />                                                                              study, Cnn novel centrosomal protein that <br /> Discussion <br />                                                                              evolved organize actin microtubule cytoskeleton in <br /> The rapid divergence cnn problematic                    absence cell membranes. The ancestral gene genes <br /> intriguing lab &#239;&#172;?rst described gene                      cnn derived remains unknown time. <br /> (Heuer et al. 1995). Surprisingly, rate divergence                        One possibility cnn chimera derived mul- <br /> per million years cnn rapid rate reported               tiple genes, suggested superfamily proteins with <br /> for vertebrate albumin genes (Doolittle 1995)                   KFC motif. Experimental data superfamily exist <br /> rapid rate 20&#226;&#8364;&#8220;30% cytoskeletal proteins pre-                 fungal genes Anucleate primary sterigmataB (AspB) in <br /> dicted report. The rapid divergence                  Aspergillus nidulans (Westfall Momany 2002), microtu- <br /> dif&#239;&#172;?cult identify domains motifs necessary Cnn                    bule organizer 1 (Mto1p) Schizosaccharomyces pombe <br /> function, essential directed molecular dissec-                (Sawin et al. 2004), vertebrate genes mmg phos- <br /> tion protein. Moreover, rapid evolution                 phodiesterase 4d interacting protein (Pde4dip) (Verde et al. <br /> publicly available orthologous gene sets makes impossible                 2001) CDK5RAP2, cnn, D. melanogaster. <br /> to completely rule orthologous relationships in-                   The KFC motif Mto1p (Sawin et al. 2004), CDK5RAP2 <br /> sect, vertebrate, fungal genes encoding KFC motif.                   (Fong et al. 2008), Cnn (Megraw et al. 2001; Zhang <br /> Although rapid evolution problematic,                       Megraw 2007) nucleates g-tubulin microtubule- <br /> always intrigued processes drive diver-                     organizing centers function mechanistically <br /> gence compensatory mechanisms exist Drosoph-                     different fungi, vertebrates, &#239;&#172;&#8218;ies (Fong et al. 2008). <br /> ila offset rapid changes essential developmental                       Unlike proteins, AspB involved nuclear migra- <br /> proteins. For reasons, characterized cnn                     tion spore formation colocalizes F-actin <br /> orthologous gene set insects, useful identi-                (Westfall Momany 2002), Mmg interacts with <br />  <br />  <br />  <br /> red boxes, bottom) comparisons MPOs useful detection small signi&#239;&#172;?cant changes sequence. (C) A <br /> schematic (top) comparison MPOs (light red) exon 4c (blue) exon 5 (brown) A. mellifera N. vitripennis suggest change <br /> has occurred. Comparisons aligned introns (top) large deletion, different nontriplet indels N. vitripennis signi&#239;&#172;?cantly <br /> changed splicing. Alignments coding sequence (middle, boxed) translated peptide encoded exon 4c nontriplet indels <br /> have generated new codons or, unknown reason, nucleotide substitution rate accelerated exon. <br />  <br />  <br />  <br />                                                                                                                       Rapid Evolution Cnn       921 <br />                                                                                                                      Figure 13 Minimal divergence <br />                                                                                                                     proteins perturbs Cnn- <br />                                                                                                                     PA function development. <br />                                                                                                                     The ectopic expression D. <br />                                                                                                                     melanogaster EGFP::Cnn-PA fu- <br />                                                                                                                     sion protein D. melanogaster, <br />                                                                                                                     D. simulans, D. yakuba, D. <br />                                                                                                                     pseudoobscura syncytial <br />                                                                                                                     development shows minimal di- <br />                                                                                                                     vergence            proteins <br />                                                                                                                     expressed heterospeci&#239;&#172;?c <br />                                                                                                                     embryo disrupts normal cleav- <br />                                                                                                                     age. Live imaging EGFP (A&#226;&#8364;&#8220;D) <br />                                                                                                                     shows tagged protein <br />                                                                                                                     localizes centrosome, but <br />                                                                                                                     D. yakuba protein less <br />                                                                                                                     punctate diffuse and <br />                                                                                                                     D. pseudoobscura amount <br />                                                                                                                     tagged protein centrosomes <br />                                                                                                                     reduced variable. Immu- <br />                                                                                                                     nostaining &#239;&#172;?xed embryos rec- <br />                                                                                                                     ognizes tagged and <br />                                                                                                                     native Cnn protein similar <br />                                                                                                                     observed live ani- <br />                                                                                                                     mals prophase (compare <br />                                                                                                                     I&#226;&#8364;&#8220;L A&#226;&#8364;&#8220;D). Although centro- <br />                                                                                                                     somes obvious D. <br />                                                                                                                     pseudoobscura &#239;&#172;?xed material, <br />                                                                                                                     signi&#239;&#172;?cant variability <br />                                                                                                                     nuclei (L). During meta- <br />                                                                                                                     phase (Q&#226;&#8364;&#8220;T) defects clear in <br />                                                                                                                     D. simulans (Q) D. yakuba <br />                                                                                                                     (S) indicated free centro- <br />                                                                                                                     somes (white arrowheads). Either <br />                                                                                                                     centrosome replication preco- <br />                                                                                                                     cious centrosomes split at <br />                                                                                                                     poles D. yakuba. The <br />                                                                                                                     phenotype seen D. pseudoobs- <br />                                                                                                                     cura interesting, spindles <br />                                                                                                                     similar spindles cnn loss- <br />                                                                                                                     of-function mutant embryos. <br />                                                                                                                     However, D. pseudoobscura <br />                                                                                                                     (T), Cnn centrosomes, <br />                                                                                                                     spindle poles multiple cen- <br />                                                                                                                     trosomes spindles are <br />                                                                                                                     multipolar. These results show <br />                                                                                                                     ,10% divergence sin- <br />                                                                                                                     gle indel coex- <br />                                                                                                                     pressed       Cnn     proteins     is <br />                                                                                                                     suf&#239;&#172;?cient signi&#239;&#172;?cantly perturb <br /> normal function. Control animals expressing EGFP-tagged Cnn appear normal cell cycle (prophase: E&#226;&#8364;&#8220;H; metaphase: M&#226;&#8364;&#8220; <br /> P). Fixed embryos (E&#226;&#8364;&#8220;T) stained anti-Cnn (green) anti-b-tubulin (red) DNA stained TOTO3 (blue). <br />  <br />  <br /> cyclic nucleotide phosphodiesterase (Verde et al. 2001)                      Functional data SESAW SPD motifs the <br /> A-kinase (Uys et al. 2011) vertebrate muscles. Addition-                   carboxy terminus limited. Global phosphorylation <br /> ally, proteins share signi&#239;&#172;?cant homology outside                  studies D. melanogaster shown SESAW and <br /> the KFC motif &#239;&#172;&#8218;anking domain, unique motifs                      SPD motifs phosphorylated (http://www.phosphopep. <br /> and domains present proteins, removal                    org/), suggesting potentially important regu- <br /> of KFC motif BLAST searches returns completely                      latory motifs used cell cycle. The function the <br /> different set &#226;&#8364;&#339;best hits.&#226;&#8364;? Taken together, diverse func-               carboxy terminus unknown, have <br /> tions limited homology suggest superfamily                      shown loss SPD motif carboxy terminus <br /> may epaktologs inherited KFC domain in-                      results low levels truncated Cnn centrosome <br /> dependently different lineages (Nagy et al. 2011a),                 high levels metaphase plate (Eisman et al. <br /> than orthologs related descent.                                            2009). A recent study reported motif Cnn <br />  <br />  <br />  <br /> 922     R. C. Eisman T. C. Kaufman <br />  Table 2 Divergence genes multi-exon MPOs Drosophila <br />  <br /> Gene/protein ID              Gene size range (kb)                Protein size range (aa)               Alternative splicing mechanism                   Exons annotated <br /> Shot-PH                                                                  3070&#226;&#8364;&#8220;3498                                   Yes (IRE)                             Exon 14 only <br /> Dp                            Incomplete sequences                                                                Yes (Unknown) <br /> Pros-PK                            10.8&#226;&#8364;&#8220;20.0                             1403&#226;&#8364;&#8220;519                                    Yes (IRE)                             Entire   gene <br /> Mud-PL                             10.3&#226;&#8364;&#8220;16.5                             1641&#226;&#8364;&#8220;2756                                   Yes (IRE)                             Exons    2&#226;&#8364;&#8220;11 <br /> Rno-PB                             11.0&#226;&#8364;&#8220;14.8                             3201&#226;&#8364;&#8220;4386                                      No                                 Entire   gene <br /> Lva-PC                              7.9&#226;&#8364;&#8220;9.5                              2406&#226;&#8364;&#8220;2837                                      No                                 Exons    2&#226;&#8364;&#8220;8 <br /> Futsch-PC                          16.7&#226;&#8364;&#8220;21.1                             4922&#226;&#8364;&#8220;6507                                      No                                 Exons    3&#226;&#8364;&#8220;8 <br /> A summary comparative analyses seven genes Drosophila species based manual annotations. These genes &#239;&#172;?rst identi&#239;&#172;?ed D. melanogaster <br /> due presence extremely long MPO compared annotated coding exons. Only target exon SHOT-PH protein evolving rapidly, analysis <br /> was exon. Dp sequences incomplete species, hindering complete analysis gene. For additional details, File S1 Figure S1, <br /> Figure S2, Figure S3, Figure S4, Figure S5. <br />  <br />  <br />  <br />  <br /> carboxy terminus conserved mouse CDK5RAP2 (Wang                                    mulation indels occurs speciation. However, pre- <br /> et al. 2010), PsiBLAST D. melanogaster motif                              sented Results, evidence nontriplet <br /> found minimal homology, E-value 7.3 &#239;&#172;&#8218;y                                indels exon 4, suggesting types changes <br /> and mouse. Clearly, future studies Cnn required                                exist single population. Future RNA-seq anal- <br /> de&#239;&#172;?ne molecular basis Cnn function.                                               yses lines North Carolina population be <br />     The conservation Cnn domain architecture                                    required verify validity cause apparent <br /> striking contrast divergence orthologous gene                                  rapid changes coding sequence. <br /> architecture, gene size, coding sequences. In general,                                   A key question regarding protein insertions deletions <br /> the intron&#226;&#8364;&#8220;exon structure cnn conserved insect                              underlying nucleotide indels triplet or <br /> orders strikingly different orders. Relative                           nontriplet changes. The presumably removes single <br /> the dipterans, Cnn encoded exons species                               codon causes frameshift. The assump- <br /> investigated trend exon fusion                                 tion global analyses protein indels result of <br /> dipterans. The common mechanism intron loss                                  perfect insertion deletion codons, complete <br /> mediated reverse transcriptase (Coulombe-Huntington                                   divergence multiple amino acids surrounding indels in <br /> and Majewski 2007; Yenerall et al. 2011), results                                Cnn-PA alignments suggests true. The <br /> suggest cnn introns lost genomic deletions                                   parsimonious explanation divergence coding <br /> (Llopart et al. 2002; Yenerall et al. 2011). Our comparison                              regions &#239;&#172;&#8218;anking exon 6/7 splice site Drosophila and <br /> of exon structure cnn orthologs shows potential in-                             exon 4c/5 splice site hymenopterans nontriplet <br /> termediate step process, especially hyme-                             indels. In examples, upstream downstream con- <br /> nopterans. Initially, intronic deletions bring multiple exons                            served regions &#239;&#172;&#8218;ank divergent regions indels <br /> into close proximity other, forming exon islands.                              eventually total triplet combinations splicing <br /> These data resolve multiple exons fuse,                                   machinery restore downstream reading frame. Al- <br /> imprecise NHEJ repair (Llopart et al. 2002; Yenerall et al.                              data required address possibilities, <br /> 2011) intronic deletions resulting intron retention                                probable mechanisms resolve nontriplet <br /> (Talerico Berget 1994) plausible mechanisms. These                               indels coding sequences. <br /> two mechanisms, loss coding exons                                   The types changes driving rapid evolution Cnn- <br /> lepidopterans incorporation nonhomologous cod-                                PA suggest buffering mechanisms the <br /> ing sequence T. castaneum, account                              sequence protein level. In study, presented <br /> divergence Cnn-PA proteins.                                                           MPOs possible buffering mechanism sequence <br />     The divergence Cnn-PA proteins appears to                               level. The buffering capacity MPOs appears inherent <br /> several types changes. Imprecise exon fusion events,                                  sequence span, coding small <br /> nonsynonymous nucleotide substitution, frequent small                                    introns paucity dinucleotides needed introduce <br /> indels coding exons, and, based analyses,                                  stop codons. This results multiple overlapping reading <br /> nontriplet indels resulting frameshifts generate new                             frames genomic region, demonstrated the <br /> codons play role divergence Cnn-PA. Within                               intron exons 6 7 North Carolina lines. In <br /> orders, Cnn-PA divergence rapid uniform                                example, splicing changed, but, similar changes <br /> phylogeny, comparisons orders                                  occur coding sequence, as-yet-unknown apparently <br /> Cnn-PA minimal homology, represent                                         present splicing mechanism retain protein function. <br /> the minimal conservation necessary retain Cnn-PA                                      Perhaps sequence level evolution selects paucity <br /> function. Analysis D. melanogaster North Carolina                                 stop codon dinucleotides buffer inevitable <br /> lines suggests nucleotide substitution responsible                               deleterious deletions occur genome size is <br /> for allelic variation population accu-                             signi&#239;&#172;?cantly reduced. <br />  <br />  <br />  <br />                                                                                                                                         Rapid Evolution Cnn             923 <br />      Although buffering sequence level makes              subset features shared <br /> possible transcribe similar primary sequence,             functions, unwise transfer function cnn <br /> resultant protein fold properly retain function.           based mutational analyses D. melanogaster genes <br /> Based Cnn-PA orthologs proteins MPO              limited homology. This orthologous gene set provides <br /> screen, rapid divergence primary sequence                valuable information future molecular transcrip- <br /> restricted structural proteins. The frequent motif         tional studies. We realize divergence cnn <br /> found study coiled-coil domain, a-helical        does &#239;&#172;?t neatly current models, in <br /> fold composed simple heptad repeats common com-           silico global analyses multiple genomes &#226;&#8364;&#339;see&#226;&#8364;? the <br /> ponent structural proteins. Since residues sub-        small-yet-signi&#239;&#172;?cant changes described. One way detect <br /> stitute heptad positions, protein motif   changes incorporate MPOs computer <br /> surprising fold diverge rapidly retain func-     programs comparisons MPO changes species <br /> tion. The rapid divergence Futsch suggests second        means detect single-nucleotide deletion. It is <br /> fold buffering capacity high-copy-number simple            hope work provides basis continued inves- <br /> repeats. Although did analyze evolution Dp,          tigations rare genomic changes presumably <br /> it shown repetitive PIGFEAST region           deleterious mutations drive rapid evolution essen- <br /> undergoing concerted evolution Drosophila un-            tial genes. <br /> equal crossing (Carmon et al. 2007, 2010), similar what <br /> we Futsch. While protein folds, disor- <br />                                                                    Acknowledgments <br /> dered regions, likely buffering capacity, the <br /> rapid divergence associated Cnn-PA orthologs likely        The authors thank Melissa Phelps Stacy Holtzman for <br /> to restricted subset protein types. An additional       outstanding technical assistance enduring moral sup- <br /> and somewhat surprising buffering capacity inherent         port; FlyBase easy access genetic genomic <br /> in insects tolerate signi&#239;&#172;?cant differences    information; Bloomington Drosophila Stock Center for <br /> in protein size closely related species. This suggests     &#239;&#172;&#8218;y stocks; Center Genomics Bioinformatics for <br /> that molecular machinery animals relatively        help algorithm search MPOs; Baylor <br /> plastic wide range size tolerances.                      Sequencing Center Fringy Richards access the <br />     While rapid divergence buffered levels,      recently sequenced modENCODE Drosophila genomes and <br /> it reasonable expect constraints protein          associated RNAseq data; Josh Goodman help in <br /> function, especially coiled-coil domains interact form       retrieving relevant sequences Mackay Raleigh <br /> oligomers. Although know oligomerization             population sequences; and, &#239;&#172;?nally, Matt Hahn his <br /> state Cnn-PA, simplest state Cnn-PA forms parallel     evisceration previous incarnation article. His <br /> homodimer, divergent Cnn-PA orthologs interfere          comments helpful, insightful, remaining <br /> with protein folding domain alignment. To test            de&#239;&#172;?ciencies pigheaded refusal his <br /> hypothesis, ectopically expressed D. melanogaster             well-proffered advice. Supported Indiana Genomics <br /> GFP::Cnn-PA fusion protein D. simulans, D. yakuba,          Initiative. <br /> D. pseudoobscura syncytial development. These <br /> experiments showed fusion protein interferes with <br /> native Cnn-PA function species inten-        Literature Cited <br /> sity dominant-negative effect increases diver- <br />                                                                    Altenhoff, A. M., C. Dessimoz, 2009 Phylogenetic func- <br /> gence native protein increases. Since GFP::Cnn-PA          tional assessment orthologs inference projects methods. <br /> fusion protein rescues cnnmfs mutant phenotype D.              PLOS Comput. Biol. 5: e1000262. <br /> melanogaster, D. simulans D. yakuba Cnn-PA             Carmon, A., M. Wilkin, J. Hassan, M. Baron, R. MacIntyre, <br /> proteins similar, unlikely results          2007 Concerted evolution Drosophila dumpy gene. <br /> are GFP tag functional changes               Genetics 176: 309&#226;&#8364;&#8220;325. <br /> species. This suggests that, functional homodimer         Carmon, A., M. J. Guertin, O. Grushko, B. Marshall, R. MacIn- <br />                                                                       tyre, 2010 A molecular analysis mutations complex <br /> form, degree protein divergence tolerated low and <br />                                                                       dumpy locus Drosophila melanogaster. PLoS ONE 5: e12319. <br /> indels rare species. However,               Conduit, P. T., J. W. Raff, 2010 Cnn dynamics drive cen- <br /> effects D. yakuba Cnn-PA orthologs            trosome size asymmetry ensure daughter centriole retention <br /> type divergence taking place essential devel-          Drosophila neuroblasts. Curr. Biol. 20: 2187&#226;&#8364;&#8220;2192. <br /> opmental proteins, tempting propose com-         Coulombe-Huntington, J., J. Majewski, 2007 Intron loss and <br /> bined effect associated speciation, speci&#239;&#172;?cally           gain Drosophila. Mol. Biol. Evol. 24: 2842&#226;&#8364;&#8220;2850. <br />                                                                    Dobbelaere, J., F. Josue, S. Suijkerbuijk, B. Baum, N. Tapon et al., <br /> in light data presented postfertilization <br />                                                                       2008 A genome-wide RNAi screen dissect centriole dupli- <br /> isolation. <br />                                                                       cation centrosome maturation Drosophila. PLoS Biol. 6: <br />     We presented comprehensive set cnn orthologs            e224. <br /> and features likely necessary Cnn       Doolittle, R. F., 1995 The multiplicity domains proteins. <br /> function associated D. melanogaster. While proteins              Annu. Rev. Biochem. 64: 287&#226;&#8364;&#8220;314. <br />  <br />  <br />  <br /> 924    R. C. Eisman T. C. Kaufman <br />  Eisman, R., T. C. Kaufman, 2007 Cytological investigation         Lupas, A., M. Van Dyke, J. Stock, 1991 Predicting coiled coils <br />     mechanism parthenogenesis Drosophila mercatorum.              protein sequences. Science 252: 1162&#226;&#8364;&#8220;1164. <br />     Fly (Austin) 1: 317&#226;&#8364;&#8220;329.                                             Mackay, T. F., S. Richards, E. A. Stone, A. Barbadilla, J. F. Ayroles <br /> Eisman, R. C., N. Stewart, D. Miller, T. C. Kaufman,                    et al., 2012 The Drosophila melanogaster Genetic Reference <br />     2006 centrosomin&#226;&#8364;&#8482;s beautiful sister (cbs) encodes GRIP-do-            Panel. Nature 482: 173&#226;&#8364;&#8220;178. <br />     main protein marks Golgi inheritance functions       Matthews, K. A., D. F. Miller, T. C. Kaufman, <br />     centrosome cycle Drosophila. J. Cell Sci. 119: 3399&#226;&#8364;&#8220;3412.            1989 Developmental distribution RNA protein products <br /> Eisman, R. C., M. A. Phelps, T. C. Kaufman,                             Drosophila alpha-tubulin gene family. Dev. Biol. 132: 45&#226;&#8364;&#8220; <br />     2009 Centrosomin: complex mix long short isoforms              61. <br />     required centrosome function early development      McQuilton, P., S. E. St Pierre, J. Thurmond; FlyBase Consor- <br />     Drosophila melanogaster. Genetics 182: 979&#226;&#8364;&#8220;997.                         tium, 2012 FlyBase 101: basics navigating FlyBase. Nu- <br /> Fitch, W. M., 1970 Distinguishing homologous analogous                 cleic Acids Res. 40: D706&#226;&#8364;&#8220;D714. <br />     proteins. Syst. Zool. 19: 99&#226;&#8364;&#8220;113.                                    Megraw, T. L., K. Li, L. R. Kao, T. C. Kaufman, 1999 The <br /> Fitch, W. M., 2000 Homology personal view                  centrosomin protein required centrosome assembly and <br />     problems. Trends Genet. 16: 227&#226;&#8364;&#8220;231.                                    function cleavage Drosophila. Development 126: <br /> Flory, M. R., M. Morphew, J. D. Joseph, A. R. Means, T. N.              2829&#226;&#8364;&#8220;2839. <br />     Davis, 2002 Pcp1p, Spc110p-related calmodulin target           Megraw, T. L., L. R. Kao, T. C. Kaufman, 2001 Zygotic de- <br />     centrosome &#239;&#172;?ssion yeast Schizosaccharomyces                  velopment functional mitotic centrosomes. Curr. Biol. <br />     pombe. Cell Growth Differ. 13: 47&#226;&#8364;&#8220;58.                                   11: 116&#226;&#8364;&#8220;120. <br /> Fong, K. W., Y. K. Choi, J. B. Rattner, R. Z. Qi,                    Mushegian, A. R., J. R. Garey, J. Martin, L. X. Liu, 1998 Large- <br />     2008 CDK5RAP2 pericentriolar protein functions             scale taxonomic pro&#239;&#172;?ling eukaryotic model organisms: com- <br />     centrosomal attachment gamma-tubulin ring complex.               parison orthologous proteins encoded human, &#239;&#172;&#8218;y, nem- <br />     Mol. Biol. Cell 19: 115&#226;&#8364;&#8220;125.                                            atode, yeast genomes. Genome Res. 8: 590&#226;&#8364;&#8220;598. <br /> Forslund, K., I. Pekkari, E. L. Sonnhammer, 2011 Domain              Nagy, A., L. B&#195;&#161;nyai, L. Patthy, 2011a Reassessing domain ar- <br />     architecture conservation orthologs. BMC Bioinformatics 12:          chitecture evolution metazoan proteins: major impact errors <br />     326.                                                                    caused confusing paralogs epaktologs. Genes 2: 516&#226;&#8364;&#8220;561. <br /> Galperin, M. Y., E. V. Koonin, 1998 Sources systematic            Nagy, A., G. Szl&#195;&#161;ma, E. Szarka, M. Trexler, L. B&#195;&#161;nyai et al., <br />     error functional annotation genomes: domain rearrange-            2011b Correction: Nagy, et al. Reassessing domain architec- <br />                                                                             ture evolution metazoan proteins: major impact gene pre- <br />     ment, non-orthologous gene displacement operon disrup- <br />                                                                             diction errors. Genes 2011, 2, 449&#226;&#8364;&#8220;501. Genes 2: 599&#226;&#8364;&#8220;607. <br />     tion. In Silico Biol. 1: 55&#226;&#8364;&#8220;67. <br />                                                                          Nene, V., J. R. Wortman, D. Lawson, B. Haas, C. Kodira et al., <br /> Gorman, O. T., W. J. Bean, Y. Kawaoka, I. Donatelli, Y. J. Guo et al., <br />                                                                             2007 Genome sequence Aedes aegypti, major arbovirus <br />     1991 Evolution in&#239;&#172;&#8218;uenza A virus nucleoprotein genes: im- <br />                                                                             vector. Science 316: 1718&#226;&#8364;&#8220;1723. <br />     plications origins H1N1 human classical swine <br />                                                                          Petrov, D. A., 2002 DNA loss evolution genome size in <br />     viruses. J. Virol. 65: 3704&#226;&#8364;&#8220;3714. <br />                                                                             Drosophila. Genetica 115: 81&#226;&#8364;&#8220;91. <br /> Gupta, R. S., 1998 Protein phylogenies signature sequences: <br />                                                                          Petrov, D. A., D. L. Hartl, 1997 Trash DNA gets <br />     reappraisal evolutionary relationships archaebacte- <br />                                                                             thrown away: high rate DNA loss Drosophila. Gene 205: <br />     ria, eubacteria, eukaryotes. Microbiol. Mol. Biol. Rev. 62: <br />                                                                             279&#226;&#8364;&#8220;289. <br />     1435&#226;&#8364;&#8220;1491. <br />                                                                          Petrov, D. A., D. L. Hartl, 1998 High rate DNA loss the <br /> Heuer, J. G., K. Li, T. C. Kaufman, 1995 The Drosophila ho- <br />                                                                             Drosophila melanogaster Drosophila virilis species groups. <br />     meotic target gene centrosomin (cnn) encodes novel centro-            Mol. Biol. Evol. 15: 293&#226;&#8364;&#8220;302. <br />     somal protein leucine zippers maps genomic             Petrov, D. A., T. A. Sangster, J. S. Johnston, D. L. Hartl, K. L. <br />     region required midgut morphogenesis. Development 121:              Shaw, 2000 Evidence DNA loss determinant ge- <br />     3861&#226;&#8364;&#8220;3876.                                                              nome size. Science 287: 1060&#226;&#8364;&#8220;1062. <br /> Holtzman, S., D. Miller, R. Eisman, H. Kuwayama, T. Niimi et al.,        Rokas, A., P. W. Holland, 2000 Rare genomic changes as <br />     2010 Transgenic tools members genus Drosophila               tool phylogenetics. Trends Ecol. Evol. 15: 454&#226;&#8364;&#8220;459. <br />     sequenced genomes. Fly (Austin) 4: 349&#226;&#8364;&#8220;362.                     Sawin, K. E., P. C. Lourenco, H. A. Snaith, 2004 Microtubule <br /> Hulsen, T., M. A. Huynen, J. Vlieg, P. M. Groenen,                   nucleation non-spindle pole body microtubule-organizing <br />     2006 Benchmarking ortholog identi&#239;&#172;?cation methods using                  centers requires &#239;&#172;?ssion yeast centrosomin-related protein <br />     functional genomics data. Genome Biol. 7: R31.                          mod20p. Curr. Biol. 14: 763&#226;&#8364;&#8220;775. <br /> Kalvakolanu, D. V., W. H. Livingston, III, 1991 Rapid            Subbiah, S., 1989 Are coiled-coil proteins evolutionarily related? <br />     inexpensive protocol generating greater 95% recombi-           J. Mol. Biol. 206: 689&#226;&#8364;&#8220;692. <br />     nants subcloning experiments. Biotechniques 10: 176&#226;&#8364;&#8220;177.          Talerico, M., S. M. Berget, 1994 Intron de&#239;&#172;?nition splicing <br /> Keeling, P. J., M. A. Luker, J. D. Palmer, 2000 Evidence           small Drosophila introns. Mol. Cell. Biol. 14: 3434&#226;&#8364;&#8220;3445. <br />     beta-tubulin phylogeny microsporidia evolved        Tatusov, R. L., E. V. Koonin, D. J. Lipman, 1997 A genomic <br />     fungi. Mol. Biol. Evol. 17: 23&#226;&#8364;&#8220;31.                                  perspective protein families. Science 278: 631&#226;&#8364;&#8220;637. <br /> Li, K., T. C. Kaufman, 1996 The homeotic target gene cen-            Uys, G. M., A. Ramburan, B. Loos, C. J. Kinnear, L. J. Korkie et al., <br />     trosomin encodes essential centrosomal component. Cell 85:           2011 Myomegalin novel A-kinase anchoring protein in- <br />     585&#226;&#8364;&#8220;596.                                                                volved phosphorylation cardiac myosin binding protein <br /> Li, K., E. Y. Xu, J. K. Cecil, F. R. Turner, T. L. Megraw et al.,           C. BMC Cell Biol. 12: 18. <br />     1998 Drosophila centrosomin protein required male mei-        Vaizel-Ohayon, D., E. D. Schejter, 1999 Mutations centro- <br />     osis assembly &#239;&#172;&#8218;agellar axoneme. J. Cell Biol. 141:           somin reveal requirements centrosomal function during <br />     455&#226;&#8364;&#8220;467.                                                                early Drosophila embryogenesis. Curr. Biol. 9: 889&#226;&#8364;&#8220;898. <br /> Llopart, A., J. M. Comeron, F. G. Brunet, D. Lachaise, M. Long,      Venkatram, S., J. J. Tasto, A. Feoktistova, J. L. Jennings, A. J. Link <br />     2002 Intron presence-absence polymorphism Drosophila                 et al., 2004 Identi&#239;&#172;?cation characterization novel <br />     driven positive Darwinian selection. Proc. Natl. Acad. Sci.          proteins affecting &#239;&#172;?ssion yeast gamma-tubulin complex func- <br />     USA 99: 8121&#226;&#8364;&#8220;8126.                                                      tion. Mol. Biol. Cell 15: 2287&#226;&#8364;&#8220;2301. <br />  <br />  <br />  <br />                                                                                                                 Rapid Evolution Cnn     925 <br />  Verde, I., G. Pahlke, M. Salanova, G. Zhang, S. Wang et al.,             Yenerall, P., B. Krupa, L. Zhou, 2011 Mechanisms intron <br />    2001 Myomegalin novel protein Golgi/centrosome               gain loss Drosophila. BMC Evol. Biol. 11: 364. <br />    interacts cyclic nucleotide phosphodiesterase. J. Biol.   Zhang, J., T. L. Megraw, 2007 Proper recruitment gamma- <br />    Chem. 276: 11189&#226;&#8364;&#8220;11198.                                                  tubulin D-TACC/Msps embryonic Drosophila centro- <br /> Wang, Z., T. Wu, L. Shi, L. Zhang, W. Zheng et al., 2010 Conserved          somes requires Centrosomin Motif 1. Mol. Biol. Cell 18: 4037&#226;&#8364;&#8220; <br />    motif CDK5RAP2 mediates localization centrosomes           4049. <br />    Golgi complex. J. Biol. Chem. 285: 22658&#226;&#8364;&#8220;22665.                   Zhang, X., J. Lee, L. A. Chasin, 2003 The effect nonsense <br /> Westfall, P. J., M. Momany, 2002 Aspergillus nidulans septin            codons splicing: genomic analysis. RNA 9: 637&#226;&#8364;&#8220;639. <br />    AspB plays pre- postmitotic roles septum, branch, and <br />    conidiophore development. Mol. Biol. Cell 13: 110&#226;&#8364;&#8220;118.                                              Communicating editor: N. Perrimon <br />  <br />  <br />  <br />  <br /> 926     R. C. Eisman T. C. Kaufman <br />       GENETICS <br />                                                        Supporting Information <br />     http://www.genetics.org/lookup/suppl/doi:10.1534/genetics.113.152546/-/DC1 <br />  <br />  <br />  <br />  <br /> Probing Boundaries Orthology: The <br />         Unanticipated Rapid Evolution of <br />                  Drosophila centrosomin <br />                                      Robert C. Eisman Thomas C. Kaufman <br />  <br />  <br />  <br />  <br />                                         Copyright &#194;&#169; 2013 Genetics Society America <br />                                                         DOI: 10.1534/genetics.113.152546 <br />                                                                                                                       &#194;&#160; <br /> &#194;&#160; <br /> Figure&#194;&#160;S1&#194;&#160;&#194;&#160;&#194;&#160;Three&#194;&#160;large&#194;&#160;multi&#226;&#8364;?exon&#194;&#160;MPOs&#194;&#160;in&#194;&#160;D&#194;&#160;melanogaster&#194;&#160;dumpy.&#194;&#160;The&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;dp&#194;&#160;gene&#194;&#160;(top)&#194;&#160;is&#194;&#160;shown&#194;&#160;with&#194;&#160;the&#194;&#160;MPOs&#194;&#160; <br /> (orange&#194;&#160;boxes)&#194;&#160;shown&#194;&#160;above&#194;&#160;the&#194;&#160;known&#194;&#160;coding&#194;&#160;exons&#194;&#160;(blue&#194;&#160;arrows).&#194;&#160;The&#194;&#160;PIGFEAST&#194;&#160;repeat&#194;&#160;region&#194;&#160;and&#194;&#160;the&#194;&#160;exon&#194;&#160;37&#194;&#160;to&#194;&#160;70&#194;&#160;region&#194;&#160; <br /> are&#194;&#160;indicated&#194;&#160;by&#194;&#160;bars.&#194;&#160;&#194;&#160;An&#194;&#160;expanded&#194;&#160;view&#194;&#160;of&#194;&#160;the&#194;&#160;exon&#194;&#160;37&#194;&#160;to&#194;&#160;70&#194;&#160;region&#194;&#160;(bottom),&#194;&#160;which&#194;&#160;contains&#194;&#160;34&#194;&#160;coding&#194;&#160;exons&#194;&#160;in&#194;&#160;13&#194;&#160;MPOs&#194;&#160;or&#194;&#160; <br /> reading&#194;&#160;frames.&#194;&#160;&#194;&#160;Incomplete&#194;&#160;sequences&#194;&#160;in&#194;&#160;the&#194;&#160;other&#194;&#160;species&#194;&#160;made&#194;&#160;a&#194;&#160;complete&#194;&#160;analysis&#194;&#160;impossible&#194;&#160;but&#194;&#160;a&#194;&#160;similar&#194;&#160;ratio&#194;&#160;of&#194;&#160;exons&#194;&#160;to&#194;&#160; <br /> MPOs&#194;&#160;was&#194;&#160;found&#194;&#160;in&#194;&#160;D.&#194;&#160;virilis,&#194;&#160;although&#194;&#160;the&#194;&#160;arrangement&#194;&#160;of&#194;&#160;corresponding&#194;&#160;exons&#194;&#160;is&#194;&#160;different&#194;&#160;between&#194;&#160;species.&#194;&#160;&#194;&#160; <br /> &#194;&#160;                                      &#194;&#160; <br />  <br />  <br />  <br />  <br /> 2&#194;&#160;SI                                            R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;&#194;&#160; <br /> &#194;&#160; <br />                                                                                                                                            &#194;&#160; <br /> &#194;&#160; <br />     Figure&#194;&#160;S2&#194;&#160;&#194;&#160;&#194;&#160;Annotation&#194;&#160;of&#194;&#160;Drosophila&#194;&#160;MUD&#226;&#8364;?PL.&#194;&#160;The&#194;&#160;manual&#194;&#160;annotation&#194;&#160;of&#194;&#160;the&#194;&#160;MUD&#226;&#8364;?PL&#194;&#160;splice&#194;&#160;variant&#194;&#160;from&#194;&#160;exon&#194;&#160;2&#194;&#160;to&#194;&#160;11&#194;&#160;is&#194;&#160;shown&#194;&#160; <br />     for&#194;&#160;eight&#194;&#160;species&#194;&#160;with&#194;&#160;the&#194;&#160;MPOs&#194;&#160;(orange&#194;&#160;boxes)&#194;&#160;above&#194;&#160;the&#194;&#160;coding&#194;&#160;exons&#194;&#160;(blue&#194;&#160;arrows)&#194;&#160;below.&#194;&#160;&#194;&#160;The&#194;&#160;first&#194;&#160;coding&#194;&#160;exon&#194;&#160;could&#194;&#160;not&#194;&#160;be&#194;&#160; <br />     assigned&#194;&#160;with&#194;&#160;high&#194;&#160;confidence&#194;&#160;in&#194;&#160;at&#194;&#160;least&#194;&#160;one&#194;&#160;species.&#194;&#160;&#194;&#160;The&#194;&#160;numbers&#194;&#160;on&#194;&#160;the&#194;&#160;right&#194;&#160;indicate&#194;&#160;the&#194;&#160;gene&#226;&#8364;&#8482;s&#194;&#160;genomic&#194;&#160;extent&#194;&#160;(top)&#194;&#160;and&#194;&#160; <br />     the&#194;&#160;translated&#194;&#160;protein&#194;&#160;size&#194;&#160;(bottom)&#194;&#160;for&#194;&#160;each&#194;&#160;model,&#194;&#160;the&#194;&#160;species&#194;&#160;is&#194;&#160;on&#194;&#160;the&#194;&#160;left.&#194;&#160;&#194;&#160;The&#194;&#160;gene&#194;&#160;models&#194;&#160;are&#194;&#160;to&#194;&#160;scale&#194;&#160;and&#194;&#160;are&#194;&#160;aligned&#194;&#160;on&#194;&#160; <br />     the&#194;&#160;left&#194;&#160;at&#194;&#160;the&#194;&#160;3&#226;&#8364;&#178;&#194;&#160;terminal&#194;&#160;stop&#194;&#160;codon.&#194;&#160;&#194;&#160;The&#194;&#160;target&#194;&#160;MPO&#194;&#160;identified&#194;&#160;in&#194;&#160;our&#194;&#160;screen&#194;&#160;for&#194;&#160;long&#194;&#160;MPOs&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;is&#194;&#160;shaded&#194;&#160;in&#194;&#160; <br />     gray.&#194;&#160;The&#194;&#160;asterisk&#194;&#160;downstream&#194;&#160;of&#194;&#160;the&#194;&#160;target&#194;&#160;MPO&#194;&#160;is&#194;&#160;a&#194;&#160;clear&#194;&#160;intron&#194;&#160;loss&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;and&#194;&#160;D.&#194;&#160;erecta.&#194;&#160;Interestingly&#194;&#160;this&#194;&#160;exon&#194;&#160; <br />     in&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;encodes&#194;&#160;a&#194;&#160;conserved&#194;&#160;functional&#194;&#160;motif&#194;&#160;in&#194;&#160;the&#194;&#160;protein.&#194;&#160; <br />     Abbreviations:&#194;&#160;(Dmel:&#194;&#160;D.&#194;&#160;melanogaster;&#194;&#160;Dere:&#194;&#160;D.&#194;&#160;erecta;&#194;&#160;Dyak:&#194;&#160;D.&#194;&#160;yakuba;&#194;&#160;Dana:&#194;&#160;D.&#194;&#160;ananassae;&#194;&#160;Dpse:&#194;&#160;D.&#194;&#160;psuedoobscura;&#194;&#160;Dwil:&#194;&#160;D.&#194;&#160; <br />     willistoni;&#194;&#160;Dmoj:&#194;&#160;D.&#194;&#160;mojavensis;&#194;&#160;Dvir:&#194;&#160;D.&#194;&#160;virilis;&#194;&#160;Dgri:&#194;&#160;D.&#194;&#160;grimshawi)&#194;&#160; <br />  <br />     &#194;&#160;                                      &#194;&#160; <br />  <br />  <br />  <br />  <br />                                                      R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;                                              3&#194;&#160;SI&#194;&#160; <br />     &#194;&#160; <br />                                                                                                                                        &#194;&#160; <br />     &#194;&#160; <br />     Figure&#194;&#160;S3&#194;&#160;&#194;&#160;&#194;&#160;Annotation&#194;&#160;of&#194;&#160;Drosophila&#194;&#160;Rno&#226;&#8364;?PB.&#194;&#160;The&#194;&#160;manual&#194;&#160;annotation&#194;&#160;of&#194;&#160;the&#194;&#160;entire&#194;&#160;RNO&#226;&#8364;?PB&#194;&#160;splice&#194;&#160;variant&#194;&#160;is&#194;&#160;shown,&#194;&#160;labeled&#194;&#160;as&#194;&#160;in&#194;&#160; <br />     Supplemental&#194;&#160;Figure&#194;&#160;2.&#194;&#160;&#194;&#160;All&#194;&#160;the&#194;&#160;variation&#194;&#160;in&#194;&#160;the&#194;&#160;protein&#194;&#160;size&#194;&#160;between&#194;&#160;species&#194;&#160;is&#194;&#160;due&#194;&#160;to&#194;&#160;many&#194;&#160;indels&#194;&#160;within&#194;&#160;the&#194;&#160;large&#194;&#160;terminal&#194;&#160; <br />     coding&#194;&#160;exon&#194;&#160;identified&#194;&#160;in&#194;&#160;our&#194;&#160;screen.&#194;&#160; <br /> &#194;&#160; <br />     &#194;&#160;                                      &#194;&#160; <br />  <br />  <br />  <br />  <br />     4&#194;&#160;SI                                            R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;&#194;&#160; <br />     &#194;&#160; <br />                                                                                                                                         &#194;&#160; <br /> &#194;&#160; <br /> Figure&#194;&#160;S4&#194;&#160;&#194;&#160;&#194;&#160;Annotation&#194;&#160;of&#194;&#160;Drosophila&#194;&#160;Lva&#226;&#8364;?PC.&#194;&#160;The&#194;&#160;manual&#194;&#160;annotation&#194;&#160;of&#194;&#160;exons&#194;&#160;2&#194;&#160;to&#194;&#160;8&#194;&#160;of&#194;&#160;the&#194;&#160;LVA&#226;&#8364;?PC&#194;&#160;splice&#194;&#160;variant&#194;&#160;is&#194;&#160;shown,&#194;&#160; <br /> labeled&#194;&#160;as&#194;&#160;in&#194;&#160;Supplemental&#194;&#160;Figure&#194;&#160;2.&#194;&#160;&#194;&#160;The&#194;&#160;first&#194;&#160;coding&#194;&#160;exon&#194;&#160;could&#194;&#160;not&#194;&#160;be&#194;&#160;assigned&#194;&#160;with&#194;&#160;high&#194;&#160;confidence&#194;&#160;in&#194;&#160;at&#194;&#160;least&#194;&#160;one&#194;&#160;species.&#194;&#160;&#194;&#160;In&#194;&#160; <br /> D.&#194;&#160;psuedoobscura&#194;&#160;the&#194;&#160;two&#194;&#160;coding&#194;&#160;exons&#194;&#160;5&#226;&#8364;&#178;&#194;&#160;of&#194;&#160;the&#194;&#160;terminal&#194;&#160;exon&#194;&#160;are&#194;&#160;interrupted&#194;&#160;by&#194;&#160;a&#194;&#160;single&#194;&#160;stop&#194;&#160;codon&#194;&#160;created&#194;&#160;by&#194;&#160;a&#194;&#160;non&#226;&#8364;?triplet&#194;&#160; <br /> indel,&#194;&#160;which&#194;&#160;may&#194;&#160;be&#194;&#160;due&#194;&#160;to&#194;&#160;a&#194;&#160;sequencing&#194;&#160;error.&#194;&#160;&#194;&#160;All&#194;&#160;other&#194;&#160;splice&#194;&#160;sites&#194;&#160;were&#194;&#160;determined&#194;&#160;by&#194;&#160;their&#194;&#160;alignment&#194;&#160;with&#194;&#160;the&#194;&#160;known&#194;&#160;D.&#194;&#160; <br /> melanogaster&#194;&#160;cDNA&#194;&#160;and&#194;&#160;will&#194;&#160;require&#194;&#160;future&#194;&#160;RNA&#194;&#160;sequencing&#194;&#160;to&#194;&#160;determine&#194;&#160;the&#194;&#160;actual&#194;&#160;transcripts&#194;&#160;present&#194;&#160;in&#194;&#160;each&#194;&#160;species.&#194;&#160; <br /> &#194;&#160;                                       &#194;&#160; <br />  <br />  <br />  <br />  <br />                                                   R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;                                              5&#194;&#160;SI&#194;&#160; <br /> &#194;&#160; <br />                                                                                                                                       &#194;&#160; <br /> &#194;&#160; <br /> Figure&#194;&#160;S5&#194;&#160;&#194;&#160;&#194;&#160;Annotation&#194;&#160;of&#194;&#160;Drosophila&#194;&#160;Futsch&#226;&#8364;?PC.&#194;&#160;The&#194;&#160;manual&#194;&#160;annotation&#194;&#160;of&#194;&#160;exons&#194;&#160;3&#194;&#160;to&#194;&#160;8&#194;&#160;of&#194;&#160;the&#194;&#160;Futsch&#226;&#8364;?PC&#194;&#160;splice&#194;&#160;variant&#194;&#160;is&#194;&#160;shown,&#194;&#160; <br /> labeled&#194;&#160;as&#194;&#160;in&#194;&#160;Supplemental&#194;&#160;Figure&#194;&#160;2.&#194;&#160;&#194;&#160;The&#194;&#160;first&#194;&#160;two&#194;&#160;coding&#194;&#160;exons&#194;&#160;could&#194;&#160;not&#194;&#160;be&#194;&#160;assigned&#194;&#160;with&#194;&#160;high&#194;&#160;confidence&#194;&#160;in&#194;&#160;at&#194;&#160;least&#194;&#160;one&#194;&#160; <br /> species.&#194;&#160;As&#194;&#160;in&#194;&#160;RNO&#226;&#8364;?PB,&#194;&#160;the&#194;&#160;protein&#194;&#160;size&#194;&#160;variation&#194;&#160;is&#194;&#160;due&#194;&#160;to&#194;&#160;frequent&#194;&#160;indels&#194;&#160;in&#194;&#160;the&#194;&#160;exons&#194;&#160;corresponding&#194;&#160;to&#194;&#160;the&#194;&#160;single&#194;&#160;large&#194;&#160;coding&#194;&#160; <br /> exon&#194;&#160;associated&#194;&#160;with&#194;&#160;our&#194;&#160;target&#194;&#160;MPO&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster.&#194;&#160;&#194;&#160;The&#194;&#160;single&#194;&#160;breaks&#194;&#160;in&#194;&#160;the&#194;&#160;large&#194;&#160;exons&#194;&#160;in&#194;&#160;D.&#194;&#160;mojavensis&#194;&#160;and&#194;&#160;D.&#194;&#160; <br /> grimshawi&#194;&#160;are&#194;&#160;due&#194;&#160;to&#194;&#160;a&#194;&#160;single&#194;&#160;base&#194;&#160;substitution&#194;&#160;that&#194;&#160;introduces&#194;&#160;an&#194;&#160;in&#226;&#8364;?frame&#194;&#160;stop&#194;&#160;codon,&#194;&#160;probably&#194;&#160;due&#194;&#160;to&#194;&#160;sequencing&#194;&#160;errors.&#194;&#160;The&#194;&#160; <br /> next&#194;&#160;to&#194;&#160;last&#194;&#160;exon&#194;&#160;in&#194;&#160;species&#194;&#160;below&#194;&#160;D.&#194;&#160;ananassae&#194;&#160;are&#194;&#160;present&#194;&#160;but&#194;&#160;we&#194;&#160;could&#194;&#160;not&#194;&#160;predict&#194;&#160;where&#194;&#160;the&#194;&#160;potential&#194;&#160;5&#226;&#8364;&#178;&#194;&#160;intron&#194;&#160;would&#194;&#160;be&#194;&#160;in&#194;&#160; <br /> actual&#194;&#160;transcripts.&#194;&#160; <br />  <br /> &#194;&#160;                                      &#194;&#160; <br />  <br />  <br />  <br />  <br /> 6&#194;&#160;SI                                             R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;&#194;&#160; <br /> &#194;&#160; <br />                                                            File&#194;&#160;S1&#194;&#160; <br />                                                               &#194;&#160; <br />                                                    Supplemental&#194;&#160;Text&#194;&#160; <br />                                                               &#194;&#160; <br /> Our&#194;&#160;screen&#194;&#160;for&#194;&#160;extensive&#194;&#160;MPOs&#194;&#160;as&#194;&#160;compared&#194;&#160;to&#194;&#160;coding&#194;&#160;exon&#194;&#160;length&#194;&#160;identified&#194;&#160;several&#194;&#160;genes&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster.&#194;&#160;&#194;&#160;Although&#194;&#160;we&#194;&#160; <br /> limited&#194;&#160;our&#194;&#160;analyses&#194;&#160;to&#194;&#160;genes&#194;&#160;with&#194;&#160;MPOs&#194;&#160;&gt;3000&#194;&#160;nt&#194;&#160;longer&#194;&#160;than&#194;&#160;an&#194;&#160;associated&#194;&#160;coding&#194;&#160;exon,&#194;&#160;it&#194;&#160;is&#194;&#160;likely&#194;&#160;the&#194;&#160;evolution&#194;&#160;of&#194;&#160;genes&#194;&#160;with&#194;&#160; <br /> less&#194;&#160;extensive&#194;&#160;MPOs&#194;&#160;will&#194;&#160;be&#194;&#160;interesting&#194;&#160;as&#194;&#160;well.&#194;&#160;&#194;&#160;For&#194;&#160;example,&#194;&#160;the&#194;&#160;largest&#194;&#160;MPO&#194;&#160;in&#194;&#160;cnn&#194;&#160;is&#194;&#160;only&#194;&#160;1000&#194;&#160;nt&#194;&#160;longer&#194;&#160;than&#194;&#160;one&#194;&#160;of&#194;&#160;the&#194;&#160; <br /> associated&#194;&#160;coding&#194;&#160;exons.&#194;&#160;&#194;&#160;Additionally,&#194;&#160;genes&#194;&#160;with&#194;&#160;significant&#194;&#160;variation&#194;&#160;in&#194;&#160;their&#194;&#160;MPO&#194;&#160;maps&#194;&#160;across&#194;&#160;some&#194;&#160;phylogenetic&#194;&#160;range&#194;&#160; <br /> should&#194;&#160;be&#194;&#160;good&#194;&#160;candidates&#194;&#160;for&#194;&#160;genomic&#194;&#160;and&#194;&#160;gene/protein&#194;&#160;evolutionary&#194;&#160;studies.&#194;&#160;&#194;&#160;Below&#194;&#160;is&#194;&#160;a&#194;&#160;very&#194;&#160;brief&#194;&#160;description&#194;&#160;of&#194;&#160;the&#194;&#160;seven&#194;&#160; <br /> genes&#194;&#160;we&#194;&#160;have&#194;&#160;manually&#194;&#160;annotated&#194;&#160;in&#194;&#160;nine&#194;&#160;Drosophila&#194;&#160;species.&#194;&#160; <br /> &#194;&#160;         The&#194;&#160;dumpy&#194;&#160;(dp)&#194;&#160;gene&#194;&#160;makes&#194;&#160;maximum&#194;&#160;use&#194;&#160;of&#194;&#160;multi&#226;&#8364;?exon&#194;&#160;MPOs&#194;&#160;and&#194;&#160;contained&#194;&#160;the&#194;&#160;longest&#194;&#160;excess&#194;&#160;beyond&#194;&#160;an&#194;&#160; <br /> annotated&#194;&#160;exon&#194;&#160;end&#194;&#160;(14,000&#194;&#160;nt).&#194;&#160;&#194;&#160;The&#194;&#160;dp&#194;&#160;locus&#194;&#160;spans&#194;&#160;over&#194;&#160;100&#194;&#160;kb,&#194;&#160;encoding&#194;&#160;a&#194;&#160;2.5&#194;&#160;MDa&#194;&#160;modular&#194;&#160;extracellular&#194;&#160;matrix&#194;&#160;protein&#194;&#160;with&#194;&#160; <br /> 308&#194;&#160;EGF&#194;&#160;modules&#194;&#160;and&#194;&#160;185&#194;&#160;DPY&#194;&#160;modules&#194;&#160;required&#194;&#160;for&#194;&#160;the&#194;&#160;anchoring&#194;&#160;of&#194;&#160;epidermal&#194;&#160;tissue&#194;&#160;to&#194;&#160;the&#194;&#160;fly&#194;&#160;cuticle&#194;&#160;(WILKIN&#194;&#160;et&#194;&#160;al.&#194;&#160;2000).&#194;&#160;The&#194;&#160; <br /> 79&#194;&#160;dp&#194;&#160;coding&#194;&#160;exons&#194;&#160;are&#194;&#160;in&#194;&#160;53&#194;&#160;MPOs,&#194;&#160;and&#194;&#160;the&#194;&#160;region&#194;&#160;from&#194;&#160;exon&#194;&#160;37&#194;&#160;to&#194;&#160;70&#194;&#160;has&#194;&#160;34&#194;&#160;coding&#194;&#160;exons&#194;&#160;in&#194;&#160;13&#194;&#160;MPOs&#194;&#160;(Supplemental&#194;&#160;Figure&#194;&#160; <br /> 1).&#194;&#160;Due&#194;&#160;to&#194;&#160;extensive&#194;&#160;repeats&#194;&#160;within&#194;&#160;the&#194;&#160;gene&#194;&#160;the&#194;&#160;exact&#194;&#160;changes&#194;&#160;in&#194;&#160;coding&#194;&#160;exon&#194;&#160;to&#194;&#160;MPO&#194;&#160;organization&#194;&#160;within&#194;&#160;the&#194;&#160;genus&#194;&#160;Drosophila&#194;&#160; <br /> are&#194;&#160;difficult&#194;&#160;to&#194;&#160;determine&#194;&#160;precisely,&#194;&#160;all&#194;&#160;species&#194;&#160;have&#194;&#160;a&#194;&#160;similar&#194;&#160;multi&#226;&#8364;?exon&#194;&#160;MPO&#194;&#160;arrangement.&#194;&#160;A&#194;&#160;recent&#194;&#160;study&#194;&#160;of&#194;&#160;dp&#194;&#160;found&#194;&#160;the&#194;&#160;area&#194;&#160; <br /> containing&#194;&#160;the&#194;&#160;greatest&#194;&#160;concentration&#194;&#160;of&#194;&#160;multi&#226;&#8364;?exon&#194;&#160;MPOs&#194;&#160;is&#194;&#160;alternatively&#194;&#160;spliced,&#194;&#160;although&#194;&#160;the&#194;&#160;actual&#194;&#160;splicing&#194;&#160;is&#194;&#160;unknown&#194;&#160; <br /> (CARMON&#194;&#160;et&#194;&#160;al.&#194;&#160;2010).&#194;&#160;Because&#194;&#160;many&#194;&#160;dp&#194;&#160;genes&#194;&#160;in&#194;&#160;the&#194;&#160;genus&#194;&#160;Drosophila&#194;&#160;appear&#194;&#160;to&#194;&#160;have&#194;&#160;assembly&#194;&#160;errors&#194;&#160;creating&#194;&#160;large&#194;&#160;deletions,&#194;&#160; <br /> possibly&#194;&#160;due&#194;&#160;to&#194;&#160;the&#194;&#160;highly&#194;&#160;repetitive&#194;&#160;nature&#194;&#160;of&#194;&#160;dp&#194;&#160;protein&#194;&#160;modules,&#194;&#160;we&#194;&#160;were&#194;&#160;not&#194;&#160;able&#194;&#160;to&#194;&#160;do&#194;&#160;a&#194;&#160;complete&#194;&#160;analysis&#194;&#160;of&#194;&#160;the&#194;&#160; <br /> divergence&#194;&#160;of&#194;&#160;target&#194;&#160;MPOs.&#194;&#160; <br /> &#194;&#160;         The&#194;&#160;target&#194;&#160;MPOs&#194;&#160;in&#194;&#160;four&#194;&#160;of&#194;&#160;the&#194;&#160;genes&#194;&#160;identified,&#194;&#160;mushroom&#194;&#160;body&#194;&#160;defective&#194;&#160;(mud)&#194;&#160;(Supplemental&#194;&#160;Figure&#194;&#160;2),&#194;&#160;rhinocerous&#194;&#160; <br /> (rno)&#194;&#160;(Supplemental&#194;&#160;Figure&#194;&#160;3),&#194;&#160;lava&#194;&#160;lamp&#194;&#160;(lva)&#194;&#160;(Supplemental&#194;&#160;Figure&#194;&#160;4),&#194;&#160;and&#194;&#160;Futsch&#194;&#160;(Supplemental&#194;&#160;Figure&#194;&#160;5),&#194;&#160;span&#194;&#160;a&#194;&#160;small&#194;&#160;and&#194;&#160; <br /> large&#194;&#160;coding&#194;&#160;exon&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster.&#194;&#160;Interestingly,&#194;&#160;the&#194;&#160;ab&#194;&#160;initio&#194;&#160;models&#194;&#160;for&#194;&#160;these&#194;&#160;four&#194;&#160;genes&#194;&#160;are&#194;&#160;variable&#194;&#160;in&#194;&#160;the&#194;&#160;genus&#194;&#160; <br /> Drosophila,&#194;&#160;including&#194;&#160;a&#194;&#160;split&#194;&#160;of&#194;&#160;the&#194;&#160;gene&#194;&#160;into&#194;&#160;two&#194;&#160;separate&#194;&#160;transcription&#194;&#160;units&#194;&#160;and&#194;&#160;the&#194;&#160;elimination&#194;&#160;of&#194;&#160;relatively&#194;&#160;large&#194;&#160;regions&#194;&#160;of&#194;&#160; <br /> coding&#194;&#160;sequence.&#194;&#160;Most&#194;&#160;of&#194;&#160;the&#194;&#160;variability&#194;&#160;in&#194;&#160;these&#194;&#160;models&#194;&#160;is&#194;&#160;associated&#194;&#160;with&#194;&#160;the&#194;&#160;region&#194;&#160;covered&#194;&#160;by&#194;&#160;our&#194;&#160;target&#194;&#160;MPOs,&#194;&#160;suggesting&#194;&#160; <br /> these&#194;&#160;coding&#194;&#160;regions&#194;&#160;confound&#194;&#160;modeling&#194;&#160;algorithms,&#194;&#160;possibly&#194;&#160;due&#194;&#160;to&#194;&#160;a&#194;&#160;lack&#194;&#160;of&#194;&#160;strong&#194;&#160;splicing&#194;&#160;signals.&#194;&#160;However,&#194;&#160;the&#194;&#160;manual&#194;&#160; <br /> annotation&#194;&#160;of&#194;&#160;these&#194;&#160;genes&#194;&#160;finds&#194;&#160;little&#194;&#160;evidence&#194;&#160;to&#194;&#160;support&#194;&#160;significant&#194;&#160;changes&#194;&#160;relative&#194;&#160;to&#194;&#160;known&#194;&#160;transcripts&#194;&#160;from&#194;&#160;D.&#194;&#160; <br /> melanogaster.&#194;&#160; <br /> &#194;&#160;&#194;&#160;        The&#194;&#160;rno&#194;&#160;locus&#194;&#160;encodes&#194;&#160;a&#194;&#160;transcription&#194;&#160;factor&#194;&#160;predicted&#194;&#160;to&#194;&#160;be&#194;&#160;a&#194;&#160;chromatin&#226;&#8364;?remodeling&#194;&#160;protein&#194;&#160;required&#194;&#160;to&#194;&#160;restrict&#194;&#160;the&#194;&#160; <br /> Ras&#194;&#160;pathway&#194;&#160;during&#194;&#160;eye&#194;&#160;development&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;(Voas&#194;&#160;&amp;&#194;&#160;Rebay,&#194;&#160;2003).&#194;&#160;&#194;&#160;The&#194;&#160;target&#194;&#160;MPO&#194;&#160;encodes&#194;&#160;a&#194;&#160;long&#194;&#160;(3929&#194;&#160;aa)&#194;&#160; <br /> coiled&#226;&#8364;?coil&#194;&#160;peptide&#194;&#160;and&#194;&#160;is&#194;&#160;present&#194;&#160;in&#194;&#160;all&#194;&#160;Drosophila&#194;&#160;species&#194;&#160;used&#194;&#160;in&#194;&#160;this&#194;&#160;study,&#194;&#160;as&#194;&#160;are&#194;&#160;the&#194;&#160;five&#194;&#160;amino&#194;&#160;terminal&#194;&#160;coding&#194;&#160;exons.&#194;&#160;Based&#194;&#160; <br /> on&#194;&#160;these&#194;&#160;results&#194;&#160;there&#194;&#160;is&#194;&#160;no&#194;&#160;clear&#194;&#160;explanation&#194;&#160;for&#194;&#160;the&#194;&#160;significant&#194;&#160;variations&#194;&#160;in&#194;&#160;gene&#194;&#160;models.&#194;&#160; <br /> &#194;&#160;         The&#194;&#160;lva&#194;&#160;gene&#194;&#160;encodes&#194;&#160;a&#194;&#160;rapidly&#194;&#160;evolving&#194;&#160;coiled&#226;&#8364;?coil&#194;&#160;golgin&#194;&#160;protein&#194;&#160;required&#194;&#160;for&#194;&#160;Golgi&#194;&#160;vesicle&#194;&#160;transport&#194;&#160;and&#194;&#160; <br /> cellularization&#194;&#160;during&#194;&#160;embryogenesis&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;(SISSON&#194;&#160;et&#194;&#160;al.&#194;&#160;2000).&#194;&#160;The&#194;&#160;lva&#194;&#160;target&#194;&#160;MPO&#194;&#160;spans&#194;&#160;exon&#194;&#160;4&#194;&#160;and&#194;&#160;exon&#194;&#160;5&#194;&#160;in&#194;&#160;the&#194;&#160; <br /> Lva&#226;&#8364;?PC&#194;&#160;isoform&#194;&#160;but&#194;&#160;the&#194;&#160;intron&#194;&#160;between&#194;&#160;these&#194;&#160;exons&#194;&#160;is&#194;&#160;retained&#194;&#160;in&#194;&#160;an&#194;&#160;alternatively&#194;&#160;spliced&#194;&#160;variant&#194;&#160;(Sisson&#194;&#160;et&#194;&#160;al,&#194;&#160;2000).&#194;&#160;The&#194;&#160;only&#194;&#160; <br /> evidence&#194;&#160;we&#194;&#160;see&#194;&#160;for&#194;&#160;model&#194;&#160;changes&#194;&#160;in&#194;&#160;the&#194;&#160;genus&#194;&#160;involve&#194;&#160;fusions&#194;&#160;with&#194;&#160;the&#194;&#160;downstream&#194;&#160;adjacent&#194;&#160;exon&#194;&#160;and&#194;&#160;the&#194;&#160;possible&#194;&#160;loss&#194;&#160;of&#194;&#160;an&#194;&#160; <br /> intron.&#194;&#160;&#194;&#160;However,&#194;&#160;based&#194;&#160;on&#194;&#160;the&#194;&#160;exon6/7&#194;&#160;splicing&#194;&#160;in&#194;&#160;cnn&#194;&#160;(described&#194;&#160;in&#194;&#160;the&#194;&#160;main&#194;&#160;text)&#194;&#160;accurate&#194;&#160;models&#194;&#160;will&#194;&#160;require&#194;&#160;sequencing&#194;&#160; <br /> transcripts&#194;&#160;or&#194;&#160;an&#194;&#160;analysis&#194;&#160;of&#194;&#160;RNA&#226;&#8364;?Seq&#194;&#160;data.&#194;&#160;&#194;&#160;Nevertheless&#194;&#160;even&#194;&#160;in&#194;&#160;the&#194;&#160;absence&#194;&#160;of&#194;&#160;cDNA&#194;&#160;or&#194;&#160;RNA&#226;&#8364;?Seq&#194;&#160;data&#194;&#160;all&#194;&#160;Drosophila&#194;&#160;lva&#194;&#160;gene&#194;&#160; <br /> models&#194;&#160;appear&#194;&#160;to&#194;&#160;be&#194;&#160;similar&#194;&#160;to&#194;&#160;D.&#194;&#160;melanogaster.&#194;&#160; <br />  <br />  <br />  <br />  <br />                                                    R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;                                               7&#194;&#160;SI&#194;&#160; <br /> &#194;&#160; <br />  &#194;&#160;         The&#194;&#160;futsch&#194;&#160;locus&#194;&#160;encodes&#194;&#160;a&#194;&#160;cytoskeletal&#194;&#160;protein&#194;&#160;required&#194;&#160;for&#194;&#160;the&#194;&#160;development&#194;&#160;of&#194;&#160;axons&#194;&#160;and&#194;&#160;dendrites&#194;&#160;in&#194;&#160;D.&#194;&#160; <br /> melanogaster&#194;&#160;(HUMMEL&#194;&#160;et&#194;&#160;al.&#194;&#160;2000),&#194;&#160;and&#194;&#160;organizes&#194;&#160;the&#194;&#160;microtubule&#194;&#160;cytoskeleton&#194;&#160;during&#194;&#160;synaptic&#194;&#160;growth&#194;&#160;(ROOS&#194;&#160;et&#194;&#160;al.&#194;&#160;2000).&#194;&#160;The&#194;&#160; <br /> Futsch&#194;&#160;protein&#194;&#160;appears&#194;&#160;to&#194;&#160;be&#194;&#160;a&#194;&#160;chimera&#194;&#160;of&#194;&#160;vertebrate&#194;&#160;genes,&#194;&#160;comprised&#194;&#160;of&#194;&#160;vertebrate&#194;&#160;MAP1B&#226;&#8364;?like&#194;&#160;amino&#226;&#8364;?&#194;&#160;and&#194;&#160;carboxy&#226;&#8364;?termini&#194;&#160; <br /> separated&#194;&#160;by&#194;&#160;a&#194;&#160;repetitive&#194;&#160;central&#194;&#160;region&#194;&#160;similar&#194;&#160;to&#194;&#160;vertebrate&#194;&#160;neurofilament&#194;&#160;proteins&#194;&#160;(HUMMEL&#194;&#160;et&#194;&#160;al.&#194;&#160;2000).&#194;&#160;The&#194;&#160;central&#194;&#160;region&#194;&#160; <br /> contains&#194;&#160;60&#194;&#160;repeats&#194;&#160;of&#194;&#160;approximately&#194;&#160;37&#194;&#160;amino&#194;&#160;acids&#194;&#160;each&#194;&#160;that&#194;&#160;may&#194;&#160;be&#194;&#160;phosphorylation&#194;&#160;targets&#194;&#160;(HUMMEL&#194;&#160;et&#194;&#160;al.&#194;&#160;2000)&#194;&#160;and&#194;&#160;are&#194;&#160; <br /> encoded&#194;&#160;by&#194;&#160;Futsch&#226;&#8364;?PB&#194;&#160;exon&#194;&#160;6&#194;&#160;which&#194;&#160;is&#194;&#160;in&#194;&#160;our&#194;&#160;target&#194;&#160;MPO.&#194;&#160;Exon&#194;&#160;6&#194;&#160;is&#194;&#160;a&#194;&#160;continuous&#194;&#160;exon&#194;&#160;in&#194;&#160;all&#194;&#160;species&#194;&#160;except&#194;&#160;D.&#194;&#160;mojavensis&#194;&#160;and&#194;&#160;D.&#194;&#160; <br /> grimshawi,&#194;&#160;which&#194;&#160;both&#194;&#160;have&#194;&#160;single&#194;&#160;base&#194;&#160;changes&#194;&#160;creating&#194;&#160;a&#194;&#160;stop&#194;&#160;codon,&#194;&#160;and&#194;&#160;D.&#194;&#160;yakuba&#194;&#160;appears&#194;&#160;to&#194;&#160;have&#194;&#160;a&#194;&#160;small&#194;&#160;insertion&#194;&#160;and&#194;&#160; <br /> stop&#194;&#160;codon.&#194;&#160;While&#194;&#160;these&#194;&#160;changes&#194;&#160;may&#194;&#160;be&#194;&#160;real,&#194;&#160;since&#194;&#160;they&#194;&#160;occur&#194;&#160;in&#194;&#160;the&#194;&#160;highly&#194;&#160;repetitive&#194;&#160;sequence&#194;&#160;it&#194;&#160;is&#194;&#160;also&#194;&#160;possible&#194;&#160;they&#194;&#160;are&#194;&#160; <br /> sequencing&#194;&#160;or&#194;&#160;assembly&#194;&#160;errors.&#194;&#160;Surprisingly,&#194;&#160;even&#194;&#160;though&#194;&#160;this&#194;&#160;exon&#194;&#160;is&#194;&#160;similar&#194;&#160;in&#194;&#160;size&#194;&#160;to&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;in&#194;&#160;the&#194;&#160;other&#194;&#160;species,&#194;&#160;the&#194;&#160; <br /> gene&#194;&#160;models&#194;&#160;for&#194;&#160;several&#194;&#160;species&#194;&#160;remove&#194;&#160;approximately&#194;&#160;half&#194;&#160;of&#194;&#160;the&#194;&#160;exon&#194;&#160;from&#194;&#160;the&#194;&#160;5&#226;&#8364;&#178;&#194;&#160;end.&#194;&#160;&#194;&#160; <br /> &#194;&#160;         Of&#194;&#160;the&#194;&#160;four&#194;&#160;genes&#194;&#160;analyzed&#194;&#160;with&#194;&#160;multi&#226;&#8364;?exon&#194;&#160;MPOs,&#194;&#160;mud&#194;&#160;has&#194;&#160;the&#194;&#160;most&#194;&#160;complex&#194;&#160;and&#194;&#160;potentially&#194;&#160;exhibits&#194;&#160;real&#194;&#160;changes&#194;&#160;in&#194;&#160; <br /> splicing.&#194;&#160;The&#194;&#160;mud&#194;&#160;gene&#194;&#160;encodes&#194;&#160;a&#194;&#160;microtuble&#194;&#160;binding&#194;&#160;protein&#194;&#160;required&#194;&#160;at&#194;&#160;the&#194;&#160;meiosis&#194;&#160;II&#194;&#160;spindle&#194;&#160;in&#194;&#160;Drosophila&#194;&#160;embryos&#194;&#160;(YU&#194;&#160;et&#194;&#160;al.&#194;&#160; <br /> 2006),&#194;&#160;and&#194;&#160;is&#194;&#160;required&#194;&#160;for&#194;&#160;the&#194;&#160;proliferation&#194;&#160;(GUAN&#194;&#160;et&#194;&#160;al.&#194;&#160;2000)&#194;&#160;and&#194;&#160;asymmetric&#194;&#160;division&#194;&#160;of&#194;&#160;neuroblast&#194;&#160;cells&#194;&#160;(IZUMI&#194;&#160;et&#194;&#160;al.&#194;&#160;2006;&#194;&#160; <br /> SILLER&#194;&#160;et&#194;&#160;al.&#194;&#160;2006).&#194;&#160;Mud&#194;&#160;protein&#194;&#160;is&#194;&#160;predicted&#194;&#160;to&#194;&#160;be&#194;&#160;functional&#194;&#160;ortholog&#194;&#160;of&#194;&#160;mammalian&#194;&#160;NuMA&#194;&#160;based&#194;&#160;on&#194;&#160;conserved&#194;&#160;domains&#194;&#160;in&#194;&#160;the&#194;&#160; <br /> carboxy&#226;&#8364;?terminus&#194;&#160;of&#194;&#160;the&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;protein&#194;&#160;(SILLER&#194;&#160;et&#194;&#160;al.&#194;&#160;2006).&#194;&#160;&#194;&#160;Our&#194;&#160;target&#194;&#160;MPO&#194;&#160;spans&#194;&#160;the&#194;&#160;Mud&#226;&#8364;?PB&#194;&#160;large&#194;&#160;exons&#194;&#160;6&#194;&#160;and&#194;&#160;7,&#194;&#160; <br /> encoding&#194;&#160;a&#194;&#160;coiled&#226;&#8364;?coil&#194;&#160;domain&#194;&#160;(GUAN&#194;&#160;et&#194;&#160;al.&#194;&#160;2000).&#194;&#160;Based&#194;&#160;on&#194;&#160;protein&#194;&#160;alignments&#194;&#160;it&#194;&#160;appears&#194;&#160;the&#194;&#160;coding&#194;&#160;portions&#194;&#160;of&#194;&#160;the&#194;&#160;exons&#194;&#160;are&#194;&#160; <br /> fused&#194;&#160;in&#194;&#160;D.&#194;&#160;yakuba,&#194;&#160;D.&#194;&#160;erecta,&#194;&#160;D.&#194;&#160;pseudoobscura&#194;&#160;and&#194;&#160;D.&#194;&#160;grimshawi&#194;&#160;as&#194;&#160;removal&#194;&#160;of&#194;&#160;an&#194;&#160;intron&#194;&#160;would&#194;&#160;result&#194;&#160;in&#194;&#160;a&#194;&#160;significantly&#194;&#160;smaller&#194;&#160; <br /> protein&#194;&#160;compared&#194;&#160;to&#194;&#160;other&#194;&#160;species&#194;&#160;in&#194;&#160;this&#194;&#160;analysis.&#194;&#160;Additionally,&#194;&#160;these&#194;&#160;proteins&#194;&#160;are&#194;&#160;several&#194;&#160;hundred&#194;&#160;amino&#194;&#160;acids&#194;&#160;shorter&#194;&#160;than&#194;&#160; <br /> species&#194;&#160;with&#194;&#160;the&#194;&#160;intron.&#194;&#160;In&#194;&#160;D.&#194;&#160;mojavensis&#194;&#160;there&#194;&#160;are&#194;&#160;two&#194;&#160;exons&#194;&#160;but&#194;&#160;the&#194;&#160;3&#226;&#8364;&#178;&#194;&#160;end&#194;&#160;of&#194;&#160;exon&#194;&#160;6&#194;&#160;is&#194;&#160;fused&#194;&#160;to&#194;&#160;exon&#194;&#160;7,&#194;&#160;producing&#194;&#160;a&#194;&#160;much&#194;&#160; <br /> smaller&#194;&#160;peptide.&#194;&#160;Finally,&#194;&#160;in&#194;&#160;D.&#194;&#160;virilis&#194;&#160;there&#194;&#160;is&#194;&#160;a&#194;&#160;short&#194;&#160;fused&#194;&#160;single&#194;&#160;coding&#194;&#160;exon&#194;&#160;and&#194;&#160;an&#194;&#160;upstream&#194;&#160;insertion&#194;&#160;of&#194;&#160;approximately&#194;&#160;5&#194;&#160;kb,&#194;&#160; <br /> bearing&#194;&#160;no&#194;&#160;sequence&#194;&#160;similarity&#194;&#160;to&#194;&#160;any&#194;&#160;of&#194;&#160;the&#194;&#160;mud&#194;&#160;genes&#194;&#160;in&#194;&#160;this&#194;&#160;study.&#194;&#160;Our&#194;&#160;mud&#194;&#160;target&#194;&#160;MPO&#194;&#160;identified&#194;&#160;the&#194;&#160;region&#194;&#160;of&#194;&#160;Mud&#194;&#160;protein&#194;&#160; <br /> that&#194;&#160;led&#194;&#160;others&#194;&#160;to&#194;&#160;identify&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;as&#194;&#160;a&#194;&#160;functional&#194;&#160;ortholog&#194;&#160;of&#194;&#160;NuMA&#194;&#160;rather&#194;&#160;than&#194;&#160;an&#194;&#160;orthologous&#194;&#160;gene.&#194;&#160;In&#194;&#160;their&#194;&#160;study&#194;&#160; <br /> they&#194;&#160;showed&#194;&#160;conserved&#194;&#160;domains&#194;&#160;in&#194;&#160;Mud&#194;&#160;retained&#194;&#160;NuMA&#194;&#160;function&#194;&#160;but&#194;&#160;the&#194;&#160;coiled&#226;&#8364;?coil&#194;&#160;region&#194;&#160;had&#194;&#160;no&#194;&#160;similarity&#194;&#160;to&#194;&#160;vertebrate&#194;&#160; <br /> NuMA&#194;&#160;(GUAN&#194;&#160;et&#194;&#160;al.&#194;&#160;2000),&#194;&#160;which&#194;&#160;are&#194;&#160;80%&#194;&#160;identical&#194;&#160;between&#194;&#160;human&#194;&#160;and&#194;&#160;mouse&#194;&#160;(WHITE&#194;&#160;and&#194;&#160;ERICKSON&#194;&#160;2006).&#194;&#160;While&#194;&#160;our&#194;&#160;results&#194;&#160; <br /> neither&#194;&#160;strengthen&#194;&#160;nor&#194;&#160;weaken&#194;&#160;orthology&#194;&#160;arguments&#194;&#160;they&#194;&#160;do&#194;&#160;suggest&#194;&#160;the&#194;&#160;evolution&#194;&#160;of&#194;&#160;coiled&#226;&#8364;?coil&#194;&#160;domains&#194;&#160;may&#194;&#160;be&#194;&#160;more&#194;&#160;plastic&#194;&#160; <br /> in&#194;&#160;Drosophila&#194;&#160;compared&#194;&#160;to&#194;&#160;vertebrates.&#194;&#160; <br />           The&#194;&#160;last&#194;&#160;two&#194;&#160;genes&#194;&#160;in&#194;&#160;our&#194;&#160;screen&#194;&#160;were&#194;&#160;identified&#194;&#160;because&#194;&#160;the&#194;&#160;target&#194;&#160;MPOs&#194;&#160;spanned&#194;&#160;exons&#194;&#160;known&#194;&#160;to&#194;&#160;be&#194;&#160;alternatively&#194;&#160; <br /> spliced&#194;&#160;by&#194;&#160;an&#194;&#160;intron&#194;&#160;retention/exclusion&#194;&#160;mechanism.&#194;&#160;The&#194;&#160;prospero&#194;&#160;(pros)&#194;&#160;gene&#194;&#160;encodes&#194;&#160;a&#194;&#160;transcription&#194;&#160;factor&#194;&#160;required&#194;&#160;for&#194;&#160;cell&#194;&#160; <br /> fate&#194;&#160;specification&#194;&#160;during&#194;&#160;central&#194;&#160;nervous&#194;&#160;system&#194;&#160;development&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster&#194;&#160;(CHU&#226;&#8364;?LAGRAFF&#194;&#160;et&#194;&#160;al.&#194;&#160;1991).&#194;&#160;Orthologous&#194;&#160;genes&#194;&#160; <br /> of&#194;&#160;the&#194;&#160;Pros/Prox1&#194;&#160;family&#194;&#160;all&#194;&#160;contain&#194;&#160;a&#194;&#160;conserved&#194;&#160;COOH&#194;&#160;terminus&#194;&#160;which&#194;&#160;contains&#194;&#160;the&#194;&#160;structurally&#226;&#8364;?unique&#194;&#160;Homeo&#226;&#8364;?Prospero&#194;&#160; <br /> domain&#194;&#160;(RYTER&#194;&#160;et&#194;&#160;al.&#194;&#160;2002),&#194;&#160;and&#194;&#160;motifs&#194;&#160;required&#194;&#160;for&#194;&#160;the&#194;&#160;complex&#194;&#160;regulation&#194;&#160;of&#194;&#160;the&#194;&#160;protein&#194;&#160;(BI&#194;&#160;et&#194;&#160;al.&#194;&#160;2003).&#194;&#160;&#194;&#160;The&#194;&#160;amino&#194;&#160;half&#194;&#160;of&#194;&#160;D.&#194;&#160; <br /> melanogaster&#194;&#160;Pros&#194;&#160;protein&#194;&#160;contains&#194;&#160;two&#194;&#160;small&#194;&#160;conserved&#194;&#160;motifs&#194;&#160;present&#194;&#160;in&#194;&#160;chicken&#194;&#160;Prox1,&#194;&#160;but&#194;&#160;shares&#194;&#160;no&#194;&#160;other&#194;&#160;similarities&#194;&#160; <br /> upstream&#194;&#160;of&#194;&#160;the&#194;&#160;homeodomain&#194;&#160;(TOMAREV&#194;&#160;et&#194;&#160;al.&#194;&#160;1996).&#194;&#160;The&#194;&#160;pros&#194;&#160;target&#194;&#160;MPO&#194;&#160;spans&#194;&#160;the&#194;&#160;exon&#194;&#160;encoding&#194;&#160;the&#194;&#160;amino&#194;&#160;half&#194;&#160;of&#194;&#160;Pros&#194;&#160;and&#194;&#160; <br /> accounts&#194;&#160;for&#194;&#160;the&#194;&#160;majority&#194;&#160;of&#194;&#160;the&#194;&#160;changes&#194;&#160;between&#194;&#160;Drosophila&#194;&#160;species.&#194;&#160;Protein&#194;&#160;alignments&#194;&#160;between&#194;&#160;Drosophila&#194;&#160;species&#194;&#160;show&#194;&#160;the&#194;&#160; <br /> encoded&#194;&#160;peptide&#194;&#160;is&#194;&#160;a&#194;&#160;mix&#194;&#160;of&#194;&#160;high&#194;&#160;conservation&#194;&#160;interspersed&#194;&#160;with&#194;&#160;regions&#194;&#160;of&#194;&#160;nearly&#194;&#160;100%&#194;&#160;divergence&#194;&#160;and&#194;&#160;gaps&#194;&#160;accumulate&#194;&#160;with&#194;&#160; <br /> phylogenetic&#194;&#160;distance,&#194;&#160;similar&#194;&#160;to&#194;&#160;all&#194;&#160;MPOs&#194;&#160;identified&#194;&#160;in&#194;&#160;the&#194;&#160;screen.&#194;&#160; <br /> &#194;&#160;         The&#194;&#160;last&#194;&#160;gene&#194;&#160;in&#194;&#160;this&#194;&#160;study&#194;&#160;is&#194;&#160;shortstop&#194;&#160;(shot),&#194;&#160;a&#194;&#160;transcriptionally&#194;&#160;complex&#194;&#160;gene,&#194;&#160;which&#194;&#160;is&#194;&#160;a&#194;&#160;member&#194;&#160;of&#194;&#160;the&#194;&#160;Plakin&#194;&#160; <br /> protein&#194;&#160;family.&#194;&#160;Shot&#194;&#160;proteins&#194;&#160;link&#194;&#160;the&#194;&#160;actin&#194;&#160;and&#194;&#160;microtubule&#194;&#160;cytoskeleton&#194;&#160;to&#194;&#160;a&#194;&#160;variety&#194;&#160;of&#194;&#160;junctional&#194;&#160;complexes&#194;&#160;in&#194;&#160;different&#194;&#160;cell&#194;&#160; <br /> types&#194;&#160;(SONNENBERG&#194;&#160;and&#194;&#160;LIEM&#194;&#160;2007)&#194;&#160;required&#194;&#160;for&#194;&#160;cytoskeleton&#194;&#160;organization&#194;&#160;during&#194;&#160;neurogenesis&#194;&#160;and&#194;&#160;in&#194;&#160;muscles&#194;&#160;and&#194;&#160;tendon&#194;&#160;cells&#194;&#160; <br />  <br />  <br /> 8&#194;&#160;SI                                              R.&#194;&#160;C.&#194;&#160;Eisman&#194;&#160;and&#194;&#160;T.&#194;&#160;C.&#194;&#160;Kaufman&#194;&#160;&#194;&#160; <br /> &#194;&#160; <br />  (SUBRAMANIAN&#194;&#160;et&#194;&#160;al.&#194;&#160;2003).&#194;&#160;The&#194;&#160;central&#194;&#160;region&#194;&#160;of&#194;&#160;Shot&#194;&#160;long&#194;&#160;isoforms&#194;&#160;form&#194;&#160;a&#194;&#160;long&#194;&#160;coiled&#194;&#160;coil&#194;&#160;(LEE&#194;&#160;et&#194;&#160;al.&#194;&#160;2000),&#194;&#160;a&#194;&#160;region&#194;&#160;partially&#194;&#160; <br /> covered&#194;&#160;by&#194;&#160;the&#194;&#160;shot&#194;&#160;MPO.&#194;&#160;Alternative&#194;&#160;3&#226;&#8364;&#178;&#194;&#160;splicing&#194;&#160;of&#194;&#160;our&#194;&#160;target&#194;&#160;MPO&#194;&#160;in&#194;&#160;shot&#194;&#160;produces&#194;&#160;eighteen&#194;&#160;mRNAs&#194;&#160;that&#194;&#160;include&#194;&#160;the&#194;&#160;5&#226;&#8364;&#178;&#194;&#160; <br /> terminus&#194;&#160;of&#194;&#160;the&#194;&#160;exon&#194;&#160;encoding&#194;&#160;a&#194;&#160;conserved&#194;&#160;peptide&#194;&#160;(154&#194;&#160;aa),&#194;&#160;whereas&#194;&#160;the&#194;&#160;Shot&#226;&#8364;?RH&#194;&#160;mRNA&#194;&#160;includes&#194;&#160;the&#194;&#160;entire&#194;&#160;exon&#194;&#160;encoding&#194;&#160;a&#194;&#160; <br /> large&#194;&#160;(3499&#194;&#160;aa&#194;&#160;in&#194;&#160;D.&#194;&#160;melanogaster)&#194;&#160;divergent&#194;&#160;peptide.&#194;&#160;Three&#194;&#160;Shot&#194;&#160;splice&#194;&#160;variants&#194;&#160;skip&#194;&#160;this&#194;&#160;exon&#194;&#160;entirely.&#194;&#160;Interestingly,&#194;&#160;27&#194;&#160;exons&#194;&#160; <br /> common&#194;&#160;to&#194;&#160;all&#194;&#160;is <br /> </body></html>