<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, May 2011, p. 1982&#226;&#8364;&#8220;1989                                                                      Vol. 55, No. 5 <br /> 0066-4804/11/$12.00 doi:10.1128/AAC.01796-10 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />        Selective Inhibitors Methionyl-tRNA Synthetase Have Potent <br />            Activity Trypanosoma brucei Infection Mice&#228;&#338;&#164;&#226;&#8364;&#160; <br />  Sayaka Shibata,1 J. Robert Gillespie,2 Angela M. Kelley,2 Alberto J. Napuli,2 Zhongsheng Zhang,1 <br />     Kuzma V. Kovzun,2 Ranae M. Pefley,2 Jocelyn Lam,2 Frank H. Zucker,1 Wesley C. Van Voorhis,2 <br />                   Ethan A. Merritt,1 Wim G. J. Hol,1 Christophe L. M. J. Verlinde,1 <br />                              Erkang Fan,1 Frederick S. Buckner2* <br />           Department Biochemistry1 Department Medicine,2 University Washington, Seattle, Washington 98195 <br />                       Received 21 December 2010/Returned modification 19 January 2011/Accepted 24 January 2011 <br />  <br />              Human African trypanosomiasis continues important public health threat extensive regions of <br />           sub-Saharan Africa. Treatment options infected patients unsatisfactory toxicity, difficult ad- <br />           ministration regimes, poor efficacy available drugs. The aminoacyl-tRNA synthetases selected as <br />           attractive drug targets essential roles protein synthesis cell <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>survival</span>. Comparative sequence <br />           analysis disclosed differences trypanosome mammalian methionyl-tRNA synthetases <br />           (MetRSs) suggested opportunities selective inhibition using drug-like molecules. Experiments using <br />           <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>RNA interference</span> single MetRS Trypanosoma brucei demonstrated gene product was <br />           essential normal cell growth. Small molecules (diaryl diamines) similar shown potent <br />           activity prokaryotic MetRS enzymes synthesized observed inhibitory activity T. <br />           brucei MetRS (50% inhibitory concentration, &lt;50 nM) bloodstream forms T. brucei cultures (50% <br />           effective concentration, low 4 nM). Twenty-one compounds close correlation enzyme <br />           binding/inhibition T. brucei growth inhibition, indicating likely acting intended <br />           target. The compounds minimal effects mammalian cell growth 20 &#226;?&#174;M, demonstrating wide <br />           therapeutic index. The potent compound tested murine model trypanosomiasis and <br />           demonstrated profound parasite suppression delayed mortality. A homology model T. brucei MetRS <br />           based MetRS structures used model binding lead diaryl diamine compounds. Future <br />           studies focus improving pharmacological properties MetRS inhibitors. <br />  <br />  <br />    Drugs meet modern standards chemotherapeutics                       sequences proteins. With notable exceptions (27), <br /> have, unfortunately, developed human African                   aaRS specifically recognize single amino acid and <br /> trypanosomiasis (HAT). The currently used drugs HAT                         attach corresponding tRNA anticodon matches <br /> include suramin (an injection drug introduced 1922),                         codons amino acid. The sequence <br /> melarsoprol (a highly toxic arsenical use 1949), pent-                 reactions catalyzed consists essential steps: (i) rec- <br /> amidine (introduced 1940s given painful intra-                    ognition correct amino acid, (ii) reaction ATP <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> muscular injections), eflornithine (introduced 1980s                 form aminoacyl-adenylate intermediate, (iii) recognition of <br /> given intravenously 6 h multigram quantities),                     cognate tRNA, (iv) transfer aminoacyl group to <br /> nifurtimox (a drug originally developed Chagas&#226;&#8364;&#8482; disease                  terminal adenosine tRNA. Interference of <br /> the 1970s used combination eflornithine                     steps inhibits formation properly charged tRNA, <br /> HAT). With 60 million people Africa risk HAT,                     leading accumulation uncharged tRNA ribosome. <br /> need effective, safe, affordable drugs remains acute                 This results disruption normal protein chain elongation <br /> as ever. The completion Trypanosoma brucei genome                        translation. Not surprisingly, aaRS enzymes been <br /> sequencing project combined advances modern drug                        shown essential genetic <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> knockdown stud- <br /> discovery techniques creates unprecedented opportunity                    ies variety organisms (3, 4, 10, 11). <br /> develop overdue, modern drugs lethal disease.                             T. brucei encodes 23 aaRS enzymes, amino acid, <br />    Analysis sequenced T. brucei genome reveals hun-                      exception 2 enzymes aspartate (4), 2 lysine <br /> dreds genes encoding enzymes, in-                      (8), 2 tryptophan (3). The methionyl-RS (MetRS) T. <br /> teresting drug development standpoint amino- <br />                                                                                 brucei particularly attracted signif- <br /> acyl-tRNA synthetases (aaRSs). These enzymes essential <br />                                                                                 icant differences mammalian orthologs (discussed <br /> to core biological process translating gene nucleotide <br />                                                                                 herein) existence compounds under <br />                                                                                 development Pharma targeting homologous enzyme in <br />    * Corresponding author. Mailing address: University Washington,           bacteria. MetRS enzymes general divided major <br /> Division Allergy Infectious Diseases, Box 357185, Seattle, WA            forms basis sequence similarity sensitivity to <br /> 98195. Phone: (206) 616-9214. Fax: (206) 685-6045. E-mail: fbuckner             inhibitors (3, 12). MetRS1 form commonly in <br /> @uw.edu. <br />    &#226;&#8364;&#160; Supplemental material article http://aac <br />                                                                                 Gram-positive bacteria, including Staphylococcus aureus and <br /> .asm.org/.                                                                      Streptococcus pyogenes. MetRS2 archaea, eu- <br />    &#228;&#338;&#164; <br />      Published ahead print 31 January 2011.                               karyotes, Gram-negative bacteria. Eukaryotic organ- <br />  <br />                                                                          1982 <br />  VOL. 55, 2011                                                                   T. BRUCEI METHIONYL-tRNA SYNTHETASE INHIBITORS                                           1983 <br />  <br />  <br /> isms contain MetRS forms, MetRS1 mito-                            lowed size exclusion gel chromatography (Superdex 75 26/60; GE Biosciences, <br /> chondrial form MetRS2 cytoplasmic form. In                              Piscataway, NJ). <br />                                                                                          Thermal shift assay. The thermal shift assay performed previously de- <br /> mammals, mitochondrial aaRS enzymes encoded                            scribed (7), modifications samples contained MetRS enzyme (0.42 <br /> nucleus imported mitochondrion import signals.                         mg/ml T. brucei enzyme, 0.32 mg/ml T. cruzi enzyme), 50 &#226;?&#174;M inhibitor, <br /> The single MetRS T. brucei encoded nucleus                           10 mM ATP magnesium salt, 10 mM L-methionine, 5% dimethyl sulfoxide <br /> presumably functions cytoplasm mitochon-                          (DMSO). The inclusion 10 mM ATP methionine assay buffer to <br />                                                                                       suppress overall thermal shift signal tight-binding inhibitors could <br /> drion (26). On basis sequence, T. brucei MetRS <br />                                                                                       cause significant shift melting temperature (Tm) MetRS. The assays were <br /> enzyme groups MetRS1 form; thus, distinct                         performed twice independently triplicate experiment. <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> mammalian cytoplasmic MetRS enzyme. Differences                                 Aminoacylation assays. Enzyme activity quantified attachment of <br /> with human mitochondrial MetRS discussed paper.                       [3H]methionine tRNA presence MetRS enzymes. Reactions were <br />    Synthetic compounds targeting bacterial MetRS enzymes                              performed 96-well filter plates Durapore membranes (MSHVN4B10; <br />                                                                                       Millipore) volumes 75 &#226;?&#174;l. The reaction performed 25 mM HEPES, <br /> are advanced preclinical development treating bac-                         pH 7.9, 10 mM MgCl2, 50 mM KCl, 0.2 mM spermine, 0.1 mg/ml bovine serum <br /> terial infections caused methicillin-resistant Staphylococcus                      albumin, 2.5 mM dithiothreitol, 10 U/ml pyrophosphatase (R4251; Sigma). <br /> aureus Clostridium difficile (5, 6). These originated diaryl                   Recombinant enzyme (4 nM) compound inhibitors (50 nM) mixed with <br /> diamine compounds development GlaxoSmithKline                                buffer preincubated 15 min. To start reaction, 400 &#226;?&#174;g/ml bulk <br />                                                                                       Escherichia coli tRNA (R4251; Sigma), 0.2 mM ATP, 250 nM [3H]methio- <br /> (10, 15, 16) subsequently Replidyne (6, 13). We                              (80 Ci/mmol) added. The plate incubated shaking room <br /> synthesized compounds relating diaryl diamines                             temperature 120 min. The reactions stopped addition 100 &#226;?&#174;l <br /> tested binding recombinant T. brucei MetRS,                           cold 10% trichloroacetic acid. The reaction components separated from <br /> on T. brucei cell cultures, murine model T. brucei                      tRNA filtration vacuum manifold washed times cold <br />                                                                                       10% trichloroacetic acid. The filter plates dried overnight, scintillation fluid <br /> infection. This report includes results molecular <br />                                                                                       added, counts plates determined scintillation plate <br /> modeling studies compounds RNA inter-                             counter. Samples run quadruplicate, average activity inhibitors <br /> ference (RNAi) studies establishing essentiality                           compared control wells inhibitors. <br /> MetRS enzyme T. brucei.                                                               Growth inhibition assays T. brucei, T. cruzi, mammalian cell cultures. <br />                                                                                       T. brucei (bloodstream form strain 427 K. Stuart, Seattle Biomedical Research <br />                                                                                       Institute, Seattle, WA) cultured HMI-9 medium containing 10% fetal bovine <br />                         MATERIALS AND METHODS                                         serum, penicillin, streptomycin 37&#194;&#176;C 5% CO2 (14). Drug sensitivity the <br />                                                                                       T. brucei strain determined 96-well microtiter plates triplicate initial <br />    Synthesis diaryl diamines. Synthesis inhibitors followed literature-     inoculum 1 &#226;&#171;&#187; 104 trypomastigotes well. Compound stock solutions were <br /> reported route. Briefly, aniline condensed malonic acid phosphorus        prepared DMSO 20 mM added serial dilutions final volume 200 <br /> oxychloride 2,4-dichloroquinoline (24). Subsequently, reaction 4-        &#226;?&#174;l/well. Parasite growth quantified 48 h addition AlamarBlue <br /> methoxybenzyl alcohol activated NaH performed dry tetrahydrofuran         (Alamar Biosciences, Sacramento, CA) (25). Pentamidine isethionate (Aventis, <br /> in presence 15-crown-5 produce 2-chloro-4-(4-methoxybenzyloxy)-quin-        Dagenham, United Kingdom) included assay positive control. <br /> oline (16), reacted excess 1,3-diaminopropane N1-[4-(4-        Standard errors assays consistently 15%. <br /> methoxy-benzyloxy)-quinolin-2-yl]-propane-1,3-diamine (15). Deprotection             Compounds screened Tulahuen strain T. cruzi expressing <br /> performed 2:8 trifluoroacetic acid-dichloromethane, condensation        &#226;?&#164;-galactosidase 96-well tissue culture plates described previously (2). In this <br /> a variety substituted unsubstituted benzaldehydes reductive amina-        assay, T. cruzi proliferates intracellular amastigotes murine 3T3 fibroblasts. <br /> tion conditions (2.5% acetic acid methanol NaCNBH3) gave final             Compounds screened triplicate determine 50% effective concentrations <br /> compounds, purification (15, 16). The newly synthesized compounds gave          (EC50s). Standard errors assays consistently 15%. <br /> satisfactory analytical data (details provided supplemental material).      The human lymphocytic cell line CRL-1855 (American Type Culture Collec- <br />    RNA interference. The T. brucei MetRS homolog (Tb927.10.1500)              tion) used cytotoxicity assessment. The cells grown RPMI <br /> target RNAi knockdown experiment. The region bases 994            medium 10% fetal bovine serum, penicillin, streptomycin 37&#194;&#176;C with <br /> 1615 open reading frame amplified (primers 1 2 Table S1      5% CO2. Cells (5 &#226;&#171;&#187; 103/well) added 96-well plates incubated with <br /> supplemental material) cloned 2T7TABlue (a gift D. Horn, London           serial dilutions compounds 48 h. At time, cell viability quantified <br /> School Hygiene Tropical Medicine) (1). Transfections RNAi exper-           addition AlamarBlue, plates incubated additional 4 h 5% <br /> iments performed previously described (23).                                   CO2, 37&#194;&#176;C. Absorbance readings obtained optical density 570 600 nm <br />    Northern analysis. RNA isolated induced noninduced cultures           used calculate viability referenced cells grown inhibitors. <br /> after 72 h growth using Qiagen RNEasy kit (no. 74104). Total RNA (11 &#226;?&#174;g             Washout experiments T. brucei cultures. T. brucei cells (strain culture <br /> per lane) separated formaldehyde gel blotted using standard              conditions described above) grown 1.5-ml volumes 12-well plates in <br /> procedures. The RNA membrane analyzed 32P-labeled DNA                 presence compound 2 pentamidine. The concentrations com- <br /> probe region MetRS overlapping portion used         pounds represented factors EC50s determined 48-h incuba- <br /> RNA interference. The probe fragment amplified primers 3 4 (see          tion. At defined times (6, 12, 24, 36, 48 h), cells removed, centrifuged, <br /> Table S1 supplemental material), correspond bases 30 657        washed, resuspended medium compounds. The cultures were <br /> the TbMetRS open reading frame. The blot stripped reprobed               allowed grow 48-h time point exposure com- <br /> DNA T. brucei &#226;?&#164;-tubulin gene standardization (see primers 5 6       pounds, time viability cultures assessed using AlamarBlue <br /> Table S1 supplemental material).                                               described above. The viability cells expressed percentage the <br />    PCR ligase-independent cloning. The gene coding T. brucei methionyl-       viability cells grown compounds 48 h. In order determine <br /> tRNA synthetase (Tb927.10.1500) amplified T. brucei brucei strain 427        time needed kill parasites, aliquots cultures (plus an <br /> genomic DNA using primers 7 8 (see Table S1 supplemental               additional aliquot taken cells grown compounds 72 h) were <br /> material). The gene coding amino acids 219 765 T. cruzi methionyl-      incubated medium compounds 10 days. The cultures were <br /> tRNA synthetase (Tc00.1047053509247.50) amplified T. cruzi TCCL              inspected light microscopy daily outgrowth living parasites. This gave <br /> Brener genomic DNA primers 9 10 (see Table S1 supplemental            sufficient time outgrowth single surviving parasite, based the <br /> material). The PCR products gel purified Qiagen kit cloned       findings previous experiments. The conditions parasite outgrowth <br /> the AVA0421 vector ligation-independent cloning described previously            observed marked (see Fig. 3). <br /> (21). The AVA0421 vector derivative pET14b vector containing              Pharmacokinetic efficacy studies mice. For pharmacokinetic studies, <br /> N-terminal 6-histidine tag followed 3C protease cleavage site.                   compound 1 dissolved 10% DMSO injected intraperitoneal <br />    Production recombinant T. brucei T. cruzi MetRSs. The expression         (i.p.) route 50 mg/kg body weight BALB/c mice (7- 8-week-old <br /> recombinant proteins performed previously described (21). The N-termi-         females, n &#226;&#171;&#189; 3). At timed intervals, 20 &#226;?&#174;l tail blood collected hepa- <br /> nal 6-His fusion proteins purified nickel affinity chromatography fol-        rinized capillary tubes. Plasma separated frozen later analysis by <br />  1984       SHIBATA ET AL.                                                                                                 ANTIMICROB. AGENTS CHEMOTHER. <br />  <br /> liquid chromatography-mass spectrometry (28). For efficacy studies, compound 1 <br /> was loaded Alzet minipumps (model 1003D) implanted subcutaneously <br /> into BALB/c mice (7- 8-week-old females, n &#226;&#171;&#189; 5 group) delivery 25 <br /> mg/kg/day 3 days. The control group given minipumps loaded with <br /> vehicle. Mice infected T. brucei strain 427 (1 &#226;&#171;&#187; 104 i.p.) 1 day prior to <br /> implantation minipumps. Parasitemia quantified blood from <br /> tail bleeding using hemacytometer. Blood levels compound 1 obtained <br /> from mice minipumps place 24 h. Concentrations of <br /> compound 1 analyzed plasma described above. <br />    Homology modeling docking studies. Homology modeling using <br /> the MODELLER software program (19). Docking studies utilized FLO/QXP <br /> docking program Metropolis Monte Carlo scheme (20). Additional details of <br /> building homology model provided supplemental material. <br />  <br />  <br />  <br />                                   RESULTS <br />    Target validation RNA interference. T. brucei amenable <br /> to RNA interference analysis establish essentiality of <br /> gene targets cell viability. Gene knockdown MetRS <br /> (Tb927.10.1500 gene) resulted dramatic growth inhibition <br /> <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>; day 8, growth inhibited factor &#226;&#172;&#402;106 <br /> (Fig. 1). Total RNA collected 72 h, Northern blots <br /> demonstrated MetRS mRNA knocked by <br /> between 57 77% compared level control cells. <br /> Thus, complete knockdown MetRS tran- <br /> script, cell growth profoundly inhibited. <br />    Binding compounds T. brucei MetRS, inhibition of <br /> MetRS enzyme, inhibition trypanosome cell cultures. A <br /> set diaryl diamines (compounds 2, 3, 7) synthe- <br /> sized tested binding recombinant T. brucei MetRS <br />                                                                                          FIG. 1. Growth T. brucei undergoing RNAi MetRS. (Top) <br /> using thermal shift assay (Table 1). The initial set com-                        A region &#226;&#172;&#402;500 nucleotides 3&#226;&#172;&#732; end gene targeted using <br /> pounds demonstrated large thermal shifts (&#226;&#338;&#172;Tm) T.                              p2T7 expression vector (1). RNAi induced (&#226;&#171;&#185;) addition of <br /> brucei MetRS, exceeding 7&#194;&#176;C. A &#226;&#338;&#172;Tm exceeding 2&#194;&#176;C usually                           tetracycline medium left uninduced (&#226;&#171;&#186;). The growth rate two <br /> considered significant. These compounds tested                          separate clones (MET 1 MET 2) tested compared of <br />                                                                                       single-marker (SM) control, does contain RNAi con- <br /> activity T. brucei cultures remarkable                           struct. Note y axis log scale. (Bottom) Northern blots the <br /> potency, particularly compound 2, EC50 8 nM. As                          reduction mRNA levels T. brucei MetRS (TbMetRS) two <br /> result, additional 18 compounds synthesized inves-                         clones induced tetracycline (TCN&#226;&#171;&#185;) induced with <br /> tigate structure-activity relationships, particularly focusing                     tetracycline (TCN&#226;&#171;&#186;) 72 h. Approximately equivalent loading the <br /> the phenyl ring left scaffold, shown                         samples demonstrated Northern blots constitutively ex- <br />                                                                                       pressed &#226;?&#164;-tubulin (Tb&#226;?&#164;-Tubulin) mRNA. Densitometry T. brucei <br /> Table 1. By substituting 3,5-bromines 3,5-chlorines                          MetRS mRNA (normalized &#226;?&#164;-tubulin) showed 77% 57% signal <br /> (compound 1), activity T. brucei cells                        knockdowns clone 1 clone 2, respectively. <br /> improved EC50 4 nM. The activities 21 compounds <br /> are displayed Table 1. Binding compounds the <br /> recombinant enzyme, measured thermal melting, and <br /> EC50s T. brucei cultures highly correlated (Fig. 2). An                          Washout experiments. In order determine exposure <br /> enzymatic assay developed directly measure                            time required kill T. brucei MetRS inhibitor, T. brucei <br /> aminoacylation activity T. brucei MetRS. The com-                              cultures incubated presence compound 2 or <br /> pounds tested enzyme 50 nM, inhibition                            pentamidine defined time periods transferred to <br /> observed high 99% (compound 1; Table 1). Enzyme                           drug-free medium growth quantification 48 h (Fig. 3A). <br /> inhibition data highly correlated thermal melt                          Incubation compound 2 24 h concentrations 2 times <br /> shifts (R2 &#226;&#171;&#189; 0.821, P &#226;&#172;? 0.0001) EC50s T. brucei                       EC50 resulted &#226;&#172;&#381;95% growth suppression, indicating that <br /> cultures (R2 &#226;&#171;&#189; 0.646, P &#226;&#172;? 0.0001). The compounds did                              relative short exposure times lead profound growth suppres- <br /> inhibit growth mammalian cell line CRL-8155 con-                            sion. However, cultures left incubate 10 <br /> centrations high 20 &#226;?&#174;M, indicating large selectivity                     days compound washed out, parasites eventually <br /> index 5,000 (Table 1).                                                  grew conditions, unless cells were <br />    In addition, binding compounds recombinant T.                        exposed 8 times EC50 72 h, 16 times EC50 for <br /> cruzi MetRS inhibition T. cruzi amastigote growth                         48 h, 64 times EC50 36 h (see values marked with <br /> evaluated (Table 1). Again, substantial &#226;&#338;&#172;Tm shifts                           asterisks Fig. 3A). For comparison, HAT drug pentam- <br /> 12.0&#194;&#176;C observed compounds interacted                               idine completely trypanocidal cultures ex- <br /> enzyme. There potent activity T. cruzi amastigotes,                       posed 8 times EC50 48 h 16 times EC50 36 h <br /> although EC50s higher seen T. brucei                           (Fig. 3B). Thus, MetRS inhibitor slightly rapid <br /> cells factor &#226;&#172;&#402;10 40.                                                       pentamidine exerting trypanocidal effects. <br />  VOL. 55, 2011                                                            T. BRUCEI METHIONYL-tRNA SYNTHETASE INHIBITORS                               1985 <br />  <br />  <br />                  TABLE 1. Binding inhibitory activity diaryl diamine compounds MetRS enzymes cell culturesa <br />  <br />  <br />  <br />  <br />                         Substituent ring position:                               % inhibition of <br />                                                                 T. brucei MetRS                       T. brucei   T. cruzi MetRS    T. cruzi    Mammalian <br />                                                                                        T. brucei <br /> Compound                                                          &#226;&#338;&#172;Tm &#226;&#171;&#190; SEM                             EC50       &#226;&#338;&#172;Tm &#226;&#171;&#190; SEM         EC50        cell EC50 <br />                                                                                      MetRS 50 <br />                   2            3         4         5       6          (&#194;&#176;C)                             (nM)             (&#194;&#176;C)         (nM)          (nM) <br />                                                                                      nM &#226;&#171;&#190; SEM <br />  <br />       1                      Cl                   Cl              12.9 &#226;&#171;&#190; 0.04        99.2 &#226;&#171;&#190; 0.09            4      12.0 &#226;&#171;&#190; 0.03           175     &#226;&#172;&#381;20,000 <br />       2                      Br                   Br              12.9 &#226;&#171;&#190; 0.15        98.8 &#226;&#171;&#190; 0.18            8      11.3 &#226;&#171;&#190; 0.09           310     &#226;&#172;&#381;20,000 <br />       3         OOEt         Br                   Br              12.2 &#226;&#171;&#190; 0.19        98.2 &#226;&#171;&#190; 0.15           30      11.3 &#226;&#171;&#190; 0.04           876     &#226;&#172;&#381;20,000 <br />       4                      Cl                                    7.7 &#226;&#171;&#190; 0.04        97.3 &#226;&#171;&#190; 0.05           35       7.9 &#226;&#171;&#190; 0.07           842     &#226;&#172;&#381;20,000 <br />       5                     OCH3                OCH3               7.8 &#226;&#171;&#190; 0.01        91.1 &#226;&#171;&#190; 0.97           70       7.5 &#226;&#171;&#190; 0.07           452     &#226;&#172;&#381;20,000 <br />       6          Cl                              Cl                8.8 &#226;&#171;&#190; 0.01        93.1 &#226;&#171;&#190; 0.23           73       9.8 &#226;&#171;&#190; 0.05           485     &#226;&#172;&#381;20,000 <br />       7                       Cl        Cl                        10.1 &#226;&#171;&#190; 0.03        91.7 &#226;&#171;&#190; 0.94           75       8.7 &#226;&#171;&#190; 0.04           830     &#226;&#172;&#381;20,000 <br />  <br />       8                       Br                                   7.1 &#226;&#171;&#190; 0.06        82.7 &#226;&#171;&#190; 4.76           75       7.1 &#226;&#171;&#190; 0.04         2,140     &#226;&#172;&#381;20,000 <br />       9        Cl             Cl                                   6.9 &#226;&#171;&#190; 0.04        77.9 &#226;&#171;&#190; 1.42           95       7.7 &#226;&#171;&#190; 0.24         1,230     &#226;&#172;&#381;20,000 <br />       10      OOCH3           Br                  Br               9.4 &#226;&#171;&#190; 0.07        95.0 &#226;&#171;&#190; 0.29          100       8.8 &#226;&#171;&#190; 0.07           920     &#226;&#172;&#381;20,000 <br />       11       Cl                       Cl                         6.1 &#226;&#171;&#190; 0.04        87.1 &#226;&#171;&#190; 0.96          133       7.3 &#226;&#171;&#190; 0.05         1,230     &#226;&#172;&#381;20,000 <br />       12                     F                     F               5.6 &#226;&#171;&#190; 0.04        45.8 &#226;&#171;&#190; 9.22          200       6.4 &#226;&#171;&#190; 0.03         1,920     &#226;&#172;&#381;20,000 <br />       13                   OOCF3                                   5.5 &#226;&#171;&#190; 0.04        56.5 &#226;&#171;&#190; 20.5          375       5.5 &#226;&#171;&#190; 0.08         2,390     &#226;&#172;&#381;20,000 <br />       14         Cl                                       Cl       1.7 &#226;&#171;&#190; 0.04        51.3 &#226;&#171;&#190; 22.2        1,000       5.1 &#226;&#171;&#190; 0.10         4,000     &#226;&#172;&#381;20,000 <br />  <br />       15                    OCF3                OCF3               1.1 &#226;&#171;&#190; 0.12        36.9 &#226;&#171;&#190; 11.0       1,250        3.4 &#226;&#171;&#190; 0.05     &#226;&#172;&#381;10,000       &#226;&#172;&#381;20,000 <br />       16                    OOEt                                   1.6 &#226;&#171;&#190; 0.10        19.1 &#226;&#171;&#190; 1.96       1,300        3.6 &#226;&#171;&#190; 0.03     &#226;&#172;&#381;10,000       &#226;&#172;&#381;20,000 <br />       17                    OCN                                    1.6 &#226;&#171;&#190; 0.04        20.2 &#226;&#171;&#190; 2.54       1,500        3.4 &#226;&#171;&#190; 0.04     &#226;&#172;&#381;10,000       &#226;&#172;&#381;20,000 <br />       18                                Cl                         2.2 &#226;&#171;&#190; 0.10        52.0 &#226;&#171;&#190; 1.13       1,300        5.3 &#226;&#171;&#190; 0.07      ND           &#226;&#172;&#381;20,000 <br />       19                  OOCH3                OOCH3               1.2 &#226;&#171;&#190; 0.10        35.3 &#226;&#171;&#190; 9.54       2,000        3.3 &#226;&#171;&#190; 0.01     &#226;&#172;&#381;10,000       &#226;&#172;&#381;20,000 <br />       20         Cl                                                1.8 &#226;&#171;&#190; 0.07        31.8 &#226;&#171;&#190; 3.55       2,500        4.0 &#226;&#171;&#190; 0.05       6,500       &#226;&#172;&#381;20,000 <br />       21         H            H         H          H       H       0.9 &#226;&#171;&#190; 0.04         9.0 &#226;&#171;&#190; 3.34      &#226;&#172;&#381;5,000        2.1 &#226;&#171;&#190; 0.04     &#226;&#172;&#381;10,000       &#226;&#172;&#381;10,000 <br />   a <br />    Binding (&#226;&#338;&#172;Tm) measured thermal melt method, enzyme inhibition (%) measured aminoacylation assays, cell growth inhibition (EC50) was <br /> measured described Materials Methods. <br />  <br />  <br />    Pharmacokinetic efficacy experiments mice. The                          EC50, compound 1 delivered subcutaneous osmotic mini- <br /> most potent compound, compound 1, subjected single-                      pumps 25 mg/kg/day efficacy experiment. The pumps <br /> dose pharmacokinetic study mice. Following i.p. injec-                      designed deliver compound 3 days. Blood levels the <br /> tion, blood sampled time intervals analyzed                         day implantation measured 0.56 &#226;&#171;&#190; 0.12 &#226;?&#174;M <br /> liquid chromatography-mass spectrometry. The terminal half-                       (average &#226;&#171;&#190; standard error mean [SEM]; n &#226;&#171;&#189; 5 mice). In T. <br /> life determined &#226;&#172;&#402;1.0 h (data shown). In order                    brucei-infected mice, nearly complete suppression parasitemia <br /> provide sustained blood level mice vitro                   observed microscopic examination blood smears 8 <br />                                                                                   days (Fig. 4). After 8 days, parasitemia evident the <br />                                                                                   mice succumbed infection. Average survival time doubled <br />                                                                                   mice treated compound 1 compared mice <br />                                                                                   treated vehicle (Fig. 4). No apparent effects evi- <br />                                                                                   dent mice treated compound 1 the <br />                                                                                   single-dose pharmacokinetic studies efficacy studies <br />                                                                                   implanted minipumps. <br />                                                                                      Homology modeling ligand docking studies trypano- <br />                                                                                   somatid MetRS enzymes. Due absence crystal <br />                                                                                   structure T. brucei T. cruzi MetRS, homology models <br />                                                                                   created docking studies gain insight bind- <br />                                                                                   ing mode diaryl diamine inhibitors. Three-dimensional <br />                                                                                   structures MetRS 6 species available templates <br />                                                                                   homology modeling. The structures Protein Data <br />                                                                                   Bank (PDB) originate Aquifex aeolicus, Thermus thermo- <br />   FIG. 2. Correlation cell growth inhibition (EC50) binding             philus, Pyrococcus abyssi, Escherichia coli, Mycobacterium <br /> recombinant parasite MetRS. Binding determined shift                smegmatis. In addition, Medical Structural Genomics of <br /> the melting temperature (&#226;&#338;&#172;Tm) protein presence of <br /> compounds. R2 values 0.92 (P &#226;&#172;? 0.0001) T. brucei (solid dia- <br />                                                                                   Pathogenic Protozoa (MSGPP) consortium recently solved the <br /> monds) 0.79 (P &#226;&#172;? 0.0001) T. cruzi (open circles). Numeric data            structure Leishmania major enzyme, eukaryotic <br /> for series compounds presented Table 1.                            MetRS structure solved (18). Sequence comparisons of <br />  1986     SHIBATA ET AL.                                                                                     ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 3. Washout experiments T. brucei compound 2 pentamidine. The bar graphs indicate percent growth cultures <br /> compared level growth cultures grown compound. The cultures incubated compound 2 (A) pentamidine <br /> (B) concentrations 2 64 times EC50 (established 48-h experiments) various periods exposure 6 48 h. Below bar <br /> graph, cultures marked asterisk cell growth observed 10 days following washout compound. <br />  <br />  <br />  <br /> the T. cruzi T. brucei MetRS proteins Leish-              Section S1 supplemental material). Upon docking into <br /> mania MetRS homologs PDB showed overall                    homology model, compound 1 binding <br /> amino acid sequence identities ranging 24 60% (see                pockets (Fig. 5). The benzyl fragment occupies methionine <br /> Table S2 supplemental material). A detailed                   pocket meta-chlorine atoms same <br /> analysis performed compare residues constituting               position sulfur atom methionine. The methio- <br /> the methionine/ATP binding pockets respective en-                  pocket formed 11 residues, are <br /> zymes (see Table S3 supplemental material). The                    hydrophobic. The 2-amino-quinolinone fragment resides a <br /> crystal structure complex E. coli MetRS                second, adjacent pocket engages hydrogen bonds <br /> L-methionyladenylate served reference structure,              nitrogens carboxylate Asp 287 and <br /> the active-site pocket defined residues 5 &#195;&#8230;              hydrogen bond phenol hydroxyl Tyr 472. <br /> the L-methionyladenylate ligand. The pocket contained 24                  Twelve residues make second pocket. The ali- <br /> residues, identity scores analyzed              phatic linker inhibitor mainly abuts Tyr 250 (Fig. 5). <br /> MetRS enzymes ranged 58 100% (see Table S4                     Comparison sequence human cytosolic enzyme to <br /> the supplemental material). The trypanosomatid or-                  sequences trypanosomatid enzymes reveals 13 dif- <br /> ganisms identical methionine/ATP binding pockets.                     ferences 25 residues predicted contact compound 1 <br />   An homology model T. brucei T. cruzi enzymes                (see Table S5 supplemental material). The human mi- <br /> built using L. major structure, highest se-             tochondrial MetRS predicted differences contact <br /> quence identity, modified conformation                  residues compound 1 compared trypanosomatid <br /> binding site mimic arrangement seen A. aeolicus             enzymes (see Table S5 supplemental material). The <br /> 2CSX structure. That conformational arrangement required               significance similarity human mitochondrial <br /> for inhibition bacterial MetRS diaryl inhibitors (see           MetRS T. brucei MetRS discussed below. <br />  VOL. 55, 2011                                                       T. BRUCEI METHIONYL-tRNA SYNTHETASE INHIBITORS                          1987 <br />  <br />  <br />  <br />  <br />                                                                             FIG. 5. Structural model T. brucei MetRS compound 1 <br />                                                                          docked active site. The dichlorinated benzyl fragment left <br />                                                                          resides methionine substrate pocket, 2-amino-quino- <br />                                                                          linone fragment occupies adjacent pocket right pre- <br />                                                                          dicted interact Asp 287. The protein molecular surface shown <br />                                                                          transparent gray. The atomic color <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> cyan protein C, yellow <br />                                                                          ligand C, blue N, red O, purple Cl. <br />  <br />  <br />  <br />                                                                          zyme essential trypanosome growth. The RNA inter- <br />    FIG. 4. Efficacy compound 1 mouse model acute T.         ference method used knock mRNA levels the <br /> brucei infection. Mice (n &#226;&#171;&#189; 5 group) infected blood-       single MetRS gene identified T. brucei genome. The <br /> stream forms T. brucei day 0. On day 1, subcutaneous Alzet <br /> minipumps delivered compound 1 (25 mg/kg/day) vehicle 3 <br />                                                                          observed RNAi results (Fig. 1) confirmed expectation that <br /> days implanted. Parasitemia (upper panel) rapidly rose       MetRS T. brucei critical normal growth protein <br /> vehicle group, mice euthanized day 4. In        synthesis obviously dependent minimum levels charged <br /> contrast, 4 5 mice treated compound 1 dramatic           tRNAMet. Cell growth inhibited factor 106; how- <br /> suppression parasitemia gradually rebounded 7 8     ever, complete killing detected, possibly the <br /> days postinfection. They survived twice long the <br /> vehicle-treated mice (lower panel). Similar results observed   RNAi method resulted incomplete knockdown the <br /> second experiment (data shown).                                      MetRS transcript 57 77% normal levels two <br />                                                                          clones. <br />                                                                             Three compounds (compounds 2, 3, 7) synthesized <br />                           DISCUSSION                                     basis published reports MetRS inhibitors had <br />                                                                          antibacterial activity (15, 16). These compounds first <br />    tRNA synthetases established targets antimicrobial            tested T. brucei cultures observed remarkably <br /> chemotherapy, foremost example bacterial iso-              potent activity, EC50s 8, 30, 75 nM, respectively <br /> leucyl-tRNA synthetase, molecular target mupir-          (Table 1). In order determine compounds bound to <br /> ocin, antibiotic used topically treatment bacterial        T. brucei MetRS enzyme, thermal melt assay using <br /> infections, including Staphylococcus aureus infections. This             recombinant enzyme employed. Pronounced shifts melt- <br /> precedent indicated potential developing selective         ing temperatures observed (&#226;&#338;&#172;Tms, 12.9, 12.2, 10.1&#194;&#176;C, <br /> inhibitors aaRS enzymes infectious pathogens           respectively). Generally, temperature shifts exceeding 2&#194;&#176;C are <br /> as trypanosomes. The MetRS enzyme particularly attrac-             considered significant; thus, data indicated com- <br /> tive target piggyback drug development neglected trop-           pounds tightly bound recombinant MetRS enzyme. <br /> ical diseases (9) existence potent MetRS               With encouraging results, additional 18 compounds <br /> inhibitors development Pharma indications             synthesized explore structure-activity relation- <br /> (22). The similarities bacterial MetRS enzymes               ships, primarily focusing phenyl group left of <br /> and T. brucei T. cruzi led hypothesis       molecule (Table 1). The compound series resulted a <br /> existing compounds bind trypanosomal MetRS enzymes                 broad range activities T. brucei cultures the <br /> and rise antitrypanosomal activity.                              active compound, compound 1, EC50 4 nM, to <br />    Before activity compounds investigated, genetic            active compound, compound 21, EC50 of <br /> experiments conducted establish MetRS en-               &#226;&#172;&#381;5,000 nM. All compounds tested thermal <br />  1988     SHIBATA ET AL.                                                                                 ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> melt assay, correlation &#226;&#338;&#172;Tm EC50 was <br /> plotted (Fig. 2, upper line). A high correlation ob- <br /> served (R2 &#226;&#171;&#189; 0.92), meaning compounds bind the <br /> enzyme highest affinity potent inhib- <br /> iting T. brucei cell growth. This evidence com- <br /> pounds act parasites inhibition target <br /> enzyme off-target mechanism. <br />    The compounds tested major <br /> trypanosome pathogen, T. cruzi, observed to <br /> have potent growth inhibition activity (Table 1). There a <br /> high correlation compounds active T. <br /> brucei T. cruzi, expected 100% se- <br /> quence identity predicted inhibitor binding sites (see <br /> Table S4 supplemental material). The EC50s T. <br /> cruzi generally higher (by factor 10 40) those <br /> observed T. brucei. Since T. cruzi amastigotes grown <br /> intracellularly mammalian 3T3 fibroblasts, higher ob- <br />                                                                        FIG. 6. Structures MetRS inhibitors clinical trials bacterial <br /> served EC50s suggest compounds partially            infections (5, 6, 25). <br /> excluded intracellular environment host cells. <br /> Experiments binding compounds recombinant T. <br /> cruzi MetRS yielded results similar observed with <br /> the T. brucei MetRS enzyme. As before, binding affinity         leading irreversible growth arrest similar of <br /> (&#226;&#338;&#172;Tm) cell growth inhibition (EC50) highly correlated      pentamidine, drug clinical use early-stage HAT. Thus, <br /> (Fig. 2, line).                                              sustained inhibition MetRS completely kills T. brucei grown in <br />    In order establish binding recombinant T.         vitro. This trypanocidal activity necessary feature the <br /> brucei MetRS related inhibition enzymatic func-       drug used late-stage HAT, drugs completely <br /> tion, aminoacylation assay adopted T. brucei         kill parasites needed clear infection central ner- <br /> enzyme. The method assesses complete enzyme reaction         vous cerebrospinal fluid. <br /> measuring incorporation [3H]methionine tRNA            Compound 1 short plasma half-life mice (&#226;&#172;&#402;1 h). For <br /> substrate. The potent compound, compound 1, inhibited          purposes studies, able attain adequate <br /> the enzyme 99.2% 50 nM, suggesting 50% in-           plasma concentrations compound mice adminis- <br /> hibitory concentration (IC50) 50 nM,         tering continuous infusion using osmotic minipumps. <br /> consistent low EC50 4 nM observed T. brucei          Plasma concentrations &#226;&#172;&#402;0.5 &#226;?&#174;M measured mice <br /> cells. An exact IC50 (possibly subnanomolar) highly         receiving minipumps. Using pumps designed deliver <br /> potent compounds determined en-             compound 3 days, profound suppression parasitemia was <br /> zyme concentration 4 nM necessary robust           observed. However, parasites completely eradi- <br /> signal assay. The correlation enzyme inhibitory      cated, delayed recrudescence parasitemia at <br /> activity thermal melt results tight (R2 &#226;&#171;&#189; 0.821,   day 7 8 mice succumbed infection. <br /> P &#226;&#172;? 0.0001) supports use thermal method       There longer survival treated group (&#226;&#172;&#402;9 10 <br /> an enzyme assay developed optimized (17).              days) controls (4 days). These data indicate com- <br />    The effect MetRS inhibitors growth mam-         pound 1 profound effect suppress parasitemia and <br /> malian lymphocytic cell line CRL-8155 assessed. Since      prolong survival mice. It suggests higher <br /> no inhibition growth observed 20 &#226;?&#174;M        blood levels longer exposure compound 1 required <br /> compounds, indicated remarkably high selectivity index       cures. Since plasma levels &#226;&#172;&#402;100 times higher the <br /> for parasites mammalian cells. The molecular mod-          EC50 (4 nM), expected complete clearance of <br /> eling provides explanation compounds            parasites basis results washout exper- <br /> unlikely bind human cytoplasmic MetRS enzyme (see            iments described above. However, factors, pro- <br /> Table S4 supplemental material). However, differ-        tein binding access parasites sequestered tissues out- <br /> ences trypanosomatid MetRS human                blood compartment, reasons complete <br /> mitochondrial MetRS pronounced (see Table S4        cures observed. <br /> supplemental material) (discussed below). These data establish         The analysis amino acids form predicted <br /> the potential selectively inhibit trypanosomatid MetRS       binding site diaryl diamine compounds allowed comparisons <br /> and inhibit trypanosome growth avoiding toxicity           trypanosomatid human MetRS enzymes <br /> mammalian cells.                                                    (see Table S5 supplemental material). Substantial dif- <br />    In vitro washout experiments conducted determine     ferences trypanosomatid human cytosolic <br /> time required compound 2 lead irreversible death   MetRS enzymes observed; specifically, 13 25 residues <br /> parasites. When bloodstream forms T. brucei exposed      different, suggesting inhibitor un- <br /> compound 2 8 times EC50 72 h, parasites did      likely bind tightly pocket enzymes. This fits <br /> recover. Similarly, exposed compound 2 16      published data diaryl diamine REP8839 (Fig. 6) <br /> times EC50 48 h, failed recover. The conditions     essentially inactive human cytosolic enzyme, Ki <br />  VOL. 55, 2011                                                             T. BRUCEI METHIONYL-tRNA SYNTHETASE INHIBITORS                                     1989 <br />  <br />  <br /> being &#226;&#172;&#381;20,000 nM (12). Fewer differences inhibitor                       2. Buckner, F. S., C. L. M. J. Verlinde, A. C. La Flamme, W. C. Van <br />                                                                                    Voorhis. 1996. Efficient technique screening drugs activity against <br /> binding site observed trypanosomatid hu-                          Trypanosoma cruzi using parasites expressing &#226;?&#164;-galactosidase. Antimicrob. <br /> man mitochondrial MetRS enzymes, 5 25 residues                        Agents Chemother. 40:2592&#226;&#8364;&#8220;2597. <br /> being different. This fits published data REP8839                3. Charriere, F., S. Helgadottir, E. K. Horn, D. Soll, A. Schneider. 2006. <br />                                                                                    Dual targeting single tRNA(Trp) requires different tryptophanyl- <br /> inhibits human mitochondrial enzyme Ki 10 nM                         tRNA synthetases Trypanosoma brucei. Proc. Natl. Acad. Sci. U. S. A. <br /> (12). Of note, REP8839 inhibits S. aureus MetRS Ki                      103:6847&#226;&#8364;&#8220;6852. <br /> of 10 pM (12), suggesting extremely tight binding pos-                  4. Charriere, F., et al. 2009. Dual targeting tRNA ASP requires two <br />                                                                                    different aspartyl-tRNA synthetases Trypanosoma brucei. J. Biol. Chem. <br /> sible. Unfortunately, know Kis diaryl diamines                    284:16210&#226;&#8364;&#8220;16217. <br /> against trypanosomatid enzymes methods                      5. Critchley, I. A., et al. 2009. Spectrum activity mode action of <br /> accurately measuring Kis highly active compounds                            REP3123, new antibiotic treat Clostridium difficile infections. J. Anti- <br />                                                                                    microb. Chemother. 63:954&#226;&#8364;&#8220;963. <br /> against MetRS (involving adaptation ATP-PPi ex- <br />                                                                                 6. Critchley, I. A., et al. 2005. Antibacterial activity REP8839, new antibi- <br /> change assay [12]) developed trypano-                        otic topical use. Antimicrob. Agents Chemother. 49:4247&#226;&#8364;&#8220;4252. <br /> somatid enzymes. If Ki trypanosomatid MetRS                      7. Crowther, G. J., et al. 2009. Buffer optimization thermal melt assays of <br /> low (&#226;&#172;?&#226;&#172;?1 nM), possible greater activity                 Plasmodium proteins detection small-molecule ligands. J. Biomol. <br />                                                                                    Screen. 14:700&#226;&#8364;&#8220;707. <br /> diaryl diamines trypanosomatid cells mammalian cells                    8. Espanol, Y., D. Thut, A. Schneider, L. R. Pouplana. 2009. A mech- <br /> (Table 1) selective activity trypanosomatid                       anism functional segregation mitochondrial cytosolic genetic <br /> enzyme. However, likely factors                      codes. Proc. Natl. Acad. Sci. U. S. A. 106:19420&#226;&#8364;&#8220;19425. <br />                                                                                 9. Gelb, M. H., et al. 2003. Protein farnesyl N-myristoyl transferases: piggy- <br /> contributing lack toxicity diaryl diamines                        medicinal chemistry targets development antitrypanosomatid <br /> against mammalian cells live animals. One possibility                       antimalarial therapeutics. Mol. Biochem. Parasitol. 126:155&#226;&#8364;&#8220;163. <br /> that compounds penetrate mammalian cells                      10. Gentry, D. R., et al. 2003. Variable sensitivity bacterial methionyl-tRNA <br />                                                                                    synthetase inhibitors reveals subpopulations Streptococcus pneumoniae <br /> as trypanosomatid cells. The observation                      distinct methionyl-tRNA synthetase genes. Antimicrob. Agents <br /> that T. cruzi (grown inside mammalian fibroblasts) EC50s                       Chemother. 47:1784&#226;&#8364;&#8220;1789. <br /> as 44-fold higher T. brucei (grown                   11. Giaever, G., et al. 2002. Functional profiling Saccharomyces cerevisiae <br />                                                                                    genome. Nature 418:387&#226;&#8364;&#8220;391. <br /> axenic culture), despite having identical binding pockets               12. Green, L. S., et al. 2009. Inhibition methionyl-tRNA synthetase by <br /> MetRS enzyme (see Table S4 supplemental material),                          REP8839 effects resistance mutations enzyme activity. Antimicrob. <br /> suggests compounds efficiently penetrate                       Agents Chemother. 53:86&#226;&#8364;&#8220;94. <br />                                                                                13. Guiles, J., I. Critchley, X. Sun. 2008. New agents Clostridium diffi- <br /> mammalian cell membrane. It little com-                      cile-associated disease. Expert Opin. Invest. Drugs 17:1671&#226;&#8364;&#8220;1683. <br /> pound passes second barrier (the mitochondrial                     14. Hirumi, H., K. Hirumi. 1989. Continuous cultivation Trypanosoma <br /> membrane) mitochondrial MetRS significantly                        brucei blood stream forms medium containing low concentration of <br />                                                                                    serum protein feeder cell layers. J. Parasitol. 75:985&#226;&#8364;&#8220;989. <br /> inhibited. As new compounds alter pharma-                      15. Jarvest, R. L., et al. 2002. Nanomolar inhibitors Staphylococcus aureus <br /> cological properties, changes affect permeability                    methionyl tRNA synthetase potent antibacterial activity gram- <br /> properties. As result, important carefully test                positive pathogens. J. Med. Chem. 45:1959&#226;&#8364;&#8220;1962. <br />                          <br /> </body></html>