<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 18 October 2011                                  Nucleic Acids Research, 2012, Vol. 40, Database issue D687&#226;&#8364;&#8220;D694 <br />                                                                                                               doi:10.1093/nar/gkr827 <br />  <br />  <br /> PhenoM: database morphological phenotypes <br /> caused <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> essential genes in <br /> Saccharomyces cerevisiae <br /> Ke Jin1,2, Jingjing Li1,2,3, Frederick S. Vizeacoumar4, Zhijian Li1,2,3, Renqiang Min1,5, <br /> Lee Zamparo1,5, Franco J. Vizeacoumar1,2, Alessandro Datti4,6, Brenda Andrews1,2,3,*, <br /> Charles Boone1,2,3,* Zhaolei Zhang1,2,3,5,* <br /> 1 <br />  Donnelly Centre Cellular Biomolecular Research, University Toronto, 160 College Street, Toronto, <br /> ON, M5S 3E1, 2Banting Best Department Medical Research, University Toronto, 112 College Street, <br /> Toronto, ON, M5G 1L6, 3Department Molecular Genetics, University Toronto, 1 King&#226;&#8364;&#8482;s College Circle, <br /> Toronto, ON, M5S 1A8, 4Samuel Lunenfeld Research Institute, Mount Sinai Hospital, 600 University Avenue, <br /> Toronto, ON, M5G 1X5, Canada, 5Department Experimental Medicine Biochemical Sciences, University <br /> of Perugia, Via del Giochetto, 06122, Perugia, Italy 6Department Experimental Medicine Biochemical <br /> Sciences, University Perugia, Perugia, Italy <br />  <br /> Received July 22, 2011; Revised September 16, 2011; Accepted September 19, 2011 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        enrichment analysis gene sets showing similar <br /> About one-fifth genes budding yeast                               morphological alterations. The current PhenoM <br /> are essential haploid viability                               version 1.0 contains 78 194 morphological images <br /> functionally assessed using standard genetic                                    1 909 914 cells covering subcellular compart- <br /> approaches gene deletion. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> facilitate                                 ments structures 775 ts alleles spanning 491 <br /> genetic analysis essential genes,                              essential genes. PhenoM freely available http:// <br /> have assembled collections yeast strains ex-                                 phenom.ccbr.utoronto.ca/. <br /> pressing temperature-sensitive (ts) alleles essen- <br /> tial genes. To explore phenotypes caused                                 INTRODUCTION <br /> essential <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>gene mutation</span> used panel genet-                               Essential genes indispensable <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span> an <br /> ically engineered fluorescent markers explore                            organism typically involved fundamental bio- <br /> morphology cells ts strain collection using                           logical processes, cell wall membrane biogen- <br /> high-throughput microscopy. Here,                               esis, ribosome biosynthesis, DNA replication and <br /> design implementation online database,                                cytoskeletal functions (1). As thoroughly <br /> PhenoM (Phenomics yeast Mutants), storing,                               characterized model organisms, budding yeast <br /> retrieving, visualizing data mining quantita-                           Saccharomyces cerevisiae frequently used study <br /> tive single-cell measurements extracted                                    genes involved conserved biological pathways. In <br /> micrographs ts mutant cells. PhenoM allows                               S. cerevisiae,  18% genes considered es- <br />                                                                                 sential growth rich medium glucose the <br /> users rapidly search retrieve raw images and <br />                                                                                 carbon source (1,2). Essential genes evolutionarily <br /> their quantified morphological data genes                                conserved yeast humans than <br /> interest. The database provides                                    non-essential genes, indicating deeper understanding <br /> data-mining tools, including PhenoBlast module                                functions yeast essential genes extend our <br /> for phenotypic comparison mutant strains                                understanding human gene function (3). Previous <br /> and Gene Ontology module functional                                       studies characterized functions yeast essential <br />  <br />  <br />  <br />  <br /> *To correspondence addressed. Tel: +1 416 978 8562; Fax: +1 416 978 8287; Email: brenda.andrews@utoronto.ca <br /> Correspondence addressed Charles Boone. Tel: + 416 946 7260; Fax: +1 416 978 8287; Email: charlie.boone@utoronto.ca <br /> Correspondence addressed Zhaolei Zhang. Tel: +416 946 0924; Fax: +1 416 978 8287; Email: zhaolei.zhang@utoronto.ca <br />  <br /> &#195;&#376; The Author(s) 2011. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  D688 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br /> genes using variety alleles allow conditional                   To facilitate access valuable single-cell morpho- <br /> partial inactivation essential gene function. Many                      logical data including raw images, implemented a <br /> mutant strains constructed genetic analysis                  web server integrated database capability <br /> of essential genes past years, including              allow visualization analysis morphological data. <br /> harbouring tetracycline (tet)-repressible promoter alleles                 PhenoM (Phenomics Yeast Mutants) web-based <br /> (1), decreased abundance mRNA perturbation alleles                      platform contains quantitative single-cell measure- <br /> (4) temperature-sensitive (ts) alleles (5,6). Among               ments morphological images yeast cells carrying <br /> methods, ts alleles considered powerful                   ts alleles essential genes generated HCS technology <br /> genetic analysis perturb speci&#239;&#172;?c aspects es-                (10&#226;&#8364;&#8220;13) [reviewed (9)]. Figure 1 shows detailed record <br /> sential gene function (5&#226;&#8364;&#8220;8).                                               allele smc4-1, describes basic information of <br />    Temperature-sensitive alleles essential genes allow                  gene phenotypic images. We developed a <br /> cell viability permissive temperature                     user-friendly fast accurate search <br /> protein encoded ts allele resembles encoded                    engine embedded data mining tools, PhenoDev <br /> by normal wild-type allele mutant                           PhenoBlast. PhenoDev set tools calculating <br /> cells morphologically indistinguishable                     phenotypic differences mutant strains or <br /> the wild-type cells. However, restrictive (non-                     mutant wild-type strain grown the <br /> permissive) temperature, ts gene product defective                  different temperatures. These tools used <br /> and result loss-of-function. Consequently                     identify genes that, compromised, rise a <br /> cells growth morphological defects re-                 speci&#239;&#172;?c abnormal <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> interest; conversely, a <br /> strictive temperature. We recently developed                        speci&#239;&#172;?c ts allele, tools quickly identify <br /> method quantify cell morphological changes thou-                    signi&#239;&#172;?cant phenotypic changes associated <br /> sands mutants parallel coupling synthetic genetic                 temperature shift. In contrast, tool, <br /> analysis high-content screening (SGA&#226;&#8364;&#8220;HCS) (9,10).                     PhenoBlast, serves quantify morphological similar- <br /> We used SGA&#226;&#8364;&#8220;HCS study essential yeast genes                         ity multiple ts alleles. This tool best used to <br /> recently published phenotypic pro&#239;&#172;?ling 497 yeast                      discover ts alleles cause abnormal phenotypes <br /> strains carrying ts conditional alleles essential genes (6).            similar ts allele interest. Our provides <br />  <br />  <br />  <br />  <br /> Figure 1. An example allele interface based allele smc4-1. The interface includes basic information allele phenotypic <br /> images allele different subcellular compartments structures. <br />                                                                    Nucleic Acids Research, 2012, Vol. 40, Database issue   D689 <br />  <br />  <br /> tools Gene Ontology (GO) enrichment analysis &#239;&#172;?nd            website (18) customized according gene <br /> common <span id='am-318' about='Thesaurus:GO_Annotation' typeof='owl:Thing'>GO annotation</span>(s) top-ranked                       set database. <br /> mutants identi&#239;&#172;?ed PhenoDev PhenoBlast. <br />    Currently, PhenoM version 1.0 holds 78 194 morpho-              DATA MINING <br /> logical images 1 909 914 cells, covering yeast <br /> subcellular compartments structures, including actin &#239;&#172;?la-       The current PhenoM database contains data 775 ts <br /> ments, DNA damage foci, mitochondria, nucleus, plasma              mutants covering 491 yeast essential genes. For each <br /> membrane mitotic spindle. These data assembled            mutant strain, 865 morphological parameters were <br /> by studying strains carrying 775 different ts alleles spanning     catalogued database, corresponding different <br /> 491 essential genes. To knowledge, PhenoM &#239;&#172;?rst          temperatures, cellular compartments cell cycle <br /> database phenotypes caused mutation yeast essen-          stages. For example, following parameters describing <br /> tial genes contains analyses quantitative data            morphology spindle quanti&#239;&#172;?ed stored <br /> derived single-cell morphological images. PhenoM              database: area, perimeter, <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> orientation. <br /> should valuable resource yeast genomics           To aid biologists extracting biologically useful informa- <br /> communities large volume data ef&#239;&#172;?cient        tion, designed implemented data mining tools, <br /> analysis tools facilitate study essential genes        PhenoDev PhenoBlast, summarize the <br /> function. Images available downloadable              phenotypic deviations genes isogenic cell popu- <br /> format facilitate future re-analysis using                 lations quantify morphological similarities among <br /> sophisticated software developed compu-          different mutants, respectively. <br /> tational experts.                                                  Phenotypic deviation: PhenoDev <br />                                                                    PhenoDev collection analytical tools that <br /> DATA SOURCE                                                        help users evaluate phenotypic deviations a <br />                                                                    given morphological parameter response condition <br /> The PhenoM database contains morphological images              changes. It consists modules: PhenoTempDev, <br /> quantitative single-cell measurements yeast strains             PhenoMutaDev PhenoCycleDev. We provide a <br /> carrying 775 different ts alleles associated 491 essen-       detailed user manual examples using tools <br /> tial genes, covering following subcellular compart-        PhenoM website; tools user-friendly inter- <br /> ments structures: actin &#239;&#172;?laments, DNA damage foci,              faces extensively tested optimized. <br /> mitochondria, nucleus, plasma membrane mitotic                    The phenotypic temperature deviation (PhenoTemp <br /> spindle. As described previous report, MATa                 Dev) module help users identify ts strains have <br /> query strains carrying different &#239;&#172;&#8218;uorescent markers           signi&#239;&#172;?cant changes speci&#239;&#172;?c morphological parameter <br /> mated ts allele array MATa haploid ts strains           grown permissive restrictive temperature <br /> expressing different Green Fluorescent Protein (GFP) and/          (see Figure 2 example). For example, researcher <br /> or Red Fluorescent Protein (RFP) fusion proteins              interested asking ts allele affects &#226;&#8364;&#732;spindle <br /> isolated using SGA technology (6,10,14). The following             length&#226;&#8364;&#8482;, use PhenoTempDev extract two <br /> cellular markers used: plasma membrane marker               sets measurements &#226;&#8364;&#732;spindle length&#226;&#8364;&#8482; two <br /> (Psr1p-GFP), reporter DNA damage (Ddc2p-GFP),                 cell populations. A statistical test allows ranking of <br /> a nuclear marker (Mad1p-Nuclear localization Signal-               ts alleles according signi&#239;&#172;?cance difference <br /> RFP), mitotic spindle reporter (GFP-Tub1), mitochon-           tested strains. In essence, tool help <br /> drial marker (OM45p-GFP) actin reporter                     researchers identify genes implicated particu- <br /> (Sac6p-GFP). Morphological images generated                lar yeast morphological trait. Phenotypic mutation devi- <br /> using SGA-HCS protocol (9,15) quantitative                 ation (PhenoMutaDev) intended quantify the <br /> measurements particular morphological feature            phenotypic deviation wild-type strain a <br /> extracted single-cell level. For reporter,    ts mutant strain selected set morphological par- <br /> independent images captured different tempera-             ameters temperature. In contrast to <br /> tures referred &#226;&#8364;&#732;sites&#226;&#8364;&#8482; database.        PhenoTempDev, detects morphological changes <br /> The quantitative measurements cell                    caused temperature shift, PhenoMutaDev can <br /> categorized factors temperature          identify mutants signi&#239;&#172;?cantly different pheno- <br /> which yeast strain grown, cell cycle stage      type wild-type cells. <br /> cell subcellular compartment structure                Finally, phenotypic cell cycle deviation (PhenoCycleDev) <br /> which feature extracted. Since yeast cells rela-      allows users examine phenotypic changes single <br /> tively simple ellipsoidal shape, dimensions cells       mutant cell cycle. As described above, we <br /> can serve proxy determine cell cycle phase (16). In        categorized cell cycle phases based on <br /> PhenoM, categorized cells phases            daughter bud mother cell ratios. Our queries <br /> the cell cycle according ratio daughter cell area     contiguous cell cycle transitions: (i) unbudded to <br /> mother cell area: (i) unbudded cell; (ii) small budded cell;       small budded cells (G1-S phase); (ii) small budded to <br /> (iii) medium budded cell; (iv) large budded cell               medium budded cells (S-G2); (iii) medium <br /> (detailed information provided PhenoM website).              budded large budded cells (G2-M). For cell cycle <br /> Gene annotations downloaded SGD (17),                    transition, measurements morphological parameters are <br /> while GO annotations downloaded GO                   extracted compared. The program ranks the <br />  D690 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  <br />  <br /> Figure 2. An example &#226;&#8364;&#732;PhenoTempDev Detail&#226;&#8364;&#8482; interface. The &#239;&#172;?ve alleles presented &#239;&#172;?gure. <br />  <br />  <br />  <br />  <br /> mutants according changes morphological measure-                   colleagues developed software tool called PhenoBlast, in <br /> ments associated transitions. With tool,             analogy BLAST, gene pairs scored based <br /> for essential mutant, users readily determine              similarities morphological phenotypes caused by <br /> phenotypic features affected particular cell                knockdown gene expression RNAi. With tool, <br /> cycle phase transition.                                                  47 phenotypic parameters used assess morpho- <br />                                                                          logical phenotypes Caenorhabditis elegans (19). We <br />                                                                          adopted PhenoBlast comparing yeast morphology <br /> PhenoBlast <br />                                                                          data. In algorithm, directly compare the <br /> The rationale using morphological data study gene                 morphology different mutant strains. <br /> function stems recent observation genes                    Instead, mutant strain, morphological meas- <br /> that annotated similar functions tend                     urements population mutant cells &#239;&#172;?rst <br /> give rise similar morphological defects perturb-                 compared population wild-type cells. We <br /> ation (6). The huge volume morphology data generated                  used Wilcoxon rank-sum test determine whether <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> high-content screens catalogued PhenoM                   particular morphological parameter (e.g. spindle length) <br /> database offers unique useful resource search               signi&#239;&#172;?cantly increased (labelled &#226;&#8364;&#732;+1&#226;&#8364;&#8482;), unchanged <br /> alleles similar morphological phenotypes                   (labelled &#226;&#8364;&#732;0&#226;&#8364;&#8482;) signi&#239;&#172;?cantly decreased (labelled as <br /> thus similar cellular functions.                                    &#226;&#8364;&#732; 1&#226;&#8364;&#8482;) mutant strain. In way, obtain a <br />    In contrast searching databases genes                vector 865 values consisting &#226;&#8364;&#732;+1&#226;&#8364;&#8482;, &#226;&#8364;&#732;0&#226;&#8364;&#8482; &#226;&#8364;&#732; 1&#226;&#8364;&#8482; for <br /> sequence similarities, searching morphological                       mutant strain, describes overall morpho- <br /> similarities pair yeast strains computa-               logical change comparison wild-type cells. We can <br /> tionally algorithmically challenging. In                     quantify morphological similarities any <br /> earlier study phenotypic changes Caenorhabditis                    mutant strains calculating similarities between <br /> elegans RNAi interference experiments, Gunsalus                   corresponding vectors (for detailed information, <br />                                                                  Nucleic Acids Research, 2012, Vol. 40, Database issue   D691 <br />  <br />  <br /> please PhenoM server). The algorithm described           mutants number objects single cell a <br /> above implemented optimized                      particular compartment; (iv) searching mutants a <br /> PhenoBlast. Users query single mutant          certain range particular cell shape parameter. All of <br /> and mutants similar phenotypic pro&#239;&#172;?les         options uni&#239;&#172;?ed Morphology Search. <br /> real time. Figure 3 shows example PhenoBlast <br /> comparison used default settings query              GO enrichment analysis <br /> smc4-1 strain. The search identi&#239;&#172;?ed mcd1-73 strain,          Depending users&#226;&#8364;&#8482; objective, running PhenoDev or <br /> which contains ts allele MCD1 gene           PhenoBlast automatically generates list mutants <br /> hit&#226;&#8364;&#8221;the smc4-1 mcd1-73 strains shared similarity          showing phenotypic abnormality. We provide a <br /> 621 morphological parameters [122 parameters signi&#239;&#172;?-             module GO enrichment analysis test whether <br /> cantly increased, 435 unchanged 64 signi&#239;&#172;?cantly              genes associated phenotypic defects are <br /> decreased (Figure 4)].                                           enriched known functions pathways. This function- <br />                                                                  ality particularly useful gauging biological sig- <br /> Search                                                           ni&#239;&#172;?cance highly ranked genes mutation <br />                                                                  causes extreme phenotypic defects. To implement GO <br /> The search engine provides options,            analysis, compared tools GoMiner <br /> Quick Search Morphology Search. In Quick Search,             (20), GOEAST (21) GO::TermFinder (22). We <br /> users directly query mutant         chose use GO::TermFinder package (v0.86) for <br /> database specifying gene name, open reading frame         PhenoM uses modular design was <br /> (ORF) name, aliases. A partial-word-match method           easy integrate system. <br /> also incorporated Quick Search. On hand, <br /> if users want &#239;&#172;?nd mutant strains share morpho- <br /> logical similarity pre-speci&#239;&#172;?ed gene, search         DATABASE IMPLEMENTATION AND USER <br /> morphology pro&#239;&#172;?les performed. The Morphology              INTERFACE <br /> Search utility &#239;&#172;&#8218;exible allows users conduct <br />                                                                  Database implementation <br /> speci&#239;&#172;?c advanced searches including: (i) searching for <br /> mutants certain cell cycle phase; (ii) searching        We designed implemented PhenoM using Struts- <br /> mutants grown certain temperature; (iii) searching          Spring-Hibernate,   lightweight enterprise-level <br />  <br />  <br />  <br />  <br /> Figure 3. An example &#226;&#8364;&#732;Search&#226;&#8364;&#8482; interface PhenoBlast. <br />  D692 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  <br />  <br /> Figure 4. An example &#226;&#8364;&#732;PhenoBlast Detail&#226;&#8364;&#8482; interface. The interface includes GO analysis result section &#226;&#8364;&#732;Search Result&#226;&#8364;&#8482; section. Smc4-1 <br /> is used query allele 5 23&#226;&#8364;&#8220;25 alleles presented &#239;&#172;?gure. <br />  <br />  <br />  <br />  <br /> developmental framework (23,24). MySQL (v5.0.77)                        charts. All source <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> development performed in <br /> used underlying relational database store                     Eclipse v3.4.2 (http://www.eclipse.org). <br /> large volume quantitative measurements links <br /> to raw morphological images &#239;&#172;?le system. At                   User interface <br /> front end, Apache Server 2.0 deployed handle user <br />                                                                             The PhenoM server provides following primary web <br /> requests forward requests Java servlets <br />                                                                             interfaces, allow users perform focused query <br /> and Java Server Pages Apache Tomcat 6.0. To <br />                                                                             analysis tasks. <br /> provide user-friendly interface, JavaScript AJAX <br />                                                                                The &#226;&#8364;&#732;Allele interface&#226;&#8364;&#8482; presents detailed informa- <br /> (Asynchronous Javascript XML) technology were <br />                                                                             tion mutant strain. It consists parts: <br /> adopted program client-side functionality and <br /> Apache TilesTM used create reusable interface                          (i) Basic information cross links databases <br /> components. For online graphical visualization                                SGD (17) BioGRID (25); <br /> image manipulation, ImageJ (v1.44) incorporated                       (ii) Phenotypic images statistical information for <br /> assist users morphological image processing Java                         subcellular compartments struc- <br /> applets, Java open source package JFreeChart                              tures. Several functional links provided, <br /> (http://www.jfree.org/jfreechart/) used plot bar                           option download original <br />                                                                 Nucleic Acids Research, 2012, Vol. 40, Database issue          D693 <br />  <br />  <br />       micrograph TIFF format, viewing image               (P = 0.04455) (Figure 4). The results PhenoBlast <br />       ImageJ, link dataset        consistent known biology (6,29&#226;&#8364;&#8220;31), and <br />       particular image. For ease analysis, images      believe useful tool suggesting <br />       annotated names correspond             new biological gene associations yeast research <br />       mutant name, reporter imaged, temperature         community. <br />       images captured site               Download: PhenoM allows users download either <br />       number (which indicates set independent            data set mutants single <br />       images used produce particular image           selected mutant tab-delimited format. <br />       mutant) and <br /> (iii) The PhenoBlast results list, containing &#239;&#172;?fty <br />       mutants share similar morphological            AVAILABILITY AND FUTURE DIRECTION <br />       pro&#239;&#172;?les (Figures 1 4).                                PhenoM freely accessed http://phenom.ccbr. <br />                                                                 utoronto.ca/. The future development PhenoM will <br />                                                                 include following aspects: (i) PhenoM include <br />    PhenoDev: series tools measure pheno-        phenotypic information additional yeast strains <br /> typic deviations strains carrying different ts alleles       carrying mutant alleles non-essential essential <br /> under varying conditions. We PhenoTempDev               genes data available groups or <br /> an example navigate interfaces function          elsewhere; (ii) data subcellular compartments or <br /> (Figure 2). Multiple options, morphological             structures incorporated database help <br /> features certain compartment, bud size                 biologists uncover association cell morph- <br /> others provided customized. An ana-              ology gene function; (iii) data mining and <br /> lysis default parameters lead              statistical analysis tools developed, especially tools <br /> PhenoTempDev results interface (Figure 2),                integrate types data protein <br /> includes sections: (i) results GO analysis       complexes protein interactions. <br /> (ii) deviation details 50 mutants listed         Our database ultimately aims provide centralized <br /> descending order.                                               platform analysis gene function using quan- <br />    PhenoBlast: comparison tool phenotyp-          titative measurements morphological images from <br /> ic similarity accessed home          yeast mammalian cells. To accomplish goal, <br /> interface PhenoM navigation bar. To              PhenoM provide submission collect data <br /> execute PhenoBlast, query mutant                 image-based HCS experiments future. <br /> relevant morphological features speci&#239;&#172;?ed. Upon <br /> typing characters query allele box, will <br /> automatically suggest list similar mutant         ACKNOWLEDGEMENTS <br /> names. To facilitate customized search, list 865       We like thank Xiaoli Wu helping design the <br /> morphological features divided parts: (i) 386     style interfaces Dr Yufang Zheng Fudan <br /> features phenotypic mutation deviation; (ii) 193           University critical readings early version the <br /> features phenotypic temperature deviation;             article. We wish thank Dr Judice L.Y. Koh her <br /> (iii) 286 features phenotypic cell cycle deviation         helpful suggestions. <br /> (detailed information provided PhenoM <br /> server). At feature chosen conduct <br /> the analysis. The PhenoBlast results interface dis-     FUNDING <br /> played following execution successful query and <br />                                                                 A Fudan University - University Toronto Exchange <br /> contains parts: (i) GO analysis results (ii) the <br />                                                                 Scholarship (in K.J.); postdoctoral fellowship <br /> details 50 mutants similar phenotypes. <br />                                                                 Best Foundation (to F.J.V.); Canadian <br /> Figures 3 4 shows example navigating through <br />                                                                 Institutes Health Research grants (GMX-201237 to <br /> PhenoBlast (Figures 3 4). The SMC4 gene encodes a <br />                                                                 B.A. C.B.) (GMX-211012 B.A., C.B. and <br /> member ubiquitous family chromosome-associated <br />                                                                 Z.Z.). Infrastructure automated imaging the <br /> ATPases (26). The Smc4 protein subunit the <br />                                                                 Boone Andrews Lab funded the <br /> condensin complex required chromosome <br />                                                                 Canadian Foundation Innovation (LEF-21475) and <br /> condensation dynamics (27,28). To search for <br />                                                                 Ontario Research Fund. Funding open access <br /> mutants sharing similar morphological pro&#239;&#172;?les the <br />                                                                 charge: Canadian Institutes Health Research <br /> smc4-1 mutant strain, input smc4-1 query <br />                                                                 (GMX-201237) (GMX-211012). <br /> allele box Search interface PhenoBlast (Figure 3) <br /> and press button &#226;&#8364;&#732;Run PhenoBlast&#226;&#8364;&#8482; using default        Con&#239;&#172;&#8218;ict statement. None declared. <br /> parameters. The automatically leads Detail <br /> interface PhenoBlast. In search result section, the <br /> &#239;&#172;?rst, &#239;&#172;?fth fourth ranked mutants, mcd1-73,           REFERENCES <br /> smc1-2 smc3-1, respectively, carry mutant alleles <br />                                                                  1. Mnaimneh,S., Davierwala,A.P., Haynes,J., Moffat,J., Peng,W.T., <br /> in genes encoding condensin subunits list                Zhang,W., Yang,X., Pootoolal,J., Chua,G., Lopez,A. et al. (2004) <br /> mutants overall enriched DNA metabolic                   Exploration essential gene functions titratable promoter <br /> process (P = 0.01218) sister chromatid cohesion                 alleles. Cell, 118, 31&#226;&#8364;&#8220;44. <br />  D694 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  2. Giaever,G., Chu,A.M., Ni,L., Connelly,C., Riles,L., Veronneau,S.,       17. Hong,E.L., Balakrishnan,R., Dong,Q., Christie,K.R., Park,J., <br />     Dow,S., Lucau-Danila,A., Anderson,K., Andre,B. et al. (2002)                Binkley,G., Costanzo,M.C., Dwight,S.S., Engel,S.R., Fisk,D.G. <br />     Functional pro&#239;&#172;?ling Saccharomyces cerevisiae genome.                 et al. (2008) Gene Ontology annotations SGD: new data <br />     Nature, 418, 387&#226;&#8364;&#8220;391.                                                       sources annotation methods. Nucleic Acids Res., 36, <br />  3. Hughes,T.R. (2002) Yeast drug discovery. Funct. Integr.                 D577&#226;&#8364;&#8220;D581. <br />     Genomics, 2, 199&#226;&#8364;&#8220;211.                                                   18. Ashburner,M., Ball,C.A., Blake,J.A., Botstein,D., Butler,H., <br />  4. Schuldiner,M., Collins,S.R., Thompson,N.J., Denic,V.,                       Cherry,J.M., Davis,A.P., Dolinski,K., Dwight,S.S., Eppig,J.T. <br />     Bhamidipati,A., Punna,T., Ihmels,J., Andrews,B., Boone,C.,                  et al.; The Gene Ontology Consortium (2000) Gene ontology: <br />     Greenblatt,J.F. et al. (2005) Exploration function               tool uni&#239;&#172;?cation biology. Nat. Genet., 25, 25&#226;&#8364;&#8220;29. <br />     organization yeast early secretory pathway            19. Gunsalus,K.C., Yueh,W.C., MacMenamin,P. Piano,F. (2004) <br />     epistatic miniarray pro&#239;&#172;?le. Cell, 123, 507&#226;&#8364;&#8220;519.                             RNAiDB PhenoBlast: web tools genome-wide phenotypic <br />  5. Ben-Aroya,S., Coombes,C., Kwok,T., O&#226;&#8364;&#8482;Donnell,K.A., Boeke,J.D.               mapping projects. Nucleic Acids Res., 32, D406&#226;&#8364;&#8220;D410. <br />     Hieter,P. (2008) Toward comprehensive                             20. Zeeberg,B.R., Feng,W., Wang,G., Wang,M.D., Fojo,A.T., <br />     temperature-sensitive mutant repository essential genes           Sunshine,M., Narasimhan,S., Kane,D.W., Reinhold,W.C., <br />     Saccharomyces cerevisiae. Mol. Cell, 30, 248&#226;&#8364;&#8220;258.                           Lababidi,S. et al. (2003) GoMiner: resource biological <br />  6. Li,Z., Vizeacoumar,F.J., Bahr,S., Li,J., Warringer,J.,                      interpretation genomic proteomic data. Genome Biol., 4, <br />     Vizeacoumar,F.S., Min,R., Vandersluis,B., Bellay,J., Devit,M.               R28. <br />     et al. (2011) Systematic exploration essential yeast gene            21. Zheng,Q. Wang,X.J. (2008) GOEAST: web-based software <br />     function temperature-sensitive mutants. Nat. Biotechnol., 29,          toolkit Gene Ontology enrichment analysis. Nucleic Acids <br />     361&#226;&#8364;&#8220;367.                                                                    Res., 36, W358&#226;&#8364;&#8220;W363. <br />  7. Hartwell,L.H., Culotti,J. Reid,B. (1970) Genetic control         22. Boyle,E.I., Weng,S., Gollub,J., Jin,H., Botstein,D., Cherry,J.M. <br />     cell-division cycle yeast. I. Detection mutants. Proc. Natl       Sherlock,G. (2004) GO::TermFinder&#226;&#8364;&#8220;open source software for <br />                                                                                 accessing Gene Ontology information &#239;&#172;?nding signi&#239;&#172;?cantly <br />     Acad. Sci. USA, 66, 352&#226;&#8364;&#8220;359. <br />                                                                                 enriched Gene Ontology terms associated list genes. <br />  8. Simchen,G. (1978) Cell cycle mutants. Ann. Rev. Genet., 12, <br />                                                                                 Bioinformatics, 20, 3710&#226;&#8364;&#8220;3715. <br />     161&#226;&#8364;&#8220;191. <br />                                                                             23. Sun,L.V., Jin,K., Liu,Y., Yang,W., Xie,X., Ye,L., Wang,L., <br />  9. Vizeacoumar,F.J., Chong,Y., Boone,C. Andrews,B.J. (2009) <br />                                                                                 Zhu,L., Ding,S., Su,Y. et al. (2008) PBmice: integrated <br />     A picture worth thousand words: genomics phenomics in <br />                                                                                 database piggyBac (PB) insertional mutations their <br />     yeast Saccharomyces cerevisiae. FEBS Lett., 583, 1656&#226;&#8364;&#8220;1661.             characterizations mice. Nucleic Acids Res., 36, D729&#226;&#8364;&#8220;D734. <br /> 10. Vizeacoumar,F.J., van Dyk,N.S., Vizeacoumar,F., Cheung,V.,              24. Yang,W., Jin,K., Xie,X., Li,D., Yang,J., Wang,L., Gu,N., <br />     Li,J., Sydorskyy,Y., Case,N., Li,Z., Datti,A., Nislow,C. et al.             Zhong,Y. Sun,L.V. (2009) Development database system <br />     (2010) Integrating high-throughput genetic interaction mapping              mapping insertional mutations mouse genome with <br />     high-content screening explore yeast spindle                         large-scale experimental data. BMC Genomics, 10(Suppl. 3), S7. <br />     morphogenesis. J. Cell. Biol., 188, 69&#226;&#8364;&#8220;81.                              25. Stark,C., Breitkreutz,B.J., Chatr-Aryamontri,A., Boucher,L., <br /> 11. Bakal,C., Aach,J., Church,G. Perrimon,N. (2007)                         Oughtred,R., Livstone,M.S., Nixon,J., Van Auken,K., Wang,X., <br />     Quantitative morphological signatures de&#239;&#172;?ne local signaling                 Shi,X. et al. <br /> </body></html>