<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Volume 15 Number 19 1987                                              Nucleic Acids Research <br />  <br /> The n d sequence PRII gene relaed DNA prime Sacchwomyces cerevsae <br />  <br />  <br /> Paolo Plevani*, Stefania Francesconi Giovanna Lucchini <br />  <br />  <br /> Dipartimnto di Genetica e Biologia dei Microrganismi, Universita di Milano, Via Celoria 26, 20133 <br /> Milano, Italy <br />  <br /> Received July 30, 1987; Revised Accepted September 11, 1987                Accession no. Y00458 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> The PRIl gene Saccharomyces cerevisiae encodes p48 polypeptide of <br /> DNA primase. We determined nucleotide sequence 1,965 bp DNA <br /> fragment containing PRIl locus. The entire coding sequence gene <br /> lies open reading frame, 409 amino acids the <br /> single polypeptide protein translation assumed start first <br /> ATG frame. The 5&apos; 3&apos; end-points PRIl mRNA determined <br /> by Sl mapping primer extension analysis. The primary structure the <br /> codon usage PRIl suggest essential gene poorly expressed in <br /> yeast cells. <br />  <br />  <br /> INTRODUCTION <br /> Purified DNA polymerases prokaryotic eukaryotic organisms are <br /> unable initiate DNA synthesis novo (1). Several evidences indicate that <br /> the nascent DNA chains (Okazaki fragments), lagging strand of <br /> the replication fork, synthesized elongation RNA primers (1). A <br /> specialized enzyme, called DNA primase, devoted synthesis such <br /> oligoribonucleotides, involved initiation DNA replication (1). <br /> Recently, DNA primase activity associated re- <br /> plicative DNA polymerase purified variety eukaryotic organisms (2). <br /> The polypeptide structure catalytic activity DNA polymerase- <br /> DNA primase complexes appear highly conserved, suggesting that <br /> they play critical role DNA replication. Because mechanism of <br /> priming leading strand origin replication similar <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> the initiation synthesis lagging strand, proposed that <br /> eukaryotic DNA polymerase-primase complexes involved forma- <br />  <br />  <br />  <br />  ) I R L Press Limited, Oxford, England.                                                      7975 <br />  Nucleic Acids Research <br />  <br /> tion initiation complex, simply function the <br /> elongation reaction (for review, ref.2). <br /> We purified yeast DNA polymerase-primase complex rapid immuno- <br /> affinity chromatography using mouse monoclonal antibody (3). A catalytic <br /> subunit exhibiting hetherogaeros   composition antigenically related poly- <br /> peptides, ranging 140 kDa 180 kDa, identified DNA <br /> polymerase core enzyme (3,4). The corresponding gene cloned by <br /> using specific antibodies, Southern analysis gene disruption experi- <br /> ments showed yeast DNA polymerase I encoded single, essential <br /> gene (5-7). The mechanistic properties yeast DNA primase exten- <br /> sively characterized, enzymatic activity easily disso- <br /> ciated DNA polymerase and, eventually, bifunctional complex be <br /> reconstituted vitro (3,8). DNA primase, yeast other <br /> eukaryotic organisms, correlates polypeptides approximately <br /> 48 kDa 58 kDa (2). Even known yeast p48 <br /> polypeptide itself sufficient DNA primase activity, antiserum <br /> raised isolated polypeptide inhibits enzymatic activity, <br /> thus providing direct evidence protein related DNA <br /> primase (4). We isolated gene encoding p48 polypeptide using this <br /> specific antiserum demonstrated unique yeast haploid <br /> genome function essential (4). We called gene PRII, <br /> because cloned gene related eukaryotic DNA primase. <br /> In order facilitate studies mechanistic functional pro- <br /> perties DNA primase, undertook determination nucleotide se- <br /> quence PRIl gene. The knowledge sequence essential to <br /> overproduce p48 polypeptide yeast E.coli, construct <br /> thermosensitive mutants site directed mutagenesis analyze regu- <br /> lation expression essential enzyme involved DNA replication. <br /> Moreover, knowledge sequence yeast DNA primase helpful <br /> in identifying genes performing similar functions eukaryotic cells. <br /> The analysis sequence 1,965 bp segment containing entire PRIl <br /> gene revealed presence open reading frame 1,227 nucleotides <br /> which encodes protein 47,623 daltons. The mapping 5&apos; 3&apos; <br />  <br />  <br /> 7976 <br />                                                                               Nucleic Acids Research <br />  <br /> end-points PRIl mRNA provides necessary information study mecha- <br /> nisms controlling PRIl transcription. <br />  <br />  <br /> MATERIALS AND METHODS <br /> Subcloning construction deletions spanning PRIl gene. <br /> The cloning PRIl           gene   Saccharomyces cerevisiae reported <br /> previously (4). The pGEM series plasmids (Promega Biotec.,Cambridge,MA) <br /> containing      polylinker cloning segment flanked T7 SP6 promoters                     was <br />  <br /> used sequencing work reported here. The PRIl coding sequence                          con- <br />  <br /> tained        1.92 kb NruI-SstI fragment (4),                ligated the <br /> SmaI SstI sites plasmid pGEM-2, create derivative plasmid pPP1. <br /> Exonuclease III           used construct      set unidirectional deletions            ac- <br />  <br /> cording procedure described Henikoff (9). Because exonuclease III <br /> fails initiate digestion DNA ends four-<span id='am-4' about='xsp:base' typeof='owl:Thing'>base</span> 3&apos;-protrusions and <br /> digestion    progresses          uniform rate,      ordered set deletion clones in <br /> the 5&apos; 3&apos; direction PRIl             gene      cutting pPPl with <br /> BamHI PstI exonuclease treatment. In fact,                           enzymes <br />  <br /> cut plasmid              polylinker region, BamHI              pro- <br />  <br /> ximal PRII insert, PstI leaves                    3&apos;-four-base protrusion that <br /> protects remainder vector nuclease attack. Aliquots                           re- <br />  <br /> moved exonuclease III digestion mixture uniform intervals, trea- <br /> ted Sl nuclease, Klenow DNA polymerase, T4 DNA ligase used to <br /> transform E.coli competent cells (9). Plasmid DNA                     prepared several <br /> transformants time point extent deletions                           determi- <br /> ned restriction analysis. Sequencing PstI-NruI region shown in <br /> Figure 1 sequencing opposite DNA strand                      achieved subclo- <br /> ning appropriate restriction fragments pGEM-2 vector using <br /> either T7            SP6 promoter primers. In     case,       sequence   3&apos; to <br /> 5&apos; direction           portion PRIl   gene        obtained using        primer in <br /> the sequencing reaction              20-mer synthetic oligonucleotide complementary to <br /> the sequence opposite strand previously determined. <br /> DNA sequencing. <br /> The subclones            sequenced dideoxy-chain termination method (10), <br />  <br />  <br />                                                                                                     7977 <br />  Nucleic Acids Research <br />  <br /> using denatured plasmid templates (11). Preparation plasmid DNA alka- <br /> line denaturation performed essentially described (11). The sequen- <br /> cing reactions carryed Promega GemSeq       K/RT accor- <br /> ding instructions manufacturer, using[     5S]-dATP   labeled <br /> nucleotide. The Klenow fragment E.coli DNA polymerase generally used, <br /> while AMV reverse transcriptase utilized resolve ambiguities even- <br /> tually sequence ladder obtained enzyme. <br /> Preparation RNA transcript mapping 5&apos; 3&apos; end-points. <br /> Poly(A)    yeast RNA prepared strain TD28 (MATa, ura3.52, inosl, canl) <br /> as previously described (4). The Sl nuclease protection experiments car- <br /> ryed essentially described Natsoulis et al. (12). Our probes to <br /> map 5&apos; 3&apos; end-points PRIl transcript were, respectively, the <br /> NruI-ScaI NdeI-SstI fragments plasmid pPP1 labeled T4 polynucleo- <br /> tide kinase ScaI site Klenow enzyme NdeI site. The <br /> fragments hybridized 20 40 pg poly(A)      RNA quantities <br /> of S1 required complete digestion non-protected fragments de- <br /> termined empirically. Protected DNA electrophoresed 6% polyacrylami- <br /> de denaturing gel parallel Maxam Gilbert sequence (13) the <br /> fragments used S1 mapping experiments. <br /> The primer extension experiment synthetic primer 5&apos; GTCTTTAC <br /> TGAATTGGTCAT 3&apos; complementary PRIl mRNA (+1 +20) synthesized with <br /> an Applyed Biosystems DNA synthesizer. The primer hybridized 20 and <br /> 40 pg   poly(A)+   RNA extended AMV reverse transcriptase the <br /> presence of[    _P]-TTP labeled nucleotide essentially described by <br /> Giaver et al. (14). The products reaction analyzed 8% se- <br /> quencing gel Sanger sequencing reaction plasmid pPP1 per- <br /> formed using synthetic oligonucleotide primer. <br /> Computer assisted analysis DNA sequence. <br /> DNA sequence data deduce amino acids composition PRIl protein, <br /> the codon usage hydrophilicity profile analyzed IBM <br /> microcomputer using MicroGenie program (Beckman). <br />  <br />  <br />  <br />  <br /> 7978 <br />                                                                      Nucleic Acids Research <br />  <br />                               I              PRI I ORF                    &gt; <br />  <br />                          4 <br />                               -* <br />      Pstl             Nrul    ScalEcoRI NindAll Acci         BglH       Ndel    Ball     Sstl <br />  <br />                                             - *          -          *          -- <br />  <br />              0.5 kb <br />  <br />  <br /> Figure 1. Restriction map DNA sequencing strategy PRIl region. <br /> The horizontal lines arrowheads indicate extent direction of <br /> the sequenced fragments. The extent PRIl open reading frame indica- <br /> ted figure. <br />  <br /> RESULTS AND DISCUSSION <br /> Sequencing strategy. <br /> The cleavage sites number 6 bp restriction enzymes previously <br /> cloned 2,450 bp region containing entire yeast PRIl gene shown in <br /> Figure 1. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> previous experiments known coding sequence was <br /> located right NruI site terminated SstI site. <br /> In fact, 1.45 kb mRNA identified PRIl mRNA detected on <br /> Northern blots yeast poly(A)        RNA using DNA probes spanning the <br /> NruI-BalI region. The upstream PstI-NruI fragment SstI-NcoI fragment, <br /> spanning 1 kb region downstream SstI site, identified on <br /> Northern blots mRNA species 0.92 kb 2.25 kb respectively (data not <br /> shown), correspond transcripts unidentified yeast genes which <br /> lie PRI1 locus. The direction transcription the <br /> PRIl gene known left right demonstrated vitro <br /> transcription translation pGEM-2 derivative plasmid                    the <br /> NruI-SstI fragment inserted control phage T7                   promoter(4). <br /> To generate plasmids direct sequencing dideoxy-chain termination <br /> method      (10), plasmid pPP1 containing NruI-SstI fragment inserted the <br /> polylinker region pGEM-2, cut PstI BamHI set targeted <br /> 5,     deletions created unidirectional digestion exonuclease III as <br /> described Materials Methods. A number overlapping deletions se- <br />  <br />  <br />                                                                                             7979 <br />  Nucleic Acids Research <br />  <br /> quenced using denatured plasmid template phage T7 promo- <br /> ter primer directly   obtain sequence mRNA (non-coding) strand <br /> of PRIl gene. <br /> The opposite DNA strand sequenced subcloning suitable restriction <br /> fragments pGEM plasmids using T7 SP6 phage promoter primers or <br /> synthetic oligonucleotides complementary proper regions previously <br /> determined non-coding strand PRIl gene (15). The large majority the <br /> clones yielded high quality sequencing ladders, purity plasmid <br /> DNA crucial DS sequencing method (11). <br /> Nucleotide sequence yeast PRIl gene. <br /> Figure 2 depicts sequence 1,965 bp PRIl region yeast <br /> Saccharomyces cerevisiae, previously located chromosome IX (4). <br /> The indicated restriction sites deduced nucleotide sequence in <br /> complete agreement independently restriction map shown in <br /> Figure 1, limits agarose gel resolution. Furthermore, enti- <br /> re coding sequence yeast PRIl gene contained NruI-SstI re- <br /> gion restriction map, deduced previous data (4). <br /> An open reading frame (ORF) encoding 409 amino acids ending TGA <br /> stop signal identified sequence. The 1,227 bp ORF encodes a <br /> polypeptide calculated molecular weight 47,623 daltons, provided <br /> that ATG downstream transcription start recognized by <br /> the translation machinery. PRIl encodes polypeptides of <br /> yeast DNA primase polypeptide mass enzyme subunit been <br /> estimated 48,000 daltons electrophoretic mobility in <br /> polyacrylamide gel presence sodium dodecyl sulfate (3). Therefore, <br /> it appears molecular weight PRI1 protein deduced DNA se- <br /> quence analysis p48 polypeptide yeast DNA primase in <br /> complete agreement. <br /> There second frame ATG codon nucleotides +45 +47 act <br /> as initiator codon generate protein slightly reduced <br /> molecular weight 46,210 daltons reasonable accor- <br /> dance peptide mass DNA primase determined electrophoretic ana- <br /> lysis. The gene encoding yeast hystidine-tRNA synthetase (HTS1) two <br />  <br />  <br /> 7980 <br />                                                                                                                      Nucleic Acids Research <br />            -30                                           llru <br />            IItAGCllCC GIJACTGGIT CAGCCACAGG IGIGAIGAGI A!CGCGAICA AICCICGCCI ItiICCIIGC ICCIICAACC AIIACIATAI CCIACGICIL <br />            AA^GAIGCGI GAGIIICGAA ACAGTIGCIC GGICAGIGCC CIIIGCCACI IIIACCIAII CIllIClIlC AGIIGAAAAG IJICAIAACG CCAAAAAGAI <br />            -100                                                    _                   _ <br />            AAAAAAIIAA TCAAAAAAIC ITIGAAGCIG AAGAGAAAAL CGIGCCAIAA TACIALCAAA AAIIACIIII AGCAACICAA AAAACICIII IAIAAACAIC <br />    I                                                                               Scal <br />    A1G ACC AA! ICA G61 AAG ACT AA G66G CCA AGC ICC 1CA <br />    Net Ihr Asn Ser Va Lys Thr Asn Gly Pro Ser Ser Ser JAC <br />                                                           AVG G6G TAC TAC TAT AAG ICC CIC 1Al CCA IIC AAA CAC All IIC AAC <br />                                                        sp Net G u Tyr Tyr lyr lys Ser leu Tyr Pro Phe lys His <br />                                                                                                            I. r Phe sn <br />    91                                                           [coRI <br />    T6G CIG AA1 CST 1CG CCC AAA CCI lCl AGG GAl AVG All AAC AA GSA ItC GC6 AIG 6CG IIC AGG ICA GG6 CC1 IAC AAA ACG IA AAI <br />    Trp Leu Asn His Ser Pro Lys Pro Ser Arg Asp Net lIe As. Arg Glu Phe Ala let Al Phe Arg Ser Cly Ala ly Iys Arq Tyr <br />                                                                                                                     yAn <br />    181 <br />    IC! 111 AAC IC! GIG CAA GA! TIC AAA 6CA A                 AlA <br />                                                   GAA AAA GCC AAC CCA GAC AGA 111 GA        G66 GCC A A <br />    Ser Phe As. Ser Val G6n Asp Phe Lys Ala G1 lIe 6lu Lys Ala Asn Pro sp Arg Phe G lu llICe Gly Ala Ile 1Ayr Asn lys Pro Pro <br />                                                                                                              AAC AAA CCG CCA <br />  <br />    271                                           Hind IIl <br />    AGA GAu GT GAI AC! CIA IIA AAA NC UA 1G AAA 661 IIA GAG AACGCCG 16 GIG                 GAl AIG GAl GAl IAC IAl GCC lIC <br />            Arg Asp Thr Leu lem Lys Ser 61u lea Lys la Lev G61 Lys C u leu Val Phe Asp All <br />                                                                           G GI  1 A <br />    Arg Giu                                                                             I    p flet Asp Asp yr sp Ala Phe <br />    361 <br />    AGA ACG 1GT IGC ICC CGA 6CC C A 611 IGC ICT AAG IGC 1GG AAA Ti1 A                        II     C CGCGC AG AA I IACG AC ACC        CI A GA G <br />    Arg Thr Cys CysSer Gly AlaGlnnaI Cys Ser Lys Cys Trp Lys Phe                            e Ser    lee la Net lys lIe Ihr Asn Ihr la leu Aq Clu <br />    451                                                                Accl <br />    GA 1T IT              TAT AAG JAC ITC AIC <br />                          1^                    1GG GTA   11 ICA    G66 AGAC6661G     G ITC IC16 GG GIA AGC GAC AAA CGG GCA CGG Gi               ACC <br />                                                                                                                                             II Ihr <br />     sp Phe          y    Tyr Lys sp I e     Irp Va    Phe Ser   Gly Arg Arg G   y A His Cys Trp VaI Ser Asp lys Arq Ala Arq          teu <br />    541 <br />    GA!     Gl CA CGA AGA AAC G61 TIA GAl IA! 61C AAC Gl All AGA AC AGA AA1 ACI GAC AAG CGA 11 <br />            Va Gn Arg Arg Asn Va leu sp yr Va sn Val e Arg Asprg Asn Thr sp Lys Arg teu <br />                                                                                            IG                            l II   AAG AGG CCI IA! CAC <br />                                                                                                                                         Pro lyr His <br />     sp                                                                                                                     teu lys Arg <br />    631 <br />    CCI CAT Cll GfT CGI IC! 11G GfG <br />    Pro His lev aArg Ser Lev G6 G.n leu Lys Pro Phe Phe Va AVSer <br />                                    CA I16 AAA CCI IIC TI1 G                            Alf IG      CIC G#A JAC GCA AI CCI IGG       GAl GAC CfA C#1 <br />                                                                                                    teu himu Asp GTu Asn Pro lrp GfA <br />                                                              GC <br />                                                                                         e Net                                   GTu Asp Csp 61n His <br />    771 <br />    l AII Ce <br />            CtAa ATCIlA11 G            CIACl <br />                                          IVA      IA! <br />                                                   GTAl       AAA CAIA GA16 l CAl ICA IIA AAG AAG IA! IG CIC GAC AAI CCA AGG AGG ICA AGC <br />                 TI r leu Leu            Leu  lyr sp     Lys G n Leu I e Asp Ser leu lys Lys lyr lrp leu Asp Asn Pro Arg Arg Ser Ser <br />    oil <br />    A&quot; GAG AAG   166  AAT GA!         AlA GAl CAG AlA 661     ACA ICG CIC IIC AAA GGC CCC AAG 61. <br />                                                                                                AGAC ICT CAC                         1GI <br />                                                                                                             lIe All <br />                                                                                                                 lIe Lys lea Arg G#A <br />                                                                                                                     AAG llA CGt <br />    Lys 61u Lys lrp Asc sp            I le Asp G6n I e Ala    Ihr Ser teu Phe Lys Gly Pro Lys C! Asp Ser His AIA                 Clu Cys <br />    901           1111I <br />    AAG GGAlT CIC lIA 1G AIG  AC! CII IA! CCG AAG CIG GA! GIG G I ACA AAG CfA ACA All CAl 11G ITA AAG 6CC CCI TIIT GI <br />                          Net Thr Lee Tyr Pro Lys leu Asp Val G61 G!G <br />    Lys 61U Asp Lee alteu                                           Vat Ihr Lys G61 Ihr lIe His leu leu Lys Ala Pro Phe Cys <br />    991 <br />    AllIe   l CA I GCTIJ AP GCG AAT GTC IG6 G61 CCI Al! GAl G6A ICC IIT C6A CCT GAA AAA GCA CCI AAG CIA All GAl CIT CAA ACA CGA <br />            His  Pre Ihi Gly Asn Val Cys Val Pro lIe sp G u Ser Phe Ala Pro G6u Lys la Pro lys leu lIe Asp teu G6n Ihr Clu <br />    10111                                                                                                      del <br />        G16 HG AAA AAI AA! GA! GIl ICA lA ACA 66! TIA CAA CCI 11 IIC <br />                                                                   CAA CfG lC CAGA IA! GIG AGT IC! CTI IIC AAA AAI CAA <br />         let u Lys sn Asn sp Va Serieu Thr       leu G Pro Phe       Asn G n Phe Gln la Tyr Va Ser Ser leu leu lys An Glu <br />                                                                                    e <br />  <br />  <br />  <br />  <br />    1111 <br />    CICG 66 ICA GIG AAA AGA GAA CGI GIA GA! GA! GAl GAA CCC 661 IC! hA CGA TIC IGA IGGIGAAAIC TTTIIGTAAC AAIAAAAAG(. <br />    tea Gly Ser al Lys Arg G Arg G1u Asp sp Asp Glu Pro Ala Ser leu Asp Phe *ue <br />  <br />            ltrZIAIAIAA      AITIAITAGA CIACIAIAGC IITIGGIICA AIITIGCIIT ACCIGTAAAC GCATTCCCT GGTAAGTCAI ICGAAGAGCC ACIICCACAG! <br />                  1361 <br />                     *--                                             Ball <br />                  6IIGGCC IIGACITCCA      IlIIGCGCAA AITICAATGA ACTCAICGCC AGCCICCCGA AACCAITCAA GAACCTGGTA CTCAGCTTTA IACGIAIAAI <br />             tLGIAIGA IGAACGAAIT GACAACCGCC TICIGGAAA AIGGCGCCCCI CCIAIAAAAA TAAATAGGG GACAGTTATC TGTTCGCGCG IM.LIIfAI <br />             M AAIIC AAAAGGTAGG AITIAITACA IAAGFTAAAC ICAGCAAACI CAAAICITCC GGAAIFTTGC AIGGGGAATI GGICICAIIV IA(CAICOAA <br />            1I9 <br />            AGCIC <br />  <br />  <br /> Figure 2. Nucleotide sequence deduced amino acids sequence PRI1. The <br /> nucleotide sequence numbered ATG initiation codon PRIl <br /> protein (+1). The amino acids sequence translated DNA se- <br /> quence. The major 51 end-points mature PRIl transcripts desi- <br /> gned horizontal arrows position -47 -61. The closed circles <br /> at position +1373 (t 1) indicate 31 end-point PRIl mRNA. The 6 bp <br /> restriction endonuclease recognition sites indicated sequence. <br />  <br /> in frame translation start sites located 60 bp apart, shown <br /> that set HTS1 transcripts initiates upstream ATG codons, while <br /> the set initiates ATGs, messages encode t-ihe <br />  <br />  <br />                                                                                                                                                   7981 <br />  Nucleic Acids Research <br /> mitochondrial cytoplasmic hystidine-tRNA synthetase, respectively (12). <br /> Because transcription mapping PRIl mRNA (see below) places 5&apos; end- <br /> points transcripts upstream ATG codon, necessa- <br /> ry destroy ATGs vitro mutagenesis and/or construct PRIl-lacZ <br /> fusions include the frame ATGs assess their <br /> role initiation translation. Recent examination ribosome <br /> scanning hypothesis (18) indicates that, AUG lies unfa- <br /> vorable nucleotide context reading frame second <br /> AUG codon, ribosomes initiate sites, producing long short <br /> versions encoded polypeptide, case PRIl <br /> gene. The optimal context initiator AUG codon higher eukaryotes is <br /> CCACCAUGG (16), consensus sequence AUG context highly <br /> expressed yeast genes AAAAAAAUGUC (17). Neither sequences is <br /> found possible translation initiation codons PRI1. A 72 bp long <br /> ORF present upstream 5&apos; transcription start position -201 <br /> to -126. However, function, any, short ORF is, present, <br /> unknown. <br /> Codon usage PRIl predicted structure gene product. <br /> As shown Table 1, translation 1,227 bp ORF PRIl gene <br /> indicates PRIl protein 40.3% non polar, 27.9% polar, 17.1% posi- <br /> tively charged 14.7% negatively charged amino acids. This represents a <br /> random distribution types amino acids. Table 1 shows the <br /> frequency codon usage. It shown correlation <br /> between extent codon bias gene level mRNA and <br /> protein expression (19,20). An examination codon usage PRIl <br /> gene reveals tha little codon bias. In fact, the <br /> CGC arginine codon, 60 possible codons used PRI1. This <br /> <span id='am-5' about='xsp:pattern' typeof='owl:Thing'>pattern</span> codon usage common poorly expressed yeast genes is <br /> in agreement observation PRIl mRNA low abundance. The co- <br /> don usage highly expressed ADH1 gene shown Table 1 faci- <br /> litate direct comparison (27). <br />  <br />  <br />  <br />  <br /> 7982 <br />                                                                                                    Nucleic Acids Research <br />  <br />                                                               Table 1. <br /> A. Codon usae PRIA protein. <br />                         Po11                           ADNI                                poll                          ADN I <br />  <br />                                                                            M IUsage Ue* <br />                                                                               Cede&quot; Ud  ACede.                       I    Cedon <br />                                                                                                                          Usage <br />                             L.         zu0::        Usage <br />  <br /> Phe(F      )     TII          10(&apos;2.4)     46           0           Tyr(Y)     TAT            11(?.7)         73            0 <br /> Pbe             T.TC          11(2.7)      5?         100           Tyr        IAC             4( 1.0)        27          100 <br /> ieu(t)           TTA          11(2.7)      28                     Term       ]AA <br /> teo             11T           13(3.?)      33          79           Tera       TAG <br />  <br /> teo             CIl            4(1.0)       10          O           NHis(s)    CAl                6(1.5)       67           9 <br /> Lev             CiC            4(1.0)       10          0           Nis        CAC                3(0.7)       33          91 <br /> teo             CTA            3(0.7)                  13           GIn(0)     CAA            13(3.2)          87         100 <br /> ter             CTG            4(1.0)       10          0           6 n        CAG                2(0.5)       13           0 <br />  <br /> Ile(1)           ATT          12(2.9)      53          42           Asn(b)     AAT            13(3.2)          62           0 <br /> Ile              ATC           4(1.0)       16         57           Asn        AAC             8(2.0)          38         100 <br /> 11e              ATA           6(1.S)       27          0           Lys(K)     AAA            19(4.6)          53          17 <br />  *et(I)          AT6           9(2.2)      100        100            Lys       AAG            17(4.2)          47          83 <br />  <br /> Va1(V)           CTT           4(1.0)       21         53            Asp(D)    CAT            25(6.1)          71          13 <br /> Va1              CTC           3(0.7)       16         47            Asp       GAC            10(2.4)          29          87 <br /> Val              CTA           5(1.2)       26          0            61u(E)    GAA            18(4.4)          72         100 <br /> Va1              GTG           7(1.7)       37          0            61u       GAG             7(1.7)          28           0 <br />  <br /> Set(S)           TCT           9(2.2)       31         67            Cys(C)    161                5(1.2)       63         100 <br /> Ser              TCC           3(0.7)       10         33            Cys       TIC                3(0.7)       37           0 <br /> Ser               TCA          8(2.0)       26          0            Term      TCA <br /> Ser              TCC           3(0.7)       10          0            Trp(U)    TGG                7(1.7)      100         100 <br />  <br /> Pro(P)           CCT           12(2.9)      55         15            Arg( R)   CG1                5(1.2)       20           0 <br /> Pro              CCC            2(0.5)       9          8            Arg       CGC                0(0.0)        0           0 <br /> Pro              CCA            5(1.2)      23         77            Arg       CGA                2(0.5)        8           0 <br /> Pro              CCC            3(0.7)      14          0            Arg       CG6                ?(0.5)        6           0 <br />  <br />   Thr( )         ACT            5( .2)      31         36            Ser       AGT                 1(0.2)       3           0 <br />  Ihr             ACC            1(0.2)       6         64            Ser       AGC                 5(1.2)      17           0 <br />  Ibr             ACA            5(1.2)      31          0            Arg       AGA                10(2.4)      40         100 <br />  Tir             ACG            5(1.?)      31          0            Arg       AGG                 6(1.5)      24           0 <br />  <br />  Ala(A)           Cci          12(2.9)      46         54            CIy(C) <br />                                                                          G       C1                6( 1.5)     60          93 <br />  Ala              CCC           6(1.5)      23         46            Gly       GGC                 1(0.2)      10 <br />  Ala             6CA            7(1.7)      27          0            GIy        CGA                1(0.2)      10               0 <br />  Ala             GCC            1(0.2)           4      0            Gly       CGC                 2(0.5)      20               0 <br />  <br />  <br />  D. Predicted mino acid composition PRII protein <br />   Amime         Acid                  Amine Acid        t     Amino Acid                 I     Amino Acid    No             t <br />           Ala           26      6.4         CIN       15      3.7        Ler          39      9.S            Ser     29             7.1 <br />           At,            25     6.1         CIl       25      6.1        Lys          36      6.6            Tlr     16             3.9 <br />           As            721     5.1          C1y      10      2.4        met           9          2.7        Trp      7             1.7 <br />           Asp           35       6.6        Nis        9      2.7        Phe          21      5.1            lyr     IS             3.7 <br />           Cys                   7.0         11e       22      5.4        Pro          722     S.4            V1      19             4.6 <br />  <br />  <br />  <br />  <br />                                                                                                                                      7983 <br />  Nucleic Acids Research <br />  <br />                           I     a@Ix               v    B     T               B            c       v     B   a <br />  <br />  <br />                                                                                            A                              I <br />  <br />  <br />  <br />      0 <br />  <br />  <br />  <br />  <br />                        s0           100          1S0            t00          to50          300             30             400 <br />  <br />  <br />  <br /> Figure 3. Hydrophilicity profile secondary structure mature PRIl <br /> amino acids sequence predicted DNA sequence analysis. Positive nega- <br /> tive numbers correspond hydrophilic hydrophobic regions, respectively <br /> (22). = K-helix;     =   -sheet; T= 3-turn. <br />  <br /> The local secondary structure hydrophilicity value PRI1 gene <br /> product          predicted assisted analysis (21,22). As shown <br /> in Figure 3, regions                        CK-helical structure                10               contigous <br /> amino acids                located residues 43-52, 94-114, 136-151, 214-233, 294-313, <br /> 320-329, 354-363, 387-403. The longest contigous &gt;-turns                                                    p-sheets        re- <br />  <br /> gions        located residues 124-135 179-195, respectively. The hydro- <br /> philicity index PRIl protein (Figure 3),                                                     interest, <br /> because hydrophilicity values shown useful predict protein <br /> antigenic determinants amino acids                               sequences      (22). In fact, point of <br /> highest local average hydrophilicity invariably located in,                                                     adjacent to, <br /> an       antigenic determinant. Our attempts produce specific monoclonal antibo- <br /> bies DNA primase mojety yeast DNA polymerase-DNA primase <br /> complex                     unsuccessful (7). The                use           proper        synthetic    pep- <br />  <br /> tides derived PRIl sequence possibly help                                                reach goal. <br /> Such specific immunological reagents                                    useful purifying yeast <br /> proteins interacting vivo DNA primase. In fact, yeast DNA polyme- <br /> rase-primase complex itself                        purified using               monoclonal antibody raised <br /> against isolated DNA polymerase                        I   (3). <br /> Transcription mapping PRIl                          gene. <br />  <br /> Poly(A)       RNA prepared Saccharomyces cerevisiae strain TD28                                                  used in <br />  <br />  <br />  <br /> 7984 <br />                 -80 <br />  <br />  <br />  <br />  <br />                60 ^Z <br />  <br />  <br />  <br />  <br />                - 40 <br />  <br />  <br />  <br />  <br /> methods allow to <br />                           .&apos;:W <br />                           12 <br />  <br />  <br />  <br />  <br />                           V: <br />                             A <br />  <br />  <br />  <br />  <br />                           .f.;.- X <br />  <br />  <br />  <br />  <br />                            : : <br />  <br />  <br />  <br />  <br />                          measure <br />                                  .9wF <br />  <br />  <br />  <br />  <br />                                 _: <br />                                  _ <br />  <br />  <br />  <br />  <br />                                   s <br />                                     .. <br />  <br />  <br />                                    ....:; <br />                                     ::.:.. <br />  <br />  <br />                                     .:::S <br />  <br />  <br />                                   !:&apos;.:.:.F: <br />  <br />  <br />                                     ..so <br />                                         &apos; <br />  <br />  <br />                                    .:: &apos;. <br />                                   ..}.e <br />                                   .:::,f <br />                                  .. -.e.:e, <br />  <br />  <br />  <br />                                .. m R,. <br />                                          .... <br />  <br />  <br />  <br />  <br />                                          :: <br />  <br />  <br />  <br />  <br />                             *: . ::. Bbres <br />                                    .i..X#,, <br />  <br />  <br />  <br />  <br />                                      .:}}::.:: <br />                                     .:!: .^. <br />                                    *;:S:: <br />                                               .:. <br />  <br />  <br />  <br />  <br />                                               .. <br />  <br />  <br />  <br />  <br />                                    .&apos;.S2}3}. &apos;. <br />  <br />  <br />  <br />  <br />                                           :::::.. <br />                                                           12 <br />  <br />  <br />  <br />  <br />                                                           . . :.:. <br />                                                           . . . . . . . .A <br />                                                           . &apos;......... <br />                                                           *.. <br />  <br />  <br />  <br />  <br /> Figure 4. Determination 5&apos; 3&apos; end-points mature PRIl <br /> transcript. The 5&apos; termini PRIl transcripts mapped Sl prote- <br /> ction (Panel A) primer extension (Panel B), according procedures <br /> described Materials Methods. Panel C, shows Sl mapping 3&apos; <br /> end-point PRIl transcription. Lanes 1 2 results obtained <br /> when using 20 40 jg, respectively, poly(A) RNA Sl mapping and <br /> primer extension experiments. After treatment Sl, protected fragments <br /> were run 6% sequencing gel (Panel A C). The products primer <br /> extension shown Panel B analyzed 8% sequencing gel. <br />  <br /> Sl mapping experiments (23) primer extension AMV <br />                                                              B <br />  <br />  <br />  <br />  <br />                                                              .... . .&apos;.:..^...,.:. <br />                                                                     .. <br />  <br />  <br />  <br />  <br /> criptase (24) identify 5&apos; end-points mature PRIl message. Both <br />                                                                               .. <br />                                                                             ...: <br />                                                                                      -100 <br />  <br />  <br />                                                                                      -80 <br />  <br />  <br />  <br />                                                                                      -60 <br />  <br />  <br />  <br />  <br />                                                                                      -40 <br />                                                                                                 1 2 <br />  <br />  <br />  <br />  <br />                                                     distance 5&apos; mRNA terminus and <br />                                                                                                     C <br />                                                                                                         Nucleic Acids Research <br />  <br />  <br />  <br />  <br />                                                                                                            +1400 <br />                                                                                                            +1380 <br />  <br />                                                                                                            +1360 <br />  <br />                                                                                                            41340 <br />  <br />  <br />                                                                                                            +1320 <br />  <br />  <br />  <br />                                                                                                             +1300 <br />  <br />  <br />  <br />  <br />                                                                                                             +1280 <br />  <br />  <br />  <br />  <br />                                                                                                                reverse   trans- <br />  <br />  <br />                                                                                                                             site <br />  <br /> whose exact position sequence known. <br /> To   map   5&apos; terminus Sl digestion, plasmid pPPl                                              cut ScaI, 5&apos; end <br /> labeled T4 polynucleotide kinase                                               [32P]-ATP   NruI-ScaI frag- <br />  <br /> ment     used     probe. As shown Figure 4, Sl mapping 5&apos; terminus <br />  <br />  <br />                                                                                                                               7985 <br />  Nucleic Acids Research <br />  <br /> identifies major bands corresponding mRNA start sites position <br /> -61 -47 strength signal dependent concentra- <br /> tion   poly(A)+   mRNA. <br /> The existance transcription initiation points confirmed by <br /> primer extension experiments, using synthetic oligonucleotide complementary <br /> to nucleotide +1 +20 mRNA sequence incubated poly(A) <br /> RNA presence reverse transcriptase triphosphates (Figure 4). <br /> Again, intensity signals increased doubling RNA concentra- <br /> tion. <br /> A comparison number yeast leader sequences shows transcription <br /> initiation occurs nucleotide sequence PyAAPu (25), does not <br /> seem strict rule, yeast genes initiate transcription into <br /> a different nucleotide context (14). The sequence GAAAAG PRIl <br /> upstream transcription start related sequence GAAG 5&apos; <br /> end CDC9 (DNA ligase) mRNA (27). A similar motif present PRIl at <br /> position -126 -121 sequences GAAG, GAAAG, GAAAAG in <br /> the 5&apos; non coding region yeast genes CDC8 (thymidylate <br /> kinase) CDC9 (26,27). The PRIl gene periodically transcribed around <br /> the G1/S phase boundary interval yeast DNA synthesis genes <br /> CDC8, CDC9, CDC21 (thymidilate synthetase) POLl (DNA polymerase I) (28). <br /> Once complete sequence deletion analysis data available, compa- <br /> rison upstream regions genes eventually identify sequence <br /> similarities suggesting possible common regulatory mechanisms. <br /> The consensus sequence TATAAA (29) yeast TATA box the <br /> PRIl gene 5&apos; transcription start, identical sequence located <br /> at position -10 -5. In yeast genes characterized date, TATA <br /> like sequences quite hetherogeneous distances varying <br /> from 50-150 nucleotides transcription start (30). Furthermore, many <br /> of poorly expressed yeast genes recognisable TATA homology <br /> (31), appears case PRIl gene, but <br /> also CDC9 TOP2 genes (14,27). Perhaps reason, se- <br /> quence complementary yeast 18S rRNA (32) low- <br /> ly expressed yeast genes. However, PRIl 5&apos; non coding region high <br />  <br /> 7986 <br />                                                         Nucleic Acids Research <br />  <br /> A+T content, typical yeast promoter regions (17). From <br /> position -200 -1 A+T composition 67.5% compared 56.5% the <br /> PRIl coding region. <br /> The determination 3&apos; end-point PRIl transcript performed by <br /> labeling plasmid pPPl NdeI site using NdeI-SstI fragment <br /> as probe S1 nuclease protection experiments. As shown Figure 4 <br /> panel C, 3&apos; end-point PRIl transcript maps position +1373(&#194;&#177;l). <br /> This site located 142 bp TGA termination codon. The sequence <br /> between termination codon 3&apos; end-point 64% A+T, characteri- <br /> stic commonly yeast genes. The specific signals responsible <br /> for polyadenylation transcription termination Saccharomyces cerevisiae <br /> are defined. Sequences similar consensus elements proposed by <br /> Bennetzen Hall (33), Zaret Sherman (34) Henikoff et al. (35) are <br /> not readily apparent 3&apos; non coding region PRIl gene, for <br /> the presence short runs T residues region termina- <br /> tion codon 3&apos; end-point. The sequence AATAA, necessary <br /> for polyadenylation genes higher eukaryotes (34), found <br /> within larger sequence located position +1251 +1258. A deletion ana- <br /> lysis 3&apos; region PRIl gene nuclear runoff experiments be <br /> necessary determine effect specific sequences PRIl transcript <br /> stability translation efficiency distinguish poly- <br /> adenylation termination transcription. <br /> The PRIl gene yeast Saccharomyces cerevisiae sequenced <br /> gene related eukaryotic DNA primase. A search possible homology <br /> with prokaryotic DNA primases E.coli dnaG protein (36), T7 <br /> gene 4 protein (37) T4 genes 41 61, did reveal signifi- <br /> cant homology. The polypeptide structure eukaryotic DNA primases be <br /> highly conserved. In fact, DNA primase associated 48 kDa 58 kDa <br /> polypeptides DNA polymerase-DNA primase complexes purified seve- <br /> ral eukaryotic organisms (2). Once eukaryotic DNA primase genes be <br /> identified characterized, interesting evaluate whether <br /> the conserved structure mantained amino acids level, <br />  <br />  <br />  <br />                                                                              7987 <br />  Nucleic Acids Research <br /> thus defining possible functional domains            essential   enzyme   involved in <br /> the initiation DNA replication. <br />  <br /> ACKNOWLEDGMENTS <br /> This work     partially supported      grant Progetto Finalizzato Inge- <br /> gneria Genetica     e   Basi Molecolari delle Malattie Ereditarie, CNR, Roma, Italy. <br />  <br />  <br /> *To corrspondence adWessed <br />  <br /> REFERENCES <br />  1. Kornberg,A. (1980) DNA replication, W.H.Freeman Co.,San Francisco. <br />  2. Campbell.J.L. (1986) Ann.Rev.Biochem. 55, 733-771 <br />  3. Plevani,P.,Foiani,M.,Valsasnini,P.,Badaracco,G.,Cheriathundam,E. and <br />      Chang,L.M.S. (1985) J.Biol.Chem. 260, 7102-7107. <br />  4. Lucchini,G.,Francesconi,S.,Foiani,M.,Badaracco,G. Plevani,P. (1987) <br />      EMBO J. 6, 737-742. <br />  5. Lucchini,G.,Brandazza,A.,Badaracco,G.,Bianchi,M. Plevani,P. (1985) <br />      Curr.Genet. 10, 245-252. <br />  6. Johnson,L.M.,Snyder,b.,Chang,L .M.S.,Davis,R.W. Campbell,J.L. (1985) <br />      Cell. 43, 369-377. <br />  7. Plevani,P. ,Lucchini,G.,Foiani,M.,Valsasnini,P.,Brandazza,A.,Bianchi,M., <br />      Magni,G. Badaracco,G. (1987) Life Sci.Adv. 6, 53-60. <br />  8. Badaracco,G.,Valsasnini,P.,Foiani,M.,Benfante,R.,Lucchini,G. Plevani,?P <br />      (1986) Eur.J.Biochem. 161, 435-440. <br />  9. Henikoff,S. (1984) Gene. 28, 351-359. <br /> 10. Sanger,F.,Nicklen,S. Coulson,A.R. (1977) Proc.Natl.Acad.Sci.USA. 74, <br />      5413-5417. <br /> 11. Hattori,M. Sakaki,Y. (1986) Anal.Biochem. 152, 232-238. <br /> 12. Natsoulis,G.,Hilger,F. Fink,G.R. (1986) Cell. 46, 235-243. <br /> 13. Maxam,A. Gilbert,W. (1980) Methods Enzymol. 65, 499-560. <br /> 14. Giaver,G.,Lynn,R.,Goto,T. Wang,J.C. (1986) J.Biol.Chem. 261, 12448- <br />      12454. <br /> 15. Strauss,E.C.,Ko <br /> </body></html>