<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Dev Genes Evol. 2013 May ; 223(3): 171&#226;&#8364;&#8220;181. doi:10.1007/s00427-012-0432-6. <br />  <br />  <br />  <br />                            <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>RNA interference</span> used disrupt gene function in <br />                            tardigrades <br />                            Jennifer R. Tenlen1,*,3, Shaina McCaskill1,2, Bob Goldstein1 <br />                            1Dept. Biology, University North Carolina-Chapel Hill, CB 3280, Chapel Hill, NC 27599, USA <br />  <br />                            2Dept.   Biology, Fayetteville State University, 1200 Murchison Rd., Fayetteville, NC 28301, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  How morphological diversity arises key question evolutionary developmental biology. As a <br />                                  long-term approach address question, developing water bear Hypsibius dujardini <br />                                  (Phylum Tardigrada) model system. We expect using close relative well-studied <br />                                  models, Drosophila (Phylum Arthropoda) Caenorhabditis elegans (Phylum Nematoda), will <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  facilitate identifying genetic pathways relevant understanding evolution development. <br />                                  Tardigrades valuable research subjects investigating organisms biological <br />                                  materials survive extreme conditions. Methods disrupt gene activity essential of <br />                                  efforts, method exists Phylum Tardigrada. We developed protocol <br />                                  disrupt tardigrade gene functions double-stranded RNA-mediated RNA interference (RNAi). <br />                                  We targeting tardigrade homologs essential developmental genes RNAi produced <br />                                  embryonic lethality, targeting green fluorescent protein did not. Disruption gene <br />                                  functions appears relatively specific criteria: targeting distinct genes resulted in <br />                                  distinct phenotypes consistent predicted gene functions, RT-PCR, RNAi <br />                                  reduced level target mRNA control mRNA. These studies represent first <br />                                  evidence gene functions disrupted RNAi phylum Tardigrada. Our results <br />                                  form platform dissecting tardigrade gene functions understanding evolution of <br />                                  developmental mechanisms <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span> extreme environments. <br />  <br />  <br />                            Keywords <br />                                  Hypsibius dujardini; tardigrade; RNA interference; evo-devo; extreme environments <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Introduction <br />                                              A key question evolutionary developmental biology morphological diversity arises <br />                                              alterations developmental patterning mechanisms. Uncovering molecular changes <br />                                              contribute phenotypic variety animals facilitated comparing <br />                                              developmental processes closely-related organisms (Simpson 2002). As long-term <br />                                              approach address question origin diversification body plans, are <br />                                              developing close relative Drosophila (Phylum Arthropoda) C. elegans (Phylum <br />                                              Nematoda), water bear Hypsibius dujardini (Phylum Tardigrada), model system <br />                                              (Fig. 1a, b). These phyla members Ecdysozoa, protostome <br />                                              superclades (Aguinaldo et al. 1997). Recent molecular evidence supports clustering of <br />                                              arthropods, tardigrades onychophorans Panarthropoda superphylum segmented <br />                                              animals paired appendages (Rota-Stabelli et al. 2010; Campbell et al. 2011; Giribet and <br />  <br />  <br />                            * <br />                             Corresponding author: Jennifer R. Tenlen, tenlenj@spu.edu, phone: +1-206-281-2007, fax: +1-206-281-2882. <br />                            3Present address: Dept. Biology, Seattle Pacific University, 3307 3rd Ave W., Suite 205, Seattle, WA 98119, USA <br />                             Tenlen et al.                                                                                              Page 2 <br />  <br />  <br />                                            Edgecombe 2012). Whether tardigrades closely related arthropods and <br />                                            onychophorans nematodes remains controversial (Giribet Edgecombe 2012). <br />                                            Although Drosophila C. elegans well-studied model systems, little known <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            functions developmental genes Ecdysozoan phyla. Tardigrades share <br />                                            features arthropods nematodes, suggesting techniques <br />                                            studying development phyla transferable tardigrades. We expect that <br />                                            using close relative model systems, orthologous genes predicted <br />                                            similar functions, mean large number developmental genes and <br />                                            developmental processes valuable understanding evolution development. <br />  <br />                                            The tardigrade species H. dujardini favorable characteristics evo-devo <br />                                            studies. This species maintained laboratory cultures containing spring water <br />                                            green algae food source (Ammermann 1967; Gabriel et al. 2007). Embryos are <br />                                            optically clear, facilitating study development live microscopy. While males have <br />                                            described wild, H. dujardini raised culture date parthenogenetic; all <br />                                            embryos diploid females (Ammermann 1967; Ramazzotti Maucci 1983). Female <br />                                            reproduction tightly linked molting, broods 1&#226;&#8364;&#8220;10 embryos deposited the <br />                                            shed exoskeleton, exuvium (Bertolani 1983; Ramazzotti Maucci 1983; Gabriel et al. <br />                                            2007). Embryos hatch 4 days deposition; complete life cycle relatively <br />                                            brief, generation time 13&#226;&#8364;&#8220;14 days room temperature (Gabriel et al. 2007). H. <br />                                            dujardini compact genome 70 Mb, genome sequencing progress <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (Gabriel et al. 2007). Our lab developed protocols cryopreservation, live microscopy, <br />                                            immunostaining facilitate studies H. dujardini (Gabriel Goldstein 2007; <br />                                            Gabriel et al. 2007). Several tardigrade species shown unique ability <br />                                            withstand extreme conditions cryptobiosis (Wright 2001; Guidetti et al. 2011; We nicz <br />                                            et al. 2011). Evidence suggests H. dujardini survive dessication, the <br />                                            mechanisms underlying dessication tolerance well-understood species (Wright <br />                                            1989, 2001). Other tardigrade species multicellular organisms survive <br />                                            exposure outer space (J&#195;&#182;nsson et al. 2008; Rebecchi et al. 2009). These traits led a <br />                                            growing tardigrades research investigating organisms biological <br />                                            materials survive extreme conditions (J&#195;&#182;nsson 2007; M&#195;&#184;bjerg et al. 2011). <br />  <br />                                            <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> address issues above, essential able disrupt gene function H. dujardini. <br />                                            However, method exists Phylum Tardigrada. RNA interference (RNAi) a <br />                                            robust well-conserved mechanism gene silencing organisms, including <br />                                            nematodes arthropods (Hannon 2002; Cerutti Casas-Mollano 2006). In C. elegans, <br />                                            RNAi systemic heritable; injection double-stranded RNA (dsRNA) the <br />                                            intestine germline adults disrupt gene activity injected animals their <br />                                            progeny (Fire et al. 1998). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            We developed protocol RNA interference H. dujardini microinjection of <br />                                            dsRNA. We targeted H. dujardini genes identified homology genes with <br />                                            known functions development organisms. We demonstrate injection of <br />                                            dsRNA resulted sequence-specific gene silencing, effective method to <br />                                            dissect gene function species valuable understanding evolution of <br />                                            developmental mechanisms, survival extreme environments. <br />  <br />                               Materials methods <br />                               Tardigrade culture <br />                                            H. dujardini cultures maintained room temperature spring water fed green <br />                                            algae, 60 mm 150 mm glass Petri dishes, described (Gabriel et al. 2007). <br />  <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                           Page 3 <br />  <br />  <br />                               Preparation genomic DNA <br />                                            To separate tardigrades algae food, took advantage H. dujardini&#226;&#8364;&#8482;s <br />                                            phototactic behavior (B.G., unpublished observation). Cultures 150 mm dishes were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            placed near full-spectrum light source hours. Tardigrades accumulated one <br />                                            plate, periodically transferred clean glass dish. Collected <br />                                            tardigrades allowed starve overnight, water changed times to <br />                                            reduce algal contamination. Tardigrades transferred low-retention microcentrifuge <br />                                            tubes (Fisher), allowed settle gravity. The supernatant removed, the <br />                                            tardigrade pellets stored &#226;&#710;&#8217;80&#194;&#176;C. Four 150 mm dishes typically yielded <br />                                            approximately 30 &#206;&#188;g compacted tardigrades. <br />  <br />                                            Genomic DNA prepared approximately 30 &#206;&#188;g mixed-stage tardigrades using <br />                                            DNeasy Blood Tissue kit (Qiagen), following manufacturer&#226;&#8364;&#8482;s protocol. After <br />                                            elution, genomic DNA concentrated ethanol precipitation, resuspended in <br />                                            nuclease-free water Tris-Cl solution. <br />  <br />                               Preparation cDNA <br />                                            Mixed-stage embryos (n &gt; 50) collected transferred microcentrifuge tube. <br />                                            Embryos centrifuged supernatant removed. Crushed pieces glass coverslips <br />                                            added tube, embryos vortexed 100 &#206;&#188;l TRIzol Reagent <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (Invitrogen) room temperature hours. 1/5 volume chloroform added the <br />                                            tube extract RNA. Extracted RNA concentrated ethanol precipitation, and <br />                                            resuspended nuclease-free water. Genomic DNA removed treatment DNaseI, <br />                                            Amp Grade (Invitrogen) directed. Oligo-dT-primed first-strand cDNA synthesis was <br />                                            performed using SuperScript III kit (Invitrogen) following manufacturer&#226;&#8364;&#8482;s <br />                                            recommendations. <br />  <br />                               Cloning <br />                                            Sequences cloned study listed Table 1; closest D. melanogaster C. <br />                                            elegans genes identified BLAST sequence listed Online Resource 1. <br />                                            Sequences aligned using ClustalW (Thompson et al. 1994) (Online Resource 2). <br />  <br />                                            Gene-specific primers used amplify target sequence genomic DNA or <br />                                            cDNA. Primer sequences listed Online Resource 3. PCR products cloned into <br />                                            pGEM-T-Easy vector (Promega), clones sequenced gene. <br />  <br />                                            Each gene amplified second PCR reaction, using gene-specific primers T7 <br />                                            polymerase promoter site added 5&#226;&#8364;&#178; end primer. The final PCR product was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cleaned using PCR Purification Kit (Qiagen) directed. DNA eluted Tris-Cl. <br />  <br />                                            Plasmid pJT06 created PCR amplification GFP pie-1:GFP expression <br />                                            vector (Strome et al. 2001), using sequence-specific primers XhoI XbaI adapters. <br />                                            The GFP PCR product cloned XhoI/XbaI sites vector L4440 (Fire Lab <br />                                            Vector Kit, Addgene). Vector-specific primers used amplify GFP flanking T7 <br />                                            polymerase promoter sites PCR (Online Resource 3). <br />  <br />                               RNA interference <br />                                            For gene, dsRNA synthesized 1 &#206;&#188;g T7-tagged DNA, using T7 <br />                                            RiboMax Express RNAi System (Promega) directed. Following ethanol precipitation, <br />                                            dsRNA resuspended nuclease-free water. Working stocks prepared 1 &#206;&#188;g/&#206;&#188;l <br />                                            concentration, dsRNA specific &#226;&#8364;&#178; &#226;&#8364;&#178; halves Hd-mag-1/mago nashi, <br />                                            prepared 1.5 &#206;&#188;g/&#206;&#188;l concentration. <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                              Page 4 <br />  <br />  <br />                                            Injection needles pulled borosilicate glass capillaries (World Precision <br />                                            Instruments) using Flaming/Brown Micropipette Puller (Sutter Instrument Co.). Needles <br />                                            back-filled dsRNA immediately prior injections. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Approximately 20&#226;&#8364;&#8220;30 adult females transferred deep depression slide rinsed <br />                                            times spring water. To anesthetize animals, water replaced with <br />                                            Levamisole (Sigma-Aldrich) dissolved spring water final concentration 5 mM. <br />                                            Animals largely immobile 15 minutes. Animals remained viable anesthetic <br />                                            hour. <br />  <br />                                            Injection slides prepared securing right triangle cut corner 22&#195;&#8212;22 <br />                                            mm 1 &#194;&#189; glass coverslip standard glass microscope slide. Halocarbon oil (Halocarbon <br />                                            Products Inc.) overlaid coverslip, tardigrades transferred to <br />                                            slide. Tardigrades braced straight edge coverslip (Fig. 1c, d). <br />  <br />                                            Injections performed using Picosptrizer II (Parker-Hamilton Corp.) Three-Axis <br />                                            Oil Micromanipulator (Narshige MMO-203), mounted Nikon Eclipse TE300 inverted <br />                                            microscope equipped 20&#195;&#8212; Plan Fluor objective. The Picospritzer set deliver <br />                                            100-ms pulses air injection needle. Needle tips broken gently stroking the <br />                                            tips edge triangular coverslip piece. In case, needle inserted <br />                                            gonad intestine, successful injection confirmed swelling gonad <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            intestine injection. To ensure tardigrades tolerate microinjection, adult <br />                                            females injected phenol red solution diluted 1:100 spring water. Of 10 adults <br />                                            injected, 8 recovered produced offspring, suggesting microinjection not <br />                                            deleterious tardigrades. In 8 adults, phenol red collected center site of <br />                                            injection (either gonad intestine), eventually dissipated. <br />  <br />                                            Injected tardigrades transferred depression slide containing spring water and <br />                                            allowed recover 30 minutes. Injected animals transferred single <br />                                            wells sterile, non-treated 96-well plates (Grenier bio) 100 &#206;&#188;l spring water and <br />                                            5 &#206;&#188;l algae added. After round egg-laying, adults transferred new <br />                                            wells easily monitor progeny. Injected animals progeny monitored <br />                                            30 days. For injected females, recorded total number broods deposited, the <br />                                            number embryos brood, time broods. Since embryos hatch 4.5 <br />                                            days deposition, noted embryos failed hatch 5 days, where <br />                                            possible, examined embryos DIC microscopy (see below). Embryos hatched <br />                                            followed began reproducing. For animals, recorded lifespan <br />                                            monitored apparent changes morphology behavior. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Microscopy <br />                                            Live embryos prepared microscopy methods. In method, <br />                                            embryos transferred drop bottled spring water epoxy-coated three-well slides <br />                                            (Cel-Line/Thermo Scientific), supported glass microspheres (diameter 37.36 &#194;&#177;0.39 &#206;&#188;m, <br />                                            Whitehouse Scientific). In second method, embryos mounted freshly-prepared <br />                                            agar pads (2% agarose spring water). Coverslips sealed molten Vaseline. 4-D <br />                                            differential interference contrast (DIC) microscopy carried Nikon Eclipse 800 <br />                                            microscope mounted Diagnostic Instruments SPOT2 camera. Images acquired <br />                                            1 &#206;&#188;m optical sections analyzed Metamorph v 6.3r7 (Molecular Devices). <br />  <br />                               Single-embryo cDNA synthesis RT-PCR <br />                                            Embryos collected deposition exuvium molting female. This <br />                                            collection strategy permitted stage embryos embryos initiated or <br />                                            completed mitosis time. In cases embryo deposited <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                           Page 5 <br />  <br />  <br />                                            exuvium, embryos separated using 26G &#194;&#189; needle (Becton- <br />                                            Dickinson). Embryos mounted imaging described assess their <br />                                            phenotypes. To synthesize cDNA single embryos, modified protocol developed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            C. elegans embryos (Robertson et al. 2004). Single H. dujardini embryos were <br />                                            transferred microscope slide cap thin-walled, 0.2 ml PCR tube. The <br />                                            water replaced 0.2 &#206;&#188;l cDNA synthesis buffer prepared reagents <br />                                            included SuperScript III First-Strand cDNA Synthesis kit (Invitrogen), supplemented <br />                                            detergent Triton-X-100 (Sigma) [1X RT Buffer, 5 mM MgCl2, 10 mM DTT, 2.5 <br />                                            &#206;&#188;M oligo(dT), 0.5 mM dNTPs, 40U RNase OUT, 0.5% Triton-X-100]. The embryo was <br />                                            briefly frozen dry ice, crushed using tip 26G &#194;&#189; needle. An additional 1.8 &#206;&#188;l <br />                                            cDNA synthesis buffer added cap, extract collected brief <br />                                            centrifugation. The extract frozen dry ice. For first-strand cDNA synthesis, 17 &#206;&#188;l of <br />                                            cDNA synthesis buffer added extract, incubated 65&#194;&#176;C 5 minutes. <br />                                            After step, SuperScript III protocol followed according manufacturer&#226;&#8364;&#8482;s <br />                                            instructions. <br />  <br />                                            For RT-PCR, 3 &#206;&#188;l first-strand cDNA product used template, using <br />                                            primers specific &#206;&#178;-tubulin (Hd-tbb-1) mago nashi [Hd-mag-1(FL)] (Online Resource <br />                                            3). Thermal cycling conditions follows: 94&#194;&#176;C 30 sec (1 cycle); 94&#194;&#176;C 30 sec, <br />                                            59&#194;&#176;C 30 sec, 72&#194;&#176;C 45 sec (35 cycles); 72&#194;&#176;C 10 min (1 cycle). All products were <br />                                            separated gel electrophoresis, stained ethidium bromide, visualized UV light. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Gel images captured digitally pixel values band measured in <br />                                            ImageJ (National Institutes Health). The ratio &#206;&#178;-tubulin mago nashi pixel intensities <br />                                            calculated wild-type Hd-mag-1(RNAi) embryos, statistical <br />                                            significance assessed student&#226;&#8364;&#8482;s t-test (unpaired, two-tailed). <br />  <br />                               Results <br />                               Molecular cloning H. dujardini genes <br />                                            Some organisms respond introduction dsRNAs sequence-non-specific manners <br />                                            (Stark et al. 1998). We reasoned organism molecular tools existed <br />                                            date, gauge specificity RNAi using biological indicator &#226;&#8364;&#8220; by <br />                                            determining extent disrupting genes predicted functions result in <br />                                            distinct predictable phenotypes. To identify H. dujardini genes predicted roles <br />                                            development, compared BLAST H. dujardini ESTs inGenBank C. elegans <br />                                            D. melanogaster genomes, ordering results E values. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> resulting lists, we <br />                                            selected H. dujardini genes loss-of-function phenotypes predicted <br />                                            based functions homologs C. elegans and/or Drosophila (Table 1, Online Resource <br />                                            1). The proteins encoded genes include putative homologs actin, Deformed/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            HoxB4, dynein light chain, 14-3-3 protein, Mago nashi, muscle myosin. The genes were <br />                                            cloned PCR amplification genomic DNA cDNA using gene-specific primers. <br />                                            While H. dujardini ESTs prepared mixed-stage animals, confirmed RT- <br />                                            PCR genes appeared expressed embryonically (data shown). <br />  <br />                                            Two gene products appeared contain complete coding sequences, determined by <br />                                            presence predicted full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> ORF alignment D. melanogaster C. elegans <br />                                            sequences (Online Resource 2). Hd-dlc-1 encodes full-length protein 89 aa with <br />                                            significant identity dynein light chain. Alignment sequences genomic DNA and <br />                                            cDNA shows Hd-dlc-1 locus contains exons separated single intron <br />                                            (Online Resource 4a). Hd-mag-1 encodes protein 147 aa significant identity to <br />                                            Mago nashi. A single exon 444 bp defines Hd-mag-1 locus; introns <br />                                            (Online Resource 4b). The remaining gene products contained 400 bp coding <br />                                            sequence. <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                            Page 6 <br />  <br />  <br />                               RNAi microinjection dsRNA <br />                                            To disrupt gene function H. dujardini, adapted protocol microinjection dsRNA <br />                                            C. elegans (Fire et al. 1998), anesthetizing animals mounting custom <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            injection chamber (see Materials Methods). To assess efficacy RNAi, each <br />                                            dsRNA injected, monitored injected adults progeny 30 days following <br />                                            injections. Microinjection dsRNA did appear deleterious viability adult <br />                                            females 86% injected females recovered (n=203). Of females recovered, 60% <br />                                            laid brood embryos (n=175). We monitored injected females their <br />                                            progeny parameters, including brood size, brood frequency, viability. For <br />                                            genes, embryonic lethality observed progeny females injected with <br />                                            dsRNA (Fig. 2a; Online Resource 5). For example, 86% progeny females injected <br />                                            Hd-act-1/actin dsRNA failed hatch (n=22), 16% Hd-mag-1/mago <br />                                            nashi(399bp)(RNAi) embryos did hatch (n=207). Conversely, injection adults with <br />                                            water dsRNA targeting green fluorescent protein (GFP) effect embryo viability <br />                                            development (Fig. 2a, Online Resource 5). In experiments, progeny hatched and <br />                                            progressed adulthood obvious defects morphology, behavior fecundity. <br />  <br />                                            In addition effects embryo viability, injection gene-specific dsRNA appeared to <br />                                            affect fecundity injected adults. While water-injected H. dujardini females produced <br />                                            average 3.4 broods 30 days injection, significant reductions brood <br />                                            numbers observed females injected dsRNA targeting Hd-act-1/actin, Hd- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ftt-1/14-3-3 Hd-dlc-1/dynein light chain (Fig. 2b, Online Resource 5). Surprisingly, <br />                                            females injected Hd-mag-1 dsRNA significant increase number broods; <br />                                            effect fecundity described loss mago nashi function other <br />                                            organisms. <br />  <br />                                            H. dujardini females laid eggs discrete broods approximately 4 days apart throughout <br />                                            reproductive lifespan (mean=4.4&#194;&#177;1.5 SD, range 3 &#226;&#8364;&#8220; 9 days, n=42 broods). Females <br />                                            produced average 2 embryos brood (mean=2.2&#194;&#177;0.9 SD, n=92 embryos). In the <br />                                            course analyzing RNAi experiments, observed percentage embryonic <br />                                            lethality decreased significantly successive brood. For example, Hd- <br />                                            mag-1(RNAi) experiments, majority progeny laid brood did hatch <br />                                            (59%, n=39), 25% progeny laid second brood did hatch (n=36) <br />                                            (Fig. 2c). Within brood, highest proportion lethality observed embryos <br />                                            laid days injection (Online Resource 6). After brood, all <br />                                            progeny hatched developed adults. These results suggest penetrance of <br />                                            phenotypes following injection dsRNA time-sensitive. <br />  <br />                                            We sought determine efficacy RNAi depended template which <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            dsRNA prepared. While presence introns dsRNA does negatively affect the <br />                                            efficacy RNAi C. elegans (Fire et al. 1998), results study inconclusive. <br />                                            For example, dsRNA specific Hd-myo-1/myosin prepared genomic DNA <br />                                            cDNA. Injection Hd-myo-1 dsRNA prepared genomic DNA, containing one <br />                                            predicted intron, effect embryo viability (100% hatched; n=82), 16.7% of <br />                                            progeny injected Hd-myo-1 dsRNA prepared cDNA failed hatch (Fig. 2a; <br />                                            Online Resource 5). Conversely, injection Hd-act-1 dsRNA prepared genomic <br />                                            DNA, contains predicted introns, strong effect animal development (Fig. <br />                                            2a; Online Resource 5). It determine minimum length contiguous <br />                                            coding sequence dsRNA necessary effectively silence target genes tardigrades. <br />  <br />  <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                              Page 7 <br />  <br />  <br />                               Sequence-specific disruption gene function <br />                                            To assess specificity injected dsRNA target gene, compared embryonic <br />                                            development progeny injected females phenotypes expected based on <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            gene&#226;&#8364;&#8482;s predicted function. H. dujardini embryos develop stereotyped <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />                                            (Gabriel et al. 2007). By ~20 hours egg deposition, wild-type embryos begin to <br />                                            elongate anterior-posterior axis, markers tissue differentiation, as <br />                                            birefringent granules produced intestine, appear (Fig. 3a). After 4 days, just prior <br />                                            hatching, well-developed structures present embryos, including pharynx and <br />                                            intestine (Fig. 3b). Below, phenotypes associated disruption genes: <br />                                            Hd-act-1/actin, Hd-dlc-1/dynein light chain, Hd-ftt-1/14-3-3, Hd-mag-1/mago nashi. <br />  <br />                                            The protein actin highly-conserved roles cell division, cell movement, other <br />                                            processes (Pollard Cooper 2009). Based actin&#226;&#8364;&#8482;s fundamental role cytokinesis, we <br />                                            predicted Hd-act-1(RNAi) embryos display defects cell division, resulting in <br />                                            multinucleated cells. Of 17 Hd-act-1(RNAi) embryos examined DIC microscopy, 10 <br />                                            arrested early development, multinucleated cells (Fig. 3d). Filming early <br />                                            embryogenesis confirmed cytokinesis disrupted (Online Resource 7). 6 17 <br />                                            embryos appeared express birefringent gut granules, marker intestinal tissue <br />                                            differentiation (Gabriel Goldstein 2007; Gabriel et al. 2007), failed elongate along <br />                                            anterior-posterior axis. 1 embryo appeared complete morphogenesis failed to <br />                                            hatch. These results consistent role actin cytokinesis cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            processes. Interestingly, females injected Hd-act-1 dsRNA laid brood died <br />                                            days, failing produce additional broods (n=14). This observation suggests <br />                                            depletion Hd-act-1 affected embryogenesis progeny, the <br />                                            physiology injected adults. Injection short interfering RNAs (siRNAs) targeting actin <br />                                            effect animal viability development (data shown), suggesting longer <br />                                            dsRNA necessary trigger RNAi. <br />  <br />                                            Dynein light chain subunit cytoplasmic dynein, complex microtubule-associated <br />                                            proteins roles cell division, nuclear migration, intracellular transport, germ cell <br />                                            proliferation (Karki Holzbaur 1999; Dorsett Schedl 2009). Among Hd-dlc-1(RNAi) <br />                                            embryos, 3 11 embryos failed hatch. These embryos initiated, did complete <br />                                            morphogenesis. Each embryo appeared contain partial pharynx, lacked birefringent <br />                                            granules seen intestines wild-type embryos (Fig. 3f). These 3 embryos produced <br />                                            different mothers, supoorting conclusion <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> likely specific to <br />                                            loss function Hd-dlc-1. <br />  <br />                                            14-3-3 proteins involved protein-protein binding, conserved roles cell <br />                                            polarity germline development (Benton et al. 2002; Morton et al. 2002). Targeting a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            14-3-3-encoding gene resulted 26% embryos failing hatch (n-39). Phenotypes were <br />                                            highly variable. Interestingly, Hd-ftt-1(RNAi) appeared affect fecundity injected <br />                                            females. While water-injected H. dujardini females produced average 3.4 broods (&#194;&#177;2.1 <br />                                            SD) 30 days injection, females injected Hd-ftt-1/14-3-3 dsRNA produced an <br />                                            average 1.4 broods (&#194;&#177;0.9 SD) (Fig. 2b, Online Resource 5). These data consistent <br />                                            possible role 14-3-3 protein tardigrade germline development. <br />  <br />                                            Mago nashi highly-conserved protein multiple roles development, particularly in <br />                                            morphology. In D. melanogaster, mago nashi function required specification of <br />                                            posterior segments maintenance germline (Boswell et al. 1991). In C. elegans, <br />                                            mago nashi required embryo elongation anterior-posterior axis, for <br />                                            sex determination (Li et al. 2000). Of 28 Hd-mag-1(RNAi) embryos failed hatch, 14 <br />                                            produced birefringent gut granules, marker intestinal tissue differentiation, failed to <br />                                            elongate anterior-posterior axis (Fig. 3e). This phenotype consistent with <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                              Page 8 <br />  <br />  <br />                                            elongation defects observed C. elegans mag-1 mutant embryos (Li et al. 2000). Of the <br />                                            remaining embryos, 12 arrested earlier stage, 2 completed morphogenesis but <br />                                            failed hatch. Among progeny hatched developed adulthood, no <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            obvious defects morphology, germline development fecundity. As test the <br />                                            specificity Hd-mag-1(RNAi), generated dsRNA shorter regions Hd- <br />                                            mag-1. Injection mago-5&#226;&#8364;&#178; dsRNA mago-3&#226;&#8364;&#178; dsRNA (Online Resource 6) <br />                                            produced phenotypes progeny similar injection Hd-mag-1(399bp) dsRNA (Online <br />                                            Resource 5). These results support conclusion Hd-mag-1 dsRNA targets Hd- <br />                                            mag-1 specifically. <br />  <br />                                            The observation dsRNA produced distinct reproducible phenotypes suggested <br />                                            cognate genes specifically targeted. To test conclusion, we <br />                                            used RT-PCR confirm injection dsRNA resulted depletion target mRNA. <br />                                            Seven wild-type seven Hd-mag-1(RNAi) embryos collected 48 hours first <br />                                            mitosis, cDNA synthesized single embryos (see Materials Methods). <br />                                            Using first-strand cDNA template, Hd-tbb-1/&#206;&#178;-tubulin Hd-mag-1 were <br />                                            amplified PCR, separated gel electrophoresis. Comparison relative ratios of <br />                                            Hd-mag-1 Hd-tbb-1 amplified wild-type Hd-mag-1(RNAi) embryos <br />                                            demonstrated expression Hd-mag-1 significantly reduced Hd-mag-1(RNAi) <br />                                            embryos (unpaired, two-tailed t-test: P &lt; 0.05) (Fig. 4b, c). We conclude injection of <br />                                            dsRNA resulted specific depletion target mRNA. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Discussion <br />                                            RNAi potent method disrupting assessing gene function organisms <br />                                            (Cerutti Casas-Mollano 2006). Long dsRNAs elicit sequence non-specific responses <br />                                            organisms, triggering, example, interferon responses mammalian cells (Stark <br />                                            et al. 1998). We dsRNA-mediated RNAi used disrupt gene <br />                                            functions tardigrade H. dujardini. We predict development method for <br />                                            RNAi species valuable investigating evolution developmental <br />                                            mechanisms survival extreme environments. <br />  <br />                                            The penetrance RNAi appears time-sensitive, progeny laid earliest <br />                                            broods affected. With exception Hd-act-1(RNAi), cases embryonic <br />                                            lethality observed progeny dsRNA-injected females, percentage embryonic <br />                                            lethality higher brood subsequent broods (Fig. 2a, c). Several reasons <br />                                            explain observation. First, eggs significant portion adult&#226;&#8364;&#8482;s body <br />                                            volume, egg laying, adults lose significant percentage volume (Ramazzotti <br />                                            Maucci 1983; Rebecchi Bertolani 1994). It possible loss volume <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            flushes dsRNA cleavage products, diluting effect RNAi brood. There <br />                                            general mechanism promoting turnover dsRNA, resulting time- <br />                                            dependent loss dsRNA (Bosher Labouesse 2000; Tuschl 2001). Finally, site of <br />                                            injection important. In organisms C. elegans, site dsRNA injection <br />                                            does affect efficacy RNAi (Fire et al. 1998), cases, injection into <br />                                            gonad produced stronger phenotype (Ahringer 2006). We observed RNAi phenotypes <br />                                            progeny females injected intestine ovary, future experiments <br />                                            determine importance site injection RNAi penetrance. It be <br />                                            interesting determine explanations correct optimizing the <br />                                            RNAi protocol. <br />  <br />                                            The long-term goal study facilitate utilization tardigrades&#226;&#8364;&#8482; close relationship <br />                                            arthropods nematodes understanding evolution development. For <br />                                            example, position tardigrades panarthropod phylum makes potentially <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                              Page 9 <br />  <br />  <br />                                            valuable outgroup determining ancestral states arthropod ancestors, alongside the <br />                                            panarthropod phylum, onychophorans, gene disruption technique yet <br />                                            exists (Janssen et al. 2010). We focused genes known affect development other <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            organisms. Homeobox (Hox) genes highly-conserved roles morphogenesis in <br />                                            multiple systems. In H. dujardini EST database, putative homolog of <br />                                            Deformed/HoxB4, required normal head morphology D. melanogaster <br />                                            (Regulski et al. 1987). However, injection dsRNA targeting Hd-dfd-1/deformed no <br />                                            effect embryo viability development, morphological abnormalities <br />                                            observed hatched progeny. This result suggest Hd-dfd-1 mRNA not <br />                                            sufficiently depleted, Hd-dfd-1 does function H. dujardini development, it <br />                                            acts redundantly gene. Hd-dfd-1 appears sufficiently conserved suggest that <br />                                            likely function H. dujardini. As H. dujardini genome sequence becomes <br />                                            available, interesting identify homeobox genes assess roles in <br />                                            development H. dujardini body plan. <br />  <br />                                            In addition homeobox genes, focused Mago nashi, ancient highly-conserved <br />                                            protein diverse phyla (Micklem et al. 1997; Newmark et al. 1997; Wiens et <br />                                            al. 2006). In organisms, Mago nashi roles body axis specification, in <br />                                            reproduction, regulating germline development involvement sex <br />                                            determination pathway (Boswell et al. 1991; Newmark Boswell 1994; Newmark et al. <br />                                            1997; Li et al. 2000; Wiens et al. 2006; Parma et al. 2007). We did observe any <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            deleterious effects germline development adults injected Hd-mag-1 dsRNA, in <br />                                            viable progeny. This result suggest Mago nashi required germline <br />                                            development H. dujardini, depletion insufficient reveal role germline <br />                                            development requiring low levels mRNA, embryonic phenotype masks <br />                                            later developmental role. Phenotypic analysis Hd-mag-1(RNAi) embryos revealed <br />                                            majority affected embryos failed elongate anterior-posterior axis, but <br />                                            able initiate differentiation certain tissues, including intestine. This phenotype <br />                                            distinct observed D. melanogster mago mutant embryos, lack <br />                                            abdominal segments posterior pole cells (germline precursor cells), indicative defects <br />                                            establishing anterior-posterior axis oocytes (Boswell et al. 1991). Instead, Hd- <br />                                            mag-1(RNAi) phenotype resembles observed C. elegans embryos lacking Mago nashi <br />                                            function, fail complete elongation (Li et al. 2000). This conserved role in <br />                                            embryonic elongation suggests Mago nashi played role elongation in <br />                                            ancestral Ecdysozoans. Alternatively, role evolved tardigrades and <br />                                            nematodes independently, distinct processes required elongation. <br />                                            Determining downstream effectors mago nashi H. dujardini C. elegans <br />                                            provide valuable insight evolution Mago nashi function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            In conclusion, demonstrated RNAi effective method disrupt gene <br />                                            function H. dujardini. This method represents protocol disrupting gene <br />                                            function member Phylum Tardigrada. Our initial analysis developmentally- <br />                                            important genes, Hd-act-1/actin, Hd-dlc-1/dynein light chain, Hd-ftt-1/14-3-3, and <br />                                            Hd-mag-1/mago nashi, suggests functions proteins encoded genes <br />                                            dissected reverse genetics approaches. We expect ability disrupt gene <br />                                            functions H. dujardini valuable studies evolution development and <br />                                            survival extreme environments. <br />  <br />                               Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                                             Page 10 <br />  <br />  <br />                               Acknowledgments <br />                                            We thank Bob McNuff continuing help tardigrade cultures, Victoria Madden help electron <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            microscopy, members Goldstein lab critical reading manuscript. J.R.T. supported the <br />                                            National Institutes Health, Minority Opportunities Research division National Institute General <br />                                            Medical Sciences (NIGMS) grant K12GM000678, NIH-funded UNC Developmental Biology Training <br />                                            Grant (T32HD046369-03). S.M. supported ARRA funding Minority Opportunities Research <br />                                            (MORE) division National Institute General Medical Sciences (NIGMS) supplementing IRACDA/ <br />                                            SPIRE grant K12GM000678. This work supported NSF grants IOS 0235658 IOS 0652007 B.G. <br />  <br />  <br />                               References <br />                                            Aguinaldo AM, Turbeville JM, Linford LS, Rivera MC, Garey JR, Raff RA, Lake JA. Evidence a <br />                                               clade nematodes, arthropods moulting animals. Nature. 1997; 387:489&#226;&#8364;&#8220;493. [PubMed: <br />                                               9168109] <br />                                            Ahringer, J. Research Community. The C. elegans. WormBook; 2006. Reverse genetics. <br />                                            Ammermann D. Die Cytologie der Parthenogenese bei dem Tardigraden Hypsibius dujardini. <br />                                               Chromosoma. 1967; 23:203&#226;&#8364;&#8220;213. [PubMed: 5594788] <br />                                            Benton R, Palacios IM, St Johnston D. Drosophila 14&#226;&#8364;&#8220;3&#226;&#8364;&#8220;3/PAR-5 essential mediator PAR-1 <br />                                               function axis formation. Dev Cell. 2002; 3:659&#226;&#8364;&#8220;671. [PubMed: 12431373] <br />                                            Bertolani, R. Reproductive Biology Invertebrates. Vol. I. John Wiley &amp; Sons; Chichester: 1983. <br />                                               Tardigrades; p. 431-441. <br />                                            Bosher JM, Labouesse M. RNA interference: genetic wand genetic watchdog. Nature Cell Biol. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               2000; 2:E31&#226;&#8364;&#8220;E36. [PubMed: 10655601] <br />                                            Boswell RE, Prout ME, Steichen JC. Mutations newly identified Drosophila melanogaster gene, <br />                                               mago nashi, disrupt germ cell formation result formation mirror-image symmetrical <br />                                               double abdomen embryos. Development. 1991; 113:373&#226;&#8364;&#8220;384. [PubMed: 1765008] <br />                                            Campbell LI, Rota-Stabelli O, Edgecombe GD, Marchioro T, Longhorn SJ, Telford MJ, Philippe H, <br />                                               Rebecchi L, Peterson KJ, Pisani D. MicroRNAs phylogenomics resolve relationship of <br />                                               Tardigrada suggest velvet worms sister group Arthropoda. Proc Natl Acad Sci U <br />                                               S A. 2011; 108:15920&#226;&#8364;&#8220;15924. [PubMed: 21896763] <br />                                            Cerutti H, Casas-Mollano JA. On origin functions RNA-mediated silencing: protists to <br />                                               man. Curr Genet. 2006; 50:81&#226;&#8364;&#8220;99. [PubMed: 16691418] <br />                                            Dorsett M, Schedl T. A role dynein inhibition germ cell proliferation fate. Mol Cell Biol. <br />                                                2009; 29:6128&#226;&#8364;&#8220;6139. [PubMed: 19752194] <br />                                            Dunn CW, Hejnol A, Matus DQ, Pang K, Browne WE, Smith SA, Seaver E, Rouse GW, Obst M, <br />                                                Edgecombe GD, S&#195;&#184;rensen MV, Haddock SHD, Schmidt-Rhaesa A, Okusu A, Kristensen RM, <br />                                                Wheeler WC, Martindale MQ, Giribet G. Broad phylogenomic sampling improves resolution of <br />                                                animal tree life. Nature. 2008; 452:745&#226;&#8364;&#8220;749. [PubMed: 18322464] <br />                                            Epstein HF, Waterston RH, Brenner S. A mutant affecting heavy chain myosin in <br />                                                Caenorhabditis elegans. J Mol Biol. 1974; 90:291&#226;&#8364;&#8220;300. [PubMed: 4453018] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fire A, Xu S, Montgomery MK, Kostas SA, Driver SE, Mello CC. Potent specific genetic <br />                                                interference double-stranded RNA Caenorhabditis elegans. Nature. 1998; 391 (6669):806&#226;&#8364;&#8220; <br />                                                811. [PubMed: 9486653] <br />                                            Gabriel WN, Goldstein B. Segmental expression Pax3/7 engrailed homologs tardigrade <br />                                                development. Dev Genes Evol. 2007; 217:421&#226;&#8364;&#8220;433. [PubMed: 17516081] <br />                                            Gabriel WN, McNuff R, Patel SK, Gregory TR, Jeck WR, Jones CD, Goldstein B. The tardigrade <br />                                                Hypsibius dujardini, new model studying evolution development. Dev Biol. 2007; <br />                                                312:545&#226;&#8364;&#8220;559. [PubMed: 17996863] <br />                                            Giribet G, Edgecombe GD. Reevaluating Arthropod Tree Life. Annu Rev Entomol. 2012; <br />                                                57:167&#226;&#8364;&#8220;186. [PubMed: 21910637] <br />                                            Guidetti R, Altiero T, Rebecchi L. On dormancy strategies tardigrades. J Insect Physiol. 2011; <br />                                                57:567&#226;&#8364;&#8220;576. [PubMed: 21402076] <br />                                            Hannon GJ. RNA interference. Nature. 2002; 418:244&#226;&#8364;&#8220;251. [PubMed: 12110901] <br />  <br />  <br />  <br />                                                 Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                                Page 11 <br />  <br />  <br />                                            Hejnol A, Obst M, Stamatakis A, Ott M, Rouse GW, Edgecombe GD, Martinez P, Baguna J, Bailly X, <br />                                                Jondelius U, Wiens M, M&#195;&#188;ller WEG, Seaver E, Wheeler WC, Martindale MQ, Giribet G, Dunn <br />                                                CW. Assessing root bilaterian animals scalable phylogenomic methods. Proc R Soc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Lond B Biol Sci. 2009; 276:4261&#226;&#8364;&#8220;4270. <br />                                            Janssen R, Eriksson BJ, Budd GE, Akam M, Prpic N-M. Gene expression patterns onychophoran <br />                                                reveal regionalization predates limb segmentation pan-arthropods. Evolution &amp; <br />                                                Development. 2010; 12:363&#226;&#8364;&#8220;372. [PubMed: 20618432] <br />                                            J&#195;&#182;nsson KI. Tardigrades potential model organism space research. Astrobiology. 2007; 7:757&#226;&#8364;&#8220; <br />                                                766. [PubMed: 17963475] <br />                                            J&#195;&#182;nsson KI, Rabbow E, Schill RO, Harms-Ringdahl M, Rettberg P. Tardigrades survive exposure to <br />                                                space low Earth orbit. Curr Biol. 2008; 18:R729&#226;&#8364;&#8220;R731. [PubMed: 18786368] <br />                                            Karki S, Holzbaur EL. Cytoplasmic dynein dynactin cell division intracellular transport. <br />                                                Curr Opin Cell Biol. 1999; 11:45&#226;&#8364;&#8220;53. [PubMed: 10047518] <br />                                            Li W, Boswell R, Wood WB. mag-1, homolog Drosophila mago nashi, regulates hermaphrodite <br />                                                germ-line sex determination Caenorhabditis elegans. Dev Biol. 2000; 218:172&#226;&#8364;&#8220;182. [PubMed: <br />                                                10656761] <br />                                            Micklem DR, Dasgupta R, Elliott H, Gergely F, Davidson C, Brand A, Gonz&#195;&#161;lez-Reyes A, St Johnston <br />                                                D. The mago nashi gene required polarisation oocyte formation of <br />                                                perpendicular axes Drosophila. Curr Biol. 1997; 7:468&#226;&#8364;&#8220;478. [PubMed: 9210377] <br />                                            Morton DG, Shakes DC, Nugent S, Dichoso D, Wang W, Golden A, Kemphues KJ. The <br />                                                Caenorhabditis elegans par-5 gene encodes 14&#226;&#8364;&#8220;3&#226;&#8364;&#8220;3 protein required cellular asymmetry in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                early embryo. Dev Biol. 2002; 241:47&#226;&#8364;&#8220;58. [PubMed: 11784094] <br />                                            M&#195;&#184;bjerg N, Halberg KA, J&#195;&#184;rgensen A, Persson D, Bj&#195;&#184;rn M, Raml&#195;&#184;v H, Kristensen RM. Survival in <br />                                                extreme environments - current knowledge adaptations tardigrades. Acta Physiol. <br />                                                2011; 202:409&#226;&#8364;&#8220;420. <br />                                            Newmark PA, Boswell RE. The mago nashi locus encodes essential product required germ <br />                                                plasm assembly Drosophila. Development. 1994; 120:1303&#226;&#8364;&#8220;1313. [PubMed: 8026338] <br />                                            Newmark PA, Mohr SE, Gong L, Boswell RE. mago nashi mediates posterior follicle cell-to- <br />                                                oocyte signal organize axis formation Drosophila. Development. 1997; 124:3197&#226;&#8364;&#8220;3207. <br />                                                [PubMed: 9272960] <br />                                            Parma DH, Bennett PE Jr, Boswell RE. Mago Nashi Tsunagi/Y14, respectively, regulate <br />                                                Drosophila germline stem cell differentiation oocyte specification. Dev Biol. 2007; 308:507&#226;&#8364;&#8220; <br />                                                519. [PubMed: 17628520] <br />                                            Pollard TD, Cooper JA. Actin, central player cell shape movement. Science. 2009; 326:1208&#226;&#8364;&#8220; <br />                                                1212. [PubMed: 19965462] <br />                                            Ramazzotti, G.; Maucci, W. Il Phylum Tardigrada (The Phylum Tardigrada). In: Beasley, CW., <br />                                                translator. Memoire dell&#226;&#8364;&#8482;Istituto Italiano di Idrobiologia Dott. Marco Marchi. 3. Vol. 41. <br />                                                Istituto Italiano di Idrobiologia; Verbania Pallanza: 1983. <br />                                            Rebecchi L, Altiero T, Guidetti R, Cesari M, Bertolani R, Negroni M, Rizzo AM. Tardigrade <br />                                                resistance space effects: results experiments LIFE-TARSE mission FOTON- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                M3 (September 2007). Astrobiology. 2009; 9:581&#226;&#8364;&#8220;591. [PubMed: 19663764] <br />                                            Rebecchi L, Bertolani R. Maturative pattern ovary testis eutardigrades freshwater and <br />                                                terrestrial habitats. Invert Reprod Dev. 1994; 26:107&#226;&#8364;&#8220;118. <br />                                            Regulski M, McGinnis N, Chadwick R, McGinnis W. Developmental molecular analysis of <br />                                                Deformed; homeotic gene controlling Drosophila head development. EMBO J. 1987; 6:767&#226;&#8364;&#8220;777. <br />                                                [PubMed: 16453752] <br />                                            Robertson SM, Shetty P, Lin R. Identification lineage-specific zygotic transcripts early <br />                                                Caenorhabditis elegans embryos. Dev Biol. 2004; 276:493&#226;&#8364;&#8220;507. [PubMed: 15581881] <br />                                            Rota-Stabelli O, Kayal E, Gleeson D, Daub J, Boore JL, Telford MJ, Pisani D, Blaxter M, Lavrov DV. <br />                                                Ecdysozoan mitogenomics: evidence common origin legged invertebrates, the <br />                                                Panarthropoda. Genome Biol Evol. 2010; 2:425&#226;&#8364;&#8220;440. [PubMed: 20624745] <br />                                            Simpson P. Evolution development closely related species flies worms. Nature Rev Genet. <br />                                                2002; 3:1&#226;&#8364;&#8220;11. <br />  <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                                 Page 12 <br />  <br />  <br />                                            Stark GR, Kerr IM, Williams BR, Silverman RH, Schreiber RD. How cells respond interferons. <br />                                                Annu Rev Biochem. 1998; 67:227&#226;&#8364;&#8220;264. [PubMed: 9759489] <br />                                            Strome S, Powers J, Dunn M, Reese K, Malone CJ, White J, Seydoux G, Saxton W. Spindle dynamics <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                role gamma-tubulin early Caenorhabditis elegans embryos. Mol Biol Cell. 2001; <br />                                                12:1751&#226;&#8364;&#8220;1764. [PubMed: 11408582] <br />                                            Telford MJ, Bourlat SJ, Economou A, Papillon D, Rota-Stabelli O. The evolution Ecdysozoa. <br />                                                Philos Trans R Soc Lond B Biol Sci. 2008; 363:1529&#226;&#8364;&#8220;1537. [PubMed: 18192181] <br />                                            Thompson JD, Higgins DG, Gibson TJ. CLUSTAL W: improving sensitivity progressive <br />                                                multiple sequence alignment sequence weighting, position-specific gap penalties and <br />                                                weight matrix choice. Nucleic Acids Res. 1994; 22:4673&#226;&#8364;&#8220;4680. [PubMed: 7984417] <br />                                            Tuschl T. RNA interference small interfering RNAs. Chembiochem. 2001; 2:239&#226;&#8364;&#8220;245. [PubMed: <br />                                                11828450] <br />                                            We&#197;&#8218;nicz W, Grohme MA, Kaczmarke &#197;?, Schill RO. Anhydrobiosis tardigrades - The decade. J <br />                                                Insect Physiol. 2011; 57:577&#226;&#8364;&#8220;583. [PubMed: 21440551] <br />                                            Wiens M, Belikov SI, Kaluzhnaya OV, Krasko A, Schr&#195;&#182;der HC, Perovic-Ottstadt S, M&#195;&#188;ller WEG. <br />                                                Molecular control serial module formation apical-basal axis spong Lubomirskia <br />                                                baicalensis: silicateins, mannose-binding lectin mago nashi. Dev Genes Evol. 2006; 216:229&#226;&#8364;&#8220; <br />                                                242. [PubMed: 16380844] <br />                                            Wright JC. Desiccation tolerance water-retentive mechanisms tardigrades. J Exp Biol. 1989; <br />                                                142:267&#226;&#8364;&#8220;292. <br />                                            Wright JC. Cryptobiosis 300 years van Leuwenhoek: What learned about <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                tardigrades? Zoologischer Anzeiger. 2001; 240:563&#226;&#8364;&#8220;582. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Dev Genes Evol. Author manuscript; available PMC 2014 May 01. <br />                             Tenlen et al.                                                                                           Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 1. <br />                                            Phylogenetic position anatomy H. dujardini. Evolutionary position tardigrades, <br />                                            relationships selected taxa shown. Within Ecdysozoa, arthropods, onychophorans <br />                                            tardigrades proposed form Panarthropoda sub-clade. Some studies place <br />                                            tardigrades sister clade nematodes nematomorphs. Drawn (Dunn et al. <br />                                            2008; Telford et al. 2008; Hejnol et al. 2009; Rota-Stabelli et al. 2010; Campbell et al. <br />                                            2011). b Scanning electron m <br /> </body></html>