<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Theor Biol. 2014 March 21; 345: 12&#226;&#8364;&#8220;21. doi:10.1016/j.jtbi.2013.12.009. <br />  <br />  <br />  <br />                            Incorporation Flexible Objectives Time-Linked Simulation <br />                            Flux Balance Analysis <br />                            Elsa W. Bircha, Madeleine Udellb, Markus W. Covertc <br />                            aChemical Engineering, Stanford University, Stanford, CA, 94305, USA <br />  <br />                            bComputational       Mathematical Engineering, Stanford University, Stanford, CA, 94305, USA <br />                            cBioengineering,       Stanford University, Stanford, CA, 94305, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                 We present modifications Flux Balance Analysis (FBA) metabolic modeling <br />                                 framework relax implicit assumptions biomass reaction. Our flexible Flux Balance <br />                                 Analysis (flexFBA) objective removes fixed proportion reactants, therefore <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 produce subset biomass reactants. Our time-linked Flux Balance Analysis (tFBA) simulation <br />                                 removes fixed proportion reactants byproducts, describe <br />                                 transitions metabolic steady states. Used together, flexFBA tFBA model time scale <br />                                 shorter regulatory growth steady state encoded biomass reaction. This <br />                                 combined short-time FBA method intended integrated modeling applications enable <br />                                 detailed dynamic depictions microbial physiology whole-cell modeling. For <br />                                 example, modeling Escherichia coli, avoids artifacts caused low-copy-number <br />                                 enzymes single-cell models kinetic bounds. Even outside integrated modeling contexts, the <br />                                 detailed predictions flexFBA tFBA complement existing FBA techniques. We show <br />                                 detailed metabolite production silico knockouts used identify correct essentiality <br />                                 predictions wrong reason. <br />  <br />  <br />                            Keywords <br />                                 Metabolism; Integrated Modeling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            1. Introduction <br />                                              Quantitative metabolic models important tools understanding engineering the <br />                                              behavior microorganisms. Flux Balance Analysis (FBA) powerful technique <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                              simulate large metabolic networks kinetic parameters unavailable. FBA <br />                                              simulations capture microorganism growth, nutritional resource consumption, waste- <br />                                              product secretion rates [1, 2]. In addition FBA generate <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>knockout</span> essentiality <br />                                              predictions treated hypotheses explore organism&apos;s metabolic capability <br />                                              [3, 4]. <br />  <br />  <br />  <br />  <br />                            Crown Copyright &#194;&#169; 2013 The Authors. Published Elsevier Ltd. All rights reserved <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Birch et al.                                                                                            Page 2 <br />  <br />  <br />                                           Classical implementations FBA quantify microbial growth using rigid biomass reaction <br />                                           represents processes cell replication single proportion reactants <br />                                           required byproducts returned. It used quantify microbial growth another <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           objective used refine flux predictions evaluate perturbations [5, 6, 7]. <br />  <br />                                           The biomass reaction produce balanced growth complete inactivity as <br />                                           predictions. For applications assumptions underlying all-or-nothing behavior <br />                                           valid results useful. However, current silico biology <br />                                           incorporates FBA integrated models combine mathematical models different <br />                                           types interact simulation [8, 9]. For applications &#226;&#8364;&#8221; notably whole-cell <br />                                           models [10] &#226;&#8364;&#8221; rigid biomass reaction limitation. <br />  <br />                                           To enable whole-cell modeling, require nuanced alternative biomass <br />                                           reaction FBA produce metabolites non-wild-type non-steady-state <br />                                           proportions. <br />  <br />                                           In work, relax implicit assumptions biomass reaction construct new <br />                                           FBA methods. The assumption balanced population average growth, encoded by <br />                                           biomass reaction&apos;s fixed proportion reactants. The second assumption steady <br />                                           state growth, encoded biomass reaction&apos;s fixed proportion byproducts reactants. <br />                                           Relaxing reactant byproduct assumptions results flexible FBA (flexFBA) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           time-linked FBA (tFBA) approaches, respectively. <br />  <br />                                           Together, balanced steady-state growth assumptions inherent biomass reaction in <br />                                           FBA make method applicable timescale longer regulatory cell process <br />                                           interactions. By combining flexFBA tFBA methods relax assumptions, <br />                                           obtain short-time FBA appropriate use whole-cell models. This short-time scale is <br />                                           consistent whole-cell models evaluate metabolic model timescales shorter <br />                                           regulatory process interactions explicitly represent. <br />  <br />                                1.1. Biomass Reaction Assumptions <br />                                           The biomass reaction ubiquitous microbial FBA lends great predictive power <br />                                           under-constrained metabolic network. It succinct mathematical form is <br />                                           composed straightforward parameter values. In addition quantifying growth, the <br />                                           biomass reaction flux used optimization objective case be <br />                                           called `biomass objective&apos; [11]. Much literature evaluates ability various FBA <br />                                           objectives mimic observed growth, gene essentiality, flux states [6, 12, 13, 14, 15], <br />                                           comparison `biomass objective&apos; performance. In contrast, discuss <br />                                           simulation regimes biomass reaction does adequately model range of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           metabolic network function, longer relevant quantification growth. <br />  <br />                                           By constraining process reactant requirements byproduct returns the <br />                                           biomass reaction combines subtly different assumptions deal (1) <br />                                           reactant-to-reactant (2) byproduct-to-reactant groups. <br />  <br />                                           Reactant-to-reactant fixed proportion biomass reaction assumes population average <br />                                           balanced growth: homogeneity cells cells time. This assumption is <br />                                           contained biomass reaction&apos;s negative coefficients. As consequence, biomass <br />                                           reaction scales fractional fulfillment process reactants whichever most <br />                                           limited. Homogeneity cells arises biomass reaction coefficients <br />                                           bulk cell composition values. For single cells short timescales homogeneity <br />                                           conflicts biological reality. Bulk phenotypes given average neglect <br />                                           variance underlying population [16, 17]. Strict temporal homogeneity metabolite <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 3 <br />  <br />  <br />                                           production ratios unreasonable transcriptional translational regulatory <br />                                           mechanisms enforce operate timescales longer typical FBA time <br />                                           step (1 sec minutes [18, 19]). Furthermore, regulatory interactions exist <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           metabolites included biomass reaction enforce proportional <br />                                           production. Experimental observations reveal essential metabolites be <br />                                           produced non-wild-type proportions [20, 21, 22, 23]. Additionally, metabolites <br />                                           included biomass reaction essential model growth. If biomass reaction <br />                                           includes process reactants non-essential cell replication, false-essential <br />                                           predictions result [24]. Previously, inflexible ratio essentiality biomass <br />                                           reaction addressed alternate biomass reaction definitions, [24, 25] reactions <br />                                           allowing similar metabolites substitute [26]; approaches are <br />                                           practical entire scale pathways metabolism. <br />  <br />                                           Byproduct-to-reactant fixed proportion biomass reaction assumes steady state <br />                                           metabolic function. This assumption contained biomass reaction&apos;s positive coefficients. <br />                                           The principle example return spent energy carrier ADP set proportional the <br />                                           ATP produced time step. Proportional byproducts reactants means the <br />                                           ADP return immediately matched capacity metabolism recharge ATP, <br />                                           consistent previous time step&apos;s metabolic conditions. Relating the <br />                                           reactant byproduct quantities reasonable, long-time assumption implied within <br />                                           single evaluation FBA optimization. Perturbations changes available media <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           resources result immediate transition new steady state single time step <br />                                           applied. This type transition unrealistic short time step evaluations <br />                                           energy carrier supply turnover limited. <br />  <br />                                2. Methods <br />                                           Mathematically biomass reaction consists coefficients mi metabolites Mi appended <br />                                           single reaction column stoichiometric matrix S equation, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                                         (1) <br />  <br />                                           values mi &lt; 0 represent consumption process reactants, mi &gt; 0 represent <br />                                           byproduct return. A majority n total mi values zero associated <br />                                           metabolites participate cell processes metabolism. Coefficient magnitudes <br />                                           anabolic products given relative quantities bulk biomass [11]; Coefficient <br />                                           magnitudes case catabolic energy carriers given requirements for <br />                                           macromolecule synthesis calculated bulk yields [11]. Units mi usually chosen <br />                                           flux biomass reaction, element flux vector v denoted vbio, <br />  <br />                                           directly interpreted microbial growth rate [1]. Metabolite accumulation       is <br />                                           set zero [1], applying steady state assumption metabolic network intermediates on <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                Page 4 <br />  <br />  <br />                                             timescale evaluation &#206;&#8221;t, typically 1 s longer. Using biomass reaction flux the <br />                                             maximization objective, optimization problem is: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                     (2) <br />  <br />  <br />  <br />                                             S&#226;&#8364;&#178; = [S | m] v&#226;&#8364;&#178; vbio appended end v Equation 1. <br />  <br />                                             Because coefficients mi quantities required basis cell mass, we <br />                                             convenient think biomass reaction flux vbio fractional fulfillment of <br />                                             requirement time. The classical FBA biomass reaction requires the <br />                                             fractional fulfillment metabolite requirements same. <br />  <br />                                2.1. Designing Biomass Reaction Alternative <br />                                             While sought relax biomass reaction&apos;s assumptions, wanted simultaneously <br />                                             preserve behavior wild-type long-time limits. We developed flexFBA to <br />                                             produce possible process reactants inhibition distant/unrelated blocked <br />                                             pathways maintaining population average cell composition wild type networks. <br />                                             We developed tFBA allow observation transient behavior integration other <br />                                             biological process models reproducing steady state growth. An additional design <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             constraint need integrated modeling methods computationally efficient and <br />                                             function human supervision. <br />  <br />                                             Neither flexFBA tFBA rely interfere other, implemented <br />                                             separately together. We treat individually initial methods explanation <br />                                             evaluation. However, flexFBA tFBA relax separate long-time assumptions <br />                                             biomass reaction, implementing achieve short-time FBA <br />                                             suitable use whole-cell models. <br />  <br />                                2.2. Reactant Flexibility: flexFBA <br />                                             As long process reactants participate reaction provided by <br />                                             network rates related nonzero constant multiplier. To remove constraint and <br />                                             produce independently, append coefficients separate reactions the <br />                                             stoichiometric matrix: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                                              (3) <br />  <br />                                             fluxes corresponding reactions fi appended v, intuitively <br />                                             represent fractional fulfillments requirement metabolites processes, they <br />                                             vary another. We combined matrix Equation the <br />                                             combined vector . The blocks appended S square diagonal general forms of <br />                                             Equations 3 5, all-zero columns neglected practical use. <br />  <br />  <br />                                                  J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                            Page 5 <br />  <br />  <br />                                           The objective criteria applied incentivize process reactant metabolite production &#226;&#8364;&#8221; <br />                                           large values fi &#226;&#8364;&#8221; simultaneously encourage proportional production &#226;&#8364;&#8221; similar values <br />                                           fi. In wild type case, objective result process reactant exchange and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           metabolic network identical biomass reaction flux maximization case in <br />                                           Equation (2). We achieve mathematically using objective, <br />  <br />                                                                                                                   (4) <br />  <br />  <br />                                           fatp fractional fulfillment energy carrier ATP. It maximizes fractional <br />                                           fulfillment ATP penalizing metabolite produced proportionally to <br />                                           ATP. The weight &#206;&#179; applied penalty terms constant, explain choose <br />                                           later. Figure 1a compares biomass reaction flux maximization flexible form. <br />  <br />                                           The &#226;&#8222;&#8220;1 norm penalty encourages sparsity arguments; incentivizing fi match <br />                                           fatp drawing fi constrained low. Similar <br />                                           motivations &#226;&#8222;&#8220;1 norm use robust regularized regression [27, 28]. In <br />                                           contrast, &#226;&#8222;&#8220;2 norm does penalize small errors, draws solution outliers. <br />                                           We &#226;&#8222;&#8220;2 norm numerically stable, failing reliable results the <br />                                           metabolic system. <br />  <br />                                           ATP production fatp used representative metabolic function, based biological <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           importance large process requirement mi. Including ATP explicitly objective is <br />                                           consistent previous FBA implementations include ATP rate yield the <br />                                           optimization criteria maintenance energy flux constraints [6]. Alternatives choice <br />                                           include functions fi values mean, prone trade-offs between <br />                                           metabolites, especially orders magnitude spanned values mi. <br />  <br />                                           We additionally introduce constraint fatp &#226;&#8240;&#165; fi, resulting simplification to <br />                                           difference penalty terms. However, constraint optional &#226;&#8222;&#8220;1 norm penalties be <br />                                           used subsequent equations instead. Because ATP coefficient mATP based bulk <br />                                           yield measurements accounts thermodynamic inefficiencies of <br />                                           metabolism, chosen maintain strictly fatp &#226;&#8240;&#165; fi constraint. We <br />                                           include optional biomass reaction flux weight &#206;&#178; optimization criteria, so <br />                                           problem is, <br />  <br />  <br />  <br />  <br />                                                                                                                         (5) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The biomass group term gbio, included su ciently large &#206;&#178;, assures no <br />                                           metabolite produced using biomass reaction FBA. Including the <br />                                           biomass group alters exFBA simulation pathway partially restricted <br />                                           upstream branch multiple process reactant metabolites, discuss subsequently. <br />                                           The problem Equation 5 forced biomass reaction solution setting <br />                                           0 &#226;&#8240;&#164; gbio = fi. <br />  <br />                                           Basic flexFBA simulation results shown Figure 1b identical metabolite <br />                                           production biomass reaction FBA wild-type network. When glycogen synthesis <br />                                           capacity removed simulated <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span>, biomass reaction predicts no <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                            Page 6 <br />  <br />  <br />                                           metabolite production, flexFBA predicts wild type level production all <br />                                           metabolites glycogen. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Simulations Figure 2 exemplify contribution biomass group gbio term. <br />                                           Pathway restriction makes metabolic precursor tyrosine phenylalanine limited <br />                                           resource. E. coli biomass contains slightly greater amounts phenylalanine tyrosine <br />                                           (mtyr &lt; mphe), 57% flux branch point heading phenylalanine. The <br />                                           process requirement tyrosine smaller means increment fractional <br />                                           fulfillment costly terms limited precursor flexible terms (fatp with <br />                                           penalty Equation (4) produce ftyr &gt; fphe. The biomass group incentivizes <br />                                           proportional production extent flux permitted: ftyr = fphe single <br />                                           knockdown (left columns). This contribution limited restricted process <br />                                           metabolites, just biomass reaction been. We preference of <br />                                           flexFBA smaller requirement downstream restricted branch biomass <br />                                           group additionally constrained &#226;&#8364;&#8221; Figure 2 glycogen pathway restriction the <br />                                           double knockdown (right columns, Figure 2). <br />  <br />                                           The addition &#206;&#178;gbio flexible objective ways increase fi similarity <br />                                           subsets process metabolites. Mathematically terms positively <br />                                           weight group production biomass group term. Taking example reactants <br />                                           DNA synthesis, entail adding &#206;&#178;dNTPgdNTP maximization criteria and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           constraint gdNTP &#226;&#8240;&#164; fj j = {dATP, dTTP, dGTP, dCTP} problem Equation 5. <br />                                           Similarly, deviations group penalized using form like flexible <br />                                           objective itself, strict limit imposed members fj &#226;&#8240;&#164; mink fk, j, k = {dATP, dTTP, <br />                                           dGTP, dCTP}. These additional terms apply assumption varying strictness <br />                                           molecular interactions maintain proportion metabolites timescale <br />                                           FBA evaluation. However, assigning groups weighting terms increases the <br />                                           parameterization potentially complicates optimization problem. We consider current <br />                                           experimental evidence insufficient guide justify including groups generally. <br />  <br />                                2.3. Time-Linked Byproducts: tFBA <br />                                           Biomass reaction FBA represent steady states growth transients <br />                                           steady states. This limitation process byproducts returned <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                           reactants provided processes optimization time step. In tFBA the <br />                                           byproducts time step t available metabolic network time step t + &#206;&#8221;t With <br />                                           modification remove long term steady state assumption biomass <br />                                           reaction. <br />  <br />                                           Additional motivation tFBA construct integrated models advantageous to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           separate process reactant requirements byproduct return. The biomass reaction be <br />                                           thought integrated model lumping cell processes together. Because biomass <br />                                           reaction expressed single linear equation, included linear for <br />                                           optimization. However, possible represent processes differential equation form, <br />                                           especially stochastic processes, framework. Once required evaluate <br />                                           FBA optimization process models separately, simulation include methods to <br />                                           assure metabolic solution exists overall simulation conserves mass &#226;&#8364;&#8221; keep <br />                                           processes metabolism consistent. A solution maintaining process-metabolism <br />                                           consistency preserving byproduct-to-reactant ratio FBA optimization <br />                                           required multiple evaluations simulated time step. We applied solution <br />                                           FBA integrated modeling Birch et al. 2012 [8], computationally <br />                                           inefficient, prone time step dependent artifacts, problematic for <br />                                           stochastic models. The preferable alternative tFBA: process-metabolism consistency can <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                               Page 7 <br />  <br />  <br />                                           enforced straightforward metabolite quantities separating process reactant <br />                                           consumption byproduct return occur time steps. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The biomass reaction summarizes molecular steps share reactants and <br />                                           byproducts, mi summarizes reactant byproduct stoichiometric coefficient ri pi <br />                                           respectively: mi = pi + ri, ri &lt; 0 pi &gt; 0. In tFBA replace mi reactant <br />                                           coefficient ri reactions appended S matrix, <br />  <br />  <br />  <br />  <br />                                                                                                                                          (6) <br />  <br />                                           separate exchange reactions xi process byproducts added, the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           matrix flux vector denoted&#197;&#339; v&#204;&#8218;. The bounds byproduct exchanges xl &#226;&#8240;&#164; x &#226;&#8240;&#164; 0 <br />                                           set uptake, based available byproduct metabolite returned the <br />                                           processes prior time step. In case simple proportionality used to <br />                                           represent processes, steady state wild type network exchange reactions will <br />                                           flux bounds values xi = xl,i &#226;&#8364;&#8220; pif f fractional fulfillment all <br />                                           metabolites steady state. For transient metabolic states bounds xl,i will <br />                                           depend ci,j, fj,(t&#226;&#8364;&#8220;&#206;&#8221;t) fractional fulfillment related reactant Mj previous <br />                                           time step. Note maintain FBA steady state assumption respect metabolic <br />                                           intermediates, addresses intermediate timescale fluctuations in <br />                                           metabolite concentrations enzyme-small molecule interactions respond quickly, <br />                                           longer regulatory responses. The optimization problem statement tFBA is: <br />  <br />  <br />  <br />  <br />                                                                                                                      (7) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           case flexFBA used. To implement tFBA retain fixed <br />                                           proportion reactants constraint Equation 7 0 &#226;&#8240;&#164; gbio = fi. <br />  <br />                                           To present impact tFBA method simulations, compare previous <br />                                           methods generating FBA time courses. Such methods consist updating media <br />                                           concentrations based resources consumed discrete time steps, most <br />                                           thorough accounting called dynamic FBA (dFBA) Mahadevan et. al. 2002 [2]. Such <br />                                           simulations capture organism growth rate shifts time step resources is <br />                                           exhausted. However, time step growth steady state respect biomass <br />                                           reaction, resulting time course sequence growth steady states. Figure 3a b <br />                                           illustrate reactant-byproduct relationship time course biomass reaction FBA <br />                                           steady states, tFBA, respectively. Upon change conditions, biomass reaction FBA <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 8 <br />  <br />  <br />                                            byproduct return informed new conditions &#226;&#8364;&#8221; arrow from <br />                                            reactants byproducts Figure 3a appears reversed respect time &#226;&#8364;&#8221; and <br />                                            immediately achieves steady state. In comparison, tFBA byproduct return function of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            reactant consumption previous time step condition. As result, transitions over <br />                                            number evaluations new steady state. At steady-state growth, time steps are <br />                                            identical result obtained long time limit. <br />  <br />                                            Figures 3c d process reactants byproducts example transition from <br />                                            acetate glucose media conditions biomass reaction FBA tFBA simulations. <br />                                            Transition time distinction, tFBA accounts difference in <br />                                            resources needed maintain states. In E. coli network purine energy carriers ATP <br />                                            ADP important example. Using biomass reaction FBA, the <br />                                            modeling assumptions additional ADP assumed exist soon be <br />                                            phosphorylated glucose media condition, tFBA simulation includes synthesis <br />                                            additional purine molecules metabolic network. The higher glucose growth <br />                                            rate requires twice number ATP plus ADP sustain [29], certainly preferable to <br />                                            account resources used synthesis. Because Figure 3 simulations employ the <br />                                            wild type network necessary media resources available produce balanced growth, <br />                                            flexFBA biomass reaction objective produce results shown. <br />  <br />                                            We note quantities reactants byproducts exchanged metabolism <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            processes representative physiological metabolite concentrations. <br />                                            These quantities accounting practicality representing discrete metabolic steps and <br />                                            cell function linear representation. However, variables this <br />                                            metabolism-process interface provide opportunity methods development in <br />                                            FBA whole-cell modeling fields. <br />  <br />                                2.4. Implementation <br />                                            The genome scale metabolic model used main text figures slight expansion on <br />                                            iMC1010 [30]. Text files used reaction network modeling information are <br />                                            included <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> associated publication. Kinetic bounds reaction <br />                                            perturbations included source <span id='am-317' about='Thesaurus:code' typeof='owl:Thing'>code</span>. Simulations implemented Python, with <br />                                            linear programming completed using CVXOPT [31] GLPK MOSEK. Source code for <br />                                            simulation figure generation available simtk.org/home/flexfbatfba. <br />  <br />                                            Simulations silico knockouts entail setting associated reaction flux constraints vl = <br />                                            vu = 0. For tFBA simulations, use process byproduct protons metabolism required. <br />                                            Fluxes accounted simulations cell gram dry cell <br />                                            weight basis, displayed unless noted. If &#206;&#178; nonzero hold its <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            value large, &#206;&#178; &#226;&#8240;&#171; 1. <br />  <br />                                            The weighting penalty terms 0 &lt; &#206;&#179; &lt; 1 production biomass <br />                                            metabolites ATP, tighter lower upper bounds depending the <br />                                            metabolic network biomass metabolite count (theoretically &#206;&#179; &#226;&#8240;&#165; 1 feasible but <br />                                            practically interferes solver). For main text simulations work value 0.1 <br />                                            used, chosen based total count process reactants terms avoid solution <br />                                            convergence biomass reaction limit. An analysis metabolite production sensitivity <br />                                            &#206;&#179; (Supplementary Figure S1) shows value chosen produce the <br />                                            desired qualitative flexFBA solution, robust various conditions multiple <br />                                            perturbations. Figure S1 includes simulations physiologically reasonable range of <br />                                            growth conditions, range metabolites constrained low production. <br />  <br />  <br />  <br />  <br />                                                 J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 9 <br />  <br />  <br />                                3. Results <br />                                3.1. Knockouts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Using combined flexFBA tFBA achieve short-time FBA, simulate the <br />                                             metabolic network dynamic response silico gene knockout perturbation. When we <br />                                             applied knockouts, observed broad types simulation results, examples of <br />                                             presented Figure 4. <br />  <br />                                             Many knockouts converge immediately steady state, Figure 4a. An example <br />                                             pgsA catalyzes step synthesis E. coli membrane components cardiolipin <br />                                             phosphatidylglycerol. Immediately constraining pgsA flux zero, the <br />                                             associated process reactant production zero (top left Figure 4a) continues the <br />                                             long time limits (top right Figure 4a). Meanwhile, unrelated biomass reactants and <br />                                             byproducts continue produced (bottom left right Figure 4a). Lack cardiolipin <br />                                             phosphatidylglycerol pgsA knockout simulation consistent experimental <br />                                             evidence strain shown lack membrane components [32] and <br />                                             survive conditions. <br />  <br />                                             A smaller group knockouts display qualitatively different metabolite synthesis <br />                                             immediately long perturbation, exemplified purA Figure 4b. Flux <br />                                             reaction novo purine synthesis set zero, short term <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             purines provided processes (top right Figure 4b). Continued purine <br />                                             availability tFBA purines returned previous step byproduct spent <br />                                             energy carrier ADP. Long-term gradual decrease occurs time step of <br />                                             returned purines sequestered cell macromolecules RNA `soluble pool&apos; <br />                                             maintenance &#226;&#8364;&#8221; mathematically mamp + matp &lt; &#226;&#710;&#8217;madp &#226;&#8364;&#8221; ability synthesize <br />                                             adenosine small molecule nutrients free declines (top right Figure 4b). <br />                                             Based flexFBA constraint fm &lt; fatp, represents biological dependance all <br />                                             metabolic activity ATP, biomass reactants decrease proportionally (bottom left <br />                                             right Figure 4b). Experimentally, purA knockouts purine auxotrophs [33], consistent <br />                                             flexFBA tFBA <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>phenotype</span> predictions. <br />  <br />                                             The flexFBA tFBA techniques allow compare knockout simulations experiment <br />                                             metabolite individually, binary essential/non-essential gene <br />                                             classifications biomass reaction FBA. Previously, detailed predictions have <br />                                             possible testing media supplements simulations, or <br />                                             computational searches FBA solution space [34]. Figure 5 emphasizes the <br />                                             increase prediction techniques summarizing results like Figure <br />                                             4. In Figure 5, genes knocked computationally true-essential predictions glucose <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             minimal media Feist et al. 2007 [24]. Process reactant absence indicated dark <br />                                             shading metabolite, short-term long-term border. We contrast these <br />                                             results analogous biomass reaction results (small grids Figure 5): using the <br />                                             biomass reaction useful contained completely shaded grid (right Figure 5). <br />  <br />                                             We make graded assessments metabolic network predictions. A knockout has <br />                                             historically called true-essential biomass reaction flux value some <br />                                             threshold gene experimentally necessary growth. These true-essential <br />                                             matches considered metric quality metabolic network reconstructions. <br />                                             However, provides information prediction correct relevant <br />                                             biological reason. We compared literature information flexFBA metabolites absences in <br />                                             Figure 5, indicate agreement color (associated references Table S1). The strains <br />                                             missing biomass metabolites analogous auxotrophs, these <br />                                             experimental literature confirms media supplemented metabolites <br />  <br />  <br />                                                  J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 10 <br />  <br />  <br />                                            unavailable simulation precursors. For genes experimental evidence was <br />                                            consistent silico phenotype, expected critical components to <br />                                            exchanged media, assigned `other agreement&apos; classification. The `other <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            agreement&apos; group consists membrane components, electron transport chain <br />                                            enzymes, nucleotide kinases. <br />  <br />                                            Shown pie inset Figure 5, majority true-essential predictions occur from <br />                                            biologically relevant consequences metabolic network. This agreement means our <br />                                            detailed results largely consistent accuracy classification implied `true&apos; of <br />                                            true-essential biomass reaction predictions. 14 metabolic phenotypes least <br />                                            partially inconsistent biochemical genetic evidence. These phenotypes include <br />                                            cases FBA technically correct predictions essentiality, faulty <br />                                            reasoning. One example metK knockout spermidine synthesis blocked in <br />                                            silico, preventing biomass reaction flux. The zero biomass reaction flux traditional <br />                                            FBA prediction metK essential gene, matches metK essentiality found <br />                                            experimentally. However, metK knockout lethal spermidine absence, as <br />                                            strains isolated polyamines [35]. Instead, reason metK knockout <br />                                            lethal involves production critical cofactors required methionine biosynthesis <br />                                            [36]. These cofactors used cycles, means FBA `balance&apos; fluxes <br />                                            cofactors produced. Our new approach allows resolve the <br />                                            discrepancy biochemical evidence silico phenotype, the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            previous FBA methods claimed correct prediction. The folA knockout prediction an <br />                                            analogous case metK; serA, serB, serC knockouts demonstrate opposite <br />                                            phenomena cycles possible FBA cells disrupted. A <br />                                            number incorrect predictions involve cofactor cycling, folA metK <br />                                            knockouts able produce metabolites synthesis reality prevented. In <br />                                            cases serA, serB, serC knockouts balance interconversion acetyl- <br />                                            CoA CoA perturbed, FBA apparently sensitive metabolism in <br />                                            vivo. <br />  <br />                                3.2. Expression Bursts <br />                                            Figure 6 compares results short-time FBA biomass reaction simulations the <br />                                            case enzyme copy number fluctuations single cell. Some process reactant metabolites <br />                                            required cell small amounts, enzymes produce exist the <br />                                            order tens cell. Increasingly, FBA approaches refined addition of <br />                                            enzymatic parameters bounds calculation [14], example using protein counts and <br />                                            turnover number constrain internal network fluxes [10]. For kinetic flux bounds be <br />                                            applied single cell integrated models, flexFBA required avoid biomass reaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            artifacts low copy number enzymes. <br />  <br />                                            Coenzyme A (CoA) small process requirement metabolite example Figure 6 <br />                                            simulations, total inclusion biomass acetyland succinyl-CoA six <br />                                            orders magnitude smaller process requirement ATP. An essential step CoA <br />                                            synthesis performed coaBC, observed average per <br />                                            cell, produced transcription events cell cycle [16]. We input an <br />                                            approximation coaBC expression translational bursts, informs kinetic bounds on <br />                                            CoA synthesis shown Figure 6a b, compared kinetic bounds smoothly <br />                                            increase according population average. The biomass reaction flexFBA predictions <br />                                            CoA synthesis rate similar, restricted kinetic bounds single <br />                                            cell lower smooth population average (Figure 6a b). The difference when <br />                                            CoA synthesis constrained smooth level, using biomass reaction, process <br />                                            reactant metabolite synthesis constrained (Figure 6c), unperturbed by <br />  <br />  <br />                                                 J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                      Page 11 <br />  <br />  <br />                                           CoA limits using flexFBA (Figure 6d). Biomass reaction accumulation macromolecular <br />                                           precursors lowered periods restriction. This deficit seen slightly lower <br />                                           fluxes compared smooth case coaBC enzyme limiting. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                4. Discussion Conclusions <br />                                           We constructed FBA objective able produce subsets process <br />                                           reactants, reproduces traditional biomass production wild-type network case. <br />                                           In addition, time-linked simulations allow observe transitions FBA steady <br />                                           states. Critically, methods satisfy requirement quick unsupervised operation such <br />                                           used integrated modeling applications, using off-the-shelf and <br />                                           open-source optimization packages. Furthermore, flexFBA functions robustly the <br />                                           single added adjustable parameter penalty term weighting &#206;&#179;. An additional strength that <br />                                           penalty weighting approaches question asked converges `whatever <br />                                           metabolism make&apos; to classic population <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>survival</span> assessment. <br />  <br />                                           The detailed results flexFBA tFBA silico knockout predictions offer new <br />                                           window metabolic reconstructions. Correctness incorrectness essentiality and <br />                                           metabolic phenotype predictions properties metabolic reconstruction gene <br />                                           associations. What methods make possible identify correct predictions are <br />                                           wrong reason, indicating problem metabolic reconstruction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Identification erroneous true-essential predictions important cause <br />                                           problems applications FBA. For example, metabolic engineering design <br />                                           lead misguided computational suggestions strain development efforts. <br />  <br />                                           Our short-time FBA necessary avoid artifacts apply FBA short-time scale <br />                                           single-cell level enzyme count kinetic parameters constraints. Using biomass <br />                                           reaction, models predict cell constrains distant pathways, including large fluxes <br />                                           catalyzed high copy number enzymes, match transcriptional fluctuations single <br />                                           rare protein. On second timescales, strict constraint unreasonable an <br />                                           evolutionary perspective mechanistic understanding. Furthermore, the <br />                                           coaBC example saw impact reaction limited enzyme produced in <br />                                           bursts, exist and, simulation simultaneously, result more <br />                                           dramatic limitation. Our methods introduce FBA solution avoids propagating <br />                                           low copy enzyme bounds implausibly. <br />  <br />                                           The biomass reaction previously limited application genome scale stoichiometric <br />                                           metabolic models population average steady state growth. FlexFBA tFBA <br />                                           bring relevant timescale intermediate range allow to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           represent cell physiological detail. The combined short-time FBA instrumental <br />                                           whole-cell simulations understanding heterogeneity underlies critical <br />                                           phenomena microbiology. <br />  <br />                                Supplementary Material <br />                                           Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                           We thank Derek Macklin Jonathan Karr helpful discussions methods presentation. This work was <br />                                           supported NIH Directors Pioneer Award (5DP1LM01150-05), Allen Distinguished Investigator Award, and <br />                                           Hellman Faculty Scholarship M.W.C; Stanford Bio-X Fellowship E.W.B; NSF Graduate Research <br />                                           Fellowship DGE-1147470 Gabilan Stanford Graduate Fellowship Funds M.U. <br />  <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                  Page 12 <br />  <br />  <br />                                References <br />                                           [1]. Varma A, Palsson BO. Metabolic capabilities Escherichia coli: I. synthesis biosynthetic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 precursors cofactors. J Theor Biol. 1993; 165(4):477&#226;&#8364;&#8220;502. [PubMed: 21322280] <br />                                           [2]. Mahadevan RR, Edwards JSJ, Doyle FJF. Dynamic flux balance analysis diauxic growth in <br />                                                 Escherichia coli. Biophys J. 2002; 83(3):1331&#226;&#8364;&#8220;1340. [PubMed: 12202358] <br />                                           [3]. Covert MW, Schilling CH, Famili I, Edwards JS, Goryanin II, Selkov E, Palsson BO. Metabolic <br />                                                 modeling microbial strains silico. Trends Biochem Sci. 2001; 26(3):179&#226;&#8364;&#8220;186. [PubMed: <br />                                                 11246024] <br />                                           [4]. Edwards JS, Palsson BO. Metabolic flux balance analysis silico analysis Escherichia <br />                                                 coli K-12 gene deletions. BMC Bioinformatics. 2000; 1:1. [PubMed: 11001586] <br />                                           [5]. Raman K, Chandra N. Flux balance analysis biological systems: applications challenges. <br />                                                 Briefings Bioinformatics. 2009; 10(4):435&#226;&#8364;&#8220;449. [PubMed: 19287049] <br />                                           [6]. Schuetz R, Kuepfer L, Sauer U. Systematic evaluation objective functions predicting <br />                                                 intracellular fluxes Escherichia coli. Molecular Systems Biology. 2007; 3:119&#226;&#8364;&#8220;119. [PubMed: <br />                                                 17625511] <br />                                           [7]. Segr&#195;&#168; D, Vitkup D, Church GM. Analysis optimality natural perturbed metabolic <br />                                                 networks. Proc Natl Acad Sci USA. 2002; 99(23):15112&#226;&#8364;&#8220;15117. [PubMed: 12415116] <br />                                           [8]. Birch EW, Ruggero NA, Covert MW. Determining Host Metabolic Limitations Viral <br />                                                 Replication Integrated Modeling Experimental Perturbation. PLoS Comput Biol. 2012; <br />                                                 8(10):e1002746. [PubMed: 23093930] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [9]. Gon&#195;&#167;alves E, Bucher J, Ryll A, Niklas J, Mauch K, Klamt S, Rocha M, Saez-Rodriguez J. <br />                                                 Bridging layers: integration signal transduction, regulation metabolism into <br />                                                 mathematical models. Mol. BioSyst. <br />                                           [10]. Karr JR, Sanghvi JC, Macklin DN, Gutschow MV, Jacobs JM, Bolival B, Assad-Garcia N, Glass <br />                                                 JI, Covert MW. A whole-cell computational model predicts phenotype genotype. Cell. <br />                                                 2012; 150(2):389&#226;&#8364;&#8220;401. [PubMed: 22817898] <br />                                           [11]. Feist AM, Palsson BO. The biomass objective function. Curr Opin Microbiol. 2010; 13(3):344&#226;&#8364;&#8220; <br />                                                 349. [PubMed: 20430689] <br />                                           [12]. Harcombe WR, Delaney NF, Leiby N, Klitgord N, Marx CJ. The Ability Flux Balance <br />                                                 Analysis Predict Evolution Central Metabolism Scales Initial Distance the <br />                                                 Optimum. PLoS Comput Biol. 2013; 9(6):e1003091. [PubMed: 23818838] <br />                                           [13]. Burgard AP, Maranas CD. Optimization-based framework inferring testing hypothesized <br />                                                 metabolic objective functions. Biotechnol Bioeng. 2003; 82(6):670&#226;&#8364;&#8220;677. [PubMed: 12673766] <br />                                           [14]. Reed JL. Shrinking Metabolic Solution Space Using Experimental Datasets. PLoS Comput <br />                                                 Biol. 2012; 8(8):e1002662. [PubMed: 22956899] <br />                                           [15]. Zomorrodi AR, Suthers PF, Ranganathan S, Maranas CD. Mathematical optimization <br />                                                 applications metabolic networks. Metab Eng. 2012; 14(6):672&#226;&#8364;&#8220;686. [PubMed: 23026121] <br />                                           [16]. Taniguchi Y, Choi PJ, Li GW, Chen H, Babu M, Hearn J, Emili A, Xie XS. Quantifying E. coli <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Proteome Transcriptome Single-Molecule Sensitivity Single Cells. Science. 2010; <br />                                                 329(5991):533&#226;&#8364;&#8220;538. [PubMed: 20671182] <br />                                           [17]. Lidstrom ME, Konopka MC. The role physiological heterogeneity microbial population <br />                                                 behavior. Nat Chem Biol. 2010; 6(10):705&#226;&#8364;&#8220;712. [PubMed: 20852608] <br />                                           [18]. Covert MW, Palsson B&#195;&#732;. Transcriptional regulation constraints-based metabolic models of <br />                                                 Escherichia coli. J Biol Chem. 2002; 277(31):28058&#226;&#8364;&#8220;28064. [PubMed: 12006566] <br />                                           [19]. Covert MW, Xiao N, Chen TJ, Karr JR. Integrating metabolic, transcriptional regulatory and <br />                                                 signal transduction models Escherichia coli. Bioinformatics. 2008; 24(18):2044&#226;&#8364;&#8220;2050. <br />                                                 [PubMed: 18621757] <br />                                           [20]. Jackowski S, Rock CO. Regulation coenzyme A biosynthesis. The Journal Bacteriology. <br />                                                 1981; 148(3):926&#226;&#8364;&#8220;932. <br />                                           [21]. Goldstein A, Goldstein DB, Brown BJ, Chou S-C. Amino acid starvation Escherichia coli <br />                                                 auxotroph. I. E ects protein nucleic acid synthesis cell devision. Biochim Biophys <br />                                                 Acta. 1959; 36:163&#226;&#8364;&#8220;172. [PubMed: 13850757] <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                   Page 13 <br />  <br />  <br />                                           [22]. Goss WA, Deitz WH, Cook TM. Mechanism Action Naladixic Acid Escherichia coli. <br />                                                 The Journal Bacteriology. 1964; 88:1112&#226;&#8364;&#8220;1118. <br />                                           [23]. Ohashi Y, Hirayama A, Ishikawa T, Nakamura S, Shimizu K, Ueno Y, Tomita M, Soga T. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Depiction metabolome changes histidine-starved Escherichia coli CE-TOFMS. Mol. <br />                                                 BioSyst. 2008; 4(2):135. [PubMed: 18213407] <br />                                           [24]. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, Broadbelt LJ, <br />                                                 Hatzimanikatis V, Palsson B&#195;&#732;. A genome-scale metabolic reconstruction Escherichia coli <br />                                                 K-12 MG1655 accounts 1260 ORFs thermodynamic information. Molecular Systems <br />                                                 Biology. 2007; 3:121. [PubMed: 17593909] <br />                                           [25]. Nookaew I, Jewett MC, Meechai A, Thammarongtham C, Laoteng K, Cheevadhanarak S, <br />                                                 Nielsen J, Bhumiratana S. The genome-scale metabolic model iIN800 Saccharomyces <br />                                                 cerevisiae validation: scaffold query lipid metabolism. BMC Syst Biol. 2008; 2(1):71. <br />                                                 [PubMed: 18687109] <br />                                           [26]. Heavner BD, Smallbone K, Barker B, Mendes P, Walker LP. Yeast 5 &#226;&#8364;&#8220; expanded <br />                                                 reconstruction Saccharomyces cerevisiae metabolic network. BMC Syst Biol. 2012; 6(1): <br />                                                 55. [PubMed: 22663945] <br />                                           [27]. Boyd, S.; Vandenberghe, L. Convex optimization. <br />                                           [28]. Sra, S.; Nowozin, S.; Wright, SJ. Optimization Machine Learning. MIT Press; 2012. <br />                                           [29]. Bennett BD, Kimball EH, Gao M, Osterhout R, Van Dien SJ, Rabinowitz JD. Absolute <br />                                                 metabolite concentrations implied enzyme active site occupancy Escherichia coli. Nature <br />                                                 Publishing Group. 2009; 5(8):593&#226;&#8364;&#8220;599. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [30]. Covert MW, Knight EM, Reed JL, Herrgard MJ, Palsson BO. Integrating high-throughput and <br />                                                 computational data elucidates bacterial networks. Nature. 2004; 429(6987):92&#226;&#8364;&#8220;96. [PubMed: <br />                                                 15129285] <br />                                           [31]. Andersen, MS.; Dahl, J.; Vandenberghe, L. CVXOPT: A Python package convex <br />                                                 optimization. <br />                                           [32]. Mileykovskaya E, Ryan AC, Mo X, Lin C-C, Khalaf KI, Dowhan W, Garrett TA. Phosphatidic <br />                                                 acid N-acylphosphatidylethanolamine form membrane domains Escherichia coli mutant <br />                                                 lacking cardiolipin phosphatidylglycerol. J Biol Chem. 2009; 284(5):2990&#226;&#8364;&#8220;3000. [PubMed: <br />                                                 19049984] <br />                                           [33]. Zalkin, H. Escherichia coli Salmonella. American Society Microbiology; Washington, <br />                                                 DC: 1996. Biosynthesis purine nucleotides. <br />                                           [34]. Imielinski M, Belta C, Halasz A, Rubin H. Investigating metabolite essentiality genome- <br />                                                 scale analysis Escherichia coli production capabilities. Bioinformatics. 2005; 21(9):2008&#226;&#8364;&#8220; <br />                                                 2016. [PubMed: 15671116] <br />                                           [35]. Tabor CW, Tabor H. Polyamines microorganisms. Microbiol Rev. 1985; 49(1):81. [PubMed: <br />                                                 3157043] <br />                                           [36]. Wei Y, Newman EB. Studies role metK gene product Escherichia coli K-12. <br />                                                 Molecular Microbiology. 2002; 43(6):1651&#226;&#8364;&#8220;1656. [PubMed: 11952912] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                               Page 14 <br />  <br />  <br />  <br />                                           Highlights <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The rigid Flux Balance Analysis (FBA) biomass reaction hinders whole-cell <br />                                           modeling. <br />                                           New flexible FBA produce subsets biomass reactants. <br />                                           Time-linked FBA removes reactant-to-byproduct long-time assumption. <br />                                           Our new methods avoid low-copy enzyme metabolic artifacts whole-cell <br />                                           modeling. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                       Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                         <br /> </body></html>