<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Systematic Identification Novel, Essential Host Genes <br /> Affecting Bromovirus RNA Replication <br /> Brandi L. Gancarz1, Linhui Hao1,2, Qiuling He3, Michael A. Newton3,4, Paul Ahlquist1,2* <br /> 1 Institute Molecular Virology, University Wisconsin&#226;&#8364;&#8220;Madison, Madison, Wisconsin, United States America, 2 Howard Hughes Medical Institute, University of <br /> Wisconsin-Madison, Madison, Wisconsin, United States America, 3 Department Statistics, University Wisconsin&#226;&#8364;&#8220;Madison, Madison, Wisconsin, United States of <br /> America, 4 Department Biostatistics Medical Informatics, University Wisconsin&#226;&#8364;&#8220;Madison, Madison, Wisconsin, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Positive-strand RNA virus replication involves viral proteins cellular proteins nearly replication step. Brome <br />      mosaic virus (BMV) well-established model dissecting virus-host interactions viruses whose <br />      RNA replication, gene expression encapsidation reproduced yeast Saccharomyces cerevisiae. <br />      Previously, laboratory identified ,100 non-essential host genes loss inhibited enhanced BMV replication at <br />      3-fold. However, isolation additional BMV-modulating host genes classical genetics results <br />      underscore genes essential cell growth contribute BMV RNA replication frequency greater <br />      non-essential genes. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> systematically identify novel, essential host genes affecting BMV RNA replication, we <br />      tested collection ,900 yeast strains, single essential gene promoter replaced doxycycline-repressible <br />      promoter, allowing repression gene expression adding doxycycline growth medium. Using strain array of <br />      ,81% essential yeast genes, identified 24 essential host genes depleted expression reproducibly inhibited or <br />      enhanced BMV RNA replication. Relevant host genes involved ribosome biosynthesis, cell cycle regulation protein <br />      homeostasis, cellular processes. BMV 2aPol levels significantly increased strains depleted heat <br />      shock protein (HSF1) proteasome components (PRE1 RPT6), suggesting genes affect BMV RNA replication <br />      directly indirectly modulating 2aPol localization, post-translational modification interacting partners. Investigating <br />      diverse functions newly identified essential host genes advance understanding BMV-host <br />      interactions normal cellular pathways, suggest new modes virus control. <br />  <br />   Citation: Gancarz BL, Hao L, He Q, Newton MA, Ahlquist P (2011) Systematic Identification Novel, Essential Host Genes Affecting Bromovirus RNA <br />   Replication. PLoS ONE 6(8): e23988. doi:10.1371/journal.pone.0023988 <br />   Editor: Volker Thiel, Kantonal Hospital St. Gallen, Switzerland <br />   Received May 27, 2011; Accepted July 28, 2011; Published August 22, 2011 <br />   Copyright: &#195;&#376; 2011 Gancarz et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Institutes Health (NIH) grant GM35072 (http://grants.nih.gov/grants/oer.htm.) The funders role study <br />   design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: P.A investigator Howard Hughes Medical Institute. <br />   * E-mail: ahlquist@wisc.edu <br />  <br />  <br />  <br /> Introduction                                                                         replication encapsidation recapitulated yeast <br />                                                                                      Saccharomyces cerevisiae expressing viral replication and/or <br />    Viruses survive limited genetic material interacting                 capsid proteins genomic RNA template <br /> and exploiting host factors essentially replication step                    [20,21,22]. The ability BMV duplicate nearly major <br /> [1,2,3,4,5,6,7,8,9,10,11,12,13,14]. Identifying host factors                 replication features natural plant hosts yeast, combined <br /> pathways exploited virus replication nature                      yeast genetics, advanced understanding BMV <br /> contributions interactions virus-encoded replication                        replication virus-host interactions [1,2,9]. <br /> factors represent major challenges opportunities under-                         Previously, tested deletions nearly non-essential yeast <br /> standing controlling viruses.                                                    genes (,80% yeast genome) identified 99 genes, that <br />    Positive-strand RNA viruses comprise virus                  deleted, altered BMV replication, revealing involvement <br /> genera include important human pathogens hepatitis                       novel host pathways viral replication, transcription, <br /> C virus, dengue virus, chikungunya virus, West Nile virus                        translation [9]. However, classical yeast genetics other <br /> [15]. Brome mosaic virus (BMV), member alphavirus-like                      approaches demonstrated genes essential cell growth <br /> superfamily human, plant, animal viruses, used                    make major contributions BMV RNA replication [23,24, <br /> a model study gene expression, RNA replication                         25,26]. To globally identify additional essential host factors <br /> virus-host interactions positive-strand RNA viruses [1,2,8,                       critical BMV RNA replication, assayed doxycycline (dox)- <br /> 10,16]. BMV genomic RNAs subgenomic (sg)                           repressible library ,900 yeast strains, allows <br /> mRNA. Genomic RNAs 1 2 encode multifunctional repli-                         repressing expression selected essential gene adding dox <br /> cation proteins 1a 2a polymerase (2aPol), respectively,                    growth media [27]. Using genome-wide approach, we <br /> are required RNA replication [17,18,19]. Genomic RNA3                            identified 24 essential host factors repressed expression <br /> encodes 3a movement protein, required infection spread                    reproducibly altered BMV RNA replication. These host factors <br /> plants. A sg mRNA, RNA4, encodes viral coat protein                       involved protein homeostasis, protein trafficking, and <br /> produced (2)RNA3 replication intermediate. BMV RNA                          translation, others. The results presented here, conjunction <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                1                               August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                          Essential Genes BMV RNA Replication <br />  <br />  <br /> with previously identified host factors [2,8,10,24,25], provide            48 hr post gal-induction virus expression. Strains were <br /> more complete understanding cellular pathways utilized                 grown 96-well plates luciferase assays 14 ml culture <br /> BMV. Dissecting role essential host genes virus              tubes Northern analysis. <br /> replication significantly advance understanding basic <br /> virus biology virus-host interactions. Additionally, results       RNA Analysis <br /> may lay foundation extending studies virus                  For 96-well Fluc assays, 2.5 ml cells lysed 16 Passive <br /> groups, potentially identifying common cellular pathways                Lysis Buffer (Promega, Madison, WI), 25 ml Luciferase Assay <br /> that targeted development broad-spectrum                 Substrate (Promega, Madison, WI) injected read 1 s <br /> antivirals.                                                                  1.6 s delay using VictorV (PerkinElmer, Waltham, MA). <br />                                                                              For 96-well Rluc assays, 5 ml cells lysed 16 Passive <br /> Methods                                                                      Lysis Buffer, 25 ml Renilla Luciferase Assay Substrate <br />                                                                              (Promega, Madison, WI) injected read 1 s a <br /> S. cerevisiae Strains Plasmids                                           1.6 s delay using VictorV. To allow comparison between <br />    The yeast Tet-Promoter Hughes Collection essential yeast               plates, median untreated samples median dox- <br /> strains purchased Open Biosystems (Huntsville, AL).                 treated samples calculated plate. Each untreated <br /> The tet-promoter mutant strains (designated prefix             sample normalized untreated median whereas <br /> PTET) provided haploid R1158 background (MATa                    dox-treated sample normalized dox-treated <br /> URA3::CMV-tTA MATa his3-1 leu2-0 met15-0), con-                    median. For pass 741 PTET strains, calculated <br /> structed one-step integration cytomegalovirus (CMV)              BMV-directed Rluc expression [Rlucdox-treated/Flucdox-treated] <br /> promoter-driven tTA* transactivator URA3 locus [28]. The              normalized dox-treated median [Rlucuntreated/ <br /> kanR-tetO7-TATA integrated promoter                   Flucuntreated] normalized untreated median. The dox- <br /> different essential gene strain R1158, allowing repression         treated untreated ratio ratios calculated converted <br /> essential gene expression addition dox growth                 fold change. High-throughput isolation total RNA <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> medium [27].                                                                 yeast cells performed previously described [35]. Northern <br />    pB12VG1 expresses BMV 1a 2aPol GAL1                      blotting performed previously described [36] that <br /> GAL10 promoters, respectively [9]. pB3BG29, based pB3Rluc                 2 mg RNA separated 1% (wt/vol) agarose-MOPS <br /> [9], uses truncated GAL1 promoter (GALL [29]) express                   (morpholinepropanesulfonic acid)-formaldehyde gels. RNAs <br /> RNA3 coat protein ORF replaced Renilla luciferase            detected using 32P-labeled probes specific positive- or <br /> (Rluc) ORF (from pRL-null; Promega, Madison, WI). pB3BG29                    negative-strand BMV RNA3 RNA4 previously described <br /> also expresses firefly luciferase (Fluc) ORF (from pGL3-Basic; <br />                                                                              [24]. The 18S rRNA probe derived pTRI RNA 18S <br /> Promega, Madison, WI) GAL10 promoter. To construct <br />                                                                              templates (Ambion, Austin, TX). Probes synthesized using <br /> pB3BG29, AgeI-AatII RNA3/Rluc-containing fragment of <br />                                                                              Epicenter Riboscribe probe synthesis kit (Madison, WI) with <br /> pB3Rluc [9] replaced AgeI-AatII FHV RNA1/Rluc-containing <br />                                                                              appropriate enzyme, i.e., T7 or SP6 polymerase. Northern <br /> fragment pBDL250-Ren [B. Lindenbach, unpublished]. <br />                                                                              blots imaged Typhoon 9200 instrument (Amersham <br /> pB3MS82 expresses BMV RNA3 derivative coat <br />                                                                              Biosciences, Piscataway, NJ) band intensities analyzed <br /> protein gene four-nucleotide insertion <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>point mutation</span>, <br />                                                                              ImageQuant software (Molecular Dynamics, Piscataway, <br /> abolishing expression coat protein [30]. The use pB3MS82 <br />                                                                              NJ). <br /> in study, studies [8,19,30,31,32,33], allows <br /> analysis RNA3 RNA4 levels avoiding possible <br /> effects coat protein expression RNA encapsidation.                    Protein extraction, Western blotting, total protein <br /> pB12VG1 pB3BG29 used reporter gene-based primary                 analysis <br /> screens pB12VG1 pB3MS82 used secondary                          Total protein extracted previously described [24] and <br /> validation testing Northern blotting.                                     equal volumes cell lysates separated 4&#226;&#8364;&#8220;15% <br />                                                                              CriterionTM TGXTM precast polyacrylamide gels (Bio-Rad, <br /> Yeast transformation growth                                              Hercules, CA). Proteins transferred PVDF membrane <br />    96-well yeast transformations based one-step                    expression target proteins detected following <br /> procedure [34]. The PTET essential yeast strains grown               antibodies dilutions: rabbit anti-BMV 1a 1:10,000, mouse <br /> saturation overnight 30uC 96-well plates (1 ml well). The          anti-BMV 2aPol 1:3,000, mouse anti-Pgk1p (Molecular <br /> cells pelleted, suspended 100 ml transformation mix               Probes, Carlsbad, CA) 1:10,000 using HRP-conjugated <br /> (0.18 M LiAc, pH 5.5, 36% polyethylene glycol-3350, 90 mM                    secondary antibodies (Thermo Scientific, Rockford, IL) and <br /> DTT, 0.5 mg/ml sheared salmon sperm DNA, 20 mg/ml                     Supersignal West Femto substrate (Thermo Scientific, Rockford, <br /> each plasmid) incubated 30uC 60 min. Cells               IL). <br /> then heat shocked 42uC 20 min, pelleted, re-suspended <br /> in 20 ml sterile water 10 ml plated solid media.         Statistical analysis <br /> Transformants selected complementation auxotrophic                   The tools R statistical package (version R-2.11.1) (http:// <br /> markers. 96-well plates transformed yeast re-formatted            www.r-project.org/) [37] used statistical analysis BMV <br /> contain 48 strains duplicate plate strains           RNA replication data obtained Northern blotting. Log <br /> analyzed absence dox (allowing essential gene expression)          transformation applied [RNA4/18S rRNA]dox-treated/ <br /> or presence dox (repressing essential gene expression).            [RNA4/18S rRNA]untreated ratios Northern blot data, where <br /> Strains containing BMV expression plasmids grown over-                  18S rRNA served normalization standard. One-sided t-statistics <br /> night medium containing raffinose, subcultured starting              used identify dox-treated strains RNA <br /> OD600 = 0.1 medium containing raffinose 610 mg/ml dox,                    replication statistically significantly altered compared to <br /> grown 24 hr subcultured starting OD600 = 0.1 medium              untreated strains. p-values t-statistics converted q- <br /> containing galactose 610 mg/ml dox. Cells analyzed 24 hr             values control false discovery rate [37]. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                          August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                           Essential Genes BMV RNA Replication <br />  <br />  <br />  <br /> Results                                                                     assayed 24 hr 48 hr post-virus induction (Fig. 1B). Two <br />                                                                             independent analyses entire library performed. <br /> Identification Essential Host Genes Affecting BMV RNA                       Since ,70% strains library exhibit growth defects <br /> Replication                                                                 presence dox (,20% exhibit growth defects <br />    To systematically identify novel, essential host genes affecting         absence dox, presumably result endogenous <br /> BMV RNA replication, screened dox-repressible library               promoter replacement) [27] BMV RNA replication <br /> essential yeast strains [27]. This collection contains ,900 yeast           levels non-specifically enhanced slow-growing cells, <br /> strains, single endogenous essential gene promoter              stringent growth requirements employed minimize false <br /> replaced tetracycline-repressible promoter [27]. Upon              positives. Additionally, expected strains to <br /> addition tetracycline derivative dox growth media,                exhibit immediate dox-induced growth defects and/or <br /> expression essential gene repressed, protein              transcriptional defects GAL promoter, Fluc values in <br /> depleted. This library used identify changes BMV RNA              untreated vs. dox-treated strains closely monitored. Accord- <br /> replication repression (dox-treated) relative continuous           ingly, included final data analysis potentially <br /> expression (untreated) specific essential host gene.                   specific effects BMV replication, strain required to: 1) <br />    To assess role essential genes BMV RNA replication,            double twice 0&#226;&#8364;&#8220;24 hr galactose-containing <br /> we co-transformed ,900 strains BMV expression              media; 2) double additional time 24&#226;&#8364;&#8220;48 hr <br /> plasmids pB12VG1 pB3BG29 (Fig. 1A). pB12VG1 expresses                   galactose-containing media; 3) Flucdox-treated <br /> BMV RNA replication proteins 1a 2aPol [9]. pB3BG29                      value 20% Flucuntreated value. As reference, wild <br /> expresses Rluc reporter-expressing BMV RNA3 cDNA deriva-                 type strain R1158 doubled 3 times 0&#226;&#8364;&#8220;24 hr 2 <br /> tive and, expression control, Fluc reporter gene (Fig. 1A).       times 24&#226;&#8364;&#8220;48 hr comparable Flucuntreated and <br /> DNA-dependent transcription produces initial (+)RNA3                     Flucdox-treated values. Strains did meet growth or <br /> transcript serves template 1a- 2aPol-dependent            Fluc value requirements passes excluded further <br /> RNA3 replication sgRNA4 synthesis (2)RNA3                         analysis (Tables S2 S3). The remaining strains satisfied <br /> intermediate (Fig. 1A). Because sgRNA4, encodes coat              growth Fluc requirements included analysis <br /> protein, derived (2)RNA3 intermediate, synthesis            (Table S4) showed .90% overlap pass 1 2 the <br /> depends on, serve reporter for, BMV RNA replica-               24 hr time point .80% overlap passes 48 hr <br /> tion [20]. Accordingly, pB3BG29 replaced coat protein             time point, confirming good reproducibility screen conditions <br /> ORF Rluc reporter gene, luciferase assays            (e.g. growth, dox-treatment, etc.) assay performance. <br /> be used rapid measure RNA4 production expression                   From analyzed strains (Table S4), identified 42 essential <br /> (Fig. 1A). A similar approach used primary screening         yeast genes that, dox-depleted, altered BMV-directed Rluc <br /> yeast non-essential gene deletion library [9]. Of 892 dox-              expression 6-fold passes time point <br /> repressible strains transformed, 151 did produce colonies         (Tables 1 2). A stringent 6-fold cutoff used the <br /> repeated transformation attempts (Table S1), suggesting            primary screen response observation strains in <br /> transformation process expression viral constructs                essential gene library showed luciferase assay variability <br /> significantly impacted fitness strains.                        previous screen non-essential genes. This increased <br />    The remaining 741 transformants re-formatted 96-well             variability likely fact genes essential for <br /> plates duplicates strain present         cell growth. Thus, dox-induced repression genes, the <br /> plate (Fig. 1B), allowing untreated dox-treated strains           experiment race specific effects relevant gene <br /> directly compared changes RNA replication. It important           virus replication nonspecific, general suppression cell <br /> to note library expression levels substituted        growth viability eventually occur strain. <br /> TET promoter different expression levels            Therefore, results subject variations small <br /> endogenous gene promoter [27]. Therefore, test control            changes growth conditions timing experiments. <br /> for possible changes viral RNA replication and/or gene                   Accordingly, 6-fold cutoff employed limit inclusion of <br /> expression, treated strain compared untreated               false positives. <br /> counterpart parental wild type strain. Strains              In principle, altered BMV-directed Rluc expression observed in <br /> were grown raffinose-containing selective medium lacking dox             42 candidate genes result general defects RNA4 <br /> (allowing essential gene expression) raffinose-containing selec-         translation viral RNA synthesis, selective defects in <br /> tive medium containing 10 mg/ml dox (repressing essential gene              replication, expression function Rluc reporter gene. To <br /> expression) 24 hr allow initial depletion essential       identify genes specifically affected BMV RNA synthesis and/ <br /> gene mRNA protein turnover strains treated dox.                 accumulation, secondary validation testing 42 candidate <br /> After 24 hr treatment period, strains sub-cultured           genes performed using Northern blotting analyze RNA3 <br /> galactose-containing selective medium 610 mg/ml dox induce               replication RNA4 production, described two <br /> the expression BMV components subsequent viral RNA                   sections. <br /> replication continued expression (untreated) repression <br /> (dox-treated) essential host factors. At 24 hr 48 hr post-           Dox-repression 19 essential genes facilitated BMV RNA <br /> virus induction, Rluc expression measured readout               accumulation <br /> BMV RNA3 replication sgRNA4 synthesis. Because                          Of 42 candidate genes identified reporter gene-based <br /> expected differences kinetics gene product depletion              primary screens, 23 genes repression enhanced BMV- <br /> and specific non-specific effects cell growth BMV          directed Rluc expression relative Fluc expression 6-fold <br /> RNA replication, measured Rluc 24 hr 48 hr post-virus             passes time point (Table 1). For secondary <br /> induction strain. To monitor potential adverse effects        validation tests, 42 candidate strains transformed with <br /> of dox viability yeast strains ensure GAL       1a- 2aPol-expressing plasmid pB12VG1 a <br /> promoter induction effective, GAL driven Fluc expression,               plasmid expressing RNA3 retaining BMV coat protein ORF <br /> which independent BMV RNA replication (Fig. 1A),             (Fig. 1A, pB3MS82). The levels RNA3 RNA4 replication <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        3                            August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                              Essential Genes BMV RNA Replication <br />  <br />  <br />  <br />                                                                             expresses Fluc RNA3. BMV-specific RNA-dependent RNA replication <br />                                                                             subgenomic mRNA synthesis initiated cDNA derivative of <br />                                                                             RNA3. DNA-dependent transcription produces initial (+)RNA3 <br />                                                                             transcript serves template 1a- 2aPol-dependent RNA3 <br />                                                                             replication sgRNA4 synthesis (2)RNA3 intermediate. X, the <br />                                                                             BMV coat protein gene gene replacing it, Rluc (used <br />                                                                             here); GAL1/GAL10, yeast promoters, Rz, self-cleaving ribozyme. (B) 892 <br />                                                                             yeast strains, single essential gene promoter replaced a <br />                                                                             doxycycline (dox)-repressible promoter, transformed BMV <br />                                                                             expression plasmids. White wells indicate strains did transform. <br />                                                                             Transformants re-formatted 96-well plates duplicates of <br />                                                                             strain present plate, allowing untreated dox- <br />                                                                             treated strains directly compared. Strains grown raffinose- <br />                                                                             containing selective medium lacking dox (allowing essential gene <br />                                                                             expression) containing 10 mg/ml dox (repressing essential gene <br />                                                                             expression) 24 hr allow initial depletion essential gene <br />                                                                             mRNA protein turnover dox-treated strains. After 24 hr <br />                                                                             treatment, strains sub-cultured galactose-containing selective <br />                                                                             medium 610 mg/ml dox induce expression BMV components and <br />                                                                             subsequent viral RNA replication. Viral RNA replication quantitated <br />                                                                             chemiluminescent Renilla luciferase assay 24 hr 48 hr <br />                                                                             post-virus induction. Cell viability promoter launching were <br />                                                                             monitored chemiluminescent firefly luciferase assay 24 hr <br />                                                                             48 hr post-virus induction. Two independent analyses library <br />                                                                             performed. <br />                                                                             doi:10.1371/journal.pone.0023988.g001 <br />  <br />  <br />  <br />                                                                              Table 1. Genes repression associated $6- <br />                                                                              fold enhanced BMV-directed Rluc expression primary <br />                                                                              screen passes. <br />  <br />  <br />                                                                                                                         Fold increase Rluc expression <br />  <br />                                                                              ORF                    Gene                Pass 1             Pass 2 <br />  <br />                                                                              YLR359W                ADE13               46                 57 <br />                                                                              YBR070C                ALG14               11                 14 <br />                                                                              YDL132W                CDC53               8.4                6.3 <br />                                                                              YOR204W                DED1                180                140 <br />                                                                              YKL078W                DHR2                8.3                8.0 <br />                                                                              YLR129W                DIP2                440                160 <br />                                                                              YMR128W                ECM16               12                 7.4 <br />                                                                              YLR274W                MCM5                33                 19 <br />                                                                              YGR103W                NOP7                46                 78 <br />                                                                              YGR119C                NUP57               30                 19 <br />                                                                              YOR122C                PFY1                7.9                7.5 <br />                                                                              YLR196W                PWP1                11                 28 <br />                                                                              YOL094C                RFC4                21                 14 <br />                                                                              YNL207W                RIO2a               16                 12 <br />                                                                              YGL044C                RNA15               41                 12 <br />                                                                              YOR340C                RPA43               17                 6.3 <br />                                                                              YKR008W                RSC4                250                310 <br />                                                                              YPL124W                SPC29               13                 10 <br />                                                                              YGR116W                SPT6                45                 17 <br />                                                                              YKL018W                SWD2                48                 9.1 <br />                                                                              YDR324C                UTP4                12                 11 <br />                                                                              YJL069C                UTP18               11                 50 <br />                                                                              YGR251W                N/Ab                33                 150 <br />  <br />                                                                              a <br />                                                                               RIO2 identified 24 hr time point genes identified <br /> Figure 1. Yeast genetic screen used identify essential host                48 hr time point. <br /> factors affecting BMV RNA replication. (A) BMV expression                    b <br />                                                                               ORF annotated Saccharomyces Genome Database. <br /> plasmids. pB12VG1 expresses replication factors 1a 2aPol. pB3BG29        doi:10.1371/journal.pone.0023988.t001 <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        4                              August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                                    Essential Genes BMV RNA Replication <br />  <br />  <br />  <br />  Table 2. Genes repression associated $6-                              products measured Northern blotting. In particular, <br />  fold inhibited BMV-directed Rluc expression primary                          used level RNA4 relative 18S rRNA primary <br />  screen passes.                                                                       measure dox-induced change BMV RNA-dependent <br />                                                                                       RNA synthesis, ratio [RNA4/18S rRNA]dox-treated/ <br />                                                                                       [RNA4/18S rRNA]untreated. RNA4 used primary <br />                                            Fold decrease Rluc expression           measure BMV RNA synthesis since, unlike RNA3, RNA4 is <br />                                                                                       produced viral RNA-dependent RNA synthesis not <br />  ORF                   Gene                Pass 1             Pass 2 <br />                                                                                       DNA-dependent transcription (Fig. 1A). Moreover, RNA4 <br />  YKL112W               ABF1                15                 9.1                     level parameter closely related Rluc expression <br />  YER168C               CCA1                7.7                7.6                     measured primary screen (Fig. 1A). <br />  YFR028C               CDC14a              12; 9.2            6.0; 7.8                   A statistically significant increase BMV (+)RNA4 accumulation <br />                                                                                       relative 18S rRNA confirmed 19 23 genes (,83% <br />  YNR038W               DBP6                26                 9.9 <br />                                                                                       confirmation) commonly applied false discovery rate 5% <br />  YPL266W               DIM1                36                 70 <br />                                                                                       (Fig. 2 Table 3). False discovery rate analysis robust <br />  YDR141C               DOP1                6.2                46                      statistical method controls multiple test variables by <br />  YJR017C               ESS1                22                 22                      calculating adjusted, stringent p-value, termed q-value <br />  YOL133W               HRT1                7.2                6.2                     [37]. (+)RNA4 levels enhanced 1.5- 8-fold 19 <br />  YGL073W               HSF1      b <br />                                            37                 17                      confirmed strains (Fig. 2 Table 3). Similarly, levels (2)RNA3, <br />                                                                                       replication intermediate serves template sgRNA4, <br />  YGL018C               JAC1                15                 45 <br />                                                                                       increased 1.4- 5-fold strains (Table S5). <br />  YAL033W               POP5                17                 12 <br />                                                                                          The 19 confirmed dox-repressed genes enhanced BMV <br />  YER012W               PRE1a               21; 8.4            13; 19                  RNA accumulation encode proteins functions varied cellular <br />  YML046W               PRP39               9.4                49                      processes, including ribosome biosynthesis (DHR2, ECM16, NOP7, <br />  YGL048C               RPT6                15                 52                      PWP1, RIO2, RPA43, UTP4, UTP18, YGR251w), cell cycle/ <br />  YKL125W               RRN3                13                 76                      DNA maintenance (ADE13 SPC29), mRNA metabolism <br />                                                                                       (RNA15, SPT6, SWD2), protein homeostasis (PFY1), translation <br />  YGR245C               SDA1                22                 14 <br />                                                                                       (DED1), trafficking (NUP57) lipid synthesis (ALG14) (Table 4). <br />  YDR472W               TRS31               7.5                22 <br />                                                                                       Possible relations functions viral replication are <br />                              c <br />  YDR327W               N/A                 460                160                     considered Discussion. <br />  YOR262W               N/Ab,         c <br />                                            9.8                13 <br />  <br />                                                                                     Dox-repression essential genes inhibited BMV RNA <br />    CDC14 PRE1 identified 24 hr 48 hr time points and <br />    data time points listed &#226;&#8364;&#732;&#226;&#8364;&#732;24 hr; 48 hr&#226;&#8364;&#8482;&#226;&#8364;&#8482;.                           accumulation <br />  b <br />    HSF1 YOR262W identified 24 hr time point genes          The primary screens additionally identified 19 essential host <br />    identified 48 hr time point.                                           genes inhibited BMV-directed Rluc expression 6-fold <br />  c <br />   ORF annotated Saccharomyces Genome Database. <br />  doi:10.1371/journal.pone.0023988.t002                                                passes time point (Table 2). Northern blotting <br />                                                                                       confirmed statistically significant decreases BMV (+)RNA4 <br />                                                                                       accumulation 5 19 genes (,26% confirmation) false <br />  <br />  <br />  <br />  <br /> Figure 2. Dox-induced repression 19 essential yeast genes enhances BMV RNA replication. Total RNA extracts obtained from <br /> wild type R1158 untreated dox-treated (10 mg/ml) essential yeast strains expressing BMV 1a, 2aPol RNA3. Accumulation positive- and <br /> negative-strand RNA3 subgenomic RNA4 detected Northern blotting using probes specific BMV RNA3 RNA4. Equal loading of <br /> total RNA verified probing 18S rRNA. Values represent mean independent experiments. <br /> doi:10.1371/journal.pone.0023988.g002 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 5                           August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                         Essential Genes BMV RNA Replication <br />  <br />  <br />  <br />  Table 3. Essential genes repression confirmed                altered viral RNA replication phenotypes depleting the <br />  enhance BMV RNA accumulation secondary validation                      products implicated genes. <br />  testing. <br />                                                                            Discussion <br />                                                                               We employed high-throughput, systematic analysis 741 <br />                                 (+)RNA4 (average <br />  ORF              Gene          % untreated)            q-value            dox-repressible essential yeast strains identify 24 novel, essential <br />                                                                            host factors alter BMV RNA replication. Our previous <br />  YJL069C          UTP18         8336312                 0.011              systematic analysis ,4,500 non-essential yeast deletion strains <br />  YNL207W          RIO2          6356190                 0.003              identified 99 genes inhibited enhanced BMV RNA <br />  YKL018W          SWD2          3896153                 0.006              replication [9]. Collectively, analyzed ,93% yeast <br />  YOR204W          DED1          282661                  0.003              genes (,5,800), 123 host genes identified date that <br />  YPL124W          SPC29         261693                  0.006 <br />                                                                            affect BMV RNA replication represent 2.3% yeast genome. <br />                                                                            As observed tomato bushy stunt virus (TBSV) [38], another <br />  YLR359W          ADE13         2586126                 0.034 <br />                                                                            positive strand RNA virus, BMV replication affected, directly or <br />  YOR122C          PFY1          232646                  0.009              indirectly, essential genes higher frequency (3.2%) than <br />  YOR340C          RPA43         227676                  0.010              non-essential genes (2.2%). <br />  YDL132W          CDC53         222613                  3.24E-04              For multiple reasons, studies date likely underestimate the <br />  YGR119C          NUP57         194619                  0.006              number host genes contribute BMV RNA replication. <br />                                                                            ,60% non-essential yeast genes genetically redundant, <br />  YMR128W          ECM16         190641                  0.011 <br />                                                                            meaning functions gene deletions partially <br />  YGR251W          N/Aa          189645                  0.014 <br />                                                                            compensated genes [39,40]. Additionally, although <br />  YDR324C          UTP4          180649                  0.003              dox-repressible library powerful tool analyzing the <br />  YLR196W          PWP1          165641                  0.011              effects essential genes virus replication, 70% the <br />  YKL078W          DHR2          161665                  0.036              strains collection exhibit growth defects presence of <br />  YGR103W          NOP7          155621                  0.011              dox 14% exhibit growth defects absence dox [27]. <br />                                                                            Moreover, expressing viral components strains <br />  YBR070C          ALG14         153620                  0.006 <br />                                                                            resulted poor growth, interfering meaningful analysis <br />  YGL044C          RNA15         153616                  0.003              strains. Finally, screens non-essential genes <br />  YGR116W          SPT6          152612                  0.003              [9] essential genes (here), virus dependence host <br />                                                                          functions likely masked continuous expression BMV <br />   ORF annotated Saccharomyces Genome Database. <br />  doi:10.1371/journal.pone.0023988.t003                                     replication proteins RNA templates, compared natural <br />                                                                            infections resulting single viral RNA template. This is <br />                                                                            analogous studies high multiplicity infection <br />                                                                            overcomes antiviral resistance cell lines [39,41]. The <br /> discovery rate 5% (Fig. 3 Table 5). (+)RNA4 levels             lower confirmation rate 19 candidate genes inhibited <br /> inhibited ,1.5- 8-fold strains (Fig. 3 Table 5). In        BMV-directed Rluc expression primary <br /> addition severe inhibition (+)RNA4 accumulation               screen candidates included false positives arising the <br /> PTET-HSF1, PTET-PRE1, PTET-RPT6, (2)RNA3 inhibited                 tendency essential gene depletion produce non-specific <br /> 5.5-, 1.8-, 5.5-fold strains, respectively (Fig. 3        inhibitory effects greater variability results, noted <br /> Table S6).                                                                 Results section. Additionally, translation protein <br />    Interestingly, genes inhibited BMV RNA replication        expression basis primary screen luciferase assays, <br /> (ESS1, HSF1, JAC1, PRE1, RPT6) perform cellular functions              secondary confirmation Northern blotting analyzes <br /> that, various ways, contribute modulating host protein levels        RNA levels normalized 18S rRNA. Subtle differences (e.g., <br /> (Table 4).                                                                 growth conditions) controlled Fluc primary screen <br />                                                                            18S rRNA Northern analysis contribute the <br /> BMV 2aPol protein levels affected dox-                         lower confirmation rate observed. Despite limitations, <br /> repressed strains                                                          study identified 24 novel, essential host genes various <br />    One possible reason altered RNA replication deregulation         cellular pathways potentially diverse roles BMV RNA <br /> of viral protein accumulation. To test this, accumulation BMV           replication. <br /> RNA replication proteins 1a 2aPol assayed Western                 Enhanced BMV RNA replication repression an <br /> blotting. Addition dox growth medium detectable               essential gene suggests that, present, host factor con- <br /> effect BMV 1a 2aPol accumulation wild type strain            tributes inhibitory response cellular process/pathway <br /> (Figs. 4 5). For 20 24 confirmed hits, BMV 1a levels dox-        competes virus. For example, 19 genes whose <br /> treated strains comparable untreated sample (Figs. 4         repression stimulated BMV RNA replication, 9 genes (DHR2, <br /> and 5). However, PTET-ADE13, PTET-DED1, PTET-PRE1                   ECM16, NOP7, PWP1, RIO2, RPA43, UTP4, UTP18, YGR251W) <br /> PTET-RPT6 dox-treated cells detectable increase 1a          functionally associated ribosome biosynthesis (Table 4). <br /> compared untreated strains (Figs. 4 5). Moreover, BMV           All genes perform participate ATP-dependent <br /> 2aPol levels significantly increased PTET-HSF1, PTET-PRE1          processes depleting products implicated genes may <br /> and PTET-RPT6 dox-treated strains (Fig. 5). In PTET-JAC1 dox-              result increased pool energy and/or nucleotides available <br /> treated cells, 2aPol levels elevated absence dox,       virus (Fig. 2, Tables 3 4). Alternatively, depleting these <br /> reduced near wild type levels presence dox (Fig. 5).          genes alter competition viral cellular <br /> These results suggest that, potential exception HSF1,          translation. UTP4 UTP18 function rRNA processing, a <br /> PRE1 RPT6, viral protein regulation unlikely cause        cellular pathway enhances TBSV replication (as shown the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      6                            August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                                         Essential Genes BMV RNA Replication <br />  <br />  <br />  <br />  Table 4. Annotated functions confirmed essential host genes effecting BMV RNA accumulation. <br />  <br />  <br />  Genea                        Gene Descriptiona,b <br />  <br />  Protein Homeostasis <br />  ESS1                         Peptidylprolyl-cis/trans-isomerase; regulates phosphorylation RNA polymerase II large subunit <br />  HSF1                         Trimeric heat shock transcription factor <br />  JAC1                         Hsp40/DnaJ family J-protein functions Hsp70 Fe-S cluster biogenesis mitochondria <br />  PRE1                         Beta 4 subunit 20S core 26S proteasome <br />  RPT6                         One ATPases 19S regulatory particle 26S proteasome <br />  PFY1                         Binds profilin, actin phosphatidylinositol 4,5-bisphosphate; cytoskeleton organization <br />  Translation <br />  DED1                         ATP-dependent DEAD-box RNA helicase, required translation initiation yeast mRNAs <br />  Trafficking <br />  NUP57                        Nucleoporin essential trafficking nucleic acids, proteins, RNA nuclear pore complex <br />  Lipid synthesis <br />  ALG14                        Component UDP-GlcNAc transferase required dolichyl-linked oligosaccharide synthesis <br />  Ribosome Biosynthesis <br />  DHR2                         DEAH-box ATP-dependent RNA helicase, required 18S rRNA synthesis <br />  ECM16                        DEAH-box ATP-dependent RNA helicase specific U3 snoRNP, required 18S rRNA synthesis <br />  NOP7                         Required large ribosomal subunit maturation; required exit G0 <br />  PWP1                         Protein WD-40 repeats involved rRNA processing <br />  RIO2                         Serine kinase involved processing 20S pre-rRNA mature 18S rRNA <br />  RPA43                        RNA polymerase I subunit A43 <br />  UTP4                         Subunit U3-contaning complexes involved production 18S rRNA <br />  UTP18                        Possible U3 snoRNP protein involved maturation pre-18S rRNA <br />  YGR251w                      Required 18S rRNA maturation <br />  mRNA Metabolism <br />  RNA15                        Component involved cleavage polyadenylation mRNA 39 ends <br />  SPT6                         Transcription elongation factor required maintenance chromatin structure transcription <br />  SWD2                         Required methylation histone H3 RNA polymerase II transcription termination <br />  Cell Cycle/DNA Maintenance <br />  ADE13                        Adenylosuccinate lyase involved purine nucleotide synthesis <br />  CDC53                        A scaffolding subunit (cullin) multiple E3 ubiquitin-ligase complexes; regulates G1-S cell cycle progression <br />  SPC29                        Inner plaque spindle pole body (SPB) component; required SPB duplication <br />  <br />  a <br />   Bold font indicates genes repression inhibits viral RNA accumulation; non-bold font indicates genes repression enhances viral RNA accumulation. <br />  b <br />   Based Saccharomyces Genome Database http://www.yeastgenome.org/. <br />  doi:10.1371/journal.pone.0023988.t004 <br />  <br />  <br /> effects UTP9 UTP15) [38], opposite effect                       virus time accumulate RNA replication products <br /> BMV RNA replication.                                                                  prior cell division (unpublished data). To avoid false <br />    Depleting genes involved processing cellular mRNA 39 ends                       positives, excluded analysis strains dox- <br /> (RNA15), regulating transcription (SPT6), modulating cellular gene                    treatment significantly slowed cell division (see Results above). <br /> expression (SWD2), controlling nucleocytoplasmic trafficking                      Thus, SPC29 PFY1 encode proteins potentially <br /> (NUP57) increase availability ribosomes and/or alter                       related cell division (respectively spindle pole body protein and <br /> the levels specific proteins, preferentially stimulating BMV RNA                   actin-binding protein) doubling times untreated dox- <br /> replication (Fig. 2, Tables 3 4) disrupting cellular pathways                  treated PTET-SPC29 PTET-PFY1 cells comparable and <br /> that compete BMV RNA translation replication                            likely account increased levels BMV RNA4 <br /> under wild type conditions. Thus, experimental depletion                     observed (Fig. 2. Table 3). Alternatively, disrupting cell <br /> genes ways analogous global shutoff host                         cycle PTET-SPC29- and/or PTET-PFY1-repressed cells alter <br /> mRNA pathways, employed mammalian viruses                             cell cycle signal transduction pathways localization of <br /> diverse mechanisms [42,43,44]. Additional studies necessary                    cellular factors normally inhibit viral RNA replication. <br /> define role mRNA metabolism genes BMV RNA                                Mutating general translation initiation factor DED1 severely <br /> replication.                                                                          inhibits translation BMV 2aPol BMV genomic RNA2 a <br />    In previous studies, observed significantly                           fashion dependent specific sequences RNA2 59 <br /> disrupting cell cycle extended doubling times non-                        noncoding region (NCR) [32]. Because RNA2 59 NCR was <br /> specifically increase BMV RNA levels cell, apparently                     present BMV expression constructs used study, <br />  <br />  <br />          PLoS ONE | www.plosone.org                                               7                               August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                           Essential Genes BMV RNA Replication <br />  <br />  <br />                                                                              indirectly facilitates viral RNA synthesis accumulation. <br />                                                                              Interestingly, genes depletion inhibited BMV <br />                                                                              RNA replication (ESS1, HSF1, JAC1, PRE1, RPT6) varied <br />                                                                              roles protein stability activation. For example, small but <br />                                                                              reproducible (,20%) inhibition BMV RNA replication resulted <br />                                                                              dox-repression PTET-ESS1, peptidyl-prolyl cis-trans <br />                                                                              isomerase (PPI) (Fig. 3 Table 5). Changes isomerase activity <br />                                                                              alter structure, stability, intracellular localization of <br />                                                                              client proteins [45], deleting mutating PPIs variable <br />                                                                              effects positive-strand RNA viruses [46,47,48,49]. For example, <br />                                                                              knockdown cyclophilin A loss PPIase activity severely <br />                                                                              inhibits hepatitis C virus replication [46,50,51]. Conversely, <br />                                                                              siRNA-mediated knockdown cyclophilin G mRNA stimulates <br />                                                                              hepatitis c virus replication [49]. Collectively, findings <br />                                                                              suggest multi-faceted, complex roles PPIs positive-strand <br />                                                                              RNA virus replication. <br />                                                                                 JAC1, member Hsp40/DnaJ family proteins, encodes <br />                                                                              specialized J-protein co-chaperone assists Hsp70 iron- <br />                                                                              sulfur (Fe-S) cluster biogenesis [52,53]. Fe-S clusters the <br />                                                                              versatile protein co-factors cell participate in <br />                                                                              electron transfer, ribosome biogenesis, regulating gene expression <br />                                                                              enzyme activity, nucleotide metabolism [54,55,56]. The <br />                                                                              inhibition BMV replication dox-repressing PTET-JAC1 <br />                                                                              (Fig. 3 Table 5) result negatively affecting the <br />                                                                              activation and/or function cellular viral factors <br />                                                                              required BMV RNA replication. Previously, showed that <br /> Figure 3. Dox-induced repression essential yeast genes <br />                                                                              YDJ1, J-protein co-chaperone Hsp70 Hsp90, is <br /> inhibits BMV RNA replication. Total RNA extracts obtained <br /> from wild type R1158 untreated dox-treated (10 mg/ml)                required activate BMV RNA replication complex, likely <br /> essential yeast strains expressing BMV 1a, 2aPol RNA3. Accumula-         modulating BMV 2aPol folding assembly the <br /> tion positive- negative-strand RNA3 subgenomic RNA4           complex [26]. Dox-repression heat shock protein, <br /> detected Northern blotting using probes specific BMV RNA3         PTET-HSF1, inhibited BMV RNA replication 84% (Fig. 3 and <br /> RNA4. Equal loading total RNA verified probing 18S rRNA.       Table 5). Thus, data suggest BMV utilizes multiple <br /> Values represent mean independent experiments. <br />                                                                              members heat shock protein family facilitate RNA <br /> doi:10.1371/journal.pone.0023988.g003 <br />                                                                              replication. <br />                                                                                 Dox-repressing PTET-PRE1, 20S proteasome core component, <br /> we did expect detectable change BMV RNA replication              inhibited BMV RNA replication 70% (Fig. 3 Table 5). <br /> the dox-repressed PTET-DED1 strain. However, observed                  Similarly, repressing PTET-RPT6, ATPases 19S <br /> increase RNA replication (Fig. 2 Table 3), suggesting            regulatory particle 26S proteasome, resulted ,90% <br /> dox-repression PTET-DED1 global depletion gene                 reduction BMV RNA replication (Fig. 3 Table 5). <br /> products alternative effect(s) viral RNA replication          Additionally, BMV 2aPol accumulation increased significantly in <br /> compared previously analyzed ded1-18 point mutant [32].               PTET-PRE1 PTET-RPT6 dox-repressed cells (Fig. 5). <br /> For example, screen conditions, repressing PTET-DED1               These results consistent previous findings multiple <br /> may alter production viral cellular proteins ways          non-essential ubiquitin-proteasome components contribute <br /> favor increased viral RNA accumulation. Further studies                  BMV RNA replication cells lacking PRE9, only <br /> necessary define potential additional role(s) DED1            non-essential 20S proteasome component, exhibit substan- <br /> BMV RNA replication.                                                         tial increase BMV 2aPol levels [9]. Our prior data that <br />    Reduced BMV RNA replication repression essential               having substantial excess 2aPol shifts replication compartments <br /> gene suggests that, present, host factor directly                small spherular compartments double membrane layers, <br />                                                                              does inhibit viral RNA replication [57]. Thus, increase <br />                                                                              2aPol accumulation dox-repressed PTET-PRE1 PTET- <br />  Table 5. Essential genes repression confirmed to <br />                                                                              RPT6 (Fig. 5) likely cause decreased RNA replication. <br />  inhibit BMV RNA accumulation secondary validation <br />                                                                              However, prior results exclude possibility PRE1 <br />  testing. <br />                                                                              RPT6 depletion affect BMV RNA replication by <br />                                                                              directly indirectly modulating 2aPol localization, post-transla- <br />                                   (+)RNA4 (average <br />                                                                              tional modification interacting partners. The 26S proteasome <br />  ORF               Gene           % untreated)           q-value             localizes predominantly nuclear envelope-ER network [58], <br />                                                                              site BMV RNA replication [16,59], numerous viruses <br />  YGL048C           RPT6           1268                   0.024 <br />                                                                              utilize ubiquitin-proteasome facilitate infection or <br />  YGL073W           HSF1           1664                   0.004               replication [60,61,62,63,64,65]. Studies define specific <br />  YER012W           PRE1           30613                  0.024               role(s) ubiquitin-proteasome BMV RNA <br />  YGL018C           JAC1           57613                  0.029               replication ongoing. <br />  YJR017C           ESS1           8165                   0.024 <br />                                                                                 With exception NUP57, RNA15, SPC29 YGR251w, <br />                                                                              essential genes identified study recognized <br />  doi:10.1371/journal.pone.0023988.t005                                       orthologs Arabidopsis thaliana (http://www.arabidopsis.org/ and <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        8                           August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                         Essential Genes BMV RNA Replication <br />  <br />  <br />  <br />  <br /> Figure 4. BMV 1a 2aPol levels dox-repressed essential yeast genes enhanced BMV RNA replication. Accumulation BMV <br /> 1a 2aPol wild type R1158 untreated dox-treated (10 mg/ml) essential yeast strains measured Western blot analysis. Total <br /> proteins extracted equal numbers yeast cells analyzed SDS/PAGE. Equal loading total protein verified measuring <br /> Pgk1p levels. <br /> doi:10.1371/journal.pone.0023988.g004 <br />  <br /> http://ppod.princeton.edu/), presenting possibility similar       HSF1 significantly inhibits BMV RNA replication. Arabidopsis <br /> genes function BMV RNA replication natural plant           encodes 18 Hsp70 family members Hsp70s shown <br /> hosts. For example, shown dox-repression              affect replication (+)RNA viruses plants, <br /> Hsp70 cofactor JAC1 heat shock protein transcription factor             including TBSV turnip mosaic virus [66,67]. <br />                                                                               Similarly, PRE1 (PBD1 PBD2 Arabidopsis) RPT6 <br />                                                                            (RPT6a RPT6b Arabidopsis) essential components the <br />                                                                            highly conserved 26S proteasome recent results our <br />                                                                            laboratory BMV RNA replication yeast plant <br />                                                                            cells depends critically ubiquitin-proteasome pathway (B. <br />                                                                            Gancarz P. Ahlquist, unpublished results) [9,68]. <br />                                                                               In summary, high-throughput analysis essential yeast <br />                                                                            genes identified diverse set host factors affect BMV RNA <br />                                                                            replication significantly expanded knowledge cellular <br />                                                                            pathways utilized BMV. Additional studies yeast in <br />                                                                            BMV&#226;&#8364;&#8482;s natural plant hosts reveal host factors <br /> Figure 5. BMV 1a 2aPol levels dox-repressed essential              affect virus provide new insights host cell function and <br /> yeast genes inhibited BMV RNA replication. Accumulation               virus-host interactions. Although targeting essential genes <br /> of BMV 1a 2aPol wild type R1158 untreated dox-treated       result deleterious effects cells patients, focusing the <br /> (10 mg/ml) essential yeast strains measured Western blot <br />                                                                            relevant cellular pathways, individual genes, may <br /> analysis. Total proteins extracted equal numbers yeast <br /> cells analyzed SDS/PAGE. Equal loading total protein         overcome issues. For example, essential genes with <br /> verified measuring Pgk1p levels.                                        substantial inhibitory effect BMV RNA replica- <br /> doi:10.1371/journal.pone.0023988.g005                                      tion, PRE1 RPT6, proteasome components, other <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       9                            August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                                                 Essential Genes BMV RNA Replication <br />  <br />  <br /> proteasome components essential (e.g. PRE9, identified                            Table S5 Negative-strand RNA3 levels essential <br /> previous screen). Proteasome inhibitors antiviral activity                              yeast genes confirmed enhance BMV RNA accumula- <br /> against multiple diverse viruses (including herpes simplex virus,                            tion secondary validation testing. <br /> hepatitis B virus, HIV, others),                                 (XLSX) <br /> multiple clinical trials, approved use patients <br />                                                                                              Table S6 Negative-strand RNA3 levels essential <br /> for purposes. Many genes identified function in <br /> pathways utilized viruses present potential                            yeast genes confirmed inhibit BMV RNA accumula- <br /> cellular targets developing broad-spectrum antivirals.                                   tion secondary validation testing. <br />                                                                                              (XLSX) <br /> Supporting Information <br />                                                                                              Acknowledgments <br /> Table S1   Dox-repressible essential yeast strains that <br />                                                                                              We thank Billy Dye James Bruce technical assistance, James Bruce <br /> did transform BMV expression plasmids after <br />                                                                                              critical reading manuscript, members lab for <br /> repeated attempts.                                                                           helpful discussions experiments. We thank Leanne Olds <br /> (XLSX)                                                                                       assistance illustrations. P.A. investigator Howard <br /> Table S2 Number dox-repressible essential yeast                                           Hughes Medical Institute. <br /> strains excluded pass time point to <br /> growth Fluc values.                                                                       Author Contributions <br /> (XLSX)                                                                                       Conceived designed experiments: BLG LH PA. Performed the <br /> Table S3 Dox-repressible essential yeast strains ex-                                         experiments: BLG LH. Analyzed data: BLG LH QH MAN PA. <br /> cluded reporter gene-based primary screen anal-                                         Contributed reagents/materials/analysis tools: BLG LH QH MAN PA. <br />                                                                                              Wrote paper: BLG PA. <br /> ysis growth and/or Fluc value. <br /> (XLSX) <br /> Table S4 Reporter gene-based primary screen data for <br /> dox-repressible essential yeast genes included final <br /> data analysis. <br /> (XLSX) <br />  <br /> References <br />  1. Ahlquist P (2006) Parallels positive-strand RNA viruses, reverse-                  16. Schwartz M, Chen J, Janda M, Sullivan M, den Boon J, et al. (2002) A positive- <br />     transcribing viruses double-stranded RNA viruses. Nat Rev Microbiol 4:                   strand RNA virus replication complex parallels form function retrovirus <br />     371&#226;&#8364;&#8220;382.                                                                                     capsids. Mol Cell 9: 505&#226;&#8364;&#8220;514. <br />  2. Ahlquist P, Noueiry AO, Lee WM, Kushner DB, Dye BT (2003) Host factors                17. Ahola T, den Boon JA, Ahlquist P (2000) Helicase capping enzyme active <br />     positive-strand RNA virus genome replication. J Virol 77: 8181&#226;&#8364;&#8220;8186.                         site mutations brome mosaic virus protein 1a cause defects template <br />  3. Belov GA, Altan-Bonnet N, Kovtunovych G, Jackson CL, Lippincott-Schwartz J,                  recruitment, negative-strand RNA synthesis, viral RNA capping. J Virol 74: <br />     et al. (2007) Hijacking components cellular secretory pathway                     8803&#226;&#8364;&#8220;8811. <br />     replication poliovirus RNA. J Virol 81: 558&#226;&#8364;&#8220;567.                                      18. Chen J, Ahlquist P (2000) Brome mosaic virus polymerase-like protein 2a is <br />  4. Belov GA, Ehrenfeld E (2007) Involvement cellular membrane traffic proteins               directed endoplasmic reticulum helicase-like viral protein 1a. J Virol <br />     poliovirus replication. Cell Cycle 6: 36&#226;&#8364;&#8220;38.                                              74: 4310&#226;&#8364;&#8220;4318. <br />  5. Belov GA, Habbersett C, Franco D, Ehrenfeld E (2007) Activation cellular Arf          19. Liu L, Westler WM, den Boon JA, Wang X, Diaz A, et al. (2009) An <br />     GTPases poliovirus protein 3CD correlates virus replication. J Virol 81:             amphipathic alpha-helix controls multiple roles brome mosaic virus protein 1a <br />     9259&#226;&#8364;&#8220;9267.                                                                                   RNA replication complex assembly function. PLoS Pathog 5: e1000351. <br />  6. Bode JG, Brenndorfer ED, Karthe J, Haussinger D (2009) Interplay                 20. Ishikawa M, Janda M, Krol MA, Ahlquist P (1997) In vivo DNA expression of <br />     host cell hepatitis C virus regulating viral replication. Biol Chem 390:              functional brome mosaic virus RNA replicons Saccharomyces cerevisiae. <br />     1013&#226;&#8364;&#8220;1032.                                                                                   J Virol 71: 7781&#226;&#8364;&#8220;7790. <br />  7. Bruce JW, Ahlquist P, Young JA (2008) The host cell sulfonation pathway                  21. Janda M, Ahlquist P (1993) RNA-dependent replication, transcription, and <br />     contributes retroviral infection step coincident provirus                       persistence brome mosaic virus RNA replicons S. cerevisiae. Cell 72: <br />     establishment. PLoS Pathog 4: e1000207.                                                      961&#226;&#8364;&#8220;970. <br />  8. Diaz A, Wang X, Ahlquist P (2010) Membrane-shaping host reticulon proteins               22. Krol MA, Olson NH, Tate J, Johnson JE, Baker TS, et al. (1999) RNA- <br />     play crucial roles viral RNA replication compartment formation function.              controlled polymorphism vivo assembly 180-subunit 120-subunit <br />     Proc Natl Acad Sci U S A 107: 16291&#226;&#8364;&#8220;16296.                                                   virions single capsid protein. Proc Natl Acad Sci U S A 96: 13650&#226;&#8364;&#8220;13655. <br />  9. Kushner DB, Lindenbach BD, Grdzelishvili VZ, Noueiry AO, Paul SM, et al.                 23. Lee WM, Ahlquist P (2003) Membrane synthesis, specific lipid requirements, <br />     (2003) Systematic, genome-wide identification host genes affecting replication            localized lipid composition changes associated positive-strand RNA <br />     positive-strand RNA virus. Proc Natl Acad Sci U S A 100: 15764&#226;&#8364;&#8220;15769.                   virus RNA replication protein. J Virol 77: 12819&#226;&#8364;&#8220;12828. <br /> 10. Mas A, Alves-Rodrigues I, Noueiry A, Ahlquist P, Diez J (2006) Host                      24. Lee WM, Ishikawa M, Ahlquist P (2001) <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>Mutation</span> host delta9 fatty acid <br />     deadenylation-dependent mRNA decapping factors required key step                   desaturase inhibits brome mosaic virus RNA replication template <br />     brome mosaic virus RNA replication. J Virol 80: 246&#226;&#8364;&#8220;251.                                  recognition RNA synthesis. J Virol 75: 2097&#226;&#8364;&#8220;2106. <br /> 11. Panavas T, Nagy PD (2003) Yeast model host study replication                 25. Noueiry AO, Diez J, Falk SP, Chen J, Ahlquist P (2003) Yeast Lsm1p-7p/Pat1p <br />     recombination defective interfering RNA Tomato bushy stunt virus.                      deadenylation-dependent mRNA-decapping factors required brome <br />     Virology 314: 315&#226;&#8364;&#8220;325.                                                                       mosaic virus genomic RNA translation. Mol Cell Biol 23: 4094&#226;&#8364;&#8220;4106. <br /> 12. Pathak KB, Sasvari Z, Nagy PD (2008) The host Pex19p plays role                     26. Tomita Y, Mizuno T, Diez J, Naito S, Ahlquist P, et al. (2003) Mutation host <br />     peroxisomal localization tombusvirus replication proteins. Virology 379:                  DnaJ homolog inhibits brome mosaic virus negative-strand RNA synthesis. <br />     294&#226;&#8364;&#8220;305.                                                                                     J Virol 77: 2990&#226;&#8364;&#8220;2997. <br /> 13. Scheller N, Mina LB, Galao RP, Chari A, Gimenez-Barcons M, et al. (2009)                 27. Mnaimneh S, Davierwala AP, Haynes J, Moffat J, Peng WT, et al. (2004) <br />     Translation replication hepatitis C virus genomic RNA depends                      Exploration essential gene functions titratable promoter alleles. Cell 118: <br />     ancient cellular proteins control mRNA fates. Proc Natl Acad Sci U S A                  31&#226;&#8364;&#8220;44. <br />     106: 13517&#226;&#8364;&#8220;13522.                                                                        28. Hughes TR, Marton MJ, Jones AR, Roberts CJ, Stoughton R, et al. (2000) <br /> 14. Taylor MP, Kirkegaard K (2008) Potential subversion autophagosomal                        Functional discovery compendium expression profiles. Cell 102: <br />     pathway picornaviruses. Autophagy 4: 286&#226;&#8364;&#8220;289.                                             109&#226;&#8364;&#8220;126. <br /> 15. van Regenmortel MHVFC, Bishop DHL, Carsten EB, Estes MK, et al. (2000)                   29. Mumberg D, Muller R, Funk M (1994) Regulatable promoters Saccharo- <br />     Virus Taxonomy. Seventh Report International Committee                             myces cerevisiae: comparison transcriptional activity <span id='am-2' about='Thesaurus:Use_For' typeof='owl:Thing'>use for</span> <br />     Taxonomy Viruses. 1st ed. San Diego: Academic Press. 1162 p.                              heterologous expression. Nucleic Acids Res 22: 5767&#226;&#8364;&#8220;5768. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      10                                 August 2011 | Volume 6 | Issue 8 | e23988 <br />                                                                                                                                  Essential Genes BMV RNA Replication <br />  <br />  <br /> 30. Ahola T, den Boon JA, Ahlquist P (2000) Helicase Capping Enzyme Active                51. Liu Z, Yang F, Robotham JM, Tang H (2009) Critical Role Cyclophilin A <br />     Site Mutations Brome Mosaic Virus Protein 1a Cause Defects Template                     Its Prolyl-Peptidyl Isomerase Activity Structure Function the <br />     Recruitment, Negative-Strand RNA Synthesis, Viral RNA Capping. J Virol                    Hepatitis C Virus Replication Complex. J Virol 83: 6554&#226;&#8364;&#8220;6565. <br />     74: 8803&#226;&#8364;&#8220;8811.                                                                            52. Kampinga HH, Craig EA (2010) The HSP70 chaperone machinery: J proteins <br /> 31. Grdzelishvili VZ, Garcia-Ruiz H, Watanabe T, Ahlquist P (2005) Mutual                         drivers functional specificity. Nat Rev Mol Cell Biol 11: 579&#226;&#8364;&#8220;592. <br />     Interference Genomic RNA Replication Subgenomic mRNA                          53. Pukszta S, Schilke B, Dutkiewicz R, Kominek J, Moczulska K, et al. (2010) Co- <br />     Transcription Brome Mosaic Virus. J Virol 79: 1438&#226;&#8364;&#8220;1451.                                   evolution-driven switch J-protein specificity Hsp70 partner. <br /> 32. Noueiry AO, Chen J, Ahlquist P (2000) A mutant allele essential, <br /> </body></html>