<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                  Structure function cytidine <br />                                                   monophosphate kinase <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                   Yersinia pseudotuberculosis, <br /> rsob.royalsocietypublishing.org <br />                                                   essential virulence but <br /> Research <br />                                                   <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>survival</span> <br /> Cite article: Walker NJ, Clark EA, Ford      Nicola J. Walker1, Elizabeth A. Clark2, Donna C. Ford1, Helen <br /> DC, Bullifent HL, McAlister EV, Duffield ML, <br /> Acharya KR, Oyston PCF. 2012 Structure        L. Bullifent1, Erin V. McAlister1,&#226;&#8364;&#160;, Melanie L. Duffield1, <br /> function cytidine monophosphate kinase <br />                                                   K. Ravi Acharya2 Petra C. F. Oyston1 <br /> from Yersinia pseudotuberculosis, essential for <br /> virulence survival. Open Biol 2:      1 <br />                                                     Biomedical Sciences, Defence Science Technology Laboratory, Porton Down, <br /> 120142.                                           Salisbury SP4 0JQ, UK <br /> http://dx.doi.org/10.1098/rsob.120142             2 <br />                                                     Department Biology Biochemistry, University Bath, Claverton Down, <br />                                                   Bath BA2 7AY, UK <br />  <br /> Received: 2 October 2012 <br /> Accepted: 13 November 2012                        1. Summary <br />                                                   The need new antibiotics pressing light emergence of <br />                                                   antibiotic-resistant strains human pathogens. Yersinia pestis, causative <br />                                                   agent plague, public health threat agent concern biode- <br /> Subject Area:                                     fence. It recently emerged clonal derivative enteric pathogen Yersinia <br /> microbiology                                      pseudotuberculosis. Previously, developed bioinformatic approach ident- <br />                                                   ify proteins suitable targets antimicrobial therapy in <br /> Keywords:                                         particular treatment plague. One target cytidine monophos- <br />                                                   phate (CMP) kinase, essential gene organisms. Previously, <br /> cytidine monophosphate kinase, crystal <br />                                                   thought CMP kinase essential Y. pseudotuberculosis, by <br /> structure, virulence, Yersinia                    modification mutagenesis approach, report production and <br />                                                   characterization Dcmk mutant. The isogenic mutant growth defect <br />                                                   relative parental strain, highly attenuated mice. We have <br />                                                   elucidated structure CMP kinase 2.32 A        &#203;&#353; , identified <br /> Author correspondence: <br />                                                   key residues active site essential activity the <br /> Petra C. F. Oyston <br />                                                   enzyme. These findings implications development novel <br /> e-mail: pcoyston@dstl.gov.uk                      CMP kinase inhibitors therapeutic use. <br />  <br />  <br />  <br />  <br />                                                   2. Introduction <br />                                                   Cytidine monophosphate (CMP) kinase member nucleoside <br />                                                   monophosphate (NMP) kinase family, plays crucial role biosynthesis <br />                                                   nucleoside precursors. In bacteria, CMP kinase catalyses transfer a <br /> &#226;&#8364;&#160; <br />                                                   phosphoryl group ATP CMP dCMP. This differs activity <br />  Present address: Imperial College, London        eukaryotic CMP kinases, catalyse conversion UMP CMP <br /> SW7 2AZ, UK.                                      respective diphosphate form [1], and, structurally, bacterial CMP kinases <br />                                                   different members NMP kinase family [2]. CMP kinase <br />                                                   over-expression suppress lethal effects inactivation UMP <br />  <br />  <br />                                                    &amp; 2012 The Authors. Published Royal Society terms Creative Commons Attribution <br />                                                    License http://creativecommons.org/licenses/by/3.0/, permits unrestricted use, provided original <br />                                                    author source credited. <br />  kinase, PyrH [3,4], indicated low level UMP kinase           plague, inhibitors bacterial CMP kinase        2 <br /> activity CMP kinase, observation confirmed later [5].             represent novel class broad-spectrum antimicrobials. <br />  <br />  <br />  <br />  <br />                                                                                                                                          rsob.royalsocietypublishing.org <br />      CMP kinase reported essential viability of <br /> Bacillus subtilis [6] Streptococcus pneumoniae [7]. However, <br /> inactivation cmk reportedly lethal Escherichia coli        3. Material methods <br /> [8] Salmonella enterica [9]. In Haemophilus influenzae, <br />                                                                         Unless stated, chemicals purchased from <br /> there copies cmk [10], reported <br />                                                                         Sigma-Aldrich (Poole, UK), enzymes purchased <br /> for E. coli. However, conflicting data regarding <br />                                                                         Promega Ltd (Southampton, UK). <br /> whether cmk essential E. coli, mutagenesis <br /> studies indicate cmk is, fact, essential organism <br /> [11,12], essentiality target influenced            3.1. Production Yersinia pseudotuberculosis <br /> strain method used. Where mutant generated, the <br />                                                                              Dcmk mutant <br />  <br />  <br />  <br />  <br />                                                                                                                                          Open Biol 2: 120142 <br /> E. coli mutant significant growth defect demon- <br /> strated cold sensitivity [8]. Previously, attempted           Construction Y. pseudotuberculosis Dcmk mutant was <br /> create cmk mutant Yersinia pseudotuberculosis, suc-        generated reported previously [13] modification <br /> cess, indicated cmk essential gene             primers designed amplify kanamycin- <br /> member Enterobacteriaceae [13].                                  resistance cassette including cognate promoter <br />      Yersinia pseudotuberculosis human pathogenic       region. The primers used were: 50 -CTGCCGGGGGCAGAC <br /> members genus, Yersinia enterocolitica          AAGAGATTTGCCTACCGAAAGGAGAGATAAATGAGCC <br /> and Yersinia pestis. Although Y. pseudotuberculosis                 ATATTCAACGGG-30 (forward) 50 -CGTATTGAGAGCG <br /> Y. enterocolitica enteric pathogens, Y. pestis causes bubonic       CAGAACTGACCGCATTGTCGTTAGAAAAACTCATCGAG <br /> and pneumonic plague. Yersinia pestis considered             CATC-30 (reverse), sequence italic font hybri- <br /> recently emerged clonal derivative Y. pseudotuberculosis             dized kanamycin-resistance gene. PCR products were <br /> [14]. The evolution Y. pestis enteropathogen              generated using plasmid pK2 [19] template, and <br /> arthropod-vectored systemic pathogen involved                  excess template digested DpnI. PCR products <br /> gene acquisition loss [15], organisms retain            purified using Millipore Microcon Ultracel YM-100 <br /> high levels genetic similarity. As result, Y. pseudotuberculosis   transformed Y. pseudotuberculosis <br /> is exploited safer model pathogen elucidating            IP32953 [20] pAJD434 [21] electroporation. Following <br /> pathogenic mechanisms Yersinia, proteins retain             overnight incubation 288C Luria Bertani (LB) broth sup- <br /> their function species, results validated       plemented 0.8 cent arabinose, transformants were <br /> subsequently plague bacillus focused efforts.            selected LB agar supplemented kanamycin <br />      Approximately 2000 cases plague reported                    (50 mg ml21) trimethoprim (100 mg ml21) 48 h at <br /> annually World Health Organization. In case               288C. Transformants screened PCR using target <br /> suspected plague infection, recommended antibiotics             gene-specific (50 -TTTGCCTACCGAAAGGAGAG-30 50 - <br /> therapy gentamicin doxycycline quinolones [16],              GCGCAGAACTGACCGCATTG-30 ) kanamycin gene- <br /> with prompt initiation therapeutic regimens essential, par-          specific primers (50 -GCCATATTCAACGGGAAACG-30 and <br /> ticularly highly acute pneumonic form disease.               50 -AAACTCACCGAGGCAGTTCC-30 ). <br /> Antibiotic resistance reported clinical isolates                 Mutant strains cured pAJD434 plasmid by <br /> [17], highly efficient transfer resistance genes           growth 378C LB medium supplemented kanamy- <br /> the midgut flea [18]. As human-<span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span>-human transmission             cin (50 mg ml21). Cured mutant strains screened for <br /> is risk plague cases, need effective        virulence plasmid pYV PCR genes located <br /> vaccines antibiotics prevent outbreaks arising              plasmid: virF yscC. The retention Yersinia <br /> enzootic areas.                                                         virulence plasmid ( pYV) confirmed culture on <br />      Previously, reported bioinformatic approach              Congo-red magnesium oxalate plates, plasmid reten- <br /> to identify potential new targets novel antimicrobials              tion results small red colonies plasmid loss results in <br /> [13]. The database essential genes (DEG) records               large pink colonies. <br /> genes thought essential range bacteria. Using                <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>Mutation</span> cmk confirmed Southern blotting. <br /> down-selection process included conservation                Genomic DNA digested overnight Bpu10I, resolved <br /> DEG, cmk identified potential lethal gene,                0.7 cent agarose gel blotted positively <br /> cmk homologues identified 11 14 prokaryotes                   charged nylon membrane (Roche). Southern hybridization <br /> listed DEG. Other selection criteria, including size,                performed using digoxigenin-labelled probe amplified <br /> enzymatic function non-membrane protein, provided                   open reading frame located immediately down- <br /> additional support CMP kinase &#226;&#8364;&#732;druggable&#226;&#8364;&#8482; target               stream cmk encoding 30S ribosomal protein S1 (rpsA; <br /> through inhibition active site. In study,                primer sequences 50 -GAAAGCTAAGCGTCACGAAG-30 and <br /> wished definitively prove CMP kinase                  50 -GTCCAAGCCGATGAAGATAC-30 ) Roche DIG <br /> essential locus mutated. We elucidated crystal            system, according manufacturer&#226;&#8364;&#8482;s instructions. <br /> structure CMP kinase Y. pseudotuberculosis vali- <br /> dated function enzyme kinase. Through <br /> homology modelling, identified potentially impor- <br />                                                                         3.2. Expression purification CMP kinase <br /> tant protein &#226;&#8364;&#8220;ligand interactions elucidated                    The Y. pseudotuberculosis cmk gene amplified PCR <br /> essentiality number interactions site-directed         genomic DNA using Phusion DNA polymerase (Finn- <br /> mutagenesis. These results provide basis           zymes, NEB) oligonucleotide primers GSTcmkfor <br /> research development novel therapeutics                 50 GGATCCATGACGGCGATAGCCCCGGTGATA30 (BamH1 <br />  site indicated) GSTcmkrev 50 GCGGCCGCTTATTTTTT                PHENIX [26]. A set reflections (5%) set aside         3 <br /> CAACGGCAAGG30 (Not1 site indicated). The amplified                Rfree calculation [27]. <br />  <br />  <br />  <br />  <br />                                                                                                                                   rsob.royalsocietypublishing.org <br /> fragment ligated blunt cloning vector pCR- <br /> Blunt II-TOPO (Invitrogen) using T4 DNA ligase (Roche). <br /> The cmk insert excised plasmid restriction        3.5. Homology modelling <br /> digestion BamH1 Not1, ligated               A model Yersinia CMP kinase created using SWISS <br /> pGEX-6P-1 (GE Healthcare) vector. Fidelity cmk gene        modeller server [28]. This model based sequence <br /> was confirmed sequencing.                                      alignment Yersinia CMP kinase E. coli CMP <br />      Escherichia coli BL21*(DE3) harbouring recombinant           kinase CDP bound (PDB <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> 2CMK). The coordinates <br /> pGEX-6P-1 plasmids cultured LB broth supplemen-           2CMK used add CDP model. <br /> ted 1 cent glucose ampicillin (50 mg ml21). <br /> Cultures grown shaking (170 rpm) 378C until <br />  <br />  <br />  <br />  <br />                                                                                                                                   Open Biol 2: 120142 <br /> an optical density 600 nm (OD600) 0.4 reached.          3.6. CMP kinase activity assay <br /> Protein expression induced 1 mM isopropyl-B- <br />                                                                   CMP kinase activity determined using coupled spec- <br /> D-thiogalactoside (Roche) incubation 4 h <br />                                                                   trophotometric assay based Blondin et al. [29]. <br /> followed harvesting centrifugation (10 min, 1700 g). <br />                                                                   Briefly, determination Km CMP, 1 ml of <br /> Cell pellets resuspended PBS sonicated in <br />                                                                   50 mM Tris pH 7.4 containing 2 mM MgCl2, 50 mM KCl, <br /> an ice water bath. The suspension clarified centrifu- <br />                                                                   1 mM phosphoenolpyruvate, 0.2 mM NADH, 1 mM ATP, <br /> gation 27 000g 30 min. Supernatants loaded onto <br />                                                                   units lactate dehydrogenase, pyruvate kinase <br /> a GSTrap FF column (GE Healthcare) equilibrated with <br />                                                                   NDP kinase, various concentrations CMP, for <br /> PBS. The column washed equilibration buffer, <br />                                                                   determination Km ATP 0.1 mM CMP var- <br /> followed cleavage buffer (50 mM Tris, 150 mM NaCl, <br />                                                                   ious concentrations ATP incubated 308C the <br /> 1 mM EDTA, 1 mM DTT pH 7). PreScission Protease (GE <br />                                                                   absorbance monitored 340 nm. When stable absorbance <br /> Healthcare) cleavage buffer added (160 units milli- <br />                                                                   reached (10 mM), CMP kinase added the <br /> litre column matrix) incubated overnight 48C. The <br />                                                                   decrease absorbance recorded. <br /> column washed cleavage buffer, fractions <br /> containing cleaved CMP kinase pooled buffer <br /> exchanged 50 mM HEPES, 150 mM NaCl, 10 mM <br />                                                                   3.7. Generation point mutations CMP kinase <br /> glycerol pH 7. <br />      For structural analysis, elution fractions containing        Point mutants Arg188Ala, Arg131Ala Arg110Ala were <br /> CMP kinase determined SDS&#226;&#8364;&#8220;PAGE analysis                generated using Stratagene Quikchange site-directed <br /> pooled buffer exchanged 50 mM MES pH 6.0,                mutagenesis kit. Mutagenesis carried the <br /> 150 mM NaCl. Protein concentrated approximately            recombinant cmk-pGEX-6P-1 plasmids, using oligonucleotide <br /> 6.4 mg ml21, using Millipore concentrator.                      primers 50 -GGATAACCGTGATCGTAACGCTGCTGTTGCA <br />                                                                   CCTTTAGTC-30 , 50 -GGTTTAATTGCTGATGGCGCGGATAT <br />                                                                   GGGGACTATCGT-30 50 -CGGCTTTCCCCCGAGTAGCTG <br /> 3.3. Crystallization CMP kinase                                AAGCATTACTGC-30 . Mutated plasmids transformed <br /> Initial screens identify conditions CMP kinase        E. coli XL-1 blue plated LB plates supplemented <br /> crystallized carried using pre-prepared 96-well          ampicillin (50 mg ml21). Colonies screened PCR <br /> screens (Molecular Dimensions) Art Robbins Phoenix          sequenced using T7 universal primers. <br /> nano-dispenser. Drops 300 nl set using sitting <br /> drop method. Optimization carried 24-well plates <br /> using hanging drop method. Improved crystals               3.8. Circular dichroism spectroscopy <br /> CMP kinase grew 7 days 2 ml drop contained         Circular dichroism (CD) studies performed Jasco <br /> a 1 : 1 volume ratio protein solution mother liquor.        J-600 spectropolarimeter 2 mm circular cell. Samples <br /> Mother liquor contained 100 mM sodium HEPES pH 7.5,               wild-type CMP kinase (0.7 mg ml21) the <br /> 1.2 M lithium sulphate 100 mM magnesium chloride.             mutants Arg188Ala (0.3 mg ml21), Arg131Ala (0.4 mg ml21) <br />                                                                   Arg110Ala (0.7 mg ml21) prepared 50 mM <br /> 3.4. Structure determination refinement                       HEPES, 150 mM NaCl 10 mM glycerol pH 7.0. Wave- <br />                                                                   <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> scans performed 208C 195 250 nm at <br /> X-ray diffraction data collected single cryo-         rate 50 nm min21. <br /> cooled crystal (100 K) 2 M lithium sulphate the <br /> Diamond Light Source (Didcot, UK) ADSC Q315 <br /> CCD detector station I02 (l &#194;&#188; 0.98 A &#203;&#353; ). Diffraction images   3.9. In vitro evaluation growth the <br /> were collected oscillation angle 1.08. Data pro- <br /> cessed rhombohedral space group R3 using XDS [22], <br />                                                                        Dcmk mutant <br /> and data scaled using SCALA [23]. Initial phases             At 288C agitation, cultures strain IP32953 the <br /> were obtained molecular replacement using             Dcmk mutant grown overnight LB LB sup- <br /> program PHASER [24]. The search model used molecu-            plemented 50 mg ml21 kanamycin, respectively. The <br /> lar replacement model Yersinia CMP kinase                overnight cultures used inoculate fresh pre-warmed <br /> generated based sequence alignment E. coli            medium, cultures incubated above. <br /> CMP kinase (PDB <span id='am-242' about='Thesaurus:code' typeof='owl:Thing'>code</span> 1CKE). Model building per-               At selected intervals, aliquots removed OD600 <br /> formed COOT [25], refinement performed               measurement enumeration viable counts. <br />      In vitro competition index (CI) experiments con-           (a) 2.500                                                                  4 <br /> ducted prior vivo CI evaluation. Cultures both <br />  <br />  <br />  <br />  <br />                                                                                                                                              rsob.royalsocietypublishing.org <br /> strains grown LB overnight 288C agitation,                     2.000 <br /> and diluted OD600 0.1 LB. These cultures seri- <br /> ally diluted viable counts determined culture LB                     1.500 <br />  <br />  <br />  <br />  <br />                                                                     OD600 <br /> agar. Wild-type mutant bacterial suspensions were <br /> mixed 1 : 1 ratio incubated 288C shaking.                     1.000 <br /> At selected intervals, aliquots removed OD600 <br /> measurement enumeration viable counts. The CI                        0.500                                              wt <br /> defined output ratio (mutant/wild-type) divided                                                                         Dcmk <br /> the input ratio (mutant/wild-type).                                                 0 <br />  <br />  <br />  <br />  <br />                                                                                                                                              Open Biol 2: 120142 <br />                                                                     (b) 1 &#195;&#8212; 1010 <br />  <br /> 3.10. In vivo studies Dcmk mutant <br /> Attenuation Y. pseudotuberculosis Dcmk mutant eval-                 1 &#195;&#8212; 109 <br /> uated CI median lethal dose (MLD) evaluation. Mutant <br />  <br />  <br />  <br />  <br />                                                                     cfu ml&#226;&#8364;&#8220;1 <br /> and wild-type strains grown separately 100 ml LB <br /> broth shaking 288C overnight. The cultures                   1 &#195;&#8212; 108 <br /> centrifuged (10 min, 6000g) pellet resuspended in <br /> 4 ml sterile 10 cent v/v aq. glycerol. Aliquots stored                                                              wt count <br /> at 2808C required. One aliquot strain                                                                      Dcmk count <br />                                                                                1 &#195;&#8212; 107 <br /> defrosted, serially diluted cultured determine cfu ml21.                          0   5        10        15           20 <br />      For CI challenge, aliquot strain                                              time (h) <br /> defrosted, diluted PBS approximately 1   109 cfu ml21, <br /> and wild-type mutant bacterial suspensions mixed a <br />                                                                     Figure 1. Growth Y. pseudotuberculosis IP32953 Dcmk mutant, <br /> 1 : 1 ratio. This suspension serially diluted ster- <br />                                                                     followed (a) optical density (b) viable count. <br /> ile PBS. Retrospective viable counts determined by <br /> plating dilutions triplicate LB agar LB agar sup- <br /> plemented kanamycin determine input ratio.                  In pure culture, growth Dcmk mutant appeared <br /> Groups mice dosed 0.1 ml            slower wild-type, eventually <br /> dilutions intravenous route. After 3 days, spleens           reached similar final density (figure 1). A fitness defect was <br /> were removed passed 70 mm sieves (Becton&#226;&#8364;&#8220;               observed vitro CI, value 0.186 indicated <br /> Dickinson) produce cell suspension 3 ml PBS. Cell        reduced fitness mutant compared wild-type. <br /> suspensions serially diluted sterile PBS plated             Mice challenged i.p. serial dilutions wild- <br /> onto LB agar, LB agar supplemented kanamycin            type Y. pseudotuberculosis Dcmk mutant. Retrospective <br /> determine output ratio. The CI determined above.         challenge doses showed mice challenged IP32953 <br />      For MLD determination, aliquot culture          received 6.4   105 &#226;&#8364;&#8220;0.64 cfu, mice challenged the <br /> defrosted diluted PBS. Retrospective viable counts           mutant received 2.2   107 &#226;&#8364;&#8220; 22 cfu. The published MLD of <br /> were determined plating dilutions triplicate LB        wild-type strain 2200 cfu [31]. In study, MLD <br /> agar. Groups mice dosed intraperitoneally (i.p.)       wild-type strain 940 cfu. The Dcmk mutant was <br /> with 0.1 ml dilutions. Mice observed twice            significantly attenuated, MLD 9.28   107 cfu <br /> daily, animals moribund deemed incap-             (figure 2), signs recorded mice receiving <br /> able survival humanely killed. Mice observed           2.2   105 cfu mutant. This confirmed by <br /> for 28 days. The MLD calculated reported [30].               vivo CI, CI 4.48   1026 indicated significant <br />                                                                     reduction fitness compared wild-type (a CI of <br />                                                                     0.2 considered significant reduction fitness <br />                                                                     model). <br /> 4. Results <br /> 4.1. Inactivation CMP kinase highly attenuating               4.2. The CMP kinase crystal structure <br /> The Y. pseudotuberculosis cmk gene inactivated replace-      There CMP kinase molecule crystallographic <br /> ment gene kanamycin-resistance cassette driven        asymmetric <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, solvent content 48 cent. The <br /> by cmk promoter. Following transformation PCR          final refined structure Rcryst 22.2 cent an <br /> product, colonies grew LB kanamycin agar,              Rfree 27.8 cent (table 1). The final model consists of <br /> were selected screening PCR. Of these,              18 water molecules sulphate ions asymmetric <br /> shown mutants lacking cmk gene carrying           <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. The Ramachandran plot generated Molprobity <br /> kanamycin cassette. Following curing pAJD434 plas-           shows residues disallowed regions [32]. <br /> mid, confirmation unstable Yersinia virulence          Clear density observed Met1 Ala225 with <br /> plasmid pYV retained, mutation            exception 12-residue gap Glu180 Ala191. <br /> mutants confirmed Southern blotting, size            The N-terminal residues observed. Side chain <br /> difference approximately 4 kb observed           density observed residues Glu57, Glu58, Arg92 <br /> Dcmk mutant wild-type.                                          Glu122, chains modelled. <br />      (a) 6                                                                                      Table 1. Crystallographic data CMP kinase structure. Numbers     5 <br />                                                                                                parentheses upper resolution shell (2.32&#226;&#8364;&#8220; 2.45 A&#203;&#353; ), where <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          rsob.royalsocietypublishing.org <br />                        5                                                                       appropriate. <br />  <br />                        4 <br />     no. survivors <br />  <br />  <br />  <br />  <br />                                                                                                                                  5 b 5 88.64 A&#203;&#353; , c 5 84.29 A&#203;&#353; ; <br />                        3 <br />                                                                                                    cell dimensions               5 b 5 9088, g 5 12088 <br />  <br />                                                                                                    space group                   R3 <br />                        2 <br />                                                                                                    resolution (A&#203;&#353; )              2.32 <br />                        1                                                                           completeness (%)              96.4 (87.4) <br />                                                                                                    no. re&#239;&#172;&#8218;ections             44 472(4160) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          Open Biol 2: 120142 <br />                        0 <br />                                WT 6.4 &#195;&#8212; 105             WT 6.4 &#195;&#8212; 104           WT 6.4 &#195;&#8212; 103        no. unique re&#239;&#172;&#8218;ections      10 266(1370) <br />    (b) 6 <br />                                          WT 640 cfu               WT 64 cfu                        redundancy (%)                4.3 (3.0) <br />                                                                                                    I/sI                          16.2 (2.8) <br />                        5                                                                           Rmerge (%)a                   4.7 (46.4) <br />                                                                                                              b <br />                                                                                                    Rcryst (%)                    22.2 <br />                        4                                                                           Rfree (%)c                    27.8 <br />     no. survivors <br />  <br />  <br />  <br />  <br />                                                                                                    Wilson B-factor (A&#203;&#353; 2)        61.6 <br />                        3 <br />                                                                                                    average B-factors main        64.2/67.2 <br />                        2                                                                              chain/side chain (A&#203;&#353; 2) <br />                                                                                                    r.m.s.d. ideal values <br />                        1 <br />                                                                                                       bonds (A&#203;&#353; )                0.008 <br />                                                                                                       angles (8)                 1.168 <br />                        0 <br />                                                                                                a <br />                            0             5               10               15              20     Rmerge &#194;&#188; ShklSijIi (hkl 2 kI(hkl)lj/ShklSIi (hkl), kIl the <br />                                                  days post-infection                           averaged intensity observations reflection hkl. <br />                                                                                                b <br />                                                                                                  Rcryst &#194;&#188; SjjFoj2jFcjj/SjFoj, Fo Fc observed <br />                                cmk 2.2 &#195;&#8212; 107           cmk 2.2 &#195;&#8212; 106           cmk 2.2 &#195;&#8212; 105 <br />                                                                                                calculated structure factors, respectively. <br />                                cmk   2.2 &#195;&#8212; 104         cmk    2.2 &#195;&#8212; 103        cmk 220 cfu     c <br />                                                                                                 Rfree equal Rcryst random set reflections (5%) <br />                                                        cmk 22 cfu                              used refinement [27]. <br /> Figure 2. Survival mice following challenge (a) wild-type <br /> Y. pseudotuberculosis IP32953 (b) Dcmk mutant. <br />                                                                                                CMP kinase (PDB code 1CKE), residues 180&#226;&#8364;&#8220;192 are <br />                                                                                                disordered absence ligand. <br />     Yersinia CMP kinase adopts overall fold                                            Within structure, anion hole formed glycine- <br /> related CMP kinases composed a-helices                                      rich loop forms mononucleotide-binding <br /> seven b-strands connected loops. These arranged                                         motif. This forms ATP-binding site, Yersinia <br /> into domains (figure 3). The central portion mol-                                 CMP kinase, sulphate ion bound. The sulphate ion <br /> ecule composed central b-sheet, b1, b2,                                   occupies position previously published <br /> b5, b6 b7, a1 a9. Helix a6 joins central                                       bacterial CMP kinase structures crystallized <br /> domain NMP-binding domain. The NMP-binding                                              absence ATP [2,33]. <br /> domain composed anti-parallel b-strands (b3 b4) <br /> and helices (a2, a3, a4 a5). Yersinia CMP kinase be <br /> described member class long-NMP kinases <br />                                                                                                4.3. The CMP kinase: CDP model <br /> owing insert NMP-binding domain,                                           A model Yersinia CMP kinase CDP bound cre- <br /> which present short variants: structure,                                 ated based sequence alignment Yersinia <br /> insert comprises anti-parallel b-strands, loop region                                    CMP kinase E. coli CMP kinase CDP bound (PDB <br /> that connects b4 b4, a5. The lid domain composed                                     code 2CMK). The proteins share 84 cent identity and <br /> of helices a7 a8. Yersinia CMP kinase E. coli CMP                                      94 cent similarity, ligand-interacting resi- <br /> kinase (PDB code 1CKE) superimpose                                        dues conserved (figure 5). We compared the <br /> root mean square deviation (r.m.s.d.) 210 Ca atoms                                       structure Yersinia CMP kinase absence ligand <br /> 0.91 A&#203;&#353; . The proteins share 84 cent identity 93                                   model CDP-bound Yersinia CMP kinase in <br /> cent similarity.                                                                               order propose potential binding-induced changes for <br />     The ligand-binding site formed pocket                                       Yersinia CMP kinase. <br /> NMP-binding domain. In related structures, CMP CDP                                             The NMP-binding domain undergoes movement upon <br /> bind cytosine <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> ligand sits                                     substrate binding, notably anti-parallel b-strands <br /> C-terminal end b5 strand, phosphate moiety                                      b3 b4. The a5-helix moves binding site, <br /> is positioned a2&#226;&#8364;&#8220; b2 boundary [8,9]. One sulphate                                  giving closed conformation. The homology model <br /> ion bound site (figure 4). Residues Glu180 Ala191                                contains section residues E180 A191, not <br /> are disordered structure. This case E. coli                            seen apo-enzyme. In homology model, section <br />                                                                                                                                                                     6 <br />                                                                        a5 <br />                                                                                                                           a8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  rsob.royalsocietypublishing.org <br />                                                                                                                            lid domain <br />                                                                                         a4 <br />                                                                                                                  a7 <br />                                          a3 <br />                                                                             a2 <br />                                                                                              b4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  Open Biol 2: 120142 <br />                                                         a6             b3 <br />                                                                                                       a1 <br />                                                                             b2 <br />                           NMP-binding domain                                             b5 <br />  <br />                                                                                                            b6 <br />                                                                                                                      b7 <br />  <br />                                                                                         b1 <br />                                                                                                                 a9 <br />  <br />  <br />  <br />                                                                    N <br />  <br />                                                                        central domain             C <br /> Figure 3. The overall fold CMP kinase shown. a-helices b-strands numbered. The N- C-termini labelled. The central domain, lid domain and <br /> NMP-binding domains coloured pink, blue orange, respectively. This figure prepared using PYMOL (v. 0.99; Schro&#194;&#168;dinger, LLC; www.pymol.org). <br />  <br />  <br />                                                                                         Briozzo et al. [2] suggested region <br />                                                                                         ordered binding. The notable side- <br />                                                                                         chain rearrangements Arg131, Arg110 and <br />                              R110                                                       Arg41. Arg131 Arg41 chains make hydrogen <br />                                               R110*                                     bonds phosphate moieties ligand. Arg110 <br />                                                                                         chain forms hydrogen bonds cytosine base <br />                                                                                         CDP, does nearby Asp132 chain. Residues <br />                                                                                         Ser36 Ala38 N-terminal end a2 helix pre- <br />                 R188* <br />                                                                                         dicted away ligand-binding site, and <br />                                                                     S36*                predicted hydrogen bond Ser36 side <br />                                                                                         chain cytosine base. Adjacent section, Tyr40 <br />                    R131              CDP                                                moves stacking interaction with <br />                                                     S36 <br />                                                                                         cytosine base CDP. The phosphate moiety the <br />       D185*                                                        G37*                 ligand a2&#226;&#8364;&#8220;b2 boundary Ser14 hydrogen <br />                                                                                         bonded CDP. <br />                        R131* <br />                                                              G37 <br />  <br />                                                                                         4.4. Activity Yersinia CMP kinase <br />                                                                                         The activity CMP kinase confirmed in vitro <br />                                                              R41*                       assay. At constant concentration 1 mM ATP, Km <br />                                           SO42&#226;&#8364;&#8220;                                         kcat values CMP 0.028 mM 91.9 s21, respect- <br />                                                     R41                                 ively, constant concentration 0.1 mM CMP, Km <br />                                                                                         ATP determined 0.04 mM kcat 74.3 s21. <br /> Figure 4. Active site CMP kinase. Green sticks: positioning sulphate          On comparison activity data available litera- <br /> ion CMP kinase active site number surrounding residues                  ture (table 2), KCMPm    Yersinia CMP kinase is <br /> shown. Blue sticks: model CDP-bound CMP kinase shown residue                  similar E. coli B. subtilis. The KCMP <br />                                                                                                                                            m    the <br /> labels denoted asterisk. Predicted hydrogen bonds shown              M. tuberculosis protein, however, approximately threefold <br /> black dashes. This figure prepared using PYMOL.                                     higher bacteria, kcat twofold <br />                                                                                         lower Yersinia E. coli proteins. Both Yersinia <br /> is mainly a-helical extension a8 helix. Resi-                          E. coli CMP kinases inhibited CMP 0.2 mM; <br /> dues Arg188 Asp185 section interact                                  however, B. subtilis protein slightly inhibited by <br /> CDP homology model. This case E. coli,                            concentrations 2 mM [37]. The Yersinia protein KATP m    is <br />                                                                                                                                                                   7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                rsob.royalsocietypublishing.org <br />                                                                                                                                                                Open Biol 2: 120142 <br /> Figure 5. Sequence alignment Streptococcus pneumoniae, Staphylococcus aureus, Mycobacterium abscessus, Yersinia pseudotuberculosis E. coli <br /> CMP kinase. Residues mutated Yersinia protein study highlighted. Alignment performed using CLUSTALW sequence alignment <br /> program [34]. <br />  <br />  <br /> Table 2. Summary kinetic data Yersinia CMP kinase bacterial CMP kinases available literature. <br />  <br />                                                                   CMP                                                   ATP <br />  <br />                                        reference                  Km (mM)                    kcat (s21)                 Km (mM)                kcat (s21) <br />  <br />   Y. pseudotuberculosis                study                 0.028                       91.9                      0.04                   74.3 <br />   E. coli                              [5,35]                     0.035                      103                        0.038                  &#226;&#8364;&#8221; <br />   M. tuberculosis                      [36]                       0.12                       52                         &#226;&#8364;&#8221;                      &#226;&#8364;&#8221; <br />   B. subtilis                          [37]                       0.04                       &#226;&#8364;&#8221;                          0.12                   &#226;&#8364;&#8221; <br />  <br />  <br />  <br /> again similar E. coli protein, B. subtilis                   disease, responsible millions deaths <br /> protein KATP <br />           m    threefold higher.                                                 history. Unfortunately, need new antibiotics is <br />     On basis structural analysis, site-directed mutants                      pressing Y. pestis organism long <br /> were generated residues predicted play key role                          concern biodefence. Previously, reported <br /> the active site. In order assess mutants                        identification cmk potentially interesting and <br /> correctly folded, CD experiments performed wild-                            essential gene target development of <br /> type CMP kinase mutants Arg131Ala,                                  novel antibiotics [13]. In study, modifying the <br /> Arg110Ala Arg188Ala. The CD spectra proteins                            approach mutating target gene, able pro- <br /> were highly similar, indicating mutants cor-                          duce mutant Y. pseudotuberculosis cmk <br /> rectly folded. When mutated proteins assayed                          inactivated. The homologous DNA sequences oli- <br /> concentration 10 mM constant concentration                             gonucleotide primers same, difference <br /> 1 mM ATP concentrations CMP 5 mM,                                   successful approach kanamycin-resist- <br /> activity seen, showing essential CMP                             ance cassette driven cmk promoter than <br /> binding active site.                                                         kanamycin cassette promoter. This explain con- <br />                                                                                     flicting reports essentiality organisms the <br />                                                                                     mutant appears highly sensitive approach used <br />                                                                                     production. <br /> 5. Discussion                                                                            Similar E. coli [8], CMP kinase required normal <br /> The need new antibiotics pressing light                        growth Y. pseudotuberculosis 288C. Owing issues of <br /> the emergence antibiotic-resistant strains human patho-                       virulence plasmid instability 378C, did evaluate <br /> gens. Y. pestis, causative agent plague, public health                  growth defect 378C, assumed <br /> threat parts world multiply antibiotic-                          growth defect temperature <br /> resistant strains reported [17],                      also. As predicted bioinformatic analysis [13], the <br /> need effective new antibiotics treat notorious                          Y. pseudotuberculosis Dcmk mutant significantly <br />  attenuated vivo, confirming potential target                                This similar Km values Y. pseudotuberclosis                            8 <br /> for novel antimicrobials pathogenic Yersinia. This                               protein 0.04 0.028 mM, respectively. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     rsob.royalsocietypublishing.org <br /> the report vivo characterization Dcmk mutant,                                  The structure Yersinia CMP kinase reveals globular <br /> as instances mutant genera-                                     protein consisting domains. This overall <br /> ted published studies primarily focused                                     fold observed related NMP kinases. Yersinia CMP <br /> impact bacterial physiology [8,9]. The fundamental                                       kinase contains insert NMP-binding domain, <br /> physiological function CMP kinase phosphorylation                                 characteristic long-NMP kinases. In absence a <br /> of CMP produced turnover nucleic acids pro-                                    structure ligand bound, homology modelling <br /> duce di- tri-phosphates, inactivation cmk                                    predicted number changes place upon <br /> results increased internal levels CMP excretion                                ligand binding number protein &#226;&#8364;&#8220;ligand interactions <br /> uracil cytidine [8,9]. As CTP required DNA                                       predicted. These interactions based the <br /> synthesis, loss CMP kinase compensated                                       high identity Yersinia E. coli CMP kinase. The <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     Open Biol 2: 120142 <br /> for Y. pseudotuberculosis similar way S. enterica,                               importance key arginine residues positions 131, <br /> where novo synthesis CTP CTP synthetase                                        110 188 particular confirmed site- <br /> encoded pyrG maintains supply CTP [8,9]. In addition                                directed mutagenesis, substitution key amino <br /> to precursor RNA DNA synthesis, CTP                                         acids showed essential CMP binding the <br /> and dCTP involved phospholipid biosynthesis.                                    active site. Such detailed insights stereochemistry of <br /> Therefore, effects inactivation                                     substrate binding CMP kinase underpin future struc- <br /> membrane synthesis, affect growth rate                                     ture-based efforts develop novel inhibitors targeting the <br /> and contribute attenuation. It documented                                    CMP-binding site. <br /> that apparently minor changes membrane compo- <br /> sition, <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>induced mutation</span> PhoPQ <br /> regulon [38], example, large impact <br /> on virulence. <br />                                                                                             6. Acknowledgements <br />      Although Yersinia closely related E. coli,                             This work supported Ministry Defence a <br /> being members Enterobacteriaceae, NMP kinases                                    Wellcome Trust (UK) equipment grant no. 088464 K.R.A. <br /> may different biochemical physico-chemical prop-                                   We thank scientists station IO3 Diamond Light <br /> erties. For example, despite high degree amino acid                                    Source (Didcot, UK) Dr Nethaji Thiyagarajan support <br /> identity conservation nucleotide binding cat-                                X-ray data collection, Rebecca Driesener the <br /> alytic amino acids, thymidylate kinase Y. pestis                                   University Southampton help enzyme kinetic <br /> E. coli exhibited significant differences thermodynamic                                  data analysis. We thank Marc Baylis technical <br /> properties phosphorylation activity [39]. However,                                 support. The atomic coordinates structure factors <br /> a difference observed CMP kinase. The reported                                  (code 4E22) deposited Protein Data Bank, <br /> Km E. coli CMP kinase 0.038 ATP phosphoryl                                  Research Collaboratory Structural Bioinformatics, Rutgers <br /> donor 0.035 CMP phosphate acceptor [5].                                      University, New Brunswick, NJ (www.rcsb.org). <br />  <br />  <br />  <br /> References <br /> 1.   Segura-Pena D, Sekulic N, Ort S, Konrad M, Lavie A.    6.  Sorokin A, Serror P, Pujic P, Azevedo V, Ehrlich SD.   11. Kato Ji, Hashimoto M. 2007 Construction of <br />      2004 Substrate-induced conformational changes           1995 The Bacillus subtilis chromosome region               consecutive deletions Escherichia coli <br />      human UMP/CMP kinase. J. Biol. Chem. 279,                  encoding homologues Escherichia coli mssA           chromosome. Mol. Syst. Biol. 3, 132. <br />      33 882&#226;&#8364;&#8220;33 889. (doi:10.1074/jbc.M401989200)                rpsA gene products. Microbiology 141,              12. Gerdes SY et al. 2003 Experimental determination <br /> 2.   Briozzo P et al. 1998 Structures Escherichia coli       311 &#226;&#8364;&#8220;319. (doi:10.1099/13500872-141-2-311)                 level analysis essential genes in <br />      CMP kinase complex CDP: new        7. Yu L et al. 2003 Solution structure function         Escherichia coli MG1655. J. Bacteriol. 185, <br />      fold nucleoside monophosphate binding               essential CMP kinase Streptococcus                   5673&#226;&#8364;&#8220;5684. (doi:10.1128/JB.185.19.5673-5684.2003) <br />      domain insights cytosine nucleotide               pneumoniae. Protein Sci. 12, 2613&#226;&#8364;&#8220;2621. (doi:10.       13. Duffield M, Cooper I, McAlister E, Bayliss M, Ford D, <br />      specificity. Struct. Folding Des. 6, 1517&#226;&#8364;&#8220; 1527.           1110/ps.03256803)                                          Oyston P. 2010 Predicting conserved essential genes <br />      (doi:10.1016/S0969-2126(98)00150-6)                    8. Fricke J, Neuhard J, Kelln RA, Pedersen S. 1995 The         bacteria: silico identification putative drug <br /> 3.   Yamanaka K, Ogura T, Koonin EV, Niki H, Hiraga S.          cmk gene encoding cytidine monophosphate kinase            targets. Mol. Biosyst. 6, 2482&#226;&#8364;&#8220;2489. (doi:10.1039/ <br />      1994 Multicopy suppressors, mssA mssB,           located rpsA operon required          c0mb00001a) <br />      smbA mutation Escherichia coli. Mol. Gen. Genet.        normal replication rate Escherichia coli.           14. Achtman M, Zurth K, Morelli C, Torrea G, Guiyoule <br />      243, 9&#226;&#8364;&#8220;16. (doi:10.1007/BF00283870)                        J. Bacteriol. 177, 517&#226;&#8364;&#8220;523.                                A, Carniel E. 1999 Yersinia pestis, cause of <br /> 4.   Yamanaka K, Ogura T, Niki H, Hiraga S. 1992            9. Beck CF, Neuhard J, Thomassen E, Ingraham JL,               plague, recently emerged clone Yersinia <br />      Identification characterization smbA            Kleker E. 1974 Salmonella typhimurium mutants              pseudotuberculosis. Proc. Natl Acad. Sci. USA 96, <br />      gene, suppressor mukB mull mutant              defective cytidine monophosphate kinase (cmk).          14 043 &#226;&#8364;&#8220;14 048. (doi:10.1073/pnas.96.24.14043) <br />      Escherichia coli. J. Bacteriol. 174, 7517 &#226;&#8364;&#8220;7526.           J. Bacteriol. 120, 1370&#226;&#8364;&#8220;1379.                          15. Parkhill J et al. 2001 Genome sequence Yersinia <br /> 5.   Bucurenci N et al. 1996 CMP kinase Escherichia    10. Fleischmann RD et al. 1995 Whole-genome random             pestis, causative agent plague. Nature 413, <br />      coli structurally related nucleoside           sequencing assembly Haemophilus influenzae          523&#226;&#8364;&#8220;527. (doi:10.1038/35097083) <br />      monophosphate kinases. J. Biol. Chem. 271,                 Rd. Science 269, 496&#226;&#8364;&#8220; 512. (doi:10.1126/science.       16. Prentice MB. 2010 Plague: Yersinia pestis. In Oxford <br />      2856&#226;&#8364;&#8220;2862. (doi:10.1074/jbc.271.5.2856)                    7542800)                                                   textbook medicine, 5th edn. (eds DA Warrell, TM <br />        CoxJD Firth), pp. 772&#226;&#8364;&#8220; 775. Oxford, UK: Oxford                crystallographic software. J. Appl. Crystallogr. 40,      33. Dhaliwal B, Ren J, Lockyer M, Charles I, Hawkins AR,         9 <br />       University Press.                                             658 &#226;&#8364;&#8220;674. (doi:10.1107/S0021889807021206)                     Stammers DK. 2006 Structure Staphylococcus <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              rsob.royalsocietypublishing.org <br /> 17.   Guiyoule A et al. 2001 Transferable plasmid-            25.   Emsley P, Lohkamp B, Scott WG, Cowtan K. 2010                 aureus cytidine monophosphate kinase complex <br />       mediated resistance streptomycin clinical             Features development coot. Acta Crystallogr.           cytidine 5&#226;&#8364;&#8482;-monophosphate. Acta Crystallogr. F <br />       isolate Yersinia pestis. Emerg. Infect. Dis. 7,            D Biol. Crystallogr. 66, 486&#226;&#8364;&#8220;501. (doi:10.1107/               Struct. Biol. Cryst. Commun. 62, 710 &#226;&#8364;&#8220;715. (doi:10. <br />       43 &#226;&#8364;&#8220;48. (doi:10.3201/eid0701.010106)                          S0907444910007493)                                            1107/S174430910602447X) <br /> 18.   Hinnebusch BJ, Rosso ML, Schwan TG, Carniel E.          26.   Adams PD et al. 2010 PHENIX: comprehensive              34. Larkin MA et al. 2007 Clustal W Clustal X <br />       2002 High-frequency conjugative transfer                   Python-based macromolecular structure              version 2.0. Bioinformatics 23, 2947&#226;&#8364;&#8220; 2948. (doi:10. <br />       antibiotic resistance genes Yersinia pestis         solution. Acta Crystallogr. D Biol. Crystallogr. 66,          1093/bioinformatics/btm404) <br />       flea midgut. Mol. Microbiol. 46, 349 &#226;&#8364;&#8220;354. (doi:10.           213 &#226;&#8364;&#8220;221. (doi:10.1107/S0907444909052925)                 35. Bertrand T et al. 2002 Sugar specificity bacterial <br />       1046/j.1365-2958.2002.03159.x)                          27.   Brunger AT. 1992 Free R-value: novel statistical            CMP kinases revealed crystal structures and <br /> 19.   Taylor VL, Titball RW, Oyston PCF. 2005 Oral                  quantity assessing accuracy crystal-               mutagenesis Escherichia coli enzyme. J. Mol. Biol. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              Open Biol 2: 120142 <br />       immunization dam mutant Yersinia                    structures. Nature 355, 472 &#226;&#8364;&#8220;475. (doi:10.1038/               315, 1099&#226;&#8364;&#8220; 1110. (doi:10.1006/jmbi.2001.5286) <br />       pseudotuberculosis protects plague.                   355472a0)                                                 36. Thum C, Schneider CZ, Palma MS, Santos DS, Basso <br />       Microbiology 151, 1919 &#226;&#8364;&#8220;1926. (doi:10.1099/mic.0.       28.   Schwede T, Kopp J, Guex N, Peitsch MC. 2003                   LA. 2009 The Rv1712 locus Mycobacterium <br />       27959-0)                                                      SWISS-MODEL: automated protein homology-                   tuberculosis H37Rv codes functional CMP <br /> 20.   Chain PSG et al. 2004 Insights evolution          modeling server. Nucleic Acids Res. 31, 3381&#226;&#8364;&#8220;3385.            Kinase preferentially phosphorylates dCMP. <br />       Yersinia pestis whole-genome comparison               (doi:10.1093/nar/gkg520)                                      J. Bacteriol. 191, 2884&#226;&#8364;&#8220;2887. (doi:10.1128/JB. <br />       Yersinia pseudotuberculosis. Proc. Natl Acad.      29.   Blondin C, Serina L, Wiesmuller L, Gilles AM, Barzu           01337-08) <br />       Sci. USA 101, 13 826&#226;&#8364;&#8220; 13 831. (doi:10.1073/pnas.              O. 1994 Improved spectrophotometric assay              37. Schultz CP et al. 1997 Structural catalytic <br />       0404012101)                                                   nucleoside monophosphate kinase activity using            properties CMP kinase Bacillus subtilis: a <br /> 21.   Maxson ME, Darwin AJ. 2004 Identification                  pyruvate kinase/lactate dehydrogenase coupling                comparative analysis homologous enzyme <br />       inducers Yersinia enterocolitica phage shock           system. Anal. Biochem. 220, 219&#226;&#8364;&#8220;221. (doi:10.                 Escherichia coli. Arch. Biochem. Biophys. 340, <br />       protein comparison regulation            1006/abio.1994.1326)                                          144&#226;&#8364;&#8220;153. (doi:10.1006/abbi.1997.9888) <br />       RpoE Cpx extracytoplasmic stress responses.     30.   Reed LJ, Muench H. 1938 A simple method               38. Gunn JS, Ryan SS, Van Velkinburgh JC, Ernst RK, Miller <br />       J. Bacteriol. 186, 4199&#226;&#8364;&#8220; 4208. (doi:10.1128/JB.186.           estimating fifty percent endpoints. Am. J. Hygiene            SI. 2000 Genetic functional analysis PmrA- <br />       13.4199-4208.2004)                                            27, 493&#226;&#8364;&#8220; 497.                                                 PmrB-regulated locus necessary lipopolysaccharide <br /> 22.   Kabsch W. 2010 Xds. Acta Crystallogr. D Biol.           31.   Lavander M, Ericsson SK, Broms JE, Forsberg A.                modification, antimicrobial peptide resistance, oral <br />       Crystallogr. 66, 125&#226;&#8364;&#8220;132. (doi:10.1107/                       2006 The twin arginine translocation                virulence Salmonella enterica serovar typhimurium. <br />       S0907444909047337)                                            essential virulence Yersinia pseudotuberculosis.       <br /> </body></html>