<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Pyrimidine Biosynthesis Is Not Essential Function for <br /> Trypanosoma brucei Bloodstream Forms <br /> Juma A. M. Ali1,2., Daniel N. A. Tagoe1., Jane C. Munday1,3, Anne Donachie1, Liam J. Morrison3,4, <br /> Harry P. Koning1* <br /> 1 Institute Infection, Immunity Inflammation, College Medical, Veterinary Life Sciences, University Glasgow, Glasgow, United Kingdom, 2 Faculty of <br /> Science, Department Zoology, Al Jabal Al Gharbi University, Gharyan, Libya, 3 Wellcome Trust Centre Molecular Parasitology, University Glasgow, Glasgow, United <br /> Kingdom, 4 Roslin Institute, University Edinburgh, Easter Bush, United Kingdom <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: African trypanosomes capable pyrimidine biosynthesis salvage preformed pyrimidines <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />      host, unknown process essential parasite. <br />  <br />      Methodology/Principal Findings: Pyrimidine requirements growth investigated using strictly pyrimidine-free <br />      media, single added pyrimidine sources. Growth rates wild-type bloodstream form Trypanosoma brucei <br />      brucei unchanged pyrimidine-free medium. The essentiality novo pyrimidine biosynthesis pathway was <br />      studied knocking PYR6-5 locus produces fusion product orotate phosphoribosyltransferase (OPRT) and <br />      Orotidine Monophosphate Decarboxylase (OMPDCase). The pyrimidine auxotroph dependent suitable <br />      extracellular pyrimidine source. Pyrimidine starvation rapidly lethal non-reversible, causing incomplete DNA <br />      content new cells. The <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> rescued addition uracil; supplementation uridine, 29deoxyuridine, <br />      cytidine allowed diminished growth rate density. PYR6-52/2 trypanosomes sensitive pyrimidine <br />      antimetabolites displayed increased uracil transport rates uridine phosphorylase activity. Pyrimidine auxotrophs <br />      able infect mice infection developed slowly infection parental, prototrophic <br />      trypanosome line. <br />  <br />      Conclusions/Significance: Pyrimidine salvage essential function bloodstream T. b. brucei. However, <br />      trypanosomes lacking novo pyrimidine biosynthesis completely dependent extracellular pyrimidine source, <br />      strongly preferring uracil, display reduced infectivity. As T. brucei able salvage sufficient pyrimidines host <br />      environment, pyrimidine biosynthesis pathway viable drug target, interruption pyrimidine <br />      supply lethal. <br />  <br />   Citation: Ali JAM, Tagoe DNA, Munday JC, Donachie A, Morrison LJ, et al. (2013) Pyrimidine Biosynthesis Is Not Essential Function Trypanosoma brucei <br />   Bloodstream Forms. PLoS ONE 8(3): e58034. doi:10.1371/journal.pone.0058034 <br />   Editor: Ben L. Kelly, Louisiana State University, United States America <br />   Received December 6, 2012; Accepted January 29, 2013; Published March 7, 2013 <br />   Copyright: &#195;&#376; 2013 Ali et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Wellcome Trust. The Wellcome Trust Centre Molecular Parasitology supported core funding the <br />   Wellcome Trust [085349], DNAT recipient Wellcome Trust scholarship. The authors gratefully acknowledge financial support Libyan <br />   Government (studentship JAMA) Medical Research Council (contract number 84733). The funders role study design, data collection and <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: Harry.de-Koning@glasgow.ac.uk <br />   . These authors contributed equally study. <br />  <br />  <br />  <br /> Introduction                                                                           eflornithine, currently administered form nifurtimox <br />                                                                                        eflornithine combination therapy (NECT) suitable <br />    Human African Trypanosomiasis (HAT, sleeping sickness)                        T. b. rhodesiense infection, treated by <br /> caused infection protozoan parasites Trypanosoma brucei                    suramin melarsoprol, Stage I (periphery) Stage II (central <br /> gambiense T. b. rhodesiense West Africa East                         nervous system) disease, respectively; severe adverse <br /> Southern Africa, respectively. In addition subspecies T. b. brucei                 effects patients [1,2]. The use NECT lowers cost toxicity <br /> and non-human infective species T. vivax T. congolense                   halt spread eflornithine resistance indefinitely <br /> cause veterinary condition African Animal Trypanosomiasis                          [3,4]. The quest new drugs led study nucleotide <br /> (AAT, nagana) livestock sub-Saharan Africa. Both                      salvage biosynthesis protozoa, initially focused on <br /> diseases continue profound health economic                                 inhibitors purine metabolism, pathogenic protozoan parasites <br /> implications poor isolated populations region. This                      (but free-living protists) lost novo purine biosynthesis <br /> problem exacerbated inadequacies existing drugs,                      pathways [5,6]. However, protozoa, including kineto- <br /> especially toxicity, parenteral route administration                    plastid parasites, redundancy purine transporters [7,8,9] and <br /> [1], high levels treatment failure reach 30%                      interconversion pathways [10,11,12] makes therapy based on <br /> in areas [2]. The drug choice late stage HAT,                              purine metabolism inhibitors extremely difficult achieve. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         1                                 March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                        Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />    In contrast, parasitic protozoa fully capable                   Care animals professional staff designated <br /> synthesizing pyrimidine ring novo [10]                 University Glasgow facility supervision qualified <br /> capable salvaging pyrimidine nucleosides and/or nucleobases                 Veterinarians. Mice infected trypanosomes humanely <br /> [13,14,15,16]. Exceptions Plasmodium spp, incapable              euthanized seriously ill infection. <br /> of pyrimidine salvage [17], amitochondriate protozoa                   Approval experiments explicitly granted the <br /> Trichomonas vaginalis, Tritrichomonas foetus Giardia spp, lack       UK Home Office, project licence PPL 60/5760 personal <br /> the novo biosynthesis pathway [18,19]. While possession             licence PIL60/2328. <br /> the biosynthesis salvage routes appear make <br /> pyrimidine metabolism unattractive drug target,                  Culture trypanosomes <br /> been established pyrimidine biosynthesis salvage                Bloodstream forms T. b. brucei strain 427 routinely <br /> is essential African trypanosomes. Moreover, salvage                cultured HMI-9 medium [29] obtained Invitrogen, <br /> biosynthesis pathways actually share pyrimidine                    supplemented 10% Heat Inactivated Fetal Bovine Serum <br /> metabolizing enzymes, shown                  Gold (FBS; PAA Laboratories) culture flasks, 37uC, 5% <br /> essential (in contrast purine metabolism) little        CO2 atmosphere. Where indicated, trypanosomes grown a <br /> no redundancy pathways. For example, dihydrofolate                      pyrimidine-free medium identical standard HMI- <br /> reductase - thymidylate synthase (DHFR-TS) essential                     9/FBS, did contain thymidine (or other <br /> trypanosomes <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> rescued high levels               pyrimidines) serum thoroughly dialysed (12&#226;&#8364;&#8220; <br /> of thymidine [20], CTP synthetase essential T. b. brucei             14 kDa cut-off) phosphate-buffered saline pH 7.4); this <br /> are unable incorporate extracellular cytosine cytidine          medium referred HMI-9-tmd standard medium <br /> nucleic acids [21]. Furthermore, T. b. brucei deoxyuridine 59-                 simply referred HMI-9. The dialysis details exact <br /> triphosphate nucleotidohydrolase (dUTPase) recently shown                  composition HMI-9 HMI-9-tmd media given the <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> essential [22] clear enzymes         Supplementary materials. <br /> same pathways equally good drug targets. <br />    However, unclear uptake                  Transport [3H]-uracil [3H]-uridine <br /> extracellular pyrimidines novo biosynthesis                Transport assays performed exactly described [30,31]. <br /> pyrimidine nucleotide, UMP, essential kinetoplastid parasites.           Briefly, trypanosomes washed appropriate assay <br /> We previously shown procyclic T. b. brucei pyrimidines            buffer (AB; 33 mM HEPES, 98 mM NaCl, 4.6 mM KCl, <br /> are mainly taken TbU1 uracil transporter [23,16]                0.55 mM CaCl2, 0.07 mM MgSO4, 5.8 mM NaH2PO4, <br /> and recently completed study pyrimidine transport activities           0.3 mM MgCl2, 23 mM NaHCO3, 14 mM glucose, pH 7.3) to <br /> bloodstream form T. b. brucei showing presence                 final concentration 108 cells ml21. 100 ml cell suspension was <br /> high affinity uracil transporter, TbU3, uptake                incubated [5,6-3H]-uracil (Perkin Elmer, 40.3 Ci/ <br /> other pyrimidines physiological levels [24]. A previous study,           mmol) [5,6-3H]-uridine (American Radiolabeled Chemicals <br /> Arakaki et al [25] showed RNAi disruption                   Inc, 30 Ci/mmol) concentrations indicated results <br /> biosynthesis enzymes, dihydroorotate dehydrogenase, led                     section, presence absence unlabeled substrate other <br /> impaired growth compensated pyrimidine                   competitive inhibitors. The incubation stopped a <br /> uptake. The rescue extracellular uracil, however,                   predetermined interval using 1 ml ice-cold 1-mM solution <br /> observed presence TbU3 inhibitor 5-fluorouracil [25].            unlabeled substrate (uracil uridine) immediate centrifu- <br /> In present study simulated complete inhibition                       gation oil (13,0006g) 1 min. The resulting cell pellet <br /> pyrimidine salvage vitro growth pyrimidine-free medium                transferred scintillation tube radioactivity was <br /> and inhibition novo biosynthesis construction           determined liquid scintillation counting. The results were <br /> genetic deletion mutant lacking final step pyrimidine               plotted appropriate equations linear non-linear regression <br /> biosynthesis pathway, trypanosomes fusion             using Prism 5 software package (GraphPad) correction for <br /> enzymes Orotidine Monophosphate Decarboxylase (PYR6,                           non-specific association radiolabel pellet, described <br /> OMPDCase) orotate phosphoribosyltransferase (PYR5,                         [30]. <br /> OPRT) [26,27]. The PYR6-52/2 trypanosomes character- <br /> ized vitro vivo. While completely non-viable            Generation auxotrophic T. brucei bloodstream forms <br /> the absence pyrimidines vitro, able grow low                The plasmid pLHTL-PYR6-5 [27] generously donated by <br /> levels pyrimidines, similar reported Leishmania donovani             Professor George Cross Rockefeller University, New York, NY. <br /> promastigotes [28]. The activity TbU3 transporter,                  This construct contains hygromycin resistance cassette (hygro- <br /> expression Uridine Phosphorylase significantly                    mycin B phosphotransferase) negative selection marker, <br /> increased PYR6-52/2 trypanosomes shifted pyrim-                   Herpes simplex Thymidine Kinase (HSVTK) open reading frame <br /> idine-free conditions. However, observation                     loxP domains [32] targeted PYR6-5 locus by <br /> parasites able establish infection mice showed              flanking sequences 496 bp immediately downstream the <br /> pyrimidine biosynthesis essential vivo, pyrimidine              target locus 365 bp commencing 134 bp upstream the <br /> salvage blood sufficient T. brucei viability growth.          ORF. <br />                                                                                   Bloodstream forms T. brucei s427 cultivated density <br /> Materials Methods                                                          ,1&#226;&#8364;&#8220;26107 cells ml21 washed Human T-Cell Solution <br />                                                                                transfection LHTL-PYR6-5 cassette (liberated by <br /> Ethics statement                                                               digestion PvuII) using Amaxa Nucleofactor electroporator <br />   The maintenance care experimental animals complied                    exactly described [27], creating PYR6-5+/2 strain. Transfor- <br /> with appropriate legislation; UK Animals (Scientific                   mants grown cloned standard HMI-9 containing <br /> Procedures) Act, 1986, national University                 hygromycin (2 mg ml21) loss second PYR6-5 allele was <br /> Glasgow maintenance care guidelines. All procedures                   induced exposure clonal lines 100 mM 5-fluoroorotic <br /> carried trained, registered licensed animal workers.                acid (5FOA; Sigma), resulting PYR6-52/2 strain, exactly as <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 2                           March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                    Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br /> described [27], cloned limiting dilution. PYR6-5              GPI8 [37] used endogenous control, primer sequences <br /> single double knockout clones confirmed PCR. DNA               59- TCTGAACCCGCGCACTTC 59-CCACTCACG- <br /> was extracted WT s427, single double                 GACTGCGTTT. For uridine phosphorylase (UP), DDCT <br /> PYR6-5 knockout strains using DNeasy Blood Tissue Kit                method used relative quantification (RQ) using WT cells in <br /> (Qiagen). Primers designed amplify 870 bp           HMI-9 calibrator internal control. Data analyzed using <br /> PYR6-5 gene. PCR performed 200 ng isolated DNA               Applied Biosystems 7500 SDS Real-Time PCR systems software. <br /> using forward (59 GTTCTCGAGTGCAAGCGGAT) reverse                        Primers used amplification UP 59- <br /> (59CACAATGCGGTCAAACTGCA) primers annealing 56uC                         TTTGACCCCTCCACCATGA 59-GATTCAGCAGGT- <br /> and extension 72uC 60 s. A Southern blot                   GAGCCACAA. The entire experiment performed three <br /> performed confirm knockouts, using restricted digest 10 mg           independent occasions, starting cell culture RNA <br /> DNA blotting performed described [33], using DNA probes             isolation. <br /> specific PYR6-5 hygromycin B phosphotransferase <br /> genes. The PYR6-5 probe generated using primers                Infectivity mice <br /> conditions given PCR confirmation, whilst                   Six-weeks-old female ICR (CD-1) Swiss outbred mice (Harlan) <br /> hygromycin probe generated PCR using forward                      divided 3 groups mice each. Mice injected <br /> (59ATGAAAAAGCCTGAACTCAC) reverse (59ACTC-                              intraperitoneally 105 bloodstream forms Trypanosoma brucei <br /> TATTCCTTTGCCCTCG) primers annealing 55uC                            brucei strains .427 WT, Pyr6-5+/2 Pyr6-52/2 200 ml of <br /> extension 72uC 60 s.                                                HMI-9 medium supplemented 10% FBS. To quantify <br />                                                                            parasitaemia, 1 ml blood daily harvested tail venepunc- <br /> Growth analysis                                                            ture infected mice appropriately diluted red blood <br />    Growth WT PYR6-52/2 strains assessed standard             cell lysis buffer (Sigma). 10 ml diluted cells examined <br /> HMI-9 HMI-9-tmd supplemented specific pyrimidines              light microscope using haemocytometer parasitae- <br /> as indicated text. Cells seeded 16105 cells ml21        mia expressed number parasites ml blood. <br /> for purpose grown 12-well plates, condition set up <br /> in 2 wells; incubation 37uC 5% CO2. Cells                  Results <br /> counted 12 24 h. The experiment performed <br />                                                                            Generation confirmation pyrimidine auxotrophic <br /> independently separate occasions. <br />                                                                            T. brucei <br />                                                                               A schematic representation generation PYR6-52/2 <br /> Alamar blue drug sensitivity assays <br />                                                                            strain shown Figure 1A. Plasmids positive selection <br />    Sensitivities PYR6-52/2 s427-WT cells 5-fluorouracil, <br />                                                                            marker hygromycin phosphotransferase (HYG) negative <br /> 5-fluoroorotic acid, 5-fluorouridine, 5-fluoro-29deoxyuridine and <br />                                                                            selection marker Herpes simplex virus thymidine kinase (HSVTK) <br /> 5-fluoro-29deoxycytidine (all Sigma) determined using the <br />                                                                            (generous donation George Cross, Rockefeller University, <br /> Alamar Blue assay exactly described [34,35], using                 New York) used. Bloodstream form T. b. brucei s427 (16106 <br /> FLUOstar Optima (BMG Labtech, Durham, NC); lexc                        cells ml21) transformed loxP-HYG-HSVTK-loxP <br /> 544 nm lem 620 nm. Seeding density 105                cassette using Amaxa Nucleofector. The transformants were <br /> (final volume 200 ml) doubling dilutions test compounds          grown selective medium containing 4.5 mg ml21 hygromycin <br /> plates incubated standard conditions 48 h,            (Sigma) cloned using limiting dilution, creating a <br /> which blue, non-fluorescent indicator dye Alamar Blue                  heterozygote PYR6-5+/2 strain. Viable clones desired <br /> (resazurin sodium salt; Sigma) added plates               insert subjected increasing drug pressure 5- <br /> incubated 24 h. Pentamidine used positive             fluoroorotic acid (5-FOA) leading loss second PYR6-5 <br /> control throughout, drug-free incubations negative control         gene (loss heterozygosity) 100 mM (Fig. 1B). Loss of <br /> (4 plate). All drugs doubly diluted 23 wells          heterozygosity (LOH) generated homozygous PYR6-52/2 <br /> starting concentration test compound 5 mM pyrimidines            confirmed using Southern blot (Fig. 1C). <br /> and 100 mM pentamidine. <br />                                                                            Growth pyrimidine auxotrophs different pyrimidine <br /> Flow cytometry                                                             sources <br />   The DNA content bloodstream trypanosomes deter-                      Under standard culture conditions clear growth <br /> mined exactly described [36]. Briefly, trypanosome samples              phenotype associated loss PYR6-5 locus, growth of <br /> were fixed o/n methanol:PBS (7:3, v/v), treated RNase              knockout cells standard HMI-9 similar WT <br /> and stained propidium iodide (both Sigma) analysed           s427 trypanosomes HMI-9-tmd supplemented 10% dialysed <br /> a FACSCalibur (Benton Dickinson) using FL2-area detector.              FBS (Fig. S1). PYR6-52/2 cells grown standard <br />                                                                            HMI-9 HMI-9-tmd, does contain pyrimidines <br /> Quantitative PCR Uridine Phosphorylase                                  does contain 1 mM hypoxanthine purine source (Table <br />   RNA isolated WT s427 PYR6-52/2 cells                        S1) supplemented FBS extensively dialysed to <br /> quantified using Nanodrop (Thermo Scientific); 2 mg RNA               remove small molecules nucleosides. As expected, PYR6- <br /> was diluted RNase-free water total volume 25 ml.                52/2 cells unable grow semi-defined medium <br /> Complementary DNA (cDNA) produced using Reverse-                     pyrimidines, trypanosome population rapidly <br /> Transcriptase (RT) kit (Primerdesign, UK). cDNA sample            declined 24 h (Fig. 2A,B). In contrast, shift purine-free <br /> was diluted 1:10 used Real Time-PCR. Amplification            conditions caused growth arrest approximately 48 h <br /> of cDNA performed 7500 Real Time PCR System                       (Fig. 2B), consistent previous observations procyclic T. <br /> (Applied Biosystems). The dissociation curve used ensure            brucei [38]. Evidently, interruption pyrimidine supply <br /> the amplification product; samples RT               rapidly makes trypanosomes non-viable investigated how <br /> cDNA used controls. The constitutively expressed gene              quickly damage irreversible (Fig. 2A), adding back <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             3                           March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                        Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 1. Generation Pyr6-5 Knockouts. (A) Schematic representation generation Orotidine Monophosphate Decarboxylase <br /> (OMPDCase) knockout. The step replacement PYR6-5 allele construct containing positive selection marker hygromycin <br /> phosphotransferase (HYG) negative marker Herpes simplex virus thymidine kinase (HSVTK) 34-bp loxP elements. The second step creates <br /> the homozygous PYR6-52/2 drug pressure 5-fluoroorotic acid (5 FOA), causing loss heterozygosity. (B) PCR analysis PYR6-5 <br /> gene, generating 870 bp amplicon described Materials Methods section confirmed absence gene Pyr6-52/2 (lane 1) and <br /> its continued presence Pyr6-5+/2 (lane 2). Lane 3 control WT s427 DNA. (C) Southern blot confirming knockout strategy, using probes for <br /> the PYR6-5 locus HYG-HSVTK cassette. Lane 1, Pyr6-52/2, lane 2, Pyr6-5+/2, Lane 3, WT s427. Band &#226;&#8364;&#732;a&#226;&#8364;&#8482; PYR6-5, band &#226;&#8364;&#732;b&#226;&#8364;&#8482; HYG-HSVTK. <br /> doi:10.1371/journal.pone.0058034.g001 <br />  <br /> 100 mM uracil various times passage PYR6-52/2                  cell population rapidly declining continuously pyrimidine- <br /> HMI-9-tmd. Cells grew density standard HMI-9                free conditions (Fig. 2A). <br /> when uracil added immediately passage (0 h control)                We tested ability cells grow 100 mM or <br /> adding uracil little 12 h resulted irreversible            1 mM (Fig. 3) natural pyrimidine nucleoside or <br /> growth arrest eventual death parasite population.              nucleobase. At lower concentration, uracil supported near- <br /> From 24 h, addition uracil redundant,              normal growth 1 mM appeared somewhat <br />                                                                               growth inhibitory growth pronounced, possibly <br />  <br />  <br />  <br />  <br /> Figure 2. Growth bloodstream form T. b. brucei media various purine pyrimidine content. A. Pyr5-62/2 trypanosomes <br /> were transferred HMI-9 HMI-9-tmd (pyrimidine-free, %) medium subsequently uracil added final concentration 100 mM at <br /> the indicated time seeding culture. Samples taken 12 h cell densities determined using haemocytometer. In cultures with <br /> conditions allowed fast growth, trypanosome population declined 36&#226;&#8364;&#8220;48 h overgrowth. Cell population &#226;&#8364;&#732;0 h&#226;&#8364;&#8482; group <br /> declined 60 h over-growth exhaustion medium. B. Comparison purine-free pyrimidine conditions. WT s427 cells were <br /> passaged mid-log cultures (grown standard HMI-9 fresh cultures medium (control,m) medium without <br />                                                                    N <br /> hypoxanthine supplemented dialysed serum (purine free, ). Pyrimidine-auxotrophic T. b. brucei (PYR6-52/2) transferred standard <br /> HMI-9 HMI-9-tmd (pyrimidine-free, &amp;) medium. Cell population &#226;&#8364;&#732;WT, HMI-9 control&#226;&#8364;&#8482; group declined sharply 26&#226;&#8364;&#8220;48 h over- <br /> growth exhaustion media. <br /> doi:10.1371/journal.pone.0058034.g002 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                4                             March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                      Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 3. Growth pyrimidine auxotrophic T. b. brucei bloodstream forms various pyrimidine sources. PYR6-52/2 cell cultures were <br /> seeded density 16105 cells ml21 cultured 37uC/5% CO2, normal HMI-9 simplified pyrimidine version supplemented with <br /> dialysed FBS indicated pyrimidine source (A) 100 mM (B) 1 mM. Samples taken 24 h cell densities determined using a <br /> haemocytometer, duplicate. The experiment shown representative similar experiments essentially identical results. In cultures <br /> with conditions allowed fast growth, trypanosome population declined 36&#226;&#8364;&#8220;48 h overgrowth. <br /> doi:10.1371/journal.pone.0058034.g003 <br />  <br /> because resulting excessive uracil influx cause                 WT trypanosomes sensitive compound EC50 <br /> imbalance pyrimidine nucleotides 29deoxyribonucle-               13.261.2 mM. Interestingly, Pyr6-52/2 strain also <br /> otides, purine pyrimidine nucleotides. Uridine           sensitized 5-fluorouridine WT cells not <br /> supported growth 100 mM better 1 mM,                  sensitive compound limit tested (5 mM). <br /> 29-deoxyuridine barely effect at 100 mM. Of the <br /> other pyrimidines, cytidine effect growth,               Expression Uridine Phosphorylase pyrimidine <br /> only 1 mM.                                                                auxotrophs <br />                                                                                 We observed cultures Pyr6-52/2 cells appeared be <br /> [3H]-Uracil uptake pyrimidine auxotrophic                                 able adapt uridine sole pyrimidine source (data not <br /> Trypanosoma brucei                                                           shown). In order utilize uridine synthesis pyrimidine <br />    We determined uracil uptake rates Pyr6-52/2 control                nucleotides need generate uracil it, using uridine <br /> s427 WT cells assess uracil uptake capacity                    phosphorylase (UP) [39]. We inferred upregulation UP <br /> bloodstream form T. b. brucei increased absence                    possible adaptation pyrimidine starvation and <br /> pyrimidine biosynthesis. WT Pyr6-52/2 grown                      performed quantitative PCR assess relative UP mRNA levels in <br /> standard HMI-9 uptake 0.15 mM [3H]-uracil measured                WT Pyr6-52/2 cells grown different media. As shown in <br /> in timecourse 120 s. The initial rate uracil uptake            Figure 5, UP expression identical WT cells grown in <br /> consistently higher Pyr6-52/2 cells (Fig. 4A). The rates         standard HMI-9 HMI-9-tmd supplemented 100 mM <br /> 0.010960.0001 0.024160.0014 pmol(107 cells)21s21 WT                  uracil, significantly increased 48 h growth HMI-9- <br /> and Pyr6-52/2 trypanosomes, respectively (n = 3; P,0.05, Stu-                tmd <br />                                                                                   supplemented 1 mM uridine (P,0.001). In Pyr6-52/2 <br /> dent&#226;&#8364;&#8482;s T-test, unpaired). The increased uptake rate             cells cultured long-term standard HMI-9 transferred to <br /> attributed expression additional uracil transporter          HMI-9-tmd/uracil 48 h UP expression levels higher than <br /> the knockout strain present WT cells, Km values           s427-WT conditions (P,0.001) level <br /> identical strains (0.3160.01 0.3460.03 mM, respec-               increased Pyr6-52/2 cells grown 48 h HMI-9- <br />                                                                              tmd <br /> tively (n = 3)) uracil transport completely                       /uridine (P,0.001). These data appear indicate T. b. <br /> insensitive uridine strains (Ki values .3 mM, n = 3;              brucei adjust UP expression levels accommodate growth <br /> Fig. 4B). Indeed, uptake 2.5 mM [3H]-uridine                   uridine sole pyrimidine source, cells are <br /> undetectable strains (data shown). However, Vmax             pyrimidine auxotrophs prototrophs. We investigated <br /> for uracil transport significantly increased Pyr6-52/2 cells          Pyr6-52/2 strains adapt long-term cultured <br /> (0.1460.01 versus 0.08760.007 pmol(107 cells)21s21, respective-              uridine sole pyrimidine source. We cells do <br /> ly; n = 3, P = 0.012) (Fig. 4C), consistent increased initial       express significantly higher UP levels s427-WT control cells <br /> rate seen time course, probably reflecting higher               grown standard HMI-9 HMI-9-tmd/uracil (P,0.01), but <br /> number uracil transporter plasma membrane               revert quickly control levels expression shifted HMI- <br /> than expression different additional transport protein.          9-tmd/uracil (Fig. 5). <br />  <br /> Sensitivity pyrimidine auxotrophic trypanosomes                     The effect pyrimidine starvation DNA content and <br /> pyrimidine analogues                                                         integrity pyrimidine auxotrophic T. b. brucei <br />    We tested pyrimidine auxotrophs sensitive                  Pyrimidine auxotrophs die relatively rapidly absence a <br /> to cytotoxic pyrimidine analogues Pyr6-52/2 cells             salvageable pyrimidine source (uracil.uridine.29deoxyuridi- <br /> are approximately order magnitude sensitive              ne.cytidine; figure 3), death population <br /> analogues, including 5-fluorouracil 5-fluoro-29deoxyuridine              progressing soon 24 hours. To investigate cause the <br /> (Table 1). The exception 5-fluoroorotic acid,                 rapid cell death examined DNA content Pyr6-52/2 cells <br /> action dependent OPRT OMPDCase,                   grown HMI-9-tmd supplemented 100 mM various <br /> Pyr6-52/2 cells completely impervious 5 mM               pyrimidines, using flow cytometry DNA-binding fluor- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               5                            March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                                   Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 4. Uracil transport T. b. brucei bloodstream forms. (A) Timecourse 0.15 mM [3H]-uracil uptake WT s427 PYR6-52/2 cells, <br /> in presence absence 1 mM unlabelled uracil, indicated. Dashed lines represent linear regression 50 s, yielding correlation <br /> coefficients 0.97 0.92 knockout WT strains, respectively. In presence excess unlabelled uracil uptake significantly <br /> different zero (F-test) strains. The experiment shown representative identical experiments highly similar outcomes and <br /> shows average SE triplicate determinations. In presence 1 mM uridine, lines WT PYR6-52/2 superimposed. (B) Uptake of <br /> [3H]-uracil PYR6-52/2 trypanosomes measured 30 s presence absence unlabelled uracil (#) uridine (&amp;) indicated <br /> concentrations. The data plotted sigmoid curve variable slope case uridine inhibition set zero minimum. <br /> The data average SE triplicate determinations experiment shown representative independent experiments with <br /> essentially identical outcomes. (C) Michaelis-Menten saturation plots uracil uptake [3H]-uracil WT (#) PYR6-52/2 (&amp;) T. b. brucei <br /> bloodstream forms. The data represents average SE identical experiments, performed triplicate. <br /> doi:10.1371/journal.pone.0058034.g004 <br />  <br /> ophore propidium iodide; Pyr6-52/2 cells grown normal HMI-                           chromosomes) (Fig. 6). This profile stable 48 hours <br /> 9 served control. We DNA content control cells                         experiment, proportion G2 phase increased <br /> presented classical distribution cells G1 phase                            somewhat period, probably reflecting mid-log phase <br /> (diploid), small proportion S-phase undergoing DNA synthesis                       growth population end experiment, <br /> and finally percentage population G2 phase (double set                      compared early log phase start (% G2 4.962.8%, <br />                                                                                         11.663.1% 13.961.9% 24 h, 36 48 h, respectively; <br />                                                                                         quantified using ModFit software package). In sharp contrast, <br />  Table 1. EC50 values pyrimidine analogues tested                              rapid increase cells displaying incomplete <br />  WT s427 PYR6-52/2 bloodstream forms grown                                    complement chromosomes Pyr6-52/2 cells grown HMI-9- <br />                                                                                         tmd <br />  standard HMI-9, using standard protocol based                                     , resulting cells fluorescence (i.e. DNA) than <br />  fluorescent indicator dye Alamar Blue.                                                 associated normal G1 phase cells, cells a <br />                                                                                         DNA content G1 G2 phase (Fig. 6). This clearly <br />                                                                                         indicates cells attempting cell division &#226;&#8364;&#732;as normal&#226;&#8364;&#8482; but <br />                              WTs427            PYR6-52/2                                unable complete chromosome synthesis lack of <br />                                                                  RF     P value <br />                                                                                         pyrimidine nucleotides, leading aberrant cells incomplete <br />                              AVG &#194;&#177; SE          AVG &#194;&#177; SE                                 fragmented chromosomes ultimately non-viable. This <br />                              (mM)          n   (mM)     n <br />                                                                                         phenomenon progressed rapidly 48 h live cells be <br />  5-Fluorouracil              35.961.5      4   2.360.07      4   0.06   ,0.001          detected. The cells counted flow cytometer <br />  5-Fluoro-29deoxyuridine     4.660.5       3   0.7760.10     3   0.16   0.002           contained incomplete presumably fragmented <br />  5-Fluoro-29deoxycytidine    43.764.4      3   4.660.9       3   0.105 ,0.001           DNA. Highly similar flow cytometry results obtained when <br />                                                                                         supplementing HMI-9-tmd cytosine, thymine thymidine, <br />  5-Fluorouridine             .5000         5   47263         3   ,0.09 ,0.001 <br />                                                                                         supplementation uracil uridine produced profiles <br />  5-Fluoroorotic acid         13.261.2      3   .5000         4   .380 ,0.001 <br />                                                                                         highly similar control (growth standard HMI-9); addition <br />  RF, resistance factor, ratio EC50 values (mM) knockout       29deoxyuridine, cytidine 29deoxycytidine resulted inter- <br />  WT strains. P value based unpaired Students t-test.                           mediate levels DNA damage 48 h (results shown). In <br />  doi:10.1371/journal.pone.0058034.t001                                                  effort quantify emergence aberrant cells flow <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                          6                             March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                             Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />                                                                                    followed <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>survival</span> parasitaemia 15 days. Figure 8A shows <br />                                                                                    WT trypanosomes virulent killed mice <br />                                                                                    4 8 days. The single allele knockout strain Pyr6-5+/2 <br />                                                                                    caused death mice day 5 animals <br />                                                                                    survived day 12. In contrast, animals inoculated with <br />                                                                                    Pyr6-52/2 survived day 15, experiment was <br />                                                                                    terminated. However, auxotrophs able survive to <br />                                                                                    multiply host, evidenced average parasitaemia, <br />                                                                                    reached similar levels strains albeit much <br />                                                                                    slowly (Fig. 8B). We conclude T. brucei salvage just <br />                                                                                    uracil and/or uridine vitro maintain infection. <br />  <br />                                                                                    Discussion <br />                                                                                       Kinetoplastid parasites able salvage preformed pyrimi- <br />                                                                                    dine nucleobases and/or nucleosides [6,7,15,40,41] as <br />                                                                                    synthesise novo glutamine aspartate [42]. The <br />                                                                                    pathways converge UMP, end-product 6-step <br />                                                                                    biosynthesis pathway nexus salvaged cytidine, <br />                                                                                    uridine, 29dUrd, 29dCtd uracil, actions cytidine <br />                                                                                    deaminase, uridine phosphorylase uracil phosphoribosyltrans- <br /> Figure 5. Comparative expression uridine phosphorylase in <br /> wild-type pyrimidine auxotrophic trypanosomes. Expres-                         ferase (UPRT). From UMP cell make pyrimidine <br /> sion uridine phosphorylase (UP) assessed Real Time PCR                ribonucleotides 29deoxyribonucleotides needs, through <br /> WT PYR6-52/2 strains grown various conditions. The results               non-redundant pathways. Salvaged thymidine utilised as <br /> are presented normalized control (group1) average               thymidine nucleotides produce pyrimidine <br /> and SE 8 replicates. 1. Control: WT grown HMI-9; 2. WT grown 48 h            nucleotides ([24]; reviewed 10 19) procyclic and <br /> in HMI-9-tmd+100 mM uracil; 3. WT grown 48 h HMI-9-tmd+1 mM <br />                                                                                    bloodstream form trypanosomes thymidine poorly <br /> uridine; 4. PYR6-52/2 grown 48 h HMI-9-tmd+100 mM uracil; 5. PYR6- <br /> 52/2 grown 48 h HMI-9-tmd+1 mM uridine; 6. PYR6-52/2long-term                   [16,24]. Therefore clear pyrimidine metabolism in <br /> adapted growth uridine, grown 48 h HMI-9-tmd+100 mM uracil;               protozoa replete good drug targets. Indeed, T. b. <br /> 7. PYR6-52/2long-term adapted growth uridine, grown 48 h                  brucei DHFR-TS, CTP synthetase dUTPase been <br /> HMI-9-tmd+1 mM uridine. Data analysed one-way ANOVA                    shown essential enzymes [20,21,22]. These enzymes <br /> with Tukey&#226;&#8364;&#8482;s correction. Horizontal asterisks indicate significant                 pathways downstream UMP shared by <br /> differences control; vertical asterisks indicate significant differenc- <br /> es individual bars indicated. <br />                                                                                    salvage route biosynthesis route. <br /> doi:10.1371/journal.pone.0058034.g005                                                 What clear biochemical <br />                                                                                    pathways obtain UMP place essential and <br /> cytometry profiles analyzed ModFit software                          potential drug target. In order therapeutically target the <br /> package models peak area. This successful                    salvage pathway UMP necessary inhibit the <br /> the 48-h time points lack viable cells                   uptake uracil, uridine, 29deoxyuridine, cytidine 29deoxy- <br /> extensive DNA damage, did allow reliable estimates                    cytidine, inhibit UPRT. With regards option it <br /> relevant peaks. However, results 24-h 36-h time                   noted cytidine 29deoxycytidine incorpo- <br /> points shown figure 7. The use thymidine sole                         rated poorly T. b. brucei nucleotide pool [21,24] it <br />                                                                                    necessary inhibit carriers uracil, <br /> pyrimidine source caused highly significant increase cells in <br />                                                                                    29deoxyuridine uridine, recently reported all <br /> G2 phase, possibly anticipated imbalance between <br />                                                                                    mediated transporter, TbU3, bloodstream <br /> thymidine nucleotides deoxycytidine nucleosides, the <br />                                                                                    forms [24]. However, report WT trypanosomes (i.e. <br /> cell generate thymidine. The peak classified &#226;&#8364;&#732;DNA <br />                                                                                    pyrimidine prototrophs) grow unimpeded absence of <br /> debris&#226;&#8364;&#8482; increased 24 h pyrimidine-free conditions and <br />                                                                                    pyrimidine source conclude pyrimidine <br /> this highly significant (P,0.01) 36 h; debris amount <br />                                                                                    transporters UPRT essential functions bloodstream <br /> was significantly increased culturing thymidine (Fig. 7). <br />                                                                                    form T. b. brucei - consistent recent report deletion of <br /> We conclude significant interruption pyrimidine <br />                                                                                    UPRT L. donovani promastigotes created pyrimidine proto- <br /> nucleotide availability leads major defects DNA synthesis. <br />                                                                                    trophic parasite normal vitro growth [43]. It be <br />                                                                                    concluded pyrimidine salvage essential function for <br /> Infectivity pyrimidine auxotrophic T. b. brucei                           trypanosomes. <br /> mice                                                                                  Arakaki et al [25] previously investigated novo <br />    The observation (Fig. 3) pyrimidine auxotrophic Pyr6-52/2                  biosynthesis route UMP essential T. b. brucei vitro; they <br /> cells grow standard HMI-9, contains thymidine                   employed RNA-interference (RNAi) reduce expression T. <br /> pyrimidine source, grow thymidine-supplemented                       brucei dihydroorotate dehydrogenase (DHODH). These authors <br /> medium dialyzed FBS strongly suggest (1) T. b. brucei                    reported knockdown enzyme did affect growth in <br /> cannot use thymidine sole pyrimidine source (2) able              standard HMI-9 greatly reduced growth pyrimidine- <br /> to salvage sufficient amounts pyrimidines non-                   depleted medium using commercial dialysed serum. Our own <br /> dialyzed serum. It speculated pyrimidine                        observations PYR6-52/2 strain entirely consistent with <br /> auxotrophic trypanosomes able survive vivo. To test                Arakaki&#226;&#8364;&#8482;s report: growth rate pyrimidine auxotrophs at <br /> this, infected groups 6 mice high inoculum 105                     slightly affected normal medium non-dialysed serum. <br /> trypanosomes s427-WT, Pyr6-5+/2 Pyr6-52/2 strains                        As shown Supplementary data, thymidine (,83 mM) the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                     7                            March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                         Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 6. Flow cytometry DNA content bloodstream form PYR6-52/2 cells. Pyrimidine auxotrophic trypanosomes either <br /> incubated standard HMI-9 or, parallel, pyrimidine-free HMI-9-tmd 48 h, stained propidium iodide prepared flow <br /> cytometric analysis. Whereas control cultures classical distribution cells G1, S G2 phase, small percentage cells with <br /> less normal diploid DNA content (debris, d), cells grown pyrimidine-free medium showed higher percentage cells partial <br /> DNA content, increased dramatically 24 48 h. <br /> doi:10.1371/journal.pone.0058034.g006 <br />  <br /> only pyrimidine added medium nucleoside                obligate intracellular protozoan, showed parasites were <br /> cannot converted uridine cytidine nucleotides T. b.              completely avirulent immunocompromised mice [45] - a <br /> brucei, redundant [24,25] clearly serum provides                phenotype attributed lack free pyrimidines animal <br /> sufficient pyrimidines growth, consistent average                cells prevents growth pyrimidine auxotrophic <br /> uracil concentration 0.1760.05 mM human plasma [44]                 bacteria [46]. <br /> high affinity uptake pyrimidines, particularly uracil, T. brucei           Inhibition pyrimidine biosynthesis pathway T. b. brucei <br /> [23,24]. We conclude pyrimidine biosynthesis                 greatly sensitises trypanosomes cytotoxic pyrimidine <br /> essential vitro growth, fact 10% FBS                 analogues 5-fluorouracil, 5-fluoro-29deoxyuridine, 5- <br /> supplies sufficient pyrimidines, indicate         fluorodeoxycytidine 5-fluorouridine (Table 1). The enhanced <br /> essential vivo growth either.                                          effect 5-fluorouracil noted Arakaki et al [25], who <br />    This tested infecting mice s427-WT, PYR6-5+/2                  attributed inhibition uracil uptake. Whilst analogue is <br /> and PYR6-52/2 trypanosomes. All strains able                    competitive inhibitor uracil transport T. brucei, none <br /> maintain infection homozygous knockout                    fluorinated pyrimidines sufficiently inhibit uracil <br /> strain clearly virulent, unambiguously establishes           uptake bloodstream forms EC50 values given Table 1, <br /> inhibition novo pyrimidine biosynthesis viable             especially 5F-29deoxyuridine 5F-29deoxycytidine [24]. As <br /> therapeutic strategy African trypanosomes. These findings             alternative explanation, propose fluorinated <br /> are similar reported promastigote L. donovani [28]          pyrimidines enter trypanosomes prodrugs subversive <br /> but, contrast authors report, contend           substrates pyrimidine salvage enzymes, converted to <br /> fact disruption pyrimidine biosynthesis compensated            nucleotides incorporated nucleic acids, the <br /> for physiological levels pyrimidines demonstrates             case mammalian cells [47] recently shown 5- <br /> pathway essential kinetoplastids, viable drug             fluorouracil T. b. brucei [24]. This incorporation efficient <br /> target. Indeed, authors recently demonstrated              absence newly synthesised pool pyrimidine <br /> L. donovani lack carbamoyl phosphate synthetase,            metabolites compete halogenated analogues <br /> pyrimidine auxotrophic, able establish &#226;&#8364;&#732;robust&#226;&#8364;&#8482; infection           level enzyme RNA and/or DNA <br /> in mice [43]. It noteworthy, however, Leishmania species          polymerases. <br /> obligated intracellular parasites manuscript          We conclude inhibitor enzymes of <br /> assessment vivo growth extracellular pyrimidine-                  novo pathway inhibitor uracil/ <br /> auxotrophic protozoan. This relevant intracellular              uridine uptake, cytotoxic nucleoside analogue, be <br /> extracellular nucleoside nucleobase levels potentially           powerful synergistic combination act on <br /> different, intracellular purines pyrimidines over-               trypanosomes misincorporation false nucleotides, <br /> whelmingly existing nucleotides, taken                  causing pyrimidine starvation inhibition of <br /> directly protozoan transporters [6]. In addition, previous               pyrimidine carriers, trypanosome populations <br /> report pyrimidine-auxotrophic Toxoplasma gondii,                   die quickly lack pyrimidines lack <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                8                             March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                        Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 7. Quantitative analysis DNA content pyrimidine-starved trypanosomes. The peak area G1, G2 debris flow <br /> cytometric analysis DNA content (Fig. 6) determined using ModFit software package 24 36 growth various culturing <br /> conditions. Growth HMI-9 (control) HMI-9-tmd addition 100 mM pyrimidine indicated. The data the <br /> average 3&#226;&#8364;&#8220;6 independent determinations statistical analysis performed using one-way ANOVA Tukey&#226;&#8364;&#8482;s correction (Prism 5, <br /> GraphPad). *, P,0.05; **, P,0.01. <br /> doi:10.1371/journal.pone.0058034.g007 <br />  <br />  <br />  <br />  <br /> Figure 8. Infectivity pyrimidine auxotrophic T. b. brucei. (A) Survival mice groups 6, inoculated 105 bloodstream form <br /> trypanosomes various clonal lines. (B) Parasitaemia mice depicted survival panel A. The average parasitaemia surviving <br /> mice shown. Detection phase-contrast microscopy detection limit 16104; infected sub-patent, value 5000 cells <br /> ml21 inserted order arrive reasonable average. Both panels: #, WT s427; &amp;, PYR6-5+/2; m, PYR6-52/2. <br /> doi:10.1371/journal.pone.0058034.g008 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                9                             March 2013 | Volume 8 | Issue 3 | e58034 <br />                                                                                                                               Pyrimidine Auxotrophy Trypanosoma brucei <br />  <br />  <br /> of purines [38]. It easily speculated kinetoplastid                                Supporting Information <br /> parasites, having evolved capacity synthesise their <br /> own purines, relatively well-adapted periods                                   Figure S1 Growth PYR6-52/2 T. b. brucei blood- <br /> relatively low purine availability, demonstrated                                     stream forms standard HMI-9 s427-WT HMI- <br /> reversibility purine starvation-induced growth arrest [38]. In                              9-tmd supplemented 10% dialysed FBS. Seeding <br /> contrast, needed develop mechanism cope                                  density 16105 cells ml21 cells manually counted <br /> with prolonged dearth pyrimidines, able make                                     24 h. On day 3 cells passaged relevant fresh <br />                                                                                                medium, 16105 cells ml21. <br /> sufficient amounts themselves, trypanosomes therefore <br />                                                                                                (TIF) <br /> unable recover short periods pyrimidine <br /> starvation. We did observe, consistently, increased expression                              Table S1 Composition standard HMI-9 medium. <br /> uridine phosphorylase, increase uracil uptake capacity                               (DOCX) <br /> (both two-fold), pyrimidine-starved trypanosomes                             Text S1 <br /> hardly constitutes major upregulation pyrimidine salvage                              (DOCX) <br /> pathway best uncertain regulated, <br /> physiological response low pyrimidine levels. Indeed, lack of <br />                                                                                                Acknowledgments <br /> a regulated response insufficient level pyrimidine <br /> nucleotides manifest major defects DNA synthesis                                 The authors like thank Prof. George A. M. Cross (Rockefeller <br /> after 24 h, leading fragmented incomplete chromo-                                  University, New York, NY, USA) generous donation PYR6-5 <br />                                                                                                replacement cassette. The authors thank Mr Abdulsalam Alkhaldi <br /> somes. <br />                                                                                                (University Glasgow) expert technical assistance. <br />    In summary, shown pyrimidine uptake or <br /> de novo biosynthesis essential African trypanosomes a <br /> drug combination targeting systems powerful <br />                                                                                                Author Contributions <br /> approach novel therapeutic approaches kinetoplastid                                 Conceived designed experiments: JCM HdK. Performed the <br /> parasites.                                                                                     experiments: JAMA DNAT AD. Analyzed data: LJM HPdK. Wrote <br />                                                                                                paper: HPdK. <br />  <br /> References <br />  1. Brun R, Blum J, Chappuis F, Burri C (2010) Human African trypanosomiasis.                  17. Van Dyke K, Tremblay GC, Lantz CH, Szustkiewicz C (1970) The source of <br />     Lancet 375: 148&#226;&#8364;&#8220;159.                                                                           purines pyrimidines Plasmodium berghei. Am J Trop Med Hyg 19: 202&#226;&#8364;&#8220; <br />  2. Brun R, Schumacher R, Schmid C, Kunz C, Burri C (2001) The phenomenon                          208. <br />     treatment failure Human African Trypanosomiasis. Trop Med Int Health                 18. Wang CC, Cheng HW (1984) Salvage pyrimidine nucleosides Trichomonas <br />     6: 906&#226;&#8364;&#8220;914.                                                                                    vaginalis. Mol Biochem Parasitol 10: 171&#226;&#8364;&#8220;184. <br />  3. Balasegaram M, Young H, Chappuis F, Priotto G, Raguenaud ME, et al. (2009)                 19. Hassan HF, Coombs GH (1988) Purine pyrimidine metabolism parasitic <br />     Effectiveness melarsoprol eflornithine first-line regimens gambiense             protozoa. FEMS Microbiol Rev 4: 47&#226;&#8364;&#8220;83. <br />     sleeping sickness Me&#194;&#180;decins Sans Frontie`res programmes. Trans R Soc               20. Sienkiewicz N, Jaros&#197;&#8218;awski S, Wyllie S, Fairlamb AH (2008). Chemical and <br />     Trop Med Hyg 103: 280&#226;&#8364;&#8220;290.                                                                     genetic validation dihydrofolate reductase&#226;&#8364;&#8220;thymidylate synthase drug <br />  4. Priotto G, Kasparian S, Mutombo W, Ngouama D, Ghorashian S, Arnold U, et                       target African trypanosomes. Mol Microbiol 69: 520&#226;&#8364;&#8220;533. <br />     al. (2009) Nifurtimox-eflornithine combination therapy second-stage African            21. Hofer A, Steverding D, Chabes A, Brun R, Thelander L (2001) Trypanosoma <br />     Trypanosoma brucei gambiense trypanosomiasis: multicentre, randomised, phase                 brucei CTP synthetase: target treatment African sleeping sickness. <br />     III, non-inferiority trial. Lancet 374: 56&#226;&#8364;&#8220;64.                                                 Proc Natl Acad Sci USA 98: 6412&#226;&#8364;&#8220;6416. <br />  5. Martin DW (1981) Metabolism purine pyrimidine nucleotides. In: Martin               22. Castillo-Acosta VM, Este&#194;&#180;vez AM, Vidal AE, Ruiz-Perez LM, Gonza&#194;&#180;lez- <br />     DW, Mayers PA, Rodwell VW, editors. Harper&#226;&#8364;&#8482;s Review Biochemistry:                           Pacanowska D (2008) Depletion dimeric all-a dUTPase induces DNA strand <br />     Lange Medical Publications, California. pp. 331&#226;&#8364;&#8220;348.                                           breaks impairs cell cycle progression Trypanosoma brucei. Int J Biochem Cell <br />  6. De Koning HP, Bridges DJ, Burchmore RJ (2005) Purine pyrimidine                            Biol 40: 2901&#226;&#8364;&#8220;2913. <br />     transport pathogenic protozoa: biology therapy. FEMS Microbiol Rev              23. De Koning HP, Jarvis SM (1998) A highly selective, high-affinity transporter for <br />     29: 987&#226;&#8364;&#8220;1020.                                                                                  uracil Trypanosoma brucei brucei: evidence proton-dependent transport. <br />  7. Landfear SM, Ullman B, Carter NS, Sanchez MA (2004) Nucleoside                             Biochem Cell Biol 76: 853&#226;&#8364;&#8220;858. <br />     nucleobase transporters parasitic protozoa. Eukaryot Cell 3: 245&#226;&#8364;&#8220;254. <br />                                                                                                24. Ali JA, Creek DJ, Burgess K, Allison HC, Field MC, et al. (2013) Pyrimidine <br />  8. De Koning HP, Jarvis SM (1997) Purine nucleobase transport bloodstream <br />                                                                                                    salvage Trypanosoma brucei bloodstream forms trypanocidal action of <br />     forms Trypanosoma brucei brucei mediated novel transporters. Mol <br />                                                                                                    halogenated pyrimidines. Mol Pharmacol. 2012 Nov 27. [Epub ahead print] <br />     Biochem Parasitol 89: 245&#226;&#8364;&#8220;258. <br />                                                                                                    PMID: 23188714 <br />  9. Al-Salabi MI, Wallace LJ, Lu&#194;&#168;scher A, Ma&#194;&#168;ser P, Candlish D, et al. (2007) <br />                                                                                                25. Arakaki TL, Buckner FS, Gillespie JR, Malmquist NA, Phillips MA, et al. (2008) <br />     Molecular interactions underlying unusually high adenosine affinity a <br />     novel Trypanosoma brucei nucleoside transporter. Mol Pharmacol 71: 921&#226;&#8364;&#8220;929.                    Characterization Trypanosoma brucei dihydroorotate dehydrogenase a <br /> 10. Berens RL, Krug EC, Marr JJ (1995) Purine pyrimidine metabolism. In:                       possible drug target; structural, kinetic RNAi studies. Mol Microbiol 68: <br />     Marr JJ, Muller M, editors. Biochemistry Molecular biology parasites.                   37&#226;&#8364;&#8220;50. <br />     London: Academic Press. pp. 89&#226;&#8364;&#8220;117.                                                        26. Gao G, Nara T, Nakajima-Shimada J, Aoki T (1999) Novel organisation and <br /> 11. Berg M, Kohl L, Van d <br /> </body></html>