<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Structure of Mth11&#229;&#8230;&#190;Mth Rpp29, an essential protein <br /> subunit of archaeal and eukaryotic RNase P <br />  <br /> We have determined the solution structure of Mth11 (Mth Rpp29),              (Fig. 1). The eukaryotic homologs of the protein contain, in <br /> an essential subunit of the RNase P enzyme <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> the archaebac-               addition <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> the conserved domain, a domain at the N terminus <br /> terium Methanothermobacter thermoautotrophicus (Mth). RNase                  that includes a nucleolar targeting signal and may have other <br /> P is a ubiquitous ribonucleoprotein enzyme primarily responsible             functions that are dispensable in an organism lacking a nucleus. <br /> for cleaving the 5&#226;&#180;&#8226; leader sequence during maturation of tRNAs in            The essentiality of Mth Rpp29 to RNase P activity was confirmed <br /> all three domains of life. In eubacteria, this enzyme is made up of          in an in vitro reconstitution assay. <br /> two subunits: a large RNA (&#226;&#172;&#8225;120 kDa) responsible for mediating                  To provide insights into the role of Mth Rpp29 in RNase P <br /> catalysis, and a small protein cofactor (&#226;&#172;&#8225;15 kDa) that modulates             assembly, we examined the interaction of the protein with the full <br /> substrate recognition and is required for efficient in vivo catalysis.       314-nt Mth RNA subunit by using NMR chemical shift pertur- <br /> In contrast, multiple proteins are associated with eukaryotic and            bation. The data have allowed us to identify residues in the <br /> archaeal RNase P, and these proteins exhibit no recognizable                 protein that interact with the nucleic acid. These data are a <br /> homology to the conserved bacterial protein subunit. In reconsti-            critical step toward developing a structural model of the complex <br /> tution experiments with recombinantly expressed and purified                 RNase P holoenzyme from Archaea and higher organisms. <br /> protein subunits, we found that Mth Rpp29, a homolog of the <br /> Rpp29 protein subunit from eukaryotic RNase P, is an essential               Materials and Methods <br /> protein component of the archaeal holoenzyme. Consistent with                Protein Expression and Purification. Escherichia coli BL21(DE3) <br /> its role in mediating protein&#226;&#8364;&#8220;RNA interactions, we report that Mth           cells were cotransformed with Mth11&#229;&#8230;&#190;pET-15b and pARG-U <br /> Rpp29 is a member of the oligonucleotide&#229;&#8230;&#190;oligosaccharide binding             (20, 21) plasmids and grown at 37&#194;&#176;C in minimal M9 media (22) <br /> fold family. In addition to a structured &#226;&#65533;&#164;-barrel core, it possesses         containing 1 g&#229;&#8230;&#190;liter 15N ammonium chloride and 2 g&#229;&#8230;&#190;liter 13C <br /> unstructured N- and C-terminal extensions bearing several highly             glucose as the nitrogen and carbon sources, respectively, sup- <br /> conserved amino acid residues. To identify possible RNA contacts             plemented with 30 &#226;&#65533;&#174;g&#229;&#8230;&#190;liter kanamycin, 50 &#226;&#65533;&#174;g&#229;&#8230;&#190;liter carbenicillin, <br /> in the protein&#226;&#8364;&#8220;RNA complex, we examined the interaction of the               and Eagle Basal Vitamin Mix (Life Technologies, Gaithersburg, <br /> 11-kDa protein with the full 100-kDa Mth RNA subunit by using                MD). The cells were induced by addition of 1 mM isopropyl <br /> NMR chemical shift perturbation. Our findings represent a critical           &#226;&#65533;&#164;-D-thiogalactoside when the cells reached OD600 &#226;&#172;&#8225;0.6 (mid-log <br /> step toward a structural model of the RNase P holoenzyme from                phase), harvested after 4 h by centrifugation (10 min, 5,000 &#226;&#171;&#187; g, <br /> archaebacteria and higher organisms.                                         4&#194;&#176;C), and stored at &#226;&#171;&#186;20&#194;&#176;C. Thawed cells were suspended in 30 <br />                                                                              ml of buffer A (50 mM potassium phosphate, pH 7.0&#229;&#8230;&#190;0.5 M KCl) <br />                                                                              and lysed on ice by sonication, and cell debris was pelleted by <br /> R    ibonuclease (RNase) P is an essential and ubiquitous en- <br />      zyme primarily responsible for cleaving the 5&#226;&#172;&#732; leader se- <br /> quence during maturation of tRNAs in all three domains of life <br />                                                                              centrifugation (30 min at 20,000 &#226;&#171;&#187; g, 4&#194;&#176;C). Mth Rpp29 was <br />                                                                              purified as reported (23) with the following modifications: the <br /> (1, 2). In bacteria, this ribonucleoprotein enzyme is made up of             (His)6-affinity tag was removed by incubation at 37&#194;&#176;C with <br /> a large RNA subunit (&#226;&#172;&#8225;120 kDa) responsible for mediating                     thrombin (10 units&#229;&#8230;&#190;mg of Mth Rpp29) for at least 12 h, resulting <br /> catalysis (3) and a small protein cofactor (&#226;&#172;&#8225;15 kDa) that                    in a 97-residue protein consisting of four plasmid-encoded <br /> modulates substrate recognition (4&#226;&#8364;&#8220;6) and is required for effi-              residues (Gly-Ser-His-Met) followed by the 93 natively encoded <br /> cient in vivo catalysis (7). In higher organisms, RNase P is made            residues. After cleavage of the affinity tag, the protein was <br /> up of one RNA and several protein subunits (8&#226;&#8364;&#8220;12).                           purified by C4 reversed-phase HPLC chromatography (Vydac <br />    Despite significant similarity between the RNA subunits of                214TP1010, 1 &#226;&#171;&#187; 25 cm; 180-ml linear gradient of 35&#226;&#8364;&#8220;50% <br /> bacterial and archaeal RNase P (13), database mining of the                  acetonitrile, 0.1% trifluoroacetic acid). Pooled fractions were <br /> available archaeal genomes has failed to identify a protein                  lyophilized and refolded by dissolving in NMR buffer (50 mM <br /> homologous to the bacterial RNase P protein. Instead, analysis               potassium phosphate, pH 7&#229;&#8230;&#190;50 mM KCl&#229;&#8230;&#190;0.02% sodium azide&#229;&#8230;&#190; <br /> of purified RNase P from Methanothermobacter thermoautotro-                  10% D2O) to a concentration of &#226;&#172;&#8225;4 mg&#229;&#8230;&#190;ml and then concen- <br /> phicus (Mth) identified four proteins (Mth11, Mth1618, Mth687,               trated by ultra filtration (Centriprep-3, Amicon). The Mth <br /> and Mth688) that are homologous to those of eukaryotic nuclear               Rpp30 (Mth688), Mth Rpp21 (Mth1618), and Pfu Pop5 (Pf1378) <br /> RNase P (14): Rpp29, Rpp21, Pop5, and Rpp30, respectively.                   proteins were obtained in a similar manner (details to be <br /> These four proteins were subsequently shown to be those                      presented elsewhere). Sample purity was assayed by SDS&#229;&#8230;&#190;PAGE <br /> required for in vitro reconstitution of archaeal RNase P. Using <br /> bacterially expressed, purified proteins and in vitro-transcribed P          This work was presented in part at the Keystone Symposium on NMR in Molecular Biology, <br /> RNA from the hyperthermophile, Pyrococcus horikoshii, precur-                February 4 &#226;&#8364;&#8220;10, 2003, Taos, NM. <br /> sor tRNA (ptRNA)-processing activity could be recovered by                   This paper was submitted directly (Track II) to the PNAS office. <br /> the addition of these four proteins, whereas no activity was                 Data deposition: The atomic coordinates have been deposited in the Protein Data Bank, <br /> observed upon addition of any single protein subunit (15).                   www.rcsb.org (PDB ID <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> 1OQK). <br />    We report here the solution structure of Mth11&#229;&#8230;&#190;Mth Rpp29,                 Abbreviations: Mth, Methanothermobacter thermoautotrophicus; OB-fold, oligonucleo- <br /> the homolog of the human RNase P protein Rpp29 (16, 17),                     tide&#229;&#8230;&#190;oligosaccharide-binding fold; ptRNA, precursor tRNA; HSQC, heteronuclear single <br /> encoded by Mth ORF 0011 (14, 18, 19). Mth Rpp29 is a small                   quantum correlation; NOE, nuclear Overhauser effect. <br /> 93-residue protein containing a domain (Pfam UPF0086) that is                &#194;&#182;To   whom correspondence should be addressed. E-mail: foster.281@osu.edu. <br /> highly conserved both within and between Archaea and Eukarya                 &#194;&#169; 2003 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> 15398 &#226;&#8364;&#8220;15403 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; December 23, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 26                                                     www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.2535887100 <br />  Fig. 1. Sequence alignment of Mth Rpp29 with Rpp29 sequences from <br /> Archaea and Eukarya. Alignment was generated with CLUSTALW (57) and <br /> colored according to similarity by using a Risler scoring matrix (58); shaded <br /> residues indicate identity, and boxed residues indicate a global similarity score <br /> &#226;&#172;&#381;0.7. Secondary structural features observed in the structure ensemble are <br /> indicated. Arrows indicate the core of the protein protected from trypsin <br /> digestion as identified by electrospray-MS (9,114 Da expected; 9,115 Da <br /> observed). <br />  <br />  <br />  <br /> with Coomassie blue staining and by electrospray-MS. Protein <br /> concentration was estimated by Bradford assay (Coomassie Plus <br /> Protein Assay Reagent, Pierce). <br />  <br /> Limited Proteolysis. Purified, refolded Mth Rpp29 was subjected <br /> to limited proteolysis by adding 5% trypsin (wt&#229;&#8230;&#190;wt) to a protein <br /> solution (&#226;&#172;&#8225;100 &#226;&#65533;&#174;M in NMR buffer) and incubating at room <br /> temperature for intervals up to 30 min, quenched with 0.6 mM <br /> PMSF, followed by electrospray mass analysis (Q-TOF-II, Mi- <br /> cromass, Manchester, U.K.). <br />  <br /> Transcription of Mth P RNA. The gene encoding Mth P RNA [Fig. <br /> 2a, 309 nt: 1,173,861 to 1,174,169 of gi&#226;&#177;&#65533;15678031 (16, 24)] was <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        BIOCHEMISTRY <br /> obtained by PCR with the Mth genomic DNA as the template <br /> and the appropriate gene-specific primers (forward: <br /> CGGGATCCTAATACGACTCACTATA*ggACCGGGCAAG <br /> CCGA AGGGC; reverse: GGaattcACCGGGCATGC- <br /> CGAGAGTAAC; BamHI and EcoRI restriction sites are ital- <br /> icized, the T7 promoter is underlined, the asterisk denotes the <br /> transcription start site, and nucleotides in lowercase are intro- <br /> duced by primers). The PCR product containing the gene, placed <br /> under control of a bacteriophage T7 RNA polymerase promoter, <br /> was then subcloned into pUC19. DNA sequencing confirmed the                         Fig. 2. In vitro reconstitution of Mth RNase P activity. (a) Sequence and <br /> correct sequence of the full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> P RNA gene. Mth P RNA was                       phylogenetically predicted secondary structure of the Mth RNase P RNA <br />                                                                                     subunit (24). (b) Reconstitution experiment shows Mth Rpp29 is essential for <br /> generated by in vitro runoff transcription with T7 RNA poly- <br />                                                                                     effective Mth RNase P activity against E. coli ptRNATyr. The negative control is <br /> merase (purified in-house) after linearization of the plasmid with                  the substrate alone; positive control is the substrate incubated with E. coli <br /> EcoRI. In vitro-transcribed RNA (316 nt) was refolded in                            RNase P; Pfu Pop5 was used in place of Mth Pop5. <br /> binding buffer (10 mM Hepes, pH 7.5&#229;&#8230;&#190;10 mM MgOAc&#229;&#8230;&#190;400 mM <br /> NH4OAc&#229;&#8230;&#190;5% glycerol&#229;&#8230;&#190;0.01% Nonidet P-40), while verifying the <br /> proper fold by monitoring the ability of the RNA to cleave                          Pyroccocus furiosus (Pf1378; see Fig. 1b) was used in place of Mth <br /> ptRNATyr under high salt conditions (25).                                           Pop5 because of presumed problems with solubility and folding <br />                                                                                     in the Mth homolog. All proteins were purified to homogeneity <br /> In Vitro Reconstitution. Reconstitution experiments were per-                       (&#226;&#172;&#381;95%) and verified to lack independent ptRNA processing <br /> formed by incubating Mth P RNA (0.1 &#226;&#65533;&#174;M) with the protein                            activity. <br /> subunits (3 &#226;&#65533;&#174;M) in 10 &#226;&#65533;&#174;l of buffer [50 mM Tris&#228;&#161;&#160;HCl, pH 7.6&#229;&#8230;&#190;10 <br /> mM MgCl2&#229;&#8230;&#190;800 mM NH4Cl&#229;&#8230;&#190;0.025 units&#229;&#8230;&#190;&#226;&#65533;&#174;l RNasin (Invitro-                               NMR Spectroscopy. NMR spectra were recorded at 25&#194;&#176;C on 600- <br /> gen), 37&#194;&#176;C, 5 min]. Activity was assayed by adding 0.02 pmol of                     and 800-MHz Bruker (Billerica, MA) Avance DMX and DRX <br /> E. coli ptRNAtyr internally labeled with [&#226;&#65533;&#163;-32P]GTP (2 nM) and                      spectrometers equipped with triple-resonance probes and triple- <br /> incubating for 3.5 h. The reactions were quenched with urea,                        axis pulsed-field gradients. NMR data processing and analysis <br /> electrophoresed on an 8% (wt&#229;&#8230;&#190;vol) polyacrylamide&#229;&#8230;&#190;7 M urea                           were performed with NMRPIPE (26) and NMRVIEW (27). NMR <br /> gel, and analyzed by autoradiography. The Pop5 homolog from                         samples typically contained 1 mM Mth Rpp29 and 50 mM <br />  <br /> Boomershine et al.                                                                                    PNAS &#229;&#8230;&#169; December 23, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 26 &#229;&#8230;&#169; 15399 <br />  Table 1. Structure refinement statistics <br /> NMR constraints                                             1,247 <br />   Distance                                                  1,059 <br />     Intraresidue                                              488 <br />     Sequential (i, i&#226;&#171;&#185;1)                                       207 <br />     Medium range (i, j &#226;&#177;&#8226; 4)                                    41 <br />     Long range (i, j &#226;&#172;&#381; 5)                                     209 <br />     Ambiguous                                                 114 <br />     (Restraints per residue: 11) <br />   Torsion (&#226;&#65533;&#190;, &#226;&#65533;&#186;)                                              162 <br />   H bond                                                       26 <br /> Violations (20 SA structures; no NOE <br />     violations &#226;&#172;&#381; 0.5 &#195;&#8230;) <br />   NOE violations (&#226;&#172;&#381; 0.2 &#195;&#8230;) <br />     Number                                               1.91 &#226;&#171;&#190; 0.684 <br />     RMSD, &#195;&#8230;                                            0.0187 &#226;&#171;&#190; 0.0009 <br />   Torsion violations <br />     Number &#226;&#172;&#381; 5 deg                                       0.55 &#226;&#171;&#190; 0.80 <br />     RMSD, deg                                            0.66 &#226;&#171;&#190; 0.15 <br />   Deviation from ideal geometry <br />     Bond length, &#195;&#8230;                                     0.0025 &#226;&#171;&#190; 0.00028 <br />     Bond angle, deg                                    0.4121 &#226;&#171;&#190; 0.0350 <br /> RMSD from mean structure (&#195;&#8230;) <br />     (residues 12&#226;&#8364;&#8220;80) <br />   Backbone                                                   0.55 <br />   Heavy atoms                                                1.26 <br /> Ramachandran analysis, % <br />   Most favored region                                       87.3 <br />   Additionally allowed region                               12.7 <br />  <br />   RMSD, rms deviation. <br />  <br />  <br />  <br /> potassium phosphate (pH 7), 50 mM KCl, and 90% H2O&#229;&#8230;&#190;10% <br /> D2O or 100% D2O. Backbone assignments were made by using <br /> standard triple resonance spectra [HNCACB, CBCA(CO)NH, <br /> and HNCO] (28, 29) recorded at 600 MHz. Side-chain 1H and <br /> 13C resonance assignments were obtained from several 3D                   Fig. 3. NMR spectra of Mth Rpp29. (a) 15N HSQC spectrum with backbone <br /> spectra: 15N-separated total correlation spectroscopy (TOCSY)             amide assignments indicated. (b) Heteronuclear {1H}-15N NOE data of Mth <br /> (800 MHz, 40 ms &#226;&#65533;&#182;m, 9.2 kHz nominal B1 field), CCONH-                     Rpp29 indicates the N- and C-terminal residues are highly flexible. The trypsin- <br /> TOCSY (600 MHz, &#226;&#65533;&#182;m 13 ms, 8.3 kHz nominal B1 field), 3D                   resistant core consists of residues 6 &#226;&#8364;&#8220; 85. <br /> HCCH-COSY (600 MHz), and HCCH-TOCSY (600 MHz, &#226;&#65533;&#182;m 12 <br /> ms, 8.3 kHz nominal B1 field). Interproton distance restraints <br />                                                                           proton chemical shifts. A final set of 20 refined structures that <br /> were obtained from assignment of nuclear Overhauser effect <br />                                                                           met the input restraints were selected for analysis; statistics <br /> (NOE) cross peaks in 3D 15N-separated NOESY (800 MHz, &#226;&#65533;&#182;m <br />                                                                           (Table 1) were calculated with PROCHECK-NMR (33). <br /> 100 ms) and 13C-separated NOESY spectra (&#226;&#65533;&#182;m 150 ms). Two <br /> 13C-separated NOESY spectra were recorded, one in 90% <br />                                                                           NMR of the Mth Rpp29&#229;&#8230;&#190;P RNA Complex. Interactions between Mth <br /> H2O&#229;&#8230;&#190;10% D2O (800 MHz) and one in 100% D2O (600 MHz). <br />                                                                           Rpp29 and Mth P RNA were examined by using 15N-edited <br />    Heteronuclear {1H}-15N NOE data (30) were obtained by <br /> recording spectra in an interleaved manner with (NOE) and                 heteronuclear single quantum correlation (HSQC) spectra (800 <br /> without (no NOE) 1H saturation achieved via a train of 120&#194;&#176;               MHz) at a 1:1 ratio of protein to RNA. The complex was <br /> pulses (14 kHz bandwidth) applied every 18 msec for 5 sec.                assembled by adding highly concentrated RNA (100 &#226;&#65533;&#174;l; 600 &#226;&#65533;&#174;M) <br /> Heteronuclear NOE values were determined from the ratios of               to a protein solution (200 &#226;&#65533;&#174;l; 300 &#226;&#65533;&#174;M), mixed by pipetting, and <br /> the peak intensities between the two spectra.                             transferred to a reduced volume NMR tube (Shigemi, Allison <br />                                                                           Park, PA); final protein and RNA concentrations were 200 &#226;&#65533;&#174;M. <br /> Mth Rpp29 Structure Determination. The structure of the Mth               Initial spectra recorded under these conditions (low salt, 25&#194;&#176;C) <br /> Rpp29 protein was determined by using the simulated annealing             were excessively broadened by nonspecific interactions; im- <br /> and energy minimization protocols within the CNS program suite            proved spectra were obtained by addition of 1 M KCl to a final <br /> (31), which identified a full range of structures that are consistent     concentration of 400 mM, and increasing the temperature to <br /> with the distance and angle restraints derived from the NMR               50&#194;&#176;C. For reference, a spectrum of free protein was recorded in <br /> data. Distance constraints were derived from the intensities of           the same buffer and temperature. Backbone amide assignments <br /> cross peaks from multidimensional NOE spectra (&#226;&#65533;&#182;m 100&#226;&#8364;&#8220;150                 at 50&#194;&#176;C were obtained by recording 15N HSQC spectra at 5&#194;&#176; <br /> ms). Peak intensities were placed into one of three categories:           intervals. Weighted average shift perturbations were calculated <br /> strong (&#226;&#172;&#65533;2.8 &#195;&#8230;), medium (2.8&#226;&#8364;&#8220;3.4 &#195;&#8230;), and weak (3.4&#226;&#8364;&#8220;5 &#195;&#8230;).                  as (((&#226;&#65533;&#166;&#226;&#338;&#172;H)2 &#226;&#171;&#185; (&#226;&#65533;&#166;&#226;&#338;&#172;N&#229;&#8230;&#190;5)2)&#229;&#8230;&#190;2)0.5 (34). <br /> Backbone torsion angle restraints for phi and psi torsion angles <br /> were obtained from analysis of the backbone chemical shifts with          Results <br /> TALOS (32). Hydrogen bond restraints were obtained from                   Mth Rpp29 Is an Essential Subunit of Mth RNase P. To test whether <br /> analysis of NOEs and the temperature dependence of amide                  Mth Rpp29 is essential to RNase P activity, Mth P RNA was <br />  <br /> 15400 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.2535887100                                                                                   Boomershine et al. <br />  Fig. 4. Mth Rpp29 solution structure. (a) Ensemble of 20 low-energy structures of Mth Rpp29 superposed on the backbone heavy atoms of residues 12&#226;&#8364;&#8220; 82; for <br /> clarity, only the main-chain atoms (N, C&#226;&#65533;&#163;, and C&#226;&#172;&#732;) of the structured core residues are shown. Strands are blue, turns and coil regions are gray, and the C-terminal <br /> helix is red. (b) Close-up showing a potential salt bridge between the conserved Lys-56 and Glu-37 residues; hydrophobic residues in the core are green. (c) Surface <br /> representation illustrates several potential surface salt bridges. The side chains of Lys and Arg residues are blue, Asp and Glu are red, and hydrophobic residues <br /> are green. Images were rendered with MOLMOL (59). <br />  <br />  <br />  <br /> incubated with ptRNA substrate in the absence and presence of                       ing secondary structural elements, whereas the invariant Gly-32 <br /> the four known protein subunits (14). Using conditions similar                      allows a sharp turn between strands &#226;&#65533;&#164;2 and &#226;&#65533;&#164;3 on opposing <br /> to those reported for P. horikoshii RNase P reconstitution (15),                    sheets. In the ensemble, the side chain of the invariant Lys-56 is <br /> no single archaeal protein was able to reconstitute measurable                      positioned at one edge of the hydrophobic core, where it appears <br /> activity (data not shown). However, enzyme activity could be                        to form a salt bridge with the side chain of Glu-37 (Fig. 4b); the <br /> obtained by preincubating the RNA with the four proteins (Fig.                      latter is either a Glu or an Asp in all Rpp29 family members (Fig. <br /> 2b). The essentiality of Rpp29 is demonstrated by the lack of                       1). The other two invariant residues, Arg-81 and Arg-85, are <br /> activity when Rpp29 was omitted from the reaction (Fig. 2b,                         located after the C-terminal helix and preceding the disordered <br /> compare lanes 5 and 6).                                                             C terminus. The surface of the structured domain (Fig. 4c) <br />                                                                                     exhibits several pairings of basic and acidic residues, indicating <br /> NMR Spectroscopy and Assignments. After removal of the (His)6-                      possible salt bridges, as well as significant hydrophobic surface <br /> tag with thrombin, expressed Mth Rpp29 consisted of the WT                          area. <br /> protein (93 residues), plus four additional N-terminal residues: <br /> GSHM. 2D 1H-15N correlated NMR spectra (Fig. 3a) indicated                          Interactions with the RNA Subunit. At low salt, Rpp29 resonance <br /> that the majority of the resonances were well dispersed, typical                    lines were excessively broadened in the presence of the RNA <br /> for a folded protein; however, near random coil chemical shifts                     subunit, presumably because of nonspecific interactions. After <br /> and narrow line widths of several resonances indicated that                         addition of salt (400 mM KCl in 50 mM phosphate) and raising <br /> regions of the protein were flexible or disordered. Limited                         the temperature to 50&#194;&#176;C, NMR spectra of the complex could be <br /> trypsin proteolysis and heteronuclear {1H}-15N NOE data (Fig.                       recorded. Comparison of the 15N HSQC spectra of Mth Rpp29 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        BIOCHEMISTRY <br /> 3b) demonstrated that both the N and C termini were flexible or                     in the absence and presence of the catalytic P RNA subunit (Fig. <br /> disordered, with the structured protein consisting of the 80                        5a) reveals many small shift perturbations and localized line <br /> central residues (residues 6&#226;&#8364;&#8220;85, see arrows in Fig. 1). Backbone                    broadening, consistent with weak binding. RNA-induced shift <br /> amide resonance assignments (Fig. 3a) could be obtained for all                     perturbations are apparent at multiple sites in the protein; the <br /> but residues 9&#226;&#8364;&#8220;13 and 85, in addition to the five prolines                          largest measurable shift perturbations were induced in the <br /> (residues 4, 55, 66, 82, and 93).                                                   amides of Arg-25 in the &#226;&#65533;&#164;1-&#226;&#65533;&#164;2 loop and Phe-63 and Thr-65 in &#226;&#65533;&#164;6 <br />  <br /> Structure Determination. The solution structure of Mth Rpp29 was <br /> well defined for residues 12&#226;&#8364;&#8220;80, with a mean backbone rms <br /> deviation of 0.6 &#195;&#8230; (Fig. 4a). Although proteolysis and hetero- <br /> nuclear NOE data indicate that residues 6&#226;&#8364;&#8220;85 are structured, <br /> insufficient restraints were obtained for residues 6&#226;&#8364;&#8220;11 and 82&#226;&#8364;&#8220;85 <br /> to properly define their structure. The core of the protein adopts <br /> the oligonucleotide&#229;&#8230;&#190;oligosaccharide-binding fold (OB-fold) (35, <br /> 36) and consists of a closed barrel of seven antiparallel &#226;&#65533;&#164; strands <br /> and a short 310 helix, &#226;&#65533;&#163;1, near the C terminus. Strands &#226;&#65533;&#164;1, &#226;&#65533;&#164;2, &#226;&#65533;&#164;6, <br /> and &#226;&#65533;&#164;7 form a four-stranded &#226;&#65533;&#164;-sheet, whereas strands &#226;&#65533;&#164;3, &#226;&#65533;&#164;4 and <br /> &#226;&#65533;&#164;5 form a three-stranded &#226;&#65533;&#164;-sheet. The core of the protein is well <br /> packed with highly conserved hydrophobic residues, particularly <br /> those at positions 12, 13, 15, 17, 19, 30, 34, 42, 44, 54, 61, 63, 71, <br /> 73, and 78. The core appears to be somewhat exposed to solvent,                     Fig. 5. Identification of protein&#226;&#8364;&#8220;RNA contacts by 1H-15N correlated NMR of <br /> particularly the side flanked by &#226;&#65533;&#164;3 and &#226;&#65533;&#164;7, although it is possible                 the 113-kDa Mth Rpp29 &#226;&#8364;&#8220;RNA complex. (a) Overlay of a portion of the HSQC <br /> that it could be partially buried by some of the N-terminal 14                      spectra of free Mth Rpp29 (black) and the 1:1 complex between the 11-kDa (93 <br />                                                                                     residues) protein and 102-kDa (314 nt) Mth P RNA (800 MHz, 50&#194;&#176;C, 50 mM <br /> residues. The surface of the protein is highly populated by less                    phosphate, 400 mM KCl). (b) Weighted average amide proton and nitrogen <br /> conserved acidic (Glu, Asp) and basic residues (Arg, His).                          weighted-average shift perturbations (34) are mapped onto the ribbon dia- <br />    Amino acid residues highly conserved across the Rpp29 family                     gram of the Rpp29 OB-fold by using a linear color ramp from gray (no change) <br /> (Fig. 1) occupy prominent places in the structure. The first and                    to red [&#226;&#338;&#172;av(NH) &#226;&#171;&#189; 0.15]. Residues in green are those for which the effect of RNA <br /> last invariant glycine residues (Gly-14, Gly-75) form turns flank-                  binding could not be assessed. <br />  <br />  <br /> Boomershine et al.                                                                                    PNAS &#229;&#8230;&#169; December 23, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169; no. 26 &#229;&#8230;&#169; 15401 <br />  (Fig. 5b). In addition, significant shift perturbations were ob-        indicate that in 400 mM salt, the interaction of Mth Rpp29 with <br /> served for residues in the flexible N- and C-terminal segments          the RNA subunit is relatively weak and exhibits spectra charac- <br /> of the domain (data not shown).                                         teristic of rapid exchange between the free and bound states. In <br />                                                                         addition, footprinting studies examining the susceptibility of Mth <br /> Discussion                                                              P RNA to digestion by RNase T1 or RNase V1 in the absence <br /> Despite significant differences in the primary sequences of the         and presence of Rpp29 indicate that the two components do not <br /> RNA and protein subunits of bacterial RNase P, their overall            interact strongly in the absence of the other proteins (H.-Y.T. <br /> structures are remarkably conserved (1, 2, 37&#226;&#8364;&#8220;39), likely a             and V.G., unpublished results). Nevertheless, comparison of the <br /> reflection of the evolutionary pressure provided by the con-            15N-edited HSQC spectra of Mth Rpp29 free and in complex <br />  <br /> served function of the enzyme. On the other hand, although the          with the P RNA yields specific information about residues that <br /> RNA subunits of archaebacterial RNase P are similar in sec-             interact with the P RNA. Given that the P RNA is much larger <br /> ondary structure to their bacterial counterparts (14, 25), se-          than Mth Rpp29 (by 10-fold) and that chemical shift perturba- <br /> quence analysis suggests that the archaeal protein subunits are         tions were observed on both ends of the protein, it is likely that <br /> structurally unrelated to the bacterial protein, exhibiting instead     the P RNA wraps around Mth Rpp29 or that the protein fits into <br /> sequence homology to proteins from the eukaryotic enzyme                a large pocket on the RNA. The increased salt concentration <br /> (14). Until recently (15), the inability to reconstitute the eu-        (400 mM) used to obtain spectra of Mth Rpp29 in complex with <br /> karyotic and archaeal enzymes from their purified components            the P RNA should reduce the possibility that the observed <br /> made it difficult to demonstrate a direct functional role for any       spectral perturbations are dominated by nonspecific interactions <br /> of the protein subunits originally identified by virtue of their        between a cationic protein and an anionic RNA. In fact, many <br /> association with biochemically purified enzyme activity (8, 14,         of the residues exhibiting the largest shift changes were not <br /> 40&#226;&#8364;&#8220;44), leaving open the possibility that an as yet undiscovered        cationic arginines or lysines (Fig. 5), but include hydrophobic <br /> bacterial-like protein was required for activity. Furthermore, the      (e.g., Val-17, Ile-27, Phe-63, Ile-73) and polar (e.g., Thr-53, <br /> poor sequence homology of the four identified subunits of               Thr-65) residues. Indeed, a recent report demonstrating a sig- <br /> archaeal RNase P to proteins of known structure furnished few           nificant role for hydrophobic residues in nucleic acid binding by <br /> clues as to their roles in RNase P-mediated ptRNA processing.           the telomere-end binding protein, Cdc13, a member of the <br /> This impasse has now been overcome by the demonstration that            OB-fold protein family (36), indicates such observations are not <br /> a functional archaeal RNase P holoenzyme can be reconstituted           unique to this system. <br /> from the in vitro-transcribed P RNA and recombinantly ex-                  We speculate that one role of Rpp29 is to stabilize the active <br /> pressed and purified Rpp29, Rpp21, Rpp30, and Pop5 proteins             conformation of the RNA subunit by forming favorable hydro- <br /> of P. horikoshii, a hyperthermophile (15), and Mth, a moderate          phobic and electrostatic interactions. However, although a single <br /> thermophile (Fig. 2b). Moreover, although the functional roles          protein is enough to complete the bacterial enzyme, no single <br /> of the individual protein subunits remain to be elucidated, both        archaeal RNase P protein can sufficiently stabilize the RNA <br /> studies demonstrate that Rpp29 is essential for enzyme activity         structure to promote activity (Fig. 2 and ref. 15). Instead, it <br /> in a reconstitution assay. This study on Mth Rpp29 now provides         appears that the protein subunits exhibit cooperativity in their <br /> a glimpse into structure&#226;&#8364;&#8220;function relationships in archaeal&#229;&#8230;&#190;            ability to promote catalysis (H.-Y.T. and V.G., unpublished <br /> eukaryotic RNase P.                                                     work). Nevertheless, with the caveat that all of the native <br />    This investigation has revealed that Rpp29 is a member of the        RNA&#226;&#8364;&#8220;protein interactions between these subunits might be <br /> OB-fold family of proteins (35, 36). Although OB-fold contain-          achieved only when all of the protein components of the <br /> ing proteins have been difficult to identify from primary se-           holoenzyme are present, the NMR chemical shift perturbation <br /> quence, they are well represented in ribonucleoprotein com-             data presented here provide insights into possible protein&#226;&#8364;&#8220; <br /> plexes from all three domains of life. Examples include the S1          nucleic acid contacts in the holoenzyme. Furthermore, based on <br /> and S17 bacterial ribosomal proteins (45), the anticodon-               the data, we suggest that the protein interacts with the RNA via <br /> recognition domain of aspartyl and lysyl tRNA synthetases,              the conserved OB-fold domain and that the highly conserved C <br /> single-stranded DNA binding proteins from Thermus thermophi-            terminus (and potentially the N terminus) of Mth Rpp29 be- <br /> lus and Thermus aquaticus (46), translation initiation factor           comes structured upon binding to the Mth P RNA (and&#229;&#8230;&#190;or other <br /> aIF-1A from Methanococcus jannaschii (47), the single-stranded          protein subunits). <br /> telomeric DNA binding protein from Saccharomyces cerevisiae,               The proposal that Rpp29 functions via direct interaction with <br /> Cdc13 (48), and the N-terminal segment of human translation             the RNA subunit is also supported by the results from earlier <br /> initiation factor &#226;&#65533;&#163; (49). Not surprisingly, Mth Rpp29 exhibits no       studies. For instance, three hybrid assays in yeast revealed that <br /> significant structural similarity to the protein subunit of bacterial   both human and yeast Rpp29 interact with their respective <br /> RNase P, which adopts an &#226;&#65533;&#163;-&#226;&#65533;&#164; sandwich fold (37&#226;&#8364;&#8220;39).                     RNase P RNA subunit (54&#226;&#8364;&#8220;56). Additionally, findings from a <br />    The surface of the structured core of Rpp29 (Fig. 4c) exhibits       genetic study with RNase MRP, an endonuclease related to <br /> a unique array of cationic, anionic, and hydrophobic patches that       RNase P, further support the idea that Rpp29 is involved in <br /> could mediate specific protein&#226;&#8364;&#8220;protein and protein&#226;&#8364;&#8220;RNA rec-             binding its cognate RNA subunit (41). In all eukaryotes, RNase <br /> ognition. In addition to the structured core containing the             MRP is one of a series of enzymes involved in rRNA processing. <br /> OB-fold, the unstructured N- and C-terminal extensions bear             RNase MRP and RNase P, although acting on different sub- <br /> several highly conserved arginine and lysine residues (Fig. 1) that     strates, are related in that they are both ribonucleoprotein <br /> could provide additional stabilizing interactions with RNA.             complexes that share all but one protein subunit and have similar <br /> Indeed, it seems likely that the hydrophobic patches and salt           RNA subunits. In a study of S. cerevisiae RNase MRP (41), one <br /> bridges of the surface of the protein, as well as the unstructured      class of extragenic suppressors of a temperature-sensitive mu- <br /> N and C termini may have critical roles in induced-fit recognition      tation rrp2&#226;&#8364;&#8220;2 in the RNA subunit of RNase MRP were mapped <br /> of the RNA and other subunits (50, 51). Such an expectation is          to alterations of a single codon in the POP4&#229;&#8230;&#190;RPP29 gene. Based <br /> not without precedent: highly charged and unstructured N or C           on this observation, it was proposed that the rpp2&#226;&#8364;&#8220;2 <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>mutation</span> in <br /> termini in the globular ribosomal proteins, L15, L21e, and L37e         the MRP RNA might disrupt RNA&#226;&#8364;&#8220;protein interactions with <br /> (52) penetrate the ribosome and make important structural               Pop4 (Rpp29), a reasonable premise that has so far been <br /> contacts with the ribosomal RNA (52, 53).                               untested. <br />    What is the nature of the interaction of Mth Rpp29 with the             The mechanistic and structural details of how Mth Rpp29 <br /> RNA subunit? NMR spectra of the RNP complex (Fig. 5a)                   interacts with the catalytic P RNA and other protein subunits <br />  <br /> 15402 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.2535887100                                                                      Boomershine et al. <br />  remain to be uncovered. Those studies should help reveal the                            We thank C. Amero, C. Cottrell, S. Kraynik, W. Woznick, and C. Yuan <br /> function of Mth Rpp29 within the holoenzyme and allow com-                              (Ohio State University) for assistance and technical support; J. W. <br /> parisons between the functions of the protein subunits of                               Brown (North Carolina State University, Raleigh) for helpful discussions <br /> bacterial and archaeal&#229;&#8230;&#190;eukaryotic RNase P. Given the stark                              and the Rpp21 expression plasmid; A. Edwards and S. Arrowsmith <br /> difference between the structure of Rpp29 and the bacterial                             (University of Toronto, Toronto) for the Rpp29, Rpp30, and Pop5 <br /> protein subunit, it is unclear why the many subunits of archaeal                        expression constructs, M. L. S. Raj (Ohio State University) for the Mth <br /> RNase P are required to supplant the function of their single                           P RNA clone; J. Reeve (Ohio State University) for Mth genomic DNA, <br /> bacterial counterpart. Although a comprehensive understanding                           M. Adams (University of Georgia, Athens) for Pfu genomic DNA; the <br /> of the mechanism of archaeal RNase P will necessitate detailed                          Ohio State University Campus Chemical Instrument Center for mass <br /> studies of all subunits and their complexes, structural charac-                         spectrometric analyses; and the Ohio State University Plant-Microbe <br /> terization of Mth Rpp29 and its interaction with the catalytic P                        Genomics Facility and Plant Biotechnology Center for DNA sequencing. <br />                                                                                         W.P.B. was supported by a National Institutes of Health Chemistry&#226;&#8364;&#8220; <br /> RNA represent important steps toward elucidating structure&#226;&#8364;&#8220; <br />                                                                                         Biology Interface training grant, and H.-Y.T. was supported by an <br /> function correlates in the holoenzyme. <br />                                                                                         American Heart Association predoctoral fellowship. This work was <br /> Note Added in Proof. The solution structure of the Rpp29 homolog from                   supported in part by Ohio State University and National Science <br /> Archaeoglobus fulgidus has recently been reported (60) and seems to                     Foundation Grants MCB-0092962 (to M.P.F.) and MCB-0091081 (to <br /> adopt the same architecture as the Mth homolog.                                         V.G.). <br />  <br />  <br />   <br /> </body></html>