<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Foundations Reproductive Lethality in <br /> Arabidopsis thaliana <br /> Rosanna Muralla, Johnny Lloyd, David Meinke* <br /> Department Botany, Oklahoma State University, Stillwater, Oklahoma, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The SeedGenes database (www.seedgenes.org) contains information 400 genes required embryo <br />      development Arabidopsis. Many EMBRYO-DEFECTIVE (EMB) genes encode proteins essential function <br />      required life cycle. This raises fundamental question. Why does elimination essential gene in <br />      Arabidopsis result embryo lethality gametophyte lethality? In words, mutant (emb) <br />      gametophytes survive participate fertilization essential cellular function disrupted? Furthermore, do <br />      mutant embryos proceed development others? <span id='am-8' about='protege:TO' typeof='owl:Thing'>To</span> address questions, established a <br />      curated dataset genes required gametophyte development Arabidopsis based information extracted the <br />      literature. This provided basis comparison EMB genes obtained SeedGenes dataset. We identified <br />      genes exhibited embryo gametophyte defects disrupted loss-of-function <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>mutation</span>. We then <br />      evaluated relationship mutant <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>phenotype</span>, gene redundancy, mutant allele strength, gene expression <span id='am-401' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, <br />      protein function, intracellular protein localization determine factors influence phenotypes lethal mutants <br />      Arabidopsis. After removing cases continued development potentially resulted gene redundancy residual <br />      function weak mutant allele, identified numerous examples viable mutant (emb) gametophytes required <br />      explanation. We propose presence gene products derived transcription diploid (heterozygous) <br />      sporocytes enables mutant gametophytes survive loss essential gene Arabidopsis. Whether gene <br />      disruption results embryo gametophyte lethality depends ability residual, parental gene <br />      products support gametophyte development. We highlight 70 preglobular embryo mutants zygotic <br />      pattern inheritance, provide valuable insights maternal-to-zygotic transition Arabidopsis timing <br />      paternal gene activation embryo development. <br />  <br />   Citation: Muralla R, Lloyd J, Meinke D (2011) Molecular Foundations Reproductive Lethality Arabidopsis thaliana. PLoS ONE 6(12): e28398. doi:10.1371/ <br />   journal.pone.0028398 <br />   Editor: Gordon Chua, University Calgary, Canada <br />   Received July 22, 2011; Accepted November 7, 2011; Published December 2, 2011 <br />   Copyright: &#195;&#376; 2011 Muralla et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research supported National Science Foundation grant DBI-0618166 Arabidopsis 2010 Program, Oklahoma Center <br />   Advancement Science Technology. The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: david.meinke@okstate.edu <br />  <br />  <br />  <br />  <br /> Introduction                                                                          isolation characterization embryo-defective (emb) mutants of <br />                                                                                       Arabidopsis [26&#226;&#8364;&#8220;28]. Identifying genes responsible these <br />    Essential genes long played important role microbial                mutant phenotypes facilitated advances T-DNA <br /> medical genetics. In recent years, factors contributed                insertional mutagenesis, enabled large-scale, forward <br /> the establishment large datasets essential genes microorgan-                 genetic screens tagged, loss-of-function mutants defective in <br /> isms: failure mutant cells proliferate readily detected               seed development [29,30]. Eventually, led establishment <br /> high-throughput gene disruption experiments; products                          SeedGenes database (www.seedgenes.org), presents <br /> essential genes define promising targets novel, antimicrobial                     detailed information genes required embryo development in <br /> compounds relevant human health; lethal mutations                           Arabidopsis [24]. The December 2010 database release includes 402 <br /> essential genes help define minimal gene set required sustain             EMB genes essential cellular function required produce <br /> living cell [1,2]. Many laboratories contributed large-scale              normal, mature embryo. Many EMB proteins likely be <br /> identification essential genes bacteria [3&#226;&#8364;&#8220;6] yeast [7&#226;&#8364;&#8220;10],             required life cycle. Mutants exhibit defects in <br /> to establishment database essential genes comparative                 embryogenesis absence functional <br /> studies [11]. Related efforts Drosophila [12&#226;&#8364;&#8220;14], Caenorhabditis [15],           gene product critical. <br /> mouse [16,17], humans [18] generated wealth                                Other research groups pursued complementary <br /> information diversity gene products required viability               approach analysis essential genes Arabidopsis through <br /> multicellular eukaryotes. Recent evolutionary studies                       isolation characterization mutants defective in <br /> explored relationship gene function, duplication,                     gametophyte development [31&#226;&#8364;&#8220;35]. Identification mutants <br /> essentiality animal systems [19&#226;&#8364;&#8220;23].                                               facilitated screening insertion lines exhibit <br />    Our desire establish comparable dataset essential genes                    reduced transmission associated selectable marker. Although <br /> in model plant [24,25] arose longstanding                    hundreds mutants defective gametophyte development have <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                          December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                               Reproductive Lethality Arabidopsis <br />  <br />  <br /> been years, identities genes responsible           Arabidopsis appears contain 750 1000 EMB genes located <br /> these mutant phenotypes confirmed. Because               genome [28,45]. Table S1 presents updated, <br /> large deletions translocations common T-DNA                   comprehensive dataset 396 EMB loci Arabidopsis. Many the <br /> transposon insertion lines Arabidopsis [36&#226;&#8364;&#8220;39],                 EMB gene identities revealed work laboratory have <br /> chromosomal aberrations result lethality fertiliza-          published before. The current dataset, excludes six <br /> tion [35], large datasets gametophyte essentials include             atypical problematic SeedGenes loci, likely represents about <br /> candidate genes represented single mutant allele              40% saturation. This sufficient begin addressing diversity <br /> viewed caution.                                                         protein functions required cell viability embryo <br />    In order characterize cellular disruptions lead            development. Regarding level certainty gene <br /> defects embryo gametophyte development,              identified responsible mutant phenotype described, we <br /> established curated dataset gametophyte essentials           recognize distinct categories EMB genes: labeled as <br /> be compared embryo essentials SeedGenes                confirmed, molecular complementation the <br /> database. We used dataset combination                   analysis additional mutant alleles disrupted gene, <br /> updated SeedGenes dataset determine factors               labeled confirmed, robust genetic data <br /> distinguish mutants defective embryo development               indicate close linkage gene mutant <br /> defective gametophyte development. Specifically, wanted             phenotype (www.seedgenes.org). Approximately 100 EMB <br /> understand mutants defective embryo development                   genes Table S1 identities remain confirmed. <br /> able survive gametophyte development essential                    Based experience insertion lines, estimate that <br /> function required life cycle disrupted,                .85% identities correct. Genes unknown function <br /> conversely, mutants exhibit gametophyte lethality                   remain problematic protein identity be <br /> instead. In addition, sought explain mutant                   used support conclusion essential process been <br /> embryos reach later stage development others,             disrupted. <br /> general conclusions drawn relationship                       To assess terminal stage development reached mutant <br /> between gene function mutant phenotype.                                  embryos prior seed desiccation, evaluated 700 <br />    This analysis led conclude pre-meiotic gene expression         mutants classified SeedGenes having embryo-defective <br /> in diploid microsporocytes megasporocytes likely             mutant phenotype. When mutant allele was <br /> important feature reproductive development Arabidopsis.                available, chose appeared strongest allele based <br /> Although past work elegantly demonstrated diversity               phenotype known location mutation. After removing <br /> transcripts male female gametophytes [40&#226;&#8364;&#8220;44],               44 EMB loci associated additional defects gametophyte <br /> approaches pursued generally distinguished                  development, divided remaining 352 &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB loci into <br /> transcripts produced meiosis produced afterward.            terminal phenotype classes, ranging preglobular cotyle- <br /> We propose gene products derived transcription              don. The results shown Figure 1A. Fifty-five percent these <br /> wild-type allele heterozygous sporocytes enable mutant               loci essential heart stage development. Arrested <br /> gametophytes survive loss essential gene,              embryos mutants typically white, of <br /> embryo lethality results residual products exhausted.         mutants reach transition cotyledon stages <br /> In Arabidopsis, pollen tubes travel short distances order reach        pale green green. More 15% &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes are <br /> the ovule, increases likelihood male gametophytes             required earliest, preglobular stages embryo development. <br /> lacking essential gene product survive participate             Although broad descriptions fail capture important <br /> fertilization. By contrast, equivalent gene disruptions plants       phenotype details, demonstrate global scale EMB <br /> long pollen tubes, notably maize, likely result          genes differ far embryo development proceed when <br /> male gametophyte lethality.                                                  functions disrupted. Furthermore, discussed later, the <br />    This raises important question: mutant                    existence substantial number embryo-defective mutants <br /> embryos disrupted essential gene survive                    Mendelian pattern inheritance, characterized 25% <br /> globular stage development, given levels gene products            mutant seeds siliques selfed heterozygotes, a <br /> stored mutant gametophytes insufficient support                    preglobular stage embryo arrest, implications timing <br /> unlimited growth? Several additional factors, including gene                 extent maternal-to-zygotic transition gene expression <br /> redundancy, residual gene function, delayed activation                    plant embryo development. <br /> specialized pathways, diffusion products maternal                   Historically, distinction embryo gametophyte <br /> tissues likely influence stage development reached mutant          mutants Arabidopsis vague subjective. Some <br /> embryos seed desiccation. In addition, gene products             embryo mutants exhibit defects gametophyte development, and <br /> may perform limited role essential cellular process             gametophyte mutants, maintained heterozygotes, <br /> than others, allow development               produce low frequency viable mutant gametes. Embryo <br /> otherwise expected. We conclude datasets genes                mutants defects male gametophyte development often <br /> required embryo gametophyte development presented                    exhibit reduced percentage nonrandom distribution of <br /> here provide useful resource comparative studies            mutant seeds heterozygous siliques. This &#226;&#8364;&#732;&#226;&#8364;&#732;certation&#226;&#8364;&#8482;&#226;&#8364;&#8482; effect was <br /> multicellular eukaryotes valuable insights molecular            originally noted Mu&#194;&#168;ller [46] later observed in <br /> basis reproductive lethality model plant.                            collections mutants isolated characterized laboratory <br />                                                                              [27,47]. The underlying mechanism disabled mutant pollen <br /> Results Discussion                                                       tubes fail reach ovules <span id='am-399' about='xsp:base' typeof='owl:Thing'>base</span> silique. Reduced <br />                                                                              pollen viability result decreased percentage mutant <br /> The SeedGenes dataset EMB genes required                              seeds overall. Embryo mutants defects female gametophyte <br /> embryo development                                                           development typically low percentage mutant seeds, <br />   Based abundance embryo-defective mutants                     randomly distributed silique, combined high <br /> the frequency duplicate alleles mutagenesis experiments,               percentage aborted ovules. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                      December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                               Reproductive Lethality Arabidopsis <br />  <br />  <br />                                                                             GAM (Gametophyte) subclass essentials represents those <br />                                                                             genes combined gametophyte defects severe, <br />                                                                             resulting fewer 2% mutant seeds expected selfed <br />                                                                             heterozygotes. Different subclasses embryo gametophyte <br />                                                                             essentials described report listed Table 1. We then <br />                                                                             searched literature published examples genes required <br />                                                                             gametophyte development Arabidopsis, recorded male and <br />                                                                             female transmission ratios documented, noted informa- <br />                                                                             tion frequencies mutant embryos produced. This <br />                                                                             provided foundation curated dataset presented here <br />                                                                             genes required gametophyte development. <br />  <br />                                                                             A curated dataset genes required gametophyte <br />                                                                             function <br />                                                                                In order create dataset gametophyte essentials <br />                                                                             comparable EMB dataset SeedGenes, excluded <br />                                                                             unconfirmed, putative gametophyte loci single <br />                                                                             mutant allele characterized flanking sequence was <br />                                                                             obtained sides insert. This eliminated <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                             consideration sizeable number candidate genes. But also <br />                                                                             increased likelihood genes identified responsible <br />                                                                             gametophyte defects correct. The resulting <br />                                                                             dataset, shown Table S2, contains total 173 genes required <br />                                                                             normal gametophyte development. This includes 70 definitive <br />                                                                             GAM loci, 4 GAM / GEM loci, 25 GEM loci, 3 GEM / EMG <br />                                                                             loci, 44 EMG loci, 27 loci defective <br />                                                                             gametophytes produce viable homozygotes are <br />                                                                             difficult classify incomplete transmission data. GAM <br />                                                                             loci differentiated based transmission rates the <br />                                                                             mutant allele. Three broad groups recognized (Table S2): <br /> Figure 1. Terminal embryo phenotypes strongest known                 strong defects (,0.4 transmission efficiency, TE) on <br /> alleles &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes evidence gametophyte <br /> defects. (A) Entire dataset 352 genes. (B) Comparison            male female sides (14 loci); severe defects <br /> unique &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; (UT) EMB genes redundant paralog,         (,0.1 TE) male (44 loci); severe defects <br /> remaining dataset (OT) &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes              (,0.1 TE) female (12 loci). These 70 GAM genes <br /> redundant (BLASTP match.e-30).                                              defined robust dataset gametophyte essentials then <br /> doi:10.1371/journal.pone.0028398.g001                                       compared 352 &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes devoid gametophyte <br />                                                                             defects (Table S3) 72 EMG GEM loci defects both <br />     Because SeedGenes database includes information,               embryo gametophyte development (Table S4). <br /> available, mutant seed ratios distributions, assembled a <br /> list SeedGenes loci appeared defects                Protein functions genes required embryo and <br /> embryo gametophyte development. These assigned               gametophyte development <br /> EMG (Embryo-Gametophyte) subclass EMB loci distinguish                   Extensive overlap exists types protein functions <br /> them &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes lacked <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>knockout</span> defects                required gametophyte development required for <br /> gametophyte development. We examined publications dealing              embryo development (Figure 2). Consequently, explain <br /> with SeedGenes loci identify additional examples EMB genes            difference embryo gametophyte mutants the <br /> associated defects gametophyte development. By defini-              basis protein function alone. Nevertheless, distinctive features of <br /> tion, mutants assigned EMG subclass known                     dataset identified. For example, interfering DNA <br /> predicted produce 10% defective seeds following self-           replication RNA modification typically disrupts embryo <br /> pollination heterozygotes reduced frequency            development, complete loss cytosolic translation in <br /> of mutant seeds overall, mutant seeds base            absence redundant gene function appears result 100% <br /> silique, excessive number aborted ovules. We                  male female gametophyte lethality, means such <br /> established subclass embryo essential genes            mutants readily maintained [48]. Blocking chloroplast <br /> severe defects gametophyte function, labeled GEM (Gameto-                translation results embryo arrest globular stage of <br /> phyte-Embryo), recognize cases heterozygotes known             development [49]. In fact, translation defects associated <br /> or predicted, based reduced gamete transmission mutant            unique (non-redundant) &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes identified date <br /> allele, produce 2% 10% mutant seeds. Both                involve chloroplast-localized proteins. Recently, proposed that <br /> EMG GEM subsets EMB genes, therefore, exhibit                      embryo lethality mutants results failure produce <br /> combination embryo gametophyte defects. The difference               fatty acids required continued growth development [49]. <br /> lies extent defects impact fertilization       Interfering translation mitochondria typically results an <br /> and embryo development. We chose 2% mutant seeds                     ovule abortion phenotype features embryo female <br /> baseline GEM subclass corresponds single          gametophyte lethality, reduced transmission male <br /> mutant seed silique, average, marginally          gametes [48]. As expected, perturbations cellular processes <br /> background rate spontaneous seed abortion [26]. By contrast,             required rapid tip growth (Figure 2, Class 11) common <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        3                       December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                                   Reproductive Lethality Arabidopsis <br />  <br />  <br />  <br />  Table 1. Classification genes required embryo gametophyte development. <br />  <br />  <br />  Class essential genes                    Subclass          <span id='am-5' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> class subclass                            Genes          Table <br />  <br />  Embryo defective                                              Defect mutant embryo characterized                      396            S1 <br />                                              True EMB          No documented gametophyte defect                           352            S3 <br />                                              UT-EMB            Unique true EMB; No obvious paralog                        152            S5 <br />  Gametophyte defective                                         Defect mutant gametophyte characterized                 173            S2 <br />                                              GAM               ,2% mutant seeds predicted observed                     70             S2 <br />                                              Viable            Gametophyte defective; homozygote viable                   14a            S2 <br />                                              Uncertain         Gametophyte defective; classification uncertain            13a            S2 <br />  Embryo / gametophyte defective                                Embryo gametophyte defective                           76             S2 <br />                                              EMG               .10% mutant seeds predicted observed                    44             S2, S4 <br />                                              GEM               2&#226;&#8364;&#8220;10% mutant seeds predicted observed                   25             S2, S4 <br />                                              GAM / GEM         Precise subclass uncertain conflicted                   4              S2 <br />                                              GEM / EMG         Precise subclass uncertain conflicted                   3              S2, S4 <br />  <br />  a <br />   Included Table S2 discussed text. <br />  doi:10.1371/journal.pone.0028398.t001 <br />  <br />  <br /> among mutants severe defects male gametophyte                        average, disruptions beginning (A region) middle (B <br /> development. Most auxotrophic mutants exhibit embryo lethality               region) gene likely eliminate protein function <br /> rather gametophyte lethality, explained            end gene (C region). We distinguished <br /> by ability maternal tissues supply limited nutrients            insertions introns located exons, although <br /> mutant gametophytes. The disruption male gametophyte                      single flanking sequence available localized an <br /> development observed histidine auxotrophs placed               intron, uncertain adjacent exon be <br /> the EMG class embryo lethals [50] reflect perturbation           disrupted. Overall, 89 152 UT-EMB loci associated <br /> the linked ATP recycling pathway loss histidine.            mutant alleles contained exon insertion <br /> The abundance EMB proteins unknown functions                         middle gene. Another 23 intron <br /> included SeedGenes database, compared                   insertion (A/B) region, 19 contained insertion <br /> gametophyte dataset, reflect reluctance journals         final gene. When compared terminal <br /> publish work individual gametophyte mutants defective                  phenotypes 89 mutants carrying A/B exon disruptions with <br /> proteins unknown functions.                                             42 mutants containing A/B intron C region <br />                                                                              disruptions, significant differences (x2 = 2.50; <br /> Genetic redundancy mutant allele strength                                p = 0.65) frequencies early versus late embryo phenotypes <br />    Two potential explanations mutants defective                   (data shown). We significant differences <br /> embryo development survive gametophyte development                     (x2 = 7.69; p = 0.17) phenotype distributions obtained when <br /> involve genetic redundancy mutant allele strength. The first <br /> model proposes duplicated genes redundant functions <br /> compensate loss essential EMB gene functions during <br /> gametophyte development. The second model suggests most <br /> embryo-defective mutant alleles retain function predicts <br /> that true null alleles EMB genes gametophyte lethals <br /> instead. Both models insufficient explain results <br /> obtained. We evaluated level redundancy 352 &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> EMB genes using BLASTP searches (www.ncbi.nlm.nih.gov) the <br /> Arabidopsis proteome. EMB proteins lacking match least <br /> an e-30 level significance viewed likely unique. <br /> Those significant matches classified potentially <br /> redundant, cases genes be <br /> functionally distinct. We chose e-30 cutoff value relatively <br /> stringent selection unique genes wanted              Figure 2. Distribution protein function classes among <br />                                                                              different collections essential genes. (1) DNA synthesis; repair; <br /> identify EMB genes <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>survival</span> mutant gametophytes               (2) RNA synthesis; modification; (3) protein synthesis; (4) Protein <br /> was likely result genetic redundancy. The resulting            modification; transport; (5) protein degradation; (6) chromosome <br /> dataset 152 &#226;&#8364;&#732;&#226;&#8364;&#732;unique true&#226;&#8364;&#8482;&#226;&#8364;&#8482; (UT) EMB loci presented Table           dynamics; (7) transcriptional regulation; (8) signaling pathways; (9) <br /> S5.                                                                          energy; electron transport; (10) metabolism; (11) cell structure; <br />    Distinguishing null alleles retain low level         membrane function; trafficking. Significant differences observed <br />                                                                              function classes 2 (x2 = 6.22; p,0.05), 3 (x2 = 6.77; p,0.05), 11 <br /> protein function particularly difficult lethal mutants because <br />                                                                              (x2 = 16.7; p,0.001). Excluded genes functions remain to <br /> homozygous mutant tissue readily analyzed. We                      classified (15 &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB; 5 EMG/GEM; 3 GAM) with <br /> therefore decided record insert locations mutant alleles          uncertain unknown functions (57 &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB; 3 EMG/GEM; 6 GAM). <br /> represented SeedGenes database, assumption              doi:10.1371/journal.pone.0028398.g002 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        4                          December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                              Reproductive Lethality Arabidopsis <br />  <br />  <br /> the unique potentially redundant datasets &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB              functions represented, purine, proline, pantothenate <br /> genes compared (Figure 1B), mutants                     biosynthesis (fac1, pts, emb2772) tRNA modification the <br /> preglobular defects appeared somewhat common                    cytosol (emb2191, emb2820) chromosome dynamics (emb2773, <br /> among UT-EMB genes, consistent model                    emb2782) microtubule assembly (ttn1, pfi, por, emb2804, qqt2). <br /> mutants late defects partially rescued expression        Surprisingly, chloroplast-localized proteins missing this <br /> a second, redundant gene.                                                  dataset genes encoding proteins unknown functions are <br />    Another point argues idea embryo-            reduced occurrence. We propose these <br /> defective mutants correspond weak alleles genes required         knockouts, residual gene products wild-type allele <br /> gametophyte development relatively examples                 transcribed diploid sporocytes enable mutant gametophytes to <br /> gametophyte lethal alleles known EMB loci                     participate fertilization. Development homozygous mutant <br /> documented literature. We uncovered cases,             seeds arrests short time later, reserves become <br /> including EMB1989/NRPB2, EMB2779/LCB1, RSW1/CESA1,                         depleted. Diffusion nutrients surrounding maternal tissues <br /> EMB24/AtASP38, EMB2776/PRP4/LIS [51&#226;&#8364;&#8220;54],                       primary source metabolites stages male <br /> number small compared total number essential           gametophyte development pollen-tube growth Arabi- <br /> genes identified date. By contrast, 30 examples          dopsis proceed vitro simple, defined medium [44,55]. <br /> been documented weak mutant alleles EMB genes that <br /> complete embryo development exhibit defects                   Nature stored gene products <br /> seedlings adult plants (Table S6). Based analyses,              In principle, gene products contributed diploid <br /> conclude vast majority cases, ability emb            sporocytes mutant gametophytes RNA or <br /> mutant gametophytes participate fertilization likely       protein, combination both. Male gametophytes long <br /> be explained residual protein function provided weak               known active transcription. Post-meiotic gene <br /> mutant allele. Other factors genetic redundancy weak           expression demonstrated maize analyzing isozyme <br /> mutant alleles must, therefore, involved supporting                  patterns dimeric enzymes pollen heterozygous plants <br /> gametophyte development heterozygous plants.                            [56]. Female gametophytes difficult study, <br />                                                                            presence surrounding maternal tissues. <br /> Significance terminal embryo phenotypes                                 Translation cytosolic mRNAs essential feature of <br />    Disruptions UT-EMB genes permit growth                      gametophyte development Arabidopsis. Mutant alleles unable <br /> development continue globular stage embryo                support essential process likely transmitted through <br /> development mutant alleles question appear           male female gametes [48]. Furthermore, pollen-tube growth in <br /> be nulls. How apparent contradiction explained?                vitro blocked presence translation inhibitors [42]. This <br /> Because diploid sporocytes small comparison multicel-            means diploid sporocytes contribute haploid <br /> lular embryos late development, unlikely provide            gametophytes sufficient levels functional proteins derived from <br /> gametophytes sufficient quantities stored gene products         essential genes required complete growth development. <br /> explain late embryo phenotypes mutants. Other           Nevertheless, possibility remains stored products some <br /> factors involved. In order evaluate loci,        essential genes proteins instead RNAs. <br /> we focused 36 UT-EMB genes A/B exon disruptions                   Knockout mutants disrupted essential subunits RNA <br /> exhibited terminal embryo phenotype transition                 polymerase unable transmit mutant allele through <br /> cotyledon stage. One striking feature resulting dataset,            female gametes exhibit reduced transmission mutant <br /> shown Table S7, diversity possible causes late        allele male gametes [51]. On female side, this <br /> embryo phenotypes observed. These include diffusion                  indicates transcription gametophyte required for <br /> essential <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> surrounding maternal tissues (bio1), late          normal development maternal sources RNA <br /> expression transcriptional regulator (wox2), delayed require-         polymerase insufficient meet transcriptional demands. On <br /> ments specialized pathways metabolism (tag1, hyd1,                  male side, microsporocytes appear contribute RNA <br /> emb1873), peripheral roles EMB proteins essential                polymerase activity male gametophytes, stored <br /> cellular processes (pex10, emb2184, emb2754). Other notable                transcript functional protein, activity sufficient <br /> features dataset include abundance genes encoding           meet transcriptional demand mutant pollen, resulting <br /> chloroplast-localized proteins (15 loci) proteins unknown         reduced rates transmission. RNA polymerase knockouts, <br /> cellular functions (12 loci). We conclude cases,          however, fail address broader question other <br /> late embryo phenotypes observed knockouts consistent          products essential genes contributed sporocytes to <br /> with known protein functions important late                 gametophytes, products proteins <br /> embryo development. In words, elimination                   transcripts. A different pattern documented <br /> specialized functions expected result lethality         SHORT SUSPENSOR (SSP) locus, transcript <br /> immediately fertilization. In cases, protein product       produced male gametophytes translated zygote and <br /> likely performs peripheral role essential process required         modulates important signaling pathway early embryo <br /> throughout growth development. The survival mutant                  development [57]. <br /> gametophytes emb/EMB heterozygotes best addressed, <br /> therefore, using EMB genes knockout phenotypes early               Transcriptional profiles essential genes male <br /> stages embryo development, cases, survival          gametophytes <br /> the mutant gametophyte contrasts documented, early                   We searched evidence stored, pre-meiotic transcripts of <br /> lethality mutant embryo.                                      EMB genes haploid gametophytes comparing published <br />    We focused dataset 25 UT-EMB genes                  transcript profiles [42&#226;&#8364;&#8220;45] different classes essential <br /> preglobular zygotic terminal phenotype,                   genes Arabidopsis: (a) 59 GAM GEM loci severe defects <br /> consistent essential gene product required              male transmission (Table S8); (b) 48 GEM EMG loci with <br /> development (Table 2). Once again, wide range protein                 moderate defects male transmission (Table S9); (c) 75 UT- <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       5                       December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                                       Reproductive Lethality Arabidopsis <br />  <br />  <br />  <br />  Table 2. Unique &#226;&#8364;&#732;&#226;&#8364;&#732;true&#226;&#8364;&#8482;&#226;&#8364;&#8482; EMB genes documented preglobular embryo phenotype. <br />  <br />  <br />  Locus                       Gene symbol                 Identity statusa          Function classb           Predicted protein function <br />  <br />  At1g03360                   RRP4                        C                         2.3                       rRNA Processing; Exonuclease <br />  At1g09770                   AtCDC5                      NC                        7                         MYB Domain Transcription Factor; Cell Cycle <br />                                                                                                              Control <br />  At1g48175                   EMB2191                     C                         2.3                       tRNA Adenosine Deaminase <br />  At1g55350                   EMB1275                     C                         12                        Calpain-Type Cysteine Protease <br />  At1g55900                   EMB1860                     C                         4.2                       Inner Mitochondrial Membrane Import Protein <br />  At1g71440                   PFI                         C                         11.1                      Tubulin Folding Cofactor E; Microtubule Dynamics <br />  At2g30920                   EMB3002                     NC                        9                         Methyltransferase; Ubiquinone Biosynthesis <br />  At2g32590                   EMB2795                     C                         6.2                       Condensin Complex Subunit; Sister Chromatid <br />                                                                                                              Segregation <br />  At2g34780                   EMB1611                     C                         13                        Uncertain <br />  At2g38280                   FAC1                        C                         10.1                      AMP Deaminase; Purine Biosynthesis <br />  At2g38670                   PECT1                       C                         10.5                      Phosphatidylethanolamine Biosynthesis <br />  At3g10220                   EMB2804                     C                         11.1                      Tubulin Folding Cofactor B; Microtubule Dynamics <br />  At3g20070                   TTN9                        C                         13                        Unknown <br />  At3g60740                   TTN1                        C                         11.1                      Tubulin Folding Cofactor D; Microtubule Dynamics <br />  At4g03240                   AtFH                        C                         12                        Frataxin; Mitochondrial Fe-S Protein Biogenesis <br />  At4g21800                   QQT2                        C                         11.1                      ATP/GTP Binding Protein; Microtubule Localization <br />  At4g39920                   POR                         C                         11.1                      Tubulin Folding Cofactor C; Microtubule Dynamics <br />  At5g13480                   FY                          C                         2.3                       Nuclear RNA Binding / Polyadenylation Protein <br />  At5g13690                   CYL1                        C                         10.4                      Alpha-N-Acetyl-Glucosaminidase <br />  At5g14800                   EMB2772                     C                         10.1                      Proline Biosynthesis <br />  At5g15540                   EMB2773                     C                         6.2                       Adherin; Sister Chromatid Cohesion <br />  At5g15920                   EMB2782                     C                         6.2                       SMC Family Protein; Chromosome Dynamics <br />  At5g24670                   EMB2820                     C                         2.3                       tRNA Adenosine Deaminase <br />  At5g48840                   PTS                         C                         10.1                      Pantothenate Synthetase <br />  At5g59440                   ZEUS1                       C                         10.1                      Thymidylate Kinase; Nucleotide Biosynthesis <br />  <br />  a <br />   Gene responsible mutant phenotype confirmed (C) confirmed (NC) allelism tests molecular complementation. <br />  b <br />   Protein function classes defined Table S1, spreadsheet tab. <br />  doi:10.1371/journal.pone.0028398.t002 <br />  <br />  <br /> EMB loci functional male gametophytes preglobular                      Arabidopsis [58&#226;&#8364;&#8220;62] limited scope support refute <br /> (25 loci; Table 2) globular (50 loci; Table S10) stage embryo               prediction. Only handful proteins derived UT-EMB <br /> arrest. We expected higher levels transcript young                  genes early stages embryo arrest included these <br /> microspores pollen tubes mature pollen grains                     datasets, GAM, GEM, EMG classes are <br /> pre-meiotic transcripts supported male gametophyte development.                   infrequently represented (Table S11). <br /> By contrast, anticipated transcript levels increase                 Transcripts UT-EMB genes early embryo defects are <br /> remain steady gametophyte development GAM                          frequently detected young microspores, levels not <br /> loci severe defects male transmission, consistent post-              increase substantially pollen development. Both the <br /> meiotic gene expression. EMG GEM loci combination                      transcript profiles inheritance patterns EMB genes <br /> of embryo defects moderate defects male gametophyte                        consistent utilization stored transcript <br /> development expected intermediate variable                        pollen development inconsistent gametophyte <br /> profiles.                                                                         expression. By contrast, genes required male gametophyte <br />    Our analysis male transcriptome datasets summarized                   development up-regulated pollen development <br /> Table 3. Overall, results different laboratories                     transcripts frequently detected pollen <br /> remarkably consistent highlight informative trends. Tran-                     maturity pollen tubes. As expected, EMG GEM loci <br /> scripts UT- EMB loci early defects seed development                  moderate defects male transmission transcript profiles <br /> are likely absent mature pollen (x2 = 17.1;                       intermediate preglobular EMB severe male <br /> p,0.001) transcripts gametophyte loci severe                       gametophyte loci. Transcript profiles EMB genes a <br /> defects male transmission. This suggests survival emb                  globular stage arrest difficult interpret remain <br /> mutant pollen tubes depends proteins synthesized                  consistent pre-meiotic transcription inconsistent with <br /> pollen development stored mRNAs translated                       widespread storage transcripts mature pollen. Globular <br /> pollen maturation. This model predicts EMB proteins                   mutant embryos appear arrest later development <br /> be present mature pollen grains transcripts                     simply EMB transcripts stored higher levels in <br /> absent. Unfortunately, published proteome datasets                        mature pollen. Whether protein products transcripts <br />  <br />  <br />       PLoS ONE | www.plosone.org                                              6                        December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                                                 Reproductive Lethality Arabidopsis <br />  <br />  <br />  <br />  Table 3. Summary pollen development transcriptome data selected essential genesa. <br />  <br />  <br />                                                                     Unique true EMB genesb                       Male gametophyte defectivesc <br />  <br />  Transcript accumulation pattern male <br />  gametophytes                                                       Preglobulard            Globulard            Moderated           Severed              Reference <br />  <br />  Transcript detected early pollen development                70%                     64%                  55%                 27%                  [42] <br />  mature pollen <br />  Transcript detected pollen development                   17%                    29%                  43%                 61%                  [42] <br />  Other patterns transcript accumulation                          13%                     7%                   2%                  12%                 [42] <br />  Transcript detected pollen                              0%                      2%                   2%                  26%                  [42] <br />  development; $26 higher late stages <br />  Transcript detected pollen                              4%                      10%                  12%                 13%                  [42] <br />  development; $26 higher early stages <br />  Comparison results obtained different <br />  laboratories <br />  Transcript detected mature pollene                              26%                     36%                  43%                 69%                  [42] <br />  Substantial transcript detected mature pollenf,g                9%                      12%                  21%                 47%                  [42] <br />  Transcript detected mature pollene,g                            13%                     14%                  29%                 42%                  [43] <br />  Transcript detected mature polleng                              13%                     19%                  21%                 51%                  [44] <br />  Transcript detected pollen tubeg                                13%                     14%                  21%                 45%                  [44] <br />  <br />  a <br />    Based published, large-scale microarray datasets wild-type plants [42&#226;&#8364;&#8220;44]. <br />  b <br />     Non-redundant genes embryo arrest stage shown; gametophyte defects observed. <br />  c <br />    Moderate (EMG GEM) severe (GAM) classes described text. <br />  d <br />     Percentages essential genes transcript detected noted. Total number genes analyzed: preglobular (23/25); globular (42/50); moderate (42/48); severe (51/ <br />     59). Excluded genes microarray dataset transcript detected pollen development. <br />  e <br />     Level detected 50% transcript levels recorded. <br />  f <br />    Level detected 25% transcript levels recorded. This cutoff value detection [42] gave results consistent obtained elsewhere <br />    [43,44]. <br />  g <br />     Most genes identified positive signal using data laboratory identified using data laboratories. <br />  doi:10.1371/journal.pone.0028398.t003 <br />  <br />  <br /> are abundant pollen retained sufficient levels                       remain functional young microspores. Surprisingly, transcripts <br /> after fertilization support embryo development remains                       similar percentage GAM GEM loci defects in <br /> an open question.                                                                       male transmission detected microsporocytes. This <br />                                                                                         contrasts known inheritance pattern gametophyte <br /> Transcriptional profiles essential genes                                          defects observed. Furthermore, frequency genes with <br /> microsporocytes                                                                         abundant transcripts (.15.0 RPKM) significantly higher <br />    To test prediction transcripts EMB genes                       UT-EMB genes early embryo defects male <br /> contributed diploid microsporocytes haploid gameto-                             gametophyte essentials (x2 = 0.61; p = 0.44), genes with <br /> phytes, evaluated recent reports transcript profiles                    transcripts detected microsporocytes (x2 = 0.22; p = 0.64). We <br /> isolated male meiocytes. Two studies utilized direct RNA                       propose cases transcription an <br /> sequencing [63,64]; relied microarray analysis [65].                       EMB gene microsporocytes supports gametophyte development, <br /> One report [64] characterized transcriptional landscape                     survival mutant (emb) gametophytes results relatively <br /> isolated microsporocytes; [63,65] emphasized                          modest levels residual transcript contributed young <br /> identification meiosis-specific gene products. Of 183 GAM,                       microspores subsequently translated pollen develop- <br /> GEM, EMG, EMB genes listed Table 2 Tables S8, S9,                            ment. By contrast, male gametophyte defects gametophyte <br /> S10 combined, just included 844 meiosis-specific                       lethals arise post-meiotic expression essential gene is <br /> loci described Yang et al. [64]. This surprising                      required supplement functional transcript remains <br /> most genes dataset encode proteins thought                          pre-meiotic expression microsporocytes. Confirmation of <br /> required multiple stages development. Similarly, 5%                       model achieved future determining whether <br /> the 183 essential genes dataset included 752                           down-regulation EMB gene microsporocytes changes an <br /> transcripts Libeau et al. [65] conclude enriched                            embryo lethal gametophyte lethal. <br /> microsporocytes compared leaves roots. This is <br /> intriguing suggests microsporocytes amplify                      Transcriptional profiles essential genes in <br /> essential gene transcripts preparation subsequent use                     megasporocytes <br /> male gametophyte development.                                                              We focused analysis genes required male gameto- <br />    This model supported analysis dataset                    phyte development data female meiocytes more <br /> Yang et al. [64], shown Table 4. As expected,                               difficult obtain evaluate. One argue the <br /> transcripts 85% UT-EMB genes early embryo                            number confirmed, female gametophyte mutants identified to <br /> defects detected microsporocytes, compared just 55%                         date fewer number male gametophyte mutants, <br /> for protein-coding genes. Many transcripts                          megasporocytes effective microsporocytes at <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                   7                           December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                                             Reproductive Lethality Arabidopsis <br />  <br />  <br />  <br />  Table 4. Summary microsporocyte transcriptome data selected essential genesa. <br />  <br />  <br />                                      Unique true EMB genesb                          Male gametophyte defectivesc <br />  <br />  Transcript detected in <br />  microsporocytesd                    Preglobular              Globular               Moderate             Severe              Microsporocyte dataset full <br />  <br />  $1.0 RPKM                           88%                      82%                    90%                  81%                 55% <br />  .5.0 RPKMe                          41%                      41%                    53%                  52%                 49% <br />  .15.0 RPKMe                         14%                      17%                    23%                  19%                 18% <br />  <br />  a <br />    Based published transcriptome dataset Yang et al. [64]. Percentages essential genes transcripts detected different abundance levels noted. <br />  b <br />    Non-redundant genes early embryo arrest gametophyte defects. Genes analyzed: preglobular [25]; globular [50]. <br />  c <br />   Moderate (EMG GEM) severe (GAM) classes described text. Genes analyzed: moderate [48]; severe [59]. <br />  d <br />    Reads kilobase million reads (RPKM); high numbers indicate abundant transcripts. <br />  Transcripts RPKM$1.0 considered present. <br />  e <br />    Genes transcript detected RPKM,1.0 excluded percentages shown. <br />  doi:10.1371/journal.pone.0028398.t004 <br />  <br />  <br /> contributing functional gene products progeny haploid cells.                       The transition maternal zygotic gene expression <br /> This cannot, however, true essential genes, RNA                       In addition evaluating sporocyte transcription gameto- <br /> polymerase knockouts transmitted female side,                     phyte viability embryo-defective mutants Arabidopsis, we <br /> which indicates essential genes transcribed                      utilized dataset EMB genes presented address <br /> female gametophytes. While logical knockouts                         central issue plant development related stored, <br /> genes required rapid tip growth exhibit male                        essential gene products. Two fundamental questions figured <br /> than female gametophyte lethality, mechanism responsible                      prominently recent discussions gene expression plant <br /> severe female defects knockouts minimal disruption male                    embryo development. One involves extent paternal allele <br /> gametophytes remains obscure. In cases, gene duplication                         contributions developing embryo endosperm tissue. <br /> may involved, redundant gene supplying missing                          The concerns development transition from <br /> function male paralog required female                        maternal zygotic gene expression takes place. Both these <br /> development disrupted.                                                             questions widely studied maize [68&#226;&#8364;&#8220;70] and <br />    The recent publication transcriptional profiles Arabidopsis                 Arabidopsis [71&#226;&#8364;&#8220;77]. However, significant conflicts remain over <br /> megasporocytes [66] female gametophytes [67] makes                             interpretation results obtained, reliability methods <br /> possible analyze maternal contributions haploid gameto-                         employed, validity conclusions drawn. <br /> phytes role pre-meiotic transcription supporting                           Our field began 20 years ago, we <br /> embryo development. Transcripts essential genes                           isolated characterized mutant (emb173) unusual <br /> detected megasporocytes. Of 75 UT-EMB genes early                         pattern inheritance: heterozygous plants produced 50% mutant <br /> embryo defects, 62% produced transcripts detected                         seeds regardless pollen genotype [36]. Additional alleles this <br /> three megasporocyte samples analyzed [66]. This                            gene, later named MEA, isolated characterized <br /> consistent model transcripts produced                           elsewhere, gene shown encode polycomb group <br /> megasporocytes used support female gametophyte devel-                          protein regulates cell proliferation, paternal allele was <br /> opment. We asked genes associated severe                            imprinted DNA methylation, particularly the <br /> defects female transmission fail produce transcripts                      endosperm [78&#226;&#8364;&#8220;80]. Several genes similar pattern of <br /> detected megasporocytes. Unexpectedly, 19 genes                          inheritance [www.seedgenes.org] differing seed phenotypes <br /> (Table S2) examined, likelihood transcripts                         mechanisms maternal influence described <br /> present megasporocyte dataset [66] significantly                       [81,82]. Evaluation genes mutant phenotypes has <br /> different (x2 = 0.24; p = 0.62) observed 75 UT-                     frequently centered maternal genome contributions and <br /> EMB genes early phenotypes, knockouts survive                              paternal genomic imprinting relation parental conflict over <br /> gametophyte development. This result mirrors trend noted                          resource allocation mother offspring. Despite considerable <br /> above microsporocyte transcripts involving male gametophyte                       origin significance loci, numbers <br /> essential genes. Furthermore, significant difference                      quite limited compared EMB genes normal, <br /> (x2 = 1.81; p = 0.40) likelihood transcripts present               zygotic pattern inheritance. <br /> megasporocytes, available egg cell contribution                        We surprised report 12 years ago that <br /> the zygote, compared datasets UT-EMB genes                             claimed widespread, delayed activation paternal genome <br /> with preglobular, globular, cotyledon mutant phenotypes. We                       seed development Arabidopsis concluded early <br /> have unable establish role pre-meiotic                        embryo endosperm development primarily, not <br /> transcripts supporting continued growth mutant embryos                      exclusively, maternal control [71]. This conclusion, which <br /> to globular stage beyond. By contrast, observed differences                     appeared inconsistent inheritance pattern early <br /> in severity phenotypes male female                                 embryo-lethal mutants, based analysis &#195;&#376;-glucuron- <br /> gametophyte mutants disrupted essential genes,                         idase (GUS) enhancer trap lines marked expression 19 <br /> embryo gametophyte lethals overall, appear cases                       different genes known active ovule early seed <br /> reflect differences residual, essential gene                         development. By comparing presence absence GUS <br /> products donated sporocytes developing gametophytes.                          staining embryos produced reciprocal crosses wild- <br /> The practical applications different patterns reproduc-                   type plants, authors concluded paternal gene each <br /> tive lethality Arabidopsis remain explored.                                  case expressed 3 4 days pollination, the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                8                           December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                               Reproductive Lethality Arabidopsis <br />  <br />  <br /> embryo reached early globular stage. Additional studies             detailed picture molecular basis reproductive lethality in <br /> with EMB30/GNOM tests allele-specific single nucleotide             Arabidopsis emerge additional genes essential <br /> polymorphisms (SNPs) transcripts produced following reciprocal           functions identified future. <br /> crosses accessions support observations. <br /> This model soon challenged work                      Materials Methods <br /> Arabidopsis [72]. With maize, published reports both <br /> consistent [68] inconsistent [69] model. Down-                 Updating published dataset EMB genes in <br /> regulating RNA polymerase II activity fertilization later         Arabidopsis <br /> found interfere endosperm early embryo                         The published, comprehensive dataset EMB genes [30] <br /> development Arabidopsis [73]. Once again, suggested            compiled 7 years ago included total 220 <br /> the embryo transcriptionally quiescent globular stage         loci, 70% uncovered forward genetic <br /> and dependent instead stored maternal transcripts.                    screen undertaken collaboration Syngenta [29]. Since that <br />    Recently, large-scale RNA sequencing effort [77] designed           time, SeedGenes database undergone major updates <br /> detect accession-specific SNPs Arabidopsis embryos 2&#226;&#8364;&#8220;4          include additional genes mutant alleles. In addition to <br /> cell stage indicated genome-wide, transcripts                 curating information literature, large-scale, <br /> maternal allele predominate paternal allele             reverse genetic approaches pursued laboratory to <br /> through globular stage development. Following             identify EMB genes missed past screens [24]. These <br /> cross Columbia Landsberg erecta accessions,              included analysis candidate genes encoding proteins that <br /> 90% 135,000 informative sequence reads derived                  shared common cellular process (aminoacylation tRNAs), <br /> 4,000 loci genome-wide appeared maternal origin             metabolic pathway (biotin histidine biosynthesis), and <br /> 2&#226;&#8364;&#8220;4 cell embryo stage. Overall, paternal contribution increased         intracellular compartment (chloroplast) products known <br /> somewhat globular stage, loci produced equal                EMB genes [48&#226;&#8364;&#8220;50,83]. We used PCR (polymerase chain <br /> amounts transcripts maternal paternal alleles,             reaction) amplify plant sequences flanking T-DNA inserts for <br /> but early development, vast majority loci            mutants mapped cloned [45]. In <br /> under maternal control. The authors stated work                  addition, analyzed candidate genes represented potential <br /> reconciled observations different laboratories              homologs essential genes identified organisms [30], <br /> maternal zygotic effects embryogenesis Arabidopsis.           genes encoded proteins thought interact known <br /> But data conflict existence large numbers        EMB protein, hundreds insertion lines corresponding to <br /> embryo-defective mutants Mendelian pattern inheritance            genes Ecker project providing knockouts every <br /> and cellular defects observed globular stage.               gene [38] failed identify viable homozygotes. Whenever <br />    Because publications maternal-to-zygotic               possible, genetic crosses heterozygotes used to <br /> transition Arabidopsis cited handful EMB genes         confirm allelism demonstrate gene responsible the <br /> with known defects early development, decided establish          mutant phenotype identified. Crosses used to <br /> more robust dataset known EMB genes arrested embryos                identify new alleles known EMB genes previously been <br /> that fail reach globular stage. The resulting collection 70       mapped cloned [45]. Methods used laboratory to <br /> genes, corresponding 18% 396 EMB genes                      analyze embryo-defective mutants presented tutorial <br /> identified date, presented Table S12. This dataset does        section SeedGenes recent publications [24,45,48&#226;&#8364;&#8220; <br /> include genes EMB30/GNOM, initial defects             50,83]. Although genetic phenotypic data most <br /> observed long embryo development arrested,               promising mutants consistently added SeedGenes <br /> other mutants late terminal phenotypes early defects             database, EMB genes identified laboratory and <br /> remain uncovered. Nevertheless, dataset hand,            Table S1 published time. <br /> we intrigued learn study maternal-to- <br /> zygotic transition Arabidopsis followed approach similar         Establishing dataset genes required for <br /> to Autran et al. [77] expanded number genes                  gametophyte development <br /> embryo stages examined, included analysis reciprocal                    Several approaches taken identify evaluate genes <br /> crosses, increased washing isolated embryos remove            reported required gametophyte development in <br /> contaminating RNA derived maternal seed coat (Michael              Arabidopsis. We began list known genes mutant <br /> Nodine David Bartel, personal communication). Their results             phenotype published 8 years ago [84], examined candidate genes <br /> for 7,000 loci present different picture, maternal         The Arabidopsis Information Resource (www.arabidopsis. <br /> and paternal transcripts genes equal amounts,             org) thought associated phenotype <br /> even 1&#226;&#8364;&#8220;2 cell stage. Transcripts 70 preglobular             information, performed variety PubMed searches <br /> EMB genes listed Table S12 exhibit similar, bi-parental                (www.ncbi.nlm.nih.gov) relevant keywords (gamete, gameto- <br /> pattern accumulation, consistent known mode                phyte, mutant, mutation, knockout, lethal, essential, male, female, <br /> inheritance.                                                                pollen). Publications describing large-scale identification of <br />    When results evaluated context               genes required male female gametophyte development <br /> decades research embryo-defective mutants, believe           [31,35] evaluated identify genes likely be <br /> is compelling evidence early activation zygotic genome           responsible gametophyte defects observed. Emphasis was <br /> in Arabidopsis, unlike situation observed animal       placed genes multiple mutant alleles flanking <br /> systems, early expression essential genes required support       sequence derived sides insert. We utilized <br /> wide range cellular functions beginning shortly fertiliza-         information presented SeedGenes publications on <br /> tion. From historical perspective, elucidating role paternal       embryo-defective mutants determine mutants exhibited <br /> gene expression plant embryo development provides                      additional defects gametophyte development. Transmission <br /> compelling illustration value large datasets genes        efficiency (TE) mutant allele, determined crosses between <br /> essential functions growth development. A                   heterozygous wild-type plants, defined as: (number of <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        9                       December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                                                      Reproductive Lethality Arabidopsis <br />  <br />  <br /> heterozygous F1 plants) / (number wild-type F1 plants).                                   Table S7 36 unique true EMB genes late pheno- <br /> Subclasses genes different mixtures embryo                                    types putative knockout alleles. <br /> gametophyte defects (GAM, GEM, EMG) levels transmis-                                  (XLS) <br /> sion reduction (e.g. 0 =; MMM, MM, M, +) defined                                  Table S8 59 GAM GEM genes severe defects in <br /> second tabbed spreadsheets Table S2. The percentage                             male gamete transmission. <br /> of mutant seeds expected selfed siliques heterozygotes                            (XLS) <br /> reduced transmission mutant allele both <br /> gametes calculated as: [(= TE)/(1+= TE )]6[(R TE)/(1+R                                   Table S9 48 GEM EMG genes moderate defects <br /> TE)]6100, &#226;&#8364;&#732;&#226;&#8364;&#732;1&#226;&#8364;&#8482;&#226;&#8364;&#8482; represents normal transmission efficiency                           male gamete transmission. <br /> the wild-type allele. This reduces [1/(1+1)]6[1/(1+1)]6100,                            (XLS) <br /> 25% mutant seeds absence defects gamete                                         Table S10 50 unique true EMB genes early <br /> transmission.                                                                                globular globular phenotypes. <br />                                                                                              (XLS) <br /> Supporting Information <br />                                                                                              Table S11 Detection proteins required male <br /> Table S1 Updated SeedGenes dataset 396 EMB genes                                          gametophyte embryo development pollen. <br /> of Arabidopsis.                                                                              (XLS) <br /> (XLS) <br />                                                                                              Table S12  70 EMB genes mutant embryos arrest <br /> Table S2   173 Arabidopsis genes gametophyte-                                         preglobular stage development. <br /> defective phenotype.                                                                         (XLS) <br /> (XLS) <br /> Table S3 352 true EMB genes evidence                                              Acknowledgments <br /> gametophyte defects.                                                                         We thank members Meinke laboratory Oklahoma State <br /> (XLS)                                                                                        University, Allan Dickerman Virginia Bioinformatics Institute, and <br />                                                                                              David Patton Syngenta contributing establishment the <br /> Table S4 72 GEM EMG genes embryo                                               SeedGenes database. Michael Nodine laboratory David Bartel at <br /> and gametophyte defects.                                                                     Massachusetts Institute Technology kindly shared unpublished data <br /> (XLS)                                                                                        transcript patterns preglobular EMB loci. <br /> Table S5 152 unique true EMB genes Arabidopsis. <br /> (XLS)                                                                                        Author Contributions <br />                                                                                              Conceived designed experiments: DM RM. Wrote paper: DM. <br /> Table S6 Examples EMB genes pheno-                                             Established, analyzed, curated embryo datasets: DM RM JL. <br /> types weak alleles RNAi lines.                                                         Established, analyzed, curated gametophyte datasets: JL DM. <br /> (XLS)                                                                                        Analyzed transcriptome proteome datasets: RM DM. <br />  <br /> References <br />  1. Osterman AL, Gerdes SY (2008) Microbial Gene Essentiality: Protocols                 14. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, et al. (2007) A genome- <br />     Bioinformatics. Methods Molecular Biology (Vol 416). Totowa: Humana                       wide transgenic RNAi library conditional gene inactivation Drosophila. <br />     Press. pp 436.                                                                               Nature 448: 151&#226;&#8364;&#8220;156. <br />  2. Msadek T (2009) Grasping shadows: Revealing elusive nature essential           15. Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, et al. (2003) Systematic <br />     genes. J Bacteriol 191: 4701&#226;&#8364;&#8220;4704.                                                           functional analysis Caenorhabditis elegans genome using RNAi. Nature 421: <br />  3. Ji Y, Zhang B, Van Horn SF, Warren P, Woodnutt G, et al. (2001) Identification               231&#226;&#8364;&#8220;237. <br />     critical Staphylococcal genes using conditional phenotypes generated               16. Mouse Genome Informatics (MGI) Website. Available: www.informatics.jax.org. <br />     antisense RNA. Science 293: 2266&#226;&#8364;&#8220;2269.                                                       Accessed 2011 November 11. <br />  4. Akerley BJ, Rubin EJ, Novick VL, Amaya K, Judson N, et al. (2002) A genome-              17. Brown SDM, Wurst W, Ku&#194;&#168;hn R, Hancock JM (2009) The functional annotation <br />     scale analysis identification genes required growth survival                mammalian genomes: The challenge phenotyping. Annu Rev Genet 43: <br />     Haemophilus influenzae. Proc Natl Acad Sci USA 99: 966&#226;&#8364;&#8220;971.                                  305&#226;&#8364;&#8220;333. <br />  5. Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, et al. (2003)                18. Silva JM, Marran K, Parker JS, Silva J, Golding M, et al. (2008) Profiling <br />     Essential Bacillus subtilis genes. Proc Natl Acad Sci USA 100: 4678&#226;&#8364;&#8220;4683.                    essential genes human mammary cells multiplex RNAi screening. Science <br />  6. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction                    319: 617&#226;&#8364;&#8220;620. <br />     Escherichia coli K-12 in-frame, single-<span id='am-321' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-322' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-397' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants: The Keio                19. Park D, Park J, Park SG, Park T, Choi SS (2008) Analysis human disease <br />     collection. Mol Syst Biol 2: 2006.0008.                                                      genes context gene essentiality. Genomics 92: 414&#226;&#8364;&#8220;418. <br />  7. Giaever G, Chu AM, Ni L, Connelly C, Riles L, et al. (2002) Functional                   20. Liang H, Li WH (2009) Functional compensation duplicated genes mouse. <br />     profiling Saccharomyces cerevisiae genome. Nature 418: 387&#226;&#8364;&#8220;391.                       Trends Genet 25: 441&#226;&#8364;&#8220;442. <br />  8. Ben-Aroya S, Coombes C, Kwok T, O&#226;&#8364;&#8482;Donnell KA, Boeke JD, et al. (2008)                    21. Chen S, Zhang YE, Long M (2010) New genes Drosophila quickly become <br />     Toward comprehensive temperature-sensitive mutant repository                        essential. Science 330: 1682&#226;&#8364;&#8220;1685. <br />     essential genes Saccharomyces cerevisiae. Mol Cell 30: 248&#226;&#8364;&#8220;258.                       22. Liao BY, Zhang J (2008) Null mutations human mouse orthologs <br />  9. Hillenmeyer ME, Fung E, Wildenhain J, Pierce SE, Hoon S, et al. (2008) The                   frequently result different phenotypes. Proc Natl Acad Sci USA 105: <br />     chemical genomic portrait yeast: Uncovering phenotype genes.                    6987&#226;&#8364;&#8220;6992. <br />     Science 320: 362&#226;&#8364;&#8220;365.                                                                    23. Liao BY, Weng MP, Zhang J (2010) Contrasting genetic paths morphological <br /> 10. Kim DU, Hayles J, Kim D, Wood V, Park HO, et al. (2010) Analysis                        physiological evolution. Proc Natl Acad Sci USA 107: 7353&#226;&#8364;&#8220;7358. <br />     genome-wide set gene deletions fission yeast Schizosaccharomyces pombe.        24. Meinke D, Muralla R, Sweeney C, Dickerman A (2008) Identifying essential <br />     Nat Biotechnol 28: 617&#226;&#8364;&#8220;623.                                                                  genes Arabidopsis thaliana. Trends Plant Sci 13: 483&#226;&#8364;&#8220;491. <br /> 11. Zhang R, Lin Y (2009) DEG 5.0, database essential genes                     25. Meinke D, Muralla R, Dickerman A (2009) An essential line inquiry. Trends <br />     prokaryotes eukaryotes. Nucleic Acids Res 37: D455&#226;&#8364;&#8220;D458.                                 Plant Sci 14: 414&#226;&#8364;&#8220;415. <br /> 12. Perrimon N, Lanjuin A, Arnold C, Noll E (1996) Zygotic lethal mutations             26. Meinke DW, Sussex IM (1979) Embryo-lethal mutants Arabidopsis thaliana: A <br />     maternal effect phenotypes Drosophila melanogaster. II. Loci second            model genetic analysis plant embryo development. Devel Biol 72: <br />     chromosomes identified P-element-induced mutations. Genetics 144:                   50&#226;&#8364;&#8220;61. <br />     1681&#226;&#8364;&#8220;1692.                                                                               27. Meinke DW (1985) Embryo-lethal mutants Arabidopsis thaliana: Analysis of <br /> 13. Boutros M, Kiger AA, Armknecht S, Kerr K, Hild M, et al. (2004) Genome-                      mutants wide range lethal phases. Theor Appl Genet 69: 543&#226;&#8364;&#8220;552. <br />     wide RNAi analysis growth viability Drosophila cells. Science 303:             28. Franzmann LH, Yoon ES, Meinke DW (1995) Saturating genetic map of <br />     832&#226;&#8364;&#8220;835.                                                                                     Arabidopsis thaliana embryonic mutations. Plant J 7: 341&#226;&#8364;&#8220;350. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      10                           December 2011 | Volume 6 | Issue 12 | e28398 <br />                                                                                                                                           Reproductive Lethality Arabidopsis <br />  <br />  <br /> 29. McElver J, Tzafrir I, Aux G, Rogers R, Ashby C, et al. (2001) Insertional                   58. Holmes-Davis R, Tanaka CK, Vensel WH, Hurkman WJ, McCormick S (2005) <br />     mutagenesis genes required seed development Arabidopsis thaliana.                     Proteome mapping mature pollen Arabidopsis thaliana. Proteomics 5: <br />     Genetics 159: 1751&#226;&#8364;&#8220;1763.                                                                        4864&#226;&#8364;&#8220;4884. <br /> 30. Tzafrir I, Pena-Muralla R, Dickerman A, Berg M, Rogers R, et al. (2004)                     59. Noir S, Bra&#194;&#168;utigam A, Colby T, Schmidt J, Panstruga R (2005) A reference map <br />     Identification genes required embryo development Arabidopsis. Plant                   Arabidopsis thaliana mature pollen proteome. Biochem Biophys Res <br />     Physiol 135: 1206&#226;&#8364;&#8220;1220.                                                                         Commun 337: 1257&#226;&#8364;&#8220;1266. <br /> 31. Pagnussat GC, Yu HJ, Ngo QA, Rajani S, Mayalagu S, et al. (2005) Genetic                60. Sheoran IS, Sproule KA, Olson DJH, Ross ARS, Sawhney VK (2006) Proteome <br />     molecular identification genes required female gametophyte development                   prolife functional classification proteins Arabidopsis thaliana (Landsberg <br />     function Arabidopsis. Development 132: 603&#226;&#8364;&#8220;614.                                          erecta) mature pollen. Sex Plant Reprod 19: 185&#226;&#8364;&#8220;196. <br /> 32. Procissi A, Laissardie`re S, Fe&#194;&#180;rault M, Vezon D, Pelletier G, et al. (2001) Five        61. Zou J, Song L, Zhang W, Wang Y, Ruan S, et al. (2009) Comparative proteomic <br />     gametophytic mutations affecting pollen development pollen tube growth                   analysis Arabidopsis mature pollen germinated pollen. J Integ Plant Biol 51: <br />     Arabidopsis thaliana. Genetics 158: 1773&#226;&#8364;&#8220;1783.                                                  438&#226;&#8364;&#8220;455. <br /> 33. Lalanne E, Michaelidis C, Moore JM, Gagliano W, Johnson A, et al. (2004)                    62. Grobei MA, Qeli E, Brunner E, Rehrauer H, Zhang R, et al. (2009) <br />     Analysis transposon insertion mutants highlights diversity mechanisms                 Deterministic protein inference shotgun proteomics data provides new <br />     underlying male progamic development Arabidopsis. Genetics 167: 1975&#226;&#8364;&#8220;1986.                   insights Arabidopsis pollen development function. Genome Res 19: <br /> 34. Johnson MA, von Besser K, Zhou Q, Smith E, Aux G, et al. (2004) Arabidopsis                     1786&#226;&#8364;&#8220;1800. <br />     hapless mutations define essential gametophytic functions. Genetics 168:                    63. Chen C, Farmer AD, Langley RJ, Mudge J, Crow JA, et al. (2010) Meiosis- <br />     971&#226;&#8364;&#8220;982.                                                                                        specific gene discovery plants: RNA-Seq applied isolated Arabidopsis male <br /> 35. Boavida LC, Shuai B, Yu HJ, Pagnussat GC, Sundaresan V, et al. (2009) A <br />                                                                                                     meiocytes. BMC Plant Biol 10: 280. <br />     collection Ds insertional mutants associated defects male gametophyte <br />                                                                                                 64. Yang H, Lu P, Wang Y, Ma H (2011) The transcriptome landscape Arabidopsis <br />     development function Arabidopsis thaliana. Genetics 181: 1369&#226;&#8364;&#8220;1385. <br />                                                                                                     male meiocytes high-throughput sequencing: The complexity and <br /> 36. Castle LA, Errampalli D, Atherton TL, Franzmann LH, Yoon ES, et al. (1993) <br />                                                                                                     evolution meiotic process. Plant J 65: 503&#226;&#8364;&#8220;516. <br />     Genetic molecular characterization embryonic mutants identified <br />     following seed transformation Arabidopsis. Mol Gen Genet 241: 504&#226;&#8364;&#8220;514.                   65. Libeau P, Durandet M, Granier F, Marquis C, Berthome&#194;&#180; R, et al. (2011) Gene <br /> 37. Tax FE, Vernon DM (2001) T-DNA-associated duplication/translocations                         expression profiling Arabidopsis meiocytes. Plant Biol 13: 784&#226;&#8364;&#8220;793. <br />     Arabidopsis. Implications mutant analysis functional genomics. Plant                66. Schmidt A, Wuest SE, Vijverberg K, Baroux C, Kleen D, et al. (2011) <br />     Physiol 126: 1527&#226;&#8364;&#8220;1538.                                                                         Transcriptome analysis Arabidopsis megaspore mother cell uncovers the <br /> 38. O&#226;&#8364;&#8482;Malley RC, Ecker JR (2010) Linking genotype phenotype using                            importance RNA helicases plant germline development. PLoS Biol 9: <br />     Arabidopsis unimutant collection. Plant J 61: 928&#226;&#8364;&#8220;940.                                          e1001155. <br /> 39. Clark KA, Krysan PJ (2010) Chromosomal translocations common                          67. Wuest SE, Vijverberg K, Schmidt A, Weiss M, Gheyselinck J, et al. (2010) <br />     phenomenon Arabidopsis thaliana T-DNA insertion lines. Plant J 64: 990&#226;&#8364;&#8220;1001.                 Arabidopsis female gametophyte gene expression map reveals similarities between <br /> 40. Johnston AJ, Meier P, Gheyselinck J, Wuest SE, Federer M, et al. (2007) Genetic                 plant animal gametes. Curr Biol 20: 506&#226;&#8364;&#8220;512. <br />     subtraction profiling identifies genes essential Arabidopsis reproduction           68. Grimanelli D, Perotti E, Ramirez J, Leblanc O (2005) Timing maternal- <br />     reveals interaction female gametophyte maternal                             to-zygotic transition early seed development maize. Plant Cell 17: <br />     sporophyte. Genome Biol 8: R204.                                                                1061&#226;&#8364;&#8220;1072. <br /> 41. Drews GN, Wang D, Steffen JG, Schumaker KS, Yadegari R (2011)                               69. Meyer S, Scholten S (2007) Equivalent parental contribution early plant <br />     Identification genes expressed angiosperm female gametophyte. J Exp                   zygotic development. Curr Biol 17: 1686&#226;&#8364;&#8220;1691. <br />     Bot 62: 1593&#226;&#8364;&#8220;1599.                                                                          70. Jahnke S, Scholten S (2009) Epigenetic resetting gene imprinted plant <br /> 42. Honys D, Twell D (2004) Transcriptome analysis haploid male gametophyte                      embryos. Curr Biol 19: 1677&#226;&#8364;&#8220;1681. <br />     development Arabidopsis. Genome Biol 4: R85.                                             71. Vielle-Calzada JP, Baskar R, Grossniklaus U (2000) Delayed activation the <br /> 43. Pina C, Pinto F, Feijo&#194;&#180; JA, Becker JD (2005) Gene family analysis                        paternal genome seed development. Nature 404: 91&#226;&#8364;&#8220;94. <br />     Arabidopsis pollen transcriptome reveals biological implications cell growth,           72. Weijers D, Geldner N, Offringa R, Ju&#194;&#168;rgens G (2001) Early paternal gene activity <br />     division control, gene expression regulation. Plant Physiol 138: 744&#226;&#8364;&#8220;756.                   Arabidopsis. Nature 414: 709&#226;&#8364;&#8220;710. <br /> 44. Wang Y, Zhang WZ, Song LF, Zou JJ, Su Z, et al. (2008) Transcriptome                        73. Pillot M, Baroux C, Vazquez MA, Autran D, Leblanc O, et al. (2010) Embryo <br />     analyses changes gene expression accompany pollen germination                    endosperm inherit distinct chromatin transcriptional states the <br />     tube growth Arabidopsis. Plant Physiol 148: 1201&#226;&#8364;&#8220;1211.                                       female gametes Arabidopsis. Plant Cell 22: 307&#226;&#8364;&#8220;320. <br /> 45. Meinke D, Sweeney C, Muralla R (2009) Integrating genetic physical                  74. Aw SJ, Hamamura Y, Chen Z, Schnittger A, Berger F (2010) Sperm entry is <br />     maps Arabidopsis thaliana: Identification mapped alleles cloned essential              sufficient trigger division central cell paternal genome is <br />     (EMB) genes. PLoS ONE 4: e7386.                                                                 required endosperm development Arabidopsis. Development 137: <br /> 46. Mu&#194;&#168;ller AJ (1963) Embryonentest zum Nachweis rezessiver Letalfaktoren bei                       2683&#226;&#8364;&#8220;2690. <br />     Arabidopsis thaliana. Biol Zentralbl 82: 133&#226;&#8364;&#8220;163.                                           75. Raissig MT, Baroux C, Grossniklaus U (2011) Regulation flexibility of <br /> 47. Meinke DW (1982) Embryo-lethal mutants Arabidopsis thaliana: Evidence                    genomic imprinting seed development. Plant Cell 23: 16&#226;&#8364;&#8220;26. <br />     gametophytic expression mutant genes. Theor Appl Genet 63: 543&#226;&#8364;&#8220;552.                  76. Xiang D, Venglat P, Tibiche C, Yang H, Risse <br /> </body></html>