<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Hindawi Publishing Corporation <br /> Archaea <br /> Volume 2010, Article ID 426239, 11 pages <br /> doi:10.1155/2010/426239 <br />  <br />  <br />  <br />  <br /> Research Article <br /> Towards Systems Approach Genetic Analysis of <br /> Archaea: Accelerating Mutant Construction Phenotypic <br /> Analysis Haloferax volcanii <br />  <br />           Ian K. Blaby, Gabriela Phillips, Crysten E. Blaby-Haas, Kevin S. Gulig, <br />           Basma El Yacoubi, Val&#194;&#180;erie Cr&#194;&#180;ecy-Lagard <br />           Department Microbiology Cell Science, University Florida, Gainesville, FL 32611-0700, USA <br />  <br />           Correspondence addressed Val&#194;&#180;erie Cr&#194;&#180;ecy-Lagard, vcrecy@ufl.edu <br />  <br />           Received 2 September 2010; Accepted 24 October 2010 <br />  <br />           Academic Editor: Li Huang <br />  <br />           Copyright &#194;&#169; 2010 Ian K. Blaby et al. This open access article distributed Creative Commons Attribution License, <br />           permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />           With availability genome sequence increasingly sophisticated genetic tools, Haloferax volcanii model <br />           Archaea halophiles. In order H. volcanii reach status equivalent Escherichia coli, Bacillus subtilis, or <br />           Saccharomyces cerevisiae, gene <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>knockout</span> collection needs constructed order identify archaeal essential gene set and <br />           enable systematic <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> screens. A streamlined gene-deletion protocol adapted potential automation implemented <br />           used generate 22 H. volcanii deletion strains identify potentially essential genes. These gene deletion mutants, <br />           generated previous studies, analyzed high-throughput fashion measure growth rates di&#239;&#172;&#8364;erent <br />           media temperature conditions. We conclude high-throughput methods suitable rapid investigation an <br />           H. volcanii mutant library suggest form basis larger genome-wide experiment. <br />  <br />  <br />  <br />  <br /> 1. Introduction                                                          [10, 11]. Yet, strict anaerobes, manipulation also <br />                                                                          restricted specialists. <br /> Since identification three-kingdom paradigm                   Thanks endeavours growing community of <br /> 30 years ago [1], quest model archaeal organisms,                investigators (see http://www.haloferax.org/), Haloferax vol- <br /> representing main groups, ongoing.                  canii filling role model archeaon model halo- <br /> <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> date, culturable Archaea, Sulfolobus sp. remain                phile. Its relatively short generation time &lt;4 hours [12, <br /> amongst intensely investigated              13], ease growth laboratory conditions, genome- <br /> subject modern postgenomics experimental techniques                   sequence availability [14], development genetic tools <br /> [2, 3], vivo studies organisms lagged,              [15&#226;&#8364;&#8220;17] (including shuttle, expression suicide vectors <br /> as robust genetic methodologies lacking (although               using auxotrophy, antibiotic-based selections) allow <br /> is addressed [4, 5]). The ability thermophiles,             gene deletions achieved relative ease. This <br /> such Thermococcus kodakaraensis Pyroccocus furiosus,              combination feats enabled rapid progress study <br /> to thrive excess 95&#226;&#8212;&#166; C great (particularly       halophiles aided implementation <br /> to industrials). However, whilst T. kodakaraensis well-            microarray proteomics experiments closely <br /> developed genetic system, optimal growth high temper-                 related organism Halobacterium salinarum [18&#226;&#8364;&#8220;21]. Hence, <br /> atures coupled anaerobic growth requirements puts                   H. salinarum/H. volcanii pair considered the <br /> these organisms reach nonspecialist laboratories               advanced model organisms Archaea. <br /> [6&#226;&#8364;&#8220;8]. The methanogens received early lead                        The construction genome-wide gene deletion libraries <br /> first available archaeal genome sequence, Methanococcus jan-             major achievements postgenomic <br /> naschii [9], genetic manipulation techniques               era. Initially, collections generated using random <br /> described closely related Methanococcus maripaludis               mutagenesis approaches, strain-sets using transposon <br />  2                                                                                                                        Archaea <br />  <br /> insertions described Escherichia coli [22],          100 &#206;&#188;l spheroplasts incubated 5 min. 100 &#206;&#188;l 60% <br /> Corynebacterium glutamicum [23], Haemophilus influenza             (v/v) PEG600 added mixed gently. The mixture <br /> [24], Pseudomonas aeruginosa [25, 26] others.            incubated room temperature 30 min. Cells were <br /> Yet, potential polar e&#239;&#172;&#8364;ects insertion down-          recovered adding recovery solution, described the <br /> stream genes lack certainty gene-inactivation        HaloHandbook, incubated 4 hrs 44&#226;&#8212;&#166; C before <br /> make transposon libraries far ideal. For reasons,       plating appropriate medium. <br /> targeted gene-deletion libraries method <br /> of choice, despite higher cost labor requirements.             2.2. Gene Deletion Strain Construction. Plasmids delete <br /> Such mutant collections constructed multiple          H. volcanii genes constructed follows. Regions of <br /> model organisms [27&#226;&#8364;&#8220;29] greatly aided gene-               approximately 600 bp upstream downstream gene <br /> function studies [30&#226;&#8364;&#8220;32]. The genetic tools available          deleted amplified polymerase chain reaction <br /> for halophiles, logistical knowledge gained      (PCR) using Phusion Hot Start polymerase (Finnzymes, <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> previous genome-wide gene deletion libraries,              Espoo, Finland) purified H. volcanii genomic DNA <br /> enabling systematic analysis applied         (prepared described [17]) primers listed in <br /> time archaeal species. Systematic, potentially           Supplemental Table 3. Primers designed they <br /> automated, analysis gene-deletion collections enable       contained 18&#226;&#8364;&#8220;20 bases homology chromosomal <br /> rapid identification strain phenotypes aid       DNA. The reverse primer upstream fragment and <br /> annotation hypothetical proteins.                               forward primer downstream fragment also <br />     As proof-of-principle case study goal,      contained linker 15 bases enabled joining the <br /> to identify suitable approaches study library,   fragments recombination (described below). The <br /> we analyzed 22 gene-deletion mutations generated            linker designed incorporate novel 15 bp spacer <br /> laboratory using easily scalable high-throughput methods.          up- downstream fragments containing <br /> Here, report statistics success endeavour    unique NdeI MluI cut sites (Figure 1). The forward <br /> and present techniques used establish accelerated           primer upstream fragment reverse primer <br /> genetics H. volcanii. Furthermore, employed high-            downstream fragment contained 15 bases of <br /> throughput array-based phenotyping approaches,                homology pTA131 (Figure 1). pTA131 linearized <br /> the potential scalability automation, aid           cutting XhoI EcoRI. After fragments and <br /> study mutants demonstrate H.           linearized vector gel-purified (QiaQuick, Qiagen), they <br /> volcanii perfectly amenable techniques (assuming       combined 5 : 5 : 1 (upstream fragment:downstream <br /> the predicted 4209 genes, mutants analyzed                fragment:linearized pTA131; Figure 1) molar ratio (cor- <br /> represent approximately 0.6% possible gene deletions).      rected include 100 ng linear vector) recombined <br /> We propose methods used form basis               using In-Fusion directed manufacturer (Clon- <br /> high-throughput phenotyping analysis H. volcanii.               tech, CA). Reactions incubated 15 min 37&#226;&#8212;&#166; C <br />                                                                    followed 15 min 50&#226;&#8212;&#166; C. Subsequent incubation, <br />                                                                    0.2 &#206;&#188;l transformed Top10 cells (Invitrogen, CA) <br /> 2. Materials Methods                                           according manufacturer&#226;&#8364;&#8482;s directions cells <br />                                                                    plated LB plus Amp, IPTG, X-gal. Clones were <br /> 2.1. Strains, Plasmids, Media, Transformation Proce-           screened PCR using M13 universal primers (which anneal <br /> dures. All strains plasmids used study             MCS pTA131) using 5  Taq Master <br /> detailed Supplementary Tables 1 2 available              Mix (Eppendorf) manufacturer&#226;&#8364;&#8482;s directions. <br /> doi:10.1155/2010/426239. H. volcanii H26 used           Clones verified Sanger sequencing using UF <br /> parent strain. E. coli routinely grown LB-Lennox            sequencing facility. The specific primers used generate <br /> (LB) (Fisher) LB agar (Fisher) 37&#226;&#8212;&#166; C, supplemented           gene-deletion construct detailed Supplemental <br /> when required ampicillin (Amp; 100 &#206;&#188;g mL&#226;&#710;&#8217;1 ),                 Table 3. <br /> isopropyl &#206;&#178;-D-1-thiogalactopyranoside (IPTG; 0.2 mM)                   Once obtained, confirmed deletion plasmids were <br /> and     bromo-chloro-indolyl-galactopyranoside       (X-gal;       passaged dam&#226;&#710;&#8217; strain Escherichia coli (Inv110; <br /> 40 &#206;&#188;g mL&#226;&#710;&#8217;1 ). When required, novobiocin added             Invitrogen, CA), transformed H. volcanii H26 (or <br /> final concentration 0.3 &#206;&#188;g mL&#226;&#710;&#8217;1 . H. volcanii cells        derivatives) described above. Deletion targeted <br /> routinely grown 44&#226;&#8212;&#166; C (unless specified) Hv-YPC,             locus selected two-step process described <br /> HvCA, CDM. H. volcanii media according                 previously Allers et al. [16]. Briefly, recombination the <br /> to recipes provided HaloHandbook [12]. Trans-           deletion plasmid chromosome single cross- <br /> formations chemically competent E. coli performed          event selected growth HvCA (i.e., in <br /> as described manufacturer&#226;&#8364;&#8482;s directions (Invitrogen,         absence uracil). Subsequent excision integrated <br /> CA). Transformation H. volcanii performed                plasmid target gene second recombination event <br /> described HaloHandbook using &#226;&#8364;&#339;standard PEG-             selected plating HvCA supplemented with <br /> mediated transformation Haloarchaea&#226;&#8364;? protocol. Briefly,         uracil (10 &#206;&#188;g mL&#226;&#710;&#8217;1 ) 5-fluoroorotic acid (5-FOA; 50 &#206;&#188;g <br /> spheroplasts adding 100 &#206;&#188;l 0.5 M EDTA (pH             mL&#226;&#710;&#8217;1 ). Gene deletion candidates screened using a <br /> 8.0) 1 mL concentrated cells incubated room              PCR-based method follows. The forward primer was <br /> temperature 10 min. 2 &#206;&#188;g plasmid DNA added             designed anneal 5  downstream fragment, and <br />  Archaea                                                                                                                                     3 <br />  <br />                                                                                 2.4. H. volcanii tRNA Extraction Analysis. Bulk tRNA <br />     NF            CF <br />     Upstream        Downstream <br />                                                              lacZa XhoI EcoRI <br />                                                                                 prepared, hydrolyzed, analyzed liquid chro- <br />                Gene <br />                                                                                 matography-tandem mass spectrometry (LC-MS/MS) as <br />             NR                    CR <br />                                                                                 described previously [34]. All tRNA purifications anal- <br />                                                                                 yses performed independently twice. <br />                        (a)                       pyrE2 <br />                                                                pTA131           2.5. High-Throughput Phenotype Analysis. For analysis of <br />                                                                                 growth liquid solid media, strains routinely <br />                                                                                 streaked cryostorage Hv-YPC solid medium and <br />                                                                                 incubated 48 hrs 44&#226;&#8212;&#166; C. Three tubes containing 5 mL of <br />                                     <br />                                                                   AmpR          Hv-YPC inoculated independently single colonies <br />                                                                                 grown 44&#226;&#8212;&#166; C 48 hrs. <br />                                                                                      For monitoring H. volcanii growth curves, 2 &#206;&#188;l of <br />                        (b)                                      (c) <br />                                                                                 culture normalized OD600 1 used inoculate <br />                                            Mlu Ndel                             300 &#206;&#188;l Hv-YPC 300 &#206;&#188;l CDM. 250 &#206;&#188;l inoculated <br />             &#226;&#710;&#188;600 bp upstream <br />                                                       &#226;&#710;&#188;600 bp downstream        medium transferred 100-well plate. Growth curves <br />                                                                                 monitored Bioscreen-C Automated Growth Curve <br />                    LacZa <br />                                                               LacZa             Analysis System (Growth Curves USA, MA) 44&#226;&#8212;&#166; C with <br />                                                                                 continuous shaking. Readings recorded 45 min <br />                                                                                 (for clarity 5 readings shown Figures 3(d) <br />                                Gene deletion construct                          3(e)). For duration experiments conducted <br />             pyrE2                                                               here, special measures necessary prevent evapo- <br />                                                                                 ration/salt precipitation Bioscreen plate. <br />                                                                                      For analysis growth solid media, wild-type and <br />                                                                                 mutant strains grown range temperatures and <br />                                                                                 salt concentrations. 100 &#206;&#188;l culture grown Hv-YPC <br />                                             AmpR                                normalized OD600 1. This culture serially <br />                                                                                 diluted, 10 &#206;&#188;l 10&#226;&#710;&#8217;4 dilution plated a <br />                                                                                 96-well plate. Each contained 100 &#206;&#188;l Hv-YPC agar <br />                                            (d)                                  containing various salt concentrations. In addition the <br /> Figure 1: Generation H. volcanii gene deletion constructs.                   standard final concentration 18% salt water, indicated <br /> Regions approximately 600 bp flanking gene                   [12], 30% salt water stock used make standard <br /> were amplified H. volcanii chromosome (a) Regions                      Hv-YPC diluted achieve final concentrations 12%, <br />                                                     <br /> of complementarity primers . Complemen-                         14%, 23% 25%. Each medium duplicated that <br /> tarity sequences 15 b, follows: NF: CGGGCCCCCCCTC-                      6 salt concentrations occupied 12 wells (equating a <br /> GAG; NR: GACGCGTTCATATGC; CF: GCATATGAACGCGTC                                   row 96-well plate). Each plate replicated 5 times, <br /> CR: CGGGCTGCAGGAATTC designed yield unique                          incubated 26&#226;&#8212;&#166; C, 30&#226;&#8212;&#166; C, 37&#226;&#8212;&#166; C, 44&#226;&#8212;&#166; C 50&#226;&#8212;&#166; C, <br /> MluI NdeI sites. Plasmid pTA131 digested EcoRI                     yielding matrix temperature salt concentration. <br /> XhoI (c) fragments recombined using In-Fusion                     Depending temperature, plates incubated 2 to <br /> (d). Refer Materials Methods details techniques               7 days. The row plate used wild-type <br /> used. <br />                                                                                 control, various mutants added subsequent <br />                                                                                 rows. <br /> the reverse primer designed anneal the <br /> upstream fragment. One pair primers (denoted                          3. Results Discussion <br /> su&#239;&#172;&#402;x ext Fwd ext Rev) designed anneal                        3.1. Accelerating Gene Deletion Constructions H. volcanii. <br /> the flanking regions gene deleted                          The deletion large number genes H. volcanii <br /> amplicon size compared wild-type predicted sizes. To                     genome required development fast method to <br /> confirm loss gene, second pair primers (su&#239;&#172;&#402;xed                      obtain deletion constructs. Construction H. volcanii gene <br />  int Fwd int Rev) designed anneal                         deletions divided phases: (1) generation <br /> target gene deleted confirm loss gene                      deletion construct involves established E. coli- <br /> the chromosome.                                                                 based molecular biology, (2) introduction and <br />                                                                                 selection correct gene-deletion candidate native <br /> 2.3. Other Plasmid Constructions. Target genes ampli-                      organism. Phase 1 hampered need amplify <br /> fied PCR purified H. volcanii genomic DNA using                         polymerase chain reaction (PCR) clone multiple <br /> Phusion Hot Start polymerase (Finnzymes, Espoo, Finland),                       fragments high GC content DNA (average 65% [14]). <br /> using primers listed Supplementary Table 3. The amplicon                     Phase 2 time consuming inherent technique and <br /> was inserted NdeI BlpI sites pJAM202c                        organism; independent recombination events be <br /> [33].                                                                           selected relatively slow-growing organism. Currently, <br />  4                                                                                                                                                                                                                                                                                                                            Archaea <br />  <br />  <br />  <br />  <br />                                             &#206;&#8221;HVO 1594 <br />  <br />  <br />  <br />  <br />                                                                                                          &#206;&#8221;HVO 1594 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   &#206;&#8221;HVO 1717 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              &#206;&#8221;HVO 1717 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                          &#206;&#8221;HVO 1716 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                        &#206;&#8221;HVO 1716 <br />                                 Wild-type <br />  <br />  <br />  <br />  <br />                                                                                       Wild-type <br />  <br />  <br />  <br />  <br />                                                                                                                                       Wild-type <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                           Wild-type <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                           Wild-type <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                           Wild-type <br />                               3800 bp <br />                                      2900 bp                                                                                        2522 bp                                                                                             2608 bp <br />                                                                                                                                                       1748 bp <br />                                                                                                                                                                                                                                                       1873 bp <br />                                                                                      543 bp                                                                                                              573 bp                                                                                        558 bp <br />  <br />  <br />  <br />  <br />                                                    (a)                                                                                                                        (b)                                                                                                    (c) <br />     (d)                                                                                                                                  (f) <br />                                                                                     1.2e7                                                                                                                             m2 2 G <br />                                                                                      1e7               15.69 min                                                     4e5                                                                 G+ <br />                                                                                                        m5 C                                                                                                                                                                2.4e8 <br />                                                                                                                                                                                                                                                                                                            24.48 min <br />                                                                   Intensity (cps) <br />  <br />  <br />  <br />  <br />                                                                                      8e6                                                                                                                                                                                       2e8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                        Imtensity (cps) <br />                                                                                                                                                                                                                                                                                                 XIC 325 m/z <br />  <br />  <br />  <br />  <br />                                                                                                                                                       (a.u./&#206;&#188;V) <br />                                                                                                                                                                      3e5                                                                                                   1.6e8 <br />                                                                                      6e6 <br />                                                                                                                                                                                                                          t6 A                                              1.2e8                            G+ <br />                                                                                      4e6                                                                                                                                                                                       8e7 <br />                                                                                                                                                                      2e5 <br />                   4e5                                                                2e6                                                                                                                                                                                       4e7 <br />                                                                                                                                                                                                                                                                                      0 <br />                  3.5e5                                                                   0                                                                           1e5                                                                                                                 24 24.4 24.8 25.2 25.6 26 26.4 26.8 <br />                                                                                                   14          15          16 <br />                                                                                                                                                                                                                                                                                                       Time (min) <br />                   3e5 <br />                                                                            m5 C                                                                                               0 <br />      (a.u./&#206;&#188;V) <br />  <br />  <br />  <br />  <br />                  2.5e5                              U <br />                                                                                                                                          (g) <br />                   2e5                                                                                                                                                                                                 m2 2 G <br />                                                                                                                                                               2.3e5 <br />                  1.5e5 <br />                                                                                                                                                                      2e5 <br />                   1e5                                                                                                                                                                                                                                                     2.4e6                        XIC 325 m/z <br />                                                                                                                                                               1.6e5                                                                                                                           No G+ <br />                                                                                                                                                   (a.u./&#206;&#188;V) <br />  <br />  <br />  <br />  <br />                   5e4                                                                                                                                                                                                   t6 A                                                  2e6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                        Imtensity (cps) <br />                                                                                                                                                               1.2e5                                                                                                       1.6e6 <br />                     0                                                                                                                                                                                                                                                     1.2e6 <br />                          12    13           14               15   16   17                                            18        19                                    8e4                                                                                                      8e5 <br />                                                             Time (min)                                                                                                                                                                                                        4e5 <br />                                                                                     8e6                                                                              4e4                                                                                                                    0 <br />                                                                                     7e6                                                                                                                                                                                                  24 24.4 24.8 25.2 25.6 26 26.4 26.8 <br />                                                                                     6e6                No m5 C                                                                0 <br />                                                                   Intensity (cps) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                       Time (min) <br />     (e)                                                                             5e6                                                  (h) <br />                  3.2e5                                                              4e6                                                                       2.7e5 <br />                    3e5                                                              3e6                                                                                                                                   m2 2 G <br />                  2.8e5                                                                                                                                        2.4e5 <br />                  2.6e5                                                              2e6 <br />      (a.u./&#206;&#188;V) <br />  <br />  <br />  <br />  <br />                  2.4e5                                                              1e6                                                                              2e5 <br />                  2.2e5                                  U                             0 <br />                                                                                                                                                   (a.u./&#206;&#188;V) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                             3e6 <br />                    2e5                                                                            14       15         16 <br />                                                                                                                                                               1.6e5                                                                                                       2.6e6               No G+ <br />                                                                                                                                                                                                                                                        Imtensity (cps) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                       XIC 325 m/z <br />                  1.8e5                                                                                                                                                                                                         t6 A                                       2.2e6 <br />                  1.6e5                                                                                                                                        1.2e5 <br />                                                                                                                                                                                                                                                                           1.8e6 <br />                  1.4e5                                                                                                                                                                                                                                                    1.4e5 <br />                                                                                                                                                                                                                                                                             1e5 <br />                  1.2e5 <br />                    1e5                                                                                                                                               8e4                                                                                                    6e5 <br />                                                                                                                                                                                                                                                                             2e5 <br />                    6e4 <br />                    4e4                                                                                                                                               4e4                                                                                                                 24   24.4 24.8 25.2 25.6     26           26.4 26.8 <br />                    2e4                                                                                                                                                                                                                                                                                   Time (min) <br />                      0                                                                                                                                                        0 <br />                          12       13         14              15                      16            17                18        19                                                 24     24.4 24.8           25.2 25.6                  26            26.4 <br />                                                             Time (min)                                                                                                                                   Time (min) <br />                                                         (d)                                                                                                                                                                       (e) <br />  <br /> Figure 2: tRNA analysis. Gene deletions (a) HVO 1594, (b) HVO 1717 (c) HVO 1716 verified locus-specific PCR using <br /> primer pairs indicated Supplementary Table 3 designed anneal outside gene. Predicted amplicon sizes indicated <br /> above band. Analysis LC-MS/MS tRNA extracted H. volcanii strains showing presence absence m5 C peak on <br /> the UV trace 254 nm H26 (d) corresponding extraction ion chromatogram H26 &#206;&#8221;HVO 1594 (e). Fragmentation <br /> analyses m5 C shown insets. LC-MS/MS analyses tRNA extracted (f) H26, (g) &#206;&#8221;HVO 1717, (h) &#206;&#8221;HVO 1716 indicate <br /> the presence absence G+ . The peaks corresponding m2 2 G t6 A shown comparison. Fragmentation analyses G+ are <br /> shown insets. <br />  <br />  <br />  <br /> described methods require cloning up- down-                                                                                                                                    proprietary topoisomerase-based techniques be <br /> stream regions flanking gene deleted                                                                                                                                  expensive applied large gene sets [16, 17]. Thus, all <br /> together (making three-way ligation reaction                                                                                                                           existing techniques require multiple rounds cloning <br /> two fragments linearized vector relatively ine&#239;&#172;&#402;cient),                                                                                                                             labor intensive expensive, neither <br /> sequential cloning (which time consuming), use                                                                                                                                desirable creation large numbers <br />  Archaea                                                                                                                                                     5 <br />  <br />                                    Wild-type         &#206;&#8221;HVO 2747             Wild-type               &#206;&#8221;HVO 0253                       Wild-type    &#206;&#8221;HVO 0339 <br />  <br />                                     2570 bp <br />                                                           2020 bp                                                                  2139 bp <br />                                                                            2509 bp                     1970 bp <br />                                                                                                                                                 1007 bp <br />  <br />  <br />  <br />  <br />                                         (a)                                  (b)                                                     (c) <br />                                                                                                  0.7 <br />                    0.8 <br />                                                                                                  0.6 <br />  <br />                                                                                                  0.5 <br />                    0.6 <br />      OD (600 nm) <br />  <br />  <br />  <br />  <br />                                                                                    OD (600 nm) <br />                                                                                                  0.4 <br />  <br />                    0.4                                                                           0.3 <br />  <br />                                                                                                  0.2 <br />                    0.2 <br />                                                                                                  0.1 <br />  <br />                     0                                                                             0 <br />                          0                    20                      40                                0               20                       40 <br />                                                          Time (hrs)                                                                Time (hrs) <br />                              H26                              HVO 2888                                       H26                        HVO 2478 <br />                              HVO 0916                         HVO 0310                                       HVO 0916                   HVO 0310 <br />                                                    (d)                                                                       (e) <br />  <br /> Figure 3: H. volcanii gene essentiality growth analysis. Gene essentiality confirmed locus-specific PCR (a) HVO 2747, (b) <br /> HVO 0253, (c) HVO 0339. Primers designed anneal outside gene used detailed Supplementary Table 3 the <br /> amplicon size compared wild-type parent strain. Amplicon sizes indicated band. Growth analysis H. volcanii H26 <br /> (wild-type) severe growth-defected strains Hv-YPC media (d) CDM (e). H26, HVO 0916, HVO 0390 are <br /> indicated diamonds, squares, circles, respectively. Triangles indicate HVO 2888 (d) HVO 2478 (e). Each point represents mean <br /> of independent biological repeats; error bars plus minus standard error. <br />  <br />  <br />  <br />  <br /> of gene deletions. To expedite generation mutants,                               method enables construction large numbers of <br /> we explored possibility recombineering deletion                                  gene deletion constructs realistic timeframe. Further- <br /> constructs employing traditional established                                more, technique principle applied <br /> molecular biology techniques using In-Fusion (Clontech,                                 rapidly generate plasmid constructs manipulate the <br /> CA; [35]). Detailed methodology technique                                   genome insertions, in-frame chromoso- <br /> included Materials Methods shown schemat-                                mal protein-fusion tags remains untested at <br /> ically Figure 1. Utilization enabled deletion                         present. <br /> constructs synthesized 24 hrs receiving                                 Traditionally, H. volcanii community preferred <br /> necessary oligonucleotides (compared 2&#226;&#8364;&#8220;4 weeks using                                 deletion verification Southern hybridization methods. <br /> previously described methods) verified gene deletions                               Here, large number putative mutants screen, <br /> in H. volcanii achieved 3&#226;&#8364;&#8220;4 weeks. Furthermore,                                 adopted 2-fold PCR-based mutant verification <br /> due apparent high accuracy technique, fewer                               process employed H. volcanii El Yacoubi et al. [17]. <br /> deletion-construct candidates needed screened prior                               In addition sensitive Southern <br /> to introduction H. volcanii reducing time spent cost                             hybridizations, PCR easily scalable. Two primers are <br /> (generally candidate plasmids screened using                               designed anneal outside a&#239;&#172;&#8364;ected gene, two <br /> PCR-based method,                                 within. The PCR reactions performed primers <br /> correct). Although month far                               confirm alteration genomic region validated <br /> time required model organisms kingdoms,                                    comparison predicted amplicon size, the <br /> this period unavoidable relative slow growth                              primers landing gene absolutely confirm its <br /> of organism recombination events                                    absence compared wild-type parent strain <br /> implicit deletion protocol. However, adoption                                 positive control. Generally, 36 candidates screened <br />  6                                                                                                                      Archaea <br />  <br /> which, cases mutants achieved, typically yielded      shuttle vector pJAM202. The clones transformed into <br /> several deletions. The theoretical 50% mutant generation          H. volcanii H26 derivatives chromosomes carried <br /> from resolution merodiploid rarely achieved,           verified integration corresponding gene deletion <br /> possibly recombination restraints resulting           construct (see Section 2). With specific genes expressed <br /> localized secondary structure. If gene deletions     high levels trans, attempts to <br /> identified, 100 candidates screened,     delete corresponding chromosomal gene copy. Deletion <br /> mutants exhibited, entire process repeated     gene situation checked described <br /> at once.                                                    Materials Methods, internal primers <br />      Using protocol, 21 genes successfully deleted      used amplify product regard- <br /> out 30 attempted (Table 1), strain constructed     successful deletion. Using methodology, we <br /> containing gene deletions (&#206;&#8221;HVO 1105 &#206;&#8221;HVO 2008).              successfully achieved deletion chromosomal copies <br /> To demonstrate validity protocol,           HVO 0339, HVO 0253, HVO 2747 suggesting that <br /> analyzed phenotypes specific mutants. HVO 1594             genes genuinely essential (Figures 3(a)&#226;&#8364;&#8220;3(c)). <br /> previously predicted responsible forma-             Testing remaining essential gene candidates is <br /> tion tRNA modification m5 C basis                ongoing. <br /> homology P. abyssi protein PAB1947 [36], mod-                  The essentiality genes expected. <br /> ification shown present H. volcanii           HVO 0339, member COG1517 family, recently <br /> tRNA species [37]. HVO 1594 deleted chro-            shown vitro responsible modification cyti- <br /> mosome using method described above, bulk                 dine agmatidine wobble position tRNA Ile- CAU <br /> tRNA wild type &#206;&#8221;HVO 1594 cells grown                  [45, 46]. Agmatidine allows modified tRNA Ile- CAU to <br /> Hv-YPC media analyzed LC-MS/MS (Figure 2).                decode AUA isoleucine codons methionine AUG <br /> As shown Figure 2, peak corresponding m5 C              codons [45, 46]. The functionally analogous modification in <br /> was absent tRNA extracted mutant.               bacteria, lysidine, essential [47]. HVO 0253 member <br /> The identity corresponding peak wild-type             YrdC/Sua5 family involved formation N6 - <br /> tRNA confirmed tandem mass fragmentation.                  threonylcarbamoyl adenosine (t6 A), universal modification <br /> Thus LC-MS/MS analysis consistent                position 37 tRNAs decoding ANN codons [48]. <br /> involvement HVO 1594 formation m5 C               The corresponding gene family essential bacteria, and <br /> tRNA.                                                             deletion yeast homologue (YGL169W) viable but <br />      HVO 1717 HVO 1716 respective homologs            leads severe growth phenotypes [48]. The essentiality <br /> of bacterial queE queC involved synthesis           HVO 0253 reveals concerning role t6 A <br /> preQ0 , precursor Queuosine modification           biosynthesis enzymes, Archaea closer bacteria to <br /> in bacterial tRNA [38, 39]. preQ0 precursor      eukaryotes. Finally, function HVO 2747, member <br /> biosynthesis Archaeosine (G+ ) [40], tRNA modification       COG1491, remains unknown, conservation in <br /> of D-loop archaeal tRNA species           Archaea, combined essentiality phenotype, <br /> sequenced date [41]. As modifications share          makes target study. Our genetic results strongly <br /> the precursor, likely hypothesis HVO 1717          suggest genes described essential but <br /> and HVO 1716 involved G+ biosynthesis. To test             future investigation utilizing previously described PTNA <br /> this, corresponding gene deletion mutants                promoter [44] confirm essentiality phenotype as <br /> constructed described Materials Methods             determine level expression required for <br /> and, avoid potential contamination preQ0 rich          viability. <br /> media [40], grown defined media (CDM). The bulk <br /> tRNA extracted mutant strains, purified            3.3. High-Throughput Growth Rate Analysis H. volcanii <br /> and digested ribonucleosides (Figure 2). Analysis           Mutants. Having deleted relatively large number genes, <br /> LC-MS/MS demonstrated peak corresponding              second major goal study develop and <br /> the G+ absent mutant strains present       implement high-throughput phenotyping physiology <br /> in wild-type. Identity peak wild-type         approaches H. volcanii. Whilst approaches <br /> sample confirmed tandem mass fragmentation                 applied analysis H. salinarum [49], <br /> (Figure 2). These results confirmed HVO 1717 (queE)          similar methods performed H. volcanii. <br /> and HVO 1716 (queC) involved G+ biosynthesis.              Array-based phenotyping approaches (used derive the <br />                                                                   &#226;&#8364;&#339;phenome&#226;&#8364;?) employed model organisms <br /> 3.2. Discovering Essential Genes H. volcanii. Using        great success ( [50&#226;&#8364;&#8220;52] recently reviewed [53]). <br /> techniques, set delete 30 genes (Table 1). 22 strains   Should H. volcanii gene deletion library constructed <br /> were successfully constructed (Table 1); however, despite         future, similar methods necessary the <br /> multiple attempts, genes deleted (Table 1)     functional assessment large number mutants. We, <br /> raising possibility essential. To deter-       therefore, initiated investigation growth studies <br /> mine bona fide essential genes,            wild-type mutant strains constructed our <br /> thus form basis H. volcanii essential gene set         laboratory years compared range <br /> alongside likes cct1, hdaI, pitA [42&#226;&#8364;&#8220;44],         conditions media types. Our methods designed <br /> of genes cloned E. coli/H. volcanii        generate large, quantitative dataset whilst maintaining <br />  Archaea                                                                                                                                    7 <br />  <br />                                                        Table 1: Gene-deletion summary. <br />  <br /> Gene                    COG          Strain growth rate Hv-YPCa      Strain growth rate CDMa,b       Phenotype/essentiality verified <br /> H26 (wild-type)          &#226;&#8364;&#8221;                     0.0227                          0.0133                             &#226;&#8364;&#8221; <br /> HVO 0156                1041                   0.0223                          0.0105 <br /> HVO 0658                0585                   0.0281                          0.0175 <br /> HVO 2001                0343                   0.0233                          0.0123                       Cold-sensitivity <br /> HVO 0236                1867                   0.0238                          0.0104 <br /> HVO 2906                0565                    NDc                             NDc <br /> HVO 2888                1243                   0.0102                          0.0115                     Severe growth defect <br /> HVO 0580                0037                   0.0271                          0.0124 <br /> HVO 2736                1444                   0.0210                          0.0115                              (20) <br /> HVO 1852                0101                   0.0151                          0.0151                        Growth defect <br /> HVO 1594                0144                   0.0107                          0.0107                        Growth defect <br /> HVO 0916                1798                   0.0055                          0.0088             Cold-sensitivity, severe growth defect <br /> HVO 1631                4112                   0.0196                          0.0097             Cold-sensitivity, slight growth defect <br /> HVO 2477                2890                   0.0219                          0.0117                       Cold-resistance <br /> HVO 1716                0603                    0.022                          0.0143 <br /> HVO 1717                0602                    0.020                          0.0118 <br /> HVO B0354               1357                   0.0266                          0.0128 <br /> HVO 0390                2016                   0.0066                          0.0064                     Severe growth defect <br /> HVO 2478                0042                   0.0243                          0.0075                 Defined media growth defect <br /> HVO 1475                2263                   0.0266                          0.0177 <br /> HVO 1105                1374                   0.0216                          0.0124 <br /> HVO 2008                1549                   0.0245                          0.0154 <br /> HVO 1105                1374,                   0.025                          0.0127 <br /> HVO 0574                2226                                Not deleted                                    Predicted essential <br /> HVO 0339                1571                                Not deleted                          Deleted gene present trans <br /> HVO 0697                2047                                Not deleted                                    Predicted essential <br /> HVO 0253                0009                                Not deleted                          Deleted gene present trans <br /> HVO 1173                1303                                Not deleted                                    Predicted essential <br /> HVO 1383                2519                                Not deleted                                    Predicted essential <br /> HVO 2747                1491                                Not deleted                          Deleted gene present trans <br /> HVO 0929                                                    Not deleted                                    Predicted essential <br /> a <br />   growth rates presented hr&#226;&#710;&#8217;1 . <br /> b 0.4% (v/v) glycerol used carbon source. <br /> c Not determined. <br />  <br />  <br />  <br />  <br /> a manageable, easily scalable, labour load. We explored               deletions investigation detrimental to <br /> two approaches strain phenotyping. The approach                  growth. Many strains maintained growth rates <br /> was perform growth curves 100-well format,                    indistinguishable wild type, however <br /> the second determination fitness strains grown               7 mutants exhibited severe growth defects (Table 1 and <br /> solid media 96-well format. For growth curves, wild-                 Figures 3(d) 3(e)). As expected, more <br /> type 21 mutant strains grown rich (Hv-YPC)                    mutants exhibited phenotypes grown defined <br /> and defined media (CDM). Secondly, solid media,                 (CDM) medium (Table 1). &#206;&#8221;HVO 0390, &#206;&#8221;HVO 0916, and <br /> same strains grown range temperatures                    &#206;&#8221;HVO 2888 displayed severe growth defects rich <br /> salt concentrations. Each experiment contained               medium (Figures 3(d) 3(e)). &#206;&#8221;HVO 0916, &#206;&#8221;HVO 0390, <br /> independent biological replicates. Using techniques,             &#206;&#8221;HVO 2478 displayed severe growth <br /> identified multiple novel phenotypes researcher                  defects defined medium (Figure 3). Complete growth- <br /> in 2 weeks (summarized Table 1, Figures 3 4                    rate calculations strains media types are <br /> Supplementary Table 4).                                                   summarized Table 1. Although growth rate constants <br />      A Bioscreen C apparatus employed obtain growth                corresponding doubling times &#226;&#710;&#188;20 hrs lower than <br /> parameters strains rich medium (Hv-YPC) de-                 previously reported [12, 13], appears an <br /> fined medium (CDM) determine gene                          artefact using Bioscreen C apparatus, culture <br />  8                                                                                                                                Archaea <br />  <br />                12%      14%        16%     18%     23%        25%          agitation, aeration, significantly lower <br />     37&#226;&#8212;&#166; C                                                            H26 <br />                                                                            large cultures regular shaking incubator. As a <br />                                                                            control this, H26 grown Hv-YPC media <br />     44&#226;&#8212;&#166; C                                                            H26   regular culture flasks determined doubling <br />                                                                            time &#226;&#710;&#188;4.5 hrs, strongly suggesting di&#239;&#172;&#8364;erences solely <br />     50&#226;&#8212;&#166; C                                                            H26   result 100-well format used. HVO 0390 member <br />                                                                            COG2016. The yeast homolog, Tma20, identified <br />                                      (a) <br />                                                                            ribosome-associated protein [54]. Loss Tma20 <br />                                                                            caused significant increase ochre opal codon read <br />                                                                            yeast [54]. The corresponding gene physically <br />                 12%         14%     16%     18%     23%        25% <br />                                                                            clusters ribosomal protein genes archaeal <br />                                                                       1    genomes ([55] Tma20 subsystem SEED data- <br />                                                                       2    <span id='am-315' about='xsp:base' typeof='owl:Thing'>base</span> http://theseed.uchicago.edu/FIG/SubsysEditor.cgi?page <br />       26&#226;&#8212;&#166; C                                                           3 <br />                                                                            =ShowSpreadsheet&amp;subsystem=DOE COG2016).               There- <br />                                                                            fore, remains untested present, role for <br />                                                                       4 <br />                                                                            HVO 0390 maintaining translation fidelity Archaea <br />                                                                       5    appears probable, given this, reduced growth rate <br />                                                                            corresponding deletion mutant surprising. <br />                                                                       1 <br />                                                                            HVO 0916 homologous Diphtine Synthase (Dph5), a <br />                                                                       2    methyltransferase diphtamide biosynthetic pathway. <br />       30&#226;&#8212;&#166; C                                                           3    Diphtamide post translational modification in <br />                                                                       4 <br />                                                                            Elongation factor 2 (EF2) Archaea eukaryotes and <br />                                                                            target diphtheria toxin [56&#226;&#8364;&#8220;58]. Although the <br />                                                                       5    precise function modification remains unknown, <br />                                                                       1    deletion genes pathway results growth phe- <br />                                                                            notypes translation defects yeast [59] severely <br />                                                                       2 <br />                                                                            impaired development mammals [60, 61]. Our analysis <br />       37&#226;&#8212;&#166; C                                                           3    suggests growth defects absence diphtamide <br />                                                                       4    observed Archaea, remains validated <br />                                                                       5    confirming absence modification EF2 to <br />                                                                            deletion HVO 0916. These results suggest obtaining <br />               1 H26                                                        growth curve analyses high-throughput fashion a <br />               2 &#206;&#8221; HVO   0916                                               viable approach study H. volcanii gene function and <br />               3 &#206;&#8221; HVO   1631 <br />               4 &#206;&#8221; HVO   2001 <br />                                                                            physiology. <br />               5 &#206;&#8221; HVO   2477 <br />                                      (b) <br />                                                                            3.4. High Throughput Phenotyping H. volcanii Mutants. A <br />                                                                            key question analysis halophiles, Archaea at <br />                       26&#226;&#8212;&#166; C                       44&#226;&#8212;&#166; C <br />                                                                            large, understand ability thrive seemingly <br />                                                                            inhospitable environments. To begin address this, and <br />                                                                            long-term intention identifying under- <br />                   1            2             1            2 <br />                                                                            lying genetic molecular basis ability, the <br />                                                                            parent strain mutants subjected growth a <br />                                                                            range temperatures (26&#226;&#8212;&#166; C&#226;&#8364;&#8220;50&#226;&#8212;&#166; C) salt concentrations <br />                         3                           3                      (12%&#226;&#8364;&#8220;25%). An interesting observation revealed <br />                                                                            comparing large numbers plates simultaneously is <br />                                                                            H. volcanii salt tolerance appears dependent <br />             1 H26pJAM202c                                                  temperature. At relatively low temperatures (26&#226;&#8212;&#166; C&#226;&#8364;&#8220; <br />             2 &#206;&#8221;HVO 2001 pJAM202::HVO 2001                                  37&#226;&#8212;&#166; C), clear preference relatively high salt <br />             3 &#206;&#8221;HVO 2001 pJAM202c                                           concentrations higher temperature (50&#226;&#8212;&#166; C), salt <br />                                      (c)                                   tolerance decreases, high salt concentration becomes <br />                                                                            inhibitive growth (Figure 4(a)). Comparison growth <br /> Figure 4: Salt temperature tolerance H. volcanii strains. ((a)      wild-type mutant strains various salt <br /> and (b)) H26 (wild-type) mutant strains (b) subjected             concentrations temperatures revealed strains <br /> to high low temperature salt concentration. The salt               exhibit cold sensitivities. These included deletions <br /> concentration temperature indicated well. (c)           HVO 0916, HVO 1631, HVO 2001 (Figure 4(b) <br /> H26 (1), &#206;&#8221;HVO 2001 (2), complemented &#206;&#8221;HVO 2001 (3)                     Supplementary Table 4). Conversely, strain bearing <br /> streaked Hv-YPC incubated 26&#226;&#8212;&#166; C 44&#226;&#8212;&#166; C indicat-           deletion HVO 2477 exhibits increased tolerance low <br /> ed.                                                                        temperature. <br />  Archaea                                                                                                                              9 <br />  <br />      As seen deleting components transla-        construction. GP, CEB BEY helped design of <br /> tion apparatus [62, 63], deleting HVO 0916, HVO 1631,          phenotypic analysis. <br /> HVO 2001 gave rise cold sensitivity phenotypes. As dis- <br /> cussed above, HVO 0916 homologous diphthamide <br /> biosynthesis protein Dph5 HVO 1631 homologous               Acknowledgments <br /> to Dph2, enzyme pathway [64]. HVO 2001 <br />                                                                    This work funded National Science Foundation <br /> encodes tRNA-guanine&#226;&#8364;&#8220;transglycosylase (TGT) [17]. TGT <br />                                                                    Grant no. MCB-05169448 V. Cr&#194;&#180;ecy-Lagard H. volcanii, <br /> is key enzyme synthesis Archaeosine (G+ ), a <br />                                                                    H26, pJAM202c gifts J. Maupin-Furlow. LC- <br /> modification tRNA specifically Archaea. G+ is <br />                                                                    MS/MS experiments performed Sophie Alvarez, and <br /> found elbow D-arm T-arm position 15 of <br />                                                                    authors thank Henri Grosjean constant suggestions <br /> almost archaeal tRNA bearing guanine position. <br />                                                                    support. <br /> We previously shown &#206;&#8221;HVO 2001 viable and <br /> lacked G+ tRNA [17]. The results growth <br /> phenotype Archaeon lacking Archaeosine, cold sensi-         References <br /> tivity. In eukaryotic bacterial tRNA, position 15 part <br /> of Mg2+ binding site [65, 66]. Mg2+ ions induce folding         [1] C. R. Woese G. E. Fox, &#226;&#8364;&#339;Phylogenetic structure the <br /> and maintain tRNA tertiary structure (16, 43, 48,               prokaryotic domain: primary kingdoms,&#226;&#8364;? Proceedings of <br /> 52). Also, Oliva et al. showed silico, Mg2+ G+ play         National Academy Sciences United States of <br /> an interchangeable role stabilizing G15-C48 Reverse              America, vol. 74, no. 11, pp. 5088&#226;&#8364;&#8220;5090, 1977. <br /> Watson-Crick geometry [67]. Together, studies imply           [2] S.-V. Albers, N.-K. Birkeland, A. J. M. Driessen et al., <br />                                                                         &#226;&#8364;&#339;SulfoSYS (Sulfolobus Systems Biology): silicon <br /> that G+ play role inducing folding maintain- <br />                                                                         cell model central carbohydrate metabolism the <br /> ing structural integrity tRNA. Therefore, absence            archaeon Sulfolobus solfataricus temperature variation,&#226;&#8364;? <br /> of G+ modification confer rigidity tRNA               Biochemical Society Transactions, vol. 37, no. 1, pp. 58&#226;&#8364;&#8220;64, <br /> and perturb folding low temperatures explaining cold-            2009. <br /> sensitivity phenotype. To independently confirm 96-             [3] T. K. Pham, P. Sierocinski, J. Van Der Oost, P. C. Wright, <br /> well plate assays, reproduced &#206;&#8221;HVO 2001 phenotype                &#226;&#8364;&#339;Quantitative proteomic analysis Sulfolobus solfataricus <br /> on 100 mm petri dishes (Figure 4(c)) complemented                   membrane proteins,&#226;&#8364;? Journal Proteome Research, vol. 9, no. <br /> the cold-sensitivity phenotype expressing HVO 2001               2, pp. 1165&#226;&#8364;&#8220;1172, 2010. <br /> in trans. As HVO 2001, HVO 1717, HVO 1716 form                  [4] S. Berkner G. Lipps, &#226;&#8364;&#339;Genetic tools Sulfolobus spp.: <br /> part pathway G+ biosynthesis,                  vectors applications,&#226;&#8364;? Archives Microbiology, vol. <br /> cold-sensitivity phenotype expected strains            190, no. 3, pp. 217&#226;&#8364;&#8220;230, 2008. <br /> bearing deletions HVO 1717 HVO 1716. However,             [5] M. Wagner, S. Berkner, M. Ajon, A. J. M. Driessen, G. <br /> did observe cold-sensitivity phenotype              Lipps, S.-V. Albers, &#226;&#8364;&#339;Expanding understanding the <br /> genes. An explanation observation possibility           genetic toolbox hyperthermophilic genus Sulfolobus,&#226;&#8364;? <br /> of preQ0 salvage rich media (on phenotype              Biochemical Society Transactions, vol. 37, no. 1, pp. 97&#226;&#8364;&#8220;101, <br />                                                                         2009. <br /> screens performed). Therefore, strains G+ is <br />                                                                     [6] T. Sato, T. Fukui, H. Atomi, T. Imanaka, &#226;&#8364;&#339;Improved <br /> still made. This hypothesis tested repeating these <br />                                                                         versatile transformation allowing multiple genetic <br /> assays defined medium, salvage PreQ0 base               manipulations hyperthermophilic archaeon Thermococ- <br /> would occur.                                                        cus kodakaraensis,&#226;&#8364;? Applied Environmental Microbiology, <br />      In conclusion, employed modern recombineer-                vol. 71, no. 7, pp. 3889&#226;&#8364;&#8220;3899, 2005. <br /> ing-based methods rapidly generate gene-deletion con-            [7] T. Sato, T. Fukui, H. Atomi, T. Imanaka, &#226;&#8364;&#339;Targeted <br /> structs, ultimately resulting construction 22 strains.        gene disruption homologous recombination hyper- <br /> Eight genes included pipeline deleted               thermophilic archaeon Thermococcus kodakaraensis KOD1,&#226;&#8364;? <br /> by methods; genetically demonstrated provided                  Journal Bacteriology, vol. 185, no. 1, pp. 210&#226;&#8364;&#8220;220, 2003. <br /> evidence probable essentiality 3 those. Further-      [8] I. Waege, G. Schmid, S. Thumann, M. Thomm, W. <br /> more, using array-based phenotyping methods, able               Hausner, &#226;&#8364;&#339;Shuttle vector-based transformation for <br /> to identify multiple growth phenotypes mutants ana-              pyrococcus furiosus,&#226;&#8364;? Applied Environmental Microbiology, <br /> lyzed using methods easily scaled encompass            vol. 76, no. 10, pp. 3308&#226;&#8364;&#8220;3313, 2010. <br /> much larger gene-deletion collections. The work presented           [9] C. J. Bult, O. White, G. J. Olsen et al., &#226;&#8364;&#339;Complete genome <br /> here paves potential road systematic creation                 sequence Methanogenic archaeon, Methanococcus jan- <br />                                                                         naschii,&#226;&#8364;? Science, vol. 273, no. 5278, pp. 1058&#226;&#8364;&#8220;1073, 1996. <br /> and analysis global H. volcanii gene-deletion library, <br />                                                                    [10] B. C. Moore J. A. Leigh, &#226;&#8364;&#339;Markerless mutagenesis in <br /> which support genetic analysis plethora of <br />                                                                         Methanococcus maripaludis demonstrates roles alanine <br /> hypothetical poorly annotated genes Archaea.                     dehydrogenase, alanine racemase, alanine permease,&#226;&#8364;? <br />                                                                         Journal Bacteriology, vol. 187, no. 3, pp. 972&#226;&#8364;&#8220;979, 2005. <br />                                                                    [11] M. A. Pritchett, J. K. Zhang, W. W. Metcalf, &#226;&#8364;&#339;Development <br /> Authors&#226;&#8364;&#8482; Contributions                                                  markerless genetic exchange method Methanosarcina <br />                                                                         acetivorans C2A use construction new genetic <br /> IKB VdC-L designed &amp; performed research wrote                   tools methanogenic archaea,&#226;&#8364;? Applied Environmental <br /> the manuscript. GP, CEB, KG BEY helped strain                  Microbiology, vol. 70, no. 3, pp. 1425&#226;&#8364;&#8220;1433, 2004. <br />  10                                                                                                                                        Archaea <br />  <br /> [12] M. Dyall-Smith, The Halohandbook: Protocols Halobacterial         [28] G. Giaever, A. M. Chu, L. Ni et al., &#226;&#8364;&#339;Functional profiling the <br />      Genetics, vol. 6.01, 2006, http://www.microbiol.unimelb.edu                Saccharomyces cerevisiae genome,&#226;&#8364;? Nature, vol. 418, no. 6896, <br />      .au/micro/sta&#239;&#172;&#8364;/mds/HaloHandbook/.                                          pp. 387&#226;&#8364;&#8220;391, 2002. <br /> [13] A. Zaigler, S. C. Schuster, J. Soppa, &#226;&#8364;&#339;Construction           [29] K. Kobayashi, S. D. Ehrlich, A. Albertini et al., &#226;&#8364;&#339;Essential <br />      usage onefold-coverage shotgun dna microarray char-                Bacillus subtilis genes,&#226;&#8364;? Proceedings National Academy <br />      acterize metabolism archaeon Haloferax volcanii,&#226;&#8364;?               Sciences United States America, vol. 100, no. 8, pp. <br />      Molecular Microbiology, vol. 48, no. 4, pp. 1089&#226;&#8364;&#8220;1105, 2003.               4678&#226;&#8364;&#8220;4683, 2003. <br /> [14] A. Hartman, C. Norais, J. Badger et al., &#226;&#8364;&#339;The complete genome         [30] M. E. Hillenmeyer, E. Ericson, R. W. Davis, C. Nislow, D. <br />      sequence Haloferax volcanii DS2, model archaeon,&#226;&#8364;? PLoS                Koller, G. Giaever, &#226;&#8364;&#339;Systematic analysis genome-wide <br />      One, vol. 5, article no. e9605, 2010.                                      fitness data yeast reveals novel gene function drug <br /> [15] T. Allers, S. Barak, S. Liddell, K. Wardell, M. Mevarech,              action,&#226;&#8364;? Genome Biology, vol. 11, no. 3, article no. r30, 2010. <br />      &#226;&#8364;&#339;Improved strains plasmid vectors conditional overex-         [31] M. E. Hillenmeyer, E. Fung, J. Wildenhain et al., &#226;&#8364;&#339;The chemical <br />      pression His-tagged proteins Haloferax volcanii,&#226;&#8364;? Applied            genomic portrait yeast: uncovering phenotype all <br />      Environmental Microbiology, vol. 76, no. 6, pp. 1759&#226;&#8364;&#8220;1769,             genes,&#226;&#8364;? Science, vol. 320, no. 5874, pp. 362&#226;&#8364;&#8220;365, 2008. <br />      2010.                                                                 [32] C. Tamae, A. Liu, K. Kim et al., &#226;&#8364;&#339;Determination antibiotic <br /> [16] T. Allers, H.-P. Ngo, M. Mevarech, R. G. Lloyd, &#226;&#8364;&#339;Devel-                hypersensitivity 4,000 single-gene-knockout mutants <br />      opment additional selectable markers halophilic                 Escherichia coli,&#226;&#8364;? Journal Bacteriology, vol. 190, no. 17, pp. <br />      archaeon Haloferax volcanii based leuB trpA genes,&#226;&#8364;?             5981&#226;&#8364;&#8220;5988, 2008. <br />      Applied Environmental Microbiology, vol. 70, no. 2, pp.           [33] S. J. Kaczowka J. A. Maupin-Furlow, &#226;&#8364;&#339;Subunit topology <br />      943&#226;&#8364;&#8220;953, 2004.                                                             20S proteasomes Haloferax volcanii,&#226;&#8364;? Journal of <br /> [17] B. El Yacoubi, G. Phillips, I. K. Blaby et al., &#226;&#8364;&#339;A gateway platform        Bacteriology, vol. 185, no. 1, pp. 165&#226;&#8364;&#8220;174, 2003. <br />      functional genomics Haloferax volcanii: deletion      [34] V. Cr&#194;&#180;ecy-Lagard, C. Brochier-Armanet, J. Urbonaviius et <br />      tRNA modification genes,&#226;&#8364;? Archaea, vol. 2, no. 4, pp. 211&#226;&#8364;&#8220;219,             al., &#226;&#8364;&#339;Biosynthesis wyosine derivatives tRNA: ancient <br />      2009.                                                                      highly diverse pathway archaea,&#226;&#8364;? Molecular Biology and <br /> [18] B. Bisle, A. Schmidt, B. Scheibe et al., &#226;&#8364;&#339;Quantitative profiling        Evolution, vol. 27, no. 9, pp. 2062&#226;&#8364;&#8220;2077, 2010. <br />      membrane proteome Halophilic archaeon,&#226;&#8364;? Molecular              [35] B. Zhu, G. Cai, E. O. Hall, G. J. Freeman, &#226;&#8364;&#339;In-Fusion <br />      Cellular Proteomics, vol. 5, no. 9, pp. 1543&#226;&#8364;&#8220;1558, 2006.               assembly: seamless engineering multidomain fusion pro- <br /> [19] C. Lange, A. Zaigler, M. Hammelmann et al., &#226;&#8364;&#339;Genome-wide                   teins, modular vectors, mutations,&#226;&#8364;? BioTechniques, vol. 43, <br />      analysis growth phase-dependent translational tran-                 no. 3, pp. 354&#226;&#8364;&#8220;359, 2007. <br />      scriptional regulation halophilic archaea,&#226;&#8364;? BMC Genomics,          [36] S. Auxilien, F. El Khadali, A. Rasmussen, S. Douthwaite, H. <br />      vol. 8, article no. 415, 2007.                                             Grosjean, &#226;&#8364;&#339;Archease Pyrococcus abyssi improves substrate <br /> [20] A. Tebbe, A. Schmidt, K. Konstantinidis et al., &#226;&#8364;&#339;Life-style                specificity solubility tRNA m5 C methyltransferase,&#226;&#8364;? <br />      changes halophilic archaeon analyzed quantitative                  Journal Biological Chemistry, vol. 282, no. 26, pp. 18711&#226;&#8364;&#8220; <br />      proteomics,&#226;&#8364;? Proteomics, vol. 9, no. 15, pp. 3843&#226;&#8364;&#8220;3855, 2009.              18721, 2007. <br /> [21] J. Twellmeyer, A. Wende, J. Wolfertz et al., &#226;&#8364;&#339;Microarray analysis     [37] R. Gupta, &#226;&#8364;&#339;Halobacterium volcanii tRNAs. Identification 41 <br />      archaeon Halobacterium salinarum strain R1,&#226;&#8364;? PLoS                   tRNAs covering amino acids, sequences 33 class <br />      ONE, vol. 2, no. 10, article no. e1064, 2007.                              I tRNAs,&#226;&#8364;? Journal Biological Chemistry, vol. 259, no. 15, pp. <br /> [22] S. Y. Gerdes, M. D. Scholle, J. W. Campbell et al., &#226;&#8364;&#339;Experimen-            9461&#226;&#8364;&#8220;9471, 1984. <br />      tal determination level analysis essential genes        [38] R. M. McCarty, A.   &#194;&#180; Somogyi, G. Lin, N. E. Jacobsen, V. <br />      Escherichia coli MG1655,&#226;&#8364;? Journal Bacteriology, vol. 185,            Bandarian, &#226;&#8364;&#339;The deazapurine biosynthetic pathway revealed: <br />      no. 19, pp. 5673&#226;&#8364;&#8220;5684, 2003.                                               vitro enzymatic synthesis PreQ0 guanosine 5  - <br /> [23] N. Suzuki, N. Okai, H. Nonaka, Y. Tsuge, M. Inui,                      triphosphate steps,&#226;&#8364;? Biochemistry, vol. 48, no. 18, pp. <br />      H. Yukawa, &#226;&#8364;&#339;High-throughput transposon mutagenesis                      3847&#226;&#8364;&#8220;3852, 2009. <br />      Corynebacterium glutamicum construction single-              [39] J. S. Reader, D. Metzgar, P. Schimmel, V. De Cr&#194;&#180;ecy- <br />      gene disruptant mutant library,&#226;&#8364;? Applied Environmental                 Lagard, &#226;&#8364;&#339;Identification genes necessary biosynthesis <br />      Microbiology, vol. 72, no. 5, pp. 3750&#226;&#8364;&#8220;3755, 2006.                         modified nucleoside queuosine,&#226;&#8364;? Journal Biological <br /> [24] B. J. Akerley, E. J. Rubin, V. L. Novick, K. Amaya, N. Judson,             Chemistry, vol. 279, no. 8, pp. 6280&#226;&#8364;&#8220;6285, 2004. <br />      J. J. Mekalanos, &#226;&#8364;&#339;A genome-scale analysis identification      [40] M. Watanabe, M. Matsuo, S. Tanaka et al., &#226;&#8364;&#339;Biosynthesis of <br />      genes required growth <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>survival</span> Haemophilus                    archaeosine, novel derivative 7-deazaguanosine specific <br />      influenzae,&#226;&#8364;? Proceedings National Academy Sciences            archaeal tRNA, proceeds pathway involving base <br />      United States America, vol. 99, no. 2, pp. 966&#226;&#8364;&#8220;971, 2002.           replacement tRNA polynucleotide chain,&#226;&#8364;? Journal of <br /> [25] M. A. Jacobs, A. Alwood, I. Thaipisuttikul et al., &#226;&#8364;&#339;Comprehen-             Biological Chemistry, vol. 272, no. 32, pp. 20146&#226;&#8364;&#8220;20151, 1997. <br />      sive transposon mutant library Pseudomonas aeruginosa,&#226;&#8364;?            [41] M. Sprinzl, C. Steegborn, F. H&#194;&#168;ubel, S. Steinberg, &#226;&#8364;&#339;Com- <br />      Proceedings National Academy Sciences United              pilation tRNA sequences sequences tRNA genes,&#226;&#8364;? <br />      States America, vol. 100, no. 2, pp. 14339&#226;&#8364;&#8220;14344, 2003.                 Nucleic Acids Research, vol. 24, no. 1, pp. 68&#226;&#8364;&#8220;72, 1996. <br /> [26] N. T. Liberati, J. M. Urbach, S. Miyata et al., &#226;&#8364;&#339;An ordered,          [42] N. Altman-Price M. Mevarech, &#226;&#8364;&#339;Genetic evidence the <br />      nonredundant library Pseudomonas aeruginosa strain PA14                 importance protein acetylation protein deacetylation <br />      transposon insertion mutants,&#226;&#8364;? Proceedings National                 halophilic archaeon Haloferax volcanii,&#226;&#8364;? Journal of <br />      Academy Sciences United States America, vol. 103,             Bacteriology, vol. 191, no. 5, pp. 1610&#226;&#8364;&#8220;1617, 2009. <br />      no. 8, pp. 2833&#226;&#8364;&#8220;2838, 2006.                                           [43] E. Bab-Dinitz, H. Shmuely, J. Maupin-Furlow, J. Eichler, and <br /> [27] T. Baba, T. Ara, M. Hasegawa et al., &#226;&#8364;&#339;Construction                      B. Shaanan, &#226;&#8364;&#339;Haloferax volcanii PitA: example functional <br />      Escherichia coli K-12 in-frame, single-<span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants:              interaction Pfam chlorite dismutase antibi- <br />      Keio collection,&#226;&#8364;? Molecular Systems Biology, vol. 2, article           otic biosynthesis monooxygenase families?&#226;&#8364;? Bioinformatics, <br />      no. 2006.0008, 2006.                                                       vol. 22, no. 6, pp. 671&#226;&#8364;&#8220;675, 2006. <br />  Archaea                                                                                                                                        11 <br />  <br /> [44] A. Large, C. Stamme, C. Lange et al., &#226;&#8364;&#339;Characterization                  hydrolysis products,&#226;&#8364;? Journal Biological Chemistry, vol. <br />      tightly controlled promoter halophilic archaeon                    255, no. 22, pp. 10710&#226;&#8364;&#8220;10716, 1980. <br />      Haloferax volcanii use analysis essential      [59]   P. A. Ortiz, R. Ulloque, G. K. Kihara, H. Zheng, T. G. <br />      cct1 gene,&#226;&#8364;? Molecular Microbiology, vol. 66, no. 5, pp. 1092&#226;&#8364;&#8220;               Kinzy, &#226;&#8364;&#339;Translation elongation factor 2 anticodon mimicry <br />      1106, 2007.                                                                 domain mutants a&#239;&#172;&#8364;ect fidelity diphtheria toxin resis- <br /> [45] T. Suzuki, Y. Ikeuchi, S. Kimura et al., &#226;&#8364;&#339;Agmatine-conjugated               tance,&#226;&#8364;? Journal Biological Chemistry, vol. 281, no. 43, pp. <br />      cytidine tRNA anticodon essential AUA decoding               32639&#226;&#8364;&#8220;32648, 2006. <br />      archaea,&#226;&#8364;? Nature Chemical Biology, vol. 6, no. 4, pp. 277&#226;&#8364;&#8220;282,       [60]   C.-M. Chen R. R. Behringer, &#226;&#8364;&#339;Ovca1 regulates cell <br />      2010.                                                                       proliferation, embryonic development, tumorigenesis,&#226;&#8364;? <br /> [46] D. Mandal, C. K&#194;&#168;ohrer, D. Su et al., &#226;&#8364;&#339;Agmatidine, modified                Genes Development, vol. 18, no. 3, pp. 320&#226;&#8364;&#8220;332, 2004. <br />      cytidine anticodon archaeal tRNA(Ile), base pairs          [61]   T. R. Webb, S. H. Cross, L. McKie et al., &#226;&#8364;&#339;Diphthamide <br />      adenosine guanosine,&#226;&#8364;? Proceedings                  modification eEF2 requires J-domain protein is <br />      National Academy Sciences United States America,               essential normal development,&#226;&#8364;? Journal Cell Science, vol. <br />      vol. 107, no. 7, pp. 2872&#226;&#8364;&#8220;2877, 2010.                                       121, no. 19, pp. 3140&#226;&#8364;&#8220;3145, 2008. <br /> [47] A. Soma, Y. Ikeuchi, S. Kanemasa et al., &#226;&#8364;&#339;An RNA-modifying           [62]   B. S. Laursen, I. Siwanowicz, G. Larigauderie et al., &#226;&#8364;&#339;Char- <br />      enzyme governs codon amino acid speci-                    acterization mutations GTP-binding domain IF2 <br />      ficities isoleucine tRNA,&#226;&#8364;? Molecular Cell, vol. 12, no. 3, pp.           resulting cold-sensitive growth Escherichia coli,&#226;&#8364;? Journal <br />      689&#226;&#8364;&#8220;698, 2003.                                                              Molecular Biology, vol. 326, no. 2, pp. 543&#226;&#8364;&#8220;551, 2003. <br /> [48] B. El Yacoubi, B. Lyons, Y. Cruz et al., &#226;&#8364;&#339;The universal              [63]   Z. Zhou M. P. Deutscher, &#226;&#8364;&#339;An essential function for <br />      YrdC/Sua5 family required formation threonyl-                 phosphate-dependent exoribonucleases RNase PH and <br /> </body></html>