<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, OCt. 1990, p. 5226-5234                                                                  Vol. 10, No. 10 <br /> 0270-7306/90/105226-09$02.00/0 <br /> Copyright C 1990, American Society Microbiology <br />  <br />  <br />         REBI, Yeast DNA-Binding Protein Many Targets, Is <br />        Essential Cell Growth Bears Some Resemblance the <br />                                Oncogene myb <br />                            QIDA JU, BERNICE E. MORROW, AND JONATHAN R. WARNER* <br />                    Department Cell Biology, Albert Einstein College Medicine, Bronx, New York 10461 <br />                                             Received 23 April 1990/Accepted 6 July 1990 <br />  <br />              REB1 DNA-binding protein recognizes sites enhancer promoter rRNA <br />            transcription upstream genes transcribed RNA polymerase II. We report cloning <br />            gene REB1 screening yeast genomic Agtll library specific oligonucleotides containing the <br />            REB1 binding site consensus sequence. The REBI gene sequenced, revealing open reading frame <br />            encoding 809 amino acids. The predicted protein highly hydrophilic, numerous OH-containing amino <br />            acids glutamines, features common general DNA-binding proteins Saccharomyces <br />            cerevisiae, ABF1, RAP1, GCN4, HSF1. There homology portion REB1 and <br />            DNA-binding domain oncogene myb. REBI essential gene maps chromosome II. <br />            However, physiological role plays cell established. <br />  <br />    In yeast Saccharomyces cerevisiae, 200 copies               REB1 consensus binding site CCGGGTA (28) CGGGTRR <br /> of rRNA genes arranged single tandem array             (where R purine) (4) upstream number of <br /> is transcribed RNA polymerase I. Each repeat contains               genes, e.g., ACT], RAP], TRP5, SIN3, SWI5, TOP] (4, <br /> 7-kilobase (kb) transcription <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> product processed            43; Morrow et al., unpublished), cen- <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> form 25S, 18S, 5.8S RNAs mature ribo-                  tromere, CEN4, subtelomeric X Y regions (4). In <br /> some. Adjacent transcription units separated 2.5-kb             case level binding REB1 observed (4, <br /> spacer. Within spacer lies enhancer specific RNA             27, 43). REB1 shown bind GAL1-GAL10 <br /> polymerase I. This enhancer active orientation,             upstream activating sequence (UAS), leading localized <br /> and unlike RNA polymerase II enhancers S. cerevisiae,              exclusion nucleosomes (8). Although site not <br /> stimulates transcription placed upstream                  essential Gal4-dependent transcription GAL] and <br /> downstream rRNA gene (6, 17).                                     GAL10 (44), involved Gal4-independent tran- <br />    The enhancer lies 2 kb upstream transcription                  scription (9). In certain circumstances highly posi- <br /> initiation site 100 <span id='am-5' about='xsp:base' typeof='owl:Thing'>base</span> pairs end 35S          tion-dependent manner, REB1 binding site act to <br /> precursor rRNA. Indeed, current evidence suggests               stimulate transcription (3, 4). In circumstances can <br /> end 35S RNA arises processing event                   inhibit transcription (43). <br /> transcription continues enhancer terminate                 In order understand function abundant <br /> within (19, 27). These considerations suggested            protein widespread binding sites, cloned the <br /> might coupling termination initiation (7, 17,             gene REB1 screening yeast Agtll library a <br /> 19).                                                                     double-stranded oligonucleotide REB1 binds spe- <br />    On assumption protein factors involved              cifically (37, 42). REB1 essential gene. The amino acid <br /> in termination activation transcription,                sequence deduced open reading frame contains 809 <br /> identified proteins, termed REB1 REB2, bind                 amino acids high density hydrophilic residues. It <br /> within yeast rRNA enhancer (28). REB1                       homology DNA-binding domain the <br /> purified homogeneity single polypeptide                  oncogene myb (20, 21). <br /> apparent Mr 125,000 (B. E. Morrow, Q. Ju, J. R. <br /> Warner, unpublished data). REB1 binds the <br /> enhancer similar sequence 200 base pairs                           MATERIALS AND METHODS <br /> upstream 35S RNA transcription initiation site. Nev- <br /> ertheless, function REB1 rRNA transcription                Strains, media, libraries. The S. cerevisiae strain used <br /> clear, deletion REB1 binding sequences               study W303 (Matala ade2-1 leu2-3,112 <br /> test gene little effect transcription (22).                   his3-11 trpl-l ura3-1 can1-100) (38) haploid deriva- <br />    It likely REB1 play general                 tives. BJ926 (Matalao trpl+ +/his prcl-1261prcl-126 pep4-31 <br /> role organization cell&apos;s DNA. By criteria           pep4-3 prbl-11331prbl-1133 canllcanl), obtained the <br /> oligonucleotide competition, high-resolution footprint anal-             Yeast Genetic Stock Center, Berkeley, Calif., protease- <br /> ysis migration sodium dodecyl sulfate-polyacrylamide              deficient strain (18) used preparing REB1 protein. <br /> gel electrophoresis (SDS-PAGE) gels (28; Morrow et al.,                     Escherichia coli strains Y1090 [1acU169 proA+ lon <br /> unpublished), REB1 identical protein studied                araD139 strA supF (trpC22: :TnlO) (pMC9)] Y1089 <br /> others names Y (8), GRF2 (4), QBP (3). The                 [lacUJ69 proA + Ion araD139 strA hflA (chr::TnlO) (pMV10)] <br />                                                                          used Agtll host strains. Y1090 used screen <br />                                                                           Xgtll recombinants, Y1089 used generate lyso- <br />                                                                          gens analysis lacZ-REBJ fusion protein (16). <br />   * <br />       Corresponding author.                                                 Preparation oligonucleotide probe. A 200-ng amount <br />                                                                   5226 <br />  VOL. 10, 1990                                                                            CLONING OF YEAST REB1              5227 <br />  <br /> of complementary oligonucleotides, JW107           template. After phenol extraction, mixture spin- <br /> JW106                                                              dialyzed remove dNTPs oligonucleotides a <br />                 JW107 GATCTACTGGGTTACCCGGGGCACCTG <br />                                                                    Centricon-30 microconcentrator. Single-stranded DNA was <br />                 JW106     ATGACCCAATGGGCCCCGTGGACCTAG              precipitated ethanol used directly sequencing. <br />                                                                       Disruption REBI. An EcoRI-KpnI fragment (A; Fig. <br /> were    separately phosphorylated [-y-32P]ATP using        2) REB1 gene cloned pGEM-Blue. The <br /> polynucleotide kinase. Labeled oligonucleotides mixed         recombinant plasmid cut BgIII treated with <br /> and annealed heating 85&#194;&#176;C 2 min, 65&#194;&#176;C bath 15        Klenow fragment make blunt ends. A 1.16-kb HindIII <br /> min, 37&#194;&#176;C 15 min, finally room temperature 15       fragment containing URA3 gene treated Kle- <br /> min. Annealed oligonucleotides ligated overnight           fragment ligated BglII site. The new <br /> 12&#194;&#176;C passed Sepharose P-60 column            EcoRI-KpnI fragment containing URA3 gene ex- <br /> remove unincorporated [y-32P]ATP. Labeled DNA fractions            cised plasmid used transform W303 (a/a) <br /> were used directly probes.                                      (33). The transformants selected medium lacking <br />    Screening Agtll library. The Xgtll library sheared S.    uracil. A Southern blot genomic DNA probed with <br /> cerevisiae S288c DNA (14) generous gift M. Snyder         nick-translated EcoRI-KpnI fragment verify dis- <br /> (Yale University) screened essentially described        ruption REBI. One transformant sporulated, <br /> before (37, 42). Duplicate nitrocellulose replicas       tetrads dissected. <br /> form plate dried 15 min room temperature.             Primer extension. Total RNA strain <br /> All subsequent steps performed 4&#194;&#176;C. Each dried             W303a. Polyadenylated [poly(A)+] RNA purified with <br /> filter immersed 6 M guanidine hydrochloride (GuHCl)         oligo(dT)-cellulose. Poly(A)+ RNA (10 jxg) lyophilized <br /> in binding buffer (25 mM Tris hydrochloride [Tris-HCl, pH          dissolved 10 ,ul annealing buffer (0.1 M Tris-HCl <br /> 8.0], 0.13 mM EDTA, 0.6 mM dithiothreitol [DTT], 87 mM             [pH 8.3], 0.14 M KCl) containing 20 ng 32P-labeled <br /> KCl, 6.2 mM MgCl2, 0.6 mM CaCl2, 7% glycerol) (13)             oligonucleotide JW233 (TCGTTTCTATTACTGTCATC; <br /> gently shaken 5 min. The bound protein                complementary nucleotides 209 190 Fig. 6). The <br /> renatured immersing filter series decreasing        mixture denatured 90&#194;&#176;C 3 min annealed at <br /> concentrations GuHCl (3.0, 1.5, 0.75, 0.38, 0.19, 0         41&#194;&#176;C 5 min. Extension buffer (0.1 M Tris-HCl [pH 8.3], <br /> M), 5 min. After renaturation, filter blocked     0.14 M KCl, 15 mM DTT, 0.45 mM dNTPs) (12 RI) 20 U <br /> in 5% instant nonfat dry milk (Carnation) binding buffer        (1 RI) reverse transcriptase added, incubation <br /> for 30 min. The blot incubated concatenated           continued 41&#194;&#176;C 2 h. The reaction stopped by <br /> oligonucleotide (106 dpm/ml) 0.25% nonfat dry milk           adding 2.1 p1 3 M sodium acetate 52 pul ethanol, and <br /> binding buffer 16 h gentle shaking. The filter        mix left dry ice 20 min. The DNA-RNA <br /> then washed twice 0.25% nonfat dry milk binding            hybrids collected centrifugation dissolved 4 <br /> buffer autoradiographed.                                       [lI RNase (1 p.g/pl). After incubation 37&#194;&#176;C 20 min, <br />    Preparation crude lysogen extract. E. coli Y1089         2.5 RIl 95% formamide containing 10 mM EDTA was <br /> infected QJ1 phage (5 PFU/cell). Lysogenic cells         added, sample heated 70&#194;&#176;C 2 min and <br /> detected temperature sensitivity. A 10-ml culture      loaded 6% sequencing gel. <br /> lysogenic cells OD6. 0.5 diluted equal           Yeast genomic PCR used gel retardation footprint <br /> volume LB 55&#194;&#176;C incubated 42&#194;&#176;C 20 min           assay. Since restriction sites available for <br /> induce expression prophage. IPTG (isopropyl-p-D-            constructing probe containing REB1 binding sites <br /> thiogalactopyranoside) added final concentration       upstream REBI gene, PCR probe gel <br /> 10 mM induce synthesis LacZ-REB1 fusion              retardation footprint assays. Oligonucleotides JW202 <br /> protein. The culture incubated 1 h 37&#194;&#176;C       (TACGTAAACTTTCAATACAT; nucleotides -360 to <br /> vigorous shaking. The culture centrifuged, cells        -341, 1 pug) JW243 (ATAACCTTGCTTGTGATGAT; <br /> were suspended lysis buffer (100 mM Tris-HCl [pH 7.4],          complementary nucleotides -71 -90, 1 jig) were <br /> 10 mM EDTA, 1 mM phenylmethylsulfonyl fluoride). The               labeled [-y-32P]ATP using polynucleotide kinase. The <br /> cells disrupted cycles rapid freeze-thaw,         labeled oligonucleotides used primers 30 cycles <br /> debris removed low-speed centrifugation,            PCR starting W303a genomic DNA template. <br /> supernatant dialyzed dialysis buffer (20 mM            The reaction mixture purified Sepharose P-60 <br /> Tris-HCl [pH 8.0], 100 mM KCI, 1 mM EDTA, 0.6 mM DTT,              column used directly gel retardation assay (1,500 <br /> 10% glycerol, 0.05% Nonidet P-40) 4&#194;&#176;C. The dialyzed             cpm, Cerenkov). Only oligonucleotide primers <br /> extract stored -70&#194;&#176;C. Portions used             32P labeled used footprint assays. <br /> directly gel retardation assay purification.      Nucleotide sequence accession number. The nucleotide <br />    DNA sequence analysis. The EcoRI fragment QJ1          sequences determined study submitted to <br /> subcloned pGEM-Blue; recombinant plasmid              GenBank accession no. M36598. <br /> cut BamHI SphI treated exonuclease III <br /> (Erase-a-base; Promega Biotech) make series dele-                                   RESULTS <br /> tions subsequently cloned used DNA <br /> sequencing. After strand sequenced, series         Cloning REBI yeast Agtll expression library. We <br /> oligonucleotides used sequence strand.           cloned REBI gene virtue high-affinity <br />    Yeast genomic PCR reaction used sequence analysis. An       binding specific DNA sequence rDNA <br /> asymmetric polymerase chain reaction (PCR) procedure           promoter region. A Agtll expression library sheared S. <br /> used synthesize single-stranded DNA yeast genomic          cerevisiae DNA screened, following GuHCl denatur- <br /> DNA sequencing (14). Oligonucleotides JW209 (nucleo-           ation-renaturation protocol, concatamerized oligonu- <br /> tides 392 411 Fig. 7) (500 ng) JW169 (complemen-         cleotide probe REB1 binding site near rRNA <br /> tary nucleotides 934 915 Fig. 7) (100 ng) used    promoter (37, 42) (see Materials Methods). Duplicated <br /> primers 30 cycles PCR starting genomic DNA        plaque lifts used avoid artifacts. Out 60,000 <br />  5228          JU ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />        A                                                B <br />                                                                                                                  *   *     ~** <br />  <br />  <br />  <br />  <br />                                                                             S.                                        0    . <br />                                                                                                                                          S~~~~ <br />  <br />  <br />                                                                                   0                                  0.~~~~~ <br />  <br />  <br />                                                                                                                           **.    **t <br />  <br />                                                                                                                            .: <br />  <br />  <br />   FIG. 1. Cloning REBI Xgtll library. Bacterial cells (Y1090) infected phage yeast genomic Xgtll library (16) and <br /> plated 9-cm petri dish plates 1,000 PFU/plate. Duplicate nitrocellulose replicas plate, cycled GuHCl, <br /> and probed labeled, concatenated oligonucleotide containing REB1 binding site (106 cpm/ml). The filter washed and <br /> autoradiographed. A positive plaque obtained screening (A) purified homogeneity steps (B C). <br />  <br /> plaques screened, 4 clearly positive                         ically DNA sequence. Therefore, 4.3-kb <br /> duplicate filters (Fig. 1A) purified homogeneity                     insert contain region encoding DNA-binding <br /> (Fig. 1B C).                                                                 domain REB1. <br />   Restriction enzyme analysis cross-hybridization tests                         Characterization REBI clone. To identify poten- <br /> on DNA plaque-purified phages showed                          tial REBI coding sequences insert, partial <br /> each identical 4.3-kb EcoRI insert. This phage                       restriction map constructed orient 4.3-kb insert in <br /> named QJ1 (Fig. 2).                                                              recombinant phage. Portions insert used to <br />   To produce fusion protein, E. coli Y1089 lysoge-                         probe Northern (RNA) blot (Fig. 5). Although several <br /> nized QJ1. A culture lysogen induced                          bands observed entire insert used a <br /> synthesize fusion protein extracted de-                         probe (lane T), A B fragments adjacent lacZ <br /> scribed Materials Methods. Gel retardation assays                         fusion region detected single band 3 kb (lanes <br /> performed crude extract showed fusion                          A B), close size predicted mRNA coding for <br /> protein form complex probe containing                         REBl protein migrates SDS gel protein of <br /> rDNA enhancer region (Fig. 3, lane 6). The double-stranded                       125,000 daltons (Da) (Morrow et al., unpublished). <br /> oligonucleotide representing REB1 binding site com-                          Use SP6 T7 transcription probes A fragment <br /> peted specific binding (lane 7). Deletion core                        confirmed orientation transcription S. cerevi- <br /> sequence CGGGTA REB1 binding site abolished                             siae left right, oriented Fig. 2. In recombi- <br /> binding (lane 8). As case authentic REBl (28), the <br /> fusion protein substantially higher affinity rDNA <br /> promoter site enhancer site (data shown). A                                                2 34 5678 <br /> footprint fusion protein bound promoter region <br /> was identical purified REB1 (Fig. 4). Approxi- <br /> mately 20 nucleotides strand protected. <br />    Both gel retardation footprint assays showed that <br /> the fusion protein yeast REB1 protein bound specif- <br />                                                                                                          Maa,. <br />                                    LaCZ       A B, C            D <br />        QJ 1 <br />                                           E       K K       K           E <br />                            F       G              H <br />        QJ2 <br />                       E        E          E             E <br />  <br />                                E          E       K K       K <br />   Genome              //                      -                     is <br />                                                                     I <br />                                                                                     FIG. 3. Gel retardation assays LacZ-REB1 fusion protein. <br />                        I <br />                                                                                  Gel retardation assays performed (28) 190-bp <br />                                           REBD    1~                             EcoRI-HindIII fragment containing enhancer REBl site (lanes 1 <br /> REB 1 ORF <br />                                                                                  3 6 7) AREBl probe, core REBl binding <br />   FIG. 2. REB1 gene clones Xgtll libraries. QJ1                 sequence, CGGGTA, deleted (lanes 4, 5, 8). REB1 protein <br /> the clone isolated; QJ2 second (see text). The lines               partially purified DEAE-Sepharose chromatography (28). <br /> represent insertions, hatched bars represent lambda phage                    The crude extract IPTG-induced QJ1 lysogen used the <br /> DNA. Relevant restriction enzyme sites indicated: E, EcoRI; K,               LacZ-REB1 fusion protein. A double-stranded oligonucleotide con- <br /> KpnI. Genome REB1 ORF location REB1 open                     taining REB1 site rRNA promoter (JW106 JW107) <br /> reading frame chromosome II. The locations oligonucleo-                used competitor. Lane 1, No protein; lane 2, REBl protein; <br /> tides used asymmetric genomic PCR indicated                   lane 3, REB1 protein plus competitor; lane 4, protein; lane 5, <br /> arrows. The BgIII site used gene disruption shown                   REB1 protein; lane 6, LacZ-REB1 protein; lane 7, LacZ-REB1 <br /> inverted triangle.                                                               protein plus competitor; lane 8, LacZ-REB1 protein. <br />  VOL. 10, 1990                                                                                  CLONING OF YEAST REB1                  5229 <br />  <br />  <br />                 A                  B                                                               T A B C D <br />                     1234567            1234567 <br />  <br />  <br />  <br />  <br />                     .I.e <br />  <br />  <br />  <br />  <br />                                        -.   ; b!!                       FIG. 5. Characterization REB1 clone. Poly(A)+ RNA was <br />                                                                       strain W303a using oligo(dT)-cellulose column. <br />                        _p. <br />                                                                       Poly(A)+ RNA (3 ,ug) loaded lane denaturing <br />                                                                       agarose gel, blotted nitrocellulose, probed fragment <br />                                                                       nick-translated phage. Lane T probed entire <br />                           lP-*                                        4.3-kb EcoRI-EcoRI fragment. Lanes A, B, C, D probed <br />                                                                       EcoRI-KpnI fragments A, B, C, D, <br />                                                                       respectively (Fig. 2). The arrow indicates mRNA REBI. <br />  <br />  <br />                                               -- <br />                                                                       oligonucleotide EcoRI site primers <br />                                                                       (Fig. 2). A product predicted size obtained; its <br />                                                                       sequence confirmed arrangement shown Fig. 2. <br />   FIG. 4. Footprint analysis LacZ-REB1 fusion protein.           REBl essential gene. REBI disrupted using the <br /> Large-scale gel retardation assays performed 350-bp       gene replacement technique Rothstein (33). A 1.16-kb <br /> EcoRI-HindIII probe containing REBI binding site         URA3 fragment inserted BglII site (Fig. 2), and <br /> rRNA promoter region. The probe labeled separately        EcoRI-KpnI fragment containing URA3 insert was <br /> strand filling Klenow fragment (A) kinase     used transform diploid strain W303 (a/a). Southern <br /> treatment (B). Either partially purified REB1 (DEAE-Sepharose         analysis Ura+ transformants showed two <br /> chromatography) (lane 4) crude QJ1 lysogen extract (lane 5)    REBI genes disrupted. Tetrad analysis the <br /> incubated DNA probe (100,000 cpm, Cerenkov) applied      spores derived transformant showed 2:2 segrega- <br /> to native gels. The uncomplexed DNA (lane 3) protein DNA <br /> complexes (lanes 4 5) isolated following chemical cleavage   tion lethality (Fig. 6A). The viable spores Ura- <br /> of DNA 1,10-phenanthroline-copper ion. Lanes 1 7 contain     functional REB1 gene. Microscopic exami- <br /> a radiolabeled DNA standard. The Maxam-Gilbert G-reaction se-         nation showed spores REBI gene was <br /> quencing ladder shown lanes 2 6.                            disrupted undergo divisions. No <br />                                                                       gross morphological abnormalities visible these <br /> nant phage QJ1, transcription lacZ promoter            REBI-disrupted cells. <br /> be transcribed produce LacZ-REB1 fusion protein                     High-stringency Southern hybridization analysis showed <br /> encoded sequences lambda A, B, C fragments. <br />    The cloned A, B, C fragments sequenced. A <br /> single open reading frame, initiation codon,                         A                         B <br /> started left end fragment A ended middle <br /> of fragment C. <br />    The entire REBI gene cloned second genomic <br /> Agtll library (obtained Clontech, Palo Alto, Calif.) that <br /> was screened oligonucleotide A fragment, <br /> located 100 nucleotides downstream EcoRI site (Fig. <br /> 2). Positive candidates checked second oligonu- <br /> cleotide A fragment. One plaque, termed QJ2, was <br /> positive oligonucleotides. <br />    DNA isolated QJ2 mapped shown Fig. 2.                 FIG. 6. REBI gene disruption complementation a <br /> Sequencing fragment H showed sequences             cloned REB1 gene. (A) A 1.16-kb URA3 gene cloned the <br /> of fragment A QJ1. Sequencing fragment G revealed             BglII site A fragment QJ1 (Fig. 2). The new EcoRI-KpnI <br /> a single open reading frame crossing EcoRI site link        fragment used transform W303 (a/a). The Ura+ transfor- <br /> with observed fragment A.                                 mants confirmed Southern analysis. The REBJ-disrupted <br />    Since original library constructed         diploid sporulated used tetrad analysis. (B) A 3.6-kb <br /> the endogenous EcoRI sites              EcoRI fragment containing fragments G H (Fig. 2) sub- <br /> end insert (16), possibility remained              cloned EcoRI site centromere plasmid pRS313. This <br />                                                                       recombinant plasmid used transform W303 (a/a) one <br /> juxtaposition fragments G H QJ2 arisen           copy REBI disrupted HIS3 gene. The His&apos; Ura+ <br /> during cloning procedure. That arrangement                 transformants sporulated used tetrad analysis. Note <br /> shown actually occurs yeast genome estab-              spore tetrads failure centromere <br /> lished carrying PCR genomic DNA                     plasmid appear spores. <br />  5230       JU ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />  -800 GAATTCGACTGATGAACAGCGCCCTTGTTCAAAGTCTGAACTAGATCCCTGTATTGGCAACGCAGCAAGTAAGCGAAGTTTCTATGATTATAACAGCGTTTACGTCGCGAGTGACGCAAT <br />  <br />  -680 TATTACGACTGCCGCTGCCACTGCCATTATCAGTAGTAATAGTGGAGACTATCAGCGTGGGCACGATGTTCGCGATGTTCCAAGAAATGTTTTGTTACAGGCAGGAGAAACAGATTTCAG <br />  <br />  -   560 TAGTGTGCTTCGGGTTGACTCCGATCTCAAGTTATCCAACATAAGTCATCATTCCCCCGTAAAACCTTCGTCAACTTCAAGTCATTCGACCTTCATTTTCGAGTCGGAAACTGACACTGA <br />  <br />  -440 TACTGATACTGACGCTGAAACAGAAAATGACATTGACGCTTACATAGACACCAGTATACCCAACCTGCTCCTATAAATTTTACGTAAACTTTCAATACATTGAAATTGTGTCTCATTTTC <br />  <br />  -320 TTCTAGAAGCTCCTCGGAAAATTGATCACGTGATATATAGCAACAACTTTGTTACCCGTCTTTCAAATTGGGCCCATGCCGGGTGGCAAACCAAAGGCGGCCAAGGGACCTACCCTCAC <br />  <br />  -   200 AAAGGACAACGACAAACAAATGGCAGCTTTCTTGACCACACAAGATAAAACAACAAAACTCAGGCTGTTATTACCCTGCTGTGGACCCAACCTAAAGTAAATAAATAAACATCATCACAA <br />  <br />      -80 GCAAGGTTATCTACACACTATCGTTATTAGTTTGAACTCACGTGTTGCTGCTGAATTAGAATATTTATAGCTCAATCAATATGCCTTCAGGTCATAACGATAAAAACGCAAATCAAGAGT <br />                                                                                           M P S G H N D K N A N Q E S <br />  <br />       41 CTGTGGAAGAGGCTGTTTTGAAATATGTCGGTGTAGGTCTAGATCACCAGAACCATGACCCTCAATTGCATACGAAGGATCTGGAAAACAAACACTCGAAAAGCAGAATATTGTGGAAA <br />       15   V E E A V L K Y V G V G L D H Q N H D P Q L H T K D L E N K H S K K Q N I V E S <br />  <br />      161 GTAGTAATGATGTTGATGTTAATAACAATGATGACAGTAATAGAAMCGAGGATAATAATGATGATTCTGAAAATATTAGCGCATTGAATGCGAACGAATCATCTTCGAATGTGGATCATG <br />       55   S N D V D V N N N D D S N R N E D N N D D S E N I S A L N A N E S S S N V D H A <br />  <br />      281 CCAACTCCAATGAACAACATAACGCAGTTATGGATTGGTATTTAAGGCAAACAGCGCATAATCAACAAGACGACGAAGATGACGAAAACAATAATAACACTGACAACGGCAATGACAGTA <br />       95   N S N E Q H N A V M D \J Y L R Q T A H N Q Q D D E D D E N N N N T D N G N D S N <br />  <br />      401 ATAATCACTTTTCTCAATCTGACATAGTCGTGGATGACGACGACGACAAGAATAAGAAAGATGCGGGTGTTGGTGTGGACGATGATCATCAATCTATGGCGATGGCCGCCGTCGCTGCTG <br />      135   N H F S Q S D I V V D D D D D K N K K D A G V G V D D D H Q S M A M A A V A A A <br />  <br />      521 CTTACACTCTATCGAAMMTAACAATAATAATAACAGTATTGCGAACGATAGCAATTCGCGTAAAAGACAGCATGATMACGGAAATAATCATGAGAATTCACAGAAAAAGCGGAAAAACA <br />      175   Y T L S K N N N N N N S I A N D S N S R K R 0 H D N G N N H E N S Q K K R K N N <br />  <br />      64 1 ATAATGATGACGATGACAGACAAATTGGAAACGTAGATCCGGAATTAACCACCTTGGGTGATGCAGATGACAACGATACTAATAATGATGTGATTGATCGTGATCAACTGGTCCATAAGG <br />      215    N D D D D R Q I G N V D P E L T T L G D A D D N D T N N D V I D R D Q L V H K A <br />  <br />      761 CCATTATCGACGCTGATTCGATTACCCAACATCCTGATTTCCAGCAATACTTGAATACTGCGGCTGATACTGACGATAACGMAAMATTAAMGCATATTAAAGATCATTTGATGCGTACAC <br />      255   I I D A D S I T Q H P D F 0 Q Y L N T A A D T D D N E K L K H I K D H L M R T H <br />  <br />      881 ACGGTTTGAATCATCAGAACAAGAATCACAATGAT GATACAGAT GAT TTAT CAAATAGTACAAAGCAATACTCT GAACT GCAGAAAGACTCTATGCTGGATAGT TCCCTCAACAAATCTA <br />      295   G L N H Q N K N H N D D T D D L S N S T K Q Y S E L Q K D S M L D S S L N K S R <br />  <br />  1 001 GAAATTATATGGAAGTCTTGCCCAAAGTTATCTCTCAAGATACTCAACCACACCAACAAAAATCTCCCTCTCATGATAATGAAGCTGGCAGCGTGGACAATTCAGAAATATCTCAACTAC <br />    335   N Y M E V L P K V I S Q D T Q P H Q K S P S H D N E A G S V D N S E I S Q L L <br />  <br />      1 121 TTCAATCAGCCGCCACAAAGGCATCATCTTTGGTATCTTTATCCTCATCCTCAGCAACGCCATCGACTTCAAGGTCTMCAATAGCAAAGCTTTTGATAAAGCCGAAGACGCCGCTTTAG <br />        375   Q S A A T K A S S L V S L S S S S A T P S T S R S N N S K A F D K A E D A A L E <br />  <br />      1241 AAAGATTTATTAACGAGTATGAGGCCATTGAACGTTTGACTAGACAACAAGTTTGTGAAAGGATATGGAGTTCCGACAGGCCAAAGGACAACTTTTGGAATAATATTTACAAAGTCTTAC <br />       415   R F I N E Y E A I E R L T R Q Q V C E R I W S S D R P K D N F W N N I Y K V L P <br />  <br />      1 361 CCTATAGATCTAGCTCCTCTATCTACAAACACATGAGAAGAAAATATCACATTTTTGAACAACGTGGTAAMTGGACCGCGGAGGAGGAACAAGAGCTAGCTAAATTATGTGCAGAAAG <br />        455   Y R S S S S I Y K H M R R K Y H I F E Q R G K \J T A E E E Q E L A K L C A E K E <br />  <br />      1481 AAGGTCAATGGGCAGAAATAGGTAAAACCTTAGGCAGAATGCCAGAAGATTGTAGGGATCGTTGGAGAAACTACGTAAAATGTGGTACCAATAGAGCATCAAATAGATGGTCCGTTGAAG <br />       495   G Q W A E I G K T L G R M P E D C R D R W R N Y V K C G T N R A S N R W S V E E <br />  <br />      160 1 AAGAAGAGCTTCTGAAAAGGTTATCAGCGATATGTTGGAAGAAGCCCAGCAACAACAGTCTCAATTGCATCCAAACTTATTGGAAGAAGAACAGCATTTACTGCAAGATGACCAGAATG <br />       535    E E L L K K V I q n M L E E A Q Q Q Q S Q L H P N L L E E E Q H L L Q D D Q N D <br />  VOL. 10, 1990                                                                             CLONING OF YEAST REB1               5231 <br />  <br />  <br /> 1725 ATCATCGCAATHNCGATGAGACGATGATGATACAGCTTCTGCAGCAGCAGCTCGTGCTGTCGTATTCGAACAN CAACACTTCTTCAC SAAGCAGCAGATGATGACGATGCTAT <br />  575   H R N N D E D D D D T A S A A A A R A V V F K N N N N F F N K S S K N N T N L L <br />  <br /> 1841 TGCGCTCGTGCGCTGCTGCTTCTTCATCCCTGGGAGAAACAAGACGAGACAACCCCACGAGTCATTAGGTATACAGCTCGATGATTTCCCAGACTCAMTGGTACCTGCTCCAT <br />  615   R S C A A A S S S L G E N K D E D K P H E S L G I Q L D D N S Q N S N V P A P S <br />  <br />  <br /> 1 961 CAGCAACAAGCACACATTCTAAGTTTGTCAAATACAATCAGACGTCACAATAATAAACTGAGGAAATCTTTGATGGGTAACGGTAAGTTAGATTTCAAGACATCATTAACTGGACCA <br />   655   A T S T H S K S L S N T I R R H N N K L R K S L N G N G K L D F K D I I N U T I <br />  <br /> 2081 TTGTCAGTGAGCGTATGGGCGGTACGAGATCACGTATTCAATGTCGTTATAATGGAATAAATTGGTCAAAAGGGAAGCAATTGCCAAAATTCAGACTGTAAAGATGATGATATGTTAT <br />  695   V S E R N G G T R S R I Q C R Y K W N K L V K R E A I A K I Q T V K D D D N L W <br />  <br /> 2201 GGATTTTTGAAAATTAAGGGATTTAGGTATAACAGAAGATTCTCAAGTAGATTGGGATGAACTTGCAGCTTTGAAGCCTGGCATGAAATTAAATGGGTTAGAGTTGAAATTGTGCTATG <br />  735   I F E K L R D L G I T E D S Q V D W D E L A A L K P G N K L N G L E L K L C Y E <br />  <br /> 2321 AAAGAATGAAGAAAAGGTCAAAGGCTATAAGCAAAATCAATTAATGAAATCAGTAAAGAGCTAGTTGATTATTTTAGCTCCAATATTTCAATGAAACAGAAAATTAATTCCGGMAAA <br />  775   R N K K K V K G Y K Q K S I N E I S K E L V D Y F S S N I S N K T E                                        50 <br />  <br /> 2441 TATAATTAGTGAAAGCGAAAACTCAATAATGTTTGATAGTACGCGTTATTTTTCATCATTTATCTTTTTTTTAGCCCTGTACATTTGGCAAATCTTGTCTTCACTTATTGCCGGATA <br />  <br /> 2561 TTGACTTTCTTTTTCTTCCTTGACGCCAACTGTTTTTATTTCCAACTTACACATCAAGC <br />   FIG. 7. Sequence REBI open reading frame. The sequence REBI determined strands dideoxy method <br /> as described text. The predicted amino acid sequence shown. The 5&apos; end transcript nucleotide -156. <br />  <br /> that REBI single-copy gene. A blot electrophoretically     upstream DNA sequences. It apparent homology to <br /> separated chromosomes S. cerevisiae probed           known genes. <br /> 32P-labeled fragment REB1. A single band corresponding            REB1 homology Myb proteins. A search the <br /> to chromosome II cross-hybridized (data shown).                GenBank NBRF data banks proteins a <br />    Cloned REBI gene functional. In order confirm        high degree homology REB1. However, search did <br /> the REB1 gene isolated sequenced functional            reveal REBI substantial similarity DNA- <br /> gene, 3.6-kb piece DNA containing G H                 binding domain Myb proteins (Fig. 8) (20, 21). It has <br /> fragments QJ2 (Fig. 2) subcloned EcoRI           suggested periodic tryptophans Myb form its <br /> site yeast centromere plasmid, pRS313 (36), carrying      DNA-binding site (1). It intriguing homology <br /> URA3 gene. The recombinant plasmid used transform           REB1 Myb includes motif. Another yeast <br /> W303 (a/a) copy REBI disrupted            DNA-binding protein, BAS1, substantial homology <br /> by HIS3 gene. On sporulation, transformant yielded         Myb (41). However, aside regions regular <br /> four viable spores (Fig. 6B). The existence viable Ura+         tryptophans, essentially sequence homology <br /> His&apos; spores demonstrated cloned REBI gene           BASR REB1. <br /> complement disrupted REBI gene.                                    5&apos; end REBI transcript. Primer extension used <br />    Characterization REBI sequence. The sequence          determine 5&apos; end REBI mRNA. By using <br /> the REBI gene shown Fig. 7. There single open          poly(A)+ RNA template oligonucleotide 187 <br /> reading frame encoding protein 809 amino acids. The           nucleotides downstream translation start codon (Fig. <br /> amino acid sequence deduced DNA sequence              7) primer, cDNA synthesized reverse tran- <br /> consistent result N-terminal sequence analysis of <br /> purified REB1 protein. If posttranslational proc-      scriptase, product compared sequencing <br /> essing, molecular weight REB1 protein deduced           reaction primer (Fig. 9). The 5&apos; end REB1 <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> DNA sequence 92,058. Purified REB1, however,               mRNA nucleotide -156 (Fig. 7). The AUG this <br /> migrated SDS-polyacryamide gel 125,000         long 5&apos; leader initiated REBI open reading frame. <br /> Da. This discrepancy characteristic yeast             REB1 binding sites REBI gene. Examination of <br /> DNA-binding proteins, e.g., GCN4 (12), RAP1 (35), ABF1             sequences upstream transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> REBI <br /> (11, 31), HSF1 (39).                                           revealed potential REB1 binding sites, TGTTACCCGT <br />    REB1 contained motifs common                -270 -260 CCGGGTGGAT -241 -233. Gel <br /> DNA-binding proteins, zinc finger (32) leucine      retardation analysis pure REB1 protein revealed two <br /> zipper (23). It unusual hydrophilicity          protein-DNA complexes (Fig. 10, lanes 5 8), suggesting <br /> charge, 12.7% basic residues (Arg plus Lys) 17.8%         sites fragment -71 -360. <br /> acidic residues (Asp plus Glu), 13.8% Ser Thr       After treatment fragment ApaI, cuts <br /> and 11.4% Gln. As expected protein charge         putative binding sites, subfragments <br /> density, regions high negative charge, e.g.,         retarded pure REB1 protein (Fig. 10, lanes 9 12). <br /> residues 117 149 533 584, play role      These observations suggest REB1 involved in <br /> the activation polymerase (12, 26).                             regulating synthesis. <br />    The sequence Fig. 7 reveals carboxy-terminal           In addition, nucleotides -318 -305 putative <br /> 114 amino acids open reading frame started       heat shock factor-binding site, CTAGAAGCTCCTCG (38). <br />  5232     JU ET AL.                                                                                                                                                MOL. CELL. BIOL. <br />  <br />                                             430                 440                450                460                470              480 <br />                                REB1     TRQQVCERIWSSDRPKDNFWNNIYKVLPYRSSSSIYKHMRRKYHIFEQRGKWTAEEEQEL <br />                                                                                          :1 11 11 : <br />                                MYB      MGAPLNCPMKSMPLFPSSAQTGITPPRQNRTDVQCQHRWQKVLNPELIKGPWTKEEDQRV <br />                                                            10                 20                 30                 40               50               60 <br />                                             490     500          510      520       530       540 <br />                                REB1     AKLCAE-KEGQWAEIGKTL-GRMPEDCRDRWRNYVKCGTNRASNRWSVEEEELLKKVISD <br />                                MYB <br />                                                       1:     11:             :: :::   1: 11::::: <br />                                         IELVQKYGPKRWSVIAKHLKGRIGKQCRERWHNHLNPEVKKTS--WTEEEDRIIYQA-HK <br />                                                 70        80        90        100         110 <br />                                               550                     560                570                580                590              600 <br />                                REB1     MLEEAQQQQSQLHPNLLEEEQHLLQDDQNDHRNNDED--DDDTASAAAARAVVFKNNNNF <br />                                MYB      RLGNRWAEIAKLLPGRTDNAIKNHINSTMRRKVEQEGYLQEPSKASQTPVATSFQKNNHL <br />                                         120                     130                140                150                160              170 <br />                                                    610                  620                630                640              650              660 <br />                                REB1     FNKSSKMMTMLLRSCAAASSSLGENKDEDKPHESLGIQLDDNSQNSMVPAPSATSTHSKS <br />                                MYB      MGFGHASPPSQLSPSGQSSVNSEYPYYHIAEAQNISSHVPYPVALHVNIVNVPQPAAAAI <br />                                         180                     190                200                210                220              230 <br />   FIG. 8. Homology REB1 Myb. The sequence REB1 compared NBRF protein data bank using FASTA <br /> program (30). A region REB1 showed substantial homology Myb proteins data bank. The computer-optimized alignment <br /> of REB1 mouse-transforming Myb oncogene (34) (NBRF identifier TVMSMA) shown, recurrent tryptophans emphasized. <br /> Note extensive overlap tryptophan regions obtained shifting Myb sequence left, cost, however, <br /> of perfect alignment amino acids. A vertical line indicates identical amino acids; colon indicates amino acids identified by <br /> the PAM250 matrix (5) having relatively high probability replacement other. In cases, e.g., R:K, G:D, L:I, the <br /> structural similarity obvious. In cases, probability determined empirically. <br />  <br />                          DISCUSSION                                                                The sequence REB1 (Fig. 7) remarkable high <br />    We recently purified (Morrow et al., unpublished)                                        concentration charged amino acids, serine threo- <br /> and cloned gene encoding REB1, specific DNA-                                           nine, glutamine. It interesting number the <br /> binding protein S. cerevisiae. Although REB1 initially                                    DNA-binding proteins, RAP1 (35), ABF1 (11, <br /> identified rDNA enhancer-binding protein (28),                                      31), HSF1 (39), share property, the <br /> since shown bind rRNA promoter region (28),                                       general size, 800 + 50 amino acids. Nevertheless, <br /> to GALJ-GALJO UAS (4, 8; Morrow et al., unpublished),                                        proteins bear little resemblance the <br /> and upstream ACT], RAP] (4), SIN3, SWI5 (43), and,                                            sequences compared. It easy could <br /> indeed, REBI itself (Fig. 10). Thus, REB1 sites                                       evolved single primordial DNA-binding protein. <br /> transcriptional control regions genes transcribed                                     In particular, characteristic motif for <br /> RNA polymerase I II. In addition, REB1 bind                                       DNA-binding regions different proteins. <br /> centromere, CEN4, subtelomeric X Y                                              Although DNA-binding region REB1 been <br /> regions (4), unlike ubiquitous DNA-binding pro-                                        localized carboxy two-thirds molecule, a <br /> teins S. cerevisiae, ABF1 RAP1.                                                   reasonable assumption resides region that <br />                                                                                                  resembles DNA-binding region Myb, motif of <br />                           P A C G T <br />                                             . &apos; _ EG-C                                                                         2 3 4            56 7 8     9 GO   12 <br />                                                   T <br />                                      ~~ @          T-A <br />  <br />  <br />                                &quot;&quot;V. <br />                                  -                 T-A <br />                                                    C -A* <br />                         _ _      &quot;                ~~~T-A <br />                                t--          I~~~~ <br />                                                -s <br />  <br />  <br />  <br />                                                                                                     FIG. 10. Binding REB1 upstream REBI gene. A 350-bp <br />                                                                                                  labeled EcoRI-HindIII fragment containing REBI site the <br />                                                                                                  promoter region used lanes 1 4. Alternatively, 290-bp <br />                                                                                                  labeled PCR product (nucleotides -360 -71 Fig. 7) containing <br />                                                                                                  putative REB1 binding sites used (lanes 5 8). Finally, <br />   FIG. 9. Primer extension analysis REBI mRNA. Poly(A)&apos;                                       PCR product cut ApaI yield fragments (lanes 9 <br /> RNA (10 p.g) 32 P-labeled oligonucleotide JW233 (complementa-                                12). These labeled DNA fragments used probes gel <br /> ry nucleotides +209 +190 Fig. 7) (20 p.g) used                                retardation assays purified REB1 (lanes 2 4, 6 8, 10 to <br /> primer extension reaction. The product (8,000 cpm, Cerenkov)                                 12). Lanes 1, 5, 9 contain protein. Binding REB1 was <br /> analyzed 6% sequencing gel (lane P). The noncoding strand                                competed double-stranded oligonucleotide containing <br /> the cloned REBJ gene sequenced using oligonu-                                    REB1 site used original screen (JW106 and <br /> cleotide (lanes ACGT). The arrow shows main product primer                                JW107; text) (lanes 3, 7, 11) unrelated <br /> extension, nucleotide -156 Fig. 7, indicated G*.                                        double-stranded oligonucleotide (lanes 4, 8, 12). <br />  VOL. 10, 1990                                                                               CLONING OF YEAST REB1                  5233 <br />  <br /> recurring tryptophans (1). A quite distinct yeast protein     fying mutant genes able suppress lethal effects <br /> resembles tryptophan motif Myb BAS1,            deletion REBI. <br /> involved constitutive low-level transcription certain       The REB1-LacZ fusion protein described Fig. 3 4 <br /> genes involved amino acid synthesis,       purified homogeneity used raise antiserum in <br /> other functions (41). The ets oncogene family         rabbits. Immunoprecipitation serum showed that <br /> encode conserved set recurrent tryptophans           REB1 protein phosphorylated certain serine resi- <br /> been proposed DNA-binding domain (10),          dues vivo (Morrow et al., unpublished). Other DNA- <br /> amino acids resemblance REB1 sequence.              binding proteins, e.g., HSF1, phosphorylated (39); <br /> None characteristic motifs DNA-binding             case HSF1, heat shock causes numerous changes in <br /> proteins appear REB1, i.e., leucine zipper          phosphorylation, without, however, having effect on <br /> region (23); zinc finger region (32); does   DNA-binding properties molecule (37). We have <br /> not appear conventional helix-loop-helix          determined environmental conditions affect <br /> associated basic region (29), numerous          phosphorylation activity REB1. It interesting that <br /> predicted helices, limited degree          c-myb phosphorylated casein kinase II that <br /> amphipathic character. Nor does sequence REB1 bear          phosphorylation causes lose DNA-binding activity <br /> any resemblance hUBF, protein factor           (25). Most oncogenic mutants myb lost the <br /> binds enhancer region upstream human rRNA            phosphorylation site. Casein kinase II identi- <br /> genes (15).                                                        fied important regulation rRNA transcription <br />    We shown (Fig. 6) REB1 essential cell          following mitogenic stimulation phosphorylation of <br /> growth, know specific physiological func-        RNA polymerase I, nucleolin, etc. (2). <br /> tion. Although REBl binds independently sites within <br /> the rDNA repeat vitro (28) vivo (22, 24),                                 ACKNOWLEDGMENTS <br /> experiments test genes suggest binding sites          We particularly grateful M. Snyder, provided Agtll <br /> do contribute rRNA transcription (22). While           library, V. Guacci, provided yeast chromosome blot. <br /> evidence suggests sequences close REB1            We thank D. Stillman, D. Chasman, R. Kornberg, K. Struhl, <br /> site rDNA enhancer element involved              M. Holland sharing results publication. <br /> termination rRNA transcription, evidence does            This work supported Public Health Service grants <br />                                                                    GM25532 CA13330 National Institutes Health and <br /> support conclusion (22). If role REB1 site      American Cancer Society grant MV-323S. <br /> the enhancer involves termination, REB1 site the <br /> promoter &quot;fail-safe&quot; termination site prevent                              LITERATURE CITED <br /> the disruption initiation complex passage        1. Anton, I. A., J. Frampton. 1988. Tryptophans myb <br /> rogue polymerase molecule escaped termination               proteins. Nature (London) 336:719. <br /> transcribing upstream gene. Alternatively, possible       2. Belenguer, P., V. Baldin, C. Mathieu, H. Prats, M. Bensaid, G. <br /> that REB1 plays role organizing rDNA repeats            Bouche, F. Amalric. 1989. Protein kinase NII the <br /> the chromosome (17) apparent experiments              regulation rDNA transcription mammalian cells. Nucleic <br />                                                                        Acids Res. 17:6625-6635. <br /> on test genes.                                                      3. Brandl, C. J., K. Struhl. 1990. A nucleosome-positioning <br />    The role REB1 transcription RNA polymerase II              sequence required GCN4 activate transcription the <br /> is clear. Although REB1 binding sites                absence TATA element. Mol. Cell. Biol. 10:4256-5265. <br /> upstream genes, case shown          4. Chasman, D. I., N. F. Lue, A. R. Buchman, J. W. LaPointe, Y. <br /> be essential upstream activating inhibiting           Lorch, R. D. Kornberg. 1990. A yeast protein influ- <br /> site. The thoroughly studied REB1 site            ences chromatin structure UASG functions a <br /> the GALJ-GALJO UAS. In binding there, REB1                       powerful auxiliary gene activator. Genes Dev. 4:503-514. <br /> significant effect structure surrounding chro-        5. Dayhoff, M. O., R. M. Schwartz, B. C. Orcutt. 1978. A <br />                                                                        model evolutionary change proteins, p. 345-352. In M. 0. <br /> matin (8), binding sequence deleted little         Dayhoff (ed.), Atlas protein sequence structure, vol. 5. <br /> effect Gal4-dependent induction transcription             National Biomedical Research Foundation, Silver Spring, Md. <br /> galactose (44). In recent experiments, however, Finley         6. Elion, E. A., J. R. Warner. 1986. An RNA polymerase I <br /> and West (9) identified element, termed GAE1,             enhancer Saccharomyces cerevisiae. Mol. Cell. Biol. 6:2089- <br /> by itself cause transcription GAL] promoter              2097. <br /> certain negatively acting &quot;operator&quot; regions removed.           7. Elion, E. A., J. R. Warner. 1987. Characterization yeast <br /> GAE1 contains sequence REB1 binds,             RNA polymerase I enhancer. UCLA Symp. Mol. Cell. Biol. <br /> not determined binding REB1 fact             New Series 52:21-29. <br /> responsible activity GAE1.                               8. Fedor, M. J., N. F. Lue, R. D. Kornberg. 1988. Statistical <br />                                                                        positioning nucleosomes specific protein-binding an <br />    In number artificial constructs, presence             upstream activating sequence yeast. J. Mol. Biol. 204:109- <br /> REB1 binding site able potentiate (3, 4)           127. <br /> inhibit (43) transcription nearby promoters. The precise       9. Finley, R. L., Jr., R. W. West, Jr. 1989. Differential <br /> location important, however, way uncharac-            repression GAL4 adjacent transcription activators by <br /> teristic usual upstream activating inhibiting ele-           operators yeast GAL upstream activating sequence. Mol. <br /> ments. Wang et al. (43) REB1 binding                   Cell. Biol. 9:4282-4290. <br /> activity reduced strains carrying mutations gene      10. Gunther, C. V., J. A. Nye, R. S. Bryner, B. J. Graves. 1990. <br /> SIN3, particularly growth galactose. Again,               Sequence-specific DNA binding proto-oncoprotein ets-1 <br /> biological significance observation unclear. In         defines transcriptional activator sequence long <br />                                                                        terminal repeat Moloney murine sarcoma virus. Genes <br /> case, availability cloned gene enable ask        Dev. 4:667-679. <br /> more precise questions function observing             11. Halfter, H., B. Kavety, J. Vandekerckhove, F. Kiefer, D. <br /> response cell withdrawal REB1, generat-            Gallwitz. 1989. Sequence, expression mutational analysis of <br /> ing temperature-sensitive mutations REBI, identi-            BAF1, transcriptional activator ARS1-binding protein of <br />  5234     JU ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />     yeast Saccharomyces cerevisiae. EMBO J. 8:4265-4272.                   mediate RNA polymerase I-dependent promoter terminator <br /> 12. Hope, I. A., K. Struhl. 1986. Functional dissection               activities. Mol. Cell. Biol. 9:1243-1254. <br />     eukaryotic transcriptional activator protein, GCN4 yeast.         28.   Morrow, B. E., S. P. Johnson, J. R. Warner. 1989. Proteins <br />     Cell 46:885-894.                                                           bind yeast rDNA enhancer. J. Biol. Chem. 264:9061- <br /> 13. Huet, J., A. Sentenac. 1987. TUF, DNA-binding factor               9068. <br />     specif <br /> </body></html>