<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Ctf7p is essential for sister chromatid <br /> cohesion and links mitotic chromosome <br /> structure <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> the DNA replication <br /> machinery <br />  <br /> CTF7 (chromosome transmission fidelity) gene in budding yeast encodes an essential protein that is required <br /> for high-fidelity chromosome transmission and contains regions of identity conserved <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> yeast to man. ctf7 <br /> mutant cells arrested prior to anaphase onset contain separated sister chromatids. Thus, Ctf7p is essential for <br /> cohesion. Cohesion is established during S phase and then maintained until mitosis. However, Ctf7p activity <br /> is required only during S phase, suggesting that Ctf7p functions in the establishment of cohesion. In addition, <br /> ctf7 genetically interacts with DNA metabolism mutations pol30 (PCNA) and ctf18 (an RF-C like protein) and <br /> ctf7 temperature sensitivity and chromosome loss are rescued by high levels of POL30. These findings provide <br /> the first evidence that links the establishment of sister chromatid cohesion to the DNA replication machinery <br /> and suggest that the assembly of cohesion (and possibly condensation) complexes are coupled to <br /> PCNA-dependent DNA replication. The analysis of Ctf7p also reveals an important connection between sister <br /> chromatid cohesion, spindle integrity and the spindle assemby checkpoint. <br /> [Key Words: CTF7; chromosome loss; BUB3; MAD2; POL30; chromatid cohesion; condensation] <br /> Received October 9, 1998; revised version accepted November 30, 1998. <br />  <br />  <br />  <br />  <br /> Cohesion between replicated sister chromatids is funda-                  cohesion complexes are inactivated and the sister chro- <br /> mental to cell viability. Cohesion tethers replicated sis-               matids segregate from one another and move toward <br /> ter chromatids together along their entire lengths from S                their spindle poles. <br /> phase until anaphase. Chromatid cohesion also sterically                    Despite the essential role of chromatid cohesion in <br /> constrains sister kinetochores so that they face away                    mitosis and cell division, only recently has progress been <br /> from each other, promoting proper chromosome attach-                     made in elucidating the molecular basis of this form of <br /> ment to microtubules nucleated from opposing spindle                     chromosome metabolism. In budding yeast, genetic and <br /> poles (Rieder and Salmon 1994; Hyman and Sorger 1995).                   biochemical analyses have identified a complex that <br /> On attachment of kinetochores to spindle microtubules,                   contains two regulators of sister chromatid cohesion, <br /> each sister chromatid experiences microtubule-depen-                     Pds1p and Esp1p (Cohen-Fix et al. 1996; Yamamoto et al. <br /> dent poleward forces. These poleward forces are resisted                 1996 a,b; Ciosk et al. 1998). Pds1p is an inhibitor of ana- <br /> by cohesion so that sister chromatids do not separate but                phase onset and is degraded at the metaphase-to-ana- <br /> instead come under tension. Observations made in mei-                    phase transition by a ubiquitin-dependent process in- <br /> otic cells have led to the proposal that tension is moni-                volving the anaphase promoting complex (APC). Pds1p <br /> tored by the cell to ensure that each sister chromatid is                proteolysis is thought to free Esp1p, which in turn pro- <br /> properly associated with microtubules. Only after all of                 motes the inactivation of chromatid cohesion. The <br /> the chromosomes are placed under tension does the cell                   Pds1p&#226;&#8364;&#8220;Esp1p complex is functionally related to the fis- <br /> signal for the onset of anaphase (Nicklas 1997). Thus,                   sion yeast Cut1p&#226;&#8364;&#8220;Cut2p complex and potential mamma- <br /> chromatid cohesion may also be a critical link in the                    lian homologs of Esp1p have been identified (Funabiki et <br /> mechanism that regulates anaphase onset. At anaphase,                    al. 1996). In addition to these regulatory proteins, several <br />                                                                          candidate cohesion proteins have been identified: <br /> 4 <br />  Present address: Centre for Molecular Medicine and Therapeutics, Uni-   Mcd1p/Scc1p, Scc2p, Smc1p, and Smc3p from budding <br /> versity of British Columbia, Vancouver, British Columbia V5Z 4H4,        yeast (Guacci et al. 1997; Michaelis et al. 1997) and <br /> Canada <br /> 5 <br />  Corresponding author.                                                   Mis4p from fission yeast (Furuya et al. 1998). Smc1p and <br /> E-MAIL skibbens@mail1.ciwemb.edu; FAX (410) 243-6311                     Smc3p are members of the recently discovered gene fam- <br />  <br />  <br />       GENES &amp; DEVELOPMENT 13:307&#226;&#8364;&#8220;319 &#194;&#169; 1999 by Cold Spring Harbor Laboratory Press ISSN 0890-9369/99 $5.00; www.genesdev.org      307 <br />  Skibbens et al. <br />  <br />  <br />  <br /> ily required for the structural maintenance of chromo-       fective in cohesion at both the centromere and arms <br /> somes (SMC) and are related by protein motifs first iden-    (Guacci et al. 1997; Michaelis et al. 1997). These results <br /> tified in Smc1p (Strunnikov et al. 1993). Inactivation of    suggest that although a common cohesion mechanism <br /> Mcd1p, Scc2p, Smc1p, Smc3p, or Mis4p causes preco-           exists for the entire chromosome, cohesion along the <br /> cious sister separation in M phase. Consistent with          chromosome can be differentially regulated. Finally, <br /> Mcd1p&#226;&#8364;&#8482;s role in the maintenance of cohesion, Mcd1p           Losada et al. (1998) recently reported that the Mcd1&#226;&#8364;&#8220; <br /> associates with chromosomes from S phase through M           Smc1&#226;&#8364;&#8220;Smc3 protein complex in Xenopus dissociates <br /> phase. Mcd1p both exists in a complex with Smc1p and         from chromatin early in mitosis, suggesting that the pro- <br /> requires Smc1p to associate with chromatin (Guacci et        cess of cohesion may be more mechanistically compli- <br /> al. 1997; Michaelis et al. 1997). Xenopus homologs of        cated than anticipated and that other factors involved in <br /> Mcd1p, Smc1p, and Smc3p, along with two other pro-           cohesion have yet to be identified. <br /> teins, form a cohesin complex that is required in inter-        Previously, a collection of 136 chromosome transmis- <br /> phase to maintain cohesion in the subsequent mitosis         sion fidelity (ctf) mutants was identified in Saccharomy- <br /> (Losada et al. 1998). These similarities between yeast and   ces cerevisiae on the basis of the increased rate of loss of <br /> vertebrate proteins suggest that the mechanisms of chro-     a nonessential chromosome fragment (Spencer et al. <br /> matid cohesion is likely to be conserved in all eukayotes.   1990). ctf mutants that allowed both transcription to <br />    In budding yeast, the cohesion factor Mcd1p is also       proceed through a centromere and the stable propagation <br /> required for chromosome condensation (Guacci et al.          of a dicentric chromosome (in vivo assays for kineto- <br /> 1997), revealing an important, but poorly understood,        chore integrity) proved to contain mutations in genes <br /> link between cohesion and condensation. Chromosome           subsequently shown to encode kinetochore components <br /> condensation ensures both disentanglement of sisters         that bind the yeast centromere (Lechner and Carbon <br /> and compaction of the chromosome arms away from the          1991; Doheny et al. 1993; Goh and Kilmartin 1993; Jiang <br /> cytokinetic furrow (Koshland and Strunnikov 1996). In-       et al. 1993). However, the vast majority of ctf mutants do <br /> terestingly, two other SMC family members, Smc2p and         not appear to correspond to integral kinetochore compo- <br /> Smc4p, are required for condensation and appear con-         nents and therefore constitute a powerful resource to <br /> served across evolution (Chuang et al. 1994; Hirano and      identify proteins involved in novel processes crucial for <br /> Mitchison 1994; Saitoh et al. 1994; Saka et al. 1994;        proper chromosome transmission. In this paper, we re- <br /> Strunnikov et al. 1995; Jessberger et al. 1996; Sutani and   port that CTF7 encodes an essential nuclear protein that <br /> Yanagida 1997). In Xenopus, Smc2p (XCAP-E) and               is required for cohesion. The analysis of CTF7 also es- <br /> Smc4p (XCAP-C) homologs combine with three addi-             tablishes a novel connection between cohesion and <br /> tional proteins to form a condensin complex that asso-       DNA replication. <br /> ciates with chromosomes (Hirano et al. 1997). Biochemi- <br /> cal analysis of Xenopus extracts have revealed that co- <br /> hesin and condensin complexes function independent of        Results <br /> one another, indicating that the processes of cohesion <br />                                                              Ctf7p is an essential nuclear protein and contains <br /> and condensation are uncoupled in this cell system <br />                                                              domains that are highly conserved <br /> (Losada et al. 1998). <br />    Although recent advances have been made in the iden-      In this study, we characterized ctf7 mutants, one <br /> tification of proteins required for cohesion and conden-     complementation group of the ctf collection that exhibit <br /> sation, little is known about how cohesion proteins are      elevated loss of a nonessential chromosome fragment <br /> assembled onto chromatin and, once assembled, how            (Spencer et al. 1990). The CTF7 gene was cloned on the <br /> those complexes are regulated. For instance, temporal        basis of complementation of the chromosome loss phe- <br /> analysis of cohesion in human cells and genetic analysis     notype and confirmed by integrative mapping (Materials <br /> of cohesion in yeast have shown that cohesion is estab-      and Methods). CTF7 was shown to encode an essential <br /> lished during S phase (Selig et al. 1992; Guacci et al.      gene by both tetrad analyses of a heterozygous CTF7/ <br /> 1997; Michaelis et al. 1997; Furuya et al. 1998; Uhlmann     ctf7&#226;&#338;&#172; strain and plasmid shuffle methods (see Materials <br /> and Nasmyth 1998). Yet, specialized factors that as-         and Methods). CTF7 is identical to the unannotated ORF <br /> semble cohesion complexes or load those complexes            YFR027W on chromosome VI (Murakami 1995) and en- <br /> onto chromatin remain to be discovered. Another funda-       codes a 281 amino acid protein with a predicted molecu- <br /> mental issue pertains to the regulation of chromatid co-     lar mass of 31.8 kD (Fig. 1A). A search of the expressed <br /> hesion. In early mitotic vertebrate cells, sister chroma-    sequence tag database (dBEST) revealed that portions of <br /> tids are paired along their entire <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>. In metaphase-     Ctf7p contained strong homologies to several human <br /> arrested cells, however, the chromosome arms are no          and mouse EST sequences (Fig. 1B). Similar homology <br /> longer paired, whereas the centromeres can remain            was found for an ORF of unknown function in fission <br /> paired for an extended period of time (Sumner 1991). A       yeast. These results show that Ctf7p, an essential yeast <br /> similar spatial and temporal regulation is observed in       protein, contains a motif that is conserved across king- <br /> meiotic cells: Cohesion is lost from the chromosome          doms and suggests that Ctf7p-related proteins may play <br /> arms during meiosis I to resolve recombinant interme-        a role important for chromosome transmission in all eu- <br /> diates, but cohesion is retained at the centromeres until    karyotic cells. Overexpressed HA-tagged Ctf7p localized <br /> meiosis II. In budding yeast, mcd1 mutant cells are de-      to the nucleus (HA-tagged Ctf7p under the control of the <br />  <br />  <br /> 308       GENES &amp; DEVELOPMENT <br />                                                                                                    Chromatid cohesion and replication <br />  <br />  <br />  <br />  <br />                                                      Figure 1. (A) Nucleotide and amino acid sequences of CTF7. The MluI cell <br />                                                      cycle box in the 5&#226;&#172;&#732; untranslated region, which directs the expression of <br />                                                      many genes required during S phase, is boxed (Johnston and Lowndes 1992; <br />                                                      Toyn et al. 1995). (B) Ctf7p homology to fission yeast, mouse and human <br />                                                      ESTs are shown. Identical residues are indicated by placement of the amino <br />                                                      acid above the sequence, conserved residues are indicated with a plus. The <br />                                                      carboxy-terminal region of Ctf7p (211&#226;&#8364;&#8220;267 amino acids) is 49%, 40%, and <br />                                                      41% identical and 70%, 63%, and 60% similar to fission yeast, mouse, and <br />                                                      human ESTs, respectively. <br />  <br />  <br />  <br /> endogenous CTF7 promoter was undetectable; Materials <br /> and Methods), consistent with Ctf7p&#226;&#8364;&#8482;s essential role in <br /> chromosome transmission. <br />  <br />  <br /> ctf7 mutants delay in mitosis with a 2C DNA content, <br /> separated DNA masses, and partially <br /> elongated spindles <br /> To investigate the function of Ctf7p in vivo, three tem- <br /> perature-sensitive alleles of CTF7 were obtained by a <br /> PCR-based mutagenesis strategy and integrated into the <br /> genome (see Materials and Methods). At the permissive <br /> temperature, all three temperature-sensitive alleles, <br /> ctf7-201, ctf7-202, and ctf7-203, exhibited increased <br /> chromosome fragment loss (Fig. 2A), recapitulating the <br /> original Ctf <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>phenotype</span>. At the restrictive temperature, <br /> ctf7 mutant strains rapidly lost viability compared with <br /> wild type, indicating that all three alleles caused an ir- <br /> reversible defect (Fig. 2B). <br />   To ascertain the terminal phenotype of the ctf7 mu- <br /> tant strains, log phase cultures of each mutant were <br /> shifted from 25&#194;&#176;C to 37&#194;&#176;C and processed to assess DNA <br /> content and cell morphology. At 25&#194;&#176;C, both wild-type <br /> and mutant strains exhibited similar DNA contents, as           Figure 2. Phenotypes of ctf7 conditional mutants. (A) ctf7 cells <br /> expected for cycling cells. However, after 3 hr growth at       (ctf7-201 shown) exhibit elevated chromosome loss over wild <br /> 37&#194;&#176;C, a significant increase in the number of cells with a      type. Faithful maintenance of a reporter chromosome fragment <br /> 2C DNA content was observed in ctf7 mutant strains              yields white colonies. Loss of the reporter chromosome results <br />                                                                 in red sectored colonies. (B) ctf7 mutants are temperature-sen- <br /> compared with CTF7 control cells (Fig. 2C). Analyses of <br />                                                                 sitive lethal. Log-phase cultures were shifted from 25&#194;&#176;C to 37&#194;&#176;C <br /> cell morphologies from this time point showed that              and assayed for cell viability. (&#228;&#352;&#8240;) ctf7-201; (&#228;&#352;&#65533;) ctf7-202; (&#228;&#8240;&#177;) ctf7- <br /> &#226;&#710;&#188;51% of ctf7 mutant cells were large budded and of              203; (*) wild type. (C) ctf7 mutants delay in mitosis. The DNA <br /> these, 63% contained separating or bi-lobed DNA                 content of log phase cultures grown at 25&#194;&#176;C or 37&#194;&#176;C for 3 hr <br /> masses (Fig. 2D; Table 1A). The bilobed DNA masses              were assessed by flow cytometry (ctf7-203 shown). (D) Micro- <br /> often remained entirely within the mother cell and oc-          graphs of ctf7 mutant cell (Phase), DNA (DAPI), and microtu- <br /> casionally contained unequal amounts of DNA. Spindles           bule (Tubulin) morphologies after 3 hr growth at 37&#194;&#176;C. <br />  <br />  <br />                                                                                                 GENES &amp; DEVELOPMENT              309 <br />  Skibbens et al. <br />  <br />  <br />  <br /> Table 1.    ctf7 morphologies at 37&#194;&#176;C                             mad2-1 and ctf7-203 bub3&#226;&#338;&#172; strains exhibited a complete <br />                                                                   loss of the 2C DNA accumulation at the restrictive tem- <br />                                                                   perature (Fig. 3; data not shown). Consistent with this <br />                                                                   result, the ctf7-203 mad2-1 double mutant strain also <br />                                                                   failed to accumulate partially elongated spindles or bi- <br /> A <br />                                                                   lobed DNA masses, but instead proceeded through mi- <br /> CTF7          63.2   13.5    6.2        11.9    3.1   0     2.1 <br /> ctf7-203      25.1    9.1    9.7         8.6   31.9   5.7   9.9   tosis as evidenced by the reappearance of G1-like (unbud- <br /> B                                                                 ded) cells (Table 1B). These results show that the mitotic <br /> CTF7          69.4   12.2    5.2         6.1    4.5   0     2.6   delay in ctf7 cells requires a functional kinetochore/ <br /> ctf7-203      19.1   15.3    7.4         0     56.3   1.9   0     spindle assembly checkpoint but not a functional DNA <br /> ctf7-203                                                          damage checkpoint. <br />   mad2-1      59.2    8.2    3.1         9.3   15     3.1   2.1 <br />  <br /> (A) ctf7 mutant cells accumulate as large budded cells contain- <br /> ing a bilobed/elongated nucleus. Cell and nuclear morphologies    ctf7 cells delay in metaphase prior to anaphase onset <br /> of log-phase CTF7 and cft7-203 strains were assayed after 150 <br />                                                                   Typically, an activated kinetochore/spindle assembly <br /> min growth at 37&#194;&#176;C. (B) The bilobed nuclear morphology of ctf7 <br /> mutants depends on the MAD2 checkpoint. Log-phase cultures <br />                                                                   checkpoint causes cells to arrest prior to anaphase onset <br /> of CTF7, ctf7-203, and ctf7-203/mad2-1 were synchronized in       and those cells contain short spindles and a single DNA <br /> G1 used &#226;&#65533;&#163;-factor, washed, and grown at 37&#194;&#176;C for 3 hr before       mass. However, the MAD/BUB-dependent mitotic delay <br /> analysis.                                                         in ctf7 mutant cells lead to an accumulation of cells that <br />                                                                   contained partially elongated spindles and separated <br />                                                                   DNA masses. This unusual phenotype could represent <br /> from these cells were also atypical in that they were             an aberrant metaphase. An alternative possibility was <br /> entirely contained within the mother cell (Fig. 2D) and           that ctf7 cells were delayed in midanaphase, suggesting a <br /> appeared longer than preanaphase spindles. To quantify            novel postmetaphase role for Mad/Bub checkpoint pro- <br /> this last observation, log phase cultures of CTF7, ctf7-          teins. To determine whether the mitotic delay occurred <br /> 203, and ctf13-30 (a kinetochore mutant that arrests              prior to anaphase onset, we assessed the levels of Pds1p <br /> prior to anaphase onset) were shifted to 35&#194;&#176;C for 3 hr,           in wild-type and ctf7 mutant cells. Pds1p is an inhibitor <br /> fixed, and immunostained for tubulin to visualize the             of anaphase onset and is degraded at the metaphase-to- <br /> spindle. Measurements of spindle lengths revealed that            anaphase transition (Cohen-Fix et al. 1996; Yamamoto et <br /> ctf7 mutant cells with bilobed DNA masses contained               al. 1996a,b). <br /> partially elongated spindles of 3.7 &#194;&#181;m (&#194;&#177;0.4, n = 7) in              Wild-type and ctf7 mutant cells were transformed <br /> length; &#226;&#710;&#188;2.5 times longer than the 1.4 &#194;&#181;m (&#194;&#177;0.3, n = 8)           with a plasmid directing the expression of HA&#226;&#8364;&#8220;Pds1p <br /> preanaphase spindles in ctf13-30 mutants but roughly              (Cohen-Fix et al. 1996). The level of HA&#226;&#8364;&#8220;Pds1p was ana- <br /> half as long as the 6.9 &#194;&#181;m (&#194;&#177;0.9, n = 11) elongated               lyzed after synchronized G1 cells were released into rich <br /> spindles in anaphase CTF7 wild-type cells. The increased          medium and grown for 3 hr at 37&#194;&#176;C. Western blot analy- <br /> chromosome loss, large-budded cell morphology, 2C                 sis revealed that wild-type cells had low levels of HA&#226;&#8364;&#8220; <br /> DNA content, abnormal spindle morphology, and un-                 Pds1p because most cells had already completed mitosis <br /> equal chromatin distribution observed in ctf7 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>mutation</span> <br /> strains indicate that loss of Ctf7p function leads to a <br /> defect in mitosis and a mitotic delay. <br />  <br />  <br /> The mitotic delay in ctf7 cells requires <br /> the kinetochore/spindle assembly checkpoint <br /> Mitotic delays typically arise by activation of either the <br /> DNA damage or kinetochore/spindle assembly check- <br /> point pathways. To determine whether ctf7 cells re- <br /> quired either of these checkpoint mechanisms to delay <br /> in mitosis, we constructed strains containing ctf7 in <br /> combination with either mutations in the DNA damage <br /> checkpoint pathway (rad9&#226;&#338;&#172; and mec1-1) (Weinert and <br /> Hartwell 1988, 1990; Weinert et al. 1994) or mutations in <br /> the kinetochore/spindle assembly checkpoint pathway <br /> (mad2-1 and bub3&#226;&#338;&#172;) (Hoyt et al. 1991; Li and Murray <br /> 1991). Log phase cultures of the resulting strains were <br /> shifted to 37&#194;&#176;C for 3 hr and their DNA content measured           Figure 3. The mitotic delay in ctf7 cells requires the kineto- <br /> by flow cytometry. Neither rad9&#226;&#338;&#172; nor mec1-1 mutations             chore/spindle checkpoint pathway. Flow cytometry analysis of <br /> abrogated the ctf7-203 mediated 2C DNA accumulation               DNA content of log phase ctf7, checkpoint mutant, and double- <br /> (Fig. 3, data not shown). In contrast, both ctf7-203              mutant strains grown for 3 hr at 25&#194;&#176;C (left) or 37&#194;&#176;C (right). <br />  <br />  <br /> 310        GENES &amp; DEVELOPMENT <br />                                                                                                  Chromatid cohesion and replication <br />  <br />  <br />  <br />                                                                  FISH to ask if sister chromatid cohesion was normal in <br />                                                                  ctf7 mutant cells. In wild-type cells arrested in mitosis, <br />                                                                  only &#226;&#710;&#188;10% of the DNA masses contained two FISH sig- <br />                                                                  nals when using a CEN16 proximal DNA probe (Fig. <br />                                                                  5A,B). In contrast, &#226;&#710;&#188;40% of the DNA masses from ctf7 <br />                                                                  cells contained two FISH signals, often with sister cen- <br />                                                                  tromeres residing at opposite sides of the nuclear masses <br />                                                                  (Fig. 5A,B). The ctf7 defect affected other chromosomes, <br />                                                                  &#226;&#710;&#188;40% of DNA masses in ctf7 cells contained two FISH <br />                                                                  signals with a CEN1 proximal probe (Fig. 5B). Finally, a <br />                                                                  similar loss of cohesion was obtained by use of centro- <br />                                                                  mere-distal (arm) probes to chromosome XVI, indicating <br />                                                                  that Ctf7p was required for cohesion along the entire <br />                                                                  length of the chromosome (Fig. 5B). These findings re- <br />                                                                  veal that Ctf7p is essential for sister chromosome cohe- <br />                                                                  sion and supports previous findings that cohesion is me- <br /> Figure 4. ctf7 cells are delayed prior to anaphase onset. (A). <br /> The Western blot shows HA&#226;&#8364;&#8220;Pds1p levels in wild-type and ctf7     diated by a mechanism common throughout the chro- <br /> cells that were synchronized in G1 and then released into me-    mosome (Guacci et al. 1997). <br /> dium and grown at 37&#194;&#176;C for 3 hr. Tubulin levels verified that <br /> equal protein loads were present in each lane. (B) Micrographs <br /> show the localization of DNA (DAPI) and HA&#226;&#8364;&#8220;Pds1p (HA&#226;&#8364;&#8220; <br /> Pds1p) in ctf7 mutant and wild-type cells treated as in A. <br />  <br />  <br />  <br /> (Fig. 4A). In contrast, ctf7 cells contained high HA&#226;&#8364;&#8220;Pds1p <br /> levels, indicating that ctf7 cells were delayed prior to <br /> anaphase onset. To confirm that ctf7 cells were delayed <br /> in metaphase, HA&#226;&#8364;&#8220;Pds1p was visualized by immunoflu- <br /> orescence in wild-type and ctf7 cells treated as above. As <br /> expected, wild-type cells with separating DNA masses <br /> did not contain HA&#226;&#8364;&#8220;Pds1p (Cohen-Fix et al. 1996). In <br /> contrast to wild-type cells, a significant population of <br /> ctf7 cells with separating DNA masses contained high <br /> levels of HA&#226;&#8364;&#8220;Pds1p (Fig. 4B). Thus, these ctf7 cells were <br /> delayed prior to anaphase onset. It was not surprising <br /> that some ctf7 cells with separating DNA masses did not <br /> contain detectable HA&#226;&#8364;&#8220;Pds1p levels, because ctf7 cells <br /> are only delayed in the cell cycle and eventually exit <br /> mitosis. These results reveal that partial spindle elonga- <br /> tion and DNA separation occurs prior to anaphase onset <br /> in ctf7 mutant cells. <br />  <br />  <br /> Ctf7p is required for sister chromatid cohesion <br /> How can cells delayed prior to anaphase onset have sepa- <br /> rated DNA masses? One possibility was that defects in <br /> sister chromosome cohesion allowed sister chromatids <br /> to separate prematurely, resulting in the movement of            Figure 5. Ctf7p is required for sister chromatid cohesion. (A) <br /> disjoined sisters toward the spindle poles. To test this         Micrographs of wild-type and ctf7 mutant cells grown for 3 hr at <br /> model, wild-type and ctf7-203 cells were synchronized in         37&#194;&#176;C in medium containing nocodazole and processed for FISH. <br /> G1 with &#226;&#65533;&#163;-factor, released into medium that contained            A single FISH signal represents paired sister chromatids, two <br /> nocodazole, and grown for 3 hr at 37&#194;&#176;C. Nocodazole               FISH signals represents disjoined sister chromatids. (red) Prop- <br /> blocks both microtubule assembly and anaphase onset              idium iodide staining DNA mass; (green) hybridization to <br />                                                                  CEN16 probe. (B) Quantification of single (nondisjoined chro- <br /> such that sister chromatids typically remain closely ap- <br />                                                                  mosomes) vs. double FISH signals (disjoined chromosomes) <br /> posed for extended periods of time. FISH has been used           within propidium iodide staining DNA masses. Synchronized <br /> previously to demonstrate the pairing of sister chroma-          G1 wild-type and ctf7 cells contained an equal percentage of <br /> tids in M-phase yeast cells and can further assess the           single FISH signals (data not shown), indicating that the in- <br /> pairing at different loci such as the centromere and chro-       crease in two FISH signals observed for ctf7 mutant cells was <br /> mosome arms (Guacci et al. 1994). Therefore, we used             not due to pre-existing aneuploid. <br />  <br />  <br />                                                                                                GENES &amp; DEVELOPMENT            311 <br />  Skibbens et al. <br />  <br />  <br />  <br /> Ctf7p is required during S phase                                     &#226;&#65533;&#163;-factor release but prior to DNA replication) (Fig. 6A,B). <br />                                                                      Cultures exhibited high viability when shifted to 37&#194;&#176;C <br /> Sister chromatid cohesion is apparently established dur-             after 80 min of growth at 23&#194;&#176;C (Fig. 6A). The increase in <br /> ing S phase and maintained until anaphase onset. To                  viability correlated with progression of DNA replication: <br /> assess the role of Ctf7p in the establishment and main-              Maximum viability was realized only after the bulk of <br /> tenance of cohesion, we used reciprocal temperature-                 DNA replication was completed (80 min) (Fig. 6B). Fur- <br /> shift experiments in ctf7 cells to map that portion of the           thermore, maximal cell viability preceded anaphase on- <br /> cell cycle that requires Ctf7p function (Fig. 6). A shift-up         set (spindle elongation) by &gt;20 min (Fig. 6A). These re- <br /> experiment marks the earliest point in the cell cycle                sults suggest that Ctf7p is required during S phase but <br /> when Ctf7p can be inactivated without reducing cell vi-              not during mitosis. <br /> ability (i.e., when Ctf7p function is no longer essential).            A shift-down experiment marks the latest point in the <br /> ctf7-203 cells were synchronized in G1 with &#226;&#65533;&#163;-factor and             cell cycle when Ctf7p must be active to produce viable <br /> then released into medium at 23&#194;&#176;C to allow the cells to              cells. ctf7-203 cultures were synchronized in G1 with <br /> proceed through the cell cycle. Starting at the release              &#226;&#65533;&#163;-factor at 23&#194;&#176;C, then released into 37&#194;&#176;C rich medium in <br /> from &#226;&#65533;&#163;-factor (t = 0), aliquots were shifted to 37&#194;&#176;C at 20           the absence of &#226;&#65533;&#163;-factor to allows cells to proceed through <br /> min intervals. At the time of the shift, aliquots were also          the cell cycle. Aliquots were then shifted down to 23&#194;&#176;C <br /> harvested for flow cytometry and immunofluorescence                  at 20 min intervals and also harvested for flow cytometry <br /> analyses to position cells within the cell cycle (see Ma-            and immunofluoresence analyses as described above. On <br /> terials and Methods). At the end of the experiment, all              release from &#226;&#65533;&#163;-factor (t = 0), cells retained high viability <br /> samples were plated at 23&#194;&#176;C to assess viability (Materials           for 80 min of growth at 37&#194;&#176;C prior to shifting down to <br /> and Methods). Cultures exhibited low viability when                  23&#194;&#176;C (Fig. 6C). Decreased cell viability (starting at t = 80) <br /> shifted to 37&#194;&#176;C any time during the first 60 min (after              correlated with entry into DNA replication (Fig. 6C,D) <br />  <br />  <br />  <br />  <br /> Figure 6. Ctf7p is required only during S phase. (A,B) Shift-up experiment: G1 ctf7-203 cells were released from &#226;&#65533;&#163;-factor and shifted <br /> from 23&#194;&#176;C to 37&#194;&#176;C at 20&#226;&#8364;&#8220;30 min increments. (C,D) Shift-down experiment: G1 ctf7-203 cells were released from &#226;&#65533;&#163;-factor into 37&#194;&#176;C <br /> medium and shifted down to 25&#194;&#176;C at 20&#226;&#8364;&#8220;30 min increments. Wild-type strains retain high viability throughout either procedures (data <br /> not shown). For each time point, an aliquot was assayed for spindle elongation/DNA mass separation (&#226;&#189;&#167;), viability (&#228;&#352;&#65533;), and DNA <br /> content (flow cytometry profile). (E, top) Viability of G1 synchronized wild-type, ctf7-203, and mcd1-1 cells released into medium <br /> containing nocodazole and permitted to arrest in mitosis at 23&#194;&#176;C prior to shifting to 37&#194;&#176;C. (E, bottom) Viability of G1 synchronized <br /> wild-type, ctf7-203, and mcd1-1 cells released into medium containing nocodazole at 37&#194;&#176;C. A 10-fold dilution for each culture is <br /> shown. <br />  <br />  <br /> 312       GENES &amp; DEVELOPMENT <br />                                                                                                  Chromatid cohesion and replication <br />  <br />  <br />  <br /> and a further decline in cell viability coincided with pro-    sister centromeres, Ctf7p is not required for the mainte- <br /> gression through S phase. Thus, Ctf7p must be active as        nance of sister chromatid cohesion. <br /> cells enter S phase to obtain viable cells. Activation later <br /> in the cell cycle resulted in cell death. A further finding <br /> from this experiment was that ctf7 mutants progressed          Genetic interactions between CTF7 and the DNA <br /> through S phase with kinetics similar to wild-type cells       replication machinery <br /> (data not shown). Thus, the lesion incurred during S <br />                                                                Typically, high copy suppressors are proteins that either <br /> phase in ctf7 mutant cells is not manifested as a defect in <br />                                                                interact with the target protein or function in a parallel <br /> DNA replication. The combination of results from the <br />                                                                pathway. To gain further insight into the role of Ctf7p in <br /> shift-up and shift-down experiments in ctf7 cells suggest <br />                                                                cohesion, high-copy suppressors of the temperature sen- <br /> that Ctf7p function is essential during S phase when co- <br />                                                                sitive ctf7 phenotype were identified. Three plasmids <br /> hesion is established but not in mitosis when cohesion is <br />                                                                were obtained that enabled ctf7 mutant cells to grow at <br /> maintained. <br />                                                                34&#194;&#176;C but not at 37&#194;&#176;C. The three plasmids contained <br />    We decided to use a second method to test whether <br />                                                                overlapping DNA sequences, suggesting suppression was <br /> Ctf7p is required during M phase and S phase. Mcd1p <br />                                                                due to a single gene. ctf7 temperature sensitivity was <br /> appears to be required during both M phase and S phase <br />                                                                suppressed by a minimal DNA sequence defined by BglII <br /> and was included as an indicator for loss of cohesion <br />                                                                to StuI restriction sites (Fig. 7A). Sequence analysis re- <br /> (Guacci et al. 1997). To test whether Ctf7p was required <br />                                                                vealed that this 1.7-kb fragment contained POL30, <br /> during mitosis, wild-type, ctf7-203, and mcd1-1 mutant <br />                                                                which encodes the DNA replication processivity factor <br /> cells synchronized in G1 were released into medium con- <br />                                                                proliferating cell nuclear antigen (PCNA) (Bauer and <br /> taining nocodazole and allowed to proceed from G1 into <br />                                                                Burgers 1990; Burgers 1991). A frame-shift mutation in <br /> mitosis at the permissive temperature of 23&#194;&#176;C. Aliquots <br />                                                                the POL30 ORF rendered the resulting plasmid inca- <br /> of the cultures were then shifted to 37&#194;&#176;C at 1 hr inter- <br />                                                                pable of rescuing ctf7-203 temperature sensitivity, indi- <br /> vals. After incubation at 37&#194;&#176;C, the cultures were plated <br />                                                                cating that POL30 is a high-copy suppressor of ctf7-203. <br /> and allowed to grow for 2 days at 23&#194;&#176;C to assess viability <br />                                                                Overexpression of POL30 also rescued the temperature <br /> (Fig. 6E). Wild-type cells were not adversely affected by <br /> this procedure and maintained high viability. As ex- <br /> pected, mcd1-1 cells exhibited a dramatic decrease in <br /> viability within the first hour of incubation at 37&#194;&#176;C. In <br /> contrast, ctf7-203 cells retained high viability even after <br /> 3 hr incubation at 37&#194;&#176;C. Thus, Ctf7p is not required dur- <br /> ing mitosis when sister chromatid cohesion is main- <br /> tained. However, Ctf7p is required during S phase when <br /> cohesion is established. ctf7-203 and mcd1-1 mutant <br /> cells both exhibited decreased viability when synchro- <br /> nized cells were allowed to proceed from G1 in to mitosis <br /> at 37&#194;&#176;C. The combination of these results suggest that <br /> Ctf7p plays a role in cohesion distinct from Mcd1p. <br />    The high viability of ctf7-203 cells allowed to proceed <br /> into mitosis before shifting them to the restrictive tem- <br /> perature implied that Ctf7p was not required for the <br /> maintenance of cohesion. To demonstrate this unam- <br /> biguously, we performed FISH analysis on wild-type and <br /> ctf7-203 cells. As before, G1 synchronized cells were re- <br /> leased into medium containing nocodazole and allowed <br /> to proceed into mitosis at the permissive temperature. <br /> After 3 hr at 23&#194;&#176;C, the cultures were shifted to 37&#194;&#176;C and <br /> held at this restrictive temperature for 2 hr. Aliquots of     Figure 7. POL30 is an overexpression suppressor of ctf7 mu- <br /> the cultures were then processed for FISH analysis by use      tant phenotypes. (A) A 7.1-kb insert was identified that sup- <br /> of both CEN1 and CEN16 proximal probes. Wild-type              pressed ctf7 temperature sensitivity at 34&#194;&#176;C. Neither vector <br /> cells retained sister chromatid cohesion following this        alone (Mock) nor vector containing insert DNA sequences to <br /> procedure, as only 22% (CEN1) and 23% (CEN16) of the           either the left or right of the Sall digestion site rescued the ctf7 <br /> DNA masses contained two FISH signals. ctf7-203 cells          mutant at 34&#194;&#176;C. A 1.7-kb fragment flanked by Bglll and Stul <br /> also retained sister chromatid cohesion, 26% (CEN1) and        restriction sites rescued ctf7 mutant temperature sensitivity <br />                                                                but not to the same level of growth as CTF7 (Vector + CTF7). <br /> 27% (CEN16) of the DNA masses contained two FISH <br />                                                                The hatched box in starting clone denotes POL30 OFR. (B) ctf7 <br /> signals (data not shown). The slight increase in the num-      mutations containing reporter chromosomes were transformed <br /> ber of DNA masses that contained two FISH signals ob-          with 2 &#194;&#181;m vector &#194;&#177; POL30 and grown on medium that allowed <br /> served for both strains was most likely due to the pro-        for chromosome fragment loss but retention of the 2 &#194;&#181;m plas- <br /> longed mitotic arrest and exposure to nocodazole. Thus,        mid (ctf7-108 is a non-temperature-sensitive strain obtained <br /> by use of both criteria of viability and localization of       from the original ctf collection). <br />  <br />  <br />                                                                                               GENES &amp; DEVELOPMENT              313 <br />  Skibbens et al. <br />  <br />  <br />  <br /> sensitivity of ctf7-201 but failed to rescue a ctf7&#226;&#338;&#172;1 dele-           high-copy suppressor of ctf7 phenotypes. These genetic <br /> tion strain (not shown).                                              interactions link Ctf7p with proteins that are required <br />    On the basis of the finding that POL30 is a high-copy              during S phase, consistent with our mapping experi- <br /> suppressor of ctf7 temperature sensitivity, we asked                  ments, and together suggest that the establishment of <br /> whether POL30 could also rescue ctf7 chromosome loss.                 sister chromatid cohesion is facilitated by components <br /> To test this possibility, POL30 was overexpressed in                  of the DNA replication machinery. <br /> strains containing either ctf7 temperature-sensitive alle- <br /> les (ctf7-201 and ctf7-203) or a non-temperature-sensi- <br /> tive allele (ctf7-108) that was obtained in the original ctf          Discussion <br /> collection (Spencer et al. 1990). Identical results were <br />                                                                       Ctf7p, a novel link between sister chromatid cohesion, <br /> obtained in each case: Overexpression of POL30 de- <br />                                                                       S phase, and the DNA replication machinery <br /> creased the chromosome loss associated with all ctf7 <br /> mutations tested (Fig. 7B).                                           In this study, we have identified Ctf7p as a protein re- <br />    To both test whether the genetic interaction between               quired for sister chromatid cohesion. HA&#226;&#8364;&#8220;Ctf7p is a <br /> CTF7 and POL30 reflected an important link between                    nuclear protein and mutations in CTF7 result in elevated <br /> the establishment of sister chromatid cohesion and                    chromosome loss, consistent with Ctf7p&#226;&#8364;&#8482;s role in chro- <br /> DNA replication machinery, and to further identify gene               mosome segregation (this paper; Spencer et al. 1990). We <br /> products that might require CTF7 function, we assayed                 discovered further that Ctf7p is required for cohesion at <br /> for synthetic lethal interactions. A strain harboring the             both centromeres and chromosome arms, as found for <br /> ctf7-203 allele was mated to strains containing either                the cohesion factor Mcd1p (this paper; Guacci et al. <br /> checkpoint, kinetochore, DNA metabolism, or other                     1997; Michaelis et al. 1997). Independent analysis has <br /> chromosome loss mutants. The inability to recover                     also shown that Ctf7p (called Eco1p) is required for co- <br /> double-mutant spores was scored as a synthetic lethal                 hesion (A. Toth, R. Ciosk, F. Uhlmann, M. Galova, A. <br /> interaction (Table 2). Synthetic lethal interactions were             Schleiffer, K. Nasmyth, pers. comm.). Therefore, al- <br /> observed for ctf7-203 in combination with either ctf18&#226;&#338;&#172;1              though there is evidence from vertebrate cell chromo- <br /> or pol30-104. CTF18 encodes an RF-C-like protein and                  somes that cohesion is differentially regulated along the <br /> RF-C complexes are known to load PCNA onto double-                    length of the chromosome (Sumner 1991), Ctf7p and <br /> stranded DNA (Burgers 1988; Kouprina et al. 1994; Ay-                 Mcd1p appear to function in aspects of cohesion that are <br /> yagari et al. 1995; Amin and Holm 1996). In contrast to               common to all regions of the chromosome. <br /> these findings, no synthetic lethal interactions were ob-                Cell viability and sister chromatid cohesion are abol- <br /> served when ctf7-203 was combined with checkpoint                     ished when ctf7 cells are placed at the restrictive tem- <br /> mutants (rad9&#226;&#338;&#172;, mec1-1, mad2-1, or bub3&#226;&#338;&#172;), kineto-                    perature during S phase but not during mitosis. These <br /> chore mutants (ctf13-30, ctf14-42, skp1-4, or skp1-3), or             findings demonstrate that Ctf7p, unlike Mcd1p, func- <br /> other mutants involved in DNA replication (ctf4-25, an                tions specifically in the establishment of cohesion (this <br /> allele of the Pol&#226;&#65533;&#163;-binding protein, and cdc6/ctf10-103,               paper; Guacci et al. 1997; Michaelis et al. 1997). An in- <br /> an allele of the ORC interacting protein). Of 12 of the               dependent allele of ctf7 (eco1-1) is also defective in the <br /> mutant strains that we tested (Table 3), synthetic lethal-            establishment but not the maintenance of cohesion (A. <br /> ity was only observed for ctf7 in combination with pol30              Toth, R. Ciosk, F. Uhlmann, M. Galova, A. Schleiffer, K. <br /> or ctf18 mutations. The specificity of these genetic in-              Nasmyth, pers. comm.), indicating that the inability to <br /> teractions is substantiated by the related functions of               reveal a function for Ctf7p in the maintenance of cohe- <br /> PCNA and Ctf18p and by the finding that POl30 is a                    sion is not an allele-specific effect. The model in which <br />  <br />  <br /> Table 2.    ctf7-203 is synthetically lethal with cft18&#226;&#338;&#172;1 and pol30-104 <br />  <br />                                                                                           Observed          Expected <br />  <br /> pol30&#226;&#8364;&#8220;104   mms <br />                   &#226;&#8364;&#8220;LEU2 &#195;&#8212; ctf7&#226;&#338;&#172;&#226;&#172;&#8249;HIS3 ctf7-203 :LEU2 <br />                                              ts <br />                                                              61% viable <br />                                                                (68 0f 112 spores) <br />                                                                wild type                     32+               28 <br />                                                                pol30                         28                28 <br />                                                                ctf7                           8                14 <br />                                                                pol30 ctf7                     0                14 <br />   ctf18&#226;&#338;&#172;&#226;&#172;&#8249;URA3 &#195;&#8212; ctf7&#226;&#338;&#172;&#226;&#172;&#8249;HIS3 ctf7-203ts:LEU2                   55% viable <br />                                                                (57 of 104 spores) <br />                                                                wild type                     30                26 <br />                                                                ctf18&#226;&#338;&#172;                        13                26 <br />                                                                ctf7                          14                13 <br />                                                                ctf18&#226;&#338;&#172; ctf7                    0                13 <br />  <br /> ctf7-203 was mated with pol30-104 and ctf18&#226;&#338;&#172;1, and the resulting diploid dissected to identify spores that contained both mutations. <br /> The expected and observed number of wild-type, single and double mutant spores are shown for each cross. Total spore viability is also <br /> shown. <br />  <br />  <br /> 314        GENES &amp; DEVELOPMENT <br />                                                                                            Chromatid cohesion and replication <br />  <br />  <br /> Table 3.   Strains used in this study <br />  <br /> Strains                                          Genotypes                                             References <br />  <br /> YBS2     MATa/MAT&#226;&#65533;&#163; ura3-52/ura3-52 lys2-801/lys2-801 ade2-101/ade2-101 trpl-&#226;&#338;&#172;63/trp1-&#226;&#338;&#172;63      this study <br />           his3-&#226;&#338;&#172;200/his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;/leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3/CTF7 CFIII (CEN3.L) TRP1 SUP11 <br /> YBS3     MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 CTF7:HIS3 <br /> YBS5     MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 CTF7&#226;&#172;&#8249;HIS3 CFIII <br />           (CEN3.L) TRP1 SUP11 pRS316CTF7 <br /> YBS204   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br />           rad9&#226;&#338;&#172;1&#226;&#172;&#8249;TRP1 <br /> YBS205   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 <br />           ctf7-203:LEU2 rad9&#226;&#338;&#172;1&#226;&#172;&#8249;TRP1 <br /> YBS213   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1,63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br />           mad2-1 <br /> YBS214   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1,63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 <br />           ctf7-203:LEU2 mad2-1 <br /> YBS504   MATa ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br />           bub3&#226;&#338;&#172;1&#226;&#172;&#8249;LEU2 <br /> YBS568   MATa ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 ctf7-203:LEU2 <br />           bub3&#226;&#338;&#172;1&#226;&#172;&#8249;LEU2 <br /> YBS255   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br /> YBS514   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 <br />           ctf7-203:LEU2 <br /> YBS268   MAT&#226;&#65533;&#163; ura3 trp1 his3 leu2 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 ctf7-203:LEU2 mec1-1:HIS3 <br /> YBS271   MAT&#226;&#65533;&#163; ura3 trp1 his3 leu2 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7&#226;&#8364;&#8220;LEU2 mec1-1:HIS3 <br /> YCTF108 MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1 leu2-&#226;&#338;&#172;1 ctf7-108 CFIII (CEN3.L) URA3 SUP11     Spencer et al. (1990) <br /> YCTF103 MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1 leu2-&#226;&#338;&#172;1 ctf10-103 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YCTF42   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf14-42 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YCTF30   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf13-30 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YCTF25   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 cf4-25 CFIII (CEN3.L) URA3 SUP11 <br /> YCTF10   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7-10 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YPH1182  MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 skp1-4:LEU2                Connelly and Heiter (1996) <br /> YPH1188  MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 skp1-3:LEU2 <br /> YCH2165 MATa leu2-3,112 ura3-52 POL30:LEU2                                                    Amin and Holm (1996) <br /> YCH2161 MATa leu2-3,112 ura3-52 pol30&#226;&#8364;&#8220;104:LEU2 <br /> YJE106   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf18&#226;&#338;&#172;1:LEU2               Kouprina et al. (1994) <br /> YJE188   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1 ctf18&#226;&#338;&#172;1&#226;&#172;&#8249;URA3 <br /> DLY285   MATa ura3 trp1 his3 leu2 trp1 mec1-1&#226;&#8364;&#8220;HIS3                                            Paulovich (1995) <br /> YCB624   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 rad9&#226;&#338;&#172;1&#226;&#172;&#8249;TRP1                Brachmann (1995) <br /> YFS451   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1,63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 mad2-1 CFIII (CEN3.L)    Pangilinan (1996) <br />           URA3 SUP11 <br /> YFP74    MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 bub3&#226;&#338;&#172;1&#226;&#172;&#8249;LEU2 <br /> VG955-7D MATa mcd1-1 trp1 leu2 bar1 gal1                                                      Guacci et al. (1997) <br /> See Materials and Methods for details. <br />  <br />  <br /> Ctf7p functions in the establishment of cohesion during       between CTF7, POL30, and CTF18 do not appear to re- <br /> S phase is further supported by the genetic interactions      flect a general response of ctf7 mutant strains to alter- <br /> observed between mutations in CTF7 and the DNA me-            ations in S-phase progression. For instance, both ctf18&#226;&#338;&#172;1 <br /> tabolism genes POL30 and CTF18 (Kouprina et al. 1994;         and pol30-104 are cold-sensitive mutations that, at el- <br /> Amin et al. 1996). POL30 was the only gene found in a         evated temperatures, exhibit normal S-phase kinetics <br /> genome-wide screen that, at high copy, suppressed ctf7        (Kouprina et al. 1994; Amin and Holm 1996). In addition, <br /> temperature sensitivity and chromosome loss. In addi-         ctf7 mutant strain temperature sensitivity was not sup- <br /> tion, pol30-104 and ctf18&#226;&#338;&#172;1 are synthetically lethal with     pressed by levels of hydroxyurea that slow DNA elonga- <br /> ctf7-203. The significance of these genetic interactions is   tion (R.V. Skibbens and D. Koshland, unpubl.). Taken <br /> augmented by their specificity; no synthetic lethal inter-    together, these observations suggest that Ctf7p functions <br /> actions were observed for ctf7 in combination with mu-        during S phase for the establishment of cohesion and <br /> tations that affect either the initiation of DNA replica-     reveals an important link between DNA replication and <br /> tion, kinetochore function, or DNA damage or spindle          sister chromatid cohesion. <br /> assembly checkpoint pathways. The genetic interactions           The connection between CTF7 and POL30 is particu- <br />  <br />  <br />                                                                                         GENES &amp; DEVELOPMENT             315 <br />  Skibbens et al. <br />  <br />  <br />  <br /> larly intriguing. PCNA forms doughnut-like sliding            structure? In preanaphase budding yeast cells, spindle <br /> clamps that are continuously loaded behind DNA poly-          pole bodies are mechanically linked by a few interdigi- <br /> merases at the replication fork (Krishna et al. 1994). Oth-   tating polar microtubules and by many kinetochore mi- <br /> ers have noted that PCNA both appears to be landing           crotubules that are indirectly connected by virtue of <br /> pads for a variety of proteins not directly related to DNA    their attachment to paired sister chromatids (O&#226;&#8364;&#8482;Toole et <br /> replication and may play a role in chromatin remodeling       al. 1997). We propose that the dissolution of sister chro- <br /> (Kelman 1997). In combination with our results, these         matid cohesion complexes weakens this linkage and pro- <br /> data suggest that the loading of cohesin complexes onto       foundly affects spindle integrity. Abnormal spindles <br /> chromatin is coupled with PCNA-dependent DNA rep-             have been reported for other mutants defective in cohe- <br /> lication. This model is attractive because it places cohe-    sion such as smc1 and mcd1 (Strunnikov et al. 1993; <br /> sin complexes proximal to newly replicated sister chro-       Guacci et al. 1997; Michaelis et al. 1997). Thus, sister <br /> matids to ensure the pairing of sister chromatids but         chromatid cohesion appears to function as part of the <br /> preclude the pairing of nonsister chromatids. Loading of      mechanical linkage that is required to maintain spindle <br /> cohesin complexes onto chromatin can be uncoupled             integrity. <br /> from replication, as has been observed in Xenopus ex-            The second question raised by our observations is why <br /> tracts and budding yeast cells, but at least in yeast this    do ctf7 mutants activate the kinetochore/spindle assem- <br /> association fails to generate functional sister chromatid     bly checkpoint and delay in mitosis? To date, the kineto- <br /> cohesion (Losada et al. 1998; Uhlmann and Nasmyth             chore/spindle damage checkpoint pathway has been im- <br /> 1998).                                                        plicated in monitoring defects in centromere, kineto- <br />    In budding yeast, Mcd1p is required for both cohesion      chore, or spindle function and responds by delaying cells <br /> and condensation along chromosome arms, providing             in mitosis (Skibbens and Hieter 1998). Currently, we <br /> the first link between these two processes (Guacci et al.     cannot rule out a model whereby ctf7 cells contain both <br /> 1997). Given this precedent, we found that Ctf7p was          defective kinetochores and sister chromatid cohesion <br /> also required for condensation at the rDNA locus, al-         complexes. However, it is not necessary to invoke a ki- <br /> though the defect in ctf7 mutants was not as pronounced       netochore defect to explain the checkpoint-mediated de- <br /> as that observed in mcd1 mutants (R.V. Skibbens and D.        lay in ctf7 cells. Interestingly, other cohesion mutants <br /> Koshland, unpubl.). Concomitant with our study, Trf4          also exhibit a mitotic delay, although it is unknown <br /> protein has also been shown to be required for both co-       whether this delay is dependent on a spindle assembly <br /> hesion and condensation (Castano et al. 1996; I. Castano      checkpoint (Strunnikov et al. 1993; Guacci et al. 1997; <br /> and M. Christman, pers. comm.). Thus, their study of          Michaelis et al. 1997). On the basis of our evidence, we <br /> Trf4p and our study of ctf7p further establishes that         envision cohesion defects to activate this checkpoint by <br /> yeast cohesin and condensin complexes share either            altering the state of tension exerted on chromosomes. <br /> common components, regulators, or assembly factors. In        Chromosomes attached to microtubules experience <br /> contrast, Losada et al. (1998) reported that depleting co-    poleward forces that place the chromosome under ten- <br /> hesin components from Xenopus extracts did not ad-            sion. Evidence from meiotic cell systems revealed that <br /> versely affect chromosome condensation. In addition,          tension exerted on chromosomes is critical for signaling <br /> Xenopus 13S condensin and 14S cohesin complexes were          anaphase onset (Nicklas 1997): A decrease or loss of ten- <br /> found to associate with chromatin independent of one          sion generates a mitotic delay. One predicted effect for <br /> another, indicating a separation of function (Losada et al.   the dissolution of chromatid cohesion is to decrease the <br /> 1998). It will be important to pursue whether the cou-        level of tension exerted on chromosomes by spindle mi- <br /> pling/uncoupling of cohesion and condensation reflect         crotubules and activate the spindle assembly checkpoint <br /> differences in organisms, cell systems (somatic versus        (Wells and Murray 1996). This model suggests that sister <br /> embryonic) or segregation cycles (mitotic versus mei-         chromatid cohesion is also a fundamental component of <br /> otic). Such differences have been well documented and         the mechanism that maintains tension on chromo- <br /> what once appeared as conflicting data has historically       somes. <br /> proven to be of tremendous scientific value.                     Typically, checkpoint mechanisms maintain cell vi- <br />                                                               ability by delaying the cell cycle and allowing cells time <br />                                                               to repair damage or correct mistakes. In mutations that <br /> Cohesion, checkpoints, and spindle integrity <br />                                                               induce such damage, the additional loss of checkpoint <br /> Our results showed that ctf7 mutant cells exhibited a         function often results in synthetic lethality. Therefore, a <br /> mitotic delay that required the kinetochore/spindle as-       third question is why is ctf7 not synthetically lethal <br /> sembly checkpoint. Consistent with the preanaphase de-        with mutations in the kinetochore/spindle assembly <br /> lay typically imposed by this checkpoint, ctf7 cells con-     checkpoint? The loss of the checkpoint has little impact <br /> tained high levels of the anaphase inhibitor Pds1p. On        on ctf7 cells for two reasons. First, one component of the <br /> the other hand, ctf7 cells contained elongated spindles       checkpoint mechanism is to block sister separation. <br /> and separated DNA masses, uncharacteristic of preana-         However, ctf7 cells are defective in sister chromatid co- <br /> phase cells. This aberrant metaphase delay raises three       hesion and subvert this component of checkpoint func- <br /> issues that further reveal important functions of sister      tion. Second, sister separation is an irreversible event. <br /> chromatid cohesion.                                           Thus, whether the cell delays or not in response to <br />    First, how does the ctf7 mutation disrupt spindle          checkpoint function, cells with precociously separated <br />  <br />  <br /> 316       GENES &amp; DEVELOPMENT <br />                                                                                                       Chromatid cohesion and replication <br />  <br />  <br />  <br /> sisters are beyond rescue. Our results are analogous to              Generation, integration and suppression of conditional <br /> those reported over a decade ago, double-strand breaks               ctf7 alleles <br /> generated in G1 haploid cells resulted in a RAD9-medi-               PCR-based mutagenesis of pRS316&#226;&#8364;&#8220;CTF7 was performed as de- <br /> ated cell cycle delay at G2 /M. Despite this delay, the              scribed (Kassenbrock et al. 1993) with minor modifications. <br /> cells were inviable (Weinert and Hartwell 1988).                     YBS5 (ctf7&#226;&#338;&#172; covered by pRS316&#226;&#8364;&#8220;CTF7) was cotransformed with <br />                                                                      pooled PCR product and MluI&#226;&#8364;&#8220;NdeI-gapped pRS315&#226;&#8364;&#8220;CTF7 (this <br />                                                                      digestion removes the entire CTF7 ORF but leaves 5&#226;&#172;&#732; and 3&#226;&#172;&#732; <br />                                                                      ends homologous to the CTF7 genomic locus). Of 12,500 Leu+ <br /> Materials and methods                                                transformants screened, 11 scored positive as temperature-sen- <br />                                                                      sitive nonpetite strains by virtue of their inability to grow on <br /> Yeast strains and medium                                             5-FOA plates at 37&#194;&#176;C and growth on glycerol plates at 25&#194;&#176;C. <br /> Medium used for growth and sporulation was described previ-          Plasmids containing putative mutated CTF7 sequences were <br /> ously (Rose et al. 1990). Yeast transformations were performed       recovered from those strains. YBS5 was then transformed with <br /> as described (Ito et al. 1983; Schiestl and Gietz 1989). S288C-      the rescued plasmids to confirm plasmid-dependent tempera- <br /> derived Saccharomyces cerevisiae yeast strains (YPH) and plas-       ture sensitivity. Three of the eleven plasmids that exhibited the <br /> mids (pRS) were described previously (Sikorski and Hieter 1989;      lowest reversion rate and conferred the tightest temperature <br /> Doheny et al. 1993; Connelly and Hieter 1996). Other strains are     sensitivity were chosen for integration into the genome. The <br /> listed in Table 3. To establish genetic interactions, two inde-      2.1-kb insert was cloned into pRS304 and the resulting plasmid <br /> pendent double heterozygous diploids were obtained, sporu-           linearized with HpaI to target integration to the LEU2 locus in <br /> lated, and haploid spores containing both mutations identified.      YBS5. After passage on 5-FOA, His+ Leu+ Ura&#226;&#710;&#8217; transformants <br /> The nonessential reporter chromosome of &#226;&#710;&#188;150 kb contains the         were temperature sensitive for growth. The temperature-sensi- <br /> short arm and centromere of chromosome III for stable propa-         tive phenotype was rescued on retransformation with pRS316&#226;&#8364;&#8220; <br /> gation and SUP11 to suppress the formation of the red byprod-        CTF7. Hence, these mutations are recessive. <br /> uct of the adenine biosynthetic pathway in ade2 mutations               For overexpression suppression, ctf7-203 was transformed <br /> (Spencer et al. 1990).                                               with a 2 &#194;&#181;m of URA3 library (Connelly et al. 1996). Of 24,900 <br />                                                                      transformants, 26 plasmids were obtained that, on retransfor- <br />                                                                      mation, rescued growth at 34&#194;&#176;C. Twenty-three of the twenty- <br />                                                                      six plasmids contained the CTF7 ORF. The remaining three <br /> Cloning and molecular characterization of CTF7                       were found to contain overlapping sequences and one plasmid <br />                                                                      was subcloned to identify the suppressor ORF. <br /> A region of DNA that rescued the chromosome fragment-loss <br /> phenotype in a ctf7 mutant strain (ctf7-108) from the original ctf <br /> collection (Spencer et al. 1990) was identified from a CEN/ <br />                                                                      Morphological and growth analyses <br /> LEU2 library and subcloned following procedures and reagents <br /> described previously (Gerring et al. 1990). This 2.1-kb region       Flow cytometry, Western blot analysis, and indirect immuno- <br /> was shown to encode the chromosome transmission fidelity             fluorescence (IIF) were performed as described previously (Ger- <br /> locus of CTF7 in the following manner. The 2.1-kb fragment           ring et al. 1990; Cohen-Fix et al. 1996) with minor modifica- <br /> was cloned into pRS303, linearized within the CTF7 ORF with          tions. Viability was determined as described previously (Mor- <br /> XbaI and integrated into the genome of YPH250. His+ transfor-        row et al. 1995) with minor modifications. Briefly, cells were <br /> mants were shown to contain HIS3-marked CTF7 integrated at           plated after experimental procedures onto rich YPD medium <br /> the proper locus by Southern analysis on the basis of the known      plates and grown at the permissive temperature for 18 hr. The <br /> restriction map of the region. The resulting strain (YBS3) was       plates were then assessed microscopically for the formation of <br /> mated with a strain from the original collection that contains       microcolonies. Typically, microcolonies were rescored after an <br /> ctf7-10 (Spencer et al. 1990) and sporulated. His+ spores (22 of     additional 24 hr. For execution experiments, the incidence of <br /> 22) exhibited wild-type levels of chromosome loss, whereas all       anaphase onset was based on either spindle elongation or sepa- <br /> 14 His&#226;&#710;&#8217; spores tested sectored dramatically when grown on non-       ration of DNA masses. In shift-down experiments, the percent <br /> selective medium. Database searches of the yeast genome indi-        of anaphase cells rose dramatically because of the cell cycle <br /> cated that CTF7 is contained on Chromosome VI. This chro-            delay observed in ctf7 mutants, consistent with the accumula- <br /> mosome assignment was confirmed by hybridization to yeast            tion of a 2C DNA content. For localization of Ctf7p, the entire <br /> genomic clone contigs (L. Riles and M. Olson, pers. comm.).          CTF7 ORF was placed in-frame with the encoded HA epitope <br />    The CTF7 ORF was replaced with the selectable marker HIS3         tag contained on pAD5 (Neiman et al. 1993). YBS5 was trans- <br /> by a PCR strategy described previously (Baudin et al. 1993). The     formed with this plasmid and pRS316-CTF7 shuffled out from <br /> deletion DNA fragments were generated by PCR by the primers          the resulting strain by use of medium plates containing 5-Fluo- <br /> 5&#226;&#172;&#732;-AGTAATAATACATAAGGAGATGCCAGCTACAACGG-                              rorotic acid, FOA (Boeke et al. 1987) prior to immunolocaliza- <br /> ACATCAACATATATCTAGATTGTACTGAGAGTGCACC-3&#226;&#172;&#732;                             tion. Immunostaining was performed by use of the anti-HA an- <br /> and 5&#226;&#172;&#732;-ACCTTGAAAAGTTTTATGCCATTTTTCCAGTGT-                            tibody 12CA5 (BabCo) in combination with fluorescein- or Cy3- <br /> CCCTTCTCGCTGTCTTTCTGTGCGGTATTTCACACCG-3&#226;&#172;&#732;.                            conjugated goat anti-mouse antibodies (Jackson Immuno- <br /> The wild-type diploid strain YPH987 was transformed with             Research) to visualize epitope-tagged Ctf7p and Pds1p (Cohen- <br /> PCR product and His3+ colonies identified to obtain a hetero-        Fix et al. 1996). Not all nuclei stained with 12CA5, indicating <br /> zygous CTF7/ctf7&#226;&#338;&#172; deletion strain (YBS2). CTF7 deletion was          that protein levels varied considerably when constructs are car- <br /> confirmed by Southern blot analysis. Tetrad analysis yielded         ried on a 2 &#194;&#181;m plasmid. YBS5 similarly transformed with the <br /> 2+:2&#226;&#710;&#8217; spore death with no recovery of His+ spores. To confirm        pAD5 vector only (containing no CTF7 insert) exhibited no <br /> that CTF7 encodes an essential protein, the diploid YBS2             nuclear staining, nor was a nuclear signal observed with sec- <br /> (ctf7&#226;&#338;&#172;::HIS3/CTF7) was transformed with pRS316&#226;&#8364;&#8220;CTF7 and              ondary antibody alone. YOL1/34 (Serotec) and fluorescein- or <br /> sporulated. All His+ Ura+ spores tested were inviable on me-         rhodamine-conjugated goat anti-rat antibodies (Cappel and Boe- <br /> dium plates containing 5-FOA (Boeke et al. 1987).                    hringer Mannhein) were used to visualize microtubules. DNA <br />  <br />  <br /> masses were visualized with DAPI (Sigma). For Western blot           <br /> analysis of HA&#226;&#8364;&#8220;Pds1, cell densities were normalized by optical       <br /> density. Equal loads of cell extracts were confirmed by Western         <br /> blot analysis of tubulin levels.                                        <br />   Reagents used to perform FISH analysis and detailed proce-        <br /> dures are described previously (Guacci et al. 1994, 1997).               <br />                                                                          <br /> </body></html>