<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                      Citation: Cell Death Disease (2013) 4, e866; doi:10.1038/cddis.2013.371 <br /> OPEN                                                                                  &amp; 2013 Macmillan Publishers Limited All rights reserved 2041-4889/13 <br />                                                                                       www.nature.com/cddis <br />  <br />  <br /> Cornelia Lange Syndrome: NIPBL <br /> haploinsufficiency downregulates canonical Wnt <br /> pathway zebrafish embryos patients fibroblasts <br /> A Pistocchi1,2, G Fazio3, A Cereda3, L Ferrari1, LR Bettini3, G Messina2, F Cotelli2, A Biondi3, A Selicorni3 V Massa*,3,4 <br />  <br />  <br /> Cornelia Lange Syndrome severe genetic disorder characterized malformations affecting multiple systems, a <br /> common feature severe mental retardation. Genetic variants genes (NIPBL (Nipped-B-like), SMC1A, SMC3, and <br /> HDAC8) believed responsible majority cases; genes encode proteins &#226;&#8364;&#732;cohesin <br /> complex&#226;&#8364;&#8482;. Cohesins exhibit temporally separated major roles cells: controlling cell cycle involved <br /> in regulating gene expression. The present study focuses role zebrafish nipblb paralog neural <br /> development, examining expression central nervous system, analyzing consequences nipblb loss of <br /> function. Neural development impaired knockdown nipblb zebrafish. nipblb-loss-of-function embryos presented <br /> with increased apoptosis developing neural tissues, downregulation canonical Wnt pathway genes, subsequent <br /> decreased Cyclin D1 (Ccnd1) levels. Importantly, <span id='am-393' about='xsp:pattern' typeof='owl:Thing'>pattern</span> canonical WNT pathway CCND1 downregulation was <br /> observed NIPBL-mutated patient-specific fibroblasts. Finally, chemical activation pathway nipblb-loss-of-function <br /> embryos rescued adverse <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span> restored physiological levels cell death. <br /> Cell Death Disease (2013) 4, e866; doi:10.1038/cddis.2013.371; published online 17 October 2013 <br /> Subject Category: Neuroscience <br />  <br />  <br />  <br /> Cornelia Lange Syndrome (CdLS) multiple malforma-                             terms morphology, axonal trajectory, synaptic specifi- <br /> tion syndrome typically involving developmental delay,                                cities, neurotransmitters.4 This fundamental specification <br /> specific facial features, behavioral abnormalities, major                         controlled dorsoventral (DV) AP gradients of <br /> congenital malformations. Almost organ affected,                           molecules. For example, components Wnt family are <br /> but commonly observed defects ones                              known induce anterior structures (such forebrain), <br /> affect neurodevelopmental, gastrointestinal, muscu-                           bone morphogenetic proteins responsible for <br /> loskeletal systems. There broad spectrum clinical                             neurogenesis dorsal neural tube.5 <br /> involvement, increasing recognition milder                                Genetically, CdLS caused autosomal-dominant <br /> phenotype previously appreciated. Clinically, common                           X-linked mutations. Approximately, 60% cases <br /> feature CdLS mental retardation associated                        genetic defects genes: NIPBL <br /> with enlarged brain ventricles, thinning atrophy white                          (Nipped-B-like), SMC1A, SMC3, recently identified <br /> matter, brainstem hypoplasia, cerebellar vermal hypo-                             HDAC8.1,6 NIPBL gene associated <br /> plasia agenesis.1&#226;&#8364;&#8220;3 During development, central                                CdLS. SMC1A subsequently identified an <br /> nervous (CNS) arises neural plate folds                          additional gene disruption produces phenotypic <br /> up anterior&#226;&#8364;&#8220;posterior (AP) axis form tube divided                      findings CdLS, CdLS case reported <br /> three main units: anlagen prosencephalon (fore-                            having <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span> SMC3. HDAC8 mutated <br /> brain), mesencephalon (midbrain), rhombence-                              CdLS patients, consistent role controlling <br /> phalon (hindbrain). Caudally, tube develop                          acetylation status SMC3. Most recently, mutations <br /> spinal cord. These coordinated events achieved                            RAD21, structural protein cohesin complex, <br /> a finely tuned balance cell proliferation pro-                            children spectrum disabilities <br /> grammed cell death. Within region, large diversity                          overlapping CdLS syndrome, presenting with <br /> neuronal cell types generated, distinct identities                    milder cognitive impairments consistent so-called <br />  <br />  <br /> 1 <br />  Department Medical Biotechnologies Translational Medicine, University Milan, Via Viotti, 3/5, 20133, Milan, Italy; 2Department Biosciences, University of <br /> Milan, Via Celoria, 26, 20133, Milan, Italy 3Department Paediatrics, S.Gerardo Hospital/Fondazione MBBM, University Milano-Bicocca, Via Pergolesi, 33, <br /> 20900, Monza, Italy <br /> *Corresponding author: V Massa, Department Paediatrics, S.Gerardo Hospital/Fondazione MBBM, University Milano-Bicocca, Via Pergolesi, 33, 20900, Monza, <br /> Italy. Tel: &#195;&#190; 39 039 2333661; Fax: &#195;&#190; 39 039 2332167; E-mail: v.massa@hsgerardo.org <br /> 4 <br />  Present address: Department Health Sciences, University Milan, Via A. Di Rudin&#196;&#177;&#194;&#180;, 8, 20142, Milan, Italy <br /> Keywords: Cornelia Lange Syndrome; cyclins; zebrafish; fibroblasts; NIPBL; apoptosis <br /> Abbreviations: CdLS, Cornelia Lange Syndrome; Ccnd1, Cyclin D1; Ccnd2, Cyclin D2; CNS, central nervous system; AP, anterior&#226;&#8364;&#8220;posterior; DV, dorsoventral; <br /> WISH, whole-mount situ hybridization; RPMI, Roswell Park Memorial Institute serum; q-PCR, quantitative RT&#226;&#8364;&#8220;PCR; TBST, tris-buffered saline/Tween 20 buffer; ABC, <br /> active form beta-catenin; Fmi/Stan, Flamingo/Starry <br /> Received 31.7.13; accepted 01.8.13; Edited G Raschella&#194;&#180; <br />                                                NIPBL: canonical Wnt pathway apoptosis <br />                                                                          A Pistocchi et al <br /> 2 <br />  <br />  <br />  <br />  <br />           Figure 1 Expression analysis nipblb. (a&#226;&#8364;&#8220;d) WISH nipblb-specific probe 24 hpf. (a) Whole-mount embryo showing modulated nipblb expression AP axis. <br />           c d indicate position histological sections reported c d, respectively. (b) Dorsal view (anterior left) different regions nipblb expression the <br />           CNS. (c) Transverse section neural tube showing nipblb transcript ubiquitously expressed DV axis. (d) Longitudinal section showing nipblb <br />           expression AP axis. (e) Comparison nipblb (in blue) krox20 (in red) expressions showing nipblb extends posteriorly far rhombomere V at <br />           24 hpf. di, diencephalon; m, midbrain; h, hindbrain; ot, otocystis. Scale bar, 100 mm <br />  <br />  <br />           coesinopathy.3 All genes encode proteins involved                                   order dissect molecular mechanisms underlying the <br />           &#226;&#8364;&#732;cohesin complex&#226;&#8364;&#8482;.                                                                       neurodevelopmental disorders. First, effects nipblb <br />             The cohesin complex evolutionarily conserved fungi                                 loss function developing CNS zebrafish, in <br />           human cells, canonical role regulation                                  particular hindbrain level, characterized. The <br />           chromatid separation cell division.7 In particular,                               results obtained functional studies zebrafish embryos <br />           cohesin complex controls sister chromatid cohesion                                    confirmed fibroblasts CdLS patients with <br />           S phase. In somatic cells vertebrates, core subunits                             known mutation NIPBL. <br />           forming complex identified: SMC1 (SMC1A or <br />           SMC1B), SMC3, SCC1, SA1 SA2. Associated                                        Results <br />           core complex, proteins assist cell <br />           division, example, NIPBL, appears facilitate                                Zebrafish nipblb expressed CNS. In accordance <br />           uploading complex, WAPL, enables complex                                     literature,12,14 nipblb ubiquitously expressed during <br />           dissociation.8                                                                               early somitogenesis zebrafish embryos. The expression <br />             A non-canonical role recently observed,                                   progressively decreased trunk appeared specifically <br />           cohesin complex represents key regulator gene                                  localized CNS 24 hpf (hours post fertilization) <br />           expression. In Drosophila, NIPBL ortholog (i.e., Nipped-B)                                   (Figure 1a). As seen dorsal view, expression nipblb <br />           shown participate long-distance activation                                (Figure 1b) detected diencephalon mesence- <br />           cut Ultrabithorax gene expression.9 In zebrafish                                      phalon, highlighted progression hindbrain ventricle <br />           (Danio rerio), cohesins regulate runx transcription factors                                  opening. Histological longitudinal transverse sections of <br />           hematopoiesis.10 Interestingly, functionally,                              24 hpf embryos clearly showed nipblb expressed <br />           cell cycle control gene expression regulation                                    entire dorsal&#226;&#8364;&#8220;ventral (DV) AP axis of <br />           temporally separated. For example, Drosophila mutants                                    CNS (Figures 1c d). Colocalization krox20, a <br />           orthologs cohesin complex genes defect axon                                     marker rhombomeres III V,15 showed nipblb <br />           retraction physiologically occurs post-mitotic neurons,                              expression extended caudally far rhombomere V this <br />           cells exited cell cycle.11 In zebrafish,                              developmental stage (Figure 1e). <br />           cohesins expressed proliferating non-proliferat- <br />           ing cells,12 mice heterozygous Nipbl                                 Inhibition Nipblb function zebrafish embryos affect <br />           severe developmental defects generally altered gene                                      CNS development. Embryos injected nipblb-morpho- <br />           expression absence cell cycle sister chromatid                                  lino (MO) began exhibit defects 24 hpf, including brain <br />           impairment.13                                                                                defects (50% nipblb-MO injected, total n &#194;&#188; 200, Figures 2a <br />             A nipbl-deficiency model zebrafish shows altered                                        b). In particular, nipblb-loss-of-function embryos were <br />           expression genes involved endodermal differentiation                                   microphthalmic microcephalic. In addition, nipblb- <br />           spectrum heart, gut, visceral organ defects                              MO-injected embryos presented short, curved <br />           bear striking similarity observed CdLS                                    tail (40% nipblb-MO injected, total n &#194;&#188; 180, Figure 2b), a <br />           patients.14 In present study, used combined                                  phenotype previously described Muto et al.14 following <br />           approaches, zebrafish embryos patient-derived cells,                                  nipblb loss function. nipblb-MO efficiency specificity <br />  <br /> Cell Death Disease <br />                                                                                         NIPBL: canonical Wnt pathway apoptosis <br />                                                                                        A Pistocchi et al <br />                                                                                                                                                                                  3 <br />  <br />  <br />  <br />  <br /> Figure 2 MO knockdown nipblb. (a b) Morphology living embryos 24 hpf. (a) Embryos injected ctrl-MO. (b) Embryos injected nipblb-MO <br /> presenting brain defects curved tail. (c) Nipbl reduced MO-injected embryos 39&#194;&#177;0.04% compared controls (expressed mean triplicate densitometric <br /> analyses normalized controls). (d) Western blot protein extracted minimum 30 embryos experimental group showing reduced levels Nipbl (316 KDa) in <br /> two different batches 24 hpf nipblb-MO-injected embryos compared controls developmental stage. Lower band beta-actin (50 KDa) <br />  <br />  <br /> were verified western blot experiments Nipbl                               number proliferative cell nuclear antigen-positive cells in <br /> protein levels reduced 39% nipblb-MO-injected                               nipblb-MO-injected embryos 24 hpf significantly <br /> embryos 24 hpf, compared stage-matched control                                 increased comparison controls (Supplementary <br /> embryos (Figures 2c d).                                                            Figure S2). <br />  <br /> nipblb downregulation affects cell <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>survival</span> cell                               nipblb loss function alters Wnt signaling pathway <br /> proliferation. When compared controls, nipblb-MO-                                 expression. Owing expression nipblb throughout <br /> injected embryos showed increased programmed cell death                                CNS, investigated CNS AP formation <br /> in CNS 24 hpf, apoptosis                                       altered nipblb-loss-of-function embryos 24 hpf. <br /> pronounced CNS control embryos (Figures 3a&#226;&#8364;&#8220;d).                                   The expression AP identity markers, hoxb2a,20 <br /> Interestingly, TUNEL staining showed apoptotic                            pax2a,21 krox20,15 altered nipblb-MO-injected <br /> cells evident tissues expressing nipblb                           embryos (Supplementary Figure S3), suggesting the <br /> gene (compare Figure 1a Figures 3b&#226;&#8364;&#8220;d). Previous                                    CNS patterned correctly. However, wnt1 <br /> studies zebrafish described p53-dependent apoptosis                                 expression22 severely altered hindbrain nipblb- <br /> following cohesin smc3 knockdown, p53 upregulation                                 MO-injected embryos 24 hpf (80%, total analyzed embryos <br /> in mutants cohesin subunit rad21.16,17 It known                        n &#194;&#188; 120; Figures 4a b). It possible observe nipblb <br /> that MO molecules elicit undesirable off-target effects                          knockdown phenotypes different degrees fusion the <br /> such activation p53 protein. Importantly, reversal                        wnt1 signal hindbrain rhombomere II the <br /> p53-dependent cell death p53 knockdown did affect                               spinal cord: severe phenotype included complete <br /> cell death caused specific gene loss function.18 <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span>                              fusion wnt1 hindbrain signals (20%, total n &#194;&#188; 120), the <br /> verify apoptosis nipblb-MO-injected embryos                            intermediate phenotype expressed multiple points wnt1 <br /> to specific loss function gene result                      fusion AP hindbrain axis (40%, total n &#194;&#188; 120), and <br /> p53 activation, co-injected p53-MO nipblb-MO. In                               severe phenotype presented single point <br /> p53-MO &#195;&#190; nipblb-MO-injected embryos, apop-                               wnt1 fusion incomplete opening ventricle <br /> totic cells comparable nipblb-loss-of-function embryos,                        (20%, total n &#194;&#188; 120, Figures 4a b). <br /> which verified neural apoptosis specifically caused                           To investigate observed defects specific <br /> knocking nipblb (n &#194;&#188; 42) (Supplementary Figure S1).                               mis-regulation wnt1 expression following nipblb loss <br /> These results suggest nipblb required <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>cell survival</span>                        function caused morphological defects in <br /> during neurulation zebrafish embryos.                                               hindbrain formation, analyzed expression atoh1, a <br />    To determine nipblb loss function alters                        marker hindbrain dorsal progenitors ventricle open- <br /> proliferation rate, embryos stained proliferative                        ing.23 atoh1 expression altered nipblb-MO-injected <br /> cell nuclear antigen antibody. As expected previously                             embryos 24 hpf (90%, total n &#194;&#188; 50; Supplementary Figures <br /> published studies cohesin loss function,10,19                           S4A B), showing anatomical patterning the <br /> that entry S phase compromised,                                developing brain affected nipblb loss function. <br />  <br />                                                                                                                                                              Cell Death Disease <br />                                               NIPBL: canonical Wnt pathway apoptosis <br />                                                                         A Pistocchi et al <br /> 4 <br />  <br />  <br />  <br />  <br />           Figure 3 Apoptosis increased CNS nipblb-loss-of-function embryos. (a b) Apoptotic cells labeled TUNEL strongly increased (b) nipblb-MO- <br />           injected embryos 24 hpf comparison control embryos (a) developmental stage. (c d) Transverse sections indicated prevalent cell death neural <br />           tube (d, arrow) nipblb-MO-injected embryos comparison controls (c). nt, neural tube. Scale bar, 100 mm <br />  <br />  <br />  <br />  <br />           Figure 4 wnt1 expression specifically mis-regulated hindbrain nipblb-MO-injected embryos. (a b) wnt1 expression controls nipblb-MO-injected <br />           embryos 24 hpf. Altered wnt1 expression hindbrain (b) nipblb-MO-injected embryos 24 hpf comparison (a) controls developmental stage. <br />           Asterisks indicate multiple points wnt1 fusion AP hindbrain axis (b) nipblb-MO-injected embryos. (c d) axin2 expression showing comparable defects of <br />           patterning nipblb-MO-injected embryos (d) comparison controls (c). Scale bar, 100 mm <br />  <br />  <br />  <br />              As defects neural tube patterning disrupt                                    Expression       profiling             WNT/beta-catenin <br />           neuronal differentiation, examined dopaminergic                                   pathway target genes patients fibroblasts and <br />           population (thyrosin-hydroxilase-positive neurons) 24 hpf                             nipblb-MO-injected embryos. In zebrafish embryos, nipblb <br />           following nipblb loss function. It known                                 loss function resulted alterations Wnt/beta-catenin <br />           formation mesencephalic dopaminergic neurons                                       pathway, modeling molecular defects underlying NIPBL- <br />           directed diffusable signals notochord,                              mutated CdLS patients. Hence, sought analyze the <br />           floor plate, isthmic organizer Shh Fgf,                              expression profile signaling pathway fibroblasts of <br />           Wnt1 extrinsic factors.24 In nipblb-loss-of-                           patients, comparing cells obtained age-, <br />           function embryos, neurodifferentiation                             ethnicity- gender-matched healthy controls. Fibroblasts <br />           population adversely affected (Supplementary                                     believed represent valid model cohesin deficiency <br />           Figures S4C D).                                                                      humans6 studies gene expression, consistent <br />              Importantly, comparable alterations gene distribution                              altered profiles observed primary cultures of <br />           observed wnt1 observed axin2 expression                                  fibroblasts lymphoblastoid cell lines.6 <br />           pattern,25 key downstream component pathway                                        It possible exclude involvement non-canonical <br />           typically used readout cascade (Figures 4c                                   pathway, DVL1 DVL226 comparable in <br />           d).                                                                                  patients control group (Supplementary Figures S5A B). <br />  <br /> Cell Death Disease <br />                                                                                              NIPBL: canonical Wnt pathway apoptosis <br />                                                                                             A Pistocchi et al <br />                                                                                                                                                                                          5 <br />  <br />  <br />  <br />  <br /> Figure 5 The canonical Wnt pathway downregulated animal human models CdLS. (a b) Human (CCND1) zebrafish (ccnd1) real-time analysis in <br /> (a) patients-specific fibroblasts (b) nipblb-MO-injected embryos showing reduction trend transcripts models (asterisks b represent Po0.01). (c d; g <br /> and h) Western blot analyses. Quantification relative protein expression normalization beta-actin. The quantification (zebrafish) (human) <br /> independent experiments data expressed fold change relative controls standard errors. (c d). The reduction CCND1 confirmed protein <br /> levels CdLS patient-derived cells (c) nipblb-MO-injected embryos (d). (e f) WISH analysis showing decreased expression levels ccnd1 nipblb-MO-injected <br /> embryos (f) compared control embryos (e) developmental stage. (g h) Levels ABC reduced CdLS models, is, human (g) zebrafish <br /> (h), compared specific controls. Asterisk h represents Po0.05. (e f) Dorsal view, anterior left <br />  <br />  <br />  <br /> Analysis expression direct targets canonical                               Consistently, mean reduction CCND1 levels almost <br /> pathway, Cyclins, quantitative PCR (q-PCR)                                       50% observed patients samples compared <br /> showed trend reduction Cyclin D1 (CCND1)                                 healthy controls (Figure 5c, Supplementary Figure 5D). <br /> NIPBL-mutated patients compared healthy controls                                       Ccnd1 significantly downregulated in <br /> (Figure 5a). As known Cyclins compensate                                     nipblb-MO-injected embryos q-PCR (Figure 5b), western <br /> for deficiency isoforms mammals,27                                       blot analysis decrease Ccnd1 expression 42% <br /> also analyzed Cyclin D2 (CCND2)                                          (Figure 5d), whole-mount situ hybridization (WISH) <br /> significantly upregulated (Po0.01) (Supplementary Figure S5C).                              analyses (Figures 5e f). The showed the <br />  <br />                                                                                                                                                                      Cell Death Disease <br />                                               NIPBL: canonical Wnt pathway apoptosis <br />                                                                         A Pistocchi et al <br /> 6 <br />  <br />  <br />  <br />  <br />           Figure 6 Molecular morphological defects CdLS zebrafish model rescued LiCl treatment. (a&#226;&#8364;&#8220;c) Dorsal view (anterior left) flat-mounted embryos <br />           showing altered wnt1 expression nipblb-MO-injected embryos (b) multiple points fusion (asterisks) comparison controls (a), restored wnt1 expression to <br />           normal pattering nipblb-MO-injected embryos treated LiCl (c). (d f) Lateral view embryos stained TUNEL assay showing treatment LiCl (f) <br />           rescues morphological cephalic defects reduces level abnormal apoptosis (e) restoring physiological levels comparable control embryos (d). (g&#226;&#8364;&#8220;j) Western blot <br />           analyses showing ABC (g&#226;&#8364;&#8220;i) Ccnd1 (h&#226;&#8364;&#8220;j) protein levels strongly increased nipblb-MO-injected LiCl-treated embryos comparison nipblb-MO-injected <br />           embryos. Western blot quantification (zebrafish) independent experiments, data expressed fold change nipblb-MO-injected embryos and <br />           nipblb-MO-injected LiCl-treated embryos controls standard errors. Asterisks g represent *Po0.05 **Po0.01 <br />  <br />  <br />  <br />           expression ccnd1 altered nipblb-MO-injected                                    experiments failed produce visible effects control <br />           embryos 24 hpf comparison controls (40%, total                                embryos (minimum 30 experiment), able to <br />           n &#194;&#188; 30).                                                                                 strikingly reverse defects induced nipblb-MO injection <br />              In addition, average 18% reduction                              (minimum 30 experiment). In particular, cephalic <br />           active form beta-catenin (ABC)28 patients compared                            structures showed restored morphology, wnt1 <br />           controls normalization (Figure 5g, Supplementary                                   distribution pattern similar control embryos <br />           Figure 5D). The pattern observed nipblb-MO-                                  (Figures 6a&#226;&#8364;&#8220;c). Apoptosis levels determined TUNEL <br />           injected embryos statistically significant reduction                           staining reduced LiCl treatment nipblb-MO- <br />           ABC levels 80% (Figure 5h).                                                           injected embryos levels comparable control <br />                                                                                                    embryos (Figures 6d&#226;&#8364;&#8220;f). Western blot analysis nipblb-MO- <br />           Rescue nipblb deficiency phenotype chemical                                        injected embryos treated LiCl showed canonical <br />           induction canonical Wnt pathway. As data sugg-                                    Wnt pathway induced, ABC levels increased by <br />           ested adverse phenotype caused knocking                                 fourfold compared controls 10-fold <br />           nipblb zebrafish embryos patients fibroblasts                           compared nipblb-MO-injected embryos (Figures 6g&#226;&#8364;&#8220;i). <br />           mediated attenuation canonical Wnt pathway                              Both differences statistically significant. Notably, <br />           subsequent decreased Ccnd1 expression, sought                                   Ccnd1 protein levels following LiCl treatment induced <br />           rescue phenotype nipblb-loss-of-function embryos                               3.7-fold compared nipblb-MO-injected <br />           chemical induction Wnt pathway lithium chloride                                embryos 2.4-fold increase compared controls. <br />           (LiCl) treatment.29 The concentration used sets                              (Figures 6h&#226;&#8364;&#8220;j). <br />  <br /> Cell Death Disease <br />                                                                         NIPBL: canonical Wnt pathway apoptosis <br />                                                                        A Pistocchi et al <br />                                                                                                                                                   7 <br />  <br /> Discussion                                                             treatment, suggesting model, primary adverse <br />                                                                        event Gsk3b-Beta-catenin complex level, that <br /> In study, different modeling strategies used to <br />                                                                        chemical compensation triggers negative feedback wnt1 <br /> analyze abnormalities underlying CdLS, focusing NIPBL, <br />                                                                        expression control. In fact, reported negative <br /> which commonly mutated gene patients affected <br />                                                                        feedback upregulation Axin2 cancer model.38 <br /> by disease.1,30 A developmental study vivo using <br />                                                                           Our CdLS zebrafish model suggests role canonical <br /> zebrafish embryos initially used identify gene pathways <br />                                                                        Wnt pathway perturbation CNS abnormalities. In murine <br /> that contribute phenotype, followed a <br />                                                                        model autosomic recessive microcephaly, brain alterations <br /> validation study primary cultures CdLS patients <br />                                                                        caused decrement expression levels Wnt1-regulated <br /> fibroblasts subsequently tested. <br />                                                                        genes, overexpression beta-catenin neural cells could <br />    First, expression nipblb CNS zebrafish <br />                                                                        rescue defects.39 Several animal models alterations in <br /> embryos 24 hpf, time neurulation process <br />                                                                        Wnt-related gene expression partially overlapping pheno- <br /> is completed, evaluated. This alleviates existing <br />                                                                        type: well-known masterblind (mbl) mutant, develops <br /> data gap literature, expression pattern zebrafish <br />                                                                        reduction telencephalon, optic vesicles ,and abnormal <br /> nipblb different developmental stages previously <br />                                                                        AP patterning, spontaneous axin1 mutant.34,35 <br /> reported.12,14 The results showed dynamic pattern expres- <br />                                                                        Since mouse (for example, Hesx1, Six3, and <br /> sion regulated spatio-temporally, specific localization at <br />                                                                        Tcf3),40&#226;&#8364;&#8220;42 zebrafish mutants43,44 described with <br /> the level cephalic neural tissues. The zebrafish          comparable phenotype AP patterning CNS during <br /> paralog nipbla reported similar expression             embryonic development. In humans, HESX1 SIX3 have <br /> pattern function developing visceral organs.14              causative hypopituitarism septo-optic <br />    Using MO technology, model haploinsufficiency               dysplasia,45,46 developmental disorder characterized by <br /> zebrafish embryos generated. This model mimics                 severe defects brain eye. In 1996, case report <br /> genetic haploinsufficiency CdLS patients mutation           described 5-year-old CdLS patient autopsy revealed <br /> NIPBL. The nipblb-loss-of-function embryos showed severe               hypoplasia optic systems, hypothalamic nuclei, corpus <br /> alterations head morphology, various degrees            callosum, cerebellar vermis. Moreover, septum pelluci- <br /> microcephaly, microphthalmia, incomplete development               dum, fornix, anterior commissure rudimentary. This <br /> of cephalic structures.                                          case showed features septo-optic dysplasia combined with <br />    This adverse phenotype correlated increased apoptosis       commissural dysplasia cerebellar vermian hypoplasia.47 <br /> the areas nipblb normally expressed, suggesting             It known decreased levels Wnt1 signaling results in <br /> the cohesins important functional role cell survival     downregulation Ccnd1 leads apoptosis several <br /> specific tissues vertebrate development. In system,         animal models.48&#226;&#8364;&#8220;50 It important note mammals, <br /> the proliferation rate unaffected. Interestingly,           partial compensation upregulation Ccnd2,27 <br /> expression distribution number markers AP develop-          observed cell system. It known that <br /> ment appeared comparable control embryos. wnt1 distribution,        GSK3b overexpression induces increased apoptosis vivo <br /> on hand, significantly impaired hindbrain         perturbing proliferation maturation, resulting loss of <br /> level, prospect pons, medulla oblongata, cerebellum,        immature neurons.51 The reduction Wnt ligand levels the <br /> mammals. We ruled wnt1 expression alterations            removal canonical pathway component beta-catenin <br /> caused abnormalities anatomy developing structures         results fewer neuroepithelial/radial glial stem cells in <br /> atoh1, expression covers entire dorsal lateral           precocious neuronal differentiation mouse embryos.52 <br /> aspect cerebellar anlage, properly distributed nipblb-      Further, shown murine models of <br /> MO-injected embryos.31,32 The wnt1 localization, however, failed       autism-like disorders, conspicuous phenotype an <br /> to restricted lateral tissues embryos. This      increase programmed cell death developing <br /> consistent previous data indicating presence under-        cerebellum.53,54 In CdLS patients, alterations develop- <br /> lying anterio-posterior pattern absence Wnt signals.33&#226;&#8364;&#8220;35    ment cerebellar structures anatomical levels known, <br />    The canonical Wnt pathway altered                   robust analyses magnetic resonance imaging <br /> in nipblb-MO-injected embryos patients fibroblasts.             help unraveling link autism features, <br /> In cases, ABC downregulated. Ccnd1, known               seizures, altered CNS development patients. <br /> direct target pathway, consequently downregulated              Increased apoptosis vivo led hypo- <br /> in models. Noticeably, abnormal cephalic phenotype            thesize that, extent, mental retardation <br /> observed nipblb-MO-injected embryos strongly reminis-            autism-like behavior typically associated CdLS <br /> cent observed Ccnd1 knockdown.                           patients roots abnormality. The <br />    Using LiCl well-known chemical activator                reported autopsy47 showed reduction Purkinje cells number <br /> Wnt pathway,34,36 negatively regulates GSK3b inhibition          cerebellum, data implicate imbalance <br /> of beta-catenin,37 possible rescue adverse               cell death control development causative. <br /> phenotype nipblb-MO-injected embryos levels: anato-             Several pieces evidence suggest possible regulation of <br /> mical cephalic structures appeared morphologically normal,         Wnt pathway cohesins non- dividing cells different <br /> programmed cell death levels restored physiological            models. In skin fibroblasts expressing mutated form of <br /> levels seen controls, Ccnd1 protein levels                 HDAC8, WNT5A, WNT2, WISP2, FZD8 be <br /> increased. Interestingly, wnt1 pattern expression appeared          downregulated compared wild-type cells.6 In Drosophila, <br /> rescued zebrafish nipblb-MO-injected embryos LiCl                SMC3 appears important control expression Flamingo/ <br />  <br />                                                                                                                               Cell Death Disease <br />                                               NIPBL: canonical Wnt pathway apoptosis <br />                                                                         A Pistocchi et al <br /> 8 <br />  <br />           Starry night (Fmi/Stan) orthologs vertebrate Celsr,                              female Italian pediatric patients, undergoing surgical procedures for <br />           known transduce Wnt signaling cascade                                dermatological testing. Skin biopsies collected CdLS patients and <br />           non-canonical pathway control cell polarity.55 It                             pediatric controls, finely shredded using scalpel sterile conditions. <br />                                                                                                   Skin fragments cultured RPMI medium supplemented 20% fetal <br />           interesting note regulation two-way, in <br />                                                                                                   bovine serum. After B7&#226;&#8364;&#8220;10 days, fibroblasts started grow, soon cell <br />           colorectal cancer, known upregulation beta-catenin                               confluence reached, cells collected, following standard trypsinization <br />           appears control expression SMC3.56                                                procedure, suspended Trizol reagent (Invitrogen, Life Technologies, Carlsbad, <br />              In conclusion, paper shows time,                           CA, USA) radio immunoprecipitation assay buffer, stored   80 1C. <br />           knowledge, regulation Wnt-canonical pathway                                 For experiments, passaged (maximum P &#194;&#188; 3) cells compared. <br />           cohesins development zebrafish embryos                                    Reverse transcription-PCR real-time q-PCR assays. RNA <br />           fibroblasts CdLS patients. This suggests potential                            extraction performed following manufacturer&#226;&#8364;&#8482;s protocol. Superscript II <br />           underlying mechanism developmental abnormalities                                enzyme (Life Technologies) used cDNA synthesis. Quantitative-reverse <br />           observed CNS CdLS patients.                                                   transcription (RT)-PCRs zebrafish embryos (minimum 30 experimental <br />                                                                                                   group) carried total volume 25 ml containing half volume SYBR <br />           Materials Methods                                                                   Green (Life Technologies). For normalization purposes, ef1-alpha RNA level was <br />           Animals. Breeding wild-type fish AB strain maintained 28 1C           tested parallel gene interest. Primers reported Supplementary <br />           14-h light/10-h dark cycle. Embryos collected natural spawning, staged        Table S1. Melting curve derivatives assessed run. <br />           according Kimmel et al.,57 raised 28 1C fish water (Instant Ocean,            q-RT-PCR experiments human samples performed Universal Probe <br />           United Pet Group, Blacksburg, VA, USA; 0.1% methylene blue) Petri dishes,            Master (Roche Diagnostics). For set experiments, Light Cycler 480II <br />           according established techniques, approved veterinarian (OVSAC)           (Roche Diagnostics) used. <br />           animal use committee (IACUC) University Oregon, agreement            Primers probes selected according Software Probe Finder <br />           local national sanitary regulations. Embryonic ages expressed somites        (Roche Diagnostics) reported Supplementary Table S2. ABL gene was <br />           (s), hours post fertilization (hpf).                                                 used reference healthy patients cells standard control. <br />              To activate Wnt/beta-catenin signaling, LiCl added fish water 30 min at <br />                                                                                                   Western blot. Fish embryos (minimum 30 experimental group) human <br />           10&#226;&#8364;&#8220;12 s stage concentration 0.3 M 28 1C. Treated embryos then <br />                                                                                                   fibroblasts lysed radio immunoprecipitation assay buffer homogenized. <br />           washed times water allowed develop 24 hpf. <br />                                                                                                   Samples boiled 10 min 95 1C 20   reducing agent (Bio-Rad, <br />           In situ hybridization, histological analysis, immunohisto-                          Hercules, CA, USA). Thirty micrograms protein samples size-fractionated by <br />           chemistry. WISH experiments carried described Thisse et al.58            pre-casted gel (4&#226;&#8364;&#8220;15%, Bio-Rad) transferred following standard protocol. The <br />           Immunohistochemistry analyses performed according method described          polyvinylidene fluoride membranes (Bio-Rad) blocked 10% nonfat dry milk in <br />           Panzer et al.59 For experiment, minimum 30 controls MO-injected        TBST (TBS, pH 7.5, containing 0.1% Tween 20) 1 h room temperature and <br />           embryos analyzed. To clone nipblb probe, PCR used (primers                subsequently incubated primary antibody. Primary antibodies used at <br />           Supplementary Table S1); krox20 probe synthesized described Oxtoby        appropriate working dilutions (Supplementary Table S3), 5% milk/TBST night at <br />           Jowett;15 wnt1 probe synthesized described Molven et al.;22 axin2         4 1C. Horseradish peroxidase-conjugated secondary antibodies (Sigma-Aldrich, <br />           probe synthesized described Shimizu et al.;25 atoh1 probe synthesized     St Louis, MO, USA) used 1 h room temperature. The antigen signal <br />           described Millimaki et al.,23 pax2a probe synthesized described         detected immune-star chemiluminescence detection (Bio-Rad) as <br />           Krauss et al.;21 hoxb2a probe synthesized described Prince                specified manufacturer. A StripAblot Stripping Buffer (Euroclone S.p.A., Milan, <br />           colleagues.20 For histological sections, stained embryos re-fixed 4%            Italy) used recover membranes. Densitometry analyses performed using <br />           paraformaldehyde, dehydrated stored methanol, wax-embedded               Alliance instrument Uviband software (Uvitec, Cambridge, UK). Beta-actin was <br />           sectioned (5 mm). For immunohistochemistry experiments, embryos exposed         used reference protein concentrations data expressed normalized <br />           specific primary antibodies (Supplementary Table S3), followed appropriate           control levels. For zebrafish embryos, experiments run three <br />           biotinylated secondary antibodies (Vector Laboratories, Burlingame, CA, USA).           independent experiments human fibroblasts duplicates. <br />           Images embryos sections acquired using microscope equipped with <br />                                                                                                   Statistical analysis. For q-RT-PCR experiments, data statistically <br />           digital camera LAS Leica Imaging software (Leica, Wetzlar, Germany). Images <br />                                                                                                   analyzed applying two-tailed t-test setting Pr0.05 significant. Data were <br />           processed using Adobe Photoshop software (Adobe System Incorporated, <br />                                                                                                   analyzed using comparative DDCt method20,60 t-test S.D. values <br />           San Jose`, CA, USA), and, necessary, different focal planes image <br />                                                                                                   refer samples triplicates. For western blot experiments, raw data analyzed <br />           taken separately later merged single image. <br />                                                                                                   applying two-tailed paired t-test. Significance set Po0.05. <br />           TUNEL staining. For TUNEL assay, minimum 24 embryos (per <br />           experimental group) fixed 4% paraformaldehyde 2 h room                 Conflict Interest <br />           temperature. Embryos washed methanol &#226;&#8364;&#8220;20 1C twice with <br />                                                                                                   The authors declare conflict interest. <br />           PBC (0.001% Triton X-100, 0.1% sodium citrate PBS) 10 min. Staining for <br />           apoptotic cells performed using AP-In situ Cell Death Detection Kit (Roche <br />           Diagnostics, Penzberg, Germany) carefully leaving labeling reagents react        Acknowledgements. We grateful Italian National Association of <br />           <span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span> time experiments. Embryos incubated 37 1C        Volunteers Cornelia Lange support inspiration, patients for <br />           1 h, washed, stained, mounted microscopic imaging.                              consenting use tissues study. We thankful Mr. Eugenio Gautiero <br />                                                                                                   (Genetic Laboratory San Gerardo Hospital, Monza, Italy) technical advice in <br />           Injections. To repress nipblb mRNA translations, ATG-targeting MO <br />                                                                                                   fibroblast isolation culture Dr. Laura Corti Prof. Emilio Berti assisting <br />           (nipblb-MO) synthesized (Gene Tools LLC, Philomath, OR, USA), nipblb-MO: <br />                                                                                                   healthy donor samples recruitment. We like express deep gratitude <br />           50 -GTCCCCATTCATGCTGAAGAAGGGA-30 , used concentration of <br />                                                                                                   Prof. Finnell, Dr. Greene, Prof. Bellipanni Dr. Gaston-Massuet commenting <br />           1 pmol/embryo 1   Danieau buffer (pH 7.6) previously reported.60 As a <br />                                                                                                   manuscript. This work funded Fondazione MBBM (Monza, Italy). <br />           control, standard control MO oligonucleotide (ctrl-MO) injected. In all <br />           experiments, nipblb-MO-injected embryos compared embryos injected <br />           ctrl-MO developmental stage. p53-MO              1. Kline AD, Krantz ID, Sommer A, Kliewer M, Jackson LG, FitzPatrick DR et al. Cornelia de <br />           designed (Gene Tools) used described Robu et al.18                                Lange syndrome: clinical review, diagnostic scoring systems, anticipatory <br />                                                                                                       guidance. Am J Med Genet A 2007; 143A: 1287&#226;&#8364;&#8220;1296. <br />           Cell cultures. Biopsies CdLS patients known mutation NIPBL             2. Krantz ID, McCallum J, DeScipio C, Kaur M, Gillis LA, Yaeger D et al. Cornelia Lange <br />           healthy controls (Supplementary Table S4) used study.                syndrome caused mutations NIPBL, human homolog Drosophila <br />           Samples obtained signed informed consent. Healthy donors, male               melanogaster Nipped-B. Nat Genet 2004; 36: 631&#226;&#8364;&#8220;635. <br />  <br />  <br /> Cell Death Disease <br />                                                                                                             NIPBL: canonical Wnt pathway apoptosis <br />                                                                                                            A Pistocchi et al <br />                                                                                                                                                                                                                   9 <br />  <br />  3. Deardorff MA, Wilde JJ, Albrecht M, Dickinson E, Tennstedt S, Braunholz D et al. RAD21                 34. van Water S, van Wetering M, Joore J, Esseling J, Bink R, Clevers H et al. Ectopic <br />     mutations cause human cohesinopathy. Am J Hum Genet 2012; 90: 1014&#226;&#8364;&#8220;1027.                                 Wnt signal determines eyeless phenotype zebrafish masterblind mutant. <br />  4. Lumsden A, Krumlauf R. Patterning vertebrate neuraxis. Science 1996; 274: 1109&#226;&#8364;&#8220;1115.                   Development 2001; 128: 3877&#226;&#8364;&#8220;3888. <br />  5. Altmann CR, Brivanlou AH. Neural patterning vertebrate embryo. Int Rev Cytol 2001;              35. Heisenberg CP, Houart C, Take-Uchi M, Rauch GJ, Young N, Coutinho P et al. A mutation <br />     203: 447&#226;&#8364;&#8220;482.                                                                                              Gsk3-binding domain zebrafish Masterblind/Axin1 leads fate transformation of <br />  6. Deardorff MA, Bando M, Nakato R, Watrin E, Itoh T, Minamino M et al. HDAC8 mutations                    telencephalon eyes diencephalon. Genes Dev 2001; 15: 1427&#226;&#8364;&#8220;1434. <br />     Cornelia Lange syndrome affect cohesin acetylation cycle. Nature 2012; 489: 313&#226;&#8364;&#8220;317.            36. Klein PS, Melton DA. A molecular mechanism effect lithium development. Proc <br />  7. Bose T, Gerton JL. Cohesinopathies, gene expression, chromatin organization. J Cell                    Natl Acad Sci USA 1996; 93: 8455&#226;&#8364;&#8220;8459. <br />     Biol 2010; 189: 201&#226;&#8364;&#8220;210.                                                                               37. Wu D, Pan W. GSK3: multifaceted kinase Wnt signaling. Trends Biochem Sci 2010; 35: <br />  8. Dorsett D, Strom L. The ancient evolving roles cohesin gene expression DNA                   161&#226;&#8364;&#8220;168. <br />     repair. Curr Biol 2012; 22: R240&#226;&#8364;&#8220;R250.                                                                 38. Lustig B, Jerchow B, Sachs M, Weiler S, Pietsch T, Karsten U et al. Negative feedback loop <br />  9. Rollins RA, Morcillo P, Dorsett D. Nipped-B, Drosophila homologue chromosomal                         Wnt signaling upregulation conductin/axin2 colorectal liver tumors. Mol <br />     adherins, participates activation remote enhancers cut Ultrabithorax genes.               Cell Biol 2002; 22: 1184&#226;&#8364;&#8220;1193. <br />     Genetics 1999; 152: 577&#226;&#8364;&#8220;593.                                                                           39. Buchman JJ, Durak O, Tsai LH. ASPM regulates Wnt signaling pathway activity the <br /> 10. Horsfield JA, Anagnostou SH, Hu JK, Cho KH, Geisler R, Lieschke G et al. Cohesin-                          developing brain. Genes Dev 2011; 25: 1909&#226;&#8364;&#8220;1914. <br />     dependent regulation Runx genes. Development 2007; 134: 2639&#226;&#8364;&#8220;2649.                                  40. Gaston-Massuet C, Andoniadou CL, Signore M, Sajedi E, Bird S, Turner JM et al. Genetic <br /> 11. Dorsett D. Running rings chromosomes trim axons target dendrites. Dev Cell                   interaction homeobox transcription factors HESX1 SIX3 required for <br />     2008; 14: 156&#226;&#8364;&#8220;158.                                                                                         normal pituitary development. Dev Biol 2008; 324: 322&#226;&#8364;&#8220;333. <br /> 12. Monnich M, Banks S, Eccles M, Dickinson E, Horsfield J. Expression cohesin                      41. Martinez-Barbera JP, Rodriguez TA, Beddington RS. The homeobox gene Hesx1 required <br />     condensin genes zebrafish development supports non-proliferative role                         anterior neural ectoderm normal forebrain formation. Dev Biol 2000; 223: 422&#226;&#8364;&#8220;430. <br />     cohesin. Gene Expr Patterns 2009; 9: 586&#226;&#8364;&#8220;594.                                                          42. Andoniadou CL, Signore M, Young RM, Gaston-Massuet C, Wilson SW, Fuchs E et al. <br /> 13. Kawauchi S, Calof AL, Santos R, Lopez-Burks ME, Young CM, Hoang MP et al. Multiple                         HESX1- TCF3-mediated repression Wnt/beta-catenin targets required normal <br />     organ defects transcriptional dysregulation Nipbl( &#195;&#190; /   ) mouse,                      development anterior forebrain. Development 2011; 138: 4931&#226;&#8364;&#8220;4942. <br />     model Cornelia Lange Syndrome. PLoS Genet 2009; 5: e1000650.                                     43. Kim CH, Oda T, Itoh M, Jiang D, Artinger KB, Chandrasekharappa SC et al. Repressor activity <br /> 14. Muto A, Calof AL, Lander AD, Schilling TF. Multifactorial origins heart gut defects nipbl-       Headless/Tcf3 essential vertebrate head formation. Nature 2000; 407: 913&#226;&#8364;&#8220;916. <br />     deficient zebrafish, model Cornelia Lange Syndrome. PLoS Biol 2011; 9: e1001181.               44. Lekven AC, Buckles GR, Kostakis N, Moon RT. Wnt1 wnt10b function redundantly at <br /> 15. Oxtoby E, Jowett T. Cloning zebrafish krox-20 gene (krx-20) expression                      zebrafish midbrain-hindbrain boundary. Dev Biol 2003; 254: 172&#226;&#8364;&#8220;187. <br />     hindbrain development. Nucleic Acids Res 1993; 21: 1087&#226;&#8364;&#8220;1095.                                   45. Dattani MT, Martinez-Barbera JP, Thomas PQ, Brickman JM, Gupta R, Wales JK et al. <br /> 16. Rhodes JM, Bentley FK, Print CG, Dorsett D, Misulovin Z, Dickinson EJ et al. Positive regulation           HESX1: novel gene implicated familial form septo-optic dysplasia. Acta Paediatr <br />     c-Myc cohesin direct, evolutionarily conserved. Dev Biol 2010; 344: 637&#226;&#8364;&#8220;649.                  Suppl 1999; 88: 49&#226;&#8364;&#8220;54. <br /> 17. Ghiselli G. SMC3 knockdown triggers genomic instability p53-dependent apoptosis                 46. Gaston-Massuet C, Kelberman D, Dattani M, Martinez-Barbera JP. Absence SIX3 mutations <br />     human zebrafish cells. Mol Cancer 2006; 5: 52.                                                         patients congenital hypopituitarism. Am J Med Genet A 2009; 149A: 2874&#226;&#8364;&#8220;2876. <br /> 18. Robu ME, Larson JD, Nasevicius A, Beiraghi S, Brenner C, Farber SA et al. p53 activation               47. Hayashi M, Sakamoto K, Kurata K, Nagata J, Satoh J, Morimatsu Y. Septo-optic dysplasia with <br />     knockdown technologies. PLoS Genet 2007; 3: e78.                                                        cerebellar hypoplasia Cornelia Lange syndrome. Acta Neuropathol 1996; 92: 625&#226;&#8364;&#8220;630. <br /> 19. Monnich M, Kuriger Z, Print CG, Horsfield JA. A zebrafish model Roberts syndrome                    48. Roue G, Pichereau V, Lincet H, Colomer D, Sola B. Cyclin D1 mediates resistance to <br />     reveals Esco2 depletion interferes development disrupting cell cycle. PLoS                apoptosis upregulation molecular chaperones consequent redistribution of <br />     One 2011; 6: e20051.                                                                                       cell death regulators. Oncogene 2008; 27: 4909&#226;&#8364;&#8220;4920. <br /> 20. Prince VE, Moens CB, Kimmel CB, Ho RK. Zebrafish hox genes: expression                          49. Wei W, Chua MS, Grepper S, So SK. Blockade Wnt-1 signaling leads anti-tumor <br />     hindbrain region wild-type mutants segmentation gene, valentino.                             effects hepatocellular carcinoma cells. Mol Cancer 2009; 8: 76. <br />     Development 1998; 125: 393&#226;&#8364;&#8220;406.                                                                        50. He B, You L, Uematsu K, Xu Z, Lee AY, Matsangou M et al. A monoclonal antibody against <br /> 21. Krauss S, Johansen T, Korzh V, Fjose A. Expression pattern zebrafish pax genes                          Wnt-1 induces apoptosis human cancer cells. Neoplasia 2004; 6: 7&#226;&#8364;&#8220;14. <br />     suggests role early brain regionalization. Nature 1991; 353: 267&#226;&#8364;&#8220;270.                             51. Gomez-Sintes R, Hernandez F, Lucas JJ, Avila J. GSK-3 mouse models study neuronal <br /> 22. Molven A, Njolstad PR, Fjose A. Genomic structure restricted neural expression                  apoptosis neurodegeneration. Front Mol Neurosci 2011; 4: 45. <br />     zebrafish wnt-1 (int-1) gene. EMBO J 1991; 10: 799&#226;&#8364;&#8220;807.                                                52. Doe CQ. Neural stem cells: balancing self-renewal differentiation. Development 2008; <br /> 23. Millimaki BB, Sweet EM, Dhason MS, Riley BB. Zebrafish atoh1 genes: classic proneural                      135: 1575&#226;&#8364;&#8220;1587. <br />     activity inner ear regulation Fgf Notch. Development 2007; 134: 295&#226;&#8364;&#8220;305.             53. Tsai PT, Hull C, Chu Y, Greene-Colozzi E, Sadowski AR, Leech JM et al. Autistic-like behaviour <br /> 24. Roussa E, Krieglstein K. Induction specification midbrain dopaminergic cells: focus                 cerebellar dysfunction Purkinje cell Tsc1 mutant mice. Nature 2012; 488: 647&#226;&#8364;&#8220;651. <br />     SHH, FGF8, TGF-beta. Cell Tissue Res 2004; 318: 23&#226;&#8364;&#8220;33.                                          54. Yochum CL, Bhattacharya P, Patti L, Mirochnitchenko O, Wagner GC. Animal model of <br /> 25. Shimizu T, Yamanaka Y, Ryu SL, Hashimoto H, Yabe T, Hirata T et al. Cooperative roles                   autism using GSTM1 <span id='am-315' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-316' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice early post-natal sodium valproate treatment. <br />     Bozozok/Dharma Nodal-related proteins formation dorsal organizer                      Behav Brain Res 2010; 210: 202&#226;&#8364;&#8220;210. <br />     zebrafish. Mech Dev 2000; 91: 293&#226;&#8364;&#8220;303.                                                                 55. Mouri K, Horiuchi SY, Uemura T. Cohesin controls planar cell polarity regulating the <br /> 26. Wallingford JB, Habas R. The developmental biology Dishevelled: enigmatic protein                    level seven-pass transmembrane cadherin Flamingo. Genes Cells 2012; 17: 509&#226;&#8364;&#8220;524. <br />     governing cell fate cell polarity. Development 2005; 132: 4421&#226;&#8364;&#8220;4436.                               56. Ghiselli G, Coffee N, Munnery CE, Koratkar R, Siracusa LD. The cohesin SMC3 target <br /> 27. Ciemerych MA, Kenney AM, Sicinska E, Kalaszczynska I, Bronson RT, Rowitch DH et al.                        beta-catenin/TCF4 transactivation pathway. J Biol Chem 2003; 278: 20259&#226;&#8364;&#8220;20267. <br />     Development mice expressing single D-type cyclin. Genes Dev 2002; 16:                             57. Kimmel CB, Ballard WW, Kimmel SR, Ullmann B, Schilling TF. Stages embryonic <br />     3277&#226;&#8364;&#8220;3289.                                                                                                 development zebrafish. Dev Dyn 1995; 203: 253&#226;&#8364;&#8220;310. <br /> 28. Staal FJ, Noort MvM, Strous GJ, Clevers HC. Wnt signals transmitted                        58. Thisse C, Thisse B, Schilling TF, Postlethwait JH. Structure zebrafish snail1 gene <br />     N-terminally dephosphorylated beta-catenin. EMBO Rep 2002; 3: 63&#226;&#8364;&#8220;68.                                       expression wild-type, spadetail tail mutant embryos. Development 1993; <br /> 29. Joly JS, Joly C, Schulte-Merker S, Boulekbache H, Condamine H. The ventral                             119: 1203&#226;&#8364;&#8220;1215. <br />     posterior expression zebrafish homeobox gene eve1 perturbed dorsalized                59. Panzer JA, Gibbs SM, Dosch R, Wagner D, Mullins MC, Granato M et al. Neuromuscular <br />     mutant embryos. Development 1993; 119: 1261&#226;&#8364;&#8220;1275.                                                          synaptogenesis wild-type mutant zebrafish. Dev Biol 2005; 285: 340&#226;&#8364;&#8220;357. <br /> 30. Russo S, Masciadri M, Gervasini C, Azzollini J, Cereda A, Zampino G et al. Intragenic              60. Nasevicius A, Ekker SC. Effective targeted gene &#226;&#8364;&#732;knockdown&#226;&#8364;&#8482; zebrafish. Nat Genet 2000; <br />     large NIPBL rearrangements revealed MLPA Cornelia Lange patients. Eur J Hum                       26: 216&#226;&#8364;&#8220;220. <br />     Genet 2012; 20: 734&#226;&#8364;&#8220;741. <br /> 31. Adolf B, Bellipanni G, Huber V, Bally-Cuif L. atoh1.2 beta3.1 new bHLH- <br />     encoding genes expressed selective precursor cells zebrafish anterior hindbrain.                            Cell Death Disease open-access journal <br />     Gene Expr Patterns 2004; 5: 35&#226;&#8364;&#8220;41.                                       <br /> </body></html>