<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Novel Role Lh3 Dependent ECM Modifications <br /> during Neural Crest Cell Migration Zebrafish <br /> Santanu Banerjee, Jesse Isaacman-Beck, Valerie A. Schneider&#194;&#164;, Michael Granato* <br /> Department Cell Developmental Biology, University Pennsylvania School Medicine, Philadelphia, Pennsylvania, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      During vertebrate development, trunk neural crest cells delaminate entire <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> dorsal neural tube and <br />      initially migrate non-segmented sheet. As enter somites, neural crest cells rearrange spatially restricted <br />      segmental streams. Extracellular matrix components likely play critical roles transition sheet-like <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />      stream-like mode migration, extracellular matrix components modifying enzymes critical this <br />      transition largely unknown. Here, identified glycosyltransferase Lh3, known modify extracellular matrix <br />      components, presumptive substrate Collagen18A1, provide extrinsic signals critical neural crest cells to <br />      transition sheet-like migration behavior migrating segmental stream. Using live cell imaging in <br />      lh3 <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>, neural crest cells fail transition sheet stream, consequently enter somites <br />      multiple streams, stall shortly entering somites. Moreover, demonstrate transgenic expression lh3 <br />      small subset somitic cells adjacent neural crest cells switch sheet stream migration restores <br />      segmental neural crest cell migration. Finally, knockdown presumptive Lh3 substrate Collagen18A1 <br />      recapitulates neural crest cell migration defects observed lh3 mutants, consistent notion Lh3 exerts its <br />      effect neural crest cell migration regulating post-translational modifications Collagen18A1. Together data <br />      suggest Lh3&#226;&#8364;&#8220;Collagen18A1 dependent ECM modifications regulate transition trunk neural crest cells non- <br />      segmental sheet like migration mode segmental stream migration mode. <br />  <br />   Citation: Banerjee S, Isaacman-Beck J, Schneider VA, Granato M (2013) A Novel Role Lh3 Dependent ECM Modifications Neural Crest Cell Migration in <br />   Zebrafish. PLoS ONE 8(1): e54609. doi:10.1371/journal.pone.0054609 <br />   Editor: Moises Mallo, Instituto Gulbenkian Cie&#203;&#8224;ncia, Portugal <br />   Received September 14, 2012; Accepted December 14, 2012; Published January 18, 2013 <br />   Copyright: &#195;&#376; 2013 Banerjee et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work funded United States National Institutes Health grant HD037975 NSF grant 0920336 M.G. The funders role in <br />   study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: granatom@mail.med.upenn.edu <br />   &#194;&#164; Current address: National Center Biotechnology Information, National Library Medicine, National Institutes Health, Bethesda, Maryland, United States of <br />   America <br />  <br />  <br />  <br /> Introduction                                                                            restrict neural crest cells migration defined region the <br />                                                                                         somite. Although extracelluar matrix (ECM) components <br />    During vertebrate development, trunk neural crest cells de-                          shown localize segmental path they <br /> laminate entire rostro-caudal axis neural tube,                      exert permissive, pro migratory roles [17], ECM compo- <br /> and continue specific migratory routes [1]. Neural crest                     nents required specific aspects segmental neural cell <br /> cells delaminate later development enter ventro-                          migration vivo identified. Similarly, enzymes <br /> lateral pathway dermomyotome epidermis                              modify ECM components post-translationally thereby <br /> [2,3]. Early delaminating neural crest cells choose ventral route                     provide unique properties regulate neural crest cell <br /> through intersomitic space intersegmental blood vessels,                      behaviors largely unknown. Zebrafish provide attractive <br /> and somites neural tube pathway                             identify role ECM components their <br /> called ventro-medial pathway [4]. Importantly, neural crest                         modifications neural crest cell migration [18]. <br /> cells entering ventromedial pathway converge broader                            In zebrafish, neural crest cells migrate restricted <br /> region neural tube distinct segmentally organized                        region somite located mid-segmentally two <br /> streams. In amniotes rodents, streams extend                      adjacent somite/segment boundaries [19,20]. We recently <br /> rostral portion somite subsequently travel                            shown secreted glycoprotein Wnt11r binds Muscle <br /> alongside ventrally projecting spinal nerves [5,6,7]. Ablation                     specific kinase (MuSK) induce Dishevelled dependent <br /> experiments zebrafish shown segmental neural crest                         signaling cascade adaxial muscle cells. In embryos compromised <br /> cell migration occur independent spinal motor axons,                             Wnt11r-MuSK-Dishevelled signaling, neural crest cells stray <br /> suggesting cues direct migration [8].                                   away mid-segmental region [8]. However, these <br />    It established somite derived signals direct segmental                  embryos, segregation neural crest cells mid-segmental <br /> neural crest cell migration [9,10]. These include Ephrin/Eph                            streams remains unaffected. This observation raised possibility <br /> receptor dependent signals [11,12,13], Semaphorin3-Neuropilin                           additional, somite derived signaling pathways regulate the <br /> dependent signals [14,15,16] Wnt-MuSK dependent signals                             transition neural crest cells migrating sheet segmentally <br /> [8], thought provide inhibitory and/or repulsive forces                       repeated streams. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                          1                               January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                                        ECM Modifications Neural Crest Cell Migration <br />  <br />  <br />    Lh3 (Lysyl hydroxylase 3, 2-oxoglutarate 5-dioxygenase 3 <br /> PLOD3), multifunctional enzyme catalyzes post- <br /> translational addition galactosyl glucosyl moieties onto <br /> collagens proteins collagen-like domains [21]. <br /> Collagens, depending particular subtypes, function as <br /> either permissive non permissive substrates neural crest cells, <br /> however complement vivo roles neural crest <br /> cell migration defined [17]. Here, present genetic <br /> evidence Lh3 dependent signaling pathway acts cell-non <br /> autonomously neural crest cells transition sheet-like <br /> to stream-like mode migration. In lh3 mutant embryos neural <br /> crest cells fail transition, leading ectopic stalled migration. <br /> We demonstrate neural crest cell migration defects <br /> occur independently motor axon guidance defects observed <br /> in lh3 mutants, lh3 activity subset muscles <br /> cells sufficient proper neural crest cell migration. Finally, we <br /> show morpholino mediated knockdown collagen18a1 leads <br /> to neural crest cell migration defects similar observe lh3 <br /> mutants. We propose Lh3 dependent ECM modifications, <br /> including Collagen18A1, regulate behavioral <br /> transition trunk neural crest cells non-segmental, sheet <br /> like mode segmental, stream like mode. <br />  <br /> Results <br /> Lh3 Required Segmental Neural Crest Cell Migration <br />    In zebrafish neural crest cells delaminate dorsal neural <br /> tube enter ventro-medial pathway uninterrupted, <br /> non-segmented group cells (Fig. 1 A B) [20]. Given sheet <br /> like appearance previously described birds mammals, we <br /> will refer mode non-segmented migration sheet like <br /> migration [22]. At interface neural tube and <br /> notochord, neural crest cells converge discreet segmental <br /> streams, stream migrating ventrally narrow path in <br /> the center somite (Fig. 1 A, C, D). We refer these <br /> individual streams &#226;&#8364;&#732;mid-segmental&#226;&#8364;&#8482; streams. As continue <br /> their migration, neural crest cell streams share path spinal <br /> motor axons (Fig. 1 C D). <br />    In lh3tv205/tv205 mutants (referred lh3 hereafter), neural crest <br /> cells delaminated properly dorsal neural tube and <br /> migrated ventrally. However, interface neural tube <br /> and notochord, wildtype neural crest cells converge into <br /> discreet mid-segmental stream, significant fraction lh3 neural <br /> crest cells failed coalesce segmental streams instead <br /> remained organized continuous sheet anterior <br /> posterior axis (asterisks Fig. 1E, F asterisks; 25%, n = 80 <br /> hemisegments). Although lh3 mutant neural crest cells <br /> entered mid-segmental path, stalled shortly after <br /> entering path (arrowheads Figure 1E, F 50%, n = 80 <br /> hemisegments), migrated lateral somite territories <br /> typically devoid neural crest cells (arrows Fig. 1E, F; 25%, <br /> n = 80 hemisegments). <br />    Based static images unable determine <br />                                                                                Figure 1. The role lh3 signaling neural crest cell migration. <br /> whether mutant neural crest cells lateral somite                  (A) Lateral view 22 hour old embryo stained pan-neural <br /> entered midsegmental path strayed away                 crest cell marker crestin (riboprobe, green) marking neural crest cells, <br /> they entered mid-segmental stream instead entered                F59 antibody marking adaxial muscle cells. (B) Schematic <br /> the somites following ectopic route. To distinguish                 diagram showing modes zebrafish neural crest cell migration: 1) <br /> these possibilities, performed live imaging wildtype (lh3            sheet-like, non segmental 2) stream-like, segmental. Neural crest <br />                                                                                cells (green) migrate central region somite after <br /> siblings) lh3 mutant embryos expressing membrane bound <br />                                                                                forming segmental streams (red). White horizontal dashed lines A, B, <br /> RFP neural crest cells (Tg[sox10:mRFP]vu234) GFP motor               C E indicate approximate position ventral boundary the <br /> neurons (Tg[mnx1:gfp]ml2). In wildtype embryos neural crest cells              neural tube. (C D) Lateral view 28 hpf embryo stained with <br /> migrate single stream mid-segmental region, remaining              crestin (green) motor neuron maker antibodies (Znp1 SV2; <br /> in close association motor axons (Fig. 1 G&#226;&#8364;&#8220;I). In contrast,               red). Horizontal dotted lines A indicate ventral boundary the <br /> analysis lh3 mutant embryos revealed                     neural tube. E F lateral views lh3 mutant embryos stained <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 2                            January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                                        ECM Modifications Neural Crest Cell Migration <br />  <br />  <br />  <br /> with crestin motor neuron markers. Arrows E F point to <br /> neural crest cells near somite boundaries. Asterisks marks neural <br /> crest cells forming sheet adjacent hemisegments. G&#226;&#8364;&#8220;L <br /> shows images time lapse movies recorded embryos <br /> expressing membrane bound RFP (Tg[sox10:mrfp]) neural crest cells <br /> and GFP motor neurons (Tg[mnx1:gfp]). In wildtype embryos (G&#226;&#8364;&#8220;I), <br /> neural crest cells migrate mid-segmentally motor <br /> axons. In lh3 mutant embryos (J&#226;&#8364;&#8220;L), neural crest cells migrate with <br /> motor axons mid-segmentally region near segment/somite <br /> boundary (arrows). Neural crest cells lh3 mutants acquire <br /> a rounded morphology (asterisks). Oblique dotted lines C L <br /> indicate position somite/segment boundary. Scale bars:10 <br /> micron. <br /> doi:10.1371/journal.pone.0054609.g001 <br />                                                                                 Figure 2. Motor axon independent role lh3 signaling in <br />                                                                                 neural crest cell migration. (A D) Lateral view double <br /> following migration defects (10/11 somitic segments 4                      transgenic embryos (Tg[sox10:mrfp];Tg[mnx1;gfp] expressing mRFP in <br /> embryos). In 6/10 affected somite segments, lh3 mutant neural                   neural crest cells GFP neurons. Pre ablation (A) post ablation <br /> crest cells correctly coalesced mid-segmental stream,                (A&#226;&#8364;&#8482;) view motor neurons neural crest cells wildtype embryos. <br /> eventually stalled (data show), consistent end-point               Neural crest cells able migrate mid-segmental <br /> analyses (Fig. 1E). In 4 11 somite segments analyzed, lh3                region 4 hours following motor neuron ablation (B). In lh3 mutants (C- <br />                                                                                 pre ablation, C&#226;&#8364;&#8482;230 min post ablation, D- 4 hours post ablation), neural <br /> mutant neural crest cells entered somite ectopic <br />                                                                                 crest cells migrated region overlying somite/segment boundary <br /> route, lateral mid-segmental path (Fig. 1J, arrow). Over             absence motor neurons. Scale bar- 10 micron. <br /> next hours misguided neural crest cells continued                  doi:10.1371/journal.pone.0054609.g002 <br /> their ectopic path close proximity somite boundary <br /> (Fig. 1K, L, arrows), path observed wildtype embryos.                Lh3 Function Adaxial Muscle Neural Crest Cell <br /> Finally, noticed lh3 mutants individual neural crest cells <br /> frequently lacked filopodial protrusions, characteristic wildtype <br />                                                                                 Convergence <br />                                                                                    During time period neural crest cell migration, lh3 <br /> neural crest cells mid-segmental stream. Instead, lh3 <br />                                                                                 mRNA detectable notochord small sub- <br /> mutant neural crest cells exhibited round morphology <br />                                                                                 population somitic muscle cells known adaxial cells [25]. <br /> (Fig. 1K, L, asterisks). Thus, lh3 mutants neural crest cells either <br />                                                                                 These adaxial cells decorate future mid-segmental path, and <br /> enter mid-segmental path stall, fail organize into <br />                                                                                 genetic studies shown absence adaxial cells, <br /> mid-segmental streams. Instead, mutant cells initially accumulate <br />                                                                                 neural crest cells fail form segmental streams [26]. Therefore, <br /> where wildtype cells transition segmental streams, then <br />                                                                                 asked lh3 activity adaxial cells functionally important for <br /> frequently enter somites entire length the <br />                                                                                 neural crest cells organize segmental streams. To test this, <br /> somite, forming ectopic, non-segmental streams. <br />                                                                                 generated transgenic zebrafish lines expressing Myc epitope <br />                                                                                 tagged length wildtype lh3 cDNA control an <br /> Neural Crest Cell Defects lh3 Mutants Occur                                  adaxial muscle specific promoter [27] (Tg[smhc1: lh3-myc]p161). In <br /> Independent Motor Axons                                                      transgenic Tg[smhc1: lh3-myc]p161 embryos lh3-myc expression is <br />    Along ventro-medial route somites, zebrafish               detectable adaxial cells time period neural crest <br /> neural crest cells share migration path spinal motor axons               cells begin migrate ventro-medial pathway (Fig. 3A). <br /> [8,23]. We previously shown misguided motor                     Although levels lh3-myc expression vary individual <br /> axons redirect neural crest cells, motor axons dispensable              adaxial muscle cells, confirmed 90% adaxial <br /> for neural crest cells enter migrate mid-segmental              muscle cells, identified expression Prox-1 [28,29], also <br /> path [8]. In lh3 mutants, motor axons fail exit spinal              express lh3-myc (Fig. 3A). This transgenic line crossed lh3 <br /> cord, exit stall soon [24]. To determine           mutants, neural crest cell migration analyzed the <br /> neural crest cell migration defects observed lh3 mutants                 offspring intercrosses (lh3/+; Tg[smhc1:lh3-myc]) <br /> caused motor axons failing migrate properly, examined                  animals. Unlike wildtype embryos neural crest cells <br /> lh3 mutant neural crest cell migration following motor neuron                   entered migrated somites mid- <br /> ablation.                                                                       segmental path, lh3 mutant embryos neural crest cells either <br />    As previously reported, absence motor neurons                      entered mid-segmental path stalled (,50% of <br /> wildtype neural crest cells enter migrate mid-                    hemisegments, n = 60; Fig. 3C, arrow), entered and <br /> segmental path (Fig. 2 A&#226;&#8364;&#8220;A&#226;&#8364;&#8482;&#226;&#8364;&#8482;&#226;&#8364;&#8220;B; n = 4/7). Ablation motor                     migrated somites ectopic, non-segmental <br /> neurons lh3 mutants did ameliorate enhance                   path (,30% hemisegments, n = 60; Fig. 3C, asterisk). In <br /> neural crest cell migration defects observed lh3 mutants. In                 contrast, transgenic expression lh3-myc adaxial muscle <br /> motor neuron ablated lh3 mutants, neural crest cells frequently                 restored neural crest cell migration significantly lh3 <br /> entered mid-segmental path (n = 4/10; Fig. 2 C&#226;&#8364;&#8220;C&#226;&#8364;&#8482;&#226;&#8364;&#8482;&#226;&#8364;&#8220;                 mutant embryos (Fig. 3D, quantified E). In embryos, <br /> D), stalled, entered extended somites              ectopic, non-segmental neural crest cell migration restored to <br /> ectopic, non-segmental routes (n = 3/10). Thus, failure lh3                  wildtype levels, suggesting adaxial lh3 expression <br /> mutant neural crest cells coalesce segmental streams                 regulates primarily transition trunk neural crest cells <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> independent motor axons. This suggests ability                   non-segmental sheet-like migration mode mid-segmental <br /> neural crest cells converge single mid-segmental stream            stream migration mode. Thus, lh3 regulates cell non-autonomous- <br /> directly dependent lh3 activity, intrinsically                 ly conversion form sheet-like, non-segmental segmental <br /> migratory neural crest cell, provided extrinsically                   neural crest cell migration. <br /> embryonic environment. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  3                            January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                                     ECM Modifications Neural Crest Cell Migration <br />  <br />  <br />  <br />  <br /> Figure 3. Lh3 activity adaxial cells rescue neural crest cell migration defects lh3 mutants. Cross sectional view 16 somite <br /> stage [Tg(smhc1:lh3-myc)] embryos showing expression lh3-myc transgene (green) prox-1 (red) positive adaxial cells. Lateral view 28 hour <br /> old lh3 sibling embryo (B), lh3 mutant (C) lh3 mutant embryo expressing lh3-myc adaxial muscle cells (D), stained crestin (green). <br /> Note stalled neural crest cells mid-segmental neural crest cells hemisegment region lh3 mutants (arrows asterisks C). <br /> These defects rescued expressing lh3-myc adaxial cells lh3 mutants (C). (E) Quantification neural crest cell migration defects. P <br /> values (** p,0.001; * p,0.05) obtained using Chi- Square test. Scale bar-10 micron. <br /> doi:10.1371/journal.pone.0054609.g003 <br />  <br />  <br /> Collagen18A1 Plays Role Neural Crest Cell Migration                        Importantly, collagen18a1 MO injection did significantly <br />    We examined lh3 influences neural crest cell                     affect adaxial cell morphology differentiation (Fig. 4F, G). <br /> migration. Lh3 multifunctional enzyme catalyzes                Further analyses using adaxial cell specific Prox-1 marker <br /> post-translational addition glycosyl moieties primarily              [28,29] revealed adaxial cell number unchanged in <br /> collagens [21], modifying extracellular matrix. We           collagen18a1 MO injected embryos (Fig. 4 H I). Moreover, in <br /> previously shown early neural crest cell migration, lh3          wildtype embryos subset adaxial cell nuclei level the <br /> and collagen18a1 co-expressed adaxial muscle cells [25],              horizontal myoseptum located close anterior somite <br /> suggesting model Lh3 influence neural crest cell            boundary, stereotyped position unaffected in <br /> migration post-translational modifications Collagen18A1.              collagen18a1 morpholino injected embryos (arrowheads, Fig. 4 H <br /> Importantly, Collagen18A1 belongs multiplexin group                I). Finally, co-injected MO1 MO2 synthetic <br /> non-fibrillar collagens contains cysteine rich domain (CRD)            mRNA encoding full-length collagen18a1. In double injected <br /> homologous WNT binding frizzled receptors,                    embryos, neural crest migration restored wildtype <br /> a unique thrombospondin (TSPN) domain, commonly                     levels, confirming neural crest cell phenotypes observed in <br /> the ectodomain semaphorins guidance receptors [30].             collagen18a1 MO injected embryos caused knockdown of <br /> Moreover, collagen18a1 previously shown critical axon          collagen18a1 function (Fig 4 D, quantified E). Thus, knockdown <br /> guidance C.elegnas, Drosophila zebrafish [25,31,32]. There-           collagen18a1 recapitulates neural crest migration defects <br /> fore, asked collagen 18a1 function important               observed lh3 mutants, albeit lower penetrance, suggesting <br /> neural crest cell migration. For used combination           Lh3 acts Collagen18A1 guide neural crest <br /> splice blocking morpholinos (MOs) knock collagen18a1                 cell migration. <br /> function. We previously shown MO1 blocks splicing of <br /> collagen18a1 mRNA [25]. We designed second spice blocking             Discussion <br /> MO, MO2, effective reducing collegen18a1 <br />                                                                                 Segmental migration neural crest cells dependent on <br /> transcript level injected embryos (Fig. 4A). In embryos injected <br />                                                                              number studied guidance cues act cognate <br /> with vehicle (Danio buffer) solution, migration <span id='am-162' about='xsp:pattern' typeof='owl:Thing'>pattern</span> neural <br />                                                                              receptors present neural crest cells, well <br /> crest cells indistinguishable observed wild type <br />                                                                              characterized ECM components deposited migration <br /> embryos (Fig. 4 B). In embryos injected combination of <br />                                                                              path [17]. For example, mammals, Nrp2-Sema3F signaling has <br /> collagen18a1 MO1 MO2 dose causing minimal lethality <br />                                                                              shown organize neural crest cells segmental streams <br /> (,10%) obvious morphological defects, neural crest cell <br />                                                                              [14,15]. Although distribution ECM components the <br /> migration defects identical observed lh3 <br />                                                                              migratory path neural crest cells suggestive specific <br /> mutants. Specifically, collagen18a1 MO1 MO2 injected <br />                                                                              functions migration, ECM components <br /> embryos neural crest cells entered mid-segmental path <br />                                                                              modifications alter migratory patterns neural crest <br /> but stalled (,25%, n = 80 hemisegments; arrows Fig. 4C, <br />                                                                              cells unclear. Here use genetic mutants, live cell <br /> quantified Fig. 4E), entered migrated through <br />                                                                              imaging transgenic rescue examine function defined <br /> the somites ectopic, non-segmental path (,15%, n = 80 <br />                                                                              ECM components ECM modifying enzymes migration <br /> hemisegments; asterisks Fig. 4C, quantified Fig. 4E). <br />                                                                              zebrafish trunk neural crest cells. Specifically, in <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               4                           January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                                         ECM Modifications Neural Crest Cell Migration <br />  <br />  <br />  <br />  <br /> Figure 4. Knockdown collagen18a1 results neural crest cell migration defects. (A) RT PCR analysis showing efficiency collagen18a1 <br /> knockdown following morpholino treatment. Arrow marks expected band following morpholino treatment sized 439 <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span> pairs. Lateral views <br /> of 28 hpf vehicle (B) collagen18a1 MO (C) collagen18a1 MO plus collagen18a1 mRNA injected embryo (D), stained crestin visualize neural <br /> crest cells. Arrows indicate neural crest cells stalled mid-segmental path (C), asterisks indicate neural crest cells ectopic path. <br /> (C, E) (E) Quantification neural crest cell migration defects collagen18a1 MO injected embryos. p values calculated using tailed Fisher <br /> Exact Probability test. Vehicle injected collagen18a1(G) MO injected embryos, stained F59 visualize adaxial cells (red), crestin to <br /> visualize neural crest cells (green) (F&#226;&#8364;&#8220;G) prox-1 antibody (H&#226;&#8364;&#8220;I). Arrowheads mark adaxial cell nuclei located anteriorly near horizontal <br /> myoseptum region (H&#226;&#8364;&#8220;I). Scale bar-10 micron. <br /> doi:10.1371/journal.pone.0054609.g004 <br />  <br /> lh3 mutants collagen18a morphants neural crest cells frequently             Lh3 activity provided small group somitic <br /> enter fail migrate mid-segmental path                 cells, adaxial muscle cells, positioned neural crest cells <br /> somites, instead enter continue migrate                 organize segmental streams. Combined, data identify an <br /> somite territories typically devoid neural crest cells. Moreover,            Lh3-Collagen18A1 dependent pathway provides extrinsic <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  5                             January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                                   ECM Modifications Neural Crest Cell Migration <br />  <br />  <br /> signals required alter migratory behavior neural crest cells       possible Collagen18A1 substrates <br /> as organize segmental streams.                                     Lh3 mediates neural crest cell migration. <br />  <br /> Lh3 Plays Novel, Motor Axon Independent Function                       How does Lh3 Collagen18A1 Influence Neural Crest <br /> Neural Crest Cell Migration                                                  Cell Migration? <br />    Neural crest cells, following delamination neural tube              There potential mechanisms which <br /> enter migratory pathways, known dorso-lateral              Collagen18A1 influence segmental neural crest cell <br /> and ventro-medial pathway, respectively. Along ventro-               migration. First, C terminus Collagen18A1 contains an <br /> medial pathway neural crest cells undergo intriguing re-                  endostatin domain previously shown inhibit epithelial cell <br /> organization transform non-segmented sheet cells           migration [35]. It conceivable Lh3 modification of <br /> into distinct segmented streams, continue               Collagen18A1 generates non-permissive territories sides <br /> migration center somite. In lh3 mutants,                 mid-segmental path, inhibiting neural crest cells to <br /> neural crest cells enter mid-segmental path           stray mid-segmental path. Loss lh3 collagen18a1 <br /> then stall, ignore mid-segmental path enter somites           make territories permissive neural crest cells, <br /> at ectopic positions, demonstrating lh3 critical segmental       resulting ectopic migratory streams. However, such <br /> neural crest cell migration.                                                 inhibitory migratory function Collagen18A1 compatible <br />    In zebrafish, neural crest cells undergo transition non-       frequent stalling neural crest cells mid- <br /> segmented sheet cells distinct segmented streams          segmental path observed collagen18a1 morphants lh3 <br /> ventral extend spinal cord, adjacent adaxial cells (Fig. 1A        mutants. Thus, Collagen18A1 simultaneously promote <br /> B). Transgenic expression lh3 adaxial cells rescued               neural crest cell migration, similar documented for <br /> migratory defects, stalling segmental path migration           neuronal migration [31]. <br /> along ectopic trajectories, albeit different extent. Although           Second, Collagen18A1 influence neural crest cell <br /> can exclude possibility adaxial cell specific               migration interactions known positive negative <br /> transgene produces Lh3 protein levels produced                migratory cues. For example, Collagen18A1 influence <br /> endogenously adaxial cells, difference rescue levels        neural crest cell migration WNT binding cysteine rich <br /> consistent idea Lh3 exhibits distinct functions        domain (CRD) and/or thrombospondin domain, commonly <br /> neural crest cell migration. First, adaxial cell dependent                ectodomains semaphorin guidance cues [30]. <br /> function provides extrinsic signals re-organize migration         Alternatively, Collagen18A1 bind thrombos- <br /> neural crest cells non-segmented sheet cells distinct         pondin domain Integrin receptors present neural crest cells, <br /> and segmented streams. Secondly, Lh3 functions support                 providing permissive substrate migration [36]. <br /> promote neural crest cell migration segmental path,                Future structure-function studies identify Collagen18A1 <br /> although function appears provided partially              domain(s) required neural crest cell migration greatly <br /> adaxial cells, transgenic expression failed fully restore              aid reveal molecular mechanisms Collagen18A1 <br /> migration segmental path. Importantly, functions              exerts function migrating neural crest cells. <br /> appear independent presence motor axons (Fig. 2)                  Independent precise mechanism Collagen18A1 <br /> both functions require, part, Collagen18A1 function.             influences neural crest cell migration, data consistent with <br />                                                                              model Lh3 activity adaxial muscle cells modifies <br /> Collagen18A1, Multifunctional ECM Component                            ECM components turn influence cytoskeletal remodeling <br /> Presumptive Lh3 Substrate Important Segmental                         neural crest cells migration. Recently, Lh3 been <br /> Neural Crest Cell Migration                                                  shown regulate organization extracellular matrix <br />    One possible scenario Lh3 regulates neural crest cell            components Fibronectin Collagens [37]. Consequent- <br /> migration modification ECM components. Lh3              ly, ECM modifications induce changes cytoskeletal <br /> an ER resident glycosylating enzyme adds O-linked glycosyl              dynamics appearance motile cells. Consistent with <br /> residues collagens collagen domain containing proteins                this, fibroblasts derived lh3 deficient mice exhibit more <br /> [21]. Although Lh3 detected extracellular space               round morphology [37], similar morphology observe in <br /> [33], previously shown removing ER retention                neural crest cells lh3 mutants collagen18a1 morphants. Thus, <br /> signal lh3 coding sequence abolishes lh3&#226;&#8364;&#8482;s ability               conceivable lh3 expression adaxial cells regulates the <br /> restore motor axon guidance. We did examined secreted               modification distribution ECM components, including <br /> form Lh3 rescues neural crest cell migration, favor idea          Collagen18A1, providing extrinsic signals required alter <br />                                                                              migratory behavior neural crest cells organize into <br /> that Lh3 acts substrates, including Collagen18A1. <br />                                                                              segmental streams. <br />    The best characterized targets Lh3 collagens [34], we <br /> have previously shown zebrafish Collagen18A1, member of <br /> the multiplexin family collagens, transiently expressed             Materials Methods <br /> adaxial cells time neural crest cell migration [25].           Ethics Statement <br /> Knockdown collagen18a1 results neural crest cell migration <br />                                                                                All experiments conducted according Animal <br /> defects similar observed lh3 mutant embryos, suggesting <br />                                                                              Protocol fully approved University Pennsylvania <br /> that Lh3 exert function neural crest cell <br />                                                                              Institutional Animal Care Use Committee (IACUC) on <br /> migration post-translational modifications Collagen18A1. <br />                                                                              September 7, 2010, protocol number 803258.Veterinary care is <br /> Importantly, severity neural crest cell migration defects <br />                                                                              supervision University Laboratory Animal <br /> observed collagen18a1 morphants lower compared those <br />                                                                              Resources (ULAR) University Pennsylvania. <br /> observed lh3 mutants. This simply residual <br /> collagen18a1 function morphant embryos. Alternatively, is <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               6                          January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                               ECM Modifications Neural Crest Cell Migration <br />  <br />  <br />  <br /> Zebrafish Genetics                                                       The amplified product digested NheI subcloned <br />   Zebrafish embryos used study raised 28uC       pCS2C18Ex3CRD generate length myc tagged <br /> required time (see [38]). Wildtype fish used               collagen18a1 cDNA pCS2C18Ex3CRD-myc. <br /> experiments TLF, mutants used diwanka/lh3tv205, <br /> [39], Tg[mnx1:gfp]ml2 [40], Tg[Sox10:mrfp]vu234 [41],Tg[smhc1:lh3-       mRNA Preparation Injection <br /> myc]p161.Transgenic fish used alone, combination             Full length myc tagged collagen18a1 mRNA from <br /> with combination lh3tv205 mutant                   pCS2C18Ex3CRD-myc vector vitro transcription using <br /> background.                                                              Ambion mMESSAGE mMachine SP6 kit (Product # <br />                                                                          AM1340). A 500 ng/uL working stock diluted 0.1M KCl <br /> Molecular Biology                                                        DEPC water final concentration 20 pg injected <br />   lh3-5x myc (referred lh3-myc) cloned previously pCS2+        embryo. <br /> vector [25]. lh3-5x myc cloned ISce-I vector downstream of <br /> smhc1 promoter using Xba/Spe1 sites.                                     Immunohistochemistry <br />                                                                             Antibody staining performed described previously [43]. <br /> Anti Sense Morpholino Analysis                                           The following primary antibodies used: znp-1 (1:200) [44]; <br />    collagen18a1 Morpholino 1:59 - TAACAACCTACCTGGATA-                    Antibody Facility, University Oregon; SV2 (1:50, Developmen- <br /> GAGCCTT - 39 [25]. collagen18a1 Morpholino 2:59 -                        tal Studies Hybridoma Bank, University Iowa), myc (9E10, <br /> CAGTGCTCAACACACCTTGTCTCC - 39 (This spice                           1:1000,Covance), GFP (JL-8, 1-100,Clonetech), prox-1 (1:2000). <br /> blocking morpholino interferes transcription           Antibodies visualized Alexa-Fluor-594 conjugated <br /> Exon 8 Intron 8 CRD containing variant                     secondary antibodies (1:500; Molecular Probes, Eugene,OR). In <br /> collagen18a1). A total 7 ng (MO1-5 ng; MO2- 2 ng)                  situ hybridization crestin probe [45] performed as <br /> collagen18a1 MO cocktail injected embryo. For RT            described [25]. <br /> PCR analysis, 5 uninjected 5 morpholino-2 treated embryos <br /> were used make cDNA. PCR performed using forward <br />                                                                          Live Cell Imaging <br /> primer Exon 5 (59-GTGTTATTGGTGATCTGAGGGTGT- <br />                                                                             Sixteen somite stages embryos briefly anesthe- <br /> 39)          reverse    primer      Exon     10   (59- <br />                                                                          tized using tricane mounted laterally 1% low melting <br /> TCCTCTTGGTCCTGTGACTTTCTG-39) collagen18a1 <br />                                                                          agarose prepared Ringer&#226;&#8364;&#8482;s solution containing tricane. Images <br /> gene. <br />                                                                          captured 1&#226;&#8364;&#8220;10 min interval using 63x water immersion <br />                                                                          lens spinning disc confocal microscope (Olympus) equipped <br /> Cloning collagen18a1 cDNA <br />                                                                          28uC temperature controlled chamber. Appropriate <br />    A 1.5 kb fragment collagen18a1 cDNA containing sequence <br />                                                                          numbers z sections used create maximum intensity <br /> from Exon 4 stop codon collagen 18a1 cloned into <br />                                                                          projection image using Slidebook (3i) NIH ImageJ. Images were <br /> TOPO blunt vector named Topo-C183&#226;&#8364;&#8482;-end using following <br />                                                                          processed using ImageJ and/or Photoshop. <br /> primers (forward 59CACTCAAGAATGAGATGAAGGGTG39 <br /> and reverse TTCTAGATTGGC GTTACGGCGAATAGA). A <br /> 59 3 kb fragment collagen18a1 encompassing UTR Exon 1-2           Motor Neuron Ablation <br /> were cloned pcrBluntII-TOPO-C18Ex1-2-59 using follow-                Embryos mounted described live imaging. Motor <br /> ing primers (forward 59 AATCGATACCATGG CAA-                              neuron cell bodies expressing GFP ablated using Micro- <br /> GAAGGTGTCTCG revere 59 GGACCTGGAGGTCCAG-                             Point nitrogen pulsed laser (Photonic Instruments) mounted on <br /> GAATACTGAG). pcrBluntII-TOPO-C18 Exon1-2-59 fragments                    spinning disc microscope 63x water immersion objective <br /> were digested Sal1-Xho1 sites release 3 kb fragment       lens. Ablations carried hemisegments per <br /> which subsequently moved Sal1-Xho1 digested Topo-               embryo. Ablations verified 30 minutes, 3 <br /> C183&#226;&#8364;&#8482;end vector generate TOPOC18Exon1-2. A 4.5 kb                     hours. Segments incomplete/partial ablation motor <br /> fragments generated using Cla1 Xb1 sites clone          neurons included analysis. Following ablation of <br /> into pCS2 vector generating pCS2C18Ex1-2 lacking exon 3               motor neurons, migration neural crest cells analyzed either <br /> collagen18a1. The Exon 3 collagen18a1 containing CRD domain           live imaging 3&#226;&#8364;&#8220;5 hours fixing embryo 3&#226;&#8364;&#8220;5 hours post <br /> (1.5 Kb) subcloned TOPOC18Ex3CRD59 using                       ablation. Fixed embryos subsequently analyzed situ <br /> primers (forward 59 AATCGATA CCATGATGGCTGAACT-                           hybridization immunohistochemistry described above. <br /> CAGGT reverse 59TGCTCTCCATCT CCACTTGCTCC <br /> GTAAT). This fragment subcloned pCS2C18Ex1-2               Statistical Analysis <br /> using Cla1- Xba1 site generating length Collagen18a1 cDNA             Chi Square Fisher exact probably tests performed <br /> (pCS2C18Ex3CRD). Finally, 5X myc tag cloned             (one tailed) data presented Figure 3D Figure 4E. <br /> signal peptide newly created unique NheI site track <br /> transgenic expression. Briefly, unique Nhe-1 site created       Acknowledgments <br /> mutating C696T using QuikChange II XL Site-Directed <br /> Mutagenesis Kit following primers: 59TTGAAAGC-                   We like thank members Granato Laboratory their <br /> CAATTTTTAATAGCTAGCATGCCAGCATCC TTC39 (for-                               insightful comments manuscript. <br /> ward)            59GAAGGATGCTGGCATGCTAGCTAT- <br /> TAAAAATTGGCTTT CAA39 (reverse). A 5X myc tag lacking                     Author Contributions <br /> a stop codon amplified ISceI-smyhc1:SV1-myc [42]           Conceived designed experiments: SB MG. Performed the <br /> with primers flanked NheI sites: (forward 59AAAAGC-                   experiments: SB VS JIB. Analyzed data: SB MG JIB. Contributed <br /> TAGCCGTAA GGTAAATCGATCGAAA39 reverse                                 reagents/materials/analysis tools: SB VS MG. Wrote paper: SB JIB <br /> 59AAAAGCT           AGCGGTGAGGTCGCCCTTGCTCTC39).                         MG. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                           7                          January 2013 | Volume 8 | Issue 1 | e54609 <br />                                                                                                                         ECM Modifications Neural Crest Cell Migration <br />  <br />  <br />  <br />  <br /> References <br />  1. Krull CE (2010) Neural crest cells motor axons avians: Common                 25. Schneider VA, Granato M (2006) The myotomal diwanka (lh3) glycosyltransfer- <br />     distinct migratory molecules. Cell Adh Migr 4.                                               ase type XVIII collagen critical motor growth cone migration. <br />  2. Erickson CA (1985) Control neural crest cell dispersion trunk               Neuron 50: 683&#226;&#8364;&#8220;695. <br />     avian embryo. Dev Biol 111: 138&#226;&#8364;&#8220;157.                                                     26. Honjo Y, Eisen JS (2005) Slow muscle regulates pattern trunk neural crest <br />  3. Erickson CA, Weston JA (1983) An SEM analysis neural crest migration               migration zebrafish. Development 132: 4461&#226;&#8364;&#8220;4470. <br />     mouse. J Embryol Exp Morphol 74: 97&#226;&#8364;&#8220;118.                                                 27. Elworthy S, Lister JA, Carney TJ, Raible DW, Kelsh RN (2003) Transcriptional <br />  4. Loring JF, Erickson CA (1987) Neural crest cell migratory pathways trunk              regulation mitfa accounts sox10 requirement zebrafish melanophore <br />     chick embryo. Dev Biol 121: 220&#226;&#8364;&#8220;236.                                                  development. Development 130: 2809&#226;&#8364;&#8220;2818. <br />  5. Bronner-Fraser M (1986) Analysis early stages trunk neural crest               28. Glasgow E, Tomarev SI (1998) Restricted expression homeobox gene <br />     migration avian embryos using monoclonal antibody HNK-1. Dev Biol 115:                    prox 1 developing zebrafish. Mech Dev 76: 175&#226;&#8364;&#8220;178. <br />     44&#226;&#8364;&#8220;55.                                                                                   29. Roy S, Wolff C, Ingham PW (2001) The u-boot <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span> identifies Hedgehog- <br />  6. Rickmann M, Fawcett JW, Keynes RJ (1985) The migration neural crest cells                 regulated myogenic switch fiber-type diversification zebrafish embryo. <br />     growth motor axons rostral half chick somite.                  Genes Dev 15: 1563&#226;&#8364;&#8220;1576. <br />     J Embryol Exp Morphol 90: 437&#226;&#8364;&#8220;455.                                                       30. Elamaa H, Peterson J, Pihlajaniemi T, Destree O (2002) Cloning three <br />  7. Serbedzija GN, Fraser SE, Bronner-Fraser M (1990) Pathways trunk neural                   variants type XVIII collagen expression patterns Xenopus <br />     crest cell migration mouse embryo revealed vital dye labelling.                 laevis development. Mech Dev 114: 109&#226;&#8364;&#8220;113. <br />     Development 108: 605&#226;&#8364;&#8220;612.                                                                31. Ackley BD, Crew JR, Elamaa H, Pihlajaniemi T, Kuo CJ, et al. (2001) The <br />  8. Banerjee S, Gordon L, Donn TM, Berti C, Moens CB, et al. (2011) A novel role                 NC1/endostatin domain Caenorhabditis elegans type XVIII collagen affects <br />     MuSK non-canonical Wnt signaling segmental neural crest cell                  cell migration axon guidance. J Cell Biol 152: 1219&#226;&#8364;&#8220;1232. <br />     migration. Development 138: 3287&#226;&#8364;&#8220;3296.                                                   32. Meyer F, Moussian B (2009) Drosophila multiplexin (Dmp) modulates motor <br />  9. Bronner-Fraser M, Stern C (1991) Effects mesodermal tissues avian neural               axon pathfinding accuracy. Dev Growth Differ 51: 483&#226;&#8364;&#8220;498. <br />     crest cell migration. Dev Biol 143: 213&#226;&#8364;&#8220;217.                                             33. Salo AM, Wang C, Sipila L, Sormunen R, Vapola M, et al. (2006) Lysyl <br /> 10. Kalcheim C, Teillet MA (1989) Consequences somite manipulation                     hydroxylase 3 (LH3) modifies proteins extracellular space, novel <br />     pattern dorsal root ganglion development. Development 106: 85&#226;&#8364;&#8220;93.                         mechanism matrix remodeling. J Cell Physiol 207: 644&#226;&#8364;&#8220;653. <br /> 11. Krull CE, Lansford R, Gale NW, Collazo A, Marcelle C, et al. (1997) <br />                                                                                              34. Myllyla R, Wang C, Heikkinen J, Juffer A, Lampela O, et al. (2007) Expanding <br />     Interactions Eph-related receptors ligands confer rostrocaudal pattern to <br />                                                                                                  lysyl hydroxylase toolbox: new insights localization activities of <br />     trunk neural crest migration. Curr Biol 7: 571&#226;&#8364;&#8220;580. <br />                                                                                                  lysyl hydroxylase 3 (LH3). J Cell Physiol 212: 323&#226;&#8364;&#8220;329. <br /> 12. McLennan R, Krull CE (2002) Ephrin-as cooperate EphA4 promote <br />                                                                                              35. Myllyharju J, Kivirikko KI (2004) Collagens, modifying enzymes their <br />     trunk neural crest migration. Gene Expr 10: 295&#226;&#8364;&#8220;305. <br />                                                                                                  mutations humans, flies worms. Trends Genet 20: 33&#226;&#8364;&#8220;43. <br /> 13. Santiago A, Erickson CA (2002) Ephrin-B ligands play dual role control <br />                                                                                              36. Tucker RP, Hagios C, Chiquet-Ehrismann R, Lawler J, Hall RJ, et al. (1999) <br />     neural crest cell migration. Development 129: 3621&#226;&#8364;&#8220;3632. <br />                                                                                                  Thrombospondin-1 neural crest cell migration. Dev Dyn 214: 312&#226;&#8364;&#8220;322. <br /> 14. Roffers-Agarwal J, Gammill LS (2009) Neuropilin receptors guide distinct phases <br />     sensory motor neuronal segmentation. Development 136: 1879&#226;&#8364;&#8220;1888.                  37. Risteli M, Ruotsalainen H, Salo AM, Sormunen R, Sipila L, et al. (2009) <br /> 15. Schwarz Q, Maden CH, Davidson K, Ruhrberg C (2009) Neuropilin-mediated                       Reduction lysyl hydroxylase 3 causes deleterious changes deposition <br />     neural crest cell guidance essential organise sensory neurons segmented           organization extracellular matrix. J Biol Chem 284: 28204&#226;&#8364;&#8220;28211. <br />     dorsal root ganglia. Development 136: 1785&#226;&#8364;&#8220;1789.                                         38. Mullins MC, Hammerschmidt M, Haffter P, Nusslein-Volhard C (1994) Large- <br /> 16. Schwarz Q, Maden CH, Vieira JM, Ruhrberg C (2009) Neuropilin 1 signaling                     scale mutagenesis zebrafish: search genes controlling development in <br />     guides neural crest cells coordinate pathway choice cell specification.              vertebrate. Curr Biol 4: 189&#226;&#8364;&#8220;202. <br />     Proc Natl Acad Sci U S A 106: 6164&#226;&#8364;&#8220;6169.                                                 39. van Eeden FJ, Granato M, Schach U, Brand M, Furutani-Seiki M, et al. (1996) <br /> 17. Perris R, Perissinotto D (2000) Role extracellular matrix neural               Mutations affecting somite formation patterning zebrafish, Danio <br />     crest cell migration. Mech Dev 95: 3&#226;&#8364;&#8220;21.                                                     rerio. Development 123: 153&#226;&#8364;&#8220;164. <br /> 18. Halloran MC, Berndt JD (2003) Current progress neural crest cell motility             40. Flanagan-Steet H, Fox MA, Meyer D, Sanes JR (2005) Neuromuscular synapses <br />     migration future prospects zebrafish model system. Dev Dyn                   form vivo incorporation initially aneural postsynaptic specializa- <br />     228: 497&#226;&#8364;&#8220;513.                                                                                tions. Development 132: 4471&#226;&#8364;&#8220;4481. <br /> 19. Eisen JS, Weston JA (1993) Development neural crest zebrafish.             41. Kirby BB, Takada N, Latimer AJ, Shin J, Carney TJ, et al. (2006) In vivo time- <br />     Dev Biol 159: 50&#226;&#8364;&#8220;59.                                                                     <br /> </body></html>