<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                        Dur3 is the major urea transporter in Candida <br />                                        albicans and is co-regulated with the urea <br />                                        amidolyase Dur <br />  <br />                                        Hemiascomycetes, including the pathogen Candida albicans, acquire nitrogen <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> urea using <br />                                        the urea amidolyase Dur1,2, whereas all other higher fungi use primarily the nickel-containing <br />                                        urease. Urea metabolism via Dur1,2 is important for resistance <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> innate host immunity in C. <br />                                        albicans infections. To further characterize urea metabolism in C. albicans we examined the <br />                                        function of seven putative urea transporters. Gene disruption established that Dur3, encoded <br />                                        by orf 19.781, is the predominant transporter. [14C]Urea uptake was energy-dependent and <br />                                        decreased approximately sevenfold in a dur3D mutant. DUR1,2 and DUR3 expression was <br />                                        strongly induced by urea, whereas the other putative transporter genes were induced less than <br />                                        twofold. Immediate induction of DUR3 by urea was independent of its metabolism via Dur1,2, <br />                                        but further slow induction of DUR3 required the Dur1,2 pathway. We investigated the role of <br />                                        the GATA transcription factors Gat1 and Gln3 in DUR1,2 and DUR3 expression. Urea <br />                                        induction of DUR1,2 was reduced in a gat1D mutant, strongly reduced in a gln3D mutant, and <br />                                        abolished in a gat1D gln3D double mutant. In contrast, DUR3 induction by urea was preserved <br />                                        in both single mutants but reduced in the double mutant, suggesting that additional signalling <br />             mechanisms regulate DUR3 expression. These results establish Dur3 as the major urea <br />           transporter in C. albicans and provide additional insights into the control of urea utilization by <br />          this pathogen. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                               The role of urea in the dimorphic human fungal pathogen <br />                                                                            Candida albicans, however, has been unclear. Although <br /> Nitrogen sources play important roles in regulating fungal <br />                                                                            Dastidar and co-workers reported in 1967 that many <br /> dimorphism. For example, Ceratocystis ulmi cells grow as <br />                                                                            strains of C. albicans grow well with urea as the sole source <br /> yeasts in the presence of proline but as hyphae in the <br />                                                                            of nitrogen (Dastidar et al., 1967), this observation drew <br /> presence of ammonia, arginine and most other nitrogen <br />                                                                            little attention based on numerous reports that C. albicans <br /> sources (Kulkarni &amp; Nickerson, 1981). Trigonopsis varia-                   lacks urease (Odds, 1988). This apparent contradiction was <br /> bilis cells grow as budding yeasts in the presence of                      recently explained by the discovery that C. albicans uses <br /> ammonium sulfate and as triangles with methionine                          urea amidolyase to hydrolyse urea (Ghosh et al., 2009). The <br /> (Sentheshanmuganathan &amp; Nickerson, 1962).                                  enzyme urea amidolyase, encoded by DUR1,2, was first <br /> Urea is an important nitrogen source for many fungi and is                 characterized in Candida utilis (Roon et al., 1972). Dur1,2 <br /> also important for microbial pathogenesis in humans.                       is a multifunctional biotin-dependent enzyme (Roon &amp; <br /> Urease is a virulence factor for some pathogenic fungi,                    Levenberg, 1972) with domains for urea carboxylase and <br /> including Cryptococcus neoformans (Cox et al., 2000a) and                  allophanate (urea carboxylate) hydrolase activity (Cooper <br /> Coccidioides immitis (Cole, 1997), and for the bacterial                   et al., 1980). Recently, we showed that higher fungi exhibit <br /> pathogens Helicobacter pylori (Eaton et al., 1991) and                     a dichotomy with regard to urea utilization (Navarathna <br /> Proteus mirabilis (Jones et al., 1990).                                    et al., 2010). All of the hemiascomycetes use urea amidolyase <br />                                                                            (Dur1,2). Most other higher fungi use urease exclusively, <br /> Abbreviations: NCR, nitrogen catabolite repression; qPCR, quantitative     except the Sordariomycetes (Magnaportha, Fusarium and <br /> PCR.                                                                       Nectria), which have both urease and urea amidolyase. Still, <br />  <br /> 270                                                                                                      045005       Printed in Great Britain <br />                                                                                                   Dur3 is the major C. albicans urea transporter <br />  <br />  <br /> the vast majority of yeasts/hemiascomycetes preserve the                   (Table 2), and cloning the SacI/SacII- and XhoI/ApaI-digested PCR <br /> dichotomy by having urea amidolyase but not urease.                        products on both sides of the SAT1 flipper cassette of plasmid pSFS1, <br />                                                                            in which expression of the caFLP gene is controlled by the SAP2 <br /> Why do Candida and Saccharomyces use an energy-                            promoter (Reu&#195;&#376; et al., 2004). The SacI&#226;&#8364;&#8220;ApaI fragment from the <br /> dependent, biotin-containing urea amidolyase system                        resulting plasmid pDUR3M3 was used to sequentially delete the <br /> when the same overall reaction could be accomplished by                    DUR3 alleles of strain SC5314 by the SAT1 flipping strategy, as <br />                                                                            described previously (Dunkel et al., 2008), resulting in the <br /> the simpler urease? This question becomes even more <br />                                                                            homozygous dur3D mutants DUR3M4A and DUR3M4B. For <br /> germane when we consider that all strains of C. albicans are               reintroduction of a functional DUR3 copy, the DUR3 coding region <br /> biotin auxotrophs (Odds, 1988), and that two to four times                 plus 0.3 kb of upstream and 0.5 kb of downstream sequence was <br /> as much biotin is required for maximum growth of                           amplified with the primers DUR31 and DUR35, and the SacI/SacII- <br /> Saccharomyces cerevisiae on urea, allantoic acid or allantoin              digested PCR product was substituted for the DUR3 upstream <br /> as sole nitrogen source (Di Carlo et al., 1953). We have                   fragment in pDUR3M3. The SacI&#226;&#8364;&#8220;ApaI fragment from the resulting <br /> suggested that urea amidolyase allows the hemiascomycetes                  plasmid pDUR3K1 was used to reintegrate DUR3 at the endogenous <br />                                                                            locus of the dur3D mutants DUR3M4A and DUR3M4B, followed by <br /> to jettison their sole Ni(II)- and Co(II)-dependent enzyme <br />                                                                            recycling of the SAT1 flipper cassette, to generate the complemented <br /> and thus to require two fewer transition metals (Navarathna                strains DUR3K2A and DUR3K2B. <br /> et al., 2010). In addition, urea amidolyase may create a urea- <br /> dependent signalling pathway, and we have hypothesized                     Mouse candidiasis model. Outbred 6&#226;&#8364;&#8220;8-week-old (18&#226;&#8364;&#8220;20 g) BALB/c <br /> that such a pathway might be related to fungal pathogenicity               female mice obtained from Charles River Laboratories were randomly <br /> (Ghosh et al., 2009).                                                      allocated to groups of five animals and housed and cared for according <br />                                                                            to guidelines of the National Cancer Institute (NCI) Animal Care and <br /> While examining the role of arginine-induced germ tube                     Use Committee. Each group of mice was inoculated intravenously in <br /> formation in the escape of C. albicans from murine                         the lateral caudal tail vein using a 27-gauge needle with a volume of <br /> macrophages (Ghosh et al., 2009), we deleted the DUR1,2                    0.1 ml containing 106 C. albicans cells (Navarathna et al., 2005, 2007). <br /> gene from the wild-type strain A72. The dur1,2D mutant                     Clinical signs of illness in each mouse were evaluated three times daily, <br />                                                                            and moribund mice that displayed sunken eyes, arched back posture, <br /> KWN6 was unable to grow on urea as the sole nitrogen <br />                                                                            dehydration, ruffled hair or difficulty walking were euthanized <br /> source, stimulate germ tube formation in response to                       immediately by placing them in a closed chamber filled with CO2, <br /> L-arginine or urea, or escape from the murine macrophage                   and processed for complete necropsy and collection of tissues for <br /> cell line RAW 264.7. These abilities were restored in the                  histopathological examination. <br /> reconstituted strains KWN7 and KWN8 (Ghosh et al., <br /> 2009).                                                                     RNA extraction and gene expression analysis by RT-PCR. RNA <br />                                                                            isolation was done using a standard hot phenol procedure (Ko&#194;&#168;hrer &amp; <br /> Given the divergence in mechanisms for urea degradation,                   Domdey, 1991). Reverse transcription was conducted using 5 mg total <br /> we examined whether a similar dichotomy exists in the                      RNA extracted from each sample using SuperScript III Reverse <br /> mechanisms for urea transport. Here, we define the major                   Transcriptase (Invitrogen) according to the manufacturer&#226;&#8364;&#8482;s instruc- <br /> urea transporter in C. albicans, and examine the regulation                tions for oligo-dT priming. Quantitative PCR (qPCR) was conducted <br />                                                                            as previously described (Navarathna &amp; Roberts, 2010) using ABsolute <br /> of urea transport and urea utilization by other known                      QPCR SYBR Green Mix (Thermo Scientific), an Opticon I <br /> nitrogen sources and two transcription factors involved in                 instrument and Opticon I software (Bio-Rad). Samples were analysed <br /> the induction of genes that are subject to nitrogen                        by PCR in triplicate and normalized to internal CDC36 mRNA levels. <br /> catabolite repression (NCR).                                               Melting curve analysis was performed to ensure that a single product <br />                                                                            was produced in each reaction (Navarathna &amp; Roberts, 2010). The <br />                                                                            qPCR primers used in this study are listed in Table 2, and all products <br />                                                                            were 75&#226;&#8364;&#8220;125 bases in <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>. <br /> METHODS <br /> Strains and growth conditions. The C. albicans strains used for this       Uptake of [14C]urea. All C. albicans cells were grown overnight in <br /> study are listed in Table 1. For all experiments, C. albicans cells were   50 ml YPD, washed and resuspended in PBS, and used as the <br /> grown overnight in 50 ml yeast peptone dextrose (YPD) medium at            inoculum (OD600 0.2) for fresh cultures in glucose phosphate proline <br /> 30 uC with aeration, as previously described (Navarathna et al., 2005).    (GPP) medium grown at 30 uC for 3&#226;&#8364;&#8220;4 h while shaking at 150 r.p.m. <br /> Cells were harvested by centrifugation at 5000 r.p.m. for 10 min,          Cell numbers were counted and adjusted so that all uptake assays <br /> washed once with 50 ml sterile, non-pyrogenic normal saline (Quality       employed an equal number of cells. Assays were done in GP buffer in <br /> Biological, Inc.) and resuspended in 10 ml saline before quantifying       14 ml round-bottomed tubes (BD) with or without 5 mM sodium <br /> cell numbers using a Petroff&#226;&#8364;&#8220;Hausser counting chamber. For gene            azide, using ~107 cells ml21 and 1 mCi (3.76104 Bq, 1.15 mg ml21, <br /> expression analysis, C. albicans cells were grown for 16 h in 50 ml        19.2 mM) [14C]urea (American Radiolabelled Chemicals). After <br /> YPD medium (Kulkarni &amp; Nickerson, 1981), washed three times with           incubation with orbital shaking at 150 r.p.m. for 5 min, unless <br /> PBS, and added to glucose peptone (GP) medium containing 10 mM             otherwise indicated, triplicate 0.5 ml samples were collected. Two <br /> of the indicated nitrogen sources. Fresh cultures were started at OD600    0.2 ml volumes of the cell suspension were sequentially overlaid onto <br /> 0.2 using overnight YPD cultures, and grown up to OD600 0.8 (for           100 ml 30 % sucrose in 5649 mm microfuge tubes and centrifuged in <br /> about 4&#226;&#8364;&#8220;5 h) for RNA extraction.                                           a Beckman Microfuge B for 30 s. After removing the upper aqueous <br />                                                                            layer, the resulting cell pellet containing 46106 cells was washed by <br /> Disruption and complementation of DUR3 (orf 19.781). A DUR3                adding 300 ml PBS and recentrifuged. After removing the supernatant <br /> deletion cassette was constructed by amplifying the DUR3 upstream          fluid, the tips of the microfuge tubes containing the cell pellets were <br /> and downstream regions from genomic DNA of strain SC5314 with              cut off and shaken in scintillation fluid before quantifying the <br /> the primer pairs DUR31/DUR32 and DUR33/DUR34, respectively                 internalized urea using a scintillation counter. <br />  <br /> http://mic.sgmjournals.org                                                                                                                     271 <br />  D. H. M. L. P. Navarathna and others <br />  <br />  <br /> Table 1. Strains used in this study <br />  <br />  Strain                                   Relevant genotype                                           Reference or source <br />  <br />  SC5314                          Wild-type                                                       Gillum et al. (1984) <br />  A72                             Wild-type                                                       ATCC* MYA-2430 <br />  KWN6                            dur1,2D/dur1,2D                                                 Ghosh et al. (2009) <br />  KWN7                            dur1,2D/dur1,2D : : DUR1,2                                      Ghosh et al. (2009) <br />  KWN8                            dur1,2D : : DUR1,2/dur1,2D : : DUR1,2                           Ghosh et al. (2009) <br />  DUR3M4A and -B                  dur3D/dur3D                                                     This study <br />  DUR3K2A and -B                  dur3D/dur3D : : DUR3                                            This study <br />  GAT1M4A                         gat1D/gat1D                                                     Dabas &amp; Morschha&#194;&#168;user (2007) <br />  GLN3M4A                         gln3D/gln3D                                                     Dabas &amp; Morschha&#194;&#168;user (2007) <br />  Dgln3GAT1M4A                    gat1D/gat1D gln3D/gln3D                                         Dabas &amp; Morschha&#194;&#168;user (2007) <br />  <br /> *American Type Culture Collection. <br />  <br />  <br />  <br /> Bioinformatics analysis. Sequences of fungal proteins were                  Jehan&#226;&#8364;&#8220;Breslow&#226;&#8364;&#8220;Wilcoxon test using GraphPad Prism software. <br /> obtained from the National Center for Biotechnology Information             qRT-PCR data were analysed using the Pfaffl method (Pfaffl, 2001) <br /> (NCBI) (http://www.ncbi.nlm.nih.gov) and the Fungal Genome                  and were normalized in two steps. CDC36, an abundantly expressed <br /> Initiative (http://www.broad.mit.edu/annotation/fgi/). DUR3 homo-           nuclear protein (Collart &amp; Struhl, 1994), was used as the internal <br /> logues were identified using S. cerevisiae Dur3 (YHL016C) as a query        control because we found its mRNA to be the most invariant for <br /> for BLASTP searches. All BLAST searches were conducted using default        normalizing C. albicans gene expression analysis (Navarathna &amp; <br /> parameters. We examined the distribution of DUR3 in the genomes             Roberts, 2010; Pendrak et al., 2004a, b, c). First, mean values for the <br /> reported previously (Navarathna et al., 2010).                              internal CDC36 control were subtracted from the corresponding <br />                                                                             experimental sample, and the variance was calculated as the square <br /> Statistics. The probability of <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>survival</span> as a function of time               root of the sum of the squares of the standard deviations. Second, <br /> was determined by the Kaplan&#226;&#8364;&#8220;Meier method, and significance                 each sample group was normalized to C. albicans grown in YPD <br /> was determined by the log-rank (Mantel&#226;&#8364;&#8220;Cox) test and the                    medium, except in Fig. 2, where GPP was used. The results are <br />  <br />  <br />  <br /> Table 2. Oligonucleotides used in this study <br /> All primers are in 59A39 orientation. Introduced restriction sites are underlined. <br />  <br />  Primer                                                  Sequence <br />  <br />  DUR31                               TTGAGCTCGATAAGACTAAATGACTGCG <br />  DUR32                               CACTCCGCGGTAATGGGACATATGAATCAGCC <br />  DUR33                               CATTGACGATGCACTCGAGGAGAAGAAAGG <br />  DUR34                               GTCAGCATAATCAGGGCCCAATCAGAAGCC <br />  DUR35                               TAATCCGCGGTATTCTCATGATTTGCCCC <br />  DUR1,2 RT LT                        TTTGTTGGTCCATCTGGTGA <br />  DUR1,2 RT RT                        TCTGGCTTCTTTGGCATCTT <br />  DUR RT LT                           AGCTTGGCAAAATGAACACC <br />  DUR RT RT                           GCTGGGGTGACATATCCATC <br />  DUR3 RT LT                          ACAGTGCTCGAGAAGGTGGT <br />  DUR3 RT RT                          AAAGCAGCTGAAGCCAATGT <br />  DUR4 RT LT                          ATCCATGGGCTGGTTATTCA <br />  DUR4 RT RT                          CACCTGCCTGGTTCAAATCT <br />  DUR7 RT LT                          GGGTTATTTGGCCAATGTCT <br />  DUR7 RT RT                          AAAAGGGGACCAATCCAATC <br />  NPR2 RT LT                          AATGGCGGAAGAATGTCAAG <br />  NPR2 RT RT                          ACTTCCAGGGTAGGGCAAGT <br />  5915 RT LT                          ACGTTGACTCCGATCGAAAC <br />  5915 RT RT                          GATTGCAGCACCAGTAGCAA <br />  5017 RT LT                          TTTGTCCAGGTGTGATTCCA <br />  5017 RT RT                          GTGATATGCCGTCGAAATCC <br />  CDC36 RT LT                         GAGCGTCCAGTATAAATCCACCAC <br />  CDC36 RT RT                         TCAAGACGGGCTCCACATTACTAT <br />  <br />  <br />  <br /> 272                                                                                                                             Microbiology 157 <br />                                                                                          Dur3 is the major C. albicans urea transporter <br />  <br />  <br /> presented as fold change using 22C t of this normalized Ct value. <br /> Statistical differences among groups were analysed with two-way <br /> analysis of variance (ANOVA) using GraphPad Prism software. <br />  <br />  <br />  <br /> RESULTS <br />  <br /> Expression of potential urea transporters <br /> The yeast S. cerevisiae metabolizes urea by means of Dur1,2 <br /> and the urea transporters Dur3 and Dur4 (Cooper, 1982). <br /> DUR1,2 and DUR3 are inducible by allophanate, whereas <br /> DUR4 encodes a constitutive urea-facilitated diffusion <br /> transporter (Cooper, 1982). The C. albicans orthologue <br /> DUR1,2 was identified by Gene Ontology (GO) annota- <br /> tion. Remarkably, the C. albicans genome contains seven <br /> genes encoding putative urea transporters: DUR3 (orf <br /> 19.781), DUR3 (orf 19.6656), DUR4 (orf 19.5677), DUR7 <br /> (orf 19.7205), NPR2 (orf 19.328), orf 19.5915 and orf <br /> 19.5017. The gene name DUR3 has been used for both orf <br /> 19.781 and orf 19.6656, but orf 19.781 is the orthologue of <br /> DUR3 of S. cerevisiae, and we therefore refer to this gene as <br /> DUR3. DUR3, DUR4 and DUR7 are fungal-specific in that <br /> they lack human or murine orthologues. <br /> Based on gene expression levels in minimal medium, we <br /> separated the putative urea transporters into two catego- <br /> ries. DUR3 and orf 19.6656 reproducibly showed high <br /> expression, with Ct values less than 22, whereas the others <br /> all had Ct values less than 24. Addition of urea instead of <br /> ammonia to the growth medium (glucose phosphate urea; <br /> GPU) resulted in a 100-fold increase in DUR3 gene <br /> expression but no increase for orf 19.6656 relative to the          Fig. 1. Effect of urea on the expression of seven putative urea <br /> basal expression in YPD (Fig. 1a). None of the other five           transporters. mRNA abundance was determined by qPCR using <br /> putative transporter genes showed significant responses to          C. albicans A72 cells grown in GPA (glucose phosphate <br /> urea relative to that in YPD (Fig. 1b).                             ammonia) or GPU medium. Experiments were performed in <br />                                                                     triplicate; error bars, SEM. Genes exhibiting higher expression are <br />                                                                     shown in (a) and low-expression genes are shown in (b). GPA and <br /> Urea induction of DUR3 is not dependent upon                        GPU expression levels were normalized to those of YPD. Cells <br /> DUR1,2 expression                                                   grown in GPP medium showed induction of DUR3 similar to that of <br /> The dur1,2D mutant KWN6 and the complemented strain                 cells grown in GPA (data not shown). <br /> KWN8 (Ghosh et al., 2009) were used to determine <br /> whether the DUR3 urea response depended on a metabolite <br /> downstream of Dur1,2, such as allophanate. Expression of            experiment, overnight C. albicans cultures grown in YPD <br /> DUR1,2 and DUR3 was increased six- and 10-fold,                     were washed three times before transfer to GPU medium to <br /> respectively, for C. albicans A72 grown with urea (glucose          examine gene expression induced by urea. Within 1 min of <br /> phosphate proline urea; GPPU) rather than without (GPP;             addition of urea, mRNA levels increased 400- to 600-fold <br /> Fig. 2a, b). Expression and induction of DUR1,2 (Fig. 2a)           in both cultures. This immediate induction was largely <br /> was lost in the dur1,2D mutant, and restored in the fully           independent of Dur1,2. However, DUR3 mRNA increased <br /> reconstituted strain KWN8 (Fig. 2a). However, in the                an additional threefold by 15 min in wild-type cells, <br /> dur1,2D mutant, DUR3 expression was still increased ~20-            whereas no further increase was seen in the mutant lack- <br /> fold by urea (GPPU) relative to the same strain in GPP.             ing DUR1,2 (Fig. 2c). Therefore, a metabolite of urea <br /> Thus, DUR3 basal expression was slightly reduced in the             downstream of Dur1,2 may account for a secondary <br /> absence of DUR1,2, but its relative induction by urea was           induction of DUR3. Alternatively, upstream feedback <br /> preserved.                                                          inhibition of further DUR3 induction may occur due to <br />                                                                     urea accumulation in the cell. <br /> However, examining the time dependence of DUR3 <br /> induction by urea in A72 (wild-type) versus KWN6                    We also tested DUR1,2 and DUR3 expression in the <br /> (dur1,2D) cells revealed a delayed effect of urea metabolism        presence of 10 mM allantoin. Allantoin, an intermediate of <br /> via Dur1,2 on DUR3 induction (Fig. 2c). For this                    purine degradation that is subsequently converted to urea, <br />  <br /> http://mic.sgmjournals.org                                                                                                        273 <br />  D. H. M. L. P. Navarathna and others <br />  <br />  <br />                                                                    induced DUR1,2 mRNA levels 6&#194;&#177;0.08-fold and DUR3 <br />                                                                    mRNA levels 1.8&#194;&#177;0.02-fold in C. albicans (data not <br />                                                                    shown). <br />  <br />                                                                    Transport of [14C]urea into C. albicans <br />                                                                    The functional importance of Dur3 in urea transport was <br />                                                                    examined by measuring [14C]urea uptake (Table 3). <br />                                                                    Preliminary experiments established that optimal uptake <br />                                                                    was attained at 5 min and gradually decreased thereafter <br />                                                                    due to catabolism (data not shown). By 5 min, urea uptake <br />                                                                    levels for a dur3D mutant were approximately three times <br />                                                                    lower than those for the wild-type C. albicans. Recon- <br />                                                                    stitution of one allele of DUR3 in the deletion mutant <br />                                                                    restored urea uptake to wild-type levels (Table 3). This <br />                                                                    finding is consistent with our observation that the dur3D <br />                                                                    mutant grew very slowly on GPU, with urea as the sole <br />                                                                    nitrogen source. The remaining urea transport in the <br />                                                                    dur3D mutant (Table 3) was likely achieved by one or more <br />                                                                    of the other, non-inducible urea transporters (Fig. 1). Urea <br />                                                                    transport by Dur3 was energy-dependent in that it was <br />                                                                    reduced four- to fivefold in the presence of 5 mM sodium <br />                                                                    azide (Table 3). <br />  <br />                                                                    Distribution of the urea transporter DUR3 in fungi <br />                                                                    Because the fungal-specific urea transporter DUR3 resides <br />                                                                    only 1000 bp upstream of DUR1,2 on chromosome 1 in C. <br />                                                                    albicans, we expected that its phylogenetic distribution might <br />                                                                    parallel that of DUR1,2, but it did not. DUR3 orthologues <br />                                                                    were present as a single copy in all of the 22 ascomycete and <br />                                                                    basidiomycete genomes searched (Navarathna et al., 2010), <br />                                                                    except for those of Ustilago maydis and Rhizopus oryzae, <br />                                                                    where it was absent, and Yarrowia lipolytica, where it was <br />                                                                    present in two copies. Y. lipolytica also has two copies of <br />                                                                    DUR1,2. This gene duplication may have occurred relatively <br />                                                                    recently, at the species level, resulting in co-duplication <br />                                                                    of DUR1,2 and DUR3. DUR3 orthologues were also not <br />                                                                    detected in three fungi not included in our previous <br />                                                                    study, Histoplasma capsulatum, Coccidioides immitis and <br />                                                                    Paracoccidioides sp. <br />  <br />                                                                    Regulation of DUR1,2 and DUR3 expression by <br /> Fig. 2. Urea induction of DUR1,2 and DUR3 gene expression.         other nitrogen sources <br /> mRNA abundance was determined by qPCR for DUR1,2 (a) and           We next explored how urea induction of DUR1,2 and <br /> DUR3 (b) in C. albicans strains A72 (wild-type), KWN6 (dur1,2D/    DUR3 relates to NCR regulation of these genes. For this <br /> dur1,2D) and the reconstituted strain KWN8 (dur1,2D : : DUR1,2/    purpose, C. albicans SC5314 cells were grown in defined <br /> dur1,2D : : DUR1,2). The cells were grown in defined GPP (filled   media containing the well-characterized nitrogen sources <br /> bars) or GPPU (open bars) medium. (c) Kinetics of induction of <br />                                                                    proline, glutamine, ammonium sulfate and asparagine, all <br /> DUR3 in the wild-type strain A72 and the dur1,2 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>knockout</span> strain <br />                                                                    at 10 mM (Fig. 3). In all cases, expression of DUR1,2 and <br /> KWN6 in GPU medium. <br />                                                                    DUR3 was higher in the GP basal media than in YPD <br />                                                                    medium. Addition of proline or glutamine at 10 mM in the <br />                                                                    presence of urea significantly repressed expression relative <br /> increased DUR1,2 mRNA levels 3.8&#194;&#177;0.5-fold and DUR3                 to urea alone (compare GPPU and GPQU with GPU in <br /> mRNA levels 9.2&#194;&#177;0.5-fold, compared with GPP. Finally,              Fig. 3). However, addition of 10 mM urea further <br /> 10 mM parabanic acid (oxaluric acid), a gratuitous inducer         stimulated expression of DUR1,2 and DUR3 mRNA <br /> of DUR1,2 in S. cerevisiae (Cooper et al., 1980), also             (compare GPQU, GPNU and GPNH4U with GPQ, GPN <br />  <br /> 274                                                                                                             Microbiology 157 <br />                                                                                                     Dur3 is the major C. albicans urea transporter <br />  <br />  <br /> Table 3. Urea uptake by wild-type and mutant C. albicans <br /> Values shown are radioactivity internalized in 4&#226;&#8364;&#8220;56106 cells after exposure to [14C]urea for 5 min, and are the <br /> mean&#194;&#177;SEM of triplicate experiments. <br />  <br />  Strain                                                  Treatment                            c.p.m. <br />  <br />  SC5314 (wild-type)                          GP   with   [14C]urea                          2707&#194;&#177;660 <br />  SC5314 (wild-type)                          GP   with   [14C]urea and NaN3                  517&#194;&#177;129 <br />  DUR3M4A (dur3D)                             GP   with   [14C]urea                           960&#194;&#177;280 <br />  DUR3K2A (dur3D+DUR3)                        GP   with   [14C]urea                          2698&#194;&#177;316 <br />  <br />  <br />  <br /> and GPNH4, respectively, in Fig. 3). Notably, the levels of                   wild-type), and abolished in the double mutant (P,0.001 <br /> gene expression were highest when urea was the sole                           for gat1D gln3D vs gat1D, Fig. 4). Interestingly, urea <br /> nitrogen source added to the basal medium. Therefore, the                     induction of the transporter DUR3 was not affected by <br /> DUR1,2 and DUR3 genes appear to be under limited NCR                          deletion of GAT1 or GLN3 individually, but it was <br /> in that their responsiveness to urea is largely independent                   significantly reduced in the double mutant (P,0.001 for <br /> of NCR regulation.                                                            gat1D gln3D vs wild-type). However, urea still induced DUR3 <br />                                                                               expression in the double mutant (P,0.01 for GPU vs YPD), <br />                                                                               indicating that other transcriptional regulators are involved. <br /> Regulation of urea utilization by GATA <br /> transcription factors <br />                                                                               NCR regulation of DUR3 involves additional <br /> To further examine the contribution of NCR to regulation of <br />                                                                               signals <br /> these urea utilization and transporter genes, we compared <br /> basal (YPD) and induced (GPU) expression of DUR1,2 and                        To confirm that, in addition to Gat1 and Gln3, other <br /> DUR3 pairwise in four strains: wild-type C. albicans and                      signalling mechanisms control DUR1,2 and DUR3 expres- <br /> mutants lacking one or both of the GATA transcription                         sion, we tested DUR1,2 and DUR3 mRNA levels in the <br /> factors Gat1 and Gln3 (Fig. 4). Induction of DUR1,2 mRNA                      gat1D, gln3D and gat1D gln3D mutants in the presence of <br /> was not reduced in the gat1D mutant, was significantly                        different nitrogen sources. DUR1,2 mRNA levels were high <br /> reduced in the gln3D mutant (P,0.001 for gln3D vs                             in gat1D cells grown in GPU, but addition of glutamine or <br />  <br />  <br />  <br />  <br />        Fig. 3. Effect of nitrogen source on DUR1,2 and DUR3 expression in C. albicans. DUR1,2 (a) and DUR3 (b) mRNA abundance <br />        was determined by qPCR for C. albicans SC5314 cells grown with the following additives in defined GP-based media: P, L- <br />        proline; U, urea; NH4, ammonia; Q, L-glutamine; N, asparagine; all at 10 mM. Similar results were obtained when A72 wild-type <br />        cells were subjected to the same treatments. Experiments were performed in triplicate; error bars, SEM. Addition of proline and <br />        glutamine significantly reduced DUR1,2 expression (b vs a5P,0.001), and addition of L-proline, glutamine, ammonia and <br />        asparagine significantly reduced expression of DUR3 (b vs a5P,0.001 by ANOVA) compared with expression levels in GPU. <br />        DUR3 expression was induced by addition of urea, regardless of the nitrogen source tested (c vs b5P,0.001, 0.001, 0.05 and <br />        0.001 for GPP, GPQ, GPNH4 and GPN, respectively, by ANOVA). <br />  <br /> http://mic.sgmjournals.org                                                                                                                  275 <br />  D. H. M. L. P. Navarathna and others <br />  <br />  <br />                                                                                      Fig. 4. Role of GATA transcription factors on <br />                                                                                      expression of DUR1,2 and DUR3. The cells <br />                                                                                      were grown in GPU and YPD. Open bars, <br />                                                                                      DUR3; filled bars, DUR1,2. Lower-case type <br />                                                                                      with an asterisk represents significant changes <br />                                                                                      in DUR3 expression in GPU relative to <br />                                                                                      expression in the wild-type strain in YPD. <br />                                                                                      Lower-case type without an asterisk indicates <br />                                                                                      significant changes in DUR1,2 expression in <br />                                                                                      GPU relative to that of the wild-type strain in <br />                                                                                      YPD medium. Experiments were performed in <br />                                                                                      triplicate; error bars, SEM. Compared with the <br />                                                                                      wild-type, the gln3D and gln3D gat1D mutants <br />                                                                                      showed significantly reduced DUR1,2 expres- <br />                                                                                      sion in GPU (P,0.001 for b vs a and c vs a), <br />                                                                                      and DUR1,2 expression was also significantly <br />                                                                                      lower in the gln3D gat1D double mutant than in <br />                                                                                      the gln3D single mutant (P,0.001 for c vs b). <br />                                                                                      In contrast, DUR3 expression was not affected <br />                                                                                      in the gln3D and gat1D single mutants, <br />                                                                                      although it was significantly reduced <br />                                                                                      (P,0.001 for b* vs a*) in the gln3D gat1D <br />                                                                                      double mutant compared with the wild-type <br />                                                                                      strain grown in GPU. The strains did not differ <br />                                                                                      in expression when grown in YPD. <br />  <br />  <br /> ammonia significantly reduced its expression (P,0.001,            enzyme Dur1,2 rather than the nickel-containing urease <br /> Fig. 5a, b). A similar <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of DUR1,2 expression was           (Navarathna et al., 2010), these fungi have in common a <br /> seen in the gln3D strain grown with GPU and GPU with              number of putative urea transporters. Among these, we have <br /> glutamine (P,0.002). However, addition of ammonia did             identified Dur3 as the major urea-inducible transporter in C. <br /> not significantly reduce DUR1,2 induction. In the gat1D           albicans. Dur3 is responsible for ~80 % of the energy- <br /> gln3D double mutant, urea failed to induce DUR1,2                 dependent urea transport into C. albicans in basal medium, <br /> expression above basal levels.                                    and presumably accounts for a much higher percentage <br />                                                                   when induced. The expression of mRNAs for several <br /> As shown in Fig. 4, deletion of GAT1 or GLN3 individually <br />                                                                   non-inducible urea transporters in C. albicans (Fig. 1) is <br /> did not alter the urea induction of DUR3 expression. <br />                                                                   consistent with our observation that the dur3D mutant <br /> Glutamine and ammonia significantly suppressed DUR3 <br />                                                                   exhibited undiminished pathogenicity in a mouse model of <br /> induction in the wild-type as well as in the gat1D and gln3D <br />                                                                   disseminated candidiasis. It is also likely that sufficient <br /> single mutants, suggesting that neither GATA transcription <br />                                                                   amounts of other nitrogen sources are available for growth, <br /> factor tested is absolutely required for the induction or <br />                                                                   so urea uptake is not limiting. The fact that the dur3D <br /> repression of this gene (Fig. 5c, d). However, induction of <br />                                                                   mutant exhibited undiminished pathogenicity is totally <br /> DUR3 by urea was diminished in the double mutant, and <br />                                                                   consistent with our previous work showing that Dur1,2 is <br /> addition of an alternative nitrogen source in this mutant <br />                                                                   needed for C. albicans to escape from macrophages. In that <br /> did not further reduce its expression. <br />                                                                   case, the urea was produced intracellularly from L-arginine. <br />                                                                   Thus, no import of exogenous urea was needed. <br /> DUR3 does not contribute to C. albicans virulence <br />                                                                   Notably, all other urea transporters are expressed at lower <br /> in mice <br />                                                                   mRNA levels than DUR3. Indeed, some of these putative <br /> The tail vein injection mouse candidaemia model was used to       urea transporters may not be functional or may have other <br /> compare the virulence of the wild-type strain SC5314, the         dominant substrates. For instance, Dur3 in S. cerevisiae has <br /> dur3D mutant DUR3M4A, and the complemented strain                 recently been shown to be a functional boron transporter <br /> DUR3K2A. No significant differences in the survival of mice       (Nozawa et al., 2006). <br /> infected with the three strains were observed (data not shown). <br />                                                                   Although urea can serve as a nitrogen source for C. albicans <br />                                                                   (Dastidar et al., 1967), the inducible transporter Dur3 and <br />                                                                   the catabolic enzyme Dur1,2 do not exhibit classical NCR <br /> DISCUSSION                                                        regulation. When rich nitrogen sources such as ammonia <br /> Although urea metabolism in the hemiascomycetes diverges          and asparagine are available, cells preferentially utilize these <br /> from that of other fungi in using the biotin-containing           sources, while repressing genes that are responsible for <br />  <br /> 276                                                                                                               Microbiology 157 <br />                                                                                            Dur3 is the major C. albicans urea transporter <br />  <br />  <br />  <br />  <br />       Fig. 5. Role of GATA transcription factors in the gene expression of DUR1,2 (a, b) and DUR3 (c, d) with or without other <br />       nitrogen sources (10 mM) in the growth media containing urea (10 mM). (a, c) Gene expression was quantified in the presence <br />       of glutamine (Q). (b, d) Gene expression was quantified in the presence of ammonia. DUR1,2 expression (a, b) and DUR3 <br />       expression (c, d) in the indicated transcription factor mutants was compared with that in the wild-type strain SC5314. <br />       Experiments were performed in triplicate; error bars, SEM. In the gat1D mutant, addition of glutamine and ammonia caused a <br />       significant reduction of DUR1,2 expression (P,0.001) compared with the levels in GPU medium. Both the gat1D and the gln3D <br />       mutant exhibited reduced DUR3 expression when glutamine (P,0.001) or ammonia (P,0.05 for gat1D and 0.001 for gln3D) <br />       was added to GPU. *, P,0.05; **, P,0.002; ***, P,0.001. <br />  <br />  <br />  <br /> breaking down poorer nitrogen sources such as proline,                 GATA(A/G) sites upstream of the ORF (Ghosh et al., <br /> arginine and urea (Cooper, 1982). When cells are starved               2009), and these are the putative binding sites for the <br /> for nitrogen, these NCR-regulated genes are induced. In S.             GATA transcription factors Gln3 and Gat1, which may <br /> cerevisiae, urea amidolyase is subject to NCR (Cox et al.,             mediate gene expression under inducing conditions (Dabas <br /> 2000b), and we have previously shown in C. albicans that               &amp; Morschha&#194;&#168;user, 2007). These sites could mediate both <br /> 5&#226;&#8364;&#8220;10 mM ammonium salts prevent germ tube formation                     direct and indirect NCR (via ammonium ions) of DUR1,2 <br /> induced by GlcNAc, arginine or urea, but not that induced              expression. However, we show here that urea induction of <br /> by 5 % CO2 (Ghosh et al., 2009). These results were                    these genes in GP medium is maintained in the presence <br /> expected, because C. albicans DUR1,2 has at least five (A/T)           of rich and poor nitrogen sources, including proline, <br />  <br /> http://mic.sgmjournals.org                                                                                                          277 <br />  D. H. M. L. P. Navarathna and others <br />  <br />  <br /> glutamine, NH4 and asparagine. The putative GATA sites           the signalling mechanisms that regulate urea transport in <br /> upstream of DUR3 probably contribute to its induction by         C. albicans. <br /> urea, although residual induction by urea in the gat1D <br />                                                                  Although deletion of the inducible urea transporter Dur3 <br /> gln3D mutant suggests that additional transcription factors <br />                                                                  did not decrease virulence in a standard mouse candidae- <br /> mediate DUR3 induction by urea. <br />                                                                  mia model, inducible acquisition of urea may be important <br /> In S. cerevisiae, the direct inducer for DUR1,2 is               for other aspects of C. albicans pathogenesis. Nitrogen <br /> allophanate, also known as urea carboxylate, rather than         metabolism contributes to the pathogenicity of Aspergillus <br /> urea itself (Cooper, 1982). Allophanate is the chemical          fumigatus (Krappmann &amp; Braus, 2005). Urease is a <br /> intermediate for the multifunctional Dur1,2. In C. albicans,     virulence factor in Cryptococcus neoformans (Cox et al., <br /> allophanate may also be an inducer for DUR1,2, although          2000a) and Coccidiodes immitis (Yu et al., 1997). In <br /> urea induction of DUR3 must involve a different                  addition, urease is a well-established virulence factor in H. <br /> mechanism. This conclusion is based on the observation           pylori. A urease mutant is unable to survive in co-culture <br /> that the dur1,2D mutant showed the same relative initial         with macrophages, whereas the urease in wild-type H. <br /> DUR3 induction by urea as the wild-type (Fig. 2b).               pylori prevents phagosome maturation (Schwartz &amp; Allen, <br /> However, following DUR3 expression over time after the           2006). We know that urea metabolism provides similar <br /> introduction of urea revealed that the dur1,2D mutant            protection from clearance by macrophages in C. albicans <br /> lacked the slow secondary induction of DUR3 expression           (Ghosh et al., 2009). Future studies will further examine <br /> seen for wild-type cells grown in GPPU (Fig. 2c).                the role of urea transport and metabolism in pathogenesis <br /> Presumably, the longer growth time in GPPU permitted             and examine the virulence of dur1,2D mutants. We will also <br /> accumulation of another, less efficient inducer (potentially     explore the potential for urea catabolism to contribute to <br /> allophanate) and/or the lifting of a partial NCR caused by       colonization of the kidney, where the organism could be <br /> the metabolism of L-proline.                                     exposed to higher concentrations of urea than in the <br /> The expression of DUR1,2 and DUR3 was minimal in the             bloodstream. <br /> complex medium YPD, suggesting that these genes could be <br /> subject to classical NCR. Although the high constitutive        <br /> expression of both genes in basal medium containing urea was <br /> significantly repressed by addition of either good (ammonia)      <br /> or poor (proline) nitrogen sources (Limjindaporn et al., 2003;  <br /> Marzluf, 1997), disruption of the NCR transcription factor <br /> Gat1 did not prevent induction of DUR1,2 by urea or its         <br /> repression by either nitrogen source. Similarly, DUR3           <br /> induction by urea was refractory to deletion of Gat1 or         <br /> Gln3, and addition of nitrogen sources significantly repressed    <br /> DUR3 expression in the presence of urea (Fig. 5c, d). Yet, <br /> induction of both genes by urea was dramatically decreased in <br /> the gat1D gln3D double mutant, and this strain was not subject   <br /> to further repression by ammonia or Gln. These observations <br />                                                                 <br /> indicate that the GATA transcription factors Gat1 and Gln3 <br />                                                                <br /> play positive roles in the urea induction of these genes, and <br />                                                               <br /> that another NCR mechanism controls the negative regulation <br />                                                     <br /> of DUR1,2 and DUR3 expression.                              <br /> Complexity in the adaptive responses to varying envi-            <br /> ronmental nitrogen sources is not unique to C. albicans.        <br /> Aspergillus AreA and Neurospora NIT2 play primary roles         <br /> in derepressing NCR (Caddick et al., 1994). Aspergillus      <br /> amdS encodes an acetamidase that releases ammonia. This <br /> gene is controlled by pathway-specific gene regulation as   <br /> well as general regulatory proteins (Davis et al., 1993). Gln3   <br /> of S. cerevisiae resembles AreA and NIT2 as a positive-          <br /> acting regulator of transcription (Cooper et al., 1990). <br />                                                           <br /> Aspergillus nirA encodes a nitrate reductase that requires      <br /> AreA for induction. A <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>mutation</span> of nirA has been shown to     <br /> overcome the requirement for AreA, suggesting that both         <br /> specific and general regulation pathways are required for      <br /> nitrogen metabolism in Aspergillus (Cove, 1979). These          <br /> insights from other fungi will guide our future studies of     <br /> </body></html>