<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Yeast Magmas Ortholog Pam16 Has Essential <br /> Function Fermentative Growth That Involves <br /> Sphingolipid Metabolism <br /> Mary K. Short1, Joshua P. Hallett1, Krisztina Tar2, Thomas Dange2, Marion Schmidt2, Robyn Moir2, <br /> Ian M. Willis2,3*, Paul T. Jubinsky1,4* <br /> 1 Department Developmental Molecular Biology, Albert Einstein College Medicine, Bronx, New York, United States America, 2 Department Biochemistry, <br /> Albert Einstein College Medicine, Bronx, New York, United States America, 3 Department Systems Computational Biology, Albert Einstein College Medicine, <br /> Bronx, New York, United States America, 4 Department Pediatrics, Section Hematology/Oncology, Yale University School Medicine, New Haven, Connecticut, <br /> United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Magmas growth factor responsive gene encoding essential mitochondrial protein mammalian cells. Pam16, the <br />      Magmas ortholog Saccharomyces cerevisiae, component presequence translocase-associated motor. A <br />      temperature-sensitive allele (pam16-I61N) used query array non-essential gene-deletion strains synthetic <br />      genetic interactions. The pam16-I61N <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> ambient temperature caused synthetic lethal sick phenotypes with <br />      genes involved lipid metabolism, perixosome synthesis, histone deacetylation mitochondrial protein import. The <br />      gene deletion array screened suppressors pam16-I61N growth defect identify compensatory pathways. <br />      Five suppressor genes identified (SUR4, ISC1, IPT1, SKN1, FEN1) involved sphingolipid metabolism. <br />      pam16-I61N cells cultured glucose non-permissive temperatures resulted rapid growth inhibition G1 cell cycle <br />      arrest, cell viability maintained. Altered mitochondria morphology, reduced peroxisome induction glycerol/ <br />      ethanol oleate, changes levels sphingolipids including C18 alpha-hydroxy-phytoceramide, were <br />      observed temperature sensitive strain. Deletion SUR4, strongest suppressor, reversed temperature <br />      sensitive fermentative growth defect, morphological changes elevated levels C18 alpha-hydroxy <br />      phytoceramide pam16-I61N. Deletion suppressor genes similar effects C18 alpha-hydroxy- <br />      phytoceramide levels restored proliferation pam16-I61N strain. In addition, pam16-I61N inhibited respiratory <br />      growth, likely reducing cardiolipin, essential mitochondrial function. Our results suggest pleiotropic <br />      effects caused impaired Pam16/Magmas function mediated changes lipid metabolism. <br />  <br />   Citation: Short MK, Hallett JP, Tar K, Dange T, Schmidt M, et al. (2012) The Yeast Magmas Ortholog Pam16 Has Essential Function Fermentative Growth That <br />   Involves Sphingolipid Metabolism. PLoS ONE 7(7): e39428. doi:10.1371/journal.pone.0039428 <br />   Editor: Michael Polymenis, Texas A&amp;M University, United States America <br />   Received October 2, 2011; Accepted May 22, 2012; Published July 10, 2012 <br />   Copyright: &#195;&#376; 2012 Short et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Institutes Health (NIH GM084228)-MS, National Institute General Medical Sciences (NIGMS GM085177)-IMW, <br />   James B. Ax Family Foundation, SAS Foundation Cancer Research, support Donna Alex Mary Robert Cancilla PTJ. The funders no <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: paul.jubinsky@yale.edu (PTJ); ian.willis@einstein.yu.edu (IMW) <br />  <br />  <br />  <br /> Introduction                                                                          embryogenesis, highest levels Magmas observed in <br />                                                                                       heart, notochord, choroid plexus, cervical ganglion, nasal mucosa <br />    Mitochondrial associated granulocyte macrophage signaling                          liver. Adult tissues high Magmas expression include <br /> molecule (Magmas) nuclear encoded protein                           muscle, pancreas, intestinal mucosa, testes [6]. <br /> mitochondrial matrix mammalian cells. It identified                          Magmas expression differs normal neoplastic <br /> transcript rapidly induced multipotential myeloid cell                 prostate [7]. The protein barely detectable normal prostate <br /> line PGMD1 granulocyte-macrophage colony-stimulating                               glands increased amounts observed higher-grade <br /> factor (GM-CSF) [1]. Magmas highly conserved [2]                            tumors. The increased Magmas malignant cells results <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> essential <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> eukaryotic cells [3,4,5]. In addition                higher amounts Magmas/cell changes the <br /> mitochondrial targeting domain [1] comparison Magmas                             number mitochondria/cell [7]. Investigations breast cancer <br /> homologs 52 species distributed animals, plants                        correlate increased Magmas expression poor outcome [8,9], <br /> fungi revealed conserved sequence motifs [2]. Kingdom                           studies pituitary adenoma Burkitt&#226;&#8364;&#8482;s lymphoma cells <br /> specific amino acid signatures, potential targeting                        suggest increased Magmas levels protect apoptosis <br /> protein locations mitochondria, presence                    [10,11]. These data suggest Magmas important role in <br /> of multiple isoforms higher eukaryotes suggest Magmas                         human cancers. <br /> may function multiple contexts [2].                                                   Human Magmas yeast ortholog, Pam 16, 42% <br />    Magmas variably expressed protein levels develop-                       identical exhibit functional conservation. Expression of <br /> mentally regulated mammalian cells [1,6]. During murine <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                1                                   July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                   Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br />  Table 1. Saccharomyces cerevisiae strains. <br />  <br />  <br />  Strain                                   Genotype                                                                                  Reference <br />  <br />  orfDarray                                MATa orfD::KanMX4 his3D leu2D met15D ura3D                                                [4] <br />  Y8835                                    MATa can1D::STE2pr-his5 lyp1D ura3D::natR leu2D0 his3D1 met15D0                           [87] <br />  Y7092                                    MATa can1D::STE2pr-his5 lyp1D ura3D0 leu2D0 his3D1 met15D0                                [87] <br />  pam16-I61N                               MATa pam16-I61N::NatR, can1D::MFA1-prHIS3 MFa1pr-LEU2 leu2D0 his3D1                       This study <br />  sur4D                                    MATa sur4D:::NatR, can1::MFA1-prHIS3 MFa1pr-LEU2 leu2D0 his3D1                            This study <br />  pam16-I61N sur4D                         MATa pam16-I61N::NatR, sur4D::URA3, can1 ::MFA1-prHIS3 MFa1pr-LEU2 leu2D0 his3D1          This study <br />  W303                                     MATa leu2-3, 112 trp1-1 can1-100 ura3-1 ade2-1 his3-11,15                                 [85] <br />  <br />  doi:10.1371/journal.pone.0039428.t001 <br />  <br />  <br /> human Magmas yeast fully complements lethal <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                  animals [2]. Pam16-I61N cells incubated 7 days YEPD <br /> of PAM16 null cells [12,13] validates use yeast               37uC remained fully viable cultured 30uC same <br /> model study fundamental properties human                 numbers colonies obtained duplicate plates grown at <br /> protein. Pam16 regulatory component presequence                   30uC 3 days (data shown). <br /> translocase-associated motor (PAM), responsible                 The temperature dependent growth pam16-I61N strain on <br /> transport proteins mitochondrial matrix [14,15,16].               fermentable (glucose) non-fermentable (glycerol/ethanol) car- <br /> Pam16 binding Pam18 results recruitment stabiliza-                  bon sources shown Figure S1. The pam16-I61N strain <br /> tion Pam18 TIM23 translocase complex, located                    exhibited strong ts phenotype media the <br /> the inner mitochondrial membrane. These interactions stimulate                temperature required completely inhibit growth lower on <br /> the ATPase activity Ssc1 (the yeast homolog Hsp70)                glycerol/ethanol. Since pam16-I61N mutation affects fermen- <br /> produces driving force responsible translocation               tative growth addition respiratory metabolism, appears that <br /> proteins mitochondrial matrix [12,13,16,17,18,19,20].                Pam16 affect metabolic pathways outside mitochon- <br />    <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> better understand Magmas function conducted genetic                  drial matrix. <br /> and functional studies PAM16. A temperature sensitive PAM16 <br /> mutation (I61N) used identify synthetic sick lethal (SSL)          Synthetic Genetic Interactions <br /> genetic interactions PAM16. In addition genetic interactions               The pam16-I61N mutation introduced Y7092 strain <br /> that support role Pam16 mitochondrial protein import,               (Table 1) serve query strain SGA analysis. A control <br /> novel interactions PAM16 identified suggesting                    strain (Y8835) screened parallel pam16-I61N xxxD <br /> involvement functions including lipid metabolism                 double mutant colonies identified failed grow well <br /> transcription. Genetic screening identified gene deletions          pam16-I61N 30uC 32uC. pam16-I61N xxxD colonies were <br /> suppressed pam16-I61N ts proliferation defect. SUR4, encoding             incubated restrictive temperature 34uC identify <br /> a fatty acid elongase involved sphingolipid biosynthesis [21,22],          gene-deletions suppress loss PAM16 function. Sixty <br /> was effective suppressor. Strains containing pam16-I61N              candidate synthetic genetic interactions initially identified <br /> with deletion SUR4 additionally characterized            using automated image analysis visual inspection. Forty-six of <br /> by proliferation studies, cell cycle analysis, morphology,                verified synthetic sick lethal (SSL) interactions by <br /> sphingolipid profiles. Our results suggest deleterious               random spore analysis (Table 2). <br /> effects impaired PAM16/Magmas activity mediated                   Fourteen 46 SSL partners petite phenotype the <br /> through changes sphingolipid metabolites.                                  original gene-deletion array severely growth <br />                                                                               impaired double mutant (Table 2). Petite strains not <br /> Results                                                                       respiration competent mitochondria. Since 227 the <br />                                                                               4285 gene-deletion array strains &#226;&#8364;&#732;&#226;&#8364;&#732;respiratory growth <br /> A Temperature-sensitive PAM16 Mutant Exhibits Defects                         absent&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype [23], 5.7-fold enrichment this <br /> in Fermentation Respiration                                               phenotype screen (p,0.01) supports role PAM16/ <br />    Microscopic examination spores dissected PAM16                     Magmas specific interactions genes involved the <br /> heterozygous null asci revealed normal colonies single            regulation oxidative metabolism [13,24]. <br /> cells (data shown) demonstrating haploid null spores                Figure S2 shows interaction network PAM16 SSL <br /> unable undergo single cell division confirming                  genes depicted nodes colored according gene ontology (GO) <br /> PAM16 essential proliferation. Thus characterize                biological process designations [25] edges representing known <br /> functional roles Magmas, conditional alleles PAM16                 physical genetic interactions. When genes categorized <br /> necessary. Temperature sensitive alleles created random               according biological function, major groups apparent: <br /> mutagenesis screened inability grow elevated              mitochondrial lipid metabolism, histone modification, mitochon- <br /> temperatures. Four unique mutant strains displayed impaired              drial protein import, peroxisome biogenesis glycolysis <br /> growth 37uC glucose glycerol/ethanol media (YPGE)                (Figure 1). These groups served basis analysis of <br /> contained single amino acid substitution. The strain              Pam16 function. <br /> grew best 30uC did grow 37uC change at <br /> amino acid 61 isoleucine asparagine (I61N). This amino                Mitochondrial Lipid Metabolism <br /> acid identical Pam16 Magmas located                    The functional group containing largest number of <br /> sequence high homology (EX3IL) conserved fungi, plants              interacting genes established synthetic relationship between <br />  <br />  <br />          PLoS ONE | www.plosone.org                                       2                                 July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                               Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br /> Table 2. pam16-I61N synthetic sick lethal partners.                                    Table 2. Cont. <br />  <br />  <br />                 Systemic                                                                  RPM2, essential gene opposite DNA strand transcription is <br /> Gene           SSLa       petite Functional groupb                              inhibited eliminated YML090W deletion (data shown). <br />                                                                                           d <br />                                                                                             The interaction GRC2 confirmed deletion YNL198c, an <br /> CRD1            ydl142c     +++              Mitochondrial Lipid Metabolism               overlapping dubious open reading frame opposite DNA strand. <br /> ETR1 *          ybr026c     ++      X        Mitochondrial Lipid Metabolism               doi:10.1371/journal.pone.0039428.t002 <br />  <br /> HFA1 *          ymr207c     +++              Mitochondrial Lipid Metabolism              PAM16 12 genes involved mitochondrial lipid metabolism <br /> LAT1 *          ynl071w     +++              Mitochondrial Lipid Metabolism              (Figure 1, blue nodes Table 2). Ten genes encode <br /> LIP2 *          ylr239c     +       X        Mitochondrial Lipid Metabolism              proteins located mitochondrial matrix (Table 2) and <br /> LIP5 *          yor196c     +++     X        Mitochondrial Lipid Metabolism              involved mitochondrial specific fatty acid synthesis the <br />                                                                                          lipoylation proteins [26,27,28]. The genes, CRD1 and <br /> LPD1 *          yfl018c     +       X        Mitochondrial Lipid Metabolism <br />                                                                                          TAZ1, essential biosynthesis cardiolipin the <br /> MCT1 *          yor221c     +++     X        Mitochondrial Lipid Metabolism <br />                                                                                          mitochondrial inner membrane. Cardiolipin important for <br /> PDA1 *          yer178w     +++              Mitochondrial Lipid Metabolism              maintaining mitochondrial membrane potential [29], vacuolar <br /> PDB1            ybr221c     ++               Mitochondrial Lipid Metabolism              homeostasis [30,31] stabilizing respiratory chain complexes <br /> RPM2            yml090wc +++        X        Mitochondrial Lipid Metabolism              [32]. <br /> TAZ1 *          ypr140w     +                Mitochondrial Lipid Metabolism <br /> BRE1            ydl074c     +++              Histone Modification <br />                                                                                          Histone Modification <br />                                                                                             The second numerous SSL gene set includes 7 genes <br /> DEP1            yal013w     ++      X        Histone Modification <br />                                                                                          involved histone modification (Figure 1, green nodes and <br /> LGE1            ypl055c     +++              Histone Modification <br />                                                                                          Table 2). Five genes involved Rpd3L histone <br /> PHO23           ynl097c     +++              Histone Modification                        deacetylase complex, regulates transcription distinct <br /> RXT2            ybr095c     ++               Histone Modification                        promoter regions [33,34]. Rpd3L negatively regulates genes <br /> SAP30           ymr263w     ++               Histone Modification                        encoding mitochondrial proteins proteins involved meiosis <br /> SIN3            yol004w     +++     X        Histone Modification <br />                                                                                          metabolism. The positively regulated genes involved in <br />                                                                                          heat shock, osmotic stress fermentative growth [35,36]. All the <br /> FMP18           ykr065c     +++              Mitochondrial Protein Import <br />                                                                                          genes histone modification group hundreds physical <br /> MMM1 *          yll006w     +++     X        Mitochondrial Protein Import                genetic interactions (Saccharomyces Genome Database) <br /> PHB2            ygr231c     +++              Mitochondrial Protein Import                shown pam16-I61N SSL partner set (Figure <br /> TOM37 *         ymr060c     +++              Mitochondrial Protein Import                S2, inner circle). These seven genetic interactions associated with <br /> TOM70           ynl121c     +++              Mitochondrial Protein Import                histone modification previously linked PAM16. <br /> YME1            ypr024w     +++     X        Mitochondrial Protein Import <br /> PEX29           ydr479c     +++              Peroxisomal b-oxidation <br />                                                                                          Mitochondrial Protein Import <br />                                                                                             A functional cluster consists genes involved in <br /> PEX30           ylr324w     +++              Peroxisomal b-oxidation <br />                                                                                          mitochondrial protein import. Pam16 constituent the <br /> PFK2 ?          ymr205c     +                Cytoplasm (Glycolysis)                      presequence translocase-associated motor (PAM) complex. Six <br /> ATP11 *         ynl315c     +++     X        Mitochondrial Matrix                        genes mitochondrial membrane protein import system <br /> CAF40 *         ynl288w     +++              Cytoplasm                                   resulted significantly impaired growth pam16-I61N <br /> CDC73 *         ylr418c     +++              Nucleus                                     (Figure 1, red nodes Table 2). MMM1, TOM37, and <br />                                                                                          TOM70 mitochondrial outer membrane proteins involved in <br /> CTK1 *          ykl139w     +++              Cytoplasm And Nucleus <br />                                                                                          protein import additionally role phospholipid <br /> DBF2 *          ygr092w     +++              Cytoplasm &amp;Bud Neck <br />                                                                                          biosynthesis calcium exchange [37,38,39]. PHB2, YME1, and <br /> GCR2 *          ynl199cd    +++              Nucleus                                     FMP18 associated mitochondrial inner membrane. <br /> KAR3 *          ypr141c     +++              Microtubule, Spindle Pole Body              Phb2 [40] subunit prohibitin complex, chaperone for <br /> MCK1 *          ynl307c     +                Cytoplasm Nucleus                       newly synthesized proteins [41]. YME1 codes catalytic <br /> MDL2 *          ypl270w     ++      X        Mitochondrion Inner Membrane                subunit mitochondrial inner membrane i-AAA protease <br />                                                                                          complex, degrades misfolded proteins mitochondrial <br /> MRT1 *          ycr077c     +++              Cytoplasm <br />                                                                                          inner membrane matrix [42,43]. Of genes group <br /> RPL19B *        ybl027w     +                Cytosolic Large Ribosomal Subunit <br />                                                                                          Fmp18 (Pam17) reported physically interact with <br /> RVS167 *        ydr388w     ++      X        Actin Cortical Patch, Cytoplasm             Pam16 [44]. Twenty-two 46 SSL partners pam16-I61N <br /> SWS2 *          ynl081c     +       X        Mitochondrial Small Ribosome                common genetic physical interactions <br /> THR1 *          yhr025w     ++               Amino Acid Biosynthesis                     previously identified genes/proteins involved in <br /> YBR238C         ybr238c     +++              Mitochondrion <br />                                                                                          mitochondrial protein import (data shown) [23]. This supports <br />                                                                                          findings Pam16 involved biological functions. <br /> YDR290W         ydr290w     +                Unknown <br /> YJR120W *       yjr120w     +++              Unknown <br />                                                                                          Peroxisome Biogenesis Glycolysis <br /> YHL039W         yhl039w     ++               Cytoplasm                                      Negative synthetic interactions observed genes <br /> *: synthetic sick lethal interaction suppressed additional deletion SUR4.       encoding integral membrane proteins peroxisome, an <br /> a <br />   Severity SSL defect low+ high+++.                                           organelle responsible production fatty acids energy <br /> b <br />   Functional group Figure 1 GO cellular component process.                    intermediates degradation complex lipids (Figure 1, <br /> c <br />  The interaction YML090W, dubious open reading frame, replaced             yellow nodes Table 2). PEX29 PEX30 genes required <br />  <br />  <br />          PLoS ONE | www.plosone.org                                                  3                                 July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                              Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br />  <br /> Figure 1. Synthetic lethal partners pam16-I61N. 28 46 SSL partners pam16-I61N grouped 4 major biological functions. <br /> They mitochondrial protein import (red), histone modification (green), mitochondrial lipid metabolism (blue), peroxisome biogenesis and <br /> glycolysis (yellow). Genes functionally related radiate central point. <br /> doi:10.1371/journal.pone.0039428.g001 <br />  <br /> for regulating peroxisome size, number distribution [45,46].            I61N screened deletion array using pam16-I61N sur4D <br /> PFK2 encodes phosphofructokinase major glycolytic enzyme.                 double mutant query strain (Table 2). Nineteen 46 SSL partners <br />                                                                             pam16-I61N retained SSL phenotype remaining <br /> Genetic Suppressors pam16-I61N Involved                           27 SSL interactions suppressed deletion SUR4. The <br /> Sphingolipid Metabolism                                                     sur4D suppression phenotypes divided functional <br />    Five gene deletions (SUR4, ISC1, IPT1, SKN1, FEN1)              lines. No SSL interactions histone modification group were <br /> identified suppressors temperature-sensitive phenotype         suppressed. Most SSL interactions mitochondrial protein <br /> pam16-I61N 34uC. All genes involved                      import genes suppressed. However, genes this <br /> sphingolipid metabolism. SUR4 encodes elongase synthe-              group (TOM37 MMM1) associated outer mitochon- <br /> sizes long chain 20&#226;&#8364;&#8220;26-carbon fatty acids C18-CoA                 drial membrane longer SSL pam16-I61N SUR4 <br /> primers involved sphingolipid biosynthesis [21]. The              deleted. Similarly, 9 12 SSL interactions pam16-I61N <br /> Fen1 elongase similar role, maximum fatty acid chain           mitochondrial lipid metabolism genes suppressed by <br /> <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> 24 carbons [22]. Isc1 (inositol phosphosphingolipid               deletion SUR4. The genes group remained SSL were <br /> phospholipase C) hydrolyzes complex phosphorylceramides                  CRD1, PDB1 RPM2. <br /> produce alpha-hydroxy-phytoceramides polar headgroups <br /> [47,48]. Deletion ISC1 causes defects mitochondrial function          Growth Characterization Wild Type, pam16-I61N, sur4D <br /> and failure up-regulate genes involved aerobic metabolism             pam16-I61N sur4D Strains <br /> after diauxic shift [49,50].                                               The growth wild type (wt), pam16-I61N, sur4D pam16- <br />    Ipt1 (inositolphosphotransferase) Skn1 (a putative glucosyl-         I61N sur4D strains evaluated temperatures used in <br /> transferase) involved synthesis mannose-(inositol-       SGA screens. Growth curves YEPD media typical <br /> P)2-ceramide (M(IP)2C), abundant complex sphingolipid              rapid initial rise cell density (characteristic fermentative <br /> in yeast [51,52]. Skn1 Ipt1 complement                growth) depletion glucose results switch to <br /> M(IP)2C synthesis limited nutrients skn1D ipt1D               respiratory metabolism (diauxic shift) slower growth prior to <br /> strains increased autophagy DNA fragmentation [53].                entry stationary phase [54,55] (Figure 2A&#226;&#8364;&#8220;C). The fermen- <br /> Deletion FEN1, IPT1 SKN1 suppressed 34uC ts                       tative doubling time strains temperature shown in <br /> phenotype pam16-I61N deletion                 Figure 2D. The wt, pam16-I61N, sur4D pam16-I61N sur4D <br /> ISC1 SUR4. Of suppressor strains, pam16-I61N                strains grew 30uC glucose growth rate the <br /> sur4D strain showed strongest growth phenotype                  mutant strains slower wt cells. <br /> chosen analysis. Notably, suppressor phenotype                 The reduced ability pam16-I61N grow glucose 32uC <br /> requires partially functional pam16 allele suppress       inability grow 34uC partially corrected wt levels <br /> the lethality PAM16 deletion (data shown).                         deletion SUR4 (Figure 2B&#226;&#8364;&#8220;D, Figure 3, four <br />                                                                             rows). This data agreement SGA screen, which <br /> Synthetic Genetic Interactions pam16-I61N sur4D                        showed proliferation pam16I61N glucose containing <br /> Double Mutant Query Strain                                                  media restored 34uC deletion SUR4. The sur4D <br />   To understand functional relationship sur4D                   suppression pam16-I61N growth defect media containing <br /> suppression pam16-I61N SSL interactions pam16-              glucose similarly occurs rho0 strains, lack functional <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        4                              July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                   Pam16 Sphingolipid Metabolism <br />  <br />  <br />                                                                                 mitochondria (Figure S3). The rho0 results suggest Pam16 <br />                                                                                 provides essential activity fermentative growth is <br />                                                                                 independent role presequence translocase other <br />                                                                                 mitochondrial functions. <br />                                                                                    No growth observed pam16-I61N diauxic shift at <br />                                                                                 32uC 34uC (Figure 2B,C; blue line). At 30uC pam16-I61N <br />                                                                                 grew slower wt strain sur4D strains. In contrast all <br />                                                                                 strains pam16-I61N sur4D unable proliferate at <br />                                                                                 30uC following diauxic shift (Figure 2A; red line). This <br />                                                                                 demonstrates deletion SUR4 increases severity the <br />                                                                                 respiratory growth defect pam16-I61N strain. <br />                                                                                    In addition long chain fatty acids, synthesis of <br />                                                                                 sphingolipids requires long chain <span id='am-319' about='xsp:base' typeof='owl:Thing'>base</span> (LCB) polar head <br />                                                                                 group. LCB synthesis normally required viability [56] and <br />                                                                                 potential functional relationship LCBs and <br />                                                                                 pam16-I61N assessed screening deletion array. We <br />                                                                                 examined sur4D suppression pam16-I61N <br />                                                                                 phenocopied pharmacological reduction LCB <br />                                                                                 synthesis using myriocin, inhibitor serine palmitoyltransfer- <br />                                                                                 ase [57]. Myriocin 400 ng/ml slowed growth strains <br />                                                                                 consistent global inhibition LCB synthesis (Figure 3). <br />                                                                                 However, dose inhibitor lower <br />                                                                                 concentration (200 ng/ml, did substantially affect <br />                                                                                 growth wt strain) suppressed pam16-I61N (ts) phenotype. <br />  <br />                                                                                 Cell Cycle Analysis Strains Containing pam16-I61N and <br />                                                                                 sur4D <br />                                                                                    The cell cycle profile wt, pam16-I61N, sur4D pam16-I61N <br />                                                                                 sur4D strains assessed flow cytometry 30uC 34uC to <br />                                                                                 determine cells arrested specific phase (Figure 4). <br />                                                                                 During normal exponential growth 30uC YEPD (Figure 4, <br />                                                                                 row 1), cell cycle analysis showed 29% wt cells G1 <br />                                                                                 compared 42% pam16-I61N cells. The sur4D pam16- <br />                                                                                 I61N sur4D strains fewer cells G1 wt pam16- <br />                                                                                 I61N strain. <br />                                                                                    When shifted non-permissive temperature 34uC <br />                                                                                 (Figure 4, row 2) wt cells continued replicate similar cell <br />                                                                                 cycle profile 6 h 24 h slightly cells accumulating in <br />                                                                                 G1 30uC. In contrast, majority pam16-I61N cells <br />                                                                                 arrested G1. This G1 arrest phenotype quantitatively <br />                                                                                 complete cells shifted 37uC (data shown). The G1 arrest in <br />                                                                                 pam16-I61N relieved pam16-I61N sur4D strain (Figure 4, <br />                                                                                 row 3), correlates improved fermentative growth the <br />                                                                                 double mutant strain 34uC (Figure 2C, D, Figure 3). <br />  <br />                                                                                 Strains Containing pam16-I61N sur4D Altered <br />                                                                                 Morphological Features <br />                                                                                    Wild type, pam16-I61N, sur4D, pam16-I61N sur4D cells <br />                                                                                 grown 32uC examined transmission electron micros- <br />                                                                                 copy (Figure 5 Figure S4). Cell walls, cell membranes the <br />                                                                                 nucleus similar strains. In contrast, pam16-I61N <br />                                                                                 mutant fewer smaller mitochondria wt cells <br />                                                                                 (Figure 5A, B; horizontal arrows). In addition, pam16-I61N <br />                                                                                 strain contained large vesicles electron dense material in <br /> Figure 2. Deletion SUR4 suppresses fermentative growth <br /> defect pam16-I61N 34uC. Effect temperature                      center thickened walls (small angled arrows). These <br /> proliferation PAM16 SUR4 mutant strains determined              structures apparent wt strain. <br /> a Bioscreen incubator. Strains grown 48 h 30uC (A), 32uC (B),          The mitochondria sur4D strain (Figure 5C) appeared <br /> or 34uC (C). Growth curves generated mean values               similar wt strain. The sur4D strain contained an <br /> optical density readings taken 30 minutes triplicate wells.          additional feature, large number highly electron dense vesicles <br /> Wild type (black), pam16-I61N (blue), sur4D (green), pam16-I61N <br /> sur4D (red). D. Doubling times (h) yeast strains log phase <br />                                                                                 (vertical arrows). pam16-I61N sur4D cells (Figure 5D) improved <br /> fermentative growth. *pam16-I61N attains log phase growth              mitochondrial morphology contained smaller vesicles com- <br /> 34uC.                                                                           pared pam16-I61N cells. Similar sur4D cells, pam16-I61N <br /> doi:10.1371/journal.pone.0039428.g002                                           sur4D strain numerous vesicles containing highly <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           5                              July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br />  <br /> Figure 3. Effects temperature disruption sphingolipid synthesis proliferation. Five fold serial dilutions wt, pam16-I61N, <br /> sur4D, pam16-I61N sur4D grown YEPD YEPD indicated concentrations serine palmitoyltransferase inhibitor (myriocin) <br /> at temperatures shown. Myriocin (400 ng/ml) impaired growth strains, 200 ng/ml 400 ng/ml suppressed pam16- <br /> I61N (ts) growth phenotype, unlike specific suppression sur4D. <br /> doi:10.1371/journal.pone.0039428.g003 <br />  <br /> electron dense material (Figure 5C, D; upward vertical arrows)                SUR4 showed abundant electron dense vesicles, be <br /> that 10&#226;&#8364;&#8220;26 fold numerous present wt                peroxisomes (Figure 5C, D Figure S4C, D). To determine if <br /> pam16-I61N cells. Live cell fluorescent images (Figure S5)             vesicles peroxisomes, performed live cell imaging <br /> strains transformed mitochondrial targeted GFP [58]               using Pot1-GFP fusion protein. Pot1 specific marker for <br /> showed restoration elongated mitochondrial morphology                   mature functional peroxisomes. <br /> pam16-I61N sur4D compared small fragmented mitochon-                      Fluorescence microscopy showed wt strain grown in <br /> dria pam16-I61N strain.                                                glucose few, small peroxisomes (0.6 peroxisomes/cell). The <br />                                                                               fluorescence intensity Pot1-GFP weak consistent with <br /> Strains Containing pam16-I61N sur4D Altered                          low protein expression observed western blot (Figure 6A, F). <br /> Numbers Peroxisomes                                                        Growing wt cells 4 h glycerol/ethanol oleic acid resulted <br />    Pex29 Pex30 peroxisomal membrane proteins                     increase peroxisome number (3.7&#226;&#8364;&#8220;4.1 peroxisomes/cell), <br /> control peroxisome size, number, distribution. The absence             increased organelle size, increased fluorescence intensity as <br /> either genes reduced pam16-I61N viability (Figure 1              expected (Figure 6A E). These changes seen in <br />                                                                               pam16-I61N. The pam16-I61N strain approximately twice as <br /> Table 2). The electron microscopy strains deleted in <br />                                                                               peroxisomes wt cells glucose exhibited increase <br />                                                                               peroxisome number fluorescence intensity following culture <br />                                                                               glycerol/ethanol oleic acid (Figure 6B, E). <br />                                                                                  The sur4D pam16-I61N sur4D strains 4.1 3.2 fold, <br />                                                                               respectively, peroxisomes wt strain grown glucose <br />                                                                               (Figure 6C&#226;&#8364;&#8220;E). In glycerol/ethanol oleic acid containing media, <br />                                                                               number fluorescent structures increased sur4D (2.2&#226;&#8364;&#8220;2.4 <br />                                                                               fold) pam16-I61N sur4D (1.9 fold glycerol/ethanol, 1.3 <br />                                                                               fold oleic acid). The intensity Pot1-GFP sur4D cells was <br />                                                                               higher pam16-I61N sur4D cells. Quantitation Pot1- <br />                                                                               GFP levels western blot confirmed strains with <br />                                                                               pam16-I61N reduced Pot1 expression (Figure 6F). <br />                                                                                  Increased peroxisome production Saccharomyces cerevisiae occurs <br />                                                                               response changes carbon source involves multiple <br />                                                                               signaling pathways leading transcriptional activation genes <br />                                                                               encoding peroxisomal proteins (reviewed [59,60]). Pot1 expres- <br />                                                                               sion repressed glucose, derepressed glycerol/ethanol and <br />                                                                               induced oleic acid containing media [61]. Although sur4D and <br />                                                                               pam16-I61N sur4D cells grow glucose 32uC (Figure 3), they <br />                                                                               higher baseline peroxisome content (Figure 6) suggesting <br />                                                                               partial derepression peroxisomal protein gene transcription. <br />                                                                               Since SUR4 deletion results increased peroxisome content in <br /> Figure 4. G1 cell cycle arrest pam16-I61N mutant strain                 glycerol/ethanol oleate strains overcome block in <br /> 34uC relieved sur4D. Flow cytometry wt (Y8835), pam16-I61N,          peroxisome induction characteristic pam16-I61N strain. <br /> sur4D pam16-I61N sur4D. Cultures kept mid-logarithmic <br /> phase YEPD 30uC shifted 34uC 6 24 h. Histograms       Sphingolipid Composition Altered pam16-I61N <br /> the number cells G1, S G2 DNA content measured by <br /> Sytox Green fluorescence. The mean percentage G1 cells indicated        Strain <br /> below G1 peak.                                                              Sphingolipids consist long-chain base amide linked long <br /> doi:10.1371/journal.pone.0039428.g004                                         chain fatty acid. They critical plasma membrane structure, <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                              July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                 Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br />  <br /> Figure 5. Effects pam16-I61N sur4D yeast morphology. Yeast strains examined transmission electron microscopy. pam16- <br /> I61N (Panel B) fewer smaller mitochondria (horizontal arrows) wt cells (Panel A). pam16-I61N cells vesicles thickened walls and <br /> no electron dense material center (upward angled arrows). The sur4D mutation restores near normal mitochondrial morphology pam16- <br /> I61N strain increases peroxisome number (upward vertical arrows Panels C D)(12,0006 magnification; scale bar = 500 nm). <br /> doi:10.1371/journal.pone.0039428.g005 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                               July 2012 | Volume 7 | Issue 7 | e39428 <br />                                        Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org   8   July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                 Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br /> Figure 6. Effects pam16-I61N sur4D peroxisome formation glucose, glycerol/ethanol oleic acid containing media. <br /> Live-cell fluorescence microscopy (top frames) corresponding differential interference contrast (DIC) image (bottom frames) wt (A), pam16- <br /> I61N (B), sur4D (C), pam16-I61N sur4D (D) cells containing GFP-tagged Pot1 identify peroxisomes. Cultures grown synthetic media 30uC <br /> with glucose grown 32uC additional 3 h glucose, glycerol/ethanol oleic acid carbon source. Quantitation of <br /> peroxisomes/cell (E) strain grown indicated carbon source. Western blot (F) Pot1-GFP (Mr = 75 kD) Pgk1 (Mr = 45 kD) wt, <br /> pam16-I61N, sur4D pam16-I61N sur4D cell lysates (40 mg protein/lane). Pgk1 used demonstrate Pot1 specifically induced by <br /> media glycerol/ethanol oleic acid carbon sources. Numbers indicate quantitation band volumes Pot1-GFP (61025) Pgk1 <br /> (61026) pixels. BD indicates band level detection. <br /> doi:10.1371/journal.pone.0039428.g006 <br />  <br /> lipid raft formation, play important roles numerous                   phytoceramides offset increase shorter carbon chain <br /> signaling pathways. The observation suppressors             phytoceramides. The levels alpha-hydroxy-phytoceramides with <br /> pam16-I61N involved sphingolipid metabolism suggested                 chain lengths C16 C20 increased pam16-I61N <br /> that changes sphingolipid levels                 mutant compared wt (Table S1 Figure 7). Most striking was <br /> biologically important. To investigate possibility measured          14 fold higher levels C18 alpha-hydroxy-phytoceramide <br /> the levels molecules strains                 (C18aHP) pam16-I61N strain compared wt (p = 0.043) <br /> quantitative mass spectrometry.                                              (Figure 7). <br />    Log phase wt, pam16-I61N, sur4D pam16-I61N sur4D strains                 Next compared deletion SUR4 affected <br /> cultured YEPD (30uC) incubated 6 h 34uC,               sphingolipid levels pam16-I61N strain. pam16-I61N sur4D <br /> temperature pam16-I61N cells lose ability                    resulted increased levels sphingolipids species <br /> proliferate, accumulate G1, remain fully viable. pam16-               compared pam16-I61N including sphingolipid bases <br /> I61N sur4D cells grow 34uC doubling times               dihydrosphingosine phytosphingosine (Table S1). The phos- <br /> 1.76 longer wt cells (Figure 2C, D). The levels                  phorylated forms bases elevated the <br /> predominant sphingoid bases, dihydrosphingosine phyto-                   unphosphorylated forms. pam16-I61N sur4D exhibited in- <br /> sphingosine, phosphorylated derivatives            creases C20:1 C22 alpha-hydroxy-phytoceramide low <br /> same wt pam16-I61N cells (Table S1). The highly abundant             levels long chain C24 C26 molecules expected <br /> C22&#226;&#8364;&#8220;26 alpha-hydroxy-phytoceramides unsaturated                      SUR4 deletion. Many changes sphingolipids were <br /> alpha-hydroxy-phytoceramides similar                  observed sur4D strain. Importantly, deletion SUR4, <br /> strains. The initial steps novo sphingolipid biosynthesis              itself effect level C18 alpha hydroxy <br /> therefore appear unaffected pam16-I61N mutation.                phytoceramides, suppressed elevated levels metabolite <br />    In contrast, amounts sphingolipids              pam16-I61N mutant essentially wt level (Table S1 and <br /> different pam16-I61N cells compared wt cells. C14 C16              Figure 7). C16aHP levels showed similar trend strains <br /> phytoceramide levels elevated pam16-I61N, C20&#226;&#8364;&#8220;                 statistically significant. <br /> C26 phytoceramides 3&#226;&#8364;&#8220;4 fold abundant pam16-                    The dramatic effect sur4D suppressing 14-fold increase <br /> I61N strain. This suggests reduced levels long chain             C18 alpha hydroxy phytoceramides pam16-I61N strain <br />                                                                              suggested similar effect underlie suppressor <br />                                                                              phenotypes sphingolipid biosynthetic genes. To test <br />                                                                              possibility, constructed suppressor strains deleting ISC1, <br />                                                                              IPT1, SKN1 FEN1 pam16-I61N strain performed <br />                                                                              sphingolipid profiling before. C18aHP 7.5 fold higher in <br />                                                                              pam16-I61N strain compared wt (Figure S7) for <br />                                                                              fen1D, weakest suppressor, double mutant strains had <br />                                                                              C18aHP levels similar wt cells. These data identify C18aHP as <br />                                                                              important molecule underlie effects observe in <br />                                                                              fermentative growth pam16-I61N strain. <br />  <br />                                                                              Cardiolipin Levels Decreased pam16-I61N and <br />                                                                              pam16-I61N sur4D <br />                                                                                 Since phospholipids regulate Isc1 activity [62] CRD1 and <br />                                                                              TAZ1 SSL PAM16 (Figure 1, Figure S2, Table 2) we <br />                                                                              measured major phospholipid content wt, pam16-I61N, sur4D <br />                                                                              pam16-I61N sur4D strains layer chromatography (Figure <br />                                                                              S6). No differences phosphatidic acid, phosphatidylserine, <br />                                                                              phosphatidylcholine, phosphatidylethanolamine phosphatidyli- <br /> Figure 7. C18 alpha-hydroxy-phytoceramide levels elevat- <br />                                                                              nositol detected suggesting did contribute the <br /> ed pam16-I61N strain compared wt similar to <br /> wt pam16-I61N sur4D. Overnight yeast cultures wt, pam16-I61N,          growth morphology characteristics observed pam16-I61N <br /> sur4D, pam16-I61N sur4D strains grown YEPD 30uC               strain non-permissive temperatures. However, cardiolipin levels in <br /> diluted early log phase grown 6 h 34uC. The mean level       pam16-I61N 50% wt strain (p,0.05) further <br /> 6 SEM (pmole/nmole phosphate) C18 alpha-hydroxy-phytoceramide             reduced pam16-I61N sur4D 14% wt (p,0.01), n = 4. Since <br /> was determined mass spectrometry. C18 alpha-hydroxy-phytocer-             cardiolipin levels reduced pam16-I61N deletion of <br /> amide 12&#226;&#8364;&#8220;15 fold higher pam16-I61N strain compared wt, <br /> *p = 0.043, n = 3. The sur4D pam16-I61N sur4D strains C18            SUR4 cardiolipin involved fermentative growth defect. <br /> alpha-hydroxy-phytoceramide levels different wt            Decreased cardiolipin levels shown impair oxidative <br /> cells.                                                                       phosphorylation [29] suggesting loss respiratory growth <br /> doi:10.1371/journal.pone.0039428.g007 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                               July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                              Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br />  <br /> Figure 8. Illustration synthetic lethal interactions, metabolic pathways morphological features pam16-I61N pam16- <br /> I61N sur4D. (A) pam16-I61N strain. Nuclear, mitochondrial, peroxisome associated negative genetic interactions leading SSL phenotype are <br /> indicated blue. These interactions demonstrate importance Pam16 mitochondrial protein import, Rpd3L function nucleus, <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      10                             July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                Pam16 Sphingolipid Metabolism <br />  <br />  <br />  <br /> peroxisome biogenesis energy metabolism. Defects Pam16 result rudimentary mitochondria impaired fermentation oxidative <br /> phosphorylation. The increase C18aHP likely results Isc1 mediated breakdown complex sphingolipids. The inability pam16-I61N to <br /> induce peroxisomes results decreased beta-oxidation VLCFA. (B) pam16-I61N sur4D strain. Negative genetic interactions leading SSL <br /> phenotype indicated blue. Deletion genes (red bold font) resulted block sphingolipid biosynthesis pathway (red <br /> bar) suppressing fermentative growth defect pam16-I61N. The pam16-I61N sur4D stain morphologically restored mitochondria, increased <br /> numbers inducible peroxisomes wt levels C18 alpha-hydroxy-phytoceramide. Metabolic intermediates complex sphingolipids <br /> synthesized endoplasmic reticulum Golgi shown bold font. Green arrows indicate changes (increased) (reduced). <br /> Abbreviations: DHS: dihydrosphingosine; DHSP: dihydrosphingosine-1-phosphate; PHS: phytosphingosine; PHSP: phytosphingosine-1-phosphate; <br /> VLCFA: long chain fatty acid. <br /> doi:10.1371/journal.pone.0039428.g008 <br />  <br /> (at temperatures) pam16-I61N sur4D (Figure 2 A&#226;&#8364;&#8220;C)              Biological Features pam16-I61N pam16-I61N sur4D <br /> mediated change cardiolipin levels.                                   After establishing importance pam16-I61N mutation <br />                                                                              sur4D suppressor evaluated biological effects of <br /> Discussion                                                                   mutations vivo. Significant morphological changes in <br />                                                                              pam16-I61N strain included small mitochondria enlarged <br /> PAM16 Genetic Interactions Function                                      clear-centered vesicles. Rudimentary mitochondria known <br />    To identify biological pathways affected Magmas, condi-                consequence defective mitochondrial protein import [67] <br /> tional mutants generated yeast ortholog PAM16. The               corroborating importance Pam16 function. The <br /> pam16-I61N mutant strain, unable proliferate                 vesicle accumulation seen pam16-I61N similar that <br /> glucose 34uC, used identify negative synthetic <br />                                                                              described mutants strains defects endoplasmic <br /> interactions determine effects impaired Pam16 <br />                                                                              reticulum Golgi trafficking phospholipids reflect <br /> function. We impaired Pam16 affects growth, cell <br />                                                                              altered sphingolipid metabolism endoplasmic reticulum <br /> morphology, mitochondrial protein import, oxidative metabolism, <br />                                                                              [68]. These large vesicles similar occur the <br /> fermentation, lipid/sphingolipid metabolism, peroxisome biosyn- <br />                                                                              unfolded protein response [69,70,71]. Another feature pam16- <br /> thesis, histone modification. <br />                                                                              I61N cells failure induce peroxisomes grown on <br />                                                                              oleate glycerol/ethanol containing media. This differs wt <br /> Suppressors pam16-I61N Participate Sphingolipid                        strains increase peroxisome biogenesis fatty acid <br /> Biosynthesis                                                                 oxidation grown carbon sources [72]. <br />    To detect additional genes functionally connected PAM16                The absence Sur4 reversed biological effects seen <br /> screened gene-deletion array second site suppressors          pam16-I61N strain. pam16-I61N sur4D cells larger <br /> pam16-I61N. Five genes (SUR4, ISC1, IPT1, SKN1 FEN1)                mitochondria, fewer large clear vesicles pam16-I61N. <br /> identified deleted rescued fermentative            The improvement mitochondrial morphology observed the <br /> respiratory proliferation defect pam16-I61N strain 34uC.           deletion SUR4 pam16-I61N result normalization <br /> All suppressors involved sphingolipid biosynthesis           Pam16 function reduced activity specific inhibitory <br /> (Figure 8B) indicating important role sphingolipids            sphingolipids (for example, C18aHP). Significantly, pam16-I61N <br /> pam16-I61N phenotype [47,63,64,65]. The lack pam16-I61N                   sur4D recovered ability increase peroxisome number size <br /> suppression myriocin treatment suggests reversal            grown YPGE oleic acid. The recovery perixosome <br /> the pam16-I61N fermentative growth phenotype deletion of <br />                                                                              synthesis double mutant correlates resumption of <br /> SUR4, ISC1, IPT1, SKN1 FEN1 specific. <br />                                                                              fermentative growth. Alleviation unfolded protein response <br />                                                                              requires lipid biosynthesis ER [69] precursor molecules <br /> sur4D Rescues Subset pam16-I61N SSL Interactions                        required lipid biosynthesis supplied peroxisomal <br />    To determine pam16-I61N SSL interactions                   degradation long chain fatty acids (VLCFA). Thus recovery <br /> compensated sur4D, gene-deletion array queried               G1 arrest increased availability lipid <br /> the double mutant (Table 2). Synthetic lethal genes             intermediates substrates energy production. <br /> mitochondrial lipid metabolism Pfk2, glycolytic enzyme, <br /> were corrected absence SUR4 demonstrating interde- <br /> pendencies pathways (Figure 1 Table 2). In <br />                                                                              Elevated C18 Alpha-hydroxy-phytoceramide Levels are <br /> contrast, deletions genes involved Rpd3L complex               Observed pam16-I61N <br /> the PEX genes remained synthetic lethal pam16-I61N                     Sphingolipids incorporated plasma organelle <br /> sur4D. Thus, compromised Rpd3L function compromised                       membranes, lipid rafts cytoplasmic protein complexes. The <br /> peroxisome biogenesis impedes growth Pam16 deficient cells             sphingolipid content membranes known affect many <br /> mechanisms distinct Pfk2 mitochondrial                 cellular processes including endocytosis, exocytosis actin <br /> lipid metabolism genes described above.                                      cytoskeleton dynamics, calcium signaling, transcription and <br />    The deletion SUR4 results increased peroxisome biosyn-              translation, cell cycle control, differentiation, stress resistance <br /> thesis restoration fermentative growth pam16-              aging (reviewed [47,63]). In addition role in <br /> I61N strain. However, deletion SUR4 unable suppress             membrane biology, sphingolipids metabolites act as <br /> slow growth phenotype pam16-I61N pex29D pam16-I61N                     intracellular signaling molecules coordinate nutrition and <br /> pex30D. This reflect requirement PEX29 PEX30                energy production sterol metabolism [73], serine utiliza- <br /> the efficient production peroxisomes, bypassed            tion, phospholipid metabolism [74]. <br /> by deletion SUR4. Increased peroxisome number pam16-I61N                  The pam16-I61N strain altered levels sphingolipid <br /> sur4D cells necessary maintaining energy production               metabolites compared wt cells (Table S1). Some changes <br /> via beta-oxidation and/or reducing detrimental reactive                  sphingolipid content pam16-I61N cells corrected towards <br /> oxygen species result impaired electron transfer             wt levels deletion SUR4. The dramatic effect <br /> in oxidative phosphorylation complexes [66].                             occurred C18aHP, 14 fold higher levels pam16- <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       11                               July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                              Pam16 Sphingolipid Metabolism <br />  <br />  <br /> I61N wt strains restored wt levels pam16-            extra mitochondrial locations cytoplasm [1], could <br /> I61N sur4D (Figure 7 Figure 8A B). Since deletion            direct interaction Magmas SMase2 their <br /> SUR4 restored fermentative proliferation               respective counterparts yeast cytoplasm. <br /> morphological changes resulting pam16-I61N, C18aHP could <br /> be important mediator effects mutation. Studies        The Inhibitory Effects Pam16-I61N Respiratory <br /> Neurospora crassa demonstrate importance C18aHP. In             Growth Independent Effect Fermentative <br /> this organism heat stress carbohydrate deprivation specifically <br /> induced C18aHP, resulted cell death [75]. <br />                                                                             Growth <br />                                                                                The Pam16 mutant strain unable sustain respiratory <br />                                                                             growth non-permissive temperature (Figure S1 Figure 2). <br /> Potential Mechanisms pam16-I61N Increases                          Unlike fermentative growth, deletion genes involved sphingo- <br /> C18aHP Levels During Fermentative Growth                                    lipid synthesis (SUR4, ISC1, IPT1, SKN1 FEN1) did correct <br />    It unclear dramatic increase C18aHP levels          respiratory growth defect (Figure 2 data shown). pam16- <br /> a cause consequence fermentative growth arrest               I61N resulted reduced cardiolipin levels mitochondrial <br /> pam16-I61N. However suppressor strains reduced                 dysfunction consistent severe compromises mitochon- <br /> pam16-I61N induced elevation C18aHP (Figure S7) suggesting               drial morphology (Figure 5 S5). Decreased levels cardiolipin <br /> that increased levels C18aHP responsible impaired            shown specifically impair respiratory super complex <br /> fermentative growth. The elevated levels C18aHP observed              formation cytochrome oxidase activity [32,79,80]. <br /> the pam16-I61N strain result increased synthesis             Studies Blp Drosophila homolog Pam16, consistent <br /> or reduced degradation.                                                     findings observed Saccharomyces cerevisiae during <br />    Since Pam16 required mitochondrial presequence                    respiratory growth. Drosophila S2 cells depleted Blp RNAi <br /> translocase activity C18aHP enriched mitochondrial                caused mitochondrial membrane depolarization, decreased ATP <br /> membranes [50], reasonable consider pam16-I61N                levels, increased ROS, cell cycle arrest uncoupling oxidative <br /> would result increased accumulation C18aHP reducing its <br />                                                                             phosphorylation [66]. Blp depletion preferentially inhibited <br /> mitochondrial utilization degradation. This require that <br />                                                                             cytochrome oxidase activity, preceded inhibition of <br /> C18aHP enter mitochondria translocase dependent <br />                                                                             protein import mitochondrial matrix. Similarly, Blp <br /> carrier protein. Alternatively pam16-I61N affect C18aHP <br />                                                                             inhibitor (SMMI: small molecule Magmas inhibitor) [81], reduced <br /> degradation pathways instead mitochondrial uptake. <br />                                                                             mitochondrial membrane potential cytochrome oxidase <br /> However possibilities consistent 5 <br />                                                                             activity prior detectable effects presequence translocase. <br /> deletion gene suppressor results. <br />    Instead data consistent hypothesis that <br /> elevated C18aHP pam16-I61N predominantly result              Magmas Human Malignancy <br /> increased synthesis complex sphingolipids                Our findings yeast suggest Magmas involved in <br /> de novo synthesis pathway (Figure 8). Deletion SCS7, gene              human malignancy. Unlike normal counterparts malignant <br /> required novo synthesis alpha-hydroxy-phytoceramides              cells use aerobic glycolysis energy requirements. Aerobic <br /> [76], did suppress pam16-I61N slow growth phenotype                 glycolysis results alterations levels cellular <br /> (data shown). This implies accumulation C18aHP              metabolites involved pathogenesis of <br /> results increased degradation complex sphingolipids             cancer (reviewed [82,83]). Since Pam16 significant effects <br /> the lower levels suppressed strains results reduced                 respiration fermentation likely Magmas <br /> degradation complex sphingolipids.                                       contributes metabolic alterations leading initiation <br />    Deletion ISC1 results decreased cellular C18aHP levels             maintenance malignant phenotype. <br /> [50]. The abundant Isc1 substrates wt strains those <br /> sphingolipids C26 long chain fatty acids (VLCFA)             Materials Methods <br /> are synthesized Sur4, abundant Isc1 product is <br /> C26 alpha-hydroxy-phytoceramide [50]. The pam16-I61N sur4D                  Yeast Strains Media <br /> strain C18aHP C26 VLCFAs                    Strains used study listed Table 1. rho0 derivatives <br /> in sur4D strains short chain ceramides used cell          isolated parental strains rounds growth to <br /> place ceramides contain VLCFAs.                                     saturation minimal medium containing 2% glucose plus 25 mg/ <br />    Our data suggests pam16-I61N induced elevation                   ml ethidium bromide [84], followed isolation single colonies <br /> C18aHP independent known mitochondrial activities              YEPD testing growth YPGE. Suppressor strains <br /> Pam16. The growth arrest phenotype pam16-I61N                    generated pam16-I61N mutant background one-step <br /> reversal pam16-I61N sur4D occurs respective rho0            replacement suppressor gene hygromycin resistance <br /> strains lacking functional mitochondria. Pam16 regulate                 cassette selection YEPD plates 200 mg/ml hygro- <br /> production C18aHP Isc1. The quantitative distribution              mycin B (Invitrogen, Carlsbad, CA). Yeast grown Yeast <br /> Isc1 microenvironments cytoplasm uncertain                 Extract Peptone plus adenine medium 2% glucose <br /> although activity occurs endoplasmic reticulum           (YEPD) 2% glycerol ethanol (YPGE) agar plates in <br /> during early growth mitochondria diauxic shift         liquid culture. Synthetic genetic array experiments used yeast <br /> [49,50,77]. Pam16 regulate Isc1 production C18aHPs              nitrogen base (Becton, Dickinson Co. Sparks, MD) added <br /> are typically enriched mitochondrial membrane [50].                  specific amino acid drop selection nourseothricin <br /> However Isc1 preferred substrates, C26 ceramides, located           (cloNAT; WERNER BioAgents, Germany), canavanine, thialy- <br /> primarily plasma membrane [47]. Neutral sphingomyelin-               sine, G418 (Sigma, St. Louis, MO) serial selection <br /> ase 2 (nSMase2) mammalian homolog Isc1 localizes              procedures. In experiments involving peroxisome analysis the <br /> plasma membrane functionally replace yeast gene                 growth medium yeast nitrogen base plus amino acids 2% <br /> [78]. It possible Isc1 degrade ceramides plasma          glucose, glycerol ethanol, 0.25% oleic acid 0.3% <br /> membrane fermentative growth. Since Magmas present                Brij35. YEPD supplemented myriocin (Sigma) the <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       12                              July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                              Pam16 Sphingolipid Metabolism <br />  <br />  <br /> indicated concentrations used serine palmitoyltransfer-          analysis software (BD Biosciences, San Jose, CA). Data shown is <br /> ase inhibition experiments.                                                 representative 3 independent experiments. <br />  <br /> PAM16 Mutations                                                             Electron Microscopy <br />    PAM16 ts mutants error-prone PCR using yeast                   Cultures wt, pam16-I61N, sur4D pam16-I61N sur4D grown <br /> strain W303 [85]. PAM16 cDNA encoding amino acids S41                       overnight YEPD 30uC split prewarmed 32uC <br /> through D142 subcloned pCR-TOPO (Invitrogen)                   YEPD grown 4&#226;&#8364;&#8220;6 h OD 600 nm reached 0.500. <br /> used template mutagenesis. The error-prone PCR                   9.5 ml culture fixed (2.5% glutaraldehyde 0.1 M <br /> products subcloned wt PAM16 pRS315                         sodium cacodylate buffer) postfixed 1% osmium <br /> transformed E. coli XL-1B (Agilent, Santa Clara, CA).                  tetroxide, followed 2% uranyl acetate, dehydrated a <br /> Bacterial colonies scraped plates plasmid DNA             graded series ethanol embedded LX112 resin (LADD <br /> extracted. W303a haploid yeast containing pam16D::HIS3 wt             Research Industries, Burlington, VT). Sections cut a <br /> copy PAM16 pRS316 transformed mutant                     Reichert Ultracut UCT, stained uranyl acetate followed by <br /> plasmid library. After eviction plasmid growth 25uC            lead citrate viewed JEOL 1200EX transmission electron <br /> on YEPD/FOA [86], surviving colonies plated YEPD                 microscope 80 kv Analytic Imaging Facility Albert <br /> YPGE 18uC, 30uC, 37uC identify ts conditional mutants.            Einstein College Medicine (AIF). At 100 cells strain <br /> The plasmid DNA sequenced determine mutation                     examined. Mitochondria peroxisomal number cell <br /> responsible conditional phenotype.                                  mitochondrial cross sectional areas measured every <br />                                                                             cell contained low power fields strain using ImageJ <br /> Synthetic Genetic Array (SGA)                                               1.40 g software (National Institutes Health, USA). Differences <br />                                                                             strains determined comparing total organelle <br />    Query strains derived Y7092 (Table 1) a <br />                                                                             area divided total cell area. Two independent experiments were <br /> nourseothricin resistant gene located adjacent chromosomal <br />                                                                             performed. <br /> pam16-I61N mutant replacing non-essential genes. The pam16- <br /> I61N sur4D double mutant derived pam16-I61N <br /> query replacing SUR4 coding region URA3. The array              Live Cell Fluorescence Microscopy <br /> of deletion strains described Tong Boone [87].                 Yeast strains carrying C-terminal GFP-tagged Pot1 identify <br /> Automated construction double mutants achieved                    peroxisomes grown overnight synthetic media supple- <br /> crossing set viable gene-deletion mutants chosen            mented 2% dextrose. After dilution OD 600 nm of <br /> query strain. Deletion strains arrayed 7 plates density        ,0.3 (to obtain log phase cultures) synthetic media supple- <br /> of 1536 colonies plate duplicates gene-deletion.           mented 2% dextrose, 2% glycerol ethanol, 0.25% <br /> Each SGA screen performed twice using Singer RoToR                    oleic acid 0.3% Brig 35 (Sigma), cultures grown an <br /> HDA robot (Singer Instruments, Somerset, UK) pinning                additional 3 h 32uC. Yeast strains plasmid encoding a <br />                                                                             mitochondrial leader sequence-GFP fusion protein (pYX232- <br /> procedures. Plates photographed 30 h final pinning <br />                                                                             mtgfp) [58] grown overnight YEPD 30uC, split into <br /> step. The growth double mutant colonies obtained using a <br />                                                                             prewarmed 32uC YEPD grown 4 h. Cells harvested <br /> control strain (Y8835) PAM16 mutant strain (pam16-I61N) <br />                                                                             prior live-cell fluorescence microscopy centrifugation at <br /> were compared visual inspection computer-based image <br />                                                                             3000 rpm 5 minutes resuspended respective <br /> analysis (ColonyImager software) [88]. All synthetic genetic <br />                                                                             media. Live-cell fluorescence strains monitored using a <br /> interactions validated random spore analysis [89]. <br />                                                                             fluorescence microscope (Olympus BX61) AIF 606 <br />                                                                             numerical aperture 1.46 objective (PlanApo). Fluorescence or <br /> Proliferation Assay                                                         differential interference contrast (DIC) images captured with <br />   Log phase cultures selected strains counted                    cooled CCD camera (Cooke Sensicam QE, Cooke Corporation, <br /> hemocytometer, diluted 56105 cells/ml, grown YEPD                 Romulus, MI) using IPLab 4.0 software (BD Biosciences). Images <br /> in microtiter plate shaken high speed indicated                 analyzed ImageJ software 1.42q (National Institutes of <br /> temperature 48 h Bioscreen C MBR (Growth Curves                    Health, USA). In Pot1-GFP experiments number of <br /> USA, Piscataway, NJ). Growth curves derived OD                    peroxisomes cell average obtained 100 <br /> 600 nm readings triplicate culture samples measured 30             cells. Two independent experiments performed. <br /> minutes. Standard deviations 0.1% mean <br /> value. Doubling times calculated slope                 Western Blot <br /> natural log OD 600 nm vs. time cell concentrations                  Forty micrograms total cellular protein yeast strain <br /> OD 0.2&#226;&#8364;&#8220;0.6. Two independent experiments performed.                     solubilized Laemmli sample buffer proteins were <br />                                                                             separated SDS-PAGE. The proteins transferred a <br /> Flow Cytometry                                                              nylon membrane (Westran S, Whatman Inc., Sanford ME) a <br />    Cell cycle cell size analyzed fluorescence activated         TRIS/glycine-20% methanol solution (100 volt-hour) 14uC. <br /> cell sorting LSRII analyzer (Becton Dickinson, Franklin                Membranes rinsed, blocked, incubated a <br /> Lakes, NJ) Albert Einstein College Medicine Flow                  monoclonal antibody GFP (Roche Applied Science, Indianap- <br /> Cytometry Core Facility. Log phase cultures yeast strains grown          olis, IN). After washing, membrane immersed 1 h with <br /> in YEPD 30uC switched 34uC times indicated,              peroxidase-labeled secondary antibody, washed times with <br /> washed buffer fixed 70% ethanol/H2O. Fixed cells             buffer containing 50 mM TRIS pH 7.5/150 mM NaCl/0.1% <br /> digested RNAse A proteinase K (Sigma) stained             Tween 20 (TBST), 15 minutes each, developed luminol <br /> Sytox green (Invitrogen). Cells sonicated 5 seconds             substrate (Immobilon, Millipore, Billerica, MA). The resulting <br /> 30% power Sonic Dismembrator Model 500 (GE                           image captured ImageQuant LAS4000 biomolecular <br /> Healthcare, Piscataway, NJ) prior analysis. Quantitation              imager signal intensities specific bands determined using <br /> percent cells cell cycle stages determined using FloJo               ImageQuant TL7.0 software (both GE Healthcare Life <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       13                              July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                              Pam16 Sphingolipid Metabolism <br />  <br />  <br /> Sciences, Piscataway, NJ) expressed band volumes. Blots             experimental used determine association. These are <br /> were stripped reprobed monoclonal antibody yeast             synthetic lethality (light green), affinity capture (navy blue), dosage <br /> Pgk1 (Invitrogen) Pgk1 protein induced glucose [90].               rescue (orange), two-hybrid (aqua) phenotypic enhancement (pea <br />                                                                            green), reconstituted complex (rust), biochemical activity (dark <br /> Sphingolipid Analysis                                                      blue). Two genes, YNL198c YML090W, open reading <br />    Yeast strains grown liquid culture YEPD overnight            frames SGD. YNL198c deleted SSL set as <br /> at 30uC, diluted 16106 cells/ml, incubated 6 h               redundant overlaps opposite DNA strand coding for <br /> 34uC shaking water bath. Cultures washed twice                GCR2. YML090W replaced RPM2, essential gene the <br /> cold water snap frozen liquid nitrogen. Lipids                 opposite DNA strand transcription inhibited not <br /> quantitatively analyzed Lipidomics Analytical <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>           destroyed YML090W deletion (Table 2). <br /> Medical University South Carolina (http://www.musc.edu/                 (TIF) <br /> BCMB/lipidomics/index.shtml) using multicomponent LC/MS <br />                                                                            Figure S3 Strains lacking functional mitochondria have <br /> analysis simultaneous ESI/MS/MS analysis Thermo <br />                                                                            temperature sensitive effects growth as <br /> Finnigan TSQ 7000 triple quadrupole mass spectrometer <br />                                                                            parental strains. Five fold serial dilutions two <br /> operating multiple reaction monitoring (MRM) positive <br />                                                                            independent rho0 strains derived wt, pam16-I61N, sur4D, <br /> ionization mode described [91]. The quantity each <br />                                                                            pam16-I61N sur4D spotted YEPD plates the <br /> sphingolipid normalized organic phosphate determina- <br />                                                                            temperatures indicated. pam16-I61N temperature sensitive growth <br /> tion. The results shown Table S1 Figure 7 mean <br />                                                                            inhibition 34uC 37uC suppression pam16-I61N sur4D <br /> values separate experiments. Sphingolipid content is <br />                                                                            occurred cells lacking functional mitochondria suggesting that <br /> expressed pmole sphingolipid/nmole phosphate. Values were <br />                                                                            effects mediated cell. n = 2. <br /> compared using one-way ANOVA followed Dunnett&#226;&#8364;&#8482;s <br />                                                                            (TIF) <br /> multiple comparison test. <br />                                                                            Figure S4 Effects pam16-I61N sur4D yeast <br /> Phospholipid Analysis                                                      morphology. Transmission electron microscopy yeast strains <br />    Relative phospholipid levels determined according           wt (A), pam16-I61N mutant strain (B), sur4D (C), pam16-I61N <br /> method Vaden et al. [92]. Yeast strains grown YEPD           sur4D (D) shown low power field demonstrate the <br /> 32uC containing 10 mCi 32Pi ml21 OD 600 nm 0.025              cells depicted Figure 5 representative entire <br /> incubated OD 0.800. Cells washed H2O             population. (5,0006 magnification; bar = 1 um). <br /> digested zymolase form spheroplasts. Phospholipids            (TIF) <br /> extracted spheroplasts dissolved 2:1 chloroform/           Figure S5 Effects pam16-I61N sur4D mito- <br /> methanol. Samples applied Whatman LK5 silica gel TLC <br />                                                                            chondrial morphology. Live-cell fluorescence microscopy of <br /> plates (GE Healthcare) separated chloroform/ethanol/ <br />                                                                            wt (A), pam16-I61N (B), sur4D (C) pam16-I61N sur4D (D) <br /> water/triethylamine 30/35/7/35 v/v 2 h, dried, run <br />                                                                            containing mitochondrial leader-GFP display mitochondrial <br /> again solvent 2 h. The signal <br />                                                                            morphology semi-permissive temperature (32uC) (left panels) <br /> intensity band captured phosphoimager screen, <br />                                                                            corresponding differential interference contrast image are <br /> and analyzed using Fujifilm FLA-7000 Phosphoimager and <br />                                                                            shown right panels. The fragmented mitochondria <br /> ImageQuant software (GE Healthcare). Four independent exper- <br />                                                                            morphology observed pam16-I61N restored wt <br /> iments performed. <br />                                                                            elongated reticular mitochondrial morphology pam16-I61N <br />                                                                            sur4D. <br /> Supporting Information                                                     (TIF) <br /> Figure S1 Growth characteristics yeast strains. Five                    Figure S6 Steady state phospholipid levels wt, <br /> fold serial dilutions wt query strain, pam16-I61N,         pam16-I61N, sur4D, pam16-I61N sur4D crd1D. <br /> spotted YEPD YPGE plates. pam16-I61N displayed temper-               Phospholipids wt, pam16-I61N, sur4D, pam16-I61N sur4D <br /> ature sensitive growth inhibition medium containing glucose          crd1D strains analyzed layer chromatography. (A) <br /> glycerol/ethanol 32uC growth arrest 34uC 37uC.               Phosphoimage representative plate showing separation of <br /> (TIF)                                                                      32 <br />                                                                               P orthophosphate labeled organic extracts indicated <br /> Figure S2 Synthetic lethal partners Pam16-I61N.                         strains. { Phospholipids low levels sur4D CDP-DAG <br /> Schematic genetic network interactions SSL partners               LPI/LPC. (B) Quantitation phospholipids strain <br /> Pam-I61N. 46 deletion strains synthetic sick lethal           expressed % total (cpm specific phospholipid/cpm total <br /> paired mutant Pam16. The synthetic lethal partners               phospholipid sample). Values shown mean 6 SEM. Only <br /> assigned Osprey software program [93] 10 17 broad           cardiolipin levels significantly different. Wt vs pam16-I61N, <br /> gene ontology (GO) process categories; including 20 Cell                *p,0.05; wt vs pam16-I61N sur4D, ** p,0.01. Student&#226;&#8364;&#8482;s unpaired <br /> Organization Biogenesis (purple), 14 Metabolism (blue), 2           two-tail t-test. (n = 4). Quantitation total phospholipid (PL) cpm <br /> in Unknown Function (gray), 2 Protein Transport (hot pink),             lane shown lower right panel. Since wt pam16- <br /> and 1 Stress Response (flesh), Cell Cycle (green), DNA         I61N similar amounts monovalent phospholipids, these <br /> Metabolism (beige), Protein Amino Acid Phosphorylation (brown),            phospholipids responsible slow growth phenotype <br /> and Transcription (turquoise). The schematic shows genes          morphological changes occurring pam16-I61N. pam16-I61N <br /> their GO category colored circle lines connecting            lower levels cardiolipin (CL) wt, pam16-I61N sur4D <br /> them designating physical genetic interaction. The genes         reduced levels. Therefore CL levels account <br /> arranged alphabetically concentric circles inner           suppression pam16-I61N fermentative growth defect <br /> circle containing genes interactions          deletion SUR4 involved respiratory <br /> members set outer circle having fewer              growth defect (Figure 2). Abbreviations: PDME, phosphatidyldi- <br /> interactions. Edge lines nodes colored represent        methylethanolamine; PA, phosphatidic acid; PE, phosphatidyleth- <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      14                               July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                                          Pam16 Sphingolipid Metabolism <br />  <br />  <br /> anolamine; PS, phosphatidylserine; PI, phosphatidylinositol; PC,                              The sphingolipid content strains growing logarithmic phase <br /> phosphatidylcholine; LPI/LPC, lysophosphatidylinositol lyso-                               34uC measured mass spectrometry. The quantity of <br /> phosphatidic acid; CDP-DAG, cytidine diphosphate diacylgly-                                   sphingolipid normalized organic phosphate. The <br /> cerol.                                                                                        values shown means 6 SEM (pmole sphingolipid/nmole <br /> (TIF)                                                                                         phosphate). n = 3. Abbreviations: aOH, alpha-hydroxy; Cer, <br />                                                                                               ceramide; dh, dihydro; Sph, sphingosine; P, phosphate. <br /> Figure S7 Elevated C18 alpha-hydroxy-phytoceramide <br />                                                                                               (XLSX) <br /> levels pam16-I61N reduced pam16-I61N double <br /> mutants suppressed ts slow growth phenotype. <br /> The indicated strains grown early log phase YEPD                                Acknowledgments <br /> 30uC shifting temperature 34uC. Cells                                      We thank Frank Macaluso Juan Jimenez Analytic Imaging <br /> harvested 6 h elevated temperature. The normalized                               Facility Albert Einstein College Medicine, Naomi Drucker and <br /> mean level 6 SEM C18 alpha-hydroxy-phytoceramide                                       Pranoy Mohapatra technical assistance, Jon Warner, Duncan <br /> determined mass spectrometry (n = 4). Wt (black), pam16-I61N,                              Wilson Kerri McIntosh reviewing manuscript helpful <br /> (blue), pam16-I61N sur4D (red), pam16-I61N isc1D (yellow), pam16-                             suggestions. We thank Charles Boone providing arrays viable <br />                                                                                               yeast gene-deletion strains SGA parent strain Y7092 these <br /> I61N ipt1D (green), pam16-I61N skn1D (purple), pam16-I61N fenD <br />                                                                                               studies. <br /> (orange). C18 alpha-hydroxy-phytoceramide 7.5 fold higher in <br /> the pam16-I61N strain compared wt experiments. <br /> Except fen1D, weakest suppressor, double mutant                                   Author Contributions <br /> strains C18 alpha-hydroxy-phytoceramide levels similar wt.                             Conceived designed experiments: MKS KT TD MS IMW PTJ. <br /> (TIF)                                                                                         Performed experiments: MKS JPH KT TD RM IMW. Analyzed the <br />                                                                                               data: MKS KT TD MS RM IMW PTJ. Contributed reagents/materials/ <br /> Table S1 Comparison sphingolipid content wild                                           analysis tools: MKS KT TD MS RM IMW PTJ. Wrote paper: MKS <br /> type, pam16-I61N, sur4D pam16-I61N sur4D strains.                                         MS IMW PTJ. <br />  <br /> References <br />  1. Jubinsky PT, Messer A, Bender J, Morris RE, Ciraolo GM, et al. (2001)                     17. Hutu DP, Guiard B, Chacinska A, Becker D, Pfanner N, et al. (2008) <br />     Identification characterization Magmas, novel mitochondria-associated                Mitochondrial protein import motor: differential role Tim44 the <br />     protein involved granulocyte-macrophage colony-stimulating factor signal                   recruitment Pam17 J-complex presequence translocase. Mol Biol <br />     transduction. Exp Hematol 29: 1392&#226;&#8364;&#8220;1402.                                                      Cell 19: 2642&#226;&#8364;&#8220;2649. <br />  2. Peng J, Huang CH, Short MK, Jubinsky PT (2005) Magmas gene structure                  18. Pais JE, Schilke B, Craig EA (2011) Reevaluation role the <br />     evolution. In Silico Biol 5: 251&#226;&#8364;&#8220;263.                                                         Pam18:Pam16 interaction translocation proteins mitochondrial <br />  3. Becker S, Gehrsitz A, Bork P, Buchner S, Buchner E (2001) The black-pearl                     Hsp70-based import motor. Mol Biol Cell 22: 4740&#226;&#8364;&#8220;4749. <br />     gene Drosophila defines novel conserved protein family required           19. Li Y, Dudek J, Guiard B, Pfanner N, Rehling P, et al. (2004) The presequence <br />     larval growth survival. Gene 262: 15&#226;&#8364;&#8220;22.                                                  translocase-associated protein import motor mitochondria. Pam16 functions <br />  4. Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, et al. (1999)                    antagonistic manner Pam18. J Biol Chem 279: 38047&#226;&#8364;&#8220;38054. <br />     Functional characterization S. cerevisiae genome gene deletion              20. Mokranjac D, Berg A, Adam A, Neupert W, Hell K (2007) Association the <br />     parallel analysis. Science 285: 901&#226;&#8364;&#8220;906.                                                      Tim14.Tim16 subcomplex TIM23 translocase crucial function of <br />  5. Gonczy P, Echeverri C, Oegema K, Coulson A, Jones SJ, et al. (2000)                           mitochondrial protein import motor. J Biol Chem 282: 18037&#226;&#8364;&#8220;18045. <br />     Functional genomic analysis cell division C. elegans using RNAi genes         21. Kobayashi SD, Nagiec MM (2003) Ceramide/long-chain base phosphate <br />     chromosome III. Nature 408: 331&#226;&#8364;&#8220;336.                                                          rheostat Saccharomyces cerevisiae: regulation ceramide synthesis Elo3p <br />  6. Jubinsky PT, Short MK, Mutema G, Witte DP (2003) Developmental expression                     Cka2p. Eukaryot Cell 2: 284&#226;&#8364;&#8220;294. <br />     Magmas murine tissues co-expression GM-CSF receptor.               22. Oh CS, Toke DA, Mandala S, Martin CE (1997) ELO2 ELO3, homologues <br />     J Histochem Cytochem 51: 585&#226;&#8364;&#8220;596.                                                             Saccharomyces cerevisiae ELO1 gene, function fatty acid elongation <br />  7. Jubinsky PT, Short MK, Mutema G, Morris RE, Ciraolo GM, et al. (2005)                         required sphingolipid formation. J Biol Chem 272: 17376&#226;&#8364;&#8220;17384. <br />     Magmas expression neoplastic human prostate. J Mol Histol 36: 69&#226;&#8364;&#8220;75.                   23. Cherry JM, Hong EL, Amundsen C, Balakrishnan R, Binkley G, et al. (2012) <br />  8. Desmedt C, Piette F, Loi S, Wang Y, Lallemand F, et al. (2007) Strong time                    Saccharomyces Genome Database: genomics resource budding yeast. <br />     dependence 76-gene prognostic signature node-negative breast cancer                Nucleic Acids Res 40: D700&#226;&#8364;&#8220;705. <br />     patients TRANSBIG multicenter independent validation series. Clin                  24. Wiedemann N, van der Laan M, Hutu DP, Rehling P, Pfanner N (2007) Sorting <br />     Cancer Res 13: 3207&#226;&#8364;&#8220;3214.                                                                     switch mitochondrial presequence translocase involves coupling motor <br />  9. van Vijver MJ, He YD, van&#226;&#8364;&#8482;t Veer LJ, Dai H, Hart AA, et al. (2002) A gene-                 module respiratory chain. J Cell Biol 179: 1115&#226;&#8364;&#8220;1122. <br />     expression signature predictor survival breast cancer. N Engl J Med            25. Barrell D, Dimmer E, Huntley RP, Binns D, O&#226;&#8364;&#8482;Donovan C, et al. (2009) The <br />     347: 1999&#226;&#8364;&#8220;2009.                                                                               GOA database 2009&#226;&#8364;&#8220;an integrated Gene Ontology Annotation resource. <br /> 10. Kawano T, Sugawara S, Hosono M, Tatsuta T, Nitta K (2008) Alteration                       Nucleic Acids Res 37: D396&#226;&#8364;&#8220;403. <br />     gene expression induced Silurus asotus lectin Burkitt&#226;&#8364;&#8482;s lymphoma cells. Biol        26. Hiltunen JK, Schonauer MS, Autio KJ, Mittelmeier TM, Kastaniotis AJ, et al. <br />     Pharm Bull 31: 998&#226;&#8364;&#8220;1002.                                                                      (2009) Mitochondrial fatty acid synthesis type II: just fatty acids. J Biol <br /> 11. Tagliati F, Gentilin E, Buratto M, Mole D, degli Uberti EC, et al. (2010)                     Chem 284: 9011&#226;&#8364;&#8220;9015. <br />     Magmas, gene newly identified overexpressed human mouse ACTH-                 27. Schonauer MS, Kastaniotis AJ, Hiltunen JK, Dieckmann CL (2008) Intersection <br />     secreting pituitary adenomas, protects pituitary cells apoptotic stimuli.                RNA processing type II fatty acid synthesis pathway yeast <br />     Endocrinology 151: 4635&#226;&#8364;&#8220;4642.                                                                 mitochondria. Mol Cell Biol 28: 6646&#226;&#8364;&#8220;6657. <br /> 12. Elsner S, Simian D, Iosefson O, Marom M, Azem A (2009) The mitochondrial                  28. Schonauer MS, Kastaniotis AJ, Kursu VA, Hiltunen JK, Dieckmann CL (2009) <br />     protein translocation motor: Structural conservation human                    Lipoic acid synthesis attachment yeast mitochondria. J Biol Chem 284: <br />     yeast Tim14/Pam18-Tim16/Pam16 co-chaperones. Int J Mol Sci 10: 2041&#226;&#8364;&#8220;                          23234&#226;&#8364;&#8220;23242. <br />     2053.                                                                                     29. Jiang F, Ryan MT, Schlame M, Zhao M, Gu Z, et al. (2000) Absence of <br /> 13. Sinha D, Joshi N, Chittoor B, Samji P, D&#226;&#8364;&#8482;Silva P (2010) Role Magmas                     cardiolipin crd1 null mutant results decreased mitochondrial membrane <br />     protein transport human mitochondria biogenesis. Hum Mol Genet 19:                        potential reduced mitochondrial function. J Biol Chem 275: 22387&#226;&#8364;&#8220;22394. <br />     1248&#226;&#8364;&#8220;1262.                                                                                30. Chen S, Tarsio M, Kane PM, Greenberg ML (2008) Cardiolipin mediates cross- <br /> 14. Frazier AE, Dudek J, Guiard B, Voos W, Li Y, et al. (2004) Pam16                       talk mitochondria vacuole. Mol Biol Cell 19: 5047&#226;&#8364;&#8220;5058. <br />     essential role mitochondrial protein import motor. Nat Struct Mol Biol 11:         31. Tamura Y, Endo T, Iijima M, Sesaki H (2009) Ups1p Ups2p <br />     226&#226;&#8364;&#8220;233.                                                                                      antagonistically regulate cardiolipin metabolism mitochondria. J Cell Biol <br /> 15. Kozany C, Mokranjac D, Sichting M, Neupert W, Hell K (2004) The J domain-                     185: 1029&#226;&#8364;&#8220;1045. <br />     related cochaperone Tim16 constituent mitochondrial TIM23                     32. Pfeiffer K, Gohil V, Stuart RA, Hunte C, Brandt U, et al. (2003) Cardiolipin <br />     preprotein translocase. Nat Struct Mol Biol 11: 234&#226;&#8364;&#8220;241.                                      stabilizes respiratory chain supercomplexes. J Biol Chem 278: 52873&#226;&#8364;&#8220;52880. <br /> 16. D&#226;&#8364;&#8482;Silva PR, Schilke B, Walter W, Craig EA (2005) Role Pam169s degenerate J             33. Carrozza MJ, Li B, Florens L, Suganuma T, Swanson SK, et al. (2005) Histone <br />     domain protein import mitochondrial inner membrane. Proc Natl                   H3 methylation Set2 directs deacetylation coding regions Rpd3S to <br />     Acad Sci U S A 102: 12419&#226;&#8364;&#8220;12424.                                                              suppress spurious intragenic transcription. Cell 123: 581&#226;&#8364;&#8220;592. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                       15                                      July 2012 | Volume 7 | Issue 7 | e39428 <br />                                                                                                                                        Pam16 Sphingolipid Metabolism <br />  <br />  <br /> 34. Keogh MC, Kurdistani SK, Morris SA, Ahn SH, Podolny V, et al. (2005)                           activation inositol phosphosphingolipid phospholipase C, Isc1p, during <br />     Cotranscriptional set2 methylation histone H3 lysine 36 recruits repressive               growth Saccharomyces cerevisiae. J Biol Chem 280: 7170&#226;&#8364;&#8220;7177. <br />     Rpd3 complex. Cell 123: 593&#226;&#8364;&#8220;605.                                                         63.   Gault CR, Obeid LM, Hannun YA (2010) An overview sphingolipid <br /> 35. De Nadal E, Zapater M, Alepuz PM, Sumoy L, Mas G, et al. (2004) The MAPK                       metabolism: synthesis breakdown. Adv Exp Med Biol 688: 1&#226;&#8364;&#8220;23. <br />     Hog1 recruits Rpd3 histone deacetylase activate osmoresponsive genes.                 64.   Kihara A, Mitsutake S, Mizutani Y, Igarashi Y (2007) Metabolism and <br />     Nature 427: 370&#226;&#8364;&#8220;374.                                                                           biological functions phosphorylated sphingolipids, sphingosine 1- <br /> 36. Sertil O, Vemula A, Salmon SL, Morse RH, Lowry CV (2007) Direct role                       phosphate ceramide 1-phosphate. Prog Lipid Res 46: 126&#226;&#8364;&#8220;144. <br />     Rpd3 complex transcriptional induction anaerobic DAN/TIR                   65.   Hannun YA, Obeid LM (2008) Principles bioactive lipid signalling: lessons <br />     genes yeast. Mol Cell Biol 27: 2037&#226;&#8364;&#8220;2047.                                                   sphingolipids. Nat Rev Mol Cell Biol 9: 139&#226;&#8364;&#8220;150. <br /> 37. Chan NC, Lithgow T (2008) The peripheral membrane subunits SAM                    66.   Roy S, Short MK, Stanley ER, Jubinsky PT (2012) Essential Role Drosophila <br />     complex function codependently mitochondrial outer membrane biogenesis.                     black-pearl Mediated Effects Mitochondrial Respiration. FASEB J. <br />     Mol Biol Cell 19: 126&#226;&#8364;&#8220;136.                                                               67.   Perkins GA, Renken CW, van der Klei IJ, Ellisman MH, Neupert W, et al. <br /> 38. Kornmann B, Currie E, Collins SR, Schuldiner M, Nunnari J, et al. (2009) An                    (2001) Electron tomography mitochondria arrest protein import <br />     ER-mitochondria tethering complex revealed synthetic biology screen.                      associated Tom19 depletion. Eur J Cell Biol 80: 139&#226;&#8364;&#8220;150. <br />     Science 325: 477&#226;&#8364;&#8220;481.                                                                    68.   Gaspar ML, Jesch SA, Viswanatha R, Antosh AL, Brown WJ, et al. (2008) A <br /> 39. Young JC, Hoogenraad NJ, Hartl FU (2003) Molecular chaperones Hsp90                        block endoplasmic reticulum-to-Golgi trafficking inhibits phospholipid <br />     Hsp70 deliver preproteins mitochondrial import receptor Tom70. Cell                     synthesis induces neutral lipid accumulation. J Biol Chem 283: 25735&#226;&#8364;&#8220; <br />     112: 41&#226;&#8364;&#8220;50.                                                                                    25751. <br /> 40. Nijtmans LG, Jong L, Artal Sanz M, Coates PJ, Berden JA, et al. (2000)                69.   Schuck S, Prinz WA, Thorn KS, Voss C, Walter P (2009) Membrane expansion <br />     Prohibitins act membrane-bound chaperone stabilization                         alleviates endoplasmic reticulum stress independently unfolded protein <br />     mitochondrial proteins. EMBO J 19: 2444&#226;&#8364;&#8220;2451.                                                  response. J Cell Biol 187: 525&#226;&#8364;&#8220;536. <br /> 41. Tatsuta T, Model K, Langer T (2005) Formation membrane-bound ring                     70.   Bleve G, Di Sansebastiano GP, Grieco F (2011) Over-expression functional <br />     complexes prohibitins mitochondria. Mol Biol Cell 16: 248&#226;&#8364;&#8220;259.                           Saccharomyces cerevisiae GUP1, induces proliferation intracellular mem- <br /> 42. Leonhard K, Stiegler A, Neupert W, Langer T (1999) Chaperone-like activity                  branes containing ER Golgi resident proteins. Biochim Biophys Acta 1808: <br />     AAA domain yeast Yme1 AAA protease. Nature 398: 348&#226;&#8364;&#8220;351.                            733&#226;&#8364;&#8220;744. <br /> 43. Dunn CD, Lee MS, Spencer FA, Jensen RE (2006) A genomewide screen                    71.   Shechtman CF, Henneberry AL, Seimon TA, Tinkelenberg AH, Wilcox LJ, et <br />     petite-negative yeast strains yields new subunit i-AAA protease complex.              al. (2011) Loss subcellular lipid transport ARV1 deficiency disrupts <br />     Mol Biol Cell 17: 213&#226;&#8364;&#8220;226.                                                                     organelle homeostasis activates unfolded protein response. J Biol Chem. <br /> 44. van der Laan M, Chacinska A, Lind M, Perschil I, Sickmann A, et al. (2005)               72.   Hiltunen JK, Mursula AM, Rottensteiner H, Wierenga RK, Kastaniotis AJ, et <br />     Pam17 required architecture translocation activity mitochon-                 al. (2003) The biochemistry peroxisomal beta-oxidation yeast <br />     drial protein import motor. Mol Cell Biol 25: 7449&#226;&#8364;&#8220;7458.                                       Saccharomyces cerevisiae. FEMS Microbiol Rev 27: 35&#226;&#8364;&#8220;64. <br /> 45. Vizeacoumar FJ, Torres-Guzman JC, Bouard D, Aitchison JD, Rachubinski RA                 73.   Guan XL, Souza CM, Pichler H, Dewhurst G, Schaad O, et al. (2009) <br />     (2004) Pex30p, Pex31p, Pex32p form family peroxisomal integral                        Functional interactions sphingolipids sterols biological <br />     membrane proteins regulating peroxisome size number Saccharomyces                       membranes regulating cell physiology. Mol Biol Cell 20: 2083&#226;&#8364;&#8220;2095. <br />     cerevisiae. Mol Biol Cell 15: 665&#226;&#8364;&#8220;677.                                                   74.   Sims KJ, Spassieva SD, Voit EO, Obeid LM (2004) Yeast sphingolipid <br /> 46. Vizeacoumar FJ, Torres-Guzman JC, Tam YY, Aitchison JD, Rachubinski RA                         metabolism: clues connections. Biochem Cell Biol 82: 45&#226;&#8364;&#8220;61. <br />     (2003) YHR150w YDR479c encode peroxisomal integral membrane                          75.   Plesofsky NS, Levery SB, Castle SA, Brambl R (2008) Stress-induced cell death <br />     proteins involved regulation peroxisome number, size, distribution               mediated ceramide synthesis Neurospora crassa. Eukaryot Cell 7: 2147&#226;&#8364;&#8220; <br />     Saccharomyces cerevisiae. J Cell Biol 161: 321&#226;&#8364;&#8220;332.                                         2159. <br /> 47. Dickson RC (2010) Roles sphingolipids Saccharomyces cerevisiae. Adv               76.   Haak D, Gable K, Beeler T, Dunn T (1997) Hydroxylation Saccharomyces <br />     Exp Med Biol 688: 217&#226;&#8364;&#8220;231.                                                                     cerevisiae ceramides requires Sur2p Scs7p. J Biol Chem 272: 29704&#226;&#8364;&#8220;29710. <br /> 48. Sawai H, Okamoto Y, Luberto C, Mao C, Bielawska A, et al. (2000)                         77.   Vaena Avalos S, Okamoto Y, Hannun YA (2004) Activation localization <br />     Identification ISC1 (YER019w) inositol phosphosphingolipid phospholi-                    inositol phosphosphingolipid phospholipase C, Isc1p, mitochondria <br />     pase C Saccharomyces cerevisiae. J Biol Chem 275: 39793&#226;&#8364;&#8220;39798.                              growth Saccharomyces cerevisiae. J Biol Chem 279: 11537&#226;&#8364;&#8220;11545. <br /> 49. Kitagaki H, Cowart LA, Matmati N, Montefusco D, Gandy J, et al. (2009) ISC1-             78.   Wu BX, Clarke CJ, Matmati N, Montefusco D, Bartke N, et al. (2011) <br />     dependent metabolic adaptation reveals indispensable role mitochondria                  Identification novel anionic phospholipid binding domains neutral <br />     induction nuclear genes diauxic shift Saccharomyces                        sphingomyelinase 2 selective binding preference. J Biol Chem 286: <br />     cerevisiae. J Biol Chem 284: 10818&#226;&#8364;&#8220;10830.                                                      22362&#226;&#8364;&#8220;22371. <br /> 50. Kitagaki H, Cowart LA, Matmati N, Vaena Avalos S, Novgorodov SA, et al.               79.   Claypool SM (2009) Cardiolipin, critical determinant mitochondrial carrier <br />     (2007) Isc1 regulates sphingolipid metabolism yeast mitochondria. Biochim                   protein assembly function. Biochim Biophys Acta 1788: 2059&#226;&#8364;&#8220;2068. <br />     Biophys Acta 1768: 2849&#226;&#8364;&#8220;2861.                                                            80.   Joshi AS, Zhou J, Gohil VM, Chen S, Greenberg ML (2009) Cellular functions <br /> 51. Dickson RC, Nagiec EE, Wells GB, Nagiec MM, Lester RL (1997) Synthesis                      cardiolipin yeast. Biochim Biophys Acta 1793: 212&#226;&#8364;&#8220;218. <br />     mannose-(inositol-P)2-ceramide, major sphingolipid Saccharomyces                  81.   Jubinsky PT, Short MK, Ghanem M, Das BC (2011) Design, synthesis, and <br />     cerevisiae, requires IPT1 (YDR072c) gene. J Biol Chem 272: 29620&#226;&#8364;&#8220;29625.                    biological activity novel Magmas inhibitors. Bioorg Med Chem Lett 21: 3479&#226;&#8364;&#8220; <br /> 52. Thevissen K, Idkowiak-Baldys J, Im YJ, Takemoto J, Francois IE, et al. (2005)                  3482. <br />     SKN1, novel plant defensin-sensitivity gene Saccharomyces cerevisiae,            82.   Scatena R (2012) Mitochondria cancer: growing role apoptosis, cancer <br />     implicated sphingolipid biosynthesis. FEBS Lett 579: 1973&#226;&#8364;&#8220;1977.                             cell metabolism dedifferentiation. Adv Exp Med Biol 942: 287&#226;&#8364;&#8220;308. <br /> 53. Thevissen K, Yen WL, Carmona-Gutierrez D, Idkowiak-Baldys J, Aerts AM, et                83.   Ward PS, Thompson CB (2012) Metabolic reprogramming: cancer hallmark <br />     al. (2010) Skn1 Ipt1 negatively regulate autophagy Saccharomyces                        warburg did anticipate. Cancer Cell 21: 297&#226;&#8364;&#8220;308. <br />     cerevisiae. FEMS Microbiol Lett 303: 163&#226;&#8364;&#8220;168.                                            84.   Fox TD, Folley LS, Mulero JJ, McMullin TW, Thorsness PE, et al. (1991) <br /> 54. Zaman S, Lippman SI, Zhao X, Broach JR (2008) How Saccharomyces                                Analysis manipulation yeast mitochondrial genes. Methods Enzymol 194: <br />     responds nutrients. Annu Rev Genet 42: 27&#226;&#8364;&#8220;81.                                               149&#226;&#8364;&#8220;165. <br /> 55. Busti S, Coccetti P, Alberghina L, Vanoni M (2010) Glucose signaling-mediated            85.   Thomas BJ, Rothstein R (1989) Elevated recombination rates transcription- <br />     coordination cell growth cell cycle Saccharomyces cerevisiae. Sensors                ally active DNA. Cell 56: 619&#226;&#8364;&#8220;630. <br />     (Basel) 10: 6195&#226;&#8364;&#8220;6240.                                                                   86.   Boeke JD, Trueheart J, Natsoulis G, Fink GR (1987) 5-Fluoroorotic acid a <br /> 56. Dickson RC (2008) Thematic review series: sphingolipids. New insights                     selective agent yeast molecular genetics. Methods Enzymol 154: 164&#226;&#8364;&#8220;175. <br />     sphingolipid metabolism function budding yeast. J Lipid Res 49: 909&#226;&#8364;&#8220;              87.   Tong AH, Boone C (2007) High-Throughput Construction systematic <br />     921.                                                                                           synthetic lethal screening saccharomyces cerevisae. Methods Microbiology <br /> 57. Delgado A, Casas J, Llebaria A, Abad JL, Fabrias G (2006) Inhibitors                        36: 369&#226;&#8364;&#8220;386. <br />     sphingolipid metabolism enzymes. Biochim Biophys Acta 1758: 1957&#226;&#8364;&#8220;1977.                   88.   Tong AH, Lesage G, Bader GD, Ding H, Xu H, et al. (2004) Global mapping of <br /> 58. Westermann B, Neupert W (2000) Mitochondria-targeted green fluorescent                         yeast genetic interaction network. Science 303: 808&#226;&#8364;&#8220;813. <br />     proteins: convenient tools study organelle biogenesis Saccharomyces        89.   Tong AH, Boone C (2006) Synthetic genetic array analysis Saccharomyces <br />     cerevisiae. Yeast 16: 1421&#226;&#8364;&#8220;1427.                                                               cerevisiae. Methods Mol Biol 313: 171&#226;&#8364;&#8220;192. <br /> 59. Saleem RA, Lo <br /> </body></html>