<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                       REVIEW <br /> Bacteriophage 2:1, 5&#226;&#8364;&#8220;14; January/February/March 2012;           G   2012 Landes Bioscience <br />  <br />  <br />  <br />                                                            Recombineering <br />              A powerful tool modification bacteriophage <br />                                  genomes <br />                                               Laura J. Marinelli,1 Graham F. Hatfull2 Mariana Piuri3,* <br /> 1 <br />  Department Medicine; Dermatology Division; David Geffen School Medicine; University California Los Angeles; Los Angeles, CA USA; 2Department Biological Sciences; <br /> University Pittsburgh; Pittsburgh, PA USA; 3Departamento Qu&#195;&#173;mica Biol&#195;&#179;gica; Facultad Ciencias Exactas y Naturales; Universidad Buenos Aires; Buenos Aires, Argentina <br />  <br />  <br />                                                       Keywords: bacteriophage, recombineering, BRED <br />  <br />  <br />      Recombineering, recently developed technique efficient                            In contrast range tools described for <br />      genetic manipulation bacteria, facilitated phage-                           targeted mutagenesis bacterial chromosomes, fewer methods <br />      derived recombination proteins advantage                              implemented targeted mutagenesis lytically <br />      using DNA substrates short regions homology. This                            growing phages, high frequency events required the <br />      developed E. coli                             absence selectable system. <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br />      adapted use bacteria. It widely used                           Historically, construction mutations bacteriophages was <br />      number different systems variety purposes,                       accomplished general mutagenesis using UV irradiation or <br />      construction chromosomal gene knockouts, deletions,                               chemical compounds generate DNA damage. <br />      insertions, point mutations, vivo cloning,                                A homologous recombination approach used <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />      mutagenesis bacterial artificial chromosomes and <br />                                                                                           make gene deletions recombinant phages,4,5 frequency <br />      phasmids, construction genomic libraries has <br />      reported. However, methods be <br />                                                                                           low screening desired mutant tedious <br />  <br />  <br />  <br />                             Do distribute. <br />      effectively applied genetic modification                                   time consuming. <br />      bacteriophage genomes, prophage lytically                             Recently, new vivo technology introduce genetic <br />      growing states. The ever-growing collection fully sequenced                       changes bacterial genomes developed. This <br />      bacteriophages raises questions answer,                               technique&#226;&#8364;&#8221;named &#226;&#8364;&#339;recombineering&#226;&#8364;?&#226;&#8364;&#8221;refers engineering <br />      including unknown functions vast numbers genes                             recombinant DNA homologous recombination.6-8 Recom- <br />      known homologs unknown function.                                      bineering employs recombination systems encoded bacterio- <br />      Recombineering phage genomes central addressing                             phages enhance frequency homologous recombination, <br />      questions, enabling simple construction mutants,                        allowing construction chromosomal gene knock outs, <br />      determination gene essentiality, elucidation gene                          deletions, insertions, point mutations, vivo cloning, mutagen- <br />      function. In turn, advances understanding phage <br />                                                                                           esis bacterial artificial chromosomes, phasmids genomic <br />      genomics present similar recombineering tools for <br />      dissecting multitude genetically na&#195;&#175;ve bacterial <br />                                                                                           libraries.6-15 <br />      systems.                                                                                The increased availability bacterial genome sequences has <br />                                                                                           facilitated use recombineering, possible to <br />                                                                                           recombineer organism sequence known. <br />                                                                                              The systems described employed recombination functions <br />                                  Introduction                                             encoded bacteriophage lambda Rac prophage, and <br />                                                                                           originally designed modification E. coli. Subsequently, <br /> Bacteriophages extremely abundant, estimated                          successfully employed Gram-negative bacteria, <br /> that number phage particles biosphere approxi-                           Salmonella Shigella,16,17 adapted for <br /> mately 1031.1 These phages source high genetic diversity                         application divergent bacteria, like insect <br /> and replete genetic novelty.2 They played key roles                         endosymbiont Sodalis glossinidius promising subject for <br /> in development bacterial genetics provide effective                        biotechnological exploitation, Pantoea ananatis.18,19 <br /> solutions development systems manipulate genetically                           Double-stranded DNA recombineering E. coli using the <br /> na&#195;&#175;ve bacteria.3 Genetic modification phages facilitate                        lambda Red involves phage-encoded proteins, Exo, <br /> study viruses and, consequence, development                            Beta Gam. Exo exonuclease degrades strand <br /> new tools bacterial manipulation.                                                     DNA double-stranded end generate ssDNA <br />                                                                                           substrate. Beta DNA pairing enzyme anneals the <br />                                                                                           recombineering substrate chromosomal target. Gam inhibits <br />     *Correspondence to: Mariana Piuri; Email: mpiuri@qb.fcen.uba.ar <br />                                                                                           E.coli RecBCD SbcD enzymes, preventing degradation of <br />     Submitted: 10/14/11; Revised: 11/15/11; Accepted: 11/16/11                            double-stranded DNA (dsDNA) substrate.20-23 For optimal <br />     http://dx.doi.org/10.4161/bact.18778                                                  dsDNA recombineering, RecBCD inactivated, either <br />  <br />  <br /> www.landesbioscience.com                                                       Bacteriophage                                                                                  5 <br />  by <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> lambda gam.6,8,24 The original model proposed           system, facilitating construction gene-knockouts point <br /> that degradation Exo, 3&apos; single stranded DNA tail           mutations hard manipulate pathogenic bacterium <br /> exposed Beta bind.8 Homologous recombination                 Mycobacterium tuberculosis, fast-growing related <br /> then occurs, following strand invasion DNA annealing               species like Mycobacterium smegmatis, routinely use a <br /> model.25-28 Recently, new model explain mechanism             model system.46,47 <br /> dsDNA recombination mediated lambda phage proteins                    Mycobacterial strains constructed recombineering contain <br /> been proposed.29,30 In alternative model, lambda exonuclease         extra chromosomal plasmid phage recombina- <br /> entirely degrades strand, leaving strand              tion genes control inducible acetamidase <br /> intact single-stranded DNA. This single-stranded intermediate          promoter.48 Similar shown E. coli, dsDNA <br /> then recombines &#206;&#178; recombinase catalyzed annealing              recombineering mycobacteria requires exonuclease <br /> replication fork.                                                         (gp60) associated recombinase (gp61),39 recom- <br />    The Rac prophage encodes RecE RecT,                 bination using ssDNA substrates requires recombinase.47 <br /> functionally equivalent lambda Exo Beta, respectively.12,31        Targeted gene replacement mutants engineered using linear <br />    Beta belongs large family ssDNA annealing proteins             dsDNA allelic exchange substrates (AESs), containing regions of <br /> (SSAPs),32-34 protein necessary single-stranded      homology upstream downstream target gene flanking <br /> oligonucleotide recombination,7 used creation point            cassette antibiotic resistance, electroporated into <br /> mutations deletions. A strand bias recombination levels            M. smegmatis M. tuberculosis recombineering strains.39 <br /> has observed comparing complementary ssDNAs.                Substrates ssDNA recombineering short oligonucleo- <br /> Higher frequencies obtained using oligonucleotide             tides&#226;&#8364;&#8221;a minimum <span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span> 50 bases recommended&#226;&#8364;&#8221; <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br /> that anneals DNA strand undergoing discontinuous                   encoding desired mutation, technique provides a <br /> synthesis (lagging strand). As result replication process,       simple efficient method constructing point mutations <br /> transient regions ssDNA accessible Beta-mediated            mycobacterial genomes.47 The overall efficiency ssDNA <br /> annealing oligo. The increased recombination efficiency            recombineering substantially higher dsDNA sub- <br /> of &#226;&#8364;&#732;&#226;&#8364;&#732;lagging strand&#226;&#8364;&#8482;&#226;&#8364;&#8482; oligos reflect increased availability   strates chromosomal plasmid targets. However, <br /> of single-stranded regions lagging vs. leading strand              optimal efficiencies obtained using oligonucleo- <br />  <br />  <br />  <br />                        Do distribute. <br /> synthesis.7,12,35                                                         tides target, anneal to, DNA strand undergoing <br />    Regulated expression bacteriophage recombination                discontinuous synthesis (lagging strand template).47 These can <br /> system typically required recombineering. These functions          display efficiencies 10,000-fold higher than <br /> can integrated chromosome contained              oligonucleotides targeting leading strand, far exceeds <br /> extrachromosomal plasmid. A lambda lysogen defective               2&#226;&#8364;&#8220;50 fold strand biases observed l Red system.7,35 It <br /> prophage used express Red proteins strong          clear biases larger mycobacteria <br /> lambda pL promoter.6 The recombineering functions               E. coli, reflect fundamental differences in <br /> been transferred variety different plasmids,36 including          DNA replication systems and/or gp61 interacts <br /> constructs functions control                replication machinery supports necessity of <br /> arabinose promoter pBAD.14 Some plasmids carry temperature-             finding recombineering systems specific bacteria or <br /> sensitive (ts) origin replication, cured       related bacterial group. <br /> cell recombination. The prophage constructs contain                Recombineering systems based mycobacteriophage <br /> ts cI857 allele phage lambda repressor allowing protein            Che9c-encoded proteins provided new approaches myco- <br /> expression controlled temperature. At low temperature            bacterial mutagenesis greatly expanded genetic toolbox <br /> (30&#194;&#176;C), recombineering functions strongly repressed,              available study pathogenic mycobacteria. <br /> expression occurs shift 42&#194;&#176;C 15 min.                     Datta et al.37 identified characterized genes that <br />    The efficiency recombineering E. coli possibility        similar Beta RecT Gram-positive Gram- <br /> of using DNA segments short regions homology (approx.             negative bacteria phages. Nine genes, including Che9c <br /> 50 bp) target sequences led search proteins         gp61, expressed E. coli control lambda pL, <br /> with similarity lambda/Rac encoded proteins               protein activity assayed using oligo recombination <br /> phages bacteria, especially adapting lambda/Rac            system.49 Seven genes adjacent known or <br /> systems use organisms, particularly       putative exonuclease gene. In E. coli system, ssDNA <br /> Gram-positive optimal results.37-39                        binding proteins tested able catalyze oligo-mediated <br />    The complete genome sequences 80 mycobacter-              recombination variable efficiency; Che9c gp61 recomb- <br /> iophage genomes40-45 provided thousands unique             ination levels approximately 1000-fold lower those <br /> genes, making ideal reservoir search novel        observed lambda &#206;&#178;. When present canonical <br /> recombineering functions. Recently, RecE RecT homologs&#226;&#8364;&#8221;               exonucleases, recombinase-exonuclease pairs <br /> the products genes 60 61, respectively&#226;&#8364;&#8221;were identified          tested able carry low level recombination with <br /> mycobacteriophage Che9c, expression substantially            linear dsDNA, coupled lambda Exo; presumably <br /> enhances mycobacterial recombination frequencies. These pro-              physical specific coupling required the <br /> teins utilized develop efficient recombineering           cognate exonuclease single strand annealing proteins. Thus, <br />  <br />  <br /> 6                                                                Bacteriophage                                               Volume 2 Issue 1 <br />  the exonuclease species-specific,         engineering lytically growing phages. This be <br /> recombinase function diverse hosts.                             applicable phages propagated the <br />    After failure observe lambda Red-mediated recombination       laboratory avoids need genetic selection the <br /> Pseudomonas syringae, based hypothesis recom-          associated problems genetic polarity. Two main approaches <br /> binases in, associated with, particular species         recombineering lytically growing phages described, <br /> be likely function close relatives distantly           using phage lambda model system, second using <br /> species,32,50,51 Swingle et al.38used combination bioinformatic   mycobacteriophages model system. <br /> tools identify proteins homologous RecE/RecT Pseudomas           A method engineering phage lambda originally <br /> syringae pv syringae B728a. The so-called recTEPSY genes           described Oppenheim et al.58-60 In general, involves <br /> located 72 kb region horizontally acquired       infecting cells phage, preparing competent cells, introducing <br /> Pseudomas syringae pv syringae B728a. This region contains 86          dsDNA ssDNA substrates electroporation analyzing <br /> annotated open reading frames (ORFs), majority            resulting plaques presence mutation. A scheme <br /> encode hypothetical proteins. Thirteen ORFs locus          general strategy used modify phages infecting E. coli <br /> annotated phage-related proteins, likely region      shown Figure 1. Because method relies physical <br /> corresponds prophage remnant prophage containing        screening identify desired mutants, critical not <br /> the recombination functions. Depending oligo concentra-         recombination frequency high, electro- <br /> tion, 450-fold difference recombination frequency           poration efficiency high (which true E. coli), such <br /> between RecTPSY expression vector control strains          high proportion cells dsDNA ssDNA <br /> observed. Additionally, dsDNA recombination observed               substrate. <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br /> when RecTPSY assisted RecEPSY.                                     Although strains carrying defective lamboid <br />    Recombineering used target bacterial chromo-          prophage expressing &#206;&#178;, exo gam W3110 MG1655 <br /> some extrachromosomal replicating molecules. As corollary      derivatives, prophage moved backgrounds by <br /> this, extended efficient straightforward        P1 transduction,6,61,62 desired strain transformed <br /> modification bacteriophage genomes. Here, discuss       plasmid expressing recombination functions.36 For that <br /> generation E. coli phage mutants using recombineering          reason, modifications, protocol originally deve- <br />  <br />  <br />  <br />                       Do distribute. <br /> lambda Red proteins, construction mycobacter-        loped lambda phage, adapted make mutations in <br /> iophage mutants using BRED (Bacteriophage Recombineering            E. coli phages. Alternatively, intact prophage serve as <br /> Electroporated DNA).52                                                 source recombination functions target for <br />                                                                        recombineering. In case, recombination <br />                 Recombineering Prophages                               functions present, prophage inducible in <br />                                                                        order recover phage particles recombineering. <br /> One relatively straightforward approach engineering phage              In cases, cells grown exponential phase. If lytically <br /> genomes use dsDNA recombineering modify integrated         growing phage going modified, cells infected a <br /> prophage, effectively treating chromosomal locus.      multiplicity infection (MOI) 1&#226;&#8364;&#8220;3 left standing allow <br /> There are, however, limitations disadvantages. First,      phage adsorb (adsorption conditions phage be <br /> this approach restricted temperate phages. Moreover,       taken consideration step). This necessary the <br /> useful inducible phages, lytically growing   phage present prophage cells. <br /> particles constructed mutant generated. While       In step, expression Beta, Exo Gam (when is <br /> often concern well-characterized systems phage       present) induced short period time. Conditions for <br /> lambda, generally so. Second, approach typically        induction determined basis selected <br /> requires use selectable marker recover desired         strain (temperature shift, addition inducer, etc.). The cells <br /> mutant, introduction marker drug resistance         gently washed, appropriate DNA substrate is <br /> gene introduce genetic polarity, increase size     introduced electroporation. When modifying intact pro- <br /> the phage genome packaging limit. These effects        phage, induction time used prophage <br /> minimized using second event&#226;&#8364;&#8221;such using site-specific       going induced recombinant lysate prepared. The <br /> recombination&#226;&#8364;&#8221;to remove selectable marker,           progeny phage screened mutation. Alter- <br /> requires multiple sequential manipulations. Strategies modify       natively, prophage targeted using selectable marker, <br /> an integrated prophage described below.             drug resistance cassette, recombinant bacterial <br />                                                                        colonies identified using selection (see below). <br />              Recombineering E. coli Phages                                A single-stranded oligo dsDNA PCR product used <br />                                                                        substrate bacteriophage recombineering (Fig. 2). When a <br /> E. coli phage mutants played key roles study            ssDNA substrate used (for <span id='am-237' about='xsp:base' typeof='owl:Thing'>base</span> substitution small deletions <br /> phage biology,53-57 construction and/or screening       insertions), typically 70 bases length are <br /> desired mutation tedious time consuming. Recom-             designed alteration introduced contained in <br /> bineering simplified mutant phage construction, increas-        center oligo. An 80-nucleotide oligo successfully <br /> ing frequency recombination events cell, enabling        used generate 326 bp deletion cII gene lambda.60 In <br />  <br />  <br /> www.landesbioscience.com                                       Bacteriophage                                                                7 <br />  &#194;&#169; 2012 Landes Bioscience. <br />                            Do distribute. <br />     Figure 1. E. coli phage recombineering. Schematic representation strategy used construct screen phage mutants recombineering E. <br />     coli. If phage modified present prophage genome, cells infected phage; subsequently, recombination <br />     functions expressed, cells electrocompetent. The substrate introduced infected cells electroporation (dsDNA, ssDNA <br />     containing deletion D, <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>point mutation</span> *), cultures recovered allow phage assembly. The recombinant lysate screened the <br />     desired mutant. Plating properties recombinant lysate compared control lysate obtained using protocol without <br />     addition substrate. The double-layer bacterial lawn technique used strain supports amplification mutant phage (that <br />     replicate isogenic strain) available. Alternatively, desired mutant screened plaque hybridization using probe only <br />     hybridize mutant wt phage. After mutant phage identified purified, stock prepared, presence the <br />     desired mutation corroborated PCR DNA sequencing. <br />  <br />  <br />  <br /> contrast seen oligo-mediated recombina-                      cleave plasmid purified Dam+ host) be <br /> tion bacteria, complementary oligos equally efficient                 used reduce residual plasmid. <br /> in promoting recombination, reflecting rolling circle                     Oppenheim et al.60 used precisely replace the <br /> replication employed phage.                                                rexA rexB genes lambda gene conferring ampicillin <br />    When dsDNA substrate used, PCR product 50 base                      resistance. The substrate target lambda chromosome was <br /> pairs homology (at end) target phage chro-                     generated described Figure 2. In case, phage lysate was <br /> mosome created following strategy outlined Figure 2.                     grown electroporation mix used form lysogens in <br />    It noted drug resistance markers                     na&#195;&#175;ve E. coli. AmpR lysogens selected, replacement of <br /> used selection going lysogen,                  rexAB genes confirmed PCR. <br /> markers employed lytically replicating phages. Care                      If, alternative, recombinant lysogens complete <br /> should taken plasmid utilized template                    prophage (carrying example, drug resistance marker) are <br /> PCR reaction used generate substrate, residual                  going selected, electroporation mixture spread <br /> plasmid reaction products transform cells effici-                  filter placed plate rich nonselective <br /> ently, generating undesirable background. Digestion PCR                     medium. After incubation hours filter transferred <br /> reaction DpnI (this cut PCR product                     plate containing selective drug. After overnight growth, <br />  <br />  <br />  <br /> 8                                                                         Bacteriophage                                                        Volume 2 Issue 1 <br />   Figure 2. Type substrates used phage recombineering. (A) ssDNA oligos (70 nt) containing small deletion (D) point mutation (*) the <br />  center, anneal lagging strand template DNA replication. (B) When mutation going constructed prophage, <br />  <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br />  target gene replaced antibiotic resistance marker (abR). In case, primers amplify resistance marker contain 50 <br />  bases homology target gene used generate dsDNA substrate. (C) Schematic representation strategy used to <br />  increase homology target PCR. A 200 bp substrate obtained 100-mer oligonucleotide 50 bases homology side <br />  deletion 72-mers. In (A&#226;&#8364;&#8220;C) each, blue green rectangles indicate homology regions flanking target gene. <br />  <br />  <br /> recombinants cells rise colonies, prophages            gel electrophoresis. Finally, lysate corroborated <br /> excised recover mutant phage particles.                                     recombinant phage prepared used studies. <br />  <br />  <br />  <br />                         Do distribute. <br />    When lytically propagated phage genome target, <br /> the recombinant lysate plated direct selection                   Construction Mycobacteriophage Mutants <br /> plaques screening. Preliminarily, dilutions                                 Recombineering <br /> control recombinant lysates plated lawns a <br /> bacterial indicator compare plating properties lysates              Mycobacteriophages&#226;&#8364;&#8221;viruses infect mycobacteria&#226;&#8364;&#8221;have <br /> (Fig. 1). If available, selective indicator strain used.           instrumental overcoming challenges per- <br /> For example, amber mutation introduced essential                forming genetic studies human pathogen, M. tuberculosis. <br /> gene, double-layer bacterial lawn technique63 used               These formed basis development shuttle phasmids,65 <br /> identify phages carry mutation. In case, double-               used transposon mutagenesis,66 reporter gene <br /> layer containing different indicator strains utilized plate          delivery,67-69 specialized transduction,70 the <br /> the recombinant lysate. The lower layer contains restrictive               construction integration-proficient vectors, enabling stable <br /> host wild-type (wt) phage multiply,                 introduction foreign genes bacterium.42,43,71-73 How- <br /> top layer contains suppressor strain supports growth                 ever, mycobacteriophages instrumental the <br /> both mutant wt phage. Amber mutants lyse                  development techniques mycobacterial mutant construc- <br /> suppressor strain form &#226;&#8364;&#339;cloudy&#226;&#8364;? plaques                   tion, genetic tools enabling mutagenesis phages themselves <br /> non-suppresor strain grow confluence (Fig. 1).                          lacking. <br />    If phage mutants selected distinguished visually,               In general, constructing mutant derivatives bacteriophages <br /> plaque hybridization assay used detect mutant phage              challenging manipulating bacterial host <br /> derivatives (Fig. 1). In order identify mutant,              chromosome, antibiotic selection useful in <br /> 15 bp DNA sequence present recombinant phage,                   lytically propagated viruses. To circumvent this, mycobacterial <br /> but original phage, needed use target            recombineering employed manipulation of <br /> probe.5 To corroborate presence desired mutation,                prophages, integrated host chromosome, a <br /> isolated plaque resuspended buffer used directly              manner similar described manipulation lambda <br /> a template PCR reaction, using primers flank                 prophages above. Unfortunately, relatively mycobacterio- <br /> deletion/insertion.15,52 An amplicon differential size expected          phages form stable lysogens, significantly limiting use this <br /> in mutant compared parent phage. MAMA-PCR                      strategy. Until recently, powerful method manip- <br /> (mismatch amplification mutation assay PCR) used                ulating lytic mycobacteriophages use shuttle <br /> screen point mutations.64 If point mutation alters                phasmids74&#226;&#8364;&#8221;phage-plasmid hybrids packaged into <br /> restriction site generated, base change identified              phage l particles propagated E. coli. In example, this <br /> after restriction amplified PCR product visualization               strategy used make defined mutations tape measure <br />  <br />  <br />  <br /> www.landesbioscience.com                                              Bacteriophage                                                                          9 <br />  gene shuttle phasmid derivative mycobacteriophage TM4 <br /> by performing mutagenesis E. coli (by l Red recombineer- <br /> ing) recovering mutant phage M. smegmatis.15 In spite <br /> of this, obvious advantages technique, use of <br /> mycobacteriophage phasmids remains limited. This due, in <br /> part, multiple steps required construct phasmids, <br /> but primarily function relatively small packaging <br /> capacity (less ~50 kb) l capsids, compared the <br /> large average size mycobacteriophage genomes (~70 kb).40,41 <br /> Thus, mycobacteriophage genomes large shuttle <br /> phasmid construction, approach broadly applicable <br /> for functional genomic studies phage population. <br />    In contrast, mycobacterial recombineering systems offer a <br /> new, potentially powerful approach constructing mutants of <br /> any lytically growing mycobacteriophage, using method termed <br /> Bacteriophage Recombineering Electroporated DNA (BRED),52 <br /> the general scheme illustrated Figure 3. A critical <br /> aspect technique based observation that, in <br /> mycobacteria, recombineering frequencies appear primarily <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br /> limited relatively poor DNA uptake efficiencies75; is, <br /> even electrocompetent mycobacteria, 1 1000 <br /> viable cells productively DNA. This presents a <br /> challenge constructing mutations no <br /> selection, chromosomal point mutations known <br /> <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, 99.9% colonies recovered arise cells that <br />  <br />  <br />  <br />                       Do distribute. <br /> were unable recombineering substrate.47 The issue <br /> is circumvented ssDNA recombineering employing a <br /> counter-selection strategy, selectable plasmid (or a <br /> second oligonucleotide incorporates selectable point <br /> mutation) co-transformed ssDNA substrate.6,7 This <br /> effectively selects non-competent cells population, <br /> and transformants screened mutation <br /> using MAMA-PCR,10,64 selectively amplifies mutant <br /> allele. In way, non-selectable mutations easily <br /> recovered high efficiency ssDNA recombineering. <br />    The technique described previous section recom- <br /> bineering E. coli phage, l, involves infecting with <br /> the phage targeted, inducing recombineering func- <br /> tions, electroporating DNA substrate;60 however, <br /> the inefficiency DNA uptake mycobacteria precludes this <br /> approach. Therefore, BRED, adopted alternative <br /> approach derived co-selection methods described <br /> above.52 In procedure, M. smegmatis strain carrying a <br /> recombineering plasmid induced acetamide, electro- <br /> competent cells prepared. This electroporation performed <br />                                                                          Figure 3. Mycobacteriophage recombineering. Schematic representation <br /> with genomic DNA phage manipulated             strategy used construct phage mutants Mycobacterium <br /> a PCR-generated synthetic DNA substrate targets             smegmatis recombineering. M. smegmatis cells inducibly expressing <br /> phage (Fig. 2). After brief recovery period, reactions         recombination functions co-electroporated phage DNA and <br /> mixed additional plating cells M. smegmatis plated        recombineering substrate; mixed M. smegmatis <br /> in infectious center assay (i.e., prior lysis, plaques     plating cells plated obtain individual plaques. Primary plaques are <br />                                                                          screened PCR flanking primers detect containing <br /> are generated infected cells, viral particles). In    mutant phage present frequency combination the <br /> way, cells taken phage DNA rise           wt phage. Once identified way, positive plaque containing the <br /> plaques bacterial lawn, effectively selecting         mutant allele diluted plated. If non-essential gene was <br /> those non-transformable; average, 100 plaques          targeted, mutant phage easily identified PCR. Essentiality <br /> are obtained 50&#226;&#8364;&#8220;100 ng phage DNA                      target gene inferred mutant allele detected a <br />                                                                          lysate pooled secondary plaques. <br /> experiment. All primary plaques contain wild-type <br />  <br />  <br /> 10                                                              Bacteriophage                                                       Volume 2 Issue 1 <br />  phage DNA, subset (5&#226;&#8364;&#8220;50%) contain mutant mismatch, thus, stringent conditions, the <br /> allele,52 readily identified PCR screening. wild-type allele amplified. <br /> Provided mutation does ablate region essential       One application BRED DADA-PCR detec- <br /> phage growth, purified mutant phage derivatives isolated tion strategy quite useful deletion genes <br /> by plating serial dilutions mixed plaques, followed essential&#226;&#8364;&#8221;or suspected essential&#226;&#8364;&#8221;for phage growth. <br /> purification PCR analysis resulting secondary plaques. We able delete number essential genes in <br /> The frequency pure mutants recovered mycobacteriophage Giles (such lysin A)52 (Hatfull lab, <br /> vary significantly depending factors unpublished data) phages constructing comple- <br /> mutant allele present primary plaque phenotype mentation strain M. smegmatis mutant isolation, which <br /> of mutant (e.g., recovered lower frequency expresses phage gene deleted. In cases, often <br /> it reduced burst size compared wild-type). Often, necessary use DADA-PCR detect primary plaques contain- <br /> analysis 12&#226;&#8364;&#8220;18 secondary plaques sufficient identify ing mutant allele. These recovered wild-type <br /> homogenously mutant strain, cases, bacteria, presumably presence wild-type helper <br /> necessary screen secondary plaques pools 5&#226;&#8364;&#8220;10 order phage primary plaque. Once plaque recovered, <br /> to recover mutant. In cases, pool containing essentiality inferred mutant allele detected <br /> the mutant identified, plating necessary completely lysates prepared re-plating mixed plaques on <br /> purify mutant. This pooling strategy essentially enriches wild-type cells; is, individual mutant particles not <br /> a mutation present low levels rise plaques secondary plating. The mixed primary <br /> initial primary plaque.                                                 plaque serially diluted plated comple- <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br />    To date, common application BRED mentation strain. Secondary plaques picked patched in <br /> the construction unmarked gene deletions; however, duplicate wild-type complementing bacteria, mutant <br /> technique used efficiently engineer in-frame inter- phage identified able infect and <br /> nal deletions, small insertions, point mutations gene replace- kill complementing M. smegmatis. This strategy provides a <br /> ments phage genome, require selection powerful effective way screen gene essentially, allowing <br /> for mutant recovery52 (Hatfull Lab, unpublished data). When determine unique genes the <br />  <br />  <br />  <br />                        Do distribute. <br /> constructing deletion insertion mutant, 200 bp dsDNA mycobacteriophages required lytic growth. <br /> substrate, having 100 bp homology flanking region             Another type genetic manipulation BRED has <br /> deleted inserted recommended, higher proportion proved successful engineering point mutations, for <br /> primary plaques contain desired mutant allele using this, ssDNA synthetic substrates employed.52 Although an <br /> dsDNA vs. ssDNA substrates, larger substrates (200 bp) vs. extensive length dependence analysis performed, <br /> shorter ones (100 bp). Insertions typically screened <br /> by PCR using primer anneals the <br /> inserted region anneals down- <br /> stream region phage chromosome. For deletions, <br /> primers flank deleted region used; when <br /> the mutant allele present primary plaque, these <br /> produce smaller PCR product often <br /> easily visible agarose gel. In cases&#226;&#8364;&#8221;usually <br /> when mutants somewhat defective&#226;&#8364;&#8221;mutant <br /> alleles present low frequencies primary <br /> plaques detected flanking primers. <br /> For situations, adapted MAMA-PCR <br /> strategy described developed technique <br /> termed Deletion Amplification Detection Assay <br /> (DADA) PCR.52 This employs DADA-PCR primer <br /> that anneals new junction created the <br /> deletion 3&apos; end anneals to <br /> the flanking DNA, up- downstream this.          Figure 4. Deletion Amplification Assay (DADA) PCR. DADA-PCR used specifically <br /> The DADA-PCR primer designed             detect presence gene deletion allele present low frequency is <br /> mutant allele, primer anneal perfectly       difficult detect mixed population mutant WT phage. The <br />                                                              DADA-PCR primer designed 3&#226;&#8364;&#8482; end anneals new junction <br /> the deletion site (this site created joining   created joining DNA ends flanking deleted region. When this <br /> of DNA ends flank deleted region),          primer anneals WT allele, creates mismatch (shown &#226;&#8364;&#732;x&#226;&#8364;&#8482; figure) <br /> but wild-type phage, terminal 3&apos;     high fidelity DNA polymerase synthesize stringent <br /> nucleotides (depending primer design)        PCR conditions. However, DADA-PCR primer complementary deletion <br /> base pair DNA template (Fig. 4). A high fide-       allele, appropriate down- (or up-) stream primer, generate a <br />                                                              product mutant allele present. <br /> lity DNA polymerase difficulty synthesizing <br />  <br />  <br /> www.landesbioscience.com                                          Bacteriophage                                                                 11 <br />  we oligonucleotides 71 nt length                          sequenced annotated bacteriophage genomes available <br /> sufficient efficiently recover mutation (Marinelli, unpub-                            NCBI database, revealing genetically diverse popula- <br /> lished data). Based observations mycobacterial                                   tion architecturally mosaic genomes. This immense reservoir <br /> ssDNA recombineering47; however, possible shorter                                 genetic information created numerous opportunities for <br /> substrates work well. Importantly, nature                           functional genomic studies aimed uncovering novel functions <br /> phage DNA replication, constructing point mutations, oligo-                             phage population. <br /> nucleotides targeting strands used transformed                                 Just Mycobacteriophages alone, enormous number <br /> together phage genomic DNA.52 The primary                                       predicted open reading frames unknown function have <br /> secondary plaques screened mutant allele using MAMA-                             found, make 3,000 phamilies unique <br /> PCR purified using techniques outlined above.                                   genes having detectable sequence similarity known <br />    Finally, shown BRED used con-                                 proteins.41-45,73,80 An increasing hunt phage <br /> struction gene replacement mutants phage genome.52                             proteins promote recombination functions been <br /> As mentioned above, antibiotic resistance cassettes useful                           particularly driven number bacteriophage genome <br /> in lytic phages; however, numerous applications&#226;&#8364;&#8221;such                                sequences available.50 These functions potentially used <br /> the construction fluorescent reporter phages67-69&#226;&#8364;&#8221;could                                develop species-specific recombineering systems diverse <br /> envisioned desirable insert foreign                               suite organisms. <br /> gene phage genome. The fairly strict packaging size con-                                 Bacteriophage recombineering shown highly <br /> straints exist dsDNA bacteriophages necessitate                               effective approach construction gene deletions, small <br /> inserting gene egfp (enhanced green fluorescent protein)                           insertions, gene replacements, point mutations lytically <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br /> in place similarly sized non-essential phage gene. For                             replicating phages.52,60 This technique will, time, <br /> type BRED, use dsDNA substrates, synthesized PCR,                                   allow perform broad functional genomic studies bac- <br /> that contain cassette choice flanked 100 bp homolog-                               teriophages, potentially revealing new exciting features of <br /> ous end target gene replaced. Such                            diverse phage population. <br /> mutations easily detected using primer pair                                  Here described modification E. coli phages <br /> anneals newly inserted gene anneals                              mycobacteriophages. However, slight modifications to <br />  <br />  <br />  <br />                               Do distribute. <br /> a region up- downstream phage genome.                                          protocols, providing appropriate recombination <br />    To date, BRED successfully used different                                systems found, methods potentially adapted for <br /> mycobacteriophages52,76-78 (Hatfull Lab, unpublished data),                           use phages. Any new bacteriophage isolated can <br /> is likely utility approaches mycobacter-                          possible source new recombination proteins. For this <br /> iophage mutagenesis realized. For example,                                 purpose, genetic manipulation phages very <br /> coupling efficient point mutagenesis mycobacterial non-                              useful recombineering perfect ally to <br /> sense suppressor strains79 provide relatively simple method                          accomplish objective. Thus, bacteriophage recombineering <br /> for making conditional lethal mutants mycobacteriophage                                   facilitate study orphan genes provide <br /> genomes. It possible utilize technique                         insights phage biology, help uncover <br /> manipulation phages infect species mycobacteria                             new valuable instruments manipulation genetically <br /> or infecting closely related bacteria.                                           naive bacteria. <br />  <br />                                   Perspectives                                                                               Acknowledgments <br />                                                                                              This work supported US National Institutes of <br /> Coordinated efforts spanning decade yielded vast                             Health grant 1R03TW008926-01, ANPCyT-PICT2009-0095, <br /> collection fully sequenced bacteriophages. About 600 fully                                CONICET-PIP2011-0222. <br />  <br />                       References                               5.   Tanji Y, Furukawa C, Na S-H, Hijikata T, Miyanaga       8.  Court DL, Sawitzke JA, Thomason LC. Genetic <br />                                                                     K, Unno H. Escherichia coli detection GFP-labeled        engineering using homologous recombination. Annu <br /> 1.   Hendrix RW, Lawrence JG, Hatfull GF, Casjens S. <br />                                                                     lysozyme-inactivated T4 bacteriophage. J Biotechnol         Rev Genet 2002; 36:361-88; PMID:12429697; http:// <br />      The origins ongoing evolution viruses. Trends <br />                                                                     2004; 114:11-20; PMID:15464594; http://dx.doi.org/          dx.doi.org/10.1146/annurev.genet.36.061102.093104 <br />      Microbiol 2000; 8:504-8; PMID:11121760; http://dx. <br />                                                                     10.1016/j.jbiotec.2004.05.011                           9. Copeland NG, Jenkins NA, Court DL. Recombineer- <br />      doi.org/10.1016/S0966-842X(00)01863-1 <br />                                                                6.   Yu D, Ellis HM, Lee EC, Jenkins NA, Copeland NG,            ing: powerful new tool mouse functional <br /> 2.   Br&#195;&#188;ssow H, Hendrix RW. Phage Genomics: Small Is <br />                                                                     Court DL. An efficient recombination chro-       genomics. Nat Rev Genet 2001; 2:769-79; PMID: <br />      Beautiful. Cell 2002; 108:13-6; PMID:11792317 <br />                                                                     mosome engineering Escherichia coli. Proc Natl           11584293; http://dx.doi.org/10.1038/35093556 <br /> 3.   Jacobs WR Jr. Mycobacterium tuberculosis: A Once <br />                                                                     Acad Sci USA 2000; 97:5978-83; PMID:10811905;           10. Swaminathan S, Ellis HM, Waters LS, Yu D, Lee EC, <br />      Genetically Intractable Organism. In: Hatfull GF, <br />                                                                     http://dx.doi.org/10.1073/pnas.100127597                    Court DL, et al. Rapid engineering bacterial artificial <br />      Jacobs Jr. WR, eds. Molecular Genetics Myco- <br />                                                                7.   Ellis HM, Yu D, DiTizio T, Court DL. High efficiency        chromosomes using oligonucleotides. Genesis 2001; 29: <br />      bacteria. Washington, DC: ASM Press, 2000:1-16. <br />                                                                     mutagenesis, repair, engineering chromosomal         14-21; PMID:11135458; http://dx.doi.org/10.1002/ <br /> 4.   Sarkis GJ, Jacobs WR, Hatfulll GF. L5 luciferase <br />                                                                     DNA using single-stranded oligonucleotides. Proc Natl       1526-968X(200101)29:1,14::AID-GENE1001.3.0. <br />      reporter mycobacteriophages: sensitive tool the <br />                                                                     Acad Sci USA 2001; 98:6742-6; PMID:11381128;                CO;2-X <br />      detection assay live mycobacteria. Mol Microbiol <br />                                                                     http://dx.doi.org/10.1073/pnas.121164898 <br />      1995; 15:1055-67; PMID:7623662; http://dx.doi.org/ <br />      10.1111/j.1365-2958.1995.tb02281.x <br />  <br />  <br />  <br />  <br /> 12                                                                                 Bacteriophage                                                                   Volume 2 Issue 1 <br />  11. Sarov M, Schneider S, Pozniakovski A, Roguev A,           27. P&#195;&#162;ques F, Haber JE. Multiple pathways recombina-        42. Morris P, Marinelli LJ, Jacobs-Sera D, Hendrix RW, <br />     Ernst S, Zhang Y, et al. A recombineering pipeline        tion induced double-strand breaks Saccharomyces          Hatfull GF. Genomic characterization mycobacter- <br />     functional genomics applied Caenorhabditis elegans.        cerevisiae. Microbiol Mol Biol Rev 1999; 63:349-404;           iophage Giles: evidence phage acquisition host <br />     Nat Methods 2006; 3:839-44; PMID:16990816;                    PMID:10357855                                                  DNA illegitimate recombination. J Bacteriol 2008; <br />     http://dx.doi.org/10.1038/nmeth933                        28. Smith GR. Homologous recombination near far                190:2172-82; PMID:18178732; http://dx.doi.org/10. <br /> 12. Zhang Y, Buchholz F, Muyrers JP, Stewart AF. A new            DNA breaks: alternative roles contrasting             1128/JB.01657-07 <br />     logic DNA engineering using recombination              views. Annu Rev Genet 2001; 35:243-74; PMID:               43. Pham TT, Jacobs-Sera D, Pedulla ML, Hendrix RW, <br />     Escherichia coli. Nat Genet 1998; 20:123-8; PMID:             11700284; http://dx.doi.org/10.1146/annurev.genet.35.          Hatfull GF. Comparative genomic analysis myco- <br />     9771703; http://dx.doi.org/10.1038/2417                       102401.090509                                                  bacteriophage Tweety: evolutionary insights con- <br /> 13. Muyrers JP, Zhang Y, Testa G, Stewart AF. Rapid           29. Maresca M, Erler A, Fu J, Friedrich A, Zhang Y,                struction compatible site-specific integration vectors <br />     modification bacterial artificial chromosomes ET-       Stewart AF. Single-stranded heteroduplex intermediates         mycobacteria. Microbiology 2007; 153:2711-23; <br />     recombination. Nucleic Acids Res 1999; 27:1555-7;             lambda Red homologous recombination. BMC Mol                PMID:17660435;          http://dx.doi.org/10.1099/mic.0. <br />     PMID:10037821; http://dx.doi.org/10.1093/nar/27.6.            Biol 2010; 11:54; PMID:20670401; http://dx.doi.org/            2007/008904-0 <br />     1555                                                          10.1186/1471-2199-11-54                                    44. Hatfull GF, Pedulla ML, Jacobs-Sera D, Cichon PM, <br /> 14. Datsenko KA, Wanner BL. One-step inactivation          30. Mosberg JA, Lajoie MJ, Church GM. Lambda red                   Foley A, Ford ME, et al. Exploring mycobacter- <br />     chromosomal genes Escherichia coli K-12 using              recombineering Escherichia coli occurs            iophage metaproteome: phage genomics edu- <br />     PCR products. Proc Natl Acad Sci USA 2000; 97:                fully single-stranded intermediate. Genetics 2010; 186:        cational platform. PLoS Genet 2006; 2:e92; <br />     6640-5; PMID:10829079; http://dx.doi.org/10.1073/             791-9; PMID:20813883; http://dx.doi.org/10.1534/               PMID:16789831; http://dx.doi.org/10.1371/journal. <br />     pnas.120163297                                                genetics.110.120782                                            pgen.0020092 <br /> 15. Piuri M, Hatfull GF. A peptidoglycan hydrolase motif      31. Gottesman MM, Gottesman ME, Gottesman S, Gellert           45. Pedulla ML, Ford ME, Houtz JM, Karthikeyan T, <br />     mycobacteriophage TM4 tape measure                 M. Characterization bacteriophage lambda reverse            Wadsworth C, Lewis JA, et al. Origins highly mosaic <br />     protein promotes efficient infection stationary            Escherichia coli phage carrying unique set             mycobacteriophage genomes. Cell 2003; 113:171-82; <br />     phase cells. Mol Microbiol 2006; 62:1569-85; PMID:            host-derived recombination functions. J Mol Biol            PMID:12705866; http://dx.doi.org/10.1016/S0092- <br />     17083467;        http://dx.doi.org/10.1111/j.1365-2958.       1974; 88:471-87; PMID:4616090; http://dx.doi.org/              8674(03)00233-2 <br />     2006.05473.x                                                  10.1016/0022-2836(74)90496-3                               46. van Kessel JC, Hatfull GF. Mycobacterial recombineer- <br /> 16. Ranallo RT, Barnoy S, Thakkar S, Urick T, Venkatesan      32. Iyer LM, Koonin EV, Aravind L. Classification              ing. Methods Mol Biol 2008; 435:203-15; PMID: <br />  <br />  <br />  <br />  <br /> &#194;&#169; 2012 Landes Bioscience. <br />     MM. Developing live Shigella vaccines using l Red             evolutionary history single-strand annealing            18370078; http://dx.doi.org/10.1007/978-1-59745- <br />     recombineering. FEMS Immunol Med Microbiol                    proteins, RecT, Redbeta, ERF RAD52. BMC                    232-8_15 <br />     2006; 47:462-9; PMID:16872384; http://dx.doi.org/             Genomics 2002; 3:8; PMID:11914131; http://dx.doi.          47. van Kessel JC, Hatfull GF. Efficient point mutagenesis <br />     10.1111/j.1574-695X.2006.00118.x                              org/10.1186/1471-2164-3-8                                      mycobacteria using single-stranded DNA recombi- <br /> 17. Swingle B, Markel E, Costantino N, Bubunenko MG,          33. Hall SD, Kolodner RD. Homologous pairing                   neering: characterization antimycobacterial drug <br />     Cartinhour S, Court DL. Oligonucleotide recombina-            strand exchange promoted Escherichia coli RecT          targets. Mol Microbiol 2008; 67:1094-107; PMID: <br />     tion Gram-negative bacteria. Mol Microbiol 2010;           protein. Proc Natl Acad Sci USA 1994; 91:3205-9;               18221264; http://dx.doi.org/10.1111/j.1365-2958.2008. <br />     75:138-48; PMID:19943907; http://dx.doi.org/10.               PMID:8159725; http://dx.doi.org/10.1073/pnas.91.8.             06109.x <br />     1111/j.1365-2958.2009.06976.x                                 3205                                                       48. Parish T, Mahenthiralingam E, Draper P, Davis EO, <br />  <br />  <br />  <br />  <br />                              Do distribute. <br /> 18. Pontes MH, Dale C. Lambda red-mediated genetic            34. Noirot P, Kolodner RD. DNA strand invasion pro-                Colston MJ. Regulation inducible acetamidase <br />     modification insect endosymbiont Sodalis               moted Escherichia coli RecT protein. J Biol Chem            gene Mycobacterium smegmatis. Microbiology 1997; <br />     glossinidius. Appl Environ Microbiol 2011; 77:1918-           1998; 273:12274-80; PMID:9575178; http://dx.doi.               143:2267-76; PMID:9245815; http://dx.doi.org/10. <br />     20; PMID:21216910; http://dx.doi.org/10.1128/AEM.             org/10.1074/jbc.273.20.12274                                   1099/00221287-143-7-2267 <br />     02166-10                                                  35. Zhang Y, Muyrers JP, Rientjes J, Stewart AF. Phage         49. Costantino N, Court DL. Enhanced levels lambda <br /> 19. Katashkina JI, Hara Y, Golubeva LI, Andreeva IG,              annealing proteins promote oligonucleotide-directed            Red-mediated recombinants mismatch repair <br />     Kuvaeva TM, Mashko SV. Use lambda Red-                 mutagenesis Escherichia coli mouse ES cells.            mutants. Proc Natl Acad Sci USA 2003; 100:15748- <br />     recombineering method genetic engineering              BMC Mol Biol 2003; 4:1; PMID:12530927; http://dx.              53; PMID:14673109; http://dx.doi.org/10.1073/pnas. <br />     Pantoea ananatis. BMC Mol Biol 2009; 10:34; PMID:             doi.org/10.1186/1471-2199-4-1                                  2434959100 <br />     19389224; http://dx.doi.org/10.1186/1471-2199-10-34       36. Datta S, Costantino N, Court DL. A set recom-           50. Lopes A, Amarir-Bouhram J, Faure G, Petit MA, <br /> 20. Kulkarni SK, Stahl FW. Interaction sbcC           bineering plasmids gram-negative bacteria. Gene            Guerois R. Detection novel recombinases in <br />     gene Escherichia coli gam gene phage            2006; 379:109-15; PMID:16750601; http://dx.doi.                bacteriophage genomes unveils Rad52, Rad51 and <br />     lambda. Genetics 1989; 123:249-53; PMID:2531105               org/10.1016/j.gene.2006.04.018                                 Gp2.5 remote homologs. Nucleic Acids Res 2010; <br /> 21. Murphy KC. The lambda Gam protein inhibits                37. Datta S, Costantino N, Zhou X, Court DL. Identifi-             38:3952-62; PMID:20194117; http://dx.doi.org/10. <br />     RecBCD binding dsDNA ends. J Mol Biol 2007;                cation analysis recombineering functions           1093/nar/gkq096 <br />     371:19-24; PMID:17583735; http://dx.doi.org/10.               Gram-negative Gram-positive bacteria         51. Marchler-Bauer A, Anderson JB, Chitsaz F, Derbyshire <br />     1016/j.jmb.2007.05.085                                        phages. Proc Natl Acad Sci USA 2008; 105:1626-31;              MK, DeWeese-Scott C, Fong JH, et al. CDD: specific <br /> 22. Marsi&#196;&#8225; N, Roje S, Stojiljkovic I, Salaj-Smic E,               PMID:18230724;           http://dx.doi.org/10.1073/pnas.       functional annotation Conserved Domain <br />     Trgovcevic Z. In vivo studies interaction           0709089105                                                     Database. Nucleic Acids Res 2009; 37:D205-10; <br />     RecBCD enzyme lambda Gam protein. J Bacteriol         38. Swingle B, Bao Z, Markel E, Chambers A, Cartinhour             PMID:18984618; http://dx.doi.org/10.1093/nar/gkn845 <br />     1993; 175:4738-43; PMID:8335632                               S. Recombineering using RecTE Pseudomonas             52. Marinelli LJ, Piuri M, Swigonova Z, Balachandran A, <br /> 23. Court R, Cook N, Saikrishnan K, Wigley D. The                 syringae. Appl Environ Microbiol 2010; 76:4960-8;              Oldfield LM, van Kessel JC, et al. BRED: simple and <br />     crystal structure lambda-Gam protein suggests            PMID:20543050;          http://dx.doi.org/10.1128/AEM.         powerful tool constructing mutant recombinant <br />     model RecBCD inhibition. J Mol Biol 2007; 371:            00911-10                                                       bacteriophage genomes. PLoS ONE 2008; 3:e3957; <br />     25-33; PMID:17544443; http://dx.doi.org/10.1016/j.        39. van Kessel JC, Hatfull GF. Recombineering Myco-             PMID:19088849; http://dx.doi.org/10.1371/journal. <br />     jmb.2007.05.037                                               bacterium tuberculosis. Nat Methods 2007; 4:147-52;            pone.0003957 <br /> 24. Yu D, Sawitzke JA, Ellis H, Court DL. Recombineer-            PMID:17179933; http://dx.doi.org/10.1038/nmeth996          53. Hsiao CL, Black LW. DNA packaging pathway <br />     ing overlapping single-stranded DNA oligonucleo-     40. Hatfull GF, Jacobs-Sera D, Lawrence JG, Pope WH,               bacteriophage T4 head assembly. Proc Natl Acad Sci <br />     tides: testing recombination intermediate. Proc Natl        Russell DA, Ko CC, et al. Comparative genomic                  USA 1977; 74:3652-6; PMID:269421; http://dx.doi. <br />     Acad Sci USA 2003; 100:7207-12; PMID:12771385;                analysis 60 Mycobacteriophage genomes: genome               org/10.1073/pnas.74.9.3652 <br />     http://dx.doi.org/10.1073/pnas.1232375100                     clustering, gene acquisition, gene size. J Mol Biol    54. V&#195;&#182;lker TA, Kuhn A, Showe MK, Bickle TA. Gene 67, <br /> 25. Stahl MM, Thomason L, Poteete AR, Tarkowski T,                2010; 397:119-43; PMID:20064525; http://dx.doi.                new, essential bacteriophage T4 head gene codes a <br />     Kuzminov A, Stahl FW. Annealing vs. invasion               org/10.1016/j.jmb.2010.01.011                                  prehead core component, PIP. II. The construction in <br />     phage lambda recombination. Genetics 1997; 147:961-       41. Pope WH, Jacobs-Sera D, Russell DA, Peebles CL, Al-            vitro unconditionally lethal mutants their <br />     77; PMID:9383045                                              Atrache Z, Alcoser TA, et al. Expanding diversity          maintenance. J Mol Biol 1982; 161:491-504; PMID: <br /> 26. Langston LD, Symington LS. Gene targeting yeast         mycobacteriophages: insights genome architec-          7154088; http://dx.doi.org/10.1016/0022-2836(82) <br />     initiated independent strand invasions. Proc           ture evolution. PLoS ONE 2011; 6:e16329;                   90403-X <br />     Natl Acad Sci USA 2004; 101:15392-7; PMID:                    PMID:21298013; http://dx.doi.org/10.1371/journal. <br />     15489271; http://dx.doi.org/10.1073/pnas.0403748101           pone.0016329 <br />  <br />  <br />  <br />  <br /> www.landesbioscience.com                                                          Bacteriophage                                                                                       13 <br />  55. M&#195;&#188;ller UR. Enzymatic construction selection         66. Bardarov S, Kriakov J, Carriere C, Yu S, Vaamonde C,      73. Sampson T, Broussard GW, Marinelli LJ, Jacobs-Sera <br />     bacteriophage G4 mutants modifications               McAdam RA, et al. Conditionally replicating myco-             D, Ray M, Ko CC, et al. Mycobacteriophages BPs, <br />     DNA secondary structure J-F intercistronic              bacte <br /> </body></html>