<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Heme Binding Proteins Bartonella henselae Are <br /> Required Undergoing Oxidative Stress During Cell <br /> and Flea Invasion <br /> MaFeng Liu1,2, Yann Ferrandez1, Emilie Bouhsira3, Martine Monteil1, Michel Franc3, Henri- <br /> Jean Boulouis1, Francis Biville1,4* <br /> 1 UMR BIPAR Universite&#194;&#180; Paris-Est, Ecole nationale ve&#194;&#180;te&#194;&#180;rinaire d&#226;&#8364;&#8482;Alfort, INRA-Anses-UPEC-ENVA, Maisons-Alfort, France, 2 Key Laboratory Zoonosis, Ministry Education, <br /> Institute Zoonosis, Jilin University, Changchun, People&#226;&#8364;&#8482;s Republic China, 3 Laboratoire parasitologie, Universite&#194;&#180; Toulouse, INP, ENVT, Toulouse, France, <br /> 4 De&#194;&#180;partement Microbiologie, Institut Pasteur, Paris, France <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Bartonella hemotropic bacteria responsible emerging zoonoses. These heme auxotroph alphaproteobacteria must <br />      import heme growth, synthesize it. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> import exogenous heme, Bartonella genomes encode for <br />      complete heme uptake enabling transportation compound cytoplasm degrading release <br />      iron. In addition, bacteria encode outer membrane heme binding proteins (Hbps). The structural genes <br />      highly homologous proteins expressed differently depending oxygen, temperature heme <br />      concentrations. These proteins hypothesized involved various cellular processes according ability <br />      bind heme regulation profile. In report, investigated roles Hbps Bartonella henselae, <br />      responsible cat scratch disease. We Hbps bind heme vitro. They able enhance efficiency of <br />      heme uptake co-expressed heme transporter Escherichia coli. Using B. henselae Hbp knockdown mutants, we <br />      proteins involved defense oxidative stress, colonization human endothelial cell and <br />      <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> flea. <br />  <br />   Citation: Liu M, Ferrandez Y, Bouhsira E, Monteil M, Franc M, et al. (2012) Heme Binding Proteins Bartonella henselae Are Required Undergoing Oxidative <br />   Stress During Cell Flea Invasion. PLoS ONE 7(10): e48408. doi:10.1371/journal.pone.0048408 <br />   Editor: Stefan Bereswill, Charite&#194;&#180;-University Medicine Berlin, Germany <br />   Received July 20, 2012; Accepted October 1, 2012; Published October 29, 2012 <br />   Copyright: &#195;&#376; 2012 Liu et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Institut Pasteur, Centre National la Recherche Scientifique (URA2172) Agence Nationale pour la <br />   Recherche [ANR-06-MIME-019-01]. MaFeng Liu supported grant China Scholarship Council. Yann Ferrandez supported grant 319 <br />   DIM Malinf (dim100103). The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: fbiville@pasteur.fr <br />  <br />  <br />  <br /> Introduction                                                                             homology compounds Fe2+ transport system, already <br />                                                                                          characterized Yersinia pestis [7] Photorhabdus luminescens [8], <br />    B. henselae recognized common                               present Bartonellae genomes. Moreover, Bartonellae genomes <br /> zoonoses acquired animal companions industrialized                               encode complete heme transport shown active in <br /> countries [1]. The bacterium causes cat scratch disease                       Bartonella quintana [9]. <br /> a number syndromes associated tumoral proliferation                           Analysis Bartonellae genomes sequenced date clearly shows <br /> of endothelial cells [2]. Most Bartonellae species appear share                       absence numerous genes proven required E. coli in <br /> a similar natural cycle involves arthropod transmission,                            order face oxidative stress [10]. Genes coding polypeptides <br /> followed exploitation mammalian host. Each Bartonella                            involved degradation hydrogen peroxide, like catalase and <br /> species appears highly adapted reservoir                         peroxidase [11], methyl sufoxide reduction (MsrA MsrB) [12] <br /> hosts causes long-lasting intra-erythrocytic bacteremia                      oxidative stress response regulation (SoxR, OxyR) [13], [14] <br /> as hallmark infection. Before colonizing erythrocytes,                          contained Bartonellae genomes. Based above <br /> bacteria need replicate competent primary                             information relating Bartonellae genome contents, be <br /> niche [3] endothelial cells, nucleated cells                      hypothesized Bartonellae highly sensitive oxidative stress. <br /> might constitute primary niche [4]. Bacterial                                However, hydrogen peroxide challenges performed B. <br /> persistence erythrocytes original strategy, considered                   bacilliformis clearly bacterium efficiently face <br /> an adaptation mode transmission bloodsucking                                exposure 1 mM H2O2 30 min [15]. Moreover, lifestyle <br /> arthropod vectors. The invasion erythrocytes                            Bartonellae suggests bacteria face oxidative stress <br /> a strategy Bartonellae species obtain heme, absolute                           blood meal arthropod vector [16]. Bartonellae <br /> requirement growth [5]. Not sequenced                             genomes small size, 1.2 2 M bases. In spite their <br /> Bartonellae genomes contain heme biosynthesis genes [6]. More-                           reduced size, encode heme binding proteins <br /> over, genomes encode siderophore biosynthesis                        (Hbps) [17]. Comparison B. quintana Hbps (HbpA, HbpB, <br /> a complete iron Fe3+ transport system. Only genes sharing strong                         HbpC HbpD) shows proteins similar in <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                           1                              October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                          Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  Table 1. Strains plasmids used study. <br />  <br />  <br />  E. coli strains                         Genotype                                                                         Source reference <br />                                           2                                                     2        2   q <br />  XL1-Blue                                F supE44 hdsR17 recA1 endA1 gyrA46 thi relA1 lac F9 proAB lacI                   Laboratory collection <br />                                          lacZDM15 Tn10 (TetR) <br />  FB8.27                                  FB8 Dlac entF::TnphoA95, TetR                                                    [44] <br />  XL1-Blue pBAD24                         XL1-Blue carrying pBAD24, AmpR                                                   This study <br />  XL1-Blue pBAD24::hbpA                   XL1-Blue carrying pBAD24::hbpA, AmpR                                             This study <br />  XL1-Blue pBAD24::hbpB                   XL1-Blue carrying pBAD24::hbpB, AmpR                                             This study <br />  XL1-Blue pBAD24::hbpC                   XL1-Blue carrying pBAD24::hbpC, AmpR                                             This study <br />  XL1-Blue pBAD24::hbpD                   XL1-Blue carrying pBAD24::hbpD, AmpR                                             This study <br />                                                             R       R <br />  FB8.27 pBAD24                           FB8.27 pBAD24, Tet , Amp                                                         This study <br />  FB8.27 pBAD24::hbpA                     FB8.27 pBAD24::hbpA, TetR, AmpR                                                  This study <br />  FB8.27 pBAD24::hbpB                     FB8.27 pBAD24::hbpB, TetR, AmpR                                                  This study <br />  FB8.27 pBAD24::hbpC                     FB8.27 pBAD24::hbpC, TetR, AmpR                                                  This study <br />  FB8.27 pAM239::hemR, pBAD24             FB8.27 pAM239::hemR, pBAD24, TetR, SpcR, AmpR                                    This study <br />  FB8.27 pAM239::hemR, pBAD24::hbpA       FB8.27 pAM239::hemR, pBAD24::hbpA, TetR, SpcR, AmpR                              This study <br />  FB8.27 pAM239::hemR, pBAD24::hbpB       FB8.27 pAM239::hemR, pBAD24::hbpB, TetR, SpcR, AmpR                              This study <br />  FB8.27 pAM239::hemR, pBAD24::hbpC       FB8.27 pAM239::hemR, pBAD24::hbpC, TetR, SpcR, AmpR                              This study <br />  FB8.27 pBAD24::hbpC                     FB8.27 pBAD24::hbpC, TetR, AmpR                                                  This study <br />                                                                                        R    R       R <br />  FB8.27 pAM239::hemR, pBAD24::hbpD       FB8.27 pAM239::hemR, pBAD24::hbpD, Tet , Spc , Amp                               This study <br />  Bartonella strains                      Genotype                                                                         Source reference <br />  B. henselae Houston-1                   Houston-1, ATCC 49882T                                                           Laboratory collection <br />  B. henselae pNS2Trc                     B. henselae carrying pNS2Trc, AmpR                                               This study <br />  B. henselae pNS2Trc::hbpA   AS          B. henselae carrying pNS2Trc::hbpA    AS,   AmpR                                 This study <br />  B. henselae pNS2Trc::hbpB   AS          B. henselae carrying pNS2Trc::hbpB    AS,   AmpR                                 This study <br />                                                                                        R <br />  B. henselae pNS2Trc::hbpC   AS          B. henselae carrying pNS2Trc::hbpC    AS, Amp                                    This study <br />                                                                                        R <br />  B. henselae pNS2Trc::hbpD   AS          B. henselae carrying pNS2Trc::hbpD    AS, Amp                                    This study <br />  Plasmids <br />  pBAD24                                  pBR322 araC, arabinose-inducible promoter, AmpR                                  Laboratory collection <br />  pNS2Trc                                 KmR                                                                              [49] <br />  pAM239::hemR                            pAM239 carrying hemR Serratia marcesns                                      [74] <br />  pBAD24::hbpA                            pBAD24 carrying hbpA B. henselae, AmpR                                      This study <br />  pBAD24::hbpB                            pBAD24 carrying hbpB B. henselae, AmpR                                      This study <br />  pBAD24::hbpC                            pBAD24 carrying hbpC B. henselae, AmpR                                      This study <br />  pBAD24::hbpD                            pBAD24 carrying hbpD B. henselae, AmpR                                      This study <br />  pNS2Trc::hbpA     AS                    pNS2Trc carrying anti-sense hbpA, kmR                                            This study <br />                                                                                  R <br />  pNS2Trc::hbpB     AS                    pNS2Trc carrying anti-sense hbpB, km                                             This study <br />  pNS2Trc::hbpC     AS                    pNS2Trc carrying anti-sense hbpC, kmR                                            This study <br />  pNS2Trc::hbpC     AS                    pNS2Trc carrying anti-sense hbpC, kmR                                            This study <br />  <br />  doi:10.1371/journal.pone.0048408.t001 <br />  <br />  <br /> size peptide sequences. The HbpA, HbpC HbpD                              conditions tested, hbpA strongly expressed hbp <br /> sequences approximately 30 kDa polypeptides nearly                  genes [19]. Based regulatory <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, hbp genes were <br /> 53% identical. HbpB distinguished Hbps,                divided groups. The contained hbpB hbpC, over <br /> polypeptide 11 kDa bigger Hbps                       expressed conditions mimick gut arthropod <br /> contains central present Hbps B. quintana               environment (high heme concentration low temperature, <br /> [17].                                                                            high O2 concentration). The transcription level hbpB hbpC <br />    To survive multiply, Bartonellae forced encounter                  higher high heme concentrations (2.5 5 mM) [19]. Those <br /> various environments hosts vectors. In mammals, free                authors suggested HbpB HbpC play critical role the <br /> heme rare [18] blood oxygen concentration low (5%                  arthropod gut [19]. The transcription hbpA, hbpD hbpE was <br /> compared 21% atmosphere) [19]. In contrast,                         increased low heme concentrations (0.05 mM) 37uC. The <br /> arthropod vectors, toxic heme level gut                  authors suggested HbpA, HbpD HbpE required <br /> oxygen concentration high [16]. The regulation pattern hbp                 free heme concentration low, blood <br /> gene expression investigated B. quintana [19]. Under             circulation mammalian host. However, recently <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 2                             October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                   Function Heme Binding Proteins B. henselae <br />  <br />  <br /> shown transcription B. henselae hbpA significantly           Immeuble Discovery - Zone Technologique 91190 Saint Aubin, <br /> increased 28uC, suggesting HbpA protect B. henselae            France) supplemented 10% fetal calf serum [29] 35uC <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> heme toxicity arthropod gut [20].                                5% CO2 atmosphere. For flea infection assays, Bartonella <br />    The function HbpA investigated B. henselae.               strains collected 5 days growth CBA plates and <br /> Controversial results excluded direct role HbpA heme            suspended PBS buffer. The bacterial suspension diluted <br /> uptake process [21], [22]. Various reports demonstrated                 PBS obtain cell density 1.986108 bacteria/ml. The <br /> HbpA B. henselae, shares homology Opa                     survival bacteria PBS buffer significantly decreased <br /> Neisseria meningitidis [23], plays role endothelial cell            24 h storage room temperature. <br /> adhesion process [24], [25]. HbpB B. tribocorum shown be <br /> required establishing long-term bacteremia rat model,           Congo Red Binding Assay <br /> its role elucidated [26]. HpbC B. henselae                  Tested strains containing pBAD derivatives expressing Hbps <br /> recently identified heme detoxifying protein [20]. Finally,          grown overnight 37uC LB medium containing 50 mg/ <br /> was suggested HbpD B. henselae required survival           ml ampicillin. Two ml LB medium containing 50 mg/ml <br /> endothelial cells [27].                                                      ampicillin inoculated OD600 0.05 overnight <br />    In report, investigated activity Hbps                 culture grown 37uC. Expression induced OD600 <br /> (HbpA, B, C, D) B. henselae using homologous                     0.6 2 h adding arabinose (0.4% final concentra- <br /> heterologous (E. coli) models. In E. coli, ability bind               tion). Cultures diluted, plated Congo red plates and <br /> Congo red heme investigated vivo vitro                     incubated 37uC 24 h. <br /> respectively. In B. henselae, hbp knockdowns checked for <br /> their growth ability, oxidative stress response capacity to <br />                                                                              E. coli Heme-dependent Growth Assays <br /> invade survive endothelial cells. Moreover, these <br />                                                                                  Tested strains grown 18 h M63 medium without <br /> mutants tested ability develop B. <br />                                                                              iron 0.4% glycerol carbon source, presence of <br /> henselae arthropod vector Ctenocephalides felis. <br />                                                                              0.02% arabinose. Cultures checked OD 600nm and <br />                                                                              adjusted OD600 = 1. A 100 ml sample bacterial suspension <br /> Materials Methods                                                        mixed 4 ml soft agar. The mixture poured onto <br /> Ethics Statement                                                             M63 plates containing 0.4% glycerol, 0.02% arabinose and <br />    Animals handled strict accordance good animal                80 mM Dip (M63D). Wells (5 mm diameter) cut the <br /> practice defined relevant European (European standards             agar filled 100 ml 50 mM, 10 mM, 5 mM, 1 mM of <br /> of welfare animals research), and/or local animal welfare             filter-sterilized Hb solution. Growth wells recorded <br /> bodies. Animal work performed Ecole Nationale ve&#194;&#180;te&#194;&#180;rinaire           2-day incubation 37uC. All experiments performed in <br /> d&#226;&#8364;&#8482;Alfort (ENVA/ANSES) reviewed approved                       triplicate. <br /> institute&#226;&#8364;&#8482;s ethics committee September 2011(agreement nu:14/ <br /> 06/2011-1). Animal work performed Ecole nationale                     Physiological Characterization hbp Knockdown Strains <br /> ve&#194;&#180;te&#194;&#180;rinaire Toulouse reviewed approved INRA                 To evaluate effect hbp knockdown growth B. henselae, <br /> Toulouse/ENVT ethics committees (agreement nu: MP/01/22/                     tested strains grown liquid Schneider&#226;&#8364;&#8482;s medium and <br /> 06/09) 3 years.                                                          CBA plates. B. henselae pNS2Trc B. henselae <br />                                                                              pNS2Trc::hbpsAS collected 5 days growth CBA <br /> Bacterial Strains Plasmids                                               plates suspended Schneider&#226;&#8364;&#8482;s medium phosphate <br />   Bacterial strains plasmids used study listed            buffered saline (PBS). For growth Schneider&#226;&#8364;&#8482;s medium, the <br /> Table 1.                                                                     OD600 bacterial suspension adjusted 0.05. Five ml <br />                                                                              samples suspension poured 6-well plates and <br /> Media Growth Conditions                                                  grown 35uC presence 5% CO2. OD600 checked at <br />    Bovine hemoglobin (Hb) 2, 29dipyridyl (Dip) obtained             days 2, 4, 5 7 inoculation. For growth CBA plates, <br /> from Sigma-Aldrich France (38297 Saint-Quentin Fallavier-                    serial dilutions bacterial suspension PBS plated CBA <br /> France). Heme dissolved immediately use                        plates colony size evaluated 6 10 days growth <br /> 0.02 M NaOH. Hb dissolved 100 mM NaCl. Heme                       35uC presence 5% CO2. All experiments were <br /> Hb solutions filter-sterilized 0.20 mm Millipore filters       performed triplicate. <br /> bacterial growth experiments. E. coli strains grown LB <br /> medium (Sigma), M63 minimal medium, aerobically 37uC [28]                 H2O2 Challenge <br /> or Congo red plates. M63 medium supplemented                        B. henselae pNS2Trc B. henselae pNS2Trc::hbpsAS were <br /> 0.4% glycerol (Gly) carbon source. Solid media soft agar              grown CBA plates days 35uC 5% CO2 <br /> respectively contained 1.5% 0.7% Difco agar. Congo red plates             atmosphere. Bacteria collected plate suspended <br /> consisted solid BHI medium (Difco) supplemented Congo                washed twice PBS buffer. The cell suspension then <br /> red dye (0.02% final concentration). Iron-depleted medium                diluted OD600 0.5. Before H2O2 challenge, dilutions <br /> obtained addition Dip 80 mM final concentration.           tested cell suspension spread CBA plates (T0). <br /> Antibiotics added following final concentrations (mg             For challenge assay, bacteria incubated 30 min PBS <br /> ml21): ampicillin (Amp), 50; kanamycin (Km), 50; spectino-               buffer presence 1 mM 10 mM H2O2 35uC <br /> mycin (Spc), 50. Arabinose (Ara) added final concentra-             5% CO2 atmosphere. After exposure H2O2, bacteria <br /> tion 0.02%, 0.2% 0.4% induction Para promoter. B.           washed twice PBS buffer dilutions plated <br /> henselae grown Columbia blood agar (CBA) plate                      CBA plates (T1). After 15-day incubation 35uC under <br /> containing 5% defibrinated sheep blood (Biome&#194;&#180;rieux; ref 43041; 5,           5% CO2 atmosphere, colonies counted. Survival rate <br /> rue des Aqueducs BP 10 - 69290 Craponne, France)                       expressed (T1/T0) X100%. All experiments were <br /> Schneider&#226;&#8364;&#8482;s medium (Gibco; Route L&#226;&#8364;&#8482;Orme des Merisiers                     performed triplicate. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               3                        October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                    Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br /> Endothelial Cell Culture Invasion Assay                                Flea Maintenance Supply <br />    Endothelial cell line Ea.hy 926 resulting fusion                Strain Ctenocephalides felis (C. felis) (Siphonaptera: Pulicidae) originat- <br /> HUVEC lung carcinoma cell line A549 (ATCC; reference                   ing wild strain harvested cat maintained on <br /> CRL 2922) cultured DMEM medium (Gibco)                             cat laboratory conditions 1990. A new generation is <br /> supplemented 10% fetal bovine serum decomplemented                    obtained 2 4 weeks. Fleas controlled PCR <br /> by heating 30 min 56uC use. When required,                       negative B. henselae using primers bh2390fo, bh2390re <br /> kanamycin added 50 mg/ml final concentration. Cells               (Table 2). <br /> were incubated 37uC humidified 5% CO2 cultured <br /> every 7 days using 0.025% trypsin 1 mM EDTA Hanks&#226;&#8364;&#8482;                  Feeding C. felis B. henselae pNS2Trc- B. henselae <br /> balanced salt solution [30]. Endothelial cells seeded 24-          pNS2Trc::hbpsAS-infected Blood <br /> well plates density 104 cells/well. After 6 days, cell                Dog blood used experiments obtained 3 beagles <br /> number estimated 1.56105/well. Bartonella strains                   (15 ml obtained dog) Ectoparasite Laboratory <br /> harvested CBA plates 5-day growth 35uC                 National Veterinary School Toulouse, France. The <br /> a 5% CO2 atmosphere washed twice modified DMEM                     absence Bartonella spp. blood dogs confirmed <br /> buffer resuspended medium. Bacterial                  PCR using primers bh2390fo, bh2390re (Table 2). Lithium <br /> number adjusted 36104/ml (1OD estimated                      heparin&#226;&#8364;&#8220;coated vacutainer tubes (Venosafe, Terumo Europe) were <br /> 6.66108/ml). For cell invasion assays, medium              used draw blood venipuncture. Blood functional comple- <br /> removed 0.5 ml modified DMEM containing 1.56104                     ment deactivated maintaining blood samples room <br /> colony-forming units (CFUs) B. henselae tested strains         temperature 2 h blood test storing at <br /> added obtain 0.1 multiplicity infection (m.o.i).         4uC. Blood samples stored 48 h 4uC. When <br /> A low m.o.i. used considered cell excess            required, kanamycin added blood 50 mg/ml final <br /> suitable visualize invading ability bacteria. After         concentration. Kanamycin previously determined no <br /> remaining 1 h 37uC, 1.5 ml modified DMEM medium                  effect C. felis feeding, viability egg production. A total 500 <br /> was supplemented well. The number bacteria                   unfed fleas (males females aged 8&#226;&#8364;&#8220;10 days) were <br /> controlled plating dilutions CBA plates, CFUs            placed plexiglas box contact glass feeder closed the <br /> were determined 15 days growth (T0). Mixtures                parafilm membrane. To stimulate flea blood-feeding, <br /> incubated 37uC 5% CO2 24 h. After 24 h, number               constant temperature (38.5uC) maintained water-jacket <br /> of viable bacteria determined plating serial dilutions           circulation glass feeder. For blood infection, <br /> mixtures CBA plates, CFUs determined 15                  500 mL bacterial suspension concentration approxi- <br /> days growth (T24). After 24 h, bacterial viability nearly              mately 1.986108 bacteria/ml PBS added 5 ml blood. <br /> 100%. The intracellular bacterial population quantified             Viability Bartonella blood 100% 2-h <br /> the gentamicin protection assay described Mehock [31].               incubation. Blood complemented bacterial suspension for <br /> Briefly, DMEM gentamicin (final concentration 250 mg/ml)              days feeding. Then, fleas fed uninfected <br /> was added mixture assay removed 2 h 37uC               dog blood 8 days. Every 24 h, glass feeder was <br /> in presence 5% CO2. A control B. henselae bacterial                 cleaned, new parafilm membrane stretched blood was <br /> suspension showed survival 2 h exposure gentamicin             changed. At time, flea feces collected. All samples <br /> (250 mg/ml). After removing modified DMEM medium contain-                  stored 220uC PCR analysis. Ethanol 70% added <br /> ing gentamicin, endothelial cells washed times             flea feces samples. <br /> with modified DMEM medium remove residual antibiotic. <br /> Endothelial cells collected 4 min incubation with <br /> 400 ml trypsin 37uC. After centrifuging 12,000 rpm, cells            Genetic Techniques <br /> were suspended 1 ml sterile water disrupted using                  E. coli cells transformed calcium chloride method <br /> a 1 ml syringe 0.4 mm620 mm needle 5 pushes                      [33]. Bartonella cells transformed electroporation as <br /> [32]. Microscopic controls revealed that, 5 pushes, cells        previously described [34]. <br /> were lysed. Cell lysates supplemented 100 ml 106PBS <br /> to overcome osmotic lysis. The number viable bacteria               DNA Manipulations <br /> determined plating lysates CBA plates. After 15 days                A small-scale plasmid DNA preparation performed using <br /> incubation 35uC 5% CO2 atmosphere, colonies                QIAprep Spin Miniprep kit (Qiagen; QIAGEN France 3 av du <br /> counted (TI). The invasion rate expressed (TI/T0)x100%.             Canada LP 809 91974 Courtaboeuf). Restriction, modification, <br /> Each assay performed double wells experiments               ligation carried according manufacturer&#226;&#8364;&#8482;s <br /> were performed triplicate.                                              recommendations. DNA fragments amplified Hybaid <br />                                                                            PCR thermocycler using Phusion DNA polymerase (Finnzymes). <br /> Survival Assay Endothelial Cells                                        Nucleotide sequencing performed Eurofins MWG Operon. <br />    To perform survival assays B. henselae pNS2Trc B. henselae       Purification DNA fragments PCR reaction, restriction <br /> pNS2Trc::hbpsAS endothelial cells, monolayer contained           reaction agarose gels performed using Macherey-Nagel <br /> mixtures challenged gentamicin killing washed times          NucleoSpinH Extract II kit. <br /> and incubated modified DMEM medium 37uC 5% CO2 <br /> for 24 48 h. Cells treated described             Construction Recombinant Vector Expressing Hbps <br /> number viable bacteria determined plating lysates         B. henselae <br /> CBA plates. After 15-day incubation 35uC 5% CO2                    Complete B. henselae hbpA, hbpB, hbpC hbpD genes C- <br /> atmosphere, colonies counted (TS24 TS48). The survival             terminal Histag (6 His) amplified PCR B. henselae <br /> rate expressed (TS24/TI TS48/TI)x100%. Each assay            chromosomal DNA using primers HbpABhamont HbpAB- <br /> performed double wells experiments performed            haval, HbpBBhamont HbpBBhaval, HbpCBhamont and <br /> triplicate.                                                                HbpCBhaval HbpDBhamont HbpDBhaval, respectively <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             4                           October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  Table 2. Primers used study. <br />  <br />  <br />  Primer                      Gene            Organism      Sequence <br />  <br />  HbpABhamont                 hbpA            B. henselae   59 CTAGCTAGCAGGAGGAATTCACCATGAATATAAAATCTTTAATGA 39 <br />  HbpABhaval                  hbpA            B. henselae   59 CGGGGTACCTCAGTGGTGGTGGTGGTGGTGGAATTTGTACGCTACACCAACACGG 39 <br />  HbpBBhamont                 hbpB            B. henselae   59 CTAGCTAGCAGGAGGAATTCACCATGAATACGAAACGTTTAATAACAG 39 <br />  HbpBBhaval                  hbpB            B. henselae   59 ATCCCCGAAGCTTATGGTTTAGTGGTGGTGGTGGTGGTGGAATTTGTAAGCGACAC 39 <br />  HbpCBhamont                 hbpC            B. henselae   59 CTAGCTAGCAGGAGGAATTCACCATGAAATCGCGTGTTCAAATAT39 <br />  HbpCBhaval                  hbpC            B. henselae   59 CGGGGTACCTCAGTGGTGGTGGTGGTGGTGAAATTTATAAGCGACACCAACACGG 39 <br />  HbpDBhamont                 hbpD            B. henselae   59 CTAGCTAGCAGGAGGAATTCACCATGACTACAAAATATTTAATCACAA 39 <br />  HbpDBhaval                  hbpD            B. henselae   59 CGGGGTACCTCAGTGGTGGTGGTGGTGGTGAAACTTGTACGCTACACCAACACGA 39 <br />  hbpAantisensamt             Antisens hbpA   B. henselae   59 CCCGGATCCTTAGAATTTGTACGCTACACC39 <br />  hbpAantisensavl             Antisens hbpA   B. henselae   59CCCTCTAGAATGAATATAAAATCTTTAATG 39 <br />  hbpBantisensamt             Antisens hbpB   B. henselae   59 CCCGGATCCTTAGAATTTGTAAGCGACACC39 <br />  hbpBantisensavl             Antisens hbpB   B. henselae   59 CCCTCTAGAATGAATACGAAACGTTTAATAAC39 <br />  hbpCantisensamt             Antisens hbpC   B. henselae   59 CCCGGATCCTTAAAATTTATAAGCGACACC39 <br />  hbpCantisensavl             Antisens hbpC   B. henselae   59 CCCTCTAGAATGAATATAAAATGGTTAATA39 <br />  hbpDantisensamt             Antisens hbpD   B. henselae   59 CCCGGATCCTCAAAACTTGTACGCTACACCA 39 <br />  hbpDantisensavl             Antisens hbpD   B. henselae   59 CCCTCTAGAATGACTACAAAATATTTAATCACA 39 <br />  bh2390fo                    bh2390          B. henselae   59-GGTGAATGTGTGCAAAGTTTTAAG 39 <br />  bh2390re                    bh2390          B. henselae   59 CCAATAAACGCCAACAAAGAC 39. <br />  Cf18Sf                      18S rDNA        C. felis      59 TGCTCACCGTTTGACTTGG 39 <br />  Cf18Sr                      18S rDNA        C. felis      59 GTTTCTCAGGCTCCCTCTCC 39 <br />  <br />  doi:10.1371/journal.pone.0048408.t002 <br />  <br />  <br /> (Table 2). The fragments amplified (890 bp hbpA, 881 bp           overnight 56uC pre-lysis step. For samples, final <br /> hbpC 875 bp hbpD) purified digested NheI            elution volume 100 mL. The concentration DNA extraction <br /> and KpnI. The hbpB fragment (1235 bp) purified digested           samples measured using nanodrop spectrophotometer <br /> with NheI Hind III. Then, fragments ligated                 (NanoDrop 2000, Thermo Scientific). <br /> pBAD24 plasmid digested NheI KpnI NheI Hind <br /> III. Ligation mixtures introduced CaCl2-competent E.            Detection B. henselae DNA Flea Feces <br /> coli strain XL1 Blue cells transformants selected LB             DNA B. henselae detected amplification 1,052 bp <br /> plates containing ampicillin. Clones screened PCR             fragment containing B. henselae bh02390 gene https://www. <br /> method corresponding primers. Six PCR-positive clones           genoscope.cns.fr/agc/microscope/home/index.php using primers <br /> then sequenced.                                                           bh2390fo, bh2390re (Table 2). DNA C. felis detected by <br />                                                                           amplification fragment C. felis 18S rDNA using primers <br /> Construction Knockdown Vector Decreasing                     Cf18Sf Cf18Sr (Table 2) [35]. Amplifications performed <br /> Amount Hbp B. henselae                                              20 ng DNA extract flea feces. Each reaction was <br />    The entire coding region hbpA, hbpB, hbpC hbpD              conducted total volume 20 mL 0.5 mM primer, <br /> amplified PCR B. henselae chromosomal DNA using               200 mM dNTP, 4 mL 56 PCR buffer G/C 0.02U/ <br /> primers hbpAantisensamt hbpAantisensavl, hbpBantisensamt              ml Taq DNA polymerase (PhusionH High-Fidelity DNA <br /> and hbpBantisensavl, hbpCantisensamt hbpCantisensavl               Polymerase, Thermo scientific). The PCR program follows: <br /> hbpDantisensamt hbpDantisensavl, respectively (Table 2).              initial denaturation step 30 s 98uC, followed 40 cycles <br /> The PCR product (846 bp hbpA, 1,182 bp hbpB, 837 bp           denaturation 10 s 98uC, annealing 30 s 55uC and <br /> hbpC 831 bp hbpD) purified, digested BamHI           extension 1 min 72uC, final extension step 72uC for <br /> XbaI ligated plasmid pNS2Trc digested                  7 min. <br /> BamHI XbaI. Ligation mixtures introduced CaCl2- <br /> competent E. coli strain XL1 Blue cells. Transformants               Expression Purification Recombinant His-tagged <br /> screened PCR corresponding primers. Six PCR-positive              Hbps <br /> clones sequenced.                                                 Strain XL1-Blue pBAD24::hbpA, XL1-Blue pBAD24::hbpB, <br />                                                                           XL1-Blue pBAD24::hbpC XL1-Blue pBAD24::hbpD were <br /> Extraction DNA Flea Feces                                         grown overnight 37uC LB medium containing 50 mg/ml <br />    DNA extracted flea feces using Nucleospin tissue          ampicillin. Then, 200 ml LB medium containing 50 mg/ml <br /> kit according manufacturer&#226;&#8364;&#8482;s instructions (NucleoSpinH             ampicillin inoculated OD600 0.05 the <br /> Tissue, Macherey-Nagel; 1, Rue Gutenberg, B. P. 135, 67722                overnight culture grown 37uC. Expression induced at <br /> Hoerdt France). The quantity biological material used DNA          OD600 0.6 2 h adding arabinose (0.4% final <br /> extraction 30&#226;&#8364;&#8220;40 mg feces. Flea feces incubated         concentration). Bacteria harvested centrifugation for <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            5                        October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                Function Heme Binding Proteins B. henselae <br />  <br />  <br /> 10 min 3,000 g 4uC, pellet suspended 20 ml            (Hybond-C Extra, GE Healthcare; VWR International S.A.S Le <br /> binding buffer (50 mM Tris-HCl, 8 M urea, 0.05% triton,                    Pe&#194;&#180;rigares - Ba&#203;&#8224;t B 201, Rue Carnot 94126 Fontenay-sous-Bois <br /> pH 8.0). Lysis bacteria obtained incubation room              cedex) according Towbin et al [37]. Non-specific binding sites <br /> temperature rotation 3 h. The suspension                 blocked 5% skim milk TBS-Tween 20(0.05%). The <br /> centrifuged 13,000 g 30 min 4uC. The supernatant                 immunoblot probed polyclonal mouse sera raised against <br /> containing soluble fraction mixed 200 ml Ni-               recombinant HbpB (1:200), followed 1:1,000 dilution of <br /> agarose beads (Qiagen) according manufacturer&#226;&#8364;&#8482;s instruc-            rabbit anti-mouse IgG horseradish peroxidase[HRP]-conjugated <br /> tions. Purified protein dialyzed twice buffer contain-       secondary antibody (cell signaling; Ozyme, 10 avenue Ampe`re, BP <br /> ing 50 mM Tris-HCl eliminate residual imidazole                 128, 78053 St Quentin en Yvelines France). Visualization the <br /> urea. The protein stable month kept           HRP signal performed using enhanced chemiluminescence <br /> 280uC 20% glycerol.                                                   (ECL) reagents (Amersham Pharmacia, Piscataway, N.J). HbpA <br />                                                                            bands visualized exposing blot autoradiographic <br /> Heme Binding Assay vitro                                                film (Labscientific, Livingston, NJ, USA). <br />    For heme blotting, proteins separated standard 12.5% <br /> SDS gels followed electrophoretic transfer nitrocellulose            Protein Assay <br /> membranes according protocol Vargas [36]. Briefly,                  The concentration protein determined using BC <br /> samples mixed loading buffer DTT                 assay protein quantitation kit (Interchim). <br /> added samples boiled electrophoresis. 1 mg of <br /> HbpA, HbpB, HbpC HbpD 3 mg BSA separated                  Statistical Analysis <br /> 12.5% SDS-PAGE. One gel stained Coomassie brilliant                  Statistical analysis performed using GraphPad Prism 5 <br /> Blue R. Another gel transferred nitrocellulose using            software Windows. Statistical significance data was <br /> general methods Towbin et al. [37]. The heme binding blot           ascertained use Student&#226;&#8364;&#8482;s t test. A value P,0.05 was <br /> done according protocol Carroll et al. [22]. Briefly,        considered significant. <br /> resulting blots rinsed Tris-buffered saline containing <br /> 0.1% Tween 20 (TBST; 10 mM pH 8.0 Tris-HCl containing                      Results <br /> 150 mM NaCl 0.1% Tween 20) times 30 min and <br /> subsequently probed 1.5 h TBS containing heme                     Hbps Able Bind Congo Red Expressed E. <br /> (1026 M) room temperature. Nitrocellulose washed              coli <br /> times 30 min TBS-Tween 20 (0.1%) room tempera-                   To determine Hbps B. henselae bind heme, first <br /> ture. Heme visualized intrinsic peroxidase activity [38]        checked Congo red binding activity, shown <br /> using enhanced chemiluminescence (ECL) reagents (Amersham                  Hbps bind Congo red [40]. Moreover, already <br /> Pharmacia, Piscataway, N.J). Hbp bands visualized                  shown expressing Congo red binding protein the <br /> exposing blot autoradiographic film (Labscientific, Living-         outer membrane E. coli conferred Congo red binding <br /> ston, N.J.).                                                               <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> [41]. We transformed E. coli XL1-Blue plasmids <br />                                                                            pBAD24, pBAD24::hbpA, pBAD24::hbpB, pBAD24::hbpC and <br /> Antibody Preparation                                                       pBAD24::hbpD determine hbp genes produce <br />   200 ml emulsion containing purified HbpB (10 mg)               Congo red binding phenotype. Strain XL1-Blue plasmid <br /> ISA 61 VG adjuvant (Seppic) (120 ml), completed NaCl                  pBAD24 did confer Congo red phenotype grown on <br /> 0.9%, inoculated twice (at 1-month interval) subcutane-             plate 0.2% arabinose (Fig. lB). However, strains <br /> ously 4 weeks old C57B6 mice (Charles River) (Ethics                  expressing HbpA, B, C D formed red colony plate <br /> Committee Anses/ENVA/UPEC agreement nu:14/06/2011-1).                      0.2% arabinose (Fig. 1 C, D, E, F). Without arabinose, <br /> Two weeks second inoculation, 200 ml blood samples               XL1-Blue pBAD24 XL1-Blue pBAD24::hbpA, XL1- <br /> were collected 3 weeks retro-orbital bleeding. Blood             Blue pBAD24::hbpB, XL1-Blue pBAD24::hbpC XL1-Blue <br /> samples centrifuged twice (3,600 rpm 5 min) obtain serum           pBAD24::hbpD, conferred Congo red binding phenotype (data <br /> which stored 220uC. Before use, non-specific antibodies             shown). Taken together, data showed Hbps can <br /> were removed incubating immune serum E. coli cell              bind Congo red confirmed Hbps exposed cell <br /> extract 1 h 4uC centrifugation 10 min 8,000 rpm.         surface expressed E. coli. <br /> The supernatant used serum. <br />                                                                            Recombinant HbpA, HbpB, HbpC HbpD can <br /> Protein Analysis Electrophoresis                                        Specifically Bind Heme vitro <br />   Proteins analyzed 12% sodium dodecyl sulfate poly-                 To produce purify recombinant HbpA, B, C, D E. coli, <br /> acrylamide gel (SDS-PAGE) electrophoresis [39], followed                amplified structural gene using B. henselae chromosomal <br /> Coomassie Blue staining.                                                   DNA template primers allowing addition His-tag at <br />                                                                            C-terminal protein. The fragments cloned the <br /> Immunoblot Analysis                                                        pBAD24 plasmid described Materials methods. Plasmids <br />    Sodium dodecyl sulfate/polyacrylamide gel electrophoresis               pBAD24::hbpA, pBAD24::hbpB, pBAD24::hbpC pBAD24::hbpD <br /> (SDS-PAGE) immunoblotting detecting decreased expres-              introduced strain XL1-Blue. To check quantities of <br /> sion HbpA B. henselae performed follows. Tested              Hbps E. coli strain XL1-Blue, SDS gel electrophoresis (SDS- <br /> strain biomass collected 5 days growth CBA plates,         PAGE) used compare protein extracts strain XL1-Blue <br /> suspended PBS buffer centrifuged. Bacterial pellets                 pBAD24::hbpshis XL1-Blue pBAD24. A supplementary visible <br /> calculated contain 40 mg proteins suspended loading          band observed SDS page gel XL1-Blue derivatives <br /> buffer heated 5 min 100uC. Proteins separated           harboring plasmid pBAD24::hbpA, pBAD24::hbpB, pBAD24::hbpC <br /> 12% SDS-PAGE transferred nitrocellulose membrane                  pBAD24::hbpD grown presence 0.4% arabinose <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             6                          October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                    Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  <br /> Figure 1. Congo red binding assays. Strains XL1-Blue pBAD24 XL1-Blue pBAD24 expressing Hbps expressed 2 h 37uC as <br /> described Materials methods. Expression evident SDS-PAGE (A). Line 1: XL1-Blue pBAD24, lines 2 5: XL1-Blue pBAD24 expressing <br /> HbpA, B, C, D, respectively (20 mg). After plating Congo red plates supplemented 0.2% arabinose, strains grown 20 h 37uC. Strain <br /> XL1-Blue pBAD24 (B) formed white colonies. Strain XL1-Blue pBAD24::hbpA (C), XL1-Blue pBAD24::hbpB (D), XL1-Blue pBAD24::hbpC (E) XL1-Blue <br /> pBAD24::hbpD (F) formed red colonies. <br /> doi:10.1371/journal.pone.0048408.g001 <br />  <br /> (Fig. 1A). Recombinant Hbp proteins purified Ni-agarose              bind heme added 1026M concentration (Fig. 2B). In contrast, <br /> affinity chromatography described Materials methods.               assay conditions, BSA unable bind heme <br /> The size purified Hbp proteins corresponded                 (Fig. 2B). The result underlines specificity proteins <br /> predicted sequences.                                              bind heme. Taken whole, conclude when <br />    To test pure Hbps specifically bind heme vitro,            purified, Hbps bind heme vitro. <br /> a standard method used detecting heme binding of <br /> cytochrome C used [36]. Pure HbpA, HbpB, HbpC, HbpD                      Hbps Increase Efficiency Heme Uptake Mediated <br /> and BSA separated SDS-polyacrylamide gels. One                   HemR Serratia marcescens E. coli <br /> gel stained Coomassie brilliant Blue R (Fig. 2A).                 Conflicting results concerning heme transport activity of <br /> Another gel transferred nitrocellulose filter perform            HbpA encouraged check ability Hbps to <br /> heme blotting ECL detection (Fig. 2B). Pure Hbps able               transport heme using E. coli hemA mutant complementation. Our <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               7                         October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                    Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  <br /> Figure 2. Detection heme binding Hbps. After SDS gel electrophoresis, gel stained Coomassie brilliant Blue R. Another gel <br /> was transferred nitrocellulose filter perform heme blotting detected ECL described Materials methods. (A) Coomassie <br /> Blue staining result. Line 1:1 mg HbpA, line 2:1 mg HbpB, line 3:1 mg HbpC, line 4:1 mg HbpD, line 5:3 mg BSA. (B) Heme binding results. Line 1:1 mg <br /> HbpA, line 2:1 mg HbpB, line 3:1 mg HbpC, line 4:1 mg HbpD, line 5:3 mg BSA. <br /> doi:10.1371/journal.pone.0048408.g002 <br />  <br /> results demonstrated Hbps B. henselae did act heme          process. HbpB HbpD efficient HbpA and <br /> transporters (Table 3). These results variance             HbpC (Table 3). To investigate efficiency heme <br /> obtained Pap31 (HbpA) B. henselae [21],                  uptake mediated Hbps HemR, grew bacteria on <br /> agreement obtained HbpA B. quintana [9]                minimal iron-depleted plate 100 mM Dip presence of <br /> Bartonella birtlesii (Biville F, unpublished data). Heme binding             different concentrations Hb. For control strain only <br /> proteins act heme porins, ability bind heme           expressing HemR strains co-expressing HbpA HbpC and <br /> can increase heme concentration bacterial surface. This               HemR, growth observed the <br /> local increase heme concentration facilitate uptake        concentration Hb added (data shown). The strain co- <br /> bacteria. We checked Hbps modulate efficiency              expressing HbpD HemR showed growth well <br /> of heme uptake process. We verified effect Hbps          containing 50 mM Hb. The strain co-expressing HbpB HemR <br /> the activity HutA B. henselae expressed E. coli. HutA B.       able grow concentration of <br /> quintana shown transport heme and, consequently, restore           Hb (data shown). Taken together, concluded HbpB <br /> growth E. coli hemA mutant. [9]. Experiments                      HbpD efficient increasing heme <br /> performed presence high heme concentrations liquid              uptake expressed E. coli, play an <br /> medium. The weak growth restoration                        important role low heme concentrations. Similar results were <br /> consequence mutations enhancing outer membrane permeabil-                 obtained introducing plasmids harboring hbps genes E. coli <br /> ity [42]. These complementation assays failed E. coli K-              entF mutant expressing HasR S. marcescens (data shown). <br /> 12 hemA mutant expressing HutA B. birtlessii grown                Binding heme Hbps increased concentration the <br /> heme-supplemented solid medium [43]. To avoid factual results                bacteria consequently facilitated uptake. <br /> concerning heme transportation activity HutA B. <br /> henselae, expressed E. coli entF mutant grow         Hbp Activity Important B. henselae Growth <br /> in presence iron chelator [44]. When heme transporter               Since Hbps modulate heme uptake efficiency when <br /> is expressed strain, addition heme medium                  expressed E. coli, similar activity hypothesized B. <br /> restores growth. In order used iron source,           henselae. In bacterium, heme serves heme an <br /> of heme required 100 times greater required use         iron source [47]. Thus, abolishing decreasing synthesis the <br /> as heme source. Such complementation assay              different Hbps B. henselae potentially affect growth <br /> used check heme transport activity HasR Serratia              capacity. <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>Knockout</span> HbpA B. henselae hypothesized as <br /> marcescens [45] characterization heme-degrading              lethal bacteria [20]. A preliminary unsuccessful assay <br /> enzymes [46], [47]. No growth observed E. coli entF               lab disrupt hbpA B. henselae agreement this <br /> strain expressing HutA B. henselae grown iron-                  hypothesis. To investigate function Hbps B. <br /> depleted medium presence hemoglobin (data shown).              henselae using genetic tool, chose knockdown <br /> To check effect Hbps B. henselae heme               method successfully used B. henselae [47], [49], <br /> uptake process, plasmids harboring hbps genes introduced                [50]. We cloned hbps B. henselae oriented reverse direction <br /> into E. coli entF mutant [44] expressing HemR heme                     anti-sense strand transcribed plasmid pNSTrc <br /> transporter S. marcescens [48]. Strains obtained tested        [49]. Plasmids pNS2Trc, pNS2Trc::hbpAAS, pNS2Trc::hbpBAS <br /> growth iron-depleted medium presence hemoglobin                 pNS2Trc::hbpCAS pNS2Trc::hbpDAS introduced B. <br /> added different concentrations. As seen table 3, HemR            henselae using electroporation. We checked knockdown <br /> led growth containing hemoglobin 50 mM              effect Hbp expression level B. henselae using western blot <br /> the presence 80 mM dip. Strains expressing hbpA, hbpB, hbpC               experiment. Multiple sequence alignment Hbps B. henselae <br /> and hbpD able grow lower concentrations Hb                     revealed high degree amino acid sequence conservation its <br /> (Table 3), suggesting HbpA, HbpB, HbpC HbpD                         N-terminal C-terminal parts HbpB, longer <br /> increase efficiency HemR-mediated heme uptake                     Hbps. According size, HbpB easily be <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               8                         October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                                Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  Table 3. The effect Hbps HemR-dependent heme                                      oxidative stress generated exposure H2O2 [15]. It was <br />  uptake.                                                                                 suggested potential role Hbps bind heme the <br />                                                                                          surface bacteria provide antioxidant barrier via <br />                                                                                          heme intrinsic peroxidase activity [57]. To examine hypoth- <br />                               Diameter halo (cm)                                  esis, tested effect hbp knockdown ability B. <br />  E. coli strain <br />                                                                                          henselae face 30 min exposure 1 mM 10 mM hydrogen <br />                               50 mM          10 mM        5 mM       1 mM <br />                                                                                          peroxide. After exposure 1 mM H2O2, survival 50% <br />  FB8.27 pBAD24                NM             NM           NM         NM                  control strain B. henselae pNS2Trc B. henselae <br />  FB8.27 pBAD24::hbpA,         NM             NM           NM         NM                  pNS2Trc::hbpCAS. There existed slightly decreased survival capac- <br />  B, C, D                                                                                 ity B. henselae pNS2Trc::hbpAAS, B. henselae pNS2Trc::hbpBAS <br />  FB8.27 pAM::hemR,            1.8260.015     NM           NM         NM                  B. henselae pNS2Trc::hbpDAS (data shown). After exposure <br />  pBAD24                                                                                  10 mM H2O2, survival 25% control strain B. <br />  FB8.27 pAM::hemR,            2.2360.04      1.7860.03    1.4860.09 NM                   henselae pNS2Trc. With strain B. henselae pNS2Trc::hbpAAS, B. <br />  pBAD24::hbpA                                                                            henselae pNS2Trc::hbpBAS B. henselae pNS2Trc::hbpCAS, sensi- <br />  FB8.27 pAM::hemR,            2.7760.11      2.3760.08    2.3160.09 1.8460.1             tivity hydrogen peroxide increased 3-4-fold (Fig. 5). <br />  pBAD24::hbpB                                                                            Decreasing HbpD levels sharply increased B. henselae <br />  FB8.27 pAM::hemR,            2.0960.04      1.6460.03    1.4260.08 NM                   sensitivity hydrogen peroxide (Fig. 5). These results indicated <br />  pBAD24::hbpC                                                                            lowering Hbp level B. henselae significantly increased its <br />  FB8.27 pAM::hemR,            2.5560.10      1.9260.09    1.6860.09 1.2260.06            sensitivity H2O2. <br />  pBAD24::hbpD <br />  <br />  E. coli strains FB8.27 pBAD24, FB8.27 pBAD24::hbpA, FB8.27 pBAD24::hbpB, <br />                                                                                          Effect Hbp Knockdown B. henselae Capacity to <br />  FB8.27 pBAD24::hbpC, FB8.27 pBAD24::hbpD, FB8.27 pAM239::hemR pBAD24,                   Invade Endothelial Cells <br />  FB8.27 pAM239::hemR pBAD24::hbpA, FB8.27 pAM239::hemR pBAD24::hbpB,                        Within cells, bacterial infection shown induce ROS <br />  FB8.27 pAM239::hemR pBAD24::hbpC FB8.27 pAM239::hemR pBAD24::hbpD <br />  tested efficiency heme utilization iron source iron-depleted <br />                                                                                          production [58]. Thus, decreasing ability undergo oxidative <br />  medium M63 (Gly 0.4%, Ara 0.02%, Dip 80 mM, Spc, Amp).                                  stress expected decrease ability B. henselae survive in <br />  Growth wells containing 1 mM, 5 mM, 10 mM, 50 mM Hb                endothelial cells. This promoted check effect Hbp <br />  described Materials methods. After 48 h growth, diameter           knockdown capacity B. henselae invade human <br />  zone turbidity aground measured quadruplicate each <br />  plate mean diameter calculated. Results expressed mean 6 <br />                                                                                          endothelial cells survive them. <br />  SD diameter (in cm) obtained plates. NM: Not measurable.              For endothelial cell invasion, B. henselae pNS2Trc::hbpBAS <br />  doi:10.1371/journal.pone.0048408.t003                                                   B. henselae pNS2Trc::hbpCAS exhibited invasion rate <br />                                                                                          control strain B. henselae pNS2Trc (about 2%) (Fig. 6). However, <br /> distinguished Hbps. Mouse anti-serum HbpB                             invasion rates B. henselae pNS2Trc::hbpAAS B. henselae <br /> was prepared tested recognition purified Hbps. Our                            pNS2Trc::hbpDAS decreased 3-fold compared control strain <br /> results demonstrated anti-HbpB antibody recognize well-                         (Fig. 6). <br /> purified HbpB HbpA. For HbpD HbpC, recognition                                   For survival endothelial cell assay, mixtures grown <br /> anti-HbpB antibodies lower absent (data shown).                               24 h 48 h gentamicin killing assay. After 24 h in <br /> Detection HbpB expression B. henselae investigated using                       endothelial cell, cell lysates spread blood plate to <br /> anti-HbpB anti-serum. We failed detect HbpB B. henselae                            check viable bacterial number. Surprisingly, bacteria were <br /> using western blot method. This result good agreement                          visible blood plate 2-week incubation. To overcome <br /> with proteomic analysis outer membrane fractions B.                       problem, grew mixture Schneider&#226;&#8364;&#8482;s liquid <br /> henselae B. quintana [51], [52], [53], [54], [55], [56],                       medium overnight, sustaining primary isolation B. henselae [29] <br /> showed HbpB detectable using method. Since                             plating blood plates. Colony count 2-week <br /> HbpA abundant Hbp B. henselae [52], [51],                              incubation showed decrease viable bacteria strains <br /> checked level B. henselae pNS2Trc B. henselae                                 tested, claimed B. henselae able to <br /> pNS2Trc::hbpAAS using mouse antibody directed HbpB.                              replicate inside endothelial cell bacterial rRNA replication <br /> As seen Fig. 3, level HbpA lower strain B. henselae                     [59]. Survival rates B. henselae pNS2Trc::hbpsAS much <br /> pNS2Trc::hbpAAS strain B. henselae pNS2Trc.                                      lower control B. henselae pNS2Trc (Fig. 7), 24 h <br />    The strains obtained tested growth blood agar                        48 h growth endothelial cells. To check effect of <br /> plates Schneider&#226;&#8364;&#8482;s medium described Materials                           overnight growth Schneider&#226;&#8364;&#8482;s medium, 600 B. henselae <br /> methods. After 6 10 days growth blood agar plates,                             pNS2Trc B. henselae pNS2Trc::hbpsAS bacteria grown <br /> colonies produced strain B. henselae harboring                                        overnight medium plated blood plates for <br /> pNS2Trc::hbpAAS, pNS2Trc::hbpBAS, pNS2Trc::hbpCAS                                     enumeration. The increase bacterial numbers calculated <br /> pNS2Trc::hbpDAS smaller formed B.                                tested strains. The increase 20&#226;&#8364;&#8220;30% B. <br /> henselae pNS2Trc (Table 4). The B. henselae strain containing                            henselae pNS2Trc B. henselae pNS2Trc::hbpsAS. The differing <br /> pNS2Trc::hbpsAS grew slowly B. henselae pNS2Trc                           survival rates B. henselae pNS2Trc B. henselae pNS2Trc::hbps <br /> in Schneider&#226;&#8364;&#8482;s medium (Fig. 4). These results indicated                             AS related growth defect Schneider&#226;&#8364;&#8482;s medium. We <br /> a decrease Hbps slowed growth B. henselae.                           conclude Hbps B. henselae play important role survival <br />                                                                                          endothelial cells. <br /> Knockdown Hbps Increased B. henselae Sensitivity to <br /> Hydrogen Peroxide                                                                        Hbps Involved Multiplication B. henselae C. <br />   Analyses Bartonellae genomes demonstrated numerous                             felis <br /> genes involved oxidative stress response present.                           It demonstrated B. henselae-infected cat fleas (C. felis) can <br /> However, shown Bartonella bacilliformis able undergo                     transmit B. henselae cats [60] B. henselae replicate in <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                           9                          October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                      Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  <br /> Figure 3. Detection HbpA expression levels B. henselae pNS2Trc B. henselae pNS2Trc::hbpA AS immunoblotting. 40 mg <br /> samples B. henselae pNS2Trc (line 1) B. henselae pNS2Trc::hbpAAS (line 2) 100 ng sample purified his-tagged HbpA (line 3) loaded <br /> on SDS-PAGE. After electrophoresis, gel stained Coomassie brilliant Blue R (A). Another gel transferred nitrocellulose filter for <br /> performing immune blotting described Materials methods (B). Measurement HbpA band intensity using Image J software gave the <br /> following results: B. henselae pNS2Trc: integrated density: 52; B. henselae pNS2Trc::hbpAAS: integrated density: 26. <br /> doi:10.1371/journal.pone.0048408.g003 <br />  <br /> the gut cat flea [61]. Inside arthropod gut, bacteria                amounts quality DNA flea feces, amplified <br /> confront oxidative stress blood meal [16]. Since Hbps             flea 18S rDNA day-7-to-10 flea feces samples using primers <br /> B. henselae play protective role H2O2-produced oxidative              Cf18Sf Cf18Sr. Flea 18S rDNA detected day- <br /> stress, prompted check Hbp knockdown effect                7-to-10 flea feces samples (data shown). This suggests that <br /> B. henselae multiplication fleas. After feeding fleas 2 days        Hbps play important role multiplication B. henselae fleas. <br /> blood containing bacteria, fleas fed blood without <br /> bacteria 8 days. For control strain B. henselae pNS2Trc,            Discussion <br /> we able detect B. henselae DNA feces day 1 to <br /> day 10 (Table 5). For strains decreased HbpA,                    In report, investigated role B. henselae Hbps in <br /> HbpB, HbpC HbpD, bacterial DNA detected                    heme utilization, oxidative stress response, cell colonization <br /> day 6. To exclude possibility result small             survival arthropod vector C. felis. Previous data had <br />  <br />  <br />  <br />  <br /> Figure 4. Effect hbps knockdown growth B. henselae Schneider&#226;&#8364;&#8482;s medium. Strains B. henselae pNS2Trc B. henselae <br /> pNS2Trc::hbpsAS cultured Schneider&#226;&#8364;&#8482;s medium, collected 5-day growth CBA blood plates suspended Schneider&#226;&#8364;&#8482;s medium. The <br /> bacterial suspension used inoculate Schneider&#226;&#8364;&#8482;s medium OD600 0.05. Cultures grown 35uC presence 5% CO2 and <br /> OD600 measured days 2, 4, 5 7 inoculation. All experiments repeated times. <br /> doi:10.1371/journal.pone.0048408.g004 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 10                         October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                                 Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  Table 4. Effect hbps knockdown growth B. henselae <br />  blood plates. <br />  <br />  <br />  Strain                        Colony size (mm) <br />  <br />                                Day 6           Day 10 <br />  <br />  B. henselae (pNS2Trc)         0.5860.12       1.0060.149 <br />  B. henselae                   NM              0.5060.105 (p,0.0001) <br />  (pNS2Trc::hbpAAS) <br />  B. henselae                   NM              0.5360.125 (p,0.0001) <br />  (pNS2Trc::hbpBAS) <br />  B. henselae                   NM              0.5160.120 (p,0.0001) <br />  (pNS2Trc::hbpCAS) <br />  B. henselae                   NM              0.44960.072 (p,0.0001) <br />  (pNS2Trc::hbpDAS) <br />  <br />  For growth test CBA plates, strains B. henselae (pNS2Trc), B. henselae <br />  (pNS2Trc::hbpA AS), B. henselae (pNS2Trc::hbpB AS), B. henselae (pNS2Trc::hbpC AS) <br />  B. henselae (pNS2Trc::hbpDAS) collected 5 days growth CBA <br />  plates suspended PBS buffer obtain 103 CFU ml21. Two- <br />  microliters cell suspension plated CBA plate. Colony <br />  sizes measured 6 10 days growth 35uC presence of <br />  5% CO2. Data mean diameter (mm) 6 SD 10 colonies one <br />  representative experiment. Standard deviations calculated using Statview <br />  software. All experiments repeated times. NM: measurable. <br />  doi:10.1371/journal.pone.0048408.t004 <br />                                                                                            Figure 5. Hbp knockdown decreases ability B. henselae to <br />                                                                                            undergo exposure H2O2. B. henselae pNS2Trc B. henselae <br /> shown recombinant HbpA B. quintana able bind                                pNS2Trc::hbpsAS challenged 10 mM H2O2 described in <br /> heme vitro, did confer heme binding phenotype vivo                         Materials methods. Experiments performed triplicate; <br /> when expressed E. coli [22]. Later claimed HbpB                          survival rates expressed described Materials methods. <br />                                                                                            (*P,0.05, **P,0.01 compared B. henselae pNS2Trc). <br /> B. quintana did bind heme [19]. Recently, shown                            doi:10.1371/journal.pone.0048408.g005 <br /> hbpC B. henselae, expressed E. coli, confers heme <br /> binding phenotype vivo [20]. <br />    Our results clearly expression Hbps B. <br /> henselae E. coli confers Congo red binding phenotype (Fig. 1), <br /> thus suggesting Hbps surface location expressed <br /> in E. coli. We demonstrate that, vitro, purified Hbps <br /> specifically bind heme. To characterize physiological impor- <br /> tance Hbps, investigated effect heme uptake <br /> process. Pap31 (HbpA) B. henselae claimed act heme <br /> porin expressed E. coli [21], conflicting data this <br /> activity published HbpA B. quintana [57]. Moreover, we <br /> failed visualize heme porin activity Hbps B. henselae <br /> when expressed E. coli (data shown). Based above <br /> results, hypothesized Hbp family Bartonella act <br /> as heme reservoir, rendering available heme- <br /> limited conditions. This hypothesis good agreement the <br /> absence genes encoding heme iron storage proteins in <br /> Bartonellae genomes. Such heme storage activity enhance the <br /> efficiency heme uptake process. We examined effect of <br /> Hbps activity heterologous heme transporters HemR and <br /> HasR S. marcescens [48], [62]. All Hbps increased heme uptake <br /> efficiency mediated HemR HasR. Efficiency low heme <br /> concentrations better HbpB HbpD HbpA <br /> and HbpC. Such differences efficacy related the <br /> differing levels Hbps E. coli. This case HbpB, not <br /> for HbpD (Fig. 1). This result suggests HbpD active <br /> when heme concentration low. This conclusion good                             Figure 6. Effect Hbp knockdown endothelial cell <br /> agreement increase hbpD expression B. quintana                            invasion. Invasion endothelial cells B. henselae pNS2Trc B. <br /> was grown presence low heme concentrations [19]. How                             henselae pNS2Trc::hbpsAS. Cells mixed bacteria 0.1 m.o.i. <br /> heme transferred heme binding proteins HemR                                 After 24 h, infected cell treated gentamicin kill <br />                                                                                            extracellular bacteria lysates plated CBA blood plate <br /> HasR heme transporters remains unknown.                                                    determine number intracellular bacteria. Invasion was <br />    During mammal flea invasion, Bartonellae face                                  calculated using equation provided Materials methods. <br /> microenvironmental shifts, stress host immune defense.                             (***P,0.005 compared B. henselae pNS2Trc). <br /> For example, shown reactive oxygen species (ROS:                               doi:10.1371/journal.pone.0048408.g006 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                            11                         October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                                  Function Heme Binding Proteins B. henselae <br />  <br />  <br />  <br />  <br /> Figure 7. Effect Hbp knockdown survival B. henselae endothelial cells. Survival B. henselae pNS2Trc B. henselae <br /> pNS2Trc::hbpsAS endothelial cell. After gentamicin killing, infected cell grown 24 h 48 h modified DMEM medium. Lysates were <br /> incubated Schneider&#226;&#8364;&#8482;s medium overnight. Then lysates collected centrifugation plated CBA blood plates determine the <br /> number intracellular bacteria. The survival rates 24 h growth (A) 48 h growth (B) calculated using equation provided Materials <br /> and methods. (*P,0.05, **P,0.01, ***P,0.005 compared B. henselae pNS2Trc). <br /> doi:10.1371/journal.pone.0048408.g007 <br />  <br /> O22, H2O2 OH2) production important immune                                  adaptive response ROS thoroughly investigated <br /> defense mechanism mammal hosts arthropod vectors                              [16]. B. henselae replicates gut cat flea able to <br /> against pathogenic bacteria [16], [63]. Recently, shown                   survive days flea feces [61]. It shown the <br /> ROS (H2O2) levels midgut higher (over 10 mM) Y. pestis                     hematophagous vector substantial need huge amounts of <br /> infected fleas. Antioxidant treatment prior infection decreased                    blood meal; digestion hemoglobin gut the <br /> ROS levels resulted higher Yersinia pestis loads [64]. An                      vector releases large quantities heme, potential pro- <br /> OxyR Y. pestis mutant showed reduced growth fleas early                      oxidant cytotoxic effects bound proteins [69], [70]. <br /> infection [64]. ROS potentially toxic host cell                  To successfully replicate cat flea gut, B. henselae must <br /> pathogenic bacteria [16], [65], [66], [67]. Host cells protected                  confront toxic ROS repair damage. Many homologues of <br /> from oxidative damage enzymes detoxify ROS,                           genes involved oxidative stress response E. coli not <br /> SOD, catalase, glutathione peroxidase (Gpx) thioredoxin                           present B. henselae genome. The absence homologues of <br /> peroxidase, detoxify H2O2 [68]. In arthropod vectors,                        genes suggests Bartonellae possess uncharacterized <br />  <br />  Table 5. Detection B. henselae DNA flea feces samples using PCR. <br />  <br />  <br />                                                                                                Flea feces samples (days) <br />  <br />  Strain                                                                                        1     2      3       4     5     6     7      8     9      10 <br />  <br />  B. henselae pNS2Trc                                                                           +     +      +       +     +     +     +      +     +      + <br />  B. henselae pNS2Trc::hbpAAS                                                                   +     +      +       +     +     +     2      2     2      2 <br />  <br />  B. henselae pNS2Trc::hbpBAS                                                                   +     +      +       +     +     +     2      2     2      2 <br />  B. henselae pNS2Trc::hbpCAS                                                                   +     +      +       +     +     +     2      2     2      2 <br />  B. henselae pNS2Trc::hbpDAS                                                                   +     +      +       +     +     +     2      2     2      2 <br />  <br />                                                                             8 <br />  About 500 fleas feed blood containing 500 ml bacteria (2610 /ml) 2 days fed uninfected blood 8 days. Flea feces collected every <br />  day. DNA extracted flea feces PCR performed described Materials methods. <br />  doi:10.1371/journal.pone.0048408.t005 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       12                              October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                                         Function Heme Binding Proteins B. henselae <br />  <br />  <br /> mechanisms response oxidative stress. One actor                                    Clearance bacteria flea feces explained by <br /> oxidative stress response recently identified heme-                            decrease survival arthropod. Our results clearly <br /> degrading enzyme enabling release iron heme [47]. In                             demonstrate rapid clearance B. henselae flea feces <br /> report, demonstrate Hbps required efficiently                              invasion assays performed using Hbp knockdown <br /> undergo exposure hydrogen peroxide. A role intrinsic                               mutants. In addition, rapid clearance bacteria in <br /> peroxidase activity heme bound Hbps hypothesized                                flea feces attributed decreased ability confront <br /> B. quintana [19] shown case Porphyromonas                               oxidative stress. Thus, ROS-detoxifying activity Hbps also <br /> gingivalis [71]. The competence Hbps detoxifying H2O2                             plays important role colonization fleas. Taken <br /> also required efficient invasion survival endothelial                           together, reveal functions Hbps heme utilization, the <br /> cells, major target cell types bacterial colonization                              oxidative stress response, cell colonization flea transmission. <br /> reservoir host(s) infected host [72]. Bacterial                             The ability Hbps bind heme provides competence in <br /> infection shown induce ROS production endothelial cells                             destroying ROS, constituting important immune defense <br /> [58]. As consequence, weaker survival capacity endothelial                           host cell arthropod vectors. Functional identifica- <br /> cells, related knockdown Hbps, related lower                            tion Hbp families cell interactions flea transmission <br /> resistance oxidative stress.                                                               help develop strategies fighting infection and <br />    For HbpA HbpD, weakening                                         transmission. It interesting future elucidate anti- <br /> endothelial cell invasion process level decreases. For                             oxidant mechanisms used vector-borne pathogens. <br /> HbpA, effect related Opa domain, has <br /> been shown involved entry host epithelial cell                           Acknowledgments <br /> Neisseria gonorrhoeae [73]. Indeed, previous data showed                            We thank Dr. Gillaspie Dr. Anderson (College Medecine, <br /> Pap31 (HbpA) B. henselae able bind endothelial cells                             University South Florida, USA) generous gift plasmid <br /> a dose-dependent manner, binding inhibited anti-                                   pNS2Trc. We thank Seppic (Paris, France) generous gift ISA <br /> Pap31 antibodies [24]. However, HbpD involvement                                       61 VG adjuvant. <br /> endothelial cell invasion process striking, explained <br /> by greater sensitivity oxidative stress produced endothelial                            Author Contributions <br /> cells, leading rapid killing bacteria. This hypothesis                        Conceived designed experiments: MFL FB. Performed the <br /> is supported fact mutant HbpD delay                                 experiments: MFL YF EB MM FB. Analyzed data: MFL FB. <br /> lysosomal fusion, leading significantly reduced viability                           Contributed reagents/materials/analysis tools: MF HJB. Wrote paper: <br /> endothelial cells [27].                                                                       MFL BF. <br />  <br /> References <br />  1. Edouard S, Raoult D (2010) [Bartonella henselae, ubiquitous agent                   17. Minnick MF, Sappington KN, Smitherman LS, Andersson SG, Karlberg O, et <br />     proteiform zoonotic disease]. Med Mal Infect 40: 319&#226;&#8364;&#8220;330.                                     al. (2003) Five-member gene family Bartonella quintana. Infect Immun 71: <br />  2. Klotz SA, Ianas V, Elliott SP (2011) Cat-scratch Disease. Am Fam Physician 83:                814&#226;&#8364;&#8220;821. <br />     152&#226;&#8364;&#8220;155.                                                                                  18. Wandersman C, Delepelaire P (2004) Bacterial iron sources: siderophores <br />  3. Harms A, Dehio C (2012) Intruders Radar: Molecular Pathogenesis                  hemophores. Annu Rev Microbiol 58: 611&#226;&#8364;&#8220;647. <br />     Bartonella spp. Clin Microbiol Rev 25: 42&#226;&#8364;&#8220;78.                                             19. Battisti JM, Sappington KN, Smitherman LS, Parrow NL, Minnick MF (2006) <br />  4. Mandle T, Einsele H, Schaller M, Neumann D, Vogel W, et al. (2005) Infection                  Environmental signals generate differential coordinated expression the <br />     human CD34+ progenitor cells Bartonella henselae results                           heme receptor gene family Bartonella quintana. Infect Immun 74: 3251&#226;&#8364;&#8220; <br />     intraerythrocytic presence B. henselae. Blood 106: 1215&#226;&#8364;&#8220;1222.                              3261. <br />  5. Sander A, Kretzer S, Bredt W, Oberle K, Bereswill S (2000) Hemin-dependent                20. Roden JA, Wells DH, Chomel BB, Kasten RW, Koehler JE (2012) Hemin <br />     growth hemin binding Bartonella henselae. FEMS Microbiol Lett 189:                     binding protein C outer membrane vesicles protects Bartonella <br />     55&#226;&#8364;&#8220;59.                                                                                        henselae toxic concentrations hemin. Infect Immun 80: 929&#226;&#8364;&#8220;942. <br />  6. Alsmark CM, Frank AC, Karlberg EO, Legault BA, Ardell DH, et al. (2004)                   21. Zimmermann R, Kempf VA, Schiltz E, Oberle K, Sander A (2003) Hemin <br />     The louse-borne human pathogen Bartonella quintana genomic derivative                 binding, functional expression, complementation analysis Pap 31 from <br />     zoonotic agent Bartonella henselae. Proc Natl Acad Sci U S A 101: 9716&#226;&#8364;&#8220;                   Bartonella henselae. J Bacteriol 185: 1739&#226;&#8364;&#8220;1744. <br />     9721.                                                                                     22. Carroll JA, Coleman SA, Smitherman LS, Minnick MF (2000) Hemin-binding <br />  7. Perry RD, Mier I, Jr., Fetherston JD (2007) Roles Yfe Feo                          surface protein Bartonella quintana. Infect Immun 68: 6750&#226;&#8364;&#8220;6757. <br />     transporters Yersinia pestis iron uptake intracellular growth. Biometals        23. Simon D, Rest RF (1992) Escherichia coli expressing Neisseria gonorrhoeae <br />     20: 699&#226;&#8364;&#8220;703.                                                                                  opacity-associated outer membrane protein invade human cervical and <br />  8. Watson RJ, Millichap P, Joyce SA, Reynolds S, Clarke DJ (2010) The role                    endometrial epithelial cell lines. Proc Natl Acad Sci U S A 89: 5512&#226;&#8364;&#8220;5516. <br />     iron uptake pathogenicity symbiosis Photorhabdus luminescens TT01.              24. Dabo SM, Confer AW, Anderson BE, Gupta S (2006) Bartonella henselae <br />     BMC Microbiol 10: 177. <br />                                                                                                   Pap31, extracellular matrix adhesin, binds fibronectin repeat III13 <br />  9. Parrow NL, Abbott J, Lockwood AR, Battisti JM, Minnick MF (2009) Function, <br />                                                                                                   module. Infect Immun 74: 2513&#226;&#8364;&#8220;2521. <br />     regulation, transcriptional organization hemin utilization locus of <br />                                                                                               25. Dabo SM, Confer AW, Saliki JT, Anderson BE (2006) Binding Bartonella <br />     Bartonella quintana. Infect Immun 77: 307&#226;&#8364;&#8220;316. <br />                                                                                                   henselae extracellular molecules: identification potential adhesins. Microb <br /> 10. Storz G, Imlay JA (1999) Oxidative stress. Curr Opin Microbiol 2: 188&#226;&#8364;&#8220;194. <br />                                                                                                   Pathog 41: 10&#226;&#8364;&#8220;20. <br /> 11. Schellhorn HE (1995) Regulation hydroperoxidase (catalase) expression in <br />                                                                                               26. Saenz HL, Engel P, Stoeckli MC, Lanz C, Raddatz G, et al. (2007) Genomic <br />     Escherichia coli. FEMS Microbiol Lett 131: 113&#226;&#8364;&#8220;119. <br />                                                                                                   analysis Bartonella identifies type IV secretion systems host adaptability <br /> 12. Weissbach H, Resnick L, Brot N (2005) Methionine sulfoxide reductases: history <br />                                                                                                   factors. Nat Genet 39: 1469&#226;&#8364;&#8220;1476. <br />     cellular role protecting oxidative damage. Biochim Biophys Acta <br />     1703: 203&#226;&#8364;&#8220;212.                                                                            27. Kyme PA, Haas A, Schaller M, Peschel A, Iredell J, et al. (2005) Unusual <br /> 13. Amabile-Cuevas CF, Demple B (1991) Molecular characterization soxRS                    trafficking pattern Bartonella henselae -containing vacuoles macrophages <br />     genes Escherichia coli: genes control superoxide stress regulon. Nucleic             endothelial cells. Cell Microbiol 7: 1019&#226;&#8364;&#8220;1034. <br />     Acids Res 19: 4479&#226;&#8364;&#8220;4484.                                                                  28. Miller JH (1972) Experiments molecular genetics.: Cold Spring Harbor <br /> 14. Storz G, Tartaglia LA, Ames BN (1990) The OxyR regulon. Antonie Van                           laboratory Press, Cold Spring Harbor, N.Y. <br />     Leeuwenhoek 58: 157&#226;&#8364;&#8220;161.                                                                  29. Riess T, Dietrich F, Schmidt KV, Kaiser PO, Schwarz H, et al. (2008) Analysis <br /> 15. Coleman SA, Minnick MF (2003) Differential expression invasion-                        novel insect cell culture medium-based growth medium Bartonella <br />     associated locus B (ialB) gene Bartonella bacilliformis response                     species. Appl Environ Microbiol 74: 5224&#226;&#8364;&#8220;5227. <br />     environmental cues. Microb Pathog 34: 179&#226;&#8364;&#8220;186.                                            30. Chang CC, Chen YJ, Tseng CS, Lai WL, Hsu KY, et al. (2011) A comparative <br /> 16. Graca-Souza AV, Maya-Monteiro C, Paiva-Silva GO, Braz GR, Paes MC, et al.                     study interaction Bartonella henselae strains human endothelial <br />     (2006) Adaptations heme toxicity blood-feeding arthropods. Insect                  cells. Vet Microbiol 149: 147&#226;&#8364;&#8220;156. <br />     Biochem Mol Biol 36: 322&#226;&#8364;&#8220;335. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                               13                              October 2012 | Volume 7 | Issue 10 | e48408 <br />                                                                                                                           Function Heme Binding Proteins B. henselae <br />  <br />  <br /> 31. Mehock JR, Greene CE, Gherardini FC, Hahn TW, Krause DC (1998)                             54. Eberhardt C, Engelmann S, Kusch H, Albrecht D, Hecker M, et al. (2009) <br />     Bartonella henselae invasion feline erythrocytes vitro. Infect Immun 66:                 Proteomic analysis bacterial pathogen Bartonella henselae and <br />     3462&#226;&#8364;&#8220;3466.                                                                                     identification immunogenic proteins serodiagnosis. Proteomics 9: 1967&#226;&#8364;&#8220; <br /> 32. Kempf VA, Volkmann B, Schaller M, Sander CA, Alitalo K, et al. (2001)                          1981. <br />     Evidence leading role VEGF Bartonella henselae-induced endothelial             55. Boonjakuakul JK, Gerns HL, Chen YT, Hicks LD, Minnick MF, et al. (2007) <br />     cell proliferations. Cell Microbiol 3: 623&#226;&#8364;&#8220;632.                                                Proteomic immunoblot analyses Bartonella quintana total membrane <br /> 33. Maniatis T, Fritch EF, Sanbrook J (1989) Molecular Cloning. A Laboratory                       proteins identify antigens recognized sera infected patients. Infect <br />     Manual; Harbor CS, editor. New York: Cold Spring Harbor Laboratory Press.                      Immun 75: 2548&#226;&#8364;&#8220;2561. <br /> 34. Riess T, Anderson B, Fackelmayer A, Autenrieth IB, Kempf VA (2003) Rapid                   56. Rhomberg TA, Karlberg O, Mini T, Zimny-Arndt U, Wickenberg U, et al. <br />     efficient transposon mutagenesis Bartonella henselae transposome                     (2004) Proteomic analysis sarcosine-insoluble outer membrane fraction of <br />     technology. Gene 313: 103&#226;&#8364;&#8220;109.                                                                 bacterial pathogen Bartonella henselae. Proteomics 4: 3021&#226;&#8364;&#8220;3033. <br /> 35. Reif KE, Stout RW, Henry GC, Foil LD, Macaluso KR (2008) Prevalence                    57. Minnick MF, Battisti JM (2009) Pestilence, persistence pathogenicity: <br />     infection load dynamics Rickettsia felis actively feeding cat fleas. PLoS One            infection strategies Bartonella. Future Microbiol 4: 743&#226;&#8364;&#8220;758. <br />     3: e2805. <br />                                                                                                58. Sahni SK, Rydkina E (2009) Host-cell interactions pathogenic Rickettsia <br /> 36. Vargas C, McEwan AG, Downie JA (1993) Detection c-type cytochromes <br />                                                                                                    species. Future Microbiol 4: 323&#226;&#8364;&#8220;339. <br />     using enhanced chemiluminescence. Anal Biochem 209: 323&#226;&#8364;&#8220;326. <br />                                                                                                <br /> </body></html>