<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yeast spindle pole body duplication gene MPS1 <br /> encodes an essential dual specificity protein kinase <br />  <br />  <br />                             duplication transit GI and S phases and give rise <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> cells <br />                        with monopolar spindles. Three distinct aberrant SPB <br />                                                morphologies have been detected in mutants, and these <br />                                                                morphologies appear to indicate which step of SPB <br />  <br />    duplication has failed in a given mutant (Winey et al., <br />         1991). Generally, mutants defective in SPB duplication <br />  <br />     exhibit a transient G2 arrest in the cell cycle. At the non- <br />           permissive temperature, the mutants arrest as large budded <br />                                         cells, with post-S phase DNA content and monopolar <br />                                                                spindles. The exception to this behavior is the mpsl-J <br />                                       mutant, which uniquely identifies a post-mating factor <br />                                                                arrest step in SPB duplication (Winey et al., 1991). At <br /> The MPS1 gene has been previously identified by a              the non-permissive temperature, mpsl-l cells exhibit no <br /> mutant allele that shows defects in spindle pole body          pause in the cell cycle and commit aberrant DNA segrega- <br /> (SPB) duplication and cell cycle control. The SPB              tion and cytokinesis. Indeed, MPSJ has been implicated <br /> is the centrosome-equivalent organelle in the yeast            in a mitotic checkpoint that arrests the cell cycle in <br /> Saccharomyces cerevisiae, and it nucleates all the micro-      response to failed SPB duplication (E.Weiss and M.Winey, <br /> tubules in the cell. We report the isolation of the            in preparation). <br /> MPSI gene, which encodes an essential protein kinase              Molecular analyses of genes involved in S.cerevisiae <br /> homolog. The MPSI open reading frame has been                  SPB duplication have identified several types of proteins <br /> fused to those that encode the LexA protein or the             that are involved in the control and/or execution of this <br /> GST protein and both of these constructs function in           process. The CDC31 gene product is a small Ca2+ binding <br /> yeast. The fusion proteins have been affinity-purified         protein of the calmodulin family that has been shown to <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> yeast extracts and the GST chimeric protein has           be a SPB component (Sprang et al., 1993; Biggins and <br /> been found to be a phosphoprotein. Both proteins have          Rose, 1994). The KARI gene product is a novel protein <br /> been used to demonstrate intrinsic in vitro protein            of 53 kDa, and may also be a SPB component (Rose and <br /> kinase activity of Mpslp against exogenous substrates          Fink, 1987; Vallen et al., 1992). Mutant alleles of KARI <br /> and itself (autophosphorylation). A <span id='am-318' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-319' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>mutation</span> predicted         show genetic interactions with CDC31 and its mutant <br /> to abolish kinase function not only eliminates in vitro <br /> protein kinase activity, but also behaves like a null          alleles (Vallen et al., 1994), and Karlp binds Cdc31p <br /> mutation in vivo, suggesting that kinase activity con-         in vitro (Biggins and Rose, 1994). Other SPB components <br /> tributes to the essential function of the protein.             include the product of the NUFI/SPCIIO gene, which is <br /> Phosphoamino acid analysis of substrates phos-                 a coiled-coil protein (Mirzayan et al., 1992; Kilmartin <br /> phorylated by Mpslp indicates that this kinase can             et al., 1993), and calmodulin, which binds the NUFI/ <br /> phosphorylate serine, threonine and tyrosine residues,         SPCI1O gene product (Geiser et al., 1993). Finally, the <br /> identifying Mpslp as a dual specificity protein kinase.        NDCJ gene is required for a late step in SPB duplication <br /> Key words: mitosis/MPSI/protein kinase/spindle pole            and it encodes a hydrophobic protein that appears to be a <br /> body/yeast                                                     constituent of the nuclear envelope (Winey et al., 1993). <br />                                                                   We report here the isolation and characterization of the <br />                                                                MPSJ gene and its gene product. The gene encodes a 764 <br />                                                                amino acid protein with significant homology to protein <br />                                                                kinases. MPSJ is the same gene as the recently reported <br /> Introduction                                                   RPKI kinase homolog (Poch et al., 1994). These workers <br /> The yeast Saccharomyces cerevisiae has as its sole micro-      found MPSJ as an interesting open reading frame (ORF) <br /> tubule organizing center the spindle pole body (SPB).          adjacent to the ACT2 gene that they had previously <br /> This organelle serves as the yeast equivalent to the           identified (Schwob and Martin, 1992). Poch et al. (1994) <br /> centrosome of mammalian cells. Like centrosomes, the           report that MPSJ is essential. They found that MPSI <br /> SPB is duplicated during a precise interval in the cell        mRNA levels were constant in mitotic cells, decreased in <br /> cycle to yield the two poles of the mitotic spindle. This      mating factor-treated cells, and increased in meiotic cells. <br /> process occurs during the GI interval of the S.cerevisiae      They also suggest that MPSJ may encode a dual specificity <br /> cell cycle and is coordinated with other cell cycle events.    protein kinase, based on significant amino acid sequence <br /> SPB duplication has been characterized cytologically, and      homology in the kinase domain between MPSJ and the <br /> several mutants have been identified that are specifically     mammalian dual specificity protein kinases PYTfI7TK <br /> defective in SPB duplication (reviewed by Winey and            (Mills et al., 1992; Lindberg et al., 1993) and esk (Douville <br /> Byers, 1993). As a class, the mutants that fail in SPB         et al., 1992). We show that the MPSJ gene product <br />  <br /> &#194;&#169; Oxford University Press                                                                                             1 655 <br />  E.Lauze et al. <br />  <br /> (Mps Ip) does indeed have protein kinase activity in vitro,                                                              ScS <br /> and that the kinase phosphorylates serine, threonine and             B                   H E          E H C              XB    K            Cbmplgon) <br />                                                                                                                                               mpsol-1? <br /> tyrosine residues. Furthermore, a GST-Mpsl fusion                                                                                                y <br /> protein is phosphorylated in yeast, suggesting that Mpslp                                                                                        y <br />                                                                                                                                                  N <br /> is a phosphoprotein in vivo. We have implicated an                                                                                               N <br /> essential, dual specificity protein kinase in yeast SPB                                                                                          N <br />                                                                                                                                                  N <br /> duplication and cell cycle control.                                                                                      F,s <br />                                                                                                                                                  N <br />                                                                                                                                                  N <br />                                                                                                                          Ffs                     N <br /> Results                                                                                                                                          N <br /> Isolation of the MPSl gene                                           0.5 kb          MPS1 ORF           -       0-0000000-0- <br /> MPSJ was isolated from the yeast genomic library con- <br /> structed by Rose et al. (1987) in the centromeric plasmid,       b            ,4DHI promos)       I    LA   I    PSI <br />                                                                                                                 AMI                     I        y <br /> YCp5O. One plasmid that complemented the mpsl-J                                   prmw) <br />                                                                               (GALl           I GST I           -PSI                j            y <br /> mutation was isolated independently nine times, and                                                                        -77772M,r&apos; <br /> all nine isolates could complement mpsl-J upon re-                            (GALI pr_ol)    I   GST   I       IIIPS1              I            N <br /> introduction into yeast. The restriction map of the -9 kb        Fig. 1. Restriction map and complementation data for the identification <br /> insert in this plasmid (YCpMPS1) is shown in Figure 1.           of the MPSJ gene (a) and the fusion protein constructs (b) used for <br /> The region required for complementation of mpsl-J was            analysis of protein kinase activity. The top line (a) depicts the original <br /> identified by deleting regions of the genomic insert in          clone of MPSJ, with restriction sites B (BamHI), H (HindIII), E <br /> YCpMPS1 as shown in Figure 1. This analysis showed               (EcoRI), C (CIaI), X (XbaI), K (KpnI), Sc (Sacl) and S (SphI). The <br />                                                                  lower lines show subclones and mutants including frameshift mutants <br /> that only a 3.5 kb fragment from one end of the insert           (FS) and the insertion/deletion <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>null allele</span> used (HIS3+, see Materials <br /> was necessary for complementation of mps1-]. This data           and methods). The fusion protein constructs (b) and the MPSJ ORF <br /> is consistent with two &apos;frameshift&apos; mutations made by            are aligned with the above restriction map. The constructs are <br /> filling in and ligating at unique restriction sites within the   described in Materials and methods. The kinase domain in MPSJ is <br />                                                                  shown as the cross-hatched region, and the asterisk on the bottom <br /> complementing fragment (Figure 1).                               construct marks the position of the &apos;kinase-dead&apos; mutation (see <br />    Various methods were used to demonstrate that the             Materials and methods). <br /> complementing fragment in YCpMPS1 contains the bona <br /> fide MPSJ gene. First, the EcoRI-SalI fragment con-              fragment internal to the MPSJ ORF was labeled and used <br /> taining the putative MPSJ gene was subcloned from                to probe genomic yeast DNA that had been cut with various <br /> YCpMPS1 to pRS306, an integrative plasmid (Sikorski              restriction enzymes, and only one fragment was detected <br /> and Hieter, 1989). The resulting plasmid (YIpMPS1) was           in each lane, except for two digests where the restriction <br /> integrated in the genome of yeast strain Wx257-5c (Table         enzymes are known to cut in the segment used as a probe <br /> I). Transformants were crossed to the mpsl-] mutant strain       (data not shown). This result is consistent with the physical <br /> Wx241-12b (Table I). The integrated marker, URA3, on             mapping of the MPSJ gene reported above, which identified <br /> YIpMPS1 showed no recombination with mpsl-] in 25                only one locale in the genome. These results suggest that <br /> tetrads, demonstrating that the cloned fragment and the          the MPSJ gene is unique in S.cerevisiae. <br /> mutation are linked. Furthermore, the URA3 integrant                The strategy used for DNA sequencing involved cloning <br /> showed linkage to trp, as predicted by the previously            restriction fragments into phage M1 3 and the use of <br /> reported map position of mpsl (10 cM distal to trpl;             custom oligonucleotide primers. DNA sequence analysis <br /> Winey et al., 1991). The second line of evidence that the        of the complementing insert in YCpMPS 1 revealed a <br /> bona fide MPSJ gene had been cloned came from mapping            single long ORF of 2292 nucleotides with the potential <br /> the complementing DNA of YCpMPS 1 to the yeast                   to encode a protein of 764 amino acids with a calculated <br /> physical map. An EcoRV fragment was used to probe the            mol. wt of 87 kDa. The derived amino acid sequence of <br /> overlapping set of X phage used to construct the yeast           Mpslp that we determined has one amino acid change in <br /> physical map as in Riles et al. (1993) (see Materials and        the N-terminus of the protein from the sequence reported <br /> methods). The probe hybridized to the X clone Olson <br /> #3686 (ATCC #70420) and to the overlapping cosmid                by Poch et al. (1994). This discrepancy arises from a <br /> Olson #9533 (ATCC #70974), which place this DNA in               difference in the DNA sequence at position +436 (T-&gt;G), <br /> MERG <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> 0964 on the left arm of chromosome IV in               resulting in amino acid number 146 (a serine) changing <br /> the vicinity of the mpsl-J mutation. Finally, plasmid-to-        to an alanine in our sequence. This change has been noted <br /> chromosome conversion has demonstrated that the mpsl-J           in the GenBank file (Accession number L08909) for <br /> mutation maps to the ORF identified as the MPSJ gene             MPSJ/RPKI. <br /> (discussed below), and a null allele of this ORF fails to                Comparison of the derived amino acid sequence of the <br /> complement the mpsl-J mutation. The finding that the             MPSJ gene with protein sequences in various databases <br /> recently reported gene RPKI (Poch et al., 1994) is the           revealed significant similarity between the C-terminal <br /> same as MPSJ, suggests that the linked genes PRP9 and            region of the putative MPSJ gene product and members <br /> ACT2 (Schwob and Martin, 1992) that Poch et al. (1994)           of the protein kinase superfamily. Closer examination <br /> used to find MPSJ must also be centromere linked on              revealed that amino acids 447-704 of Mpslp could be <br /> chromosome IVL.                                                  aligned with the consensus sequences for the 11 sub- <br />                                                                  domains of a protein kinase domain as defined by Hanks <br /> Characterization of the MPS1 gene                                et al. (1988), and that Mpslp contains all consensus amino <br /> The minimal complementing region of YCpMPS1 was                  acids in each subdomain (Figure 2). As noted by Poch <br /> further characterized by Southern analysis. An EcoRV             et al. (1994), the protein kinases most similar to MPSJ <br />  <br /> 1656 <br />  <br />  <br /> Fig. 2. Alignment of the kinase domains of MPSJ and of the mammalian genes PYT/tITK (Mills et al., 1992; Lindberg et al., 1993) and esk <br /> (Douville et al., 1992). The MPSJ amino acid sequence is shown, beginning at amino acid number 446. Dashes in the esk and PYT/TTK sequences <br /> indicate positions of identity with the MPSI sequence, and a period (.) in the esk sequence is a position where it is identical to PYT/TTK, but differs <br /> from MPSI. The two mammalian kinases are virtually identical to one another, exhibiting 96% identity over their entire <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span>. Approximately 45% <br /> of the residues in the MPSJ kinase domain are identical to the mammalian kinases, and 67% are similar when allowing for conservative amino acid <br /> changes. The subdomains (domains I-XI) of the kinase domain were defined by Hanks et al. (1988). The conserved amino acids in serine/threonine <br /> kinases (SER/THR) are after Lindberg et al. (1992), where the upper case amino acids are invariant, and the amino acids in lower case are nearly <br /> invariant. The italicized E in subdomain V is not part of the SER/THR consensus, but a feature of dual specificity kinases (Lindberg et al., 1992). <br /> The position of the &apos;kinase-dead&apos; mutation in subdomain VII is shown as the underlined A at the position of the D it replaced. <br />  <br /> are the dual specificity kinases esk from mice (Douville                       (Rothstein, 1991). The correct transplacement of <br /> et al., 1992) and PYT/TTK from humans (Mills et al.,                           mpslA::HIS3 into the MPSJ locus in four separate trans- <br /> 1992; Lindberg et al., 1993) (Figure 2). The product of                        formants was confirmed by a polymerase chain reaction <br /> the yeast MPSJ gene and the mammalian gene products                            (PCR) protocol (see Materials and methods). Tetrads from <br /> are similarly organized, each with the kinase domain in                        each of the four sporulated diploids gave rise to only two <br /> the C-terminal half of the protein. However, the N-terminal                    viable segregants, and all viable segregants were histidine <br /> region of MPSJ and the 60 amino acids C-terminal of the                        auxotrophs (n = 37). These results indicate that the <br /> kinase domain show no similarity to PYT/TTK, esk, or                           transformed diploids segregate a lethal mutation that is <br /> any sequence in the available databases.                                       linked to the HIS3 gene, and presumably the lethality is <br />                                                                                due to the loss of MPSJ function. The MPSJ gene on a <br /> MPS1 is an essential gene                                                      plasmid (YCpMPS 1) was found to rescue the lethal <br /> A deletion/<span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>insertion mutation</span> in the MPSJ ORF was                              mutant, and the plasmid is required for mitotic viability <br /> created to determine if MPSJ is an essential gene for                          as these cells will not grow on 5-fluoroorotic acid (5-FOA) <br /> yeast as suggested by the conditional-lethal mpsl-J allele                     plates which select against the URA3 marker on YCpMPS 1 <br /> (Winey et al., 1991). A ClaI-BamHI fragment was                                (Boeke et al., 1987). A final demonstration that the null <br /> removed from the MPSJ gene and replaced with a                                 allele disrupts the MPSJ gene involved selecting diploids <br /> ClaI-BamHI fragment carrying the yeast HIS3+ gene                              from a cell (mpsl-J, his3, LEU2, strain Wx241-12b, Table <br /> (Struhl, 1985; see Materials and methods). A plasmid                           I) to spore mating, using spores from the disruption diploid <br /> carrying the mpslA::HIS3 allele is not able to complement                      (mpslA::HIS3, his3A200, leu2-3, 112). These diploids <br /> the mpsl-1 mutation, indicating that the MPSJ gene                             (His&apos;, Leu+) where found to be temperature-sensitive for <br /> function had been disrupted by the insertion/deletion                          growth, indicating that the disruption had abolished MPSI <br /> mutant. An mpslA::HIS3-containing fragment was used                            function and allowed expression of the mpsl-J <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> to transform a diploid yeast strain (D8bx5cA, Table I) to                      in diploids with the genotype mpsl-J/mpslA::HIS3. <br /> histidine prototrophy in a one-step gene transplacement                           To determine if the putative kinase activity was import- <br />  <br /> Fig. 3. Protein kinase assays using the GST (a) and the LexA (c) fusion proteins. Affinity purification of the correct fusion proteins is shown by <br /> immunoblot analysis (b and d). GST (G) (a, lanes I and 3) or GST-MpslA2 (GM) (a, lanes 2 and 4) was affinity purified from yeast and used in <br /> kinase assays under conditions for autophosphorylation (a, lanes I and 2), or substrate phosphorylation (a, lanes 3 and 4) using MBP (see Materials <br /> and methods). Similarly, LexA-bicoid (LB) (Golemis and Brent, 1992) or LexA-MpslA63 (LM) were immunoprecipitated and assayed for <br /> autophosphorylation activity (c, lanes 1 and 2, respectively). The positions of MBP and the autophosphorylated Mpslp fusion proteins are indicated, <br /> and are shown directly by immunoblotting using anti-GST antibody (b, lane I GST, lane 2 GST-MpsIA2), or by using anti-LexA antibody (d, lane <br /> 1 LexA-bicoid, lane 2 LexA-MpslA63). <br />  <br /> ant to MPSJ function, a &apos;kinase-dead&apos; allele was con-                        tion and substrate phosphorylation activity (Figure 3a, <br /> structed using a PCR-based protocol (see Materials and                       lanes 2 and 4; and 3c, lane 2; substrate phosphorylation <br /> methods). The particular <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> (D580-&gt;A580,                         by LexA-MpsIA63 not shown). In contrast, no kinase <br /> Figure 2) was chosen on the basis of the mutagenesis of                      activity was detected in the immunoprecipitates of the <br /> the yeast cAMP kinase (Gibbs and Zoller, 1991). In that                      GST protein alone or the LexA-bicoid fusion protein <br /> gene, the mutation of D228-*A228 in subdomain VII of the                     (Figure 3a, lanes 1 and 3; and 3c, lane 1), indicating that <br /> catalytic domain (consensus sequence DFG, see Figure 2)                      the activity seen was specific to the affinity purification <br /> was the sole single amino acid change of wild-type to                        of the Mpslp fusion protein. <br /> alanine that rendered the enzyme non-functional in vitro                        The possibility exists that the protein kinase activity <br /> and the gene non-functional in vivo. Similar genetic results                 shown in Figure 3a is not intrinsic to Mpslp, but is due <br /> were found with the mpslD/A allele. The mutation was                         to a co-purifying kinase in both the LexA-MpslA63 <br /> made in the GST::MPSIA2 construct (see Materials and                         immunocomplexes and the GST-MpsIA2 complexes. <br /> methods), which normally complements both the mpsl-l                         The &apos;kinase-dead&apos; mutation (D580-&gt;A580, see Materials <br /> mutation and the mpslA::HIS3 null allele. The GST::                          and methods) was introduced into the GST::MPSJA2 <br /> mpslD/A mutant construct, however, fails to complement                       construct and the mutant protein was isolated and assayed <br /> mpsl-J and the mpsiA::HIS3 allele, indicating that protein                   as above to determine if the observed protein kinase <br /> kinase activity is required for the essential function of                    activity is intrinsic to Mpslp. Figure 4b, lane 2 shows <br /> MPSJ.                                                                        that the mutant protein can be isolated from yeast and <br />                                                                              detected by immunoblot analysis using anti-GST antibody. <br /> Mpslp has protein kinase activity                                            However, this mutant protein has no detectable protein <br /> Chimeric proteins containing nearly all of Mpslp have                        kinase activity, either in assays of autophosphorylation or <br /> been used to demonstrate that Mpslp has protein kinase                       in assays for the phosphorylation of exogenous substrate <br /> activity in vitro. The MPSJ ORF was fused to two different                   (Figure 4a, lanes 2 and 4). We conclude that the observed <br /> heterologous protein tags, GST and LexA, under the                           protein kinase activity is intrinsic to Mps lp, and that it is <br /> control of heterologous promoters for increased expression                   the sole activity found in the glutathione bead/GST- <br /> levels of the chimeric proteins in yeast (see Figure 1 and                   Mps 1A2 preparations as shown in Figures 3 and 4. <br /> Materials and methods). These fusion proteins, GST-                             The immunoblot detection of GST-MpslA2 and <br /> MpslA2 and LexA-MpsIA63, were enriched from yeast                            GST-MpsI A2D/A (kinase-dead protein) shown in Figure <br /> lysates by affinity purification on glutathione resin or by                  4b, lanes 1 and 2, indicates that the proteins have different <br /> immunoprecipitation, respectively. The material isolated                     apparent mol. wts when isolated from yeast. The inactive <br /> from yeast extracts using glutathione agarose or anti-LexA                   GST-MpslIA2D/A protein migrates at the expected mol. <br /> antibodies was analyzed by immunoblotting using anti-                        wt of 112 kDa, whereas the active GST-MpsIA2 protein <br /> GST or anti-LexA antibodies, respectively, (Figure 3b,                       migrates at a higher apparent mol. wt and the electro- <br /> lane 2; and 3d, lane 2) and proteins of slightly higher than                 phoretic band is less distinct. Increases in apparent mol. wt <br /> the expected mol. wts for the two chimeric proteins were                     and indistinct bands can be indicative of post-translational <br /> detected. These precipitated proteins were used in kinase                    modification of the polypeptide in question. The GST- <br /> assays (as described in Materials and methods) under                         Mps1A2 and GST-Mpsl1A2D/A proteins were subjected <br /> conditions for autophosphorylation or for the phosphoryla-                   to protein phosphatase treatment with protein phosphatase <br /> tion of the exogenous substrate myelin basic protein                         2A (PP2A, Materials and methods) to determine if the <br /> (MBP). Both chimeric proteins exhibited autophosphoryla-                     different mobilities of these proteins are caused by <br />  <br />  <br />           PP2A      -               -       -        +   +             Fig. 5. Analysis of in vivo 32P-labeled GST (G) (lanes and 2) and <br />           MC        -               -       -        +   -             GST-MpslA2 (M) (lanes 3 and 4) proteins. Cells were labeled and <br />                                                                        the GST proteins were affinity purified as described in Materials and <br />                                                                        methods. Glutathione agarose loaded with GST or GST-MpsIA2 was <br />                                                                        subjected to thrombin cleavage, creating two fractions; the material <br />          116 <br />                     ....... <br />                                    40b               _   0-* GM D/A    remaining on the beads (lane 1, GST; lane 3, GST-MpslA2) and the <br />                                                                        material liberated by the cleavage [lane 2, GST; lane 4, MpslA2 (M)]. <br />  <br />  <br />           66                                                           inhibitor microcystin resulted in a mobility similar to the <br />                                                                        untreated protein (Figure 4b, lanes 3 and 4). It appears <br />                                                                        that the retarded mobility of the GST-MpslA2 protein is <br />           45-                                                          largely a result of phosphorylation, and that this may be <br />                                                                        due to autophosphorylation since the inactive GST- <br />                                                                        Mps1A2D/A protein appears to lack these modifications. <br />           29                                                              We wished to demonstrate independently that Mpslp <br />                                                                        is a phosphoprotein in vivo. Yeast cells containing a <br />                                                                        plasmid-bome GST::MPSIA2 gene were labeled with <br />                                                                        [32P]orthophosphate (see Materials and methods), and <br />                                          ((GOST                        GST-MpslA2 was isolated and analyzed by gel electro- <br />                                                                        phoresis and autoradiography. We found that the GST <br /> Fig. 4. Analysis of the GST-Mps1A2D/A (kinase-dead) mutant             protein itself is phosphorlylated in yeast (Figure 5, lanes <br /> protein. GST-MpslA2 (GM) and GST-Mps1A2D/A (GM D/A)                    1 and 2), although the 32p signal observed for the significant <br /> proteins were affinity purified from yeast and used in assays for      amount of protein present is quite low. GST-MpslA2 <br /> autophosphorylation and substrate phosphorylation (a), and <br /> immunoblotting with anti-GST antibodies (b, see Materials and          contains a thrombin cleavage site (Miller et al., 1993) that <br /> methods). Lanes 1 and 3 in (a) and lane 1 in (b) were loaded with      allows the cleavage of the Mps 1A2 moiety from GST. <br /> affinity-purified GST-MpslA2, and lanes 2 and 4 in (a) and lane 2 in   The thrombin cleavage of GST-MpslA2 isolated from <br /> (b) were loaded with affinity-purified GST-Mps1A2D/A. (b) also         32P-labeled cells was carried out as described by Miller <br /> shows immunoblot detection of phosphatase-treated GST-MpslA2 <br /> using anti-GST antibodies (lanes 3-5). The proteins were incubated     et al. (1993) and the Mpslp moiety from the chimeric <br /> with nothing (b, lane 3), the inhibitor microcystin (MC) and PP2A      protein was released from the GST protein and found to <br /> (b, lane 4), or PP2A alone (b, lane 5) as described in Materials and   be a phosphoprotein as well (Figure 5, lane 4). In this <br /> methods.                                                               experiment, 32P-labeled GST is not detected (Figure 5, <br />                                                                        lane 3), either because GST is not phosphorylated in the <br /> phosphorylation. The mobility of the inactive GST-                     chimeric protein or because the level of phosphorylated <br /> Mps1A2D/A protein is unaffected by phosphatase treat-                  GST is too low to detect. Consistent with the apparent <br /> ment (data not shown), whereas Figure 4b shows that                    phosphorylation of GST-MpslA2 as detected by phos- <br /> PP2A treatment of the active GST-MpslA2 protein does                   phatase treatment, we have found this protein to be a <br /> alter its mobility (lane 5), such that it has the same apparent        phosphoprotein in vivo. <br /> mol. wt as the inactive GST-MpsIA2D/A protein. In                         Finally, the amino acid specificity of the Mpslp protein <br /> addition, the shift in mobility appears to be due to                   kinase was determined by phosphoamino acid analysis of <br /> phosphatase activity alone, as the addition of the PP2A                in vitro labeled GST-MpslA2 from autophosphorylation <br />  <br />                                                                                                                                       1659 <br />  E.Lauze et al. <br />  <br />                                                                       Mps Ip itself, have not been identified and mutations <br />                                                                       of the phosphorylation sites on Mps Ip have not been <br />                                                                       examined. <br />                                                                          The MPSJ gene has recently been independently <br />                                                                       reported by Poch et al. (1994). These workers also report <br />                                                                       that MPSJ is an essential gene using null alleles distinct <br />                                                                       from the one reported here. Poch et al. (1994) examined <br />                                                                       the transcript levels of MPSJ mRNA and found that the <br />                                                                       mRNA levels were constant in mitotic cells. Interestingly, <br />                                                                       they report that MPSI mRNA levels fall in mating factor- <br />                                                                       treated cells. We have previously reported that MPS] is <br />                                                                       required for the completion of SPB duplication upon <br />                                                                       release from mating factor arrest (Winey et al., 1991). <br />                                                                       The reduction in MPSJ mRNA levels in cells exposed to <br />                                                                       mating factor may be a mechanism to block SPB duplica- <br />                                                                       tion at the appropriate stage in mating cells. Synchroniza- <br />                                                                       tion of SPB duplication in the mating partners is required <br />                                                                       for efficient karyogamy, which is initiated by the SPBs <br />                                                                       (Byers and Goetsch, 1975; Rose et al., 1986). Poch et al. <br />                                                                       (1994) also report an increase in MPS] mRNA levels in <br />                                                                       meiotic cells. We have observed that a/a mps1-limps]-] <br />                                                                       strains fail to form spores at the non-permissive tempera- <br />                                                                       ture of 30&#194;&#176;C (M.Winey, unpublished results), indicating <br />                                                                       that MPSJ is required for meiosis. <br /> Fig. 6. Phosphoamino acid analysis of in vitro labeled MBP (a), and      Poch et al. (1994) constructed a depletion allele of <br /> GST-MpslA2 (b) as described in Materials and methods. The             MPS1, where the MPS] gene was placed under the control <br /> positions of phosphoserine (pS), phospothreonine (pT) and             of the GALIO promoter. Yeast cells containing this allele <br /> phosphotyrosine (pY) are indicated and were determined by the         could be cultured on galactose-containing media, but <br /> migration of unlabeled markers, as described by Kamps and Sefton      perished when transferred to glucose-containing media. <br /> (1989). <br />                                                                       Cells dying upon transfer to glucose-containing media <br />                                                                       were analyzed by flow cytometry and DAPI staining of <br /> reactions and from in vitro labeled MBP (as in Figures 3              DNA in whole cells. The authors observed failure to arrest <br /> and 4). Figure 6a shows that phosphoserine and phospho-               in the cell cycle, and cells with aberrant DNA contents. <br /> threonine residues were detected in in vitro labeled MBP,             They interpret these results as the cells failing to enter S <br /> and that the major signal is from phosphothreonine. Similar           phase, but carrying out mitosis. In contrast, we have <br /> analysis of autophosphorylated GST-MpsIA2 showed a                    reported that the mpsl-J mutants, when released from x <br /> phosphotyrosine signal as well, although phosphothreonine             factor arrest at the non-permissive temperature, complete <br /> and phosphoserine are the predominant signals (Figure                 DNA synthesis and then missegregate their chromosomes <br /> 6b). It is clear, however, that Mps 1 p will autophosphorylate        (Winey et al., 1991), yielding a flow cytometric profile of <br /> on all three hydroxy amino acids, indicating that Mpslp               DNA content reminiscent of the MPSJ depletion experi- <br /> can function as a dual specificity protein kinase in vitro.           ment (Poch et al., 1994). We propose that the MPSJ <br />                                                                       depletion allele reported by Poch et al. (1994) gives rise <br /> Discussion                                                            to failure in SPB duplication and lack of cell cycle arrest <br />                                                                       in response to the defective spindle, identical to the <br /> We have described the isolation and characterization of               conditional alleles. Poch et al. (1994) recognized that <br /> the S.cerevisiae MPSJ gene. This gene was previously                  failure of the cells to arrest and their continued aberrant <br /> identified by a mutant allele that causes defects in SPB              cell cycles were indicative of a defect in a checkpoint <br /> duplication and cell cycle regulation (Winey et al., 1991).           control. We concur, and have demonstrated that MPSJ is <br /> We have demonstrated that this gene encodes an essential,             required for a checkpoint that monitors some aspect of <br /> dual specificity protein kinase that is a phosphoprotein              SPB duplication or function (E.Weiss and M.Winey, in <br /> in vivo. The MPSI-encoded protein is 764 amino acids                  preparation). <br /> long, wherein the kinase domain is located in the                        Protein kinases have been implicated in the regulation <br /> C-terminal region of the protein and the N-terminal region            of many cellular events, and the behavior of centrosomes <br /> is unique. Mps p kinase activity is similar to several other          is no exception. While no kinase besides MPSJ has been <br /> dual specificity kinases, in that phosphorylation of tyrosine         directly implicated in spindle pole duplication, phos- <br /> can be detected by autophosphorylation, and the level of              phorylation of centrosome components has been correlated <br /> tyrosine phosphorylation is modest (Lindberg et al., 1992).           with in vitro assembly (Felix et al., 1994). Furthermore, <br /> Furthermore, the Mpslp kinase favors threonine phos-                  protein kinases have been implicated in modification <br /> phorylation over serine phosphorylation on the exogenous              of centrosome activity. A variety of experiments have <br /> substrate MBP, which has also been observed for the                   demonstrated that the in vitro microtubule nucleation <br /> mammalian esk kinase (Douville et al., 1992). The sig-                capacity of isolated centrosomes is decreased by kinase <br /> nificance of these patterns of phosphorylation by Mpslp               inhibitors or by treatment with the MPM2 antibody, which <br /> is unknown, because in vivo substrates of the kinase, save            recognizes an M phase-specific phosphorylated epitope, <br />  <br /> 1660 <br />                                                                                                                               MPS1 protein kinase <br />  <br /> and increased by treatment with cyclin-associated protein                   agarose was washed extensively in buffer B plus phosphatase inhibitors. <br /> kinase complexes (Centonze and Borisy, 1990; Buendia                        Thrombin cleavage was carried out as described by Miller et al. <br />                                                                             (1993). Post-cleavage supernatants were pooled and acetone precipitated <br /> et al., 1992; Ohta et al., 1993). As suggested by these                     (Ausubel et al., 1994), and subjected to SDS-PAGE analysis and <br /> results, centrosomes do contain phosphoproteins (Kalt and                   autoradiography. <br /> Schliwa, 1993). Furthermore, the level of phosphorylation <br /> at the centrosome, as detected by MPM2 antibody staining,                   Nucleic acid techniques <br /> does change during the cell cycle, peaking during mitosis.                   DNA was manipulated by -standard techniques as described in Ausubel <br />                                                                              et al. (1994). DNA was prepared from E.coli using Wizard kits (Promega), <br /> This phenomenon has been observed in both mammalian                          and from yeast by the method of Hoffman and Winston (1987). Sequence <br /> cells (Centonze and Borisy, 1990) and in S.pombe (Masuda                     analysis was accomplished by using fragments of the MPSJ gene <br /> et al., 1992). Finally, it has been suggested that centro-                   introduced in M13 mpl8 or mpl9 (Messing, 1983) or by using custom <br /> somes may have protein kinases as components, but their                      oligonucleotides as primers (DNA Express, Fort Collins CO). DNA <br /> role is unknown (Kalt and Schliwa, 1993). To date, the                       sequencing was carried out using the Sequenase 2.0 kit designed for this <br />                                                                              purpose (United States Biochemical) following the instructions provided <br /> localization of Mpslp in the yeast cell is unknown.                         by the supplier. Analysis of the DNA sequence of MPSJ and its derived <br />    The mammalian dual specificity protein kinases, esk                       amino acid sequence was carried out using programs in the GCG <br /> and PYT/ITK, have the most amino acid sequence homo-                         software package from the Genetics Computer Group, Madison, WI. <br />                                                                                 DNA hybridization analysis utilized a labeled EcoRV fragment internal <br /> logy to the MPSI-encoded protein, but that similarity is                    to the ORF of the MPSJ gene. DNA was labeled by random primed <br /> limited to the kinase domain. The mammalian genes were                       incorporation of digoxigenin (DIG)-labeled deoxyuridine triphosphate, <br /> thought to be involved in cell cycle control because their                  using the Genius Nonradioactive Nucleic Acid Detection System <br /> expression is limited to proliferating cells (both stem cells               (Boehringer Mannheim Biochemicals) and following the labeling proto- <br />                                                                             col provided by the supplier. The DIG-labeled probe was hybridized to <br /> and transformed cells) that are actively going through cell <br />                                                                             the Olson collection of overlapping genomic clones (Riles et al., 1993; <br /> cycles (Douville et al., 1992; Mills et al., 1992). Consistent              American Type Culture Collection, Rockville Pike, MD) in 5 x SSC, 50% <br /> with this idea, more recent data from Hogg et al. (1994)                    deionized formamide, 5% (w/v) Genius blocking reagent (Boehringer <br /> and Schmandt et al. (1994) have shown cell cycle                            Mannheim), 0.1 % N-lauroylsarcosine (sodium salt), 0.02% SDS at 420C <br /> periodicity for the expression of the PYTFFTK gene and                      for 16 h. An anti-DIG antibody conjugated to alkaline phosphatase <br /> for the level of kinase activity. While these fluctuations                  (Boehringer Mannheim) was bound to hybridized DIG-labeled DNA, <br />                                                                             and the chemiluminescent alkaline phosphatase substrate Lumiphos <br /> in PYT/TTK activity suggest some cell cycle role for this                   (Boehringer Mannheim) was applied to the filters at alkaline pH. Light <br /> kinase, its function is not known. In our initial attempts to               emission was then detected by exposure of filters to X-ray film. <br /> determine if MPSJ and PYTMK are functional homologs,                            A PCR protocol was used to confirm proper integration of the MPSJ <br /> chimeric genes (the N-terminus of Mpslp with the PYT/                       null allele (mpslA::HIS3, see Figure 1) into the genome of a diploid <br /> TTK catalytic domain) have failed to rescue mpsl-J                          yeast recipient strain (D8bx5cA, Table I). Reactions were carried out <br />                                                                             under standard conditions using Taq polymerase (Promega) in a Perkin- <br /> mutant cells at the restrictive temperature. (J.Bachant and                 Elmer 480 Thermal cycler with priming oligos synthesized by DNA- <br /> M.Winey, unpublished results).                                              Express (Fort Collins, CO). Three primers were used together in the <br />    We will need to understand the regulation of MPSJ                        analysis. One (BKS1, 5&apos;-GAATC-TTTTCATTATTG-3&apos;) binds 3&apos; of the <br />                                                                             site of integration and should prime both chromosome IV homologs in <br /> activity and identify the proteins that interact with Mpslp                 the transformants. A second oligonucleotide (BS02, 5&apos;-CGATGAC- <br /> to better understand the role of this gene in SPB duplication               TTTGACAAA-3&apos;), specific for the region of MPSJ removed in <br /> and in cell cycle control. Regulation of Mpslp by phos-                     mpslA::HIS3, was used to detect the non-recombinant chromosome, <br /> phorylation seems to be a reasonable possibility since                      yielding a 1542 nucleotide fragment with the above common primer <br /> GST-MpslA2 is a phosphoprotein in vivo. Regulation of                       (BKS1). The mpslA::HIS3 allele was detected by an oligonucleotide <br /> mRNA is also a possibility, since mRNA abundance is                         specific for the HIS3 sequence (HIS3A, 5&apos;-GATTAGCGACCAGCCGG- <br />                                                                             3&apos;) and, in correct integrants, oligos HIS3A and the common oligonucleo- <br /> altered in certain parts of the yeast life cycle (Poch et al.,              tide (BKSI) yield a band of 1401 nucleotides. PCR was performed on <br /> 1994). Identification of gene products that interact with                   four independent His&apos; transformants with the mpslA::HIS3 allele, and <br /> Mpslp will be crucial to understanding the role of this                     all four yielded the two expected bands. Tetrad analysis of all four <br /> dual specificity protein kinase in SPB duplication and in                   diploids showed that a recessive lethal mutation was segregating and <br />                                                                             was linked to HIS3. Cells from the viable spore clones were analyzed <br /> cell cycle control.                                                         by PCR and found to contain only the 1542 nucleotide band indicative <br />                                                                             of the normal MPSJ gene. <br />                                                                                 PCR was also used to create a point mutation in the kinase domain <br /> Materials and methods                                                       of the MPSJ gene. PCR was carried out as described above, but plasmid <br />                                                                             DNA containing the MPSJ gene was used as the template for mutagenesis. <br /> Strains, cell culture and genetic techniques                                A 42 residue oligonucleotide (MPSC, 5&apos;-CGGTACCGCGTTTGC- <br /> The yeast strains used in this study are listed in Table I. Yeast media     TATACCAAAAGCAATGATTTTTAA-3&apos;, the bold nucleotide being the <br /> and genetic and molecular techniques were as described by Ausubel           mutation) was synthesized that incorporated the KpnI site in the kinase <br /> et al. (1994). Escherichia coli DH5a (Sikorski and Hieter, 1989) was        domain and the point mutation (nucleotide A1739-*C]739 changing a <br /> cultured and transformed as described by Ausubel et al. (1994). Yeast       D580 to an A580, numbering after Poch et al., 1994) in this primer. A <br /> 5-FOA plates were prepared and used as described by Boeke et al. ( 1987).   second primer (MPSG, 5&apos;-CATTGGATGATGCAATAA-3&apos;) was used so <br />    For in vivo labeling, yeast strains containing the GST::MPSJA2 fusion    the PCR product could be cleaved with XbaI and KpnI for re-introduction <br /> construct (described below) and GST alone were grown to 0.4 OD600/ml        into the MPSJ gene (see Figure 1). Once cloned, the DNA sequence of <br /> in uracil-minus, 2% raffinose medium at 30&apos;C. Transcription was induced     the entire XbaI-KpnI fragment in GST::mpsJA2D/A was determined to <br /> by addition of galactose to 4% final concentration (Miller et al., 1993).   confirm that only the intended mutation had been introduced into the gene. <br /> Cells were induced for 5 h at 30&#194;&#176;C. Cells were then starved for phosphate <br /> in uracil-minus, 4% galactose, phosphate-depleted medium (Warner,           Plasmid constructs <br /> 1991) for 1 h. 2.5 mCi of [32Plorthophosphate were then added to the        The original isolate of MPSI in YCp5O is shown in Figure 1. An <br /> cells and allowed to incorporate for 2 h. The cells were collected and      integrative plasmid containing the MPSJ region was constructed by <br /> resuspended in buffer B (described below) plus phosphatase inhibitors       inserting the EcoRI-SaIl fragment of MPSI into pRS306 (Sikorski and <br /> (10 mM NaF and 10 mM sodium orthovanadate). Following glass-bead            Hieter, 1989). The integration of this plasmid into yeast chromosomal <br /> lysis with 2 vol of acid-washed glass beads (0.45-0.52 mm, Thomas           DNA was promoted by linearizing the plasmid in the MPSJ gene. <br /> Scientific), the extracts were incubated with glutathione agarose (100 ml   The null allele, mpslA::HIS3, was constructed as an insertion/deletion <br /> agarose suspended at 50% v/v in buffer B; Sigma) for 1 h at 4&apos;C. The        mutation. The ClaI-BamHI fragment of MPSJ (internal to the ORF, <br />  <br />                                                                                                                                               1661 <br />  E.Lauze et al. <br />  <br /> Figure 1) was replaced with the 1692 bp ClaI-BamHI fragment                       Phosphoamino acid analysis was performed as described in Kamps <br /> containing the HIS3+ genes (Struhl, 1985). This allele was excised from        and Sefton (1989) and Cooper et al. (1983) on material from in vitro <br /> the plasmid in which it was constructed as an EcoRI-KpnI fragment              kinase assays. The kinase assay reactions were run on 12% SDS -PAGE <br /> for transformation in a one-step gene transplacement (Rothstein, 1991).        and transferred to Immobilon PVDF, as described above. Following <br /> Gene transplacements were confirmed by the PCR protocol described              exposure of the membrane to film, the bands of interest (autophosphoryl- <br /> above.                                                                         ated Mpslp fusion proteins and phosphorylated MBP) were excised and <br />    Two plasmids were constructed for affinity purification of the MPSJ         eluted from the membrane (Cooper etal., 1983; Kamps and Sefton, 1989). <br /> gene product from yeast. A fusion of MPSJ to the LexA gene of Ecoli               Affinity-purified GST-MpsIA2 and GST-MpslA2D/A protein was <br /> was accomplished by inserting the ClaI-SalI MPSJ fragment, after               phosphatase treated using protein phosphatase 2A (a gift from Dr Natalie <br /> converting the ClaI site to an EcoRI by the addition of linkers (New           Ahn) while still conjugated to the glutathione agarose (in non-detergent <br /> England Biolabs), into the EcoRI and Sail sites of the plasmid pEG202          buffer, see above). PP2A was added to the proteins at a final concentration <br /> (Golemis and Brent, 1992). This chimeric gene (LexA::MPS1A63) is               of 500 U/ml and incubated at 30&#194;&#176;C for 1 h. The phosphatase inhibitor <br /> under the control of the constitutive ADHI promoter, and deletes the           microcystin was added to identical phosphatase reactions at a final <br /> first 63 amino acids of the MPSJ ORF. A second fusion of the MPSJ              concentration of 2 jM. Following the incubation, the glutathione agarose <br /> gene was made to the Schistosoma japonicum GST gene under the                  was needle-aspirated dry, resuspended in 20 ,ul of 2&gt;x Laemmli buffer, <br /> control of the inducible GAL] promoter in the pEG(KT) vector described         boiled for 5 min and separated by SDS-PAGE. <br /> by Miller et al. (1993). In this construct (GST::MPSJA2), an -3 kb <br /> Hincll fragment containing MPSJ was inserted into the PvuII site of <br /> pEG(KT), resulting in removal of only the first two amino acids of the          <br /> MPSJ ORF. <br />                                                                                 <br /> Protein techniques and kinase assays                                           <br /> Immunoblot analysis of protein samples separated by 12% SDS-PAGE                <br /> (Ausubel et al., 1994) involved transferring the proteins to Immobilon         <br /> PVDF membrane (Millipore) for 2 h at 0.75 A in transfer buffer (20%             <br /> MeOH, 0.1% SDS, 5 mM Tris, 0.65 M glycine) at 4&#194;&#176;C using a Hoefer               <br /> apparatus. Membranes were incubated in blocking buffer (100 mM                 <br /> Tris-HCl pH 7.5, 0.9% NaCl, 0.1% Tween 20, 2% BSA) for 30 min at               <br /> room temperature, or overnight at 4&#194;&#176;C. Membranes were then washed             <br /> extensively in TBST (100 mM Tris-HCl pH 7.5, 0.9% NaCl, 0.1%                   <br /> Tween 20) and subsequently incubated in the primary antibody for 1 h        <br /> at room temperature. For detection of GST-MpslA2p, the primary                 <br /> antibody was used at 1:500 (Pharmacia goat anti-GST). Rabbit polyclonal        <br /> anti-LexA antibody (a gift from Dr Roger Brent) was used at a 1:1000           <br /> dilution. After washing in TBST, the membranes were then incubated             <br /> in secondary antibody for 1 h at room temperature. Secondary antibodies        <br /> were used at a 1:20 000 dilution for detection using the ECL chemi- <br /> luminescence system (Amersham). <br />    Yeast strain EGY48 (Table I) transformed with the LexA::MPSJA63              <br /> plasmid was grown in histidine-minus selective medium to 0.2-0.5 <br /> OD600/ml. Cells were collected by centrifugation, rinsed twice in ice-         <br /> cold sterile distilled water, and disrupted by vortex mixing 10 times      <br /> (30 s vortex, 30 s on ice) in 0.5 ml buffer B (50 mM Tris-HCl, pH 7.5;         <br /> 50 mM NaCl; 0.2% Triton X-100; leupeptin, 5 mg/ml; soybean trypsin             <br /> inhibitor, 5 mg/ml; aprotinin, 5 mg/ml; PMSF, 1 mM) with 2 vol of              <br /> 0.45-0.52 mm glass beads (Thomas Scientific). The supernatant from               <br /> the lysed cells was incubated with anti-LexA antibody prebound to             <br /> protein A-Sepharose (Pharmacia) for 2 h at 4&#194;&#176;C. The Sepharose was                  <br /> then washed in buffer B three times, buffer Bi [50 mM Tris-HCl                 <br /> (pH 7.5), 50 mM NaCl, 0.1% Triton X-100] three times, buffer A                  <br /> [50 mM Tris-HCl (pH 7.5), 50 mM NaCl, 0.02% Triton X-100] three                <br /> times, and non-detergent buffer [50 mM Tris-HCl (pH 7.5), 50 mM                   <br /> NaCI] once, and used in kinase assays (described below) or for                 <br /> immunoblot analysis.                                                              <br />    The yeast strain BJ2168 (Table I) containing the GST::MPSJA2 or                 <br /> GST::mpsJA2D/A construct was grown to 0.4 OD600/ml in selective                 <br /> medium (uracil-minus, 2% raffinose) at 30&#194;&#176;C. The cells were then                  <br /> induced to express the chimeric genes by the addition of galactose              <br /> (Sigma) to 4% final concentration. Following 6 h of induction, the cells      <br /> were collected by centrifugation, rinsed twice in ice-cold sterile distilled    <br /> water and glass-bead beaten as described above. The supematant from            <br /> the lysis was incubated with glutathione agarose (200 ml agarose                <br /> suspended at 50% v/v in buffer B, Sigma) for 1 h at 4&#194;&#176;C. For use in            <br /> kinase assays or for imumunoblot analysis, the beads were washed three         <br /> times each in buffer B, buffer B1 and buffer A sequentially, and once             <br /> in non-detergent buffer.                                                       <br />    The affinity-enriched proteins were incubated in 25 gl kinase assay          <br /> buffer minus ATP [KAB-ATP: 50 mM Tris pH 7.5, 10 mM MgCI2,                      <br /> 0.5 mM dithiothreitol (DTT), 1 mM PMSF] for 5 min at 30&#194;&#176;C. The                     <br /> buffer was removed, and 25 ,l of kinase assay buffer was added. The             <br /> assay buffers were: KAB-autophosphorylation (50 mM Tris pH 7.5,                    <br /> 10 mM MgCl2, 2 mM MnCl2, 0.5 mM DTT, 1 mM PMSF, 10 ,uM ATP,                    <br /> 10 ,uCi [y-,2P]ATP) and KAB-substrate phosphorylation [50 mM Tris               <br /> pH 7.5, 10 mM MgCl2, 0.5 mM DTT, 1 mM PMSF, 10 ,uM ATP, 10 ,uCi                   <br /> [y-32P]ATP, 4 ,ig MBP (Sigma)]. Reactions were incubated at 30&#194;&#176;C for           <br /> 30 min and then terminated by the addition of 6.5 jl of Sx Laemmli             <br /> sample buffer (Ausubel et al., 1994) and boiled for 5 min.                     <br /> </body></html>