<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Biomol NMR. Author manuscript; available PMC 2013 September 04. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Biomol NMR. 2004 May ; 29(1): 101&#226;&#8364;&#8220;102. doi:10.1023/B:JNMR.0000019469.77977.72. <br />  <br />  <br />  <br />                            Letter Editor: NMR assignment hypothetical protein <br />                            HI0004 Haemophilus influenzae &#226;&#8364;&#8220; putative essential gene <br />                            product <br />                            Deok Cheon Yeh, James F. Parsons, Lisa M. Parsons, Fang Liu, Edward Eisenstein, and <br />                            John Orban* <br />                            Center Advanced Research Biotechnology, University Maryland, Biotechnology Institute, <br />                            9600 Gudelsky Drive, Rockville, MD 20850, U.S.A. <br />  <br />  <br />                            Keywords <br />                                 Haemophilus influenzae; NMR assignment; structural genomics <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Biological context <br />                                            Haemophilus influenzae gram negative bacterium known cause meningitis, <br />                                            sepsis, upper respiratory infections, inner ear infections, particularly small children. <br />                                            Its 1.83 Mb genome relatively small, consisting approximately 1700 open reading <br />                                            frames, free-living organism genome sequenced entirely <br />                                            (Fleischmann et al., 1995). At time genome sequence released, approximately <br />                                            32% gene products unknown poorly understood functions. A significant <br />                                            fraction sequence homologues prokaryotes, archaea, eukaryotes, and <br />                                            classified hypothetical proteins. In effort better understand the <br />                                            biochemical functions hypothetical proteins, involved concerted <br />                                            program solve dimensional structures using X-ray crystallography NMR <br />                                            spectroscopy (Eisenstein et al., 2000). A summary current project status found <br />                                            http://s2f.umbi.umd.edu <br />  <br />                                            One protein 154 residue HI0004 150 sequence <br />                                            homologues organisms known function three-dimensional structure. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Mutagenesis experiments indicate putative essential role 259 gene products of <br />                                            unknown function Haemophilus influenzae HI0004 set, making an <br />                                            important target structural functional annotation (Akerley et al., 2002). Further, the <br />                                            HI0004 family sequence relatives contains human homologue, C21orf57, a <br />                                            putative transcript located chromosome 21 near Down syndrome locus (Reymond et <br />                                            al., 2001). Structure determination HI0004 allow homology modeling a <br />                                            large number sequences organisms provide basis understanding of <br />                                            biochemical functions molecular level. We report 1H, 13C, 15N <br />                                            chemical shift assignments HI0004, providing step detailed structure <br />                                            calculations solution. <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2004 Kluwer Academic Publishers. Printed Netherlands. <br />                            * <br />                              <span id='am-1' about='protege:TO' typeof='owl:Thing'>To</span> correspondence addressed. orban@umbi.umd.edu. <br />                             Yeh et al.                                                                                                            Page 2 <br />  <br />  <br />                               Methods experiments <br />                                         The native HI0004 gene cloned pET-28a vector using standard procedures and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         transformed Escherichia coli BL21 (DE3) cells (Novagen). Uniformly 13C/15N-labeled <br />                                         samples HI0004 prepared growing cells minimal media 15NH4Cl <br />                                         13C6-glucose sole nitrogen carbon sources, respectively. The media were <br />                                         supplemented ZnCl2 (2 mg/L), MnSO4 (1 mg/L), CoCl2 (0.1 mg/L), CuSO4 <br />                                         (0.1 mg/L) thiamine (1 mg/L). E. coli cells grown A600 0.8 protein <br />                                         expression induced 1 mM IPTG. After additional 3 h, cells harvested, <br />                                         suspended lysis buffer (0.1 M Tris, 1 mM EDTA, 1 mM DTT, pH 8.0), lysed by <br />                                         sonication. The lysed cells centrifuged ammonium sulfate added the <br />                                         supernatant 40% saturation level. The resulting precipitate removed by <br />                                         centrifugation ammonium sulfate added supernatant 70% <br />                                         saturation level. The precipitate collected centrifugation, dissolved lysis buffer, <br />                                         dialysed solution 25 mM HEPES, 1 mM EDTA, 1 mM DTT pH 7.2. <br />                                         The desalted solution loaded Poros HQ-50 anion exchange column (Applied <br />                                         Biosystems) eluted 0&#226;&#8364;&#8220;800 mM NaCl gradient. The homogeneity individual <br />                                         fractions determined SDS-PAGE 102 containing pure HI0004 were <br />                                         combined, dialysed NMR buffer (50 mMpotassium phosphate, 100 mM NaCl, 0.5 <br />                                         mM EDTA, 1 mM DTT, pH 7.0) concentrated. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Both 15N- 15N/13C-labeled NMR samples HI0004 prepared concentrations of <br />                                         approximately 1.5 mM 450 &#194;&#181;L NMR buffer containing 90% H2O/10% D2O. The <br />                                         following experiments acquired: two-dimensional 15N-HSQC,13C-HSQC, and <br />                                         (HB)CB(CGCD)HD/(HB)CB(CGCD)HE, three-dimensionalHNCACB, <br />                                         CBCA(CO)NH, HNCO, HNCA, HBHA(CO)NH, H(CCO)NH-TOCSY, (H)C (CO)NH- <br />                                         TOCSY, HCCH-TOCSY, HCCH-COSY, 15NNOESY HSQC, 13C-NOESY HSQC. In <br />                                         addition, constant time 13C-HSQC spectrum recorded 15% uniformly 13C- <br />                                         labeled sample (1.0 mM) determine stereospecific assignments methyl groups Leu <br />                                         Val residues (Neri et al., 1989). The methyl groups 12 14 Val residues all <br />                                         17 Leu residues stereospecifically assigned way. With exception 15N- <br />                                         NOESY HSQC, NMR spectra collected 7 day period NMR Facility at <br />                                         Madison (NMRFAM, Wisconsin) Bruker DRX-500 equipped z-gradient triple <br />                                         resonance cryoprobe. All experiments recorded 298K. Spectra processed using <br />                                         nmrPipe (Delaglio et al., 1995) analyzed Sparky (Goddard Kneller, UCSF). <br />  <br />                               Extent assignments data deposition <br />                                         Sequence specific backbone assignments 143 145 possible main chain <br />                                         crosspeaks 15N-HSQC spectrum HI0004 (154 residues minus 8 prolines N- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         terminal Met) illustrated Figure 1. Residues backbone HN/N assignments <br />                                         G2 M133. The che <br /> </body></html>