<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                               GBE <br />  <br /> Correlating Traits Gene Retention, Sequence <br /> Divergence, Duplicability Essentiality Vertebrates, <br /> Arthropods, Fungi <br /> Robert M. Waterhouse*,1,2, Evgeny M. Zdobnov1,2,3, Evgenia V. Kriventseva*,1,2 <br /> 1 <br />  Department Genetic Medicine Development, University Geneva Medical School, Geneva, Switzerland <br /> 2 <br />  Swiss Institute Bioinformatics, Geneva, Switzerland <br /> 3 <br />  Imperial College London, South Kensington Campus, London, United Kingdom <br /> *Corresponding author: E-mail: robert.waterhouse@unige.ch; evgenia.kriventseva@isb-sib.ch. <br /> Accepted: 4 December 2010 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> Delineating ancestral gene relations large set sequenced eukaryotic genomes allowed rigorously examine <br /> links evolutionary functional traits. We classified 86% 1.36 million protein-coding genes 40 <br /> vertebrates, 23 arthropods, 32 fungi orthologous groups linked 90% Gene Ontology or <br /> InterPro annotations. Quantifying properties ortholog phyletic retention, copy-number variation, sequence <br /> conservation, examined correlations gene essentiality functional traits. More half vertebrate, <br /> arthropod, fungal orthologs universally present lineage. These universal orthologs preferentially <br /> distributed groups single-copy multicopy genes, sequence evolution predominantly single- <br /> copy orthologous groups markedly constrained. Essential genes representative model organisms, Mus <br /> musculus, Drosophila melanogaster, Saccharomyces cerevisiae, significantly enriched universal orthologs within <br /> each lineage, essential-gene-containing groups consistently exhibit greater sequence conservation without. <br /> This study eukaryotic gene repertoire evolution identifies shared fundamental principles highlights lineage-specific <br /> features, confirms essential genes highly retained conclusively supports &#226;&#8364;&#732;&#226;&#8364;&#732;<span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>knockout</span>-rate prediction&#226;&#8364;&#8482;&#226;&#8364;&#8482; of <br /> stronger constraints essential gene sequence evolution. However, distinction sequence conservation of <br /> single- versus multicopy orthologs quantitatively prominent orthologous groups without <br /> essential genes. The previously underappreciated difference tolerance gene duplications contrasting <br /> evolutionary modes &#226;&#8364;&#732;&#226;&#8364;&#732;single-copy control&#226;&#8364;&#8482;&#226;&#8364;&#8482; versus &#226;&#8364;&#732;&#226;&#8364;&#732;multicopy license&#226;&#8364;&#8482;&#226;&#8364;&#8482; reflect major evolutionary mechanism that <br /> allows extended exploration gene sequence space. <br /> Key words: orthologs, essential genes, molecular evolution, vertebrates, arthropods, fungi. <br />  <br />  <br /> Introduction                                                                        brate, arthropod, fungal lineages offer unprecedented <br />                                                                                     opportunities comprehensively catalog gene genealogies <br /> Proteins constitute major cellular machinery are <br />                                                                                     relate increasingly detailed characteriza- <br /> inherited genes encoded genomic DNA, the <br />                                                                                     tions eukaryotic gene function. <br /> continual evolutionary processes gene duplications, los- <br />                                                                                         Comparative sequence analysis allows identification <br /> ses, sequence mutations alter repertoire, abun- <br />                                                                                     ancestral relations genes, is, homology. With <br /> dance, sequence identity. Taking advantage the <br />                                                                                     reference specific species radiation, homologous rela- <br /> availability genetic blueprints numerous eukaryotic <br />                                                                                     tions define orthologs, is, genes arose vertical <br /> species, set explore trends protein-coding <br />                                                                                     decent single gene common ancestor <br /> gene repertoire evolution sampled lineages <br />                                                                                     (Fitch 1970; Koonin 2005). Gene duplications descen- <br /> in large-scale consistent manner. With total 95 <br />                                                                                     dent lineages, referred inparalogs, co- <br /> selected species spanning hundreds millions of <br />                                                                                     orthologs comprise orthologous group descended <br /> years evolution (Hedges Kumar 2009), verte- <br />  <br />  <br /> &#194;&#170; The Author(s) 2010. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/ <br /> 2.5), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br /> Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010                                                               75 <br />  Waterhouse et al.                                                                                                  GBE <br />  <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> gene common ancestor. Delineation           sampling mouse gene knockouts did identify impact <br /> orthologs given lineage defines core       gene essentiality rate protein sequence evolu- <br /> set genes common ancestor. The conservation        tion (Liao et al. 2006). In Saccharomyces cerevisiae, gene <br /> of protein sequence identities orthologous group            evolutionary rates negatively correlated adverse <br /> members indicative strength selection inferred       effects knockouts fitness parallel growth assays <br /> by rate gene sequence divergence. Selection intensity      comparing required maximal growth dis- <br /> on gene retention deduced phyletic distribu-          pensable genes failed identify significant difference <br /> tions member genes; presence absence           (Hirsh Fraser 2001). Saccharomyces cerevisiae&#226;&#8364;&#8220;Candida <br /> considered species. Constraints gene duplicability         albicans comparisons evolutionary rates did cor- <br /> be manifested gene copy-number variations indepen-          relate dispensability, however, true du- <br /> dently evolving lineages. Detailing ancestral gene relations      plicated genes (those within-species homologs) not <br /> therefore enables quantification properties ortholog        singletons (unique genes, within-species homologs) <br /> sequence divergence, phyletic retention, copy-number          (Yang et al. 2003). For duplicates singletons, Cae- <br /> variation. Although orthologous relations defined         norhabditis elegans RNAi data suggested amino acid <br /> by gene function, identifying &#226;&#8364;&#732;&#226;&#8364;&#732;equivalent&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes mod-        replacement levels lower essential <br /> ern species provides working hypothesis         genes (Castillo-Davis Hartl 2003). Ignoring duplications <br /> similar functionality, especially single-copy orthologs.      selecting ortholog compared <br /> As such, confirmed putative functional annotations            eukaryotic species estimate evolutionary rates, these <br /> of orthologous group members&#226;&#8364;&#8221;including gene ontolo-               RNAi data provided evidence sequence evolution in- <br /> gies, protein domains, gene essentiality&#226;&#8364;&#8221;define puta-         dispensable proteins constrained selection (Luz and <br /> tive biological features characterizing group whole.     Vingron 2006). Dispensability correlated evo- <br /> The question formulated link func-       lutionary rates comparisons S. cerevisiae more <br /> tional characteristics quantifiable evolutionary proper-     closely related fungi, rate differences genes <br /> ties orthologs sequenced genomes.                           lethal nonlethal effects pronounced on- <br />     Essential genes operationally defined molecular ge-    ly comparing closer relatives (Zhang He 2005). <br /> netics gene knockouts result (conditional) lethality   Thus, unequivocal support knockout-rate prediction <br /> or infertility described strongly contributing    remained elusive. <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> organismal fitness. A na&#196;&#177;&#194;&#168;ve expectation indis-          Gene essentiality indicates critical contribution or- <br /> pensable genes given species indispens-         ganismal fitness, does necessarily any <br /> ability equivalents species. Under         specific biological roles. Detailed functional characteriza- <br /> assumption, studies bacteria identified broader phyletic       tions required explore evolutionary <br /> distributions essential genes (Jordan et al. 2002; Gerdes      traits genes linked particular cellular processes, facili- <br /> et al. 2003), phyletic retention levels proved         tated state art functional annotations provided <br /> to predictive feature essentiality bacterial    Gene Ontology (GO) (GO-Consortium 2010) and <br /> and yeast genes (Gustafson et al. 2006). Similarly, gene          InterPro (Hunter et al. 2009) resources. The rigorously <br /> silencing RNA-interference (RNAi) nematodes identi-         curated and, thus, accurate comprehensive <br /> fied greater proportion mutants targeted genes         gene functional annotations represented GO&#226;&#8364;&#8482;s <br /> with orthologs eukaryotes (Castillo-Davis Hartl      founding model organisms: mouse (Mus musculus), fly (Dro- <br /> 2003). Furthermore, quantifying propensity gene loss          sophila melanogaster), yeast (S. cerevisiae). Detailed <br /> among clusters orthologous groups seven distantly           mouse, fly, yeast GO annotations therefore, respec- <br /> related eukaryotes revealed enrichment yeast essential         tively, facilitate inferred putative functionality orthologs <br /> genes clusters losses (Krylov et al. 2003).         vertebrate, arthropod, fungal lineages. <br />     A second expectation, formulated Wilson et al.          Matches InterPro signatures protein domains, the <br /> (1977) known &#226;&#8364;&#732;&#226;&#8364;&#732;knockout-rate prediction,&#226;&#8364;&#8482;&#226;&#8364;&#8482; antici-     majority annotated, provide hints <br /> pates stringent constraints sequence evolution        describing likely biological roles genes recogniz- <br /> of essential genes. Although gained support        able sequence signatures. Together, GO InterPro resour- <br /> studies bacteria (Jordan et al. 2002), confounding factors     ces offer large-scale functional characterization describing <br /> such covariation evolutionary rates levels gene     confirmed putative biological features majority <br /> expression, somewhat inconsistent observations         eukaryotic protein-coding genes. <br /> from studies eukaryotes, yielded inconclusive        A number previous studies investigated con- <br /> results. Substitution rates mouse rat orthologs       nections evolutionary traits functional proper- <br /> suggested essential genes slower evolving,       ties premise gene functionality impacts the <br /> difference observed controlling fast-evolving       strength negative selection. Although studies have <br /> immunity genes (Hurst Smith 1999), greater            focused relating gene essentiality rates sequence <br />  <br />  <br /> 76       Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010 <br />  Correlating Traits Gene Retention, Sequence Divergence, Duplicability Essentiality                                 GBE <br />  <br /> evolution summarized above, additional examined charac-               SilkDB, VectorBase wFleaBase (current releases July <br /> teristics included gene expression levels, propensity gene           2009), 32 fungi UniProt (Release 15.0, March <br /> loss, gene compactness, placements protein inter-              2009) (supplementary table 1, Supplementary Material on- <br /> action gene regulatory networks, example, gene se-                line). Preprocessing gene sets selected longest <br /> quence evolutionary rates negatively correlated                protein-coding transcript gene annotated mul- <br /> expression level positively correlated propensity               tiple transcripts resulting nonredundant set of <br /> for gene loss (Krylov et al. 2003). Beyond correlating pairs             1,363,300 protein-coding genes subsequent orthology <br /> of specific traits, composite variables derived multivar-           delineation analysis. <br /> iate statistical approaches reveal intercorrelations <br />                                                                          Gene Annotations. Gene essentiality data mouse (M. <br /> among traits highlight emergent gene properties. Using <br />                                                                          musculus), fly (D. melanogaster), yeast (S. cerevisiae) <br /> these approaches, Wolf et al. (2006) interpreted principal <br />                                                                          retrieved Database Essential Genes <br /> component seven characteristics reflecting <br />                                                                          (DEG 5.4) (Zhang Lin 2009). Alternative gene essential- <br /> a gene&#226;&#8364;&#8482;s &#226;&#8364;&#732;&#226;&#8364;&#732;importance&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;status&#226;&#8364;&#8482;&#226;&#8364;&#8482;, strong positive con- <br />                                                                          ity data retrieved querying: 1) Mouse Genome In- <br /> tributions expression level, number paralogs, essen- <br />                                                                          formatics (MGI) (Bult et al. 2010) resources pre/peri/ <br /> tiality protein interactions, large negative contributions <br />                                                                          postnatal lethality phenotypes, 2) FB (Tweedie et al. <br /> from evolutionary rate, propensity gene loss (Wolf <br />                                                                          2009) &#226;&#8364;&#732;&#226;&#8364;&#732;phenotypic class: lethal&#226;&#8364;&#8482;&#226;&#8364;&#8482;, 3) Saccharomyces <br /> et al. 2006). Concomitant analysis gene character- <br />                                                                          Genome Database (SGD) (Engel et al. 2010) &#226;&#8364;&#732;&#226;&#8364;&#732;systematic <br /> istics applying multivariate approaches use- <br />                                                                          deletion <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>: inviable.&#226;&#8364;&#8482;&#226;&#8364;&#8482; InterPro GO annotations <br /> ful provide perspectives principal factors influencing <br />                                                                          describing putative gene functional attributes re- <br /> the evolution genes gene repertoires. <br />                                                                          trieved InterPro UniProt (UniProt-Consortium <br />     In study, applying consistent methodology the <br />                                                                          2010) resources, respectively. GO term parent&#226;&#8364;&#8220;child <br /> delineation orthologous gene relations compar- <br />                                                                          relationships InterPro GO mappings re- <br /> ative analysis 40 vertebrates, 23 arthropods, 32 fungi <br />                                                                          trieved GO. <br /> allowed identify core sets genes descended <br /> from common ancestor lineage. By quan- <br /> tifying evolutionary properties gene retention, sequence              Orthologous Group Classification <br /> divergence, duplicability, able examine <br />                                                                          Orthology Delineation. The classification protein- <br /> correlations traits links gene functional <br />                                                                          coding genes orthologous groups based clus- <br /> characteristics. We focused testing specific hypotheses <br />                                                                          tering procedure all-against-all Smith&#226;&#8364;&#8220;Waterman protein <br /> using intuitively interpretable pairwise relations gene <br />                                                                          sequence comparisons using PARALIGN (Saeb&#195;&#184; et al. 2005) <br /> characteristics major eukaryotic lineages, supported <br />                                                                          implemented OrthoDB methodology (Waterhouse <br /> by principal component analysis examine intercorre- <br />                                                                          et al. 2010). The clustering procedure starts iden- <br /> lations traits. With essential genes rep- <br />                                                                          tification best reciprocal hits e value cutoff of <br /> resentative model organisms, aimed explore the <br />                                                                          1   10 6, followed triangulation e value <br /> expectation broad phyletic distributions essential genes <br />                                                                          cutoff 1   10 3, requiring member sequences over- <br /> across dozens eukaryotic species distinct <br />                                                                          lap; clusters expanded include more <br /> lineages. Furthermore, utilizing consistent measures of <br />                                                                          closely related within-species inparalogs. This procedure <br /> sequence evolution, set test knockout-rate <br />                                                                          scrutinized genome projects <br /> prediction fungal, arthropod, vertebrate lin- <br />                                                                          (Richards et al. 2008; Elsik et al. 2009; Kirkness et al. 2010; <br /> eages. Employing functional attributes gene essentiality <br />                                                                          Werren et al. 2010), extensive manual examination or- <br /> with comprehensive gene ontologies protein domain <br />                                                                          thologous groups (Waterhouse et al. 2007; Wyder et al. 2007; <br /> signatures, explored relations gene functional- <br />                                                                          Lemay et al. 2009; Matsui et al. 2009) confirmed their <br /> ity evolutionary traits reflecting strengths selection on <br />                                                                          biological relevance acceptable accuracy. Because orthol- <br /> gene retention, sequence divergence, duplicability. <br />                                                                          ogy defined relative common ancestor spe- <br />                                                                          cies considered, determining hierarchical <br /> Materials Methods                                                    nature orthologous classifications, procedure built or- <br />                                                                          thologous groups radiation phyloge- <br /> Data Sources                                                             nies 40 vertebrates, 23 arthropods, 32 fungi. <br /> Gene Sets. The complete predicted protein-coding gene                    Orthology Type. Orthologous groups exhibit different <br /> sets 95 eukaryotic species retrieved publically             phyletic distributions member genes, allowing them <br /> available genomic resources. These included 40 verte-                    classified different types according gene <br /> brates Ensembl (Release 55, July 2009), 23 arthro-                  copy-numbers phylogenies. Univer- <br /> pods AphidBase, BeeBase, BeetleBase, FlyBase (FB),                  sal groups, separately defined lineage, were <br />  <br />  <br /> Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010                                        77 <br />  Waterhouse et al.                                                                                                 GBE <br />  <br /> required gene members 90%              vertebrates, arthropods, fungi. Mapping essential genes <br /> species (missing arthropods            DEG resource orthologous groups line- <br /> more vertebrates fungi), accounting          age identified 1,673 mouse orthologous groups (2,054 <br /> for possible artifacts incomplete genome sequencing           mouse genes), 324 fly orthologous groups (334 fly genes), <br /> and/or annotation. The remaining orthologous groups           1,074 yeast orthologous groups (1,110 yeast genes). <br /> deemed nonuniversal. Universal orthologous groups             Employing alternative species-specific resources identified <br /> further partitioned single-copy groups,         1,560 mouse orthologous groups (1,954 MGI genes), <br /> gene member half species, re-          2,066 fly orthologous groups (2,234 FB genes), 1,074 <br /> maining multicopy groups, gene          yeast orthologous groups (1,109 SGD genes). Analyses <br /> of half species. This binary distinction   DEG-defined essential genes mouse yeast FB- <br /> universal nonuniversal orthologous groups           defined fly genes presented main text, al- <br /> between single-copy multicopy orthologous groups               ternative sets supporting conclusions described <br /> was chosen simplicity consistency alternative         Supplementary Material online. <br /> (stricter possibly intuitive) categorizations presented <br /> in Supplementary Material online support the <br /> same conclusions (supplementary figs. 1&#226;&#8364;&#8220;3, Supplementary <br /> Material online). <br />                                                                    Results Discussion <br />                                                                    Core Gene Sets Vertebrates, Arthropods, and <br /> Average Percent Identity. The average amino acid per- <br />                                                                    Fungi <br /> cent identity members orthologous <br /> group provides measure level overall group pro-        The Majority Genes Exhibit Traceable Orthology <br /> tein sequence conservation. The average percent identity       Functional Annotations. Our orthology delineation <br /> each orthologous group calculated mean           procedure established ancestral gene relations 95 <br /> between-species pairwise percent identities member pro-         eukaryotic species level major lineages. <br /> teins. Excluding within-species pairwise identities effectively    Of total 1,363,300 protein-coding genes, 86% were <br /> calculates measure group sequence conservation         classified 16,031, 18,937, 13,535 orthologous <br /> all member species independent gene copy-           groups levels common ancestor verte- <br /> number. For universal orthologous groups defined            brates, arthropods, fungi, respectively (fig. 1A). Assum- <br /> each lineage, <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> broad phyletic dis-     ing genes descended common ancestor are <br /> tributions, provides absolute measure group             likely share general functionality enabled tentative ex- <br /> sequence conservation.                                             trapolation functional attributes ascribed more <br /> Related Groups. The all-against-all Smith&#226;&#8364;&#8220;Waterman pro-            members group whole. Accordingly, orthologous <br /> tein sequence comparisons enabled identification ho-       group descriptions summarized associated Inter- <br /> mologous relations orthologous groups. Comparing             Pro GO annotations individual member genes such <br /> two orthologous groups, average alignment score         92% 1.18 million orthologous group <br /> between-group gene comparisons provide basis           member genes classified orthologous groups de- <br /> calculating corresponding e value, indicative number      scribed InterPro domains GO terms 81% <br /> of matches score good ex-         attributes. The larger classified proportion verte- <br /> pected occur chance. The number orthologous               brate genes (93.4%) compared arthropods (76.5%) <br /> groups database scoring used          fungi (81.6%) likely reflects higher levels evolu- <br /> (substitution matrix gap penalties)           tionary divergence species sampled ar- <br /> taken account calculate e value. Because em-       thropod fungal phylogenies, limit the <br /> pirical calculations averaged scores corresponding      detection distant homology. Estimated divergence times <br /> log-scaled averaged e values all-against-all gene         major eukaryotic lineages (Hedges Kumar 2009) sug- <br /> comparisons correlation matches exactly         gest compared 450 My vertebrate evolu- <br /> to lambda value scoring used PARALIGN         tion, arthropods diverged 700 My and <br /> (supplementary fig. 4, Supplementary Material online), aver-       fungi probably span billion years. In addition, <br /> aged between-group gene-to-gene e values used           rates vertebrate protein sequence evolution signifi- <br /> approximation e values describing homologous rela-          cantly slower compared arthropods (Wyder et al. <br /> tions orthologous groups, considering homology               2007) fungi (Dujon 2006). The smaller proportion clas- <br /> support e value cutoffs 1   10 3 1   10 10.              sified genes arthropods influenced more <br />                                                                    variable annotation approaches resulting total gene <br /> Essential Orthologous Groups. Essential genes                 counts (supplementary fig. 5 table 1, Supplementary <br /> model organisms mapped orthologous groups               Material online). <br />  <br />  <br /> 78       Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010 <br />  Correlating Traits Gene Retention, Sequence Divergence, Duplicability Essentiality                                              GBE <br />  <br />  <br />  <br />  <br />       FIG. 1.&#226;&#8364;&#8221;Orthology classification 40 vertebrates, 23 arthropods, 32 fungi. (A) The average numbers classified annotated proteins <br /> per proteome shown lineage. Eighty-six percent total 1.36 million genes classified orthologous groups, 92% of <br /> classified genes assigned orthologous groups described GO InterPro attributes both. (B) Copy-number distributions of <br /> universal orthologous groups mouse (Mus musculus), fly (Drosophila melanogaster), yeast (Saccharomyces cerevisiae). More half of <br /> orthologous groups universal lineage single copy multicopy. Orthologous groups members than <br /> 90% vertebrate, arthropod, fungal species define sets universal orthologous groups lineage constitute half of <br /> mouse, fly, yeast orthologous groups (inset pie charts). The proportion predominantly multicopy universal orthologous groups notably larger in <br /> mouse compared fly yeast. <br />  <br />  <br />  <br />   The comprehensive catalog hierarchical orthologous                        multiple independent duplications majority de- <br /> groups protein-coding genes eukaryotic lin-                  scendant lineages, prominently vertebrates, ef- <br /> eages associated GO InterPro attributes freely                     fectively issuing &#226;&#8364;&#732;&#226;&#8364;&#732;multicopy license&#226;&#8364;&#8482;&#226;&#8364;&#8482; entire <br /> accessible OrthoDB resource, http://cegg.                          lineage. <br /> unige.ch/orthodb (Waterhouse et al. 2010).                                        This trend &#226;&#8364;&#732;&#226;&#8364;&#732;single-copy control&#226;&#8364;&#8482;&#226;&#8364;&#8482; versus multicopy li- <br />                                                                                cense, observed lineages, suggests <br /> More Than Half Orthologous Groups Are                                       major modes gene function influences <br /> Universal Each Lineage Are Almost All                               eukaryotic gene repertoire evolution. Investigations corre- <br /> Single Copy All Multicopy. Defining universal orthol-                       lating gene evolutionary functional traits there- <br /> ogous groups lineage having gene members                        fore consider appropriate controls account this <br /> in 90% sampled species classified half                   dichotomy gene duplicability. <br /> of mouse, fly, yeast groups universal verte- <br /> brate, arthropod, fungal lineages, respectively (fig. 1B).                  Sequence Conservation Universal Orthologous <br /> These orthologs exhibit broad phyletic distributions                      Groups Is Markedly Higher Single-Copy <br /> are likely stringent selection gene retention.                 Than Multicopy Orthologs. Single- multicopy or- <br /> Nevertheless, seemingly indispensable universal single-                   thologous groups considered universal verte- <br /> copy orthologs lost (Wyder et al. 2007) missed                    brate, arthropod, fungal lineages originate the <br /> whole-genome sequencing annotation. Partitioning                            common ancestor considered species phylog- <br /> these fractions universal orthologous groups according                      eny. Despite ages, single-copy groups ex- <br /> to member gene copy-numbers revealed                            hibit significantly higher average protein sequence identities <br /> majority groups single copy mul-               compared multicopy groups (fig. 2, supplementary fig. <br /> ticopy (fig. 1B). This distribution suggests strong gene                  1, Supplementary Material online). Although vertebrate <br /> dosage constraints likely preserve single-copy status                   lineage exhibits accumulated duplicates in <br /> the majority universal orthologs, especially arthropods                  arthropods fungi (fig. 1B), striking contrast between <br /> and fungi, allowing exceptional cases &#226;&#8364;&#732;&#226;&#8364;&#732;copy-                     sequence conservation single-copy genes versus the <br /> number runaways.&#226;&#8364;&#8482;&#226;&#8364;&#8482; At end spectrum,                           divergence multicopy genes consistently observed in <br /> relaxation copy-number restrictions appears lead                      lineages. <br />  <br />  <br /> Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010                                                      79 <br />  Waterhouse et al.                                                                                                                    GBE <br />  <br />  <br />  <br />  <br />      FIG. 2.&#226;&#8364;&#8221;Phyletic profiles sequence conservation levels essential compared nonessential genes mouse (Mus musculus), fly <br /> (Drosophila melanogaster), yeast (Saccharomyces cerevisiae). Orthologous groups essential genes enriched universal orthologs show <br /> constrained sequence evolution. (A) The majority mouse (74%), fly (82%), yeast (85%) essential gene-containing orthologous groups are <br /> universal&#226;&#8364;&#8221;they belong orthologous groups members 90% vertebrate, arthropod, fungal species, about <br /> 50% remaining, nonessential, orthologous groups universal. Orthologous groups essential genes significantly likely <br /> to universal nonessential groups (Fisher&#226;&#8364;&#8482;s Exact Tests P , 1   10 65). Distinguishing predominantly single-copy predominantly <br /> multicopy universal orthologous groups reveals universal essential groups single copy fly yeast. (B) Among single- and <br /> multicopy universal orthologous groups, essential genes display greater sequence conservation, measured mean interspecies <br /> protein sequence identities orthologous group members, without. Notched boxes medians orthologous group percent <br /> identities limits upper lower quartiles, box widths proportional number orthologous groups category. For <br /> significance tests, supplementary table 2 (Supplementary Material online). <br />  <br />  <br />    This indicative substantially greater impact or-                common ancestors vertebrates, arthropods, or <br /> ganismal fitness mutations genes evolving sin-                    fungi. At e value cutoff 1   10 3, 4,869 unique gene <br /> gle-copy control. Mutations better buffered                         sequences estimated vertebrate ancestor, which <br /> the case multicopy genes, maintained                    1.5 times arthropods (3,145) more <br /> duplication principally gene dosage constraints                        double fungi (2,212). <br /> rather functional innovations (Aury et al.                       These consistent insights ancestral gene contents <br /> 2006). Nevertheless, subset eukaryotic genes                      major eukaryotic lineages support ideas ex- <br /> a multicopy license allowed explore protein se-                     panding gene universe perceived organismal complexity. <br /> quence space, turn facilitate functional diver- <br /> gence fine-tuning novelties models of <br />                                                                               Evolutionary Traits Essential Genes <br /> subfunctionalization neofunctionalization (Lynch and <br /> Conery 2000; Hahn 2009).                                                      The delineation orthologous groups phyloge- <br />                                                                               nies vertebrate, arthropod, fungal species se- <br /> The Ancestral Repertoire Distinct Protein-                                 quenced genomes describes core gene sets three <br /> Coding Sequences Has Grown Fungi                                      different eukaryotic lineages. This provided context from <br /> Arthropods Vertebrates. Detectable homology be-                            examine evolutionary characteristics the <br /> tween orthologous groups (see Materials Methods)                       subset experimentally defined essential genes through <br /> indicative common sequence ancestry full-                     concomitant analysis gene dispensability evolution- <br /> <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> member genes (ancient paralogs) individual se-                   ary traits orthologous group phyletic distributions se- <br /> quence regions (protein domains). At e value cutoff 1                   quence diversities. Employing viability data model <br />   10 3, half universal orthologous groups                         organisms (see Materials Methods) identified 14.2% <br /> mouse (41.2%), fly (59.5%), yeast (43.9%) homol-                      mouse, 19.6% fly, 23.7% yeast orthologous <br /> ogous universal orthologous group                    groups contained essential gene (essential groups). <br /> their respective lineages (supplementary fig. 6, Supplemen-                   The mapping essential genes orthologous groups <br /> tary Material online). Counting sets homologous                      vertebrate, arthropod, fungal line- <br /> universal groups summing groups                         ages allowed explore broader evolutionary context <br /> homologs provides lower estimate total numbers                       single model organism species, derive <br /> of distinct protein-coding sequences genomes                        predictive associations essentiality. <br />  <br />  <br /> 80        Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010 <br />  Correlating Traits Gene Retention, Sequence Divergence, Duplicability Essentiality                                 GBE <br />  <br /> Essential Genes Are Enriched Universal                                vertebrate lineage, majority ar- <br /> Orthologs. Comparing proportions universal essen-                 thropod fungal universal groups single copy. <br /> tial groups vertebrates, arthropods, fungi revealed                  This prompts speculation negative effects of <br /> that essential groups significantly likely                gene expression level imbalances diminished verte- <br /> universal nonessential groups (Fisher&#226;&#8364;&#8482;s Exact Tests                 brates, relaxing single-copy controls allowing <br /> P , 1   10 65; fig. 2A, supplementary fig. 2 table                   multicopy gene buffering mutations lead ex- <br /> 2, Supplementary Material online). The preferentially univer-            tended exploration protein sequence space. Such suppos- <br /> sal phyletic distributions essential gene-containing groups           edly redundant gene copies provide buffer against <br /> in examined eukaryotic lineages consistent               gene inactivation, however, compensation essential func- <br /> with observed broad phyletic distributions essential genes            tions paralogs assumed, duplicates in <br /> in bacteria, yeasts, nematodes (Jordan et al. 2002;                  mouse necessarily confer functional redundancy <br /> Castillo-Davis Hartl 2003; Gerdes et al. 2003; Krylov                (Liao Zhang 2007). In fact, retention gene duplicates <br /> et al. 2003; Gustafson et al. 2006). This correlation            facilitated coordinated reduction expression <br /> supported principal component analysis (supplementary                 levels, rendering copies necessary fulfill biolog- <br /> fig. 7 table 3, Supplementary Material online), which,               ical role ancestral gene creating redundancy <br /> in agreement observed opposing contributions                     level molecular function (Qian et al. 2010). <br /> essentiality propensity loss gene&#226;&#8364;&#8482;s importance              However, differences gene duplicability re- <br /> (Wolf et al. 2006), highlighted major coordinated contri-                flect relative strengths purifying selection lin- <br /> butions gene essentiality phyletic retention levels.              eages imposed factors effective population sizes. <br /> Furthermore, comparing lineages reveals                   The greater permissiveness slightly deleterious events in <br /> universal vertebrate, arthropod, fungal groups make                   vertebrates allow accumulation redundant <br /> up similar majority essential groups mouse (74%),                gene copies purged intensive se- <br /> fly (82%), yeast (85%), respectively, suggests                 lection pressures (Lynch Conery 2000). <br /> that quarters experimentally identi- <br /> fied essential genes individual organisms likely               Sequence Evolution Essential Genes Is More <br /> have orthologs lineage. Nevertheless, minor                Constrained. Among single- multicopy universal <br /> fraction essential groups exhibit nonuniversal phyletic dis-          orthologous groups, essential genes exhibit <br /> tributions, suggesting essential genes cru-            greater sequence conservation (fig. <br /> cial biological processes specific selected model              2B, supplementary fig. 2 table 2, Supplementary Mate- <br /> organisms closer relatives.                                    rial online). This difference clear orthologous groups <br />    Our large-scale study, therefore, ultimately confirms             partitioned universal single- versus multicopy <br /> propensity essential genes belong universal orthol-             groups control effects age copy-number <br /> ogous groups fungi confidently extends                   constraints sequence evolution. Universal groups are <br /> trend arthropod vertebrate lineages.                          descendants ancestral genes retained across <br />                                                                          lineage represent orthologous groups of <br /> The Majority Universal Essential Groups Are                           common age old common ancestor. <br /> Single Copy Yeast Fly. Distinguishing                     The remaining nonuniversal groups ancient <br /> single- multicopy universal orthologous groups reveals               groups experienced multiple gene losses, youn- <br /> marked differences lineages respect                 ger clade-specific groups, display variable ranges of <br /> to essentiality. Yeast essential groups enriched univer-          sequence identities (supplementary fig. 3, Supplementary <br /> sal single-copy multicopy groups, fly                Material online). Subsequent partitioning age-con- <br /> mostly single copy greater proportion                trolled universal orthologous groups single- multi- <br /> of multicopy groups, mouse, multicopy                    copy groups (as shown fig. 1B) distinguishes <br /> groups enriched (Fisher&#226;&#8364;&#8482;s Exact Tests P , 1   10 50;                 effects copy-number constraints the <br /> fig. 2A, supplementary fig. 2 table 2, Supplementary                 effects essentiality sequence evolution. <br /> Material online). This reflected total proportions                 The lower level amino acid substitutions essen- <br /> in lineage, mouse exhibits larger propor-              tial groups indicative stronger purifying selection <br /> tion universal multicopy groups (44.9%) compared                 evolution vertebrates, arthropods, and <br /> fly (17.4%) yeast (14.4%) (fig. 1B). Principal component             fungi supports knockout-rate prediction slower <br /> analysis suggests relaxation copy-number                sequence evolution essential genes. However, quan- <br /> constraints distinguishing feature verte-             titative distinction genes known essential <br /> brates (supplementary fig. 7 table 3, Supplementary                  functions substantially prominent <br /> Material online). Thus, highly constrained                    distinction single-copy constrained <br /> essential genes, maintained duplications appear prevalent                genes multicopy license. Thus, through <br />  <br />  <br /> Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010                                        81 <br />  Waterhouse et al.                                                                                                                                                 GBE <br />  <br /> Table 1 <br /> Essential Nonessential Orthologous Groups Related Groups Mouse (Mus musculus), Fly (Drosophila melanogaster), Yeast <br /> (Saccharomyces cerevisiae) <br />                                                Mouse                                                Fly                                             Yeast <br />   Cutoff                                  ES            NE                 ft                ES            NE                 ft              ES            NE                ft <br />           3                                                                      24                                                 13                                              2 <br /> 1   10             Relatives              667          1,978         1.2   10              1,123          2,207         9.0   10              420           828         7.6   10 <br />                    No relatives           565          3,214                                 565          1,701                               495         1,099 <br />           10                                                                     12                                                 3                                               2 <br /> 1   10             Relatives              192            376         5.7   10                653          1,187         9.8   10              173           321         7.8   10 <br />                    No relatives         1,040          4,816                               1,035          2,721                               742         1,606 <br />      NOTE.&#226;&#8364;&#8221;Universal essential orthologous (ES) groups nonessential (NE) groups compared universal orthologous groups related groups e value <br /> cutoffs 1   10 3 1   10 10. Essential groups animals likely relatives (P values Fisher&#226;&#8364;&#8482;s Exact Tests [ft] enrichment). See Materials Methods for <br /> definitions universal, essential, related orthologous groups. For results alternative essentiality data sets, supplementary table 4 (Supplementary Material online). <br />  <br />  <br />  <br />  <br /> a consistent, whole-genome scale analysis, higher                                      Material online). Member gene GO InterPro attributes <br /> constraints sequence evolution essential genes                                       facilitated assignment principal GO terms (e.g., ge- <br /> across vertebrates, arthropods, fungi.                                                     neric GO Slim terms) universal mouse, fly, yeast or- <br />                                                                                                thologous groups using GO child parent relations and <br /> Essential Groups Animals Are More Likely                                                 InterPro GO mappings (GO-Consortium 2010). Of the <br /> Have Homologs. If essential gene duplications resulted                                      23 biological processes, 16 10 enriched essential <br /> severely imbalanced functionality critical biological pro-                                  genes mouse fly, respectively, 2 show <br /> cesses essential groups expected limited                                   significant enrichment yeast. Examining categories de- <br /> to appearing lineage unique ortholo-                                      fined generic GO Slim subsets biological processes, <br /> gous groups. Instead, essential groups appear likely                                   molecular functions, cellular components confirms this <br /> have relatives, suggesting essential gene ancestors                                  difference processes functions compo- <br /> more frequently given rise novel related orthologous                                        nents instead exhibit similar proportions enriched terms in <br /> groups vertebrates arthropods (table 1, supplemen-                                      lineages (supplementary table 5 fig. 8, Sup- <br /> tary table 4, Supplementary Material online). Alternatively,                                   plementary Material online). <br /> ancient genes frequently maintained duplicates                                          Although variations annotation strategies contrib- <br /> gained essential functions. Although 54% universal                                          ute observed differences, general paucity of <br /> mouse essential groups universal group relatives,                                    categories enriched essential groups fungi compared <br /> 38% universal nonessential groups related                                      animals appears suggest fungal processes <br /> universal orthologous groups 1   10 3 cutoff. The                                       functions robust gene knockouts. Robust- <br /> difference similar fly groups, 67% universal                                   ness key attribute complex systems facilitates <br /> essential versus 56% universal nonessential groups                                     evolvability itself selected (Kitano 2004). It is <br /> relatives. However, yeast, small 3% difference                                   tempting speculate stronger effective se- <br /> statistically significant (46% essential vs. 43% nonessential).                                lection evolution fungi confer <br />     This trend consistent studies compared evo-                                   greater robustness fungal processes functions. <br /> lutionary rates duplicated genes (with within-species ho- <br /> mologs) singletons (unique, within-species                                        Divergence Rates Vary Functional <br /> homologs) showed conserved genes con-                                            Categories Lineages. Orthologous groups genes <br /> strained sequence evolution likely rise                                      involved processes translation precursor me- <br /> to maintained duplicates (Davis Petrov 2004; Jordan                                        tabolism (generation precursor metabolites energy) <br /> et al. 2004). Thus, functionally important genes con-                                 highest levels sequence conservation verte- <br /> stituent domains appear utilized fre-                                        brates, arthropods, fungi (fig. 3). At end the <br /> quently evolution eukaryotes.                                                scale, immune responses adhesion processes shared <br />                                                                                                divergent categories reproductive processes rela- <br /> Functional Perspective                                                                         tively divergent vertebrates arthropods, whereas <br />                                                                                                divergence DNA metabolism common arthropods <br /> The Majority Biological Processes Are Enriched                                              fungi. Structural molecule translation factor activ- <br /> for Essential Genes Animals. Identifying proportions                                        ities commonly conserved molecular functions, whereas <br /> of essential gene-containing orthologous groups GO                                          divergent include receptor, signal transducer, and <br /> functional categories revealed significant enrichment es-                                   nuclease activities, carbohydrate chromatin <br /> sential genes principal biological processes mouse                                    binding (supplementary table 5 fig. 8, Supplementary <br /> and fly (fig. 3, supplementary table 5, Supplementary                                          Material online). Among cellular components, ribosome <br />  <br />  <br /> 82             Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010 <br />  Correlating Traits Gene Retention, Sequence Divergence, Duplicability Essentiality                                            GBE <br />  <br />  <br />  <br />  <br />      FIG. 3.&#226;&#8364;&#8221;Orthologous group sequence conservation principal GO biological processes mouse (Mus musculus), fly (Drosophila melanogaster), <br /> and yeast (Saccharomyces cerevisiae). Processes ordered median orthologous group sequence identities fly categories divergent <br /> in red conserved blue (more standard deviation greater than&#226;&#8364;&#8221;dark blue&#226;&#8364;&#8221;or than&#226;&#8364;&#8221;dark red&#226;&#8364;&#8221;the mean identity). Several <br /> mouse fly biological processes enriched essential gene-containing orthologous groups (striped shading) contrast processes in <br /> yeast. Notched boxes medians orthologous group percent identities limits upper lower quartiles, box heights are <br /> proportional number orthologous groups category. For GO identifiers, median identities, orthologous group counts, Fisher&#226;&#8364;&#8482;s Exact <br /> Tests enrichment essential groups, supplementary table 5 (Supplementary Material online). <br />  <br />  <br />  <br /> appears highly conserved, divergent                      tens groups examining <br /> extracellular elements vertebrates arthropods                      protein domain signatures orthologous groups with <br /> components nucleus arthropods fungi. These                      numerous related groups reveals proliferation <br /> results agreement pairwise studies fly&#226;&#8364;&#8220;                   functional domains created distinguishing <br /> mosquito (Zdobnov et al. 2002) human&#226;&#8364;&#8220;chicken                               superfamilies lineage (fig. 4). At e value cutoff of <br /> (Chicken-Genome-Consortium 2004) comparisons                             1   10 3 (see Materials Methods), 9.0% vertebrate <br /> identified structural molecules involved protein                 7.3% arthropod orthologous groups than <br /> transport conserved signal transducers                    50 related groups, fungi, 5.4% ortholo- <br /> immune-related genes divergent. Similarly, mul-                   gous groups 20 related groups. Even essen- <br /> tispecies comparisons human yeast associated                     tial genes orthologous groups <br /> fast-evolving protein families regulatory roles re-                  highest numbers relatives (fig. 4), agreement <br /> sponses stimuli immune challenges, con-                      observation functionally important genes or <br /> trast slow-evolving families involved transport, protein                   domains frequently reutilized eukary- <br /> synthesis, primary metabolism (Lopez-Bigas et al. 2008).                   otic evolution (table 1). <br />    Orthologous group average sequence identities, there-                         The highly abundant zinc finger proteins feature both <br /> fore, clearly distinguish fast- slowly evolving                   vertebrates arthropods, proliferation of <br /> functional categories, highlighting common dis-                      rhodopsin-like G protein&#226;&#8364;&#8220;coupled receptors (GPCRs) ver- <br /> tinct evolutionary pressures functional subsets genes                   tebrates contrasts expanded superfamily arthropod <br /> in eukaryotic lineages.                                             peptidases. Cytochrome P450s prominent arthropods <br />                                                                               fungi, families transporters characterize the <br /> Distinct Domain Proliferations Characterize                                   fungi, protein kinases prominent lin- <br /> Vertebrate, Arthropod, Fungal Proteomes.                                  eages. These domains characterize abun- <br /> Shared sequence homology orthologous groups                             dant eukaryotic protein-coding genes, other <br /> may derive expanded repertoires functionally ben-                     domains highlight lineage-specific biology verte- <br /> eficial protein domains lineage. Many orthologous                     brate Kruppel-associated boxes involved transcriptional <br /> groups related (share common sequence ancestry)                        repression major histocompatibility complex proteins <br />  <br />  <br /> Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010                                                    83 <br />  Waterhouse et al.                                                                                                                     GBE <br />  <br />  <br />  <br />  <br />      FIG. 4.&#226;&#8364;&#8221;Independent proliferation functional domains created distinguishing protein superfamilies vertebrates (A), arthropods (B), <br /> and fungi (C). The majority orthologous groups identifiable homology groups related groups, however, <br /> those numerous relatives characterized specific protein domains. The superfamily zinc finger C2H2-like proteins common to <br /> vertebrates arthropods, expansion vertebrate rhodopsin-like GPCRs contrasts arthropod peptidases. The fungi are <br /> characterized families transporters major facilitator superfamily general substrate transporters. Cytochrome P450s prominent in <br /> arthropods fungi protein kinases feature lineages. Orthologous groups essential genes with <br /> some highest numbers relatives. Related groups defined average pairwise Smith&#226;&#8364;&#8220;Waterman e value members of <br /> each group lineage cutoff 1   10 3. <br />  <br />  <br />  <br /> of acquired immune (supplementary table 6,                         family expansions, notably rhodopsin-like GPCRs, <br /> Supplementary Material online). Arthropod-specific domains                    include vertebrate olfactory chemokine receptors <br /> include pheromone/odorant-binding proteins oxygen-                        absent arthropods. In contrast, arthropod che- <br /> carrying hemocyanins, subfamilies glycoside                        moreceptors related rhodopsin-like GPCRs but <br /> hydrolases cellulose-binding domains characterize                         operate instead ligand-gated ion channels form <br /> fungal-specific biology.                                                      large repertoire gustatory receptors Arthropoda <br />    Zinc finger motifs form DNA-binding domains                     olfactory receptors terrestrial insects (Sato et al. 2008; <br /> many transcription factors mediate protein&#226;&#8364;&#8220;protein in-                    Pen&#203;&#339;alva-Arana et al. 2009). The usually secreted S1/S6 pep- <br /> teractions (Brayer Segal 2008). The evolution zinc fin-                tidases employ histidine&#226;&#8364;&#8220;aspartate&#226;&#8364;&#8220;serine catalysis cleave <br /> ger proteins animal lineages followed                         target proteins diverse processes including fertilization, <br /> distinct paths, expansions AD-type arthropods                  development, intestinal digestion, blood coagulation, apo- <br /> that mirrored vertebrates, proliferation ver-               ptosis, immunity (Di Cera 2009). Arthropod diversity is <br /> tebrates zinc finger proteins acquired vertebrate-                reflected variety food sources rely, <br /> specific Kruppel-associated boxes (Copley 2008). The prolif-                  especially larvae nymphs, abundance ar- <br /> eration proteins regulatory interaction-me-                  thropod proteases driven digestive require- <br /> diating functions facilitated evolution                           ments. These expansions reflect key roles of <br /> organismal complexity higher eukaryotes. GPCRs sig-                    proteases characteristic arthropod processes molting <br /> nal-transducer transmembrane proteins include hor-                       metamorphosis immune defense open <br /> mone, olfactory, neurotransmitter, light receptors                        circulatory system. Cytochrome P450s important for <br /> (Fredriksson et al. 2005). Vertebrates exhibit extensive GPCR                 synthesis breakdown hormones in <br />  <br />  <br /> 84        Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010 <br />  Correlating Traits Gene Retention, Sequence Divergence, Duplicability Essentiality                                        GBE <br />  <br /> detoxification processes critical clearance              Literature Cited <br /> potentially harmful xenobiotics. Their prominent diversity               Aury J, et al. 2006. Global trends whole-genome duplications <br /> in arthropods certainly linked incredible va-                revealed ciliate Paramecium tetraurelia. Nature. <br /> riety environmental challenges, particularly respect                  444(7116):171&#226;&#8364;&#8220;178. <br /> to coevolution arthropods plants (Feyereisen                  Brayer K, Segal D. 2008. Keep fingers DNA: protein-protein <br /> 2006). The numerous cytochrome P450s fungi contribute                      interactions mediated C2H2 zinc finger domains. Cell Biochem <br /> to ability occupy large variety ecological niches               Biophys. 50(3):111&#226;&#8364;&#8220;131. <br /> that require specialized secondary metabolism (Kelly               Bult C, Kadin J, Richardson J, Blake J, Eppig J. 2010. The Mouse Genome <br /> et al. 2009), numerous transporters critical                    Database: enhancements updates. Nucleic Acids Res. <br /> for exchanging solutes environments.                               38:D586&#226;&#8364;&#8220;D592 (Database issue). <br />                                                                          Castillo-Davis C, Hartl D. 2003. Conservation, relocation duplication <br />                                                                               genome evolution. Trends Genet. 19(11):593&#226;&#8364;&#8220;597. <br /> Conclusions                                                              Chicken-Genome-Consortium. 2004. Sequence comparative anal- <br />                                                                               ysis chicken genome provide unique perspectives on <br /> This study provides consistent view gene evolutionary <br />                                                                               vertebrate evolution. Nature. 432(7018):695&#226;&#8364;&#8220;716. <br /> traits large set vertebrate, arthropod, fungal <br />                                                                          Copley R. 2008. The animal genome: comparative genomics and <br /> species spanning millions years divergence sheds                    evolution. Philos Trans R Soc Lond B Biol Sci. 363(1496):1453&#226;&#8364;&#8220;1461. <br /> light fundamental principles eukaryotic gene               Davis J, Petrov D. 2004. Preferential duplication conserved proteins in <br /> repertoire evolution. Classification protein-coding genes                  eukaryotic genomes. PLoS Biol. 2(3):E55. <br /> into orthologous groups genes descended                    Di Cera E. 2009. Serine proteases. IUBMB Life. 61(5):510&#226;&#8364;&#8220;515. <br /> common ancestors reveals majority groups ex-                 Dujon B. 2006. Yeasts illustrate molecular mechanisms eukaryotic <br /> hibit broad phyletic distributions genes univer-                  genome evolution. Trends Genet. 22(7):375&#226;&#8364;&#8220;387. <br /> sally present lineage. Most genes evolve            Elsik C, et al. 2009. The genome sequence taurine cattle: window <br />                                                                               ruminant biology evolution. Science. 324(5926):522&#226;&#8364;&#8220;528. <br /> under single-copy control multicopy license <br />                                                                          Engel S, et al. 2010. Saccharomyces Genome Database provides mutant <br /> frequently duplicate entire lineage. Evolution of <br />                                                                               phenotype data. Nucleic Acids Res. 38:D433&#226;&#8364;&#8220;D436 (Database issue). <br /> single-copy orthologs constrained sequence                Feyereisen R. 2006. Evolution insect P450. Biochem Soc Trans. 34(Pt <br /> level contrasts elevated divergence main-                       6):1252&#226;&#8364;&#8220;1255. <br /> tained duplicates. In addition, evolutionary perspective             Fitch W. 1970. Distinguishing homologous analogous proteins. <br /> on gene essentiality assessed model organisms firmly sup-                  Syst Zool. 19(2):99&#226;&#8364;&#8220;113. <br /> ports hypotheses genes stronger                  Fredriksson R, Lagerstro&#194;&#168;m M, Schio&#194;&#168;th H. 2005. Expansion the <br /> selection gene retention gene sequence con-                      superfamily G-protein-coupled receptors chordates. Ann N Y <br /> servation vertebrates, arthropods, fungi.                              Acad Sci. 1040:89&#226;&#8364;&#8220;94. <br />                                                                          Gerdes S, et al. 2003. Experimental determination level <br />    Our methodologically consistent large-scale approach <br />                                                                               analysis essential genes Escherichia coli MG1655. J Bacteriol. <br /> provides evidence principles shared <br />                                                                               185(19):5673&#226;&#8364;&#8220;5684. <br /> among major eukaryotic lineages highlights               GO-Consortium. 2010. The Gene Ontology 2010: extensions and <br /> lineage-specific idiosyncrasies taken ac-                 refinements. Nucleic Acids Res. 38:D331&#226;&#8364;&#8220;D335 (Database issue). <br /> count cross-lineage comparisons. Some princi-               Gustafson A, Snitkin E, Parker S, DeLisi C, Kasif S. 2006. Towards the <br /> ples, stronger constraints sequence                        identification essential genes using targeted genome sequencing <br /> evolution essential genes, previously hypothesized                    comparative analysis. BMC Genomics. 7:265. <br /> or evidenced using smaller data sets, quantita-             Hahn M. 2009. Distinguishing evolutionary models the <br /> tively prominent distinction tolerance gene copy-                   maintenance gene duplicates. J Hered. 100(5):605&#226;&#8364;&#8220;617. <br /> number variations largely underappreciated.                     Hedges SB, Kumar S. 2009. The timetree life. Oxford: Oxford <br />                                                                               University Press. <br />                                                                          Hirsh A, Fraser H. 2001. Protein dispensability rate evolution. <br /> Supplementary Material                                                        Nature. 411(6841):1046&#226;&#8364;&#8220;1049. <br />                                                                          Hunter S, et al. 2009. InterPro: integrative protein signature <br /> Supplementary figures 1&#226;&#8364;&#8220;8 tables 1&#226;&#8364;&#8220;6 available at <br />                                                                               database. Nucleic Acids Res. 37:D211&#226;&#8364;&#8220;D215 (Database issue). <br /> Genome Biology Evolution online (http://www.gbe.                     Hurst L, Smith N. 1999. Do essential genes evolve slowly? Curr Biol. <br /> oxfordjournals.org/).                                                         9(14):747&#226;&#8364;&#8220;750. <br />                                                                          Jordan I, Rogozin I, Wolf Y, Koonin E. 2002. Essential genes more <br /> Acknowledgments                                                               evolutionarily conserved nonessential genes bacteria. <br />                                                                               Genome Res. 12(6):962&#226;&#8364;&#8220;968. <br /> The authors like thank Prof Iwona Stroynowski, Dr               Jordan I, Wolf Y, Koonin E. 2004. Duplicated genes evolve slower than <br /> Ivo Pedruzzi, Dr Fredrik Tegenfeldt, members                   singletons despite initial rate increase. BMC Evol Biol. 4:22. <br /> Computational Evolutionary Genomics Group useful dis-                Kelly D, Krasevec N, Mullins J, Nelson D. 2009. The CYPome <br /> cussions, anonymous referees con-                    (Cytochrome P450 complement) Aspergillus nidulans. Fungal <br /> structive suggestions.                                                        Genet Biol. 46(Suppl 1):S53&#226;&#8364;&#8220;S61. <br />  <br />  <br />  <br /> Genome Biol. Evol. 2:75&#226;&#8364;&#8220;86. doi:10.1093/gbe/evq083 Advance Access publication December 9, 2010                                                 85 <br />  Waterhouse et al.                                                                                                                      GBE <br />  <br /> Kirkness E, et al. 2010. Genome sequences human body louse              Sato K, Pellegrino M, Nakagawa T, Vosshall L, Touhara K. 2008. Insect <br />     primary endosymbiont provide insights permanent               olfactory receptors heteromeric ligand-gated ion channels. <br />     parasitic lifestyle. Proc Natl Acad Sci U S A. 107(27):12168&#226;&#8364;&#8220;12173.            Nature. 452(7190):1002&#226;&#8364;&#8220;1006. <br /> Kitano H. 2004. Biological robustness. Nat Rev Genet. 5(11):826&#226;&#8364;&#8220;837.           Tweedie S, et al. 2009. FlyBase: enhancing Drosophila Gene <br /> Koonin E. 2005. Orthologs, paralogs, evolutionary genomics. Annu               Ontology annotations. Nucleic Acids Res. 37:D555&#226;&#8364;&#8220;D559 (Data- <br />     Rev Genet. 39:309&#226;&#8364;&#8220;338.                                                         <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> issue). <br /> Krylov D, Wolf Y, Rogozin I, Koonin E. 2003. Gene loss, protein sequence       UniProt-Consortium. 2010. The Universal Protein Resource <br />                                                                                    (UniProt) 2010. Nucleic Acids Res. 38:D142&#226;&#8364;&#8220;D148 (Database <br />     divergence, gene dispensability, expression level, interactivity are <br />                                                                                    issue). <br />     correlated eukaryotic evolution. Genome Res. 13(10):2229&#226;&#8364;&#8220;2235. <br />                                                                                Waterhouse R, et al. 2007. Evolutionary dynamics immune-related <br /> Lemay D, et al. 2009. The bovine lactation genome: insights the <br />                                                                                    genes pathways disease-vector mosquitoes. Science. <br />     evolution mammalian milk. Genome Biol. 10(4):R43. <br />                                                                                    316(5832):1738&#226;&#8364;&#8220;1743. <br /> Liao B, Scott N, Zhang J. 2006. Impacts gene essentiality, expression <br />                                                                                Waterhouse R, Zdobnov E, Tegenfeldt F, Li J, Kriventseva E. 2010. <br />     <span id='am-165' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, gene compactness evolutionary rate                      OrthoDB: hierarchical catalog eukaryotic orthologs 2011. <br />     mammalian proteins. Mol Biol Evol. 23(11):2072&#226;&#8364;&#8220;2080.                           Nucl. Acids Res. doi: 10.1093/nar/gkq930. Advance Access October <br /> Liao B, Zhang J. 2007. Mouse duplicate genes essential                   23, 2010. <br />     singletons. Trends Genet. 23(8):378&#226;&#8364;&#8220;381.                                   Werren J, et al. 2010. Functional evolutionary insights the <br /> Lopez-Bigas N, De S, Teichmann S. 2008. Functional protein divergence              genomes parasitoid Nasonia species. Science. <br />     evolution Homo sapiens. Genome Biol. 9(2):R33.                       327(5963):343&#226;&#8364;&#8220;348. <br /> Luz H, Vingron M. 2006. Family specific rates protein evolution.     <br /> </body></html>