<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>DOI:10.1093/jnci/djs471                                                                              &#194;&#169; The Author 2012. Published Oxford University Press. <br /> &#239;&#187;&#191;                                          This Open Access article distributed terms Creative Commons Attribution Non-Commercial <br />                                                   License (http://creativecommons.org/licenses/bync/3.0/uk/) permits unrestricted noncommercial use, <br />                                                                    distribution, reproduction medium, provided original work properly cited. <br /> Article <br />  <br />  <br /> Effect SMURF2 Targeting Susceptibility MEK Inhibitors <br /> in Melanoma <br /> Michael P. Smith, Jennifer Ferguson, Imanol Arozarena, Robert Hayward, Richard Marais, Anna Chapman, Adam Hurlstone, <br /> Claudia Wellbrock <br />  <br /> Manuscript received April 5, 2012; revised September 27, 2012; accepted September 30, 2012. <br /> Correspondence <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span>: Claudia Wellbrock, PhD, Molecular Cancer Studies, Wellcome Trust Centre Cell Matrix Research, University Manchester, Michael Smith <br /> Bldg, Oxford Rd, Manchester, M13 9PT, UK (e-mail: Claudia.Wellbrock@manchester.ac.uk). <br />  <br />  <br />  <br />  <br /> 	Background	        The mitogen-activated protein&#226;&#8364;&#8220;kinase pathway consisting kinases RAF, MEK, ERK central cell pro- <br />                     liferation <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span> deregulated 90% melanomas. MEK inhibitors currently trialled <br />                     clinic, despite efficient target inhibition, cytostatic cytotoxic activity limits efficacy. <br />  <br /> 	   Methods	        We assessed cytotoxicity MEK inhibitors (PD184352 selumetinib) melanoma cells toluidine-blue <br />                     staining, caspase 3 cleavage, melanoma-sphere growth. Western blotting quantitative real-time poly- <br />                     merase chain reaction applied determine SMAD-specific E3 ubiquitin protein ligase 2 (SMURF2), PAX3, <br />                     MITF expression. Human melanoma samples (n&#194;&#160;=&#194;&#160;77) various stages analyzed SMURF2 and <br />                     PAX3 expression. <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> performed target SMURF2 MEK inhibition vivo melanoma <br />                     xenografts mice zebrafish. All statistical tests two-sided. <br />  <br /> 	    Results	       Activation transforming growth factor &#206;&#178; (TGF-&#206;&#178;) signalling sensitized melanoma cells cytotoxic effects <br />                     MEK inhibition. Melanoma cells resistant cytotoxic effects MEK inhibitors counteracted TGF-&#206;&#178; sig- <br />                     nalling overexpression E3 ubiquitin ligase SMURF2, resulted increased expression of <br />                     transcription factors PAX3 MITF. High MITF expression protected melanoma cells MEK inhibitor <br />                     cytotoxicity. Depleting SMURF2 reduced MITF expression substantially lowered threshold MEK inhibi- <br />                     tor&#226;&#8364;&#8220;induced apoptosis. Moreover, SMURF2 depletion sensitized melanoma cells cytotoxic effects selu- <br />                     metinib, leading cell death concentrations approximately 100-fold lower concentration required to <br />                     induce cell death SMURF2-expressing cells. Mice treated selumetinib dosage 10&#226;&#8364;&#8240;mg/kg body <br />                     weight daily produced response, combination SMURF2 depletion, selumetinib suppressed <br />                     tumor growth 97.9% (95% confidence interval&#194;&#160;=&#194;&#160;38.65% 155.50%, P = .005). <br />  <br /> 	Conclusions	       Targeting SMURF2 novel therapeutic approach increasing antitumor efficacy MEK inhibitors. <br />  <br /> 		                  J Natl Cancer Inst 2013;105:33&#226;&#8364;&#8220;46 <br />  <br />  <br />  <br /> The mitogen-activated protein (MAP)&#226;&#8364;&#8220;kinase pathway, consist-                    continuous long-term exposure drug (6&#226;&#8364;&#8220;8). However, dose- <br /> ing kinases RAF, MEK, ERK, hyperactivated                 limiting toxicity short half-life inhibitor prevents these <br /> than 90% melanomas, mainly triggered mutated                conditions satisfied vivo (3,5). Moreover, frequently <br /> BRAF-kinase, constitutively activates MEK (1,2). Currently,               BRAF-mutant cells display high resistance MEK inhibitor&#226;&#8364;&#8220; <br /> inhibitors MEK evaluated clinic,               induced apoptosis despite efficient target inhibition (9,10), which <br /> they single-agent activity, overall responses                 points activation additional pathways cells that <br /> much lower expected (3,4). Importantly, doses MEK                   enable survive low levels ERK activation. <br /> inhibitor used reported trials effectively produce target inhibi-            Thus, addition recently discovered mechanisms reinstating <br /> tion suppress proliferation (4,5). These observations suggest               MAP-kinase pathway activation presence inhibitor (11&#226;&#8364;&#8220; <br /> that cytostatic effects sufficient produce major                14), abrogation cytotoxic effects MEK inhibitors brought <br /> response patients majority tumors intrinsi-              parallel pathways considered possible <br /> cally refractory cytotoxic effects MEK inhibition.                        resistance mechanism. Phosphatidylinositol-3-kinase its <br />     Indeed, studies melanoma cell lines, clear               upstream regulators activate pathway (10,15,16), trials <br /> cytotoxic effects MEK inhibitors melanoma cells generally                 using phosphatidylinositol-3-kinase pathway inhibitors combined <br /> occur concentrations considerably higher                     MEK inhibitors currently underway. However, like the <br /> concentrations needed produce antimitotic effect                 MAP-kinase pathway, phosphatidylinositol-3-kinase signalling is <br />  <br />  <br /> jnci.oxfordjournals.org                                                                                                               JNCI&#226;&#8364;&#402;|&#226;&#8364;&#402;Articles&#226;&#8364;&#402;33 <br />  central nontumor cells, toxicity inhibitors   dithiothreitol, 0.01% weight/volume bromophenol blue) lysis <br /> a limiting factor (17,18).                                               buffer [50&#226;&#8364;&#8240;mm 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid, <br />     We, therefore, aimed identify melanoma specific proteins          pH 7.5, 150&#226;&#8364;&#8240;mM sodium chloride, 1.5&#226;&#8364;&#8240;mM magnesium chloride, <br /> that counteract MEK inhibitor&#226;&#8364;&#8220;induced cytotoxic effects. Such            1&#226;&#8364;&#8240;mM ethylene glycol tetraacetic acid, 10% glycerol, 1% Triton <br /> proteins useful predictive markers sensitivity mel-    X-100, 1&#226;&#8364;&#8240;mM phenylmethanesulfonyl fluoride, 0.2&#226;&#8364;&#8240;mM sodium <br /> anoma cells MEK inhibitors. Moreover, targeting proteins         orthovanadate, 10&#226;&#8364;&#8240;mg/mL leupeptin, 10&#226;&#8364;&#8240;mg/ml aprotinin) 20 <br /> may 1)&#194;&#160;improve initial response MEK inhibitors enhanc-         minutes 4&#194;&#186;C analyzed standard immunoblotting pro- <br /> ing cytotoxic effects, overcoming primary resistance,     tocols. The protein loaded lane, and <br /> 2)&#194;&#160; prevent secondary acquired resistance develops         primary antibodies detected luminescence using peroxi- <br /> selection primary resistant cells drug treatment.             dase-coupled secondary antibodies (Jackson, Stratech, Newmarket, <br />                                                                          UK). The primary antibodies used were: phospho-ERK (mouse <br />                                                                          monoclonal MAPK-YT, 1:10,000 dilution; Sigma); ERK2 (rabbit <br /> Materials Methods                                                    polyclonal C-14, 1:10,000 dilution), PAX3 (goat-polyclonal N-19, <br /> Cell Culture, Reagents, Transfections                                1:1000 dilution), SMURF2 (rabbit polyclonal H-50, 1:1000 <br /> Human melanoma cell lines, derived patients pri-        dilution) Santa Cruz Biotechnolgy (Santa Cruz, CA); MITF <br /> mary (888mel) metastatic (A375, WM266-4, SK-Mel2, 501mel,             (mouse monoclonal, C5, 1:500 dilution; Neomarkers, Lab Vision, <br /> and WM164) melanoma lesions, grown Dulbecco&#226;&#8364;&#8482;s modi-              Runcorn, UK); PARP (mouse monoclonal C2-10, #556362, 1:3000 <br /> fied Eagle medium containing 10% fetal calf serum (PAA, Yeovil,          dilution; BD Biosciences, Oxford, UK); cleaved caspase 3, <br /> UK). A375, WM266-4, WM164, SK-Mel2 cells                   SMAD2, phospho-SMAD2, SMAD3, SMAD4 (all rabbit <br /> American Type Culture Collection (LGC, Middlesex, UK), 501mel            polyclonal, 1:1000 dilution) Cell Signaling (Boston, MA). <br /> and 888mel gift Steven Rosenberg (National Cancer <br /> Institute, Bethesda, MD). All cell lines positive mela-      Detection Cell&#194;&#160;Death <br /> noma marker MITF: SK-Mel2 cells harbor NRAS <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span>,             The cellular sub-G1 fraction determined fluorescence- <br /> the melanoma cell lines confirmed harbor BRAF            activated cell sorting using propidium iodide staining stand- <br /> mutation. Human dermal fibroblasts gift Guillaume            ard protocols. Quantification live cells performed crystal <br /> Jacquement (University Manchester, Manchester, UK)                violet toluidine blue staining formalin-fixed cells, subse- <br /> grown Dulbecco&#226;&#8364;&#8482;s modified Eagle medium containing 10%                 quent spectrophotometrical analysis performed measuring <br /> fetal calf serum. Normal human melanocytes (Cascade Biologics,           OD 555 solubilized dye. <br /> Invitrogen, Carlsbad, CA) cultured medium 154 human <br /> melanocyte growth supplement 2 (Cascade Biologics). PD184352             RNA Isolation Quantitative Real-Time Polymerase <br /> was Axon Medchem (Groningen, The Netherlands),                  Chain Reaction (qPCR) Analysis <br /> selumetinib (AZD6244) Selleck Chemicals (Newmarket,             RNA isolated TRIZOL reagent (Invitrogen), and <br /> UK). Transforming growth factor &#206;&#178; (TGF-&#206;&#178;) Sigma                 selected genes amplified qPCR using SYBR green <br /> (St Louis, MO). Cells transfected plasmid DNA using            (Qiagen) incorporation amplification reaction. Primer <br /> Attractene transfection reagent (Qiagen, Valencia, CA)          sequences (forward; reverse) follows: PAX3 (aggat- <br /> small interfering RNAs (siRNAs) using INTERFERin siRNA-                  gcggctgatggaactcactg; ccaggatgatgcggc- <br /> transfection reagent (Polyplus, Illkirch, France) according       cgggccc ggg); MITF (CCGTCTCTCACTGGATTGGT; <br /> manufacturer&#226;&#8364;&#8482;s instructions. For generation A375 cells stably     TACTTGGTGGGGTTTTCGAG); SMURF2 (gcaacaa- <br /> transfected control vector pLKO differ-          ggccaggtgtatt;            accacttgctgttgctgttg); <br /> ent SMAD-specific E3 ubiquitin protein ligase 2 (SMURF2)&#226;&#8364;&#8220;spe-            GAPDH (caatgacccc ttcattgacc; gacaagcttc- <br /> cific small hairpin RNAs (shRNAs), cells transfected       ccgttctcag); &#206;&#178;-actin (Gcaagcaggagt atgac- <br /> respective circular plasmids (set 5 shRNAs, #RHS4533; Open            gag; Caaataaagccatgccaatc). <br /> Biosystems, Huntsville, AL) selected puromycin (1&#194;&#181;g/ <br /> ml) resistance. Clones S2-C4 S2-C14 isolated cell          siRNAs and&#194;&#160;shRNAs <br /> populations transfected different shRNA sequences. For          All siRNAs Thermo Scientific Dharmacon <br /> MEK inhibitor resistance colony formation assay, A375, WM266-            (Loughborough, UK). The sequences individual siRNAs <br /> 4, SK-Mel28 cells transfected circular pEF-MITF            follows: SMURF2#1 (GAUGAGAACACUCCAAUUA), <br /> or vector plasmids. Cells plated 10-cm dishes          SMURF2#2 (GACCAUACCUUCUGUGUUG), SMURF2#3 <br /> incubated 1&#194;&#181;M PD184352 3 weeks formalin            (CAAAGUGGAAUCAGCAUUA),              SMURF2#4        (GAACAAC <br /> fixed, stained crystal violet, photographed. Quantification     ACAAUUUACAGA), MITF#1 (GAACGAAGAAGAAGAUUU <br /> was achieved spectrophotometrical analysis measuring opti-        AUU), MITF#2 (AAAGCAGUACCUUUCUACCAC), MITF#3 <br /> cal density 555&#226;&#8364;&#8240;nm (OD 555)&#194;&#160;of solubilized dye.                   (GACCUAACCUGUACAACAAUU), PAX3#1 (GAAACACCG <br />                                                                          UGCCGUCAGUUU), PAX3#2 (GAGACUGGCUCCAUAC <br /> Immunoblotting                                                           GUCUU) scrambled (nontargeting) control sequence (AAUAUAA <br /> Melanoma cells (3&#226;&#8364;&#8240;&#195;&#8212;&#226;&#8364;&#8240;105) lysed 150-&#194;&#181;L sodium dodecyl sul-        UCACUAUCAGGUGC); S2-p ready-to-use SMURF2 <br /> fate sample buffer (62.5&#226;&#8364;&#8240;mM Tris-hydrochloride [pH 6.8 25&#194;&#176;C],         SMARTpool. The circular plasmids containing SMURF2-specific <br /> 2% weight/volume sodium dodecyl sulfate, 10% glycerol, 50&#226;&#8364;&#8240;mM             shRNAs Open Biosystems (Thermo Fisher, Loughborough, <br />  <br /> 34&#226;&#8364;&#402;Articles&#226;&#8364;&#402;|&#226;&#8364;&#402;JNCI                                                                                          Vol. 105, Issue 1 | January 2, 2012 <br />  UK). S2-sh#1 corresponds clone TRCN0000003475, S2-sh#2         vehicle dimethyl sulfoxide (DMSO). The drug added the <br /> corresponds clone TRCN0000003477.                                  fish medium, larvae grown 34&#194;&#176;C ambient temperature <br />                                                                       chorion water. Before drug addition (day 1)&#194;&#160;and 3&#194;&#160;days drug <br /> Melanoma Three-Dimensional Spheroid&#194;&#160;Growth                            addition, anesthetized larvae (MS222, Sigma) imaged using a <br /> A375 melanoma cells corresponding SMURF2 shRNA                Leica SP5 confocal microscope (Leica Microsystems). Images were <br /> derivatives stably expressing green fluorescent protein resus-   processed using Volocity software (Perkin Elmer, Cambridge, UK.) <br /> pended Dulbecco&#226;&#8364;&#8482;s modified Eagle medium containing 5% <br /> fetal calf serum 1.5% methylcellulose (Sigma). The cell sus-      Xenografts Mice.&#226;&#8364;&#402;     For xenografts, 4&#226;&#8364;&#8240;&#195;&#8212;&#226;&#8364;&#8240;106 A375pLKO A375 <br /> pension transferred 96-well plate (1000 cells well),   S2-C14 cells 0.1&#226;&#8364;&#8240;mL phosphate-buffered saline inoculated <br /> and spheres allowed form period 48 72 hours.     subcutaneously flank female CD1 nude mice <br /> Spheres transferred 0.5-mL fibrillar bovine dermal     (Charles River, Margate, Kent, UK). When mice developed <br /> collagen (2.3&#226;&#8364;&#8240;mg/mL; Nucaton, Leimuiden, The Netherlands)             melanoma nodules approximately 100&#226;&#8364;&#8240;mm3, mice randomly <br /> with sphere 24-well plate. Once collagen    assigned group, drug administration initiated (day <br /> set, Dulbecco&#226;&#8364;&#8482;s modified Eagle medium containing 10% fetal calf       0). Treatment oral gavage daily vehicle (5% DMSO, <br /> serum added, approximately 16 hours, drugs         95% water), 3&#226;&#8364;&#8240;mg 10&#226;&#8364;&#8240;mg/kg body weight selumetinib vehicle. <br /> added medium indicated concentrations. Dead cells      Tumor size determined days 7, 12, 19, 26, 29 calliper <br /> identified staining nonfixed cells ethidium bromide        measurements tumor <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span>, width, depth, volume was <br /> (100&#194;&#160;&#194;&#181;g/mL; Sigma) 10 minutes, followed washes      calculated volume&#194;&#160;=&#194;&#160;0.5236&#194;&#160;&#195;&#8212; length &#195;&#8212; width &#195;&#8212; depth (mm). <br /> phosphate-buffered saline. Green fluorescent protein&#226;&#8364;&#8220;express- <br /> ing spheres monitored, ethidium bromide intensity was <br />                                                                       Statistical Analysis <br /> assessed using Leica DM IL HC inverted microscope a <br />                                                                       All experiments performed minimum times. Data <br /> FC340 Cooled Monocrome camera (Leica Microsysytems). <br />                                                                       represent results assays performed triplicate more, and <br />                                                                       error bars represent 95% confidence intervals (CIs). All statistics <br /> Melanoma cDNA Tissue&#194;&#160;Array <br />                                                                       based continuous variables. Comparisons more <br /> Two melanoma TissueScan cDNA arrays, MERT101 and <br />                                                                       treatment groups using predominately one- <br /> MERT102 (Origene, Rockville, MD), analyzed the <br />                                                                       way analysis variance Tukeys&#226;&#8364;&#8482;s, Dunnett&#226;&#8364;&#8482;s Bonferroni&#226;&#8364;&#8482;s <br /> expression SMURF2 PAX3. The arrays consisted cDNA <br />                                                                       post hoc test multiple comparisons. For comparisons between <br /> derived stage III (n&#194;&#160;=&#194;&#160;36) stage IV (n&#194;&#160;=&#194;&#160;41) melanomas, <br />                                                                       groups, Student&#226;&#8364;&#8482;s t test applied. Drug dose&#226;&#8364;&#8220;response <br /> staged according revised TNM classification mini- <br />                                                                       curves analyzed nonlinear regression curve fit model. <br /> mum stage grouping (19), normal skin (n&#194;&#160; =&#194;&#160; 6), which <br />                                                                       A&#194;&#160;ranking linear regression fit Wald&#226;&#8364;&#8220;Wolfowitz runs test was <br /> was supplemented additional cDNA derived human <br />                                                                       used compare SMURF2 PAX3 expression cDNA <br /> fibroblasts, keratinocytesm, melanocytes. The expression in <br />                                                                       array data. P values than.05 considered statistically <br /> melanocytes set one. The cDNA samples preampli- <br />                                                                       significant. All statistical tests two-sided. All statistical analysis <br /> fied using TaqMan PreAmp Master Mix Kit (PN4384267, <br />                                                                       calculations performed using GraphPad Prism version <br /> Applied Biosystems, Carlsbad, CA) according manufacturer&#226;&#8364;&#8482;s <br />                                                                       4.00 Mac OS (GraphPad Software Inc, San Diego, CA). <br /> instructions. qPCR carried TaqMan Gene Expression <br /> Master Mix (PN4369016, Applied Biosystems). &#206;&#178;-Actin expression <br /> was used normalize relative SMURF2 PAX3 expression. <br /> Experiments performed multiplex PCR reactions using           Results <br /> TaqMan probes (Applied Biosystems) SMURF2 (Hs00224203,            MITF Expression Resistance MEK Inhibitor&#226;&#8364;&#8220; <br /> FAM labelled) PAX3 (Hs00240950, FAM labelled) differ-       Induced Apoptosis Melanoma&#194;&#160;Cells <br /> entially labelled &#206;&#178;-actin probe (Hs99999903, VIC labelled).           To identify melanoma specific proteins allowing survival the <br />                                                                       presence MEK inhibitor, exposed BRAF-mutant A375 mela- <br /> Melanoma Xenografts                                                   noma cells cytotoxic concentrations PD184352 4&#194;&#160;days and <br /> All procedures involving animals approved Animal          isolated individual clones able grow presence <br /> Ethics Committees Institute Cancer Research accord-      drug (Figure&#194;&#160;1, A). Immunoblot analysis individual clones for <br /> ance National Home Office regulations Animals          expression central regulator melanoma cell fate, MITF <br /> (Scientific Procedures) Act 1986 according guidelines      (21&#226;&#8364;&#8220;24), identified increased expression compared parental <br /> of Committee National Cancer Research Institute (20).      cell line A375 (Figure&#194;&#160;1, B). Further analysis confirmed con- <br />                                                                       trast parental A375 cells, A375-R1 A375-R4 cells able <br /> Xenografts Zebrafish.&#226;&#8364;&#402; Zebrafish (Danio rerio) raised         survive presence MEK inhibitor, indicated <br /> and maintained University Manchester Biological             absence increased subG1 fraction MEK inhibi- <br /> Services <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>. Zebrafish xenografts generated injection       tor exposure (Figure&#194;&#160;1, C). However, resistant clones still <br /> of approximately 1000 melanoma cells pericardiac space       responded MEK inhibitor reduced ERK phosphoryla- <br /> surrounding heart embryos 48 hours postfertiliza-    tion (Figure&#194;&#160;1, D) reduced progression S-phase com- <br /> tion. Subsequently, groups larvae condition, randomly      pared parental cells (cells S-phase: R1: 25.6% vs 7.12%, <br /> assigned treated 20&#226;&#8364;&#8240;nM 100&#226;&#8364;&#8240;nM PD184352         difference&#194;&#160;=&#194;&#160;18.48%, 95% CI&#194;&#160;=&#194;&#160;13.53 23.42, P = .002; R4: 27.87% <br />  <br /> jnci.oxfordjournals.org                                                                                                   JNCI&#226;&#8364;&#402;|&#226;&#8364;&#402;Articles&#226;&#8364;&#402;35 <br />  Figure&#194;&#160; 1.&#226;&#8364;&#8218; MITF resistance MEK inhibitor&#226;&#8364;&#8220;induced apoptosis. A)       (pERK) ERK2 (loading control) parental A375 cells (A) MEK <br /> Schematic protocol used selection MEK inhibitor resist-    inhibitor resistant clones (R1, R4) treated DMSO indi- <br /> ant clones. A375 melanoma cells exposed 1&#194;&#160;&#194;&#181;M PD184352 (PD)           cated concentrations PD184352 (PD). E) Quantification cells the <br /> for 4&#194;&#160;days, resulted greater 98% cell death. Surviving cells   S-phase fraction FACS analysis parental A375 cells MEK <br /> were expanded 0.5&#194;&#160;&#194;&#181;M PD184352 additional 3 weeks. Individual       inhibitor&#226;&#8364;&#8220;resistant clones treated DMSO (D) 1&#194;&#181;M PD184352 (PD) <br /> clones grown presence 0.5&#194;&#160; &#194;&#181;M drug isolated.         72 hours. Bars represent means independent experi- <br /> B) Western blot sensitive (S) parental A375 cells MEK inhibitor&#226;&#8364;&#8220;      ments; error bars refer 95% confidence intervals; ***P &lt; .001. A&#194;&#160;two- <br /> resistant clones MITF ERK2 (loading control). C) Quantification      sided Student&#226;&#8364;&#8482;s t test used. F) Parental A375 resistant clones <br /> of cells subG1 fraction fluorescence-activated cell sorting      R1 R4 cultured absence (DMSO) presence 0.5&#194;&#160;&#194;&#181;M <br /> (FACS) analysis parental A375 cells MEK inhibitor&#226;&#8364;&#8220;resistant clones    PD184352 12&#194;&#160;days. Cells fixed, photographed, stained with <br /> treated dimethyl sulfoxide (DMSO) (D) 1&#194;&#181;M PD184352 (PD)          crystal violet. The intensity staining reflects number cells that <br /> 72 hours. Bars represent means independent experiments;           grown various conditions. Individual photographs of <br /> error bars refer 95% confidence intervals; ***P &lt; .001; NS&#194;&#160;=&#194;&#160;P &gt; .05.     cells grown presence PD148352 shown. Scale bar&#194;&#160;=&#194;&#160;50&#194;&#160;&#194;&#181;m. <br /> A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test used. D) Western blot phosphoERK <br />  <br />  <br /> vs 7.7%, difference 20.17%, 95% CI&#194;&#160; =&#194;&#160; 14.93 25.40, P&#194;&#160; &lt;&#194;&#160; .001)           difference&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;0.25, 95% CI&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;0.30 &#226;&#710;&#8217;0.21, P &lt; .001; SKMEL28: <br /> (Figure&#194;&#160;1, E). As consequence, cultured presence            OD 555&#194;&#160;=&#194;&#160;0.105 vs 0.335, difference&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;0.23, 95% CI&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;0.28 &#226;&#710;&#8217;0.18, <br /> drug, resistant cells displayed slow continuous growth,           P &lt; .001; WM2664: OD 555&#194;&#160;=&#194;&#160;0.033 vs 0.155, difference&#194;&#160;=&#194;&#160;0.12, 95% <br /> was seen long-term growth experiment. When, 12&#194;&#160; days              CI&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;0.15 &#226;&#710;&#8217;0.10, P &lt; .001) (Figure&#194;&#160;2, B). These initial findings <br /> of treatment, number cells culture dish assessed          indicated MITF required sufficient produce <br /> staining crystal violet, staining intensity plates contain-      resistance MEK inhibitor&#226;&#8364;&#8220;induced cell&#194;&#160;death. <br /> ing A375-R1 A375-R4 cells considerably higher in <br /> plates containing A375 cells. The difference cell number         Effect SMURF2 MITF Expression MEK Inhibitor <br /> visible cells photographed (Figure&#194;&#160;1, F).                          Resistance <br />     MITF depletion resistant A375-R1 A375-R4 cells           To identify mechanism leading increased MITF expression in <br /> RNA interference (RNAi) sensitized cells PD184352-induced             acquired resistant A375-R1 R4 cells, analyzed cells for <br /> caspase 3 cleavage (Figure&#194;&#160;2, A). Furthermore, ectopic overexpression        expression MITF upstream regulators BRN2 MITF <br /> of MITF different sensitive cell lines increased number         detected increased expression PAX3 (Figure&#194;&#160;3, A). PAX3 <br /> of colonies growing 3 weeks presence MEK inhibitor           positive transcriptional regulator MITF (25) RNAi- <br /> when compared cell lines transfected                 mediated depletion A375 WM266-4 melanoma cells resulted <br /> an vector (OD 555 stained cells: A375: 0.046 vs 0.300,              reduced MITF levels (Supplementary Figure&#194;&#160;1, A, available online). <br />  <br /> 36&#226;&#8364;&#402;Articles&#226;&#8364;&#402;|&#226;&#8364;&#402;JNCI                                                                                                 Vol. 105, Issue 1 | January 2, 2012 <br />  Figure&#194;&#160; 2.&#226;&#8364;&#8218; Effect MITF MEK inhibitor resistance. A) Western blot   MITF expression 24 hours transfection. Note chosen <br /> for MITF, cleaved caspase 3, pERK resistant clones A375-R1     short-film exposure does detect endogenous MITF expression in <br /> and A375-R4 treated scrambled control MITF-specific small       WM266-4 A375 cells. The cells cultured 3 weeks pres- <br /> interfering RNAs (siRNAs) (MI #1, MI #2, MI #3) 48 hours, followed    ence PD184352, resistant colonies stained crystal vio- <br /> by 24-hour exposure 1&#194;&#160; &#194;&#181;M PD184352 (PD) dimethyl sufloxide          let. A&#194;&#160;representative stained cell image shown left graph, <br /> (DMSO [D]). &#206;&#178;-Actin used loading control. B) A375, SK-Mel28,       plots mean number crystal violet&#226;&#8364;&#8220;positive cells three <br /> and WM266-4 cells transfected MITF expression vector         independent assays; error bars refer 95% confidence intervals; ***P <br /> or vector (Vec). The Western blot (upper right panels) shows     &lt; .001. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test used. <br />  <br />  <br />    PAX3 MITF regulation placed downstream                cell death, hypothesized counteracting TGF-&#206;&#178; signalling <br /> TGF-&#206;&#178; melanocytes, SMAD4/SMAD2-containing                      provide mechanism MITF-mediated MEK inhibitor <br /> complex directly suppresses PAX3 expression (26). We           resistance. Indeed, detected statistically significantly (R1: P <br /> described mechanism conserved melanoma cells,                &lt; .001; R4: P &lt; .001) increased RNA expression SMURF2 <br /> treatment melanoma cells TGF-&#206;&#178; suppressed PAX3                (Figure&#194;&#160;3, E), E3 ubiquitin ligase regulates degradation <br /> and MITF RNA expression compared nontreated mela-               activity SMAD proteins (27,28) MEK inhibitor&#226;&#8364;&#8220; <br /> noma cells (normal human melanocytes: relative PAX3 expression         resistant cell lines compared parental A375 cells. We found <br /> untreated cells&#194;&#160;=&#194;&#160;1 vs 0.43, TGF-&#206;&#178;-treated cells, difference&#194;&#160;=&#194;&#160;0.57,   SMURF2 suppresses SMAD2 melanoma cells <br /> 95% CI&#194;&#160; =&#194;&#160; 0.21 0.92, P &lt; .001; relative MITF expression            (Figure&#194;&#160;3, E; Supplementary Figure&#194;&#160;2, A, available online), that <br /> untreated cells&#194;&#160;=&#194;&#160;1 vs 0.17, TGF-&#206;&#178;-treated cells, difference&#194;&#160;=&#194;&#160;0.83,   high SMURF2 levels paralleled high PAX3 expression A375-R1 <br /> 95% CI&#194;&#160;=&#194;&#160;0.48 1.19, P &lt; .001; A375: relative PAX3 expression        A375-R4 cells (Figure&#194;&#160; 3, F). This high PAX3 expression is <br /> untreated cells&#194;&#160;=&#194;&#160;1 vs 0.28 TGF-&#206;&#178;-treated cells, difference&#194;&#160;=&#194;&#160;0.72,    dependent SMURF2 SMURF2 depletion using four <br /> 95% CI&#194;&#160; =&#194;&#160; 0.36 1.07, P &lt; .001; relative MITF expression            individual siRNAs completely abolished increased PAX3 expression <br /> untreated cells&#194;&#160;=&#194;&#160;1 vs 0.64 TGF-&#206;&#178;-treated cells, difference&#194;&#160;=&#194;&#160;0.36,    (Figure&#194;&#160; 3, F). Accordingly, MITF expression severely <br /> 95% CI&#194;&#160;=&#194;&#160;0.05 0.72, P&#194;&#160;=&#194;&#160;0.02) (Figure&#194;&#160;3, B). Treatment A375-R1      reduced (Supplementary Figure&#194;&#160;2, C D, available online). <br /> and R4 cells TGF-&#206;&#178; PD184352 induced caspase 3 cleav-                 Depletion SMURF2 A375-R1 A375-R4 cells did <br /> age (Figure&#194;&#160;3, C). Moreover, treatment melanoma cells         impact viability, additional treatment PD184352 <br /> TGF-&#206;&#178; PD184352 reduced <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>cell survival</span> acquired resistant        induced cell death (Supplementary Figure, 2, B E, available <br /> melanoma cells, approximately 80% reduction cell numbers          online). A&#194;&#160; dose&#226;&#8364;&#8220;response analysis showed SMURF2 deple- <br /> (R1: 78.5%, 95% CI&#194;&#160;=&#194;&#160;75.74% 83.16%, P &lt; .001; R4: 81.5%, 95%           tion RNAi overcomes MEK inhibitor resistance, GI50 <br /> CI&#194;&#160;=&#194;&#160;79.75% 83.07%, P &lt; .001) (Figure&#194;&#160;3, D).                           (concentration MEK inhibitor required reduce cell num- <br />    Because activating TGF-&#206;&#178; signalling suppressed MITF                    ber 50% compared DMSO control) PD184352 in <br /> expression sensitized melanoma cells MEK inhibitor&#226;&#8364;&#8220;induced         A375-R1 A375-R4 cells reduced 3- 10-fold compared <br />  <br /> jnci.oxfordjournals.org                                                                                                       JNCI&#226;&#8364;&#402;|&#226;&#8364;&#402;Articles&#226;&#8364;&#402;37 <br />  Figure&#194;&#160;3.&#226;&#8364;&#8218; Effects transforming growth factor &#206;&#178; (TGF-&#206;&#178;) SMURF2       used. F) Quantitative real-time PCR analysis PAX3 expression <br /> on melanoma cell sensitivity MEK inhibition. A) Western blot sen-     parental A375 cells resistant clones R1 R4 transfected <br /> sitive (S) parental A375 cells MEK inhibitor resistant clones R3, R6,   scrambled control individual SMURF2-specific <br /> R1, R4 MITF, PAX3, ERK2 (loading control). B) Quantitative      small interfering RNAs (siRNAs) (#1&#226;&#8364;&#8220;#4). A375 cells treated a <br /> real-time polymerase chain reaction (PCR) analysis PAX3 MITF         scrambled control siRNA set 1.&#194;&#160; Bars represent means <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> expression normal human melanocytes (NHM) A375 cells          independent experiments; error bars refer 95% confidence <br /> unstimulated (&#226;&#8364;&#8220;) stimulated (transforming growth factor &#206;&#178; [TGF-&#206;&#178;])       intervals. ***P &lt; .001. Two-sided ANOVA Dunnett&#226;&#8364;&#8482;s post hoc test <br /> for 24 hours 5&#226;&#8364;&#8240;ng/ml TGF-&#206;&#178;. Bars represent means inde-      used. G) Dose&#226;&#8364;&#8220;response curve cell survival presence of <br /> pendent experiments; error bars refer 95% confidence intervals;          PD184352. A375 cells resistant clones R1 R4 treated <br /> ***P &lt; .001; *P = .02. A&#194;&#160; two-sided Student&#226;&#8364;&#8482;s t used. C) Western        scrambled control SMURF2-specific siRNA (siS2#1) 48 <br /> blot A375-R1 A375-R4 cells cleaved caspase 3, pERK,and           hours, followed indicated concentrations PD184352 24 <br /> ERK2 (loading control). The cells unstimulated (&#226;&#8364;&#8220;) stimu-    hours fixed, stained toluidine blue, quanti- <br /> lated (TGF-&#206;&#178;) 24 hours 5&#226;&#8364;&#8240;ng/ml TGF-&#206;&#178; presence dime-      fied measuring optical density 555&#226;&#8364;&#8240;nm (OD 555)&#194;&#160; solubilized <br /> thyl sulfoxide (DMSO [D]) 2&#194;&#160;&#194;&#181;M PD184352 (PD). D) Quantification       dye. For cell line, DMSO-treated cells set 100%, and <br /> cell survival. A375-R1 A375-R4 cells treated TGF-&#206;&#178; 48     percentage cell survival defined drug concentrations deter- <br /> hours, followed 24-hour exposure 2&#194;&#160;&#194;&#181;M PD184352 (PD) DMSO           mined. Drug dose&#226;&#8364;&#8220;response curves analyzed nonlinear <br /> (D), cell number assessed toluidine staining. DMSO-       regression curve fit model. H) A375-pLKO small hairpin SMURF2- <br /> treated cells set 100%. Bars represent means inde-          expressing A375 cells (clone S2&#226;&#8364;&#8220;C14; Supplementary Figure&#194;&#160; 3, <br /> pendent experiments; error bars refer 95% confidence intervals;          available online) cultured 3 weeks presence 0.5&#194;&#160;&#194;&#181;M <br /> ***P &lt; .001. Two-sided analysis variance (ANOVA) Bonferroni         PD184352, resistant colonies stained crystal violet, and <br /> post hoc test used. E) Quantitative real-time PCR Western blot      OD 555 quantified. Values represent results inde- <br /> analysis SMURF2 expression parental A375 cells resistant      pendent experiments. ***P &lt; .001. A&#194;&#160; two-sided Student&#226;&#8364;&#8482;s t test was <br /> clones R1 R4. SMURF2 expression A375 cells set 1.&#194;&#160;Bars        used. Black circles represent cells stably transfected empty <br /> represent means independent experiments; error bars refer        pLKO vector, black triangles represent cells stably transfected a <br /> to 95% confidence intervals; ***P &lt; .001. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test      SMURF2&#226;&#8364;&#8220;small hairpin RNA&#226;&#8364;&#8220;expressing vector. <br />  <br /> 38&#226;&#8364;&#402;Articles&#226;&#8364;&#402;|&#226;&#8364;&#402;JNCI                                                                                              Vol. 105, Issue 1 | January 2, 2012 <br />  Figure&#194;&#160;4.&#226;&#8364;&#8218; MITF primary resistance MEK inhibition. A) DNA content       NS&#194;&#160;=&#194;&#160;P &gt; .05; ***P &lt; .001. Two-sided ANOVA Tukey&#226;&#8364;&#8482;s post hoc test was <br /> fluorescence-activated cell sorting (FACS) analysis (assessing fractions    used. A&#194;&#160;Western blot showing MITF expression ERK2 loading con- <br /> cells G1, S, G2/M, subG1 phase) indicated cell lines treated      trol shown. D) Western blot MITF, cleaved caspase 3, phospho- <br /> with 1&#194;&#181;M PD184352 (PD) dimethyl sulfoxide (DMSO) 72 hours.              ERK (pERK) 888mel 501mel cells treated scrambled control <br /> Western blots ERK phosphorylation (pERK) concentrations 0.2           MITF-specific small interfering RNAs (siRNAs) (MI#1&#226;&#8364;&#8220;#3) 48 hours, <br /> and 1&#194;&#160;&#194;&#181;M PD184352. ERK2 serves loading control. B) Quantification        followed 24-hour exposure 1&#194;&#160;&#194;&#181;M PD184352 (PD) DMSO (D). ERK2 <br /> cells subG1 fraction FACS analysis indicated melanoma       used loading control. E) Quantification cell survival. 501mel <br /> cell lines (R&#194;&#160;=&#194;&#160;resistant, S&#194;&#160;=&#194;&#160;sensitive) treated 1&#194;&#181;M PD184352 72     cells treated scrambled control MITF-specific siRNAs (MI#2 <br /> hours. Bars represent means independent experiments; error          #3) 48 hours, followed 24-hour exposure 0.5&#194;&#160;&#194;&#181;M PD184352 <br /> bars refer 95% confidence intervals. NS&#194;&#160; =&#194;&#160; P &gt; .05; ***P &lt; .001. Two-      (PD) DMSO (D). They fixed stained toluidine blue, <br /> sided analysis variance (ANOVA) Tukey&#226;&#8364;&#8482;s post hoc test used.        optical density 555&#226;&#8364;&#8240;nm quantified. DMSO-treated cells <br /> C) Quantification toluidine staining live cells indicated mela-   set 100%, treatment 5&#194;&#160;&#194;&#181;M PD184352 used posi- <br /> noma cell lines treated 3&#194;&#160;&#194;&#181;M PD184352 24 hours DMSO only.          tive control. Bars represent means independent experiments; <br /> DMSO-treated cells set 100%. Bars represent means           error bars refer 95% confidence intervals; NS&#194;&#160;=&#194;&#160;P &gt; .05; ***P &lt; .001. <br /> independent experiments; error bars refer 95% confidence intervals.         Two-sided ANOVA Tukey&#226;&#8364;&#8482;s post hoc test used. <br />  <br />  <br /> with A375 cells (A375: 95% CI [GI50]&#194;&#160;=&#194;&#160;1.47 2.66&#194;&#160;&#194;&#181;M; R1/siS2:               555 stained cells, pLKO vs shSMURF: 0.3872 vs 0.1030, differ- <br /> 95% CI [GI50]&#194;&#160;=&#194;&#160;0.53 0.72&#194;&#160;&#194;&#181;M; R4/siS2: 95% CI [GI50]&#194;&#160;=&#194;&#160;0.11                 ence&#194;&#160;=&#194;&#160;0.28, 95% CI&#194;&#160;=&#194;&#160;0.21 0.36, P &lt; .001) (Figure&#194;&#160;3, H). <br /> to 0.18&#194;&#160;&#194;&#181;M) (Figure&#194;&#160;3, G). Most important, stable expression a <br /> SMURF2-specific shRNA A375 cells (Supplementary Figure&#194;&#160;3,                   MITF Expression Primary Resistance MEK <br /> available online) suppressed development acquired MEK                   Inhibitor&#226;&#8364;&#8220;Induced Cytotoxicity <br /> inhibitor resistance, number PD184352-resistant colo-               We extended study primary (innate) resistance to <br /> nies derived small hairpin SMURF2 (shSMURF2) cells                  MEK inhibition BRAF-mutant melanoma cells. High-MITF- <br /> 3 weeks drug exposure severely reduced compared                    expressing cells (WM164, 888mel, 501mel), repre- <br /> with vector pLKO cells treated MEK inhibitor (OD                    sentative certain cohorts melanomas (29), responded to <br />  <br /> jnci.oxfordjournals.org                                                                                                          JNCI&#226;&#8364;&#402;|&#226;&#8364;&#402;Articles&#226;&#8364;&#402;39 <br />  PD184352 MEK inhibition low doses inhibitor            affected SMURF2 depletion (Supplementary Figure&#194;&#160; 1, <br /> (0.2&#194;&#160;&#194;&#181;M), detectable reduced ERK phosphorylation (Figure&#194;&#160;4,         C, available online), rescued MEK inhibitor effect the <br /> A). However, PD184352 induced cytostatic effect           cell number (A375, S2 siRNA transfected PD184352 treated vs <br /> high-MITF-expressing cells, produced cell death low-MITF-        MITF-A375, S2 siRNA transfected PD184352 treated: 13.47% vs <br /> expressing cell lines (WM266-4, A375, SKMel-2), seen         81.18%, difference&#194;&#160;=&#194;&#160;67.71%, 95% CI&#194;&#160;=&#194;&#160;73.95% 61.47%, P &lt; <br /> in increased subG1 fraction decreased cell numbers           .001) (Figure&#194;&#160; 5, F). This suggests reduced MITF expression <br /> SKMel-2, A375 WM266-4 cells (Figure&#194;&#160;4, A&#226;&#8364;&#8220;C).                       responsible cell death induced SMURF2 depletion in <br />     As seen acquired resistant cells, depletion MITF       PD184352-treated cells. <br /> from high-MITF- expressing primary resistant cell lines (888mel <br /> and 501mel cells) resulted sensitization PD184352, led     SMURF2 Expression Melanoma Effect Its <br /> to activation apoptotic program caspase 3 cleav-     Depletion Melanoma&#194;&#160;Cells <br /> age (Figure&#194;&#160;4, D) reduced cell numbers low concentrations       In panel melanoma cell lines, SMURF2 depletion no <br /> of drug (Figure&#194;&#160;4, E).                                                 major effect cell viability (Figure&#194;&#160; 6, A). However, compared <br />                                                                        single treatments, combined targeting SMURF2 MEK <br /> SMURF2 Depletion MEK Inhibitor&#226;&#8364;&#8220;Induced                             produced statistically significant (P &lt; .001) cell death, nota- <br /> Cytotoxicity                                                           bly primary resistant cell lines (Figure&#194;&#160;6, A). On other <br /> When analyzed primary MEK inhibitor&#226;&#8364;&#8220;resistant melanoma              hand, SMURF2 depletion noticeable effect melanocytes <br /> cells PAX3, increased expression resistant         (Figure&#194;&#160; 6, B). We did observe major effect cell <br /> cell lines, paralleled increased MITF expression (Figure&#194;&#160;5,      survival SMURF2 depleted MEK inhibited non- <br /> A). Furthermore, line PAX3 acting upstream MITF,            MITF-expressing BRAF-mutant colon cancer cells (Figure&#194;&#160;6, <br /> PAX3 depletion A375 cells enhanced proapoptotic               C), supporting melanoma-specific role SMURF2. When <br /> effect PD184352 detectable increased caspase 3 cleavage          melanoma cells grown spheres three-dimensional colla- <br /> (Supplementary Figure&#194;&#160; 1, B, available online). In addition, PAX3      gen, SMURF2 depletion enhanced PD184352-induced cell death, <br /> depletion reduced number living cells presence   visible ethidium bromide uptake (Figure&#194;&#160; 6, D). Furthermore, <br /> of PD184352 (A375 control siRNA transfected, PD184352 treated          targeting SMURF2 fibroblasts did stimulate tumor-promot- <br /> vs A375 PAX3 siRNA transfected, PD184352 treated: 75.12% vs            ing effects produced TGF-&#206;&#178; (30), increased <br /> 44.93%, difference&#194;&#160;=&#194;&#160;30.19%, 95% CI&#194;&#160;=&#194;&#160;27.50% 41.80%, P &lt;            expression VEGF SPARC (Figure&#194;&#160; 6, E), reveals an <br /> .001) (Figure&#194;&#160; 5, B). However, ectopic expression MITF res-         important difference SMURF2 function TGF-&#206;&#178; <br /> cued A375 cells PAX3 targeted RNAi MEK             stimulated signalling. <br /> by PD184352 (A375 PAX3 siRNA transfected, PD184352 treated                 In advanced stage III IV melanomas, SMURF2 RNA <br /> vs MITF-A375 PAX3 siRNA transfected PD184352 treated:              expression statistically significantly increased compared with <br /> 44.93% vs 85.07%, difference&#194;&#160; =&#194;&#160; 40.15%, 95% CI&#194;&#160; =&#194;&#160; 45.69%          normal skin (mean different skin samples primary skin cells <br /> 34.60%, P &lt; .001) (Figure&#194;&#160;5, B). On hand, MITF          [normal melanocytes&#194;&#160;=&#194;&#160;1] 1.05-fold vs stage III 15.40-fold, differ- <br /> was depleted RNAi ectopic MITF-expressing A375 cells,       ence&#194;&#160;=&#194;&#160;14.35, 95% CI&#194;&#160;=&#194;&#160;21.91-fold 6.80-fold, P &lt; .001; normal <br /> became responsive PD184352, cell number reduced         1.05-fold vs stage IV 8.01-fold, difference&#194;&#160;=&#194;&#160;6.96, 95% CI&#194;&#160;=&#194;&#160;10.69- <br /> (Figure&#194;&#160;5, B).                                                         fold 3.25-fold, P &lt; .001) (Figure&#194;&#160;6, F). There correlation <br />     TGF-&#206;&#178; induced caspase 3 cleavage response presence       (r2&#194;&#160;=&#194;&#160;0.788) SMURF2 PAX3 expression (Figure&#194;&#160;6, G), <br /> of PD184352 (Figure&#194;&#160; 5, C). Importantly, caspase 3 cleavage       suggesting regulatory link proteins is <br /> in response TGF-&#206;&#178; seen MEK inhibitor&#226;&#8364;&#8220;sensi-         preserved tumors. <br /> tive WM266-4 cells MEK inhibitor&#226;&#8364;&#8220;resistant 501mel <br /> cells (Figure&#194;&#160;5, C). The action TGF-&#206;&#178; PD184352         Effect SMURF2 Depletion MEK Inhibition In&#194;&#160;Vivo <br /> reflected effects cell survival, notably MEK inhibi-     In approach allows rapid assessment cytotoxic effects <br /> tor&#226;&#8364;&#8220;resistant melanoma cells (Figure&#194;&#160;5, D).                            in vivo setting, MEK inhibitor&#226;&#8364;&#8220;resistant 501mel MEK <br />     When analyzed panel melanoma cells, sta-        inhibitor&#226;&#8364;&#8220;sensitive A375 cells injected zebrafish <br /> tistically significantly (P = .001) increased SMURF2 expression     embryos (Figure&#194;&#160;7, A; Supplementary Figure&#194;&#160;4, available online). <br /> the primary MEK inhibitor&#226;&#8364;&#8220;resistant cell lines compared       Both, pLKO-control SMURF2-depleted cells (ie, cells sta- <br /> MEK inhibitor&#226;&#8364;&#8220;sensitive cell lines. (Figure&#194;&#160;5, E). When SMURF2         bly expressing SMURF2-specific shRNAs; Figure&#194;&#160; 3, H) <br /> was depleted RNAi melanoma cell lines, resulted          grew statistically significant difference (P &gt; .05) size over <br /> reduced PAX3 MITF expression (Supplementary Figures 2              3&#194;&#160;days (Figure&#194;&#160;7, A; Supplementary Figure&#194;&#160;4, B, available online). <br /> and 3, available online), reduced PAX3 MITF            Xenografts formed sensitive A375 cells showed statistically <br /> expression sensitizes melanoma cells MEK inhibition (Figure&#194;&#160;1,      significant tumor growth period 3&#194;&#160;days (day 1 day 4)&#194;&#160;at <br /> Figure&#194;&#160; 2, A, Figure&#194;&#160; 5, B), SMURF2 depletion strongly increased       20nM PD184352 (P &gt; .05) (Supplementary Figure&#194;&#160;4, B, available <br /> cell death melanoma cells presence PD184352               online). Resistant 501mel xenografts grew presence <br /> (A375,scrambled control siRNA transfected, PD184352 treated            100&#226;&#8364;&#8240;nM drug (Figure&#194;&#160;7, A), additional SMURF2 depletion <br /> vs A375, S2 siRNA transfected PD184352 treated: 79.58% vs              reduced xenograft volume 0.5-fold initial <br /> 13.47%, difference&#194;&#160;=&#194;&#160;66.11%, 95% CI = 57.27% 74.94%, P &lt;            size measured day 1 (DMSO: pLKO 3.26-fold vs shSMURF2 <br /> .001) (Figure&#194;&#160;5, F). However, ectopic expression MITF,        2.78-fold size increase compared day 1, difference&#194;&#160; =&#194;&#160; &#226;&#710;&#8217;0.48, <br />  <br /> 40&#226;&#8364;&#402;Articles&#226;&#8364;&#402;|&#226;&#8364;&#402;JNCI                                                                                          Vol. 105, Issue 1 | January 2, 2012 <br />  Figure&#194;&#160;5.&#226;&#8364;&#8218; SMURF2 confers resistance MEK inhibition melanoma           toluidine blue, optical density 555&#226;&#8364;&#8240;nm quantified. <br /> cells. A) Western blot indicated cell lines MITF, PAX3,       DMSO-treated cells set 100%. Bars represent means from <br /> ERK2 (loading control). R&#194;&#160;=&#194;&#160;resistant, S&#194;&#160;=&#194;&#160;sensitive. B) Quantification   independent experiments; error bars refer 95% confidence <br /> cell survival. A375 MITF-A375 cells (stably expressing MITF         intervals; ***P &lt; .001. Two-sided ANOVA Tukey&#226;&#8364;&#8482;s post hoc test was <br /> an ectopic promoter) treated scrambled control, PAX3-            used. E) Quantitative real-time polymerase chain reaction Western <br /> (PAX3#1) MITF (MI#1)-specific small interfering RNAs (siRNAs),            blot analysis SMURF2 expression melanoma cell lines. The <br /> for 48 hours, followed 24-hour treatment 1&#194;&#160; &#194;&#181;M PD184352           expression SMURF2 shown fold expression compared with <br /> dimethyl sulfoxide (DMSO). They stained toluidine             normal human melanocytes (NHM), SMURF2 expression was <br /> blue, optical density 555&#226;&#8364;&#8240;nm quantified. DMSO-treated         set 1.&#194;&#160;Bars represent means independent experiments; <br /> cells set 100%. Bars represent means independ-            error bars refer 95% confidence intervals; ***P &lt; .001. Two-sided <br /> ent experiments; error bars refer 95% confidence intervals; ***P &lt;        ANOVA Dunnett&#226;&#8364;&#8482;s post hoc test used. F) Quantification of <br /> .001. Two-sided analysis variance (ANOVA) Tukey&#226;&#8364;&#8482;s post hoc test      cell survival. A375 MITF-A375 cells treated scrambled <br /> was used. C) Western blot WM266-4 501mel cells cleaved            control (SC) SMURF2 pooled siRNAs (S2-p) 48 hours, followed <br /> caspase 3, pERK, ERK2 (loading control). The cells           24-hour exposure 1&#194;&#160;&#194;&#181;M PD184352 DMSO, quantified using <br /> unstimulated (&#226;&#8364;&#8220;) stimulated (transforming growth factor &#206;&#178; [TGF&#226;&#8364;&#8220;           toluidine blue. DMSO-treated cells set 100%. Bars represent <br /> &#206;&#178;]) 24 hours 5&#226;&#8364;&#8240;ng/ml TGF-&#206;&#178; presence DMSO (D)           means independent experiments; error bars refer 95% <br /> 1&#194;&#160; &#194;&#181;M PD184352 (PD). D) Quantification cell survival. The indicated       confidence intervals; NS&#194;&#160;=&#194;&#160;P &gt; .05; ***P &lt; .001. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s <br /> cell lines treated TGF-&#206;&#178; 48 hours, followed 24-hour         t test used. <br /> exposure 1&#194;&#160;&#194;&#181;M PD184352 (PD) DMSO (D). They stained <br />  <br />  <br /> 95% CI&#194;&#160; =&#194;&#160; &#226;&#710;&#8217;0.22 1.19, P &gt; 0.05; PD184352: PLKO 1.90-fold                 95% CI&#194;&#160; =&#194;&#160; 1.60 1.81&#194;&#160; &#194;&#181;M; S2C4: (GI50)&#194;&#160; =&#194;&#160; 26.62&#226;&#8364;&#8240;nM, 95% <br /> vs shSMURF2 0.57-fold size increase compared day 1, dif-                CI&#194;&#160;=&#194;&#160;20.42 36.73&#226;&#8364;&#8240;nM; S2C14 (GI50): 23.3&#226;&#8364;&#8240;nM, 95% CI&#194;&#160;=&#194;&#160;17.24 <br /> ference&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;1.33, 95% CI&#194;&#160;=&#194;&#160;&#226;&#710;&#8217;0.64 &#226;&#710;&#8217;2.05, P &lt; .001) (Figure&#194;&#160;7, A).           31.60) (Figure&#194;&#160; 7, B). In mice, selumetinib reduces xenograft <br /> A&#194;&#160;similar effect observed A375 xenografts (Supplementary             growth doses 25 30&#226;&#8364;&#8240;mg/kg twice daily (9,31). However <br /> Figure&#194;&#160;4, B, available online).                                              pharmacokinetic analyses revealed dose 10&#226;&#8364;&#8240;mg/kg once <br />    SMURF2 depletion led approximately 100-fold enhanced                day produces drug exposures mice similar achieved <br /> sensitivity melanoma cells selumetinib, MEK inhibitor                clinical use (32). Therefore, used doses 3 10&#226;&#8364;&#8240;mg/ <br /> that produces limited responses melanoma patients (3) (A375:              kg daily, did statistically significant effect <br /> (GI50)&#194;&#160;=&#194;&#160;2.2&#194;&#160;&#194;&#181;M, 95% CI&#194;&#160;=&#194;&#160;2.114 2.286; pLKO: (GI50)&#194;&#160;=&#194;&#160;1.69&#194;&#160;&#194;&#181;M,            (P&#194;&#160;&gt;&#194;&#160;.05) growth parental A375-pLKO tumors (Figure&#194;&#160;7, <br />  <br /> jnci.oxfordjournals.org                                                                                                      JNCI&#226;&#8364;&#402;|&#226;&#8364;&#402;Articles&#226;&#8364;&#402;41 <br />  Figure&#194;&#160;6.&#226;&#8364;&#8218; SMURF2 overexpression specificity targeting SMURF2         test used. D) Spheres green fluorescent protein&#226;&#8364;&#8220;expressing A375 <br /> melanoma cells. A) Quantification cell survival. The indicated cell lines    cells, A375-pLKO-expressing A375 cells, small hairpin SMURF 2(shS- <br /> were treated scrambled control SMURF2-specific small interfering        MURF2)&#226;&#8364;&#8220;expressing A375 cells (clone S2&#226;&#8364;&#8220;C14) three-dimensional col- <br /> RNAs (siRNAs) (siSMURF2#1) 48 hours, followed 24-hour exposure           lagen treated DMSO PD184352 3&#194;&#160;days. Nonfixed spheres <br /> to dimethyl sulfoxide (DMSO) 1&#194;&#160; &#194;&#181;M PD184352. They             stained ethidium bromide (EtBr), images taken. Scale <br /> were stained toluidine blue, optical density 555&#226;&#8364;&#8240;nm         bar&#194;&#160;=&#194;&#160;200&#194;&#160;&#194;&#181;m. E) Quantitative real-time polymerase chain reaction (PCR) <br /> quantified. DMSO-treated cells set 100%. Bars represent means           analysis VEGF SPARC expression primary fibroblasts either <br /> from independent experiments; error bars refer 95% confidence          untreated (&#226;&#8364;&#8220;) treated (transforming growth factor [TGF-&#206;&#178;]) 5&#226;&#8364;&#8240;ng/ml <br /> intervals; ***P &lt; .001.Two-sided analysis variance (ANOVA) withTukey&#226;&#8364;&#8482;s       TGF-&#206;&#178; transfected scrambled control individual SMURF2- <br /> post hoc test used. B) Quantification cell survival. Normal melano-      specific siRNAs (S2#1 #2). Bars represent means independ- <br /> cytes (NHM) treated scrambled control SMURF2 pool              ent experiments; error bars refer 95% confidence intervals; NS&#194;&#160;=&#194;&#160;P &gt; .05; <br /> of 4 (S2-p) siRNAs 48 hours, followed 24-hour exposure 1&#194;&#160;&#194;&#181;M           ***P &lt; .001. A&#194;&#160;two-sided Dunnett&#226;&#8364;&#8482;s t test used. F) Quantitative real- <br /> PD184352 (PD) DMSO (D). They then fixed stained           time PCR analysis SMURF2 expression normal skin cells stage III <br /> toluidine blue quantified. DMSO-treated cells set 100%. Bars        (n&#194;&#160;=&#194;&#160;36) stage IV (n&#194;&#160;=&#194;&#160;41) melanomas. The expression normalized <br /> represent means independent experiments; error bars refer            &#206;&#178;-actin. Box plots mean values SMURF2 expression shown. <br /> to 95% confidence intervals; NS&#194;&#160; =&#194;&#160; P &gt; .05. A&#194;&#160; two-sided Student&#226;&#8364;&#8482;s t test      Normal&#194;&#160;=&#194;&#160;normal skin (n&#194;&#160;=&#194;&#160;6), supplemented additional cDNA derived <br /> was used. C) Quantification cell survival. The colon cancer cell lines       human fibroblasts, keratinocytes, melanocytes. The expression <br /> RKO HT29 treated scrambled control SMURF2-               melanocytes set 1.&#194;&#160;***P &lt; .001. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test to <br /> specific siRNAs (#1 #2) 48 hours, followed 24-hour exposure       compare expression stage III stage IV melanomas normal was <br /> 1&#194;&#160;&#194;&#181;M PD184352 DMSO. They fixed stained toluidine          used. G) Linear regression analysis relative SMURF2 PAX3 expres- <br /> blue quantified. DMSO-treated cells set 100%. Bars repre-           sion levels stage III melanomas. Goodness Fit: r2 + 0.7883. A&#194;&#160;ranking <br /> sent means independent experiments; error bars refer 95%          linear regression fit Wald&#226;&#8364;&#8220;Wolfowitz runs test used compare <br /> confidence intervals; NS&#194;&#160; =&#194;&#160; P &gt; .05; ***P&#194;&#160; &lt;&#194;&#160; .001. A&#194;&#160; two-sided Student&#226;&#8364;&#8482;s t   SMURF2 PAX3 expression cDNA array data. <br />  <br />  <br /> 42&#226;&#8364;&#402;Articles&#226;&#8364;&#402;|&#226;&#8364;&#402;JNCI                                                                                                     Vol. 105, Issue 1 | January 2, 2012 <br />  Figure&#194;&#160;7.&#226;&#8364;&#8218; SMURF2 depletion sensitizes melanoma cells MEK inhibition       26, 29 groups mice error bars represent 95% <br /> in vivo. A) RFP-expressing 501mel pLKO small hairpin SMURF2 (shS-          confidence intervals. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test used calculate <br /> MURF2) cells injected zebrafish larvae 48 hours post-     P values day 29. D) Nude mice bearing tumors SMURF2-depleted <br /> fertilization, larvae treated 100&#226;&#8364;&#8240;nM PD184352        A375-S2-C14 cells treated vehicle (DMSO) selu- <br /> or vehicle dimethyl sulfoxide (DMSO). Three days drug addi-         metininb (3 10&#226;&#8364;&#8240;mg/kg daily) 29 consecutive days. The results <br /> tion, xenografts imaged. Scale bar&#194;&#160;=&#194;&#160;36&#194;&#160;&#194;&#181;m. Quantification        mean volumes measured days 7, 12, 19, 26, 29 groups <br /> xenograft volumes (n&#194;&#160;=&#194;&#160;19) performed drug addition day          mice error bars represent 95% confidence intervals. <br /> 1 day 4 using Volocity software. Fold xenograft size indicates     A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test used calculate P values day 29. <br /> change volume day 1 day 4.&#194;&#160; Black squares represent            E) Quantification tumor volume day 29. Results shown represent a <br /> individual fish injected pLKO-501mel cells; black triangles repre-       single experiment mice group treated. Bars rep- <br /> sent individual fish injected 501mel-shSMURF2 cells. The horizontal      resent means group mice; error bars refer 95% confi- <br /> bar indicates mean value fold xenograft size respective     dence intervals; **P = .003; *P = .03; NS&#194;&#160;=&#194;&#160;P &gt; .05. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s <br /> group zebrafish. A&#194;&#160;two-sided Student&#226;&#8364;&#8482;s t test used calculate P      t test used. The horizontal line 100&#226;&#8364;&#8240;mm3 indicates mean tumor <br /> values; NS&#194;&#160; =&#194;&#160; P &gt; .05; ***P &lt; .001. B) Dose&#226;&#8364;&#8220;response curve cell sur-      volume beginning drug treatment. F) Model survival and <br /> vival presence selumetinib. The indicated cell lines treated   growth regulation melanoma cells. The cooperation MEK MITF <br /> with indicated concentrations selumetinib 24 hours     required efficient survival. MEK predominantly regulates cell cycle <br /> were fixed, stained toluidine blue, quantified. DMSO-treated         progression ERK activates survival signals. MITF con- <br /> cells set 100%, drug dose&#226;&#8364;&#8220;response curves analyzed           tributes regulation cell cycle progression required to <br /> with nonlinear regression curve fit model. C) Nude mice bearing             suppress apoptosis. SMURF2 depletion reduced cell cycle <br /> tumors A375-pLKO cells (empty vector) treated           progression, entirely line decreased MITF expression <br /> vehicle (DMSO) selumetininb (3 10&#226;&#8364;&#8240;mg/kg daily) 29 consec-      (37). Under conditions, MITF levels appear sufficient pro- <br /> utive days. The results mean volumes measured days 7, 12, 19,         vide survival signals long MEK fully active. <br />  <br />  <br />  <br />  <br /> jnci.oxfordjournals.org                                                                                                          JNCI&#226;&#8364;&#402;|&#226;&#8364;&#402;Articles&#226;&#8364;&#402;43 <br />  C E). However, growth tumors derived SMURF2-                        We shown previously MITF contributes cell cycle <br /> depleted A375 cells greatly reduced dosing regimens,            progression downstream BRAF (37), it <br /> with 10&#226;&#8364;&#8240;mg/kg selumetinib completely suppressing tumor               cooperates MEK provide survival signals (Figure&#194;&#160;7, F). <br /> growth 3&#226;&#8364;&#8240;mg/kg resulting approximately 77% reduction              Although MEK activate survival signals ERK (7,8), <br /> tumor size compared tumors control mice treated            cooperation MEK MITF required for <br /> vehicle DMSO (shSMURF2/3mg/kg: difference&#194;&#160; =&#194;&#160; 77.15%, 95%                   survival. In context, identified SMURF2 regulator of <br /> CI&#194;&#160; =&#194;&#160; 10.62% 143.72%, P = .03; shSMURF2/10mg/kg: differ-                MITF expression PAX3 (Figure&#194;&#160;7, F). SMURF2 depletion <br /> ence&#194;&#160;=&#194;&#160;97.9%, 95% CI&#194;&#160;=&#194;&#160;38.65% 155.50%, P = .005) (Figure&#194;&#160;7,              reduced cell cycle progression (Supplementary Figure&#194;&#160;2, B, <br /> D E).                                                                   available online), entirely line decreased MITF <br />                                                                             expression. However MITF levels appear sufficient to <br />                                                                             provide survival signals long MEK fully active because <br /> Discussion                                                                  observed effects cell viability SMURF2 depletion <br /> The efficacy anticancer drugs closely linked            alone. Thus, SMURF2 itself relevant mela- <br /> cytotoxic potential. Cytostatic drugs stop tumor growth,           noma cells MEK inhibited. Importantly, found <br /> which increases risk development acquired resist-            depleting SMURF2 melanocytes did any <br /> ance allows survival selection cells         effect growth survival. This suggests targeting <br /> are able continue proliferate presence drug.            SMURF2 normal cells does affect viability growth prob- <br /> Therefore, enhancing cytotoxic potential drug             ably compensatory mechanisms, assumption that <br /> only improve tumor response, reduce emer-          supported fact SMURF2-/- mice viable <br /> gence acquired resistance. We increased expression         develop healthy normal adulthood (28). These mice <br /> MITF, PAX3, SMURF2 allows melanoma cells escape                  eventually develop tumors; however, occurs long <br /> proapoptotic effects MEK inhibition. We identified SMURF2             latency age 20&#194;&#160; months (38,39), suggesting <br /> a regulator PAX3 expression and, consequently, MITF expres-              loss SMURF2 function does produce immediate <br /> sion demonstrate exploiting mechanism dramatically            adverse effects. <br /> enhances cytotoxic potential MEK inhibitors PD184352                 SMURF2 HECT-domain E3 ubiquitin ligase, reg- <br /> (CI-1040) selumetinib (AZD6244) vitro in&#194;&#160;vivo.                   ulates turnover inhibitory SMADs associated TGF-&#206;&#178; <br />     In principle, MEK inhibitors exert cytotoxic effects             receptors regulatory SMADs (27). SMURF2 regu- <br /> directly activating apoptotic programme (6&#226;&#8364;&#8220;8,33). However,              lates cell polarity, motility, senescence SMAD- independ- <br /> for occur, high drug concentrations long-term drug              ent manner (40&#226;&#8364;&#8220;43). These findings suggest SMURF2 plays an <br /> exposure required, suggesting MEK&#226;&#8364;&#8482;s contribution            important role various aspects relevant tumor initiation and <br /> regulation apoptosis major. Rather, appears pre-      progression SMAD-dependent SMAD-independent <br /> dominant function MEK regulate cell cycle progression              manner. We SMURF2 regulates SMAD2 levels mela- <br /> through ERK, agreement observations loss              noma cells (Figure&#194;&#160;3, E; Supplementary Figure&#194;&#160;2, available online). <br /> of ERK phosphorylation leads G1 arrest,              However, SMAD2 shown regulate sup- <br /> the induction apoptosis (7,10). In line findings,          pression PAX3 melanocytes (26), complete mechanism as <br /> identified cells responded MEK inhibitor PD184352               SMURF2 suppresses PAX3 clear involve <br /> with target inhibition reduced proliferation able        additional factors SMAD2. We detected high <br /> to propagate long term, situation likely          expression levels SMURF2 tumors compared normal <br /> tumor. We cells resistant proapoptotic          tissue (42,44), suggesting deregulation ubiquitin ligase in <br /> effects MEK inhibition that, ERK activity           cancer. On hand, neutralization SMURF2 activity <br /> low, sufficient promote cell cycle progression. This find-   ubiquitin-specific peptidase USP15 important TGF- <br /> ing great relevance highlights challenge tar-       &#206;&#178;-mediated glioblastoma progression (45), suggesting tumor-spe- <br /> geting MAP-kinase pathway single-agent treatment. It is,           cific differences SMURF2 function. A&#194;&#160; tumor-specific function <br /> therefore, great importance identify factors predict        supported finding SMURF2 depletion colon <br /> and enhance sensitivity MEK inhibitors.                     cancer cells did prime cells MEK inhibitor cytotoxic- <br />     We identified melanocyte-specific transcription factor              ity despite mutant BRAF status. <br /> MITF predictive marker sensitivity melanoma cells               We identified SMURF2 potential therapeutic target in <br /> toward cytotoxic effect MEK inhibitors. We high           melanoma. However, clear limitations study. First <br /> MITF expression protects cells MEK inhibitor&#226;&#8364;&#8220;induced cell              used xenograft assays assess combined therapy vivo, and <br /> death, striking considering MITF amplified        approach does consider heterogeneity real <br /> 20% melanomas (34). MITF central regulator melanoma              tumor situation. Second, SMURF2 inhibitor available, we <br /> cell fate controls differentiation, proliferation, inva-     rely knock-down experiments study, might <br /> sion (21,22,24). Although know exact mechanism             produce different effects inhibition SMURF2 activity. <br /> to MITF regulates survival cells MEK inhibited,          Finally, clinical data showing increased expression <br /> MITF known regulator melanoma cell survival,           SMURF2 MITF correlates MEK inhibitor resistance, <br /> induce expression BCL-2 ML-inhibitor apoptosis             hope able collect samples future and <br /> protect cells cytotoxic agents (34&#226;&#8364;&#8220;36).                                analyze potential resistance regulators. <br />  <br /> 44&#226;&#8364;&#402;Articles&#226;&#8364;&#402;|&#226;&#8364;&#402;JNCI                                                                                             Vol. 105, Issue 1 | January 2, 2012 <br />      Recent experiences targeting kinases patients                      	 14.	Johannessen CM, Boehm JS, Kim SY, et&#194;&#160; al. COT drives resistance to <br /> exposed inevitable risk developing resistance com-                       RAF inhibition MAP kinase pathway reactivation. Nature. <br />                                                                                            2010;468(7326):968&#226;&#8364;&#8220;972. <br /> plex kinase networks feedback mechanisms (10,13&#226;&#8364;&#8220;16,46,47). <br />                                                                                     	 15.	Nazarian R, Shi H, Wang Q, et&#194;&#160; al. Melanomas acquire resistance to <br /> Therefore, targeting nonkinase provide advantage. E3                            B-RAF(V600E) inhibition RTK N-RAS upregulation. Nature. <br /> ubiquitin ligases gained lot attention potential drug targets                      2010;468(7326):973&#226;&#8364;&#8220;977. <br /> when MDM2/p53 inhibitors Nutlin1&#226;&#8364;&#8220;3 discovered,                         	 16.	 Villanueva J, Vultur A, Lee JT, et&#194;&#160;al. Acquired resistance BRAF inhibi- <br /> initial efforts identify E3 ubiquitin ligase inhibitors proved                    tors mediated RAF kinase switch melanoma overcome by <br />                                                                                            cotargeting MEK IGF-1R/PI3K. Cancer Cell. 2010;18(6):683&#226;&#8364;&#8220;695. <br /> difficult (48). On hand, recent successes inhibitors <br />                                                                                     	 17.	Liu P, Cheng H, Roberts TM, Zhao JJ. Targeting phosphoinositide <br /> of apoptosis encouraging (48), suggesting developing                            3-kinase pathway cancer. Nat Rev Drug Discov. 2009;8(8):627&#226;&#8364;&#8220;644. <br /> SMURF2-specific inhibitor possible. Alternatively, fur-                    	 18.	 Pal SK, Reckamp K, Yu H, Figlin RA. Akt inhibitors clinical develop- <br /> ther characterization SMURF2, PAX3, MITF regula-                                ment treatment cancer. Expert Opin Investig Drugs. 2010;19(11): <br /> tion melanoma lead identification better druggable                        1355&#226;&#8364;&#8220;1366. <br />                                                                                     	 19.	 Balch CM, Gershenwald JE, Soong SJ, et&#194;&#160;al. Final version 2009 AJCC <br /> proteins targeted MEK inhibitor combination thera- <br />                                                                                            melanoma staging classification. J Clin Oncol. 2009;27(36):6199&#226;&#8364;&#8220;6206. <br /> pies. Our results strongly suggest melanoma therapies                  	 20.	 Workman P, Aboagye EO, Balkwill F, et&#194;&#160;al. Guidelines welfare and <br /> might potential increase frequency complete                             use animals cancer research. Br J Cancer. 2010;102(11):1555&#226;&#8364;&#8220;1577. <br /> responses reduce risk development acquired                        	 21.	 Levy C, Khaled M, Fisher DE. MITF: master regulator melanocyte devel- <br /> resistance.                                                                                opment melanoma oncogene. Trends Mol Med. 2006;12(9):406&#226;&#8364;&#8220;414. <br />                                                                                     	 22.	 Arozarena I, Bischof H, Gilby D, Belloni B, Dummer R, Wellbrock C. In <br />                                                                                            melanoma, beta-catenin suppressor invasion. Oncogene. 2011;30(45): <br />                                                                                            4531&#226;&#8364;&#8220;4543. <br /> References                                                                          	 23.	 Wellbrock C, Marais R. Elevated expression MITF counteracts B-RAF- <br /> 	  1.	 Davies H, Bignell GR, Cox C, et&#194;&#160;al. Mutations BRAF gene human             stimulated melanocyte melanoma cell proliferation. J Cell Biol. 2005; <br />        cancer. Nature. 2002;417(6892):949&#226;&#8364;&#8220;954.                                             170(5):703&#226;&#8364;&#8220;708. <br /> 	 2.	Wellbrock C, Hurlstone A. BRAF therapeutic target melanoma.              	 24.	 Wellbrock C, Weisser C, Geissinger E, Troppmair J, Schartl M. Activation <br />        Biochem Pharmacol. 2010;80(5):561&#226;&#8364;&#8220;567.                                              p59(Fyn) leads melanocyte dedifferentiation influencing MKP-1- <br /> 	 3.	Kirkwood JM, Bastholt L, Robert C, et&#194;&#160; al. Phase II, open-<span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span>, rand-                regulated mitogen-activated protein kinase signaling. J Biol Chem. 2002; <br />        omized trial MEK1/2 inhibitor selumetinib monotherapy ver-                277(8):6443&#226;&#8364;&#8220;6454. <br />        sus temozolomide patients advanced melanoma. Clin Cancer Res.        	 25.	Kubic JD, Young KP, Plummer RS, Ludvik AE, Lang D. Pigmentation <br />        2012;18(2):555&#226;&#8364;&#8220;567.                                                                 PAX-ways: role Pax3 melanogenesis, melanocyte stem cell main- <br /> 	 4.	Infante JR, Fecher LA, Nallapareddy S, et&#194;&#160; al. Safety efficacy results            tenance, disease. Pigment Cell Melanoma Res. 2008;21(6):627&#226;&#8364;&#8220;645. <br />        first-in-human study oral MEK 1/2 inhibitor GSK1120212.      	 26.	 Yang G, Li Y, Nishimura EK, et&#194;&#160;al. Inhibition PAX3 TGF-beta mod- <br />        J Clin Oncol. 2010;28:15(Suppl):<span id='am-1' about='protege:abstract' typeof='owl:Thing'>abstract</span> 2503.                                      ulates melanocyte viability. Mol Cell. 2008;32(4):554&#226;&#8364;&#8220;563. <br /> 	 5.	 Adjei AA, Cohen RB, Franklin W, et&#194;&#160;al. Phase I&#194;&#160;pharmacokinetic phar-      	 27.	 Inoue Y, Imamura T. Regulation TGF-beta family signaling E3 ubiq- <br />        macodynamic study oral, small-molecule mitogen-activated protein             uitin ligases. Cancer Sci. 2008;99(11):2107&#226;&#8364;&#8220;2112. <br />        kinase kinase 1/2 inhibitor AZD6244 (ARRY-142886) patients           	 28.	 Tang LY, Yamashita M, Coussens NP, et&#194;&#160;al. Ablation Smurf2 reveals an <br />        advanced cancers. J Clin Oncol. 2008;26(13):2139&#226;&#8364;&#8220;2146.                              inhibition TGF-beta signalling multiple mono-ubiquitination <br /> 	 6.	 Gray-Schopfer VC, Karasarides M, Hayward R, Marais R. Tumor necrosis                 Smad3. EMBO J. 2011;30(23):4777&#226;&#8364;&#8220;4789. <br />        factor-alpha blocks apoptosis melanoma cells BRAF signaling       	 29.	 Hoek KS, Schlegel NC, Brafford P, et&#194;&#160;al. Metastatic potential melano- <br />        inhibited. Cancer Res. 2007;67(1):122&#226;&#8364;&#8220;129.                                          mas defined specific gene expression profiles BRAF signature. <br /> 	 7.	VanBrocklin MW, Verhaegen M, Soengas MS, Holmen SL. Mitogen-                          Pigment Cell Res. 2006;19(4):290&#226;&#8364;&#8220;302. <br />        activated protein kinase inhibition induces translocation Bmf pro-     	 30.	 Massague J. TGFbeta Cancer. Cell. 2008;134(2):215&#226;&#8364;&#8220;230. <br />        mote apoptosis melanoma. Cancer Res. 2009;69(5):1985&#226;&#8364;&#8220;1994.                	 31.	 Haass NK, Sproesser K, Nguyen TK, et&#194;&#160;al. The mitogen-activated protein/ <br /> 	 8.	 Eisenmann KM, VanBrocklin MW, Staffend NA, Kitchen SM, Koo HM.                       extracellular signal-regulated kinase kinase inhibitor AZD6244 (ARRY- <br />        Mitogen-activated protein kinase pathway-dependent tumor-specific sur-              142886) induces growth arrest melanoma cells tumor regression <br />        vival signaling melanoma cells inactivation proapoptotic          combined docetaxel. Clin Cancer Res. 2008;14(1):230&#226;&#8364;&#8220;239. <br />        protein bad. Cancer Res. 2003;63(23):8330&#226;&#8364;&#8220;8337.                              	 32.	 Denton CL, Gustafson DL. Pharmacokinetics pharmacodynamics of <br /> 	 9.	 Davies BR, Logie A, McKay JS, et&#194;&#160;al. AZD6244 (ARRY-142886), potent                 AZD6244 (ARRY-142886) tumor-bearing nude mice. Cancer Chemother <br />        inhibitor mitogen-activated protein kinase/extracellular signal-regu-            Pharmacol. 2011;67(2):349&#226;&#8364;&#8220;360. <br />        lated kinase kinase 1/2 kinases: mechanism action vivo, pharmacoki-    	 33.	 Cragg MS, Jansen ES, Cook M, Harris C, Strasser A, Scott CL. Treatment <br />        netic/pharmacodynamic relationship, potential combination                B-RAF mutant human tumor cells MEK inhibitor requires <br />        preclinical models. Mol Cancer Ther. 2007;6(8):2209&#226;&#8364;&#8220;2219.                           Bim enhanced BH3 mimetic. J Clin Invest. 2008;118(11): <br /> 	 10.	Gopal YN, Deng W, Woodman SE, et&#194;&#160; al. Basal treatment-induced                    3651&#226;&#8364;&#8220;3659. <br />        activation AKT mediates resistance cell death AZD6244 (ARRY-        	 34.	 Garraway LA, Widlund HR, Rubin MA, et&#194;&#160;al. Integrative genomic analyses <br />        142886) BRAF-mutant human cutaneous melanoma cells. Cancer Res.                  identify MITF lineage survival oncogene amplified malignant mela- <br />        2010;70(21):8736&#226;&#8364;&#8220;8747.                                                              noma. Nature. 2005;436(7047):117&#226;&#8364;&#8220;122. <br /> 	 11.	 Emery CM, Vijayendran KG, Zipser MC, Sawyer AM, Niu L, Kim JJ, et&#194;&#160;al.        	35.	Dynek JN, Chan SM, Liu J, Zha J, Fairbrother WJ, Vucic D. <br />        MEK1 mutations confer resistance MEK B-RAF inhibition. Proc                  Microphthalmia-associated transcription factor critical transcriptional <br />        Natl Acad Sci U S A. 2009;106(48):20411&#226;&#8364;&#8220;20416.                                      regulator melanoma inhibitor apoptosis melanomas. Cancer Res. <br /> 	 12.	Wagle N, Emery C, Berger MF, et&#194;&#160; al. Dissec <br /> </body></html>