<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>2548-2554 Nucleic Acids Research, 1995, Vol. 23, No. 13                                                1995 Oxford University Press <br />  <br />  <br /> Characterization new snRNAs <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Saccharomyces cerevisiae: snR34, snR35 snR36 <br /> Dmitry A. Samarsky, Andrey G. Balakin Maurille J. Fournier* <br />  <br /> Department Biochemistry Molecular Biology Program Molecular Cellular Biology, Lederle <br /> Graduate Research Center, University Massachusetts, Amherst, MA 01002, USA <br /> Received February 7, 1995; Revised Accepted May 10, 1995                             GenBank accession nos L33802-L33804 (incl.) <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                            seen good progress characterizing new snRNAs. Most the <br />                                                                     snRNAs analyzed vertebrates yeast, humans, <br /> Genes novel snRNAs Saccharomyces                       rodents Xenopus receiving special attention the <br /> cerevisiae isolated, sequenced tested                 vertebrates. Nearly 25 snRNAs identified from <br /> for essentiality. The RNAs encoded genes               vertebrates somewhat number been <br /> designated snR34, snR35 snR36 respectively                  characterized yeast Saccharomyces cerevisiae. SnRNAs <br /> contain 203, 204 182 nucleotides. Each RNA                   described plant sources, trypanosomes, slime <br /> derived single copy gene RNAs              molds protozoa, number species analyzed from <br /> believed nucleolar, i.e. snoRNAs, based                    sources small (2). <br /> extraction properties association fibrillarin.                Only snRNAs known conserved between <br /> SnR34 snR35 contain trimethylguanosine cap,                   vertebrates yeast vertebrates, however, is <br /> but feature absent snR36. The novel RNAs               early predict number snRNAs universal. <br /> lack elements conserved snoR-                   snRNAs known conserved animals, plants and <br /> NAs, including box C, box D long sequence                       yeast include: splicing snRNAs, RNase P RNA the <br /> complementarities rRNA. Genetic disruption                     snoRNAs U3, U14 7-2/MRP RNA (1-4,6-8). Interest- <br /> analyses showed RNAs dispensable                  ingly, small RNA structural similarities U3 been <br /> and haploid strain lacking RNAs                   archaebacterium Sulfolobus acidocaldarius, <br /> previously characterized fourth snoRNA (snR33)                   suggesting snRNA functions confined eukaryotes <br /> also viable. No differences levels precursors             (11). <br /> or mature rRNAs apparent gene                        We estimate S.cerevisiae contain upwards 50-60 <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> strain. Possible roles new RNAs                snRNAs, based high resolution electrophoretic analysis of <br /> ribosome biogenesis discussed.                                  RNA prepared isolated nuclei (12; A. Balakin M. J. <br />                                                                     Fournier, unpublished results; below). Equivalent <br /> INTRODUCTION                                                        analyses nuclear RNA appear available other <br /> It clear past years eukaryotic nuclei    cell sources. If band yeast RNA detected corresponds <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> contain complex populations small stable RNAs (snRNAs). At       authentic snRNA, 40-50% snRNAs <br /> least dozen RNAs play vital roles RNA synthesis. Of   organism described, leaving 25-35 <br /> the snRNAs characterized far small subset occurs      species (or more) characterized. We report structures <br /> nucleoplasm, majority associated           essentiality results new snRNAs S.cerevisiae. <br /> nucleolus. The nucleoplasmic species include spliceosomal       All snoRNAs. <br /> snRNAs RNAs known suspected being <br /> involved mRNA synthesis tRNA processing (reviewed          MATERIALS AND METHODS <br /> 1-9). Because location small nucleolar RNAs <br /> (snoRNAs) long expected participate ribosome        Strains media <br /> biogenesis. This prediction borne far six <br /> snoRNAs shown required normal processing               Escherichia coli DH5aF&apos; [F&apos;/supE44 AlacU 169 (480 lacZ- <br /> ribosomal RNA (2,3,6,10).                                           AM 15) hsdR17 recA 1 endA 1 gyrA96 thi- 1 relA 1] used all <br />    The total number nuclear small RNAs known      cloning procedures. E.coli strain HB 101 [supE44 hsdS20(rm-) <br /> any organism. Early estimates low, especially mam-         recA 13 ara-14 proA2 lacYl galK2 rpsL20 xyl-5 mtl-I] used <br /> malian cells, primarily lack detection low abundance   source bacterial DNA hybridization analyses. Both <br /> species. As detection fractionation procedures improved,        strains grown liquid solid LB medium (1% tryptone, <br /> estimates snRNA pool size increased. The past years     0.5% yeast extract, 1% NaCl). <br />  <br />  <br />  * <br />      To correspondence addressed <br />                                                                                  Nucleic Acids Research, 1995, Vol. 23, No. 13 2549 <br />  <br />    Yeast snRNA direct sequencing DNA subgenomic        Preparation labeled probes hybridization <br /> library prepared protease-deficient strain BJ2407      procedures <br /> (a/a prbl-11221prbl-1122 prcl-4071prcl-407 pep4-31pep4-3 <br /> leu2/leu2 trplltrpl ura3-521ura3-52; 13,14). Gene disruptions        The cDNAs corresponding snR34, snR35 snR36 were <br /> and genetic manipulations carried yeast strain   synthesized one-sided (anchored) PCR (25). Oligonucleotide <br /> MH2 (a/a ade2-J0J/ade2-101 his3/his3 trpl-Alltrpl-Al                 primers complementary 3&apos; portions RNA (identi- <br /> ura3-521ura3-52 leu2-3, 11211eu2-3,112; M. Fitzgerald-Hayes,         fied above) used reverse transcription. Poly(A) tails were <br /> unpublished results). Cells grown liquid solid YEPD       added single-stranded cDNAs terminal deoxynucleotidyl <br /> (1% yeast extract, 2% peptone, 2% glucose) YNBD (0.67%            transferase (Gibco BRL Life Technologies). Double-stranded <br /> yeast nitrogen <span id='am-160' about='xsp:base' typeof='owl:Thing'>base</span>, 2% glucose), supplemented appropriate      cDNAs produced PCR using snRNA-specific <br /> amino acids nucleic acid bases, 30&#194;&#176;C             oligonucleotides oligo(dT). Restriction fragments derived <br /> indicated. Diploid cells sporulated room temperature      amplified cDNAs cloned pBluescript <br /> liquid medium containing 1% potassium acetate asci          IISK(-) vector, fused T7 RNA polymerase promoter in <br /> dissected treatment zymolyase (ICN Biochemicals           reverse orientation permit production anti-sense RNAs. <br /> Inc.).                                                               Transcription carried vitro described earlier (26), <br />                                                                      using plasmids linearized appropriate restriction enzymes, <br />                                                                      [a-3 P]UTP (800 Ci/mmol; DuPont NEN) T7 RNA poly- <br /> Preparation analysis RNA                                      merase (kindly provided C. Martin). Gel-purified radioactive <br />                                                                      transcripts used probes hybridization analyses. A <br />                                                                      radiolabeled DNA probe used SNR35 gene replacement <br /> A nuclear RNA fraction enriched known nucleolar RNAs          analysis prepared NEBlot Kit (New England Biolabs) <br /> prepared described earlier (12). Partial sequences gel-puri-   [a-32P]dCTP (800 Ci/mmol; DuPont NEN). <br /> fied snRNAs obtained using RNA sequencing kit               Oligonucleotides used identify control snRNAs included: (i) <br /> Pharmacia, according instructions supplied manufac-        5&apos;-CGCCTTCCGCGCCGTATGTGTGTGTGACC-3&apos;, U1; <br /> turer. Reverse transcription essentially described    (ii) 5&apos;-CATAGGATGGGTCAAGATCATCGCGCC-3&apos;, U3; <br /> others (15) using oligonucleotide primers complementary       (iii) 5&apos;-CGATGGGTTCGTAAGCGTACTCCTACCGTGG-3&apos;, <br /> 3&apos;-terminal sequences new RNAs (5&apos;-CTlTlCTTCGA-               U14. <br /> CATCCCAATTC snR34, 5&apos;-CCTGCCGGCCCTTGC-                              Three oligonucleotides complementary different regions of <br /> ATGGGCGTCTT snR35 5&apos;-CGTTCTAGTTACAGT                         35S pre-rRNA labeled [y-32P]ATP (6000 Ci/mmol; <br /> ACGATACC snR36). The extreme 3&apos;-terminal nucleotides          DuPont NEN) T4 polynucleotide kinase (New England <br /> each RNA determined direct chemical sequencing (16).         Biolabs) used analyze patterns pre-rRNA processing. <br /> RNA Northern primer extension analyses prepared          The probes were: 5&apos;-TTATTAAATlTTlCTTTAATGATCCTTCC- <br /> by hot phenol/glass beads procedure (17). The occurrence        GCA, complementary 18S RNA ITS1 junction <br /> 5&apos;-trimethylguanosine caps tested probing blots small      region; 5&apos;-CACCTAGCGACTCTCTCCACCG, complementary <br /> RNA precipitated TMG-specific antibodies. Association           ETS 1 region 543 nt upstream 5&apos;-end 18S RNA; <br /> with fibrillarin (NOP 1) examined electrophoretic analysis    5&apos;-CGCCTAGACGCTCTCTTCTTATCG, complementary to <br /> of RNA precipitated fibrillarin antibodies, labeling      ITS2 region, 40 nt upstream 5&apos;-end 25S RNA. <br /> with [32P]pCp. The antibodies used rabbit Rl 131 anti-TMG          Hybridizations labeled DNA, RNA oligonucleotide <br /> and anti-fibrillarin monoclonal 72B9, kindly provided R.          probes carried described previously (12). <br /> Luhrmann K. M. Pollard E. M. Tan. Analyses yeast <br /> rRNA rRNA precursor levels performed described <br /> earlier (18,19). RNA prepared cultures logarithmic       Cloning <br /> growth.                                                              A subgenomic DNA library prepared yeast DNA <br />                                                                      digested HindIII fractionated 0.8% agarose gel. <br />                                                                      Based results Southern hybridization assays, fragments <br /> Isolation analysis DNA                                        ranging size 6 8 kb selected, electroeluted from <br />                                                                      gel ligated HindlIl-digested pUC18 vector. E.coli <br /> Large small scale preparations yeast DNA carried     cells transformed clones SNR genes identified <br /> as described previously (20,21, respectively). Plasmids         colony hybridization using snRNA-specific antisense RNA <br /> isolated E.coli boiling miniprep procedure (22). E.coli    probes. Shorter DNA fragments containing SNR genes were <br /> chromosomal DNA used dot-blot hybridization assays           subcloned plasmid pUC18 used gene disruption <br /> snRNA probes purified described (23).                         analyses. The fragments were: (i) 1.3 kb SacI-PstI fragment <br />    DNA sequencing accomplished pUC18 pBlues-             (SNR34); (ii) 1.2 kb PstI fragment (SNR35); (iii) 1.0 kb BamHI <br /> cript II SK(-) vectors carrying 150-400 bp fragments snRNA        fragment (SNR36). <br /> coding sequences flanking DNA. Sequence data were <br /> generated strands using snRNA-specific primers,             Construction null alleles <br /> forward reverse M13 primers SK KS pBluescript- <br /> specific primers. The protocol assay materials       The SNR34 SNR36 genes disrupted pUC18 with <br /> dsDNA Cycle Sequencing System kit (BRL Life Technologies             BamHI restriction fragments containing yeast LEU2 HIS3 <br /> Inc.). Computer DNA RNA sequence analyses carried           genes respectively. The marker genes derived from <br /> out programs developed Genetics Computer Group           plasmids YDp-L YDp-H (27). Unique restriction sites <br /> from University Wisconsin (24).                               located SNR gene coding sequences used the <br />  2550 Nucleic Acids Research, 1995, Vol. 23, No. 13 <br />  <br /> disruptions. These BbsI SNR34 EcoRV SNR36. <br /> A PstI fragment containing SNR35 gene transferred from <br /> pUC18 pBR322, AceI site eliminated prior                          *2ZnIt <br /> to cloning operation. The SNR35 gene disrupted by <br /> replacing AseI fragment (380 bp) containing portion the <br /> snR35 coding sequence 3&apos;-flanking DNA BamHI <br /> fragment containing yeast TRPI gene derived plasmid                                                             SnR35 <br /> YDp-W (27).                                                                                                         &quot;, <br />                                                                                                                          sn R34 <br />                                                                                          ,-&gt;11 R &apos;) <br /> Construction strains inactive SNR genes <br />                                                                                         SI)102        - <br />                                                                                                       - <br />                                                                                         nRI3? <br /> Plasmids containing disrupted SNR genes restricted                            911R3..5. <br />  <br /> Sacd PstI obtain snr34::LEU2 fragment; PstI obtain a <br /> snr35::HIS3 fragment BamHI obtain snr36::TRPJ <br /> fragment. The plasmid digests used directly transform                                             ..   --        snR36 <br /> diploid yeast MH2 cells lithium acetate procedure (28). <br /> Transformants screened selective media disruption <br /> of chromosomal alleles verified Southern analysis. <br /> Haploid segregants obtained tetrad dissection. <br />   The snr34 snr36 strains crossed (YS604 x YS603) <br /> and resulting diploid dissected yield segregant YS606 <br /> containing null alleles. Similarly, snr331snr35 double <br /> mutant (YS607) constructed crossing snr33 snr35 <br /> strains (YS591 x YS605; 12 YS591 strain). Double <br /> mutants YS606 YS607 crossed, diploids dissected <br /> and segregants containing mutations (YS608) four <br /> (YS609) SNR genes isolated. The parental new strains <br /> have following genotypes: YS591 (a ade2-101 trpl-Al <br /> ura3-52 leu2-3,112 his3 snr33::URA3); YS603 (a ade2-101             Figure 1. Fractionation snR34, snR35 snR36 RNAs. Nuclei were <br />                                                                     prepared yeast spheroplasts, purified centrifugation 0.5 M <br /> trpl -Al ura3-52 leu2-3,112 his3 snr36::HIS3); YS604 (a             sucrose cushion extracted 25 mM HEPES-KOH, pH 7.6, 5 mM <br /> ade2-101 trpl-Al ura3-52 leu2-3,112 his3 snr34::LEU2);              magnesium acetate, 0.25 M sucrose NP-40. RNAs isolated phenol <br /> YS605 (a ade2-101 trpl-Al ura3-52 leu2-3,112 his3                   extraction labeled [32P]pCp fractionated electrophoresis in <br /> snr35::TRPI); YS606 (a ade2-101 trpl -Al ura3-52 leu2-3,112         8% polyacrylamide. RNA bands containing snR34, snR35 snR36 were <br />                                                                     subjected second cycle electrophoresis obtain individual species. <br /> snr34::LEU2 snr36::HIS3); YS607 (a ade2-101 trpl -Al ura3-52        Bands corresponding novel previously characterized snRNAs are <br /> leu2-3,112 his3 snr33::URA3 snr35::TRPI); YS608 (a ade2-1O1         marked. The electrophoretic <span id='am-161' about='xsp:pattern' typeof='owl:Thing'>pattern</span> shown snRNAs size range <br /> trpl-Al ura3-52 leu2-3,112 his3 snr34::LEU2 snr35::TRPJ             158-222 nt. <br /> snr36::HIS3); YS609 (a ade2-101 trpl-Al ura3-52 leu2-3,112 <br /> his3 snr33::URA3 snr34::LEU2 snr35::TRPJ snr36::HIS3). <br />                                                                     useful restriction sites cloned               17 transcription <br /> RESULTS                                                             vector. <br />                                                                       Results Southern analyses showed snRNA be <br /> Identirication new snRNAs gene cloning                       derived single genomic locus revealed cloning <br /> The snRNAs featured study selected characteriz-    strategy (Fig. 2). All coding sequences occur HindIlI <br /> ation based success obtaining direct sequence data    fragments -6-8 kb. This fraction genomic DNA than <br /> case. The individual RNAs purified preparation       used develop library enriched coding sequences. <br /> nuclear RNA. The purification procedure involved: (i) salt          Each SNR genes isolated library using <br /> extraction isolated nuclei, using conditions enrich        anti-sense RNA probes. The sequences ofthe novel snRNA genes <br /> known nucleolar snRNAs; (ii) phenol extraction; (iii) labeling      shown Figure 3. <br /> RNA [32P]pCp; (iv) fractionation individual snRNAs by <br /> high resolution gel electrophoresis. A portion gel pattern   Properties novel snRNAs genes <br /> obtained shown Figure 1. Partial sequences developed <br /> for purified RNA direct enzymatic sequencing. Searches      The 5&apos;-ends defined primer extension 3&apos;-ends by <br /> of GenBank database showed snRNAs       chemical sequencing. The sizes new RNAs revealed <br /> novel.                                                              be: (i) snR34, 203 bases; (ii) snR35, 204 bases; (iii) snR36, 182 <br />   Our cloning strategy involved hybridization screening          bases. The cDNA genomic DNA sequences agreed, indicat- <br /> genomic DNA clones labeled anti-sense RNA transcripts.         ing genes contain introns transcripts <br /> The cDNAs synthesized DNA primers complemen-              edited. None RNAs contain box C D elements <br /> tary 3&apos;-ends RNAs. A poly(A) tail added   conserved snRNAs box A B <br /> reverse transcript double-stranded cDNA prepared         sequences unique far U3 RNAs (reviewed in <br /> PCR amplification. This material sequenced identify          2,4,6). <br />                                                                                                               Nucleic Acids Research, 1995, Vol. 23, No. 13 2551 <br />  <br />                              A                                             B                                      C <br />                                                                                                                       -0 <br />                                        0    I   EC:   cr   -    -   =                                                 ._ I EC:                                CC                              = <br />                                    =        E   o      o   C    -   M                         o       _.. =         x EX O                           O - = <br />                                                                                                                                                                                           - <br />  <br />  <br />                                                                                 -.(U u         V   *-   U) &gt;          _-                             U U - U) &gt; <br />                              kb    I        m   XU    LU       X CL I.             M _        wI        0   0C                   m                   WWZ 0.0 <br />  <br />                             23.1 I                                                                                    I                                                :: . . . . . . . <br />  <br />                              9.4w                                                                                                                                  a <br />                              6.5 <br />                                                                                                                                  ..          .: .w{::: <br />  <br />                              4. 4      _o                                                                                              *.: <br />  <br />  <br />  <br />  <br />                                                                                S <br />                                                                                                                                         ;:             ::::            .:: <br />                                                                                                                                        ...... :.                         ::: * <br />  <br />                                                                                a,       Sl <br />                                                                                                                        ..: <br />  <br />  <br />  <br />  <br />                                                                          -~ ~       ~        ~~r                      -.:. <br />                                                                                                                       _E                          :: <br />                                                                                                                                                                    :.: <br />                                                                                                                       _r <br />                                                                                                                       .....                                                                   f <br />                                                                                                                                              .:                          ...                  :: <br />  <br />  <br />  <br />  <br />                                                                                                                            ...        ..i.mo.oqK.. <br />  <br />  <br />  <br />  <br />                              0.6       _                                       w                                      ilILL <br />  <br />  <br /> Figure 2. Southern analysis genomic DNA. Yeast genomic DNA digested number restriction enzymes, fractionated 0.8% agarose gels and <br /> electroblotted nylon membranes. The blots probed 32P-labeled antisense snRNA transcripts. The panels correspond genes SNR34 (A), <br /> SNR35 (B) SNR36 (C). Sizes estimated HindIII digest A DNA labeled 32p. <br />  <br />    Northern assays total cell RNA yielded single bands                            All snRNAs dispensable <br /> case: precursors variants detected (related data <br /> are shown below). Immunoprecipitation assays antibodies                                 The essentiality snRNAs examined replacing the <br /> specific trimethylguanosine yielded signals snR34                                normal genes disrupted alleles. The null-alleles were <br /> snR35, snR36, indicating snR36 lacks                                   constructed Figure 5. Hybridization data snR35 gene <br /> modification (Fig. 4). All RNAs precipitated                               disruptions shown Figure 6A. Interestingly, transformation <br /> fibrillarin-specific antibody, indicating association NOPI                           snr35::TRPI cassette resulted disruption both <br /> (data shown). Precipitation efficiencies differed, seen                           chromosomal SNR35 alleles analyzed clones. (Fig. <br /> other snoRNAs (2), positive signals evident case.                           6A, lane 4). Northern analysis total RNA clone <br /> Patterns fibrillarin-positive snRNAs presented                                showed snR35 absent (Fig. 6B), indicating SNR35 <br /> elsewhere, comprehensive study (A. Balakin                             gene product dispensable. <br /> M. Fournier, preparation). Based snoRNA-like                                       Haploids containing disrupted alleles SNR34 SNR36 <br /> extraction properties (salt-resistant) association fibrilla-                        genes isolated tetrad dissection identified by <br /> rin, new RNAs believed nucleolar.                                       screening selectable markers. The disruption gene <br />    Screening GenBank database complete gene                                  verified Southern Northern analyses; snRNA <br /> sequences did reveal significant relatedness known                                patterns shown Figure 7. Cells lacking individual <br /> genes. Negative Northern Southern hybridization results                                  snRNAs viable YEPD medium 30&apos;C no <br /> were obtained total RNA DNA E.coli                                     significant deviation growth rate evident comparison <br /> fission yeast Schizosaccharomyces pombe genomic DNA                                      wild-type strain. <br /> from Arabidopsis thaliana mice (data shown). Comple-                                   The possibility simultaneous loss snRNAs might <br /> mentarity precursor ribosomal RNA examined                                  synergistic effect growth examined. Two <br /> assess potential direct interaction RNAs.                                 strains containing multiple SNR gene knock-outs con- <br /> Several segments 10 complementary base pairs                                   structed two-step cross-and-dissection procedure (see <br /> identified, however, seen especially promising,                                Materials Methods details). One strain (YS608) lacked all <br /> similar extents complementarity exist spliceosomal                               ofthe new snRNAs. The second strain (YS609) lacked these <br /> snRNAs pre-rRNA.                                                                         fourth non-essential species, snR33 described earlier <br />    TATA-like elements occur 5&apos;-flanking segments                              (12). No growth rate differences apparent mutant and <br /> new gene, positions -81 SNR34, -92 SNR35 -108                                 wild-type cells. Comparisons YEP YNB solid <br /> for SNR36. The sequences spacings good agreement                                  media different carbon sources different temperatures. <br /> with consensus patterns reported earlier                               The conditions examined were: 2% glucose, 2% galactose, 2% <br /> snRNA genes sequenced S.cerevisiae (29). This consensus                                 glycerol 2% potassium acetate 15, 25, 32 370C (data <br /> sequence, TATAAARRGNR, located 84-100 nt                                         shown). Cell shape bud formation appeared normal <br /> coding sequence. A recent search revealed gene                                  mutant cells grown liquid YEP YNB media with <br /> snR36 adjacent 3&apos; non-coding region newly                                  various carbon sources different temperatures. <br /> discovered gene 3-phosphoserine aminotransferase (P. Belhu-                                No differences levels precursor mature rRNAs were <br /> meur, N. Fortin M. W. Clark, unpublished results). The                               apparent gene knock-out strain YS609, relative a <br /> genes tail-to-tail orientation coding regions                           control (MH2) haploid strain. The pre-rRNA patterns were <br /> separated 140 nt.                                                                         analyzed hybridization probes complementary ETS 1, <br />  2552 Nucleic Acids Research, 1995, Vol. 23, No. 13 <br />  <br />  SNR34                                                                                                           1      2       3 <br />              10         20         30         40         50                                           Ul         _           41 <br />   1 GTCTCTTCCT TGTATTCGCT TGCTGATATT CCGTACCCTA GATTTCTTAC <br />  51 TTTCGAAATA CATTACCCGG ATCAATTCCA TTATTTTCGC CTTTCAAAAA                                            U3        4_ <br /> 101 ATTTCCCGGT TGTATAAAAA GGCAGGACTT CGACAGTCAA CAACAATTTT <br /> 151 GAGTATATTT ACCACATTTT TTCTGATTTT TTTTTAAGAG GGTGAA,TCA                                         snR35         4. <br />  <br /> 201 AAATA          TTTACAC       AACGATGCCA CAGT              AACCTGTCTT <br /> 251 CTAACAGTA      TTAATTGCCA GTA               CAAGA         AACCAG                               snR344 <br /> 301     ccAGAA     ATTTA         AGGAT              TCCGAT       T2 <br /> 351 CTCAAACGA      GCGATAGAA TGGGATG            AAGAAAGTC     ACATTACTCA                           snR36         _m <br /> 401 TATATTTACG TACTGTCTCT TGCAGGTAAA AAGTCTTGAA GATCAGTTTC <br /> 451 AATATTGAAA CTAACAAATT TAGTTAGTTT CTGTACTTAG TCCGAGGTAT                                          U14 <br />                                                                                                           Gal                &apos;p <br />                                                                                                                 ..Sp <br /> 501 GTATGCTAAG ATTAGAGGAT GGTAAAAAAA TCAGCTATGA AAATACATGT                                          U14w <br />                                                                                                      UtI         . <br />  <br />  <br />       SNR35 <br />                10        20          30         40         50                 Figure 4. Immunoprecipitation TMG antibodies. Immunoprecipitated <br />       ACAAGCGTCT TCTTTAATAA CTCAAGGGAC CCTAATACGC CGTTCTTTAT <br />                                                                               small RNAs fractionated electrophoretically, transferred filters and <br />                                                                               identified radiolabeled oligonucleotide probes. Control RNAs included <br />  51 ATTCAACTCC GGGTAATGGA GATAGTATAT ATATATTTTT TTTTCCTCCC                    Ul, U3 U14. Two U14 variants included, TMG-minus (wild-type) <br />                                                                               TMG-plus, RNA polymerase II, galactose-dependent allele <br /> 101 AGTGCAACTA     TAAAAAGAGA TGAAGAATTT CTAAAGAGTA CTACCTCGGT                (Gald). Lane 1, total RNA; lane 2, RNA precipitated non-immune human <br /> 151 TTTACTTCTT GTATATTTAC TCTTTTTATT TCAAATTTTA AAAAAAAACC                    serum; lane 3, RNA precipitated TMG antibody. <br /> 201 ACAAAATT AATCGTGCGG ATAATAA                 CAGGACTA      AAACCGTZ <br /> 251 GTTTATATCG AlG&quot;TCTCT7TTT GGTATAAGCG TCAAGTCCAT ,CGGAGAGAT <br /> 301 ATCATTTTGT      A     CTA    GCCCTT       T GTAGGAAA      GAGGGAAGT       DISCUSSION <br /> 351 TAGGCTTC       AACA           GACGCCTAT     CAAGGGC       TAGGACAGA       The snRNAs described currently unique S.cerevisiae. No <br /> 401 ATCTATTCT TATTTTGACG TAGAAATAAC GTAGTTAAGA TTGAAGAGGA                     homologs present GenBank database. Negative <br />                                                                               results obtained hybridization assays RNA and <br /> 451 AAGAGGTATT CTATATACAA TAAATATCAG TGTCCGTACT TCAGATCCCT <br />                                                                               DNA different animal, plant fungal species. <br /> 501 GAAAAAGTTC ATCATACCGG TAAAAACCAC CAACATTAAT                               Hybridization results misleading, however, snRNA <br />                                                                               homologs modest homologies. For example, U14 is <br />   SNR36                                                                       conserved organisms tested far, probes for <br />             10         20         30         40         50                    S.cerevisiae U 14 cross-hybridize RNA yeast <br />   1 TCGGAAACGA GTTAAGATAA AATATAAGAA GAGGAAAGTA TTCCCAATTA <br />                                                                               S.pombe different hybridization wash conditions. <br />  51 TGTTTTCATT TAATTGGTAG ACGCGACAAA AAAAAGTATA ATATATTGGG                    Nevertheless, determined S.pombe contains U 14 and <br /> 101 AGAGATAGTT CAGTCTCACC CACATTCTCT TTCCCI                   CCTGCTCGG       gene RNA isolated sequenced <br />                                                                               (D. Samarsky M. Fournier, preparation). <br /> 151 TTGTTAATT      CCAATATTA CGATCTTGAG         TGGATGACAA mAAmZ                 Of snRNAs characterized far,.most believed be <br /> 201   ATAcA                                   AACATT <br />                                 T CAATTTGCTA AT                CTTTGA         nucleolar. Yeast nucleoli defined morphologically <br /> 251 GTACGAGGAT ATCGCTA           TTATTA       ,GTATCGTATTGTAATTGA             higher eukaryotes generally accepted <br />                                                                               procedure isolating complex exists present. For this <br /> 301   cG=TcA= CATATCGTAT         AATTCGGCCT TTATTTCAAG AAAAAGAAGA <br />                                                                               reason, classification nucleolar RNAs yeast based on <br /> 351 GAAGT                                                                     association rRNAs nucleolar proteins, generally NOPI, <br />                                                                               yeast counterpart vertebrate fibrillarin. Co-puri- <br /> Figure 3. Sequences genes snR34, snR35 snR36. The sequences    fication nucleolus used metazoan organisms, but <br /> of snRNA coding segments flanking regions shown. The 3&apos;-ends   adopted yeast. Using alternative approach, a <br /> the coding regions defined direct chemical sequencing individual <br /> snRNAs 5&apos;-ends primer extension analysis. The sequences            nuclear RNA fraction enriched snoRNAs prepared by <br /> corresponding snRNAs highlighted italics underlining.       differential extraction. The new snRNAs described are <br /> Candidate TATA elements identified bold letters. The TATA elements   believed nucleolar they: (i) exhibit extraction properties <br /> conforn consensus sequence proposed earlier S.cerevisiae SNR genes   like known nucleolar RNAs (U3 U14; 12); (ii) are <br /> (TATAAARRGNR; 30).                                                            precipitable antibodies fibrillarin. While are <br />                                                                               accepted criteria, note somewhat tenuous, both <br />                                                                               indirect. <br />                                                                                  Yeast nucleolar RNAs believed transcribed RNA <br /> ITS 1 ITS2 regions 35S rRNA. The levels mature 18S                  polymerase II. Most contain TMG caps, presumed to <br /> and 25S rRNA assessed visual inspection RNA gels                   formed hypermethylation m7G capped transcripts. Most <br /> stained ethidium bromide (data shown).                               coding units TATA-like elements, however, these <br />  A <br />  <br />  <br />  <br />  <br /> B <br />  <br />  <br />  <br />  <br /> C <br />  <br />  <br />  <br />  <br /> A <br />          -t- <br />  <br />  <br />      BamHI <br />  <br />  <br />  <br />  <br /> Figure <br />              Sacl <br />  <br />  <br />  <br />  <br />               l~~ <br />  <br />  <br />  <br />  <br />            PstI <br />  <br />  <br />  <br />  <br />          1 2 3 4 5 <br />                   l <br />                       _~ .          Bbsl <br />                                     SNR34 <br />  <br />  <br />  <br />  <br />                                  Asel <br />                                       LEIJU-. <br />  <br />  <br />  <br />  <br />                                        P1 <br />                                      ...T. <br />  <br />  <br />  <br />                                    SNR3 5 <br />  <br />  <br />  <br />  <br />                                     mmmft.. <br />  <br />  <br />  <br />  <br />                                     B <br />                                       EcoRV <br />                                      SNR36 <br />                                                 Asel <br />  <br />  <br />  <br />  <br />                                         .H.....S. <br />                                                        ~ <br />  <br />  <br />  <br />  <br /> Plasmids containing inactive snRNA genes prepared disrupting the <br /> RNA coding sequences selectable yeast marker genes. The structures of <br /> the relevant DNA segments restriction sites shown. <br />  <br />  <br />  <br />  <br />                                                 1 2 3 4 5 <br />                                                            ~ <br />                                                                0.1 kb <br />  <br />  <br />  <br />  <br />                                                                 ~   ~~~ <br />  <br />  <br />  <br />  <br />                                                                 0.1kb <br />                                                                 L-- <br />                                                                            PstI <br />  <br />  <br />  <br />  <br />                                                                           ~~~~ <br />  <br />  <br />                                                                             Pstl <br />  <br />  <br />  <br />  <br />                                                                           0.1kb <br />  <br />  <br />  <br />  <br />          5. Construction null alleles snR34, snR35 snR36 genes. <br />                                                                                    Bamr HI <br />                                                                                              BB <br />                                                                                                     et <br />  <br />                                                                                                     z <br />                                                                                                     cn <br />  <br />  <br />  <br />  <br />                                                                                                     1 2 3 4 <br />  <br />  <br />                                                                                                      &quot;I&quot; <br />                                                                                                      a: <br />                                                                                                          L <br />  <br />  <br />  <br />  <br />                                                                                                          ) <br />  <br />  <br />  <br />  <br />                                                                                                      1 2 3 4 <br />                                                                                                              L) <br />                                                                                                              Z <br />                                                                                                                   Nucleic Acids Research, 1995, Vol. 23, No. 13 2553 <br />  <br />  <br />                                                                                                                   - <br />  <br />  <br />  <br />  <br />                                                                                                                   cn <br />                                                                                                                   u0 <br />                                                                                                                       _ <br />  <br />  <br />  <br />  <br />                                                                                                                       uu <br />                                                                                                                           r,I <br />  <br />  <br />                                                                                                                       I;mU~ c4UJ <br />                                                                                                                           C <br />                                                                                                                           r_ <br />                                                                                                                                 U <br />  <br />  <br />  <br />  <br />                                                                                                                                 u <br />                                                                                                                                 zn <br />                                                                                                                                     J <br />  <br />  <br />  <br />  <br />                                                                                                                                     _ <br />  <br />  <br />  <br />  <br />                                                                                                                                     f <br />  <br />                                                                                                                                     X-  , <br />  <br />  <br />  <br />  <br />                                                                                                                                         C <br />                                                                                                                                         r_ <br />                                                                                                                                              z <br />                                                                                                                                              LI) <br />  <br />  <br />  <br />  <br />                                                                                                                                                V <br />                                                                                                                                                Cl! <br />                                                                                                                                                z <br />                                                                                                                                                V) <br />                                                                                                                                                      - <br />  <br />  <br />  <br />  <br />                                                                                                                                                      - 2.Okb <br />  <br />  <br />  <br />  <br />                                                                                                                                                      - <br />                                                                                                                                                      - <br />  <br />  <br />                                                                                                                                                      - <br />                                                                                                                                                          3.6kb <br />  <br />  <br />  <br />  <br />                                                                                                                                                          5.8S <br />                                                                                                                                                          5S <br />  <br />                                                                                                                                                          tRNA <br />  <br />  <br />  <br />  <br />                                                                                              Figure 7. Southern Northern data strains containing null alleles the <br />                                                                                              snR34 snR36 genes. Diploid cells transformed With snR34 snR36 gene <br />                                                                                              fragments (PstI-SacI BamH I respectively; Fig. SA C) identified <br />                                                                                              growth selective media genomic DNAs subjected Southern <br />                                                                                              analysis treatment PstI BamHI respectively. Haploids containing <br />                                                                                              disrupted snr alleles prepared tetrad dissection analyzed growth <br />                                                                                              selective media snRNA content. Southern (A) Northern (B) <br />                                                                                              assay results shown cells derived spores single tetrads. The data <br />                                                                                              presented panels sets clones. <br />  <br />  <br />  <br />                                                                                              shown involved expression (2). The 5&apos; <br />                                                                                              non-coding regions ofthe new genes elements and <br />                                                                                              positions sequences elements consistent with <br />                                                                                              previously characterized yeast SNR genes. This feature implies <br />                                                                                              expression regulation related snRNAs occur <br />                                                                                                                                                                  cr, <br />                                                                                                                                                                  L#n <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  c <br />                                                                                                                                                                  U, <br />  <br />  <br />  <br />                                                                                                                                                                  up <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  Cr <br />  <br />                                                                                                                                                                  I <br />                                                                                                                                                                  tn <br />                                                                                                                                                                  (0 <br />                                                                                                                                                                  rr, <br />                                                                                                                                                                   C <br />                                                                                                                                                                   UM <br />                                                                                                                                                                        Z <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  1 2 3 4 <br />  <br />  <br />  <br />                                                                                                                                                                        CIO <br />                                                                                                                                                                        Z <br />                                                                                                                                                                         ~ <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        LI) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        V) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        amm <br />  <br />  <br />  <br />  <br />                                                                                                                                                                   1 2 3 4 <br />                                                                                                                                                                          trno <br />                                                                                                                                                                             L <br />                                                                                                                                                                              c <br />                                                                                                                                                                              L) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              c <br />                                                                                                                                                                              U, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              CY) <br />                                                                                                                                                                              I <br />                                                                                                                                                                             (0 <br />                                                                                                                                                                               4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               c <br />                                                                                                                                                                               U) <br />                                                                                                                                                                                    z <br />                                                                                                                                                                                    C) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    (0 <br />                                                                                                                                                                                    CY) <br />                                                                                                                                                                                    z <br />                                                                                                                                                                                    u) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    4P <br />                                                                                                                                                                                           -   2.1kb <br />                                                                                                                                                                                          l.lOkb <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          - <br />                                                                                                                                                                                           - <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          -5.8S <br />                                                                                                                                                                                               5S <br />                                                                                                                                                                                           =J tRNA <br />  <br />  <br />  <br />  <br />                                                                                              common mechanisms. The absence TMG cap snR36 <br />                                                                                              RNA suggests RNA results processing. Two classes <br />                                                                                  -5.8 S      cap-free nucleolar snRNAs known exist yeast, both <br />          _Un_ _0 _4_         1 .6 kb <br />                              -                                                   -5S         derived processing. One class consists species processed <br />                                                                                              pre-mRNA introns, e.g. U18 (A. Balakin M. J. Fournier, <br />          __s4_M           _ -1.2 kb <br />                                                                                              preparation) snR38 snR39 (J. Ni M. J. Fournier in <br />                                                                                  -it RNA <br />                                                                                              preparation). The second corresponds RNAs processed from <br />                                                                                              transcripts appear intronic (U 14 snR190; 30). <br />                                                                                              It possible snR36 transcribed RNA polymerase <br /> Figure 6. Analysis DNA RNA cells containing disrupted snR35                      III. In plants, example, U3 MRP RNAs shown <br /> gene alleles. Diploid cells transformed PstI restriction fragments                 transcribed RNA polymerase III (31,32). <br /> from snr35 plasmid shown Figure 5. Transformants identified                      The gene disruption analyses did provide clues about <br /> growth selective medium subjected Southern analysis, following <br /> digestion PstI. The Southern data (A) indicate allele                 functions new RNAs. None genes essential for <br /> disrupted isolates, alleles damaged strain (lane 4).            cell viability loss RNAs detectable effect on <br /> No snR35 RNA detected Northern analysis diploid (B).                        growth rate temperatures different media. The <br />  2554 Nucleic Acids Research, 1995, Vol. 23, No. 13 <br />  <br /> absence obvious <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> triple quadruple                 3    Mattaj,I.W., Tollervey,D. Seraphin,B. (1993) FASEB J., 7, 47-53. <br /> mutants lacking new snRNAs snR33 eliminates                      4    Filipowicz,W. Kiss,T. (1993) Mol. Biol. Rep., 18, 149-156. <br /> possibility strong synergistic effect growth. A similar              5    Sollner-Webb,B. (1993) Cell, 75, 403-405. <br />                                                                              6    Fournier,M.J. Maxwell,E.S. (1993) Trends Biochem. Sci., 18, <br /> situation described Parker et al. snRNA                       131-135. <br /> species snR3, snR4, snR5, snR8 snR9 (29),              7    Clayton,D.A. (1994) Proc. NatL Acad. Sci. USA, 91, 4615-4617. <br /> splicing snRNAs. A strain lacking RNAs apparent              8    Darr,S.C., Brown,J.W. Pace,N.R. (1992) Trends Biochem. Sci., 17, <br /> growth phenotype mutant disrupted snRNAs                      178-182. <br /> snRlO RNA phenotype strains lacking                      9   Wahle,E. Keller,W. (1992) Annu. Rev. Biochem., 61, 419-440. <br />                                                                              10   Sollner-Webb,B., Tycowski,K.T. Steitz,J.A. (1995) In Zimmer- <br /> snRIO; snRlO essential, loss yields cs phenotype                 mann,R.A. Dahlberg,A.E. (eds), Ribosomal RNA: Structure, Evolution, <br /> and impainrent rRNA processing (33). Of non-spliceosomal                   Gene Expression Function Protein Synthesis. CRC Press, New <br /> yeast snRNAs characterized date, including RNAs                            York, NY, press. <br /> described here, &lt;30% essential exhibit detectable growth            11   Potter,S., Durovic,P. Dennis,P.P. (1995) Science, 268, 1056-1060. <br /> phenotype (2). Dispensability limited snRNA coding                 12   Balakin,A.G., Schneider,G.S., Corbett,M.S., Ni,. Fournier,M.J. (1993) <br />                                                                                   Nucleic Acids Res., 21, 5391-5397. <br /> units. Studies random disruptions suggest 70%           13   Jones,E.W. (1977) Genetics, 85, 23-33. <br /> yeast genomic sequences dispensable (34). In addition, 45             14   Zubenko,G.S., Mitchell,A.P. Jones,E.W. (1980) Genetics, 96, <br /> putative protein coding regions identified sequence                     137-146. <br /> chromosome III, 62% shown non-essential (35).                     15   Treizenberg,S.J., (1992) In Ausubel,F.A., Brent,R., Kingston,R.E., <br />    The yeast nucleolar snRNAs known essential                  Moore,D.D., Seidman,J.G., Smith,J.A. Struhl,K. (eds), Current <br />                                                                                   Protocols Molecular Biology. Greene Publishing Wiley- <br /> required processing pre-rRNA, precise functions                   Interscience, New York, NY, pp. 4.8.1-4.8.5. <br /> have established (reviewed 2-4,10). U3, U14              16   Peattie,D.A. (1979) Proc. Natl. Acad. Sci. USA, 76, 1760-1764. <br /> snR30 essential early processing steps, MRP          17   Kohrer,K. Domdey,H. (1991) Methods Enzymol., 194, 398-405. <br /> species needed later cleavage upstream 5.8S RNA. The             18   Li,H.V. Fournier,M.J. (1990) Mol. Cell. Biol., 10, 1145-1152. <br /> roles non-essential snRNAs remain enigma. No function              19   Brown,T. (1993) In Ausubel,F.A., Brent,R., Kingston,R.E., Moore,D.D., <br /> is formal possibility, likely dispensable            Seidman,J.G., Smith,J.A. Struhl,K. (eds), Current Protocols in <br />                                                                                   Molecular Biology. Greene Publishing Wiley-Interscience, New York, <br /> RNAs functionally redundant mediate processes                  NY, pp. 4.9.2-4.9.4. <br /> are beneficial, critical. Consistant view     20   Rose,M.D., Winston,F. Hieter,P. (1990) Methods Yeast Genetics: A <br /> finding loss non-essential snoRNA, snR3 1, does              Lab <br /> </body></html>