<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yeast autonomously replicating sequence <br /> binding factor involved nucleotide <br /> excision repair <br /> Simon H. Reed,1 Masahiro Akiyama,3 Bruce Stillman,2 Errol C. Friedberg1,4 <br /> 1 <br />   Laboratory Molecular Pathology, Department Pathology, University Texas Southwestern Medical Center, Dallas, <br /> Texas 75235 USA; 2Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 11724; 3Department Molecular <br /> Biology, Graduate School Biological Sciences, Nara Institute Science Technology <br /> (NAIST),Ikoma, Nara 630-01, Japan <br />  <br />  <br />  <br /> Nucleotide excision repair (NER) yeast effected concerted action large complex proteins. <br /> Recently, identified stable subcomplex containing yeast Rad7 Rad16 proteins. Here, report <br /> the identification autonomously replicating sequence binding factor 1 (ABF1) component the <br /> Rad7/Rad16 NER subcomplex. Yeast ABF1 protein encoded essential gene required DNA <br /> replication, transcriptional regulation, gene silencing. We ABF1 plays direct role NER in <br /> vitro. Additionally, consistent role ABF1 protein NER vivo, certain <br /> temperature-sensitive abf1 mutant strains defective DNA replication specifically defective in <br /> the removal photoproducts NER sensitive killing ultraviolet (UV) radiation. These studies <br /> define novel unexpected role ABF1 protein NER yeast. <br /> [Key Words: DNA repair; Saccharomyces cerevisiae; ABF1; Rad7; Rad16] <br /> Received September 1, 1999; revised version accepted October 14, 1999. <br />  <br />  <br />  <br />  <br /> Autonomously replicating sequence (ARS) binding factor                  With respect role transcription, ABF1-binding <br /> 1 (ABF1) abundant DNA-binding protein encoded                  sites identified promoters genes and <br /> by essential gene Saccharomyces cerevisiae (Rhode              mutational analysis revealed positive nega- <br /> et al. 1992). ABF1 recognizes specific DNA sequence                tive regulatory effects gene expression (Brand et al. <br /> motif present numerous sites yeast genome,                 1987; Buchman Kornberg 1990; Della Seta et al. <br /> including ARS&#226;&#8364;&#8482;s, promoter elements mating-type si-               1990). Transcriptional silencing mating-type loci in <br /> lencing sequences (Rhode et al. 1992). Genetic analyses              yeast effected localized alterations chromatin <br /> have demonstrated ABF1 protein plays roles ARS               structure (Rine Herskowitz 1987). Silencing re- <br /> activity DNA replication, transcriptional activa-             quires presence cis-acting elements (which in- <br /> tion, transcriptional silencing HM mating-type                clude ARS), trans-acting factors histones H3 <br /> loci (Diffley Stillman 1989; Rhode et al. 1989;                  H4, RAP1 protein, Sir proteins, ABF1 (Klar et <br /> Walker et al. 1989).                                                 al. 1979; Feldman et al. 1984; McNally Rine 1991). A <br />    ARS&#226;&#8364;&#8482;s contain multiple functional elements (Marah-                direct role ABF1 protein silencing estab- <br /> rens Stillman 1992; Rao et al. 1994; Rao Still-              lished isolation temperature-sensitive alleles of <br /> man 1995). Element A required binding ori-             ABF1 specifically defective modality (Loo <br /> gin recognition complex (ORC) (Marahrens Stillman                et al. 1995). <br /> 1992; Rao et al. 1994; Rao Stillman 1995),                  In contrast genes required process of <br /> element B1 participates ORC&#226;&#8364;&#8220;DNA interactions. Ele-                nucleotide excision repair (NER) yeast, Rad7 and <br /> ment B3 identified ABF1-binding site                 Rad16 proteins appear play specialized role this <br /> (Marahrens Stillman 1992; Rao et al. 1994; Rao               process. They required specifically removal of <br /> Stillman 1995). Additionally, temperature-sensitive abf1             pyrimidine dimers transcriptionally repressed re- <br /> mutants shown defective DNA repli-                gions genome, (such HM mating-type loci), <br /> cation (Rhode et al. 1992), providing direct genetic evi-            transcriptionally activated homologs <br /> dence role ABF1 protein initiation DNA            (Terleth et al. 1990; Verhage et al. 1994). Additionally, <br /> replication.                                                         Rad7 Rad16 required NER nontran- <br />                                                                      scribed strand transcriptionally active genes, not <br /> 4 <br />  Corresponding author.                                               repair transcribed strand genes (Terleth <br /> E-MAIL friedberg.errol@pathology.swmed.edu; FAX (214) 648-4067.      et al. 1990; Verhage et al. 1994). The precise role Rad7 <br />  <br />  <br /> 3052      GENES &amp; DEVELOPMENT 13:3052&#226;&#8364;&#8220;3058 &#194;&#169; 1999 Cold Spring Harbor Laboratory Press ISSN 0890-9369/99 $5.00; www.genesdev.org <br />                                                                                              ABF1: novel function DNA repair <br />  <br />  <br />  <br /> and Rad16 proteins NER structurally dis-       bands identified (&#226;&#710;&#188;112 kD), known immunob- <br /> tinctive regions genome clear,          lotting contain Rad16 protein, stained intensely <br /> higher eukaryotic homologs proteins iden-        Rad7 band (&#226;&#710;&#188;75 kD) (Fig. 1A). Tandem mass <br /> tified. Paetkau et al. (1994) reported interaction       spectrometry analysis 112-kD band revealed the <br /> Rad7 protein Sir3, protein involved silencing     presence ABF1 protein addition previously <br /> mating-type loci. However, current evidence suggests           identified Rad16 protein (see Materials Methods). <br /> that Rad7 directly involved gene silencing,      We confirmed result immunoblotting affin- <br /> there indications factors required gene        ity-purified ABF1 antibody [see Materials Methods <br /> silencing involved NER.                            (Diffley Stillman 1988, 1989), does cross <br />    Another feature distinguishes RAD7             react Rad16 protein (Fig. 1B)]. <br /> RAD16 genes genes involved NER             Details purification Rad7/Rad16 complex <br /> yeast deletion of genes confers   reported previously (Reed et al. 1998). Briefly, <br /> moderate sensitivity ultraviolet (UV) radiation (Ter-       His-tagged Rad7 protein purified nickel <br /> leth et al. 1990). Furthermore, extracts rad7           column fractionation step. It noted <br /> rad16-deletion mutants defective early          His-tagged Rad7 strain expresses Rad7 similar <br /> event damage-specific DNA incision NER            levels wild-type strain created. <br /> vitro vivo (Reed et al. 1998). However, mu-         All Rad7 Rad16 protein immobilized the <br /> tants severely deficient (but totally defective)    nickel matrix proteins copurified subse- <br /> post-incision oligonucleotide excision repair synthe-      quent purification steps. In contrast, noted &#226;&#710;&#188;70% <br /> sis DNA vitro (Reed et al. 1998). These observations     ABF1 protein flow-through fraction <br /> have led suggestion Rad7 Rad16 pro-        nickel column. Hence, estimate only <br /> teins play role displacement (excision) oligo-     &#226;&#710;&#188;30% cellular ABF1 protein associated with <br /> nucleotide fragments generated bimodal incision         Rad7 Rad16. <br /> reaction. This process turn directly coupled <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> dissociation bound NER machinery DNA <br /> and/or chromatin modulation NER. In re-         Depleting ABF1 cells results UV radiation <br /> gard, Rad16 member SWI/     sensitivity defective NER <br /> SNF superfamily ATPases, believed          ABF1 protein encoded essential gene a <br /> to subunit chromatin-modulating complex              limited repertoire conditional-lethal mutants are <br /> (Bang et al. 1992).                                            available (Rhode et al. 1992). Therefore, investigated <br />    Here report unexpected role ABF1 protein       functional significance physical association <br /> NER yeast. We ABF1 protein copurifies             Rad7/Rad16 complex using experimental <br /> with yeast Rad7 Rad16 proteins [which form           approach facilitates temperature-dependent deg- <br /> stable complex vivo vitro (Wang et al.          radation selected gene products living yeast cells <br /> 1997; Reed et al. 1998) multiple fractionation         (the N-degron strategy) (Dohmen et al. 1994). Fusing a <br /> steps. A mutant strain depleted ABF1 protein         temperature-dependent degradation signal ABF1 gen- <br /> N-degron strategy (Dohmen et al. 1994) sensitive       erated mutant strain designated abf1td, ABF1 <br /> to killing following exposure UV radiation, defec-      protein selectively inactivated, obviating <br /> tive NER vitro. Furthermore, defective NER            need laboriously screen informative condi- <br /> vitro specifically complemented purified re-       tional-lethal mutants. <br /> combinant ABF1 protein. Finally, subset condi- <br /> tional-lethal abf1 mutants shown previously defec- <br /> tive DNA replication shown sensitive to <br /> killing UV light, agents produce DNA <br /> damage repaired pathways distinct NER. <br /> Furthermore, demonstrate UV radiation sen- <br /> sitivity mutants directly correlates in- <br /> ability remove UV radiation-induced lesions DNA. <br />                                                                Figure 1. (A) Polyacrylamide gel stained Coomassie blue <br />                                                                shows presence Rad16 (&#226;&#710;&#188;112 kD) Rad7 (&#226;&#710;&#188;75 kD) pro- <br /> Results                                                        teins. The purification protocol gel electrophoresis proce- <br />                                                                dures described previously (Reed et al. 1998). The pres- <br /> ABF1 copurifies Rad7 Rad16 proteins                   ence proteins confirmed Western blotting and <br /> Recently, reported purification Rad7/Rad16-        tandem mass spectrometry (see Materials Methods). <br />                                                                Note intensity 112-kD band greater that <br /> containing protein complex (Reed et al. 1998). Gel filtra- <br />                                                                75-kD band. (B) Western blot analysis shows cofraction- <br /> tion purified complex revealed apparent mo-          ation purified ABF1, Rad16, Rad7 proteins a <br /> lecular mass &#226;&#710;&#188;300 kD (Reed et al. 1998), significantly      glycerol gradient described (Reed et al. 1998). Antibodies to <br /> larger summed molecular masses Rad7            Rad7 Rad16 prepared described (Reed et al. 1998). <br /> Rad16 (&#226;&#710;&#188;190 kD). In present studies, electrophoretic       Antibodies ABF1 protein described Materials and <br /> analysis purified complex revealed larger      Methods. <br />  <br />  <br />                                                                                             GENES &amp; DEVELOPMENT            3053 <br />  Reed et al. <br />  <br />  <br />  <br />    As shown Figure 2, ABF1 protein depleted pro- <br /> gressively extracts 1&#226;&#8364;&#8220;5 hr incubation at <br /> the restrictive temperature (37&#194;&#176;C). Rad7 Rad16 pro- <br /> teins degraded conditions, a <br /> significantly reduced rate compared ABF1 protein <br /> (Fig. 2). The time required steady-state levels ABF1, <br /> Rad7, Rad16 proteins decrease 50% under <br /> these conditions (measured quantitative densitome- <br /> try scanning) calculated 2, 3.5, 4 hr, respec- <br /> tively. Such degradation observed wild-type              Figure 3. Western blot analysis shows abf1td <br /> strains 37&#194;&#176;C (data shown). Additionally,               mutant strain maintained elevated temperature, ABF1 <br /> NER proteins Rad23 (Fig. 2),             protein extracts competent NER depleted (A). The pro- <br />                                                                     tein detectable extract prepared abf1td strain <br /> cross-reacting proteins (Fig. 2b), showed obvious deg- <br />                                                                     maintained permissive temperature rad7/rad16 <br /> radation 5-hr period 37&#194;&#176;C. The codegradation           double-deletion mutant strain (A). Under conditions, <br /> Rad7 Rad16 ABF1 provides supporting evidence               Rad7 protein detected extracts abf1td strain but <br /> that proteins exist complex vivo.                 extracts rad7/rad16 double-deletion mutant (B). <br />    To correlate temperature-dependent degradation            The center lane B deliberately overloaded demonstrate <br /> ABF1 protein defective NER, prepared whole-                 absence Rad7 protein. The band migrating higher <br /> cell extracts (WCE) abf1td strain held              molecular mass Rad7 (*) nonspecific cross-reacting <br /> restrictive temperature (37&#194;&#176;C) 4 hr. Under con-           species strain. (C) Similar data Rad16 protein. <br /> ditions, ABF1 protein depleted (Fig. 3A), Rad7 <br /> protein (Fig. 3B) Rad16 protein (Fig. 3C) still <br />                                                                     synthesis extracts wild-type strain (data not <br /> readily detectable immunoblotting. We tested the <br />                                                                     shown). Finally, purified ABF1 did complement de- <br /> NER activity extract showed severely <br />                                                                     fective repair synthesis rad7 rad16 single- <br /> deficient compared extracts prepared the <br />                                                                     mutant extracts (data shown), rad7/rad16 <br /> abf1td strain maintained permissive temperature <br />                                                                     double-mutant extracts (Fig. 4, lanes 8&#226;&#8364;&#8220;10). These results <br /> (23&#194;&#176;C), comparable known NER-defective <br />                                                                     demonstrate direct role ABF1 protein NER in <br /> rad2 mutant (Fig. 4, cf. lanes 1&#226;&#8364;&#8220;3 lanes 11&#226;&#8364;&#8220;13). De- <br />                                                                     vitro. <br /> fective NER ABF1-depleted extract comple- <br /> mented mixing extract rad2 mutant <br /> (Fig. 4, cf. lanes 11&#226;&#8364;&#8220;14), demonstrating extract <br /> was nonspecifically inactivated. Furthermore, the <br /> addition purified recombinant ABF1 protein expressed <br /> in baculovirus (see Materials Methods) hence <br /> free yeast Rad7 Rad16 proteins, restored repair <br /> synthesis abf1td mutant extracts (Fig. 4, lanes 4&#226;&#8364;&#8220;6), <br /> but rad2 mutant extracts (data shown). Puri- <br /> fied ABF1 protein stimulatory effect repair <br />  <br />  <br />  <br />  <br />                                                                     Figure 4. NER measured monitoring repair synthesis of <br />                                                                     plasmid substrate carrying defined <span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span> damage (bottom <br />                                                                     bands) relative levels nonspecific DNA synthesis a <br />                                                                     larger control plasmid (top bands). The details assay are <br />                                                                     described (Wang et al. 1997). (A) Ethidium-stained gels indi- <br />                                                                     cating relative amounts DNA loaded. (B) An autoradio- <br />                                                                     graph gel. ABF1+ extracts (ABF1TD) prepared the <br /> Figure 2. An N-degron strain constructed expresses       abf1td strain permissive temperature support robust re- <br /> temperature-degradable ABF1TD fusion protein (Dohmen et al.         pair synthesis (lane 1). However, synthesis defective in <br /> 1994). When strain maintained 37&#194;&#176;C varying          rad2 deletion mutant known required NER (lanes <br /> periods time, progressive loss ABF1TD         2,12), extracts rad7/rad16 double-deletion mutant <br /> fusion protein monitored Western blotting monoclo-       (lane 7). Defective deficient NER observed extracts <br /> nal antibodies HA epitope protein (A). Rad7 (B)    strain expressing temperature-degradable ABF1 protein <br /> Rad16 protein (C) degraded, rate deg-     (lanes 3,13). Defective NER complemented mixing rad2 <br /> radation significantly slower ABF1TD fu-       mutant ABF1-depleted (by temperature degradation) ex- <br /> sion protein. Rad23 protein (D) various proteins     tracts (lane 14), adding increasing amounts purified <br /> cross-reacted antibodies (e.g., lower band B) did   ABF1 protein ABF1-depleted extracts (lanes 4&#226;&#8364;&#8220;6). Addi- <br /> not detectable degradation. Antibodies Rad23 protein        tion ABF1 protein effect level repair synthe- <br /> were prepared described (Wang et al. 1997).                      sis rad7/rad16 deletion mutant (lanes 8&#226;&#8364;&#8220;10). <br />  <br />  <br /> 3054          GENES &amp; DEVELOPMENT <br />                                                                                                    ABF1: novel function DNA repair <br />  <br />  <br />  <br />                                                                      assay. This assay monitors loss cyclobutane py- <br />                                                                      rimidine dimer (CPD) 6-4 photoproducts geno- <br />                                                                      mic DNA specific monoclonal antibodies raised <br />                                                                      lesions (Mori et al. 1991). Figure 6 shows <br />                                                                      kinetics loss 6-4 photoproducts abf1-1 <br />                                                                      (Fig. 6B) abf1-5 (Fig, 6A) mutants wild- <br />                                                                      type parental strain (Fig, 6A,B). The abf1-1 mutant is <br />                                                                      defective removal lesions compared with <br />                                                                      wild-type parental strain semipermissive tem- <br />                                                                      perature 32&#194;&#176;C (Fig. 6B, cf. lanes 3 4). In contrast, <br />                                                                      abf1-5 strain wild-type parental strain <br />                                                                      showed progressive loss lesions function time <br />                                                                      temperature (Fig. 6A, cf. lanes 3 4). At the <br />                                                                      permissive temperature 23&#194;&#176;C, lesion removal ob- <br /> Figure 5. UV-radiation sensitivity abf1-1 abf1-5 mu-      served strains. However, condi- <br /> tants. <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>Survival</span> measured quantitating colony-forming          tions kinetics lesion removal abf1-1 strain <br /> ability described (Huang et al. 1998) highest tempera-     reduced compared wild type (Fig. 6A B, <br /> ture strains grow (32&#194;&#176;C abf1-1 28&#194;&#176;C       lanes 1 2). <br /> for abf1-5). (&#228;&#352;?) Wild type 32&#194;&#176;C; (&#227;&#8364;&#171;) abf1-1 32&#194;&#176;C; (&#228;&#8240;&#173;) abf1-5      Most temperature-sensitive mutations yeast reflect <br /> at 28&#194;&#176;C; (&#228;&#352;&#8240;) wild type 28&#194;&#176;C.                                      amino acid changes result destabilization pro- <br />                                                                      teins elevated temperature. Such mutations often <br /> DNA-binding domain mutants ABF1                                   result alterations (but all) phenotypes <br /> are sensitive UV radiation                                        permissive temperature. For example, Rhode <br />                                                                      et al. (1992) reported permissive tempera- <br /> Conditional-lethal abf1 mutants isolated in <br />                                                                      ture, ABF1-1 mutant protein able bind <br /> two independent laboratories basis screens for <br />                                                                      ABF1-binding site (a requirement DNA replica- <br /> defective transcriptional silencing (Loo et al. 1995) and <br />                                                                      tion), strain showed obvious replication de- <br /> defective DNA replication (Rhode et al. 1992). Sensitiv- <br />                                                                      fect. Therefore, suggested protein must <br /> ity mutants UV radiation examined <br />                                                                      bind cognate site extent vivo. We pro- <br /> at nonpermissive permissive temperatures si- <br /> lencing. No increased UV-radiation sensitivity ob- <br /> served conditions (data shown), sug- <br /> gesting silencing function ABF1 protein does <br /> not play role NER. In contrast, examined <br /> conditional-lethal mutants obtained screen for <br /> defective DNA replication, strain designated abf1-1 <br /> with <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> DNA-binding domain ABF1 <br /> protein (Rhode et al. 1992), clearly sensitive UV <br /> light semipermissive temperature 32&#194;&#176;C (Fig. 5). <br /> The abf1-1 mutant strain abnormally sensitive <br /> to DNA-damaging agents methylmethane sulfonate <br /> or hydrogen peroxide. Both agents known to <br /> produce DNA damage repaired pathways other <br /> than NER (Friedberg et al. 1995; data shown). A sec- <br /> ond independently isolated mutant strain (abf1-2) shown <br /> to carry identical mutation ABF1 DNA-binding <br /> domain (Rhode et al. 1992) abnormally sensitive <br /> to killing UV light (data shown). A replica- <br /> tion-defective strain (abf1-5), carries mutation <br /> outside DNA-binding motif defective in <br /> binding ABF1 recognition sequence (Rhode et al. <br /> 1992), abnormally sensitive UV radiation (Fig.            Figure 6. Kinetics loss UV radiation-induced 6-4 pho- <br /> 5). These observations indicate vivo role ABF1           toproducts genomic DNA. Removal (6-4) photoproducts <br /> DNA repair specifically implicate DNA-binding                measured immunological slot blot assay (see Mate- <br /> domain protein process.                               rials Methods). Monoclonal (6-4) photoproduct antibodies <br />                                                                      used probe presence lesions genomic <br /> DNA-binding domain mutants ABF1 defective                     DNA following exposure cells UV subse- <br /> in photoproduct removal vivo                                      quent incubation noted conditions allow DNA <br />                                                                      repair. Strains maintained permissive (23&#194;&#176;C) and <br /> To directly examine vivo removal DNA damage                semipermissive (32&#194;&#176;C) temperatures (see Materials Meth- <br /> following exposure UV radiation, used slot blot              ods Results details). <br />  <br />  <br />                                                                                                   GENES &amp; DEVELOPMENT            3055 <br />  Reed et al. <br />  <br />  <br />  <br /> pose similar explanation observations involv-       (rad7::LEU2),     MGSC97       (rad7::URA3),       W303236 <br /> ing lack sensitivity UV radiation defective         (rad16::URA3) generously supplied Dr. Jaap Brouwer <br /> NER 6-4 photoproducts vivo permissive tem-       (University Leiden, The Netherlands). Other strains used <br /> perature.                                                     SX46a (MATa, ade2, his3-532, trp1-289, ura3-52) and <br />                                                               isogenic partners MGSC 139 (rad14::LEU2) SX46a <br />   Essentially similar results obtained re- <br />                                                               (rad1::URA3), rad4-10 (MATa, rad4-10, ade2, his3-832, trp1- <br /> moval CPD analyzed (data shown). These ob- <br />                                                               289). SX46A (ubr1::his3, ubr1::HIS3, isogenic SX46A); <br /> servations correlate UV sensitivity abf1-1     SX46Aabf1td (abf1td, isogenic SX46A); SX46A ubr1::HIS3, <br /> abf1-2 mutants directly inability remove UV        abf1td (abf1td, isogenic ubr1::HIS3); YNN281 (MATa trp1D, <br /> radiation-induced lesions DNA.                     his3D200, ura3-52, lys2-801, ade2-1, gal), JCA30 (YNN281, <br />                                                               ABF1 HIS3); JCA31 (YNN281, abf1-1, HIS3; JCA32 (YNN281, <br /> Discussion                                                    abf1-2, HIS3); JCA35 (YNN281, abf1-5, HIS3). These strains <br />                                                               generous gift Judith Campbell (Caltech, CA). W303- <br /> In present study, demonstrated ABF1          1A (MATa, ade2-1, his3-11, 15, leu2-3, 112, trp1-1, ura3-1, can1- <br /> protein component subcomplex nucleo-         100; JRY3765 (MATa, abf1-102, ade2, his3-11, 15, leu2-3, 112, <br /> tide excision repairosome. Our experiments       lys2-801, ura3; JRY4996 (MATa1, abf1-101, ste14, ade2, leu2-3, <br /> addition previously noted roles DNA replica-        112, lys2-801, ura3, kind gift Jasper Rine (Univer- <br />                                                               sity California). Yeast strains abf1td ubr1::abf1td were <br /> tion, transcription, gene silencing, ABF1 protein is <br />                                                               maintained 23&#194;&#176;C YPD medium supplemented 0.5 mM <br /> involved directly process NER. ABF1 there- <br />                                                               CuSO4. The abf1 mutant strains Jasper Rine Judith <br /> fore identified novel DNA repair protein.                Campbell maintained YPD permissive tempera- <br />    The association protein Rad7/Rad16        ture 23&#194;&#176;C. The remaining strains maintained 30&#194;&#176;C in <br /> protein subcomplex, required events       YPD medium. <br /> NER follow bimodal damage-specific incision of <br /> DNA, suggests like Rad7 Rad16 proteins, ABF1 <br />                                                               UV radiation sensitivity <br /> subserves role specialized functions NER. <br /> Our experiments demonstrate abf1 mutants            Sensitivity killing UV radiation measured growing <br /> harbor mutations DNA-binding domain             strains appropriate temperature (see below), following <br />                                                               inoculation starter cultures (5 ml) single colony and <br /> protein, defective ability bind <br />                                                               growing late-log phase. An aliquot culture then <br /> the ABF1 DNA recognition sequence, UV-radia- <br />                                                               used inoculate 25 ml YPD culture grown mid- <br /> tion sensitive defective NER photoproducts. In      log phase prior harvesting. Serial 10-fold dilutions per- <br /> contrast, abf1 mutants DNA-replication defec-        formed culture cells plated 200&#226;&#8364;&#8220;300 <br /> tive, carry mutations regions ABF1,       colonies expected grow unirradiated control plates. <br /> not sensitive killing UV radiation NER          Irradiated plates exposed UV radiation germi- <br /> proficient.                                                   cidal UV lamp fluence 1 J/m2 sec. Plates incu- <br />    A possible explanation findings      bated appropriate temperature dark colonies <br /> overall ability ABF1 protein nonspecifically        large counted. All manipulations car- <br /> bind DNA altered abf1-1 abf1-2 mutants,         ried safe light avoid photoreactivation. Experimental <br />                                                               conditions survival abf1td mutant strain as <br /> resulting defective DNA repair. A intriguing sce- <br />                                                               follows. Cells grown overnight YPD supplemented with <br /> nario ABF1-binding sites play specific role in <br />                                                               CuSO4 (5 mM) 23&#194;&#176;C. Following plating YPD (supple- <br /> NER similar DNA replication. We esti-          mented CuSO4) UV irradiation, cells placed at <br /> mate &#226;&#710;&#188;2000 ABF1-binding sites           32&#194;&#176;C determine survival. For temperature-sensitive <br /> yeast genome. Conceivably, efficiency NER           abf1 mutants defective silencing, cells grown overnight <br /> yeast influenced proximity process       incubated plates post-irradiation 23&#194;&#176;C. Conditions for <br /> such sites. In regard, provocative Rad16   abf1 mutant alleles temperature sensitive replication <br /> a member SWI/SNF family DNA-dependent               follows: strains grown overnight 23&#194;&#176;C sub- <br /> ATPases involved chromatin remodeling.                     sequently incubated 32&#194;&#176;C 28&#194;&#176;C (abf1-1 abf1-5, respec- <br />    Previous biochemical studies reconstituted       tively) determine UV sensitivity. <br /> vitro NER assay shown Rad7 Rad16 stimu- <br /> late NER reaction (Guzder et al. 1997). The authors       Plasmid construction <br /> note ATPase activity Rad16 attenuated       Plasmid pabf1td derived plasmid pPW66R containing <br /> the presence high levels damage, suggest       URA3 gene Pcup1-Ub-Arg-DHFRts-ha-cdc281-95. This <br /> the complex plays role scanning DNA damage.          plasmid [temperature-degradable (td) vector] generously <br /> Future studies necessary elucidate precise role    provided Dr. Alexander Varshavsky (Caltech). The cdc281-95 <br /> of ABF1 protein ABF1 DNA-binding site          fragment removed pPW66R digesting HindIII. <br /> process NER.                                               A 300-bp fragment ABF1-coding region amplified by <br />                                                               PCR using following primers: 5&#226;&#172;&#732;TATAGAAGCTTCCATG- <br />                                                               GACAAATTAGTCGTGAATTATTATGA-3&#226;&#172;&#732; 5&#226;&#172;&#732;-TATAAAGC- <br /> Materials methods                                         TTAAAATGTTTGTTCCTATGCGAATTCTT-3&#226;&#172;&#732;. The PCR <br />                                                               product HindIII sites engineered ends. Digesting <br /> Yeast strains <br />                                                               PCR product HindIII enzyme generated ABF1 frag- <br /> The RAD+ strain W303-B (MATa, ade2-1, trp1-1, leu2-3, 112,    ment cloned directly frame td vector. The <br /> his3-11, 15, ura3-1) isogenic strains MGSC104         ligated product used transform Escherichia coli DH5 &#226;?&#163; <br />  <br />  <br /> 3056          GENES &amp; DEVELOPMENT <br />                                                                                                     ABF1: novel function DNA repair <br />  <br />  <br /> cells transformants selected. The ABF1 fragment             Tandem mass spectrometry performed. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> 112-kD <br /> could cloned orientation clones tested     band, peptide fragment (Mr = 1791.88) identified uniquely the <br /> selected ABF1 fragments correct orientation ABF1      tryptic peptide comprising amino acids 638&#226;&#8364;&#8220;652 ABF1 <br /> expression.                                                          protein (NEDDKLPHEVAEQLR). Two ABF1 peptides <br />                                                                      sequenced digest (data shown). <br /> Construction temperature-degradable yeast abf1 <br /> mutant strain                                                        In vivo DNA repair abf1 mutant alleles <br /> The pabf1td plasmid linearized partial digest        In vivo DNA repair measured growing abf1-1 and <br /> NcoI, cleaves ABF1 fragment. A second NcoI          abf1-5 mutant strains isogenic parental strains permis- <br /> site exists URA3 selectable marker pPW66R          sive temperature 23&#194;&#176;C, inoculating starter culture (5 ml) <br /> vector. Therefore, partial digest conditions necessary       single colony growing late-log phase. An aliquot <br /> linearize plasmid. Yeast strains SX46A SX46A                 culture inoculated 100 ml YPD the <br /> ubr1::HIS3 subsequently transformed. Recombinants            culture grown mid-log phase prior harvesting. Cells were <br /> URA3 prototrophs selected integration abf1td         diluted PBS cell density 2 &#195;&#8212; 107 /ml. Prior irradia- <br /> allele genome confirmed Southern blot analysis.      tion, samples divided equally fractions main- <br />                                                                      tained 3 hr permissive temperature 23&#194;&#176;C the <br />                                                                      semipermissive temperature 32&#194;&#176;C abf1 mutant alle- <br /> Preparation WCE abf1td mutant strain <br />                                                                      les. The strains irradiated dishes depth of <br /> Strain abf1td grown permissive condition (23&#194;&#176;C)        cell suspension &lt;3 mm. Each strain exposed UV <br /> YPD supplemented 0.5 mM CuSO4 mid-log phase              light dose 20 J germicidal lamp fluence 1 <br /> cells collected centrifugation. WCE containing levels     J/m2 sec. An unirradiated control sample prepared, and <br /> ABF1 produced permissive temperature prepared         irradiated samples returned respective tempera- <br /> described previously (Wang et al. 1997). To deplete temperature-     tures dark allow period repair. An aliquot from <br /> degradable ABF1 protein vivo, cells resuspended       sample taken directly irradiation. All manipula- <br /> YPD medium prewarmed 37&#194;&#176;C absence CuSO4             tions carried safe light avoid photoreactivation. <br /> incubated 37&#194;&#176;C 1&#226;&#8364;&#8220;5 hr prior collection preparation     Samples taken times indicated genomic DNA <br /> of WCE.                                                              prepared standard techniques. Equal amounts (5 &#194;&#181;g) DNA <br />                                                                      time point applied Hybond N plus transfer <br />                                                                      membrane slot-blot transfer apparatus. DNA dena- <br /> ABF1 protein antisera <br />                                                                      tured fixed membrane adding NaOH final <br /> ABF1 protein produced recombinant baculovirus vec-          concentration 0.4 M DNA sample. Removal UV <br /> tor. The ABF1 gene (Diffley Stillman 1989) cloned       radiation-induced photoproducts genomic DNA ex- <br /> the transfer vector pVL1392 inserted BaculoGold        amined Western blotting monoclonal CPD 6-4 pho- <br /> baculovirus DNA (PharMingen). The resulting virus plaque         toproduct antibodies (Mori et al. 1991). <br /> purified ABF1 protein purified recombinant bacu- <br /> lovirus-infected SF9 insect cells procedure similar that <br /> described previously (Diffley Stillman 1988). Anti-ABF1p         Acknowledgments <br /> antibody affinity purified described (Diffley Stillman <br />                                                                      We thank Judith Campbell Jasper Rine yeast strains, <br /> 1989). <br />                                                                      Patty Wendel technical assistance laboratory col- <br />                                                                      leagues extensive discussions review manuscript. <br /> In vitro NER                                                         Supported research grant CA12428 U.S. public <br />                                                                      Health Service (ECF). <br /> Yeast WCE capable supporting NER prepared de- <br />                                                                        The publication costs article defrayed by <br /> scribed previously (Wang et al. 1997). NER monitored by <br />                                                                      payment page charges. This article hereby <br /> measuring DNA repair synthesis pUC18 containing N-ace- <br />                                                                      marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 USC section <br /> tyl-2-aminofluorene (AAF) adducts. AAF-modified DNA was <br />                                                                      1734 solely indicate fact. <br /> prepared treating pUC18 AAF purifying DNA <br /> on 5%&#226;&#8364;&#8220;20% sucrose gradient. Plasmid DNA recovered <br /> from extracts restricting HindIII enzyme, linear- <br />                                                                      References <br /> izes substrate reveals discrete band 1% agarose gel <br /> following electrophoresis. In standard NER assay mea-       Bang, D.D., R. Verhage, N. Goosen, J. Brouwer, P. van de <br /> sures repair synthesis, undamaged DNA template used            Putte. 1992. Molecular cloning RAD16, gene involved in <br /> an internal control. Background incorporation radioactivity          differential repair Saccharomyces cerevisiae. Nucleic Ac- <br /> (typically &#226;&#710;&#188;5% damaged plasmid) subtracted            ids Res. 20: 3925&#226;&#8364;&#8220;3931. <br /> from incorporated damaged template. DNA vi-          Brand, A.H., G. Micklem, K. Nasmyth. 1987. A yeast si- <br /> sualized gels UV light staining ethidium            lencer contains sequences promote autonomous <br /> bromide. Repair synthesis visualized autoradiography          plasmid replication transcriptional activation. Cell <br /> dried agarose gels quantitated PhosphorImaging.                  51: 709&#226;&#8364;&#8220;719. <br />                                                                      Buchman, A.R. R.D. Kornberg. 1990. A yeast ARS-binding <br />                                                                         protein activates transcription synergistically combina- <br /> Identification ABF1 protein tandem mass spectrometry <br />                                                                         tion weak activating factors. Mol. Cell. Biol. <br /> Details analysis available request. Proteins         10: 887&#226;&#8364;&#8220;897. <br /> proteolytically trypsinized gel. Extracted peptides      Della Seta, F., I. Treich, J.M. Buhler, A. Sentenac. 1990. <br /> purified reverse-phase microcolumn eluted 60%               ABF1 binding sites yeast RNA polymerase genes. J. Biol. <br /> methanol/5% formic acid nanoelectrospray capillary.              Chem. 265: 15168&#226;&#8364;&#8220;15175. <br />  <br />  <br />                                                                                                   GENES &amp; DEVELOPMENT             3057 <br />  Reed et al. <br />  <br />  <br /> Diffley, J.F. B. Stillman. 1988. Purification yeast protein      tor 1 initiation DNA replication transcriptional <br />    binds origins DNA replication transcrip-              control Saccharomyces cerevisiae. Mol. Cell. Biol. <br />    tional silencer. Proc. Natl. Acad. Sci. 85: 2120&#226;&#8364;&#8220;2124.                 12: 1064&#226;&#8364;&#8220;1077. <br /> &#226;&#8364;&#8221;&#226;&#8364;&#8221;&#226;&#8364;&#8221;. 1989. Similarity transcriptional silencer             Rine, J. I. Herskowitz. 1987. Four genes responsible a <br />    binding proteins ABF1 RAP1. Science 246: 1034&#226;&#8364;&#8220;1038.                position effect expression HML HMR Sac- <br /> Dohmen, R.J., P. Wu, A. Varshavsky. 1994. Heat-inducible              charomyces cerevisiae. Genetics 116: 9&#226;&#8364;&#8220;22. <br />    degron: A method constructing temperature-sensitive             Terleth, C., P. Schenk, R. Poot, J. Brouwer, P. van Putte. <br />    mutants. Science 263: 1273&#226;&#8364;&#8220;1276.                                       1990. Differential repair UV damage rad mutants of <br /> Feldman, J.B., J.B. Hicks, J.R. Broach. 1984. Identification       Saccharomyces cerevisiae: A possible function G2 arrest <br />    sites required repression silent mating type locus         UV irradiation. Mol. Cell. Biol. 10: 4678&#226;&#8364;&#8220;4684. <br />    yeast. J. Mol. Biol. 178: 815&#226;&#8364;&#8220;834.                                  Verhage, R., A.M. Zeeman, N. Groot, F. Gleig, D.D. Bang, P. <br /> Friedberg, E.C., G.C. Walker, W. Siede. 1995. DNA repair              van Putte, J. Brouwer. 1994. The RAD7 RAD16 <br />    mutagenesis, ASM Press, Washington, DC.                            genes, essential pyrimidine dimer removal <br /> Guzder, S.N., P. Sung, L, Prakash S. Prakash, 1997. The               silent mating type loci, required repair <br />    DNA dependent ATPase activity yeast nucleotide exci-                nontranscribed strand active gene Saccharo- <br />    sion repair factor 4 role DNA damage recognition.           myces cerevisiae. Mol. Cell. Biol. 14: 6135&#226;&#8364;&#8220;6142. <br />    J. Biol. Chem. 272: 21665&#226;&#8364;&#8220;21668.                                    Walker, S.S., S.C. Francesconi, B.K. Tye, S. Eisenberg. 1989. <br /> Huang, W., W.J. Feaver, A.E. Tomkinson, E.C. Friedberg.               The OBF1 protein DNA-binding site important <br />    1998. The N-degron protein degradation strategy inves-             function autonomously replicating sequence in <br />    tigating function essential genes: Requirement rep-         Saccharomyces cerevisiae. Mol. Cell. Biol. 9: 2914&#226;&#8364;&#8220;2921. <br />    lication protein A proliferating cell nuclear antigen pro-      Wang, Z., S. Wei, S.H. Reed, X. Wu, J.Q. Svejstrup, W.J. Feaver, <br />    teins nucleotide excision repair yeast extracts. Mutat.         R.D. Kornberg, E.C. Friedberg. 1997. The RAD7, <br />    Res. 408: 183&#226;&#8364;&#8220;194.                                                     RAD16, RAD23 genes Saccharomyces cerevisiae: Re- <br /> Klar, A.J., S. Fogel, K. Lusnak. 1979. Gene conversion         quirement transcription-independent nucleotide exci- <br />    mating-type locus Saccharomyces cerevisiae. Genetics                sion repair vitro interactions gene prod- <br />    92: 777&#226;&#8364;&#8220;782.                                                           ucts. Mol. Cell. Biol. 17: 635&#226;&#8364;&#8220;643. <br /> Loo, S., P. Laurenson, M. Foss, A. Dillin, J. Rine. 1995. Roles <br />    ABF1, NPL3, YCL54 silencing Saccharomyces <br />    cerevisiae. Genetics 141: 889&#226;&#8364;&#8220;902. <br /> Marahrens, Y. B. Stillman. 1992. A yeast chromosomal ori- <br />    gin DNA replication defined multiple functional ele- <br />    ments. Science 255: 817&#226;&#8364;&#8220;823. <br /> McNally, F.J. J. Rine. 1991. A synthetic silencer mediates <br />    SIR-dependent functions Saccharomyces cerevisiae. Mol. <br />    Cell. Biol. 11: 5648&#226;&#8364;&#8220;5659. <br /> Mori, T., M. Nokane, T. Hattori, T. Matsunaga, M. Ihara, O. <br />    Nikaido. 1991. Simultaneous establishment monoclonal <br />    antibodies specific cylcobutane pyrimidine dimer <br />    (6-4) photoproduct mouse immunized with <br />    UV irradiated DNA. Photochem. Photobiol. 54: 225&#226;&#8364;&#8220;232. <br /> Paetkau, D.W., J.A. Riese, W.S. MacMorran, R.A. Woods, and <br />    R.D. Gietz. 1994. Interaction yeast RAD7 SIR3 <br />    proteins: Implications DNA repair chromatin struc- <br />    ture. Genes &amp; Dev. 8: 2035&#226;&#8364;&#8220;2045. <br /> Rao, H. B. Stillman. 1995. The origin recognition complex <br />    interacts bipartite DNA binding site yeast <br />    replicators. Proc. Natl. Acad. Sci. 92: 2224&#226;&#8364;&#8220;2228. <br /> Rao, H., Y. Marahrens, B. Stillman. 1994. Functional con- <br />    servation multiple elements yeast chromosomal repli- <br />    cators. Mol. Cell. Biol. 14: 7643&#226;&#8364;&#8220;7651. <br /> Reed, S.H., S. Boiteux, R. Waters. 1996. UV-induced endo- <br />    nuclease III-sensitive sites mating type loci Saccha- <br />    romyces cerevisiae repaired nucleotide excision re- <br />    pair: RAD7 RAD16 required removal <br />    HML alpha. Mol. &amp; Gen. Genet. 250: 505&#226;&#8364;&#8220;514. <br /> Reed, S.H., Z. You, E.C. Friedberg. 1998. The yeast RAD7 <br />    RAD16 genes required postincision events during <br />    nucleotide excision repair. In vitro vivo studies with <br />    rad7 rad16 mutants purification Rad7/Rad16- <br />    containing protein complex. J. Biol. Chem. 273: 29481&#226;&#8364;&#8220; <br />    29488. <br /> Rhode, P.R., K.S. Sweder, K.F. Oegema, J.L. Campbell. 1989. <br />    The gene encoding ARS-binding factor I essential the <br />    viability yeast. Genes &amp; Dev 3: 1926&#226;&#8364;&#8220;193 <br /> </body></html>