<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                 Volume 15 Number 9           September 2013          pp. 1075&#226;&#8364;&#8220;1085        1075 <br /> www.neoplasia.com <br />  <br />  <br />  <br />  <br />                                                                                     Jing Shao*,&#226;&#8364;&#160;, Yong Teng&#226;&#8364;&#161;, Ravi Padia&#226;&#8364;&#160;, <br />           COP1 GSK3&#206;&#178; Cooperate                                                  Sungguan Hong&#226;&#8364;&#160;, Hyangsoon Noh&#226;&#8364;&#160;, Xiayang Xie&#194;&#167;, <br />           Promote c-Jun Degradation                                              Jeff S. Mumm&#194;&#167;, Zheng Dong&#194;&#167;, Han-Fei Ding&#226;&#8364;&#161;, <br />                                                                                     John Cowell&#226;&#8364;&#161;, Jaejik Kim&#226;&#8364;&#161;,&#194;&#182;, Jiahuai Han* <br />           Inhibit Breast Cancer                                                 Shuang Huang&#226;&#8364;&#160;,# <br />           Cell Tumorigenesis1,2                                                     *State Key Laboratory Cellular Stress Biology, School <br />                                                                                     Life Sciences, Xiamen University, Xiamen, Fujian, <br />                                                                                     China; &#226;&#8364;&#160;Department Biochemistry Molecular Biology, <br />                                                                                     Medical College Georgia, Georgia Regents University, <br />                                                                                     Augusta, GA; &#226;&#8364;&#161;Cancer Center, Georgia Regents University, <br />                                                                                     Augusta, GA; &#194;&#167;Department Cellular Biology Anatomy, <br />                                                                                     Medical College Georgia, Georgia Regents University, <br />                                                                                     Augusta, GA; &#194;&#182;Department Biostatistics, Georgia <br />                                                                                     Regents University, Augusta, GA; #E-Institute Shanghai <br />                                                                                     Municipal Education Committee, Shanghai University <br />                                                                                     Traditional Chinese Medicine, Shanghai, China <br />  <br />  <br />  <br />  <br />           <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />           High abundance c-Jun detected invasive breast cancer cells aggressive breast tumor malignancies. Here, <br />           demonstrate major cause high c-Jun abundance invasive breast cancer cells prolonged c-Jun <br />           protein stability owing poor poly-ubiquitination c-Jun. Among known c-Jun&#226;&#8364;&#8220;targeting E3 ligases, iden- <br />           tified constitutive photomorphogenesis protein 1 (COP1) E3 ligase responsible c-Jun degradation less <br />           invasive breast cancer cells depletion COP1 reduced c-Jun poly-ubiquitination leading stabilization <br />           c-Jun protein. In panel breast cancer cell lines, observed inverse association levels COP1 <br />           c-Jun. However, overexpressing COP1 unable decrease c-Jun level invasive breast cancer cells, <br />           indicating efficient c-Jun protein degradation necessitates additional event. Indeed, glycogen <br />           synthase kinase 3 (GSK3) inhibitors elevated c-Jun abundance invasive breast cancer cells GSK3&#206;&#178; <br />           nonphosphorylable c-Jun&#226;&#8364;&#8220;T239A mutant displayed greater protein stability poorer poly-ubiquitination compared <br />           wild-type c-Jun. The ability simultaneously enforced expression COP1 constitutively active GSK3&#206;&#178; <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br />           decrease c-Jun abundance invasive breast cancer cells allowed conclude c-Jun negatively regulated <br />           coordinated action COP1 GSK3&#206;&#178;. Importantly, co-expressing COP1 active GSK3&#206;&#178; blocked <br />           vitro cell growth/migration vivo metastasis invasive breast cancer cells. Gene expression profiling breast <br />           tumor specimens revealed higher COP1 expression correlated better recurrence-free <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>survival</span>. Our <br />           study supports notion COP1 suppressor breast cancer progression. <br />  <br />           Neoplasia (2013) 15, 1075&#226;&#8364;&#8220;1085 <br />  <br />  <br />  <br />  <br /> Address correspondence to: Shuang Huang, PhD, Department Biochemistry Molecular Biology, Medical College Georgia, Georgia Regents University, Augusta, <br /> GA 30912. E-mail: shuang@gru.edu <br /> 1 <br />   This work supported funding National Institutes Health HL083335, Shanghai Eastern Scholar Fund, E-Institutes Shanghai Municipal Education <br /> Commission Project E03008. <br /> 2 <br />  This article refers supplementary materials, designated Figures W1 W5 Table W1 available online www.neoplasia.com. <br /> Received 11 May 2013; Revised 27 June 2013; Accepted 2 July 2013 <br /> Copyright &#194;&#169; 2013 Neoplasia Press, Inc. All rights reserved 1522-8002/13/$25.00 <br /> DOI 10.1593/neo.13966 <br />  1076         Regulation c-Jun COP1 GSK3&#206;&#178;            Shao et al.                                             Neoplasia Vol. 15, No. 9, 2013 <br />  <br /> Introduction                                                                         The objective study answer protein level of <br /> c-Jun member activating protein 1 family transcription             c-Jun regulated breast cancer cells. By performing cycloheximide- <br /> factors [1]. By forming heterodimers members activat-               chasing experiment, c-Jun protein more <br /> ing protein 1 family, c-Jun regulates expression variety              stable invasive breast cancer cells invasive ones. Pro- <br /> genes important diverse cellular functions including cell growth,             longed c-Jun protein stability c-Jun being <br /> cell migration, invasion [2]. As identified proto-           poly-ubiquitinated invasive breast cancer cells. To characterize the <br /> oncogenes, extensive studies exerted characterize                mechanism mediating c-Jun protein degradation, showed that <br /> role c-Jun cancer development including breast cancer. An early             depleting COP1 inhibiting GSK3 activity prevented poly- <br /> study showed forced c-Jun expression able convert non-                ubiquitination c-Jun increased c-Jun abundance invasive <br /> invasive/hormone-dependent breast cancer MCF7 line inva-                    breast cancer cells. Moreover, simultaneous enforcement of <br /> sive hormone-resistant line [3]. A later study using clinical breast          COP1 constitutively active GSK3&#206;&#178; (GSK3&#206;&#178;S9A) expression was <br /> tumor specimens revealed c-Jun detected invasive            able effectively downregulate c-Jun invasive breast cancer cells. <br /> of breast tumors level correlated increased angiogenesis             These results indicate high COP1 expression GSK3&#206;&#178;- <br /> [4]. The role c-Jun promote breast tumor progression metas-             mediated phosphorylation necessary c-Jun protein turnover. <br /> tasis supported recent vivo studies: 1) depletion c-Jun           This notion supported observation non&#226;&#8364;&#8220;GSK3&#206;&#178;- <br /> reduced cell migration invasion ErbB2-induced mammary                      phosphorylable c-Jun significantly stable wild-type <br /> tumors ErbB2 mammary tumor transgenic mouse model [5]                      c-Jun. Importantly, showed co-expressing COP1 and <br /> 2) overexpression c-Jun sufficient confer nonmetastatic                 GSK3&#206;&#178;S9A suppressed vitro cell growth/migration vivo <br /> breast cancer cells capability metastasize [6]. An active             metastasis invasive breast cancer cells. Gene expression analysis of <br /> role c-Jun breast tumorigenesis lie ability promote           breast tumor specimens revealed COP1 mRNA expres- <br /> cell proliferation [7] migration/invasion [8].                                sion independent positive prognostic factor recurrence-free <br />     High abundance c-Jun detected various aggressive tumor               survival patients breast cancer. <br /> types [9,10]. Immunohistochemistry revealed c-Jun level was <br /> often low present cells normal benign breast <br /> tissues; contrast, immunoreactivity c-Jun detected high              Materials Methods <br /> intensity usually detected significant percentage cells breast <br /> carcinoma specimens [11]. With limited number human breast                 Cell Culture, Inhibitors, siRNAs <br /> cancer cell lines, previously showed c-Jun             All cell lines used study purchased American Tissue <br /> higher invasive lines invasive ones [8]. Although abun-          Culture Collection (ATCC, Manassas, VA) cultured Dulbecco&#226;&#8364;&#8482;s <br /> dance c-Jun governed transcriptional, posttranscriptional,           modified Eagle&#226;&#8364;&#8482;s medium supplemented 10% FBS humidi- <br /> translational, posttranslational levels [2], mechanism             fied atmosphere containing 5% CO2 37&#194;&#176;C. Inhibitors used the <br /> elevated c-Jun level invasive breast cancer cells understood.      study PYR-41 Calbiochem (San Diego, CA); MG132 and <br />     Under normal circumstances, c-Jun protein known                     Chloroquine Tocris (Bristol, United Kingdom); SP600125, <br /> be highly unstable [12] level regulated                  SB216763, 4,5,6,7-tetrabromobenzotriazole (TBB) Sigma <br /> ubiquitination/proteasome-dependent mechanism [13]. Ubiquitin E3 li-              (St Louis, MO). All E3 ligase&#226;&#8364;&#8220;targeting siRNAs control siRNA <br /> gases add poly-ubiquitin chain c-Jun include constitutive             purchased Thermo Scientific (West Palm Beach, FL). <br /> photomorphogenesis protein 1 (COP1) [14], cullin 4 (CUL4) [15], <br /> F-box WD repeat domain containing 7 E3 ubiquitin protein ligase               Determination c-Jun Protein Stability <br /> (FBW7) [16], Itchy E3 ubiquitin protein ligase (ITCH) [17], mitogen-                 Overnight cultured MCF7, MDA-MB-231, MDA-MB-436, <br /> activated protein kinase kinase kinase 1 (MEKK1) [18], sensitive           T47D cells co-transfected plasmids encoding human <br /> apoptosis, zinc ring finger protein (SAG) [19]. COP1 unique               influenza hemagglutinin (HA)-tagged c-Jun green fluorescent <br /> others dual ability act E3 ligase adaptor   protein (GFP) 1 day. Cells trypsinized reseeded for <br /> to recruit substrate de-etiolated homolog 1/damage-specific DNA                12 hours followed addition 10 &#206;&#188;g/ml cycloheximide. At vary- <br /> binding protein 1/cullin 4/ring-box 1, E3 ubiquitin protein ligase                ing time points, cells harvested cell lysates subjected <br /> (DET1/DDB1/CUL4/RBX1) ubiquitin ligase complex [15]. The ex-                      immunoblot analysis detect HA-tagged c-Jun GFP using <br /> pression E3 ligases displays tissue specificity varies various    respective antibodies. The intensities bands quantified by <br /> types stages cancers; therefore, great investigate    Odyssey Image System (LI-COR Biosciences, Lincoln, NE). <br /> whether c-Jun&#226;&#8364;&#8220;targeting E3 ligases involved <br /> in regulation c-Jun protein abundance breast cancer cells.              Immunoprecipitation <br />     Function c-Jun shown regulated phosphoryla-                 To determine c-Jun poly-ubiquitination breast cancer cells, <br /> tion. Casein kinase II phosphorylates c-Jun, leading suppres-              HA-tagged c-Jun Flag-tagged ubiquitin co-transfected into <br /> sion c-Jun activity [20]. In contrast, c-Jun N-terminal kinases                MDA-MB-231, MDA-MB-436, MCF7, T47D cells 1 day <br /> (JNKs) activate c-Jun directly phosphorylating c-Jun [21]. Interest-           followed treatment MG132 8 hours. Cells lysed in <br /> ingly, JNKs activate c-Jun&#226;&#8364;&#8220;targeting E3 ligase ITCH, estab-          buffer containing 1% sodium dodecyl sulfate, 150 mM NaCl, and <br /> lishing negative feedback loop [17]. Recognition c-Jun E3 ligase          10 mM Tris-HCl (pH 8.0) followed 10-minute boiling. Samples <br /> FBW7 subsequent degradation necessitate c-Jun phos-               sheared sonication device diluted buffer contain- <br /> phorylated glycogen synthase kinase 3 beta (GSK3&#206;&#178;) [22]. However,              ing 10 mM Tris-HCl (pH 8.0), 150 mM NaCl, 1 mM EDTA, and <br /> it unknown c-Jun phosphorylation prerequisite deg-        1% Triton X-100. After 30-minute centrifugation 20,000g, the <br /> radation COP1 others.                                                      resulting supernatants immunoprecipitated HA monoclonal <br />  Neoplasia Vol. 15, No. 9, 2013                                             Regulation c-Jun COP1 GSK3&#206;&#178;         Shao et al.     1077 <br />  <br /> antibody (mAb) immunoprecipitates subjected immuno-         using 24-well formatted collagen-coated transwell previously <br /> blot analysis detect Flag-ubiquitin using Flag mAb.                  described [8]. Briefly, 1 &#195;&#8212; 105 cells 100 &#206;&#188;l serum-free medium <br />                                                                         added upper chamber transwell allowed to <br /> Cell Fractionation                                                      migrate 4 hours. Nonmigrating cells upper chambers were <br />    Cell fractionation performed previously described          removed cotton swabs, migrating cells lower surface of <br /> [23]. Briefly, cells lysed Buffer A [10 mM Hepes (pH 7.9),      chamber stained using crystal violet. Amount migrating <br /> 1.5 mM MgCl2, 10 mM KCl, 0.5 mM PMSF, 0.5 mM DTT]                   cells determined counting stained cells phase- <br /> followed centrifugation 2000g 10 minutes obtain crude    contrast microscope. <br /> nuclear pellet cytoplasmic supernatant (cytoplasmic fraction). <br /> The crude nuclear pellets purified resuspension      Zebrafish Metastatic Model <br /> Buffer B [20 mM Hepes (pH 7.9), 25% (vol/vol) glycerol, 0.45 M              The effect combined expression COP1 GSK3&#206;&#178;S9A was <br /> NaCl, 1.5 mM MgCl 2 , 1 mM EDTA, 0.5 mM PMSF,                       evaluated recently developed zebrafish metastatic model [24]. <br /> 0.5 mM DTT]. The lysates subjected high-speed centrifuga-       All experimental procedures approved Institutional <br /> tion 1 hour obtain nuclear fraction.                             Animal Care Use Committee Georgia Regents University. Briefly, <br />                                                                         control COP1/GSK3&#206;&#178;S9A-expressing MDA-MB-231 cells were <br /> Lentiviral Constructs Packaging                                     labeled fluorescent dye CM-Dil (Life Technologies, Carlsbad, <br />    Coding sequences COP1 GSK3&#206;&#178;S9A mutant sub-               CA), approximately 200 labeled cells microinjected <br /> cloned pCDH Lentivector (System Biosciences, Mountain View,        perivitelline space 48-hour post fertilization (hpf) zebrafish <br /> CA). Lentiviruses packaged 293FT cells according         embryos using pressure microinjector. The embryos kept at <br /> manufacturer&#226;&#8364;&#8482;s instruction.                                             34&#194;&#176;C imaged anaesthetic confocal microscopy at <br />                                                                         48-hour post injection. <br /> Nuclear Proteasome Activity Assay <br />    Nuclear 20S proteasome activities determined using overnight    Statistical Analysis <br /> cultured cells aid 20S Proteasome Assay Kit (Cayman            Statistical analyses laboratory studies performed either <br /> Chemical, Ann Arbor, MI) measures ability fractions     Student&#226;&#8364;&#8482;s t test analysis variance. To determine potential <br /> to cleave proteasome-specific substrate SUC-LLVY-AMC. Briefly, cells    correlation COP1 expression clinicopathologic param- <br /> were resuspended Buffer A [10 mM Hepes, 10 mM KCl, 2 mM              eters, obtained data COP1 expression microarray analysis <br /> MgCl2, 0.1 mM EDTA, 1 mM PMSF (pH7.9)] incubated                clinical features 126 patients breast cancer re- <br /> on ice 25 minutes followed adding 10% NP-40 final         ceived tamoxifen monotherapy, chemotherapy, radiation therapy <br /> concentration 1%. Cells vortexed 15 seconds        Gene Expression Omnibus database (91 samples <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> microcentrifuged 5000g 5 minutes. The collected pellets (cell    GSE18229 35 samples GSE40954). The cutoff value of <br /> nucleus) analyzed 20S proteasome activity.                     high low COP1 expression groups based median <br />                                                                         value. Associations COP1 expression clinicopathologic <br /> Immunofluorescence Staining                                             parameters determined using Pearson &#207;&#8225;2 test. In univariate sur- <br />    Cells culture collagen-coated glass coverslips overnight,    vival analysis, Kaplan-Meier method log-rank test used <br /> then fixed 4% paraformaldehyde permeabilize 1%            compare recurrence-free survival curves high low <br /> Triton X-100 15 minutes. To visualize c-Jun proteasome          COP1 expression groups. In multivariate survival analysis, Cox <br /> S5A subunit, coverslips incubated c-Jun mAb S5A           proportional hazard regression model used identify important <br /> polyclonal antibody followed incubation fluorescein isothio-    factors disease-free survival. To identify factors, initial <br /> cyanate rhodamine-labeled secondary antibodies. The fluorescence     model COP1 expression considered particular <br /> staining visualized fluorescence microscope (Axiovert       clinicopathologic parameter added model time. By <br /> 200M; Zeiss, Munich, Germany). 4&#226;&#8364;&#178;,6-Diamidino-2-phenylindole            repeating procedure, final regression model important <br /> (DAPI) used stain nucleus breast cancer cells.                factors identified likelihood ratio test. P &lt; .05 considered <br />                                                                         statistically significant. <br /> Cell Proliferation Study <br />    To determine ability enforced COP1 GSK3&#206;&#178;S9A <br /> expression affect cell proliferation, MDA-MB-231 MDA-            Results <br /> MB-436 cells transduced lentiviral vector vector <br /> containing COP1 GSK3&#206;&#178;S9A 3 days. Transduced cells           Ubiquitin/Proteasome-Dependent Mechanism Controls <br /> subjected 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium       c-Jun Abundance Breast Cancer Cells <br /> bromide (MTT) assay evaluate cell proliferation previously           With limited human breast cancer cell lines, previously showed <br /> described [23]. Briefly, 5 &#195;&#8212; 104 cells seeded 24-well cul-    abundance c-Jun higher invasive breast can- <br /> ture plates allowed grow 1 3 days addition     cer cells invasive ones [8]. To extend early finding, <br /> of MTT. Cell proliferation determined reading plates          performed immunoblot analysis examine c-Jun abundance <br /> 595 nm dissolving MTT formazan DMSO.                         10 established breast cancer cell lines included 5 invasive and <br />                                                                         5 invasive lines (Figure 1B). High level abundance c-Jun <br /> Cell Migration Assay                                                    seen invasive lines, low barely detectable <br />   The ability enforced COP1 GSK3&#206;&#178;S9A expression affect cell   invasive cell lines (Figure 1, A B). To determine whether <br /> migration MDA-MB-231 MDA-MB-436 cells determined             difference c-Jun protein stability invasive <br />  1078        Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.                                           Neoplasia Vol. 15, No. 9, 2013 <br />  <br />  <br />  <br />  <br /> Figure 1. Abundance c-Jun breast cancer cells determined protein stability. (A) Immunoblot analysis c-Jun breast cancer <br /> cell lines. (B) In vitro invasion breast cancer cell lines. Data means &#194;&#177; SE (n = 3). (C) Cells co-transfected HA-tagged <br /> c-Jun GFP 24 hours followed addition 20 &#206;&#188;g/ml cycloheximide varying times. Cells lysed, cell lysates were <br /> subjected immunoblot analysis detect HA-tagged c-Jun GFP. The relative HA-tagged c-Jun standardized GFP. <br /> (D) Cells treated vehicle (DMSO), 50 &#206;&#188;M PYR-41, 10 &#206;&#188;M MG-132 8 hours lysed immunoblot analysis detect <br /> c-Jun &#206;&#178;-actin. (E) T47D cells treated 10 &#206;&#188;M MG-132 varying times lysed immunoblot analysis detect <br /> c-Jun &#206;&#178;-actin. (F) Cells treated 50 &#206;&#188;M chloroquine, 10 mM NH4Cl, 10 &#206;&#188;M MG-132 8 hours lysed immunoblot <br /> analysis detect c-Jun &#206;&#178;-actin. <br />  <br />  <br />  <br /> and invasive lines, co-transfected HA-tagged c-Jun GFP         ure 1C ), results indicate greater c-Jun protein stability <br /> into invasive (MDA-MB-231 MDA-MB-436)                 contributing factor high c-Jun abundance invasive breast <br /> invasive lines (MCF7 T47D) followed performing cyclo-               cancer cells. <br /> heximide-chasing experiment measure stability HA-tagged             c-Jun protein degraded ubiquitin/proteasome- <br /> c-Jun protein lines. HA-tagged c-Jun relatively stable        dependent mechanism various cell types, including melanoma <br /> and detected 40% reduction                lung cancer cells [25,26]. To determine ubiquitin/ <br /> HA-tagged c-Jun MDA-MB-231 MDA-MB-436 cells                   proteasome plays role regulating c-Jun abundance breast <br /> 2 hours cycloheximide addition (Figure 1C ). In contrast,          cancer cells, treated MCF7, MDA-MB-231, MDA-MB-436, and <br /> than 80% HA-tagged c-Jun degraded time                  T47D cells ubiquitin E1 ligase inhibitor PYR-41 [27] <br /> period T47D MCF7 cells (Figure 1C). Because GFP exhibited           proteasome inhibitor MG132 8 hours. Immunoblot analysis <br /> similar protein stability invasive invasive lines (Fig-   showed treatment inhibitor markedly increased the <br />  Neoplasia Vol. 15, No. 9, 2013                                                 Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.    1079 <br />  <br /> amount c-Jun invasive cells (MCF7 T47D), did        Status c-Jun Poly-ubiquitination Determines c-Jun <br /> not significantly alter c-Jun abundance invasive cells (MDA-MB-231       Abundance Breast Cancer Cells <br /> and MDA-MB-436; Figure 1D). MG132-induced accumulation                      To determine difference c-Jun protein stability is <br /> c-Jun protein rapid; 1 hour MG132 treatment elevated            linked variation proteasome activity invasive and <br /> more eight-fold c-Jun level T47D cells (Figure 1E). These        invasive cells, examined subcellular localization of <br /> results suggest ubiquitin/proteasome responsible         c-Jun MCF7 MDA-MB-436 cells. Immunoblot analysis of <br /> efficient c-Jun protein degradation invasive breast cancer cells.   cytoplasmic nuclear fractions showed c-Jun simi- <br /> Because lysosome regulate protein stability, treated          larly present nucleus lines, nuclear localization <br /> MCF7 T47D cells lysosome inhibitors chloroquine                c-Jun affected MG132 treatment (Figure 2A). We <br /> NH4Cl 8 hours. Immunoblot analysis showed signifi-         measured proteasome activities nucleus MCF7, <br /> cantly altered c-Jun level (Figure 1F), excluding participation    MDA-MB-231, MDA-MB-436, T47D cells. Up 1-hour <br /> of lysosome c-Jun protein degradation.                                   reaction period, did observe significant difference nuclear <br />  <br />  <br />  <br />  <br /> Figure 2. The extent poly-ubiquitination c-Jun different invasive invasive breast cancer cells. (A) Cells were <br /> treated 10 &#206;&#188;M MG132 left untreated 8 hours. Cells lysed preparation cytoplasmic (Cy) nuclear (Nu) frac- <br /> tions, fractions analyzed immunoblot analysis detect c-Jun histone deacetylase 1 (HDAC1; indicator nuclear fraction). <br /> (B) Proteasome activities breast cancer cell lines. Data means &#194;&#177; SE (n = 3). (C) Immunofluorescence staining c-Jun PMSA5 <br /> in MDA-MB-231 10 &#206;&#188;M MG-132&#226;&#8364;&#8220;treated T47D cells. DAPI used nuclear staining. (D) Cells co-transfected HA-tagged <br /> c-Jun Flag-tagged ubiquitin 1 day followed 10 &#206;&#188;M MG-132 treatment 8 hours. Cells lysed, cell lysates immuno- <br /> precipitated HA mAb, immunoprecipitates subjected immunoblot analysis detect Flag-tagged ubiquitin (Ub). <br />  1080        Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.                                         Neoplasia Vol. 15, No. 9, 2013 <br />  <br />  <br />  <br />  <br /> Figure 3. COP1 E3 ligase mediating c-Jun turnover breast cancer cells. (A) T47D MCF7 cells transfected 50 nM <br /> siRNA pool CUL4, FBW7, ITCH, MEKK1, COP1, SAG 3 days. Cells lysed, cell lysates analyzed by <br /> immunoblot analysis detect c-Jun, COP1, &#206;&#178;-actin. (B) T47D MCF7 transfected sequence-distinct COP1 siRNAs <br /> (50 nM) lysed immunoblot analysis detect c-Jun, COP1, &#206;&#178;-actin. (C) T47D cells transfected 50 nM control <br /> or COP1 siRNA 2 days co-transfected HA-tagged c-Jun GFP 1 day. After adding 20 &#206;&#188;g/ml cycloheximide for <br /> varying times, cells harvested cell lysates analyzed immunoblot analysis detect HA-tagged c-Jun, GFP, COP1. <br /> The relative HA-tagged c-Jun standardized GFP. (D) T47D cells treated 50 nM control <br /> or COP1 siRNA 2 days co-transfected HA-tagged c-Jun Flag-tagged ubiquitin 1 day. Cells treated with <br /> 10 &#206;&#188;M MG-132 treatment 8 hours subjected immunoprecipitation HA mAb. Immunoprecipitates analyzed immuno- <br /> blot analysis detect Flag-tagged ubiquitin (Ub). (E) Immunoblot analysis c-Jun COP1 breast cancer cell lines. <br />  <br />  <br /> proteasome activity lines (Figure 2B). Immunofluores-          precipitates anti&#226;&#8364;&#8220;Flag mAb showed HA-tagged c-Jun was <br /> cence staining showed similar <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span> c-Jun protea-       robustly poly-ubiquitinated MCF-7 T47D cells, the <br /> somal PMSA5 co-localization MDA-MB-231 MG132-treated                poly-ubiquitination HA-tagged c-Jun weak MDA- <br /> T47D cells (Figure 2C ). Identical pattern observed            MB-231 MDA-MB-436 cells (Figure 2D). These results suggest <br /> MDA-MB-436 MG132-treated MCF7 cells (Figure W1). These                 effectiveness c-Jun poly-ubiquitinated determines <br /> results indicate difference c-Jun protein stability    abundance c-Jun breast cancer cells. <br /> invasive invasive breast cancer cells caused the <br /> variation nuclear proteasome activity co-localization c-Jun       COP1 E3 Ligase Is Required c-Jun Degradation in <br /> with proteasome.                                                       Breast Cancer Cells <br />    Subsequently, investigated status c-Jun poly-ubiquitination     c-Jun substrate CUL4, FBW7, ITCH, MEKK1, COP1, <br /> in MG132-treated MCF7, MDA-MB-231, MDA-MB-436,                         SAG E3 ligases. To identify E3 ligase regulates c-Jun <br /> T47D cells co-transfected HA-tagged c-Jun               abundance breast cancer cells, treated MCF7 T47D cells <br /> and Flag-tagged ubiquitin. Immunoblot analysis HA immuno-               predesigned siRNA pool E3 ligases. While <br />  Neoplasia Vol. 15, No. 9, 2013                                              Regulation c-Jun COP1 GSK3&#206;&#178;     Shao et al.   1081 <br />  <br /> all effectively knocked respective targets (Fig-      knockdown T47D cells control T47D cells (Figure 3C ). <br /> ure W2), COP1 siRNA pool markedly elevated               Immunoprecipitation experiment revealed extent <br /> of c-Jun cell lines (Figure 3A). To confirm results gener-   c-Jun poly-ubiquitination significantly reduced COP1- <br /> ated siRNA pool, MCF7 T47D cells treated          knockdown cells compared control cells (Figure 3D). <br /> sequence distinct COP1 siRNAs 3 days. Immunoblot analysis               To substantiate findings, investigated potential func- <br /> showed siRNAs increased c-Jun abundance (Figure 3B).        tional link c-Jun COP1 breast cancer cell <br /> We examined effect COP1 knockdown c-Jun pro-              lines. Immunoblot analysis revealed lines relatively high <br /> tein stability introducing HA-tagged c-Jun control          c-Jun abundance (BT549, Hs578t, MDA-MB-231, MDA- <br /> and COP1-knockdown T47D cells. Cycloheximide-chasing experi-             436) displayed COP1 relatively low <br /> ment showed HA-tagged c-Jun stable COP1-                c-Jun abundance (BT474, MCF7 T47D; Figure 3E ). This <br />  <br />  <br />  <br />  <br /> Figure 4. Activity GSK3&#206;&#178; required efficient c-Jun protein turnover breast cancer cells. (A) COP1 lentivirally overexpressed <br /> in MDA-MB-231 MDA-MB-436 cells. Populations cells lysed immunoblot analysis detect COP1, c-Jun, &#206;&#178;-actin. <br /> (B) T47D MCF7 cells treated 10 &#206;&#188;M SP600125, 10 mM LiCl, 10 &#206;&#188;M SB216763, 50 &#206;&#188;M TBB 12 hours lysed <br /> for immunoblot analysis detect c-Jun &#206;&#178;-actin. (C) HA-tagged c-Jun c-Jun&#226;&#8364;&#8220;T239A co-transfected T47D cells GFP for <br /> 1 day followed addition 20 &#206;&#188;g/ml cycloheximide varying times. Cells harvested, cell lysates analyzed by <br /> immunoblot analysis detect HA-tagged c-Jun GFP. The relative HA-tagged c-Jun standardized GFP. <br /> (D) T47D cells co-transfected HA-tagged c-Jun Flag-tagged ubiquitin 1 day treated 10 &#206;&#188;M MG-132 for <br /> 8 hours. Cells lysed, cell lysates subjected immunoprecipitation HA mAb. Immunoprecipitates analyzed by <br /> immunoblot analysis detect Flag-tagged ubiquitin (Ub). (E) MDA-MB-231 MDA-MB-436 cells lentivirally transduced with <br /> COP1 constitutively active GSK3&#206;&#178; 3 days. Cells lysed, cell lysates subjected immunoblot <br /> analysis detect COP1, GSK3, c-Jun, &#206;&#178;-actin. <br />  1082       Regulation c-Jun COP1 GSK3&#206;&#178;   Shao et al.                                          Neoplasia Vol. 15, No. 9, 2013 <br />  <br />  <br />  <br />  <br /> Figure 5. Simultaneously expressing COP1 constitutively active GSK3&#206;&#178; suppress breast tumorigenesis. (A) MDA-MB-231 and <br /> MDA-MB-436 cells lentivirally transduced COP1 constitutively active GSK3&#206;&#178; 3 days. Popula- <br /> tions transduced cells subjected MTT assay measure cell proliferation. Data means &#194;&#177; SE (n = 3). #P &lt; .01 <br /> versus vector (control). (B) Populations transduced cells subjected transwell assay measure cell migration. Data <br /> are means &#194;&#177; SE (n = 3). *P &lt; .001 versus vector (control). (C) Control COP1/GSK3&#206;&#178;S9A-transduced MDA-MB-231 cells <br /> were labeled fluorescent dye CM-Dil microinjected perivitelline space 48-hpf zebrafish embryos. Embryos <br /> were imaged confocal microscope. <br />  <br />  <br /> inverse correlation c-Jun COP1 abundance implicates     cells, populations transduced cells analyzed c-Jun <br /> that COP1 negatively regulates abundance c-Jun breast     abundance. Immunoblot analysis showed COP1 overexpression <br /> cancer cells.                                                       unable decrease level c-Jun (Figure 4A), suggesting that <br />                                                                     event additional high COP1 expression required rapid <br />                                                                     c-Jun protein degradation. <br /> c-Jun Protein Degradation Is Sensitive Inhibition            The activities CK2, JNK, GSK3 shown regu- <br /> GSK3 Activity                                                       late c-Jun activity stability [17,20&#226;&#8364;&#8220;22]. Especially, recognition of <br />    The inverse correlation c-Jun COP1 abundance         c-Jun E3 ligase FBW7 necessitates GSK3-mediated phosphoryla- <br /> prompted test enforcing COP1 expression invasive   tion c-Jun [22]. To test phosphorylation CK2, JNK, or <br /> breast cancer cells downregulate c-Jun. We      GSK3 required c-Jun turnover, MCF7 T47D cells were <br /> lentivirally introduced COP1 MDA-MB-231 MDA-MB-436         treated specific inhibitors JNK (SP600125), GSK3 (LiCl and <br />  Neoplasia Vol. 15, No. 9, 2013                                                 Regulation c-Jun COP1 GSK3&#206;&#178;          Shao et al.     1083 <br />  <br /> SB216763), CK2 (TBB). Immunoblot analysis showed               Simultaneously Enhancing COP1 Expression Elevating <br /> GSK inhibitors JNK CK2 inhibitor led significant         GSK3&#206;&#178; Activity Suppress Breast Tumorigenesis <br /> increase c-Jun (Figure 4B), indicating necessity       We previously showed presence c-Jun essential for <br /> GSK3 activity instability c-Jun protein. We generated       cell migration metastasis invasive breast cancer cells [8]. Others <br /> HA-tagged non&#226;&#8364;&#8220;GSK3&#206;&#178;-phosphorylable c-Jun (c-Jun&#226;&#8364;&#8220;T239A)                   reported c-Jun promotes breast cancer cell prolifera- <br /> performing Thr&#226;&#8224;&#8217;Ala mutagenesis amino acid residue 239                 tion controlling expression cyclins E2F [7]. We thus <br /> c-Jun, GSK phosphorylation site. Cycloheximide-chasing experi-          hypothesized reduced c-Jun abundance combined expres- <br /> ment showed HA-tagged c-Jun&#226;&#8364;&#8220;T239A significantly               sion COP1 constitutively active GSK3&#206;&#178; lead the <br /> stable HA-tagged c-Jun (Figure 4A), depletion COP1              suppression breast tumorigenesis. To test hypothesis, len- <br /> did affect stability HA-tagged c-Jun&#226;&#8364;&#8220;T239A (Figure W3).          tivirally introduced COP1 GSK3&#206;&#178;S9A com- <br /> Compared HA-tagged c-Jun, HA-tagged c-Jun&#226;&#8364;&#8220;T239A                      bination MDA-MB-231 MDA-MB-436 cells. MTT assay <br /> poorly poly-ubiquitinated (Figure 4D). These results suggest           showed raising COP1 level enforcing GSK3&#206;&#178;S9A expression <br /> GSK3&#206;&#178;-mediated phosphorylation prerequisite efficient poly-        moderately reduced cell proliferation, co-expression <br /> ubiquitination c-Jun subsequent degradation.                     led greater degree inhibition cell proliferation (Figure 5A). <br />    We enforced expression COP1 constitutive active           Transwell migration assay showed forced co-expression COP1 <br /> GSK3&#206;&#178; (GSK3&#206;&#178;S9A) MDA-MB-231                 GSK3&#206;&#178;S9A blocked 90% cell migration, whereas <br /> MDA-MB-436 cells. Although did reduce                  expressing inhibited 20% 40% cell migra- <br /> amount c-Jun, combination diminished 90%             tion compared control (Figures 5B W4). We further <br /> 70% c-Jun MDA-MB-231 MDA-MB-436 cells, respec-                    analyzed effect combined COP1 GSK3&#206;&#178;S9A expression <br /> tively (Figure 4E ). These results suggest COP1 GSK3&#206;&#178;              metastasis MDA-MB-231 MDA-MB-436 cells using a <br /> work facilitate effective c-Jun protein degradation breast   well-established zebrafish metastasis model [24,28,29]. Control cells <br /> cancer cells.                                                               cells co-expression COP1 GSK3&#206;&#178;S9A were <br />                                                                             microinjected perivitelline space 48-hpf zebrafish embryos, <br />                                                                             embryos imaged 2 days. Metastasis detected <br />                                                                             44 48 46 56 embryos injected control MDA-MB- <br />                                                                             231 MDA-MB-436 cells, respectively (Figures 5C W5). In <br />                                                                             contrast, metastasis seen 10 54 4 42 embryos <br />                                                                             injected MDA-MB-231 MDA-MB-436 cells co-expressing <br />                                                                             COP1 GSK3&#206;&#178;S9A, respectively, remaining embryos dis- <br />                                                                             played cells site injection (Figures 5C W5). These <br />                                                                             results demonstrate simultaneously enhancing COP1 expression <br />                                                                             elevating GSK3&#206;&#178; activity effectively suppress breast cancer <br />                                                                             cell tumorigenesis. <br />  <br />                                                                             COP1 Expression Correlates Recurrence-Free Survival <br />                                                                             Patients Breast Cancer <br />                                                                                As laboratory study recapitulate clinical breast <br />                                                                             malignancy, explored potential clinical relevance COP1 <br />                                                                             expression human breast cancer. To so, assessed the <br />                                                                             relationship COP1 mRNA expression clinical features <br />                                                                             breast tumors using microarray data 126 patients breast can- <br />                                                                             cer obtained Gene Expression Omnibus. There no <br />                                                                             significant associations COP1 expression (divided low <br />                                                                             high expression groups median) pathologic parameters, <br />                                                                             including histologic grade, tumor diameter, estrogen proges- <br />                                                                             terone receptor expression, lymph node status (Table W1). How- <br />                                                                             ever, univariate survival analysis (Kaplan-Meier method, log-rank <br />                                                                             test) showed higher COP1 expression significantly associated <br />                                                                             high recurrence-free survival patients (P = .006; Figure 6A). <br />                                                                             By multivariate analysis (proportional hazard method) encom- <br />                                                                             passing basic prognostic factors tumor diameter, histologic <br /> Figure 6. COP1 expression positive prognostic factor               grade, lymph node status addition COP1 expression (initial <br /> recurrence-free survival patients breast cancer. (A) Univariate     regression model), COP1 expression status remained indepen- <br /> survival method (Kaplan-Meier method) patients breast               dent positive prognostic factor final multivariate model (P = <br /> cancer indicates strong positive prognostic value COP1 expres-         .041, likelihood ratio test) tumor diameter lymph <br /> sion (P = .006, log-rank test). (B) Multivariate survival analysis <br /> (proportional hazard method) shows positive, independent prog- <br />                                                                             node status, hazard ratio 1.69 patients high <br /> nostic importance COP1 expression (P &lt; .05, likelihood ratio             low COP1 expression (i.e., patients low COP1 expression <br /> test), addition independent prognostic impact tumor            1.69 times higher hazard recurrence breast cancer <br /> diameter lymph node status. HR, hazard ratio.                           high COP1 expression; Figure 6B). These results <br />  1084         Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.                                                      Neoplasia Vol. 15, No. 9, 2013 <br />  <br /> also support notion COP1 negative regulator breast          COP1 promotes tumorigenesis consequence tumor for- <br /> tumor progression.                                                           mation; however, mice partial loss COP1 function tissue- <br />                                                                              specific COP1 ablation strongly suggest COP1 tumor <br />                                                                              suppressor [14,35]. Using publicly available breast tumor gene expres- <br /> Discussion                                                                   sion microarray database, reduced COP1 expression <br /> High c-Jun abundance detected invasive breast cancer cell         correlated poorer disease-free survival COP1 expression is <br /> lines specimens aggressive breast tumors [8,11]. However,             independent positive prognostic factor recurrence-free survival <br /> how c-Jun abundance regulated breast cancer cells          (Figure 6). Our study adds strong evidences COP1 suppressor <br /> understood. In study, showed c-Jun protein         breast tumor progression. <br /> stable invasive breast cancer cells invasive ones (Figure 1),      Although mechanistic study performed established <br /> indicating prolonged protein stability major cause high         breast cancer cell lines, observation high COP1 expression <br /> c-Jun abundance invasive breast cancer cells aggressive breast         correlated better disease-free survival patients breast <br /> tumor specimens [8,11]. The stability c-Jun protein breast             cancer supports results generated experimental studies. <br /> cancer cells appeared depend c-Jun efficiently             To conclude, study suggests COP1 GSK3&#206;&#178; cooperate <br /> poly-ubiquitinated nuclear proteasome activity            promote c-Jun degradation and, hence, subsequent inhibition of <br /> the ability c-Jun localize proteasomes (Figure 2). These        breast tumor progression metastasis. <br /> findings led hypothesize lack c-Jun&#226;&#8364;&#8220;targeting E3 <br /> ubiquitin ligase invasive breast cancer cells. With aid target- <br /> specific siRNAs, identified COP1 E3 ligase responsible          References <br /> c-Jun turnover breast cancer cells COP1 depletion enhanced              [1] Karin M, Liu Z, Zandi E (1997). AP-1 function regulation. Curr <br />                                                                                   Opin Cell Biol 9, 240&#226;&#8364;&#8220;246. <br /> c-Jun protein stability diminished c-Jun poly-ubiquitination in <br />                                                                               [2] Shaulian E Karin M (2002). AP-1 regulator cell life death. Nat <br /> less invasive breast cancer cells (Figure 3). In human primary T lym-             Cell Biol 4, E131&#226;&#8364;&#8220;E136. <br /> phocytes, proto-oncogene c-Abl phosphorylates c-Jun, turn            [3] Smith LM, Wise SC, Hendricks DT, Sabichi AL, Bos T, Reddy P, Brown PH, <br /> prevents c-Jun ITCH interaction stabilizes c-Jun [30].               Birrer MJ (1999). cJun overexpression MCF-7 breast cancer cells pro- <br /> Receptor activated C-kinase 1 stabilizes c-Jun releasing           duces tumorigenic, invasive hormone resistant <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span>. Oncogene 18, <br /> FBW7 complex [31]. In study, showed levels                    6063&#226;&#8364;&#8220;6070. <br />                                                                               [4] Vleugel MM, Greijer AE, Bos R, van der Wall E, van Diest PJ (2006). <br /> c-Jun COP1 inversely correlated breast cancer cells (Fig-              c-Jun activation associated proliferation angiogenesis invasive <br /> ure 3), implicating low abundance COP1 principal cause                breast cancer. Hum Pathol 37, 668&#226;&#8364;&#8220;674. <br /> of prolonged c-Jun protein stability invasive breast cancer cells.         [5] Jiao X, Katiyar S, Willmarth NE, Liu M, Ma X, Flomenberg N, Lisanti MP, <br />    Phosphorylation specific site prerequisite ubiquitina-          Pestell RG (2010). c-Jun induces mammary epithelial cellular invasion and <br /> tion E3 ubiquitin ligases. For example, Skp2 ubiquitinates               breast cancer stem cell expansion. J Biol Chem 285, 8218&#226;&#8364;&#8220;8226. <br />                                                                               [6] Zhang Y, Pu X, Shi M, Chen L, Song Y, Qian L, Yuan G, Zhang H, Yu M, <br /> and promotes degradation forkhead box protein O1 (FOXO1).                  Hu M, et al. (2007). Critical role c-Jun overexpression liver metastasis of <br /> However, Skp2-mediated event requires Akt-specific phosphoryla-              human breast cancer xenograft model. BMC Cancer 7, 145. <br /> tion FOXO1 Ser-256 [32]. c-Jun ubiquitinated FBW7              [7] Shen Q, Uray IP, Li Y, Krisko TI, Strecker TE, Kim HT, Brown PH <br /> various cell types, GSK3&#206;&#178;-mediated phosphorylation Thr-239                 (2008). The AP-1 transcription factor regulates breast cancer cell growth via <br />                                                                                   cyclins E2F factors. Oncogene 27, 366&#226;&#8364;&#8220;377. <br /> is necessary c-Jun ubiquitination place [22]. In study, <br />                                                                               [8] Chen H, Zhu G, Li Y, Padia RN, Dong Z, Pan ZK, Liu K, Huang S (2009). <br /> we showed blocking GSK3 activity increased c-Jun abundance                   Extracellular signal&#226;&#8364;&#8220;regulated kinase signaling pathway regulates breast cancer cell <br /> in invasive breast cancer cells (Figure 4). Specifically, mutating           migration maintaining slug expression. Cancer Res 69, 9228&#226;&#8364;&#8220;9235. <br /> Thr-239 Ala stabilized c-Jun protein prevented c-Jun poly-             [9] Okutomi Y, Shino Y, Komoda F, Hirano T, Ishihara T, Yamaguchi T, Saisho <br /> ubiquitination (Figure 4). Our data suggest GSK3&#206;&#178;-mediated              H, Shirasawa H (2003). Survival regulation pancreatic cancer cells by <br /> phosphorylation Thr-239 event required COP1-                c-Jun. Int J Oncol 23, 1127&#226;&#8364;&#8220;1134. <br />                                                                              [10] Neyns B, Katesuwanasing, Vermeij J, Bourgain C, Vandamme B, Amfo K, <br /> FBW7-mediated c-Jun protein degradation. This possibility              Lissens W, DeSutter P, Hooghe-Peters E, DeGr&#195;&#168;ve J (1996). Expression <br /> supported observation simultaneously overexpressing COP1              jun family genes human ovarian cancer normal ovarian surface <br /> and elevating GSK3&#206;&#178; activity effectively diminish c-Jun abundance             epithelium. Oncogene 12, 1247&#226;&#8364;&#8220;1257. <br /> in invasive breast cancer cells (Figure 4).                                  [11] Tiniakos DG, Scott LE, Corbett IP, Piggott NH, Horne CH (1994). Studies <br />                                                                                   c-jun oncogene expression human breast using new monoclonal antibody, <br />    In vitro vivo studies demonstrated convincing role <br />                                                                                   NCL-DK4. J Pathol 172, 19&#226;&#8364;&#8220;26. <br /> of c-Jun breast tumorigenesis. For example, enforced expression        [12] Jariel-Encontre I, Salvat C, Steff AM, Pariat M, Acquaviva C, Furstoss O, and <br /> c-Jun confers noninvasive hormone-dependent MCF7 cells                   Piechaczyk M (1997). Complex mechanisms c-fos c-jun degradation. <br /> an invasive hormone-resistant phenotype [3]. In MMTV-ErbB2                    Mol Biol Rep 24, 51&#226;&#8364;&#8220;56. <br /> transgenic breast cancer model, c-Jun deficiency led reduced cell         [13] Westermarck J (2010). Regulation transcription factor function targeted <br /> migration, invasion, mammosphere formation erbB2-induced                   protein degradation: overview focusing p53, c-Myc, c-Jun. Methods <br />                                                                                   Mol Biol 647, 31&#226;&#8364;&#8220;36. <br /> mammary tumors [5]. In study, showed simultaneously             [14] Migliorini D, Bogaerts S, Defever D, Vyas R, Denecker G, Radaelli E, Zwolinska <br /> raising COP1 level elevating GSK3&#206;&#178; activity suppressed vitro               A, Depaepe V, Hochepied T, Skarnes WC, et al. (2011). Cop1 constitutively <br /> cell growth/migration vivo metastasis invasive breast cancer            regulates c-Jun protein stability functions tumor suppressor mice. <br /> cells (Figure 5). Taken together, study presents strong evidence              J Clin Invest 121, 1329&#226;&#8364;&#8220;1343. <br /> that high COP1 expression GSK3&#206;&#178; activity negatively impact          [15] Wertz IE, O&#226;&#8364;&#8482;Rourke KM, Zhang Z, Dornan D, Arnott D, Deshaies RJ, and <br />                                                                                   Dixit VM (2004). Human De-etiolated-1 regulates c-Jun assembling CUL4A <br /> breast tumor progression metastasis.                                          ubiquitin ligase. Science 303, 1371&#226;&#8364;&#8220;1374. <br />    There evidences up-regulation loss COP1            [16] Nateri AS, Riera-Sans L, Da Costa C, Behrens A (2004). The ubiquitin <br /> cancers [14,33,34]. It remains determined overexpression              ligase SCFFbw7 antagonizes apoptotic JNK signaling. Science 303, 1374&#226;&#8364;&#8220;1378. <br />  Neoplasia Vol. 15, No. 9, 2013                                                          Regulation c-Jun COP1 GSK3&#206;&#178;                 Shao et al.        1085 <br />  <br /> [17] Gao M, Labuda T, Xia Y, Gallagher E, Fang D, Liu YC, Karin M (2004).       [27] Yang Y, Kitagaki J, Dai RM, Tsai YC, Lorick KL, Ludwig RL, Pierre SA, Jensen <br />      Jun turnover controlled JNK-dependent phosphorylation E3          JP, Davydov IV, Oberoi P, et al. (2007). Inhibitors ubiquitin-activating enzyme <br />      ligase Itch. Science 306, 271&#226;&#8364;&#8220;275.                                                  (E1), new class potential cancer therapeutics. Cancer Res 67, 9472&#226;&#8364;&#8220;9481. <br /> [18] Xia Y, Wang J, Xu S, Johnson GL, Hunter T, Lu Z (2007). MEKK1              [28] Zhang L, Zhou F, Drabsch Y, Gao R, Snaar-Jagalska BE, Mickanin C, Huang H, <br />      mediates ubiquitination degradation c-Jun response osmotic         Sheppard KA, Porter JA, Lu CX, et al. (2012). USP4 regulated AKT phos- <br />      stress. Mol Cell Biol 27, 510&#226;&#8364;&#8220;517.                                                  phorylation directly deubiquitylates TGF-&#206;&#178; type I receptor. Nat Cell Biol 14, <br /> [19] Gu Q, Bowden GT, Normolle D, Sun Y (2007). SAG/ROC2 E3 ligase                   717&#226;&#8364;&#8220;726. <br />      regulates skin carcinogenesis stage-dependent targeting c-Jun/AP1    [29] Thomas C, Rajapaksa G, Nikolos F, Hao R, Katchy A, McCollum CW, <br />      I&#206;&#186;B-&#206;&#177;/NF-&#206;&#186;B. J Cell Biol 178, 1009&#226;&#8364;&#8220;1023.                                            Bondesson M, Quinlan P, Thompson A, Krishnamurthy S, et al. (2012). ER&#206;&#178;1 <br /> [20] Lin A, Frost J, Deng T, Smeal T, al-Alawi N, Kikkawa U, Hunter T, Brenner           represses basal-like breast cancer epithelial mesenchymal transition by <br />      D, Karin M (1992). Casein kinase II negative regulator c-Jun DNA        destabilizing EGFR. Breast Cancer Res 14, R148. <br />      binding AP-1 activity. Cell 70, 777&#226;&#8364;&#8220;789.                                   [30] Gao B, Lee SM, Fang D (2006). The tyrosine kinase c-Abl protects <br /> [21] Derijard B, Hibi M, Wu IH, Barrett T, Su B, Deng T, Karin M, Davis RJ           c-Jun ubiquitination-mediated degradation T cells. J Biol Chem 281, <br />      (1994). JNK1: protein kinase stimulated UV light Ha-Ras binds         29711&#226;&#8364;&#8220;29718. <br />      phosphorylates c-Jun activation domain. Cell 76, 1025&#226;&#8364;&#8220;1037.            [31] Zhang J, Zhu F, Li X, Dong Z, Xu Y, Peng C, Li S, Cho YY, Yao K, Zykova <br /> [22] Wei W, Jin J, Schlisio S, Harper JW, Kaelin WG Jr (2005). The v-Jun             TA, et al. (2012). Rack1 protects N-terminal phosphorylated c-Jun from <br />      <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>point mutation</span> allows c-Jun escape GSK3-dependent recognition                Fbw7-mediated degradation. Oncogene 31, 1835&#226;&#8364;&#8220;1844. <br />      destruction Fbw7 ubiquitin ligase. Cancer Cell 8, 25&#226;&#8364;&#8220;33.                [32] Huang H, Regan KM, Wang F, Wang D, Smith DI, van Deursen JM, and <br /> [23] Hong S, Noh H, Chen H, Padia R, Pan ZK, Su SB, Jing Q, Ding HF,                 Tindall DJ (2005). Skp2 inhibits FOXO1 tumor suppression through <br />      Huang S (2013). Signaling p38 MAPK stimulates nuclear localization        ubiquitin-mediated degradation. Proc Natl Acad Sci USA 102, 1649&#226;&#8364;&#8220;1654. <br />      microprocessor component p68 processing selected primary microRNAs.     [33] Dornan D, Bheddah S, Newton K, Ince W, Frantz GD, Dowd P, Koeppen <br />      Sci Signal 6, ra16.                                                                 H, Dixit VM, French DM (2004). COP1, negative regulator of <br /> [24] Rouhi P, Jensen LD, Cao Z, Hosaka K, Lanne T, Wahlberg E, Steffensen JF,            p53, overexpressed breast ovarian adenocarcinomas. Cancer Res 64, <br />      Cao Y (2010). Hypoxia-induced metastasis model embryonic zebrafish.          7226&#226;&#8364;&#8220;7230. <br />      Nat Protoc 5, 1911&#226;&#8364;&#8220;1918.                                                       [34] Lee YH, Andersen JB, Song HT, Judge AD, Seo D, Ishikawa T, Marquardt <br /> [25] Lopez-Bergami P, Huang C, Goydos JS, Yip D, Bar-Eli M, Herlyn M, Smalley            JU, Kitade M, Durkin ME, Raggi C, et al. (2010). <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>Definition</span> ubiquitination <br />      KS, Mahale A, Eroshkin A, Aaronson S, et al. (2007). Rewired ERK-JNK sig-           modulator COP1 novel therapeutic target human hepatocellular carcinoma. <br />      naling pathways melanoma. Cancer Cell 11, 447&#226;&#8364;&#8220;460.                               Cancer Res 70, 8264&#226;&#8364;&#8220;8269. <br /> [26] He H, Gu Q, Zheng M, Normolle D, Sun Y (2008). SAG/ROC2/RBX2               [35] Vitari AC, Leong KG, Newton K, Yee C, O&#226;&#8364;&#8482;Rourke K, Liu J, Phu L, Vij R, <br />      E3 ligase promotes UVB-induced skin hyperplasia, skin tumors,            Ferrando R, Couto SS, et al. (2011). COP1 tumour suppressor causes <br />      simultaneously targeting c-Jun/AP-1 p27. Carcinogenesis 29, 858&#226;&#8364;&#8220;865.            degradation ETS transcription factors. Nature 474, 403&#226;&#8364;&#8220;406. <br />  Supplemental Information                                              FBW7 F      AATGGCCAAGGGCAACAA <br />  <br /> Antibody Information                                                  FBW7 R      CATTCCTGGAGGCCTGTAGGT <br />  <br />     c-Jun rabbit mAb: CS-9165, Cell Signaling Technology (Danvers,    ITCH F      GATGGCTCCAGATCCAAGGA <br />     MA); titer 1:1000 (immunoblot analysis) <br />                                                                       ITCH R      TGCATCTTCAGGGTCATCTGAT <br />     COP1 rabbit polyclonal antibody: A300-894A, Bethyl Laboratories <br />     (Montgomery, TX); titer 1:1000 (immunoblot analysis)              CUL4A F     GAATCAGAGTGACTCAGGCCCTATA <br />     GSK3 rabbit mAb: SC-5676, Cell Signaling Technology; <br />     titer 1:1000 (immunoblot analysis)                                CUL4A R     GGCCAGTAGCCCATTGTGA <br />     HA rabbit mAb: CS-5017, Cell Signaling Technology;                SAG F       AACAATCGCTGCCCTCTCT <br />     titer 1:1000 (immunoblot analysis) <br />     &#206;&#178;-Actin mouse mAb: SC-47778, Santa Cruz Biotechnology (Santa      SAG R       CTCATTTGCCGATTCTTTGGA <br />     Cruz, CA); titer 1:500 (immunoblot analysis) <br />                                                                       MEKK1 F     CAAACCGCCGTGTTAACAAA <br />     Flag mouse mAb: F 3165, Sigma; titer 1:1000 (immuno- <br />     blot analysis)                                                    MEKK1 R     GGCCCTATCTGCTGCAGTAAGT <br />     c-Jun mouse mAb: BD-610327, BD Biosciences (San Jose, CA); <br />     titer 1:100 (immunofluorescence staining)                         &#206;&#178;-Actin F   CCAGCTCACCATGGATGATG <br />     PSMA5 (proteasome S5A subunit) rabbit polyclonal antibody:        &#206;&#178;-Actin R   ATGCCGGAGCCGTTGTC <br />     CS-2457, Cell Signaling Technology; titer 1:50 (immuno- <br />     fluorescence staining) <br />                                                                       Sequences COP1 siRNAs <br /> Information Primers Used Quantitative Reverse <br /> Transcription&#226;&#8364;&#8220;Polymerase Chain Reaction                                   siCOP1 #1, AAAAGAAAUGACCUGCAAUUCGATTGGA- <br />                                                                           TCCAATCGAATTGCAGGTCATTTC <br /> COP1 F        CCACAGGTGGCAGATATTTCAA <br />                                                                           siCOP1 #2, AAAAAAACAGCCUUGGUAUAAUATTGGA- <br /> COP1 R        CATTGGCCAAATCAAGGTTATCTT                                    TCCAATATTATACCAAGGCTGTTT <br />  Figure W1. Colocalization c-Jun PMSA5. Immunofluorescence staining c-Jun PMSA5 MDA-MB-436 10 &#206;&#188;M MG132- <br /> treated MCF7 cells. DAPI used stain nucleus. <br />  <br />  <br />  <br />  <br />                                   Figure W2. siRNA-mediated knockdown effect c-Jun&#226;&#8364;&#8220;targeting <br />                                   E3 ligases. MCF7 T47D cells transfected 50 nM con- <br />                                   trol siRNA pool CUL4, FBW7, ITCH, MEKK1, COP1, or <br />                                   SAG 3 days. Total RNA extracted cells and <br />                                   subjected quantitative reverse transcription&#226;&#8364;&#8220;polymerase chain <br />                                   reaction respective primers. &#206;&#178;-Actin mRNA used as <br />                                   internal control standardization. <br />  <br />  <br />  <br />  <br /> Figure W3. Depletion COP1 does alter protein stability c-Jun&#226;&#8364;&#8220;T239A. T47D cells treated control COP1 siRNA for <br /> 2 days. HA-tagged c-Jun&#226;&#8364;&#8220;T239A co-transfected cells GFP 1 day followed addition 20 &#206;&#188;g/ml cyclo- <br /> heximide varying times. Cells harvested, cell lysates analyzed immunoblot analysis detect HA-tagged c-Jun <br /> and GFP. The relative HA-tagged c-Jun standardized GFP. <br />  Figure W4. Simultaneously expressing COP1 constitutively                     Figure W5. Simultaneously expressing COP1 constitutively <br /> active GSK3&#206;&#178; block cell migration invasive breast cancer cells.               active GSK3&#206;&#178; suppress breast cancer cell metastasis. MDA-MB-436 <br /> COP1 GSK3&#206;&#178;(CA) lentivirally transduced MDA-MB-231                  cells lentivirally transduced COP1 constitutively active <br /> and MDA-MB-436 cells together. Populations trans-             GSK3&#206;&#178; 3 days. Control COP1/GSK3&#206;&#178;S9A-transduced <br /> duced cells added transwells allowed migrate                MDA-MB-436 cells labeled fluorescent dye CM-Dil <br /> 4 hours. Images crystal violet&#226;&#8364;&#8220;stained migratory cells                microinjected perivitelline space 48-hpf zebrafish <br /> undersurface transwells.                                                      embryos. Embryos imaged confocal microscope. <br />  <br />  <br />  <br />                                     Table W1. Clinicopathologic Correlates COP1 Expression Human Breast Cancer. <br />  <br />  <br />                                     Clinical Features               COP1 Expression*                                P Value&#226;&#8364;&#160; <br />  <br />                                                                     Low (%)                 High (%) <br />  <br />                                     Histologic grade                                                                .28 <br />                                       1                              5                       6 <br />                                       2-3                           26                      77 <br />                                     Tumor diameter                                                                  .72 <br />                                       &lt;2 cm                          6                      22 <br />                                       &#226;&#8240;&#165;2 cm                         26                      70 <br />                                     Lymph node status                                                               .71 <br />                                       Negative                      14                      48 <br />                                       Positive                      15                      40 <br />                                     ER receptor                                                                     .46 <br />                                       Negative                      13                      44 <br />                                       Positive                      20                      46 <br />                                     PR receptor                                                                     .89 <br />                                       Negative                      15                      42 <br />                                       Positive                      12                      35 <br />  <br />                                     PR indicates progesterone receptor. <br />                                     *Low/high median (low, COP1 &lt; 0; high, COP1 &#226;&#8240;&#165; 0 expression value). <br />                                     &#226;&#8364;&#160; <br />                                   <br /> </body></html>