<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Microbiology (2012) 85(3), 513&#226;&#8364;&#8220;534 &#228;&#352;?                                                          doi:10.1111/j.1365-2958.2012.08124.x <br />                                                                                                          First published online 12 July 2012 <br />  <br />  <br />  <br /> Phosphoglucomutase absent Trypanosoma brucei and <br /> redundantly substituted phosphomannomutase and <br /> phospho-N-acetylglucosamine mutase                                                  mmi_8124 513..534 <br />  <br />  <br />  <br />  <br /> Giulia Bandini,1&#226;&#8364;&#160; Karina Mari&#195;&#177;o,1&#226;&#8364;&#161;                                      enzymes. This example organism of <br /> M. Lucia Sampaio G&#195;&#188;ther,1 Amy K. Wernimont,2                            PGM activity completely replaced way <br /> Sabine Kuettel,1 Wei Qiu,2 Shamshad Afzal,1&#194;&#167;                            explains why, uniquely, T. brucei able <br /> Anna Kelner,1 Raymond Hui2                                          lose PGM gene. The RNAi data TbPMM also <br /> Michael A. J. Ferguson1*                                                showed essential gene parasite <br /> 1 <br />  Division Biological Chemistry Drug Discovery,                   growth. <br /> College Life Sciences, University Dundee, Dundee <br /> DD1 5EH, UK. <br /> 2 <br />  Structural Genomics Consortium, University Toronto, <br />                                                                         Introduction <br /> Toronto, Ontario, Canada.                                               The kinetoplastid Trypanosoma brucei causative <br />                                                                         agent human African trypanosomiasis, known as <br />                                                                         &#226;&#8364;&#732;sleeping sickness&#226;&#8364;&#8482;, cattle disease Nagana. <br /> Summary                                                                 Human African trypanosomiasis fatal untreated <br /> The     enzymes      phosphomannomutase       (PMM),                    constitutes major public health problem sub- <br /> phospho-N-acetylglucosamine mutase (PAGM)                           Saharan Africa (Favre et al., 2008). The disease char- <br /> phosphoglucomutase (PGM) reversibly catalyse                        acterized asymptomatic period weeks or <br /> transfer phosphate C6 C1 hydro-                      months progresses early stage, character- <br /> xyl groups mannose, N-acetylglucosamine                          ized malaria-like symptoms, advanced stage, <br /> glucose respectively. Although genes candidate                    severe neurological mental disorders appear. <br /> PMM PAGM enzymes                           There clear need new therapeutics control the <br /> Trypanosoma brucei genome, is, surprisingly,                      disease. <br /> no candidate gene PGM. The TbPMM TbPAGM                            Trypanosoma brucei related trypanosomatid <br /> genes cloned expressed Escherichia coli                     parasites Trypanosoma cruzi Leishmania sp. synthe- <br /> and TbPMM enzyme crystallized struc-                    size complex cell-surface glycoconjugates, of <br /> ture solved 1.85 &#195;&#8230; resolution. Antibodies                     essential parasite <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>survival</span> and/or infectivity. <br /> recombinant proteins localized endogenous TbPMM                         Examples infective bloodstream form T. brucei <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> glycosomes bloodstream form parasite,                  include variant surface glycoprotein (VSG), trans- <br /> while TbPAGM localized cytosol gly-                     ferrin receptor (Steverding et al., 1995; Cross, 1996; <br /> cosomes. Both recombinant enzymes able                          Mehlert et al., 1998; Pays Nolan, 1998; Mehlert and <br /> interconvert glucose-phosphates, acting                   Ferguson, 2007), p67 lysosomal glycoprotein (Peck <br /> their definitive substrates. Analysis sugar                      et al., 2008) membrane-bound histidine acid phos- <br /> nucleotide levels parasites TbPMM TbPAGM                     phatase TbMBAP1 (Engstler et al., 2005). This led to <br /> knocked <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-366' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) suggests                        investigation enzymes sugar nucleotide bio- <br /> that, vivo, PGM activity catalysed                        synthesis potential therapeutic targets (Davis et al., <br />                                                                         2004; Urbaniak et al., 2006a,b; Turnock Ferguson, <br /> Accepted 2 June, 2012. *For correspondence. E-mail m.a.j.               2007; Turnock et al., 2007; Stokes et al., 2008; Denton <br /> ferguson@dundee.ac.uk; Tel. (+44) 1382 384219; Fax (+44) 1382 <br /> 386373. Present addresses: &#226;&#8364;&#160;Department Molecular Cell            et al., 2010; Lackovic et al., 2010; Mari&#195;&#177;o et al., 2010; <br /> Biology, Boston University, Henry Goldman School Dental Medi-        2011). <br /> cine, Boston, MA 02118, USA; &#226;&#8364;&#161;National Institute Bioprocessing         Sugar nucleotides activated forms sugars used <br /> Research Training, Dublin-Oxford Glycobiology Laboratory, <br /> Conway Institute, University College Dublin, Belfield, Dublin 4,        donors glycosylation reactions. They synthe- <br /> Ireland; &#194;&#167;Division Gene Regulation Expression, College Life   sized novo pathway, requiring conver- <br /> Sciences, University Dundee, Dundee DD1 5EH, UK.                     sion precursor sugar/sugar nucleotide, and/or a <br /> Re-use article permitted accordance Terms <br /> and Conditions set http://wileyonlinelibrary.com/onlineopen#     salvage pathway, sugar activated using a <br /> OnlineOpen_Terms                                                        kinase pyrophosphorylase. In eukaryotes, <br /> &#194;&#169; 2012 Blackwell Publishing Ltd <br />  514 G. Bandini et al. &#228;&#352;? <br />  <br />                                                                                            Fig. 1. Biosynthesis UDP-GlcNAc, <br />                                                                                            UDP-Glc, UDP-Galp, GDP-Man GDP-Fuc <br />                                                                                            T. brucei. Glucose taken the <br />                                                                                            environment phosphorylated Glc-6-P, <br />                                                                                            starting point novo <br />                                                                                            synthesis sugar nucleotides. Glc-6-P is <br />                                                                                            normally converted Glc-1-P action a <br />                                                                                            PGM; however, conventional PGM <br />                                                                                            homologue identified T. brucei. <br />                                                                                            Salvage pathways present the <br />                                                                                            biosynthesis GDP-Man UDP-GlcNAc <br />                                                                                            Man GlcN, UDP-Galp can <br />                                                                                            synthesized novo UDP-Glc <br />                                                                                            action UDP-galactose 4-epimerase. <br />                                                                                            The metabolites enzymes referred in <br />                                                                                            paper marked black. HK, <br />                                                                                            hexokinase; PGI, glucose-6-phosphate <br />                                                                                            isomerase; GFAT, <br />                                                                                            glucosamine-fructose-6-phosphate <br />                                                                                            aminotransferase; GalE, UDP-galactose <br />                                                                                            4-epimerase; PMI, phosphomannose <br />                                                                                            isomerase; MPGT, GDP-mannose <br />                                                                                            pyrophosphorylase. <br />  <br />  <br />  <br />  <br /> sugar nucleotides formed used cytosol                    and/or calreticulin-mediated glycoprotein quality control <br /> and/or transported lumen Golgi apparatus                  system, requires UDP-Glc donor substrate (Sousa <br /> and/or endoplasmic reticulum used                 et al., 1992; Caramelo et al., 2004). In T. brucei UGGT has <br /> glycosyltransferases (Freeze Elbein, 2008).                         shown essential parasite growth survival <br />    In years, knowledge T. brucei sugar              40&#194;&#176;C (Izquierdo et al., 2009). UDP-Glc pre- <br /> nucleotide biosynthesis (Fig. 1) widened demon-                  sumed donor synthesis <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span> J (b-D- <br /> strate steps biosynthesis GDP-fucose             glucosylhydroxymethyluracil), rare deoxynucleotide of <br /> (GDP-Fuc) (Turnock et al., 2007), UDP-galactose (UDP-                   unknown function DNA trypanosomatids <br /> Gal) (Roper et al., 2002; 2005; Urbaniak et al., 2006b),                (Borst Sabatini, 2008 references within). Finally, <br /> UDP-N-acetylglucosamine (UDP-GlcNAc) (Stokes et al.,                    action UDP-Glc 4&#226;&#8364;&#178;-epimerase, UDP-Glc the <br /> 2008; Mari&#195;&#177;o et al., 2011) GDP-mannose (GDP-Man)                    source essential sugar nucleotide UDP-Gal in <br /> (Denton et al., 2010) essential parasite growth.                T. brucei T. cruzi (Roper et al., 2002; 2005; MacRae <br /> Furthermore, these, (Mari&#195;&#177;o et al.,                   et al., 2006) hexose transporters take <br /> 2010), reports enzymes sugar                   D-Gal (Barrett et al., 1995; Tetaud et al., 1996). <br /> nucleotide biosynthesis located kinetoplastid                   Generally, UDP-Glc synthesized three-step <br /> peroxisome-like organelle, glycosome.                               process: (i) phosphorylation glucose glucose-6-P via <br />    Despite advances, importance                      hexokinase, (ii) transfer phosphate group C6 to <br /> metabolite, relatively little known synthesis           C1 produce glucose-1-phosphate (Glc-1-P) phos- <br /> UDP-glucose (UDP-Glc) T. brucei. In eukaryotic cells,                phoglucomutase (PGM, E.C. 5.4.2.2) (iii) coupling of <br /> UDP-Glc required synthesis diverse glucose-               glucose-1-P UTP UDP-Glc pyrophosphorylase <br /> containing glycoconjugates secondary metabolites                    (UGP). In T. brucei, enzymes responsible first <br /> (Flores-D&#195;&#173;az et al., 1997). It plays crucial role         step characterized (Nwagwu and <br /> &#226;&#8364;&#732;quality control&#226;&#8364;&#8482; newly synthesized glycoproteins taking             Opperdoes, 1982; Mari&#195;&#177;o et al., 2010) T. cruzi and <br /> place endoplasmic reticulum (Trombetta Parodi,               Leishmania easily identifiable PGM genes (Penha <br /> 2003; Ruddock Molinari, 2006), unfolded glycopro-                et al., 2005; 2009). The absence putative T. brucei <br /> tein glucosyltransferase (UGGT), calnexin-                  PGM gene perplexing. <br />  <br />                                                                   &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                       PMM, PAGM phosphoglucomutase activity T. brucei 515 <br />  <br />    The PGMs belong a-D-phosphohexomutase                    eukaryotic PMM family. Eukaryotic PMMs members of <br /> (PHM) superfamily enzymes eukaryotic              haloacid dehydrogenase superfamily phosphotrans- <br /> phospho-N-acetylglucosamine mutases (PAGM, E.C.                    ferases. TbPMM, like eukaryotic PMMs, belongs <br /> 5.4.2.3) bacterial phosphomannomutase/phospho-                 type II class superfamily (Kedzierski et al., <br /> glucomutases (PMM/PGM, E.C. 5.4.2.2) phospho-                  2006; Silvaggi et al., 2006). This deduced the <br /> glucosamine mutases (E.C. 5.4.2.10) (Shackelford et al.,           location cap domain (aa 85&#226;&#8364;&#8220;183), class <br /> 2004). Although eukaryotic phosphomannomutases                     motif II III (Fig. 2A). Eukaryotic <br /> (PMM, E.C. 5.4.2.8) reversibly catalyse transfer        PMMs characterized conserved DVDGT motif <br /> phosphate group mannosyl phosphate                (motif I, aa 9&#226;&#8364;&#8220;13), localized protein N-terminus. The <br /> C6 C1 positions, enzymes belong halo-             Asp motif (D9) phosphorylated the <br /> acid dehydrogenase superfamily phosphotransferases              catalytic cycle. TbPMM shows 57% sequence identity its <br /> (Collet et al., 1998).                                             S. cerevisiae homologue. <br />    The apparent lack gene encoding T. brucei PGM             A BLASTp search T. brucei predicted protein <br /> could explained TbPGM bears little resemblance         database Homo sapiens S. cerevisiae PAGM <br /> to conventional PGMs, conversion glucose-             amino acid sequences (NCBI accession nos. NP_ <br /> phosphates catalysed TbPMM and/or TbPAGM.                056414.1 NP_010856.1 respectively) revealed a <br /> The likely as, eukaryotic           single putative TbPAGM gene (GeneDB ID: Tb927.8.980) <br /> PMMs PAGMs generally highly specific             (Turnock Ferguson, 2007). Two putative PAGMs <br /> sugar phosphate substrates, ability PMMs           identified T. cruzi (GeneDB ID: TcCLB. <br /> PAGMs interconvert glucose-phosphates                  503733.70, TcCLB.508569.80). Both genes more <br /> shown (Fernandez-Sorensen Carlson, 1971; Boles                 97% identical, probably accounting hap- <br /> et al., 1994; Hofmann et al., 1994; Oesterhelt et al., 1996;       lotypes (Esmeraldo non-Esmeraldo-like) present in <br /> Pirard et al., 1997; 1999; Kato et al., 2005; Qian et al.,         CL-Brener strain sequenced (El-Sayed et al., <br /> 2007).                                                             2005a). A single PAGM gene detected in <br />    In paper, identified genes encoding functional      L. major (GeneDB ID: LmjF07.0805). For TbPAGM, the <br /> T. brucei phosphomannomutase (TbPMM) phospho-                  predicted amino acid sequence contains conserved <br /> N-acetylglucosamine mutase (TbPAGM) studied              motifs a-D-PHM family, phosphate- <br /> subcellular location metabolic activities bloodstream       binding site (RPSGTE, aa 564&#226;&#8364;&#8220;569), Mg2+ binding site <br /> form T. brucei. As interconverting mannose-            (DGDADR, aa 338&#226;&#8364;&#8220;343), invariant resi- <br /> N-acetylglucosamine-phosphates respectively,             dues R564, S566, G418, D338, D340, D342, <br /> shown capable producing Glc-1-P glucose-             R343 (marked asterisks, Fig. 2B) (Shackelford et al., <br /> 6-phosphate (Glc-6-P), albeit different efficiencies.         2004). However, TbPAGM, sugar-binding site (aa <br /> TbPMM crystallized structure solved            435&#226;&#8364;&#8220;442) contains unusual variation, nor- <br /> high resolution. The dual specificities enzymes           mally highly conserved A437 residue replaced a <br /> suggested explanation absence TbPGM                serine. As observed PAGMs, phosphoserine <br /> gene parasite knockdown TbPMM                   loop (TASHNP, aa 80&#226;&#8364;&#8220;85, black box, Fig. 2B), essential <br /> TbPAGM RNA interference (RNAi) showed                 catalytic activity, translocated earlier position <br /> enzymes mutually redundant PGM activity.             compared a-PHM family members, order to <br />                                                                    accommodate bulky N-acetyl group present the <br />                                                                    phosphorylated GlcNAc substrate (Shackelford et al., <br /> Results                                                            2004). TbPAGM shares 28% 35% identity its <br />                                                                    human yeast orthologues respectively. <br /> Identification T. brucei PMM PAGM <br />                                                                       According recent quantitative proteomic analysis, <br /> encoding genes <br />                                                                    TbPAGM TbPMM present protein level <br /> Saccharomyces cerevisiae PMM (NCBI accession no.:                  life cycle stages relative concentrations of <br /> NP116609) previously used template                  proteins (procyclic levels divided bloodstream form <br /> BLASTp search T. brucei predicted proteins                  levels) 1.06 &#226;&#171;&#190; 0.30 1.17 &#226;&#171;&#190; 0.30 respectively <br /> Tb10.70.0370 identified putative TbPMM. Puta-             (Urbaniak et al., 2012). <br /> tive PMM genes Leishmania <br /> major (GeneDB ID: LmjF36.1960) T. cruzi (GeneDB <br />                                                                    The activities recombinant TbPMM TbPAGM on <br /> ID: TcCLB.510187.480) genomes (Turnock Fergu- <br />                                                                    mannose- N-acetylglucosamine-phosphates <br /> son, 2007) Leishmania mexicana PMM been <br /> structurally characterized (LmxM.36.1960; Kedzierski               The TbPMM TbPAGM open reading frames (ORFs) <br /> et al., 2006). As shown Fig. 2A, TbPMM belongs           amplified PCR genomic DNA obtained <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  516 G. Bandini et al. &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 2. TbPMM TbPAGM homology eukaryotic orthologues. Amino acids conserved proteins highlighted in <br /> dark grey. Sequences protozoa (T. brucei, T. cruzi L. major), yeasts (S. cerevisiae Candida albicans) mammals (Homo <br /> sapiens PMM1) used alignments. <br /> A. Sequences eukaryotic PMM family. The conserved motifs indicated black boxes, arrow points Asp <br /> residue phosphorylated reaction. <br /> B. PAGM sequence alignment. Conserved motifs a-D-PHM family indicated black boxes, asterisks indicate invariant <br /> residues arrow points P-Ser residue. <br />  <br />  <br />                                                                 &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                      PMM, PAGM phosphoglucomutase activity T. brucei 517 <br />  <br />                                                                                      Fig. 3. TbPMM converts Man-1-P to <br />                                                                                      Man-6-P. Recombinant TbPMM was <br />                                                                                      incubated Man-1-P the <br />                                                                                      Glc-1,6-biphosphate cofactor products <br />                                                                                      reaction analysed by <br />                                                                                      HPAEC-PAD. A peak corresponding to <br />                                                                                      Man-6-P observed presence the <br />                                                                                      recombinant enzyme (B), its <br />                                                                                      absence (A). C. Kinetic analysis TbPMM <br />                                                                                      apparent Km Man-1-P of <br />                                                                                      327 &#226;&#171;&#190; 66 mM maximum velocity of <br />                                                                                      2.6 &#226;&#171;&#190; 0.4 nmol min-1 milligram protein <br />                                                                                      (Table 1). <br />  <br />  <br />  <br />  <br /> T. brucei strain 427, sequenced (EMBL nucleotide                   (Fig. 3B), Man-6-P detected when <br /> sequence database accession number FR851306                    TbPMM absent (Fig. 3A). We used system <br /> FR851307) cloned expression vector contain-            TbPMM convert GlcNAc-1-P to <br /> ing N-terminal His6-tags expressed E. coli. After           GlcNAc-6-P, case (Fig. S3A). The <br /> Nickel column purification, TbPMM protein              procedure used determine kinetic <br /> purified size exclusion chromatography (Fig. S1)            properties enzyme Man-1-P (Fig. 3C). Different <br /> the His-tag removed incubation TEV protease            concentrations Glc-1,6-biP tested selecting <br /> resulting band 27.9 kDa SDS-PAGE (Fig. S1).             optimal concentration 1 mM. The apparent Km, Vmax <br /> For TbPAGM, His-tag cleaved PreScission               catalytic efficiency recombinant TbPMM using <br /> protease&#226;&#8222;&#162; untagged protein purified            Man-1-P substrate shown Table 1. It not <br /> by size exclusion chromatography (Fig. S2). The molecu-            possible saturate reaction, TbPMM activity <br /> lar masses identities tagged untagged               showed inhibition highest Man-1-P concentrations <br /> recombinant proteins verified MALDI-TOF             (Fig. 3C). Consequently, equation high-substrate <br /> LC-MS/MS peptide mass fingerprinting respectively (data            inhibition used calculate kinetic parameters of <br /> not shown). Although recombinant enzymes                 reaction. The pH dependence TbPMM PMM activity <br /> active His-tag, untagged recom-           analysed pH range 4.0 10.0. The enzyme <br /> binant enzymes used activity assays and,           showed activity pH 5.0 8.0 optimum <br /> case TbPMM, crystallization trials.                         pH 6.5 (data shown). <br />    A high-pH anion exchange chromatography pulsed                The HPAEC-PAD used the <br /> amperometric detection (HPAEC-PAD) used                 activity recombinant TbPAGM GlcNAc-1-P and <br /> to assay activity recombinant TbPMM,                 GlcNAc-6-P, Glc-1,6-biP cofactor (Fig. 4B). <br /> described Experimental procedures. It clear            TbPAGM able convert Man-1-P Man-6-P in <br /> TbPMM convert Man-1-P Man-6-P presence               vitro (Fig. S3B); however, discussed later, activity <br /> of glucose-1,6-biphosphate (Glc-1,6-biP) cofactor             able compensate knockdown TbPMM <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  518 G. Bandini et al. &#228;&#352;? <br />  <br /> Table 1. Comparison kinetic properties recombinant TbPMM TbPAGM different sugar phosphates substrates. <br />  <br />                                    TbPMM                                                             TbPAGM <br />  <br />                                                                                                                                 IC50 (mM)a <br />                  Km             vmax                Kcat/Km        Km             vmax                Kcat/Km <br /> Substrate        (mM)           (nmol min-1)        (M-1 s-1)      (mM)           (nmol min-1)        (M-1 s-1)     GlcNAc-6-P         GlcNAc-1-P <br />  <br /> Man-1-P          327 &#226;&#171;&#190; 66b        2.6 &#226;&#171;&#190; 0.4         3.7 &#194;&#165; 105 <br /> Glc-6-P           96 &#226;&#171;&#190; 8c       0.331 &#226;&#171;&#190; 0.008       5.7 &#194;&#165; 104      107 &#226;&#171;&#190; 17b       0.23 &#226;&#171;&#190; 0.03        9.4 &#194;&#165; 104     8.3 &#226;&#171;&#190; 0.4          6.3 &#226;&#171;&#190; 2.6 <br /> GlcNAc-6-P                                                          14 &#226;&#171;&#190; 6c       0.051 &#226;&#171;&#190; 0.006       1.6 &#194;&#165; 105 <br />  <br /> a. IC50 GlcNAc phosphates competitive inhibitors formation Glc-1-P. <br /> b. The kinetic constants calculated using equation high-substrate inhibition based best-fit non-linear lines. <br /> c. The kinetic constants calculated Michaelis&#226;&#8364;&#8220;Menten equation. <br />  <br />  <br />  <br /> in vivo. The kinetic parameters conversion                       10.0 pH range. The enzyme showed broad optimum <br /> GlcNAc-6-P GlcNAc-1-P TbPAGM determined                          pH 7.2 9.1 (data shown). <br /> using TbUAP pyrophosphorylase coupled assay <br /> with colorimetric detection (Stokes et al., 2008) a <br />                                                                             Both TbPMM TbPAGM convert Glc-6-P <br /> 96-well plate format, described Experimental <br />                                                                             Glc-1-P <br /> procedures. The TbPAGM enzyme followed classic <br /> Michaelis&#226;&#8364;&#8220;Menten kinetics (Fig. 4C) apparent Km,                    The ability TbPMM TbPAGM catalyse reac- <br /> Vmax catalytic efficiency recombinant TbPMM                     tion assessed incubating recombinant <br /> using GlcNAc-6-P substrate shown Table 1.                     enzyme presence Glc-1-P. In cases the <br /> The pH dependence TbPAGM studied 4.0                       formation Glc-6-P detected HPAEC-PAD <br />  <br />  <br />                                                                                                   Fig. 4. TbPAGM interconverts GlcNAc <br />                                                                                                   phosphates. Recombinant TbPAGM was <br />                                                                                                   incubated GlcNAc-1-P without <br />                                                                                                   Glc-1,6-biphosphate cofactor, as <br />                                                                                                   indicated, products reaction <br />                                                                                                   analysed HPAEC-PAD. A peak <br />                                                                                                   corresponding GlcNAc-6-P observed in <br />                                                                                                   presence recombinant enzyme and <br />                                                                                                   cofactor (B), cofactor was <br />                                                                                                   removed reaction (A). C. Kinetic <br />                                                                                                   analysis TbPMM apparent Km <br />                                                                                                   GlcNAc-6-P 14 &#226;&#171;&#190; 6 mM maximum <br />                                                                                                   velocity 0.051 &#226;&#171;&#190; 0.006 nmol min-1 per <br />                                                                                                   milligram protein (Table 1). <br />  <br />  <br />  <br />  <br />                                                                      &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                       PMM, PAGM phosphoglucomutase activity T. brucei 519 <br />  <br /> (Fig. S4). As Glc-1-P substrate UGP,             crystallization LmPMM (Kedzierski et al., 2006). In both <br /> were able set assay conversion Glc-6-P         cases, TbPMM behaved monomer oligomers <br /> to Glc-1-P coupling TbPMM TbPAGM TbUGP                 observed (Fig. S5). It noted when <br /> to produce UDP-Glc presence UTP (Mari&#195;&#177;o et al.,          TbPMM analysed size exclusion chromatography <br /> 2010). Thus, single peak co-eluted UDP-Glc             high salt purification process, retention <br /> could detected HPAEC-UV A260,               time consistent apparent molecular weight <br /> either TbPMM TbPAGM coupled recombinant                (MW) homodimer (data shown). Because the <br /> TbUGP (Fig. 5B D). Removal PHM              dimeric state TbPMM observed high <br /> reaction resulted loss UDP-Glc peak (Fig. 5A         concentrations salt, physiological condi- <br /> and C).                                                            tions, concluded biological assembly TbPMM to <br />    The kinetic parameters conversion Glc-6-P         monomer. <br /> Glc-1-P TbPMM TbPAGM determined                      Comparison TbPMM crystal structure the <br /> discontinuous coupled assay TbUGP colorimet-             monomers L. mexicana human enzymes <br /> ric detection. This assay measures pyrophosphate               showed overall structure conserved <br /> (PPi) product Glc-1-P + UTP &#226;&#8224;&#8217; UDP-Glc + PPi                 (Fig. 7A), residues forming active site <br /> coupling reaction conversion PPi inorganic           (Fig. 7B). As previously described, residues involved in <br /> phosphate detection molybdate reagent (see              catalysis located core domain (aa 5&#226;&#8364;&#8220;82 and <br /> Experimental procedures). The kinetic values TbPMM              187&#226;&#8364;&#8220;246): D9 (Motif I) corresponds Asp is <br /> and TbPAGM listed Table 1. Interestingly, TbPAGM            phosphorylated reaction, S45 (motif II) K187 <br /> activity inhibited highest concentrations Glc-       (Motif III) involved binding phosphate, and <br /> 6-P saturation observed kinetic         D9, D11 (Motif I), D206 D214 (Motif IV) co-ordinate the <br /> analysis (Fig. 5F). No inhibition detected        Mg2+ ion (Fig. 7B C) (Allen Dunaway-Mariano, <br /> the case TbPMM (Fig. 5E).                                       2004; Quental et al., 2010). The cap domain contributes <br />    As affinity TbPAGM GlcNAc-6-P              residues involved substrate recognition (E119, R121, <br /> 10-fold higher Glc-6-P, investigated           M124, N126, R132, R139, S177), active site is <br /> GlcNAc-6-P act competitive inhibitor             located groove interface core cap <br /> formation Glc-1-P. This confirmed running            domains (Fig. 7A) (Kedzierski et al., 2006; Silvaggi et al., <br /> assay 100 mM Glc-6-P (i.e. apparent Km                2006). The electron density map solved structure <br /> TbPAGM Glc-6-P) presence increasing con-             revealed sulphate ion monomer&#226;&#8364;&#8482;s active site, in <br /> centrations GlcNAc-6-P, IC50                    approximately position 1-phosphate <br /> 8.3 &#226;&#171;&#190; 0.4 mM (Fig. 6A). GlcNAc-1-P inhibited              moiety Glc-1,6-biP cofactor (Kedzier- <br /> conversion Glc-6-P Glc-1-P comparable IC50         ski et al., 2006). This ion co-ordinates directly R139 <br /> 6.3 &#226;&#171;&#190; 2.6 mM (Fig. 6B).                                            R132, linking waters S177 R121 <br />                                                                    (Fig. 7C). Overlaying structure 2i55, LmPMM <br />                                                                    structure Glc-1,6-biP bound, phospho-sugar fits <br /> Determination analysis TbPMM structure <br />                                                                    quite closed pocket TbPMM. In Fig. 7B, <br /> The crystal structure full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> TbPMM solved          strict conservation pocket. In the <br /> (PDB accession number 3F9R) molecular replacement               LmPMM structure, D10, D207 D215 engaged in <br /> using modified form L. mexicana PMM (LmPMM;               co-ordinating Mg2+ ion D10 supposed form the <br /> PDB accession number 2i54). The crystallographic               phosphoprotein intermediate TbPMM structure, <br /> model refinement data TbPMM structure                  Asp residues conserved (D9, D11, D206 and <br /> reported Table 2. Two polypeptide units seen           D214), Mg2+ ion electron <br /> in crystals asymmetric <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, chains A B. When              density. No additional magnesium added protein <br /> symmetry mates extended, monomer formed                prior crystallization, density magnesium <br /> dimer symmetry mates, i.e. A : A B : B.           ion distal enzyme another <br /> These dimers mimicked dimerization mode seen                magnesium-binding site PMMs structures <br /> both human LmPMM structures (Fig. S6).                         (2i55, 2fue). <br />    The oligomeric state TbPMM studied analytical            As quaternary structure TbPMM main dif- <br /> ultracentrifugation (AUC) size exclusion chromatogra-          ference enzyme described <br /> phy. Recombinant TbPMM analysed AUC                  eukaryotic PMMs, homodimers (Kepes and <br /> different buffers: containing sodium chloride physi-        Schekman, 1988; Kedzierski et al., 2006; Silvaggi et al., <br /> ological concentrations (10 mM HEPES pH 7.5, 150 mM                2006), analysed residues hypothetical <br /> NaCl) salt (10 mM Tris-HCl pH 7.5,             dimer interface. Analysis using PISA-EBI identified 12 <br /> 1 mM DTT). This buffer used                residues potentially involved H-bonding salt bridge <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  520 G. Bandini et al. &#228;&#352;? <br />  <br />                                                                Fig. 5. TbPMM TbPAGM <br />                                                                convert Glc-6-P Glc-1-P. <br />                                                                TbPMM TbPAGM used <br />                                                                coupled assay TbUGP <br />                                                                presence Glc-6-P and <br />                                                                UTP, indicated. The products <br />                                                                reactions analysed by <br />                                                                HPAEC UV detection. <br />                                                                Formation UDP-Glc (indicative <br />                                                                conversion Glc-6-P to <br />                                                                Glc-1-P) observed both <br />                                                                TbPMM (B) TbPAGM (D). No <br />                                                                sugar nucleotide peak be <br />                                                                detected absence either <br />                                                                phospho-sugar mutase (A and <br />                                                                C). The kinetic analyses of <br />                                                                recombinant TbPMM (E) and <br />                                                                TbPAGM (F) performed <br />                                                                using TbUGP-coupled assay <br />                                                                colorimetric detection. The <br />                                                                results Table 1. <br />  <br />  <br />  <br />  <br />                           &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                            PMM, PAGM phosphoglucomutase activity T. brucei 521 <br />  <br />                                                                        consistent interface residues having role in <br />                                                                        multimer formation. <br />  <br />  <br />                                                                        Subcellular localization TbPMM TbPAGM in <br />                                                                        bloodstream form T. brucei <br />                                                                        Subcellular localization enzymes studied <br />                                                                        immunofluorescence microscopy. Paraformaldehyde- <br />                                                                        fixed wild type bloodstream form T. brucei cells incu- <br />                                                                        bated polyclonal mouse anti-TbPMM (Fig. 8A) anti- <br />                                                                        TbPAGM (Fig. 8B) combined rabbit polyclonal <br />                                                                        anti-GAPDH, glycosomal marker (images d) with <br />                                                                        rabbit anti-enolase, cytosolic marker (images e h). <br />                                                                        Cells washed incubated mix Alexafluor <br />                                                                        488 anti-mouse IgG (green) Alexafluor 594 anti-rabbit <br />                                                                        IgG (red). Anti-TbPMM showed punctate staining <span id='am-398' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />                                                                        substantial colocalization anti-GAPDH (Fig. 8A, <br />                                                                        images d), absence colocalization with <br />                                                                        anti-enolase (Fig. 8A, images e h), indicating that <br />                                                                        TbPMM glycosomal enzyme. On hand, <br />                                                                        TbPAGM showed punctate diffuse staining, in <br />                                                                        agreement partial colocalization GAPDH <br />                                                                        (Fig. 8B, images d) enolase (Fig. 8B, images <br />                                                                        e h), respectively, indicating TbPAGM located in <br />                                                                        glycosomes cytosol. <br />                                                                           The subcellular localization TbPMM TbPAGM <br />                                                                        studied digitonin latency, described previ- <br />                                                                        ously (Galland et al., 2007), using Western blotting the <br />                                                                        proteins released escalating digitonin concentrations <br />                                                                        (Fig. 8C). TbPMM poorly released lower <br />                                                                        concentrations digitonin increased gradually with <br />                                                                        increasing digitonin concentration, reaching maximum <br />  <br />  <br /> Fig. 6. GlcNAc-6-P GlcNAc-1-P inhibit formation         Table 2. Data collection crystallographic refinement. <br /> UDP-Glc vitro. When TbPAGM used source PGM <br /> activity step UDP-Glc formation, biosynthesis   Data collection <br /> this sugar nucleotide inhibited presence       Space group                                                P21212 <br /> GlcNAc-6-P (A) GlcNAc-1-P (B). Thus, sugar phosphates         Cell dimensions <br /> compete Glc-6-P binding TbPAGM IC50 values           (&#195;&#8230;)                                                    105.93 <br /> 8.3 &#226;&#171;&#190; 0.4 mM 6.3 &#226;&#171;&#190; 2.6 mM respectively (Table 1).                    b (&#195;&#8230;)                                                    46.99 <br />                                                                          c (&#195;&#8230;)                                                    94.49 <br />                                                                        Wavelength                                                 1.5418 <br />                                                                        Resolution                                                 50&#226;&#8364;&#8220;1.8 (1.86) <br /> interactions (N91, K99, R103, A106, D107, D109, P111,                  Unique reflections                                         44 877 <br /> R114, T116, V118, Y120 N133) 14 residues                       Rmerge                                                     0.122 (0.862) <br /> involved hydrophobic interactions                   Mean I/s(I)                                                17.83 (1.40) <br />                                                                        Completeness (%)                                           99.9 (99.3) <br /> TbPMM structure. These residues conserved                  Redundancy                                                 6.2 (4.4) <br /> the LmPMM interface, exception amino                 PDB <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span>                                                   3F9R <br /> acids. Two (I94 Y120) partially involved             Refinement <br />                                                                        Resolution                                                 50&#226;&#8364;&#8220;1.85 <br /> in hydrophobic interactions correspond                      Number reflections                                      41 076 <br /> LmPMM residues V95 F121. The substitution                    Test set                                                   2074 <br /> occurs R103, LmPMM H104 (Fig. 7D). Inter-               Rwork/Rfree                                                0.202/0.266 <br />                                                                        RMS deviations <br /> estingly, PISA-EBI macromolecular analysis tool finds                Bond lengths (&#195;&#8230;)                                         0.016 <br /> a low complexation significance score (CSS) TbPMM                    Bond angles (&#194;&#176;)                                          1.471 <br /> (0.017) suggesting monomeric quaternary                      Average B value (&#195;&#8230;2)                                       31.63 <br />  <br /> structure. As comparison, CSS LmPMM 0.249, <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  522 G. Bandini et al. &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 7. High-resolution crystal structure TbPMM. <br /> A. Comparison TbPMM structure (green) monomers LmPMM (yellow) H. sapiens PMM1 (magenta). <br /> B. Active sites TbPMM (blue) Glc-1,6-biP cofactor modelled compared LmPMM (grey). All Asp residues be <br /> involved phosphoprotein intermediate (D9) Mg2+ co-ordination (D11, D206 D214) conserved, S45 R187, <br /> which involved phosphate moiety recognition. S45 shown possible conformations. <br /> C. A sulphate ion position phosphate group substrate expected. The ion co-ordinated R139 <br /> and R132 directly R121 S177 water molecules. TbPMM structure shown blue LmPMM grey. <br /> D. Comparison surface interfaces TbPMM (green) LmPMM (beige). The mutated residues labelled. <br />  <br />  <br />  <br /> at 0.1 mg digitonin/mg protein. This pattern similar             cipitated single band correct apparent MW from <br /> to aldolase (aldo), lumenal glycosome resident                2 &#194;&#165; 108 cell equivalents trypanosome detergent lysate <br /> enzyme, indicating TbPMM glycosomal                       (Fig. 8D, lane 2). <br /> enzyme. Lower MW bands observed higher con-                        Digitonin latency used study cellular <br /> centrations digitonin 0.1% Triton X-100 (Fig. 8C,              location TbPAGM. Unlike TbPMM, TbPAGM sub- <br /> lanes 7 8 respectively) probably proteolytic              stantially released low concentrations digitonin, a <br /> degradation TbPMM, non-specificity                  pattern similar enolase (eno), cytosolic protein. <br /> the affinity-purified antibody specifically immunopre-              However, release TbPAGM increased gradually <br />  <br />                                                                   &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                       PMM, PAGM phosphoglucomutase activity T. brucei 523 <br />  <br />                                                                                       Fig. 8. Subcellular localization TbPMM <br />                                                                                       TbPAGM. Immunofluorescence <br />                                                                                       microscopy performed on <br />                                                                                       paraformaldehyde-fixed bloodstream form <br />                                                                                       T. brucei cells stained mouse <br />                                                                                       anti-TbPMM (A) mouse anti-TbPAGM (B) <br />                                                                                       sera combined glycosomal marker rabbit <br />                                                                                       anti-GAPDH (images d) cytosolic <br />                                                                                       marker rabbit anti-enolase (images e h). <br />                                                                                       Merged images shown images c g <br />                                                                                       corresponding phase contrast images are <br />                                                                                       shown images d h. A scale bar (10 mm) <br />                                                                                       shown A, image e. The subcellular <br />                                                                                       localization TbPMM TbPAGM also <br />                                                                                       studied digitonin latency, i.e. release of <br />                                                                                       specific proteins, judged Western <br />                                                                                       blotting antibodies TbPMM and <br />                                                                                       TbPAGM, glycosomal marker aldolase <br />                                                                                       (aldo) cytosolic marker enolase (eno), <br />                                                                                       increasing concentrations digitonin <br />                                                                                       (C). In TbPMM blot, arrow <br />                                                                                       indicates intact TbPMM bottom <br />                                                                                       arrow indicates TbPMM proteolytic products. <br />                                                                                       The specificity TbPMM antibody (D) was <br />                                                                                       tested Western blotting an <br />                                                                                       immunoprecipitate using antibody <br />                                                                                       2 &#194;&#165; 108 bloodstream form cells (lane 2) <br />                                                                                       25 ng recombinant <br />                                                                                       TbPMM (lane 1). The MW markers shown <br />                                                                                       left. <br />  <br />  <br />  <br />  <br /> with increase digitonin concentration (Fig. 8A,             expressing tetracycline inducible double-stranded RNA <br /> compare lanes 2 7) indicating PAGM dual             (dsRNA) targeting TbPMM TbPAGM respectively. <br /> localization cell, glycosomal cytosolic.                  Induction dsRNA targeting TbPMM resulted a <br />                                                                    68 &#226;&#171;&#190; 18% knockdown TbPMM mRNA 48 h (as <br />                                                                    judged qRT-PCR replicates) reduction <br /> RNAi knockdown TbPMM TbPAGM the <br />                                                                    growth rate led cell death 72 h, demonstrat- <br /> effects sugar nucleotide metabolism bloodstream <br />                                                                    ing essentiality TbPMM (Fig. 9A). Phenotypic <br /> form T. brucei <br />                                                                    analyses knockdown cell line carried at <br /> The contributions TbPMM TbPAGM biosyn-               24 h 48 h post dsRNA induction. The expression of <br /> thesis UDP-Glc UDP-Galp studied RNAi.               TbPAGM protein evaluated Western blot case <br /> Fragments 464 497 bp TbPMM TbPAGM                    enzyme, interconvert Man-1-P Man- <br /> ORFs, respectively, cloned p2T7TABlue            6-P, upregulated induced TbPMM RNAi <br /> resulting constructs used generate cell lines          cells compensate loss TbPMM, was <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  524 G. Bandini et al. &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 9. Characterization TbPMM RNAi effects. <br /> A. Growth curves TbPMM RNAi cell line induced tetracycline (solid line) un-induced (dashed line). <br /> B. Western blot total cell lysates un-induced (lanes 1 3) induced (lanes 2 4) TbPMM RNAi cells stained Ponceau <br /> (lanes 1 2) antibodies TbPAGM (lanes 3 4). <br /> C. The levels sugar nucleotide determined LC-MS/MS wild type cells (white), un-induced TbPMM RNAi cell line (striped) or <br /> the TbPMM RNAi cell line induced 24 h (grey) 48 h (black). The P-values (P) determined t-test; double asterisk indicates a <br /> P &lt; 0.005. <br /> D. The extent N-glycosylation sVSG221 assessed anti-sVSG221 Western blot cell lysates wild type (WT) cells <br /> (lane 1) TbPMM RNAi cells induced (+) un-induced (-) tetracycline 1 day 2 days, indicated. The lower band in <br /> lanes 2 4, marked spot, consistent sVSG221 lacking N-glycans. <br />  <br />  <br />  <br />  <br />                                                                    &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                        PMM, PAGM phosphoglucomutase activity T. brucei 525 <br />  <br /> not case (Fig. 9B). Sugar nucleotides extracted           enzymes mutually redundant the <br /> from induced un-induced TbPAGM TbPMM RNAi                  conversion Glc-6-P Glc-1-P. Thus, reduc- <br /> cell lines wild type cells, separated HPLC         tion GDP-sugars levels observed knock- <br /> quantified electrospray ionization tandem mass spec-            TbPMM expression, likewise reduction in <br /> trometry, previously described (Turnock Ferguson,           UDP-GlcNAc levels observed case TbPAGM <br /> 2007). In TbPMM RNAi cell line GDP-Man levels              knockdown, significant reduction UDP-Glc UDP- <br /> were reduced induction, expected. Thus, GDP-               Gal observed cell line. <br /> Man reduced 55% 50% (P &lt; 0.005) wild <br /> type levels 24 48 h dsRNA induction respec- <br /> tively, downstream GDP-Man metabolite, GDP- <br />                                                                    Discussion <br /> Fuc, reduced 40% (P = 0.001) wild type levels            Although genes encoding putative PMM PAGM <br /> after 48 h (Fig. 9C). However, condi-           enzymes T. brucei genome, bio- <br /> tions UDP-sugar levels similar         chemically confirmed here, gene encoding putative <br /> wild type un-induced cells (Fig. 9C). These data           PGM, enzyme responsible synthesis Glc-1-P <br /> that, unlike GDP-Man GDP-Fuc, synthesis                 Glc-6-P, (Berriman et al., 2005; <br /> UDP-Glc UDP-Gal critically dependent             Turnock Ferguson, 2007). The absence T. brucei <br /> expression TbPMM. Further evidence mannose star-             PGM gene particularly perplexing. First, because <br /> vation induced TbPMM RNAi cell line obtained            T. cruzi Leishmania easily identifiable PGM <br /> by Western blotting anti-VSG antibodies (Fig. 9D).            genes T. cruzi PGM enzyme biochemi- <br /> Thus, evident 24 h clear           cally characterized (Penha et al., 2005; 2009) and, <br /> 48 h, TbPMM knockdown leads N-glycosylation               second, known route identified <br /> of sVSG221, previously described conditional            T. brucei metabolite UDP-Glc (Turnock Ferguson, <br /> null mutant T. brucei GDP-Man pyrophosphorylase                 2007) UGP (Mari&#195;&#177;o et al., 2010). As UGP uses <br /> (Denton et al., 2010).                                             Glc-1-P UTP substrates, known <br />    The induction dsRNA targeting TbPAGM caused                  route Glc-1-P Glc-6-P PGM activity, follows <br /> only slight reduction cell growth (Fig. 10A),            T. brucei possess enzyme PGM activity. <br /> though level TbPAGM protein 2 3 days              Furthermore, flux UDP-Glc, also <br /> of induction reduced 15% wild type levels             Glc-1-P, known considerable blood- <br /> 30% un-induced levels respectively (Fig. 10B). No               stream form procyclic form T. brucei (Turnock and <br /> compensatory difference RNA levels TbPMM                Ferguson, 2007). This UDP-Glc obligate <br /> was observed TbPAGM RNAi cell line,                  precursor UDP-Gal (Roper et al., 2002; 2005; Shaw <br /> un-induced induced 2 3 days (data shown).            et al., 2003; Urbaniak et al., 2006b) that, turn, used to <br /> After 48 h induction, sugar nucleotide levels        synthesize galactose-containing glycoproteins of <br /> TbPAGM RNAi cell line agreed reasonably                  parasite, including highly abundant VSG coat gly- <br /> those un-induced wild type cells (Fig. 10C).                coprotein procyclins bloodstream pro- <br /> However, 72 h induction, UDP-GlcNAc levels            cyclic forms parasite respectively. The biochemical <br /> were 25% 30% (P &lt; 0.05) un-induced wild                 characterization TbPMM TbPAGM described in <br /> type levels respectively, sugar nucleotide         paper resolves metabolic paradox, both <br /> levels remained comparable un-induced                  enzymes capable converting Glc-6-P Glc-1-P, as <br /> cells (Fig. 10C). Despite significant knockdown             interconverting definitive mannose- and <br /> TbPAGM protein levels, evidence GlcNAc starvation,             N-acetylglucosamine-phosphate substrates. Certain other <br /> i.e. N-glycosylation sVSG221 previously                PMMs PAGMs shown similarly pro- <br /> described conditional <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>null mutants</span> T. brucei            miscuous organisms (Fernandez-Sorensen and <br /> UDP-GlcNAc pyrophosphorylase glucosamine-6-P                   Carlson, 1971; Boles et al., 1994; Hofmann et al., 1994; <br /> N-acetyltransferase (Stokes et al., 2008; Mari&#195;&#177;o et al.,           Oesterhelt et al., 1996; Pirard et al., 1999; Kato et al., <br /> 2011), barely detectable (Fig. 10D). It noted              2005; Qian et al., 2007), known example <br /> previously growth defects occur blood-              PGM activity completely replaced PMM and/or <br /> stream form T. brucei UDP-GlcNAc levels fall           PAGM. Analysis synteny trypanosomatid <br /> 20% wild type levels (Stokes et al., 2008)         genomes (Fig. S7; El-Sayed et al., 2005b) suggests that <br /> appear RNAi knockdown TbPAGM achieved                  T. brucei deleted PGM gene point in <br /> here deep significantly affect protein            evolution, presumably PMM PAGM <br /> glycosylation cell growth.                                     attained sufficient catalytic flexibility support Glc-6-P <br />    In conclusion, analysis sugar nucleotide             Glc-1-P conversion well. The opposite situation was <br /> levels TbPMM TbPAGM RNAi cell lines suggests                recently reported protozoan parasite, Giardia <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  526 G. Bandini et al. &#228;&#352;? <br />  <br />  <br />  <br />  <br /> Fig. 10. Characterization TbPAGM RNAi effects. <br /> A. Growth curves TbPAGM RNAi cell line induced tetracycline (solid line) un-induced (dashed line). <br /> B. Western blot TbPAGM protein total lysates wild type cells (lane 5) TbPAGM RNAi cells un-induced (lanes 2 4) induced <br /> (lanes 1 3) tetracycline day 2 (lanes 1 2) 3 (lanes 3 4). The Licor quantitative data, normalized aldolase (aldo) <br /> signal, plotted blot. <br /> C. The levels sugar nucleotide determined LC-MS/MS wild type cells (white), un-induced TbPAGM RNAi cell line (stippled) <br /> or TbPMM RNAi cell line induced 48 h (squared) 72 h (bold horizontal stripes). The P-values (P) determined t-test; the <br /> asterisk indicates P &lt; 0.05. <br /> D. The extent N-glycosylation sVSG221 assessed anti-sVSG221 Western blot cell lysates wild type (WT) cells <br /> (lane 5) TbPAGM RNAi cells induced (+) un-induced (-) tetracycline 2 days 3 days, indicated. The lower band <br /> in lanes 1 3, marked spot, consistent sVSG221 lacking N-glycans. <br />  <br />  <br />  <br />  <br />                                                                      &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                       PMM, PAGM phosphoglucomutase activity T. brucei 527 <br />  <br /> lamblia. This organism obvious PMM gene                 Man-6-P, which, reversible phosphomannose <br /> instead evolved PGMs: canonical glucose                  isomerase reaction, lead fructose-6-phosphate <br /> phosphate-specific PGM divergent able                 enter glycolytic pathway. The ability to <br /> convert mannose-phosphates (Mitra et al., 2010).                   divert significant portion carbon metabolism into <br />    The PGM activities TbPMM TbPAGM appear                GDP-sugar nucleotides fuel glycoprotein synthesis in <br /> be mutually redundant, RNAi knockdown            rapidly dividing trypanosomes, automatically shunt <br /> in turn significant effect UDP-Glc UDP-Gal           glycolysis glycoprotein synthesis is <br /> levels. This despite fact Western blot unpub-      repressed non-dividing cells, suggests mechanism <br /> lished proteomic data strongly suggest TbPAGM              help trypanosomes manage transformations <br /> much abundant TbPMM. Indeed, interesting           non-dividing metacyclic trypomastigotes and <br /> that TbPAGM, interconvert mannose-                  rapidly dividing slender trypomastigotes slender <br /> phosphates, substitute TbPMM that,                  non-dividing stumpy trypomastigotes. <br /> therefore, RNAi knockdown TbPMM affects protein gly-               In organisms, PMM PAGM activities are <br /> cosylation suppresses parasite growth. This consis-         located cytosol. However, bloodstream form <br /> tent previous studies using conditional null mutants          T. brucei, TbPMM localizes glycosomes and <br /> that downstream enzymes pathway                TbPAGM localizes glycosomes cytosol. <br /> GDP-Man metabolite GDP-Fuc essential           The glycosomes T. brucei related trypanosomatids <br /> (Turnock et al., 2007; Denton et al., 2010). In contrast,      belong, glyoxysomes plants per- <br /> RNAi knockdown TbPAGM described insuffi-               oxisomes eukaryotes, microbody family of <br /> cient significantly affect protein glycosylation sup-        organelles. When compared microbodies, <br /> press growth. This despite fact conditional        trypanosomatid glycosomes unique essential <br /> null mutants glucosamine-6-P N-acetyltransferase            role carbohydrate metabolism, i.e. seven steps <br /> UDP-GlcNAc pyrophosphorylase down-                   glycolysis, gluconeogenesis, glycerol metabolism and <br /> stream enzymes leading UDP-GlcNAc clearly                   pentose-phosphate pathway (Michels et al., 2006; <br /> essential (Stokes et al., 2008; Mari&#195;&#177;o et al., 2011). The          Haanstra et al., 2008). Glycosomal proteins typically <br /> notion phospho-sugar mutase(s) (in addition         targeted organelles Peroxisomal Targeting <br /> to TbPMM TbPAGM) exist T. brucei           Sequences (PTS), located C-terminus <br /> perform PGM function excluded searches               (PTS-1) N-terminus (PTS-2) protein. Internal target- <br /> updated trypanosomatid genomes TriTrypDB (Aslett                ing sequences proposed tar- <br /> et al., 2010) protein family (PFAM) domains         geting mechanism, T. cruzi PGM T. brucei <br /> relevant PMM, PAGM PGM activities (PF03332,                 triosephosphate isomerase (Penha et al., 2009; Galland <br /> PF02878, PF02879, PF02880 PF00408). These                      et al., 2010) proteins reach organelles by <br /> searches returned PMM PAGM genes                  &#226;&#8364;&#732;piggybacking&#226;&#8364;&#8482; glycosome-targeted proteins <br /> African trypanosomes, returned PMM,                   (Titorenko et al., 2002). As canonical trypanosome PTS <br /> PAGM PGM genes T. cruzi Leishmania.                sequences apparent TbPAGM TbPMM <br />    The TbPMM enzyme unusual features                 [although T. cruzi L. major PMM orthologues do <br /> compared eukaryotic PMMs. First, apparent             contain PTS-1 sequences (Opperdoes Szikora, <br /> Km Man-1-P relatively high (327 &#226;&#171;&#190; 66 mM) com-            2006)], likely use piggybacking <br /> parison values reported Arabidopsis thaliana            mechanism unidentified internal targeting <br /> (29.7 mM) (Qian et al., 2007), Galderia sulphuraria                sequence. An inefficient import TbPAGM glyco- <br /> (50 mM) (Oesterhelt et al., 1996), human PMM1                  somes explain partial localization enzyme <br /> (3.2 mM) PMM2 (18 mM) (Pirard et al., 1997; 1999).             cytosol bloodstream form T. brucei. In case, <br /> Second, unlike aforementioned enzymes, TbPMM                TbPMM TbPAGM join growing list enzymes <br /> inhibited high levels Man-1-P. Third, PMMs,          novo sugar nucleotide biosynthesis been <br /> including L. mexicana (Kedzierski et al., 2006),       localized glycosomes. These include hexokinase <br /> typically homodimers, TbPMM appears monomer                phosphoglucose isomerase (Visser Opperdoes, <br /> under physiological conditions. The properties             1980), glucosamine-6-P N-acetyltransferase (Mari&#195;&#177;o <br /> T. brucei GDP-Man pyrophosphorylase, shows sig-              et al., 2011), TbUAP (Stokes et al., 2008), TbUGP (Mari&#195;&#177;o <br /> nificant product inhibition (Denton et al., 2010),             et al., 2010), GalE (Roper et al., 2005), phosphomannose <br /> TbPMM, shows product inhibition Man-6-P            isomerase (Colasante et al., 2006) GDP-mannose <br /> Man-1-P direction, assist coupling GDP-sugar            4,6-dehydratase (Turnock et al., 2007). It possible that <br /> synthesis glycoprotein synthesis. Thus, reduction          sugar nucleotide biosynthesis machinery <br /> glycoprotein precursor synthesis reduce                  localized glycosome T. brucei rate <br /> demand GDP-Man lead accumulation                  synthesis metabolites controlled, some <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  528 G. Bandini et al. &#228;&#352;? <br />  <br /> degree, &#226;&#8364;&#732;compartmentalization&#226;&#8364;&#8482;. Such mechanism                thiogalactopyranoside (IPTG). After harvesting, cells <br /> already proposed alternative allosteric regu-            resuspended 50 mM Tris-HCl pH 7.3, 150 mM NaCl, <br /> lation glycolysis T. brucei (Haanstra et al., 2008).            100 mM imidazole, 1 mM DTT, 1 mg ml-1 lysozyme Com- <br />                                                                        plete Protease Inhibitor Cocktail Tablets (Roche) then <br />    In summary, expressed characterized the <br />                                                                        lysed using French press. The cell lysate cleared by <br /> PMM PAGM enzymes T. brucei determined                       centrifugation (17 000 g, 4&#194;&#176;C, 30 min), filtered a <br /> their subcellular locations high-resolution crystal            0.2 mm cellulose acetate membrane (Whatman) loaded <br /> structure TbPMM. Further, analysing substrate              5 ml Ni2+ HiTrap&#226;&#8222;&#162; Chelating HP column (GE <br /> specificities, provided solution appar-              Healthcare). Fractions collected checked SDS- <br /> ently paradoxical absence PGM gene T. brucei                PAGE. The TbPMM containing fractions pooled and <br /> demonstrating TbPMM TbPAGM inter-                    digested 0.1 mg PreScission&#226;&#8222;&#162; protease GST-tagged <br />                                                                        (a kind gift Daan van Aalten, University Dundee) over- <br /> convert Glc-6-P Glc-1-P. Selective knockdown of <br />                                                                        night 4&#194;&#176;C. The PreScission&#226;&#8222;&#162; protease removed by <br /> TbPMM TbPAGM inducible RNAi analysis                     incubating sample 2 h 4&#194;&#176;C 50 ml Glutathione <br /> sugar nucleotide levels suggests mutually                Sepharose&#226;&#8222;&#162; 4 Fast Flow (GE Healthcare) slurry by <br /> redundant particular function. It clear          separating beads centrifugation. A fraction the <br /> the routes key sugar-1-phosphates (Man-1-P,                   untagged TbPMM, present supernatant, used for <br /> GlcNAc-1-P Glc-1-P) key sugar                   mouse rabbit immunization. The remaining recombinant <br /> nucleotides (GDP-Man, GDP-Fuc, UDP-GlcNAc, UDP-                        protein used coupling CNBr Sepharose beads. <br /> Glc UDP-Gal) dependent just enzymes: <br /> TbPMM TbPAGM, thera-                      TbPMM protein expression purification activity <br /> peutic potential.                                                      assays crystallization trials <br />  <br />                                                                        TbPMM ORF amplified described cloned <br />                                                                        pET15-MHL vector (GenBank ID EF456738) and <br /> Experimental procedures                                                transformed E. coli BL21-(DE3)-V2R-pRARE2. A single <br /> DNA isolation manipulation                                         colony inoculated 100 ml LB medium contain- <br />                                                                        ing ampicillin/chloramphenicol (100 mg ml-1 34 mg ml-1 <br /> Plasmid DNA purified Escherichia coli (DH5a) using            respectively) inside 250 ml baffled flask incubated with <br /> the Qiagen Miniprep Maxiprep kit appropriate. Gel                shaking 250 r.p.m. overnight 37&#194;&#176;C. The culture was <br /> extraction performed using QIAquick kits. Custom oligo-            transferred 1.0 l TB antibiotic formula- <br /> nucleotides obtained University Dundee                tion pre-added. The culture allowed grow LEX <br /> oligonucleotide facility. T. brucei genomic DNA isolated           (Harbinger Biotechnology Engineering) an <br /> from 1 &#194;&#165; 108 bloodstream form cells using DNAzol (Helena               OD600 5&#226;&#8364;&#8220;6, cooled 15&#194;&#176;C induced 0.5 mM IPTG <br /> Biosciences). All T. brucei cell cultures mycoplasma-free.         overnight temperature. The culture harvested by <br />     All plasmids verified sequencing (DNA Sequenc-             centrifugation. The resulting pellets resuspended to <br /> ing Service, College Life Sciences, University Dundee;           approximately 40 ml l-1 cell culture Binding Buffer (50 mM <br /> http://www.dnaseq.co.uk). Recombinant proteins iden-              HEPES pH 7.5, 500 mM NaCl, 5 mM imidazole 5% glyc- <br /> tified Proteomics Mass Spectrometry Facility,               erol) protease inhibitors (1 mM benzamidine 1 mM <br /> College Life Sciences, University Dundee.                        phenylmethyl sulphonyl fluoride) added, stored -80&#194;&#176;C. <br />     Protein sequence multiple alignments assembled                   On day purification, pellets thawed <br /> using ClustalW (Chenna et al., 2003) Jalview (Water-               overnight 4&#194;&#176;C. Each pellet 1 l culture pre- <br /> house et al., 2009).                                                   treated 0.5% CHAPS 500 units benzonase for <br />                                                                        40 min room temperature, subsequently sonicated. <br />                                                                        After 6 min sonication, cell lysate centrifuged using a <br /> TbPMM expression purification                                  Beckman JA-25 rotor 24 000 r.p.m. 20 min 4&#194;&#176;C. The <br /> antibody production                                                    cleared lysate loaded 1.0&#226;&#8364;&#8220;2.5 ml Ni-NTA (Qiagen) <br />                                                                        open column (pre-equilibrated Binding Buffer) at <br /> The TbPMM (Tb927.10.6440) ORF amplified PCR                     approximately 1.5&#226;&#8364;&#8220;2.0 ml min-1. The Ni-NTA column was <br /> from T. brucei genomic DNA Platinum Taq DNA poly-                 washed 150 ml Wash Buffer (50 mM HEPES <br /> merase High Fidelity (Invitrogen) using forward reverse            pH 7.5, 500 mM NaCl, 30 mM imidazole 5% glycerol) at <br /> primers containing NdeI XhoI restriction sites (under-             2&#226;&#8364;&#8220;2.5 ml min-1. The protein eluted Elution <br /> lined) respectively: 5&#226;&#8364;&#178;-GGAATTCCATATGAAAAGAGTTCTT                      Buffer (50 mM HEPES pH 7.5, 500 mM NaCl, 250 mM imi- <br /> TTACTCTTTGAC-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-CCGCTCGAGTTACTTCATGG                            dazole 5% glycerol). The eluted sample applied a <br /> CAATTATTTTTTCC-3&#226;&#8364;&#178;. The NdeI XhoI restriction sites                 Sephadex S200 16/60 gel filtration column (GE Healthcare) <br /> were used clone TbPMM ORF pET15b                         pre-equilibrated Gel filtration Buffer (10 mM HEPES, <br /> (Novagen) modified PreScission&#226;&#8222;&#162; Protease (PP)                   pH 7.5, 500 mM NaCl) AKTA explorer (GE <br /> cleavage site, generating following expression construct:          Healthcare). The fractions corresponding eluted protein <br /> pET15b-His6-PP-TbPMM. The recombinant TbPMM                        peak pooled treated TEV protease <br /> expressed E. coli BL21(DE3). The cells grown over-             overnight cut His tag. The mixture loaded onto <br /> night 16&#194;&#176;C induction 0.75 mM isopropyl b-D-1-            1.0 ml Ni-NTA open column cut protein was <br />  <br />                                                                  &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                         PMM, PAGM phosphoglucomutase activity T. brucei 529 <br />  <br /> collected flow through. Its identity purity       stopped addition equal volume 0.2 M NaOH, <br /> evaluated mass spectroscopy SDS-PAGE gel. The                raising pH 12. The samples analysed High <br /> sample concentrated using 15 ml Amicon Ultra             Pressure Anion Exchange Chromatography coupled a <br /> centrifugal filter device (Millipore) 10 mg ml-1 stored   Pulse Amperometric Detector (HPAEC-PAD, Dionex) using a <br /> 4&#194;&#176;C.                                                                CarboPac PA1 column conditions adapted Zhou <br />                                                                     et al. (2002). For kinetic analysis, concentration of <br />                                                                     Man-1-P reaction varied 10 1000 mM. <br /> TbPAGM protein expression purification activity             Each concentration analysed triplicates. A high- <br /> assays antibody production                                      substrate inhibition equation (1) based non-linear fit was <br />                                                                     used calculate kinetic parameters reaction. <br /> TbPAGM (Tb927.8.980) amplified PCR T. brucei <br />                                                                                                   Vmax <br /> genomic DNA Platinum Taq DNA polymerase High Fidel-                                 v=                                                 (1) <br />                                                                                                   Km S <br /> ity (Invitrogen) using forward reverse primers containing                                  1+    + <br /> NdeI XhoI restriction sites (underlined) respectively:                                         S   Ki <br /> 5&#226;&#8364;&#178;-GGAATTCCATATGGTGCTGCAGGCT-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;-CGCCT                           All kinetic data fitted using GraFit5. <br /> CGAGCTACGCTCCACCGCAGA-3&#226;&#8364;&#178;. The NdeI XhoI <br /> restriction sites used clone TbPAGM ORF          Activity assays conversion Glc-6-P <br /> pET15b (Novagen) modified PreScission&#226;&#8222;&#162; Protease <br />                                                                     Glc-1-P <br /> (PP) cleavage site, generating following expression con- <br /> struct: pET15b-His6-PP-TbPAGM. The recombinant TbPAGM               The conversion Glc-1-P Glc-6-P recombinant TbPMM <br /> was expressed E. coli BL21(DE3). Cultures grown          TbPAGM followed HPAEC-PAD. The condi- <br /> 37&#194;&#176;C, LB medium containing ampicillin, OD600           tions reaction buffer described for <br /> approximately 0.5. IPTG added final concentration          Man-1-P Man-6-P conversion, 25 ng of <br /> of 0.5 mM cultures grown overnight 16&#194;&#176;C.            enzyme used substrate changed Glc-1-P. <br /> The cell pellets obtained centrifugation resus-            The conversion Glc-6-P Glc-1-P TbPMM and <br /> pended 50 mM Tris-HCl pH 7.7, 200 mM NaCl, 50 mM                 TbPAGM analysed using coupled assay with <br /> imidazole, presence DNase I, EDTA-free Complete           TbUGP. The formation final product (UDP-glucose) <br /> Protease Inhibitor Cocktail Tablets (Roche) 1 mg ml-1           followed HPAEC (Dionex) using CarboPac <br /> lysozyme. The cells lysed using French press         PA1 column UV detector set 260 nm. Different <br /> cell lysate cleared centrifugation (20 000 g, 4&#194;&#176;C, 30 min)       amounts recombinant proteins (50&#226;&#8364;&#8220;400 ng) incu- <br /> filtered 0.2 mm cellulose acetate membrane                bated 15 min room temperature 100 ml reaction volume <br /> (Whatman) loaded 5 ml Ni2+ HiTrap&#226;&#8222;&#162; Chelating             containing 2 mM Bis-Tris propane pH 7.3, 10 mM MgCl2, <br /> HP column (GE Healthcare). The TbPAGM-containing frac-              1 mM glucose-1,6-biphosphate, 500 mM Glc-6-P, 1 mM DTT, <br /> tions pooled digested 0.1 mg PreScission&#226;&#8222;&#162;          0.1 mg ml-1 BSA, 500 mM UTP 1 mg rx-1 TbUGP (Buffer <br /> protease GST-tagged described above. The cleaved                 CA). The reaction stopped addition equal <br /> protein purified gel filtration using Superdex     volume 0.2 M NaOH. <br /> G-200 10/30 column (Amersham) eluted 50 mM Tris-                Kinetic parameters determined using colorimetric <br /> HCl pH 7.7, 200 mM NaCl.                                            assay 96-well plate format (Cellstar) described in <br />                                                                     Stokes et al. (2008). Briefly, 25 ng recombinant TbPMM or <br />                                                                     TbPAGM incubated 100 ml reaction volume 15 min <br /> Analytical ultracentrifugation                                      room temperature buffer CA, containing 0.08 U rx-1 <br />                                                                     pyrophosphatase (Sigma). The concentration Glc-6-P was <br /> Recombinant untagged TbPMM analysed sedi- <br />                                                                     varied 10 1000 mM. After addition 100 ml of <br /> mentation velocity Beckman Optima XL-1 Analytical <br />                                                                     Biomol Green (Biomol International) 20 min incubation, <br /> Ultracentrifuge AN50-Ti rotor 4&#194;&#176;C, 32 000 r.p.m. <br />                                                                     A620 measured SpectraMax 340PC (Molecular <br /> The quaternary structure protein studied 0.25, <br />                                                                     Devices) plate reader. Triplicates run point. The <br /> 0.5 0.75 mg ml-1 different buffers: (i) 10 mM <br />                                                                     high-substrate inhibition equation used <br /> HEPES pH 7.5, 150 mM NaCl, (ii) 10 mM Tris-HCl pH 7.5, <br />                                                                     fit kinetic data TbPAGM. <br /> 1 mM DTT. Absorbance data collected analysed <br />                                                                        IC50 GlcNAc-1-P GlcNAc-6-P measured using <br /> using SEDFIT software (Schuck, 2004). TbPMM was <br />                                                                     colorimetric coupled assay. Briefly, 25 ng TbPAGM was <br /> assumed globular protein amino acid compo- <br />                                                                     incubated conditions described above. A fixed <br /> sition used determine density. <br />                                                                     concentration Glc-6-P 100 mM used, the <br />                                                                     GlcNAc-1-P GlcNAc-6-P concentration varied <br />                                                                     1 250 mM. The detection carried as <br /> Activity assay conversion Man-1-P Man-6-P <br />                                                                     above. <br /> To assay activity, 10 ng recombinant untagged TbPMM <br /> was incubated 100 ml reaction volume 20 min room          Activity assays conversion GlcNAc-6-P <br /> temperature buffer D (2 mM Bis-Tris propane pH 7.3, 5 mM         GlcNAc-1-P <br /> MgCl2, 1 mM glucose-1,6-biphosphate, 50 ng BSA and <br /> 250 mM Man-1-P). For TbPAGM, 100 ng enzyme incu-             The conversion GlcNAc-1-P GlcNAc-6-P recombi- <br /> bated buffer D 1 h 37&#194;&#176;C. The reaction            nant TbPAGM followed HPAEC-PAD. The condi- <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  530 G. Bandini et al. &#228;&#352;? <br />  <br /> tions reaction buffer (Buffer D)                    2001). The eluted sugar nucleotides analysed mul- <br /> described Man-1-P Man-6-P conversion,                  tiple reaction monitoring LC-MS/MS (Turnock Ferguson, <br /> expect 25 ng enzyme used substrate              2007). The P-values determined using Student&#226;&#8364;&#8482;s t-test. <br /> changed GlcNAc-1-P. For TbPMM, 250 ng enzyme was <br /> incubated Buffer D containing 800 mM GlcNAc-1-P 1 h <br /> at 37&#194;&#176;C analysis HPAEC-PAD.                                   Crystallization data collection <br />    The conversion GlcNAc-6-P GlcNAc-1-P TbPAGM <br /> was analysed using coupled assay TbUAP colo-                Crystals TbPMM obtained setting protein <br /> rimetric detection 96-well plate format (Cellstar)              sample obtained described hanging drop <br /> described above. Briefly, 25 ng recombinant TbPAGM               vapour diffusion experiment 20&#194;&#176;C. The successful crystal- <br /> incubated buffer CA (2 mM Bis-Tris propane pH 7.3,                   lization conditions leading structure were: 2 M <br /> 10 mM MgCl2, 1 mM glucose-1,6-biphosphate, 1 mM DTT,                    (NH4)2SO4, 0.2 M NaCl, 0.1 M sodium cacodylate, pH 6.0. <br /> 0.1 mg ml-1 BSA, 500 mM UTP 1.5 mg rx-1 TbUAP).                        Crystals data collection transferred paratone-N <br /> Kinetic parameters determined varying concen-               cryoprotectant, flash frozen liquid nitrogen data <br /> tration GlcNAc-6-P 5 100 mM. Triplicates                 collected Rigaku FRE Superbright anode a <br /> were run point.                                                RAXIS-IV plate imager. A complete data set collected, <br />                                                                         indexed scaled HKL2000 program (Otwi- <br />                                                                         nowski Minor, 1997). <br /> Parasite culture <br />  <br /> Bloodstream form T. brucei parasites (strain 427, variant               Structural determination model refinement <br /> MITat1.2) expressing T7 polymerase tetracycline repres- <br /> sor protein G418 selection (Wirtz et al., 1999)              The structure TbPMM solved molecular replace- <br /> grown HMI-9t medium 37&#194;&#176;C 5% CO2. HMI-9T                 ment using program PHASER LmexPMM struc- <br /> variant HMI-9 medium described Hirumi                     ture (2i54) modified side-chains replaced the <br /> Hirumi (1994) thioglycerol used instead                     Tb sequence FFAS03 (Jaroszewski et al., 2005). The <br /> b-mercaptoethanol (Greig et al., 2009).                                 model refined 1.85A data using initial round of <br />                                                                         ARP/WARP, followed iterative manual rebuilding COOT <br />                                                                         refinement REFMAC 5.5.0109 (ccp4 Suite, Murshu- <br /> Generation RNAi constructs transformation                     dov et al., 1997). The final model refined good <br />                                                                         geometry statistics, checked MOLPROBITY <br /> bloodstream form T. brucei <br />                                                                         outliers Ramachandran plot. Final data infor- <br /> RNAit       (http://trypanofan.path.cam.ac.uk/software/RNAit.           mation Table 2. TbPMM residues this <br /> html; Redmond et al., 2003) used identify suitable               paper numbered start methionine. <br /> RNAi internal coding sequences TbPMM (Tb927.10.6440, <br /> nt. 204&#226;&#8364;&#8220;667) TbPAGM (Tb927.8.980, nt. 676&#226;&#8364;&#8220;1172). The <br /> 464 nt (TbPMM) 497 bp (TbPAGM) fragments, contain-                  Production antibodies cell localization TbPMM <br /> ing XhoI BamHI restriction sites 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; respec-         TbPAGM immunofluorescence <br /> tively, synthesized Dundee Cell Products were <br /> then cloned p2T7TABlue (Alibu et al., 2005) using XhoI         Two adult Balb/c mice used raise polyclonal antibod- <br /> and BamHI sites generate p2T7-TbPMM p2T7-                        ies E. coli overexpressed untagged TbPMM <br /> TbPAGM. These constructs digested NotI, precipi-              protein Freund&#226;&#8364;&#8482;s complete adjuvant. Each animal <br /> tated, washed 70% ethanol redissolved sterile               received immunizations Freund&#226;&#8364;&#8482;s incomplete <br /> water, used T. brucei transformation                adjuvant 2 months. A pool mice sera used <br /> Amaxa nucleofection previously described (Burkard et al.,            Western blots immunofluorescence. Similar proce- <br /> 2007). The final constructs verified sequencing (DNA            dures performed TbPAGM. <br /> Sequencing Service, College Life Sciences, University                For immunofluorescence experiments, wild type blood- <br /> Dundee) transformation.                                          stream form T. brucei cells grown HMI-9T medium <br />                                                                         (Hirumi Hirumi, 1994; Greig et al., 2009) density 1 <br />                                                                         2 &#194;&#165; 106 cells ml-1, harvested centrifugation 800 g for <br /> Sugar nucleotide analysis                                               10 min 4&#194;&#176;C resuspended trypanosome dilution <br />                                                                         buffer (20 mM Na2HPO4, 2 mM NaH2PO4, 5 mM KCl, 80 mM <br /> Sugar nucleotide extraction analysis performed              NaCl, 1 mM MgSO4, 20 mM glucose pH 7.8) density of <br /> previously described (Turnock Ferguson, 2007). Briefly,             2 &#194;&#165; 107 cells ml-1. Aliquots (10 ml) placed 13 mm pre- <br /> cells pelleted centrifugation, washed ice-cold               cooled coverslips inside 12-well plate, covered, incubated for <br /> phosphate buffer saline lysed 70% ethanol                 15 min 4&#194;&#176;C subsequently fixed 1 ml freshly pre- <br /> presence 20 pmol GDP-glucose internal standard                pared 4% paraformaldehyde phosphate-buffered saline <br /> (Sigma). The lysate centrifuged remove insoluble                 (PBS) 30 min 4&#194;&#176;C followed washes 1 ml of <br /> material supernatant extracted butan-1-ol               PBS. Cells permeabilized 0.1% Triton X-100 in <br /> remove lipids. Sugar nucleotides extracted                PBS 10 min room temperature. Samples then <br /> resulting aqueous phase using EnviCarb graphitized carbon               blocked 5% fish skin gelatin (FSG) PBS containing 10% <br /> columns (Supelco) previously described (Rabina et al.,               normal goat serum. The coverslips incubated humid <br />  <br />                                                                   &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                        PMM, PAGM phosphoglucomutase activity T. brucei 531 <br />  <br /> chamber 1:1000 mouse anti-TbPMM mixed              5 ml pooled mice anti-TbPMM sera added the <br /> 1:2000 rabbit anti-glyceraldehyde-3-phosphate dehydroge-            supernatant incubated 1 h 4&#194;&#176;C. Subsequently, <br /> nase (GAPDH) antiserum 1:4000 rabbit anti-enolase anti-          50 ml protein G Dynabeads (Invitrogen) added and <br /> serum diluted 1% FSG PBS, 0.05% Triton X-100. Both            incubated 1 h 4&#194;&#176;C. The beads recovered by <br /> anti-GAPDH anti-enolase kindly provided Paul            placing magnet 2 min washed times with <br /> Michels (Catholic University Louvain, Belgium). Samples          cold 20 mM Tris-HCl pH 7.2, 0.15 M NaCl, 0.03% SDS and <br /> were subsequently washed 1% FSG PBS, 0.05%                  1% Triton X-100. The beads boiled SDS sample <br /> Triton X-100 incubated 50 ml 1:500 diluted Alexa        buffer 0.1 M DTT, run 4&#226;&#8364;&#8220;12% Nupage gel and <br /> 488-conjugated goat anti-mouse IgG 1:500 diluted Alexa          Western blotted described above. <br /> 594-conjugated goat anti-rabbit IgG 1 h. Coverslips were <br /> washed mounted glass slides drop Hydro- <br /> mount containing 2.5% DABCO left set dark         Western blot anti-TbPAGM anti-sVSG <br /> 30 min. Microscopy performed Zeiss LSM 510 <br /> META confocal microscope.                                           TbPAGM RNAi cells induced tetracycline 2 <br />                                                                     3 days, lysed SDS sample buffer, loaded (5 &#194;&#165; 106 cell <br />                                                                     equivalents lane) SDS-PAGE gel blotted nitro- <br /> Digitonin latency Western blotting                              cellulose. The membrane blocked 2% (w/v) FSG in <br />                                                                     blocking buffer described developed 1:1000 <br /> Digitonin latency performed previously reported              mouse anti-TbPAGM mixed 1:4000 rabbit anti-aldolase. <br /> (Galland et al., 2007). Briefly, T. brucei bloodstream form cells   The membrane washed times 0.1% Tween-20 <br /> (2 &#194;&#165; 109 cells) washed twice 10 ml STE buffer          PBS, incubated 1:15 000 donkey anti-mouse green <br /> (250 mM sucrose, 25 mM Tris-HCl pH 7.4 1 mM EDTA)               infrared conjugate (IRDye 800 CW, Licor) mixed 1:20 000 <br /> and resuspended 1.5 ml STEN (STE containing 0.15 M            donkey anti-rabbit red conjugate (IRDye 680, Licor) and <br /> NaCl). Aliquots (0.15 ml) treated equal volume         scanned Licor Odyssey Infrared Imaging System. The <br /> digitonin various concentrations presence 0.1 mM       Licor imager software used quantify signal both <br /> TLCK, 1 mg ml-1 leupeptin, 1 mg ml-1 aprotinin 1 mM             infrared channels gel loading normalized against <br /> PMSF. Digitonin stock prepared 10 mg ml-1 DMSO            aldolase signal. For sVSG analysis, TbPAGM RNAi <br /> and diluted STEN required concentrations. Complete        cells induced tetracycline 2 3 days <br /> extraction obtained parallel treating equivalent       TbPMM RNAi cells 2 days, lysed in <br /> number cells 0.1% Triton X-100 STEN. The lysates         SDS sample buffer, loaded (5 &#194;&#165; 106 cell equivalents lane) <br /> were incubated 5 min room temperature                SDS-PAGE gel Western blotted nitrocellulose. The <br /> insoluble materials removed centrifugation 16 000 g       Licor development described above, instead rabbit <br /> 2 min. The pellets discarded; aliquots superna-         anti-sVSG221 used 1:2000 dilution. <br /> tants run reducing 4&#226;&#8364;&#8220;12% gradient NuPage gel <br /> (Invitrogen) transferred nitrocellulose Western blot- <br /> ting. Note: Different amounts samples loaded <br />                                                                     Quantitative reverse transcriptase PCR (qRT-PCR) <br /> onto separate gels antibody Western blot, order to <br /> keep detection ECL film exposure linear           RNA extracted using RNeasy RNA extraction kit <br /> during Western blot development; TbPMM 1 &#194;&#165; 107 cell         (Quiagen). cDNA synthesized 1 mg RNA using <br /> equivalents lane, TbPAGM 5 &#194;&#165; 106 cell equivalents       oligo(dT) random primers qScript cDNA synthe- <br /> lane, aldolase (aldo) 1 &#194;&#165; 105 cell equivalents lane,    sis (Quanta Biosciences), diluted 1 100 <br /> PEX13 5 &#194;&#165; 106 lane enolase (eno) 1 &#194;&#165; 105 cell           nuclease-free water. qRT-PCR reactions included 3 ml of <br /> equivalents lane. The membranes blocked               diluted cDNA, SYBR Green Master Mix (Quanta Bio- <br /> 0.25% bovine serum albumin, 0.05% Tween-20 (Sigma),                 sciences), 0.6 mM forward (atgagggctttgataaagcgagc) <br /> 0.15 M NaCl 50 mM Tris-HCl pH 7.4. The membranes            reverse (cgtccgagttgctcaacctgc) gene-specific primers. <br /> probed parallel following antibodies 1:1000 diluted     Amplification carried iCycler iQ5 PCR thermal <br /> mouse anti-TbPMM, 1:1000 mouse anti-TbPAGM, 1:4000                  cycler (Bio-Rad). Mean standard deviation deter- <br /> rabbit anti-aldolase, 1:5000 rabbit anti-Pex13 1:5000 rabbit     mined combining replicate analyses. <br /> anti-enolase 1 h 37&#194;&#176;C. The membranes washed <br /> and incubated 1 h room temperature 1:100 000 goat <br /> anti-mouse goat anti-rabbit horseradish peroxidase conju-        Acknowledgements <br /> gate. After washing, membranes developed by <br /> chemiluminescent detection (Super Signal WestPico, Thermo           The authors like thank Mark Agacan Uni- <br /> Scientific) film (Hyperbond ECL, GE Healthcare).                versity Dundee Proteomics Facility (Wellcome Strategic <br />    For immunoprecipitation native TbPMM (to check        Award 083481) performing AUC proteomics <br /> mono-specificity), 2 &#194;&#165; 108 T. brucei bloodstream form cells         experiments Alan R. Prescott (Division Cell Signalling <br /> were lysed 1% (w/v) SDS 20 mM Tris-HCl pH 6.8 con-            Immunology, University Dundee) generous help <br /> taining 0.1 M DTT heated 50&#194;&#176;C 15 min. SDS            microscopy. We like thank Michael <br /> diluted 0.03% 1% (w/v) Triton X-100 20 mM Tris-          Urbaniak SILAC data Alan Fairlamb high <br /> HCl pH 6.8 0.15 M NaCl presence 0.1 mM               substrate inhibition equation. We grateful Daan van <br /> TLCK, 1 mg ml-1 leupeptin, 1 mg ml-1 aprotinin 1 mM             Aalten providing reagents Paul Michels providing <br /> PMSF. The insoluble material removed centrifugation,         anti-GAPDH anti-enolase antibodies. <br />  <br /> &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />  532 G. Bandini et al. &#228;&#352;? <br />  <br /> References                                                                  Crabtree, J., Aggarwal, G., et al. (2005b) Comparative <br />                                                                             genomics trypanosomatid parasitic protozoa. Science <br /> Alibu, V.P., Storm, L., Haile, S., Clayton, C., Horn, D.                309: 404&#226;&#8364;&#8220;409. <br />   (2005) A doubly inducible RNA interference               Engstler, M., Weise, F., Bopp, K., Grunfelder, C.G., Gunzel, <br />   rapid RNAi plasmid construction Trypanosoma brucei.                    M., Heddergott, N., Overath, P. (2005) The membrane- <br />    Mol Biochem Parasitol 139: 75&#226;&#8364;&#8220;82.                                        bound histidine acid phosphatase TbMBAP1 essential <br /> Allen, K.N., Dunaway-Mariano, D. (2004) Phosphoryl                      endocytosis membrane recycling Trypanosoma <br />    group transfer: evolution catalytic scaffold. Trends                brucei. J Cell Sci 118: 2105&#226;&#8364;&#8220;2118. <br />    Biochem Sci 29: 495&#226;&#8364;&#8220;503.                                               Favre, E., Wissmann, B., Welburn, S., Lutumba, P. <br /> Aslett, M., Aurrecoechea, C., Berriman, M., Brestelli, J.,                  (2008) The burden human African trypanosomiasis. <br />    Brunk, B.P., Carrington, M., et al. (2010) TriTrypDB: func-            PLoS Negl Trop Dis 2: e333. <br />    tional genomic resource Trypanosomatidae. Nucleic              Fernandez-Sorensen, A., Carlson, D.M. (1971) Purifica- <br />    Acids Res 38: D457&#226;&#8364;&#8220;D462.                                                 tion properties phosphoacetylglucosamine mutase. <br /> Barrett, M.P., Tetaud, E., Seyfang, A., Bringaud, F., Baltz,            J Biol Chem 246: 3485&#226;&#8364;&#8220;3493. <br />    T. (1995) Functional expression characterization            Flores-D&#195;&#173;az, M., Alape-Gir&#195;&#179;n, A., Persson, B., Pollesello, P., <br />    Trypanosoma brucei procyclic glucose transporter, THT2.                  Moos, M., von Eichel-Streiber, C., et al. (1997) Cellular <br />    Biochem J 312 (Part 3): 687&#226;&#8364;&#8220;691.                                         UDP-glucose deficiency caused single <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>point mutation</span> <br /> Berriman, M., Ghedin, E., Hertz-Fowler, C., Blandin, G.,                    UDP-glucose pyrophosphorylase gene. J Biol Chem <br />    Renauld, H., Bartholomeu, D.C., et al. (2005) The genome                 272: 23784&#226;&#8364;&#8220;23791. <br />    African trypanosome Trypanosoma brucei. Science                 Freeze, H., Elbein, A. (2008) Glycosylation precursors. In <br />    309: 416&#226;&#8364;&#8220;422.                                                            Essentials Glycobiology. Varki, A., Cummings, R., Esko, <br /> Boles, E., Liebetrau, W., Hofmann, M., Zimmermann,                      J., Freeze, H., Stanley, P., Bertozzi, C., et al. (eds). New <br />    F.K. (1994) A family hexosephosphate mutases Sac-                  York, NY, USA: Cold Spring Harbour Laboratory Press, pp. <br />    charomyces cerevisiae. Eur J Biochem 220: 83&#226;&#8364;&#8220;96.                         47&#226;&#8364;&#8220;62. <br /> Borst, P., Sabatini, R. (2008) Base J: discovery, biosyn-             Galland, N., Demeure, F., Hannaert, V., Verplaetse, E., Ver- <br />    thesis, possible functions. Annu Rev Microbiol 62:                   tommen, D., Smissen, P.V.D., et al. (2007) Characteriza- <br />    235&#226;&#8364;&#8220;251.                                                                 tion role receptors PEX5 PEX7 the <br /> Burkard, G., Fragoso, C.M., Roditi, I. (2007) Highly effi-              import proteins glycosomes Trypanosoma brucei. <br />    cient stable transformation bloodstream forms Trypa-               Biochim Biophys Acta 1773: 521&#226;&#8364;&#8220;535. <br />    nosoma brucei. Mol Biochem Parasitol 153: 220&#226;&#8364;&#8220;223.                     Galland, N., Walque, S., Voncken, F.G.J., Verlinde, C.L. <br /> Caramelo, J.J., Castro, O.A., Prat-Gay, G., Parodi, A.J.             M.J., Michels, P.A.M. (2010) An internal sequence <br />    (2004) The endoplasmic reticulum glucosyltransferase rec-                targets Trypanosoma brucei triosephosphate isomerase to <br />    ognizes nearly native glycoprotein folding intermediates.                glycosomes. Mol Biochem Parasitol 171: 45&#226;&#8364;&#8220;49. <br />    J Biol Chem 279: 46280&#226;&#8364;&#8220;46285.                                          Greig, N., Wyllie, S., Patterson, S., Fairlamb, A.H. (2009) <br /> Chenna, R., Sugawara, H., Koike, T., Lopez, R., Gibson, T.J.,               A comparative study methyglyoxal metabolism in <br />    Higgins, D.G., Thompson, J.D. (2003) Multiple                        trypanosomatids. FEBS J 276: 376&#226;&#8364;&#8220;386. <br />    sequence alignment Clustal series programs.                Haanstra, J.R., Van Tuijl, A., Kessler, P., Reijnders, W., <br />    Nucleic Acids Res 31: 3497&#226;&#8364;&#8220;3500.                                         Michels, P.A.M., Westerhoff, H.V., et al. (2008) Compart- <br /> Colasante, C., Ellis, M., Ruppert, T., Voncken, F. (2006)               mentation prevents lethal turbo-explosion glycolysis in <br />    Comparative proteomics glycosomes bloodstream                    trypanosomes. Proc Natl Acad Sci USA 105: 17718&#226;&#8364;&#8220;17723. <br />    form procyclic culture form Trypanosoma brucei                     Hirumi, H., Hirumi, K. (1994) Axenic culture African <br />    brucei. Proteomics 6: 3275&#226;&#8364;&#8220;3293.                                         trypanosome bloodstream forms. Parasitol Today 10: <br /> Collet, J.F., Stroobant, V., Pirard, M., Delpierre, G., Van             80&#226;&#8364;&#8220;84. <br />    Schaftingen, E. (1998) A new class phosphotransferases              Hofmann, M., Boles, E., Zimmermann, F.K. (1994) Char- <br />    phosphorylated aspartate residue amino-                      acterization essential yeast gene encoding <br />    terminal DXDX(T/V) motif. J Biol Chem 273: 14107&#226;&#8364;&#8220;14112.                  N-acetylglucosamine-phosphate mutase. Eur J Biochem <br /> Cross, G.A. (1996) Antigenic variation trypanosomes:                     221: 741&#226;&#8364;&#8220;747. <br />    secrets surface slowly. Bioessays 18: 283&#226;&#8364;&#8220;291.                         Izquierdo, L., Atrih, A., Rodrigues, J.A., Jones, D.C., and <br /> Davis, A.J., Perugini, M.A., Smith, B.J., Stewart, J.D., Ilg, T.,           Ferguson, M.A. (2009) Trypanosoma brucei UDP- <br />    Hodder, A.N., Handman, E. (2004) Properties GDP-                  glucose:glycoprotein glucosyltransferase unusual sub- <br />    mannose pyrophosphorylase, critical enzyme drug                    strate specificity protects parasite stress. <br />    target Leishmania mexicana. J Biol Chem 279: 12462&#226;&#8364;&#8220;                   Eukaryot Cell 8: 230&#226;&#8364;&#8220;240. <br />    12468.                                                                 Jaroszewski, L., Rychlewski, L., Li, Z., Li, W., Godzik, A. <br /> Denton, H., Fyffe, S., Smith, T.K. (2010) GDP-mannose                   (2005) FFAS03: server profile-profile sequence <br />    pyrophosphorylase essential bloodstream form                alignments. Nucleic Acids Res 33: W284&#226;&#8364;&#8220;W288. <br />    Trypanosoma brucei. Biochem J 425: 603&#226;&#8364;&#8220;614.                            Kato, N., Mueller, C.R., Wessely, V., Lan, Q., Chris- <br /> El-Sayed, N.M., Myler, P.J., Bartholomeu, D.C., Nilsson, D.,                tensen, B.M. (2005) Aedes aegypti phosphohexomutases <br />   Aggarwal, G., Tran, A.N., et al. (2005a) The genome                       uridine diphosphate-hexose pyrophosphorylases: <br />   sequence Trypanosoma cruzi, etiologic agent chagas                  comparison primary sequences, substrate specificities <br />    disease. Science 309: 409&#226;&#8364;&#8220;415.                                           temporal transcription. Insect Mol Biol 14: 615&#226;&#8364;&#8220; <br /> El-Sayed, N.M., Myler, P.J., Blandin, G.L., Berriman, M.,                   624. <br />  <br />                                                                     &#194;&#169; 2012 Blackwell Publishing Ltd, Molecular Microbiology, 85, 513&#226;&#8364;&#8220;534 <br />                                                                                         PMM, PAGM phosphoglucomutase activity T. brucei 533 <br />  <br /> Kedzierski, L., Malby, R.L., Smith, B.J., Perugini, M.A.,           Pays, E., Nolan, D.P. (1998) Expression function of <br />   Hodder, A.N., Ilg, T., et al. (2006) Structure Leishmania        surface proteins Trypanosoma brucei. Mol Biochem <br />   mexicana phosphomannomutase highlights similarities                 Parasitol 91: 3&#226;&#8364;&#8220;36. <br />   human isoforms. J Mol Biol 363: 215&#226;&#8364;&#8220;227.                     Peck, R.F., Shiflett, A.M., Schwartz, K.J., McCann, A., <br /> Kepes, F., Schekman, R. (1988) The yeast SEC53 gene               Hajduk, S.L., Bangs, J.D. (2008) The LAMP-like <br />   encodes phosphomannomutase. J Biol Chem 263: 9155&#226;&#8364;&#8220;                  protein p67 plays essential role lysosome of <br />   9161.                                                               African trypanosomes. Mol Microbiol 68: 933&#226;&#8364;&#8220;946. <br /> Lackovic, K., Parisot, J.P., Sleebs, N., Baell, J.B., Debien, L.,   Penha, L.L., Mendon&#195;&#167;a-Previato, L., Previato, J.O., Scharf- <br />   Watson, K.G., et al. (2010) Inhibitors Leishmania GDP-           stein, J., Heise, N., Lima, A.P. (2005) Cloning and <br />   mannose pyrophosphorylase identified high-throughput             characterization phosphoglucomutase Trypano- <br />   screening small-molecule chemical library. Antimicrob            soma cruzi functional complementation Saccha- <br />   Agents Chemother 54: 1712&#226;&#8364;&#8220;1719.                                     romyces cerevisiae PGM null mutant. Glycobiology 15: <br /> Macrae, J.I., Obado, S.O., Turnock, D.C., Roper, J.R.,                1359&#226;&#8364;&#8220;1367. <br />   Kierans, M., Kelly, J.M., et al. (2006) The suppression        Penha, L.L., Sant&#226;&#8364;&#8482;Anna, C.B., Mendon&#195;&#167;a-Previato, L., <br />   galactose metabolism Trypanosoma cruzi epimastigotes             Cunha-e-Silva, N.L., Previato, J.O., Lima, A.P. (2009) <br />   causes changes cell surface molecular architecture           Sorting phosphoglucomutase glycosomes Trypano- <br />   cell morphology. Mol Biochem Parasitol 147: 126&#226;&#8364;&#8220;136.                soma cruzi mediated internal domain. Glycobiol- <br /> Mari&#195;&#177;o, K., Sampaio-G&#195;&#188;ther, M.L., Wernimont, A.K., Amani,             ogy 19: 1462&#226;&#8364;&#8220;1472. <br />   M., Hui, R., Ferguson, M.A. (2010) Identification, sub-       Pirard, M., Collet, J.F., Matthijs, G., Van Schaftingen, E. <br />   cellular localization, biochemical properties high-             (1997) Comparison PMM1 phosphomannomu- <br />   resolution crystal structure Trypanosoma brucei UDP-             tases expressed rat liver human cells. FEBS Lett <br />   glucose pyrophosphorylase. Glycobiology 20: 1619&#226;&#8364;&#8220;                   411: 251&#226;&#8364;&#8220;254. <br />   1630.                                                             Pirard, M., Achouri, Y., Collet, J.F., Schollen, E., Matthijs, G., <br /> Mari&#195;&#177;o, K., G&#195;&#188;ther, M.L.S., Wernimont, A.K., Qiu, W., Hui, R.,        Van Schaftingen, E. (1999) Kinetic properties tis- <br />   Ferguson, M.A.J. (2011) Characterization, localization,         sular distribution mammalian phosphomannomutase <br />   essentiality high-resolution crystal structure               isozymes. Biochem J 339 (Part 1): 201&#226;&#8364;&#8220;207. <br />   Glucosamine-6-phosphate N-acetyltransferase Trypa-           Qian, W., Yu, C., Qin, H., Liu, X., Zhang, A., Johansen, I.E., <br />   nosoma brucei. Eukaryot Cell 10: 985&#226;&#8364;&#8220;997.                           Wang, D. (2007) Molecular functional analysis of <br /> Mehlert, A., Ferguson, M.A. (2007) Structure gly-          phosphomannomutase (PMM) higher plants and <br />   cosylphosphatidylinositol anchor Trypanosoma                 genetic evidence involvement PMM ascorbic <br />   brucei transferrin receptor. Mol Biochem Parasitol 151:             acid biosynthesis Arabidopsis Nicotiana benthami- <br />   220&#226;&#8364;&#8220;223.                                                            ana. Plant J 49: 399&#226; <br /> </body></html>