<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The intron tRNA-TrpCCA dispensable growth <br /> and translation Saccharomyces cerevisiae <br />  <br />  <br /> SHUNSUKE MORI,1 TAKUYA KAJITA,1 TOSHIYA ENDO,1 TOHRU YOSHIHISA2,3,4 <br /> 1 <br />  Department Chemistry, Graduate School Science, Nagoya University, Nagoya, 464-8602, Japan <br /> 2 <br />  Research Center Materials Science, Nagoya University, Nagoya, 464-8602, Japan <br /> 3 <br />  Precursory Research Embryonic Science Technology, Japan Science Technology Agency, Kawaguchi, 332-0012, Japan <br />  <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> A eukaryotic tRNA genes harbor intron nucleotide 39 anticodon, removal intron an <br /> essential processing step tRNA maturation. While tRNA introns important roles modification certain <br /> nucleotides, essentiality tRNA intron eukaryotes tested extensively. This partly the <br /> eukaryotic genomes multiple genes encoding isoacceptor tRNA. Here, examined intron tRNA-TrpCCA <br /> genes, copies scattered genome yeast, Saccharomyces cerevisiae, essential growth translation <br /> of yeast vivo. We devised procedure remove tRNA introns yeast genome iteratively marker <br /> cassettes containing positive negative markers. Using procedure, removed introns tRNA- <br /> TrpCCA genes, intronless strain grew normally expressed tRNA-TrpCCA similar the <br /> wild-type genes. Neither incorporation 35S-labeled amino acids TCA-insoluble fraction major protein <span id='am-85' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> on SDS-PAGE/2D gel affected complete removal intron, expression levels proteins marginally <br /> affected. Therefore, tRNA-TrpCCA intron dispensable growth bulk translation yeast. This raises possibility <br /> that mechanism selective pressure translational efficiency maintains tRNA intron yeast genome. <br /> Keywords: tRNA; intron; tRNA splicing; tRNA processing; Saccharomyces cerevisiae <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                   formed anticodon loop intron. On other <br />                                                                                hand, eukaryotic intron does motif and <br /> tRNAs play essential role protein translation adaptor <br />                                                                                thought recognized splicing machinery through <br /> molecules nucleic acid sequence amino <br />                                                                                &#226;&#8364;&#732;&#226;&#8364;&#732;ruler mechanism&#226;&#8364;&#8482;&#226;&#8364;&#8482; measure distance between <br /> acid sequence. During maturation, tRNAs undergo com- <br />                                                                                cleavage sites body mature tRNA (Abelson <br /> plex sophisticated processing intracellular dynamics <br />                                                                                et al. 1998). Recently, noncanonical introns interrupt <br /> in eukaryotes (Yoshihisa 2006; Hopper et al. 2010; Phizicky <br />                                                                                parts tRNAs discovered various archaea <br /> and Hopper 2010). Primary transcripts tRNAs receive 59 <br />                                                                                certain eukaryotes (for review, Heinemann et al. 2010). <br /> and 39 trimming catalyzed RNase P RNase Z, re- <br />                                                                                In extreme cases, introns inserted tRNA- <br /> spectively. Then, CCA sequence added 39 end. <br />                                                                                ProUGG gene Pyrobaculum islandicum, split tRNA <br /> Various positions bases sugars modified. <br />                                                                                transcripts joined form mature tRNA splicing- <br /> These modifications place appropriate time points <br />                                                                                like mechanism <span id='am-84' about='xsp:base' typeof='owl:Thing'>base</span>-pairing ap- <br /> and places cell. <br />                                                                                pended segments (Sugahara et al. 2007; Fujishima et al. <br />    Some tRNAs encoded genes intron. The <br />                                                                                2009). <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> remove tRNA introns, proteinaceous <br /> canonical tRNA intron, eukaryotes archaea, lo- <br />                                                                                enzymes catalyze splicing steps. tRNA splicing <br /> cated nucleotide 39 anticodon (Abelson et al. <br />                                                                                endonucleases, govern cleavage splice sites, are <br /> 1998; Heinemann et al. 2010). In archaeal tRNA genes, typ- <br />                                                                                conserved amino acid sequences eukaryotes <br /> ical splice sites consist bulge-helix-bulge motif is <br />                                                                                archaea, intracellular localization shows <br />                                                                                diversity eukaryotic cells (Trotta et al. 1997; Yoshihisa <br />                                                                                et al. 2003; Paushkin et al. 2004; Englert et al. 2007; <br />    4 <br />     Corresponding author.                                                      Yoshihisa et al. 2007). Furthermore, organisms adopted <br />    E-mail tyoshihi@biochem.chem.nagoya-u.ac.jp. <br />    Article published online ahead print. Article publication date   different reaction pathways ligate tRNA exons <br /> at http://www.rnajournal.org/cgi/doi/10.1261/rna.2851411.                      (Filipowicz Shatkin 1983; Phizicky et al. 1986; Culver <br />  <br /> 1760                                 RNA (2011), 17:1760&#226;&#8364;&#8220;1769. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2011 RNA Society. <br />                                                                                        tRNA-Trp intron dispensable yeast <br />  <br />  <br />  <br /> et al. 1997; Englert Beier 2005; Weitzer Martinez        overproduction. These facts suggest functional redundancy <br /> 2007; Schwer et al. 2008; Englert et al. 2010; Popow et al.      tRNA-SerCGA ncm5U-modified tRNA-SerUGA <br /> 2011). Despite diversity, intron removal itself     (Johansson Bystro&#194;&#168;m 2004; Johansson et al. 2008) and <br /> essential functionalization tRNAs organism.      marginal effects intron removal tS(CAG)C may <br />    The studied eukaryote tRNA splicing Sac-      covered redundancy. Therefore, strictest <br /> charomyces cerevisiae, genome harbors 61 intron-    sense, requirement tRNA intron eukaryotic cells <br /> containing tRNA genes encoding 10 different isoacceptor          tested isoacceptor tRNAs. <br /> tRNAs (Abelson et al. 1998; Lowe Chan 2010). The                Here, examined essentiality canonical intron in <br /> introns vary size 14 nt 60 nt,        tRNA-TrpCCA genes S. cerevisiae, model case to <br /> locate canonical position, nucleotide 39       study essentiality tRNA introns yeast genome. <br /> anticodon. It genes encoding   tRNA-TrpCCA unique Trp codon, <br /> isoacceptor tRNAs introns, indicating yeast        UGG, nuclear genome, cognate <br /> needs maintain tRNA introns genes. How-          isoacceptor tRNA exists. Since tRNA-TrpCCA encoded by <br /> ever, fully understood tRNA intron       genes sequence including intron <br /> contributes life yeast. The introns tRNAs      (Atkin et al. 1992), developed systematic protocol to <br /> are, indeed, required proper modifications. For        replace multiple tRNA genes yeast chromosomes <br /> example, pseudouridylation positions 25, 34, 36,        intronless version. Using technique, suc- <br /> and 67 tRNA-IleUAU catalyzed Pus1p, pseudouri-             ceeded yielding strain tRNA-TrpCCA in- <br /> dylation positions 34 36 applied       tron genome demonstrated feasibility this <br /> intron-containing precursor vitro (Szweykowska-Kulinska       technique examine necessity introns other <br /> et al. 1994). Similarly, Trm4p converts C34 tRNA-             isoacceptor tRNA genes. Surprisingly, find <br /> LeuCAA 5-methylcytidine nucleotide exists       significant difference wild-type tRNA- <br /> on precursor (Strobel Abelson 1986; Grosjean et al.      TrpCCA intronless strains growth protein synthe- <br /> 1997). Modification 34th nucleotide, wobble           sis. This indicates intron little, any, impact <br /> position, considered affect decoding ability tRNAs      functionality tRNA-TrpCCA S. cerevisiae. <br /> (Agris et al. 2007). Besides, class I aminoacyl-tRNA syn- <br /> thetase (ARS) recognizes anticodon cognate tRNA, <br />                                                                  RESULTS <br /> and recognition affected modifications the <br /> anticodon (Senger et al. 1997). Despite facts, tRNA <br />                                                                  Strategy sequential removal tRNA-TrpCCA <br /> introns considered critical element tRNA <br />                                                                  introns yeast genome <br /> biogenesis deletion intron-dependent modifi- <br /> cation enzymes little effect yeast growth (Simos et al.   In S. cerevisiae, tRNA-TrpCCA genes clustered <br /> 1996; Motorin Grosjean 1999).                                scattered genome; genes exist on <br />    On hand, direct experiments examine              chromosome VII chromosomes X, XI, <br /> essentiality eukaryotic tRNA introns ham-           XIII, XVI (Fig. 1A). All genes same <br /> pered fact isoacceptor tRNAs         sequence interrupted single 34-nt intron (Fig. <br /> encoded multiple genes. Even case simple          1B; intron sequences enclosed square). Conservation <br /> organism like S. cerevisiae, intron-containing isoacceptor       intron, removed tRNA <br /> tRNAs encoded 10 degenerated genes. While           maturation, great contrast variability 59- <br /> suppressor tRNA <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> used study function-          39-flanking sequences tRNA-TrpCCA genes (Fig. <br /> ality intron-deleted tRNA genes expressed trans (Strobel   1B). It suggests tRNA-TrpCCA introns likely <br /> and Abelson 1986; Ho Abelson 1988), method              selected genome S. cerevisiae. It true in <br /> cannot measure decoding activity original antico-       comparison orthologous chromosomal regions <br /> dons. The exception yeast tRNA-SerCGA,           corresponding tRNA-TrpCCA locus, tW(CCA)G2, <br /> encoded single gene, tS(CGA)C SUP61. The intron          closely related yeasts Saccharomyces genus (Fig. <br /> was proven nonessential replacing chromosomal           1C). The intron conserved 59- 39- <br /> tS(CGA)C intronless version (Ho Abelson              flanking regions tW(CCA)G2 locus. <br /> 1988). However, sequence tRNA-SerCGA quite                To achieve sequential alterations multiple tRNA- <br /> similar tRNA-SerUGA encoded tS(UGA)P            TrpCCA genes complete removal tRNA-TrpCCA <br /> SUP16; nucleotides different          introns yeast genome, repetitive usage selection <br /> two. Overproduction tS(UGA)P suppress lethality        markers essential. For purpose, adopted marker <br /> ts(cga)cD, elp3D mutation, causes defect       cassettes devised Akada et al. (2002). In cassettes, <br /> ncm5U modification U34&#226;&#8364;&#8482;s narrows decoding            auxotrophic selection marker accompanied neg- <br /> ability U34-containing tRNAs like tRNA-SerUGA, abol-          ative selection marker, GAL1pTGIN11M86, encodes <br /> ishes suppression ts(cga)D lethality tS(UGA)P         toxic protein control GAL1 promoter. <br />  <br />                                                                                                   www.rnajournal.org       1761 <br />  Mori et al. <br />  <br />  <br />  <br />                                                                                                  phic selection, consecutive plating on <br />                                                                                                  galactose medium allowed select <br />                                                                                                  clones lost marker cassette by <br />                                                                                                  homologous recombination the <br />                                                                                                  hisG sites. All recombination pro- <br />                                                                                                  cesses confirmed PCR. <br />                                                                                                     Initially, tried construct single <br />                                                                                                  intron-deletion mutants, of <br />                                                                                                  tRNA-TrpCCA loci converted to <br />                                                                                                  intronless version. All mu- <br />                                                                                                  tants viable growth de- <br />                                                                                                  fects normal laboratory conditions, <br />                                                                                                  indicating introns <br />                                                                                                  essential function itself. Then, <br />                                                                                                  performed sequential deletion the <br />                                                                                                  tRNA-TrpCCA introns yeast ge- <br />                                                                                                  nome. Six repetitive integration/marker <br />                                                                                                  rescue procedures (for tW(CCA)G1, <br />                                                                                                  tW(CCA)K, tW(CCA)M, tW(CCA)P, <br />                                                                                                  tW(CCA)G2, tW(CCA)J, this <br />                                                                                                  order), complementing mu- <br />                                                                                                  tations wild-type tRNA-TrpCCA <br />                                                                                                  gene plasmid, enabled achieve <br />                                                                                                  complete intron deletion six <br />                                                                                                  tRNA-TrpCCA genes yeast genome <br />                                                                                                  (see Fig. 2A&#226;&#8364;&#8220;D). <br />  <br />  <br />                                                                                                  All tRNA-TrpCCA introns be <br />                                                                                                  removed yeast genome <br /> FIGURE 1. tRNA-TrpCCA genes genome S. cerevisiae. (A) Localization tRNA-           To verify yeast transformants <br /> TrpCCA genes genome S. cerevisiae described schematically. Locus names tRNA-    lost introns tRNA-TrpCCA <br /> TrpCCA genes expressed according Saccharomyces Genome Database. White circles <br /> represent centromere loci. (B) Nucleotide sequences yeast tRNA-TrpCCA genes              genes expected, tested trans- <br /> compared Clustal X. Nucleotides corresponding 59- 39-flanking regions shown     formants basis criteria. <br /> in lower case. Positions nucleotides conserved genes        First, analyzed locus mod- <br /> marked asterisks, sequences intron enclosed squares. (C) Nucleotide  ified homologous recombination. The <br /> sequences tRNA-TrpCCA locus, tW(CCA)G2 S. cerevisiae, compared five <br /> yeast genomes Saccharomyces genus described B.                                 corresponding region yeast chro- <br />                                                                                                 mosome amplified PCR with <br />                                                                                                 &#226;&#8364;&#732;&#226;&#8364;&#732;primers a,&#226;&#8364;&#8482;&#226;&#8364;&#8482; shown Figure 2A, and <br /> The marker set sandwiched hisG sequences,                 compared wild-type strain <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> enable pop-out recombination integration               plasmid harboring tRNA gene intron (Fig. <br /> yeast genome. We added Camr marker make                     2A). By examining tRNA-TrpCCA loci yeast <br /> plasmid construction easy Escherichia coli. Since use               genome, confirmed integration process pro- <br /> GIN11M86 negative selection marker marker res-                   ceeded expected. We final trans- <br /> cue, use vector plasmid URA3 marker,                formants, sextuple intron-deletion clones, lost in- <br /> can used positive negative selection marker,            trons tRNA-TrpCCA loci. <br /> to complement defects derived tRNA-TrpCCA gene                          Second, asked unannotated tRNA-TrpCCA <br /> mutations necessary (Boeke et al. 1984). Then, using                   locus exons known tRNA-TrpCCA <br /> these modified marker cassettes, constructed series               intron S. cerevisiae genome exists, especially <br /> plasmids, contained intronless version                genome sextuple intron-deletion clones. To <br /> a tRNA-TrpCCA gene flanking region divided                 test possibility, PCR primers designed according <br /> marker cassette. A chromosomal tRNA-TrpCCA gene                       exons tRNA-TrpCCA (Fig. 2A, primers b), <br /> replaced corresponding intronless fragment                    chromosomal regions corresponding pre-tRNA- <br /> homologous recombination. Transformants tar-                     TrpCCA amplified wild-type series of <br /> geted tRNA-TrpCCA gene selected auxotro-                     intron-deletion strains (Fig. 2B). The amounts PCR <br />  <br /> 1762       RNA, Vol. 17, No. 9 <br />                                                                                               tRNA-Trp intron dispensable yeast <br />  <br />  <br />  <br />                                                                         fragment derived tRNA-TrpCCA genes the <br />                                                                         intron gradually decreased progression of <br />                                                                         intron removal. While considerable intron- <br />                                                                         containing DNA fragment observed PCR sample <br />                                                                         quintuple intron deletion clone (one wild-type gene <br />                                                                         plus 5 intronless genes), band mobility the <br />                                                                         intron-containing fragment seen sample the <br />                                                                         sextuple deletion clone. These results collectively indicate <br />                                                                         that, sextuple intron-deletion strain, tRNA-TrpCCA <br />                                                                         introns completely removed yeast genome. <br />                                                                            Third, tested tRNA-TrpCCA genes <br />                                                                         handled essential yeast, negate a <br />                                                                         possibility tRNA-TrpCCA genes unusual sequences <br />                                                                         compensate defects intronless tRNA-TrpCCA genes. We <br />                                                                         disrupted tW(CCA) genes replacing gene a <br />                                                                         different marker left marker tW(CCA) locus. <br />                                                                         Combination iterative deletions tetrad analyses re- <br />                                                                         vealed complete (sextuple) deletion mutant tRNA- <br />                                                                         TrpCCA lethal, mutants six <br />                                                                         tRNA-TrpCCA genes grow slowly (Fig. 2C, columns 2,3). <br />                                                                         Interestingly, growth defects varied different quin- <br />                                                                         tuple disruptants, probably reflecting difference tran- <br />                                                                         scription efficiency tRNA gene left chromosome. <br />                                                                         The growth defects quintuple sextuple disrup- <br />                                                                         tants complemented introducing pTYSC583, a <br />                                                                         URA3 plasmid tW(CCA)K, strain TYSC1406, <br />                                                                         sextuple disruptant pTYSC583, cured <br />                                                                         plasmid FOA medium (Fig. 2D, row 3), in- <br /> FIGURE 2. Confirmation intron deletion. (A) Deletion           dicating tRNA-TrpCCA gene required for <br /> intron tRNA-TrpCCA gene integration procedure          yeast viability, genes yeast genome cannot <br /> confirmed PCR agarose-gel electrophoresis. DNA fragments <br /> amplified probes marked &#226;&#8364;&#732;&#226;&#8364;&#732;a&#226;&#8364;&#8482;&#226;&#8364;&#8482; panel        substitute functionality tW(CCA) loci (Fig. 2D). <br /> analyzed 2.5% agarose gels. Analyzed tRNA-TrpCCA loci shown         Finally, asked intronless strain con- <br /> on left. (WT) Wild-type strain BY418, (1/6 Dint) single intron      structed harbors suppressor mutation(s) compen- <br /> deletion tW(CCA)G1 locus, (2/6 Dint) double intron deletion of <br />                                                                         sates growth defects caused intron deletion, using the <br /> tW(CCA)G1 tW(CCA)K loci, (3/6 Dint) triple intron deletion of <br /> tW(CCA)G1, tW(CCA)K, tW(CCA)M loci, (4/6 Dint) quadru-              strain. We crossed complete intronless strain, <br /> ple intron deletion tW(CCA)G1, tW(CCA)K, tW(CCA)M,               SMSC233 (a cell), TYSC1406 (a cell) obtain <br /> tW(CCA)P loci, (5/6 Dint) quintuple intron deletion tW(CCA)G1,       diploid strain [MATa/a tw(cca)g1-Dint/D tw(cca)g2-Dint/D <br /> tW(CCA)K, tW(CCA)M, tW(CCA)P, tW(CCA)G2 loci, (6/6 Dint) <br /> sextuple intron deletion tW(CCA)G1, tW(CCA)K, tW(CCA)M, <br />                                                                         tw(cca)j-Dint/D tw(cca)k-Dint/D tw(cca)m-Dint/D tw(cca)p- <br /> tW(CCA)P, tW(CCA)G2, tW(CCA)J loci, (P) plasmid harbors        Dint/D] pTYSC583. The growth diploid was <br /> an intronless tRNA-TrpCCA gene control. Assignment          essentially control heterozygote <br /> bands shown schematically right. (B) The samples    [W(CCA)G1/D tW(CCA)G2/D tW(CCA)J/D tW(CCA)K/D <br /> subjected PCR single set primers hybridized the <br /> tRNA-TrpCCA exons (primers &#226;&#8364;&#732;&#226;&#8364;&#732;b&#226;&#8364;&#8482;&#226;&#8364;&#8482; shown gray arrows). Amplified       tW(CCA)M/D tW(CCA)P/D] (Fig. 2D, rows 4,5). Since the <br /> fragments analyzed 10% native polyacrylamide gel. (C) A       SMSC233 TYSC1406 strains constructed indepen- <br /> diploid strain TYSC1361 [tw(cca)g1DT CgHIS3/- tw(cca)g2DT               dently, unlikely intronless strain viable be- <br /> LEU2/- tw(cca)jDTkanMX/+ tw(cca)kDTCgTRP1/+ tw(cca)mDT                  cause recessive suppressor mutation specifically <br /> hphMX/+ tw(cca)pDTADE2/+] sporulated, resulting <br /> tetrads dissected YPD (upper). A white circle indicates         cancels growth defects intron deletion. <br /> existence tiny colony it. The remaining tRNA-TrpCCA <br /> genes estimated marker phenotypes haploid descendant <br /> are shown lower part. Alphabets represent chromosomes        Functionality intronless tRNA-TrpCCA genes <br /> where tW(CCA) locus locates. (D) The complete intronless strain, <br /> SMSC233 (6/6Dint), control wild-type strain, BY418 (WT),    The fact construct yeast strain tRNA- <br /> crossed haploid strain TYSC1406 tRNA-TrpCCA          TrpCCA introns genome completely removed <br /> gene chromosomes pTYSC583 [tW(CCA)K 2m URA3]            indicates tRNA-TrpCCA intron does possess <br /> (6/6D). Resulting diploids (6/6D3WT, 6/6D36/6Dint) those <br />                                                                         essential role(s) yeast growth. To extend this <br /> parents pTYSC583 spotted FOA plate test <br /> whether strains lose URA3-plasmid tW(CCA)K.          observation, first, compared growth yeast strains <br /> Dilution rates shown panel.                               different numbers tRNA-TrpCCA genes converted to <br />  <br />                                                                                                          www.rnajournal.org      1763 <br />  Mori et al. <br />  <br />  <br />  <br /> the intronless version. We grew strains rich                 time point easily estimated PCR certain <br /> medium (YPD) 23&#194;&#176;C, 30&#194;&#176;C, 37&#194;&#176;C                 tW(CCA) locus. As shown Figure 4, quantitative analyses <br /> of strains including sextuple intronless strain showed        PCR products chip electrophoresis revealed that <br /> quite similar growth wild-type strain (Fig. 3A).         population intronless strain did decrease but <br />    Similar results observed strains                slightly increased 50 generations contin- <br /> grown synthetic media, media nonferment-                  uous growth independent experiments. These results, <br /> able carbon sources like glycerol lactate. The intronless           again, indicate complete removal tRNA-TrpCCA <br /> strains grew presence 1 M NaCl low pH                 intron does cause negative effects yeast growth under <br /> stress (pH 5.5) (Fig. 3B; data shown). We tested               usual laboratory conditions. <br /> sensitivity alkylating reagent, methyl methane-                 Next, analyzed expression maturation tRNA- <br /> sulfonate (MMS). It suggested unspliced pre-                   TrpCCA derived intronless genes vivo. After its <br /> tRNAs nucleus act signal counteract              end-maturation, intron-containing pre-tRNA-TrpCCA <br /> DNA damage (Ghavidel et al. 2007). We detect                  109 nt <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> converted mature form with <br /> any difference viability wild-type com-              length 75 nt splicing. When analyzed tRNA-TrpCCA <br /> plete intronless strains cells treated               Northern blotting, similar amounts tRNA-TrpCCA <br /> a sublethal concentration MMS (Fig. 3C). Therefore, com-             detected wild-type, quintuple, sextuple <br /> plete removal tRNA-TrpCCA introns does                intron-deletion strains (Fig. 5A, upper panel). This not <br /> strong impact yeast growth growth condi-                   expected distance internal promoter <br /> tions examined far.                                                  elements tRNA genes, box A box B, known to <br />    Then, examined growth intronless strain                affect transcriptional efficiency (Ciliberto et al. 1982; <br /> more quantitatively. First, measured growth rates             Fabrizio et al. 1987). The mobility tRNA-TrpCCA on <br /> wild-type intronless strains liquid YPD 30&#194;&#176;C.                 urea-PAGE alkaline buffer affected by <br /> Their doubling times 130 6 4 min 136 6 6 min,                  intron deletion, suggesting tRNA-TrpCCA these <br /> respectively. Because difference minute              strains received appropriate end-maturation probably <br /> the range standard deviations, tried detect            nucleotide modifications affecting electrophoretic mobility <br /> a possible difference growth competitive culturing          alkaline conditions. On hand, the <br /> in YPD. If difference growth rates,          samples hybridized probe the <br /> the strains dominant population long in-               tRNA-TrpCCA intron, weak apparent signals derived <br /> cubation co-culture. The population strains              primary transcript, 59-processed, end-ma- <br />                                                                         tured pre-tRNA-TrpCCA observed lane the <br />                                                                         quintuple intron-deletion strain, signal seen <br />                                                                         lane sextuple deletion strain (Fig. 5A, middle <br />                                                                         panel). These results indicate intron contain- <br />                                                                         ing pre-tRNA-TrpCCA transcribed sextuple intron- <br />                                                                         deletion strain and, addition, tRNA-TrpCCA is <br />                                                                         transcribed end-matured intronless genes as <br />                                                                         wild-type genes vivo. <br />                                                                            If tRNA-TrpCCA intronless genes completes its <br />                                                                         maturation processes, aminoacylated Trp- <br />                                                                         RS. We measured tRNA-TrpCCA the <br />                                                                         intron deletion strains charged using acidic gel. We <br />                                                                         prepared total tRNAs strains shown above <br />                                                                         acidic conditions (pH 4.5) protect aminoacyl <br />                                                                         group conjugated 39 end tRNAs hydrolysis. <br />                                                                         A portion RNA samples treated alkaline for <br />                                                                         deacylation, alkaline-treated nontreated <br />                                                                         samples subjected Northern blotting. Large parts <br /> FIGURE 3. Growth tRNA-TrpCCA intronless yeasts. (A) Growth        tRNA-TrpCCA quintuple sextuple intron- <br /> tRNA-TrpCCA intronless strains YPD compared         deletion strains wild-type strain <br /> WT (BY418). The plates cultured 23&#194;&#176;C, 30&#194;&#176;C, 37&#194;&#176;C. The <br /> yeast strain streaked sector shown lower right. (B)   migrated slowly absence alkaline treatment <br /> Growth wild-type (WT) intronless (6/6Dint) strains       treated alkaline, indicating the <br /> compared conditions described left serial       tRNA-TrpCCA aminoacylated (Fig. 5B). Levels of <br /> dilution. Cells cultured 30&#194;&#176;C 3 d YPD SD 4 <br />                                                                         aminoacylation intronless strains indistinguishable <br /> d YPGly YPD 1 M NaCl. (C) Viability wild-type <br /> and intronless cells treated 0.12% MMS 80 min    wild-type strain. Therefore, tRNA-TrpCCA <br /> 30&#194;&#176;C (Ghavidel et al. 2007) analyzed serial dilution YPD.     intronless genes properly aminoacylated vivo. <br />  <br /> 1764       RNA, Vol. 17, No. 9 <br />                                                                                                 tRNA-Trp intron dispensable yeast <br />  <br />  <br />  <br />                                                                         protein compositions wild-type sextuple <br />                                                                         intron-deletion strains 2D gel electrophoresis. Protein <br />                                                                         extracts prepared log phase cells subjected iso- <br />                                                                         electric focusing (IEF) pH gradient 4 7 and <br />                                                                         SDS-PAGE 12% polyacrylamide gel (Fig. 6C). <br />                                                                         Comparison 500 spots gels showed most <br />                                                                         proteins migrated similar positions similar <br />                                                                         amounts. On hand, noticed signal <br />                                                                         intensities limited numbers spots apparently dif- <br />                                                                         ferent strains. Some examples shown <br />                                                                         pseudocolor expression the gel (Fig. 6C, <br />                                                                         right-most panel, circled spots). Taken together, con- <br />                                                                         clude complete removal tRNA-TrpCCA introns <br />                                                                         yeast genome little, any, effect growth <br />                                                                         bulk translation S. cerevisiae, causes minor <br />                                                                         effects yeast protein composition. <br />  <br />                                                                         DISCUSSION <br />                                                                         Is tRNA intron, removed during <br /> FIGURE 4. Competitive growth wild-type tRNA-            maturation, essential eukaryotic cells? The simplest <br /> TrpCCA intronless yeasts. The wild-type (WT) tRNA-TrpCCA <br />                                                                         way answer question examine or- <br /> intronless (Dint) yeast cells co-cultured liquid YPD 30&#194;&#176;C. <br /> Six independent cultures (a&#226;&#8364;&#8220;f) initiated 1:1 mixture     ganism tRNA introns removed genome <br /> these strains, logarithmical growth kept periodic      genetically constructed. However, difficult to <br /> dilution fresh medium. Samples taken time point     achieve experiment eukaryotes mul- <br /> subjected PCR amplification tW(CCA)K locus analyzed <br /> by chip electrophoresis. (A) Electropherograms samples from <br />                                                                         tiple isoacceptor tRNAs encoded multiple intron-containing <br /> the inoculants 50th generation shown     genes. Therefore, we, instead, asked intron a <br /> the wild-type (WT) intronless strains (Dint). Band assign-      certain isoacceptor tRNA essential life a <br /> ment shown left. (B) DNA amounts PCR products         eukaryote, S. cerevisiae. To analyze subject, developed <br /> the wild-type tW(CCA) tw(cca)kDint alleles culture <br /> were quantified electropherograms, ratios,           systematic protocol replace multiple tRNA genes the <br /> [tw(cca)kDint] / ([tw(cca)kDint] + [tW(CCA)K]), plotted             yeast chromosomes intronless version. Although <br /> against culture length. <br />  <br />  <br />  <br />    Finally, examined tryptophanylated tRNA- <br /> TrpCCA intronless genes normally acts translation. <br /> We examined rates bulk protein synthesis by <br /> measuring incorporation 35S-labeled amino acids (Met/ <br /> Cys) trichloroacetic acid (TCA)-insoluble fraction. <br /> We observe difference incorporation <br /> among wild-type, quintuple, sextuple intron-deletion <br /> strains shown Figure 6A, suggesting tryptopha- <br /> nylated tRNA-TrpCCA derived intronless genes is <br /> utilized bulk translation, intron- <br /> containing genes vivo. Second, analyzed effect of <br />                                                                         FIGURE 5. tRNA-TrpCCA intronless genes matured <br /> intron removal individual protein synthesis analyzing             aminoacylated efficiently wild-type genes. (A) <br /> the protein composition. First, cell extracts prepared             Total RNAs extracted strain shown the <br /> the wild-type, quintuple, sextuple intron-deletion cells            panel, 3 mg aliquots subjected Northern blotting with <br /> in log stationary phases subjected SDS-               probes shown left. In middle panel, bands corresponding to <br />                                                                         primary transcript, 59-processed precursor, end-matured <br /> PAGE 8%&#226;&#8364;&#8220;14% polyacrylamide gradient gel.                        precursor marked closed, gray, open triangles, <br /> Essentially, protein patterns observed             respectively. (B) Total RNAs prepared indicated strains <br /> extracts strains (Fig. 6B), suggesting              acidic conditions retain aminoacyl groups 39 end of <br />                                                                         tRNAs. Samples (3 mg lane) pre-treated (+ lanes) or <br /> there essential difference protein compositions of <br />                                                                         (&#226;&#8364;&#8220; lanes) alkaline deacylation, subjected to <br /> these yeast cells log stationary culture conditions            Northern blotting. Closed open triangles represent positions <br /> with resolution. We analyzed difference             aminoacylated deacylated tRNA species, respectively. <br />  <br />  <br />                                                                                                             www.rnajournal.org         1765 <br />  Mori et al. <br />  <br />  <br />  <br />                                                                                                 intronless yeast grows normally un- <br />                                                                                                 der various conditions, including those <br />                                                                                                 damaging chromosomal DNA, recovery <br />                                                                                                 correlates nuclear <br />                                                                                                 accumulation certain intron-contain- <br />                                                                                                 ing pre-tRNAs (Ghavidel et al. 2007). <br />                                                                                                 We detect difference in <br />                                                                                                 35 <br />                                                                                                    S-amino acids incorporation pro- <br />                                                                                                 tein fractions wild-type and <br />                                                                                                 completely intronless cells. The expres- <br />                                                                                                 sion level aminoacylation efficiency <br />                                                                                                 tRNA-Trp CCA derived the <br />                                                                                                 intronless genes comparable to <br />                                                                                                 wild-type genes vivo, <br />                                                                                                 suggesting tRNA-TrpCCA the <br />                                                                                                 intronless genes undergoes normal <br />                                                                                                 aminoacylation/peptidyl transfer cycle in <br />                                                                                                 addition normal maturation. These <br />                                                                                                 results indicate intron tRNA- <br />                                                                                                 TrpCCA dispensable growth and <br />                                                                                                 bulk translation S. cerevisiae under <br /> FIGURE 6. Complete removal tRNA-TrpCCA intron yeast genome does laboratory conditions. The effects in- <br /> a big impact bulk translation protein composition. (A) Protein synthesis measured tron removal analyzed by <br /> as 35S-methionine/cysteine incorporation TCA-insoluble fraction. Averages three <br /> independent experiments shown standard deviations. (B) Major protein contents comparing protein composition SDS- <br /> the wild-type (wt), quintuple (5/6 Dint), sextuple (6/6 Dint) intron deletion strains PAGE/2D gel analyses. Again, found <br /> compared SDS-PAGE/Coomassie Brilliant Blue staining. SDS-extracts overall protein composition not <br /> strains log stationary phases analyzed 8%&#226;&#8364;&#8220;14% polyacrylamide gel. For affected significantly complete re- <br /> log phase cell extracts, 0.10 OD660 equivalents yeast cells loaded, 0.20 OD660 <br /> equivalents loaded stationary cell extracts. (C) Protein compositions wild-type moval tRNA-TrpCCA intron. How- <br /> cells (wt) sextuple intron deletion cells (6/6 Dint) compared 2D electrophoresis. ever, expression proteins al- <br /> Fifty mg total proteins analyzed IEF (pH 4&#226;&#8364;&#8220;7) subjected 12% SDS- tered intron removal (Fig. 6C). <br /> polyacrylamide gel. A gel enclosed dashed rectangle panel merged <br />                                                                                                 Identity proteins interest <br /> in right-most panel (merged). In panel, contrast reversed, signals were <br /> expressed pseudocolor: (green) wild-type, (red) sextuple intron deletion.                  reveal kind the <br />                                                                                                 codon sequences suscepti- <br />                                                                                                 ble decoding tRNA-TrpCCA. <br /> removal intron certain tRNA gene                             Another surprise yeast cells intronless <br /> achieved previously yeast, applied                  tRNA genes maintain similar tRNA-TrpCCA <br /> a minor Ser isoacceptor, encoded single gene.                 wild-type genes. It known promoter <br /> In report, removed introns                  tRNA gene RNA polymerase III consists box A <br /> tRNA-TrpCCA genes, gene product                           box B, located tRNA coding region <br /> isoacceptor tRNA decoding Trp codon, model case                    (Hall et al. 1982; Marck et al. 2006). The distance between <br /> to analyze essentiality tRNA introns yeast                elements affects transcription efficiency, that <br /> genome. We construct yeast strain                     removal intron alters expression levels some <br /> intronless tRNA-TrpCCA genes genome                        tRNAs Xenopus oocytes vivo yeast extracts in <br /> demonstrate feasibility procedure modify heavily                  vitro (Ciliberto et al. 1982; Fabrizio et al. 1987). Similarly, <br /> degenerated genes iteratively. We confirmed complete                    intron removal SUP53/tL(CAA)C gene, encoding <br /> loss tRNA-TrpCCA intron (1) PCR analysis                   tRNA-LCAA, yeast renders gene occupied by <br /> the annotated tRNA-TrpCCA loci, (2) PCR analysis set                 TFIIIC vivo (Donze Kamakaka 2001). Furthermore, <br /> of primers amplifying tRNA segment encompassing                       effect TFIIIC occupancy tRNA locus has <br /> anticodon loop, (3) Northern blotting probe                      big impact blocking expansion heterochromatin <br /> against known tRNA-TrpCCA intron.                                       tRNA gene. However, present study showed <br />    One big surprises intronless yeast cells              intron removal does affect tRNA- <br /> seem grow translate proteins                     TrpCCA despite fact tRNA-TrpCCA genes a <br /> problems. The growth intronless strain in-                       longer (34 bp) intron SUP53 (32 bp) neither <br /> distinguishable wild-type strain                   box A- box B-like sequences tRNA- <br /> the co-culture analyses rich medium, YPD. The                          TrpCCA intron. This suggests boxes A B of <br />  <br /> 1766       RNA, Vol. 17, No. 9 <br />                                                                                                tRNA-Trp intron dispensable yeast <br />  <br />  <br />  <br /> tRNA-TrpCCA sufficient affin- <br /> ity TFIII proteins, irrespective the <br /> distance elements, to <br /> assemble transcription initiation <br /> complex RNA polymerase III. Alter- <br /> natively, mechanisms level <br /> of transcriptional elongation in <br /> stability control tRNA com- <br /> pensate reduction promoter activity <br /> of intronless tRNA-TrpCCA genes. <br />    The results great contrast <br /> to fact S. cerevisiae maintains <br /> the intron tRNA-TrpCCA genes <br /> at position same <br /> sequence (Fig. 1B). The sequences the <br /> tRNA-TrpCCA introns conserved <br /> among yeasts Saccharomyces <br /> genus compared 59- 39- <br /> flanking regions tRNA gene (Fig. <br /> 1C). On hand, tRNA- FIGURE 7. Comparison tRNA-TrpCCA genes moderately related yeast genomes. <br />                                                  Sequences tRNA-Trp         genes moderately related yeast genomes annotated in <br /> TrpCCA genes distantly re- Genomic tRNA DatabaseCCA            (http://lowelab.ucsc.edu/GtRNAdb/) compared, results <br /> lated yeasts, intron sequences are expressed phylogram. The lower right inset shows phylogenetic relation six <br /> more conserved species yeast species according Fischer et al. (2006). Clusters tRNA genes genome <br /> when compared shadowed. (S. cer) Saccharomyces cerevisiae, (Y. lip) Yarrowia lipolytica, (D. han) <br />                                                  Debaryomyces hansenii, (C. gla) Candida glabrata, (E. gos) Eremothecium gossypii, (K. lac) <br /> species (Fig. 7). When parts genomic Kluyveromyces lactis. <br /> sequences including tRNA-TrpCCA on <br /> six moderately related yeast genomes <br /> are analyzed Clustal X, tRNA genes             MATERIALS AND METHODS <br /> genome clustered. Among these, Debaryomyces hansenii <br />                                                                         Yeast strains, primers, plasmids <br /> harbors tRNA-TrpCCA genes intron, while <br /> more distantly related yeast Yarrowia lipolytica harbors                Yeast genetic techniques essentially described Guthrie and <br /> intron-containing tRNA-TrpCCA genes. These facts suggest                Fink (1991), molecular biological techniques in <br /> that organisms avoid having intron-containing intron-               Sambrook Russell (2001). Yeast strains used constructed <br /> less genes isoacceptor tRNA simultaneously               study summarized Supplemental Table SI. Primers <br /> genomes. If certain gene isoacceptor tRNA acquires              plasmids Supplemental Tables SII SIII, respectively. <br />                                                                         To construct pTYE379 pTYE380, 4.95 kb BglII-NotI frag- <br /> an intron, sequence spreads genome, and <br />                                                                         ment pGG216 6.03 kb BglII-NotI fragment from <br /> maintenance intron genome does to <br />                                                                         pGG217 (Akada et al. 2002) inserted BamHI- <br /> rely functionality tRNA translation. Alterna-              PspOMI sites pDNR-1r (Clontech) derivative modified <br /> tively, possible minor differences               multicloning site, respectively. <br /> translational activity tRNA-TrpCCA intron- <br /> containing genes intronless genes, like the <br />                                                                         Construction tRNA-TrpCCA intronless yeasts <br /> difference 2D gel analyses (Fig. 6C), swings the <br />                                                                         growth analysis <br /> balance reach extreme cases. <br />    tRNA-TrpCCA S. cerevisiae second                  A chromosomal region certain tRNA-TrpCCA gene its <br /> strict case tRNA intron dispensable                 flanking region amplified PCR separately ExTaq <br /> growth eukaryotic cells. The final answer obtained           (Takara Bio) cloned vector, pTYE374, derivative of <br /> by extending analysis rest isoacceptor                pBluescript SK(-), loxP site. The amplified regions are <br /> tRNAs S. cerevisiae. It early conclude      summarized Supplemental Table SIV. To make intronless <br />                                                                         version tRNA-TrpCCA gene, applied overlap extension <br /> the tRNA introns dispensable any <br />                                                                         mutagenesis set primers, tRNA-TrpCCA gene <br /> major roles tRNA maturation functionality. How- <br />                                                                         specific forward reverse primers, tRNA-TrpCCA exon <br /> ever, results lead new idea; tRNA intron            primers (tWccaG2_ex1_3 tWccaG2_ex2_5) provided <br /> maintained yeast genome essen-                joint exons. A marker cassette derived pTYE379 <br /> tiality tRNA function translation,             pTYE380, harbors auxotrophic marker (URA3 ADE2) <br /> some cellular mechanisms, maintenance                  negative marker (GAL1pTGIN11M86) sandwiched two <br /> genome integrity, chromatin organization, others.                    hisG fragments (Akada et al. 2002), Camr, introduced <br />  <br />                                                                                                            www.rnajournal.org       1767 <br />  Mori et al. <br />  <br />  <br />  <br /> between tRNA-TrpCCA gene flanking region. All           pH 7.5, 1 mM EDTA] run 7.0 M urea/10% w/v <br /> sequences PCR fragments confirmed cloning.            polyacrylamide gel TBE, RNAs dissolved <br />    For integration intronless tRNA-TrpCCA gene yeast      AE [10 mM sodium acetate, pH 4.5, 1.0 mM EDTA] run on <br /> genome, plasmid intronless gene digested          10 M urea/10% w/v polyacrylamide gel 0.10 M NaOAc, pH 5.0. <br /> appropriate restriction enzymes, resulting DNA fragments        Deacylation tRNAs performed 50 mM Tris-HCl, pH 9.0 at <br /> were introduced appropriate yeast strain. Transformants      37&#194;&#176;C 1 h (Steiner-Mosonyi et al. 2003). After transfer charged <br /> an auxotrophic marker selected SCD medium [0.67%             nylon membranes (Hybond N+, GE Healthcare Biosciences), <br /> w/v Difco Yeast Nitrogen Base Amino Acids (BD Bio-              particular RNA decorated appropriate digoxigenin- <br /> sciences), 0.5% w/v Vitamin Assay Casamino Acids (BD Bio-               labeled oligonucleotide probe (Roche Diagnostics), signal <br /> sciences), 2% w/v glucose] appropriate supplements.            developed ECF (GE Healthcare Biosciences) read with <br /> Counter selection obtain clones lost marker cassette        Storm 860 Image Analyzer (GE Healthcare Biosciences). <br /> was performed YPGal 2% w/v galactose induce the <br /> toxic GIN11M86 gene. The set procedures repeated           Protein analysis <br /> six times remove introns tRNA-TrpCCA genes in <br /> the yeast strain BY418. Integration DNA fragment          Two-dimensional gel electrophoresis performed IEF as <br /> corresponding chromosomal region confirmed PCR               dimension SDS-PAGE second dimension. To <br /> electrophoresis 2.5% w/v agarose gel.                              prepare protein samples IEF, yeast cells suspended 10 mM <br />    For analyzing population complete intronless cells         Tris-HCl, pH 8.0, 1 mM EDTA treated final 0.28 M <br /> competitive growth wild-type cells, strains         NaOH 6.2% v/v 2-mercaptoethanol ice 4 min, and <br /> inoculated YPD 1:1 mixture incubated 30&#194;&#176;C. When          proteins precipitated adding 10 volumes ice-cold 10% <br /> OD660 reached 0.4, culture brought 128-fold        w/v TCA. After collecting precipitates centrifugation, pellets <br /> dilution fresh YPD allow continuous logarithmic growth,           washed ice-cold acetone dissolved IEF <br /> and rest culture incubated 12 h achieve   Sample Buffer [7.0 M urea, 2.0 M thiourea, 4% w/v CHAPS, 2% v/v <br /> saturation growth. These procedures repeated seven times.       IPG Buffer, pH 4.0-7.0, 0.12% v/v DeStreak Reagent (GE Health- <br /> DNA samples prepared saturation cultures         care)]. The samples analyzed Immobiline Dry Strip, pH <br /> subjected PCR above. Amounts PCR products derived         4.0&#226;&#8364;&#8220;7.0 IPGphor Electrophoresis System (GE Healthcare) <br /> the wild-type intronless alleles tW(CCA)K tW(CCA)G1           according manufacturer&#226;&#8364;&#8482;s instructions, developed strips <br /> loci measured chip electrophoresis MultiNA             subjected SDS-PAGE 12% polyacrylamide gels. Gels <br /> Microchip Electrophoresis System (Shimadzu).                            stained SYPRO Red (Sigma-Aldrich) analyzed with <br />                                                                         Typhoon 9400 Image Analyzer (GE Healthcare). <br /> Disruption tRNA-TrpCCA genes                                         35 <br />                                                                              S-amino acid-labeling monitor protein synthesis <br /> Disruption tRNA-TrpCCA genes performed PCR- <br />                                                                         Protein synthesis monitored measuring 35S methionine/ <br /> based method using pCgHIS3 (Candida glabrata HIS3), pCgTRP1 <br />                                                                         cysteine incorporation TCA insoluble fraction. Briefly, 100 <br /> (C. glabrata TRP1), pBS-kanMX (kanMX4), pBS-hygMX <br />                                                                         OD600 cells cultured SD medium prepared Difco Yeast <br /> (hphMX), pJJ282 (LEU2) pTYE512 (ADE2) templates. <br />                                                                         Nitrogen Base Amino Acids Ammonium Sulfate (BD <br /> First, constructed double disruption strains TYSC1281 [MATa <br />                                                                         Biosciences) supplemented appropriate sulfate-free nutri- <br /> tw(cca)g1DTCgHIS3 tw(cca)g2DTLEU2] based YCB436 and <br />                                                                         ents labeled 2.0 MBq Tran35S-<span id='am-3' about='rdfs:label' typeof='owl:Thing'>Label</span> (MP Biomedicals) <br /> TYSC1283 [MATa tw(cca)jDTkanMX tw(cca)kDTCgTRP1] based <br />                                                                         indicated times. Three 200-ml aliquots harvested in <br /> on BY418, crossed strains dissected resulting diploid <br />                                                                         1.0 mL 15% w/v TCA, heated 94&#194;&#176;C 5 min, insoluble <br /> to obtain MATa strain quadruple disruption TYSC1345. We <br />                                                                         materials recovered GF-75 glass filter (Advantec). After <br /> also constructed MATa quadruple disruption TYSC1349 <br />                                                                         sequential washing 15% w/v TCA ethanol, filter was <br /> [tw(cca)g1DTCgHIS3 tw(cca)g2DTLEU2 tw(cca)mDThphMX <br />                                                                         dried, radioactivity trapped filter measured <br /> tw(cca)pDTADE2] TYSC1281 iterative disruption. These <br />                                                                         liquid scintillation counter, LSC-5100 (Aloka). <br /> two haploids crossed obtain diploid TYSC1361 <br /> [tw(cca)g1DTCgHIS3/D tw(cca)g2DTLEU/D tw(cca)jDTkanMX/+ <br /> tw(cca)kDTCgTRP1/+ tw(cca)mDThphMX/+ tw(cca)pDTADE2/+].                 SUPPLEMENTAL MATERIAL <br /> For complementing chromosomal disruption tRNA-TrpCCA genes, <br /> we constructed pTYSC583, wild-type tW(CCA)K gene           Supplemental material available article. <br /> was inserted pYO326, 2m plasmid URA3 marker. <br /> By introducing pTYSC583 TYSC1361 performing the <br />                                                                         ACKNOWLEDGMENTS <br /> tetrad analysis, obtained TYSC1406, sextuple tW(CCA)- <br /> deletion strain pTYSC583.                                          We thank present previous members laboratory for <br />                                                                         help suggestions, especially Ms. Masako Fujishiro constructing <br /> RNA analysis                                                            plasmids. This work supported Grant-in-aid Sci- <br />                                                                         entific Research Ministry Education, Culture, Sports, Sci- <br /> RNA extractions yeast guanidine thiocyanate buffer          ence, Technology Japan, Takeda Science Foundation, <br /> and acidic conditions performed previously          PRESTO Japan Science Technology Agency. <br /> described Yoshihisa et al. (2003) Sarkar et al. (1999), <br /> respectively. The RNAs kept TE [10 mM Tris-HCl,          Received May 31, 2011; accepted June 20, 2011. <br />  <br /> 1768       RNA, Vol. 17, No. 9 <br />                                                                                                       tRNA-Trp intron dispensable yeast <br />  <br />  <br />  <br /> REFERENCES                                                                  Johansson MJO, Esberg A, Huang B, Bjo&#194;&#168;rk GR, Bystro&#194;&#168;m AS. 2008. <br />                                                                                 Eukaryotic wobble uridine modifications promote functionally <br /> Abelson J, Trotta CR, Li H. 1998. tRNA splicing. J Biol Chem 273:               redundant decoding system. Mol Cell Biol 28: 3301&#226;&#8364;&#8220;3312. <br />     12685&#226;&#8364;&#8220;12688.                                                            Lowe TM, Chan P. 2010. GtRDB: The genomic tRNA database. http:// <br /> Agris PF, Vendeix FAP, Graham WD. 2007. tRNA&#226;&#8364;&#8482;s wobble decoding                  lowelab.ucsc.edu/GtRNAdb/. <br />     genome: 40 years modification. J Mol Biol 366: 1&#226;&#8364;&#8220;13.          Marck C, Kachouri-Lafond R, Lafontaine I, Westhof E, Dujon B, <br /> Akada R, Hirosawa I, Kawahata M, Hoshida H, Nishizawa Y. 2002.                  Grosjean H. 2006. The RNA polymerase III-dependent family of <br />     Sets integrating plasmids gene disruption cassettes con-             genes hemiascomycetes: comparative RNomics, decoding strate- <br />     taining improved counter-selection markers designed repeated            gies, transcription evolutionary implications. Nucleic Acids Res <br />     use budding yeast. Yeast 19: 393&#226;&#8364;&#8220;402.                                    34: 1816&#226;&#8364;&#8220;1835. <br /> Atkin AL, Henry RW, Roy KL, Bell JB. 1992. Characterization          Motorin Y, Grosjean H. 1999. Multisite-specific tRNA:m5C-methyl- <br />     tRNATrp genes Saccharomyces cerevisiae. Gene 119: 57&#226;&#8364;&#8220;63.                 transferase (Trm4) yeast Saccharomyces cerevisiae: Identification of <br /> Boeke JD, LaCroute F, Fink GR. 1984. A positive selection mutants           gene substrate specificity enzyme. RNA 5: 1105&#226;&#8364;&#8220;1118. <br />     lacking orotidine-59-phosphate decarboxylase activity yeast:         Paushkin SV, Patel M, Furia BS, Peltz SW, Trotta CR. 2004. Identifi- <br />     5-fluoro-orotic acid resistance. Mol Gen Genet 197: 345&#226;&#8364;&#8220;346.                cation human endonuclease complex reveals link between <br /> Ciliberto G, Traboni C, Cortese R. 1982. Relationship           tRNA splicing pre-mRNA 39 end <br /> </body></html>