<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Structural insight into the functional mechanism <br /> of Nep1/Emg1 N1-specific pseudouridine <br /> methyltransferase in ribosome biogenesis <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        INTRODUCTION <br /> Nucleolar Essential Protein 1 (Nep1) is required for                            Ribosome biogenesis is a complex stepwise process that <br /> small subunit (SSU) ribosomal RNA (rRNA) matur-                                 begins with the transcription of ribosomal RNA (rRNA), <br /> ation and is mutated in Bowen&#226;&#8364;&#8220;Conradi Syndrome.                                 and continues with a coordinated processing pathway by <br /> Although yeast (Saccharomyces cerevisiae) Nep1                                  which the rRNA is processed and ribosomal proteins are <br />                                                                                 assembled [reviewed in (1&#226;&#8364;&#8220;3)]. In Saccharomyces cerevisiae <br /> interacts with a consensus sequence found in <br />                                                                                 (yeast), a single 35S rRNA transcript is cleaved at several <br /> three regions of SSU rRNA, the molecular details                                sites, and then further processed by exo- and endonucle- <br /> of the interaction are unknown. Nep1 is a SPOUT                                 ases <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> produce the mature 18S, 25S and 5.8S rRNAs (2). <br /> RNA methyltransferase, and can catalyze methyla-                                Ribosomal RNA processing and ribosome assembly <br /> tion at the N1 of pseudouridine. Nep1 is also                                   require several non-ribosomal protein factors, as well as <br /> involved in assembly of Rps19, an SSU ribosomal                                 small nucleolar RNA (snoRNA) molecules (2,4). These <br /> protein. Mutations in Nep1 that result in decreased                             factors not only guide cleavage steps, but also carry out <br /> methyl donor binding do not result in lethality, sug-                           site-speci&#239;&#172;&#65533;c rRNA modi&#239;&#172;&#65533;cations that include methylation <br /> gesting that enzymatic activity may not be required                             and pseudouridylation (2). <br /> for function, and RNA binding may play a more im-                                  Nucleolar essential protein 1 (Nep1) is a highly <br /> portant role. To study these interactions, the crystal                          conserved protein required for ribosome biogenesis and <br />                                                                                 found in organisms <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> archaea to humans. Nep1 was <br /> structures of the scNep1 dimer and its complexes <br />                                                                                 &#239;&#172;&#65533;rst identi&#239;&#172;&#65533;ed in &#239;&#172;&#65533;ssion yeast (Schizosaccharomyces <br /> with RNA were determined. The results demonstrate                               pombe) as a high-copy number suppressor of a mating <br /> that Nep1 recognizes its RNA site via <span id='am-164' about='xsp:base' typeof='owl:Thing'>base</span>-specific                             de&#239;&#172;&#65533;ciency caused by a <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>mutation</span> in the effector domain <br /> interactions and stabilizes a stem-loop in the bound                            of Ras1 and thus named Mra1 (multicopy suppressor of <br /> RNA. Furthermore, the RNA structure observed                                    Ras 1) (5). It has also been designated the name Emg1 for <br /> contradicts the predicted structures of the                                     essential for mitotic growth 1 (6). In yeast, Nep1 is <br /> Nep1-binding sites within mature rRNA, suggesting                               localized to the nucleolus, interacts with a known nucle- <br /> that the Nep1 changes rRNA structure upon binding.                              olar protein, Nop14, and is required for 18S rRNA mat- <br /> Finally, a uridine base is bound in the active site of                          uration (6,7). To release the yeast 18S SSU rRNA, <br /> Nep1, positioned for a methyltransfer at the C5                                 cleavage must occur at four sites, designated sites A0, <br /> position, supporting its role as an N1-specific                                 A1, A2 and D (1,2). Cleavage at site A0 and A1 truncates <br />                                                                                 the 50 -end of the rRNA transcript, while cleavage at A2 <br /> pseudouridine methyltransferase. <br />                                                                                 results in separation of a 20S pre-rRNA species. Final <br />  <br />  <br />  <br />  <br /> *To whom correspondence should be addressed. Tel: +3014050462; Fax: +3013140386; Email: nlaronde@umd.edu <br /> Present address: <br /> Agnieszka Szyk, Cell Biology and Biophysics <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, NINDS Porter Neuroscience Research Center Building 35, Room 3B-203, 35 Convent Drive, <br /> MSC 3700, Bethesda, MD 20892-3700, USA. <br />  <br /> &#195;&#376; The Author(s) 2010. Published by Oxford University Press. <br /> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. <br />  2446 Nucleic Acids Research, 2011, Vol. 39, No. 6 <br />  <br />  <br /> cleavage at site D produces the mature 18S SSU rRNA.              RNA-binding screens have indicated that Nep1 binds to <br /> Depletion of yeast Nep1 (scNep1) results in loss of            the consensus sequence C/UUCAAC (23). This sequence <br /> cleavage at site A2, which results in an accumulation of       is found in three regions of 18S rRNA in yeast including <br /> a 21S SSU rRNA species (7). This is a result of cleavage at    the stem loop of helix 31, bases 1188&#226;&#8364;&#8220;1193, and part of <br /> site A3, which normally happens after site A2 cleavage in      the stem of helix 42, bases 1565&#226;&#8364;&#8220;1570. In the RNA <br /> the maturation of the 50 -end of the 5.8S rRNA species (1).    three-hybrid screen that identi&#239;&#172;&#65533;ed this consensus <br /> Nep1 has also been identi&#239;&#172;&#65533;ed as the human gene mutated         sequence, all the binding sites were located in the <br /> in Bowen&#226;&#8364;&#8220;Conradi syndrome, a lethal genetic disease            unpaired region of a stem loop structure predicted by <br /> that results in low birth weight, small head, joint            RNA secondary structure predictions (23). Both these <br /> deformities and failure to thrive (8&#226;&#8364;&#8220;10). The mutation         binding sites are near helix 41, where Rps19 was <br /> results in a switch of D86 to glycine in the human Nep1        shown to bind using cryo-EM reconstructions (23,25). <br /> protein (8).                                                   In addition, one of the sites, between bases 1565 and <br />     Prior structural work on Nep1 includes the X-ray           1570, overlaps with the predicted binding site of snR57 <br /> crystal structures and backbone nuclear magnetic               (bases 1569&#226;&#8364;&#8220;1583). <br /> resonance (NMR) assignments of the archaeal Nep1                  Recent work has identi&#239;&#172;&#65533;ed Nep1 as a pseudouridine <br /> dimer from Methanocaldococcus jannaschii and a mono-           N1-methyltransferase (14). Wurm et al. (14) showed that <br /> meric structure of scNep1 (11&#226;&#8364;&#8220;13). Those structural            both human and M. jannaschii Nep1 are capable of <br /> analyses showed that Nep1 is structurally homologous           catalyzing methylation on pseudouridine-containing <br /> to SPOUT RNA methyltransferases, provided a muta-              RNA sequences, especially C/U	CAAC. This sequence <br /> tional analysis of Nep1 and allowed NMR mapping of             is the Nep1 site found in the stem loop of helix 31 of <br /> some of the residues that interact with RNA (11,13,14).        yeast 18S rRNA mentioned above (14). This <br /> SPOUT methyltransferases are a group of proteins with          pseuoduridine, located at position 1189, is hypermodi&#239;&#172;&#65533;ed <br /> an S-adenosylmethionine (SAM)-binding knotted                  in yeast, with a 3-amino-3-carboxypropyl group on the N3 <br /> methyltransferase fold originally de&#239;&#172;&#65533;ned to contain the        position and a methyl group on the N1 position <br /> SpoU and TrmD methyltransferases (15). This family             (m1acp3	) (26). Nep1 is the &#239;&#172;&#65533;rst identi&#239;&#172;&#65533;ed example of <br /> now includes a large number of methyltransferases con-         an N1-speci&#239;&#172;&#65533;c pseudouridine methyltransferase (14). <br /> taining similar knotted folds (16). The members of the            Given that it plays an essential, though yet unclear, role <br /> family each contain a SPOUT methyltransferase domain           in maturation of the 18S rRNA, we have determined the <br /> and may contain additional inserted domains required for       X-ray crystal structures of dimeric Nep1 from scNep1 <br /> selectivity of RNA substrates. Thus far, SPOUT                 and Archaeoglobis fulgidus (afNep1) at 1.85 and 1.45 A&#203;&#353;, <br /> methyltransferases have been implicated in the methyla-        respectively, bound to S-adenosyl homocysteine (SAH), <br /> tion of ribose 20 -OH (20 -O-ribose), guanine N1 (m1G),        and scNep1 bound to one molecule of cognate RNA at <br /> uridine C3 (m3U) and N3 of pseudouridine (m3	)                 1.90 A&#203;&#353;, and two molecules of cognate RNA at 3.00 A&#203;&#353;. The <br /> (16,17). Within the family, Nep1 proteins are most             structures of Nep1 with RNA have revealed recognition of <br /> closely related to the TrmD and RsmE subfamilies,              a base &#239;&#172;&#8218;ipped uridine in the active site of the enzyme, and <br /> which catalyze m1G and m3U modi&#239;&#172;&#65533;cations, respectively          a hairpin loop in the RNA-binding site. The RNA bound <br /> (16). All SPOUT methyltransferases characterized thus far      in the active site contains the consensus sequence UUCAA <br /> are dimeric, as is Nep1.                                       C, and does not contain pseudouridine, and therefore is a <br />     Nep1 plays a yet unclear role in the loading of Rps19      substrate analog. Consistent with the role as a pseudourid- <br /> into the SSU. Rps19 is an SSU ribosomal protein found          ine N1-methyltransferase, structural evidence indicates <br /> only in archaeal and eukaryotic ribosomes and is also          that the substrate analog is positioned for acceptance of <br /> known as a protein mutated in roughly 25% of cases of          a methylation at the C5 position of the uridine moiety, <br /> Diamond&#226;&#8364;&#8220;Blackfan Anemia, a rare congenital disease             which is the equivalent of the N1 position of pseudourid- <br /> characterized by defects in red blood cell development         ine. This is the &#239;&#172;&#65533;rst structure of a SPOUT <br /> and predisposition to cancers (18,19). Rps19 is an essential   methyltransferase bound to cognate RNA. In addition, <br /> protein, and its depletion also results in the loss of         Nep1 is the &#239;&#172;&#65533;rst example of a SPOUT-class methyl- <br /> cleavage at site A2 (11,19&#226;&#8364;&#8220;22). In yeast studies performed     transferase that functions as an N1-pseudouridine- <br /> by Buchhaupt et al. (23) overexpression of Rps19 partially     speci&#239;&#172;&#65533;c methyltransferase, and therefore these results <br /> suppresses the <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> observed with loss of Nep1            provide a structural analysis of a unique new group of <br /> function. Surprisingly, this suppression is enhanced by        enzymes. <br /> deleting snr57, the C/D box snoRNA that guides <br /> 20 -O-ribose methylation at 18S G1572 (23,24). In fact, <br /> deletion of Nep1 is not lethal in an snR57 deletion <br /> mutant (23). Taken together, it appears that Nep1 aids         MATERIALS AND METHODS <br /> in Rps19 loading and is essential in the presence of <br />                                                                Protein expression and puri&#239;&#172;&#65533;cation <br /> snr57. This function is either through direct interaction <br /> with Rps19, or alteration of the rRNA structure (23).          The full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> Nep1 gene was PCR ampli&#239;&#172;&#65533;ed from <br /> Since the defects in rRNA processing observed in Nep1          A. fulgidus and S. cerevisiae genomic DNA (American <br /> and Rps19 depletion are identical, it is likely that some of   Type Culture Collection) and subcloned into a plasmid <br /> the ribosome biogenesis defect caused by loss of Nep1 is a     vector containing an N-terminal 6  histidine tag <br /> consequence of improper Rps19 assembly.                        followed by a Tobacco Etch Virus (TEV) protease <br />                                                                         Nucleic Acids Research, 2011, Vol. 39, No. 6 2447 <br />  <br />  <br /> cleavage site (Protein Expression Laboratory, SAIC,            Nep1&#226;&#8364;&#8220;RNA complex crystallization <br /> Frederick, MD, USA). These constructs were transformed <br />                                                                ScNep1 was mixed in a 1:1 molar ratio with RNA oligos <br /> into Escherichia coli RosettaTM-DE3-pLysS cells                of     varying     lengths    and    sequences    purchased <br /> (Novagen). Expression cultures were grown at 37  C and         HPLC-puri&#239;&#172;&#65533;ed from IDT and resuspended in 10 mM <br /> induced with 1 mM IPTG at OD600 of 0.6 for 4 h for             Tris pH 8.0. The mixtures were analyzed by gel electro- <br /> afNep1. For scNep1, cultures were grown at 37  C until         phoresis on a 6% TBE-polyacrylamide gel and visualized <br /> OD600 of 0.6, then cooled to 20  C and induced with 1 mM       by UV after ethidium bromide staining to determine stoi- <br /> IPTG for overnight expression. After expression, the cells     chiometric complex formation. A 14-base RNA oligo con- <br /> were harvested by centrifugation. For afNep1, cells were       taining the sequence 50 -GGGCUUCAACGCCC-30 was <br /> resuspended in 50 ml of 50 mM Tris pH 8.0, 300 mM              the most successful at producing stoichiometric <br /> NaCl, 0.2% b-mercaptoethanol, 0.2  BugbusterTM                 complexes. <br /> (Novagen) and 0.1 mg/ml DNase1 (Roche) per liter of               Initial crystallization conditions were obtained through <br /> expression culture and stirred at room temperature.            utilization of several sparse matrix screens (Emerald <br /> After 30 min, the lysate was transferred to centrifuge         Biostructures Inc., Nextal, Hampton Research) with the <br /> tubes and placed in a 75  C water bath for 15 min to           sitting drop vapor diffusion method. Crystals of the <br /> denature the E. coli proteins then immediately centrifuged     afNep1 protein were obtained from drops containing <br /> for 20 min at 18 000 rpm in a ultracentrifuge to remove        3&#226;&#8364;&#8220;4 M sodium chloride, 100 mM HEPES buffer pH <br /> insoluble material. For scNep1, cells were resuspended         7.0&#226;&#8364;&#8220;8.0 placed over 1 ml reservoirs. Crystals of free <br /> in 50 ml of 25 mM Tris pH 8.0, 200 mM NaCl, 0.2%               scNep1 were obtained from several conditions, including <br /> b-mercaptoethanol, 0.2  BugbusterTM and 0.1 mg/ml              50 mM BIS&#226;&#8364;&#8220;TRIS, pH 6.5, 50 mM ammonium sulfate, <br /> DNase1 (Roche) per liter of expression culture and             30% pentaerythritol ethoxylate (15/4 EO/OH) and 20% <br /> stirred at 4  C. After 45 min, the lysate was transferred to   glycerol, used for data collection. The Nep1/1RNA <br /> centrifuge tubes and centrifuged for 20 min at 18 000 rpm.     complex crystallized in 100 mM HEPES, pH7.0 and <br /> The supernatants were passed through a 0.22 mM &#239;&#172;&#65533;lter and       10% PEG (polyethylene glycol) 6000 and crystals of the <br /> loaded onto a 5 ml HisTrapTM HP column (GE                     Nep1/2RNA complex were obtained in 50 mM MES, pH <br /> Healthcare) equilibrated in buffer containing 50 mM Tris       6.0, 15 mM magnesium sulfate and 7% PEG 4000. <br /> pH 8.0, 300 mM NaCl and 0.2% b-mercaptoethanol for             Diffraction quality crystals were obtained after 4-6 days <br /> afNep1 and 25 mM Tris pH 8.0, 200 mM NaCl and 0.2%             at 20  C. <br /> b-mercaptoethanol for scNep1. The bound Nep1 was <br /> washed in 100 mM imidazole in equilibration buffer and         Data collection and processing <br /> then eluted with a gradient from 100 mM to 1 M imidazole <br /> in equilibration buffer in 20 column volumes. The              Crystals were &#239;&#172;&#8218;ash frozen in mother liquor containing <br /> fractions containing Nep1 were pooled and dialysed over-       20% glycerol for all crystals. Diffraction data for the <br /> night into 25 mM Tris pH 8.0, 300 mM NaCl, 1 mM                native and Se-Met were collected at 100 K at the <br /> EDTA and 0.2% b-mercaptoethanol for afNep1 and                 NE-CAT beamline at the Advanced Photon Source <br /> 25 mM Tris pH 8.0, 200 mM NaCl, 1 mM EDTA and                  (APS), Argonne, Illinois. Data from the Se-Met crystals <br /> 0.2% b-mercaptoethanol for scNep1. In the case of              were collected at one wavelength (peak). All data were <br /> afNep1, attempts made to remove the tags by TEV                integrated and merged using HKL2000. Table 1 contains <br /> protease cleavage were unsuccessful. The tag was cleaved       details regarding data statistics for all data sets. <br /> in scNep1 and the resulting protein contained a single <br /> extra glycine residue at the N-terminus. The cleaved           Structure determination and re&#239;&#172;&#65533;nement <br /> scNep1 protein was passed over a 5 ml HisTrap HP               For afNep1, structures were solved using single wave- <br /> column (GE Healthcare) a second time to remove                 length anomalous diffraction (SAD) data with Se-MET <br /> uncleaved protein and TEV protease. The proteins were          protein crystals, collected at 2.0 A&#203;&#353;. The AutoSharp <br /> concentrated to 5&#226;&#8364;&#8220;10 mg/ml and further puri&#239;&#172;&#65533;ed to &gt;99%         program suite was used to obtain the phases using the <br /> by size exclusion chromatography using a Superdex 75TM         SAD method, apply solvent &#239;&#172;&#8218;attening and density modi- <br /> column (GE Healthcare) equilibrated in 20 mM Tris pH           &#239;&#172;&#65533;cation to the initial electron density map, and perform <br /> 8.0, 300 mM NaCl, 1 mM EDTA and 0.2%                           automatic model building with wARP (27). The complete <br /> b-mercaptoethanol for afNep1, and a Superdex 200TM             model was &#239;&#172;&#65533;nalized by rebuilding in COOT (28) and re- <br /> column (GE Healthcare) equilibrated in 20 mM Tris pH           &#239;&#172;&#65533;nement with REFMAC5 (29). The model was then used <br /> 8.0, 200 mM NaCl, 1 mM EDTA and 0.2%                           to phase the native data set using MOLREP as part of <br /> b-mercaptoethanol for scNep1. The protein was                  CCP4i (30&#226;&#8364;&#8220;32). The scNep1 structures were solved by mo- <br /> concentrated again to 8 mg/ml for afNep1 and 20 mg/ml          lecular replacement in MOLREP using PDB ID 2V3J <br /> for scNep1. This preparation was stored at 4  C until crys-    (13). All models were subjected to several rounds of <br /> tallization screening. Se-Met afNep1 was expressed in          building in COOT and re&#239;&#172;&#65533;nement using REFMAC5 or <br /> minimal media with all amino acids supplemented except         PHENIX (28,29,33,34). TLS groups were determined <br /> for methionine, which was replaced by seleno-methionine        using the TLSMD server (35). Rfree was monitored by <br /> (Se-Met). Reducing agent (0.2% b-mercaptoethanol) was          using 5% of the re&#239;&#172;&#8218;ections as a test set in each case. <br /> added to Se-Met protein immediately prior to crystalliza-      Re&#239;&#172;&#65533;nement statistics are provided in Table 1. The coord- <br /> tion screening.                                                inates and structure factors have been deposited in the <br />  2448 Nucleic Acids Research, 2011, Vol. 39, No. 6 <br />  <br />  <br /> Table 1. Data collection and re&#239;&#172;&#65533;nement statistics <br />  <br /> PDB <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span>                                   afNep1              afNep1              scNep1              scNep1+1RNA            scNep1+2RNA <br />                                            Se-Met SAD          3O7B                3OII                3OIJ                   3OIN <br />  <br /> Data collection <br />   Space group                              P3121               P3121               P212121             P212121                P212121 <br />   Cell dimensions <br />     a, b, c (A&#203;&#353;)                           70.3, 70.3, 95.3    70.1, 70.1, 95.2    46.0, 84.7, 112.5   44.8, 88.7, 115.8      83.4, 91.9, 96.9 <br />     a, b, g (  )                           90.0, 90.0, 120.0   90.0, 90.0, 120.0   90.0, 90.0, 90.0    90.00, 90.00, 90.00    90.00, 90.00, 90.00 <br />   Molecules/Asym. <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>                     1 (1 Protein)       1 (1 Protein)       2 (2 Protein)       3 (2 Protein: 1 RNA)   4 (2 Protein: 2 RNA) <br />   Wavelength (A&#203;&#353;)                          0.97919             0.99998             1.00931             1.00931                1.00931 <br />   Resolution (A&#203;&#353;)                          30&#226;&#8364;&#8220;2.0              30&#226;&#8364;&#8220;1.45             30&#226;&#8364;&#8220;1.8              30&#226;&#8364;&#8220;1.9                 30&#226;&#8364;&#8220;2.9 <br />   Rsym (last shell)                        0.041 (0.142)       0.075 (0.517)       0.061 (0.403)       0.066 (0.392)          0.078 (0.361) <br />   I/sI                                     37.3 (16.3)         32.9 (5.8)          22.6 (1.7)          25.5 (2.3)             21.2 (2.0) <br />   Completeness (%)                         99.6 (99.9)         99.0 (100.0)        94.4 (63.1)         96.8 (80.2)            97.0 (76.5) <br />   Redundancy                               7.6 (7.6)           14.3 (10.0)         6.9 (3.2)           7.1 (4.2)              6.3 (2.9) <br /> Re&#239;&#172;&#65533;nement <br />   Resolution (A&#203;&#353;)                                              29&#226;&#8364;&#8220;1.45             20&#226;&#8364;&#8220;1.85             28&#226;&#8364;&#8220;1.9                 30&#226;&#8364;&#8220;3.0 <br />   Rwork/Rfree (%)                                              19.8/21.1           17.1/21.6           19.2/23.0              17.4/25.2 <br />   No. of protein residues monomers A/B                         216                 216/208             217/217                211/217 <br />   No. of non-protein molecules <br />   Nucleic Acid (bases)                                         0                   0                   14                     28 <br />   Ligand (SAH)                                                 1                   2                   1                      2 <br />   Solvent                                                      242                 224                 258                    64 <br />   Mean B-factors (A&#203;&#353;)2 (TLS re&#239;&#172;&#65533;ned)                            23.3                35.0                48.1                   68.9 <br /> RMS deviations <br />   Bond lengths (A&#203;&#353;)                                            0.010               0.021               0.008                  0.008 <br />   Bond angles (  )                                             1.281               1.881               1.069                  1.257 <br /> Ramachandran plot <br />   Most favorable (%)                                           94.2                93.4                92.8                   87.5 <br />   Additionally allowed (%)                                     5.8                 6.3                 7.0                    12.5 <br />   Generously allowed (%)                                       0.0                 0.3                 0.3                    0.0 <br />   Disallowed (%)                                               0.0                 0.0                 0.0                    0.0 <br />  <br />  <br />  <br /> Protein Data Bank (accession codes 3O7B for afNep1,                         models, density was observed in the SAM-binding pocket <br /> 3OII, 3OIN and 3OIJ for scNep1 free dimer, 1RNA                             into which SAH was modeled. <br /> complex and 2RNA complex, respectively). <br />                                                                             Dimerization interface of scNep1 <br /> RESULTS                                                                     Although the structure of Nep1 from yeast has been pre- <br />                                                                             viously reported, the deposited structure (PDB ID 2V3J) <br /> Structures and comparison of the dimeric Nep1 from                          was monomeric (13). In the crystal structure of scNep1 <br /> S. cerevisiae (yeast) and A. fulgidus                                       bound to SAH presented here, the molecule is dimeric. <br /> The overall structure of the Nep1 protein, shown in                         In both archaea and yeast, a large interacting surface is <br /> Figure 1, consists of a central b-sheet surrounded by                       observed between Nep1 molecules that buries surface area <br /> a-helices on either side, characteristic of a Rossmann                      in excess of 845 A&#203;&#353;2 for afNep1 and around 1475 A&#203;&#353;2 for <br /> fold (11,13). The central b-sheet is comprised of six                       scNep1. The dimer interface is formed primarily between <br /> b-strands, &#239;&#172;&#65533;ve of which are parallel to each other and                      residues on helices aA and aE and the N-terminal end of <br /> one anti-parallel (b3), with an additional two loosely                      ac0 . The interactions form in a 2-fold symmetric interface <br /> ordered strands completing the sheet (b4, b5). The order                    that can be seen in Figure 1B and C (Supplementary <br /> in which sequential b-strands form the b-sheet is                           Figure S1 for afNep1). The residues at the dimer interface <br /> b5-b4-b3-b2-b1-b7-b6-b8. The helices within the                             of scNep1 are highly conserved among eukaryotes, but <br /> Rossmann domain are located between b1 and b2 (aA,                          less with archaeal organisms (Supplementary Figures S2 <br /> aAb, ad0 ), b3 and b4 (ac0 , Hc) and after b8 (aE). Two                     and S3). The scNep1 interface consists of a mixture of <br /> additional helices, Hb0 and aa0 are present in a small                      electrostatic interactions and hydrogen bonds on the <br /> Nep1 speci&#239;&#172;&#65533;c 40 amino-acid insertion between b1 and a2                      edges and a few hydrophobic interactions at the center, <br /> (residues 48&#226;&#8364;&#8220;88). This insertion and the orientation of aa0                 also several water molecules were observed in the charged <br /> is characteristic of Nep1 SPOUT methyltransferases, and                     regions of the interface. This shows a less tight dimer <br /> also exhibits the most difference between the archaeal and                  interface than that observed for the afNep1 interface, <br /> yeast structures. This domain is moderately conserved                       which contains a stabilizing disul&#239;&#172;&#65533;de bond, commonly <br /> based on sequence comparison with eukaryotic Nep1 se-                       used to stabilize protein structure in thermophiles <br /> quences (Figure1A). Electron density was observed for all                   (Supplementary Figure S1) (36). Interestingly, in scNep1, <br /> the residues except 1&#226;&#8364;&#8220;27 for both monomers, 56&#226;&#8364;&#8220;64 for                       the side chain of the C-terminal Leu 252, forms hydropho- <br /> monomer A and 54&#226;&#8364;&#8220;65 and 153&#226;&#8364;&#8220;157 for monomer B, of                           bic packing interactions at the interface and the carboxyl <br /> scNep1, and for all but the last three for afNep1. In both                  group at the end of the chain forms a hydrogen bond with <br />                                                                                           Nucleic Acids Research, 2011, Vol. 39, No. 6 2449 <br />  <br />  <br />  <br />  <br /> Figure 1. Overall structure of Nep1 and dimer interface. (A) Sequence alignment from of Nep1 from S. cerevisiae (NEP1_YEAST), Homo sapiens <br /> (NEP1_HUMAN), Drosophila melanogaster (NEP1_DROME) and A. fulgidus (NEP1_ARCFU). Yellow, red and blue dots indicate residues <br /> involved in dimerization, RNA binding and SAM binding respectively. Secondary structure for yeast (top) and A. fulgidus (bottom) shown as <br /> open bars for a-helices and arrows for b-strands and numbered according to Leulliot et al. for yeast, and Taylor et al. for A. fulgidus (11,13). (B) The <br /> scNep1 dimer. RNA binding and catalytic regions of the molecules are colored red and dark blue respectively. The molecules of SAH are shown in <br /> sticks. Secondary structure elements are labeled according to Leulliot et al. (13). (C) Front view of the dimerization interface of Nep1. Residues are <br /> labeled with chain names and residue numbers. (D) Top view of the interface showing the interaction of the C-terminus in the interface. Residues <br /> that are identical in eukaryotic sequences are labeled in red text. All 3D structure &#239;&#172;&#65533;gures were generated in PyMOL (www.pymol.org). <br />  2450 Nucleic Acids Research, 2011, Vol. 39, No. 6 <br />  <br />  <br /> the side chain of Ser 228 and Asn 229 and the backbone                      determined that an ideal complex was obtained when <br /> amide nitrogen of Asn 229 of the other monomer                              scNep1 and a 14-base RNA fragment with the sequence <br /> (Figure1C and D). Therefore, the C-terminus participates                    G1G2G3C4U5U6C7A8A9C10G11C12C13C14 were mixed in a <br /> in dimer formation. In the previously reported structure of                 1:1 ratio. We obtained crystals of Nep1 bound to this <br /> scNep1, the construct used contained a C-terminal histi-                    14-base RNA fragment in two crystal forms (Figure 2A <br /> dine tag that may have disrupted this interaction and                       and B). In one crystal form, a dimer of scNep1 is bound to <br /> weakened the dimer to some extent, resulting in                             one RNA molecule (Figure 2A). In the second crystal <br /> monomers in the crystals (13). In the absence of RNA,                       form, the dimer binds two separate RNA molecules <br /> one monomer of scNep1 aligns with the other with an                         (Figure 2B). In both cases, the RNA forms a stem-loop <br /> RMSD of 0.34 A&#203;&#353;, and small conformational differences                       structure in which eight bases participate in canonical base <br /> are observed between loop regions of the two monomers.                      pairs in the stem, and six bases in the loop (UUCAAC) <br />                                                                             have base-speci&#239;&#172;&#65533;c contacts with the protein. Electron <br /> Interaction of scNep1 with RNA                                              density was not observed for residues 1&#226;&#8364;&#8220;27 and 56&#226;&#8364;&#8220;64 of <br /> Using the binding sequences for scNep1 obtained through                     both monomers in the RNA complexes, and residues <br /> RNA three-hybrid screening by Buchhaupt et al. (23), we                     76&#226;&#8364;&#8220;82 of monomer A of the two-RNA complex. There <br /> designed a series of oligonucleotides and tested their                      are no signi&#239;&#172;&#65533;cant differences between the RNA molecules <br /> binding to scNep1 in stoichiometric complexes. We                           bound in the two complexes. The six bases in the form <br />  <br />  <br />  <br />  <br /> Figure 2. Overall structure of the Nep1/RNA complexes. Structures of the (A) one RNA and (B) two RNA complexes with SAH (pink) and RNA <br /> (magenta and yellow) shown in sticks. (C) Electrostatic surface of the Nep1 complex showing front, top and bottom views. The surface is colored red <br /> to blue, negative to positive charge (calculated using vacuum electrostatics in PyMOL). <br />                                                                                    Nucleic Acids Research, 2011, Vol. 39, No. 6 2451 <br />  <br />  <br /> the basis for speci&#239;&#172;&#65533;c recognition by the protein. In the                 positively charged surface that is not contacted by RNA in <br /> comparison of the sequences of RNA found to bind                         these structures (Figure 2C). This positively charged <br /> Nep1 by RNA three-hybrid screens, all the sequences con-                 surface is conserved in the afNep1 as well <br /> tained the consensus sequence C/UUCAAC, and the most                     (Supplementary Figure S3). This is consistent with the <br /> tightly binding were those sequences that contained the                  idea that Nep1 binds to RNA on the pre-ribosome, <br /> consensus sequence within the loop of a predicted                        which makes it likely that it will contact signi&#239;&#172;&#65533;cantly <br /> stem-loop (23). In the crystal containing the one RNA                    more RNA than is observed with these minimal con- <br /> per dimer complex, the empty RNA-binding site is                         structs. The regions that are not contacted by RNA in <br /> involved in four hydrogen bonds with the RNA                             these structures contain many conserved residues as well, <br /> backbone from a symmetry-related complex.                                indicating that this surface is important for Nep1 function <br />    In both complexes, the RNA binds on highly positively                 (Supplementary Figure S2). <br /> charged surfaces of the dimer as predicted in previous                      ScNep1 provides several key interactions that confer the <br /> reports (11,13). However, the protein contains signi&#239;&#172;&#65533;cant                observed speci&#239;&#172;&#65533;city for a UUCAAC sequence (Figure 3, <br />  <br />  <br />  <br />  <br /> Figure 3. Nep1-RNA interactions. (A) Stereo &#239;&#172;&#65533;gures of RNA bound in the cleft between the two monomers. Residues are colored according to <br /> monomer from which they originate (green-monomer A; cyan-monomer B). Dash lines represent connections between atoms within hydrogen <br /> bonding distance. RNA (yellow) truncated to show only the bases that contact the protein for simplicity. (B) Stereo view of a close-up of the <br /> uridine base interactions with 2Fo&#226;&#8364;&#8220;Fc map contoured at 1s. Protein residues are colored as in (A). U6 is shown in magenta. Distances between <br /> atoms are indicated by red text. <br />  2452 Nucleic Acids Research, 2011, Vol. 39, No. 6 <br />  <br />  <br /> schematic in Supplementary Figure S4). The &#239;&#172;&#65533;rst base of         hydrogen bonds with conserved A: Arg 136, one <br /> the consensus sequence, U5, is contacted through                between the Arg NH2 and the C10 N3, and the other <br /> base-speci&#239;&#172;&#65533;c contacts through bidentate hydrogen bond           between the Arg NE and C10 O2. There is also an inter- <br /> interactions between the NH1 and NH2 of conserved               action between the C10 ribose O20 and the backbone <br /> Arg 136 of monomer A (A:Arg 136 NH1 and NH2)                    carbonyl of A: Arg136. Arg 136 contacts C10 via three <br /> and the C2 carbonyl oxygen (O2) of the uridine base             hydrogen bonds, U5 via two hydrogen bonds, and stacks <br /> (Figure 3A and Supplementary Figure S4). There are no           in between A9 and C7. Consequently, Arg 136 is invariant <br /> contacts between the protein and C4 carbonyl oxygen             in all sequences. In a functional analysis by Taylor et al. <br /> (O4), which explains the observation that a cytosine,           (11) mutations of Arg 136, as well as Arg 88, Arg 129 and <br /> which differs from uracil solely by an amine group at           Arg 132 to Ala all result in diminished RNA-binding <br /> C4, can substitute at this position (23).                       activity as measured by yeast three-hybrid assays with <br />    The next uracil in the consensus sequence, U6, is the site   consensus sequences. <br /> of modi&#239;&#172;&#65533;cation by the enzyme. It is &#239;&#172;&#8218;ipped out from the <br /> loop and bound in a pocket between the two monomers, <br /> near the bound SAH molecule, in the enzyme active site          Non-speci&#239;&#172;&#65533;c interactions <br /> (Figure 3A and B). It is recognized by a direct hydrogen <br /> bond with A:Arg 132 NE (Ne of arginine 132 of monomer           Of the eight bases that form the stem in canonical <br /> A) at the U6 O2 (C2 carbonyl oxygen of U6), and                 Watson&#226;&#8364;&#8220;Crick base pairs, only one, G11, is contacted at <br /> water-mediated hydrogen bond interactions between the           the base by a protein residue. B: Glu 74 accepts a <br /> U6 O4 and the backbone amide of B: Ser 233 and the U6           hydrogen bond from G11 N2, and contacts the G11 <br /> O4 and N3 and OE (side chain carbonyl oxygen) of                ribose O20 via a water-mediated interaction (Figure 3 <br /> conserved B: Gln 94. Arg 132 is held &#239;&#172;&#65533;rmly in position          and Supplementary Figure S4). Other backbone inter- <br /> by hydrogen bonding between Arg 132 NH1 and NH2                 actions between the protein and the RNA include <br /> and conserved A: Asp 101 OD1 and OD2. In addition,              contacts between A: Gln 143 and C10 O20 and phosphate <br /> hydrogen bonds are observed between the U6 O2 and               between C10 and A9, A: Arg 129 and C11 O2P, A: Arg 211 <br /> NH1 and NH2 of B:Arg 88 and between the ribose O4               and the phosphate between U5 and U6. Of these, Arg 129 <br /> and the NH1 of B:Arg 88. This would hold U6 &#239;&#172;&#65533;rmly in            and Gln 143 are highly conserved, while Arg 211 is only <br /> position, and may serve as a means to stabilize negative        weakly conserved (Figure 1A). Based on this data, we can <br /> charge on the base during methyl transfer. Arg 88, in turn,     conclude that very little speci&#239;&#172;&#65533;city is determined in the <br /> is held in place by interactions between its NH2 and NE         stem-loop portion of the RNA. <br /> and the OD1 and OD2 of conserved A:Asp 90. Asp 90 is <br /> the equivalent of the human Asp 86 that is mutated to <br /> glycine in Bowen&#226;&#8364;&#8220;Conradi syndrome. That mutation                Catalytic residues in the Nep1 active site <br /> would have the effect of destabilizing the interaction <br /> with the U6 substrate both through disruption of the inter-     The methylation reaction that Nep1 catalyzes would <br /> action with the Arg 88 and hydrophobic contacts with Ile        require the deprotonation of, and transfer of the methyl <br /> 91 through repositioning. Finally C5 of the uridine base is     group from SAM to, the N1 of the pseudouridine. <br /> positioned a mere 3.9 A&#203;&#353; away from the sulfur atom of           Stabilization of a negative charge on the uridine ring <br /> SAH, in perfect position for a methyl transfer to the C5        would result in a decrease of the N1 pKa, which was <br /> position of the uracil (Figure 3). If a pseudouridine were in   measured at 8.97 for free pseudouridine (37). The likely <br /> its place, then N1, which is an equivalent position to C5,      candidate for this stabilization would be invariant Arg 88, <br /> would be the target of methylation.                             for which both primary amine nitrogens are within <br />    In the remaining consensus sequence, C7 is recognized        hydrogen bonding distance of U6 O2, the C2 carbonyl <br /> through backbone interactions between the carbonyl              oxygen (3.1 and 2.7 A&#203;&#353;; Figure 3B and Supplementary <br /> oxygen, O2, and the backbone amides on either side of           Figure S5), and which is stabilized by bidentate <br /> A: Arg 132, and a base stacking interaction with A: Arg         hydrogen bonds with invariant Asp 90, which is mutated <br /> 136. A8 is sandwiched between a stacking interaction with       in Bowen&#226;&#8364;&#8220;Conradi Syndrome (8). Mutation of Arg 88 to <br /> conserved A: Arg 129 and van der Waal&#226;&#8364;&#8482;s contact with            Asp resulted in a signi&#239;&#172;&#65533;cant loss of Nep1 RNA-binding <br /> conserved A: Ile 159, and donates a hydrogen bond from          activity (13). Arg 88 and Asp 90 are located at the begin- <br /> N6 to the backbone carbonyl oxygen of A: Thr 127. A8 N6         ning of aA, and are surrounded by Pro 89 and Ile 91, <br /> is also involved in a water-mediated hydrogen bond inter-       which are also invariant. Ile 91 is in van der Waals <br /> action with the side chain OH of Thr 127 and A8 N1 is in        contact distance of the uridine ring in the active site. <br /> hydrogen bonding distance of the backbone amide of A:           This region is labeled as the &#226;&#8364;&#732;Catalytic Loop&#226;&#8364;&#8482; in Figure <br /> Arg 129. A9 is recognized through hydrogen bonds                1A and B. Arg 132 may also provide further stabilization, <br /> between the backbone carbonyl of A: Arg 129 and the             since it is also within hydrogen bonding distance (3.0 A&#203;&#353;) of <br /> N6, a van der Waal&#226;&#8364;&#8482;s interaction between the side chain         O2. These interactions would increase deprotonation at <br /> of A: Val 128 and the C2, and a stacking hydrophobic            the N1 through localization of the negative charge to O2 <br /> interaction with conserved A: Leu 140. Although Val             where it would be stabilized by interaction or proton <br /> 128 is not highly conserved, it is Val, Thr or Cys in           donation by Arg 88 (Supplementary Figure S5). This <br /> other sequences, which would be able to maintain the            would in turn enable N1 to accept the methyl group <br /> interaction with A9 C2. Finally C10 is speci&#239;&#172;&#65533;ed by two          from SAM. <br />                                                                                      Nucleic Acids Research, 2011, Vol. 39, No. 6 2453 <br />  <br />  <br /> Conformational changes in the scNep1 protein when                         of overall secondary structure, as the structure changes to <br /> bound to RNA                                                              accommodate interactions with the RNA (Figure 4A <br />                                                                           and B). The domain, which is packed against ac&#226;&#8364;&#8482;, houses <br /> When a monomer of free Nep1 is compared to monomers <br />                                                                           Arg 88, which may be important for catalysis, and Glu 74 <br /> from the RNA Nep1 complex, an RMSD of 0.76 A&#203;&#353; is <br />                                                                           that interacts with G11 (Figure 3A). The buried surface <br /> observed for 415 out of 426 residues. Most of the inter- <br />                                                                           area in the RNA bound structure (1338 A&#203;&#353;2) is slightly <br /> action between the RNA and the protein are mediated by <br />                                                                           decreased, due to an opening of the interface to accom- <br /> the helix ac0 and adjacent loops on the monomer that does <br />                                                                           modate the RNA, which buries an additional 721 A&#203;&#353;2 on <br /> not provide the SAM to the active site (monomer A in <br />                                                                           the monomer containing the methyl donor and 351 A&#203;&#353;2 on <br /> Figures 1, 2 and 3). However, the most signi&#239;&#172;&#65533;cant con- <br />                                                                           the other monomer. <br /> formational changes are seen in the small Nep1 speci&#239;&#172;&#65533;c <br /> inserted domain on the SAM-binding side of the dimer <br />                                                                           Two-RNA versus One-RNA <br /> interface (Figure 4A and B). This domain, consisting of <br /> a 3&#226;&#8364;&#8220;10 helix (Hb0 ), small a-helix (aa0 ) and a b-hairpin,                Both the 1:1 and 1:2 complexes for the complexes between <br /> undergoes a shift in positioning towards the dimer inter-                 Nep1 and RNA were obtained using the same preparation <br /> face when RNA is bound. In addition, there is alteration                  of Nep1 to RNA in a 1:1 ratio. The isolation of these two <br />  <br />  <br />  <br />  <br /> Figure 4. Conformational changes upon RNA binding. (A) Front and (B) Bottom views of the Nep1/1 RNA complex (green-monomer A, <br /> cyan-monomer B) superposed on the free dimer (magenta). (C) Front view of the alignment of the 1 RNA and 2 RNA (yellow) complexes. <br /> (D) Bottom view of both RNA complexes and the free dimer. Monomer B, shown here on the right, was used to align the monomers. Asterisk <br /> indicates the Nep1 speci&#239;&#172;&#65533;c domain where signi&#239;&#172;&#65533;cant structural change occurs, and ac&#226;&#8364;&#8482; indicates the RNA-binding helix that shifts relative to the <br /> aligned monomer. <br />  2454 Nucleic Acids Research, 2011, Vol. 39, No. 6 <br />  <br />  <br />  <br />  <br /> Figure 5. Comparisons of SAH ligand binding. (A) SAH bound in the SAM-binding pocket in the Nep1/1RNA complex, showing electron density <br /> 2Fo&#226;&#8364;&#8220;Fc map contoured at 1s. (B) Alignment of the SAM-binding pockets of Nep1/1RNA complex monomer B (cyan), Nep1/2RNA monomer B <br /> (green), Nep1/2RNA monomer A (magenta) and free Nep1 dimer (orange) showing side chains of residues that show the most conformational <br /> differences between the complexes. Spheres indicate bound waters, and are labeled &#226;&#8364;&#732;W&#226;&#8364;&#8482; in B. <br />  <br />  <br /> complexes in the crystal structures suggests that both                observations, and the presence of additional conserved <br /> complexes exist in solution, which implies differences in             positive charged surface (Figure 2C), it is possible that <br /> the binding af&#239;&#172;&#65533;nities between the two sites. Comparisons              both RNA-binding sites would be occupied on rRNA, <br /> of the structures of these complexes reveal differences in            perhaps with varied binding af&#239;&#172;&#65533;nity or speci&#239;&#172;&#65533;city, and <br /> the structures of the RNA-binding sites as a function of              additional binding surface would be used to interact on <br /> occupancy with RNA (Figure 4C and D). The RMSD for                    the surface of the rRNA. <br /> the alignment of the protein molecules in the complexes <br /> averages 0.85 A&#203;&#353; for 421 residues of the protein chains, with <br /> the largest difference being between chain A of the 1:1               DISCUSSION <br /> complex and chain B of the 2:1 complex. This number is                Prior to this work, the structures of the scNep1 in mono- <br /> larger than the RMSD for the comparison between the                   meric form, and the archaeal M. jannaschii Nep1 dimer, <br /> Nep1/1RNA complex and the free enzyme, indicating a                   allowed for the classi&#239;&#172;&#65533;cation of Nep1 as a SPOUT <br /> larger conformational difference between the Nep1/1RNA                methyltransferase, characterization of the Nep1 speci&#239;&#172;&#65533;c <br /> and Nep1/2RNA complexes than between the free enzyme                  structural elements and analysis of the SAM-binding <br /> and the Nep1/1RNA complex. Upon closer inspection, we                 pocket, and, in combination with mutagenesis and <br /> observed that the positioning of one monomer relative to              NMR mapping on the M. jannaschii Nep1, allowed for <br /> the other is slightly different between the two complexes.            a prediction of the RNA-binding site (11&#226;&#8364;&#8220;14). This <br /> In the Nep1/2RNA complex, the two monomers are                        report provides a view of dimeric scNep1 in apo form <br /> slightly closer together near the RNA-binding site,                   and in complex with RNA, allowing a &#239;&#172;&#65533;rst look at a <br /> indicating that RNA binding pulls the two monomers                    SPOUT methyltransferase bound to RNA, a high reso- <br /> closer together. As a result, the SAH molecule is pulled              lution analysis of the interactions of the only known <br /> closer to the RNA. The SAH molecule is contacted mostly               N1-speci&#239;&#172;&#65533;c pseudouridine methyltransferase with a sub- <br /> by backbone interactions, but one residue, Asp 214,                   strate analog, and a picture of the RNA structure that <br /> hydrogen bonds the ribosyl 30 OH via one of its side-chain            Nep1 stabilizes upon binding. These data con&#239;&#172;&#65533;rm that <br /> carbonyl oxygens (Figure 5A). In the Nep1/2RNA                        the RNA binds to the cleft between the two monomers, <br /> complex, for one monomer site the Asp 214 side-chain is               which agrees with previously reported NMR mapping <br /> rotated away from the SAM-binding site and no longer                  data and mutational analysis that reported mutation of <br /> contacts the bound SAM (Figure 5B and Supplementary                   the M. jannaschii equivalents of R88, R129, R132 and <br /> Figure S5A). This results in a slightly altered positioning           R136 signi&#239;&#172;&#65533;cantly reduced RNA-binding activity (11,14). <br /> of the homocysteinyl moiety in the active site, with the              In Nep1, the binding site accommodates a stem-loop <br /> sulfur atom positioned farther away (4.5 A&#203;&#353;) from the                 structure, with the majority of binding observed between <br /> uridine base (Figure 5B and Supplementary Figure                      the RNA loop and the protein. Both the RNA <br /> S5A). In addition, the SAM-binding pocket on the side                 three-hybrid screening of scNep1 and the speci&#239;&#172;&#65533;city <br /> of the dimer not bound to RNA in the Nep1/1RNA                        determination on M. jannaschii Nep1 indicate that <br /> complex is not visibly occupied by SAH. Based on these                Nep1 binds more tightly to a predicted stem-loop <br />                                                                                        Nucleic Acids Research, 2011, Vol. 39, No. 6 2455 <br />  <br />  <br />  <br />  <br /> Figure 6. Possible role for Nep1 in S19 loading. (A, B) Three dimensional and (C) Secondary structure map of the head region of eukaryotic 18S <br /> rRNA, truncated to remove bases 1212&#226;&#8364;&#8220;1439 for simplicity. (A) and (B) were generated from PDB ID 3JYV [cryo-EM model of the 40S subunit of <br /> the Thermomyces lanuginosus ribosome (25)]. (C) is based on the secondary structure prediction for S. cerevisiae 18S rRNA from the Comparative <br /> RNA Web Site (www.rna.ccbb.utexas.edu). Regions in tertiary structure and secondary are colored orange, cyan and green for helix 31, 41 and 42 <br /> respectively. Red color indicates Nep1 consensus sites, with U1189 and U1566, the bases that would bind in the Nep1 active site, in small spheres. <br /> The snR57-binding site is highlighted in yellow in (C). Purple denotes parts of rRNA within 4 A&#203;&#353; of S19 (shown as blue cartoon). Asterisk indicates <br /> the site of interaction between helix 41 and 42. <br />  <br />  <br /> structure, suggesting that this is the preferred substrate                  Bowen&#226;&#8364;&#8220;Conradi Syndrome, also plays a role by holding <br /> for Nep1 (23). However, for M. jannaschii Nep1, se-                         Arg 88 in place in the structure. There are no candidates <br /> quences that do not contain a stem can also bind                            for direct hydrogen binding near the C5 of the uridine in <br /> (14,23). We have no information about the structure of                      our structure that may allow speci&#239;&#172;&#65533;c selection of pseudo- <br /> the free RNA, and no similar loops containing this                          uridine, but the absence of an activating nucleophile <br /> sequence were found in the structure databases, but                         in the vicinity of C6 in these structures rules out the <br /> the stem-loop is predicted to form at 37  C by RNAfold                      possibility that this protein functions as an m5U <br /> (38), although we are not sure if U6 would be &#239;&#172;&#8218;ipped                        methyltransferase, as would be suggested if uridine was <br /> out in the absence of protein. The RNA seen in our struc-                   the substrate (39,40). In addition, m5U modi&#239;&#172;&#65533;cation has <br /> tures fails to contact much of the positively charged                       not been observed in yeast 18S rRNA. However, this <br /> surface of the protein, leaving open the possibility that                   means that uridine is indistinguishable from pseudourid- <br /> the actual binding site is larger than the selected consensus               ine in the Nep1 active site, but Nep1 will only catalyze <br /> sequence.                                                                   methyl transfer on pseudouridine. This is supported by <br />    It has been proposed that Nep1 is the &#239;&#172;&#65533;rst example of                    &#239;&#172;&#8218;uorescence binding data for M. jannaschii Nep1 <br /> an N1-speci&#239;&#172;&#65533;c pseudouridine methyltransferase and that                      reported by Wurm et al. (14) that shows no signi&#239;&#172;&#65533;cant <br /> it catalyzes the only such modi&#239;&#172;&#65533;cation known in 18S                         differences in RNA-binding Kd when pseudouridine is <br /> rRNA, the N1-methylation on the hypermodi&#239;&#172;&#65533;ed                                replaced with uridine in high af&#239;&#172;&#65533;nity RNA-binding sites. <br /> m1acp3	 at position 1189 in yeast (14). The structural                         Although it is likely that the N1-methyltransferase <br /> evidence, seen in the speci&#239;&#172;&#65533;c positioning of the C5 of                      activity is important for Nep1 function, as supported by <br /> uridine in the enzyme active site, supports this. The                       the severe defect caused by mutations at Asp 90, it has <br /> uridine is held in place by several contacts that impart                    been reported that mutations in the SAM-binding site of <br /> this speci&#239;&#172;&#65533;city, and the interaction with Arg 88 suggests                   Nep1 that reduces its ligand-binding activity by 100-fold <br /> that this residue plays a role in catalysis, by promoting                   results in no disruption of Nep1 activity (13). Therefore it <br /> deprotonation of the ring (Supplementary Figure S5B).                       is plausible that binding of Nep1 to the rRNA in the <br /> Asp 90, which is equivalent to Asp 86 that is mutated in                    absence of methyltransfer activity is suf&#239;&#172;&#65533;cient to carry <br />  2456 Nucleic Acids Research, 2011, Vol. 39, No. 6 <br />  <br />  <br /> out at least some of its role in ribosome biogenesis. In       (42). In any case, the genetic interaction between the <br /> addition, overexpression of yeast ribosomal protein            snR57 and Nep1 supports the possibility of Nep1 <br /> Rps19 compensates for de&#239;&#172;&#65533;ciency in Nep1 function (23).         binding in the region near helix 42. <br /> An explanation put forth for these observations was that          Methanocaldococcus jannaschii Nep1 binds to yeast 18S <br /> Nep1 may play a role that is independent of Nep1               rRNA sequences corresponding to the loop of helix 31 <br /> methyltransferase activity (13,23). A possible role may        (14), and afNep1 similarly is capable of binding yeast <br /> be to act as a chaperone of RNA folding during a               rRNA sequences corresponding to the Nep1-binding site <br /> speci&#239;&#172;&#65533;c assembly step. The observation of a two RNA            near helix 42 (data not shown). Residues involved in key <br /> complex in which both binding sites are occupied               interactions with the RNA, for example Arg 88, Arg 136, <br /> suggests that it is plausible that rRNA occupies both          Arg 129, Leu 140 and Leu 159, are identical in afNep1. <br /> sites when Nep 1 binds in vivo. Based on structural            Superposition of the archaeal dimer on the yeast struc- <br /> analysis, the two binding sites are non-equivalent, high-      tures showed that the RNA interaction mode is likely to <br /> lighted by differences in binding to SAH (Figure 5B and        be very highly conserved between archaea and yeast <br /> Supplementary Figure S5A). This idea that both sites           (Supplementary Figure S3D). Although these structures <br /> would be occupied in the ribosome is further supported         do not provide all the answers on Nep1 function, it does <br /> by the presence of extensive positively charged surface not    provide the basis for continued studies on the ancient <br /> contacted by the stem-loops in the structures. Two of the      structural mechanisms utilized by Nep1 in rRNA <br /> three possible Nep1-binding sites in yeast 18S rRNA are        processing. <br /> located within 50 A&#203;&#353; of each other in the 3D structure of <br /> the eukaryotic ribosome (25) (Figure 6A and C). Those <br /> sites are the loop terminating helix 31 and a portion of       ACCESSION NUMBERS <br /> helix 42 (Figure 6C). Based on the EM reconstruction of        3O7B, 3OII, 3OIJ and 3OIN. <br /> the eukaryotic ribosome (25), in order for Nep1 to bind <br /> the site at helix 41, signi&#239;&#172;&#65533;cant secondary structure <br /> alteration would have to occur (Figure 6 and                <br /> Supplementary Figure S6). Half of the expected stem <br />                                                                 <br /> loop that is expected to form when Nep1 binds is part <br /> of helix 42. Therefore, it is possible that when Nep1 <br /> binds, there is a rearrangement of the interactions to          <br /> produce the expected stem-loop (Figure 6 and <br /> Supplementary Figure S6). This rearrangement would         <br /> also result in the two Nep1 sites being moved closer           <br /> together, since the distance between active sites in the        <br /> Nep1 dimer is  20 A&#203;&#353;, while it is predicted to be 50 A&#203;&#353;       <br /> apart in the mature rRNA (Figure 6A). The loop at         <br /> the end of helix 42 interacts with the loop of helix 41 <br /> (Figure 6B). Helix 41 is the site of most of the contacts <br />                                                                FUNDING <br /> between the rRNA and S19 (Figure 6B and C). Therefore, <br /> a possible model is that Nep1 interacts with the rRNA at       National Science Foundation (MCB0953493, CAREER <br /> two sites&#226;&#8364;&#8221;the loop of helix 31, where it catalyzes the         Award to N.L.); Maryland Department of Business and <br /> methylation of 	1189 and the stem of helix 42, where it        Economic Development Nano-Biotechnology Award (to <br /> acts as an RNA chaperone and breaks apart helix 42,            the University of Maryland); and Achievement Rewards <br /> rearranging the area and the interactions with helix 41        for College Scientists (ARCS) Fellowship (to S.T); <br /> to promote S19 binding. In this scenario, more of the          National Center for Research Resources at the National <br /> Nep1 surface would be expected to contact rRNA, as pre-        Institutes of Health, Advanced Photon Source on the <br /> dicted by this structure. This is assuming that rRNA in        Northeastern Collaborative Access Team beamlines <br /> early maturation stages have structures near the predicted     (award RR-15301); U.S. Department of Energy, Of&#239;&#172;&#65533;ce <br /> structure of the mature rRNA. However, there is evidence       of Basic Energy Sciences, Advanced Photon Source <br /> is that pre-rRNA structure can differ from mature rRNA         (Contract No. DE-AC02-06CH11357). Funding for open <br /> structure (41). In the absence of snR57, which is predicted    access     charge:   National   Science    Foundation <br /> to bind between bases 1568 and 1584, a Nep1 deletion is        (MCB0953493, CAREER Award to N.L.). <br /> no longer lethal, indicating that Nep1 is only required <br /> when snR57 is present (23). Since snR57 guides the methy-      <br /> lation of G1572 (24), a site near helix 42 (Figure 6C), this <br /> interaction between Nep1 and snR57 may be due to struc-      <br /> tural changes that occur as a result of methylation. <br /> Another possibility, however, is that the interaction          <br /> between Nep1 and its binding site in that region may          <br /> promote the dissociation of snR57 from rRNA. This dis-             <br /> sociation may be necessary for proper rRNA structure, or     <br /> even for N7 methylation of G1575 catalyzed by Bud23p              <br /> </body></html>