<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Jan. 2004, p. 473&#226;&#8364;&#8220;480                                                                                Vol. 186, No. 2 <br /> 0021-9193/04/$08.00&#226;&#171;&#185;0 DOI: 10.1128/JB.186.2.473&#226;&#8364;&#8220;480.2004 <br /> Copyright &#194;&#169; 2004, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  The Sorbitol Phosphotransferase System Is Responsible Transport <br />          2-C-Methyl-D-Erythritol Salmonella enterica <br />                        Serovar Typhimurium <br />                               Charles A. Testa,&#226;&#8364;&#160; Rita M. Cornish, C. Dale Poulter* <br />                                Department Chemistry, University Utah, Salt Lake City, Utah 84112 <br />                                               Received 23 June 2003/Accepted 8 October 2003 <br />  <br />             2-C-methyl-D-erythritol 4-phosphate committed intermediate biosynthesis isoprenoid <br />           precursors isopentenyl diphosphate dimethylallyl diphosphate. Supplementation growth medium <br />           2-C-methyl-D-erythritol shown complement disruptions Escherichia coli gene for <br />           1-deoxy-D-xylulose 5-phosphate synthase, enzyme synthesizes immediate precursor 2-C-methyl- <br />           D-erythritol 4-phosphate. In order utilized isoprenoid biosynthesis, 2-C-methyl-D-erythritol be <br />           phosphorylated. We construction Salmonella enterica serovar Typhimurium strain RMC26, in <br />           essential gene encoding 1-deoxy-D-xylulose 5-phosphate synthase disrupted insertion of <br />           synthetic mevalonate operon consisting yeast ERG8, ERG12, ERG19 genes, responsible for <br />           converting mevalonate isopentenyl diphosphate control arabinose-inducible promoter. <br />           Random mutagenesis RMC26 produced defects sorbitol phosphotransferase prevented <br />           transport 2-C-methyl-D-erythritol cell. RMC26 mutant strains RMC26 unable grow on <br />           2-C-methyl-D-erythritol incubated buffer containing mevalonate deuterium-labeled 2-C-methyl-D- <br />           erythritol. Ubiquinone-8 isolated cells analyzed deuterium content. Efficient incorpo- <br />           ration deuterium observed RMC26. However, evidence deuterium incorporation into <br />           isoprenoid chain ubiquinone Q8 RMC26 mutants. <br />  <br />  <br />    With 33,000 different compounds known date,                  D-glyceraldehyde 3-phosphate pyruvate 1-deoxy- <br /> isoprenoids diverse groups compounds                   D-xylulose 5-phosphate (DXP) (30). MEP synthase cata- <br /> found nature (9). Isoprenoid molecules important                lyzes NADPH-dependent rearrangement reduction <br /> wide variety cellular functions, including electron transport             DXP MEP (27). DXP precursor bio- <br /> (quinones), stabilization membranes (sterols), cell wall bio-             synthesis vitamins B1 B6. Thus, MEP com- <br /> synthesis (dolichols), signaling (prenylated proteins hor-               mitted precursor biosynthesis IPP DMAPP. MEP <br /> mones), photosynthesis (chlorophylls), photoprotection (caro-                converted 2-C-methyl-D-erythritol 2,4-cyclodiphosphate <br /> tenoids), protein synthesis (modified tRNAs). Despite                  (cMEPP) consecutive actions 4-diphosphocytidyl-2- <br /> high degree structural diversity, isoprenoids derived             C-methyl-D-erythritol synthase, 4-diphosphocytidyl-2-C-meth- <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> simple five-carbon molecules, isopentenyl diphos-                   yl-D-erythritol 2-phosphate synthase, cMEPP synthase (22, <br /> phate (IPP) dimethylallyl diphosphate (DMAPP). Al-                       28, 29, 31, 39). A reductive ring-opening cMEPP produces <br /> though higher isoprenoid molecules derived IPP                  1-hydroxy-2-methyl-2-buten-4-yl diphosphate (4, 11, 21). In the <br /> and DMAPP, fundamental building blocks produced                    final step, protein encoded ispH (formerly lytB) converts <br /> by different biosynthetic pathways.                                      1-hydroxy-2-methyl-2-buten-4-yl diphosphate mixture of <br />    In Archaea, Eukarya, gram-positive bacteria, IPP                 IPP DMAPP (2, 3, 13, 32, 38). The segregation the <br /> and DMAPP synthesized acetyl coenzyme A                      MVA MEP pathways makes attractive target <br /> mevalonate (MVA) pathway (Fig. 1). The steps                development antibiotics herbicides. <br /> MVA pathway condense molecules acetyl coenzyme A                       Disruptions genes encoding proteins the <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> form 3-hydroxy-3-methylglutaryl-coenzyme A (HMG-CoA)                      MEP pathway lethal (3, 4, 10, 11, 13, 20, 32). Disruptions <br /> (24, 33). HMG-CoA reduced MVA, con-                      dxs complemented supplementation growth <br /> verted MVA diphosphate steps (5, 8). IPP obtained                  medium deoxyxylulose (DX) 2-C-methyl-D-erythritol <br /> by ATP-dependent decarboxylation MVA diphosphate                    (ME), disruptions dxr complemented ME <br /> then isomerized DMAPP (1, 15). The 2-C-methyl-D-eryth-                    (6, 16, 19, 37, 40, 43). In order utilized MEP <br /> ritol 4-phosphate (MEP) pathway orthogonal MVA                     pathway, exogenous DX ME transported the <br /> pathway (Fig. 1) bacteria plant chlo-               cell phosphorylated. DX converted DXP D-xylu- <br /> roplasts. The step MEP pathway condensation              lokinase, enzyme normally phosphorylates D-xylulose <br />                                                                              (46), point transport mechanism uptake <br />                                                                              DX known. Likewise, mechanism uptake of <br />    * Corresponding author. Mailing address: Department Chemis-            ME established. <br /> try, University Utah, Salt Lake City, UT 84112. Phone: (801) 581- <br /> 6685. Fax: (801) 581-4391. E-mail: poulter@chem.utah.edu. <br />                                                                                 Work laboratory (20) Boronat col- <br />    &#226;&#8364;&#160; Present address: Echelon Biosciences, Inc., Salt Lake City, UT          leagues (10) demonstrated ability plasmid-encoded <br /> 84108.                                                                       MVA operon restore cell viability strains Escherichia <br />  <br />                                                                        473 <br />  474      TESTA ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />                                                                                    1 M MVA lactone 1.02 volumes 1 M KOH followed incubation the <br />                                                                                    mixture 37&#194;&#176;C 30 min. DX prepared previously described (7) and <br />                                                                                    used concentration 2 mM. L-arabinose (Ara) used concentration <br />                                                                                    0.02%. <br />                                                                                       General methods. Minipreparations plasmid DNA restriction analysis <br />                                                                                    using QIAprep Spin Miniprep kits (Qiagen). Large-scale plasmid <br />                                                                                    preparations using Qiagen Plasmid Midi Prep kits. PCR products <br />                                                                                    purified QIAquick PCR purification kits (Qiagen) agarose gels <br />                                                                                    (GIBCO) soaking band overnight TE purification using <br />                                                                                    QIAquick gel extraction kits (Qiagen). Genomic DNA isolated using Easy <br />                                                                                    DNA kits (Invitrogen). Restriction digests, ligations, transformations, elec- <br />                                                                                    troporations performed described Sambrook et al. (41). PCRs were <br />                                                                                    carried using Perkin-Elmer GeneAmp PCR 2400 DNA thermal <br />                                                                                    cycler KlenTaq LA polymerase mix (Clontech), unless noted. <br />                                                                                    Chain reaction cloning (CRC) performed described Pachuk et al. (34). <br />                                                                                    E. coli DH5&#226;?&#163; (GIBCO) used host plasmid constructs. S. enterica <br />                                                                                    serovar Typhimurium strain TR6579 used shuttle plasmids constructed in <br />                                                                                    E. coli S. enterica serovar Typhimurium. Transductions mediated the <br />                                                                                    high-frequency P22 mutant HT105/1 int-201 previously described (42). Phage <br />                                                                                    P22 lysates prepared previously described (14). Random mutagenesis <br />                                                                                    performed described Hughes Roth (23) using TE3461 (17) for <br />                                                                                    Mud-Cam insertions selecting LB-CAM-MVA-Ara LB-CAM. &#226;?&#173;-Red- <br />                                                                                    mediated recombination performed described Price-Carter et al. (36). <br />                                                                                    DNA sequenced Health Sciences Center Sequencing Facility, Eccles <br />                                                                                    Institute Human Genetics, University Utah. Sequence searches per- <br />                                                                                    formed National Center Biotechnology Information WU GSC <br />                                                                                    BLAST server, Genome Sequencing Center, Washington University School of <br />                                                                                    Medicine. <br />              FIG. 1. MVA (left) MEP (right) pathways.                             Plasmid strain construction. Plasmids bacterial strains used this <br />                                                                                    study listed Table 2. All parental S. enterica serovar Typhimurium strains <br />                                                                                    phages used gifts J. Roth (University California&#226;&#8364;&#8221;Davis). <br />                                                                                       Construction pCAT53-3. The SalI fragment pUC4K containing the <br /> coli disrupted dxs. The bacterium blocked                     gene conferring Kanr ligated XhoI site pFCO1 (20) form <br /> in novo biosynthesis IPP DMAPP MEP                                pFCO1/kan. Transformants bearing ligated products selected LB- <br /> pathway able synthesize molecules MVA.                        KAN. The PBAD promoter araC gene excised pBAD/gIII (In- <br />                                                                                    vitrogen) digestion NsiI, treatment Klenow blunt end, diges- <br /> To identify genes involved MEP pathway, chromo-                         tion SacI, followed ligation resulting fragment pFCO1/kan, <br /> somal copy dxs Salmonella enterica serovar Typhimurium                       digested SapI, treated Klenow, digested with <br /> was disrupted synthetic MVA operon (20). For                          SacI form pCAT53-3. <br /> chromosomal disruption, araBAD promoter pBAD                                 Construction pRMC14. S. enterica serovar Typhimurium dxs PCR <br />                                                                                    amplified genomic template, ligated directly pGEM-T (Promega), <br /> was inserted upstream MVA pathway genes a <br />                                                                                    digested Bsu36I EcoRV subsequently treated Klenow give <br /> kanamycin resistance (Kanr) cassette appended downstream                        4-kb fragment. The 7-kb Klenow-treated BsrBI/PspOMI fragment from <br /> allow easy detection insertion mutants. To facilitate cor-                    pCAT53-3 ligated 4-kb fragment pRMC13 create pRMC14 <br /> rect insertion, MVA operon flanked regions                           using CRC. The bridge oligonucleotides crcDXSARA crcKANDXS were <br /> homology dxs. Boronat colleagues (10) described                        used thermal cycler settings denaturation 94&#194;&#176;C 30 s, followed an <br />                                                                                    annealing cycle 52&#194;&#176;C 30 s ligation 66&#194;&#176;C 1 min 30 cycles. <br /> a related construct synthetic MVA operon                             Construction RMC26. An &#226;&#172;&#402;8-kb BsrBI/SpeI fragment pRMC14 was <br /> inserted tryptophan biosynthetic operon E. coli                    electroporated TT22236 containing &#226;?&#173;-Red plasmid pTP223 (35). Linear <br /> chromosomal dxs disrupted chloramphenicol acetyl-                       recombinants selected plating LB-KAN-ME. The correct insertion <br /> transferase (CAT) cassette. We report work impli-                         initially verified phenotypically growth LB-KAN-ME LB-KAN- <br />                                                                                    MVA-Ara lack growth LB-KAN. Subsequent verification was <br /> cates sorbitol phosphotransferase (PTS) the <br />                                                                                    accomplished PCR using primers nested ispA (sISPA1) araC <br /> transport phosphorylation ME (for review PTS,                      (asARAC1) end insertion primers nested yajO (asYAJO1) <br /> reference 45).                                                                     Kan cassette (sKAN1) end insert followed by <br />                                                                                    sequencing PCR products. The genes ispA yajO flank dxs. The strain <br />                                                                                    designated RMC25. The insertion backcrossed LT2 result in <br />                        MATERIALS AND METHODS <br />                                                                                    RMC26, disruption designated dxs::MVAoperon. <br />    Materials. Restriction endonucleases, Klenow fragment, T7 DNA poly-            Random mutagenesis RMC26. Mutagenesis experiments conducted <br /> merase purchased New England Biolabs. Ampicillin, kanamycin              using TE3461 donor strain standard P22 transduction RMC26 as <br /> (KAN), chloramphenicol (CAM) purchased Sigma. Shrimp al-             recipient, resulting random Mud-Cam insertions (17, 23). Mutants were <br /> kaline phosphatase purchased U.S. Biochemical Corp. Ampligase         selected plating transduction crosses directly LB-CAM-MVA-Ara. After <br /> purchased Epicentre Technologies Corporation. KlenTaq-LA DNA poly-            incubation 2 days 37&#194;&#176;C, plates replica printed LB-CAM-ME <br /> merase mix purchased Clontech. Taq/TaqStart DNA polymerase, E.            LB-CAM-MVA-Ara. Colonies exhibiting growth MVA-Ara on <br /> coli DH5&#226;?&#163; competent cells, agarose purchased GIBCO. Oligo-           ME streaked green-CAM-MVA-Ara plates. Nonlysogenic colonies <br /> nucleotides used study listed Table 1 custom synthesized   (white colonies green plates) cross streaked P22 (H5) ensure <br /> by Protein/DNA Core Facility Utah Regional Cancer Center.               phage sensitivity. When phage-sensitive mutants verified, single colonies <br />    Growth media supplements. Luria-Bertani (LB) used       streaked LB-CAM-MVA-Ara plates. When cells grew, plates <br /> supplementation growth conditions (41). KAN used concentra-       replica plated described previously. Cells exhibiting proper pheno- <br /> tion 40 &#226;?&#174;g/ml, CAM used concentration 20 &#226;?&#174;g/ml, ampicillin      type used inoculate liquid cultures, insertion point Cam <br /> was used concentration 100 &#226;?&#174;g/ml. ME [1,1-2H2]2-C-methyl-D-eryth-      cassette determined single-primer (semirandom) PCR genomic <br /> ritol ([1,1-2H2]ME) synthesized described Duvold et al. (16) used   DNA. Semirandom PCR (23) performed using Taq/TaqStart prim- <br /> at concentration 50 &#226;?&#174;g/ml. MVA prepared hydrolysis 1 volume      ers smud33, smud54, smud59, smud86. Thermal cycler settings as <br />  475 <br /> METHYL ERYTHRITOL TRANSPORT IN SEROVAR TYPHIMURIUM <br />  <br />  <br />  <br />  <br />                                                                                                          TABLE 1. Oligonucleotide sequences used study <br />                                                      Oligonucleotide                                                                   Sequencea <br />                                                      crcDXSARA            5&#226;&#172;&#732;   CAATGAATCACGCAGGCGATCTCGCCGCAGCCGAACGACC 3&#226;&#172;&#732; <br />                                                      crcKANDXS            5&#226;&#172;&#732;   TGCAGGTCGAGGGGGGGCCTGAGGGGCGAAAGTCGTAAA 3&#226;&#172;&#732; <br />                                                      sISPA1               5&#226;&#172;&#732;   CGTTAGACTTGGGCGTTGAG 3&#226;&#172;&#732; <br />                                                      asARAC1              5&#226;&#172;&#732;   CTTTTGAGCACCACCCGGAT 3&#226;&#172;&#732; <br />                                                      asYAJO1              5&#226;&#172;&#732;   AGTTAATGCCGCCCTCAAGG 3&#226;&#172;&#732; <br />                                                      sKAN1                5&#226;&#172;&#732;   GGCAGAGCATTACGCTGACT 3&#226;&#172;&#732; <br />                                                      SygbPFRT2            5&#226;&#172;&#732;   GGGCAAACTCATCGATAAACCATCCAGGGATTTAACATGGCAGCCACTTTATTGGACGTTTGCGCCGTGGTGCCGGCGAAGCCACTGGAGCACCTCAA <br />                                                                           3&#226;&#172;&#732; <br />                                                      ASygbPFRT2           5&#226;&#172;&#732;   CCAAACGCGTGTACGTCAAAACCGTGTCCAATTCGCATTATGCTTTCTCCTGGTGGATGGTTCGGGTCAGATAAACGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      sychBcamPCR          5&#226;&#172;&#732;   ATGATGACCCATTGGCCTTCTCCGGCAAAATTAAATCTGTTTTTATATATCACCGGACGAAGCCACTGGAGCACCTCAA 3&#226;&#172;&#732; <br />                                                      asychBcamPCR         5&#226;&#172;&#732;   TTAGAGTAACTCTCGATGCAATGGGGAGAGGTTGACACCCTTCGCCACAAAAGCATTGAGCGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      sychBCamseq2         5&#226;&#172;&#732;   GGTGGCTATGACAGCAAAA 3&#226;&#172;&#732; <br />                                                      asychBCamseq2        5&#226;&#172;&#732;   GGATCCTTACTCGTCTTCATTCAC 3&#226;&#172;&#732; <br />                                                      sygbBcamPCR          5&#226;&#172;&#732;   ATGCGAATTGGACACGGTTTTGACGTACACGCGTTTGGCGGAGAAGGGCCGATTATTATTAAGCCACTGGAGCACCTCAA 3&#226;&#172;&#732; <br />                                                      asygbBcamPCR         5&#226;&#172;&#732;   TCATTTCGCCGCCTTCATCAGCAACGCGACCGCTTCGCAGGCAATGCCTTCTCCGCGTCCCGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      sygbBCamseq2         5&#226;&#172;&#732;   TAAATCTTCAACCACCACAAAATCTTCC 3&#226;&#172;&#732; <br />                                                      asygbBCamseq2        5&#226;&#172;&#732;   GGATCCAATTTTATCTGACCCGAA 3&#226;&#172;&#732; <br />                                                      sgcpEcam2            5&#226;&#172;&#732;   TTATTTTTCAACCTGCAGCACGTCAATCCGACGCGCTTCAAGCCACTGGAGCACCTCAA 3&#226;&#172;&#732; <br />                                                      asgcpEcam2           5&#226;&#172;&#732;   ATGCATAACCAGGCTCCGATTCAACGTAGAAAATCGACACGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      sgcpEseq             5&#226;&#172;&#732;   CCAGGCAGATAATCGTTCATG 3&#226;&#172;&#732; <br />                                                      sgcpEseq             5&#226;&#172;&#732;   GTATCAAGGAAAACCTGTCG 3&#226;&#172;&#732; <br />                                                      smud33               5&#226;&#172;&#732;   ACCTTTGGTCACCAACGCTTTTCC 3&#226;&#172;&#732; <br />                                                      smud54               5&#226;&#172;&#732;   CCGAATAATCCAATGTCC 3&#226;&#172;&#732; <br />                                                      smud59               5&#226;&#172;&#732;   CAGATCCCGAATAATCCAATGTCCT 3&#226;&#172;&#732; <br />                                                      smud86               5&#226;&#172;&#732;   GCAAGCCCCACCAAATCTAATCCCA 3&#226;&#172;&#732; <br />                                                      sgutEFRT2            5&#226;&#172;&#732;   ACTTGAGCAGAAAGTTCATCTGACAGGAGCGGTATCATGACACGGGTTCGCATTGAGAAAGGCGCCGGCGGCTGGGGCAAGCCACTGGAGCACCTCAA <br />                                                                           3&#226;&#172;&#732; <br />                                                      asgutEFRT2           5&#226;&#172;&#732;   ATGGTGGTTTGATAAATCACGCTCATGTCTGAACCTCTTATTGATAGATAAAGCCGGAAACAAACCAGGCGATAAGTACGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      SgutEseq             5&#226;&#172;&#732;   CATATGTCACCGATCTCACCACCGCT 3&#226;&#172;&#732; <br />                                                      ASgutEseq            5&#226;&#172;&#732;   GGATCCGCTCATGTCTGAACCTCTTA 3&#226;&#172;&#732; <br />                                                      Ssr1E                5&#226;&#172;&#732;   CCATGGCACGGGTTCGCATTGAGAAAGGC 3&#226;&#172;&#732; <br />                                                      STYasgutEPCR         5&#226;&#172;&#732;   CGGATTCTTAATGATGATGATGATGATGTTGATAGATAAAGCCGGAAA CAAACC 3&#226;&#172;&#732; <br />                                                      SsrlACam             5&#226;&#172;&#732;   AGAAAACTAAAAACCCTACAGGAGAGAACAATGATCGAAAAAGCCACTGGAGCACCTCAA 3&#226;&#172;&#732; <br />                                                      ASsrlACam            5&#226;&#172;&#732;   CAATGCGAACCCGTGTCATGATACCGCTCCTGTCAGATGACGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      SsrlBCam             5&#226;&#172;&#732;   GGCTTTATCTATCAATAAGAGGTTCAGACATGAGCGTGATAAGCCACTGGAGCACCTCAA 3&#226;&#172;&#732; <br />                                                      ASsrlBCam            5&#226;&#172;&#732;   CAACGGCAACTTGATTCATGTTTTTTCTCCTTAAGCGACACGGGGAGAGCCTGAGCAAA 3&#226;&#172;&#732; <br />                                                      SsrlAX               5&#226;&#172;&#732;   TTGCGATCACAATAACACT 3&#226;&#172;&#732; <br />                                                      ASsrlAX              5&#226;&#172;&#732;   TGTTAGTTGCGCCAGTTT 3&#226;&#172;&#732; <br />                                                      SsrlBX               5&#226;&#172;&#732;   CAGATGCTGATTACGTTTCG 3&#226;&#172;&#732; <br />                                                      ASsrlBX              5&#226;&#172;&#732;   GATTAAAGCGGGAAATCTGC 3&#226;&#172;&#732; <br />                                                       a <br />                                                           Italics indicate homology CAT. <br /> VOL. 186, 2004 <br />  476       TESTA ET AL.                                                                                                                              J. BACTERIOL. <br />  <br />  <br />       TABLE 2. Bacterial strains plasmids used study <br />                                                                       Reference <br />   Strain plasmid                     Description <br />                                                                       source <br />  <br /> E. coli <br />   DH5&#226;?&#163;                      Host cloning vectors                 GIBCO             FIG. 2. Diagram MVA operon inserted dxs. araC and <br />                                                                                      PBAD upstream yeast ERG genes, Kanr cassette is <br /> S. typhimurium                                                                       located downstream. <br />    TR6579                   Host cloning vectors                 J. R. Roth <br />    TT22236                  LT2/pTP223                               J. R. Roth <br />    RMC25                    LT2 dxs::MVA operon                      This work <br />    RMC26                    LT2 dxs::MVA operon                      This work       homology regions flanking disruptions used sequencing. The <br />    RMC28                    RMC26/pTP223                             This work       CAT insertions srlA srlB initially constructed TT22236. After <br />    CR4                      RMC26 ispD::Cam                          This work       verification correct recombination event sequencing, srlA srlB <br />    TE3461                   Mud-cam source                           J. R. Roth      insertions crossed RMC26 standard P22 transduction, resulting in <br />    CR5                      RMC26 srlE::Cam                          This work       strains RMC78 RMC79, respectively. The primers used construct srlA <br />    CR24                     LT2 srlE::Cam                            This work       insertion SsrlACam ASsrlBCam. Primers SsrlBCam ASsrlBCam <br />    CR29                     CR5 pBAD/srlE                            This work       used srlB insertion. The srlA insertion verified using primers <br />    RMC78                    RMC26 srlA::Cam                          This work       SsrlAX ASsrlAX. The analogous primers SsrlBX ASsrlBX used to <br />    RMC79                    RMC26 srlB::Cam                          This work       sequence srlB insertion. Strains RMC26, CR5, RMC78, RMC79 were <br />                                                                                      plated LB-KAN-MVA-Ara LB-KAN-ME verify <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br /> Plasmids                                                                                Partial purification ubiquinone Q8. Ubiquinone-8 (Q8) purified as <br />   pBAD/gIII                 Source araC PBAD                  Invitrogen      described Charon et al. (12) minor modifications. Mass spectrometry was <br />   pGEM-T Easy               TA cloning vector; Ampr                  Promega         performed using positive ion atmospheric pressure chemical ionization mass <br />   pGEM-T                    TA cloning vector; Ampr                  Promega         spectrometry (APCI) positive ion liquid secondary ion mass spectrometry <br />   pTP223                    &#226;?&#173;-Red genes                              J. R. Roth      (LSIMS). APCI performed direct injection sample dissolved in <br />   pFCO1                     ERG genes inserted                    20              1:4:1:1 chloroform-methanol&#226;&#8364;&#8220;n-butanol&#226;&#8364;&#8220;water. A mobile phase 1:1 water- <br />                                pBluescript SK&#226;&#171;&#185;                                       acetonitrile used flow rate 1 ml/min. LSIMS performed using <br />   pUC4K                     Contains Kanr cassette                   Pharmacia       m-nitrobenzyl alcohol matrix argon. <br />   pFCO1/kan                 pFCO1 Kanr cassettes                This work <br />                                pUC4K <br />   pCAT53-3                  MVA operon                               This work                                        RESULTS <br />   pRMC13                    Serovar Typhimurium dxs               This work <br />                                pGEM-T                                                   Disruption chromosomal dxs S. enterica serovar Typhi- <br />   pRMC14                    Plasmid-encoded dxs::MVA                 This work       murium. A dxs mutant S. enterica serovar Typhimurium was <br />                                operon                                                constructed disruption gene synthetic operon <br />   pBAD/myc-his A            Arabinose-induced expression             Invitrogen      containing genes Saccharomyces cerevisiae re- <br />                                vector; Ampr <br />   pBAD/srlE                 srlE cloned pBAD/myc-               This work       sponsible converting MVA IPP (20) control of <br />                                A                                                 PBAD promoter (10) (Fig. 2). pCAT53-3 constructed <br />                                                                                      plasmid pFCO1 adding Kanr cassette 3&#226;&#172;&#732; end <br />                                                                                      yeast ERG8 (phosphomevalonate kinase), ERG12 (MVA <br />                                                                                      kinase), ERG19 (MVA diphosphate decarboxylase) and <br /> follows: initial denaturation 94&#194;&#176;C 30 s, followed 20 cycles high-    adding araC gene PBAD promoter 5&#226;&#172;&#732; end. The <br /> stringency PCR samples heated 94&#194;&#176;C (with hold time)          modified operon (MVAoperon) excised pCAT53-3 <br /> and cooled 53&#194;&#176;C (no hold time), followed 1-min elongation step 72&#194;&#176;C.      used disrupt plasmid-encoded dxs using CRC form <br /> Thirty cycles low-stringency PCR ensued, samples heated to <br /> 94&#194;&#176;C (no hold time) cooled 40&#194;&#176;C (no hold time), followed 1-min <br />                                                                                      pRMC14. The traditional blunt-end ligation MVA <br /> extension 72&#194;&#176;C. Thirty additional cycles high-stringency PCR followed       operon dxs attempted successful. The <br /> described above.                                                                     ligation likely failed size MVAoperon (&#226;&#172;&#402;7 <br />    Insertions identified semirandom PCR verified using standard two-         kb). A stable integration S. enterica serovar Typhi- <br /> primer PCR. In mutants, MEP pathway genes probed insertions <br />                                                                                      murium chromosome generated &#226;?&#173;-Red-mediated ho- <br /> using PCR. <br />    Construction directed knockouts srlE, srlA, srlB, ispD, ispE, ispF,     mologous recombination (23). Selection performed using <br /> ispG RMC26. Disruptions srlE, srlA, srlB, MEP pathway genes           KAN resistance ME auxotrophy. Homologous re- <br /> ispD ispG constructed insertion CAT cassette using &#226;?&#173;-Red-       combination facilitated using pTP223, carries the <br /> mediated linear recombination described disrupting dxs               phage &#226;?&#173; genes encoding 5&#226;&#172;&#732;-3&#226;&#172;&#732; exonuclease (exo) facilitate <br /> MVAoperon. In cases, insertion CAT cassette resulted a <br /> partial gene deletion. All primers designed PCR amplify CAT <br />                                                                                      interstrand invasion DNA (bet) host cell RecBCD- <br /> cassette pACYC184 incorporating approximately 40 bp homology          inactivating protein (gam). This construct synthesizes IPP from <br /> the target end amplificate. The amplificate gel purified   MVA, supplemented Ara, ME. <br /> and used directly transform recipient cells CAM resistance. The srlE          Random mutagenesis RMC26. Mutagenesis per- <br /> and MEP pathway disruptions constructed RMC28 using primers <br />                                                                                      formed phage-mediated random &#226;&#8364;&#339;hopping&#226;&#8364;? Mud-Cam <br /> sgutEFRT2 asgutEFRT2, SygbPFRT2 ASygbPFRT2, sygbBcamPCR <br /> and asygbBcamPCR, sychBcamPCR asychBcamPCR, sgcpEcam2                    element, contains CAT cassette, RMC26 <br /> asgcpEcam2 yield CR3 (srlE::Cam), CR2 (ispD::Cam), CT10 (ispE::Cam),              genome. Mutants exhibited desired phenotype of <br /> CT11 (ispF::Cam), CR20 (ispG::Cam), respectively. When correct inser-          growth MVA growth ME isolated, the <br /> tion verified phenotype (growth MVA-Ara, growth ME), PCR             disrupted gene identified semirandom PCR using prim- <br /> analysis, sequencing, disruptions backcrossed RMC26. The <br /> primers SgutEseq ASgutEseq (srlE::Cam), sygbPseq asygbPseq <br />                                                                                      ers nested CAT cassette. A total mutants exhib- <br /> (ispD::Cam), sychBCamseq2 asychBCamseq2 (ispE::Cam), sygbBCamseq2                iting phenotype isolated, approximately <br /> and asygbBCamseq2 (ispF::Cam), sgcpEseq asgcpEseq (ispG::Cam)                200,000 screened. <br />  VOL. 186, 2004                                          METHYL ERYTHRITOL TRANSPORT IN SEROVAR TYPHIMURIUM                           477 <br />  <br />  <br />                                                                         analyzed positive-ion APCI positive-ion LSIMS. In <br />                                                                         APCI, peak m/z 727 observed corresponding M &#226;&#171;&#185; <br />                                                                         H&#226;&#171;&#185; Q8 (MW &#226;&#171;&#189; 726). The LSIMS spectrum peak at <br />                                                                         m/z 729, reflecting reduction quinone correspond- <br />                                                                         ing quinol mass spectral analysis, previously <br />                                                                         reported (12). <br />                                                                            Purified samples Q8 following incubations were <br />                                                                         analyzed LSIMS demonstrate ME utilization com- <br />                                                                         promised srlE disabled: (i) LT2 incubated LB; <br />    FIG. 3. Genetic evidence SrlE, SrlA, SrlB involved in <br /> the utilization exogenous ME. Growth patterns RMC26, CR5,         (ii) RMC26 incubated [1,1-2H2]ME; (iii) RMC26 was <br /> RMC78, RMC79 LB-KAN-ME (left) LB-KAN-MVA-Ara                 incubated [1,1-2H2]ME MVA-Ara; (iv) CR5 was <br /> (right) shown.                                                      incubated [1,1-2H2]ME MVA-Ara. The LSIMS spec- <br />                                                                         tra shown Fig. 5. APCI spectra Q8 isolated from <br />                                                                         incubation ii peak m/z 743, LSIMS peak was <br />    Disruption srlE, srlA, srlB, MEP pathway genes (ispD          m/z 745, indicating complete incorporation deuterium <br /> and ispG) RMC26 LT2. BLAST searches se-                   isoprenoid chain Q8. Incubation iii gave an <br /> quences obtained semirandom PCR products                APCI peak 743 LSIMS peak 745, indicating that <br /> nine mutants returned sequences srlE. The gene                deuterium-labeled IPP incorporated exclusively the <br /> encoding SrlE protein RMC28 disrupted homol-              Q8 chain. Q8 isolated incubation iv gave APCI <br /> ogous recombination subsequently transduced                    peak 727 LSIMS peak 729, demonstrating no <br /> RMC26 CR5. In addition exhibiting phenotype              deuterium incorporation Q8 srlE disrupted. <br /> of growth MVA-Ara growth ME, CR5 viable                   Characterization remaining insertion mutants. Four <br /> when supplemented DX. Restoration growth DX                  isolated insertion mutations RMC26 mapped srlE. <br /> supplementation indicated endogenous MEP pathway               The remaining resistant semirandom PCR <br /> remained intact suggested mechanism DX                 subsequent sequencing. None remaining mu- <br /> utilization different ME. The srlE::CAT dis-           tants insertions known genes MEP pathway <br /> ruption successfully crossed wild-type LT2. The           probed two-primer PCR. Initial attempts complement <br /> resulting strain, CR24, viable LB-CAM, indicating           mutations S. enterica serovar Typhimurium <br /> srlE essential gene. The srlA srlB genes        genomic library unsuccessful. <br /> disrupted RMC26, resulting strains, RMC78 and <br /> RMC79, exhibited phenotype growth MVA-Ara                                         DISCUSSION <br /> no growth ME. Figure 3 displays growth patterns of <br /> RMC26, CR5, RMC78, RMC79 plated                      Random mutagenesis RMC26 generated mutants <br /> LB-KAN-MVA-Ara LB-KAN-ME. Each strains                       desired phenotype disruptions srlE, which <br /> was viable medium supplemented MVA-                   encodes protein sorbitol PTS. PCR using genomic <br /> Ara, RMC26 viable ME supplementation,                 DNA mutants template confirmed insertion in <br /> indicating SrlA, SrlB, SrlE involved ME             srlE region. Insertion CAT cassette srlE produced <br /> utilization. All directed knockouts RMC26 MEP              MVA-Ara-dependent phenotype RMC26, confirming its <br /> pathway genes resulted desired phenotype growth            involvement ME utilization. During PTS-mediated phos- <br /> MVA-Ara growth ME (unpublished results).                      phorylation, phosphate group PEP transferred to <br />    Q8 isolation analysis. Feeding experiments [1,1-            N1 position histidine residue EI subsequently <br /> 2 <br />   H2]ME established SrlE required uptake uti-           transferred N1 position histidine HPr, with <br /> lization ME cell. Q8 partially purified          steps sugar specific. The phosphate is <br /> following cultures: LT2 grown LB, RMC26 grown LB-                 transferred HPr histidine N3 EIIA the <br /> KAN-[1,1-2H2]ME, RMC26 grown LB-KAN-[1,1-2H2]ME-                     sugar-specific transport complex, followed transfer cys- <br /> MVA-Ara, CR5 grown LB-CAM-[1,1-2H2]ME-MVA-                       teine residue EIIB finally sugar molecule (45). <br /> Ara. LT2 grown LB used control Q8 isolation.            Because coordinated mechanism EIIA, EIIB, and <br /> Q8 derived unlabeled isoprene units molecular                EIIC subunits PTS system, srlA srlB impli- <br /> weight (MW) 726, Q8 derived [1,1-2H2]ME                 cated srlE ME utilization. Insertions CAT <br /> has MW 742 (Fig. 4). Q8 isolated LT2                  cassettes srlA srlB gave MVA-Ara-dependent phe- <br />                                                                         notype similar seen srlE, confirming role all <br />                                                                         genes ME utilization. <br />                                                                            Incorporation ME DX isoprenoids reported <br />                                                                         steps MEP pathway dis- <br />                                                                         covered (6, 19, 37, 40, 43). It clear compounds <br />                                                                         phosphorylated utilized substrates <br />                                                                         MEP pathway. The sorbitol PTS provides mechanism <br />                                                                         import phosphorylation ME. In vitro experiments <br />  FIG. 4. Q8 derived MVA (left) [1,1-2H2]ME (right).       demonstrated D-xylulokinase (encoded xylB), the <br /> MW.                                                                     enzyme responsible phosphorylating D-xylulose, phos- <br />  478     TESTA ET AL.                                                                                                     J. BACTERIOL. <br />  <br />  <br />                                                                     phorylates DX (46), transport mechanism for <br />                                                                     uptake cells reported. The viability CR5 <br />                                                                     DX present growth medium indicates the <br />                                                                     mechanism utilization DX MEP pathway is <br />                                                                     different ME. One presumably identify <br />                                                                     gene (or genes) responsible transporting exogenous <br />                                                                     DX performing similar set experiments described in <br />                                                                     work using DX instead ME screening <br />                                                                     process. <br />                                                                        Involvement srlE gene product utilization of <br />                                                                     exogenously supplied ME isoprenoid biosynthesis ver- <br />                                                                     ified biochemically analysis Q8. S. enterica serovar Ty- <br />                                                                     phimurium strain RMC26 readily incorporated deuterium <br />                                                                     <span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> ME isoprenoid chain Q8. Interest- <br />                                                                     ingly, RMC26, synthesizes isoprenoids ME <br />                                                                     MVA, substantial preference utilization ME <br />                                                                     incubated mixture MVA-Ara deuterium- <br />                                                                     labeled ME. In contrast, incubation CR5, SrlE the <br />                                                                     sorbitol PTS disabled, mixture of <br />                                                                     substrates produced Q8 deuterium chain. <br />                                                                     Thus, functional copy SrlE protein required ME <br />                                                                     utilization. <br />                                                                        The sorbitol PTS complex appears responsible for <br />                                                                     phosphorylation ME. In particular, SrlE protein the <br />                                                                     EIIB subunit sorbitol PTS analogous protein <br />                                                                     transfers phosphate sugar glucose PTS (18). <br />                                                                     Although D-sorbitol ME polyhydroxyl compounds, the <br />                                                                     absolute stereochemistries chiral centers pre- <br />                                                                     cisely map another, clear operon <br />                                                                     responsible utilization sorbitol responsible for <br />                                                                     transport phosphorylation ME. <br />                                                                        In cases overproduction enzymes MEP path- <br />                                                                     way stimulates synthesis isoprenoids (25, 26). For exam- <br />                                                                     ple, higher levels lycopene obtained isopentenyl <br />                                                                     diphosphate isomerase overexpressed (25). The level ly- <br />                                                                     copene increased DXP synthase MEP <br />                                                                     synthase overproduced (44). It possible further <br />                                                                     stimulate synthesis lycopene ME overproduc- <br />                                                                     tion SrlA, SrlE, SrlB proteins DXP <br />                                                                     synthase MEP synthase. <br />                                                                        It notable insertions seen known biosyn- <br />                                                                     thetic genes MEP pathway directed knock- <br />                                                                     outs genes RMC26 gave strains phe- <br />                                                                     notype CR5 (unpublished results). We able to <br />                                                                     specifically disrupt MEP pathway genes beyond <br />                                                                     MEP synthase RMC26, exhibiting phenotype <br />                                                                     growth MVA-Ara growth ME. This would <br />                                                                     imply able identify mutations render <br />                                                                     genes inactive, isolated mutants <br />                                                                     appear involved ME uptake phosphorylation. In <br />                                                                     addition srlE mutants, unknown mutants do <br />                                                                     absolute requirement MVAoperon and <br />                                                                     readily lose recombination crossed a <br />                                                                     tightly linked marker, thiI::Tn10d-Tet. If random <br />                                                                     insertions MEP pathway genes, MVA operon <br />                                                                     essential provide cellular isoprenoids, demon- <br />    FIG. 5. LSIMS spectra Q8 isolated feeding experiments: <br /> (A) wild-type LT2 supplementation; (B) RMC26 supple-        strated inability CR4 lose MVA operon the <br /> mented [1,1-2H2]ME; (C) RMC26 supplemented [1,1-          cross. If assumed 1,000 genes encoded <br /> 2 <br />   H2]ME-MVA-Ara; (D) CR5 supplemented [1,1-2H2]ME-MVA-         S. enterica serovar Typhimurium chromosome, five <br /> Ara.                                                                potential MEP pathway targets screen (ispD <br />  VOL. 186, 2004                                                    METHYL ERYTHRITOL TRANSPORT IN SEROVAR TYPHIMURIUM                                                479 <br />  <br />  <br /> through ispH) belonging proposed ME trans-                                Escherichia coli: fate 3-H 4-H atoms 1-deoxy-D-xylulose. <br />                                                                                           J. Chem. Soc. Chem. Commun. 17:1857&#226;&#8364;&#8220;1858. <br /> port (srlA, srlB, srlE, genes encoding HPr                       20.   Hahn, F. M., L. M. Eubanks, C. A. Testa, B. S. Blagg, J. A. Baker, C. D. <br /> EI), estimated 100 mutants exhibiting phenotype                                 Poulter. 2001. 1-Deoxy-D-xylulose 5-phosphate synthase, gene product <br /> growth MVA-Ara growth ME expected                                   open reading frame (ORF) 2816 ORF 2895 Rhodobacter capsula- <br />                                                                                           tus. J. Bacteriol. 183:1&#226;&#8364;&#8220;11. <br /> for completely random insertions. Our finding                          21.   Hecht, S., W. Eisenreich, P. Adam, S. Amslinger, K. Kis, A. Bacher, D. <br /> mutants 200,000 low. We evidence explain                             Arigoni, F. Rohdich. 2001. Studies nonmevalonate pathway to <br /> this discrepancy.                                                                         terpenes: role GcpE (IspG) protein. Proc. Natl. Acad. Sci. USA <br />                                                                                           98:14837&#226;&#8364;&#8220;14842. <br />                                                                                     22.   Herz, S., J. Wungsintaweekul, C. A. Schuhr, S. Hecht, H. Lu  &#194;&#168;ttgen, S. Sagner, <br />                          ACKNOWLEDGMENTS                                                  M. Fellermeier, W. Eisenreich, M. H. Zenk, A. Bacher, F. Rohdich. 2000. <br />                                                                                           Biosynthesis terpenoids: YgbB protein converts 4-diphosphocytidyl-2-C- <br />   We thank John Roth helpful discussions, Lisa Eubanks pro-                       methyl-D-erythritol 2-phosphate 2-C-methyl-D-erythritol 2,4-cyclodiphos- <br /> viding DXP synthase, David Fox MEP synthase.                                      phate. Proc. Natl. Acad. Sci. USA 97:2486&#226;&#8364;&#8220;2490. <br />   R.M.C. NIH trainee supported grant GM 08537. This                       23.   Hughes, K. T., J. R. Roth. 1988. Transitory cis complementation: a <br /> project supported NIH grant GM 25521.                                              method providing transposition functions defective transposons. Ge- <br />                                                                                           netics 119:9&#226;&#8364;&#8220;12. <br />                                                                                     24.   Jordan, P. M., P. N. Gibbs. 1983. Mechanism action beta-oxoacyl- <br />                                 REFERENCES <br />                                                                                           CoA thiolase rat liver cytosol. Direct evidence order addition <br />  1. Agranoff, B. W., H. Eggerer, U. Henning, F. Lynen. 1959. Isopentenyl              acetyl-CoA molecules formation acetoacetyl-CoA. <br />     pyrophosphate isomerase. J. Am. Chem. Soc. 81:1254&#226;&#8364;&#8220;1255.                              Biochem. J. 213:153&#226;&#8364;&#8220;158. <br />  2. Altincicek, B., E. C. Duin, A. Reichenberg, R. Hedderich, A. K. Kollas, M.      25.   Kajiwara, S., P. D. Fraser, K. Kondo, N. Misawa. 1997. Expression an <br />     Hintz, S. Wagner, J. Wiesner, E. Beck, H. Jomaa. 2002. LytB protein               exogenous isopentenyl diphosphate isomerase gene enhances isoprenoid <br />     catalyzes terminal step 2-C-methyl-D-erythritol-4-phosphate path-          biosynthesis Escherichia coli. Biochem. J. 324:421&#226;&#8364;&#8220;426. <br />     way isoprenoid biosynthesis. FEBS Lett. 532:437&#226;&#8364;&#8220;440.                         26.   Kim, S.-W., J. D. Keasling. 2001. Metabolic engineering nonme- <br />  3. Altincicek, B., A. Kollas, M. Eberl, J. Wiesner, S. Sanderbrand, M. Hintz, E.         valonate isopentenyl diphosphate synthesis pathway Escherichia coli en- <br />     Beck, H. Jomaa. 2001. LytB, novel gene 2-C-methyl-D-erythritol           hances lycopene production. Biotechnol. Bioeng. 72:408&#226;&#8364;&#8220;415. <br />     4-phosphate pathway isoprenoid biosynthesis Escherichia coli. FEBS        27.   Koppisch, A. T., D. T. Fox, B. S. Blagg, C. D. Poulter. 2002. E. coli MEP <br />     Lett. 499:37&#226;&#8364;&#8220;40.                                                                      synthase: steady-state kinetic analysis substrate binding. Biochemistry <br />  4. Altincicek, B., A. K. Kollas, S. Sanderbrand, J. Wiesner, M. Hintz, E. Beck,          41:236&#226;&#8364;&#8220;243. <br />     H. Jomaa. 2001. GcpE involved 2-C-methyl-D-erythritol 4-phos-     28.   Kuzuyama, T., M. Takagi, Kaneda, K., T. Dairi, H. Seto. 2000. Forma- <br />     phate pathway isoprenoid biosynthesis Escherichia coli. J. Bacteriol.           tion 4-(cytidine 5&#226;&#172;&#732;-diphospho)-2-C-methyl-D-erythritol 2-C-methyl- <br />     183:2411&#226;&#8364;&#8220;2416.                                                                        D-erythritol 4-phosphate 2-C-methyl-D-erythritol 4-phosphate cytidylyl- <br />  5. Amdur, B. H., H. Rilling, K. Bloch. 1957. The enzymatic conversion             transferase. Tetrahedron Lett. 41:703&#226;&#8364;&#8220;706. <br />     mevalonic acid squalene. J. Am. Chem. Soc. 79:2646&#226;&#8364;&#8220;2647.                     29.   Kuzuyama, T., M. Takagi, K. Kaneda, H. Watanabe, T. Dairi, H. Seto. <br />  6. Arigoni, D., S. Sagner, C. Latzel, W. Eisenreich, A. Bacher, M. H. Zenk.          2000. Studies nonmevalonate pathway: conversion 4-(cytidine 5&#226;&#172;&#732;- <br />     1997. Terpenoid biosynthesis 1-deoxy-D-xylulose higher plants              diphospho)-2-C-methyl-D-erythritol 2-phospho derivative 4-(cyti- <br />     intramolecular skeletal rearrangement. Proc. Natl. Acad. Sci. USA                     dine 5&#226;&#172;&#732;-diphospho)-2-C-methyl-D-erythritol kinase. Tetrahedron Lett. 41: <br />     94:10600&#226;&#8364;&#8220;10605.                                                                       2925&#226;&#8364;&#8220;2928. <br />  7. Blagg, B. S., C. D. Poulter. 1999. Synthesis 1-deoxy-D-xylulose      30.   Lange, B. M., M. R. Wildung, D. McCaskill, R. Croteau. 1998. A family <br />     1-Deoxy-D-xylulose-5-phosphate. J. Org. Chem. 64:1508&#226;&#8364;&#8220;1511.                           transketolases directs isoprenoid biosynthesis mevalonate-inde- <br />  8. Bloch, K., S. Chaykin, A. H. Phillips, A. Waard. 1959. Mevalonic acid          pendent pathway. Proc. Natl. Acad. Sci. USA 95:2100&#226;&#8364;&#8220;2104. <br />     pyrophosphate isopentenyl pyrophosphate. J. Biol. Chem. 234:2595&#226;&#8364;&#8220;           31.   Lu&#194;&#168;ttgen, H., F. Rohdich, S. Herz, J. Wungsintaweekul, S. Hecht, C. A. <br />     2604.                                                                                 Schuhr, M. Fellermeier, S. Sagner, M. H. Zenk, A. Bacher, W. Eisen- <br />  9. Bochar, D. A., J. A. Freisen, C. V. Stauffacher, V. W. Rodwell. 1999.             reich. 2000. Biosynthesis terpenoids: YchB protein Escherichia coli <br />     Biosynthesis mevalonic acid acetyl-CoA, p. 15&#226;&#8364;&#8220;44. In D. Cane (ed.),           phosphorylates 2-hydroxy group 4-diphosphocytidyl-2-C-methyl-D- <br />     Comprehensive natural products chemistry, vol. 2. Pergamon Press, Oxford,             erythritol. Proc. Natl. Acad. Sci. USA 97:1062&#226;&#8364;&#8220;1067. <br />     United Kingdom. <br />                                                                                     32.   McAteer, S., A. Coulson, N. McLennan, M. Masters. 2001. The lytB gene <br /> 10. Campos, N., M. Rodriguez-Concepcion, S. Sauret-Gueto, F. Gallego, L. M. <br />                                                                                           Escherichia coli essential specifies product needed isoprenoid <br />     Lois, A. Boronat. 2001. Escherichia coli engineered synthesize iso- <br />                                                                                           biosynthesis. J. Bacteriol. 183:7403&#226;&#8364;&#8220;7407. <br />     pentenyl diphosphate dimethylallyl diphosphate mevalonate: a <br />                                                                                     33.   Miziorko, H. M., M. D. Lane. 1977. 3-Hydroxy-3-methylgutaryl-CoA <br />     novel genetic analysis 2-C-methyl-D-erythritol 4-phos- <br />                                                                                           synthase. Participation acetyl-S-enzyme enzyme-S-hydroxymethylgu- <br />     phate pathway isoprenoid biosynthesis. Biochem. J. 353:59&#226;&#8364;&#8220;67. <br />                                                                                           taryl-SCoA intermediates reaction. J. Biol. Chem. 252:1414&#226;&#8364;&#8220;1420. <br /> 11. Campos, N., M. Rodriguez-Concepcion, M. Seemann, M. Rohmer, A. <br />     Boronat. 2001. Identification gcpE novel gene 2-C-methyl-D-      34.   Pachuk, C. J., M. Samuel, J. A. Zurawski, L. Snyder, P. Phillips, C. <br />     erythritol 4-phosphate pathway isoprenoid biosynthesis Escherichia             Satishchandran. 2000. Chain reaction cloning: one-step method direc- <br />     coli. FEBS Lett. 488:170&#226;&#8364;&#8220;173.                                                         tional ligation multiple DNA fragments. Gene 243:19&#226;&#8364;&#8220;25. <br /> 12. Charon, L., J. F. Hoeffler, C. Pale-Grosdemange, L. M. Lois, N. Campos, A.      35.   Poteete, A. R., A. C. Fenton. 2000. Genetic requirements phage <br />     Boronat, M. Rohmer. 2000. Deuterium-labelled isotopomers 2-C-                  lambda red-mediated gene replacement Escherichia coli K-12. J. Bacteriol. <br />     methyl-D-erythritol tools elucidation 2-C-methyl-D-eryth-           182:2336&#226;&#8364;&#8220;2340. <br />     ritol 4-phosphate pathway isoprenoid biosynthesis. Biochem J. 346:737&#226;&#8364;&#8220;      36.   Price-Carter, M., J. Tingey, T. A. Bobik, J. R. Roth. 2001. The alterna- <br />     742.                                                                                  tive electron acceptor tetrathionate supports B12-dependent anaerobic <br /> 13. Cunningham, F. X., Jr., T. P. Lafond, E. Gantt. 2000. Evidence role          growth Salmonella enterica serovar Typhimurium ethanolamine or <br />     LytB nonmevalonate pathway isoprenoid biosynthesis. J. Bac-             1,2-propanediol. J. Bacteriol. 183:2463&#226;&#8364;&#8220;2475. <br />     teriol. 182:5841&#226;&#8364;&#8220;5848.                                                          37.   Putra, S. R., L. M. Lois, N. Campos, A. Boronat, M. Rohmer. 1998. <br /> 14. Davis, R. W., D. Bolstein, J. R. Roth. 1980. Advanced bacterial genetics.         Incorporation [2,3&#226;&#8364;&#8220;13C2]- [2, 4&#226;&#8364;&#8220;13C2]-D-1-deoxyxylulose ubiqui- <br />     Cold Spring Harbor Laboratory, Cold Spring Harbor, New York.                          Escherichia coli mevalonate-independent pathway iso- <br /> 15. Dhe-Paganon, S., J. Magrath, R. H. Abeles. 1994. Mechanism meva-               prenoid biosynthesis. Tetrahedron Lett. 39:23&#226;&#8364;&#8220;26. <br />     lonate pyrophosphate decarboxylase: evidence carbocationic transition     38.   Rohdich, F., S. Hecht, K. Gartner, P. Adam, C. Krieger, S. Amslinger, D. <br />     state. Biochemistry 33:13355&#226;&#8364;&#8220;13362.                                                   Arigoni, A. Bacher, W. Eisenreich. 2002. Studies nonmevalonate <br /> 16. Duvold, T., P. Cali, J.-M. Bravo, C. Pale-Grosdemange, M. Rohmer.                 terpene biosynthetic pathway: metabolic role IspH (LytB) protein. Proc. <br />     1997. Incorporation 2-C-methyl-D-erythritol, putative isoprenoid inter-          Natl. Acad. Sci. USA 99:1158&#226;&#8364;&#8220;1163. <br />     mediate mevalonate-independent pathway, ubiquinone              39.   Rohdich, F., J. Wungsintaweekul, M. Fellermeier, S. Sagner, S. Herz, K. Kis, <br />     menaquinone Escherichia coli. Tetrahedron Lett. 38:4769&#226;&#8364;&#8220;4772.                      W. Eisenreich, A. Bacher, M. H. Zenk. 1999. Cytidine 5&#226;&#172;&#732;-triphosphate- <br /> 17. Elliot, T. 1993. Transport 5-aminolevulinic acid dipeptide permease         dependent biosynthesis isoprenoids: YgbP protein Escherichia coli <br />     Salmonella typhimurium. J. Bacteriol. 175:325&#226;&#8364;&#8220;331.                                 catalyzes formation 4-diphosphocytidyl-2-C-methylerythritol. Proc. <br /> 18. Erni, B., H. Trachsel, P. W. Postma, J. P. Rosenbusch. 1982. Bacterial            Natl. Acad. Sci. USA 96:11758&#226;&#8364;&#8220;11763. <br />     phosphotransferase system. Solubilization purification glucose-      40.   Sagner, S., C. Latzel, W. Eisenreich, A. Bacher, M. H. Zenk. 1998. Dif- <br />     specific enzyme II membranes Salmonella typhimurium. J. Biol.                 ferential incorporation 1-deoxy-D-xylulose monoterpenes caro- <br />     Chem. 257:13726&#226;&#8364;&#8220;13730.                                                                tenoids higher plants. J. Chem. Soc. Chem. Commun. 2:221&#226;&#8364;&#8220;222. <br /> 19. Giner, J. L., B. Jaun, D. Arigoni. 1998. Biosynthesis isoprenoids     41.   Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />  480      TESTA ET AL.                                                                                                                         J. BACTERIOL. <br />  <br />     laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring            mevalonate isopentenyl diphosphate pathway Escherichia coli enhances <br />     Harbor, N.Y.                                                                     lycopene production. Biotechnol. Bioeng. 72:408&#226;&#8364;&#8220;415. <br /> 42. Schmieger, H. 1972. Phage P22 mutants increased decreased trans-     45. Tchieu, J. H., V. Norris, J. S. Edwards, M. H. Saier, Jr. 2001. The <br />     duction abilities. Mol. Gen. Genet. 119:75&#226;&#8364;&#8220;88.                                   complete phosphotranferase Escherichia coli. J. Mol. Microbiol. <br /> 43. Schwender, J., J. Zeidler, R. Groner, C. Muller, M. Focke, S. Braun, F. W. <br /> </body></html>