<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Experimental selection hypoxia-tolerant <br /> Drosophila melanogaster <br /> Dan Zhoua, Nitin Udpab,1, Merril Gerstenb,1, DeeAnn W. Viska,c,1, Ali Bashirb, Jin Xuea, Kelly A. Frazera,b, <br /> James W. Posakonyb,c, Shankar Subramaniamb,d, Vineet Bafnab,e, Gabriel G. Haddada,f,g,2 <br /> a <br />  Department Pediatrics, University California San Diego, La Jolla, CA 92093; bBioinformatics Systems Biology Graduate Program, University of <br /> California San Diego, La Jolla, CA 92093; cDivision Biological Sciences, Section Cell Developmental Biology, University California San Diego, <br /> La Jolla, CA 92093; dDepartment Bioengineering, University California San Diego, La Jolla, CA 92092; eDepartment Computer Science and <br /> Engineering, University California San Diego, La Jolla, CA 92092; fDepartment Neuroscience, University California San Diego, La Jolla, CA 92093; <br /> and gDivision Pulmonology, Rady Children&#226;&#8364;&#8482;s Hospital, San Diego, CA 92123 <br />  <br /> Edited* Shu Chien, University California San Diego, La Jolla, CA, approved December 21, 2010 (received review July 21, 2010) <br />  <br /> Through long-term laboratory selection (over 200 generations),                 underlies adaptation. We discovered a) genetic mech- <br /> have generated Drosophila melanogaster populations tolerate                  anisms allowed D. melanogaster adapt tolerate extremely <br /> severe, normally lethal, levels hypoxia. Because initial experi-            low O2 environments, b) genomic intervals were <br /> ments suspecting genetic mechanisms underlying adaptation,                   selected for, X chromosome, occurred during <br /> we compared genomes hypoxia-selected &#239;&#172;&#8218;ies                   long-term hypoxia, c) Notch pathway played important <br /> of controls using deep resequencing. By applying unique comput-                   role hypoxia tolerance. <br /> ing analytical methods identi&#239;&#172;?ed number DNA regions <br /> under selection, X chromosome. Several hyp-                  Results <br /> oxia-selected regions contained genes encoding regulating                  Hypoxia-Selected Regions Genetic Pro&#239;&#172;?les AF Genome. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> <br /> Notch pathway. In addition, previous expression pro&#239;&#172;?ling revealed                 determine DNA signatures hypoxia selec- <br /> an activation Notch pathway hypoxia-selected &#239;&#172;&#8218;ies.                  tion, sequenced control (C1 C2) AF (H1 and <br /> We con&#239;&#172;?rmed contribution Notch activation hypoxia                       H2) populations hypoxia selection 180 <br /> tolerance using speci&#239;&#172;?c &#206;&#179;-secretase inhibitor, N-[N-(3,5-Di&#239;&#172;&#8218;uoro-               generations separate environmental chambers &gt;60&#195;&#8212; cov- <br /> phenacetyl)-L-alanyl]-S-phenylglycine t-butyl ester (DAPT),                 erage, using Illumina GA II sequencer. We aligned between <br /> signi&#239;&#172;?cantly reduced adult <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span> life span hypoxia-                 120 million 200 million 54-bp paired-end reads pop- <br /> selected &#239;&#172;&#8218;ies. We demonstrated &#239;&#172;&#8218;ies loss-of-function               ulation D. melanogaster reference sequence (Table S1). <br /> Notch mutations RNAi-mediated Notch knockdown signif-                    Because individual genotypes number individuals <br /> icant reduction hypoxia tolerance, gain-of-func-              sampled given <span id='am-315' about='xsp:base' typeof='owl:Thing'>base</span> standard linkage disequilibrium <br /> tion dramatic opposite effect. Using UAS-Gal4 system,                (LD) information determined pooled se- <br /> also showed speci&#239;&#172;?c overexpression Notch intracellular                quence data, standard tests selection used (3, 4). <br /> domain glial cells critical conferring hypoxia tolerance.              Consequently, used complementary approaches de- <br /> Unique analytical tools genetic bioinformatic strategies                  termine hypoxia-selected regions genetic pro&#239;&#172;?les the <br /> allowed discover Notch activation plays major role                AF genome. <br /> this hypoxia tolerance Drosophila melanogaster.                                   In &#239;&#172;?rst case, used coarse-grained approach compare <br />                                                                                   SNP distributions control AF pools. In approach, <br /> evolution   | next-generation sequencing                                          developed unique statistic determine potential regions <br />                                                                                   selection using sequence data generated the <br />  <br /> O     xygen homoeostasis essential development, growth, <br />       integrity cells, tissues, organisms. Limited oxygen <br /> supply cells tissues (hypoxia) wide range physio- <br />                                                                                   pooled populations. Sf(C1, H1) log ratio control AF <br />                                                                                   scaled <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>mutation</span> rates provides comparison effective <br />                                                                                   population sizes. We identi&#239;&#172;?ed Sf(C1, H1) &gt;&gt; 0 indicative of <br /> logic potentially pathologic consequences, ranging                       deviation neutrality consistent purifying selection <br /> ischemic/hypoxic heart disease, stroke, pulmonary hyper-                      hypoxia (reduction effective population size). We used Sf(C1, <br /> tension number obstetrical/perinatal complications,                    C2) empirical control false discovery rate (FDR) com- <br /> high-altitude illnesses, organ transplantation, &#239;&#172;?nally                  putation. We investigated concordance AF <br /> intratumor hypoxia cancer progression. Despite clinical                   populations different chambers comparing in- <br /> importance societal disease impact wide range                    dependent estimates Sf(C1, H1) Sf(C1, H2). Using Sf(C1, <br /> disorders, molecular underpinnings susceptibility toler-                H1) cutoff 4, corresponding FDR &#226;&#710;&#188;1%, over- <br /> ance cells tissues lack O2 understood. Many              lapping 50-kbp windows, observed remarkable concordance <br /> studies investigated mechanisms lead injury                 regions selection H1 H2. A total of <br /> cells deprived O2, potentially treat prevent                 1,509,436 bp comprising 24 distinct hypoxia-selected regions and <br /> consequences hypoxia necessitates understanding of <br />                                                                                                                                                                                  GENETICS <br />  <br />  <br />  <br />  <br />                                                                                   containing total 188 genes selection H1 <br /> the inherent tissue mechanisms critical tolerance and <br /> survival. To so, use long-term laboratory selection strategy <br /> that unmasks mechanisms play important role hypoxia                    Author contributions: D.Z., A.B., K.A.F., S.S., V.B., G.G.H. designed research; D.Z. and <br /> tolerance genetic model, Drosophila melanogaster (1, 2). In                  D.W.V. performed research; D.Z., J.W.P., S.S., V.B., G.G.H. contributed new reagents/ <br /> this attempt, starting 27 isofemale D. melanogaster strains,                 analytic tools; D.Z., N.U., M.G., D.W.V., V.B. analyzed data; D.Z., N.U., M.G., <br /> and applying decreasing levels O2 &gt;200 generations,                    D.W.V., J.X., K.A.F., S.S., V.B., G.G.H. wrote paper. <br />  <br /> generated Drosophila populations tolerate severe levels                   The authors declare con&#239;&#172;&#8218;ict interest. <br /> hypoxia, lethal original parental lines. These                   *This Direct Submission article prearranged editor. <br /> hypoxia-adapted &#239;&#172;&#8218;ies (AF) pass tolerance trait generation                1 <br />                                                                                   N.U., M.G., D.W.V. contributed equally work. <br /> to generation trait persists absence hypoxic               2 <br />                                                                                   To correspondence addressed. E-mail: ghaddad@ucsd.edu. <br /> stress (i.e., generations normoxic environment),               This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> suggesting genetic physiological mechanism                   1073/pnas.1010643108/-/DCSupplemental. <br />  <br />  <br />  <br /> www.pnas.org/cgi/doi/10.1073/pnas.1010643108                                                           PNAS | February 8, 2011 | vol. 108 | no. 6 | 2349&#226;&#8364;&#8220;2354 <br />  and H2. Twenty regions (&gt;80%) located                     identi&#239;&#172;?ed entire extended gene region, introns <br /> 10-Mb interval chromosome X remaining 4 lo-                      containing largest number SNPs indels (Fig. 1H). <br /> cated 1.2-Mb interval chromosome 3R (Fig. 1 A&#226;&#8364;&#8220;D <br /> and Fig. S1). These results demonstrate populations                  Candidate Genes Underlying Hypoxia Tolerance AF. There 188 <br /> &#239;&#172;&#8218;ies, independently selected hypoxia tolerance (i.e., differ-             genes located hypoxia-selected regions (Table S2). To <br /> ent environmental chambers) intervals ge-                    identify set potentially causal genes hypoxia tolerance, <br /> nome undergo high degree &#239;&#172;?xation (hypoxia-selected regions)                 started initially 28 genes previously <br /> and suggest genes required adaptation severe                     implicated hypoxia similar phenotypes (such oxi- <br /> hypoxic conditions localized distributed              dative stress aging). We &#239;&#172;?ltered remaining 160 genes for <br /> genome. The observation reinforced distribution              evidence selective sweep using complementary tests: i) <br /> C1 vs. H1 H2 &#239;&#172;?xed SNPs genome: Whereas                   McDonald&#226;&#8364;&#8220;Kreitman test H1 C1 evidence of <br /> populations median value 28 &#239;&#172;?xed SNPs (range 0&#226;&#8364;&#8220;363)                    adaptive evolution based correlation &#239;&#172;?xed (i.e., &gt;90% <br /> per 50-kb interval genome, large difference be-                frequency) nonsynonymous mutations population (P &#226;&#8240;&#164; <br /> tween C1 H1 H2 hypoxia-selected regions                     0.05) (5), ii) &#226;&#8240;&#165;1.5-fold transcriptomic change hypoxia (1), <br /> the AF populations threefold higher &#239;&#172;?xed SNPs compared                      iii) sorting intolerant tolerant (SIFT) evaluation the <br /> with control &#239;&#172;&#8218;ies (&#226;&#710;&#188;74; Poisson P = 2.57 &#195;&#8212; 10&#226;&#710;&#8217;13). Further-                  impact &#239;&#172;?xed, nonsynonymous mutations functions of <br /> more, conservation H1 H2 &#239;&#172;?xed SNPs                       proteins encoded genes regions (P &#226;&#8240;&#164; 0.05) (6). A <br /> higher hypoxia-selected regions compared                total 68 genes 160 genes identi&#239;&#172;?ed of <br /> non-hypoxia&#226;&#8364;&#8220;selected regions (93% vs. 78%; hypergeometric P =                    observed using &#239;&#172;?ne-granularity assessment. <br /> 3.12 &#195;&#8212; 10&#226;&#710;&#8217;43). Consistent hypoxic stress-mediated pop-                    These include 12 genes interact with, activate, targets <br /> ulation bottleneck leading overall loss diversity AF             Notch pathway (Table S2). For example, two <br /> populations, observed higher genetic similarity be-                    members Notch repressor complex, i.e., Hairless and <br /> tween H1 H2 (66% &#239;&#172;?xed SNPs common), com-                          HDAC4, located hypoxia-selected regions AF. <br /> pared concordance 35&#226;&#8364;&#8220;37% AF control                        HDAC4 contains SNP &#239;&#172;?xed AF populations sig- <br /> lines, despite identical ancestry (Fig. 1 E F).                          ni&#239;&#172;?cant impact function (SIFT P value = 0.05) virtually <br />    In second case, used &#239;&#172;?ne-grained approach fo-                    absent control populations (A1004S isoform B). The <br /> cused analysis loci high-con&#239;&#172;?dence allelic differ-               &#239;&#172;?ne-grained genome-wide analysis identi&#239;&#172;?ed additional poly- <br /> ences (between control AF) considered likely represent                    morphisms Notch pathway-related genes, including Notch, <br /> allele selection linkage selected alleles AF pop-               Delta, fringe, sgg (Fig. 2) (Table S3 Table S4). <br /> ulations. This analysis limited 45% euchromatic                   Because a) number genes Notch pathway <br /> genome (including 70&#226;&#8364;&#8220;75% exon loci) &#226;&#8240;&#165;20&#195;&#8212; coverage                    related Notch signaling selected AF con <br /> high base-calling quality. This approach identi&#239;&#172;?ed SNPs small                trast control &#239;&#172;&#8218;ies b) previous expression pro&#239;&#172;?ling <br /> indels loci AF populations differed                   studies demonstrated Notch pathway activated AF <br /> the control reference alleles. This approach con&#239;&#172;?rmed                   &#239;&#172;&#8218;ies comparison control &#239;&#172;&#8218;ies (1), focused investigation <br /> of &#239;&#172;?ndings described above. The majority detected high-                   role Notch pathway hypoxia tolerance. <br /> con&#239;&#172;?dence polymorphisms occurred chromosome X (959 <br /> SNPs, 259 indels), chromosome 3R (371 SNPs, 318 indels)                     Notch Activation Is Critical Hypoxia Tolerance Drosophila <br /> containing largest number. Also detected signi&#239;&#172;?-                 melanogaster. To dissect contribution Notch activation to <br /> cant number polymorphisms chromosome 3L compar-                        hypoxia tolerance, used genetic tools pharmacologic <br /> atively chromosomes 2L 2R (Fig. 1G). Polymorphisms                    agents. We &#239;&#172;?rst examined role Notch hypoxia tolerance <br />  <br />  <br />  <br />  <br /> Fig. 1. Genomic regions selection hypoxia tolerance. The Sf statistic computed overlapping 50-kbp windows, comparing hypoxia <br /> samples (H1 H2) normoxia (C1). Regions Sf values &gt;4 shown, corresponding 1% FDR (based comparing normoxia samples). The <br /> results chromosomes (A) X (B) 3R near perfect concordance H1 vs. C1 (red) H2 vs. C2 (green, inverted). Regions deemed signi&#239;&#172;?cant <br /> for hypoxic samples shown blue intervals. (C) Regions 6 7, including genes Fur2, mei-41, hang, para, (D) region 24, containing <br /> Hairless, shown greater detail. Note value statistic plotted median region decays goes edge the <br /> selected region. (E F) Venn diagrams, describing frequency &#239;&#172;?xed SNPs (per 50 kbp) (E) just regions selection (F) entire genome, <br /> were tabulated. Orange represents H1, green represents H2, blue represents C1. (G H) Bar graphs indicating distribution SNPs small indels <br /> in extended gene regions chromosome gene region. Loci falling gene region gene counted gene. <br />  <br />  <br /> 2350 | www.pnas.org/cgi/doi/10.1073/pnas.1010643108                                                                                                 Zhou et al. <br />                                                                                     hypoxia tolerance (12, 13). Several available GAL4 lines were <br />                                                                                    crossed UAS&#226;&#8364;&#8220;Notch intracellular domain (UAS-NICD) <br />                                                                                    transgenic stock generate progeny speci&#239;&#172;?c Notch <br />                                                                                    activation speci&#239;&#172;?c cells/tissues speci&#239;&#172;?c developmen- <br />                                                                                    tal stages (Table S5). The results showed speci&#239;&#172;?c expression <br />                                                                                    NICD neurons and/or glial cells conferred hypoxia <br />                                                                                    survival progeny. For example, progeny derived <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                    crosses NICD up-regulated speci&#239;&#172;?c subset of <br />                                                                                    glial cells showed remarkable increase hypoxia tolerance and <br />                                                                                    survival. As shown Fig. 4, Fig. S2, Table S5, Eaat- <br />                                                                                    GAL4 P{GawB}17A-Gal4&#226;&#8364;&#8220;driven NICD overexpression in <br />                                                                                    glial cells signi&#239;&#172;?cantly enhanced eclosion rate adult life <br />                                                                                    span (after eclosion) hypoxia. <br />                                                                                       To explore mechanisms underlying Notch-mediated <br />                                                                                    hypoxia tolerance, tested role Notch downstream <br />                                                                                    gene, E(spl)m&#206;&#177;, located E(spl) genomic region and <br />                                                                                    signi&#239;&#172;?cantly up-regulated hypoxia-selected &#239;&#172;&#8218;ies (1). <br />                                                                                    We generated &#239;&#172;?rst homozygous &#239;&#172;&#8218;y use subsequent cross <br />                                                                                    provide Notch overexpression background [i.e., (Eaat1- <br />                                                                                    GAL4/Eaat1-GAL4; UAS-NICD/UAS-NICD) stock, EN line]. <br />                                                                                    This EN line crossed homozygous &#239;&#172;&#8218;ies carry <br />                                                                                    UAS-E(spl)m&#206;&#177;RNAi. The progeny cross carried Eaat1- <br />                                                                                    GAL4, UAS-NICD, UAS-E(spl)m&#206;&#177;RNAi allowed to <br />                                                                                    knock target E(spl)m&#206;&#177; gene Notch overex- <br />                                                                                    pression background determine Notch activation requires <br />                                                                                    E(spl)m&#206;&#177; regulate hypoxia tolerance. As shown Fig. 5, Notch <br />                                                                                    activation, conferred hypoxia tolerance RNAi <br />                                                                                    E(spl)m&#206;&#177;, totally abolished E(spl)m&#206;&#177; knockdown, <br />                                                                                    demonstrating critical role E(spl)m&#206;&#177; regulating Notch- <br />                                                                                    induced hypoxia tolerance. <br /> Fig. 2. Enrichment &#239;&#172;?xed SNPs Indels extended Notch pathway. <br /> The Notch pathway adapted KEGG (42) adding Notch inter-                Discussion <br /> actors literature create expanded Notch signaling pathway. <br /> Genes differentially expressed larva (expression levels Zhou et al.) (1) <br />                                                                                    D. melanogaster used powerful genetic model for <br /> are cyan (up-regulated) yellow (down-regulated), genes showing               century. Because genes pathways evolu- <br /> change expression gray, untested genes white. Genes             tionarily conserved D. melanogaster humans, Dro- <br /> which SNP and/or indels &#239;&#172;?xed indicated stars: red         sophila effective tools dissecting <br /> for genes located hypoxia-selected region blue others.        genetic mechanisms human diseases, including develop- <br />                                                                                    mental neurological disorders, cancer, cardiovascular disease, <br />                                                                                    metabolic storage diseases (for selected reviews refs. <br /> using homozygous-viable Notch mutants: N[Ax-tsl], N[fa-1],                   14&#226;&#8364;&#8220;17). The current study taken advantage Drosophila <br /> and N[spl-1] (7&#226;&#8364;&#8220;9). The N[fa-1] mutation caused insert-                   model used population genetics, deep-sequencing, mo- <br /> ion transposable element (opus) second intron                   lecular strategies identify potential causative gene(s) underlying <br /> Notch gene, N[Ax-tsl] generated ethyl methanesulfonate-                  hypoxia tolerance. The use experimental selection methods fol- <br /> induced mutagenesis N[fa-1] N[Ax-tsl] having loss-                   lowed deep sequencing pooled individuals unique ex- <br /> of-function mutations. Unlike N[fa-1] N[Ax-tsl], N[spl-1]                   perimental technique right. In recent publication (18), <br /> a Notch gain-of-function allele carrying <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>point mutation</span> EGF                   similar evolutionary experiment performed, dif- <br /> repeat 14 Notch protein replaces Isoleucine578                ferent <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Their analysis (different ours) does not <br /> a Threonine (10). We N[fa-1] N[Ax-tsl] hy-                     point speci&#239;&#172;?c regions genome-wide signature se- <br /> persensitive hypoxia lowered survival rate,                   lection. Rather, suggest adaptation &#226;&#8364;&#339;incom- <br /> much milder hypoxic conditions (i.e., 6% O2) (Fig. 3A). In con-                    plete sweep models.&#226;&#8364;? By contrast, experiments analysis <br /> trast, N[spl-1] exhibited remarkable hypoxia tolerance sur-                    provide de&#239;&#172;?nitive evidence selective sweep. Speci&#239;&#172;?- <br /> vived 4% O2, like AF &#239;&#172;&#8218;ies. We used RNAi                           cally, identi&#239;&#172;?ed 24 distinct hypoxia-selected regions con- <br /> strategy determined &#239;&#172;&#8218;ies knockdown Notch                    taining total 188 genes hypoxia-selected population <br /> a hypoxia-sensitive phenotype (Fig. 3B). These results clearly in-                 compared naive ones. Interestingly, 20 re- <br /> dicate Notch function critical survival hypoxia.                 gions (&gt;80%) located chromosome X. Because Drosoph- <br />    Because up-regulation genes encoding aph-1 nct                       ila males hemizygous X chromosome possibility <br />                                                                                                                                                            GENETICS <br />  <br />  <br />  <br />  <br /> subunits &#206;&#179;-secretase suggested activation Notch signaling               recombination, recessive X-linked alleles readily available <br /> in AF &#239;&#172;&#8218;ies involve &#206;&#179;-secretase, used speci&#239;&#172;?c                        selection males. A consequence rapid accu- <br /> &#206;&#179;-secretase inhibitor, N-[N-(3,5-Di&#239;&#172;&#8218;uorophenacetyl)-L-alanyl]-S-                   mulation favorable mutations stronger purifying se- <br /> phenylglycine t-butyl ester (DAPT) (11) examined life                    lection deleterious recessive alleles X chromosome rel- <br /> span severe hypoxia AF population. We DAPT                    ative autosomes (19&#226;&#8364;&#8220;21). <br /> treatment reduced signi&#239;&#172;?cantly median maximum                         The hypoxia-selected regions hypoxia-selected &#239;&#172;&#8218;ies <br /> life span AF &#239;&#172;&#8218;ies (Fig. 3C) signi&#239;&#172;?cant effect                 contain genes belong Notch signaling pathway <br /> control &#239;&#172;&#8218;ies.                                                                      and/or regulate activity Notch. For example, Hairless <br />                                                                                    HDAC4 important Notch signaling down- <br /> Spatial&#226;&#8364;&#8220;Temporal Activation Notch Its Downstream Genes                   stream genes Notch pathway activated binding <br /> Hypoxia Tolerance. The UAS/GAL4 used determine                       NICD DNA-binding complex CSL (22). NICD <br /> the critical spatial&#226;&#8364;&#8220;temporal activation Notch pathway                   competes CSL binding corepressor complex, which <br />  <br /> Zhou et al.                                                                                            PNAS | February 8, 2011 | vol. 108 | no. 6 | 2351 <br />                                     A                                                                    C <br />  <br />  <br />  <br />  <br />                                    B                     da-GAL4 <br />                                                          UAS-N.dsRNA <br />                                                          (UAS-N.dsRNA virgin female) x (da-GAL4 male) <br />                                                    120   (da-GAL4 virgin female) x (UAS-N.dsRNA male) <br />  <br />                                                    100                                                                       Median        Maximum <br />                                Eclosion Rate (%) <br />  <br />  <br />  <br />  <br />                                                                                                                             Lifespan       Lifespan <br />                                                     80                                                      0.00 mM DAPT <br />                                                                                                                            15.2 &#194;&#177; 1.04    20.1 &#194;&#177; 1.25 <br />                                                                                                                 (n=300) <br />                                                     60 <br />                                                                                                             0.25 mM DAPT <br />                                                                                                                            12.3 &#194;&#177; 0.58*   15.1 &#194;&#177; 0.55** <br />                                                     40                                                          (n=300) <br />                                                                                                             0.50 mM DAPT <br />                                                     20                                                                     11.7 &#194;&#177; 0.58*   13.1 &#194;&#177; 0.72** <br />                                                                                                                 (n=300) <br />  <br />                                                     0 <br />                                                          Room Air                 6% O2 <br />  <br /> Fig. 3. Hypoxia tolerance Notch mutants &#206;&#179;-secretase inhibitor-treated &#239;&#172;&#8218;ies. (A) Contribution Notch hypoxia tolerance determined cul- <br /> turing homozygous-viable Notch mutants 6% O2 (relatively mild) 4% O2 (severe) hypoxic conditions. Canton-S used control. N[spl-1], which <br /> is gain-of-function Notch allele, dramatically increased survival rate hypoxic conditions. In contrast, N[fa-1] N[Ax-tsl] loss-of-function <br /> mutants little eclosion severe hypoxic condition showed reduced survival rate mild hypoxic condition (*P &lt; 0.01, compared with <br /> Canton-S control). (B) RNAi-mediated Notch knockdown induces increased sensitivity hypoxia &#239;&#172;&#8218;ies. Flies carrying UAS-N.dsRNA transgene X <br /> chromosome used determine function Notch hypoxia tolerance. Two crosses used generate &#239;&#172;&#8218;ies N.dsRNA expression either <br /> in progeny [cross A: (UAS-N.dsRNA virgin female) &#195;&#8212; (da-GAL4 male)] female progeny [cross B: (da-GAL4) &#195;&#8212; (UAS-N.dsRNA)]. Cross A progeny <br /> showed increased sensitivity hypoxia (*P &lt; 0.01 compared cross B control). This hypersensitivity rescued cross B progeny the <br /> function Notch knocked females. Both male female progeny included scoring A B crosses. (C) &#206;&#179;-Secretase ac- <br /> tivation plays important role hypoxia tolerance hypoxia-selected &#239;&#172;&#8218;ies. Five-day-old adult hypoxia-selected &#239;&#172;&#8218;ies collected treated 0.25 or <br /> 0.50 mM DAPT life span determined 1.5% O2. Median life span time 50% death occurred sample, maximum <br /> life span time 90% sample dead. Compared vehicle-treated controls, &#239;&#172;&#8218;ies treated DAPT showed signi&#239;&#172;?cant reduction of <br /> both median (*P &lt; 0.01) maximum life span (**P &lt; 0.01). The median maximum life spans calculated using GraphPad Prism 4, statistical <br /> signi&#239;&#172;?cance calculated Student&#226;&#8364;&#8482;s t test. <br />  <br />  <br />  <br /> consists Hairless, CtBP, Groucho, additionally                                               generated lines overexpressing NICD role Notch in <br /> recruits histone deacetylases (HDAC) (23). Such enrichment                                           hypoxia tolerance requires downstream gene E(spl)m&#206;&#177;. A <br /> polymorphic differences Notch pathway-related genes                                          previous study shown E(spl)m&#206;&#177; activated Notch and <br /> evidence Notch activation revealed expression pro&#239;&#172;?ling                                            functions negative feedback loop accurately adjust Notch <br /> suggested potentially important role Notch hypoxia tol-                                         signaling (37), important distinguishing cell <br /> erance. This role Notch investigated using Notch                                         survival property Notch function inhibiting cell <br /> mutants, RNAi-mediated knockdown, pharmacological in-                                               differentiation. Accumulating evidence demonstrates hyp- <br /> hibitory reagents. These experiments demonstrated Notch                                            oxia induces Notch activation mammals, including humans. For <br /> activation confers hypoxia survival. In addition, remarkable                                          example, chronic constant hypoxia up-regulates Notch1 expres- <br /> hypoxia survival observed &#239;&#172;&#8218;ies NICD overexpression                                          sion mouse heart (38) Notch1 regulates melanoma de- <br /> in speci&#239;&#172;?c neuronal and/or glial cells, demonstrating impor-                                        velopment protecting cells hypoxia-induced cell death <br /> tance maintaining integrity neuronal function hyp-                                         (39). Furthermore, Notch3 plays major role development <br /> oxia tolerance organism. Although precise                                             hypoxic pulmonary hypertension (40) humans and <br /> underlying mechanisms regarding mode action Notch                                             rodents. The role Notch hypoxia discovered study <br /> remain elusive, work speci&#239;&#172;?c NICD                                              bound important Drosophila in <br /> activation particular cells central nervous confers                                    humans, raising distinct possibility Notch signaling as <br /> survival protects hypoxia-induced death. Indeed, pre-                                          potential target translational therapeutic strategies. <br /> vious studies demonstrated anti-apoptotic function of <br /> Notch (24&#226;&#8364;&#8220;31) current study provides evidence demon-                                            Materials Methods <br /> strating activation Notch highly differentiated cells                                    Drosophila Stocks Culture. The UAS-NICD 4XSu(H)-lacZ stocks were <br /> can maintain survival-promoting property. Furthermore,                                            provided J. Posakony. All GAL4 driver lines N[Ax-tsl], N[fa-1], N[spl- <br /> recent whole-exome sequencing study 50 Tibetan subjects                                              1] mutants obtained Bloomington Drosophila Stock Center at <br /> revealed frequency noncoding intronic SNP located                                         Indiana University. Drosophila stocks cultured standard cornmeal/ <br /> at EPAS1/HIF2&#206;&#177; locus signi&#239;&#172;?cantly higher high al-                                         yeast media. To assay cell type speci&#239;&#172;?city NICD overexpression ho- <br />                                                                                                         mozygous UAS-NICD &#239;&#172;&#8218;ies crossed Eaat1-GAL4 &#239;&#172;&#8218;ies. To test NICD <br /> titude-adapted Tibetan subjects lowlander controls, <br />                                                                                                         transcriptional up-regulation, double-homozygous (E/E; N/N) males were <br /> indicating possible involvement HIF2&#206;&#177; hypoxia tolerance                                           crossed 4xSu(H)-lacZ virgin females. The UAS-RNAi-m&#206;&#177; stock obtained <br /> during evolution generations (32). It interesting                                       Vienna Stock Collection. <br /> note studies demonstrated interaction be- <br /> tween Notch HIF hypoxic condition (33&#226;&#8364;&#8220;36). In                                             Whole-Genome Resequencing. Genomic DNA isolated pool 100 <br /> current study, addition proving Notch critical                                        male 100 female adult &#239;&#172;&#8218;ies collected hypoxia-selected populations <br /> hypoxia tolerance, demonstrated newly                                             generation-matched control populations standard phenol:chloroform <br />  <br />  <br /> 2352 | www.pnas.org/cgi/doi/10.1073/pnas.1010643108                                                                                                                           Zhou et al. <br />                                    Room Air           5% Oxygen <br />  <br /> Fig. 4. Increased hypoxia tolerance &#239;&#172;&#8218;ies NICD overexpression speci&#239;&#172;?c glial cells. (A) Overexpression NICD driven Eaat1-GAL4 instar <br /> larval brain. The <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span> Eaat1-GAL4&#226;&#8364;&#8220;driven NICD (green) overexpression determined progeny (Eaat1-GAL4) &#195;&#8212; (UAS-NICD) cross. <br /> Counterstaining Repo (red) Elav (blue) reveals NICD overexpression glia (repo+) cells. Z-projection 27 slices obtained by <br /> confocal microscopy. (B) NICD overexpression-induced activation Notch glial cells determined crossing (Eaat1-GAL4/Eaat1-GAL4; UAS-NICD/UAS- <br /> NICD) Su(H)-LacZ reporter. The LacZ protein (green) detected cells locations A, demonstrating Eaat1-GAL4&#226;&#8364;&#8220;driven NICD <br /> overexpression functionally active. Z-projection 9 slices obtained confocal microscopy. (C) Glial-speci&#239;&#172;?c overexpression NICD enhances the <br /> eclosion rate hypoxia. #P value = 0.205; *P value &lt;0.001 (unpaired Student&#226;&#8364;&#8482;s t test). Bars represent mean &#194;&#177; SEM vials (n = 6). (D) Glial-speci&#239;&#172;?c <br /> overexpression NICD enhances adult survival hypoxia. Adult survival hypoxia evaluated counting number newly eclosed experi- <br /> mental control &#239;&#172;&#8218;ies transferring new individual vial. The following day, number dead &#239;&#172;&#8218;ies adult-only vial subtracted from <br /> the previous day&#226;&#8364;&#8482;s total newly eclosed &#239;&#172;&#8218;ies original vial counted added adult-only vial avoid cumbersome number of <br /> vials tests done. Hence, number &#239;&#172;&#8218;ies adult-only vial (due adult death), later increase (due addition newly eclosed <br /> adults original vial adult-only vial). <br />  <br />  <br /> extraction followed treatment DNase-free RNase. DNA quality            allelic frequencies differed control hypoxia-tolerant &#239;&#172;&#8218;ies, <br /> assessed spectrophotometry (260/280 260/230) gel electropho-            representing regions potential selection. Both analyses used Maq v.0.7.1 <br /> resis. A total 3 &#206;&#188;g sheared DNA (Covaris) used construct         (41) default parameters map reads populations <br /> a library paired-end sequencing. The DNA fragments subjected           (H1, H2, C1, C2) D. melanogaster reference genome downloaded <br /> end repair using End-IT DNA End-Repair Kit (Epicentre) ligated        FlyBase (http://www.&#239;&#172;&#8218;ybase.org). As experimental science, con- <br /> to Illumina PE adapters. The adapter-ligated products puri&#239;&#172;?ed              cordance results obtained methods provides validation not <br /> Qiaquick spin columns (Qiagen) PCR ampli&#239;&#172;?ed high-&#239;&#172;?delity DNA              results obtained methods used. <br /> Polymerase 12 cycles using Illumina&#226;&#8364;&#8482;s PE primer set. Cluster generation was <br /> performed using Illumina cluster station cluster generation kit v2.        DAPT Treatment. Adult &#239;&#172;&#8218;ies collected control hypoxia- <br /> The 54 + 54 paired-end sequencing performed using genome analyzer II           selection chamber (10 vials chamber, 10 &#239;&#172;&#8218;ies vial, 5 vials male, 5 <br /> (Illumina) sequencing kit v3. The &#239;&#172;&#8218;uorescent images processed          vials female) (n = 100 chamber). DAPT dissolved ethanol <br />                                                                                                                                                                        GENETICS <br />  <br />  <br />  <br />  <br /> sequences using Illumina base-calling pipeline (GA Pipeline-1.4.0). The D.     diluted 5% sucrose solution reach &#239;&#172;?nal experimental con- <br /> melanogaster reference genome, annotation genes               centration 0.25 0.50 mM (&#239;&#172;?nal ethanol concentration &lt;1%). Five per- <br /> and repeats, downloaded University California (Santa Cruz,         cent sucrose 1% ethanol used control. The DAPT control <br /> CA) (UCSC) database (http://genome.ucsc.edu/).                                     solution applied 150 &#206;&#188;L &#239;&#172;?lter paper vial day. <br />                                                                                    The dead &#239;&#172;&#8218;ies counted 24 h determine life span. The median <br /> Data Analysis. The next-generation sequencing data pools           maximum life spans calculated using GraphPad Prism 4 (GraphPad <br /> were derived 200 &#239;&#172;&#8218;ies descended 27 parental strains. Neither             Software), statistical signi&#239;&#172;?cance calculated Student&#226;&#8364;&#8482;s t test. <br /> individual genotypes number individuals sampled region could <br /> be determined, precluding use standard analysis tools identify differ-       Hypoxia Tolerance Vulnerability Tests. The survival rate Notch mutants <br /> ences control hypoxia-tolerant populations. We used          hypoxia determined culturing computer-controlled <br /> two complementary analysis methods. One focused identifying individual          environmental chamber. After 3 wk culture, numbers eclosed and <br /> loci high-con&#239;&#172;?dence allelic differences control hypoxia       total pupae counted. The ratio elcosed pupae total <br /> tolerant populations. The identi&#239;&#172;?ed genomic regions characterized         number pupae calculated presented eclosion rate. <br />  <br />  <br /> Zhou et al.                                                                                                PNAS | February 8, 2011 | vol. 108 | no. 6 | 2353 <br />                                                                                                48 h normoxia. The &#239;&#172;&#8218;ies moved control vial (for another <br />                                                                                               48 h discarded) vial eggs moved 5% <br />                                                                                               oxygen chamber 12-h dark 12-h light cycle temperature 22 &#194;&#177; <br />                                                                                               1 (&#226;?&#176;C. In parallel, parental line cross wasere tested normoxia <br />                                                                                               control. After 4 wk, sets &#239;&#172;&#8218;ies assayed number pupal <br />                                                                                               cases full. Six vials condition completed in <br />                                                                                               two2 different experiments minimum 500 pupal cases scored cal- <br />                                                                                               culate eclosion rate condition. Adult survival hypoxia was <br />                                                                                               evaluated counting number newly eclosed experimental control <br />                                                                                               &#239;&#172;&#8218;ies transferring new individual vial. The following day, the <br />                                                                                               number adult dead &#239;&#172;&#8218;ies adult-only vial subtracted the <br />                                                                                               previous day&#226;&#8364;&#8482;s total newly eclosed &#239;&#172;&#8218;ies original vial were <br />                                                                                               counted added adult-only vial avoid cumbersome number of <br />                                                                                               vials tests done. Hence, number &#239;&#172;&#8218;ies adult-only vial go <br />                   Room Air               5% Oxygen                                            (due adult death), later increase (due addition newly <br />                                                                                               eclosed adults original vial adult-only vial). <br /> Fig. 5. Notch activation-conferred hypoxia tolerance requires E(spl)m&#206;&#177;. The EN- <br /> line &#239;&#172;&#8218;ies, homozygous Eaat1-GAL4 driver UAS-NICD,                      The statistical signi&#239;&#172;?cance eclosion rate mutants, NICD- <br /> were crossed homozygous UAS-E(spl)m&#206;&#177;. RNAi stock knock E(spl)m&#206;&#177;                    overexpressed &#239;&#172;&#8218;ies, controls calculated unpaired t test. <br /> on background Notch overactivation. Hypoxia tolerance progeny                       See SI Materials Methods methods. <br /> was determined eclosion rate 5% O2 hypoxic condition. Compared <br /> with Notch overactivation control (EN/w1118), knocking E(spl)m&#206;&#177;                   ACKNOWLEDGMENTS. We thank O. Gavrialov, M. Y. Hsiao, Y. Lu-Bo, <br /> the Notch overactivation background abolished hypoxia tolerance (P &lt; 0.01;                    J. Wang, N. Morgan technical assistance. Confocal microscopy was <br /> t test). Bars represent mean &#194;&#177; SEM (n = 3) group/treatment.                      performed University California San Diego Neuroscience <br />                                                                                               Microscopy Shared Facility (P30 NS047101). This work supported by <br />                                                                                               Eunice Kennedy Shriver National Institute Child Health Human <br />                                                                                               Development (5P01HD032573); National Institute Neurological Dis- <br />    The UAS-NICD stock crossed speci&#239;&#172;?c GAL4 transgenic &#239;&#172;&#8218;ies de-                    orders Stroke (5R01NS037756); National Heart, Lung, Blood <br /> termine effect speci&#239;&#172;?c spatiotemporal NICD overexpression hypoxia                   Institute (5R33HL087375); National Human Genome Research Institute <br /> tolerance. Each cross contained 10 virgin female homozygous UAS-NICD &#239;&#172;&#8218;ies                     (5R01HG004962); National Science Foundation (DBI-0641037 NSF-III- <br /> and 5 male homozygous GAL4 transgenic &#239;&#172;&#8218;ies allowed lay eggs                       # <br /> </body></html>