<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Apr. 1993, p. 2152-2161                                                               Vol. 13, No. 4 <br /> 0270-7306/93/042152-10$02.00,&apos;0 <br /> Copyright X) 1993, American Society Microbiology <br />  <br />  <br />      GSP1 GSP2, Genetic Suppressors prp20-1 Mutant in <br />        Saccharomyces cerevisiae: GTP-Binding Proteins Involved <br />              Maintenance Nuclear Organization <br />                        PIERRE BELHUMEUR,t ARIANNA LEE, ROSE TAM, TIZIANO DiPAOLO, <br />                                    NATHALIE FORTi. A.ND MICHAEL W. CLARK* <br />                        Department Biology, McGill University, Montreal, Quebec H3A IBJ, Canada <br />                      Received 13 August 1992/Returned modification 12 October 1992/Accepted 14 January 1993 <br />  <br />             The temperature-sensitive <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span> prp20-1 Saccharomyces cerevisiae exhibits pleiotropic <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> <br />          associated general failure maintain proper organization nucleus. Its mammalian homolog, <br />          RCC1, reported involved negative control chromosome condensation also <br />          believed assist coupling DNA replication entry mitosis. Recent studies Xenopus RCC1 <br />          strongly suggested role protein formation maintenance DNA replication <br />          machinery. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> elucidate nature various components required PRP20 control pathway S. <br />          cerevisiae, undertook search multicopy suppressors prp2O thermosensitive mutant. Two genes, <br />          GSPI GSP2, identified encode identical polypeptides 219 220 amino acids. <br />          Sequence analyses proteins contain ras consensus domains involved GTP binding <br />          metabolism. The levels GSP1 transcript 10-fold GSP2. As S. cerevisiae R4S2, <br />          GSP2 expression exhibits carbon source dependency, GSPI expression does not. GSPI essential <br />          gene, GSP2 required cell viability. We GSP1p nuclear, bind GTP an <br />          vitro assay, finally, mutation GSPlp activates small ras-like proteins increasing the <br />          stability GTP-bound form causes dominant <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>lethal phenotype</span>. We believe gene products <br />          serve regulating activities multicomponent PRP20 complex. <br />  <br />  <br />    During proliferation, is, increase cell number,        prematurely condensed chromosomes (55). RCClp thus <br /> the nucleus eukaryotic cells progresses series       thought negatively regulate chromosome condensation by <br /> of dramatic structural rearrangements culminate            holding chromosomes interphase position until <br /> in separation condensed chromosomes nuclear            cell ready enter mitosis allow the <br /> reformation. The complex nature events predicts              chromosomes condense. The important role RCCIp plays <br /> that machinery cell duplication process spa-            maintenance chromosome conformation and/or <br /> tially, temporally, ordered highly regu-          nuclear placement demonstrated Xeno- <br /> lated. The elaborate positive negative control-         pus RCClp. During vitro nuclear assembly assays, DNA <br /> ler proteins govern cell&apos;s entry proliferation      replication requires presence RCClp (14). RCClp was <br /> stages, DNA synthesis (S) mitosis (M), rapidly         thought affect DNA replication direct enzymatic <br /> being delineated (16, 17, 22, 39). The details             involvement synthesis process, playing an <br /> activities regulators affect spatial arrangements        essential role initial establishment DNA repli- <br /> of nuclear components cell cycle, though,        cation complex chromosome. <br /> remains unclear. Information obtained examination               The multiple nuclear events overseen RCClp and <br /> of ubiquitous nuclear protein, RCClp, revealing              homologs various organisms appear regulated by <br /> specific linkages cell cycle regulatory functions         association protein proteins, of <br /> and rearrangements nuclear components seen              GTP-binding characteristics (4, 18, 31, 35). <br /> the cell cycle.                                                       RCClp purified human HeLa cells a <br />    RCClp reported attributes cell            stoichiometric association protein 25 kDa termed <br /> cycle regulatory protein direct involvement          RAN (6, 40). Partial protein sequence analysis RAN <br /> in placement chromosomes nucleus             shows contains sequence similarities ras super- <br /> the interphase cell cycle (38, 55). At restrictive <br /> temperatures, stages cell cycle, mamma-            family GTP-binding consensus and, specifically, signif- <br /> lian cell lines contain temperature-sensitive (ts)            icant homology human gene called TC4 (4, 6, 15). <br /> mutation RCC1 enter mitosis DNA                   Consistent protein sequence predictions, RAN <br /> synthesis completed (38). It               protein shown bind GTP vitro, RCC1 <br /> proposed RCClp cell cycle                   protein specifically catalyze exchange GTP on <br /> checkpoint monitors DNA replication (17). A single              RAN (5). Furthermore, fission yeast Schizosaccharo- <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>point mutation</span> coding region RCC1 generate             mycespombe, homolog RCC1 identified and <br /> anomalous chromosome structures reported               termed PIMI (35). At restrictive temperature, chromosomes <br />                                                                       piml ts mutant undergo condensation enter <br />                                                                       mitosis presence hydroxyurea, agent <br />   * <br />     Corresponding author.                                             normally arrest cells S phase by <br />   t Present address: Department Microbiology Immunology,       inhibiting DNA synthesis. Along PIM1 wild-type <br /> Universite Montreal, Montreal, Quebec H3C 3J7, Canada.             allele, gene termed SPI1 isolated multicopy <br />                                                                    2152 <br />  VOL. 13, 1993                                         GSP1 AND GSP2: NEW S. CEREV7SL4E GTP-BINDING PROTEINS                      2153 <br />  <br /> suppressor piml mutant. Analysis amino acid                               TABLE 1. Yeast strains <br /> sequence gene predicts polypeptide highly homol-                                                               Source or <br /> ogous RAN/TC4 and, therefore, categorizes SPIl                    Strain                 Genotype                 reference <br /> small GTP-binding protein. <br />    The association RCClp small GTP-binding                 EJ101            Mata hisl prbl-1122 prcl-126    1 <br /> protein strongly implies nuclear function RCClp          SS328            MAATa ade2-101 his3A200         1 <br /> could mediated ratios GDP/GTP-                                  lys2-801 ura3-52 <br /> bound forms protein. Small GTP-binding pro-            prp2O/2C         MA4Ta prp2O-1 ade2-101          1 <br />                                                                                         his3A200 lys2-801 ura3-52 <br /> teins known control multitude cytoplas-          SEY6210          M4Tot leu2-3,112 ura3-52        A. H. Bussey <br /> mic functions, including (i) sorting amplification                               his3A200 zys2-801 trplA901 <br /> transmembrane signals, (ii) directing protein synthesis                             suc2A9 <br /> translocation, (iii) guiding vesicular traffic cyto-     HAB251-15B       6210 autodiploid MAMTa/MATa     A. H. Bussey <br /> plasm, (iv) controlling cell proliferation (8, 11, 27, 41, 49,   HAB251AGSP1      Same HAB251-15B          This study <br /> 52). Elaborate multicomponent, highly specific control                                  GSP1+/gspl::HIS3 <br /> mechanisms regulate activities           HAB251AGSP2      Same HAB251-15B          This study <br /> the cytoplasmic GTP-binding proteins. It reason-                               GSP2&apos;/gsp2::HIS3 <br /> able prediction RCC1-RAN activity complex                PUB175           MA4Ta gspl::HIS3 leu2-3,112     This study <br />                                                                                         ura3-52 his3A200 lys2-801 <br /> recruiting unidentified cytoplasmic and/or nuclear                            trplA901 suc2A9 (plasmid <br /> proteins GTP/GDP regulation proper conforma-                              pRS316+GSPl; URA3) <br /> tion chromosomes required essential <br /> nuclear functions DNA replication. <br />    To advantage extensive knowledge mutant <br /> collections amassed budding yeast             ment function, minimum independent transfor- <br /> Saccharomyces cerevisiae, investigations RCC1-                mants assayed, result is <br /> RAN complex undertaken organism. The S.                 presented report. <br /> cerevisiae homolog RCC1 termed PRP20. Since               Plasmids. The 2p yeast genomic libraries used the <br /> first identified screen RNA-splicing mutants (56),       suppressor screen, plasmids YEp351 -352, as <br /> have shown prp2O splicing defect          pRS316, described (10, 23, 51). The <br /> complex set phenotypes associated maintenance            Bluescript vector used gene subcloning sequencing <br /> of nuclear structure. This gene sequence func-           Stratagene (San Diego, Calif.). <br /> tional homology RCCJ; theprp20-1 yeast ts mutant             DNA manipulations sequencing. Standard procedures <br /> complemented human RCCI gene (1, 18). We report               used recombinant DNA manipulations (46). Site- <br /> here high-copy-number gene suppression techniques            directed mutagenesis (28) GSPI pRS316 as <br /> against theprp20-1 ts allele, proteins interact       single-stranded DNA substrate (the sense strand) contain- <br /> prp2Op genetically identified, GSPlp GSP2p.            ing entire BglII-SstI fragment (see Fig. 2A). The muta- <br /> The proteins greater 95% identity              genic oligonucleotide Gly-21 Val change 5&apos;- <br /> other, present strong homology ras super-            CCAGTACCAACATCACCGA-3&apos; (the underlined A would <br /> family, especially human TC4 (15) SPIl S.              C complementary wild-type <br /> pombe (35). We characterized GSP1 gene prod-             sequence). To confirm presence predicted muta- <br /> uct. GSP1 nuclear protein binds GTP vitro        tion absence spurious mutations, deter- <br /> vivo. As demonstration direct involvement              mined sequence entire coding region. <br /> PRP20-GSP1 (2) complex nuclear structure maintenance,               The sequencing DNA strands performed with <br /> the overexpression GSPI GSP2 shown prevent           Sequenase kit (U.S. Biochemical, Cleveland, Ohio) with <br /> disruption nucleoplasm caused prp20-1 ts allele.       a-35S-dATP (Amersham Canada Limited, Oakville, On- <br />                                                                      tario). DNA oligonucleotide primers (either universal, re- <br />                MATERIALS AND METHODS                                 verse Bluescript, internal sequenced region) <br />                                                                      used sequencing. <br />    Strains, media, growth conditions. Yeast strains used            The chromosomal localization GSPI GSP2 was <br /> in work described Table 1. Eschenchia coli               determined chromosomal blots S. cerevisiae <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> DH10B (Bethesda Research Laboratories, Inc.) rou-                Clontech (Palo Alto, Calif.) conditions specified the <br /> tinely used plasmid host, E. coli CJ236 used         manufacturer. Specific DNA probes labelled the <br /> a site-directed mutagenesis experiment (28). Bacterial           Oligolabelling Kit (Pharmacia LKB Biotechnology, Piscat- <br /> yeast media prepared described previously (43,               away, N.J.) [a-32P]dCTP (Amersham Canada Limited). <br /> 46) SC-uracil, hydrolysate casein             The GSPJ probe derived 3.0-kb fragment con- <br /> (Casamino Acids; Difco) added 0.25% source            taining entire reading frame, including -600 bp upstream <br /> amino acids. YPG medium contained, addition glycerol           initiation codon SstI restriction site (see <br /> at 3% (vol/vol), ethanol 2% (vol/vol). Preparation media       Fig. 2A, upper diagram). The probe GSP2 1.8-kb <br /> containing 5-fluoro-orotic acid described         BglII-EcoRV restriction fragment (see Fig. 2A, lower dia- <br /> (50). The permissive restrictive temperatures growth         gram). Under hybridization conditions used these <br /> of yeast ts mutants 24 34&#194;&#176;C, respectively. Yeast            chromosome blots, cross-hybridization seen between <br /> transformations performed lithium acetate                genes. The <span id='am-2' about='rdfs:comment' typeof='owl:Thing'>comment</span> the <br /> method (24) initial screen suppressors,        detection RNA transcripts judged Northern <br /> which DNA introduced prp202C electropora-                (RNA) blot analysis null strains (data shown). <br /> tion (2).                                                               RNA analysis. Total RNA extracted exponen- <br />   Whenever various plasmids carrying different yeast                 tially growing cells YPD YPG 30&#194;&#176;C glass <br /> genomic fragments tested ability comple-           bead method (43). The RNA analyzed 1.2% agarose <br />  2154     BELHUMEUR ET AL.                                                                                          MOL. CELL. BIOL. <br />  <br /> gel 20 mM MOPS (morpholinepropanesulfonic acid)-5 <br /> mM sodium acetate-1 mM EDTA-0.66 M formaldehyde <br /> (19). Each lane contained 10 ,ug RNA, blots (nitro- <br /> cellulose filters Schleicher &amp; Schuell) analyzed <br /> with labelled GSPI GSP2 probes (similar specific activ- <br /> ity) described above. Hybridization blots done <br /> in 3x SSC (lx SSC 0.15 M NaCl plus 0.015 M sodium <br /> citrate)-lOx Denhardt&apos;s solution (46)-0.1% sodium dodecyl <br /> sulfate (SDS)-10% dextran sulfate-100 jig competitor <br /> DNA ml 65&#194;&#176;C 16 h. Final washing conditions the <br /> blots 0.1x SSC-0.1% SDS 65&#194;&#176;C. The loading of <br /> equal amounts mRNA lane verified a <br /> BglII-XhoI 0.6-kb DNA fragment L29 ribosomal                    High-copy plasmid                  Low-copy plasmid <br /> protein DNA sequence detecting 0.55-kb transcript (26).            FIG. 1. Growth 34&#194;&#176;C prp20-1 cells containing various plas- <br />    Autoradiograms scanned LKB Ultrascan XL            mids. prp2o/2C cells transformed Ura+ 24&#194;&#176;C URA3- <br /> Laser Densitometer.                                                based plasmids carrying different inserts streaked the <br />    Generation GSPlp antibody. A 697-bp PvuII-AccI frag-         minimal plates incubated 34&#194;&#176;C. The vector used was <br /> ment GSPJ locus containing coding region in-            YEp352 (high-copy-number plasmid) pRS316 (low-copy- <br /> cluded 50 bp downstream stop codon. The fragment            number plasmid). Sectors contain cells following plasmids: <br /> was blunt ended filling overhang Klenow             A, PRP20; B, GSPI; C, GSP2; D, vector alone. <br /> polymerase ligated vector pGEX-3X (Pharmacia <br /> LKB Biotechnology) previously EcoRI <br /> cleaved blunt ended Klenow polymerase. The                binding proteins. The GTP-binding proteins yeast cell <br /> resulting plasmid directed production hybrid protein      extract detected based various protocols (30, 36, <br /> of approximately 50 kDa composed 27-kDa portion            45). After separation proteins SDS-12% PAGE, <br /> the glutathione S-transferase fused GSPlp (23 kDa).         transferred immediately nitrocellulose filter <br /> This fusion process resulted loss amino   (Schleicher &amp; Schuell) treatment gel. To <br /> acids GSPlp. Overproduction GST-GSP1 fusion           detect GTP-binding proteins, incubated blot 1 h in <br /> E. coli led formation inclusion bodies (37). The         blocking solution containing 1% gelatin, 0.1% Tween 20, and <br /> hybrid protein inclusion bodies purified pre-      1 x PBS room temperature (48). The filter rinsed <br /> parative SDS-polyacrylamide gel electrophoresis (PAGE)             briefly binding buffer (50 mM Tris-HCI [pH 7.5], 0.3% <br /> (29) used immunization rabbits standard          Tween 20, 5 mM MgCl, 1 mM Na2-EGTA). Binding GTP <br /> methods described reference 13.                              assayed [a-3 P]GTP (3,000 Ci/mmol; Amersham <br />    Immunological techniques. Protein samples separated        Canada Limited) 1 ,uCi binding buffer ml 1 h at <br /> on SDS-12% polyacrylamide gels. The proteins elec-            room temperature (45). After binding, blot rinsed <br /> trotransferred nitrocellulose filter (Schleicher &amp;          times binding buffer 1 2 h then <br /> Schuell) semi-dry transfer cell (Bio-Rad) according       exposed Kodak X-(OMAT)AR film intensifying <br /> to manufacturer&apos;s specifications. The complete methods         screen -80&#194;&#176;C. <br /> used react proteins filters antiserum <br /> are described reference 13. After initial blocking                               RESULTS <br /> step, nitrocellulose filter incubated overnight at <br /> room temperature 1/1,000 dilution preimmune            Isolation nucleotide sequence analysis GSPI and <br /> serum GST-GSP1 antiserum TPBS. The filters          GSP2 genes. To identify gene products directly inter- <br /> then processed Vectastain ABC kit (Vector Labo-           act PRP20 protein, genes isolated could <br /> ratory, Burlingame, Calif.).                                       suppress ts lethal phenotype prp20-1 mutant by <br />    Labelling immunoprecipitations yeast proteins            increasing cellular copy number. A yeast 2,u YEp24- <br /> were conducted follows. EJ101 yeast cells grown         based genomic library (10) transformed the <br /> saturation minimal medium containing 1 mM (NH4)2SO4             prp2O/2C strain electroporation selected for <br /> and 5% glucose supplemented appropriate amino             uracil independence permissive temperature. After 5 <br /> acids. Cells washed resuspended optical             days 24&#194;&#176;C, 20,000 colonies replica plated and <br /> density 600 nm 0.02 medium        transferred 34&#194;&#176;C. Fourteen thermoresistant transformants <br /> (NH4)2SO4, 100 ,uM. At mid-log phase,         identified, counterparts, always <br /> cells resuspended small volume fresh medium           maintained permissive temperature avoid genomic <br /> without (NH4)2SO4 optical density 2 600 nm (cells      rearrangements, picked analysis. <br /> per milliliter). Labelling took place room temper-         Restriction endonuclease mapping rescued plasmids <br /> ature 3 h presence 250 ,uCi [35S]H2SO4 -43 Ci/mg      revealed wild-type PRP20 allele obtained twice <br /> of S; ICN). After cells washed twice lx              new complementation groups, termed GSPJ <br /> phosphate-buffered saline (PBS), lysates prepared          GSP2 (for GTP-binding suppressor PRP20), were <br /> RIPA (radioimmunoprecipitation) buffer (46) glass           picked times, respectively. As shown in <br /> bead method (21). Total lysate (100 jxg) immunoprecip-         Fig. 1, GSPI expressed multicopy plasmid comple- <br /> itated 5 ,ul anti-GSP1 antibody, results      ments ts phenotype prp20-1 mutation 34WC <br /> analyzed SDS-12% PAGE autoradiography.                      PRP20 gene (sectors A B). GSP2 <br />    The immunofluorescence techniques prefixed cells        (sector C) displays ability suppress defective <br /> and nucleolar-specific staining procedures followed           prp20-1 function reduced level comparison <br /> described previously (12, 13).                                     GSP1. There appears gene dosage effect from <br />    [a-32P]GTP probing nitrocellulose filter GTP-        suppressors viability strain prp2O/2C be <br />  VOL. 13, 1993                                      GSP1 AND GSP2: NEW S. CEREVISIAE GTP-BINDING PROTEINS                                                              2155 <br />  <br /> restored GSPI expressed low-copy-number plas-                  B <br />                                                                                                                                                             A <br /> mid, case PRP20 gene.              F-                                                4-        GP <br />                                                                                                     aspi <br /> GSP2 low-copy-number plasmid, though, sup- <br /> port growth prp20-1 mutant restrictive <br /> temperature. It noteworthy that, unlike PRP20 gene,                                                       HiHI                    I T SI <br /> neither suppressors overcome lethal pheno-                   |                     ;           Hl ;  | <br />                                                                                                                                      f~~~~gpl-.NS <br /> type prp2O/2C 37&#194;&#176;C, support growth <br /> of prp2OA strain (data shown). The ability GSP <br /> genes rescue prp2O mutant allele specific. <br />   The prp20-1 suppression activity alleles was <br /> located endonuclease restriction mapping gene dele-               KB                             R                                         K <br /> tions. The nucleotide sequence genes                                                                                                      GSP2 <br /> determined. As shown Fig. 2B, GSPI gene pre-                                GSP2                      SERI <br />  <br /> dicted encode polypeptide 219 amino acids, and <br /> GSP2 encodes 220 residues. The proteins strikingly                    B <br />                                                                                       A <br />                                                                                                                              R <br />  <br /> similar, sharing 97% identical amino acid residues                               | HlS3l <br /> being 82% identical coding region nucle-                                                             gip2.,HIPS3 <br /> otide level. Their overall homology ras superfamily <br /> resides discrete, highly conserved domains believed       B <br />                                                                  Saks <br />                                                                                                                G-1     + <br />                                                                                                                            YDPTIZ-3S <br />                                                                                                                                                            G-2 <br /> to involved guanine nucleotide metabolism (8). GSP1        Gsil <br />                                                                                                  LVQGo GVGI                                                             49 <br />                                                                                   NSAPAAN-GEVPTFKLVLVGDGGTGKTTFVKRHLTGEFEKKYIATIGVEV <br /> and GSP2 lack C-terminal domain           QS2 <br />                                                                  GSil <br />                                                                                   NSAPAQNNAEVPTFKLVLVGDGGTGKTTFVKRHLTGEFEKKYIATIGVEV <br />                                                                                   MAQPQN----VPTFKLVLVGDGGTGKTTFVKRHLTGEFEKKYIATLGVEV <br />                                                                                                                                                                         50 <br />                                                                                                                                                                         46 <br /> present various ras family members believed    1C4              MAAQGEPQVQ---FKLVLVGDGGTGKTTFVKRHLTGEFEKKYVATLGVEV                                    47 <br /> be responsible membrane anchoring. Hybridization                               * <br />                                                                                       ... <br />                                                                                         .                   *******~*** ** ********* *** ***          * .* . **** <br />  <br />  <br /> these genes yeast chromosome blot (see Materials        BRAD             I                DThGQ3 <br />                                                                                                                      G-3 <br /> and Methods) revealed GSP1 chromosome VII                          HPLSFYTNFGEIKFDVWDTAGQEKFGGLRDGYYINAQCAIIMFDVTSRIT                                    99 <br /> GSP2 chromosome XV. No cross-hybridization be-             Gpill <br />                                                                  311              HPLSFYTNFGEIKFDVWDTAGQEKFGGLRDGYYINAQCAIIMFDVTSRIT <br />                                                                                   HPLHFHTNFGEICFNVWUDTAGQEKLGGLRDGYYIQGQCGIIMFDVTSRIT <br />                                                                                                                                                                        100 <br />                                                                                                                                                                         96 <br /> tween genes seen hybridization           TC4              HPLVFHTNRGPIKFNVWDTAGQEKFGGLRDGYYIQAQCAIIMFDVTSRVT <br />                                                                                   ***      * .**    *   *   * .********* .********* . . ** .*********. * <br />                                                                                                                                                                         97 <br /> conditions used study. The GSP2 gene                                                                    G-4 <br /> found adjacent locus SERI (Fig. 2A, lower           XMAS                                                       0KCDL <br /> diagram). The SERI locus encodes yeast analog                                                                                                              149 <br />                                                                                                                                                                        150 <br /> enzyme 3-phosphoserine aminotransferase (EC 2.6.1.52) (3,        3111 <br />                                                                  TC4 <br />                                                                                                                                                                        146 <br />                                                                                                                                                                        147 <br /> 54).                                                                              ***** . *        ********************.* . . ***** . *. ***** <br />    The GSP1 GSP2 proteins exhibit 88% identity                            G-5 <br /> two proteins putative GTP-binding ability (Fig.       BRAS             VS8AKY <br />                                                                  Gpil             DISAKSNYNFEKPFLWLARKLAGNPQLEFVASPALAPPEVQVDEQLMQQY                                   199 <br /> 2B). In similar overproduction suppressor screen forpiml,      G12              DISAKSNYNFEKPFLWLARKLAGNPQLEFVASPALAPPEVQVDEQLHHQY <br />                                                                                   DISAKSNYNFEKPFLWLARKLVGNPNLEFVASPALAPPEVQVDQQLLAQY <br />                                                                                                                                                                        200 <br /> a ts allele S. pombe homolog budding yeast         ?C4              DISAKSNYNFEKPFLWLARKLIGDPNLEFVAMPALAPPEVVMDPALAAQY <br />                                                                                                                                                                        196 <br />                                                                                                                                                                        197 <br /> PRP20 mutant, single gene called SPII identified (35).                      *********************                    *.*.*****   ********       .*    .*   ** <br />  <br /> TC4, mammalian homolog yeast GTP-binding            as1              QQEMEQATALPLPDEDDADL                     219 <br /> proteins, previously isolated degenerate oligonu-       GSP2             QQEMDQATAWWLDEDDADL                      220 <br />                                                                                   QQEMNEAA.AMPLPDEDDADL                    216 <br /> cleotide probes ras GTP-binding consensus sequence                        EHDLEVAQNPALPDEDD-DL                     216 <br />                                                                                       ...-* <br />                                                                                          . ****** ** <br /> on human teratocarcinoma cDNA library (15). The ex-                                  * <br />  <br />  <br />  <br /> traordinary similarity distantly related species         FIG. 2. Structures GSPI GSP2. (A) Gene structures of <br /> strongly indicates critical cellular functions    GSP1 GSP2. The closed boxes represent identified reading <br /> performed proteins.                                     frames, arrows indicate deduced direction transcrip- <br />                                                                  tion. Indicated restriction sites are: A, AgeI; B, BgIII; C, ClaI; E, <br />    GSPI essential cell viability, GSP2 not. To   EcoRI; K, KpnI; P, PstI; R, EcoRV; S, SstI. The disruption <br /> assess importance GSPI GSP2 viability       constructs depicted. (B) Amino acid sequence comparisons. <br /> the budding yeast cells, disruptions genes       Deduced amino acid sequences GSP1 GSP2 aligned with <br /> were constructed. The GSP1 coding region interrupted         ras-like proteins, S. pombe SPIl (35) human <br /> after 30 amino acids introduction HIS3 gene     TC4 (15) sequences. The domains involved guanine <br /> an EcoRI site. The GSP2 gene disrupted 29 amino        nucleotide metabolism indicated (8). Identical amino acids <br /> acids AgeI site selectable marker (Fig.      represented asterisks, conservative changes indicated <br /> 2A). The inactivated copy gene introduced            dots. GenBank accession numbers genes are <br />                                                                  L08690 (GSPI) L08691 (GSP2). <br /> separately autodiploid strain HAB251 perform a <br /> one-step gene replacement (44). Since strain HAB251 was <br /> derived introduction episomal HO gene under <br /> the control Gall/JO inducible promoter yeast <br /> AM Ta SEY6210 strain, strain homozygous       indicating gspl ::HIS3 recessive lethal allele <br /> all genes AL4T loci (9a). Southern blot      required spore germination.   This possibility was <br /> analysis chromosomal DNA diploid strains             ruled close examination microcolonies derived <br /> HAB251AGSP1 HAB251AGSP2 containing dis-                  spore assumed received disrupted <br /> rupted alleles gspl::HIS3 gsp2::HIS3 confirmed          allele; undergone divisions arrest- <br /> each diploid contained wild-type allele inacti-      ing. In contrast, members tetrads derived from <br /> vated allele (data shown).                                   HAB251AGSP2 viable showed 2:2 segregation for <br />    When HAB251AGSP1 HAB251AGSP2 induced                 histidine prototrophy conferred gsp2::HIS3. Having <br /> to sporulate, haploid meiotic products        verified interrupted allele really lost activity <br /> former able grow absence histidine,            suppress ts lethal phenotype prp20-1 <br />  2156     BELHUMEUR ET AL.                                                                                                                  MOL. CELL. BIOL. <br />  <br />                                                                                            strain PUB175, sensitivity 5-fluoro-orotic acid was <br />                                                                                            assayed. Subsequent growth strain medium <br />                                                                                            indicate overexpression GSP2p support <br />                                                                                            viability cell totally devoid GSP1 gene product; Fig. <br />                                                                                            3 shows case. Surprisingly, considering their <br />                                                                                            respective efficiencies suppressing prp20-1 ts pheno- <br />                                                                                            type, growth GSPJ-disrupted strain, sup- <br />                                                                                            ported overexpressed GSPlp GSP2p episomal <br />                                                                                            manner (Fig. 3, sectors A B), comparable. <br />                                                                                               Different levels expression GSP1 GSP2 mRNAs. <br />                                                                                            One possible explanation weak suppression the <br />                                                                                            p7p20-1 ts mutation GSP2 comparison seen <br />    FIG. 3. Overexpression GSP2 support growth gsplA                            GSP1 steady-state levels GSP1 <br /> strain. Plasmids YEp351 derivatives introduced strain <br /> PUB175 selection Leu+ transformants. Cells                                mRNA higher GSP2 mRNA. Northern blot <br /> streaked selective medium (control, left plate) shuffling                         analysis RNA preparations exponentially growing <br /> medium containing 5-fluoro-orotic acid (5-FOA) (right plate). Sec-                         cells SS328, prp20/2C parental strain, SEY6210, <br /> tors contain cells following plasmids: A, GSP1; B, GSP2;                          haploid strain used generate autodiploid HAB251, <br /> C, YEp351.                                                                                 probed GSP1 GSP2 mRNAs. When cells <br />                                                                                            grown glucose, significantly GSP1 <br />                                                                                            mRNA GSP2 mRNA (Fig. 4A, compare lanes 1 3 <br /> mutation, concluded GSP2                                   5 7). Using transcript L29 ribosomal <br /> required cell viability.                                                               protein control mRNA expression rates, densitomet- <br />   We investigated possibility overexpression                              ric scanning autoradiograms showed was <br /> GSP2p support growth gspl-disrupted                                       10-fold RNA GSP1 GSP2 (Fig. 4B). <br /> strain. The strategy used replace, plasmid                                  Both RNAs run position corresponding 0.9 kb size. <br /> shuffling, URA3 GSPI plasmid LEU2 GSP2 plasmid                                    Expression of suppressors&apos; transcripts did not <br /> in gspl null strain monitor viability. The                                    display cell cycle regulation judged Northern <br /> HAB251AGSP1 diploid transformed URA3-based                                          blot analysis RNA preparations cells released after <br /> centromeric plasmid (pRS316; reference 51) bearing                                     a-factor block (data shown). Since RASI expression <br /> GSPI induced sporulate, tetrads dissected.                                 exhibits carbon dependency, mRNA levels down- <br /> Haploid strains obtained displayed His&apos; Ura+                                   regulated yeast cells grown nonfermentable carbon <br /> phenotype, (i) having GSPI chromosomal                                      sources (9), possibility explored GSP mRNA <br /> copy inactivated (ii) exhibiting sensitivity 5-fluoro-                              levels. If plots GSP1 mRNA compari- <br /> orotic acid; compound able counterselect cells                                   son L29 ribosomal protein mRNA, which <br /> displaying Ura- phenotype, URA3-based plasmid                                     decreases cells grown YPG versus YPD, likely reflect- <br /> loss (7). One disrupted strains, PUB175, used                                  ing slower rate growth cells different <br /> for plasmid shuffling experiments. GSP2 carried                                    media (Fig. 4A, lower panels), significant differ- <br /> LEU2-based 2,i vector (YEp351) introduced                                        ence GSP1 mRNA cells are <br />  <br />  <br />                                                      (1 <br />                               A         e-11t-                           55_e                       B <br />                                     x &gt;- x &gt;                            x&gt;- x &gt;- <br />                                     LU   J   Li -i                      LU J I Li __j <br />                                     C    00D0 0D                         00( 0 0f <br />  <br />  <br />                                                                                                                              E GLUCOSE <br />                                                                                                U)cO <br />                                                                                                                               E GLYCEROL <br />                                                                                                LZ <br />                                                           -   25S   -                          Z)     E <br />                                                                                                CC         6- <br />                                                           -   18S -                            O- &#194;&#176; <br />                                                                                         GSP2   1 Sj <br />                             GSP1 <br />                             mRNA <br />                                                                         3o   #0 40 <br />                                                                                         mRNA   U00 <br />                                                                                                w  C <br />                                                                                                   - <br />  <br />  <br />                                                                                                Z &lt;5 <br />                                                                                                           4- <br />  <br />  <br />                                                                                                CM <br />  <br />                                                                                                           2- <br />  <br />  <br />  <br />                                    :rR           NrRNAs <br />                                                                                                                KL   GSPI   GSP2 <br />  <br />  <br />                                                           L29 RNA,      ,z,* <br />                                     1    2 3     4                      5 6 7 8 <br />    FIG. 4. Northern blot analysis GSP1 GSP2 mRNAs. (A) Samples (10 pg) total RNA extracted strains SS328 SEY6210 <br /> grown glucose (DEX) glycerol (GLY) analyzed probing GSP1 mRNA left blot GSP2 mRNA right <br /> blot. Lower panels are, first, photographs gels transfer (rRNAs), second, signals obtained reprobing filters <br /> for control, L29 ribosomal protein transcript. (B) Histogram derived densitometric scanning autoradiography the <br /> blots, relative L29 ribosomal protein mRNA. <br />  VOL. 13, 1993                                                 GSP1 AND GSP2: NEW S. CEREVISLAE GTP-BINDING PROTEINS                    2157 <br />  <br />                      0          Pi         I   Pi                           electrotransferred nitrocellulose filter subse- <br />                                  1         2   1        2 <br />                                                                             quently incubated presence a-32P-labelled GTP. <br />                                                                             Only immune serum (Fig. SC, lane 2) did protein bands <br />                                                                             bind GTP. The sizes detected proteins 23 <br />                                                                             17 kDa. The 23-kDa GTP-binding band runs coincident <br />          43 kOi- <br />                                                                             GSPlp Western blot (immunoblot) the <br />                                                                             35S-labelled samples (Fig. 5A B, lane 2). This [32P]GTP <br />                               :: ..: <br />                                                                             binding proteins exhibits competitive binding <br />         25 5kO-                                                             unlabelled GTP. ATP, CTP, UTP not <br />                                                             -GSP1           compete labelled binding (data shown). Further <br />                                                                             controls verify 23-kDa GTP-binding <br />                                                                             band GSP1. First, GSPlp overproduced yeast <br />                                                                             cells, coincident increase GSP1, revealed immuno- <br />                                                                             blot analysis, 23-kDa GTP-binding band, revealed by <br />                      A                 B            c                       immunoprecipitation GTP filter assays, observed <br />   FIG. 5. Immunological analysis GSP1 protein.           (A) West-   (data shown). Second, wild-type cellular concentra- <br /> ern blot analysis GSP1 protein whole-cell extract.        Wild-type   tions GSP1, GSPlp retained double-stranded <br /> yeast total protein extract (75 ,ug) separated           SDS-12%     DNA column presence wild-type PRP20p. In <br /> PAGE, transferred nitrocellulose filter, probed anti-         inactivated extracts ts prp20-1 mutant strain, neither <br /> GSP1 antiserum.   (B) Immunoprecipitation analysis GSP1 protein.         PRP20P GSPlp observed binding double- <br /> 35S-labelled extract subjected immunoprecipitation              stranded DNA (31). The eluted fractions prp2Op <br /> preimmune serum (PI) (lane 1) anti-GSP1 antiserum (I) (lane 2).          mutant double-stranded DNA column contained 23-kDa <br /> The immunocomplexes, adsorbed protein A-Sepharose, were <br /> separated SDS-12% PAGE. A 23-kDa protein band correspond- <br />                                                                             GTP-binding protein (data shown). <br /> ing GSPlp indicated 53-kDa protein (asterisk)               gspl-val2l allele exhibits dominant lethal phenotype. <br /> corresponding PRP20 polypeptide. (C) GTP-binding ability              GSP1, GSP2, SPII, TC4 classified members the <br /> GSP1 polypeptide. Proteins immunoprecipitated preimmune                ras superfamily based protein sequence homology. Al- <br /> serum (lane 1) anti-GSP1 antibodies (lane 2) obtained              mammalian protein termed RAN, very <br /> unlabelled whole-cell extract separated SDS-12% PAGE,               similar TC4, shown bind GTP vitro (5), <br /> transferred nitrocellulose filter, probed [a-32P]GTP          vivo data describing phenotype an <br /> (see Materials Methods). Protein bands corresponding GSPIp           altered allele defective GTP metabolism. We therefore <br /> and 17-kDa polypeptide indicated. kD, kilodaltons.                 mutated highly conserved domain shared between <br />                                                                             ras members, changing Gly-21 Val residue in <br />                                                                             GSPlp (Fig. 2B). This equivalent substitution mam- <br /> grown glucose glycerol (Fig. 4A, lane 1 versus lane 2              malian ras yeast RAS2 proteins to <br /> lane 3 versus lane 4; Fig. 4B). On hand, GSP2                     dramatically decrease intrinsic GTPase activity (25, 36, <br /> mRNA steady-state levels exhibit relative increase (about                 53). This gspl-val2l allele, cloned a <br /> fourfold; Fig. 4A, lane 5 versus lane 6 lane 7 versus lane               centromeric (pRS316) 2,u (YEp352) plasmid trans- <br /> 8; Fig. 4B) cells grown glycerol versus glucose.                formed strain prp20/2C (at permissive temperature) or <br /> The relative values used generate histogram Fig. 4B               parental strain SS328 reproducibly gave 1,000-fold <br /> were obtained SS328, indistinguishable                   fewer colonies wild-type version GSPI the <br /> from      SEY6210.                                                 vector (typical results shown Fig. 6, which <br />   GSPlp    binds GTP vitro. Antibodies       GSP1                 shows strain SS328 transformed similar amounts 2,u <br /> polypeptide raised fusion GSP1 car-                   plasmids). If GTP-bound form GSPlp stabilized, <br /> boxyl terminus amino terminus GST protein                    lead dominant lethal effect caused the <br /> (Pharmacia) (see Materials Methods). Two months                   improper regulation essential process. Similar obser- <br /> the initial injections subsequent boosts SDS-                      vations decreased numbers viable transformants have <br /> PAGE-purified fusion protein, immunoblot containing 75                   demonstrated equivalent mutation introduced <br />  ,ug total yeast extract shows 23-kDa band reacting               CDC42Sc gene product, involved cell <br /> with GSP1 antiserum (Fig. 5A). These antibodies                    polarity member ras superfamily (58). <br /> used examine GSP1 protein bind GTP                     These results strongly suggest activity GSP1 <br /> in vitro t_-32P-labelled GTP filter binding assay. The                protein requires proper guanine nucleotide metabolism. <br /> autoradiograph SDS-PAGE immunoprecipitations                       GSPlp predominantly localized nucleus. The <br /> 35S-labelled total yeast cell extract preimmune                    PRP20 protein previously located nucleus by <br /> serum antiserum GSPlp (lanes 1 2, respectively)                  immunofluorescence (18). If, assumed suppressor <br /> is presented Fig. SB. A distinct 23-kDa band seen                 screen, PRP20p GSPlp physically interact, would <br /> the antiserum lane, bands precipitated                   predict GSPlp nuclear location. <br /> the preimmune serum present                       Wild-type cells harvested early log phase of <br /> control containing protein A-Sepharose cell                    growth prepared immunofluorescent mapping of <br /> extract (data shown). The protein bands                      GSPlp (13). Although diffuse staining the <br /> coprecipitate GSP1 (43, 40, 39, 32, 28, 17 kDa)                    cytoplasm observed cells, Fig. 7A shows the <br /> coincide size range bands seen                      majority GSP1 protein resides yeast nucleus. <br /> coprecipitate antibody PRP20 protein (18, 31).               (Arrowheads panels A&apos; B&apos; indicate convex ap- <br /> The 53-kDa (Fig. 5B, *) protein likely PRP20p,                   pearance nucleus Nomarski differential inter- <br /> 23-kDa protein GSPlp. Figure 5C SDS-PAGE                     ference contrast. This convex structure readily stains with <br /> duplicate set immune precipitations unlabelled               DNA intercalating fluorescent dyes [13].) No specific stain- <br /> yeast extract used. The proteins gel                      ing seen preimmune serum (Fig. 7B). <br />  2158    BELHUMEUR ET AL.                                                                                          MOL. CELL. BIOL. <br />  <br />                                insert                   GSP1 wt                    GSP1 VAL-21 <br />  <br />  <br />  <br />  <br />   FIG. 6. Dominant lethal phenotype gspl-val2l allele. A similar plasmid DNA introduced SS328 by <br /> transformation selecting Ura+ 30&#194;&#176;C. The vector used YEp352. wt, wild type. <br />  <br />  <br />   Overproduction GSPlp GSP2p prevents nucleolar              GTP-binding proteins intimately involved the <br /> disruption prp20-1 mutant. One phenotypic            activity PRP20. <br /> characteristics prp20-1 mutant allele disruption      Whereas GSPI essential gene, specific <br /> the proper organization nucleoplasm (1). This nucle-        phenotype gsp2 disruption. This result appears incon- <br /> oplasmic disruption visualized using distinctive        sistent protein sequence prediction two <br /> shape fungal nucleolus, single crescent profile,    genes encode identical proteins. An analogous situa- <br /> a marker integrity nucleoplasm (Fig. 8D,            tion reported S. cerevisiae RHOJ RHO2 <br /> arrowheads indicate crescent profile yeast nucle-       ras-like genes (33). It possible that, overex- <br /> olus). In prp20-1 strain, nucleolus frag-          pressed GSP2p complement disruption GSP1, <br /> mented placed nonpermissive temperature            GSP2p additional function assayed. <br /> times short 15 min (1). (Fig. 8A shows prp2O/2C          The slight changes sequences proteins <br /> strain 1 h 34&#194;&#176;C. The arrows indicate fragments         indicative different cellular activities. Differences <br /> nucleolus.) We used nucleolar-specific silver staining      GSPlp GSP2p primary structures consist of <br /> procedure (12) GSP1 GSP2 suppressor strains          conservative amino acid changes, Gly-8 Ala-9 and <br /> demonstrate overabundance of            Glu-204 Asp-205, nonconservative replace- <br /> gene products prp2O/2C strain rescues          ments, Ala-6 Gln-6 Gln-197 His-198, including a <br /> cells prp20-1 ts lethality, prevents         gap (positions 7 8) introduced GSPlp sequence to <br /> fragmentation nucleolus (Fig. 8B C, respectively        allow comparison. It interesting similar <br /> [arrowheads indicate intact nucleoli]).                        minor differences seen mammalian proteins <br />                                                                    RAN TC4 (5, 6). Partial protein sequencing RANp <br />                         DISCUSSION                                 corresponding 60% TC4p revealed there <br />   The PRP20 function associated essential control       differences: positions 206 207 TC4 are <br /> step proper maintenance nuclear organization. To         Asn-Pro, equivalent amino acids RAN are <br /> elucidate nature molecular mechanisms          Thr-Thr. There data available comparison of <br /> PRP20 influences nuclear structure yeast cells,          N termini proteins. It noteworthy these <br /> isolated genes S. cerevisiae encode nuclear            structural changes reside outside (flanking) putative core <br />  <br />  <br />  <br />  <br />   FIG. 7. fmmunofluorescence mapping GSP1 protein yeast nucleus. Log-phase growing wild-type cells stained with <br /> anti-GSP1 antibodies (A A&apos;) preimmune serum (B B&apos;). Arrowheads panels A&apos; B&apos; indicate convex nuclei revealed the <br /> Nomarski differential interference optics Zeiss Optical. The bar panel B&apos; equivalent 5 ,um. <br />  VOL. 13, 1993                                                    GSP1 AND GSP2: NEW S. CEREVISIAE GTP-BINDING PROTEINS                 2159 <br />  <br />                                                                              GSPI-2 homolog (termed SPI1), single gene was <br />                                                                              reported (35). To knowledge, report an <br />                                                                              SPI2 gene S. pombe. <br />                                                                                 GSPlp GSP2p contain highly conserved <br />                                                                              motifs involved guanine nucleotide metabolism, making <br />                                                                              like members ras gene family <br />                                                                              (8). We shown GSPlp binds [32P]GTP vitro. In <br />                                                                              model GSPlp bound GTP (active conforma- <br />                          -4 z. <br />                         -4.                                                  tion) associates PRP20p chromosomes, <br />                             .. <br />                                  v. <br />                                                                              predict mutation GSPlp leading an <br />                                  ,f.   i,                                    increased stabilization activated form chromo- <br />                                                                              result interference DNA replication and <br />                                                                              mitotic arrest likely cause cell death. Using the <br />                                                                              mammalian ras protein model, reproduced this <br />                                                                              similar mutation, changing glycine residue position 21 <br />                                                                              GSPlp valine. In mammals RAS2 yeast cells, <br />         A                                       B                            change results stabilized form ras bound GTP <br />                                                                              (25, 36, 53). This mutated version GSP1 exhibits a <br />                                                                              dominant lethal phenotype independent genomic <br />                                                                              background recipient cells copy number of <br />                                                                              vector. Monitoring cascade events place <br />                                                                              cell dies conditional expression gspl-val2l <br />                                                                              provide interesting clues activities the <br />                                                                              PRP20-GSP1 complex. <br />                                                                                 In addition expected labelling 23-kDa band <br />                                                                              corresponds GSPlp, peptide 17 kDa was <br />                                                                              revealed GTP-binding protein filter assay. We not <br />                    A, .:I,.,I                            111-1 <br />                                                                              believe 17-kDa protein degradation product of <br />                                                                              GSPlp contaminating protein antibody in <br />                                             4                                Western blot analysis, 23-kDa band observed. This <br />                                                                              17-kDa peptide, 32-kDa GTP-binding <br />         c                                       D                            protein, seen fractions eluted double-stranded <br />                                                                              DNA-cellulose column PRP20p GSPlp appear <br />    FIG. 8. Overexpression GSPJ GSP2 gene products                 interdependent manner (31). <br /> prevents fragmentation nucleolus seen prp2o/2C                To support direct interaction two <br /> mutant strain. (A) prp20/2C strain 1 h 34&#194;&#176;C. The arrows             proteins, PRP20p coimmunoprecipitate an <br /> indicate fragments darkly silver-stained nucleo-          antibody raised GSPlp. As well, shown <br /> lus. (B) prp20/2C strain contains GSP1 gene 2pL <br /> plasmid vector. (C) prp2O/2C strain containing GSP2 gene            double-stranded DNA-binding complex containing <br /> 2,u plasmid. (D) prp20/2C strain contains wild-type PRP20          PRP20p GSPlp elutes fast protein liquid <br /> gene 2p plasmid vector. The strains panels B, C, D          chromatography sizing column estimated molecular <br /> all grown overnight 34&#194;&#176;C. The crescent-shaped profile              mass greater 150 kDa (31). To knowledge, are <br /> single, darkly silver-stained nucleolus indicated arrow-           data demonstrate direct physical interactions <br /> heads panels B, C, D. All strains prepared               proteins yeast cells. Although is <br /> staining method described Clark (12) nucleolar-            evidence guanine nucleotide exchange RANp is <br /> specific silver staining yeast cells. The bar panel B equivalent    catalyzed mammalian RCClp (5), recent studies <br /> to 5 g&#194;&#177;m.                                                                    failed direct interactions RCC1 and <br />                                                                              RAN Xenopus egg extracts antibody Xenopus <br />                                                                              RCClp (14). Still, sequence homology TC4IRAN and <br /> domain involved GTP metabolism. In case                        overexpression mammalian RCC1 rescue the <br /> GSP1 GSP2, differences biological                         lethality prp20-1 mutant, reasonable believe that <br /> significance, likely explanation discrep-                 PRP20p catalyze GDP/GTP exchange GSPlp <br /> ancies results disruption experiments                  acts GSPJ-GSP2 guanine nucleotide- <br /> their different levels expression.                                        releasing factor. <br />   The relative abundance mRNAs encoding                        It clear multicomponent PRP20-GSP1 <br /> proteins different. There approximately 10 times                  complex generates control arrangement the <br /> more GSP1 transcript GSP2 transcript. Moreover,                      nucleoplasm; still, obvious GTP hydrolysis <br /> showed GSP2 expression, judged steady-state                       plays essential role maintaining functional conforma- <br /> levels mRNAs, exhibits carbon source dependency.                          tion cellular compartment. More detailed aspects of <br /> These features clearly distinguish GSP1 GSP2.                       activities PRP20-GSP1 complex cell <br /> Given likely GSPlp cells grown                cycle regulation likely studies the <br /> a nonfermentable carbon source, possible GSP2p                    protein components coimmunoprecipitate with <br /> serves     back-up pool GSP1-GSP2 essential                     PRP20p-GSPlp (18 study). Also, continuing with <br /> function. We reported GSP2 essential cells                  analogy GSP1 ras-like proteins, there <br /> are grown glucose 30&#194;&#176;C. We add gsp2 null                      nucleotide diphosphate kinase, catalyze the <br /> strain grows glycerol different temperatures                  transphosphorylation GDP GTP (32). It shown <br /> (data shown). In describing S. pombe TC4/RANI                        nucleotide diphosphate kinase directly transphospho- <br />  2160     BELHUMEUR ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br /> rylates GDP bound tubulin, leads microtubule                      DNA replication. Mol. Cell. Biol. 12:3337-3345. <br /> polymerization (42). Also, genetic interactions                 15. Drivas, G. T., A. Shih, E. Coutavas, M. G. Rush, P. <br /> demonstrated prune locus Drosophila mela-                      D&apos;Eustachio. 1990. Characterization novel ras-like genes <br /> nogaster encoding protein homology GAP                          expressed human teratocarcinoma cell line. Mol. Cell. Biol. <br /> awd, fly nucleotide diphosphate kinase (52). Using S.                       10:1793-1798. <br />                                                                           16. Enoch, T., P. Nurse. 1990. Mutation fission yeast cell <br /> cerevisiae model genes described,                   cycle control genes abolishes dependence mitosis DNA <br /> be possible identify study, biochemical genetic                 replication. Cell 60:665-673. <br /> means, components regulatory systems                   17. Enoch, T., P. Nurse. 1991. Coupling M phase S phase: <br /> used maintenance proper nuclear organization                  controls maintaining dependence mitosis chromosome <br /> enable better understand complex essential                      replication. Cell 65:921-923. <br /> interactions structure nucleoplasm                 18. Fleishmann, M., M. W. Clark, W. Forrester, M. Wickens, T. <br /> nuclear activities.                                                           Nishimoto, M. Aebi. 1991. Analysis yeast prp2O muta- <br />                                                                               tions functional complementation human homologue <br />                                                                               RCC1, protein involved control chromosome con- <br />                       ACKNOWLEDGMENTS                                         densation. Mol. Gen. Genet. 227:417-423. <br />   We thank Howard Bussey group helpful discus-           19. Fourney, R. M., J. Miyakoshi, R. S. Day III, M. C. Paterson. <br /> sions. We thank A. Herscovics (Montreal) H. M. Fried                 1988. Northern blotting: efficient RNA staining transfer. <br /> (Chapel Hill) kindly providing yeast genomic library          Focus 10:5-6. <br /> L29 ribosomal protein clone, respectively, D. Oki assistance      20. Fukumoto, Y., K. Kaibuchi, Y. Hori, H. Fujioka, S. Araki, T. <br /> with manuscript preparation.                                                  Ueda, A. Kikuchi, Y. Takai. 1990. Molecular cloning and <br />   This work supported postdoctoral fellowship               characterization novel type regulatory protein (GDI) for <br /> Medical Research Council (MRC) Canada P.B., graduate                  rho proteins, ras p21-like small GTP-binding proteins. <br /> student fellowship R.T. Fonds recherche en sante du            Oncogene 5:1321-1328. <br /> Quebec (FRSQ), operating grant M.W.C.                  21. Harrow, E., D. Lane. 1988. Antibodies: laboratory man- <br /> National Cancer Institute (NCI) Canada.                                    ual. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />                                                                           22. Hartwell, L., T. Weinert. 1989. Checkpoints: controls that <br />                             REFERENCES                                        ensure order cell cycle events. Science 246:629-634. <br />  1. Aebi, M., M. W. Clark, U. Vijayraghavan, J. Abelson. 1990.        23. Hill, J. E., A. M. Myers, T. J. Koerner, A. Tzagoloff. 1986. <br />     A yeast mutant, PRP20, altered mRNA metabolism                     Yeast/E. coli shuttle vectors multiple unique restriction <br />     maintenance nuclear structure, defective gene              sites. Yeast 2:163-167. <br />     homologous human gene RCCI involved            24. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor- <br />     control chromosome condensation. Mol. Gen. Genet. 224:                 mation intact yeast cells treated alkali cations. J. <br />     72-80.                                                                    Bacteriol. 153:163-168. <br />  2. Becker, D. M., L. Guarente. 1991. High-efficiency transfor-       25. Kataoka, T., S. Powers, C. McGill, 0. Fasano, J. Strathern, J. <br />     mation yeast electroporation. Methods Enzymol. 194:182-             Broach, M. Wigler. 1984. Genetic analysis yeast RAS1 <br />     187.                                                                      RAS2 genes. Cell 37:437-445. <br />  3. Belhumeur, P., N. Fortin, A. Lee, M. W. Clark. Unpub-             26. Kaufer, N. F., H. M. Fried, W. F. Schwindinger, M. Jasin, and <br />     lished data.                                                              J. R. Warner. 1983. Cycloheximide resistance yeast: gene <br />  4. Bischoff, F. R., G. Maier, G. Tilz, H. Ponstingl. 1990. A             protein. Nucleic Acids Res. 11:3123-3135. <br />     47-kDa human nuclear protein recognized antikinetochore            27. Kikuchi, A., T. Sasaki, S. Araki, Y. Hata, Y. Takai. 1989. <br />     autoimmune sera homologous protein encoded                 Purification characterization bovine brain cytosol of <br />     RCC1, gene implicated onset chromosome conden-                GTPase activating proteins specific smg p21, GTP <br />     sation. Proc. Natl. Acad. Sci. USA 87:8617-8621.                          binding protein having effector domain c-ras p2ls. J. <br />  5. Bischoff, F. R., H. Ponstingl. 1991. Catalysis guanine             Biol. Chem. 264:9133-9136. <br />     nucleotide exchange Ran mitotic regulator RCC1.             28. Kunkel, T. A. 1985. Rapid efficient site-directed mutagene- <br />     Nature (London) 354:80-82.                                                sis phenotypic selection. Proc. Natl. Acad. Sci. USA <br />  6. Bischoff, F. R., H. Ponstingl. 1991. Mitotic regulator protein        82:488-492. <br />     RCC1 complexed nuclear ras-related polypeptide.             29. Laemmli, U. K. 1970. Cleavage structural proteins the <br />     Proc. Natl. Acad. Sci. USA 88:10830-10834.                                assembly head bacteriophage T4. Nature (London) <br />  7. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A positive               227:680-685. <br />     selection mutants lacking orotidine-5&apos;-phosphate decarbox-        30. Lapetina, E. G., B. R. Reep. 1987. Specific binding alpha <br />     ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen.       32P GTP cytosolic membrane-bound proteins human <br />     Genet. 197:345-346.                                                       platelets correlates activation phospholipase C. <br />  8. Bourne, H. R., D. R. Sanders, F. McCormick. 1991. The                 Proc. Natl. Acad. Sci. USA 84:2261-2265. <br />     GTPase superfamily: conserved structure molecular mech-           31. Lee, A., M. W. Clark. Unpublished data. <br />     anism. Nature (London) 349:117-127.                                   32. Liotta, L. A., P. S. Steeg. 1990. Clues function of <br />  9. Breviario, D., A. Hinnebusch, J. Cannon, K. Tatchell, R.              nm23 Awd proteins development, signal transduction, <br />     Dhar. 1986. Carbon source regulation RAS1 expression                tumor metastasis provided studies Dictyostelium <br />     Saccharomyces cerevisiae phenotypes ras2- cells.               discoideum. J. Natl. Cancer Inst. 82:1170-1172. <br />     Proc. Natl. Acad. Sci. USA 83:4152-4156.                              33. Madaule, P., R. Axel, A. M. Myers. 1987. Characterization <br />  9a.Bussey, A. H. Personal communication.                                     members rho family yeast Saccharomy- <br /> 10. Carlson, M., D. Botstein. 1982. Two differentially regulated          ces cerevisiae. Proc. Natl. Acad. Sci. USA 84:779-783. <br />     mRNAs different 5&apos; ends encode secreted intracellular        34. Matsui, Y., A. Kikuchi, S. Araki, Y. Hata, J. Kondo, Y. <br />     forms yeast invertase. Cell 28:145-154.                                Teranishi, Y. Takai. 1990. Molecular cloning character- <br /> 11. Chardin, P. 1991. Small GTP-binding proteins ras family:           ization novel type regulatory protein (GDI) smg p25A, <br />     conserved functional mechanism? Cancer Cells 3:117-126.                 ras p21-like GTP-binding protein. Mol. Cell. Biol. 10:4116- <br /> 12. Clark, M. W. 1991. Nucleolar-specific silver staining optical         4122. <br />     electron <br /> </body></html>