<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Kinetic Modelling GlmU Reactions &#226;&#8364;&#8220; Prioritization of <br /> Reaction Therapeutic Application <br /> Vivek K. Singh, Kaveri Das, Kothandaraman Seshadri* <br /> Innovative Medicine Infection (iMed Infection), AstraZeneca India Pvt. Ltd., Bangalore, Karnataka, India <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Mycobacterium tuberculosis(Mtu), successful pathogen, developed resistance existing anti-tubercular drugs <br />      necessitating discovery drugs novel action. Enzymes involved peptidoglycan biosynthesis attractive targets for <br />      antibacterial drug discovery. The bifunctional enzyme mycobacterial GlmU (Glucosamine 1-phosphate N-acetyltransferase/ <br />      N-acetylglucosamine-1-phosphate uridyltransferase) target enzyme drug discovery. Its C- N- terminal <br />      domains catalyze acetyltransferase (rxn-1) uridyltransferase (rxn-2) activities respectively final product is <br />      involved peptidoglycan synthesis. However, bifunctional nature GlmU poses difficulty deciding function <br />      intervened therapeutic advantage. Genetic analysis showed essential gene unclear whether <br />      activities critical cell <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span>. Often enzymatic activity suitable high-throughput assay is <br />      chosen random screening, appropriate biological function inhibited maximal effect. Prediction <br />      rate-limiting function dynamic network analysis reactions option identify appropriate function. <br />      With view provide insights biochemical assays appropriate activity inhibitor screening, kinetic modelling <br />      studies GlmU undertaken. Kinetic model Mtu GlmU-catalyzed reactions built based available kinetic <br />      data Mtu deduction Escherichia coli data. Several model variants constructed including coupled/ <br />      decoupled, varying metabolite concentrations presence/absence product inhibitions. This study demonstrates that <br />      coupled model low metabolite concentrations, inhibition of GlmU reactions cause significant decrement <br />      overall GlmU rate. However higher metabolite concentrations, rxn-2 showed higher decrement. Moreover, with <br />      available intracellular concentration metabolites vivo variant model, uncompetitive inhibition rxn-2 <br />      caused highest decrement. Thus, physiologically relevant metabolite concentrations, targeting uridyltranferase activity of <br />      Mtu GlmU better choice therapeutic intervention. <br />  <br />   Citation: Singh VK, Das K, Seshadri K (2012) Kinetic Modelling GlmU Reactions &#226;&#8364;&#8220; Prioritization Reaction Therapeutic Application. PLoS ONE 7(8): e43969. <br />   doi:10.1371/journal.pone.0043969 <br />   Editor: Benfang Lei, Montana State University, United States America <br />   Received November 3, 2011; Accepted July 30, 2012; Published August 27, 2012 <br />   Copyright: &#195;&#376; 2012 Singh et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The authors support funding report. <br />   Competing Interests: The authors read journal&#226;&#8364;&#8482;s policy following conflicts: This work funded Astra Zeneca, employer all <br />   authors. There patents, products development marketed products declare. This does alter authors&#226;&#8364;&#8482; adherence PLOS ONE policies <br />   sharing data materials, detailed online guide authors. <br />   * E-mail: kothandaraman.seshadri@gmail.com <br />  <br />  <br />  <br />  <br /> Introduction                                                                            phosphate-acetyltransferase/N-acetylglucosamine-1-phosphate- <br />                                                                                         uridyltransferase (GlmU) (Figure 1). GlmU exists bifunc- <br />    Tuberculosis (TB) infectious disease caused                                 tional enzyme bacteria including Mtu, catalysing two <br /> Mycobacterium tuberculosis (Mtu) plagued humans                             consecutive reactions &#226;&#8364;&#8220; first, acetyltransferase reaction convert- <br /> centuries. In 2009 alone, 1.7 million people died TB                           ing alpha-D-glucosamine-1-phosphate (GlcN1P) N-acetyl- <br /> 9.4 million new TB cases reported [1]. The burden                             alpha-D-glucosamine-1-phosphate (GlcNAc1P) coupled with <br /> TB compounded emergence multidrug-                                 conversion acetyl coenzyme A (AcCoA) coenzyme A <br /> resistant (MDR) extensively drug-resistant (XDR) strains                         (CoA) second, uridyltransferase reaction converting <br /> Mtu resistant line second line anti-                    GlcNAc1P UDPGlcNAc utilizing UTP uridyl group donor <br /> TB drugs respectively. Consequently, pressing need                           releasing pyrophosphate (PPi) product (see scheme 1) <br /> for novel anti-TB drugs inhibit novel targets                        [4], [5]. The acetyltransferase (rxn-1; EC 2.3.1.157) and <br /> MDR XDR strains tackled drug                                  uridyltransferase (rxn-2; EC 2.7.7.23) activities enzyme <br /> sensitive Mtu strains.                                                                  catalyzed C- N-terminal domains respectively [4]. <br />    The mycobacterial cell wall consists covalently linked                            This bifunctional nature GlmU offers increased opportu- <br /> complex mycolic acids arabinogalactan, linked                           nity inhibition, given fact enzyme already <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> peptidoglycan. The pathway peptidoglycan biosynthesis                        suggested attractive target TB drugs [6], [7]. <br /> <span id='am-3' about='Thesaurus:Has_Target' typeof='owl:Thing'>has target</span> antibacterial agents                            However, inhibition reactions lead to <br /> cycloserine fosfomycin [2], [3]. In pathway, UDP-N-                            level overall decrement GlmU rate, is <br /> acetyl-D-glucosamine (UDPGlcNAc) essential precursor                          important determine reaction maximal impact as <br /> peptidoglycan synthesized enzyme - glucosamine-1-                         design assays appropriately target enzyme. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                          1                              August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                              Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  <br /> Figure 1. Pathway context GlmU-catalyzed reactions. <br /> doi:10.1371/journal.pone.0043969.g001 <br />  <br />  <br />  <br />  <br /> Scheme1                                                                     biochemical reactions focussed present model (see <br /> Rxn{1(Acetyltransferasereaction): GlcN1PzAcCoA~GlcNAc1PzCoA                 scheme 1). The stoichiometric equations reactions were <br /> Rxn{2(Uridyltransferasereaction): UTPzGlcNAc1P~UDPGlcNAczPPi <br />                                                                             obtained KEGG [15], [16], [17] cross-verified <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                             literature sources (2), [4]. As noticed scheme 1, the <br />                                                                             functioning rxn-2 dependent rxn-1 virtue of <br /> Generalformof orderedbi{bireaction                                          sharing intermediate GlcNAc1P reactions, is <br /> AzB~PzQ                                                                     product rxn-1 acts substrate rxn-2. It is <br /> Rxn{1: A~GlcN1P; B~AcCoA; P~GlcNAc1P; Q~CoA <br />                                                                             noteworthy GlcNAc1P released acetyltransferase <br />                                                                             domain prior binding uridyltransferase domain GlmU <br /> Rxn{2: A~UTP; B~GlcNAc1P; P~UDPGlcNAc; Q~PPi <br />                                                                             [18], thus, eliminating possibility substrate (GlcNAc1P) <br />                                                                             channelling. <br />  <br />    Kinetic modelling offers computational means assess             Rate equations <br /> dynamic behaviour biochemical reaction systems                 It evident GlmU reactions (scheme 1) follow <br /> employed past select drug targets simulating effect        Michaelis Menten ordered bi-bi mechanism [19], [4] involving an <br /> of enzyme inhibition dynamics reaction systems [8], [9],          obligatory order binding substrates enzyme. In GlmU <br /> [10]. It interesting note recently, genome-scale        rxn-1, GlcN1P substrate bind free enzyme (E) <br /> kinetic models attempted [11], [12], typically               followed binding AcCoA E-GlcN1P complex. A <br /> kinetic models restricted small set reactions [13], [14]        similar binding order presumed reverse direction <br /> due requirement enzyme kinetic parameters. GlmU-               GlcNAc1P binds free enzyme followed the <br /> catalysed reactions forms ideal kinetic                  binding CoA. In GlmU rxn-2, UTP binds free enzyme <br /> modelling based analysis, evaluate relative control          followed binding GlcNAc1P reverse direction, <br /> on overall GlmU rate identify inhibition           UDPGlcNAc binds free enzyme followed binding of <br /> would cause maximal effect overall rate.                             PPi [4]. The binding products (substrates reverse reactions) <br />    Reported work construction kinetic model          enzyme taken account include effect of <br /> GlmU-catalyzed reactions Mtu application GlmU                 product inhibition model. <br /> reactions achieving therapeutic goal. Furthermore, basis            Assigning substrates products GlmU reactions the <br /> of modelling studies, preferred mode inhibition              general form ordered bi-bi reaction depicted scheme 1, a <br /> initial metabolite concentrations design vitro assay       general rate equation derived based rapid equilibrium <br /> were proposed, bias assay selection          kinetics following method outlined Segel [20]. The reaction <br /> of specific type inhibitor reaction choice.               equilibria constructed (see Figure 2) derive rate equation <br />                                                                             included: (1) binding products enzyme (2) the <br /> Methods                                                                     binding different types hypothetical inhibitors (I) the <br />                                                                             enzyme, derived general rate equation account <br /> Biochemical reactions model                                          reverse reaction, product inhibition inhibition various <br />   The reactions catalysed GlmU i.e. acetyltransferase (rxn-1)            types hypothetical inhibitors depending values assigned <br /> and uridyltransferase (rxn-2) constitute set                terms contained rate equation. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              2                          August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                           Kinetic Modelling GlmU Reactions <br />  <br />  <br />    Briefly, steps derivation rate equation involve: <br />    (1) Writing general rate equation based concentration <br /> of enzyme complexes yield products                                              v~ <br />                                                                                                            &#194;&#189;A  &#194;&#189;B        &#194;&#189;P  &#194;&#189;Q  <br />                                                                                                         Vf    :    {Vr       : <br />               v~(kz2 &#194;&#189;E{A{B ){(k{2 &#194;&#189;E{P{Q )                                                                KA KB <br />                                                                                                                      <br />                                                                                                                          KP KQ <br />                                                                                                                                                   :::equation   2 <br />                                                                                          &#194;&#189;A  &#194;&#189;P                      &#194;&#189;A  &#194;&#189;B       &#194;&#189;P  &#194;&#189;Q  <br />                                                                                       1z      z     za glmu rxnz        :      z     : <br /> where k+2 k22 turnover numbers forward reverse                           KA KP                       KA KB        KP KQ <br /> reactions respectively.                                                                                                              <br />                                                                                           &#194;&#189;A                    &#194;&#189;P  <br />                                                                                       z       :b A glmu rxn z       :b P glmu rxn <br />    (2) Dividing sides equation total enzyme                          KA                     KP <br /> concentration, [Et], right-hand equation                                                                                 <br />                                                                                           &#194;&#189;A  &#194;&#189;B                        &#194;&#189;P  &#194;&#189;Q  <br /> is expressed sum concentration enzyme species:                   z       :   :b A B glmu rxn z         :   :b P Q glmu rxn <br />                                                                                          KA KB                          KP KQ <br />  <br />  <br /> &#194;&#189;E z&#194;&#189;E{A z&#194;&#189;E{P z&#194;&#189;E{I z&#194;&#189;E{A{B z&#194;&#189;E {P{Q z::: <br />  <br />  <br />    (3) Expressing concentrations enzyme species                    Kinetic parameters model <br /> terms free enzyme concentration derived basis                              The dynamics GlmU-catalyzed reactions simulated by <br /> equilibria (see Figure 2):                                                           solving rate equations; however, solve them, values the <br />                                                                                      terms right-hand equations (such Vf, Vr, KM) <br />                                                                                      need assigned. Kinetic parameters constitute the <br />                           &#194;&#189;A                 &#194;&#189;A  &#194;&#189;B  <br />           &#194;&#189;E{A ~&#194;&#189;E :          ; &#194;&#189;E{A{B ~&#194;&#189;E :    :    ; :::                           terms right-hand rate equations described this <br />                           KA                 KA KB                                   section. The terms dealing initial concentration metabolites <br /> where K,metabolite. = Michaelis constant metabolite specified                 discussed later section. <br /> between ,.                                                                              The kinetic parameters rxn-2 available Mtu from <br />   (4) Performing algebraic operations, moving [Et] right-                     UTP GlcNAc1P vitro concentration response experiments <br /> hand defining Vf ~kz2 &#194;&#189;Et  ; Vr~k{2 &#194;&#189;Et   equation                      carried AstraZeneca (unpublished). However, substituting <br /> becomes:                                                                             values rate equations model did not <br />                                                                                      quantitatively reproduce experimentally obtained vitro <br />                                                                                      concentration response curves UTP GlcNAc1P. The <br />                                                                                      non-agreement difference rate equations used <br /> v~ <br />                                                                                      experiment model. While experimentally <br />                     &#194;&#189;A  &#194;&#189;B        &#194;&#189;P  &#194;&#189;Q                                             obtained data fitted simple Michaelis-Menten (MM) equation, <br />                    Vf    :   {Vr      : <br />                     KA KB         KP KQ <br />                                                                :::equation   1       model simulated ordered bi-bi mechanism in <br />     &#194;&#189;A  &#194;&#189;P  &#194;&#189;I         &#194;&#189;A  &#194;&#189;B       &#194;&#189;P  &#194;&#189;Q          &#194;&#189;A  &#194;&#189;I  <br />  1z    z     z   z         :    z       :      z        :                            accordance previous reports ([19], [4]). To overcome issue, <br />     KA KP Kic          KA KB        KP KQ           KA Kiu A <br />                                                                                      parameter values reproduce experimental <br />     &#194;&#189;P  &#194;&#189;I       &#194;&#189;A  &#194;&#189;B      &#194;&#189;I           &#194;&#189;P  &#194;&#189;Q      &#194;&#189;I  <br />  z     :      z      :     :        z        :    :                                  GlcNAc1P concentration response curve used further <br />     KP Kiu P     KA KB Kiu A B            KP KQ Kiu P Q <br />                                                                                      simulations (see Table 1 parameters used simulation <br />                                                                                      Table S1 information derivation kinetic <br />                                                                                      parameters). GlcNAc1P concentration response curve chosen <br />    In rate equation (equation-1), metabolite inhibitor                   alignment parameter values substrate for <br /> notations [A], [B], [P], [Q] [I] represent                           GlmU rxn-2, UTP, present intracellular milieu high <br /> concentration respective metabolites inhibitor; Vf = max-                     concentration ( = 8.3mM (see Table 2)). At concentration, <br /> imal rate forward direction; Vr = maximal rate reverse                         UTP available enzyme saturating level, is <br /> direction; Kic = inhibition constant hypothetical competitive                     case performing GlcNAc1P concentration response <br /> inhibitor; Kiu_,metabolite. = inhibition constant hypothetical                  experiment. Further, obtain best fit GlcNAc1P <br /> inhibitor uncompetitve metabolite indicated                     concentration curve, parameter estimation functionality CO- <br /> within ,.. Further, terms representing ratios inhibitor                   PASI [21] used. The input data were: (1) list of <br /> concentration inhibition constant rate equation                       GlcNAc1P concentrations corresponding reaction rates <br /> substituted variables follows: a_glmu_rxn = [I]/Kic; b_A_                      obtained experimentally; (2) parameter values obtained <br /> glmu_rxn = [I]/Kiu_A; b_P_glmu_rxn = [I]/Kiu_P; b_A_B_glmu_rxn                       experimentally range varied to <br /> = [I]/Kiu_A_B; b_P_Q_glmu_rxn = [I]/Kiu_P_Q. This representa-                    minimize error experimental simulated <br /> tion eliminates need assumption inhibitor                             reaction rates. Following range parameter values used: <br /> concentration inhibition constants provides                            0.015nmol/min # Vf #0.020nmol/min; 0.04mM # KUTP <br /> normalised measure inhibition strengths various types                       #0.04mM; 0.01mM # KGlcNAc1P #0.1mM experimentally <br /> hypothetical inhibitors. It noticed values                   obtained parameter values are: Vf = 0.020nmol/min; KUTP <br /> such variables zero, rate equation represents                             = 0.04mM; KGlcNAc1P = 0.04mM. Evolutionary programming was <br /> uninhibited enzymatic reaction, effect inhibition                    used method optimization number generations <br /> the model simulated assigning non-zero values                     = 200, population size = 20 random number generator = 1 <br /> variables. Equation-2 resulting general rate equation                   [21]. The best fit (root mean square error = 0.51) obtained <br /> this manipulation. The rate equations GlmU reactions                         parameter values (Vf = 0.020nmol/min; KUTP <br /> deduced general rate equation substituting                          = 0.04mM; KGlcNAc1P = 0.033mM). These values used <br /> corresponding metabolites A, B, P Q (as provided                          simulations. <br /> scheme 1) used model. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       3                             August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                    Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  <br /> Figure 2. Equilibria enzyme species ordered bi-bi mechanism enzymatic reaction. A, B = First second substrate of <br /> enzyme E; P, Q = First second substrates reverse reaction, binding enzyme accounts product inhibition; I = Different types of <br /> hypothetical inhibitor, type determined form enzyme binds to: I binding free E (forming E-I complex) competitive inhibitor <br /> with respect A, I binding E-A complex (forming E-A-I complex) uncompetitive inhibitor respect A I binding E-A-B complex <br /> (forming E-A-B-I complex) uncompetitive inhibitor respect A B; Kic = Inhibition constant hypothetical competitive inhibitor; <br /> Kiu_,metabolite. = Inhibition constant hypothetical uncompetitive inhibitor inhibitor behaves uncompetitive metabolite <br /> indicated ,.. <br /> doi:10.1371/journal.pone.0043969.g002 <br />  <br />     The KM values rxn-1 taken literature [5],            vitro assays. To construct decoupled model, one <br /> Vf abstracted experimental data rxn-2 (as                     hypothetical metabolite GlcNAc1P_2 defined. In decou- <br /> obtained AstraZeneca (unpublished)) scaling factor                 pled model, metabolite act substrate rxn- <br /> literature [5] (see Table 1 parameters used simulation             2 original metabolite representing GlcNAc1P (formed as <br /> and Table S1 information derivation kinetic                      product rxn-1) serve substrate rxn-2. Thus, in <br /> parameters).                                                                   decoupled model, variables exist represent GlcNAc1P: <br />                                                                                GlcNAc1P GlcNAc1P_2. The concentrations two <br /> Variants model                                                          variables independent other, making GlmU <br />     Several model variants constructed explore                     reactions independent (decoupled) other. Concomitant <br /> following possibilities: Presence/absence product inhibition;               change rate equation rxn-2, i.e. GlcNAc1P_2 assigned as <br /> Coupled/decoupled model &#226;&#8364;&#8220; reaction coupling product                  second substrate, represent decoupling. <br /> rxn-1 acting substrate rxn-2; Low/medium/high/                           The concentrations metabolites modulated <br /> intracellular metabolite concentrations &#226;&#8364;&#8220; concentrations                depending condition models represented. The <br /> metabolites kept 0.1xKM, KM, 10xKM intracellular levels;             concentrations precursors products model were <br /> In vitro vivo model &#226;&#8364;&#8220; representing condition biochemical           kept variable in vitro biochemical assay condition constant <br /> assay vs. condition inside cell respectively.                                expected inside vivo cellular environment the <br />     The effect product inhibition simulated assigning             precursor substrates synthesised reactions <br /> literature derived values [22] Michaelis constants relevant              upstream GlmU-catalysed reactions similarly products <br /> products GlmU reactions model, absence                  consumed reactions downstream GlmU- <br /> product inhibition simulated making Michaelis                      catalysed reactions (see Figure 1). <br /> constants equal large number ( = 109mM) the <br /> affinity enzyme products reduces negligible.                 Initial concentrations metabolites <br /> Product inhibition rxn-1 GlcNAc1P rxn-2                              Initial concentrations metabolites acting substrates <br /> UDPGlcNAc accounted model [22] (see Table S1                    kept low ( = 0.1xKM), medium ( = KM), high ( = 10xKM) or <br /> for information derivation kinetic parameters).                      equal intracellular levels (obtained diverse bacterial <br />     The model construction described behaves coupled                 sources including Mycobacterium tuberculosis ATCC 25618 Eco <br /> model intermediate GlcNAc1P acts product                   [23], [24], [25], [26], [27]) simulate dynamics the <br /> rxn-1 substrate rxn-2. Coupled model constructed              reaction diverse scenarios (see Table 2). Initial <br /> as represent dependence rxn-2 rxn-1, product             concentrations metabolites acting products (such as <br /> of rxn-1, GlcNAc1P, serving substrate rxn-2. The evidence             CoA, UDPGlcNAc PPi) kept equal zero the <br /> for coupling comes fact Mtu does              conditions low, medium high metabolite concentrations. <br /> route synthesizing GlcNAc1P GlmU rxn-1 (KEGG                        Under intracellular metabolite concentration scenario, the <br /> [15], [16], [17]). On hand, decoupled model                      concentrations metabolites maintained different levels <br /> constructed aim studying behaviour GlmU                vitro vivo variants models. For vitro variant, the <br /> reaction independent GlmU reaction, usually              concentrations intermediate metabolites (such GlcNAc1P) or <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 4                             August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                                                      Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  Table 1. Kinetic parameters used simulating model.                                      influence different types inhibitors. The simulations were <br />                                                                                                  performed inhibition strength (i.e. I/Ki ratio) kept equal 20, <br />                                                                                                  typical ratio inhibitor concentration its <br />  Rxn-1 Parameters                          Rxn-2 Parameters                                      corresponding inhibition constant. The computations per- <br />                                                                                                  formed Matlab using Matlab toolboxes &#226;&#8364;&#8220; SBML toolbox [29] <br />  Vf               = 0.01089mM/min          Vf               = 0.0004mM/min <br />                                                                                                  SBTOOLBOX2 [30]. <br />  KGlcN1P          = 0.061mM                KUTP             = 0.040Mm <br />  KAcCoA           = 0.224mM                KGlcNAc1Pa       = 0.033mM                            Results Discussion <br />                                            KGlcNAc1P_2b     = 0.033mM <br />  Vr               = 0mM/min                Vr               = 0mM/min <br />                                                                                                  Reproduction vitro concentration response curve <br />                                                                                                     The model aligned GlcNAc1P concentration response curve <br />  KGlcNAc1P        = 0.003mM                KUDPGlcNAc       = 0.132mM/min <br />                                                                                                  able reproduce experimentally observed curve (see <br />  KCoA             = 109mM                  KPPi             = 109mM                              Figure 3). The vitro biochemical assay simulation conditions viz., <br />  a <br />   This parameter present coupled model absent decoupled model; <br />                                                                                                  (1) constant concentration 0.25mM UTP GlcNAc1P; (2) <br />  b <br />   This parameter present decoupled model absent coupled model; See                     depletion substrates; (3) accumulation products, <br />  Table S1 details derivation kinetic parameters.                                       maintained simulation. <br />  doi:10.1371/journal.pone.0043969.t001 <br />  <br />                                                                                                  Product inhibition leads quantitative differences the <br /> metabolites act products (such CoA, UDPGlcNAc                                    model dynamics <br /> and PPi) initialized zero (as expected vitro                                The influence product inhibition dynamical behaviour <br /> assay condition), vivo variant, concentrations                           model investigated building separate versions the <br /> such metabolites initialized respective intracellular                              model (see &#226;&#8364;&#732;&#226;&#8364;&#732;Variants model&#226;&#8364;&#8482;&#226;&#8364;&#8482; section &#226;&#8364;&#732;&#226;&#8364;&#732;Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482; for <br /> levels. Furthermore, decoupled variant model,                                     details). The product inhibition rxn-1 increased progressively <br /> concentration GlcNAc1P_2 kept equal intracellular                                  increasing metabolite concentration increase in <br /> level vitro vivo scenario acts                       normalized product concentration ([product]/Ki_product) relative to <br /> substrate rxn-2; concentration variable represent-                              normalised substrate concentration ([substrate]/KM_substrate) (data not <br /> ing GlcNAc1P maintained described above. As discussed                                   shown). On contrary, product inhibition rxn-2 decreased <br /> the previous section, concentrations precursors products                                  progressively increasing metabolite concentration to <br /> were kept constant represent vivo situation. This achieved                             decrease normalized product concentration ([product]/Ki_product) <br /> by defining metabolites boundary metabolites (i.e.                                       relative normalised substrate concentration ([substrate]/ <br /> boundaryCondition = &#226;&#8364;&#732;&#226;&#8364;&#732;True&#226;&#8364;&#8482;&#226;&#8364;&#8482;), implies said metabolites                           KM_substrate) (data shown). As expected, rates reactions <br /> are boundary reaction modelled                                    higher version product inhibition compared <br /> their concentrateons determined rate equations                                   version product inhibition. Furthermore, difference <br /> when participate reaction(s) (see [28] details).                                caused rate rxn-1 comparatively higher that <br />                                                                                                  caused rate rxn-2 product reaction <br /> Simulation model                                                                          (GlcNAc1P) causes stronger product inhibition rxn-1 (KGlcNAc1P <br />    All described variants model represented                                  = 0.003mM) compared caused product reaction <br /> Systems Biology Markup Language (SBML) [28],                                        (UDPGlcNAc) rxn-2 (KUDPGlcNAc = 0.132mM). These phenom- <br /> standard format represent Systems Biology models. To                                     ena observed both, vitro vivo variants model. <br /> perform simulations, Matlab script written reads                              The product inhibition included simulations <br /> SBML file representing model, performs time course                                       owing experimental evidence [22] provide biological <br /> steady state simulations uninhibited model, followed                               relevance simulations. <br /> time course steady state simulations model the <br />  <br />  Table 2. Initial metabolite concentrations boundary conditions various variants model. <br />  <br />  <br />  Metabolite          Low (mM)       Medium (mM)           High (mM)      Intracellular (mM)                    Intracellular conc/KM                 BoundaryConditiona <br />  <br />                                                                                                                                                      In vivo model       In vitro model <br />  <br />  GlcN1P              0.0061         0.061                 0.61           0.08                                  1.31                                  True                False <br />  AcCoA               0.0224         0.224                 2.24           1.71                                  7.63                                  True                False <br />                                                                                   vivo           vitro          vivo           vitro <br />  GlcNAc1P            0.0            0.0                   0.0            0.082             ; 0.0               2.48             ; 0.0                False               False <br />  GlcNAc1P_2b         0.0033         0.033                 0.33           0.082                                 2.48                                  True                False <br />                                                                                vivo           vitro <br />  CoA                 0.0            0.0                   0.0            1.2             ; 0.0                 -                                     True                False <br />  UTP                 0.004          0.04                  0.4            8.3                                   207.50                                True                False <br />                                                                                vivo           vitro              vivo           vitro <br />  UDPGlcNAc           0.0            0.0                   0.0            9.2             ; 0.0                 69.70             ; 0.0               True                False <br />                                                                                vivo           vitro <br />  PPi                 0.0            0.0                   0.0            0.5             ; 0.0                 -                                     True                False <br />  <br />  a <br />   Boundary condition &#226;&#8364;&#732;&#226;&#8364;&#732;True&#226;&#8364;&#8482;&#226;&#8364;&#8482; metabolite indicates concentration determined set reactions metabolite occurs substrate <br />  product i.e., metabolite boundary reaction component rest model (see [28] details); b This metabolite only <br />  present decoupled version model; vivo Represents vivo variant model; vitro Represents vitro variant model. <br />  doi:10.1371/journal.pone.0043969.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                                5                                              August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                   Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  <br /> Figure 3. Experimental vs. simulated concentration response curves. GlcNAc1P concentration response curve; Curves obtained from <br /> experiment: Black; Curves obtained simulation: Gray; v = Velocity GlmU rxn-2. Assays carried 25uC assay buffer containing <br /> 50 mM Hepes KOH pH 7.5, 5 mM MgCl2. 5 mM DTT, 0.3 units/ml pyrophosphatase phosphate formed detected using malachite green <br /> reagent Innova Biosciences. For GlcNAc1P KM determination UTP fixed 250 mM. <br /> doi:10.1371/journal.pone.0043969.g003 <br />  <br />  <br /> Coupled vs. decoupled models                                                   representing concentrations precursors products as <br />    Coupled model constructed represent dependence                constant. Furthermore, rate rxn-2 decoupled model remained <br /> rxn-2 rxn-1, representing condition closer cellular environ-             constant simulation owing constant supply and <br /> ment. In contrast, decoupled model constructed study                consumption reactants products respectively. <br /> behaviour GlmU reaction independently, usually <br /> done vitro assays. Discussed contrast            In silico inhibition GlmU reactions <br /> the dynamic behaviour versions model presence                  In silico inhibition of GlmU reactions performed <br /> of product inhibition wide range metabolite concentra-               goal prioritize GlmU reaction, inhibiting which, would <br /> tions.                                                                         cause maximal decrement overall GlmU rate. Moreover, <br />    As shown Figure 4 (panels 1A 1B), vitro                     objective inhibiting GlmU reaction(s) disruption of <br /> condition, major difference observed functioning              synthesis end-product (i.e. UDPGlcNAc), rate rxn- <br /> rxn-2 begins delay coupled model, time                 2 used marker overall GlmU reaction rate. It must <br /> period required synthesis GlcNAc1P; contrast                  pointed choice rxn-2 marker overall <br /> decoupled model, rxn-2 begins instantaneously                   GlmU rate based fact synthesis end-product <br /> available non-zero concentration GlcNAc1P_2. This delay                     GlmU reactions proportional rate rxn-2. <br /> shortens increase initial metabolite concentrations.                   In decoupled model, inhibition GlmU rxn-1 caused no <br /> Furthermore, medium metabolite concentrations, rate                  decrement overall Glmu rate. This fact that <br /> rxn-2 initially increases increase GlcNAc1P                     functioning rxn-2 independent rxn-1 decoupled <br /> concentration eventually drops zero                         model. Inhibition rxn-2 followed expected kinetic behaviour <br /> exhaustion UTP (another substrate rxn-2). Such sharp                   potency competitive inhibitor decreased increas- <br /> decrease rate rxn-2 observed decoupled model               ing metabolite concentrations vice versa uncompetitive <br /> owing equal concentration substrates               inhibitors (see right (for vivo condition) right (for <br /> equal KM Mtu. At high metabolite concentration, sharp decrease              vitro condition) sections Figure 5). <br /> in rate rxn-2 observed coupled model                       Inhibition GlmU reactions coupled model in <br /> decoupled model showed gradual decrease rxn-2 rate (data                  vivo condition provided following insights: <br /> not shown), explained drop GlcNAc1P_2                      Inhibitor of GlmU reactions potent low <br /> concentration.                                                                 ( = 0.1xKM) metabolite concentration. As expected, compet- <br />    In contrast vitro condition, vivo simulation showed       itive inhibitors GlmU reactions appeared <br /> reaction proceeds steady state coupled             potent uncompetitive inhibitors low metabolite <br /> well decoupled models (see panels 2A 2B Figure 4). Flux              concentrations. The percent inhibition caused competitive <br /> through reactions coupled model equal asymp-                    inhibitors GlmU reactions similar (see left section column <br /> totically steady state, rates reactions                 &#226;&#8364;&#732;&#226;&#8364;&#732;Low&#226;&#8364;&#8482;&#226;&#8364;&#8482; Figure 5). This indicates reactions have <br /> decoupled model remain unequal steady state. No abrupt drop                 similar control GlmU rate low metabolite concentrations, <br /> in rates of reactions observed observed vitro          apparent computed flux control coefficients <br /> condition. This attributed apparent constant supply              (see Table S2) (see [31] theory flux control coefficient <br /> of precursors constant consumption products caused                   computation). Flux control coefficient computation indicated <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 6                            August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                      Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  <br /> Figure 4. Dynamic behaviour rates GlmU reactions coupled vs. decoupled models. Plots corresponding medium ( = KM) <br /> metabolite concentrations; v = Rates GlmU rxn-1 (broken lines) rxn-2 (solid lines); Panel 1A: In vitro variant GlmU rxn-1; Panel 1B: In vitro variant <br /> GlmU rxn-2; Panel 2A: In vivo variant GlmU rxn-1; Panel 2B: In vivo variant GlmU rxn-2; Coupled model: Black lines; Decoupled model: Gray lines. <br /> doi:10.1371/journal.pone.0043969.g004 <br />  <br /> that control GlmU rate shifts rxn-2                     tained intracellular levels (as obtained diverse <br /> increase metabolite concentrations, evident high                bacterial sources including Mtu ATCC 25618 Eco [23], [24], <br /> potency inhibitors rxn-2 medium high metabolite                     [25], [26], [27]), uncompetitive (against E-UTP-GlcNAc1P <br /> concentrations (see below).                                                      complex) inhibition rxn-2 caused highest decrement in <br />   GlmU rxn-2 inhibition potent medium ( = KM)                          overall GlmU rate. This followed decrement caused by <br /> high ( = 10xKM) metabolite concentration. With increase                       uncompetitive (against E-UTP complex) inhibition rxn-2, which <br /> in initial metabolite concentrations, relative potency                low (,33%) (see left section column &#226;&#8364;&#732;&#226;&#8364;&#732;Intracellular&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> inhibitors rxn-2 increased respect rxn-1                      Figure 5). <br /> inhibitors. At medium metabolite concentrations, decrement                      The high potency uncompetitive inhibition (against E-UTP- <br /> caused uncompetitive (against E-UTP-GlcNAc1P complex)                         GlcNAc1P complex) rxn-2 attributed to: (1) high normalised <br /> inhibition rxn-2 highest, followed competitive                 metabolite concentration (defined asMetabolite      concentration <br />                                                                                                                            K M metabolite ) UTP <br /> and uncompetitive (against E-UTP complex) inhibitions rxn-2, <br />                                                                                  ( = 207.50) intracellular level; (2) significant normalised <br /> which equal. The decrement higher that <br />                                                                                  metabolite concentration GlcNAc1P ( = 24.99) simulation (at <br /> caused competitive uncompetitive inhibitions rxn-1, <br />                                                                                  steady state). <br /> which equal (see left section column <br />                                                                                     Furthermore, potency inhibition largely <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Medium&#226;&#8364;&#8482;&#226;&#8364;&#8482; Figure 5). At high metabolite concentrations, <br />                                                                                  dependent intracellular UTP concentration, be <br /> uncompetitive inhibition (against E-UTP-GlcNAc1P complex) of <br /> rxn-2 caused highest decrement GlmU overall rate. All                     expected tight control cellular environment, the <br /> other types inhibition rxn-2 type inhibition rxn-1            inhibition overcome compensatory alteration in <br /> caused low decrement GlmU overall rate (see left section              UTP concentration. Thus, appears uncompetitive (E-UTP- <br /> column &#226;&#8364;&#732;&#226;&#8364;&#732;High&#226;&#8364;&#8482;&#226;&#8364;&#8482; Figure 5).                                                    GlcNAc1P complex) inhibition rxn-2 best choice to <br />   Uncompetitive inhibition (against E-UTP-GlcNAc1P                               achieve therapeutic goal reducing overall GlmU rate to <br /> complex) GlmU rxn-2 best choice achieve                             ultimately interrupt cell wall biosynthesis process Mtu. <br /> therapeutic objective. When silico inhibition GlmU                            The challenge identify uncompetitive inhibitors that <br /> reactions performed metabolite concentrations main-                     bind E-UTP-GlcNAc1P complex inhibit GlmU rxn- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                   7                             August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                   Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  <br /> Figure 5. Effect silico inhibition GlmU reactions various conditions. Metabolite concentrations used simulation: Low <br /> ( = 0.1xKM), Medium ( = KM), High ( = 10xKM) Intracellular levels; Inhibition strength (I/Ki ratio) maintained 20; Numbers figure indicate <br /> percent decrement GlmU overall rate various types inhibition; Linear color-coded scale Gray White indicating decreasing level of <br /> effect inhibition GlmU rate). <br /> doi:10.1371/journal.pone.0043969.g005 <br />  <br /> 2. Easy sound, finding uncompetitive inhibitors                  relevant metabolite concentrations. As step, vitro <br /> have difficult present case demands                     assay designed screen compound libraries to <br /> inhibitor required bind pocket              identify uncompetitive (against E-UTP-GlcNAc1P complex) <br /> substrate-binding pockets. A recent evidence existence               inhibitors targeting GlmU rxn-2. To select inhibitors an <br /> an allosteric binding site (Haemophilus influenzae (Hin)) GlmU              assay, significant fraction enzyme exist E-UTP- <br /> uridyltransferase (rxn-2 catalysing) domain [32] offers cue               GlcNAc1P complex, necessitates high concentration of <br /> the direction study advance i.e.                      UTP GlcNAc1P assay mixture. The assay a <br /> allosteric binding site exploited using structure-based              decoupled assay, reactions GlmU assayed <br /> approaches design desired uncompetitive inhibitors.               independent GlmU reaction, or, coupled assay <br /> In recent study performed Mtu, analogue                        GlmU reactions assayed the <br /> GlcN1P uncompetitive (against E-GlcN1P                      readout synthesis final end-product i.e. UDPGlcNAc or <br /> complex) inhibitor GlmU rxn-1 albeit poor Ki value                   PPi. In decoupled assay, deciding initial concentrations of <br /> (18.69mM) [33].                                                                metabolites assay mixture trivial task the <br />    Inhibition GlmU reactions coupled model vitro                metabolite concentrations kept low/high select <br /> condition led interesting observation described below:                competitive/uncompetitive inhibitor respectively given <br />    Inhibited rate greater uninhibited rate. In general,                   GlmU reaction. However, coupled assay, initial substrates <br /> the inhibition <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> displayed vitro version model              provided assay are: GlcN1P, AcCoA (i.e. <br /> similar vivo model, phenomenon                   substrates rxn-1) UTP (1st substrate rxn-2). The second <br /> certain metabolite concentrations, GlmU overall rate                    substrate rxn-2, GlcNAc1P, generated action <br /> presence inhibitor exceeded compared absence                      GlmU rxn-1 course assay. Thus maintain the <br /> inhibitor. This resulted slow exhaustion UTP inhibited            concentration GlcNAc1P desired range assay <br /> system compared uninhibited sheer low rate               mixture, initial concentrations GlcN1P AcCoA need to <br /> reaction inhibited system. At initial phase, inhibited rate          chosen carefully GlcNAc1P concentration does not <br /> rxn-2 slower uninhibited (thus conserving UTP),                  rise high cause significant product inhibition rxn-1 and <br /> then pays time point uninhibited rate rxn-2               drop low formation E-UTP-GlcNAc1P <br /> reaches zero inhibited continues residual                complex hindered. <br /> UTP. Such phenomenon usually observed vitro                       The series simulations indicate best maintain the <br /> assays reactions allowed reach late              initial concentrations AcCoA GlcN1P twice KM <br /> stage kinetics. This observed vivo situations             values UTP times KM value assay meant <br /> because UTP supplied GlmU rxn-2 UTP                          screen uncompetitive (against E-UTP-GlcNAc1P complex) <br /> synthesizing pathways.                                                         inhibitors rxn-2. With said initial concentrations of <br />                                                                                AcCoA, GlcN1P UTP, GlcNAc1P concentration hovers <br /> Prediction initial metabolite concentrations suitable                       KM twice KM UTP hovers 6 &#226;&#8364;&#8220; 10 <br /> for vitro biochemical assay screening GlmU                       times KM value. Such higher KM concentrations of <br /> rxn-2 inhibitors                                                               GlcNAc1P UTP lead rise E-UTP-GlcNAc1P <br />  As apparent simulations, uncompetitive (against E-                complex, turn, favour selection uncompet- <br /> UTP-GlcNAc1P complex) inhibition rxn-2 appears cause                     itive (against E-UTP-GlcNAc1P complex) inhibitors rxn-2 <br /> maximal impact overall GlmU rate physiologically                  (see Figure 6). With higher initial concentrations AcCoA and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 8                            August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                               Kinetic Modelling GlmU Reactions <br />  <br />  <br />  <br />  <br /> Figure 6. Dynamics GlcNAc1P (black line) UTP (gray line) normalised concentrations proposed assay condition. <br /> The normalized concentrations GlcNAc1P UTP stay 1 significant portion simulation time period, favourable <br /> condition assay identifying uncompetitive (against E-UTP-GlcNAc1P complex) inhibitors GlmU rxn-2. <br /> doi:10.1371/journal.pone.0043969.g006 <br />  <br /> GlcN1P, concentration GlcNAc1P rise. This lead                          bias assay selection type inhibitor. Thus <br /> to high product inhibition rxn-1, undesirable                         current work presents example application of <br /> assay. It noted rate equations model                       computational approaches early stages drug discovery so <br /> derived rapid equilibrium assumption (see &#226;&#8364;&#732;&#226;&#8364;&#732;Rate equations&#226;&#8364;&#8482;&#226;&#8364;&#8482;                         make informed choices target preferred mode <br /> section &#226;&#8364;&#732;&#226;&#8364;&#732;Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482; details), implies rate                       inhibition late-stage failures avoided at <br /> catalysis slower kinetic processes                         minimised. We premise present work help <br /> binding dissociation substrate, product inhibitor. In                          identify high-affinity uncompetitive inhibitors GlmU rxn-2 and <br /> case assumption breaks down, concentrations GlmU,                         potentiate optimisation lead drug molecules <br /> substrates inhibitor important                               treatment tuberculosis. <br /> determining vitro potency inhibitor the <br /> factors binding dissociation rate constants. However,                         Supporting Information <br /> whether rapid equilibrium holds not, selection of <br /> uncompetitive (against E-UTP-GlcNAc1P complex) inhibitor                                  Table S1 Derivation GlmU kinetic parameters. <br /> would stipulate concentrations UTP GlNAc1P                                (PDF) <br /> be high E-UTP-GlcNAc1P complex available the <br />                                                                                           Table S2 Flux control coefficients GlmU reactions <br /> assay mixture desired uncompetitive inhibitor would <br />                                                                                           various metabolite concentrations. <br /> bind. The prediction suitable assay condition improve <br />                                                                                           (PDF) <br /> further details Mtu associated kinetics product <br /> inhibition available. <br />                                                                                           Acknowledgments <br /> Conclusion                                                                                We thank Janani Venkatraman early inputs simulation design. We <br />   GlmU essential enzyme synthesis important                           thank Vaishali Humnabadkar, Gulebahar Sheikh Sunita DeSousa for <br /> precursor peptidoglycan biosynthesis, attractive                          concentration-response experiments relating Figure 3. We gratefully <br /> anti-TB drug target. In study, kinetic modelling paradigm                        acknowledge Dr.Anand Kumar Dr.Santanu Datta critical <br /> used simulate dynamics GlmU-catalyzed reactions                          suggestions manuscript. <br /> predict effect inhibition GlmU reactions overall <br /> GlmU rate. Based simulations,                                Author Contributions <br /> inhibition GlmU rxn-2, preferably uncompetitive inhibitor                         Conceived designed experiments: KS KD VKS. Performed the <br /> against E-UTP-GlcNAc1P complex, cause maximal                                   experiments: VKS. Analyzed data: VKS KS KD. Contributed <br /> impact GlmU rate physiologically relevant metabolite                             reagents/materials/analysis tools: KD KS VKS. Wrote paper: KS <br /> concentrations. Further, initial metabolite concentrations                       VKS KD. <br /> coupled biochemical assay mixture predicted to <br />  <br /> References <br /> 1. Who (2010) World Health Organization: Tuberculosis Data.                                3. Wishart DS, Knox C, Guo AC, Shrivastava S, Hassanali M, et al. (2006) <br /> 2. Wishart DS, Knox C, Guo AC, Cheng D, Shrivastava S, et al. (2008) DrugBank:                DrugBank: comprehensive resource silico drug discovery and <br />    knowledgebase drugs, drug actions drug targets. Nucleic acids research           exploration. Nucleic acids research 34: D668&#226;&#8364;&#8220;D672. <br />    36: D901&#226;&#8364;&#8220;D906. <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                            9                             August 2012 | Volume 7 | Issue 8 | e43969 <br />                                                                                                                                          Kinetic Modelling GlmU Reactions <br />  <br />  <br />  4. Zhang Z, Bulloch EMM, Bunker RD, Baker EN, Squire CJ (2009) Structure                     Activity Escherichia coli GlmU. Antimicrobial Agents Chemotherapy 53: <br />     function GlmU Mycobacterium tuberculosis. Acta Crystallographica                      2306&#226;&#8364;&#8220;2311. <br />     Section D: Biological Crystallography 65: 275&#226;&#8364;&#8220;283.                                      20.   Irwin HS (1993) Enzyme Kinetics: Behavior analysis rapid equilibrium <br />  5. Zhou Y, Xin Y, Sha S, Ma Y (2011) Kinetic properties Mycobacterium                         steady-state enzyme systems: New York: Wiley Classics Library Edition, <br />     tuberculosis bifunctional GlmU. Archives Microbiology 193: 751&#226;&#8364;&#8220;757.                        John Wiley &amp; Sons, Inc. 22&#226;&#8364;&#8220;24 p. <br />  6. Sassetti CM, Boyd DH, Rubin EJ (2003) Genes required mycobacterial                  21.   Hoops S, Sahle S, Gauges R, Lee C, Pahle Jr, et al. (2006) COPASIa&#203;&#8224;J&#226;&#8364;&#8482;&#226;&#8364;&#8482;a <br />     growth defined high density mutagenesis. Molecular microbiology 48: 77&#226;&#8364;&#8220;84.                 COmplex PAthway SImulator. Bioinformatics 22: 3067&#226;&#8364;&#8220;3074. <br />  7. Zhang W, Jones VC, Scherman MS, Mahapatra S, Crick D, et al. (2008)                     22.   Mengin-Lecreulx D, Van Heijenoort J (1994) Copurification glucosamine-1- <br />     Expression, essentiality, microtiter plate assay mycobacterial GlmU,                phosphate acetyltransferase N-acetylglucosamine-1-phosphate uridyltrans- <br />     bifunctional glucosamine-1-phosphate acetyltransferase N-acetylgluco-                 ferase activities Escherichia coli: characterization glmU gene product as <br />     samine-1-phosphate uridyltransferase. International Journal Biochemistry               bifunctional enzyme catalyzing subsequent steps pathway UDP- <br />     Cell Biology 40: 2560&#226;&#8364;&#8220;2571.                                                                   N-acetylglucosamine synthesis. Journal bacteriology 176: 5788&#226;&#8364;&#8220;5795. <br />  8. Bakker BM, Michels PAM, Opperdoes FR, Westerhoff HV (1997) Glycolysis                23.   Vadali RV, Bennett GN, San KY (2004) Cofactor engineering intracellular <br />     bloodstream form Trypanosoma brucei understood terms                         CoA/acetyl-CoA effect metabolic flux redistribution Escherichia <br />     kinetics glycolytic enzymes. Journal Biological Chemistry 272: 3207&#226;&#8364;&#8220;                coli. Metabolic Engineering 6: 133&#226;&#8364;&#8220;139. <br />     3215.                                                                                   24.   Marcellin E, Nielsen LK, Abeydeera P, Kro&#194;&#168;mer JO (2009) Quantitative analysis <br />  9. Eisenthal R, Cornish-Bowden A (1998) Prospects antiparasitic drugs. Journal               intracellular sugar phosphates sugar nucleotides encapsulated <br />     Biological Chemistry 273: 5500&#226;&#8364;&#8220;5505.                                                       streptococci using HPAEC PAD. Biotechnology Journal 4: 58&#226;&#8364;&#8220;63. <br /> 10. Singh VK, Ghosh I (2006) Kinetic modeling tricarboxylic acid cycle               25.   Bennett BD, Kimball EH, Gao M, Osterhout R, Van Dien SJ, et al. (2009) <br />     glyoxylate bypass Mycobacterium tuberculosis, application to <br />                                                                                                   Absolute metabolite concentrations implied enzyme active site occupancy in <br />     assessment drug targets. Theoretical Biology Medical Modelling 3: 27. <br />                                                                                                   Escherichia coli. Nature chemical biology 5: 593&#226;&#8364;&#8220;599. <br /> 11. Jamshidi N, Palsson B&#195;&#732; (2008) Formulating genome-scale kinetic models the <br />                                                                                             26.   Newton GL, Arnold K, Price MS, Sherrill C, Delcardayre SB, et al. (1996) <br />     post-genome era. Molecular Systems Biology 4: 171. <br />                                                                                                   Distribution thiols microorganisms: mycothiol major thiol most <br /> 12. Smallbone K, Simeonidis E, Swainston N, Mendes P (2010) Towards genome- <br />                                                                                                   actinomycetes. Journal bacteriology 178: 1990&#226;&#8364;&#8220;1995. <br />     scale kinetic model cellular metabolism. BMC Systems Biology 4: 6. <br /> 13. Jamshidi N, Edwards JS, Fahland T, Church GM, Palsson BO (2001) Dynamic                 27.   Kukko-Kalske E, Lintunen M, Inen MK, Lahti R, Heinonen J (1989) <br />     simulation human red blood cell metabolic network. Bioinformatics 17:                  Intracellular PPi concentration directly dependent inorganic <br />     286&#226;&#8364;&#8220;287.                                                                                      pyrophosphatase Escherichia coli K-12 cells. Journal bacteriology 171: <br /> 14. Young JD, Henne KL, Morgan JA, Konopka AE, Ramkrishna D (2008)                                4498&#226;&#8364;&#8220;4500. <br />     In <br /> </body></html>