<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Oct. 2011, p. 1295&#226;&#8364;&#8220;1305                                                                                           Vol. 10, No. 10 <br /> 1535-9778/11/$12.00 doi:10.1128/EC.05093-11 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />   The Essentiality Fungus-Specific Dam1 Complex Is Correlated <br />      One-Kinetochore&#226;&#8364;&#8220;One-Microtubule Interaction Present <br />          Cell Cycle, Independent Nature <br />                            Centromere&#228;&#338;&#164;&#226;&#8364;&#160; <br />                                              Jitendra Thakur Kaustuv Sanyal* <br />                  Molecular Mycology Laboratory, Molecular Biology Genetics <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Jawaharlal Nehru Centre for <br />                                  Advanced Scientific Research, Jakkur, Bangalore 560 064, India <br />                                                Received 27 April 2011/Accepted 29 April 2011 <br />  <br />              A fungus-specific outer kinetochore complex, Dam1 complex, essential Saccharomyces cerevisiae, nones- <br />           sential fission yeast, absent metazoans. The reason reductive evolution functionality this <br />           complex remains unknown. Both Candida albicans Schizosaccharomyces pombe regional centromeres as <br />           opposed short-point centromeres S. cerevisiae. The interaction microtubule kinetochore is <br />           established S. cerevisiae C. albicans early cell cycle, contrast multiple <br />           microtubules bind kinetochore mitosis S. pombe. Moreover, Dam1 complex associated <br />           kinetochore cell cycle S. cerevisiae C. albicans mitosis S. pombe. <br />           Here, Dam1 complex essential viability indispensable proper mitotic chromosome <br />           segregation C. albicans. The kinetochore localization Dam1 complex independent kinetochore- <br />           microtubule interaction, function complex monitored spindle assembly checkpoint. Strikingly, <br />           Dam1 complex required prevent precocious spindle elongation premitotic phases. Thus, constitutive <br />           kinetochore localization associated one-microtubule-one kinetochore type interaction, <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> <br />           centromere, correlated essentiality Dam1 complex. <br />  <br />  <br />    The separation sister chromatids mitosis driven                envelope breaks mitosis. Finally, number of <br /> by dynamic interaction macromolecular struc-                     MTs bind single chromosome differs organ- <br /> tures: (i) kinetochore (KT), complex proteinaceous struc-                isms: Saccharomyces cerevisiae MT binds KT <br /> ture built centromere DNA (10, 47), (ii) mitotic                (52), S. pombe MTs bind KT (13), in <br /> spindle formed microtubules (MTs). The process chro-                     metazoans multiple MTs bind KT (20 25 humans) <br /> mosome segregation studied extensively wide                      (34). Thus, considering timing onset KT-MT <br /> range organisms, simple unicellular yeasts humans.                  interaction, number MTs associated KT and <br /> While major sequence events required chromosome                     <span id='am-87' about='xsp:pattern' typeof='owl:Thing'>pattern</span> breakdown nuclear envelope, grad- <br /> segregation remains same, species-specific differ-                   ual evolution evident budding yeast fission yeast <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> ences exist. The microtubule organizing centers (MTOCs)                     metazoans. <br /> budding yeasts, called spindle pole bodies (SPBs), embed-                     The process chromosome segregation mediated nu- <br /> ded nuclear envelope assemble intranuclear                       clear (kinetochore nonkinetochore/interpolar) cyto- <br /> mitotic spindle (5). Thus, interaction spindle MTs                 plasmic (astral) MTs originate SPBs: kinetochore <br /> KTs does require breakdown nuclear envelope                     microtubules (kMTs) connect SPBs KTs generate pole- <br /> during closed mitosis budding yeasts, KT-MT inter-                  ward pulling forces sister chromatid separation ana- <br /> action established early cell cycle (12, 23). The fission            phase A. The Dam1 complex (described below) S. cerevisiae <br /> yeast Saccharomyces pombe undergoes closed mitosis (23),                  shown coupler transduces MT depoly- <br /> but SPBs remain outside nuclear envelope inter-                     merization activity poleward pulling forces. Nonkineto- <br /> phase (27) spindle MTs nucleated interact                     chore/interpolar MTs (IPMTs), interdigitate spin- <br /> KTs duplicated SPBs enter nuclear mem-                       dle midzone help plus-end MT binding proteins (4, <br /> brane following mitotic initiation (14). Metazoan cells, con-               32), elongate generate outward pushing forces further <br /> trast, undergo open mitosis (22, 45) MTs, origi-                  separate sister chromatids anaphase B. Cytoplasmic/ <br /> nating MTOCs, gain access KTs nuclear                    astral MTs protrude cytoplasm (41) regulate <br />                                                                                spindle length alignment (20). The interaction astral <br />    * Corresponding author. Mailing address: Molecular Mycology Lab-            MTs cell cortex generates backward force acts on <br /> oratory, Molecular Biology Genetics <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Jawaharlal Nehru Cen-            spindle poles keeps specific distance apart <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> tre Advanced Scientific Research, Jakkur, Bangalore 560 064, In-           (7, 33, 39, 41, 46). <br /> dia. Phone: 91 80 2208 2878. Fax: 91 80 2208 2766. E-mail: sanyal                 Unlike fission yeast metazoans, KT-MT interaction <br /> @jncasr.ac.in. <br />    &#226;&#8364;&#160; Supplemental material article http://ec <br />                                                                                starts premitotic phase S. cerevisiae (5). Several lines of <br /> .asm.org/.                                                                     evidence suggest mitotic spindle S. cerevisiae kept <br />    &#228;&#338;&#164; <br />      Published ahead print 13 May 2011.                                  short anaphase onset facilitate KT-MT attachment and <br />  <br />                                                                         1295 <br />  1296       THAKUR AND SANYAL                                                                                                                    EUKARYOT. CELL <br />  <br />  <br /> to avoid premature chromosome segregation (2, 30, 31). Thus,                         Construction conditional mutants DAM1 (Orf19.4837), ASK1 <br /> a proper KT-MT interaction important spindle length                        (Orf19.4675), SPC19 (Orf19.4473), DAD2 (Orf19.3551). The copy of <br />                                                                                   DAM1, ASK1, SPC19 replaced HIS1 strain SN148. 5&#226;&#172;&#732; Sequences <br /> maintenance premitotic phases budding                          long primers (see Table S3 supplemental material) homolo- <br /> yeast cell cycle. Improper KT-MT interaction (unattached KTs                      gous sequences upstream downstream gene 3&#226;&#172;&#732; ends were <br /> or improper tension) activates spindle assembly checkpoint                    homologous HIS1 gene used construct deletion cassettes PCR <br /> (SAC) (Mad1p, Mad2p, Mad3p, Bub1p, Bub2p, Bub3p,                           using plasmid HIS1GFP (19) template. Each deletion construct carries <br />                                                                                   90 110 bp upstream downstream homology regions flanked HIS1 <br /> and Mps1p SAC yeast) prevents meta- <br />                                                                                   gene. Transformants selected complete medium lacking histidine (CM- <br /> phase-anaphase transition (36). Eventually inactivation                    His medium). In resulting strains J101, J103, J105, remaining wild-type <br /> SAC proper KT-MT interaction allows sister chromatid                           allele DAM1, ASK1, SPC19, respectively, placed control of <br /> separation.                                                                       MET3 promoter. We cloned open reading frame (ORF), includ- <br />    A fungus-specific 10-protein outer KT complex, namely,                     ing start codon gene (that carries unique restriction site), BamHI <br />                                                                                   (DAM1 ASK1) BamHI/PstI (SPC19) fragments corresponding <br /> Dam1 complex, shown involved KT-MT                              site(s) pCaDis (6). Each resulting plasmids linearized ClaI <br /> interaction S. cerevisiae (1, 18, 26, 38, 49). In organism,               (for DAM1 ASK1) SwaI (for SPC19) transformed J101, J103, or <br /> mitotic spindles shown highly compromised                      J105 rise J102, J104, J106, respectively (see Table S2 the <br /> various mutants Dam1 complex (8, 9, 16, 24). All 10                        supplemental material genotypes strains used). <br /> proteins Dam1 complex essential viability, local-                     To delete copy DAD2, cloned HIS1 gene pBluescript KS <br />                                                                                   II(&#226;&#171;&#186;) EcoRI/BamHI fragment generate pDad2-1. We cloned <br /> ized KT cell cycle, help chromo-                     DAD2 upstream (523 bp) downstream (583 bp) sequences sites <br /> some segregation forming rings MTs S. cerevi-                    flanking HIS1 pDad2&#226;&#338;&#172;1 BamHI/SacI HindIII/KpnI fragments, respec- <br /> siae (48). In contrast, Dam1 complex localized KT                   tively. The resulting plasmid pDad2-3 digested SacI KpnI and <br /> only mitosis proteins complex                   transformed BWP17 (51) rise J107. To place remaining allele <br />                                                                                   DAD2 control PCK1 promoter, URA3 sequence cloned <br /> essential viability fission yeast, mutations these <br />                                                                                   HindIII fragment site pBluescript rise pDad2-4. <br /> proteins lead increased chromosome missegregation. Func-                       Subsequently, pDad2-5 generated cloning DAD2 upstream sequence (459 <br /> tional differences complex conserved process                    bp) KpnI XhoI fragments pDad2pck1a. The PCK1 promoter was <br /> chromosome segregation raises interesting question:                        released pCA01 (29) BamHI-BglII fragment cloned the <br /> redundancy essentiality Dam1 complex corre-                      BamHI site pDad2-5 pDad2-6. Finally, pDad2-7 constructed by <br />                                                                                   cloning DAD2 ORF pDad2-6 BamHI-NotI fragment. A 3.6-kb <br /> lated timing KT-MT interaction (premitotic                         KpnI-NotI fragment released pDad2-7 used transform J107 to <br /> mitotic) localization complex (constitutive                rise J108. <br /> or mitotic) KT, type centromere (point                     Construction mad2 null mutant conditional double mutants the <br /> regional) determines essentiality complex?                       Dam1 complex proteins mad2 null mutant background. To delete both <br />                                                                                   copies MAD2, deletion cassettes constructed. MAD2 upstream and <br />    The putative homologs 10 components Dam1 <br />                                                                                   downstream sequences PCR amplified (see Table S1 supplemental <br /> complex present Candida albicans (35). C. albicans,                    material) cloned BamHI/SmaI SalI/XhoI sites pBluescript to <br /> most commonly isolated fungal pathogen, causes candidiasis                     generate pMad2-1. The construct deletion copy MAD2 was <br /> immunocompromised patients (15, 21, 50). In C. albicans                       generated cloning Candida maltosa LEU2 (37) (PCR amplified vector <br /> nuclear spindle visible cell cycle, nuclear                pSN42, gift Suzanne M. Noble) EcoRI SalI sites pMad-1. The <br />                                                                                   resulting plasmid pMad2-2 digested SacII XhoI used trans- <br /> envelope remains intact mitosis (17), KT-MT in-                        form C. albicans SN148, J102, J104, J106 rise J109, J111, J113, and <br /> teraction established early cell cycle (42, 44).                    J115, respectively. To delete remaining copy MAD2, constructed <br /> Similar S. cerevisiae, MT binds KT C.                         pMad2-3 cloning C. albicans ARG4 SmaI SalI sites pMad2-1. The <br /> albicans (25). C. albicans 3- 5-kb-long CENP-A-rich                        resulting plasmid pMad2-3 digested BamHI XhoI transformed <br />                                                                                   J109, J111, J113, J115 rise J110, J112, J114, J116, respec- <br /> centromere sequences unique different each <br />                                                                                   tively. <br /> other various chromosomes (3, 43). The properties C.                           Construction protein A-tagged strains. A tandem affinity purification (TAP) <br /> albicans centromeres appear intermediate                      tag cassette chromosomally fused C terminus DAM1 DAD2 <br /> of S. cerevisiae (125-bp-long centromeres) S. pombe (40-                   gene. A calmodulin binding domain (CBD) protein A domain two <br /> 110-kb-long centromeres), C. albicans good                    epitopes present TAP tag. The CaURA3-TAP cassette amplified from <br />                                                                                   vector pPK335 (11) using long primer pairs (see Table S3 supplemental <br /> study functional evolution Dam1 complex. <br />                                                                                   material) <span id='am-84' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> locus (Dam1 Dad2) 5&#226;&#172;&#732; end a <br />    In study, proteins                     CaURA3-TAP cassette 3&#226;&#172;&#732; end. These TAP-tagging cassettes trans- <br /> evolutionarily conserved outer KT complex, namely, Dam1,                          formed C. albicans J101 J107, allele gene was <br /> Ask1, Dad2, Spc19, essential viability                        deleted using CaHIS1 rise J117 J118. All strains analyzed for <br /> localized KT cell cycle C. albicans. The                 correct integration confirmatory PCRs using primers listed Table S3 the <br />                                                                                   supplemental material. <br /> KT recruitment Dam1 complex, however, indepen-                             Media growth conditions. Conditional mutant strains J102, J104, and <br /> dent MTs. Our results Dam1 complex                       J106, carrying DAM1, ASK1, SPC19, respectively, control the <br /> involved KT-MT-mediated process chromosome seg-                         MET3 promoter grown YPDU (1% yeast extract, 2% peptone, 3% <br /> regation mitosis, including spindle length maintenance                     glucose, 0.01% uridine) permissive medium YPDU plus 5 mM <br />                                                                                   cysteine plus 5 mM methionine nonpermissive medium. Conditional mutant <br /> throughout cell cycle. Based results, propose <br />                                                                                   strain J108, carrying DAD2 control PCK1 promoter, grown <br /> that essentiality Dam1 complex correlated                       YPSU (1% yeast extract, 2% peptone, 2% succinate 0.01% uridine) the <br /> with timing onset KT-MT interaction                         permissive medium YPDU nonpermissive medium. All C. albicans <br /> number MTs KT type centromere.                         strains grown 30&#194;&#176;C. <br />                                                                                      Viability assays. Conditional mutants DAM1, ASK1, SPC19, DAD2 were <br />                                                                                   grown overnight inducing medium (YPU-succinate PCK1 promoter- <br />                        MATERIALS AND METHODS <br />                                                                                   driven expression CM&#226;&#8364;&#8220;Met-Cys MET3 promoter-driven expression). <br />   Strain construction. Strains used study listed Table S2   Cells grown overnight pelleted down, washed water, transferred into <br /> supplemental material.                                                            repressive media (YPU plus 3% glucose PCK1 promoter repression YPDU <br />  VOL. 10, 2011                                                                        Dam1 COMPLEX IS ESSENTIAL IN CANDIDA ALBICANS                                  1297 <br />  <br />  <br />  <br />  <br />    FIG. 1. Dam1 complex localized kinetochore cell cycle C. albicans. (A) Western blot analysis using total cell lysates <br /> from BWP17 (wild type) J117 (Dam1-Prot A) J118 (Dad2-Prot A) cells anti-protein A antibodies (top). DAPI-stained cells J117 <br /> (Dam1TAP) J118 (Dad2TAP), different stages cell cycle, immunostained anti-protein A (red) anti-tubulin (green) <br /> antibodies. (B) Indirect immunostaining anti-Dad2 (red) anti-myc (Mif2) (green) antibodies C. albicans strain CAMB1 expressing a <br /> key KT component, Myc-Mif2. (C) Anti-protein A ChIP DNA J117 J118 cells analyzed PCR primers specific CEN7 <br /> (assembly 21-CaChr7 coordinates 427369 427560) region 17 kb away (assembly 21-CaChr7 coordinates 444584 444875) (FAR <br /> CEN7). T, total DNA; &#226;&#171;&#185;, IP DNA anti-protein A antibodies; &#226;&#171;&#186;, bead-only control antibodies. <br />  <br />  <br />  <br /> plus 5 mM Met plus 5 mM Cys MET3 promoter repression), grown                 DAPI. Palette adjustment performed optimal intensity each <br /> 30&#194;&#176;C. Cells collected different time points, counted, plated        image. Z stacks collected 0.4- 0.5-&#226;?&#174;m intervals, stacked projection <br /> plates containing inducing medium. The number viable cells formed               images processed Adobe Photoshop. <br /> colonies counted 2 days incubation 30&#194;&#176;C. Percent viability            ChIP assay. ChIP assays anti-CENP-A antibodies performed as <br /> plotted different time points (see Fig. 4; Fig. S1 supplemental      described (40, 43). ChIP assays anti-protein A antibodies J117 <br /> material).                                                                              (Dam1-Prot A) J118 (Dad2-Prot A) strains performed using same <br />    DAPI staining, calcofluor staining, FACS analysis. Various C. albicans           protocol following modifications: J117 cells cross-linked with <br /> strains grown overnight inducing medium. These cells pelleted              1% formaldehyde 90 min, J118 cells cross-linked 120 min. The <br /> down, washed water, transferred repressible media, grown 30&#194;&#176;C.         DNA recovered reversing cross-linking precipitated using herring <br /> Cells harvested various time points growth repressible media       sperm DNA ethanol. <br /> were fixed 70% alcohol 10 min. Cells pelleted down, sus- <br /> pended water, incubated 4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino-2-phenylindole (DAPI) solu- <br /> tion (50 ng/ml) 10 min. For calcofluor staining, cells incubated in <br /> calcofluor solution (no. 18909; Sigma) 5 min subsequently washed with <br />                                                                                                                           RESULTS <br /> 10% potassium hydroxide solution. For fluorescent-activated cell sorter (FACS) <br />                                                                                            Identification Dam1 complex C. albicans. Using <br /> analysis, cells collected various time points fixed, subsequent steps <br /> were followed described previously (44). Images captured using LSM           polypeptide sequences S. cerevisiae Dam1 complex compo- <br /> 510 META software using laser confocal microscope (Carl Zeiss, Germany).              nents queries BLAST analysis, identified putative <br />    Nocodazole treatment. For nocodazole treatment, J118 (dad2/DAD2TAP)                  homologous open reading frames (ORFs) of 10 <br /> cells grown overnight YPDU reinoculated YPDU initial             proteins form Dam1 complex C. albicans (see Table <br /> optical density 600 nm (OD600) 0.2. Nocodazole (no. M1404; Sigma) was <br /> added concentration 20 &#226;?&#174;g/ml OD600 0.4 (one generation) was <br />                                                                                         S1 supplemental material). By using conserved domain <br /> achieved. Cells grown additional 4 h harvesting immuno-          search service (NCBI), conserved motifs 10 <br /> localization chromatin immunoprecipitation (ChIP) assays.                           ORFs identified. These motifs share considerable simi- <br />    Subcellular immunolocalization. Subcellular immunolocalization per-              larity amino acid sequence. The overall sequence similarity, <br /> formed using protocols described (44). The dilutions primary antibod- <br /> ies used following: affinity-purified rabbit anti-Dad2 antibody, 1:50; <br />                                                                                         however, low cases. <br /> rabbit anti-protein A antibody (no. P3775; Sigma), 1:1,000; affinity-purified rab-         The Dam1 complex localized kinetochore through- <br /> bit anti-CENP-A antibody (44), 1:500; mouse anti-Myc (no. OP10L; Calbio-                cell cycle. The subcellular localization Dam1 and <br /> chem), 1:50; rat anti-tubulin (no. YOL1/34; Invitrogen), 1:100. The fluorescent         Dad2 C. albicans performed indirect immunofluo- <br /> secondary antibodies obtained Invitrogen used dilution of <br />                                                                                         rescence microscopy using anti-protein A antibodies strains <br /> 1:500 Alexa Fluor goat anti-rabbit IgG 568 (no. A11011), 1:100 Alexa <br /> Fluor goat anti-rat IgG 488 (no. A11006), 1:500 Alexa Fluor anti-mouse          J117 (dam1/DAM1TAP) J118 (dad2/DAD2TAP), respec- <br /> 488 (no. A11001). Coimmunolocalization experiments performed using             tively. The full-length copy DAM1 DAD2 present in <br /> same protocol.                                                                          J117 J118 fused protein A, epitope the <br />    Microscopy, image capture, image processing. All microscopic images              tandem affinity purification (TAP) tag, C terminus of <br /> were taken LSM 510 META software using laser confocal microscope (Carl <br /> Zeiss, Germany) following lasers specific flurophores: Ar laser (band <br />                                                                                         ORF (strains listed Table S2 supplemental <br /> pass, 500 550 nm) Alexa Fluor 488, He/Ne laser (band pass, 565 615 nm)        material). Western blot analysis confirmed expression of <br /> for Alexa fluor 568, two-photon laser near infrared (band pass, &#226;&#172;&#402;780 nm)          protein A-tagged Dam1 Dad2 (Fig. 1A, top). We observed <br />  1298     THAKUR AND SANYAL                                                                                                 EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 2. Kinetochore localization Dad2 does depend integrity kinetochore-microtubule interaction C. albicans. (A) Untreated <br /> (DMSO) nocodazole-treated (NOC) J118 (dad2/DAD2TAP) cells stained DAPI anti-protein A anti-tubulin antibodies. A <br /> barely visible signal anti-tubulin antibodies NOC-treated cells confirmed loss spindle structures. However, Dad2 signals are <br /> comparable DMSO-treated NOC-treated cells. (B) ChIP assays anti-protein A antibodies using DMSO-treated NOC-treated J118 <br /> cells confirmed Dad2 recruitment centromeres independent kinetochore-microtubule interaction. T, total DNA; &#226;&#171;&#185;, IP with <br /> anti-protein A antibodies; &#226;&#171;&#186;, bead-only control. <br />  <br />  <br /> bright dot-like signals cells colocalized DAPI-        mitotic spindle disrupted. Untreated cells exhibited <br /> stained nuclei (Fig. 1A). Each dot represents cluster 16           normal spindles (Fig. 2A). However, intensities Dad2 <br /> centromeres. Unbudded cells exhibited dot cell close            TAP signals NOC-treated cells comparable in <br /> to SPB, large-budded cells later stages cell        untreated cells, suggesting disruption spindle <br /> cycle exhibited dots cosegregated nuclei con-             did alter Dad2 localization KTs (Fig. 2A). To further <br /> nected mitotic spindles. The localization patterns suggest           confirm result, performed ChIP assays untreated and <br /> that Dad2 Dam1 present KTs cell               NOC-treated J118 cells using anti-protein A antibodies. The <br /> cycle. The colocalization Dad2 key KT component,               immunoprecipitated (IP) DNA samples analyzed PCR <br /> Mif2p (Myc tagged), strain CAMB1 (43) confirmed             using specific set primers designed CEN regions all <br /> Dad2 KT localized (Fig. 1B). To determine vivo associa-           chromosomes (primers listed Table S3 the <br /> tions proteins centromeres, performed          supplemental material). No significant difference Dad2 TAP <br /> standard chromatin immunoprecipitation (ChIP) assays               binding CEN regions observed (Fig. 2B). <br /> anti-protein A antibodies strains J117 J118. The immu-           Taken together, results confirm Dad2 localization at <br /> noprecipitated (IP) DNA samples analyzed PCR using              KTs independent KT-MT interaction. <br /> a specific set primers designed CEN7 sequences (prim-              The Dam1 complex essential viability C. albicans. <br /> ers listed Table S3 supplemental material). CEN7          To examine essentiality Dam1 complex C. albi- <br /> regions J117 J118 enriched IP DNA, indi-             cans, constructed conditional mutants genes, <br /> cating Dam1 Dad2 (Fig. 1C) associated                DAM1, ASK1, DAD2, SPC19. The copy gene <br /> centromeres. Taken together, results indicate Dam1           replaced HIS1 marker, remaining copy was <br /> and Dad2, components Dam1 complex, KT localized              placed control regulatable promoter (the MET3 <br /> throughout cell cycle C. albicans.                               promoter DAM1, ASK1, SPC19 PCK1 pro- <br />    Kinetochore recruitment Dam1 complex indepen-              moter DAD2). The MET3 promoter repressed the <br /> dent presence microtubules. To examine depen-             presence cysteine (Cys) methionine (Met) (6), the <br /> dency Dam1 complex spindle MTs KT recruit-            PCK1 promoter repressed glucose used car- <br /> ment, observed KT localization Dad2 presence           bon source (29). Strains J102 (dam1/MET3prDAM1), J104 <br /> of spindle poison nocodazole (NOC). Indirect immunoflu-             (ask1/MET3prASK1), J106 (spc19/MET3prSPC19) failed <br /> orescence performed untreated (DMSO only) no-                grow plates containing 5 mM Cys 5 mM Met (Fig. 3). <br /> codazole-treated J118 (dad2/DAD2TAP) cells using anti-tubu-             Similarly, strain J108 (dad2/PCK1prDAD2) failed grow <br /> lin anti-protein A antibodies. Nocodazole treatment led          glucose used carbon source. The parent strains <br /> G2/M arrest followed elongated bud <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>phenotype</span>              J101 (dam1/DAM1), J103 (ask1/ASK1), J105 (spc19/SPC19), <br /> barely visible tubulin staining (Fig. 2A). The absence tubulin       J107 (dad2/DAD2) did growth defects. The <br /> staining G2/M-arrested NOC-treated cells confirmed              inability conditional mutants grow repressible <br />  VOL. 10, 2011                                                           Dam1 COMPLEX IS ESSENTIAL IN CANDIDA ALBICANS                     1299 <br />  <br />  <br />                                                                            conditions. A prolonged incubation (&#226;&#172;&#381;6 h) led transition <br />                                                                            large bud elongated bud (pseudohyphal) pheno- <br />                                                                            type case (Fig. 4A; Fig. S1A supple- <br />                                                                            mental material). FACS analysis mutants confirmed <br />                                                                            majority cells 4N DNA content 4 h <br />                                                                            protein depletion (Fig. 4B; Fig. S1B). Almost all <br />                                                                            arrested cells large buds (99%) depleted of <br />                                                                            Dam1 Ask1 single undivided DNA masses (Fig. <br />    FIG. 3. Dam1 complex essential viability C. albicans. C. <br />                                                                            4C). On hand, depletion Spc19 Dad2 <br /> albicans cells expressing wild-type copies (YFG/YFG), de-         resulted large-budded cells containing single undi- <br /> leted copy, remaining wild-type copy control          vided nucleus (75 90%) unequally divided nuclei (10 <br /> either native promoter (yfg/YFG) MET3/PCK1 promoter             25%) (Fig. 5). Nuclear missegregation increased drastically <br /> (yfg/MET3prYFG yfg/PCK1prYFG) indicated gene (DAM1, <br />                                                                            2 6 h growth nonpermissive conditions in <br /> ASK1, SPC19, DAD2) streaked plates containing inducible <br /> (&#226;&#171;&#186;CM Suc) repressible (&#226;&#171;&#185;CM Glu) medium (CM, 5 mM                  cases concomitant loss cell viability, suggesting that <br /> cysteine 5 mM methionine; Suc, succinate; Glu, glucose). The           chromosome missegregation sole cause loss cell <br /> MET3 promoter, expressed absence CM (&#226;&#171;&#186;CM)           viability mutant cells (Fig. 4D; Fig. S1C). The <br /> repressed presence CM (&#226;&#171;&#185;CM), used controlled         parent strains J101 (Dam1), J103 (Ask1), J105 (Spc19), and <br /> expression DAM1, ASK1, SPC19. The PCK1 promoter, is <br /> expressed presence succinate (Suc) repressed glucose      J107 (Dad2), grown similar conditions, exhibited normal <br /> (Glu), used control expression DAD2. Plate photographs       cell cycle progression (with obvious cell cycle arrest), proper <br /> were taken 2 3 days incubation 30&#194;&#176;C.                        nuclear segregation, unaltered cell viability throughout <br />                                                                            growth similar conditions (Fig. 4D; Fig. <br />                                                                            S1C). All results strongly indicate Dam1 <br /> conditions confirmed genes essential            complex C. albicans essential proper nuclear segrega- <br /> for viability.                                                             tion, G2/M progression cell cycle, comple- <br />    The Dam1 complex required G2/M progression                       tion mitosis. <br /> through cell cycle completion mitosis C. albicans.              Depletion essential kinetochore protein C. albicans <br /> Conditional mutant strains DAM1 (J102), ASK1 (J104),                    leads severe spindle defects. Most cells Dam1 <br /> SPC19 (J106), DAD2 (J108) showed accumulation                   conditional mutants grown nonpermissive conditions ar- <br /> large budded cells 4 h growth nonpermissive                 rested large bud stage unsegregated nucleus each <br />  <br />  <br />  <br />  <br />    FIG. 4. Dam1 complex required G2/M progression cell cycle completion mitosis C. albicans. (A) DIC images <br /> showing phenotypes parent strain (dam1/DAM1) conditional dam1 mutant (dam1/MET3prDAM1) (0 h) (8 h) <br /> growth medium repressive MET3 promoter. (B) FACS analysis showing cellular DNA content strains 6 h growth <br /> in medium repressive MET3 promoter. (C) Fixed wild-type cells cells depleted Dam1 stained DAPI 4 h growth in <br /> repressive medium. Dam1-depleted cells single unsegregated nuclei stuck mother bud neck, parent cells undergo proper <br /> nuclear segregation. (D) Graphs showing proportion cells exhibiting improper nuclear division wild-type cells (gray triangles) or <br /> conditional mutant cells (blue circles) collected various time points growth nonpermissive medium. Cell viability wild-type (gray <br /> diamonds) mutant (red squares) strains corresponding time points shown graph. Ask1, Spc19, Dad2 mutants show <br /> similar results (see Fig. S1 supplemental material). Bar, 5 &#226;?&#174;m. <br />  1300     THAKUR AND SANYAL                                                                                                      EUKARYOT. CELL <br />  <br />  <br />                                                                            cases. We examined compared spindle mor- <br />                                                                            phology known inner KT mutants CENP-A (Cse4) and <br />                                                                            CENP-C (Mif2) conditional mutant strains Dam1 <br />                                                                            complex, namely, J102 (Dam1), J104 (Ask1), J106 (Spc19), and <br />                                                                            J108 (Dad2), grown nonpermissive conditions 4 h <br />                                                                            (partial depletion) 8 h (nearly complete depletion). As <br />                                                                            opposed wild type (BWP17) (Fig. 6), conditional mutant <br />                                                                            strains CAKS3b (CSE4/PCK1prCSE4) CAMB2 (mif2/ <br />                                                                            PCK1prMIF2), grown nonpermissive conditions, <br />                                                                            showed arrested cells G2/M stage short <br />                                                                            (&#226;&#172;&#381;85%) medium-long (&#226;&#172;?15%) spindles (Fig. 6). Two the <br />                                                                            subunits Dam1 complex study, Spc19- and <br />                                                                            Dad2-depleted cells, showed similar spindle defects (Fig. 6). <br />                                                                            The spindle defects exhibited depletion other <br />                                                                            subunits, Dam1 Ask1, different the <br />                                                                            rest KT mutants C. albicans. <br />                                                                               Subunits Dam1 complex maintain dynamics inter- <br />                                                                            polar astral MTs. Astral MTs highly dynamic, and <br />                                                                            appearance undetectable wild-type bud- <br />                                                                            ding hyphal cells C. albicans (Fig. 7A B). Dam1- <br />                                                                            depleted cells showed uniform short-spindle phenotype but <br />   FIG. 5. Proportion cells indicated phenotypes showing        lacked astral MTs (Fig. 7C D, upper). More 45% of <br /> chromosome missegregation Dam1- Dad2-depleted cells.                 dam1 mutant cells exhibited SPB-like dots situated very <br />  <br />  <br />  <br />  <br />    FIG. 6. Depletion essential kinetochore protein C. albicans leads severe spindle defects. Wild-type cells grown YPDU or <br /> YPDU plus 10% serum, conditional mutants inner kinetochore proteins (CENP-A/Cse4 CENP-C/Mif2) tested <br /> outer kinetochore proteins Dam1 complex (Dad2 Spc19) grown nonpermissive medium 4 h (left) 8 h (right), fixed, and <br /> stained DAPI anti-tubulin antibodies. The percentage cells specific spindle phenotypes associated strain calculated <br /> from cells elongated bud stage (right). Bar, 5 &#226;?&#174;m. <br />  VOL. 10, 2011                                                        Dam1 COMPLEX IS ESSENTIAL IN CANDIDA ALBICANS                         1301 <br />  <br />  <br />  <br />  <br />   FIG. 7. Dam1 Ask1, subunits Dam1 complex, maintain polymerization/depolymerization dynamics interpolar astral <br /> microtubules. Wild-type cells grown YPDU (A) YPDU plus 10% serum (B) conditional mutants Dam1 (dam1prMET3DAM1/dam1) <br /> and Ask1 (ask1prMET3ASK1/ask1) grown nonpermissive medium 4 h (C E) 8 h (D F) fixed stained DAPI and <br /> anti-tubulin antibodies. More 50% short spindles resulting Dam1 depletion showed staining IPMTs midzone thus <br /> appeared dots (probably representing SPBs) situated close (C D, lower). (E F) Almost short spindles <br /> (&#226;&#172;&#381;99%) caused Ask1 depletion resulted extensive growth astral microtubules cell cortex. The percentage cells with <br /> specific spindle phenotypes associated strain calculated cells elongated bud stage (extreme right). Bar, 5 &#226;?&#174;m. <br />  <br />  <br />  <br />  <br /> close barely visible MTs two <br /> dots, indicating apparent absence interpolar MTs (Fig. <br /> 7C D, lower). Interestingly, Ask1-depleted cells showed a <br /> uniform phenotype short, misaligned spindles a <br /> brightly stained extensive network cytoplasmic (astral) mi- <br /> crotubules (Fig. 7E F). <br />    Dad2 localized spindle midzone. Western blot <br /> analysis immunostaining Dad2 antibodies confirmed <br /> that strain J108 (dad2/PCK1prDAD2) expressed Dad2 suc- <br /> cinate media levels higher wild type <br /> (BWP17) (Fig. 8A B). The overexpression Dad2 did not <br /> alter chromosome segregation fidelity cell viability com- <br /> pared wild type (data shown). Immuno- <br /> staining anti-Dad2 antibodies J108 overexpressing <br /> Dad2 revealed bright dot-like signal midzone in <br /> addition KT localization (the dot-like signals <br /> are KTs near SPBs) (Fig. 8C; Fig. S2 the <br /> supplemental material). In extremely rare cases (&#226;&#172;?1% total <br /> population), Dad2 signals visible spindle <br /> axis (see Fig. S2). However, midzone localization rarely                 FIG. 8. Dad2 localized spindle midzone. Dad2 over- <br />                                                                           expressed control PCK1 promoter succinate-con- <br /> visible wild-type BWP17 cells (DAD2/DAD2) (Fig. 8C),                   taining medium. (A) Western blot analysis cell lysates the <br /> probably low cellular levels Dad2 present                wild-type strain (BWP17) Dad2-overexpressing (O/E) strain J108 <br /> spindle midzone. Thus, addition KT localization, Dad2           (dad2/PCK1prDAD2) grown succinate using anti-Dad2 anti- <br /> probably localized plus end nuclear microtu-             PSTAIRE (loading control) antibody. (B) BWP17 J108 cells were <br /> bules.                                                                    grown succinate, fixed, stained DAPI anti-Dad2 anti- <br />                                                                           bodies. (C) Succinate-grown wild-type BWP17 Dad2-overexpress- <br />    In Dad2- Spc19-depleted cells observed, albeit            ing J108 (dad2/PCK1prDAD2) cells fixed stained DAPI <br /> infrequently, unbundled fibers mitotic spindles connecting             anti-tubulin anti-Dad2 antibodies. Apart usual KT <br /> two nuclear masses (Fig. 8D).                                             localization (two spots overlapping DAPI-stained nuclear mass), <br />    The spindle assembly checkpoint (SAC) monitors KT-MT                   additional signal spindle midzone (shown arrow) was <br />                                                                           visible anaphase cells. (D) Dad2-depleted cells (J108 grown in <br /> interaction mediated Dam1 complex. Several lines                glucose) fixed stained DAPI anti-tubulin antibody. <br /> evidence described indicate subunits                The unbundling spindle fibers observed (although rarely) in <br /> Dam1 complex C. albicans essential KT-MT-                   long short spindles. Bar, 5 &#226;?&#174;m. <br />  1302     THAKUR AND SANYAL                                                                                                      EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 9. Function Dam1 complex surveillance Mad2-mediated spindle assembly checkpoint C. albicans. (A) J102 <br /> (dam1/MET3prDAM1) J112 (dam1/MET3prDAM1 mad2/mad2) cells grown nonpermissive medium 4 h stained DAPI. <br /> J102 (dam1 mutant) cells exhibited large bud arrest unsegregated DNA mass, mother-bud neck spindle assembly <br /> checkpoint active. J112 (dam1 mad2 double mutant) cells grown similar conditions, hand, exhibited cells various stages <br /> of cell cycle, including large budded cells improperly segregated nuclear masses. FACS analysis J102 J112 cells indicated times <br /> of growth nonpermissive medium shown (lower). (B) J116 (spc19MET3/pr SPC19 mad2/mad2) cells grown nonpermissive medium <br /> for 8 h (depleted Spc19) stained calcofluor white DAPI. Spc19-depleted cells absence Mad2 led formation of <br /> cells multiple buds fragmented nuclear masses. Ask1 Dam1 showed similar phenotypes. <br />  <br />  <br />  <br />  <br /> mediated process chromosome segregation. To examine                     material), suggesting defects budding the <br /> whether large bud arrest various Dam1 complex mutants               Dam1 complex subunits depleted mad2 mutant <br /> was activation SAC, senses improper                strain. <br /> KT-MT interaction, deleted copies MAD2, es-                     Dam1 complex-dependent KT-MT interaction restricts <br /> sential SAC component, conditional mutant strains J102,                 spindle length premitotic cells avoid premature chromo- <br /> J104, J106 construct conditional double mutant                      segregation. Tubulin staining J112 (mad2/mad2 dam1/ <br /> strains J112 (mad2/mad2 dam1/MET3prDAM1), J114 (mad2/                      MET3prDAM1), J114 (mad2/mad2 ask1/MET3prASK1), and <br /> mad2 ask1/MET3prASK1), J116 (mad2/mad2 spc19/                          J116 (mad2/mad2 spc19/MET3prSPC19) strains grown non- <br /> MET3prSPC19). Microscopic observation (differential inter-                 permissive conditions exhibited long spindles (&#226;&#172;&#402;5 &#226;?&#174;m) un- <br /> ference contrast [DIC] imaging) J112 cells grown non-                budded cells (Fig. 10A). The control parent strain J110 (mad2/ <br /> permissive conditions MET3 promoter showed large                 mad2) showed single dot-like signal stained anti- <br /> bud arrest phenotype (Fig. 9A) prolonged growth                 tubulin antibodies, probably representing SPB, in <br /> (&#226;&#172;&#381;8 h). The appearance unbudded G1 cells 2N DNA                    unbudded cells (Fig. 10A). This result eliminates possibility <br /> content determined FACS analysis confirmed G2/M                    Mad2 restricting spindle length premitotic cells. <br /> arrest caused Dam1 depletion relieved absence             Immunostaining anti-Cse4 antibodies cells exhib- <br /> Mad2-mediated checkpoint surveillance (Fig. 9A). In spite               ited Cse4 dot-like signal associated single <br /> release G2/M arrest, observed improvement                    nuclear mass, revealing occurrence unclustering of <br /> viability growth rate double mutants compared             KTs (the precocious separation chromosomes) were <br /> those conditional single Dam1 complex mutants. The                  unattached improperly attached MTs (Fig. <br /> viability double mutants marginally               10B). These results, previous observations (spindle <br /> single mutants (Fig. 9B), significant change growth              defects metaphase), suggest proper KT-MT <br /> rate observed (data shown). These results confirm             interaction mediated Dam1 complex required re- <br /> the function Dam1 complex mediating KT-MT inter-                 strict spindle elongation premitotic cells. Figure 10C sum- <br /> actions surveillance Mad2-dependent spin-              marizes various roles Dam1 complex components differ- <br /> dle assembly checkpoint C. albicans. Cells J114 (mad2/               ent stages cell cycle C. albicans. <br /> mad2 ask1/MET3prASK1) J116 (mad2/mad2 spc19/ <br /> MET3prSPC19) conditional double mutants showed                                                   DISCUSSION <br /> similar results (data shown). Dam1-, Ask1-, Spc19- <br /> depleted cells underwent division absence Mad2.                    In work, subunits Dam1 com- <br /> However, cells behaved abnormally subsequent cell                 plex, evolutionarily conserved fungus-specific protein com- <br /> cycle stages. We observed multiple budded cells chains               plex present outer KT, essential viability the <br /> cells attached multiple (&#226;&#172;&#381;2)                  human pathogenic yeast C. albicans. In spite low level of <br /> DAPI-stained nuclear masses (see Fig. S3 supplemental               overall homology protein sequence S. cerevi- <br />  VOL. 10, 2011                                                        Dam1 COMPLEX IS ESSENTIAL IN CANDIDA ALBICANS                         1303 <br />  <br />  <br />  <br />  <br />    FIG. 10. Dam1 complex restricts spindle length premitotic cells avoid precocious chromosome separation. (A) Cells indicated <br /> genotypes grown nonpermissive medium fixed stained DAPI anti-tubulin antibodies. Unbudded cells exhibited long spindles <br /> approaching metaphase length (2 5 &#226;?&#174;m) Dam1 complex mutants Mad2 deleted. The ask1 mad2 double mutant showed <br /> branched spindles unbudded cells. Unbudded multiply budded cells formed elongated buds (due stress response) showed <br /> aberrantly segregated nuclear masses connected mitotic spindle. The mad2 mutant cells depleted Spc19 showed unbundled <br /> mitotic spindle fibers connecting aberrantly segregated nuclear masses. (B) Cells indicated genotype grown nonpermissive media were <br /> fixed stained DAPI anti-Cse4 anti-tubulin antibodies. Unbudded cells long spindles exhibited Cse4 dots associated with <br /> a single nuclear mass, indicating precocious separation chromosomes premitotic cells. (C) A model highlighting role Dam1 <br /> complex maintenance spindle length morphology cell cycle. Bar, 5 &#226;?&#174;m. <br />  <br />  <br /> siae counterparts, subunits Dam1 complex                dramatic gradual change requirement Dam1 <br /> conserved function chromosome segregation. Subcellular                 complex chromosome segregation species be <br /> localization ChIP assays confirm Dam1 complex             correlated evolution complexity mecha- <br /> associated KT cell cycle. The deple-              nism chromosome segregation. In S. cerevisiae C. albi- <br /> tion of proteins Dam1 complex results             cans, MT binds KT, KT-MT interaction is <br /> in loss cell viability concomitant increase                established early cell cycle, contrast to <br /> chromosome missegregation. As result, conditional mutant                MTs KT S. pombe, KT-MT interaction is <br /> strains nonpermissive conditions arrest large bud stage             established mitosis. Interestingly, Dam1 com- <br /> mostly short spindles. Although KT localization           plex shown localized KTs are <br /> Dam1 complex independent KT-MT interaction,                 associated spindle (i.e., cell cycle S. <br /> cell cycle arrest G2/M surveillance Mad2-          cerevisiae C. albicans mitosis S. pombe). <br /> dependent spindle assembly checkpoint, confirming evolu-              Hence, essentiality Dam1 complex correlated with <br /> tionarily conserved role Dam1 complex KT-MT in-                 maintaining proper MT-one KT interaction, is <br /> teraction shown previously S. cerevisiae S. pombe. The             established interphase (Fig. 11). <br /> Dam1 complex shown involved restricting                   The KT localization S. cerevisiae Dam1 shown to <br /> premature spindle elongation premitotic cells.                         compromised absence spindle MTs (16, 30). How- <br />    Despite indispensable role chromosome segregation            ever, results immunofluorescence ChIP assays the <br /> S. cerevisiae, Dam1 complex nonessential fission yeast          present study unequivocally prove KT recruitment of <br /> and absent metazoans, indicating re-              Dam1 complex dependent spindle MTs C. <br /> ductive evolution complex adaptive evolution            albicans. This suggests functional differences in <br /> alternative mechanism governs KT-MT interaction. Such                Dam1 complex S. cerevisiae C. albicans terms of <br />  1304     THAKUR AND SANYAL                                                                                                         EUKARYOT. CELL <br />  <br />  <br />                                                                      premitotic cells. This led propose Dam1 complex <br />                                                                      restricts interphase spindle length elongation. Figure 10 sum- <br />                                                                      marizes comparative analysis Dam1 complex from <br />                                                                      different organisms. Future experiments test essentiality <br />                                                                      Dam1 complex C. albicans creating conditions that <br />                                                                      allow MT binding kinetochore may <br />                                                                      provide direct link function Dam1 com- <br />                                                                      plex number MTs KT. The essentiality the <br />                                                                      Dam1 complex, exclusive presence fungal <br />                                                                      species mammalian homolog, makes attractive <br />                                                                      target develop safer potent antifungal drug to <br />                                                                      treat candidiasis. <br />  <br />                                                                                                ACKNOWLEDGMENTS <br />                                                                        We thank B. S. Suma help confocal microscopy. <br />         FIG. 11. Reductive evolution Dam1 complex.              This work supported grant Department Bio- <br />                                                                      technology, Government India, K.S., JNCASR. <br />                                                                        J.T. senior research fellow supported Department of <br />                                                                      Biotechnology, Government India. <br /> their mechanistic roles KT-MT interaction. Indeed, we <br /> found spindle defects associated Dam1 Ask1,                                         REFERENCES <br /> subunits Dam1 complex C. albicans, unique (dis-         1. Asbury, C. L., D. R. Gestaut, A. F. Powers, A. D. Franck, T. N. Davis. <br />                                                                          2006. The Dam1 kinetochore complex harnesses microtubule dynamics to <br /> cussed below) observed S. cerevisiae.               produce force movement. Proc. Natl. Acad. Sci. U. S. A. 103:9873&#226;&#8364;&#8220;9878. <br />    We observed spindle defects associated Spc19         2. Bachant, J., S. R. Jessen, S. E. Kavanaugh, C. S. Fielding. 2005. The <br />                                                                          yeast S phase checkpoint enables replicating chromosomes bi-orient and <br /> Dad2 similar inner (Cse4/CENP-A Mif2/                restrain spindle extension S phase distress. J. Cell Biol. 168:999&#226;&#8364;&#8220;1012. <br /> CENP-C; present study) middle (Mis12/Mtw1 [42]) KT                3. Baum, M., K. Sanyal, P. K. Mishra, N. Thaler, J. Carbon. 2006. For- <br /> proteins C. albicans. Mtw1, protein present          mation functional centromeric chromatin specified epigenetically in <br />                                                                          Candida albicans. Proc. Natl. Acad. Sci. U. S. A. 103:14877&#226;&#8364;&#8220;14882. <br /> linker layer kinetochore, showed similar spindle defects       4. Brinkley, B. R., J. Cartwright. 1971. Ultrastructural analysis mitotic <br /> in C. albicans (42). Thus, majority cells short          spindle elongation mammalian cells vitro. Direct microtubule counts. <br /> spindles small population medium-long spindles             J. Cell Biol. 50:416&#226;&#8364;&#8220;431. <br />                                                                       5. Byers, B., L. Goetsch. 1975. Behavior spindles spindle plaques in <br /> found common consequence kinetochore mutants                  cell cycle conjugation Saccharomyces cerevisiae. J. Bacteriol. <br /> irrespective layer presumably present.          124:511&#226;&#8364;&#8220;523. <br />                                                                       6. Care, R. S., J. Trevethick, K. M. Binley, P. E. Sudbery. 1999. The MET3 <br /> However, outer kinetochore proteins study,             promoter: new tool Candida albicans molecular genetics. Mol. Micro- <br /> Dam1 Ask1, exhibited exclusively short spindle pheno-             biol. 34:792&#226;&#8364;&#8220;798. <br /> type single nuclear mass depleted C. albicans        7. Carminati, J. L., T. Stearns. 1997. Microtubules orient mitotic <br />                                                                          spindle yeast dynein-dependent interactions cell cortex. <br /> cells. This observation suggests possible functional difference        J. Cell Biol. 138:629&#226;&#8364;&#8220;641. <br /> between proteins rest analyzed         8. Cheeseman, I. M., et al. 2001. Implication novel multiprotein Dam1p <br /> the layers kinetochore. Indeed, S. cerevisiae       complex outer kinetochore function. J. Cell Biol. 155:1137&#226;&#8364;&#8220;1145. <br />                                                                       9. Cheeseman, I. M., M. Enquist-Newman, T. Mu      &#194;&#168;ller-Reichert, D. G. Drubin, <br /> Dam1 Ask1 promote chromosome segregation                  G. Barnes. 2001. Mitotic spindle integrity kinetochore function <br /> more efficiently KT proteins tethered             linked Duo1p/Dam1p complex. J. Cell Biol. 152:197&#226;&#8364;&#8220;212. <br />                                                                      10. Cleveland, D. W., Y. Mao, K. F. Sullivan. 2003. Centromeres and <br /> noncentromeric sequence plasmid chromo-              kinetochores: epigenetics mitotic checkpoint signaling. Cell 112:407&#226;&#8364;&#8220; <br /> some conditionally inactivated centromere (26, 28).               421. <br />    Besides short long spindles, components showed       11. Corvey, C., et al. 2005. Carbon source-dependent assembly Snf1p <br />                                                                          kinase complex Candida albicans. J. Biol. Chem. 280:25323&#226;&#8364;&#8220;25330. <br /> mutant-specific defects spindle morphology C. albicans.        12. De Souza, C. P., S. A. Osmani. 2007. Mitosis, just open closed. <br /> Most short spindles Dam1 mutants                Eukaryot. Cell 6:1521&#226;&#8364;&#8220;1527. <br /> very weak staining midzone, probably           13. Ding, R., K. L. McDonald, J. R. McIntosh. 1993. Three-dimensional <br />                                                                          reconstruction analysis mitotic spindles yeast, Schizosaccha- <br /> faster depolymerization/destabilization interpolar MTs. The           romyces pombe. J. Cell Biol. 120:141&#226;&#8364;&#8220;151. <br /> abnormal growth astral MTs absence Ask1           14. Ding, R., R. R. West, D. M. Morphew, B. R. Oakley, J. R. McIntosh. <br />                                                                          1997. The spindle pole body Schizosaccharomyces pombe enters and <br /> be stabilization MTs inhibiting rate        leaves nuclear envelope cell cycle proceeds. Mol. Biol. Cell <br /> of depolymerization. We anticipate abnormally                 8:1461&#226;&#8364;&#8220;1479. <br /> elongated astral MTs interact cell cortex generate      15. Eggimann, P., J. Garbino, D. Pittet. 2003. Epidemiology Candida <br />                                                                          species infections critically ill non-immunosuppressed patients. Lancet <br /> a backward force SPBs. This prevent cell cycle stage-             Infect. Dis. 3:685&#226;&#8364;&#8220;702. <br /> specific spindle elongation alignment, resulting short        16. Enquist-Newman, M., et al. 2001. Dad1p, component Duo1p/ <br /> and misaligned spindles.                                                 Dam1p complex involved kinetochore function mitotic spindle integ- <br />                                                                          rity. Mol. Biol. Cell 12:2601&#226;&#8364;&#8220;2613. <br />    Since Dam1 complex C. albicans localized through-       17. Finley, K. R., J. Berman. 2005. Microtubules Candida albicans hyphae <br /> out cell cycle, asked Dam1 complex helps              drive nuclear dynamics connect cell cycle progression morphogenesis. <br />                                                                          Eukaryot. Cell 4:1697&#226;&#8364;&#8220;1711. <br /> in KT-MT interaction interphase. The Dam1 complex             18. Franck, A. D., et al. 2007. Tension applied Dam1 complex <br /> mutants absence Mad2 provided opportunity             promotes microtubule elongation providing direct mechanism length <br /> study role complex premitotic interphase cells.           control mitosis. Nat. Cell Biol. 9:832&#226;&#8364;&#8220;871. <br />                                                                      19. Gerami-Nejad, M., J. Berman, C. A. Gale. 2001. Cassettes PCR- <br /> The presence long metaphase-like spindles (3 5 &#226;?&#174;m)              mediated construction green, yellow, cyan fluorescent protein fusions <br /> these unbudded cells revealed premature spindle elongation            Candida albicans. Yeast 18:859&#226;&#8364;&#8220;864. <br />  VOL. 10, 2011                                                                       Dam1 COMPLEX IS ESSENTIAL IN CANDIDA ALBICANS                                    1305 <br />  <br /> 20. Goshima, G., J. M. Scholey. 2010. Control mitotic spindle length.                 gene deletion studies diploid human fungal pathogen Candida albi- <br />     Annu. Rev. Cell Dev. Biol. 26:21&#226;&#8364;&#8220;57.                                                     cans. Eukaryot. Cell 4:298&#226;&#8364;&#8220;309. <br /> 21. Gudlaugsson, O., et al. 2003. Attributable mortality nosocomial candi-          38.   Nogales, E., V. H. Ramey. 2009. Structure-function insights yeast <br />     demia, revisited. Clin. Infect. Dis. 37:1172&#226;&#8364;&#8220;1177.                                       Dam1 kinetochore complex. J. Cell Sci. 122:3831&#226;&#8364;&#8220;3836. <br /> 22. Gu&#194;&#168;ttinger, S., E. Laurell, U. Kutay. 2009. Orchestrating nuclear enve-        39.   Oliferenko, S., M. K. Balasubramanian. 2002. Astral microtubules mon- <br />     lope disassembly reassembly mitosis. Nat. Rev. Mol. Cell Biol.                itor metaphase spindle alignment fission yeast. Nat. Cell Biol. 4:816&#226;&#8364;&#8220;820. <br />     10:178&#226;&#8364;&#8220;191.                                                                        40.   Padmanabhan, S., J. Thakur, R. Siddharthan, K. Sanyal. 2008. Rapid <br /> 23. Heath, I. B. 1980. Variant mitoses lower eukaryotes: indicators                evolution Cse4p-rich centromeric DNA sequences closely related <br />     evolution mitosis. Int. Rev. Cytol. 64:1&#226;&#8364;&#8220;80.                                          pathogenic yeasts, Candida albicans Candida dubliniensis. Proc. Natl. <br /> 24. Hofmann, C., et al. 1998. Saccharomyces cerevisiae Duo1p Dam1p,                      Acad. Sci. U. S. A. 105:19797&#226;&#8364;&#8220;19802. <br />     novel proteins involved mitotic spindle function. J. Cell Biol. 143:1029&#226;&#8364;&#8220;       41.   Palmer, R. E., D. S. Sullivan, T. Huffaker, D. Koshland. 1992. Role of <br />     1040.                                                                                    astral microtubules actin spindle orientation migration the <br /> 25. Joglekar, A. P., et al. 2008. Molecular architecture kinetochore-                 budding yeast <br /> </body></html>