<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body><span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>Survival</span> Response Increased Ceramide Involves <br /> Metabolic Adaptation Novel Regulators of <br /> Glycolysis Lipolysis <br /> Niraj K. Nirala1,2., Motiur Rahman1,2., Stanley M. Walls3, Alka Singh1,2, Lihua Julie Zhu1,2,4, <br /> Takeshi Bamba5, Eiichiro Fukusaki5, Sargur M. Srideshikan6, Greg L. Harris3, Y. Tony Ip2, Rolf Bodmer7, <br /> Usha R. Acharya1,2* <br /> 1 Program Gene Function Expression, University Massachusetts Medical School, Worcester, Massachusetts, United States America, 2 Program Molecular <br /> Medicine, University Massachusetts Medical School, Worcester, Massachusetts, United States America, 3 Cell Molecular Biology Program, San Diego State <br /> University, San Diego, California, United States America, 4 Program Bioinformatics Integrative Biology, University Massachusetts Medical School, Worcester, <br /> Massachusetts, United States America, 5 Department Biotechnology, Graduate School Engineering, Osaka University, Osaka, Japan, 6 Laboratory Cell and <br /> Developmental Signaling, National Cancer Institute, Frederick, Maryland, United States America, 7 Development Aging Program, Sanford-Burnham Medical <br /> Research Institute, La Jolla, California, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The sphingolipid ceramide elicits stress responses, however, organisms survive despite increased ceramide how <br />      poorly understood. We demonstrate AKT/FOXO pathway regulates survival increased ceramide <br />      environment metabolic adaptation involving changes glycolysis lipolysis novel downstream targets. We <br />      ceramide kinase mutants accumulate ceramide leads reduction energy levels compromised <br />      oxidative phosphorylation. Mutants increased activation Akt consequent decrease FOXO levels. These <br />      changes lead enhanced glycolysis upregulating activity phosphoglyceromutase, enolase, pyruvate kinase, and <br />      lactate dehydrogenase provide energy. A second major consequence AKT/FOXO reprogramming mutants the <br />      increased mobilization lipid gut novel lipase targets, CG8093 CG6277 energy contribution. <br />      Ubiquitous reduction targets knockdown experiments results semi total lethality mutants, <br />      demonstrating importance activating them. The efficiency adaptive mechanisms decreases age and <br />      leads reduction adult life span mutants. In particular, mutants develop cardiac dysfunction age, likely <br />      reflecting high energy requirement well-functioning heart. The lipases regulate physiological triacylglycerol <br />      homeostasis important energy metabolism midgut specific reduction wild type flies results in <br />      increased sensitivity starvation accumulation triglycerides leading cardiac defects. The central findings of <br />      increased AKT activation, decreased FOXO level activation phosphoglyceromutase pyruvate kinase also <br />      observed mice heterozygous ceramide transfer protein suggesting conserved role pathway mammals. <br />      These data reveal novel glycolytic non-autonomous lipolytic pathways response increased ceramide for <br />      sustenance high energy demanding organ functions like heart. <br />  <br />  <br />   Citation: Nirala NK, Rahman M, Walls SM, Singh A, Zhu LJ, et al. (2013) Survival Response Increased Ceramide Involves Metabolic Adaptation Novel <br />   Regulators Glycolysis Lipolysis. PLoS Genet 9(6): e1003556. doi:10.1371/journal.pgen.1003556 <br />   Editor: Kaveh Ashrafi, University California San Francisco, United States America <br />   Received November 7, 2012; Accepted April 23, 2013; Published June 20, 2013 <br />   Copyright: &#195;&#376; 2013 Nirala et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research supported NIH grant (RO1EY016469) URA. YTI supported NIH grant (DK83450) member UMass DERC <br />   (DK32520). SMW Rees-Stealy Research Foundation Fellow, ARCS Foundation supported Scholar American Heart Association Predoctoral Fellow. The <br />   funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: usha.acharya@umassmed.edu <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                           important contributors obesity insulin resistance. Dyslipidemia <br />                                                                                        obesity characterized elevated free fatty acids, triglycerides <br />     Energy balance regulated complex homeostatic control lipid                (TAG), VLDL decreased HDL cholesterol. Increased levels free <br /> and glucose metabolism involving signaling pathways multiple                        fatty acids lipids diacylglycerol ceramide mediate <br /> tissues organs. Aberrant glucose lipid metabolism arising                 insulin resistance redistribution fat metabolites tissues suited <br /> defects pathways accompanies number diseases including                   lipid storage [1,2]. <br /> cardiovascular disorders, type II diabetes metabolic syndrome                         The sphingolipid, ceramide, integral component cell <br /> associated obesity insulin resistance. Understanding regula-                  membranes bioactive lipid [3]. Ceramide elicits many <br /> tory mechanisms governing glucose lipid metabolism funda-                       cell-stress responses including apoptosis, senescence, inflammation, <br /> mental identifying potential therapeutic targets treatment                  mitochondrial dysfunction recent studies affect <br /> these metabolic diseases. Altered lipid metabolism                       cellular metabolism. It impair insulin stimulated glucose uptake, <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                               1                               June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                              AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />   Author Summary                                                                We set globally discover genes metabolites change <br />                                                                              ceramide kinase mutant flies (dcerk1) gene expression and <br />   Ceramide belongs important class lipids called                    metabolic profiling. Integrating information analyses with <br />   sphingolipids. An increase ceramide levels elicits              genetic biochemical experiments, dcerk1 mutants <br />   stress responses. We used Drosophila mutated                       survive increased activation AKT decreased FOXO <br />   ceramide kinase, enzyme involved metabolizing                        level. In dcerk1 mutants, increase ceramide leads decreased <br />   ceramide understand organism adapts stress                    ATP level compromised mitochondrial oxidative phosphory- <br />   imposed ceramide. We mutants,                          lation. Metabolic reprogramming ceramide involves AKT induced <br />   increased ceramide decreases energy production                         glycolytic utilization glucose activation phosphoglycerate <br />   mitochondrial oxidative phosphorylation. The mutants                       mutase utilization triglycerides activation intestine <br />   compensate decreased energy upregulating glycol- <br />                                                                              specific lipases, CG8093 CG6277 dcerk1. These novel <br />   ysis, central pathway intermediary metabolism. <br />                                                                              downstream targets AKT/FOXO glycolysis lipolysis. With <br />   Another important mechanism activated these <br />   ceramide kinase mutants lipolysis, involved                 age, compensatory mechanisms fail leading abnormal cardiac <br />   breakdown triglycerides energy contribution. The                    function decreased life span mutants. Gut specific reduction <br />   mutants increased hydrolysis triglycerides                  lipases wild type mutant flies results TAG <br />   gut, probably provide sufficient energy optimal                     accumulation, increased sensitivity starvation cardiac dysfunc- <br />   performance heart, high energy demanding organ.                   tion. Using mice heterozygous ceramide transfer protein (CERT) as <br />   The upregulation pathways mutant result                    second model ceramide increase, substantiate AKT activation, <br />   increased activation AKT decreased FOXO. The                   decrease FOXO level activation glycolytic genes, phospho- <br />   modulation glycolysis lipolysis AKT/FOXO                      glycerate mutase pyruvate kinase mammalian [28]. We <br />   mediated novel targets identified study. Failure                discovered novel connection PI3K-AKT-FOXO <br />   adaptive mechanisms aging flies lead                        pathway survival metabolic adaptation ceramide <br />   triglyceride accumulation, cardiac abnormalities, ulti-                environment participates energy response involving phospho- <br />   mately decrease lifespan mutant animals.                   glycerate mutase gut specific lipases, CG8093 CG6277 as <br />   The mechanisms identified clinical                         physiological targets. <br />   implications diseases associated ceramide as <br />   diabetes obesity. <br />                                                                              Results <br /> it serves intermediate linking glucocorticoids saturated           dcerk1 mutants exhibit metabolic reprogramming <br /> fatty acids insulin resistance implicated                Ceramide kinase mutants increase steady state <br /> pathogenesis lipotoxic cardiomyopathy [4&#226;&#8364;&#8220;6]. Strategies              concentration ceramides significant decrease levels <br /> pharmacologically genetically decrease ceramide beneficial           ceramide 1-phosphate fly extracts (Figure S1). The increase <br /> effects reversing insulin resistance, preventing apoptosis             C24:1 ceramide dramatic (about 280%) other <br /> pancreatic b-cells cardiomyocytes [4,7].                                 ceramides 30&#226;&#8364;&#8220;50% increase dcerk1 mutants over <br />    Despite involvement ceramide stress responses,             control flies. Thus dcerk1 mutants survive increased <br /> organisms develop survive increased ceramide levels              ceramide environment. An organism&#226;&#8364;&#8482;s attempt adapt stress or <br /> certain circumstances. For example, Drosophila mutants ceramide           altered environment involves changes gene expression and <br /> kinase brainwashing (mammalian homolog alkaline                       metabolite levels aid reestablishing homeostasis. Therefore, <br /> ceramidase) viable mutants ceramidase lethal                reasoned elucidating transcriptional metabolite responses <br /> despite similar increase ceramide levels [8&#226;&#8364;&#8220;11]. Mice                 change mutants relative control help understand <br /> Arabidopsis deficient ceramide kinase mice deficient               possible adaptive mechanisms operate dcerk1. We carried a <br /> sphingomyelin synthase 1 2 viable despite accumulation            transcriptome-wide analysis w1118 dcerk1 flies. 310 transcripts <br /> ceramide [12&#226;&#8364;&#8220;16]. A likely explanation organisms               changed significantly dcerk1 compared control, 152 <br /> implement adaptive responses allow survive                  genes upregulated (. = 2.0 fold) 158 genes being <br /> maintain equilibrium face stress increased ceramide.        downregulated (, = 0.5 fold, Table S1). <span id='am-8' about='protege:TO' typeof='owl:Thing'>To</span> identify pathways and <br /> These adaptive mechanisms critical importance under-               predicted gene functions corresponding altered transcripts, we <br /> standing metabolic homeostasis, received limited attention          employed DAVID gene ontology (GO) annotation functional <br /> especially multi-cellular organisms. In study, address       classification tools [29]. The GO class increased genes included <br /> issue using Drosophila loss function <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> ceramide kinase          serine type endopeptidases hydrolases (Figure 1A). Most the <br /> (CERK). Our earlier studies established CERK important                 serine type endopeptidases mainly expressed larval/adult <br /> regulator phospholipase C (PLC) signaling photoreceptor               mid gut (based FlyAtlas expression patterns) suggesting are <br /> homeostasis [10]. A mutation dCERK led proteolysis PLC,             likely involved mobilization dietary protein. Among the <br /> consequent loss PLC activity failure light signal                  increased hydrolases glycosyl hydrolases involved in <br /> transduction. These defects increased ceramide levels            metabolism starch, sucrose galactose also <br /> and ceramide 1-phosphate (C1P) levels C1P did               expressed mainly adult gut. Thus, significant <br /> change significantly photoreceptors dcerk1 mutants.                    upregulation midgut genes involved breakdown dietary <br />    In recent years, Drosophila emerged useful organism           proteins sugars dcerk1. Other classes upregulated genes dcerk1 <br /> study metabolism [17]. The fly organ systems fat             transporters, oxidoreductases genes constitute the <br /> body (adipose tissue), oenocytes (hepatocyte-like cells), gut                peritrophic envelope, lining composed chitin glycoproteins, <br /> (gastrointestinal tract) malphigian tubules (kidneys)               separating food midgut epithelium. The GO category <br /> parallel mammals. Several studies highlight conserved           downregulated genes antimicrobial peptides (AMPs) involved <br /> mechanisms carbohydrate, lipid energy homeostasis flies            innate immune response. Important metabolic genes that <br /> and genes discovered Drosophila provided information                 change dcerk1 flies phosphoenolpyruvate carboxykinase (PEPCK, <br /> relevant human physiology including heart [18&#226;&#8364;&#8220;27].            downregulated) involved gluconeogenesis, fatty acid synthesis <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     2                             June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 1. Metabolic reprogramming dcerk1 mutants. A. Functional classification genes expression altered dcerk1 mutants <br /> compared control w1118 flies. RNA extracted control mutant flies hybridized Affymetrix Drosophila 2.0 microarrays and <br /> experiments carried triplicate. Gene expression analysis revealed 152 genes increased (2-fold more) 158 genes were <br /> decreased (0.5 fold less) dcerk1 compared w1118. DAVID gene functional annotation tool functional classification tool used to <br /> annotate classify increased decreased genes functionally related gene groups. The GO terms obtained increased and <br /> decreased genes listed number genes altered category statistical significance based modified Fisher Exact P- <br /> Value. Serine type endopeptidases hydrolases upregulated immune response genes downregulated mutant. B. Metabolomic <br /> profiling reveals changes glycolysis TCA cycle metabolites dcerk1 compared w1118. Control mutant fly extracts subjected in <br /> triplicate GC/MS, LC/MS LC/MS/MS platforms compare relative levels metabolites. The fold change, dcerk1/w1118 shown each <br /> metabolite. A p value ,0.05 q value ,0.1 (false discovery rate) considered highly statistically significant (marked red *). Among <br /> glycolysis intermediates, pyruvate lactate meet stringent criteria; pyruvate level decreased lactate level increased mutant. <br /> Among TCA cycle metabolites, oxaloacetate a-ketoglutarate decreased citrate malate increased. Fumarate succinate are <br /> also increased meet stringent statistical criteria approaching significance (p 0.05 20.1 q 0.1 less, marked blue *). Most <br /> amino acids (other cysteine methionine) sugars significant differences. Among fatty acids, Dodecanedioate, long chain <br /> fatty acid decreased significantly mutant. This degradation acetyl CoA succinate, important for <br /> gluconeogenesis starvation. <br /> doi:10.1371/journal.pgen.1003556.g001 <br />  <br /> (downregulated), carnitine palmitoyltransferase (b-oxidation, upregu-          identified classified different functional pathways <br /> lated) pyruvate kinase (upregulated) involved glycolysis.               (Table S3). The fold changes (dcerk1/w1118) various amino <br />    Since gene expression changes known targets FOXO                 acids, fatty acids, sugars intermediates glycolysis, TCA <br /> as PEPCK AMPs observed, analyzed                          cycle, shown Figure 1B. Two important metabolic pathways <br /> differentially expressed genes microarray dataset              different w1118 mutants glycolysis and <br /> potential targets FOXO. Recent ChIP-chip analysis revealed                  TCA cycle. dcerk1 exhibited increase lactate decrease in <br /> about 700 direct FOXO targets adult Drosophila [30]. Using             pyruvate, glycolytic intermediates. While oxaloacetate a- <br /> information coupled ChIP peak Annopackage identified                   ketoglutarate decreased, TCA cycle intermediates, <br /> that 40 genes microarray data contained FOXO binding                  citrate, malate, fumarate succinate increased. The <br /> sites potential targets FOXO (Table S2) [31].                  altered steady state concentrations metabolites indicate <br />                                                                                change flux pathways. Glycolysis, TCA cycle and <br />                                                                                mitochondrial oxidative phophorylation share control energy <br /> Metabolomic analysis dcerk1 mutants reveals changes                         metabolism interact match cellular ATP demand with <br /> in glycolysis TCA cycle                                                    ATP production. <br />    In order understand changes gene expression                          To assess mitochondrial oxidative phosphorylation, we <br /> influence metabolism, compared metabolic profiles                    measured enzyme activities complexes the <br /> w1118 dcerk1 mass spectrometry. About 175 metabolites                   electron transport chain mitochondrial-enriched fractions <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       3                             June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                            AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> control mutant flies. As seen Figure 2A,                    FOXO level ubiquitination proteosomal degradation <br /> activities complexes II, III, IV V decreased I          FOXO [42]. To address possibility Drosophila, reduced <br /> increased mutant compared control. To test                     ubiquitin mediated proteosomal degradation dcerk1 using a <br /> compromised oxidative phosphorylation led alteration                  dominant temperature sensitive (DTS) mutation, DTS5 that <br /> energy level, measured ATP level mitochondria isolated                affects b6 proteosomal subunit [43]. Since ubiquitous <br /> from w1118 mutant flies. Indeed, dcerk1 showed 40%                    expression DTS5 resulted lethality, overexpressed <br /> decrease ATP level relative w1118 (Figure 2B). Since                  DTS5 photoreceptors using GMR GAL4 driver and <br /> metabolic profiling emphasized changes glycolysis,              measured FOXO transcript level heads. If FOXO targeted <br /> examined changes gene expression enzymes              degradation dcerk1, expressing DTS5 result in <br /> the glycolytic pathway addition pyruvate kinase,            restoration FOXO level case (Figure 3D). <br /> shown upregulated microarray analysis. Quantitative                Thus increased activation AKT leads downregulation of <br /> PCR analysis different glycolytic genes showed 15-fold             FOXO transcription factor dcerk1 mutant flies. <br /> increase phosphoglycerate mutase (Pglym), 3&#226;&#8364;&#8220;4 fold increase                 To evaluate importance increased AKT activity dcerk1 <br /> in enolase (Eno) pyruvate kinase (Pyk) expression,                 homozygous mutants, generated akt4226, dcerk1 double homo- <br /> enzymes catalyze later reactions glycolytic pathway             zygotes. While dcerk1 akt4226 homozygous flies viable, <br /> (Figure 2C). Interestingly, traditional regulators glycolysis            double mutants early larval lethal. Hence, compromised AKT <br /> such hexokinase phosphofructokinase un-                   function incompatible survival dcerk1 flies. <br /> changed decreased. To test increase transcript levels <br /> translated increased activity, measured activity               Metabolic reprogramming dcerk1 involves increased <br /> various enzymes. Pglym, Pyk, Eno Lactate dehydrogenase                  glycolytic flux activation phosphoglycerate <br /> (Ldh) showed increase enzyme activity (Figure 2D). Meta-                 mutase downstream glycolytic enzymes <br /> bolically changes transcript enzyme activity result                To test AKT required increased glycolytic flux <br /> in 1.7 fold increase lactate dcerk1 (Figure 1B). Thus             mutants, generated dcerk1 flies reduced AKT function <br /> significant portion glucose metabolized glycolytic                wild type copy AKT replaced the <br /> pathway converted lactate action lactate                   hypomorphic allele AKT. dcerk1 flies heterozygous akt 4226 <br /> dehydrogenase. The results suggest increased                 (referred dcerk1.akt/dcerk1) survive adulthood have <br /> reliance glycolysis dcerk1 compensates decreased            used subsequent experiments evaluating effects of <br /> ability produce ATP oxidative phosphorylation.                   reducing AKT function dcerk1. The data studies with <br />                                                                             Pglym shown showed significant increase both <br /> Increased activation AKT required survival                     transcript enzyme activity mutant flies (Figures 3E, 3F). <br /> dcerk1 mutant flies                                                         dcerk1.akt/dcerk1 flies showed reduction Pglym transcript and <br />    Since dcerk1 mutants specific changes glycolysis             enzymatic activity. akt4226/+ flies tested above <br /> differentially expressed genes transcriptional profiling data        experiments did differ significantly w1118 control <br /> contained FOXO binding sites, decided test AKT,                  suggesting heterozygous state AKT not <br /> central modulator metabolism upstream regulator                   responsible changes (Figures 3E, 3F). Densitometric <br /> FOXO altered dcerk1. AKT master regulator survival,          scanning Western blots reflects 50% decrease total AKT <br /> proliferation cellular metabolism [32,33]. AKT activation               30% decrease pAKT levels akt4226/+ (Figures S2A, <br /> requires TOR dependent phosphorylation Ser-473 (Ser-505               S2B). As mentioned earlier, AKT alters metabolic balance through <br /> Drosophila) C-terminus [34,35]. We measured AKT                      downregulation FOXO. We tested involvement regulating <br /> activation dcerk1 flies measuring level Ser-505                Pglym, overexpressing FOXOTM, constitutively active form <br /> phosphorylated AKT protein Western analysis. Phosphorylated              protein localizes nucleus [44]. We expressed <br /> AKT (seen major minor band) increased              FOXOTM using muscle driver, myosin heavy chain (MHC)- <br /> total AKT level different w1118 (Figure 3A). The blot          GAL4 analyzed body Pglym transcript level [45]. <br /> also shows reduced levels total phospho AKT akt4226,            Indeed, FOXOTM overexpression muscles resulted in <br /> hypomorphic allele AKT used study [36]. Densitometric            significant decrease Pglym transcript (Figure 3G), suggesting <br /> scanning Western blots shows 50% increase pAKT                  Pglym regulation FOXO dependent. <br /> level dcerk1 mutant (Figure 3B). To test activity              The measurement steady state levels glucose (mammals) <br /> upstream activator PI3K increased, used vivo reporter,          trehalose (Drosophila) indicator glycolytic utilization of <br /> which contains PH domain general receptor phospho-               sugars energy metabolism. Therefore, tested AKT is <br /> inositide fused GFP (tGPH) [37]. tGPH increased cell           required maintaining normal trehalose levels dcerk1 mutants. <br /> membrane PI3K activation increases PIP3 level. We                      Trehalose levels fly extract dcerk1 90% control <br /> observed increased membrane association tGPH dcerk1                flies, reduction akt dosage dcerk1 mutants resulted in <br /> compared control adult mid gut (Figure 3C, panel)             increase trehalose (Figure 3H). Circulating trehalose levels <br /> and larval fat body (Figure 3C, panel). These results                followed similar trend body trehalose <br /> suggest increased PI3K activation leads increased AKT                    measurements flies (data shown). We globally <br /> activation dcerk1. A likely reason increased activation       reduced Pglym expression ubiquitous RNAi mediated knock- <br /> PI3K increased availability PIP2 dcerk1              using actin-Gal4 driver (Figure S3). Notably, this <br /> mutant [10].                                                                reduction resulted semi lethality 40% expected <br />    AKT mediates alterations metabolism               number flies reaching adulthood dcerk1. Trehalose levels <br /> regulation FOXO Drosophila single homolog           increased 20% w1118 50% dcerk1 Pglym <br /> the FOXO transcription factor family [38&#226;&#8364;&#8220;41]. To test status            levels reduced (Figure 3I). Hence, results suggest <br /> of FOXO, measured FOXO transcript level mutant flies                  increased glycolytic flux Pglym important for <br /> and significantly decreased (Figure 3D). AKT              survival maintaining near physiological levels of <br /> activation mammalian cells shown lead low                 trehalose dcerk1 mutants. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    4                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 2. dcerk1 mutants decreased mitochondrial oxidative phosphorylation, lower ATP level, increased glycolysis <br /> compared control flies. (A). The activity of electron transport chain complexes measured spectrophotometrically using specific <br /> substrates inhibitors mitochondria isolated control mutant flies. Specific activity mg mitochondrial protein calculated and <br /> then normalized w1118. Mitochondria prepared 1000 flies 3 replicates carried out. Error bars represent standard deviation. (B). <br /> ATP level measured w1118 mutant fly mitochondria bioluminescence assay using luciferase catalyzed oxidation luceferin ATP <br /> dependent. The ATP calculated mg mitochondrial protein normalized w1118. The relative level ATP mutant <br /> mitochondria 60% w1118. n = 3, error bars represent standard deviation. (C). QPCR analysis transcripts encoding glycolytic enzymes 16&#226;&#8364;&#8220; <br /> 18 fold increase Pglym, 3&#226;&#8364;&#8220;4 fold increase Eno 4&#226;&#8364;&#8220;5 fold increase Pyk levels dcerk1 relative w1118. n = 3, error bars represent standard <br /> deviation. (D). Measurement specific activities glycolytic enzymes increase Pglym, Eno, Pyk Ldh activities dcerk1 compared to <br /> w1118 hexokinase significantly different. Specific activity enzyme determined mg protein normalized to <br /> w1118. n = 3, error bars represent standard deviation. <br /> doi:10.1371/journal.pgen.1003556.g002 <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       5                             June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                               AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 3. Increased glycolytic flux dcerk1 compared w1118 activation Pglym downstream glycolytic enzymes <br /> through AKT. (A). Total AKT phospho AKT monitored control mutant flies Western blots. One band observed total AKT it <br /> is significantly different w1118 mutant. Two major bands minor band observed phospho AKT increased <br /> in dcerk1 compared w1118. The blots total AKT phospho AKT reduced significantly hypomorphic allele akt4226 <br /> used study. (B). Quantification total AKT phospho AKT levels. Densitometric scanning total AKT phospho AKT bands from <br /> Western blots carried out. The phospho AKT graph represents average obtained combining values bands observed on <br /> blots. The values normalized w1118 taken 100%. (C). tGPH staining monitored control (upper left panel) dcerk1 (upper right panel) <br /> adult midgut. The lower left panel shows tGPH staining control lower right panel shows mutant larval fat body. There increased <br /> tGPH staining cell membranes (compare arrows) indicating increased PIP3 levels increased PI3K activity. In addition plasma <br /> membrane, nuclear tGPH increased dcerk1 gut reported nuclear tGPH visible cell types PIP3 levels <br /> are increased cells. Midguts 15&#226;&#8364;&#8220;20 flies visualized experiment independent experiments carried out. For <br /> visualizing tGPH fat body, 15&#226;&#8364;&#8220;20 early instar larvae used experiment independent experiments carried out. Scale <br /> bar represents 50 mm. (D). FOXO transcript level measured QPCR shows 60% reduction in dcerk1 flies compared w1118. FOXO <br /> transcript level increases mutant heads proteosomal degradation inhibited overexpressing DTS5 subunit dcerk1 photoreceptors <br /> compared w1118 DTS5 driver dcerk1 head samples. (E, F). Transcript level (E) enzyme activity (F) Pglym significantly <br /> decreased AKT activity genetically reduced dcerk1 suggesting AKT mediates Pglym activation dcerk1. (G). Overexpression constitutively <br /> active FOXO muscles (MHC-GAL4) significantly decreases body Pglym transcript level. n = 3, error bars represent standard deviation. (H). <br /> Reduction AKT dcerk1 increases total body trehalose level. Trehalose level calculated mg protein normalized w1118 level. n = 3, <br /> error bars represent standard deviation. (I). While actin GAL4 driver UASPglymRNAi transgene individually effect trehalose level, <br /> combining increases animal trehalose 20% w1118 50% dcerk1. n = 3, error bars represent standard deviation. <br /> doi:10.1371/journal.pgen.1003556.g003 <br />  <br />  <br /> Metabolic adaptation dcerk1 involves increased                             lipoprotein particles called lipophorins. So, decided test if <br /> mobilization TAG novel lipases, CG8093                         energy rich TAG stores altered dcerk1 so, does the <br /> CG6277                                                                        AKT/FOXO pathway play role process. <br />    Our experiments far suggest dcerk1 flies attempt                 We measured animal TAG levels ad libitum fed and <br /> adapt reduced energy availability. Triacylglycerol (TAG)            20 h starved w1118 dcerk1 flies. The mutant flies showed 20% <br /> of major energy reserves, stored cytoplasmic lipid            TAG w1118 fed state 50% TAG the <br /> droplets primarily adipose tissue [46]. In Drosophila,              starved state (Figure 4A). The decrease TAG starved state <br /> addition fat body cells, tissues, notably, gut,             accompanied increased free fatty acid glycerol whole <br /> oocytes, larval imaginal discs oenocytes accumulate TAG              fly extracts (Figures 4B, 4C). Since TAG levels significantly <br /> as intracellular lipid droplets [47]. During times energy need             altered starved state, assessed ability dcerk1 to <br /> such starvation, stored TAG hydrolyzed depots             survive starvation stress. As shown Figure 4D, susceptibility <br /> activation lipolytic mechanisms [48]. Fatty acids generated            mutant flies starvation similar w1118. These results <br /> lipolysis lipases used energy production b-             suggest dcerk1 mutant likely rely increased TAG hydrolysis <br /> oxidation. Stored lipids transported hemolymph             survive starvation stress. To test AKT mediated compensation <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      6                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                 AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 4. CG8093 CG6277 novel lipases gut activated AKT/FOXO dcerk1. (A). Relative TAG level ad libitum fed <br /> dcerk1 80% w1118 increases 120% AKT reduced. Relative TAG level 20 hr starved dcerk1 50% w1118 significantly <br /> increases AKT reduced. n = 3, error bars represent standard deviation. (B). Free fatty acids measured lysates prepared starved <br /> w1118 dcerk1 flies. Mutants increased free fatty acids compared w1118. (C). Glycerol measured lysates prepared starved w1118 <br /> and dcerk1 flies. Mutant shows increased glycerol compared w1118. (D). w1118, dcerk1, akt/+ dcerk1.akt/dcerk1 flies subjected starvation and <br /> the number surviving flies recorded 6 hr intervals. 200 flies divided 10 groups genotype used experiment. Three <br /> independent experiments performed average percentage survival relative w1118 shown time interval. Reduction AKT <br /> increases starvation sensitivity dcerk1 flies. (E). QPCR analysis shows transcript levels lipases CG8093 CG6277 (identified by <br /> microarray analysis) increased dcerk1 relative w1118 fed 20 h starved state. Both transcript levels decreased upon <br /> reduction AKT dcerk1. n = 3, error bars represent standard deviation. (F). Overexpression constitutively active FOXO midgut decreases <br /> whole animal transcript levels CG8093 CG6277 fat body specific expression does significantly affect dcerk1. FOXO <br /> overexpression driver quantified QPCR using FOXO primers normalized. n = 3, error bars represent standard deviation. (G). Midgut <br /> specific overexpression constitutively active FOXO increases animal TAG level dcerk1. The TAG level dcerk1 expressing esg-GAL4 driver is <br /> taken 100%. (H). Constitutive FOXO represses CG8093 luciferase reporter assays. A 2.4 kb fragment (genomic region 1773 bp to <br /> 4177 bp transcription start site) containing FOXO core motifs upstream CG8093 luciferase reporter vector 1.3 kb fragment (region <br /> between 2842 bp 4177 bp) deficient FOXO motifs cotransfected constitutive dFOXO A3. FOXO represses transcription the <br /> 2.4 kb fragment repression binding sites deleted. (I). Dissected adult midguts stained Oil Red O significantly <br /> reduced lipid staining starved dcerk1 compared w1118. dcerk1 mutants reduced AKT comparatively Oil Red O staining the <br /> midgut. 15&#226;&#8364;&#8220;20 guts examined genotype experiment, carried times. Scale bar represents 50 mm. Magnified view <br /> of dissected adult midguts w1118 dcerk1.akt/dcerk1 Oil Red O stained puncta significantly reduced dcerk1. Scale <br /> bar represents 50 mm. <br /> doi:10.1371/journal.pgen.1003556.g004 <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        7                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br /> allowed dcerk1 survive starvation, subjected dcerk1.akt/dcerk1            interaction transcription factors indirectly <br /> flies starvation stress. These flies died faster rate dcerk1       mediated FOXO dependent induction transcriptional <br /> showing reduction AKT function dcerk1 led starvation                   repressor protein. A possible candidate sugarbabe, which <br /> sensitivity (Figure 4D). TAG level flies significantly             encodes zinc finger protein. An earlier study analyzed sugar <br /> higher dcerk1, fed starved states (Figure 4A).          dependent gene expression profiles Drosophila larvae identified <br /> The starvation sensitivity flies despite increased TAG level           CG6277 negatively regulated sugarbabe [49]. Our <br /> suggested unable properly hydrolyze TAG                 analysis sugarbabe transcript dcerk1 mutants suggests level <br /> stores lipolysis AKT activity reduced. Thus AKT                     low, consistent observed upregulation <br /> dependent lipases provide critical function dcerk1 flies.              CG6277 transcript level. Also, analysis FOXO binding sites <br />     In order identify lipases activated dcerk1,         identifies potential regions 10 kb upstream of <br /> tested transcript levels known fat body lipases Drosophila,               sugarbabe (data shown). Thus possible mode regulation <br /> which include Brummer (ATGL homolog), CG11055 (putative                         FOXO activates sugarbabe, turn negatively <br /> homolog hormone sensitive lipase) CG8552 (homolog                     regulates CG6277. Thus downregulation FOXO dcerk1 could <br /> major triglyceride lipase related insect) [19,48]. The transcript          lead activation CG6277. These results suggest increase in <br /> level Brummer decreased dcerk1 (0.5 fold compared                  CG8093 CG6277 AKT/FOXO lead to <br /> control) lipases did changes               increased lipolysis decreased TAG level dcerk1. <br /> (data shown). Thus traditional lipases upregulated                To examine increased hydrolysis fat gut did <br /> increased TAG utilization dcerk1. We pursued potential                  occur result increased lipase activity, guts isolated from <br /> candidates microarray studies. Our microarray analysis                     w1118, dcerk1 dcerk1.akt/dcerk1 stained neutral lipid <br /> identified 60 lipases identified             dye Oil Red O starved conditions (Figure 4I). Oil Red O <br /> Drosophila, expression levels genes, CG8093 CG6277,                  staining evident w1118 gut dcerk1 showed little staining. <br /> that encode predicted TAG lipases increased dcerk1.                     Midgut isolated dcerk1.akt/dcerk1 stained Oil <br /> CG8093 acid lipase a/b hydrolase family related                 Red O (Figure 4I). Oil Red O puncta representing lipid droplets <br /> human lysosomal acid lipase CG6277 neutral lipase                    clearly visible w1118 dcerk1 reduced AKT were <br /> showing homology lipoprotein lipase pancreatic                        substantially decreased dcerk1 midgut (Figure 4I, panel). <br /> triacylglycerol lipase family. Both CG8093 CG6277 contain                      To ensure changes gene expression increased <br /> the catalytic triad Ser, His, Asp residues important lipase              utilization lipid gut to compromised gut <br /> activity. Earlier studies involving microarray analysis Drosophila           integrity defective epithelial renewal, tested dcerk1 gut were <br /> demonstrated lipases repressed high sugar              morphologically different w1118 gut. We observed gut <br /> situ hybridization studies showed expressed                    <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>, width ability ingest food different between <br /> midgut [49]. Our QPCR results showed genes                 control mutant flies (data shown). Epithelial integrity and <br /> highly expressed midgut (Figure S4). To test lipases            epithelial renewal assessed described Figure S5 and <br /> could facilitate TAG utilization dcerk1, examined             data suggest gut integrity epithelial renewal gut <br /> transcript levels fed starved states (Figure 4E). Both genes             largely uncompromised dcerk1. <br /> were upregulated dcerk1. This AKT mediated adaptive <br /> response transcript levels downregulated when <br />                                                                                 Midgut specific knock CG8093 CG6277 <br /> AKT reduced dcerk1. To evaluate involvement FOXO, <br /> we overexpressed FOXOTM using gut (esg-GAL4) fat body                       results TAG accumulation, starvation sensitivity and <br /> (C564-GAL4) specific GAL4 drivers [50,51]. Gut specific overex-                 cardiac defects <br /> pression FOXOTM decreased animal lipase transcripts                       Our data suggest increased expression CG8093 and <br /> while fat body GAL4 did alter expression significantly dcerk1            CG6277 leads increased lipolysis concomitant decrease in <br /> (Figure 4F). Overexpression FOXOTM gut resulted                  TAG level dcerk1. To ascertain observation, asked if <br /> in higher TAG level dcerk1 compared control (Figure 4G).                  impairing function lipases dcerk1 lead to <br />     The expression lipases regulated directly              accumulation TAG mutant. We attempted reduce <br /> indirectly FOXO. Our earlier analysis identified FOXO                        levels ubiquitous expression UAS-CG8093 RNAi and <br /> binding sites upstream CG8093 (Table S2)                 UAS-CG6277 RNAi constructs dcerk1. However condi- <br /> potential target FOXO. To test negative regulation                     tions resulted lethality wild type dcerk1 flies. Since <br /> CG8093 FOXO mediated sites, performed                    lipases expressed predominantly gut, next <br /> luciferase reporter assays. A 2.4 kb fragment containing                    drove RNAi constructs using midgut specific driver esg- <br /> FOXO motifs regulatory region CG8093                                GAL4 able obtain adult flies. RNAi resulted 80% <br /> (pGL4CG8093 2.4 kb) fragment motifs                     reduction transcript levels lipases control dcerk1 <br /> deleted (pGL4CG8093 1.3kbdel) cloned reporter vector                  (Figure S6). Midgut specific knockdown of lipases <br /> containing luciferase gene (pGL4). Drosophila S2 cells                 resulted significant increase animal TAG levels in <br /> cotransfected constitutively active FOXO                dcerk1 (Figures 5A, 5B). Reduction lipases resulted in <br /> these constructs luciferase activity measured 30 h                increased TAG control flies (Figures 5A, 5B). Gut specific <br /> [40]. Constitutively active FOXO suppresses luciferase activity             reduction lipases resulted increased sensitivity to <br /> of pGL4CG8093 2.4 kb considerably relieved                       starvation wild type mutant background compared to <br /> pGL4CG8093 1.3kbdel, deficient FOXO binding                     GAL4 driver RNAi control flies (Figures 5C, 5D). Staining of <br /> sites (Figure 4H). This suggests FOXO act sites              guts isolated knockdown flies Oil Red O showed <br /> regulate CG8093 transcription. Our search FOXO binding                      increase number Oil Red O puncta indicative defective <br /> sites limited 10 kb upstream microarray target genes              mobilization lipid stores (Figure S7). These results collectively <br /> and CG6277 identified analysis. The promoter                    suggest lipases important utilization TAG and <br /> region CG6277 does contain obvious FOXO binding sites                    maintenance TAG homeostasis mutant also <br /> suggesting transcriptional regulation takes place            wild type flies. Additionally, life span wild type dcerk1 <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        8                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                               AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 5. Knockdown CG8093 CG6277 midgut increases body TAG, starvation sensitivity w1118 dcerk1 <br /> flies knockdown CG6277 leads cardiac defects. (A). Midgut specific reduction CG8093 increases body TAG level to <br /> 325&#226;&#8364;&#8220;350% control w1118 flies 275% dcerk1 mutants. The TAG level driver taken 100%. n = 3, error bars represent standard <br /> deviation. (B). Midgut specific reduction CG6277 increases animal TAG 150% control w1118 flies 275%&#226;&#8364;&#8220;300% dcerk1 <br /> mutants. The TAG level driver taken 100%. n = 3, error bars represent standard deviation. (C). Reduction CG8093 lipase increases the <br /> starvation sensitivity wild type dcerk1 flies. CG8093 RNAi flies, esgGAL4 driver flies, 8093 RNAi combined esgGAL4 wild type dcerk1 are <br /> subjected starvation number surviving flies recorded 6 hr intervals. 200 flies divided 10 groups genotype used in <br /> one experiment. Three independent experiments performed average percentage survival shown time interval. (D). Reduction <br /> in CG6277 lipase increases starvation sensitivity wild type dcerk1 flies. CG6277 RNAi flies, esgGAL4 driver flies, 6277 RNAi combined to <br /> esgGAL4 wild type dcerk1 subjected starvation number surviving flies recorded 6 hr intervals. (E). Quantification of <br /> cardiac function parameters RNAi knockdown CG6277 wild type dcerk1. Knockdown results decreased heart rate wild type flies a <br /> similar trend dcerk1, increased heart period backgrounds, longer diastolic interval control mutant significant change in <br /> systolic interval. n = 11&#226;&#8364;&#8220;13 group. Bar represents standard error mean statistical significance determined using Student&#226;&#8364;&#8482;s t test; <br /> * denotes P, = 0.05-0.01, ** P, = 0.01-0.001 *** denotes P, = 0.001-0.0001. <br /> doi:10.1371/journal.pgen.1003556.g005 <br />  <br />  <br /> flies significantly reduced gut specific knockdown                  energy substrates [52]. Cardiac dysfunction linked altered <br /> CG6277 (data shown).                                                       energy metabolism instances [53]. To test two <br />                                                                                lipases played role process, tested reduction in <br /> Heart function dcerk1 mutants lipase Pglym                          wild type flies compromised cardiac function using variety of <br /> knockdown                                                                      heart performance assays developed Drosophila <br />   The heart high demand energy relies fatty acids             [21]. Heart function compromised one-week old gut <br /> derived lipolysis lipoprotein bound TAG lipases                  specific knockdown CG6277 manifested decreased heart <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       9                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br /> rate increased heart period primarily increased                    important scenarios ceramide increased. One <br /> diastolic interval (Figure 5E). Knockdown CG6277 dcerk1                   example mouse ceramide transfer protein mutant, which <br /> mutant flies resulted increased heart period trend                accumulates ceramide dies embryonic day 11.5 [28]. <br /> towards decreased heart rate (Figure 5E). One-week old dcerk1                   These embryos severe mitochondrial degeneration, de- <br /> alone did significant changes heart parameters,                 creased phospho AKT level cardiac defects suggesting likely <br /> only slight trend slower heart rate dilation                    failure adaptive mechanism. CERT heterozygous mice are <br /> (increased systolic diastolic diameters). While gut specific                viable, mitochondrial damage stress severe as <br /> knockdown CG8093 wild type dcerk1 flies did               homozygotes. We tested central results AKT <br /> significant changes parameters tested heart                  activation decrease FOXO level observed CERT <br /> function, heart period affected dcerk1 expansion                heterozygous mice. Indeed, increased phosphoAKT level <br /> diastolic interval, pointing importance lipase heart         decreased levels different FOXO transcripts (Figures 7A, <br /> function dcerk1 flies (Figure S8).                                           7B). While results liver extracts shown here, down <br />    Since knockdown lipases resulted increased sensitivity             regulation FOXO observed tissues as <br /> to starvation cardiac defects, tested effects                intestine skeletal muscle. We observe increase in <br /> downregulating glycolytic gene, Pglym sensitivity                     transcripts phosphoglycerate mutase pyruvate kinase, <br /> starvation cardiac functions wild type mutant                        suggesting likely increase glycolysis animals <br /> backgrounds. Ubiquitious knockdown Pglym results                        (Figure 7C). We tested mammalian lipase family <br /> significant increase sensitivity starvation stress wild type           members showed homology CG8093 (LIPA, LIPF, LIPM) <br /> and dcerk1 (Figure S9). Evaluation various heart function                    CG6277 (pancreatic lipoprotein 1, pancreatic lipoprotein <br /> parameters Pglym knockdown flies wild type                     related protein, lipoprotein lipase, endothelial lipase) not <br /> dcerk1 mutant backgrounds showed decrease diastolic                    detect changes transcript levels. The lack validation targets <br /> systolic diameters resulting somewhat constricted <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span>                triglyceride utilization mice suggests CERT <br /> without significant changes parameters (Figure S9). This            heterozygotes ideal model lipolytic mechanisms <br /> reminiscent cardiac constricted phenotype eas mutants,              reflects complexities involved lipid sensing the <br /> show aberrations phosphatidylethanolamine synthesis                      gut coordination organs mammalian <br /> elevated SREBP activity [26].                                                   systems. <br />  <br /> Failure adaptive mechanisms dcerk1 mutants leads                          Discussion <br /> to cardiac defects decreased life span <br />     To test adaptive mechanisms observe young dcerk1                  In study, attempted understand dcerk1 flies <br /> flies (less week) sustained fails age, tested        survive adapt increased ceramide level, disrupts <br /> AKT activation measured FOXO transcript level aged                       physiological metabolic pathways. Our results <br /> dcerk1 (6 weeks). We observe decrease AKT activation 4-                collectively suggest presence increased ceramide, the <br /> fold increase FOXO transcript level older flies (data                 important coping strategies organismal survival are <br /> shown). Consequently, Pglym activity decreases significantly old             geared energy production metabolic remodeling <br /> dcerk1 flies (Figure 6A) leading increased trehalose level (data         involving upregulation genes function breakdown of <br /> shown). Similarly, transcript levels lipases decrease              dietary nutrients, increased glycolysis utilization stored fat <br /> age (Figure 6B), leading 50% increase TAG level old dcerk1             gut. This increased activation PI3K/AKT/ <br /> (data shown). These results suggest efficiency                   FOXO pathway subsequent activation novel downstream <br /> compensatory mechanisms decrease age mutant flies.                  effectors (summarized model, Figure 7D). Failure this <br /> The decompensation leads deterioration indices                    compensatory mechanism (due aging genetic reduction) leads <br /> heart performance young three-week old dcerk1 flies                 TAG accumulation, increased starvation sensitivity cardiac <br /> (Figure 6C), time point begin observe noticeable               abnormalities. <br /> decrease transcript levels lipases mutant. These                  While dcerk1 flies viable, genetic reduction AKT results in <br /> changes follow similar trend observed CG6277                    lethality showing survival dcerk1 dependent increased <br /> lipase knocked healthy three-week old wild type flies                activation AKT. AKT activation leads increased glycolytic <br /> (Figure 6D). In situations addition heart rate             flux mutants. Increase lactate level dcerk1 (in the <br /> heart period changes, observe dilated heart chamber,                    presence sufficient oxygen) reminiscent aerobic glycolysis <br /> perhaps increased dependence heart function                   observed normal proliferating cells cancer cells. In <br /> lipase aging. We chose interim period weeks                    normal proliferating cells, increasing glycolytic intermediates has <br /> because CG6277 lipase knockdown flies survive                    proposed support macromolecular synthesis a <br /> weeks. Thus, decrease CG6277 compromises cardiac function                  similar phenomenon called Warburg effect observed cancer <br /> likely altered energy balance resulting decreased TAG               cells glycolysis increased provide energy cells <br /> hydrolysis observed wild type flies transcript levels            defective mitochondrial respiration [54,55]. AKT does not <br /> are reduced RNAi aging dcerk1 mutants AKT                         activate traditional targets glycolysis hexokinase and <br /> mediated compensatory changes begin fail. We tested                     phosphofructokinase dcerk1 Pglym downstream <br /> whether impaired mitochondrial function age related                         glycolytic enzymes. Historically Pglym viewed a <br /> pathology triglyceride accumulation decline cardiac              rate-limiting step glycolysis studies suggested its <br /> function impacted life span dcerk1. While 90% mutants             importance heart, phagocytic cells cancer cells [56,57]. <br /> die day 60, 20% controls dead time,                  Thus, glycolytic pathway, control metabolic flux can <br /> suggesting adult life span mutant flies shortened                 achieved exerting control different enzymes depending <br /> (Figure 6E).                                                                    cellular conditions Pglym important node in <br />     We finally asked adaptive mechanisms increased               normal proliferating stressed cells. Addition of <br /> ceramide identified using Drosophila dcerk mutants likely              ceramide analogs reported inhibit AKT activation <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       10                           June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                               AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 6. Failure adaptive mechanisms leads cardiac dysfunction reduction adult life span dcerk1. (A). Measurement of <br /> Pglym enzyme activity young (1 week) aged (6 weeks) w1118 dcerk1 flies shows significant decrease activity older dcerk1. (B). <br /> Comparison transcript levels CG8093 CG6277 QPCR 1week 6 week old flies, reveals decrease transcripts aged dcerk1 <br /> flies. (C). Comparison cardiac performance indices 1 3 week old esgGAL4 dcerk1 flies reveals progressive decline heart function <br /> with decreased heart rate, increased heart period dilated heart chamber. n = 15, bar represents standard error mean statistical significance <br /> was determined using Student&#226;&#8364;&#8482;s t test; * denotes P, = 0.05-0.01, ** P, = 0.01-0.001 *** denotes P, = 0.001-0.0001. (D). Knockdown of <br /> CG6277 3 week old wild type flies shows decreased heart rate, increased heart period, dilated heart chamber arrhythmia. n = 10, bar <br /> represents standard error mean statistical significance determined using Student&#226;&#8364;&#8482;s t test; * denotes P, = 0.05-0.01, ** P, = 0.01-0.001 is <br /> and *** denotes P, = 0.001-0.0001. (E). dcerk1 reduced adult life span compared w1118 flies. The survivorship curve dcerk1 w1118 <br /> represents data 600 flies duplicate. <br /> doi:10.1371/journal.pgen.1003556.g006 <br />  <br /> without affecting upstream signaling events [58,59]. Possible                  effects use pharmacological amounts of <br /> mechanisms include activation protein phosphatase 2A,                 ceramide analogs, likely mimic endogenous <br /> could dephosphorylate AKT interference membrane                       ceramide. Nevertheless, effects negated by <br /> recruitment AKT atypical protein kinase C [59,60]. These                 overexpression constitutively active AKT [59]. How AKT <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      11                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                              AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />  <br />  <br /> Figure 7. Activation AKT, decrease FOXO activation glycolytic genes CERT heterozygous mice. (A). Total AKT and <br /> phosphoAKT levels monitored liver extracts wild type (+/+) CERT heterozygous (+/2) mice. PhosphoAKT level increased CERT <br /> heterozygotes total AKT significantly changed. (B). Comparison different FOXO liver tissue wild type (+/+) CERT heterozygous <br /> (+/2) mice shows significant decrease transcript levels. (C). Comparison glycolytic genes skeletal muscle wild type (+/+) CERT <br /> heterozygous (+/2) mice shows greater 5 fold increase transcript levels phosphoglycerate mutase pyruvate kinase 1.8 fold <br /> increase hexokinase. (D). A simple graphical model depicting metabolic reprogramming dcerk1 increased AKT activation decreased <br /> FOXO maintain energy balance. dcerk1 mutants glucose utilization increased glycolytic flux mediated Pglym downstream <br /> enzymes. They increased utilization triglycerides mediated CG8093 CG6277. <br /> doi:10.1371/journal.pgen.1003556.g007 <br />  <br /> activation achieved dcerk1? While mechanisms                 scriptional regulation known the <br /> contribute AKT activation, idea favor PIP2           regulation lipases FOXO. In Drosophila, dFOXO been <br /> available PI3K utilization PIP2 PLC severely               shown directly regulate acid lipase, dlip4 [66]. Recent studies <br /> decreased degradation dcerk1 [10]. While              hormone dependent module consisting salt induced <br /> increase tGPH suggesting increase PIP3,             kinase SIK3, histone deacetylase HDAC4, regulate <br /> able quantify PIP3 level mass spectrometry. However, PIP2               FOXO activity lipolysis [67]. In C.elegans, fat mobilization <br /> level shows 60% increase dcerk1 compared control [10].                  induction triglyceride lipase K04A8.5, target of <br />    Studies Drosophila highlighted role Brummer lipase          DAF16 increases longevity [68]. In mammalian systems, FOXO <br /> (ATGL homolog) fat storage mobilization fat body              shown regulate lipolysis directly controlling <br /> and mice shown ATGL mediated lipolysis important               expression ATGL PPARc [69]. A forkhead factor, <br /> PPARa-PGC1 complex activity heart function [19,61].                       FKHR shown upregulate lipoprotein lipase expression <br /> Magro, target nuclear receptor DHR96, hydrolyzes                     skeletal muscle [70]. While FOXO binding sites been <br /> dietary lipids gut stored cholesterol esters                identified promoter region lipoprotein lipase, can <br /> highlighting importance Drosophila gut maintaining              modulate expression directly demonstrated [71]. <br /> TAG cholesterol homeostasis [62,63]. In mammalian systems,                Further understanding role FOXOs different organs <br /> understanding lipid sensing intestine coordination          allow exploration therapeutic value targeting <br /> with organs liver, brain signaling                  FOXO metabolic diseases. <br /> pathways organs alter glucose energy homeostasis               Many recent studies sphingolipids relevant to <br /> are just beginning explored [64]. Our results suggest                   changes carbohydrate fat levels. In Drosophila, overexpres- <br /> utilization stored TAG gut important mechanism              sion glucosylceramide synthase catalyzes synthesis of <br /> in times energy need starvation stress CG8093              glucosylceramide increases TAG carbohydrate levels a <br /> CG6277 critical targets AKT/FOXO process. In                 reduction causes decrease fat storage [72]. Drosophila ceramide <br /> Drosophila larval model type 2 diabetes accompanied             synthase mutants reduction fat storage [73]. In <br /> high sugar TAG levels, gene expression changes reveal                mammalian systems, mice lacking sphingomyelin synthase and <br /> CG8093 CG6277 significantly downregulated (15                    ceramide kinase (both increase ceramide levels) are <br /> and 5 fold respectively) suggesting lipases role        resistant high fat diet induced obesity [74,75]. It be <br /> hypertriglyceridemia associated diabetes [65]. Metabolic                 interesting test animals increased lipolytic <br /> control lipolysis primarily studied level               mechanisms based observations Drosophila. Our results <br /> <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-sensing signal transduction cascades tran-               dcerk mutant presented mutants of <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     12                            June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                              AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br /> Drosophila sphingolipid enzymes (our unpublished observations)                Starvation assay <br /> suggest increase ceramide cellular compartment                In experiment, 10 batches 20 flies genotype <br /> eventually leads increase ceramide mitochondria               transferred vials containing small piece sponge soaked <br /> suggesting central effect ceramide accumulation               water. Dead flies counted 6 h intervals. <br /> organism impairment mitochondrial function. This impact on <br /> mitochondria validated recent mice knock                 Longevity assay <br /> studies sphingomyelin synthase 2 CERT mutants,                   Newly eclosed adult w1118 dcerk1 collected 24 <br /> which mitochondrial dysfunction [16,28].                                 hr period divided batches 20 flies vial. Flies were <br />    The mechanisms outlined mediate                        transferred fresh food vials scored survival 48 h. <br /> adaptation ceramide, important responses to <br /> reduced energy availability. The &#226;&#8364;&#732;protective effects&#226;&#8364;&#8482; adaptive <br />                                                                               Fly heartbeat analysis <br /> mechanisms revealed suggest potential <br />                                                                                  Semi-intact fly heart preparations used heart param- <br /> therapeutic implications ceramide mediated stress involving <br />                                                                               eter measurements according published protocols [20]. High <br /> decreased energy leading high sugar TAG levels, ultimately <br />                                                                               speed 1 min movies taken rate .100 frames per <br /> resulting cardiac dysfunction. Also, importantly, identifica- <br />                                                                               second using Hamamatsu CCD camera Leica DM LFSA <br /> tion novel downstream targets AKT/FOXO pathway                      microscope 106 dipping immersion lens. The images were <br /> opens possibility new therapeutic targets treatment              processed using Simple PCI imaging software (Hamamatsu Inc.). <br /> cardiac dysfunction, hyperglycemia hypertriglyceridemia.                  Cardiac parameters including heart rate, heart period, diastolic <br />                                                                               systolic intervals, diameters, fractional shortening and <br /> Materials Methods                                                         arrhythmic index generated using MatLab-based image <br />                                                                               analysis program. <br /> Fly stocks husbandry <br />    Drosophila stocks raised standard corn meal agar and <br /> maintained 25uC. 2&#226;&#8364;&#8220;7 day old flies used experiments            Adult gut larval fat body staining <br /> unless indicated. Akt4226, actin-GAL4 driver, MHC-                    For Oil red O staining, midguts 2&#226;&#8364;&#8220;7 days old adult flies <br /> GAL4 driver tGPH flies obtained Bloomington                     dissected fixed 4% paraformaldehyde/PBS for <br /> Stock Center (Indiana University). UAS-FOXO gift                   30 min room temperature. The midguts washed four <br /> Marc Tatar (Brown University, Providence), esg-Gal4 Norbert              times PBS incubated 20 30 min 0.1% Oil Red <br /> Perrimon (Harvard Medical School, Boston) C564-GAL4 flies                O stain (prepared fresh isopropanol:water passed <br /> Neal Silverman (UMass, Worcester) DTS flies Eric                0.45 mm syringe filter). Midguts washed three <br /> Baerahcke (UMass, Worcester). UAS RNAi lines CG8093,                      times PBS, mounted 20% glycerol/PBS. The guts <br /> CG6277 Pglym obtained VDRC (Vienna). Akt4226                    imaged immediately using Differential Interference <br />                                                                               Contrast microscopy Nikon Eclipse E-600 microscope. <br /> was recombined dcerk1 obtain dcerk1.akt/dcerk1 flies. UASRNAi <br />                                                                               For visualization tGPH, adult midgut fat body early <br /> lines CG8093, CG6277 Pglym chromosome <br />                                                                               instar larvae dissected PBS visualized <br /> were recombined dcerk1 obtain dcerk1.UASCG8093RNAi, <br />                                                                               immediately using Nikon Eclipse E-600 microscope NIS- <br /> dcerk1.UASCG6277RNAi dcerk1.UASPglymRNAi lines. UAS- <br />                                                                               Elements Imaging Software. <br /> FOXO, esg-GAL4, actin-GAL4, C564-GAL4 MHC-GAL4 <br /> drivers second chromosome used. <br />                                                                               Quantitative RT-PCR <br />                                                                                  Total RNA extracted using TRIzol reagent cDNA was <br /> Microarray analysis <br />        1      1118                                                            synthesized using SuperScript III first&#226;&#8364;&#8220;Strand Synthesis kit <br />    dcerk w       flies collected, RNA extracted using <br />                                                                               (invitrogen). Real time PCR performed ABI PRISM <br /> TRIzol (Invitrogen) purified RNeasy columns (QIAGEN). <br />                                                                               7000 sequence Detection System (Applied Biosystems) using <br /> All samples prepared triplicate facilitate subsequent <br />                                                                               SYBR Green Supermix (Invitrogen). Reactions normalized <br /> statistical analysis. Purified RNA sent Expression Analysis <br />                                                                               RP49 levels. Detailed methodology primers used are <br /> (Durham, NC), labeled hybridized Affymetrix <br />                                                                               provided Supplemental Materials Methods (Text S1). <br /> Drosophila 2.0 microarrays. The details analysis are <br /> described Supplemental Materials Methods (Text S1). <br />                                                                               Luciferase assays <br />                                                                                  Constructs. A 2.4-kb fragment containing FOXO core <br /> Metabolic profiling <br />        1       1118                                                           motif sites WWAACA putative regulatory region CG8093 <br />    dcerk w    (100 flies each, triplicate) collected and <br />                                                                               (chr2R 10725084 &#226;&#8364;&#8220; 10727488) identified. This fragment was <br /> frozen. The samples prepared analyzed LC/MS, LC/                  cloned pGL4-basic vector, firefly luciferase reporter <br /> MS/MS GC/MS platforms Metabolon (Durham, NC).                          gene (Promega). The fragment amplified PCR genomic <br />                                                                               DNA flies using following primers F59GGGGGCTAGCGA- <br /> ATP, TAG, trehalose, glycerol free fatty acid                             CAGCTACTTACATCCAAGCAATA39 R59GGGGCTCG <br /> measurements                                                                  AGCAACGAGCTCAAATTGGGTAAGA-39and cloned between <br />    TAG measurement carried using Serum triglyceride               NheI XhoI sites (pGLCG8093 2.4). The FOXO deficient <br /> determination kit (TR0100; Sigma, St. Louis, MO). Total body              fragment generated using primers F59GGGGGCTAGCC <br /> circulating trehalose measured using glucose reagent                   ACGAGAAATGTCAGGAAGTAATG39 R59GGGGGC <br /> (GAGO-20, Sigma) followed porcine kidney trehalase (Sigma,                 TAGCCACGAGAAATGTCAGGAAGTAATG39 cloned in <br /> T8778). ATP measurements carried using ATP assay                  pGL4 vector NheI XhoI sites (pGLCG8093 1.3kbdel). <br /> kit (Calbiochem). Glycerol free fatty acid determined using          The pAc5.1 vector constitutively active version of <br /> coupled enzyme assay kits (Zenbio, NC). Detailed methodology               dFOXO pAc5.1 vector gifts Drs. Oscar Puig and <br /> provided Supplemental Materials Methods (Text S1).                     Jaakko Mattila. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     13                          June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                  AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br />    Cell culture transfection. Drosophila Schneider (S2) cells                CG6277 shows approximately 20-fold increase transcript level <br /> were grown Schneider medium 10% heat-inactivated fetal                   midgut compared head CG8093 shows 5 fold <br /> bovine serum, penicillin, streptomycin 25uC suspension. For            increase suggesting genes highly expressed adult <br /> transfection, 16105 S2 cells plated 96 -well plate         midgut. n = 3, error bars represent standard deviation. <br /> triplicates. Cells transfected using Effectene transfection reagent         (TIF) <br /> (Qiagen) according manufacturer&#226;&#8364;&#8482;s instructions. For transfec- <br />                                                                                  Figure S5 Gut epithelial integrity epithelial renewal not <br /> tions, 100 ng expression vector (pAcFOXOA3 pAc5.1) and <br />                                                                                  compromised dcerk1 mutants. A. w1118 dcerk1 gut stained <br /> 100 ng reporter vector (pGLCG8093 2.4, pGLCG80931.3del) <br />                                                                                  Armadillo (b-Catenin homolog), important component of <br /> along renilla plasmid (20 ng) used. After 30 h transfection, <br />                                                                                  adherens junctions. The nuclei stained DAPI. Armadillo <br /> luciferase assays performed using dual-glow luciferase assay <br />                                                                                  staining significantly different w1118 dcerk1 <br /> system manufacturer&#226;&#8364;&#8482;s protocol (Promega). Each experiment <br />                                                                                  suggesting epithelial integrity compromised mutant. <br /> was repeated times. <br />                                                                                  Scale bar represents 25 mm. B. This panel shows w1118 dcerk1 <br />                                                                                  midguts expressing esg-GAL4 GFP, marks enteroblasts and <br /> Immunoblotting                                                                   intestinal stem cells involved epithelium renewal. GFP staining is <br />   Flies crushed SDS-sample buffer; extracts                     significantly different control mutant guts. The <br /> centrifuged separated SDS-PAGE followed Western                        guts stained DAPI. Scale bar represents 50 mm. C. <br /> blotting using antibodies AKT (1:1000, Cell Signaling)                    Survival dcerk1 significantly different w1118 upon <br /> phospho Drosophila AKT Ser505 (1:1000, Cell Signaling).                          ingestion Pseudomonas entomophila. Increased epithelial renewal is <br />                                                                                  required gut make damage caused oral pathogens <br /> Measurement activity glycolytic enzymes                                    flies low epithelial renewal succumb bacterial infection <br />   Soluble extracts prepared approximately 1000 flies                   earlier control flies. However, dcerk1 flies died similar rate <br /> per batch using different buffers different enzymes                      controls fed Pseudomonas entomophila suggesting epithelial <br /> spectrophotometric assays carried detailed                        renewal response gut damage dcerk1 compromised. <br /> Supplemental Materials Methods (Text S1).                                    (TIF) <br />                                                                                  Figure S6     Gut specific knockdown CG8093 CG6277 in <br /> Assays mitochondrial oxidative phosphorylation                               w1118 dcerk1. Combining expression esg-GAL4 driver and <br /> complexes                                                                        UASCG8093RNAi transgene effectively reduces CG8093 tran- <br />   Mitochondria isolated approximately 1000 flies                   script level w1118 dcerk1 flies. Similarly, combining <br /> batch. The activity of complexes determined                 expression esg-GAL4 driver UASCG6277RNAi transgene <br /> spectrophometrically detailed Supplemental Materials                   effectively reduces CG6277 transcript level w1118 dcerk1 flies. <br /> Methods (Text S1).                                                               n = 3, error bars represent standard deviation. <br />                                                                                  (TIF) <br /> Supporting Information <br />                                                                                  Figure S7 Knockdown CG8093 CG6277 w1118 results <br /> Figure S1 Ceramide levels increase ceramide 1-phosphate                    increased Oil Red O puncta midgut. Dissected adult <br /> levels significantly affected dcerk1 mutant fly extracts              midguts stained Oil Red O significantly increased <br /> compared w1118. (A). d14 long chain <span id='am-243' about='xsp:base' typeof='owl:Thing'>base</span> ceramide                        staining RNAi knockdown w1118 compared RNAi controls. <br /> different fatty acids, C20:0, C22:0, C24;1 C24:0 estimated               Oil Red O puncta significantly right panels and <br /> by mass spectrometry sphingolipid enriched fractions prepared                 puncta appear bigger CG8093 knockdown. 15&#226;&#8364;&#8220;20 guts <br /> from w1118 dcerk1 flies. The ceramide calculated                examined genotype experiment, n = 3. <br /> based total carbon content normalized w1118. All                  (TIF) <br /> ceramides significant increase (P, = 0.001-0.0001) in <br />                                                                                  Figure S8 Quantification cardiac function parameters in <br /> mutant compared w1118. n = 3, error bars represent standard <br />                                                                                  RNAi knockdown CG8093 wild type dcerk1. Knockdown <br /> deviation. (B). d14 long chain base ceramide 1-phosphate (C1P) <br />                                                                                  CG8093 does significant changes the <br /> with fatty acids C20:0, C22:0 C24:0 estimated mass <br />                                                                                  parameters tested backgrounds. However, heart period is <br /> spectrometry. A 20% decrease C24:0 C1P observed mutant <br />                                                                                  increased expansion diastolic interval dcerk1 when <br /> relative w1118. n = 3, error bars represent standard deviation. <br />                                                                                  CG8093 knocked down. n = 11&#226;&#8364;&#8220;13 group. Bar <br /> (TIF) <br />                                                                                  represents standard error mean. <br /> Figure S2    Western blot total AKT phospho AKT level                  (TIF) <br /> akt4226/+ heterozygotes. (A). Western blot total AKT and <br />                                                                                  Figure S9 Assessment starvation sensitivity cardiac <br /> phospho AKT w1118, dcerk1 akt4226/+. (B). Quantification of <br />                                                                                  function parameters RNAi knockdown Pglym wild type <br /> the blots densitometric scanning shows 50% decrease total <br />                                                                                  dcerk1. Ubiquitous knockdown Pglym results significant <br /> AKT 30% reduction pAKT heterozygote. n = 3. <br />                                                                                  increase starvation sensitivity backgrounds. Knockdown <br /> (TIF) <br />                                                                                  Pglym decreases diastolic systolic diameters both <br /> Figure S3 Ubiquitous RNAi knockdown Pglym w1118                        backgrounds. n = 11&#226;&#8364;&#8220;13. Bar represents standard error mean. <br /> dcerk1. Combining actin GAL4 driver UASPglymRNAi                            (TIF) <br /> transgene results significant reduction Pglym transcript level <br />                                                                                  Table S1 List increased decreased genes microarray <br /> in w1118 dcerk1. n = 3, error bars represent standard deviation. <br />                                                                                  analysis. List genes expression increase 2 fold more <br /> (TIF) <br />                                                                                  decrease 0.5 fold dcerk1 compared w1118 flies shown. <br /> Figure S4 CG6277 CG8093 high expression                          The columns represent Affymetrix ID, CG number, log fold change, P <br /> adult midgut. QPCR analysis carried using total RNA                       value, normalized log 2 transformed intensity present call. <br /> extracted adult head dissected adult midgut w1118.                 (PDF) <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        14                             June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                                AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br /> Table S2 List candidate genes microarray analysis                               ATP, free fatty acid glycerol, QPCR analysis, immunofluo- <br /> contain FOXO binding sites. The table shows genes                                   rescence microscopy, activity measurements glycolytic en- <br /> microarray analysis potential FOXO binding sites                              zymes, assays mitochondrial oxidative phosphorylation com- <br /> identified. The columns FBgn ID chromosome location,                               plexes feeding Pseudomonas entomophila. <br /> strand, gene start end positions, CG numbers, peak start                             (DOCX) <br /> end positions, distance peak, shortest distance peak and <br /> whether overlapping nearest.                                                        Acknowledgments <br /> (PDF) <br />                                                                                              We thank Drs. Perrimon Tatar, Bloomington stock center, VDRC <br /> Table S3 Fold change (dcerk1/w1118) metabolites                                 fly stocks Drs. Puig, Mattila, Dekker, Sanyal reagents. We <br /> classification different pathways. The table shows relative                             thank Dr. Jairaj Acharya helpful discussions comments the <br /> changes different metabolites control mutant flies.                           manuscript. We grateful Dr. Yong Xu Wang comments the <br /> The metabolites classified functional pathways. A p                           manuscript. We thank Kathya Acharya help figures. <br /> value 0.05 q value (false discovery rate) 0.1 less <br /> is considered statistically significant.                                                     Author Contributions <br /> (PDF) <br />                                                                                              Conceived designed experiments: URA TB RB. Performed the <br /> Text S1 Methodology detailed Supplemental Materials                                   experiments: NKN MR URA SMW AS TB. Analyzed data: URA TB <br /> Methods. Additional details provided microarray analysis,                            EF GLH YTI RB. Contributed reagents/materials/analysis tools: NKN <br /> metabolic assays including measurements TAG, trehalose,                                   MR URA SMW LJZ SMS. Wrote paper: URA. <br />  <br />  <br /> References <br />  1. Samuel VT, Shulman GI (2012) Mechanisms insulin resistance: common                   23. Teleman AA (2009) Molecular mechanisms metabolic regulation insulin in <br />     threads missing links. Cell 148: 852&#226;&#8364;&#8220;871.                                                Drosophila. Biochem J 425: 13&#226;&#8364;&#8220;26. <br />  2. Bikman BT, Summers SA (2011) Ceramides modulators cellular                     24. Birse RT, Choi J, Reardon K, Rodriguez J, Graham S, et al. (2010) High-fat- <br />     whole-body metabolism. J Clin Invest 121: 4222&#226;&#8364;&#8220;4230.                                         diet-induced obesity heart dysfunction regulated TOR pathway <br />  3. Hannun YA, Obeid LM (2008) Principles bioactive lipid signalling: lessons                 Drosophila. Cell Metab 12: 533&#226;&#8364;&#8220;544. <br />     sphingolipids. Nat Rev Mol Cell Bio 9: 139&#226;&#8364;&#8220;150.                                     25. Partridge L, Alic N, Bjedov I, Piper MD (2011) Ageing In Drosophila: role of <br />  4. Holland WL, Brozinick JT, Wang LP, Hawkins ED, Sargent KM, et al. (2007)                     insulin/Igf TOR signalling network. Exp Gerontol 46: 376&#226;&#8364;&#8220;381. <br />     Inhibition ceramide synthesis ameliorates glucocorticoid-, saturated-fat-,        26. Lim HY, Wang W, Wessells RJ, Ocorr K, Bodmer R (2011) Phospholipid <br />     obesity-induced insulin resistance. Cell Metab 5: 167&#226;&#8364;&#8220;179.                                   homeostasis regulates lipid metabolism cardiac function SREBP <br />  5. Nikolova-Karakashian MN, Rozenova KA (2010) Ceramide stress response.                     signaling Drosophila. Genes Dev 25: 189&#226;&#8364;&#8220;200. <br />     Adv Exp Med Biol 688: 86&#226;&#8364;&#8220;108.                                                            27. Na J, Musselman LP, Pendse J, Baranski TJ, Bomer R, et al. (2013). A <br />  6. Jiang XC, Goldberg IJ, Park TS (2011) Sphingolipids cardiovascular                       Drosophila model high sugar diet-induced cardiomyopathy. PLoS Genet 9: <br />     diseases: lipoprotein metabolism, atherosclerosis cardiomyopathy. Adv Exp                e1003175. <br />     Med Biol 721: 19&#226;&#8364;&#8220;39.                                                                     28. Wang X, Rao RP, Kosakowska-Cholody T, Massod MA, Southon E, et al. <br />  7. Holland WL, Miller RA, Wang ZV, Sun K, Barth BM, et al. (2011) Receptor-                     (2009) Mitochondrial degeneration apoptosis primary cause of <br />     mediated activation ceramidase activity initiates pleiotropic actions              embryonic lethality ceramide transfer protein mutant mice. J Cell Biol 184: <br />     adiponectin. Nat Med 17: 55&#226;&#8364;&#8220;63.                                                              143&#226;&#8364;&#8220;158. <br />  8. Rohrbough J, Rushton E, Palanker L, Woodruff E, Matthies HI, et al. (2004)               29. Huang da W, Sherman BT, Lempicki RA (2009) Systematic integrative <br />     Ceramidase regulates synaptic vesicle excocytosis trafficking. J Neurosci 24:            analysis large gene lists using DAVID bioinformatics resources. Nat Protoc 4: <br />     7789&#226;&#8364;&#8220;78038.                                                                                  44&#226;&#8364;&#8220;57. <br />  9. Acharya JK, Dasgupta U, Rawat SS, Yuan C, Sanxaridis P, et al. (2008) Cell               30. Alic N, Andrews TD, Giannakou ME, Papatheodorou I, Slack C, et al. (2011) <br />     non-autonomous function ceramidase photoreceptor homeostasis. Neuron                   Genome-wide dFOXO targets topology transcriptomic response to <br />     57: 69&#226;&#8364;&#8220;79.                                                                                   stress insulin signaling. Mol Syst Biol 7: 502. <br /> 10. Dasgupta U, Bamba T, Chiantia S, Karim P, Abou Tayoun AN, et al. (2009)                  31. Zhu LJ, Gazin C, Lawson ND, Page`s H, Lin SM, et al. (2010) ChIPpeakAnno: a <br />     Ceramide kinase regulates phospholipase C phosphatidylinositol 4, 5,                     Bioconductor package annotate ChIP-seq ChIP-chip data. BMC <br />     bisphosphate phototransduction. Proc Natl Acad Sci USA 106: 20063&#226;&#8364;&#8220;20068.                  Bioinformatics 11: 237. <br /> 11. Yuan C, Rao RP, Jesmin N, Bamba T, Nagashima K, et al. (2011) CDase                 32. Hanada M, Feng J, Hemmings BA (2004) Structure, regulation function of <br />     pan-ceramidase Drosophila. Mol Biol Cell 22: 33&#226;&#8364;&#8220;43.                                       PKB/AKT-a major therapeutic target. Biochim Biophys Acta 1697: 3&#226;&#8364;&#8220;16. <br /> 12. Liang H, Yao N, Song JT, Luo S, Lu H, et al (2003) Ceramides modulate                    33. Hers I, Vincent EE, Tavare&#194;&#180; JM (2011) Akt signalling health disease. Cell <br />     programmed cell death plants. Genes Dev 17: 2636&#226;&#8364;&#8220;2641.                                    Signal 23: 1515&#226;&#8364;&#8220;1527. <br /> 13. Mitsutake S, Yokose U, Kato M, Matsuoka I, Yoo JM, et al. (2007). The                    34. Alessi DR, Andjelkovic M, Caudwell B, Cron P, Morrice N, et al. (1996) <br />     generation behavioral analysis ceramide kinase-null mice, indicating                Mechanism activation protein kinase B insulin IGF-1. EMBO J 15: <br />     function cerebellar Purkinje cells. Biochem Biophys Res Commun 363: 519&#226;&#8364;&#8220;                  6541&#226;&#8364;&#8220;6551. <br />     524.                                                                                     35. Sarbassov DD, Guertin DA, Ali SM, Sabatini DM (2005) Phosphorylation and <br /> 14. Graf C, Zemann B, Rovina P, Urtz N, Schanzer A, et al. (2008) Neutropenia                    regulation Akt/PKB rictor-mTOR complex. Science 307: 1098&#226;&#8364;&#8220;1101. <br />     impaired immune response Streptococcus pneumoniae ceramide                    36. Stocker H, Andjelkovic M, Oldham S, Laffargue M, Wymann MP, et al. (2002) <br />     kinase-deficient mice. J Immunol 180: 3457&#226;&#8364;&#8220;3466.                                             Living lethal PIP3 levels: viability flies lacking PTEN restored PH <br /> 15. Hailemariam TK, Huan C, Liu J, Li Z, Roman C, et al. (2008). Sphingomyelin                   domain mutation Akt/PKB. Science 295: 2088&#226;&#8364;&#8220;2091. <br />     synthase activation attenuates NF kappaB activation. Arterioscler Thromb Vasc            37. Britton JS, Lockwood WK, Li L, Cohen SM, Edgar BA (2002) Drosophila&#226;&#8364;&#8482;s <br />     Biol 28: 1519&#226;&#8364;&#8220;1526.                                                                          insulin/PI3-kinase pathway coordinates cellular metabolism nutritional <br /> 16. Yano M, Watanabe K, Yamamoto T, Ikeda K, Senokuchi T, et al. (2011)                          conditions. Dev Cell 2: 239&#226;&#8364;&#8220;249. <br />     Mitochondrial dysfunction increased reactive oxygen species impair insulin           38. Brunet A, Bonni A, Zigmond MJ, Lin MZ, Juo P, et al. (1999) Akt promotes cell <br />     secretion sphingomyelin synthase 1-null mice. J Biol Chem 286: 3992&#226;&#8364;&#8220;4002.                 survival phosphorylating inhibiting Forkhead transcription factor. Cell <br /> 17. Baker KD, Thummel CS (2007) Diabetic larvae obese flies-Emerging                         96: 857&#226;&#8364;&#8220;868. <br />     studies metabolism Drosophila. Cell Metab 6: 257&#226;&#8364;&#8220;266.                              39. Gross DN, Wan M, Birnbaum MJ (2009) The role FOXO regulation of <br /> 18. Rulifson EJ, Kim SK, Nusse R (2002) Ablation insulin-producing neurons                 metabolism. Curr Diab Rep 9: 208&#226;&#8364;&#8220;214. <br />     flies: Growth diabetic phenotypes. Science 296: 1118&#226;&#8364;&#8220;1120.                           40. Puig O, Marr MT, Ruhf ML, Tjian R (2003) Control cell number by <br /> 19. Gronke S, Mildner A, Fellert S, Tennagels N, Petry S, et al. (2005) Brummer                  Drosophila FOXO: downstream feedback regulation insulin receptor <br />     lipase evolutionary conserved fat storage regulator Drosophila. Cell                pathway. Genes Dev 17: 2006&#226;&#8364;&#8220;2020. <br />     Metab 1: 323&#226;&#8364;&#8220;330                                                                         41. Junger MA, Rintelen F, Stocker H, Wasserman JD, Vegh M, et al. (2003) The <br /> 20. Ocorr K, Akasaka T, Bodmer R (2007) Age-related cardiac disease model                     Drosophila Forkhead transcription factor FOXO mediates reduction cell <br />     Drosophila. Mech Ageing Dev 128: 112&#226;&#8364;&#8220;116.                                                    number associated reduced insulin signaling. J Biol 2: 20. <br /> 21. Leopold P, Perrimon N (2007) Drosophila genetics internal                 42. Plas DR, Thompson CB (2003) Akt activation promotes degradation tuberin <br />     milieu. Nature 450: 186&#226;&#8364;&#8220;188.                                                                 FOXO3a proteosome. J Biol Chem 278: 12361&#226;&#8364;&#8220;12366. <br /> 22. Hietakangas V, Cohen SM (2009) Regulation tissue growth <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>              43. Belote JM, Fortier E (2002) Targeted expression dominant negative <br />     sensing. Annu Rev Genet 43: 389&#226;&#8364;&#8220;410.                                                         proteasome mutants Drosophila melanogaster. Genesis 34: 80&#226;&#8364;&#8220;82. <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                    15                                June 2013 | Volume 9 | Issue 6 | e1003556 <br />                                                                                                               AKT/FOXO Mediated Metabolic Adaptation Ceramide <br />  <br />  <br /> 44. Hwangbo DS, Gershman B, Tu MP, Palmer M, Tatar M (2004) Drosophila                           step underpinning disruption PKB-directed signalling ceramide. <br />     dFOXO controls lifespan regulates insulin signalling brain fat body.              Biochem J 410: 369&#226;&#8364;&#8220;379. <br />     Nature 429: 562&#226;&#8364;&#8220;566.                                                                   61.   Haemmerle G, Moustafa T, Woelkart G, Buttner S, Schmidt A, et al. (2011) <br /> 45. Demontis F, Perrimon N (2010) FOXO/4E-BP signaling Drosophila muscles                     ATGL-mediated fat catabolism regulates cardiac mitochondrial function via <br />     regulates organism-wide proteostasis aging. Cell 143: 1&#226;&#8364;&#8220;13.                           PPAR-alpha PGC-1. Nat Med 17: 1076&#226;&#8364;&#8220;1085. <br /> 46. Walther TC, Farese RV Jr (2009) The life lipid droplets. Biochim Biophys            62.   Sieber MH, Thummel CS (2009) The DHR96 nuclear receptor controls <br />     Acta 1791: 459&#226;&#8364;&#8220;466.                                                                          triacylglycerol homeostasis Drosophila. Cell Metab 10: 481&#226;&#8364;&#8220;490. <br /> 47. Kunhlein RP (2011) The contribution Drosophila model lipid droplet           63.   Sieber MH, Thummel CS (2012) Coordination triaclglyceroland cholesterol <br />     research. Prog Lipid Res 50: 348&#226;&#8364;&#8220;356.                                                        homeostasis DHR96 Drosophila LipA Homolog magro. Cell Metab <br /> 48. Arrese EL, Soulages JL (2010) Insect fat body: energy, metabolism,                       15: 122&#226;&#8364;&#8220;127. <br />     regulation. Annu Rev Entomol 55: 207&#226;&#8364;&#8220;225.                                              64.   Rasmussen BA, Breen DM, Lam TKT (2012) Lipid sensing gut, brain and <br /> 49. Zinke I, Schutz CS, Katzenberger JD, Bauer M, Pankratz MJ (2002) <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>Nutrient</span>                    liver. Trends Endocrinology Metabolism 23: 49&#226;&#8364;&#8220;55. <br />     control gene expression Drosophila: Microarray analysis starvation        65.   Musselman LP, Fink JL, Narzinski K, Venkatesh Ramachandran P, Hathir- <br />     sugar-dependent response. EMBO J 21: 6162&#226;&#8364;&#8220;6173.                                              amani SS, et al. (2011) A high-sugar diet produces obesity insulin resistance <br /> 50. Micchelli CA, Perrimon N (2006) Evidence stem cells reside adult                 wild-type Drosophila. Dis Model Me <br /> </body></html>