<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Downregulation yidC Escherichia coli Antisense <br /> RNA Expression Results Sensitization Antibacterial <br /> Essential Oils Eugenol Carvacrol <br /> Supriya Deepak Patil, Rajnikant Sharma, Santosh Srivastava, Naveen Kumar Navani, Ranjana Pathania* <br /> Chemical Biology Drug Discovery Lab, Department Biotechnology, Indian Institute Technology Roorkee, Roorkee, Uttarakhand, India <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: The rising drug resistance pathogenic bacteria inefficiency current antibiotics meet clinical <br />      requirements augmented need establish new innovative approaches antibacterial drug discovery <br />      involving identification novel antibacterial targets inhibitors. Being obligatory bacterial growth, essential gene <br />      products considered vital drug targets. The bacterial protein YidC highly conserved pathogens is <br />      essential membrane protein insertion holds immense potential promising target antibacterial <br />      therapy. <br />  <br />      Methods/Principal Findings: The aim study explore feasibility efficacy expressed antisense- <br />      mediated gene silencing specific downregulation yidC Escherichia coli. We induced RNA silencing yidC which <br />      resulted impaired growth host cells. This followed search antibacterial compounds sensitizing YidC <br />      depleted cells act inhibitors essential protein products. The present findings affirm that <br />      reduction YidC synthesis results bacterial growth retardation, warrants use enzyme viable target <br />      search novel antibacterial agents. Moreover, yidC antisense expression E. coli resulted sensitization antibacterial <br />      essential oils eugenol carvacrol. Fractional Inhibitory Concentration Indices (FICIs) point high level synergy <br />      yidC silencing eugenol/carvacrol treatment. Finally, known YidC inhibitors, RNA silencing <br />      approach applied study forward rapid means screen novel potential YidC inhibitors. <br />  <br />      Conclusions/Significance: The present results suggest YidC promising candidate target screening antibacterial <br />      agents. High level synergy reported yidC silencing eugenol/carvacrol treatment indicative a <br />      potential antibacterial therapy. This report indicating essential gene yidC therapeutic target the <br />      antibacterial essential oils eugenol carvacrol E. coli. <br />  <br />   Citation: Patil SD, Sharma R, Srivastava S, Navani NK, Pathania R (2013) Downregulation yidC Escherichia coli Antisense RNA Expression Results in <br />   Sensitization Antibacterial Essential Oils Eugenol Carvacrol. PLoS ONE 8(3): e57370. doi:10.1371/journal.pone.0057370 <br />   Editor: Hendrik W. van Veen, University Cambridge, United Kingdom <br />   Received November 29, 2012; Accepted January 21, 2013; Published March 4, 2013 <br />   Copyright: &#195;&#376; 2013 Patil et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: Financial support work provided Department Science Technology (DST), New Delhi, India Fast Track Scheme (grant <br />   no. SR/FT/LS-129/2008) awarded RP. SDP thankful Council Scientific Industrial Research (CSIR); RS SS thank Ministry Human Resource <br />   Development (MHRD), Government India financial support. The funding agencies role study design, data collection analysis, decision <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />   publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: ranjanapathania@gmail.com <br />  <br />  <br />  <br /> Introduction                                                                              Most antibiotics current medical use were <br />                                                                                        identified whole-cell (<span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> based) screening assays in <br />    Evolution multidrug resistance bacterial pathogens led                    compounds potent antibacterial activity are <br /> to situation diminutive treatment                           selected first, followed identification cellular targets <br /> options infections certain pathogens. Consequently,                       [2,6]. However, consequently led rediscovery and <br /> need new antibacterial agents increased [1&#226;&#8364;&#8220;4]. Regardless                      identification classes antibiotics proved <br /> of rise resistant pathogens, pace new antibiotic                         inadequate beat problem drug resistance. Moreover, it <br /> approvals falling [1,5,6]. Over past decades, order                   difficult task deduce targets mode action of <br /> solve problem drug resistance, existing antibiotics                   antibacterial agents identified random cell screens. <br /> chemically modified [1]. However, led relatively narrow                    Therefore, target-directed antibacterial discovery approaches <br /> diversity compound classes aiming limited number targets                      created merging random whole-cell screening with <br /> [7]. Therefore, solution lies isolating synthesizing new                     genetic methods. Differential growth assays based bacterial <br /> antibacterial agents non-classical cellular targets. Antimicro-                   antisense technology applied target-directed whole-cell <br /> bials target unexplored proteins bacteria likely                 screening [2,9]. Antisense expression approach used to <br /> vulnerable existing target-based mechanisms resistance                           down-regulate expression specific gene post-transcrip- <br /> because distinctive modes action [8].                                      tional level means inducible promoter plasmid <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         1                                March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                               yidC Promising Antibacterial Target <br />  <br />  <br /> vector containing gene fragment antisense orientation. If         gene silencing effectiveness E. coli [10]. By using approach, <br /> gene essential, antisense RNA (asRNA) synthesis lead              possible decrease cellular level yidC mRNA <br /> inhibition host cell growth binding corresponding               consequently leading retarded growth host cells. Reduction of <br /> messenger RNA (mRNA) making non-functional [10,11].                   expression gene sensitize cells compounds inhibit <br /> This achieved largely increasing degradation mRNA [11]              activity gene product related process [16,29]. Our <br /> as inhibition translation hybridizing sequences             results yidC antisense expressing clone sensitized to <br /> flanking ribosome binding site (RBS) start codon          membrane disintegrating membrane bound ATPase inhibi- <br /> mRNA [12,13]. Moreover, surplus relative availability                   tory antibacterial essential oils eugenol carvacrol. To best <br /> asRNAs probably facilitate out-compete target                   knowledge, report indicates the <br /> mRNAs ribosome [10]. Using approach Staphylococcus               essential gene yidC therapeutic target antibacterial essential <br /> aureus, inhibitors targeting essential enzymes fatty acid         oils eugenol carvacrol. <br /> synthesis pathway discovered [9,14,15]. Although asRNA <br /> approach used elaborate mechanism action               Materials Methods <br /> natural products E. coli [16], dearth published <br /> reports describing application approach screening            Strains Plasmids <br /> target specific antibacterial agents Gram-negative bacteria.                 Antisense cloning vector pHN678 kindly provided Dr <br />    Proteins mandatory bacterial <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>survival</span>                Liam Good (Department Pathology Infectious Diseases, <br /> conserved pathogens considered excellent targets               Royal Veterinary College, London, UK). The multiple cloning site <br /> inhibitor screening [2,6]. The availability complete genome               (MCS) pHN678 flanked paired-termini impart stability <br /> sequences allows identify proteins conserved           antisense RNA expression [10]. E. coli DH5a strain was <br /> the medically important pathogens absent higher eukaryotes            used host cell experiments grown Luria <br /> thereby excluding potential toxicity [2]. Many essential proteins            Bertani (LB) medium (Merck, Germany) 37uC 250 rpm <br /> have identified bacteria prospective drug targets                 shaking specified. Chloramphenicol (Sigma <br /> [17,18]. However, current drugs target bacterial              Chemical Company, St. Louis, Mo, USA) concentration of <br /> pathways [19]. Conversely, identification new classes                  30 mg/ml included maintain plasmid. <br /> antibacterial compounds directed unexplored <br /> targets bacteria critical combat emerging resistance [8].           Transcript Target Site Selection <br /> One essential E. coli proteins, YidC 60-kDa membrane                Part yidC gene sequence antisense expression was <br /> protein [20]. Its functions involve membrane protein translocation           chosen based criteria described previously [30]. It been <br /> and insertion. YidC functions jointly Sec translocase           shown earlier antisense sequence small <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>, <br /> can function separately facilitate insertion proteins         located start codon include promoter and <br /> into cell membrane [21]. It evolutionarily conserved               coding regions minimal secondary structures. <br /> protein shown experimentally essential                The secondary structures predicted RNAfold (http://rna. <br /> growth E. coli [22]. It shown YidC depletion leads             tbi.univie.ac.at/cgi-bin/RNAfold.cgi). The primers designed <br /> decrease functional assembly cytochrome o oxidase              expressed asRNAs hybridize sequences <br /> F1Fo ATPase strong reduction proton-motive force             flanking RBS start codon yidC mRNA. The primers <br /> in E. coli [23]. All Gram-positive bacteria gene           used study given table 1. <br /> encoding YidC homolog genome, most, like the <br /> oral pathogenic bacterium Streptococcus mutans contain second         Cloning Antisense RNA Expression Constructs, Insert <br /> gene YidC homolog [24]. Being common feature                 DNA Sequencing Bioinformatic Analysis <br /> Gram-negative Gram-positive bacteria, attractive                   Selected nucleotide sequence yidC murA essential genes <br /> target development broad-spectrum antibacterial agents.           amplified E. coli MG1655 genomic DNA (using primers <br /> Additionally, YidC different targets existing drugs [19],         given table 1), digested NcoI (Fermentas, USA) XhoI <br /> is essential bacterial growth [22] exhibits divergence            (Fermentas, USA) cloned MCS pHN678 with <br /> eukaryotes bacteria [25] sustain intrinsic             similarly digested ends. Recombinant plasmids transformed <br /> significance promising antibacterial target.                            E. coli DH5a competent cells selected LB agar <br />    Natural products affluent resource leads              supplemented chloramphenicol (30 mg/ml). <br /> antibacterial drug discovery [3]. Drugs originated plant                   In order confirm inserts, plasmids isolated the <br /> sources used healing various human ailments                transformants sequenced Ocimum Biosolutions, India to <br /> thousands years form traditional medicines [7].                 determine DNA sequences inserts orientations. <br /> Essential oils, known volatile oils extracted plants.       The DNA sequences compared annotated <br /> They aromatic volatile nature                genomic sequence E. coli MG1655 (<span id='am-2' about='Thesaurus:GenBank_Accession_Number' typeof='owl:Thing'>Genbank accession number</span> <br /> known possess antibacterial properties [26]. Eugenol                  NC_000913) determine origin DNA inserts their <br /> carvacrol antibacterial components essential oils             orientation using BLAST (Basic Local Alignment Search Tool) <br /> which known inhibit number Gram-negative                     NCBI (National Center Biotechnology Information) <br /> Gram-positive pathogenic bacteria [27]. They kill bacteria                (http://blast.ncbi.nlm.nih.gov/Blast.cgi). <br /> causing membrane disruption increased permeability of <br /> bacterial cells. They known possess membrane bound               E. coli Growth Inhibition Experiments <br /> ATPase inhibitory activity E. coli [28].                                     The growth inhibitory effect antisense expression yidC and <br />    The aim study explore feasibility efficacy         murA essential genes (antisense expression murA used a <br /> of expressed antisense-mediated gene silencing specific                  positive control) E.coli examined assessing cell growth. <br /> downregulation yidC E. coli. An antisense expression vector,           E. coli DH5a carrying pHN678, pHN678 yidC antisense <br /> pHN678 used, designed expressed asRNA                  insert (pHN678-yidC) pHN678 murA antisense insert <br /> molecules paired termini increase stability          (pHN678-murA) grown overnight 10 ml LB broth <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                            March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                                          yidC Promising Antibacterial Target <br />  <br />  <br />  <br />  Table 1. Primers used study. <br />  <br />  <br />                                                                                                     Product size         Purpose of <br />  Primer                   Primer sequence (59&#226;&#8364;&#8220;39)                             Target                (<span id='am-164' about='xsp:base' typeof='owl:Thing'>base</span> pair)          amplicon             Reference/Source <br />  <br />  yidC-XF                  cgtccatggcatgaaagacacgaacag                         Antisense yidC     134                  Cloning yidC      This study <br />                                                                                                                          antisense into <br />                                                                                                                          pHN678 <br />  yidC-XR                  tgtctcgaggtattaaaatgccacc <br />  murA-XF                  cgtccatgggagcagcatttttagcgc                         Antisense murA     130                  Cloning murA      [30] <br />                                                                                                                          antisense into <br />                                                                                                                          pHN678 <br />  murA-XR                  tgtctcgaggctatgggcgattcgc <br />  yidC-F                   ctggggcttctccattatca                                yidC gene fragment 279                     qPCR yidC         This study <br />  yidC-R                   agttcaacggaacccatcag <br />  16SrRNA-F                cagccacactggaactgaga                                16S rRNA gene         204                  qPCR reference    This study <br />                                                                               fragment                                   gene <br />  16SrRNA-R                gttagccggtgcttcttctg <br />  <br />  doi:10.1371/journal.pone.0057370.t001 <br />  <br />  <br /> supplemented chloramphenicol (30 mg/ml) 37uC                           Relative Real-time (RT) Quantitative PCR (qPCR) <br /> shaking 250 rpm. Following overnight growth, cells                         Total RNA isolated cells using TRI reagent <br /> diluted 100 fold fresh LB broth medium supplemented                         (MRC, Cincinnati, Ohio, USA) manufacturer&#226;&#8364;&#8482;s instructions <br /> chloramphenicol (30 mg/ml). This diluted culture (5 ml)                         followed DNase I (Fermentas, USA) treatment for <br /> spotted LB chloramphenicol plates (for uninduced conditions)                     removal contaminating genomic DNA. Total RNA (1 mg) was <br /> and LB chloramphenicol plates supplemented Isopropyl-                          converted strand-specific cDNA 25 ml reaction consisting of <br /> beta-thiogalactopyranoside (IPTG) (Fermentas, USA) concentra-                       1 X RT reaction buffer, 4 mM dNTP, 1.6 mM gene <br /> tions ranging 1 20 mM (for induced conditions). These                       specific reverse primers, 20U RNase inhibitor, 1 ml of <br /> plates incubated 37uC 14 hours. Growth uninduced                     lmProm-II Reverse Transcriptase (Promega, Madison, USA). <br /> and induced plates compared naked eyes.                                    Each 25 ml PCR reaction contained 12.5 ml Light cycler <br />     Antisense expression checked liquid medium                       480 SYBR Green I master (Roche diagnostics, Germany), 0.2 mM <br /> described previously [30] modifications. Various                          primer 2 ml (450 ng) cDNA. Relative qPCR was <br /> concentrations IPTG (40, 80, 160 320 mM) added                          carried Smart Cycler (Cepheid, Sunnyvale, CA) with <br /> in triplicates Mueller Hinton (MH) broth (Merck, Germany)                        primers yidC-F/R specific yidC target gene, primers <br /> supplemented chloramphenicol (30 mg/ml) separate wells                      16S rRNA-F/R specific 16S rRNA reference gene <br /> of 96-well plate (Genaxy, India) making final volume 100 ml.                  (endogenous gene) (Table 1). <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> fluorescent values obtained <br /> A control IPTG included. Overnight cultures                    real-time PCR analysis, threshold cycles (CT- <br /> of E. coli DH5a carrying plasmids expressing asRNA (Table 2)                     value) obtained. The relative expression ratio calculated <br /> MH broth supplemented chloramphenicol (30 mg/ml)                          given described previously [31]. <br /> subcultured (1% inoculum) fresh medium incubated further <br /> till optical density 600 nm (OD600) reached 0.5 0.7. This                       Here, DCT target gene (control2test) = CT value yidC <br /> culture diluted standardized OD600 readings                      gene control RNA (RNA uninduced cells)2CT <br /> approximately 105 cfu/ml 100 ml diluted culture                    value yidC gene test RNA (RNA IPTG <br /> was added 96-well plate make final volume                 induced cells). Also, DCT endogenous gene (control2test) = CT <br /> 200 ml incubated 37uC 150 rpm shaking. The growth                       value 16S rRNA gene control RNA (RNA from <br /> was monitored OD600 readings hour using spectramax                         uninduced cells)2CT value 16S rRNA gene test <br /> plus plate reader (molecular devices, USA).                                         RNA (RNA IPTG induced cells). All samples analyzed <br />  <br />  <br />  Table 2. Plasmids used study. <br />  <br />  <br />                                                                                                      Antisense target location* <br />  Plasmid                  Description resistance                      Purpose                    length                              Reference/Source <br />  <br />  pHN678                   IPTG inducible promoter (Ptrc),                 Antisense RNA expression n/a                                       [10] <br />                           paired-termini flanking MCS, CamR               vector <br />  pHN678-murA              pHN678 derivative, IPTG-inducible promoter      Inducible expression    254 +76 murA (130 nt.)            [30] <br />                           (Ptrc), CamR, murA antisense insert             murA antisense <br />  pHN678-yidC              pHN678 derivative, IPTG-inducible promoter      Inducible expression yidC280 +54 yidC (134 nt)            This work <br />                           (Ptrc), CamR, yidC antisense insert             antisense <br />  <br />  *antisense target locations specified relative start codon. <br />  doi:10.1371/journal.pone.0057370.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                      3                                March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                               yidC Promising Antibacterial Target <br />  <br />  <br />  <br />  <br /> Figure 1. Effect yidC antisense expression growth E. coli. E. coli DH5a harbouring pHN678 (negative control antisense expression), <br /> pHN678-yidC pHN678-murA (positive control antisense expression) cultured LB broth supplemented chloramphenicol (30 mg/ml). <br /> The overnight cultures diluted fold fresh LB broth supplemented chloramphenicol (30 mg/ml) 5 ml spotted LB <br /> chloramphenicol plates (upper panel) IPTG (lower panel). There change observed phenotype E. coli DH5a <br /> harbouring vector (pHN678) insert induction IPTG 14 hrs incubation 37uC. However, growth E. coli DH5a cells <br /> expressing antisense murA yidC showed growth suppression induction. <br /> doi:10.1371/journal.pone.0057370.g001 <br />  <br /> in triplicate. Quantitation target yidC mRNA normalized               piperine, a-terpineol, c-terpinene, eugenol, carvacrol, aloe emo- <br /> against 16S rRNA mRNA calculated relative untreated               din, gallic acid, rutin apigenin) screened sensitization <br /> sample. Mean values error bars, representing standard                   YidC depleted strain E. coli compared control cells not <br /> deviation experimental replicates, plotted           expressing yidC antisense. The FICI values synergy were <br /> IPTG doses.                                                                  calculated described previously [16]. Synergy defined FICI <br />                                                                              values #0.5 [33]. <br /> Minimum Inhibitory Concentration (MIC) Determination <br />    All antibiotics plant natural products including           Results <br /> berberine, piperine, a-terpineol, c-terpinene, eugenol, carvacrol, <br /> aloe emodin, rutin, apigenin gallic acid purchased                  Transcript Target Site Selection Cloning Antisense <br /> from Sigma-Aldrich, USA. Each added series                       RNA Expression Constructs <br /> dilutions MH broth 96-well plates triplicates make                The fragment yidC sequence chosen antisense <br /> final volume 100 ml. Then, 100 ml MH broth containing                  expression consisted nucleotides region 280 to <br /> approximately 105 cfu/ml E. coli cells prepared growth              +54 (134 nt (nucleotides)) sequence produce <br /> inhibition experiments added dilutions. MIC                 minimum secondary structures compared short <br /> deduced lowest concentration antimicrobial agent                sequences flanking yidC start codon. The yidC murA <br /> at growth tested E. coli strain inhibited               antisense constructs confirmed sequencing the <br /> completely 12 hours incubation 37uC.                             plasmids isolated transformants BLAST analysis of <br />                                                                              obtained sequences. <br /> Determining Sensitization Fractional Inhibitory <br /> Concentration Indices (FICIs) Synergy Studies                            The Growth Inhibitory Effect yidC Antisense <br />    Cell-based assays MH broth described growth                     Expression Bacterial Cells <br /> inhibition experiments performed panel known                  The growth inhibitory effect yidC antisense expression on <br /> antibiotics different chemical classes plant natural products         bacterial cells examined assessing cell growth (Figure 1 <br /> with antibacterial activity obtain fold sensitization values [the         2). Figure 1 shows addition IPTG clear <br /> ratio IC50 value (the concentration cell                inhibitory effect growth yidC murA antisense <br /> growth inhibited 50% compared control) non-induced                  expressing E. coli DH5a cells silencing genes essential <br /> condition induced condition] [32]. Various antibiotics           growth. However, E. coli DH5a harbouring pHN678 <br /> belonging different classes based mode action <br />                                                                              antisense insert showed similar growth on <br /> namely cell wall synthesis inhibition (vancomycin phospho- <br />                                                                              uninduced induced plate. The growth curves for <br /> mycin), protein synthesis inhibition (kanamycin hygromycin), <br />                                                                              different levels yidC murA asRNA expression are <br /> nucleic acid synthesis inhibition (mitomycin novobiocin), cell <br />                                                                              presented figure 2 shows silencing yidC and <br /> membrane disruption (polymyxin B colistin), metabolic <br />                                                                              murA lowered final OD600 E. coli cells. <br /> antagonism (trimethoprim) fatty acid synthesis inhibition <br /> (triclosan) plant natural products (berberine, <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               4                            March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                                yidC Promising Antibacterial Target <br />  <br />  <br />  <br />  <br /> Figure 2. Effect IPTG concentration growth E. coli DH5a harbouring yidC antisense expressing vector (pHN678-yidC). Graph <br /> on left represents growth E. coli DH5a harbouring yidC antisense expressing vector (pHN678-yidC) graph right represents growth E. coli <br /> DH5a harbouring murA antisense expressing vector (pHN678-murA) used positive control. Antisense essential genes induced IPTG at <br /> different concentrations (40, 80, 160 320 mM). Addition IPTG clear inhibitory effect growth E. coli DH5a cells harbouring <br /> antisense inserts increased increasing IPTG concentration. <br /> doi:10.1371/journal.pone.0057370.g002 <br />  <br />  <br /> Effect yidC Antisense Expression Corresponding                         Sensitization E. coli Expressing yidC Antisense Plant <br /> Gene Transcripts                                                             Derived Antibacterial Natural Products <br />    The reduction growth bacterial cells yidC antisense               E. coli growth monitored presence absence <br /> expression caused decay specific transcripts             plant natural products (berberine, piperine, a-terpineol, c- <br /> gene yidC antisense mechanism subsequent reduced                 terpinene, eugenol, carvacrol, aloe emodin, rutin, apigenin and <br /> expression protein YidC. We used relative qPCR compare                 gallic acid) yidC silencing asRNA expression. <br /> mRNA expression uninduced induced E. coli cells                       Silencing yidC caused bacteria sensitive to <br /> harbouring pHN678-yidC assumed proxy protein                  eugenol carvacrol (Figure 4 5). Upon yidC asRNA <br /> measure bacteria [34]. We observed yidC mRNA                         expression, bacterial growth completely inhibited lower <br /> expression diminished progressively increase IPTG <br /> concentration (figure 3) used induction. However, the <br /> expression cells treated 80 160 mM <br /> IPTG. The level unrelated, constitutively expressed 16S <br /> rRNA gene transcript, assayed control RNA, did change <br /> under culture conditions revealed CT values <br /> (data shown). <br />  <br /> Susceptibility Testing Various Antibiotics known <br /> Modes Action <br />    The sensitization yidC asRNA expressing E. coli cells towards <br /> various antibiotics known mode action studied cell- <br /> based assays. We used murA asRNA expressing clone positive <br /> control susceptibility testing phosphomycin well <br /> established inhibitor MurA. Induction IPTG rendered E. <br /> coli cells expressing murA antisense susceptible phospho- <br /> mycin (apparent MIC decreased 62.5 ng/ml in <br />                                                                              Figure 3. Effect yidC antisense expression corresponding <br /> case control cells harbouring pHN678 7.8 ng/ml in <br />                                                                              gene transcript. Relative Real-time quantitative PCR (qPCR) was <br /> case cells harbouring pHN678-murA antisense construct). Our               carried compare yidC mRNA expression uninduced and <br /> results indicate 70 mM IPTG, murA asRNA expressing               induced pHN678-yidC harbouring E. coli DH5a cells. The CT values were <br /> clone exhibits 8 fold increase (IC50 0 mM divided IC50              averaged triplicate samples expression ratio was <br /> 70 mM IPTG) sensitivity phosphomycin (Figure 4). The                   calculated. Comparison expression ratio yidC gene its <br /> optimized cell-based assay performed serial dilutions            antisense induction E. coli different IPTG concentrations showed <br />                                                                              yidC mRNA expression decreased increasing IPTG concentra- <br /> of antibiotics. Results showed yidC asRNA                tion. However, expression cells treated 80 <br /> clone did comparable sensitization                    160 mM IPTG. <br /> antibiotics tested (Figure 4).                                               doi:10.1371/journal.pone.0057370.g003 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               5                             March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                                 yidC Promising Antibacterial Target <br />  <br />  <br />  <br />  <br /> Figure 4. Fold sensitization yidC antisense expressing E. coli DH5a cells various antibacterial agents. Various antibiotics belonging <br /> to different classes based mode action cell wall synthesis inhibition (vancomycin phosphomycin), protein synthesis <br /> inhibition (kanamycin hygromycin), nucleic acid synthesis inhibition (mitomycin novobiocin), cell membrane disruption (polymyxin B and <br /> colistin), metabolic antagonism (trimethoprim) fatty acid synthesis inhibition (triclosan) plant natural products membrane <br /> bound ATPase activity inhibiting essential oils eugenol carvacrol, cell division protein FtsZ inhibiting alkaloid berberine other <br /> plant natural products named materials methods (data shown) screened sensitization YidC depleted strain E. coli DH5a as <br /> compared control cells (E. coli DH5a harbouring vector expressing antisense murA essential gene). Fold sensitization represents the <br /> change antibiotic IC50 values antisense induction. E. coli DH5a cells expressing yidC antisense showed considerable sensitization eugenol <br /> (16 fold) carvacrol (8 fold) compared control cells. However natural products antibiotics known mode action did not <br /> result fold sensitization equal greater murA antisense expressing E. coli cells phosphomycin (positive control). <br /> doi:10.1371/journal.pone.0057370.g004 <br />  <br /> concentration essential oils eugenol carvacrol                  expressing asRNA. Moreover strain sensitive other <br /> compared cells expressing yidC asRNA. Declined YidC                    antibiotics [36]. Similarly E. coli, demonstrated cells <br /> expression resulted reduction MIC eugenol                expressing antisense fabI sensitized FabI inhibitor <br /> 2.5 mg/ml IPTG 0.156 mg/ml 70 mM IPTG.                         triclosan trimethoprim (inhibitor dihydrofolate reduc- <br /> Similarly MIC carvacrol reduced 0.3 mg/ml                 tase) rifampicin (inhibitor RNA polymerase) [10]. Due to <br /> 0.03 mg/ml induction 70 mM IPTG. However,                       extreme sensitivity asRNA expressing strain specific <br /> control strains harbouring vector pHN678                   inhibitors, RNA-silencing based screens enable detection of <br /> insert pHN678-murA antisense construct affected. In               compounds (such natural product secondary metabolites which <br /> addition, calculated FIC indices revealed synergy yidC            produced small quantities [37]) gone <br /> asRNA expression eugenol (FICI = 0.13) carvacrol                      undetected traditional cell based antibacterial screening. <br /> treatment (FICI = 0.2) suggested synergy existed                  The applicability approach discover novel antibacterial <br /> between eugenol/carvacrol addition silencing control               agents proved S. aureus natural product <br /> gene murA (FICI = 1).                                                         inhibitors enzymes fatty acid synthesis pathway like <br />                                                                               platensimycin [15], platencin [14], phomallenic acids [9] etc. <br /> Discussion                                                                    discovered using approach. Thus, target specific <br />                                                                               cell screening based asRNA expression promising <br />    Rapid evolution drug resistance mechanisms bacterial              approach antibacterial drug discovery. Although asRNA based <br /> pathogens unfortunately exceeded capacity develop new              gene silencing approach used elaborate the <br /> antibiotics. Novel antibacterial drug screening strategies based           mechanism action berberine E. coli [16], published reports <br /> gene silencing asRNA expression competently make                       describing application approach screening target <br /> available drugs battle emerging infections. A strain expressing            specific antibacterial agents Gram-negative bacteria lacking. <br /> asRNA directed specific gene transcript                      YidC, bacterial membrane integral protein immense <br /> reduced pool cognate protein sensitized               potential target antibacterial therapy its <br /> inhibitors protein products shown previous               conservation pathogens essentiality host survival [22]. <br /> studies [16]. Antisense based RNA silencing highly specific            The yidC gene encoding E. coli membrane insertase highly <br /> the specificity exemplified various methods like                  conserved Gram-negative pathogens. Sequence <br /> western blotting [35], complementation target gene, qRT-               alignment E. coli yidC BLAST showed 99% gene similarity in <br /> PCR [30] etc. It demonstrated S. aureus strain                  Shigella flexneri Shigella dysenteriae, 91% gene similarity in <br /> expressing asRNA directed fabF sensitized 12-fold              Citrobacter koseri, 90% gene similarity Salmonella bongori, Salmonella <br /> cerulenin, FabF inhibitor, compared uninduced cells               enterica subsp. enterica serovar Typhi Salmonella enterica subsp. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                6                             March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                                 yidC Promising Antibacterial Target <br />  <br />  <br />  <br />  <br /> Figure 5. Effect antisense RNA mediated yidC silencing E. coli susceptibility eugenol carvacrol. Growth curves E. coli <br /> DH5a carrying control (pHN678 pHN678-murA) pHN678-yidC plasmids. (A) In absence antibacterial agent. (B) In presence sub <br /> lethal concentration eugenol. (C) In presence sub lethal concentration carvacrol. The growth curve experiment carried described in <br /> materials methods. Antisense expression induced 70 mM IPTG. Data means 6 standard deviations experiment performed in <br /> triplicate. <br /> doi:10.1371/journal.pone.0057370.g005 <br />  <br /> enterica serovar Typhimurium 89% gene similarity                       21CFR172.515; Eugenol: 21CFR582.60). Eugenol major <br /> Enterobacter cloacae. No inhibitors YidC reported             component (approximately 85%) clove oil carvacrol one <br /> far. Therefore, set target yidC E. coli asRNA                 major components oregano thyme oils [38]. These <br /> expression confirm effect downregulation growth           molecules known disintegrate bacterial membrane and <br /> E. coli. Our results expression antisense 280          inhibit membrane bound ATPase activity E. coli cells [28]. It <br /> +54 nt yidC results diseased phenotype host E. coli cells.           shown interactions mRNA- protein- <br /> Further, growth inhibition effect yidC antisense                   level inhibitors directed genetic target be <br /> expression decay yidC mRNA transcript                  synergistic [39]. Therefore, membrane bound ATPase <br /> evidently results decrease YidC synthesis                 inhibitory activity eugenol carvacrol, checked for <br /> diseased phenotype host cells.                                             synergy YidC depletion Foc (membrane subunit the <br />    The plant-derived essential oils, eugenol carvacrol             F1Fo ATPase) substrate YidC owing YidC <br /> long history use flavouring agents food preservatives           depletion causes adverse effect assembly functional <br /> to antimicrobial activity. These compounds                    membrane bound ATPases [23]. Additionally, comparison, <br /> classified GRAS (generally recognized safe) Food             plant natural products including alkaloids like berberine and <br /> Drug Administration (FDA), agency United States                        piperine; essential oil components like a-terpineol c-terpinene; <br /> Department Health Human Services (Carvacrol:                           anthraquinone like aloe emodin; polyphenol like gallic acid and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                7                             March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                                                    yidC Promising Antibacterial Target <br />  <br />  <br /> flavanoids like rutin apigenin screened sensitization.                        cell assays screen target specific inhibitors. However, over <br /> Here report novel important observation                                      expression yidC inhibitory E. coli growth (data <br /> downregulation yidC causes E. coli susceptible                           shown). Hence antisense based method suitable screening <br /> to inhibition eugenol carvacrol. In control assay,                         specific inhibitors proteins overexpression leads <br /> silencing essential gene, murA, sensitization eugenol                        cell death. The RNA silencing approach applied study <br /> or carvacrol observed sensitization phosphomycin                          offers means speedy identification putative YidC inhibitors. <br /> was apparent. MurA (UDP-N-acetylglucosamine enolpyruvyl                                    Antimicrobial synergies mRNA- protein-level inhib- <br /> transferase) established target phosphomycin E.                          itors described previously [39]. Synthetic RNA silencers <br /> coli [40]. On basis derived FICI values results                         emerged novel class antibacterials [42]. Similar <br /> from growth curves, clear synergy exists yidC                           antisense oligonucleotides directed yidC mRNA be <br /> silencing essential oils eugenol carvacrol treatment.                          designed used combination essential oils eugenol and <br /> We hypothesize synergetic effect reason                        carvacrol antibacterial therapy. Our studies taken together <br /> YidC depletion results decline functional membrane bound                             recent research groups [16,30] establish the <br /> ATPases, activity inhibited essential oils                      usefulness antisense RNA technology E. coli explore mode <br /> eugenol carvacrol. Our study shows yidC depleted cells                         action antibacterial agents. Using technology, have <br /> not comparable sensitization antibiotics                              shown time yidC therapeutic target of <br /> different modes action indicating drugs targeting YidC                         antibacterial essential oils eugenol carvacrol E. coli. <br /> unlikely susceptible resistance mechanisms bacteria <br /> use defeat current antibiotic classes. <br />                                                                                            Acknowledgments <br />    A complementary gene dosage based approach identifying <br /> drug-target interactions multicopy suppression profiling. In                       We thank Dr. Liam Good Department Pathology Infectious <br /> approach, specific target protein expressed                         Diseases, Royal Veterinary College, London, UK providing the <br /> exceeds quantity antibacterial molecules leading                                 plasmid pHN678 V Rajesh Iyer Jitendra Sahoo for <br /> suppression growth inhibitory phenotype. This approach                              technical support. <br /> been successfully applied E. coli determine mode action of <br /> an antibacterial small molecule identification                               Author Contributions <br /> established targets various known antibiotics [41]. The                            Conceived designed experiments: RP NKN SS SDP. Performed <br /> fact multicopy suppression measurement target protein                            experiments: SDP RS. Analyzed data: SDP RP NKN. Contributed <br /> overexpression used determine target antibacterial                         reagents/materials/analysis tools: SDP RP. Wrote paper: RP SDP <br /> molecule indicate similar concept used design                             NKN. <br />  <br /> References <br />  1. Davies J (2007) Microbes word. A drastic re-evaluation                15. Wang J, Soisson SM, Young K, Shoop W, Kodali S, et al. (2006) Platensimycin <br />     antimicrobial treatment needed overcome threat antibiotic-resistant           selective FabF inhibitor potent antibiotic properties. Nature 441: 358&#226;&#8364;&#8220; <br />     bacteria. EMBO reports 8: 616.                                                             361. <br />  2. Miesel L, Greene J, Black TA (2003) Genetic strategies antibacterial drug          16. Boberek JM, Stach J, Good L (2010) Genetic evidence inhibition bacterial <br />     discovery. Nature Reviews Genetics 4: 442&#226;&#8364;&#8220;456.                                             division protein FtsZ berberine. PloS 5: e13745. <br />  3. Roemer T, Davies J, Giaever G, Nislow C (2011) Bugs, drugs chemical                17. Projan SJ (2002) New (and new) antibacterial targets-from and <br />     genomics. Nature Chemical Biology 8: 46&#226;&#8364;&#8220;56.                                                novel drugs come? Current Opinion Pharmacology 2: 513. <br />  4. Spellberg B, Powers JH, Brass EP, Miller LG, Edwards Jr JE (2004) Trends            18. Chan PF, Macarron R, Payne DJ, Zalacain M, Holmes DJ (2002) Novel <br />     antimicrobial drug development: implications future. Clinical Infectious           antibacterials: genomics approach drug discovery. Current Drug Targets- <br />     Diseases 38: 1279&#226;&#8364;&#8220;1286.                                                                    Infectious Disorders 2: 291&#226;&#8364;&#8220;308. <br />  5. Vergidis PI, Falagas ME (2008) Multidrug-resistant Gram-negative bacterial             19. Payne DJ, Gwynn MN, Holmes DJ, Pompliano DL (2006) Drugs bad bugs: <br />     infections: emerging threat potential novel treatment options. Current             confronting challenges antibacterial discovery. Nature Reviews Drug <br />     opinion investigational drugs (London, England: 2000) 9: 176.                           Discovery 6: 29&#226;&#8364;&#8220;40. <br />  6. Xu HH, Trawick JD, Haselbeck RJ, Forsyth R, Yamamoto RT, et al. (2010)                 20. Scotti PA, Urbanus ML, Brunner J, Gier JWL, von Heijne G, et al. (2000) <br />     Staphylococcus aureus TargetArray: comprehensive differential essential gene               YidC, Escherichia coli homologue mitochondrial Oxa1p, component <br />     expression mechanistic tool profile antibacterials. Antimicrobial agents           Sec translocase. The EMBO journal 19: 542&#226;&#8364;&#8220;549. <br />     chemotherapy 54: 3659&#226;&#8364;&#8220;3670.                                                        21. Serek J, Bauer-Manz G, Struhalla G, Van Den Berg L, Kiefer D, et al. (2004) <br />  7. Pathania R, Brown ED (2008) Small lethal: searching new antibacterial              Escherichia coli YidC membrane insertase Sec-independent proteins. <br />     compounds novel modes action. Biochemistry Cell Biology 86: 111&#226;&#8364;&#8220;               The EMBO journal 23: 294&#226;&#8364;&#8220;301. <br />     115.                                                                                   22. Samuelson JC, Chen M, Jiang F, Mo&#194;&#168;ller I, Wiedmann M, et al. (2000) YidC <br />  8. Moir DT, Opperman TJ, Butler MM, Bowlin TL (2012) New classes                           mediates membrane protein insertion bacteria. Nature 406: 637&#226;&#8364;&#8220;641. <br />     antibiotics. Current Opinion Pharmacology.                                          23. van der Laan M, Urbanus M, Ten Hagen-Jongman C, Nouwen N, Oudega B, <br />  9. Young K, Jayasuriya H, Ondeyka JG, Herath K, Zhang C, et al. (2006)                        et al. (2003) A conserved function YidC biogenesis respiratory chain <br />     Discovery FabH/FabF inhibitors natural products. Antimicrobial agents              complexes. Proceedings National Academy Sciences 100: 5801. <br />     chemotherapy 50: 519&#226;&#8364;&#8220;526.                                                          24. Hasona A, Crowley PJ, Levesque CM, Mair RW, Cvitkovitch DG, et al. (2005) <br /> 10. Nakashima N, Tamura T, Good L (2006) Paired termini stabilize antisense                    Streptococcal viability diminished stress tolerance mutants lacking the <br />     RNAs enhance conditional gene silencing Escherichia coli. Nucleic acids             signal recognition particle pathway YidC2. Proceedings National <br />     research 34: e138&#226;&#8364;&#8220;e138.                                                                    Academy Sciences United States America 102: 17466&#226;&#8364;&#8220;17471. <br /> 11. Masse&#194;&#180; E, Escorcia FE, Gottesman S (2003) Coupled degradation small               25. Zhang YJ, Tian HF, Wen JF (2009) The evolution YidC/Oxa/Alb3 family in <br />     regulatory RNA mRNA targets Escherichia coli. Genes &amp;                           domains life: phylogenomic analysis. BMC evolutionary biology 9: <br />     development 17: 2374&#226;&#8364;&#8220;2383.                                                                 137. <br /> 12. Dryselius R, Aswasti SK, Rajarao GK, Nielsen PE, Good L (2003) The                     26. Prabuseenivasan S, Jayakumar M, Ignacimuthu S (2006) In vitro antibacterial <br />     translation start codon region sensitive antisense PNA inhibition                 activity plant essential oils. BMC Complementary Alternative <br />     Escherichia coli. Oligonucleotides 13: 427&#226;&#8364;&#8220;433.                                            Medicine 6: 39. <br /> 13. Knudsen H, Nielsen PE (1996) Antisense properties duplex-and triplex-               27. Ouattara B, Simard RE, Holley RA, Piette GJP, Be&#194;&#180;gin A (1997) Antibacterial <br />     forming PNAs. Nucleic acids research 24: 494&#226;&#8364;&#8220;500.                                          activity selected fatty acids essential oils meat spoilage <br /> 14. Wang J, Kodali S, Lee SH, Galgoci A, Painter R, et al. (2007) Discovery                 organisms. International journal food microbiology 37: 155&#226;&#8364;&#8220;162. <br />     platencin, dual FabF FabH inhibitor vivo antibiotic properties.          28. Gill A, Holley R (2006) Inhibition membrane bound ATPases Escherichia coli <br />     Proceedings National Academy Sciences 104: 7612&#226;&#8364;&#8220;7616.                            Listeria monocytogenes plant oil aromatics. International journal food <br />                                                                                                microbiology 111: 170&#226;&#8364;&#8220;174. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                             8                                  March 2013 | Volume 8 | Issue 3 | e57370 <br />                                                                                                                                     yidC Promising Antibacterial Target <br />  <br />  <br /> 29. Donald RGK, Skwish S, Forsyth R, Anderson JW, Zhong T, et al. (2009) A                   36. Forsyth R, Haselbeck RJ, Ohlsen KL, Yamamoto RT, Xu H, et al. (2002) A <br />     Staphylococcus aureus Fitness Test Platform Mechanism-Based Profiling                 genome-wide strategy identification essential genes Staphylococcus <br />     Antibacterial Compounds. Chemistry &amp; biology 16: 826&#226;&#8364;&#8220;836.                                    aureus. Molecular microbiology 43: 1387&#226;&#8364;&#8220;1400. <br /> 30. Goh S, Boberek JM, Nakashima N, Stach J, Good L (2009) Concurrent growth                 37. Olano C, Lombo&#194;&#180; F, Me&#194;&#180;ndez C, Salas JA (2008) Improving production of <br />     rate transcript analyses reveal essential gene stringency Escherichia coli.           bioactive secondary metabolites actinomycetes metabolic engineering. <br />     PloS 4: e6061.                                                                           Metabolic engineering 10: 281&#226;&#8364;&#8220;292. <br /> 31. Kim BC, Gu MB (2006) Expression analysis stress-specific responsive genes          38. Burt S (2004) Essential oils: antibacterial properties potential <br />     two-stage continuous cultures Escherichia coli using cDNA microarray real-            applications foods&#226;&#8364;&#8220;a review. International journal food microbiology 94: <br />     time RT-PCR analysis. Enzyme microbial technology 39: 440&#226;&#8364;&#8220;446.                           223&#226;&#8364;&#8220;253. <br /> 32. Meng J, Kanzaki G, Meas D, Lam CK, Crummer H, et al. (2012) A genome-                    39. Dryselius R, Nekhotiaeva N, Good L (2005) Antimicrobial synergy between <br />     wide inducible phenotypic screen identifies antisense RNA constructs silencing               mRNA-and protein-level inhibitors. Journal Antimicrobial Chemotherapy 56: <br />     Escherichia coli essential genes. FEMS Microbiology Letters.                                 97&#226;&#8364;&#8220;103. <br /> 33. Odds F (2003) Synergy, antagonism, chequerboard puts                40. Kahan FM, Kahan JS, Cassidy PJ, Kropp H (2006) The mechanism action of <br />     them. Journal Antimicrobial Chemotherapy 52: 1&#226;&#8364;&#8220;1.                                         fosfomycin (phosphonomycin). Annals New York Academy Sciences <br /> 34. Dryselius R, Nikravesh A, Kulyte&#194;&#180; A, Goh S, Good L (2006) Variable                           235: 364&#226;&#8364;&#8220;386. <br />     coordination cotranscribed genes Escherichia coli following antisense              41. Pathania R, Zlitni S, Barker C, Das R, Gerritsma DA, et al. (2009) Chemical <br />     repression. BMC microbiology 6: 97.                                                          genomics Escherichia coli identifies inhibitor bacterial lipoprotein <br /> 35. Ji Y, Yin D, Fox B, Holmes DJ, Payne D, et al. (2006) Validation antibacterial            targeting. Nature Chemical Biology 5: 849&#226;&#8364;&#8220;856. <br />     mechanism action using regulated antisense RNA expression Staphylo-             <br /> </body></html>