<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> The single-stranded DNA-binding protein of Deinococcus <br /> radiodurans <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: Deinococcus radiodurans R1 is one of the most radiation-resistant organisms known <br />                  and is able <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> repair an unusually large amount of DNA damage without <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>induced mutation</span>. Single- <br />                  stranded DNA-binding (SSB) protein is an essential protein in all organisms and is involved in DNA <br />                  replication, recombination and repair. The published genomic sequence <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> Deinococcus <br />                  radiodurans includes a putative single-stranded DNA-binding protein gene (ssb; DR0100) requiring <br />                  a translational frameshift for synthesis of a complete SSB protein. The apparently tripartite gene <br />                  has inspired considerable speculation in the literature about potentially novel frameshifting or RNA <br />                  editing mechanisms. Immediately upstream of the ssb gene is another gene (DR0099) given an ssb- <br />                  like annotation, but left unexplored. <br />                  Results: A segment of the Deinococcus radiodurans strain R1 genome encompassing the ssb gene <br />                  has been re-sequenced, and two errors involving omitted guanine nucleotides have been <br />                  documented. The corrected sequence incorporates both of the open reading frames designated <br />                  DR0099 and DR0100 into one contiguous ssb open reading frame (ORF). The corrected gene <br />                  requires no translational frameshifts and contains two predicted oligonucleotide/oligosaccharide- <br />                  binding (OB) folds. The protein has been purified and its sequence is closely related to the Thermus <br />                  thermophilus and Thermus aquaticus SSB proteins. Like the Thermus SSB proteins, the SSBDr functions <br />                  as a homodimer. The Deinococcus radiodurans SSB homodimer stimulates Deinococcus radiodurans <br />                  RecA protein and Escherichia coli RecA protein-promoted DNA three-strand exchange reactions <br />                  with at least the same efficiency as the Escherichia coli SSB homotetramer. <br />                  Conclusions: The correct Deinococcus radiodurans ssb gene is a contiguous open reading frame <br />                  that codes for the largest bacterial SSB monomer identified to date. The Deinococcus radiodurans <br />                  SSB protein includes two OB folds per monomer and functions as a homodimer. The Deinococcus <br />                  radiodurans SSB protein efficiently stimulates Deinococcus radiodurans RecA and also Escherichia coli <br />                  RecA protein-promoted DNA strand exchange reactions. The identification and purification of <br />                  Deinococcus radiodurans SSB protein not only allows for greater understanding of the SSB protein <br />                  family but provides an essential yet previously missing player in the current efforts to understand <br />                  the extraordinary DNA repair capacity of Deinococcus radiodurans. <br />  <br />  <br />  <br />  <br />                                                                                                                                              Page 1 of 12 <br />                                                                                                                       (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                     http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br />  <br /> Background                                                    was derived from the Deinococcus radiodurans R1 type <br /> Deinococcus radiodurans R1, a tetrad-forming gram positive    strain obtained directly from the ATCC (ATCC 13939). <br /> soil bacterium, is among the most radiation resistant         The ssb gene was also independently cloned into two dif- <br /> organisms known [1]. The D37 &#206;&#179; irradiation dose, (the         ferent cloning vectors and re-sequenced. In each case, the <br /> dose at which an irradiated population of cells is reduced    sequence revealed two additional guanine nucleotides <br /> to 37%) for D. radiodurans is approximately 6,000 Gy.         that are not present in the sequence deposited in the data- <br /> This dose is 200 times that required to reduce Escherichia    <span id='am-317' about='xsp:base' typeof='owl:Thing'>base</span> of the National Center for Biotechnology Informa- <br /> coli <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span> to the same extent [1]. A 6,000 Gy dose of      tion (NCBI) (Fig. 1) [4]. (The original submitted sequence <br /> radiation introduces approximately 300 DNA double             has accession #AE000513. The current sequence has <br /> strand breaks, greater than 3,000 single strand breaks, and   accession #NC_001263 and is identical to the original). <br /> more than 1,000 sites of base damage per D. radiodurans       With the corrections reported here, a contiguous ORF that <br /> haploid genome ([1] and references therein). D. radio-        encodes a complete ssb protein is revealed. A recognizable <br /> durans is also able to sustain growth without induced         D. radiodurans ribosomal binding site (AAGGAG) exists at <br /> <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> in the presence of considerable levels of ambi-      an optimal 8 nucleotides upstream of the initiation codon <br /> ent radiation (6 krads/h) [2]. Presumably, this organism      (Fig. 1) [8-10]. The corrected ssb sequence has been depos- <br /> possesses a robust DNA damage repair system. Single-          ited with GenBank, accession number AY293617. The <br /> stranded DNA-binding protein (SSB) is an essential pro-       predicted D. radiodurans SSB protein is the largest bacterial <br /> tein in all known organisms and is required for DNA rep-      SSB polypeptide identified to date (301 amino acids, <br /> lication, recombination and repair [3].                       including the initiating methionine), exceeding in <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br />                                                               the T. thermophilus VK1 SSB by 35 amino acid residues. <br /> The genome of D. radiodurans consists of 2 chromosomes, <br /> a megaplasmid, and a plasmid. Each of these genomic ele-      The presence of a contiguous open reading frame encom- <br /> ments is present at 4&#226;&#8364;&#8220;10 copies per cell [4]. The genome      passing the regions designated DR0099 and DR0100 was <br /> has been sequenced [4]. In this sequence, the SSB protein     confirmed by PCR amplification of this region from <br /> appeared to be encoded by a tripartite ssb gene, requiring    genomic DNA, and cloning it into an E. coli expression <br /> two translational frameshifts or some other mechanism to      vector. Excellent inducible expression of a protein of the <br /> allow expression of a full length functional SSB protein      predicted size was obtained from the pEAW328 vector <br /> [2,4-6]. The apparent need for frameshifting in this gene     (see Methods) (Fig. 2). This protein was purified and con- <br /> has led to considerable speculation about both the pres-      firmed to be D. radiodurans SSB protein by N-terminal <br /> ence of novel frameshifting or RNA editing mechanisms         sequencing of the first nine amino acids (ARGMNHVYL). <br /> in D. radiodurans and their possible roles in normal          As expected, the N-terminal methionine residue was <br /> metabolism [2,4-6]. There has even been some uncer-           absent in the D. radiodurans SSB protein purified from E. <br /> tainty as to whether D. radiodurans possesses a functional    coli. The calculated molecular weight of the purified D. <br /> ssb gene. We now report that the ssb sequences as origi-      radiodurans SSB protein monomer (300 amino acids with- <br /> nally published (genes DR0099 and DR0100) include             out the initiating methionine) is 32,591. We estimate that <br /> two errors that render the gene interpretation opaque. The    the D. radiodurans SSB protein purified as described here <br /> correct sequence reveals a longer and completely contigu-     is more than 99% homogeneous. Additionally, the puri- <br /> ous ORF for ssb, encoding an SSB protein closely related to   fied D. radiodurans SSB protein was free of detectable DNA <br /> the recently characterized Thermus thermophilus and Ther-     endo- and exonucleases. <br /> mus aquaticus SSB proteins [7]. The D. radiodurans SSB pro- <br /> tein has been over-expressed and purified, and like the       D. radiodurans SSB protein is closely related to the T. <br /> Thermus SSB proteins, has been shown to form a                thermophilus and T. aquaticus SSB proteins <br /> homodimer in solution. The D. radiodurans SSB                 The protein sequence predicted for D. radiodurans SSB <br /> homodimer stimulates both the D. radiodurans RecA pro-        protein shares 43% identity and 58% similarity with the <br /> tein and E. coli RecA protein-promoted DNA three-strand       T. thermophilus HB-8 SSB protein, 43% identity and 58% <br /> exchange reactions at a level comparable to or greater than   similarity with the T. thermophilus VK-1 SSB protein and <br /> the stimulation of these reactions by the E. coli SSB         44% identity and 61% similarity with the T. aquaticus YT- <br /> homotetramer.                                                 1 SSB protein (Fig 3A and 3B). The N-terminal segment of <br />                                                               the D. radiodurans SSB protein shares 38% identity and <br /> Results and Discussion                                        49% similarity with the E. coli SSB protein. The C-terminal <br /> The D. radiodurans ssb gene encodes a contiguous ORF          segment of the D. radiodurans SSB protein shares 39% <br /> The region encompassing the ssb gene, which is flanked by     identity and 64% similarity with the E. coli SSB protein. <br /> DR0098, the ribosomal protein S6 gene and DR0101, the <br /> ribosomal protein S18 gene, was PCR amplified from            Virtually all bacterial SSB proteins identified to date con- <br /> genomic DNA and sequenced (Fig. 1). The genomic DNA           tain only one oligonucleotide/oligosaccharide-binding <br />  <br />  <br />                                                                                                                   Page 2 of 12 <br />                                                                                             (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                          http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br />  <br />                                                                                                          M   L   L   S   S   E   P <br />     up/DR0099 -81 cgtcctggtggtcaaggaccgcccggagtggaagaccaagaaggcctgagccttttatgtcattgacataattgactctgcTTGTTACTATCTAGTGAAC 19 <br />   Correct ssb up -168 cgtcctggtggtcaaggaccgcccggagtggaagaccaagaaggcctgagccttttatgtcattgacataattgactctgcttgttactatctagtgaac -69 <br />  <br />                          A R A V R Q Q H R T L V I L P A T Q G E F V M A R G M N H V Y L I <br />        DR0099       20 CCGCAAGGGCCGTCCGCCAGCAACATCGAACTCTAGTTATTTTGCCAGCTACCCAAGGAGAATTTGTTATGGCCCGAGGCATGAACCACGTCTACCTGAT 119 <br />      Correct ssb   -68 ccgcaagggccgtccgccagcaacatcgaactctagttattttgccagctacccaaggagaatttgttATGGCCCGAGGCATGAACCACGTCTACCTGAT 32 <br />                                                                                RBS         M A R G M N H V Y L I <br />                         G A L A R D P E L R Y T G N G M A V F E A T V A G E D R V I G N D <br />        DR0099      120 CGGCGCACTCGCCCGTGATCCCGAACTTCGCTACACCGGCAACGGCATGGCCGTTTTCGAAGCCACCGTCGCAGGCGAAGACCGAGTGATCGGCAATGAT 219 <br />      Correct ssb    33 CGGCGCACTCGCCCGTGATCCCGAACTTCGCTACACCGGCAACGGCATGGCCGTTTTCGAAGCCACCGTCGCAGGCGAAGACCGAGTGATCGGCAATGAT 132 <br />                         G A L A R D P E L R Y T G N G M A V F E A T V A G E D R V I G N D <br />  <br />                        G R E R N L P W Y H R V S I L G K P A E W Q A E R N L K G G D A V V <br />        DR0099      220 GGCCGTGAACGCAATCTGCCCTGGTACCACCGCGTGTCGATTCTGGGCAAGCCCGCCGAATGGCAGGCCGAGCGCAACCTCAAGGGCGGCGACGCCGTGG 319 <br />      Correct ssb   133 GGCCGTGAACGCAATCTGCCCTGGTACCACCGCGTGTCGATTCTGGGCAAGCCCGCCGAATGGCAGGCCGAGCGCAACCTCAAGGGCGGCGACGCCGTGG 232 <br />                        G R E R N L P W Y H R V S I L G K P A E W Q A E R N L K G G D A V V <br />  <br />                          V E G T L E Y R Q W E A P E A                 A N A A Q S T S K P S A W N S S A P <br />        DR0099      320 TGGTTGAAGGTACCCTGGAATACCGTCAGTGGGAAGCGCCCGAAG-CGGCAAACGCAGCGCAGTCAACGTCAAAGCCCTCCGCATGGAACAGCTCGGCAC 418 <br />      Correct ssb   233 TGGTTGAAGGTACCCTGGAATACCGTCAGTGGGAAGCGCCCGAAGGCGGCAAACGCAGCGCAGTCAACGTCAAAGCCCTCCGCATGGAACAGCTCGGCAC 332 <br />                          V E G T L E Y R Q W E A P E G G K R S A V N V K A L R M E Q L G T <br />                          S P N                                 M S G A M N E V L V L G N V T R D P E I <br />    DR0099/0100     419 CCAGCCCGAACtgattcaggacgctggcggcggcgtgcgcATGAGCGGCGCGATGAATGAAGTCCTGGTCCTCGGCAACGTGACCCGTGACCCCGAGATT 518 <br />     Correct ssb    333 CCAGCCCGAACTGATTCAGGACGCTGGCGGCGGCGTGCGCATGAGCGGCGCGATGAATGAAGTCCTGGTCCTCGGCAACGTGACCCGTGACCCCGAGATT 432 <br />                         Q P E L I Q D A G G G V R M S G A M N E V L V L G N V T R D P E I <br />  <br />                        R Y T P A G D A V L S L S I A V N E N Y Q D R Q A                           S A R K K V H Y I <br />        DR0100      519 CGCTACACCCCTGCGGGTGACGCTGTGCTGAGCCTGTCGATTGCGGTCAACGAGAACTACCAGGACCGTCAGG-CCAGCGCCAGGAAAAGGTTCACTATA 617 <br />      Correct ssb   433 CGCTACACCCCTGCGGGTGACGCTGTGCTGAGCCTGTCGATTGCGGTCAACGAGAACTACCAGGACCGTCAGGGCCAGCGCCAGGAAAAGGTTCACTATA 532 <br />                        R Y T P A G D A V L S L S I A V N E N Y Q D R Q G Q R Q E K V H Y I <br />  <br />                          D A T L W R D L A E N M K E L R K G D P V M I M G R L V N E G W T <br />        DR0100      618 TCGACGCCACGCTCTGGCGCGACCTCGCCGAGAACATGAAGGAGCTCCGTAAGGGCGACCCGGTCATGATCATGGGCAGGCTCGTCAACGAGGGCTGGAC 717 <br />      Correct ssb   533 TCGACGCCACGCTCTGGCGCGACCTCGCCGAGAACATGAAGGAGCTCCGTAAGGGCGACCCGGTCATGATCATGGGCAGGCTCGTCAACGAGGGCTGGAC 632 <br />                          D A T L W R D L A E N M K E L R K G D P V M I M G R L V N E G W T <br />                         D K D G N K R N S T R V E A T R V E A L A R G A G N A N S G Y A A <br />        DR0100      718 CGATAAAGACGGCAACAAGCGCAACAGCACAAGAGTAGAGGCGACGCGAGTCGAAGCCCTTGCCCGAGGCGCAGGCAATGCCAATTCGGGCTACGCCGCA 817 <br />      Correct ssb   633 CGATAAAGACGGCAACAAGCGCAACAGCACAAGAGTAGAGGCGACGCGAGTCGAAGCCCTTGCCCGAGGCGCAGGCAATGCCAATTCGGGCTACGCCGCA 732 <br />                         D K D G N K R N S T R V E A T R V E A L A R G A G N A N S G Y A A <br />                        A T P A A P R T Q T A S S A A R P T S G G Y Q S Q P S R A A N T G S <br />        DR0100      818 GCCACCCCTGCCGCACCTCGCACGCAGACCGCGAGCAGTGCCGCACGCCCGACGTCTGGCGGCTACCAGAGCCAGCCCAGCCGCGCGGCGAACACGGGGA 917 <br />      Correct ssb   733 GCCACCCCTGCCGCACCTCGCACGCAGACCGCGAGCAGTGCCGCACGCCCGACGTCTGGCGGCTACCAGAGCCAGCCCAGCCGCGCGGCGAACACGGGGA 832 <br />                        A T P A A P R T Q T A S S A A R P T S G G Y Q S Q P S R A A N T G S <br />                          R S G G L D I D Q G L D D F P P E E D D L P F <br />   DR0100/down      918 GCCGTTCGGGGGGCCTAGATATTGATCAAGGTCTCGACGATTTTCCGCCGGAAGAAGACGACCTGCCCTTTtaatgcaggccctgggggcctagcgcccc 1017 <br /> Correct ssb/down   833 GCCGTTCGGGGGGCCTAGATATTGATCAAGGTCTCGACGATTTTCCGCCGGAAGAAGACGACCTGCCCTTTtaatgcaggccctgggggcctagcgcccc 932 <br />                          R S G G L D I D Q G L D D F P P E E D D L P F <br />  <br />  <br />    DR0100down 1018 cggcttccggccacaccaaggacacccaacatgacccag 1056 <br /> Correct ssb down 933 cggcttccggccacaccaaggacacccaacatgacccag 971 <br />  <br />  <br /> Nucleotide <br /> Figure 1 sequence of the D. radiodurans R1 ssb gene <br /> Nucleotide sequence of the D. radiodurans R1 ssb gene. The predicted amino acid sequence is shown above the nucle- <br /> otide sequence for the DR0099 (red) and DR0100 (green)[4] and below for the corrected ssb (blue) sequences. Predicted non- <br /> coding regions are shown in lower case. The nucleotides omitted in the published sequence are shown in bold in the corrected <br /> ssb sequence and are indicated by triangles. The reading frames affected by the previous errors are shown in boxes. The trans- <br /> lational frameshift region predicted by the earlier sequence is highlighted by a gray box. The putative Ribosomal Binding Site in <br /> the corrected ssb sequence is underlined. The sequences are numbered according to the first predicted initiation codon in the <br /> sequences (TTG in DR0099 and ATG in the corrected ssb). The DR0099 and DR0100 genes have accession number <br /> NC_001263. The corrected ssb sequence, reported here, has accession number AY293617. <br />  <br />  <br />  <br />  <br /> (OB) fold per monomer and function as homotetramers                ilar to that of the T. thermophilus and T. aquaticus ssb gene <br /> [11,12]. The very recently characterized T. thermophilus           structures and is predicted to contain two OB folds. The <br /> and T. aquaticus SSB proteins have broken that <span id='am-318' about='xsp:pattern' typeof='owl:Thing'>pattern</span>,            high similarity of the D. radiodurans SSB protein sequence <br /> and contain 2 OB folds per monomer [7,11] and function             to the Thermus SSB proteins (Fig 3) suggested that the D. <br /> as homodimers [7]. In the formation of homodimers, T.              radiodurans SSB protein would also form a homodimer. <br /> thermophilus and T. aquaticus SSB proteins maintain the            This prediction was confirmed (see below). The structure <br /> bacterial trend of 4 OB folds per SSB protein oligomer.            of the D. radiodurans SSB protein is consistent with and <br /> The D. radiodurans SSB protein gene structure is quite sim-        reinforces the close phylogenetic relationship between <br />  <br />  <br />                                                                                                                          Page 3 of 12 <br />                                                                                                  (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                          http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br />                                                                    a standard molecular weight curve generated for the col- <br />  <br />  <br />  <br />  <br />                                                           Dr <br />                                                                    umn, these values correspond to 85 kDa and 101 kDa <br />  <br />  <br />  <br />  <br />                                          B <br />                                                                    respectively. The native molecular weight approximation <br />  <br />  <br />  <br />  <br />                                        SS <br />                                  uc d <br />                                                                    for D. radiodurans SSB protein is 2.6 times the molecular <br />  <br />  <br />  <br />  <br />                               ind uce <br />                               pu ed <br />                       rs <br />  <br />  <br />  <br />  <br />                                     ed <br />                   rke                                              weight of a D. radiodurans SSB monomer (32.6 kDa), or <br />                                                                    1.3 times that predicted for a D. radiodurans SSB <br />                            ind <br />  <br />  <br />                                 rifi <br />                                                                    homodimer (65 kDa). The native molecular weight <br />               ma <br />  <br />                        un <br />    kDa                                                             approximation for E. coli SSB protein under these condi- <br />                                                                    tions is 5.4 times that of the molecular weight of an E. coli <br />                                                                    SSB monomer (18.8 kDa), or 1.3 times that of an E. coli <br />                                                                    SSB homotetramer (75 kDa). E. coli SSB protein is known <br />                                                                    to form a homotetramer in solution [13,14]. Given the <br />    97 &#226;&#8364;&#8220;                                                            aberrant chromatographic behavior of the native E. coli <br />                                                                    SSB homotetramer, the apparently similar behavior of the <br />    66 &#226;&#8364;&#8220;                                                            native D. radiodurans SSB is most consistent with a <br />                                                                    homodimer. Due to the high sequence identity and simi- <br />                                                                    larity of the OB folds between E. coli SSB protein and D. <br />    45 &#226;&#8364;&#8220;                                                            radiodurans SSB protein, we anticipate that the proteins <br />                                                                    will have similar properties. In the experiment of Fig. 4, <br />                                                                    both proteins chromatograph with an apparent mass cor- <br />                                                                    responding to 1.3 times that of their calculated molecular <br />    31 &#226;&#8364;&#8220;                                                            weight, assuming the D. radiodurans SSB is a homodimer <br />                                                                    and the E. coli SSB is a homotetramer. <br />  <br />                                                                    The sedimentation equilibrium data were best described <br /> 21.5 &#226;&#8364;&#8220;                                                             as a single species. There was no evidence for multiple spe- <br />                                                                    cies and modeling attempts using associations or simple <br />                                                                    mixtures did not yield physically realistic parameters for <br /> 14.4 &#226;&#8364;&#8220;                                                             some fitting variables. The results of global fitting of the <br />                                                                    all data at absorbance less than 2 gave a molecular weight <br /> protein <br /> Figure <br /> Expression2 and purification of the D. radiodurans R1 SSB          of 63,300 &#194;&#177; 70, in excellent agreement with the value of <br /> Expression and purification of the D. radiodurans R1               65,183 for a dimer based on the sequence. In fact an <br /> SSB protein. Protein expression was obtained from the              increase in the calculated partial specific volume of only <br /> pEAW328 vector in BL21 Codon Plus (DE3) (Stratagene) E.            0.008 would yield perfect agreement. <br /> coli cells. Proteins were examined on a standard SDS-poly- <br /> acrylamide gel. Lanes are (1) molecular weight markers (Bio-       Since sedimentation equilibrium shows the protein to be <br /> Rad, Low Range), as noted with the labels at left; (2) Whole       a homogeneous population of dimers, the results from <br /> cell extract of uninduced cells; (3) whole cell extract of cells <br />                                                                    column chromatography would suggest that the shape of <br /> after induction of the D. radiodurans ssb gene; (4) purified D. <br /> radiodurans R1 SSB protein.                                        the molecule deviates from that of the globular standards. <br />  <br />                                                                    The C-terminus of SSB has special functional significance. <br />                                                                    Although the length and the sequence of the C-terminal <br />                                                                    regions (the region extending past the OB fold(s)) is vari- <br />                                                                    able across bacterial species, the last 10 amino acids at the <br /> Deinococcus and bacteria of the Thermus group of extremo-          C-terminus are highly acidic and well conserved across <br /> philes [2,4-6].                                                    bacterial species [3]. An acidic C-terminus is also seen in <br />                                                                    the D. radiodurans SSB protein C-terminal tail (Fig 3). <br /> D. radiodurans SSB protein forms a homodimer                       Studies of E. coli SSB C-terminal truncation mutants have <br /> Native molecular weight approximation by gel filtration            indicated that the last 10 amino acids of E. coli SSB protein <br /> analysis (Fig 4) and sedimentation equilibrium experi-             is essential for <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> and is probably involved in <br /> ments confirmed our prediction that D. radiodurans SSB             protein-protein interactions [15]. This may be of func- <br /> protein exists as a homodimer. In the gel filtration analy-        tional significance to the D. radiodurans SSB. As a <br /> sis, D. radiodurans SSB protein displayed a fractional reten-      homodimer, SSBDr contains 4 OB folds, similar to the E. <br /> tion (Kav) on a Sephacryl S-200 column of 0.21. E. coli SSB        coli SSB homotetramer. However, the same D. radiodurans <br /> protein, ran for comparison, displayed a Kav of 0.18. Using <br />  <br />  <br />                                                                                                                        Page 4 of 12 <br />                                                                                                  (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                                    http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br />  <br />                 A                            OB fold                              OB fold <br />  <br />                      Drad       1   1&#226;&#8364;&#8482;   2    3    &#206;&#177;   4 451 452 5   1   1&#226;&#8364;&#8482;   2    3    &#206;&#177;   4 451 452 5                 301 <br />                      Taq                                                                                                264 <br />                      TthHB8                                                                                             263 <br />                      TthVK1                                                                                             266 <br />                      Gmet                                                                                               147 <br />                      Neur                                                                                               149 <br />                      PaerPAO1                                                                                           165 <br />                      EcoliK12                                                                                           178 <br />  <br />                 B <br />  <br />  <br />  <br />  <br /> Figure schematics <br /> Protein  3          and amino acid sequence alignments of D. radiodurans R1 SSB protein and other SSB proteins <br /> Protein schematics and amino acid sequence alignments of D. radiodurans R1 SSB protein and other SSB pro- <br /> teins. A) Schematic representation of the D. radiodurans R1 SSB protein and other SSB proteins highlighting the OB fold <br /> regions. The Drad, Taq, TthHB8 and TthVK1 SSB proteins contain two OB folds each. The characteristic motifs that make up an <br /> OB fold are highlighted with open boxes/arrows and numbered. The Gmet, Neur, PaerPAO1 and EcoliK12 SSB proteins contain <br /> one OB fold each and align with both the N-terminal OB fold and the C-terminal OB fold of the proteins that contain two OB <br /> folds. Only the C-terminal alignments are shown. The structural assignments are according to the OB fold rule defined by Mur- <br /> zin [11,32]. B) Amino acid sequence alignment of D. radiodurans R1 SSB protein and other SSB proteins from closely and dis- <br /> tantly related bacteria. The Drad, Taq, TthHB8 and TthVK1 SSB protein sequences are divided into N- and C-terminal fragments <br /> in order to highlight that each fragment contains an OB fold. The arrows indicate &#206;&#178;-sheets and the rectangle indicates the &#206;&#177;- <br /> helix contained in the secondary structure of the OB fold of SSB protein from E. coli strain K12 [11,32]. Sequence identity is <br /> shown by white fonts on black boxes, and sequence similarity is shown by black font on gray boxes. Sequence similarity is <br /> defined by the following amino acid groupings: STA, NEQK, NHQK, NDEQ, QHRK, MILV, MILF, HY, FYW. Abbreviations: <br /> Drad-N or -C: D. radiodurans strain R1, N- or C- terminal fragment; Taq-N or -C: T. aquaticus, N- or C- terminal fragment; <br /> TthHB8-N or -C: T. thermophilus strain HB8, N- or C- terminal fragment; TthVK1-N or -C: T. thermophilus strain VK1, N- or C- <br /> terminal fragment; Gmet: Geobacter metallireducens; Neur: Nitrosomonas europaea ATCC 19718; PaerPAO1: Pseudomonas aerugi- <br /> nosa PAO1. EcoliK12: E. coli strain K12. <br />  <br />  <br />  <br />                                                                                                                                 Page 5 of 12 <br />                                                                                                           (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                         http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br />                                                                   ally weakens the binding of E. coli SSB homotetramer to <br />                                                                   nucleic acids [15]. The authors of this study conclude that <br />                                                                   the amino acid sequence contained between the OB fold <br />                             E. coli SSB                           and the highly acidic 10 amino acid tail acts purely as a <br />                                                                   spacer to shield the negative charges of the C-terminus <br />                                                                   from the bound DNA on the SSB protein [15]. In support <br />                                  D. radiodurans SSB               of this idea, a crystal structure of E. coli SSB protein, in the <br />                                                                   absence of DNA, suggests that the C-terminal region (the <br />                                                                   region not involved in the OB fold) protrudes from the <br />                                                                   tetramer and that each of the four C-terminal regions take <br /> Log M r <br />  <br />  <br />  <br />  <br />                                                                   on different conformations [16]. <br />  <br />                                                                   We speculate that the acidic terminus of the D. radiodurans <br />                                                                   SSB protein is also important for in vivo protein-protein <br />                                                                   interactions, but that there may be some interesting func- <br />                                                                   tional differences reflecting the reduced number of C-ter- <br />                                                                   minal regions in the D. radiodurans SSB oligomer <br />                                                                   compared to that of E. coli and other bacterial SSB oligom- <br />                                                                   ers. Further studies involving the D. radiodurans SSB C-ter- <br />                                                                   minus will help to further understand the role of the <br />                                                                   spacer and acidic tail regions of bacterial SSB proteins. <br />  <br />                                                                   A D. radiodurans SSB protein homodimer facilitates RecA <br />                                                                   promoted DNA three-strand exchange <br />                                                                   Single-stranded DNA binding proteins, in general, help <br />                                  K av                             stimulate recombinase promoted in vitro DNA strand <br />                                                                   exchange reactions. The E. coli system has been most <br /> Molecular <br /> protein oligomer <br /> Figure    4 weight approximation of the D. radiodurans SSB        intensely studied and E. coli SSB protein has been shown <br /> Molecular weight approximation of the D. radio-                   to have both pre-synaptic and post-synaptic roles in E. coli <br /> durans SSB protein oligomer. Gel filtration of the stand-         RecA protein-promoted DNA strand exchange reactions <br /> ards and SSB proteins were performed as described in the <br />                                                                   [17,18]. The addition of E. coli SSB during the presynaptic <br /> Methods section using a S-200 Sephacryl HR column. The <br /> protein standards &#206;&#178;-amylase (200 kDa), alcohol dehydroge-         formation of RecA filaments helps to remove ssDNA sec- <br /> nase (150 kDa), bovine serum albumin (66 kDa), carbonic           ondary structure, allowing formation of contiguous RecA <br /> anhydrase (29 kDa), and cytochrome c (12.4 kDa) were used         filaments [17]. Post-synaptically, E. coli SSB facilitates the <br /> to calibrate the column as shown by open circles. The elution     recombination reaction by binding the displaced ssDNA <br /> volume of blue dextran was used to calculate the void vol-        and preventing the reversal of the strand exchange reac- <br /> ume and the total volume of the column was known. The             tion [18]. <br /> best-fit line was generated of the log Mr of the protein stand- <br /> ards versus Kav of the standards. D. radiodurans SSB protein      The SSB proteins from different organisms have fre- <br /> and E. coli SSB protein samples were injected onto the col-       quently been interchanged and found to stimulate the <br /> umn in independent experiments and their elution volumes          reactions of recombinases from other organisms [19-21]. <br /> were used to calculate their Kav. The position of D. radio- <br />                                                                   These observations suggest that there is little species-spe- <br /> durans SSB protein on the standard curve is indicated by a <br /> closed square and the position of E. coli SSB protein on the      cific protein-protein interaction. The E. coli SSB protein <br /> standard curve is indicated by an open square.                    greatly stimulates the D. radiodurans RecA protein-pro- <br />                                                                   moted DNA strand exchange reaction [22], and in fact an <br />                                                                   SSB protein is required to observe significant reaction. We <br />                                                                   have now found that the D. radiodurans SSB protein stim- <br />                                                                   ulates the DNA strand exchange reactions promoted by <br /> SSB homodimer contains only two C-terminal regions                both RecADr and the E. coli RecAEc (Fig. 5). In the reactions <br /> whereas the E. coli SSB homotetramer contains four.               promoted by both D. radiodurans RecA protein (Fig 5 Pan- <br />                                                                   els A, B and C) and the E. coli RecA protein (Figure 5 Panel <br /> In vitro analyses of the E. coli SSB protein showed that the      D), less than half as much D. radiodurans SSB protein was <br /> C-terminal third of the protein is not needed for tetramer        needed to generate a yield of nicked circular product equal <br /> formation or DNA binding and that the last 10 amino               to that seen in the reactions with E. coli SSB under the <br /> acids, containing the highly conserved acidic region, actu-       same conditions. The reduced requirement for the D. <br />  <br />  <br />                                                                                                                        Page 6 of 12 <br />                                                                                                  (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                    http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br /> radiodurans SSB is again consistent with the formation of     isoamyl alcohol and then once with 24:1 chloroform/iso- <br /> a D. radiodurans SSB homodimer (Fig 5). Since the D. radi-    amyl alcohol. The non-aqueous layer was removed, and <br /> odurans SSB forms a homodimer (with 4 total OB folds)         the DNA was ethanol precipitated. The dry DNA pellet <br /> and E. coli SSB forms a homotetramer (with 4 total OB         was dissolved in 200 &#194;&#181;l TE, and 1 &#194;&#181;l was used in subse- <br /> folds), the concentration of SSBDr should be multiplied by    quent PCR amplifications. <br /> a factor of 2 to provide an SSBDr-SSBEc comparison that <br /> reflects equal numbers of OB folds (see dashed line in Fig.   Sequencing approaches <br /> 5, panels C and D).                                           The sequence of the ssb gene was checked in three inde- <br />                                                               pendent analyses using different PCR primers and <br /> Note that the maximum product generation is slightly dif-     sequencing primers. In a direct approach, primers starting <br /> ferent for the D. radiodurans and E. coli RecA protein-pro-   in the flanking DR0098 ribosomal protein S6 gene (5&apos; <br /> moted reactions. The optimal conditions are slightly          CATCAAGGCTTCGGGCAAC 3&apos;) and the DR0101 ribos- <br /> different for the two recombinases, and the conditions        omal protein S18 gene (5&apos; TTGCGTTCGCCGCTGTTTCC <br /> used in these reactions were those optimal for the E. coli    3&apos;) were used to PCR amplify the ssb gene and flanking <br /> RecA protein.                                                 regions. Sequencing primers were used to directly <br />                                                               sequence this PCR fragment with a 2&#226;&#8364;&#8220;4-fold coverage. In <br /> Conclusions                                                   addition to this direct approach, the ssb gene was inde- <br /> We conclude that frameshifting does not occur within the      pendently PCR amplified and inserted into two separate <br /> D. radiodurans ssb gene. The gene consists of a contiguous    plasmids, pRAD1 (obtained as a gift from Mary E. Lid- <br /> open reading frame encompassing the regions now               strom) [9] and pET21A (Novagen) and sequenced from <br /> labeled DR0099 and DR0100. The predicted amino acid           these plasmids. The first clone, with the ssb gene inserted <br /> sequence of the D. radiodurans SSB protein is closely         between the XhoI and HindIII cloning sites of vector <br /> related to the SSB proteins from T. aquaticus and T. ther-    pRAD1, used the PCR forward primer containing the XhoI <br /> mophilus. The purified D. radiodurans SSB protein, like the   site (underlined) (5&apos; CTACCGCTCGAGAGTGGAAGAC- <br /> SSB proteins from T. aquaticus and T. thermophilus [7],       CAAGAAGGCCTGAGC 3&apos;) and the reverse primer con- <br /> forms a homodimer. The D. radiodurans SSB homodimer           taining         the         HindIII         site         (5&apos; <br /> stimulates D. radiodurans RecA and also E. coli RecA pro-     TATGAAGCTTTTAGTGGTGGTGGTGATGATGGCCAC- <br /> tein-promoted DNA strand exchange reactions with at           CAAAGGGCAGGTCG 3&apos;). The forward primer for the <br /> least the same efficiency as the E. coli SSB homotetramer     pRAD1 insertion PCR was designed to anneal in the <br /> stimulates these reactions. Elucidation of the complete D.    upstream flanking ribosomal protein S6 gene while the <br /> radiodurans ssb gene and purification of the D. radiodurans   reverse primer was designed to anneal at the end of the ssb <br /> SSB protein opens the way for further studies of the DNA      gene and included a C-terminal histidine tag sequence. <br /> metabolism of this unusually DNA repair-proficient            The ssb gene was sequenced in this clone with a two-fold <br /> organism.                                                     coverage. This clone was not used for any further experi- <br />                                                               mentation contained within this report. A second, inde- <br /> Methods                                                       pendently created clone called pEAW328, with the ssb <br /> Strain                                                        gene inserted between the EcoRI and the NdeI cloning sites <br /> The Deinococcus radiodurans R1 type strain was obtained       of pET21A, used the PCR forward primer containing the <br /> directly from the American Type Culture Collection            NdeI site (5&apos; CGTATTCCATATGGCCCGTGGCATGAAC- <br /> (ATCC 13939).                                                 CAC 3&apos;) and the reverse primer containing the EcoRI site <br />                                                               (5&apos; CGGAATTCTTAAAAGGGCAGGTCGTC 3&apos;). The for- <br /> Genomic DNA isolation                                         ward primer for the pET21A insertion PCR was designed <br /> Genomic DNA was purified from Deinococcus radiodurans         to anneal at the initiating codon of the correct ssb gene <br /> R1 type strain cells using the following protocol. TGY        and contained a CGT codon (italicized) rather than the <br /> media (5% tryptone, 3% yeast extract, 1% glucose) was         natural CGA codon for Arg in order to replace a low use <br /> used in a 5 mL overnight 30&#194;&#176;C culture growth. The cells       Arg codon (the third amino acid in the primary sequence) <br /> were pelleted and resuspended in 2 mL suspension buffer       with a higher use Arg codon in preparation for overexpres- <br /> (10 mM Tris-HCl (pH 8.0), 1 mM Na-EDTA, 0.35 M                sion. The reverse primer for the pET21 insertion PCR was <br /> sucrose). Lysozyme (2 mg) was added and the suspension        designed to anneal at the natural termination codon of <br /> was incubated for 4 hours at 37&#194;&#176;C. Lysis buffer (3 ml; 100    the correct ssb gene. No tag sequence was used in the <br /> mM Tris-HCl (pH 8.0), 0.3 M NaCl, 20 mM EDTA, and             pET21A vector construction. Three separate pET21A <br /> 2% (w/v) SDS, with 2% (v/v) &#206;&#178;-mercaptoethanol added           insertion clones were sequenced twice. All restriction <br /> just before use) was added to lyse the cells. The suspen-     enzymes were purchased from New England Biolabs. <br /> sion was extracted with equal volumes of TE saturated <br /> phenol once, then once with 25:24:1 phenol/chloroform/ <br />  <br />  <br />                                                                                                                  Page 7 of 12 <br />                                                                                            (page number not for citation purposes) <br />  <br /> DNA <br /> Figurestrand <br />          5 exchange reactions promoted by D. radiodurans RecA and E. coli RecA with SSB titrations <br /> DNA strand exchange reactions promoted by D. radiodurans RecA and E. coli RecA with SSB titrations. Reac- <br /> tions were carried out as described in the Methods section. Circular single-stranded DNA (css) was preincubated with either <br /> D. radiodurans or E. coli RecA. ATP and SSB protein (either D. radiodurans or E. coli SSB protein as indicated) were then added <br /> and incubated, followed by the addition of homologous linear double-stranded DNA (lds) which initiated the DNA three- <br /> strand exchange reaction. The nicked circular double-stranded DNA product (nc) is distinguishable by agarose gel electro- <br /> phoresis and quantifiable. Panels A and B show the agarose gel electrophoresis results of reactions promoted by D. radiodurans <br /> RecA with various monomer concentrations of D. radiodurans SSB protein and E. coli SSB protein, respectively. These results <br /> are quantitated in panel C, with the data from reactions with D. radiodurans SSB (closed circles) and E. coli SSB (closed triangles) <br /> coming from Panels A and B respectively. Panel D shows the quantitated results of similar reactions promoted by E. coli RecA <br /> with D. radiodurans SSB (closed circle) and E. coli SSB (closed triangle) (agarose gel not shown). Since an SSBDr monomer has <br /> two OB folds and an SSBEc monomer has only one, the improved reactions seen in the reactions containing the former protein <br /> could simply reflect the higher effective concentration of OB folds when monomeric SSB concentrations are compared. In pan- <br /> els C and D, the dashed lines represent a plot in which the percentage reaction product generated in the reactions using SSBDr <br /> are plotted against the actual concentration of OB folds in these reactions (twice the actual concentration of D. radiodurans SSB <br /> monomers). The dashed lines (--X--) thus allow a direct comparison of the reactions observed with the D. radiodurans SSB pro- <br /> tein with the reactions observed with the E. coli SSB protein (closed triangles). The production of nicked circular double- <br /> stranded DNA product is calculated as a percentage of total duplex DNA (the sum of linear double-stranded DNA substrate, <br /> nicked circular double-stranded DNA product and any network products near the well). <br />  <br />  <br />  <br />                                                                                                                                                       Page 8 of 12 <br />                                                                                                                                 (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                      http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br /> Protein sequence analysis                                       ~65% of the initial volume used to resuspend the cells. <br /> Standard BLAST pair wise analysis (Blosum62 matrix at           The resuspended solution was centrifuged for 20 min at <br /> default settings) of the protein sequences was used to cal-     30,000 g to remove non-dissolved proteins. The superna- <br /> culate percent identity and similarity values between           tant was dialyzed against TGE Buffer. Following dialysis, <br /> proteins. ClustalX was used to generate the multiple            the solution was cleared of precipitate by a 5 min centrif- <br /> sequence alignments.                                            ugation at 3,800 g and the supernatant was dialyzed <br />                                                                 against P (20 mM) buffer (20 mM potassium phosphate, <br /> Expression and purification of D. radiodurans SSB Protein       pH 7.5, 10% (w/v) glycerol, 0.1 mM EDTA, 1 mM &#206;&#178;-mer- <br /> As described above, the D. radiodurans ssb gene PCR prod-       captoethanol). Again the solution was cleared by centrifu- <br /> uct was inserted into the EcoRI and the NdeI cloning sites      gation following dialysis. A hydroxyapatite column (Bio- <br /> of pET21A (Novagen) to yield construct pEAW328. The             Rad) was pre-equilibrated with P (20 mM). EDTA was not <br /> ssb gene in this construct did not contain a histidine tag or   used in buffers applied to the column other than the <br /> any modification that would lead to a translation product       EDTA contained in the load volume. The hydroxyapatite <br /> that would differ from that encoded by the chromosomal          bed volume used and found adequate was 110% of the <br /> D. radiodurans ssb gene. Construct pEAW328 was trans-           initial volume used to resuspend the cells. The dialyzed <br /> formed into BL21 Codon Plus (DE3) (Stratagene) E. coli          protein was loaded on the column and eluted over a 10 <br /> cells. These cells were grown in LB broth in the presence       column volume linear gradient going from Buffer P (20 <br /> of 100 &#194;&#181;g/ml of ampicillin and 25 &#194;&#181;g/ml of chloramphen-         mM) to Buffer P (152 mM, i.e. 152 mM potassium phos- <br /> icol at 35&#194;&#176;C to an optical density at 600 nm of 0.8. The        phate, pH 7.5, 10% (w/v) glycerol, 1 mM &#206;&#178;-mercaptoeth- <br /> cells were then induced with 0.4 mM IPTG and grown at           anol). The SSB completed elution approximately half way <br /> 35&#194;&#176;C for three more hours before harvest. Harvested cells       through the linear gradient. Fractions containing mostly <br /> were resuspended in Buffer A (25 mM Tris-HCl, pH 8.3,           SSB and some very minor degradation bands by SDS- <br /> 12% w/v glycerol, 0.5 mM EDTA) with the buffer addition         PAGE were pooled and dialyzed against TGE Buffer. The <br /> corresponding to five times the cell volume. Lysozyme           dialyzed protein was then loaded onto a DEAE Sepharose <br /> was added to a final concentration of 0.2 mg/ml. Cells          (Amersham Pharmacia Biotech) column (bed volume of <br /> were stirred at 4&#194;&#176;C for 1 hour and then sonicated on ice.       110% of the initial volume used to resuspend the cells was <br /> All subsequent purification steps were performed at 4&#194;&#176;C.        found adequate) and eluted in a 10 column volume linear <br /> Cell debris and insoluble material were removed by 3 suc-       gradient of TGE Buffer going from 0.0 M NaCl to 0.3 M <br /> cessive 20 min centrifugations at 38,000 g with insoluble       NaCl. Degraded protein eluted before the pure SSB pro- <br /> material being removed between centrifugations. Solid           tein eluted. Elution was complete at about half way <br /> NaCl was then dissolved in the supernatant to a concen-         through the gradient. Fractions containing pure SSB pro- <br /> tration of 0.18 M NaCl. DNA and proteins were precipi-          tein were pooled and dialyzed into storage buffer (20 mM <br /> tated by drop-by-drop addition of 10% (w/v)                     Tris-HCl, pH 8.3, 0.5 M NaCl, 50% (w/v) glycerol, 1 mM <br /> polyethylenimine, pH 7.5, with constant stirring to a final     EDTA, 1 mM &#206;&#178;-mercaptoethanol), aliquotted, snap frozen <br /> concentration of 0.4% (w/v) polyethylenimine. The solu-         in liquid nitrogen, and stored at -80&#194;&#176;C. <br /> tion was stirred for 15&#226;&#8364;&#8220;60 minutes and then centrifuged <br /> for 15 min at 10,000 g. The SSB protein remained in the         The extinction coefficient for the D. radiodurans SSB pro- <br /> pellet at this point and was eluted from the pellet with        tein was determined as described previously [23,24]. <br /> Buffer B (25 mM Tris-HCl, pH 8.3, 0.4 M NaCl, 12% w/v           Eleven determinations at three different concentrations of <br /> glycerol, 0.5 mM EDTA, 1 mM &#206;&#178;-mercaptoethanol) in a             D. radiodurans SSB protein gave an average extinction <br /> volume equal to the volume initially used to resuspend          coefficient of &#206;&#181;280 = (4.1 &#194;&#177; 0.2) &#195;&#8212; 104 M-1cm-1. The N-ter- <br /> the cells. The polyethylenimine pellet was broken up and        minal sequence analysis was performed by the Protein <br /> resuspended using a plastic spatula and then a glass            and Nucleic Acid Chemistry Laboratories, Department of <br /> homogenizer or small mortar and pestle. This suspension         Molecular Biology and Pharmacology, Washington Uni- <br /> was stirred for 30 min and then centrifuged for 15 min at       versity School of Medicine, St. Louis, Mo., without the lab- <br /> 10,000 g. The SSB was found in the supernatant. Solid           oratory personnel knowing the identity of the protein. <br /> Ammonium Sulfate was slowly added to the supernatant <br /> while stirring over the course of 30 minutes to a 30% sat-      Native Molecular Weight Determination by Gel Filtration <br /> uration of the solution. This solution was allowed to con-      The native molecular weight of D. radiodurans SSB was <br /> tinue stirring for approximately 3 hours and the                approximated by gel filtration FPLC. A Sephacryl S-200 <br /> precipitated proteins, including much of the SSB, were          HR column (31.4 cm &#195;&#8212; 1.0 cm) was used at a flow rate of <br /> collected by centrifugation at 10,000 g for 20 min. The         0.05 ml/min. The buffer in all experiments was 50 mM <br /> Ammonium Sulfate pellet was dissolved in TGE Buffer (50         Tris-HCl, pH 7.5, 100 mM KCl, as recommended by <br /> mM Tris-HCl, pH 8.3, 20% w/v glycerol, 1 mM EDTA, 1             Sigma for the protein standards. Chromatography was <br /> mM &#206;&#178;-mercaptoethanol) with gentle stirring at a volume          performed at 4&#194;&#176;C while A280 was measured. The column <br />  <br />  <br />                                                                                                                    Page 9 of 12 <br />                                                                                              (page number not for citation purposes) <br />  BMC Microbiology 2004, 4                                                       http://www.biomedcentral.com/1471-2180/4/2 <br />  <br />  <br />  <br /> was calibrated using Sigma Gel Filtration Molecular              cific volume was calculated from the composition as <br /> Weight Markers: blue dextran (2,000 kDa), beta-amylase           0.722 mL/gm. The dialysate density was measured as <br /> (200 kDa), alcohol dehydrogenase (150 kDa), bovine               1.007518 gm/mL at 4&#194;&#176;C using an Anton Paar DMA5000 <br /> serum albumin (66 kDa), carbonic anhydrase (29 kDa),             density meter. <br /> and cytochrome c (12.4 kDa). The standards were loaded <br /> independently at the concentrations recommended by               The data from the three samples at six speeds were glo- <br /> Sigma in 100 &#194;&#181;l sample volumes. Approximately 70 &#194;&#181;g of           bally tested against models of a single species, two nonin- <br /> E. coli SSB protein (18.8 kDa per monomer) and 200 &#194;&#181;g            teracting species and two species in equilibrium. <br /> of D. radiodurans SSB protein (32.6 kDa per monomer)             Programs for analysis were written in Igor Pro (Wavemet- <br /> were independently loaded on the column in 100 &#194;&#181;l                rics Inc., Lake Oswego, OR) by Darrell R. McCaslin. <br /> sample volumes. Standards and SSB loads were dissolved <br /> or dialyzed respectively into the recommended buffer plus        DNA Three-Strand Exchange Reactions <br /> 5% (w/v) glycerol. The elution volume of blue dextran            Enzymes and Reagents <br /> determined the void volume (Vo), and the total volume            The E. coli RecA and D. radiodurans RecA proteins were <br /> (Vt) was determined by volume measurements of the col-           purified by polyethylenimine precipitation followed by a <br /> umn before packing and measurement of the column tub-            DEAE-Sepharose column and a hydroxyapatite column as <br /> ing used. The peak elution volumes (Ve) were calculated          described [26]. Protein concentrations were determined <br /> from the chromatogram and fractional retentions, Kav,            by absorbance at 280 nm using the extinction coefficients <br /> were calculated using the equation: Kav = (Ve - Vo)/(Vt - Vo).   &#206;&#181;280 = 2.23 &#195;&#8212; 104 M-1 cm-1 for E. coli RecA [27], 1.41 &#195;&#8212; 104 <br /> A standard curve was determined by plotting the Kav of the       M-1 cm-1 for D. radiodurans RecA [22] and 2.38 &#195;&#8212; 104 M- <br /> protein standards against the log10Mr of the standards. The      1cm-1 for E. coli SSB [28]. E. coli SSB protein was purified <br />  <br /> native molecular weight of D. radiodurans SSB protein was        as previously described [25]. <br /> approximated by comparing its Kav value to the standard <br /> curve. The native molecular weight of E. coli SSB protein        Preparation of DNA substrates <br /> was determined for comparison in like manner. E. coli SSB        Duplex supercoiled DNA and circular ssDNA (css DNA) <br /> protein was purified as previously described [25], and an        substrates from bacteriophage M13mp7 (7238 bp) were <br /> extinction coefficient of 2.38 &#195;&#8212; 104 M-1cm-1 was used to         purified as described [29-31]. The linear dsDNA substrate <br /> calculate the concentration of E. coli SSB.                      was prepared from M13mp7 supercoiled dsDNA, which <br />                                                                  was cut with BsmBI and purified by electrophoresis on <br /> Sedimentation Equilibrium Measurements                           1.0% agarose gel. The concentrations of ssDNA and <br /> To prepare samples for sedimentation equilibrium a 1 mL          dsDNA solutions were determined by absorbance at 260 <br /> D. radiodurans SSB 157 &#194;&#181;M protein sample (in 35 mM               nm, using 36 and 50 &#194;&#181;g ml-1 A260-1, respectively, as conver- <br /> Tris-HCl (pH 7.7), 25% w/v glycerol, 300 mM NaCl, 1              sion factors. The concentrations of DNA and proteins <br /> mM EDTA, 0.5 mM &#206;&#178;-mercaptoethanol, 1 mM EDTA                     reported below are the final concentrations after addition <br /> buffer) was dialyzed for 1 hour at 4&#194;&#176;C against 50 mM Tris-       of all components and DNA concentrations are in terms <br /> HCl, pH 7.5, 100 mM NaCl, 1 mM EDTA dialysis buffer,             of total nucleotides. <br /> and then again against 2 L of fresh dialysis buffer over- <br /> night at 4&#194;&#176;C. The protein concentration of the resulting         DNA three-strand exchange reaction <br /> sample was 80.4 &#194;&#181;M. The final dialysate was used to dilute       The DNA three-strand exchange reactions were carried out <br /> aliquots of this stock to concentrations of 4.67 &#194;&#181;M, 12.4        at 37&#194;&#176;C in solutions containing 25 mM Tris-OAc (80% <br /> &#194;&#181;M and 20.4 &#194;&#181;M.                                                  H+, pH 7.5), 10 mM Mg(OAc)2 (Fisher Scientific), 1 mM <br />                                                                  DTT (Research Organics), 3 mM Potassium Glutamate, <br /> 100 &#194;&#181;L of each sample was placed in 12 mm double-sector          5% (w/v) glycerol and an ATP regeneration system (12 <br /> charcoal-filled Epon centerpieces with about 105 &#194;&#181;L of the       mM phosphocreatine and 10 units/ml phosphocreatine <br /> final dialysate as reference. Centrifugation was performed       kinase (Boehringer Mannheim)). Two &#194;&#181;M RecA protein <br /> at 4&#194;&#176;C in a Beckman model XL-A analytical ultracentri-           (from E. coli or D. radiodurans) was pre-incubated with 6 <br /> fuge. The protein gradients were recorded at 280 nm every        &#194;&#181;M css DNA in the reaction buffer and regeneration sys- <br /> 2&#226;&#8364;&#8220;3 hours until they became superimposable. Data were            tem for 10 min. ATP (3 mM) and SSB (indicated concen- <br /> collected at six speeds (5200, 8200, 11000, 14000, 16500,        trations) were then added, followed by another 20 min <br /> 18500 rpm). With the various speeds and starting concen-         incubation. The reactions were initiated by addition of 10 <br /> trations, absorbance in the gradients ranged from ~0 to          &#194;&#181;M lds DNA. The reactions were incubated for 2 hr and <br /> &gt;2.8, which corresponds to ~0 to &gt;60 &#194;&#181;M protein. Base-           stopped by the addition of 1.2 &#194;&#181;l 10% SDS, 0.3 &#194;&#181;l 0.5 M <br /> line absorbances were measured for each sample after             EDTA and 0.6 &#194;&#181;l 20 mg/ml Proteinase K followed by 30 <br /> high speed depletion and was less than 0.03 in all cases         min incubation. Aliquots mixed with 2.5 &#194;&#181;l 6 &#195;&#8212; loading <br /> and was subtracted prior to curvefittings. The partial spe-      buffer (15% Ficoll, 0.25% bromphenol blue, 0.25% <br />  <br /> xylene cyanole FF) were loaded on an 1% agarose gel and     <br /> electrophoresed at 25&#226;&#8364;&#8220;35 V for 16 hr at room tempera-   <br /> ture. To visualize the DNA bands, the gels were stained        <br /> with ethidium bromide, and exposed to UV light. Gel     <br /> images were captured with a digital CCD camera utilizing          <br /> GelExpert software (Nucleotech). The intensity of DNA                  <br /> bands was quantitated with the software package TotalLab          <br /> v1.10 from Phoretix.          <br /> List of abbreviations                                              <br /> ssb, single-strand DNA-binding protein gene; SSB, single-          <br /> stranded DNA-binding protein; ORF, open reading frame;  <br /> OB, oligonucleotide/oligosaccharide-binding; ATCC,            <br /> American Type Culture Collection; PCR, polymerase     <br /> chain reaction; NCBI, National Center for Biotechnology         <br /> Information; RBS, ribosomal binding site; SSBDr, D. radi-                  <br /> odurans single-stranded DNA-binding protein; ssDNA,   <br /> single-stranded DNA; css, circular single-stranded DNA;          <br /> lds, linear double-stranded DNA; nc, nicked circular dou-         <br /> ble-stranded DNA ; Drad, D. radiodurans strain R1; Taq,  <br /> Thermus aquaticus; TthHB8, Thermus thermophilus strain     <br /> HB8; TthVK1, Thermus thermophilus strain VK1; Gmet, Geo-         <br /> bacter metallireducens; Neur, Nitrosomonas europaea ATCC              <br /> 19718; PaerPAO1, Pseudomonas aeruginosa PAO1;                     <br /> EcoliK12, Escherichia coli strain K12.                        <br /> Authors&apos; contributions                                      <br /> J.M.E. predicted and confirmed the full length D. radio-             <br /> durans ssb gene, identified one of two original sequencing             <br /> errors in the ssb gene, performed the sequence alignments,           <br /> purified the D. radiodurans SSB protein, determined the                <br /> oligomeric state of the protein by analytical gel filtration,        <br /> assisted in the sedimentation equilibrium analysis and             <br /> drafted the manuscript. N.H. performed and quantitated           <br /> the DNA three-strand exchange experiments. E.A.W. iden-          <br /> tified one of two original sequencing errors in the D. radi-      <br /> odurans ssb gene, participated in the confirmation of the           <br /> full length ssb gene and cloned the D. radiodurans ssb gene     <br /> in the over-expression vector. M.M.C. conceived of the                         <br /> study, participated in its design and coordination and                 <br /> edited the manuscript. All authors read and approved the        <br /> final manuscript.                                                       <br /> </body></html>