<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A New Method Discovery Essential Proteins <br /> Xue Zhang1,2*, Jin Xu1,2, Wang-xin Xiao3 <br /> 1 Key Laboratory High Confidence Software Technologies, Ministry Education, Peking University, Beijing, China, 2 School Electronics Engineering Computer <br /> Science, Peking University, Beijing, China, 3 Research Institute Highway Ministry Transport, Beijing, China <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Background: Experimental methods identification essential proteins costly, time-consuming, and <br />      laborious. It challenging task protein essentiality experiments. With development high <br />      throughput technologies, vast protein-protein interactions available, enable identification of <br />      essential proteins network level. Many computational methods task proposed based the <br />      topological properties protein-protein interaction (PPI) networks. However, currently available PPI networks each <br />      species complete, i.e. false negatives, noisy, i.e. high false positives, network topology-based centrality <br />      measures sensitive noise. Therefore, exploring robust methods identifying essential proteins <br />      great value. <br />  <br />      Method: In paper, new essential protein discovery method, named CoEWC (Co-Expression Weighted Clustering <br />      coefficient), proposed. CoEWC based integration topological properties PPI network co- <br />      expression interacting proteins. The aim CoEWC capture common features essential proteins date <br />      hubs party hubs. The performance CoEWC validated based PPI network Saccharomyces cerevisiae. <br />      Experimental results CoEWC significantly outperforms classical centrality measures, also <br />      outperforms PeC, newly proposed essential protein discovery method outperforms 15 centrality measures on <br />      PPI network Saccharomyces cerevisiae. Especially, predicting 500 proteins, 50% <br />      improvements obtained CoEWC degree centrality (DC), better centrality measure identifying protein <br />      essentiality. <br />  <br />      Conclusions: We demonstrate robust essential protein discovery method developed integrating the <br />      topological properties PPI network co-expression interacting proteins. The proposed centrality measure, <br />      CoEWC, effective discovery essential proteins. <br />  <br />   Citation: Zhang X, Xu J, Xiao W-x (2013) A New Method Discovery Essential Proteins. PLoS ONE 8(3): e58763. doi:10.1371/journal.pone.0058763 <br />   Editor: Vladimir N. Uversky, University South Florida College Medicine, United States America <br />   Received December 25, 2012; Accepted February 6, 2013; Published March 21, 2013 <br />   Copyright: &#195;&#376; 2013 Zhang et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work partially supported Key project National Natural Science Fund (No. 61133010 No. 61127005) National Natural <br />   Science Foundation China (No. 50708085 No. 50978127). The funders role study design, data collection analysis, decision publish, or <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: lindajia03@126.com <br />  <br />  <br />  <br /> Introduction                                                                            With advances high-throughput experimental technolo- <br />                                                                                      gies, Y2H mass spectrometry, large amounts of <br />    Genome-wide gene deletion studies small fraction                   protein-protein interaction (PPI) data produced, which <br /> genes genome indispensable <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span>                               make possible study proteins network level. In order <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> reproduction organism [1,2]. These genes referred                       break experimental constraints, recently researchers have <br /> essential genes, essential proteins just products                     paid attention computational methods based on <br /> essential genes. The deletion essential proteins result              network topological characteristics. The correlations between <br /> lethality infertility. The identification essential proteins           network topological features protein essentiality been <br /> important understanding minimal requirements                        explored researchers. It observed several <br /> for survival organism, finding human disease                      species, Saccharomyces cerevisiae, Caenorhabditis elegans <br /> genes [3] new drug targets. The genome-wide identification                    Drosophila melanogaster [9,10], hub proteins PPI <br /> essential genes valuable rational drug design [4]. Essential                  network, highly connecting proteins, more <br /> proteins pathogenic organisms taken potential                       likely essential low connections [11]. This the <br /> targets new antibiotics [5].                                                     so-called centrality-lethality rule [11]. Several researchers tried <br />    Several experimental methods discovery essential                       explain correlation different hypotheses [12&#226;&#8364;&#8220;15]. <br /> proteins conducted, single gene knockouts [6],                     Although controversies exist explanations, most <br /> <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>RNA interference</span> [7] conditional knockouts [8]. However,                         researchers confirmed correlation topological <br /> these experimental methods time-consuming                               centrality protein essentiality [10,16&#226;&#8364;&#8220;18]. <br /> laborious, require large amounts resources.                           Computational methods seen useful preprocessing <br />                                                                                      techniques help experimental methods quickly find <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       1                                March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                    Method Discovery Essential Proteins <br />  <br />  <br /> essential proteins. Many centrality measures proposed            neighbors consideration. As result, CoEWC expected to <br /> to capture correlation network topological properties          identify essential proteins date hubs party hubs well. The <br /> and protein essentiality. Local network features based centrality          performance CoEWC tested studied species of <br /> measures include degree centrality (DC) [11],sum edge                   Saccharomyces cerevisiae. Compared previous centrality <br /> clustering (SoECC) [18], local average connectivity (LAC) [19],            measures better predicting precision, CoEWC <br /> and density maximum neighborhood component (DMNC) [20].                 achieves higher predicting precision identification of <br /> Global network characteristics based centrality measures include           essential proteins. The experimental results demonstrate that <br /> betweenness centrality (BC) [21], closeness centrality (CC)            centrality measures, based appropriate integration of <br /> [22]. Other previously proposed centrality measures include                network topological properties gene expression, more <br /> subgraph centrality [23], eigenvector centrality [24], information         robust effective, based network topological <br /> centrality [25], bottle neck [26,27], method integrating        features, discovery essential proteins, CoEWC is <br /> network topology gene expression data (PeC) [28]. Compar-              good example integration. <br /> ative studies kinds measures local features <br /> based measures effective identifying essential proteins       Methods <br /> [28&#226;&#8364;&#8220;29]. <br />    Since currently available PPI networks species         Motivations <br /> not complete, i.e. false negatives, noisy, i.e. high false           As reported [32], hubs yeast interactome network can <br /> positives, especially obtained high-throughput                classified date party hubs basis partners&#226;&#8364;&#8482; <br /> technologies, identification essential proteins based            expression profiles, moreover, party hubs date hubs have <br /> network topology challenging. Most centrality measures       similar probability essential. Therefore, exploring co- <br /> are sensitive noise PPI network. In addition,        expression protein interacting neighbors PPI <br /> known false negatives false positives PPI networks        network identify protein&#226;&#8364;&#8482;s essentiality reasonable. <br /> are hard cleaned out. Therefore, robust centrality measures             If use Pearson Correlation Coefficient (PCC) capture the <br /> for discovery essential proteins great value.           co-expression, yeast interactome non- <br /> Biological information integrated network topology           essential hubs tend co-express neighbors PCC <br /> to improve precision essential protein discovery methods            values large range negative positive. We the <br /> [28,30]. In [28], authors proposed PeC method integrating           protein YJR091C example illustrate phenomenon. <br /> edge clustering coefficient gene co-expression. In [30],                  YJR091C non-essential hub protein yeast proteome. It <br /> essential proteins explored based integration               maximal degree, 280, yeast PPI network. YJR091C <br /> network topological features types GO annotations:              ranges according DC SoECC mainly its <br /> cellular localization biological process.                              large degree. Now let co-expression neighbors. <br />    As reported [13], essential proteins tend form highly             Figure 1 shows pearson correlation coefficients YJR091C <br /> connected clusters function independently. Some                280 neighbors. The PCC values ranges 20.846 to <br /> researchers began pay attention relationship             0.802. The sum PCC values 3.37, YJR091C <br /> protein essentiality cluster property [18,31]. According         gets 451th place according sum PCC. This tells PCC is <br /> to [32], hubs yeast interactome network classified           suitable discriminate non-essential proteins like <br /> into date party hubs basis partners&#226;&#8364;&#8482; expression        YJR091C DC SoECC. <br /> profiles. This distinction suggests model organized modularity           Another motivation proposed centrality measure, <br /> for yeast proteome. Modules connected date             CoEWC, demonstrated toy network figure 2. <br /> hubs act regulators, mediators adaptors, party           Since edge clustering coefficient (ECC) measures two <br /> hubs represent integral elements modules tend            interacting nodes high probability co-clustered, <br /> function lower level organization proteome. That is,        according <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> SoECC [18,28], edges AC BC <br /> party hubs co-clustered neighbors PPI               weight determining node C&#226;&#8364;&#8482;s essentiality, edges <br /> network date hubs not. In addition, party hubs date <br />                                                                            CD1, CE1 CF1. However, goes intuition. By <br /> hubs similar probability essential [32]. Cluster- <br />                                                                            intuition, edges CD1, CE1 CF1 weight on <br /> based centrality measures, clustering coefficient sum of <br />                                                                            determining node C&#226;&#8364;&#8482;s essentiality. That is, basis co- <br /> edge clustering coefficient, effective identifying <br />                                                                            expression, reasonable clustering properties <br /> essential proteins date hubs. <br />                                                                            node&#226;&#8364;&#8482;s neighbors consideration clustering <br />    With respect various difficulties progresses, we <br />                                                                            property node itself. <br /> propose new centrality measure, named CoEWC, integrating <br />                                                                               By observing topological properties date hubs and <br /> PPI data gene expression data. CoEWC determines a <br />                                                                            party hubs, know essential proteins kinds <br /> protein&#226;&#8364;&#8482;s essentiality based high probability <br />                                                                            hubs different clustering property themselves, but <br /> to co-expressed neighbors its <br />                                                                            neighbors tend common features, i.e. clustering <br /> neighbors takes densely connected clusters. Different <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                            property. Moreover, cheerful clustering property can <br /> SoECC PeC, emphasize co-clustering relationship <br />                                                                            discriminate non-essential hubs extent. Centrality <br /> between protein neighbors, CoEWC pay attention <br /> to clustering property protein&#226;&#8364;&#8482;s neighbors          measures based idea effective essential <br /> the protein itself. As know, proteins cluster tend          proteins date hubs based ECC. Clustering <br /> share similar biological functions neighbors             coefficient (CC) measures node&#226;&#8364;&#8482;s neighbors are <br /> proteins similar functions tend co-expressed. Therefore,        connected other, used capture a <br /> we think co-expression protein interacting          node&#226;&#8364;&#8482;s clustering property. According CC, edges CD1, CE1 <br /> neighbors PPI network capture co-clustering relation-           CF1 weight determining node C&#226;&#8364;&#8482;s essentiality <br /> ship protein neighbors extent.                 edges AC BC toy network. <br /> Moreover, CoEWC takes clustering properties protein&#226;&#8364;&#8482;s <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             2                            March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                       Method Discovery Essential Proteins <br />  <br />  <br />  <br />  <br /> Figure 1. PCCs YJR091C neighbors. <br /> doi:10.1371/journal.pone.0058763.g001 <br />  <br />  <br /> New centrality measure: CoEWC                                                functional gene product. We consider gene expressions <br />    In paper, new centrality measure, CoEWC, proposed               proteins functional RNAs non-protein coding <br /> based integration PPI network gene expression data.            genes exist. For protein u, gene expressions s <br /> The basic ideas CoEWC follows: (1) A highly                    different times denoted Ge(u) = {g(u,1),g(u,2),&#226;&#8364;&#166;,g(u,s)}. The <br /> connected protein likely essential low                  probability proteins co-expressed evaluated based <br /> connected one; (2) Essential proteins tend form densely                   pearson correlation coefficient (PCC). The clustering <br /> connected clusters; (3) Essential Proteins cluster          property protein evaluated based clustering <br /> a chance co-expressed; (4) Party hubs date hubs               coefficient (CC). <br /> have similar probability essential               Pearson correlation coefficient. Pearson correlation coef- <br /> different clustering property. In CoEWC, protein&#226;&#8364;&#8482;s essentiality         ficient (PCC) measure correlation two <br /> determined number protein&#226;&#8364;&#8482;s neighbors                  variables, giving value +1 -1 inclusive. It widely <br /> probability protein co-expressed neighbors           used sciences measure strength linear <br /> well neighbors&#226;&#8364;&#8482; clustering properties.                                dependence variables. The PCC pair genes (X <br />    To method simply clearly,                    Y), encode corresponding paired proteins (u v) <br /> following definitions descriptions. The PPI network                   interacting PPI network, defined as: <br /> represented undirected graph G(V, E), node vMV <br /> represents protein edge e(u,v)ME denotes interaction                                                                      <br />                                                                                               1 Xs    g(X ,i){g(X ) : g(Y ,i){g(Y ) <br /> between proteins u v. Gene expression process              PCC(X ,Y )~                                              &#195;&#176;1&#195;&#382; <br /> which information gene used synthesis                                  s{1 i~1      s(X )           s(Y ) <br />  <br />  <br />                                                                                 Where s number samples gene expression data; <br />                                                                              g(X,i) (or g(Y,i)) expression level gene X (or Y) sample <br />                                                                              specific condition; g(X ) (or g(Y )) represents mean <br />                                                                              expression level gene X (or Y) s(X ) (or s(Y )) represents the <br />                                                                              standard deviation expression level gene X (or Y). <br />                                                                                 The pearson correlation coefficient pair proteins (u v) <br />                                                                              defined the PCC corresponding paired <br />                                                                              genes (X Y), PCC(u,v) = PCC(X,Y). If PCC(u,v) a <br />                                                                              positive value, positive linear correlation u and <br />                                                                              v. <br />                                                                                 As know, co-clustered proteins tend share similar <br /> Figure 2. A toy network. A, A1, B, B1 C nodes toy             functions proteins similar functions tend co- <br /> network. D, E, F complete sub-networks size 20, 30,       expressed. That is, proteins u v larger value of <br /> and 40. Node C connects node D, E, F respectively, say       PCC(u,v) likely cluster function <br /> D1, E1 F1.                                                               similarly. <br /> doi:10.1371/journal.pone.0058763.g002 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               3                             March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                    Method Discovery Essential Proteins <br />  <br />  <br />  <br />  <br /> Figure 3. Comparison number essential proteins detected CoEWC previously proposed centrality <br /> measures. <br /> doi:10.1371/journal.pone.0058763.g003 <br />  <br />    Clustering coefficient. In graph theory, clustering coeffi-              Where Nu set neighbors protein u ku denotes the <br /> cient measure degree nodes graph tend             number immediately connected neighbors u. <br /> cluster together. Evidence suggests real-world                   CC(u) local variable characterizes clustering <br /> networks, particular social network, create tightly knit            property protein u. A protein u larger value CC(u) is <br /> groups characterized relatively high density ties [33,34].         expected impact neighbors, has <br /> Yeast PPI network small world network. Two versions           demonstrated section 2. <br /> this measure exist: global local. The global version          CoEWC method. It proved exist a <br /> designed overall indication clustering            number protein complexes play key role carrying out <br /> network, local gives indication embedded-            biological functionality [35] essential proteins tend form <br /> ness single nodes. Here, refer local clustering               protein complexes [36]. In addition, essential proteins same <br /> coefficient.                                                               cluster tend co-expressed. It centrality measures <br />    The local clustering coefficient node graph quantifies        exploring co-clustering co-expression properties a <br /> how close neighbors clique (complete graph).            protein work task identifying essential proteins, <br /> Watts Strogatz introduced measure 1998 determine             just like PeC does. PeC outperforms previous centrality <br /> whether graph small world network [34]. The local                   measures indeed. However, reported [32], hubs be <br /> clustering coefficient protein u PPI network defined       divided date hubs party hubs, kinds hubs <br /> as                                                                         tend essential similar probability. PeC mainly <br />                                                                            emphasizes co-clustering co-expression properties a <br />                           P                                                protein neighbors, effective identify <br />                              v[Nu e(u,v)De(u,v)[E <br />                  CC(u)~                           :             &#195;&#176;2&#195;&#382;        essential proteins date hubs co-clustered <br />                               ku |&#195;&#176;ku {1&#195;&#382;=2 <br />                                                                            neighbors. According analysis toy network in <br />                                                                            section motivations, SoECC capture wrong features <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             4                            March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                       Method Discovery Essential Proteins <br />  <br />  <br /> for date hubs, make effective identifying essential         Results Discussion <br /> proteins date hubs. <br />    Although date hubs party hubs different co-                 Test data <br /> clustering property, neighbors similar               To evaluate performance proposed new centrality <br /> co-clustering property. For party hub, neighbors            measure, CoEWC, PPI network gene expression data of <br /> generally member densely connected module                 Saccharomyces cerevisiae used, characterized by <br /> that hub involves in. So exploring clustering property         <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>knockout</span> experiments widely used evaluation of <br /> each hub&#226;&#8364;&#8482;s neighbors, capture hub&#226;&#8364;&#8482;s high degree            methods essential proteins discovery. The test data used this <br /> property. For date hubs, mediate different densely                paper come [28]. We briefly follows. <br /> connected modules. Generally neighbor date hub                        The PPI data Saccharomyces cerevisiae downloaded from <br /> involves densely connected cluster, clusters             DIP database [37]. There 24,743 interactions 5093 <br /> neighbors involve different. By exploring           proteins total self-interactions repeated <br /> neighbors&#226;&#8364;&#8482; clustering property, capture high             interactions filtered. The PPI network consists 21 <br /> degree property date hub, filter hubs neighbors           components. The largest component consists 5052 proteins. <br /> seldom connected proteins. Hubs large number                 Essential proteins Saccharomyces cerevisiae collected from <br /> disconnected neighbors tend non-essential.                             databases, MIPS, SGD, DEG SGDP. Out of <br />    In order capture characteristics essential proteins based       5093 proteins PPI network, 1167 proteins are <br /> on standpoints, propose new centrality measure                essential 1165 proteins largest component <br /> which named CoEWC. We use PCC capture co-                       PPI network. <br /> clustering co-expression properties protein                   The gene expression data Saccharomyces cerevisiae retrieved <br /> neighbors, use local clustering coefficient capture high          [38], containing 6,777 gene products 36 samples total. <br /> connectivity protein neighbors&#226;&#8364;&#8482; clustering         There 4,981 proteins corresponding gene expression <br /> property.                                                                    data 112 proteins corresponding gene <br />    For protein u, CoEWC(u) defined sum PCC            expression data 6 proteins essential. For proteins <br /> between u neighbors weighted corresponding            corresponding gene expression data, simply set <br /> neighbor&#226;&#8364;&#8482;s clustering coefficient. The <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> given                zero values. <br /> equation (3). <br />                                                                              Compare CoEWC centrality measures <br />                             X                                                   In order validate performance proposed new <br />             CoEWC(u)~           v[Nu <br />                                        PCC(u,v)|CC(v)             &#195;&#176;3&#195;&#382;        centrality measure, CoEWC, carry comparison between <br />                                                                              state-of-the-art centrality measures: Degree Cen- <br />                                                                              trality (DC) [11], Sum Edge Clustering Coefficient (SoECC) <br />   Where Nu denotes set immediately connected                      [18], PeC [28] Clustering Coefficient (CC) [34]. <br /> neighbors node u PPI network.                                             The reasons choose centrality measures to <br />   From analysis <span id='am-240' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> CoEWC,                       compare follows. DC proved good <br /> CoEWC identify essential proteins party hubs                   indicator protein essentiality researchers [11,28], and <br /> and date hubs, discriminate non-essential hubs                 comparing it, want ability CoEWC to <br /> whose neighbors mainly disconnected single proteins.                     identify essential proteins hub proteins. SoECC better <br />  <br />  <br />  <br />  <br /> Figure 4. Comparison results jackknife methodology. <br /> doi:10.1371/journal.pone.0058763.g004 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               5                            March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                    Method Discovery Essential Proteins <br />  <br />  <br />  <br />  Table 1. The relationships CoEWC centrality measures predicting 200 proteins. <br />  <br />  <br />                                                                  Non-essential            Non-essential         Percentage non-essential <br />                                                                  proteins              proteins           proteins {Mi - CoEWC} low <br />  Centrality measures (Mi)          |CoEWC &gt; Mi|   |Mi - CoEWC|   {Mi -CoEWC}              {CoEWC - Mi }         CoEWC <br />  <br />  Degree Centrality (DC)            60             140            95                       33                    49.5% <br />  Clustering Coefficient (CC)        19            181            123                      50                    69.9% <br />  Sum ECC (SoECC)                78             122            70                       27                    52.9% <br />  PeC                               155            45             19                       12                    21.1% <br />  <br />  doi:10.1371/journal.pone.0058763.t001 <br />  <br />  <br /> method discovery essential proteins centrality           From figure 3 CoEWC significantly outperforms <br /> measures based network topological property [18,28]. PeC           centrality measures based network topological features <br /> is centrality measure based integration PPI data          (DC, CC SoECC) predicting essential proteins yeast <br /> and gene expression data outperforms 15 previous centrality            PPI network. CoEWC outperforms PeC predicting more <br /> measures yeast PPI network: Betweenness Centrality (BC),                essential proteins PeC does. Especially, CoEWC obtains <br /> Closeness Centrality, Subgraph Centrality (SC), Eigenvector                50% improvement DC CC predicting 500 <br /> Centrality (EC), Information Centrality (IC), Bottle Neck (BN),            proteins, obtains 20% improvement SoECC. <br /> Density Maximum Neighborhood Component (DMNC), Local                    There 4% improvement CoEWC PeC for <br /> Average Connectivity-based method (LAC), Sum ECC                        predicting 600 proteins. <br /> (SoECC), Range-Limited Centrality (RL), L-index (LI), Leader <br /> Rank (LR), Normalized a-Centrality (NC), Moduland-                     Validated jackknife methodology <br /> Centrality (MC). Therefore, compare CoEWC                        Now use jackknife methodology [39] test comparison <br /> PeC, don&#226;&#8364;&#8482;t compare mainstream centrality                     proposed centrality measure CoEWC four <br /> measures outperformed PeC yeast PPI network                  previously proposed centrality measures (DC, CC, SoECC and <br /> identifying essential proteins. PeC aims capture co- <br />                                                                            PeC). The comparison results shown figure 4. In figure 4, <br /> clustering property protein neighbors a <br />                                                                            proteins ordered highest value lowest value for <br /> topological view biological view. However, CoEWC aims to <br />                                                                            centrality measure cumulative counts essential <br /> capture properties date hubs party hubs the <br />                                                                            proteins plotted. The areas curve (AUC) for <br /> two hubs different clustering property. CC used to <br />                                                                            CoEWC centrality measures compared. In addition, <br /> show improvements obtained properly <br />                                                                            random assortments plotted comparison. <br /> integrating gene expression data, just like CoEWC does. <br />    Figure 3 gives comparison number essential                   As shown figure 4, clear sorted curve CoEWC <br /> proteins detected CoEWC previously proposed              appears better centrality measures: DC, CC <br /> centrality measures. Proteins ranked according values         SoECC. For 180 ranked proteins, CoEWC ties PeC. <br /> calculated centrality measure. For centrality                 Then sorted curve CoEWC increasingly better that <br /> measure, certain number proteins selected                  PeC increase number ranked proteins. All <br /> candidates essential proteins, number true         results centrality measures better of <br /> essential proteins determined.                                          randomized sortings. In figure 4, AUC CoEWC is <br />  <br />  <br />  <br />  <br /> Figure 5. Comparison percentage essential proteins different proteins CoEWC four <br /> centrality measures: DC, CC, SoECC PeC. <br /> doi:10.1371/journal.pone.0058763.g005 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             6                           March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                      Method Discovery Essential Proteins <br />  <br />  <br />  <br />  Table 2. List 26 proteins predicted CoEWC are <br />  ignored centrality measures DC, CC, SoECC, and <br />  PeC predicting 200 proteins. <br />  <br />  <br />  Rank     Protein Name      Degree       CoEWC      Essentiality <br />  <br />  104      YDR365C           23           1.698583   essential <br />  124      YDL232W           18           1.503577   essential <br />  127      YJL033W           19           1.49437    essential <br />  128      YGL099W           14           1.492625   essential <br />  130      YBR234C           23           1.473234   essential <br />  131      YIL075C           32           1.472131   essential <br />  139      YLR200W           10           1.435959   non-essential <br />  145      YDL087C           23           1.401966   essential <br />  147      YKL095W           38           1.390057   essential <br />  151      YHR081W           5            1.358778   non-essential <br />                                                                           Figure 6. YOL142W interacting neighbors. <br />  152      YPR088C           14           1.351337   essential             doi:10.1371/journal.pone.0058763.g006 <br />  154      YOL094C           37           1.327599   essential <br />  156      YHL030W           21           1.310346   non-essential         detected CoEWC centrality measure Mi. {Mi - <br />  158      YOR259C           28           1.282556   essential             CoEWC} (or {CoEWC -Mi}) denotes set proteins identified <br />  161      YBL041W           9            1.275247   essential             Mi (or CoEWC) CoEWC (Mi), |Mi - CoEWC| is <br />                                                                           number proteins identified Mi CoEWC. <br />  163      YNL182C           21           1.268987   essential <br />                                                                              From table 1, common proteins identified by <br />  170      YMR314W           16           1.246819   essential <br />                                                                           CoEWC DC, CC 30%, common <br />  178      YBR126C           29           1.219773   non-essential         proteins predicted CoEWC SoECC 40%, and <br />  179      YOL142W           6            1.219508   essential             common proteins predicted CoEWC PeC less <br />  181      YBL023C           14           1.211368   essential             80%. The small overlap predicted proteins of <br />  187      YNL290W           34           1.176549   essential             CoEWC DC, CC shows CoEWC special centrality <br />                                                                           measure different classical centrality <br />  190      YFL008W           24           1.156859   essential <br />                                                                           measures. In addition, investigated non-essential proteins <br />  191      YPL012W           20           1.153688   essential <br />                                                                           predicted centrality measures, 50% <br />  193      YER025W           26           1.140787   essential             non-essential proteins predicted network <br />  194      YOR210W           16           1.138906   essential             topology-based centrality measures (DC, CC SoECC) are <br />  199      YKL068W           35           1.119936   non-essential         low values CoEWC (less 0.128) are <br />                                                                           21.1% non-essential proteins predicted PeC very <br />  doi:10.1371/journal.pone.0058763.t002                                    low values CoEWC (less 0.128). <br />                                                                              Secondly, evaluate different proteins identified by <br /> 4.3174e+005, AUC PeC 4.0450e+005. This tells             CoEWC centrality measures. Figure 5 gives <br /> that CoEWC effective PeC task identifying         number proteins predicted different <br /> essential proteins. Therefore, idea capturing                proteins identified CoEWC identified DC, CC, <br /> properties date hubs party hubs using co-              SoECC PeC. As shown figure 5, percentage essential <br /> expression protein neighbors weighted                proteins identified CoEWC consistently higher that <br /> corresponding neighbor&#226;&#8364;&#8482;s clustering coefficient better       identified centrality measures different <br /> only capturing co-clustering property protein           proteins them. Take CC example, the <br /> neighbors.                                                                largest different number proteins CoEWC. Out the <br />                                                                           200 proteins, 181 proteins differently identified CC and <br /> Analysis differences CoEWC the <br /> compared centrality measures <br />    To analyze CoEWC performs               Table 3. Information neighbors YOL142W. <br /> identification essential proteins, study relationship and <br /> difference compared centrality measures <br />                                                                            Proteins           PCC            CC           Essentiality <br /> (DC, CC, SoECC, PeC) predicting small fraction of <br /> proteins. The 200 proteins selected centrality            YDR280W            0.8046         0.3399       essential <br /> measure.                                                                   YER025W            0.639          0.1354       essential <br />    Firstly, compare CoEWC centrality                YNL265C            20.357         0.1648       non-essential <br /> measures (DC, CC, SoECC PeC) investigating many <br />                                                                            YGR195W            0.771          0.4083       essential <br /> proteins predicted CoEWC the <br />                                                                            YGR095C            0.7414         0.3897       essential <br /> other centrality measures. The number overlaps between <br /> CoEWC centrality measures shown                 YOL021C            0.8391         0.375        essential <br /> table 1. |CoEWC &gt; Mi| denotes number common proteins                doi:10.1371/journal.pone.0058763.t003 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            7                                March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                     Method Discovery Essential Proteins <br />  <br />  <br />  <br />  <br /> Figure 7. Properties 125 neighbors YLR295C. <br /> doi:10.1371/journal.pone.0058763.g007 <br />  <br /> by CoEWC, respectively. Out 181 proteins CoEWC,                 neighbors PPI network, rank proteins according to <br /> 72.4% proteins essential, 32% proteins            SoPCC. YLR295C gets rank 121 according SoPCC and <br /> 181 proteins CC essential. Then PeC example,              gets rank 123 according PeC. Figure 7 gives the <br /> which smallest difference CoEWC. There 45                  properties YLR295C 125 neighbors captured PCC, <br /> different proteins identified CoEWC PeC. Out 45                CC ECC. In figure 7, neighbor proteins 24 <br /> different proteins, CoEWC identified 73.3% essential              neighbor IDs essential, proteins non- <br /> proteins PeC identified 57.8% essential                essential. <br /> proteins. The similar results obtained DC                  From distributions PCC, CC ECC figure 7, can <br /> SoECC.                                                                      quarter neighbors co-clustered with <br />    There 26 proteins predicted CoEWC               YLR295C (with ECC value equals 1), few <br /> included 200 proteins centrality        neighbors large CC values. Almost ECC values the <br /> measures. These 26 proteins shown table 2. In table 2,            neighbors larger zero, half neighbors&#226;&#8364;&#8482; <br /> show following information 26 proteins: rank              CC values zero. Among YLR295C&#226;&#8364;&#8482;s interacting proteins, <br /> CoEWC, protein name, degree, value CoEWC essentiality.               essential proteins tend non-zero CC values, accords <br /> As shown table 2, 26 proteins 80.8% essential.            assumption essential proteins tend co-clustered <br />    Take YOL142W example. YOL142W essential                      neighbors. According <span id='am-241' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> SoECC, <br /> protein degree 6. The interactions                    sum ECC, YLR295C&#226;&#8364;&#8482;s SoECC value large due <br /> YOL142W neighbors shown figure 6. To                 high degree. According definition PeC, the <br /> study characteristic YOL142W neighbors,              sum product ECC PCC, PeC value YLR295C <br /> the following information neighbors: PCC value, CC value,            considerably smaller SoECC negative <br /> and essentiality table 3. From table 3, 5            values PCC. Moreover, according definition CoEWC, <br /> neighbors 6 neighbors essential proteins,           sum product PCC CC, CoEWC <br /> that YOL142W co-expressed 5 neighbors            value YLR295C smaller SoECC PeC, due <br /> also essential. All CC values neighbors significantly        negative PCC values smaller CC values. From figure 7, <br /> larger average CC value PPI network             understand CoEWC discriminate <br /> is 0.097. Table 3 tells co-clustered essential proteins        YLR295C non-essential DC, SoECC PeC cannot. <br /> tend co-expressed CoEWC capture                        Table 4 shows list non-essential proteins high <br /> property well.                                                              degree low value CoEWC. In order compare with <br />    Take non-essential protein, YLR295C, example.              centrality measures, values DC, CC, <br /> YLR295C 125 neighbors, 24 essential.              SoECC PeC. Since values predicted different centrality <br /> YLR295C gets rank 16, 2388, 8 according DC, CC,               measures comparable directly, 1110th <br /> and SoECC, respectively. According definition DC, CC              proteins&#226;&#8364;&#8482; values, sorted descending order according to <br /> and SoECC corresponding ranks YLR295C according                  centrality measure, reference values. The reference values <br /> to centrality measures, conclude YLR295C            DC, CoEWC, CC, SoECC PeC 12, 0.127, 0.1428, <br /> is hub protein co-clustered neighbors,       3.057 0.55, respectively. From table 4, these <br /> and connections neighbors (its          non-essential proteins discriminated DC and <br /> CC value 0.0017). It obvious YLR295C              SoECC. By considering co-expression properties a <br /> discriminated DC SoECC.                                              protein interacting proteins, CoEWC PeC have <br />    In addition, order compare CoEWC PeC,              improved discrimination ability non-essential <br /> also compute sum PCC (SoPCC) protein               proteins. As shown table 4, non-essential proteins with <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              8                           March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                         Method Discovery Essential Proteins <br />  <br />  <br />  <br />  Table 4. List non-essential proteins high degree low value CoEWC. <br />  <br />  <br />  Protein Name                Essentiality               DC           CoEWC                  CC                    SoECC               PeC <br />  <br />  YCL018W                     non-essential              156          0.0723                 0.0244                21.2481             0.1766 <br />  YBR127C                     non-essential              113          0.0502                 0.0207                11.7541             20.2884 <br />  YMR106C                     non-essential              110          -0.2571                0.0283                18.9553             20.543 <br />  YLR288C                     non-essential              99           20.1364                0.0033                31.4151             3.3742 <br />  YLR191W                     non-essential              97           21.5164                0.0215                24.9253             21.6514 <br />  YLR447C                     non-essential              95           20.0429                0.0035                26.2248             2.4992 <br />  YOL055C                     non-essential              93           0.0942                 0.0140                8.1786              20.3599 <br />  YHR135C                     non-essential              84           20.2160                0.0203                13.849              20.3505 <br />  YGR040W                     non-essential              79           0.1058                 0.0207                8.6171              0.3374 <br />  YLR453C                     non-essential              78           0.0411                 0.003                 23.4248             0.3608 <br />  YER118C                     non-essential              72           20.2675                0.0274                14.6054             20.2627 <br />  YDL059C                     non-essential              67           0.1163                 0.0407                7.3506              20.0661 <br />  YCL027W                     non-essential              67           0.0730                 0.009                 18.7385             4.1962 <br />  YBL085W                     non-essential              67           20.2364                0.0212                13.4245             20.8018 <br />  YGR254W                     non-essential              67           20.0167                0.0298                7.08                20.0244 <br />  YAR014C                     non-essential              65           20.20841               0.016346              13.29514            22.5911 <br />  YDR171W                     non-essential              61           20.28158               0.023497              4.637564            0.0205 <br />  YHR140W                     non-essential              60           20.55414               0.249718              30.04586            22.13 <br />  YML048W                     non-essential              60           20.44267               0.136723              18.25437            20.4479 <br />  YGR262C                     non-essential              60           20.09723               0.029379              7.759587            0.4257 <br />  YJL098W                     non-essential              59           20.26835               0.04851               10.58723            20.5065 <br />  YLR096W                     non-essential              58           20.27887               0.047792              11.02443            20.6211 <br />  YJL095W                     non-essential              58           20.03866               0.050817              13.54987            0.7098 <br />  YGL237C                     non-essential              57           20.39525               0.025063              10.00474            0.8943 <br />  YNL135C                     non-essential              57           0.046016               0.022556              8.115793            0.1006 <br />  YDR386W                     non-essential              57           20.65712               0.030702              6.746308            21.4038 <br />  YCL040W                     non-essential              55           20.11312               0.020875              5.221536            0.0757 <br />  YDL101C                     non-essential              55           0.069512               0.041077              9.862956            20.4066 <br />  YGL173C                     non-essential              52           20.80869               0.032428              5.137779            20.8367 <br />  YER179W                     non-essential              50           20.14783               0.04                  6.687021            20.357 <br />  YKL065C                     non-essential              50           20.3737                0.173061              15.15515            20.7295 <br />  <br />  doi:10.1371/journal.pone.0058763.t004 <br />  <br />  <br /> a high degree consistently low value CoEWC. From               features essential proteins distinguish non- <br /> the analysis, CoEWC help filter false           essential proteins. <br /> predictions centrality measures. So integration PCC              To tackle difficulties, propose new centrality <br /> and CC effective prediction essential proteins.                 measure, named CoEWC, based integration PPI data <br />                                                                               gene expression data. CoEWC aims capture common <br /> Conclusions                                                                   features essential proteins date hubs party hubs by <br />                                                                               integrating PCC CC together. CoEWC applied PPI <br />    With large PPI data available species,              network Saccharomyces cerevisiae. The experimental results show <br /> the discovery essential proteins network level          CoEWC significantly outperforms network topology- <br /> hot topic. Many network topology-based centrality measures                based centrality measures: DC, CC SoECC, that <br /> the discovery essential proteins proposed. However,              CoEWC outperforms PeC, currently proposed centrality <br /> the currently available PPI networks species                     measure based integration PPI data gene <br /> incomplete (false negatives) noisy (high false positives).           expression data. <br /> At time, network topology-based methods                     Although CoEWC performs discovery essential <br /> depend reliability available protein-protein interac-           proteins, space improve prediction <br /> tions sensitive network. Moreover,                   precision. First, integration PCC CC simple in <br /> essential proteins distinct clustering properties, i.e. date        paper, exist abstruse relationship between <br /> hubs party hubs, time essential non-essential             PCC CC. Second, exist excellent <br /> proteins common features, i.e. high degree              method capture distinct properties essential <br /> hub proteins. It challenging capture true distinct <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               9                              March 2013 | Volume 8 | Issue 3 | e58763 <br />                                                                                                                           Method Discovery Essential Proteins <br />  <br />  <br /> proteins non-essential proteins. Finally, gene                              Author Contributions <br /> expression data, protein related data, biological <br />                                                                                             Participated revising draft: JX. Conceived designed the <br /> process, domain information, localization,                               experiments: XZ WXX. Performed experiments: XZ WXX. Analyzed <br /> valuable task identifying essential proteins.                                    data: XZ WXX. Contributed reagents/materials/analysis tools: XZ <br />                                                                                             WXX. Wrote paper: XZ. <br />  <br /> References <br />  1. Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, et al. (1999)              20. Lin CY, Chin CH, Wu HH, Chen SH, Ho CW, et al. (2008) Hubba: hub objects <br />     Functional characterization S. cerevisiae genome gene deletion                analyzer-a framework interactome hubs identification network biology. <br />     parallel analysis. Science 285: 901&#226;&#8364;&#8220;906.                                                    Nucleic Acids Res, 36:438&#226;&#8364;&#8220;443. <br />  2. Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, et al. (2003) Systematic              21. Joy MP, Brock A, Ingber DE, Huang S (2005) High-betweenness proteins the <br />     functional analysis Caenorhabditis elegans genome using RNAi. Nature     <br /> </body></html>