<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 10 June 2013                                                 Nucleic Acids Research, 2013, Vol. 41, No. 15 7485&#226;&#8364;&#8220;7500 <br />                                                                                                                   doi:10.1093/nar/gkt490 <br />  <br />  <br /> Structure-function analysis 50 end yeast U1 <br /> snRNA highlights genetic interactions the <br /> Msl5 Mud2 branchpoint-binding complex other <br /> spliceosome assembly factors <br /> Beate Schwer1,*, Jonathan Chang2 Stewart Shuman2,* <br /> 1 <br />     Microbiology Immunology Department, Weill Cornell Medical College, New York, NY 10065, USA and <br /> 2 <br />     Molecular Biology Program, Sloan-Kettering Institute, New York, NY 10065, USA <br />  <br /> Received March 28, 2013; Revised May 7, 2013; Accepted May 12, 2013 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                         budding yeast Saccharomyces cerevisiae been <br />                                                                                  elucidated genetically biochemically (1&#226;&#8364;&#8220;3). The yeast <br /> Yeast pre-mRNA splicing initiates formation                             spliceosome transits assembly, activation, cataly- <br /> complex comprising U1 snRNP bound 50                                      sis disassembly steps programmed U1, <br /> splice site (50 SS) Msl5 Mud2 heterodimer                                U2, U4, U5 U6 snRNPs army proteins that <br /> engaged branchpoint (BP). Here, present                                interact snRNPs pre-mRNA. The &#239;&#172;?rst <br /> a mutational analysis U1 snRNA, shows                               stage spliceosome assembly entails formation a <br /> that enlarging 50 leader                                complex comprising U1 snRNP bound intronic <br /> TMG cap 3ACUUAC8 motif anneals                                   50 splice site (50 SS; consensus sequence: 50 -GUAUGU) and <br /> the 50 SS tolerated, tight constraints                           Msl5 Mud2 protein heterodimer engaged the <br /> the downstream spacer 3ACUUAC8                                       intron branchpoint (BP) (consensus sequence: 50 -UACU <br /> helix 1 U1 fold. We exploit U1 alleles 50                            AAC). Bridging interactions U1 snRNP and <br />                                                                                  Msl5 Mud2 serve stabilize complex prepare a <br /> extensions, variations 3ACUUAC8 motif, <br />                                                                                  scaffold recruitment U2 snRNP BP. <br /> downstream mutations longer helix 1                                        The yeast U1 snRNP consists trimethylguanosine <br /> discover new intra-snRNP synergies U1                                       (TMG)-capped 568-nt U1 snRNA, toroidal Sm protein <br /> subunits Nam8 Mud1 trimethyl-                                        ring composed seven subunits (present U2, <br /> guanosine (TMG) cap. We novel mutations                                 U4 U5 snRNPs), 10 additional protein subunits <br /> in U1 snRNA bypass essentiality                                  unique yeast U1 snRNP: Prp39, Prp40, Snu71, <br /> DEAD-box protein Prp28. Structure-guided muta-                                   Snu56, Snp1, Mud1, Luc7, Prp42, Nam8 Yhc1 <br /> genesis Msl5 distinguished essential amino                               (4&#226;&#8364;&#8220;10). Yeast U1-speci&#239;&#172;?c subunits Snp1, Mud1 and <br /> acids contact BP sequence                               Yhc1 conserved human U1 snRNP U1-70K, <br /> BP-binding residues inessential. We report                              U1-A U1-C, respectively. The 50 sequence yeast <br /> new synthetic genetic interactions U1 snRNP                               U1 RNA&#226;&#8364;&#8221;m2,2,7GpppAUACUUACC&#226;&#8364;&#8221;is conserved in <br />                                                                                  smaller human U1 snRNA (164 nt) and <br /> with Msl5 Mud2 nuclear cap- <br />                                                                                  contains hexanucleotide motif (underlined) complemen- <br /> binding subunit Cbc2. Our results fortify idea                               tary consensus 50 SS. This U1 50 sequence anneals <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> that spliceosome assembly occur distinct                                 pre-mRNA nucleate early assembly complex. <br /> genetically     buffered    microscopic     pathways                             The U1 snRNP protein subunits interact U1 RNA <br /> involving cross-intron-bridging interactions                              sequences and/or secondary structures, contact pre- <br /> U1 snRNP 50 SS complex Mud2 Msl5 BP                                     mRNA contact protein components the <br /> complex.                                                                         spliceosome initial assembly (11&#226;&#8364;&#8220;13). <br />                                                                                  Additional non-snRNP elements aid U1 pre-mRNA <br />                                                                                  complex assembly, including pre-mRNA m7G <br /> INTRODUCTION                                                                     cap structure association yeast nuclear <br /> The composition, function dynamics pre-                               cap-binding complex (CBC: Sto1 Cbc2 heterodimer) <br /> mRNA splicing machinery (the &#226;&#8364;&#732;spliceosome&#226;&#8364;&#8482;)                               (14). The U1 snRNP ultimately displaced the <br />  <br />  <br /> *To correspondence addressed. Tel: +212 746 6518; Email: bschwer@med.cornell.edu <br /> Correspondence addressed Stewart Shuman. Tel: +212 639 7145; Fax: +212 717 3623; Email: s-shuman@ski.mskcc.org <br />  <br /> &#195;&#376; The Author(s) 2013. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/3.0/), permits non-commercial re-use, distribution, reproduction medium, provided original work properly cited. For commercial <br /> re-use, contact journals.permissions@oup.com <br />  7486 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br /> pre-mRNA U1 U2-containing spliceosome point              spacing U1 50 SS RNA duplex helix 1 is <br /> when U5 U4 U6 tri-snRNP complex joins en route              important U1 snRNP function. Changes increase <br /> to forming pre-mRNA U2 U5 U6 spliceosome.                     <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> helix 1 benign se synthetic in <br /> Dissociation U1 snRNP thought triggered          combination agents spliceosome assembly. <br /> the essential DEAD-box protein Prp28 (15,16), acting         Viable mutations 3ACUUAC8 element display <br /> disrupt short U1:50 SS RNA duplex remodel                allele-speci&#239;&#172;?c synergies. We indentify novel mutations <br /> protein&#226;&#8364;&#8220;RNA contacts 50 SS (or both).                    U1 snRNA bypass essentiality Prp28. Our <br />    Although yeast U1 RNA essential viability, sub-       results reveal rich network intramolecular inter- <br /> stantial chunks U1 primary structure (more          molecular genetic interactions yeast U1 snRNP, <br /> half) dispensable cell growth (17,18). This situation   especially BP-binding protein Msl5. <br /> inspired elegant highly successful genetic screen by <br /> the Rosbash laboratory yeast mutations caused <br /> synthetic lethality viable U1 snRNA           MATERIALS AND METHODS <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> large internal deletion                 U1 expression plasmids mutants <br /> combined G27A change stem-loop-1 (SL1) <br /> sequence interacts U1-70K (19). The &#226;&#8364;&#732;Mutant-          A 1.3-kb DNA segment bearing SNR19 (U1) gene was <br /> U1-Die&#226;&#8364;&#8482; &#226;&#8364;&#732;MUD&#226;&#8364;&#8482; screen identi&#239;&#172;?ed genes encoding            ampli&#239;&#172;?ed S. cerevisiae genomic DNA PCR using <br /> four U1-speci&#239;&#172;?c subunits: Mud1, Snu56/Mud10,             forward primer introduced XhoI site 550 nt <br /> Nam8/Mud15 Prp42/Mud16 (5,19). The screen              upstream transcription start site (+1) U1 <br /> identi&#239;&#172;?ed Cbc2/Mud13 subunit nuclear CBC             snRNA reverse primer downstream HindIII <br /> the Mud2 subunit yeast BP-binding protein                site position +755. The DNA fragment inserted <br /> complex (Mud2 yeast homolog mammalian              centromeric plasmids pRS316 (URA3) pRS415 <br /> U2AF65) (20,21). The &#239;&#172;?ndings Mud1, Nam8, Cbc2              (LEU2). The resulting yeast expression plasmids p316-U1 <br /> and Mud2 inessential yeast vegetative        p415-U1 include 568-nt U1 snRNA sequence plus <br /> growth immediately highlighted network genetically         550 187 nt upstream downstream sequences. <br /> buffered functions centered U1 snRNP              Mutations (insertions, deletions single nucleotide <br /> early spliceosome assembly. The Rosbash laboratory              changes) introduced U1 plasmid two- <br /> then bootstrapped viable mud2  mutant execute          stage PCR overlap extension mutagenic primers. <br /> &#226;&#8364;&#732;Mud2 synthetic lethal&#226;&#8364;&#8482; (MSL) screen identify       The U1 genes sequenced completely con&#239;&#172;?rm <br /> Msl5, yeast BP-binding protein, Msl1, U2B00         unwanted changes acquired ampli&#239;&#172;?- <br /> subunit U2 snRNP (22,23).                                cation cloning. <br />    Additional directed genome-wide synthetic genetic <br /> array analyses greatly expanded network mu-         Msl5 expression plasmids mutants <br /> tational synergies involving: U1 snRNP subunits             CEN LEU2 plasmids bearing wild-type mutated <br /> Mud1, Nam8, Luc7 Yhc1; TMG cap structure                MSL5 genes control native MSL5 <br /> of U1 snRNA (via inactivation TMG synthase Tgs1);            promoter described (31). New missense muta- <br /> m7G cap binding nuclear CBC; Msl5 Mud2               tions L169A, R172A, L176A, I189A, R190A, L256A and <br /> BP binding complex (24&#226;&#8364;&#8220;32). Such genetic interactions           L259A KH-QUA2 domain P97A, P98A, <br /> among individually dispensable players (or            Y100A Y100F PPxY motif introduced <br /> benign mutations essential factors) act            MSL5 two-stage PCR overlap extension mu- <br /> common pathway meet operational de&#239;&#172;?nition redun-          tagenic primers. The PCR products digested and <br /> dancy. Genetic redundancy does necessitate         inserted pRS415-based MSL5 expression <br /> synthetic interactor proteins RNA elements perform           plasmid (31). The MSL5 genes sequenced completely <br /> the task, suggests spliceosome          con&#239;&#172;?rm unwanted changes acquired <br /> assembly accomplished stabilized different        ampli&#239;&#172;?cation cloning. CEN HIS3 MSL5 <br /> microscopic sub-pathways.                                       plasmids constructed subcloning 2.2-kb MSL5 <br />    In present study, interrogate structure&#226;&#8364;&#8220;function <br />                                                                 fragments (excised XhoI SacI pRS415- <br /> relations 50 end yeast U1 snRNA, studying the <br />                                                                 MSL5) pRS413. <br /> effects 50 extensions; nucleobase changes 3ACUU <br /> AC8 sequence pairs 50 SS; insertions, <br />                                                                 Yeast strains tests U1 function vivo <br /> deletions nucleobase changes &#239;&#172;&#8218;anking <br /> 9 <br />   CUU11 segment interacts U1-C connects           To develop plasmid shuf&#239;&#172;&#8218;e assay gauging mutational <br /> the U1 50 SS RNA duplex helix 1 U1 RNA fold           effects U1 function, generated U1D strain that <br /> (4,8). We &#239;&#172;?nd extending segment upstream        relies viability maintenance wild-type U1 <br /> 3 <br />   ACUUAC8 element effect growth syn-           gene CEN URA3 plasmid, p316-U1. In brief, we <br /> thetically lethal sick mutations affecting TMG          &#239;&#172;?rst replaced U1 locus position +1 +532 <br /> capping, m7G cap-binding CBC, U1 snRNP protein               kanMX cassette BY4743 diploid strain and <br /> Nam8 Msl5 Mud2 BP-binding complex.                      transformed heterozygous diploid p316-U1. <br /> Insertions 1 3 nt downstream 3ACUUAC8              The diploid sporulated, asci dissected and <br /> viable, display synthetic phenotypes; contrast, in-      haploid U1D Ura+ progeny recovered. U1D (p316- <br /> sertions  5 nucleotides lethal, suggesting      U1) cells resistant G418 unable grow on <br />                                                                      Nucleic Acids Research, 2013, Vol. 41, No. 15 7487 <br />  <br />  <br /> medium containing 0.75 mg/ml 5-&#239;&#172;&#8218;uoroorotic acid              RESULTS <br /> (FOA). To assay function wild-type mutated        Effect U1 50 extensions distance cap the <br /> U1 alleles, U1D (p316-U1) cells transfected CEN    3 <br />                                                                ACUUAC8 sequence <br /> LEU2 U1 plasmids. Individual Leu+ transformants were <br /> selected streaked agar medium containing              Our 50 end yeast U1 snRNA sparked <br /> FOA. The plates incubated 20, 30 37  C,       earlier &#239;&#172;?ndings (i) nuclear CBC stoi- <br /> mutants failed form macroscopic colonies      chiometric subunit U1 snRNP yeast tgs1  cells <br /> temperatures 10 d deemed lethal. Individual       lack methyltransferase enzyme converts m7G <br /> FOA-resistant colonies viable U1 alleles           caps TMG caps (ii) cold-sensitive (cs) growth <br /> grown mid-log phase yeast extract, peptone,            <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> yeast tgs1  cells rescued mutations <br /> dextrose (YPD) broth adjusted A600           cap-binding pocket nuclear CBC no <br /> values. Aliquots (3 ml) serial 10-fold dilutions     effect growth se (7,32). Our inference that <br /> spotted YPD agar plates, incubated        ectopic binding CBC residual m7G U1 snRNA <br /> at temperatures ranging 18 37  C.                    cap key factor tgs1  cs phenotype that <br />    We developed plasmid shuf&#239;&#172;&#8218;e assays test       reducing CBC af&#239;&#172;?nity m7G U1 snRNA cap <br /> mutational effects U1 function mud2 , tgs1 ,           restored normal growth. Because U1 snRNA cap is <br /> nam8 , mud1 , swt21 , brr1 , cbc2-Y24A, swm2             located just 2 nt upstream 3ACUUAC8 sequence <br /> tgs1D cbc2-Y24A cells (7,27,29&#226;&#8364;&#8220;32) using standard genetic    pairs pre-mRNA 50 SS, considered that <br /> manipulations mating, sporulation dissection. We      steric hindrance CBC U1 50 end interfere <br /> thereby generated mud2  U1D (p316-U1), nam8  U1D             U1 function, case increasing physical <br /> (p316-U1), mud1  U1D (p316-U1), tgs1  U1D (p316-             distance CBC 3ACUUAC8 sequence <br /> U1), swt21  U1D (p316-U1), swm2  U1D (p316-U1),              suppress tgs1  cs defect. <br /> brr1  U1D (p316-U1), cbc2-Y24A U1D (p316-U1)                 To test idea, extended 50 sequence U1 <br /> tgs1  cbc2-Y24A U1D (p316-U1) cells,              gene 5, 10, 15, 20, 25 30 nt, depicted Figure 1A. <br /> unable grow FOA-containing medium, unless         The wild-type (WT) 50 -extended U1 alleles the <br /> had transformed wild-type U1 functional       control native U1 promoter placed CEN <br /> U1 mutant allele CEN LEU2 plasmid FOA            LEU2 plasmids tested function vivo plasmid <br /> selection. To investigate genetic interactions U1         shuf&#239;&#172;&#8218;e yeast strain deleted chromosomal U1 <br /> mutants essential MSL5 PRP28 genes,          locus bearing WT U1 gene CEN URA3 <br /> &#239;&#172;?rst generated heterozygous msl5 /MSL5 U1D/U1            plasmid. All 50 -extended U1 alleles supported <br /> prp28 /PRP28       U1D/U1       diploids       crossing   growth yeast U1  cells medium containing FOA, <br /> msl5D::natMX p316-MSL5 prp28D::natMX p316-               drug selects CEN URA3 U1 plasmid. <br /> PRP28 cells U1D::kanMX p316-U1 cells             The U1 +5, +10, +15, +20, +25 +30 strains grew as <br /> opposite mating type, selecting diploids YPD              U1 WT cells 20, 25, 30, 34 37  C, gauged <br /> medium containing G418 clonNat plating          spotting serial dilutions respective strains on <br /> to FOA-containing medium. The heterozygous diploids          YPD agar (Figure 1B). To assess newly <br /> were transformed CEN URA3 U1 MSL5                  inserted sequences 50 end U1 gene were <br /> (p316-U1-MSL5) CEN URA3 U1 PRP28 (p316-U1-                transcribed U1 snRNA, performed primer ex- <br /> PRP28) plasmids. [In plasmids, U1 gene ( 550       tension analysis total RNA isolated yeast cells <br /> to +755) arranged head-to-head con&#239;&#172;?guration     bearing WT 50 -extended U1 alleles (Figure 1C). <br /> the MSL5 gene PRP28 gene ( 520 +2120).]            A 50 32P-labeled DNA oligonucleotide complementary to <br /> Ura+ heterozygous diploids subjected sporulation     yeast U1 snRNA extended reverse transcriptase to <br /> and tetrad dissection, haploid prp28  U1D        yield discrete cDNA corresponding size the <br /> (p316-U1-PRP28 msl5  U1D (p316-U1-MSL5)                  distance (in nucleotides) primer 50 end 50 <br /> progeny recovered. These cells unable grow      end U1 snRNA. A second 50 32P-labeled DNA <br /> on FOA medium, double-deletion strains      oligonucleotide complementary U2 snRNA was <br /> complemented cotransformation p(CEN LEU2             included reverse transcription reactions a <br /> U1) plus p(CEN HIS3 MSL5) p(CEN HIS3               control. Denaturing polyacrylamide gel electrophoresis <br /> PRP28). <br />                                                              (PAGE) analysis primer extension products <br />                                                              revealed 50 ends U1 snRNAs shifted <br /> Primer extension analysis U1 50 ends                      serially &#226;&#8364;&#732;upstream&#226;&#8364;&#8482; 5-nt intervals, expected the <br /> U1D cells bearing CEN LEU2 U1 plasmids grown         U1 DNA sequences. Thus, DNA additions did not <br /> YPD broth mid-log phase. Cellular RNA isolated        alter site transcription initiation directed the <br /> using RNeasy kit (Qiagen). Aliquots (20 mg RNA)        50 -&#239;&#172;&#8218;anking U1 gene promoter. The amounts U1 <br /> were used templates reverse transcriptase-catalyzed   cDNA synthesized total RNAs derived the <br /> extension 50 32P-labeled primers 50 -d(GAATGGAAAC         WT 50 -extended U1 yeast strains similar, signify- <br /> GTCAGCAAACAC) 50 -d(CTTAAAAAGTCTCTT                      ing steady-state levels U1 snRNA not <br /> CCCGTC) complementary U1 U2                  affected 50 leader sequences. The 50 ends and <br /> snRNA, respectively. The primer extension reactions          steady-state levels U2 snRNAs unaltered the <br /> were performed described previously (33).                 U1 mutations. <br />  7488 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br />                                                                           ectopic binding CBC U1 m7G cap tgs1  <br />                                                                           strain severe cap distanced the <br />                                                                           body U1 benign 50 RNA leader <br />                                                                           sequence (e.g. distance allows freer <br />                                                                           access CBC U1 cap growth temperatures). <br />                                                                           If case, thought cbc2-Y24A <br />                                                                           mutation cap-binding pocket nuclear CBC, <br />                                                                           effect se yeast cell growth <br />                                                                           (Figure 2; cbc2-Y24A U1-WT) suppresses the <br />                                                                           tgs1  cs phenotype (Figure 2; tgs1  cbc2-Y24A U1-WT), <br />                                                                           ameliorate synergy tgs1  50 U1 <br />                                                                           extensions. A necessary step en route issue to <br />                                                                           query effects U1 50 extensions cbc2-Y24A <br />                                                                           background. This experiment revealed gradient syn- <br />                                                                           thetic sick synthetic lethal interactions cbc2- <br />                                                                           Y24A U1 mutant series, the+5,+10,+20 <br />                                                                           +25 extensions caused cs growth defects increasing <br />                                                                           severity (re&#239;&#172;&#8218;ected upward shift restrictive <br />                                                                           temperature) +15 +30 extensions lethal <br />                                                                           temperatures cbc2-Y24A background <br />                                                                           (Figure 2; cbc2-Y24A). Tyr24 Cbc2 equivalent <br />                                                                           Tyr20 mammalian Cbc2 forms p&#226;&#8364;&#8220;cation <br />                                                                           stack m7G nucleobase confer high-af&#239;&#172;?nity <br />                                                                           cap binding. The synergistic effects cbc2-Y24A with <br />                                                                           altered U1 snRNAs seen resonate the <br />                                                                           recently reported synthetic lethality sickness cbc2- <br />                                                                           Y24A null mutations proteins involved early <br />                                                                           steps spliceosome assembly (Nam8, Mud1, Swt21, <br />                                                                           Mud2, Ist3 Brr1) benign muta- <br />                                                                           tions Msl5, essential BP-binding protein (32). <br />                                                                           These combinatorial effects highlight contribu- <br /> Figure 1. U1 snRNAs extended 50 ends functional. (A) The         tions spliceosome assembly CBC engaged 50 <br /> DNA sequences shown 50 ends wild-type U1 (WT)          pre-mRNA cap genetically buffered compo- <br /> the mutant variants +5, +10 forth, named according the <br /> number nucleotides inserted upstream 3ACTTAC8 segment <br />                                                                           nents splicing machinery. The present results <br /> (highlighted gray) pairs intron 50 SS. (B) The growth    identify U1 snRNA 50 end novel component of <br /> phenotypes U1  p(CEN LEU2 U1) cells bearing indicated U1           genetic network. In light aforementioned <br /> alleles assessed follows. Liquid cultures grown mid-log   &#239;&#172;?ndings, surprising growth phenotypes <br /> phase 30  C adjusted A600. Aliquots (3 ml) serial   50 -extended U1 alleles tgs1D cbc-Y24A back- <br /> 10-fold dilutions cells spotted YPD agar. The plates were <br /> incubated indicated temperatures photographed 2 d        ground virtually identical tgs1  <br /> (30, 34 37  C), 3 d (25  C) 4 d (20  C). (C) Primer extension      background (Figure 2). <br /> analyses 32P-labeled primers complementary U1 snRNA                  To delineate additional genetic interactions U1 50 <br /> (nt 161&#226;&#8364;&#8220;182 ) U2 snRNA (nt 140&#226;&#8364;&#8220;160 ) performed using           extensions, constructed series yeast strains in <br /> template total cellular RNA isolated indicated U1  p(CEN <br /> LEU2 U1) strains. The reaction products analyzed denaturing <br />                                                                           genes encoding inessential splicing factors <br /> PAGE visualized autoradiography. The sizes (nt) 32P-labeled     deleted U1  p(CEN URA3 U1) background, <br /> marker DNAs analyzed parallel indicated left.         allowing tests synergy plasmid shuf&#239;&#172;&#8218;e. <br />                                                                           Although ablation U1 snRNP subunit Nam8 has <br />    Having shown U1 snRNA tolerates installation                   effect yeast vegetative growth itself, nam8  was <br /> new RNA sequences upstream 3ACUUAC8                                lethal combination U1 +30 allele caused a <br /> sequence, constructed tgs1  U1  p(CEN URA3                           strong cs growth defect combination &#239;&#172;?ve other <br /> U1) strain test separating U1 cap                     U1 50 extension alleles, manifest growth at <br /> 50 SS interaction element affect tgs1 . To                      25  C, minimal growth 30  C varying degrees of <br /> surprise, alleviating tgs1  cs phenotype,                 slow growth 34  C (Figure 2). The U1 50 extensions <br /> the U1 50 extensions exacerbated impact losing                     impact absence U1 snRNP <br /> the TMG cap. A gradient increasing synthetic                           subunit Mud1, alleles +30 were <br /> sickness tgs1  seen U1 +5, +25, +10                      viable 25  C mud1  background (Supplementary <br /> and +20 alleles (Figure 2). The +15 +30 U1 alleles                    Figure S1). The U1 50 extensions +15, +20 +25 <br /> were lethal tgs1  background.                                      displayed cs synthetic sickness null mutation of <br />                                                                           early splicing factor Swt21 interacts genetically <br /> Synthetic genetic interactions U1 50 extensions <br />                                                                           CBC, Tgs1, Yhc1 Prp28 (27,34); here, the <br /> A possible explanation synthetic lethality/sickness               U1 +30 allele impact, feeble growth at <br /> of tgs1  50 -extended U1s effect                  25 20  C swt21  background (Supplementary <br />                                                                                        Nucleic Acids Research, 2013, Vol. 41, No. 15 7489 <br />  <br />  <br />  <br />  <br /> Figure 2. Synthetic genetic interactions U1 50 extensions. Plasmid shuf&#239;&#172;&#8218;e assays used test 50 extended U1 snRNAs are <br /> functional various genetic backgrounds. Synthetic lethality indicated failure form macroscopic colonies FOA agar 10 d 20, 30 <br /> and 37  C. Cultures viable FOA-resistant cells grown YPD broth 37  C growth phenotypes assessed spotting serial 10- <br /> fold dilutions described Figure 1B. The U1 alleles speci&#239;&#172;?ed left genetic background test mutational synergy is <br /> indicated right. <br />  <br />  <br />  <br />  <br /> Figure S1). The U1 +20, +25, +30 alleles grew poorly                    mud2  cells grew normally WT U1 snRNA, U1 <br /> at 25 20  C absence Brr1, splicing factor                   +5 allele barely viable mud2  background, and <br /> implicated snRNP biogenesis (35) (Supplementary                          +10, +15, +20, +25 +30 alleles lethal <br /> Figure S1).                                                                 (Figure 2). By contrast, U1 50 extensions displayed <br />    The strongest mutational synergies U1 50 exten-                   little synergy null mutation Swm2 <br /> sions seen Mud2 subunit yeast                          (Supplementary Figure S1), named swm2  is <br /> heterodimeric BP-binding protein ablated. Although                      synthetic mud2  (29). <br />  7490 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br /> Effects insertions deletions immediately                              mutational synergies, revealed [+1] and <br /> downstream 3ACUUAC8 sequence                                          [+3] insertions lethal combination tgs1 , <br />                                                                              cbc2-Y24A, nam8 , mud1 , swt21  mud2 . Thus, <br /> The 50 AUACUUACCU10 single-stranded segment U1 <br />                                                                              single extra nucleotide suf&#239;&#172;?ced render U1 <br /> snRNA precedes folded U1 RNA tertiary structure <br />                                                                              snRNA function dependent inessential com- <br /> that initiates nucleotide U11 (i.e. four-helix junction <br />                                                                              ponents U1 snRNP early spliceosome. <br /> depicted Figure 4A). The 8CCU10 segment connecting <br />                                                                                 We tested effects shortening linker <br /> the 50 SS complementary motif helix 1 interacts the <br />                                                                              segment, deleting nucleotides C9 U10, singly and <br /> U1-C subunit U1 snRNP (4,8). Here, queried the <br />                                                                              combination. The predicted 50 structures deleted <br /> importance spacing 50 SS complemen- <br />                                                                              U1 RNAs shown Figure 4C. When tested by <br /> tary motif helix 1 introducing insertions 1&#226;&#8364;&#8220;30 nt <br />                                                                              plasmid shuf&#239;&#172;&#8218;e, U1 DC9 DC9U10 mutations were <br /> between C8 C9 (Figure 3). The insertion alleles were <br />                                                                              lethal. [Primer extension analysis showed mutant <br /> tested bioactivity plasmid shuf&#239;&#172;&#8218;e U1  strain. <br />                                                                              DC9U10 U1 snRNA synthesized plasmid- <br /> The [+1] [+3] insert strains viable 18&#226;&#8364;&#8220;37  C, <br />                                                                              borne DC9U10 gene vivo achieved steady-state <br /> albeit slower growing WT U1 cells low tempera- <br />                                                                              levels high chromosome-encoded wild- <br /> tures (Figure 3). By contrast, insertions  5 nt between <br />                                                                              type U1 (Supplementary Figure S2).] By contrast, yeast <br /> C8 C9 uniformly lethal (Figure 3). We performed <br />                                                                              U1-DU10 cells viable grew WT cells <br /> primer extension analysis RNA isolated yeast <br />                                                                              37 34  C, slow growing 25 20  C <br /> cells wild-type chromosomal U1 gene had <br />                                                                              (Figure 4B). The U1-DU10 allele synthetic lethal with <br /> been transformed CEN plasmids bearing series <br />                                                                              tgs1 , cbc2-Y24A, nam8 , mud1 , swt21  mud2 . <br /> of U1 insertion alleles (Supplementary Figure S2). <br /> Denaturing PAGE analysis 50 32P-labeled primer <br />                                                                              Effects extending <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span>-pairing potential helix 1 <br /> extension products revealed 50 ends the <br /> plasmid-encoded [+1], [+3], [+5], [+10], [+20]                           Helix 1 conserved respect presence and <br /> [+30] U1 snRNAs shifted incrementally upstream                          position yeast human U1 snRNAs not <br /> compared endogenous wild-type U1 snRNA. The                         conserved level primary structure. Whereas the <br /> levels radiolabeled cDNAs derived U1 snRNA                       50 strand helix 1 yeast U1 11UAAGAU16 <br /> insertion mutants high cDNA cor-                     (Figure 4A), corresponding segment human helix <br /> responding endogenous wild-type U1 snRNA                                  1 12GCAGG16 (8). To probe effects increasing the <br /> (Supplementary Figure S2), signifying (i) steady-                   length yeast helix 1, changed 9CU10 dinucleotide <br /> state levels U1 snRNA affected                        9UA10, extending base-pairing potential to <br /> inserted sequences (ii) lethality [+5],                       sequence 30 end U1 RNA helix 1 <br /> [+10], [+20] [+30] U1 alleles attributable                       span 8 bp instead 6 bp (Figure 4A). This U1 <br /> to failure produce mutant snRNAs. We surmise                       allele, named H1L (&#226;&#8364;&#732;helix 1 long&#226;&#8364;&#8482;), supported normal <br /> that tight constraints length (and/or                       yeast growth temperatures tested (Figure 4B). <br /> sequence) linker segment interacts                          Though benign se, H1L mutant displayed a <br /> U1-C. This theme underscored systematically                        broad range synthetic genetic interactions other <br /> tested viable [+1] [+3] insert alleles                           splicing factors. At severe end spectrum, <br />  <br />  <br />  <br />  <br /> Figure 3. Effects insertions immediately downstream 3ACUUAC8 sequence. The DNA sequences shown 50 ends WT U1 and <br /> the mutant variants [+1], [+3], etc., named according number nucleotides inserted positions C8 C9. The 3ACTTAC8 segment <br /> that pairs intron 50 SS highlighted gray. The [+5], [+10], [+20] [+30] mutants failed complement U1  plasmid shuf&#239;&#172;&#8218;e <br /> assay deemed lethal. The viable U1-[+1] U1-[+3] strains spot-tested growth indicated temperatures parallel with <br /> WT cells Figure 1B, plates incubated 5 d 25, 20 18  C. The synthetic lethal interactions U1-[+1] U1- <br /> [+3] mutants indicated the &#239;&#172;?gure. <br />                                                                                       Nucleic Acids Research, 2013, Vol. 41, No. 15 7491 <br />  <br />  <br /> U1-H1L synthetic lethal mud2 . H1L barely                      The C8U mutant barely grew 37 34  C failed <br /> viable 20&#226;&#8364;&#8220;34  C mud1  background did                      grow 25, 20 18  C (Figure 5A). The C4U strain <br /> grow 37  C (Figure 4B). H1L synergized tgs1                     grew best 34  C failed grow 25, 20 18  C <br /> cbc2-Y24A, resulting feeble growth 37 34  C                   (Figure 5A). We C4U lethal 34  C in <br /> severe cs defects lower temperatures (Figure 4B).                    combination tgs1 , cbc2-Y24A, nam8 , mud1 , <br /> H1L caused slow growth temperatures                           swt21  mud2 . A new allele, U1-A7G, viable <br /> nam8  swt21  backgrounds (Figure 4B). These                             slow growing 25, 30 34  C, did thrive <br /> results increasing size stability helix                 higher lower temperatures (Figure 5A). A7G was <br /> 1 enforces reliance dispensable spliceosome                    inviable combination tgs1 , cbc2-Y24A, nam8 , <br /> components assembly factors, especially Mud2.                           mud1 , swt21  mud2 . <br />                                                                                New instructive &#239;&#172;?ndings emerged analysis <br />                                                                             mutations U5A, U5G U5C. In agreement with <br /> Novel mutational effects 3ACUUAC8 sequence <br />                                                                             Siliciano Guthrie (36), U5A cells were <br /> The base-pairing interaction 50 end yeast                    viable modestly slow growing temperatures <br /> U1 snRNA consensus 50 SS yeast pre-mRNAs                         tested (Figure 5A). The U5G mutant viable and <br /> is shown Figure 5. Early studies effects                       virtually growth effects U5A higher <br /> nucleobase changes segment yeast U1                          temperatures slightly affected cold tem- <br /> RNA identi&#239;&#172;?ed following lethal pyrimidine-to-purine                     peratures (Figure 5A). The U5A U5G phenotypes <br /> mutations: C4A, C4G, C8A C8G (36,37). Pyrimidine-                       caused installation purine se to <br /> to-pyrimidine mutants C4U C8U viable slow                      establishment extra A:U G:U base pair with <br /> growing 30  C (36). The U5A allele, eliminates                   50 SS lieu native U:U mispair. Thus, re- <br /> U:U mismatch U1 50 SS duplex (Figure 5),                    markable U5C mutant grew WT U1 <br /> viable slow growing 30  C (36). Here, examined                    strain temperatures (Figure 5A), signifying a <br /> the effects known viable mutations,                 Py5:U mispair U1 50 SS RNA hybrid suf&#239;&#172;?ces for <br /> novel nucleobase changes 3ACUUAC8 sequence,                       seemingly normal U1 snRNA activity. A corollary of <br /> the activity genetic interactions U1 snRNA.                      result characteristic pseudouridine <br />  <br />  <br />  <br />  <br /> Figure 4. Effects deletions immediately downstream 3ACUUAC8 extending base-pairing potential helix 1. (A) The primary and <br /> predicted secondary structures 50 end wild-type U1 RNA H1L mutant shown. The 3ACUUAC8 segment pairs the <br /> intron 50 SS highlighted gray. Helix 1 shaded orange. The mutation 9CU10 9UA10 (colored red) extends helix 1 base pairs, which <br /> are shaded yellow. (B) Yeast strains bearing U1 WT, H1L DU10 alleles indicated genetic backgrounds spot-tested growth the <br /> temperatures speci&#239;&#172;?ed. (C) The primary predicted secondary structures  U10,  C9  C9U10 deletion mutants U1 snRNA are <br /> shown. The DC9 DC9U10 alleles failed complement U1  deemed lethal. Genetic interactions H1L DU10 indicated below <br /> the respective structures (A) (C). <br />  7492 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br />  <br />  <br /> Figure 5. Effects mutations 3ACUUAC8 sequence. (A) The base-pairing interaction U1 snRNA 3ACUUAC8 sequence the <br /> consensus pre-mRNA 50 SS shown. U1  p(CEN LEU2 U1) cells bearing indicated U1 alleles spot-tested growth indicated <br /> temperatures. (B) Yeast strains bearing U1 WT, U5A U5C alleles indicated genetic backgrounds spot-tested growth the <br /> temperatures speci&#239;&#172;?ed. Synthetic lethal interactions U5A U5C indicated bottom. <br />  <br />  <br />  <br />  <br /> modi&#239;&#172;?cation U5 nucleobase U1 snRNA (38)                      34  C failed grow 30, 25 20  C (Figure 5B). <br /> (which occur U5C mutant) critical               These results fortify case role Nam8 cap <br /> for U1 snRNA function wild-type                        binding CBC U1 snRNP assembly non-consen- <br /> background.                                                            sus 50 SS (30,32,39). The U5C allele unconditionally <br />   We surveyed function U1                   synthetic lethal mud2  background (Figure 5B). <br /> snRNA U5C variant affected deleting optional                       The synergy tests performed U5A <br /> splicing factors U1 snRNP subunits. The U1 U5C                     snRNA mutant. The slow growth phenotype U5A <br /> mutation unaffected swt21  displayed                      evident temperatures swt21  and <br /> mild synthetic growth defect mud1  tempera-                tgs1  backgrounds (Figure 5B). The true of <br /> tures tested (Figure 5B). U5C exacerbated cs pheno-                U5A nam8 , highlighting distinctive genetic <br /> type tgs1 , pushing restrictive temperature                     interaction nam8  U5A versus U5C the <br /> upwards (Figure 5B). Strong genetic interactions U5C                loss Nam8 exerted stronger growth effect U5C <br /> were seen nam8  cbc2-Y24A, U5C                    (Figure 5B). U5A synthetically lethal mud2  <br /> nam8  U5C cbc2-Y24A strains grew slowly 37                  cbc2-Y24A. <br />                                                                         Nucleic Acids Research, 2013, Vol. 41, No. 15 7493 <br />  <br />  <br /> New genetic interactions Msl5 subunits U1        revealed extensive network atomic contacts <br /> snRNP, Mud2 Cbc2                                            conserved amino acid chains (many of <br />                                                                 hydrophobic) RNA nucleobases and <br /> Saccharomyces cerevisiae Msl5 (BP-binding protein) or- <br />                                                                 sugars, RNA phosphates surface <br /> chestrates spliceosome assembly binding intron <br />                                                                 exposed make relatively protein contacts <br /> BP sequence 50 -UACUAAC establishing cross- <br />                                                                 (Supplementary Figure S3). Given seven <br /> intron-bridging interactions components of <br />                                                                 chains mutated singly alanine Table 1 iden- <br /> the splicing machinery (20,22,31,40&#226;&#8364;&#8220;42). Msl5 (a 476-aa <br />                                                                 tical SF1 Msl5 (Supplementary Figure S3), the <br /> polypeptide) essential yeast vegetative growth. The <br />                                                                 seventh Msl5 residue Leu169 conserved Ile SF1, <br /> central BP RNA-binding domain Msl5&#226;&#8364;&#8221;composed of <br />                                                                 RNA&#226;&#8364;&#8220;protein contacts observed these <br /> KH QUA2 modules (43)&#226;&#8364;&#8221;is &#239;&#172;&#8218;anked N- and <br />                                                                 chains SF1 structure canonical yeast <br /> C-terminal domains imputed functions in <br />                                                                 BP RNA recognition element, presumption that <br /> protein&#226;&#8364;&#8220;protein interactions. By gauging ability of <br />                                                                 RNA contacts likely the <br /> Msl5 mutants complement msl5 , recently re- <br />                                                                 equivalent Msl5 chains. These contacts listed in <br /> ported Msl5 N-terminal Mud2-binding domain <br />                                                                 Table 1. (A complete list imputed BP RNA contacts <br /> (aa 35&#226;&#8364;&#8220;68) downstream PPxY100 motif proposed <br />                                                                 Msl5 residues subjected <br /> to interact WW-domain proteins (44) inessential, <br />                                                                 alanine scanning provided Supplementary Figure <br /> as C-terminal proline-rich domain (aa 382&#226;&#8364;&#8220;476) and <br />                                                                 S4.) The striking &#239;&#172;?nding 4 13 conserved <br /> two zinc-binding CxxCxxxxHxxxxC motifs (aa 273&#226;&#8364;&#8220;286 <br />                                                                 chains, imputed contacts BP <br /> and 299&#226;&#8364;&#8220;312) (31). A subset conserved BP <br />                                                                 (Supplementary Table S1), essential msl5  <br /> RNA-binding amino acids central KH-QUA2 <br />                                                                 complementation. <br /> domain (aa 146&#226;&#8364;&#8220;269) essential pairwise (Ile189- <br />                                                                    We tested newly identi&#239;&#172;?ed fully viable MSL5 <br /> Arg190; Leu256-Leu259) trios (Leu169-Arg172- <br />                                                                 alleles mutational synergies mutations other <br /> Leu176), RNA-binding residues dispens- <br />                                                                 genes affecting early splicing events. L176A supported <br /> able (31). We used collection viable Msl5 <br />                                                                 wild-type growth combined mud1 , nam8 , <br /> mutants illuminate synthetic genetic interactions be- <br />                                                                 mud2 , tgs1  cbc2-Y24A (Table 2). Thus, the <br /> tween Msl5 Mud2, Nam8, Tgs1 Cbc2 (31,32). <br />                                                                 inferred contacts Leu176 BP adenosine <br /> The results suggested network important <br />                                                                 (to adenine N3 atom ribose O20 ) dispens- <br /> but functionally buffered protein&#226;&#8364;&#8220;protein protein&#226;&#8364;&#8220; <br />                                                                 able yeast growth genetic backgrounds tested. By <br /> RNA interactions Msl5 Mud2 complex at <br />                                                                 contrast, L169A I189A lethal combination <br /> the BP, U1 snRNP 50 SS CBC pre- <br />                                                                 mud1 , nam8 , mud2 , tgs1  cbc2-Y24A <br /> mRNA cap. <br />                                                                 (Table 2). Leu169 imputed intercalate and <br />    Here, queried collection Msl5 mutants that <br />                                                                 make van der Waals contacts BP adenine the <br /> have growth defect se synthetic interactions <br />                                                                 upstream &#239;&#172;&#8218;anking adenine. Ile189 contacts N1 and <br /> with components U1 snRNP evaluated <br />                                                                 N6 atoms BP adenine. The emerging theme <br /> previously. We focused &#239;&#172;?rst Mud1 subunit. We <br />                                                                 loss U1 snRNP components, weaken- <br /> found mud1  synthetically lethal (i) N- <br />                                                                 ing CBC interactions mRNA cap, buffered <br /> terminal truncation eliminated Mud2- <br />                                                                 cross-intron Msl5 contacts BP that, when <br /> binding site (aa 35&#226;&#8364;&#8220;68); (ii) triple-alanine mutation <br />                                                                 weakened mutations characterized presently, results <br /> P97A-P98A-Y100A PPxY100 motif; (iii) a <br />                                                                 failure vegetative growth, presumably insuf&#239;&#172;?- <br /> C-terminal truncation eliminated aa 402&#226;&#8364;&#8220;425                cient splicing essential pre-mRNAs. <br /> (Supplementary Table S1). Otherwise benign double- <br /> alanine mutations Msl5 KH-QUA2 domain were <br /> either lethal (N163A-V165A) severely sick (V195A-            New genetic interactions Msl5 50 end of <br /> K196A, T265A-R267A K252A-R253A) mud1                 U1 snRNA <br /> background (Supplementary Table S1). By contrast,         Viable U1 snRNA mutants tested synthetic <br /> was synthetic lethality/sickness mud1            interactions viable Msl5 mutations con- <br /> C273A&#226;&#8364;&#8220;C276A C299A&#226;&#8364;&#8220;C302A alleles respectively           structing U1  msl5  double <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>knockout</span> strain a <br /> disrupt proximal distal zinc knuckle modules         resident CEN URA3 U1 MSL5 plasmid sustain <br /> Msl5 (Supplementary Table S1).                                  growth. This strain co-transformed various com- <br />    To better delineate essential constituents BP     binations CEN HIS3 MSL5 CEN LEU2 U1 <br /> RNA binding site yeast Msl5, tested effects        plasmids, individual His+ Leu+ transformants were <br /> single alanine substitutions lethal alanine-   tested growth FOA 20, 30 37  C. Allelic <br /> cluster alleles identi&#239;&#172;?ed previously (31). Tests msl5        pairs failed support growth FOA tem- <br /> complementation showed that: (i) R172A, R190A               perature deemed synthetic lethal (Table 3). FOA-re- <br /> L259A unconditionally lethal; (ii) L256A displayed         sistant survivors grown YPD liquid culture <br /> a tight cs growth phenotype; (iii) L169A, L176A         tested growth spotting serial dilutions YPD <br /> I189A grew normally YPD agar temperatures             agar 20, 25, 30, 34 37  C. The results compiled in <br /> (Table 1). The NMR structure human SF1 (the                  Table 3, +++ signi&#239;&#172;?es growth indistinguishable <br /> ortholog Msl5) bound RNA (50 -AUACUAAC                 wild-type U1 MSL5 cells, ++ denotes smaller <br /> AA) containing consensus yeast BP sequence (43)             colony size + indicates pinpoint colonies. Several of <br />  7494 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br /> Table 1. Effect KH-QUA2 domain mutations Msl5 function vivo <br />  <br /> MSL5                  Contacts BBP RNA structure:                                                msl5  complementation <br />                          1    2    3     4    5    6   7    8    9                 <br /> Allele                pU pA pC pU pA pA pC pA pA                               18                 25                    30                   37  <br />  <br /> WT                                                                             +++                +++                  +++                   +++ <br /> L169A                 A6 base; A5 base                                         +++                +++                  +++                   +++ <br /> R172A                 A6pC7 phosphate                                          &#226;&#8364;&#8220;                  &#226;&#8364;&#8220;                    &#226;&#8364;&#8220;                     &#226;&#8364;&#8220; <br /> L176A                 A6 base N3, ribose O20                                   +++                +++                  +++                   +++ <br /> I189A                 A6 base N1 N6                                            +++                +++                  +++                   +++ <br /> R190A                 A9 base                                                  &#226;&#8364;&#8220;                  &#226;&#8364;&#8220;                    &#226;&#8364;&#8220;                     &#226;&#8364;&#8220; <br /> L256A                 A2 ribose O20 ; C3 base N3, ribose O40                   &#226;&#8364;&#8220;                  &#226;&#8364;&#8220;                    +                     ++ <br /> L259A                 U4 base C2 O2, ribose O40                                &#226;&#8364;&#8220;                  &#226;&#8364;&#8220;                    &#226;&#8364;&#8220;                     &#226;&#8364;&#8220; <br />  <br /> Complementation msl5D indicated MSL5 alleles assayed plasmid shuf&#239;&#172;&#8218;e (31). Individual Leu+ transformants streaked agar <br /> medium containing FOA. Growth scored incubation 7 d 18, 25, 30 37  C. Lethal mutants failed form colonies at <br /> any temperature. Individual FOA-resistant colonies viable MSL5 alleles grown mid-log phase YPD broth adjusted equivalent <br /> A600. Aliquots (3 ml) serial 10-fold dilutions spotted YPD agar plates, incubated 18, 25, 30 37  C. Growth was <br /> scored follows: (+++) colony size indistinguishable strains bearing wild-type MSL5; (++) slightly reduced colony size; (+) pinpoint <br /> macroscopic colonies formed; (&#226;&#8364;&#8220;) growth. <br />  <br />  <br />  <br />  <br /> Table 2. Synthetic interactions Msl5 KH-QUA2 domain mutations               +++ ++ growth higher temperatures. The same <br />                                                                                V195A K196A K252R alleles lethal combin- <br /> MSL5 allele                            Complementation <br />                                                                                ation DU10 [+1] U1 alleles (Table 3). At the <br />                 mud1         mud2            nam8          cbc2-Y24A   tgs1    end spectrum, MSL5-L176A did synergize <br />                                                                                50 extensions H1L, did display syn- <br /> WT              +++          +++             +++           +++         +++     thetic sickness [+1] DU10 (Table 3). <br /> L169A           &#226;&#8364;&#8220;            &#226;&#8364;&#8220;               &#226;&#8364;&#8220;             &#226;&#8364;&#8220;           &#226;&#8364;&#8220; <br /> L176A           +++          +++             +++           +++         +++ <br /> I189A           &#226;&#8364;&#8220;            &#226;&#8364;&#8220;               &#226;&#8364;&#8220;             &#226;&#8364;&#8220;           &#226;&#8364;&#8220;       Dissecting genetic interactions Msl5 <br />                                                                                PPxY100 motif <br /> Complementation msl5D genetic backgrounds speci&#239;&#172;?ed by <br /> otherwise functional MSL5 alleles assayed plasmid shuf&#239;&#172;&#8218;e.               The msl5-(P97A-P98A-Y100A) allele synthetically <br /> L169A I189A transformants failed form FOA-resistant                     lethal combination 50 extensions, U5C, DU10 <br /> colonies temperature tested &#239;&#172;?ve strain back-             [+1] mutations U1 snRNA (Table 3; 97AAxA100). <br /> grounds. By contrast, L176A supported growth FOA and, spot <br /> testing, +++ growth YPD agar temperatures &#239;&#172;?ve strains.        The U1-H1L msl5-(P97A-P98A-Y100A) strain viable <br />                                                                                slow growing cold-sensitive (Table 3). The <br />                                                                                PPxY100 motif suggested mediate cross-intron- <br /> the allelic combinations elicited ts cs growth failures;                    bridging interaction Msl5 Prp40 subunit the <br /> these denoted Table 3 growth                        U1 snRNP (22). Prp40 583-aa protein containing two <br /> scores applied permissive temperatures                         tandem WW modules N-terminus (aa 1&#226;&#8364;&#8220;70) and <br /> these strains.                                                                 FF domains dispersed downstream (44,47,48). The <br />    The U1 50 extensions, U5C mutation H1L,                         isolated tandem WW domain binds vitro synthetic <br /> DU10 [+1] changes distal 50 SS complementarity                      peptide PSPPPVYDA corresponding Msl5 residues 94&#226;&#8364;&#8220; <br /> segment uniformly synthetic lethal MSL5-                             102 (44). A key question synthetic lethality <br /> (69&#226;&#8364;&#8220;476), N-terminal truncation disrupts                           P97A-P98A-Y100A mutations 50 end U1 <br /> interaction BP binding protein Mud2/                               snRNA causally related loss imputed <br /> U2AF65 (42,45,46) (Table 3). Indeed, synergies                          binding PPxY motif Prp40 WW domain. If <br /> MSL5-(69-476) U1 snRNA mutations                                      case, deletion WW domain Prp40 <br /> phenocopied mud2  (Figures 2&#226;&#8364;&#8220;5). By contrast,                         phenocopy mutational synergies msl5-(P97A- <br /> MSL5-(55&#226;&#8364;&#8220;476), deletes Mud2-                            P98A-Y100A). We idea test replacing <br /> binding interface, permitted +++ growth yeast                               chromosomal PRP40 locus truncated allele, <br /> bearing 50 extensions, U5C H1L variants U1                          prp40-(77-583) (henceforth named prp40-DWW), which <br /> snRNA low temperatures did support growth                        cleanly subtracts tandem WW modules. In agreement <br /> 34&#226;&#8364;&#8220;37  C (Table 3).                                                            recent report Go&#194;&#168;rnemann et al. (49), found <br />    Lesions L169A I189A RNA-binding site                          prp40-DWW strain grew PRP40 <br /> Msl5 synthetically lethal U1 mutant                      control strain YPD agar temperatures (not <br /> alleles surveyed and, allelic pairs viable,                     shown). We proceeded construct U1  prp40-DWW <br /> the cells sick cold-sensitive (Table 3). Other                        p(CEN URA3 U1) strain suitable plasmid shuf&#239;&#172;&#8218;e of <br /> changes KH-QUA2 domain Msl5 displayed                                U1 snRNA variants. The instructive &#239;&#172;?ndings the <br /> less severe synthetic interactions U1 mutations.                      +5, +15, +20, +30 U5C alleles U1 syn- <br /> For example, V195A K196A K252R elicited cs defects                         thetic lethal msl5-(P97A-P98A-Y100A) all <br /> in combination U1 50 extensions displayed                         viable combination prp40-DWW (Supplementary <br />                                                                                       Nucleic Acids Research, 2013, Vol. 41, No. 15 7495 <br />  <br />  <br /> Table 3. Synergies U1 snRNA mutations Msl5 mutations <br />  <br /> U1 snRNA                                                                        MSL5 <br />                                                     97       100 <br />                    55&#226;&#8364;&#8220;476            69&#226;&#8364;&#8220;476              AAxA          L169A           L176A          I189A         V195 K196A            K252R <br />  <br /> WT                 +++              ++             +++                +++              +++           +++            +++                  +++ <br /> +5                 +++              lethal         lethal             ++ /csa          +++           +              +++ /csb             +++ /csc <br /> +15                +++ /tsd         lethal         lethal             lethal           +++           lethal         +++ /cse             +++ /csc <br /> +20                +++ /tsd         lethal         lethal             + /csf           +++           lethal         +++ /cse             ++ /csa <br /> +30                +++ /tsd         lethal         lethal             lethal           +++           lethal         +++ /cse             ++ /csa <br /> U5C                +++ /tsd         lethal         lethal             lethal           +++           lethal         ++ /csa              ++ /csa <br /> H1L                +++ /tsd         lethal         ++ /csg            lethal           +++           lethal         ++ /csg              + <br />  U10               lethal           lethal         lethal             lethal           +             lethal         lethal               lethal <br /> [+1]               + /tsh           lethal         lethal             lethal           ++            lethal         lethal               lethal <br />  <br /> Complementation msl5D U1 genetic backgrounds speci&#239;&#172;?ed assayed plasmid shuf&#239;&#172;&#8218;e. Allelic combinations failed form colonies <br /> at temperature denoted lethal. Viable strains tested growth YPD agar scored described Table 1. ts cs phenotypes <br /> were follows: <br /> a <br />  ++ 34&#226;&#8364;&#8220;37  C. <br /> b <br />  +++ 25&#226;&#8364;&#8220;37  C. <br /> c <br />  +++ 34&#226;&#8364;&#8220;37  C. <br /> d <br />  +++ 18&#226;&#8364;&#8220;30  C. <br /> e <br />  +++ 30&#226;&#8364;&#8220;37  C. <br /> f <br /> + 34&#226;&#8364;&#8220;37  C. <br /> g <br />  ++at 30&#226;&#8364;&#8220;37  C. <br /> h <br />  + 18&#226;&#8364;&#8220;25  C. <br />  <br />  <br />  <br /> Figure S5). The prp40-DWW U1-U5C strain grew                       phosphorylation, replaced Tyr100 phenylalanine. <br /> all temperatures. prp40-DWW strains U1 50 exten-                      The Y100F allele displayed synthetic defects com- <br /> sions grew 37, 34 30  C slow                          bination mud1 , nam8 , cbc2-Y24A, tgs1  or <br /> growing 20  C. The prp40-DWW U1-H1L strain grew                         mud2  (Figure 6A). We conclude genetically <br /> slowly temperatures. These results compat-                  buffered essential functions Tyr100 related to <br /> ible model severe synergies U1                   phosphorylation status. <br /> snRNA 50 mutations msl5-(P97A-P98A-Y100A)                            We proceeded test effects P97A, P98A, Y100A <br /> mediated exclusively (or predominantly) Prp40                      Y100F tandem U1 alleles which <br /> WW domain. We surmise PPxY100 motif                            97AAxA100 triple-mutant synthetic lethal <br /> Msl5 plays important, albeit genetically buffered, role                 (i.e. +5, +20, +30, U5C [+1]) sick (H1L). P97A <br /> in pre-mRNA splicing naught                        Y100F sustained +++ growth U1 mutant <br /> Prp40 WW domain.                                                           background 20, 30, 34 37  C (not shown). P98A <br />    To better understand PPxY motif functions,                   supported ++ growth 20&#226;&#8364;&#8220;37  C +5, +20, +30, <br /> tested effects single mutations P97A, P98A                      U5C H1L strains; msl5-P98A U1-[+1] strain <br /> Y100A Msl5 activity genetic backgrounds                      grew +++ 30, 34 37  C ++ 20 25  C <br /> for msl5-(P97A-P98A-Y100A) triple-mutant                         (not shown). By contrast, msl5-Y100A sick com- <br /> was lethal (e.g. mud1 , nam8 , cbc2-Y24A tgs1 )                        bination U1-[+1] +30. The msl5-Y100A U1- <br /> or sick (mud2 ). Initial control experiments veri&#239;&#172;?ed                  [+1] strain formed pinpoint colonies 37 30  C <br /> the single mutations PPxY impact comple-                      failed grow 25 20  C (Supplementary Figure <br /> mentation msl5  (not shown). Complementation tests                      S6). The msl5-Y100A U1 +30 strain grew slowly at <br /> for mutational synergies revealed P97A P98A                   high temperature did grow low temperature <br /> alleles genetic interactions mud1 , nam8 ,                     (Supplementary Figure S6). Y100A displayed cs <br /> cbc2-Y24A, tgs1  mud2  (Figure 6A). By contrast,                    growth defect combination +20 50 extension <br /> Y100A allele synthetically lethal mud1                        little synthetic phenotype U1-H1L and <br /> nam8  synthetic sick mud2 , effectively pheno-                    +5 (Supplementary Figure S6). Thus, loss Tyr100 <br /> copying synthetic defects 97AAxA100 triple-                     suf&#239;&#172;?ced synergize benign mutations of <br /> mutant backgrounds (Figure 6A). Y100A                         U1 snRNA, albeit forme fruste impact of <br />                                                                            97 <br /> also synthetically sick cbc2-Y24A tgs1                               AAxA100 mutant U1 backgrounds. <br /> (Figure 6A), evinced smaller colony size 34 and <br /> 37  C cold-sensitivity (Figure 6B). These results impli-                Bypass essentiality Prp28 U1 snRNA <br /> cate loss Tyr100 principally responsible                  mutations <br /> synthetic interactions Msl5 97AAxA100 mutant                    Yeast Prp28 essential pre-mRNA splicing factor and <br /> proteins involved spliceosome assembly.                                  member DEAD-box family nucleic acid-de- <br />    Tyr100 identi&#239;&#172;?ed site Msl5 phosphor-                    pendent NTPases (51,52). Although Prp28 been <br /> ylation vivo (50). To probe synthetic pheno-                 imputed RNA helicase, catalytic activities <br /> types msl5-Y100A related absence tyrosine                  inherent yeast Prp28 characterized. <br />  7496 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br />  <br />  <br /> Figure 6. Genetic interactions Msl5 PPxY100 motif. (A) Complementation msl5D indicated MSL5 alleles assayed plasmid <br /> shuf&#239;&#172;&#8218;e. Individual Leu+ transformants streaked agar medium containing FOA. Growth scored incubation 7 d 18, 25, 30 or <br /> 37  C. Lethal mutants failed form colonies temperature. Individual FOA-resistant colonies viable MSL5 alleles were <br /> grown mid-log phase YPD broth adjusted equivalent A600. Serial 10-fold dilutions spotted YPD agar plates, then <br /> incubated 25, 30, 34 37  C. Growth scored follows: (+++) colony size indistinguishable strains bearing wild-type MSL5 all <br /> temperatures; (++) slightly reduced colony size; (cs) pinpoint colonies 25  C. (B) Spot testing growth MSL5 WT versus Y100A the <br /> indicated strain backgrounds temperatures speci&#239;&#172;?ed. <br />  <br />  <br />  <br /> Nonetheless, Prp28 implicated genetically displace-                  transformants screened growth FOA at <br /> ment U1 snRNP 50 SS transition                  18&#226;&#8364;&#8220;37  C U1 variants bypass the <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> pre-mRNA U1 U2-containing spliceosome                          Prp28 requirement rise FOA-resistant <br /> pre-mRNA U2 U5 U6 spliceosome (15,16,34). Initial                          colonies. The important distinction and <br /> insights came observation deleterious                    prior tests U1 bypass Prp28 assay <br /> effects hyper-stabilizing U1 50 SS base-paring inter-               mandates U1 variants bypass prp28  they <br /> action genetically enhanced cs mutant prp28,                  source U1 snRNA cell. Four U1 <br /> but conditional mutants helicase-like                  snRNA mutants passed test: C4U, U5C, DU10 and <br /> splicing factors (16). In turn, prp28-cs phenotype               [+3]. The viable U1  prp28  cells bearing bypass <br /> be suppressed extra U1 snRNA gene C4U                         U1 alleles tested growth YPD agar 20, 25, <br /> change weakens base-pairing consensus 50 SS                30, 34 37  C (Figure 7). <br /> (16). The case forti&#239;&#172;?ed &#239;&#172;?nding essen-                     The hierarchy bypass strengths noteworthy. The <br /> tiality Prp28 vegetative growth bypassed                   DU10 variant U1 snRNA best Prp28 <br /> by mutations essential U1 snRNP subunits Yhc1,                      bypass suppressors, insofar DU10 prp28  cells grew <br /> Prp42 Snu71, deletions inessential                   wild-type cells 37, 34 30  C, as <br /> subunits Nam8 Mud1 (15,34). Deletion splicing                   gauged colony size (Figure 7). The progressive cs <br /> factor Swt21 rescued lethality prp28  (34).                    growth defect DU10 prp28  strain 25 and <br /> These results highlighted need Prp28                   20  C (Figure 7) exactly seen DU10 <br /> U1 snRNP ejection early spliceosome                            PRP28 mutant strain (Figure 4,  U10 WT). Thus, DU10 <br /> alleviated certain alterations presumed                     appears fully independent Prp28. <br /> weaken U1 50 SS contacts. This idea consolidated                       U5C best Prp28 bypass suppressors. <br /> the demonstration ectopic expression mutant U1                     U5C prp28  cells grew 37, 34 30  C, albeit slower <br /> snRNAs C4U, U5C C8U (on wild-type U1                           wild-type strain, unable grow 20&#203;&#353; C <br /> snRNA) restore viability prp28  (15).                        (Figure 7). Because U5C snRNA supported appar- <br />    Here, surveyed collection viable U1 snRNA                     ently normal growth temperatures PRP28 back- <br /> mutants Prp28 bypass, constructing U1  prp28                      ground (Figure 5A), surmise bypass effect of <br /> double-knockout strain resident CEN URA3 U1                         U5C cold-sensitive. This result suggests higher tem- <br /> PRP28 plasmid sustain growth. This strain                           peratures help destabilize weakened U1-U5C 50 SS <br /> transformed CEN LEU2 U1 plasmids,                             pairing permits U1 snRNP ejection without <br />                                                                                     Nucleic Acids Research, 2013, Vol. 41, No. 15 7497 <br />  <br />  <br />  <br />  <br /> Figure 7. U1 mutations bypass requirement essential PRP28 gene. Yeast prp28  U1  cells harboring indicated U1 allele&#226;&#8364;&#8221;WT, C4U, <br /> U5C, DU10 [+3]&#226;&#8364;&#8221;on CEN LEU2 plasmid wild-type PRP28 (CEN HIS3) CEN HIS3 plasmid (prp28 ) grown in <br /> liquid cultures 37  C mid-log phase. The cultures adjusted A600 0.1, aliquots serial dilutions spotted YPD agar. <br /> Photographs plates incubation 2 d (30, 34, 37  C), 4d (25  C) 5 d (20  C) shown. <br />  <br />  <br /> the assistance Prp28. However, pairing                     missense mutations essential components be <br /> more stable lower temperatures, Prp28 required.               tolerated level vegetative growth, effects of <br />    Although C4U appeared effective Prp28                    simultaneous subtraction functionally overlapping <br /> bypass comparing growth C4U prp28                          microscopic pathways seen synthetic lethality or <br /> wild-type strain (Figure 7), bear mind C4U se                 sickness. <br /> elicits phenotype PRP28 background                             The composition U1 snRNP more <br /> (Figure 5). (Side-by-side comparisons C4U PRP28                    complex budding yeast mammals, respect <br /> and C4U prp28  strains revealed difference growth;                  size U1 RNA (568 versus 164 nt) the <br /> not shown.)                                                               number U1-speci&#239;&#172;?c protein subunits (10 versus 3). <br />    The [+3] insertion 50 SS complementarity                   The larger yeast U1 RNA include binding sites for <br /> motif &#239;&#172;?rst hairpin restored slow growth prp28                  yeast-speci&#239;&#172;?c U1 subunit proteins, are <br /> cells 37, 34, 30 25  C did bypass 20  C                 Prp39, Prp40, Snu71, Snu56, Luc7, Prp42 Nam8. <br /> (Figure 6). Because [+3] insertion did cause                    Mammalian cells putative homologs Luc7 <br /> strong cs phenotype PRP28 background (Figure 3                     (hLuc7), Nam8 (TIA-1) Prp40 (FBP11) (1,13,53), <br /> and side-by-side growth comparisons shown),                        subunits U1 snRNP. By <br /> conclude [+3] bypass effect cold-sensitive, a`                assimilating seven yeast proteins stoichiometric <br /> la U5C.                                                                   subunits, yeast U1 snRNP serve recipient of <br />                                                                           splicing regulation (via signals pre-mRNA inter- <br />                                                                           actions splicing factors) that, case of <br /> DISCUSSION                                                                mammalian splicing, conveyed armada of <br />                                                                           trans-acting splicing factors in&#239;&#172;&#8218;uence 50 SS usage <br /> The present study reveals structure&#226;&#8364;&#8220;function relationships                and/or formation U1 50 SS complex (54,55), most <br /> at 50 end yeast U1 snRNA contribu-                   equivalents budding yeast. <br /> tions expansive network genetic interactions                        The present study informative regarding cross-in- <br /> among proteins RNA elements drive spliceosome                    tron contacts U1 50 SS Mud2 Msl5 BP <br /> assembly. We 50 dinucleotide leader                 complexes, analysis effects mutations in <br /> the TMG cap conserved 3ACUUAC8 motif                       BP-binding protein Msl5. Because Msl5 essential in <br /> enlarged 30 nt compromising cell growth. By                    budding yeast, mutational analyses the <br /> contrast, tight functional constraints                   yeast intron BP focused exclusively base- <br /> downstream dinucleotide spacer 3ACUUAC8                           pairing interactions U2 snRNP (56&#226;&#8364;&#8220;58), is <br /> and helix 1 U1 snRNA fold, insertions                      little known necessity function Msl5 BP <br /> of  5 nt lethal. We exploited collection bio-                  interactions vivo. Here, extended initial struc- <br /> logically active U1 alleles 50 extensions, variations             ture-guided mutagenesis RNA-binding site Msl5 <br /> the 3ACUUAC8 sequence, short downstream insertions/                       consolidate following points. First, alanine <br /> mutations longer H1 helix discover new intra-                    mutations Msl5 amino acids that <br /> snRNP synergies inessential U1 components                        contact consensus BP unconditionally condi- <br /> Nam8, Mud1 TMG cap. We new                             tionally lethal: Arg172, Arg190, Leu256 Leu259. By <br /> synergies non-snRNP splicing factor complexes                    contrast, alanine mutations Msl5 amino <br /> Msl5 Mud2 CBC bind intron BP                             acids contact consensus BP elicit growth <br /> pre-mRNA 50 cap, respectively. These results fortify                  phenotypes. However, allele-speci&#239;&#172;?c synergies <br /> idea spliceosome assembly facilitated               benign RNA-binding site mutations mu- <br /> several genetically buffered microscopic pathways                         tations U1 snRNA, U1 snRNP subunits Mud1 and <br /> involving cross-intron-bridging interactions U1                    Nam8, Tgs1, Mud2 Cbc2. These results discriminate <br /> snRNP 50 SS complex Mud2 Msl5 BP complex                         essential optional BP RNA contacts imply that <br /> and      cross-exon-bridging      interactions                     effects hypomorphic Msl5 changes weaken BP <br /> CBC m7GpppRNA complex U1 50 SS complex.                          binding buffered interactions U1 snRNP <br /> Although elimination weakening set inter-                    mediated, part, Mud2 subunit of <br /> actions deletions inessential components                         Msl5 Mud2 heterodimer. <br />  7498 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br />  <br />  <br /> Figure 8. Disposition PxY100 structure SF1 bound U2AF65. (A) Stereo view crystal structure (pdb 4FXW) complex <br /> between SF1-(20-128) (colored cyan) U2AF65 RRM3 domain (colored gray). Selected SF1 chains shown stick models are <br /> numbered according positions yeast Msl5. (B) Alignment amino acid sequences N-terminal domains Msl5 SF1, <br /> corresponding SF1 structure shown (A). Positions chain identity/similarity indicated  . The a-helices comprising the <br /> coiled-coil SF1 indicated cylinders alignment. The PPxY motif connecting loop denoted bracket. <br />  <br />  <br />  <br />    The N-terminal domains Msl5 human SF1                        (Figure 8A). The SF1 a-helices include side <br /> responsible interactions C-terminal RRM3                      chains contact U2AF65 directly. Because side <br /> domains Mud2 U2AF65, respectively. Initial struc-                   chains conserved yeast Msl5 (highlighted in <br /> tural studies identi&#239;&#172;?ed SF1 peptide segment                             Figure 8B), presume comprise homologous <br /> (13PSKKRKRSRWNQD25) bound U2AF65,                                interface Mud2. The salient point the <br /> the principal contact point insertion                       new SF1&#226;&#8364;&#8220;U2AF65 structure equivalent the <br /> conserved SF1 tryptophan chain (Trp22 SF1;                         Msl5 97PPxY100 motif (84EPxY87 SF1) located within <br /> Trp35 Msl5) hydrophobic pocket U2AF65                         loop connects a-helices com- <br /> (59) (see Figure 8). Although SF1 segment                          ponent interface Mud2/U2AF65 (Figure 8A). <br /> conserved yeast Msl5, mutational studies showed                    Our studies aromatic Tyr100 residue of <br /> the Msl5 W35A change did phenocopy synthetic                       Msl5 loop plays important role vivo is <br /> genetic interactions mud2  (31). Moreover, compound                     revealed splicing factors U1 <br /> mutations conserved Mud2 amino acids that,                       snRNA absent mutated. The earlier suggestion <br /> U2AF65, comprise SF1 peptide-binding site did                      vitro studies PPxY peptide Msl5 <br /> affect Mud2 function vivo genetic backgrounds                  mediates Msl5 binding WW domains Prp40 is <br /> Mud2 essential (31). These results suggested                   consistent &#239;&#172;?ndings deletion the <br /> Mud2 Msl5 interface complex                        WW domain Prp40 did phenocopy mutational <br /> seen U2AF65 SF1 peptide structure. The detailed                     synergies msl5-(P97A-P98A-Y100A) U1 alleles <br /> anatomy SF1 USAF65 interface (and extension                      (Supplementary Figure S4). Also, noted a <br /> the Mud2 Msl5 interface) clari&#239;&#172;?ed                        prp40-DWW mud1  strain displayed wild-type growth at <br /> X-ray NMR structures complex SF1-                         temperatures (not shown), contrast uncondi- <br /> (20-128) U2AF65 RRM3 (45,46). RRM3 interacts                       tional synthetic lethality msl5-(P97A-P98A-Y100A) <br /> with newly de&#239;&#172;?ned domain module SF1, composed                      mud1D (Supplementary Table S1). The location of <br /> two long a-helices form antiparallel coiled coil                   Tyr100 exposed surface loop distant from <br />                                                                                   Nucleic Acids Research, 2013, Vol. 41, No. 15 7499 <br />  <br />  <br /> the putative Mud2-Msl5 interface (Figure 8A) suggests                    12. Zhang,D. Rosbash,M. (1999) Identi&#239;&#172;?cation proteins <br />                                                                              cross-link pre-mRNA yeast commitment complex. <br /> that Tyr100 mediates interaction Msl5 with <br />                                                                              Genes Dev., 13, 581&#226;&#8364;&#8220;592. <br /> another component yeast splicing apparatus,                       13. Puig,O., Bragado-Nilsson,E., Koski,T. Se&#194;&#180;raphin,B. (2007) The <br /> either domain Mud2 Msl5 present                           U1 snRNP-associated factor Luc7p affects 50 splice site selection <br /> SF1-U2AF65 structure, component U1                               yeast human. Nucleic Acids Res., 35, 5874&#226;&#8364;&#8220;5885. <br /> snRNP, splicing factor.                                   14. Lewis,J.D., Go&#194;&#168;rlich,D. Mattaj,I.W. (1996) A yeast cap <br />                                                                              binding protein complex (yCBC) acts early step pre- <br />                                                                              mRNA splicing. Nucleic Acids Res., 24, 3332&#226;&#8364;&#8220;3336. <br />                                                                          15. Chen,J.Y., Stands,L., Staley,J.P., Jackups,R.R., Latus,L.J. and <br /> SUPPLEMENTARY DATA                                                           Chang,T.H. (2001) Speci&#239;&#172;?c alterations U1-C protein U1 <br />                                                                              small nuclear RNA eliminate requirement Prp28p, an <br /> Supplementary Data available NAR Online:                              essential DEAD box splicing factor. Mol. Cell, 7, 227&#226;&#8364;&#8220;232. <br /> Supplementary Table 1 Supplementary Figures 1&#226;&#8364;&#8220;6.                     16. Staley,J.P. Guthrie,C. (1999) An RNA switch 50 splice <br />                                                                              site requires ATP DEAD box protein Prp28p. Mol. Cell, <br />                                                                              3, 55&#226;&#8364;&#8220;64. <br />                                                                          17. Siliciano,P.G., Kivens,W.J. Guthrie,C. (1991) More half <br /> FUNDING                                                                      yeast U1 snRNA dispensable growth. Nucleic Acids Res., <br />                                                                              19, 6367&#226;&#8364;&#8220;6372. <br /> National Institutes Health (NIH) [GM52470 S.S.]                    18. Liao,X., Kretzner,L., Se&#194;&#180;raphin,B. Rosbash,M. (1990) <br /> and [GM50288 B.S.]. S.S. American Cancer                            Universally conserved yeast-spec&#239;&#172;?c U1 snRNA sequences are <br />                                                                              important essential U1 snRNP function. Genes Dev., <br /> Society Research Professor. B.S. recipient                         4, 1766&#226;&#8364;&#8220;1774. <br /> Hearst Foundation Award. Funding open access                         19. Liao,X.C., Tang,J. Rosbash,M. (1991) An enhancer screen <br /> charge: NIH [GM52470].                                                       identi&#239;&#172;?es gene encodes yeast U1 snRNP A protein: <br />                                                                              implications snRNP protein function pre-mRNA splicing. <br /> Con&#239;&#172;&#8218;ict statement. None declared.                                Genes Dev., 7, 419&#226;&#8364;&#8220;428. <br />                                                                          20. Abovich,N., Liao,X.C. Rosbash,M. (1994) The yeast MUD2 <br />                                                                              protein: interaction PRP11 de&#239;&#172;?nes bridge between <br />                                                                              commitment complexes U2 snRNP addition. Genes Dev., 8, <br /> REFERENCES                                                                   843&#226;&#8364;&#8220;854. <br />                                                                          21. Colot,H.V., Stutz,F. Rosbash,M. (1996) The yeast splicing <br />  1. Fabrizio,P., Dannenberg,J., Dube,P., Kastner,B., Stark,H.,               factor Mud13p commitment complex component and <br />     Urlaub,H. Lu&#194;&#168;hrmann,R. (2009) The evolutionarily conserved           corresponds CBP20, small subunit nuclear cap- <br />     core design catalytic activation step yeast                binding complex. Genes Dev., 10, 1699&#226;&#8364;&#8220;708. <br />     spliceosome. Mol. Cell, 36, 593&#226;&#8364;&#8220;608.                                 22. Abovich,N. Rosbash,M. (1997) Cross-intron bridging <br />  2. Warlocki,Z., Odenwa&#194;&#168;lder,P., Schitzova,J., Platzmann,F., Stark,H.,       interactions yeast commitment complex conserved in <br />     Urlaub,H., Ficner,R., Fabrizio,P. Lu&#194;&#168;hrmann,R. (2009)                mammals. Cell, 89, 403&#226;&#8364;&#8220;412. <br />     Reconstitution steps Saccharomyces cerevisiae splicing    23. Tang,J., Abovich,N. Rosbash,M. (1996) Identi&#239;&#172;?cation and <br />     puri&#239;&#172;?ed spliceosomal components. Nat. Struct. Mol. Biol.,           characterization yeast gene encoding U2 small nuclear <br />     16, 1237&#226;&#8364;&#8220;1243.                                                           ribonucleoprotein particle B&#226;&#8364;&#8482;&#226;&#8364;&#8482; protein. Mol. Cell. Biol., 16, <br />  3. Hoskins,A.A., Friedman,L.J., Gallagher,S.S., Crawford,D.J.,              2787&#226;&#8364;&#8220;2795. <br />     Anderson,E.G., Wombacher,R., Ramirez,N., Cornish,V.W.,               24. Fortes,P., Kufel,J., Fornerod,M., Polycarpou-Schwarz,M., <br />     Gelles,J. Moore,M.J. (2011) Ordered dynamic assembly             Lafontaine,D., Tollervey,D. Mattaj,I.W. (1999) Genetic and <br />     single spliceosomes. Science, 331, 1289&#226;&#8364;&#8220;1295.                         physical interaction involving yeast nuclear cap-binding <br />  4. Tang,J., Abovich,N., Fleming,M.J., Se&#194;&#180;raphin,B. Rosbash,M.           complex. Mol. Cell. Biol., 19, 6543&#226;&#8364;&#8220;6553. <br />     (1997) Identi&#239;&#172;?cation characterization yeast homolog      25. Tong,A.H., Evangelista,M., Parsons,A.B., Xu,H., Bader,G.D., <br />     U1 snRNP-speci&#239;&#172;?c protein C. EMBO J., 13, 4082&#226;&#8364;&#8220;4091.                      Page,N., Robinson,M., Raghibizadeh,S., Hogue,C.W. and <br />  5. Gottschalk,A., Tang,J., Puig,O., Salgado,J., Neubauer,G.,                Bussey,H. (2001) Systematic genetic analysis ordered arrays <br />     Colot,H.V., Mann,M., Se&#194;&#180;raphin,B., Rosbash,M.                        yeast deletion mutants. Science, 294, 2364&#226;&#8364;&#8220;2368. <br />     Lu&#194;&#168;hrmann,R. (1998) A comprehensive biochemical genetic          26. Wilmes,G.M., Bergkessel,M., Bandyopadhyay,S., Shales,M., <br />     analysis yeast U1 snRNP reveals &#239;&#172;?ve novel proteins. RNA,          Braberg,H., Cagney,G., Collins,S.R., Whitworth,G.B., Kress,T.L. <br />     4, 374&#226;&#8364;&#8220;393.                                                              Weissman,J.S. (2008) A genetic interaction map RNA- <br />  6. Fortes,P., Bilbao-Corte&#194;&#180;s,D., Fornerod,M., Rigaut,G.,                    processing factors reveals links Sem1/Dss1-containing <br />     Raymond,W., Se&#194;&#180;raphin,B. Mattaj,I.W. (1999) Luc7p, novel           complexes mRNA export splicing. Mol. Cell, 32, <br />     yeast U1 snRNP protein role 50 splice site recognition.        735&#226;&#8364;&#8220;746. <br />     Genes Dev., 13, 2425&#226;&#8364;&#8220;2438.                                           27. Hausmann,S., Zheng,S., Costanzo,M., Brost,R.L., Garcin,D., <br />  7. Schwer,B., Erdjument-Bromage,H. Shuman,S. (2011)                     Boone,C., Shuman,S. Schwer,B. (2008) Genetic and <br />     Composition yeast snRNPs snoRNPs absence                biochemical analysis yeast human cap trimethylguanosine <br />     trimethylguanosine caps reveals nuclear cap binding protein         synthase: functional overlap TMG caps, snRNP components, <br />     gained U1 component implicated cold-sensitivity tgs1           pre-mRNA splicing factors, RNA decay pathways. J. Biol. <br />     cells. Nucleic Acids Res., 39, 6715&#226;&#8364;&#8220;6728.                                Chem., 283, 31706&#226;&#8364;&#8220;31718. <br />  8. Pomeranz Krummel,D.A., Oubridge,C., Leung,A.K.W., Li,J.          28. Costanzo,M., Baryshnikova,A., Bellay,J., Kim,Y., Spear,E.D., <br />     Nagai,K. (2009) Crystal structure human spliceosomal U1               Sevier,C.S., Ding,H., Koh,J.L., Tou&#239;&#172;?ghi,K., Mostafavi,S. et al. <br />     snRNP 5.5 A&#203;&#353; resolution. Nature, 458, 475&#226;&#8364;&#8220;480.                        (2010) The genetic landscape cell. Science, 327, 425&#226;&#8364;&#8220;431. <br />  9. Weber,G., Trowitzsch,S., Kastner,B., Lu&#194;&#168;hrmann,R.                29. Chang,J., Schwer,B. Shuman,S. (2010) Mutational analyses of <br />     Wahl,M.C. (2010) Functional organization Sm core           trimethylguanosine synthase (Tgs1) Mud2: proteins implicated <br />     crystal structure human U1 snRNP. EMBO J., 29, 4172&#226;&#8364;&#8220;4184.             pre-mRNA splicing. RNA, 16, 1018&#226;&#8364;&#8220;1031. <br /> 10. Leung,A.K.W., Nagai,K. Li,J. (2011) Structure             30. Qiu,Z.R., Schwer,B. Shuman,S. (2011) Determinants of <br />     spliceosomal U4 snRNP core domain implications               Nam8-dependent splicing meiotic pre-mRNAs. Nucleic Acids <br />     snRNP biogenesis. Nature, 473, 536&#226;&#8364;&#8220;539.                                  Res., 39, 3427&#226;&#8364;&#8220;3445. <br /> 11. Van der Feltz,C., Anthony,K., Brilot,A. Pomeranz                 31. Chang,J., Schwer,B. Shuman,S. (2012) Structure-function <br />     Krummel,D.A. (2012) Architecture spliceosome.                     analysis genetic interactions yeast branchpoint binding <br />     Biochemistry, 51, 3321&#226;&#8364;&#8220;3333.                                             protein Msl5. Nucleic Acids Res., 40, 4539&#226;&#8364;&#8220;4552. <br />  7500 Nucleic Acids Research, 2013, Vol. 41, No. 15 <br />  <br />  <br /> 32. Qiu,Z.R., Chico,L., Chang,J., Shuman,S. Schwer,B. (2012)               N-terminal domain splicing factor 1 30 -splice site <br />     Genetic interactions hypomorphic mutations m7G cap               recognition. Nucleic Acids Res., 41, 1343&#226;&#8364;&#8220;1354. <br />     binding pocket yeast nuclear cap binding complex: essential      46. Wang,W., Maucuer,A., Gupta,A., Manceau,V., Thickman,K.R., <br />     role Cbc2 meiosis splicing MER3 pre-mRNA. RNA,               Bauer,W.J., Kennedy,S.D., Wedekind,J.E., Green,M.R. and <br />     18, 1996&#226;&#8364;&#8220;2011.                                                             Kielkopf,C.L. (2013) Structure phosphorylated SF1 bound to <br /> 33. Schwer,B., Mao,X. Shuman,S. (1998) Accelerated mRNA                    U2AF65 essential splicing factor complex. Structure, 21, <br />     decay conditional mutants yeast mRNA capping enzyme.                 197&#226;&#8364;&#8220;208. <br />     Nucleic Acids Res., 26, 2050&#226;&#8364;&#8220;2057.                                     47. Gasch,A., Wiesner,S., Martin-Malpartida,P., Ramirez-Espain,X., <br /> 34. Hage,R., Tung,L., Du,H., Stands,L., Rosbash,M. Chang,T.H.              Ruiz,L. Macias,M.J. (2006) The structure Prp40 FF1 <br />     (2009) A targeted bypass screen identi&#239;&#172;?es Ynl187p, Prp42p,                 domain interaction crn-TPR1 motif Clf1 gives <br />     Snu71p, Cbp80p stable U1 snRNP/pre-mRNA                            new insight binding mode FF domains. J. Biol. <br />     interaction, Mol. Cell. Biol., 29, 3941&#226;&#8364;&#8220;3952.                              Chem., 281, 356&#226;&#8364;&#8220;364. <br /> 35. Noble,S.M. Guthrie,C. (1996) Transcriptional pulse-chase           48. Bonet,R., Ruiz,L., Morales,B. Macias,M.J. (2009) Solution <br />     analysis reveals role <br /> </body></html>