<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 15, 552&#226;&#8364;&#8220;562, February 2004 <br />  <br />  <br /> <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>RNA Interference</span> Inhibition Mus81 Reduces <br /> Mitotic Recombination Human Cells <br /> Veronique Blais,* Hui Gao,* Cherilyn A. Elwell,*&#226;&#8364;&#160; Michael N. Boddy,* <br /> Pierre-Henri L. Gaillard,* Paul Russell,&#226;&#8364;&#161; Clare H. McGowan&#226;&#8364;&#161;&#194;&#167; <br />  <br /> *Departments Molecular Biology &#226;&#8364;&#161;Cell Biology, The Scripps Research Institute, La Jolla, <br /> California 92037 <br />  <br /> Submitted August 11, 2003; Revised October 15, 2003; Accepted October 15, 2003 <br /> Monitoring Editor: Mark Solomon <br />  <br />  <br />  <br /> Mus81 highly conserved endonuclease homology XPF subunit XPF-ERCC1 complex. In yeast Mus81 <br /> associates second subunit, Eme1 Mms4, essential endonuclease activity vitro vivo <br /> function. Human Mus81 binds homolog fission yeast Eme1 vitro vivo. We recombinant <br /> Mus81-Eme1 cleaves replication forks, 3&#226;&#180;&#8226; flap substrates, Holliday junctions vitro. By use differentially tagged <br /> versions Mus81 Eme1, Mus81 associates Mus81 Eme1 associates Eme1. Thus, <br /> complexes containing Mus81-Eme1 units function coordinate substrate cleavage vivo. Down- <br /> regulation Mus81 RNA interference reduces mitotic recombination human somatic cells. The recombination <br /> defect rescued expression bacterial Holliday junction resolvase. These data provide direct evidence role of <br /> Mus81-Eme1 mitotic recombination higher eukaryotes support hypothesis Mus81-Eme1 resolves <br /> Holliday junctions vivo. <br />  <br />  <br /> INTRODUCTION                                                         duplex free double strand end. The double strand end <br />                                                                      repaired homologous recombination, process involv- <br /> Maintaining genomic integrity essential component of <br />                                                                      ing resection strand free end, invasion the <br /> the process cell duplication division. Cells espe- <br />                                                                      intact linear duplex, creation new replication fork <br /> cially vulnerable DNA damage process of <br /> DNA replication. Replication forks encounter relatively         (Haber, 1999). The importance homologous recombina- <br /> innocuous forms damage single strand break           tion replication restart established prokaryotes. <br /> a modified <span id='am-237' about='xsp:base' typeof='owl:Thing'>base</span> increase damage generating              Evidence important pathway eu- <br /> double strand break fixing <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>mutation</span>. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> cope            karyotes accumulating (Cox et al., 2000; Michel, 2000; <br /> the need complete replication presence DNA              McGlynn Lloyd, 2002). Two activities resolve Hol- <br /> damage, organisms evolved number mecha-               liday junctions linear duplex identified in <br /> nisms replicating damaged sites             human somatic cell extracts (Chen et al., 2001; Constantinou <br /> ways cause minimal genome instability. One option               et al., 2002). One, known Resolvase A, introduces sym- <br /> involves use bypass translesion polymerases           metrical related nicks strands like polarity, asso- <br /> act sites template damaged           ciates branch migration activities (Constantinou et al., <br /> way deciphered replicative poly-            2001, 2002). The identities proteins comprise this <br /> merases (Friedberg et al., 2002). A second mechanism, in-            activity known. The second human activity that <br /> volves regression fork annealing nascent           resolves Holliday junctions associated Mus81 (Chen <br /> strands form Holliday junction structure (Higgins et al.,       et al., 2001; Constantinou et al., 2002). Mus81-associated en- <br /> 1976). The leading strand extended, using            donuclease cleaves Holliday junction structures linear <br /> nascent lagging strand template. Subsequent migration           duplexes does cut symmetrically related sites and <br /> of Holliday junction nascent end re-             products ligatable vitro (Chen et al., 2001). <br /> stores replication fork, allowing replication continue.       Mus81 component highly conserved endonuclease <br /> Use template-switching mechanism does di-              homology XPF subunit XPF&#226;&#8364;&#8220;ERCC1 com- <br /> rectly repair damage, does allow replication to <br />                                                                      plex (Boddy et al., 2000; Interthal Heyer, 2000; Chen et <br /> proceed downstream damaged site circum- <br />                                                                      al., 2001; Mullen et al., 2001). In yeast, Mus81 mutants are <br /> vents problems associated incomplete replication. <br /> Stalled replication forks repaired             hypersensitive DNA-damaging agents impede DNA <br /> recombination repair process Holliday junction            replication, Mus81 mutation lethal combination <br /> formed regressed fork resolved intact              disruption RecQ like helicases Rqh1 Sgs1 <br />                                                                      (Boddy et al., 2000; Mullen et al., 2001). In humans, mutations <br />                                                                      RecQ like helicases BLM WRN associated with <br />  Article published online ahead print. Mol. Biol. Cell 10.1091/   genomic instability increased incidence cancer (Hick- <br /> mbc.E03&#226;&#8364;&#8220; 08 &#226;&#8364;&#8220; 0580. Article publication date www. <br /> molbiolcell.org/cgi/doi/10.1091/mbc.E03&#226;&#8364;&#8220; 08 &#226;&#8364;&#8220; 0580. <br />                                                                      son, 2003). BLM WRN demonstrated to <br />  &#226;&#8364;&#160; <br />    Present address: Department Medicine, University Califor-   branch-migrate Holliday junctions vitro (Constantinou et <br />    nia, San Francisco, San Francisco, CA 94143.                      al., 2000; Karow et al., 2000). Thus, genetic interactions <br />  &#194;&#167; <br />    Corresponding author. E-mail address: chmcg@scripps.edu.          enzymatic properties Mus81 RecQ helicases <br />  <br />  <br /> 552                                                                                  &#194;&#169; 2004 The American Society Cell Biology <br />                                                                                                                     Mus81 Required Recombination <br />  <br />  <br /> suggest act independently stalled replication                     Expression Recombinant Proteins RNA Interference <br /> forks promote replication restart.                                          (RNAi) <br />    Mus81 associates second protein, known Eme1                       3HaMus81 wild-type endonuclease dead (Chen et al., 2001) cloned <br /> in fission yeast Mms4 budding yeast. Eme1/Mms4                       pCDNA3 (Invitrogen, Carlsbad, CA) using EcoRI XhoI sites. Eme1 <br /> essential Mus81 endonuclease activity vitro                  FLAG tagged C&#226;&#172;&#732; terminus using oligonucleotide <br />                                                                                5&#226;&#172;&#732;GCCCGCTCGAGTCACTTGTCATCGTCGTCCTTGTAGTCAGCACTATC- <br /> vivo function (Boddy et al., 2001; Kaliraman et al., 2001;                     TAAAGA-3&#226;&#172;&#732; inserted pCDNA3 using EcoRI XhoI. Mus81 was <br /> Mullen et al., 2001; Doe et al., 2002). Based phenotypes                FLAG tagged C&#226;&#172;&#732; terminus using oligonucleotide <br /> of fission yeast mutants Mus81 Eme1, i.e., sensitivity                  5&#226;&#172;&#732;-CTCGAGTCACTTGTCATCGTCGTCCTTGTAGTCGGTCAAGGGGCC- <br /> to agents impede DNA replication, recombination-de-                       GTAGC-3&#226;&#172;&#732;. 3HaEme1 cloning Eme1 pcDNA-3Ha (Mel- <br />                                                                                chionna et al., 2000) using NdeI XhoI. Human cells transfected <br /> pendent failure complete meiosis, evidence                         using Effectene (QIAGEN, Valenica, CA) FuGENE 6 (Roche Diagnostics, <br /> Mus81 containing complexes resolve Holliday junctions                     Indianapolis, IN)) according manufacturer&#226;&#8364;&#8482;s instructions. For expres- <br /> linear duplex vitro, proposed Mus81 Eme1                        sion Sf9 cells, Gst-Mus81 (Chen et al., 2001) cloned pFastBac <br /> are subunits nuclear Holliday junction resolvase (Boddy                   (Invitrogen) using EcoRI HindIII, Eme1-FLAG cloned using <br />                                                                                EcoRI XhoI. The Bac-to-Bac (Invitrogen) used generate <br /> et al., 2001). A similar function proposed human                   recombinant viruses. All constructs verified sequencing. Two 19- <br /> enzyme (Chen et al., 2001). On hand, specific                     nucleotide regions (corresponding 178 &#226;&#8364;&#8220;197 292&#226;&#8364;&#8220;311 Mus81 open <br /> strains budding yeast, Mus81 Mms4 mutants                        reading frame; Chen et al., 2001) selected cloned pSuper and <br /> have profound meiotic recombination defect, suggesting                       used described Brummelkamp et al., (2002). Polymerase chain reaction <br />                                                                                (PCR) carried HeLa cell cDNA library (BD Biosciences Clonetech, <br /> that Mus81-Mms4 major meiotic Holliday junction                       Palo Alto, CA) using sequences present forms Eme1 (5&#226;&#172;&#732;- <br /> resolvase budding yeast (De Los Santos et al., 2001, 2003).                 CGGAATTCACCATGGCTCTAAAGAAGTCATCACC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GC- <br /> Furthermore, highly purified recombinant Mus81-Mms4                         CCGCTCGAGTCAGTCAGCACTATCTAAAGAGAG-3&#226;&#172;&#732;). The PCR products <br /> Mus81-Eme1 partially purified Mus81-complexes                         cloned pTopo (Invitrogen). Restriction enzyme analysis clones <br />                                                                                gave <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> corresponding Eme1B. Sequencing clones verified <br /> human cells extracts greater activity 3&#226;&#172;&#732; flap                    transcript corresponding Eme1B expressed HeLa cells. NLS- <br /> replication fork structures Holliday junctions (Ka-                  RusA-2Ha wild-type inactive cloned pCDNA3 expression in <br /> liraman et al., 2001; Constantinou et al., 2002; Whitby et al.,                human cells using pRep1-RusA pRep1-RusA-D70N (Boddy et al., <br /> 2003). The preference Mus81 shows substrates led                     2001) starting constructs. <br /> to suggestion replication forks 3&#226;&#172;&#732; flaps, not <br />                                                                                Gene Cloning <br /> Holliday junctions, relevant vivo substrate of <br />                                                                                Three clones corresponding sequence AK0055926 retrieved <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Mus81 (Kaliraman et al., 2001; Constantinou et al., 2002; Doe                  American Type Culture Collection (Manassas, VA) collection human ex- <br /> et al., 2002; Bastin-Shanower et al., 2003).                                   pressed tagged sequences. ATCC clone 2899969 (coding Eme1A) came <br />    To gain insight function Mus81 human                    cervical carcinoma library, ATCC clone 3905809 (coding Eme1B) <br /> cells identified human homologs Eme1 bind                      came form uterine tumor library, ATCC clone 814686 (coding for <br />                                                                                Eme1C) came form normal B-cell library. The clones sequenced <br /> Mus81 vitro vivo. The sequence designated                         directly. Where difference clones observed multiple reads <br /> Eme1A independently identified shown interact                       directions used confirm specific nucleotides. <br /> with Mus81 groups (Ciccia et al., 2003; Ogrunc <br /> and Sancar, 2003). We recombinant Mus81-Eme1                         Nuclease Assays Western Analysis <br /> efficiently cleaves replication forks 3&#226;&#172;&#732; flap substrates                Nuclease assays carried described previously (Chen et al., 2001; <br /> vitro. Mus81-Eme1 generates lower, readily detectable,                   Constantinou et al., 2002). Antibody Ha-epitope Babco (Rich- <br /> yield linear duplex product Holliday junction struc-                   mond, CA). Antibody FLAG-epitope (FLAG-M2) Sigma- <br />                                                                                Aldrich (St. Louis, MO). Antibody Mus81 described Chen et al. <br /> tures. Resolution Holliday junctions requires coordinate                    (2001). Cells lysates, immune-precipitates, immune-blots analysis was <br /> cleavage opposing strands junction. By use                        carried Chen et al. (2001). <br /> differentially tagged versions Mus81 Eme1, find <br /> that Mus81 associates Mus81, Eme1 associates                     Immune-Fluorescence Microscopy <br /> with Eme1. Thus, complexes containing Mus81-                       Conditions growing XPA cells, localized UV-irradiation, develop- <br /> Eme1 units function coordinate substrate cleavage                     ing immune-fluorescence images described Gao et al. (2003). <br /> in vivo. Down-regulation Mus81 RNA interference <br /> reduces mitotic recombination human somatic cells. The                      RESULTS <br /> recombination defect rescued expression bacterial <br /> Holliday junction resolvase. These data provide direct evi-                    Identification Human Homolog Eme1 <br /> dence role Mus81-Eme1 mitotic recombination                     Reiterative PSI-BLAST searches (www.ncbi.nlm.nih.gov/ <br /> higher eukaryotes support hypothesis Mus81-                       blast/Blast) used identify mammalian homologs of <br /> Eme1 resolves Holliday junctions vivo.                                      Schizosaccharomyces pombe Eme1 (SpEme1). In itera- <br />                                                                                tion, uncharacterized open reading frame (AL356173) <br /> MATERIALS AND METHODS                                                          Neurospora crassa identified having significant <br />                                                                                similarity spEme1. The open reading frame has <br /> Cell Culture Mitotic Recombination Assays                                  previously noted homology Mus81 bind- <br /> HeLa cells, 293 (human embryonic kidney cells), GM847L22 (Prince et al.,   ing partner budding yeast Mms4 (Mullen et al., 2001). <br /> 2001) grown DMEM, supplemented 10% enriched calf serum, <br /> 100 &#226;?&#174;g/ml penicillin streptomycin. For routine culture, GM847L22 were <br />                                                                                Reiteration search identified highly related, <br /> maintained presence 400 &#226;?&#174;g/ml G418. Sf9 cells grown Excell-401   overlapping human sequences significant similarity to <br /> media (JRH Biosciences, Lenexa, KS) 50 &#226;?&#174;g/ml penicillin strepto-      SpEme1 (Figure 1A). Three clones corresponding se- <br /> mycin. To assay mitotic recombination 5 &#226;&#171;&#187; 105 cells plated G418 free   quence AK0055926 retrieved American Type Cul- <br /> medium 16 h transfection. Thymidine (2 mM) added culture <br /> medium, cells grown 16 h. Cells cultured normal <br />                                                                                ture Collection sequenced confirm iden- <br /> growth medium 24 h. Cells fixed 2% formaldehyde     tity. Surprisingly, clones differ <br /> phosphate-buffered saline (PBS) 10 min, washed PBS twice,         slightly designated Eme1A, Eme1B, Eme1C <br /> assayed &#226;?&#164;-galactosidase activity incubation PBS containing 1 mg/ml   (Figure 1B). The identification single genomic locus <br /> X-Gal, 4 mM potassium ferrocyanide, 4 mM potassium ferricyanide, 2 mM <br /> MgCl2 37&#194;&#176;C overnight (Herzing Meyn, 1993). The number blue cells <br />                                                                                (17q21.33) National Center Biotechnology Infor- <br /> was scored using 20&#226;&#171;&#187; objective inverted light microscope. Statistical    mation human genome database suggested three <br /> significances calculated using Student&#226;&#8364;&#8482;s t test.                        variants derive single gene. The absence specific <br />  <br />  <br /> Vol. 15, February 2004                                                                                                                                   553 <br />  V. Blais et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Identification human homolog SpEme1. (A) A human cDNA encoding open reading frame 571 amino acids with <br /> similarity spEme1 (Boddy et al., 2001) identified PSI-BLAST algorithm. Alignment Eme1 homologs generated using the <br /> Clustal W program. Identities shown reverse. (B) Three sequence variants human Eme1 obtained American Type Culture <br /> Collection. Eme1A (ATCC clone 2899969) encodes protein 583 amino acids, Eme1B (ATCC clone 3905809) encodes protein 571 amino <br /> acids, Eme1C (ATCC clone 814686) encodes protein 542 amino acids. <br />  <br />  <br />  <br /> sequences (corresponding amino acids 303&#226;&#8364;&#8220;331 372&#226;&#8364;&#8220;             FLAG-Eme1A FLAG-Eme1C immune-precipitates (our <br /> 384 Eme1A) suggested different forms            unpublished data); however, Eme1B-associated endonucle- <br /> due incomplete splicing. However, absence single         ase activity consistently higher. <br /> codon (at position 138) longest sequence,                To determine forms Eme1 expressed HeLa <br /> substitution arginine cysteine (at position 525)      cells, oligonucleotide primers common variants <br /> the shortest one, compatible interpretation.       used amplify sequences HeLa cell cDNA <br /> To determine which, any, forms human              library. The products PCR cloned restriction <br /> Eme1 binds Mus81, epitope-tagged constructs          enzyme analysis clones gave pattern corresponding <br /> variants tested ability interact Mus81          Eme1B. Sequencing 2 clones verified transcript <br /> and support Mus81-associated endonuclease activity.               corresponding Eme1B expressed HeLa cells. Al- <br /> HeLa cells transiently transfected 3HaMus81            analysis does exclude possibility the <br /> FLAG-tagged Eme1. As shown Figure 2A, 3HaMus81                A C form Eme1 expressed cell types, at <br /> detected immune-precipitates forms Eme1.          low levels HeLa cells, transcripts corresponding B <br /> However, 3HaMus81 associated FLAG-                form protein readily detectable, we <br /> Eme1B reproducibly higher FLAG-Eme1A C.                  conducted analysis Eme1 using B form of <br /> Human Mus81 robust activity 3&#226;&#172;&#732; flap substrate          protein. <br /> in vitro (Constantinou et al., 2002); therefore, Mus81              We asked endonuclease activity detected <br /> Eme1 immune-complexes assayed associated endo-              Eme1 immune-precipitate dependent Mus81. <br /> nuclease activity using substrate (Figure 2B). The           As shown Figure 2C, FLAG-Eme1 associated with <br /> activity 3HaMus81 greatly increased samples           wild-type Mus81 mutant version Mus81 that <br /> had cotransfected FLAG-Eme1B,                 lacks associated endonuclease activity (Chen et al., 2001). <br /> FLAG-Eme1A FLAG-Eme1C. Likewise, differ-                 Endonuclease activity detected FLAG-Eme1 im- <br /> ent forms Eme1 immune-precipitated using                 mune-precipitate cells simultaneously <br /> FLAG antibody, B form readily detectable activity.           transfected wild-type 3HaMus81, cells that <br /> Longer exposures reveal endonuclease activity                transfected FLAG-Eme1 (Figure 2D). <br />  <br />  <br /> 554                                                                                                       Molecular Biology Cell <br />                                                                                               Mus81 Required Recombination <br />  <br />  <br />  <br />  <br /> Figure 2. Eme1 interacts Mus81 and <br /> has Mus81-dependent endonuclease activ- <br /> ity. HeLa cells transiently transfected <br /> with 3HaMus81 presence absence <br /> of FLAG-Eme1. Forty-eight hours after <br /> transfection, lysates immune-precipi- <br /> tates probed presence 3Ha- <br /> Mus81 FLAG-Eme1. (A) 3HaMus81 <br /> was detected FLAG immune-precipitates <br /> from cells express FLAG-Eme1A, B, or <br /> C. (B) Ha FLAG immune-precipitates <br /> were assayed associated endonuclease <br /> activity using 3&#226;&#172;&#732; flap substrate. Coex- <br /> pression 3HaMus81 FLAG-Eme1B <br /> resulted highest activity. (C) FLAG- <br /> Eme1B detected Ha immune-precip- <br /> itates cells express wild-type 3Ha- <br /> Mus81 (WT) endonuclease dead <br /> version Mus81 (AA). 3HaMus81 (wild- <br /> type endonuclease dead version) <br /> were detected FLAG immune-precip- <br /> itate cells express FLAG-Eme1B <br /> (D) FLAG-Eme1B immune-precipitates <br /> from cells cotransfected with <br /> wild-type endonuclease dead <br /> 3HaMus81 cleave 3&#226;&#172;&#732; flap structures. <br /> 3HaMus81WT immune-precipitates have <br /> associated endonuclease activity was <br /> increased cells cotransfected <br /> with FLAG-Eme1. S, substrate alone. The <br /> black dot represents radioactive phos- <br /> phate labeling substrate. <br />  <br />  <br />  <br /> Thus, Eme1 associated endonuclease activity dependent       nucleoli (Figure 3A). To determine Eme1 re- <br /> coexpression wild-type Mus81. As reported previously           cruited regions damaged DNA cell cycle-depen- <br /> (Chen et al., 2001), Ha-immune-precipitates cells   dent manner, cells synchronized G1/S bound- <br /> been transfected 3HaMus81 detectable endonuclease        ary double-thymidine block. Arrested cells, cells <br /> activity absence transfected Eme1 (Figure 2D).          released 5 h, UV-irradiated <br />                                                                   polycarbonate filters containing 10-&#226;?&#174;m-diameter <br /> Eme1 Localizes Nucleoli Regions Damaged              pores 15 min later, samples fixed stained for <br /> DNA                                                               regions damage Eme1. To prevent repair of <br /> Recent studies human Mus81 showed localizes         UV-photoproducts NER, simian virus 40 (SV40)-trans- <br /> nucleoli regions damaged DNA S-phase cells           formed fibroblasts defect XPA used this <br /> (Gao et al., 2003). If Eme1 acting complex Mus81,      analysis (Mone et al., 2001). When thymidine-arrested <br /> it distinctive pattern subcellular        cells UV irradiated, regions damage clearly <br /> localization. This tested using indirect-immune-fluo-         visualized (Figure 3B) using cyclobutane pyrimi- <br /> rescence cells transfected vectors          dine dimer-specific monoclonal antibody TDM2 (Mizuno <br /> encoding 3HaEme1 FLAG-Mus81. As previously                    et al., 1991; Mori et al., 1991). 3HaEme1 the <br /> seen, Mus81 retained nucleoli cells       nucleoli thymidine-arrested cells, no <br /> subjected situ extraction fixation (Gao et al.,      evidence Eme1 recruited regions damaged <br /> 2003). 3HaEme1 exactly colocalize               DNA (1 25 cases). By contrast, 5 h thymidine <br /> Mus81 regions nucleus (Figure 3A).         release majority cells replicating DNA, <br /> Costaining 3HaEme1 nucleolar marker nucleo-            Eme1 nucleoli regions of <br /> phosmin/B23 confirmed Eme1 retained            damage (25 30 cases). Thus, pattern Eme1 local- <br />  <br />  <br /> Vol. 15, February 2004                                                                                                   555 <br />  V. Blais et al. <br />  <br />  <br />  <br />  <br />                                                                                         Figure 3. Eme1 colocalizes Mus81 in <br />                                                                                         nucleoli recruited regions dam- <br />                                                                                         age S-phase cells. HeLa cells were <br />                                                                                         transfected vectors encoding <br />                                                                                         3HaEme1 FLAG-Mus81 grown <br />                                                                                         coverslips 24 h subjected in <br />                                                                                         situ extraction fixation. (A) Samples <br />                                                                                         processed immune-fluorescence <br />                                                                                         using antibodies Ha epitope <br />                                                                                         (red), FLAG epitope (green) nu- <br />                                                                                         cleophosmin/B23 (green). Samples were <br />                                                                                         costained 4,6-diamidino-2-phenylin- <br />                                                                                         dole reveal DNA. (B) XPA cells were <br />                                                                                         transfected 3HaEme1 FLAG- <br />                                                                                         Mus81, grown coverslips, synchro- <br />                                                                                         nized G1/S boundary double- <br />                                                                                         thymidine (Tdr) block. A UV-opaque filter <br />                                                                                         containing pores &#226;&#172;&#402;10 &#226;?&#174;m diameter <br />                                                                                         placed cells exposed to <br />                                                                                         60 mJm&#226;&#171;&#186;2 254-nm light cultured <br />                                                                                         15 min situ fraction fixa- <br />                                                                                         tion. Cells stained Eme1 (red) and <br />                                                                                         TDM-2 (green) detect regions <br />                                                                                         UV-induced cyclobutane pyrimidine <br />                                                                                         dimer. Top, typical cell thymi- <br />                                                                                         dine time irradiation. Bottom, typ- <br />                                                                                         ical cell 5 h release thymi- <br />                                                                                         dine block. <br />  <br />  <br />  <br /> ization damaged undamaged cells identical            implies enzyme exist complexes containing <br /> that recently reported Mus81 (Gao et al., 2003).              catalytic units. To determine Mus81-Eme1 ex- <br />                                                                   ists higher order complexes needed coordinate <br /> Activity Recombinant Mus81-Eme1                                cleavage Holliday junctions, human cells simulta- <br /> Eme1 essential Mus81 activity function fission      neously transfected plasmids encoding different <br /> yeast (Boddy et al., 2001). Likewise, Mms4 essential       forms Mus81, 3HaMus81, FLAG-Mus81. As shown in <br /> Mus81 activity function budding yeast (Kaliraman et        Figure 5A, FLAG-Mus81 detected immune-pre- <br /> al., 2001; Mullen et al., 2001). To test Eme1 essen-   cipitate 3HaMus81. Control samples cells were <br /> tial activity human Mus81, insect cells in-       transfected construct cross- <br /> fected baculoviruses encoding Gst-Mus81, FLAG-               reactivity immune-precipitating antibody. Like- <br /> Eme1, (Figure 4A). Immune-precipitated Gst-Mus81          wise, 3HaEme1 cotransfected FLAG-Eme1, <br /> and FLAG-Eme1 assayed using 3&#226;&#172;&#732; flap, replication         3HaEme1 detected immune-precipitates FLAG- <br /> fork, Holliday junction structure (X12). Gst-Mus81     Eme1 (Figure 5B). This analysis does distinguish the <br /> had detectable activity substrates. Like-      number Mus81, Eme1, molecules coprecipitate <br /> wise, immune-precipitated FLAG-Eme1B detectable            other; however, does demonstrate least <br /> endonuclease activity (Figure 4B). In contrast, Gst-         molecules Mus81, Eme1, associate vivo. <br /> Mus81 FLAG-Eme1 coexpressed immune-precip- <br /> itates Mus81 readily cleaved 3&#226;&#172;&#732; flap replication       RNA Interference Mus81 Expression Reduces <br /> fork structures. A lower detectable specific activity     Viability <br /> was observed using X12 substrate (Figure 4B). Thus,      The role Mus81 human cells investigated using <br /> the endonuclease activity Mus81 depends Eme1            RNAi suppress Mus81 expression (Brummelkamp et al., <br /> vice versa. A similar pattern activity seen immune-     2002). As shown Figure 6A, Mus81 protein levels were <br /> precipitates FLAG-Eme1 purification recom-        substantially reduced cells transfected with <br /> binant Mus81-Eme1 GSH-Sepharose chromatography                 pSuper vectors containing 19-nucleotide sequences tar- <br /> (our unpublished data). The data obtained immune-pre-          regions Mus81 mRNA (pSuper-178 pSuper- <br /> cipitation recombinant Mus81-Eme1 shown          292) (Brummelkamp et al., 2002). No loss Mus81 seen <br /> allows direct comparison activity immune-pre-         cells transfected control vector (pSuper). The effect <br /> cipitated endogenous Mus81. Like endogenous Mus81, re-            Mus81-RNAi cell viability determined assay- <br /> combinant Mus81-Eme1 lower activity Hol-          ing colony <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>survival</span>. As shown Figure 6B, suppression of <br /> liday junction substrate. As previously seen              Mus81 expression associated significant reduc- <br /> endogenous transfected Mus81, reaction product            tion ability cells form viable colonies. Loss of <br /> was linear duplex (Chen et al., 2001). No cleavage de-        viability specifically associated reduced <br /> tected mutant form Mus81 (Mus81AA) used.            Mus81 expression seen transfection with <br />                                                                   different constructs cause reduced Mus81 expres- <br /> Association Mus81-Eme1 Catalytic Units                         sion seen transfection control RNAi <br /> The generation linear duplex products requires coordi-         vectors. Flow cytometric analysis did reveal significant <br /> nate cleavage opposing strands X12 substrate        change cell cycle distribution 48 72 h trans- <br />  <br />  <br /> 556                                                                                                  Molecular Biology Cell <br />                                                                                                 Mus81 Required Recombination <br />  <br />  <br />  <br />  <br /> Figure 4. Endonuclease activity recom- <br /> binant Mus81-Eme1. Gst-Mus81 FLAG- <br /> Eme1B immune precipitated Sf9 <br /> cells infected viruses encoding in- <br /> dicated proteins. (A) Mus81 immune-pre- <br /> cipitates probed presence of <br /> Gst-Mus81 FLAG-Eme1B. (B) Recom- <br /> binant Mus81-Eme1 cleaves 3&#226;&#172;&#732; flap, repli- <br /> cation fork, Holliday junction (X12) <br /> structures vitro. The activity associated <br /> with Mus81 immune-precipitates from <br /> HeLa cells shown comparison (En). S, <br /> substrate alone. The black dot represents <br /> radioactive phosphate labeling sub- <br /> strates. <br />  <br />  <br />  <br /> fection pSuper-178 pSuper-292. Experiments de-        ically inactive &#226;?&#164;-galactosidase (LacZ) genes rise <br /> termine specific cause cell death infor-    LacZ&#226;&#171;&#185; cells gene conversion, unequal sister chromatid <br /> mative.                                                          exchange, intrachromosomal recombination (Herzing <br />                                                                  Meyn, 1993; Meyn, 1993; Prince et al., 2001). This system <br /> RNAi Suppression Mus81 Reduces Mitotic                        previously used demonstrate cells from <br /> Recombination                                                    ataxia telangiectasia patients increased mitotic recom- <br /> A Mus81 meiotic defect depends recombination         bination rates (Meyn, 1993) loss Werner <br /> been reported fission yeast budding yeast (In-       syndrome protein (WRN) associated decreased pro- <br /> terthal Heyer, 2000; Boddy et al., 2001; Kaliraman et al.,   ductive mitotic recombination (Prince et al., 2001; Saintigny <br /> 2001). To determine Mus81 needed mitotic          et al., 2002). A feature reporter &#226;?&#164;-galactosidase <br /> recombination human cells, took advantage            activity scored directly single cells, is <br /> SV40-transformed human fibroblast line, GM847L22 (Prince         compatible transient down-regulation use of <br /> et al., 2001), contains single integrated copy    RNAi. Furthermore, does depend long-term viabil- <br /> mitotic recombination reporter plasmid pLrec (Herzing        ity culture. After transfection plasmids that <br /> Meyn, 1993; Meyn, 1993). A schematic Lrec cassette     suppress Mus81 expression (pSuper-178, pSuper-292) or <br /> shown Figure 7A. It contains direct repeats genet-     control plasmids (pSuper pCDNA), GM847L22 cells were <br />  <br />  <br /> Figure 5. Mus81 Eme1 self-associate. <br /> 293 cells transfected 3HaMus81, <br /> FLAG-Mus81, both. Forty hours after <br /> transfection, lysates Ha immune-pre- <br /> cipitates probed presence of <br /> 3HaMus81 FLAGMus81. (A) FLAG- <br /> Mus81 detected Ha immune-precip- <br /> itates cells express 3HaMus81. (B) <br /> 3Ha-Eme1 detected FLAG immune- <br /> precipitates cells express FLAG- <br /> Eme1. <br />  <br />  <br />  <br /> Vol. 15, February 2004                                                                                                        557 <br />  V. Blais et al. <br />  <br />  <br />  <br />  <br />                                                                                         Figure 6. Suppression Mus81 by <br />                                                                                         RNAi results reduced viability. (A) <br />                                                                                         Transfection pSuper-178, pSuper- <br />                                                                                         292, pSuper results re- <br />                                                                                         duced Mus81 protein. NonTx, untrans- <br />                                                                                         fected cells. (B) HeLa cells were <br />                                                                                         transfected pSuper-178, pSuper-292, <br />                                                                                         pSuper plated low density. Col- <br />                                                                                         onies counted 10 d growth <br />                                                                                         staining Geimsa. Triplicate <br />                                                                                         plates counted point. Re- <br />                                                                                         sults average independent <br />                                                                                         transfections. <br />  <br />  <br /> grown presence thymidine increase incidence   pSuper-178, pSuper-292 plasmids. Cells processed <br /> of recombination (Lundin et al., 2002). After 24 h   stained &#226;?&#164;-galactosidase activity exactly described <br /> growth normal medium, cells stained &#226;?&#164;-galacto-     above. A similar percentage (84 &#226;&#171;&#190; 2%) &#226;?&#164;-galactosidase pos- <br /> sidase activity frequency recombination         itive cells present cases. Therefore, interpret these <br /> scored. Untransfected cultures generated 1470 &#226;&#171;&#190; 180 recom-     data indicate down-regulation Mus81 suppresses <br /> binants million cells (Figure 7B). A similar number     recombination inactive LacZ alleles rather <br /> LacZ&#226;&#171;&#185; cells seen transfection control vectors.   expression &#226;?&#164;-galactosidase activity se. <br /> The number recombinants reduced fourfold (p &#226;&#171;&#189; <br /> 0.0003) twofold (p &#226;&#171;&#189; 0.0006) cells been <br /> transfected Mus81-RNAi plasmids, pSuper-292,          RusA Rescues Mus81-dependent Recombination <br /> and pSuper-178, respectively. These data suggested        RusA rescues meiotic defect hypersensitivity to <br /> suppression Mus81 reduces mitotic recombination         agents cause replication fork stalling Mus81 mutants <br /> could indicate Mus81-RNAi interfered &#226;?&#164;-ga-      (Boddy et al., 2001; Doe et al., 2002). We reasoned if <br /> lactosidase expression. To determine Mus81-RNAi&#226;&#8364;&#8220;       suppression Mus81 human cells results accumu- <br /> transfected cells capable expressing active &#226;?&#164;-galac-   lation Holliday junctions, reduction recombination <br /> tosidase, cells transfected plasmid carrying     rescued expression active RusA (Bolt and <br /> single intact copy &#226;?&#164;-galactosidase pSuper,     Lloyd, 2002). As shown Figure 7B, expression wild-type <br />  <br />  <br /> 558                                                                                                 Molecular Biology Cell <br />                                                                                               Mus81 Required Recombination <br />  <br />  <br />  <br /> Figure 7. Suppression Mus81 re- <br /> sults reduced mitotic recombina- <br /> tion rescued expression a <br /> bacterial Holliday junction resolvase, <br /> RusA. (A) pLrec contains direct re- <br /> peat inactive LacZ genes sep- <br /> arated neomycin resistance <br /> gene (black box). Expression under <br /> the control SV40 promoter <br /> (gray box). Base pairs (693) identi- <br /> cal sequence LacZ alleles <br /> are indicated arrows. Lx2 inac- <br /> tive insertion site indi- <br /> cated X. &#226;&#338;&#172;Lx1 lacks 727 base pairs <br /> of 5&#226;&#172;&#732; sequence (Herzing Meyn, <br /> 1993). The cell-line GM847L22 con- <br /> tains single intact copy pLrec <br /> (Prince et al., 2001). (B) Incidence of <br /> LacZ&#226;&#171;&#185; cells. Cells (5 &#226;&#171;&#187; 105) were <br /> plated G418-free medium 16 h be- <br /> fore transfection indicated <br /> plasmid. The DNA trans- <br /> fected kept constant use of <br /> empty vector. Thymidine (2 mM) <br /> was added culture medium <br /> and cells grown 16 h. Cells <br /> were cultured normal growth me- <br /> dium 24 h, stain- <br /> ing &#226;?&#164;-galactosidase activity. Du- <br /> plicate dishes used monitor <br /> cell number expression Mus81 <br /> and RusA. Error bars represent data <br /> from separate experiments. (C) <br /> Ha immune-blots cell lysates indi- <br /> cate active mutant forms <br /> of RusA expressed compara- <br /> ble levels. (D) Immune-precipitates <br /> of wild-type RusA active on <br /> X12 structures 3&#226;&#172;&#732; flaps struc- <br /> tures vitro. The 80, 8, 2, 1 <br /> indicate relative proportion the <br /> enzyme added constant amount <br /> of substrate. Mutant versions of <br /> RusA, RusAN70A, Mus81-Eme1, <br /> Mus81AA, did cleave either <br /> structure. Wild-type Mus81-Eme1 <br /> converted X12 3&#226;&#172;&#732; flap struc- <br /> tures linear duplex products. <br /> Quantification product yield was <br /> calculated using Image Quant (Am- <br /> ersham Biosciences, Piscataway, NJ) <br /> software. S, substrate alone. <br />  <br />  <br />  <br /> Ha-tagged RusA did significantly affect incidence      common substrate vivo. We tested ability <br /> recombination cells transfected           RusA Mus81-Eme1 cleave X12 3&#226;&#172;&#732; flap <br /> vector (p &#226;&#171;&#189; 0.65), suggesting level expression,   structure vitro. Both enzymes recovered from <br /> RusA does drive increased recombination human              HeLa cells using Ha-antibody. As shown Figure <br /> cells. In contrast, active, nuclear RusA (RusAWT)        7D, Mus81-Eme1 readily detectable activity both <br /> cotransfected plasmids encoding Mus81-RNAi in-            substrates. Mus81-Eme1 showed 10- 12-fold ac- <br /> creased incidence recombination levels seen      tivity 3&#226;&#172;&#732; flap compared X12 the <br /> untransfected control cultures (Figure 7B). An endonuclease       range enzyme used. Likewise, RusA cleaved <br /> dead version RusA (RusAN70) did significantly in-          X12 structure, 80-fold dilution the <br /> crease number recombinants Mus81-RNAi trans-            immune-precipitate RusA detectably cleaved X12 <br /> fected cells. Immune blotting showed wild-type       structure. By contrast, incubating equal amounts RusA <br /> mutant forms RusA equally expressed (Figure 7C).          3&#226;&#172;&#732; flap structure yielded detectable product. <br /> Ha-immune-precipitates endonuclease activity assays           Even detector sensitivity set 10-fold higher prod- <br /> confirmed wild-type, mutant RusA, active         uct detected samples RusA was <br /> on Holliday junction (X12) substrate (Figure 7D). The           incubated 3&#226;&#172;&#732; flap structure. This analysis suggests <br /> observation RusA rescues Mus81-dependent recombi-            RusA cleaves 3&#226;&#172;&#732; flaps vitro, does &#226;&#172;?1000- <br /> nation suggests enzymes acting         fold efficiency acts X12 structure. <br />  <br />  <br /> Vol. 15, February 2004                                                                                                    559 <br />  V. Blais et al. <br />  <br />  <br /> DISCUSSION                                                             The hypothesis Mus81 component Holliday <br />                                                                     junction resolvase vivo challenged basis <br /> To better understand function Mus81 human cells,          ability preferentially cleave branched Y-structures <br /> we set identify human homolog Mus81                 3&#226;&#172;&#732; flaps vitro. Ciccia et al. (2003) showed recom- <br /> binding partner Eme1 Mms4. A human sequence                 binant human Mus81-Eme1 &#226;&#172;&#402;12-fold activity when <br /> limited homology fission yeast Eme1 identified            assayed using mobile Holliday junction (X26) structure <br /> PSI-BLAST. While work progress, groups              compared 3&#226;&#172;&#732; flap replication fork. Cleavage a <br /> reported identification sequence refer         fixed Holliday junction (X0) &#226;&#172;&#402;75-fold 3&#226;&#172;&#732; flap <br /> Eme1A. Ogrunc Sancar (2003) designated protein             (Ciccia et al., 2003). Our analysis recombinant Mus81- <br /> HsMms4 showed binds Mus81 vivo           Eme1 agreement with previous studies in <br /> immune complexes transfected Mus81-Mms4 (Eme1A)                  showing Mus81-Eme1 preferentially cleaves 3&#226;&#172;&#732; flaps <br /> have endonuclease activity. Ciccia et al. (2003) identified    replication forks vitro (Kaliraman et al., 2001; Con- <br /> Eme1A, use bacterial expression showed          stantinou et al., 2002; Bastin-Shanower et al., 2003; Whitby et <br /> the proteins associate form endonuclease complex.         al., 2003). However, contend ability cleave <br /> Our analysis vitro substrate specificity recombi-      branched structures vitro does disqualify the <br /> nant Mus81-Eme1 closely parallels Ciccia et al. (2003)      Mus81&#226;&#8364;&#8220;Eme1 complex resolving Holliday junctions in <br /> and indicates Mus81-Eme1 cleaves 3&#226;&#172;&#732; flaps effi-           vivo. The data presented presented Ciccia et <br /> ciently Holliday junction structures.                          al. (2003) clearly demonstrate recombinant human <br />    In study, variants Eme1 retrieved        Mus81-Eme1 capable resolving Holliday junction into <br /> the human expressed sequence tag collection American             linear duplexes vitro. Mus81-Eme1 does behave like <br /> Type Culture Collection. Because sequences included             prokaryotic Holliday junction resolvase RuvC, by <br /> both potential polymorphic variations splice variants,          displaying strong sequence selectivity, catalyzing <br /> all tested ability interact Mus81        symmetrical cleavage junctions (Chen et al., 2001; Con- <br /> and support Mus81-associated endonuclease activity.              stantinou et al., 2002). Nevertheless, Mus81-Eme1 capable <br /> Even promoter drove expression, Eme1B               catalyzing biologically critical process converting a <br /> was present slightly higher levels            four-way DNA junction linear duplexes vitro, <br /> proteins. We variants associate       vivo. Partially purified prepara- <br /> activate Mus81 differing degrees. The higher endonucle-          tions human Mus81 robust Holliday junction activ- <br /> ase activity Eme1B containing immune-precipitates                ity dependent Mus81 protein (e.g., Figure 2 of <br /> seemed result mainly increased association          Constantinou et al., 2001). Comparison activity with <br /> 3HaMus81, possible Eme1B stimulated             relatively weak activity recombinant Mus81-Eme1 <br /> Mus81 activity Eme1A Eme1C. The 13-amino               (this work; Ciccia et al., 2003), suggests cofactors may <br /> acid sequence insert Eme1A present homolo-           improve Holliday junction resolution activity Mus81- <br /> gous murine sequence (accession no. CB182498). Further-             Eme1 extracts (Boddy et al., 2001). The identification of <br /> more, presence consensus splice donor acceptor           human Eme1 facilitate characterization ancillary <br /> sites flanking insert suggest construct corre-        factors enhance Mus81-Eme1 function. <br /> sponding Eme1A result incomplete splicing.               Mus81 Eme1 colocalize nucleoli human cells <br /> Nevertheless, Eme1A bind activate Mus81 (Ciccia          proteins recruited regions damaged DNA, <br /> et al., 2003; Ogrunc Sancar, 2003). The likely source        specifically S phase (Gao et al., 2003; Figure 3). The re- <br /> Eme1C difficult predict. The clone derived           cruitment Mus81 Eme1 localized region UV <br />                                                                     damage, specifically S-phase cells, consistent the <br /> from normal tissue. Relative sequences, it <br />                                                                     hypothesis Mus81&#226;&#8364;&#8220;Eme1 complex functions re- <br /> lacks residues 303&#226;&#8364;&#8220;331 contains cysteine place of <br />                                                                     combination repair human cells. <br /> arginine carboxy terminus protein. This substi- <br />                                                                        Sequence structural analysis junction resolving <br /> tution particularly intriguing present block <br />                                                                     enzymes different sources suggests have <br /> of significant homology fission yeast Eme1 (Figure 1A). <br />                                                                     evolved independently different ancestors <br /> Determining variants Eme1 differ-          (Aravind et al., 2000; Lilley White, 2001). Despite these <br /> entially expressed, single site substitutions        differences, characterized Holliday junction resolving en- <br /> are polymorphic variants human populations,              zymes bind junction DNA dimeric form (Lilley and <br /> whether associated human disease           White, 2001). The association catalytic units enables <br /> interest.                                                           paired cleavage opposed strands four-way junc- <br />    As reported previously, immune-precipi-            tion. By use differentially tagged version Mus81 and <br /> tates 3HaMus81 associated endonuclease activity          Eme1, molecules Mus81 of <br /> the absence transfected Eme1 (Chen et al., 2001). By con-        Eme1 associate. Further analysis needed deter- <br /> trast, immune-precipitates Eme1 active unless           ability Mus81-Eme1 resolve Holliday <br /> Mus81 cotransfected. Given Mus81 expressed Sf9          junctions linear duplex dependent correct <br /> cells active absence Eme1, assume           coordination active Mus81-Eme1 heterodimers in <br /> activity Mus81-immune-precipitate reflects ability      vivo. <br /> Mus81 associate endogenous human Eme1. This idea               The mitotic phenotypes associated loss Mus81- <br /> is supported substantial increase Mus81-associated        function fission yeast budding yeast similar. <br /> endonuclease activity detected transfection Eme1. The         Disruption Mus81 results increased sensitivity geno- <br /> reason transfected Eme1 fails form active complex        toxic agents impede DNA replication Mus81 mu- <br /> endogenous Mus81 clear. Transfection Eme1A C           tation lethal combination disruption RecQ <br /> leads reproducible reduction activity associated        homologs. Nevertheless, yeast viable absence <br /> with endogenous Mus81 (Figure 2B), suggesting trans-           Mus81 (Boddy et al., 2000; Interthal Heyer, 2000). As <br /> fected Eme1A C disrupt endogenous Mus81 con-             step directly determining role Mus81-Eme1 in <br /> taining complex.                                                    human cells, used RNAi suppress expression of <br />  <br />  <br /> 560                                                                                                    Molecular Biology Cell <br />                                                                                                         Mus81 Required Recombination <br />  <br />  <br /> Mus81. This analysis allowed draw conclu-            nealing (SDSA), strand sister chromatid is <br /> sions function Mus81 human cells. The        used template extension free 3&#226;&#172;&#732; end (Kaliraman <br /> is down-regulation Mus81 results significant loss     et al., 2001; Constantinou et al., 2002; Bastin-Shanower et al., <br /> of cell viability. Suppression Mus81 protein expression         2003). SDSA attractive model double strand break <br /> was maximal 48 &#226;&#8364;&#8220;72 h transfection; however,          repair. Because accomplished forming a <br /> at time distinct morphological cell cycle defect        Holliday junction, account mitotic recombination <br /> could determined. At later time points, number           crossover (Paques Haber, 1999). However, <br /> detached, dead cells culture increased line         models SDSA based presence double <br /> reduced number Mus81-RNAi&#226;&#8364;&#8220;treated cells               strand ends generated break. In case replication <br /> able form viable colonies. Analysis Mus81-RNAi&#226;&#8364;&#8220;trans-        restart, cleavage regressed fork generate only <br /> fected cultures failed define clear cause cell death.      double strand end. Thus, replication fork recapture <br /> Mus81-Eme1 required repair problems arising           inevitably results formation Holliday <br /> DNA replication yeast. In humans, Mus81-Eme1 re-             junction (McGlynn Lloyd, 2002). If Mus81 deficiency <br /> cruited sites DNA damage, specifically cells     resulted failure cleave 3&#226;&#172;&#732; flap generated SDSA, it <br /> replicating DNA (Gao et al., 2003). It possible repli-     lead Holliday junction accumulation, thus <br /> cation problems arise frequently human cells,          expect expression Holliday junction <br /> Mus81 essential absence extraneous replica-          resolvase rescue defect SDSA. The ability RusA to <br /> tional stress. It determine         rescue Mus81-dependent recombination suggests Hol- <br /> germ-line disruption Mus81 Eme1 effect          liday junctions accumulate Mus81-deficient cells. Clearly, <br /> as acute loss somatic cells.                                    interpretation dependent knowing specificity <br />    A second finding studies inhibition         RusA. The ability RusA act retain <br /> Mus81 results reduced mitotic recombination human            activity expressed fused nuclear localization signals <br /> cells. The possibility reduced recombination rate         yeast human cells exploited several <br /> was indirect consequence loss viability associ-       studies (Doe et al., 2000; Boddy et al., 2001; Doe et al., 2002; <br /> ated Mus81-RNAi considered. We                 Saintigny et al., 2002; Bastin-Shanower et al., 2003; Odagiri et <br /> Mus81-RNAi&#226;&#8364;&#8220;treated cells expressed active &#226;?&#164;-galactosidase          al., 2003). Extensive analysis RusA shown its <br /> at frequency level control cells       nuclease activity highly specific Holliday junctions and <br /> when single intact copy LacZ open-reading frame           unlikely cleave structures vivo (Bolt and <br /> was present plasmid. The rate recombination            Lloyd, 2002; Doe et al., 2002). Consistent reports, <br /> calculated incidence Lac&#226;&#171;&#185; cells million live cells   HeLa cell expressed RusA did detectably <br /> at time harvest; therefore, Mus81-RNAi cells      cleave 3&#226;&#172;&#732; flap structures, protein robust <br /> lost point included analysis. By       activity X12 structure. The data suggest even <br /> the criteria trypan blue exclusion, cell cycle profile      expressed heterologous system, RusA retains its <br /> &#226;?&#164;-galactosidase expression remaining cells viable         preference symmetrical structures. Although cannot <br /> and capable expressing active &#226;?&#164;-galactosidase time       know certain targets RusA vivo, available <br /> of analysis. It important determine             data strongly support hypothesis acts on <br /> Mus81-deficient cells die unable process       Holliday junctions. Thus, rescue Mus81-dependent <br /> intermediates arise specifically DNA replication.      recombination Holliday junction resolvase, coupled <br /> Mus81 protein detectable quiescent cells. The in-        biochemical evidence Mus81-Eme1 resolves Hol- <br /> crease abundance seen serum stimulation reentry          liday junction vitro, suggests Mus81-Eme1 resolves <br /> into replicative cycle suggest Mus81 required specif-      Holliday junctions human cells. <br /> ically cells actively dividing (Gao et al., 2003). <br />    Evidence Mus81-Eme1 active 3&#226;&#172;&#732; flap repli- <br /> cation fork structures Holliday junctions vitro      ACKNOWLEDGMENTS <br /> predict number possible roles Mus81-Eme1 re-           We grateful Ray Monnat providing GM847L22 advice use of <br /> combination repair. A role directly cleaving replication        cells, Reuven Agami providing pSuper, members the <br /> forks proposed (Kaliraman et al., 2001; Con-              Scripps Cell Cycle group encouragement advice. P.-H.L.G. and <br />                                                                    M.N.B. Research Special Fellows Leukemia Lymphoma Society. <br /> stantinou et al., 2002; Doe et al., 2002). However, recently   This work funded National Cancer Institute grant awarded to <br /> emerged lines evidence suggest Mus81 does act          C.H.M. <br /> directly replication forks vivo. First, extreme sen- <br /> sitivity yeast strains lack Mus81 camptothecin, a <br /> topoisomerase inhibitor thought cause fork col-         REFERENCES <br /> lapse, strongly suggests Mus81 required repli-       Aravind, L., Makarova, K.S., Koonin, E.V. (2000). Survey summary: <br /> cation fork collapse (Doe et al., 2002; Bastin-Shanower et al.,    Holliday junction resolvases related nucleases: identification new <br /> 2003) consistent hypothesis Mus81         families, phyletic distribution evolutionary trajectories. Nucleic Acids <br />                                                                    Res. 28, 3417&#226;&#8364;&#8220;3432. <br /> activity required cleave stalled forks. Second, mutations <br /> in genes act early recombination (Rad51, Rad52,        Bastin-Shanower, S.A., Fricke, W.M., Mullen, J.R., Brill, S.J. (2003). The <br /> Rad54) suppress synthetic lethality Mus81-Sgs1              mechanism mus81-mms4 cleavage site selection distinguishes the <br />                                                                    homologous endonuclease rad1-rad10. Mol. Cell Biol. 23, 3487&#226;&#8364;&#8220;3496. <br /> strains (Fabre et al., 2002; Bastin-Shanower et al., 2003). If <br /> Mus81-Eme1 acts directly replication forks growth           Boddy, M.N., Gaillard, P.H., McDonald, W.H., Shanahan, P., Yates, J.R., and <br />                                                                    Russell, P. (2001). Mus81-eme1 essential components Holliday junc- <br /> defects rescued disruption genes.         tion resolvase. Cell 107, 537&#226;&#8364;&#8220;548. <br />    Does observation expression RusA rescues the <br />                                                                    Boddy, M.N., Lopez-Girona, A., Shanahan, P., Interthal, H., Heyer, W.D., and <br /> Mus81-dependent recombination defect differentiate role        Russell, P. (2000). Damage tolerance protein mus81 associates FHA1 <br /> of Mus81 remaining pathways? The observation           domain checkpoint kinase cds1. Mol. Cell Biol. 20, 8758 &#226;&#8364;&#8220; 8766. <br /> Mus81-Eme1 efficiently cleaves 3&#226;&#172;&#732; flaps vitro suggests        Bolt, E.L., Lloyd, R.G. (2002). Substrate specificity RusA resolvase <br /> role trimming flaps arise extension 3&#226;&#172;&#732;    reveals DNA structures targeted RuvAB RecG vivo. Mol. Cell <br /> end process synthesis-dependent strand an-           10, 187&#226;&#8364;&#8220;198. <br />  <br />  <br />  <br /> Vol. 15, February 2004                                                                                                                       561 <br />  V. Blais et al. <br />  <br /> Brummelkamp, T.R., Bernards, R., Agami, R. (2002). A stable          Kaliraman, V., Mullen, J.R., Fricke, W.M., Bastin-Shanower, S.A., Brill, S.J. <br /> expression short interfering RNAs mammalian cells. Science 296, 550 &#226;&#8364;&#8220;         (2001). Functional overlap Sgs1-Top3 Mms4-Mus81 endo- <br /> 553.                                                                                nuclease. Genes Dev. 15, 2730 &#226;&#8364;&#8220;2740. <br /> Chen, X.B., Melchionna, R., Denis, C.M., Gaillard, P.H., Blasina, A., Van        Karow, J.K., Constantinou, A., Li, J.L., West, S.C., Hickson, I.D. (2000). <br /> Weyer, I., Boddy, M.N., Russell, P., Vialard, J., McGowan, C.H. (2001).         The Bloom&#226;&#8364;&#8482;s syndrome gene product promotes branch migration Holliday <br /> Human Mus81-associated endonuclease cleaves Holliday junctions vitro.            junctions. Proc. Natl. Acad. Sci. USA 97, 6504 &#226;&#8364;&#8220; 6508. <br /> Mol. Cell 8, 1117&#226;&#8364;&#8220;1127. <br />                                                                                     Lilley, D., White, M. (2001). The junction-resolving enzymes. Nat. Rev. <br /> Ciccia, A., Constantinou, A., West, S.C. (2003). Identification charac-     Mol. Cell Biol. 2, 433&#226;&#8364;&#8220; 443. <br /> terization human Mus81/Eme1 endonuclease. J. Biol. Chem. 278, <br /> 25172&#226;&#8364;&#8220;25178.                                                                        Lundin, C., Erixon, K., Arnaudeau, C., Schultz, N., Jenssen, D., Meuth, M., <br />                                                                                     Helleday, T. (2002). Different roles nonhomologous end joining and <br /> Constantinou, A., Chen, X.B., McGowan, C.H., West, S.C. (2002). Holliday        homologous recombination following replication arrest mammalian cells. <br /> junction resolution human cells: junction endonucleases distinct        Mol. Cell Biol. 22, 5869 &#226;&#8364;&#8220;5878. <br /> substrate specificities. EMBO J. 21, 5577&#226;&#8364;&#8220;5585. <br />                                                                                     McGlynn, P., Lloyd, R.G. (2002). Recombinational repair restart of <br /> Constantinou, A., Davies, A.A., West, S.C. (2001). Branch migration         damaged replication forks. Nat. Rev. Mol. Cell Biol. 3, 859 &#226;&#8364;&#8220; 870. <br /> Holliday junction resolution catalyzed activities mammalian cells. <br /> Cell 104, 259 &#226;&#8364;&#8220;268.                                                                 Melchionna, R., Chen, X.B., Blasina, A., McGowan, C.H. (2000). Threonine <br />                                                                                     68 required radiation-induced phosphorylation activation cds1. <br /> Constantinou, A., Tarsounas, M., Karow, J.K., Brosh, R.M., Bohr, V.A., Hick- <br />                                                                                     Nat. Cell Biol. 2, 762&#226;&#8364;&#8220;765. <br /> son, I.D., West, S.C. (2000). Werner&#226;&#8364;&#8482;s syndrome protein (WRN) migrates <br /> Holliday junctions co-localizes RPA replication arrest. EMBO          Meyn, M.S. (1993). High spontaneous intrachromosomal recombination rates <br /> Rep. 1, 80 &#226;&#8364;&#8220; 84.                                                                    ataxia-telangiectasia. Science 260, 1327&#226;&#8364;&#8220;1330. <br /> Cox, M.M., Goodman, M.F., Kreuzer, K.N., Sherratt, D.J., Sandler, S.J.,         Michel, B. (2000). Replication fork arrest DNA recombination. Trends <br /> Marians, K.J. (2000). The importance repairing stalled replication forks.        Biochem. Sci. 25, 173&#226;&#8364;&#8220;178. <br /> Nature 404, 37&#226;&#8364;&#8220; 41. <br />                                                                                     Mizuno, T., Matsunaga, T., Ihara, M., Nikaido, O. (1991). Establishment of <br /> De Los Santos, T., Hu <br /> </body></html>