<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Bacterial Resistance Antisense Peptide Phosphorodiamidate <br /> Morpholino Oligomers <br /> Susan E. Puckett,a* Kaleb A. Reese,a Georgi M. Mitev,a Valerie Mullen,a Rudd C. Johnson,a Kyle R. Pomraning,b Brett L. Mellbye,c* <br /> Lucas D. Tilley,c* Patrick L. Iversen,c Michael Freitag,b Bruce L. Gellera,c <br /> Department Microbiologya Department Biochemistry Biophysics,b Oregon State University, Corvallis, Oregon, USA, AVI BioPharma, Inc., Corvallis, <br /> Oregon, USAc <br />  <br /> Peptide phosphorodiamidate morpholino oligomers (PPMOs) synthetic DNA mimics bind cRNA inhibit bacterial <br /> gene expression. The PPMO (RFF)3RXB-AcpP (where R arginine, F, phenylalanine, X 6-aminohexanoic acid, B &#226;?&#164;-alanine, <br /> and AcpP acyl carrier protein) complementary 11 bases essential gene acpP (which encodes acyl carrier protein). <br /> The MIC (RFF)3RXB-AcpP 2.5 &#226;?&#174;M (14 &#226;?&#174;g/ml) Escherichia coli W3110. The rate spontaneous resistance E. coli <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> (RFF)3RXB-AcpP 4 &#226;&#180;&#8250; 10&#226;&#180;&#353;7 mutations/cell division. A spontaneous (RFF)3RXB-AcpP-resistant mutant (PR200.1) iso- <br /> lated. The MIC (RFF)3RXB-AcpP 40 &#226;?&#174;M (224 &#226;?&#174;g/ml) PR200.1. The MICs standard antibiotics PR200.1 and <br /> W3110 identical. The sequence acpP identical PR200.1 W3110. PR200.1 resistant PPMOs <br /> conjugated (RFF)3RXB peptides similar composition <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> cationic nonpolar residues. Genomic se- <br /> quencing PR200.1 identified <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> sbmA, encodes active transport protein. In separate experiments, a <br /> (RFF)3RXB-AcpP-resistant isolate (RR3) selected transposome library, insertion mapped sbmA. Ge- <br /> netic complementation PR200.1 RR3 sbmA restored susceptibility (RFF)3RXB-AcpP. Deletion sbmA caused resis- <br /> tance (RFF)3RXB-AcpP. We conclude resistance (RFF)3RXB-AcpP linked peptide phosphoro- <br /> diamidate morpholino oligomer, dependent composition repeating pattern amino acids, caused mutations in <br /> sbmA. The data suggest (RFF)3R-XB PPMOs transported plasma membrane SbmA. <br />  <br />  <br />  <br /> A    ntibiotic resistance bacteria continues prob- <br />      lem. The number antibiotic-resistant pathogens increas- <br /> ing, level resistance standard antibiotics increasing, and <br />                                                                                      cationic amphipathic. Previous investigations sug- <br />                                                                                      gest repeated peptide motif cationic residue fol- <br />                                                                                      lowed hydrophobic residues im- <br /> the percentage isolates resistance multiple antibiotics                   portant feature efficient membrane penetration (39). More <br /> has risen dramatically recent years (3, 37). At time,                recently, compared variety membrane-penetrating <br /> number antibiotics developed decreased sig-                    peptides abilities enhance efficacy peptide phos- <br /> nificantly, particularly targeting Gram-negative bacteria.                     phorodiamidate morpholino oligomers (PPMOs) dif- <br /> Most new antibiotics approved use                   ferences peptides vary pattern alternating <br /> United States past 40 years new classes antibiotics                cationic nonpolar residues amino acid composi- <br /> but simply chemical derivatives antibiotic classes                   tions (27). <br /> that discovered mid-20th century (8). There                            Despite progress improving efficacy potency of <br /> urgent need new antibiotics, particularly novel                    antisense oligomers made, little known about <br /> innovative strategies targeting bacterial pathogens cause                    bacterial resistance compounds. Some naturally occur- <br /> serious diseases (3, 22).                                                            ring antimicrobial peptides, characteristics sim- <br />     Genomics created attractive potential developing                      ilar synthetic peptides used make peptide oli- <br /> innovative strategies address problem antibiotic resis-                  gomers, appear cause resistance bacteria (38). One <br /> tance. Synthetic antisense oligomers, peptide nucleic acids                  report resistance antisense morpholino oligomer a <br /> (14), phosphorothioates (16), phosphorodiamidate morpho-                         mutation region virus genome targeted oligomer <br /> lino oligomers (PMOs) (11, 15), silence expression bacterial <br /> genes. Gene-silencing oligomers decrease expression reporter <br /> genes luciferase, activate endogenous genes &#226;?&#164;-ga-                       Received 24 April 2012 Returned modification 21 May 2012 <br />                                                                                         Accepted 25 June 2012 <br /> lactosidase, inhibit growth kill bacteria targeting es- <br />                                                                                         Published ahead print 17 September 2012 <br /> sential genes (10). Antisense oligomers targeted specific, essen- <br />                                                                                         Address correspondence Bruce L. Geller, gellerb@orst.edu. <br /> tial bacterial genes reduce infections increase <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>survival</span> in <br />                                                                                         * Present address: Susan E. Puckett, Department Microbiology and <br /> mouse models infection (12, 15, 40).                                                 Immunology, Weill Cornell Medical College, New York, New York, USA; Brett L. <br />     Antisense oligomers require assistance cross outer mem-                      Mellbye, Department Microbiology, Oregon State University, Corvallis, Oregon, <br /> brane Gram-negative bacteria molecular                              USA; Lucas D. Tilley, Department Microbiology Molecular Genetics, <br /> weight polar characteristics. Short amphipathic peptides                       University Vermont, Burlington, Vermont, USA. <br /> been attached antisense oligomers, greatly im-                          S.E.P. K.A.R. contributed equally article. <br /> proved entry Gram-negative bacteria increased                            Copyright &#194;&#169; 2012, American Society Microbiology. All Rights Reserved. <br /> their potency (11, 13, 27).                                                             doi:10.1128/AAC.00850-12 <br />     Membrane-penetrating peptides diverse sequences, but <br />  <br />  <br /> December 2012 Volume 56 Number 12                                 Antimicrobial Agents Chemotherapy         p. 6147&#226;&#8364;&#8220; 6153                           aac.asm.org   6147 <br />  Puckett et al. <br />  <br />  <br />  <br /> (28). Resistance antibiotic important charac-           spread 20 agar plates (60 mm 15 mm) LB plus 20 &#226;?&#174;M peptide <br /> teristic determined drug development. The frequency            PMO. The cultures grown overnight 37&#194;&#176;C aeration, and <br /> of antibiotic resistance ultimately manifest itself clinic      colonies enumerated. <br /> and play role use particular indication.                  Screening transposome mutants. Transposome EZ-Tn5&#226;&#172;?R6K&#226;?&#165;ori/ <br />     In report, characterize spontaneous resistance             Kan-2&#226;&#172;&#381;Tnp (Epicentre, Madison, WI) electroporated E. coli <br />                                                                             XL1-Blue MRF=, 3 &#226;&#171;&#187; 103 transductants selected LB-kanamy- <br /> PPMO compare cross-resistance antibiotics, PPMOs <br />                                                                             cin (15 &#226;?&#174;g/ml) plates. The transductants pooled stored phos- <br /> with different peptides PMO, PPMOs                phate-buffered saline 15% glycerol &#226;&#171;&#186;75&#194;&#176;C. The pooled transduc- <br /> same peptide targeted different genes. Furthermore,              tants thawed, 1 &#226;&#171;&#187; 104 CFU spread LB plate that <br /> same gene causes PPMO resistance identified isolates             included 20 &#226;?&#174;M (RFF)3RXB-AcpP. Insertion mutations PPMO- <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> independent strategies selection.                               resistant mutants sequenced rescue cloning described the <br />                                                                             manufacturer (Epicentre). Insertions sbmA confirmed using PCR <br /> MATERIALS AND METHODS                                                       described previously (18). Briefly, PCR mixtures contained chromo- <br /> Bacterial strains. Wild-type Escherichia coli K-12 strain W3110 used    somal DNA extracted bacteria using commercial kit (DNeasy; <br /> for selecting spontaneous mutants resistant PPMO            Qiagen, Valencia, CA) primers (IDT Technologies, Coralville, IA) <br /> (RFF)3RXB-AcpP (where R arginine, F, phenylalanine, X 6-amino-        flank insertion site: 5=-GATTGCCGTTATCTTCTGGC 5=-G <br /> hexanoic acid, B &#226;?&#164;-alanine, AcpP acyl carrier protein). Sponta-   CTCAAGGTATGGGTTACTTCC. Thirty PCR cycles denaturation at <br /> neous mutants resistant (RFF)3RXB-AcpP selected         95&#194;&#176;C 0 s, annealing 45&#194;&#176;C 0 s, extension 72&#194;&#176;C 1 min were <br /> growth Mueller-Hinton II broth supplemented 8&#226;&#171;&#187; MIC               carried out. PCRs run 1605 air thermocycler (Idaho Technol- <br /> (RFF)3RXB-AcpP. Liquid cultures grown Mueller-Hinton         ogy, Idaho Falls, ID). <br /> II LB broth. LB agar used growth solid medium. Transfor-          Sequencing acpP. The acpP allele strain analyzed am- <br /> mants pSE380myc-luc (11) grown LB medium supple-               plified PCR (18), using template single bacterial colony picked <br /> mented 50 &#226;?&#174;g/ml ampicillin (LBA; Sigma-Aldrich, St. Louis, MO).        growth plate, Promega Taq polymerase (Madison, WI), the <br />     Oligopeptide transport mutants PA0183 (opp), PA0333 (opp dpp),          following primers (Invitrogen, Carlsbad, CA): 5=-AACGTAAAATCGTG <br /> PA0410 (opp tpp), PA0643 (opp dpp tpp), PA0610 (opp dpp tpp),           GTAAGACC-3= 5=-TAACGCCTGGTGGCCGTTGATG-3=. The <br /> which derived parent strain Morse 2034 [trpE9851 leu 277          PCR products gel purified using Qiagen MinElute PCR purifica- <br /> F&#226;&#171;&#186;IN(rrnD-rrnE)], described previously (36) gifts        tion kit (Valencia, CA) sequenced using primers shown <br /> from J. W. Payne (University Wales, Bangor, United Kingdom).             core laboratory Center Genome Research, Oregon <br />     In-frame, nonpolar <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>knockout</span> strains E. coli JW3496 (dctA knockout       State University. <br /> mutant), JW5730 (eptA knockout mutant), JW0368 (sbmA knockout               Genomic sequencing. Genomic DNA W3110 wild type and <br /> mutant) isogenic parent strain, BW25113 (2), provided        PR200.1 generated standard procedures (1). DNA sheared by <br /> from Keio collection National BioResource Project (NIG; Ja-      sonication processed Illumina high-throughput sequencing as <br /> pan). The knockout strains grown LB broth 50 &#226;?&#174;g/ml kana-       previously described (31, 32). Data analyses individual point mu- <br /> mycin (Sigma-Aldrich). The IPTG (isopropyl-&#226;?&#164;-D-thiogalactopyrano-           tations carried described previously (32). <br /> side)-inducible sbmA expression plasmid (which pSbmA, from <br /> strain b0377) vector control (pNTR-SD) (34) pro-        RESULTS <br /> vided National BioResource Project (NIG; Japan) grown LB      Spontaneous mutants resistant peptide PMOs. Spontaneous <br /> broth 20 &#226;?&#174;g/ml ampicillin. <br />                                                                             resistance apparent growth occasionally occurred in <br />     PPMOs. PPMOs synthesized AVI BioPharma, Inc. (Corvallis, <br /> OR), described previously (39). The <span id='am-393' about='xsp:base' typeof='owl:Thing'>base</span> sequence PPMOs tar-      cultures included (RFF)3RXB-AcpP (X 6-amino- <br /> geted acpP (AcpP) 5=-CTTCGATAGTG-3=, PPMOs tar-           hexanoic acid B &#226;?&#164;-alanine) concentrations the <br /> geted ftsZ (FtsZ) 5=-TCCATTGGTTC-3=, PPMOs            MIC. Growth MIC observed cultures that <br /> targeted luc (Luc) 5=-AACGTTGAGIG. Inosine place guanine      included AcpP PPMOs different peptides attached to <br /> in Luc PPMO necessary make oligomer soluble aqueous       PMO, (RX)6B-AcpP (RXR)4XB-AcpP. The <br /> solutions avoiding guanine quartet structure. The scrambled-base-    rate spontaneous resistance (RFF)3RXB-AcpP measured <br /> sequence (Scr) control 5=-TCTCAGATGGT-3=.                                4 &#226;&#171;&#187; 10&#226;&#171;&#186;7 mutations/cell generation. <br />     Antibiotics. All antibiotics bleomycin purchased           Susceptibility antibiotics growth rate. Colonies were <br /> Sigma-Aldrich. Bleomycin purchased Enzo Life Science (Farm-        isolated single liquid culture W3110 grown 8&#226;&#171;&#187; <br /> ingdale, NY). <br />                                                                             MIC (20 &#226;?&#174;M, 112 &#226;?&#174;g/ml) (RFF)3RXB-AcpP. One colony <br />     MICs. MICs determined microdilution method (5) in <br /> Mueller-Hinton II broth. For determination MICs strains XL1-Blue     (PR200.1) picked random characterized. <br /> MRF= RR3, Mueller-Hinton II broth supplemented 1%              PR200.1 equally susceptible parent strain W3110 each <br /> tryptone.                                                                   antibiotic tested (MICs, 4 &#226;?&#174;g/ml, 1.25 &#226;?&#174;g/ml, 1.25 &#226;?&#174;g/ml, 0.125 <br />     Luciferase expression. Spontaneous (RFF)3RXB-AcpP-resistant mu-         &#226;?&#174;g/ml, 10 &#226;?&#174;g/ml ampicillin, tetracycline, kanamycin, <br /> tants chemically competent transformed described pre-      polymyxin B, rifampin, respectively). These results indicate <br /> viously (29) pSE380myc-Luc (11). Overnight cultures grown         particular PPMO-resistant isolate resistant to <br /> aerobically 37&#194;&#176;C LBA diluted 2 &#226;&#171;&#187; 10&#226;&#171;&#186;2 LBA       antibiotics general. <br /> without various concentrations (8, 20, 50 &#226;?&#174;M) (RFF)3RXB-Luc           The doubling times strains PR200.1 W3110 iden- <br /> (RFF)3RXB-Scr grown aerobically 37&#194;&#176;C 7 h. Samples an-       tical, difference growth rate observed liquid or <br /> alyzed luciferase expression luminometry described previously <br />                                                                             solid medium. <br /> (11). <br />     Rate spontaneous resistance. The rate spontaneous resistance       Sequences acpP alleles. The target PMO, acpP, was <br /> peptide PMO measured method Luria Delbruck (23)        sequenced PR200.1 W3110, sequences found <br /> described previously (33). An overnight culture diluted 1 &#226;&#171;&#187; 104      identical (data shown). <br /> CFU/ml LB medium divided 20 1-ml aliquots. Each aliquot             MICs AcpP PPMOs attached various peptides. MICs for <br /> was grown overnight 37&#194;&#176;C aeration, 1 &#226;?&#174;l 50 &#226;?&#174;l   different AcpP PPMOs measured using strains PR200.1 and <br />  <br />  <br /> 6148    aac.asm.org                                                                                               Antimicrobial Agents Chemotherapy <br />                                                                                                                                            Morpholino Oligomer Resistance <br />  <br />  <br />  <br /> TABLE 1 MIC AcpP PPMOs pure cultures E. coli <br />                                                                                                                                MIC (&#226;?&#174;M [&#226;?&#174;g/ml]) <br />                                                                                b <br /> Motif, PPMO no.                                            Conjugated peptide                                                  W3110                            PR200.1 <br /> Motif 1 (C-N-N) <br />  NG-05-0200                                                RFFRFFRFFRXB                                                        2.5 (14)                         40 (222) <br />  NG-05-0653                                                DRDFDFDRDFDFDRDFDFDRXB                                              2.5 (14)                         40 (222) <br />  NG-23-248                                                 RXXRXXRXXRXB                                                        20 (102)                         80 (204) <br />  NG-06-0199                                                KFFKFFKFFKXB                                                        10 (54)                          80 (435) <br />  <br /> Motif 2 (C-N), NG-06-0073                                  RXRXRXRXRXRXB                                                       1.25 (7)                         1.25 (7) <br />  <br /> Motif 3 (C-N-C) <br />  NG-06-0076                                                RXRRXRRXRRXRXB                                                      1.25 (7)                         1.25 (7) <br />  NG-07-0795                                                RFRRFRRFRRFRXB                                                      1 (6)                            16 (94) <br />  <br /> No motif, NG-05-0246                                       RTRTRFLRRTXB                                                        20 (111)                         40 (111) <br /> a <br />   Motif 1 cationic-nonpolar-nonpolar (C-N-N). Motif 2 cationic-nonpolar (C-N). Motif 3 cationic-nonpolar-cationic (C-N-C). <br /> b <br />   X 6-aminohexanoic acid, B &#226;?&#164;-alanine, O ornithine, D indicates isomeric form residue follows. <br />  <br />  <br />  <br /> W3110 indicators (Table 1). All AcpP PPMOs tested                              Exponential cultures grown 18 h (RFF)3RXB- <br /> the base sequence different peptides attached. The                          FtsZ, targeted ftsZ, scrambled-base-sequence (Scr) <br /> attached peptides differed amino acid composi-                         control. Samples culture plated, viable <br /> tions pattern repeating sequences cationic                     cells counted. PR200.1 grew normal cell density, whereas <br /> nonpolar residues. Repeating patterns amino acids, in-                          viable cell count parent strain W3110 reduced 2 <br /> cluding cationic nonpolar residues, important features                        orders magnitude presence (RFF)3RXB-FtsZ (Fig. 1). <br /> membrane-penetrating peptides (17, 41, 42). Four AcpP                             The scrambled-base-sequence control effect growth <br /> PPMOs, including (RFF)3RXB-AcpP, peptides repeat-                             W3110 PR200.1. <br /> ing amino acid motif cationic-nonpolar-nonpolar (C-N-N),                                  In experiments, exponential cultures W3110 and <br /> and composed D-amino acids instead                            PR200.1 grown 7 h various concentrations of <br /> usual L-amino acids. One AcpP PPMO conjugated (RX)6B,                             (RFF)3RXB-Luc, targeted luciferase reporter gene, or <br /> which repeating motif cationic-nonpolar (C-N). Two                              scrambled-base-sequence control, (RFF)3RXB-Scr. A plasmid <br /> other AcpP PPMOs conjugated peptides repeating                            expresses luciferase transferred PR200.1 prior <br /> motif cationic-nonpolar-cationic (C-N-C): (RXR)4XB                                experiment. Samples culture analyzed by <br /> (RFR)4XB. Another AcpP PMO conjugated RTRTRFLRR                                   luminometry luciferase activity. The results that <br /> TXB, repeat pattern does conform                          (RFF)3RXB-Luc did inhibit luciferase PR200.1 the <br /> the repeat patterns. All PPMOs various pep-                          3 concentrations tested (Fig. 2). In comparison, W3110 showed <br /> tides attached AcpP PMO previously char-                           inhibition luciferase proportional concentration <br /> acterized effective inhibiting growth E. coli                      PPMO added. The scrambled-base-sequence control did not <br /> (27).                                                                                    inhibit luciferase strain. There differences the <br />     The results PR200.1 resistant AcpP                            growth (optical density) of cultures (data shown). <br /> PMO C-N-N peptide motif tested fully suscepti-                              Peptide transport mutants. The results presented indi- <br /> ble (RX)6B-AcpP PMO (RXR)4XB-AcpP PMO (Table <br /> 1). However, PR200.1 resistant (RFR)4XB-AcpP, which <br /> shares C-N-C motif (RXR)4XB-AcpP but, like <br /> (RFF)3RXB-AcpP, contains phenylalanine instead 6-amino- <br /> hexaonoic acid. Compared susceptible parent strain <br /> W3110, resistance (RFF)3RXB-AcpP PR200.1 increased the <br /> MIC 16-fold. PR200.1 resistant D-isomeric form of <br /> (RFF)3RXB. The MIC RTRTRFLRRTXB-AcpP, lacks a <br /> repeating amino acid motif includes phenylalanine, in- <br /> creased 2-fold PR200.1 used indicator com- <br /> pared obtained W3110 used indicator. The <br /> MICs scrambled-base-sequence PPMOs composed each <br /> of peptides used AcpP PPMOs undetectable                           FIG 1 Viable cell count 18-h cultures. Stationary cultures W3110 or <br /> (&#226;&#172;&#381;80 &#226;?&#174;M) case.                                                                  PR200.1 diluted 5 &#226;&#171;&#187; 105 CFU/ml Mueller-Hinton broth divided <br />     (RFF)3RXB-PPMOs targeted various genes. PR200.1                               three. (RFF)3RXB-FtsZ (FtsZ) scrambled-base-sequence (Scr) control <br /> tested susceptibility PPMOs, (RFF)3 pep-                        PPMO (160 &#226;?&#174;M) PPMO added. Cultures grown aerobically at <br />                                                                                          37&#194;&#176;C 18 h, samples diluted plated determine <br /> tide motif different base sequences. One PPMO comple-                             viable cells. Error bars indicate standard deviations. *, highly significant (P &#226;&#172;? <br /> mentary ftsZ, essential gene involved cell division.                   0.01) difference compared no-PPMO scrambled-base-se- <br /> The PPMO targeted luciferase reporter gene (luc).                          quence (Scr) control-treated culture. <br />  <br />  <br />  <br /> December 2012 Volume 56 Number 12                                                                                                                      aac.asm.org 6149 <br />  Puckett et al. <br />  <br />  <br />  <br />                                                                                  lates indicated transposome inserted exact <br />                                                                                  position sbmA isolate, suggesting two <br />                                                                                  colonies clones. The isolates named RR3. <br />                                                                                      RR3 characterized measuring MICs various stan- <br />                                                                                  dard antibiotics (Table 3). All standard antibiotics tested the <br />                                                                                  MIC using XL1-Blue MRF= RR3 indicator, <br />                                                                                  including peptide antibiotics, colistin polymyxin B. How- <br />                                                                                  ever, RR3 4-fold resistant peptide antibi- <br />                                                                                  otics bleomycin phleomycin. PR200.1 4-fold resis- <br /> FIG 2 Luciferase activity cultures treated PPMOs. Growing cultures       tant bleomycin (MIC &#226;&#171;&#189; 5.6 &#226;?&#174;M [8 &#226;?&#174;g/ml]) phleomycin <br /> were treated 7 h PPMO (no PPMO) 3 concentrations        (MIC &#226;&#171;&#189; 5.2 &#226;?&#174;M [8 &#226;?&#174;g/ml]) W3110 (bleomycin MIC &#226;&#171;&#189; 1.4 <br /> PPMO [(RFF)3RXB-Luc] targeted luciferase reporter gene scrambled-      &#226;?&#174;M [2 &#226;?&#174;g/ml]; phleomycin MIC &#226;&#171;&#189; 1.3 &#226;?&#174;M [2 &#226;?&#174;g/ml]). <br /> base-sequence control [(RFF)3RXB-Scr]. After 7 h, samples culture <br /> were measured luciferase activity luminometry. The experiment             The MICs (RFF)3R-AcpP (RXR)4XB-AcpP were <br /> repeated 3 times, error bars indicate standard deviations. *, highly   measured using RR3 XL1-Blue MRF= indicator strains. <br /> significant (P &#226;&#172;? 0.01) difference compared cultures PR200.1       The results RR3 32-fold resistant to <br /> same concentrations (RFF)3RXB-Luc, cultures W3110        (RFF)3RXB-AcpP 8-fold resistant (RXR)4XB- <br /> concentrations (RFF)3RXB-Scr, culture PPMO. <br />                                                                                  AcpP (Table 3). RR3 resistant PPMO with <br />                                                                                  D-amino acids ([D-(RFF)3R]XB-AcpP). Scrambled-base-se- <br />                                                                                  quence control PPMOs did inhibit growth either <br /> cate resistance PPMOs linked peptide moiety. We                RR3 XL1-Blue MRF=. <br /> hypothesized PPMO resistance caused mutation                      RR3 XL1-Blue MRF= genetically complemented with <br /> in known oligopeptide transporters. To test this,               pSbmA used measure MIC (RFF)3RXB-AcpP. <br /> MIC measured using various strains mutations one,                    pSbma fully restored susceptibility RR3 PPMO, when <br /> two, oligopeptide transporters (Table 2). The results               induced IPTG (Table 3). Interestingly, MIC signifi- <br /> show (RFF)3RXB-AcpP MIC PR200.1                     cantly complemented strains used indicators <br /> the parent (nonmutant) strain. The scrambled-base-sequence                       strains pSbmA used indicators. <br /> control (RFF)3RXB-Scr showed detectable MIC (&#226;&#172;&#381;160 &#226;?&#174;M).                        Complementation pSbmA restored susceptibility to <br />     Genomic sequencing. The genomes PR200.1 parent                    bleomycin phleomycin. <br /> strain, W3110, sequence compared. The results indi- <br /> cated total 3 genes mutations PR200.1 compared               DISCUSSION <br /> the W3110 sequence: dctA, eptA, sbmA. In dctA, 2                  This study aware characterize bacterial <br /> transition mutations bases 3958154 (T &#194;&#161; A) 395153 (A &#194;&#161;                    resistance antisense antibacterial compound. Initially, <br /> G), codon 396, caused missense                  growth occasionally unexpectedly observed rou- <br /> Ile Ala. In eptA, transversion mutation base                 tine MIC assays (RFF)3XB-AcpP present cultures levels <br /> 4339795 (T &#194;&#161; A) affected codon 259 caused missense                    4- 8-fold MIC. Similar growth observed <br /> from Ser Thr. In sbmA, transversion mutation                 MIC assays (RXR)4XB-AcpP (RX)6B-AcpP. We <br /> base 396121 (T &#194;&#161; G) changed codon 87 (Ser Ala). No                       speculate greater number X (6-aminohexanoic acid) <br /> deletions insertions detected gene.                               residues lack F PPMOs respon- <br />     Characterization deletion mutants. Mutants in-frame                  sible apparent lack spontaneous resistance these <br /> knockout mutations dctA, eptA, sbmA tested sus-                  PPMOs conditions used MIC assay. Alterna- <br /> ceptibility (RFF)3RXB-AcpP, susceptibilities com-                    tively, genetic loci involved resistance to <br /> pared parent strain (BW25113). The MIC                        (RFF)3RXB PPMOs (putative) resistance PPMOs <br /> (RFF)3RXB-AcpP (2 &#226;?&#174;M, 11 &#226;?&#174;g/ml) using                     conjugated peptides, (RXR)4XB (RX)6B. How- <br /> the dctA eptA knockout strain parent strain. The MIC ob- <br /> tained using sbmA knockout strain 32 &#226;?&#174;M (179 &#226;?&#174;g/ml). <br />     The MIC (RXR)4XB-AcpP measured using sbmA-                        TABLE 2 MIC (RFF)3R-AcpP (NG-05-0200) using oligopeptide <br /> knockout strain indicator 2 &#226;?&#174;M (11 &#226;?&#174;g/ml),                 transport mutants <br /> the isogenic parent strain.                                                                                      MIC <br />     Complementation pSbmA. PR200.1 genetically                          E. coli strain     Mutation/<span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span>                &#226;?&#174;M          &#226;?&#174;g/ml <br /> complemented IPTG-inducible expression plasmid that <br />                                                                                  Morse 2034         Wild-type oligopeptide            5           28 <br /> encodes sbmA (pSbma) control. The complemented                                           transport <br /> strain grown IPTG used measure MIC                        PA0183             opp knockout, oligopeptide        5           28 <br /> (RFF)3RXB-AcpP. The MICs 1 &#226;?&#174;M 32 &#226;?&#174;M in-                                              permease deletion <br /> duced, sbmA complemented strain vector control                     PA0333             dpp opp knockout,             5           28 <br /> strain, respectively.                                                                                 PA0183 plus dipeptide <br />     Transposome mutants. E. coli XL1-Blue MRF= mu-                                                permease deletion <br /> tagenized transposome EZ-Tn5, library 1 &#226;&#171;&#187; 104                  PA0643             tpp, dpp, opp                 5           28 <br /> mutants spread selection plates included 20 &#226;?&#174;M                                            knockout, PA0333 plus <br /> (RFF)3R-AcpP. Two colonies grew selection plate,                                       tripeptide permease <br />                                                                                                       mutant <br /> mutated gene sequenced. The sequences iso- <br />  <br />  <br /> 6150    aac.asm.org                                                                                                 Antimicrobial Agents Chemotherapy <br />                                                                                                                   Morpholino Oligomer Resistance <br />  <br />  <br />  <br /> TABLE 3 MICs standard antibiotics PPMOs using transposome mutant RR3, isogenic parent strain XL1-Blue MRF=, sbmA <br /> complemented strains <br />                                                       MIC (&#226;?&#174;M [&#226;?&#174;g/ml]) <br />                                                                                                       XL1-Blue MRF= <br /> PPMO no.              Antibiotic PPMO              XL1-Blue MRF=            RR3                    (pSbmA)                    RR3(pSbmA) <br />                       Polymyxin B                     0.8 (1)                  0.8 (1)                0.8 (1)                    0.8 (1) <br />                       Colistin                        0.9 (1)                  0.9 (1)                0.9 (1)                    0.9 (1) <br />                       Erythromycin                    34 (25)                  34 (25)                34 (25)                    34 (25) <br />                       Rifampin                        6 (5)                    6 (5)                  6 (5)                      6 (5) <br />                       Bleomycin                       0.2 (0.25)               0.7 (1)                0.01 (0.016)               0.01 (0.016) <br />                       Phleomycin                      0.3 (0.5)                1.3 (2)                0.02 (0.03)                0.02 (0.03) <br /> NG-05-0200            (RFF)3RXB-AcpP                  2 (11)                   64 (355)               2 (11)                     2 (11) <br /> NG-05-0653            [D-(RFF)3R]XB-AcpP              2 (11)                   64 (355)               2 (11)                     2 (11) <br /> NG-06-0076            (RXR)4XB-AcpP                   1 (6)                    8 (48)                 0.1 (0.6)                  0.1 (0.6) <br /> NG-05-0655            (RFF)3RXB-Scr                   &#226;&#172;&#381;128 (&#226;&#172;&#381;714)              &#226;&#172;&#381;128 (&#226;&#172;&#381;714)            &#226;&#172;&#381;128 (&#226;&#172;&#381;714)                &#226;&#172;&#381;128 (&#226;&#172;&#381;714) <br /> NG-06-0078            (RXR)4XB-Scr                    &#226;&#172;&#381;128 (&#226;&#172;&#381;714)              &#226;&#172;&#381;128 (&#226;&#172;&#381;714)            &#226;&#172;&#381;128 (&#226;&#172;&#381;714)                &#226;&#172;&#381;128 (&#226;&#172;&#381;714) <br />  <br />  <br />  <br /> ever, rigorously pursued resistance        However, say target-site mutations do <br /> two PPMOs, certainly possible spontaneous resis-          occur uncharacterized mutants. Neverthe- <br /> tance occur appropriate conditions.                            less, target-site mutations statistically improbable con- <br />     The rate spontaneous resistance (RFF)3RXB-AcpP             sidering 4 wobble bases target region of <br /> similar rate spontaneous mutation individual genes      acpP possibly lead decrease efficacy without <br /> E. coli, typically 10&#226;&#171;&#186;6 10&#226;&#171;&#186;7 mutations/gene/        changing amino acid sequence targeted protein. <br /> generation (6, 24). This suggests genes which,            Another hypothesis resistance PR200.1 caused <br /> when mutated, rise PPMO-resistant phenotype.             mutation oligopeptide transporter. However, oligopep- <br /> However, rate mutation vary widely, measure-          tide transport mutants just susceptible (RFF)3RXB- <br /> ment rate mutation function variables (25),       AcpP isogenic parent strain. This showed resistance in <br /> including concentration antibiotic used selection         PR200.1 caused mutation known oligopeptide <br /> the number genes loci capable causing resistance pheno-       transporters tested. <br /> type. Ultimately, rate resistance PPMOs vivo              In effort identify mutation PR200.1 is <br /> conditions specific infections meaningful           responsible resistance (RFF)3RXB-AcpP, genome of <br /> measure usefulness clinic.                               PR200.1 sequenced. The results showed missense mutations <br />     The spontaneous mutant PR200.1 susceptible small-         3 genes compared sequence PPMO-suscepti- <br /> molecule antibiotics tested. This shows resistance           ble strain: dctA, eptA, sbmA. In-frame, nonpolar deletion mu- <br /> PPMO caused change physiology result        tations gene showed three, strain with <br /> resistance antibiotics general. Such general changes           sbmA deletion resistant (RFF)3RXB-AcpP. Further- <br /> known occur include reduction net negative charge        more, complementation PR200.1 sbmA restored suscepti- <br /> of lipopolysaccharide Gram-negative bacteria (7, 30),             bility PPMO. These results mutations sbmA <br /> changes capsule polysaccharide (4), changes expression          cause resistance (RFF)3RXB-AcpP. <br /> outer membrane porins (30), alterations outer membrane lipid              The MIC (RFF)3RXB-AcpP slightly lower the <br /> composition result decreased membrane permeability               strain sbmA deletion PR200.1 used the <br /> (30), activation overexpression multidrug efflux pumps         indicator. However, strains originated different parent <br /> (21). We tested variety antibiotics, hydro-       strains, probably accounts difference suscepti- <br /> philic (ampicillin, kanamycin), hydrophobic (rifampin, tetracy-          bility. The parent strain strain sbmA deletion was <br /> cline), amphiphilic (polymyxin B) enter             slightly susceptible PPMO parent strain <br /> Gram-negative bacteria outer membrane porins (am-            PR200.1. <br /> picillin, tetracycline) outer membrane lipid bilayer          The transposome mutant RR3 resistant both <br /> (rifampin, polymyxin B). The results suggested mutation         (RFF)3RXB-AcpP (RXR)4XB-AcpP. This differs re- <br /> in PR200.1 specific (RFF)3RXB-AcpP PPMOs simi-            sult spontaneous mutant PR200.1 strain the <br /> lar peptide moieties. Later, following identification mu-     sbmA deletion, resistant (RFF)3RXB-AcpP not <br /> tation sbmA, PR200.1 mildly resistant (4-fold)        (RXR)4XB-AcpP. This suggests RR3 polar effect the <br /> to peptide antibiotics bleomycin phleomycin.                     gene downstream sbmA (yaiW) responsible re- <br />     We hypothesized resistance caused mutation          sistance (RXR)4XB-AcpP. yaiW predicted DNA-binding <br /> the sequence acpP targeted PPMO. We previously            transcriptional regulator. A mutation sbmA apparently suffi- <br /> shown one-base mismatch near 3=end PPMO tar-             cient cause resistance (RFF)3RXB-AcpP, (RXR)4- <br /> geted acpP Burkholderia cepacia complex raised MIC         AcpP. <br /> by factor 8- 32-fold (15). However, present study       PR200.1 susceptible (RXR)4XB-AcpP resistant to <br /> found mutation target region acpP resistant     (RFR)4XB-AcpP, PPMOs share C- <br /> mutant. Therefore, hypothesis case disproven.            N-C repeat motif. This indicates amino acid composition <br />  <br />  <br /> December 2012 Volume 56 Number 12                                                                                            aac.asm.org 6151 <br />  Puckett et al. <br />  <br />  <br />  <br /> of PPMOs important repeating pattern            sbmA deletion somewhat susceptible to <br /> of amino acids determining resistance PR200.1. The similar-        (RFF)3RXB-AcpP, albeit high concentrations. We speculate <br /> ity resistance PR200.1 (RFF)3RXB-AcpP             PPMOs able cross plasma membrane passing <br /> (RFR)4XB-AcpP complete susceptibility (RXR)4XB-AcpP               lipid bilayer manner cross the <br /> may suggest X (6-aminohexanoic acid) accounts dif-          outer membrane. There additional active transporters <br /> ference. This supported result (Table 1) level     specificities nucleic acid oligomers. The possibility <br /> resistance (RXX)3RXB-AcpP 4-fold level re-        suggested ability PMOs (not conjugated peptide) <br /> sistance (RFF)3RXB-AcpP,               inhibit gene expression strains porous outer membranes <br /> repeating pattern cationic nonpolar amino acids           allow passage large oligomers (11, 12). <br /> former contains X. Perhaps unusual 6-carbon backbone                In summary, results suggest bacterial resistance a <br /> of X causes conformational change disallows interaction           PPMO determined peptide PMO. The <br /> with SbmA. Alternatively, sbmA mutants resis-            rate occurrence spontaneous resistance (RFF)3RXB-AcpP <br /> tant peptides F (phenylalanine). There positive trend       similar spontaneous changes bacterial phe- <br /> between number F residues peptide resistance.          notypes. In PR200.1 RR3, resistance caused mutations in <br /> This supported results higher resistance          sbmA. Our results, combination known role SbmA <br /> PPMOs F [such (RFF)3RXB-AcpP (RFR)4XB-                  peptide antibiotic uptake, suggest SbmA acts trans- <br /> AcpP], resistance PPMOs fewer F residues (such           porter (RFF)3RXB-AcpP periplasm cytoplasm. <br /> RTRTRFLRRTXB-AcpP), resistance PPMOs F <br /> [such (RXR)4XB-AcpP (RXR)6XB-AcpP],                      ACKNOWLEDGMENTS <br /> (RXX)3RXB-AcpP exception trend.                            This work supported AVI BioPharma, Inc., Howard <br />     sbmA encodes active transporter bleomycin           Hughes Medical Institute (through undergraduate student research fel- <br /> peptide antibiotics (19, 26, 35, 43). Our results consistent    lowships Susan E. Puckett Valerie Mullen). Preparation Illumina <br /> SbmA acting active transporter (RFF)3RXB-AcpP. sbmA           sequencing libraries data analyses supported start-up funds <br /> homologs widely conserved bacteria (9). The homolog            OSU Computational Genome Biology Initiative Michael <br /> of sbmA Rhizobium meliloti, bacA, required symbiosis      Freitag. <br /> alfalfa (9). The homolog Brucella abortus virulence factor          We thank Andrew Karplus critical discussion. <br /> important intracellular survival macrophages (20). It            Bruce L. Geller employed AVI BioPharma, Inc., and <br />                                                                          Oregon State University. <br /> been proposed physiological substrates SbmA or- <br /> ganic signaling molecules (43). An assay measure uptake of <br />                                                                          REFERENCES <br /> PPMOs currently available used define fur- <br />                                                                           1. Ausubel FM, et al. 1998. Current protocols molecular biology. John <br /> ther role SbmA resistance PPMO.                                 Wiley &amp; Sons, Inc, New York, NY. <br />     The substrate specificity SbmA investigated           2. Baba T, et al. 2006. Construction Escherichia coli K-12 in-frame, single- <br /> found quite flexible. Initially, specificity proposed       <span id='am-315' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-316' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants: Keio collection. Mol. Syst. Biol. 2:2006.0008. <br /> be associated thiazole oxazole structural motif (43).              doi:10.1038/msb4100050. <br />                                                                           3. Boucher HW, et al. 2009. Bad bugs, drugs: ESKAPE! An update <br /> Later, proline-rich antimicrobial peptides shown trans- <br />                                                                              Infectious Diseases Society America. Clin. Infect. Dis. 48:1&#226;&#8364;&#8220;12. <br /> ported SbmA (26). However, (RFF)3RXB-AcpP                  4. Campos MA, et al. 2004. Capsule polysaccharide mediates bacterial re- <br /> these structural features. If SbmA transporter                     sistance antimicrobial peptides. Infect. Immun. 72:7107&#226;&#8364;&#8220;7114. <br /> (RFF)3RXB-AcpP, specificity SbmA apparently lim-            5. Clinical Laboratory Standards Institute. 2006. Methods dilution <br /> ited thiazole- oxazole-containing compounds proline-             antimicrobial susceptibility tests bacteria grow aerobically, p 10.2&#226;&#8364;&#8220; <br />                                                                              10.3. Approved standard, 7th ed. Clinical Laboratory Standards Insti- <br /> rich peptides. Our results suggest substrate specificity         tute, Wayne, PA. <br /> SbmA flexible accommodate polypeptides               6. Drake J, Charlesworth WB, Charlesworth D, Crow JF. 1998. Rates of <br /> thiazole, oxazole, proline. With peptides used         spontaneous mutation. Genetics 148:1667&#226;&#8364;&#8220;1686. <br /> conjugates, specificity appears linked spacing        7. Ernst RK, Guina T, Miller SI. 2001. Salmonella typhimurium outer <br />                                                                              membrane remodeling: role resistance host immunity. Microbes <br /> cationic nonpolar amino acid residues context of <br />                                                                              Infect. 3:1327&#226;&#8364;&#8220;1334. <br /> the peptide. It noteworthy all-D-enantiomer con-         8. Fischback MA, Walsh CT. 2009. Antibiotics emerging pathogens. <br /> jugate (NG-05-0653) MIC values all-L-enan-               Science 325:1089 &#226;&#8364;&#8220;1093. <br /> tiomer conjugate (NG-05-0200) parental resistant strains.         9. Gazebrook J, Ichige A, Walker GC. 1993. A Rhizobium meliloti homolog <br /> This contrast results shown all-D-isomer              Escherichia coli peptide antibiotic-transport protein SbmA essen- <br />                                                                              tial bacteroid development. Genes Dev. 7:1485&#226;&#8364;&#8220;1497. <br /> proline-rich antimicrobial peptide Bac7(1-35), ineffec-        10. Geller BL. 2005. Antisense antibiotics. Curr. Opin. Mol. Ther. 7:109 &#226;&#8364;&#8220;113. <br /> tive compared all-L-isomer form (26). It suggested       11. Geller BL, et al. 2003. Inhibition gene expression Escherichia coli by <br /> th <br /> </body></html>