<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Younger Genes Are Less Likely Be Essential Older <br /> Genes, Duplicates Are Less Likely Be Essential than <br /> Singletons Same Age <br /> Wei-Hua Chen,1 Kalliopi Trachana,1 Martin J. Lercher,*,2 Peer Bork*,1 <br /> 1 <br />  Structural Computational Biology <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, European Molecular Biology Laboratory, Heidelberg, Germany <br /> 2 <br />  Institute Computer Science, Heinrich-Heine-University, Du&#194;&#168;sseldorf, Germany <br /> *Corresponding author: E-mail: bork@embl.de; lercher@cs.uni-duesseldorf.de. <br /> Associate editor: Arndt von Haeseler <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> Recently duplicated genes believed overlap function expression. A priori, likely be <br /> essential. Although observed yeast, mouse singletons duplicates reported equally often <br /> essential. This contradiction partly explained experimental biases. We older genes <br /> (i.e., genes earlier phyletic origin) likely essential, regardless duplication status. At given <br /> phyletic gene age, duplicates likely essential compared singletons. The &#226;&#8364;&#732;&#226;&#8364;&#732;paradoxical&#226;&#8364;&#8482;&#226;&#8364;&#8482; high <br /> essentiality mouse gene duplicates caused different age profiles singletons duplicates, the <br /> latter tending derived older genes. <br /> Key words: gene essentiality, yeast, mouse, phyletic age, linking genotype <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br />  <br />  <br /> In model organisms mouse yeast, phenotypic                          cates statistically significantly lower in <br /> changes caused single-gene mutations assayed                         singletons (Su Gu 2008; Makino et al. 2009). <br /> a genome-wide scale (Kelly et al. 2001; Blake et al. 2011). Of                     However, authors did explore immediate <br /> particular essential genes, removal re-                      conclusions studies (Su Gu 2008; Makino <br /> sults death infertility. Many expressed genes perform-                    et al. 2009): 1) genes derived old duplications more <br /> ing important molecular functions nonessential. In                          likely essential singletons 2) developmental <br /> these cases, likely gene deletion partially               duplicates likely essential developmental <br /> compensated gene overlapping function                           singletons (or singletons whole). Both conclu- <br /> and expression.                                                                 sions hold true older latest versions <br />    Gene duplication believed important source                       mouse phenotype data sets (table 1). This appears <br /> of functional redundancy (Ohno 1970). Accordingly,                         contradict duplication-functional redundancy <br /> the proportion essential genes (PE) duplicates                      concept, consider issue unresolved. <br /> much lower singletons yeast (Gu et al.                               What factors duplication status affect gene es- <br /> 2003). However, expected trend confirmed                           sentiality? Developmental genes likely essential <br /> in mouse, proportion essentials dupli-                       nondevelopmental genes (Makino et al. 2009), this <br /> cates comparable (Liao Zhang 2007) lower                         apply duplicates singletons alike. It also <br /> (table 1) singletons.                                                suggested hubs protein&#226;&#8364;&#8220;protein interaction networks <br />    The contradicting results mouse initially inter-                     likely essential (Jeong et al. 2001); however, this <br /> preted evidence widespread functional redun-                         observation probably reflects biases proteins large <br />  <br />                                                                                                                                                      Letter <br /> dancy duplicates (Liao Zhang 2007);                                 essential protein complexes (Zotenko et al. 2008). <br /> interpretation hotly disputed (Su Gu 2008; Liang                           Previous studies indicated phyletic origin (age) <br /> and Li 2009; Makino et al. 2009). At time (Liao                        genes, defined evolutionarily distant species <br /> Zhang 2007; Su Gu 2008; Liang Li 2009; Makino                           group homologs (Wolf et al. 2009), is <br /> et al. 2009), ;5,000 mouse genes tested                           correlated gene features (Hao et al. 2010). <br /> in <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> experiments. Biases expected                           Genes originated early tend conserved across <br /> subset mouse genes, genes known severe muta-                         species, highly broadly expressed, broadly useful <br /> tional phenotypes selected higher priority.                       (Hao et al. 2010). Thus, hypothesized knocking <br /> Two follow-up studies (Su Gu 2008; Makino et al. 2009)                      phyletically old genes likely severe phe- <br /> discovered knockout data enriched                      notypic effects: old genes essential. <br /> genes derived old duplications developmental                           <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> test idea, classified mouse yeast genes <br /> genes; correcting biases, overall PE dupli-                  different age groups according earliest phyletic <br />  <br /> &#194;&#169; The Author(s) 2012. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License <br /> (http://creativecommons.org/licenses/by-nc/3.0), permits unrestricted non-commercial use, distribution, and <br /> reproduction medium, provided original work properly cited.                                                  Open Access <br /> Mol. Biol. Evol. 29(7):1703&#226;&#8364;&#8220;1706. 2012 doi:10.1093/molbev/mss014 Advance Access publication January 19, 2012                                1703 <br />  Chen et al. &#194;&#183; doi:10.1093/molbev/mss014                                                                                                                                                                                                MBE <br /> Table 1. Proportions Essential Genes Different Gene                                                                                          true mouse yeast (fig. 1). Thus, duplicated <br /> Categories Two Phenotypical Data Sets Mouse.                                                                                            genes tend likely essential. Furthermore, <br />                                                                   Proportion Essential Genes (%)                                                singletons duplicated genes, fraction of <br />                                                                 Current                                 Data Makino                           essential genes increases increasing age; thus, older <br /> Categories                                                     Data Seta                                  et al. (2009)                            genes likely essential (fig. 1). The <br /> All genes                                                     43.3                                             42.07                               trends observed figure 1 reproduced restricting <br /> Duplicates                                                    43.9 (41.6b)                                     41.92                               analysis developmental genes nondeve- <br /> Singletons                                                    41.1                                             42.61                               lopmental genes (Supplementary figs. S1 S2, Supple- <br /> All developmental genes                                       62.53                                            59.51                               mentary Material online; raw data, see <br /> Developmental duplicates                                      64.75                                            60.9 <br /> Developmental singletons                                      53.1                                             53.36                               Supplementary table, Supplementary Material online). <br /> Old duplications (Ksc &#226;&#8364;&#161; 2)                                    47.31                                            44.94                                   Gene duplicates ages: age gene fam- <br /> a <br />   MGI 4.4 (October 2010). <br />                                                                                                                                                    ily (phyletic age; fig. 1) age recent du- <br /> b <br />   If genes valid phyletic ages used.                                                                                                 plication event (duplication age). The effect phyletic age <br /> c <br />   If gene multiple duplicates, pairwise Ks (the number synonymous                                                                     likely similar duplicates singletons. In addi- <br /> substitutions synonymous site) gene duplicates be <br /> calculated, lowest Ks value used. Synonymous substitutions most <br />                                                                                                                                                    tion, functional redundancy expected strongly af- <br /> genes Ks   2 reached saturation, hence, corresponding                                                                       fected age duplication event, duplicates <br /> genes tend older genes Ks , 2.                                                                                                derived ancient duplications likely es- <br />                                                                                                                                                    sential genes derived recent duplications (Su and <br />                                                                                                                                                    Gu 2008). In mouse gene duplicates, essentiality reaches <br /> origin (see Materials Methods). We classified genes                                                                                         plateau Fungi/Metazoa group does in- <br /> specific taxonomic groups yeast (fig. 1A)                                                                                       crease older age groups. According to <br /> and broad taxonomic groups mouse (fig. 1C).                                                                                                reasoning above, plateau caused com- <br /> Because large differences yeast mouse,                                                                                          parably young duplication age. We two <br /> we did attempt direct cross-species comparisons                                                                                            oldest groups contain higher fractions younger dupli- <br /> and did attempt map histories common                                                                                           cates &#226;&#8364;&#732;&#226;&#8364;&#732;Fungi/Metazoa&#226;&#8364;&#8482;&#226;&#8364;&#8482; group (Supplementary <br /> timescale.                                                                                                                                         fig. S3, Supplementary Material online). <br />    We age group, PE sin-                                                                                              In phyletic age group, duplicates likely to <br /> gletons higher duplicated genes;                                                                                         essential singletons (fig. 1). Why same <br />  <br />  <br />                              A                                                                                                                                                    B <br />                                       % essential genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       10 20 30 40 <br />  <br />  <br />  <br />                                                                                                                                                              Singlets <br />                                                                   10 20 30 40 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              Duplicates <br />                                                                   0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       0 <br />  <br />  <br />  <br />  <br />                                                                                   s                        ta                   oa                                        es                                  s                 es <br />                                                                                 sm                      yo                    az                ot                      al                                  et               at <br />                                                                                                   r                                         yc                       t                                gl <br />                                                                                               ka                         et              om                  y   ce                              Si <br />                                                                                                                                                                                                        n               pl <br />                                                                                                                                                                                                                           ic <br />                                                                rg                            Eu                          /M              c                                                                           Du <br />                                                              ro                                                    n   gi              As                   r om <br />                                                                                                                                                        ha <br />                                   lul                                                                           Fu <br />                                el                                                                                                                    cc <br />                              C                                                                                                                 Sa <br />  <br />                              C <br />                                       % essential genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                              Singlets             D <br />                                                                                                                                                                                            40 <br />                                                                   50 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              Duplicates <br />                                                                                                                                                                                            20 <br />                                                                   30 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                  s                es <br />                                                                   10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                              et                at <br />                                                                                                                                                                                                         n  gl                ic <br />                                                                                      s <br />                                                                                                 ot <br />                                                                                                                      oa              oa               at <br />                                                                                                                                                                            ia                      Si                   pl <br />                                                                                 ism           ry                     az              az            rd                      al                                        Du <br />                                                                       n                                         et              et            o                     m <br />                                                                   rga                     uk                   i/M               M           Ch                  am <br />                                                              ro                          E                                                                      M <br />                                               la                                                          ng <br />                                  llu                                                               Fu <br />                              Ce <br />  <br /> FIG. 1. In yeast (A) mouse (C), genes recent phyletic origins likely essential, duplicated genes compared <br /> with singletons phyletic age. However, ignoring age, overall proportion essential genes singletons higher yeast (B) but <br /> lower mouse (D) compared duplications. Filled circles (C) indicate proportion essential genes corresponding <br /> duplication groups higher closer overall PE singletons (41.1%; dashed horizontal line); hollow circles indicate <br /> that PE lower. <br />  <br /> 1704 <br />  Younger Duplicated Genes Are Less Likely Be Essential &#194;&#183; doi:10.1093/molbev/mss014                                        MBE <br /> not true disregarding age, previous stud-              Materials Methods <br /> ies (Liao Zhang 2007; Su Gu 2008; Liang Li <br /> 2009; Makino et al. 2009)? This fact instance              We determined phyletic origins genes yeast, <br /> Simpson&#226;&#8364;&#8482;s paradox (Simpson 1951), arise                      mouse, fly using method described Wolf et al. <br /> when dependence categorical variables (essen-               (2009) modifications (for details, Supple- <br />                                                                        mentary text, Supplementary Material online, for <br /> tiality duplication status) variable (phyletic <br />                                                                        results, Supplementary table, Supplementary Material <br /> age) disregarded. To illustrate mathematics behind <br />                                                                        online). We separated genes singletons duplicates <br /> this paradox, divided age groups duplicated <br />                                                                        previously described (Liao Zhang 2007; Makino et al. <br /> genes parts reflecting coarse <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of <br /> age: including age groups (the &#226;&#8364;&#732;&#226;&#8364;&#732;old part,&#226;&#8364;&#8482;&#226;&#8364;&#8482; filled           2009). We grouped duplicates gene families using <br /> circles fig. 1C) higher PEs               clustering-based method (Markov cluster algorithm <br /> overall singletons including remaining                   [MCL]; Enright et al. 2002) used ancient <br /> two groups lower PEs (the &#226;&#8364;&#732;&#226;&#8364;&#732;young part,&#226;&#8364;&#8482;&#226;&#8364;&#8482; open circles            origin members age corresponding family. <br />                                                                           We obtained phenotypic data species <br /> in fig. 1C). This partitioning results higher overall pro- <br />                                                                        online gene essentiality database (Chen et al. 2012), <br /> portion essential genes old duplicate part <br />                                                                        originally published Saccharomyces <br /> (Pold <br />    E 544:89%) higher compared overall sin- <br />                                                                        Genome Deletion Project (Cherry et al. 1997), Mouse <br /> gletons (Psingleton <br />              E       541:1%), corresponding <br /> proportion                young      duplicate             Genome Informatics (Blake et al. 2011), authors of <br /> (Pyoung 522:97%) lower. The overall PE duplicates re-            Chen et al. (2010), respectively. We restricted anal- <br />    E <br /> gardless age calculated weighted              yses genes tested phenotypic data <br /> average:                                                               sets. <br />  <br />             PE 5 fold   Pold <br />                          E   &#195;&#190; fyoung   Pyoung <br />                                          E     ;                       Supplementary Material <br /> where fold fyoung fraction duplicates contained         Supplementary text, Supplementary figures, Supple- <br /> in &#226;&#8364;&#732;&#226;&#8364;&#732;old&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;young&#226;&#8364;&#8482;&#226;&#8364;&#8482; parts, respectively (with fold &#195;&#190; fyoung   mentary table available Molecular Biology and <br /> 5 1) (for details, Supplementary text Supple-             Evolution online (http://www.mbe.oxfordjournals.org/). <br /> mentary table, Supplementary Material online). In theory, <br /> the overall PE high 44.89% low as <br /> 22.97%, depending values fold fyoung. In             Acknowledgments <br /> study, vast majority duplicates de- <br /> rived old gene families (fold 5 84.66%), resulting          SysteMtb grant #241587 P.B. provided funding the <br /> overall PE 41.6% duplicates (see Supplementary table,           open access license. <br /> Supplementary Material online). Thus, surprising result <br /> of higher essentiality mouse duplicates compared               References <br /> with singletons caused different age profile single-        Blake JA, Bult CJ, Kadin JA, Richardson JE, Eppig JT. Mouse Genome <br /> tons duplicated gene families.                                         Database G. 2011. The Mouse Genome Database (MGD): <br />     Our results differ significantly recent publication             premier model organism resource mammalian genomics and <br /> on Drosophila melanogaster (Chen et al. 2010). Based                    genetics. Nucleic Acids Res. 39:D842&#226;&#8364;&#8220;D848. <br /> RNAi knockdowns ;440 genes, Chen et al.                  Chen S, Zhang YE, Long M. 2010. New genes Drosophila quickly <br /> ;30% young genes (,35 myr) essential compared                      essential. Science 330:1682&#226;&#8364;&#8220;1685. <br />                                                                        Chen WH, Minguez P, Lercher MJ, Bork P. 2012. OGEE: online <br /> with ;35% old genes (.40 myr). The authors concluded <br />                                                                            gene essentiality database. Nucleic Acids Res. 40:D901&#226;&#8364;&#8220;D906. <br /> that &#226;&#8364;&#732;&#226;&#8364;&#732;young genes essential old genes terms           Cherry JM, Ball C, Weng S, et al. (11 co-authors). 1997. Genetic <br /> viability&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Chen et al. 2010). We reanalyzed data using             physical maps Saccharomyces cerevisiae. Nature 387: <br /> our methods, differ Chen et al. age                 67&#226;&#8364;&#8220;73. <br /> classification separate analysis duplicates          Enright AJ, Van Dongen S, Ouzounis CA. 2002. An efficient <br /> singletons (for raw data, Supplementary table,                     algorithm large-scale detection protein families. Nucleic <br /> Supplementary Material online). We                          Acids Res. 30:1575&#226;&#8364;&#8220;1584. <br />                                                                        Gu Z, Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH. 2003. Role <br /> proportion essential genes singletons and <br />                                                                            duplicate genes genetic robustness null mutations. <br /> duplicates general increases increasing age                 Nature 421:63&#226;&#8364;&#8220;66. <br /> five age groups, fluctuations (Supplementary                 Hao L, Ge X, Wan H, Hu S, Lercher M, Yu J, Chen W-H. 2010. Human <br /> fig. S4, Supplementary Material online). However, PE                   functional genetic studies biased medically most <br /> in duplicates lower singletons sim-               relevant primate-specific genes. BMC Evol Biol. 10:316. <br /> ilar age, differences statistically significant        Jeong H, Mason SP, Barabasi AL, Oltvai ZN. <br /> </body></html>