<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                      NIH Public Access <br />                       Author Manuscript <br />                       Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                   Published final edited form as: <br />                    Chem Res Toxicol. 2012 November 19; 25(11): 2542&#226;&#8364;&#8220;2552. doi:10.1021/tx300337j. <br />  <br />  <br />  <br />                   Novel Oxidatively Activated Agents Modify DNA are <br />                   Enhanced Ercc1 Silencing <br />                   Amy R. Jones*, Tiffany R. Bell-Horwath*, Guorui Li*, Stephanie M. Rollmann**,&#194;&#167;, Edward <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                   J. Merino*,&#194;&#167; <br />                   **Department Biological Sciences, University Cincinnati, Cincinnati, Ohio 45221 <br />  <br />                   *Department     Chemistry, University Cincinnati, Cincinnati, Ohio 45221-0172 <br />  <br />                   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                         Agents chemically modify DNA form backbone cancer treatments. A key problem <br />                         DNA modifying agents lack specificity. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> address issue, designed novel <br />                         molecular scaffolds, termed An-Hq An-Hq2, activated hallmark some <br />                         cancers: elevated concentrations reactive oxygen species. Elevated reactive oxygen species are <br />                         linked oncogenesis increase aggressive cancers. The agents are <br />                         quinones that, oxidation, form highly electrophilic species. In vitro studies identified the <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                         mode addition DNA. The aniline portion An-Hq serves enhance nucleophilic addition <br />                         ethyl phenyl ether instead forming common Michael additions. Structural <br />                         characterization showed agents add 2&#226;&#8364;&#178;-deoxyguanosine N2,N3-positions. The product <br />                         formed bulky hydroxy-N2,3-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct. In addition, the <br />                         oxidatively activated agents added 2&#226;&#8364;&#178;-deoxyadenosine 2&#226;&#8364;&#178;-deoxycytidine, thymidine <br />                         2&#226;&#8364;&#178;-deoxyinosine. These findings confirmed primer extension analysis 392 <span id='am-239' about='xsp:base' typeof='owl:Thing'>base</span> pair <br />                         DNA. The full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> primer extension product reduced 69.0 &#194;&#177; 0.6% oxidative <br />                         activation An-Hq2 compared controls. Little sequence dependence observed 76% of <br />                         guanine, adenine, cytosine residues showing increase extension stops and <br />                         fold controls. Benzetheno-nucleobase addition double stranded DNA confirmed <br />                         LC/MS self-complementary oligonucletide. Experiments carried confirm in <br />                         vivo DNA damage. Because lesion identified vitro, reasoned nucleotide excision <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                         repair involved reversing effects oxidatively activated agents and <br />                         enhance toxicity Drosophila melanogaster. Using RNAi based approach, Ercc1 silenced <br />                         <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span> monitored injection agent. As expected, bulky cross-linking DNA <br />                         modifying agents, cisplatin chlorambucil, showed statistically significant enhanced toxicity in <br />                         Drosophila silenced Ercc1. In addition, 5-fluorouracil, does produce bulky lesions, <br />                         showed selective toxicity. An-Hq An-Hq2 showed statistically significant toxicity in <br />                         Drosophila silenced Ercc1. Examination cytotoxicity shows renal carcinoma cell lines a <br />                         target agents median IC50 1.8 &#206;&#188;M. Taken together, data shows the <br />                         designed oxidatively-activated agents form distinct, bulky DNA modifications prove difficult <br />                         cancer cells possessing elevated reactive oxygen species <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>phenotype</span> overcome. The <br />                         modification produced relatively unique anticancer agents. <br />  <br />  <br />  <br />  <br />                   &#194;&#167; <br />                    Correspondence sent to: Dr. Edward J. Merino, Department Chemistry, University Cincinnati, Cincinnati, OH <br />                   45221-0172, Phone number: 513-556-9200, Fax Number: 513-556-9239, merinoed@uc.edu. Dr. Stephanie M. Rollmann, Department <br />                   Biological Sciences, University Cincinnati, Cincinnati, OH 45221-0006, USA, Phone number: 513-556-9729, Fax number: <br />                   513-556-5299, stephanie.rollmann@uc.edu. <br />                    Jones et al.                                                                                                 Page 2 <br />  <br />  <br />  <br />                       Keywords <br />                             Reactive oxygen species; ROS; DNA-modifying agent; 2&#226;&#8364;&#178;-deoxyguanosine; anti-cancer agent; <br />                             Ercc1; Renal carcinoma <br />  <br />  <br />                       INTRODUCTION <br />                                      The harsh effects current chemotherapy treatments motivating factors <br />                                      increasing selectivity novel anticancer agents.1 Agents chemically modify DNA <br />                                      form backbone cancer treatments. A typical DNA-modifying agent cisplatin, <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      used treat fifty percent cancer patients.2 These agents represent large portion <br />                                      current phased-out anticancer agents. The design DNA-modifying agents has <br />                                      slowed drastically perception agents high levels effects <br />                                      modification DNA, subsequent cytotoxicity, non-target cells. This lack <br />                                      specificity termed off-target reactivity, limits tolerated doses and, therefore, <br />                                      decreases efficacy.3 Instead, agents attached transport active scaffolds <br />                                      enhance uptake target cancer cells.4,5 Side effects, however, problematic, <br />                                      improving selectivity major goal development novel anticancer agents. We <br />                                      developing novel agents address problems. <br />  <br />                                      A potential path forward cancer drug development advantage hallmark of <br />                                      cancer cells, elevated reactive oxygen species (ROS).6,7 Persistent elevation ROS <br />                                      numerous cancers, including renal cell carcinoma, melanoma, and <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      leukemia.8&#226;&#8364;&#8220;10 ROS occurs major endogenous forms cell: superoxide, <br />                                      hydrogen peroxide, singlet oxygen, hydroxyl radical.11&#226;&#8364;&#8220;13 Mitochondrial dysfunction, <br />                                      common cancer, known increase ROS.14 Increased ROS damages DNA, leading to <br />                                      <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span>.15 Consequently, surprise levels ROS-induced DNA damage <br />                                      correlates cancer prognosis.16 In turn, mutations cause enhanced activation of <br />                                      oncogenes.17 A critical role ROS oncogene transformed cancer cells shown <br />                                      investigating c-Myc induced damage.18 Furthermore, Ras oncogene locked <br />                                      active state concentration cellular antioxidant enzymes lowered causing <br />                                      elevated ROS.19,20 As result, chemical research teams actively searching for <br />                                      means utilize elevated ROS order generate selective anticancer agents.21&#226;&#8364;&#8220;23 Our lab <br />                                      designed oxidatively activated DNA modifying agents selectivity target the <br />                                      elevated ROS phenotype cancer cells.24 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      Our new oxidatively activated agents possess attractive anticancer properties.24 These <br />                                      agents, An-Hq An-Hq2, utilize novel mechanism activation highly <br />                                      selective (Figure 1A). The agents unreactive stable water, strongly <br />                                      activated ROS forms, excluding hydrogen peroxide. Addition oxidative <br />                                      equivalents leads rate enhancement greater 1700-fold compared no <br />                                      oxidant present. When reactivity examined, nitrogen <br />                                      mustard, agents actually oxidatively activated quinones. More specifically, we <br />                                      demonstrated agents capable adding guanine. The product a <br />                                      hydroxy-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct, guanine possessed added <br />                                      phenol. <br />  <br />                                      The possible formation benzethenoguanine left important questions be <br />                                      addressed. Specifically, questioned new agents actually DNA modifying <br />                                      agents. We set structurally characterize lesion produced nucleosides on <br />                                      double stranded DNA, examine reaction mechanism vitro (Figure 1A). To <br />                                      determine vivo DNA adduct formation, studies using powerful genetic model system,25 <br />  <br />  <br />                                       Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 3 <br />  <br />  <br />                                  Drosophila melanogaster, accomplished (Figure 1B). D. melanogaster shown <br />                                  effective, whole-animal model cancer research drug discovery.26&#226;&#8364;&#8220;29 Many <br />                                  key features DNA repair cell death conserved Drosophila and <br />                                  mammals.30 Approximately 75% human disease related genes functional ortholog <br />                                  flies.31,32 Moreover, studies Drosophila instrumental elucidating Ras <br />                                  signaling, major components conserved mammals.33 <br />  <br />                                  Here, investigated vitro DNA lesions produced novel oxidatively <br />                                  activated agents NMR. We agents required nucleoside base amine to <br />                                  initiate reactivity. A primer extension assay used explore lesion produced a <br />                                  392-nucleotide PCR product. We discovered nucleosides possessing amines <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  modified double stranded DNA, producing bulky lesions. These agents product <br />                                  adduct small double stranded DNA. Such lesions led predict that <br />                                  nucleotide excision repair important mechanism repairing adducts.34 Hence, <br />                                  used targeted RNAi knockdown approach based GAL4/UAS (Figure 1B) <br />                                  evaluate bulky DNA modification occurring vivo.35 We silenced Ercc1 gene, <br />                                  protein forms complex XPF key step nucleotide excision double <br />                                  strand break repair pathway.36 We Drosophila lacking Ercc1 expression showed <br />                                  elevated toxicity known DNA modifying agents, cisplatin chlorambucil. The <br />                                  elevated Ercc1-dependent sensitivity did occur Drosophila treated 5- <br />                                  fluorouracil, does produce bulky lesion. These results confirmed D. <br />                                  melanogaster effective model organism ascertaining genetic mechanisms of <br />                                  DNA modifying agents. We examined novel agents, An-Hq An-Hq2. We <br />                                  demonstrate flies Ercc1 expression knocked enhanced <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  sensitivity novel agents, particularly An-Hq, supporting role these <br />                                  oxidatively activated agents modifying DNA vivo. <br />  <br />                       MATERIAL AND METHODS <br />                       Synthesis An-Hq An-Hq2 <br />                                  The synthesis An-Hq An-Hq2 accomplished.24 NMR (1H 13C) spectra were <br />                                  recorded Bruker AMX 400MHz spectrometer. Chemical resonances reported &#206;&#180; <br />                                  (ppm) units using 13C residual 1H signals deuterated solvents references. High- <br />                                  resolution mass spectra (ESI) recorded Micromass Q-TOF 2 (Waters). Analytical <br />                                  layer chromatography (TLC) performed silica gel 60 GF254 (Merck). <br />                                  Compounds greater 99% pure according HPLC. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       Reaction An-Hq Nucleoside Analogs <br />                                  For nucleoside reactivity studies, An-Hq (22 &#206;&#188;l dissolved DMSO, 1mmol) added <br />                                  phosphate buffer (2 mL, 25mM NaH2PO4 5% acetonitrile 95% H2O, pH 8.0) <br />                                  contained 0.4 mmol listed nucleoside. Na2Ir2Cl6 (22 &#206;&#188;l dissolved water, 1 <br />                                  mmol) added, reaction mixed. Product monitored using Beckman <br />                                  Coulter System Gold HPLC equipped diode array detector (260 nm detection) for <br />                                  lesion formation twice day 7 days. HPLC conditions follows: Cosmosil <br />                                  5C18-PAQ Waters column used (4.6ID, 150mm length). The gradient (solvent <br />                                  A=95% water, 5% acetonitrile solvent B=5% water, 95% acetonitrile) linear: 0% B <br />                                  5 min, 100% B 20 min, 100% B 4 min, 0% B 2 min held 4 min. For <br />                                  mass spectrometry (MS), peaks collected, dried remove solvent (if stable, they <br />                                  directly injected MS), analyzed MS. All yields 0.5 and <br />                                  3%, depending nucleoside adduct. <br />  <br />  <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                             Page 4 <br />  <br />  <br />                                   The MS performed follows: isolated products resuspended 100 &#206;&#188;L 0.25% <br />                                   acetic acid 15% acetonitrile. Infusion instrument occurred rate 5 &#206;&#188;L/min. <br />                                   The mass spectrometry performed Thermo Fisher Scientific LTQ-FT, hybrid <br />                                   instrument consisting linear ion trap Fourier transform ion cyclotron resonance <br />                                   mass spectrometer. The entire elutant introduced LTQ-FT, using standard <br />                                   electrospray ionization source instrument spray voltage 5 kV capillary <br />                                   temperature 275&#194;&#176;C. Autogain control used set 500,000 maximum injection <br />                                   time 1250 ms FT-ICR scans. Collision induced dissociation, MS/MS, was <br />                                   executed linear trap AGC setting 10,000 maximum injection time of <br />                                   500 ms. FT-ICR scans acquired positive ion mode 100,000 resolving <br />                                   power m/z 400. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       2&#226;&#8364;&#178;-Deoxyguanosine-An-Hq Adduct NMR Analysis <br />                                   All 1H NMR experiments carried Bruker DGX-501, 500 MHz instrument. <br />                                   Chemical resonances reported &#206;&#180; (ppm) units, using residual 1H signals <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                   deuterated solvents references. The reaction listed scaled 100-fold (total volume <br />                                   200 mL). The reaction incubated dark 25&#194;&#176;C days. Solvent was <br />                                   evaporated adduct purified stages. First, Biotage SP1 Flash <br />                                   system, outfitted 5.5g RediSep Rf Gold C18 column, used purify reaction <br />                                   mixture, using 97% water 3% methanol buffer A methanol buffer B. The <br />                                   gradient 0% methanol 10% 10 column volumes, 10% 100% 3 column <br />                                   volumes held 5 column volumes. The sample (~75% pure HPLC) was <br />                                   collected 13 column volumes lyophilized remove solvent. The second stage <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   employed HPLC purification, using Grace Alltima HP C8 semi-preparative column (3&#206;&#188;m, <br />                                   7 &#195;&#8212; 53mm) 2.5ml/min. Solvent A 95% water 5% acetonitrile solvent B was <br />                                   95% acetonitrile 5% water. The gradient 0% B 5 min, 30% B 8 min, 100% <br />                                   B 3 min held 4 min. Absorbance monitored 260nM. The <br />                                   collected sample eluted 8 min. After HPLC purification, product dried and <br />                                   dissolved 500 &#206;&#188;L d6-DMSO evaluated NMR 15 hrs determine 1H- <br />                                   NMR. <br />  <br />                       Solvolysis An-Hq Methanol Using Na2IrCl6 Oxidant <br />                                   An-Hq (10 mg, 0.039 mmol) dissolved methanol (2 mL). Na2IrCl6&#194;&#183;6H2O (43 mg, 0.078 <br />                                   mmol) N,N-diisopropylethylamine (4.5 mg, 0.039 mmol) added mixture. The <br />                                   mixture reacted rt 0.5 hr. The reaction filtered silica gel washed <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   acetonitrile. The filtrate concentrated residue purified flash <br />                                   chromatography silica gel provide product (9 mg, 0.017 mmol, 90% yield) a <br />                                   yellow oil. 1H NMR (CDCl3, 400 MHz) &#206;&#180; 7.23 (t, J=8.2 Hz, 2 H), 6.80 (d, J=10.4 Hz, 2 H), <br />                                   6.69 (m, 3 H), 6.26 (d, J=10.0 Hz, 2 H), 3.76 (t, J=6.2 Hz, 2 H), 3.54 (t, J=6.2 Hz, 2 H), 3.41 <br />                                   (q, J=7.1 Hz, 2 H), 3.36 (s, 3 H), 0.87 (t, J=7.1 Hz, 3 H); HRMS (ESI, positive) m/z calcd. <br />                                   C17H22NO3 [M+H]+: 288.1600, found: 288.1614. <br />  <br />                       DNA Damage Visualization 392-Nucleotide PCR Product <br />                                   Primer extension experiments performed using 392-nucleotide dsDNA, synthesized <br />                                   pUC19 plasmid vector (New England Biolabs). A 30 cycle PCR amplification (55&#194;&#176;C <br />                                   30 s, 75&#194;&#176;C 45 s, 95&#194;&#176;C 30 s) performed, using 19-nucleotide forward <br />                                   primer, GGCCTCTTCGCTATTACGC, starting nucleotide position 287 the <br />                                   vector, 19-nucleotide reverse primer, ATACGCAAACCGCCTCTC, starting at <br />                                   position 672.37 The oligonucleotide purified using Cycle Pure Kit (Omega BioTek), <br />                                   yielding final concentration 40 &#206;&#188;g/&#206;&#188;L. DNA reacted agent, An-Hq2, and <br />                                   Na2IrCl6 5 hrs 37&#194;&#176;C (720 &#206;&#188;g DNA, 0.2 mM sodium phosphate, pH=8, 0.5 mM An- <br />                                   Hq2, 2 mM iridium). For pre-quenched control, DNA added 24 hrs the <br />  <br />                                    Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 5 <br />  <br />  <br />                                   agent oxidant mixed. The end products benzoquinone N- <br />                                   phenyldiethanolamine. Each sample reacted DNA added primer extension <br />                                   mix (1X vent buffer, 100 &#206;&#188;M dNTP, 200 nM primer, 0.05 U/&#206;&#188;L vent(exo-) DNA <br />                                   polymerase). The primer fluorescently-labeled IRDye700. Twelve cycles were <br />                                   accomplished (55&#194;&#176;C 15 s, 72&#194;&#176;C 1 min, 95&#194;&#176;C 30 s). Denaturing load dye was <br />                                   added samples, 12% denaturing PAGE gel performed. The gel was <br />                                   visualized using Odyssey Infrared Imaging System (LiCor) 169 &#206;&#188;m resolution and <br />                                   700-channel. Sequencing performed standard methods, using manual sequencing <br />                                   392-nucleotide PCR product acyclo-terminators, fluorescently labeled <br />                                   M13 primer substituted. Experiments performed triplicate, standard errors <br />                                   calculated. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       LC/MS Product Analysis Reaction An-Hq2 Oligonucleotide <br />                                   High purity oligonucleotides purchased Eurofins MWG operon. Electrophoresis <br />                                   showed oligonucletide greater 95% pure. The DNA sequence 5&#226;&#8364;&#178;- <br />                                   GCGCAATTGCGC-3&#226;&#8364;&#178;. The DNA annealed 25 mM sodium phosphate. Prior to <br />                                   reaction, DNA desalted placed 5mM Ammonium Acetate buffer, pH 8.0. The <br />                                   100 &#206;&#188;L reaction contained 2 mM DNA, 10 mM An-Hq2, 10 mM a <br />                                   (diacetoxyiodo)benzene oxidant. The reaction left room temperature dark for <br />                                   days. Analysis mass spectroscopy utilized Thermo Scientific LTQ-FT, hybrid <br />                                   instrument consisting linear ion trap Fourier transform ion cyclotron resonance <br />                                   mass spectrometer. The injection volume 10 &#206;&#188;L. Liquid chromatography was <br />                                   accomplished using Waters Symmetry C18 5 &#206;&#188;m, 2.1 &#195;&#8212; 150 m column, Finnigan Surveyor <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   MS pump, Finnigan Micro AS autosampler. The flow rate 200 &#206;&#188;L/min the <br />                                   gradient ranged 2% (v/v) acetonitrile 5mM ammonium formate 15% 35 min. <br />                                   Autogain control used set 500,000 maximum injection time 1250 ms for <br />                                   FT-ICR scans. FT-ICR scans acquired negative ion mode 100,000 <br />                                   resolving power m/z 400. Mass accuracy errors 500 ppb scan. <br />  <br />                       Drosophila Stocks Husbandry <br />                                   Drosophila maintained standard cornmeal, agar, molasses media 25&#194;&#176;C under <br />                                   12:12 hr light:dark cycle. To induce targeted gene silencing, following RNAi line was <br />                                   obtained Vienna Drosophila RNAi Center:35 UAS-Ercc1RNAi (v12622VDRC). This <br />                                   line crossed daughterless-GAL4 (da-GAL4) driver line order ubiquitously <br />                                   inactivate gene expression. The isogenic host strain, w1118 (60100VDRC), crossed da- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   GAL4 control genetic background effects. The da-GAL4 line kindly provided <br />                                   Dr. Mike Grotewiel (Virginia Commonwealth University.38 <br />  <br />                       RNAi Mediated Suppression Targeted Genes <br />                                   To confirm knockdown efficiency transgenic RNAi approach, RT-PCR assays were <br />                                   conducted. For sample, adult male Drosophila homogenized, total <br />                                   RNA isolated using Trizol (Invitrogen, Carlsbad, CA). Total RNA DNase <br />                                   treated using DNA-free (Ambion, Austin, TX), according manufacturer&#226;&#8364;&#8482;s instructions. <br />                                   Total RNA (0.5 &#206;&#188;g) reverse transcribed using Accuscript High Fidelity First Strand <br />                                   cDNA synthesis kit (Agilent Technologies, Santa Clara, CA), resulting cDNA was <br />                                   used RT-PCR assays. Primers follows: Ercc1-F 5&#226;&#8364;&#178;- <br />                                   CGTGCTGTACCTCTCGC-3&#226;&#8364;&#178; Ercc1-R 5&#226;&#8364;&#178;-CTGAGGAACGGTTCCTG-3&#226;&#8364;&#178;. <br />                                   QuantumRNA &#206;&#178;-actin Internal Standards (Ambion, Austin, TX) used amplify &#206;&#178;- <br />                                   actin (control) according manufacturer&#226;&#8364;&#8482;s instructions, following optimization &#206;&#178;- <br />                                   actin:competimer ratio. PCR products separated electrophoretically 2% agarose <br />                                   gel visualized ethidium bromide. RT-PCR assays independent RNA <br />                                   isolations genotype performed. <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                               Page 6 <br />  <br />  <br />                       Microinjection Drosophila <br />                                   Microinjections performed individual adult male Drosophila and <br />                                   seven days old. Individuals held position microinjection using gentle vacuum. <br />                                   A pulled glass micropipette attached Picospritzer III (Parker Hannifin, Cleveland, <br />                                   OH) used deliver anticancer agents fly abdomen. A 0.5 &#206;&#188;L volume agent <br />                                   injected ~40 psi compressed air. Agents dissolved 100% DMSO, and <br />                                   solutions fresh daily. Blue food dye (0.125 mg/mL) added solutions to <br />                                   confirm delivery agent. Agents tested included An-Hq, An-Hq2, chlorambucil, <br />                                   cisplatin, 5-fluorouracil. Microinjections vehicle (control) also <br />                                   performed. For agent, individuals genotype injected 10 mM <br />                                   concentration. A concentration 10 mM used preliminary analysis showed <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   effective eliciting response. Post injection, flies placed fresh standard <br />                                   Drosophila media 25&#194;&#176;C 12:12 hr light:dark cycle. Individuals scored for <br />                                   survival day followed total 7 days examine longer-term effects of <br />                                   agents. To analyze significance data, chi-squared calculated. <br />  <br />                       IC50 Determination An-Hq2 <br />                                   An-Hq2 submitted NCI Developmental Therapeutics Program screening. The <br />                                   method uses total protein content developed Monks et. al.39 Briefly, cells are <br />                                   treated An-Hq2 fixed plate surface. The number cells proportional <br />                                   relative protein measured sulforhodamine B. <br />  <br />                       RESULTS <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       In vitro Reactions Nucleosides Oligonucleotides <br />                                   Our data established An-Hq An-Hq2 highly selective cytotoxic agents against <br />                                   certain cancers. As new convention,40 numbering 2&#226;&#8364;&#178;-deoxyguanosine (dG) is <br />                                   retained manuscript. The numbering An-Hq phenol derivatives is <br />                                   denoted double prime, discussing phenol addition dG. We <br />                                   designed agents release hydroquinone moiety oxidation leave an <br />                                   activated nitrogen mustard. Instead, hydroquinone active portion <br />                                   molecule. Initially, An-Hq reacted dG, using Na2IrCl6 oxidant to <br />                                   determine addition occurred. We mass product corresponded to <br />                                   formation benzetheno-2&#226;&#8364;&#178;-deoxyguanosine. Previous literature demonstrates that <br />                                   benzoquinone add 2&#226;&#8364;&#178;-deoxyguanosine produce 3&#226;&#8364;&#179;-hydroxy-1,N2- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   benzetheno-2&#226;&#8364;&#178;-deoxyguanosine.41 The step reaction mechanism Michael <br />                                   addition. Thus, probed ability An-Hq react form benzetheno-2&#226;&#8364;&#178;- <br />                                   deoxyguanosine adduct different. <br />  <br />                                   In order determine agents modify DNA benzoquinone, An-Hq oxidized in <br />                                   presence methanol product determined. Na2IrCl6 used induce oxidation <br />                                   methanol (Figure 2A). Solvolysis methanol serves simple nucleophile a <br />                                   means trap site nucleophilic addition. Upon oxidation, An-Hq 90% oxidized to <br />                                   new compound 15 min. 1H-NMR used analyze isolated product; region <br />                                   3 8 ppm shown Figure 2A. There eighteen protons region, <br />                                   indicating product additional methoxy group. The singlet peak 3.4 ppm an <br />                                   integration 2.9 corresponds added methoxy group. The aromatic region is <br />                                   symmetric, indicating addition did occur C3&#226;&#8364;&#179; C2&#226;&#8364;&#179; phenol through <br />                                   Michael addition. We used mass spectrometry (MS) ensure product consistent a <br />                                   single methanol addition. Therefore, An-Hq does behave like benzoquinone. Instead, <br />                                   oxidative activation leads potent electrophile, addition C4&#226;&#8364;&#179;. This infers the <br />  <br />  <br />  <br />                                    Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                            Page 7 <br />  <br />  <br />                                  DNA lesion produced different benzoquinone, agents simply <br />                                  releasing benzoquinone mechanism action. <br />  <br />                                  As difference reactivity oxidative activation, investigated lesion <br />                                  produced NMR analysis (Figure 2B). For consistency, NMR labeled starting <br />                                  A products. The DNA lesion formed incubation An-Hq compared <br />                                  formed benzoquinone. The product An-Hq, Na2IrCl6, dG water- <br />                                  phosphate buffer used isolate dG-adduct formed. The oxidant used Na2IrCl6 <br />                                  possesses correct potential oxidize agents dG.42 The reaction of <br />                                  benzoquinone dG established Jowa et al. Chenna and <br />                                  Singer.41,43 We followed procedure, entails incubation dG in <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  dimethylformamide (DMF) potassium carbonate. This non-aqueous formed the <br />                                  benzetheno-adduct high yield. After reaction, purified products greater than <br />                                  95% purity levels (see Figure S1). Analysis positive ion MS showed products m/ <br />                                  z value 358.1146 elemental composition C16H16N5O5+ (Figure 3). MS/MS <br />                                  shows deglycosylation, indicating ribose portion nucleoside modified <br />                                  (see Figure S2). <br />  <br />                                  Reaction benzoquinone revealed addition N1,N2 previously observed <br />                                  (Figure 2B, bottom). Ten resonances 4&#226;&#8364;&#8220;10 ppm shown. The B shift a <br />                                  singlet 9.4 ppm ascribed terminal phenol hydrogen C1&#226;&#8364;&#179;. Two strong <br />                                  singlets, C D, observed 8.0. These singlets C8-hydrogen C2&#226;&#8364;&#179;- <br />                                  hydrogens, C2&#226;&#8364;&#179; deshielded. Importantly, C8-hydrogen indicated <br />                                  five-membered ring guanine modified. Accordingly, N7 the <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  addition site. Resonances E F remaining C5&#226;&#8364;&#179; C6&#226;&#8364;&#179; singlet phenol <br />                                  hydrogens. Taken together, phenol resonances indicate dG positions have <br />                                  added phenol. The triplet 6.2 ppm, G, C3&#226;&#8364;&#178;-hydrogen. The resonances H, I, <br />                                  J ribose protons, hydroxyl ribose. Finally, shift K hydrogen at <br />                                  N2. We hydrogen easily exchangeable and, hence, integration low. <br />                                  We added drop D2O differentiate exchangeable non-exchangeable protons <br />                                  (Figure S3). We resonances B, H, I, K exchangeable confirming our <br />                                  NMR assignments. <br />  <br />                                  Analysis An-Hq-dG adduct accomplished similar manner (Figure 2B, top). <br />                                  Again, resonances observed key differences. Several resonances were <br />                                  identical 3&#226;&#8364;&#179;-hydroxy-1,N2-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct. Shift D, C8- <br />                                  hydrogen, confirmed An-Hq does modify N7 five-membered ring. Likewise, <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  resonances B, G, H, I, J directly overlap (compare Figure 2B bottom) indicating <br />                                  modification ribose. The NMR identified distinct hydrogens. Two new <br />                                  hydrogen peaks seen. First, A shift 9.9 ppm strong singlet. Such strongly <br />                                  deshielded hydrogen come amido-hydrogen N1. In addition, K-shift <br />                                  N2 hydrogen observed 4.3 ppm. Importantly, shift A (bolded Figure 2B) <br />                                  indicated N1 position longer phenol-dG adduct. This was <br />                                  interesting because, taken appearance C8-hydrogen, NMR reveals that <br />                                  phenol added N2 N3. To support data An-Hq producing a <br />                                  different hydroxy-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct, determined stability acid. It <br />                                  observed N2,3-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adducts prone to <br />                                  degradation incubation acid weaker glycosidic bond present.44 Incubation at <br />                                  pH 1 elevated temperatures led degradation adduct compared 3&#226;&#8364;&#179;- <br />                                  hydroxy-1,N2-benzetheno adducts (see Figure S4). It noted tautomers <br />                                  drawn assign correctly NMR data (see Figure S5). In addition proposed <br />                                  mechanism form guanine adduct listed Figure S5. Importantly, data showed <br />  <br />  <br />  <br />                                  Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                         Page 8 <br />  <br />  <br />                                  An-Hq generates distinct lesion benzoquinone. The distinct lesion stemmed from <br />                                  oxidative activation mechanism An-Hq. <br />  <br />                                  We performed series HPLC analyses determine DNA bases modified by <br />                                  An-Hq (Figure 3). Nucleosides incubated An-Hq Na2IrCl6. After the <br />                                  incubation period, reactions analyzed HPLC UV detection set 260 nm. For <br />                                  nucleoside, reaction chromatogram (black) control chromatograms (grey) are <br />                                  shown. Each controls excludes single reaction component: An-Hq, dG, and <br />                                  Na2IrCl6. The reaction dG An-Hq shown left Figure 3. A clearly <br />                                  observable product seen 13 min comparing controls reaction traces. The <br />                                  product characteristic band absorbance spectrum (inset). MS analysis showed a <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  predominate ion m/z 358, elemental composition C16H16N5O5+ less <br />                                  300 ppb error. It noted small product 18 min which <br />                                  possessed m/z value equal deglycosylation product (data shown). To top <br />                                  right Figure 3, examined 7-deaza-dG reactivity. The NMR revealed N7 was <br />                                  involved reaction. Based data, substitution dG 7-deaza-dG should <br />                                  interfere formation reaction product. By performing reaction 7- <br />                                  deaza-dG, new series products observed 15 min. Again, products have <br />                                  similar UV absorbance. The MS spectra mass change 1 amu C17H17N4O5+, <br />                                  correct 7-deaza-dG adduct added phenol. This data proved N7 is <br />                                  involved reaction. To validate NMR results, reacted 2&#226;&#8364;&#178;- <br />                                  deoxyinosine An-Hq (Figure 3B, left). 2&#226;&#8364;&#178;-deoxyinosine lacks N2 and, <br />                                  according NMR results, react. The reaction prepared identically the <br />                                  reaction dG. We saw product formation, confirming NMR results N2 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  required product formation. <br />  <br />                                  We examined An-Hq adduct formation nucleoside. Analysis reaction <br />                                  2&#226;&#8364;&#178;-deoxyadenosine An-Hq revealed new product 15 min (Figure 3, middle <br />                                  left). The absorbance similar dG adduct formed. The product formed in <br />                                  low yield. The yield ~0.4%. The MS analysis adduct gave mass indicative that <br />                                  phenols added (addition 188 amu). Addition phenols observed in <br />                                  literature involving quinone addition nucleosides.43 The reaction 2&#226;&#8364;&#178;- <br />                                  deoxycytidine (Figure 3, middle right) gave product, limited <br />                                  characterization HPLC. Finally, did observe adduct formation incubation <br />                                  thymidine (Figure 3, right). The key difference thymidine lacks a <br />                                  nucleoside aryl amine. Altogether, data demonstrates oxidative activation of <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  An-Hq derivatives generates electrophile add exocyclic amino groups <br />                                  adenine, guanine, cytosine. The yield reactions double stranded DNA will <br />                                  differ nucleosides aryl amines adenine cytosine accessible the <br />                                  major groove, N2 guanine accessible minor groove. <br />  <br />                                  With ability modify nucleosides different parts the <br />                                  helix, set investigate agents&#226;&#8364;&#8482; reactivity double stranded DNA <br />                                  (dsDNA), biologically relevant substrates. Reaction An-Hq2 Na2IrCl6 large <br />                                  DNA strand allow identify specific DNA base sequences susceptible damage <br />                                  agents (Figure 4). A 392-nucleotide section pUC19 plasmid, positions 287&#226;&#8364;&#8220; <br />                                  678, amplified PCR. The fluorescently labeled primer used studies overlaps at <br />                                  positions 370&#226;&#8364;&#8220;389. Vent(exo-) polymerase used extension manual <br />                                  sequencing. The nucleotides listed Figure 4 sequence lanes correspond template <br />                                  strand. For example, bands G lane equivalent G nucleotides 392- <br />                                  nucleotide template. Agent oxidant required stop primer extension. When the <br />                                  unmodified DNA extended, full-length PCR product 301 bp observed. <br />                                  Comparison unreacted primer full-length product allowed calculation percent <br />  <br />  <br />                                  Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                              Page 9 <br />  <br />  <br />                                  yield primer extension (Figure 4A). Several controls utilized identify unique <br />                                  damage produced activated An-Hq2. The controls included minus Na2IrCl6 control <br />                                  (&#226;&#710;&#8217;OX), end products produced An-Hq2 oxidation (EP), pre-quenched reaction <br />                                  control (PQ), negative control containing DMSO (NR). Each control yielded 3.1&#226;&#8364;&#8220; <br />                                  3.2% full-length product demonstrated oxidative activation required. The end <br />                                  products gave An-Hq2 oxidation produced 3.1 &#194;&#177; 0.4% extension yield, similar the <br />                                  minus oxidant control. This control established end products cause extension <br />                                  stops high yield. When 392-nucleotide DNA incubated oxidized An- <br />                                  Hq2, similar yield 3.2 &#194;&#177; 0.1% full-length extension product observed. The no <br />                                  reagent control yielded 3.1 &#194;&#177; 0.1% extension yield. As positive control, incubated the <br />                                  392-nucleotide DNA cisplatin. Cisplatin known DNA modifying agent and, <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  therefore, induce replication stops DNA damage sites. As expected, observed no <br />                                  full-length extension product cisplatin positive control. Next, An-Hq2 tested to <br />                                  investigate DNA modifying capabilities. Treatment 392-nucleotide DNA <br />                                  An-Hq2 followed oxidative activation led reduction full-length product <br />                                  3.2% 1.0 &#194;&#177; 0.2%, decrease 69 &#194;&#177; 0.6%. The decline product formation <br />                                  shows An-Hq2 derivatives modify dsDNA stop progression DNA <br />                                  polymerase vitro. <br />  <br />                                  The primer extension assay provides evidence damage induced An-Hq2 and <br />                                  derivatives guanine specific (Figure 4B). The positive control, cisplatin, reacts <br />                                  guanine repeats. This seen CIS lane gel, largest replication <br />                                  stop position 413. The corresponding sequence 5&#226;&#8364;&#178;-GGGG. The negative controls <br />                                  (&#226;&#710;&#8217;OX, EP, PQ, NR) limited replication stops. We attributed stops the <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  polymerase&#226;&#8364;&#8482;s inability pass certain structural features hard replicate cloning <br />                                  region pUC19. In contrast, activated An-Hq2 RXN lane capable modifying <br />                                  bases positions. When looking positions 410&#226;&#8364;&#8220;470, nucleotides showed a <br />                                  greater two-fold enhancement early terminations. In particular, guanine at <br />                                  positions 412&#226;&#8364;&#8220;414, 422&#226;&#8364;&#8220;423, 430&#226;&#8364;&#8220;436 exhibited replication stops. Stops adenine <br />                                  occur. For example, replication stops seen 410 430. Similar extension <br />                                  stops cytosine positions 415&#226;&#8364;&#8220;419 440&#226;&#8364;&#8220;441 observed. There few <br />                                  predominate stops extension, 76% base types showing an <br />                                  increase extension stops fold. In contrast, observed little <br />                                  modification thymine. The thymine positions 392-nucleotide sequence denoted <br />                                  asterisks Figure 4C. The sequence 5&#226;&#8364;&#178;-TT position 421 422 displayed limited <br />                                  extension stops compared pre-quench control. Several thymine residues <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  showed change extension stops. In fact, thymine nucleotides nineteen <br />                                  difference PQ RXN (Figure 4C, grey asterisks). This data <br />                                  illustrates An-Hq2 derivatives modify DNA vitro, modification <br />                                  occurs types DNA bases. This data in-line nucleoside reactivity <br />                                  studies Figure 3. <br />  <br />                                  Reactivity DNA explored vitro using 12-nucleotide oligonucleotide. <br />                                  The sequence chosen self-complementary sequence 5&#226;&#8364;&#178;-GCGCAATTGCGC-3&#226;&#8364;&#178;, <br />                                  molecular weight 3646 g/mol elemental composition of <br />                                  C116H147N46O70P11. Reaction An-Hq2 produce addition benzetheno-group <br />                                  oligonucleotide increase molecular weight 3736 g/mol an <br />                                  elemental composition C122H149N46O71P11. This difference quite large easily <br />                                  detected FT-MS. The oligonucleotide annealed form dsDNA, reacted An- <br />                                  Hq2 oxidant, incubated days. The reaction mixture analyzed <br />                                  LC/MS determine An-Hq2 formed benzetheno-adduct double <br />                                  stranded DNA vitro (Figure 5). The total ion chromatogram seen of <br />                                  Figure 5A, predominate ion peak 12.0 min ion peak 13.5 min. The <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                           Page 10 <br />  <br />  <br />                                  MS spectrum 12.0 min peak shown Figure 5B. The 12.0 min spectrum a <br />                                  large ion current m/z 1214, triply negatively charged unmodified <br />                                  oligonucleotide (Figure 5B). Also observed doubly negatively charged ion the <br />                                  unmodified oligonucleotide m/z value 1821. The MS spectrum 13.5 min <br />                                  peak shown Figure 5C. The major ion observed total ion current 13.5 min <br />                                  corresponds m/z value 1244, expected m/z triply negatively <br />                                  charged benzetheno-oligonucleotide. Also observed doubly negatively charged <br />                                  modified oligonucleotide m/z 1866. The elemental composition determined <br />                                  C122H149N46O71P11 error 447 ppb (Figure S6). The major ion <br />                                  observed 13.5 min FT-MS ion pair triply negatively charged <br />                                  oligonucleotide trifluoroacetic acid (TFA) m/z value 1282; TFA <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  ion pair unmodified oligonucleotide seen 12.0 min FT-MS m/z <br />                                  1252. We note modification nucleotide positions the <br />                                  dsDNA, modification location mass. This result is <br />                                  significant proves An-Hq2 add dsDNA form lesion mass <br />                                  shown nucleoside studies. <br />  <br />                       Genetic Knockdown Nucleotide Excision Repair <br />                                  Although An-Hq derivatives modify double stranded DNA, known if <br />                                  DNA modification occurs cellular context. Therefore, turned D. <br />                                  melanogaster, powerful whole-animal genetic model. Because DNA lesion <br />                                  identified vitro, reasoned nucleotide excision repair important means <br />                                  repair adduct occurring vivo. Thus, prediction knockdown <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Ercc1, gene nucleotide excision repair pathway, sensitize Drosophila to <br />                                  oxidatively-activated agents. This sensitization result limited avenues repair <br />                                  remain, low basal levels An-Hq activation. Furthermore, predicted we <br />                                  expedient response DNA modifying agents. DNA damaging agents, to <br />                                  reactive nature, rapidly metabolized cells plasma. For example, cisplatin is <br />                                  hydrolyzed blood minutes hours active forms.45 The active agents quickly <br />                                  react DNA, glutathione, thioredoxin, protein produce irreversible adducts.46,47 <br />                                  Similar profiles observed chlorambucil, rapidly produces DNA lesions in <br />                                  minutes.48 <br />  <br />                                  As step testing hypotheses, began validating Drosophila suitable <br />                                  model study agents known mechanisms action. The effects agents, <br />                                  cisplatin, chlorambucil, 5-fluorouracil, examined. Specifically, conducted a <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  targeted transgenic RNAi knockdown experiment using GAL4/UAS which <br />                                  expression Ercc1 silenced. We confirmed silencing Ercc1 expression da- <br />                                  GAL4 driver using RT-PCR (Figure S7). Because rapid induction agents&#226;&#8364;&#8482; <br />                                  mechanisms action, monitored tolerated doses agents Drosophila over <br />                                  course 7 days. To so, recorded differences survival relative control, <br />                                  following delivery DNA modifying agent vehicle control.35 In majority <br />                                  injections healthy animals just vehicle alone, death noted in <br />                                  approximately 20%. This attributed stress injection process was <br />                                  expected. <br />  <br />                                  First, examined effects cisplatin, cross-links proximal guanines create <br />                                  helix distorting adducts. When cross-linking occurs, nucleotide excision repair required to <br />                                  correct adduct. In fact, ERCC1 activity used prognostic marker cisplatin- <br />                                  based cancer treatments.49,50 As predicted, Drosophila lacking Ercc1 expression (da- <br />                                  GAL4/UAS-Ercc1RNAi) injected cisplatin, rapid reduction survival was <br />                                  observed relative control line (da-GAL4/+) Day 1 (X2 = 5.0, P &lt; 0.025; Figure <br />                                  6A). On Day 1, 40% flies expressing Ercc1RNAi died comparison the <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                            Page 11 <br />  <br />  <br />                                  control flies. The 40% difference Drosophila lacking Ercc1 significant compared <br />                                  injection vehicle Day 1 (X2 = 5.0, P &lt; 0.025). No significant difference survival <br />                                  observed vehicle cisplatin injected Drosophila Ercc1 not <br />                                  silenced. Finally, seven day period, observed reduced survival genotypes <br />                                  agent delivery consistent highly toxic nature cisplatin. Hence, Ercc1 <br />                                  results D. melanogaster accurately replicate observations cisplatin Ercc1 in <br />                                  cancer. <br />  <br />                                  To confirm Drosophila-based assay sensitive DNA modifying agents <br />                                  producing bulky lesions, examined response Drosophila lacking Ercc1 expression <br />                                  chlorambucil. Chlorambucil nitrogen mustard produce bulky DNA- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  DNA cross-links, require Ercc1 repair. In case chlorambucil, significant <br />                                  difference response agent observed Day 1 Drosophila lacking <br />                                  Ercc1 expression control line(X2 = 3.8, P &lt; 0.05; Figure 6B). This suggests flies <br />                                  wild-type Ercc1 expression levels tolerate formation cross-links that <br />                                  Drosophila lacking Ercc1 higher sensitivity, 50% death Day 1. All the <br />                                  Drosophila lacking Ercc1 expression received chlorambucil died days <br />                                  compared 80% survival wild-type Drosophila. Additionally, significant difference in <br />                                  survival flies lacking Ercc1 expression observed flies injected <br />                                  chlorambucil vehicle (X2 = 6.7, P &lt; 0.001). However, significant <br />                                  survival difference observed agent vehicle wild-type flies. This further <br />                                  supports model demonstrating Ercc1 vital repair bulky lesions <br />                                  induced chlorambucil. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  We investigated second agent provide additional validation model system. The <br />                                  second agent, 5-fluorouracil, mainly repaired base excision repair, signals the <br />                                  incorrect incorporation uracil. Consequently, did expect downregulation Ercc1 <br />                                  affect mechanism action, and, therefore, considered negative control. <br />                                  When similar experiments conducted 5-fluorouracil, significant <br />                                  discrimination flies Ercc1 silencing controls (Figure 6C). On Day 1, <br />                                  injected control Drosophila died. Similarly, Ercc1 knockdown only <br />                                  20% died 5-fluorouracil injection, Drosophila perished. <br />                                  Survival agent delivery did significantly differ injection vehicle alone <br />                                  genotype. Thus, little change mortality observed treatment 5- <br />                                  fluorouracil indicative DNA modification occurring repaired. Therefore, <br />                                  validated Drosophila, model organism, used determine an <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  agent inducing DNA damage vivo. <br />  <br />                                  Finally, investigated modification DNA oxidatively activated agents was <br />                                  repaired Drosophila model pathway involving Ercc1. Experiments similar <br />                                  using DNA-modifying agents accomplished, using An-Hq An-Hq2. <br />                                  Because bulky lesion produced DNA modification, unlikely Drosophila <br />                                  lacking Ercc1 expression able repair damage. Therefore, expected that <br />                                  flies exhibit greater sensitivity An-Hq An-Hq2. We explored <br />                                  An-Hq. Upon injection An-Hq, observed 10% Drosophila died on <br />                                  Day 1 controls (Figure 7A). In contrast, Drosophila lacking Ercc1 expression showed <br />                                  60% death Day 1. This result significant P &lt; 0.002 X2 = 5.5. Drosophila <br />                                  lacking Ercc1 expression significant reduction survival agent delivery <br />                                  relative injection vehicle (X2 = 8.5 P &lt; 0.003). These results unique to <br />                                  An-Hq; An-Hq2 showed similar trends flies lacking Ercc1 showing significant <br />                                  reduction survival (50% death) Day 1 compared control line (X2 = 3.8, P &lt; 0.05; <br />                                  Figure 7B) compared delivery vehicle (X2 = 6.7, P &lt; 0.01). Wild-type <br />                                  Drosophila did differ significantly treated An-Hq2 versus vehicle Day 1. <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 12 <br />  <br />  <br />                                  This data emphasizes novel oxidatively activated agents induce DNA modification <br />                                  vivo strongly supports role DNA damaging agents highly selective <br />                                  cytotoxicity. <br />  <br />                       Potency Against Renal Carcinoma Cells <br />                                  We sought elucidate target cell type novel agents. In previous work, <br />                                  MTT assay used quantify potency types cancer cell lines. <br />                                  Interestingly, cancer cells tested displayed weak potency treated <br />                                  An-Hq2.24 Two renal cancer cells displayed low IC50 value. We, therefore, assessed <br />                                  renal cells sensitive An-Hq2 treatment. An-Hq2 evaluated effects on <br />                                  viability using sulforhodamine B total protein content assay NCI Developmental <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Therapeutics Program. Seven renal cancer cell lines examined: 786-0, A498, ACHN, <br />                                  CAKI-1, RXF 393, SN12C, UO-31. Data NCI fit sigmoid the <br />                                  IC50 fitting error determined. Most renal cancer cells tested sensitive An-Hq2 <br />                                  median IC50 value 1.8 &#206;&#188;M. Importantly, cell lines display high sensitivity <br />                                  IC50 values ACHN CAKI-1 cells 360 &#194;&#177; 90 370 &#194;&#177; 40 nM, <br />                                  respectively. One cell line, SN12C, displayed low potency IC50 value 21 &#194;&#177; 1.2 <br />                                  &#206;&#188;M. Since seven renal carcinoma cell lines IC50 values 5 &#206;&#188;M, infer <br />                                  cancer targeted oxidatively-activated agents. <br />  <br />                       CONCLUSION <br />                                  DNA modifying agents highly used anticancer treatments, non- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  selective. We designed agents activation occurs elevated reactive <br />                                  oxygen species present cancers. Previously, demonstrated our <br />                                  designed agents selective certain cancer cell types. Expansion studies in <br />                                  renal cell carcinoma cell lines shows agents potency mid- <br />                                  nanomolar range. This important finding research teams utilizing <br />                                  ROS-activated agents.7,51 For example, boronic esters limit off-target effects nitrogen <br />                                  mustards protease inhibitors.52&#226;&#8364;&#8220;55 Our work shows renal cancer cells natural target <br />                                  approaches. In manuscript, explored mechanism action these <br />                                  agents evaluation DNA lesion produced. These novel oxidatively activated <br />                                  agents behave differently current DNA modifying agents. They nitrogen <br />                                  mustards, simple quinone. Instead, occurs aniline ring tethered to <br />                                  quinone induces nucleophilic addition 4&#226;&#8364;&#179;-carbon DNA. DNA addition occurs <br />                                  exocylic amine bearing residue, means capable modifying out <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  DNA bases. Our primer extension experiment revealed that modification is <br />                                  occurring, likely minor groove guanine major groove adenine and <br />                                  cytosine. The end product bulky hydroxy-N2,3-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine addition <br />                                  apt strong replication stop. By making array lesions, these <br />                                  oxidatively activated agents expected impart strong need DNA repair. Thus, <br />                                  DNA repair, altered cancers, essential component cytotoxic <br />                                  mechanism action novel anticancer agents. <br />  <br />                                  Our Drosophila model shown loss Ercc1 function plays major role repair <br />                                  bulky DNA lesions. It noted assess cellular DNA modifying <br />                                  ability novel agents, validated rapid assay. In fact, detection DNA <br />                                  lesions formed anticancer agents challenging process. This difficulty the <br />                                  fact lesions need form impart cytotoxicity. The RNAi-mediated <br />                                  silencing Ercc1 Drosophila serve simple means determine therapeutic is <br />                                  DNA damage agent. This study illustrates DNA modifying agents selective, <br />                                  form highly disruptive lesions. <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                                             Page 13 <br />  <br />  <br />                       Supplementary Material <br />                                    Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                       Acknowledgments <br />                                    FUNDING SUPPORT <br />  <br />                                    Funding came University Cincinnati form startup funding faculty research award to <br />                                    E.J.M. graduate student research fellowship T.R.B.H. Furthermore, acknowledge NIH (Grant <br />                                    NIHRR19900) acquisition LTQ-MS used studies. Drosophila research supported grant <br />                                    National Institute Health S.M.R. (GM080592). <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                    We like acknowledge Jim Deddens, John Layne, Larry Sallans technical assistance and/or helpful <br />                                    discussions. We thank Vienna Drosophila RNAi center providing Drosophila stocks. <br />  <br />  <br />                       ABBREVIATIONS <br />  <br />                                  ROS               reactive oxygen species <br />                                  dG                2&#226;&#8364;&#178;-deoxyguanosine <br />                                  An-Hq             4-(2-(ethyl(phenyl)amino)ethoxy)phenol <br />                                  An-Hq2            4,4&#226;&#8364;&#178;-(2,2&#226;&#8364;&#178;-(phenylazanediyl)bis(ethane-2,1-diyl)bis(oxy))diphenol <br />                                  NMR               nuclear magnetic resonance <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       References <br />                                    1. Chabner B, Roberts T. Timeline - Chemotherapy war cancer. Nat Rev Cancer. 2005; <br />                                       5:65&#226;&#8364;&#8220;72. [PubMed: 15630416] <br />                                    2. Dhar S, Kolishetti N, Lippard S, Farokhzad O. Targeted delivery cisplatin prodrug safer and <br />                                       effective prostate cancer therapy vivo. Proc Natl Acad Sci USA. 2011; 108:1850&#226;&#8364;&#8220;1855. <br />                                       [PubMed: 21233423] <br />                                    3. Lovejoy, K.; Lippard, S. Dalton Trans. 2009. Non-traditional platinum compounds improved <br />                                       accumulation, oral bioavailability, tumor targeting; p. 10651-10659. <br />                                    4. Puckett C, Barton J. Targeting ruthenium complex nucleus short peptides. Bioorg Med <br />                                       Chem. 2010; 18:3564&#226;&#8364;&#8220;3569. [PubMed: 20430627] <br />                                    5. Zhilina Z, Ziemba A, Nielsen P, Ebbinghaus S. PNA-nitrogen mustard conjugates effective <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                       suppressors HER-2/neu biological tools recognition PNA/DNA interactions. Bioconj <br />                                       Chem. 2006; 17:214&#226;&#8364;&#8220;222. <br />                                    6. Trachootham D, Alexandre J, Huang P. Targeting cancer cells ROS-mediated mechanisms: a <br />                                       radical therapeutic approach? Nat Rev Drug Discov. 2009; 8:579&#226;&#8364;&#8220;591. [PubMed: 19478820] <br />                                    7. Pelicano H, Carney D, Huang P. ROS stress cancer cells therapeutic implications. Drug <br />                                       Resist Updat. 2004; 7:97&#226;&#8364;&#8220;110. [PubMed: 15158766] <br />                                    8. Sudarshan S, Sourbier C, Kong HS, Block K, Romero VAV, Yang Y, Galindo C, Mollapour M, <br />                                       Scroggins B, Goode N, Lee MJ, Gourlay CW, Trepel J, Linehan WM, Neckers L. Fumarate <br />                                       hydratase deficiency renal cancer induces glycolytic addiction hypoxia-inducible <br />                                       transcription factor 1 stabilization glucose-dependent generation reactive oxygen species. Mol <br />                                       Cell Biol. 2009; 29:4080&#226;&#8364;&#8220;4090. [PubMed: 19470762] <br />                                    9. Wittgen H, van Kempen L. Reactive oxygen species melanoma therapeutic implications. <br />                                       Melanoma Res. 2007; 17:400&#226;&#8364;&#8220;409. [PubMed: 17992124] <br />                                    10. Hole P, Darley R, Tonks A. Do reactive oxygen species play role myeloid leukemias? Blood. <br />                                         2011; 117:5816&#226;&#8364;&#8220;5826. [PubMed: 21398578] <br />                                    11. Cadet J, Douki T, Ravanat J. Oxidatively generated base damage cellular DNA. Free Radic Biol <br />                                         Med. 2010; 49:9&#226;&#8364;&#8220;21. [PubMed: 20363317] <br />  <br />  <br />  <br />                                       Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                                 Page 14 <br />  <br />  <br />                                  12. Solivio MJ, Nemera DB, Sallans L, Merino EJ. Biologically relevant oxidants cause bound <br />                                      proteins readily oxidatively cross-link guanine. Chem Res Toxicol. 2012; 25:326&#226;&#8364;&#8220;336. <br />                                      [PubMed: 22216745] <br />                                  13. Alp O, Zhang Y, Merino EJ, Caruso JA. Selenium effects arsenic cytotoxicity protein <br />                                      phosphorylation human kidney cells using chip-based nanoLC-MS/MS. Metallomics. 2011; <br />                                      3:482&#226;&#8364;&#8220;490. [PubMed: 21479294] <br />                                  14. Valko M, Rhodes C, Moncol J, Izakovic M, Mazur M. Free radicals, metals antioxidants in <br />                                      oxidative stress-induced cancer. Chem Biol Interact. 2006; 160:1&#226;&#8364;&#8220;40. [PubMed: 16430879] <br />                                  15. Adhikary A, Becker D, Collins S, Koppen J, Sevilla M. C5&#226;&#8364;&#178; - C3&#226;&#8364;&#178;-sugar radicals produced via <br />                                      photo-excitation one-electron oxidized adenine 2&#226;&#8364;&#178;-deoxyadenosine derivatives. <br />                                      Nucleic Acids Res. 2006; 34:1501&#226;&#8364;&#8220;1511. [PubMed: 16537838] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  16. Murtas D, Piras F, Minerba L, Ugalde J, Floris C, Maxia C, Demurtas P, Perra M, Sirigu P. <br />                                      Nuclear 8-hydroxy-2&#226;&#8364;&#178;-deoxyguanosine survival biomarker patients cutaneous <br />                                      melanoma. Oncol Rep. 2010; 23:329&#226;&#8364;&#8220;335. [PubMed: 20043092] <br />                                  17. Loft S, Poulsen H. Cancer risk oxidative DNA damage man. J Mol Med. 1996; 74:297&#226;&#8364;&#8220;312. <br />                                      [PubMed: 8862511] <br />                                  18. Vafa O, Wade M, Kern S, Beeche M, Pandita T, Hampton G, Wahl G. c-Myc induce DNA <br />                                      damage, increase reactive oxygen species, mitigate p53 function: A mechanism oncogene- <br />                                      induced genetic instability. Mol Cell. 2002; 9:1031&#226;&#8364;&#8220;1044. [PubMed: 12049739] <br />                                  19. Hu Y, Lu W, Chen G, Wang P, Chen Z, Zhou Y, Ogasawara M, Trachootham D, Feng L, Pelicano <br />                                      H, Chiao PJ, Keating MJ, Garcia-Manero G, Huang P. K-ras(G12V) transformation leads to <br />                                      mitochondrial dysfunction metabolic switch oxidative phosphorylation glycolysis. <br />                                      Cell Res. 2012; 22:399&#226;&#8364;&#8220;412. [PubMed: 21876558] <br />                                  20. Hole PS, Pearn L, Tonks AJ, James PE, Burnett AK, Darley RL, Tonks A. Ras-induced reactive <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      oxygen species promote growth factor-independent proliferation human CD34(+) hematopoietic <br />                                      progenitor cells. Blood. 2010; 115:1238&#226;&#8364;&#8220;1246. [PubMed: 20007804] <br />                                  21. Jourden JLM, Cohen SM. Hydrogen peroxide activated matrix metalloproteinase inhibitors: A <br />                                      Prodrug Approach. Angew Chem Int Edit. 2010; 49:6795&#226;&#8364;&#8220;6797. <br />                                  22. Kuang Y, Balakrishnan K, Gandhi V, Peng X. Hydrogen peroxide inducible DNA cross-linking <br />                                      agents: targeted anticancer prodrugs. J Am Chem Soc. 2011; 133:19278&#226;&#8364;&#8220;19281. [PubMed: <br />                                      22035519] <br />                                  23. Hagen H, Marzenell P, Jentzsch E, Wenz F, Veldwijk MR, Mokhir A. Aminoferrocene-based <br />                                      prodrugs activated reactive oxygen species. J Med Chem. 2012; 55:924&#226;&#8364;&#8220;934. [PubMed: <br />                                      22185340] <br />                                  24. Li G, Bell T, Merino EJ. Oxidatively activated DNA-modifying agents selective cytotoxicity. <br />                                      ChemMedChem. 2011; 6:869&#226;&#8364;&#8220;875. [PubMed: 21374823] <br />                                  25. Richgels PK, Rollmann SM. Genetic variation odorant receptors contributes variation in <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      olfactory behavior natural population Drosophila melanogaster. Chem Senses. 2012; <br />                                      37:229&#226;&#8364;&#8220;240. [PubMed: 22038943] <br />                                  26. Bier E. Drosophila, golden bug, emerges tool human genetics. Nat Rev Gen. 2005; 6:9&#226;&#8364;&#8220; <br />                                      23. <br />                                  27. Bell A, McBride S, Dockendorff T. Flies ointment Drosophila modeling enhance drug <br />                                      discovery. Fly. 2009; 3:39&#226;&#8364;&#8220;49. [PubMed: 19164936] <br />                                  28. Das T, Cagan R. Drosophila novel therapeutic discovery tool thyroid cancer. Thyroid. <br />                                      2010; 20:689&#226;&#8364;&#8220;695. [PubMed: 20578898] <br />                                  29. Kislukhin G, Murphy ML, Jafari M, Long AD. Chemotherapy-induced toxicity highly heritable <br />                                      Drosophila melanogaster. Pharmacogenet Genom. 2012; 22:285&#226;&#8364;&#8220;289. <br />                                  30. Sekelsky J, Brodsky M, Burtis K. DNA repair Drosophila: Insights Drosophila genome <br />                                      sequence. J Cell Biol. 2000; 150:F31&#226;&#8364;&#8220;F36. [PubMed: 10908583] <br />                                  31. Pandey UB, Nichols CD. Human disease models Drosophila melanogaster role fly <br />                                      therapeutic drug discovery. Pharmacol Rev. 2011; 63:411&#226;&#8364;&#8220;436. [PubMed: 21415126] <br />                                  32. Reiter L, Potocki L, Chien S, Gribskov M, Bier E. A systematic analysis human disease- <br />                                      associated gene sequences Drosophila melanogaster. Genome Res. 2001; 11:1114&#226;&#8364;&#8220;1125. <br />                                      [PubMed: 11381037] <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                                   Page 15 <br />  <br />  <br />                                  33. Pagliarini R, Quinones A, Xu T. Analyzing function tumor suppressor genes using a <br />                                      Drosophila model. Meth Mol Biol. 2003; 223:349&#226;&#8364;&#8220;382. <br />                                  34. Laat WL, Jaspers NG, Hoeijmakers JH. Molecular mechanism nucleotide excision repair. <br />                                      Genes Dev. 1999; 13:768&#226;&#8364;&#8220;785. [PubMed: 10197977] <br />                                  35. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, Fellner M, Gasser B, Kinsey K, Oppel S, <br />                                      Scheiblauer S, Couto A, Marra V, Keleman K, Dickson BJ. A genome-wide transgenic RNAi <br />                                      library conditional gene inactivation Drosophila. Nature. 2007; 448:151&#226;&#8364;&#8220;156. [PubMed: <br />                                      17625558] <br />                                  36. Aboussekhra A, Biggerstaff M, Shivji MK, Vilpo JA, Moncollin V, Podust VN, Proti&#196;&#8225; M, <br />                                      H&#195;&#188;bscher U, Egly JM, Wood RD. Mammalian DNA nucleotide excision repair reconstituted with <br />                                      purified protein components. Cell. 1995; 80:859&#226;&#8364;&#8220;868. [PubMed: 7697716] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  37. Solivio MJ, Joy TJ, Sallans L, Merino EJ. Copper generated reactive oxygen leads formation of <br />                                      lysine-DNA adducts. J Inorg Biochem. 2010; 104:1000&#226;&#8364;&#8220;1005. [PubMed: 20684045] <br />                                  38. Martin I, Jones MA, Grotewiel M. Manipulation Sod1 expression ubiquitously, the <br />                                      nervous muscle, impacts age-related parameters Drosophila. FEBS Lett. 2009; <br />                                      583:2308&#226;&#8364;&#8220;2314. [PubMed: 19540235] <br />                                  39. Monks A, Scudiero D, Skehan P, Shoemaker R, Paull K, Vistica D, Hose C, Langley J, Cronise P, <br />                                      Vaigro-Wolff A. Feasibility high-flux anticancer drug screen using diverse panel cultured <br />                                      human tumor cell lines. J Natl Cancer Inst. 1991; 83:757&#226;&#8364;&#8220;766. [PubMed: 2041050] <br />                                  40. Cooke MS, Loft S, Olinski R, Evans MD, Bialkowski K, Wagner JR, Dedon PC, M&#195;&#184;ller P, <br />                                      Greenberg MM, Cadet J. Recommendations standardized description nomenclature <br />                                      concerning oxidatively damaged nucleobases DNA. Chem Res Toxicol. 2010; 23:705&#226;&#8364;&#8220;707. <br />                                      [PubMed: 20235554] <br />                                  41. Chenna A, Singer B. Synthesis benzene metabolite adduct, 3&#226;&#8364;&#179;-Hydroxy-1, N2-benzetheno-2&#226;&#8364;&#178;- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      deoxyguanosine, Its site-specific incorporation DNA oligonucleotides. Chem Res <br />                                      Toxicol. 1997; 10:165&#226;&#8364;&#8220;171. [PubMed: 9049427] <br />                                  42. Munk BH, Burrows CJ, Schlegel HB. Exploration mechanisms transformation 8- <br />                                      hydroxy guanine radical FAPyG density functional theory. Chem Res Toxicol. 2007; <br />                                      20:432&#226;&#8364;&#8220;444. [PubMed: 17316026] <br />                                  43. Jowa L, Witz G, Snyder R, Winkle S, Kalf GF. Synthesis characterization deoxyguanosine- <br />                                      benzoquinone adducts. J Appl Toxicol. 1990; 10:47&#226;&#8364;&#8220;54. [PubMed: 2335711] <br />                                  44. Khazanchi R, Yu P, Johnson F. N-2,3-Etheno-2&#226;&#8364;&#179;-deoxyguanosine [8,9-dihydro-9-oxo-2-&#226;&#8364;?deoxy-3- <br />                                      beta-D-ribofuranosylimidazo[2,1-B]purine] - practical synthesis characterization. J Org <br />                                      Chem. 1993; 58:2552&#226;&#8364;&#8220;2556. <br />                                  45. Verschraagen M, van der Born K, Zwiers T, van der Vijgh W. Simultaneous determination of <br />                                      intact cisplatin metabolite monohydrated cisplatin human plasma. J Chromatog B-Analyt <br />                                      Technol Biomed Life Sci. 2002; 772:273&#226;&#8364;&#8220;281. [PubMed: 12007772] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  46. Vermorken JB, van der Vijgh WJ, Klein I, Hart AA, Gall HE, Pinedo HM. Pharmacokinetics of <br />                                      free total platinum species short-term infusion cisplatin. Cancer Treat Rep. 1984; <br />                                      68:505&#226;&#8364;&#8220;513. [PubMed: 6538459] <br />                                  47. Paz MM, Zhang X, Lu J, Holmgren A. A new mechanism action anticancer drug <br />                                      mitomycin C: mechanism-based inhibition thioredoxin reductase. Chem Res Toxicol. 2012; <br />                                      25:1502&#226;&#8364;&#8220;1511. [PubMed: 22694104] <br />                                  48. Bank BB, Kanganis D, Liebes LF, Silber R. Chlorambucil pharmacokinetics DNA binding in <br />                                      chronic lymphocytic leukemia lymphocytes. Cancer Res. 1989; 49:554&#226;&#8364;&#8220;559. [PubMed: 2910477] <br />                                  49. Besse B, Massard C, Haddad V, Andre F, Dunant A, Pirker R, Olaussen KA, Brambilla E, Fouret <br />                                      P, Soria JC. ERCC1 influence incidence brain metastases patients non-squamous <br />                                      NSCLC treated adjuvant cisplatin-based chemotherapy. Ann Oncol. 2011; 22:575&#226;&#8364;&#8220;581. <br />                                      [PubMed: 20801905] <br />                                  50. Arora S, Kothandapani A, Tillison K, Kalman-Maltese V, Patrick SM. Downregulation XPF- <br />                                      ERCC1 enhances cisplatin efficacy cancer cells. DNA Repair (Amst). 2010; 9:745&#226;&#8364;&#8220;753. <br />                                      [PubMed: 20418188] <br />                                  51. Peng X, Gandhi V. ROS-activated anticancer prodrugs: new strategy tumor-specific damage. <br />                                      Ther Del. 2012; 3:823&#226;&#8364;&#8220;833. <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                             Page 16 <br />  <br />  <br />                                  52. Kuang Y, Balakrishnan K, Gandhi V, Peng X. Hydrogen Peroxide Inducible DNA Cross-Linking <br />                                      Agents: Targeted Anticancer Prodrugs. J Am Chem Soc. 2011; 133:19278&#226;&#8364;&#8220;19281. [PubMed: <br />                                      22035519] <br />                                  53. Major Jourden JL, Cohen SM. Hydrogen Peroxide Activated Matrix Metalloproteinase Inhibitors: <br />                                      A Prodrug Approach. Angew Chem Int Ed Engl. 2010; 49:6795&#226;&#8364;&#8220;6797. [PubMed: 20715043] <br />                                  54. Hagen H, Marzenell P, Jentzsch E, Wenz F, Veldwijk MR, Mokhir A. Aminoferrocene-Based <br />                                      Prodrugs Activated Reactive Oxygen Species. J Med Chem. 2012; 55:924&#226;&#8364;&#8220;934. [PubMed: <br />                                      22185340] <br />                                  55. Cao S, Wang Y, Peng X. ROS-Inducible DNA Cross-Linking Agent New Anticancer Prodrug <br />                                      Building Block. Chem Eur J. 2012; 18:3850&#226;&#8364;&#8220;3854. [PubMed: 22378607] <br /> $watermark-text <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                              Page 17 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Figure 1. Identification Determinants Selective Activation Oxidatively-Activated <br />                                  Agent <br />                                  (A) Structure oxidatively activated agents An-Hq An-Hq2. When ROS present, <br />                                  An-Hq activated, leading addition phenol guanine. We seek determine <br />                                  structure dependence damage. (B) Genes responsible producing proteins involved <br />                                  DNA repair targeted using RNAi-mediated silencing. Delivery known DNA <br />                                  modifying agents lead selective death D. melanogaster model. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 18 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Figure 2. An-Hq Derivatives React Guanine Differently Benzoquinone <br />                                  (A) Quinones react paths: C4&#226;&#8364;&#179; addition, black, C2&#226;&#8364;&#179;-Michael addition, grey. <br />                                  The product path shown. To determine reaction path, product from <br />                                  oxidation methanol isolated. NMR MS shown. Oxidative activation <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  An-Hq methanol proceeds C4&#226;&#8364;&#179; addition based grey NMR assignments and <br />                                  integrals. (B) Top. NMR reaction An-Hq, 2&#226;&#8364;&#178;-deoxyguanosine, iridium <br />                                  oxidant phosphate buffer. On left reaction assigned NMR resonances <br />                                  superimposed product. On right H1-NMR 4&#226;&#8364;&#8220;10 ppm. The <br />                                  assignments grey. An N1-proton signal observed. Bottom. The reaction 2&#226;&#8364;&#178;- <br />                                  deoxyguanosine, base, benzoquinone according literature. On left reaction <br />                                  assignments. On right H1-NMR. Importantly, N1 peak 10ppm <br />                                  missing benzoquinone reaction product NMR. The different reaction path An- <br />                                  Hq leads different guanine lesion. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                       Page 19 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Fi <br /> </body></html>