<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                   Article <br />  <br />                                                                                                                             pubs.acs.org/biochemistry <br />  <br />  <br />  <br />  <br /> Expression Translationally Fused TAP-Tagged Plasma Membrane <br /> Proton Pump Arabidopsis thaliana <br /> Rachel B. Rodrigues, Gregorz Sabat, Benjamin B. Minko&#239;&#172;&#8364;, Heather L. Burch, Thao T. Nguyen, <br /> and Michael R. Sussman* <br /> Department Biochemistry, Biotechnology Center, University Wisconsin, 425 Henry Mall, Madison, Wisconsin 53706, United <br /> States <br />   * <br />   S Supporting Information <br />  <br />  <br />  <br />   <span id='am-3' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>: The Arabidopsis thaliana plasma membrane proton ATPase genes, AHA1 AHA2, <br />   highly expressed isoforms 11 gene family collectively essential for <br />   embryo development. We report translational fusion tandem a&#239;&#172;&#402;nity-puri&#239;&#172;?cation tag the <br />   5&#226;&#8364;&#178; end AHA1 open reading frame genomic clone. Stable expression TAP-tagged AHA1 <br />   Arabidopsis rescues embryonic lethal <span id='am-11' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-12' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span> endogenous double aha1/aha2 <br />   knockdowns. Western blots SDS-PAGE Blue Native gels enrichment AHA1 in <br />   plasma membrane fractions indicate hexameric quaternary structure. TAP-tagged AHA1 rescue <br />   lines exhibited reduced vertical root growth. Analysis plasma membrane soluble <br />   proteomes identi&#239;&#172;?ed plasma membrane-localized proteins alterred abundance TAP- <br />   tagged AHA1 rescue lines compared wild type. Using a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry, we <br />   uniquely identi&#239;&#172;?ed additional AHA isoforms, AHA9 AHA11, copuri&#239;&#172;?ed TAP- <br />   tagged AHA1. In conclusion, generated transgenic Arabidopsis lines TAP-tagged <br />   AHA1 transgene complemented essential endogenous AHA1 AHA2 functions have <br />   shown plants used purify AHA1 protein identify planta interacting <br />   proteins mass spectrometry. <br />  <br />  <br />  <br /> T      Arabidopsis thaliana H+-ATPase genes, AHA1 and <br />        AHA2, belong protein family 100 kDa, <br /> plasma membrane-localized P-type ATPases. AHA proteins <br />                                                                             dant, nature AHA1 AHA2 complicated reverse- <br />                                                                             genetic approaches elucidate speci&#239;&#172;?c cellular functions for <br />                                                                             proteins.20 <span id='am-10' about='protege:TO' typeof='owl:Thing'>To</span> create facile for <br /> belong electrogenic class P-type ATPases responsible              investigating AHA1 speci&#239;&#172;?c functions, developed transgenic <br /> for generating primary active membrane potentials,               endogenous double aha1/aha2 knockdown lines rescued a <br /> includes Ca2+-ATPase sarcoplasmic reticulum                  tandem a&#239;&#172;&#402;nity-puri&#239;&#172;?cation (TAP)-tagged AHA1, thereby <br /> the Na+/K+-ATPase mammalian cells.1 Speci&#239;&#172;?cally, AHAs                    replacing &#226;&#710;&#188;70% AHA protein crude seedling and <br /> link ATP catalysis active transport protons            vegetative extracts fusion protein capable facilitating <br /> cell, establishing transmembrane proton motive force              isoform-speci&#239;&#172;?c methods including puri&#239;&#172;?cation, biochemical <br /> (PMF) drives secondary transport processes.2&#226;&#710;&#8217;4 The                analysis, mass spectrometry. <br /> PMF composed pH gradient electrical                        An increasing de&#239;&#172;?ning protein-interaction net- <br /> membrane potential reach excess &#226;&#710;&#8217;250 mV                   works greatly facilitated emerging proteomic, <br /> roots.5 AHAs highly expressed cell types specialized             genomic, imaging technologies.21&#226;&#710;&#8217;23 An especially <br /> active transport, including phloem tissue, guard cells,                 signi&#239;&#172;?cant contribution identi&#239;&#172;?cation protein <br /> epidermal root hairs, major consumers cellular                   interactions growing access modern mass spectrometry <br /> ATP.4,6&#226;&#710;&#8217;8                                                                   (MS) resources ability perform quantitative <br />    Of 11 genes encoding functional proteins, AHA1                   experiments.24 Recently, large Arabidopsis protein-inter- <br /> AHA2 highly expressed isoforms,                        action networks published including yeast two- <br /> abundant tissues developmental stages based                   hybrid approach identi&#239;&#172;?ed 6200 interactions 2700 <br /> EST expression data, estimated make 80%                 proteins25 split ubiquitin approach focused membrane <br /> all AHA activity.5,9,10 The plasma membrane proton pumps                proteins revealed 541 interactions 239 proteins.26 <br /> hypothesized play roles control tip growth                  No AHA1 AHA2 protein interactions identi&#239;&#172;?ed the <br /> generating localized proton &#239;&#172;&#8218;uxes11,12 highly regulated             yeast two-hybrid interactome database, high- <br /> to control guard cell opening closure response biotic             con&#239;&#172;?dence candidates, annotated leucine-rich repeat <br /> and abiotic stimuli.13&#226;&#710;&#8217;16 Using reverse-genetic analysis, roles for <br /> AHA3 pollen development, AHA4 salt stress, AHA10                  Received: August 12, 2013 <br /> in proanthocyanidin vacuole biosynthesis                      Revised: January 2, 2014 <br /> reported.17&#226;&#710;&#8217;19 However, essential, genetically redun-               Published: January 3, 2014 <br />  <br />                               &#194;&#169; 2014 American Chemical Society        566                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                    Article <br />  <br /> kinases, generated split ubiquitin approach.                    Forward: GATTTCTTCTGGGTGAAGATGTCG- <br /> Furthermore, proteins previously published interact              ACTCTCGAAGATATCAAGAACGAGA <br /> the plasma membrane proton pumps identi&#239;&#172;?ed                      Reverse: TCTCGTTCTTGATATCTTCGAGAGT- <br /> these interactome studies.27&#226;&#710;&#8217;29,14,30,31                                 CGACATCTTCACCCAGAAGAAATC <br />    A&#239;&#172;&#402;nity puri&#239;&#172;?cation epitope-tagged proteins combined                   The gAHA1 fragment subcloned Agrobacterium <br /> with mass spectrometry-based proteomic analysis             transformation vector pGreenII0179 SacI XhoI <br /> widely established method characterization protein            sites.39 The N-terminal TAPi tag ampli&#239;&#172;?ed high- <br /> complexes, tandem a&#239;&#172;&#402;nity-puri&#239;&#172;?cation tags                  &#239;&#172;?delity polymerase gateway cloning vector nta- <br /> successfully used investigate protein complexes plants         pi.289.gw32 using following primers designed append <br /> these a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry (AP&#226;&#710;&#8217;MS) ap-                   SalI restriction overhangs: <br /> proaches.32&#226;&#710;&#8217;36,23 A&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry meth-                Forward: AAGTCGACCGTGGTGGACAACAAGTTCAA <br /> ods combined chemical cross-linking                 Reverse: TTGTCGACGAAGAAGATCCTCCTCCTCCC- <br /> the puri&#239;&#172;?cation low-abundance plasma membrane protein                 AGTGCGCCGCTG <br /> complexes plants.36                                                    Adenosine overhangs added Taq polymerase, and <br />    Herein, report functional rescue aha1/aha2                  fragment cloned pCR2.1. Ntapi SalI restriction <br /> double-<span id='am-89' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>knockout</span> plants TAP-tagged AHA1 genomic                    fragments cloned unique AHA1 SalI site. <br /> transgene (Ntapi:gAHA1). We observed reduced vertical root               Constructs con&#239;&#172;?rmed sequencing used to <br /> growth Ntapi:gAHA1 transgenic rescue lines,              transform aha1&#226;&#710;&#8217;6/aha1&#226;&#710;&#8217;6;AHA2/aha2&#226;&#710;&#8217;4 plants Agro- <br /> phenotypic consequences overall plant growth apparent             bacterium-mediated &#239;&#172;&#8218;oral dip, strain GV3101 carrying <br /> under standard laboratory growth conditions. To investigate              pSOUP.40 Transgenic seeds selected hygromycin <br /> the basis short-root phenotype assess validity         media (half-strength M&amp;S, 1% sucrose, 0.7% agar, 25 &#206;&#188;g/ <br /> of using plants future AHA1 functional studies,             mL hygromycin). Hygromycin-resistant (HygR) aha1/ <br /> used metabolically labeled mass spectrometry approach               aha1;aha2/aha2 plants complemented presence the <br /> characterize proteins showing increased decreased abun-               N-terminal Ntapi:gAHA1 construct isolated T2 <br /> dance Ntapi:gAHA1 compared wild-type plants. We used               population PCR genotyping. <br /> a&#239;&#172;&#402;nity puri&#239;&#172;?cation followed tandem mass spectrometry                  Plasma Membrane Preparation. Arabidopsis seedlings <br /> identify AHA1-interacting protein candidates Ntapi:AHA1               grown 9 days half-strength M&amp;S liquid media as <br /> copuri&#239;&#172;?cation investigated e&#239;&#172;&#8364;ect vivo elicitation          described previously.41 Nine day old seedlings extracted in <br /> on copurifying proteins. The Ntapi:gAHA1 transgenic plants               2&#195;&#8212; (v/w) homogenization bu&#239;&#172;&#8364;er (230 mM sorbitol, 50 mM <br /> reported provide important tool AHA1 isoform-           Tris-HCl, pH 7.5, 10 mM KCl, 1 mM potassium metabisul&#239;&#172;?te, <br /> speci&#239;&#172;?c biochemical analysis, identi&#239;&#172;?cation new protein               1 mM PMSF, 10 mM sodium &#239;&#172;&#8218;uoride, 2 mM sodium <br /> interactions, veri&#239;&#172;?cation interactions previously identi-         pyrophosphate, 1 mM ammonium molybdate, complete <br /> &#239;&#172;?ed using heterologous techniques.                                       EDTA-free protease inhibitor cocktail (Roche)). Microsomes <br />  <br /> &#226;&#8211;&#160;    EXPERIMENTAL PROCEDURES <br />    Plant Materials Growth Conditions. Mutants <br />                                                                          collected previously described.20 The microsomal pellet <br />                                                                          resuspended resuspension bu&#239;&#172;&#8364;er (330 mM sorbitol, 5 <br />                                                                          mM KCl, 5 mM K2HPO4, pH 7.8) homogenized a <br /> (ecotype Columbia) carrying homozygous T-DNA insertions                  Te&#239;&#172;&#8218;on homogenizer, plasma membranes enriched by <br /> in AHA1 (aha1&#226;&#710;&#8217;6, SALK016325) heterozygous insertions                 phase partition described previously.42 Protein concen- <br /> in AHA2 (aha2&#226;&#710;&#8217;4, SALK082786) created crossing                 trations determined Bradford assays. <br /> described previously.20 Seeds germinated plates                     Protein Puri&#239;&#172;?cation. Plasma membrane protein (100&#226;&#710;&#8217;500 <br /> containing half-strength Murashige &amp; Skoog (M&amp;S) salts, 1%               &#206;&#188;g) solubilized 10:1 (w/w) n-dodecyl-&#206;&#178;-D-maltoside <br /> (w/v) sucrose, 0.7% (w/v) agar. Seedlings transferred           resuspension bu&#239;&#172;&#8364;er 1 h 4 &#194;&#176;C. Unsolubilized material <br /> to soil/perlite mixture grown constant light 21 &#194;&#176;C.         pelleted benchtop microcentrifuge speed 30 <br />    Root Growth Assays. Root growth asssays conducted                min 4 &#194;&#176;C. Solubilized upper-phase protein (0.1 mg, 100&#226;&#710;&#8217;200 <br /> as described previously.20,37                                            &#206;&#188;L total volume) incubated 1 h 4 &#194;&#176;C 15 &#206;&#188;L of <br />    DNA Extraction Plant Genotyping. Plant genomic                    rabbit IgG agarose resin (Sigma, A9294) 10 &#206;&#188;L IgG <br /> DNA extracted using method Krysan et al.             Sepharose 6 Fast Flow resin (GE Life Sciences, 17-0969-01) <br /> elimination phenol/chloroform step.38 The presence             equilibrated resuspension bu&#239;&#172;&#8364;er 0.1% (w/v) DDM. <br /> aha1&#226;&#710;&#8217;6 aha2&#226;&#710;&#8217;4 T-DNA insertion wild-type alleles               Resin pelleted 0.8 rcf 1 min, supernatant was <br /> determined PCR using allele-speci&#239;&#172;?c primers spanning              discarded. Resin washed times 4 &#194;&#176;C 10 min with <br /> T-DNA AHA1 AHA2 genomic junctions (T-DNA LB:                      100&#195;&#8212; volume resuspension bu&#239;&#172;&#8364;er containing 1% NP40 <br /> T C A AA C A G G A T T T T C G C C T G CT ; S0 1 6 3 2 5 L P :           followed &#239;&#172;?nal wash 100&#195;&#8212; volume resuspension <br /> CGTCTCAACAAAAGTCTCTTTCA; S016325 RP:                                     bu&#239;&#172;&#8364;er containing 1% NP40 0.01% SDS. Resin pelleted <br /> CGAAAGATCAACCTCGTGAGT; S082786 LP: ATGTTC-                               1 min 0.8 rcf wash, supernatant was <br /> ATTGCAAAGGTGGTG; S082786 RP: CCCAT-                                  discarded. Protein eluted overnight 4 &#194;&#176;C 30 units <br /> TAGCTCGTGGTTATTG).20                                                     AcTEV protease (Invitrogen). <br />    Cloning Transformation TAP-Tagged AHA1.                           SDS-PAGE. SDS-PAGE analysis performed Novex <br /> Cloning 9.9 kb genomic region containing AHA1                  10% Tris-glycine gels (Invitrogen) using 1&#195;&#8212; Novex Tris- <br /> 3377 bp upstream 1100 bp downstream DNA                        Glycine SDS running bu&#239;&#172;&#8364;er NuPAGE 4&#226;&#710;&#8217;12% Bis-Tris <br /> (gAHA1) described previously.20 A unique SalI restriction-           gels using 1&#195;&#8212; MOPS running bu&#239;&#172;&#8364;er 80 V 3 4 h. <br /> enzyme site created 5&#226;&#8364;&#178; end AHA1 open                   Protein samples diluted 1:1 Laemmli sample bu&#239;&#172;&#8364;er with <br /> reading frame using QuikChange II XL kit (Stratagene)            5% &#206;&#178;-mercaptoethanol (BioRad). For western blot, gels were <br /> the following mutagenic primers:                                         incubated room temperature 20 min 100 mL of <br />                                                                    567                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                   Article <br />  <br /> equilibration bu&#239;&#172;&#8364;er (2&#195;&#8212; NuPAGE transfer bu&#239;&#172;&#8364;er, 10%                      100 000 resolving power. The &#239;&#172;?ve ions, excluding <br /> methanol, 1:1000 NuPAGE antioxidant). Proteins                 unassigned +1 charge states, repeatedly isolated <br /> transferred PVDF using iBlot dry blotting                 2.5 Da window CID analysis linear ion trap <br /> (Invitrogen) using program P3 12 min. Immunoblotting                using normalized collision energy 35.0 activation Q <br /> was performed according standard protocols using HRP-                   time 0.25 30 ms, respectively. Dynamic exclusion <br /> conjugated rabbit anti-goat IgG colorometric visualization          used 40 s repeat count 1 the <br /> or rabbit anti-CBP (Calmodulin-binding protein, Genscript)              duration analysis. The TAIR9 proteome database was <br /> followed IRDye goat anti-rabbit (LI-COR) membranes               searched using Mascot v2.2, allowing missed tryptic <br /> visualized LI-COR Odyssey infrared scanner.                      cleavage, +2 +3 charge states, peptide tolerance 20 <br />    Blue Native PAGE. Three micrograms plasma membrane                ppm, MS/MS ion tolerance 0.6 Da. Carbamidomethy- <br /> protein combined 1% digitonin membrane-                     lation set &#239;&#172;?xed modi&#239;&#172;?cation, phosphorylated S/T/ <br /> solubilization bu&#239;&#172;&#8364;er (50 mM Tris-HCl, 150 mM NaCl, 1                Y, deamidated N/Q, oxidated M set variable <br /> mM EDTA, pH 7.5) previously described.43 NativePAGE                  modi&#239;&#172;?cations. A reverse database appended con- <br /> sample bu&#239;&#172;&#8364;er (Invitrogen) added supernatant             currently searched. Data &#239;&#172;?ltered 1% false discovery rate <br /> &#239;&#172;?nal concentration 1&#195;&#8212;, NativePAGE 5% G-250 sample                using in-house developed software (available http://www. <br /> additive added &#239;&#172;?nal concentration 0.06%. Samples            biotech.wisc.edu/sussmanlab/home), ratio calculations <br /> were run NativePAGE Novex 3&#226;&#710;&#8217;12% Bis-Tris gels 150 V               performed using Census.45 Every sample manually <br /> for 20 min dark blue cathode bu&#239;&#172;&#8364;er followed 45 min              median-normalized prior analysis. <br /> with light blue cathode bu&#239;&#172;&#8364;er. Gels washed 20 min              Quantitative Analysis Metabolically Labeled <br /> equilibration bu&#239;&#172;&#8364;er (80 mL ddH20, 10 mL 20&#195;&#8212; transfer              Proteomes. Peptides Mascot scores 30 peak <br /> bu&#239;&#172;&#8364;er, 10 mL methanol, 100 &#206;&#188;L NuPage antioxidant),            correlations 0.8 discarded analysis. Peptide <br /> transferred Immobilon-FL membranes (Millipore) using              ratios averaged achieve single ratio protein <br /> iBlot dry blotting (Invitrogen). Membrane allowed            identi&#239;&#172;?ed. Proteins showing 14N/15N reciprocal ratios of <br /> to dry 30 min. Immunoblotting performed according              &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8 identi&#239;&#172;?ed experimental pair were <br /> standard protocol using rabbit anti-CBP IRDye goat anti-            subjected manual veri&#239;&#172;?cation. Peptides blasted against <br /> rabbit (LI-COR). Membranes visualized LI-COR                TAIR10 protein database (www.arabidopsis.org) to <br /> Odyssey infrared-imaging system.                                        determine uniqueness, nonunique peptides discarded. <br />    Metabolic Labeling Mass Spectrometry. Seedlings                  Reciprocal protein ratios proteins identi&#239;&#172;?ed or <br /> for metabolically labeled mass spectrometry experiments            peptides 14N/15N 15N/14N tissue pairs tested <br /> grown 1&#195;&#8212; M&amp;S <span id='am-1' about='Thesaurus:Micronutrient' typeof='owl:Thing'>micronutrient</span> solution supplemented                statistical signi&#239;&#172;?cance using two-sample, two-tailed <br /> 1.5 mM CaCl2, 0.75 mM MgSO4, 0.625 mM KH2PO4, 2.3 mM                    student&#226;&#8364;&#8482;s t test p &#226;&#8240;&#164; 0.05. Protein-abundance changes <br /> MES salts, 29.2 mM sucrose. Natural abundance (14N)              strengthened protein identi&#239;&#172;?ed both <br /> heavy (15N) NH4NO3 ( 0.825 g) 14N 15N KNO3 (0.96                 biological replicates direction change both <br /> g) added 1 L media, pH adjusted 5.7           replicates agreed. Proteins showing opposite abundance <br /> with KOH. Fourteen milligrams seeds grown 75 mL               changes reciprocal pairs discarded. <br /> of sterile media supplemented ampicillin. Seeds                  Mass Spectrometry IgG-Puri&#239;&#172;?ed Proteins. IgG- <br /> strati&#239;&#172;?ed dark 2 nights 4 &#194;&#176;C grown         puri&#239;&#172;?ed, TEV-eluted protein samples TCA/acetone- <br /> constant light gentle shaking 9 days. Seedlings           precipitated, pellets suspended 7.4 &#206;&#188;L 8 M <br /> spun-dry weighed, equal fresh weights heavy              urea, 50 mM NH4HCO3 (pH 8.5), 1 mM Tris-HCl 5 <br /> light seedlings combined. Microsome plasma                     min subsequently diluted 30 &#206;&#188;L 1.25 &#206;&#188;L 25 mM <br /> membrane puri&#239;&#172;?cation proceeded above. Methanol/chloro-               DTT, 2.5 &#206;&#188;L MeOH, 18.75 &#206;&#188;L 25 mM NH4HCO3 (pH 8.5) <br /> form extraction performed plasma membrane                    reduction alkylation. Samples incubated 50 &#194;&#176;C <br /> soluble fractions previously described.44 Protein                15 min cooled ice room temperature. IAA (1.5 &#206;&#188;L <br /> suspended 8 M urea/50 mM NH4HCO3 gentle                         55 mM) added, samples incubated room <br /> sonication diluted 1 M urea using 50 mM NH4HCO3. A               temperature dark 15 min. The reaction quenched <br /> BCA protein assay performed determine protein                    4 &#206;&#188;L 25 mM DTT. Eight microliters trypsin (10 ng/ <br /> concentration. Aliquots samples reduced 5 mM               &#206;&#188;L Trypsin Gold, Promega Corp., 25 mM NH4HCO3) and <br /> DTT &#226;&#710;&#188;50 &#194;&#176;C 45 min, alkylated 15 mM                          6.5 &#206;&#188;L 25 mM NH4HCO3 (pH 8.5) added &#239;&#172;?nal <br /> iodoacetamide room temperature 45 min, digested              volume 50 &#206;&#188;L. Digestion proceeded 2 h 42 &#194;&#176;C <br /> overnight 37 &#194;&#176;C 50:50 mixture trypsin/lys-C (total         followed additional 5 &#206;&#188;L trypsin &#239;&#172;?nal enzyme/ <br /> protein/enzyme ratio 100:1). Samples acidi&#239;&#172;?ed 0.3%            substrate ratio 1:40. Digestion proceeded overnight 37 &#194;&#176;C. <br /> TFA desalted using Waters tC18 sep-pak columns                      Digestions quenched 2.5% TFA 0.3% &#239;&#172;?nal <br /> according manufacturer&#226;&#8364;&#8482;s instructions. After drying          concentration, 8 &#206;&#188;L (&#226;&#710;&#188;700 ng) loaded nanoLC&#226;&#710;&#8217; <br /> samples, peptides solubilized LC&#226;&#710;&#8217;MS quality 0.1%                MS/MS analysis. Microsomal two-phase plasma mem- <br /> formic acid. Approximately 5 &#206;&#188;g samples loaded               brane-enriched fractions delipidated methanol/ <br /> analysis using Agilent 1100 LC inline LTQ             chloroform separation prior digestion. One-hundred micro- <br /> Orbitrap XL. Peptides separated in-house packed            grams microsomal protein 10 &#206;&#188;g plasma membrane- <br /> reversed-phase analytical column (Magic-C18, 200 &#195;&#8230;, 3 &#206;&#188;m,               enriched fractions precipitated methanol/chloroform/ <br /> Michrom Bioresources, Inc.) postloading mobile phase             water (4:1:1), protein pellet organic aqueous <br /> elution gradient 0&#226;&#710;&#8217;40% B 3 h (A, 0.1% formic acid; B,           interphase washed neat methanol. The protein dried <br /> 95% CH3CN 0.1% formic acid) 200 nL/min. Data-                    digested 100 &#206;&#188;L total volume 1:40 <br /> dependent analysis performed LTQ Orbitrap XL                 ratio trypsin/substrate. Peptides analyzed nanoLC&#226;&#710;&#8217; <br /> using FTMS preview mode MS1 scan acquisition                MS/MS using Agilent 1100 nano&#239;&#172;&#8218;ow (Agilent <br />                                                                   568                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                    Article <br />  <br /> Technologies) connected linear hybrid ion trap-orbitrap            test AHA1 transgene fused tandem a&#239;&#172;&#402;nity- <br /> mass spectrometer (LTQ-Orbitrap XL, Thermo Fisher                       puri&#239;&#172;?cation (TAP) tag rescue embryonic lethality. A <br /> Scienti&#239;&#172;?c) equipped nanoelectrospray ion source.                 TAP-tag construct optimized use plants containing two <br /> HPLC performed in-house fabricated 15 cm C18                  tandem protein A domains separated calmodulin- <br /> column packed MAGIC C18AQ 3 &#206;&#188;m particles (Michrom                  binding protein domain (CBP) TEV protease site was <br /> BioResources, Inc.) laser-pulled tip (P-2000, Sutter               ligated 5&#226;&#8364;&#178; end genomic AHA1 open reading frame32 <br /> Instrument) using 360 &#206;&#188;m &#195;&#8212; 75 &#206;&#188;m fused silica tubing. Sample            (Supporting Information Figure S1). <br /> loading desalting 15 &#206;&#188;L/min using trapping              Arabidopsis (Col-0, aha1/aha1;AHA2/aha2) trans- <br /> column line autosampler (Zorbax 300SB-C18, 5                formed Ntapi:gAHA1 clone Agrobacterium- <br /> &#206;&#188;M, 5 &#195;&#8212; 0.3 mm, Agilent Technologies). Peptide elution                  mediated &#239;&#172;&#8218;oral dip, successful transformation events were <br /> solvent A 0.1% formic acid water solvent B 0.1%          selected hygromycin media.40,47 Plants allowed self, <br /> formic acid 95% acetonitrile water. The &#239;&#172;&#8218;ow rate             PCR genotyping T2 population identi&#239;&#172;?ed aha1/ <br /> 200 nL/min. The column loaded, desalted,                        aha1;aha2/aha2 double-knockdown plants carrying the <br /> equilibrated 20 min 1% solvent B followed                  Ntapi:gAHA1 transgene. We recovered independent rescue <br /> increase 40% solvent B 195 min, ramped 60% solvent           lines expressing Ntapi:gAHA1 (Ntapi:gAHA1a Ntapi:gA- <br /> B 20 min, increased 100% solvent B 5 min, held           HA1b; TAPa TAPb). A band approximately 125 kDa <br /> at 100% solvent B additional 3 min. The column re-           detected western blots denaturing SDS protein gels <br /> equilibrated 1% B 30 min. MS/MS data acquisition             Ntapi:gAHA1 microsomal fractions probed protein A. <br /> gathered data-dependent mode follows: MS survey scans           Following two-phase partitioning, majority protein A <br /> <span id='am-6' about='protege:FROM' typeof='owl:Thing'>from</span> m/z 300 2000 collected centroid mode               staining enriches plasma membrane phase (Figure 1). In <br /> resolving power 100 000. MS/MS spectra collected on <br /> the &#239;&#172;?ve abundant signals survey scan. Dynamic <br /> exclusion 40 s precursors 0.55 m/z 1.05 <br /> m/z previously selected precursors employed to <br /> increase dynamic range maximize peptide <br /> identi&#239;&#172;?cations. Singly charged ions ions the <br /> charge state assigned rejected from <br /> consideration MS/MS. <br />    Quantitative Analysis. <span id='am-7' about='rdfs:label' typeof='owl:Thing'>Label</span>-free raw data &#239;&#172;?les were <br /> converted Mascot generic format (mgf) initial open- <br /> source mzXML format conversion using Trans-Proteomic <br /> Pipeline (TPP) software suite version 4.4. Mgf &#239;&#172;?les were <br /> searched Arabidopsis thaliana TAIR9 database with <br /> decoy reverse entries common contaminants included <br /> for total 27 279 Arabidopsis protein-coding genes 62 <br /> 523 total entries using in-house Mascot search engine 2.2.07 <br /> (Matrix Science) variable methionine oxidation and <br /> asparagine glutamine deamidation. Peptide mass tolerance <br /> was set 20 ppm fragment mass 0.8 Da. Protein <br /> annotations, identi&#239;&#172;?cation signi&#239;&#172;?cance, spectral-based <br /> quanti&#239;&#172;?cation using Sca&#239;&#172;&#8364;old software (version <br /> 3.6.3, Proteome Software Inc., Portland, OR.) Protein                   Figure 1. Detection Ntapi:AHA1 total microsomes (Mic), <br /> identi&#239;&#172;?cations accepted established              plasma membrane (PM), intracellular membranes (non-PM). Size <br /> greater 95% probability contained                 markers kDa. Upper panel coomassie stained, lower panel <br /> peptides using 1% false discovery rate. Protein probabilities         western blot probed HRP-conjugated rabbit anti-goat IgG. <br /> were assigned using Protein Prophet.46 Proteins containing <br /> nonunique peptides assigned MS/MS <br /> analysis grouped satisfy principles                addition major band 110 130 kDa <br /> parsimony. Metabolically labeled raw data acquired,                 markers, commonly observe slightly higher smaller <br /> converted, searched above. Mascot search results            &#226;&#710;&#188;70&#226;&#710;&#8217;80 kDa bands western blots. Smaller bands be <br /> transformed pepXML format using TPP software in-             partial proteolysis C-terminal regulatory domain, <br /> house-generated false-discovery-rate &#239;&#172;?lter. FDR peptide             larger bands alterred migration resulting <br /> identi&#239;&#172;?cation set 10% for low-quality MS/MS spectra          bound lipid detergent molecules. <br /> to allowed available manual inspection desired.              Previous studies Nicotiana plumbaginifolia plasma <br /> Relative quanti&#239;&#172;?cation 14N/15N peptide ratios performed          membrane proton pump, PMA2, indicate protein is <br /> using TPP tools Census software (Scripps Research                   present membrane hexameric complexes, especially <br /> Institute). <br />  <br /> &#226;&#8211;&#160; <br />                                                                         activation fusicoccin subsequent 14-3-3 protein <br />                                                                         binding.48,49 To determine molecular weight non- <br />    RESULTS                                                              denatured Ntapi:AHA1, puri&#239;&#172;?ed plasma membranes run <br />   Complementation TAP-Tagged AHA1 Trans-                         blue native gels probed rabbit anti-CBP antibody. <br /> gene. Our lab previously shown wild-type AHA1               We observed bands migrating close 242 720 <br /> AHA2 transgenes successfully rescue aha1/aha1;aha2/aha2                 kDa markers. Proteins migrate higher expected in <br /> double-knockdown embryo lethality.20 We wished             blue native gels based detergents dyes used bound <br />                                                                   569                         dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                        Article <br />  <br /> lipids; thus, bands likely correspond dimeric                   Ntapi:gAHA1b progeny performed the <br /> hexameric complexes observed previously (Figure 2).48                        genotype transformed seedling aha1&#226;&#710;&#8217;6;aha1&#226;&#710;&#8217;6/ <br />                                                                              aha2&#226;&#710;&#8217;4;aha2&#226;&#710;&#8217;4. <br />                                                                                  Ntapi:gAHA1 Plants Exhibit Reduced Root <span id='am-8' about='xsp:length' typeof='owl:Thing'>Length</span>. <br />                                                                              When grown vertically standard half-strength M&amp;S 1% <br />                                                                              (w/v) sucrose <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> media, observed approximately 20% <br />                                                                              reduced root length Ntapi:gAHA1 plants compared wild- <br />                                                                              type aha1/aha2 double knockdown mutants rescued a <br />                                                                              wild-type AHA1 transgene (gAHA1) (Figure 3A). We looked <br />                                                                              additional growth changes time &#239;&#172;?rst bolting <br />                                                                              (Figure 3B) plant growth soil (Figure 3C), did <br />                                                                              observe readily apparent di&#239;&#172;&#8364;erences. Work lab has <br />                                                                              previously shown aha2 single-knockdown mutants exhibit <br />                                                                              root-growth phenotypes seedlings grown stress <br />                                                                              conditions (high external pH, high external potassium, and <br />                                                                              toxic cations) alter rely proton motive force.20,37 <br />                                                                              To determine e&#239;&#172;&#8364;ects TAP-tagged AHA1 protein the <br />                                                                              proton motive force, grew Ntapi:gAHA1 plants the <br />                                                                              presence gentamicin lithium chloride. Under these <br />                                                                              conditions, aha2 knockdowns enhanced root growth <br />                                                                              compared wild-type plants reduced proton <br />                                                                              motive force energizes reduced uptake toxic <br />                                                                              compounds. Ntapi:gAHA1 lines similar root-growth <br />                                                                              phenotypes aha2 single knockdowns rescued a <br /> Figure 2. Three micrograms Ntapi:gAHA1 wild-type plasma                gAHA1 transgene (Figure 3D). <br /> membrane protein run blue native gels. Proteins                      Proteome Analysis Ntapi:gAHA1 Plants. With the <br /> transferred membranes probed rabbit anti-CBP antibodies.         ultimate goal using Ntapi:gAHA1 plants planta <br />                                                                              physiological investigation AHA1, wished establish <br />    The e&#239;&#172;&#402;ciency aha1/aha2 double-knockdown rescue                      closely transgenic lines resembled wild-type plants beyond <br /> Ntapi:gAHA1 assessed comparing genotypes                       merely visible phenotypes. To end, compared the <br /> aha1&#226;&#710;&#8217;6/aha1&#226;&#710;&#8217;6;AHA2/aha2&#226;&#710;&#8217;4;Ntapi:gAHA1a progeny                            plasma membrane soluble proteomes Ntapi:gAHA1 <br /> the ratios expected basis Mendelian segregation                plants wild-type gAHA1 rescue lines using a <br /> either single unlinked insertional events (Table 1).                metabolically labeled mass spectrometry approach determine <br />                                                                              expressing Ntapi:AHA1 proteomic consequences. <br /> Table 1. Progeny aha1&#226;&#710;&#8217;6/aha1&#226;&#710;&#8217;6;AHA2/aha2&#226;&#710;&#8217;4                                Equal wet weights seedlings grown natural-abundance <br /> Parental Plant Carrying Ntapi:gAHA1 Transgene                            (14N) isotopically heavy nitrogen (15N) liquid media were <br />                                   plants    experiment    experiment         mixed prior protein extraction. Subsequent downstream <br />             genotype               obsa         Ab            Bc             sample processing mass spectrometry analysis identical <br />     AHA2/aha2&#226;&#710;&#8217;4                    8        18.33         4.58               paired tissues, relative abundance peptide from <br />     AHA2/AHA2                      5        9.17          2.29               original tissue determined calculating ratio of <br />     aha2&#226;&#710;&#8217;4/aha2&#226;&#710;&#8217;4                  0        0             0                  14 <br />                                                                                 N/15N peptide intensity.52 A ratio 1.0 indicates no <br />     AHA2/aha2&#226;&#710;&#8217;4;Ntapi:gAHA1        55       41.25         51.56              di&#239;&#172;&#8364;erence peptide abundance metabolically labeled <br />     AHA2/AHA2;Ntapi:gAHA1          31       20.625        25.78              tissue pairs; ratio greater 1.0 indicates increased peptide <br />     aha2&#226;&#710;&#8217;4/aha2&#226;&#710;&#8217;4;Ntapi:gAHA1      11       20.625        25.78              abundance 14N-grown tissue compared 15N-grown <br />     total                          110      110           110                tissue; ratio 1.0 indicates decreased peptide <br />                                             p = 0.0002d   p = 0.004          abundance 14N-grown tissue <br /> a <br />   Number plants observed PCR genotyping. bNumber plants                We generated 14N/15N reciprocal 15N/14N tissue pairs of <br /> expected Mendelian segregation single transgene insertion.         liquid-grown, 9 day old wild-type/wild-type, wild-type/gAHA1, <br /> c <br />   Number plants expected Mendelian segregation unliked          wild-type/Ntapi:gAHA1a plants performed biological <br /> transgene insertional events. dResult chi-squared goodness &#239;&#172;?t          duplicates experimental pair. Proteomic analysis of <br /> tests observed expected genotypes using 4 degrees of <br /> freedom.                                                                     soluble plasma membrane proteomes performed. <br />                                                                              The number unique peptides proteins identi&#239;&#172;?ed by <br />                                                                              tandem mass spectrometry metabolically labeled <br /> The presence multiple insertions complex transgenic loci              tissue pair summarized Supporting Information Tables S1 <br /> following Agrobacterium-mediated transformation com-                      S2. Approximately half protein identi&#239;&#172;?cations all <br /> mon.50,51 Progeny ratios di&#239;&#172;&#8364;ered signi&#239;&#172;?cantly                      samples based single peptide discarded <br /> expected scenarios based chi-squared analysis                    analysis (Supporting Information Table S3). For <br /> closely resembled scenario involving unlinked transgene                proteins identi&#239;&#172;?ed peptide, peptide ratios <br /> insertional events. A reduction number rescued                     averaged achieve single 14N/15N protein ratio. Protein <br /> progeny expected number observed                   ratios plasma membrane (Figure 4) soluble (Figure 5) <br /> due altered expression, translation, tra&#239;&#172;&#402;cking, biochemical            fractions plotted ratios 14N/15N reciprocal <br />                                                                              15 <br /> activity fusion protein. A similar segregation analysis               N/14N genotype pairs. <br />                                                                        570                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                           Article <br />  <br />  <br />  <br />  <br /> Figure 3. Growth Ntapi:gAHA1 plants. (A) Vertical root growth (mm) plants half-strength M&amp;S 1% (w/v) sucrose media was <br /> measured 7 10 days. Bars indicate standard error, two-tailed, two-sample Student t tests used determine statistical di&#239;&#172;&#8364;erence (p &lt; <br /> 0.005, n &#226;&#8240;&#165; 12). (B) Sixteen seedling genotype transferred soil monitored visually bloting. (C) Representative experiment <br /> shown. Vegetative growth representative plants genotype 32 days. (D) Vertical root growth (mm) 4 day old wild-type, <br /> Ntapi:gAHA1a, Ntapi:gAHA1b seedlings transferred stress media 5 days. Bars indicate standard error, Student t tests performed <br /> as (p &lt; 0.05, n = 7). (E) Vertical root growth (mm) 4 day old wild-type, Ntapi:gAHA1a, gAHA1 seedlings transferred stress media <br /> for 5 days. <br />  <br />    Following manual veri&#239;&#172;?cation proteins showing 14N/15N                   rescue lines observable phenotypes standard <br /> and reciprocal ratios &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8, proteins met             laboratory growth conditions. In wild-type/Ntapi:gAHA1 tissue <br /> required criteria (see Experimental Procedures) 14Nwild-                   pairs, identi&#239;&#172;?ed plasma membrane proteins were <br /> type/15Nwild-type reciprocal plasma membrane soluble                   di&#239;&#172;&#8364;erentially abundant reciprocal experiments with <br /> protein fractions. In wild-type/gAHA1 tissue pairs, AHA1                 20% abundance change experiment (Table 3). <br /> itself showed statistically signi&#239;&#172;?cant reciprocal abundance               The majority proteins identi&#239;&#172;?ed decreased abundance <br /> changes biological replicates (Table 2). Despite                  Ntapi:gAHA1 seedlings compared wild-type, two, the <br /> reduced levels AHA1 protein membrane, gAHA1-                        phosphate transporter ATPT2 AHA1 itself, increased <br />                                                                         571                          dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                        Article <br />  <br />  <br />  <br />  <br /> Figure 4. Reciprocal plasma membrane protein ratios metabolically labeled tissue pairs. Wild-type (WT), wild-type AHA1-rescued (gAHA1), and <br /> Ntapi:gAHA1-rescued plants grown natural-abudance (14N) heavy isotope (15N) nitrogen media. Equal fresh weights (A) wild-type/ <br /> wild-type, (B) wild-type/gAHA1, (C) wild-type/Ntapi:gAHA1 genotype pairs biologically replicated experiments (upper lower <br /> plots) mixed prior homogenization, sample processing, MS analysis. Protein ratios (14N/15N) calculated averaging median- <br /> normalized peptide ratios determined Census, minimum MASCOT score 30. Calculated protein ratios reciprocally labeled tissue <br /> pairs plotted proteins identi&#239;&#172;?ed peptides samples. The number proteins showing reciprocal ratios &#226;&#8240;&#165;1.2 and <br /> &#226;&#8240;&#164;0.8 sample indicated. <br />  <br />  <br />  <br />  <br /> Figure 5. Reciprocal soluble protein ratios metabolically labeled tissue pairs. Wild-type (WT), wild-type AHA1-rescued (gAHA1), and <br /> Ntapi:gAHA1-rescued plants grown natural-abudance (14N) heavy isotope (15N) nitrogen media. Equal fresh weights (A) wild-type/ <br /> wild-type, (B) wild-type/gAHA1, (C) wild-type/Ntapi:gAHA1 genotype pairs biologically replicated experiments (upper lower <br /> plots) mixed prior homogenization, sample processing, MS analysis. Protein ratios (14N/15N) calculated averaging median- <br /> normalized peptide ratios determined Census, minimum MASCOT score 30. Protein ratios reciprocally labeled tissue pairs were <br /> plotted proteins identi&#239;&#172;?ed peptides samples. The number proteins showing reciprcal ratios &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8 each <br /> sample indicated. <br />  <br />  <br />                                                                       572                         dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                                          Article <br />  <br /> Table 2. Manually Veri&#239;&#172;?ed Protein Abundance Changes gAHA1 vs Wild-Type Plasma Membrane Fractions <br />                                                                             experiment 1                                                                 experiment 2 <br />                                                    14 <br />         accession              annotation               NgAHA1/ 15NWT                  14 <br />                                                                                            NWT/ 15NgAHA1                      14 <br />                                                                                                                                    NgAHA1/ 15NWT                     14 <br />                                                                                                                                                                         NWT/ 15NgAHA1 <br />       AT2G18960                  AHA1                     0.20 (5)a                         5.64 (6)                                     0.15 (3)                         4.67 (8) <br />                                                                                             p = 0.0003b                                                                   p = 0.001 <br />       AT5G44610                  MAP18                    1.33 (3)                          0.74 (2) <br /> a <br />  The number unique peptides ratios averaged calculate total protein ratio indicated parentheses. bTwo sample, two-tailed <br /> Student t tests used determine statistical signi&#239;&#172;?cance reciprocal protein ratios unique peptides detected each <br /> sample. <br />  <br /> Table 3. Manually Veri&#239;&#172;?ed Protein Abundance Changes Ntapi:gAHA1 vs Wild-Type Plasma Membrane Fractions <br />                                                                experiment 1                                     experiment 2                                        experiment 3 <br />                                             14            15          15        14              14         15          15           14              14         15          15 <br />      accession             annotation            NTAPa/ NWT            NWT/ NTAPa                    NTAPa/ NWT             NWT/ NTAPa               NTAPb/ NWT                 NWT/14NTAPb <br />                                                                a <br />     AT2G18960       AHA1                          1.06 (6)                 0.79 (9)                   1.93 (6)              0.62 (7)                     1.34 (9)               0.70 (5) <br />                                                                            p = 0.01b                                        p = 1 &#195;&#8212; 10&#226;&#710;&#8217;5                                        p = 1 &#195;&#8212; 10&#226;&#710;&#8217;4 <br />     AT2G38940       ATPT2                         1.69 (4)                 0.88 (3)                                         0.76 (2)                     1.08 (4)               0.69 <br />                                                                            p = 0.05 <br />     AT3G01290       band 7                        1.53 (6)                 0.69 (8)                   1.17 (4)              1.05 (6)                     1.74 (7)               0.47 (4) <br />                                                                            p = 6 &#195;&#8212; 10&#226;&#710;&#8217;9                                                                                         p = 2 &#195;&#8212; 10&#226;&#710;&#8217;6 <br />     AT1G66970       SVL2                          0.84 (7)                 1.37 (10)                  0.66 (8)              1.57 (8)                     0.80 (8)               0.95 (8) <br />                                                                            p = 2 &#195;&#8212; 10&#226;&#710;&#8217;5                                     p = 4 &#195;&#8212; 10&#226;&#710;&#8217;7 <br />     AT1G72150       PATL1                         0.94 (7)                 1.55 (7)                   0.72 (6)              2.47 (8)                     0.78 (10)              1.07 (11) <br />                                                                            p = 1 &#195;&#8212; 10&#226;&#710;&#8217;5                                     p = 6 &#195;&#8212; 10&#226;&#710;&#8217;6                                        p = 0.03 <br />     AT3G16450       jacalin family                0.96                     2.20 (4)                                         0.64                         0.75 (4)               1.38 (4) <br />                                                                                                                                                                                 p = 1 &#195;&#8212; 10&#226;&#710;&#8217;4 <br />     AT3G16460       jacalin family                0.65 (5)                 1.36 (8)                   0.91 (6)              1.35 (7)                     0.80 (11)              1.45 (16) <br />                                                                            p = 5 &#195;&#8212; 10&#226;&#710;&#8217;6                                     p = 8 &#195;&#8212; 10&#226;&#710;&#8217;4                                        p = 2 &#195;&#8212; 10&#226;&#710;&#8217;11 <br />     AT3G05900       neuro&#239;&#172;?lament protein-         0.64 (10)                1.21 (11)                  0.91 (4)              1.29 (4)                     0.70 (8)               1.44 (10) <br />                       related <br />     AT4G20260       DREPP family                  0.95 (6)                 1.24 (8)                   0.80 (3)              1.71 (6)                     0.93 (7)               1.15 (5) <br />                                                                            p = 0.009                                        p = 0.008                                           p = 0.006 <br /> a <br />  The number unique peptides ratios averaged calculate total protein ratio indicated parentheses. bTwo sample, two-tailed <br /> Student t tests used determine statistical signi&#239;&#172;?cance reciprocal protein ratios unique peptides detected each <br /> sample. A experiment performed comparing Ntapi:gAHA1b wildtype plants distinguish e&#239;&#172;&#8364;ects speci&#239;&#172;?c Ntapi:AHA1 <br /> expression versus transformation artifacts. <br />  <br /> abundance. The known cellular roles identi&#239;&#172;?ed proteins                                        use washing conditions containing 0.1% SDS additional <br /> include membrane transport (ATPT2), cell wall structure                                          salts reducing background protein contamination while <br /> (SVL2), vesicle tra&#239;&#172;&#402;cking (PATL1), microtubule structure                                     retaining speci&#239;&#172;?cally bound Ntapi:AHA1. <br /> and organization (DREPP).53&#226;&#710;&#8217;58                                                                       Identi&#239;&#172;?cation Ntapi:AHA1-Copurifying Proteins. <br />    TAP-Tag-Mediated Puri&#239;&#172;?cation AHA1. With goal                                           After establishing speci&#239;&#172;?c IgG puri&#239;&#172;?cation Ntapi:AHA1, we <br /> of using Ntapi:gAHA1 plants facilitate e&#239;&#172;&#402;cient isoform-                                       pursued identi&#239;&#172;?cation Ntapi:AHA1-copurifying proteins <br /> speci&#239;&#172;?c puri&#239;&#172;?cation AHA1 Arabidopsis tissue,                                          using a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry approach. <br /> investigated ability speci&#239;&#172;?cally purify Ntapi:AHA1                                   Ntapi:gAHA1 wild-type plants grown 14N and <br /> IgG puri&#239;&#172;?cation resin. We developed modi&#239;&#172;?ed tandem                                             15 <br />                                                                                                     N liquid media combined prior homogenization as <br /> a&#239;&#172;&#402;nity-puri&#239;&#172;?cation protocol purify Ntapi:AHA1 using                                      described previously. More technical puri&#239;&#172;?cation <br /> the &#239;&#172;?rst a&#239;&#172;&#402;nity-puri&#239;&#172;?cation step limit possibility losing <br />                                                                                                  replicates performed 14NNtapi:gAHA1/15Nwild-type <br /> protein interactors increased sample handling.60 Puri&#239;&#172;?ca- <br />                                                                                                  reciprocal 14Nwild-type/15NNtapi:gAHA1 tissue pairs. <br /> tion Ntapi:AHA1 plasma membrane fractions IgG <br />                                                                                                  Eluted proteins subjected tandem mass spectrometry <br /> resin demonstrated western blot Figure 6. Including <br /> 0.1% SDS IgG resin washing bu&#239;&#172;&#8364;er decreases                                                analysis, proteins showing reciprocal 14N/15N ratios <br /> nonspeci&#239;&#172;?cally bound proteins retaining high Ntapi:A-                                      deviating 1.0 wild-type Ntapi:gAHA1 <br /> HA1 antibody reactivity eluted fractions. Using normalized                                    seedlings subjected manual analysis. Four proteins <br /> spectral counts relative quanti&#239;&#172;?cation abundance                                  showed increased puri&#239;&#172;?cation Ntapi:gAHA1 plants <br /> Ntapi:AHA1 membrane puri&#239;&#172;?ed samples, protein                                          compared wild-type including additional AHA isoforms <br /> enrichment upper-phase fractions IgG resin                                            (Table 5). The enrichment AHA9 AHA11 from <br /> approximately 3.5-fold mild washing conditions (Table                                      Ntapi:gAHA1 versus wild-type protein extracts validated by <br /> 4). Addition 150 mM NaCl reduces MS identi&#239;&#172;?cation                                     searching raw data peaks corresponding unique <br /> of contaminating proteins results loss                                            peptides isoform (Figure 7). Because high <br /> Ntapi:AHA1 staining western blots TEV-eluted fractions                                     degree sequence identity isoforms, other <br /> (Supporting Information Figure S2). We routinely                                       AHA peptides discarded analysis; however, list <br />                                                                                           573                                      dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                       Article <br />  <br />                                                                                          proteins,61 wished investigate treatment vivo <br />                                                                                          compounds known hypothesized cause AHA <br />                                                                                          activation e&#239;&#172;&#8364;ect identi&#239;&#172;?cation NTapi:AHA1-copurify- <br />                                                                                          ing proteins. The fungal phytotoxin fusicoccin (FC) binds to <br />                                                                                          stabilizes AHA phosphorylated C-terminus 14-3-3 <br />                                                                                          protein complex, causing permanent activation pump.62 <br />                                                                                          We hypothesized treating liquid-grown seedlings FC <br />                                                                                          increase mass spectrometry identi&#239;&#172;?cation 14-3-3 <br />                                                                                          proteins IgG-puri&#239;&#172;?ed eluates, failed identify in <br />                                                                                          nonelicited copuri&#239;&#172;?cation experiments (Supporting Informa- <br />                                                                                          tion Table S5). Nine day old Ntapi:gAHA1 liquid-grown 14N + <br />                                                                                          FC/15N &#226;&#710;&#8217; FC reciprocally treated pairs subjected to <br />                                                                                          combined homogenization, puri&#239;&#172;?cation, MS analysis as <br />                                                                                          previously described. Treatment seedlings 1 &#206;&#188;M FC for <br />                                                                                          30 min increased puri&#239;&#172;?cation AHA11 14-3-3 <br /> Figure 6. Puri&#239;&#172;?cation Ntapi:AHA1 visulized western blot. One-                      protein isoforms including GRF1, GRF2, GRF3 (Table 6). <br /> hundred micrograms solubilized plasma membrane protein                            Auxin regulates plasma membrane proton pump activity by <br /> incubated IgG puri&#239;&#172;?cation resin, washed times 0.1%                       phosphorylation, protein kinases and/or phos- <br /> SDS resuspension bu&#239;&#172;&#8364;er followed times AcTEV,                        photases responsible identi&#239;&#172;?ed.63 Additionally, <br /> eluted overnight AcTEV protease. Eluted proteins acetone-                      inhibitory e&#239;&#172;&#8364;ect auxin root growth well-established <br /> precipitated resuspended directly Laemmli sample bu&#239;&#172;&#8364;er                       likely involves regulation proton pump activity.64 When <br /> reducing agent. One microgram nonplasma membrane (non-PM) <br />                                                                                          seedlings treated 1 &#206;&#188;M indole-3-acetic-acid 15 <br /> and plasma membrane (PM) wild-type Ntapi:gAHA1 protein and <br /> half TEV-eluted proteins loaded 4&#226;&#710;&#8217;12% NuPAGE gels.                            min, observed decreased copuri&#239;&#172;?cation protein kinase <br /> The left panel silver stained total protein, right panel             PHOT1 (Table 6). <br />  <br />                                                                                          &#226;&#8211;&#160; <br /> western blot probed rabbit anti-CBP followed LI-COR IRDye <br /> goat anti-rabbit antibody. Size-marker values kDa.                                     DISCUSSION <br /> Table 4. Successive Puri&#239;&#172;?cation Ntapi:AHA1 Liquid-                               Herein, functional rescue aha1/aha2 <br /> Grown Seedlings                                                                          knockdown embryonic lethality Arabidopsis stable <br />                                                                                          transformation N-terminal TAP-tagged genomic <br />                                           spectral                  fold                 AHA1 transgene, used a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass <br /> step                  fraction             Ctsa      NSAFb      enrichmentc <br />                                                                                          spectrometry approach copurify candidate Ntapi:AHA1- <br />      1   microsomes                         75       0.0012                              interacting proteins. The ability identify proteins interacting <br />      2   non-PM membranes                   63       0.0010                              AHA1 planta particular strength approach <br />      3   plasma membranes                   273      0.0051        4.25 <br />                                                                                          feasible heterologous methods used identify <br />      4   IgG bound, TEV-eluted              99       0.0175        14.6 <br />            (mild washd)                                                                  protein interactions, including yeast two-hybrid split <br />      5   IgG bound, TEV-eluted              100      0.5639        470                   ubiquitin systems. <br />            (stringent washe)                                                                While observed slight reduction number of <br /> a <br />   Spectral counts representative sample determined &#226;&#8364;&#339;number                    endogenous double knockdown, transgene-rescued progeny <br /> of assigned spectra&#226;&#8364;? Sca&#239;&#172;&#8364;old. bNormalized spectral abundance                          expected numbers based Mendelian segregation, <br /> factor calculated according method Zhang et al.59                              result non-WT transgene expression or <br /> c <br />   Enrichment Ntapi:AHA1 microsomal samples determined                         altered translation, tra&#239;&#172;&#402;cking, activity tagged protein, <br /> by increasing NSAF values. dTwice 100&#195;&#8212; volume 1% NP40,                         resulting embryonic lethality. Western blots <br /> with 100&#195;&#8212; volume 1% NP40, 0.01% SDS. eThree times 100&#195;&#8212;                              two-phase separated membranes showed Ntapi:AHA1 <br /> volume 0.1% SDS, 150 mM NaCl. <br />                                                                                          enriched plasma membrane fractions compared to <br />                                                                                          nonplasma membrane fractions, expected. This result <br /> of identi&#239;&#172;?ed AHA peptides 14N/15N ratios                               alleviated concerns TAP tag deleteriously <br /> included Supporting Information (Table S4).                                       a&#239;&#172;&#8364;ecting proper maturation tra&#239;&#172;&#402;cking Ntapi:AHA1 <br />    Response Ntapi:AHA1-Copurifying Proteins                                     protein, rule possibility a <br /> Vivo Treatment. Because autoinhibited state AHA                                portion protein mistra&#239;&#172;&#402;cked misfolded. <br />  <br /> Table 5. Ntapi:AHA1-Copurifying Proteins Reciprocal Metabolically Labeled Tissue Pairs <br />           accession                   annotation                    unique peptidesa                14 <br />                                                                                                          NTAP/15NWT ratiob                   14 <br />                                                                                                                                                   NWT/15NTAP ratioc <br />          AT2G18960               AHA1                                          7                              14.10                                    0.03 <br />          AT1G80660               AHA9                                          1                              24.34                                    0.04 <br />          AT5G62670               AHA11                                         2                              1.73                                     NDd <br />          AT4G10480               unknown/&#206;&#177;-NAC                                 2                              1.35                                     0.71 <br />          AT1G62480               unknown/Ca2+-binding                          2                              1.32                                     0.72 <br />  <br /> a <br />  Peptides BLASTed agains TAIR10 database determine uniqueness. bPeptide ratios IgG-puri&#239;&#172;?ed, TEV-eluted proteins from <br /> 14 <br />   NNtapi:gAHA1/15NCol-0 WT metabolically labeled, liquid-grown tissue pairs. cPeptide ratios IgG-puri&#239;&#172;?ed, TEV-eluted proteins from <br /> reciprocally labeled tissue pairs. dPeptide intensity 14N-grown tissue low detection raw data Mascot set 14N search mode. <br />  <br />                                                                                    574                           dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                Article <br />  <br />  <br />  <br />  <br /> Figure 7. Peptides unique AHA isoforms enrich Ntapi:AHA1 TAP-tag-mediated puri&#239;&#172;?cation. Metabolically labeled <br /> 14 <br />   NNtapi:gAHA1/15Nwild-type (red) reciprocal 14Nwild-type/15NNtapi:gAHA1 (blue) plasma membrane pairs incubated IgG resin, <br /> washed, eluted overnight TEV protease. The raw data inspected manually peaks corresponding indicated unique peptides <br /> from AHA1, AHA9, AHA11. <br />  <br /> Table 6. Response Ntapi:AHA1-Copurifying Proteins Vivo Treatments <br />          accession                 annotation               treatment                 14 <br />                                                                                            NTAP+/15NTAP&#226;&#710;&#8217;a                            14 <br />                                                                                                                                           NTAP&#226;&#710;&#8217;/15NTAP+b <br />                                                                    c <br />        AT2G18960                   AHA1                       FC                              0.93 (8)                                       0.98 (6) <br />                                                                                               p = 0.74 <br />                                                               IAAd                            1.00 (5)                                       1.14 (7) <br />                                                                                               p = 0.52 <br />        AT5G62670                   AHA11                      FC                              1.10 (3)e                                      0.67 (3) <br />                                                                                               p = 0.03f <br />        AT4G09000                   GRF1                       FC                              1.41(4)                                        0.95 (4) <br />                                                                                               p = 0.006 <br />        AT1G78300                   GRF2                       FC                              1.13 (3)                                       0.80 (3) <br />                                                                                               p = 0.003 <br />        AT5G38480                   GRF3                       FC                              1.25 (4)                                       0.76 (5) <br />                                                                                               p = 0.003 <br />        AT3G45780                   PHOT1                      IAA                             0.87 (2)                                       1.17 (5) <br />        AT2G26080                   ATGLDP2                    IAA                             1.23 (3)                                       0.94 (3) <br />                                                                                               p = 0.02 <br />        AT3G23990                   HSP60                      IAA                             1.21 (4)                                       0.81 (2) <br /> a <br />  Liquid-grown Ntapi:gAHA1 seedlings: 14N seedlings treated, 15N seedlings mock treated. bReciprocal experiment: 14N seedlings mock treated, 15N <br /> seedlings treated. cFusicoccin treatment: 1 &#206;&#188;M 30 min. dIndole-3-acetic acid treatment: 1 &#206;&#188;M 15 min. eNumber unique peptides included <br /> in protein ratio average indicated parentheses. fStatistical signi&#239;&#172;?cance based two-sample, two-tailed Student&#226;&#8364;&#8482;s t test. <br />  <br />    Ntapi:gAHA1-rescue lines exhibited minimal observable                      e&#239;&#172;&#8364;ect.56 The DREPP family protein identi&#239;&#172;?ed study, <br /> phenotypic consequences &#226;&#710;&#188;20% reduced vertical                      PcaP1, microtubule destabilizing activity results the <br /> root growth standard media. However, mass spectrometry                     negative regulation hypotcotyl elongation.58 A closely related <br /> revealed subtle plasma membrane proteomic phenotype                      gene, PcaP2, negatively regulates root hair tip growth.67 These <br /> Ntapi:gAHA1 plants. Several proteins showing abundance                        proteins bind Ca2+/calmodulin complexes; therefore, <br /> di&#239;&#172;&#8364;erences Ntapi:gAHA1 versus wild-type plants                        Ntapi:AHA1 construct interefereing normal <br /> reported roles cell wall structure, vesicle tra&#239;&#172;&#402;cking,                 regulation activity protein respect to <br /> cell elongation, proteins contributing factors             calmodulin binding. Together, altered abundance these <br /> to observed Ntapi:gAHA1 short-root phenotype. SVL2                     proteins Ntapi:gAHA1 seedlings contributing to <br /> related plasma membrane anchored, glycerophosphoryl                    reduced vertical root growth. It possible tip <br /> diester phosphodiesterase protein SHV3. SHV3 plays role                  growing systems a&#239;&#172;&#8364;ected. For example, pollen tube <br /> cell wall rigidity, knockout mutants exhibit ruptured root                growth endogenous knockout pollen carrying the <br /> hairs swollen guard cells.57 PATL1 expressed root tip               Ntapi:gAHA1 transgene negatively a&#239;&#172;&#8364;ected, may <br /> cells plays important role membrane tra&#239;&#172;&#402;cking                   explain reduced numbers Ntapi:gAHA1 rescue progeny <br /> cell plate maturation, important processes cell                      observed. <br /> elongation.54,65,66 AtPrx34 known cell wall associated                     Because essential nature AHA1 AHA2 the <br /> peroxidase expressed root cells plays role cell                  number important plant processes are <br /> elongation. Knockout lines closely related gene, AtPrx33,                hypothesized function, hypothesize interacting <br /> have short roots, double knockouts exhibit additive                    proteins, including protein kinases important regulation <br />                                                                         575                               dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                        Article <br />  <br /> of AHA phosphorylation multiple sites C-terminal            material available free charge Internet http:// <br /> regulatory region,68 identi&#239;&#172;?ed. Initial exper-             pubs.acs.org. <br />  <br />                                                                          &#226;&#8211;&#160; <br /> imentation identify Ntapi:AHA1-interacting proteins identi- <br /> &#239;&#172;?ed additional copurifying AHA isoforms, AHA9                         AUTHOR INFORMATION <br /> AHA11. We shown AHA proteins form <br />                                                                          Corresponding Author <br /> multimeric complexes plasma membranes, those <br /> complexes homomultimeric heteromultimeric                    *E-mail: msussman@wisc.edu. Phone: 608-262-8608. Fax: 608- <br /> respect isoform composition shown. Our                   262-6748. <br /> copuri&#239;&#172;?cation results suggest complexes mixed          Funding <br /> their isoform composition opens possibility         U.S. Department Energy O&#239;&#172;&#402;ce Basic Energy Sciences <br /> the isoform composition native AHA complexes play               grant DEFG02-88ER13938 Michael R. Sussman. Molecular <br /> important regulatory signaling roles.69 These results          Biosciences Training Grant, NIH National Research Service <br /> contrast recent publication Bobik et al., showed           Award T32 GM07215 Rachel Rodrigues (Nee Nelson). <br /> that Nicotiana plumbaginifolia H+-ATPase isoforms PMA2               Notes <br /> PMA4 did oligomerize other.70 The di&#239;&#172;&#8364;erent                 The authors declare competing &#239;&#172;?nancial interest. <br />  <br />                                                                          &#226;&#8211;&#160; <br /> results di&#239;&#172;&#8364;erences experimental design, <br /> including use natural versus overexpression promoters              ABBREVIATIONS USED <br /> di&#239;&#172;&#8364;erences biological activity H+-ATPase isoforms <br /> from Arabidopsis versus Nicotiana. We observed increased            AHA, Arabidopsis H+-ATPase; PMF, proton motive force; TAP, <br /> copuri&#239;&#172;?cation AHA11 treatment fungal toxin             tandem a&#239;&#172;&#402;nity puri&#239;&#172;?cation; M&amp;S, Murashige &amp; Skoog; AP&#226;&#710;&#8217; <br /> FC, suggesting isoform composition quaternary                MS, a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry; CBP, calmodulin- <br /> AHA complexes constantly &#239;&#172;&#8218;ux responsive             binding protein <br /> environment. One obvious limitation rescue plants <br /> do express AHA2, which, high expression <br /> levels shared sequence identity AHA1, would <br />                                                                          &#226;&#8211;&#160;    REFERENCES <br />                                                                            (1) Ku&#204;&#710;hlbrandt, W. (2004) Biology, structure mechanism P- <br /> predict common isoform complex                    type ATPases. Nat. Rev. Mol. Cell Biol. 5, 282&#226;&#710;&#8217;295. <br /> AHA1. In absence AHA2, isoforms                        (2) Buckhout, T. J. (1989) Surcose transport isolated plasma- <br /> artifactually incorporating multimeric AHA complexes.               membrane vesicles sugar beet (Beta vulgaris L.) Evidence an <br />    The autoinhibited state AHA1 reduce ability             electrogenic sucrose-proton symport. Planta 178, 393&#226;&#710;&#8217;399. <br /> identify interacting proteins. We tested             (3) Bush, D. R. (1993) Proton-coupled sugar amino acid <br />                                                                          transporters plants. Annu. Rev. Plant Physiol. Plant Mol. Biol. 44, <br /> expand identi&#239;&#172;?cation copurifying proteins combining            513&#226;&#710;&#8217;542. <br /> in vivo treatments hypothesized known regulate              (4) Palmgren, M. G. (2001) Plant plasma membrane H+-ATPases: <br /> AHA activity metabolic labeling, a&#239;&#172;&#402;nity puri&#239;&#172;?cation,            Powerhouses <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> uptake. Annu. Rev. Plant Physiol. Plant Mol. <br /> mass spectrometry. The dynamic copuri&#239;&#172;?cation PHOT1                Biol. 52, 817&#226;&#710;&#8217;845. <br /> 14-3-3 response exogenous auxin fusicoccin suggest                (5) Sussman, M. H. R. (2012) The effect genetically reduced <br /> that experiments reveal similar, potentially            plasma membrane protonmotive force vegetative growth of <br /> novel, AHA1-interacting proteins response speci&#239;&#172;?c                  Arabidopsis. Plant Physiol. 158, 1158&#226;&#710;&#8217;1171. <br /> conditions. A current limitation inability discern               (6) Sussman, M. R. (1994) Molecular analysis proteins plant <br /> whether proteins di&#239;&#172;&#8364;erential abundance puri&#239;&#172;?ed                   plasma membrane. Annu. Rev. Plant Physiol. Plant Mol. Biol. 45, 211&#226;&#710;&#8217; <br /> samples vivo elicitation re&#239;&#172;&#8218;ect true Ntapi:AHA1-                234. <br />                                                                            (7) DeWitt, N. D., Harper, J. F., Sussman, M. R. (1991) <br /> interacting proteins indicate changes abundance             Evidence plasma membrane proton pump phloem cells of <br /> these proteins plasma membrane fractions prior                     higher plants. Plant J. Cell Mol. Biol. 1, 121&#226;&#710;&#8217;128. <br /> puri&#239;&#172;?cation IgG resin. We addressed problem                   (8) DeWitt, N. D., Sussman, M. R. (1995) Immunocytological <br /> also analyzing aliquot unpuri&#239;&#172;?ed plasma membrane                   localization epitope-tagged plasma membrane proton pump (H+- <br /> samples tandem mass spectormetry; however,                      ATPase) phloem companion cells. Plant Cell 7, 2053&#226;&#710;&#8217;2067. <br /> increased sample complexity nonpuri&#239;&#172;?ed samples result               (9) Schmid, M., Davison, T. S., Henz, S. R., Pape, U. J., Demar, M., <br /> in decreased peptide identi&#239;&#172;?cation inconclusive results,             Vingron, M., Scho&#204;&#710;lkopf, B., Weigel, D., Lohmann, J. U. (2005) A <br /> especially abundant proteins. Introducing cross-linking         gene expression map Arabidopsis thaliana development. Nat. Genet. <br /> reagents maintain protein interactions increasing           37, 501&#226;&#710;&#8217;506. <br /> denaturing quality washing steps remove nonspeci&#239;&#172;?cally               (10) Winter, D., Vinegar, B., Nahal, H., Ammar, R., Wilson, G. V., <br />                                                                          Provart, N. J. (2007) An &#226;&#8364;&#339;electronic fluorescent pictograph&#226;&#8364;? <br /> bound proteins puri&#239;&#172;?cation resin facilitate the <br />                                                                          browser exploring analyzing large-scale biological data sets. <br /> identi&#239;&#172;?cation interacting proteins future. <br />  <br /> &#226;&#8211;&#160; <br />                                                                          PLoS One 2, e718-1&#226;&#710;&#8217;e718-12. <br />                                                                            (11) Certal, A. C., Almeida, R. B., Carvalho, L. M., Wong, E., <br />      ASSOCIATED CONTENT                                                  Moreno, N., Michard, E., Carneiro, J., Rodrigue&#204;?z-Le&#204;?on, J., Wu, H.-M., <br /> * Supporting Information <br />  S                                                                       Cheung, A. Y., Feijo&#204;?, J. A. (2008) Exclusion proton ATPase <br /> Full predicted protein sequence Ntapi:AHA1 expressed              apical membrane associated cell polarity tip <br /> protein; information metabolically labeled             growth Nicotiana tabacum pollen tubes. Plant Cell 20, 614&#226;&#710;&#8217;634. <br />                                                                            (12) Feijo&#204;?, J. A., Sainhas, J., Hackett, G. R., Kunkel, J. G., Hepler, <br /> proteome experiments including total protein peptide                 P. K. (1999) Growing pollen tubes possess constitutive alkaline band <br /> identi&#239;&#172;?cations; list AHA peptides identi&#239;&#172;?ed                    clear zone growth-dependent acidic tip. J. Cell Biol. 144, <br /> metabolically labeled, nonelicited Ntapi:gAHA1 copuri&#239;&#172;?cation             483&#226;&#710;&#8217;496. <br /> experiments; list 14-3-3 general regulatory factor proteins             (13) Merlot, S., Leonhardt, N., Fenzi, F., Valon, C., Costa, M., Piette, <br /> identi&#239;&#172;?ed metabolically labeled, nonelicited Ntapi:gAHA1              L., Vavasseur, A., Genty, B., Boivin, K., Muller, A., Giraudat, J., and <br /> copuri&#239;&#172;?cation experiments 14N/15N ratios. This                 Leung, J. (2007) Constitutive activation plasma membrane H <br />  <br />                                                                    576                            dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                 Article <br />  <br /> +-ATPase prevents abscisic acid-mediated stomatal closure. EMBO J.                 profiling tandem affinity purified 14-3-3 protein complexes in <br /> 26, 3216&#226;&#710;&#8217;3226.                                                                     Arabidopsis thaliana. Proteomics 9, 2967&#226;&#710;&#8217;2985. <br />   (14) Liu, J., Elmore, J. M., Fuglsang, A. T., Palmgren, M. G.,                    (31) Wittho&#204;&#710;ft, J., Caesar, K., Elgass, K., Huppenberger, P., Kilian, J., <br /> Staskawicz, B. J., Coaker, G. (2009) RIN4 functions plasma                Schleifenbaum, F., Oecking, C., Harter, K. (2011) The activation <br /> membrane H+-ATPases regulate stomatal apertures                          Arabidopsis P-ATPase 1 brassinosteroid receptor BRI1 is <br /> pathogen attack. PLoS Biol. 7, e1000139-1&#226;&#710;&#8217;e1000139-16.                             independent threonine 948 phosphorylation. Plant Signaling Behav. <br />   (15) Hayashi, Y., Nakamura, S., Takemiya, A., Takahashi, Y.,                     6, 1063&#226;&#710;&#8217;1066. <br /> Shimazaki, K.-I., Kinoshita, T. (2010) Biochemical character-                   (32) Rohila, J. S., Chen, M., Cerny, R., Fromm, M. E. (2004) <br /> ization vitro phosphorylation dephosphorylation                   Improved tandem affinity purification tag methods isolation of <br /> plasma membrane H+-ATPase. Plant Cell Physiol. 51, 1186&#226;&#710;&#8217;1196.                      protein heterocomplexes plants. Plant J. Cell Mol. Biol. 38, 172&#226;&#710;&#8217; <br />   (16) Kinoshita, T., Hayashi, Y. (2011) New insights                 181. <br /> regulation stomatal opening blue light plasma membrane                    (33) Rubio, V., Shen, Y., Saijo, Y., Liu, Y., Gusmaroli, G., Dinesh- <br /> H(+)-ATPase. Int. Rev. Cell Mol. Biol. 289, 89&#226;&#710;&#8217;115.                                Kumar, S. P., Deng, X. W. (2005) An alternative tandem affinity <br />   (17) Robertson, W. R., Clark, K., Young, J. C., Sussman, M. R.               purification strategy applied Arabidopsis protein complex isolation. <br /> (2004) An Arabidopsis thaliana plasma membrane proton pump                      Plant J. Cell Mol. Biol. 41, 767&#226;&#710;&#8217;778. <br /> essential pollen development. Genetics 168, 1677&#226;&#710;&#8217;1687.                          (34) Chang, I.-F. (2006) Mass spectrometry-based proteomic <br />   (18) Gevaudant, F., Duby, G., von Stedingk, E., Zhao, R.,                        analysis epitope-tag affinity purified protein complexes in <br /> Morsomme, P., Boutry, M. (2007) Expression constitutively                 eukaryotes. Proteomics 6, 6158&#226;&#710;&#8217;6166. <br /> activated plasma membrane H+-ATPase alters plant development                    (35) Leene, J. V., Stals, H., Eeckhout, D., Persiau, G., Slijke, E. V. D., <br /> increases salt tolerance. Plant Physiol. 144, 1763&#226;&#710;&#8217;1776.                           Isterdael, G. V., Clercq, A. D., Bonnet, E., Laukens, K., Remmerie, N., <br />   (19) Baxter, I. R., Young, J. C., Armstrong, G., Foster, N.,                     Henderickx, K., Vijlder, T. D., Abdelkrim, A., Pharazyn, A., Onckelen, <br /> Bogenschutz, N., Cordova, T., Peer, W. A., Hazen, S. P., Murphy, A.                H. V., Inze&#204;?, D., Witters, E., Jaeger, G. D. (2007) A tandem affinity <br /> S., Harper, J. F. (2005) A plasma membrane H+-ATPase                        purification-based technology platform study cell cycle <br /> required formation proanthocyanidins seed coat                   interactome Arabidopsis thaliana. Mol. Cell. Proteomics 6, 1226&#226;&#710;&#8217; <br /> endothelium Arabidopsis thaliana. Proc. Natl. Acad. Sci. U.S.A. 102,            1238. <br /> 2649&#226;&#710;&#8217;2654.                                                                          (36) Qi, Y., Katagiri, F. (2009) Purification low-abundance <br />   (20) Haruta, M., Burch, H. L., Nelson, R. B., Barrett-Wilt, G., Kline,           Arabidopsis plasma-membrane protein complexes identification of <br /> K. G., Mohsin, S. B., Young, J. C., Otegui, M. S., Sussman, M. R.              candidate components. Plant J. Cell Mol. Biol. 57, 932&#226;&#710;&#8217;944. <br /> (2010) Molecular characterization mutant Arabidopsis plants                 (37) Haruta, M., Sussman, M. R. (2012) The effect a <br /> reduced plasma membrane proton pump activity. J. Biol. Chem. 285,                  genetically reduced plasma membrane protonmotive force on <br /> 17918&#226;&#710;&#8217;17929.                                                                       vegetative growth Arabidopsis thaliana. Plant Physiol. 158, 1158&#226;&#710;&#8217; <br />   (21) Lalonde, S., Ehrhardt, D. W., Loque&#204;?, D., Chen, J., Rhee, S. Y.,            1171. <br /> and Frommer, W. B. (2008) Molecular cellular approaches                 (38) Krysan, P. J., Young, J. C., Tax, F., Sussman, M. R. (1996) <br /> detection protein-protein interactions: Latest techniques                   Identification transferred DNA insertions Arabidopsis genes <br /> current limitations. Plant J. 53, 610&#226;&#710;&#8217;635.                                         involved signal transduction ion transport. Proc. Natl. Acad. Sci. <br />   (22) Miernyk, J. A., Thelen, J. J. (2008) Biochemical approaches             U.S.A. 93, 8145&#226;&#710;&#8217;8150. <br /> for discovering protein-protein interactions. Plant J. Cell Mol. Biol. 53,          (39) Hellens, R. P., Edwards, E. A., Leyland, N. R., Bean, S., and <br /> 597&#226;&#710;&#8217;609.                                                                           Mullineaux, P. M. (2000) pGreen: A versatile flexible binary Ti <br />   (23) Fukao, Y. (2012) Protein-protein interactions plants. Plant              vector Agrobacterium-mediated plant transformation. Plant Mol. <br /> Cell Physiol. 53, 617&#226;&#710;&#8217;625.                                                         Biol. 42, 819&#226;&#710;&#8217;832. <br />   (24) Gingras, A.-C., Gstaiger, M., Raught, B., Aebersold, R.                  (40) Clough, S. J., Bent, A. F. (1998) Floral dip: A simplified <br /> (2007) Analysis protein complexes using mass spectrometry. Nat.                 method Agrobacterium-mediated transformation Arabidopsis <br /> Rev. Mol. Cell Biol. 8, 645&#226;&#710;&#8217;654.                                                   thaliana. Plant J. Cell Mol. Biol. 16, 735&#226;&#710;&#8217;743. <br />   (25) Arabidopsis Interactome Mapping Consortium (2011) Evidence                   (41) Nelson, C. J., Huttlin, E. L., Hegeman, A. D., Harms, A. C., and <br /> for network evolution Arabidopsis interactome map. Science 333,              Sussman, M. R. (2007) Implications 15N-metabolic labeling for <br /> 601&#226;&#710;&#8217;607.                                                                           automated peptide identification Arabidopsis thaliana. Proteomics 7, <br />   (26) Chen, J., Lalonde, S., Obrdlik, P., Noorani Vatani, A., Parsa, S.           1279&#226;&#710;&#8217;1292. <br /> A., Vilarino, C., Revuelta, J. L., Frommer, W. B., Rhee, S. Y. (2012)           (42) Larsson, C., Widell, S. Isolation plant plasma membranes <br /> Uncovering Arabidopsis membrane protein interactome enriched                    production inside-out vesicles, Aqueous Two-Phase Systems: <br /> transporters using mating-based split ubiquitin assays classification          Methods Protocols (Hatti-Kaul, R., Ed.) pp 159&#226;&#710;&#8217;166, Humana <br /> models. Front. Plant Sci. 3, 124.                                                  Press, Totowa, NJ. <br />   (27) Jahn, T., Fuglsang, A. T., Olsson, A., Bru&#204;&#710;ntrup, I. M., Collinge,           (43) Lee, M. C. S., Hamamoto, S., Schekman, R. (2002) <br /> D. B., Volkmann, D., Sommarin, M., Palmgren, M. G., Larsson, C.                Ceramide biosynthesis required formation oligomeric <br /> (1997) The 14-3-3 protein interacts directly C-terminal                   H+-ATPase Pma1p yeast endoplasmic reticulum. J. Biol. Chem. <br /> region plant plasma membrane H+-ATPase. Plant Cell 9, 1805&#226;&#710;&#8217;                 277, 22395&#226;&#710;&#8217;22401. <br /> 1814.                                                                               (44) Wessel, D., Flu&#204;&#710;gge, U. I. (1984) A method the <br />   (28) Morandini, P., Valera, M., Albumi, C., Bonza, M. C., Giacometti,            quantitative recovery protein dilute solution presence of <br /> S., Ravera, G., Murgia, I., Soave, C., De Michelis, M. I. (2002) A             detergents lipids. Anal. Biochem. 138, 141&#226;&#710;&#8217;143. <br /> novel interaction partner C-terminus Arabidopsis thaliana                (45) Park, S. K., Venable, J. D., Xu, T., Yates, J. R., 3rd (2008) A <br /> plasma membrane H+-ATPase (AHA1 isoform): Site mechanism                       quantitative analysis software tool mass spectrometry-based <br /> of action H+-ATPase activity differ 14-3-3 proteins.              proteomics. Nat. Methods 5, 319&#226;&#710;&#8217;322. <br /> Plant J. Cell Mol. Biol. 31, 487&#226;&#710;&#8217;497.                                               (46) Nesvizhskii, A. I., Keller, A., Kolker, E., Aebersold, R. <br />   (29) Fuglsang, A. T., Guo, Y., Cuin, T. A., Qiu, Q., Song, C.,                   (2003) A statistical model identifying proteins tandem mass <br /> Kristiansen, K. A., Bych, K., Schulz, A., Shabala, S., Schumaker, K. S.,           spectrometry. Anal. Chem. 75, 4646&#226;&#710;&#8217;4658. <br /> Palmgren, M. G., Zhu, J.-K. (2007) Arabidopsis protein kinase                   (47) Zhang, X., Henriques, R., Lin, S.-S., Niu, Q.-W., Chua, N.- <br /> PKS5 inhibits plasma membrane H+-ATPase preventing                          H. (2006) Agrobacterium-mediated transformation Arabidopsis <br /> interaction 14-3-3 protein. Plant Cell 19, 1617&#226;&#710;&#8217;1634.                         thaliana using floral dip method. Nat. Protoc. 1, 641&#226;&#710;&#8217;646. <br />   (30) Chang, I.-F., Curran, A., Woolsey, R., Quilici, D., Cushman, J.              (48) Kanczewska, J., Marco, S., Vandermeeren, C., Maudoux, O., <br /> C., Mittler, R., Harmon, A., Harper, J. F. (2009) Proteomic                    Rigaud, J.-L., Boutry, M. (2005) Activation plant plasma <br />  <br />                                                                              577                           dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                 Article <br />  <br /> membrane H+-ATPase phosphorylation binding 14-3-3                        sec14-related protein, zucchini (Cucurbita pepo). J. Plant Physiol. <br /> proteins converts dimer hexamer. Proc. Natl. Acad. Sci. U.S.A.            163, 1150&#226;&#710;&#8217;1158. <br /> 102, 11675&#226;&#710;&#8217;11680.                                                                   (67) Kato, M., Aoyama, T., Maeshima, M. (2013) The Ca2+- <br />   (49) Ottmann, C., Marco, S., Jaspert, N., Marcon, C., Schauer, N.,               binding protein PCaP2 located plasma membrane involved in <br /> Weyand, M., Vandermeeren, C., Duby, G., Boutry, M., Wittinghofer,                  root hair development possible signal transducer. Plant J. Cell Mol. <br /> A., Rigaud, J.-L., Oecking, C. (2007) Structure 14-3-3                    Biol. 74, 690&#226;&#710;&#8217;700. <br /> coordinated hexamer plant plasma membrane H+-ATPase                       (68) Rudashevskaya, E. L., Ye, <br /> </body></html>