<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yki/YAP, Sd/TEAD Hth/MEIS Control Tissue <br /> Specification Drosophila Eye Disc Epithelium <br /> Tianyi Zhang1., Qingxiang Zhou1., Francesca Pignoni1,2* <br /> 1 Department Ophthalmology, Center Vision Research, SUNY Eye Institute, SUNY Upstate Medical University, Syracuse, New York, United States America, <br /> 2 Departments Biochemistry Molecular Biology, Neuroscience Physiology, SUNY Upstate Medical University, Syracuse, New York, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      During animal development, accurate control tissue specification growth critical generate organisms of <br />      reproducible shape size. The eye-antennal disc epithelium Drosophila powerful model identify the <br />      signaling pathway transcription factors mediate coordinate processes. We Yorkie <br />      (Yki) pathway plays major role tissue specification developing fly eye disc epithelium time organ <br />      primordia regional identity domains specified. RNAi-mediated inactivation Yki, partner Scalloped (Sd), or <br />      increased activity upstream negative regulators Yki cause dramatic reorganization eye disc fate map <br />      leading specification entire disc epithelium retina. On contrary, constitutive expression Yki suppresses <br />      eye formation Sd-dependent fashion. We knockdown transcription factor Homothorax (Hth), <br />      known partner Yki developmental contexts, induces ectopic retina domain, Yki Scalloped <br />      regulate Hth expression, gain-of-function activity Yki partially dependent Hth. Our results support a <br />      critical role Yki- partners Sd Hth - shaping fate map eye epithelium independently universal <br />      role regulator proliferation <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>survival</span>. <br />  <br />   Citation: Zhang T, Zhou Q, Pignoni F (2011) Yki/YAP, Sd/TEAD Hth/MEIS Control Tissue Specification Drosophila Eye Disc Epithelium. PLoS ONE 6(7): <br />   e22278. doi:10.1371/journal.pone.0022278 <br />   Editor: Andreas Bergmann, University Texas MD Anderson Cancer Center, United States America <br />   Received April 24, 2011; Accepted June 17, 2011; Published July 19, 2011 <br />   Copyright: &#195;&#376; 2011 Zhang et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported National Institutes Health (NIH) grant R01EY13167 FP, Research Prevent Blindness (RPB) Unrestricted Grant <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />   Department Ophthalmology, SUNY Upstate Medical University, Lions District 20-Y1 Central New York. The funders role study design, <br />   data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: pignonif@upstate.edu <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                         major regulator organ body size. In agreement this <br />                                                                                      view, shown regulate cell proliferation survival <br />    Animal body size shape genetically determined                           larval eye disc [11&#226;&#8364;&#8220;14], implicated <br /> evolutionarily conserved signaling pathways control <span id='am-315' about='xsp:pattern' typeof='owl:Thing'>pattern</span>-                    early patterning tissue. The core transduction <br /> ing growth. The activity pathways integrated                      elements pathway include kinases control the <br /> produce organisms similar dimensions body pattern                       cytoplasmic versus nuclear localization non-DNA-binding <br /> given biological species. How different signaling pathways                           transcriptional co-activator protein called Yki flies YES- <br /> contribute processes intensely studied imaginal                   Associated-Protein (YAP) vertebrates. These include kinase <br /> discs Drosophila, including eye-antennal disc epithelium.                     Hpo phosphorylates activates kinase Warts (Wts). <br /> Although significant overlap control tissue                       Phosphorylated Wts, then, associates protein Mob-as- <br /> specification growth larval eye epithelium,                         tumor-suppressor (Mats) to, turn, phosphorylate Yki. Phos- <br /> primary role ascribed signaling pathways                    phorylated Yki sequestered cytoplasm, effectively <br /> they contribute predominant fashion patterning                     preventing controlling gene expression. Thus, the <br /> proliferation.                                                                       presence Hpo signaling, Yki inactive. Conversely, the <br />    For instance, changes level distribution secreted                absence Hpo signaling, Yki active, nuclear, associates <br /> morphogen Decapentaplegic (Dpp), BMP 2/4 type factor, affects                      DNA-binding partners form complexes regulate gene <br /> proliferation disc. Indeed, Dpp acts inducer                 expression. <br /> or suppressor depending timing, spatial distribution likely                    One direct partner TEAD/TEF-type DNA-binding <br /> concentration (reviewed [1]). However, striking effect                   protein Scalloped (Sd). Sd Yki contribute up-regulation <br /> changes Dpp expression early fate map dramatic                     genes Diap1, Bantam CyclinE, promoting cell <br /> changes location, size, ectopic gain organ                        survival proliferation developing wing epithelium [12&#226;&#8364;&#8220; <br /> primordial [2&#226;&#8364;&#8220;5].                                                                    13,15]. In developing eye, however, Yki clearly <br />    Conversely, Hippo -Yorkie (Hpo-Yki) signaling pathway                      critical proliferation, Sd appears make minor contribution <br /> known virtually universal regulator proliferation                        [14]. Recently, transcription factors Homothorax (Hth), a <br /> survival vertebrates invertebrates (reviewed [6&#226;&#8364;&#8220;10]).                 TALE-type homeobox protein, Teashirt, zinc finger protein, <br /> In Drosophila, thought function essentially tissues             shown associate Yki contribute the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                1                                   July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                    Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> up-regulation pro-proliferation microRNA Bantam eye                     primordium. At L3 stage, wave secreted factors sweeps <br /> progenitor cells [14].                                                            DP cell layer posterior anterior induces eye <br />    The eye-antennal disc powerful model studying                  progenitors stop dividing, acquire specific cell fate begin <br /> the genetic control proliferation tissue specification. It            differentiate photoreceptor neurons accessory cells <br /> gives rise neural (including sensory organs)                  form single eye ommatidium (Figs. 1A, S1A) <br /> non-neural fly head structures. The &#226;&#8364;&#732;eye&#226;&#8364;&#8482; portion epithelium               (reviewed [17&#226;&#8364;&#8220;18]). Recent studies shown PE <br /> (called eye disc) consists sheet cells gives rise eye,            contributes development DP cell layer, the <br /> ocelli (additional light-sensory organs) cuticle fly head.             formation adult head structures reorgani- <br /> The developing epithelium folded flattened sac                 zation discs metamorphosis (reviewed [16]). <br /> opposing cell layers separated lumen continuous                    Despite this, understanding genetic control of <br /> much disc margin (Figs. 1A, S1A). During L1 L2                     specification, cell morphology differentiation PE is <br /> larval stages, eye disc grows proliferation acquires              infancy. <br /> regional identity. By larval stage (L3), layers                  In paper identify major players (Yki, Sd Hth) <br /> be readily distinguished morphology and, them, groups                   development PE. We transcriptional <br /> of cells fated rise defined regions adult           regulators control regional specification PE blocking <br /> fly head (Figs. 1A, S1A). The disc proper (DP) cell layer                     formation ectopic retina organ primordium cell <br /> columnar, pseudostratified morphology, known rise                  layer promoting PE/cuticle formation. In addition, we <br /> to adult eye surrounding cuticle. The squamous                            implicate upstream regulators Yki Hippo signaling <br /> peripodial portion epithelium (PE)                       pathway proper patterning eye disc. We show <br /> understood contributes cuticle ventral posterior                gene function required PE time regional <br /> regions fly head. Differences morphology                    specification (L2 stage) promote PE identity. We propose that <br /> two cell layers detected early L2 (reviewed [16]).             activities reflect function Hpo/Yki pathway tissue <br /> During L2 stage, transcription factors Eyeless (Ey),                      specification distinct activity regulating tissue <br /> Teashirt (Ths), Eyes absent (Eya), Sine oculis (So), Dachshund                growth survival, partner-specificity, addition Yki <br /> (Dac), collectively called Retina Determination Factors (RDFs),                   activity levels, contributes significant way differential <br /> come co-expressed (Ey, Tsh &#226;&#8364;&#8220;.Eya &#226;&#8364;&#8220;.So &#226;&#8364;&#8220;.Dac)                       activity Yki specification versus proliferation different <br /> portion DP define eye retina organ                        regions eye disc. <br />  <br />  <br />  <br />  <br /> Figure 1. Loss Yki Sd changes fate map eye disc. In figures: posterior left; Gal4 driver Act.IC.Gal4 unless otherwise <br /> stated; UAS transgenes used marked panels; magnification shown panel applies panels figure unless marked; XY <br /> images projections Z optical planes unless stated; ZX ZY images disc scans edited display selected regions of <br /> the discs marked yellow lines XY view; colors chose optimize image quality reflect corresponding <br /> detection channel; precise genotypes temperature crosses Table S1. A) Schematic diagram developing eye disc L3 stage with <br /> expression pattern molecular markers used work. Larger image provided Fig. S1A. B) In wt (top), PE cells squamous DP cells <br /> are columnar. In sd-RNAi-expressing discs (bottom), cell layers display columnar appearance pseudostratified DP monolayer. As <br /> shown uneven staining apical versus basal portions cell membranes position neuronal nuclei shown panel D, the <br /> polarity cells disrupted; apical faces disc lumen cases. C, D) wt (left), yki-RNAi (middle) sd-RNAi (right) discs; C) L3 <br /> discs stained RDF Eya (red) retinal cells apical membrane marker E-cadherin (green) highlight cell morphology. Mutant <br /> discs display DP-like retina forming cell layers. D) L3 discs stained neuronal marker Elav (red) membrane marker Dlg (green). <br /> Mutant discs contain developing mirror-image, neuronal arrays; cell layers, neurons begin form posterior rows added <br /> progressively expanding ommatidial field anterior. <br /> doi:10.1371/journal.pone.0022278.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                             2                                 July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                             Hpo/Yki Pathway Tissue Specification <br />  <br />  <br />  <br /> Results                                                                      function PE involved complex long range <br />                                                                              effects. We, therefore, used PE-specific Gal4 line pnr-Gal4 [19] <br /> Loss Yki Sd function causes transformation                      drive 2 copies UAS-GFP copy UAS-yki-RNAi <br /> the PE DP/retina                                                        transgene. In control discs, GFP expression exclusively <br />    In course RNAi-based genetic screen [Zhang &amp;                    restricted dorsal PE cell layer the <br /> Pignoni, unpublished] identify early players patterning            larval stages (Fig. 2A, shown). At L2 stage (when gene <br /> and development fly eye (ey-Flip; Act.IC.Gal4 UAS-GFP;                function required, shown below), GFP detected large <br /> UAS-Dicer2/UAS-RNAi; Fig. S1B legend driver                      dorsal section PE cell layer entire eye-antennal disc <br /> expression pattern), discovered disc-wide knock-down             (Fig. 2A). In pnr-Gal4 yki-RNAi discs, dorsal half PE <br /> of Yki partner Sd induced dramatic reorganization            epithelium columnar displayed developing omma- <br /> eye-disc fate map (here refer Table S1 exact            tidial array L3 stage (Fig. 2B). As shown magnified <br /> genotypes). Instead squamous (PE) columnar (DP)               panel Fig. 2B, transition columnar, transformed PE <br /> cell layer, sd-RNAi- yki-RNAi-expressing L3 discs displayed           cells unchanged squamous PE cells occur sharply a <br /> 2 columnar DP-like layers (Fig. 1B, shown). The yki-RNAi             cell diameters. Unfortunately, use genetic <br /> sd-RNAi discs considerably smaller similar staged wild             background explore signs non-cell-autonomy are <br /> type (wt) discs cell layers strongly expressed RD               present co-expressed GFP longer detected the <br /> factor Eya, marker retinal progenitors developing           stage cell fate readily assessed (late L3). This <br /> retinal cells (Fig. 1C). In addition, cells expressing pan-neural        unexpected, pnr-Gal4 PE-specific driver, hence <br /> marker Elav seen DP layer (as wt)            expression down-regulated time transformed PE- <br /> also transformed PE, indicating neurogenesis                 cells locked DP fate. <br /> ommatidia assembly occurred cell layers (Fig. 1D). We               In conclusion, Yki activity required PE cell layer to <br /> confirmed Yki protein expression was, fact, down-regulated           establish proper tissue identity. For ease scoring, however, <br /> by yki-RNAi line clones (Fig. S2A) similar                  additional experiments carried using ey-Flip; <br /> phenotypes induced other, previously published, RNAi-                Act.IC.Gal4 driver causes transformation entire PE. <br /> transgenes yki sd (Table S2, Fig. S3A, B) [13]. Thus, we <br /> believe changes detected disc morphology fate             Normal development ectopic retina sd-RNAi <br /> are product down-regulation Yki Sd                      discs <br /> expression.                                                                     During normal development, eye neurogenesis initiates the <br />    Surprisingly, previous studies Yki Sd eye disc         posterior margin L3 disc expands anteriorly the <br /> not uncovered function cell fate [11&#226;&#8364;&#8220;14]. Loss-of-function           DP cell layer, progressively anterior rows eye <br /> yki, particular, reported affect cell proliferation       progenitor cells induced begin neuronal differentiation <br /> survival profoundly cell fate              (reviewed [17,18]). A stripe dpp (dpp-lacz) expression marks <br /> markers used, neuronal marker Elav, used assess                transition zone groups progenitor cells commit a <br /> development mutant clones [14]. In cases                   neuronal fate begin differentiate (Figs. 1A, 2C). Within the <br /> traditional loss-of-function mutant clones,                 DP, RDFs Ey tsh (tsh-lacZ) strongly expressed in <br /> Minute, generated [11&#226;&#8364;&#8220;12,14]. In case, similar                    progenitor cells ahead dpp stripe, absent the <br /> approach utilized, sd yki silenced               developing ommatidial field posterior dpp; the <br /> RNAi [13]. To investigate source discrepancy,            neuronal markers Elav 22C10 mark differentiating <br /> generated sd-RNAi- yki-RNAi-expressing clones                      neurons posterior region (Figs. 1A, 1D, 2C). The Elav- <br /> MARCM method. As expected, RNAi-expressing clones                       positive nuclei located apically columnar DP cells, <br /> reduced size compared GFP-only control clones               axons, marked 22C10, run basal of <br /> cases, severely case yki-RNAi (Fig.               disc extend posteriorly optic stalk (Figs. 1C, <br /> S4). In case, did detect change fate; regardless        2C, S5). <br /> whether clones laid DP PE cell layer (Fig. S4).               The development ectopic retina transformed <br /> These results consistent previously reported clonal             PE follows pattern normal retina. In sd-RNAi disc, <br /> phenotypes differences gene <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>knock-out</span>                   dpp-positive stripe seen progressively anterior <br /> approaches (rather use mutant alleles opposed              positions neuronal field expands, stripe sweeps <br /> RNAi transgenes) responsible discrepancy mutant               disc posterior anterior cell layers <br /> phenotypes. As explained discussion, techniques differ          (Fig. 2D). Often always, development neuronal <br /> in ways: extent mutant tissue generated timing            array transformed cell layer lagged neurogenesis in <br /> of gene function inactivation, pointing non-cell-autonomy          normal DP layer. Ey tsh strongly expressed both <br /> or early (L1/L2) gene function explanations                   layers anterior domains, consistent expression retina <br /> observations.                                                                progenitor cells (Fig. 2D). Whereas expression pan-neural <br />    In conclusion, loss Yki Sd function results striking           markers Elav 22C10 observed posterior region of <br /> transformation PE cell layer second DP                  cell layer, expected developing neuronal array. In <br /> developing retina (PE-to-PD/retina transformation), indicating               layers, Elav positive nuclei located near apical <br /> that activity proteins required establishment        (lumenal) surface columnar cells, 22C10 positive axons <br /> of PE identity.                                                              extended basal (external) posterior <br />                                                                              margin (Figs. 1D, 2C, 2D, S5). <br /> Yki function required PE establish PE-fate                         In summary, PE-to-DP transformation appears be <br />    Since ey-Flip; Act.IC.Gal4 driver drives expression           complete, pertains retina domain. The <br /> PE ad DP cell layers, yki appears function non-cell-             anterior-posterior apical-basal polarity transformed <br /> autonomous fashion, sought investigate                     tissue unchanged, eye morphogenesis begins similarly in <br /> transformation <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>phenotype</span> resulted loss Yki                cell layers posterior margin disc. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        3                              July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                 Hpo/Yki Pathway Tissue Specification <br />  <br />  <br />  <br />  <br /> Figure 2. Gene silencing PE sufficient induce PE&#226;&#8364;&#8220;to-DP transformation. A) pnr-Gal4 drives expression ,early-mid L2 <br /> discs. Top: XY ZX views disc; bottom: ZX another. Gal4 expressed broad domain dorsal PE. B) pnr-Gal4 UAS-ykiRNAi disc <br /> stained Dlg (for cell shape) Elav (for neurons). Right: confocal XY image L3 disc ZX ZY projections; left: high magnification ZY of <br /> the disc. PE-restricted expression yki-RNAi transforms dorsal region PE DP/retina. High magnification panel shows the <br /> transition transformed non-transformed region occurs cell diameters. C, D) L3 discs stained red (top bottom) dpp (dpp- <br /> lacZ) marks transition zone eye progenitor cells developing neurons, RDFs tsh (tsh-lacZ) Ey mark eye <br /> progenitor cells, pan-neural membrane marker 22C10 highlights differentiating neurons axons. Where shown, membrane <br /> marker E-cadherin green. C) wt L3 discs normal expression dpp-lacZ transition zone morphogenesis ommatidial array <br /> begins (visible depression DP wt discs, marked transformed discs), tsh-lacZ Ey eye progenitor cells the <br /> anterior portion disc (expression Ey PE layer seen), 22C10 stained neurons projects axons posteriorly, basal <br /> side DP optic stalk brain (not shown). Asterisk marks axonal fascicle outside eye disc (Bolwig&#226;&#8364;&#8482;s nerve). D) sd-RNAi L3 discs <br /> show mirror image duplications expression domain thicker, DP-like cell layers. The domains dpp-lacZ expression offset, <br /> indicating that, disc, neurogenesis layer lags layer. This case. <br /> doi:10.1371/journal.pone.0022278.g002 <br />  <br /> Sd function required L2 stage suppress retina                     morphology ectopic Eya expression posterior disc <br /> identity PE                                                             (Fig. 3A), extent PE-to-DP transformation somewhat <br />    The specification retina primordium wt eye disc               variable disc disc. By early L3 (72 h AEL 25uC), sd-RNAi <br /> occurs L2 stage, eye morphogenesis begins L3,            discs showed strong signs transformation (Fig. 3A). Thus, the <br /> and time essentially disc cells proliferating.         morphological changes Eya induction associated PE-to- <br /> Early L2, PE DP cells acquire distinct morphology                 DP transformation apparent L2 stage. This <br /> [20]. At time, formation retina primordium begins               indicates lack Sd activity L2 earlier results failure to <br /> the induction Eya expression posterior margin              maintain PE identity progressive conversion PE-like <br /> disc DP cell layer [21]. To better understand process            tissue DP. Interestingly, changes appeared emanate <br /> PE-to-DP/retina transformation, eye discs analyzed                posterior margin epithelium (Fig. 3A, S6). This <br /> earlier developmental stage.                                                   posterior-to-anterior gradient ectopic eye primordium formation <br />    Discs expressing sd-RNAi indistinguishable wt               similar observed normal development [21]. In wt <br /> just prior mid-L2 (54 hr AEL 25uC) (Fig. 3A). At time, PE             discs, Eya expression triggered cells posterior margin <br /> cell layer thinner columnar DP wt sd-                 begin secrete Dpp. The similar pattern primordium <br /> RNAi discs. At mid-to-late L2 (60 hr AEL 25uC), clearly               development cell layers sd-RNAi discs suggests that <br /> so hours later (66 hr AEL 25uC), PE showed columnar                  Dpp acts common inductive signal posterior margin. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          4                                July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                Hpo/Yki Pathway Tissue Specification <br />  <br />  <br />  <br />  <br /> Figure 3. Sd function required L2 PE development. A) ZX view developing L2 discs stained Eya (red) E-cadherin <br /> (green). Ectopic Eya expression clearly seen 60 hr AEL (,mid-late L2), discs indistinguishable 54 hr AEL (,mid L2). B) Top <br /> panels: XY ZX views wt disc stained Yki (green), E-cadherin (blue) DAPI (red). XY shows PE cell layer only. Yki protein detected <br /> throughout PE apical regions DP cells. Nuclear Yki detection level available Ab wt eye disc. Bottom panel: XY <br /> and ZX views wt disc stained &#195;&#376;-galactosidase reporter transgene Diap1-2-lacZ (green) E-cadherin (red). XY shows PE cell layer only. <br /> Expression direct target Diap1-2-lacZ cell layers apparently higher PE DP L2, L3 (not <br /> shown). C, D) The red bar graphs marks phenocritical period L2. The percentage flies pharate adults showing transformation- <br /> related phenotypes shift experiment shown right. Developmental stage marked below; development proceeds more <br /> than twice fast 29uC 18uC, approximate time molt larval stages occurs different times marked grey columns. <br /> Control progeny (maintained 18uC throughout) develop normal adults. The lack sharp transition likely indirect method Sd <br /> regulation (Gal80ts = /Gal4 R sd-RNAi = /Sd) delays changes Sd protein levels. Time temperature shift 12 hr apart C 6 hr apart in <br /> D. See materials methods legend Fig. S6 details. <br /> doi:10.1371/journal.pone.0022278.g003 <br />  <br />  <br />    As mentioned above, developmental analysis mutant                    53% 14%. Later times sd inactivation <br /> phenotype indicated transformation apparent                 resulted transformation-related phenotypes. Thus, inactiva- <br /> the L2 stage. Yki is, indeed, expressed L2 eye discs (Fig. 3B) and,         tion endogenous Sd function time embryogenesis <br /> although Sd antibodies available, expression reporter             mid-late L2, later, compromises tissue identity <br /> construct Yki-Sd control (Diap1&#226;&#8364;&#8220;2 lacZ; [12]) detected             PE indicating protein activity required stage <br /> both cell layers stage (Fig. 3B). However, RNAi expression             lock PE fate. Conversely, Fig. 3D shows restoration <br /> under ey-Flip; Act.IC.Gal4 begins embryo, eye disc             Sd function (shift-down experiment) prior mid-late L1 <br /> first forms. Hence, requirement Yki Sd PE formation             causes minimal disruption PE specification (non-transformation <br /> may reflect protein function time prior late L2 stage.           phenotypes $80% flies); restoration Sd function at <br />    To identify phenocritical period protein activity,               L1/L2 molt (in L2 L3) insufficient to <br /> utilized Gal4/Gal80ts [22]. The Gal80ts protein                ensure proper PE development (the penetrance transfor- <br /> temperature sensitive form Gal80 negative regulator                  mation phenotypes increase ,50% flies early L2 to <br /> Gal4. Thus, permissive temperature (18uC), Gal80ts blocks               nearly 100% mid-late L2). Thus, conclude Sd function <br /> Gal4 activity expression sd-RNAi transgene                       necessary promote PE identity early L2. These <br /> prevented (18uC = wt Sd); non-permissive                    results consistent emergence Eya-positive <br /> temperature (29uC), Gal80ts inactive Gal4 active, resulting          ectopic retina domain L2 stage shown Fig. 3A. <br /> in sd-RNAi transgene expression (29uC = loss-of-function Sd).                 Together, analyses point L2 stage time Sd <br /> Using reagents, temporal expression sd-RNAi                       function suppressing retina formation promoting PE <br /> controlled shifting 1 hr embryo collections 18uC                    identity (Fig. 3C). Noticeably, critical period eye disc <br /> 29uC, vice versa, different time points larval develop-           retina specification eye-antennal disc. <br /> ment. In shift-up (18uC 29uC) shift-down (29uC 18uC) <br /> experiments, extent PE-to-DP transformation assessed                The TALE homeobox factor Hth contributes PE <br /> at adult stage scoring eye/head phenotypes correlated              development <br /> with varying degrees PE-to-DP transformation developing                     Recently, DNA-binding transcription factor been <br /> discs (see Fig. S6 detailed approach data).                            implicated Yki-regulated gene expression. Mann colleagues <br />    Using approach, identified time L1/L2               shown homeobox protein Homeothorax (Hth) and <br /> molt mid-late L2 phenocritical period Sd gene                   Yki bind vitro directly regulate transcription gene <br /> function. As shown Fig. 3C, penetrance transformation-               Bantam eye progenitor cells [14]. Since Hth expressed <br /> related phenotypes high (92% flies more),              PE cell layer [23](Fig. 4A) mis-expression in <br /> sd silenced (shift experiment) prior mid L2.               DP suppress eye morphogenesis (assessed Eya and <br /> Gene silencing mid-late end L2 resulted                  Elav expression) [24,25], decided investigate Hth <br /> drastic drop number flies showing signs transformation,           plays role PE formation. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          5                               July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                  Hpo/Yki Pathway Tissue Specification <br />  <br />  <br />  <br />  <br /> Figure 4. Hth controls PE formation expression regulated Yki Sd. A) Wild type L3 disc stained Hth (red) Dlg <br /> (green); top: XY view PE layer; middle: XZ view disc. Bottom: XY view DP layer. Notice Hth broadly expressed PE DP cell <br /> layers, excluded eye field. B) ZX views hth-RNAi expressing discs stained Eya (red) E-cadherin (green) (top) Elav <br /> (red) Dlg (green) (bottom). Discs transformation PE DP ectopic retina field (Eya-positive cells) ectopic neuronal <br /> development (Elav-positive clusters). C) ZX views L3 eye discs expressing exogenous Hth-GFP fusion protein (left), Hth-GFP plus sd- <br /> RNAi (middle), Hth-GFP plus yki-RNAi (right) stained anti-GFP (green) detect Hth-GFP, Eya (red) Elav (blue). No expression Eya, Elav <br /> is detected. Co-expression sd-RNAi yki-RNAi does rescue loss retina Hth mis-expression. D) Top: ZX views Yki-expressing L3 <br /> eye disc stained Hth (red) Dlg (green). Over-expression Yki induces ectopic Hth expression (compare panel A). Bottom: ZX views sd- <br /> RNAi-expressing L3 eye disc stained Hth (red) UAS-GFP (blue). Down-regulation Sd cell marked GFP results loss Hth expression. <br /> This consistent development ectopic retina transformed PE shown panel B. <br /> doi:10.1371/journal.pone.0022278.g004 <br />  <br />  <br />    Knock-down endogenous Hth achieved expressing                        In summary, Hth required PE development and <br /> UAS-hth-RNAi transgenes (Table S2, Fig. S2B). Discs                   appears function concert downstream Yki <br /> stained L3 stage retinal neuronal markers.                       Sd establishment PE-identity. <br /> Interestingly, hth-RNAi expressing discs displayed two <br /> mirror-image DP-like cell layers PE, similarly yki-RNAi               Sd Hth mediate Yki function <br /> expressing discs. Eya Elav expressed cell layers                  Since Sd Hth shown bind Yki and <br /> indicating retina primordia formed neurogenesis                provide functional specificity developmental contexts, we <br /> was progress (Fig. 4B). A strong phenotype similar trans-                 sought investigate Sd Hth related Yki the <br /> formations induced yki-RNAi sd-RNAi evident                        maintenance PE identity suppression retinal fate. <br /> ,24% hth-RNAi discs (Table S3); discs showed                       Therefore, induced over-expression Yki investi- <br /> partial transformation. Similar results obtained                    gated co-expression sd-RNAi hth-RNAi affected Yki <br /> second, independent hth-RNAi line (Fig. S3C). Thus, conclude                 function. <br /> that loss Hth function results PE-to-DP transformation                 Since Yki activity required suppress retinogenesis PE <br /> development ectopic retina PE.                                     cell layer, conversely, increasing Yki activity DP cell layer <br />    To test Yki Sd mediate effect Hth                 interfere normal development. Over-expression of <br /> negative regulators retina identity, Hth-GFP over-expressed              exogenous Yki eye disc resulted greatly enlarged <br /> alone sd-RNAi yki-RNAi. As previously                       disorganized discs folds epithelial layers. <br /> reported, discs Hth-GFP over-expression reduced size               Although cellular morphology Yki-expressing DP layer <br /> and lacked retina domain shown lack Eya Elav               remained columnar, neuronal development detected <br /> expression (Fig. 5C; [24,25]). Neither co-expression sd-RNAi,                Elav expression (Fig. 5A). This phenotype detected in <br /> nor co-expression yki-RNAi, restore eye development                    Yki-overexpressing clones (MARCM method) full <br /> within disc (Fig. 4C). Hence, Hth does function upstream                penetrance (not shown). In addition, expression RDFs Ey, <br /> of Yki Sd promoting PE identity.                                          tsh (tsh-lacz), Eya Dac absent (Fig. 5A), indicating that <br />    On contrary, Hth expression eye disc             formation and/or maintenance eye primordium was <br /> controlled Yki Sd. Hth expressed DP                 compromised. Therefore, high levels Yki suppress retina <br /> Yki-expressing discs, acquires DP-like pattern                  fate highest levels RD gene network. <br /> expression (on early progenitors, developing                     To test Sd mediates effect Yki negative <br /> neuronal field) transformed PE sd-RNAi discs (Fig. 4D).               regulator retina identity, Yki sd-RNAi expressed <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           6                                 July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                 Hpo/Yki Pathway Tissue Specification <br />  <br />  <br />  <br />  <br /> Figure 5. Suppression retina formation Yki over-expression depends Sd Hth. A) XY ZX views wt (left) Yki over- <br /> expressing discs (right) stained Ey (red) plus Eya (green) (top), tsh-lacZ (middle), Dac (red) plus Elav (green) (bottom). Over-expression Yki <br /> within eye disc results suppression eye formation highest level (tsh Ey) RD cascade. As shown Fig. 4D, discs express <br /> Hth throughout. These flies die pharate adults lack heads. Insets discs DIC (top panels), UAS-GFP <br /> expression (middle panel) discs&#226;&#8364;&#8482; outlines. B) XY ZX views discs co-expressing Yki sd-RNAi. Discs stained Eya (red) plus E- <br /> cadherin (green) (top) Dac (red) plus Elav (green) (bottom); sd-RNAi restores eye formation DP layer Yki-expressing discs, also <br /> induces ectopic retina PE (compare wt sd-RNAi disc Fig. 1C). C) XY ZX views disc co-expressing Yki, hth-RNAi, dicer2 <br /> (to increase RNAi efficacy). Discs stained Elav (green) Eya (red); hth-RNAi restores eye development Yki-expressing discs minimally <br /> and DP cell layer (compare wt panel A hth-RNAi disc Fig. 4B). <br /> doi:10.1371/journal.pone.0022278.g005 <br />  <br /> separately eye epithelium. Whereas                  Fig. 5C). Contrary observed Yki sd-RNAi discs, <br /> Yki-expressing discs showed abnormal DP layer, lacking                      expression markers detected cell layer <br /> expression eye markers (Fig. 5A), co-expression sd-RNAi                  (Fig. 5C). Incomplete knock-down Hth explain less <br /> led formation double DP (Fig. 5B), similar             striking effect hth-RNAi compared sd-RNAi, though <br /> observed sd-RNAi-only discs (Figs. 1C, 2D, S3B). Thus, Yki                  staining anti-Hth antibodies revealed strong suppression of <br /> cannot suppress retina identity partner Sd.                        Hth expression (Fig. S2B; Table S3). Alternatively, Hth may <br />    In similar experiments, Yki hth-RNAi expressed                     mediate some, all, Yki functions PE thus <br /> separately test Hth, like Sd, required           work different capacity Sd. <br /> Yki suppress retina identity. In case,             These findings, previous evidence hth <br /> hth-RNAi restore retina markers Yki-expressing                   regulation Yki Sd, suggest model Sd is <br /> discs, strikingly sd-RNAi. The expression          absolutely required Yki function establishing PE identity, <br /> several RDFs restored different extents: Ey expression             Hth expression initially induced maintained by, then <br /> robustly, Eya Dac weakly (Fig. S7). When Dicer 2                  contributes to, Yki-Sd activity. Though consistent with <br /> also co-expressed, stronger rescue Eya signs neuronal              Hth working downstream mediators Yki-Sd complex, <br /> development (Elav) detected discs (4/20 discs;                    Hth shown complex Yki [14], Hth <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          7                                July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                  Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> could alternatively, also, work component Yki-Sd-Hth                  For pathway loss-of-function, sought compromise the <br /> complex.                                                                        upstream negative regulation Yki. However, type of <br />                                                                                 experiments, endogenous level Yki protein effectively limits <br /> The Hippo pathway affects development eye                                extent hyper-activation achieved loss a <br /> primordium                                                                      single upstream regulator (likely far levels achieved with <br />    The Yki transcription factor key component Hpo                   exogenous Yki). In order achieve highest possible level of <br /> signaling pathway, partners (Sd Hth) shown                endogenous Yki activation, chose investigate specific <br /> to contribute regulation pathway&#226;&#8364;&#8482;s proliferative              genetic backgrounds. Since negative regulation upstream <br /> anti-apoptotic targets retina progenitor cells eye disc           factors converges Yki separate branches <br /> [12&#226;&#8364;&#8220;15]. However, loss proliferation increased cell death              [9,29], reasoned stronger activation Yki be <br /> not appear cause type tissue transformation observed               achieved simultaneously impairing branches. This be <br />                                                                                 achieve ways: 1) down-regulating Wts, direct <br /> in yki-RNAi sd-RNAi backgrounds. In fact, disc- <br />                                                                                 negative regulator Yki branches converge, 2) <br /> wide (ey-Flip; Act.IC.Gal4) PE-restricted (pnr-Gal4) loss of <br />                                                                                 knocking-down upstream components branch, in <br /> proliferation induces PE-to-DP transformations, suppres- <br />                                                                                 mutant combination upstream factors fat (ft) expanded <br /> sion apoptosis yki-RNAi- sd-RNAi-expressing discs rescue <br />                                                                                 (ex) (ft ex double mutant) [30,31]. We pursued approaches <br /> the transformation phenotype (Fig. S8, shown). A possible <br />                                                                                 retina specification compromised in <br /> scenario Yki partners mediate proliferation and <br />                                                                                 genetic backgrounds. Expression Eya DP layer was <br /> survival context Hpo pathway, control <br />                                                                                 greatly reduced nearly absent, Hth expression was <br /> specification independently (through genetic pathway). <br />                                                                                 greatly expanded, ft ex double mutant wts-RNAi-expressing <br /> Alternatively, Hpo pathway components regulate activity <br />                                                                                 discs (Fig. 6B, C). <br /> of Yki complexes involved processes, differences in <br />                                                                                    More importantly, down-regulation Yki protein expressing <br /> Yki activity levels and/or Yki partners account the <br />                                                                                 yki-RNAi wts-RNAi background led reversal the <br /> different outputs. <br />                                                                                 block retina development. Discs co-expressing yki-RNAi and <br />    To investigate Hpo pathway involved                   wts-RNAi display yki-RNAi-like double-retina phenotype <br /> specification role played transcription factors,                    (Fig. 6D). In fact, wts-RNAi yki-RNAi discs similar to <br /> manipulated expression pathway components lie                       severe phenotype seen yki-RNAi experiments. <br /> upstream Yki, assessed development retina                    These findings consistent known role Wts a <br /> the L3 disc. Since components negative effect Yki                negative regulator Yki, strongly support model which <br /> activity, expected gain-of-function conditions upstream                   level down-regulation Yki upstream components of <br /> factors mimic loss-of- function yki phenotypes and, conversely,              pathway required DP normal development <br /> loss-of-function conditions mimic gain-of-Yki-function.                      retina primordium, despite requirement Yki DP- <br />    For pathway gain-of-function, over-expressed wt forms                 cells&#226;&#8364;&#8482; proliferation. <br /> of Hpo, Wts membrane-tethered (activated) form Mats <br /> [26-28]. Interestingly, over-expression negative regulators <br />                                                                                 Discussion <br /> of Yki mimicked effect yki-RNAi sd-RNAi reflected by <br /> the smaller disc size PE-to-DP transformation (compare                         The process regional specification eye-antennal disc of <br /> Fig. 6A 1C). The RDF Eya strongly expressed 2                     Drosophila involves number signaling pathways regulate <br /> mirror-image, columnar epithelial layers (Fig. 6A). These results               expression identity-defining transcriptional regulators, often <br /> support role Yki promoting PE formation                        referred selector factors. We Hippo <br /> consistent role Hpo pathway regulating Yki                    signaling pathway transcription factors Yki, Sd, Hth <br /> activity process.                                                       play critical roles regional specification disc. <br />  <br />  <br />  <br />  <br /> Figure 6. Changes Hpo pathway activity induce gain loss retina. A) ZX views discs over-expressing negative regulators Yki <br /> including Hpo (top), Wts (middle) Myr-Mats (bottom). Ectopic retina development (red) occurs presumptive PE (compare wt and <br /> transformed discs Fig. 1C). B, C) XY (B) ZX (C) views discs double mutant ex ft (left) expressing wts-RNAi (right) stained Eya (B) or <br /> Hth (C). Discs reduced Eya increased Hth expression. Discs form retinae based Elav staining (not shown). Loss Eya and <br /> increased Hth observed Yki over-expressing discs (Fig. 5A). This consistent increase nuclear Yki activity expected the <br /> absence upstream negative regulators. D) Co-expression yki-RNAi wts-RNAi expressing discs restores retina formation. Discs similar to <br /> the severe phenotype seen yki-RNAi-only experiments, apparent PE. The eye-antennal discs overall small eye disc is <br /> shaped like hollow ball Eya-positive Elav-positive cells. Top panel shows XY view entire eye discs maximum projection all <br /> confocal planes (max); panels XY, ZX ZY views single scanned, confocal plane. <br /> doi:10.1371/journal.pone.0022278.g006 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           8                                July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                             Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> Specifically, factors required establishment        cells begin emerge late L1-early L2 stage become <br /> the peripodial cell layer eye disc. Yki function required       phenotypically mutant somewhat later, sufficient degradation <br /> the PE maintain tissue identity appears works concert           gene products (mRNA/protein) taken place. This <br /> with Sd Hth. Moreover, negative regulation Yki                characteristic renders clonal analysis problematic studying <br /> the Hippo/Warts genetic pathway required DP                     phenotypes gene/protein activity prior mid late L2 <br /> modulate Yki activity promotes proliferation                stage. On contrary, ey-Flip Act.IC.Gal4 UAS-RNAi discs, <br /> survival retina progenitor cells interfering           gene down-regulation begins embryonic stage because <br /> specification. The specification role Yki partners         RNAi transgene expression initiates constitutive in <br /> eye epithelium central proper disc development, occurs          embryonic eye-antennal disc. Consistent scenario, we <br /> early stages regional specification disc appears       identified early mid-late L2 interval critical <br /> be distinct general role proliferation survival.        period Sd function preventing PE-to-DP transformation <br />                                                                              phenotype. Thus, traditional clonal analysis likely be <br /> Multiple functions Yki Sd eye disc                             inadequate assessing gene function requirements process. <br />    Yki Sd shown form complex regulate                 While provides explanation observations, does not <br /> the anti-apoptotic gene Diap1 wing eye discs                   resolve issue cell-autonomous versus non-cell-autonomous <br /> Drosophila [12,13]. In eye, Yki strongly required            mechanisms. Further testing needed method eliminate <br /> proliferation survival, Sd makes lesser contribution           gene function earlier, efficiently proliferation-independent <br /> to processes [12&#226;&#8364;&#8220;14].                                                  way developed answer question. <br />    Does Yki function complex Sd context of <br /> regional specification eye disc how? The evidence             Multiple functions Hth eye disc <br /> presented shows 1) RNAi-induced knock-down                     Thus far, major functions Hth uncovered in <br /> either gene induces essentially identical PE-to-DP transformations           DP cell layer: 1) Hth, Tsh, suppresses the <br /> and 2) sd-RNAi suppress gain-of-function, anti-retina           expression late RD genes eya so, slowing the <br /> effect Yki over-expression. These findings,             conversion early eye progenitors (Ey, Tsh Hth positive) into <br /> biochemical evidence protein-protein interactions [12,13,15],             mature (Ey, Tsh, Eya So positive) eye precursor cells, <br /> suggest Yki Sd control transcription            2) Hth Tsh Yki enhances proliferation by <br /> in complex PE specification.                                        inducing expression proliferation-promoting microRNA <br />    Surprisingly, comparison loss-of-function analyses mosaic         Bantam [14,17,23&#226;&#8364;&#8220;25,38]. The combined effect two <br /> discs versus RNAi-mediated disc-wide knock-downs uncovered               activities results generation abundant pool eye <br /> a discrepancy induced phenotypes. Specifically, loss-of-              progenitor cells. Protein-protein interactions factors <br /> function clones yki sd signs cell fate           documented vitro and/or vivo, leading the <br /> transformation. This case traditional flip-FRT loss-of-           proposal complexes (inclusive Hth/Tsh Hth/Yki/ <br /> function clones generated using mutant alleles [11,12,14]         Tsh) perform major functions [14]. <br /> for knock-down MARCM clones induced RNAi expression                          The data presented uncover critical function of <br /> (Fig. S4 work [13]). We believe differences          Hth eye disc. In PE cell layer, fated rise to <br /> reagents explain discrepancy, yki-RNAi             portions head cuticle, Hth prevents conversion tissue <br /> or sd-RNAi lines used disc-wide versus clonal analyses.          retina. As eye progenitors DP, anti-retina <br /> Hence, discrepancy depend different loss-of-            activity Hth involves suppression late-RDFs (Eya, So, Dac). <br /> function approaches employed. The approaches differ                      However, unlike role eye progenitors, entails down- <br /> significantly ways: 1) mutant tissue induced,           regulation early-RDFs Ey Tsh. Thus, Hth suppresses <br /> 2) timing loss-of-gene function.                                  late-RDFs, early-RDFs, early eye disc progenitors in <br />    The scenario indicate Yki-Sd complex                 conjunction Tsh [24], suppresses retina formation <br /> controls specification non-cell-autonomous fashion, that,          level early-RDFs PE process that <br /> in mosaic discs, nearby wt cells rescue gene-silenced              appears involve Yki Sd, Tsh (whose expression is <br /> tissue. In case, simplest model Yki induces                 normally absent PE). In agreement context <br /> expression secreted factor antagonizes retina identity. But        dependent role Hth, hth-RNAi reverses anti-retina effect of <br /> other models possible. In support hypothesis,           Yki over-expression, part, relieving Yki suppression of <br /> secreted ligands Wingless (Wg), Egfr, Jak/Stat signaling          early-RDF Ey (Fig. S7), and, shown Bessa Casares <br /> pathway (Wg, Vein Unpaired respectively) shown                 [20], expression exogenous Tsh PE cell layer leads to <br /> to function downstream Yki different contexts providing                formation ectopic retina. <br /> evidence non-cell-autonomous Hpo-Yki activity [32-37]. These                 Lastly, shown Hth expression genetic, <br /> pathways extensively studies fly eye Wg            positive control Yki Sd, yki-RNAi sd- <br /> signaling, ability suppress retina formation          RNAi suppress gain-of-function activity Hth. These <br /> DP, fits profile likely candidate. However, down-regulation           findings, reported association Hth Yki <br /> of Wg signaling, using temperature sensitive allele Wgts, does           proteins S2 cells regulatory region Bantam [14], <br /> not result development ectopic retina PE             suggest potentially complex relationship Hth and <br /> (not shown). Thus, non-autonomous component involved              Yki development PE. The observation Yki and <br /> the establishment PE, analysis needed identify          Sd control Hth expression does preclude possibility that <br /> potential factors.                                                           Hth protein functions complex factors. <br />    The second scenario indicate requirement gene                 Indeed, related example offered RDF Eya, first <br /> function earlier developmental stage adequately            induces partners RDF Sine oculis regulate <br /> probed mosaic approach. Generation mosaic clones              retina development [3,39,40]. We propose Hth is <br /> by FRT-mediated chromosomal-exchange dependent                     direct indirect targets Yki/Sd complex PE. <br /> start cell proliferation late L1. Hence, genotypically mutant          Thereafter, association Hth modify Yki/Sd activity <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                              July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                  Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> eventually resulting transcriptional silencing critical RDF <br /> genes, Tsh. <br />  <br /> Yki proliferation versus specification <br />    As summarized introduction, Hippo/Yki signaling <br /> pathway general regulator cell proliferation survival in <br /> metazoans (recently reviewed [29]). In handful cases, the <br /> Hippo pathway shown regulate processes than <br /> proliferation cell death, maintenance the <br /> undifferentiated state progenitor cell types neural tube <br /> and gut [41,42] and, cases, aspects neuronal <br /> differentiation. In one, pathway controls choice opsin <br /> gene expressed R8 cells adult fly eye [43]; the <br /> other, required maintenance dendritic harbors of <br /> body-wall sensory neurons Drosophila larva [44]. While <br /> transcriptional regulation Yki thought play role these <br /> non-proliferation-related processes, difficult draw parallels <br /> between cases eye disc. <br />    Two recent reports implicate Yki processes related to <br /> tissue specification. In case, transient, expanding burst Yki <br /> activity rows cells border established <br /> wing field results marginal expansion wing primordium, <br /> ultimately ensuring proper wing size [45]. In second example, <br /> the YAP/Yki TEAD4/Sd proteins specify trophoecto- <br /> derm distinct inner cell mass (ICM) mammalian <br /> embryos [46]. It thought circumferential cell-cell <br /> contacts experienced ICM cells triggers Hpo activity and <br /> cytoplasmic retention YAP, extensive cell-cell <br /> contacts experienced outer cells not, allowing nuclear <br /> accumulation YAP [46,47]. <br />    Based similarities example, tempting to <br /> hypothesize extensive cell-cell contacts squamous               Figure 7. Yki, Sd Hth functions different regions the <br /> cell layer, columnar one, promote                       L2 eye disc. In model, different combinations co-factors <br /> efficient nuclear localization Yki PE, DP cells.           associate Yki bring different outcomes distinct regions <br />                                                                                  L2 eye disc epithelium. A Yki/Hth/Tsh complex ensures the <br /> At moderate activity levels DP, Yki promote                    maintenance expansion retina progenitors pool DP cell <br /> proliferation survival interfering retina specifi-              layer promoting proliferation suppressing apoptosis; whereas, in <br /> cation, higher activity levels PE, Yki                PE, Hth induced by, work concert with, Yki <br /> able promote cell proliferation survival                Sd control regional specification suppressing retina identity <br /> PE-identity suppressing retina formation. This scenario                 promoting development PE derivatives (this work). The induction <br /> be consistent observed need down-regulation Yki                  Hth Yki/Sd direct indirect (dashed arrow), the <br />                                                                                  negative regulation genes RD network have <br /> activity Warts DP (Fig. 6) stronger expression              be, part, indirect based non-cell-autonomous <br /> Diap1-2-lacZ PE (Fig. 3B). However, observation              features mutant clones. Whether Yki, Tsh and/or other <br /> two cell layers differ significantly availability factors        partners, controls proliferation survival developmentally <br /> in Yki-based complexes [14,20] suggests &#226;&#8364;&#732;Yki-complex                        advanced &#226;&#8364;&#732;Eya-positive + Hth-negative&#226;&#8364;&#8482; retina progenitors (named here <br /> composition&#226;&#8364;&#8482; plays critical role PE versus DP/retina                    &#226;&#8364;&#732;&#226;&#8364;&#732;eye precursor&#226;&#8364;&#8482;&#226;&#8364;&#8482; cells) likely, based small size yki-RNAi <br /> distinction developing fly eye.                                           expressing clones region disc, requires further <br />                                                                                  testing. Though shown diagram simplicity, Ey also <br />    Yki association specific co-factors would, therefore, modify             expressed PE Eya expressed restricted <br /> the output Hpo-Yki pathway bring different                       posterior-lateral portion PE, L2 eye disc; shown, Tsh is <br /> outcomes distinct regions eye epithelium. A Yki/Hth/Tsh                detected DP cell layer. <br /> complex ensure maintenance expansion retina                     doi:10.1371/journal.pone.0022278.g007 <br /> progenitors pool [14], Yki/Sd, possibly Hth, <br /> complex contribute regional specification eye                contrary, PE, behaves critical co-factor control <br /> disc ensuring formation PE-derived head structures (this                   tissue identity. Unlike Sd, Tsh expression shown be <br /> work) (Fig. 7). That Yki/YAP-based complexes include                       restricted specifically DP cell layer [20]. Thus, Yki must <br /> variety different co-factors supported recent                     perform PE-promoting tasks transcriptional complexes that <br /> examples, including mentioned role Yki, Hth                     include Tsh. Interestingly, misexpression Tsh the <br /> Tsh promoting eye progenitors&#226;&#8364;&#8482; proliferation fly [14],             PE shown induce retina development tissue <br /> its interaction IRS1 [48] promote proliferation neural                [20]. Whether Tsh does so, part, diverting Yki <br /> precursors Smad1 [49] enhance BMP-mediated                            activity away &#226;&#8364;&#732;anti-retina-fate&#226;&#8364;&#8482; functions remains be <br /> suppression neuronal differentiation embryonic stem cells                  determined. Hth, present broadly cells layer the <br /> in mouse.                                                                    L2 stage, play general role contributing Yki&#226;&#8364;&#8482;s <br />    In L2 eye disc, Sd protein believed available                function proliferation tissue specification. <br /> throughout, does appear contribute greatly Yki-                       One possible scenario tissue specification functions of <br /> induced proliferation cell layer (Fig. S4; [14]). On               Yki PE critically dependent association Sd, <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           10                                July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> and availability Tsh specifically DP interferes             anti-E-cad (1:20)- sources: rabbit anti-b-galactosidase <br /> with ability Sd Yki associate work fate-          (1:1000, Cappel), goat anti-Hth (1:200, Santa Cruz), rabbit anti- <br /> determining tasks, preventing interference retina             Ey (1:2000) [59], rabbit anti-Yki [31]. Cy2, Cy3, Cy5 secondary <br /> formation.                                                                     antibodies conjugated (Jackson Immuno Research Laboratories) <br />    As shown paucity examples, early              used 1:200. Images obtained Leica DM5500Q <br /> days deciphering Yki partners regulate cell fate.               confocal processed Adobe-Photoshop. <br /> Nonetheless, role apparently separate contribution <br /> to cell proliferation survival, likely involves number            Supporting Information <br /> distinct molecular mechanisms including co-factor specificity and <br /> differing levels Yki activity [14,45,46, work].                        Figure S1 Disc development, expression molecular <br />                                                                                markers Gal4. A) Detailed schematic developing L3 eye <br />                                                                                disc expression patterns molecular markers used identify <br /> Materials Methods                                                          eye disc cell types work. B) XY ZX views GFP <br /> Drosophila stocks                                                              expression developing eye discs expressing UAS-GFP the <br />    For targeted expression RNAi protein, used drivers             control &#226;&#8364;&#732;&#226;&#8364;&#732;ey-Flip; Actin.IC.Gal4&#226;&#8364;&#8482;&#226;&#8364;&#8482; drivers. The ey-Flip; <br /> pnr-Gal4, ey-Gal4 flip-out UAS/Gal4 [50], ey-Flip        Actin.IC.Gal4 combination drives expression broadly the <br /> Actin5C.y+.Gal4 Actin5C.CD2.Gal4, referred Act.Interrup-              eye-antennal disc; cells eye disc cells <br /> tion-Cassette.Gal4 Act.IC.Gal4 [3], [51], [52]. TRiP-Harvard                antenna express UAS-GFP transgene. The UAS-RNAi <br /> Medical School (NIH/NIGMS R01-GM084947) transgenic RNAi                        transgenes co-expressed UAS-dicer 2 (to enhance the <br /> stocks: UAS-sdRNAi (JF02514), UAS-ykiRNAi (HMS00041), UAS-                     efficiency RNAi) UAS-GFP (to mark expressing <br /> hthRNAi (HMS01112); UAS-CycERNAi (JF02473), UAS-mycRNAi                        tissue) control ey-Flip; Actin.IC.Gal4 drivers all <br /> (JF01761), UAS-NotchRNAi (JF01891) [53,54]. VDRC RNAi                      experiments unless stated (exact genotypes discs <br /> stocks: UAS-hthRNAi (12763) [55], UAS-wtsRNAi (106174) [37]. Jiang&#226;&#8364;&#8482;s           shown regular supplemental figures listed Table S1). <br /> RNAis: UAS-sdRNAiN, UAS-sdRNAi N+C, UAS-ykiRNAiN, UAS-ykiR-                    (PDF) <br /> NAiC [13]. Other lines: ey-Gal4 [56], UAS-hpo [26], UAS-yki [11], UAS-         Figure S2 yki-RNAi hth-RNAi efficiently down- <br /> Myc-wts [27], UAS-Myr-mats [28], UAS-hthGFP [57], UAS-dicer2                   regulate expression Yki Hth respectively. A) <br /> (VDRC # v60009), UAS-P35, UAS-GFP, dpp-lacz, tsh-lacz (tsh1), tub-             L3 disc containing clones yki-RNAi expressing cells (marked by <br /> Gal80ts (Bloomington), Diap1-2-lacZ [12], exG1 ftGTV/ Cyo GFP              GFP, green) stained anti-Yki Ab; XY shows DP cell layer. <br /> exG1 ftfd/ Cyo [30].                                                           The Yki protein efficiently down-regulated yki-RNAi <br />                                                                                expressing cells, continues robustly expressed both <br /> Genetics crosses                                                               cell layers regions disc. B) L3 discs stained anti- <br />    ey-Flip Act.IC.Gal4 drivers used experiments unless             Hth Ab. Hth strongly expressed wt (left) Yki over- <br /> explicitly stated. See Table S1 detailed genotypes                     expressing (middle) discs, detected presence of <br /> temperatures. The yki-RNAi-expressing clones generated                    hthRNAi. <br /> by ey-Flip MARCM method [58]. Gal80ts temperature shift                    (TIF) <br /> experiments: Sd protein expression altered shifting                     Figure S3 Multiple RNAi lines induce consistent mutant <br /> temperature 18uC (Sd expressed; Gal80ON R RNAiOFF)                     phenotypes. A, B) transgenic yki-RNAi (A) sd-RNAi (B) <br /> and 29uC (Sd not-expressed; Gal80OFF R RNAiON). Control                        lines Jiang&#226;&#8364;&#8482;s lab induce strong PE-DP transformation <br /> progeny (e.g. maintained 18uC throughout) developed                    phenotypes expressed eye disc, does TRiP hth- <br /> normal adults. Development proceeds twice fast                 RNAi line HMS01112 (C). These lines shown in <br /> 29uC 18uC. Embryos collected cages 1 hr               Fig. 1C directed different regions respective <br /> room temperature, plates placed 18uC 29uC                   mRNA targets (see Table S2). Hence, effect the <br /> needed time shift. Dissections carried L2             down-regulation intended mRNA targets. In addition, <br /> and L3 stage needed, head phenotypes scored adult               Gal4 line, ey-Gal4, similar expression induced <br /> stage. Adult head phenotypes (I&#226;&#8364;&#8220;V) correlate varying                 transformation phenotype (not shown), indicating effect is <br /> degrees PE-to-DP transformation developing disc                 restricted binary Flip-out Gal4 driver. <br /> identified combining analysis adult larval stages            (TIF) <br /> selected time points shift 18uC 29uC. At early-mid <br /> L2 (96 hour AEL 18uC) induced high percentage adults                Figure S4 Loss sd yki function clones does not <br /> protruding eyes (III), severely deformed heads (IV) headless                induce ectopic retina formation PE. ZX (top) and <br /> adults eyes fused thorax (V) (Figs. 3C, S6C). Analysis          XY (bottom) views L3 discs clones expressing GFP (left), <br /> this sample late L3 stage showed eye discs complete             GFP + sd-RNAi (middle), GFP + yki-RNAi (right). No PE-to- <br /> nearly complete PE-to-DP transformation (Fig. S6). On                      DP/retina transformation observed yki-RNAi- sd- <br /> contrary, shifts developing larvae 144 h AEL (early L3 stage)            RNAi-expressing clones; yki-RNAi-expressing clones consis- <br /> induced phenotypes not-associated PE-to-DP transformation                 tently smaller wt clones, reflecting dramatic effect loss <br /> (I, II) (Figs. 3C, S6C). The shift 120 h AEL (mid-late L2 stage)            Yki proliferation; sd-RNAi-expressing clones somewhat <br /> induced various phenotypes II IV (Figs. 3C, S6C),              smaller wt significantly larger yki-RNAi clones. Ten <br /> discs showed complete, partial, transformation (Fig. S6).           wt (3 discs), 32 sdRNAi (4 discs), 40 ykiRNAi (6 discs) clones <br />                                                                                PE cell layer individually scored the <br /> Immunohistology                                                                presence ectopic Eya expression XY, ZX ZY views. <br />                                                                                None detected. <br />    Antibody stainings discs ($12 scored, unless explicitely stated) <br />                                                                                (TIF) <br /> were standard protocol. Primary antibodies - DSHB <br /> (http://dshb.biology.uiowa.edu/): mouse anti-Eya (1:100), mouse                Figure S5  Photoreceptor neurons axonal projections in <br /> anti-Dac (1:200), mouse anti-DLG (1:100), rat anti-Elav (1:20), rat            wt transformed discs. XY views wt sd-RNAi discs <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         11                               July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                                      Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> stained 22C10 highlight axonal projections. The axons                                      conclude Sd function necessary promote PE <br /> emanating transformed-PE display overall organiza-                                     identity late L1 early L2. This result consistent with <br /> tion remarkably similar axonal projections normal                                  emergence Eya-positive ectopic retina domain the <br /> DP layer. Arrows mark nerve Bolwig&#226;&#8364;&#8482;s larval eye.                                     L2 stage shown Fig. 3A. <br /> This axonal bundle extends neuronal cell bodies located                                    (TIF) <br /> the larval head, close contact PE cell layer eye <br />                                                                                                    Figure S7 Co-expression hth-RNAi partially <br /> disc (basal side), brain optic stalk (see also <br />                                                                                                    reverses loss eye primordium induced Yki <br /> 22C10 panels Fig. 2C ZX views disc). <br />                                                                                                    over-expression. XY ZX views Yki hth-RNAi <br /> (TIF) <br />                                                                                                    expressing discs. Discs stained Ey, Eya Dac (red) as <br /> Figure S6      Temperature shift experiment correla-                                           marked panel. Insets ZX-projections Eya Dac <br /> tion adult disc phenotypes induced sd-RNAi                                               panels digitally-enhanced expression Dac Eya for <br /> expression. A&#226;&#8364;&#8220;D) Range phenotypes seen flies pharate                                     clarity. <br /> adults (A) temperature shift experiments, corresponding L3 disc                                 (TIF) <br /> abnormalities (B), penetrance percentage flies displaying <br />                                                                                                    Figure S8 Changes cell proliferation cell death do <br /> a specific phenotype (C, D). Discs signs PE-to-DP/retina <br />                                                                                                    alter regional identity disc. A) ZX views of <br /> transformation result type III, IV V adult fly phenotypes, <br />                                                                                                    RNAi-expressing discs stained Eya (red) E-cad (green). <br /> whereas discs evidence transformation produce <br />                                                                                                    RNAi knock-down G1.S transition CycE, growth <br /> type I (normal) type II flies. Total number flies scored (C) <br />                                                                                                    regulator Myc, Notch, critical regulator proliferation the <br /> and (D) marked time point. Adult head phenotypes <br />                                                                                                    eye primordium, produced discs overall smaller size <br /> (III&#226;&#8364;&#8220;V) correlate varying degrees PE-to-DP transfor- <br />                                                                                                    sd-RNAi expressing discs. Nonetheless, overall disc mor- <br /> mation developing disc identified analysis adult <br />                                                                                                    phology undisturbed thin, squamous PE cell layer <br /> and larval stages selected time points. The shift 18uC to <br />                                                                                                    devoid signs retina formation Eya-positive, <br /> 29uC early-mid L2 (96 hour AEL 18uC) induced mostly <br />                                                                                                    columnar DP cell layer. B) Suppression apoptosis co- <br /> adults protruding eyes (III), severely deformed heads (IV) or <br />                                                                                                    expression baculovirus anti-apoptosis factor P35 does not <br /> headless adults eyes fused thorax (V) (panel C). Analysis <br />                                                                                                    suppress transformation phenotype yki-RNAi discs. Except <br /> of sample late L3 stage shows eye discs complete or <br />                                                                                                    decrease activated-Caspase3 positive cells, yki-RNAi (top) <br /> nearly complete transformations (panel B, 96 hr top). On the <br />                                                                                                    yki-RNAi+P35 (bottom) discs nearly indistinguishable. <br /> contrary, shifts 144 h AEL (early L3 stage, panel B, bottom) <br />                                                                                                    (TIF) <br /> induced phenotypes not-associated transformation of <br /> the PE (I, II) (panel C). The shift 120 h AEL (mid-late L2 stage,                               Table S1     List genotypes analyzed related figures. <br /> panel B, middle) induced various phenotypes II IV (panel                                   (DOC) <br /> C), discs complete, partial, transforma- <br />                                                                                                    Table S2 RNAi lines used work. <br /> tions (panel B). C, D) Using approach, identified time <br />                                                                                                    (DOC) <br /> between L1/L2 molt mid-late L2 phenocritical <br /> period Sd gene function. Panel C shows transformation-                                    Table S3 Relative strength gene silencing. <br /> related phenotypes (III, IV, V) seen high penetrance                                        (DOC) <br /> whenever gene silencing (shift up) began prior mid L2 <br /> (108 hr). At mid-late (120 hr) late L2 (132 hr), drastic drop                                Acknowledgments <br /> in transformation occurs, ,50% ,15% <br />                                                                                                    Many thanks M. Zuber A. Viczian helpful comments the <br /> (See Fig. 3D); later time points transformation-related <br />                                                                                                    manuscript, members Pignoni, Zuber Viczian labs for <br /> phenotypes. Thus, inactivation endogenous Sd function                                       stimulating discussions, A. Ferguson, L. Presutto, D. Money for <br /> time embryogenesis 120 hr (18uC), later,                                       assistance genetic crosses manuscript preparation. We thank the <br /> compromises tissue identity PE indicating protein                                      TRiP Harvard Medical School (NIH/NIGMS R01-GM084947) for <br /> activity required ,mid-late L2 promote PE identity.                                    providing transgenic RNAi lines used study, D.J. Pan, J. Jiang, <br /> Conversely, panel D shows restoration Sd function (by                                      K. Irvine, R. Mann, U. Walldorf, IDSHB, Bloomington stock <br /> shift-down) prior mid-late L1 (30 hr, 29uC) resulted minimal                                 center, VDRC antibodies and/or fly stocks. <br /> disruption PE formation (non-transformation phenotypes in <br /> $80%); Sd restoration L1/L2 molt (36 hr, 29u)                                    Author Contributions <br /> thereafter (in L2 L3) insufficient ensure proper PE                                     Conceived designed experiments: TZ QZ FP. Performed the <br /> development (the penetrance transformation phenotypes                                       experiments: TZ QZ. Analyzed data: TZ QZ FP. Wrote paper: FP <br /> increase ,50% early L2 100% late L2). Thus,                                       TZ QZ. <br />  <br /> References <br />  1. Baker NE (2007) Patterning signals proliferation Drosophila imaginal discs.              6. Pan D (2007) Hippo signaling organ size control. Genes Development <br />     Current Opinion Genetics Development 17(4): 287&#226;&#8364;&#8220;293.                                        21(8): 886&#226;&#8364;&#8220;897. <br />  2. Chanut F, Heberlein U (1997) Role decapentaplegic initiation progression              7. Harvey K, Tapon N (2007) The Salvador-Warts-Hippo pathway - an <br />     morphogenetic furrow developing Drosophila retina. Development                       emerging tumour-suppressor network. Nature Reviews Cancer 7(3): 182&#226;&#8364;&#8220; <br />     124(2): 559&#226;&#8364;&#8220;567.                                                                                   191. <br />  3. Pignoni F, Zipursky SL (1997) Induction Drosophila eye development                        8. Reddy BVVG, Irvine KD (2008) The Fat Warts signaling pathways: New <br />     decapentaplegic. Development 124(2): 271&#226;&#8364;&#8220;278.                                                      insights regulation, mechanism conservation. Development <br />  4. Chen R, Halder G, Zhang Z, Mardon G (1999) Signaling TGF-b homolog                          135(17): 2827&#226;&#8364;&#8220;2838. <br />     decapentaplegic functions reiteratively network genes controlling retinal         9. Badouel C, Garg A, McNeill H (2009) Herding Hippos: Regulating growth in <br />     cell fate determination Drosophila. Development 126(5): 935&#226;&#8364;&#8220;943.                                flies man. Curr Opin Cell Biol 21(6): 837&#226;&#8364;&#8220;843. <br />  5. Curtiss J, Mlodzik M (2000) Morphogenetic furrow initiation progression                    10. Kango-Singh M, Singh A (2009) Regulation organ size: Insights the <br />     eye development Drosophila: The roles decapentaplegic, hedgehog eyes              Drosophila Hippo signaling pathway. Developmental Dynamics 238(7): <br />     absent. Development 127(6): 1325&#226;&#8364;&#8220;1336.                                                             1627&#226;&#8364;&#8220;1637. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                            12                                 July 2011 | Volume 6 | Issue 7 | e22278 <br />                                                                                                                                          Hpo/Yki Pathway Tissue Specification <br />  <br />  <br /> 11. Huang J, Wu S, Barrera J, Matthews K, Pan D (2005) The Hippo signaling                       36. Shaw RL, Kohlmaier A, Polesello C, Veelken C, Edgar BA, et al. (2010) The <br />     pathway coordinately regulates cell proliferation apoptosis inactivating                  Hippo pathway regulates intestinal stem cell proliferation Drosophila adult <br />     Yorkie, Drosophila homolog YAP. Cell 122(3): 421&#226;&#8364;&#8220;434.                                     midgut regeneration. Development 137(24): 4147&#226;&#8364;&#8220;4158. <br /> 12. Wu S, Liu Y, Zheng Y, Dong J, Pan D (2008) The TEAD/TEF family protein                       37. Ren F, Wang B, Yue T, Yun E-, Ip YT, et al. (2010) Hippo signaling regulates <br />     Scalloped mediates transcriptional output Hippo growth-regulatory                         Drosophila intestine stem cell proliferation multiple pathways. Proc Natl <br />     pathway. Developmental Cell 14(3): 388&#226;&#8364;&#8220;398.                                                      Acad Sci U S A 107(49): 21064&#226;&#8364;&#8220;21069. <br /> 13. Zhang L, Ren F, Zhang Q, Chen Y, Wang B, et al. (2008) The TEAD/TEF                          38. Lopes CS, Casares F (2010) hth maintains pool eye progenitors its <br />     family transcription factor Scalloped mediates Hippo signaling organ size                  downregulation Dpp Hh couples retinal fate acquisition cell cycle <br />     control. Developmental Cell 14(3): 377&#226;&#8364;&#8220;387.                                                      exit. Dev Biol 339(1): 78&#226;&#8364;&#8220;88. <br /> 14. Peng HW, Slattery M, Mann RS (2009) Transcription factor choice                       39. Bonini NM, Leiserson WM, Benzer S (1993) The eyes absent gene: Genetic <br />     Hippo signaling pathway: Homothorax Yorkie regulation microRNA                        control <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> differentiation developing Drosophila eye. Cell <br />     bantam progenitor domain Drosophila eye imaginal disc. Genes                   72(3): 379&#226;&#8364;&#8220;395. <br />     Development 23(19): 2307&#226;&#8364;&#8220;2319.                                                               40. Bonini NM, Bui QT, Gray-Board GL, Warrick JM (1997) The Drosophila eyes <br /> 15. Goulev Y, Fauny JD, Gonzalez-Marti B, Flagiello D, Silber J, et al. (2008)                       absent gene directs ectopic eye formation pathway conserved flies <br />     SCALLOPED interacts YORKIE, nuclear effector Hippo                               vertebrates. Development 124(23): 4819&#226;&#8364;&#8220;4826. <br />     tumor-suppressor pathway Drosophila. Current Biology 18(6): 435&#226;&#8364;&#8220;441.                      41. Cao X, Pfaff SL, Gage FH (2008) YAP regulates neural progenitor cell number <br /> 16. Atkins M, Mardon G (2009) Signaling dimension: The peripodial                       TEA domain transcription factor. Genes Development 22(23): <br />     epithelium eye disc development. Developmental Dynamics 238(9):                               3320&#226;&#8364;&#8220;3334. <br />     2139&#226;&#8364;&#8220;2148.                                                                                   42. Camargo FD, Gokhale S, Johnnidis JB, Fu D, Bell GW, et al. (2007) YAP1 <br /> 17. Pappu KS, Mardon G (2004) Genetic control retinal specification                           increases organ size expands undifferentiated progenitor cells. Current <br />     determination Drosophila. Int J Dev Biol 48(8-9): 913&#226;&#8364;&#8220;924.                                    Biology 17(23): 2094. <br /> 18. Silver SJ, Rebay I (2005) Signaling circuitries development: Insights            43. Mikeladze-Dvali T, Wernet MF, Pistillo D, Mazzoni EO, Teleman AA, et al. <br />     retinal determination gene network. Development 132(1): 3&#226;&#8364;&#8220;13.                                    (2005) The growth regulators warts/lats melted interact bistable loop to <br /> 19. Singh A, Choi KW (2003) Initial state Drosophila eye dorsoventral                  specify opposite fates Drosophila R8 photoreceptors. Cell 122(5): 775&#226;&#8364;&#8220;787. <br />     specification equivalent ventral. Development 130(25): 6351&#226;&#8364;&#8220;6360.                      44. Emoto K, Parrish JZ, Jan LY, Jan YN (2006) The tumour suppressor Hippo acts <br /> 20. Bessa J, Casares F (2005) Restricted teashirt expression confers eye-specific                    NDR kinases dendritic tiling maintenance. Nature 443(7108): <br />     responsiveness Dpp Wg signals eye specification Drosophila.                     210&#226;&#8364;&#8220;213. <br />     Development 132(22): 5011&#226;&#8364;&#8220;5020.                                                              45. Zecca M, Struhl G (2010) A feed-forward circuit linking Wingless, Fat-Dachsous <br /> 21. Kenyon KL, Ranade SS, Curtiss J, Mlodzik M, Pignoni F (2003) Coordinating                        signaling, Warts-Hippo pathway Drosophila wing growth. PLoS <br />     proliferation tissue specification promote regional identity Drosophila            Biology 8(6). <br />     head. Developmental Cell 5(3): 403&#226;&#8364;&#8220;414.                                                      46. Nishioka N, Inoue KI, Adachi K, Kiyonari H, Ota M, et al. (2009) The Hippo <br />                                                                                                      signaling pathway components Lats Yap pattern Tead4 activity to <br /> 22. McGuire SE, Mao Z, Davis RL (2004) Spatiotemporal gene expression targeting <br />                                                                                                      distinguish mouse trophectoderm inner cell mass. Developmental Cell <br />     TARGET gene-switch systems Drosophila. Science&#226;&#8364;&#8482;s STKE <br />                                                                                                      16(3): 398&#226;&#8364;&#8220;410. <br />     [Electronic Resource] : Signal Transduction Knowledge Environment 2004(220). <br />                                                                                                  47. Zhao B, Wei X, Li W, Udan RS, Yang Q, et al. (2007) Inactivation YAP <br /> 23. Bessa J, Gebelein B, Pichaud F, Casares F, Mann RS (2002) Combinatorial <br />                                                                                                      oncoprotein Hippo pathway involved cell contact inhibition and <br />     control Drosophila eye development Eyeless, Homothorax, Teashirt. <br />                                                                                                      tissue growth control. Genes Development 21(21): 2747&#226;&#8364;&#8220;2761. <br />     Genes Development 16(18): 2415&#226;&#8364;&#8220;2427. <br />                                                                                                  48. Fernandez-L A, Northcott PA, Dalton J, Fraga C, Ellison D, et al. (2009) YAP1 <br /> 24. Pai CY, Kuo TS, Jaw TJ, Kurant E, Chen CT, et al. (1998) The Homothorax <br />                                                                                                      amplified up-regulated hedgehog-associated medulloblastomas and <br />     homeoprotein activates nuclear localization homeoprotein, <br />                                                                                                      mediates Sonic hedgehog-driven neural precursor proliferation. Genes and <br />     Extradenticle, suppresses eye development Drosophila. Genes and <br />                                                                                                      Development 23(23): 2729&#226;&#8364;&#8220;2741. <br />     Development 12(3): 435&#226;&#8364;&#8220;446. <br />                                                                                                  49. Alarco&#194;&#180;n C, Zaromytidou A-, Xi Q, Gao S, Yu J, et al. (2009) Nuclear CDKs <br /> 25. Pichaud F, Casares F (2000) homothorax iroquois-C genes required                 drive Smad transcriptional activation turnover BMP TGF-b <br />     establishment territories developing eye disc. Mech Dev 96(1):                     pathways. Cell 139(4): 757&#226;&#8364;&#8220;769. <br />     15&#226;&#8364;&#8220;25.                                                                                       50. Brand AH, Perrimon N (1993) Targeted gene expression means altering <br /> 26. Wu S, Huang J, Dong J, Pan D (2003) hippo encodes ste-20 family protein                        cell fates generating dominant phenotypes. Development 118(2): 401&#226;&#8364;&#8220;415. <br />     kinase restricts cell proliferation promotes apoptosis conjunction          51. Ito K, Awano W, Suzuki K, Hiromi Y, Yamamoto D (1997) The Drosophila <br />     salvador warts. Cell 114(4): 445&#226;&#8364;&#8220;456.                                                        mushroom body quadruple structure clonal units contains a <br /> 27. Lai ZC, Wei X, Shimizu T, Ramos E, Rohrbaugh M, et al. (2005) Control                         virtually identical set neurones glial cells. Development 124(4): 761&#226;&#8364;&#8220;771. <br />     cell proliferation apoptosis mob tumor suppressor, Mats. Cell 120(5):              52. Newsome TP, A    &#203;&#353; sling B, Dickson BJ (2000) Analysis Drosophila photoreceptor <br />     675&#226;&#8364;&#8220;685.                                                                                         axon guidance eye-specific mosaics. Development 127(4):  <br /> </body></html>