<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Dec. 1990, p. 6123-6131                                                                   Vol. 10, No. 12 <br /> 0270-7306/90/126123-09$02.00/0 <br /> Copyright C 1990, American Society Microbiology <br />  <br />  <br />                  A Suppressor RNA Polymerase II <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>Mutation</span> of <br />                     Saccharomyces cerevisiae Encodes Subunit <br />                      Common RNA Polymerases I, II, III <br />                      JACQUES ARCHAMBAULT, KEITH T. SCHAPPERT, AND JAMES D. FRIESEN* <br />                        Department Genetics, The Hospital Sick Children, 555 University Avenue, <br />                               Department Medical Genetics, University Toronto, <br />                                             Toronto, Ontario M5G 1X8, Canada <br />                                           Received 29 May 1990/Accepted 3 September 1990 <br />  <br />               RNA polymerase H (RNAPII) complex multisubunit enzyme responsible synthesis pre-mRNA <br />            eucaryotes. The enzyme large subunits associated smaller polypeptides, <br />            common RNA polymerase species. We initiated genetic analysis RNAPII <br />            introducing mutations RP021, gene encoding largest subunit RNAPII Saccharomyces <br />            cerevisiae. We used yeast genomic library isolate plasmids suppress temperature-sensitive <br />            mutation RP021 (rpo214), goal identifying gene products interact largest subunit <br />            RNAPII. We increased expression wild-type RP026, single-copy, essential gene encoding a <br />            155-amino-acid subunit common RNAPI, RNAPII, RNAPIII, suppressed rpo2l4 temperature- <br />            sensitive mutation. Mutations constructed vitro resulted single amino acid changes the <br />            carboxy-terminal portion RP026 gene product. One temperature-sensitive mutation, some <br />            mutations did generate <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, lethal combination rpo214. These <br />            results support idea RP026 RP021 gene products interact. <br />  <br />   The high degree conservation structure                   catalytic subunit bovine type 2A protein phosphatase <br /> function transcriptional apparatus eucaryotes                postulated modulate activity RNAPII <br /> well established. For example, shown                     altering level phosphorylation largest subunit <br /> transcription factors higher eucaryotes, TATA              enzyme. <br /> sequence-binding protein (TFIID) CCAAT-binding pro-                       Suppressors temperature-sensitive mutation within <br /> teins functional homologs yeast Saccharomyces                 largest subunit RNAPII isolated. Ten <br /> cerevisiae (12-14). In addition, extensive similarity exists              independent, extragenic suppressors belonging single <br /> among forms RNA polymerase (RNAP)                     complementation group shown alleles the <br /> variety organisms, including single procaryotic            KEX2 gene, encodes protease required the <br /> enzyme (1, 2, 6, 10, 19, 46). In S. cerevisiae, RNAPII is <br />                                                                           processing alpha-factor killer-toxin precursors (31). <br /> composed 10 subunits. The largest subunits (the                    <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> study, clear KEX2 influence RNAPII, <br /> products RP021 [24, 48] RP022 [48] genes,                 precise mechanism remains obscure. <br /> called RPBJ RPB2) homologous bacterial &apos; <br /> and proteins, respectively (1, 46). In addition, RNAPII                    As continuing series genetic studies, have <br /> contains smaller polypeptides,               isolated linker-insertion mutations gene encoding <br /> subunits RNAPI RNAPIII (reviewed reference 43).                 largest subunit RNAPII (RP021; J. Archambault et <br />    The well-developed genetics S. cerevisiae makes               al., unpublished data). One allele particular, rpo214, <br /> attractive organism study multiprotein complexes               causes rapid growth arrest cells shifted nonper- <br /> such RNAPII. The isolation mutations                     missive temperature (Archambault et al., unpublished data). <br /> various subunits yeast RNAPII begun help                Pleiotropic phenotypes associated mutation include <br /> establish individual roles transcription (2, 3, 22, 24,          partial requirement inositol slow-growth <br /> 34, 35, 41, 50) promises fruitful approach           phenotype permissive temperature. The inability <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> subunit genes isolated (39). A counterpoint                  grow medium lacking inositol documented for <br /> mutant isolation search suppressors (26),            mutations genes encoding subunits RNAPII <br /> already proven useful identifying new proteins involved             (RP021, RP022, RP024) (3, 50) accounted <br /> the processes transcription. Suppressors HIS4 tran-               reduction level transcription INOI <br /> scriptional defect caused absence GCN4,                     gene, encodes enzyme inositol-1-phosphate syn- <br /> BASJ, BAS2 transcription activators identified loci              thase. <br /> (SIT] 4) (3). Two these, SIT] SIT2, allelic                 As approach identification genetic loci af- <br /> the genes encoding largest subunits RNAPII.                    fecting transcription RNAPII gain further <br /> SIT3, mutant form wild type (wt) high              understanding defect associated rpo214 <br /> copy number, suppress HIS4 defect absence of <br /> the TATA sequence. SIT4 encodes protein homologous                   temperature-sensitive mutation, isolated plasmids <br />                                                                           yeast genomic library suppress rpo214 <br />                                                                           temperature-sensitive defect. One suppressor plasmid was <br />                                                                           shown carry wt RP026 gene, encodes subunit <br />   * <br />       Corresponding author.                                               common RNAPI, -II, -III. <br />  <br />                                                                    6123 <br />  6124      ARCHAMBAULT ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />                         TABLE 1. Strains used                                 GAL] promoter (27, 47) yeast integrative plasmid <br />  Strain&quot;                           Relevant genotype                          carrying URA3 gene. Plasmid pRPO26cDNA (2,um <br />                                                                               TRPJ) isolated cDNA library described by <br /> JAY212b .......... MATa rpo214                                                McKnight McConaughy (32). This library con- <br /> JAY435 .......... MATa rpo214(pRPO26) <br /> JAY430 .......... MATa rpo214(pRPO26dl)                                       structed cloning yeast cDNAs ADCJ pro- <br /> JAY429 .......... MA Ta rpo214(pRPO26d2)                                      moter CYCI terminator yeast episomal vector <br /> JAY431 .......... MATa rpo214(pRPO26d3)                                       (2,um TRPI ). A yeast integrative plasmid, pJAY86, carrying <br /> JAY427 .......... MATa rpo214(pRPO26d4)                                       deletion allele RP026 (ARP026) constructed the <br /> JAY432 .......... MATa rpo214(pRPO26d5)                                       isolation RP026 gap repair excision. Plasmid <br /> JAY433 .......... MATa rpo214(pJAY83)                                         pJAY86 (URA3 ARP026) carries internal deletion in <br /> JAY436 .......... MATa(pRPO26)                                                RP026 spans Bsu36I sites nucleotides 230 and <br /> JAY444 .......... MA Ta RP026:: LEU2(pRPO26)                                  1858. The 5&apos; 3&apos; boundaries ARP026 allele at <br /> JAY446 .......... MA Ta RP026:: LEU2(pRPO26)(pRPO26cDNA)                      nucleotides 1 2131, respectively. <br /> JAY467 .......... MA Ta rpo214(pRPO26cDNA) <br /> JAY472 .......... MATa rpo214 RP026:: pJAY97                                     Characteristics rpo2-4 allele used suppressor <br /> JAY476 .......... MATa RP026:: pJAY97                                         isolation. The rpo214 allele isolated collection of <br /> JAY490 .............. MATa rpo214 RP026:: pJAY97(prpo26-LI111)                mutant RP021 alleles generated vitro linker insertion <br /> JAY491 .......... MATa rpo214 RP026:: pJAY97(prpo26-LR111)                    mutagenesis (Archambault et al., unpublished data). The site <br /> JAY492 ......... MATa rpo214 RP026:: pJAY97(prpo26-LI118)                     mutation determined DNA sequencing. The <br /> JAY493 .......... MATa rpo214 RP026:: pJAY97(prpo26-LK118)                    rpo214 mutation result insertion of <br /> JAY494 ......... MATa rpo214 RP026:: pJAY97(prpo26-LR118)                     12-bp XhoI linker (5&apos;-CTCGAGCTCGAG-3&apos;) HaeIII <br /> JAY495 .......... MATa rpo214 RP026:: pJAY97(prpo26-LI125)                    site position 3174 RP021 sequence (1). This <br /> JAY496 ......... MATa rpo214 RP026:: pJAY97(prpo26-LK125)                     mutation results insertion amino acids, Leu Glu <br /> JAY497 ......... MATa rpo2-4 RP026:: pJAY97(prpo26-LI138) <br /> JAY498 .......... MATa rpo214 RP026:: pJAY97(prpo26-PA117)                    Leu Glu, amino acids 954 955 RP021 <br /> JAY499 ......... MATa rpo214 RP026:: pJAY97(prpo26-PA131)                     gene product. Yeast cells carrying rpo214 mutation are <br /> JAY500 ......... MATa rpo214 RP026:: pJAY97(prpo26-EK144)                     temperature sensitive growth undergo round of <br /> JAY501 ......... MATa rpo214 RP026:: pJAY97(pRPO26HA)                         cell division shift nonpermissive temperature <br /> JAY502 .......... MATa rpo214 RP026:: pJAY97(pFL39)                           (37&#194;&#176;C). A slow-growth phenotype partial inositol <br /> JAY503 .........    MATTa RP026:: pJAY97(prpo26-LI111)                        requirement permissive temperature (30&#194;&#176;C) also <br /> JAY504 .........    MATa RP026:: pJAY97(prpo26-LR111)                         associated rpo214 mutation. <br /> JAY505 ..........   MATa RP026:: pJAY97(prpo26-L1118)                            Construction yeast genomic library. To construct the <br /> JAY506 .........    MATa RP026:: pJAY97(prpo26-LK118)                         plasmid library used isolation RP026, DNA from <br /> JAY507 ............ MATa RP026:: pJAY97(prpo26-LR118)                         yeast strain JAY357 partially digested restric- <br /> JAY508 .            MATx RP026:: pJAY97(prpo26-LI125) <br /> JAY509 .            MATTa RP026:: pJAY97(prpo26-LK125)                        tion endonuclease Sau3A, size fractionated sucrose <br /> JAY510 .            MATt RP026:: pJAY97(prpo26-LI138)                         gradient, cloned BamHI site plasmid pINT2 <br /> JAY511 .            MATot RP026:: pJAY97(prpo26-PA117)                        (URA3 CEN3 ARSI) (38); 40,000 independent bacterial <br /> JAY512 .            MATa RP026:: pJAY97(prpo26-PA131)                         colonies obtained. More 80% clones carry <br /> JAY513 .            MATot RP026:: pJAY97(prpo26-EK144)                        yeast DNA inserts ranging size 5 15 kb. Yeast <br /> JAY514.             MA T RP026:: pJAY97(pRPO26HA)                             strain JAY357 fast-growing derivative JAY214 (MA Ta <br /> JAY515 .            MAT RP026:: pJAY97(pFL39)                                 ade2-1 cannl-100 his3-11,15 leu2-3,112 trpl-J ura3-1 PGAL- <br />   All JAY strains constructed laboratory. They     rpo214). The pGAL-rpo2J4 allele constructed by <br /> derived W303-1A (MATa canl-100 his3-11,15 leu2-3,112 trpl-I ura3-1       replacing endogenous RP021 gene rpo214 allele <br /> ade2-1) W303-1B (MATat canl-100 his3-11,15 leu2-3,112 trp1-1 ura3-1        control GAL1O promoter. The characteris- <br /> ade2-1). Both strains obtained R. Rothstein.                        tics yeast strains JAY357 JAY214 presented in <br />   b Characteristics presented (Archambault et al., unpub- <br /> lished data). JAY212 derivative W303-1A RP021 gene   (Archambault et al., unpublished <br /> been replaced rpo214.                                                      data). <br />                                                                                  Nucleotide sequence analysis. Both strands 2.0-kb <br />                                                                               KpnI-SpeI RP026 fragment sequenced chain <br />                MATERIALS AND METHODS                                          termination method (42), using combination deletions <br />    Strains media. The S. cerevisiae strains used listed               commercially available primers primers synthesized <br /> in Table 1. Growth media prepared described                        basis RP026 sequence. Single-stranded or <br /> Sherman et al. (44). When needed, 5-fluoro-orotic acid                        double-stranded DNA templates used. <br /> (5-FOA) added concentration 1 mg/ml (7). All                         DNA manipulation mutagenesis. Site-directed muta- <br /> plasmids propagated Escherichia coli JF1754 (22).                     genesis RP026 performed described Kunkel et <br /> Yeast transformation described Ito et al.                      al. (28). The RP026 mutant alleles created sub- <br /> (25).                                                                         cloned plasmid pFL39. Hydroxylamine mutagenesis of <br />    Plasmids. Plasmid pJAY83 derivative pINT2                       plasmid pRPO26HA performed vitro essentially as <br /> which ClaI site located near CEN3 filled                     described previously (11). All DNA manipulations were <br /> using Klenow fragment DNA polymerase I. Plasmid                        essentially described Maniatis et al. (30). <br /> pRPO26 contains 2.1-kb XbaI-ClaI RP026 fragment (the                           Isolation RP026 gap repair excision. Plasmid <br /> ClaI site derived pINT2 polylinker adjacent                    pJAY86 linearized unique Bsu36I site intro- <br /> the BamHI site used construct library) cloned                 duced yeast cells. Repair integration plasmid <br /> into unique XbaI ClaI sites pJAY83. Plasmid                        RP026 locus confirmed DNA blot analysis for <br /> pRPO26HA contains RP026 fragment cloned                        transformants (data shown). For indepen- <br /> pFL39 (TRPI CEN6 ARS; F. Lacroute). Plasmid                              dent transformants, plasmid integrated yeast <br /> pJAY97 contains portion RP026 gene (Sau3A                            chromosome recovered E. coli digestion the <br /> fragment nucleotides 1026 1451) cloned downstream                     genomic DNA SpeI previously described (37). <br />  VOL. 10, 1990                                                   SUPPRESSOR OF AN RNA POLYMERASE II MUTATION                            6125 <br />  <br />                                                                                                                       PHENOTYPES <br />   STRAIN                                                                            Sequences                         30CC     3kC <br />                                RP026 FRAGMENT <br />                                                                                     remaining <br /> JAY   435                                                                          1-2131 (pRP026)                      +          + <br />  <br />  <br /> JAY   430                                                                          1- 1889 (pRP026d 1)                  +          + <br />  <br />  <br /> JAY   429                                                                          1- 1023 (pRP026d2) <br /> JAY   431                                                                          1-717 (pRP026d3)                     +_ <br />  <br />  <br /> JAY   427                                                                          1-273 (pRP026d4) <br /> JAY   432                                                                          1448-2131 (pRP026d5)                 +_ <br />  <br />  <br />  <br /> JAY   433                                                                           vector (pJAY83) <br />   FIG. 1. Suppression rpo214 temperature-sensitive phenotype DNA subclones. Various deletions created 2.0-kb <br /> suppressing fragment subcloned plasmid pJAY83. These subclones introduced yeast strain JAY212 selecting for <br /> Ura+ transformants. The transformants tested growth 37&#194;&#176;C. The bar indicates region remaining subclone. The endpoints <br /> of deletions determined DNA sequencing. +, Growth; -, absence growth. <br />  <br />  <br />   Nucleotide sequence accession number. The nucleotide                 HindIII RP026 fragment probe. The position the <br /> sequence  reported appear EMBL, GenBank,              intron indicated Fig. 2. The location ORF is <br /> and DDBJ nucleotide sequence data bases accession                consistent results deletion analysis presented <br /> number M33924.                                                         above. The 5&apos; 3&apos; ends cDNAs also <br />                                                                        indicated Fig. 2. <br />                            RESULTS                                        The sequence ORF identical sequence of <br />                                                                        recently isolated gene encoding 17,908-Da subunit of <br />    Cloning suppressor rpo214. A yeast genomic library          yeast RNAPII, cloned basis partial <br /> was constructed (see Materials Methods) used <br /> to transform yeast strain carrying rpo214 allele                 amino acid sequence determination protein (49). We <br /> (JAY212) order isolate plasmids suppress            refer gene RP026, sixth-largest subunit <br /> rpo214 temperature-sensitive mutation. Of 40,000 transfor-             RNAPII. <br /> mants obtained, 100 acquired temperature-resistant                  Examination predicted amino acid sequence the <br /> phenotype 37&#194;&#176;C. Plasmid DNA 40 100 clones             RP026 gene product revealed distinct regions. A very <br /> was isolated E. coli retested ability suppress       acidic region located 40 amino acids the <br /> the temperature-sensitive defect strain JAY212 (rpo214).            amino terminus protein, basic region located <br /> Two 40 plasmids continued display suppression;               middle portion protein amino acids 61 <br /> presumably remainder original 40 temperature-               97. These regions predicted hydro- <br /> resistant clones revertants temperature-sensitive          philic (29). The carboxy-terminal region RP026 gene <br /> phenotype strain JAY212. Restriction digest analysis             product contains heptad repetition leucine residues. <br /> these plasmids HindlIl showed common yeast                     RP026 single-copy gene located chromosome XVI. <br /> DNA fragment 1.5 kb which, used probe,                    The copy number RP026 yeast genome was <br /> hybridized plasmids high-stringency condi-               determined DNA blot hybridization total genomic <br /> tions. A common 2.0-kb KpnI-SpeI fragment encompassing                 DNA (Fig. 3A). When 1.5-kb HindlIl fragment used <br /> the 1.5-kb Hindlll fragment shown carry sup-                probe RP026, difference observed the <br /> pressing activity. Deletions 2.0-kb DNA fragment            <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span> bands obtained high-stringency relatively <br /> were constructed determine portions DNA                low stringency hybridization conditions. This result suggests <br /> inserts essential suppression. Deletion sequences           single copy RP026 present yeast genome. <br /> between residues 1023 1889 abolished suppression (Fig.                The chromosomal location RP026 determined by <br /> 1), indicating suppressor gene span region.         probing DNA blot yeast chromosomes been <br />    Nucleotide sequence analysis. The nucleotide sequence            separated pulse-field gel electrophoresis (provided R. <br /> the entire 2.0-kb KpnI-SpeI suppressing fragment deter-            Wevrick H. Willard). The RP026 probe hybridized in <br /> mined (Fig. 2). The sequence revealed 155-amino-acid                 region chromosomes XIII XVI (Fig. 3B). These <br /> open reading-frame (ORF) interrupted 76-nucleotide-               chromosomes distinguished reprobing same <br /> long intervening sequence. The putative intron contains            DNA blot chromosome XIII-specific probe the <br /> three highly conserved sequence motifs present yeast            ADH2 gene (33) probes simultaneously. On the <br /> introns (21): 5&apos; splice junction GTATGT, canonical             basis results, RP026 gene assigned to <br /> branch point TACTAAC sequence near 3&apos; splice junc-                 chromosome XVI. <br /> tion, 3&apos; splice junction AG. The position                RP026 essential gene. The availability cloned <br /> intervening sequence amino terminus predicted            RP026 gene allowed determine gene is <br /> ORF typical yeast introns. The locations             required vegetative growth. A disrupted allele of <br /> intron boundaries confirmed sequencing ide-              RP026 constructed vitro replacing sequences <br /> pendent RP026 cDNAs isolated expression library               lie nucleotides 1023 1448 LEU2 <br /> constructed McKnight McConaughy (32). These                     gene (Fig. 4). By using one-step gene disruption method <br /> three cDNAs isolated hybridization using 1.5-kb            (40), chromosomal allele RP026 replaced the <br />  6126   ARCHAMBAULT ET AL.                                                                                    MOL. CELL. BIOL. <br />  <br />                     TCTAGAAAAAGACATATCAGGACATATATGACACTTCCAATTTTTGGTTACTAGTGAGTC                      60 <br />                     GTCATGAATAATCATGTGCATTTGTAAACCATTTTCCCCCACACATGGTTTGCTACAAAT                      120 <br />                     AGGACATTTTAATTTAGGATGATCATTTTTTATATGCGATTGCAATTGTGACCATATCCG                      180 <br />                     AAACTCTTTGCAACAACCAGCAAAAGTACATTGGTAAGGATTTTCTACCTCAGGATCGTG                      240 <br />                     ATGTTTAGAAATGTGATTTCTTAGCCTATATGGCCTCTGAAAGCTTTTATTGCAGTGTGG                      300 <br />                     ACAGGTTAGTTTATGTAAATGAACAGATAAAATATGTGCCCTTAATTGTGGATGCTTGTA                      360 <br />                     GAATCGGAGGTTGCATCCTTCCTCTGGACAAATGAAAGATTTGGTATGCGTTACTTCGTG                      420 <br />                     TCGCTTCAGTTGCTGGCGAGTCGTCACTCCTTTTCCACAATAAGAACATTGGAATGGTTT                      480 <br />                     CGTATCAGAATGCGTATACAAGTGTCTCTCTAAGTGACTCTTTTTAACGAAGGATTTTGC                      540 <br />                     ACACTATCACACTGAAATGCTCTTAAACCCTGATGTACGCTTAATTGGTGTTCAGTCAAA                      600 <br />                     ATTGAAGGTCTTGTAAATGCCTATCACAGCATCATAGTCACAGAAATATGTCTTTGGCCT                      660 <br />                     ATTGCTCGATGAGCTTCTAGTTGATGTTAAACTGTTCAATGATTCTGAACTCTCTGATCG                      720 <br />                     TGATATGGGAATGGTCTCCTGTTTAAGTTCGGCCAGTGGCATTCCTTCATTATTTAGAAC                      780 <br />                     CTCTCCTCCCATTTGACAGCGATTTCAACAGTTACTACTGTGCTTAATGCCCTTATTTGA                      840 <br />                     TAGTTAGTTCTTCTTATAATAAATAATCATTGTATATTGATTATTCGTATAGTATGATAT                      900 <br />                     TTAATTGAAAAAAAAATTTTTTTTTTTTTAAGAATATCATTCAAAGGCATCAATCACAAC                      960 <br />                                                             3*          1,2* <br />                     CTTGAAGAAAGGCTAAAAAGACACATTTTGCAGGTAACAGTGTAAAGATTAAGGCTACAA                      1020 <br />                           M S D Y E E A <br />                     GTGATCATGTCAGACTACGAGGAGGCGTATGTGCATTCATATTGGAGAAATTGCGTGCTT 1080 <br />                                                                               F   N   D   G   N   E <br />                     TAGTTTTAATTATATTTACTAACTTCTATCATTGTGATACAGGTTTAACGACGGTAATGA 1140 <br />                       N   F   E   D   F   D   V   E   H   F   S   D   E   E   T   Y   E   E   K   P <br />                     GAACTTCGAAGATTTTGATGTAGAGCATTTTTCTGATGAGGAGACTTATGAGGAAAAACC 1200 <br />                      Q F K D G E T T D A N G K T I V T G G N <br />                     TCAATTCAAGGATGGTGAAACAACCGATGCCAACGGTAAGACCATCGTTACTGGTGGTAA 1260 <br />                      G P E D F Q Q H E Q I R R K T L K E K A <br />                     TGGCCCAGAAGATTTTCAACAGCATGAGCAAATAAGAAGAAAGACACTTAAGGAAAAGGC 1320 <br />                      I P K D Q R A T T P Y M T K Y E R A R I <br />                     CATCCCAAAAGACCAAAGAGCAACTACTCCATACATGACCAAATATGAAAGGGCAAGAAT 1380 <br />                      L G T R A L Q I S M N A P V F V D L E G <br />                     TTTAGGTACAAGAGCCCTACAAATTTCCATGAATGCGCCAGTTTTTGTAGATTTAGAAGG 1440 <br />                      E T D P L R I A M K E L A E K K I P L V <br />                     TGAAACTGATCCATTGCGTATTGCCATGAAGGAGTTGGCTGAAAAGAAAATTCCTTTGGT 1500 <br />                      I R R Y L P D G S F E D W S V E E L I V <br />                     TATTAGAAGATATTTACCAGATGGTTCCTTTGAGGACTGGAGTGTGGAGGAACTCATTGT 1560 <br />                      D L - <br />                     GGATTTGTGATTACACCTGCATTTTCCTTTATGTGTATTTGCTTTGTTTGTATATTATTA 1620 <br />                                              1,2,3* <br />                     TAACTATAAAGAACGATTTATTTTGTAATGCAAATAACTGGACGAGAGAACCACGCTATA 1680 <br />                     ACCTATTTGGTAATAAAAATGTTTGTATGAATAAAAAAGCTGCTGACTGCTTTCTTTGAT 1740 <br />                        ATGTGTATATAGAATATTTATCGGTGATTGATCTAAGAAAAGCTTACCTCTGAAAGTTCT 1800 <br />                        CTCTTCGTCGCTTGCTGATTTGTAGTAAACAATTTGAAAAGTTTAGCAAATTCTTCCTCA 1860 <br />                        GGATTTTCAAAACCGGCTTCCTTTAAAGATCAATGACCTCATTCAAGGGAACGTTTAAAT 1920 <br />                        CGTGTCCTCTTCCAATGGCTTTTAAACTTTCTTCCAAATTCTTCACGCTCGGGGAAACTG 1980 <br />                        CGATAAGTTCTTCCCCATGATAGATAAGAATATGGGAAAGCTTACACCTTCTTCTCCAAC 2040 <br />                        TTCCGCGGTACTGGTGTCATTGTCAGGTACCATTTTGGACCATTCCTCATCTGTCAATGT 2100 <br />                        CTTCCCCAATGTTGCATATATCTTGGTAAGATC                                            2133 <br />   FIG. 2. Sequence 2.0-kb KpnI-SpeI suppressing fragment predicted amino acid sequence RP026 subunit. *, <br /> Locations 5&apos; 3&apos; ends RP026 cDNAs (1,2,3). <br />  <br /> RP026::LEU2 allele strain carrying plasmid-borne           ature-sensitive phenotype. As mentioned above, se- <br /> copy (URA3 marked) RP026 gene. We deter-              quence RP026 ORF carried suppressor plasmid <br /> mined cells lost plasmid-borne copy           identical reported Woychik et al. (49). To <br /> RP026 retained viability. First, transformant          confirm RP026 gene carried suppressor <br /> integration RP026 ::LEU2 allele occurred        plasmid wt sequence entire <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> gene, <br /> chromosomal locus chosen basis DNA blot             sequenced independent copies RP026 had <br /> analysis. Cells strain (JAY444) grown non-        isolated wt yeast strain method gap <br /> selective medium 10 generations            repair excision (see Materials Methods). The se- <br /> tested ability grow medium lacking leucine      quences copies determined (from nucleotides <br /> including 5-FOA (1 mg/ml). Only Ura- cells grow            50 2133) compared sequence RP026 <br /> 5-FOA. No cells recovered grow medium          gene originally isolated suppression rpo214 tem- <br /> containing 5-FOA (Fig. 4). Thus, loss plasmid con-         perature-sensitive phenotype. In cases, nucleotide <br /> taining URA3 RP026 lethal, indicating RP026       differences ORF upstream (976 <br /> gene essential. In contrast, strain JAY444            nucleotides) downstream (567 nucleotides) untranslated <br /> transformed second plasmid carrying RP026               regions. These results indicate increased gene dosage of <br /> cDNA (JAY436), cells capable growth 5-FOA-contain-          wt RP026 sufficient bring suppression rpo214. <br /> ing medium recovered. This result demonstrates             Increased expression RP026 cDNA suppresses the <br /> the RP026 gene product performs essential function         rpo2-4 temperature-sensitive phenotype. The observation <br /> that likely does belong family functionally        increased dosage RP026 sufficient suppress the <br /> interchangeable proteins.                                         temperature-sensitive defect imposed rpo214 muta- <br />    Increased dosage RP026 suppresses rpo214 temper-        tion prompted investigate increasing expres- <br />  VOL. 10, 1990                                                                  SUPPRESSOR OF AN RNA POLYMERASE II MUTATION                        6127 <br />  <br />                                                                                                              IN <br />                           A)                                                      B) <br />                                              _        O <br />                                                     N                                  MOI(N <br />                                                                                          (N <br />                                    V <br />                                          o <br />                                          u        (//; <br />                                                                                          0 <br />                                                                                                  0         a)00- <br />                                   COa    2 <br />  <br />  <br />                                                                                                                          X <br />                                                                                                                          IV <br />  <br />  <br />                                         *4 <br />                                                           -   5.0 5        i <br />                                                                                                                          xv <br />                                                                                                                          XIS <br />                                                                                                                          1 <br />                                                           -4     27                                                      xii <br />                                                                                                                          XIV <br />                                                               -} 48                     114, <br />                                                                                           F~~                            X <br />                                                                                                                          xi <br />  <br />                                                           _ 981                                                          yin <br />                                                           - lg90                                                         III <br />                                                                                                                          VI <br />                                              _w           -   1 <br />                                                           -1 77 <br />                                                                   59 <br />  <br />  <br />                                                           -0 94 <br />                                                           -083 <br />  <br />  <br />  <br />   FIG. 3. DNA blot analysis chromosomal location RP026. (A) DNA blot analysis. Total genomic yeast DNA isolated wt <br /> yeast strain digested various restriction endonucleases. The probe used 1.5-kb HindIll fragment encompassing RP026 <br /> ORF. DNA sizes expressed kilobase pairs. (B) Chromosomal location RP026. Yeast chromosomes separated by <br /> contour-clamped homogeneous electric-field gel electrophoresis (16). The probe used described above. The positions the <br /> various yeast chromosomes indicated right. <br />  <br /> sion     RP026 cDNA lead              suppression,                 TRPJ) tested ability suppress temperature <br /> presumably result overproduction RP026                              sensitivity inositol auxotrophy rpo2l4 mutant cells. <br /> gene product. To end, wt RP026 cDNA expressed                                   Strain JAY467 carrying RP026 cDNA acquired temper- <br /> from promoter alcohol dehydrogenase 1 gene                                  ature resistance ability grow medium <br /> (pADC1) carried high-copy-number plasmid (2Rm                                 lacking inositol (Fig. 5), indicating increased transcrip- <br />  <br />  <br />                           A) <br />  <br />  <br />  <br />                                                                                                             H <br />                           B) <br />                                     qT~~~~~~~~~~~~~~~~~;. <br />                                            5/ <br />                                                                    s,-,- <br />  <br />  <br />                                                                                                                      v <br />  <br />  <br />  <br />  <br />   FIG. 4. Disruption RP026 gene effect cell viability. (A) Structure disrupted allele RP026. The open box represent <br /> the RP026 ORF. (B) Growth phenotypes cells spotted agar medium containing lacking 5-FOA 5 days incubation 30&apos;C. <br />  6128    ARCHAMBAULT ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />                                                                       Conditional expression RP026. A yeast strain ex- <br />                                                                    presses RP026 strong inducible promoter, PGALI&apos; <br />       .-                                                           constructed verify increasing expression the <br />                                                                    chromosomal RP026 gene results suppression the <br />                                                                    rpo214 temperature-sensitive phenotype. The GAL] pro- <br />                                                                    moter induced 1,000-fold cells grown medium <br />                                                                    containing galactose repressed presence of <br />                                                                    glucose (27, 47). The strategy used depicted Fig. 6. <br />                                                                    Plasmid pJAY97, contains truncated RP026 ORF <br />                                                                    fused PGALIJ used transform JAY212 cells (rpo2l- <br />                                                                    4), creating strain JAY472. Integration directed the <br />                                                                    RP026 locus linearization plasmid restric- <br />                                                                    tion endonuclease AflII. Integration resulted duplication <br />                                                                    RP026 gene; truncated RP026 ORF placed <br />                                                                    control RP026 promoter, complete <br />                                                                    RP026 ORF controlled PGALI- As expected, growth <br />                                                                    strain JAY472 (rpo214 RP026::pJAY97) galactose <br />                                                   Ii         2 dependent. This strain longer temperature sensitive <br />                                               S w l        -&gt;      despite presence rpo214 mutation. This result <br />                 rpo2r4 <br />   FIG. 5. Suppression          phenotyrpes increased expres- demonstrates suppression obtained overex- <br /> sion RP026 cDNA. (A) Structure th RP026 cDNA used   pression wt RP026 gene normal chromosomal <br /> transform yeast strain JAY212. The RP026 cDNA cloned      location, presumably result overexpression the <br /> downstream alcohol dehydrogenase promoter (ADCI)        RP026 gene product. <br /> upstream CYCI transcription terminiator 2,um plasmid.     In addition, strain JAY476 (RPO21 RP026:: pJAY97) was <br /> (B) Phenotype strain JAY212 untransforr ned transformed constructed. This strain galactose dependent. The <br /> the RP026 cDNA. Cells spotted incubated 5 days on <br /> agar medium 30 37&#194;&#176;C medium lacking inositol 30&apos;C. <br />                                                                    construction strains conditionally express RP026 <br /> Strain W303-1A (wt) used control                          provides rapid efficient assay determining the <br />                                                                    effects mutations RP026, cells trans- <br />                                                                    formed mutant RP026 alleles phenotypes can <br /> tion wt RP026 ORF sufficient suppression. This         assayed glucose-containing medium (see below). <br /> result strongly suggests supprevssion rpo214              Isolation temperature-sensitive mutation RP026. As <br /> mutation occurs overexpressic:n RP026 gene          step understanding role RP026 gene <br /> product.                                                           product function RNAPI, -II, -III, tempera- <br />  <br />  <br />  <br />                                              j            CF      -- <br />  <br />                                   /1- <br />                                                           x <br />  <br />  <br />  <br />  <br />                                                ..1            I              7- r <br />  <br />  <br />                                                                                 vp <br />  <br />  <br />  <br />  <br />   FIG. 6. Construction yeast strains conditionally express RP026. (A) Structures plasmid (pJAY97) used integration the <br /> RP026 locus resulting partial gene duplication. (B) Growth phenotypes yeast strain JAY212 W303-1A transformed with <br /> pJAY97. Cells spotted medium containing glucose galactose sole carbon source. <br />  VOL. 10, 1990                                                     SUPPRESSOR OF AN RNA POLYMERASE                           II   MUTATION   6129 <br />  <br />  <br />                     A)          - <br />                                                              \\4L l[lil - <br />                                                                                                                        0         0 <br />  <br />  <br />  <br />  <br />                   RP026 <br />                   ALLELE                                                                    B) <br />  <br />  <br />                     WT              _   __   -_1                                ++                         0!                    0D <br />                                                                                                                  pmmq pin        - <br />                   L I II                                                   +     +       LI I I I     L <br />                   LRI 1    &apos;l                                              +     +       LRII <br />                   LKi 18 <br />                           18                                               + <br />                                                                            + <br />                                                                                  + <br />  <br />  <br />                                                                                  - <br />                                                                                          L1l18 <br />                                                                                          LK 18 <br />                                                                                                             .1 <br />                   LRR 18                                                   +             LRI 18 <br />                                                                                                       I- <br />                    LI 125                                                  +    +1-      LI 125 <br />                    LK125                           K                       +             LK 1 25            r <br />                                                                                                       r <br />                    LI   138 <br />                                                        A <br />                                                                            +     -       L1138 <br />                    PIA1 17 <br />                                                               |..__   ._ <br />  <br />                                                                            +     +       PA1 17 <br />                                   A~~~~ <br />                                   A ~ <br />                                 -~~~~  - - -----                           +     +       PA131 <br />                    PAI 31 <br />                    EK 144                                                  ts    -       EK144 <br />                                                                                          WT <br />                                                                                          VECTOR <br />                                                                                                        -         hom   wol b- <br />    FIG. 7. Locations phenotypes point mutations RP026. (A) Summary locations various point mutations the <br /> RP026 gene phenotypes combination RP021 rpo214 23&#194;&#176;C. The RP026 gene product diagrammed schematically. The <br /> filled box amino terminus represents acidic region, hatched box represents basic region, bars carboxy terminus <br /> represent heptad repeat leucine residues. +, Growth; -, absence growth. (B) Growth phenotypes strain JAY472 (rpo214 <br /> RP026:: pJAY97) transformed various RP026 alleles. The cells spotted agar medium lacking uracil containing glucose <br /> or galactose sole carbon source. The temperatures cells incubated indicated. <br />  <br />  <br />  <br /> ture-sensitive allele RP026 isolated using                     containing medium, growth strain dependent on <br /> plasmid-shuffling strategy (7, 11). A single-copy replicating              productive interaction mutant gene products. <br /> plasmid, pRP026HA (RP026 TRPI CEN4 ARSI),                              Some mutant rpo26 alleles (LK118, LR118, LK125, LH138, <br /> mutagenized hydroxylamine vitro (see Materials                 EK144) inviable combination rpo214 (Fig. <br /> Methods) introduced yeast strain JAY444                       7). This phenomenon, combination mutations <br /> [trpl-l RP026:: LEU2 (pRP026)]. Approximately 8,500                        lethal resulting cell death, <br /> Trp+ transformants grown 23&#194;&#176;C medium contain-                   termed synthetic lethality (18, 45); genetic <br /> ing 5-FOA select cells lost plasmid pRP026.                indication interaction gene products. It is <br /> The 5-FOA-resistant colonies tested growth                interesting leucine mutations furthest <br /> 37&#194;&#176;C. One temperature-sensitive lethal mutation RP026                   away carboxy terminus (LIlll, LR111, L1118) <br /> was isolated carried G-to-A transition nucleotide                showed synthetic lethality. As site rpo26 <br /> 1532, resulting glutamic acid-to-lysine substitution               mutation moved closer carboxy terminus, growth in <br /> amino acid 144 RP026 gene product. This mutation                 presence rpo214 increasingly impaired (LK118, <br /> recessive, shown ability wt RP026 gene                   LR118, LK125, L1138), reduced level <br /> rescue temperature-sensitive phenotype (data                       mutation involved conservative change (LI125). <br /> shown).                                                                    Taken together, results point role carboxy- <br />    Synthetic lethality RP021 RP026 mutations. To                    terminal portion RP026 gene product interaction with <br /> determine role, any, heptad repetition                    RP021 gene product. Such interaction direct or <br /> leucines carboxy end RP026 gene product,                     occur contact proteins, as <br /> these residues mutated vitro isoleucine, lysine,             subunits RNAPII. <br /> arginine. As well, proline residues located the <br /> heptad repetition leucines changed alanine resi-                                          DISCUSSION <br /> dues. The positions mutations phenotypes <br /> are summarized Fig. 7. None mutations                       Eucaryotic RNAPs complex enzymes two <br /> discernible effect RP026 function; mutant                    large subunits associated smaller polypeptides <br /> alleles able complement RP026 LEU2 null                        (reviewed reference 43). We begun genetic analysis <br /> allele PGALI-RPO26 allele (data shown). Comple-                 S. cerevisiae RNAPII goal defining role of <br /> mentation detected 23 37&#194;&#176;C medium                       individual subunits transcription. To end, have <br /> containing lacking inositol.                                           constructed linker insertion mutations (Archambault et al., <br />    The mutations constructed leucine-                 unpublished data) gene encoding largest subunit of <br /> repeat region carboxy end RP026 (see above)                 enzyme (RP021). One allele, rpo214, displays tem- <br /> transferred strain containing GALI-RP026                  perature-sensitive phenotype partial inositol <br /> rpo2l4 alleles tested growth. On glucose-                     requirement slow-growth phenotype permissive <br />  6130    ARCHAMBAULT ET AL.                                                                                          MOL . CELL . B IOL . <br />  <br /> temperature. To gain better understanding molecular     RP021 RP026 gene products interact functionally; <br /> defect underlying various phenotypes, used       surprising, enzyme <br /> yeast genomic library isolate plasmids suppress      complex. Such interaction required the <br /> the rpo214 mutation. The suppressor plasmids carry         structural functional integrity RNAPII. It interest <br /> genes products interact functionally largest      subunits implicated DNA binding (9, <br /> subunit RNAPII. Here, report isolation             23); possibly basic region, located central <br /> characterization suppressor plasmid carries         portion RP026 protein, interact DNA. <br /> RP026, essential, single-copy gene encodes             Another feature RP026 gene product has <br /> 17,980-Da subunit common RNAPs (49).                emerged sequencing analysis presence a <br />    A primary finding work increased expression   highly acidic region amino terminus protein. <br /> of RP026 suppress phenotypes associated             This region potential fold alpha helix (15, <br /> rpo214. More specifically, shown suppression        20). The function domain unknown. The presence <br /> of rpo214 occur following conditions: (i)          highly charged regions RP026 gene prod- <br /> increased copy number RP026 gene placing       uct provide explanation size discrepancy <br /> low-copy-number, replicating plasmid (CEN ARS); (ii) in-         predicted molecular weight protein (Mr <br /> creased expression RP026 placing RP026 cDNA             17,908) molecular weight observed polyacryl- <br /> under control ADCJ promoter high-copy-           amide gels (Mr 23,000). Alternatively, difference might <br /> number plasmid (2,um); (iii) increased transcription      phosphorylation RP026 gene product (4, 5, <br /> the RP026 gene placing control strong     8) posttranslational modifications. The availability <br /> GAL] promoter. In cases, determined       cloned RP026 gene prove useful analysis <br /> the RP026 ORF expressed wt sequence. These             transcription RNAPI, -II, -III. <br /> observations strongly support suggestion in-            The isolation subunit RNAPI, -II, -III a <br /> creased wt RP026 gene product sufficient     suppressor mutation largest subunit RNAPII is <br /> bring suppression. They consistent            example use classical reverse genetics to <br /> hypotheses: (i) rpo214 mutation increases dissocia-      characterize functional interaction subunits <br /> tion constant (Kd) RP026 RP021 gene              eucaryotic RNAP. It reinforces validity this <br /> products RP026 gene product           approach identification gene products partici- <br /> component RNAP complex, (ii) rpo214 allele         pate transcription. <br /> encodes unstable protein stabilized result <br /> of forming larger complex. In case,                               ACKNOWLEDGMENTS <br /> overproduction RP026 gene product shift           We thank colleagues, especially Emmanuel Maicas Alan <br /> equilibrium complex formation result stabi-      Davidson, discussions, Jason Zacks Peter Warburton for <br /> lize rpo214 gene product. The observation increas-      technical assistance, Jacqueline Segall critical reading the <br /> ing transcription rpo214, placing gene       manuscript constant support course work. We <br /> control strong GALIO promoter, does suppress      thank Rachel Wevrick Huntington Willard providing a <br /> temperature-sensitive phenotype relevant possi-       yeast chromosome blot. <br /> bilities (data shown) suggests increased       J.A. holds studentship Medical Research Council of <br /> amount rpo214 gene product sufficient bring     Canada. This work supported grant Medical <br /> about suppression, determined          Research Council Canada. <br /> increased transcription rpo214 results greater accumu-                             LITERATURE CITED <br /> lation encoded gene product. <br />     Suppression increased expression wt gene          1. Allison, L. A., M. Moyle, M. Shales, C. J. Ingles. 1985. <br /> been observed elsewhere. For example, Clark-Adams et al.             Extensive homology largest subunits eukaryotic <br />                                                                      prokaryotic RNA polymerases. Cell 42:599-610. <br />  (17) screened high-copy-number clones          2. Allison, L. A., K.-C. Wong, V. D. Fitzpatrick, M. Moyle, and <br />  suppress solo insertion mutations yeast. With             C. J. Ingles. 1988. The C-terminal domain largest subunit <br />  selection, isolated wt histone locus HTAI-            RNA polymerase II Saccharomyces cerevisiae, Droso- <br /> HTBI, contains genes previously identified genet-          phila melanogaster, mammals: conserved structure with <br />  ically SPTIJ SPT12.                                          essential function. Mol. Cell. Biol. 8:321-329. <br />     In effort begin understanding regions      3. Arndt, K. T., C. A. Styles, G. R. Fink. 1989. A suppressor <br /> RP026 gene product important function(s),              H1S4 transcriptional defect encodes protein homol- <br />  temperature-sensitive allele RP026 isolated using         ogy catalytic subunit protein phosphatases. Cell <br />  random chemical mutagenesis. The temperature-sensitive              56:527-537. <br />                                                                   4. Bell, G. I., P. Valenzuela, W. Rutter. 1976. Phosphorylation <br />  mutation results glutamic acid-to-lysine substitution       yeast RNA polymerases. Nature (London) 261:429-431. <br />  codon 144. In addition, mutations constructed        5. Bell, G. I., P. Valenzuela, W. J. Rutter. 1977. Phosphory- <br />  carboxy end RP026 region heptad repeat            lation yeast DNA-dependent RNA polymerases vivo in <br />  leucines. Site-directed mutagenesis leucine residues       vitro. J. Biol. Chem. 252:3082-3091. <br />  indicated essential vegetative growth.     6. Biggs, J., L. L. Searles, A. L. Greenleaf. 1985. Structure of <br />  However, mutations lethal combined             eukaryotic transcription apparatus: features gene for <br />  rpo214. Such synthetic lethality genetic indication       largest subunit Drosophila RNA polymerase II. Cell <br />  interaction RP021 RP026 gene products            42:611-621. <br />  involve carboxy-terminal portion           7. Boeke, J. D., F.    Lacroute, G. R. Fink. 1984. A positive <br />                                                                      selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br />  RP026 gene product. Other examples inviable combina-             ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br />  tions mutant suppressor alleles observed           Genet. 197:345-346. <br />  genes encoding components cytoskeleton S.          8. Breant, B., J.-M. Buhler, A. Sentenac, P. Fromageot. 1983. <br />  cerevisiae (36).                                                    On phosphorylation yeast RNA polymerase A B. <br />     The genetic analysis presented established         Eur. J. Biochem. 130:247-251. <br />  VOL. 10, 1990                                                      SUPPRESSOR OF AN RNA POLYMERASE II MUTATION                           6131 <br />  <br />  9. Breant, B., J. Huet, A. Sentenac, P. Fromageot. 1983.             29. Kyte, J., R. F. Doolittle. 1982. A simple method for <br />     Analysis yeast RNA polymerases subunit-specific anti-             displaying hydropathic character protein. J. Mol. Biol. <br />     bodies. J. Biol. Chem. 258:11968-11973.                                   157:105-132. <br /> 10. Broyles, S. S., B. Moss. 1986. Homology RNA               30. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />     polymerases poxviruses, prokaryotes, eukaryotes: nu-               cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />     cleotide sequence transcriptional analysis vaccinia virus          Cold Spring Harbor, N.Y. <br />     genes encoding 147-kDa 22-kDa subunits. Proc. Natl. Acad.         31. Martin, C., R. A. Young. 1989. KEX2 mutations suppress <br />     Sci. USA 83:3141-3145.                                                    RNA polymerase II mutants alter temperature range of <br /> 11. Budd, M., J. L. Campbell. 1987. Temperature-sensitive                 yeast cell growth. Mol. Cell. Biol. 9:2341-2349. <br />     mutations yeast DNA polymerase. Proc. Natl. Acad. Sci.         32. McKnight, G. L., B. L. McConaughy. 1983. Selection of <br />     USA 84:2838-2842.                                                         functional cDNAs complementation yeast. Proc. Natl. <br /> 12. Buratowski, S., S. Hahn, P. A. Sharp, L. Guarente. 1988.              Acad. Sci. USA 80:4412-4416. <br />     Function yeast TATA element-binding protein mam-            33. Mortimer, R. K., D. Schild, C. R. Contopoulou, J. A. Kans. <br />     malian transcription system. Nature (London) 334:37-42.                   1989. Genetic map Saccharomyces cerevisiae, edition 10. <br /> 13. Cavailini, B., J. Huet, J.-L. Plassat, A. Sentenac, J.-M. Egly,       Yeast 5:321-403. <br />     P. Chambon. 1988. A yeast activity substitute HeLa        34. Nonet, M., D. Sweetser, R. A. Young. 1987. Eukaryotic <br />     cell TATA box factor. Nature (London) 334:77-80.                          RNA polymerase conditional mutant rapidly ceases mRNA <br /> 14. Chodosh, L. A., J. Olesen, S. Hahn, A. S. Baldwin, L. Guarente,           synthesis. Mol. Cell. Biol. 7:1602-1611. <br />     P. A. Sharp. 1988. A yeast human CCAAT-binding              35. Nonet, M., D. Sweetser, R. A. Young. 1987. Functional <br />     protein heterologous subunits functionally inter-           redundancy structural polymorphism large subunit of <br />     changeable. Cell 53:25-35.                                                RNA polymerase II. Cell 50:909-915. <br /> 15. Chou, P. Y., G. D. Fasman. 1978. Prediction secondary          36. Novick, P., B. C. Osmond, D. Botstein. 1989. Suppressors of <br />     structure proteins amino-acid sequence. Adv.                yeast actin mutations. Genetics 121:659-674. <br />     Enzymol. 45:45-148.                                                   37. Orr-Weaver, T. L., J. W. Szostak, R. J. Rothstein. 1983. <br /> 16. Chu, G., D. Vollrath, R. W. Davis. 1986. Separation large          Genetic application yeast transformation linear gapped <br />     DNA molecules contour-clamped homogenous electric                      plasmids. Methods Enzymol. 101:202-210. <br />     fields. Science 234:1582-1585.                                        38. Percival-Smith, A., J. Segall. 1986. Characterization and <br /> 17. Clark-Adams, C. D., D. Norris, M. A. Osley, J. S. Fassler,            mutational analysis cluster genes expressed pref- <br />     F. Winston. 1988. Changes histone gene dosage alter tran-              erentially sporulation Saccharomyces cerevisiae. Mol. <br />     scription yeast. Genes Dev. 2:150-159.                                 Cell. Biol. 6:2443-2451. <br /> 18. Dobzhansky, T. 1946. Genetics natural populations. XIII.           39. Riva, M., S. Memet, J.-Y. Micoin, J. Huet, I. Treich, J. Dassa, <br />     Recombination variability populations Drosophila                R. Young, J.-M. Buhler, A. Sentenac, P. Fromageot. 1986. <br />     pseudoobscura. Genetics 31:269-290.                                       Isolation structural genes yeast RNA polymerases by <br /> 19. Falkenburg, D., B. Dworniczak, D. M. Faust, E. K. F.                  immunological screening. Proc. Natl. Acad. Sci. USA 83:1554- <br />     Bautz. 1987. RNA polymerase II Drosophila. Relation             1558. <br />     140,000 Mr subunit beta subunit Escherichia coli RNA        40. Rothstein, R. J. 1983. One step gene disruption yeast. <br />     polymerase. J. Mol. Biol. 195:929-937.                                    Methods Enzymol. 101:202-211. <br /> 20. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis       41. Ruet, A., A. Sentenac, P. Fromageot, B. Winsor, F. Lac- <br />     accuracy implications simple methods predicting            route. 1980. A mutation B220 subunit gene affects the <br />     secondary structure globular proteins. J. Mol. Biol.               structural functional properties yeast RNA polymerase B <br />     120:97-120.                                                               vitro. J. Biol. Chem. 255:6450-6455. <br /> 21. Green, M. R. 1986. Pre-mRNA splicing. Annu. Rev. Genet.               42. Sanger, F., S. Nicklen, A. R. Coulson. 1977. DNA sequenc- <br />     20:671-708.                                                               ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br /> 22. Himmelfarb, H. J., E. M. Simpson, J. D. Friesen. 1987.                USA 74:5463-5467. <br />     Isolation characterization temperature-sensitive RNA           43. Sentenac, A. 1985. Eukaryotic RNA polymerases. Crit. Rev. <br />     polymerase II mutants Saccharomyces cerevisiae. Mol. Cell.             Biochem. 18:31-91. <br />     Biol. 7:2155-2164.                                                    44. Sherman, F., G. R. Fink, J. B. Hicks. 1986. Laboratory <br /> 23. Huet, J., M. Riva, A. Sentenac, P. Fromageot. 1985. Yeast             course manual methods yeast genetics. Cold Spring <br />     RNA polymerase C subunit. J. Biol. Chem. 260:15304-               Harbor Laboratory, Cold Spring Harbor, N.Y. <br />     15310.                                                                45. Sturtevant, A. H. 1956. A highly specific complementary lethal <br /> 24. Ingles, C. J., H. J. Himmelfarb, M. Shales, A. L. Greenleaf,          Drosophila melanogaster. Genetics 41:118-123. <br />     J. D. Friesen. 1984. Identification, molecular cloning, mu-       46. Sweetser, D., M. Nonet, R. A. Young. 1987. Prokaryotic and <br />     tagenesis Saccharomyces cerevisiae RNA polymerase genes.               eukaryotic RNA polymerases homologous core subunits. <br />     Proc. Natl. Acad. Sci. USA 81:2157-2161.                                  Proc. Natl. Acad. Sci. USA 84:1192-1196. <br /> 25. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor-         47. Yocum, R. R., S. Hanley, R. West, Jr <br /> </body></html>