<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A novel SMC protein complex in <br /> Schizosaccharomyces pombe contains the Rad18 <br /> DNA repair protein <br />  <br />  <br />                            by arresting the cell cycle following damaging treatments <br />                                                                  (for review, see Caspari and Carr, 1999). <br />      In previous work we used the rad18-X mutant <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> show <br />  <br />                                                                  that the Rad18 protein was involved in repair of ionizing <br />                                          radiation damage (Lehmann et al., 1995). Double-strand <br />  <br />                                                                  breaks are the principal lesions responsible for killing <br />                                                                  cells by ionizing radiation. Both Verkade et al. (1999) and <br /> In Schizosaccharomyces pombe, rad18 is an essential              we (unpublished observations) have shown that rad18-X <br /> gene involved in the repair of DNA damage produced               cells are deficient in repair of radiation-induced double- <br /> by ionizing radiation and in tolerance of UV-induced             strand breaks. <br /> DNA damage. The Rad18 protein is a member of                        Epistasis analysis in response to UV-irradiation suggests <br /> the SMC (structural maintenance of chromosomes)                  that Rad18 is not involved in NER, but that it does play <br /> superfamily, and we show that, like the other SMC                a role in the second excision repair pathway for UV <br /> proteins in condensin and cohesin, Rad18 is a compon- <br />                                                                  damage. The first step in this pathway is mediated by UV <br /> ent of a high-molecular-weight complex. This complex <br />                                                                  damage-endonuclease (UVDE) (Yonemasu et al., 1997), <br /> contains at least six other proteins, the largest of which <br />                                                                  which nicks UV-irradiated DNA on the 5&#226;&#172;&#732; side of UV <br /> is Spr18, a novel SMC family member closely related <br />                                                                  photoproducts (Avery et al., 1999; Yoon et al., 1999). In <br /> to Rad18, and likely to be its heterodimeric partner. <br />                                                                  a uvde background, the rad18-X <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> still sensitized <br /> SMC proteins have ATP-binding domains at the <br />                                                                  the cells to UV-irradiation (Yonemasu et al., 1997), and <br /> N- and C-termini, and two extended coiled-coil domains <br />                                                                  further epistasis analysis suggested that this was due to a <br /> separated by a hinge in the middle. We show that the <br />                                                                  role for Rad18 in a DNA damage tolerance pathway <br /> N-terminal ATP-binding domain of Rad18 is essential <br />                                                                  (Murray et al., 1997). Furthermore, deletion of the rad18 <br /> for all functions, and overexpression of an N-terminal <br />                                                                  gene demonstrated that it was essential for cell prolifera- <br /> mutant has a dominant-negative effect. We have identi- <br />                                                                  tion, and our data led us to propose tentatively that this <br /> fied an important mutation (S1045A) near the <br /> C-terminus of Rad18 that separates its repair and                essential role was an involvement in DNA replication <br /> essential roles. Potential models for the role of the            (Lehmann et al., 1995). Figure 1 summarizes the pathways <br /> Rad18&#226;&#8364;&#8220;Spr18 complex during DNA repair are dis-                   in which Rad18 is involved, based on this genetic analysis. <br /> cussed.                                                             Sequence analysis of Rad18 (Lehmann et al., 1995) <br /> Keywords: ATPase/coiled coils/DNA repair/fission yeast/          showed that the 131 kDa Rad18 protein was a member <br /> SMC proteins                                                     of the SMC (structural maintenance of chromosomes) <br />                                                                  superfamily. SMC proteins have globular N- and <br />                                                                  C-terminal domains, which are involved in ATP and Mg2&#226;&#171;&#185; <br />                                                                  binding, and two extended &#206;&#177;-helical coiled-coil domains <br />                                                                  involved in protein&#226;&#8364;&#8220;protein interactions, separated by a <br /> Introduction                                                     hinge region (for reviews, see Hirano, 1998, 1999; <br /> Many genes involved in the response to DNA damage have           Jessberger et al., 1998). Recent work using Saccharomyces <br /> been identified in the fission yeast Schizosaccharomyces         cerevisiae, S.pombe and Xenopus laevis has identified two <br /> pombe (see Lehmann, 1996). Epistasis analysis has enabled        protein complexes containing heterodimeric pairs of SMC <br /> these genes to be assigned to different DNA repair               proteins. Smc2 and Smc4 (and their orthologues) form <br /> pathways. Following treatment with ionizing radiation,           the core of condensin, a five-subunit complex involved in <br /> repair of DNA double-strand breaks requires genes known          chromosome condensation (Hirano et al., 1997; Sutani <br /> to be involved in homologous recombination, suggesting           et al., 1999). Smc1 and Smc3 (and their orthologues) are <br /> that this mechanism is likely to be the major means of           the core of a five-subunit cohesin complex, which is <br /> repairing such DNA damage in fission yeast. Photoprod-           required for sister chromatid cohesion (Michaelis et al., <br /> ucts induced in DNA by UV-irradiation can be repaired            1997; Losada et al., 1998). There is also evidence <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> both by the classical nucleotide excision repair (NER)           bovine cells to suggest that SMC1 and SMC3 proteins <br /> system, which is conserved in all eukaryotes, and by a           form part of a complex with DNA ligase III and DNA <br /> second excision repair process, which is specific to             polymerase &#206;&#181;, which has DNA strand exchange activity <br /> S.pombe and a few other organisms (Yonemasu et al.,              (Jessberger et al., 1996). <br /> 1997; Yasui and McCready, 1998). In addition to these               We describe in this paper the purification of a high- <br /> pathways that repair DNA damage directly, cells are able         molecular-weight complex with DNA-dependent ATPase <br /> to tolerate UV damage during DNA replication (Murray             activity, which contains Rad18 and at least six other <br /> et al., 1997) and to utilize cell cycle checkpoint mechan-       polypeptides. The largest of these is a novel SMC protein <br /> isms to ameliorate the deleterious effects of DNA damage         (Spr18), which is likely to be the heterodimeric partner <br />  <br /> &#194;&#169; European Molecular Biology Organization                                                                              1691 <br />  M.I.Fousteri and A.R.Lehmann <br />  <br />  <br />  <br />  <br /> Fig. 1. Involvement of Rad18 in different cellular responses to DNA <br /> damage. <br />  <br />  <br /> of Rad18. We show by site-directed mutagenesis that the <br /> N-terminal ATP-binding domain of Rad18 is required for <br /> all its functions, and that a C-terminal mutation can <br /> separate repair from essential functions. <br />  <br /> Results <br /> Rad18 is part of a high-molecular-weight complex <br /> To determine whether Rad18, like other SMC proteins, is <br /> a member of a protein complex, we fractionated extracts <br /> of S.pombe in gel filtration columns. For these experiments <br /> we constructed cell strain myc-Rad18, in which the rad18 <br /> gene was myc-tagged in the genome at the 5&#226;&#172;&#732; end, <br /> remaining under the control of its own promoter. When <br /> extracts of myc-Rad18 cells were passed down a Superdex <br /> 200 column, all the Rad18 protein was found in a high- <br /> molecular-weight peak (Mr &#226;&#172;&#381;700 kDa). To obtain a more <br /> accurate estimate of the size of this complex, the peak               Fig. 2. Rad18 is part of a high-molecular-weight complex. Extracts of <br /> fractions from the column were pooled and analysed on                 myc-Rad18 (A) or myc-Rad18-X cells (C) were precipitated with <br /> a Superose 6 column (Figure 2A). The results show that                ammonium sulfate, fractionated on Superdex 200 and the high- <br />                                                                       molecular-weight peak analysed by gel filtration on a Superose 6 <br /> Rad18 is part of a complex whose apparent Mr is ~1.6 MDa.             column. Fractions were subjected to immunoblotting with anti-myc <br /> This complex was stable in the presence of up to 0.7 M                antibody. The high-molecular-weight band on the gel is non-specific <br /> NaCl (not shown). Using a polyclonal antibody raised                  cross-reacting material. (B) Wild-type strain 501 was subjected to the <br /> against a Rad18 peptide, we also showed that the gel                  same treatment as above, but analysed with anti-Rad18 antibody. <br /> filtration profile was identical whether or not the Rad18 <br /> protein was N-terminally myc-tagged (Figure 2A and B). <br /> No change in the gel filtration profile was detected when             ive washing, bound proteins were eluted with a peptide <br /> extracts of &#206;&#179;-irradiated cells were used (data not shown).            corresponding to the c-myc epitope, and analysed both <br />     As discussed below, the mutation in rad18-X is located            by immunoblotting and colloidal Coomassie staining. <br /> in the hinge region, immediately adjacent to the second               Immunoblotting showed that the bound and eluted proteins <br /> coiled-coil region (see Figure 5A). This raised the possibil-         contained myc-tagged Rad18 (Figure 3B, lane 2). When <br /> ity that the mutation might disrupt the coiled-coil structure         extracts from untagged control cells (strain 501) were <br /> of the protein and interfere with complex formation. The              subjected to identical purification procedures, no corres- <br /> Rad18 profiles on gel filtration columns using normal                 ponding protein was bound to the beads (Figure 3B, <br /> and rad18-X extracts were, however, indistinguishable                 lane 4). Coomassie staining revealed that, apart from <br /> (compare Figure 2C and A), even at salt concentrations                Rad18, there were six specific bands in the eluate from <br /> up to 0.7 M NaCl. The rad18-X mutation does not,                      the tagged cells (Figure 3C, lane 3) that were not found <br /> therefore, interfere with complex formation.                          in similar experiments using untagged control cells <br />                                                                       (Figure 3C, lanes 1 and 2). One of these specific bands, <br /> Purification of Rad18                                                 p124, had a molecular weight close to that of Rad18 and <br /> We have developed a purification procedure for the Rad18-             was of similar intensity. <br /> containing complex, using extracts from myc-Rad18 cells <br /> (Figure 3A). After ammonium sulfate precipitation, the                ATPase activity <br /> extracts were subjected to gel filtration on Superdex 200.            Mutagenesis studies described in detail below implicate <br /> The high-molecular-weight fractions containing Rad18                  ATP-binding activity of Rad18 in all its functions, sug- <br /> were pooled and fractionated on a heparin&#226;&#8364;&#8220;Sepharose                   gesting that the protein might have ATPase activity. We <br /> column. The 0.6 M KCl fractions, which contained the                  have therefore measured ATPase activity of the Rad18 <br /> Rad18 protein, were incubated with protein G beads                    complex. We assayed the peptide eluate from the immuno- <br /> covalently cross-linked to anti-myc antibody. After extens-           affinity beads for ATPase activities in the presence and <br />  <br /> 1692 <br />                                                                                                Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br />  <br />  <br /> Fig. 3. The Rad18 complex. (A) Scheme for purification of the Rad18 complex from myc-Rad18 cell extracts (WCE, whole cell extracts; FT, flow- <br /> through). (B) Samples of the proteins loaded onto (Load) and eluted from (B&#226;&#8364;&#8482;nd) the anti-myc immuno-affinity beads were immunoblotted with anti- <br /> myc or anti-Spr18 antibodies. (C) Proteins bound to the beads were eluted with the myc peptide and analysed by SDS&#226;&#8364;&#8220;PAGE. The gel was fixed and <br /> stained with colloidal Coomassie Blue. Lanes 1 and 2, two extracts of strain 501; lane 3, myc-Rad18. (D) ATPase activity of the Rad18 complex. <br /> Experiments were carried out three times, typical results are shown. <br />  <br />  <br /> absence of DNA. Eluate from the myc-Rad18 cell extracts                    found a candidate open reading frame (ORF) in cosmid <br /> had significant ATPase activity, which was stimulated by                   C14C4 from chromosome 1 (DDBJ/EMBL/GenBank <br /> single-stranded, and to a greater extent by double-stranded                accession No. Z98596) with an orthologue in the <br /> DNA (Figure 3D). In contrast, eluate from untagged                         S.cerevisiae database (ORF SCYOL034W; DDBJ/EMBL/ <br /> control strain 501 containing an equal amount of the non-                  GenBank accession No. Z74776). The predicted 123 kDa <br /> specific 73 kDa protein (see Figure 3C) had minimal                        protein was related to the SMC family, but was most <br /> activity, which was unchanged by the addition of DNA.                      closely related to Rad18 (see Figure 4A). We designated <br /> The complex is therefore associated with specific DNA-                     it Spr18 (SMC partner of rad18). Alignments of Rad18 <br /> stimulated ATPase activity.                                                with Spr18 and its S.cerevisiae orthologue are shown in <br />                                                                            Figure 4B. There is 28% identity between the S.pombe <br /> Spr18, a heterodimeric partner of Rad18                                    and S.cerevisiae orthologues, and 20% identity between <br /> In the SMC family, Smc2 and Smc4 form a heterodimeric                      Rad18 and Spr18. Like rad18, spr18 has a small intron <br /> component of the condensin complex (Hirano et al., 1997),                  (49 bases) close to the 5&#226;&#172;&#732; end of the gene (between nt 187 <br /> which is involved in chromosome condensation. Smc1                         and 188). <br /> and Smc3 are similarly part of cohesin, involved in sister                    In order to determine whether Spr18 interacts with <br /> chromatid cohesion (Michaelis et al., 1997). We therefore                  Rad18 in vivo, we carried out immunoprecipitations using <br /> anticipated that Rad18 might have a heterodimeric partner,                 the myc-Rad18 strain (Figure 4C). Extracts were immuno- <br /> and we considered the possibility that the p124 polypeptide                precipitated with antibody directed against a peptide from <br /> in the complex might be a novel SMC protein partner of                     Spr18 (Figure 4C, lanes 2, 4, 6 and 8) or, as control, with <br /> Rad18. We searched the S.pombe sequence database and                       anti-myc antibody (lanes 1, 3, 5 and 7). The immuno- <br />  <br />                                                                                                                                          1693 <br />  M.I.Fousteri and A.R.Lehmann <br />  <br />  <br />  <br />  <br /> 1694 <br />                                                                                                 Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> precipitates were electrophoresed in an SDS&#226;&#8364;&#8220;polyacryl-                      Viability and the Ura4&#226;&#171;&#185; <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>phenotype</span> segregated in a 2:0 <br /> amide gel and immunoblotted with the anti-myc antibody.                     manner, confirming that spr18, like rad18 and the SMC <br /> Figure 4C shows that myc-tagged Rad18 co-immunopre-                         genes, is essential for cell proliferation. In order to <br /> cipitated with Spr18 in these experiments in the presence                   determine the terminal phenotype of the spr18 deletion <br /> of salt concentrations up to 0.75 M (lanes 1&#226;&#8364;&#8220;6). Similar                    strain, the spores were germinated in liquid medium with <br /> results were obtained when the blots were probed with                       selection for Ura4&#226;&#171;&#185;, and cell number and morphology <br /> anti-Spr18 (data not shown). The results were not affected                  were determined. Germinating spores underwent one or <br /> by treatment of the cell extracts with DNase I prior to                     two divisions and ceased growth with a variety of abnormal <br /> immunoprecipitation, demonstrating that the interaction                     morphologies. Several examples are shown in Figure 4D. <br /> was not mediated by binding to DNA. When the immuno-                        Many cells were elongated, in some cases excessively so <br /> precipitation experiments were carried out with control                     (Figure 4D, examples 1 and 5), often with several septa <br /> untagged strain 501, no bands were detected on the                          (examples 3 and 5), and the nuclear material was barely <br /> immunoblot (Figure 4C, lanes 7 and 8). These results                        detectable, either being present in a single nucleus <br /> demonstrate that Rad18 and Spr18 do indeed interact,                        (example 4) or spread amorphously through part of the <br /> consistent with the idea that the two proteins form a                       cell (example 1). A proportion of cells (10&#226;&#8364;&#8220;15%) showed <br /> heterodimer in S.pombe cells, although we have not                          a cut phenotype (Figure 4D, example 2). The distribution <br /> formally eliminated the possibility that the interaction                    of these different phenotypes is shown in Figure 4E. The <br /> might be mediated by another protein. In results to be                      similarity of the terminal phenotype of the spr18 deletion <br /> presented elsewhere, we have shown a similar interaction                    mutant, rad18 deletion (Lehmann et al., 1995) and wild- <br /> between the human orthologues of these proteins                             type cells overexpressing dominant-negative mutants of <br /> (E.M.Taylor, J.Moghraby and A.R.Lehmann, unpublished                        rad18 (see below), provides further evidence that Rad18 <br /> observations).                                                              and Spr18 perform similar functions in the cell. <br />  <br /> Spr18 is the p124                                                           Mutagenesis of rad18 defines important structural <br /> The peptide elution fractions from the immuno-affinity                      domains <br /> column obtained using both the myc-Rad18 and 501                            The characteristic SMC structure of Rad18, with globular <br /> extracts (Figure 3) were analysed by immunoblotting with                    head and tail regions and two coiled-coil domains separated <br /> the anti-Spr antibody. Spr18, like Rad18, was detected in                   by a hinge, is depicted schematically in Figure 5A. To <br /> the bound and eluted proteins from the myc-Rad18 extract                    gain insight into the roles of the different domains, we <br /> (Figure 3B, lane 6), but not in the untagged control                        have used site-directed mutagenesis to introduce a series <br /> extracts (lane 8). The band detected using the anti-Spr18                   of mutations into rad18 cDNA. The cDNA was cloned <br /> antibody corresponded with that of the p124 protein in                      under the control of the attenuated thiamine-repressible <br /> the Coomassie-stained gel. These results confirm that p124                  nmt1 promoter in the vectors pRep41MH N (Leu&#226;&#171;&#185;) or <br /> is the SMC protein Spr18 and are consistent with the                        pRep42MH N (Ura&#226;&#171;&#185;) (Maundrell, 1993; Craven et al., <br /> suggestion that Rad18&#226;&#8364;&#8220;Spr18 forms a heterodimeric core                      1998), so that the Rad18 protein was N-terminally myc- <br /> of a large complex similar to the other SMC complexes,                      tagged. The effects of the mutations were examined in <br /> cohesin and condensin. To investigate whether the Rad18&#226;&#8364;&#8220;                    different ways, as follows: (i) the ability of the mutant <br /> Spr18 complex contains other known DNA damage                               plasmids to rescue the lethal effect of a rad18 deletion <br /> response genes, we carried out immunoblotting of the                        was examined by transformation into the rad18::ura4&#226;&#171;&#185;/ <br /> proteins eluted from the anti-myc affinity column using a                   rad18&#226;&#171;&#185; diploid strain; (ii) the mutated plasmids were <br /> series of antibodies. We were unable to detect the presence                 overexpressed in wild-type cells to look for dominant- <br /> of the DNA damage response proteins Rad9, Rad26,                            negative effects of the mutant gene on cell growth and <br /> Cds1, Rqh1 or Rad21 in the Rad18&#226;&#8364;&#8220;Spr18 complex (data                        viability, as well as on sensitivity to UV- and &#206;&#179;-irradiation; <br /> not shown).                                                                 and (iii) the mutated genes were introduced into rad18-X <br />                                                                             cells and their impact on the sensitivity of the mutant to <br /> Spr18 is an essential gene                                                  ionizing and UV-irradiation was examined following their <br /> If Spr18 is the partner protein of Rad18, we anticipated                    transient overexpression. <br /> that, like Rad18, it would be an essential gene. We deleted                    The mutations and their effects are summarized in <br /> the entire ORF of the spr18 gene from the genome in a                       Figure 5 and Table I. Several of the mutant constructs <br /> diploid strain by replacing one copy of the gene with the                   had a dominant-negative effect on cell viability <br /> ura4&#226;&#171;&#185; marker. Following sporulation and plating in the                      (Figure 5A). The proportion of the population that lost <br /> absence of uracil, no haploid colonies were obtained,                       viability depended on the mutation, but the phenotype of <br /> suggesting that the spr18 gene was essential for cell                       the dying cells was quite similar in all cases, and resembled <br /> proliferation. In order to confirm this, tetrads were dis-                  the terminal phenotype of deletion mutants of rad18 <br /> sected from the spores and plated on yeast extract plates.                  (Lehmann et al., 1995) or spr18 (see above). For simplicity <br />  <br />  <br /> Fig. 4. Spr18, the SMC partner protein of Rad18. (A) Phylogenetic tree of all the SMC family members in S.cerevisiae and S.pombe. (B) Sequence <br /> alignments of Rad18 with Spr18 and its S.cerevisiae orthologue (designated scspr). Identical amino acids are highlighted in black, and conserved <br /> residues are in grey. (C) Extracts of myc-Rad18 cells were immunoprecipitated with either anti-myc or anti-Spr18 antibody in the presence of the <br /> salt concentrations indicated. The immunoprecipitates were immunoblotted and probed with anti-myc antibody. The right hand lanes (7 and 8) show <br /> results with untagged strain 501. (D) Terminal phenotypes of the spr18 deletion mutant. Diploid spr18&#226;&#171;&#185;/spr18::ura4&#226;&#171;&#185;cells were sporulated and <br /> grown in the absence of uracil for 36 h. Cells were fixed and stained with DAPI and calcofluor. (E) Frequency of different phenotypes after 24 h <br /> growth. <br />  <br />  <br />                                                                                                                                           1695 <br />  M.I.Fousteri and A.R.Lehmann <br />  <br />  <br />  <br />  <br /> Fig. 5. Effects of site-directed mutations in Rad18. (A) Schematic of Rad18 protein with positions of the mutations, their ability to rescue the lethal <br /> phenotype of the rad18 deletion mutant and the effect of their overexpression on the viability of the wild-type strain (&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;, all cells dead; <br /> &#226;&#171;&#185;&#226;&#171;&#185;, most cells dead but a few retain viability; &#226;&#171;&#185;, 30&#226;&#8364;&#8220;40% cells retain viability). (B and C) Survivals after &#206;&#179;- or UV-irradiation of wild-type (B) or <br /> rad18-X cells (C) overexpressing the different mutant plasmids. Experiments were carried out three times, typical results are shown (Mrep42, empty <br /> pREP42 MH vector). <br />  <br />  <br />  <br /> we will refer to this as the Rad18 phenotype, although                         were small, a few septated without prior mitosis, resulting <br /> there is no intended implication that this phenotype is                        in very long cells either without a nucleus or with only <br /> unique to rad18 mutants. This phenotype consisted of                           one nucleus often with multiple septa. Examples are shown <br /> cells with a variety of abnormal morphologies in which                         in Figure 6A (compare with overexpressed wild-type <br /> nuclear segregation was clearly aberrant. Many of them                         rad18 in Figure 6B). <br />  <br /> 1696 <br />                                                                                                    Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br />  <br />  <br /> Fig. 6. Dominant-negative effects of mutations in the rad18 gene. (A and B) Effects of overexpressing (A) rad18K129Q, rad18L843P or <br /> rad18R706A and (B) wild-type rad18 on cell growth and morphology. (C) Wild-type cells overexpressing either wild-type rad18, the rad18K129E <br /> mutant plasmid, or both, were incubated in the absence of thiamine and cell numbers counted over a 24 h time period. (D) Western blots from cells <br /> overexpressing myc-tagged rad18K129E alone (lane 1) or with myc-tagged rad1 (lane 2), rad2 (lane 3), rad3 (lane 4), pcn1 (lane 5), rad17 (lane 7) <br /> and HA-tagged cdc6 (lane 6). <br />  <br />  <br />  <br /> Table I. Summary of effects of overexpressing mutant constructs <br />  <br />                       K129E          K129Q          R706C         R706A          L843P          L843F             S1045A   S1045T       D1072A <br />  <br /> WT viabilitya         0              35             &#226;&#171;&#185;             42             20             &#226;&#171;&#185;                 &#226;&#171;&#185;        &#226;&#171;&#185;            &#226;&#171;&#185; <br /> WT UVb                dead           S              N             S              dead           N                 S        S            N <br /> WT &#206;&#179; b                dead           S              N             S              dead           N                 S        N            N <br /> Rad18-X UVc           dead           SS             SS            SS             SS             R                 SS       SS           SS <br /> Rad18-X &#206;&#179; c           dead           SSS            SS            SSS            SS             R                 SSS      SSS          SS <br /> Rad18&#226;&#710;&#8224;d               no             no             yes           partial        no             yes               yes      no           no <br /> aNumbers  denote percentage of viable cells. &#226;&#171;&#185;, &#226;&#172;&#381;70% viability. <br /> bN, normal; S, more sensitive than normal. <br /> cR,restores wild-type resistance; SS, sensitivity comparable to rad18-X; SSS, sensitivity greater than rad18-X. <br /> dYes, does; no, does not rescue lethality of deletion strain. <br />  <br />  <br />  <br />  <br />    The N-terminal domain of Rad18 contains the highly                         liquid culture, cell growth was prevented (Figure 6C) and <br /> conserved Walker Type A box found in many ATPases.                            the cells died with the Rad18 phenotype. <br /> We mutated the conserved Lys129 residue in the GXGKS                             We also generated a milder mutation of Lys129 to <br /> motif to glutamic acid (rad18K129E). When rad18K129E                          glutamine (K129Q). When plasmids containing this con- <br /> was overexpressed in wild-type cells, it had a dramatic                       struct were overexpressed in wild-type cells, the resulting <br /> dominant-negative effect on viability, completely abol-                       colonies were very small, and in liquid culture, a significant <br /> ishing the colony-forming ability of the cells. A similar                     proportion of the cells (65%) showed the Rad18 phenotype <br /> result has recently been obtained for a different mutation                    and had reduced viability (Figure 6A). The viable cells <br /> in the ATP-binding site by Verkade et al. (1999). When                        were sensitized to both UV- and ionizing radiation <br /> the rad18K129E was overexpressed in cells growing in                          (Figure 5B). The mutant plasmid failed to rescue the <br />  <br />                                                                                                                                              1697 <br />  M.I.Fousteri and A.R.Lehmann <br />  <br />  <br /> viability of the deletion mutant and slightly increased the     mutations in rad18. Mutations in the ATP-binding site in <br /> sensitivity of the rad18-X cells to &#206;&#179;-irradiation (Figure 5A    the N-terminus and disruption of the coiled-coil domain <br /> and C).                                                         had the most dramatic effects, suggesting that the ATP- <br />    The C-terminal globular region contains motifs LSGG          binding site and coiled-coil structures are essential for all <br /> and DE, conserved in all SMC proteins, the latter probably      functions. Effects of mutations in the conserved C-terminal <br /> representing part of the Walker Type B motif involved in        domain were less severe. The S1045A mutation in the <br /> ATP&#226;&#8364;&#8220;Mg2&#226;&#171;&#185; binding. We mutated Ser1045 from the LSGG              conserved LSGG motif enabled us to separate repair from <br /> sequence to either alanine or threonine (rad18S1045A or         essential functions of the protein, since it only affected <br /> rad18S1045T), and Asp1072 to alanine (rad18D1072A).             the former. <br /> Overexpression of these mutated genes had no significant <br /> effect on the viability of wild-type cells. However, the        Overexpression of damage response genes <br /> rad18S1045A mutant plasmid sensitized the cells to both         rescues the dominant-negative phenotype <br /> &#206;&#179;- and UV-irradiation damage, whereas rad18S1045T               We have looked for genetic interactions between rad18 <br /> had less effect and rad18D1072A had minimal                     and other genes by examining their ability to suppress the <br /> effect (Figure 5B). The S1045A and S1045T plasmids              dominant-negative effects of the K129E mutation in the <br /> slightly increased the &#206;&#179;-irradiation sensitivity of the         ATP-binding site. If the normal and mutant cDNAs were <br /> rad18-X mutant (Figure 5C). Importantly, however, the           co-overexpressed from separate plasmids with the same <br /> rad18S1045A mutant plasmid, but neither of the others,          nmt1 promoter (in pRep41 and 42), the normal plasmid <br /> was able to rescue the lethality of the rad18 deletion          was able to rescue the growth-inhibitory effects of the <br /> strain (Figure 5A). This significant result shows that          mutant plasmid (Figure 6C). We used this co-expression <br /> the S1045A mutation is able to separate the repair from         system to investigate whether several other genes (rad1, <br /> the essential phenotype of the cells. It is possible that the   rad2, rad3, rad17, pcn1 and cdc6) could also rescue the <br /> rad18 mutant plasmid rescued the rad18 deletion because         adverse effects of the mutant plasmid. We found that <br /> the mutant gene was overexpressed. However, in results          when either the cell cycle checkpoint genes rad1 or rad3, <br /> to be presented in detail elsewhere, we have eliminated         the rad2 gene encoding a structure-specific nuclease or the <br /> this possibility. We have integrated the S1045A mutation        cdc6 gene encoding DNA polymerase &#206;&#180; was overexpressed <br /> into the genomic rad18 gene, in which it is expressed at        together with rad18K129E, the growth inhibition was <br /> normal levels. This strain is viable and sensitive to           overcome, whereas pcn1 encoding PCNA and the check- <br /> radiation.                                                      point gene rad17 had no effect. In order to ensure that <br />    Coiled-coil motifs are amphipathic &#206;&#177;-helices character-      this was not merely the result of the second plasmid <br /> ized by heptad repeats with hydrophobic amino acids at          reducing the expression of the mutant gene, we used <br /> the first and fourth residues of each heptad (Lupas et al.,     Western blotting to determine the levels of expression <br /> 1991). We mutated Leu843, in the middle of a long heptad        after 14 h in the absence of thiamine. Since the genes <br /> repeat, either to proline (L843P), which would be expected      were also myc-tagged (in most cases), we were able to <br /> to destroy the coiled-coil structure, or to phenylalanine       detect both Rad18 and the co-expressing proteins using <br /> (L843F), which is hydrophobic like leucine, but has a           an antibody directed against the myc epitope tag. The <br /> completely different structure. The latter mutation behaved     expression of the mutant rad18K129E was not affected <br /> like wild type in all respects (Figure 5B): it restored         by co-expression from the second plasmid (Figure 6D). <br /> viability to the rad18 deletion strain and rescued the          Our data therefore provide evidence for a genetic inter- <br /> sensitivity to DNA damage of the rad18-X cells                  action between rad18 and rad1, rad2, rad3 and cdc6. <br /> (Figure 5C). In contrast, L843P had a dramatic effect.          These interactions will be studied in detail in future work. <br /> Only a few wild-type cells overexpressing the plasmid <br /> were able to form colonies, which were very small, and          Discussion <br /> in liquid culture, most of the cells (80%) had the Rad18 <br /> phenotype. Viability of the deletion strain was not restored    Rad18&#226;&#8364;&#8220;Spr18 and other SMC protein complexes <br /> with this mutant plasmid.                                       Although our understanding of many DNA repair pro- <br />    The mutation in rad18-X, which is sensitive to UV- and       cesses has increased dramatically over the last few years, <br /> &#206;&#179;-irradiation and elevated temperatures, results in the         the functions of many DNA repair proteins are still poorly <br /> change R706C. We considered the possibility that this           understood. As shown by the scheme in Figure 1, Rad18 <br /> change might exert its effects because the mutant protein       plays an important role in several repair processes, but its <br /> contains a cysteine residue, which might form aberrant          exact function is unknown. The other SMC protein family <br /> disulfide bridges. We therefore mutated Arg706 to alanine       members are essential for chromosome condensation <br /> (R706A) and compared its effects with those of R706C.           (SMC2 and 4) and cohesion (SMC1 and 3) (see Hirano, <br /> Overexpression of R706C had no effect on either wild-           1998; Jessberger et al., 1998), although the role that these <br /> type or rad18-X cells and restored the viability of the         proteins play at the molecular level is poorly understood. <br /> deletion strain. The effects of R706A were, however, more       We have shown in this paper that Rad18 resembles other <br /> severe, the phenotype being quite similar to that of            SMC proteins in that it forms a heterodimeric complex <br /> rad18K129Q (see Figure 5B and C and Table I). These             with a closely related partner protein, Spr18, and Rad18&#226;&#8364;&#8220; <br /> findings ruled out the possibility that the phenotype of        Spr18 is part of a higher molecular weight complex. This <br /> rad18-X resulted from aberrant disulfide bridges, and           complex has a molecular weight &#226;&#172;&#381;1 MDa, is stable in <br /> rather implicated R706 as an important residue for              high salt, contains at least five other proteins and has <br /> Rad18 function.                                                 DNA-stimulated ATPase activity. Similarly, the condensins <br />    In summary, we have found a gradation in severity of         from Xenopus and S.pombe contain multiple subunits <br />  <br /> 1698 <br />                                                                                     Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> including, in Xenopus, XCAP-C and XCAP-E (Hirano <br /> et al., 1997) and in S.pombe, Cut3 and Cut14 (Tanaka <br /> et al., 1999), the orthologues of SMC2 and SMC4. <br /> Likewise the cohesins, which include SMC1 and SMC3 <br /> (Losada et al., 1998), and the dosage compensation <br /> complex from Caenorhabditis elegans, which contains the <br /> SMC protein DPY27 (Chuang et al., 1996), also contain <br /> multiple subunits. A complex designated RC-1, which was <br /> able to carry out strand exchange reactions, is composed of <br /> bovine SMC1 and SMC3 together with DNA polymerase &#206;&#181; <br /> and DNA ligase III (Jessberger et al., 1996). Both Rad18 <br /> and Spr18 have orthologues in the S.cerevisiae and <br /> C.elegans databases and we have isolated human ortholog- <br /> ues of both (E.M.Taylor, J.Moghraby and A.R.Lehmann,             Fig. 7. Models for involvement of the Rad18&#226;&#8364;&#8220;Spr18 complex in DNA <br /> unpublished results). In view of the similarity in structural    repair. The diagram shows the roles of SMC proteins in chromosome <br />                                                                  condensation and cohesion as proposed by Hirano (1999), and two <br /> composition between the Rad18&#226;&#8364;&#8220;Spr18 complex, con-                possible roles for Rad18&#226;&#8364;&#8220;Spr18 in repair of double-strand breaks. <br /> densin and cohesin, we consider Rad18&#226;&#8364;&#8220;Spr18 to be the            Black lines represent double-stranded DNA, and the grey symbols <br /> third conserved SMC complex in eukaryotes.                       represent SMC proteins with globular N- and C-termini and hinge <br />                                                                  regions, with extended coiled coils in between. <br /> Structure&#226;&#8364;&#8220;function relationships in Rad18 <br /> Structural studies on bacterial SMC-like proteins have           1990; Gregory et al., 1991). This mutation provides an <br /> been carried out recently by Melby and coworkers (Melby          opportunity for further investigation of the repair functions <br /> et al., 1998). MukB is the only SMC-like protein in              of Rad18 and its homologues. <br /> Escherichia coli and has an antiparallel homodimeric <br /> structure, with the globular domains attached by rigid           Function of the Rad18&#226;&#8364;&#8220;Spr18 complex <br /> coiled-coil domains to the hinge. The coiled-coil regions        The SMC protein complexes function in chromosome- <br /> form the basis for the dimerization. The hinge appeared          associated processes, such as condensation, cohesion and <br /> quite flexible, permitting a scissoring movement. It is          dosage compensation. It is thought that the putative <br /> probable that in the Rad18 complex, Rad18 and Spr18              scissoring action, which is possible because of the flexibil- <br /> form antiparallel heterodimers.                                  ity of the SMC dimer (Melby et al., 1998), enables the <br />    Site-directed mutagenesis studies (Figures 5 and 6) have      complexes to move DNA molecules relative to each other <br /> shown that the globular N-terminal domain and the coiled-        or to other cellular structures. The condensin complex <br /> coil regions are required for both repair and essential          from Xenopus is able to supercoil DNA in the presence <br /> functions of Rad18. This is consistent with the structure        of topoisomerase I (Kimura and Hirano, 1997) and to <br /> suggested above. If we assume that Rad18 forms a similar         effect topological reconfigurations of DNA molecules <br /> structure but as a heterodimer with Spr18, we can provide        (Kimura et al., 1999). The S.cerevisiae SMC1p and <br /> a ready explanation for the dominant-negative effects of         SMC2p proteins are able to bind to DNA, and this DNA- <br /> overexpressed mutant Rad18 (Figure 6 and Verkade et al.,         binding activity resides in the C-terminus of the molecule <br /> 1999). In the majority of the mutants in which the coiled-       (Akhmedov et al., 1998). We find that Rad18 binds <br /> coil domains remain intact, the overexpressed mutant             tightly to DNA-cellulose columns (our unpublished data), <br /> Rad18 protein will dimerize and thereby sequester the            suggesting that Rad18 is likely to be a DNA-binding <br /> cellular Spr18, preventing it from dimerizing with the           protein, which would be consistent with the DNA-stimu- <br /> endogenous protein. The L843P mutant probably causes             lated ATPase activity that we have found associated with <br /> local disruption of the &#206;&#177;-helical structure of the more          the Rad18&#226;&#8364;&#8220;Spr18 complex. We previously suggested that <br /> C-terminal coiled coil, but the more N-terminal coiled-          Rad18 was involved in recombinational repair processes <br /> coil domain is likely to be unaffected and still capable of      (Lehmann et al., 1995). Our current work together with <br /> interacting with Spr18.                                          published data on the functions of SMC proteins lead us <br />    Interestingly, mutations in the conserved C-terminal          to speculate that the function of the Rad18&#226;&#8364;&#8220;Spr18 complex <br /> globular domain had relatively mild effects. Particularly        is to bring together and/or hold broken DNA molecules <br /> interesting is the S1045A mutation (Figure 5), which             together in the vicinity of double-strand damage to allow <br /> destroyed the repair function of the protein, whilst retaining   repair by recombination to take place (see Figure 7). Such <br /> its essential function. This was quite surprising in view        damage could be either a double-strand break produced <br /> of the fact that the motif LSGG, encompassing Ser1045,           directly by ionizing radiation, or a UV photoproduct <br /> is conserved in all SMC proteins (with the exception of          opposite a gap or stalled replication fork generated during <br /> Spr18, which contains the sequence QSGG). It is also             replication of UV damage. We envisage that the Rad18&#226;&#8364;&#8220; <br /> found in a large family of ATP transporter proteins (Ames        Spr18 complex either holds the ends of the break together <br /> et al., 1992), in which it has been proposed that the LSGG       or alternatively holds the broken chromosome in register <br /> motif forms an important linker region between different         with its sister or homologue to allow the Rhp51 RecA <br /> domains. In the latter protein family the serine residue is      homologue to initiate strand exchange. In this context the <br /> essential for the transporter function of the S.cerevisiae       Rad18&#226;&#8364;&#8220;Spr18 complex presumably plays a complementary <br /> Ste6p protein involved in transport of the a-factor phero-       role to the cohesins, which are known to hold sister <br /> mone (Browne et al., 1996), and it is mutated in the CFTR        chromatids together, and are also required for a normal <br /> protein in several cystic fibrosis patients (Kerem et al.,       response to radiation, since mutants in S.pombe rad21, a <br />  <br />                                                                                                                             1699 <br />  M.I.Fousteri and A.R.Lehmann <br />  <br />  <br /> non-SMC component of cohesin, are sensitive to radiation.                      Immunoprecipitation <br />                                                                                For these experiments we constructed an S.pombe strain myc-tagged in <br /> Recent data suggest that the cohesin complex is present                        the genome (myc-Rad18). A fragment of DNA containing 950 bp of <br /> at specific sites along the chromosome arms rather than                        rad18 upstream sequence and the coding sequence up to the NcoI site <br /> coating the lengths of the chromosomes (Blat and                               at nt 1760 was cloned into the integrating vector pSta18 (Carr et al., <br /> Kleckner, 1999; Tanaka et al., 1999). In the context of                        1989), which contains the sup3.5 tRNASerUGA insert (Hottinger, 1982). <br /> recombinational repair of double-strand DNA damage, it                         The His6-myc2 coding sequence was inserted as an NdeI fragment from <br />                                                                                pGEMMH (Craven et al., 1998) into an NdeI site engineered at the start <br /> may be that cohesins are necessary to maintain the overall                     of the rad18 ORF. This plasmid was transformed into S.pombe strain <br /> cohesion of the sister chromatids, whereas the Rad18&#226;&#8364;&#8220;                          501 (leu1.32 ura4.D18 ade6.704) and selection was applied for growth <br /> Spr18 complex has a more specific role at sites of damage.                     in the absence of adenine. (The ade6.704 mutation in the 501 strain is <br /> The suggestion of a role for Rad18 in recombination is                         suppressed by sup3.5.) Ade&#226;&#171;&#185; colonies were grown and analysed for the <br /> supported by the 4-fold reduction in intrachromosomal                          integration of the tagged sequence by Southern blotting and immuno- <br />                                                                                blotting. A similar procedure was followed for the S.pombe rad18-X <br /> recombination recently reported in a Rad18 mutant of                           myc-tagged strain (myc-Rad18-X). <br /> Arabidopsis thaliana (Mengiste et al., 1999). A more                              For immunoprecipitations, crude extract containing 2 mg of total <br /> complex role for Rad18 in response to DNA damage has                           protein was incubated overnight with either the anti-Spr18 antibody or <br /> been suggested by the work of Verkade et al. (1999), who                       the 9E10 anti-myc antibody in ab binding buffer [40 mM HEPES pH 7.8, <br />                                                                                2 mM EDTA, 2 mM EGTA, 60 mM &#206;&#178;-glycerophosphate, 1 mM NaF, <br /> provided evidence to suggest that under some circum-                           0.5 mM phenylmethylsulfonyl fluoride (PMSF), 0.2 mM dithiothreitol <br /> stances Rad18 was required to maintain a checkpoint                            (DTT), 0.1 mM orthovanadate, 10% glycerol and protease inhibitor <br /> following DNA damage, although establishment of the                            cocktail, as described below] in the presence of different concentrations <br /> checkpoint was clearly normal. Our finding of a genetic                        of NaCl. The immune complexes were captured on protein G&#226;&#8364;&#8220;agarose <br />                                                                                beads, washed several times, boiled with gel loading buffer and elec- <br /> interaction between some of the checkpoint genes and                           trophoresed in 10% SDS&#226;&#8364;&#8220;PAGE gels. They were blotted onto PVDF <br /> rad18 is consistent with these observations. We obtained                       membrane filters and probed with the anti-myc antibody. <br /> no evidence, however, for physical association in the <br /> Rad18&#226;&#8364;&#8220;Spr18 complex of several checkpoint proteins                             Purification of the Rad18 complex <br /> tested. The relationship of Rad18 to the checkpoint                            For purification, 30 l of exponentially growing myc-Rad18 cells were <br />                                                                                harvested, washed and the cell pellet resuspended in an equal volume <br /> machinery is likely, therefore, to be complex and indirect.                    of lysis buffer A (40 mM HEPES pH 7.8, 0.4 M KCl, 80 mM <br />    Both Rad18 and Spr18 are essential for cell proliferation.                  &#206;&#178;-glycerophosphate, 12 mM NaF, 5 mM EGTA, 5 mM EDTA, 1 mM <br /> Our results in Figure 4D and Figure 6 show that in the                         DTT, 1 mM PMSF, 0.1 mM orthovanadate, 10% glycerol, and a protease <br /> absence of functional Spr18, or when dominant-negative                         inhibitor cocktail consisting of 5 &#194;&#181;g/ml each of trypsin inhibitor, <br /> mutant rad18 constructs are overexpressed, cells appear                        pepstatin, leupeptin, aprotinin, 10 &#194;&#181;g/ml bestatin and E-64, and <br />                                                                                50 &#194;&#181;g/ml chymotrypsin inhibitor). The cells were snap-frozen in liquid <br /> to be unable to replicate their DNA, but nevertheless they                     nitrogen and lysed using a blender, following addition of copious <br /> continue to grow, lay down septa and develop grossly                           quantities of dry ice. The lysate was clarified by centrifugation and <br /> abnormal morphologies. This is indicative of a role both                       subjected to ammonium sulfate precipitation. The 15&#226;&#8364;&#8220;45% cut containing <br /> in DNA replication and in mitotic control. Occasional                          the Rad18 protein was dissolved in lysis buffer and dialysed overnight <br />                                                                                against the same buffer. The extract containing ~300 mg of protein was <br /> double-strand breaks occur during DNA replication and                          applied in multiple injections to a Superdex 200, 10/30 FPLC column <br /> these are thought to be repaired by recombination. Cells                       (Pharmacia Biotech). Fractions of 0.25 ml were collected and analysed <br /> deficient in the recombination-repair genes rhp51 or rhp54                     by Western blotting using the anti-myc monoclonal antibody. The high- <br /> grow poorly (Muris et al., 1993, 1996), presumably                             molecular-weight fractions containing the Rad18 protein were pooled <br /> because they have difficulty in dealing with damage arising                    and fractionated on heparin&#226;&#8364;&#8220;Sepharose Cl-6B (Pharmacia Biotech) <br />                                                                                equilibrated in buffer B/0.1 M KCl (25 mM HEPES pH 7.8, 2 mM <br /> during replication. Double mutants of rhp51 or rhp54 with                      EGTA, 3 mM MgCl2, 0.1 mM DTT, 0.5 mM PMSF, 12% glycerol). <br /> the DNA damage checkpoint genes are inviable (Muris                            After extensive washing, proteins were eluted with buffer B/0.2, 0.4, <br /> et al., 1996), showing that checkpoint-mediated mitotic                        0.6 and 1 M KCl. The 0.6 M KCl eluate that contained the Rad18 <br /> delay is necessary for the cell to deal with this kind of                      protein was incubated overnight at 4&#194;&#176;C with protein G&#226;&#8364;&#8220;agarose beads <br />                                                                                (Pharmacia Biotech) crosslinked with anti-myc antibody in buffer B/ <br /> replicative stress. We propose that the inviability of rad18                   0.3 M KCl, containing 0.4 mM EDTA and protease inhibitor cocktail. <br /> and spr18 mutants arises from their involvement in linking                     The beads were washed once with the same buffer, four times in 10 vol. <br /> replication, repair and mitotic control. Further work will                     of buffer B/0.5 M KCl, 0.1% NP-40, and twice in buffer B/0.1 M KCl, <br /> be necessary to unravel the origin of these complex                            0.01% NP-40. Bound proteins were eluted twice at 30&#194;&#176;C for 1 h each <br /> phenotypes.                                                                    with peptide corresponding to the myc epitope (AEEQKLISEEDL) at <br />                                                                                3 mg/ml in buffer B/0.1 M KCl, 0.2 mg/ml insulin and 1 &#194;&#181;g/ml aprotinin <br />                                                                                and E64. The eluted fractions were dialysed twice in buffer B containing <br />                                                                                0.1 M KCl and 15% glycerol to eliminate the peptide. The proteins were <br /> Materials and methods                                                          separated on 10% SDS&#226;&#8364;&#8220;PAGE and stained with colloidal Coomassie <br />                                                                                Brilliant Blue (Brilliant Blue G, colloidal concentrate; Sigma). Alternat- <br /> Antibodies                                                                     ively they were analysed by immunoblotting with antibodies specific for <br /> The 9E10 anti-myc monoclonal antibody used for immunoblotting was              different DNA damage response proteins. <br /> obtained from Santa Cruz. For immuno-affinity chromatography we                   In experiments to estimate the size of the complex, pooled Superdex <br /> used supernatants from CRL-1729 hybridoma cells obtained from the              200 fractions that contained the high-molecular-weight peak, derived as <br /> American Type Culture Collection, and cultured in RPMI with 10%                above, were further subjected to gel filtration on a Superose 6 FPLC <br /> fetal calf serum. The supernatant was bound to protein G&#226;&#8364;&#8220;agarose,              column (Pharmacia Biotech). In this case 0.5 ml fractions were collected <br /> extensively washed and covalently crosslinked using dimethylpimel-             and analysed by Western blotting. The column was calibrated with <br /> imidate (Harlow and Lane, 1988). Polyclonal antibodies to Rad18                HMW Gel Filtration Calibration Markers (Pharmacia Biotech). <br /> and Spr18 protein were raised in sheep against the peptides <br /> VYSLAKKEEEYKLLWEQSRE and RKREREILQNKNQGQSTLNS-                                 ATPase assay <br /> LKDR, respectively. These antibodies identified the corresponding              For assaying ATPase activity, elution from the affinity column with the <br /> in vitro translated proteins. They also detected proteins of the appropriate   myc peptide was carried out at 30&#194;&#176;C for 20 min only, and the eluate <br /> size in cell extracts, which were not detected by the corresponding pre-       was dialysed against buffer B containing 0.05 M KCl. Protein eluates <br /> immune sera. Antibodies against other damage response proteins were            were then incubated for 2 h at 30&#194;&#176;C in the presence of 0.1 &#194;&#181;Ci [&#206;&#179;-32P]ATP <br /> generously supplied by Drs T.Caspari, J.Murray and A.M.Carr.                   (7000 Ci/mmol; ICN) in a 10 &#194;&#181;l reaction volume containing 25 mM <br />  <br />  <br />  <br />  <br />  <br />  <br /> Tris&#226;&#8364;&#8220;HCl pH 7.5, 4 mM MgCl2, 20 &#194;&#181;M ATP, 1 mM DTT and                          <br /> 50 &#194;&#181;g/ml BSA. The reaction was stopped by adding 5 &#194;&#181;l of 0.5 M                 <br /> EDTA pH 8.0. One microlitre of each reaction mixture was then spotted          <br /> on a PEI-cellulose thin layer chromatography (TLC) plate (Sigma)             <br /> and developed in 0.75 M KH2PO4. Radiolabelled ATP and inorganic              <br /> phosphate were quantitated on a Storm 840 PhosphorImager (Molecular             <br /> Dynamics). DNA-dependent ATPase activity was assayed in the presence          <br /> or absence of double- or single-stranded DNA at a final concentration          <br /> of 15 ng/&#194;&#181;l.                                                                   <br /> Site-directed mutagenesis                                                    <br /> A rad18 cDNA clone was constructed by PCR and subcloning from                <br /> total S.pombe cDNA. The internal NdeI site was destroyed by site-directed <br />                                                                             <br /> mutagenesis, and after removing PCR errors by further mutagenesis, the <br />                                                                              <br /> cDNA was cloned as an NdeI&#226;&#8364;&#8220;SalI fragment into the pRep41 MH, and <br />                                                                               <br /> pRep42 MH epitope-tagging expression vectors (Craven et al., 1998). <br /> Mutations in the cDNA were introduced at specific sites using the           <br /> procedure of Kunkel et al. (1987), and all the mutant clones were             <br /> sequenced completely to ensure that only the desired mutations had been      <br /> introduced. We regularly found that subcloning of rad18 constructs was        <br /> unsuccessful when the ligated DNA was transformed into a variety of          <br /> commonly used E.coli strains. We therefore transformed the ligation           <br /> mixes directly into S.pombe and successfully obtained the desired          <br /> products.                                                                 <br />                                                                             <br /> Cell <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>survival</span> and complementation of the essential                           <br /> phenotype                                                                      <br /> Mutant constructs were introduced either into wild-type or rad18-X cells     <br /> in the presence of thiamine to maintain the nmt1 promoter on the plasmid     <br /> in the repressed state. For survival experiments, cells were grown in the     <br /> absence of thiamine for 16 h to induce steady levels of expression from       <br /> the nmt1 promoter. They were then exposed to UV- or &#206;&#179;-irradiation and       <br /> 1000 cells were plated out as described previously (Lehmann et al.,         <br /> 1995). Wild-type cells were plated on minimal medium, and rad18-X            <br /> on YEP. For effects on cell viability, wild-type cells were grown for      <br /> 12 h in the absence of thiamine. Samples were then plated both in the          <br /> presence and absence of thiamine. Other samples were taken every hour        <br /> for the next 14 h and the cells were counted, fixed in methanol and             <br /> stained with DAPI and calcofluor.                                             <br />    To investigate the ability of the mutant constructs to rescue the         <br /> inviability of a rad18 deletion, rad18 mutants in pRep41MH (Leu&#226;&#171;&#185;) <br />                                                                                <br /> (Craven et al., 1998) were transformed into the rad18&#226;&#171;&#185;/rad18::ura4 <br />                                                                             <br /> diploid strain in the presence of thiamine. Colonies were picked and <br /> induced to sporulate by growth on low nitrogen plates. The spores were       <br /> digested with helicase, and plated (without thiamine) in the absence of       <br /> leucine, or in the absence of both uracil and leucine. Rescue of the            <br /> deletion strain was indicated by growth in the absence of both uracil           <br /> and leucine.                                                                  <br />   <br /> </body></html>