<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                           Network rewiring important <br />                            mechanism gene essentiality change <br />     SUBJECT AREAS: <br />                            Jinho Kim1,2, Inhae Kim1, Seong Kyu Han1, James U. Bowie3 &amp; Sanguk Kim1,2 <br />      COMPUTATIONAL <br />         BIOLOGY AND <br />      BIOINFORMATICS        1 <br />                             Division Molecular Life Science, 2Division ITCE, Pohang University Science Technology, Pohang 790-784, Korea, <br />     SYSTEMS BIOLOGY        3 <br />                             Department Chemistry Biochemistry, UCLA-DOE Institute Genomics Proteomics, University California Los Angeles, <br />            BIOPHYSICS      Los Angeles, California 90095-1570, USA. <br />  <br /> MOLECULAR EVOLUTION <br />                            Gene essentiality changes crucial organismal evolution. However, unclear essentiality of <br />                            orthologs varies species. We investigated underlying mechanism gene essentiality changes <br />            Received        yeast mouse based framework network evolution comparative genomic analysis. <br />   10 September 2012        We yeast nonessential genes essential mouse network connections rapidly <br />                            increase engagement protein complexes. The increased interactions allowed previously <br />            Accepted        nonessential genes members vital pathways. By accounting changes gene essentiality, we <br />     23 October 2012        firmly reestablished centrality-lethality rule, proposed relationship essential genes and <br />                            network hubs. Furthermore, discovered number connections associated essential and <br />           Published <br />                            non-essential genes depends essential ancestral species. Our study describes the <br />   29 November 2012         time network evolution occurs change gene essentiality. <br />  <br />  <br />  <br />  <br />                            G <br />    Correspondence and <br />                                     ene essentiality varies species dramatic phenotypic changes gene can <br />                                     undergo1. For instance, deletion MAP kinase kinase1 (Map2k1) did affect fitness yeast, but <br />   requests materials <br />                                     loss function caused embryonic lethality mouse2,3. In contrast, deletion serine/threonine- <br />  addressed    protein kinase ICK caused lethality yeast apparent phenotypic effect mouse4. Generally, orthologs <br />   S.K. (sukim@postech.     considered deliver function different species. Given case, how <br />                   ac.kr)   does essentiality functional gene change species? <br />                               The C-L rule explains highly connected proteins network likely essential cell <br />                            viability5. However, weak correlation network connections gene essentiality led controver- <br />                            sies C-L rule6&#226;&#8364;&#8220;8. A system-level understanding gene essentiality change chance <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />                            understand design principles key biological processes provide opportunity predicting important <br />                            gene functions. <br />                               Here, investigated mechanisms gene essentiality changes framework network expansion <br />                            evolution. We hypothesized network rewiring significant effect gene essentiality changes <br />                            rewiring interactions enables genes integrated new pathways9 new interactions can <br />                            increase probability involved vital biological process. <br />  <br />                            Results <br />                            Gene essentiality frequently changes evolution. We significant portion 2,144 mouse <br />                            genes yeast orthologs changed essentialities mouse yeast (Fig. 1a). We arranged the <br />                            orthologous pairs yeast mouse genes phenotypic groups based changing essentiality <br />                            patterns. We 91 genes essential yeast mouse (E2E), 246 genes nonessential yeast but <br />                            essential mouse (N2E), 659 genes essential yeast nonessential mouse (E2N), 1,149 genes are <br />                            nonessential yeast mouse (N2N). The list yeast mouse gene orthologs essentiality <br />                            measurements accessed Supplementary Table S1. <br />  <br />                            Increase network connections explains gene essentiality changes. We hypothesized frequent gene <br />                            essentiality changes observed related interaction rewiring, allows genes integrate into, or <br />                            separate <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span>, important biological pathways10&#226;&#8364;&#8220;13. To test hypothesis, examined increase network <br />                            connections yeast mouse protein-protein interaction (PPI) networks (Fig. 1b). It suggested <br />                            number protein interactions highly correlated complexity organism14,15. Protein <br />                            interactions measured experiments yeast mouse separately network connections <br />                            yeast mouse compared ortholog mapping (see Materials Methods). We that <br />  <br />  SCIENTIFIC REPORTS | 2 : 900 | DOI: 10.1038/srep00900                                                                                                   1 <br />                                                                                                         www.nature.com/scientificreports <br />  <br />  <br />  <br />  <br /> Figure 1 | Increase network connections gene essentiality changes yeast mouse. (a) Gene essentiality changes yeast and <br /> mouse. The numbers essential nonessential genes yeast (left) mouse (right) presented. (b) A network evolution model describing gene <br /> essentiality changes. (c) Network connections phenotypic classes. The panels display average number network connections in <br /> yeast mouse respectively. Error bars indicate standard error. The panel shows fold increase average number connections mouse <br /> relative yeast. <br />  <br /> all classes essentiality changes increased average             genes compared genes (p 5 6.76 3 1027; Fig. 1c), whereas <br /> network connections mouse relative yeast,              increase E2N genes significantly smaller the <br /> increase quite different classes. In particular, N2E           average (p 5 1.30 3 1024). <br /> genes highest increase network connections,                  Because large evolutionary distance yeast mouse, <br /> E2N genes smallest increase phenotypic                 investigated species pairs diverged closer <br /> groups. The increase connectivity significant N2E               distance yeast mouse. We genes <br />  <br />  <br />  <br />  <br /> Figure 2 | Comparison network connections various species. (a) Increase network connections complexity organisms. The fold increase <br /> in number connections relative yeast plotted. (b) Increase network connections various species pairs. The fold increases network <br /> connections worm yeast, chicken worm, mouse chicken presented. <br />  <br />  <br /> SCIENTIFIC REPORTS | 2 : 900 | DOI: 10.1038/srep00900                                                                                                    2 <br />                                                                                                   www.nature.com/scientificreports <br />  <br />  <br />  <br />  <br /> Figure 3 | Functional enrichment analysis essentiality changing genes. (a) Comparison Biological processes N2E, E2N, N2N genes those <br /> of E2E genes yeast mouse. (b) Enrichment biological processes phenotypic groups. Gene ontology terms significantly <br /> enriched (p , 0.001) N2E genes presented. (c) Network connections Map2k1 yeast, worm, chicken, mouse. Interaction partners Map2k1 <br /> in yeast (left) mouse (right) depicted orthologs connected dotted lines. <br />  <br /> SCIENTIFIC REPORTS | 2 : 900 | DOI: 10.1038/srep00900                                                                                            3 <br />                                                                                                      www.nature.com/scientificreports <br />  <br />  <br />  Table 1 | Developmental processes N2E genes mouse <br /> Developmental process                                                                       N2E genes <br /> blastocyst development                        Cul3, Smarcb1, Ada, Sp3, Junb <br /> mammary gland development                     Phb2, Atp7b <br /> in utero embryonic development                Prmt1, Sin3a, Cul3, Slc30a1, Ccnb2, Smarcb1, Msh2, Ube2a, Mapk1, Myo1e, Mecom, Sp3, Ccnb1, <br />                                                Plcg1, Junb, Lig4, Fgfr1, Ada, Hsf1, Map2k1 <br /> immune development                     Exo1, Msh2, Maea, Ung, Sp3, Rps19, Slc11a2, Xrcc6, Lig4, Blm, Sgpl1, Msh6, G6pdx, Ccnb2, Mlh1, <br />                                                Myo1e, Tcea1, Ada, Hells, Sod2, Dnaja3 <br /> hemopoietic lymphoid organ                 G6pdx, Ccnb2, Msh2, Myo1e, Maea, Sp3, Rps19, Slc11a2, Lig4, Blm, Sgpl1, Tcea1, Ada, Hells, Sod2, <br /> development                                    Dnaja3 <br /> positive regulation developmental          Hmgb1, Junb, Lig4, Xrcc6, Fgfr1, Ada, Map2k1, Mapk14 <br /> process <br /> tube development                              Phb2, Hmgb1, Timeless, Sp3, Ppp3r1, Ptges3, Fgfr1, Ada <br /> gland development                             Phb2, Atp7b, Fgfr1 <br /> chordate embryonic development                Msh2, Ube2a, Phgdh, Sp3, Junb, Lig4, Fgfr1, Map2k1, Prmt1, Sin3a, Cul3, Slc30a1, Ccnb2, Smarcb1, <br />                                                Mapk1, Myo1e, Mecom, Ccnb1, Plcg1, Ada, Hsf1, Atm <br /> embryonic development ending               Msh2, Ube2a, Phgdh, Sp3, Junb, Lig4, Fgfr1, Map2k1, Prmt1, Sin3a, Cul3, Slc30a1, Ccnb2, Smarcb1, <br /> birth egg hatching                          Mapk1, Myo1e, Mecom, Ccnb1, Plcg1, Ada, Hsf1, Atm <br /> blood vessel development                      Myo1e, Mapk1, Sphk2, Vezf1, Junb, Ppap2b, Sgpl1, Atg5, Fgfr1, Map2k1, Mapk14 <br />  <br />  <br /> gradually increased network connections course evolu-            Many N2E genes integrated BPs vital the <br /> tion (Fig. 2a) N2E genes increased network connections fastest            development multicellular organisms (Fig. 3b Table 1). <br /> among phenotypic groups comparison closer species             Interactions N2E proteins highly enriched developmental <br /> (Fig. 2b). These results suggest essential genes unicellular organ-   processes single misregulation cause embryonic leth- <br /> isms nonessential multicellular organisms, fail             ality. For example, expanded network connections Map2k1, an <br /> rapidly expand network connections course evolution.          N2E gene, involved key pathways multicellular organisms <br />                                                                               (Supplementary Table S3). Map2k1 participates placenta develop- <br /> N2E genes integrated vital biological pathways. Next             ment mouse newly evolved interactions. It interaction <br /> asked increased connections create new connections             partners yeast PPI network, network connections <br /> core biological functions increased essentiality. It          increased 23 mouse PPI network (Fig. 3c). Consequently, <br /> been suggested genes essential participating            deletion Map2k1 lethal yeast, causes embryonic <br /> core pathways9, evidence hypothesis heretofore          lethality mice2,3. Among interaction partners Map2k1 is <br /> lacking. We new interactions gained network expan-             epidermal growth factor receptor, EGFR, regulates epi- <br /> sion tend cause integration N2E genes vital pathways         dermal growth factor pathway crucial cell growth and <br /> essential genes (Fig. 3a). Functional enrichment analysis gene             morphogenesis16. <br /> ontology biological processes (BPs) carried interac- <br /> tions formed N2E, E2N, N2N, E2E genes yeast mouse               Gene essentiality change related protein complex member- <br /> (Supplementary Table S2). The analysis reveals interactions           ship. We asked N2E genes quickly increased their <br /> N2E genes gained network expansion dramatically                     network connections molecular level. We examined mem- <br /> increased participation essential BPs E2E genes. Speci-           bership changes protein complexes yeast mouse, and <br /> fically, yeast, interactions N2E genes share 50% BPs E2E        N2E genes showed highest rate engaging protein <br /> genes, mouse, fraction sharply increases 74%. Whereas           complexes groups (p 5 3.55 3 10210; Fig. 4a). For <br /> interactions E2N genes share 77% BPs E2E genes yeast,           example, Map2k1 member protein complex yeast, <br /> the fraction decreases 59% mouse.                                       member Ksr1 scaffold protein complex in <br />  <br />  <br />  <br />  <br /> Figure 4 | Protein complex membership evolution gene essentiality changes. (a) Fraction genes newly involved protein complexes are <br /> compared phenotypic group. (b) Evolutionary rates (dN/dS) phenotypic group yeast. The evolutionary rates (dN/dS) calculated <br /> from nucleotide sequences 3,392 orthologous open reading frames (ORFs) Saccharomyces species including S. cerevisiae, S. paradoxus, <br /> S. mikatae, S. bayanus. <br />  <br />  <br /> SCIENTIFIC REPORTS | 2 : 900 | DOI: 10.1038/srep00900                                                                                             4 <br />                                                                                                             www.nature.com/scientificreports <br />  <br />  <br />                                                                                      information flow biological networks. Betweenness centrality is <br />                                                                                      measure node&#226;&#8364;&#8482;s centrality network equal number of <br />                                                                                      shortest paths pairs nodes pass node. <br />                                                                                      Proteins high betweenness centrality tend interact many <br />                                                                                      different functional groups21 important controlling <br />                                                                                      information flow network22,23. We discovered the <br />                                                                                      betweenness centrality N2E genes higher N2N <br />                                                                                      E2N genes number network connections <br />                                                                                      (Fig. 5). Of groups, E2E genes highest betweenness <br />                                                                                      centrality importance information flow PPI <br />                                                                                      network. However, N2E genes showed dramatic increase be- <br />                                                                                      tweenness centrality highly connected (.16 network <br />                                                                                      connections). The increased betweenness centrality affects func- <br />                                                                                      tional role N2E genes reforming modular architecture the <br />                                                                                      PPI network. Although N2E N2N genes nonessential <br />                                                                                      yeast, extensive rewiring network connections N2E genes <br /> Figure 5 | Information flow essentiality changing genes.                       complex organisms enables connect various <br /> Betweenness centrality phenotypic groups presented                functional modules, controlling information flow around <br /> number network connection. Circles correspond mean betweenness             newly evolved essential genes. <br /> of data points interval log scale. Error bars indicate standard error.      Our findings evolution networks allow firmly <br />                                                                                      reestablish C-L rule showing highly connected genes in <br /> multicellular organisms17. This suggests protein complex mem-                   network essential network rewiring prop- <br /> bership important mechanism expanding network                          erly considered. The C-L rule debated appar- <br /> connections affect gene essentiality changes18,19.                          ent weak correlation network connection gene <br />    To increase network connections rapidly, N2E genes                       essentiality6&#226;&#8364;&#8220;8. We suspected poor correlation have <br /> acquired new interaction sites fast adaptive evolution. To                   occurred evolution gene essentiality consid- <br /> test possibility, examined evolutionary rates E2E, N2E,               ered previously (Fig. 6). According C-L rule, essential genes in <br /> E2N, N2N genes various yeast species, discovered                     yeast relatively high connectivity. If rewiring leads to <br /> N2E genes rapidly evolved. Evolutionary rates yeast genes                    nonessential mouse (E2N), connections decrease <br /> were calculated ratio nonsynonymous substitutions (dN)                  relative essential mouse genes (see above), evolu- <br /> synonymous substitutions (dS) complete genomes                      tionary time occurred descend level nones- <br /> Saccharomyces species20. As shown Fig. 4b, N2E genes                       sential gene nonessential yeast (N2N). Similarly, if <br /> rapid evolutionary rate compared E2E (p 5 5.67 3 1025)                        nonessential gene essential mouse (N2E), connec- <br /> E2N genes (p 5 2.79 3 1027). Interestingly, evolutionary rates                   tions generally added rapidly (see above), insufficient evolu- <br /> of N2E N2N genes similar (p 5 0.82). The rapid evolution-                   tionary time occurred achieve connection level a <br /> ary rate N2N genes probably low selective pressure                   gene essential yeast remained essential in <br /> nonessential genes.                                                                  mouse. As shown Fig. 6, consider genes con- <br />                                                                                      served essentiality yeast mouse, correlation between <br /> Discussion                                                                           connectivity essentiality extremely high (R250.97). <br /> Having confirmed network evolution influences gene essential-                   In words, set common starting point con- <br /> ity changes, asked interaction rewiring impacted                      nectivity race, essential genes acquire connections than <br />  <br />  <br />  <br />  <br /> Figure 6 | The influence evolutionary history C-L rule. When genes considered regardless evolutionary history (left panel), the <br /> correlation connectivity essentiality relatively weak. If genes conserved essentiality considered (right panel), correlation is <br /> dramatically improved. Error bars indicate standard error. <br />  <br /> SCIENTIFIC REPORTS | 2 : 900 | DOI: 10.1038/srep00900                                                                                                         5 <br />                                                                                                                        www.nature.com/scientificreports <br />  <br />  <br /> non-essential genes. Thus, C-L rule does explain relationship                     various high-throughput data28. Mouse protein complex data obtained from <br />                                                                                           CORUM database lists 454 manually curated mouse complexes29. <br /> between gene essentiality network connection. It suggests <br /> that interaction rewiring properly considered predict-                      Calculation evolutionary rate (dN/dS). The evolutionary rates (dN/dS) the <br /> ing gene essentiality genome-wide scale mapping                       genes Saccharomyces species computed using nucleotide sequences for <br /> orthologs24.                                                                              3,392 orthologous open reading frames (ORF) S. cerevisiae, S. paradoxus, <br />    The relationship gene essentiality changes                             S. mikatae, S. bayanus20. A maximum likelihood phylogeny constructed <br />                                                                                           ORF using PHYLIP30. Then, number synonymous nucleotide <br /> increase network connections true relatively young                         substitutions synonymous site (dS) number nonsynonymous <br /> genes yeast mouse. Among mouse                              substitutions nonsynonymous site (dN) calculated using PAML <br /> genes yeast orthologs, 2,189 essen-                     program31. <br /> tial (X2E) 12,207 nonessential (X2N). We X2E <br /> has significantly network connections X2N mouse                          1. Chen, W. H., Minguez, P., Lercher, M. J. &amp; Bork, P. OGEE: online gene <br /> PPI network (p52.16 3 10272). Meanwhile, yeast genes                               essentiality database. Nucleic Acids Res 40, D901&#226;&#8364;&#8220;906 (2012). <br /> mouse orthologs, 427 essential (E2X) 3,983                           2. Bissonauth, V., Roy, S., Gravel, M., Guillemette, S. &amp; Charron, J. Requirement for <br /> were nonessential (N2X). Similarly, E2X signifi-                           Map2k1 (Mek1) extra-embryonic ectoderm placentogenesis. <br />                                                                                               Development 133, 3429&#226;&#8364;&#8220;3440 (2006). <br /> cantly network connections N2X (p 5 5.33 3 10221).                              3. Nadeau, V. et al. Map2k1 Map2k2 genes contribute normal <br /> These biases network connections young genes suggest                               development syncytiotrophoblasts placentation. Development 136, <br /> genes engaging interactions likely essential.                               1363&#226;&#8364;&#8220;1374 (2009). <br /> When young genes arose, likely nonessential                          4. Anderson, S. J. &amp; Perlmutter, R. M. A signaling pathway governing early <br />                                                                                               thymocyte maturation. Immunol Today 16, 99&#226;&#8364;&#8220;105 (1995). <br /> because ancestral species survived share                      5. Jeong, H., Mason, S. P., Barabasi, A. L. &amp; Oltvai, Z. N. Lethality centrality in <br /> network connections parental genes9. As underwent                             protein networks. Nature 411, 41&#226;&#8364;&#8220;42 (2001). <br /> interaction rewiring, gained interactions                          6. Coulomb, S., Bauer, M., Bernard, D. &amp; Marsolier-Kergoat, M.-C. Gene essentiality <br /> essential chances member vital pathways.                              topology protein interaction networks. Proceedings. Biological sciences/ <br />                                                                                               The Royal Society 272, 1721&#226;&#8364;&#8220;1725 (2005). <br />    To knowledge, study highlights time                        7. Gandhi, T. K. B. et al. Analysis human protein interactome comparison <br /> interaction rewiring key evolution gene essentiality.                          yeast, worm fly interaction datasets. Nature Genetics 38, 285&#226;&#8364;&#8220;293 (2006). <br /> Relating network rewiring phenotypic changes improve                            8. Batada, N. N., Hurst, L. D. &amp; Tyers, M. Evolutionary physiological importance <br /> our understanding functional evolution genes.                                       hub proteins. PLoS computational biology 2, e88 (2006). <br />                                                                                           9. Chen, S., Zhang, Y. E. &amp; Long, M. New Genes Drosophila Quickly Become <br />                                                                                               Essential. Science 330, 1682&#226;&#8364;&#8220;1685 (2010). <br /> Methods                                                                                   10. Kim, J. et al. Rewiring PDZ domain-ligand interaction network contributed to <br /> Essential nonessential genes yeast mouse. <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>Phenotype</span> data mouse gene             eukaryotic evolution. PLoS Genet 8, e1002510 (2012). <br /> deletions obtained Mouse Genome Informatics (www.informatics.jax.org/).         11. Sun, M. G., Sikora, M., Costanzo, M., Boone, C. &amp; Kim, P. M. Network evolution: <br /> These phenotypes identified random gene disruption, gene trap                       rewiring signatures conservation signaling. PLoS Comput Biol 8, <br /> mutagenesis, targeted deletion25. Genes annotated essential phenotypes,        e1002411 (2012). <br /> embryonic lethality (MP: 0002080), prenatal lethality (MP: 0002081), <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>survival</span> postnatal   12. Chen, S. et al. Reshaping global gene expression networks sex-biased gene <br /> lethality (MP: 0002082), abnormal reproductive morphology (MP: 0002160),            expression integration young gene. EMBO J 31, 2798&#226;&#8364;&#8220;2809 (2012). <br /> abnormal reproductive physiology (MP: 0001919) classified essential        13. Chen, S. et al. Frequent recent origination brain genes shaped evolution of <br /> genes. All mouse genes classified nonessential genes. This process              foraging behavior Drosophila. Cell reports 1, 118&#226;&#8364;&#8220;132 (2012). <br /> identified 2,071 essential 12,928 nonessential mouse genes.                               14. Vogel, C. &amp; Chothia, C. Protein family expansions biological complexity. <br />    Gene essentiality data yeast manually compiled Comprehensive              PLoS Comput Biol 2, e48 (2006). <br /> Yeast Genome Database (http://mips.helmholtz-muenchen.de/genre/proj/yeast/)               15. Xia, K., Fu, Z., Hou, L. &amp; Han, J. D. Impacts protein-protein interaction <br /> and large-scale experiments26. The dataset contained 1,178 essential 4,904                domains organism network complexity. Genome Res 18, 1500&#226;&#8364;&#8220;1508 <br /> nonessential yeast genes.                                                                     (2008). <br />                                                                                           16. Hu, M. C. &amp; Rosenblum, N. D. Genetic regulation branching morphogenesis: <br />                                                                                               lessons learned loss-of-function phenotypes. Pediatr Res 54, 433&#226;&#8364;&#8220;438 <br /> Construction yeast mouse PPI networks. We constructed yeast mouse                  (2003). <br /> PPI networks integrating 22 protein interaction databases10: Bio-molecular         17. McKay, M. M., Ritt, D. A. &amp; Morrison, D. K. Signaling dynamics KSR1 <br /> Interaction Network Database (BIND), Human Protein Reference Database                     scaffold complex. Proc Natl Acad Sci U S A 106, 11022&#226;&#8364;&#8220;11027 (2009). <br /> (HPRD), Molecular Interaction database (MINT), DIP, IntAct, BioGRID,                  18. van Dam, T. J. &amp; Snel, B. Protein complex evolution does involve extensive <br /> Reactome, Protein-Protein Interaction Database (PPID), BioVerse, CCS-HI1,             network rewiring. PLoS Comput Biol 4, e1000132 (2008). <br /> comprehensive resource mammalian protein complexes (CORUM), IntNetDB,              19. Jensen, L. J., Jensen, T. S., Lichtenberg, U., Brunak, S. &amp; Bork, P. Co-evolution of <br /> Mammalian Protein-Protein Interaction Database (MIPS), Online Predicted                   transcriptional post-translational cell-cycle regulation. Nature 443, 594&#226;&#8364;&#8220;597 <br /> Human Interaction Database (OPHID), Ottowa, PC/Ataxia, Sager, Transcriptome,                  (2006). <br /> Complexex, Unilever, protein-protein interaction database PDZ-domains                 20. Hirsh, A. E., Fraser, H. B. &amp; Wall, D. P. Adjusting selection synonymous <br /> (PDZBase), protein interaction dataset literature. We removed low-             sites estimates evolutionary distance. Mol Biol Evol 22, 174&#226;&#8364;&#8220;177 (2005). <br /> confidence interactions supported direct experimental evidence. The <br />                                                                                           21. Dunn, R., Dudbridge, F. &amp; Sanderson, C. M. The use edge-betweenness <br /> resulting integrated PPI network comprises 101,777 interactions 11,043 <br />                                                                                               clustering investigate biological function protein interaction networks. BMC <br /> proteins. Based integrated PPI network, constructed yeast mouse <br />                                                                                               bioinformatics 6, 39 (2005). <br /> PPI networks ortholog mapping. The interactions transferred yeast and <br />                                                                                           22. Wagner, A. &amp; Fell, D. A. The small world inside large metabolic networks. Proc <br /> mouse orthologs interacting pair present. Orthologous gene <br />                                                                                               Biol Sci 268, 1803&#226;&#8364;&#8220;1810 (2001). <br /> pairs obtained Inparanoid database (http://inparanoid.sbc.su.se). Only <br />                                                                                           23. Jeong, H., Tombor, B., Albert, R., Oltvai, Z. N. &amp; Barabasi, A. L. The large-scale <br /> the 100% confidence orthologous pair ortholog group used analysis. <br />                                                                                               organization metabolic networks. Nature 407, 651&#226;&#8364;&#8220;654 (2000). <br /> The final yeast PPI network comprises 14,024 interactions 1,367 yeast <br />                                                                                           24. Deng, J. et al. Investigating predictability essential genes distantly <br /> proteins; mouse PPI network comprises 78,582 interactions 9,210 mouse <br />                                                                                               related organisms using integrative approach. Nucleic Acids Res 39, 795&#226;&#8364;&#8220;807 <br /> proteins. <br />                                                                                               (2011). <br />                                                                                           25. Liao, B.-Y. &amp; Zhang, J. Null mutations human mouse orthologs frequently <br /> Gene ontology analysis. To investigate bio-processes mediated interactions          result different phenotypes. Proceedings National Academy Sciences of <br /> E2E, N2E, E2N, N2N genes, analyzed GO annotations direct network                United States America 105, 6987&#226;&#8364;&#8220;6992 (2008). <br /> neighbors. We used DAVID27 gene set enrichment analysis. Statistically                26. Guldener, U. et al. CYGD: Comprehensive Yeast Genome Database. Nucleic <br /> overrepresented bio-process terms group analyzed fold                    Acids Res 33, D364&#226;&#8364;&#8220;368 (2005). <br /> enrichment calculated comparing frequencies genes GO                 27. Huang da, W., Sherman, B. T. &amp; Lempicki, R. A. Systematic integrative <br /> annotation gene group genome. The analyses conducted                 analysis large gene lists using DAVID bioinformatics resources. Nat Protoc 4, <br /> yeast mouse, separately. Only bio-processes overrepresented                44&#226;&#8364;&#8220;57 (2009). <br /> p-value lower 0.001 employed.                                                   28. Benschop, J. J. et al. A consensus core protein complex compositions for <br />                                                                                               Saccharomyces cerevisiae. Mol Cell 38, 916&#226;&#8364;&#8220;928 (2010). <br /> Protein complex data. We obtained yeast protein complex data curated               29. Ruepp, A. et al. CORUM: comprehensive resource mammalian protein <br /> consensus set catalogs 518 protein complexes combination                   complexes--2009. Nucleic Acids Res 38, D497&#226;&#8364;&#8220;501 (2010). <br />  <br />  <br /> SCIENTIFIC REPORTS | 2 : 900 | DOI: 10.1038/srep00900                                                                                                                              6 <br />                                                                                                                                 www.nature.com/scientificreports <br />  <br /> 30. Felsenstein, J. PHYLIP (phylogeny inference package). Version. 3.6.                      Additional information <br />     Distributed author, Department Genetics, University Washington,                 Supplementary Information accompanies paper http://www.nature.com/ <br />     Seattle. (2003).                                                                             scientificreports <br /> 31. Yang, Z. PAML: program package phylogenetic analysis maximum <br />     likelihood. Comput Appl Biosci 13, 555&#226;&#8364;&#8220;556 (1997).                                           Competing financial interests: The authors declare competing financial interests. <br />                                                                                                  License: This work licensed Creative Commons <br />                                                                                                  Attribution-NonCommercial-ShareAlike 3.0 Unported License. To view copy this <br />                                                                                                  license, visit http://creativecommons.org/licenses/by-nc-sa/3.0/ <br /> Acknowledgements                                                                                 How cite article: Kim, J., Kim, I., Han, S.K., Bowie, J.U. &amp; Kim, S. Network rewiring is <br /> This work supported Korean National Research Foundation gra <br /> </body></html>