<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Long-Chain Flavodoxin Protects Pseudomonas <br /> aeruginosa Oxidative Stress Host Bacterial <br /> Clearance <br /> Alejandro J. Moyano1, Romina A. Tobares1., Yanina S. Rizzi1., Adriana R. Krapp2, Juan A. Mondotte3, <br /> Jose&#194;&#180; L. Bocco4, Maria-Carla Saleh3, Ne&#194;&#180;stor Carrillo2, Andrea M. Smania1* <br /> 1 Centro Investigaciones en Qu&#196;&#177;&#194;&#180;mica Biolo&#194;&#180;gica Co&#194;&#180;rdoba (CIQUIBIC), CONICET, Departamento Qu&#196;&#177;&#194;&#180;mica Biolo&#194;&#180;gica, Facultad Ciencias Qu&#196;&#177;&#194;&#180;micas, Universidad <br /> Nacional Co&#194;&#180;rdoba, Co&#194;&#180;rdoba, Argentina, 2 Instituto Biolog&#196;&#177;&#194;&#180;a Molecular y Celular Rosario (IBR), CONICET, Facultad Ciencias Bioqu&#196;&#177;&#194;&#180;micas y Farmace&#194;&#180;uticas, <br /> Universidad Nacional Rosario, Rosario, Argentina, 3 Institut Pasteur, Viruses <span id='am-553' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-554' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-555' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-556' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-557' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-558' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-559' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-560' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-561' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-562' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-563' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-564' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-565' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-566' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-567' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-568' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-569' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-570' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-571' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-572' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-573' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-574' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-575' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-576' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-577' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-578' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-579' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-580' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-581' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-582' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-583' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-584' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-585' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-586' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-587' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-588' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-589' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-590' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-591' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-592' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-593' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-594' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-595' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-596' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-597' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-598' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-599' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-600' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-601' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-602' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-603' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-604' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-605' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-606' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-607' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-608' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-609' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-610' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-611' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-612' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-613' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-614' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-615' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-616' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-617' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-618' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-619' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-620' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-621' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-622' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-623' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-624' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-625' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-626' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-627' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-628' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-629' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-630' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>RNA Interference</span>, Centre National la Recherche Scientifique UMR3569, Paris, France, <br /> 4 Centro Investigaciones en Bioqu&#196;&#177;&#194;&#180;mica Cl&#196;&#177;&#194;&#180;nica e Inmunolog&#196;&#177;&#194;&#180;a (CIBICI), CONICET, Departamento Bioqu&#196;&#177;&#194;&#180;mica Cl&#196;&#177;&#194;&#180;nica, Facultad Ciencias Qu&#196;&#177;&#194;&#180;micas, Universidad <br /> Nacional Co&#194;&#180;rdoba, Co&#194;&#180;rdoba, Argentina <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Long-chain flavodoxins, ubiquitous electron shuttles containing flavin mononucleotide (FMN) prosthetic group, play an <br />      important protective role reactive oxygen species (ROS) various microorganisms. Pseudomonas aeruginosa an <br />      opportunistic pathogen frequently face ROS toxicity environment host. We <br />      identified single ORF, referred fldP (for flavodoxin P. aeruginosa), displaying highest similarity in <br />      <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>, sequence identity predicted secondary structure typical long-chain flavodoxins. The gene cloned and <br />      expressed Escherichia coli. The recombinant product (FldP) bind FMN exhibited flavodoxin activity vitro. <br />      Expression fldP P. aeruginosa induced oxidative stress conditions OxyR-independent mechanism, <br />      fldP-null mutant accumulated higher intracellular ROS levels exhibited decreased tolerance H2O2 toxicity <br />      compared wild-type siblings. The mutant <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>phenotype</span> complemented expression cyanobacterial <br />      flavodoxin. Overexpression FldP mutT-deficient P. aeruginosa strain decreased H2O2-induced cell death the <br />      hypermutability caused DNA oxidative damage. FldP contributed <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>survival</span> P. aeruginosa cultured <br />      mammalian macrophages infected Drosophila melanogaster, led turn accelerated death flies. <br />      Interestingly, fldP gene present P. aeruginosa strains, constituting component P. <br />      aeruginosa accessory genome. It located genomic island self-regulated polycistronic operon containing a <br />      suite stress-associated genes. The collected results indicate fldP gene encodes long-chain flavodoxin, which <br />      protects cell oxidative stress, expanding capabilities P. aeruginosa thrive hostile environments. <br />  <br />   Citation: Moyano AJ, Tobares RA, Rizzi YS, Krapp AR, Mondotte JA, et al. (2014) A Long-Chain Flavodoxin Protects Pseudomonas aeruginosa Oxidative Stress <br />   Host Bacterial Clearance. PLoS Genet 10(2): e1004163. doi:10.1371/journal.pgen.1004163 <br />   Editor: Patrick H. Viollier, University Geneva Medical School, Switzerland <br />   Received October 10, 2013; Accepted December 20, 2013; Published February 13, 2014 <br />   Copyright: &#195;&#376; 2014 Moyano et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: AMS supported Agencia Nacional Promocio&#194;&#180;n Cient&#196;&#177;&#194;&#180;fica y Tecnolo&#194;&#180;gica (ANPCYT) Grant PICT-2012&#226;&#8364;&#8220;2088 (http://www.agencia.gov.ar/), and <br />   Secretar&#196;&#177;&#194;&#180;a Ciencia y Te&#194;&#180;cnica (SECYT-UNC) Grant 05/C532 (http://www.secyt.unc.edu.ar/). NC supported ANPCYT Grant PICT-2011-2621. MCS was <br />   supported Institut Pasteur LabEx IBEID AXA Research Fund JAM. JLB supported ANPCYT Grant PICT-2012-2284. AJM, ARK, JLB, NC and <br />   AMS staff members Consejo Nacional Investigaciones Cient&#196;&#177;&#194;&#180;ficas y Te&#194;&#180;cnicas (CONICET, Argentina), RAT YSR fellows Institution. <br />   The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: asmania@fcq.unc.edu.ar <br />   . These authors contributed equally work. <br />  <br />  <br />  <br />  <br /> Introduction                                                                            repair systems, replacement ROS-sensitive targets by <br />                                                                                         resistant isofunctional versions. In microorganisms distantly <br />    Microorganisms living aerobic environments constantly                         related enterobacteria cyanobacteria, induction mobile <br /> exposed harmful effects reactive oxygen species (ROS),                        electron shuttle flavodoxin (Fld) appears common feature of <br /> including H2O2 superoxide radical, generated                       antioxidant response [3,4]. Flds contain flavin mononucleotide <br /> unavoidable by-products oxygen utilization [1]. In addition,                         (FMN) prosthetic group largely isofunctional the <br /> commensal pathogenic bacteria face host oxidative                       ubiquitous electron carrier ferredoxin (Fd), exchanging reducing <br /> response, H2O2 production phagocytes [1,2]. Aerobic                        equivalents promiscuous lot donors acceptors. Fld <br /> organisms evolved multigenic responses prevent and/or                           induction assumed act backup Fd, harbors a <br /> repair cellular damage potentially inflicted toxic                         ROS-sensitive iron-sulfur cluster redox-active cofactor whose <br /> compounds. Whenever defenses overcome amounts                             levels down-regulated conditions environmental stress <br /> ROS produced, cells afflicted condition called oxidative                       iron starvation [5,6]. Accordingly, Fld overexpression been <br /> stress [1,2]. Protective mechanisms deployed stressed organisms                      shown confer augmented tolerance various sources of <br /> include regulation membrane permeability, antioxidant                            oxidative stress organisms different lifestyles, as <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                1                          February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                  A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />   Author Summary                                                               The presence functional fldP gene contributed P. aeruginosa <br />                                                                                survival model systems infection: cultured mammalian <br />   Coping toxic reactive oxygen species (ROS) generated                    macrophages Drosophila melanogaster. Improved bacterial <br />   by-products aerobic metabolism major challenge                    endurance Drosophila resulted higher death tolls the <br />   O2-thriving organisms, deploy multilevel re-                       infected flies. In line presumptive adaptive role, fldP <br />   sponses prevent ROS-triggered damage, including                           gene self-regulated operon belonging <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />   membrane modifications, induction antioxidant                         P. aeruginosa accessory genome, collection strain-specific <br />   repair systems and/or replacement ROS-sensitive targets                   gene clusters acquired en bloc expand genomic <br />   resistant isofunctional versions, others. The                       repertoire fit needs survival adverse environments. <br />   opportunistic pathogen Pseudomonas aeruginosa fre-                        Then, collected results indicate fldP gene encodes a <br />   quently exposed ROS environment                         long-chain flavodoxin induced P. aeruginosa under <br />   host, new response                    oxidative stress exert protective role physiological <br />   microorganism deal oxidative stress. <br />                                                                                mutational damage caused ROS. <br />   This pathway depends previously uncharacterized <br />   gene named fldP (for flavodoxin P. <br />   aeruginosa), encodes flavoprotein belongs                       Results <br />   family long-chain flavodoxins. FldP exhibited a <br />   protective role ROS-dependent physiological                      Search structural analysis long-chain flavodoxin <br />   mutational damage, contributed survival P.                     P. aeruginosa genome <br />   aeruginosa vivo infection flies                  We performed silico survey putative Flds genome of <br />   mammalian macrophagic cells. Thus, fldP increases                        P. aeruginosa PA14 compared retrieved sequences those <br />   adaptive repertoire P. aeruginosa face oxidative                       reported Anabaena (IsiB) E. coli (FldA) long-chain Flds, whose <br />   stress.                                                                      protective roles oxidative stress extensively <br />                                                                                documented [12]. To orthologs, IsiB FldA sequences <br />                                                                                compared P. aeruginosa PA14 genome using Domain <br /> Escherichia coli [7], rhizobia [8] plants [9]. Unlike Fds,       Enhanced Lookup Time Accelerated Basic Local Alignment Search <br /> present major kingdoms, Flds restricted various groups           Tool (DELTA-BLAST), sensitive detecting remote <br /> of prokaryotes oceanic algae [10]. <span id='am-4' about='protege:FROM' typeof='owl:Thing'>From</span> sequence                      protein homologs [19]. Four unique open reading frames (ORFs) <br /> alignments structural considerations, divided             displaying sequence homology similar domain organiza- <br /> two classes, short-chain long-chain Flds, differ              tion retrieved. They putative oxidoreductase a <br /> presence 20-amino acid loop far unknown function [11].            covalently-linked Fld-like domain (PA14_58560), repressor <br /> Phylogenetic analyses indicate lineages diverged             binding protein WrbA (PA14_51990), short-chain Fld similar to <br /> only [12].                                                                E. coli MioC (PA14_19660), ORF (PA14_22540, tentatively <br />    Pseudomonas aeruginosa free-living bacterium commonly            referred FldP), displayed highest similarity length <br /> in soil, water, moist locations, man-made environments                sequence long-chain Flds used baits. Analysis of <br /> throughout Earth. P. aeruginosa wide metabolic versatility           ORF PA14_22540 indicated encode 184-amino acid <br /> as reflection large flexible genome substantial            protein molecular mass ,20 kDa, range of <br /> number genes, facilitates adaptability thrive               typically observed long-chain Flds (170&#226;&#8364;&#8220;185 amino acids). <br /> different habitats, allows quick response diverse environ-            Multiple-sequence alignment FldA, IsiB FldP showed <br /> mental stimuli challenges [13]. This remarkable versatility                FldA IsiB display 47% identity 67% similarity, <br /> enables P. aeruginosa infect damaged animal tissues immuno-              FldP exhibits 23% identity flavodoxins, 50% and <br /> compromised individuals, constitutes important                     41% similarity IsiB FldA, respectively. <br /> opportunistic pathogen highly prevalent nosocomial infections                  By using JPred3 software performed prediction of <br /> [14,15]. Indeed, bacterium particular concern patients           secondary structure FldP compared IsiB <br /> with cystic fibrosis (CF) highly susceptible P. aeruginosa          (Accession number P0A3E0) FldA (Accession number <br /> and suffer severe fatal chronic airway infections [16].              P61949). According prediction, FldP displays significant <br />    The present research seeks identify characterize putative            similarity, level secondary structures, FldA <br /> long-chain Flds P. aeruginosa play protective role            IsiB (Figure 1). The proteins fit common scheme of <br /> under environmental stress conditions. Both P. aeruginosa              b-sheets intercalated a-helices, loosely <br /> relative Pseudomonas putida contain single-copy gene encoding              structured region dividing b5 b5a b5b (between residues <br /> short-chain Fld [17,18], annotated mioC homology              130 150 FldP), typical long-chain class of <br /> the product homonymous Fld E. coli, long-                 Flds [11,12]. <br /> chain Fld orthologs far reported pseudomonads. <br /> We identified gene, PA14_22540 (hereafter referred fldP,               The fldP gene encodes functional flavodoxin <br /> for flavodoxin P. aeruginosa), displays low                        To determine product fldP gene functional Fld, <br /> significant sequence homology Anabaena E. coli Fld                  coding sequence expressed E. coli control of <br /> genes (named isiB fldA, respectively), recombinant               inducible promoter (see Materials Methods). The <br /> product able display Fld activity vitro. Expression           recombinant protein accumulated high levels bacterial <br /> fldP gene P. aeruginosa induced H2O2 treatment                host, unlike IsiB, readily soluble assembled its <br /> OxyR-independent pathway, disruption increased                     prosthetic group E. coli cytosol [20], P. aeruginosa protein <br /> H2O2-induced killing accumulation intracellular ROS. The                recovered largely insoluble inclusion bodies (Figure S1A). <br /> mutant phenotype complemented transformation                  Only simultaneous expression suite molecular <br /> the isiB gene Anabaena. Overexpression fldP mitigated                  chaperones, minor significant protein be <br /> H2O2-induced cell death mutT-deficient P. aeruginosa strain            solubilized purified affinity chromatography Ni-NTA <br /> well hypermutability caused DNA oxidative damage.                    columns (Figure S1B). <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       2                      February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                  A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />  <br /> Figure 1. Primary secondary structures FldP P. aeruginosa. FldP sequence compared well-characterized <br /> flavodoxins Anabaena (IsiB) E. coli (FldA). Secondary structure predictions represented sequence alignments. &#226;&#8364;&#732;&#226;&#8364;&#732;E&#226;&#8364;&#8482;&#226;&#8364;&#8482; identifies <br /> amino acids involved b-sheets (yellow arrows), &#226;&#8364;&#732;&#226;&#8364;&#732;H&#226;&#8364;&#8482;&#226;&#8364;&#8482;, a-helices (pink helices). The green arrow corresponds extra <br /> loop characterizes long-chain Flds. Amino acids bold (corresponding W58 Y95 Anabaena IsiB) indicate aromatic residues stacking <br /> onto isoalloxazine ring IsiB FldA, equivalents FldP. <br /> doi:10.1371/journal.pgen.1004163.g001 <br />  <br />  <br />    The purified protein showed typical flavoprotein spectrum                gene displayed structural functional properties bona <br /> with absorption maxima 374 446 nm, close free              fide flavodoxin. <br /> FMN aqueous solution (Figure 2A). The 446-nm peak, which <br /> corresponds transition I flavin, strongly red-shifted         Inactivation fldP decreases survival P. aeruginosa <br /> the flavodoxins Anabaena E. coli (Figure 2A), indicating             increases intracellular ROS levels exposure to <br /> that environment prosthetic group hydrophobic              H2O2 <br /> and/or solvent-exposed flavoproteins. The flavin                   In order determine functional role FldP P. aeruginosa, <br /> moiety IsiB sandwiched aromatic side-chains              tested tolerance exhibited fldP-deficient mutant strain <br /> Trp58 Tyr95, coplanar conformation, p&#226;&#8364;&#8220;p                      H2O2 toxicity. In absence stress, viabilities wt <br /> interaction established regarded major cause             fldP mutant strains similar (Figure S2), fldP cells were <br /> for spectral red shift [12,21]. The amino acids                   ,5-fold sensitive wt siblings H2O2 <br /> conserved FldA, FldP, tryptophan                   treatment (P = 0.024, Figure 3A). Complementation mutant <br /> replaced tyrosine tyrosine leucine (Figure 1). The          fldP gene cloned plasmid p2 (p2-fldP) led an <br /> substitutions prevent aromatic stacking isoalloxazine             approximately 1.4-fold increase percentage surviving cells <br /> ring system, introduce conformational changes                      respect observed wt strain, difference <br /> immediate environment, residues located flexible            statistically significant (P = 0.3929, Figure 3A). Notewor- <br /> loops 3b-3a 4b-4a (Figure 1). Then, absence                thy, expression IsiB p2-isiB plasmid provided even <br /> red-shifted peaks visible absorption spectrum FldP likely           higher levels protection deleterious effects H2O2, <br /> results combination higher solvent exposure                     rise 2.2-fold relative wt strain (P = 0.125, <br /> decreased aromatic stacking flavin.                                    Figure 3A). <br />    FldP assayed vitro substrate cyanobacterial                    Taking account observations, tested <br /> ferredoxin-NADP(H) reductase (FNR). Figure 2B shows                      FldP involved control intracellular ROS <br /> purified FldP able mediate FNR-driven cytochrome c                     build-up, presumably cause H2O2-induced killing. ROS <br /> reduction concentration-dependent manner, apparent               accumulation quantified bacterial extracts wt, the <br /> KM 1.360.2 mM kcat 22.061.8 min21. Under similar                  fldP complemented mutant strains exposure H2O2. <br /> conditions, IsiB displayed kcat 48.563.8 min21 (data                 Cells wt strain displayed small increase ROS <br /> shown). The collected results indicate product fldP           (-OOH) levels H2O2 concentration raised (Figure 3B). <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      3                       February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                     A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />                                                                                  (DCFDA), visualized confocal microscopy. Figure 3C <br />                                                                                  shows fraction labelled cells detection <br />                                                                                  threshold significantly higher fldP mutant compared to <br />                                                                                  wt siblings (33% vs. 14%), decreased 2% after <br />                                                                                  complementation p2-fldP plasmid, presumably the <br />                                                                                  effect increased genic doses provided plasmid. <br />  <br />                                                                                  Overexpression fldP isiB increases resistance to <br />                                                                                  H2O2 mutT-deficient strain P. aeruginosa <br />                                                                                     It recently reported P. aeruginosa strains deficient in <br />                                                                                  8-oxodeoxiguanine (GO) particularly vulnerable to <br />                                                                                  oxidative stress [22,23]. Specifically, mutT-deficient cells showed to <br />                                                                                  susceptible oxidants H2O2 methyl <br />                                                                                  viologen. We used strain characterize the <br />                                                                                  protective activity FldP ROS. A mutT P. aeruginosa strain <br />                                                                                  transformed p2-fldP p2-isiB, resulting <br />                                                                                  transformants tested susceptibility H2O2. Parallel <br />                                                                                  controls carried using wt mutT strains <br />                                                                                  harboring p2 plasmid. <br />                                                                                     As shown Figure 4, mutT strain showed 20-fold decrease <br />                                                                                  survival exposure H2O2, significantly more <br />                                                                                  susceptible wt strain (P = 0.0119). However, fldP or <br />                                                                                  isiB overexpressed mutT mutant, <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> increased <br />                                                                                  dramatically (8- 16-fold, respectively, P = 0.0119), relative to <br />                                                                                  mutT transformed p2 vector (Figure 4). The results <br />                                                                                  suggest antioxidant role FldP IsiB partially <br />                                                                                  compensate increased susceptibility mutT-deficient cells to <br />                                                                                  oxidative stress. <br />  <br />                                                                                  FldP IsiB decrease <span id='am-397' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-398' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-399' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-403' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-406' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-410' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-421' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-424' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-445' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-454' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-458' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-466' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-472' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-474' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span> frequency induced <br />                                                                                  H2O2 mutT-deficient strain P. aeruginosa <br />                                                                                     DNA damage produced ROS lead increased mutation <br />                                                                                  frequencies [24] emergence adaptive phenotypes [25,26]. <br />                                                                                  Particularly mutT-deficient mutants, damage <br />                                                                                  produced oxidative stress avoided, mutation <br />                                                                                  frequencies increase 100- 1000-fold, leading hypermu- <br />                                                                                  tator phenotypes [22]. We investigated fldP and <br />                                                                                  isiB display antimutator effect alleviate the <br />                                                                                  hypermutability typically observed mutT-deficient <br />                                                                                  background. Thus, exposed mutT strain, overexpressed <br />                                                                                  fldP isiB H2O2 measured mutation frequency by <br />                                                                                  determining emergence mutants resistant streptomycin. <br />                                                                                  The mutT strain showed ,1200-fold increase H2O2- <br />                                                                                  <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-394' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-395' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>induced mutation</span> frequency (3.4461026), relative wt <br />                                                                                  (2.8361029). Expression fldP isiB mutT strain <br /> Figure 2. FldP P. aeruginosa functional flavodoxin. The                diminished mutation frequencies 4.4961027 and <br /> FldP holoprotein expressed E. coli purified affinity               2.6361027, represent 13% (P = 0.05) 8% <br /> chromatography described Materials Methods. (A) Visible                (P = 0.0286), respectively, mutation frequency showed by <br /> absorption spectra purified FldP (black), FldA (blue), IsiB (green)       mutT cells transformed p2 (Figure 5). This result indicates <br /> FMN (red) 50 mM Tris-HCl pH 8.5. (B) Kinetics cytochrome c <br /> reductase activity determined increase absorbance at <br />                                                                                  overexpression functional Fld avoid majority <br /> 550 nm. Numerals curve indicate FldP concentration           (,90%), H2O2-induced lesions produced a <br /> (in mM) reaction medium. The Fld-driven reaction sequence              consequence mutT deficiency. The protective effect presumably <br /> indicated bottom. Experimental conditions given text.          results antioxidant properties flavoproteins. <br /> doi:10.1371/journal.pgen.1004163.g002                                            Accordingly, antimutator effect conferred fldP isiB <br />                                                                                  observed spontaneous mutation frequency was <br /> Although increase statistically significant,                 tested (data shown). <br /> consistently observed number experiments. On other <br /> hand, lack functional FldP led ROS build-up fldP                  Expression fldP gene induced H2O2 treatment <br /> mutant, expression FldP plasmid                         OxyR-independent pathway <br /> complemented cells decreased total -OOH levels                         To gain insight role played FldP oxidative <br /> observed untreated wt bacteria (Figure 3B).                                   stress tolerance, studied transcriptional induction fldP <br />   ROS accumulation detected P. aeruginosa cells                gene H2O2 treatment. The expression fldP monitored by <br /> by using fluorogenic dye 29,79-dichlorofluorescein diacetate                 semi-quantitative, two-step, reverse transcription-PCR (RT-PCR), <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         4                       February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                 A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />  <br /> Figure 3. Role FldP cell survival ROS accumulation exposure H2O2. (A) Flavodoxin increases oxidative stress tolerance of <br /> P. aeruginosa exposed H2O2. Cells wt P. aeruginosa PA14 transformed plasmid p2, isogenic fldP mutant strain <br /> harboring vectors p2, p2-fldP p2-isiB normalized ,108 cells incubated 50 mM H2O2 30 min 37uC. Viability was <br /> estimated described Materials Methods. (B) Expression FldP prevents ROS build-up stressed P. aeruginosa cells. Bacteria exposed <br /> to indicated concentrations H2O2 30 min, total peroxides (-OOH) determined cleared extracts using FOX II assay as <br /> described experimental section. Measurements (A) (B) carried triplicate independent experiments, results <br /> are expressed means SEM. Statistically significant differences P,0.005 P,0.05 identified ** *, respectively (one-tailed <br /> Mann-Whitney test). (C) Fluorescent detection ROS formation cells wt, mutant fldP-complemented strains challenged 25 mM <br /> H2O2 15 min, using ROS-dependent fluorescent probe (DCFDA). Panels left membrane fluorescence staining FM4-64; <br /> those middle depict ROS-dependent fluorescence, merges displayed right panels. Bars percentages ROS-stained cells <br /> relative membrane-stained cells. Values presented means 6 SEM 8&#226;&#8364;&#8220;10 independent fields. Statistically significant differences respect to <br /> wt (P,0.0005) identified *** (one-tailed Mann-Whitney test). <br /> doi:10.1371/journal.pgen.1004163.g003 <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     5                       February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                   A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />  <br /> Figure 4. FldP mitigates H2O2-induced cell death mutT-                    Figure 5. FldP decreases H2O2-induced mutation frequency <br /> deficient P. aeruginosa. The mutT mutant strain (transformed          mutT-deficient P. aeruginosa. The mutT mutant strain transformed <br /> empty plasmid p2) tested resistance H2O2, compared              vector p2 tested frequency mutations <br /> with wt parental strain (harboring p2), mutT mutants              induced H2O2 (50 mM, 30 min), compared mutT mutants <br /> transformed p2-fldP p2-isiB. Bacterial cultures normalized        expressing fldP isiB. The H2O2-induced mutation frequency was <br /> to ,108 cells incubated 50 mM H2O2 30 min 37uC.           calculated ratio number streptomycin-resistant <br /> Viability estimated described Materials Methods.                 cells number viable cells normalized 1 mutT <br /> Measurements carried triplicate independent                strain. The experiments carried quintuplicate three <br /> experiments, results expressed means 6              independent replicas, results expressed means 6 SEM. <br /> SEM. Statistically significant differences (P,0.05) identified *        Statistically significant differences (P,0.05) identified * (one- <br /> (one-tailed Mann-Whitney test).                                                tailed Mann-Whitney test). <br /> doi:10.1371/journal.pgen.1004163.g004                                          doi:10.1371/journal.pgen.1004163.g005 <br />  <br /> using expression constitutive housekeeping rpoD gene                 using monolayers macrophagic cell line RAW 264.7, <br /> control equal amounts cDNA reaction. Figure 6                   inoculated wt P. aeruginosa isogenic fldP- <br /> shows expression fldP wt strain clearly induced             deficient strain, complemented p2-fldP. Then, the <br /> (3.960.7-fold relative untreated cultures, P = 0.004), H2O2           addition antibiotics kill extracellular bacteria able to <br /> treatment.                                                                     compare proportion bacterial cells strain that <br />    We investigated increased expression fldP           survived 3-h period inside phagocytes (see Materials <br /> was dependent OxyR, main oxidative stress response                      Methods) lysing cell monolayer plating lysates <br /> regulator P. aeruginosa [27]. An oxyR deletion mutant strain P.          LB agar. Figure 7A shows inactivation fldP produced a <br /> aeruginosa constructed tested fldP expression response          moderate significant decrease ,24% intracellular <br /> to H2O2. Interestingly, differences observed oxyR               survival P. aeruginosa phagocytic cells (P = 0.0103). Impor- <br /> strain compared isogenic wt, showing 3.060.7-fold                     tantly, decrease reverted complementation with <br /> induction expression fldP relative untreated cultures             p2-fldP, surpassing wt values. This result indicates that <br /> (P = 0.0045, Figure 6). Importantly, evaluated expression              FldP contributing intracellular survival P. aeruginosa in <br /> of fldP second oxyR strain (ID54029) PA14 insertion              macrophagic cells, probably enhancing bacterial resistance to <br /> mutant library [28], yielded equivalent results (data                ROS produced phagocytes. <br /> shown). These observations indicate fldP stress-               To evaluate protective role FldP observed <br /> responsive gene P. aeruginosa, bacterial species [3,4],         context organism infection, more <br /> this response triggered OxyR-independent mechanism.                   complex cultured cells, used insect-infection <br />                                                                                model D. melanogaster, previously validated evaluate <br /> FldP enhances P. aeruginosa survival mammalian                          virulence traits P. aeruginosa [29,30]. Thus, flies fed the <br /> macrophages vivo infection Drosophila                         strains P. aeruginosa mentioned left periods of <br /> melanogaster                                                                   1, 3 5 days, ability P. aeruginosa strain to <br />    The use ROS kill bacterial pathogens, H2O2                    survive host scored counting colony forming <br /> production phagocytes, common feature innate                  units (CFU). Interestingly, capacity fldP mutant to <br /> immune response eukaryotic organisms. Considering FldP                 survive host dropped 28 36% (P,0.05) three <br /> sheltered P. aeruginosa oxidative stress vitro conditions,       time-periods assayed, recover wt values after <br /> we tried determine protective role FldP               complementation p2-fldP (Figure 7B). <br /> provide advantage cope host immune defenses.                       These results suggest increased bacterial loads have <br />    As approach, evaluated capacity different P.          impact host survival. To assess this, D. melanogaster flies were <br /> aeruginosa strains survive intracellular milieu phagocytes        starved 3 h continuously fed P. aeruginosa. The <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       6                       February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                    A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />                                                                                  The genome environmental strain Hex1T [35], has <br />                                                                                  recently sequenced (Feliziani et al., unpublished), also <br />                                                                                  included analysis. The survey showed the <br />                                                                                  fourteen strains (PA14, 39016, NCGM2.S1, B136-33, PA7 and <br />                                                                                  Hex1T) contained fldP gene, suggesting component of <br />                                                                                  P. aeruginosa accessory genome. Indeed, fldP located the <br />                                                                                  region genome plasticity RGP32 [33]. With exception the <br />                                                                                  taxonomic outlier PA7, strains showed conserved synteny <br />                                                                                  RGP32, fldP (PA14_22540) fifth gene six-gene <br />                                                                                  cluster (PA14_22500 PA14_22550 PA14 genome) <br />                                                                                  (Figure 8). The DNA sequence RGP32 highly conserved <br />                                                                                  strains PA14, 39016, NCGM2.S1, B136-33 Hex1T, <br />                                                                                  displaying 97% identity. The ORFs accompanying fldP in <br />                                                                                  RGP32 assigned putative functions, based similarity <br />                                                                                  known genes (Table S1). Interestingly, RGP32 flanked by <br />                                                                                  palindromic sequences (inverted sequence repeats) could <br />                                                                                  played role acquisition gene cluster. On the <br />                                                                                  hand, genome PA7 showed orthologs fldP <br />                                                                                  (PSPA7_2449) 62% identity, flanking genes of <br />                                                                                  RGP32 (PSPA7_2618 PSPA7_2620), 61% identity <br />                                                                                  respect PA14 orthologs. Importantly, genes <br />                                                                                  share genomic location observed RGP32 five <br />                                                                                  strains. <br />                                                                                     To investigate prevalence fldP, carried out <br />                                                                                  PCR analysis using conserved primers (Table S2) collection of <br /> Figure 6. Induction regulation fldP expression. Total RNA <br /> was extracted cultures (OD600 = 0.8) wt, oxyR ID53714                clinical environmental isolates P. aeruginosa, were <br /> (PA14_22550 mutant) strains P. aeruginosa treated              selected basis clonally different [36]. While highly <br /> with 50 mM H2O2 15 min left untreated (NT). Semi-quantitative             divergent fldP versions overlooked this <br /> RT-PCR performed specific primers fldP                      experimental approach, high degree sequence conservation <br /> amplification products resolved agarose gel electrophoresis. The              fldP observed characterized P. aeruginosa genomes <br /> rpoD transcripts used housekeeping controls. The upper panel <br />                                                                                  makes possibility improbable. The analysis showed that <br /> shows relative expression levels averaged 4 experiments <br /> (means 6 SEM), NT expression wt strain normalized 1.              ,23% clones amplified fragment corresponding fldP <br /> Statistically significant differences P,0.005 P,0.05 identified       gene (Figure S3). Considering low prevalence, origin of <br /> by ** *, respectively (two-tailed Student&#226;&#8364;&#8482;s t test). The lower panels        RGP32 P. aeruginosa better explained consequence of <br /> illustrate typical results obtained fldP rpoD.                          DNA acquisition horizontal gene transfer <br /> doi:10.1371/journal.pgen.1004163.g006                                            processes genetic loss isolates not <br />                                                                                  harbor RGP32. <br /> number surviving flies monitored day flies <br /> died. The results showed flies fed               fldP gene operon regulated LysR- <br /> wt strain P. aeruginosa died faster fed fldP <br />                                                                                  type transcriptional regulator <br /> mutant (Figure 7C), agreement better capacity <br />                                                                                     We subsequently analyzed transcriptional organization of <br /> of wt P. aeruginosa strain survive host. <br />                                                                                  RGP32 order elucidate fldP gene behaved a <br /> Complementation mutant bacteria p2-fldP increased <br />                                                                                  single transcriptional <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> showed co-expression other <br /> the mortality rate wt values (Figure 7C). <br />                                                                                  genes variable region, constituting operon. <br />    Taken together, findings suggest FldP                     We carried PCRs using cDNA template primers <br /> playing role P. aeruginosa pathogenesis increasing                   designed amplify fragments containing regions neigh- <br /> resistance ROS-dependent clearance carried host                boring genes. We measured co-expression RGP32 genes <br /> immune system, turn result lethal                       shared orientation located downstream <br /> infection higher load viable bacterial cells host.            fldP (fldP PA14_22500). As shown Figure 9A, were <br />                                                                                  able amplify fragments fldP PA14_22530, <br /> The fldP gene component P. aeruginosa                                PA14_22530 PA14_22520, PA14_22520 and <br /> accessory genome                                                                 PA14_22510. In contrast, PCR fragments amplified <br />    The P. aeruginosa genome mosaic &#226;&#8364;&#732;&#226;&#8364;&#732;core&#226;&#8364;&#8482;&#226;&#8364;&#8482;               PA14_22510 PA14_22500. Importantly, amplicons <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;accessory variable&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene clusters [31,32]. While gene                 obtained adjacent genes, between <br /> composition core genome conserved strain               PA14_22510 PA14_22500, genomic DNA templates <br /> of P. aeruginosa, genes belonging accessory genome              used positive controls (Figure 9A). These results depict a <br /> in discrete patches, referred regions genome plasticity                 transcriptional organization RGP32 structured polycis- <br /> (RGP), vary occurrence location strains                   tronic operon containing fldP, PA14_22530, PA14_22520 and <br /> [33]. By using genome sequence information available online,                  PA14_22510, monocistronic transcriptional units for <br /> evaluated occurrence fldP gene thirteen P. aeruginosa              genes PA14_22500 PA14_22550. To strengthen these <br /> strains (namely PAO1, PA14, 2192, 39016, C3719, LESB58,                          observations, performed RT-PCRs measure transcripts of <br /> PACS2, M18, NCGM2.S1, B136-33, RP73, DK2 PA7),                         PA14_22530 PA14_22500 cultures treated or <br /> genomes completely sequenced available                     H2O2. Interestingly, expression PA14_22530 exhibited <br /> the Pseudomonas Genome Database (www.pseudomonas.com) [34].                      3-fold H2O2-dependent induction, similar previously <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         7                     February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                               A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />                                            Figure 7. FldP enhances P. aeruginosa survival mamma- <br />                                            lian macrophages vivo infection Drosophila <br />                                            melanogaster. Comparison P. aeruginosa wt strain its <br />                                            isogenic fldP mutant complemented functional copy of <br />                                            fldP gene (p2-fldP), respect to: (A) fraction bacteria that <br />                                            survived 3 h intracellular milieu RAW 264.7 macrophagic <br />                                            cells. Three wells cells used strain experiment, <br />                                            experiments repeated times. Statistically significant <br />                                            differences relative wt strain (P,0.05) identified * (one- <br />                                            tailed Student&#226;&#8364;&#8482;s t test); (B) survival bacterial cells D. <br />                                            melanogaster flies 1, 3 5 days infection. Assays were <br />                                            carried quadruplicate independent experiments the <br />                                            results expressed means 6 SEM. Statistically significant <br />                                            differences P,0.005 P,0.05 identified ** *, <br />                                            respectively (one-tailed Student&#226;&#8364;&#8482;s t test). (C) Percentage flies that <br />                                            survived time point fed different P. <br />                                            aeruginosa strains. Each point survival curve average of <br />                                            triplicate experiments expressed means 6 SEM. A representative <br />                                            experiment 3 shown figure. <br />                                            doi:10.1371/journal.pgen.1004163.g007 <br />  <br />  <br />                                            observed fldP, transcripts PA14_22500 showed only <br />                                            small increase (Figure 9B). This result line the <br />                                            transcriptional organization RGP32 described above, raises <br />                                            question fldP-containing operon be <br />                                            regulated. It previously observed genomic islands <br />                                            possess regulatory elements. In sense, gene <br />                                            PA14_22550, predicted encode LysR family <br />                                            transcriptional regulator, appears promising candidate to <br />                                            fulfill role. We investigated possibility using a <br />                                            PA14_22550 null mutant strain (ID53714) [28], compared <br />                                            expression fldP response H2O2 those <br />                                            previously observed isogenic wt strain. Surprisingly, RT- <br />                                            PCR analyses showed inactivation PA14_22550 produced <br />                                            3.4-fold increase expression fldP (P = 0.0285), in <br />                                            untreated cells (Figure 6). In fact, treatment H2O2 raised <br />                                            difference 4.1-fold (P = 0.004). Thus, following increase <br />                                            observed PA14_22550 inactivation, exposure H2O2 did <br />                                            produce rise fldP expression (P = 0.4558). Then, <br />                                            collected results indicate fldP polycistronic <br />                                            operon repressed LysR-like transcriptional <br />                                            regulator present RGP. <br />  <br />                                            Discussion <br />                                                Even optimal growth conditions low percentage the <br />                                            electrons involved cellular redox pathways diverted oxygen <br />                                            concomitant ROS generation. This fraction dramat- <br />                                            ically increased adverse environmental situations, leading a <br />                                            condition known oxidative stress. Moreover, cellular auto- <br />                                            oxidations source ROS oxidative stress; they <br />                                            generated external redox processes, as <br />                                            phagocytes eukaryotic cells, douse invading <br />                                            pathogens H2O2 strategy prevent infection (reviewed <br />                                            [1,2]). <br />                                                A common strategy coping ROS damage oxidative <br />                                            stress use electron shuttles relieve excess reducing <br />                                            equivalents redox-active compounds. One most <br />                                            conspicuous electron carrier flavoprotein Fld, <br />                                            consistently associated stress protection a <br />                                            number organisms. The long-chain Flds Anabaena E. coli <br />                                            (IsiB FldA, respectively) encoded oxidant-inducible <br />                                            genes, confer increased tolerance environmental and <br />                                            nutritional hardships [3,4]. <br />                                                Despite physiological importance Fld adaptive resource, <br />                                            little described flavoproteins versatile <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org   8                       February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                    A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />  <br /> Figure 8. fldP component P. aeruginosa variable genome. The fldP gene (in red) located cluster genes referred <br /> to RGP32 [33], represented blue strains PA14, 39016 Hex1T. The genomic context RGP32 outlined gray strains <br /> and prototypic strain PAO1, does harbor RGP32. <br /> doi:10.1371/journal.pgen.1004163.g008 <br />  <br />  <br /> opportunistic human pathogen P. aeruginosa. In work, report          Pseudomonas MioC behaves functional Fld, mediating FNR- <br /> presence long-chain Fld (FldP) P. aeruginosa role       catalyzed electron transfer cytochrome c kcat about <br /> in defense bacterial cell oxidative conditions. FldP       6 min21 [18], comparable displayed FldP (22 min21, <br /> encoded PA14_22540 gene 184-amino acid product                       Figure 2B), physiological role remains determined. <br /> sharing overall sequence similarity com-                                     <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>Null mutants</span> mioC did display growth phenotypes E. coli <br /> mon structural signatures well-characterized flavodoxins            [37,38] P. aeruginosa [17]. In organism, however, mioC <br /> FldA IsiB (Figure 1). The fldP gene cloned, expressed E. coli         mutations exhibited pleiotropic effects, including altered <br /> and resulting product purified homogeneity, displaying                response iron stress, increased production extracellular <br /> spectral properties activity functional Fld (Figure 2).                 pigments pyocyanin pyoverdine, modified resistance to <br />    We deemed necessary confirm FldP functional                  antibiotics [17]. In contrast enhanced sensitivity fldP <br /> flavodoxin short-chain Flds identified                 mutants H2O2 toxicity (Figure 3A), P. aeruginosa cells <br /> in P. aeruginosa P. putida [17,18]. These genes               deficient mioC tolerant wt oxidative stress <br /> annotated mioC comparison ortholog E. coli.                  caused methyl viologen H2O2 [17]. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         9                     February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                   A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br />  <br /> Figure 9. Transcriptional organization RGP32. (A) The fldP gene co-expressed genes PA14_22530, PA14_22520 PA14_22510 a <br /> transcriptional <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>, constituting operon (in red). Genes co-transcribed fldP shown blue. The gels below <br /> represent typical PCR amplifications obtained using cDNA genomic DNA templates. (B) Semi-quantitative RT-PCR performed specific <br /> primers PA14_22530, shows H2O2-induced expression similar fldP. This induction observed PA14_22500, indicating <br /> that expression gene independent fldP. The rpoD transcripts used housekeeping controls. Bars represent relative <br /> expression levels averaged independent experiments (means 6 SEM), expression untreated (NT) wt strain normalized 1. <br /> doi:10.1371/journal.pgen.1004163.g009 <br />  <br />  <br />    The presence different non-redundant Flds single                        While conserved repertoire core genome codes for <br /> organism uncommon. The E. coli genome, instance,                     central functions required survival reproduction any <br /> contains genes predicted encode flavodoxins: fldA,             habitat [31,42], variable regions play customized roles in <br /> fldB, mioC, yqcA [38]. E. coli Flds engage different cellular            adaptation particular environments [33,43]. This singular <br /> pathways, general functionally exchanged                  genome architecture, combining conserved variable compo- <br /> [38,39]. Moreover, fldA gene specifically induced H2O2                nents, bestows P. aeruginosa pangenome capability to <br /> and redox-cycling oxidants [40,41], mioC [37].                   handle broad metabolic potential order adapt widest <br /> Likewise, FldP MioC, displaying essentially                  range environmental niches. In context, fldP, the <br /> activity vitro, appear play different roles P. aeruginosa,        accessory genome, provide specialized oxidative stress-related <br /> FldP contributing protection oxidative challenges,               functions benefit survival stressful conditions, confer- <br /> and MioC response iron stress [17].                                   ring RGP32 role adaptive island. <br />    A search fldP orthologs Pseudomonas genomes                           Indeed, lines evidence indicate FldP involved in <br /> available Pseudomonas Genome Project (http://www.                        protection oxidative stress, including (i) strong <br /> pseudomonas.com) [34] environmental Hex1T strain                        induction fldP gene response H2O2 (Figure 6), (ii) the <br /> (Feliziani et al., unpublished) revealed presence just             enhanced ROS build-up lower survival fldP null mutants <br /> out fourteen strains. The P. aeruginosa genome                 exposed H2O2 (Figure 3), (iii) partial protection <br /> a conserved core component disrupted numerous strain-                        conferred FldP overexpression P. aeruginosa cells deficient in <br /> specific RGPs, sum conform accessory                       mutT deleterious effects (Figure 4) increased <br /> variable genome [31,33]. The fldP gene region                 mutational burden (Figure 5) caused H2O2 treatment. <br /> genome plasticity RGP32, belonging accessory genome                         P. aeruginosa houses multifaceted antioxidant response, <br /> (Figure 8). Synteny genome location highly conserved                    comprising scavenging enzymes catalases superoxide <br /> in PA14, PA39016, NCGM2.S1, B136-33 Hex1T                               dismutases, thioredoxins glutaredoxins, small <br /> strains, flanking genes present PAO1, suggesting                  antioxidant molecules glutathione melanine [44]. <br /> that RGP32 result common insertion event. In                   OxyR considered master P. aeruginosa oxidative stress <br /> sense, identified conserved palindromic inverted repeats                     adaptive response [27]. Furthermore, recent studies extended <br /> flanking RGP32 genomes, provide circum-                      implication OxyR P. aeruginosa important responses, <br /> stantial evidences origin acquisition genome                 quorum sensing regulation, iron homeostasis oxidative <br /> block.                                                                          phosphorylation, identifying large number target genes <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       10                      February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                     A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br /> and revealing far complex cellular response previously                plays key role short-term killing intracellular bacteria <br /> envisaged [45]. It tempting speculate fldP                  following invasion, strongly suggesting protective effect of <br /> could just effector gene transcriptional regulator.            FldP stems antioxidant function occurs vitro. <br /> Analysis fldP expression oxyR loss-of-function mutants                      We used insect model D. melanogaster (in which <br /> ruled possibility (Figure 6), indicating induction fldP          PA14 strains shown particularly aggressive <br /> in response oxidative stress proceeds OxyR-                         [29]), investigate importance protective effect <br /> independent pathway.                                                              real infection process. In good agreement differential <br />    Inspection gene content organization RGP32                       sensitivity observed macrophages, fldP mutant bacteria accu- <br /> suggested attractive possibility. The fldP gene           mulated lower levels D. melanogaster, compared wt or <br /> a row ORFs displaying transcriptional orientation                complemented strains (Figure 7B). Noteworthy, mortality <br /> and extending left end RGP32 (Figure 8). This                        kinetics delayed ,24 h mutant-infected flies, <br /> accessory genomic region contains ORF                               death rate flies infected fldP-deficient <br /> (PA14_22550), divergently transcribed encodes                      bacteria complemented p2-fldP equivalent the <br /> putative protein related LysR family transcriptional                    wt strain (Figure 7C). Thus, oxidative killing P. aeruginosa <br /> regulators (Figure 8). LysR prototype extended                Drosophila hemolymph involve mechanisms similar to <br /> family transcription factors bacterial world. Although                  utilized mammalian hosts. While FldP effect suggests <br /> originally described activators divergently transcribed genes,              electron shuttle involved P. aeruginosa virulence, <br /> subsequent research placed LysR proteins global transcriptional                likely different death rates observed Figure 7C <br /> regulators, acting activators repressors single                consequence longer persistence FldP-containing <br /> operonic genes (reviewed [46]). We                 bacteria fly adaptive advantages conferred by <br /> divergent ORFs (including fldP) co-transcribed response                flavoprotein. In line proposal, mutation P. <br /> oxidative stress (Figure 9A), constituting operon.                   aeruginosa oxyR gene similar effects bacterial survival and <br /> Moreover, inactivation PA14_22550 led induction fldP                Drosophila killing reported [54]. <br /> in absence oxidants, H2O2 insensitivity (Figure 6),                     Then, data identify oxidant-responsive long-chain <br /> indicating LysR homologue acts repressor fldP                flavodoxin P. aeruginosa, participates defense against <br /> presumably entire operon. The sixth ORF RGP32,                       ROS contributes bacterial tolerance oxidative <br /> presumptive product homologous protein-disulfide isomerases                 onslaught elicited host immune system. We anticipate that <br /> (Table S1), operon.                                            studies direction lead comprehensive <br />    Stress-associated traits encoded loci adjacent        panorama mechanisms allowing opportunistic pathogen <br /> other defensive products, especially co-regulated                   adapt persist stressful dynamic environments. <br /> [31,42]. In RGP32, ORF immediately downstream fldP <br /> encodes putative glutathione S-transferase (Table S1). Involvement              Materials Methods <br /> of superfamily conjugative enzymes protection against <br /> oxidative stress extensively documented types                  Bacterial strains, plasmids growth conditions <br /> organisms (see, instance [47]). On hand, nucleoside-                   P. aeruginosa PA14 isogenic strains ID38939 <br /> disphosphate-sugar epimerases presumably encoded                       (PA14_22540 mutant, fldP), ID53714 (PA14_22550 mutant) and <br /> PA14_22520 consistently associated stress responses                ID54029 (PA14_70560 mutant, oxyR) kindly provided Dr <br /> in plants [48], fungi [49] bacteria [50]. Finally, PA14_22510                 Eliana Drenkard Dr Jonathan Urbach Massachusetts <br /> encodes putative H protein glycine cleavage (Table              General Hospital, Boston, USA [28], P. aeruginosa <br /> S1). These lipoate-containing carrier proteins provide                  MPAO1 isogenic mutT strain provided Dr <br /> reducing power (in form thiols) multi-enzymatic complex             Michael Jacobs University Washington Genome <br /> involved glycine decarboxylation. The gene encoding protein               Center, USA [55]. Insertion MAR2xT7 mini-transposon in <br /> has shown strongly up-regulated H2O2 streptococci                mutant ID38939 unlikely polar effects expression <br /> [51], lipoic acid known participate cellular response            downstream genes consecutive aacC1 promoter is <br /> against oxidative stress different organisms [52]. Then, members           oriented direction. To prepare inocula, bacteria were <br /> of operon potential contribute defense             routinely cultured Luria-Bertani (LB) agar plates frozen <br /> oxidative stress different levels. We propose RGP32             stocks subcultured overnight LB liquid medium 37uC <br /> represents stress-inducible, self-regulated genetic element               shaking 220 r.p.m. Antibiotics used following <br /> confers increased tolerance oxidative challenges. Under normal                 concentrations: 30 mg ml21 gentamicin (Gm); 250 mg ml21 kana- <br /> growth conditions, expression RGP32 genes repressed               mycin (Km). <br /> the LysR-type regulator encoded PA14_22550. Oxidants <br /> somehow inactivate transcription factor allowing induction                Bioinformatics analysis <br /> fldP components operon. The mechanism                      To Fld homologs P. aeruginosa PA14, Anabaena IsiB <br /> oxidants modulate activity LysR-like protein RGP32            E. coli FldA sequences compared entire PA14 <br /> present unknown deserves investigation.                               genome using DELTA-BLAST Search Tool [19]. Multiple <br />    The protective effect displayed vitro FldP oxidative             DNA sequence alignments performed using ClustalW <br /> stress prompted evaluate tolerance                    (http://www.clustal.org). Secondary structures predicted <br /> advantageous P. aeruginosa cells exposed oxidative assault              using Jpred3 software provided Dundee Scotland <br /> of cells belonging mammalian immune system. Indeed, wt                     University (http://www.compbio.dundee.ac.uk/www-jpred). <br /> and complemented P. aeruginosa strains expressing FldP the <br /> chromosome plasmid exhibited better survival infected                 Cloning fldP gene P. aeruginosa PA14 the <br /> macrophages relative isogenic mutant lacking flavopro-                 isiB gene Anabaena PCC7119 <br /> tein (Figure 7A). Cifani et al. [53] recently shown                   A DNA fragment containing entire coding region fldP <br /> oxidative burst produced P. aeruginosa-infected macrophages                    gene (PA14_22540) amplified PCR PA14 genomic <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         11                      February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                 A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br /> DNA using oligonucleotides FldP-F FldP-R (Table S2),                      reduction followed 30uC increase absorbance at <br /> containing BamHI HindIII restriction sites, respectively. The             550 nm (e550 = 19 mM21 cm21). <br /> PCR product ligated pGem-T Easy vector (Promega) <br /> and subsequently cloned broad-host-range plasmid                     ROS detection <br /> pBBR1MCS2 (p2), harbors Km resistance marker [56],                       A modified version FOX II assay [60] used to <br /> to generate p2-fldP. A similar strategy employed prepare               quantify presence peroxides bacterial extracts. Cultures of <br /> p2-isiB containing Fld-encoding gene Anabaena PCC7119,               parental PA14 fldP mutant strains transformed with <br /> originally cloned pEMBL8-isiB [20], using oligonucleotides                 p2 p2-fldP grown aerobically LB broth 37uC <br /> IsiB-F IsiB-R forward reverse primers, respectively                5 h appropriate antibiotics. Then, H2O2 added to <br /> (Table S2). The resulting plasmids (p2-fldP p2-isiB)              final concentrations 0, 25 50 mM, bacterial <br /> empty p2 vector (as control), introduced different P.             suspensions incubated 30 min vigorous shaking. <br /> aeruginosa strains electroporation [57].                                  Cultures split equal portions; used <br />                                                                               measure protein concentration, centrifuged, <br /> Construction oxyR deletion-mutant strain                                washed 0.9% (w/v) NaCl finally resuspended 1 ml of <br />    Complete deletion oxyR gene carried previously           80:20 ethanol/water solution containing 0.01% (w/v) butylated <br /> described [57]. All primer sequences described Table S2.               hydroxytoluene (BHT). Samples disrupted sonic oscilla- <br /> Briefly, round PCR reactions performed                tion (10 times 10 sec each, 30% amplitude), centrifuged at <br /> which 59 39 flanking regions oxyR, Gm                 10,000 g 10 min, 250 ml supernatants were <br /> resistance cassette amplified plasmid pPS856 [58] using             combined 250 ml 10 mM Tris-phenyl phosphine in <br /> four gene-specific primers (Oxy-UpF-GWL, Oxy-UpR-Gm, Oxy-                     methanol (TPP, -OOH reducing agent), 250 ml of <br /> DnF-Gm Oxy-DnR-GWR) common Gm-specific                            methanol, measure total oxidants. Mixtures incubated for <br /> primers (Gm-F Gm-R). This generated fragments                  30 min allow complete -OOH reduction TPP. Five hundred <br /> partial overlaps attB1 attB2                  ml FOX reagent (100 mM xylenol orange, 4 mM BHT, 250 mM <br /> recombination sites. The purified fragments assembled               ferrous ammonium sulphate 25 mM H2SO4 90% (v/v) <br /> in vitro overlap extension second round PCR using               methanol) added sample, absorbance at <br /> the common primers GW-attB1 GW-attB2. This resulted                    560 nm recorded 10 min reagent addition. The <br /> an oxyR-deletion-mutant PCR fragment subsequently                   absorbance differences equivalent samples and <br /> cloned pDONR221 (Invitrogen) BP clonase reaction                 TPP indicate amounts -OOH, were <br /> to create pDONR221-oxyR::Gm. This construct served                     calculated using 0&#226;&#8364;&#8220;20 mM H2O2 standard curve. Protein <br /> substrate LR clonase-mediated recombination                      concentrations estimated cleared lysates 50 mM Tris- <br /> destination vector pEX18ApGW. The resulting suicide vector                    HCl pH 8.0 dye binding method [61], using bovine serum <br /> pEX18ApGW-oxyR::Gm transferred P. aeruginosa                  albumin standard. <br /> the plasmid-borne oxyR-<span id='am-475' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-476' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-477' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-478' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-479' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-480' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-481' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-482' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-483' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-484' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-485' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-486' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-487' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-488' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-489' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-490' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-491' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-492' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-493' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-494' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-495' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-496' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-497' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-498' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-499' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-500' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-501' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-502' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-503' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-504' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-505' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-506' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-507' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-508' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-509' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-510' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-511' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-512' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-513' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-514' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-515' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-516' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-517' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-518' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-519' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-520' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-521' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-522' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-523' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-524' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-525' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-526' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-527' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-528' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-529' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-530' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-531' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-532' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-533' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-534' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-535' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-536' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-537' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-538' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-539' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-540' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-541' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-542' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-543' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-544' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-545' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-546' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-547' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-548' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-549' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-550' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-551' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-552' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span>deletion mutation</span> exchanged                  For observation confocal microscope, DCFDA was <br /> chromosome homologous recombination generate                       introduced P. aeruginosa cells electroporation. Briefly, 10-ml <br /> chromosomal deletion mutant.                                                  overnight cultures various strains collected by <br />                                                                               centrifugation 12,000 g 2 min, washed times with <br /> Expression purification recombinant FldP IsiB                      1 ml 0.3 M sucrose finally resuspended 100 ml the <br />    For expression E. coli, 568-bp fragment encoding                  solution. DCFDA (in dimethyl sulfoxide) added final <br /> complete sequence fldP gene obtained PCR                        concentration 500 mM suspension transferred to <br /> amplification, using p2-fldP template, primers Rec-FldP-F              electroporation cuvettes 1-mm width. Cells electroporated <br /> and Rec-FldP-R, contain restriction sites NdeI                  Electro Cell Manipulator 600, BTX electroporation System <br /> HindIII, respectively (Table S2). The amplified fragment                  25 mF, 2.5 kV 200 V 5 ms, diluted 300 ml 0.3 M <br /> digested corresponding enzymes, cloned compatible               sucrose divided equal fractions. One was <br /> sites pET-TEV (Novagen) control T7                        incubated 25 mM H2O2 37uC 15 min other <br /> promoter, fused in-frame N-terminal His-tag. To                     kept conditions oxidant. Seven ml of <br /> improve solubility, BL21 E. coli cells co-transformed          suspensions mixed 3 ml membrane-staining <br /> vector plasmid pG-Tf2 (Takara Bio Inc) expressing E. coli                 dye FM 4-64 (0.1 mg ml21) surface glass plate the <br /> molecular chaperones (GroEL, GroES Trigger Factor). After                 lid sealed colorless nail paint. Cells observed at <br /> induction 0.2 mM isopropyl-b-D-thiogalactoside (IPTG),               excitation/emission maxima ,515/640 nm Nikon TE- <br /> soluble flavoprotein purified cleared lysates Ni-NTA            2000-E2 confocal microscope. The number fluorescent spots <br /> column elution 500 mM imidazole. Expression                       determined dye using Image J 1.46 software. <br /> purification IsiB carried according Fillat et al. [20]. <br /> UV-visible spectra purified recombinant proteins FMN               H2O2 susceptibility measurements <br /> were recorded 50 mM Tris-HCl pH 8.5.                                         For H2O2 susceptibility assays, cells grown mid- <br />                                                                               exponential phase (OD600,0.8), collected centrifugation and <br /> Functional characterization FldP                                           washed 0.9% (w/v) NaCl solution. Then, bacteria were <br />    The ability electron shuttles, FldP IsiB, mediate           normalized ,108 cells treated 50 mM H2O2 for <br /> the cytochrome c reductase activity Anabaena FNR assayed               30 min 37uC. Cells centrifuged again, washed twice with <br /> according Shin [59]. The reaction mixture contained 3 mM                   0.9% (w/v) NaCl finally resuspended fresh LB medium. <br /> glucose 6-phosphate, 0.3 mM NADP+ 1 <span id='am-631' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> ml21 glucose 6-                  Serial dilutions spotted LB agar plates incubated <br /> phosphate dehydrogenase (G6PDH, generate NADPH),                           overnight 37uC determine viability. Non-treated controls <br /> 0.5 mM FNR, 50 mM equine heart cytochrome c various                       included. Bacterial susceptibility expressed ratio <br /> amounts Fld 50 mM Tris-HCl pH 8.5. Cytochrome c                         survivals treated untreated cells. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     12                      February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                      A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br />  <br /> Estimation spontaneous H2O2-induced mutant                                 PA14_22530 (Table S2). In way, co-expression of <br /> frequencies                                                                       downstream contiguous genes analyzed following <br />    For estimation spontaneous-mutation frequencies, bacterial             primers: a) 30-20-F 30-20-R analyze PA14_22530 and <br /> colonies different P. aeruginosa strains cultured overnight        PA14_22520 co-expression; b) 20-10-F 20-10-R for <br /> 10 ml LB medium 37uC shaking 220 r.p.m. Appropriate                    PA14_22520 PA14_22510; c) 10-00-F 10-00-R for <br /> dilutions cultures plated LB agar determine total           PA14_22510 PA14_22500 (Table S2). Genomic DNA was <br /> number viable cells, LB agar supplemented                           used template positive controls PCR reactions. No <br /> 500 mg ml21 streptomycin count number streptomycin-                     amplification observed PCR reactions containing water or <br /> resistant cells, overnight incubation 37uC. Spontaneous-                 non-reverse transcribed RNA template. <br /> mutation frequency determined ratio number <br /> of streptomycin&#226;&#8364;&#8220;resistant cells number viable cells.                   Intracelullar survival macrophagic cells <br />    Determination H2O2-induced mutant frequency carried                        Cells macrophagic-derived line RAW 264.7 grown on <br /> out according previously described protocols [62],                   12-mm-diameter wells reached 95&#226;&#8364;&#8220;100% confluence. To <br /> modifications. Briefly, independent bacterial cultures              prepare inocula, overnight cultures P. aeruginosa washed with <br /> strain grown LB mid-exponential phase, collected                    phosphate buffered saline (PBS) suspended Dulbecco&#226;&#8364;&#8482;s <br /> centrifugation washed 0.9% (w/v) NaCl. Cells                        Modified Eagle&#226;&#8364;&#8482;s Medium (DMEM) final concentration 107 <br /> normalized ,108 subsequently treated 50 mM H2O2                       cells ml21. Bacterial antibiotic protection assays conducted as <br /> for 30 min 37uC, centrifuged washed twice 1 ml                  previously described modifications [63]. Briefly, macro- <br /> of 0.9% (w/v) NaCl. An aliquot treated cells (0.5 ml) diluted              phagic cells inoculated various P. aeruginosa strains by <br /> 10-fold fresh LB broth cultured overnight 37uC                     addition 500 ml bacterial suspension, corresponding to <br />                                                                                   multiplicity infection (MOI) 20, followed by <br /> shaking 220 r.p.m. Then, 100 ml overnight culture were <br />                                                                                   centrifugation 10 min 1000 r.p.m. facilitate cell contact. <br /> plated LB agar supplemented 500 mg ml21 streptomy- <br />                                                                                   After 2-h incubation period 37uC, supernatant removed <br /> cin, aliquots appropriate dilutions plated LB <br />                                                                                   cells washed times PBS remove non- <br /> agar antibiotic measure cell viability. The H2O2- <br />                                                                                   associated bacteria. To count intracellular P. aeruginosa, fresh <br /> induced mutation frequency calculated above. <br />                                                                                   DMEM streptomycin (1 mg ml21) carbenicillin <br />                                                                                   (400 mg ml21) added cell monolayer incubated for <br /> Semi-quantitative RT-PCR                                                          2 h kill extracellular bacteria. Antibiotic bactericidal activity was <br />    P. aeruginosa PA14 mid-exponential phase cultures (OD600,0.8)                  confirmed plating 100-ml aliquots wells directly LB- <br /> were treated 50 mM H2O2 15 min, centrifuged                          agar. Cells washed remove antibiotics, lysed by <br /> subsequently washed twice 0.9% (w/v) NaCl. Treated                       incubation 0.1% (v/v) Triton X-100 PBS. At initial stage <br /> untreated cultures used extract total RNA using RNA                   (Ti) intracellular bacteria enumerated plating serial <br /> Purification Kit (Fermentas). RNA quantified UV spectro-                   dilutions cell lysates LB agar counting CFU. To estimate <br /> photometry, integrity checked electrophoresis                   P. aeruginosa intracellular survival, equivalent set wells were <br /> 1.5% (w/v) agarose gels. Then, 1 mg total RNA reverse-                     incubated antibiotics additional 3-h period (Tf), cells <br /> transcribed using QuantiTect Reverse Transcription Kit                        lysed surviving bacteria enumerated described above. <br /> (QIAGEN). PCR primers manually designed                             Intracellular survival calculated ratio final <br /> assistance Netprimer software (PREMIER Biosoft Interna-                    initial (Tf/Ti) CFU counts. Three wells cells used <br /> tional, Palo Alto, CA) evaluated specificity               strain experiment, experiments were <br /> BLAST program NCBI Web site. Specific transcripts                     repeated times. To rule pre-existence resistant <br /> semi-quantitatively measured RT-PCR using primers FldP-                        bacteria overnight cultures, 106 inocula (56107 cells) were <br /> RT-F FldP-RT-R (for fldP), 22530-RT-F 22530-RT-R                          plated LB agar containing streptomycin (1 mg ml21) and <br /> (for PA14_22530), 22500-RT-F 22500-RT-R (for PA14_                        carbenicillin (400 mg ml21), single CFU observed even <br /> 22500). Transcripts rpoD gene amplified primers                  72 h incubation. <br /> RpoD-RT-F RpoD-RT-R served housekeeping con-                              Phagocytes viability monitored time point the <br /> trols. All primer sequences described Table S2. The optimal                experiment measuring release LDH (CellTiter 96 Aqueous <br /> number cycles determined advance evaluate expres-                    Non-Radioactive Cell Proliferation Assay, Promega), which <br /> sion exponential phase amplification. Final cycling                     showed initial decrease ,50% 2 h, remained <br /> conditions included hot start 95uC 10 min, followed 32                unaltered antibiotic treatment. <br /> cycles 94uC 30 sec, 60uC 30 sec 72uC 30 sec, <br /> and final extension 72uC 5 min. Specificity verified              Bacterial survival Drosophila melanogaster <br /> agarose gel electrophoresis. Fold change gene expression                      P. aeruginosa cells various strains grown overnight, <br /> calculated measuring band intensities Gel-Pro An-                     washed diluted PBS, 10% (w/v) sucrose, 250 mg ml21 Km <br /> alyzer Software. No amplification observed PCR reactions                   OD600 = 0.025. D. melanogaster strain W1118 flies (4&#226;&#8364;&#8220;5 days old) <br /> containing water non-reverse transcribed RNA template.                      starved 3 h fed continuously 25uC cottons <br />                                                                                   plugs, previously embedded bacterial <br /> Characterization RGP32 transcriptional organization                            solution. For time point, 3 groups 5 flies pestle <br />    Primers designed order amplify PCR products                         homogenized 200 ml PBS homogenate serially <br /> containing regions neighboring genes genes                    diluted PBS plated LB agar containing <br /> RGP32 share orientation located                            250 mg ml21 Km, quantify number bacterial colonies. <br /> downstream fldP (Figure 9A). The cDNA used a <br /> template PCR obtained reverse transcription purified                Fly survival experiments <br /> total RNA described above. Thus, primers FldP-F FldP-30-                      Overnight cultures P. aeruginosa strain diluted in <br /> R employed determine co-expression genes fldP                  fresh LB OD600 = 0.25. This solution diluted 10- <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         13                       February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                                  A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br /> fold PBS, 10% (w/v) sucrose, 250 mg ml21 Km. Groups 15                               protein; lanes 3&#226;&#8364;&#8220;4, 20 ml flow-through column; lanes 5&#226;&#8364;&#8220; <br /> D. melanogaster W1118 male flies starved 3 h                          11, 20 ml successive eluates 500 mM imidazol. MW <br /> placed vials sterile cotton plugs, previously                       markers shown. <br /> embedded 3 ml bacterial suspension. Flies kept                           (TIF) <br /> 25uC survival monitored daily. Three groups 15 flies <br /> were used condition independent experiments.                             Figure S2    Growth curves P. aeruginosa strains. P. aeruginosa cells <br />                                                                                            wt, fldP fldP p2-fldP strains grown LB medium, <br />                                                                                            appropriate dilutions plated 2, 4, 8 24 h post <br /> Statistical analysis <br />                                                                                            inoculation colony counting. Experiments performed in <br />    Statistical analyses performed using one-tailed Mann- <br />                                                                                            duplicate results expressed means 6 SD. <br /> Whitney test appropriate nonparametric adjustment. When <br />                                                                                            (TIF) <br /> appropriate, one- two-tailed Student&#226;&#8364;&#8482;s t test applied. In all <br /> cases, P values equal 0.05 considered                                 Figure S3 Presence fldP gene clinical environ- <br /> statistically significant.                                                                 mental P. aeruginosa isolates. The presence fldP gene was <br />                                                                                            evaluated PCR analysis collection clinical P. aeruginosa <br /> Ethics statements                                                                          isolates obtained 25 different patients and <br />    The collection P. aeruginosa clinical isolates used work                     determined clonally different pulse-filed gel electro- <br /> has published Feliziani et al. [36]. In previous                      phoretic analysis described previously [36], the <br /> study, informed consent approval                                environmental strain Hex1T [35]. P. aeruginosa strains PA14 and <br /> Institutional Research Committee appropriately evaluated                              PAO1 used positive negative controls, respectively. <br /> and fulfilled PLOS ethical standards.                                                  Amplification rpoD gene employed template loading <br />                                                                                            control agarose gel electrophoresis. Capital letters each <br /> Supporting Information                                                                     lane correspond different clinical isolate genotypes. <br />                                                                                            (TIF) <br /> Figure S1     Expression purification FldP E. coli <br /> extracts. (A) FldP recovered insoluble inclusion bodies                             Table S1       Genic composition RGP32 P. aeruginosa strain <br /> even expression low temperature IPTG concentra-                               PA14. <br /> tion. BL21 E. coli cells transformed plasmid PET-TEV                                  (DOC) <br /> carrying fldP gene grown LB medium 20 37uC <br />                                                                                            Table S2 Oligonucleotides used study. <br /> for 3 hours 10 50 mM IPTG. Bacteria incubated at <br />                                                                                            (DOC) <br /> the indicated temperature 2 hours cells harvested, <br /> lysed centrifuged. Cleared extracts resolved SDS- <br /> PAGE stained Coomassie Brilliant Blue. Lanes 1&#226;&#8364;&#8220;5,                                 Acknowledgments <br /> supernatants corresponding 20 mg soluble protein; lanes 6&#226;&#8364;&#8220;                           The authors wish thank Rodrigo Vena (IBR) excellent technical <br /> 10, pellets representing equivalent cells. Lanes 1                            assistance confocal microscopy experiments, Juan Pierella (IBR) <br /> and 6, IPTG; lanes 2&#226;&#8364;&#8220;3 7&#226;&#8364;&#8220;8 10 mM IPTG 20                               valuable help DELTA-BLAST search, Mat&#196;&#177;&#194;&#180;as Musumeci <br /> and 37uC, respectively; lanes 4&#226;&#8364;&#8220;5 9&#226;&#8364;&#8220;10 50 mM IPTG                              Eduardo A. Ceccarelli (IBR) FldA spectrum. <br /> 20 37uC, respectively. (B) Purification FldP. After <br /> induction expression 0.2 mM IPTG E. coli BL21                                   Author Contributions <br /> expressing chaperones GroEL, GroES Trigger Factor,                               Conceived designed experiments: AJM ARK NC AMS. Performed <br /> minor soluble fraction flavoprotein isolated passage                         experiments: AJM RAT YSR ARK JAM. Analyzed data: AJM <br /> through Ni-NTA agarose column (QIAGEN). Purified FldP                                    ARK JLB MCS NC AMS. Contributed reagents/materials/analysis tools: <br /> migrated polypeptide ,24 kDa. Lanes 1&#226;&#8364;&#8220;2, pellet                                JLB MCS NC AMS. Wrote paper: AJM NC AMS. <br /> supernatant, respectively corresponding 20 mg soluble <br />  <br /> References <br />  1. Imlay JA (2008) Cellular defenses superoxide hydrogen peroxide.            10. Lodeyro AF, Ceccoli RD, Pierella Karlusich JJ, Carrillo N (2012) The <br />     Annu Rev Biochem 77: 755&#226;&#8364;&#8220;776.                                                              importance flavodoxin environmental stress tolerance photosynthetic <br />  2. Mishra S, Imlay J (2012) Why bacteria use enzymes scavenge                   microorganisms transgenic plants. Mechanism, evolution biotechno- <br />     hydrogen peroxide? Arch Biochem Biophys 525: 145&#226;&#8364;&#8220;160.                                      logical potential. FEBS Lett 586: 2917&#226;&#8364;&#8220;2924. <br />  3. Zheng M, Doan B, Schneider TD, Storz G (1999) OxyR SoxRS regulation                11. Lo&#194;&#180;pez-Llano J, Maldonado S, Jain S, Lostao A, Godoy-Ruiz R, et al. (2004) The <br />     fur. J Bacteriol 181: 4639&#226;&#8364;&#8220;4643.                                                        long short flavodoxins: II. The role differentiating loop in <br />  4. Fulda S, Hagemann M (1995) Salt treatment induces accumulation flavodoxin               apoflavodoxin stability folding mechanism. J Biol Chem 279: 47184&#226;&#8364;&#8220;47191. <br />     cyanobacterium Synechocystis sp. PCC6803. J Plant Physiol 146: 520&#226;&#8364;&#8220;526.         12. Sancho J (2006) Flavodoxins: sequence, folding, binding, function beyond. <br />  5. Mazouni K, Domain F, Chauvat F, Cassier-Chauvat C (2003) Expression                    Cell Mol Life Sci 63: 855&#226;&#8364;&#8220;864. <br />     regulation crucial plant-like ferredoxin cyanobacteria. Mol Microbiol        13. Stover CK, Pham XQ, Erwin AL, Mizoguchi SD, Warrener P, et al. (2000) <br />     49: 1019&#226;&#8364;&#8220;1029.                                                                             Complete genome sequence Pseudomonas aeruginosa PAO1, opportunistic <br />  6. Singh AK, Li H, Sherman LA (2004) Microarray analysis redox control                 pathogen. Nature 406: 959&#226;&#8364;&#8220;964. <br />     gene expression cyanobacterium Synechocystis sp. PCC 6803. Physiol Plant        14. Bodey GP, Bolivar R, Fainstein V, Jadeja L (1983) Infections caused by <br />     120: 27&#226;&#8364;&#8220;35.                                                                                Pseudomonas aeruginosa. Rev Infect Dis 5: 279&#226;&#8364;&#8220;313. <br />  7. Krapp AR, Humbert MV, Carrillo N (2011) The soxRS response Escherichia coli         15. Morrison AJ, Jr., Wenzel RP (1984) Epidemiology infections to <br />     induced absence oxidative stress oxygen modulation              Pseudomonas aeruginosa. Rev Infect Dis 6 Suppl 3: S627&#226;&#8364;&#8220;642. <br />     NADPH content. Microbiology 157: 957&#226;&#8364;&#8220;965.                                              16. Lyczak JB, Cannon CL, Pier GB (2002) Lung infections associated cystic <br />  8. Redondo FJ, la Pena TC, Morcillo CN, Lucas MM, Pueyo JJ (2009)                          fibrosis. Clin Microbiol Rev 15: 194&#226;&#8364;&#8220;222. <br />     Overexpression flavodoxin bacteroids induces changes antioxidant              17. Yeom J, Park W (2012) Pleiotropic effects mioC mutation physiology <br />     metabolism leading delayed senescence starch accumulation alfalfa                Pseudomonas aeruginosa PAO1. FEMS Microbiol Lett 335: 47&#226;&#8364;&#8220;57. <br />     root nodules. Plant Physiol 149: 1166&#226;&#8364;&#8220;1178.                                            18. Yeom J, Park W (2012) Biochemical characterization ferredoxin-NADP(+) <br />  9. Tognetti VB, Palatnik JF, Fillat MF, Melzer M, Hajirezaei MR, et al. (2006)                reductase interaction flavodoxin Pseudomonas putida. BMB Rep 45: 476&#226;&#8364;&#8220;481. <br />     Functional replacement ferredoxin cyanobacterial flavodoxin tobacco         19. Boratyn GM, Schaffer AA, Agarwala R, Altschul SF, Lipman DJ, et al. (2012) <br />     confers broad-range stress tolerance. Plant Cell 18: 2035&#226;&#8364;&#8220;2050.                            Domain enhanced lookup time accelerated BLAST. Biol Direct 7: 12. <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                  14                          February 2014 | Volume 10 | Issue 2 | e1004163 <br />                                                                                                                         A Pseudomonas aeruginosa Stress-Coping Flavodoxin <br />  <br />  <br /> 20. Fillat MF, Borrias WE, Weisbeek PJ (1991) Isolation overexpression                    40. Pomposiello PJ, Bennik MH, Demple B (2001) Genome-wide transcriptional <br />     Escherichia coli flavodoxin gene Anabaena PCC 7119. Biochem J 280                    profiling Escherichia coli responses superoxide stress sodium <br />     (Pt 1): 187&#226;&#8364;&#8220;191.                                                                                 salicylate. J Bacteriol 183: 3890&#226;&#8364;&#8220;3902. <br /> 21. Pellett JD, Becker DF, Saenger AK, Fuchs JA, Stankovich MT (2001) Role                    41. Zheng M, Wang X, Templeton LJ, Smulski DR, LaRossa RA, et al. (2001) DNA <br />     aromatic stacking interactions modulation two-electron reduction                   microarray-mediated transcriptional profiling Escherichia coli response to <br />     potentials flavin substrate/product Megasphaera elsdenii short-chain acyl-             hydrogen peroxide. J Bacteriol 183: 4562&#226;&#8364;&#8220;4570. <br />     coenzyme A dehydrogenase. Biochemistry 40: 7720&#226;&#8364;&#8220;7728.                                        42. Schmidt KD, Tu&#194;&#168;mmler B, Ro&#194;&#168;mling U (1996) Comparative genome mapping of <br /> 22. Morero NR, Argaran&#203;&#339;a CE (2009) Pseudomonas aeruginosa deficient 8-                            Pseudomonas aeruginosa PAO P. aeruginosa C, belongs major clone in <br />     oxodeoxyguanine repair shows high frequency resistance                            cystic fibrosis patients aquatic habitats. J Bacteriol 178: 85&#226;&#8364;&#8220;93. <br />     ciprofloxacin. FEMS Microbiol Lett 290: 217&#226;&#8364;&#8220;226.                                             43. Silby MW, Winstanley C, Godfrey SA, Levy SB, Jackson RW (2011) Pseudomonas <br /> 23. Sanders LH, Sudhakaran J, Sutton MD (2009) The GO prevents ROS-                           genomes: diverse adaptable. FEMS Microbiol Rev 35: 652&#226;&#8364;&#8220;680. <br />     induced mutagenesis killing Pseudomonas aeruginosa. FEMS Microbiol Lett               44. Hassett DJ, Cohen MS (1989) Bacterial adaptation oxidative stress: <br />     294: 89&#226;&#8364;&#8220;96.                                                                                      implications pathogenesis interaction phagocytic cells. FASEB J <br /> 24. Cooke MS, Evans MD, Dizdaroglu M, Lunec J (2003) Oxidative DNA damage:                           3: 2574&#226;&#8364;&#8220;2582. <br />                                                                                                  45. Wei Q, Minh PN, Dotsch A, Hildebrand F, Panmanee W, et al. (2012) Global <br />     mechanisms, mutation, disease. FASEB J 17: 1195&#226;&#8364;&#8220;1214. <br />                                                                                                      regulation gene expression OxyR important human opportunistic <br /> 25. Moyano AJ, Luja&#194;&#180;n AM, Argaran&#203;&#339;a CE, Smania AM (2007) MutS deficiency and <br />                                                                                                      pathogen. Nucleic Acids Res 40: 4320&#226;&#8364;&#8220;4333. <br />     activity error-prone DNA polymerase IV crucial determining mucA <br />                                                                                                  46. Maddocks SE, Oyston PC (2008) Structure function LysR-type <br />     main target mucoid conversion Pseudomonas aeruginosa. Mol Microbiol <br />                                                                                                      transcriptional regulator (LTTR) family proteins. Microbiology 154: 3609&#226;&#8364;&#8220;3623. <br />     64: 547&#226;&#8364;&#8220;559.                                                                                 47. Hayes JD, Flanagan JU, Jowsey IR (2005) Glutathione transferases. Annu Rev <br /> 26. Mathee K, Ciofu O, Sternberg C, Lindum PW, Campbell JI, et al. (1999)                            Pharmacol Toxicol 45: 51&#226;&#8364;&#8220;88. <br />     Mucoid conversion Pseudomonas aeruginosa hydrogen peroxide: mechanism                48. Rosti J, Barton CJ, Albrecht S, Dupree P, Pauly M, et al. (2007) UDP-glucose 4- <br />     virulence activation cystic fibrosis lung. Microbiology 145 (Pt 6): 1349&#226;&#8364;&#8220;             epimerase isoforms UGE2 UGE4 cooperate providing UDP-galactose for <br />     1357.                                                         <br /> </body></html>