<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, June 1989. p. 2682-2694                                                                    Vol. 9, No. 6 <br /> 0270-7306/89/062682-13$02.00/0 <br /> Copyright &#194;&#169; 1989, American Society Microbiology <br />  <br />  <br />         Yeast Pheromone Response Pathway: Characterization a <br />          Suppressor That Restores Mating Receptorless Mutants <br />                                    KAREN L. CLARK AND GEORGE F. SPRAGUE, JR.* <br />          Institiute Molecillar Biology (Hid Depar-tmnent Biology, University Or egoni, Eiugene, Oregonl 97403 <br />                                         Received 28 December 1988/Accepted 14 March 1989 <br />  <br />            Saccharomyces cerevisiae haploid cells, ax a, mate appropriately stimulated pheromone <br />          secreted opposite cell type (a-factor a-factor, respectively). The binding pheromone its <br />          receptor signal initiates series intracellular changes lead specific physiological <br />          alterations required mating. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> identify components signal transduction pathway, sought <br />          pseudorevertants restored mating competence receptor mutants (MATTa ste3::LEU2). The suppressor <br />          srml-l isolated recessive <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>mutation</span> conferred temperature-sensitive growth strains and <br />          mating ability MAToa ste3::LEU2 strains nonpermissive temperature. In addition, srml-l <br />          mutants shifted nonpermissive temperature, exhibited phenotypes characteristic of <br />          pheromone response, induction FUSI transcription accumulation cells G, phase cell <br />          cycle. The srml-l mutation suppressed deletion a-factor-receptor gene cells. Together, these <br />          phenotypes suggest wild-type SRMI product component pheromone response pathway. <br />          Deletion STE4 STE5, required haploid cell types mating response pheromone, <br />          suppressed srml-1, suggesting SRMI product function STE4 STE5 <br />          products. SRMI essential gene expressed haploid cell types product their <br />          mating, a/a diploids. Homozygous srml-l a/a diploids temperature sensitive did arrest <br />          G,. Thus, SRMI product role vegetative life cycle cells. <br />  <br />    The yeast Saccharomvces cerevisiae haploid cell               transduction interaction heterotrimeric GTP-binding <br /> types, (x, reproduce mitosis. However,                 proteins (G proteins) (for recent review, reference 64). <br /> when cocultured, cell types exit mitotic cell                   We like understand processes the <br /> cycle undergo mating process form cell type,            binding pheromone receptor causes physiolog- <br /> the a/ax diploid cell. Interaction cells                 ical events discussed above. This requires identification of <br /> requires secretion response peptide pheromones                 components pheromone response pathway, path <br /> (for recent reviews, references 10, 21, 59). Thus, (x cells          propagates signal cell surface intracel- <br /> secrete ax-factor cells respond, cells secrete          lular targets. The cellular component pathway is <br /> a-factor (x cells respond. The responses phero-              pheromone receptor. Specificity pheromone recog- <br /> mone include increased transcription genes prod-                nition provided receptor; rest pathway <br /> ucts involved mating (15, 18, 38, 65), increased                  common ax cells (2, 19, 36, 45). By <br /> capacity agglutinate cells opposite mating type           analogy mammalian receptors discussed above, one <br /> (3, 13), preparation nucleus mating (52), arrest          expect G protein involved pheromone <br /> in G1 phase cell cycle (7, 68). Each member            signaling. In fact, homolog subunit G proteins is <br /> mating pair elongates partner (projection forma-              encoded GPAI (SCG1) gene (11, 41, 43). The GPAI <br /> tion) (34; J. Rine, Ph.D. thesis, University Oregon,                  product undoubtedly component pheromone <br /> Eugene, 1979); cell fusion followed nuclear fusion               response pathway, mutations GPAI suppress the <br /> creates a/a zygote.                                                   mating defect caused deletion a-factor- ac- <br />    Pheromones interact target cells specific              factor-receptor genes (23, 41). Further support notion <br /> receptors. The ax-factor receptor encoded STE2 (4, 25),            G protein component pheromone response <br /> a gene expressed cells (44). The STE3 gene,              pathway comes finding STE4 STE18 <br /> is expressed cells (61), thought encode             products, required response phero- <br /> a-factor receptor necessary sufficient             mones, homologous , y subunits, respec- <br /> response a-factor (2, 14, 45). The hydropathic profiles            tively, G proteins (67). Other possible elements the <br /> these receptors, sequence homology,                    pheromone response pathway include STE5, STE7, <br /> predict similar structure, seven potential              STE]I, STE12 gene products. Although roles of <br /> transmembrane segments followed cytoplasmic car-                    proteins defined precisely, are <br /> boxyl-terminal domain hydrophilic (8, 14, 44). Indeed,           required cells pheromone response and <br /> several features proposed structure veri-              mating (19, 36). <br /> fied, STE3 product (9a). This structure                 To identify elements involved pheromone <br /> similar receptors, including                signaling, screened mutants suppress mating <br /> mammalian rhodopsins (47), 3-adrenergic receptors (12),              defect caused deletion a-factor-receptor gene. <br /> and muscarinic acetylcholine receptors (32). These mam-              Since cells lack a-factor receptors, not <br /> malian receptors believed initiate intracellular signal           respond a-factor unable mate. Cells that <br />                                                                          acquire new mutation activates pheromone re- <br />   * Corresponding author.                                                sponse pathway regain ability mate. Here, we <br />                                                                   2682 <br />  VOL. 9, 1989                                                                       A SUPPRESSOR OF RECEPTOR MUTATIONS                   2683 <br />  <br />                                                                  TABLE 1. Strains <br />      Strain                                                  Genotype                                                      Source <br /> DC5                       MATa leO2-3. -112 Ii.s3 gail2 ((an!                                        J. Strathern <br /> SY1229                    Isogenic DC5 ilro3                                             This work <br /> SY1263                    Isogenic SY1229 MATot                                          This work <br /> 227                       MATa lNsl/ Jr!J. Hicks 1. Herskowitz <br /> EG123                     MATa tirpl leit2 ira3 his4-519 (canI                                       K. Tatchell <br /> SY1346                    Isogenic EG123 lYs2                                            This work <br /> SM1229                    Isogenic EG123 tnfal::LEU2 tnfa2::URA3                         S. Michaelis l. Herskowitz <br /> SY1347                    Isogenic SM1229 lvs2 lYs5                                   This work <br /> X2180-1A-L1-U1            MATa gal2 Ivs2 mr-3 mal inel                                               J. Rothman T. Stevens <br /> HR125-5d                  MATa leii2-3. -112 lr(3-52 his3 liis4 trpl gal2                            R. Jensen 1. Herskowitz <br /> SY816                     Isogenic HR125-5d MATo.                                        R. Jensen I. Herskowitz <br /> SY817                     Isogenic HR125-5d MATa/MATAt                                   R. Jensen I. Herskowitz <br /> YY508                     Isogenic SY816 ste3::LEU2                                      D. Hagen <br /> 320                       MATa leil ura3 ade2 (cail-Il (cv1h2-21 r-olel                              Y. Kassir <br /> YY601&quot;                    MATa ste3::LEU2 lei2-3, -112 LEU2 mra3 ade2                             This work <br /> YY750                     MATot ste3::LEU2 itr-3 leO2 trpl his4-519 (an!l                            This work <br /> XR28-50c                  MATa leu2 trpl airo7(Am) lys2                                              J. Rine <br /> SY1077                    320 x mutagenized YY508: harbors srm11l-I                                  This work <br /> SY1115&quot;                   MAToi srIl-l lee2-3, -112 ura3 tirpl lis4                                  Segregant SY1077 <br /> YY811&quot;                    MATot ste3::LEU2 srmil-I Ile2-3. -112 imra3 his3 lhis4 trpl                Segregant SY1077 <br /> SY1154                    MATa ura3 leu2 mnetl4 ade2-loc Ode! his6 urplan sup4 SUP4-3             A. Bender <br /> SY1160                    lsogenic SY1154 baHrl::LEU2                                    A. Bender <br /> SY1325&quot;                   MATot sste3::LEU2 srml-l itrl3 Ien2 trpl atilel his4 Ihis3 HIS3 his6 Segregant YY811 x SY1154 <br />                             HIS6 mnetl4 MET14 <br /> SY1327&quot;                   MATot siIIl-1 uir(a3 leu2 trpl tide! ade2 his4 his3 HIS3 his6 HJS6   Segregant YY811 x SY1154 <br /> SY1355                    Isogenic SY1327 ste3::URA3                                     This work <br /> SY1357                    Isogenic SY1327 ste4::LEU2                                     This work <br /> SY1359                    Isogenic SY1327 ste5::LEU2                                     This work <br /> SY1383                    Isogenic SY1327 MATa ste2::LEU2                            This work <br /> SY1333&quot;                   MATot ste3::LEU2 leit2 itra3 trpl (idel ade2 ADE2 metl4 METJ4        Segregant YY811 x SY1154 <br /> SY1334&quot;                   MATot leu2 ira3 trpl adel ade2 liis4 hiis3 HIS3 lhis6 HIS6 metl4  Segregant YY811 x SY1154 <br />                             MET14 <br /> SY1392&quot;                   MATot srmnl-l ara3 lea2 idel trpl liis3 HIS3 his4 HIS4 lhis6      Segregant YY811 x SY1154 <br />                             HIS6 <br /> SY1393                    Isogenic SY139&apos; MATa                                           This work <br /> SY1394                    Isogenic SY1392 MATa ste2::LEU2                            This work <br /> SY1411&quot;                   MATa ste3::LEU2 leu2 wra3 tide! ade2 trpl /his3 HJS3 his6 HIS6       Segregant YY811 x SY1154 <br />                             metI4 MET14 <br /> SY1225&quot;                   MA Ta/MA Tot srmI-1/srinl- itra3lwra3 +1/eil aide2/+ (/liS3 HIS3)1      Diploid cosegregants of <br />                             (his3 HIS3) (/i.s4 HIS4)/(his4 HIS4)                               SY1077 <br />    These strains ((111l-ll CANI. (clz2-21 CYH2. mmil RMEI. <br />  <br /> report isolation characterization suppressor                     transforming SY1392 ste2::LEU2 DNA <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> (srml-1) deletion a-factor receptor gene.                           AB506 (kindly provided A. Burkholder L. Hartwell) <br />                                                                               pGAL-HO plasmid simultaneously. Similarly, strain <br />                MATERIALS AND METHODS                                          SY1383 generated simultaneous transformation of <br />   Strains media. The strains used experiments                   SY1327 ste2::LEU2 sequences AB506 the <br /> are presented Table 1. Details construction                 pGAL-HO plasmid. <br /> strains presented below. The one-step gene replacement                       Strains SY1346 SY1347 created selecting for <br /> method Rothstein (53) used create isogenic                  h&apos;s mutants EG123 (from K. Tatchell A. Mitchell) and <br /> srml-l mutant strains. Strain SY1357 generated                         SM1229 (kindly provided S. Michaelis), respectively, on <br /> transforming SY1327 (the relevant genotype MAToh srml-l                    ox-amino adipic acid medium (9). A itra3 mutant DC5 (from <br /> STE3 STE4 STE5) ste4::LEU2 DNA plasmid p121,                        J. Strathern) selected 5-fluoro-orotic acid medium <br /> kindly provided V. MacKay. Similarly, SY1327                           (by D. Hagen; 5),creating SY1229 switched <br /> transformed ste5::LEU2 DNA plasmid pSL567                        MAThs pGAL-HO (by A. Bender) create SY1263. <br /> create strain SY1359 ste3::URA3 (a complete dele-                        The media used YEPD (same YPD; 56), SD (56), <br /> tion STE3) create SY1355. The genotypes                        SD-URA SD-LYS (SD supplemented adenine, L- <br /> transformants confirmed Southern analysis (58),                       leucine, L-histidine, L-arginine, L-methionine, L-tyrosine, <br /> using nick-translated pSL8 (for STE3 URA3), p121 (for                     L-tryptophan, L-phenylalanine, L-threonine, L-lysine or <br /> STE4), SF9 (for STE5; provided V. MacKay)                            uracil; 56), SD supplemented individual nutrients <br /> probes. Strain SY1393 MATa derivative SY1392 (the                     (e.g., SD plus URA). <br /> relevant genotype MATot sr,nIl-I STE3 STE2)                           Mutant isolation. Four different cultures MATot ste3:: <br /> HO-promoted mating-type interconversion using construc-                     LEU2 strains mutagenized ethyl methane sul- <br /> tion places HO endonuclease gene control                   fonate described previously (42), surviving cells (4 <br /> of GALI promoter (pGAL-HO; 27). Strain SY1394                         25% mutagenized population) screened the <br />  2684     CLARK AND SPRAGUE                                                                                    MOL. CEt-L. BIOL. <br />  <br /> ability mate. Immediately inactivation ethyl           4-kb EcoRI fragment pSL737 yielded plasmid <br /> methane sulfonate, cells diluted YEPD broth        (pSL739) able complement sr-n/l-I <br /> plated YEPD plates density 0.7 x 10s 1.5 x 105       mutation. The BgIII-SalI fragment plasmid was <br /> viable cells plate. After 3 days alt room temperature.     able complement s,til-1 cloned YCp5O <br /> cells replica plated lawns MATa cells plates         (creating pSL749). In contrast, plasmid pSL742, was <br /> selective growth diploids. The selections   obtained deletion 0.8-kb BaimtiHI fragment from <br /> for diploid growth room temperature. Since         pSL739, lacked complementing activity (see Fig. 3A). <br /> anticipated bypass suppressor mutations lethal          The second method used locate SRMI gene within <br /> mutations, expected selection yield             insert transposon-mediated mutagenesis (22; system <br /> mutants partially inactive gene product         instructions generously provided M. Hoyt D. <br /> room temperature severe <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> elevated           Botstein). Plasmid pSL739 used mutagenesis. <br /> temperatures. The fourth selection duplicate        The ability various mutated plasmids complement <br /> room temperature 30&#194;&#176;C obtain larger group              sitIl-I determined described above. The location of <br /> bypass suppressors. The diploids sporulated dis-          transposon noncomplementing several <br /> sected. meiotic products analyzed             complementing plasmids determined restriction en- <br /> presence suppressor ste3::LEU2.                            donuclease digestion (see Fig. 3A). <br />    The selections examined 5 x 10&quot; muta-            Presumptive null alleles SRMI created cloning <br /> genized cells strain YY508. The culture replica       2.9-kb BglII fragment YEp13 (16) containing LEU2 <br /> plated lawns strain 320 cells (MA Ta STE3) SD plus        gene 1.2-kb Bta,nHI fragment pSL641 containing the <br /> uracil (20 mg/liter). The second cultures           URA3 gene unique BainHI site pSL742 create <br /> replica plated lawns strain YY601 (MATa stie3::LEU2)         pSL786 pSL828. respectively. pSL641 plasmid made <br /> on SD plus uracil. A total 11 diploids appeared        G. McCaffrey yeast URA3 gene cloned <br /> analyzed. One diploid apparently product rare         polylinker flanked BatnHI sites. <br /> mating event c ste3::LEU2 segregants              Several plasmids constructed RNA analysis. <br /> able mate, diploids showed evidence suppressor          pSL757 pSL760 2.3-kb BamHHI-Sall 0.8-kb <br /> mutations intractable genetically,           BimHI fragments, respectively, pSL749 cloned into <br /> harbored readily identifiable suppressor ste3::LEU2.          pSPT18 (Pharmacia), used make SP6- or <br /> The fourth selection examined 2 x 106 mutagenized cells         T7-generated RNA probes SRMJ mRNA. pSL743, which <br /> strain YY750, replica plated lawns strain         0.8-kb BaniHI fragment cloned YIp5 (pBR322 <br /> XR28-50c (MA Ta STE3) SD plus tryptophan (20 mg/liter).         containing yeast URA3 gene; 63), nick translated <br /> At 22TC, 121 diploids formed, 274 diploids ap-            used DNA probe SRMI URA3 mRNA. <br /> peared 30&#194;&#176;C. Nine diploids analyzed;             Two sets nested deletions created sequencing <br /> product rare mating events, seven            SRMI (see Fig. 3B). The 2.4-kb XlioI-PstI fragment of <br /> difficult analyze genetically. The suppressor identified     pSL749 cloned pBluescriptR-KS+ pBlue- <br /> the selection involving strains YY508 320 chosen       script&apos;-SK+ (Stratagene, Inc.) strands be <br /> intensive examination. This suppressor called sirInil-I        sequenced. Nested deletions plasmid by <br /> for suppressor receptor mutations.                              exonuclease III mung bean nuclease digestion (20; <br />    Cloning sequencing. To clone SRMI gene               Stratagene), using Exo/Mung deletion kit according to <br /> complementation temperature-sensitive lethality          instructions manufacturer (Stratagene). Single- <br /> sriml-l, transformed strain YY811 (the relevant genotype        stranded DNA prepared plasmids infect- <br /> is MATot srmnl-l ste3::LEU2) YCp5O library yeast         ing plasmid-bearing strains helper bacteriophage <br /> genomic sequences, 10 15 kilobase pairs (kb) <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> (51;     M13K07 (66) essentially described Stratagene. After 8 <br /> gift J. Thorner T. Stevens). YCp5O yeast shuttle       18 h incubation, single-stranded DNA prepared as <br /> vector harboring yeast URA3 gene yeast cen-              usual. Sequencing dideoxy-chain termina- <br /> tromere (51). Spheroplasts strain YY811 prepared,          tion method (54), using 7-deaza-dGTP nucleotide <br /> and samples transformed equal amounts             mixes (40). <br /> DNA. The transformations treated follows. For             A plasmid constructed test complementing <br /> one transformation, transformants selected room            activity open reading frame (ORF) revealed se- <br /> temperature screened ability grow           quencing. XlioI linkers added HindIll site of <br /> 370C. Of 1,180 transformants (which corresponds sam-          plasmid p126H (28). The EcoRI-XhoI fragment new <br /> pling genomic equivalent), able grow        plasmid, containing UAS TATA sequences the <br /> at 37&#194;&#176;C. For transformation, transformants          GAL] gene, cloned E(oRI XhoI sites of <br /> selected 37&#194;&#176;C initial period growth (16 h)       pSL749. Thus, GAL] UAS fused upstream the <br /> allow regeneration cell walls. This selection yielded      presumptive SRMI ORF. <br /> one transformant able grow 37&#194;&#176;C. DNA prepared               DNA manipulations. Bacterial transformations, bacterial <br /> from transformants used transform bacteria          DNA preparations, plasmid constructions per- <br /> ampicillin resistance. Examination plasmids revealed        formed standard methods (37). Yeast transformations <br /> that contained presumably identical 10.6-kb inserts (see      performed essentially described Beggs (1). Yeast <br /> Fig. 3); later analyses plasmid obtain-         DNA (plasmid chromosomal) prepared proto- <br /> ed directly 37&#194;&#176;C. This plasmid (pSL737) complemented            col Nasmyth Reed (46). Southern analyses (58) were <br /> su nl-l retransformed YY811            performed, hybridizations described pre- <br /> other sr-inzl-l mutants.                                           viously RNA analysis (61), using nick-translated plas- <br />    The SRMI gene mapped insert              mids kit Bethesda Research Laboratories. <br /> methods. The subclone fragments YCp5O               RNA analysis. RNA prepared yeast cells as <br /> and determine complement                described previously (61). Poly(A)+ RNA isolated with <br /> temperature sensitivity strain YY811 SY1115. Deletion        Hybond-mAP paper Amersham Corp. suggested by <br />  VOL. 9, 1989                                                           A SUPPRESSOR OF RECEPTOR MUTATIONS                   2685 <br />  <br /> the manufacturer. Hybridization analysis performed                                  RESULTS <br /> described (38, 61), using SP6-generated RNA probes <br /> of FUSI URA3. RNA probes opposite polarities               Isolation srml-l. To mate, haploid cells must <br /> identify SRMJ mRNA pSL757                 interact specific receptors secreted pheromones. <br /> pSL760, using SP6 T7 RNA polymerases Ribo-             Mutants make functional receptors phero- <br /> probe transcription Promega Biotec. DNA probes          mone unable mate (for recent reviews, 10, 21, 59). <br /> for SRMI URA3 generated nick translation           Binding pheromone receptor presumably generates <br /> pSL743, using kit Bethesda Research Laboratories. A        internal signal causes cell respond a <br /> Biomed Instruments scanning densitometer used              characteristic fashion prepare mate. To identify <br /> quantitate bands visible autoradiogram. The URA3        components internal signaling mechanism, began <br /> mRNA internal standard lane.                     strain internal substitution LEU2 <br />     Arrest assays. Cell cycle arrest induced shifting expo-    gene gene encoding a-factor receptor (ste3:: <br />                                                                   LEU2) isolated mutants regained ability to <br /> nentially growing cultures room temperature (22           mate. <br /> 24&#194;&#176;C) 33&#194;&#176;C 4 h assayed described previously            Strain YY508 (MA Ta ste3::LEU2) mutagenized with <br /> (15). At 200 cells sample counted      ethyl methane sulfonate detailed Materials Meth- <br /> knowing identity sample, experiment       ods. Approximately 5 x 106 cells examined, one <br /> done times. Unbudded cells cells           diploid possessing readily identifiable suppressor ste3:: <br /> have undergone projection formation, called shmoos,           LEU2 isolated. Two temperatures used assay <br /> considered G1 phase cell cycle. The morphol-      phenotypes conferred suppressor; 37&#194;&#176;C used <br /> ogy pheromone-arrested cells examined exposing          assess viability, 33&#194;&#176;C used assess mating <br /> strain SY1160 (MATa barl::LEU2; one-step gene             ability. The following genetic evidence indicates the <br /> replacement BARI strain SY1154) 625 ng a-factor       diploid SY1077 harbors suppressor ste3: :LEU2. Diploid <br /> (Sigma Chemical Co.) ml 4 h 30&#194;&#176;C.                      SY1077 grow 37&#194;&#176;C segregated fully viable <br />     Mating assays. Quantitative mating assays haploid          temperature-sensitive spores tetrad. In <br /> strains performed filter mating assay described      addition, MATa ste3::LEU2 segregants, those <br /> previously (19, 24) modifications: (i)      temperature sensitive mate, could <br /> threefold excess tester cells used; (ii) mating    mate 33&#194;&#176;C room temperature qualitative <br /> allowed proceed 6 h room temperature 33&#194;&#176;C;       mating assays. All MATa ste3::LEU2 segregants that <br /> (iii) mating cultures titered selective medium     grow 37&#194;&#176;C unable mate temperatures <br /> 30&#194;&#176;C determine number diploids room              tested. The segregation <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> observed an <br /> temperature different selective medium determine      outcross ste3 segregants mate <br /> number diploids plus haploids mating efficiency          33&#194;&#176;C (YY811 = MATa ste3::LEU2 srml-J) crossed to <br /> was determined (the experimental haploids). The <br />                                                                   SY1154 (MATa SRMI STE3). Two viable tempera- <br />                                                                   ture-sensitive spores tetrad observed, and <br /> mating efficiency defined number diploids ob-        three-fourths segregants mate 33&#194;&#176;C, but <br /> served selective medium divided number        one-half mate room temperature. Again, only <br /> of diploids plus experimental haploids observed            MATot ste3: :LEU2 segregants temperature <br /> second selective medium. Strain 227 usual tester      sensitive mate, 33&#194;&#176;C. Thus, tempera- <br /> a strains, isogenic strains SY1346 SY1347        ture-sensitive growth phenotype ability restore <br /> tester strains experiments determined a-          mating ax strains harboring ste3::LEU2 allele cosegre- <br /> factor required mating exhibited MATot             gated crosses. This cosegregation indicates single <br /> ste3::LEU2 srml-I cells. Strain SY1263 tester       mutation, designated srml-I suppressor receptor <br /> strains.                                                          mutations, responsible phenotypes. <br />    The mating efficiencies plasmid-bearing strains           MATaIMATot SRMIIsrml-J diploids able grow at <br /> assayed essentially described above. Cultures grown       37&#194;&#176;C, MATa/MATa srml-llsrml-1 diploids temper- <br /> overnight SD-URA. The cells pelleted              ature sensitive growth. Thus, srml-l recessive with <br /> centrifugation, suspended YEPD, used inoculate          respect temperature-sensitive growth phenotype. To <br /> YEPD 6 x 106 cells ml. These cultures           determine mating suppression phenotype of <br /> grown 1.5 x 107 cells ml room temperature,        srmI-I recessive, analyzed mating efficiencies <br /> and mating efficiencies determined described above.       potentially mating-competent diploids. MATalmata2 dip- <br /> To ensure plasmid-bearing cells assayed,        loids functional STE3 allele mate efficiently a <br /> used ura3 mutant strains (SY1263 test MATa strains         cells (62). Therefore, assayed ability MATotlmata <br /> SY1229 test MATot strains) tester strains. Diploids     ste3::LEU21ste3::LEU2 srml-JISRMI strains mate. We <br /> were identified plating mating mix SD plus <br />                                                                   diploids low mating effi- <br />                                                                   ciencies MATa ste3::LEU2 SRMI strains did (data not <br /> histidine (20 mg/liter), diploids plus plasmid-bearing        shown), indicating mating suppression phenotype of <br /> haploids identified plating SD-URA. As de-             srrml-I recessive. <br /> scribed above, mating efficiency calculated           The analysis described relied qualitative mating <br /> titer cells able grow SD plus histidine divided   tests, noted ste3::LEU2 srml-J strains <br /> titer cells able grow SD-URA.                            mated wild-type STE3 strains. To determine <br />    Qualitative mating tests (patch matings) using strain 227   degree srml-J restored mating receptor <br /> the tester described previously (60).                mutants, performed quantitative mating tests (Table 2). <br />    Microscopy. Micrographs fixed cells taken       The mating efficiencies 33&#194;&#176;C different ste3::LEU2 <br /> Zeiss Axioplan photomicroscope Nomarski optics,              srmnl-l mutants varied 10-4 10-2 constant <br /> using x 100 objective.                                          given strain. Some suppression observed at <br />  2686      CLARK AND SPRAGUE                                                                                                              MOL. CELL. BIOL. <br />  <br />                                                 TABLE 2. Suppression stc&apos;3::LEU2 srmll-I <br />                                                                                           MaIting efficiency strain with: <br />                           Genotype <br />  Strain                                                         M&apos;ATa MFal MFa2&quot;                          MATa lFa/ MFa2&quot;                  MATa mfal ,nfh2 <br />                    STE3              SRMI                22&apos;C                      33WC                           (33WC)                         (33&#194;&#176;) <br /> SY1334               +                +                  0.46                 0.34 0.69                                                       &lt;3.2 x 10     6 <br />  <br /> SY1327               +                -                 0.70       0.92       0.73                                                              1.6 x 10    4 <br />  <br /> SY1325               -                -              1.5 x 10   3.1 x 102-5. x 10                                                               1.0 x 10 <br /> SY1333               -                               2.8 x 10 &apos;.6 x1(      &lt;9.5x10 s                                                          &lt;9.5 x 10 <br />     Results average duplicate triplicate experiments. Strain 227 tester. <br />   &apos; Results atverage duplicate experiments. The MATa M,Fal MFa2 strain SY1346. MATa        mzfal mif.2 strain SY1347. Genotypes are <br /> presented Table 1. <br />  <br /> the permissive temperature, suggesting .sr,nIl-I prod-              isolates (including SY1383) assayed room tempera- <br /> uct altered activity room temperature.                   ture 33&#194;&#176;C. Again, 33&#194;&#176;C, quite good mating was <br /> expected given mutant isolation scheme used.                             observed; roughly 6% mutant population mated. <br />    Lack requirement a-factor. These results suggest                   Remarkably, 0.8% MATa ste2::LEU2 srmi-l <br /> that srmnl-i activate signal transduction pathway                 mutants mated room temperature. This value much <br /> a pheromone-independent fashion. However, ex-                      higher mating efficiency observed MATa ste2 <br /> planations possible. One truncated, partially                  SRMI strains (1.6 x 10-i SY1383 harboring pSL749 <br /> functional receptor synthesized ste3::                     [Table 3]) suggests, argued experiments <br /> LEU2 allele (which internal substitution                 involving Matct ste3 srmEn l-I strains, sr ml-l mutant <br /> complete deletion [14]) altered receptor                 product fully functional room temperature. Conse- <br /> interact a-factor create weak signal enhanced              quently, low temperatures, srml-l mutation <br /> by sirt l-l mutant product. Alternatively, srmIl-l                 creates signal pheromone response pathway that <br /> allow cryptic a-factor receptor function. In case,               allows mating, albeit inefficiently. <br /> mating occur mating partner secreting                   Induction FUSI transcription. The s,nIl-l mutation was <br /> a-factor. We examined ability srmn l-I mutants mate                identified basis suppression mating defect <br /> with strain (SY1347) unable make a-factor                   ox ste3::LEU2 mutants. Since mating culmination of <br /> because genes encoding a-factor deleted (39).                   number physiological changes response phero- <br /> srinl-I mutants, SRMI strains, able mate                     mone, examined particular physiological changes, such <br /> with pheromone-deficient strain (Table 2). Moreover,                    transcriptional induction FUSI (38, 65) cell cycle <br /> the MAToC ste3::LEU2 srml-I mutant mating                       arrest (see below), determine srmnl-l mutants <br /> efficiency pheromone-producing phero-                      exhibit aspects normal pheromone re- <br /> mone-deficient strains. These results indicate a-factor              sponse. When exponentially growing cultures sr,nl-1 <br /> not required (x srml-l mutants mate                      mutants shifted room temperature 33&#194;&#176;C, they <br /> observed mating presence cryptic                   increased amounts FUSI transcript (Fig. 1). In <br /> truncated a-factor receptor. In addition, srinl-l suppresses                 contrast, SRMI strains increase FUSI transcrip- <br /> the mating defect mutant allele STE3                    tion shifted 33&#194;&#176;C (Fig. 1C). The increase FUSI <br /> entire STE3 promoter coding sequences deleted (sste3::                   transcript levels srInl-l mutants ranged 7- 17-fold <br /> URA3; data shown). Thus, .si-nl-l appears circumvent                  comparable regardless cell type (a ax) or <br /> all receptor-mediated functions.                                             status receptor gene (Fig. 1 data shown). <br />    Suppression a-factor-receptor mutation. Genetic                    Moreover, induction seen result temperature shift <br /> physiological experiments indicate pheromone                    comparable seen MATot STE3 srmnl-l <br /> receptors activate common response pathway (2, 19, 36,                     mutants incubated permissive temperature treated <br /> 45). To determine SRMI pathway                         a-factor (Fig. 1). Thus, srtml-1 mutants exhibit phe- <br /> common cx cells defines CL-specific                notype characteristic pheromone response are <br /> part, examined ability srn)1-l suppress                       incubated nonpermissive temperature absence <br /> mutation cL-factor-receptor gene (ste2::LEU2). A                      pheromone. <br /> MATot STE2 sriinl-l strain (SY1327) changed MATa                        The magnitude FUS1 induction ot sr I1-I <br /> ste2::LEU2 srinl-1 (strain SY1383) gene replacement at <br /> the STE2 locus HO-mediated mating-type intercon-                                   TABLE 3. srninl-l suppresses ste2::LEU2 not <br /> version MAT locus (see Materials Methods). To                                           ste4::LEU2 ste5::LEU2 <br /> create isogenic srmIl-l SRMI strains, transformed <br /> SY1383 YCp5O pSL749, YCp5O derivative                                                                                Mating efficiency&quot; of <br /> carrying SRMI gene (see below) determined                          Strain                  Genotype                       strain harboring: <br /> mating efficiencies transformants 33&#194;&#176;C (Table 3).                                                                       pSL749            YCp5O <br /> The mating efficiency MATa ste2::LEU2 srml-I                                         MATa ste2::LEU2 srmnl-J <br /> mutants harboring SRMI plasmid, pSL749, low <br />                                                                              SY1383                                              1.6 x 10-s      4.4 x 10--- <br />                                                                              SY1357         MATot ste4::LEU2 srml-/            &lt;1.9 x 10-7      &lt;6.4 x 10- 6 <br /> (10-5 [Table 3]). However, mating efficiency MATa                     SY1359         MATot ste5::LEU2 sriml-1           &lt;1.1 X 10- 7     &lt;1.8 x 10-6 <br /> ste2::LEU2 srinl-l mutants harboring YCp5O 4 x <br /> 10-2. Thus, srm I-I suppresses deletion STE2 STE3                     &quot; The mating efficiencies strains SY1383. SY1357. SY1359 harboring <br /> about extent (compare Tables 2 3).                              pSL749 (SRMI YCp5O) YCp5O determined 33&#194;&#176;C (see Materials <br />                                                                              Methods). Two transformants assayed, results were <br />    As second way compare mating efficiency                     averaged. No diploids observed SY1357 SY1359. data <br /> and SRMI, MATa ste2::LEU2 srmnl-I                        presented represent limit detection assay. <br />  VOL. 9. 1989                                                                 A SUPPRESSOR OF RECEPTOR MUTATIONS                                2687 <br />  <br />                                                                                   TABLE 4. Accumulation G1 srtnIl-I mutants <br />       A                                  B        1   2     3                           Re-                        %   GI&quot;          % Shmoo&apos; <br />                  1    2    3                                             Strain     MAT cep-      SRMI <br />                                                                                             tor                22&#194;&#176;C        33&#194;&#176;C      22&#194;&#176;C    33&#194;&#176;C <br />                                                                        SY1333             -       +     41       5      43      4   0       0 <br />      FUS1                                FUS1                          SY1411             +       +     35       1      49      7   0       0 <br />                                                                        SY1325             -       -     52       7      73      7 2 &#194;&#177; 2 35 &#194;&#177; 9 <br />      URA3                                URA3                          SY1327&apos;            +       -     51       4      70      5 3 &#194;&#177; 3 15 &#194;&#177; 9 <br />                                                                        SY1355&apos;            -       -     62       3      86      3 15&#194;&#177; 1 64 &#194;&#177; 1 <br />                                                                        SY1383&apos;            -       -     70   &#194;&#177;   9      69      9 27&#194;&#177; 12 38 &#194;&#177; 15 <br />                                                                        SY1392             +       -     51   &#194;&#177;   3      68      5 3 &#194;&#177; 2 20 &#194;&#177; 7 <br />       Temp.      22 22 33             Temp. 22 22 33                   SY1393             +       -     52   &#194;&#177;   11     68      6 9 &#194;&#177; 6 19 &#194;&#177; 7 <br />                                                                        SY1394             -       -     59   &#194;&#177;   7      63      5 17&#194;&#177; 7 13 &#194;&#177; 5 <br />         aF       -+-                         aF   -   +                SY1225        aa     -&apos;      -     36   &#194;&#177;   4      29      8   0       0 <br />                                                                           &quot; The percentage cells GI (mean &#194;&#177; standard deviation) presented. <br />                                                                        Each strain tested times. <br />                                                                           &quot;The percentage cells total population projection <br />                       C          1    2      3    4 <br />                                                                        formations (shmoos) (mean &#194;&#177; standard deviation) presented. <br />                                                                          ` <br />                                                                             SY1327 SY1355 isogenic STE3 locus. SY1327 and <br />                                                                        SY1383 isogenic MAT STE2 loci. <br />                                                                           d SY1392 SY1393 isogenic MAT locus, SY1393 <br />                                      1234                              SY1394 isogenic STE2 locus. <br />                                                                           &quot; SY1225 a/la diploid. STE2 STE3 expressed. SY1425 <br />                       FUSi                                             homozygous srml-I mutation. <br />  <br />                       URA 3          a <br />                                                                        tions. called shmoos (Table 4 Fig. 2). In contrast, the <br />                                                                        morphology SRMI cells affected shift <br />                                                                        (Table 4 Fig. 2). The shmoos induced 33&#194;&#176;C srml-J <br />                       Temp. 22 33 22 33                                mutants similar induced wild-type cells by <br />                       SRM1                                             pheromone (Fig. 2). Hence, altered activity srml-I <br />                                                                        protein caused temperature shift mimics cell cycle <br />    FIG. 1. Induction FUSI transcription. (A) Total RNA          arrest response pheromone response pathway. <br /> harvested exponentially growing cultures strain SY1115            Interestingly, extent arrest proportion of <br /> (MATa srml-l). In cases, cells grown 22&#194;&#176;C         cells showing projections varied receptor status of <br /> (-) (+) induction a-factor (aF) 90 min,    srtnl-I mutants (Table 4). When STE3 replaced by <br /> cases, cells shifted 22 33&#194;&#176;C 1 h. The RNA was <br /> fractionated 1.5% agarose-6% formaldehyde gel, blotted         ste3::URA3 (a complete deletion STE3), strain that <br /> nitrocellulose, hybridized SP6 probes FUSI URA3       lacked receptors exhibited arrest more <br /> mRNAs described Materials Methods. Similar results      shmoos (compare SY1327 SY1355 [Table 4]). This result <br /> obtained cultures incubated 33&#194;&#176;C 1 4 h. (B)   suggests wild-type receptor dampening <br /> Total RNA harvested cultures strain YY811 (MATa            effect signal generated shifting srml-l mutants to <br /> ste3::LEU2 srmnl-l). The cultures grown described panel    330C. <br /> A incubated 33&#194;&#176;C 4 h (rather 1 h)       We examined effect srmnl-l cell cycle <br /> before harvested. The RNAs treated described         morphology cells. To compare srml-I pheno- <br /> panel A. (C) Total RNA harvested cultures strain           type cells, constructed isogenic strains. MATa <br /> YY811 harboring pSL749 (SRMI YCp5O; lanes 1 2) YCpSO <br /> (lanes 3 4). The cultures grown described panel B,     srmnl-l strains switched MATa using HO endo- <br /> and RNAs treated described panel A.                    nuclease gene control GALI promoter (see <br />                                                                        Materials Methods). Both cells recovered <br />                                                                        regimen; cells behaved like original a <br /> mutants, caused temperature shift      phero-          strain, indicating transformation switching pro- <br /> mone,   observed (38, 65).            cedures deleterious. The srmnl-l mutants exhib- <br /> The reduced induction seen probably high              ited phenotype srrml-I mutants cell <br /> basal level FUSI transcription srmnl-l mutants,           accumulation GI projection formation (compare <br /> than reduction quantity FUSI transcript                 SY1327 SY1383, SY1392 SY1393 [Table 4]). <br /> present induction, basal level FUSI RNA            Thus, srml-l mutants exhibit characteristic features of <br /> greatly reduced plasmid pSL749 (SRMI cloned                  pheromone response shifted nonpermissive <br /> YCp5O; below) present srml-I strain (Fig. 1C).            temperature. <br /> The high basal level indication                  Two differences srml-l srml-l mutants <br /> mutant product fully functional permissive          worth noting. First, contrast findings a <br /> temperature.                                                           srmnl-l mutants, presence absence receptor did not <br />   Accumulation G1 phase cell cycle. Another              influence phenotype srmnl-l mutants (SY1393 and <br /> response  pheromone arrest cells G, phase          SY1394 [Table 4]). Second, srml-l mutants <br /> of cell cycle (7, 68). To examine cell cycle arrest             showed dramatic change proportion cells G1 <br /> srml-l mutants, shifted exponentially growing cultures              projections cultures shifted per- <br /> of mutants nonpermissive temperature 4 h,               missive nonpermissive temperature, srml-l mutants had <br /> fixed cells formaldehyde, determined cell             essentially proportion cells G, with <br /> cycle profile population. Shifting srmnl-l mutants           projections temperatures (Table 4). As already <br /> 33&#194;&#176;C caused accumulate GI form projec-               noted, believe mutant protein present srmnl-l <br />  2688          CLARK AND SPRAGUE                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />   A <br />                     N~~~~~~O <br />  <br />  <br />  <br />        .40.    N <br />               04 <br />  <br />  <br />  <br />  <br />                                                 Ei *i, <br />                                                    X X               46                        F                     n <br />  <br />         01% <br />                    &quot;on)   wov <br />                                                              ,tW*. <br />                                                                        .. <br />                                                                             &apos;* <br />  <br />  <br />  <br />                                                                                                 A. I&apos;ll,-N <br />                                                                                                           4., <br />                                                                                                                  e 4p <br />                                                                                                                       fd <br />                            fA*.      I <br />  <br />   FIG. 2. Cellular morphology srml-J mutants. Related wild-type (A D) srml-l mutant (B E) cells growing exponentially at <br /> 22&#194;&#176;C (A B) shifted 33&#194;&#176;C 4 h (D E). For comparison, exponentially growing MATa barl :LEU2 cells (C) exposed to <br /> a-factor (625 ng/ml) 4 h (F). The strains SY1333 (A D; MATa ste3::LEU2 SRMI), SY1325 (B E; MATa ste3::LEU2 srml-I), <br /> and related strain SY1160 (MATa SRMI barl::LEU2). Photographs fixed cells taken described Materials Methods. <br />  <br /> strains fully functional temperatures permissive        determine increased projection formation at <br /> growth cell division. The subtle differences               33&#194;&#176;C dominant altered function srml-l <br /> phenotypes srml-J srml-I mutants suggest              protein, examined morphology MATalmata2ste3 :: <br /> the cell types react differently differences            LEU21ste3::LEU2 srml-JISRMJ diploids. These diploids did <br /> quantity quality mutant srml-I product.                     arrest exhibit projections 33 37&#194;&#176;C (data not <br />   As noted above, a/a cells homozygous srml-J                 shown), indicating srml-l does dominant <br /> temperature sensitive growth. In addition,          phenotype 33&#194;&#176;C. <br /> they did exhibit cell cycle arrest projection formation          Cloning expression SRM1. SRMJ cloned by <br /> (strain SY1225 [Table 4]). This phenotype reminiscent           complementation temperature-sensitive phenotype of <br /> cdc36 cdc39 mutants, arrest G1 haploid.             srml-J described Materials Methods, using a <br /> However, a/a diploids homozygous cdc36                library yeast genomic fragments cloned YCp5O (51). <br /> cdc39 arrest uniform position cell cycle           Two apparently identical clones 10.6-kb insert were <br /> although temperature sensitive (49, 57). To            obtained. One, named pSL737 (Fig. 3), shown to <br /> determine srml-J allelic cdc36, cdc39,             complement temperature sensitivity mating <br /> with mutant genes confer G1 arrest,               phenotypes ste3::LEU2 srml-J mutants; plasmid- <br /> cdc28 cdc37 (49), srml-llcdc diploids                  bearing cells grow restrictive temperature and <br /> mating. The cdc mutations complemented srml-I, anal-              mate temperature. <br /> ysis meiotic products heterozygous diploids              To determine pSL737 really harbors SRMJ and <br />  revealed srml-1 allelic cdc genes      gene fortuitously complements temper- <br />  (data shown).                                                    ature sensitivity srml-J, examined plas- <br />    The experiments described performed 33&#194;&#176;C.            mid integrates homologous recombination SRM1 <br />  We examined phenotype srml-J mutants 37&#194;&#176;C.            locus. We transformed SY1115 (the relevant genotype is <br />  Shifting srml-l mutants 37&#194;&#176;C caused accumulate            MATa srml-1) BglII-cut pSL744 (EcoRI SphI of <br />  G1 essentially extent occurred              pSL737 YIp5) identified transformants the <br />  incubation 33&#194;&#176;C. However, fewer projections formed           plasmid integrated locus corresponding the <br />  37&#194;&#176;C formed 33&#194;&#176;C (data shown). To                cloned DNA. One transformant crossed a <br />  VOL. 9, 1989 <br />  <br />                          A <br />  <br />  <br />  <br />                        B/.Sa <br />  <br />  <br />                          -L <br />  <br />  <br />  <br />                          B <br />                                   ~ ~I <br />                                   E B <br />                                           1 kb <br />  <br />  <br />  <br />  <br />                               -~~~~~~~~~~~~ ----- <br />  <br />  <br />  <br />  <br />                                             x_ <br />  <br />  <br />  <br />                                             -.40 <br />                                                    -~~ <br />                                                          P PE S <br />  <br />  <br />  <br />  <br />                                                          -- <br />                                                      -b0- -4-~ <br />                                                          -~ <br />                                                               - <br />  <br />  <br />  <br />  <br />                                                               - <br />                                                                   ~-0 <br />  <br />  <br />  <br />  <br />                                                                   ~-00-1 <br />                                                                         x <br />  <br />  <br />  <br />                                                                         t <br />  <br />  <br />  <br />    FIG. 3. SRMJ gene. (A) A yeast genomic fragment (E1) cloned YCp5O ( <br />                                                                             Bg <br />                                                                             1 <br />                                                                                  A SUPPRESSOR OF RECEPTOR MUTATIONS <br />  <br />  <br />  <br />  <br />                                                                                     t <br />                                                                                     p <br />                                                                                      I <br />                                                                                          B <br />                                                                                          + <br />                                                                                              Bs 4; <br />  <br />  <br />  <br />  <br />                                                                                               0.5 kb <br />                                                                                                          P <br />                                                                                                              ACTIVITY <br />  <br />  <br />                                                                                                              _ <br />  <br />  <br />  <br />  <br />                                                                                                              Sa/B <br />                                                                                                                  |S <br />                                                                                                                     +T1 <br />                                                                                                                                          2689 <br />  <br />  <br />  <br />  <br />                                                                                       ) complements temperature-sensitive phenotype <br /> of srml-J mutant (the plasmid pSL737). The arrows box ( I ) indicate sites transposon insertion disrupt <br /> complementation, arrows box ( T ) indicate insertion sites disrupt complementation. The subclones pSL737 <br /> (E) (+) (-) complement srmi-J ORF SRMI (0) shown. Abbreviations: E, EcoRI; B, BamHI; Sa, Sau13A; <br /> P, Pstl; S, Sall; X, Xhol; Bg, BgIII. The entire clone tested Xhol BgIII sites. (B) The box represents fragment pSL737 <br /> that subcloned sequencing. Sets deletions created Xhol PstI sites, arrows indicate deletion subclones <br /> that sequenced direction sequencing (see Materials Methods). The ORF (0) direction transcription (-) are <br /> also shown. <br />  <br /> wild-type strain (X2180-1A-L1-U1), meiotic prod-                    147 include 11 acidic amino acids 19 total residues, and <br /> ucts analyzed. None segregants diploid                  residues 470 482 include 8 acidic 2 basic amino acids <br /> (15 tetrads examined) temperature sensitive, indicating                13 total residues (Fig. 4). Comparison sequence of <br /> that pSL744 integrated srml-J locus. Therefore,                  ORF National Biomedical Research Foundation <br /> the cloned DNA linked SRMI gene.                                  protein data <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> using program FASTP (35) did not <br />    To determine multiple copies SRMI gene                    reveal strong homologies. We searched sequences <br /> were present yeast genome, performed high-                        similar consensus sequences known functional <br /> stringency hybridizations yeast genome                  groups proteins, e.g., zinc fingers (29), calcium-binding <br /> cloned SRMJ gene. The DNA fragments observed                        EF hands (31), protein kinase sequence motifs (17), and <br /> Southern blots predicted restriction map                sequence motifs common subunit G proteins (64). <br /> of SRMJ gene, indicating copy                      The closest identity residues 60 66 (GLGPLAK) to <br /> SRMJ gene present yeast genome. These experi-                     consensus nucleotide, particularly GTP, binding site <br /> ments revealed sequences pSL739 (an                    (GXGXXGK [16]). <br /> srml-l-complementing plasmid created deletion                        A transcript approximately 1.6 kb detected all <br /> 4-kb EcoRI fragment pSL737 [Fig. 3A]) colinear                       cell types (a, a, ala) (Fig. 5). The quantity this <br /> within genome (data shown).                                         transcript unaffected addition pheromone (Fig. 5). <br />    The SRMJ gene mapped 10.6-kb insert                    Hybridization poly(A)+ RNA single-stranded probes <br /> subcloning transposon-mediated mutagenesis (22)                      revealed SRMI transcript orientation <br /> (Fig. 3). These procedures defined small region clone              ORF (data shown). <br /> to interest, sequenced 1.8 kb region (Fig.                SRMI essential gene. Two presumptive null alleles of <br /> 3). One ORF encoding 482 amino acids (if ATG                      SRMI created replacing 0.8-kb BamHI fragment <br /> codon serves start codon protein) ob-                    SRMI LEU2 URA3 (Fig. 3 Materials <br /> served (Fig. 4). Because ORF disrupted deletions                 Methods). A heterozygous srml-lIsrml::LEU2 diploid <br /> or transposons destroy complementing activity,                      created one-step gene replacement. Tetrad analysis <br /> presume corresponds SRMI product. Moreover,                       room temperature revealed diploid segregated two <br /> fusion GAL] promoter upstream ORF (at                    viable inviable spores. None viable spores <br /> XhoI site) produced hybrid gene showed galactose-                    Leu+, indicating SRMJ essential gene. <br /> dependent complementation srml-1.                                        Because srml-J mutants tended arrest G1 unbudded <br />   The hydropathic profile (33) deduced protein se-                   cells cells projections, expected srml:: <br /> quence predicts hydrophilic protein. Two regions                   LEU2 cells display similar phenotype. However, the <br /> protein stretches charged residues: residues 129                 inviable (srml::LEU2) spores germinated arrested as <br />  2690     CLARK AND SPRAGUE                                                                                         MOL. CELL. BIOt. <br />  <br />                                   -88   CTCGAGAATATTAAAAGTCATCAAGACAT1CCTCTTTAAGTCAAATCCTGGATCCA(,AAACTTAAAAAG1TrGTCATATAAAGTeTTAAA <br />    1   ATGGTCAAAAGAACAGTCGCCACCAATGGAGACGCGTCTGGCGCTCATAGAGCAAAGAAAATGTCTAAAACTCATGCTTCTCATATCATAAACGCTCAAGAAGACTACAAGCATATGTAC <br />    1   MetValLysArgThrValAlaThrAsnGlyAspAlaSerGlyAlaHisArgAlaLysLysMetSerLysThrHisAlaSerHisIlelleAsnAlaGlnGluAspTyrLysiHisMetTyr <br />  121   TTAAGTGTCCAACCATTGGATATATTTTGCTGGGGTACTGGTTCTATGTGTGAGCTTGGGTTGGGTCCACTAGCCAAAAATAAGGAAGTTAAAAGACCAAGGTTAAACCCATTTTTACCT <br />   41   LeuSerValGlnProLeuAspIlePheCysTrpGlyThrGlySerMetCysGluLe*lyLeuGlyProLeuAlaLyssnLysGIuValLysArgProArgLeuAsnProPheLeuPro <br /> 241    CGCGATGAGGCCAAGATCATTTCTTTTGCAGTCGGTGGTATGCACACCTTAGCCCTTGATGAAGAAAGCAATGTATGGTCCTGGGGTTGCAATGACGTGGGTGCGTTGGGCAGAGATACT <br />  81    ArgAspGluAlaLysIleIleSerPheAlaValGlyGlyMetHisThrLeuAlaLeuAspGluGluSerAsnValTrpSerTrpGlyCysAsnAspValGlyAlaLeuGlyArgAspThr <br /> 361    TCCAACGCCAAGGAACAACTCAAGGACATGGACGCCGATGACTCAAGTGATGACGAAGATGGTGATTTGAACGAATTGGAATCTACGCCCGCTAAAATCCCAAGAGAGTCATTTCCTCCC <br />  121   SerAsnAlaLysGluGlnLeuLysAspMetAspAlaAspAspSerSerAspAspGluAspGlyAspLeuAsnGluLeuGluSerThrProAlaLysIleProArgGluSerPheProPro <br />  481   TTAGCGGAGGGACACAAGGTCGTGCAATTGGCAGCTACAGACkATATGAGTTGCGCCCTGTTCAGCAATGGCGAGGTATACGCATGGGGTACTTTCCGTTGCAATGAAGGTATCCTAGGG <br />  161   LeuAlaGluGlyHisLysValValGlnLeuAlaAlaThrAspAsnMetSerCysAlaLeuPheSerAsnGlyGluValTyrAlaTrpGlyThrPheArgCysAsnGluGlyIleLeuGly <br /> 601    TTTTACCAAGATAAAATTAAAATTCAAAAAACTCCATGGAAAGTGCCTACATTCTCTAAGTACAATATTGTCCAATTAGCTCCCGGTAAGGATCACATACTTTTCTTAGATGAAGAGGGT <br /> 201    PheTyrGlnAspLysIleLysIleGlnLysThrProTrpLysValProThrPheSerLysTyrAsnIleValGlnLeuAlaProGlyLysAspHisIeLeuPheLeuAspGluGluGly <br /> 721    ATGGTGTTCGCCTGGGGTAACGGTCAGCAAAACCAATTAGGAAGAAAAGTTATGGAAAGGTTCCGTTTAAAGACTTTGGATCCCAGACCATTTGGGTTAAGACATGTAAALATACATCGCA <br /> 241    MetValPheAlaTrpGlyAsnGlyGlnGlnAsnGlnLeuGlyArgLysValMetGluArgPheArgLeuLysThrLeuAspProArgProPheGlyLeuArgHisValLysTyrIleAla <br />  841   TCTGGCGAAAACCATTGCTTTGCTTTAACAAAAGACAACAAGTTAGTAAGCTGGGGGTTGAATCAGTTTGGTCAATGTGGTGTTTCAGAGGATGTGGAAGACGGTGCGCTTGTTACAAAG <br />  281   SerGlyGluAsnHisCysPheAlaLeuThrLysAspAsnLysLeuValSerTrpGlyLeuAsnGlnPheGlyGlnCysGlyValSerGluAspValGluAspGlyAlaLeuValThrLys <br />  961   CCAA.AAAGACTAGCGCTACCTGATAACGTTGTTATCAGAAGCATTGCTGCTGGTGAACATCACTCATTGATCCTATCCCAAGATGGCGATTTGTATTCGTGTGGTAGACTAGACATGTTT <br />  321   ProLysArgLeuAlaLeuProAspAsnValValIleArgSerIleAlaAlaGlyGluHisHisSerLeuIleLeuSerGlnAspGlyAspLeuTyrSerCysGlyArgLeuAspMetPhe <br /> 1081   GAAGTTGGTATCCCAAAAGACAATTTACCCGAATATACTTACAAAGATGTTCATGGAAAAGCACGTGCCGTGCCACTTCCAACTAAGTTGAATAATGTTCCGAAGTTCAAAAGCGTTGCG <br />  361   GluValGlyIleProLysAspAsnLeuProGluTyrThrTyrLysAspValHisGlyLysAlaArgAlaValProLeuProThrLysLeuAsnAsnValProLysPheLysSerValAla <br /> 1201   GCGGGGTCTCACCATTCAGTCGCGGTAGCTCAAAATGGCATTGCCTATTCTTGGGGTTTTGGTGAAACCTACGCCGTTGGATTAGGCCCTTTTGAGGACGACACTGAAGTTCCAACCAGA <br />  401   AlaGlySerHisHisSerValAlaValAlaGInAsnGlyIleAlaTyrSerTrpGlyPheGlyGluThrTyrAlaValGlyLeuGlyProPheGluAspAspThrGluValProThrArg <br /> 1321   ATCAAGAATACTGCTACACAGGATCATAACATCATTTTAGTCGGGTGTGGAGGCCAATTTTCCGTGTCTGGTGGTGTTAAGCTGTCTGATGAGGACGCAGAAAAGAGAGCGGATGAAATG <br />  441   IeLy5AsnThrAlaThrGInAspHisAsnllelleLeuValGlyCysGlyGlyGlnPheSerValSerGlyGlyValLysLeuSerAspGluAspAlaGluLysArgAlaAspGluMet <br /> 1441   GATGATTAAGGTAGTAGTACAAAGATAAATAAAAGAAAATCGACATAATGCACCTCAGTATAAATTGTAAAATATAACATAAATCAAAGAAAAACGAGAAAACGATATGATTATCCGATA <br />  481   AspAsp   482 <br />  <br /> 1561 <br />  <br /> 1681   ATGAAATCAACACTGAGTAAGTATTATGTAATCCATCCGCTCGTCCAGCCCTTCAGGTTCTTCCCTCTTCTGCACGT      1757 <br />   FIG. 4. Sequence SRMI translation product. A region 1.835 kb Xliol site Pstl site SRMI Was <br /> sequenced described Materials Methods. The translation ORF shown nucleotide sequence. Two in-frame <br /> methionine residues near amino terminus underlined twice. The boxed amino acids weak match nucleotide binding site <br /> (GXGXXGK [16]). Two stretches charged (predominantly acidic) residues underlined. The standard three-letter abbreviations for <br /> amino acids used. <br />  <br /> one large-budded cell physically separated       advantage. Consistent interpretation obser- <br /> two unbudded (G1) cells 12% time. The rest          vation continued propagation dip- <br /> the large-budded cells represent cells arrested G1       loids yielded tetrads viable spores <br /> that remain physically joined, G1 initiates end     resulted producing tetrads viable <br /> mitosis cytokinesis cell separation (48). As viable      spores (data shown). An alternate hypothesis based <br /> a spores produced, inferred inviable         analysis TUBI gene, encodes x-tubulin. <br /> spores cells, indicating <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>null allele</span>     Aneuploidy observed deletions TUBI were <br /> has phenotype cell types. A total         created (55), SRMI product, like TUBI <br /> independent srml-I/srmm::LEU2 transformants (of dif-            product. required fidelity chromosome segregation. <br /> ferent diploids) analyzed results.                   Lack suppression ste4 steS mutations. Several <br />    Given results, properties additional trans-    genes identified products required for <br /> formants unexpected suggested diploid cells        pheromone response mating. These include STE4 and <br /> the quantity mutant srinl-I product gene         STE5, required ox cells (19, 36). To <br /> insufficient optimal growth. These transformants (both          determine order action respect SRMI in <br /> srml-l/srml ::LEU2 srm1-//srml:: URA3) yielded tet-             pheromone response pathway, double <br /> rads viable spores. Some spores         mutants MATot ste4::LEU2 s}srml-I MATo ste5::LEU2 <br /> tetrads Leu- Ura&apos;, indicating sr-inl null          srl-t71/- gene disruption STE4 STE5 strain <br /> allele present. Outcrossing Ura1 haploids          SY1327 (MATot si/-it-1). The ste4::LEU2 srmll-1 ste5:: <br /> to wild-type strain revealed haploid Ura strains         LEU2 smnl-l mutants (SY1357 SY1359, respectively) <br /> were aneuploid harbored sirIIIl-l srnzl::URA3          did arrest exhibit projections were <br /> on distinct chromosomes segregated meiosis (data        shifted 33&#194;&#176;C (data shown), starting strain <br /> shown). Thus, set diploid transformants            ste3:: URA3 derivative starting strain did arrest <br /> have undergone nondisjunction event mitosis          exhibit projection formation (SY1327 SY1355 [Table <br /> caused particular transformants growth              4]). SY1357 SY1359 temperature sensitive for <br />  VOL. 9. 1989                                                                  A SUPPRESSOR OF RECEPTOR MUTATIONS                      2691 <br />  <br />                              1     2       3       4    5                   The srinl-l mutants exhibited phenotype mimicked <br />                                                                          response wild-type cells pheromone. In particular, at <br />                                                                          elevated temperatures, srInl-I mutants induced expression <br />                  SRM1                                                    FUSI, accumulated G1 phase cell cycle, and <br />                                                                          formed projections. The extent accumulation G1 was <br />                                                                          great occurs pheromone induction. <br />                  URA3                                                    One explanation failure cells arrest G, is <br />                                                                          partial adaptation signal generated permissive <br />                                                                          temperature srinIl-l mutation. The high basal level of <br />                 MAT                              a,&quot;               FUS1 RNA ability receptorless srin l-I mutants to <br />                                                                          mate inefficiently, room temperature, suggest at <br />             Pheromone        -     +               +                     low temperatures srinl-l mutation creates feeble signal <br />     FIG. 5. SRMI transcription. Total RNA harvested             pheromone response pathway. If cells were <br /> exponentially growing cultures MA Ta (a), MATot (oa), MA Ta/a_     adapted presence signal low temperature, they <br /> (a/a) cells untreated (-) treated appropriate      respond increased signal high temperature <br /> pheromone 90 min (-+ indicates a-factor cells, a-factor    arrest G1. The result a <br />  cells). The strains isogenic series. HR125-5d. SY816. and <br />                                                                          nonhomogeneous response. <br /> SY817 (Table 1). The RNA fractionated 1.5% agarose-6%              Surprisingly, strength GI accumulation pheno- <br /> formaldehyde gel. blotted nitrocellulose. hybridized         type sensitive presence absence phero- <br /> nick-translated pSL743, detects SRMI URA3 transcripts. <br /> as   described Materials Methods.                                 mone receptor, especially a( cells; extent accumu- <br />                                                                          lation GI projection formation STE3 <br />                                                                          srmI2-l cells ste3 srtnIl-J cells. This finding suggests <br /> growth, unexpected a/ax sriml-1/srmil-I               receptor quench srinI-I-generated signal. In <br /> diploids, respond pheromone,                    concert possibility, the <br /> temperature sensitive. In second assay suppression              carboxyterminus at-factor receptor component of <br /> the ste mutants srinl-, asked double                 signal attenuation enables cells recover <br /> mutants mate. To create isogenic SRMI srmn-I                   a-factor-induced cell cycle arrest (30, 50). Perhaps the <br /> strains, transformed SY1357 SY1359 pSL749                    SRMl product constituent pheromone <br /> (SRMI cloned YCp5O) YCp5O determined                   response pathway target receptor-mediated signal <br /> the mating efficiencies resulting transformants.           attenuation. <br /> Strains bearing plasmid undetectable levels                   Contrary simple expectation model, did <br /> mating, indicating srinIl-I mutation did suppress           observe increase G1 accumulation projection <br /> the mating defect ste4 steS mutant strains (Table           formation STE2 (the (x-factor-receptor gene) de- <br /> 3). Thus, sriml-J requires functional STE4 STE5 prod-                leted ssrmIl-I mutants (Table 4). However, srm Il-l and <br /> ucts activate pheromone response pathway.                         sri-Itl- mutants differ regards well. For exam- <br />                                                                          ple, compared sr,nl-l cells, srInl-I cells exhibit <br />                            DISCUSSION                                    features pheromone response temperatures per- <br />                                                                          missive growth; room temperature, srmIl-l ste2 cells <br />    Identification srml-1. We sought identify components            appreciable mating efficiency high proportion <br /> of pheromone response pathway yeast. When                    cells G1 phase cell cycle have <br /> study initiated, well-defined components            formed projections. The basis subtle difference the <br /> pathway pheromones receptors. To                      phenotype srm-l-1 cells unknown, although <br /> identify intracellular components pathway,                    discrepancies sort noted study <br /> screened pseudorevertants deletion a-factor-             genes involved pheromone response. For exam- <br /> receptor gene (ste3::LEU2). As MATax ste3::LEU2 strains                  ple, mutation CDC39 sufficient suppress mating <br /> are unable mate, searched mating-proficient                    defect ste4-3 cells, cells, mutation both <br /> strains, reasoning acquisition mating ability           CDC39 SST2 required suppression (26, 57). <br /> due mutation downstream elements                  Role SRMI. The phenotypes sr,n      I-I mutants, induc- <br /> pheromone response pathway. Such mutation initi-                 tion FUSI. accumulation Gl, mating absence <br /> ate signaling autonomously, absence a-factor.              pheromone receptor, suggest SRMI product <br /> hence allow receptor-deficient strain mate.                       component pheromone response pathway. Since <br />    Using strategy, isolated suppressor ste3::               sini l-I recessive mutation allows receptorless <br /> LEU2, called srinil-1. With similar scheme, screening              strains mate, inactivation SRM1 probably step in <br /> for suppressors (x-factor-receptor deletion, Jahng et al.          pheromone response pathway. Hence, absence of <br /> (23) isolated mutations genes: GPAI (SCGI). yeast             pheromone. wild-type SRM1 function maintain <br /> homolog subunit mammalian G proteins (11. 41.               expression pheromone-responsive genes low basal <br /> 43), CDC72, CDC73. As expected cell cycle arrest               level allow entry S phase cell cycle. The <br /> is response pheromone. suppressors                  results presented suggest possible locations for <br /> temperature sensitive growth, suppress              SRM1 pathway. <br /> mating defect high temperature. In addition, previously                One imagine signal transduction pathway is <br /> identified cell-division-cycle gene. CDC36, similar                linear, branches receptor various <br /> phenotype mutant (23). SRMI allelic              targets signal. In words, terminal physio- <br /> these suppressors (unpublished observations). Thus, muta-                logical change monitor, e.g., transcription induction and <br /> tions genes cause temperature-sensitive            GI arrest, activated chain events single <br /> cell cycle arrest G1 restore mating ability recep-            path. Genetic evidence indicates early step the <br /> torless mutants.                                                         pathway involves heterotrimeric G protein (11, 23, 41, 67). <br />  2692     CLARK AND SPRAGUE                                                                                           MOL. CELL. BIOL. <br />  <br /> By analogy mammalian systems (see reference 64),        conjunction undiscovered G proteins are <br /> thought absence pheromone, a, 1, -y         essential viability. <br /> subunits G protein complexed inactive,            In summary, mutation SRMI confers mating ability to <br /> the subunit bound GDP. When pheromone binds           strains lack pheromone receptors. The phenotypes of <br /> the receptor, GTP exchanged GDP 13y subunits        srml-l mutants suggest SRM1 involved pher- <br /> are released complex. In contrast usual              omone response pathway normal vegetative growth. <br /> situation mammalian systems, S. cerevisiae thought     A complete understanding nature role of <br /> that 13y complex, ot subunit bound        SRM1 pheromone response awaits biochemical <br /> GTP, propagates signal pheromone bound         genetic analysis. <br /> receptor interacting effector unidentified. <br /> Because STE4 likely encodes 13 subunit yeast G                               ACKNOWLEDGMENTS <br /> protein (67), finding srml-J suppress deletion        We thank members laboratory discussions; Vivian <br /> of STE4 implies SRM1 functions 13y complex.        MacKay, Anne Burkholder, Lee Hartwell plasmids gene <br /> One possibility SRM1 component signal          disruptions; Jeremy Thorner Tom Stevens yeast genomic <br /> transduction pathway operates receptor            library; Susan Michaelis Ira Herskowitz strain SM1229; M. <br /> and G protein. A second possibility SRM1               Andrew Hoyt David Botstein transposon mutagenesis <br /> interacts G-protein subunits affects activ-     system; Nick Davis, David Hagen, Karen Sprague for <br /> ity. For instance, SRM1 promote reassociation         comments manuscript. <br />                                                                       This work supported National Research Service award <br /> subunit 13y complex, catalyzing GTP            1F32 GM10711 (to K.L.C.) Public Health Service research grant <br /> hydrolysis ax subunit. A possibility SRM1     GM30027 (to G.F.S.), National Institutes Health. <br /> is required expression ax subunit structural gene. In   G.F.S. recipient Faculty Research Award FRA-282 from <br /> this scenario, inactivation SRM1 cause depletion       American Cancer Society. <br /> the G protein subunit free 13y subunits could <br /> interact effector initiate signaling. Similarly,                              LITERATURE CITED <br /> SRM1 involved repressing expression 1           1. Beggs, J. D. 1978. Transformation yeast replicating <br /> y subunit structural gene. Inactivation SRM1                  hybrid plasmid. Nature (London) 275:104-109. <br /> therefore cause increase free 13y complex.                    2. Bender, A. D., G. F. Sprague, Jr. 1986. Yeast peptide <br />    Although inability srml-1 suppress ste4 muta-             pheromones, a-factor ao-factor, activate common response <br />                                                                        mechanism target cells. Cell 47:929-937. <br /> tions implies SRM1 acts G protein signal         3. Betz, R., W. Duntze, T. R. Manney. 1978. Mating-factor- <br /> transduction, second interpretation offered.             mediated sexual agglutination Saccharomyces cerevisiae. <br /> This interpretation supposes absence               FEMS Microbiol. Lett. 4:107-110. <br /> pheromone, low level signal propagated        4. Blumer, K. J., J. E. Reneke, J. Thorner. 1988. The STE2 <br /> the pathway. For example, free                gene product ligand binding component a-factor <br /> 13y complex. In support view, basal level tran-            receptor Saccharomnyc es cerevisiae. J. Biol. Chem. 263: <br /> scription pheromone-inducible genes FUSI                 10836-10842. <br />                                                                     5. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A positive <br /> reduced 40-fold ste4 mutants (38). By extrapola-            selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> tion, imagine suppression srml-l              ylase activity yeast: 5-fluoro-orotic acid resistance. Mol Gen. <br /> mating defect receptor mutants dependent low            Genet. 197:345-346. <br /> level signal exists cells stimulated phero-     6. Broach, J. R., J. N. Strathern, J. B. Hicks. 1979. Transfor- <br /> mone. In view, SRM1 functions G protein,                mation yeast: development hybrid cloning vector and <br /> rather it.                                                 isolation CANI gene. Gene 8:121-133. <br />    SRM1 role ih vegetative life cycle            7. Bucking-Throm, E., W. Duntze, L. H. Hartwell, T. R. <br /> addition role mating. a/a diploids contain        Manney. 1973. Reversible arrest haploid yeast cells the <br />                                                                        initiation DNA synthesis diffusible sex factor. Exp. Cell <br /> signal transduction G protein transcription             Res. 76:99-110. <br /> structural genes repressed (11, 41, 67). Nonetheless,            8. Burkholder, A. C., L. H. Hartwell. 1985. The yeast a-factor <br /> diploids homozygous srml-l temperature sensitive               receptor: structural properties deduced sequence of <br /> for growth and, moreover, arrest growth               STE2 gene. Nucleic Acids Res. 13:8463-8475. <br /> particular position cell cycle. Additional evidence     9. Chatoo, B. B., F. Sherman, D. A. Azubalis, T. A. Fjellstedt, D. <br /> SRM1 role vegetative life cycle comes                Mehnert, M. Ogur. 1979. Selection lys2 mutants the <br /> analysis srml null phenotype haploid cells. Spores           yeast Saccharomyces cerevisiae utilization a-amino- <br /> harboring srml::LEU2 inviable form projec-              adipate. Genetics 93:51-65. <br /> tions; arrest G1 phase cell cycle     9a.Clark, K. L., N. G. Davis, D. K. Wiest, J.-J. Hwang-Shum, and <br />                                                                        G. F. Sprague, Jr. 1988. Response yeast cells a-factor <br /> uncertain. One model explain difference phe-             pheromone: topology receptor identification a <br /> notype srml::LEU2 srml-l mutants srml-I                 component response pathway. Cold Spring Harbor Symp. <br /> mutants substantially reduced residual SRM1 ac-               Quant. Biol. 53:611-620. <br /> tivity nonpermissive temperature, condition          10. Cross, F., L. H. Hartwell, C. Jackson, J. B. Konopka. 1988. <br /> activates pheromone response pathway allows                Conjugation Saccharoimvces c erevisiae. Annu. Rev. Cell <br /> vegetative functions SRM1 fulfilled. However,          Biol. 4:429-457. <br /> complete absence SRM1 product, case          11. Dietzel, C., J. Kurjan. 1987. The yeast SCGI gene: a <br /> srml::LEU2 strains, inability carry functions           Ga-like protein implicated a- a-factor response <br /> required vegetative growth dictates phenotype. Each            pathway. Cell 50:1001-1010. <br />                                                                    12. Dixon, R. A. F., B. K. Kibilka, D. J. Strader, J. L. Benovic, <br /> of possible roles SRM1 mating discussed            H. G. Dohlman, T. Frielle, M. A. Bolanowski, C. D. Bennet, E. <br /> above extended include similar role                 Rands, R. E. Diehl, R. A. Mumford, E. E. Slater, I. S. Sigal, <br /> vegetative life cycle. For example, SRM1 involved           M. S. Caron, R. J. Lefkowitz, C. D. Strader. 1986. Cloning <br /> GTP hydrolysis subunit, SRM1 work                   gene cDNA mammalian 3-adrenergic receptor and <br />  VOL. 9, 1989                                                                 A SUPPRESSOR OF RECEPTOR MUTATIONS                          2693 <br />  <br />     homology rhodopsin. Nature (London) 321:75-79.                     177:753-754. <br /> 13. Fehrenbacher, G., K. Perry, J. Thorner. 1978. Cell-cell         35. Lipman, D. J., W. R. Pearson. 1985. Rapid sensitive <br />     recognition Saccharomyces cerev&apos;isiae: regulation mating-         protein similarity searches. Science 227:1435-1441. <br />     specific adhesion. J. Bacteriol. 134:893-901.                       36. MacKay, V., T. R. Manney. 1974. Mutations affecting <br /> 14. Hagen, D. C., G. McCaffrey, G. F. Sprague, Jr. 1986.                sexual conjugation related processes Saccharomvces <br />     Evidence yeast STE3 gene encodes receptor                 cerevisiae. II. Genetic analysis nonmating mutants. Genetics <br />     peptide pheromone factor: gene sequence implications          76:272-288. <br />     structure presumed receptor. Proc. Natl. Acad. Sci.      37. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />     USA 83:1418-1422.                                                       cloning: laboratory manual. Cold Spring Harbor Laboratory, <br /> 15. Hagen, D. C., G. F. Sprague, Jr. 1984. Induction             Cold Spring Harbor, N.Y. <br />     yeast a-specific STE3 gene peptide pheromone a-factor.       38. McCaffrey, G., F. J. Clay, K. Kelsay, G. F. Sprague, Jr. <br />     J. Mol. Biol. 178:835-852.                                              1987. Identification regulation gene required cell <br /> 16. Halliday, K. 1984. Regional homology GTP-binding proto-              fusion mating yeast Saccharomvces cerevisiae. <br />     oncogene products elongation factors. J. Cyclic Nucleotide          Mol. Cell. Biol. 7:2680-2690. <br />     Res. 9:435-448.                                                     39. Michaelis, S., I. Herskowitz. 1988. The a-factor pheromone <br /> 17. Hanks, S. K., A. M. Quinn, T. Hunter. 1988. The protein             Saccharomyces cerev&apos;isiae essential mating. Mol. Cell. <br />     kinase family: conserved features deduced phylogeny          Biol. 8:1309-1318. <br />     catalytic domains. Science 241:42-52.                               40. Misuzawa, S., S. Nishimura, F. Seela. 1986. Improvement of <br /> 18. Hartig, A., J. Holly, G. Saari, V. L. MacKay. 1986. Multiple        dideoxy chain termination method DNA sequencing by <br />     regulation STE2, mating-type-specific gene Saccharonvy-         use deoxy-7-deazaguanosine triphosphate place dGTP. <br />     ces cerevisiae. Mol. Cell. Biol. 6:2106-2114.                           Nucleic Acids Res. 14:1319-1324. <br /> 19. Hartwell, L. 1980. Mutants Saccharomvces cerei&apos;isiae unre-       41. Miyajima, I., M. Nakafuku, N. Nakayama, C. Brenner, A. <br />     sponsive cell division control polypeptide mating hor-            Miyajima, K. Kaibuchi, K.-I. Arai, Y. Kaziro, K. Matsu- <br />     mone. J. Cell Biol. 85:811-822.                                         moto. 1987. GPAI, haploid-specific essential gene, encodes a <br /> 20. Henikoff, S. 1984. Unidirectional digestion exonuclease III        yeast homolog mammalian G protein involved <br />     creates targeted breakpoints DNA sequencing. Gene 28:               mating factor signal transduction. Cell 50:1011-1019. <br />     351-359.                                                            42. Moir, D., S. E. Stewart, B. C. Osmond, D. Botstein. 1982. <br /> 21. Herskowitz, I. 1988. Life cycle budding yeast Saccharo-          Cold-sensitive cell-division-cycle mutants yeast: isolation, <br />     myces cerevisiae. Microbiol. Rev. 52:536-553.                           properties, pseudoreversion studies. Genetics 100:547-563. <br /> 22. Huisman, O., W. Raymond, K.-U. Froehlich, P. Errada, N.             43. Nakafuku, M., H. Itoh, S. Nakamura, Y. Kaziro. 1987. <br />     Kleckner, D. Botstein, M. A. Hoyt. 1987. A TnIO-lacZ-               Occurrence Saccharomnvces cerevisiae gene homologous <br />     kanR-URA3 gene fusion transposon insertion mutagenesis              cDNA coding subunit mammalian G proteins. <br />     fusion analysis yeast bacterial genes. Genetics              Proc. Natl. Acad. Sci. USA 84:2140-2144. <br />     116:191-199.                                                        44. Nakayama, N., A. Miyajima, K. Arai. 1985. Nucleotide <br /> 23. Jahng, K.-Y., J. Ferguson, S. I. Reed. 1988. Mutations         sequence STE2 STE3, cell type-specific sterile genes <br />     gene encoding subunit Saccharoinvces cereWisiae G            Saccharomvces cerevisiae. EMBO J. 4:2643-2648. <br />     protein indicate role mating pheromone signaling. Mol. Cell.   45. Nakayama, N., A. Miyajima, K. Arai. 1987. Common signal <br />     Biol. 8:2484-2493.                                                      transduction shared STE2 STE3 haploid cells <br /> 24. Jarvis, E. E., D. C. Hagen, G. F. Sprague, Jr. 1988.                Saccharomvces cerei&apos;isiae: autocrine cell-cycle arrest from <br />     Identification DNA segment necessary sufficient        forced expression STE2. EMBO J. 6:249-254. <br />     a-specific gene control Saccharomvces cereWisiae: impli-     46. Nasmyth, K. A., S. I. Reed. 1980. Isolation genes by <br />     cations regulation a-specific a-specific genes. Mol.         complementation yeast: molecular cloning cell-cycle <br />     Cell. Biol. 8:309-320.                                                  gene. Proc. Natl. Acad. Sci. USA 77:2119-2123. <br /> 25. Jenness, D. D., A. C. Burkholder, L. H. Hartwell. 1983.         47. Nathans, J., D. S. Hogness. 1984. Isolation nucleotide <br />     Binding a-factor pheromone yeast cells: chemical            sequence gene encoding human rhodopsin. Proc. Natl. <br />     genetic evidence a-factor receptor. Cell 35:521-529.             Acad. Sci. USA 81:4851-4855. <br /> 26. Jenness, D. D., B. S. Goldman, L. H. Hartwell. 1987.            48. Pringle, J., L. H. Hartwell. 1981. The Saccharomyces <br />     Saccharomrvces cere0isiae mutants unresponsive a-factor              cerevisiae cell cycle, p. 97-142. In J. N. Strathern, E. W. Jones. <br />     pheromone: a-factor binding extragenic suppression. Mol.            J. R. Broach (ed.), The molecular biology yeast <br />     Cell. Biol. 7:1311-1319.                                                Saccharoinvces: life cycle inheritance. Cold Spring Harbor <br /> 27. Jensen, R. E., I. Herskowitz. 1984. Directionality              Laboratory, Cold Spring Harbor, N.Y. <br />     regulation cassette substitution yeast. Cold Spring Harbor    49. Reed, S. I. 1980. The selection S. cerei&apos;isiae mutants defec- <br />     Symp. Quant. Biol. 49:97-104.                                           tive start event cell division. Genetics 95:561-577. <br /> 28. Johnston, M., R. W. Davis. 1984. Sequences regulate        50. Reneke, J. E., K. J. Blumer, W. E. Courchesne, J. Thorner. <br />     divergent GALI-GALIO promoter Saccharoinyces cere-               1988. The carboxy-terminal segment yeast a-factor recep- <br />     *isiae. Mol. Cell. Biol. 4:1440-1448.                                   tor regulatory domain. Cell 55:221-234. <br /> 29. Klug, A., D. Rhodes. 1987. &apos;Zinc fingers&apos;: novel protein      51. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. <br />     motif nucleic acid recognition. Trends Biochem. Sci. 12:            Fink. 1987. The Saccharomvces cerevisiae genomic plasmid <br />     464-469.                                                                bank based centromere-containing shuttle vector. Gene <br /> 30. Konopka, J. B., D. D. Jenness, L. H. Hartwell. 1988. The            60:237-243. <br />     C-terminus S. cerei&apos;isiae a-pheromone receptor mediates      52. Rose, M., B. Price, G. R. Fink. 1986. Saccharomyces <br />     adaptive response pheromone. Cell 54:609-620.                     cerevisiae nuclear fusion requires prior activation f <br /> </body></html>