<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Presence&#226;&#8364;&#8220;Absence Variation A. thaliana Is Primarily <br /> Associated Genomic Signatures Consistent <br /> with Relaxed Selective Constraints <br /> Stephen J. Bush,1 Atahualpa Castillo-Morales,1 Jaime M. Tovar-Corona,1 Lu Chen,1,z Paula X. Kover,1 and <br /> Araxi O. Urrutia*,1 <br /> 1 <br />  Department Biology Biochemistry, University Bath, Bath, United Kingdom <br /> z <br />  Present address: Human Genetics, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, UK <br /> *Corresponding author: E-mail: a.urrutia@bath.ac.uk. <br /> Associate editor: Stephen Wright <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> The sequencing multiple genomes plant species revealed polymorphic gene exon loss. Genes <br /> associated disease resistance overrepresented showing structural variations, suggesting adaptive <br /> role gene exon presence&#226;&#8364;&#8220;absence variation (PAV). <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> shed light possible functional relevance poly- <br /> morphic coding region loss mechanisms driving process, characterized genes lost entire exons <br /> or coding regions 17 fully sequenced Arabidopsis thaliana accessions. We significant <br /> enrichment genes associated certain functional categories observed, PAV events largely restricted genes <br /> with signatures reduced essentiality: PAV genes tend newer additions genome, tissue specific, lowly <br /> expressed. In addition, PAV genes located regions lower gene density higher transposable element density. <br /> Partial coding region PAV events associated marginal reduction gene expression level affected <br /> accession occurred genes higher levels alternative splicing Col-0 accession. Together, results <br /> suggest adaptive scenarios ruled out, PAV events explained invoking them. <br /> Key words: exon deletion, presence&#226;&#8364;&#8220;absence variation, genome evolution, transposable elements, adaptive <br /> evolution, Arabidopsis. <br />  <br /> Introduction                                                                        intraspecies variation genes, inclusive large <br />                                                                                     deletions (Gan et al. 2011) 775 genes acces- <br /> Intraspecies variation gene content represents impor- <br />                                                                                     sion deletions spanning 50% their <br /> tant source heterogeneity genome species <br />                                                                                     coding region sequence (Gan et al. 2011). A comparison of <br /> and potentially contributes organism&#226;&#8364;&#8482;s adaptability in <br />                                                                                     80 Arabidopsis genomes 9% total genes <br /> response external pressures (Feuk et al. 2006). Cataloguing <br />                                                                                     A. thaliana showed presence&#226;&#8364;&#8220;absence variation (PAV) <br />  <br />  <br />  <br />  <br />                                                                                                                                                         Article <br /> significant gains losses coding regions or <br /> between species allow deeper understanding                            averaging 444 absent genes accession (Tan et al. 2012). <br /> mechanisms underlying molecular evolution genomes                               Characterization coding region PAV shown certain <br /> and assist identifying functional variation agronom-                      gene categories significantly enriched. For instance, 52 of <br /> ically elite varieties staple crops (Wang, You, et al. 2013). To                 154 nucleotide-binding site leucine-rich repeat (NBS-LRR) <br /> this end, studies examined polymorphic                         R (resistance) genes deleted 1 <br /> partial gene loss plant species. For instance,                     50 rice cultivars (Xu et al. 2012). Similar overrepresentation <br /> resequencing 50 rice genomes, 1,327 possible gene loss                        R genes A. thaliana observed (Bakker <br /> events (2.4% total gene set) detected relative                          et al. 2006; Shen et al. 2006), soybean, genes <br /> to Nipponbare reference accession (Xu et al. 2012).                             enriched structural variation likely involved <br /> Significant intraspecies variation gene content                         nucleotide binding biotic defense (McHale et al. 2012). <br /> been reported maize (Swanson-Wagner et al. 2010), sor-                           Enrichment particular functional gene categories among <br /> ghum (Zheng et al. 2011), soybean (McHale et al. 2012).                         genes affected structural polymorphism suggests these <br /> Previous studies model plant Arabidopsis thaliana,                           structural polymorphisms functional role, allow- <br /> using resequencing microarrays Illumina sequencing-by-                          ing accessions better adapted environmental <br /> synthesis reads, shown significant variations total                    conditions face. <br /> nuclear genome sequence naturally occurring strains                              However, hypothesis explicitly tested. If <br /> (Clark et al. 2007; Ossowski et al. 2008). A recent study                      significant polymorphic deletions adaptive, would <br /> using 18 fully sequenced A. thaliana genomes that,                            expect affected genes multiple signatures <br /> relative reference accession Col-0, 93.4% proteins                        selection. On hand, structural <br />  <br /> &#195;&#376; The Author 2013. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http:// <br /> creativecommons.org/licenses/by-nc/3.0/), permits non-commercial re-use, distribution, reproduction medium, <br /> provided original work properly cited. For commercial re-use, contact journals.permissions@oup.com                 Open Access <br /> Mol. Biol. Evol. 31(1):59&#226;&#8364;&#8220;69 doi:10.1093/molbev/mst166 Advance Access publication September 25, 2013                                              59 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                MBE <br /> polymorphisms affect genes evolving relaxed          GOslim term. The proportion poorly characterized genes is <br /> constraints, adaptive significance           greater CDS-PAV genes, 60% having <br /> questioned.                                                       associated GOslim term biological process. When <br />    Here characterize genes affected PAV spanning            examining genes associated GOslim terms found <br /> whole exons A. thaliana, investigate genomic          E-PAV CDS-PAV genes significantly enriched <br /> features, any, associated polymorphisms.        genes associated signal transduction nucleotide <br /> Our results provide insights likely functional           binding (fig. 1 supplementary fig. S2, Supplementary <br /> impact structural variation protein-coding genes.           Material online). Furthermore, E-PAV genes appear <br />                                                                   significantly enriched genes associated GOslim <br /> Results <br />                                                                   term &#226;&#8364;&#339;other binding,&#226;&#8364;? includes proteins bind to <br /> In order characterize PAV A. thaliana, examined          lipids, metal ions, ATP, cofactors (fig. 1). <br /> previously identified polymorphic deletions 17 fully           Significant overrepresentation functional categories among <br /> sequenced Arabidopsis accessions transcriptome          PAV genes consistent previous assessment large <br /> data available (Gan et al. 2011) (see Materials          coding region indels soybean genome (McHale et al. <br /> Methods). We compiled set deletions spanned             2012) gene deletions A. thaliana (Tan et al. <br /> entire exons 17 accessions relative Col-0        2012). This observed classifying genes using a <br /> reference genome. A subset annotated deletions         broader set GO &#226;&#8364;&#339;GOslim&#226;&#8364;? terms (supplemen- <br /> experimentally validated (Gan et al. 2011). To rule   tary fig. S3, Supplementary Material online). <br /> the possibility wrongly identifying deletions differ-       When classifying genes family, observe overrep- <br /> ences assemblies, exons confirmed missing         resentation members NBS-LRR family&#226;&#8364;&#8221;involved <br /> by searching homology Col-0 exon           pathogen detection (DeYoung Innes 2006)&#226;&#8364;&#8221;among <br /> other accessions (see Materials Methods).                     E-PAV genes (families &#226;&#8364;&#339;NBS-LRR active TNL,&#226;&#8364;? adjusted <br />    A total 794 exons classified missing    P value = 8.57   10 35, &#226;&#8364;&#339;NBS-LRR active CNL,&#226;&#8364;? adjusted <br /> one 17 accessions, corresponding 411 genes (~1.5%        P value = 4.63   10 5; fig. 1), consistent previous findings <br /> the total gene set) including 81 genes coding      (Shen et al. 2006). Furthermore, examining 3,753 <br /> region completely absent accession (sup-      Pfam ID gene associations (supplementary figs. S4 S5, <br /> plementary table S1, Supplementary Material online). Exon         Supplementary Material online), observe overrepresen- <br /> losses uniformly distributed gene:         tation members NB-ARC (APAF-1, R proteins, and <br /> missing exon sequences near              CED-4) LRR domain containing families (note &#226;&#8364;&#339;NBS- <br /> ends gene (supplementary fig. S1, Supplementary           LRR&#226;&#8364;? refers composite NBS LRR domains and <br /> Material online).                                                 NBS domain known &#226;&#8364;&#339;NB-ARC&#226;&#8364;? [McHale <br />    Overall, ~0.3% genes accession          et al. 2006]). No enrichment particular gene family was <br /> least missing exon, representing 10 50 kb         observed CDS-PAV genes (data shown). <br /> of missing sequence accession (supplementary table S2,            These significant enrichments gene functional and <br /> Supplementary Material online). A total 200 genes          domain annotations line previous findings in <br /> exon loss affecting accession, consistent           Arabidopsis (Tan et al. 2012) plant species <br /> with previous study reporting &#226;&#8364;&#339;common history&#226;&#8364;?             (Swanson-Wagner et al. 2010; Zheng et al. 2011; McHale <br /> deletion events A. thaliana (Santuari et al. 2010).            et al. 2012) proposed reflect adaptive <br />    Because partial deletions spanning exons           role large polymorphic deletions. <br /> have distinct functional implications compared full <br /> coding region deletions, 330 genes partial coding        Genes Affected PAV Show Signatures Consistent <br /> region loss spanning exon       Relaxed Selective Constraints <br /> accession (exon PAV [E-PAV]) 81 genes           To determine PAV genes generally associated <br /> coding region polymorphic deletions affecting        fast evolving proteins potentially positive selection, <br /> accession (full coding DNA sequence PAV [CDS-PAV])                examined rates nonsynonymous synonymous <br /> were examined separately.                                         changes gene (dN/dS). Using randomization test, <br />                                                                   E-PAV genes significantly higher dN/dS <br /> Genes Involved Signal Transduction Both                    ratio compared genes exons present, but <br /> Nucleotide Protein Binding Are Overrepresented                genes dN/dS ratio 1 (fig. 2 and <br /> among PAV Genes                                                   supplementary tables S1 S3, Supplementary Material <br /> In order characterize PAV genes, assessed     online). CDS-PAV genes nonsignificant increase dN/ <br /> these genes overrepresented particular gene classes       dS compared intact genes (those affected dele- <br /> or gene ontology (GO) categories. To so, used          tions spanning exon accession; fig. 2 and <br /> classification schemes: &#226;&#8364;&#339;GO,&#226;&#8364;? condensed set GO terms         supplementary table S3, Supplementary Material online). <br /> (GOslim), Pfam protein domain database family            To examine selective pressures associated with <br /> classification scheme (Gan et al. 2011) (see Materials     PAV genes, examined nucleotide diversity. We considered <br /> Methods). Of 330 E-PAV genes,             nucleotide diversity replacement sites silent sites <br /> poorly characterized 50% having associated        (defined noncoding sites synonymous sites of <br />  <br /> 60 <br />  Presence&#226;&#8364;&#8220;Absence Variation A. thaliana . doi:10.1093/molbev/mst166                                                                  MBE <br />      (a)                                                                 (b) <br />  <br />  <br />  <br />  <br />      (c)                                                                 (d) <br />  <br />  <br />  <br />  <br /> FIG. 1. Distribution E-PAV genes (n = 330)&#226;&#8364;&#8221;those one, all, exons missing accession&#226;&#8364;&#8221;by GOslim categories for <br /> molecular function (a), biological process (b) cellular component (c), family (d). Both expected observed number E-PAV genes per <br /> category represented bar. Where significant enrichment (P   0.05) observed expected E-PAV genes a <br /> particular category, asterisk shown bars. Only categories E-PAV gene shown. <br />  <br />  <br /> protein-coding regions) gene, according Gan et al.                 genes) D &gt; 2 (randomization test P = 0.93 P = 1 E- <br /> (2011). PAV genes associated higher                      CDS-PAV genes, respectively). As demographic charac- <br /> nucleotide diversity silent replacement sites (sup-                teristics Arabidopsis population result shift in <br /> plementary table S3, Supplementary Material online).                           average Tajima&#226;&#8364;&#8482;s D general pool genes, is <br />    Although higher dN/dS nucleotide diversity                          possible hard thresholds informative. <br /> suggestive relaxed selective constraints, <span id='am-87' about='xsp:pattern' typeof='owl:Thing'>pattern</span>                   Indeed, intact genes Arabidopsis the <br /> also consistent scenario positive and/or balancing                   average Tajima&#226;&#8364;&#8482;s D estimate shifted negative values. <br /> selection. To differentiate possible scenarios,                  Thus, PAV genes fall short hard threshold + 2 <br /> Tajima&#226;&#8364;&#8482;s D calculated gene (see Materials                     higher D estimate general pool of <br /> Methods). A threshold &#194;&#177;2 considered point                        genes, suggestive balancing selection. However, E-PAV <br /> at D significantly departs null expectation                     genes significant differences Tajima&#226;&#8364;&#8482;s D esti- <br /> of neutral evolution given gene. Of 330 E-PAV                      mates compared intact genes CDS-PAV genes have; <br /> genes E-PAV, 24 D &lt;  2 2 D &gt; 2                         fact, significantly lower estimate D (fig. 2 supple- <br /> (AT1G12180, D = 2.17, AT5G35460, D = 2.05,                         mentary tables S1 S3, Supplementary Material online). <br /> which functionally uncharacterized). Among CDS-PAV                         It possible PAV genes higher range of <br /> genes, seven D &lt;  2 D &gt; 2.                             D values compared intact genes, hiding higher propor- <br /> Compared set intact genes, signifi-                   tion genes positive balancing selection that <br /> cant differences proportion PAV genes                    reflected overall changes mean. To <br /> D &lt; 2 (randomization test P = 1 E- CDS-PAV                        test this, compared distributions Tajima&#226;&#8364;&#8482;s D <br />  <br />                                                                                                                                                  61 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                                                  MBE <br />  <br />                                                                                                                                   20 <br />  <br />                                                                          4 <br />  <br />                   0.4                                                                                                             15 <br />  <br />  <br />  <br />  <br />                                                                                                               Exp. level (RPKM) <br />                                                                          3 <br />      dN/dS <br />  <br />  <br />  <br />  <br />                                                        Age <br />                                                                                                                                   10 <br />                                                                          2 <br />                   0.2 <br />  <br />                                                                                                                                    5 <br />                                                                          1 <br />  <br />  <br />  <br />                   0.0                                                    0                                                         0 <br />  <br />                           Intact     E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                           Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                                     Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV <br />  <br />  <br />  <br />                    0.00 <br />                                                                                                                                   0.5 <br />  <br />  <br />                                                                          9 <br />                   &#226;&#710;&#8217;0.25                                                                                                           0.4 <br />                                                        No. paralogs <br />      Tajima&apos;s D <br />  <br />  <br />  <br />  <br />                                                                                                                                   0.3 <br />                                                                          6 <br />  <br />  <br />                                                                                                               Tau <br />                   &#226;&#710;&#8217;0.50 <br />  <br />  <br />                                                                                                                                   0.2 <br />  <br />                   &#226;&#710;&#8217;0.75                                                  3 <br />                                                                                                                                   0.1 <br />  <br />  <br />                   &#226;&#710;&#8217;1.00 <br />                                                                          0                                                        0.0 <br />  <br />                             Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                           Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                                     Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV <br />  <br /> FIG. 2. Genetic features associated intact (having exons P/A variation), E-PAV (having one, all, exons missing least <br /> one accession), CDS-PAV (having entire CDS missing accession) genes. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> left right, bottom: dN/dS, age, expression <br /> level, Tajima&#226;&#8364;&#8482;s D, number paralogs, tau. See supplementary table S3, Supplementary Material online, values means statistical analysis. <br />  <br />  <br /> estimates sets genes (intact, E-PAV, CDS-                           (3.32%) observe CDS-PAV genes <br /> PAV). However, did observe evidence in-                                  (8.82%). This finding suggest significant propor- <br /> creased dispersion D PAV genes (supplementary                                  tion CDS-PAV genes undergoing stronger puri- <br /> fig. S6, Supplementary Material online). To examine                             fying positive selection relative intact genes. Together, <br /> this possibility, examined proportion PAV genes                               results suggest rule the <br /> below fifth 95th percentile &#226;&#8364;&#339;intact&#226;&#8364;?                           effect balancing selection acting individual PAV <br /> distribution (D =  2.05 1.39, respectively). If signifi-                          genes general trend balancing selection PAV genes <br /> cantly higher proportion E-PAV CDS-PAV genes                                  does readily apply. The excess negative D values among <br /> found compared intact set positive end                               PAV genes coupled higher levels nucleotide di- <br /> the distribution, infer existence detectable                            versity significant increases dN/dS ratios con- <br /> subset PAV genes undergoing balancing                                    sistent scenario weaker purifying selection could <br /> selection. However, pattern clearly observed&#226;&#8364;&#8221;                             explained positive selection. <br /> only 2.33% E-PAV&#226;&#8364;&#8221;and CDS-PAV genes exceed                                         We examined number parameters been <br /> threshold value. At opposite end distribution,                            previously associated studies gene essentiality <br /> observe overrepresentation proportion E-PAV                                explore functional importance PAV genes, <br /> genes estimates D lower threshold                                 including gene&#226;&#8364;&#8482;s age (Chen, Trachana, et al. 2012) the <br />  <br /> 62 <br />  Presence&#226;&#8364;&#8220;Absence Variation A. thaliana . doi:10.1093/molbev/mst166                                                 MBE <br /> number paralogs (Hanada et al. 2009; Makino et al.       2003). To this, used reference accession (Col-0) <br /> 2009), weaker associations expression level     calculated TE density gene intergenic <br /> (Cherry 2010) tissue specificity (Wolf et al. 2006).           sequence 1- 100-kb windows centered gene&#226;&#8364;&#8482;s <br />    Compared newer genes, older genes likely       midpoint counting number bases TE <br /> be essential (Chen, Trachana, et al. 2012). After using        annotations (see Materials Methods). E-PAV genes were <br /> phylogenetic relationships plant genomes create            approximately 2-fold increase the <br /> proxy gene age, observed 330 genes affected        bases annotated TE compared genes <br /> by E-PAV likely newer additions              intact accessions (e.g., TE sequence accounts <br /> genome (fig. 2 supplementary table S3, Supplementary           ~30% nongenic sequence 10-kb window <br /> Material online). It possible E-PAV genes      surrounding E-PAV gene; fig. 3 supplementary table <br /> greater number paralogous genes compensate           S4, Supplementary Material online). Significant enrichment of <br /> for loss function. Consistent this,       specific TE superfamilies observed, notably, DNA <br /> those genes missing exons higher number para-         transposons LTR retrotransposons (supplementary <br /> logs compared genes exons present (fig. 2      table S4, Supplementary Material online). <br /> and supplementary table S3, Supplementary Material online).            In addition, genes missing exons <br /> However, opposite result observed analyzing           have, average, shorter distance gene boundary <br /> CDS-PAV genes&#226;&#8364;&#8221;these average 4.2 paralogs           nearest TE genes exons present <br /> compared genes exon losses (fig. 2 supple-        (2.5 kb compared 5.7 kb; randomization test, P = 9.9e 5; <br /> mentary table S3, Supplementary Material online), suggesting       supplementary table S3, Supplementary Material online. <br /> their function essential. We assessed expres-     When calculating minimum distance nearest TE, <br /> sion patterns genes affected exon presence&#226;&#8364;&#8220;absence,          classified superfamily, E-PAV genes significantly closer <br /> because broadly highly expressed genes typically as-       TE type: rolling circle TEs, DNA transposons, LTR <br /> sociated higher levels selection (Yang 2009). Using      retrotransposons, long interspersed elements (LINEs), and <br /> randomization test, genes exon losses        short interspersed elements (SINEs) (supplementary table <br /> one accessions, compared intact genes,           S3, Supplementary Material online). Similar findings were <br /> had lower expression levels higher tissue specificity (sup-    obtained analyzing TE content surrounding <br /> plementary table S3, Supplementary Material online). In ad-        regions CDS-PAV genes (supplementary table S3, <br /> dition, observed exons missing        Supplementary Material online). <br /> accession are, average, shorter exons present           Certain TE sequence motifs associated with <br /> accessions (170 bp vs. 284 bp, randomization test P = 9.9e 5;      recombination hotspots drive exon loss through <br /> supplementary table S3, Supplementary Material online).            promoting ectopic recombination events (Oliver Greene <br /> However, exons affected polymorphic deletions          2009; Horton et al. 2012). To explore genes affected <br /> are shorter average compared nondeleted exons,             PAV local enrichment hotspot motifs, <br /> E-PAV genes longer unaffected genes (2,360 bp             examined density motifs and <br /> compared 2,142 bp, respectively; randomization test           genes (see Materials Methods). However, we <br /> P = 0.008; supplementary table S3, Supplementary Material          observed significant differences hotspot motif occu- <br /> online). By contrast, CDS-PAV genes&#226;&#8364;&#8221;where polymorphic              pancy nongenic regions windows surrounding E- <br /> deletions encompass gene&#226;&#8364;&#8482;s entire coding region&#226;&#8364;&#8221;were           PAV genes compared intact genes (in window sizes of <br /> found shorter unaffected genes (640 bp compared         1 100 kb centered gene&#226;&#8364;&#8482;s midpoint; supplementary <br /> with 2,142 bp, randomization test P = 9.9e 5; supplementary        table S4, Supplementary Material online). Nevertheless, sig- <br /> table S3, Supplementary Material online).                          nificant enrichment hotspot motif occupancy was <br />    Overall, findings certain func-        observed genic sequence windows centered on <br /> tional categories overrepresented genes exon        E-PAV genes compared centered intact genes <br /> loss, generally significant coding region loss prevalent   (fig. 3 supplementary table S4, Supplementary Material <br /> among novel, lowly expressed poorly functionally               online). When comparing CDS-PAV genes intact set, <br /> characterized genes. These genes evolved              observed consistent pattern higher hotspot motif <br /> more recently Arabidopsis genome likely          density genic regions marginally higher <br /> be reduced selective constraint.                             proportion hotspot motifs nongenic regions that <br />                                                                    surround them, windows 3 kb size (P &lt; 0.01; sup- <br /> PAV Genes Are Located Genomic Regions That                      plementary table S5, Supplementary Material online). <br /> Are Gene-Poor Transposable Element-Rich                            Taken together, results PAV genes are <br /> When characterizing genomic context genes affected          located gene-poor TE-rich regions genome, <br /> by PAV, genes exon coding         supporting hypothesis PAV associated <br /> region loss separated longer intergenic distances (fig. 3   relaxed selective constraints. Enrichments sequence <br /> and supplementary table S3, Supplementary Material online).        motifs previously associated recombination hotspots <br /> Transposable element density PAV genes             PAV genes suggest exon <br /> assessed gene-poor areas associated            deletion events resulted recombination <br /> higher transposable element (TE) density (Wright et al.            events involving recombination hotspot motifs. <br />  <br />                                                                                                                                63 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                                              MBE <br />                                                                                              Exon Loss Is Associated Marginal Reduction in <br />                                                                                              Expression Level <br />                                     6000 <br />                                                                                              The aforementioned results suggest E-PAV associated <br />                                                                                              reduced selective constraints. To assess exon <br />      Intergenic distance (bp) <br />  <br />  <br />  <br />                                                                                              loss likely resulted reduced functionality the <br />                                                                                              genes affected, compared expression levels genes with <br />                                     4000                                                     missing exons accessions. If exon loss <br />                                                                                              causes follows diminished functionality previous <br />                                                                                              mutations, expect expression significantly <br />                                                                                              reduced accessions affected E-PAV. Using <br />                                     2000                                                     RNAseq transcription profiles Arabidopsis accession <br />                                                                                              (Gan et al. 2011), compared expression patterns of <br />                                                                                              individual genes accessions affected exon deletions with <br />                                                                                              accessions gene remained intact. To this, <br />                                                                                              transformed expression data accession Z scores <br />                                                             0 <br />                                                                                              (Cheadle et al. 2003). We looked genes <br />                                                                   Intact   E&#226;&#710;&#8217;PAV   CDS&#226;&#710;&#8217;PAV   exon loss occurred single accession (210 <br />                                                             40                               genes). For gene, took 1) expression level of <br />                                                                                              gene affected accession 2) mean expres- <br />                                                                                              sion level gene 17 unaffected accessions <br />                                                                                              (the 16 study plus reference genome, Col-0). <br />                                                             30 <br />                                                                                              We half genes examined expression <br />                                 % TE bases (density) <br />  <br />  <br />  <br />  <br />                                                                                              level mean 37% expression level equal it. <br />                                                                                              However, average, expression levels affected acces- <br />                                                             20 <br />                                                                                              sion departed little mean expression unaffected <br />                                                                                              accessions (0.15 standard deviations). In 27 genes (13% of <br />                                                                                              cases), expression level gene affected exon deletion <br />                                                                                              higher mean expression unaffected ac- <br />                                                             10                               cessions 14 cases showing statistically significant dif- <br />                                                                                              ference (fig. 4 supplementary table S6, Supplementary <br />                                                                                              Material online). These 27 genes generally poorly charac- <br />                                                                                              terized 12 having functional category annotations. <br />                                                              0 <br />                                                                                              Most genes affected exon deletions low expression <br />                                                                   Intact   E&#226;&#710;&#8217;PAV   CDS&#226;&#710;&#8217;PAV   levels begin with, exceptions notable, <br />                                                             2.5                              rotamase CYP4 (AT3G62030; involved variety of <br />                                                                                              cellular functions related metabolism response sev- <br />                                                                                              eral types stress), average expression level in <br />                                                             2.0                              unaffected accessions 400 rpkm, 1% of <br />                                                                                              genes detectable expression Col-0. <br />                                 % hotspot bases (density) <br />  <br />  <br />  <br />  <br />                                                                                                 It possible moderate effect exon loss gene <br />                                                             1.5                              expression levels explained overrepresentation of <br />                                                                                              alternatively spliced exons set missing exons. <br />                                                                                              This allow production viable protein prod- <br />                                                             1.0                              ucts absence. In order test this, quantified <br />                                                                                              alternative splicing 15,540 Arabidopsis genes, including <br />                                                                                              103 330 E-PAV associated genes using &#226;&#8364;&#339;comparable <br />                                                             0.5                              alternative splicing index&#226;&#8364;? (see Materials Methods), which <br />                                                                                              corrects distorting effect variation transcript <br />                                                                                              coverage genes (reviewed Chen, Tovar-Corona, <br />                                                             0.0                              et al. [2012]). E-PAV genes significantly <br />                                                                   Intact   E&#226;&#710;&#8217;PAV   CDS&#226;&#710;&#8217;PAV   higher number alternative splicing events compared with <br /> FIG. 3. Genomic context intact (having exons P/A <br /> variation), E-PAV (having one, all, exons missing                     FIG. 3. Continued <br /> least accession), CDS-PAV (having entire CDS missing                       gene&#226;&#8364;&#8482;s midpoint, percentage recombinogenic motifs the <br /> least accession) genes. Averaged values genes set                    genic sequence 1-kb window centered gene&#226;&#8364;&#8482;s midpoint. See <br /> given for, bottom, intergenic distance, percentage                    supplementary tables 3 4 (Supplementary Material online) for <br /> TE bases nongenic sequence 10-kb window centered                         values specific TE families window sizes. <br />  <br />  <br /> 64 <br />  Presence&#226;&#8364;&#8220;Absence Variation A. thaliana . doi:10.1093/molbev/mst166                                                              MBE <br />                                                                             random expectations Arabidopsis (Tan et al. 2012), <br />                                                                             greatest significant enrichment PAV genes also <br />                                                                             reported NB-ARC domains. <br />                                                                                 These functional and/or gene family enrichments be <br />                                                                             suggestive adaptive role PAV events aiding spe- <br />                                                                             cific ecotypes adapting local environment. Our <br />                                                                             results&#226;&#8364;&#8221;showing genes associated with, example, <br />             &#226;&#8212;? &#226;&#8212;? &#226;&#8212;? <br />                 &#226;&#8212;?&#226;&#8212;? <br />                  &#226;&#8212;?    &#226;&#8212;?&#226;&#8212;? &#226;&#8212;?&#226;&#8212;? &#226;&#8212;?&#226;&#8212;? <br />                   &#226;&#8212;?&#226;&#8212;?&#226;&#8212;?&#226;&#8212;?&#226;&#8212;?                   &#226;&#8212;? <br />                                          &#226;&#8212;?&#226;&#8212;?   &#226;&#8212;?    &#226;&#8212;?&#226;&#8212;? <br />                                                   &#226;&#8212;?&#226;&#8212;?    &#226;&#8212;?   &#226;&#8212;?&#226;&#8212;? &#226;&#8212;?&#226;&#8212;? &#226;&#8212;? &#226;&#8212;? &#226;&#8212;? &#226;&#8212;? <br />                                                             &#226;&#8212;? <br />                                                                             resistance likely affected PAV&#226;&#8364;&#8221;are, at <br />                                                                             glance, consistent hypothesis. In addition, we <br />                                                                             able confirm previous report CDS-PAV three <br />                                                                             members R gene family&#226;&#8364;&#8221;the single-exon gene <br />                                                                             AT5G05400 multiexon genes AT5G18350 and <br />                                                                             AT5G49140 (Shen et al. 2006)&#226;&#8364;&#8221;a family known sig- <br />                                                                             natures positive selection A. thaliana (Mondragon- <br />                                                                             Palomino et al. 2002). However, comprehensive analysis for <br />                                                                             evidence selection does support general <br />     &#226;&#710;&#8217;4               &#226;&#710;&#8217;2            0                2                   4 <br />                                                                             interpretation. <br />                                                                                 dN/dS ratios widely used estimates of <br />            Z&#226;&#710;&#8217;score gene 1+ missing exons 1 ecotype <br />                                                                             selective pressure acting protein coding genes dN/dS <br /> FIG. 4. Distribution Z scores standardized transcript abundance      &gt;&gt; 1 indicative definitive signature positive <br /> data affected accession. Data 210 genes      selection (Hurst 2002). Although are, average, a <br /> or missing exons 17 A. thaliana accessions (relative    higher number substitutions E-PAV genes compared <br /> to Col-0).                                                                  intact genes, clear signature adaptation <br />                                                                             suggest comparatively relaxed negative, than <br /> intact genes (3.35 1.13, respectively; randomization test               stronger positive, selection. <br /> P = 0.046).                                                                     We PAV genes significantly higher <br />    Overall, findings suggest exon losses             nucleotide diversity silent replacement sites. Both <br /> marginal effect expression profile genes                   observations suggestive weaker purifying selection; <br /> accessions affected. The higher levels alternative splicing              however, expected PAV genes were <br /> among genes affected exon loss raises possibility             higher balancing selection. Indeed, evidence <br /> significant proportion lost exons normally alternatively             suggest diversity resistance-associated <br /> spliced, reducing selection pressure exons                 genes maintained balancing selection (Van der Hoorn <br /> a functional protein product produced                     et al. 2002), overrepresented PAV <br /> absence anyway.                                                             genes. Balancing selection proposed stably main- <br />                                                                             tain intact gene absent allele (Tan et al. <br /> Discussion                                                                  2012). <br /> Intraspecies structural variations genes proposed                  So, balancing selection parsimonious explana- <br /> to play important role adaptation particular                   tion PAV genes associated higher nucleotide <br /> populations variation environmental conditions (Feuk                  diversity? A classic scenario transspecies polymorphism, <br /> et al. 2006). Here characterized presence&#226;&#8364;&#8220;absence                   associated balancing selection, assessed <br /> coding sequence variation 17 fully sequenced A. thaliana                 given limited sequence variation data available for <br /> genomes, relative reference accession Col-0, affecting               A. lyrata, A. thaliana&#226;&#8364;&#8482;s closest sequenced relative. It possible <br /> 411 genes including 81 instances coding region de-                 &#226;&#8364;&#339;gene/exon present&#226;&#8364;? &#226;&#8364;&#339;gene/exon absent&#226;&#8364;? <br /> letions. We significant enrichment genes associ-                 alleles selection maintained different <br /> ated GO terms protein nucleotide binding                A. thaliana populations, allowing better adapt to <br /> well signal transduction. Both gene family Pfam anno-                local environment. This consistent the <br /> tation enrichment analysis revealed significant enrichments                 increase nucleotide diversity, scenario be <br /> of gene members disease resistance associated NBS-                 distinguished alternative neutral models. Conditional <br /> LRR gene families. Significant deviations random expec-                neutrality PAV loci, functional gene ceased <br /> tations observed previous studies PAV genes                 adaptive environments, be <br /> in plants, similar overrepresentation resistance-asso-              ruled (e.g., case resistance genes cor- <br /> ciated gene families PAV genes. For instance, sor-                 responding pathogen absent [Gos Wright 2008]). In <br /> ghum (Zheng et al. 2011), PAV genes enriched                    case, absent allele selective advantage <br /> Pfam categories, including NB-ARC domain-containing                     point result relaxed constraints asso- <br /> family. In soybean (McHale et al. 2012), PAV-affected genes                 ciated PAV genes Arabidopsis populations. <br /> have enriched members NB-                  Moreover, model generalized relaxed constraints affecting <br /> ARC family, GO category &#226;&#8364;&#339;defense re-                      PAV loci lead increased nucleotide diversity <br /> sponse.&#226;&#8364;? CDS-PAV genes shown deviate                      slight increases dN/dS. <br />  <br />                                                                                                                                             65 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                   MBE <br />     Tajima&#226;&#8364;&#8482;s D, comparison estimators   (the pop-       tend accumulate near genes lower selective <br /> ulation <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> rate 4Nem)&#226;&#8364;&#8221;the number segregating               pressures located regions efficient purging TE <br /> sites average number pairwise differences        sequence. Indeed, set E-PAV genes, higher <br /> sequences (Tajima 1989)&#226;&#8364;&#8221;offers reliable estimate          density TEs vicinity. In addition, that <br /> selective pressures acting gene incorporates infor-      genes undergoing PAV increased proportion of <br /> mation distribution segregating alleles           motifs associated recombination hotspots their <br /> species. This allows accurate estimations degree        sequence. Both findings consistent PAV events being <br /> and direction departure sequence evolution             associated genes located genomic regions evolving <br /> neutral expectation (although nonselectionist interpretations       reduced selective constraints. Moreover, higher TE con- <br /> of D possible, recent population expansion      tent hotspot motifs consistent suggestion <br /> bottlenecking negative positive D, respectively)            unequal recombination homologs a <br /> (Tajima 1989). Tajima&#226;&#8364;&#8482;s D values provide evidence            major mechanism generating P/A polymorphisms (Tan <br /> for E-PAV CDS-PAV genes balancing             et al. 2012). However, noted recombino- <br /> selection. Taking dN/dS, nucleotide diversity, D estimates      genic motif necessary sufficient recombina- <br /> together, PAV genes appear evolving                tion event occur (Johnston Cutler 2012), such, <br /> relaxed constraints.                                                connection, any, PAV remains speculative. <br />     A signature relaxed selection associated PAV genes         All features considered suggest al- <br /> is combined variety features associ-       individual deletions adaptive <br /> ated lower gene essentiality. We PAV genes          value, overall coding region loss disproportionally affects <br /> have lower expression levels higher tissue specificity;    genes reduced selective pressures. So these <br /> of features associated higher rates         results reconciled enrichment certain gene families <br /> substitutions reduced gene essentiality (Wolf et al. 2006;      GO functional terms? The enrichment specific func- <br /> Cherry 2010). Older genes considered essen-          tional categories gene families PAV genes (fig. 1) <br /> tial (Chen, Trachana et al. 2012) associated (in      leads implication adaptive pressures favoring PAV on <br /> humans, flies Aspergillus) higher expression level       genes related specific biological processes (Tan et al. 2012). <br /> and stronger purifying selection (Wolf et al. 2009). We       However, shown, PAV genes associated a <br /> that PAV genes are, average, newer additions              variety features suggestive lower selective constraints. <br /> genome exons affected PAV              We argue enrichment certain GO categories <br /> an orthologous exon A. lyrata (663/794). We note            and/or gene families genes associated particular <br /> both E-PAV CDS-PAV genes enriched reverse tran-          genomic feature does not, itself, allow draw conclu- <br /> scriptase domains (supplementary figs. S4 S5,                   sions adaptive processes genes be <br /> Supplementary Material online) E-PAV genes trans-           undergoing. Consistent this, intact genes <br /> posase domains (supplementary fig. S4, Supplementary                associated gene categories PAV genes are <br /> Material online), suggesting exonization TEs origin       enriched signatures reduced selection <br /> of PAV-affected exons.                                         (supplementary table S7, Supplementary Material online). <br />     In addition, fact gene expression marginally   This notable sets genes involved in, example, <br /> reduced accessions affected exon deletion events sug-         signal transduction, nucleic acid binding, NBS-LRR <br /> gests lost exons limited impact        family&#226;&#8364;&#8221;categories enriched PAV genes (fig. 1). For <br /> on gene functionality. This possibly explained cases     instance, compare set E-PAV genes set of <br /> by alternative splicing, associated     genes exons present set NBS-LRR genes to <br /> an increased frequency exon loss humans, mice,            set genes belonging families, both <br /> rats&#226;&#8364;&#8221;alternatively spliced forms likely conserved    E-PAV NBS-LRR genes comparatively newer additions <br /> between species constitutive exons (Modrek Lee             genome, higher dN/dS ratio, higher number of <br /> 2003). In A. thaliana, genes E-PAV           alternative splicing events, higher number paralogs, a <br /> under weaker purifying selection greater number          higher proportion SNPs, closer TEs (sup- <br /> of alternative splice events compared intact genes. This       plementary table S7, Supplementary Material online). We <br /> observation suggests alternatively spliced exons likely    note proportion polymorphic sites higher not <br /> to reduced selective constraints compared con-        PAV genes genes functional category. To <br /> stitutive exons, exon deletions      demonstrate PAV genes bias comparison of, <br /> of detrimental effect loss constitutive exon. To    example, set NBS-LRR genes set genes <br /> the best knowledge, time exon loss    belonging families, repeat analysis restricted <br /> events associated elevated alternative splicing      intact genes observe result (supplemen- <br /> levels species species.                tary table S7, Supplementary Material online). <br />     The genomic context genes linked          The fact observed fewer PAV genes previ- <br /> patterns sequence evolution features associated         ous study examining 80 fully sequenced Arabidopsis genomes <br /> gene essentiality. A recent study A. thaliana correlated     (n = 2,741; Tan et al. 2012) likely differences meth- <br /> the presence TEs adjacent genes sequence variation       odology. First, analysis uses 17 genomes assembled using a <br /> within gene (Wang, Weigel, et al. 2013), suggesting TEs        combination read-to-reference genome (Col-0) alignment <br />  <br /> 66 <br />  Presence&#226;&#8364;&#8220;Absence Variation A. thaliana . doi:10.1093/molbev/mst166                                                     MBE <br /> and novo approaches, and&#226;&#8364;&#8221;importantly&#226;&#8364;&#8221;for tran-            Zu-0) obtained Gan et al. (2011). We did not <br /> scriptome data available (Gan et al. 2011),       use data Po-0 unusually high frequency <br /> the 80 accessions reported Cao et al. (2011). Second,        heterozygosity high similarity Oy-0 (Gan et al. 2011). <br /> use conservative methodology defining significant       Each genome fully sequenced assembled, using <br /> deletions Tan et al. (2012) define PAV genes using      combination novo assembly read mapping the <br /> is referred &#226;&#8364;&#339;broad <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>&#226;&#8364;?: &#226;&#8364;&#339;one    reference accession, Col-0. <br /> particular locus genomes compared the <br /> others.&#226;&#8364;? This allows gene called PAV gene        Detecting Missing Exons Relative Col-0 <br /> if copy exists different locus. To minimize inclusion   For analysis, selected set deletions spanning least <br /> of rearrangement events deletions, Tan et al. (2012) exam-      exon accession relative Col-0 <br /> ined predicted PAV genes using BlastN refer-       reference genome wider set deletion events de- <br /> ence accession, excluding &#226;&#8364;&#339;absent&#226;&#8364;? category           scribed Gan et al. (2011). Exons absent Col-0 refer- <br /> gene counterpart matches &gt;50% <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span>.           ence genome present accessions included <br /> Our <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> PAV restrictive deemed        analysis. Confirmation deletions described by <br /> an exon gene deleted genome alignments showed          original authors analyzed deletion breakpoints (Gan <br /> that deletion spanned exon gene      et al. 2011). In data set, deletion breakpoints esti- <br /> with single identifiable <span id='am-85' about='xsp:base' typeof='owl:Thing'>base</span> remaining. Finally, Tan    mated ~30 bp, left right consensus se- <br /> et al. (2012) study used genomes assembled according        quences established growing inward these <br /> TAIR8 annotated positions, data assembled          estimates using read-mapping information. If was <br /> according TAIR10. There small risk, therefore, having   deletion, ends overlap. Gan et al. (2011) <br /> incorporated now-obsolete gene models findings.         confirmed alignments left right consensus <br /> Regardless methodological differences result-       sequences, excluding errors sequencing misassem- <br /> ing variation sample size, worth noting          bly. We confirmed presence absence each <br /> results contradiction previous studies      individual exon 17 accessions relative Col-0 <br /> examining PAV Arabidopsis species,            genome annotation using BlastN default parameters <br /> we similar deviations random expectations         (Altschul et al. 1990). Sequence alignments obtained <br /> functional annotations genes. Our analysis sequence          using best hit homolog Smith&#226;&#8364;&#8220;Waterman algo- <br /> evolution genic features PAV genes             rithm (fasta35 parameters&#226;&#8364;&#8220;a&#226;&#8364;&#8220;A) (Pearson 2000). We <br /> rule possibilities conditional neutrality PAV        confirmed exon missing 1) alignment could <br /> loci balancing selection acting PAV              2) nucleotide positions in <br /> genes, allowing adaptation environmental conditions         Col-0 exon mapped nucleotide accession. <br /> of specific ecotypes. Instead, findings presented show <br /> that PAV events explained nonadaptive interpre-        Functional Category Enrichment Analysis <br /> tation genes reduced constraints more <br />                                                                    Four gene classification schemes obtained. GOslim terms <br /> susceptible spread allele variants containing signifi- <br />                                                                    obtained TAIR (ftp://ftp.arabidopsis.org/home/tair/ <br /> cant deletions. <br />                                                                    Ontologies/Gene_Ontology/ATH_GO_GOSLIM.txt, dated 9 <br />     In summary, results suggest significant <br />                                                                    July 2013 [last accessed October 8, 2013]), excluding terms <br /> enrichment functional categories PAV genes was <br />                                                                    unsupported experimental computational analysis, <br /> observed, exon loss events observed newer, poorly <br />                                                                    is, evidence codes ND, NR, NAS. GO term annota- <br /> functionally characterized genes associated signatures <br />                                                                    tions obtained Ensembl BioMart (17 July 2013) <br /> linked essential genes evolving lower purifying <br />                                                                    (Smedley et al. 2009). &#226;&#8364;&#339;Pfam&#226;&#8364;? terms obtained from <br /> or balancing selection. This reduce potential func- <br />                                                                    Pfam v27.0 (17 July 2013) (Punta et al. 2012). In addition, <br /> tional relevance structural variations genes. <br />                                                                    7,119 genes classified 49 distinct families Gan <br /> We conclude adaptive model PAV <br />                                                                    et al. (2011). Statistical significance enrichment both <br /> cannot ruled out, observed functional enrichments <br />                                                                    GOslim, GO terms, Pfam class family membership <br /> among PAV genes increased nucleotide diversity can <br />                                                                    E-PAV CDS-PAV-affected genes assessed <br /> also interpreted invoking selection. <br />                                                                    using Monte Carlo random sampling (1000 randomizations), <br />                                                                    P value enrichment category obtained <br /> Materials Methods                                              using Z test. The significance individual categories was <br /> Genome Sequence Annotations                                    corrected multiple testing Benjamini&#226;&#8364;&#8220;Hochberg <br /> Exon coordinates A. thaliana strain Col-0 obtained        procedure. <br /> from The Arabidopsis Information Resource (TAIR) (ftp://ftp. <br /> arabidopsis.org/home/tair/Genes/TAIR10_genome_release/             Sequence Evolution Analysis <br /> TAIR10_gff3/TAIR10_GFF3_genes.gff, dated 20 March 2012             To approximate selective constraint gene, calculated <br /> [last accessed October 8, 2013]). The genomes 17                dN/dS. For gene, obtained local alignment of <br /> A. thaliana accessions (Bur-0, Can-0, Ct-1, Edi-0, Hi-0, Kn-0,     Col-0 CDS A. lyrata ortholog, using the <br /> Ler-0, Mt-0, No-0, Oy-0, Rsch-4, Sf-2, Tsu-0, Wil-2, Ws-0, Wu-0,   Smith&#226;&#8364;&#8220;Waterman algorithm (fasta35 parameters&#226;&#8364;&#8220;a&#226;&#8364;&#8220;A) <br />  <br />                                                                                                                                    67 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                        MBE <br /> (Pearson 2000). dN/dS calculated using Yang              excluded analysis. EST alignments used <br /> Nielson model, implemented yn00 package phy-            create exon template. Alternative splicing events per <br /> logenetic analysis maximum likelihood (PAML) (Yang                gene identified comparing alignment coordinates for <br /> 2009). Using substitution estimates, above, SNP data          individual EST exon annotations. As low EST cov- <br /> from (Gan et al. 2011), estimated Tajima&#226;&#8364;&#8482;s D (Tajima         erage increase number falsely positive claims that <br /> 1989) gene. Nucleotide diversity calculated according      exon constitutive, spliced, excluded genes <br /> Gan et al. (2011).                                                   10 fewer ESTs. ESTs assigned genes using gene <br />                                                                      annotation coordinates. A comparable alternative splicing <br /> Paralog Number Gene Age Annotations                              index avoids transcript coverage biases obtained <br /> Ortholog paralog data obtained BioMart                 using transcript normalization method described in <br /> (Vilella et al. 2009). A proxy gene age established          Kim et al. (2007). Briefly, gene 100 random samples <br /> using taxonomic classifications, based phylostrati-           10 ESTs selected. Finally, number alternative <br /> graphic method (Domazet-Los&#203;&#8225;o et al. 2007). If candidate        splicing events calculated random sample (as <br /> ortholog identified A. thaliana gene 15       detailed earlier), overall average calculated gene. <br /> plant algal species minimum identity 30%, the <br /> gene considered old &#226;&#8364;&#339;broadest&#226;&#8364;? taxonomic         Randomization Test <br /> category held common (see supplementary table S8,                 A randomization test used obtain numerical P values <br /> Supplementary Material online). This allowed make              assess statistical significance variation the <br /> use ortholog data despite divergence times relative            characteristics PAV-affected genes compared intact <br /> A. thaliana known closest relatives&#226;&#8364;&#8221;at ~5         genes. In brief, contrasted genomic feature parameters <br /> million years A. lyrata (Kuittinen et al. 2004) 20 million   E-PAV (n = 330) CDS-PAV genes (n = 81) distri- <br /> years Brassica rapa (Yang et al. 1999).                          bution means genomic feature s = 10,000 <br />                                                                      randomly generated subsets equal number genes <br /> Gene Expression <br />                                                                      drawn complete gene set. The numerical P value <br /> Expression specificity calculated tissue specificity        calculated follows: let q number times the <br /> index (tau) (Yanai et al. 2005), using massively parallel        mean value PAV set exceeded mean value the <br /> signature sequencing (MPSS) database (Brenner et al. 2000;           randomly generated subset. Letting r = s   q, P value <br /> Meyers et al. 2004; Nakano et al. 2006). Expression levels      test r + 1/s + 1. <br /> calculated using RNAseq transcript abundance data, abso- <br /> lute read values corrected sequence length acces-         Supplementary Material <br /> sion (known rpkm values: gene, number reads            Supplementary tables S1&#226;&#8364;&#8220;S8 figures S1&#226;&#8364;&#8220;S5 available <br /> per kilobase million mapped reads) (Gan et al. 2011).            Molecular Biology Evolution online (http://www.mbe. <br />                                                                      oxfordjournals.org/). <br /> TE Hotspot Motif Density <br /> TE coordinates A. thaliana strain Col-0 obtained            Acknowledgments <br /> from TAIR (file &#226;&#8364;&#339;TAIR10_Transposable_Elements,&#226;&#8364;? dated 20             The authors acknowledge valuable comments by <br /> March 2012). For analyses, identified instance       anonymous reviewers. This work supported a <br /> all 25 hotspot-associated motifs (of 5&#226;&#8364;&#8220;9 bp) described            University Bath fee studentship S.J.B., CONACyT schol- <br /> Horton et al. (2012) Col-0 reference genome. TE           arships A.C.M. J.M.T.C., UK&#226;&#8364;&#8220;China scholarship for <br /> hotspot motif density gene calculated           excellence University Bath research studentship L.C., <br /> proportion base pairs occupied TE hotspot motif         BBSRC grant (grant number BB/F022697/1) P.X.K., and <br /> within windows size 1 100 kb centered nucleotide        Royal Society Dorothy Hodgkin Research Fellowship <br /> at gene&#226;&#8364;&#8482;s midpoint. Windows consist coding           (DH071902), Royal Society research grant (grant number <br /> noncoding sequence region length (window size)/          RG0870644), Royal Society research grant fellows <br /> 2 up- downstream midpoint base. Both TE               (grant number RG080272) A.U.O. <br /> hotspot motif density calculated number TE or <br /> motif bases, respectively, relative number intergenic      References <br /> or genic bases contained window,          Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. 1990. Basic local <br /> total number bases window.                                     alignment search tool. J Mol Biol. 215:403&#226;&#8364;&#8220;410. <br />                                                                      Bakker EG, Toomajian C, Kreitman M, Bergelson J. 2006. A genome-wide <br /> Alternative Splicing Events                                              survey R gene polymorphisms Arabidopsis. Plant Cell 18: <br />                                                                          1803&#226;&#8364;&#8220;1818. <br /> Alternative splicing events identified using methods        Boguski MS, Lowe TMJ, Tolstoshev CM. 1993. dbEST&#226;&#8364;&#8221;database for <br /> described Chen et al. (2011). In brief, number                 expressed sequence tags. Nat Genet. 4:332&#226;&#8364;&#8220;333. <br /> alternative splicing events gene identified aligning      Brenner S, Johnson M, Bridgham J, et al. (24 co-authors). 2000. Gene <br /> expressed sequence tag (EST) data obtained dbEST                    expression analysis massively parallel signature sequencing <br />                                                                          (MPSS) microbead arrays. Nat Biotechnol. 18:630&#226;&#8364;&#8220;634. <br /> (Boguski et al. 1993) genome sequence (ftp://ftp.ncbi.        Cao J, Schneeberger K, Ossowski S, et al. (17 co-authors). 2011. Whole- <br /> nih.gov/repository/dbEST, accessed May 1, 2011). Those              genome sequencing multiple Arabidopsis thaliana populations. <br /> ESTs aligning regions annotated gene                     Nat Genet. 43:956&#226;&#8364;&#8220;963. <br />  <br /> 68 <br />  Presence&#226;&#8364;&#8220;Absence Variation A. thaliana . doi:10.1093/molbev/mst166                                                                    MBE <br /> Cheadle C, Vawter MP, Freed WJ, Becker KG. 2003. Analysis micro-              analyses mRNA small RNA. Nucleic Acids Res. 34: <br />     array data using Z score transformation. J Mol Diagn. 5:73&#226;&#8364;&#8220;81.               D731&#226;&#8364;&#8220;D735. <br /> Chen L, Tovar-Corona JM, Urrutia AO. 2011. Increased levels noisy         Oliver KR, Greene WK. 2009. Transposable elements: powerful facilita- <br />     splicing cancers, oncogene-derived transcripts. Hum           tors evolution. Bioessays 31:703&#226;&#8364;&#8220;714. <br />     Mol Genet. 20:4422&#226;&#8364;&#8220;4429.                                                 Ossowski S, Schneeberger K, Clark RM, Lanz C, Warthmann N, Weigel D. <br /> Chen L, Tovar-Corona JM, Urrutia AO. 2012. Alternative splicing:               2008. Sequencing natural strains Arabidopsis thaliana with <br />     potential source functional innovation eukaryotic                  short reads. Genome Res. 18:2024&#226;&#8364;&#8220;2033. <br />     genome. Int J Evol Biol. 2012:10.                                        Pearson WR. 2000. Flexible sequence similarity searching the <br /> Chen W-H, Trachana K, Lercher MJ, Bork P. 2012. Younger genes                FASTA3 Program Package. Methods Mol Biol. 132:185&#226;&#8364;&#8220;219. <br />     likely essential older genes, duplicates    Punta M, Coggill PC, Eberhardt RY, et al. (16 co-authors). 2012. The Pfam <br />     likely essential singletons age. Mol Biol Evol.       protein families database. Nucleic Acids Res. 40:D290&#226;&#8364;&#8220;D301. <br />     29:1703&#226;&#8364;&#8220;1706.                                                            Santuari L, Pradervand S, Amiguet-Vercher A-M, Thomas J, Dorcey E, <br /> Cherry JL. 2010. Expression level, evolutionary rate, cost            Harshman K, Xenarios I, Juenger T, Hardtke C. 2010. Substantial <br />     expression. Genome Biol Evol. 2:757&#226;&#8364;&#8220;769.                                     deletion overlap divergent Arabidopsis genomes revealed <br /> Clark RM, Schweikert G, Toomajian C, et al. (18 co-authors). 2007.               intersection short reads tiling arrays. Genome Biol. 11:R4. <br />     Common sequence polymorphisms shaping genetic diversity               Shen J, Araki H, Chen L, Chen JQ, Tian D. 2006. Unique evolutionary <br />     Arabidopsis thaliana. Science 317:338&#226;&#8364;&#8220;342.                                   mechanism R-genes presence/absence polymorphism <br /> DeYoung BJ, Innes RW. 2006. Plant NBS-LRR proteins pathogen                   Arabidopsis thaliana. Genetics 172:1243&#226;&#8364;&#8220;1250. <br />     sensing host defense. Nat Immunol. 7:1243&#226;&#8364;&#8220;1249.                      Smedley D, Haider S, Ballester B, Holland R, London D, Thorisson G, <br /> Domazet-Los&#203;&#8225;o T, Brajkovic&#194;&#180; J, Tautz D. 2007. A phylostratigraphy                Kasprzyk A. 2009. BioMart&#226;&#8364;&#8221;biological queries easy. BMC <br />     approach uncover genomic history major adaptations              Genomics 10:22. <br />     metazoan lineages. Trends Genet. 23:533&#226;&#8364;&#8220;539.                             Swanson-Wagner RA, Eichten SR, Kumari S, Tiffin P, Stein JC, Ware D, <br /> Feuk L, Carson AR, Scherer SW. 2006. Structural variation human           Springer NM. 2010. Pervasive gene content variation copy <br />     genome. Nat Rev Genet. 7:85&#226;&#8364;&#8220;97.                                              number variation maize undomesticated progenitor. <br /> Gan X, Stegle O, Behr J, et al. (23 co-authors). 2011. Multiple reference        Genome Res. 20:1689&#226;&#8364;&#8220;1699. <br />     genomes transcriptomes Arabidopsis thaliana. Nature 477:         Tajima F. 1989. Statistical method testing neutral mutation <br />     419&#226;&#8364;&#8220;423.                                                                     hypothesis DNA polymorphism. Genetics 123:585&#226;&#8364;&#8220;595. <br /> Gos G, Wright SI. 2008. Conditional neutrality adjacent NBS-LRR       Tan S, Zhong Y, Hou H, Yang S, Tian D. 2012. Variation of <br />     disease resistance loci natural populations Arabidopsis lyrata.        presence/absence genes Arabidopsis populations. BMC <br />     Mol Ecol. 17:4953&#226;&#8364;&#8220;4962.                                                      Evol Biol. 12:86. <br /> Hanada K, Kuromori T, Myouga F, Toyoda T, Li WH, Shinozaki K. 2009.          Van der Hoorn RAL, De Wit PJGM, Joosten MHAJ. 2002. Balancing <br />     Evolutionary persistence functional compensation duplicate             selection favors guarding resistance proteins. Trends Plant Sci. 7: <br />     genes Arabidopsis. Genome Biol Evol. 1:409&#226;&#8364;&#8220;414.      <br /> </body></html>