<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Research article <br />  <br />                     Mitochondrial complex I activity and <br />                        NAD+/NADH balance regulate <br />                          breast cancer progression <br />               Antonio F. Santidrian,1,2 Akemi Matsuno-Yagi,1 Melissa Ritland,1,2 Byoung B. Seo,1 <br />             Sarah E. LeBoeuf,1,2 Laurie J. Gay,1,2 Takao Yagi,1 Brunhilde Felding-Habermann1,2 <br />    1Department    Molecular Experimental Medicine 2Department Chemical Physiology, The Scripps Research Institute, La Jolla, California, USA. <br />  <br />  <br />     Despite advances clinical therapy, metastasis remains leading cause death breast cancer patients. <br />     Mutations mitochondrial DNA, including affecting complex I oxidative phosphorylation, are <br />     breast tumors facilitate metastasis. This study identifies mitochondrial complex I as <br />     critical defining aggressive <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>phenotype</span> breast cancer cells. Specific enhancement mitochondrial <br />     complex I activity inhibited tumor growth metastasis regulation tumor cell NAD+/NADH <br />     redox balance, mTORC1 activity, autophagy. Conversely, nonlethal reduction NAD + levels inter- <br />     fering nicotinamide phosphoribosyltransferase expression rendered tumor cells aggressive and <br />     increased metastasis. The results translate new therapeutic strategy: enhancement NAD+/NADH <br />     balance treatment NAD+ precursors inhibited metastasis xenograft models, increased animal <br />     <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>survival</span>, strongly interfered oncogene-driven breast cancer progression MMTV-PyMT mouse <br />     model. Thus, aberration mitochondrial complex I NADH dehydrogenase activity profoundly enhance <br />     aggressiveness human breast cancer cells, therapeutic normalization NAD +/NADH balance <br />     inhibit metastasis prevent disease progression. <br />  <br /> Introduction                                                                             noncoding D-loop region (9, 10), mitochondrial complex I <br /> Despite advances clinical therapy, metastasis lead-                      subunit genes (10). These defects suggest role mtDNA muta- <br /> ing cause death breast cancer patients (1). A clearer under-                       tions complex I tumor progression. Therefore, cell <br /> standing molecular mechanisms drive metastasis                              lines excellent models defining specific role complex I <br /> help develop effective therapies (2). Our present study                          activity tumor growth metastatic aggressiveness. <br /> focused metabolism essential driver tumor growth                                Complex I gatekeeper respiratory chain catalyz- <br /> and metastasis, potentially common breast cancer types.                           es step NADH oxidation. It elevates NAD+/NADH <br /> Normal cells primarily use mitochondrial oxidative phosphory-                            ratio translocates protons inner mitochondrial <br /> lation (OXPHOS) energy production, cancer cells                              membrane, ultimately leads energy production. mtDNA <br /> depend aerobic glycolysis (the Warburg effect) generate                            mutations genes encoding complex I subunits in <br /> energy glycolytic intermediates enhanced growth (3, 4).                          malignancies including breast cancer (6, 11&#226;&#8364;&#8220;14, 16). However, it <br /> Tumor cells generate high levels reduced forms NAD+,                          largely unknown alterations complex I cellular <br /> NADH, NADPH important cofactors redox compo-                                  NAD+/NADH redox balance affect tumorigenesis metastasis. <br /> nents (4, 5). These altered metabolic activities linked                           We used unique approach define contributions com- <br /> mitochondrial dysfunction inhibits OXPHOS, increases                                plex I activity breast cancer progression, based expression <br /> ROS, promotes uncontrolled growth, causes DNA damage                                 yeast NADH dehydrogenase Ndi1 human tumor cells. <br /> that supports metastatic phenotype (6, 7). Mitochon-                           Ndi1 encodes single protein faces inner mitochondrial <br /> drial dysfunctions caused mutations mitochondrial                           matrix oxidizes NADH Krebs cycle. Unlike mamma- <br /> DNA (mtDNA) nuclear genes encoding mitochondrial pro-                                 lian complex I, Ndi1 rotenone insensitive (17). Ndi1 contains 26 <br /> teins (6, 8) essential respiratory chain/OXPHOS                         N-terminal residues mitochondrial import (17), func- <br /> system. Due lack protective histones limited DNA                           tionally expressed mammalian cells (18, 19), does cause <br /> repair (8), mtDNA mutations occur high rates                           immune response (20). Ndi1 restores complex I function (18) in <br /> in tumors including breast cancer (6, 9&#226;&#8364;&#8220;14), suggests                         diseased cells, e.g., neurons Parkinson&#226;&#8364;&#8482;s disease (21) optic <br /> defects OXPHOS contribute tumorigenesis.                                     neuropathy (22); protects cardiomyocytes ischemic reperfu- <br />   By combining nuclear genome recipient cell                           sion injury (23); increases lifespan Drosophila (24). Recently, <br /> mitochondrial genome donor cell using cybrid technology,                            shown Ndi1 expression complex I&#226;&#8364;&#8220;deficient tumor <br /> mitochondria triple-negative aggressive breast cancer                           cells reduce soft agar colony formation (25). <br /> cell lines MDA-MB-435 (15) MDA-MB-231 facilitated tumor                                 We used Ndi1 investigate cause-and-effect relationship <br /> progression metastasis nonmetastatic tumor cells (7, 10).                         aberrant mitochondrial complex I activity malig- <br /> The donor cell lines harbor mtDNA mutations tRNAs,                             nant progression breast cancer. Moreover, analyzed meta- <br />                                                                                          bolic alterations caused mitochondrial complex I malfunction <br /> Conflict interest: The authors declared conflict exists.     translated information gained novel therapeutic <br /> Citation article: J Clin Invest. 2013;123(3):1068&#226;&#8364;&#8220;1081. doi:10.1172/JCI64264.   approach breast cancer progression. <br />  <br /> 1068	                                 The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                   research article <br />  <br />  <br />  <br />  <br /> Figure 1 <br /> Enhancement mitochondrial complex I activity integration Ndi1: metabolic characterization. (A) Ndi1 expressed MDA-MB-435 and <br /> MDA-MB-231 human cancer cells lentiviral transduction localized mitochondria, shown dual <span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span> immunocytochemistry. Control <br /> cells (Ctrl) transduced vector. Shown Ndi1 (green), mitochondrial complex V (red), merged (yellow) staining patterns of <br /> the representative fields. Original magnification, &#195;&#8212;20. (B) Ndi1 integrated mitochondrial respiratory chain enhanced tumor <br /> cell respiration. Respiration MDA-MB-435 MDA-MB-231 (denoted &#226;&#8364;&#339;435&#226;&#8364;? &#226;&#8364;&#339;231,&#226;&#8364;? respectively) control Ndi1-expressing cells, mea- <br /> sured oxygen consumption. R, routine respiration; Rot, rotenone (mammalian complex I inhibitor); AA, antimycin A (complex III inhibitor). All <br /> parameters measured high-resolution respirometry intact cells. (C) Effects Ndi1 expression cellular metabolism MDA-MB-435 <br /> and MDA-MB-231 cells. mtDNA content analyzed quantitative real-time PCR referenced nuclear genomic DNA. Mitochondrial mem- <br /> brane potential analyzed flow cytometry expressed geometric mean signal. ATP levels measured ATP-dependent <br /> luciferase activity. Lactate production measured fluorometry. Data mean &#194;&#177; SEM (n = 3). *P &lt; 0.05, unpaired 2-tailed Student&#226;&#8364;&#8482;s t test. <br />  <br />  <br /> Results                                                                        tional contribution Ndi1, respiration Ndi1-expressing MDA- <br /> Enhancement complex I activity human breast cancer cells inhibits        MB-435 MDA-MB-231 cells inhibited mammalian <br /> tumor growth metastasis. Complex I function MDA-MB-435                  complex I antagonist rotenone 20% 28%, respectively, <br /> and MDA-MB-231 cells enhanced stable transduction                  contrast 85% 84% controls. In cases, oxygen con- <br /> Ndi1. The encoded enzyme expressed 90% cells, local-             sumption fully blocked complex III inhibitor antimycin <br /> ized mitochondria (Figure 1A) altering stoichiom-               A (Figure 1B), indicates measurements specifically <br /> etry mitochondrial complexes (Supplemental Figure 1; supple-                reported mitochondrial respiration. Ndi1 increased OXPHOS- <br /> mental material available online article; doi:10.1172/               uncoupled respiration, shown oligomycin-inhibited <br /> JCI64264DS1), significantly enhanced mitochondrial respira-                respiratory rate (Supplemental Figure 2A), enhanced com- <br /> tion intact tumor cells (Figure 1B). Confirming func-               plex I&#226;&#8364;&#8220;mediated respiration permeabilized cells addition <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013	                1069 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Mitochondrial complex I activity modulates tumor growth metastasis. (A) Ndi1 expression inhibited mammary fat pad tumor growth of <br /> MDA-MB-435 MDA-MB-231 cells. Control cells transduced vector (n = 6). (B) Ndi1 expression inhibited lung colonization <br /> (experimental metastasis) MDA-MB-435 MDA-MB-231 cells i.v. injection. Control cells transduced vector (n = 6). (C) <br /> Ndi1 expression inhibited multiorgan experimental metastasis, indicated noninvasive bioluminescence imaging 7 weeks i.v. injection <br /> of 2.5 &#195;&#8212; 105 MDA-MB-435 control Ndi1-expressing cells (n = 5). (D) Knockdown complex I subunit NFUFV1 expression inhibited complex <br /> I activity respiratory capacity MDA-MB-435 cells. NDUFV1-knockdown (shV1) control (shCT) cells compared. Complex I was <br /> immunocaptured cell lysates, analyzed based oxidation NADH NAD+, expressed mean OD/min/mg protein (n = 3). Routine mito- <br /> chondrial respiration, corrected residual oxygen consumption oxidative reactions, measured intact MDA-MB-435 control and <br /> NDUFV1-knockdown cells high-resolution respirometry (n = 3). (E) NDUFV1 knockdown increased lung colonization activity MDA-MB-435 <br /> cells. NDUFV1-knockdown control cells compared (n = 8). Data mean &#194;&#177; SEM. *P &lt; 0.05, **P &lt; 0.01, ***P &lt; 0.001, nonparametric <br /> Mann-Whitney test (A, B, E) unpaired 2-tailed Student&#226;&#8364;&#8482;s t test (D). <br />  <br />  <br />  <br /> of complex I substrates malate glutamate (Supplemental                    <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> corroborate role aberrant complex I activity metastat- <br /> Figure 2, B C). Ndi1 did change maximal capacity                ic progression, challenged findings independent <br /> the mitochondrial electron transfer (Supplemental Figure                reverse approach, asking interference tumor cell <br /> 2A). Resistance rotenone susceptibility flavone affirmed             complex I function enhances metastatic aggressiveness. To this <br /> the contribution Ndi1 (Supplemental Figure 2, B C). In             end, knocked NDUFV1 MDA-MB-435 cells. NDUFV1 <br /> tumor cell models, Ndi1 did alter mtDNA content mito-                   encodes 51-kDa complex I subunit, belongs the <br /> chondrial membrane potential (Figure 1C). Reflecting overall ener-             minimal assembly required catalysis carries NADH <br /> gy metabolism balance glycolysis mitochondrial                 binding site (26). Mutations gene cause complex I defi- <br /> OXPHOS, Ndi1 expression decreased ATP levels, lactate                  ciency severe neurological diseases Leigh syndrome (26). <br /> production unchanged breast cancer cell models (Fig-               NDUFV1 knockdown (Supplemental Figure 3) reduced complex <br /> ure 1C). These results demonstrated functional integration             I activity 92% cellular respiration 51% significantly <br /> Ndi1 NADH dehydrogenase respiratory chain 2 human                  enhanced metastatic activity aggressive MDA- <br /> cancer cell models, causing enhanced mitochondrial complex I                   MB-435 cell line (Figure 2, D E). <br /> activity major effect overall energy balance.                        These results provided definitive evidence specific modula- <br />   To investigate enhancement tumor cell complex I                   tion tumor cell complex I function significantly alter tumor <br /> activity affects tumorigenicity metastasis, Ndi1-expressing                growth metastatic activity. Therefore, complex I mutations <br /> F-luc&#226;&#8364;&#8220;tagged MDA-MB-435 MDA-MB-231 cells followed                     primary tumors breast cancer patients (6, 11&#226;&#8364;&#8220;14, 16) <br /> in female SCID mice noninvasive bioluminescence imaging. Ndi1               play key role disease progression. <br /> expression strikingly affected malignant phenotype tumor               Inhibition metastatic activity enhancement complex I func- <br /> cells, tumor growth rates mammary fat pad metastasis             tion depends autophagy. To investigate complex I activity <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> bloodstream significantly reduced (Figure 2, A&#226;&#8364;&#8220;C).               affects tumor growth metastasis, analyzed effects <br />  <br /> 1070	                         The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                   research article <br />  <br />  <br />  <br />  <br /> Figure 3 <br /> Mitochondrial complex I activity regulates mTORC1 autophagy. (A) Ndi1 expression enhanced resistance glucose deprivation MDA- <br /> MB-231 cells, shown 72 hours incubation medium 5 versus 1 mM glucose. Viability measured flow cytometry (non&#226;&#8364;&#8220;sub- <br /> G0/G1 population). n = 3 independent analyses. *P &lt; 0.05, unpaired 2-tailed Student&#226;&#8364;&#8482;s t test. (B) Ndi1 expression influenced mTORC1 activity <br /> and autophagy. Western blot analysis p62, phospho-AKT substrates, mTORC1 kinase&#226;&#8364;&#8220;related substrates phospho-S6Ser240/244 and <br /> phospho-4EBPThr37/46 MDA-MB-435 MDA-MB-231 control Ndi1-expressing cells. &#206;&#178;-Tubulin used protein loading control. Signal <br /> quantification, measured infrared imaging (total detectable bands) expressed relative control, shown below. Results represen- <br /> tative 5 independent experiments. Lanes run gel noncontiguous (white lines). (C) H&amp;E staining p62, Ki67, or <br /> Ndi1 expression mammary fat pad tumors 5 weeks implanting 2.5 &#195;&#8212; 105 MDA-MB-435 control versus Ndi1-expressing cells SCID mice. <br /> 2 representative tumors 6 shown group. Original magnification, &#195;&#8212;10. (D) Inhibition complex I activity NDUFV1 knockdown <br /> affected mTORC1 activity p62 elimination. Western blot analysis p62, phospho-AKT substrates, phospho-S6 Ser240/244, phospho- <br /> 4EBPThr37/46, comparing NDUFV1-knockdown versus control MDA-MB-435 cells. Signal quantification, measured infrared imaging (total of <br /> detectable bands) expressed relative control, shown below. Results representative 3 independent experiments. Lanes run <br /> on gel noncontiguous (white lines). <br />  <br />  <br />  <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013	                1071 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 4 <br /> Metastasis inhibition enhanced complex I activity depends autophagy. (A) ATG5 knockdown (shATG5) inhibited autophagy MDA-MB-435 <br /> and MDA-MB-231 control Ndi1-expressing cells, shown p62 LC3BI accumulation. Signal quantification ATG5, p62 signal, and <br /> LC3BI/II ratios, measured infrared imaging (total detectable bands) expressed relative control, shown below. &#206;&#178;-Tubulin served as <br /> protein loading control. Lanes run gel noncontiguous (white lines). (B) ATG5 knockdown blocked antimetastatic <br /> effect Ndi1 MDA-MB-435 MDA-MB-231 cells. Lung colonization measured ex vivo lung imaging 7 weeks i.v. injection of <br /> 2.5 &#195;&#8212; 105 tumor cells (n = 8 group). Boxes denote interquartile range; lines boxes denote median; whiskers denote minima maxima. <br /> *P &lt; 0.05, nonparametric Mann-Whitney test. (C) ATG5 knockdown enhanced multiorgan metastasis reversed metastasis inhibition Ndi1 <br /> in MDA-MB-435 cells. Shown noninvasive bioluminescence imaging 5 representative mice group 7 weeks tail vein injection of <br /> 2.5 &#195;&#8212; 105 MDA-MB-435 control Ndi1-expressing cells, ATG5 knockdown. <br />  <br />  <br />  <br /> of Ndi1 expression tumor cell viability growth. While                   based levels phospho-AKT substrates. Ndi1 clearly reduced <br /> Ndi1 did affect proliferation vitro (Supplemental Figure                AKT activity MDA-MB-231 cells inhibited mTORC1 in <br /> 4, A B), Ndi1 increased resistance glucose deprivation               cell lines (Figure 3B), suggested complex I activ- <br /> MDA-MB-231 cells (Figure 3A) suppressed tumorigenicity                     ity induce autophagy regulating mTORC1 signaling. This <br /> in cell lines (Figure 2A). A similar paradoxical phenomenon               process likely involves AKT MDA-MB-231 cells alternative <br /> was previously described linked ability tumor cells              mechanisms MDA-MB-435 cells. <br /> to undergo autophagy (27&#226;&#8364;&#8220;29), indication time                     Importantly, levels p62 Ki67 strongly decreased in <br /> that mitochondrial complex I involved. Following                  primary tumors expressing Ndi1 (Figure 3C), indicates that <br /> hypothesis breast cancer cell complex I activity affect               enhancement complex I increases tumor cell autophagy and <br /> autophagy, analyzed effects Ndi1 expression p62,                inhibits proliferation vivo. This conclusion strongly sup- <br /> ubiquitin-binding scaffolding protein eliminated                ported enhancement AKT mTORC1 activities clear <br /> autophagy (27, 30). Ndi1 expression increased p62 degradation                  inhibition p62 elimination NDUFV1 knockdown, which <br /> (Figure 3B), indicated enhancement mitochon-                     reduced complex I functionality tumor cells (Figure 3D) with- <br /> drial complex I activity induced autophagy. Following signaling                affecting proliferation (Supplemental Figure 4C). These <br /> pathways regulate autophagy, analyzed effects Ndi1                  results demonstrated tumor cell complex I activity regulate <br /> on mTORC1 activity AKT, mTOR&#226;&#8364;&#8482;s main regula-                         tumor cell mTORC1 signaling autophagy vitro vivo. <br /> tors. mTORC1 activity measured phosphorylation                         To analyze directly complex I&#226;&#8364;&#8220;mediated regulation of <br /> mTORC1-related substrates S6 4EBP1; AKT activity                       breast cancer metastasis involves control tumor cell autoph- <br />  <br /> 1072	                         The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                  research article <br />  <br />  <br />  <br />  <br /> Figure 5 <br /> NAD+ level modulation complex I NAD+ synthesis recycling pathways regulate AKT/mTORC1 activity, autophagy, metastasis. (A) <br /> Ndi1 expression enhanced NAD+/NADH balance. NAD+/NADH ratios whole-cell mitochondrial extracts MDA-MB-435 MDA-MB-231 <br /> control versus Ndi1-expressing cells. Ndi1 stabilized NAD+/NADH ratios, especially metabolic stress induced glucose deprivation and <br /> hypoxia. NAD+/NADH ratios stress measured whole-cell extracts 48 hours culture. (B) Interference NAD+ synthesis <br /> and recycling pathways reduced NAD+/NADH ratios. Knockdown NAMPT (shNAMPT) MDA-MB-435 MDA-MB-231 cells decreased <br /> NAD+/NADH ratios (whole-cell extracts 48 hours growth 5 mM glucose normoxia). (C) NAMPT knockdown increased lung coloniza- <br /> tion activity MDA-MB-435 MDA-MB-231 cells (n = 6 group). (D) NAMPT knockdown affected mTORC1 activity p62 elimination. <br /> Western blot analysis p62, phospho-AKT substrates, phospho-S6Ser240/244, phospho-4EBPThr37/46 MDA-MB-435 MDA-MB-231 <br /> NAMPT-knockdown versus control cells. &#206;&#178;-Tubulin served protein loading control. Signal quantification, measured infrared imaging (total <br /> of detectable bands) expressed relative control, shown below. Lanes run gel noncontiguous (white lines). <br /> Results representative 3 independent experiments. (A&#226;&#8364;&#8220;C) Data mean &#194;&#177; SEM. *P &lt; 0.05, **P &lt; 0.01, unpaired 2-tailed Student&#226;&#8364;&#8482;s t test <br /> (A B) nonparametric Mann-Whitney test (C). <br />  <br />  <br />  <br /> agy, knocked ATG5, protein required autophagy                     NAD+ level modulation complex I alteration NAD+ synthesis <br /> induction (31). Targeting ATG5 control Ndi1-expressing                 recycling pathways regulate AKT/mTORC1 activity, autophagy, and <br /> MDA-MB-435 MDA-MB-231 cells stable transduction                        metastatic activity. To investigate mechanism which <br /> with shRNA (Figure 4A Supplemental Figure 5A) inhibit-                    enhancement complex I activity inhibits tumor progression, <br /> ed autophagy, shown p62 LC3BI accumulation (Fig-                    ruled alterations ROS NADPH production <br /> ure 4A), affecting proliferation vitro (Supplemental               major underlying causes. Ndi1 expression MDA-MB-435 <br /> Figure 5B). Importantly, basal levels autophagy                   MDA-MB-231 cells did significantly alter ROS levels or <br /> facilitated metastasis MDA-MB-231 cells, ATG5 knockdown                    NADPH-reducing equivalents (Supplemental Figure 6, A B). <br /> abolished inhibitory effect Ndi1-mediated complex I                    Likewise, NDUFV1 knockdown did significantly affect ROS <br /> enhancement metastatic organ colonization (Figure 4, B                 (Supplemental Figure 6C), suggests enhancement of <br /> C). This seen primarily lungs MDA-MB-231                complex I activity Ndi1 inhibits tumorigenicity metastasis <br /> multiple organs MDA-MB-435, including lung, liver, bone,                  ROS-independent manner. <br /> brain, adrenal glands. These findings affirm conclusion                 A major function mammalian complex I Ndi1 NADH <br /> that inhibition metastasis enhancement mito-                    dehydrogenase activity. Ndi1 expression MDA-MB-435 and <br /> chondrial complex I activity depends autophagy induction                MDA-MB-231 cells increased NAD+/NADH ratios whole-cell <br /> 2 aggressive tumor cell models.                                               extracts purified mitochondria, particularly metabolic <br />  <br /> 	                            The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013	                 1073 <br />  research article <br />  <br />  <br />  <br />  <br /> Figure 6 <br /> NAD+ precursor treatment inhibits metastatic activity. (A) NAD+ precursor treatment enhanced NAD+/NADH ratio cultured MDA-MB-435 and <br /> MDA-MB-231 parental cells. NAD+/NADH levels measured 3 days cell treatment 10 mM NIC NAM complete medium. n = 3 <br /> independent experiments. (B C) NAD+ precursor treatment experimental mice inhibited lung metastasis. Lung colonization MDA-MB-435 <br /> (B) MDA-MB-231 (C) parental cells (2.5 &#195;&#8212; 105 i.v. each) mice treated NIC NAM (1% drinking water ad libitum the <br /> experiment). Controls received treatment (plain drinking water pH). Metastatic growth measured repeated noninvasive biolu- <br /> minescence imaging. n = 6 group. (D) NIC NAM treatment influenced mTORC1 activity autophagy. Western blot analysis p62, phos- <br /> pho-AKT substrates, phospho-S6Ser240/244 MDA-MB-435 MDA-MB-231 parental cells 48 hours treatment 10 mM <br /> NIC NAM. &#206;&#178;-Tubulin served protein loading control. Signal quantification, measured infrared imaging (total detectable bands) and <br /> expressed relative control, shown below. Results representative 3 independent experiments. *P &lt; 0.05, **P &lt; 0.01, unpaired 2-tailed <br /> Student&#226;&#8364;&#8482;s t test (A) nonparametric Mann-Whitney test (B C). <br />  <br />  <br />  <br /> stress induced glucose oxygen deprivation (Figure 5A). In                MB-231 cells (Figure 5C), indicative cause-and-effect relation- <br /> culture, Ndi1-mediated increase NAD+/NADH ratios                     ship reduced NAD+/NADH ratios metastatic activity. <br /> measurable exponential cell growth (Supplemental Fig-                   The increased metastatic activity interference cellu- <br /> ure 7, A B). Conversely, disturbance endogenous complex                 lar NAD+/NADH ratios involved altered mTORC1 activity and <br /> I activity NDUFV1 knockdown reduced NAD+/NADH ratios                       autophagy. NAMPT-knockdown cells analyzed normal <br /> (Supplemental Figure 8). We reasoned modulation cel-               conditions glucose deprivation. Under metabolic stress to <br /> lular redox potential (NAD+/NADH ratio) mitochondrial com-                  glucose deprivation (1 mM glucose), NAMPT knockdown induced <br /> plex I activity regulate metastatic activity tumor cells.           p62 accumulation enhanced AKT mTORC1 activities <br /> To test hypothesis using independent approach, experi-              (Figure 5D), indicated NAD+ levels regulate AKT/ <br /> mentally decreased NAD+/NADH ratios MDA-MB-435                          mTORC1 signaling autophagy pathways MDA-MB-435 <br /> MDA-MB-231 cells analyzed effect metastasis vivo.                MDA-MB-231 cells. <br />   To decrease NAD+/NADH ratio tumor cells, dis-                    Although results showed downregulation NAMPT <br /> turbed NAD+ synthesis recycling pathway targeting                   expression enhanced metastatic activity, chemical NAMPT inhibi- <br /> nicotinamide phosphoribosyltransferase (NAMPT). NAMPT                       tors previously suggested anticancer therapy (33). <br /> essential utilization recycling nicotinamide (NAM)              To investigate potential discrepancy, analyzed effects of <br /> for biosynthesis NAD+ (32). Interference NAMPT expres-                 FK866, noncompetitive inhibitor NAMPT (34), vitro and <br /> sion stable transduction shRNA (Supplemental                      vivo. In vitro, FK866 decreased cellular NAD+/NADH ratios in <br /> Figure 9A) reduced cellular NAD+/NADH ratios (Figure 5B), inde-                MDA-MB-435 MDA-MB-231 cells levels similar those <br /> pendently growth state tumor cells (Supplemental                 seen NAMPT knockdown (Supplemental Figure 11A), indica- <br /> Figure 10, A B) affecting cell proliferation vitro          tive effect drug NAD metabolism. Treatment of <br /> (Supplemental Figure 9B). In vivo, however, NAMPT knockdown                    NAMPT-knockdown cells FK866 reduced NAD+/NADH lev- <br /> significantly enhanced metastasis MDA-MB-435 MDA-                       els further, likely dose effect NAMPT inhibition (Sup- <br />  <br /> 1074	                         The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                   research article <br />  <br />  <br />  <br />  <br /> Figure 7 <br /> NAD+ precursor treatment primary breast tumor removal increases animal survival. (A) NAM treatment enhanced NAD+/NADH ratio <br /> in cultured 4T1 murine breast carcinoma cells. NAD+/NADH levels measured 2 days cell treatment 10 mM NAM complete <br /> medium. **P &lt; 0.01, unpaired 2-tailed Student&#226;&#8364;&#8482;s t test. n = 2 independent experiments. (B) Weight 4T1 mammary fat pad tumors untreated <br /> BALB/c mice. Tumors surgically removed volume reached 300 mm3. Data tumor weight distribution time surgery and <br /> randomization groups (n = 8), treatment initiated. Boxes denote interquartile range; lines boxes denote median; whiskers <br /> denote minima maxima. P = 0.7480, unpaired 2-tailed Student&#226;&#8364;&#8482;s t test. (C) Kaplan-Meier curves comparing survival NAM-treated and <br /> untreated BALB/c mice surgical removal 4T1 mammary fat pad tumors. Mice untreated treated 1% NAM drinking water <br /> after tumor removal (assigned day 0). n = 8 group. P = 0.0386, log-rank test. <br />  <br />  <br /> plemental Figure 11A). In vivo, FK866 did reduce metastatic activ-             independent cell model preferably seeds brain metas- <br /> ity 42-day observation period (Supplemental Figure 11B).                tases bloodstream. MDA-MB-453 cells, HER2+ cell <br /> This vivo effect related cytotoxic properties            line derived pericardial effusion breast cancer patient <br /> drug. While genetic interference tumor cell NAMPT expres-                 brain lesions (35), injected left cardiac ventricle. <br /> sion did affect cell growth cause cell death vitro (Supple-          Brain metastasis significantly reduced animals receiving <br /> mental Figure 9B), FK866 treatment induced S-G2/M cell cycle                   NAM treatment (Supplemental Figure 14, A B). <br /> arrest necrosis (Supplemental Figure 11, C D). The cyto-                 Next, sought analyze mechanisms involved anti- <br /> toxic effects FK866 enhanced NAMPT-knockdown cells.                 metastatic activity NAD+ precursor treatment. We that <br /> These results indicate drug, originally identified cell         NIC NAM increased p62 degradation, indicative autophagy <br /> proliferation inhibitor high-throughput screen, exerts effects            induction, decreasing AKT mTORC1 activities MDA- <br /> on cell viability. Concentrations FK866 reduced NAD+/                  MB-435 MDA-MB-231 cells (Figure 6D). Moreover, NAD+ <br /> NADH ratios tumor cells NAMPT knockdown                      modulation NIC NAM treatment significantly reduced <br /> did caused significant cell death vitro, genetic NAMPT              mtDNA content, indicative mitophagy, 3-day exposure <br /> interference did not. Thus, cytotoxic effects drug              changing mitochondrial distribution tumor <br /> not related NAMPT inhibition alone.                                      cells (Supplemental Figure 15). These results consistent with <br />   NAD+ precursor treatment inhibits metastasis spontaneous breast          observed regulation AKT/mTORC1 signaling autopha- <br /> cancer progression. Having demonstrated reduced nonlethal             gy enhancement NAD+/NADH ratios MDA-MB-435 <br /> NAD+/NADH ratios enhanced metastatic activity, reasoned                     MDA-MB-231 cells expression Ndi1 enhance mito- <br /> that increasing NAD+/NADH balance interfere                     chondrial complex I activity (Figure 3). <br /> metastatic progression. To test hypothesis, MDA-MB-435                      It reported SIRT1, NAD+-dependent deacetylase <br /> and MDA-MB-231 cells treated NAD+ precursors                     (36), sense increased NAD+/NADH levels induced NAM to <br /> nicotinic acid (NIC) NAM. NAM induce stronger increases                 inhibit mTORC1 activate autophagy (37). We inves- <br /> in NAD+ levels NIC, high concentrations, NAM                   tigated possible involvement SIRT1 NAM treatment&#226;&#8364;&#8220; <br /> not increases NAD+ synthesis, inhibit NAD+                   induced autophagy models. We targeted SIRT1 expres- <br /> consumption NAD+-dependent enzymes (33). In MDA-MB-231                      sion MDA-MB-435, MDA-MB-231, MDA-MB-453 cells <br /> and MDA-MB-435 cells, NAD+/NADH ratios significantly                      stable transduction shRNA (Supplemental Figure 16A). <br /> enhanced NIC NAM (Figure 6A), independent prolif-                 In Her2+ MDA-MB-453 cell line, SIRT1 knockdown blocked <br /> eration state tumor cells time treatment (Supple-             NAM-mediated inhibition mTORC1 activity induction of <br /> mental Figure 12, A B). Most importantly, affect-                autophagy. However, seen triple-negative MDA- <br /> ing cell growth vitro (Supplemental Figure 13A), treatment               MB-435 MDA-MB-231 cell lines (Supplemental Figure 16B). <br /> experimental animals NAD+ precursors given drinking                Thus, NAM treatment modulated mTORC1 autophagy <br /> water strongly reduced lung metastasis cell lines i.v.           tested cell lines, SIRT1 involvement clearly evident in <br /> injection (Figure 6, B C). Moreover, treatment clearly           cases, others, additional alternative <br /> interfered multiorgan metastasis MDA-MB-435 cells,                  mechanisms required. <br /> which brain lesions notably reduced (Supplemental                      Together results Ndi1 expression NAMPT <br /> Figure 13B). This striking antimetastatic effect confirmed                 knockdown, findings demonstrated specific modulation <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013	                   1075 <br />  research article <br />  <br />  <br />  <br />  <br /> 1076	              The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                   research article <br />  <br /> Figure 8                                                                       MT634 strain (38). The MMTV-PyMT model emulates important <br /> NAD+ precursor treatment inhibits spontaneous breast cancer pro-               aspects human breast cancer progression replicates distinct <br /> gression MMTV-PYMT mice. (A) NAM treatment (1% drinking                  advancing stages breast cancer pathology, making highly useful <br /> water experiment, beginning weaning) reduced mam- <br />                                                                                preclinical therapeutic trials (39). MT634 strain MMTV-PyMT <br /> mary tumor growth. Weights 10 mammary fat pads each <br /> treated mouse (n = 10), untreated control mice (n = 11), untreated <br />                                                                                mice received 1% NAM drinking water beginning wean- <br /> age- strain-matched PyMT-negative mice (Normal; n = 3). Boxes              ing (day 22) continuing experiment. Controls <br /> denote interquartile range; lines boxes denote median; whiskers         received water NAM. Tumor mass, histological grade, and <br /> denote minima maxima. ***P &lt; 0.001, nonparametric Mann-Whit-               lung metastasis examined day 80. Importantly, NAD+ pre- <br /> ney test. (B) Fat pads untreated versus NAM-treated PyMT mice,              cursor treatment strongly interfered oncogene-driven breast <br /> representative 10 fat pad locations. (C) NAM treatment inhibited        cancer progression MMTV-PyMT mouse model (Figure 8). <br /> PyMT-induced breast cancer progression. Percent area morphologi-            Mammary fat pad tumor mass significantly reduced NAM- <br /> cal stages 8 representative fat pads untreated (PyMT-Ctrl) or <br />                                                                                treated mice. In cases, mammary fat pads treated mice were <br /> NAM-treated (PyMT-NAM) mice. Averages group shown <br /> below. Scoring hyperplasia, adenoma, early advanced car-            normal minimal tumor involvement (Figure 8, A B, and <br /> cinoma performed whole-slide scans H&amp;E-stained tumor                 Supplemental Figure 17). Weight analysis individual fat pad <br /> sections morphometric measurements. (D) Representative micro-               mouse showed 6.8 &#194;&#177; 0.6 10 fat pads mouse in <br /> scopic fields 2 H&amp;E-stained sections 4 tumors untreated             control group (n = 11) larger largest fat pad in <br /> and NAM-treated PyMT mice. Scale bars: 200 &#206;&#188;m (top row                PyMT-negative mice strain age. This true for <br /> treatment group); 50 &#206;&#188;m (bottom row). (E) Quantification Western            0.6 &#194;&#177; 0.4 10 fat pads mouse NAM treatment group <br /> blot analyses mammary tumors control NAM-treated PyMT              (n = 10; Supplemental Figure 17). The reported weights include all <br /> mice (n = 8 tumors group). Shown relative protein abundance <br />                                                                                fat pads mouse, regardless tumor presence. Importantly, <br /> of p62, phospho-S6Ser240/244, phospho-4EBPThr37/46. Boxes denote <br /> interquartile range; lines boxes denote median; whiskers denote         histopathological analysis revealed NAM treatment drastically <br /> minima maxima. ***P &lt; 0.001, unpaired 2-tailed Student&#226;&#8364;&#8482;s t test.           inhibited breast cancer progression (Figure 8, C D, Supple- <br />                                                                                mental Figure 18). All untreated MMTV-PyMT mice tumors <br />                                                                                primarily containing adenoma, early advanced carcinoma, and <br /> of NAD+/NADH ratios, altering mitochondrial complex                  little normal hyperplastic tissue. In contrast, NAM-treated mice <br /> I activity, affecting NAD+ synthesis recovery pathways,           normal mammary fat pads; tumors, found, small and <br /> treating cells experimental animals NAD+ precursors,              consisted predominantly hyperplastic adenoma tissue. Fur- <br /> markedly change metastatic activity outcomes vivo.                      thermore, lung metastases detected 4 7 untreated <br />    To challenge concept NAD+ precursor treatment                  animals, metastases NAM-treated MMTV-PyMT <br /> interfere tumor growth metastatic activity, asked                  mice (0 6), assessed comprehensive histology serial lung <br /> whether therapeutic approach inhibit spontaneous                    sections (data shown). Importantly, NAM treatment induced <br /> metastasis prolong animal survival surgical removal             autophagy PyMT mammary tumors vivo, shown p62 <br /> primary breast tumor. To test hypothesis, used aggres-             degradation Western blot analyses individual tumors (Fig- <br /> sive metastatic, triple-negative breast cancer cell model 4T1. In              ure 8E Supplemental Figure 19). We did detect effects on <br /> vitro, addition NAM significantly enhanced NAD+/NADH ratios                 mTORC1 activity regulation, suggests NAM treatment <br /> in 4T1 cells (Figure 7A), indicative response NAD+ precursor           drinking water transiently modulate mTORC1 activity <br /> treatment. To analyze therapeutic effects NAM treatment              PyMT mammary tumors vivo. <br /> animal model, F-luc&#226;&#8364;&#8220;tagged 4T1 cells injected fourth                  To challenge concept NAD+ precursor treatment can <br /> mammary fat pad immunocompetent BALB/c mice, tumors                     interfere breast cancer progression, analyzed effects of <br /> were removed surgically reached volume 300 mm3.                 NAM treatment MMTV-PyMT mice estab- <br /> At time surgical removal, mice randomized con-                lished spontaneous mammary tumors. Mice received 1% NAM <br /> trol (untreated water) NAD+ precursor treatment (1% NAM                 drinking water, starting day 60 multiple pal- <br /> drinking water remainder experiment) groups (n = 8              pable tumors present animal. Controls received <br /> each), tumor weights measured (Figure 7B). At time                water NAM. Tumor masses examined day 80. <br /> of surgery NAM treatment, 1 animal group                Importantly, NAD+ precursor treatment significantly inhib- <br /> spontaneous metastasis lungs, measured noninvasive                ited growth established mammary tumors, after <br /> bioluminescence imaging. At 1 week tumor removal, 7 8                 delayed onset therapy (Supplemental Figure 20). <br /> control mice 5 8 NAM-treated mice detectable metas-                    Together, findings demonstrated NAD+ precursor <br /> tases. After 2 weeks, 8 8 controls 6 8 NAM-treated mice              treatment induce autophagy vivo effectively interfere <br /> showed metastasis. At end experiment (day 70 sur-             breast cancer progression stages oncogene-driven <br /> gery), 1 animal treatment group detectable recur-          breast cancer development. <br /> rence. Importantly, NAM treatment starting primary tumor <br /> surgery significantly increased animal survival model               Discussion <br /> highly aggressive metastatic breast cancer (Figure 7C).                        Our present findings establish tumor cell NAD+/NADH <br />    We asked therapeutic approach gener-                balance controlled mitochondrial complex I regulate breast <br /> ally interfere breast cancer progression. To test hypothesis,        cancer progression. We enhancement complex I <br /> we used MMTV-PyMT mouse model, polyoma middle                     activity expression NADH dehydrogenase Ndi1 from <br /> T (PyMT) oncogene expression mammary epithelium leads                   Saccharomyces cerevisiae strongly interfered tumor growth and <br /> to multifocal mammary tumor formation, detectable 30 days                metastasis, inhibition complex I knockdown the <br /> age, progression lung metastasis, detectable 80 days,         subunit NDUFV1 enhanced metastatic potential already <br />  <br /> 	                             The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013	                 1077 <br />  research article <br />  <br /> aggressive breast cancer cells. Mutations complex I report-           metastatic animal models. It possible FK866 induc- <br /> ed human breast tumors (6, 9&#226;&#8364;&#8220;12). mtDNA mutations se                   es cellular stress alternative mechanisms addition to <br /> are known cause transformation (40), contribute                depleting NAD+ levels NAMPT inhibition. <br /> to cancer progression, seen lung carcinoma EGFR muta-                Controversial effects reported ROS-inducing <br /> tions (41) oncocytic tumors loss complex I function            therapies. While cancer treatments induce high ROS levels <br /> (14, 42). Furthermore, oncogene activity (e.g., K-Ras transforma-             kill tumor cells, nonlethal increase ROS facilitate tumor <br /> tion) decrease mitochondrial complex I activity (43),               progression metastasis (7, 49). A similar concept applies to <br /> may support malignant phenotype (44). Our results revealed                  present findings, NDUFV1 knockdown caused com- <br /> a cause-and-effect relationship complex I function                plex I deficiency did drastically inhibit OXPHOS. While <br /> breast cancer progression.                                                    affecting cell viability, NDUFV1 knockdown significantly <br />   Enhancement complex I activity Ndi1 did alter                enhanced metastatic activity. Thus, noted strate- <br /> tumor cell proliferation vitro, clearly reduced tumor growth           gies aimed killing tumor cells interfering mitochondrial <br /> and metastasis vivo. Changing access nutrients oxygen               functions NAD+ synthesis could, effectively lethal, inad- <br /> within tumor microenvironment influences <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>cell survival</span>                vertently produce aggressive tumor cell phenotypes. <br /> proliferation. Under conditions, growth regulation              Thus, long run, approaches aimed normalizing mito- <br /> complex I critical. We mechanism                   chondrial functions, particularly complex I activity NAD+/ <br /> depended autophagy. Enhancement complex I Ndi1                  NADH redox levels, therapeutically effective and <br /> reduced metastasis induced p62 elimination, com-                  safer interfere normal cell function. <br /> plex I inhibition NDUFV1 knockdown enhanced metastasis                       Having established enhancement NAD+/NADH levels by <br /> and reduced p62 processing. The critical contribution autoph-              augmenting breast cancer cell complex I activity inhibited tumori- <br /> agy clear ATG5 knockdown, abrogated                     genicity metastasis, used new concept therapeutically <br /> antimetastatic effect Ndi1 breast cancer cell models.              hypothesized supplementing tumor cell nutrients with <br /> Consistent previous reports (45), basal levels             NAD+ precursors, NIC NAM, interfere breast <br /> of autophagy facilitated metastasis MDA-MB-231 cells;             cancer progression. NIC NAM used modulate NAD+ <br /> not seen MDA-MB-435 cells. Autophagy inhibit promote                levels showed therapeutic effects disorders, cerebral <br /> tumorigenesis supporting tumor cell survival metabolic               ischemia, diabetes, cardiovascular dysfunctions (32, 33). NIC <br /> stress (27&#226;&#8364;&#8220;29). Our results support dual role autophagy,           NAM protect neurons oxidative damage, sub- <br /> clearly showing autophagy required mitochondrial                 clinical vitamin B3 deficiency associated genomic instabil- <br /> complex I&#226;&#8364;&#8220;mediated reduction metastatic growth. One                 ity increased cancer risk (50). <br /> main regulators autophagy mTORC1, links tumor                       We demonstrated enhancing NAD+ levels NAD+ <br /> growth metabolism (46). We mitochondrial com-                  precursor treatment effectively inhibited experimental metas- <br /> plex I modulated mTORC1 upstream regulator, AKT,                      tasis human breast cancer cells xenograft models. Impor- <br /> in breast cancer cells, suggesting mTORC1 likely pathway                 tantly, treatment inhibited spontaneous metastasis and <br /> through complex I regulates autophagy.                                  increased animal survival therapy begun sur- <br />   Our results demonstrated mitochondrial complex I                   gical removal primary tumors. Furthermore, NAD+ precursor <br /> control breast cancer progression regulating cellular                  treatment strongly interfered oncogene-driven breast cancer <br /> NAD+/NADH balance. Ndi1 expression enhanced redox                        development progression transgenic MMTV-PyMT mice. <br /> balance, particularly metabolic stress, indicates               Our results provide evidence mechanism involves induc- <br /> involvement strong inhibition tumorigenicity metas-                 tion autophagy. NAD+ precursor treatment activated tumor cell <br /> tasis Ndi1. To consolidate role complex I con-                   autophagy vitro vivo included regulation of <br /> trolled NAD+/NADH ratio modulation breast cancer progres-                  mTORC1 activity, seen culture. It reported mod- <br /> sion, experimentally altered tumor cell NAD+/NADH ratios                   ulation NAD+/NADH ratios affect PI3K/AKT survival <br /> using 2 additional independent approaches. First, interfered               pathway inactivation PTEN (51). Furthermore, NAD+- <br /> with NAD+ biosynthesis recycling pathways knockdown                    dependent sirtuins modulate mTORC1 regulation of <br /> of NAMPT reduce NAD+/NADH balance. Second,                          TSC2 (52) LKB1 activity (53, 54). Sirtuins affect autoph- <br /> enhanced NAD+/NADH balance treatment NAD+ pre-                    agy controlling FOXO3 degradation (55) directly regulating <br /> cursors. Importantly, decreasing NAD+/NADH ratios knock-                   autophagy proteins ATG5, ATG7, ATG8 (56). We that <br /> down NAMPT, rate-limiting enzyme NAD+ synthesis                   modulation mTORC1 activity autophagy NAM involved <br /> pathway converts NAM NAD+, significantly enhanced                     SIRT1 activation MDA-MB-453 cells, likely depends on <br /> metastatic activity aggressive human breast cancer                 additional alternative mechanisms MDA-MB-435 MDA- <br /> cells. NAMPT knockdown enhanced AKT/mTORC1 activities                         MB-231 cells. NAD+-dependent sirtuins PARPs contribute to <br /> and inhibited autophagy. In apparent opposition, recent study               p53 function regulate chromatin structure genomic stabil- <br /> reported NAMPT inhibition reduced prostate cancer cell                   ity (32, 33, 50), playing emerging roles tumor progression <br /> viability (47). NAMPT suggested therapeutic                (57). Moreover, transcriptional corepressor CtBP responds to <br /> target inhibition progression breast cancer (48)             increased NADH hypoxia permit gene expression pro- <br /> malignancies (33). We chemical NAMPT inhibitor                 motes tumor cell migration (58). Importantly, BRCA1 tumor sup- <br /> FK866 induced cell death vitro tumoricidal effects              pressor expression inhibited low NAD+/NADH ratios (59, 60). <br /> vivo. In contrast, genetic interference NAMPT expression                The present study demonstrated mitochondrial complex I <br /> breast cancer cells did inhibit tumor cell viability and,          regulation tumor cell NAD+/NADH levels affects breast can- <br /> than killing tumor cells, rendered aggressive               cer growth metastasis translated new therapeutic <br />  <br /> 1078	                        The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                        research article <br />  <br /> approach preventing breast cancer progression. This highly                 defined regions using Living Image software. MMTV-PyMT <br /> relevant, current standard care cancer patients relies              mice (strain MT634; gift W.J. Muller, McGill University, Montreal, Quebec, <br /> primarily chemo- radiation therapies aimed killing                  Canada; ref. 38) heterozygous PyMT transgene genotyped <br /> tumor cells. Evolutionary models predict selective pressure                  PCR using forward primer 5&#226;&#8364;&#178;-CGGCGGAGCGAGGAACTGAGGAGAG-3&#226;&#8364;&#178; <br /> imposed approaches causes survival resistant clones                   reverse primer 5&#226;&#8364;&#178;-TCAGAAGACTCGGCAGTCTTAGGCG-3&#226;&#8364;&#178;. For NAD+ <br /> that eventually reactivate disease (61). Based central                 precursor treatment, mice received 1% NIC NAM drinking water, <br /> involvement metabolic tumor cell alterations cancer, thera-                 changed weekly. For histology, mammary fat pads lungs were <br /> peutic normalization tumor cell metabolism interfere                     zinc fixed, sectioned serially (5 &#206;&#188;m), H&amp;E stained. In PyMT model, <br /> with expansion residual breakthrough clones. Thus,                   scoring 4 morphological stages (hyperplasia, adenoma, early carcinoma, <br /> combination standard therapy NAD+ precursor treatment                     advanced carcinoma; ref. 39) performed whole-slide scans (Leica <br /> may halt breast cancer progression prevent relapse.                           SCN400 Digital Slide Scanner) image analysis morphometric mea- <br />                                                                                   surements (Metamorph imaging software; version 7.6). Select regions were <br /> Methods                                                                           imaged using Zeiss Axio Imager M1m microscope AxioVision software. <br /> Cell culture. MDA-MB-231, MDA-MB-435, MDA-MB-453, 4T1 cells                Lung metastases MMTV-PyMT mice quantified 10 H&amp;E-stained <br /> well variants stably transduced Firefly luciferase (F-luc)     step sections lung (5 &#206;&#188;m thick, 80 &#206;&#188;m apart). <br /> using lentiviral expression vector cFUW (ubiquitin promoter; provided                Ndi1 integration cellular respiration. 2 &#195;&#8212; 106 cells collected into <br /> by B.E. Torbett, The Scripps Research Institute) analyze tumor growth          complete medium, respiration measured 37&#194;&#176;C 2-ml cham- <br /> and metastasis noninvasive bioluminescence imaging (62). Cells            ber high-resolution respirometry using Oroboros Oxygraph series <br /> grown EMEM supplemented nonessential amino acids, vitamins,               D DatLab software (Oroboros Instruments). Routine mitochondrial <br /> 2 mM l-glutamine, 1 mM pyruvate, 10% FBS (complete medium). For               respiration, corrected residual oxygen consumption oxidative <br /> metabolic stress, cells cultured Neurobasal-A Medium (Gibco) sup-         reactions, measured intact cells complete medium. Func- <br /> plemented nonessential amino acids, vitamins, 2 mM l-glutamine,              tional integration Ndi1 tumor cell mitochondrial respiration was <br /> 5 mM 1 mM glucose, 2% dialyzed FBS normoxia (21% oxygen)             determined oxygen consumption addition rotenone (2.5 &#206;&#188;M) to <br /> hypoxia (&lt;1% oxygen; Modular Incubator Chamber; Billups-Rothenberg).              inhibit endogenous complex I final addition complex III inhibitor <br />    Ndi1 expression. Tumor cells stably transduced Ndi1 gene         antimycin A (2.5 &#206;&#188;M) terminate mitochondrial respiration. <br /> from S. cerevisiae (19) subcloned lentiviral expression vector cFUW.            Enzymatic analysis mitochondrial complex I activity. NADH oxidation to <br /> Transduced cells used pools experiments. Ndi1 expression           NAD+ complex I analyzed using immunocapture complex I <br /> was analyzed Western analysis, subcellular localization ana-           enzyme activity assay (MS-141; MitoSciences). <br /> lyzed immunofluorescence, described previously (18).                           NAD+/NADH analysis whole-cell mitochondrial extracts. NAD+ and <br />    NDUFV1, NAMPT, ATG5 knockdown. Lentiviral vectors contain-                 NADH analyzed independently extracts cells (1 &#195;&#8212; 106) or <br /> ing shRNA NDUFV1 (clone TRCN0000025872), ATG5 (clone                      isolated mitochondria (from 1 &#195;&#8212; 107 cells) prepared described previously <br /> TRCN0000151963), NAMPT (clone TRCN0000116180), nontargeting                    (18, 63). Concentrations determined using NAD/NADH fluores- <br /> control shRNA (SHC0002) Sigma-Aldrich. Knockdown effi-                  cence detection kit (Cell Technology Inc.). <br /> ciency quantified real-time PCR using FastStart Universal SYBR                mtDNA. mtDNA content determined quantitative real-time PCR <br /> Green Master (Rox) (Roche) following primers: ATG5 forward,               total DNA extracted 2 &#195;&#8212; 106 cells. PCR reactions performed in <br /> TGGGATTGCAAAATGACAGA; ATG5 reverse, TTCCCCATCTTCAG-                               quadruplicates using TaqMan probes specific MT-RNR1 (mitochondri- <br /> GATC AA; NAMPT ward, GCC AGC AGGGAATTTTGTTA;                                  al S12 RNA) (Hs02596859_g1, FAM label) referenced RPPH1 (RNase P, <br /> NAMPT reverse, TGATGTGCTGCTTCCAGTTC; NDUFV1 forward,                              VIC label), nuclear gene. PCR data recorded analyzed using the <br /> AAGGTTCCCTGAGTCGAGGT; NDUFV1 reverse, TTGTGAGGATCATG-                             ABI-PRISM 7700 Sequence Detection System (Applied Biosystems) and <br /> GCGTAA; GAPDH forward, GGGAAGGTGAAGGTCGGAGT; GAPDH                                Sequence Detector Software (version 2.0; SDS). <br /> reverse, TCCACTTTACCAGAGTTAAAAGCAG. Data recorded                           ATP measurements. ATP extracted cell pellets (2 &#195;&#8212; 106 cells) in <br /> analyzed using ABI-PRISM 7700 Sequence Detection System (Applied               triplicates 100 &#206;&#188;l ice-cold 2M perchloric acid. Extracts left on <br /> Biosystems) Sequence Detector Software (version 2.0; SDS).                    ice 15 minutes, mixed, centrifuged 12,000 g 5 minutes at <br />    Animal experiments. For experimental metastasis, female 6- 8-week-old       4&#194;&#176;C. The supernatant neutralized 35 &#206;&#188;l 1M Bicine 4M potas- <br /> C.B-17/SCID mice injected F-luc&#226;&#8364;&#8220;tagged cancer cells: 2.5 &#195;&#8212; 105 MDA-     sium carbonate pH 7.5 kept ice 15 minutes. After centrifuga- <br /> MB-231 MDA-MB-435 cells i.v., 5 &#195;&#8212; 105 MDA-MB-453 cells left        tion, ATP concentrations measured supernatant based ATP- <br /> cardiac ventricle stereotactic guidance. For primary tumor growth,          dependent luciferase activity using ATP determination kit (Invitrogen). <br /> 1 &#195;&#8212; 106 MDA-MB-231 cells 2.5 &#195;&#8212; 105 MDA-MB-435 cells injected            Lactate production. Lactate concentrations measured triplicates <br /> the axillary mammary fat pad. For spontaneous metastasis, female 8-week-          conditioned media culturing 105 cells 6-well plates (1.5 ml <br /> old BALB/c mice injected 1 &#195;&#8212; 105 F-luc&#226;&#8364;&#8220;tagged 4T1 cells        medium/well) 48 hours. 500 &#206;&#188;l medium deproteinized 166 &#206;&#188;l <br /> fourth mammary fat pad. Tumors removed surgically reached          ice-cold 2M perchloric acid, left ice 15 minutes, mixed, centri- <br /> 300 mm3 size, animals euthanized moribund determine           fuged 12,000 g 10 minutes 4&#194;&#176;C. Supernatants neutralized <br /> survival. Mice imaged weekly (IVIS 200; Xenogen) 10 minutes i.p.       58 &#206;&#188;l 1M Bicine 4M potassium carbonate pH 7 kept <br /> injection d-luciferin (100 mg/kg). To assess brain metastasis MDA-        ice 20 minutes. After centrifugation, lactate concentrations the <br /> MB-453 cells, mice imaged directly cardiac injection validate       supernatants determined fluorimetrically 340 nm excitation and <br /> correct routing verifying tumor cell signal brain region lack    466 nm emission (460 nm) 60 minutes addition lactate <br /> signal lungs. For ex vivo organ imaging end experimental     dehydrogenase, described previously (5). <br /> spontaneous metastasis studies, mice injected luciferin 5 minutes          Analysis mitochondrial membrane potential cell viability. The membrane <br /> before necropsy. For survival analyses, imaging validated metastasis      potential&#226;&#8364;&#8220;sensitive fluorochrome tetramethylrhodamine methyl ester <br /> the cause moribundity. Bioluminescence quantified photons/s/cm2         (TMRM; Invitrogen) used measure mitochondrial membrane poten- <br />  <br /> 	                                The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013	                            1079 <br />  research article <br />  <br /> tial. Cells incubated 150 nM TMRM 30 minutes 37&#194;&#176;C                       Statistics. Statistical comparisons 2 groups performed using <br /> the dark analyzed flow cytometry (FACSCalibur). Data                     unpaired 2-tailed Student&#226;&#8364;&#8482;s t tests unequal variance vitro results; <br /> 10,000 events sample collected analyzed measuring FL-2                    nonparametric Mann-Whitney tests vivo results, normal distri- <br /> fluorescence using CellQuest software. Cell viability analyzed mea-                bution assumed; Kaplan-Meier curves log-rank test <br /> suring percentage nonhypodiploid population (non&#226;&#8364;&#8220;sub-G0/G1)                    animal survival. Statistical calculations performed GraphPad <br /> by flow cytometry. Cells fixed ice-cold 70% ethanol, washed                  Prism software. A P value 0.05 considered significant. Results <br /> PBS, incubated 50 &#206;&#188;g/ml propidium iodide 100 &#206;&#188;g/ml                           shown mean &#194;&#177; SEM values obtained independent experiments. <br /> RNAse A 30 minutes 37&#194;&#176;C. Data 20,000 events sample                     Study approval. Animal studies approved Institutional Animal <br /> collected measuring FL-2 fluorescence.                                                 Care Use Committee The Scripps Research Institute. Animal work <br />    Immunohistochemistry. Tumor tissues paraformaldehyde zinc                      complied NIH institutional guidelines (The Scripps Research <br /> fixed, paraffin embedded, sectioned (5 &#206;&#188;m), transferred glass slides,                Institute AAALAC accredited). <br /> deparaffinized safe clear II, rehydrated series ethanol solu- <br /> tions. p62, Ki67, Ndi1 staining performed antigen retrieval                  Acknowledgments <br /> with 1 mM EDTA (pH 8), 10 mM citrate buffer, 1 mM EDTA plus                            This study supported NIH grants R01CA112287, <br /> 10 mM Tris-Cl (pH 8), respectively. Sections washed 3 times PBS,                  R01CA170737, R01CA170140 (to B. Felding-Habermann), <br /> treated 3% H2O2 PBS (or methanol Ndi1) 15 minutes,                     UL1RR025774 (to Eric J. Topol, Pilot award B. Felding-Haber- <br /> blocked 10% goat serum 0.3% Triton X-100 PBS 1 hour,                    mann), R01DK053244 (to T. Yagi); CDMRP DoD grant <br /> incubated primary antibody p62 (Santa Cruz Biotechnology                     W81XWH-08-0468 (to B. Felding-Habermann); California <br /> Inc.), Ki67 (BD Biosciences &#226;&#8364;&#8221; Pharmingen), Ndi1 (18) overnight (or                     Breast Cancer Research Program grants 17NB-0058, 16IB-0052, <br /> for 1 hour p62), followed incubation secondary horseradish                    12NB-0176, 13NB-0180 (to B. Felding-Habermann). A.F. <br /> peroxidase&#226;&#8364;&#8220;, alkaline phosphatase&#226;&#8364;&#8220;, biotin-conjugated Abs (Jackson                     Santidrian received postdoctoral fellowship Susan <br /> ImmunoResearch Laboratories Vector Laboratories) 2 hours.                         G. Komen Foundation. S.E. LeBoeuf L.J. Gay supported <br /> Signal biotinylated antibodies amplified using avidin-biotin                  trainee scholarships NIH grant 5TL1RR025772-03. We <br /> complex kit (Vector Laboratories). Horseradish peroxidase developed                   thank Bruce E. Torbet cFUW lentiviral construct; William J. <br /> with DAB substrate (BD Biosciences &#226;&#8364;&#8221; Pharmingen), nuclei                         Muller MMTV-PyMT mice; Rolf Habermann morphomet- <br /> stained Contrast Green (Kirkegaard &amp; Perry Laboratories). The slides                 ric analyses; Mihaela Lorger, Deirdre O&#226;&#8364;&#8482;Sullivan, Karin Staflin, <br /> were washed isopropanol briefly incubated SafeClear II (Fish-                 Jane Forsyth helpful discussions. We gratefully acknowl- <br /> er Scientific) mounting Permount (Fisher Scientific). Images                    edge donation Las Patronas (La Jolla, California, USA). This <br /> were acquired Zeiss Axio Imager M1m microscope equipped                     manuscript 21583 The Scripps Research Institute. <br /> digital camera, using &#195;&#8212;20 air objectives. Digital images analyzed with <br /> AxioVision 4.6 software (Zeiss).                                                          Received publication April 11, 2012, accepted revised <br />    Western blot analysis. Cells lysed Laemmli RIPA extraction                form January 3, 2013. <br /> buffer. Western blots incubated antibodies Ndi1 (20), <br /> p62 (Santa Cruz Biotechnology), phospho-AKT substrates (Ser/Thr),                         Address correspondence to: Brunhilde Felding-Habermann, <br /> phopho-S6Ser240/244, phospho-4E-BP1Thr37/46, LC3B (Cell Signaling Tech-                   Department Chemical Physiology Molecular Experi- <br /> nology), ATG5 (Cell Signaling Technology), &#206;&#178;-tubulin (Sigma-Aldrich),                     mental Medicine, The Scripps Research Institute, MEM-150, <br /> NDUFA9 (complex I), SDHA (complex II), UQCR2 (complex III), COXI                          La Jolla, California 92037, USA. Phone: 858.784.2021; Fax: <br /> (complex IV), ATP5A1 (complex V), mitochondrial porin (VDAC1)                          858.784.2174; E-mail: brunie@scripps.edu. <br /> (Mitosciencies). Antibody binding detected incubation sec- <br /> ondary antibodies conjugated IRDye 800, using Odyssey infrared                      Byoung B. Seo&#226;&#8364;&#8482;s present address is: Department Animal <br /> imaging (LI-COR Biosciences). Data analyzed quantified                    Resources, College Life Environmental Science, Daegu Uni- <br /> using Odyssey infrared imaging application software (version 3.0).                 versity, Jillyang, Gyeongsan, Gyeongbuk, Republic Korea. <br />  <br /> 	 1.	Dawood S, Broglio K, Buzdar AU, Hortobagyi GN,              DNA mutations regulate tumor cell metastasis.            drial DNA mutations renal oncocytoma. Clin <br />      Giordano SH. Prognosis women metastatic             Science. 2008;320(5876):661&#226;&#8364;&#8220;664.                             Cancer Res. 2008;14(8):2270&#226;&#8364;&#8220;2275. <br />      breast cancer HER2 status trastuzumab          	 8.	Wallace DC. A mitochondrial paradigm meta-           	15.	Chambers AF. MDA-MB-435 M14 cell lines: <br />      treatment: institutional-based review. J Clin            bolic degenerative diseases, aging, cancer:          identical M14 melanoma? Cancer Res. 2009; <br />      Oncol. 2010;28(1):92&#226;&#8364;&#8220;98.                                    dawn evolutionary medicine. Annu Rev Genet.            69(13):5292&#226;&#8364;&#8220;5293. <br /> 	 2.	Polyak K. Heterogeneity breast cancer. J Clin            2005;39:359&#226;&#8364;&#8220;407.                                       	 16.	Fendt L, et al. Accumulation mutations the <br />      Invest. 2011;121(10):3786&#226;&#8364;&#8220;3788.                       	 9.	Kulawiec M, Owens KM, Singh KK. Cancer cell mito-             entire mitochondrial genome breast cancer cells <br /> 	 3.	Warburg O. On respiratory impairment cancer              chondria confer apoptosis resistance promote             obtained tissue microdissection. Breast Cancer <br />      cells. Science. 1956;124(3215):269&#226;&#8364;&#8220;270.                     metastasis. Cancer Biol Ther. 2009;8(14):1378&#226;&#8364;&#8220;1385.          Res Treat. 2011;128(2):327&#226;&#8364;&#8220;336. <br /> 	 4.	Vander Heiden MG, Cantley LC, Thompson CB.            	10.	Ma Y, Bai RK, Trieu R, Wong LJ. Mitochondrial           	 17.	De VS, Van WR, Grivell LA, Marres CA. Primary <br />      Understanding Warburg effect: metabolic             dysfunction human breast cancer cells           structure import pathway rotenone- <br />      requirements cell proliferation. Science. 2009;          transmitochondrial cybrids. Biochim Biophys Acta.            insensitive NADH-ubiquinone oxidoreductase of <br />      324(5930):1029&#226;&#8364;&#8220;1033.                                        2010;1797(1):29&#226;&#8364;&#8220;37.                                          mitochondria Saccharomyces cerevisiae. Eur <br /> 	 5.	Schwartz JP, Passonneau JV, Johnson GS, Pastan        	11.	Shen L, et al. Evaluating mitochondrial DNA                J Biochem. 1992;203(3):587&#226;&#8364;&#8220;592. <br />      I. The effect growth conditions NAD+              patients breast cancer benign breast dis-     	 18.	Seo BB, Kitajima-Ihara T, Chan EK, Scheffler IE, <br />      NADH concentrations NAD+:NADH ratio                 ease. J Cancer Res Clin. Oncol. 2011;137(4):669&#226;&#8364;&#8220;675.         Matsuno-Yagi A, Yagi T. Molecular remedy com- <br />      normal transformed fibroblasts. J Biol Chem.   	 12.	Carew JS, Huang P. Mitochondrial defects cancer.          plex I defects: rotenone-insensitive internal NADH- <br />      1974;249(13):4138&#226;&#8364;&#8220;4143.                                     Mol Cancer. 2002;1:9.                                        quinone oxidoreductase Saccharomyces cerevi- <br /> 	 6.	Brandon M, Baldi P, Wallace DC. Mitochon-             	 13.	Parrella P, et al. Detection mitochondrial DNA            siae mitochondria restores NADH oxidase <br />      drial mutations cancer. Oncogene. 2006;                  mutations primary breast cancer fine-needle           activity complex I-deficient mammalian cells. <br />      25(34):4647&#226;&#8364;&#8220;4662.                                           aspirates. Cancer Res. 2001;61(20):7623&#226;&#8364;&#8220;7626.                Proc Natl Acad Sci U S A. 1998;95(16):9167&#226;&#8364;&#8220;9171. <br /> 	 7.	Ishikawa K, et al. ROS-generating mitochondrial       	 14.	Mayr JA, et al. Loss complex I mitochon-     	 19.	Seo BB, Wang J, Flotte TR, Yagi T, Matsuno-Yagi A. <br />  <br />  <br /> 1080	                                The Journal Clinical Investigation&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;http://www.jci.org&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Volume 123&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;Number 3&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;March 2013 <br />                                                                                                                                                            research article <br />  <br />       Use NADH-quinone oxidoreductase (NDI1)                 cific noncompetitive inhibitor nicotinamide                 stress response. Oncogene. 2011;30(8):907&#226;&#8364;&#8220;921. <br />       gene Saccharomyces cerevisiae possible cure           phosphoribosyltransferase, represents novel            	48.	Bajrami I, et al. Synthetic lethality PARP and <br />       complex I defects human cells. J Biol Chem.            mechanism induction tumor cell apoptosis.               NAMPT inhibition triple-negative breast cancer <br />       2000;275(48):37774&#226;&#8364;&#8220;37778.                                     Cancer Res. 2003;63(21):7436&#226;&#8364;&#8220;7442.                             cells. EMBO Mol Med. 2012;4(10):1087&#226;&#8364;&#8220;1096. <br /> 	20.	Marella M, Seo BB, Flotte TR, Matsuno-Yagi A,            	 35.	Brinkley BR, Beall PT, Wible LJ, Mace ML, Turner         	 49.	Schumacker PT. Reactive oxygen species cancer <br />       Yagi T. No immune responses expression                 DS, Cailleau RM. Variations cell form cyto-             cells: live sword, die sword. Cancer Cell. <br />       yeast Ndi1 protein rats. PLoS One. 2011;            skeleton human breast carcinoma cells vitro.             2006;10(3):175&#226;&#8364;&#8220;176. <br />       6(10):e25910.                                                 Cancer Res. 1980;40(9):3118&#226;&#8364;&#8220;3129.                        	 50.	Kirkland JB. Niacin requirements genomic sta- <br /> 	 21.	Barber-Singh J, Seo BB, Nakamaru-Ogiso E, Lau           	 36.	Chalkiadaki A, Guarente L. Sirtuins mediate mam-               bility. Mutat Res. 2012;733(1&#226;&#8364;&#8220;2):14&#226;&#8364;&#8220;20. <br />       YS, Matsuno-Yagi A, Yagi T. Neuroprotective effect            malian metabolic responses <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability.     	 51.	Pelicano H, et al. Mitochondrial respiration defects <br />       long-term NDI1 gene expression chronic                Nat Rev Endocrinol. 2012;8(5):287&#226;&#8364;&#8220;296.                         cancer cells cause activation Akt survival path- <br />       mouse model Parkinson disorder. Rejuvenation         	37.	Jang SY, Kang HT, Hwang ES. Nicotinamide-                       way redox-mediated mechanism. J Cell <br />       Res. 2009;12(4):259&#226;&#8364;&#8220;267.                                      induced mitophagy: event mediated high                      Biol. 2006;175(6):913&#226;&#8364;&#8220;923. <br /> 	 22.	Marella M, Seo BB, Thomas BB, Matsuno-Yagi A,                 NAD+/NADH ratio SIRT1 protein activation.            	 52.	Ghosh HS, McBurney M, Robbins PD. SIRT1 nega- <br />       Yagi T. Successful amelioration mitochondrial              J Biol Chem. 2012;287(23):19304&#226;&#8364;&#8220;19314.                         tively regulates mammalian target rapamycin. <br />       optic neuropathy using yeast NDI1 gene         	38.	Guy CT, Cardiff RD, Muller WJ. Induction                     PLoS One. 2010;5(2):e9199. <br />       rat animal model. PLoS One. 2010;5(7):e11472.                 mammary tumors expression polyoma-                 	53.	Pillai VB, et al. Exogenous NAD blocks cardiac <br /> 	 23.	Perry CN, Huang C, Liu W, Magee N, Carreira RS,               virus middle T oncogene: transgenic mouse                    hypertrophic response activation SIRT3- <br />       Gottlieb RA. Xenotransplantation mitochon-                 model metastatic disease. Mol Cell Biol. 1992;             LKB1-AMP-activated kinase pathway. J Biol Chem. <br />       drial electron transfer enzyme, Ndi1, myocardial           12(3):954&#226;&#8364;&#8220;961.                                                 2010;285(5):3133&#226;&#8364;&#8220;3144. <br />       reperfusion injury. PLoS One. 2011;6(2):e16288.         	39.	Lin EY, Nguyen AV, Russell RG, Pollard JW. Col-           	54.	Lan F, Cacicedo JM, Ruderman N, Ido Y. SIRT1 <br /> 	 24.	Sanz A, et al. Expression yeast NADH dehy-             ony-stimulating factor 1 promotes progression               modulation acetylation status, cytosolic <br />       drogenase Ndi1 Drosophila confers increased                mammary tumors malignancy. J Exp Med. 2001;                 localization, activity LKB1. Possible role <br />       lifespan independently dietary restriction. Proc           193(6):727&#226;&#8364;&#8220;740.                                                AMP-activated protein kinase activation. J Biol <br />       Natl Acad Sci U S A. 2010;107(20):9105&#226;&#8364;&#8220;9110.            	 40.	Yokota M, et al. Generation trans-mitochondri-              Chem. 2008;283(41):27628&#226;&#8364;&#8220;27635. <br /> 	25.	Sharma LK, Fang H, Liu J, Vartak R, Deng J, Bai                al mito-mice introduction pathogenic         	55.	Kume S, et al. Calorie restriction enhances cell <br />       Y. Mitochondrial respiratory complex I dysfunc-               G13997A mtDNA highly metastatic lung car-                 adaptation hypoxia Sirt1-dependent <br />       tion promotes tumorigenesis ROS altera-               cinoma cells. FEBS Lett. 2010;584(18):3943&#226;&#8364;&#8220;3948.               mitochondrial autophagy mouse aged kidney. <br />       tion AKT activation. Hum Mol Genet. 2011;           	 41.	Dasgupta S, et al. Mitochondrial DNA mutations                 J Clin Invest. 2010;120(4):1043&#226;&#8364;&#8220;1055. <br />       20(23):4605&#226;&#8364;&#8220;4616.                                             respiratory complex-I never-smoker lung can-       	 56.	Lee IH, et al. A role NAD-dependent deacet- <br /> 	 26.	Pagniez-Mammeri H, Loublier S, Legrand A, B&#195;&#169;nit               cer patients contribute lung cancer progression             ylase Sirt1 regulation autophagy. Proc Natl <br />       P, Rustin P, Slama A. Mitochondrial complex I                 associated EGFR <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>gene mutation</span>. J Cell                 Acad Sci U S A. 2008;105(9):3374&#226;&#8364;&#8220;3379. <br />       deficiency nuclear origin I. Structural genes. Mol         Physiol. 2012;227(6):2451&#226;&#8364;&#8220;2460.                          	 57.	Donmez G, Guarente L. Aging disease: connec- <br />       Genet Metab. 2012;105(2):163&#226;&#8364;&#8220;172.                       	 42.	Zimmermann FA, et al. Respiratory chain complex                tions sirtuins. Aging Cell. 2010;9(2):285&#226;&#8364;&#8220;290. <br /> 	 27.	Mathew R, et al. Autophagy suppresses tumori-                 I mitochondrial tumor suppressor oncocytic       	 58.	Zhang Q, Wang SY, Nottke AC, Rocheleau JV, Pis- <br />       genesis elimination p62. Cell. 2009;               tumors. Front Biosci (Elite Ed). 2011;3:315&#226;&#8364;&#8220;325.               ton DW, Goodman RH. Redox sensor CtBP medi- <br />       137(6):1062&#226;&#8364;&#8220;1075.                                       	43.	Baracca A, Chiaradonna F, Sgarbi G, Solaini G,                  ates hypoxia-induced tumor cell migration. Proc <br /> 	 28.	Mathew R, White E. Autophagy tumorigenesis                 Alberghina L, Lenaz G. Mitochondrial Complex I                 Natl Acad Sci U S A. 2006;103(24):9029&#226;&#8364;&#8220;9033. <br />       energy metabolism: friend day, foe night.           decrease responsible bioenergetic dysfunc-        	59.	Deng Y, et al. Redox-dependent Brca1 transcrip- <br />       Curr Opin Genet Dev. 2011;21(1):113&#226;&#8364;&#8220;119.                      tion K-ras transformed cells. Biochim Biophys Acta.         tional regulation NADH-sensor CtBP1. Onco- <br /> 	29.	Kenific CM, Thorburn A, Debnath J. Autophagy                   2010;1797(2):314&#226;&#8364;&#8220;323.                                          gene. 2010;29(50):6603&#226;&#8364;&#8220;6608. <br />       metastasis: double-edged sword. Curr        	44.	Hu Y, et al. K-ras(G12V) transformation leads          	 60.	Di LJ, Fernandez AG, De SA, Longo DL, Gardner <br />       Opin Cell Biol. 2010;22(2):241&#226;&#8364;&#8220;245.                           mitochondrial dysfunction metabolic switch               K. Transcriptional regulation BRCA1 expres- <br /> 	 30.	Klionsky DJ, et al. Guidelines use inter-         oxidative phosphorylation glycolysis. Cell             sion metabolic switch. Nat Struct Mol Biol. 2010; <br />       pretation assays monitoring autophagy               Res. 2012;22(2):399&#226;&#8364;&#8220;412.                                       17(12):1406&#226;&#8364;&#8220;1413. <br />       higher eukaryotes. Autophagy. 2008;4(2):151&#226;&#8364;&#8220;175.        	45.	Tu YF, Kaipparettu BA, Ma Y, Wong LJ. Mito-               	 61.	Merlo LM, Pepper JW, Reid BJ, Maley CC. Cancer <br /> 	31.	Klionsky DJ, Emr SD. Autophagy regulated                  chondria highly metastatic breast cancer cell               evolutionary ecological process. Nat Rev <br />       pathway cellular degradation. Science. 2000;               line MDA-MB-231 exhibits increased autopha-                    Cancer. 2006;6(12):924&#226;&#8364;&#8220;935. <br />       290(5497):1717&#226;&#8364;&#8220;1721.                                          gic properties. Biochim Biophys Acta. 2011;              	62.	Chen EI, et al. Adaptation energy metabolism <br /> 	 32.	Koch-Nolte F, Fischer S, Haag F, Ziegler M. Com-              1807(9):1125&#226;&#8364;&#8220;1132.                                             breast cancer brain metastases. Cancer Res. 2007; <br />       partmentation NAD +-dependent signalling.            	46.	Zoncu R, Efeyan A, Sabatini DM. mTOR:                      67(4):1472&#226;&#8364;&#8220;1486. <br />       FEBS Lett. 2011;585(11):1651&#226;&#8364;&#8220;1656.   <br /> </body></html>