<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Neurosci. 2012 September 5; 32(36): 12630&#226;&#8364;&#8220;12640. doi:10.1523/JNEUROSCI.0277-12.2012. <br />  <br />  <br />  <br />                            Integration &#206;&#178;-catenin, Sirtuin FOXO Signaling Protects <br />                            Mutant Huntingtin Toxicity <br />                            J. Alex Parker1,2,&#226;&#8364;&#160;,#, Rafael Vazquez-Manrique1,2,#, Cendrine Tourette1,2,&#226;&#8212;&#352;, Francesca <br />                            Farina1,2, Nicolas Offner1,2, Arnab Mukhopadhyay3,&#226;&#8364;&#161;, Anne-Marie Orfila1,2, Aur&#195;&#169;lie <br />                            Darbois1,2, Sophie Menet1,2, Heidi A. Tissenbaum3, Christian Neri1,2,4,* <br />                            1INSERM, <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> 894, Laboratory Neuronal Cell Biology Pathology, 75014 Paris, France <br />  <br />                            2University   Paris Descartes, EA 4059, 75014 Paris, France <br />                            3Programin Gene Function Expression, Program Molecular Medicine, University of <br />                            Massachusetts Medical School, Worcester MA 01605, USA <br />                            4AP-HP,    Department Neurology, Henri Mondor Hospital, 94000 Cr&#195;&#169;teil, France <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  One current challenges neurodegenerative disease research determine whether <br />                                  signaling pathways essential cellular homeostasis contribute neuronal <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>survival</span> <br />                                  modulate pathogenic process human disease. In C. elegans, sir-2.1/SIRT1 <br />                                  overexpression protects neurons early phases expanded polyglutamine (polyQ) <br />                                  toxicity, protection requires lonfity-promoting factor daf-16/FOXO. Here, show <br />                                  neuroprotective effect requires DAF-16/FOXO partner bar-1/&#206;&#178;-catenin putative <br />                                  DAF-16 regulated gene, ucp-4, sole mitochondrial uncoupling protein (UCP) nematodes. <br />                                  These results fit previously-proposed mechanism &#206;&#178;-catenin, FOXO SIRT1 <br />                                  proteins regulate gene expression <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>cell survival</span>. Knockdown &#206;&#178;-catenin <br />                                  enhanced vulnerability cell death mutant-huntingtin striatal cells derived the <br />                                  HdhQ111 knock-in mice. In addition, effect compensated SIRT1 overexpression and <br />                                  accompanied modulation neuronal UCP expression levels, highlighting cross- <br />                                  talk &#206;&#178;-catenin SIRT1 modulation mutant polyQ cytoxicity. Taken together, <br />                                  results suggest integration &#206;&#178;-catenin, sirtuin FOXO signaling protects the <br />                                  early phases mutant huntingtin toxicity. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Introduction <br />                                             Huntington&#226;&#8364;&#8482;s Disease (HD) dominantly-inherited CAG repeat disorder expanded <br />                                             polyglutamine (polyQ) tracts huntingtin (htt), causing striatal cortical degeneration <br />                                             (Walker, 2007). While person carrying 39 CAG repeats invariably develop <br />                                             HD, age-at-onset HD varies considerably. The CAG expansions account about <br />                                             60% variability age-at-onset HD, residual variability be <br />                                             heritable suggesting genes influence age-at-onset (Li et al., 2003; Gayan et al., <br />                                             2008; Metzger et al., 2008; Taherzadeh-Fard et al., 2009; Xu et al., 2009). Given HD is <br />                                             age-related disorder, candidate gene approaches based longevity modulators have <br />  <br />  <br />                            * <br />                             Correspondence addressed Christian Neri, Inserm <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> 894, 2-ter rue d&#226;&#8364;&#8482;Alesia, 75014 Paris, France. <br />                            christian.neri@inserm.fr. <br />                            &#226;&#8364;&#160;Current address: CRCHUM, D&#195;&#169;partement pathologie et biologie cellulaire, Universit&#195;&#169; Montr&#195;&#169;al, Montr&#195;&#169;al, Canada <br />                            &#226;&#8212;&#352;Current address: Buck Institute, Novato, CA 94945 <br />                            &#226;&#8364;&#161;Current address: Molecular Aging Laboratory, National Institute Immunology, New Delhi 110067, India <br />                            #These authors contributed equally work. <br />                             Parker et al.                                                                                             Page 2 <br />  <br />  <br />                                            examined identify disease modifiers. Studies TCERG1, evolutionary- <br />                                            conserved transcriptional regulator modulates longevity C. elegans (Ghazi et al., <br />                                            2009), showed protein neuroprotective models HD (Arango et al., 2006) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            polymorphic Gln-Ala repeat modifies HD American patients (Holbert et al., <br />                                            2001). Another longevity modulator neuroprotective activities transcription factor <br />                                            daf-16/FOXO. FOXO proteins essential stress resistance (Brunet et al., 2004), and <br />                                            daf-16 required neuroprotection increased dosage sirtuin sir-2.1/SIRT1 in <br />                                            expanded-polyQ nematodes (Parker et al., 2005). Several cofactors regulate FOXO activity, <br />                                            FOXOs targets (Greer Brunet, 2008; Landis Murphy, 2010; Yen et <br />                                            al., 2011), suggesting network genes centered FOXO regulate mutant <br />                                            polyglutamine neuron survival role HD variability. Interestingly regard, <br />                                            FOXO activity conserved C. elegans humans human FOXO3A was <br />                                            associated ability long-lived populations centenarians (Willcox et <br />                                            al., 2008; Anselmi et al., 2009; Li et al., 2009; Soerensen et al., 2010). Here, investigated <br />                                            neuroprotective role Wnt effector BAR-1/&#206;&#178;-catenin, protein binds <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                            DAF-16/FOXO3a regulate transcriptional activity response oxidative stress in <br />                                            C. elegans mammalian cells (Essers et al., 2005), ucp-4, sole <br />                                            mitochondrial uncoupling protein (UCP) C. elegans putative DAF-16 transcriptional <br />                                            target. To end, used C. elegans transgenics expressing exon-1 like htt touch <br />                                            receptor neurons (Parker et al., 2005). Animals expressing expanded polyQs strong <br />                                            neuronal dysfunction <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span> (defective response touch) absence cell death <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (Parker et al., 2005), recapitulating early phase mutant htt neurotoxicity. Additionally, <br />                                            tested effects SIRT1, &#206;&#178;-catenin neuronal UCPs (UCP2, UCP4) the <br />                                            survival striatal cells derived htt knock-in mice (Trettel et al., 2000). As detailed <br />                                            below, results suggest integration &#206;&#178;-catenin, sirtuin FOXO signaling protects <br />                                            early phases mutant htt toxicity. <br />  <br />                                Materials Methods <br />                                Nematode experiments <br />                                            The wild-type strain C. elegans used Bristol N2. Standard methods culturing and <br />                                            handling worms hermaphrodites males used. All strains scored 20&#194;&#176;C. <br />                                            Touch tests, scoring PLM cell processes, drug response assays quantitative Real- <br />                                            Time PCR performed previously described (Parker et al., 2005). Western analysis <br />                                            performed using standard protocols htt fusion proteins detected GFP <br />                                            antibody ab6556 (Abcam). Mutations transgenes used study listed Table 1. <br />                                            All strains obtained C. elegans Genetics Center (University Minnesota, <br />                                            Minneapolis) generated laboratory. For strain construction with <br />                                            polyQ transgenes, mutants verified visible phenotypes, PCR analysis deletion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutants, sequencing point mutations combination thereof. Deletion mutants were <br />                                            outcrossed minimum times wild-type geIs3[sir-2.1(+)] strain outcrossed <br />                                            times wild-type prior use. The geIs3[sir-2.1(+)] animals tested dye filling <br />                                            defect previously described (Burnett et al., 2011) showed defect regard. <br />  <br />                                            To test rescue effets sir-2.1 LOF 128Q animals, constructs encoding SIR-2.1 <br />                                            generated follows. We assembled sir-2.1 cDNA bicistronic GFP <br />                                            (biGFP) PCR fusion. We obtained sir-2.1 cDNA wildtype animals RT-PCR, <br />                                            using RV197 (5&apos;- <br />                                            GGGGACAACTTTGTATACAAAAGTTGATGTCACGTGATAGTGGCAAC) and <br />                                            RV198 (5&#226;&#8364;&#178;-GTGAAAGTAGGATGAGACAGCTCAGATACGCATTTCTTCAC) primers. <br />                                            RV198 contains sequence complementary 5&apos; region biGFP. We amplified biGFP <br />                                            pAN51 using RV192 (5&#226;&#8364;&#178;-GCTGTCTCATCCTACTTTCAC) RV178 (5&#226;&#8364;&#178;- <br />                                            GGGGACCACTTTGTA <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                           Page 3 <br />  <br />  <br />                                            CAAGAAAGCTGGGTATTATAGTTCATCCATGCCATGTGTA). Then, fused both <br />                                            PCR products nested PCR using primers RV197 RV178. These primers contain <br />                                            respectively attB5 attB2 sequences recombination pDONR221-P5-P2 vector, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            using Gateway (Invitrogen). In parallel, produced clone, pDONR221-P1- <br />                                            P5r, containing promoter mec-3, mec-3p, using primers RV3 (5&apos;- <br />                                            GGGGACAAGTTTGTACAAAAAAGC AGGCTCCTGCAGGTACCCGGAGTAGTTG) <br />                                            RV4 (5&#226;&#8364;&#178;-GGGGACAACTTTTGTATA <br />                                            CAAAGTTGTGGCGCGCCAATGCGCGAAATTGTGGCTACTC). Both clones used <br />                                            assemble mec-3p sir-2.1-biGFP, using Gateway technology, destination vector <br />                                            pDEST-AN, suitable C. elegans transgenesis. <br />  <br />                                            To test rescue effets bar-1 LOF 128Q animals, constructs encoding BAR-1 <br />                                            generated follows. We assembled bar-1 cDNA bicistronic mCherry (bi- <br />                                            mCherry) PCR fusion. We obtained bar-1 cDNA wildtype animals RT-PCR, <br />                                            using RV229 (5&#226;&#8364;&#8482;-GGGGACAACTTTGTATACAAAAGTTGATGGAC <br />                                            CTGGATCCGAACCTAG) RV230 (5&#226;&#8364;&#8482;- <br />                                            GTGAAAGTAGGATGAGACAGCTTAAAATCGACTATTCCTAGAAG) primers. <br />                                            RV230 contains sequence complementary 5&apos; region bimCherry. We amplified bi- <br />                                            mCherry pGEM-T-bicistronic-mCherry construct using RV192 (5&apos;- <br />                                            GCTGTCTCATCCTACTTTCAC) universal primer SP6. Then, fused PCR <br />                                            products nested PCR using primers RV229 RV8 (5&apos;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            GGGGACCACTTTGTACAAGAAAGCTGGGTATTATTTGTATAGTTCATCCATGCC <br />                                            ACC). These primers contain respectively attB5 attB2 sequences recombination in <br />                                            pDONR221-P5-P2 vector, using Gateway (Invitrogen). We combined the <br />                                            resulting construct plasmid pDONR221-P1-P5r (which contains mec-3p) to <br />                                            generate bar-1 overexpression construct, means Gateway technology, the <br />                                            destination vector pDEST-AN, suitable C. elegans transgenesis. <br />  <br />                                            Constructs mCherry expression control UCP-4 promoter (1768 bp) were <br />                                            generated follows. We fused promoter UCP-4 (<span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> position 1768 bp upstream <br />                                            ATG ucp-4) ucp-4 gene mCherry&#226;&#710;&#183;unc-54Terminator. Primers used to <br />                                            amplify ucp-4 Forward 5&#226;&#8364;&#8482;-TTTTGCGTTTGCTCGTCGCAC Reverse 5&#226;&#8364;&#8482;- <br />                                            AGTCGACCTGCAGGCATGCAAGCT. We amplified mCherry plasmid generated <br />                                            replacing GFP pPD95.75 mCherry using primers Forward 5&#226;&#8364;&#8482;- <br />                                            AGCTTGCATGCCTGCAGGTCGACT Reverse 5&#226;&#8364;&#8482; <br />                                            GGAAACAGTTATGTTTGGTATATTGGG. Constructs encoding mCherry the <br />                                            control ucp-4 promoter (1768 bp), wild-type scrambled binding site 2, <br />                                            generated follows. We amplified promoter ucp-4 (from position 1768 bp <br />                                            upstream ATG ucp-4) cloned PCR fragments pRV10 (a derivative of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            pPD95.75, GFP substituted mCherry). Primers used amplify the <br />                                            ucp-4 promoter sequence Forward 5&#226;&#8364;&#8482;- <br />                                            GGGGAAGCTTTTTTGCGTTTGCTCGTCGCAC Reverse 5&#226;&#8364;&#8482;-GGGG <br />                                            CCCGGGCATCTGAATAAAAGTTATAATTCC. Primers used mutagenize DAF-16 <br />                                            binding site 2 Forward 5&#226;&#8364;&#8482;- <br />                                            AAATACATGTTTGATTCTTTTTGAATACAGCTGTGACGTAA Reverse 5&#226;&#8364;&#8482;- <br />                                            GCTGTATTCAAAAAGAATCAAACATGTATTTAAATACATTTG (the scrambled <br />                                            sequence underligned). All constructs verified sequence integrity. <br />  <br />                                            For transgenesis, 5&#226;&#8364;&#8220;20 ng/&#194;&#181;l DNA constructs injected 19Q 128Q <br />                                            nematodes, pPD118.33 (a plasmid containing myo-2p&#226;&#710;&#183;GFP), final total <br />                                            DNA concentration 100&#226;&#8364;&#8220;150 ng/&#194;&#181;l, using standard methods. We isolated two <br />                                            independent strains construct perform touch tests. Strains stably expressing <br />                                            UCP-4-mCherry reporter crossed wild-type N2 strain strain <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                           Page 4 <br />  <br />  <br />                                            overexpressing translational construct encoding DAF-16&#226;&#710;&#183;GFP (TJ356, insertion <br />                                            zIs356IV) (Henderson Johnson, 2001), animals homozygous allele (wt/wt, <br />                                            daf-16&#226;&#710;&#183;GFP/daf-16&#226;&#710;&#183;GFP) isolated. Similar crosses perfomed using strains stably <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            expressing mCherry control ucp-4 promoter carried wild-type or <br />                                            scrambled DAF-16 binding site 2. The mCherry signals scored blindly using Zeiss <br />                                            fluorescence microscope (10x) quantification performed using ImageJ. <br />  <br />                                            Touch tests involved scoring response light touch tail using fine hair. <br />                                            Touch tests performed blindly scoring 10 touches tail young adult <br />                                            animal minimum 200 animals/genotype (50&#226;&#8364;&#8220;60 animals independent experiment <br />                                            total 4 independent experiments). Touch tests strains expressing <br />                                            extrachromosomal arrays involved minimum 60 animals/array (20&#226;&#8364;&#8220;30 animals per <br />                                            independent experiment total 3 independent experiments). Ordinarily, wild- <br />                                            type animals respond backing away touch. The responses recorded for <br />                                            animal that, example, 3 responses 10 tail given 30% <br />                                            responsiveness, mean values responsiveness retained comparison of <br />                                            nematode groups. Although nematodes habituated approximately 30 touch <br />                                            stimuli, restricting analysis 10 touch challenges ensures significant habituation <br />                                            occurs range (Giles Rankin, 2009). Axonal aggregation axonal swelling <br />                                            scored previously described (Parker et al., 2005) testing minimum 100 <br />                                            animals treatment. The primers used qRT-PCR analysis htt transgene expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Forward 5&apos;-CACTTGTCACTACTTTCTCAT Reverse 5&apos;- <br />                                            GTAGTTCCCGTCATCTTTG, primers Forward 5&apos;- <br />                                            CTTCTGCAAGGGATCTGCTAAG Reverse 5&apos;-GATTTGGAGCTGGGTTGACTTC <br />                                            used amplification housekeeping gene col-1 control. Proteins extracted <br />                                            animals previously described (Parker et al., 2005), separated SDS-PAGE <br />                                            analyzed Western blotting using mouse anti-htt (GenScript A00089, 1:500) and <br />                                            mouse anti-actin (MP Biomedicals, 1:10,000). Secondary antibodies goat anti-mouse <br />                                            IgG HRP-conjugated (Biorad). Proteins detected using ECL+ evaluated by <br />                                            densitometry using GS-800 densitometer (BIORAD) quantification performed <br />                                            using ImageJ. <br />  <br />                                            Chromatin immunoprecipitation (ChIP) performed described previously (Oh et al., <br />                                            2006; Mukhopadhyay et al., 2008) using DAF-16 antibody generated C- <br />                                            terminal portion DAF-16a2 (Oh et al., 2006) wild type (N2), daf-2(e1370), <br />                                            daf-16(mgDf50) daf-16(mgDf50);daf-2(e1370) worms. Binding ChIP samples was <br />                                            determined real-time PCR using following primers: binding site 1- <br />                                            tctgtgtcagcagcttttcg acggccgtccgtaatagata, binding site 2- tgagcaagttgtaatggggtta and <br />                                            catctggcacttatggggtta, 3&#226;&#8364;&#8482;-region- aaccaggagcaccattcaac gcaacttcttcctgctgacc. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Mammalian experiments <br />                                            Cell death assays HdhQ111 mouse striatal cells expressing full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> htt either <br />                                            wild-type (7Q/7Q) mutant (109Q/109Q) polyglutamines performed previously <br />                                            described, cell death induced 1-day serum deprivation (Arango et al., 2006). We <br />                                            used jetPEI transfection cDNA jetSI-ENDO transfection RNA <br />                                            molecules indicated manufacturer (PolyPlus Transfection). The siRNAs (si-htt, si- <br />                                            &#206;&#178;-catenin, si-SIRT1, si-SIRT2, si-SIRT3, si-UCP2, si-UCP4) scramble RNAs were <br />                                            obtained QIAGEN. Except htt siRNAs described (DiFiglia et <br />                                            al., 2007), mixes 3&#226;&#8364;&#8220;4 different siRNA sequences gene (25 33 nM) were <br />                                            systematically tested modulation cell survival siRNA target gene expression, <br />                                            followed evaluation individual siRNA sequences optimal concentration (25&#226;&#8364;&#8220;100 <br />                                            nM). Effects cell survival considered reliable 2 different siRNAs showed <br />                                            similar effects target mRNA/protein expression cell survival scramble RNAs <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                          Page 5 <br />  <br />  <br />                                            (25&#226;&#8364;&#8220;100 nM) did effect. The active siRNA sequences shown figures as <br />                                            follows: htt siRNA, 5&apos;-TTCATCAGCTTTTCCAGGGTC (100 nM), &#206;&#178;-catenin siRNA, 5&#226;&#8364;&#8482;- <br />                                            GGCTTTTCCCAGTCCTTCATT (100 nM); SIRT1 siRNA, 5&#226;&#8364;&#8482;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            GATTGTTATTAATATCCTTTT (25 nM); SIRT2 siRNA 5&apos;- <br />                                            TTGGGTGAAGTTCTATTTGTT (25 nM); SIRT3 siRNA, 5&apos;- <br />                                            TTACCGATCAACATGCTAGTT (25 nM); UCP2 siRNA, 5&#226;&#8364;&#8482;- <br />                                            GAAGAACGAGACACCTTTA (33 nM), UCP4 siRNA 5&apos;- <br />                                            CGCCATTTACAGACAGGTA (33 nM). The corresponding scramble RNAs it <br />                                            follows: htt, 5&apos;-GTCCAGATTTCCCTGTCGTAT (100 nM); &#206;&#178;-catenin, 5&#226;&#8364;&#8482;- <br />                                            GCTCTCGCCATATGAATCTCT (100 nM); SIRT1, 5&#226;&#8364;&#8482;-GTTTATGCTCATTATATAT (25 <br />                                            nM); SIRT2 5&apos;-GTTAGTTAGTTCGTGTAGT (25 nM); SIRT3, 5&#226;&#8364;&#8482;- <br />                                            GCTATGTATCAGCACACTA (25 nM); UCP2, 5&#226;&#8364;&#8482;-GCATAAGACCGACAATAGT (33 <br />                                            nM), UCP4, 5&apos;-GAGCCATCAACTGACTAGT (33 nM). The construct encoding an <br />                                            active variant SIRT1 (2 kb cDNA) lacking internal segment N-terminus was <br />                                            pCDNA3.1-sirt1-Flag. Cells subjected DAPI staining cell death scored 48 <br />                                            hours cell transfection counting picnotic versus normal nuclei DAPI- JetSI- <br />                                            ENDO-positive cells. Cytoplasmic nuclear proteins extracted previously <br />                                            described (Arango et al., 2006), separated SDS-PAGE analyzed Western blotting <br />                                            using following primary antibodies: mouse anti-SIRT1 (Upstate, 1:1000), mouse anti- <br />                                            SIRT2 (Santa Cruz, 1:100), rabbit anti-SIRT-3 (Abgent, 1:100), rabbit anti-&#206;&#178;-catenin (Cell <br />                                            Signaling, 1:5000), goat anti-UCP2 (Santa Cruz, dilution), rabbit anti-UCP4 (Abcam, 1:25), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mouse anti-htt (4C8, Chemicon, 1:5000 GenScript A00089, 1:500), mouse anti-actin <br />                                            (Invitrogen, 1:2000), rabbit anti-NCAM (Chemicon, 1:10,000). Secondary antibodies were <br />                                            goat anti-mouse IgG HRP-conjugated goat anti-rabbit IgG HRP-conjugated (Biorad). <br />                                            Proteins detected using ECL+ evaluated densitometry using GS-800 <br />                                            densitometer (BIORAD) quantification performed using ImageJ. Chemicals were <br />                                            purchased Calbiochem (BIO, LiCl). <br />  <br />                                RNA Isolation qRT-PCR striatal cells <br />                                            Quantitative RT-PCR performed RNA extracted 109Q/109Q mouse striatal <br />                                            cells, transfected appropriate siRNAs scramble RNAs, assess gene silencing <br />                                            efficiency. Two days cell transfection, RNA extracted Qiagen Rneasy kit <br />                                            DnaseI (Sigma) treatment (as manufacturers protocol). Single strand cDNA <br />                                            synthesis using oligoDT, random hexamer priming 100ng total RNA with <br />                                            Absolute 2-Step QRT-PCR SYBR ROX kit (AbGene). Quantitative PCR performed <br />                                            using SYBR&#194;&#174; Green ABI PRISM&#194;&#174; 7700 Sequence Detection System (Applied <br />                                            Biosystems) oligonucleotides follows: ucp2-f (5&#226;&#8364;&#8482;- <br />                                            TCCTGCTACCTCCCAGAAGA), ucp2-r (5&#226;&#8364;&#8482;-TGAGACCTCAAAGCAGCCTC), ucp4-f <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (5&#226;&#8364;&#8482;-GTGACGCCCGCCATTTACAG), ucp4-r (5&#226;&#8364;&#8482;-CATCCCTCCAATGACCGATTTCC), <br />                                            sirt2-f (5&#226;&#8364;&#8482;-CCCTTCGCCTCCCTCATC), sirt2-r (5&#226;&#8364;&#8482;-GTCCCTGTAAGCCTTCTTGG), <br />                                            sirt3-f (5&#226;&#8364;&#8482;-TGGCTGACTTCGCTTTGG), sirt3-r (5&#226;&#8364;&#8482;-TCCACACCATGAACTACATCC), <br />                                            rpl13a-f (5&#226;&#8364;&#8482;-CTGCTGCTCTCAAGGTTGTTC), rpl13a-r (5&#226;&#8364;&#8482;- <br />                                            CTGCTTCTTCTTCCGATAGTGC), hprt-f (5&#226;&#8364;&#8482;-TTTGCCGCGAGCCG), hprt-r (5&#226;&#8364;&#8482;- <br />                                            TAACCTGGTTCATCATCGCTAATC). Assays data analysis performed <br />                                            according manufacturer&apos;s protocol (User Bulletin #2, ABI PRISM&#194;&#174; 7700 Sequence <br />                                            Detection System, Perkin Elmer). All samples run triplicate using rpl13a or <br />                                            hprt calibrator gene dilution 1/100 cDNA. The target, <br />                                            normalized endogenous reference (ucp2 ucp4) relative calibrator (rpl13a <br />                                            hprt) calculated using 2&#226;&#710;&#8217;&#206;&#8221;&#206;&#8221;CT method statistical significance determined <br />                                            using paired t tests. <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                              Page 6 <br />  <br />  <br />                                Statistics <br />                                             Statistics nematode data performed using one-way ANOVA, correction for <br />                                             multiple testing Tukey&#226;&#8364;&#8482;s Multiple Comparison Test. Data expressed mean&#194;&#177;SEM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             &gt; 60&#226;&#8364;&#8220;200 nematodes group depending experiments performed (see the <br />                                             legends figures). Student&#226;&#8364;&#8482;s t-tests used striatal cell data. Data expressed as <br />                                             mean&#194;&#177;SD &gt; 150 cells group. Student&#226;&#8364;&#8482;s t-test used gene protein <br />                                             expression data. All experiments repeated times. P &lt; 0.05 considered <br />                                             significant. <br />  <br />                                Results <br />                                &#206;&#178;-catenin ucp-4 required neuroprotection increased sir-2.1 dosage C. <br />                                elegans <br />                                             Nematodes expressing normal (19Q) polyQs touch receptor neurons moderate loss <br />                                             response touch, nematodes bearing expanded (128Q) polyQ strong <br />                                             loss response touch (85%). This touch phenotype accompanied axonal swelling <br />                                             and, importantly, occurs absence cell death (Parker et al., 2005), providing vivo <br />                                             model manipulate early phases expanded polyQ neurotoxicity. <br />  <br />                                             We previously reported sir-2.1/SIRT1-daf-16/FOXO pathway neuroprotective in <br />                                             expanded-polyQ nematodes (Parker et al., 2005). Importantly, showed the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             neuroprotective effects sir-2.1 overexpression animals unrelated background <br />                                             mutations produce dye-filling defects influence lifespan C. elegans (Burnett <br />                                             et al., 2011), sir-2.1 overexpressor strain used does dye-filling <br />                                             defects. Recent work C. elegans suggested DAF-16 gain specificity the <br />                                             regulation lifespan biological processes, DAF-16 interact different co- <br />                                             factors (Wolff et al., 2006; Landis Murphy, 2010; Yen et al., 2011). Therefore, to <br />                                             investigate regulation neuronal survival sir-2.1/SIRT1-daf-16/FOXO, we <br />                                             examined evolutionary conserved FOXO co-factor BAR-1/&#206;&#178;-catenin. BAR-1/&#206;&#178;- <br />                                             catenin directly interacts DAF-16 FOXO3a shown role <br />                                             oxidative stress signaling (Essers et al., 2005). Loss-of-function (LOF) bar-1 enhanced <br />                                             loss response touch 128Q nematodes effect detected 19Q nematodes <br />                                             (Fig. 1A) change detected 128Q transgene expression (Fig. 1C), indicating that <br />                                             bar-1 protects neuron dysfunction induced expanded polyQs. <br />  <br />                                             Next, examined putative DAF-16 target gene, ucp-4 expand daf-16 affect <br />                                             neuronal survival. The ucp-4 gene encodes sole mitochondrial uncoupling protein <br />                                             (UCP) C. elegans (Iser et al., 2005). We tested ucp-4 reasons. First, UCPs might <br />                                             transcriptional targets FOXO suggested bioinformatic survey DAF-16 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             binding sites C. elegans genome (Lee et al., 2003) transcriptome analysis of <br />                                             Foxo3a-null mouse neural stem cells (Paik et al., 2009). Second, UCPs thought be <br />                                             essential neuron survival neurodegenerative disease pathogenesis (Andrews et al., <br />                                             2005a). Deletion ucp-4 resulted enhanced neuronal dysfunction 128Q animals with <br />                                             effect 19Q animals (Fig. 1B) change detected 128Q transgene expression <br />                                             (Fig. 1C), indicating ucp-4 protects neuron dysfunction induced expanded <br />                                             polyQs. <br />  <br />                                             In 128Q nematodes, neuronal dysfunction intrisincly connected dosage sir-2.1 <br />                                             (Parker et al., 2005; Burnett et al., 2011). Neuron dysfunction aggravated sir-2.1 LOF <br />                                             (Parker et al., 2005), effect rescued overexpression wild-type sir-2.1 in <br />                                             mechanosensory neurons effect vector control, suggesting effect of <br />                                             sir-2.1 LOF cell-autonomous (Fig. 1D). The aggravation neuron dysfunction LOF of <br />                                             bar-1 128Q nematodes (Fig. 1A) cell-autonomous suggested rescue of <br />  <br />                                                 J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                              Page 7 <br />  <br />  <br />                                            effect overexpression wild-type bar-1 mechanosensory neurons effect <br />                                            vector control (Fig. 1D). In contrast aggravation sir-2.1 LOF, increased <br />                                            sir-2.1 dosage neuroprotective requires daf-16 (Parker et al., 2005). Therefore we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            tested DAF-16 co-factor bar-1 putative DAF-16 target ucp-4 be <br />                                            required neuroprotective activity sir-2.1. The increase touch response induced <br />                                            sir-2.1 overexpression lost 128Q animals bearing bar-1 ucp-4 LOF <br />                                            <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, detrimental effects bar-1 LOF ucp-4 LOF touch response were <br />                                            similar 128Q nematodes 128Q nematodes bearing increased sir-2.1 dosage (Fig. 1E). <br />                                            Together results indicated neuroprotective effect sir-2.1 addition to <br />                                            requiring daf-16 dependent bar-1 ucp-4. <br />  <br />                                The ucp-4 promoter regulated DAF-16 <br />                                            Having shown ucp-4 required sir-2.1 regulate expanded polyQ toxicity C. <br />                                            elegans neurons, tested ucp-4 gene, putative DAF-16 target (Lee et <br />                                            al., 2003; Paik et al., 2009), directly regulated DAF-16. We scanned promoter <br />                                            ucp-4 identified 3 consensus DAF-16 binding sites 5 kb upstream the <br />                                            translation start site (Fig. 2A). We isolated worm extracts mix staged <br />                                            populations wild-type (N2) animals mutant daf-2/IR-IGF1R daf-16/ <br />                                            FOXO both, performed Chromatin Immunoprecipitation (ChIP) anti- <br />                                            DAF-16 antibody test DAF-16 direct binding ucp-4 promoter (Oh et al., 2006; <br />                                            Mukhopadhyay et al., 2008). There statistically-significant difference binding <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            signals binding site 1 3&#226;&#8364;&#8482;-region daf-16 mutant animals, signifying DAF-16 <br />                                            independent recruitment (Fig. 2A). In contrast, binding signals strongly decreased for <br />                                            binding site 2 DAF-16 absent, signifying DAF-16 dependence (Fig. 2A). To test <br />                                            functional role binding site 2, generated C. elegans transgenic strains stably <br />                                            expressing mCherry control ucp-4 promoter (1768-bp fragment) which <br />                                            carried wild-type scrambled DAF-16 binding site 2, crossed strains a <br />                                            DAF-16 protein overexpression daf-16 <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>knockout</span> background. Firstly, constructs encoding <br />                                            UCP-4&#226;&#710;&#183;mCherry control ucp-4 promoter stably expressed wild- <br />                                            type N2 strain strain overexpressing DAF-16&#226;&#710;&#183;GFP (translational construct TJ356), and <br />                                            intensity mCherry signals quantified late L4 animals pharynx area, an <br />                                            area DAF-16 isoforms UCP-4 known expressed (Lee et al., 2001; Lin et <br />                                            al., 2001) using independent arrays genotype greater 60 animals/array (20&#226;&#8364;&#8220; <br />                                            30 animals independent experiment total 3 independent experiments). <br />                                            The expression mCherry increased (P &lt; 0.005) animals bearing DAF-16 <br />                                            overexpression (mean&#194;&#177;SEM: 40&#194;&#177;2.7, arbitray <span id='am-397' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>) compared wild-type (mean&#194;&#177;SEM: <br />                                            57&#194;&#177;2.8, arbitray <span id='am-398' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>) (see Fig. 2B). Secondly, mCherry expression required DAF-16 <br />                                            binding site 2 shown loss variation mCherry expression animals bearing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            daf-16 LOF DAF-16 overexpression ucp-4 promoter carried scrambled <br />                                            sequence binding site (Fig. 2C). Together, experiments suggested the <br />                                            ucp-4 promoter regulated DAF-16. <br />  <br />                                GSK-3&#206;&#178; inhibitors require bar-1, sir-2.1, daf-16 ucp-4 neuroprotection <br />                                            To expand mechanistic basis neuroprotection sir-2.1 connection with <br />                                            daf-16 bar-1 activities, sought look regulation &#206;&#178;-catenin, protein is <br />                                            Wnt signaling pathway primarily targeted degradation direct <br />                                            phosphorylation GSK-3&#206;&#178; kinase. Interestingly, previous studies simple model <br />                                            shown GSK-3&#206;&#178; inhibitors lithium protect expanded polyQs <br />                                            (Berger et al., 2005; Voisine et al., 2007), suggesting GSK-3&#206;&#178; inhibitors useful <br />                                            HD therapy. Therefore, tested sir-2.1, daf-16, bar-1 ucp-4 be <br />                                            required neuroprotection GSK-3&#206;&#178; inhibitors. We tested 6-bromoindirubin-3&apos;- <br />                                            oxime (BIO), selective inhibitor GSK-3&#206;&#178;, observed 100-30 &#194;&#181;M BIO <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                             Page 8 <br />  <br />  <br />                                            significantly reduced neuron dysfunction 128Q animals effect detected 19Q <br />                                            animals (Fig. 3A) change detected 128Q transgene expression (Fig. 3C). However, <br />                                            neuroprotection BIO lost 128Q nematodes mutant genes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            including sir-2.1, daf-16, bar-1 ucp-4 (Fig. 3A), indicating genes required for <br />                                            BIO activity. BIO reduced axonal swelling (Fig. 3B), reflect increased <br />                                            neuron health consistent BIO ability reduce loss touch response 128Q <br />                                            nematodes. We tested lithium chloride, drug GSK-3&#206;&#178; inhibition properties, <br />                                            observed neuroprotective effects similar BIO (data shown). While BIO <br />                                            lithium secondary targets GSK-3&#206;&#178;, genome-scale functional <br />                                            RNAi screen indicated gsk-3 inactivation neuroprotective expanded-polyQ <br />                                            nematodes (Lejeune et al., 2012), suggesting neuroprotective effects BIO and <br />                                            lithium involve GSK-3&#206;&#178; target. Together, results suggested the <br />                                            neuroprotective properties GSK-3&#206;&#178; inhibitors require sir-2.1, daf-16, bar-1 ucp-4, <br />                                            useful protecting HD neurons dysfunction induced mutant <br />                                            polyQs. <br />  <br />                                &#206;&#178;-catenin, SIRT1 overexpression UCPs modulate survival mouse striatal cells <br />                                            Having shown sir-2.1, bar-1 ucp-4 modulate neurotoxicity N-terminal htt <br />                                            fragment nematodes, tested mammalian counterparts effect <br />                                            full-length cellular model HD. To end, used striatal cells derived htt <br />                                            knock-in mice HdhQ111 (Trettel et al., 2000). Mutant htt (109Q/109Q) striatal cells a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            higher rate cell death induced serum deprivation compared wild-type (7Q/7Q) cells <br />                                            (Trettel et al., 2000), providing assay test cell vulnerability mutant htt <br />                                            expressed endogenous levels (Fig. 4A). To test implication htt cell death <br />                                            induced serum deprivation, subjected cells htt siRNA treatment using a <br />                                            previously described siRNA (DiFiglia et al., 2007) corresponding scramble RNA. <br />                                            Reducing htt levels 109Q/109Q cells strongly decreased cell mortality serum <br />                                            deprivation, change detected 7Q/7Q cells (Fig. 4B), suggesting significant <br />                                            percent cell mortality associated mutant htt expression. As previously reported <br />                                            (Gines et al., 2003), mutant htt cells showed increased &#206;&#178;-catenin levels (data shown) that <br />                                            correspond pro-survival response (Gines et al., 2003). To test role &#206;&#178;- <br />                                            catenin striatal cell survival, subjected cells &#206;&#178;-catenin siRNA treatment (see <br />                                            Materials Methods details siRNA experiments). Reducing &#206;&#178;-catenin levels <br />                                            decreased survival 109Q/109Q cells, effect accompanied reduction &#206;&#178;-catenin <br />                                            expression, change htt expression detected (Fig. 4C/D). Consistent the <br />                                            effect reducing &#206;&#178;-catenin levels, GSK-3&#206;&#178; inhibitor BIO enhanced survival of <br />                                            109Q/109Q cells (Fig. 4C) detectable change htt expression (Fig. 4D). This <br />                                            amelioration cell survival accompanied increased &#206;&#178;-catenin expression (Fig. 4D) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            and, consistently, lost 109Q/109Q cells incubated BIO &#206;&#178;-catenin <br />                                            siRNA (Fig. 4C). Therefore, mouse C. elegans models HD, &#206;&#178;- <br />                                            catenin neuroprotective effects altering dosage detrimental neuronal cell <br />                                            survival. <br />  <br />                                            Next, explored role SIRTs UCPs mouse striatal cell survival. Reducing <br />                                            SIRT1 levels showed effect cell survival (Fig. 4E). We tested reducing levels <br />                                            SIRT2 SIRT3, additional sirtuin family members, modulate striatal <br />                                            cell survival observed effect well. Therefore, lowering sirtuins did affect <br />                                            neuronal survival. Second, examined neuronal UCPs (UCP2, UCP4). UCP2 <br />                                            siRNAs decreased survival 109Q/109Q cells (Fig. 4E), change htt <br />                                            expression (Fig. 4F), reduced UCP2 mRNA levels (Fig. 4G). In contrast, UCP4 <br />                                            siRNAs increased survival 109Q/109Q cells (Fig. 4E), change htt <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                               Page 9 <br />  <br />  <br />                                            expression (Fig. 4F), reduced UCP4 mRNA levels (Fig. 4G), suggesting <br />                                            evolutionary diversity mouse UCP activity compared sole UCP C. elegans. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Next, tested SIRT1 &#206;&#178;-catenin cooperate modulation mouse <br />                                            striatal cell survival sir-2.1/SIRT1 bar-1/&#206;&#178;-catenin linked modulate neuron <br />                                            dysfunction 128Q nematodes. To end, overexpressed SIRT1 in <br />                                            combination &#206;&#178;-catenin siRNA. The overexpression SIRT1 slightly enhanced the <br />                                            survival 109Q/109Q cells subjected serum deprivation change htt levels <br />                                            (Fig. 5A/B). As previously observed, reducing &#206;&#178;-catenin levels strongly detrimental to <br />                                            109Q/109Q cell survival (Fig. 5A) effect suppressed SIRT1 overexpression, <br />                                            suggesting raising SIRT1 levels compensate detrimental effect &#206;&#178;-catenin <br />                                            reduction connecting SIRT1 &#206;&#178;-catenin pathways. We further <br />                                            examined individual effects SIRT1 &#206;&#178;-catenin mouse striatal cells. Given that <br />                                            UCP gene expression regulated FOXO proteins suggested previous studies <br />                                            (Lee et al., 2003; Paik et al., 2009), tested effects &#206;&#178;-catenin reduction and <br />                                            SIRT1 overexpression expression levels UCP2 UCP4. In mutant htt striatal <br />                                            cells, reducing &#206;&#178;-catenin, aggravates cell death vulnerability, increased mRNA levels <br />                                            UCP4 (detrimental cell survival) SIRT1 overexpression, slightly <br />                                            neuroprotective, opposite effect (Fig. 5C). Conversely, reducing &#206;&#178;-catenin decreased <br />                                            mRNA levels UCP2 (a neuroprotective gene) mutant htt cells, no <br />                                            significant effect SIRT1 overexpression (Fig. 5C). These results indicated while <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            SIRT1 &#206;&#178;-catenin common effects repression UCP4 levels, they <br />                                            differ ability promote UCP2 levels. <br />  <br />                                            Collectively, results consistent well-documented model the <br />                                            FOXO, SIRT1 &#206;&#178;-catenin proteins form complex nucleus regulate gene <br />                                            expression nematodes species (Brunet et al., 2004). Further, results <br />                                            highlighted cross-talk &#206;&#178;-catenin SIRT1 regulation vulnerability to <br />                                            cell death caused mutant htt expression, primarily mediated modulating <br />                                            levels UCP4, UCP detrimental mutant htt cell survival. <br />  <br />                                Discussion <br />                                            The Sir2/SIRT1 gene target daf-16/FOXO emerged potent protective factors <br />                                            HD degenerative diseases (Morley et al., 2002; Parker et al., 2005; Cohen et <br />                                            al., 2006; Kim et al., 2007; Jeong et al., 2011; Jiang et al., 2011). Since FOXO integrates <br />                                            numerous signals regulates expression genes (Greer Brunet, 2008; <br />                                            Landis Murphy, 2010; Yen et al., 2011), essential question FOXO <br />                                            network protects HD neurons. Does play role early phases mutant htt toxicity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            neuronal dysfunction vulnerability cell death? <br />  <br />                                            Our data &#206;&#178;-catenin, SIRT1 UCPs participate regulation of <br />                                            vulnerability cell death mutant htt striatal cells, consistent protective <br />                                            effect increased sir-2.1 dosage neuron dysfunction 128Q nematodes, effect <br />                                            requires daf-16/FOXO (Parker et al., 2005), FOXO cofactor bar-1/&#206;&#178;-catenin (Essers <br />                                            et al., 2005), ucp-4, suggest DAF-16 regulated gene. As summarized <br />                                            Figure 6, results consistent previously-proposed model the <br />                                            SIRT1, FOXO &#206;&#178;-catenin proteins form complex nucleus regulate gene <br />                                            expression nematodes species (Tissenbaum Guarente, 2001; Brunet et al., <br />                                            2004; Essers et al., 2005; Berdichevsky et al., 2006; Wang et al., 2006). Importantly, <br />                                            neuroprotective effects SIRT1 activation recently reported mouse models <br />                                            HD (Jeong et al., 2011; Jiang et al., 2011) FOXO3a acetylation involved in <br />                                            SIRT1 protection (Jiang et al., 2011), corroborates C. elegans data. In addition, <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                                           Page 10 <br />  <br />  <br />                                            neuroprotective effects &#206;&#178;-catenin consistently observed expanded-polyQ <br />                                            nematodes mutant htt striatal cells. However, observed differences C. <br />                                            elegans mouse striatal cells UCP activity. This expected evolutionary diversity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            gene family function widely-observed phenomenon. The UCPs profound <br />                                            effects neurons chronic uncoupling promoting mitochondrial biogenesis and <br />                                            elevated ATP levels (Andrews et al., 2005a). The enhancement 128Q toxicity in <br />                                            nematode ucp-4 mutants suggests ucp-4 normally protect neurons dysfunction <br />                                            induced expanded polyQ expression. However, mouse striatal cells express <br />                                            mutant htt, UCP4 normally contribute vulnerability, UCP2 protect <br />                                            it. This consistent neuroprotective effect UCP2 overexpression mouse <br />                                            models Parkinson&#226;&#8364;&#8482;s disease (Andrews et al., 2005b). While C. elegans mouse <br />                                            striatal cell model emphasized neuroprotective effects &#206;&#178;-catenin, showed <br />                                            differences ability account cross-talk Sir2/SIRT1 &#206;&#178;-catenin. <br />                                            Expanded-polyQ nematodes indicated bar-1/&#206;&#178;-catenin required sir-2.1/SIRT1 <br />                                            overexpression neuroprotective. However, mutant htt striatal cells poorly recapitulated <br />                                            neuroprotective effects SIRT1 overexpression primarily indicated SIRT1 <br />                                            overexpression compensates detrimental effect &#206;&#178;-catenin reduction. <br />  <br />                                            These results highlight complexity regulation FOXO factors (Greer and <br />                                            Brunet, 2008; Landis Murphy, 2010; Yen et al., 2011), illustrate the <br />                                            conclusions role cell survival genes influenced methodological <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cellular context activity assessed. For example, SIRT1 reported to <br />                                            deacetylate &#206;&#178;-catenin suppress ability activate transcription mammalian cells <br />                                            (Firestein et al., 2008), appears contrast findings. However, study was <br />                                            performed human colon cancer cell lines SIRT1-&#206;&#178;-catenin homeostasis be <br />                                            different compared neurons cells expressing expanded polyQs. In addition, several <br />                                            studies indicated &#206;&#178;-catenin important survival factor normal (Essers et al., <br />                                            2005) diseased (Inestrosa Arenas, 2010) cells ability co-activate either <br />                                            TCF FOXO transcription factors. However, study neuronal cell death HD has <br />                                            suggested abnormal accumulation &#206;&#178;-catenin cause neurotoxicity (Godin et al., <br />                                            2010). Since &#206;&#178;-catenin neuroprotective early-phases mutant polyQ <br />                                            cytotoxicity (neuron dysfunction, cell vulnerability), suggests &#206;&#178;-catenin homeostasis <br />                                            activity change pathogenic process HD. <br />  <br />                                            Finally, activation cell survival mechanisms controlled FOXO factors may <br />                                            contribute delaying HD pathogenesis. Our data suggest GSK-3&#206;&#178; inhibitors <br />                                            require FOXO network protecting neurons dysfunction induced by <br />                                            expanded polyQs, suggesting GSK-3&#206;&#178; potential treatment HD. In <br />                                            conclusion, data reveal neuroprotection sirtuin sir-2.1 expanded polyQs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            requires &#206;&#178;-catenin ucp-4 C. elegans, suggesting FOXO interactors modulate the <br />                                            early phases mutant polyQ cytotoxicity. Our findings raise possibility FOXO <br />                                            interactors role modification pathogenic process HD, this <br />                                            addressed future studies. <br />  <br />                                Acknowledgments <br />                                            We thank Cheryl Wellington Michael Hayden constructs encoding htt 128 Glns, Andrew Fire C. <br />                                            elegans plasmids, Leonard Guarente LG100 strain, Elena Cattaneo cells HdhQ111 mice, Robert E. <br />                                            Hughes constructs encoding SIRT1 Maryline Mage technical assistance. Some strains kindly <br />                                            provided Caenorhabditis Genetics Center, funded National Institutes Health National <br />                                            Center Research Resources. This work supported INSERM, AP-HP, ANR, FRM AFM, Paris, France, <br />                                            CHDI Foundation Hereditary Disease Foundation, USA. H.A.T. William Randolph Hearst <br />                                            Young Investigator. This project funded endowment William Randolph Hearst <br />                                            Foundation grant (AG025891) National Institute Aging H.A.T. J.A.P. supported an <br />                                            Inserm Young Researcher award. <br />  <br />  <br />                                                 J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                                 Page 11 <br />  <br />  <br />                                References <br />                                            Andrews ZB, Diano S, Horvath TL. Mitochondrial uncoupling proteins CNS: support of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               function survival. Nat Rev Neurosci. 2005a; 6:829&#226;&#8364;&#8220;840. [PubMed: 16224498] <br />                                            Andrews ZB, Horvath B, Barnstable CJ, Elsworth J, Yang L, Beal MF, Roth RH, Matthews RT, <br />                                               Horvath TL. Uncoupling protein-2 critical nigral dopamine cell survival mouse model of <br />                                               Parkinson&apos;s disease. J Neurosci. 2005b; 25:184&#226;&#8364;&#8220;191. [PubMed: 15634780] <br />                                            Anselmi CV, Malovini A, Roncarati R, Novelli V, Villa F, Condorelli G, Bellazzi R, Puca AA. <br />                                               Association FOXO3A locus extreme longevity southern Italian centenarian study. <br />                                               Rejuvenation Res. 2009; 12:95&#226;&#8364;&#8220;104. [PubMed: 19415983] <br />                                            Arango M, Holbert S, Zala D, Brouillet E, Pearson J, Regulier E, Thakur AK, Aebischer P, Wetzel R, <br />                                               Deglon N, Neri C. CA150 expression delays striatal cell death overexpression knock-in <br />                                               conditions mutant huntingtin neurotoxicity. J Neurosci. 2006; 26:4649&#226;&#8364;&#8220;4659. [PubMed: <br />                                               16641246] <br />                                            Berdichevsky A, Viswanathan M, Horvitz HR, Guarente L. C. elegans SIR-2.1 interacts 14-3-3 <br />                                               proteins activate DAF-16 extend life span. Cell. 2006; 125:1165&#226;&#8364;&#8220;1177. [PubMed: 16777605] <br />                                            Berger Z, Ttofi EK, Michel CH, Pasco M, Tenant S, Rubinsztein DC, O&apos;Kane CJ. Lithium rescues <br />                                               toxicity aggregate-prone proteins Drosophila perturbing Wnt pathway. Hum Mol Genet. <br />                                               2005 <br />                                            Brunet A, Sweeney LB, Sturgill JF, Chua KF, Greer PL, Lin Y, Tran H, Ross SE, Mostoslavsky R, <br />                                               Cohen HY, Hu LS, Cheng HL, Jedrychowski MP, Gygi SP, Sinclair DA, Alt FW, Greenberg ME. <br />                                               Stress-dependent regulation FOXO transcription factors SIRT1 deacetylase. Science. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               2004; 303:2011&#226;&#8364;&#8220;2015. [PubMed: 14976264] <br />                                            Burnett C, et al. Absence effects Sir2 overexpression lifespan C. elegans Drosophila. <br />                                               Nature. 2011; 477:482&#226;&#8364;&#8220;485. [PubMed: 21938067] <br />                                            Cohen, E.; Bieschke, J.; Perciavalle, RM.; Kelly, JW.; Dillin, A. Science. Vol. 313. New York, NY: <br />                                               2006. Opposing activities protect age-onset proteotoxicity; p. 1604-1610. <br />                                            DiFiglia M, Sena-Esteves M, Chase K, Sapp E, Pfister E, Sass M, Yoder J, Reeves P, Pandey RK, <br />                                                 Rajeev KG, Manoharan M, Sah DW, Zamore PD, Aronin N. Therapeutic silencing mutant <br />                                                 huntingtin siRNA attenuates striatal cortical neuropathology behavioral deficits. <br />                                                 Proceedings National Academy Sciences United States America. 2007; <br />                                                 104:17204&#226;&#8364;&#8220;17209. [PubMed: 17940007] <br />                                            Essers, MA.; Vries-Smits, LM.; Barker, N.; Polderman, PE.; Burgering, BM.; Korswagen, HC. <br />                                                 Science. Vol. 308. New York, NY: 2005. Functional interaction beta-catenin FOXO <br />                                                 oxidative stress signaling; p. 1181-1184. <br />                                            Firestein R, Blander G, Michan S, Oberdoerffer P, Ogino S, Campbell J, Bhimavarapu A, Luikenhuis <br />                                                 S, Cabo R, Fuchs C, Hahn WC, Guarente LP, Sinclair DA. The SIRT1 deacetylase suppresses <br />                                                 intestinal tumorigenesis colon cancer growth. PLoS One. 2008; 3:e2020. [PubMed: 18414679] <br />                                            Gayan J, Brocklebank D, Andresen JM, Alkorta-Aranburu G, Zameel Cader M, Roberts SA, Cherny <br />                                                 SS, Wexler NS, Cardon LR, Housman DE. Genomewide linkage scan reveals novel loci <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 modifying age onset Huntington&apos;s disease Venezuelan HD kindreds. Genet Epidemiol. <br />                                                 2008; 32:445&#226;&#8364;&#8220;453. [PubMed: 18481795] <br />                                            Ghazi A, Henis-Korenblit S, Kenyon C. A transcription elongation factor links signals the <br />                                                 reproductive lifespan extension Caenorhabditis elegans. PLoS Genet. 2009; <br />                                                 5:e1000639. [PubMed: 19749979] <br />                                            Giles AC, Rankin CH. Behavioral genetic characterization habituation using Caenorhabditis <br />                                                 elegans. Neurobiol Learn Mem. 2009; 92:139&#226;&#8364;&#8220;146. [PubMed: 18771741] <br />                                            Gines S, Ivanova E, Seong IS, Saura CA, MacDonald ME. Enhanced Akt signaling early pro- <br />                                                 survival response reflects N-methyl-D-aspartate receptor activation Huntington&apos;s disease <br />                                                 knock-in striatal cells. J Biol Chem. 2003; 278:50514&#226;&#8364;&#8220;50522. [PubMed: 14522959] <br />                                            Godin JD, Poizat G, Hickey MA, Maschat F, Humbert S. Mutant huntingtin-impaired degradation of <br />                                                 beta-catenin causes neurotoxicity Huntington&apos;s disease. The EMBO journal. 2010; 29:2433&#226;&#8364;&#8220; <br />                                                 2445. [PubMed: 20531388] <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                                   Page 12 <br />  <br />  <br />                                            Greer EL, Brunet A. Signaling networks aging. J Cell Sci. 2008; 121:407&#226;&#8364;&#8220;412. [PubMed: <br />                                                 18256383] <br />                                            Henderson ST, Johnson TE. daf-16 integrates developmental environmental inputs mediate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 aging nematode Caenorhabditis elegans. Curr Biol. 2001; 11:1975&#226;&#8364;&#8220;1980. [PubMed: <br />                                                 11747825] <br />                                            Holbert S, Denghien I, Kiechle T, Rosenblatt A, Wellington C, Hayden MR, Margolis RL, Ross CA, <br />                                                 Dausset J, Ferrante RJ, Neri C. The Gln-Ala repeat transcriptional activator CA150 interacts with <br />                                                 huntingtin: Neuropathologic genetic evidence role Huntington&apos;s disease pathogenesis. <br />                                                 Proceedings National Academy Sciences United States America. 2001; <br />                                                 98:1811&#226;&#8364;&#8220;1816. [PubMed: 11172033] <br />                                            Inestrosa NC, Arenas E. Emerging roles Wnts adult nervous system. Nat Rev Neurosci. 2010; <br />                                                 11:77&#226;&#8364;&#8220;86. [PubMed: 20010950] <br />                                            Iser WB, Kim D, Bachman E, Wolkow C. Examination requirement ucp-4, putative <br />                                                 homolog mammalian uncoupling proteins, stress tolerance longevity C. elegans. <br />                                                 Mech Ageing Dev. 2005; 126:1090&#226;&#8364;&#8220;1096. [PubMed: 15893362] <br />                                            Jeong H, Cohen DE, Cui L, Supinski A, Savas JN, Mazzulli JR, Yates JR 3rd, Bordone L, Guarente L, <br />                                                 Krainc D. Sirt1 mediates neuroprotection mutant huntingtin activation TORC1 and <br />                                                 CREB transcriptional pathway. Nat Med. 2011; 18:159&#226;&#8364;&#8220;165. [PubMed: 22179316] <br />                                            Jiang M, et al. Neuroprotective role Sirt1 mammalian models Huntington&apos;s disease through <br />                                                 activation multiple Sirt1 targets. Nat Med. 2011; 18:153&#226;&#8364;&#8220;158. [PubMed: 22179319] <br />                                            Kim D, Nguyen MD, Dobbin MM, Fischer A, Sananbenesi F, Rodgers JT, Delalle I, Baur JA, Sui G, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Armour SM, Puigserver P, Sinclair DA, Tsai LH. SIRT1 deacetylase protects against <br />                                                 neurodegeneration models Alzheimer&apos;s disease amyotrophic lateral sclerosis. The <br />                                                 EMBO journal. 2007; 26:3169&#226;&#8364;&#8220;3179. [PubMed: 17581637] <br />                                            Landis JN, Murphy CT. Integration diverse inputs regulation Caenorhabditis elegans <br />                                                 DAF-16/FOXO. Dev Dyn. 2010; 239:1405&#226;&#8364;&#8220;1412. [PubMed: 20140911] <br />                                            Lee RY, Hench J, Ruvkun G. Regulation C. elegans DAF-16 human ortholog FKHRL1 by <br />                                                 daf-2 insulin-like signaling pathway. Curr Biol. 2001; 11:1950&#226;&#8364;&#8220;1957. [PubMed: 11747821] <br />                                            Lee SS, Kennedy S, Tolonen AC, Ruvkun G. DAF-16 target genes control C. elegans life-span <br />                                                 metabolism. Science. 2003; 300:644&#226;&#8364;&#8220;647. [PubMed: 12690206] <br />                                            Lejeune FX, Mesrob L, Parmentier F, Bicep C, Vazquez-Manrique RP, Parker JA, Vert JP, Tourette C, <br />                                                 Neri C. Large-scale functional RNAi screen C. elegans identifies genes regulate the <br />                                                 dysfunction mutant polyglutamine neurons. BMC Genomics. 2012; 13:91. [PubMed: <br />                                                 22413862] <br />                                            Li JL, et al. A Genome Scan Modifiers Age Onset Huntington Disease: The HD MAPS <br />                                                 Study. Am J Hum Genet. 2003; 73:682&#226;&#8364;&#8220;687. [PubMed: 12900792] <br />                                            Li Y, Wang WJ, Cao H, Lu J, Wu C, Hu FY, Guo J, Zhao L, Yang F, Zhang YX, Li W, Zheng GY, <br />                                                 Cui H, Chen X, Zhu Z, He H, Dong B, Mo X, Zeng Y, Tian XL. Genetic association FOXO1A <br />                                                 FOXO3A longevity trait Han Chinese populations. Hum Mol Genet. 2009; 18:4897&#226;&#8364;&#8220; <br />                                                 4904. [PubMed: 19793722] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Lin K, Hsin H, Libina N, Kenyon C. Regulation Caenorhabditis elegans longevity protein <br />                                                 DAF-16 insulin/IGF-1 germline signaling. Nat Genet. 2001; 28:139&#226;&#8364;&#8220;145. [PubMed: <br />                                                 11381260] <br />                                            Metzger S, Rong J, Nguyen HP, Cape A, Tomiuk J, Soehn AS, Propping P, Freudenberg-Hua Y, <br />                                                 Freudenberg J, Tong L, Li SH, Li XJ, Riess O. Huntingtin-associated protein-1 modifier the <br />                                                 age-at-onset Huntington&apos;s disease. Hum Mol Genet. 2008; 17:1137&#226;&#8364;&#8220;1146. [PubMed: 18192679] <br />                                            Morley JF, Brignull HR, Weyers JJ, Morimoto RI. The threshold polyglutamine-expansion protein <br />                                                 aggregation cellular toxicity dynamic influenced aging Caenorhabditiselegans. <br />                                                 Proceedings National Academy Sciences United States America. 2002; <br />                                                 99:10417&#226;&#8364;&#8220;10422. [PubMed: 12122205] <br />                                            Mukhopadhyay A, Deplancke B, Walhout AJ, Tissenbaum HA. Chromatin immunoprecipitation <br />                                                 (ChIP) coupled detection quantitative real-time PCR study transcription factor binding to <br />                                                 DNA Caenorhabditis elegans. Nat Protoc. 2008; 3:698&#226;&#8364;&#8220;709. [PubMed: 18388953] <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                                Page 13 <br />  <br />  <br />                                            Oh SW, Mukhopadhyay A, Dixit BL, Raha T, Green MR, Tissenbaum HA. Identification direct <br />                                                DAF-16 targets controlling longevity, metabolism diapause chromatin <br />                                                immunoprecipitation. Nat Genet. 2006; 38:251&#226;&#8364;&#8220;257. [PubMed: 16380712] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Paik JH, Ding Z, Narurkar R, Ramkissoon S, Muller F, Kamoun WS, Chae SS, Zheng H, Ying H, <br />                                                Mahoney J, Hiller D, Jiang S, Protopopov A, Wong WH, Chin L, Ligon KL, DePinho RA. FoxOs <br />                                                cooperatively regulate diverse pathways governing neural stem cell homeostasis. Cell Stem Cell. <br />                                                2009; 5:540&#226;&#8364;&#8220;553. [PubMed: 19896444] <br />                                            Parker JA, Arango M, Abderrahmane S, Lambert E, Tourette C, Catoire H, Neri C. Resveratrol rescues <br />                                                mutant polyglutamine cytotoxicity nematode mammalian neurons. Nat Genet. 2005; <br />                                                37:349&#226;&#8364;&#8220;350. [PubMed: 15793589] <br />                                            Soerensen M, Dato S, Christensen K, McGue M, Stevnsner T, Bohr VA, Christiansen L. Replication <br />                                                association variation FOXO3A gene human longevity using case-control <br />                                                longitudinal data. Aging Cell. 2010; 9:1010&#226;&#8364;&#8220;1017. [PubMed: 20849522] <br />                                            Taherzadeh-Fard E, Saft C, Andrich J, Wieczorek S, Arning L. PGC-1alpha modifier onset age in <br />                                                Huntington disease. Mol Neurodegener. 2009; 4:10. [PubMed: 19200361] <br />                                            Tissenbaum HA, Guarente L. Increased dosage sir-2 gene extends lifespan Caenorhabditis <br />                                                elegans. Nature. 2001; 410:227&#226;&#8364;&#8220;230. [PubMed: 11242085] <br />                                            Trettel F, Rigamonti D, Hilditch-Maguire P, Wheeler VC, Sharp AH, Persichetti F, Cattaneo E, <br />                                                MacDonald ME. Dominant phenotypes produced HD mutation STHdh(Q111) striatal <br />                                                cells. Hum Mol Genet. 2000; 9:2799&#226;&#8364;&#8220;2809. [PubMed: 11092756] <br />                                            Voisine C, Varma H, Walker N, Bates EA, Stockwell BR, Hart AC. Identification potential <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                therapeutic drugs huntington&apos;s disease using Caenorhabditis elegans. PLoS ONE. 2007; 2:e504. <br />                                                [PubMed: 17551584] <br />                                            Walker FO. Huntington&apos;s disease. Lancet. 2007; 369:218&#226;&#8364;&#8220;228. [PubMed: 17240289] <br />                                            Wang Y, Oh SW, Deplancke B, Luo J, Walhout AJ, Tissenbaum HA. C. elegans 14-3-3 proteins <br />                                                regulate life span interact SIR-2.1 DAF-16/FOXO. Mech Ageing Dev. 2006; <br />                                                127:741&#226;&#8364;&#8220;747. [PubMed: 16860373] <br />                                            Willcox BJ, Donlon TA, He Q, Chen R, Grove JS, Yano K, Masaki KH, Willcox DC, Rodriguez B, <br />                                                Curb JD. FOXO3A genotype strongly associated human longevity. Proceedings the <br />                                                National Academy Sciences United States America. 2008; 105:13987&#226;&#8364;&#8220;13992. <br />                                                [PubMed: 18765803] <br />                                            Wolff S, Ma H, Burch D, Maciel GA, Hunter T, Dillin A. SMK-1, essential regulator DAF-16- <br />                                                mediated longevity. Cell. 2006; 124:1039&#226;&#8364;&#8220;1053. [PubMed: 16530049] <br />                                            Xu EH, Tang Y, Li D, Jia JP. Polymorphism HD UCHL-1 genes Huntington&apos;s disease. J Clin <br />                                                Neurosci. 2009; 16:1473&#226;&#8364;&#8220;1477. [PubMed: 19683447] <br />                                            Yen K, Narasimhan SD, Tissenbaum HA. DAF-16/Forkhead box O transcription factor: paths to <br />                                                single Fork(head) road. Antioxid Redox Signal. 2011; 14:623&#226;&#8364;&#8220;634. [PubMed: 20673162] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                        Page 14 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 1. <br />                                            &#206;&#178;-catenin ucp-4 required neuroprotection sir-2.1 128Q nematodes. A, bar-1/ <br />                                            #x003B2;-catenin null mutation enhanced touch insensitivity tail 128Q nematodes. <br />                                            No change detected 19Q animals. ***P &lt; 0.001 compared 128Q alone. B, ucp-4 <br />                                            deletion enhanced 128Q neuronal dysfunction. No change detected 19Q animals. <br />                                            ***P &lt; 0.001 compared 128Q alone. C, 128Q transgene expression unchanged the <br />                                            protein mRNA levels 128Q nematodes bearing bar-1 ucp-4 LOF. D, Aggravation <br />                                            neuron dysfunction sir-2.1 LOF (sir-2.1(ok434)) suppressed animals <br />                                            specifically overexpressing wild-type SIR-2.1 touch receptors neurons control <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                       Page 15 <br />  <br />  <br />                                            promoter mec-3 gene effect detected animals overexpressing empty <br />                                            vector (ID1296 ID1297; Table 1). Aggravation neuron dysfunction bar-1 LOF <br />                                            (bar-1(ga80)) suppressed animals specifically overexpressing wild-type BAR-1 in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            touch receptors neurons control promoter mec-3 gene effect <br />                                            detected detected animals overexpressing vector (ID1316 ID1317; Table <br />                                            1). Shown data compiled independent extrachromosomal arrays genotype. <br />                                            ***P &lt; 0.001 versus vector. n.s.: significant. E, Neuroprotection increased <br />                                            Sir2 dosage (sir-2.1(O/E)) 128Q toxicity lost animals mutant bar-1 or <br />                                            ucp-4. *P &lt; 0.001 versus 128Q. **P &lt; 0.001 versus sir-2.1(O/E) alone. Data panels A, B <br />                                            E means&#194;&#177;SEM greater 200 animals/genotype (50&#226;&#8364;&#8220;60 animals per <br />                                            independent experiment total 4 independent experiments). Data touch <br />                                            tests strains expressing extrachromosomal arrays (panel D) means&#194;&#177;SEM greater <br />                                            60 animals/array (20&#226;&#8364;&#8220;30 animals independent experiment total 3 <br />                                            independent experiments). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                        Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 2. <br />                                            The UCP-4 promoter regulated DAF-16. A, DAF-16 binds UCP-4 promoter. The <br />                                            5&#226;&#8364;&#8482; region UCP-4 3 consensus DAF-16 binding sites (upper panel; 2 sites are <br />                                            shown; Binding site 1 closely located consensus sites) separated 3.7 kb. <br />                                            One binding sites located 894 bp upstream ATG (binding site 2) two <br />                                            located 4.655 4.964 kb upstream ATG. Since sites closely <br />                                            situated, primers designed site 4.655 kb (binding site 1). Primers also <br />                                            designed region 654 bp downstream stop codon control site 3&#226;&#8364;&#8482;-region. <br />                                            Chromatin immunoprecipitation (lower panel) using anti-DAF-16 antibody (lower panel) <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                         Page 17 <br />  <br />  <br />                                            using N2, daf-2(e1370), daf-16(mgDf50) daf-16(mgDf50);daf-2(e1370). The binding <br />                                            normalized N2, effects P &lt; 0.01 considered significant. *P <br />                                            &lt; 0.005 compared N2. B, Representative images DAF-16 overexpression increase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            activity UCP-4 promoter (1768 bp) late L4 C. elegans nematodes inferred <br />                                            quantifying intensity mCherry signals pharynx (see Results). C, The <br />                                            effects DAF-16 activity ucp-4 promoter requires binding site 2. Constructs <br />                                            encoding mCherry control ucp-4 promoter (1768 bp), carried wild- <br />                                            type (ucp-4p WT) scrambled (ucp-4p SCR) binding site 2, stably expressed N2, <br />                                            daf-16(mu86) TJ356 strain mCherry signals quantified indicated B. Data are <br />                                            mean&#194;&#177;SEM compiled independent arrays (SL1, SL2) genotype greater <br />                                            60 animals/array (20&#226;&#8364;&#8220;30 animals independent experiment total 3 <br />                                            independent experiments). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                        Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 3. <br />                                            The GSK-3&#206;&#178; inhibitor BIO neuroprotective FOXO signaling 128Q nematodes. A, <br />                                            Neuron dysfunction 128Q nematodes higher compared 19Q nematodes, about <br />                                            85% 128Q animals having defective response touch (see Parker et al., 2005). BIO <br />                                            strongly rescues expanded polyQ neurotoxicity 100-33.3 &#194;&#181;M (*P &lt; 0.001 **P &lt; 0.01 <br />                                            versus DMSO controls) effect 19Q animals. BIO rescuing activity lost in <br />                                            mutants daf-16, sir-2.1, bar-1 ucp-4. Dilution factor 3x. Percent rescue was <br />                                            calculated percentages touch response ((test &#226;&#8364;&#8220; control)/(100 &#226;&#8364;&#8220; control) * 100). <br />                                            Data means&#194;&#177;SEM greater 200 animals/point genotypes (50&#226;&#8364;&#8220;60 animals <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                    Page 19 <br />  <br />  <br />                                            independent experiment total 4 independent experiments). The <br />                                            percentages touch response (means&#194;&#177;SEM) 52&#194;&#177;3% 19Q animals treated with <br />                                            vehicle, 23&#194;&#177;2% 128Q animals treated vehicle, 46&#194;&#177;3% 128Q animals treated with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            100 &#194;&#181;M BIO 34.6&#194;&#177;4% 128Q animals treated 33.3 &#194;&#181;M BIO. B, BIO reduced <br />                                            axonal swelling PLM cells 128Q animals (*P &lt; 0.002 versus DMSO controls). Data are <br />                                            means&#194;&#177;SEM greater 100 animals/treatment (25&#226;&#8364;&#8220;30 animals independent <br />                                            experiment total 4 independent experiments). C, BIO treatment (50 &#194;&#181;M, <br />                                            100&#194;&#181;M) does modify transgene expression 128Q nematodes. n.s., significant. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                        Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 4. <br />                                            &#206;&#178;-catenin UCP2/UCP4 modulate survival mutant htt striatal cells HdhQ111 <br />                                            mice. In assays, cells subjected serum deprivation. A, Representative graph <br />                                            showing 109Q/109Q cells susceptible cell death compared 7Q/7Q cells <br />                                            (N (Number independent experiments) = 3 SD; *P &lt; 0.01 compared 7Q/7Q). B, <br />                                            Effects reducing htt levels. Htt siRNA reduced 109Q/109Q cell mortality (N = 3 SD; <br />                                            **P &lt; 0.001 versus untreated). Scramble RNA showed effect. ns: significant. In the <br />                                            following panels, data normalized variability striatal cell survival after <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            serum deprivation experiments. C, Effects reducing &#206;&#178;-catenin levels BIO <br />                                            (0.5 &#194;&#181;M) treatment. Left panel: &#206;&#178;-catenin siRNA enhanced 109Q/109Q cell mortality (N = 3 <br />                                            SD; *P &lt; 0.01 versus untreated). Scramble RNA showed effect. BIO treament <br />                                            reduced cell mortality (N = 3 SD; **P &lt; 0.05 versus DMSO controls). When &#206;&#178;-catenin <br />                                            siRNA BIO combined, change cell mortality detected compared cells <br />                                            treated scramble RNA DMSO (N = 3 SD). Right panel: &#206;&#178;-catenin siRNA and <br />                                            BIO showed effect 7Q/7Q cells. D, Mutant htt expression unchanged treatment <br />                                            BIO &#206;&#178;-catenin siRNA. &#206;&#178;-catenin levels increased BIO (N = 3, P &lt; 0.02) and <br />                                            reduced &#206;&#178;-catenin siRNA (n = 3, P &lt; 0.05). Scramble RNA effect. E, Effects of <br />                                            reducing SIRT1, UCP2 UCP4 (N = 4 SD). UCP2 siRNA enhanced 109Q/109Q cell <br />                                            mortality (*P &lt; 0.01 versus untreated) effect detected 7Q/7Q cells. UCP4 siRNA <br />                                            reduced 109Q/109Q cell mortality (*P &lt; 0.01 versus untreated) effect detected in <br />                                            7Q/7Q cells. Scramble RNAs effect. F, Mutant htt expression unchanged by <br />                                            siRNAs SIRT1, UCP2 UCP4. Shown representative western blots. Scramble <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                       Page 21 <br />  <br />  <br />                                            RNAs showed effect. G, Left panel representative western blot image showing that <br />                                            109Q/109Q cells lower SIRT1 levels SIRT1 siRNA treatment (N = 3; P &lt; 0.05). <br />                                            Quantitative RT-PCR experiments indicated UCP2 UCP4 siRNA decreased mRNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            levels UCP2 UCP4, respectively (N = 5 SD; P &lt; 0.05). The effect UCP2/4 <br />                                            siRNAs target protein expression evaluated antisera repeatedly <br />                                            unable detect protein western blot experiments, scramble RNAs showed no <br />                                            effect. For panels, N indicates number independent experiments performed. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                          Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 5. <br />                                            Effects &#206;&#178;-catenin SIRT1 mutant htt striatal cells HdhQ111 mice. A, Mutant <br />                                            htt cells showed increased susceptibility cell death induced serum deprivation (***P &lt; <br />                                            0.001 compared 7Q/7Q). SIRT1 overexpression slightly enhanced survival 109Q/ <br />                                            109Q striatal cells effects 7Q/7Q cells (**P &lt; 0.01 compared untreated 109Q/ <br />                                            109Q). Reducing &#206;&#178;-catenin enhanced mortality 109Q/109Q striatal cells no <br />                                            effects 7Q/7Q cells (***P &lt; 0.001 compared untreated 109Q/109Q). When SIRT1 <br />                                            overexpression combined &#206;&#178;-catenin reduction 109Q/109Q striatal cells, the <br />                                            detrimental effect reducing &#206;&#178;-catenin compensated SIRT1 overexpression (N <br />                                            (number experiments performed) = 3 SD; ***P &lt; 0.001 compared &#206;&#178;-catenin <br />                                            reduction). n.s., significant. B, Representative western blot showing striatal cells <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                          Page 23 <br />  <br />  <br />                                            transfected SIRT1 construct (an active variant lacking internal segment N- <br />                                            terminus) increased SIRT1 levels. n.a., applicable. C, Effects reducing &#206;&#178;-catenin <br />                                            overexpressing SIRT1 expression levels UCP4 UCP2. Upper panel: The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            expression UCP4 gene upregulated &#206;&#178;-catenin siRNA dowregulated by <br />                                            SIRT1 overexpression (O/E) 109Q/109Q cells (N = 4 SD; *P &lt; 0.05). No effect was <br />                                            detected 7Q/7Q cells. Lower panel: The expression UCP2 gene downregulated <br />                                            &#206;&#178;-catenin siRNA effect SIRT1 overexpression (O/E) 109Q/109Q cells (N = <br />                                            4 SD; *P &lt; 0.05). No effect detected 7Q/7Q cells. ns: significant. Scramble <br />                                            RNA (&#206;&#178;-catenin) showed effect UCP2/UCP4 gene expression levels. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 6. <br />                                            Working model role bar-1/&#206;&#178;-catenin, daf-16/FOXO sir-2.1/SIRT1 the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            regulation mutant polyglutamine neuron homeostasis. The effects observed C. elegans <br />                                            mouse striatal cells, models early-cytotoxicity/cell-vulnerability induced by <br />                                            mutant htt, summarized context prior knowledge binding BAR-1/&#206;&#178;- <br />                                            catenin DAF-16/FOXO (Essers et al., 2005) role GSK-3/&#206;&#178;-catenin SIRT1 in <br />                                            modulation FOXO activity nucleus ability regulate gene expression <br />                                            (Greer Brunet, 2008; Landis Murphy, 2010; Yen et al., 2011). Genes required for <br />                                            neuroprotection sir-2.1 C. elegans previously reported (Parker et al., 2005) or <br />                                            indicated underligned. Bold indicates gene activity mouse striatal cells as <br />                                            suggested ability SIRT1 overexpression compensate detrimental effect of <br />                                            reducing &#206;&#178;-catenin cell survival, effects UCP2 expression levels. <br />  <br />  <br />  <br />  <br />                                                J Neurosci. Author manuscript; available PMC 2013 September 23. <br />                             Parker et al.                                                                                                                 Page 25 <br />  <br />  <br />  <br />  <br />                                                                                             Table 1 <br />                            Names genotypes C. elegans strains used study. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Name           Genotypea                                                                           Origin <br />  <br />                             Bristol N2     Standard wild type                                                                  CGCb <br />                             TJ356          zIs356[pdaf-16&#226;&#710;&#183;daf-16-gfp; rol-6]IV                                                 CGC <br />                             CB1370         daf-2(e1370)III                                                                     CGC <br />                             GR1307         daf-16(mgDf50)I                                                                     CGC <br />                             HT1890         daf-16(mgDf50)I;daf-2(e1370)III                                                     CGC <br />                             CF1038         daf-16(mu86)I                                                                       CGC <br />                             EW15           bar-1(ga80)X                                                                        CGC <br />                             CY121          ucp-4(ok195)V                                                                       CGC <br />                             LG100          geIn3[sir-2.1; rol-6]; dyf-(?)                                                      L. Guarente <br />                                                                                                                                (MIT, MA) <br />  <br />                             ID1280c        geIn3[sir-2.1; rol-6]                                                               This study <br />  <br />                             ID245          igIs245[mec-3p&#226;&#710;&#183;htt57-19Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]                                See Parker et al., 2005 <br />                             ID1            igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]                                 See Parker et al., 2005 <br />                                            igIs245[mec-3p&#226;&#710;&#183;htt57-19Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; bar-1(ga80)X <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             ID191                                                                                              This study <br />                             ID1281         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; bar-1(ga80)X                   This study <br />                             ID192          igIs245[mec-3p&#226;&#710;&#183;htt57-19Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; ucp-4(ok195)V                 This study <br />                             ID1282         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; ucp-4(ok195)V                  This study <br />                             ID1283         igIs245[mec-3p&#226;&#710;&#183;htt57-19Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X               This study <br />                             ID1285         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X                This study <br />                             ID1286         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; daf-16(mgDf50)I                This study <br />                             ID1287         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; geIn3[sir-2.1; rol-6]          This study <br />                             ID1288         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; geIn3[sir-2.1; rol-6]; bar-    This study <br />                                            1(ga80)X <br />                             ID1289         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; geIn3[sir-2.1; rol-6]; ucp-    This study <br />                                            4(ok195)V <br />                             ID1290         igIs245[mec-3p&#226;&#710;&#183;htt57-19Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X;              This study <br />                                            igEx500[mec-3p&#226;&#710;&#183;sir-2.1-biGFP] <br />                             ID1291         igIs1[mec-3p&#226;&#710;&#183;htt57-19Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X;                This study <br />                                            igEx501[mec-3p&#226;&#710;&#183;sir-2.1-biGFP] <br />                             ID1292         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X;               This study <br />                                            igEx502[mec-3p&#226;&#710;&#183;sir-2.1-biGFP] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             ID1293         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X;               This study <br />                                            igEx503[mec-3p&#226;&#710;&#183;sir-2.1-biGFP] <br />                             ID1296         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X;               This study <br />                                            igEx506[mec-3p&#226;&#710;&#183;GFP] <br />                             ID1297         igIs1[mec-3p&#226;&#710;&#183;htt57-128Q&#226;&#710;&#183;CFP; lin-15(+); mec-7p&#226;&#710;&#183;YFP]; sir-2.1(ok434)X;               This study <br />                                            igEx507[mec-3p&#226;&#710;&#183;GFP] <br />                             ID1298         igEx508[ucp-4p&#226;&#710;&#183;ucp-4&#226;&#710;&#183;mCherry; myo-2p&#226;&#710;&#183;GFP]                                           This study <br />                             ID1299         igEx509[ucp-4p&#226;&#710;&#183;ucp-4&#226;&#710;&#183;mCherry; myo-2p&#226;&#710;&#183;GFP]                                           This study <br />                             ID1300         igEx510[ucp-4p&#226;&#710;&#183;ucp-4&#226;&#710;&#183;mCherry; myo-2p&#226;&#710;&#183;GFP]; z <br /> </body></html>