<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>       ReseaRch papeR <br />  <br /> Nucleus 3:3, 286-289; May/June 2012; &#194;&#169; 2012 Landes Bioscience <br />  <br />  <br />  <br />       Did modulation expression noise shape <br />        evolution dimensional genome <br />               organizations eukaryotes? <br />                                                                 Kuljeet singh sandhu <br />                                        Genome Technology Biology Group; Genome Institute singapore; singapore <br />  <br />  <br />       Keywords: chromatin interactions, expression noise, evolution, epigenetics, 3D genome organization, nuclear architecture <br />  <br />  <br />           The pervasive role distant chromatin interactions transcriptional regulation increasingly evident. There <br />           possibility greater diversity chromatin interactions genomic locus contribute stochastic <br />           variation gene expression. however, issue addressed. here, I present lines evidence, which <br />           suggest variation trans chromatin interactions occur cost expression noise. Genomic regions <br />           nucleosome depletion, abundant rapid transcription essential gene clusters exhibit relatively fewer <br />           trans chromatin interactions nucleus. Moreover, loci greater number interactions tend higher <br />           expression noise. Based observations, I hypothesize dimensional organization eukaryotic <br />           genomes evolved selective pressure minimize expression noise essential gene clusters in <br />  <br />  <br />                     &#194;&#169; 2012 Landes Bioscience. <br />           nucleus. <br />  <br />  <br />  <br />                             Introduction                                       minimum. One way locus experience variation in <br />  <br />  <br />                                        Do distribute. <br />                                                                                expression highly diverse chromatin interactions. Large num- <br /> Accumulating evidence suggests prominent role nonran-                   ber chromatin interactions locus generally occur the <br /> dom dimensional organization chromatin essential                   time cell spatial constraints nucleus, <br /> genomic functions eukaryotic nucleus.1,2 The earliest evi-              represent aggregation diverse interactions across <br /> dence comes observation random insertion                  population cells. Highly varying interactions respect <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> a gene distinct genomic locations modulates transcription               time space allow gene maintain constant <br /> distinctly.3 Long range cis trans chromosomal interactions                 expression level. I test hypothesis using genome wide inter- <br /> correlate strongly active repressive epigenetic states             chromosomal (trans) chromatin interaction data yeast.7 <br /> of gene.1 Moreover, spatial positioning genes the <br /> nucleus matters expression. The nuclear core associated                                             Results <br /> with high transcription activity, nuclear lamina interac- <br /> tions chromosomal domains associated stable silenc-                Using trans chromatin interactions biological rep- <br /> ing genes.4,5 Recent reports suggest genes               licates,7 I calculated normalized number interactions, i.e., <br /> not independent transcriptional units own; instead               degree (&#206;&#180;), 3 kb region yeast genome (Material <br /> interact extensive manner.6 Despite                Methods section). I following key observations: <br /> advances, clear dimensional organiza-                  (1) As apparent Figure 1A&#226;&#8364;&#8220;D, genomic regions with <br /> tion genome evolved non-random conformation.                     low nucleosome occupancy, abundant expression rapid rate <br /> Transcriptional activity (or inactivity) coordination genes             nascent transcription significantly lower degree trans <br /> are arguably best studied constraints eukaryotic genome                 interactions regions moderate levels nucleosome <br /> organization. Here, I present lines evidence, sug-              occupancy, gene expression transcription rate (p = 3.8e-13, <br /> gest modulation expression noise               1.3e-04, 3.2e-05 respectively), suggesting nucleosome <br /> of evolutionary constraints, shaped                   depleted domains highly transcribing genes, selected <br /> three dimensional conformation eukaryotic genome.                           high variation trans chromatin interactions. <br />     Highly varying expression levels genes deleterious             Similarly, regions low gene activity repressed genes <br /> if happens essential gene loci              exhibit lower degree (p = 8.8e-05), presumably ensure <br /> evolutionary constraint shape 3D conformation genome                 stable silencing delicate regulation precise transcrip- <br /> in manner expression noise essential loci              tional level. <br />  <br />  Correspondence to: Kuljeet Singh Sandhu; Email: sandhuks@gis.a-star.edu.sg kulsinsan@gmail.com <br />  Submitted: 02/04/12; Revised: 04/02/12; Accepted: 04/04/12 <br />  http://dx.doi.org/10.4161/nucl.20263 <br />  <br />  <br /> 286                                                                     Nucleus                                                      Volume 3 Issue 3 <br />        ReseaRch papeR                                                                                                                     ReseaRch papeR <br />  <br />  <br />  <br />  <br />                    &#194;&#169; 2012 Landes Bioscience. <br />                                        Do distribute. <br />  Figure 1. (a) Linear view degree trans chromatin interactions, nucleosome occupancy gene expression yeast genome. For simplic- <br />  ity, 16 chromosomes concatenated increasing order. Rolling mean 10 consecutive windows 3 kb plotted track. highlighted in <br />  orange regions high degree identified using cut-off shown Figures S6 S7. (B&#226;&#8364;&#8220;e) Degree distributions genomic regions with <br />  high, moderate low (B) nucleosome occupancy, (c) gene expression, (D) transcription rate (mol/min) (e) density essential genes. detailed <br />  degree distribution distinct representations essential genes given Figure S1. (F) Distribution expression noise regions high <br />  low degree different cut-offs. Dashed horizontal line represents mean expression noise. p values (B&#226;&#8364;&#8220;F) calculated using Mann- <br />  Whitney U test. Ns, significant. <br />  <br />  <br />  <br />     (2) The observations prompted explore essen-                    These observations line recent report which <br /> tial gene clusters, generally located nucleosome                   suggests inter-chromosomal DNA interactions occur the <br /> depleted domains highly expressed,8 selected                cost increased expression noise.10 Since expression noise <br /> the highly varying chromatin interactions. Lower degree                      data available far measured protein level gene <br /> domains higher density essential genes compared                 transcription, possibility noise gene transcription <br /> the ones moderate density, indeed, suggests essential                 partly compensated post-transcriptional level <br /> gene clusters preferentially located genomic domains                 regulatory mechanisms explored context, <br /> of lower variability chromatin interactions (Fig. 1E; Fig. S1,               explain relatively weaker significance, observed in <br /> p = 2.3e-03). Category regions essential gene com-                   Figure 1F. Moreover, alterations chromatin organization <br /> prises lowly expressed genes shows lower                   account intrinsic noise, expression <br /> degree. Therefore, I propose essential gene clusters             noise data includes extrinsic noise contributed factors like <br /> have evolutionary selected high variability                 fluctuating micro-environments cells. <br /> chromatin interactions, consequently high expression <br /> noise, nucleus.                                                                                        Discussion <br />     (3) To test variation chromatin <br /> interactions modulates expression noise, I analyzed expres-                 The recent boom proximity ligation based techniques in <br /> sion noise data available literature.9 I obtained maximal            combination generation sequencing enabled the <br /> expression noise 3 kb bin. As shown Figure 1F,                  mapping all-to-all chromatin interactions nucleus.7,11,12 <br /> regions higher degree positively associate high expres-               Interestingly, highly expressed housekeeping genes shown <br /> sion noise (p = 3e-02, 1e-02), supporting hypothesis               converge transcriptionally active multi-gene complexes <br /> diversity chromatin interactions contribute stochastic                efficient coordinated transcription.7 Such coordina- <br /> noise expression levels.                                                     tion transcription disturbed locus involved <br />  <br />  <br /> www.landesbioscience.com                                                 Nucleus                                                                         287 <br />  stochastically moves active compartment or, worse       distinct restriction enzymes (EcoRI HindIII), used <br /> case, mis-fires repressed compartment. Therefore,          calculate normalized degree chromatin interactions. <br /> loci expected relatively stable transcription         Nucleosome occupancy data taken Kaplan et al.18 and <br /> foci sites, ones essential toler-      RNA-Seq data set taken Nagalakshmi et al.19 Data <br /> ate noisy expression. Indeed, highly varying expression level       nascent transcription rate taken Pelechano et al.20 <br /> has proposed facilitator adaptive gene expres-          List essential genes taken Saccharomyces Genome <br /> sion evolution certain genes advantageous        Deletion Project (http://www-sequence.stanford.edu/group/ <br /> in certain context.13 However, importantly, essential genes tend        yeast_deletion_project/downloads.html). Expression noise data <br /> to low expression noise noisy expression essential        originally generated Newman et al.9 scaled 0 to <br /> genes deleterious organism.8 Moreover, order        1 Li et al.21 DNA double strand break data Dmc1&#206;&#8221; mutant <br /> to minimize expression noise, essential genes evolved          taken Buhler et al. Most experimental data sets <br /> to locate clusters chromosome harbor consis-           taken generated yeast cells grown YPD growth media <br /> tently open state chromatin, shown others.8 I        respective studies. <br /> essential gene clusters maintain lower expression noise            Analysis. First, yeast genome divided bins of <br /> limiting total number interactions, possibly restricting    3 kb (equivalent average size EcoRI/HindIII fragments <br /> the global mobility loci involved. Essential gene clusters       yeast genome). The centers restriction fragments were <br /> might tethered nuclear sites abundant rapid tran-      mapped 3 kb bins. Data sets biological replicates (EcoRI <br /> scription. The hypothesis strongly supported observa-         HindIII) were, then, quantile normalized. The quality con- <br /> tion nucleosome depleted, highly rapidly transcribed           trol plots shown Figure S5A. For biological <br /> regions lower degree trans, relatively higher degree        replicates, undirected network trans chromatin interactions <br /> of cis chromatin interactions (Fig. 1A&#226;&#8364;&#8220;C; Figs. S2 S3). It       3 kb bins constructed using igraph library (http:// <br /> also notable maximal extent cis interactions locus      igraph.wikidot.com/) R-package (http://www.r-project.org/). <br />  <br />                 &#194;&#169; 2012 Landes Bioscience. <br /> is far lower (3.4-fold) trans interactions, suggesting <br /> the intra-territorial spatial constraints selectively restricting the <br /> mobility genomic domains high gene activity. Similarly, the <br />                                                                         Number interactions (degree) 3 kb region calcu- <br />                                                                         lated using &#226;&#8364;&#732;degree&#226;&#8364;&#8482; function igraph. Geometric mean two <br />                                                                         replicates used analysis. Binning data cal- <br /> regions low (or no) gene activity lower degree           culation geometric mean normalized EcoRI HindIII <br />  <br />  <br />                                     Do distribute. <br /> of trans interactions hinting maintenance stable            data sets make analysis robust possible biases <br /> repressive state low delicate transcription limiting         caused <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> local GC content distal ends <br /> the mobility locus. Indeed, attachment sub-nuclear          restriction fragments individual data sets (Fig. S5B C). <br /> structures like nucleoli nuclear periphery, associated     Rolling mean 10 consecutive windows taken plotting <br /> with transcriptional repression, significantly limits mobil-        visualization purpose. The yeast genes (S288C genome) were <br /> ity chromatin domains.14 Conversely, regions mod-           assigned 3 kb bins mapping location transcription <br /> erate levels expression relatively greater degree trans      start sites (TSS) bins. Mean nucleosome occupancy, gene <br /> interactions susceptible higher expression noise.       expression nascent transcription rate calculated each <br /> Higher expression noise moderately expressed genes          3 kb window. The cut-off values degree, nucleosome occu- <br /> observed too.15 Furthermore, reports suggest             pancy, gene expression transcription rate elaborated in <br /> that certain chromatin interactions deleterious        Figure S6. More liberal stringent cut-offs alter the <br /> mutagenic.16 By extrapolation, region having higher number         conclusions article (Fig. S7). To calculate essential gene <br /> trans chromatin interactions, risk experi-        density, neighboring windows 3 kb (12 kb total) were <br /> encing erroneous interactions. Restricting total number        considered. High moderate density essential genes was <br /> of interactions minimize risk mutagenic          demarcated &gt; 3 1&#226;&#8364;&#8220;3 genes 12 kb region respec- <br /> interactions. Indeed, DNA double strand break signal strongly           tively. A detailed degree distribution distinct representa- <br /> associates regions higher degree chromatin interac-      tions essential genes given Figure S1. Since expression <br /> tions (Fig. S4). Therefore, I propose nonrandom          noise data available yeast genes, I considered <br /> dimensional organization eukaryotic genome                maximum noise value available 3 kb bin. The key findings <br /> evolved selection constraint reduce expression         study scrutinized possible sequence <br /> noise risk erroneous mutagenic interactions essen-      mapping biases removing telomeric (15 kb) centro- <br /> tial gene loci. Having said that, I rule evo-      meric (15 kb side) regions analysis (Fig. S8A&#226;&#8364;&#8220;D). <br /> lutionary constraints like coordinated expression functionally       Moreover, yeast chromosomes known tethered cen- <br /> related genes, reported elsewhere.6,17 More data sets       tromeres and, therefore, abundant trans interactions sites <br /> future help scrutinize hypothesis.                            proximal centromeres (&#194;&#177; 50 kb, Fig. S8E). I reassessed the <br />                                                                         major observations study excluding regions &#194;&#177; 50 kb <br />                     Material Methods                                centromere. The analysis consistently adheres the <br />                                                                         hypothesis (Fig. S8F&#226;&#8364;&#8220;I). Similarly, key observations the <br /> Data sets. Genome wide chromatin interaction data taken             study robust variations replicates, <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Duan et al.7 Biological replicate data, generated using        measured using Shannon&#226;&#8364;&#8482;s entropy normalized EcoRI and <br />  <br />  <br /> 288                                                              Nucleus                                                     Volume 3 Issue 3 <br />  HindIII libraries (Figs. S9 S10). For pair-wise comparisons                                                                         Acknowledgments <br /> between distributions, Mann-Whitney U test used. All                                        K.S.S. acknowledges financial support A*STAR <br /> statistical analysis performed R package (http://www.r-                                      Singapore Joint Council Office (JCO) grant (12302EG016). <br /> project.org/). <br />                                                                                                                                       Supplemental Material <br />                  Disclosure Potential Conflicts Interest                                      Supplemental     materials                 here: <br /> No potential conflicts disclosed.                                                  www.landesbioscience.com/journals/nucleus/article/20263 <br />                                                                   9.    Newman JR, Ghaemmaghami S, Ihmels J, Breslow DK,              16. Lin C, Y <br /> </body></html>