<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Biochem Parasitol. 2011 January ; 175(1): 21&#226;&#8364;&#8220;29. doi:10.1016/j.molbiopara.2010.08.005. <br />  <br />  <br />  <br />                            Identification inhibitors putative malaria drug targets <br />                            novel antimalarial compounds <br />                            Gregory J. Crowthera, Alberto J. Napulia, James H. Gilliganb, Kerstin Gagaringb, Rachel <br />                            Borboab, Carolyn Francekb, Zhong Chenb, Eleanor F. Dagostinob, Justin B. Stockmyerb, Yu <br />                            Wangb, Philip P. Rodenbougha, Lisa J. Castanedaa, David J. Leiblya, Janhavi Bhandaria, <br />                            Michael H. Gelba, Achim Brinkerb, Ingo Engelsb, Jennifer Taylorb, Arnab K. Chatterjeeb, <br />                            Pascal Fantauzzic, Richard J. Glynneb, Wesley C. Van Voorhisa, Kelli L. Kuhenb <br />  <br />                            aUniversity Washington, Seattle, WA, USA b Genomics Institute Novartis Research <br />                            Foundation (GNF), San Diego, CA, USA c Medicines Malaria Venture, Geneva, Switzerland <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  The efficacy marketed antimalarial drugs compromised evolution parasite <br />                                  resistance, underscoring urgent need new drugs new mechanisms action. We <br />                                  taken high-throughput approach identifying novel antimalarial chemical inhibitors <br />                                  prioritized drug targets P. falciparum, excluding targets inhibited currently <br />                                  used drugs. A screen commercially available libraries identified 5,655 low molecular weight <br />                                  compounds inhibit growth P. falciparum cultures EC50 values 1.25 &#206;&#188;M. These <br />                                  compounds tested 384- 1536-well biochemical assays activity nine <br />                                  Plasmodium enzymes: adenylosuccinate synthetase (AdSS), choline kinase (CK), deoxyuridine <br />                                  triphosphate nucleotidohydrolase (dUTPase), glutamate dehydrogenase (GDH), guanylate kinase <br />                                  (GK), N-myristoyltransferase (NMT), orotidine 5&#226;&#8364;&#178;-monophosphate decarboxylase (OMPDC), <br />                                  farnesyl pyrophosphate synthase (FPPS) S-adenosylhomocysteine hydrolase (SAHH). These <br />                                  enzymes selected using TDRtargets.org, believed excellent potential drug <br />                                  targets based criteria likely essentiality, druggability, amenability high- <br />                                  throughput biochemical screening. Six targets inhibited the <br />                                  antimalarial scaffolds potential use drug development, target validation <br />                                  studies exploration P. falciparum biochemistry biology. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Keywords <br />                                  target-based drug development; enzyme activity assays; antimalarial compounds <br />  <br />  <br />                            1. Introduction <br />                                              Approximately 3 billion people, half world&#226;&#8364;&#8482;s population, live at-risk regions for <br />                                              malaria infection. This leads 250 million malaria cases year nearly one <br />  <br />  <br />                            &#194;&#169; 2010 Elsevier B.V. All rights reserved. <br />                            Corresponding author: K.L.K., Genomics Institute Novartis Research Foundation (GNF), 10675 John Jay Hopkins Drive, San <br />                            Diego CA 92121, USA; phone: 858-812-1819; fax: 858-812-1819; kkuhen@gnf.org. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Crowther et al.                                                                                             Page 2 <br />  <br />  <br />                                              million deaths (World Malaria Report, WHO, 2009, <br />                                              http://www.who.int/malaria/world_malaria_report_2009/en/index.html). The etiologic agent <br />                                              Plasmodium falciparum organism responsible majority deaths <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              malaria, 90% occur Africa 85% children age 5. <br />                                              The presence widespread drug resistance hampering effectiveness of the <br />                                              available drug arsenal [1&#226;&#8364;&#8220;6] notable exception artemisinin-based derivatives. <br />                                              Artemisinin-based combination therapies (ACT) recommended first-line treatments <br />                                              falciparum malaria countries endemic disease [7]. Though ACT remains <br />                                              effective, recent concerns efficacy artemisinin-based derivatives has <br />                                              declined Thai-Cambodian border [8&#226;&#8364;&#8220;10]. Potential clinical resistance closely <br />                                              monitored areas Southeast Asia artemisinin used monotherapy for <br />                                              decades antimalarial drug resistance previously developed. The potential <br />                                              development artemisinin resistance parasite loss ACT crisis, as <br />                                              ACTs recommended worldwide new agents currently phase 3 clinical <br />                                              trials based ACT. In battle fight malaria, introduction new drugs novel <br />                                              mechanisms action essential. <br />  <br />                                              Data essentiality putative drug targets Plasmodium growth limited few <br />                                              individual genes tested gene disruption studies [11&#226;&#8364;&#8220;19]. However, <br />                                              chemical inhibition target genetic essentiality does translate to <br />                                              pharmacological efficacy pathogen [20], genetic data essentiality <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              available comprehensive genome-wide dataset Plasmodium. Inhibition P. <br />                                              falciparum replication chemical inhibition potential drug targets can, instead, <br />                                              provide chemical validation targets. To end, undertook effort provide <br />                                              chemical tools explore biology prioritized Plasmodium targets, provide a <br />                                              starting point chemical validation targets. We developed biochemical assays to <br />                                              test enzymatic activity prioritized targets. A set compounds been <br />                                              pre-annotated having inhibitory activity P. falcpiarum proliferation erythrocytes <br />                                              tested inhibitory activity biochemical assays. We reasoned that <br />                                              screening compounds known cell-based activity might, part, <br />                                              address issues plagued biochemical approaches. These include <br />                                              cellular permeability, efflux metabolism small-molecule inhibitors. <br />  <br />                                              We screened GNF&#226;&#8364;&#8482;s non-proprietary compound collection identified subset of <br />                                              5,655 compounds potent inhibitory activity P. falciparum growth vitro as <br />                                              described [21]. These compounds non-proprietary available <br />                                              commercially, facilitating broad exploration mechanism action academic <br />                                              community. We structures compounds available at <br />                                              http://www.ebi.ac.uk/chembldb/ http://www.TDRtargets.org/. We included this <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              dataset dose-response data activity strains P. falciparum, 3D7 W2, as <br />                                              EC50 values, cytotoxicity information human hepatocellular carcinoma cell <br />                                              line Huh7, measure promiscuity compounds historical high- <br />                                              throughput screens conducted GNF. Where available, compounds tested re- <br />                                              ordered powders supplier. This collection screened high-throughput <br />                                              biochemical assays Plasmodium enzymes considered promising drug <br />                                              targets based data compiled TDRtargets.org [22] literature data. The <br />                                              results target-based biochemical screens discussed below. Notably we <br />                                              identified chemical inhibitors 6 9 targets tested sub-library few <br />                                              thousand compounds, preselected cellular activity. <br />  <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                              Page 3 <br />  <br />  <br />                               2. Materials methods <br />                               2.1. Identification antimalarial compounds <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              A high-throughput screen GNF&#226;&#8364;&#8482;s non-proprietary compound collection performed <br />                                              using erythrocyte-based P. falciparum strain 3d7 infection assay essentially described <br />                                              [21]. Compounds screened 1.25 &#206;&#188;M; hits inhibiting growth by <br />                                              &#226;&#8240;&#165;50% reconfirmed dose-response format determine EC50 values, cellular <br />                                              cytotoxicity (quantified CC50) assessed Huh7 hepatocellular carcinoma cell <br />                                              line. Compounds selectivity index &lt;5 (CC50/EC50) excluded further <br />                                              consideration, compounds quench fluorescence. Reconfirmed hit <br />                                              compounds (5,655 total) re-ordered powders 3,086 compounds were <br />                                              available purchase suppliers. <br />  <br />                               2.2. Target selection <br />                                              In selecting enzymes screen (Table 1), employed TDRtargets.org [22] other <br />                                              sources identify Plasmodium enzymes candidates compound screening using the <br />                                              following criteria: i) enzyme likely required <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> parasite; ii) the <br />                                              enzyme small-molecule binding pocket exploited drug-like <br />                                              molecule inhibit enzymatic activity; iii) exploitable differences the <br />                                              parasite human enzymes and/or pathways belong drug might <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              developed minimal adverse effects host; iv) enzyme amenable to <br />                                              recombinant expression (for precedent selected targets [23&#226;&#8364;&#8220;30]; and <br />                                              v) enzymatic activity measured high-throughput format. Of criteria we <br />                                              focused especially i, iv, v druggability (ii) selectivity (iii) currently <br />                                              difficult predict. (As example latter, difluoromethylornithine inhibits the <br />                                              human T. brucei ornithine decarboxylases, selectively kills T. brucei cells, <br />                                              apparently rapid turnover human enzyme [31].) The nine <br />                                              enzymes chosen criteria adenylosuccinate synthetase (AdSS), choline kinase <br />                                              (CK), deoxyuridine triphosphate nucleotidohydrolase (dUTPase), farnesyl pyrophosphate <br />                                              synthase (FPPS), glutamate dehydrogenase (GDH), guanylate kinase (GK), N- <br />                                              myristoyltransferase (NMT), orotidine 5&#226;&#8364;&#178;-monophosphate decarboxylase (OMPDC), S- <br />                                              adenosylhomocysteine hydrolase (SAHH). <br />  <br />                               2.3 Expression purification recombinant Plasmodium proteins <br />                                              Recombinant histidine-tagged enzymes Plasmodium falciparum P. vivax were <br />                                              expressed Escherichia coli purified immobilized metal affinity chromatography, <br />                                              essentially described previously [32]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.4 Biochemical assay details <br />                                              Specific activities enzymes studied reported previously [33], along <br />                                              comparisons literature values possible. <br />  <br />                                              Enzyme activity assays miniaturized optimized 1536-well plates test <br />                                              compounds screened dose-response format 12 points &#194;&#189;-log serial dilutions <br />                                              (0.0005 &#226;&#8364;&#8220; 100 &#206;&#188;M), exception SAHH, screened single-point format <br />                                              384-well plates. Incubations room temperature unless noted. All <br />                                              reactions included final [DMSO] 1%. The initial collection antimalarial compounds <br />                                              available set DMSO solutions pre-existing GNF compound library <br />                                              (available formats: 5,655 compounds, 1536-well dose-response format; 5,543 compounds, <br />                                              384-well single point format), 6 targets screened using collections (Table 3). <br />                                              As original stocks antimalarial compounds depleted, remaining <br />                                              targets (FPPS, dUTPase, NMT) screened using DMSO solutions reconstituted <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                             Page 4 <br />  <br />  <br />                                              3,086 compounds available repurchasing time screens. Hits were <br />                                              reconfirmed using re-ordered purified powders available vendor tested <br />                                              dose-response format. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Chemical reactions detection strategies enzyme shown Table 2; <br />                                              additional details noted below. In general, assay concentrations substrates were <br />                                              approximately 3 times respective Km&#226;&#8364;&#8482;s. Compounds, general, added enzyme <br />                                              buffer reagents 10 minutes prior initiation reaction addition substrate. If <br />                                              compounds hit multiple target-based assays triaged lack specificity <br />                                              (i.e., likely false positives) included final hit list (Supplementary Table S1). <br />  <br />                                              AdSS assayed buffer 50 mM Tris, pH 7.0, 5 mM DTT, 0.1% BSA, 0.001% <br />                                              Tween-20, 2.5 mM MgCl2. Final concentrations 500 ng/mL AdSS (10 nM), 30 <br />                                              &#206;&#188;M GTP, 60 &#206;&#188;M IMP, 1 mM aspartate. Incubation time 40 minutes, the <br />                                              GDP produced AdSS detected Transcreener GDP detection mix (Cisbio), <br />                                              used according manufacturer&#226;&#8364;&#8482;s instructions. Homogeneous time-resolved fluorescence <br />                                              (HTRF) GDP produced monitored excitation 337 nm dual emission at <br />                                              620 665 nm. <br />  <br />                                              CK assayed buffer 100 mM HEPES, pH 7.5, 150 mM NaCl 1 mM <br />                                              MgCl2. Final concentrations 312.5 ng/mL CK (5 nM), 10 &#206;&#188;M ATP, 280 &#206;&#188;M <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              choline. Incubation time 30 minutes, ADP produced CK was <br />                                              detected Transcreener ADP detection mix (Cisbio), AdSS. <br />  <br />                                              dUTPase assayed buffer 25 mM MOPS, pH 8.0, 10 mM KCl, 1.25 mM <br />                                              MgCl2, 0.1 mg/mL BSA, 0.005% Triton X 20% glycerol. Final concentrations 2 <br />                                              ng/mL dUTPase (97 pM) 14 &#206;&#188;M dUTP. Incubation time 30 minutes. Production of <br />                                              PPi monitored PPiLight kit (Lonza) conversion ATP measurement of <br />                                              luminescent signal corresponding [ATP]. The PPi detection mix diluted 1:4 with <br />                                              reaction buffer prior use. <br />  <br />                                              FPPS assayed buffer 50 mM HEPES, pH 7.5, 0.625 mM MgCl2, 1 mM <br />                                              NaCl, 0.25 mM TCEP, 5% BSA, 10% glycerol, 0.01% Triton X-100. Final <br />                                              concentrations 1.3 &#206;&#188;g/mL FPPS (45 nM), 25 &#206;&#188;M geranyl diphosphate, 25 &#206;&#188;M <br />                                              isopentenyl diphosphate. Incubation time 30 minutes. Production PPi monitored <br />                                              PPiLight kit (Lonza), dUTPase. <br />  <br />                                              GDH assayed buffer 100 mM Tris, pH 8.0, 1 mM DTT, 0.1 mg/mL BSA, <br />                                              0.005% Triton X-100. Final concentrations 0.5 &#206;&#188;g/mL GDH (10 nM), 400 &#206;&#188;M <br />                                              NADP+, 10 mM glutamate. Incubation time 45 minutes. The fluorescence the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              NADPH produced monitored excitation 355 nm emission 486 nm. <br />  <br />                                              GK assayed buffer 50 mM Tris, pH 7.5, 50 mM KCl, 2 mM MgCl2 0.1 <br />                                              mg/mL BSA. Final concentrations 1.5 ng/mL GK (130 pM), 7 &#206;&#188;M GMP, 10 &#206;&#188;M <br />                                              ATP. Incubation time 20 minutes, ADP GDP produced GK were <br />                                              detected Transcreener ADP detection mix (Cisbio), AdSS. <br />  <br />                                              NMT assayed buffer 50 mM Tris, pH 7.5, 6.25 mM NaCl, 0.625 mM <br />                                              MgCl2, 10% BSA, 0.005% Triton X-100. Final concentrations 1.5 &#206;&#188;g/mL NMT <br />                                              (30 nM), 15 uM myristoyl-CoA, 90 uM GSSYSRKNK, synthetic peptide obtained <br />                                              Peptides International GenScript order parameters 85% purity no <br />                                              N- C-terminal modifications. This peptide based N-terminal sequence of <br />                                              adenylate kinase 2, substrate N-myristoyltransferase P. falciparum vivo [34]. <br />                                              Incubation time 45 minutes 30 &#194;&#176;C, ThioGlo [35] added final <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                             Page 5 <br />  <br />  <br />                                              concentration 15 &#206;&#188;M detect free &#226;&#8364;&#8220;SH groups CoA produced NMT. ThioGlo <br />                                              fluorescence monitored excitation 360 nm emission 540 nm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              OMPDC assayed buffer 100 mM Tris, pH 8.0, 50 mM NaCl, 1 mM MgCl2, <br />                                              1 mM DTT, 0.3 mg/mL BSA. Final concentrations 3 ng/mL OMPDC (77 pM), 2 <br />                                              &#206;&#188;M OMP, 1 &#206;&#188;M ATP, 1 &#206;&#188;g/mL human CMP kinase [36], used convert <br />                                              UMP (produced OMPDC) + ATP UDP + ADP. Incubation time 180 minutes, <br />                                              ADP detected Transcreener ADP detection mix (Cisbio), for <br />                                              AdSS. <br />  <br />                                              SAHH assayed buffer 50 mM potassium phosphate, pH 8.0. Final concentrations <br />                                              5 &#206;&#188;g/mL SAHH (100 nM), 4 &#206;&#188;M S-adenosylhomocysteine, 2.5 &#206;&#188;g/mL adenosine <br />                                              deaminase P. falciparum (added consume adenosine produced SAHH, <br />                                              promoting continued progress forward reaction). Incubation time 90 minutes, <br />                                              ThioGlo [35] added final concentration 25 &#206;&#188;M detect free &#226;&#8364;&#8220;SH <br />                                              groups homocysteine produced SAHH. ThioGlo fluorescence monitored via <br />                                              excitation 360 nm emission 540 nm. The SAHH screen unique that <br />                                              compounds initially tested single concentration 8.3 &#206;&#188;M; therefore, initial hits <br />                                              defined compounds inhibiting SAHH &#226;&#8240;&#165;25% concentration. <br />  <br />                               3. Results <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.1. Identification antimalarial compounds <br />                                              We identified set 5,655 compounds inhibit growth P. falciparum erythrocytes <br />                                              described [21]. ~7,500 primary screen hits tested 8-point dose-response curves <br />                                              reconfirmation antimalarial activity EC50 value determination drug- <br />                                              resistant strains: sulfadoxine-resistant 3D7 strain multi-drug resistant W2 strain <br />                                              (resistant chloroquine, quinine, pyrimethamine, sulfadoxine, cycloguanil). Known <br />                                              inhibitors compound collection (e.g., artemisinin, mefloquine, dihydrofolate reductase <br />                                              inhibitors, quinine) identified, providing internal validation assay sensitivity. <br />                                              Counterscreens included fluorescence quencher assessment cytotoxicity Huh7 <br />                                              human hepatocellular carcinoma cell line. Compounds EC50 &lt; 1.25 &#206;&#188;M P. <br />                                              falciparum 3d7 W2 strains selectivity index (SI) Huh7 cell line least <br />                                              5-fold did exhibit fluorescence quenching retained study. <br />  <br />                               3.2. Choline kinase <br />                                              CK catalyzes step synthesis phosphatidylcholine, essential for <br />                                              intraerythrocytic growth Plasmodium parasite [26, 37, 38]. The Km CK choline <br />                                              determined ~200 &#206;&#188;M, consistent value 140 &#206;&#188;M reported [26]. The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              chemical inhibitor hexadecyltrimethylammonium bromide (HDTAB) used a <br />                                              positive control CK inhibition [37]; however, physicochemical properties HDTAB <br />                                              interfered homogeneous time resolved fluorescence (HTRF) signal detection. <br />  <br />                                              As listed Table 2, CK screen resulted 69 hits 1.2% hit rate. IC50 values ranged <br />                                              0.06 &#206;&#188;M 11 &#206;&#188;M P. falciparum EC50 values ranged 0.013 &#206;&#188;M 1.25 <br />                                              &#206;&#188;M. 23 powders available reconfirmation powders, reconfirmed in <br />                                              triplicate IC50 values ranging 2.9 &#206;&#188;M 19.0 &#206;&#188;M. Corresponding P. falciparum <br />                                              EC50 values 10-fold potent. Of hits identified (see <br />                                              Supplementary Table S1), compound CK-1 (a 3-chlorobenzothiophene amide; shown in <br />                                              Figure 1A) chemically tractable structure suitable starting point for <br />                                              chemical optimization. CK-2 CK-3 (Fig. 1A) analogs scaffold. While <br />                                              activities enzyme weaker (20-fold) cell-based activities, rank orders <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                          Page 6 <br />  <br />  <br />                                              potency compounds the cell-based target-based <br />                                              assays. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.3. S-Adenosylhomocysteine hydrolase <br />                                              SAHH, component methylation cycle methyl groups donated lipids, <br />                                              nucleic acids, proteins, molecules, certainly essential highly <br />                                              druggable (Table 1). The similarity parasite human active sites concern, with <br />                                              residue adenosine binding site &#226;&#8364;&#8220; Cys59 PfSAHH, Thr60 HsSAHH &#226;&#8364;&#8220; <br />                                              differing [39]. The Km SAHH S-adenosylhomocysteine was <br />                                              determined ~2 &#206;&#188;M, close literature value 1.2 &#206;&#188;M [24]. The SAHH screen <br />                                              resulted 38 hits hit rate 0.68% (Table 2). 20 powders 38 hits were <br />                                              available, these, seven compounds reconfirmed dose-response format two <br />                                              triaged based overlap target screens (AdSS, below), suggesting these <br />                                              promiscuous inhibitors. IC50 values ranged 0.1 &#206;&#188;M 9.1 &#206;&#188;M the <br />                                              corresponding P. falciparum EC50 values ranged 0.9 &#206;&#188;M 7.6 &#206;&#188;M. Of five <br />                                              reconfirmed specific hits identified (Supplementary Table S1), SAHH-1 (Fig. 1B) an <br />                                              interesting 8-hydroxyquinoline-based hit serve potential starting point for <br />                                              medicinal chemistry. Other hits tool compounds, not <br />                                              chemically attractive drug discovery, include SAHH-2, quinazosin, known <br />                                              vasodilator alpha-1 adrenergic receptor target [40] SAHH-3, known as <br />                                              phanquinone, antimicrobial agent [41]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.4. Enzymes involved nucleotide metabolism (AdSS, dUTPase, GK, OMPDC) <br />                                              With AdSS, purine salvage pathway, obtained Km values 14 &#206;&#188;M IMP, <br />                                              105 &#206;&#188;M GTP, 0.05 mM aspartate, somewhat different previously <br />                                              reported values 22.8 &#206;&#188;M, 4.8 &#206;&#188;M, 1.4 mM, respectively [29]. The AdSS screen <br />                                              resulted 9 hits 0.2% hit rate (Table 2). Hadacidin used positive control <br />                                              inhibitor [42], IC50 value 62 &#206;&#188;M. Of hits, available powders and <br />                                              reconfirmed. Two compounds triaged inhibited SAHH activity, <br />                                              leaving 3 candidate AdSS inhibitors (Supplementary Table S1). However, these <br />                                              compounds unlikely starting points medicinal chemistry chemically labile <br />                                              functionalities sulfonylureas (AdSS-1), imines derived aldehydes (AdSS-2) <br />                                              quaternary aromatic amines (AdSS-3). AdSS-1 (Fig. 1C), N-sulfonylurea, be <br />                                              useful tool compound; however, publicly available information cites cytostatic <br />                                              mechanism action human cells (in PubChem: 120394059 120394064). AdSS-2 (Fig. <br />                                              1C), benzyaldehydederived imine, relatively specific malaria inhibition; this <br />                                              compound useful tool compound AdSS. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              GK selected target based involvement nucleic acid synthesis. GK converts <br />                                              (d)GMP (d)GDP, turn used make (d)GTP, essential nucleic <br />                                              acid synthesis. The likely essentiality GK is, however, doubt, Plasmodium <br />                                              thymidylate kinase (2.7.4.9) convert dGMP dGDP [43, 44]. An additional <br />                                              enzyme, ribonucleoside-diphosphate reductase (1.17.4.1), convert dGDP GDP. Thus, <br />                                              GK blocked, (d)GDP dGMP. Nevertheless, the <br />                                              expression GK liver stage Plasmodium life cycle [45] suggests possible <br />                                              importance stage possible usefulness target complementing those <br />                                              prominent erythrocyte stages. <br />  <br />                                              The Km P. vivax GK GMP 3.5 &#206;&#188;M, lower value 22 &#206;&#188;M reported for <br />                                              P. falciparum GK [28]. The GK screen resulted 17 hits (0.3% hit rate) shown in <br />                                              Table 2. The positive control iodoacetamide IC50 value 36.5 &#206;&#188;M assay <br />                                              conditions used. The range guanylate kinase IC50 values 17 hits 0.5 &#206;&#188;M&#226;&#8364;&#8220; 7.8 <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                               Page 7 <br />  <br />  <br />                                              &#206;&#188;M. Seven compounds available powders; reconfirmed 3 showing <br />                                              specificity PvGK (Supplementary Table S1) shown Figure 1D. GK-1 two <br />                                              nitro groups does represent chemically attractive scaffold unless analogs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              nitro groups remain active. Nevertheless, compound relatively specific <br />                                              does appear promiscuous inhibitor historical HTS run GNF. <br />                                              Interestingly, screens overlapping activity pathogen-based screens <br />                                              related presence nitro groups. GK-2 GK-3 (Figure 1D) are <br />                                              potentially useful tool compounds interrogate malaria parasite biology confirm <br />                                              PvGK target compounds. However, GK-2 ammonium salt with <br />                                              low likelihood oral bioavailability, GK-3 imine napthoquinone may <br />                                              stability issues trap electrophiles; thus, ideal starting point medical <br />                                              chemistry efforts. <br />  <br />                                              OMPDC pyrimidine biosynthesis enzyme downstream well-validated target <br />                                              dihydroorotate dehydrogenase [46]. The approximate Km orotidine 5&#226;&#8364;&#178;-monophosphate <br />                                              (OMP) ~0.2 &#206;&#188;M, lower previously reported value 13.4 &#206;&#188;M (PMID <br />                                              15683248). The OMPDC screen resulted 26 hits approximate hit rate 0.5% <br />                                              (Table 2). IC50 values ranged 0.22 &#206;&#188;M 10 &#206;&#188;M P. falciparum EC50 values <br />                                              ranged 0.176 &#206;&#188;M 1.7 &#206;&#188;M. 11 powders available reconfirmation these <br />                                              reconfirmed, identified fluorescence quenchers therefore <br />                                              false positives. Unfortunately, attractive hit chemical standpoint, OMPDC-1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              (Fig. 1E), weak inhibitor OMPDC (IC50 19.7 &#206;&#188;M), potent the <br />                                              cellular P. falciparum assay. OMPDC-1 useful tool compound investigation <br />                                              effects parasite OMPDC inhibition. <br />  <br />                                              dUTPase protects DNA integrity preventing buildup dUTP conversion to <br />                                              dUMP, converted dTTP used DNA synthesis [47]. The Km of <br />                                              dUTPase dUTP measured 4.5 &#206;&#188;M, somewhat higher literature value of <br />                                              1.9 &#206;&#188;M [48]. The positive control used dUTPase inhibition 2&#226;&#8364;&#178;-deoxy-5&#226;&#8364;&#178;-O- <br />                                              (triphenylmethyl)uridine, IC50 value 2.4 &#206;&#188;M assay conditions. <br />                                              The dUTPase screen resulted 95 hits (3.1% hit rate) positive control inhibitor <br />                                              performed expected. 91 compounds available powders retested dose- <br />                                              response reconfirmation screen IC50 values. In parallel, hits counterscreened for <br />                                              potential inhibit components PPi detection kit Lonza. The <br />                                              counterscreen tested putative hits ability inhibit direct detection ~5 &#206;&#188;M PPi, <br />                                              using detection reagents primary dUTPase. The positive control showed the <br />                                              expected inhibition profile dUTPase assay inhibition counterscreen. <br />                                              However, hits inhibitors counterscreen. In conclusion, did not <br />                                              identify hits inhibitors dUTPase. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.5. Enzymes involved pathways adding lipid groups proteins (NMT FPPS) <br />                                              NMT transfers myristate (saturated C14 fatty acid) groups N-terminal glycine residues of <br />                                              substrate proteins, increasing hydrophobicity promoting associations with <br />                                              membranes proteins [49]. NMT Km 7 &#206;&#188;M myristoyl-CoA 21 &#206;&#188;M <br />                                              synthetic peptide GSSYSRKNK. <br />  <br />                                              The positive control inhibitor NMT inhibition, aurintricarboxylate, IC50 ~3 <br />                                              &#206;&#188;M. The NMT screen resulted 7 hits overall hit rate 0.2%. The positive control <br />                                              inhibitor, aurintricarboxylate, performed expected. 5 compounds available and <br />                                              reconfirmed inihibition. NMT-1 (Fig. 1F) did score past screens performed at <br />                                              GNF, suggesting specific activity NMT. While single nitro group, with <br />                                              concomitant risk mutagenic potential [50], compound useful tool <br />                                              compound validate NMT viable drug target. None structures are <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                            Page 8 <br />  <br />  <br />                                              chemically attractive drug discovery, contain structural alerts, as <br />                                              nitro groups NMT-1 NMT-3 ammonium salts NMT-2, 4, 5 (see <br />                                              Supplementary Table S1), possibly used tool compounds NMT <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              inhibition. <br />  <br />                                              FPPS involved post-translational modification proteins protein <br />                                              farnesylation. FPPS produces farnesyl diphosphate (a.k.a. farnesyl pyrophosphate FPP), <br />                                              used protein farnesyltransferase (PFT), validated target Plasmodium <br />                                              [51], attachment farnesyl groups proteins. Km&#226;&#8364;&#8482;s FPPS 8 &#206;&#188;M geranyl <br />                                              diphosphate 9 &#206;&#188;M isopentenyl diphosphate. The positive control inhibitor (1- <br />                                              hydroxy-2-phenylethylidene)bis-phosphonic acid IC50 7.7 &#206;&#188;M FPPS and <br />                                              EC50 value ~7.05 &#206;&#188;M P. falciparum vitro. The FPPS screen resulted 147 <br />                                              hits. Unfortunately, exception positive control, (1-hydroxy-2- <br />                                              phenylethylidene)bis-phosphonic acid, hits reconfirmed FPPS assay were <br />                                              hits counterscreen (conducted described dUTPase). In conclusion, no <br />                                              compounds hit specificity FPPS exception positive control <br />                                              inhibitor. <br />  <br />                               3.6. Glutamate dehydrogenase <br />                                              GDH provides NADPH biosynthesis glutathione reductase, oxidative stress- <br />                                              fighting enzyme thought essential [25]. In contrast mammalian GDH enzyme, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              PfGDH highly specific NADP+ NADPH, suggesting exploitable differences <br />                                              mammalian parasite enzymes. Km&#226;&#8364;&#8482;s estimated 16 &#206;&#188;M for <br />                                              NADP+ 0.8 mM glutamate, consistent literature values 20 &#206;&#188;M 1 mM, <br />                                              respectively [25]. The GDH screen resulted 164 hits, 2.9% hit rate; however, activity <br />                                              hits GDH weak. 78 powders available tested for <br />                                              reconfirmation hits reconfirmed assay conditions used. To decrease the <br />                                              assay stringency increase sensitivity inhibitors, concentration substrate <br />                                              [NADP+] reduced 400 &#206;&#188;M 50 &#206;&#188;M, hits reconfirmed. In conclusion, <br />                                              hits identified GDH. <br />  <br />                               4. Discussion <br />                                              The availability large compound collections automated robotic systems ultra high- <br />                                              throughput screening commonplace pharmaceutical companies, less <br />                                              frequent academia. Here collection antimalarial chemical inhibitors <br />                                              identified screening non-proprietary compound collections. In application, we <br />                                              attempted link antimalarial compounds possible protein targets. The <br />                                              cell-active compounds tested high-throughput assays enzymes believed to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              potential drug targets based considerations essentiality, druggability, <br />                                              selectivity, amenability high-throughput screening (section 2.2 Table 1). These <br />                                              enzymes included involved nucleotide metabolism (AdSS, dUTPase, GK, and <br />                                              OMPDC) attachment lipid groups proteins (NMT FPPS) as <br />                                              components key metabolic pathways (CK, GDH, SAHH). While were <br />                                              unable detect inhibitors dUTPase, FPPS GDH, identified inhibitors AdSS, <br />                                              GK, OMPDC, NMT, CK, SAHH potential starting points medicinal chemistry or <br />                                              tool compounds Plasmodium enzymes. In view, results validate <br />                                              disprove wisdom target selection strategy. We disappointed find <br />                                              potent inhibitors enzymes chosen; however, screen larger chemical library <br />                                              retrieved compounds suitable development leads good activity <br />                                              enzymes parasite cells. <br />  <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                              Page 9 <br />  <br />  <br />                                              IC50 values preliminary hits enzymes checked repurchased compounds <br />                                              (Table 3 Supplementary Table S1). Reversible inhibitors generally act their <br />                                              intracellular targets IC50 lower EC50 parasite. However, the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              compound concentrations of target-based hits required inhibit enzyme activity <br />                                              higher concentrations needed inhibit P. falciparum growth. There a <br />                                              number possible explanations cellular enzymatic discrepancy. First, of <br />                                              targets inhibited antimalarial compounds only, the <br />                                              primary, target compound; inhibition protein (and/or non-protein <br />                                              targets) contribute cellular activity [52]. Second, possible mechanism <br />                                              compound affecting parasitic activity completely unrelated the <br />                                              biochemical activity reported here. Third, EC50 values lower relative to <br />                                              biochemical IC50 values test compound accumulates higher concentrations inside <br />                                              parasite. Lastly, comparing enzyme activity antimalarial activity be <br />                                              particularly helpful, given unavoidable differences intracellular vitro <br />                                              conditions (e.g. substrate concentrations), unless comparing structure activity <br />                                              relationships (SAR) set chemically related analogs relative enzyme and <br />                                              antimalarial inhibition. Derivation pharmacophore correlation enzyme- <br />                                              based parasite-based inhibition needed identify meaningful SAR correlations and <br />                                              link target hits observed cell-based <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span>phenotype</span>. Our highest priority follow <br />                                              enzyme inhibitors IC50&#226;&#8364;&#8482;s &lt; EC50&#226;&#8364;&#8482;s, investigations associations of <br />                                              high-value compounds targets prove worthwhile chemically validate targets. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Three targets (dUTPase, GDH FPPS) lacked hits compounds tested. It is <br />                                              possible enzymes essential parasite assay conditions. <br />                                              Alternatively, broader library antimalarial compounds selected <br />                                              lacking chemical composition capable inhibiting enzymes. <br />                                              Regarding possibility, enzymes predicted moderate high <br />                                              druggability according silico methods (see Table 1). Our approach offered test of <br />                                              silico predictions suggested benefit refinement. <br />  <br />                                              To test inhibition particular biochemical target causal antimalarial <br />                                              activity, approaches possible. These include: (a) allowing cultures evolve <br />                                              resistance compound determining mutations occur gene the <br />                                              presumed target [53]; (b) studying compounds chemically related original hit and <br />                                              looking correlation IC50&#226;&#8364;&#8482;s presumed target EC50&#226;&#8364;&#8482;s the <br />                                              parasite [54]; (c) determining overexpression putative target overcomes the <br />                                              inhibition observed presence compound [55] (although genetic manipulation of <br />                                              Plasmodium challenging); (d) detecting buildup presumed target&#226;&#8364;&#8482;s substrate <br />                                              depletion product extracts compound-treated cells [56&#226;&#8364;&#8220;58], and/or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              determining parasite growth restored supplementation media with <br />                                              missing product [59]; (e) using compounds affinity probes isolate target <br />                                              proteins cell lysates [60]. <br />  <br />                                              Despite extensive screening work reported here, mechanisms action 5,655 <br />                                              antimalarial compounds generally remain elusive. However, modifications present <br />                                              approach lead greater success identifying mechanisms action. For example, <br />                                              studying compounds EC50&#226;&#8364;&#8482;s &gt;1.25 &#206;&#188;M enable discovery weak <br />                                              activity parasite EC50/IC50 data strongly point particular target. <br />                                              Furthermore, considering roughly 2 5% organism&#226;&#8364;&#8482;s genes legitimate <br />                                              potential drug targets [61], number proteins tested relatively small to <br />                                              labor-intensive requirement optimizing separate biochemical assay protein. <br />                                              A generic method assessing compound-protein binding, thermal melting [32] or <br />                                              Surface Plasmon Resonance [62], allow proteins studied simultaneously, <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                               Page 10 <br />  <br />  <br />                                              increasing odds identified likely targets antiparasitic <br />                                              compounds. Finally, possible similar screening efforts instead focus on <br />                                              finding new scaffolds previously validated targets. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              In conclusion, data support original proposition target-based drug <br />                                              discovery accelerated starting cell-active compounds. While early to <br />                                              say enzyme-compound associations identified screens represent <br />                                              compound&#226;&#8364;&#8482;s true mechanism action inhibiting parasite growth, compounds <br />                                              utility exploring biology surrounding possible targets. Such <br />                                              tool compounds employed basic studies enzyme function pathway biology <br />                                              starting points structure-activity relationship (SAR) selectivity studies. <br />                                              Screens kind performed yield valuable information do <br />                                              result identification biochemically potent lead compounds drug development. <br />  <br />                               Supplementary Material <br />                                              Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                              These efforts possible combination pharmaceutical resources (Novartis), funding by <br />                                              Medicines Malaria Venture (MMV) partnership academia (the University Washington). We thank <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Thierry Diagana Novartis Institute Tropical Diseases (NITD) Singapore help planning project <br />                                              critical review manuscript, Christophe Bodenreider NITD sharing unpublished data. <br />  <br />  <br />                               References <br />                                              1. Valderramos SG, Fidock DA. Transporters involved resistance antimalarial drugs. Trends <br />                                                 Pharmacol Sci. 2006; 27:594&#226;&#8364;&#8220;601. [PubMed: 16996622] <br />                                              2. Ekland EH, Fidock DA. In vitro evaluations antimalarial drugs relevance clinical <br />                                                 outcomes. Int J Parasitol. 2008; 38:743&#226;&#8364;&#8220;7. [PubMed: 18406409] <br />                                              3. Nkrumah LJ, Riegelhaupt PM, Moura P, et al. Probing multifactorial basis Plasmodium <br />                                                 falciparum quinine resistance: evidence strain-specific contribution sodium-proton <br />                                                 exchanger PfNHE. Mol Biochem Parasitol. 2009; 165:122&#226;&#8364;&#8220;31. [PubMed: 19428659] <br />                                              4. Sidhu AB, Valderramos SG, Fidock DA. pfmdr1 mutations contribute quinine resistance and <br />                                                 enhance mefloquine artemisinin sensitivity Plasmodium falciparum. Mol Microbiol. 2005; <br />                                                 57:913&#226;&#8364;&#8220;26. [PubMed: 16091034] <br />                                              5. Sidhu AB, Uhlemann AC, Valderramos SG, Valderramos JC, Krishna S, Fidock DA. Decreasing <br />                                                 pfmdr1 copy number plasmodium falciparum malaria heightens susceptibility mefloquine, <br />                                                 lumefantrine, halofantrine, quinine, artemisinin. J Infect Dis. 2006; 194:528&#226;&#8364;&#8220;35. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 16845638] <br />                                              6. Sidhu AB, Verdier-Pinard D, Fidock DA. Chloroquine resistance Plasmodium falciparum malaria <br />                                                 parasites conferred pfcrt mutations. Science. 2002; 298:210&#226;&#8364;&#8220;3. [PubMed: 12364805] <br />                                              7. Eastman RT, Fidock DA. Artemisinin-based combination therapies: vital tool efforts to <br />                                                 eliminate malaria. Nat Rev Microbiol. 2009; 7:864&#226;&#8364;&#8220;74. [PubMed: 19881520] <br />                                              8. Dondorp AM, Nosten F, Yi P, et al. Artemisinin resistance Plasmodium falciparum malaria. N <br />                                                 Engl J Med. 2009; 361:455&#226;&#8364;&#8220;67. [PubMed: 19641202] <br />                                              9. Lim P, Alker AP, Khim N, et al. Pfmdr1 copy number arteminisin derivatives combination <br />                                                 therapy failure falciparum malaria Cambodia. Malar J. 2009; 8:11. [PubMed: 19138391] <br />                                              10. White NJ. Qinghaosu (artemisinin): price success. Science. 2008; 320:330&#226;&#8364;&#8220;4. [PubMed: <br />                                                   18420924] <br />                                              11. Odom AR, Van Voorhis WC. Functional genetic analysis Plasmodium falciparum <br />                                                   deoxyxylulose 5-phosphate reductoisomerase gene. Mol Biochem Parasitol. 2010; 170:108&#226;&#8364;&#8220;11. <br />                                                   [PubMed: 20018214] <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                    Page 11 <br />  <br />  <br />                                              12. Slavic K, Straschil U, Reininger L, et al. Life cycle studies hexose transporter of <br />                                                  Plasmodium species genetic validation essentiality. Mol Microbiol. 2010 <br />                                              13. Vaughan AM, O&#226;&#8364;&#8482;Neill MT, Tarun AS, et al. Type II fatty acid synthesis essential for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  malaria parasite late liver stage development. Cell Microbiol. 2009; 11:506&#226;&#8364;&#8220;20. [PubMed: <br />                                                  19068099] <br />                                              14. Sasaki N, Hirai M, Maeda K, et al. The Plasmodium HU homolog, binds plastid DNA <br />                                                  sequence-independent manner, essential parasite&#226;&#8364;&#8482;s survival. FEBS Lett. 2009; 583:1446&#226;&#8364;&#8220; <br />                                                  50. [PubMed: 19358847] <br />                                              15. Gunther S, Matuschewski K, Muller S. <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>Knockout</span> studies reveal important role Plasmodium <br />                                                  lipoic acid protein ligase A1 asexual blood stage parasite survival. PLoS One. 2009; 4:e5510. <br />                                                  [PubMed: 19434237] <br />                                              16. Sijwali PS, Koo J, Singh N, Rosenthal PJ. Gene disruptions demonstrate independent roles the <br />                                                  falcipain cysteine proteases Plasmodium falciparum. Mol Biochem Parasitol. 2006; <br />                                                  150:96&#226;&#8364;&#8220;106. [PubMed: 16890302] <br />                                              17. El Bissati K, Zufferey R, Witola WH, Carter NS, Ullman B, Ben Mamoun C. The plasma <br />                                                  membrane permease PfNT1 essential purine salvage human malaria parasite <br />                                                  Plasmodium falciparum. Proc Natl Acad Sci U S A. 2006; 103:9286&#226;&#8364;&#8220;91. [PubMed: 16751273] <br />                                              18. Omara-Opyene AL, Moura PA, Sulsona CR, et al. Genetic disruption Plasmodium <br />                                                  falciparum digestive vacuole plasmepsins demonstrates functional redundancy. J Biol Chem. <br />                                                  2004; 279:54088&#226;&#8364;&#8220;96. [PubMed: 15491999] <br />                                              19. Krnajski Z, Gilberger TW, Walter RD, Cowman AF, Muller S. Thioredoxin reductase essential <br />                                                  survival Plasmodium falciparum erythrocytic stages. J Biol Chem. 2002; 277:25970&#226;&#8364;&#8220;5. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  [PubMed: 12004069] <br />                                              20. Payne DJ, Gwynn MN, Holmes DJ, Pompliano DL. Drugs bad bugs: confronting challenges <br />                                                  antibacterial discovery. Nat Rev Drug Discov. 2007; 6:29&#226;&#8364;&#8220;40. [PubMed: 17159923] <br />                                              21. Plouffe D, Brinker A, McNamara C, et al. In silico activity profiling reveals mechanism of <br />                                                  action antimalarials discovered high-throughput screen. Proc Natl Acad Sci U S A. 2008; <br />                                                  105:9059&#226;&#8364;&#8220;64. [PubMed: 18579783] <br />                                              22. Aguero F, Al-Lazikani B, Aslett M, et al. Genomic-scale prioritization drug targets: TDR <br />                                                  Targets database. Nat Rev Drug Discov. 2008; 7:900&#226;&#8364;&#8220;7. [PubMed: 18927591] <br />                                              23. Menz RI, Cinquin O, Christopherson RI. The identification, cloning functional expression of <br />                                                  gene encoding orotidine 5&#226;&#8364;&#178;-monophosphate (OMP) decarboxylase Plasmodium <br />                                                  falciparum. Ann Trop Med Parasitol. 2002; 96:469&#226;&#8364;&#8220;76. [PubMed: 12194707] <br />                                              24. Nakanishi M, Iwata A, Yatome C, Kitade Y. Purification properties recombinant <br />                                                  Plasmodium falciparum S-adenosyl-L-homocysteine hydrolase. J Biochem. 2001; 129:101&#226;&#8364;&#8220;5. <br />                                                  [PubMed: 11134963] <br />                                              25. Wagner JT, Ludemann H, Farber PM, Lottspeich F, Krauth-Siegel RL. Glutamate dehydrogenase, <br />                                                  marker protein Plasmodium falciparum--cloning, expression characterization the <br />                                                  malarial enzyme. Eur J Biochem. 1998; 258:813&#226;&#8364;&#8220;9. [PubMed: 9874251] <br />                                              26. Choubey V, Guha M, Maity P, et al. Molecular characterization localization Plasmodium <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  falciparum choline kinase. Biochim Biophys Acta. 2006; 1760:1027&#226;&#8364;&#8220;38. [PubMed: 16626864] <br />                                              27. Gunaratne RS, Sajid M, Ling IT, Tripathi R, Pachebat JA, Holder AA. Characterization N- <br />                                                  myristoyltransferase Plasmodium falciparum. Biochem J. 2000; 348(Pt 2):459&#226;&#8364;&#8220;63. [PubMed: <br />                                                  10816442] <br />                                              28. Kandeel M, Nakanishi M, Ando T, et al. Molecular cloning, expression, characterization and <br />                                                  <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> Plasmodium falciparum guanylate kinase. Mol Biochem Parasitol. 2008; 159:130&#226;&#8364;&#8220;3. <br />                                                  [PubMed: 18374996] <br />                                              29. Jayalakshmi R, Sumathy K, Balaram H. Purification characterization recombinant <br />                                                  Plasmodium falciparum adenylosuccinate synthetase expressed Escherichia coli. Protein Expr <br />                                                  Purif. 2002; 25:65&#226;&#8364;&#8220;72. [PubMed: 12071700] <br />                                              30. Whittingham JL, Leal I, Nguyen C, et al. dUTPase platform antimalarial drug design: <br />                                                  structural basis selectivity class nucleoside inhibitors. Structure. 2005; 13:329&#226;&#8364;&#8220;38. <br />                                                  [PubMed: 15698576] <br />  <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                 Page 12 <br />  <br />  <br />                                              31. Phillips MA, Coffino P, Wang CC. Cloning sequencing ornithine decarboxylase gene <br />                                                  Trypanosoma brucei. Implications enzyme turnover selective difluoromethylornithine <br />                                                  inhibition. J Biol Chem. 1987; 262:8721&#226;&#8364;&#8220;7. [PubMed: 3036823] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              32. Crowther GJ, Napuli AJ, Thomas AP, et al. Buffer optimization thermal melt assays of <br />                                                  Plasmodium proteins detection small-molecule ligands. Journal Biomolecular Screening. <br />                                                  2009; 14:700&#226;&#8364;&#8220;707. [PubMed: 19470714] <br />                                              33. Crowther GJ, He P, Rodenbough PP, et al. Use thermal melt curves assess quality of <br />                                                  enzyme preparations. Anal Biochem. 2010; 399:268&#226;&#8364;&#8220;75. [PubMed: 20018159] <br />                                              34. Rahlfs S, Koncarevic S, Iozef R, et al. Myristoylated adenylate kinase-2 Plasmodium falciparum <br />                                                  forms heterodimer myristoyltransferase. Mol Biochem Parasitol. 2009; 163:77&#226;&#8364;&#8220;84. <br />                                                  [PubMed: 18973776] <br />                                              35. Wright SK, Viola RE. Evaluation methods quantitation cysteines proteins. Anal <br />                                                  Biochem. 1998; 265:8&#226;&#8364;&#8220;14. [PubMed: 9866701] <br />                                              36. Liou JY, Dutschman GE, Lam W, Jiang Z, Cheng YC. Characterization human UMP/CMP <br />                                                  kinase phosphorylation D- L-form deoxycytidine analogue monophosphates. Cancer <br />                                                  Res. 2002; 62:1624&#226;&#8364;&#8220;31. [PubMed: 11912132] <br />                                              37. Choubey V, Maity P, Guha M, et al. Inhibition Plasmodium falciparum choline kinase by <br />                                                  hexadecyltrimethylammonium bromide: possible antimalarial mechanism. Antimicrob Agents <br />                                                  Chemother. 2007; 51:696&#226;&#8364;&#8220;706. [PubMed: 17145794] <br />                                              38. Ancelin ML, Vial HJ. Quaternary ammonium compounds efficiently inhibit Plasmodium <br />                                                  falciparum growth vitro impairment choline transport. Antimicrob Agents Chemother. <br />                                                  1986; 29:814&#226;&#8364;&#8220;20. [PubMed: 3524430] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              39. Tanaka N, Nakanishi M, Kusakabe Y, et al. Crystal structure S-adenosyl-L-homocysteine <br />                                                  hydrolase human malaria parasite Plasmodium falciparum. J Mol Biol. 2004; 343:1007&#226;&#8364;&#8220; <br />                                                  17. [PubMed: 15476817] <br />                                              40. Greenslade FC, Scott CK, Newquist KL, Krider KM, Chasin M. Heterogeneity biochemical <br />                                                  actions vasodilators. J Pharm Sci. 1982; 71:94&#226;&#8364;&#8220;100. [PubMed: 6276530] <br />                                              41. Mett H, Gyr K, Zak O, Vosbeck K. Duodeno-pancreatic secretions enhance bactericidal activity of <br />                                                  antimicrobial drugs. Antimicrob Agents Chemother. 1984; 26:35&#226;&#8364;&#8220;8. [PubMed: 6236746] <br />                                              42. Raman J, Mehrotra S, Anand RP, Balaram H. Unique kinetic mechanism Plasmodium <br />                                                  falciparum adenylosuccinate synthetase. Mol Biochem Parasitol. 2004; 138:1&#226;&#8364;&#8220;8. [PubMed: <br />                                                  15500910] <br />                                              43. Kandeel M, Kitade Y. Molecular characterization, heterologous expression kinetic analysis of <br />                                                  recombinant Plasmodium falciparum thymidylate kinase. J Biochem. 2008; 144:245&#226;&#8364;&#8220;50. <br />                                                  [PubMed: 18477629] <br />                                              44. Kandeel M, Kitamura Y, Kitade Y. The exceptional properties Plasmodium deoxyguanylate <br />                                                  pathways potential area metabolic drug discovery studies. Nucleic Acids Symp Ser <br />                                                  (Oxf). 2009:39&#226;&#8364;&#8220;40. <br />                                              45. Tarun AS, Peng X, Dumpit RF, et al. A combined transcriptome proteome survey malaria <br />                                                  parasite liver stages. Proc Natl Acad Sci U S A. 2008; 105:305&#226;&#8364;&#8220;10. [PubMed: 18172196] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              46. Gujjar R, Marwaha A, El Mazouni F, et al. Identification metabolically stable <br />                                                  triazolopyrimidine-based dihydroorotate dehydrogenase inhibitor antimalarial activity in <br />                                                  mice. J Med Chem. 2009; 52:1864&#226;&#8364;&#8220;72. [PubMed: 19296651] <br />                                              47. Nguyen C, Kasinathan G, Leal-Cortijo I, et al. Deoxyuridine triphosphate nucleotidohydrolase a <br />                                                  potential antiparasitic drug target. J Med Chem. 2005; 48:5942&#226;&#8364;&#8220;54. [PubMed: 16161998] <br />                                              48. Quesada-Soriano I, Leal I, Casas-Solvas JM, et al. Kinetic thermodynamic characterization of <br />                                                  dUTP hydrolysis Plasmodium falciparum dUTPase. Biochim Biophys Acta. 2008; 1784:1347&#226;&#8364;&#8220; <br />                                                  55. [PubMed: 18586121] <br />                                              49. Gelb MH, Van Voorhis WC, Buckner FS, et al. Protein farnesyl N-myristoyl transferases: <br />                                                  piggyback medicinal chemistry targets development antitrypanosomatid antimalarial <br />                                                  therapeutics. Mol Biochem Parasitol. 2003; 126:155&#226;&#8364;&#8220;63. [PubMed: 12615314] <br />                                              50. Orsiere T, De Meo M, Rathelot P, et al. Implication nitro group reduction mutagenic and <br />                                                  chromosome damaging activities 22 new 5-nitroisoquinolines Salmonella mutagenicity <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                  Page 13 <br />  <br />  <br />                                                  test cytokinesis-blocked micronucleus assay. Food Chem Toxicol. 2003; 41:275&#226;&#8364;&#8220;90. <br />                                                  [PubMed: 12480303] <br />                                              51. Fletcher S, Cummings CG, Rivas K, et al. Potent, Plasmodium-selective farnesyltransferase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  inhibitors arrest growth malaria parasites: structure-activity relationships of <br />                                                  ethylenediamine-analogue scaffolds homology model validation. J Med Chem. 2008; <br />                                                  51:5176&#226;&#8364;&#8220;97. [PubMed: 18686940] <br />                                              52. Hopkins AL. Network pharmacology: paradigm drug discovery. Nat Chem Biol. 2008; <br />                                                  4:682&#226;&#8364;&#8220;90. [PubMed: 18936753] <br />                                              53. Dharia NV, Sidhu AB, Cassera MB, et al. Use high-density tiling microarrays identify <br />                                                  mutations globally elucidate mechanisms drug resistance Plasmodium falciparum. <br />                                                  Genome Biol. 2009; 10:R21. [PubMed: 19216790] <br />                                              54. Mukkamala D, No JH, Cass LM, Chang TK, Oldfield E. Bisphosphonate inhibition a <br />                                                  Plasmodium farnesyl diphosphate synthase general method predicting cell-based activity <br />                                                  enzyme data. J Med Chem. 2008; 51:7827&#226;&#8364;&#8220;33. [PubMed: 19053772] <br />                                              55. Luesch H, Wu TY, Ren P, Gray NS, Schultz PG, Supek F. A genome-wide overexpression screen <br />                                                  yeast small-molecule target identification. Chem Biol. 2005; 12:55&#226;&#8364;&#8220;63. [PubMed: <br />                                                  15664515] <br />                                              56. Hankins EG, Gillespie JR, Aikenhead K, Buckner FS. Upregulation sterol C14-demethylase <br />                                                  expression Trypanosoma cruzi treated sterol biosynthesis inhibitors. Mol Biochem <br />                                                  Parasitol. 2005; 144:68&#226;&#8364;&#8220;75. [PubMed: 16165233] <br />                                              57. Seymour KK, Lyons SD, Phillips L, Rieckmann KH, Christopherson RI. Cytotoxic effects of <br />                                                  inhibitors novo pyrimidine biosynthesis Plasmodium falciparum. Biochemistry. 1994; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  33:5268&#226;&#8364;&#8220;74. [PubMed: 7909690] <br />                                              58. Vial HJ, Thuet MJ, Ancelin ML, Philippot JR, Chavis C. Phospholipid metabolism new target <br />                                                  malaria chemotherapy. Mechanism action D-2-amino-1-butanol. Biochem Pharmacol. <br />                                                  1984; 33:2761&#226;&#8364;&#8220;70. [PubMed: 6431995] <br />                                              59. Roberts F, Roberts CW, Johnson JJ, et al. Evidence shikimate pathway apicomplexan <br />                                                  parasites. Nature. 1998; 393:801&#226;&#8364;&#8220;5. [PubMed: 9655396] <br />                                              60. Kato N, Sakata T, Breton G, et al. Gene expression signatures small-molecule compounds link <br />                                                  protein kinase Plasmodium falciparum motility. Nat Chem Biol. 2008; 4:347&#226;&#8364;&#8220;56. [PubMed: <br />                                                  18454143] <br />                                              61. Hopkins AL, Groom CR. The druggable genome. Nat Rev Drug Discov. 2002; 1:727&#226;&#8364;&#8220;30. <br />                                                  [PubMed: 12209152] <br />                                              62. Piliarik M, Vaisocherova H, Homola J. Surface plasmon resonance biosensing. Methods Mol Biol. <br />                                                  2009; 503:65&#226;&#8364;&#8220;88. [PubMed: 19151937] <br />                                              63. Hyde JE. Targeting purine pyrimidine metabolism human apicomplexan parasites. Curr <br />                                                  Drug Targets. 2007; 8:31&#226;&#8364;&#8220;47. [PubMed: 17266529] <br />                                              64. Vial HJ, Wein S, Farenc C, et al. Prodrugs bisthiazolium salts orally potent antimalarials. <br />                                                  Proc Natl Acad Sci U S A. 2004; 101:15458&#226;&#8364;&#8220;63. [PubMed: 15492221] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              65. Jarzyna R, Lenarcik E, Bryla J. Chloroquine potent inhibitor glutamate dehydrogenase in <br />                                                  liver kidney-cortex rabbit. Pharmacol Res. 1997; 35:79&#226;&#8364;&#8220;84. [PubMed: 9149320] <br />                                              66. Werner C, Stubbs MT, Krauth-Siegel RL, Klebe G. The crystal structure Plasmodium <br />                                                  falciparum glutamate dehydrogenase, putative target novel antimalarial drugs. J Mol Biol. <br />                                                  2005; 349:597&#226;&#8364;&#8220;607. [PubMed: 15878595] <br />                                              67. Martin MB, Grimley JS, Lewis JC, et al. Bisphosphonates inhibit growth Trypanosoma <br />                                                  brucei, Trypanosoma cruzi, Leishmania donovani, Toxoplasma gondii, Plasmodium <br />                                                  falciparum: potential route chemotherapy. J Med Chem. 2001; 44:909&#226;&#8364;&#8220;16. [PubMed: <br />                                                  11300872] <br />  <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                        Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Figure 1. <br />                                              Chemically tractable inhibitors discovered biochemical screens. Abbreviations refer <br />                                              enzymes inhibited: (A) choline kinase (CK), (B) S-adenosylhomocysteine hydrolase <br />                                              (SAHH), (C) adenylosuccinate synthetase (AdSS), (D) guanylate kinase (GK), (E) OMP <br />                                              decarboxylase (OMPDC), (F) N-myristoyltransferase (NMT). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                                Page 15 <br />  <br />  <br />  <br />  <br />                                                                                            Table 1 <br />                            Selection promising drug targets biochemical screens <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Enzyme (PlasmoDB ID)                       Essentiality                  Druggability score*   Genome-wide percentile ranking** <br />  <br />                             Adenylosuccinate synthase (PF13_0287)           Part purine salvage                0.8                        99.5% <br />                                                                             pathway, should <br />                                                                             essential because <br />                                                                             Plasmodium cannot <br />                                                                             synthesize purines de <br />                                                                             novo [63]. <br />  <br />                             Choline kinase (PF14_0020)                      Chemical validation                0.6                        97.9% <br />                                                                             Kennedy pathway for <br />                                                                             phosphatidylcholine <br />                                                                             synthesis suggested by <br />                                                                             work Henri Vial <br />                                                                             [64]. <br />  <br />                             dUTPase (PF11_0282)                             dUTPase likely                0.6                        99.8% <br />                                                                             essential it <br />                                                                             prevents buildup of <br />                                                                             dUTP makes dUMP <br />                                                                             dTTP synthesis [47]. <br />                             Farnesyl pyrophosphate synthase                 Since protein                         0.8                        99.3% <br />                             (PVX_092040)                                    farnesyltransferase a <br />                                                                             valid drug target in <br />                                                                             Plasmodium [51], FPPS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                             well <br />                                                                             supplies the <br />                                                                             substrate PFT. A <br />                                                                             recent paper reported an <br />                                                                             ability predict cell- <br />                                                                             killing activity <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                             bisphosphonates&#226;&#8364;&#8482; <br />                                                                             inhibition FPPS [54]. <br />                             Glutamate dehydrogenase (PF14_0164)             Thought major                 0.8                        99.7% <br />                                                                             source NADPH for <br />                                                                             glutathione reductase, <br />                                                                             thought be <br />                                                                             essential [65]. This is <br />                                                                             based fact <br />                                                                             Plasmodium is <br />                                                                             &#226;&#8364;&#339;exposed multiple <br />                                                                             oxidative stress its <br />                                                                             high metabolic rate, the <br />                                                                             degradation heme and <br />                                                                             reactive oxygen species <br />                                                                             imposed host <br />                                                                             immune system&#226;&#8364;? [66]. <br />  <br />                             Guanylate kinase (PVX_099895)                   Based metabolic                    0.2                        98.4% <br />                                                                             maps, guanylate kinase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                             appeared only <br />                                                                             route which <br />                                                                             Plasmodium convert <br />                                                                             GMP GDP dGMP <br />                                                                             dGDP. (A possible <br />                                                                             bypass noted the <br />                                                                             text.) Also, expression <br />                                                                             liver stage [45] <br />                                                                             suggests importance in <br />                                                                             stage. <br />  <br />                             N-myristoyltransferase (PF14_0127)              As argued Gelb et al.              0.8                        98.4% <br />                                                                             [49], &#226;&#8364;&#339;Genetic studies <br />                                                                             shown the <br />                                                                             NMT gene essential <br />                                                                             viability range of <br />                                                                             species, including <br />                                                                             Drosophila <br />                                                                             melanogaster, S. <br />  <br />  <br />  <br />  <br />                                                   Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                                                  Page 16 <br />  <br />  <br />  <br />  <br />                                Enzyme (PlasmoDB ID)                           Essentiality                   Druggability score*       Genome-wide percentile ranking** <br />                                                                                    cerevisiae Candida <br />                                                                                    albicans and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                    Cryptococcus <br />                                                                                    neoformans.&#226;&#8364;? <br />  <br />                                OMP decarboxylase (PF10_0225)                       Part novo                     N/A                              98.6% <br />                                                                                    pyrimidine synthesis <br />                                                                                    pathway, should <br />                                                                                    essential in <br />                                                                                    Plasmodium it <br />                                                                                    does a <br />                                                                                    pyrimidine salvage <br />                                                                                    pathway [63]. Also, <br />                                                                                    chemical validation of <br />                                                                                    novo pyrimidine <br />                                                                                    pathway comes from <br />                                                                                    studies dihydroorotate <br />                                                                                    dehydrogenase [46]. <br />  <br />                                S-adenosylhomocysteine hydrolase (PFE1050w)         Part methylation                   0.8                              99.9% <br />                                                                                    cycle blockage <br />                                                                                    fatal [39]. <br />  <br />                            * <br />                             Druggability scores taken TDRtargets.org. Scores range 0.0 1.0, 1.0 signifying maximum druggability ~0.2 signifying <br />                            average score. <br />                            ** <br />                               TDRtargets.org used perform multiparameter weighted search, points awarded various features considered desirable a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            potential drug target. Percentiles computed rank-order list genes Plasmodium genome. Points awarded as <br />                            follows: 8 points gene life cycle stage (merozoite, early late ring, early late schizont, early late trophozoite) in <br />                            expression ranks quintile (80th 100th percentile) given erythrocyte life cycle stage; 4 points gene life <br />                            cycle stage expression 60th 80th percentile; 50 points gene genetic validation (according manual curation of <br />                            literature); 35 points gene associated publications (according automated analysis PubMed); 50 points each <br />                            gene associated crystal structures; 30 points gene 3D structural model (ModBase); 25 points each <br />                            gene lacking ortholog humans; 20 points gene lacking predicted transmembrane domain; 20 points gene coding protein <br />                            100 kilodaltons; 100 points gene coding enzyme; 40 points gene orthologs essential model <br />                            organism; 30 points gene activity assay available. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Mol Biochem Parasitol. Author manuscript; available PMC 2012 January 1. <br />                             Crowther et al.                                                                                                                            Page 17 <br />  <br />  <br />  <br />  <br />                                                                                                Table 2 <br />                            Screening reactions detection strategies <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Enzyme (EC number)                 Chemical reactions                Detection strategy                    Control wells* <br />  <br />                                Adenylosuccinate synthetase (6.3.4.4)   Primary: GTP + IMP + L-           Detect GDP using Cisbio               Hadacidin [42] <br />                                                                        aspartate &#226;&#8224;&#8217; GDP + Pi +            Transcreener ADP kit. <br />                                                                        adenylosuccinate <br />                                Choline kinase (2.7.1.32)               Primary: ATP + choline &#226;&#8224;&#8217;          Detect ADP using Cisbio               No choline <br />                                                                        ADP + phosphocholine              Transcreener ADP kit. <br />  <br />                                dUTPase (3.6.1.23)                      Primary: dUTP + H2O &#226;&#8224;&#8217;             Detect PPi Lonza PPiLight        CAS: 14270-73-6,84472-83-3 <br />                                                                        dUMP + PPi                        kit. <br />  <br />                                Farnesyl pyrophosphate synthase         Primary: geranyl diphosphate      Detect PPi Lonza PPiLight        compound 11 [67] <br />                                (2.5.1.&#226;&#710;&#8217;)                               + isopentenyl diphosphate &#226;&#8224;&#8217;       kit. <br />                                                                        farnesyl diphosphate + PPi <br />  <br />                                Glutamate dehydrogenase (1.4.1.4)       Primary: L-glutamate + H2O        Detect NADPH fluorescence.            No glutamate <br />                                                                        + NADP+ &#226;&#8224;&#8217; 2-oxoglutarate <br />                                                                        + NH3 + NADPH + H+ <br />  <br />                                Guanylate kinase (2.7.4.8)              Primary: ATP + GMP &#226;&#8224;&#8217;              Detect ADP GDP using              Iodoacetamide <br />                                                                        ADP + GDP                         Cisbio Transcreener ADP kit. <br />  <br />                                N-myristoyltransferase (2.3.1.97)       Primary: myristoyl-CoA +          Detect CoA&#226;&#8364;&#8482;s free &#226;&#8364;&#8220;SH groups          aurintricarboxylate <br />                                                                        glycylpeptide &#226;&#8224;&#8217; CoA + N-          ThioGlo. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                        myristoylglycylpeptide <br />  <br />                                OMP decarboxylase (4.1.1.23)            Primary: OMP &#226;&#8224;&#8217; UMP +              Detect ADP using Cisbio               No OMP <br />                                                                        ATP CMP kinase: UMP +             Transcreener ADP kit. <br />                                                                        ATP &#226;&#8224;&#8217; UDP + ADP <br />  <br />                                S- adenosylhomocysteine hydrolase       Primary: S-                       Detect homocysteine&#226;&#8364;&#8482;s free &#226;&#8364;&#8220;SH        No S- adenosylhomocysteine <br />                                (3.3.1.1)                               adenosylhomocysteine + H2O        groups ThioGlo. <br />                                                                        &#226;&#8224;&#8217; adenosine + homocysteine <br />                                                                        Adenosine deaminase: <br />                                 <br /> </body></html>