<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.13 no.13 pp.3127-3135, 1994 <br />  <br />  <br /> The yeast NOP4 gene product essential <br /> nucleolar protein required pre-rRNA processing <br /> and accumulation 60S ribosomal subunits <br />  <br /> Chao Sun John L.Woolford, Jr1                                   At different small RNA molecules specific- <br />                                                                  ally localized metazoan nucleoli. By virtue their <br /> Department Biological Sciences, Carnegie Mellon University,   sequence complementarity pre-rRNA demon- <br /> Mellon Institute, 4400 Fifth Avenue, Pittsburgh, PA 15213, USA   strated association pre-rRNA pre-ribosomal <br /> &apos;Corresponding author                                            particles, small nucleolar RNAs (snoRNAs) are <br /> Communicated I.Mattaj                                         hypothesized involved ribosome assembly <br />                                                                  (reviewed Fournier Maxwell, 1993; Mattaj et al., <br /> The Saccharomyces cerevisiae NOP4 gene isolated              1993). The U3 U8 snoRNAs shown be <br /> by screening kgtll yeast genomic DNA library                   necessary pre-rRNA processing (Kass et al., 1990; <br /> with monoclonal antibody yeast nucleolar             Savino Gerbi, 1990; Peculis Steitz, 1993). <br /> protein. NOP4 encodes 78 kDa protein contains                Several non-ribosomal nucleolar proteins known <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> two prototypical RNA recognition motifs (RRMs)                   required ribosomal subunit synthesis. Fibrillarin is <br /> flanking imperfect RRM lacking characteristic                 associated U3, U8 U13 snoRNAs, is <br /> RNP1 RNP2 motifs. In addition, fourth             required early steps rRNA processing (Tyc and <br /> incomplete RRM. NOP4 single copy essential gene             Steitz, 1989; Kass et al., 1990). Nucleolin binds both <br /> present chromosome XVI, RADI PEP4.                rDNA pre-rRNA shuttles nucleolus <br /> To examine function Nop4p, constructed               cytoplasm. These findings suggest roles nucleolin <br /> conditional <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>null allele</span> NOP4 placing gene             rRNA transcription assembly, transport <br /> under control glucose-repressible GAL]                nucleolar molecules (Herrera Olson, 1986; Bugler <br /> promoter. When cells shifted galactose-con-             et al., 1987; Lapeyre et al., 1987; Borer et al., 1989; <br /> taining medium glucose-containing medium, NOP4                Caizergues-Ferrer et al., 1989). B23, called N038, <br /> transcription terminated, Nop4 protein depleted            numatrin ribocharin, associated preribosomal <br /> and cell growth impaired. Nop4 protein depletion              particles facilitate ribosome assembly (Spector <br /> results diminished accumulation 60S ribosomal              et al., 1984; Hugle et al., 1985; Schmidt-Zachmann and <br /> subunits, assignable defect ribosome biogenesis          Franke, 1988; Zhang et al., 1989). Like nucleolin, B23 <br /> arising lack production mature 25S rRNA             shuttles nucleolus cytoplasm be <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> 27S precursor rRNA.                                         involved transport nucleolar components (Borer et <br /> Key words: pre-rRNA processing/ribosome assembly/                al., 1989). <br /> RNA recognition motifs/yeast nucleolar protein                      The mechanisms pre-rRNA processing ribosome <br />                                                                  assembly investigated yeast Saccharo- <br /> Introduction                                                     myces cerevisiae. Mutants defective pre-rRNA pro- <br />                                                                  cessing ribosome assembly genes encoding <br /> Most steps eukaryotic ribosome biogenesis occur            snoRNAs nucleolar proteins identified and <br /> the nucleolus, subnuclear compartment defined           manipulated study functions nucleolar molecules <br /> location actively transcribed rDNA (Karpen et al.,            (reviewed Warner, 1990; Woolford, 1991; Woolford <br /> 1988; reviewed Hadjiolov, 1985; Sommerville, 1986;            Warner, 1991; Fournier Maxwell, 1993). Four of <br /> Nigg, 1988; Scheer Benavente, 1990; Warner, 1990;            12 yeast snoRNAs, U3, U14, snRIO snR30, have <br /> Hemandez-Verdun, 1991 Scheer et al., 1993). Nucleolar            shown necessary important pre-rRNA <br /> functions necessary ribosome biogenesis include tran-        processing (reviewed Tollervey Hurt, 1990; Fournier <br /> scription rDNA, post-transcriptional modification         Maxwell, 1993). Nop I p, yeast homologue of <br /> processing pre-rRNA, transport molecules            fibrillarin, associated known yeast <br /> from nucleolus, assembly ribosomal macro-             snoRNAs required methylation processing <br /> molecules coordinated fashion. These processes              pre-rRNA (Schmmang et al., 1989; Tollervey et al., <br /> require function large number factors,           1991, 1993). Ssblp, Garlp Soflp associated with <br /> ribosomal non-ribosomal, including RNAs, enzymes             subset snoRNAs important necessary for <br /> and proteins.                                              steps pre-rRNA processing (Clark et al., <br />    Factors required transcription rDNA, including          1990; Girard et al., 1992; Jansen et al., 1993). Nop3p <br /> RNA polymerase I associated transcription factors            (Npl3p) Nsrlp shown associated <br /> and topoisomerases, identified characterized       snoRNAs necessary ribosome biogenesis <br /> in (Ross et al., 1988; reviewed Reeder,           (Bossie et al., 1992; Kondo Inouye, 1992; Lee et al., <br /> 1990; Sollner-Webb Mougey, 1991). However,                1992; Russell Tollervey, 1992). Spb4p, Drslp, Dbp3p <br /> functions small number non-ribosomal          CA9 putative RNA dependent helicases required <br /> nucleolar molecules participate ribosome synthesis       assembly 40S 60S ribosomal subunits <br /> have investigated.                                          (Sachs Davis, 1990; Ripmaster et al., 1992; P.Weaver <br />  <br /> &#194;&#169; Oxford University Press                                                                                            3127 <br />  C.Sun J.L.Woolford, Jr <br />  <br /> and T.H.Chang, personal communication; C.O&apos;Day                        enriched nuclei (data shown). Because apparent <br /> J.Abelson, personal communication). RRP2 (NMEJ)                           size protein different previously <br /> encodes RNA moiety RNase MRP necessary                     identified yeast nucleolar protein, used 28.3a <br /> for processing pre-5.8S rRNA (Shuai Wamer, 1991;                   antibody screen kgtl 1 yeast genomic DNA expression <br /> Lindahl et al., 1992; Schmitt Clayton, 1993; Chu et                   library identify novel yeast nucleolar protein gene. <br /> al., 1994).                                                                  One positive clone identified screening 8 X 105 <br />    To investigate role nucleolar proteins               plaques. Western immunoblotting lysate Escher- <br /> ribosome biogenesis, isolated new yeast                         ichia coli transfected phage identified 180 kDa <br /> nucleolar protein gene, NOP4. We utilized monoclonal                    protein cross-reacted antibody 28.3a and <br /> antibody novel nucleolar antigen screen                    anti-p-galactosidase antibody (data shown).      The com- <br /> kgtll yeast genomic DNA expression library. Nop4p                      plete gene recovered probing library yeast <br /> an essential nucleolar protein containing RNA                     genomic DNA cloned YEp 13 yeast DNA amplified <br /> recognition motifs. Depletion Nop4p impairs pre-rRNA                   PCR phage clone. We verified single <br /> processing, particularly steps leading production              plasmid identified YEpl 3 library, YEp1 3NOP4, <br /> of 25S rRNA. Consequently, assembly 60S ribosomal                      contains gene encoding yeast nucleolar antigen by <br /> subunits disrupted. NOP4/NOP77 identified                     following criteria. (i) The intensity nucleolar staining <br /> in screen synthetic lethal <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span> mutation                 antibody 28.3a increased cells containing the <br /> in NOPI gene (Berges et al., 1994).                                   high copy plasmid YEp1 3NOP4 relative wild type cells <br />                                                                           (Figure 1, compare A D). (ii) A significantly greater <br />                                                                           100 kDa antigen detected by <br /> Results                                                                   immunoblot analysis extracts cells transformed <br /> Cloning NOP4                                                           YEpl3NOP4, compared wild type cells or <br /> Mouse monoclonal antibody 28.3a specifically stains                   YEp13 vector transformed cells (data shown). We <br /> yeast nucleolus (Figure IA D), identifiable                      named gene NOP4, nucleolar protein 4. <br /> crescent-shaped structure occupies approximately one-                NOP4 localized 10 kb yeast DNA insert <br /> third nucleus adjacent nuclear envelope                      YEpl3NOP4, determined comparison the <br /> (Sillevis-Smitt et al., 1973). Identical immunofluorescence               restriction map DNA original kgtl 1 <br /> staining patterns obtained antisera                 clone (Figure 2) Southern hybridization. <br /> bona fide yeast nucleolar proteins Noplp Nsrlp (data <br /> not shown). Western immunoblot analysis indicated                    Nop4p contains multiple RNA recognition motifs <br /> antibody 28.3a recognizes yeast polypeptide migrates               The nucleotide sequence NOP4 deduced amino <br /> on SDS-polyacrylamide gels apparent molecular                     acid sequence Nop4p shown Figure 3. The <br /> size 100 kDa present subcellular fractions                   sequence revealed single open reading frame could <br />  <br />            Anti-Nop4p                                          DAPI                                              DIC <br />  <br />  <br />  <br />  <br /> Fig. 1. Yeast Nop4p detected monoclonal antibody 28.3a localized nucleolus. SC252 (wild type) cells (A-C) JWY4905 cells <br /> (containing NOP4 high copy plasmid) (D-F) grown YEPGlu 30&#194;&#176;C density I x 108 cells ml, fixed. (A D) Cells <br /> incubated mouse monoclonal antibody 28.3a followed rhodamine-conjugated anti-mouse IgG antibody detect Nop4p antigen. (B E) <br /> Cells treated 4&apos;,6 diamidino-2-phenylindole (DAPI) visualize nuclear mitochondrial DNA. (C F) Cells viewed Nomarski DIC <br /> optics. The exposure time rhodamine fluorescence images SC252 cells (A) 5-fold longer JWY4905 cells (D), order to <br /> obtain comparable signal intensity. <br />  <br /> 3128 <br />                                                                                                       Nucleolar protein required ribosome biogenesis <br />  <br /> encode 685 amino acid protein predicted molecular               respectively (data      shown), predicted the <br /> mass 77 824 Da isoelectric point 9.8. Nop4p                 restriction map YEp352NOP4 (Figure 2). NOP4 was <br /> contains RNA recognition motifs (RRMs, called                  localized left arm chromosome XVI, between <br /> RBDs RNP-80). This motif                       PEP4 RA4D], hybridization NOP4 DNA <br /> RNA binding proteins, including number nucleolar                    blot electrophoretically separated yeast chromo- <br /> proteins (reviewed Bandziulis et al., 1989; Mattaj,                   somes subsequently grid cloned overlapping <br />  1989; Scherly et al., 1989; Keene Query, 199 1; Kenan               genomic restriction fragments (data shown) (Link and <br /> et al., 1991; Dreyfuss et al., 1993). Three tandem RRMs                  Olson, 1991; Riles et al., 1993). <br /> are present amino-terminal half Nop4p,                    A null allele NOP4 created removal 1 <br /> amino acids 28 358. A fourth incomplete RRM                       kb Spel-ClaI fragment containing 5&apos; 323 codons of <br /> present amino acids 533 574. A short acidic                  NOP4 open reading frame replacement this <br /> domain (14/18 amino acids aspartate glutamate                     fragment yeast TRPJ gene (Figure 2B). One <br /> residues) present region second                copy NOP4 diploid JM749 replaced the <br /> third RRMs, amino acids 250 267. The carboxy-                    nop4::TRPJ allele transformation homologous <br /> terminal domain Nop4p contains charged                           recombination (Rothstein, 1983). The expected gene <br /> residues; 24 68 amino acids arginines                 replacement confirmed genomic Southern <br /> lysines, 13 glutamate aspartate residues. A                   hybridization (data shown). The resulting heterozygote, <br /> diagram motifs Nop4p shown Figure 4.                         JWY49 14, NOP4Inop4::TRPJ genotype, sporulated <br />    The discrepancy apparent molecular size                   40 tetrads dissected. Thirty tetrads gave two <br /> of Nop4p determined SDS-PAGE (100 kDa)                         viable spores ascus, 10 gave viable <br /> the predicted size (78 kDa) accounted                  spore. All viable spores Trp-. To eliminate <br /> presence RRMs. Many proteins containing RRMs                      possibility NOP4 essential spore <br /> migrate slowly predicted sizes (Wilk <br /> et al., 1985; Adam et al., 1986; Sachs et al., 1986; Jong <br /> et al., 1987; Lapeyre et al., 1987; Bossie et al., 1992; <br /> Lee et al., 1992; Russell Tollervey, 1992). <br /> NOP4 single copy essential gene <br /> Genomic Southern blot analysis indicated NOP4                                                                                                                                                                  7. <br />  <br /> single copy gene. A 2.2 kb EcoRI fragment containing                     7               AA.2%7 A.A. <br /> most NOP4 coding sequence hybridized single                                                                                                            ..U.           &quot;.7   &apos;7           &apos;7A.A          T71 <br />  <br />  <br />  <br /> Pstl genomic DNA fragment, ClaI genomic DNA                              p     N         N <br />                                                                                                                    7        7A..A- 7&apos;.: <br />                                                                                                                                F                  F <br />  <br /> fragments Bglll genomic DNA fragments,                                 -=:A71;             .AA                     -TTA <br />  <br />  <br />                                                                                  ......... <br />  <br />  <br />  <br />   A <br />                 sc~                                             Sa <br />   P      R     S-1       B           C-   E3R     S   -                                                                    A,:::A_A. M:AA_.--                             .7 .................1: -AA. <br />                                                                                                                                                                                     V F V F N                 V <br />                                                                                                                                                                                                                      A    7 <br />  <br />  <br />  <br />                                                                                              ALAIT     7        7 7 A.   A A-A.7 7        A       77       7 A.- <br />  <br />  <br />  <br />  <br />                         -. .-11 Cl-re- <br />                                                                                                                                                                                                         A. T,7. 7 <br />  <br />  <br />  <br />  <br />                                                                                          A   A,:           1A              t 1: AAA,-:.::             A.                  A A A              A   A             7A <br />  <br />  <br />  <br />  <br />   B <br />                 SsSa <br />   P       R    Sp        B           C    B R     Sc                 P <br />  <br />  <br />                              NOP#4 <br />  <br />  <br />  <br />  <br />                                                                             Fig. <br />                                                                                3. <br />                                                                                 Nucleotie sequence <br />                                                                                                 of- NP4 and7prdicted       ainoaci <br />                                                                             sequenceof  Nop4p.Thenucleotide sequence isshown line <br />                                                                           deduedmin acid sequenc sigeletrcoei7sono                  A <br />  <br />  <br />  <br />                                                                          -th botmA <br />                                                                               ieA Te N rcgnto motifs (RMs) ar boxe the~ <br />               RPI------------ <br />               I                                                             conserved <br />                                                                                    octamer sequence RNP ~~~~andhex.7- Ame euneRP r <br />                                                                             shaded, <br />                                                                                  potential nuclear loaiztoseuneFr <br />                                                                              unelnd <br />                                                                                  Tha                        eBn                 ceso nube NOP4. isU0.59- <br /> Fig. 2. Restriction map yeast genomic DNA containing NOP4 and <br /> map nop4::TRPJ disruption allele. (A) Restriction map the <br /> 6.6 kb Pstl fragment yeast genomic DNA cloned plasmid <br /> YEp352NOP4. (B) nop4::TRPJ null allele. A 1009 bp Spel-Clal <br /> fragment containing 5&apos; half NOP4 gene replaced with <br /> the TRPJ gene, shown. The direction transcription NOP4 is <br /> indicated arrowhead. B, Bg1II; C, Clal; P, Pstl; R, EcoRI; Sa, <br /> Sacl; Sc, ScaI; Sp, Spel; Ss, Sspl. <br />  <br />                                                                                                                                                                                                                               3129 <br />  C.Sun J.L.Woolford, Jr <br />  <br />                    N   1-   .[    .1                                                                      ... <br />  <br />  <br />  <br />                                  RRM1       RRM?      Acidic   RRM3                      RRM z&apos; <br />  <br /> Fig. 4. Schematic motifs Nop4p. The Nop4 protein contains RRMs containing RNPI (M) RNP2 (EC), flanking imperfect RRM, and <br /> a partial RRM containing RNP1 octamer sequence. There short acidic domain (U) RRM2 RRM3, basic domain (X) <br /> near carboxy-terminus Nop4p. <br />  <br /> germination, constructed haploid strain JWY4917,                   A 4 <br /> which contains nop4::TRPI chromosome XVI and <br /> NOP4 plus URA3 plasmid pRS316NOP4. This strain <br /> was unable grow medium containing 5-fluoroorotic                     C    3 <br /> acid (5-FOA). Taken together, results indicate                 0 <br /> NOP4 essential vegetative growth yeast. <br />                                                                                 2 <br />                                                                           0 <br /> Construction conditional lethal null allele                        1 <br />  <br /> NOP4 <br /> In order examine function Nop4p, strain JWY4919 <br /> was constructed, functional copy of <br /> NOP4 plasmid-borne GAL]-NOP4 promoter fusion                               0 <br /> that expressed galactose-containing medium (YEP-                              0     10      20     30         40      50 <br /> Gal) glucose-containing medium (YEPGlu).                                               Time (hours) <br /> JWY4919 grows doubling time 3.2 h YEPGal, <br /> similar wild type NOP4 strains (Figure 5A data                               B <br /> not shown), indicating GALJ-NOP4 construct is <br /> fully functional. Within 30 min shifting JWY4919                                                                        :    Nop 4p <br /> from YEPGal YEPGlu, NOP4 mRNA detected <br /> by Northern analysis (data shown). Western immuno- <br /> blot analysis indicated Nop4p decreased the <br /> shift YEPGlu; 15 h Nop4p detected                                         0 1/2h 1 h 2h 4h t5h <br /> (Figure SB). JWY4919 divided 2 h                     Fig. 5. Yeast cells stop dividing depletion Nop4p. (A) Growth <br /> the 15 h shifting YEPGal YEPGlu.                    curve JWY4919 (GALJ-NOP4) grown YEPGal medium after <br /> Fifteen hours medium shift, doubling time                 shift YEPGlu medium. Data represented log OD,/ODt = 0, <br /> of JWY4919 began increase significantly (Figure SA).                 t time hours shifting medium. (B) Depletion of <br /> This diminished growth termination NOP4 expres-                 Nop4p switching cells YEPGlu medium. Cell extracts prepared <br />                                                                         culture used growth curve assayed by <br /> sion reversible; -80% JWY4919 cells grown                    Western immunoblot analysis using monoclonal antibody 28.3a. Equal <br /> in YEPGlu 40 h able form colonies plated                 amounts protein loaded lane, judged by <br /> on YEPGal medium, GALI-NOP4 expressed. No                      immunodetection ribosomal protein Ll (data shown). The <br /> colonies formed cells plated YEPGlu                        second, rapidly migrating band detected antibody 28.3a <br /> medium. Thus, cells stopped dividing                  probably degradation product 100 kDa Nop4 polypeptide; it <br />                                                                         absent extracts prepared proteinase deficient yeast <br /> not dead resume growth Nop4p provided.                  strains BJ2168 BJ3501 (data shown). <br />  <br /> Depletion Nop4p results deficit 60S                          peaks sedimenting quickly 80S monoribosomes <br /> ribosomal subunits                                                      polyribosomes evident extracts JWY4919 <br /> To determine Nop4p essential production                  12 h shift YEPGal YEPGlu (Figure 6D, <br /> of ribosomes, amounts 40S 60S ribosomal                      arrows). These peaks represent halfmer polyribosomes <br /> subunits, 80S monoribosomes, polyribosomes                     contain mRNAs associated integral number <br /> assayed depletion Nop4p. JWY4917 containing                     ribosomes plus stalled 48S preinitiation complex. <br /> the low copy plasmid pRS316 bearing NOP4                      Such halfmers accumulate 60S subunit assembly is <br /> control promoter used positive control.             aborted (Rotenberg et al., 1988; Ripmaster et al., 1992; <br /> Extracts prepared JWY4917 JWY4919 grown                        Deshmukh et al., 1993; Goyer et al., 1993). The deficit <br /> in YEPGal shifted YEPGlu 4 h 12 h                    60S ribosomal subunits JWY4919 pro- <br /> fractionated sucrose velocity gradient centrifugation.               nounced 12 h shift 4 h, parallel with <br /> Assaying amounts free 40S 60S ribosomal subunits                 decrease Nop4p (Figure 5B). These <br /> by means proven sensitive measure                   effects amounts 60S ribosomal subunits probably <br /> defects ribosome assembly (Moritz et al., 1990, 1991;                direct result depletion Nop4p, being <br /> Ripmaster et al., 1992, 1993; Deshmukh et al., 1993).                   indirect result impaired growth rate, 60S <br /> When expression NOP4 terminated JWY4919,                      ribosomal subunits diminished noticeable <br /> fewer free 60S ribosomal subunits accumulated relative                  decrease rate division JWY4919 cells. <br /> to free 40S subunits (Figure 6C D). The relative <br /> amounts 40S 60S subunits unchanged                       Nop4p required processing pre-rRNA <br /> JWY4917, Nop4p continued synthesized                     We investigated deficit 60S ribosomal <br /> in wild type amounts (Figure 6, A B). Additional                    subunits observed depletion Nop4p resulted from <br />  <br /> 3130 <br />                                                                                                                                                _~ - <br />                                                                                         Nucleolar protein required ribosome biogenesis <br />  <br />                       A                     JWY 4917                       B                                   JWY 4917 <br />                            1.0                                                 1.0 <br />  <br />  <br />  <br />  <br />                       EC                                                   E <br />  <br />                      CD    0.5 <br />                                                                           0.5 <br />                                                                            o) <br />                      co <br />  <br />  <br />  <br />  <br />                            0.0 <br />                                                                Bottom          0.0 <br />                                                                                                                                         Bottom <br />  <br />                       C                    JWY 4919                       D                                    JWY 4919 <br />                            1.0                                                 1.0 <br />  <br />  <br />  <br />  <br />                       E                                                    E <br />                                                                            C <br />                       c,                                                  o    0.5 <br />                       co. <br />                                                                           co <br />                                                                    VV <br />  <br />  <br />  <br />  <br />                            0.0 <br />                                                             B( ottom           0.0 <br />                                                                                                                                         Bottom <br /> Fig. 6. Depletion Nop4p results deficit 60S ribosomal subunits. Polyribosome profiles shown JWY4917 (NOP4) JWY4919 <br /> (GALI-NOP4). Cells shifted YEPGal YEPGlu 4 h (A C) 12 h (B D). Free ribosomal subunits polyribosomes in <br /> cell extracts separated 7-47% sucrose gradients. Peaks representing 40S 60S ribosomal subunits 80S monoribosomes labelled <br /> and halfmer polyribosomes indicated vertical arrows. <br />  <br /> defects synthesis processing pre-rRNA. JWY4919                                                   Gal                    Glu 4h Glu 12h            NOP4 <br /> cells grown YEPGal shifted YEPGlu                        Chase (min)     0      3       10        20 30   0     3 10 0     3   10   0    3   10 <br />  <br /> 4 h 12 h, pulse-labelled 3 min L-                               35S                                                    5                                35S <br /> ([3H]methyl)methionine, chased 2000-fold                              32S    -                                    @                         r             -   32s <br /> excess cold methionine 3 10 min. Synthesis and <br />                                                                                                                                                                          27S A <br /> processing rRNA JWY4919 grown YEPGal                                                                          00                      a <br />                                                                                                                                                                      - <br />  <br />                                                                                                                                                                          27S B <br /> the NOP4 strain JWY4917 grown YEPGlu                                    27S A <br />                                                                                 27S B   - <br />                                                                                                                              ..   Osl                  ..   l| w         25S <br /> identical previously observed wild type patterns                              25S <br />                                                                                                           *am <br />  <br />  <br /> (Figure 7). Four hours shifting YEPGal to <br />                                                                                         - <br />                                                                                                           0-me <br /> YEPGlu, synthesis processing pre-rRNAs                                                                             I           AN   ij&amp; <br />                                                                                                                                                        m             -   20s <br /> JWY4919 similar pattems observed these <br /> above-mentioned wild type controls (Figure 7). The 35S <br />                                                                                                                                     6 -, s 0                      j- 18S <br />                                                                                  20S    - <br />                                                                                                 .11. q: <br />                                                                                                           .:   4 <br /> rRNA primary transcript, 32S pre-rRNA processing                                                                 lb <br />                                                                                  18S                           .4-   a <br /> intermediate, 27SA 27SB precursors mature <br /> 25S rRNA 20S precursor 18S mature rRNA <br /> were detected 3 min pulse, small amounts <br /> of mature 25S 18S rRNAs. After 3 min 10 <br /> min chase, majority rRNAs detected mature                      Fig. 7. Depletion Nop4p impairs pre-rRNA processing blocks <br /> 25S 18S rRNA. Only small amounts 27S 20S                        accumulation 25S rRNA. Synthesis accumulation pre-rRNA <br /> processing intermediates evident 10 min                     assayed JWY4919 grown YEPGal 4 h 12 h after <br /> chase. In experiments, processing intermediates                   shifting YEPGal YEPGlu medium, JWY4917 <br />                                                                            grown YEPGlu medium. Cells pulse-labelled 3 min with <br /> were detected 20 30 min chase (data shown). <br />                                                                            L-([3H]methyl)methionine, chased 3 10 min 2000- <br /> In contrast, 12 h shifting JWY4919 YEPGlu,                        fold excess unlabelled methionine. Data JWY4919 grown in <br /> rRNA processing severely perturbed (Figure 7). The                     YEPGal medium obtained separate experiment. The rRNA <br /> 35S pre-rRNA 32S, 27SA 20S processing                          precursor, processing intermediates mature rRNAs labelled. <br />  <br />  <br />  <br />                                                                                                                                                                          3131 <br />  C.Sun J.L.Woolford, Jr <br />  <br />                       Gal            Glu4h         Glu 12h           nuclear localization sequences (NLSs) required entry <br />                                                                      proteins nucleus (reviewed Garcia-Bustos <br />   Chase (min&quot; 0       3 10 30 0       3 10 30 0 3 10 30              et al., 1991). The sequence KRrreqlkqartKRTR (618-633) <br />        35S-                                                          resembles bipartite nuclear targeting sequence (Robbins <br />        32S -                                                         etal., 1991). <br />                                                                         The pronounced feature Nop4p presence <br />                                                                      RNA recognition motifs, tandemly arranged <br />        27S A-                                                        amino-terminal half protein, fourth <br />        27S B -                                                       truncated RRM located carboxyl-terminal half of <br />          25S-                                                        Nop4p. Within -80-100 amino acid long canonical <br />                                                                      RRM highly conserved octapeptide RNP1 the <br />                                                     A.E4. <br />                                                                      slightly conserved hexapeptide RNP2 located 30-40 <br />                                                                      amino acids amino-terminal RNP1. There 10 other <br />                                                                      conserved individual residues located specific positions <br />          20S- <br />                                                                      RRMs (Bandziulis et al., 1989; Mattaj, 1989; <br />                                                                      Keene Query, 1991; Kenan et al., 1991; Dreyfuss et <br />          18S-                                                        al., 1993). Structural studies RRM-containing <br />                                                                      proteins RRM contains kappaj structure, <br />                                                                      ,B strands form antiparallel 3 sheet, <br /> Fig. 8. Nop4p required pre-rRNA processing accumulation   RNP2 RNP1 located inner P1 P3 <br /> of 25S rRNA. Cells JWY4919 grown YEPGal 4 h 12 h <br /> after shifting YEPGlu medium pulse-labelled 2 min   strands f sheet (Nagai et al., 1990; Hoffman et al., <br /> [3H]uracil, chased 3, 10 30 min excess unlabelled    1991; Wittekind et al., 1992). <br /> uracil. The rRNA precursor, processing intermediates mature          RRM1 RRM3 Nop4p contain prototypical <br /> rRNAs labelled.                                                  RNP1 RNP2 sequences, overall quite similar <br />                                                                      sequence number RRMs. Both Chou- <br /> intermediates, small mature 18S               Fasman (Chou Fasman, 1978) Gamier-Robson <br /> rRNA, detected 3 min pulse, 27SB                  methods (Gamier et al., 1978) predict RRM1 and <br /> rRNA precursor detected. After 10 min chase,              RRM3 Nop4p prototypical Papa structures, in <br /> precursors intermediates detected, little           RNP1 octamer RNP2 hexamer contained <br /> mature 25S rRNA accumulated. However, substantial                    ,3 ,B1 strands. <br /> amounts 18S rRNA produced. The ratio newly                    RRM2 Nop4p somewhat imperfect. The highly <br /> synthesized 25S rRNA 18S rRNA detected 10                 conserved Lys/Argl Phe/Tyr8 RNP1 Cys and <br /> min chase cells shifted 12 h YEPGlu 15-fold            Met, respectively, conserved Phe/Tyr2, Gly/ <br /> lower cells grown YEPGal shifted YEPGlu             Lys4 Leu6 RNP2 Ile, Arg Met, respectively, <br /> for 4 h.                                                             RRM2. Although overall sequence similarity the <br />    To exclude possibility impaired production            remaining 69 amino acids RRM family reasonably <br /> 25S rRNA resulted effects methylation                 high (-50%), predicted secondary structure RRM2 <br /> rRNA, pulse-chase experiments performed using              does strong match canonical <br /> [3H]uracil. Identical results obtained; amounts              RRMs. In particular, RNP1 predicted an <br /> of 25S rRNA significantly diminished relative                 a-helix. Thus, RRM2 Nop4p diverged from <br /> 18S rRNA Nop4 protein depleted (Figure 8).                   functional RRM. <br /> Thus deficit 60S ribosomal subunits observed              RRM4 Nop4p does completely match the <br /> upon depletion Nop4p resulted impaired pro-                   sequence canonical RRMs. It contains RNP1 <br /> cessing pre-rRNA reduced production mature                  sequence matches consensus perfectly, there <br /> 25S rRNA.                                                             20 residues amino-terminal 14 residues <br /> Discussion                                                            carboxy-terminal RNP1 similar in <br />                                                                       RRMs. There RNP2 sequence RRM4. <br /> We identified characterized novel yeast                       The predicted RNA recognition motifs Nop4p, as <br /> nucleolar protein gene, NOP4. Nop4p essential                   requirement Nop4p pre-rRNA processing, <br /> protein necessary production 25S rRNA                  suggest Nop4p associated snoRNAs, or <br /> 60S ribosomal subunits.                                               bind rRNA precursors, processing intermediates, <br />    Nop4p contains different structural modules,            products. Because Nop4p contains multiple RRMs, it <br /> observed nucleolar proteins. There acidic              contact RNA molecule simul- <br /> domain amino acids 250 267 basic                    taneously. Preliminary co-immunoprecipitation experi- <br /> domain carboxy-terminus,                   ments antibody 28.3a Nop4p failed to <br /> function surfaces interaction molecules.            detect U3, U14 snR0 snoRNAs precipitation <br /> Unlike nucleolar proteins, Nop4p does                  complexes. When extracts prepared cells <br /> contain extended glycine- arginine-rich (GAR)                  irradiated UV light crosslink RNA protein, <br /> domain (summarized Girard et al., 1992). Embedded                  snR 190 co-immunoprecipitated 28.3a antibody <br /> within basic carboxy-terminal domain Nop4p             (C.Sun J.Woolford, unpublished data). Thus, Nop4p <br /> sequences KRRR (618-621), KRTR (630-633), KKPKK                       loosely associated component putative <br /> (648-652) KRKRK (679-683), resemble                         snoRNP containing snR190, indirectly <br />  <br />  <br /> 3132 <br />                                                                                                 Nucleolar protein required ribosome biogenesis <br />  <br />                   l1S            5.8S             25S <br /> 35S                                                                     p <br />                                                                                    Table I. Yeast strains <br />  <br />                                                                                    Strain         Genotype                                   Source <br /> 32S? [ <br />                    _~~~~~J                  ________________ <br />                                                                                    JM749          MATa/MATa ura3-52/ura3-52 trpl-            J.Maddock <br />                                                                                                   AJOJ/trpl-A101 Iys2-80111ys2-801 <br />                                                                                                   leu2-AI/leu2-AJ his3-A200/his3-A200 <br />                                                                                    SC252          MATa ura3-52 adel leu2-3,112 GAL2          J.Hopper <br /> 20S                                                           mmmmv%^^, 27SA                                                                 This study <br />                                                                                    JWY4905        MATa ura3-52 adel leu2-3,112 GAL2 <br />                                                                                                   pYEp352NOP4 <br />                                                      I_                            JWY4914        MATa/MATTa ura3-521ura3-52 trpl- <br />                                                                                                   AJOJ/trpl-AIOI lys2-8011lys2-801 <br />                                                                                                                                              This study <br />                                                                             27SB <br />                                                                                                   leu2-AJl1eu2-AI his3-A200/his3-A200 <br />                                                                                                   NOP41nop4::TRPI <br />                                                                                    JWY4916        MATa/MATa ura3-521ura3-52 trpl-            This study <br />                                                                                                   A1011trpl-A101 lys2-8011lys2-801 <br />  <br />  <br />                                   Fl <br />                            7S <br />                                                                                                   leu2-AJ/leu2-A his3-A200/his3-A200 <br />  <br />                                                      l                             JWY4917 <br />                                                                                                   NOP41nop4::TRPI pRS316NOP4 <br />                                                                                                   MATa ura3-52 trpl-A101 lys2-801 <br />                                                                                                   leu2-AJ his3-A200 nop4::TRPI <br />                                                                                                   pRS3 16NOP4 <br />                                                                                                                                              This study <br />  <br />                   18S           5.8S              25S <br />                                 |Rrp2| <br />                                                                                    JWY4919        MATa ura3-52 trpl-A101 Iys2-801            This study <br />                                                                                                   leu2-AI his3-A200 nop4::TRPI <br />           Garip NJopip U3 U14                   Spb4p Nopip                                       pUN9OGAL I -NOP4 <br />                   Nop3p ~               ~ <br />           Nsr1p C9p3 snR130 <br />  <br />                                                                                    rRNA processing consistent deficiency 60S <br /> Fig. 9. Yeast small nucleolar RNAs nucleolar proteins                     ribosomal subunits observed depletion Nop4p. A <br /> function different steps pre-rRNA processing. <br />                                                                                    detailed examination required identify the <br />                                                                                    precise step steps pre-rRNA processing which <br /> associated snR 190 larger macromolecular struc-                            Nop4p necessary. <br /> tures nucleolus. <br />    Some yeast nucleolar proteins                     required multiple     Materials methods <br /> steps pre-rRNA processing biogenesis the <br /> 40S 60S ribosomal subunits, nucleolar                            Strains, plasmids media <br /> proteins necessary steps pre-rRNA processing                            Strains used study described Table I. Yeast cells were <br /> leading assembly 40S 60S ribosomal                             grown 30&#194;&#176;C YEPGlu (I % yeast extract, 2% peptone, 2% glucose), <br /> subunits, (Figure 9). Depletion Noplp                           YEPGal (1% yeast extract, 2% peptone, 2% galactose) synthetic <br />                                                                                    medium (Sherman et al., 1986) containing glucose galactose <br /> Nop3p results delayed processing 35S primary                             carbon source. Methionine uracil omitted synthetic <br /> rRNA transcript, diminished production rRNA                                medium used pulse-chase experiments increase efficiency of <br /> processing intermediates mature 18S rRNA,                                  radioactive labelling. Yeast cells transformed DNA using the <br /> eventually deficit mature 25S rRNA (Tollervey et                              lithium acetate method (Ito et al., 1983). Yeast strains SC252 and <br /> al., 1991; Russell Tollervey, 1992). Depletion                              JWY4905 used indirect immunofluorescence microscopy. <br />                                                                                    JWY4905 derived SC252 transformation high <br /> inactivation Garlp, Soflp Nsrlp results increased                         copy plasmid YEpI3NOP4. <br /> accumulation 35S pre-rRNA, diminished                                      A 6.6 kb PstI DNA fragment containing NOP4 isolated from <br /> amounts 32S, 27SA 20S processing inter-                                 plasmid YEpl3NOP4 subcloned plasmid vector YEp352 (Hill <br /> mediates mature 18S rRNA. However, production                               et al., 1986) generate YEp352NOP4. Plasmid pRS316NOP4 was <br />                                                                                    constructed subcloning 6.4 kb BamHI-SacI DNA fragment isolated <br /> both 27SB precursor 25S rRNA mature                                 YEp352NOP4 plasmid pRS316 (Sikorski Hieter, 1989). <br /> 25S rRNA significantly decreased (Girard et al.,                               A nop4::TRPI null allele lacking 5&apos; half NOP4 constructed <br /> 1992; Lee et al., 1992; Jansen et al., 1993). Depletion                         shown Figure 2B. YEp352NOP4 digested SpeI ClaI. <br /> inactivation Spb4p, Drs lp Nop4p impairs production                          The overhanging 5&apos; ends filled Klenow enzyme 1.3 <br /> of 25S rRNA, 18S rRNA (Sachs Davis, 1990;                              kb PvuII fragment containing yeast TRPI gene (Jones Prakash, <br />                                                                                    1990) ligated filled SpeI ClaI sites. The resulting <br /> Ripmaster et al., 1992). Unlike case cells depleted                        construct digested BglI. The 10.5 kb Bgll fragment containing <br /> of Noplp, Garlp, Soflp, Nop3p Nsrlp,                                nop4::TRPI allele gel purified used replace copy of <br /> increase 35S pre-rRNA decrease                              NOP4 diploid strain JM749 transformation homologous <br /> 27SA 20S rRNA processing intermediates mature                                recombination (Rothstein, 1983), create diploid strain JWY4914. <br />                                                                                       In order construct JWY4917 containing genomic nop4::TRPI <br />  18S rRNA depletion Nop4p. Rather, depletion                            null allele wild type NOP4 low copy plasmid pRS316, <br /> Nop4p results decrease the 27SB                              JWY4914 transformed plasmid pRS316NOP4 containing <br /> rRNA processing intermediate significant decrease                            URA3. The resulting strain JWY4916 sporulated tetrads dis- <br /> in mature 25S rRNA. This suggests                               sected. A Trp+ Ura+ haploid spore clone (JWY4917) selected. <br />                                                                                       The GALI-NOP4 promoter fusion constructed follows. A 2.3 <br /> Nop4p functions rRNA processing ribosome                                     kb SspI-ScaI DNA fragment contains <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> NOP4 gene <br /> assembly steps formation 27SA processing                              (Figure 2) subcloned SmaI site plasmid L1-1 18, con- <br /> intermediate, distinct requiring Noplp,                          structed laboratory subcloning EcoRI-SalI fragment of <br /> Nop3p, Garlp, Nsrlp Soflp. The defect pre-                                  plasmid pBM258T (Johnston Davis, 1984) containing GAL] <br />  <br />  <br />                                                                                                                                                       3133 <br />  C.Sun J.L.Woolford, Jr <br />  <br /> promoter plasmid pUN90 (Elledge Davis, 1988). This plasmid         Fractionation characterization ribosomes <br /> containing GALJ-NOP4 promoter fusion yeast HIS3 gene            Ribosomal subunits, ribosomes polyribosomes assayed by <br /> was transformed haploid strain JWY4917. Transformants             sucrose gradient centrifugation cell extracts, described previously <br /> plated medium containing 5-FOA galactose select loss       (Deshmukh et al., 1993), yeast cells grown 100 ml <br /> plasmid pRS316NOP4. The resulting strain JWY4919 contains          medium 40 A260 units cell lysate layered 35 ml <br /> functional NOP4 gene, GALI-NOP4 conditional allele.                     linear 7-47% sucrose gradient. <br /> Screening akgtll yeast genomic expression library                     Pulse-chase analysis pre-rRNA processing <br /> YEp13 yeast genomic library                                                 The synthesis, processing, turnover pre-rRNA assayed by <br /> Akgtl 1 yeast genomic DNA expression library (a gift Dr Robert         pulse-chase analysis using L-([3H]methyl)methionine, described in <br /> Hamatake, NIEHS) immunoscreened yeast nucleolar                Deshmukh et al. (1993). Similar procedures followed pulse- <br /> specific monoclonal antibody 28.3a (a gift Drs Connie Copeland         chase experiments using [3H]uracil, JWY4919 containing <br /> and Michael Snyder, Yale University) using protocol Young        pRS316 vector bearing URA3 used, enable growth synthetic <br /> Davis (1991). Phage plated -5 X 104 plaques 150 mm plate.       complete medium lacking uracil. Cells pulse-labelled 50,uCi <br /> A 10-fold dilution 28.3a cell line culture supernatant used     [3H]uracil (51 Ci/mmol) ml 2 min chased 30 min <br /> the primary antibody probe. This primary antibody detected           300,ug/ml cold uracil. Yeast RNA subjected electrophoresis <br /> alkaline phosphatase-conjugated secondary antibody (Promega Corp.,          24 cm long gels 6.5 h 200 V. <br /> Madison, WI), following manufacturer&apos;s instructions. <br />    Genomic DNA containing portion NOP4 cloned kgtl 1            Indirect immunofluorescence microscopy <br /> amplified PCR standard conditions (Sherman et al., 1989),          Indirect immunofluorescence microscopy performed cells grown <br /> using pair oligonucleotides (CS 1: 5&apos;-GGTGGCGACGACTCCTGG-              YEPGlu, described Copeland Snyder (1993). Purified 28.3a <br /> AGCCCG-3&apos; CS2: 5&apos;-TTGACACCAGACCAACTGGTAATG-3&apos;),                         monoclonal antibody used concentration 20 .g/ml. The <br /> complementary sequences flanking EcoRI site kgtl . The        secondary antibody used rabbit anti-mouse immunoglobulin G <br /> PCR product digested EcoRI remove phage DNA, labelled           conjugated rhodamine (Cappel, Organon Teknika Corp., Durham, <br /> with 32P random primer reactions (Feinberg Vogelstein, 1984)         NC). Slides viewed using Zeiss Axiophot microscope equipped <br /> and used colony hybridization probe screen library yeast       100 X objective lens Nomarski DIC optics. Photographs <br /> genomic DNA cloned plasmid YEp13, described Gergen et              taken CCD camera (Photometric Inc., Tuscon, AZ) under <br /> al. (1979).                                                                 constant illumination conditions. Photographs transforming <br />                                                                             digital image Kodak Tmax-100 film using a <br /> DNA RNA methods                                                         Kodak slidewriter. <br /> Extraction yeast DNA RNA, Southern Northern hybridization, <br /> and construction analysis recombinant DNA molecules were <br /> performed described Sambrook et al. (1989). DNA sequence          Acknowledgements <br /> determined dideoxy nucleotide termination method (Sanger et          We thank Drs Connie Copeland Michael Snyder (Yale University) <br /> al., 1977), using single-stranded DNA templates cloned Bluescript    generous gift monoclonal antibody 28.3a directed a <br /> plasmid (Stratagene, La Jolla, CA). Sequenase version 2.1 used      yeast nucleolar protein, Dr Robert Hamatake (NIEHS) kgtl I <br /> sequencing reactions, following manufacturer&apos;s instructions (USB        expression library. We grateful Dr Greg Fisher Microscope <br /> Corp., Cleveland, OH). Sequence analysis, database searches protein     Imaging Facility, Carnegie Mellon University, help microscopy. <br /> structure predictions performed using Sequence Analysis            Thanks Drs Michael Cusick (Texas A &amp; M University), Pat O&apos;Conner <br /> Software Package programs developed Genetics Com-           Christopher Burd (University Pennsylvania) useful discussions <br /> puter Group University Wisconsin Biotechnology Center             RNA recognition motifs. We thank colleagues laboratory <br /> (Devereux et al., 1984). Computer software accessed         valuable discussions suggestions course work. <br /> courtesy Pittsburgh Supercomputing Group.                            Thanks Drs Elizabeth Jones, Javier L6pez Carol Woolford and <br />                                                                             colleagues laboratory critical reading manuscript. <br /> Chromosome mapping NOP4                                                  This study supported Public Health Service Grant GM28301 <br /> A blot electrophoretically separated yeast chromosomes            J.L.W. <br /> ordered library yeast genomic fragments cloned bacteriophage X <br /> and cosmids (a gift Drs L.Riles M.Olson, Washington University <br /> and University Washington) hybridized 32P-labelled <br /> 2.2 kb EcoRI DNA fragment containing NOP4.                                  References <br /> Growth curve viability tests                                            Adam,S.A., Nakagawa,T.Y., Swanson,M.S., Woodruff,T. Dreyfuss,G. <br /> Strain JWY4919 growing exponentially 50 ml YEPGal medium                (1986) Mol. Cell. Biol., 6, 2932-2943. <br /> harvested centrifugation 4000 g 5 min, washed        Bandziulis,R.J., Swanson,M.S. Dreyfuss,G. (1989) Genes Dev., 3, <br /> equal volume YEPGlu medium remove galactose, inoculated              431-437. <br /> into 50 ml YEPGlu medium. Growth viability strain         Berges,T., Petfalski,E., Tollervey,D. Hurt,E. (1994) EMBO J., 13, <br /> YEPGal medium shifting YEPGlu medium assayed              3136-3148. <br /> previously described (Moritz et al., 1991). In order maintain cultures   Borer,R.A., Lehner,C.F., Eppenberger,H.M. Nigg,E.A. (1989) Cell, <br /> in exponential growth, cells diluted fresh medium           56, 379-390. <br /> the OD610 reached 0.7 units.                                                Bossie,M.A., DeHoratius,C., Barcelo,G. Silver,P. (1992) MoI. Biol. <br />                                                                                Cell, 3, 875-893. <br /> Western immunoblot analysis                                                 Bugler,B., Bourbon,H., Lapeyre,B., Wallace,M.C., Chang,J.H., <br /> Western immunoblotting performed previously described (Moritz           Amalric,F. Olson,M.O.J. (1987)          J. Biol. Chem., 262, <br /> et al., 1991), yeast cell lysates prepared boiling         10922-10925. <br /> cells (109 cells ml) directly Laemmli sample buffer (65 mM Tris,     Caizergues-Ferrer,M., Mariottini,P., Curie,C., Lapeyre,B., Gas,N., <br /> pH 6.8, 10% glycerol, 1% SDS, 1% P-mercaptoethanol) 5 min,             Amalric,F. Amaldi,E (1989) Genes Dev., 3, 324-333. <br /> vortexing hard times 30 s equal volume glass beads       Chou,P.Y. Fasman,G.D. (1978) Adv. Enzymol., 47, 45-148. <br /> to minimize degradation Nop4p. The cell       lysate subjected    Chu,S., Archer,R.H., Zengel,J.M. Lindahl,L. (1994) Proc. NatlAcad. <br /> electrophoresis 10% SDS-polyacrylamide gels, electrophor-              Sci. USA, 91, 659-663. <br /> etically transferred BA-S 85 nitrocellulose membranes (Schleicher &amp;      Clark,M.W., Yip,M.L., Campbell,J. Abelson,J. (1990) J. Cell Biol., <br /> Schuell, Keene, NH). The blots incubated 28.3a antibody        111, 1741-1751. <br /> for 2 h room temperature, washed times TBST          Copeland,C. Snyder,M. (1993) Yeast, 9, 235-249. <br /> solution (10 mM Tris, pH 8.0, 150 mM NaCl, 0.05% Tween 20)               Deshmukh,M., Tsay,Y.-F., Paulovich,A.G. Woolford,J.L.,Jr (1993) <br /> remove free primary antibody. The Nop4p antigen-antibody complexes            Mol. Cell. Biol., 13, 2835-2845. <br /> were detected alkaline phosphatase-conjugated secondary antibody         Devereux,J., Haeberli,P. Smithies,O. (1984) Nucleic Acids Res., 12, <br /> as described above.                                                            387-395. <br />  <br />  <br /> 3134 <br />                                                                                         Nucleolar protein required ribosome biogenesis <br />  <br /> Dreyfuss,G., Matunis,M.J., Pifiol-Roma,S. Burd,C.G. (1993) Annu.        Sachs,A.B. Davis,R.W. (1990) Science, 247, 1077-1079. <br />    Rev. Biochemii., 62, 289-321.                                            Sachs,A.B., Bond,M.W. Kornberg,R.D. (1986) Cell, 45, 827-835. <br /> Elledge,S.J. Davis,R.W. (1988) Gene, 70, 303-312.                       Sambrook,J., Fritsch,E.F. Maniatis,T. (1989) Molecular Cloning. A <br /> Feinberg,A.P. Vogelstein,B. (1984) Anal. Biochem., 137, 266-267.           Laboratory Manual. 2nd edn. Cold Spring Harbor Laboratory Press, <br /> Fournier,M.J. Maxwell,E.S. (1993) Trends Biochem. Sci., 18,                Cold Spring Harbor, NY. <br />     13 1-136.                                                               Sanger,F., Nicklen,S. Coulson,A.R. (1977) Proc. Natl Acad. Sci. <br /> Garcia-Bustos,J., Heitman,J. Hall,M.N. (1991) Biochim. Biophvs.            USA, 74, 5463-5467. <br />    Acta, 1071, 83-101.                                                      Savino,R. Gerbi,S.A. (1990) EMBO J., 9, 2299-2308. <br /> Garnier,J., Osguthorpe,D.J. Robson,B. (1978) J. Mol. Biol., 120,        Scheer,U. Benavente,R. (1990) BioEssays, 12, 14-20. <br />    97- 120.                                                                 Scheer,U., Thiry,M. Goessens,G. (1993) Trends Cell Biol., 3, <br /> Gergen,J., Stern,R. Wensink,P. (1979) Nucleic Acids Res., 7,               236-241. <br />    2115-2136.                                                               Scherly,D., Boelens,W., van Venrooij,W.J., Dathan,N.A., Hamm,J. and <br /> Girard,J.-P., Lehtonen,H., Caizergues-Ferrer,M., Amalric,F., Tollervey,D.      Mattaj,I.W. (1989) EMBO J., 8, 4163-4170. <br />    Lapeyre,B. (1992) EMBO J., 11, 673-682.                              Schmidt-Zachmann,M.S. Franke,W.W. (1988) Chromosoma, 96, <br /> Goyer,C., Altmann,M.,             Lee,H.S.,   Blanc,A.,   Deshmukh,M.,         417-426. <br />    Woolford,J.L.,Jr, Trachsel,H. Sonenberg,N. (1993) Mol. Cell.         Schmitt,M.E. Clayton,D.A. (1993) Mol. Cell. Biol., 13, 7935-7941. <br />    Biol., 13, 4860-4874.                                                    Schmmang,T., Tollervey,D., Kem,H., Frank,R. Hurt,E.C. (1989) <br /> Hadjiolov,A.A. (1985) In Alfert,M., Bermann,W., Goldstein,L.,                  EMBO J., 8, 4015-4024. <br />    Porter,K.R. Site,P. (eds), The Nucleolus Ribosome Biogenesis.    Sherman,F., <br /> </body></html>