<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Disruption Yarrowia lipolytica TPS1 Gene Encoding <br /> Trehalose-6-P Synthase Does Not Affect Growth in <br /> Glucose Impairs Growth High Temperature <br /> Carmen-Lisset Flores1*, Carlos Gancedo1, Thomas Petit2 <br /> 1 Department Metabolism Cell Signalling, Instituto Investigaciones Biome&#194;&#180;dicas &#226;&#8364;&#732;&#226;&#8364;&#732;Alberto Sols&#226;&#8364;&#8482;&#226;&#8364;&#8482; CSIC-UAM, Madrid, Spain, 2 Universite&#194;&#180; La Re&#194;&#180;union, Laboratoire <br /> de Chimie des Substances Naturelles et des Sciences des Aliments, Institut Universitaire Technologie, De&#194;&#180;partement Ge&#194;&#180;nie Biologique, Saint Pierre, France <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      We cloned Yarrowia lipolytica TPS1 gene encoding trehalose-6-P synthase complementation lack of <br />      growth glucose Saccharomyces cerevisiae tps1 mutant. Disruption YlTPS1 achieved cassette <br />      placed 39half coding region overlap sequence promoter essential gene YlTFC1. <br />      The Yltps1 mutant grew glucose Y. lipolytica hexokinase extremely sensitive inhibition trehalose-6-P. <br />      The presence glucokinase, insensitive trehalose-6-P, constitutes 80% glucose phosphorylating <br />      capacity growth glucose account growth <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Trehalose content 1 nmol/mg dry <br />      weight Y. lipolytica, increased strains expressing YlTPS1 control YlTEF1promoter a <br />      disruption YALI0D15598 encoding putative trehalase. mRNA levels YlTPS1 low did respond thermal <br />      stresses, YlTPS2 (YALI0D14476) YlTPS3 (YALI0E31086) increased 4 6 times, repectively, heat treatment. <br />      Disruption YlTPS1 drastically slowed growth 35uC. Homozygous Yltps1 diploids showed decreased sporulation <br />      frequency ascribed low level YALI0D20966 mRNA homolog S. cerevisiae MCK1 encodes a <br />      protein kinase activates early meiotic gene expression. <br />  <br />   Citation: Flores C-L, Gancedo C, Petit T (2011) Disruption Yarrowia lipolytica TPS1 Gene Encoding Trehalose-6-P Synthase Does Not Affect Growth Glucose <br />   Impairs Growth High Temperature. PLoS ONE 6(9): e23695. doi:10.1371/journal.pone.0023695 <br />   Editor: Robert Alan Arkowitz, Institute Developmental Biology Cancer Research, France <br />   Received June 3, 2011; Accepted July 22, 2011; Published September 12, 2011 <br />   Copyright: &#195;&#376; 2011 Flores et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work partially supported Grants BFU2007-66063-C02-01/BMC BFU2010-19628-CO2-02 Spanish Ministry Science and <br />   Innovation (MICINN; URL:www.micinn.es) CG. The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. No additional external funding received study. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: clflores@iib.uam.es <br />  <br />  <br />  <br /> Introduction                                                                             yeast hyphal form [17] excrete organic acids [18,19] and <br />                                                                                          potential host expression heterologous proteins [20]. Y. <br />    Trehalose, non-reducing disaccharide formed glucose                          lipolytica used model study physiological processes <br /> units, important varied functions different organisms [1,2].                  like lipid accumulation [21] peroxisome biogenesis pexophagy <br /> In yeasts trehalose synthesized two-step pathway [3]: first,                     [22]. Differences kinetic regulatory properties important Y. <br /> trehalose-6-phosphate (T6P) formed glucose-6P UDP-                           lipolytica enzymes [23,24,25] transcriptional regulation some <br /> glucose enzyme T6P synthase (Tps1) encoded TPS1                            genes respect S. cerevisiae [26,27] been <br /> gene [4] dephosphorylated T6P phosphatase (Tps2)                           described. Therefore high sensitivity Y. lipolytica <br /> encoded gene TPS2 [5]. Two proteins catalytic                       hexokinase T6P appeared worthwhile isolate TPS1 gene of <br /> activity, Tps3 Tsl1, appear form complex Tps1                          yeast analyze effects disruption. The isolation of <br /> Tps2 [6]. Mutations genes involved trehalose biosynthesis                      gene presents potential technological Aspergillus <br /> affect glucose metabolism, morphology virulence yeasts                         niger degree expression tpsA gene encodes T6P <br /> fungi [2], cause lethal phenotypes insects nematodes [7,8]                    synthase, influences rate citric acid production [28,29] Y. <br /> are embryo lethal affect inflorescence branching                            lipolytica excretes acid conditions [18,19]. We report here <br /> structures plants [9]. In Saccharomyces cerevisiae Kluyveromyces lactis            Y. lipolytica single gene encoding T6P synthase, its <br /> mutations gene TPS1 cause inability grow glucose                            disruption does preclude growth glucose decreases <br /> [4,10,11]. This phenotype ascribed loss                           sporulation efficiency slows growth 35uC. In addition <br /> inhibitory effect T6P hexokinase [2,12] mathematical                           report disruption YlTPS3 abolishes increase trehalose <br /> modelization glycolysis confirmed importance                          observed heat shock. <br /> control mechanism S. cerevisiae [13]. The inhibition hexokinase <br /> by T6P widespread yeasts [12,14] strength                            Materials Methods <br /> variable; inhibited hexokinase reported yeast <br /> Yarrowia lipolytica Ki 3.5 mM [12,15]. Y. lipolytica                      Strains culture conditions <br /> dimorphic yeast separated early yeast evolutionary                           The yeasts strains used shown Table 1. Y. lipolytica was <br /> trunk [16]. It attracted attention ability shift               cultured synthetic medium 0.17% yeast nitrogen <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span> <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                   1                           September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                                Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br /> without amino acids ammonium sulfate (Difco, Detroit, MI)                         The following plasmids S. cerevisiae constructed: <br /> and 0.1% glutamate pH 6. S. cerevisiae cultured similarly but <br /> using ammonium sulfate nitrogen source. Auxotrophic                             &#226;&#8364;&#8220; pCLF1 carrying YlTPS1 gene isolated cDNA <br /> requirements added final concentration 20 mg/ml                         library [32] ability complement lack growth in <br /> and 2% glucose generally used carbon source. Liquid                           glucose S. cerevisiae tps1 strain. <br /> cultures shaken 30uC. Sporulation medium based                      &#226;&#8364;&#8220; pCLF2, centromeric plasmid carries YlTPS1, was <br /> commercial V8 drink essentially [30]. Freshly constructed                         constructed follows. The BamHI fragment plasmid <br /> diploid strains patched medium incubated                         pAN10 [36] carrying promoter terminator regions <br /> 23uC weeks. Sporulation followed malachite                      S.cerevisiae ADH1 inserted pRS316 [37] <br /> green staining [31]. Spores recovered sporulated                           linearized BamHI. A 1.5 kb blunt-ended NotI fragment <br /> cultures digestion Zymolyase 20T (Seikagaku Co.,<span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span>-                       YlTPS1 pCLF1 inserted blunt-ended <br /> kyo, Japan) treatment mineral oil. Cells                             HindIII site plasmid. <br /> hydrophobic phase spread selective plates colonies                     &#226;&#8364;&#8220; pCLF7 expresses YlTPS3 control ScADH1 <br /> isolated checked crossing ability relevant genes                     promoter. YlTPS3 (YALI0E31086) annotated intron <br /> tested PCR. Thermal stresses transferring                        containing gene. Using FirstChoice RLM-RACE Kit <br /> yeast cultures 30uC shaking water bath 40uC 4uC                   (Ambion) checked correctness ATG the <br /> and keeping time indicated experiment.                          cDNA predicted sequence. Primer design amplify cDNA <br /> Temperature equilibration took place minutes.                      genomic DNA based fact exon is <br />                                                                                      23 bp long. Primer 1006 covers exon first <br /> Libraries, primers plasmids                                                      19 bp second exon; primer 1007 produced <br />    A Y. lipolytica cDNA library control S. cerevisiae               PCR product 3168 bp containing cDNA YlTPS3. <br /> PGK1 promoter plasmid pFL61 [32] genomic library Y.                      This product cloned pCR-Blunt vector (Invitrogen) <br /> lipolytica [33] used. Yeast transformations described                   resulting plasmid digested NotI SpeI blunt- <br /> in [34] Y. lipolytica [35] S. cerevisiae. Primers used             ended cloned pDB20 [38] URA3 marker <br /> PCR reactions shown Tables S1 S2. All PCR products                        substituted LEU2. The cDNA YlTPS3 was <br /> were sequenced verify identity.                                             sequenced introduced plasmid. <br />  <br />  <br />  <br />  Table 1. Yeast strains used work. <br />  <br />  <br />  Strains Y. lipolytica               Genotype                                                                 Reference <br />  <br />  PO1a                                   MATA leu2-270 ura3-302                                                   [34] <br />  E129                                   MATA lys11-23 ura3-302 leu2-270 xpr2-322                                 C. Gaillardin (Grignon, France) <br />  A1-5                                   MATB met6                                                                S. Mauersberger (Dresden,Germany) <br />  CJM 364                                MATA leu2-270 Dura3-302 tps1::YlURA3                                     This work <br />  CJM 645                                MATB met6                                                                <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> cross A1-56CJM 364 <br />  CJM 651                                MATA tps1::YlURA3 met 6                                                  From cross A1-56CJM 364 <br />  CLF 279                                tps1::YlURA3 leu2-270 met6 pCLF4                                         This work <br />  CJM 613                                MATA leu2-270 ura3-302 pCLF5                                             This work <br />  CJM 683                                MATA leu2-270 ura3-302 nth1::LEU2                                        This work <br />  CJM 402                                MATA tps1::YlURA3 lys11-23 ura3-302 leu2-270 xpr2-322                    This work. Derived E129 <br />  CJM 649                                MATB tps1::YlURA3 leu2-270 ura3-302 met6                                 From cross A1-56CJM 364 <br />  CJM 722                                Diploid homozygous TPS1                                              From cross E1296A1-5 <br />  CJM 723                                Diploid heterozygous TPS1/tps1                                           From cross CJM 4026A1-5 <br />  CJM 724                                Diploid homozygous tps1/tps1                                             From cross CJM 4026CJM 649 <br />  CJM 748                                MATA tps3::LEU2 ura3 leu2-270                                            This work <br />  CJM 703                                tps1::YlURA3 nth1::YlLEU2 met 6                                          From cross CJM 6496CJM 683 <br />  CJM 667                                MATA leu2-270 ura3-302 pCLF8                                             This work <br />  CJM 687                                MATB tps1::YlURA3 leu2-270 ura3-302 met6 pCLF9                           This work <br />  <br />  Strains S. cerevisiae               Genotype                                                                 Reference <br />  <br />  CJM 567                                Wild type                                                                J.M. Gancedo (Madrid, Spain) <br />  CJM 486                                MATa ade2-1 his311,15 ura3-1 leu2-3 112 trp1-1 tps1::HIS3                [12] <br />  CJM 391                                MATa ade2-1 his311,15 ura3-1 leu2-3 112 trp1-1 tps1::HIS3 pCLF1          This work <br />  CJM 397                                MATa ade2-1 his311,15 ura3-302 leu2-3 112 trp1-1 tps1::HIS3 pCLF2        This work <br />  CJM 767                                MATa ade2-1 his311,15 ura3-1 leu2-3 112 trp1-1 tps1::HIS3 pCLF7          This work <br />  <br />  Plasmids pCLF described Materials Methods. <br />  doi:10.1371/journal.pone.0023695.t001 <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                             2                            September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                    Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />   The following plasmids Y. lipolytica constructed                RT-qPCR <br />                                                                               Total RNA Y. lipolytica extracted flash-frozen <br /> &#226;&#8364;&#8220; pCLF3 carries fragment 5.4 kb contains Y.                  cells [43] processed described therein. The quality RNA <br />   lipolytica TPS1 gene isolated screening Y. lipolytica       checked using Agilent 2100 Bioanalyzer. The primers for <br />   genomic library [33] YlTPS1 probe.                                RT-qPCR shown Table S2. They checked for <br /> &#226;&#8364;&#8220; pCLF4 expresses YlTPS1 control                      specificity using Primer-BLAST NCBI Y. <br />   promoter. It constructed follows: YlURA3 marker               lipolytica CLIB122 genomic sequence. Total RNA reverse- <br />   plasmid pCL49 [26] substituted YlLEU2                  transcribed cDNA using High Capacity RNA-to-cDNA <br />   pCL49L. The SphI-BamHI fragment pCL49L replaced                   Master Mix (Applied Biosystems). The cDNA levels then <br />                                                                            analyzed using LightCycler 480 ROCHE the <br />   2.8 kb HpaI-BamHI fragment pCLF3 bearing the <br />                                                                            LightCycler 480 SYBR Green I Master mix (Roche) each <br />   YlTPS1 ORF 1 Kb upstream sequence. <br />                                                                            primer 250 nM. Each sample tested triplicate. After <br /> &#226;&#8364;&#8220; pCLF5 carries coding region YlTPS1 (a 1.5 Kb NotI                 completion RT-qPCR melting-curve data collected to <br />   fragment) plasmid pCLF1 control                    verify PCR specificity, absence contamination primer <br />   YlTEF1 promoter plasmid pCL49L.                                       dimers. The gene YALI0F27533 (ARP4) used normalize the <br /> &#226;&#8364;&#8220; pCLF8 carries fusion YlTPS1 promoter lacZ. A                 data. <br />   1214 bp DNA fragment includes 14 initial amino acids <br />   YlTPS1 obtained PCR, using oligonucleotides 1010               Extracts assay enzymatic activities <br />   1011 cloned pGEM-T easy, digested NotI                    Cell free extracts prepared breaking yeasts buffer <br />   BamHI inserted plasmid pINA354B [39] digested              glass beads cycles 1 min vortexing 1 min on <br />   enzymes. The resulting plasmid linearized              ice. The buffer 20 mM imidazole pH 7, addition of <br />   ApaI direct integration YlLEU2 locus. Correct                1 mM DTT 1 mM EDTA T6P synthase assayed. <br />   integration checked PCR Southern analysis.                    The extract centrifuged cold 15 min 13000 rpm <br /> &#226;&#8364;&#8220; pCLF9 carries coding region ScTPS1 obtained PCR                Eppendorf table centrifuge supernatant used for <br />   using primers 1012 1013 plasmid pCL49L.                           determination enzyme activities. T6P synthase activity was <br />                                                                            determined step method [3] mixture 50 mM <br />                                                                            imidazole pH 7, 0.1 M KCl, 10 mM MgCl2, 1 mM EDTA, <br />                                                                            10 mM glucose 6-P 5 mM UDP-glucose. Samples taken <br /> Disruption YlTPS1 <br />                                                                            different time intervals boiled 3 minutes stop the <br />    In consecutive PCR reactions I-SceI restriction site a <br />                                                                            reaction. The UDP formed measured spectrophotometrically <br /> deletion 658 bp created. Primers 1001 1002 with <br />                                                                            coupled assay NADH, pyruvate kinase lactate <br /> complementary ends including recognition site meganu- <br />                                                                            dehydrogenase. A blank added glucose-6P run in <br /> clease I-SceI, 1000 1003 used. With pCLF3 template <br />                                                                            parallel sample. Hexokinase glucokinase were <br /> and mentioned primers fragments corresponding 59 <br />                                                                            assayed glucose fructose [15]. b-galactosidase was <br /> and 39regions disruption cassette obtained. These              assayed 80 mM sodium phosphate buffer, 8 mM KCl, 0.8 mM <br /> products used template primers 1000 1003             MgSO4, 40 mM b mercaptoethanol, 2.6 mM 2-nitrophenyl <br /> PCR reaction obtain disrupted YlTPS1 copy. The product            b-D- galactopyranoside described Wallenfels [44]. Protein <br /> cloned pGEM-T easy, digested I-SceI ligated             assayed commercial BCA protein assay kit (Pierce). <br /> 1.2 kb YlURA3 fragment flanked I-SceI sequences plasmid <br /> pINA-URA3-I-SceI [40]. The 5268 bp fragment NotI used to <br />                                                                            Trehalose determination <br /> disrupt chromosomal copy YlTPS1 gene. <br />                                                                               The cultures harvested centrifugation, washed with <br />                                                                            water frozen use. Trehalose extracted boiling <br /> Disruption YlTPS3 (YALI0E31086)                                         water described [45]. Trehalose determined enzymat- <br />    A piece 3611 bp containing YlTPS3 gene obtained           ically using Trehalose assay Kit Megazyme (Bray, Co. <br /> PCR genomic DNA primers 1004 1005 cloned                  Wicklow, Ireland). <br /> in pGEM-T easy (Promega). The resulting plasmid digested <br /> with XhoI, remove fragment 758 bp, ligated 2.1 kb         Extraction assay intracellular metabolites <br /> band containing YlLEU2 gene NcoI digestion                  Yeast cells quickly filtered Millipore <br /> plasmid pINA62 [41] filling-in ends DNA                  AAWPO4700 snap frozen liquid nitrogen. The frozen <br /> segments. The chromosomal copy YlTPS3 disrupted using             pellets dropped boiling ethanol treated [46]. <br /> 4996 bp NotI fragment previous plasmid. Correct                   Metabolites determined spectrophotometrically [47]. <br /> disruption checked PCR.                                             For calculations, assumed 1 g wet yeast an <br />                                                                            intracellular volume 0.6 ml [48]. <br />                                                                               Nucleotide accession number.- The sequence the <br /> Disruption YlNTH1 (YALI0D15598)                                         YlTPS1 cDNA deposited GenBank database the <br />    A fragment 2049 bp containing YALI0D15598 [42]                       accession number AJ011032. <br /> encoding putative neutral trehalase (YlNTH1) obtained by <br /> PCR using primers 1008 1009 cloned pGEM-T easy.                 Results <br /> The resulting plasmid digested NheI XhoI substitute <br /> an internal 816 bp fragment NTH1 gene 2.1 kb                 Cloning characteristics Y. lipolytica TPS1 gene <br /> fragment containing YlLEU2 gene plasmid pINA62 [41]                 We used phenotypic complementation absence of <br /> digested NcoI. A 3342 bp NotI fragment construction         growth glucose S. cerevisiae tps1 mutant [2,49] isolate the <br /> was used disrupt chromosomal copy. Correct disruption           TPS1 gene Y. lipolytica. We transformed S. cerevisiae tps1 <br /> checked PCR Southern analysis.                                      mutant strain cDNA library Y. lipolytica the <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       3                       September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                             Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br /> control S. cerevisiae PGK1 promoter (see Materials                       glucose phosphorylating enzyme expressed growth this <br /> Methods) selected transformants grew glucose. Plasmid                   sugar [55,56]. Since T6P inhibition Y. lipolytica hexokinase the <br /> pCLF1 isolated different transformants carried DNA insert                    highest reported (Ki 3.5 mM) [12,15] studied effect the <br /> of 1.5 kb DNA sequence revealed ORF encoding                       disruption TPS1 yeast. Attempts disrupt YlTPS1 <br /> putative protein 486 amino acids shares 70% identity                  placing disruption cassette nucleotides 188 406 after <br /> with different fungal T6P synthases. This Y.lipolytica cDNA inserted                ATG failed, displaced nucleotide 710 a <br /> either multicopy centromeric plasmid                          correct disruption obtained (Figure 2 b,c,d). We attribute the <br /> complemented glucose negative phenotype S. cerevisiae tps1               failures disruption cassettes located 59region the <br /> mutant restored ability synthesize trehalose (Figure 1).            coding sequence interference expression the <br /> These results indicate DNA cloned encodes bona fide Tps1                 neighbouring TFC1 gene (Figure 2a). Lack TFC1 lethal in <br /> protein Y. lipolytica functional S. cerevisiae. We refer       S. cerevisiae [57] Schizosaccharomyces pombe [58] is <br /> to gene encoding protein YlTPS1. The 1 kb upstream                      likely case Y. lipolytica. The YlTPS1 disruptants <br /> region YlTPS1 contains CCCCT motif [50] S. cerevisiae               grew glucose contrast behavior tps1 mutants of <br /> implicated heat stress-controlled transcription                        S. cerevisiae. The distinct phenotype caused difference <br /> [51,52,53,54]. About 245 bp upstream ATG YlTPS1                        glucose phosphorylating equipment yeasts, <br /> found ATG ORF transcription runs divergent                       lack significant activity Tps1 both. We in <br /> that YlTPS1 (Figure 2a). The protein putatively encoded                  glucose grown cultures Y. lipolytica expression gene <br /> ORF highly similar S. cerevisiae Tfc1, DNA-                YALI0E15488 encoding glucokinase (Flores Gancedo, <br /> binding subunits yeast transcription factor TFIIIC. Part              unpublished results) exceeded gene encoding <br /> TFC1 promoter likely overlaps coding sequence YlTPS1.                   hexokinase (Figure 3). Also enzyme measurements showed that <br /> The relative position ORFs Y. lipolytica different              glucokinase constitutes main phosphorylating activity Y. <br /> that Hemiascomycetes TPS1 appears                      lipolytica (Table 2). Since glucokinase insensitive inhibition by <br /> synteny block covers genes (Figure 2a). The                      T6P [12,15], growth glucose Yltps1 mutant be <br /> sequence TPS1 promoter region widely used PO1a                        explained scarce contribution hexokinase glucose <br /> strain exhibited GT deletion 260 change C/T position                  phosphorylating activity. Moreover, disruption YlTPS1 slightly <br />                                                                                     decreased proportion hexokinase activity (Table 2). <br /> 2314 relative ATG respect sequence appears in <br />                                                                                     Concentration hexose mono bis- phosphates ATP were <br /> Ge&#194;&#180;nolevures [42]. <br />                                                                                     affected respect wild type growth in <br />                                                                                     glucose (Table 3) contrast happens S. cerevisiae tps1 <br /> Disruption chromosomal copy YlTPS1 does                               mutant accumulates compounds loses ATP upon <br /> affect growth glucose                                                            glucose addition. This result consistent lack control of <br />    Lack growth glucose S. cerevisiae tps1 mutants                 glycolytic flux T6P Y. lipolytica. The YlTPS1 disruptants <br /> attributed loss inhibition T6P hexokinase 2 [12]                    showed slightly shorter duplication time wild type (Wt <br />                                                                                     14964 min, tps1 13964 min, tps1/pCLF5 15168 min, means of <br />                                                                                     experiments) (Figure S1). No immediate explanation be <br />                                                                                     provided difference. <br />  <br />                                                                                     Levels trehalose Y. lipolytica low increase <br />                                                                                     disruption gene encoding putative trehalase <br />                                                                                     heat shock <br />                                                                                        Levels trehalose Y. lipolytica grown glucose to <br />                                                                                     stationary phase glycerol 1 nmol/mg dry weight. <br />                                                                                     A possible explanation result genes <br />                                                                                     encoding trehalose biosynthetic pathway enzymes not <br />                                                                                     expressed growth glucose, measured the <br />                                                                                     expression genes Y.lipolytica. In Ge&#194;&#180;nolevuress <br />                                                                                     database [42], YALI0D14476 annotated similar S. cerevisiae <br />                                                                                     TPS2 YALI0E31086 shows highest homology S. <br />                                                                                     cerevisiae TPS3/TSL1. All genes expressed during <br />                                                                                     growth glucose levels YlTPS2 YlTPS3 were <br />                                                                                     low compared YlTPS1 (Figure 3). Although a <br />                                                                                     western blot analysis fusion Tps1 HA epitope <br />                                                                                     indicated protein expressed growth glucose <br />                                                                                     (results shown) activity low (Table 4). This low <br />                                                                                     activity additional cause lack effect the <br />                                                                                     YlTPS1 disruption growth glucose. Expression YlTPS1 <br />                                                                                     control strong YlTEF1 promoter increased <br /> Figure 1. Phenotypic complementation S.cerevisiae tps1                         trehalose content allowed detection Tps activity was <br /> mutant expression YlTPS1 gene. The strains indicated                      undetectable wild type strain (Table 4). This result <br /> were streaked minimal medium glucose galactose carbon                 suggested low trehalose level low activity of <br /> sources incubated 30uC 4 days. Trehalose content the <br /> strains cultured glucose, mutant tps1 cultured                   biosynthetic pathway. In addition low synthesis low <br /> galactose, given table (results independent cultures).             trehalose content activity trehalase. We <br /> The multicopy plasmid pCLF1 centromeric pCLF2.                      identified gene, YALI0D15598, sequence Y. <br /> doi:10.1371/journal.pone.0023695.g001                                               lipolytica Ge&#194;&#180;nolevures database [42] shows homology the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               4                        September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                           Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />  <br />  <br /> Figure 2. Relative positions TPS1 TFC1 genes yeast species disruption chromosomal copy the <br /> YlTPS1 gene. a) Diagram chromosomal neighborhood TPS1 different yeasts species. Notice unusual relative position TPS1 TFC1 <br /> in Y. lipolytica. The order transcriptional direction genes correspond annotation Ge&#194;&#180;nolevures [42] S. cerevisiae which <br /> the annotation SGD used [90]. Names genes appear arrow, arrows indicate genes unknown function. <br /> Chromosome designation indicated. WGD, genome duplication. b) Scheme chromosomal region YlTPS1 its <br /> disruption. The location coding region YlTFC1 shown. The I-SceI sites flanking URA3 gene introduced PCR described in <br /> Materials Methods. Short vertical bars indicate piece DNA used chromosomal disruption. c, d) Southern analysis different <br /> disruptants: genomic DNA digested HindIII (c) StuI (d) hybridized indicated probe. The sizes DNA bands bp are <br /> shown. Attempts disrupt YlTPS1 placing YlURA3 SalI sites HindIII EcoRI unsuccessful. <br /> doi:10.1371/journal.pone.0023695.g002 <br />  <br /> S.cerevisiae NTH1 NTH2 genes encoding neutral trehalases.                    plants also, inhibition trehalase allows measurement of <br /> Measurements YlNTH1 mRNA showed expressed                        undetectable trehalose levels [59,60,61]. <br /> during growth glucose (Figure 3). We disrupted YlNTH1                       Heat shock increases trehalose yeasts [52,62]. In Y. <br /> measured increased trehalose content resulting mutant.                lipolytica heat shock 40uC 2 h increased trehalose, although <br /> Such increase dependent activity Tps1                  final value varied depending strain (from 7 20 nmol/ <br /> double mutant nth1 tps1 trehalose detected (Table 4). In                 mg dry weight). This treatment considered strong this <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           5                         September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                                      Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />  <br />  <br /> Figure 3. mRNA levels corresponding genes encoding glucose phosphorylating enzymes proteins related trehalose <br /> metabolism Y. lipolytica. Yeasts strains CJM 645, wild type, CJM 651, Yltps1::URA3, grown minimal medium- glucose indicated in <br /> Materials Methods. mRNA levels quantified RT-qPCR described Materials Methods. Three independent cultures analyzed <br /> and technical replicas run. Expression gene normalized YlARP4. The columns represent mean <br /> values biological experiments bars indicating standard deviation. The genes studied encode following proteins: GLK1, <br /> glucokinase; HXK1, hexokinase; TPS1, trehalose-6-P synthase; TPS2, trehalose-6-P phosphatase; TPS3, subunit trehalose synthase complex; NTH1, <br /> neutral trehalase. <br /> doi:10.1371/journal.pone.0023695.g003 <br />  <br />  <br /> yeast does grow temperatures 35uC. We measured                           phenotype S. cerevisiae tps1 mutant YlTPS3 cDNA <br /> levels mRNA corresponding genes related trehalose                         (Figure 5). We suggest YlTps3 necessary maintain the <br /> metabolism heat treatment (Figure 4). Levels YlTPS1                             stability trehalose synthase complex heat shock Y. <br /> mRNA did increase spite presence CCCCT                              lipolytica. This reveals difference S. cerevisiae the <br /> sequence promoter. Measurements b-galactosidase                                absence Tps3 does affect trehalose content heat <br /> produced fusion YlTPS1 promoter E. coli lacZ                            shock [6]. <br /> were consistent result (3463 mUnits/mg protein time                            Disruption YlTPS1 severely decreased growth 35uC, only <br /> 0 vs 3964 mUnits/mg protein 2 h, independent                                  small colonies visible 7 days temperature <br /> experiments). Relative abundance mRNAs corresponding                               (Figure 6). A plasmid carrying YlTPS1 restored wild type <br /> YlTPS2 YlTPS3 increased 4 6 times respectively                           phenotype. The S.cerevisiae TPS1 gene slightly improved growth of <br /> the heat treatment. The higher increase YlTPS3 suggested                           Yltps1 mutant 35uC. <br /> important role gene heat response. A disruption                          Treatment Y.lipolytica 4uC 2 20 hours did not <br /> YlTPS3 abolished trehalose accumulation heat treatment                              modify significantly mRNA levels corresponding genes <br /> raising question Tps3 T6P synthase                          trehalose biosynthetic pathway (Figure 4), contrast with <br /> hypothesis ruled similarity                        behavior TPS1 TPS2 S. cerevisiae expression <br /> the YlTPS1 YlTPS3 sequences. This possibility                               increase treatment 10uC [63]. <br /> unlikely absence trehalose Yltps1 mutant heat                          In S. cerevisiae transcription factors Hsf1 Msn2/4 are <br /> shock lack complementation glucose negative                         implicated response heat shock stresses <br />  <br />  <br />  Table 2. Phosphorylating activity glucose fructose activities hexokinase glucokinase Y. lipolytica. <br />  <br />  <br />  Strain relevant genotype           Phosphorylating activity (mU/mg protein)                                Specific activity (mU/mg protein) <br />  <br />                                         Glucose                                  Fructose                       Hexokinasea)                      Glucokinasea) <br />  <br />  CJM 645 YlTPS1                         203620                                   68619                          49 [24]                           154 [76] <br />  CJM 651 Yltps1                         208615                                   4766                           34 [16]                           174 [84] <br />  <br />  Strains cultured minimal medium glucose indicated Materials Methods. Extracts assay phosphorylating activities described Materials <br />  Methods. <br />  a) <br />    Hexokinase glucokinase activities calculated phosphorylation results glucose fructose using Fructose/Glucose phosphorylation ratio of <br />    1.4 hexokinase ([92], unpublished results). Figures brackets indicate percentage corresponding activity extract. Results mean values <br />    6 standard deviations independent cultures. <br />  doi:10.1371/journal.pone.0023695.t002 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                   6                            September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                                   Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />  <br />  Table 3. Concentration metabolites (mM) WT Yltps1 strains growth glucose. <br />  <br />  <br />  Strains relevant <br />  genotype                        Glucose-6-P                Fructose-6-P             Fructose-1,6-P2             Trioses phosphate       ATP <br />  <br />  CJM 645 YlTPS1                  0.1860.06                  0.0460.01                0.3360.02                   0.0860.02               0.4760.10 <br />  CJM 651 Yltps1                  0.2460.01                  0.0660.01                0.2860.01                   0.0960.01               0.5460.08 <br />  <br />  Yeast strains cultivated minimal medium glucose sampled exponential phase growth. Metabolite extraction quantification described <br />  Materials Methods. Results mean values 6 standard deviations independent cultures. <br />  doi:10.1371/journal.pone.0023695.t003 <br />  <br />  <br /> [52,64,65]. In Y. lipolytica protein Mhy1 (YALI0B21582p)                        Tps activity Y. lipolytica gene expressed the <br /> shows high similarity zinc finger domain Msn2/4                   control strong promoter, restoration growth in <br /> and binds STRE sequences [66]. A BLAST search Y.                        glucose trehalose content S. cerevisiae tps1 mutant, by <br /> lipolytica database genes encoding homologues ScHSF1                         sequence similarity T6P synthases organisms. <br /> yielded gene YALI0E13948. Levels mRNA corresponding                           Although sequences high similarity Tps1 be <br /> those genes increased 3 times heat shock (Figure 4)                     identified Y. lipolytica Ge&#194;&#180;nolevuress database [42] only <br /> consistent possible implication heat shock regulated                  active trehalose synthesis expressed S. cerevisiae was <br /> processes.                                                                          YlTPS1. In yeasts fungi gene encoding a <br />                                                                                     protein Tps activity Aspergillus niger A. <br /> Disruption YlTPS1 impairs sporulation                                            fumigatus [69,70] T6P synthases high sequence homology <br />    When Y. lipolytica diploids homozygous tps1 <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span>                     different roles physiology organism have <br /> (CJM 724) placed sporulation conditions sporulation                     described. The chromosomal close vicinity YlTPS1 and <br /> frequency reduced respect wild type (CJM 722)                   YlTFC1 deserves attention. A short intergenic region been <br /> or heterozygous TPS1/tps1 (CJM 723) diploids. A similar                             taken indicative sharing certain regulatory elements the <br /> behaviour tps1/tps1 diploids S. cerevisiae [67,68] ascribed               genes question [71,72]. Assuming average promoter <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br /> to decreased expression MCK1, gene stimulates                           1 kb [72] promoter TFC1 overlap the <br /> expression IME1 encodes transcriptional activator                     coding sequence YlTPS1 viceversa. This neighborhood <br /> sporulation [68]. Based sequence homology identified                       opposite transcription direction explains the <br /> ORF YALI0D20966 exhibits 41% identity 59%                                 failures disrupt YlTPS1 insertions located near 59end of <br /> similarity S. cerevisiae MCK1. We measured levels                     coding sequence interfere expression of <br /> of mRNA corresponding YALI0D20966 genes                               YlTFC1 result lethality. Although closely located pairs of <br /> implicated trehalose metabolism wild type diploid                  genes divergent transcription tend conserved in <br /> in homozygous tps1 mutation sporulation conditions.                  evolution [72] TPS1 TFC1 separated yeast <br /> After 8 days sporulation medium level YALI0D20966                         species Hemiascomycetes phylogenetic [16]. <br /> mRNA wild type increased 15 times                         This separation allowed evolution mechanisms <br /> homozygous tps1/tps1 strain reached maximum 3 fold                          linking expression TPS1 multiple signals case in <br /> (Figure 7). This behaviour parallels MCK1 S. cerevisiae                  yeasts. <br /> suggesting implication YALI0D20966 sporulation Y.                          Deffects trehalose biosynthetic pathway produce variety <br /> lipolytica. Levels RNA corresponding YlTPS1, YlTPS2, YlTPS3                   effects different organisms like bacteria [73] yeasts [2], plants <br /> or YlNTH1 did vary significantly strain                     [74] insects [7] nematodes [8]. In S. cerevisiae loss control <br /> same conditions.                                                                    hexokinase activity T6P causes lack of <br />                                                                                     growth tps1 mutant glucose [12], S. pombe whose <br /> Discussion                                                                          hexokinases sensitive T6P [75] phenotype not <br />                                                                                     observed similar mutation [76]. The growth glucose the <br />    We isolated characterized gene encoding                             tps1 mutant Y. lipolytica hexokinase highly sensitive to <br /> trehalose-6-P synthase dimorphic yeast Y. lipolytica. The                  T6P [12,15] explained presence T6P <br /> identity gene supported increase trehalose                  insensitive glucokinase constituting roughly 80% glucose <br />  <br />  Table 4. Trehalose content Tps1 activity different strains Y. lipolytica. <br />  <br />  <br />  Strain relevant genotype                        Trehalose                                            Trehalose-6-P synthase <br />                                                      (nmol/mg dry weight)                                 (mU/mg protein) <br />  <br />  CJM 645 TPS1                                        ,1                                                   ,5 <br />  CJM 613 TPS1pCLF5                                   8.962.1                                              5565 <br />  CJM 683 TPS1 nth1                                   1862                                                 n.d. <br />  CJM 703 nth1tps1                                    ,1                                                   n.d. <br />  <br />  Yeasts grown minimal medium glucose; trehalose trehalose-6-P synthase assayed described Materials Methods. Results mean values <br />  6 standard deviations independent cultures. n.d., determined. <br />  doi:10.1371/journal.pone.0023695.t004 <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                                7                           September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                        Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />  <br />  <br /> Figure 4. Levels mRNA corresponding genes YlTPS1, <br /> YlTPS2, YlTPS3, MHY1 (YALI0B21582) HSF1 (YALI0E13948) <br /> during thermal stresses. A sample exponentially growing <br /> culture strain CJM 645 minimal medium glucose (time zero) was <br /> taken transfer 40uC 4uC 2 20 hours. mRNA levels <br /> were quantified RT-qPCR described Materials Methods. <br /> Two independent cultures analyzed technical replicas <br /> were run. Expression gene normalized to <br /> that YlARP4. Values shown relative time zero. The <br /> columns represent mean values experiments bars                Figure 6. Effect YlTPS1 disruption growth 356C. The <br /> indicating standard deviation.                                            strains streaked minimal medium glucose plates, incubated at <br /> doi:10.1371/journal.pone.0023695.g004                                         indicated temperatures pictures taken 3 7 days after <br />                                                                               inoculation. <br />                                                                               doi:10.1371/journal.pone.0023695.g006 <br /> phosphorylating capacity yeast. A similar explanation might <br /> also apply phenotype described tps1 mutant of <br /> Hansenula polymorpha [14] yeast glucokinase                    increase ATP concentration measured Yltps1 mutant, <br /> hexokinase present growth glucose [77]. The levels              finding parallels results obtained tpsA disruptant of <br /> of metabolites disrupted strain accordance             A. nidulans [78]. With current knowledge clear explanation <br /> lack effect mutation growth glucose. A slight             results advanced. Due &#226;&#8364;&#732;&#226;&#8364;&#732;turbo design&#226;&#8364;&#8482;&#226;&#8364;&#8482; of <br />                                                                               glycolysis [13] regulation initial steps pathway is <br />                                                                               necessary. In mammals glucose-6-phosphate controls hexokinase <br />                                                                               [79] S. cerevisiae T6P plays similar role. Yeasts fungi in <br />                                                                               lack T6P does affect growth glucose [14,78] shall <br />                                                                               possess mechanisms regulate steps glycolysis. It <br />                                                                               asked significance T6P inhibition of <br />                                                                               hexokinase organisms appears play a <br />                                                                               significant role control glycolysis. One possibility it <br />                                                                               serve control unrecognized function hexokinase, <br />                                                                               consequence protein structure <br />                                                                               shared hexokinases organisms high <br />                                                                               glycolytic flux taken advantage control first <br />                                                                               irreversible step glucose metabolism. In Y. lipolytica differences in <br />                                                                               kinetic regulatory properties important glycolytic enzymes <br />                                                                               like phosphofructokinase [23] pyruvate kinase [24] indicate that <br />                                                                               yeast regulate glycolysis differently S. cerevisiae. <br />                                                                                  The decrease sporulation observed homozygous tps1 <br />                                                                               diploids parallels findings tps1 mutants fungi like S. <br />                                                                               cerevisiae [67,68,80] Cryptococcus neoformans [81] Stagonospora <br />                                                                               nodorum [82]. In S. cerevisiae, defect adscribed a <br />                                                                               low expression MCK1 inducer gene IME1 whose <br />                                                                               expression triggers sporulation [68]. The low level Yltps1 <br />                                                                               diploids mRNA corresponding gene YALI0D20966 that <br /> Figure 5. Lack complementation Sctps1 YlTPS3. The S. <br /> cerevisiae tps1 mutant (CJM 486) transformed plasmids pCLF1 <br />                                                                               appears Y. lipolytica homolog ScMCK1 suggest a <br /> (YlTPS1), pCLF7 (YlTPS3) pDB20 (void) streaked minimal            similar mechanism decreased sporulation yeast and <br /> medium glucose galactose incubated 30uC 4 days.            relationship TPS1 sporulation already <br /> doi:10.1371/journal.pone.0023695.g005                                         present ancient yeast like Y. lipolytica. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         8                        September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                                         Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />                                                                                             biosynthetic pathway S. cerevisiae depends repetitions a <br />                                                                                             CCCCT stretch (STRE sequence) promoters [65,87]. <br />                                                                                             Function STRE sequences S. cerevisiae requires Msn2/ <br />                                                                                             Msn4 proteins [64]. The corresponding gene(s) known Y. <br />                                                                                             lipolytica. Hurtado Rachubinsky [66] observed high <br />                                                                                             sequence homology Zn finger domain Mhy1 that <br />                                                                                             Msn2/4 showed protein able bind STRE <br />                                                                                             sequences vitro. These authors reported levels MHY1 <br />                                                                                             mRNA increased heat shock 35uC. Our results <br />                                                                                             heat shock 40uC levels MHY1 mRNA <br />                                                                                             increase suggesting MHY1 play role regulatory <br />                                                                                             response stress. It noticed high GC <br />                                                                                             content Y. lipolytica DNA [88] cause presence of <br />                                                                                             CCCCT sequences promoters genes not <br />                                                                                             related responses stress. <br />                                                                                                Y. lipolytica does grow temperatures 35uC. The <br /> Figure 7. Levels mRNA corresponding genes                                     finding disruption YlTPS1 impairs growth limit <br /> during sporulation. A wild type diploid homozygous tps1 <br /> diploid transferred minimal medium glucose V8 <br />                                                                                             temperature suggests trehalose plays protective role against <br /> sporulation medium (see Materials Methods). Samples taken                          changes produced condition [65]. <br /> from initial culture glucose (time zero) 8 days V8.                        Many evidences different organisms trehalose <br /> mRNA levels quantified RT-qPCR described Materials                        biosynthetic pathway, addition primary role, an <br /> Methods. Three independent sporulation experiments analyzed                            influence variety processes range growth on <br /> and technical replicas run. Expression                     certain substrates temperatures, differences virulence in <br /> gene normalized YlARP4. Levels mRNA shown <br /> relative time zero. The columns represent mean values                     pathogens. The targets pathway different depending on <br /> the experiments bars indicating standard deviation.                                organism closely related yeast species like C. <br /> doi:10.1371/journal.pone.0023695.g007                                                       neoformans C. gattii important variations effects <br />                                                                                             caused perturbations pathway [89]. The finding in <br />    Trehalose Y. lipolytica different conditions                             Y. lipolytica separated early evolution yeasts [16], <br /> 1 nmol/mg dry weight. Disruption gene encoding putative                              trehalose biosynthetic pathway does regulate glycolysis <br /> neutral trehalase overexpression YlTPS1 increased trehalose                           suggests regulatory property acquired later along <br /> content. A similar situation occurred vascular plants                           yeast evolution. <br /> trehalose thought absent; incubation validamycin <br /> A, inhibitor trehalase, showed existence                                   Supporting Information <br /> disaccharide [59,60]. Hydrolysis trehalose trehalase a <br />                                                                                             Figure S1 Growth wild type Yltps1 strains. The <br /> low level Tps1 activity responsible low levels of <br />                                                                                             strains grown described Materials Methods and <br /> the sugar Y. lipolytica. It speculated main role of <br />                                                                                             growth followed measuring optical density. CJM645 (Wild <br /> Tps1 provide T6P intermediate pathways different from <br />                                                                                             type), CJM651 (Yltps1) CLF279 (Yltps1/pCLF4). A represen- <br /> trehalose synthesis. Some bacteria produce biosurfactants or <br />                                                                                             tative curve shown strain. <br /> glycolipids require T6P synthesis [83,84]. Y. lipolytica <br /> also produces biosurfactants growing aqueous media                              (TIF) <br /> their detailed structure known [85].                                                 Table S1 Primers used DNA cloning. <br />    Heat shock increased levels trehalose changed                             (DOC) <br /> levels mRNA corresponding YlTPS2 YlTPS3 not <br /> those YlTPS1. A similar lack response A. nidulans tpsA                         Table S2 Primers used RT-qPCR. <br /> been described [78]. The increase mRNA corresponding                                  (DOC) <br /> YlTPS3 absence trehalose heat shocked <br /> Yltps3 mutant indicate important role protein                             Acknowledgments <br /> stability Y. lipolytica trehalose biosynthetic complex. While                     We thank J.M. Nicaud (CNRS, INRA, Institut MICALIS, Jouy-en-Josas, <br /> S. cerevisiae complex consists proteins, Tps1,Tps2, Tsl1                        France) biological materials, Brigitte Tre&#194;&#180;ton (CNRS, INRA, Institut <br /> and Tps3, sequence similar Tps3/Tsl1                                MICALIS, Jouy-en-Josas, France) indications sporulation <br /> found Y. lipolytica database. Decrease trehalose levels                           conditions, J.L. Souciet (Universite&#194;&#180; Louis Pasteur, Strasbourg, France) for <br /> during heat shock S. cerevisiae requires disruption                          discussion synteny Juana M. Gancedo (this Institute) for <br /> Tsl1 Tps3 [6]. In S. cerevisiae different mechanisms                            critical reading manuscript. <br /> transcriptional activation genes, stabilization certain <br /> RNAs [86] activation trehalose synthase complex [87]                             Author Contributions <br /> contribute trehalose accumulation heat shock. Such detailed                           Conceived designed experiments: C-LF CG TP. Performed the <br /> studies available Y. lipolytica. Transcriptional                            experiments: C-LF CG TP. Analyzed data: C-LF CG TP. Contributed <br /> response heat shock case genes trehalose                            reagents/materials/analysis tools: CG C-LF. Wrote paper: CG C-LF. <br />  <br /> References <br />  1. Elbein AD, Pan YT, Pastuszak I, Carroll D (2003) New insights trehalose:            3. Cabib E, Leloir LF (1958) The biosyntesis trehalose phosphate. J Biol Chem <br />     multifunctional molecule. Glycobiology 13: 17&#226;&#8364;&#8220;27.                                           231: 259&#226;&#8364;&#8220;275. <br />  2. Gancedo C, Flores CL (2004) The importance functional trehalose                     4. Bell W, Klaassen P, Ohnacker M, Boller T, Herweijer M, et al. (1992) <br />     biosynthetic pathway life yeasts fungi. FEMS Yeast Res 4: 351&#226;&#8364;&#8220;359.           Characterization 56-kDa subunit yeast trehalose-6-phosphate synthase <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      9                           September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                                                Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />       cloning gene reveal identity product CIF1, regulator        33. Nuttley WM, Brade AM, Eitzen GA, Glover JR, Aitchison JD, et al. (1993) <br />       carbon catabolite inactivation. Eur J Biochem 209: 951&#226;&#8364;&#8220;959.                                     Rapid identification characterization peroxisomal assembly mutants in <br />  5.   De Virgilio C, Burckert N, Bell W, Jeno P, Boller T, et al. (1993) Disruption                Yarrowia lipolytica. Yeast 9: 507&#226;&#8364;&#8220;517. <br />       TPS2, gene encoding 100-kDa subunit trehalose-6-phosphate                    34. Barth G, Gaillardin C (1996) Yarrowia lipolytica. In: Wolf K, ed. Nonconventional <br />       synthase/phosphatase complex Saccharomyces cerevisiae, causes accumulation                yeasts biotechnology. Berlin Heidelberg New York: Springer. pp 313&#226;&#8364;&#8220;388. <br />       trehalose-6-phosphate loss trehalose-6-phosphate phosphatase activity.               35. Ito H, Fukuda Y, Murata K, Kimura A (1983) Transformation intact yeast <br />       Eur J Biochem 212: 315&#226;&#8364;&#8220;323.                                                                     cells treated alkali cations. J Bacteriol 153: 163&#226;&#8364;&#8220;168. <br />  6.   Reinders A, Burckert N, Hohmann S, Thevelein JM, Boller T, et al. (1997)                    36. Navas MA, Cerdan S, Gancedo JM (1993) Futile cycles Saccharomyces cerevisiae <br />       Structural analysis subunits trehalose-6-phosphate synthase/                      strains expressing gluconeogenic enzymes growth glucose. Proc <br />       phosphatase complex Saccharomyces cerevisiae function heat                  Natl Acad Sci U S A 90: 1290&#226;&#8364;&#8220;1294. <br />       shock. Mol Microbiol 24: 687&#226;&#8364;&#8220;695.                                                           37. Sikorski RS, Hieter P (1989) A shuttle vectors yeast host strains <br />  7.   Chen Q, Haddad GG (2004) Role trehalose phosphate synthase trehalose                     designed efficient manipulation DNA Saccharomyces cerevisiae. Genetics <br />       hypoxia: flies mammals. J Exp Biol 207: 3125&#226;&#8364;&#8220;3129.                               122: 19&#226;&#8364;&#8220;27. <br />  8.   Kormish JD, McGhee JD (2005) The C. elegans lethal gut-obstructed gob-1 gene             38. Becker DM, Fikes JD, Guarente L (1991) A cDNA encoding human CCAAT- <br />       trehalose-6-phosphate phosphatase. Dev Biol 287: 35&#226;&#8364;&#8220;47.                                         binding protein cloned functional complementation yeast. Proc Natl Acad <br />  9.   Paul MJ, Primavesi LF, Jhurreea D, Zhang Y (2008) Trehalose metabolism                      Sci U S A 88: 1968&#226;&#8364;&#8220;1972. <br />       signaling. Annu Rev Plant Biol 59: 417&#226;&#8364;&#8220;441.                                                 39. Blanchin-Roland S, Cordero Otero RR, Gaillardin C (1994) Two upstream <br /> 10.   Gonzalez MI, Stucka R, Blazquez MA, Feldmann H, Gancedo C (1992)                                activation sequences control expression XPR2 gene yeast <br />       Molecular cloning CIF1, yeast gene necessary growth glucose. Yeast                  Yarrowia lipolytica. Mol Cell Biol 14: 327&#226;&#8364;&#8220;338. <br />       8: 183&#226;&#8364;&#8220;192.                                                                                 40. Wang H, Le Clainche A, Le Dall MT, Wache Y, Pagot Y, et al. (1998) Cloning <br /> 11.   Luyten K, Koning W, Tesseur I, Ruiz MC, Ramos J, et al. (1993) Disruption                    characterization peroxisomal acyl CoA oxidase ACO3 gene the <br />       Kluyveromyces lactis GGS1 gene causes inability grow glucose                   alkane-utilizing yeast Yarrowia lipolytica. Yeast 14: 1373&#226;&#8364;&#8220;1386. <br />       fructose suppressed mutations reduce sugar uptake. Eur J Biochem             41. Gaillardin C, Ribet AM (1987) LEU2 directed expression beta-galactosidase <br />       217: 701&#226;&#8364;&#8220;713.                                                                                   activity phleomycin resistance Yarrowia lipolytica. Curr Genet 11: 369&#226;&#8364;&#8220;375. <br /> 12.   Blazquez MA, Lagunas R, Gancedo C, Gancedo JM (1993) Trehalose-6-                           42. Sherman DJ, Martin T, Nikolski M, Cayla C, Souciet JL, et al. (2009) <br />       phosphate, new regulator yeast glycolysis inhibits hexokinases. FEBS                  Genolevures: protein families synteny complete hemiascomycetous <br />       Lett 329: 51&#226;&#8364;&#8220;54.                                                                                yeast proteomes genomes. Nucleic Acids Res 37: D550&#226;&#8364;&#8220;554. <br /> 13.   Teusink B, Walsh MC, van Dam K, Westerhoff HV (1998) The danger                          43. Belinchon MM, Flores CL, Gancedo JM (2004) Sampling Saccharomyces cerevisiae <br />       metabolic pathways turbo design. Trends Biochem Sci 23: 162&#226;&#8364;&#8220;169.                           cells rapid filtration improves yield mRNAs. FEMS Yeast Res 4: <br /> 14.   Reinders A, Romano I, Wiemken A, De Virgilio C (1999) The thermophilic                          751&#226;&#8364;&#8220;756. <br />       yeast Hansenula polymorpha does require trehalose synthesis growth high          44. Wallenfels K (1962) [23] [beta]-Galactosidase (crystalline). In: Sidney PC, <br />       temperatures does normal acquisition thermotolerance. J Bacteriol                    Nathan OK, eds. Methods Enzymology Academic Press. pp 212&#226;&#8364;&#8220;219. <br />       181: 4665&#226;&#8364;&#8220;4668.                                                                             45. Kienle I, Burgert M, Holzer H (1993) Assay trehalose acid trehalase <br /> 15.   Petit T, Gancedo C (1999) Molecular cloning characterization gene                    purified Saccharomyces cerevisiae. Yeast 9: 607&#226;&#8364;&#8220;611. <br />       HXK1 encoding hexokinase Yarrowia lipolytica. Yeast 15: 1573&#226;&#8364;&#8220;1584.                 46. Gamo FJ, Portillo F, Gancedo C (1993) Characterization mutations that <br /> 16.   Dujon B (2006) Yeasts illustrate molecular mechanisms eukaryotic genome                  overcome toxic effect glucose phosphoglucose isomerase strains of <br />       evolution. Trends Genet 22: 375&#226;&#8364;&#8220;387.                                                            Saccharomyces cerevisiae. FEMS Microbiol Lett 106: 233&#226;&#8364;&#8220;237. <br /> 17.   Morin M, Monteoliva L, Insenser M, Gil C, Dominguez A (2007) Proteomic                      47. Bergmeyer HU, Bergmeyer J, Grassi M (1987) Methods enzymatic analysis. <br />       analysis reveals metabolic changes yeast hypha transition Yarrowia                 In: Bergmeyer HU, ed. Verlag Chemie. <br />       lipolytica. J Mass Spectrom 42: 1453&#226;&#8364;&#8220;1462.                                                  48. Conway EJ, Downey M (1950) An outer metabolic region yeast cell. <br /> 18.   Holz M, Forster A, Mauersberger S, Barth G (2009) Aconitase overexpression                      Biochem J 47: 347&#226;&#8364;&#8220;355. <br />       changes product ratio citric acid production Yarrowia lipolytica. Appl            49. Thevelein JM, Hohmann S (1995) Trehalose synthase: guard gate of <br />       Microbiol Biotechnol 81: 1087&#226;&#8364;&#8220;1096.                                                             glycolysis yeast? Trends Biochem Sci 20: 3&#226;&#8364;&#8220;10. <br /> 19.   Kamzolova SV, Shishkanova NV, Morgunov IG, Finogenova TV (2003)                             50. Kobayashi N, McEntee K (1993) Identification cis trans components a <br />       Oxygen requirements growth citric acid production Yarrowia lipolytica.               novel heat shock stress regulatory pathway Saccharomyces cerevisiae. Mol Cell <br />       FEMS Yeast Res 3: 217&#226;&#8364;&#8220;222.                                                                      Biol 13: 248&#226;&#8364;&#8220;256. <br /> 20.   Madzak C, Gaillardin C, Beckerich JM (2004) Heterologous protein expression                 51. Bulman AL, Hubl ST, Nelson HC (2001) The DNA-binding domain yeast <br />       secretion non-conventional yeast Yarrowia lipolytica: review.                      heat shock transcription factor independently regulates N- C- <br />       J Biotechnol 109: 63&#226;&#8364;&#8220;81.                                                                        terminal activation domains. J Biol Chem 276: 40254&#226;&#8364;&#8220;40262. <br /> 21.   Beopoulos A, Cescut J, Haddouche R, Uribelarrea JL, Molina-Jouve C, et al.                  52. Conlin LK, Nelson HC (2007) The natural osmolyte trehalose positive <br />       (2009) Yarrowia lipolytica model bio-oil production. Prog Lipid Res 48:                regulator heat-induced activity yeast heat shock transcription factor. <br />       375&#226;&#8364;&#8220;387.                                                                                        Mol Cell Biol 27: 1505&#226;&#8364;&#8220;1515. <br /> 22.   Sakai Y, Oku M, van der Klei IJ, Kiel JA (2006) Pexophagy: autophagic                       53. Marchler G, Schuller C, Adam G, Ruis H (1993) A Saccharomyces cerevisiae UAS <br />       degradation peroxisomes. Biochim Biophys Acta 1763: 1767&#226;&#8364;&#8220;1775.                               element controlled protein kinase A activates transcription response a <br /> 23.   Flores CL, Martinez-Costa OH, Sanchez V, Gancedo C, Aragon JJ (2005) The                        variety stress conditions. EMBO J 12: 1997&#226;&#8364;&#8220;2003. <br />       dimorphic yeast Yarrowia lipolytica possesses atypical phosphofructokinase:              54. Moradas-Ferreira P, Costa V, Piper P, Mager W (1996) The molecular defences <br />       characterization enzyme encoding gene. Microbiology 151:                         reactive oxygen species yeast. Mol Microbiol 19: 651&#226;&#8364;&#8220;658. <br />       1465&#226;&#8364;&#8220;1474.                                                                                  55. Gancedo JM, Clifton D, Fraenkel DG (1977) Yeast hexokinase mutants. J Biol <br /> 24.   Hirai M, Tanaka A, Fukui S (1975) Difference pyruvate kinase regulation                      Chem 252: 4443&#226;&#8364;&#8220;4444. <br />       groups yeasts. Biochim Biophys Acta 391: 282&#226;&#8364;&#8220;291.                            56. Herrero P, Galindez J, Ruiz N, Martinez-Campa C, Moreno F (1995) <br /> 25.   Le Dall M, Nicaud J, Treton BY, Gaillardin CM (1996) The 3-phosphoglycerate                     Transcriptional regulation Saccharomyces cerevisiae HXK1, HXK2 and <br />       kinase gene yeast Yarrowia lipolytica de-represses gluconeogenic                      GLK1 genes. Yeast 11: 137&#226;&#8364;&#8220;144. <br />       substrates. Curr Genet 29: 446&#226;&#8364;&#8220;456.                                                         57. Swanson RN, Conesa C, Lefebvre O, Carles C, Ruet A, et al. (1991) Isolation of <br /> 26.   Flores CL, Gancedo C (2005) Yarrowia lipolytica mutants devoid pyruvate                      TFC1, gene encoding DNA-binding subunits yeast transcription <br />       carboxylase activity unusual growth phenotype. Eukaryot Cell 4:                         factor tau (TFIIIC). Proc Natl Acad Sci U S A 88: 4887&#226;&#8364;&#8220;4891. <br />       356&#226;&#8364;&#8220;364.                                                                                    58. Huang Y, Hamada M, Maraia RJ (2000) Isolation cloning subunits <br /> 27.   Jardon R, Gancedo C, Flores CL (2008) The gluconeogenic enzyme fructose-                        fission yeast TFIIIC complex includes ortholog human <br />       1,6-bisphosphatase dispensable growth yeast Yarrowia lipolytica                regulatory protein TFIIICbeta. J Biol Chem 275: 31480&#226;&#8364;&#8220;31487. <br />       gluconeogenic substrates. Eukaryot Cell 7: 1742&#226;&#8364;&#8220;1749.                                       59. Goddijn OJ, Verwoerd TC, Voogd E, Krutwagen RW, Graaf PT, et al. <br /> 28.   Arisan-Atac I, Wolschek MF, Kubicek CP (1996) Trehalose-6-phosphate                             (1997) Inhibition trehalase activity enhances trehalose accumulation in <br />       synthase A affects citrate accumulation Aspergillus niger conditions                transgenic plants. Plant Physiol 113: 181&#226;&#8364;&#8220;190. <br />       high glycolytic flux. FEMS Microbiol Lett 140: 77&#226;&#8364;&#8220;83.                                       60. Mu&#194;&#168;ller J, Aeschbacher RA, Wingler A, Boller T, Wiemken A (2001) Trehalose <br /> 29.   Wolschek M, Kubicek CP (1998) Biochemistry citric acid accumulation                       trehalase Arabidopsis. Plant Physiol 125: 1086&#226;&#8364;&#8220;1093. <br />       Aspergillus niger. In Kristiansen B, Linden J, Mattey M, eds. Citric acid                   61. Mu&#194;&#168;ller J, Boller T, Wiemken A (1995) Effects validamycin A, potent <br />       biotechnology. London: Taylor Francis. pp 11&#226;&#8364;&#8220;33.                                            trehalase inhibitor, phytohormones trehalose metabolism roots and <br /> 30.   Gaillardin CM, Charoy V, Heslot H (1973) A study copulation, sporulation                     root nodules soybean cowpea. Planta 197: 362&#226;&#8364;&#8220;368. <br />       meiotic segregation Candida lipolytica. Arch Mikrobiol 92: 69&#226;&#8364;&#8220;83.                    62. Arguelles JC (1997) Thermotolerance trehalose accumulation induced by <br /> 31.   Yarrow D (1998) Methods isolation maintenance identification                     heat shock yeast cells Candida albicans. FEMS Microbiol Lett 146: 65&#226;&#8364;&#8220;71. <br />       yeasts. In: Kurtzman CP, Fell JW, eds. The yeasts: A taxonomic study. New                   63. Kandror O, Bretschneider N, Kreydin E, Cavalieri D, Goldberg AL (2004) <br />       York: Elsevier. pp 77&#226;&#8364;&#8220;100.                                                                      Yeast adapt near-freezing temperatures STRE/Msn2,4-dependent <br /> 32.   Lopez MC, Nicaud JM, Skinner HB, Vergnolle C, Kader JC, et al. (1994) A                         induction trehalose synthesis certain molecular chaperones. Mol Cell <br />       phosphatidylinositol/phosphatidylcholine transfer protein required differ-               13: 771&#226;&#8364;&#8220;781. <br />       entiation dimorphic yeast Yarrowia lipolytica yeast mycelial         64. Martinez-Pastor MT, Marchler G, Schuller C, Marchler-Bauer A, Ruis H, et al. <br />       form. J Cell Biol 125: 113&#226;&#8364;&#8220;127.                                                                 (1996) The Saccharomyces cerevisiae zinc finger proteins Msn2p Msn4p are <br />  <br />  <br />  <br />          PLoS ONE | www.plosone.org                                                          10                            September 2011 | Volume 6 | Issue 9 | e23695 <br />                                                                                                                                 Effects TPS1 Disruption Yarrowia lipolytica <br />  <br />  <br />       required transcriptional induction stress response element                  77. Kramarenko T, Karp H, Jarviste A, Alamae T (2000) Sugar repression the <br />       (STRE). EMBO J 15: 2227&#226;&#8364;&#8220;2235.                                                                   methylotrophic yeast Hansenula polymorpha studied using hexokinase-negative, <br /> 65.   Ye Y, Zhu Y, Pan L, Li L, Wang X, et al. (2009) Gaining insight                        glucokinase-negative double kinase-negative mutants. Folia Microbiol <br />       response logic Saccharomyces cerevisiae heat shock combining expression                (Praha) 45: 521&#226;&#8364;&#8220;529. <br />       profiles metabolic pathways. Biochem Biophys Res Commun 385:                           78. Fillinger S, Chaveroche MK, van Dijck P, Vries R, Ruijter G, et al. (2001) <br />       357&#226;&#8364;&#8220;362.                                                                                        Trehalose required acquisition tolerance variety stresses the <br /> 66.   Hurtado CA, Rachubinski RA (1999) MHY1 encodes C2H2-type zinc finger                          filamentous fungus Aspergillus nidulans. Microbiology 147: 1851&#226;&#8364;&#8220;1862. <br />       protein promotes dimorphic transition yeast Yarrowia lipolytica.                79. Wilson JE (2003) Isozymes mammalian hexokinase: structure, subcellular <br />       J Bacteriol 181: 3051&#226;&#8364;&#8220;3057.                                                                     localization metabolic function. J Exp Biol 206: 2049&#226;&#8364;&#8220;2057. <br /> 67.   Neves MJ, Hohmann S, Bell W, Dumortier F, Luyten K, et al. (1995) Control                80. Van Aelst L, Hohmann S, Bulaya B, Koning W, Sierkstra L, et al. (1993) <br />       glucose influx glycolysis pleiotropic effects studied different isogenic            Molecular cloning gene involved glucose sensing yeast Saccharomyces <br />       sets Saccharomyces cerevisiae mutants trehalose biosynthesis. Curr Genet 27:              cerevisiae. Mol Microbiol 8: 927&#226;&#8364;&#8220;943. <br />       110&#226;&#8364;&#8220;122.                                                                                    81. Lin X, Heitman J (2005) Chlamydospore formation hyphal growth in <br /> 68.   De Silva-Udawatta MN, Cannon JF (2001) Roles trehalose phosphate <br />                                                                                                       Cryptococcus neoformans. Eukaryot Cell 4: 1746&#226;&#8364;&#8220;1754. <br />       synthase yeast glycogen metabolism sporulation. Mol Microbiol 40: <br />                                                                                                   82. Lowe RG, Lord M, Rybak K, Trengove RD, Oliver RP, et al. (2009) Trehalose <br />       1345&#226;&#8364;&#8220;1356. <br />                                                                                                       biosynthesis involved sporulation Stagonospora nodorum. Fungal Genet Biol <br /> 69.   Wolschek MF, Kubicek CP (1997) The filamentous fungus Aspergillus niger <br />                                                                                                       46: 381&#226;&#8364;&#8220;389. <br />       contains &#226;&#8364;&#732;&#226;&#8364;&#732;differentially regulated&#226;&#8364;&#8482;&#226;&#8364;&#8482; trehalose-6-phosphate synthase-encoding <br />       genes, tpsA tpsB. J Biol Chem 272: 2729&#226;&#8364;&#8220;2735.                                           83. Kretschmer A, Wagner F (1983) Characterization biosynthetic intermediates <br /> 70.   Al-Bader N, Vanier G, Liu H, Gravelat FN, Urb M, et al. (2010) Role                          trehalose dicorynomycolate Rhodococcus erithropolis grown n-alcanes. <br />       trehalose biosynthesis Aspergillus fumigatus development, stress response,               Biochim Biophys Acta 753: 306&#226;&#8364;&#8220;313. <br />       virulence. Infect Immun 78: 3007&#226;&#8364;&#8220;3018.                                                      84. Vergne I, Daffe M (1998) Interaction mycobacterial glycolipids host cells. <br /> 71.   Adachi N, Lieber MR (2002) Bidirectional gene organization: common                            Front Biosci 3: d865&#226;&#8364;&#8220;876. <br />       architectural feature human genome. Cell 109: 807&#226;&#8364;&#8220;809.                               85. Amaral PF, Lehocky M, Barros-Timmons AM, Rocha-Leao MH, Coelho MA, <br /> 72.   Davila Lopez M, Martinez Guerra JJ, Samuelsson T (2010) Analysis gene                        et al. (2006) Cell surface characterization Yarrowia lipolytica IMUFRJ 50682. <br />       order conservation eukaryotes identifies transcriptionally functionally                  Yeast 23: 867&#226;&#8364;&#8220;877. <br />       linked genes. PLoS One 5: e10654.                                                           86. Castells-Roca L, Garcia-Martinez J, Moreno  <br /> </body></html>