<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 19 No. 21 pp. 5801&#194;&#177;5812, 2000 <br />  <br />  <br /> LCD1: essential gene involved checkpoint <br /> control regulation MEC1 signalling <br /> pathway Saccharomyces cerevisiae <br />  <br />  <br /> John Rouse Stephen P.Jackson1                                    instrumental regard. The &#194;&#174;rst yeast checkpoint gene <br /> Wellcome Trust Cancer Research Campaign Institute Cancer <br />                                                                      identi&#194;&#174;ed Weinert Hartwell (1988), who <br /> and Developmental Biology Department Zoology, University   showed RAD9 gene required G2/M <br /> Cambridge, Tennis Court Road, Cambridge CB2 1QR, UK                  checkpoint response DNA damage. Subsequent work <br /> 1 <br />   Corresponding author <br />                                                                      laboratories shown RAD9 the <br /> e-mail: spj13@mole.bio.cam.ac.uk                                     MEC1 pathway&#195;?a complex protein phosphorylation <br />                                                                      cascade activated response DNA damage <br /> We identi&#194;&#174;ed YDR499W Saccharomyces cerevisiae                   inhibition DNA replication (reviewed Lowndes <br /> open reading frame homology check-                   Murguia, 2000). Mec1p member family of <br /> point proteins, including S.cerevisiae Rfc5p                     large protein kinases, termed PIKKs, sequence <br /> Schizosaccharomyces pombe Rad26. Disruption                       similarity catalytic domain phosphatidylinositol <br /> YDR499W (termed LCD1) results lethality                   3-kinase. In circumstances, Mec1p functions redund- <br /> rescued increasing cellular deoxyribonucleotide                   antly Tel1p (Greenwell et al., 1995; Morrow et al., <br /> levels. Cells lacking LCD1 sensitive                   1995; Sanchez et al., 1996; Vialard et al., 1998), another <br /> range DNA-damaging agents, including UV irradi-                   PIKK closest S.cerevisiae relative Mec1p. <br /> ation, inhibition DNA replication. LCD1                PIKKs conserved evolution, homo- <br /> is necessary phosphorylation activation               logues Mec1p Tel1p in <br /> Rad53p response DNA damage DNA repli-                       Schizosaccharomyces pombe (termed Rad3 Tel1, <br /> cation blocks, Chk1p activation response               respectively) humans (ATR ATM). In <br /> DNA damage. LCD1 required ef&#194;&#174;cient DNA                   S.pombe, Rad3 appears constitutively bound the <br /> damage-induced phosphorylation Rad9p                      Rad26 protein (Lindsay et al., 1998; Martinho et al., <br /> the association Rad9p FHA2 domain                     1998), component Rad3 pathway <br /> Rad53p DNA damage. In addition, cells lacking                  that, mutated, leads <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> severe that <br /> LCD1 completely defective G1/S G2/M                   caused mutations Rad3 (Al-Khodairy et al., 1994; <br /> DNA damage checkpoints. Finally, reveal                      Lindsay et al., 1998). This indicates Rad26 plays a <br /> endogenous Mec1p co-immunoprecipitates Lcd1p                    crucial role Rad3 pathway. <br /> both treatment DNA-damaging                       Schizosaccharomyces pombe Rad3 essential for <br /> agents. These results indicate Lcd1p pivotal               cell viability, disruption S.cerevisiae MEC1 <br /> checkpoint regulator, involved essential                 gene causes lethality (Kato Ogawa, 1994; Weinert <br /> and checkpoint functions Mec1p pathway.                       et al., 1994). However, lethality suppressed by <br /> Keywords: checkpoint/DNA damage/MEC1/protein                         increasing intracellular concentration deoxyribo- <br /> phosphorylation/RAD53                                                nucleotides (dNTPs), overexpression the <br />                                                                      catalytic subunits (Rnr1p, Rnr3p) ribonucleotide <br />                                                                      reductase (RNR) tetramer (Desany et al., 1998), which <br />                                                                      regulates rate-limiting step dNTP synthesis, by <br /> Introduction                                                         disrupting gene encoding Sml1p (Zhao et al., 1998), <br /> DNA damage deleterious consequences                     directly binds inhibits Rnr1p (Chabes et al., <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>survival</span> cells adopted multiple strategies              1999). At present, molecular basis essential <br /> tolerating damage genetic material. Several                 function MEC1 normal cell cycle unclear. <br /> different, conserved repair systems physically              In addition, case S.pombe cells lacking Rad3 <br /> remove bypass speci&#194;&#174;c types DNA lesion (Friedberg              (Lindsay et al., 1998), S.cerevisiae cells disrupted for <br /> et al., 1995). Also, response DNA damage, cells            MEC1 function exquisitely sensitive DNA <br /> slow progression different cell cycle             replication inhibitor hydroxyurea (HU) wide <br /> phases, provide time repair occur prevent           range DNA-damaging agents (Weinert et al., 1994; <br /> mutations propagated. These cell cycle delays,            Sanchez et al., 1996; Desany et al., 1998). Indeed, an <br /> termed `checkpoints&apos; (Weinert Hartwell, 1988),                   intact MEC1 gene required DNA replication <br /> prevent replication damaged DNA (G1/S intra-S                 checkpoint DNA damage checkpoint responses at <br /> checkpoints) segregation damaged chromosomes                   cell cycle stages (Weinert et al., 1994; Paulovich and <br /> (the G2/M checkpoint). In addition, DNA replication              Hartwell, 1995). Although increasing intracellular dNTP <br /> checkpoint ensures cell appropriate DNA             levels overcomes lethality cells lacking MEC1, the <br /> content entering mitosis.                                     checkpoint defects sensitivity genotoxic insults of <br />    Much attention focused identi&#194;&#174;cation                cells, cells containing hypomorphic alleles of <br /> genes involved DNA repair checkpoint control,             MEC1, affected RNR overexpression by <br /> the budding yeast Saccharomyces cerevisiae                  deletion SML1 (Desany et al., 1998; Zhao et al., 1998). <br />  <br /> &#195;&#163; European Molecular Biology Organization                                                                                  5801 <br />  J.Rouse S.P.Jackson <br />  <br />  <br />    It demonstrated Mec1p              major cell cycle checkpoint defects. Furthermore, we <br /> intrinsic protein kinase activity, shown   reveal activation key effector molecules the <br /> several key regulators Mec1p-dependent signalling        MEC1 pathway requires LCD1 gene, Lcd1p <br /> pathway phosphorylated MEC1-dependent               interacts physically Mec1p. These results identify <br /> manner response DNA damage DNA replication          Lcd1p key checkpoint component provide <br /> is blocked. For example, phosphorylation state          insights molecular basis Mec1p-dependent <br /> protein kinase activity Rad53p increase response       signalling responses. <br /> DNA-damaging agents HU MEC1-dependent <br /> fashion (Allen et al., 1994; Sun et al., 1996), the <br /> presence agents, cells harbouring mutations in <br />                                                                 Results <br /> RAD53 rapidly lose viability (Allen et al., 1994; Sun et al.,   Saccharomyces cerevisiae LCD1 contains sequence <br /> 1996; Fay et al., 1997). Rad53p acts downstream       motifs DNA repair cell cycle <br /> effector Mec1p-dependent signalling pathway. Like        checkpoint proteins <br /> MEC1, RAD53 essential gene, overexpression         When sequence S.cerevisiae Rfc5p used a <br /> RNR subunits deletion SML1 suppress lethal        BLAST search yeast protein database, best <br /> phenotype null mutations RAD53 (Allen et al.,         matches corresponded small subunits RFC, <br /> 1994; Desany et al., 1998; Zhao et al., 1998). The Chk1p        Rad24p (Grif&#194;&#174;ths et al., 1995; Green et al., 2000) one <br /> protein kinase lies downstream Mec1p branch        ORF situated chromosome IV: Ydr499w <br /> of pathway partly distinct        (Figure 1A). For reasons discussed below, hereafter <br /> branch involves Rad53p (Sanchez et al., 1999;              refer YDR499W gene LCD1. Lcd1p high <br /> J.Rouse S.P.Jackson, unpublished data). Signi&#194;&#174;-             proportion acidic residues (pI 5.3) predicted <br /> cantly, work laboratories shown           molecular mass 87 kDa. The observed sequence <br /> phosphorylation Rad53p Chk1p response              homology (25% identity, 45% similarity; Figure 1B) <br /> DNA damage requires intact RAD9 gene,               extends region ~180 amino acid residues that <br /> exposure cells DNA damage results rapid            align C-terminal half Rfc5p. Several the <br /> sustained hyperphosphorylation Rad9p, manner            residues conserved Rfc5p Lcd1p found <br /> that depends MEC1 (de la Torre-Ruiz et al., 1998;            conserved Rfc5 proteins diverse <br /> Emili, 1998; Vialard et al., 1998; Sanchez et al., 1999). In    species (data shown), suggesing residues are <br /> this regard then, Rad9p functions downstream Mec1p,          likely important biological function. Interest- <br /> but upstream effector kinases Chk1p Rad53p.          ingly, noticed sequence homology Lcd1p <br />    The RAD24 epistasis group (RAD24, RAD17, MEC3                Rad50p (data shown), involved DNA <br /> and DDC1) functions additively RAD9,              double-strand break repair S.cerevisiae, between <br /> separate branch pathway, mediating phosphoryl-        Lcd1p S.pombe Rad26, &#194;&#174;ssion yeast checkpoint gene <br /> ation activation Rad53p response DNA               obvious homologue S.cerevisiae (Figure 1A). <br /> damage (Paulovich et al., 1997; la Torre-Ruiz et al.,        The degree homology Lcd1p Rad26, <br /> 1998). Mutations genes result sensitivity    however, low occurs small patches <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> DNA damage, HU, cause defects        (Figure 1C) largest patch showing 23% overall <br /> G1/S G2/M DNA damage checkpoints (Lydall                identity. This region predicted form coiled- <br /> Weinert, 1995; Paulovich Hartwell, 1995; Longhese           coil structure proteins. Rad26 does have <br /> et al., 1997). Notably, Rad24p sequence similarity       detectable homology, however, Rfc5p Rad50p. <br /> the small subunits replication factor C (RFC; Grif&#194;&#174;ths       Because similar sequences Lcd1p several <br /> et al., 1995). The RFC complex, comprises large         organisms involved cell cycle checkpoints and/or <br /> subunit (Rfc1p) small subunits (Rfc2p&#194;&#177;5p;              DNA repair, investigated potential roles this <br /> Cullmann et al., 1995), key regulator initiation     uncharacterized protein cellular processes. <br /> DNA synthesis. Two groups shown Rad24p <br /> forms distinct complex small RFC subunits,       Disruption LCD1 causes lethality be <br /> not Rfc1p (Shimomura et al., 1998; Green et al.,           suppressed increasing cellular dNTP pools <br /> 2000). In addition, cells harbouring temperature-sensitive      To investigate functions Lcd1p, analysed the <br /> alleles RFC2 (Noskov et al., 1998) RFC5            consequences ablating gene encoding it. Sporulation <br /> (Sugimoto et al., 1996, 1997) defective DNA damage         subsequent tetrad analysis diploid cells which <br /> and replication checkpoints, fail activate Rad53p     copy LCD1 disrupted showed maximum <br /> response genotoxic agents.                                   spores tetrad viable, all <br />    <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> database searches, identi&#194;&#174;ed                 cases viability failed segregate marker used to <br /> S.cerevisiae YDR499W previously uncharacterized            disrupt LCD1 (data shown). This suggests an <br /> open reading frame (ORF) product amino acid           intact LCD1 gene required cell growth. There are <br /> sequence homology DNA repair DNA                 potential explanations lethality LCD1 <br /> damage checkpoint proteins, including budding yeast             disruption. As mentioned earlier, Lcd1p sequence <br /> Rfc5p Rad50p, S.pombe Rad26. Here,              similarity S.cerevisiae Rfc5p S.pombe Rad26. <br /> that disruption gene (termed LCD1 lethal,           Since RFC5 shown interact genetically with <br /> checkpoint-defective, DNA damage sensitive) results          MEC1 pathway, Rad26 interacts Rad3 <br /> lethality rescued increasing cellular dNTP       (the S.pombe homologue Mec1p), speculated that <br /> levels. Cells lacking LCD1 extremely sensitive DNA       Lcd1p involved regulation MEC1 <br /> damage inhibition DNA replication,           pathway. As discussed above, MEC1 plays essential <br />  <br /> 5802 <br />                                                                                                                 LCD1: novel checkpoint gene <br />  <br />  <br />  <br />  <br /> Fig. 1. Lcd1p shows sequence homology checkpoint proteins disruption LCD1 results lethality rescued RNR3 <br /> overexpression. (A) Schematic representation regions sequence homology Lcd1p various checkpoint proteins. (B C) The entire <br /> amino acid sequence S.cerevisiae Ydr499w (B) entire S.pombe Rad26 amino sequence (C) used query WU-BLAST search at <br /> the Saccharomyces Genome Database (genome-www.stanford.edu/Saccharomyces/). Sequences aligned using ClustalX program the <br /> Boxshade Server (www.ch.embnet.org/). Identical amino acid residues shaded black similar residues highlighted grey. (D) Ten-fold <br /> serial dilutions cells containing pLCD1, pRNR3-TRP1 vector (pRS414) contains TRP1 gene, plated onto <br /> SC-TRP SC-TRP containing 5-FOA, incubated 4 days 30&#194;&#176;C. LCD1 indicates parental strain carrying pRS414 pLCD1. <br />  <br />  <br />  <br /> role unperturbed cell cycle, lethality                     To investigate possibility thoroughly, <br /> Dmec1 cells rescued overexpressing                      LCD1-disrupted cells containing RNR3-overexpression <br /> catalytic subunits RNR (Desany et al., 1998)                        plasmid (pRNR3) transformed pLCD1, a <br /> disrupting SML1 gene (Zhao et al., 1998). Notably,                       plasmid harbouring URA3 gene LCD1 the <br /> Southern blotting PCR analyses showed                       control promoter. These cells next <br /> LCD1 disrupted wild-type haploid cells,                   encouraged lose pRNR3 growth non-selective <br /> could disrupted haploid cells overexpressing Rnr3p                     medium, resulting cells tested the <br /> or lacking SML1 (data shown). This suggested                    retention loss plasmid markers their <br /> lethality LCD1 disruption rescued increas-                    ability grow presence 5-&#194;&#175;uoro-orotic acid <br /> ing cellular dNTP pools.                                                     (FOA). This drug toxic cells containing functional <br />  <br />                                                                                                                                             5803 <br />  J.Rouse S.P.Jackson <br />  <br />  <br />                                                                          compared wild-type cells. Introduction pLCD1 <br />                                                                          plasmid, bearing LCD1 gene regulation its <br />                                                                          promoter, fully rescued sensitivity Dlcd1Dsml1 <br />                                                                          cells types DNA damage examined (data not <br />                                                                          shown). Signi&#194;&#174;cantly, cells lacking LCD1 an <br />                                                                          order magnitude sensitive MMS UV the <br />                                                                          Dmec1Dsml1 strain (Figure 2A B). Furthermore, cells <br />                                                                          lacking LCD1 MEC1 signi&#194;&#174;cantly more <br />                                                                          sensitive DNA damage cells lacking LCD1 alone, <br />                                                                          indicating genes partially epistatic <br />                                                                          (Figure 2A&#194;&#177;C). <br />                                                                             To LCD1 involved cellular responses <br />                                                                          inhibition DNA replication, examined effect <br />                                                                          exposing Dlcd1 mutant cells HU. This drug inhibits <br />                                                                          DNA replication, cells defective DNA replication <br />                                                                          checkpoint (e.g. Dmec1Dsml1 cells) survive the <br />                                                                          presence HU. As shown Figure 2D, cells lacking <br />                                                                          LCD1 extremely sensitive presence HU, even <br />                                                                          Dmec1Dsm1 mutant cells. These data imply <br />                                                                          LCD1 plays crucial role allowing cells survive <br /> Fig. 2. LCD1 disruption results hypersensitivity DNA damage    DNA replication hindered. <br /> hydroxyurea. Dsml1, Dlcd1Dsml1, Dmec1Dsml1 Dlcd1Dmec1Dsml1 <br /> cells grown mid-log phase (OD600 0.6) liquid culture. At <br /> this point, case MMS (B) HU (D), relevant drug      LCD1 required Rad53p activation response <br /> added cells grown 30&#194;&#176;C 2 h. Cell suspensions    DNA damage stalled DNA replication forks <br /> were diluted 100-fold, plated YPD agar incubated 30&#194;&#176;C    activation Chk1p response DNA <br /> 3 days. In case UV irradiation (A), cells diluted, spread    damage <br /> onto YPD agar plates placed germicidal UV lamp <br /> and irradiated 254 nm, delivery rate 3 J/m2/s. For            Given phenotypic similarities cells lacking <br /> experiments involving IR (C), diluted cells irradiated       MEC1 cells lacking LCD1, examined effect of <br /> Torrex X-ray machine, delivery rate 3.3 Gy/min,           disrupting LCD1 function individual effector <br /> spreading YPD agar.                                                 molecules MEC1 signalling pathway. When normal <br />                                                                          cells exposed DNA damage complete <br />                                                                          DNA replication, phosphorylation state kinase <br /> URA3, cells lose pLCD1 plasmid                  activity Rad53p increase MEC1-dependent manner <br /> should die presence. These studies revealed cells            (Allen et al., 1994; Sun et al., 1996). This increase in <br /> still retaining pRNR3 grew medium containing                  Rad53p activity thought elict cell cycle checkpoints <br /> lacking 5-FOA, did harbour                   (Allen et al., 1994; Sun et al., 1996; Fay et al., 1997) and <br /> plasmid able grow absence                 orchestrate DNA repair (see Discussion). To <br /> the presence 5-FOA (and thus, absence Rnr3p                 activation Rad53p depends LCD1, <br /> overexpression, require pLCD1 plasmid survive;                    Rad53p kinase activity determined treating cells <br /> Figure 1D). In parallel studies, veri&#194;&#174;ed haploid                 different agents, measuring ability Rad53p <br /> cells disrupted SML1 able sustain deletion               autophosphorylate, described previously (Pellicioli <br /> of LCD1 (data shown). Taken together, data                     et al., 1999). Rad53p kinase activity low untreated <br /> indicate increased dNTP pools necessary                     cells (U; Figure 3A, panel), exposure wild- <br /> cellular survival absence LCD1.                                type cells MMS (M) UV-mimetic drug <br />                                                                          4-nitroquinoline oxide (4-NQO; N) stimulates kinase <br /> Cells lacking LCD1 sensitive DNA damage                           activity 20- 25-fold. Similarly, addition phleomycin <br /> and inhibition DNA replication                                        (P; Figure 3A) induction restriction endonuclease <br /> Further analyses LCD1-disrupted cells revealed                   EcoRI (data shown), induce DNA DSBs, <br /> they apparently normal cell morphology cell                  leads activation Rad53p, albeit lower extent. <br /> cycle distribution, grow 23, 30 37&#194;&#176;C, albeit                 Strikingly, activation Rad53p response of <br /> slowly wild-type cells temperatures                agents severely reduced cells lacking LCD1 <br /> (data shown). Given sequence homology                    (Figure 3A), highest concentrations drugs <br /> LCD1 certain DNA damage response genes,                      used (data shown). In control experiments, DNA <br /> similarity LCD1 MEC1 regard                      damage-induced autophosphorylating band 90 kDa (the <br /> essential function(s), prompted investigate               apparent mass Rad53p western blot analysis) was <br /> potential role LCD1 responding DNA damage.                      absent strains disrupted RAD53, non- <br /> Cells lacking LCD1 extremely sensitive UV                         speci&#194;&#174;c bands lower molecular weight present <br /> irradiation (Figure 2A) presence increasing                strains (data shown Pellicioli et al., 1999). <br /> concentrations alkylating agent methyl-methane-                   Consistent data, DNA damage-induced <br /> sulfonate (MMS; Figure 2B)&#195;?agents primarily cause                   hyperphosphorylation Rad53p occurs wild-type <br /> helix-distorting modi&#194;&#174;cations DNA. Dlcd1 cells               strains essentially abolished LCD1- <br /> hypersensitive ionizing radiation (IR; Figure 2C),                 disrupted cells (data shown). Furthermore, induc- <br /> agent causes DNA double-strand breaks (DSBs),                       tion Rad53p kinase activity response HU also <br />  <br /> 5804 <br />                                                                                                         LCD1: novel checkpoint gene <br />  <br />  <br />                                                                          together, results reveal LCD1 plays crucial role <br />                                                                          regulating Rad53p activity response number of <br />                                                                          genotoxic agents. <br />                                                                             It recently shown Chk1p protein kinase also <br />                                                                          phosphorylated MEC1-dependent manner in <br />                                                                          response DNA damage (Sanchez et al., 1999; J.Rouse <br />                                                                          S.P.Jackson, unpublished data), thought that <br />                                                                          Chk1p catalyses DNA damage-induced phosphorylation <br />                                                                          stabilization anaphase inhibitor Pds1p, thereby <br />                                                                          arresting cells mitosis (Cohen-Fix Koshland, 1997; <br />                                                                          Sanchez et al., 1999). A potential role LCD1 in <br />                                                                          phosphorylation haemagglutinin (HA)-tagged Chk1p <br />                                                                          (Sanchez et al., 1999) examined western blot <br />                                                                          analysis. Treatment wild-type cells MMS 4- <br />                                                                          NQO, lesser extent phleomycin, induced an <br />                                                                          electrophoretic mobility shift HA-Chk1p (Figure 3C) <br />                                                                          compared HA-Chk1p untreated cells. The band <br />                                                                          cross-reacted HA antibody observed <br />                                                                          cells express HA-Chk1p, the <br />                                                                          electrophoretic mobility shift observed indicative of <br />                                                                          protein phosphorylation, treatment cell extracts with <br />                                                                          phage l phosphatase reverses shift (Sanchez et al., <br />                                                                          1999; data shown). In contrast, Chk1p did become <br />                                                                          phosphorylated response DNA-damaging <br />                                                                          agents examined cells lacking LCD1, introduction <br />                                                                          plasmid expressing Lcd1p endogenous levels <br />                                                                          restored DNA damage-induced phosphorylation of <br />                                                                          Chk1p observed wild-type cells (Figure 3C). Thus, an <br />                                                                          intact LCD1 gene required DNA damage-induced <br />                                                                          activation Chk1p. <br />  <br />                                                                          LCD1 required DNA damage-induced <br />                                                                          phosphorylation Rad9p <br />                                                                          Previous work established DNA damage results in <br />                                                                          generation series hyperphosphorylated forms <br />                                                                          S.cerevisiae Rad9p migrate slowly on <br />                                                                          SDS&#194;&#177;polyacrylamide gels unmodi&#194;&#174;ed protein <br /> Fig. 3. LCD1 required activation Rad53p response HU or <br />                                                                          (Emili et al., 1998; Vialard et al., 1998). Furthermore, in <br /> DNA-damaging agents activation Chk1p response DNA       G1 phase cell cycle, DNA damage-induced <br /> damage. Cells (all carrying pRNR3 plasmid) grown mid-        phosphorylation Rad9p dependent MEC1 on <br /> log phase incubated presence MMS (0.02%; M), 4-NQO         RAD24 epistasis group (Emili et al., 1998; Vialard <br /> (5 mg/ml; N), phleomycin (5 mg/ml; P) (A C) HU (0.1 M; B),     et al., 1998). In contrast, S- M-phase, MEC1 and <br /> without drug addition (U), 2 h 30&#194;&#176;C. After lysis TCA, <br /> extracts subjected situ analysis Rad53p activity           TEL1 function redundantly regard RAD24 <br /> (A B, panels) western blot analysis assess Rad53p          epistasis group required (de la Torre-Ruiz et al., <br /> protein (bottom panels; 12% SDS&#194;&#177;polyacrylamide gels), using gel          1998). We investigated potential role of <br /> conditions compacted phosphoforms protein. Rad53p       LCD1 Rad9p phosphorylation western immunoblot <br /> autophosphorylation quantitated using phosphoimager (middle <br /> panels). Phosphorylation HA-tagged Chk1p analysed by <br />                                                                          analysis. As shown Figure 4A, Rad9p becomes <br /> electrophoresis extracts cells transformed pHA-CHK1         hyperphosphorylated G1-arrested wild-type cells after <br /> on 10% SDS&#194;&#177;polyacrylamide gels (C). Molecular weight markers             exposed 4-NQO (the diminished overall <br /> (kilodaltons) shown right-hand upper panel     Rad9p signal DNA damage multiple <br /> each case.                                                               phosphorylation variants result changes in <br />                                                                          Rad9p levels; legend Figure 4). Notably, this <br />                                                                          induction Rad9p phosphorylation markedly reduced <br /> abrogated Dlcd1 mutant cells (Figure 3B, panel).                  strains disrupted LCD1 reduced, <br /> Importantly, case Rad53p activation defect                albeit lesser extent, cells lacking MEC1 <br /> the LCD1-disrupted strain restored introduction               (Figure 4A). By contrast, consistent previous <br /> of pLCD1 (Figure 3A B), expresses Lcd1p                     work (de la Torre-Ruiz, 1998), asynchronous cultures <br /> endogenous levels, western blot analysis revealed               wild-type, Dlcd1 mutant Dmec1 mutant cells, there <br /> the levels Rad53p protein remained essentially constant               little difference gross phosphorylation Rad9p in <br /> in various genetic backgrounds conditions                        response MMS 4-NQO (data shown). <br /> employed (Figure 3A B, lower panels; electrophoresis                 These data reveal parallels LCD1 and <br /> conditions compacted phospho forms Rad53p                    MEC1 terms effects Rad9p phosphorylation <br /> were used aid comparison protein levels). Taken                    DNA damage. <br />  <br />                                                                                                                                 5805 <br />  J.Rouse S.P.Jackson <br />  <br />  <br />                                                                            ation Rad9p allowed demonstration overall <br />                                                                            levels Rad9p essentially equal samples). <br />                                                                            Taken together, results indicate LCD1 is <br />                                                                            required ef&#194;&#174;cient DNA damage-induced phos- <br />                                                                            phorylation Rad9p that, absence LCD1 <br />                                                                            function, Rad9p longer able bind FHA2 <br />                                                                            domain Rad53p exposure DNA damage. <br />  <br />                                                                            LCD1 required DNA damage checkpoint <br />                                                                            G1 G2/M phases cell cycle <br />                                                                            Work laboratories led conclusion <br />                                                                            intact RAD9 gene activation Rad53p are <br />                                                                            required DNA damage checkpoint. The defect in <br />                                                                            DNA damage-induced Rad53p activation Rad9p <br />                                                                            phosphorylation seen Dlcd1 [pRNR3] cells suggested <br />                                                                            LCD1 play role checkpoint control. To test <br />                                                                            employed assay developed Garvik et al. <br />                                                                            (1995) uses temperature-sensitive allele CDC13. <br />                                                                            Cdc13p binds telomeres, defects gene lead to <br />                                                                            generation single-stranded telomeric DNA that <br />                                                                            causes prolonged metaphase arrest (Garvik et al., 1995; <br />                                                                            Lydall Weinert, 1995; Gardner et al., 1999). In this <br />                                                                            assay, cells synchronized G1 a-factor released at <br />                                                                            temperature restrictive cdc13-1 allele and, <br /> Fig. 4. LCD1 required phosphorylation Rad9p response <br /> to DNA damage association Rad53p-FHA2 with <br />                                                                            following S phase, DNA damage generated. <br /> phosphorylated Rad9p. (A) Cells (all carrying pRNR3 plasmid)           Checkpoint-pro&#194;&#174;cient cells arrest metaphase large <br /> were grown mid-log phase incubated presence absence       budded cells, single nucleus positioned the <br /> of 4-NQO (5 mg/ml) 2 h 30&#194;&#176;C, arresting cells G1         bud neck. Checkpoint-de&#194;&#174;cient cells, hand, <br /> addition a-factor 2 h. Cells lysed TCA, extracts        proceed past metaphase and, aid detection cells, <br /> were subjected western blot analysis anti-Rad9p polyclonal <br /> antibodies, electrophoresis 6.5% SDS&#194;&#177;polyacrylamide gels.         second temperature-sensitive <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>mutation</span>, cdc15-2, used <br /> (B) Cells treated described (A) native cell extracts   prevent entry cell cycle arresting cells <br /> prepared (see Materials methods). Association FHA2              late mitosis large budded cells partitioned <br /> domain Rad53p fused GST (upper panel) analysed              nucleus (Lydall Weinert, 1995). For cultures of <br /> described previously (Durocher et al., 1999). Alternatively, aliquots of <br /> native cell extracts treated phage l phosphatase (10 U) for <br />                                                                            cdc13-1 cdc15-2 cells containing lacking pRNR3, there <br /> 30 min 30&#194;&#176;C presence 2 mM MnCl2, dephosphorylate           total metaphase arrest restrictive <br /> Rad9p. After addition SDS&#194;&#177;PAGE sample buffer, samples              temperature, indicating ef&#194;&#174;cient checkpoint function <br /> subjected western blotting anti-Rad9p antibodies,            (Figure 5A). In contrast, cdc13-1 cdc15-2 cells lacking <br /> electrophoresis 6.5% SDS&#194;&#177;polyacrylamide gels (lower panel).             LCD1 fail mediate checkpoint arrest instead <br /> Molecular weight markers (kilodaltons) shown right-hand <br /> side panel.                                                        proceed cdc15 arrest point. Indeed, Dlcd cells <br />                                                                            degree checkpoint defect assay as <br />                                                                            cells lacking RAD9 gene (Figure 5A). Consistent with <br />                                                                            data, release a-factor arrest, incu- <br />    After DNA damage, Rad53p interacts phos-                       bation cells restrictive temperature to <br /> phorylated forms Rad9p, mediated                 result activation Rad53p wild-type cells, but <br /> the FHA domains Rad53p (Sun et al., 1998;                           Dlcd1 [pRNR3] cells Drad9 [pRNR3] cells <br /> Durocher et al., 1999). To interaction                 (Figure 5B). Importantly, introduction pLCD1 into <br /> dependent LCD1, took cultures wild-type, Dlcd1                    Dlcd1 strain restored cdc13-1-induced DNA <br /> or Dlcd1 cells complemented pLCD1, grew                       damage checkpoint (data shown) Rad53p acti- <br /> these presence absence 4-NQO 2 h,                     vation (Figure 5B). Thus, cells lacking LCD1 a <br /> then generated extracts them. The extracts                       major defect G2/M DNA damage checkpoint, and <br /> incubated glutathione&#194;&#177;agarose beads containing                        correlates inability activate Rad53p. Cells <br /> glutathione S-transferase (GST) GST fusion                   lacking LCD1 signi&#194;&#174;cantly sensitive the <br /> the C-terminal FHA domain (FHA2) Rad53p. Finally,                       microtubule poison, nocodazole, compared cells <br /> after washing beads, tested Rad9p association                   lacking spindle checkpoint gene BUB3 (data not <br /> by western immunoblot analysis. Consistent previous                   shown), unlikely LCD1 plays role <br /> &#194;&#174;ndings (Durocher et al., 1999), case wild-type                  cell cycle checkpoint monitors spindle integrity <br /> cells Rad9p binds ef&#194;&#174;ciently Rad53p-FHA2 following                      prevents mitosis kinetochore correct <br /> exposure DNA-damaging agent (Figure 4B, upper                       number microtubles attached. <br /> panel). Signi&#194;&#174;cantly, disruption LCD1 leads                           A role LCD1 G1 DNA damage checkpoint was <br /> dramatic decrease binding, defect comple-                   investigated. To this, cells arrested G1 with <br /> mented pLCD1 (Figure 4B, upper panel; shown                       a-factor culture split two; half UV <br /> the lower panel, treatment extracts l protein                      irradiated half left untreated. After release <br /> phosphatase electrophoresis led dephosphoryl-                    arrest, effect UV irradiation ability of <br />  <br /> 5806 <br />                                                                                                         LCD1: novel checkpoint gene <br />  <br />  <br />                                                                           exhibit delay budding release G1 <br />                                                                           (Figure 5C), suggesting cells completely <br />                                                                           defective G1 DNA damage checkpoint. Taken <br />                                                                           previous series experiments, this <br />                                                                           indicates LCD1 plays absolutely crucial role in <br />                                                                           DNA damage checkpoint response various stages the <br />                                                                           cell cycle. <br />  <br />                                                                           Lcd1p interacts Mec1p <br />                                                                           The phenotypic similarities cells defective LCD1 or <br />                                                                           MEC1 suggested products genes might <br />                                                                           interact. To case, generated an <br />                                                                           LCD1 expression plasmid LCD1 the <br />                                                                           control promoter yields Lcd1p product <br />                                                                           contains HA epitope tag N-terminus. <br />                                                                           Importantly, plasmid (pHA-LCD1) to <br />                                                                           fully complement DNA damage hypersensitivity <br />                                                                           phenotype Dlcd1 [pRNR3] cells complement <br />                                                                           <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> chromosomal LCD1 disruption <br />                                                                           (i.e. pRNR3 lost Dlcd1 cells containing <br />                                                                           pHA-LCD1 loss viability; data shown). <br />                                                                           Western blot analysis revealed band ~90 kDa that <br />                                                                           cross-reacts anti-HA antibodies extracts cells <br />                                                                           transformed pHA-LCD1 (Figure 6A, lane 2), not <br />                                                                           extracts cells transformed pLCD1 (Figure 6A, <br />                                                                           lane 1). Next, introduced pHA-LCD1 pLCD1 into <br />                                                                           cells genomic copy MEC1 an <br />                                                                           N-terminal FLAG tag. The presence epitope tag <br />                                                                           did disrupt known functions MEC1 <br />                                                                           gene (D.Durocher, unpublished data). Extracts these <br />                                                                           cells (Figure 6A, lane 4), cells untagged <br />                                                                           MEC1 (Figure 6A, lane 3), showed band ~260 kDa <br />                                                                           cross-reacts anti-FLAG antibodies western <br />                                                                           blot analysis. <br />                                                                              To Lcd1p interact Mec1p, we <br />                                                                           generated extracts strains, immunoprecipitated <br />                                                                           HA-Lcd1p FLAG-Mec1p tested precipitates for <br />                                                                           presence protein western immunoblot <br />                                                                           analysis. As shown Figure 6B, FLAG-tagged Mec1p <br /> Fig. 5. LCD1 required G1 G2/M DNA damage                   present anti-HA immunoprecipitates, using extracts <br /> checkpoints. Cells harbouring cdc13-1 cdc15-2 temperature-            cells expressing HA-Lcd1p, cells <br /> sensitive alleles grown early log phase 23&#194;&#176;C, arrested G1   expressing untagged Lcd1p. In reciprocal experiment, <br /> by addition a-factor (5 mg/ml) 120 min, released YPD      HA-Lcd1p observed anti-FLAG immunoprecipi- <br /> at 37&#194;&#176;C. (A) Samples taken 60 min, &#194;&#174;xed stained with <br /> DAPI. The percentage budded cells nucleus bud         tates extracts cells expressing FLAG-tagged <br /> (post-M phase cells) counted population ~200 cells         genomic Mec1p, cells expressing untagged <br /> &#194;&#175;uorescence microscopy. (B) After 180 min, cells lysed TCA        Mec1p used (Figure 6C). No anti-HA anti-FLAG <br /> and subjected situ analysis Rad53p kinase activity (upper        cross-reacting bands detected immunoprecipi- <br /> panel). Rad53p autophosphorylation quantitated using a <br /> phosphoimager (lower panel). (C) Cells arrested G1 by <br />                                                                           tations performed extracts parental strain <br /> addition a-factor 120 min, washed released YPD         expressing Lcd1p Mec1p untagged forms (data <br /> 30&#194;&#176;C. Samples taken 15 min intervals proportion        shown). In Figure 6B C, Mec1p appears a <br /> cells buds counted population ~150 cells.                doublet ~260 kDa western blotting. It unlikely <br />                                                                           forms Mec1p phosphorylated variants, as <br />                                                                           treatment cell extracts protein phosphatases before <br /> these cells re-enter cell cycle examined. As                   immunoprecipitation effect doublet (data <br /> shown Figure 5C, treatment wild-type cells UV                  shown). Notably, little detectable difference <br /> causes marked delay cell cycle re-entry, judged                levels Mec1p wild-type cells compared with <br /> the appearance buds. The time taken 50% wild-                   cells lacking Lcd1p, suggesting interaction of <br /> type cells start budding release G1 increased               Lcd1p Mec1p required stabilize Mec1p <br /> from ~40 min UV irradiation ~110 min                           protein. In addition, interaction Lcd1p Mec1p <br /> following UV irradiation. In contrast, cells disrupted                does appear affected treatment cells with <br /> for RAD9, delay pronounced, ~50%                    MMS (Figure 6B C) 4-NQO (data shown) is <br /> cells having budded ~70 min. Strikingly,                       affected washing immunoprecipitates 1 M <br /> exposed UV light, cells lacking LCD1 completely did                    sodium chloride (data shown). These results indicate <br />  <br />                                                                                                                                5807 <br />  J.Rouse S.P.Jackson <br />  <br />  <br />                                                                            ~25% overall identity stretch 180 amino acid <br />                                                                            residues (Figure 1A B). Although region Rfc5p <br />                                                                            similar Lcd1p characterized, this <br />                                                                            domain likely functionally important, of <br />                                                                            residues conserved Rfc5 family mem- <br />                                                                            bers different species. The RFC subunits form a <br />                                                                            multiprotein complex vivo region(s) each <br />                                                                            subunit required interactions been <br />                                                                            de&#194;&#174;ned. It interesting examine Lcd1p <br />                                                                            associate RFC RFC complex that <br />                                                                            contains Rad24p (Shimomura et al., 1998; Green et al., <br />                                                                            2000) and, so, functional effects this <br />                                                                            be. <br />                                                                               We LCD1 similar protein in <br />                                                                            S.cerevisiae genome S.pombe Rad26, the <br />                                                                            homology low extends short stretches of <br />                                                                            amino acid sequence (Figure 1A C). To date, <br />                                                                            S.cerevisiae orthologues S.pombe checkpoint <br />                                                                            Rad genes identi&#194;&#174;ed, notable exception <br />                                                                            Rad26. Since Rad26 crucial <br />                                                                            components Rad3 pathway (Al-Khodairy et al., <br />                                                                            1994; Lindsay et al., 1998; Edwards et al., 1999), seems <br />                                                                            likely organisms possess orthologue of <br />                                                                            protein. To date, however, Aspergillus nidulans <br />                                                                            uvsD gene Rad26 homologue been <br />                                                                            unambiguously identi&#194;&#174;ed (De Souza et al., 1999), with <br />                                                                            sequence homology extending ORF. <br />                                                                            Notably, phenotype cells lacking LCD1 some <br />                                                                            ways similar S.pombe rad26.d mutants, raising <br />                                                                            possibility LCD1 Rad26 functionally <br />                                                                            equivalent despite fact possess weak <br />                                                                            homology another. Whatever case, clear <br />                                                                            that, like S.pombe Rad26, S.cerevisiae LCD1 plays a <br />                                                                            crucial role DNA damage signalling responses <br />                                                                            (Figure 7). Consistent this, manuscript <br /> Fig. 6. Lcd1p interacts Mec1p. (A) Extracts (150 mg) Dlcd1       revision, YDR499W identi&#194;&#174;ed Paciotti <br /> cells transformed pLCD1 (lane 1) pHA-LCD1 (lane 2)          et al. (2000; referred DDC2) key <br /> were electrophoresed 7.5% SDS&#194;&#177;polyacrylamide gel subjected        regulator cell cycle checkpoint responses. <br /> to western blot analysis anti-HA antibodies (left panel). Extracts <br /> (150 mg) cells endogenous copy MEC1 gene <br /> did (lane 4) did (lane 3) bear N-terminal FLAG&#194;&#177;histidine tag <br /> were electrophoresed 6.5% SDS&#194;&#177;polyacrylamide gel subjected        The essential function LCD1 <br /> to western blot analysis anti-FLAG antibodies (right panel). (B   As case MEC1 RAD53, disruption of <br /> C) Cells containing pHA-LCD1, endogenous copy          LCD1 causes lethality suppressed elevation <br /> MEC1 gene bore N-terminal FLAG&#194;&#177;histidine tag, incubated in <br /> the presence absence MMS (0.02%) 2 h 30&#194;&#176;C. Native             dNTP levels. The molecular basis suppression <br /> extracts prepared, aliquots (2 mg) incubated anti-      present unclear, possible <br /> HA antibodies (B) anti-FLAG antibodies (C) conjugated protein        explanations. It shown &#194;&#174;ring late <br /> G&#194;&#177;Sepharose. After shaking 2 h 4&#194;&#176;C, beads washed       replication origins advanced cells lacking RAD53 <br /> subjected western blot analysis indicated antibody. As a <br /> control, extracts cells FLAG-tagged MEC1 gene bearing a <br />                                                                            MEC1 (Santocanale Dif&#194;&#175;ey, 1998; Shirahige <br /> pLCD1 cells untagged genomic copy MEC1, containing           et al., 1998). Activation replication complexes and <br /> pHA-LCD1 parental cells (data shown), subjected         nucleotide supply co-ordinated prevent <br /> same analysis.                                                             nucleotide depletion, LCD1, MEC1 <br />                                                                            RAD53 function pathway ensures this <br />                                                                            occurs. Consistent idea, temperature-sensitive <br /> that Lcd1p forms tight apparently constitutive                       mutations DBF4, slow origin &#194;&#174;ring, suppress <br /> complex Mec1p vivo.                                                lethality cells lacking MEC1 (Desany et al., 1998). <br />                                                                            Another explanation essential roles LCD1, <br />                                                                            MEC1 RAD53, mutually exclusive, that <br /> Discussion                                                                 genes act pathway up-regulates dNTP <br /> Lcd1p homology S.cerevisiae Rfc5p                               biosynthesis normal S phase allow ef&#194;&#174;cient DNA <br /> S.pombe Rad26                                                              replication&#195;?perhaps disrupting modulating the <br /> In searches protein sequence databases, LCD1                    Sml1p&#194;&#177;Rnr1p interaction (Zhao et al., 1998; Chabes <br /> consistently emerged showing homology Rfc5p,                    et al., 1999). <br />  <br /> 5808 <br />                                                                                                          LCD1: novel checkpoint gene <br />  <br />  <br />                                                                            cells lacking MEC1. The ability Rad53p-FHA2 to <br />                                                                            bind Rad9p extracts asynchronous cultures of <br />                                                                            Dlcd1 cells severely decreased compared wild-type <br />                                                                            cells. We DNA damage-induced phosphoryl- <br />                                                                            ation Chk1p requires intact LCD1 gene, which <br />                                                                            probably re&#194;&#175;ection defect Rad9p <br />                                                                            phosphorylation seen cells, Chk1p phosphoryl- <br />                                                                            ation dependent RAD9 (Sanchez et al., 1999). Taken <br />                                                                            together, results LCD1 required for <br />                                                                            changing activity key signal transducers that <br />                                                                            operate MEC1-dependent DNA damage signalling <br />                                                                            pathway (Figure 7). <br />                                                                               Cells lacking LCD1 display high degree of <br />                                                                            sensitivity low concentrations HU, indicating a <br />                                                                            role Lcd1p responding stalled DNA replication <br />                                                                            forks. Also, activation Rad53p DNA damage or <br />                                                                            replication blocks severely reduced cells lacking <br />                                                                            LCD1. At present, molecular mechanism(s) whereby <br />                                                                            Mec1p pathway `senses&apos; stalled replication forks <br />                                                                            remains unclear. Proteins DNA polymerase e <br />                                                                            (Pol2p) RFC subunits, roles DNA <br />                                                                            synthesis, function checkpoint responses (Navas <br />                                                                            et al., 1995; Sugimoto et al., 1996; Shimomura et al., <br /> Fig. 7. The Lcd1p/Mec1p signalling pathway. Lcd1p interacts           1998), Sgs1p DNA helicase needed for <br /> Mec1p controls activation key effector molecules            faithful DNA replication required activation of <br /> Rad53p, response DNA damage inhibition DNA replication.        Rad53p response stalled replication forks (Frei and <br /> An intact RAD9 gene required activation Rad53p          Gasser, 2000). It clearly whether <br /> Chk1p response DNA damage, RAD17/RAD24 epistasis <br /> group required DNA damage-induced activation Rad53p. <br />                                                                            Lcd1p interacts regulates functions of <br /> Black boxes represent discrete complexes involved DNA damage        DNA replication-associated proteins. <br /> response. HU-induced activation Rad53p involves different group <br /> of genes, e.g. POL2 (right-hand side). Chk1p mediates arrest mitosis,   LCD1 key regulator cell cycle checkpoints <br /> but involved DNA repair, Rad53p effector         The hypersensitivity Dlcd1 cells DNA damage and <br /> molecules involved lesion repair checkpoint regulation. <br />                                                                            DNA replication blocks raises possibility Lcd1p <br />                                                                            plays role slowing cell cycle progression these <br />                                                                            circumstances prevent cell cycle catastrophe. Indeed, <br /> The role Lcd1p cellular responses DNA <br />                                                                            DNA damage induced using temperature- <br /> damage blocked DNA replication <br />                                                                            sensitive cdc13 allele, Dlcd1 cells fail arrest in <br /> Cells lacking LCD1 extremely sensitive UV, IR and <br />                                                                            metaphase proceed cell cycle the <br /> MMS, suggesting major defect responding types                  kinetics cells lacking RAD9. This failure Dlcd1 <br /> of DNA lesions generated agents.                         cells arrest metaphase response cdc13-1-induced <br /> Strikingly, Dlcd1 cells signi&#194;&#174;cantly sensitive                 DNA damage correlates lack Rad53p activation. <br /> MMS UV Dmec1 cells. Furthermore, absence                       Notably, G1 G2/M defects <br /> of intact MEC1 gene does hypersensitize                     severe exhibited cells lacking <br /> Dlcd1 cells DNA damage (Figure 2A&#194;&#177;C), suggesting                        Rad9p, suggesting LCD1 acts upstream Rad9p and/ <br /> that terms MEC1 function genes epistatic.                  functions checkpoint pathways addition to <br /> Although possibilities exist and, discussed below,                involve Rad9p (Figure 7). <br /> this raises possibility addition role                   It shown recently disruption the <br /> regulating Mec1p pathway, Lcd1p functions                     S.cerevisiae CHK1 gene results defective G2/M DNA <br /> ways potentiate cellular survival presence                    damage checkpoint, Dchk1 cells hypersensitive to <br /> DNA damage. Regulation signalling DNA                         DNA-damaging agents (Sanchez et al., 1999; J.Rouse and <br /> repair pathways MEC1 pathway, LCD1,                      S.P.Jackson, unpublished data). Also, arresting Drad9 <br /> is currently investigation laboratory.                       cells metaphase shown little no <br />    Previous work established FHA domains                   effect viability exposure IR (Aboussekhra <br /> Rad53p bind speci&#194;&#174;cally phosphorylated forms                     et al., 1996). These observations imply sensitivity <br /> Rad9p arise DNA damage,                        cells lacking LCD1, example, primarily to <br /> required Rad53p activation (Sun et al., 1998; Durocher                 checkpoint defect, instead result an <br /> et al., 1999). In line this, ability FHA2                   inability mediate ef&#194;&#174;cient repair DNA lesions. <br /> interact Rad9p extracts cells exposed DNA                  Results recent studies consistent this <br /> damage severely reduced cells lacking LCD1,                 notion. For example, demonstrated genes <br /> correlates lack activation Rad53p Chk1p                     RAD24 epistasis group required ef&#194;&#174;cient non- <br /> induced DNA damage. DNA damage-induced Rad9p                            homologous end-joining DSBs, arti&#194;&#174;cially impos- <br /> phosphorylation G1-arrested cells reduced                  ing cell cycle delays partially rescues defect <br /> LCD1 absent reduction great             (de la Torre-Ruiz Lowndes, 2000). Also, Sir and <br />  <br />                                                                                                                                  5809 <br />  J.Rouse S.P.Jackson <br />  <br />  <br /> Ku DSB repair proteins re-localize discrete nuclear                    start ATG &#194;&#174;rst 1040 bp ORF, ampli&#194;&#174;ed cloned <br />                                                                             pGEM-T. The resulting vector, pGEM499, digested XhoI <br /> foci sites DNA damage MEC1-dependent manner <br />                                                                             HindIII excise 350 bp upstream LCD1 &#194;&#174;rst 688 bp of <br /> (Martin et al., 1999; McAinsh et al., 1999; Mills et al.,                   coding sequence. The LEU2 gene 1.2 kb upstream start ATG <br /> 1999), proteins checkpoint                      ampli&#194;&#174;ed cloned pCR-Script. An XhoI&#194;&#177;HindIII fragment <br /> defects. Finally, light, Rad55p, protein involved              containing LEU2 ORF 1 kb 5&#194;&#162; non-coding sequence was <br /> recombinational repair (Kanaar et al., 1998)                     inserted pGEM499 create pGEM499::LEU2. The NcoI&#194;&#177;NotI <br />                                                                             fragment (5 mg) vector transformed cells containing <br /> checkpoint regulation, phosphoryl-                      pRNR3. Disruption LCD1 checked genomic PCR and <br /> ated Mec1p pathway-dependent manner (Bashkirov                         con&#194;&#174;rmed Southern blot analysis. <br /> et al., 2000). In light points, currently                      To generate Dlcd1Dsml1 Dlcd1Dmec1Dsml1 cells, Dlcd1 <br /> investigating potential role Lcd1p facilitating               [pRNR3] strain crossed Dmec1Dsml1 cells. Diploids were <br /> repair various forms DNA damage.                                      sporulated and, tetrad dissection, strains isolated by <br />                                                                             following segregation sensitivity HU relevant markers. Loss <br />                                                                             RNR3 plasmid encouraged growth presence of <br /> Physical genetic interactions Lcd1p                             tryptophan verifed loss ability grow absence this <br /> and Mec1p                                                                   amino acid. <br /> The phenotypic similarities cells lacking LCD1 or <br />                                                                             In situ kinase assay Rad53p <br /> MEC1 suggested products genes                       Brie&#194;&#175;y, cells grown early log phase, point relevant <br /> interact. Indeed, &#194;&#174;nd Lcd1p Mec1p                        genotoxic agent added appropriate concentration the <br /> ef&#194;&#174;ciently immunoprecipitated another,                        indicated time. Cells collected centrifugation resuspended in <br /> under stringent conditions. Furthermore, exposure cells                  4 vols trichloroacetic acid (TCA) (20%), addition 1 vol. of <br /> to DNA damage detectable effect extent                     glass beads, cells lysed using mini bead beater. The resulting lysate <br />                                                                             centrifuged, supernatant discarded 2 vols SDS&#194;&#177;PAGE <br /> stability Lcd1p&#194;&#177;Mec1p interaction. Taken                    sample buffer, adjusted 0.2 M Tris&#194;&#177;HCl pH 8.8, added the <br /> with functional data, observation Mec1p                        pellet mixed thoroughly. After boiling, samples centrifuged and <br /> interacts Lcd1p indicates actions                supernatants stored &#194;&#177;20&#194;&#176;C required. Samples run 10% <br /> proteins cell likely closely interlinked. It               SDS&#194;&#177;polyacrylamide gels transferred Immobilon-P absence <br />                                                                             methanol, resulting &#194;&#174;lters subjected situ analysis of <br /> will great determine                          Rad53p kinase activity described previously (Pellicioli et al., 1999). <br /> interaction direct investigate is <br /> required checkpoint and/or essential functions                  Analysis G1 DNA damage checkpoint <br /> of proteins. It interesting determine                 Cells grown early log phase arrested G1 addition of <br /> what proportion total cellular Lcd1p exists complex                   a-factor (5 mg/ml) 60&#194;&#177;90 min, point arrest veri&#194;&#174;ed the <br />                                                                             absence budded cells (&lt;1%). For UV irradiation, cells collected <br /> with Mec1p. The observation cells lacking LCD1                     centrifugation, spread YPD plates irradiated germicidal <br /> more sensitive MMS UV Dmec1 cells suggests                       lamp 254 nm indicated dose, rate 3 J/m2/s. Cells were <br /> that Lcd1p function, partially, independently                  scraped YPD released G1 arrest extensive washing <br /> of MEC1, regulating pathways.                              &#194;&#174;lter apparatus YPD, incubated 30&#194;&#176;C. Samples were <br />                                                                             taken 15 min intervals proportion budded cells counted <br />                                                                             light microscopy. <br />  <br /> Materials methods                                                       Analysis G2/M DNA damage checkpoint <br />                                                                             Cell cycle arrest G2/M phase, response cdc13-induced DNA <br /> Plasmids antibodies                                                     damage, assayed previously described (Garvik et al., 1995). <br /> pBAD79, TRP1 plasmid bearing RNR3 gene control       Brie&#194;&#175;y, cells grown early log phase YPD 23&#194;&#176;C (permissive <br /> constitutive glyceraldehyde-3-phosphate dehydrogenase (GAP) pro-            temperature cdc13-1 cdc15-2), point cells were <br /> moter, pRS415-HA-CHK1 kind gifts Dr Steve Elledge             synchronized G1 addition a-factor (5 mg/ml) ~120 min. <br /> and referred pRNR3 pHA-CHK1, respectively. pLCD1          When budded cells accounted &gt;1% population, cells were <br /> constructed gap repair follows. The YDR499W ORF 500 bp            released arrest washing &#194;&#174;ve times YPD incubation in <br /> upstream start ATG ampli&#194;&#174;ed (primers 499C 499D)        pre-warmed YPD restrictive temperature (36&#194;&#176;C). At times <br /> 45 bp homology bases 1939&#194;&#177;1984 2101&#194;&#177;2146               indicated, aliquots cells removed, &#194;&#174;xed 70% ethanol and <br /> multiple cloning site centromeric plasmid pRS416 (which contains     stained 4&#194;&#162;,6-diamidino-2-phenylindole (DAPI). The percentage of <br /> the URA3 marker), respectively. Cells transformed BamHI&#194;&#177;          budded cells nucleus bud (post-M phase cells) was <br /> HindIII-digested pRS416 499C/499D PCR product (5 mg),           counted population ~200 cells &#194;&#175;uorescence microscopy. <br /> transformants selected growth medium lacking uracil. <br /> Plasmids isolated positive colonies transformed              Preparation native cell extracts <br /> Escherichia coli, presence YDR499W gene veri&#194;&#174;ed           To prepare native extracts, cells grown early log phase and <br /> restriction digestion sequencing. Polyclonal anti-Rad53p anti-      incubated presence absence DNA-damaging agents 2 h at <br /> Rad9p antibodies kind gifts Dr Noel Lowndes. pHA-LCD1             30&#194;&#176;C. After centrifugation, cells washed twice water in <br /> was constructed mutating start codon YDR499W pLCD1       1 vol. 23 extraction buffer (100 mM HEPES pH 7.4, 400 mM sodium <br /> NcoI restriction site using QuikChange site-directed mutagenesis     acetate, 100 mM magnesium acetate, 1 mM EDTA, 20% glycerol, 1 mM <br /> kit (Stratagene). Oligonucleotides encoding tag ligated   sodium orthovanadate, 1 mM microcystin-LR, 0.1% 2-mercaptoethanol <br /> NcoI site vector. Mouse monoclonal anti-FLAG antibodies (clone       cocktail protease inhibitors). After packing cells syringe <br /> M2) Sigma, anti-HA antibodies (12CA5)                   sealed tip, hole pierced tip, cells <br /> Boehringer rabbit polyclonal anti-histidine tag antibodies         extruded stream liquid nitrogen. The frozen material <br /> from Santa Cruz Biotechnology.                                              placed ground dry ice ground coffee grinder for <br />                                                                             ~3 min. This mixture placed ice, dry ice had <br /> Yeast strains                                                               sublimed, lysed cells resuspended 1 vol. cold 23 extraction <br /> Standard genetic techniques used manipulating yeast strains        buffer, ice. The lysate centrifuged 13 500 r.p.m. 4&#194;&#176;C the <br /> (Adams et al., 1997), media growth yeast strains         supernatant aliquoted stored &#194;&#177;80&#194;&#176;C use. <br /> described previously (Rose et al., 1990). Dmec1Dsml1 cells kind <br /> gift Dr Rodney Rothstein. cdc13-1 cdc15-2 cells cells bearing      Western blotting immunoprecipitation <br /> an N-terminally FLAG-His6-tagged genomic copy MEC1 gene              Rad53p Rad9p western immunoblots performed described <br /> were kind gift Dr Daniel Durocher laboratory. To generate    previously (Vialard et al., 1998). In co-immunoprecipitation experiments, <br /> cells lacking LCD1, fragment gene, spanning 700 bp upstream    aliquots native extract (2 mg protein) cells expressing either <br />  <br />  <br /> 5810 <br />                                                                                                                 LCD1: novel checkpoint gene <br />  <br /> HA-Lcd1p FLAG-Mec1p, both, incubated protein G&#194;&#177;              Rad53p DNA replication checkpoint colocalizes with <br /> Sepharose beads conjugated mouse monoclonal anti-FLAG antibodies          Rad53p S-phase-speci&#194;&#174;c foci. Genes Dev., 14, 81&#194;&#177;96. <br /> (1 mg) anti-HA monoclonal antibodies (1 mg) shaking platform    Friedberg,E.C., Walker,G.C. Siede,W. (1995) DNA Repair and <br /> 4&#194;&#176;C 2 h. After centrifugation, beads washed times      Mutagenesis. ASM Press, Washington, DC. <br /> WB buffer (50 mM HEPES pH 7.4, 0.2% NP-40 protease inhibitors          Gardner,R., Putnam,C.W. Weinert,T. (1999) RAD53, DUN1 and <br /> 0.1% 2-mercaptoethanol) containing 0.75 M sodium chloride twice       PDS1 de&#194;&#174;ne parallel G2/M checkpoint pathways budding yeast. <br /> WB buffer salt. The beads boiled 5 min SDS sample        EMBO J., 18, 3173&#194;&#177;3185. <br /> buffer electrophoresed 6.5% polyacrylamide gels, subjected   Garvik,B., Carson,M. Hartwell,L. (1995) Single-stranded DNA <br /> western blot analysis appropriate antibody.                         arising telomeres CDC13 mutants constitute speci&#194;&#174;c signal <br />                                                                              RAD9 checkpoint. Mol. Cell. Biol., 15, 6128&#194;&#177;6138. <br />                                                                            Green,C.M., Erdjument-Bromage,H., Tempst,P. Lowndes,N.F. <br /> Acknowledgements                                                             (2000) A novel Rad24 checkpoint protein complex closely related to <br />                                                                              replication factor C. Curr. Biol., 10, 39&#194;&#177;42. <br /> We thank members Jackson laboratory, especially Daniel          Greenwell,P.W., Kronmal,S.L., Porter,S.E., Gassenhuber,J., Obermaier,B. <br /> Durocher, Brandi Williams Andrew McAinsh, helpful discus-            Petes,T.D. (1995) TEL1, gene involved controlling telomere <br /> sions. We grateful Steve Elledge, Ted Weinert Michael             <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> Saccharomyces cerevisiae, homologous human ataxia <br /> Resnick providing yeast strains plasmids, Noel Lowndes        telangiectasia gene. Cell, 82, 823&#194;&#177;829. <br /> providing antibodies, Daniel Durocher providing reagents        Grif&#194;&#174;ths,D.J.F., Barbet,N.C., McCready,S., Lehmann,A.R. Carr, <br /> before publication. This work funded Leukaemia Research           A.M. (1995) Fission yeast Rad17: homolog budding yeast Rad24 <br /> Fund Association International Cancer Research.                  shares regions sequence similarity DNA polymerase <br />                                                                              accessory proteins. EMBO J., 14, 5812&#194;&#177;5823. <br />                                                                            Kanaar,R., Hoeijmakers,J.H.J. van Gent,D.C. (1998) Molecular <br /> References                                                                   mechanisms DNA double strand break repair. Trends Cell Biol., 8, <br />                                                                              483&#194;&#177;489. <br /> Aboussekhra,A., Vialard,J.E., Morrison,D.E., la Torre-Ruiz,M.A.,        Kato,R. Ogawa,H. (1994) An essential gene, ESR1, required for <br />   Cernakova,L., Fabre,F. Lowndes,N.F. (1996) A novel role        mitotic cell growth, DNA repair meiotic recombination in <br />   budding yeast RAD9 checkpoint gene DNA damage-dependent                 Saccharomyces cerevisiae. Nucleic Acids Res., 22, 3104&#194;&#177;3112. <br />   transcription. EMBO J., 15, 3912&#194;&#177;3922. <br />                                                                            Lindsay,H.D., Grif&#194;&#174;ths,D.J.F., Edwards,R.J., Christensen,P.U., Murray, <br /> Adams,A., Gottschling,D.E., Kaiser,C.A. Stearns,T. (1997) Methods        J.M., Osman,F., Walworth,N. Carr,A.M. (1998) S phase-speci&#194;&#174;c <br />   Yeast Genetics. Cold Spring Harbor Laboratory Press, Cold Spring        activation Cds1 kinase de&#194;&#174;nes subpathway checkpoint <br />   Harbor, NY.                                                                response Schizosaccharomyces pombe. Genes Dev., 12, 382&#194;&#177;395. <br /> Al-Khodairy,F., Fotou,E., Sheldrick,K.S., Grif&#194;&#174;ths,D.J.F., Lehmann,        Longhese,M.P., Paciotti,V., Fraschini,R., Zaccarini,R., Plevani,P. and <br />   A.R. Carr,A.M. (1994) Identi&#194;&#174;cation characterization            Lucchini,G. (1997) The novel DNA damage checkpoint protein Ddc1p <br />   new elements involved checkpoint feedback controls &#194;&#174;ssion        phosphorylated periodically cell cycle response to <br />   yeast. Mol. Biol. Cell, 5, 147&#194;&#177;160.                                        DNA damage budding yeast. EMBO J., 16, 5216&#194;&#177;5226. <br /> Allen,J.B., Zhou,Z., Siede,W., Friedberg,E.C. Elledge,S.J. (1994)      Lowndes,N.F. Murguia,J.R. (2000) Sensing responding DNA <br />   The Sad1/Rad53 protein kinase controls multiple checkpoints            damage. Curr. Opin. Genet. Dev., 10, 17&#194;&#177;25. <br />   DNA damage-induced transcription yeast. Genes Dev., 8,                Lydall,D. Weinert,T. (1995) Yeast checkpoint genes DNA <br />   2401&#194;&#177;2415.                                                                 damage processing: implications repair arrest. Science, 270, <br /> Bashkirov,V.I., King,J.S., Bashkirova,E.V., Schmuckli-Maurer,J.          1488&#194;&#177;1491. <br />   Heyer,W.-D. (2000) DNA repair protein Rad55 terminal substrate      Martin,S.G., Laroche,T., Suka,N., Grunstein,M. Gasser,S.M. (1999) <br />   DNA damage checkpoints. Mol. Cell. Biol., 20, 4393&#194;&#177;4404.                Relocalization telomeric Ku SIR proteins response DNA <br /> Chabes,A., Domkin,V. Thelander,L. (1999) Yeast Sml1, protein           strand breaks yeast. Cell, 97, 621&#194;&#177;633. <br />   inhibitor ribonucleotide reductase. J. Biol. Chem., 274,              Martinho,R.G., Lindsay,H.D., Flaggs,G., DeMaggio,A.J., Hoekstra,M.F., <br />   36679&#194;&#177;36683.                                                               Carr,A.M. Bentley,N.J. (1998) Analysis Rad3 Chk1 protein <br /> Cohen-Fix,O. Koshland,D. (1997) The anaphase inhibitor                kinases de&#194;&#174;nes different checkpoint responses. EMBO J., 17, <br />   Saccharomyces cerevisiae, Pds1p, target DNA damage             7239&#194;&#177;7249. <br />   checkpoint pathway. Proc. Natl Acad. Sci. USA, 94, 14361&#194;&#177;14366.          McAinsh,A.D., Scott-Drew,S., Murray,J. Jackson,S.P. (1999) DNA <br /> Cullmann,G., Fien,K., Kobayashi,R. Stillman,B. (1995)                    damage triggers disruption telomeric silencing Mec1p- <br />   Characterisation &#194;&#174;ve replication factor-C genes                  dependent relocation Sir3p. Curr. Biol., 9, 963&#194;&#177;966. <br />   Saccharomyces cerevisiae. Mol. Cell. Biol., 15, 4661&#194;&#177;4671.               Mills,K.D., Sinclair,D.A Guarente,L. (1999) MEC1-dependent <br /> de la Torre-Ruiz,M.A. Lowndes,N.F. (2000) The Saccharomyces              redistribution Sir3 silencing protein telomeres double <br />   cerevisiae DNA damage checkpoint required ef&#194;&#174;cient repair        strand breaks. Cell, 97, 609&#194;&#177;620. <br />   double strand breaks non-homologous end joining. FEBS Lett.,          Morrow,D.M., Morrow,M., Tagle,D.A., Shiloh,Y., Collins,F.S. and <br />   467, 311&#194;&#177;315.                                                              Hieter,P. (1995) TEL1, Saccharomyces cerevisiae homolog the <br /> de la Torre-R <br /> </body></html>