<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>    Colon <br />  <br />                                     ORIGINAL ARTICLE <br />  <br />                                     Reduced LIMK2 expression colorectal cancer <br />                                     re&#239;&#172;&#8218;ects role limiting stem cell proliferation <br />                                     Filipe C Louren&#195;&#167;o,1 June Munro,1 Jennifer Brown,1 Julia Cordero,1 Rhoda Stefanatos,1 <br />                                     Karen Strathdee,1 Clare Orange,2 Stephan M Feller,3 Owen J Sansom,1 Marcos Vidal,1 <br />                                     Graeme I Murray,4 Michael F Olson1 <br />  <br /> &#226;&#8211;&#184; Additional material            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> <br /> published online only. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> view      Objective Colorectal cancer (CRC) major                      Signi&#239;&#172;?cance study <br /> please visit journal online <br /> (http://dx.doi.org/10.1136/         contributor cancer mortality morbidity. LIM kinase <br /> gutjnl-2012-303883).                2 (LIMK2) promotes tumour cell invasion metastasis. <br /> 1                                   The objectives study determine               What known subject? <br />   Beatson Institute Cancer <br /> Research, Glasgow, UK               LIMK2 expression associated CRC progression              &#226;&#8211;&#184; LIM kinases overexpressed tumour <br /> 2 <br />   Department Pathology,          patient outcome, use genetically modi&#239;&#172;?ed                types. <br /> Division Cancer Sciences         Drosophila mice determine LIMK2 deletion              &#226;&#8211;&#184; They contribute regulation actomyosin <br /> and Molecular Pathology,            affects gastrointestinal stem cell regulation tumour <br /> Western In&#239;&#172;?rmary, <br />                                                                                                        cytoskeletal dynamics, play roles in <br /> Glasgow, UK <br />                                     development.                                                       cell motility, invasion metastasis. <br /> 3 <br />   Weatherall Institute           Design LIMK2 expression activity measured               &#226;&#8211;&#184; The possible involvement LIM kinases CRC <br /> Molecular Medicine, John            immunostaining tumours CRC-prone mice,                     unknown. <br /> Radcliffe Hospital, University   human CRC cell lines 650 human tumours. LIMK <br /> Oxford, Oxford, UK <br /> 4                                   knockdown Drosophila Limk2 deletion mice                What new &#239;&#172;?ndings? <br />   Department Pathology,                                                                           &#226;&#8211;&#184; LIMK2 expression substrate <br /> Division Applied Medicine,       allowed assessment contributions to <br /> School Medicine              gastrointestinal stem cell homeostasis tumour                  phosphorylation progressively decrease with <br /> Dentistry, University            development.                                                       tumour stage. <br /> Aberdeen, Aberdeen, UK              Results LIMK2 expression reduced intestinal               &#226;&#8211;&#184; Deletion dLIMK Drosophila midgut results <br />                                     tumours cancer-prone mice, human CRC              stem cell proliferation organ thickening. <br /> Correspondence to <br /> Professor Michael F Olson,          cell lines tumours. Reduced LIMK2 expression             &#226;&#8211;&#184; LIMK2 deletion increases mouse intestinal stem <br /> Beatson Institute Cancer        substrate phosphorylation associated shorter             cell proliferation vitro. <br /> Research, Garscube Estate,          patient <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>survival</span>. Genetic analysis Drosophila midgut          &#226;&#8211;&#184; In context mouse model of <br /> Switchback Road, Glasgow                                                                               colitis-associated CRC, Limk2 deletion increases <br />                                     intestinal epithelial cells isolated genetically <br /> G61 1BD, UK;                                                                                           dysplasia intestinal tumour size. <br /> m.olson@beatson.gla.ac.uk           modi&#239;&#172;?ed mice revealed conserved role LIMK2 in <br />                                     constraining gastrointestinal stem cell proliferation. Limk2     How impact clinical practice in <br /> Received 5 October 2012             deletion increased colon tumour size colitis- <br /> Revised 12 February 2013                                                                             foreseeable future? <br />                                     associated colorectal mouse cancer model.                        &#226;&#8211;&#184; This study reveals actomyosin cytoskeleton <br /> Accepted 24 March 2013 <br /> Published Online First              Conclusions This study revealed LIMK2 expression              regulators LIMK2 patterns of <br /> 12 April 2013                       activity progressively decrease advancing stage,          expression CRC distinct tumour <br />                                     supports hypothesis selective                types including breast prostate, meaning <br />                                     pressure reduced LIMK2 expression CRC relieve            drugs targeting cytoskeleton be <br />                                     negative constraints imposed gastrointestinal                 likely context-dependent effects. <br />                                     stem cells. <br />  <br />  <br />  <br />                                     INTRODUCTION                                                      The actomyosin cytoskeleton provides struc- <br />                                     Colorectal cancer (CRC) common               tural underpinning gives cells shape and <br />                                     form disease men second                 allows respond changes external <br />                                     common women worldwide, major                      physicomechanical conditions. Importantly, dynamic <br />                                     contributor cancer mortality morbidity.                 reorganisation cytoskeletal structures allows <br />                                     Although possibility exists CRC cells             tumour cells invade local tissue ultimately <br />                                     origin derived reprogrammed                     metastasise distant sites.6 As result, commonly <br />                                     epithelial cells transdifferentiated bone marrow-           held concept expression actomyosin <br />                                     derived cells,1 experimental evidence strongly sup-            regulatory proteins increase tumorigenesis <br />              Open Access            ports conclusion intestinal crypt stem cells          progression. However, given multiple roles <br />           Scan access more <br />              free content           predominant source intestinal               actomyosin cytoskeleton plays cell homeostasis, <br />                                     tumours emerge.2&#226;&#8364;&#8220;5 Therefore, mutations                     example, apoptosis human embryonic <br />                                     changes gene expression observed CRC                 stem cells,7 alternative possibility may <br />     To cite: Louren&#195;&#167;o FC,           promote tumour initiation growth                   selection reduced expression loss acto- <br />     Munro J, Brown J, et al. Gut    effects stem cell homeostasis.                        myosin regulators cancers. <br />     2014;63:480&#226;&#8364;&#8220;493. <br />  <br />  <br /> 480                                                                                        Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883 <br />                                                                                                                                     Colon <br />  <br />    The LIM kinase family consists members; LIM kinase              Technology); &#206;&#178;-catenin (BD Biosciences); GFP (Abcam); Olfm4 <br /> 1 (LIMK1) LIM kinase 2 (LIMK2), play central roles              (Abcam); Bmi1 (Cell Signaling Technology); Erk2 (gift <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> in RhoGTPase regulation actin cytoskeleton phosphor-            Chris Marshall, Institute Cancer Research); Stat1 (Cell <br /> ylating co&#239;&#172;?lin proteins Serine3 inactivating &#239;&#172;?la-            Signaling Technology). Alexa-Fluor680 (Molecular Probes) or <br /> mentous actin severing activity.8 LIMK1 LIMK2 50%                IRDye800 (Rockland)-conjugated secondary antibodies were <br /> overall identity 70% identity kinase domains,            detected infra-red imaging (Li-Cor Odyssey). Goat anti- <br /> are regulated activation loop phosphorylation                  mouse goat anti-rabbit horseradish peroxidase-conjugated <br /> RhoGTPase effector kinases including ROCK, PAK                        antibodies Pierce. <br /> MRCK.8 LIM kinases act central network hubs regula- <br /> tion cytoskeletal dynamics proposed                Histology immunohistochemistry <br /> attractive potential cancer drug targets.9 Consistent posi-        Histology immunohistochemistry performed as <br /> tive role cancer, expression LIMK1 reported             described.18 Antibodies working concentrations used for <br /> be elevated breast,10 11 malignant melanoma12 prostate13           immunohistochemistry immuno&#239;&#172;&#8218;uorescence fol- <br /> tumours. However, studies reported date             lowing: LIMK2, 1:200 (Santa Cruz Biotechnology, Inc); <br /> that examined expression LIM kinases CRC.                       Phospho-Co&#239;&#172;?lin, 1:100 (Cell Signaling Technology); GFP, 1:250 <br />    In study, LIMK2 expression reduced            (BD Biosciences); Texas-Red phalloidin, 1:250 (Molecular <br /> intestinal tumours genetically modi&#239;&#172;?ed mice,                  Probes, Invitrogen); &#206;&#178;-catenin, 1:1000 (BD Biosciences). <br /> human CRC cell lines tumours. Protein levels progres-            DAB-stained slides imaged using Hamamatsu <br /> sively lower advancing tumour grade, reduced LIMK2               Nanozoomer NDP slide scanner (Hamamatsu Photonics) and <br /> expression substrate protein phosphorylation asso-           Digital Slide Server (Slidepath) software. For immuno&#239;&#172;&#8218;uores- <br /> ciated shorter patient survival times. Increased LIMK2 pro-          cence images Nikon A1R confocal microscope used. For <br /> moter methylation observed tumours relative patient             Drosophila immuno&#239;&#172;&#8218;uorescence, tissues dissected in <br /> matched normal tissue. Using genetic methods, identi&#239;&#172;?ed              phosphate-buffered saline (PBS) &#239;&#172;?xed 30&#226;&#8364;&#8220;45 min 4% <br /> repressive role LIMK2 intestinal stem cell proliferation        para-formaldehyde. After &#239;&#172;?xation, samples washed three <br /> Drosophila melanogaster mice. Using mouse model               times PBS+0.1% Triton X-100 (PBST) incubated in <br /> colitis-associated CRC, determined LIM kinase 2 knock-            primary antibodies overnight 4&#194;&#176;C. Samples washed <br /> out (Limk2-KO) mice increased intestinal tumour size              subjected secondary antibody staining 2 h room <br /> dysplasia. These data support hypothesis select-        temperature followed washing mounting Vectashield <br /> ive pressure reduced LIMK2 expression CRC relieve               containing DAPI (Vector Laboratories, Inc). Primary secondary <br /> negative constraints imposed gastrointestinal stem cells.              antibodies incubated PBST+0.5% bovine serum albumin. <br />                                                                           The antibodies used anti-phospho-Histone 3 (1:100 dilution, <br /> MATERIALS AND METHODS                                                     Cell Signaling Technology), anti-GFP (1:2000 dilution from <br /> Cell culture                                                              Abcam) anti-Armadillo (1:3 dilution, developed by <br /> Mouse embryo &#239;&#172;?broblast cells isolated cultured                E. Wieschaus obtained Developmental Studies <br /> described D&#226;&#8364;&#8482;Abaco Olson.14 Intestinal epithelial cultures          Hybridoma Bank developed auspices National <br /> were isolated cultured described Sato et al.15                  Institute Child Health Human Development main- <br /> Colorectal tumour cell lines maintained, cell lysates            tained University Iowa, Department Biology, Iowa <br /> prepared, described Emaduddin et al.16                              City, IA, USA). Primary antibodies visualised Alexa 488, <br />                                                                           Alexa 532 594-conjugated secondary antibodies (Molecular <br /> Promoter methylation                                                      Probes, Invitrogen). <br /> For de-novo methyltransferase inhibitor treatments, SW48 cells <br /> were plated six-well plates 2&#195;&#8212;105 well. The following           Mice <br /> day, RG108 (100 mM &#239;&#172;?nal), 5-azacytidine (5-Aza) (10 mM &#239;&#172;?nal)              Lgr5-EGFP-ires-Cre:ER-T2 (Lgr5&gt;GFP) mice described previ- <br /> or 1% dimethyl sulfoxide vehicle (v/v) diluted                    ously.19 Limk2 gene-trap embryonic stem cells (sequence tag: <br /> Dulbecco&#226;&#8364;&#8482;s modi&#239;&#172;?ed eagle medium supplemented 10% fetal               RRE287; MMRRC stock # 001883-UCD) obtained from <br /> calf serum applied cells 48 h 37&#194;&#176;C. Genomic DNA             BayGenomics used blastocyst injections Beatson <br /> from cell lines tissues prepared using QIAamp DNA                Institute using standard methods. Experimental mice a <br /> Mini Kit (Qiagen). Bisulphite conversion genomic DNA               mixed C57BL/6J 129/ola strain background. Limk2-KO mice <br /> performed using EZ DNA Methylation-Gold Kit (Zymo                     viable born Mendelian ratios (observed genotype birth <br /> Research), followed PCR using custom-designed primers                  49/183=26.8%) macroscopic abnormalities. All proce- <br /> (Invitrogen). PCR products puri&#239;&#172;?ed using QIAquick                dures performed appropriate licences according to <br /> PCR Puri&#239;&#172;?cation Kit (Qiagen) cloned using 2.1-TOPO                UK Home Of&#239;&#172;?ce guidelines. <br /> vector, transformation TOP10 Escherichia coli <br /> (Invitrogen). Blue/white screening utilised select positive        Colitis-associated CRC (AOM+DSS) <br /> colonies DNA isolation sequencing. Sequencing analysis            On day 1, mice genotype (8&#226;&#8364;&#8220;12-week-old littermates) <br /> was using CLC Genomics V.5.0 software.                               weighed injected intraperitoneally 12.5 mg/kg of <br />                                                                           azoxymethane (AOM) solution (Sigma-Aldrich). On day 6 the <br /> Cell extraction immunoblotting                                        drinking supply &#239;&#172;?lled dextran sodium sulfate (DSS) solu- <br /> Whole cell lysates prepared western blotted                   tion (DSS salt, MW36 000&#226;&#8364;&#8220;56 000 Da, MP Biomedicals). On day <br /> described previously.17 Primary antibodies used routinely            8 drinking supply re&#239;&#172;?lled DSS solution day 10 <br /> used 1:1000 western blotting. Antibodies used were:                replaced autoclaved water 14 days. This cycle (5 days DSS <br /> co&#239;&#172;?lin (Cell Signaling Technology); LIMK1 (Cell Signaling                 plus 14 days water) repeated times. DSS regimes <br /> Technology); LIMK2 (Santa Cruz Biotechnology, Inc.);                      following: cycle 1&#226;&#8364;&#8220;1%; cycle 2&#226;&#8364;&#8220;1.5%; cycle 3&#226;&#8364;&#8220;1.5%. The <br /> &#206;&#177;-tubulin   (&#207;&#402;-Aldrich);    phospho-co&#239;&#172;?lin    (Cell  Signaling            mice culled dissected approximately day 70. Tumour <br />  <br /> Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883                                                                  481 <br />   Colon <br />  <br /> volume determined assuming quasi half-spherical              Statistical analysis <br /> shapeV &#194;&#188; &#194;&#189;&#195;&#176;4=3pr3 &#195;&#382;=2 .                                                 Unless indicated, statistical signi&#239;&#172;?cance determined one <br />                                                                         way analysis variance followed Tukey&#226;&#8364;&#8482;s post-hoc tests. <br /> Lgr5-GFP+ stem cell FACS sorting, RNA puri&#239;&#172;?cation                   Data shown means&#194;&#177;SD unless indicated. Box plots indi- <br /> quantitative reverse transcriptase PCR                                  cate medians bounded quartiles whiskers denoting 5th&#226;&#8364;&#8220; <br /> Crypt-villus organoids removed matrigel centrifugation          95th percentiles. Survival analysis performed SPSS <br /> or freshly isolated intestinal crypts single-cell dissociated   V.19 using method Kaplan&#226;&#8364;&#8220;Meier log-rank test to <br /> TrypLE (Life Technologies) combined DNAse I (2000 U/               compare individual groups. <br /> ml) 30 min 37&#194;&#176;C. Dissociated cells passed a <br /> 40 mm cell strainer incubated DAPI (1:500) dead <br /> cell exclusion. DAPI&#226;&#710;&#8217;/GFP+-positive cells analysed             RESULTS <br /> sorted using FacsAria (BD Bioscience). Cells sorted         LIMK2 expression human CRC <br /> full organoid culture medium plus Y27632 (10 mM). RNA               To determine LIMK2 protein levels affected in <br /> puri&#239;&#172;?ed approximately 5000 GFP+ cells using                    CRC cell lines, &#239;&#172;?rst validated speci&#239;&#172;?city antibody <br /> Arcturus PicoPure RNA isolation kit (Applied Biosystems).               recognises LIMK2a LIMK2b variants17 western <br /> Complementary DNA synthesised using Quantitect                  blotting lysates wild-type Limk2 +/&#226;&#710;&#8217; Limk2 &#226;&#710;&#8217;/&#226;&#710;&#8217; mouse <br /> Reverse Transcription Kit (Qiagen). Quantitative reverse tran-          embryo &#239;&#172;?broblasts (&#239;&#172;?gure 1A; non-speci&#239;&#172;?c band denoted by*). <br /> scriptase (RT)-PCR primers murine Limk2, Lgr5, Olfm4,               Western blotting lysates panel CRC cell lines revealed <br /> Ascl2, CD44, Bmi1, 18s Hprt acquired Qiagen               expression LIMK1, ERK2 LIM kinase substrate <br /> (Quantitect Primer Assay).                                              co&#239;&#172;?lin relatively constant, LIMK2 expression was <br />                                                                         variable numerous cell lines (&#239;&#172;?gure 1B). LIMK2 expression in <br />                                                                         HCT116 SW48 cells lower compared normal human <br /> X-gal staining mouse frozen sections <br />                                                                         colon tissue (&#239;&#172;?gure 1C) protein lysate mass/mass basis (&#239;&#172;?rst <br /> Tumour adjacent normal tissue specimens Ah-CreER <br />                                                                         second vs lane) normalised &#206;&#177;-tubulin levels <br /> APCFl/+ PTENFl/+ Limk2+/&#226;&#710;&#8217; mice isolated, washed PBS <br />                                                                         (vs fourth lane). To determine epigenetics be <br /> and mounted OCT (Tissue-Tek, Sakura) immediately <br />                                                                         in&#239;&#172;&#8218;uencing LIMK2 expression, chose SW48 cells exem- <br /> frozen using Cryospray (CellPath Ltd). Twelve 14 mm thick- <br />                                                                         plar treated de-novo methyltransferase inhibi- <br /> ness sections stained using &#206;&#178;-Gal Staining Kit <br />                                                                         tors RG108 5-Aza.24 Both treatments showed elevated <br /> (Invitrogen, Life Technologies) 48 h 37&#194;&#176;C. <br />                                                                         LIMK2 levels 48 h (&#239;&#172;?gure 1D). Sequencing bisul&#239;&#172;?te <br />                                                                         converted genomic DNA revealed methylation 7 CpG <br /> Flies                                                                   islands ( positions indicated vertical red lines &#239;&#172;?gure 1D <br /> Fly cultures incubations carried standard               line diagram) LIMK2a promoter (&#226;&#710;&#8217;629 &#226;&#710;&#8217;541 relative to <br /> molasses media, food changes 2 days, 25&#194;&#176;C            start transcription) seven CpG islands the <br /> controlled 12 h light-dark cycles described in.20 For RNAi           LIMK2b promoter (+104 +314) reduced 5-Aza <br /> experiments, male dLIMK IR 26 294 28 948 &#239;&#172;&#8218;y lines               treatment (&#239;&#172;?gure 1D, pie charts represent results 10 inde- <br /> crossed virgin female dicer;esg-gal4&gt;GFP;gal80 &#239;&#172;&#8218;ies. F1            pendent DNA isolates). We analysed LIMK2 gene expression <br /> females collected incubation 29&#194;&#176;C 14 days        CRC microarray datasets obtained using Oncomine.25 <br /> effect RNAi. All initial Drosophila stocks used described       There statistically signi&#239;&#172;?cant decreases LIMK2 expression <br /> previously obtained Bloomington Drosophila            adenocarcinoma samples caecum, colon, colon mucin- <br /> Stock Center.                                                           ous, rectal, rectal mucinous rectosigmoid tissue relative to <br />                                                                                                                                    27 <br />                                                                         normal colon tissue (&#239;&#172;?gure 1E; left side;26 right side). LIMK2 <br /> Human tissue microarrays                                                gene expression signi&#239;&#172;?cantly lower lamina proprial or <br /> A CRC tissue microarray (TMA) constructed containing                epithelial adenomas carcinomas relative matched normal <br /> normal colon mucosa (n=50), primary (n=650) metastatic              cells separated laser capture microdissection <br /> CRC (n=285) previously described O&#226;&#8364;&#8482;Dwyer et al,21                 normal, adenoma carcinoma samples28 (&#239;&#172;?gure 1F). <br /> Hope Murray22 Murray et al.23 The metastases               Similarly, LIMK2 expression signi&#239;&#172;?cantly lower carcin- <br /> all lymph node metastasis corresponding stage 3                omas relative normal colorectal samples independent <br />                                                                                                28              29 <br /> (Dukes C) cancers. The histopathology cancers                studies (&#239;&#172;?gure 1G; right side). These results indicate <br /> reported previously described,21 22 clinicopathological          LIMK2 gene expression frequently signi&#239;&#172;?cantly <br /> details summarised supplementary table S1 (available             reduced CRC. <br /> online only). The authors con&#239;&#172;?rm tissue samples              We examined Limk2 levels murine normal intestines <br /> collected strict institutional review board-approved guide-       intestinal tumours genetically modi&#239;&#172;?ed cancer-prone <br /> lines/protocols informed consent (National Research Ethics         mice. We observed Limk2 expression intestines <br /> Service, North Scotland ref. nos. 08/S0801/81 11/NS/             wild-type mice, highest levels <span id='am-394' about='xsp:base' typeof='owl:Thing'>base</span> crypts <br /> 0015). All collected human tissues treated strict con-       (&#239;&#172;?gure 2A), higher magni&#239;&#172;?cation also <br /> &#239;&#172;?dentiality according appropriate applicable laws           marked expression GFP driven promoter of <br /> protect con&#239;&#172;?dentiality personal information. No names            stem cell marker Lgr519 (&#239;&#172;?gure 2B, upper panels). Similarly, <br /> or identity information revealed              colon sections Limk2 degree of <br /> development tissue arrays, end users, result         overlap Lgr5 expressing cells (&#239;&#172;?gure 2B, lower panels). <br /> are exempt consideration Beatson Institute                  Importantly, staining patterns absent mice homo- <br /> Committee Human Subjects Medical Research                  zygous gene-trapped Limk2 loci (&#239;&#172;?gure 2A,B), valid- <br /> being anonymised non-traceable. Cancer Research UK                  ating antibody use immunohistochemistry. To <br /> guidelines followed handling analysis           determine Limk2 deletion associated reduced <br /> material.                                                               signalling output, stained sections antibody that <br />  <br /> 482                                                                          Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883 <br />                                                                                                                                           Colon <br />  <br />  <br />  <br />  <br /> Figure 1 Limk2 expression human colorectal cancers (CRC). (A) Western blot mouse embryo &#239;&#172;?broblast cell lysates indicated genotyptes. <br /> *Non-speci&#239;&#172;?c band (B) Western blot human CRC cell lysates. (C) Western blot CRC cell line normal human colon lysates. Equivalent <br /> protein amounts run left lanes, equivalent &#206;&#177;-tubulin amounts run &#239;&#172;?rst left lanes right lane. (D) Western <br /> blot SW48 cell lysates following 48 h treatment indicated novo methyltransferase inhibitor. CpG methylation LIMK2a LIMK2b <br /> promoters 5-azacytidine treatment. Line diagrams indicate position CpG islands (red lines boxed region) relative the <br /> transcriptional start site promoter. Pie charts indicate proportion methylation multiple independent clones CpG island. <br /> (E) Log2 median-centred LIMK2 messenger RNA expression indicated normal versus adenocarcinoma tissue samples from; left side26 right side.27 <br /> (F) Log2 median-centered LIMK2 mRNA expression indicated microdissected normal lamina propria epithelia, versus corresponding adenoma or <br /> carcinoma samples Skrzypczak et al.28 (G) Log2 median-centered LIMK2 mRNA expression normal versus carcinoma samples left side,28 <br /> right side.29 LIMK2, LIM kinase 2. <br />  <br />  <br /> detects phosphorylation            Limk2 substrate co&#239;&#172;?lin,           crypts, clearly reduced Limk2-KO sections <br /> which previously validated        co&#239;&#172;?lin protein phosphor-       (&#239;&#172;?gure 2C). <br /> ylation speci&#239;&#172;?city.30 Similar        Limk2, pC&#239;&#172;&#8218; localised               We examined Limk2 expression intestinal tumours <br /> throughout intestinal cells      highest levels base     two-hit genetically modi&#239;&#172;?ed mouse model that <br />  <br /> Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883                                                                        483 <br />   Colon <br />  <br />  <br />  <br />  <br /> Figure 2 Limk2 expression mouse intestine tumours. (A) Limk2 expression (red) intestine sections wild-type (WT) Limk2 <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>knockout</span> <br /> (KO) mice (B) Intestine colon sections indicated WT Limk2-KO mice crossed Lgr5&gt;GFP mice cut stained <br /> for Limk2 (red) expression. Merged image DAPI staining cell nuclei. (C) Intestine sections WT Limk2-KO mice cut and <br /> stained phosphorylated co&#239;&#172;?lin ( pC&#239;&#172;&#8218;; red). Merged image DAPI staining cell nuclei. (D) Immunohistochemical analysis Limk2 <br /> expression intestinal tumour adjacent normal tissue Ah-Cre:ER, ApcFl/+, PtenFl/+ mouse. (E) &#206;&#178;-Galactosidase activity normal and <br /> dysplastic crypts detected X-gal staining frozen intestinal tissue sections heterozygous Limk2 gene-trap, Ah-Cre, ApcFl/+, PtenFl/+ mice. <br />  <br />  <br /> requires stochastic loss heterozygosity Apc Pten.31                These data indicate apparently high expression of <br /> Limk2 expression Ah-Cre:ER, Apc Fl/+, Pten Fl/+ mice                   Limk2 mouse intestinal cells repressed, probably via <br /> markedly lower tumours surrounding normal tissue                   promoter methylation consequent inhibition, during <br /> (&#239;&#172;?gure 2D; tumour border outlined red dashed line). Mice                 tumorigenesis. <br /> bearing copy &#206;&#178;-galactosidase gene-trap Limk2                     To determine patterns LIMK2 expression changes <br /> locus context Ah-Cre:ER, ApcFl/+, PtenFl/+ genotype               associated oncogenesis tumour progression, we <br /> showed reduced &#206;&#178;-galactosidase activity (visualised X-gal)               used LIMK2 antibody stain representative human intes- <br /> in dysplastic crypts compared normal crypts, indicating inhib-             tinal adenoma samples. Similar normal tissue, LIMK2 expres- <br /> ition Limk2 promoter transcriptional activity (&#239;&#172;?gure 2E).                  sion relatively high low grade adenomas markedly <br />  <br /> 484                                                                                 Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883 <br />                                                                                                                                      Colon <br />  <br />  <br />  <br />  <br /> Figure 3 LIM kinase 2 (LIMK2) expression human colorectal cancer (CRC) progression. (A) Immunohistochemical analysis LIMK2 expression in <br /> human intestinal low grade high grade adenomas. (B) Immunohistochemical analysis LIMK2 expression tissue microarray (TMA) of <br /> human colorectal tumours lymph node metastases. (C) LIMK2 staining intensities TMA human CRC. (D) Cumulative survival curves for <br /> high low LIMK2 expressing patients. <br />  <br /> decreased high grade adenomas low high levels               supplementary table S2, available online only). LIMK2 expres- <br /> dysplasia (&#239;&#172;?gure 3A). We stained well-characterised TMA            sion lymph node metastases signi&#239;&#172;?cantly lower all <br /> comprised 650 patient samples normal intestines                 stages Dukes C (&#239;&#172;?gure 3B,C, supplementary table S2, <br /> tissue Dukes A Dukes C tumours, representa-            available online only). To determine association between <br /> tive lymph node metastases.21 By assigning staining intensity             LIMK2 expression survival, grouped staining categories <br /> scores 1 (low) 6 (high), established median               low (scores 1&#226;&#8364;&#8220;3) high (scores 4&#226;&#8364;&#8220;6) that <br /> staining intensity normal intestines scored 5 relatively          patients low LIMK2 expression signi&#239;&#172;?cantly <br /> little variation (&#239;&#172;?gure 3B,C). LIMK2 expression progressively             (&#207;&#8225;2=14.51, p&lt;0.001; log rank Mantel&#226;&#8364;&#8220;Cox test) lower mean <br /> dropped tumours, Dukes stage having signi&#239;&#172;?cantly             survival times (93.9 months) high LIMK2 expressing <br /> lower LIMK2 expression predecessor (&#239;&#172;?gure 3B,C,              patients (118.2 months) (&#239;&#172;?gure 3D supplementary table <br />  <br /> Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883                                                                    485 <br />   Colon <br />  <br />  <br />  <br />  <br /> Figure 4 Phosphorylated co&#239;&#172;?lin human colorectal cancer (CRC) progression. (A) Immunohistochemical analysis pCFL expression tissue <br /> microarray (TMA) human colorectal tumours lymph node metastases. (B) pCFL staining intensities TMA human CRC. (C) Cumulative <br /> survival curves high low pCFL-expressing patients. <br />  <br /> S3, available online only). When median mean survival            positive pixel algorithm32 33 yielded Pearson product&#226;&#8364;&#8220;moment <br /> times estimated scoring rank, clear            correlation coef&#239;&#172;?cient r=0.59 (&#239;&#172;?gure 5B). When both <br /> trend highest survival patients scoring 6 lowest sur-        LIMK2 pCFL variables analysed together, best patient <br /> vival patients scoring 1 (see supplementary table S4, avail-         mean survival time high LIMK2 pCFL <br /> able online only). To determine reduced LIMK2                    (118.9 months, 38.7% total patients; &#239;&#172;?gure 5C, supple- <br /> expression levels associated lower signalling output,          mentary table S8, available online only), worst mean sur- <br /> we stained TMA antibody detects co&#239;&#172;?lin phos-           vival time low LIMK2 pCFL (84.6 months, 36.1% <br /> phorylation.30 Levels pCFL signi&#239;&#172;?cantly progres-             total patients; &#239;&#172;?gures 5C, supplementary table S8, avail- <br /> sively reduced relative normal intestine             able online only). These data reveal clear patterns reduced <br /> Dukes stage, exceptions pCFL Dukes B tumours            LIMK2 expression activity CRC, signi&#239;&#172;?cant associ- <br /> did differ Dukes A, did pCFL lymph node                  ation patient mortality. <br /> metastases differ Dukes C tumours (&#239;&#172;?gures 4A,B, see <br /> supplementary table S5, available online only). Mean patient             LIMK2 promoter methylation human tumours <br /> survival signi&#239;&#172;?cantly (&#207;&#8225;2=14.81, p&lt;0.001; log-rank                   The DNMT inhibitor-sensitive promoter methylation CRC <br /> Mantel&#226;&#8364;&#8220;Cox test) longer high pCFL patients (119.3 months)             cell line (&#239;&#172;?gure 1D) suggested similar effect promoter <br /> than low pCFL patients (91.2 months) (&#239;&#172;?gure 4C                methylation account reduction LIMK2 expres- <br /> supplementary table S6, available online only). When                 sion human tumours (&#239;&#172;?gure 3). By isolating genomic DNA <br /> median mean survival times estimated pCFL              frozen tumour sections represented TMA along <br /> scoring rank, clear trend highest survival          matched normal tissue isolated patients, we <br /> patients scoring 6 lowest survival patients scoring 1 (see        able compare patterns promoter methylation using <br /> supplementary table S7, available online only). When examining           bisul&#239;&#172;?te sequencing. The seven CpG islands LIMK2a and <br /> on patient patient basis, obvious correlation          LIMK2b promoters typically methylated tumours <br /> between LIMK2 expression pCFL levels (&#239;&#172;?gure 5A),               matched normal tissues (results patient example <br /> when compared using SlidePath tissue image analysis           depicted &#239;&#172;?gure 6A). When percentages CpG island <br />  <br /> 486                                                                            Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883 <br />                                                                                                                                      Colon <br />  <br />  <br />  <br />  <br /> Figure 5 Association LIM kinase 2 (LIMK2) expression co&#239;&#172;?lin phosphorylation patient outcomes. (A) Staining LIMK2 pCFL from <br /> paired patient sections. (B) LIMK2 pCFL staining intensities measured using SlidePath tissue image analysis positive pixel <br /> algorithm.32 33 (C) Distribution outcomes combinations high low LIMK2 pCFL. <br />  <br />  <br /> methylation LIMK2a LIMK2b promoters compiled                 increased stem cell proliferation.42 To measure whether <br /> from patients, signi&#239;&#172;?cantly higher CpG methyla-             dLIMK reduction affected cell proliferation directly, &#239;&#172;?xed <br /> tion tumours compared matched normal tissues. These                 midguts stained phosphorylated Histone 3 ( pH3) <br /> results support conclusion reduced LIMK2 expression              antibody marker mitotic cells.41 While pH3 staining was <br /> in human CRC increased promoter methylation.                    low absent midguts wild-type &#239;&#172;&#8218;ies under <br />                                                                           homeostatic conditions, dLIMK knockdown independ- <br /> Stem cell homeostasis D melanogaster midgut                            ent RNAi transgenes signi&#239;&#172;?cantly ( p&lt;0.001) increased the <br /> To understand repression LIMK2 expression con-               mean number pH3-positive cells relative control geno- <br /> tribute CRC, used D melanogaster genetically tract-            type &#239;&#172;?vefold (&#239;&#172;?gure 7D,E). Co-staining DAPI <br /> able model examine role LIM kinase                    revealed patterns condensed chromatin supported the <br /> gastrointestinal homeostasis.34&#226;&#8364;&#8220;36 Importantly, experiments            conclusion pH3-positive cells mitotic (&#239;&#172;?gure 7D, right <br /> the Drosophila posterior midgut, homologous               panels). Haematoxylin eosin (H&amp;E) stained cross-sections <br /> mammalian intestine, established orthologues             paraf&#239;&#172;?n-embedded posterior midguts control &#239;&#172;&#8218;ies were <br /> genes frequently mutated human CRC (eg, APC, &#206;&#178;-catenin,                typical appearance, continuous cell layer sur- <br /> etc.) play important roles regulation intestinal stem           rounding substantial lumen (&#239;&#172;?gure 6D, left panel). In contrast, <br /> cell proliferation tissue homeostasis.37&#226;&#8364;&#8220;39 We used                   H&amp;E stained midgut cross-sections UAS-dLIMK RNAi &#239;&#172;&#8218;ies <br /> inverted-repeat <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>RNA interference</span> transgenes (RNAi)              signi&#239;&#172;?cantly thicker (&#239;&#172;?gure 6D, right panel, &#239;&#172;?gure 7G). <br /> transcriptional control Gal4/UAS (upstream activating              These data indicate Drosophila, LIM kinase normally <br /> sequence) system40 reduce expression single                     contributes intestinal homeostasis limiting proliferation <br /> Drosophila LIM kinase homologue (dLIMK) stem cells                     stem cells. <br /> expressing transcription factor Gal4 driven escargot <br /> (esg) promoter.41 When dLIMK knocked esg-Gal4                 LIMK2 deletion increases Lgr5-positive mouse intestinal <br /> driven expression UAS-dLIMK RNAi &#239;&#172;&#8218;ies carrying               stem cells <br /> UAS-GFP transgene mark stem cells, readily apparent             We wished determine conserved <br /> that, compared control &#239;&#172;&#8218;ies (wild-type), dLIMK knockdown               role Limk2 regulating mouse intestinal stem cell homeo- <br /> signi&#239;&#172;?cantly increased proportion GFP-expressing stem              stasis. Using method allows isolation long- <br /> cells/progenitor cells (&#239;&#172;?gure 7A,B) demarcated          term in-vitro propagation self-renewing intestinal epithelial <br /> their relatively elevated expression &#206;&#178;-catenin homologue           cells,15 examined Limk2 deletion affected prolifer- <br /> Armadillo Delta (&#239;&#172;?gure 7C). This <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> consistent              ation Lgr5-positive stem cell population.19 When initially <br />  <br /> Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883                                                                   487 <br />   Colon <br />  <br />  <br />  <br />  <br /> Figure 6 CpG methylation LIMK2a LIMK2b promoters normal tumour tissue. (A) Line diagrams indicate position CpG islands (red <br /> lines boxed region) relative transcriptional start site promoter. Pie charts indicate proportion methylation multiple <br /> independent clones CpG island genomic DNA representative patient. (B) Cumulative LIMK2a LIMK2b promoter methylation <br /> data matched normal tumour tissue derived patients. <br />  <br /> isolated wild-type Limk2-KO mice, intestinal epithelial              increased number cells indistinguishable from <br /> cells showed signi&#239;&#172;?cant differences expression                  wild-type stem cell counterparts. By inference, epigenetic <br /> various stem cell markers western blotting RNA micro-                  modi&#239;&#172;?cations LIMK2 promoters resulted tran- <br /> array (see supplementary &#239;&#172;?gure S1A,B, available online only),                scriptional repression positively selected human <br /> while percentages Lgr5&gt;GFP-expressing cells did                  CRC increased response stem cells inducers of <br /> not signi&#239;&#172;?cantly differ (see supplementary &#239;&#172;?gure S1C, available              proliferation. <br /> online only). However, cells cultured passaged <br /> twice (every 7 days) presence proliferation-inducing           Limk2 deletion promotes crypt dysplasia tumour size in <br /> growth factors epidermal growth factor, R-spondin1                       mouse model colitis-associated CRC <br /> Noggin embedded three-dimensional Matrigel,                   To test effect LIMK2 deletion mouse model of <br /> was greater proportion Lgr5&gt;GFP-positive cells                   colitis-associated CRC, 6&#226;&#8364;&#8220;12-week-old mice injected with <br /> Limk2-KO organoids relative wild-type revealed                      single dose AOM (12.5 mg/kg), potent carcinogen that <br /> immuno&#239;&#172;&#8218;uorescence microscopy (&#239;&#172;?gure 8A; white arrows illus-                  induces formation O6-methyl-guanine, followed three <br /> trating example GFP-positive cells). When cells multiple                cycles DSS administration drinking water (&#239;&#172;?gure 9A). <br /> independent organoids dissociated GFP-expressing                    DSS administration cause chronic in&#239;&#172;&#8218;ammation, enhancing <br /> cells counted FACS (&#239;&#172;?gure 8B, 10 000 events gated con-                incidence AOM-induced tumours.43 Limk2-de&#239;&#172;?cient <br /> dition), signi&#239;&#172;?cant ( p&lt;0.05, Student&#226;&#8364;&#8482;s t test)                  mice display higher sensitivity AOM+DSS protocol <br /> increase percentage Lgr5&gt;GFP-positive stem cells                signi&#239;&#172;?cant increase weight variation (&#239;&#172;?gure 9B,C), <br /> the Limk2-KO relative wild-type (&#239;&#172;?gure 8C). Western blotting              re&#239;&#172;&#8218;ecting increased weight loss second third <br /> lysates multiple independent organoids corroborated                 DSS cycles. At end treatment regimen, Limk2-KO <br /> elevation Lgr5 driven GFP expression (indicating greater                mice marked increase crypt-villus dysplasia hyper- <br /> proportion GFP-positive cells) Limk2-KO (&#239;&#172;?gure 8D).                plasia middle distal colon (&#239;&#172;?gure 9D). The <br /> When GFP-positive cells isolated collected &#239;&#172;&#8218;ow                   caecum displayed severe dysplastic hyperproli- <br /> cytometry, RT-PCR analysis revealed signi&#239;&#172;?cant differences                ferative phenotype, common feature this <br /> in expression number stem cell markers including                 tumour model (&#239;&#172;?gure 9D,E). There marked trend <br /> Lgr5, Bmi1, Olfm4, CD44 Ascl2 wild-type                  higher tumour incidence Limk2-KO mice (&#239;&#172;?gure <br /> Limk2-KO, indicating equivalent levels &#226;&#8364;&#732;stemness&#226;&#8364;&#8482;                  9F). The tumours adenomas high &#206;&#178;-catenin <br /> GFP-positive cells genotype (&#239;&#172;?gure 8E). These results              expression. The striking signi&#239;&#172;?cant effects on <br /> indicate conditions strongly promote prolif-             histology volume &#206;&#178;-catenin-positive tumour foci <br /> eration intestinal stem cells, deletion Limk2 provides           Limk2-KO mice. Limk2-KO foci signi&#239;&#172;?cantly larger <br /> competitive advantage relative wild-type cells, giving rise            (&#239;&#172;?gure 9G) locally invasive, showing higher grade of <br />  <br /> 488                                                                                Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883 <br />                                                                                                                                     Colon <br />  <br /> Figure 7 Knockdown dLIMK <br /> increases Drosophila intestinal stem <br /> cell proliferation tissue thickening. <br /> (A) Fly intestinal stem progenitor <br /> cells marked esg-driven GFP <br /> and endogenous Armadillo (red) <br /> expression, &#239;&#172;&#8218;ies expressing dLIMK <br /> RNAi inverted repeat hairpins have <br /> increased stem cell numbers compared <br /> to control genotype (wild-type; <br /> WT). (B) GFP/DAPI-positive cells were <br /> counted &#239;&#172;?ve sections &#239;&#172;?ve WT <br /> and dLIMK siRNA &#239;&#172;&#8218;y lines. <br /> (C) Cells expressing esg&gt;GFP also <br /> marked elevated expression of <br /> Armadillo Delta relative to <br /> neighbouring cells. (D) Proliferation <br /> determined staining mitotic <br /> marker phosphorylated Histone 3 <br /> ( pH3) control (WT) dLIMK <br /> targeting RNAi. Higher magni&#239;&#172;?cation <br /> images pH3 DAPI staining <br /> reveal appearance condensed <br /> chromosomes mitotic cells. <br /> (E). Quanti&#239;&#172;?cation pH3 numbers in <br /> &#239;&#172;&#8218;ies expressing independent <br /> dLIMK RNAi hairpins revealed <br /> statistically signi&#239;&#172;?cant increases relative <br /> to control WT &#239;&#172;&#8218;ies. (F) Cross-sections <br /> of &#239;&#172;?xed H&amp;E stained midguts from <br /> control (WT) dLIMK RNAi-expressing <br /> &#239;&#172;&#8218;ies. (G) Average gut wall widths <br /> determined separate <br /> measurements cross-section <br /> from &#239;&#172;?ve WT dLIMK RNAi &#239;&#172;&#8218;ies. <br /> *p&lt;0.05 Student&#226;&#8364;&#8482;s t test. <br />  <br />  <br />  <br />  <br /> dysplasia (&#239;&#172;?gure 9E,G). These results indicate                   gastrointestinal homeostasis tumorigenesis,44 using <br /> removal Limk2 colitis-associated CRC mouse model                  intestinal epithelial cells isolated genetically modi&#239;&#172;?ed mice. <br /> had mild effect tumour initiation, signi&#239;&#172;?cant              These studies revealed conserved role LIMK2 constrain- <br /> impact tumour growth acquisition                  ing gastrointestinal stem cells Drosophila mice, that <br /> invasive phenotype. This relief                relief constraint allowed expansion stem cell <br /> proliferative constraint gastrointestinal stem cells con-        numbers. Given accumulating evidence indicates stem <br /> ditions favour continuous self-renewal, contrib-          cells predominant cells origin colorectal <br /> uting CRC.                                                             tumours,2&#226;&#8364;&#8220;5 logical conclusion study a <br />                                                                           positive selection reduced LIMK2 expression and/or activity <br /> DISCUSSION                                                                CRC initiation progression allow propagation of <br /> We LIMK2 expression reduced intestinal              gastrointestinal stem cells. Consistent conclusion, <br /> tumours cancer-prone genetically modi&#239;&#172;?ed mice,           genetic deletion Limk2 reduce expression AOM/ <br /> human CRC cell lines tumours. To gain insight                DSS colitis-associated CRC mouse model resulted increased <br /> potential roles LIMK2, undertook genetic analysis            average intestinal tumour volume severe dysplastic <br /> Drosophila midgut, emerging informative model              hyperproliferative phenotype. <br />  <br /> Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883                                                                  489 <br />   Colon <br />  <br /> Figure 8 Limk2 deletion increases <br /> cultured mouse intestinal stem cell <br /> proportion. (A) Self-renewing intestinal <br /> epithelial cells isolated from <br /> wild-type (WT) Limk2 knockout <br /> (KO) mice propagated vitro in <br /> the presence growth factors <br /> EGF, R-spondin1 Noggin while <br /> embedded three-dimensional <br /> Matrigel. Organoids &#239;&#172;?xed, <br /> stained &#239;&#172;?lamentous actin with <br /> phalloidin, visualised for <br /> Lgr5-driven GFP expression and <br /> phalloidin staining confocal <br /> microscopy. (B) Organoids were <br /> isolated disrupted yield single <br /> cell suspensions. After brief <br /> incubation propidium iodide (PI), <br /> FACS analysis 10 000 PI-negative <br /> live cells used determine the <br /> proportion Lgr5&gt;GFP-positive cells <br /> in genotype. (C) Graph results <br /> from independent FACS <br /> experiments. *p&lt;0.05 Student&#226;&#8364;&#8482;s t <br /> test. (D) Equivalent cell numbers from <br /> disrupted organoids intestinal <br /> epithelial cells WTand Limk2 KO <br /> mice used prepare cell <br /> lysates western blotted as <br /> indicated. (E) RT-PCR analysis stem <br /> cell marker expression sorted <br /> GFP-positive cells WT and <br /> Limk2-KO mice expressing Lgr5&gt;GFP. <br /> The expression levels indicated <br /> stem cell marker did vary <br /> signi&#239;&#172;?cantly WT and <br /> Limk2-KO mice, indicating equivalent <br /> &#226;&#8364;&#732;stemness&#226;&#8364;&#8482;. <br />  <br />  <br />  <br />  <br />   The reduced LIMK2 expression CRC surprising given           invasion, reports negative roles LIM kinase functions <br /> the reports elevated LIM kinase expression cancers       cell cycle regulation overlooked. <br /> including breast,10 11 malignant melanoma12 prostate13           Elevated LIMK2 levels following DNA damage to <br /> tumours. Because role LIM kinases regulating acto-      promote G2/M arrest co&#239;&#172;?lin,46 consistent a <br /> myosin cytoskeletal dynamics, assumed         proposed role LIMK2 mitosis regulating micro- <br /> would consistently elevated LIM kinase expression              tubule organisation spindle orientation.47 48 Given that <br /> advanced cancers drive local invasion distant metastasis.     expression LIMK2a LIMK2b variants been <br /> Indeed, showed inhibition LIM kinase activity          reported reduced thyroid cancers,46 important <br /> RNAi chemical inhibition blocked collective invasion       re-consider LIMK2 contributes speci&#239;&#172;?c cancers rather <br /> breast cancer squamous cell carcinomas blocking           attempting construct uni&#239;&#172;?ed theory. <br /> invasive ability leading cells using in-vitro three-dimensional     In actual fact, role LIM kinases regulating co&#239;&#172;?lin <br /> invasion models.45 However, focusing LIM kinase            phosphorylation &#239;&#172;?lamentous actin turnover nuanced, and <br /> activity in&#239;&#172;&#8218;uences processes involved cell motility           circumstances actually favourable for <br />  <br /> 490                                                                       Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883 <br />                                                                                                                                     Colon <br />  <br /> Figure 9 Limk2 deletion increases <br /> tumour volume dysplasia a <br /> colitis-associated colorectal cancer <br /> mouse model. (A). Timeline of <br /> experimental procedure. (B) Mean <br /> weights wild-type Limk2 <br /> knockout mice experimental <br /> procedure relative starting weight <br /> (100%). (C) Coef&#239;&#172;?cient variation in <br /> weight curves wild-type Limk2 <br /> knockout mice. *p&lt;0.05 Student&#226;&#8364;&#8482;s t <br /> test. (D) Cross-sections colon and <br /> caecum wild-type Limk2 <br /> knockout mice indicated. Boxes <br /> indicate regions depicted higher <br /> magni&#239;&#172;?cation images. (E) Serial <br /> sections colon tissue wild-type <br /> and Limk2 knockout mice indicated. <br /> Boxes indicate regions depicted in <br /> higher magni&#239;&#172;?cation images. Elevated <br /> &#206;&#178;-catenin staining relative to <br /> surrounding tissue used identify <br /> tumours de&#239;&#172;?ne dimensions. <br /> (F) Number &#206;&#178;-catenin-positive foci <br /> per colon section wild-type and <br /> Limk2 knockout mice indicated. p <br /> Value Student&#226;&#8364;&#8482;s t test. (G) Volume <br /> of &#206;&#178;-catenin-positive foci from <br /> wild-type Limk2 knockout mice as <br /> indicated. *p&lt;0.05 Student&#226;&#8364;&#8482;s t test. <br />  <br />  <br />  <br />  <br /> increased invasive ability reduced LIM kinase             gastrointestinal stem cells, additional contributions <br /> expression activity. We previously ectopic expres-         reduced LIMK2 signal output ion promoting tumour cell inva- <br /> sion LIMK1 decreased motility Ras-transformed Swiss             siveness. As result, actually distinct ways that <br /> 3T3 &#239;&#172;?broblasts,49 overexpression LIMK1 MTLn3                  reduction LIMK2 expression contributes CRC; through <br /> rat mammary carcinoma cells inhibited lamellipod extensions,50            relief constraints stem cells early tumorigenesis, by <br /> EGF-stimulated motility, vivo invasion.51 The relative             promoting cell invasiveness advanced stages. <br /> proportions LIM kinase co&#239;&#172;?lin, activity             An inference study LIMK2 levels <br /> co&#239;&#172;?lin phosphatases Slingshot, contribute regulating           decreased sizeable proportion CRC patients, these <br /> the co&#239;&#172;?lin activity cycle key determinant invasive         cancers effectively treated che- <br /> and metastatic potential.52 53 The continuous decreases                motherapeutics knocking and/or inhibiting <br /> LIMK2 expression co&#239;&#172;?lin phosphorylation advancing                LIMK2 sensitises cancer cells DNA damaging agents,17 <br /> tumour grade suggests that, addition effects                 Aurora A inhibitors,54 Ras inhibition55 microtubule- <br />  <br />  <br /> Louren&#195;&#167;o FC, et al. Gut 2014;63:480&#226;&#8364;&#8220;493. doi:10.1136/gutjnl-2012-303883                                                                  491 <br />   Colon <br />  <br /> destabilising drugs vincristine vinblastine.56                                     19   Barker N, van Es JH, Kuipers J, et al. Identi&#239;&#172;?cation stem cells small intestine <br /> Therefore, knowledge LIMK2 expression levels CRC                                          colon marker gene Lgr5. Nature 2007;449:1003&#226;&#8364;&#8220;7. <br />                                                                                                20   Cordero JB, Macagno JP, Stefanatos RK, et al. Oncogenic Ras diverts host TNF <br /> have utility patient strati&#239;&#172;?cation development per-                                   tumor suppressor activity tumor promoter. Dev Cell 2010;18:999&#226;&#8364;&#8220;1011. <br /> sonalised therapies.                                                                           21   O&#226;&#8364;&#8482;Dwyer D, Ralton LD, O&#226;&#8364;&#8482;Shea A, et al. The proteomics colorectal cancer: <br />                                                                                                     identi&#239;&#172;?cation protein signature associated prognosis. PLoS One 2011;6: <br /> Acknowledgements The authors like thank Colin Nixon colleagues                         e27718. <br /> in histology services biological research service Beatson Institute          22   Hope NR, Murray GI. The expression pro&#239;&#172;?le RNA-binding proteins primary and <br /> Cancer Research. They acknowledge support Think Pink breast cancer                  metastatic colorectal cancer: relationship heterogeneous nuclear <br /> group funded purchase NDP slide scanner.                                                 ribonucleoproteins prognosis. Hum Pathol 2011;42:393&#226;&#8364;&#8220;402. <br /> Contributors Study concept design: FCL, JC, MV, GIM, MFO; acquisition                   23   Murray GI, Patimalla S, Stewart KN, et al. Pro&#239;&#172;?ling expression cytochrome <br /> data: FCL, JM, JB, CO, GIM, MFO; analysis interpretation data: FCL, JC, OJS,                 P450 breast cancer. Histopathology 2010;57:202&#226;&#8364;&#8220;11. <br /> MV, GIM, MFO; drafting manuscript: FCL, MFO; critical revision                   24   Foulks JM, Parnell KM, Nix RN, et al. Epigenetic drug discovery: targeting DNA <br /> manuscript important intellectual content: FCL, OJS, MV, GIM, MFO; statistical                  methyltransferases. J Biomol Screen 2012;17:2&#226;&#8364;&#8220;17. <br /> analysis: FCL, GIM, MFO; obtained funding: OJS, MV, GIM, MFO; administrative,                  25   Rhodes DR, Yu J, Shanker K, et al. ONCOMINE: cancer microarray database and <br /> technical, material support: RS, KS, SMF.                                                        integrated data-mining platform. Neoplasia 2004;6:1&#226;&#8364;&#8220;6. <br />                                                                                                26   Kaiser S, Park YK, Franklin JL, et al. Transcriptional recapitulation subversion of <br /> Funding This study funded Cancer Research UK.                                                embryonic colon development mouse colon tumor models human colon <br /> Competing interests None.                                                                           cancer. Genome Biol 2007;8:R131. <br />                                                                                                27   The Cancer Genome Atlas Network. Comprehensive molecular characterization of <br /> Patient consent Obtained.                                                                           human colon rectal cancer. Nature 2012;487:330&#226;&#8364;&#8220;7. <br /> Ethics approval All tissue samples collected strict institutional review            28   Skrzypczak M, Goryca K, Rubel T, et al. Modeling oncogenic signaling colon <br /> board-approved guidelines/protocols informed consent (National Research                        tumors multidirectional analyses microarray data directed maximization of <br /> Ethics Service, North Scotland).                                                                 analytical reliability. PLoS One 2010;5:e13091. <br />                                                                                                29   Hong Y, Downey T, Eu KW, et al. A &#226;&#8364;&#732;metastasis-prone&#226;&#8364;&#8482; signature early-stage <br /> Provenance peer review Not commissioned; externally peer reviewed. <br />                                                                                                     mismatch-repair pro&#239;&#172;?cient sporadic colorectal cancer patients implications for <br /> Data sharing statement Data shared request.                                              possible therapeutics. Clin Exp Metastasis 2010;27:83&#226;&#8364;&#8220;90. <br /> Open Access This Open Access article distributed accordance Creative         30   Samuel MS, Lopez JI, McGhee EJ, et al. Actomyosin-mediated cellular tension drives <br /> Commons Attribution Non Commercial (CC BY-NC 3.0) license, permits                  increased tissue stiffness &#206;&#178;-catenin activation induce epidermal hyperplasia <br /> distribute, remix, adapt, build work non-commercially, license                  tumor growth. Cancer Cell 2011;19:776&#226;&#8364;&#8220;91. <br /> derivative works different terms, provided original work properly cited      31   Marsh V, Winton DJ, Williams GT, et al. Epithelial Pten dispensable intestinal <br /> use non-commercial. See: http://creativecommons.org/licenses/by-nc/3.0/                          homeostasis suppresses adenoma development progression Apc <br />                                                                                                     <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span>mutation</span>. Nat Genet 2008;40:1436&#226;&#8364;&#8220;44. <br />                                                                                                32   Conway C, Dobson L, O&#226;&#8364;&#8482;Grady A, et al. Virtual microscopy enabler of <br />                                                                                                     automated/quantitative assessment protein expression TMAs. Histochem Cell <br /> REFERENCES                                                                                          Biol 2008;130:447&#226;&#8364;&#8220;63. <br />  1    Davies EJ, Marsh V, Clarke AR. Origin maintenance intestinal cancer           33   Mohammed ZM, Going JJ, McMillan DC, et al. Comparison visual and <br />       stem cell. Mol Carcinog 2011;50:254&#226;&#8364;&#8220;63.                                                       automated assessment HER2 status impact outcome primary <br />  2    Schepers AG, Snippert HJ, Stange DE, et al. Lineage tracing reveals Lgr5+ stem cell           operable invasive ductal breast cancer. Histopathology 2012;61:675&#226;&#8364;&#8220;84. <br />       activity mouse intestinal adenomas. Science 2012;337:730&#226;&#8364;&#8220;5.                           34   Polesello C, Roch F, Gobert V, et al. Modeling cancers Drosophila. Prog Mol Biol <br />  3    Barker N, Ridgway RA, van Es JH, et al. Crypt stem cells cells-of-origin            Transl Sci 2011;100:51&#226;&#8364;&#8220;82. <br />       intestinal cancer. Nature 2009;457:608&#226;&#8364;&#8220;11.                                               35   Rudrapatna VA, Cagan RL, Das TK. Drosophila cancer models. Dev Dyn <br />  4    Merlos-Suarez A, Barriga FM, Jung P, et al. The intestinal stem cell signature                2012;241:107&#226;&#8364;&#8220;18. <br />       identi&#239;&#172;?es colorectal cancer stem cells predicts disease relapse. Cell Stem Cell      36   Cordero JB, Sansom OJ. Wnt signalling role stem cell-driven intestinal <br />       2011;8:511&#226;&#8364;&#8220;24.                                                                                regeneration hyperplasia. Acta physiol 2012;204:137&#226;&#8364;&#8220;43. <br />  5    Ziskin JL, Dunlap D, Yaylaoglu M, et al. In situ validation intestinal stem cell   37   Lee WC, Beebe K, Sudmeier L, et al. Adenomatous polyposis coli regulates <br />       signature colorectal cancer. Gut 2013;62:1012&#226;&#8364;&#8220;23.                                          Drosophila intestinal stem cell proliferation. Development 2009;136:2255&#226;&#8364;&#8220;64. <br />  6    Olson MF, Sahai E. The actin cytoskeleton cancer cell motility. Clin Exp              38   Lin G, Xu N, Xi R. Paracrine Wingless signalling controls self-renewal Drosophila <br />       Metastasis 2009;26:273&#226;&#8364;&#8220;87.                                                                    intestinal stem cells. Nature 2008;455:1119&#226;&#8364;&#8220;23. <br />  7    Samuel MS, Olson MF. Dying alone: tale rho. Cell Stem Cell 2010;7:135&#226;&#8364;&#8220;6.            39   Cordero J, Vidal M, Sansom O. APC master regulator intestinal homeostasis <br />  8    Scott RW, Olson MF. LIM kinases: function, regulation association human              transformation: &#239;&#172;&#8218;ies vertebrates. Cell Cycle 2009;8:2926&#226;&#8364;&#8220;31. <br />       disease. J Mol Med 2007;85:555&#226;&#8364;&#8220;68.                                                       40   Brand AH, Perrimon N. Targeted gene expression means altering cell fates <br />  9    Manetti F. Recent &#239;&#172;?ndings con&#239;&#172;?rm LIM domain kinases emerging target                        generating dominant phenotypes. Development 1993;118:401&#226;&#8364;&#8220;15. <br />       candidates cancer therapy. Curr Cancer Drug Targets 2012;12:543&#226;&#8364;&#8220;60.                  41   Micchelli CA, Perrimon N. Evidence stem cells reside adult Drosophila <br /> 10    Bagheri-Yarmand R, Mazumdar A, Sahin AA, et al. LIM kinase 1 increases tumor                  midgut epithelium. Nature 2006;439:475&#226;&#8364;&#8220;9. <br />       metastasis human breast cancer cells regulation urokinase-type             42   Jiang H, Patel PH, Kohlmaier A, et al. Cytokine/Jak/Stat signaling mediates <br />       plasminogen activator system. Int J Cancer 2006;118:2703&#226;&#8364;&#8220;10.                                  regeneration homeostasis Drosophila midgut. Cell 2009;137:1343&#226;&#8364;&#8220;55. <br /> 11    McConnell BV, Koto K, Gutierrez-Hartm <br /> </body></html>