<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                      Vol. 29 no. 3 2013, pages 355&#226;&#8364;&#8220;364 <br /> BIOINFORMATICS                                            ORIGINAL PAPER                                               doi:10.1093/bioinformatics/bts688 <br />  <br />  <br /> Data text mining                                                                                     Advance Access publication December 11, 2012 <br />  <br />  <br /> A novel link prediction algorithm reconstructing <br /> protein&#226;&#8364;&#8220;protein interaction networks topological similarity <br /> Chengwei Lei Jianhua Ruan* <br /> Department Computer Science, The University Texas San Antonio, San Antonio, TX 78249, USA <br /> Associate Editor: Jonathan Wren <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                         functional genomics data, offering unprecedented opportu- <br /> Motivation: Recent advances technology dramatically                      nities biological computational scientists under- <br /> increased availability protein&#226;&#8364;&#8220;protein interaction (PPI) data         stand cell systems level (Przulj, 2011). For example, <br /> stimulated development methods improving sys-                global analysis PPI networks revealed important connec- <br /> tems level understanding cell. However, efforts              tions topology function (Han et al., 2004; Jeong <br /> significantly hindered high level noise, sparseness highly         et al., 2001; Yu et al., 2007). PPI networks used <br /> skewed degree distribution PPI networks. Here, present novel             predicting gene functions, functional pathways protein <br /> algorithm reduce noise present PPI networks. The key idea           complexes, supervised unsupervised methods <br /> our algorithm proteins sharing higher-order topo-               (Asthana et al., 2004; Bader Hogue, 2002; Chua et al., <br /> logical similarities, measured novel random walk-based proced-              2006; Friedel et al., 2009; King et al., 2004; Lee et al., 2008; <br /> ure, likely interacting belong           Sharan et al., 2007; Ulitsky Shamir, 2009; Wang et al., <br /> protein complex.                                                                 2007a; Wang et al., 2010). Furthermore, effort been <br /> Results: Applying algorithm yeast PPI network,            devoted recently incorporating PPI networks obtain a <br /> the edges reconstructed network higher biological rele-              better mechanistic understanding complex diseases im- <br /> vance original network, assessed multiple types in-            prove diagnosis treatment diseases (Chuang et al., <br /> formation, including gene ontology, gene expression, essentiality,               2007; Hannum et al., 2009; Hidalgo et al., 2009; Ideker and <br /> conservation species known protein complexes.                        Sharan, 2008; Kim et al., 2011). <br /> Comparison existing methods shows network recon-                      However, growing size complexity PPI networks <br /> structed method highest quality. Using independ-              poses multiple challenges biologists. First, PPI networks often <br /> ent graph clustering algorithms, reconstructed                 high false-positive rate higher false-negative <br /> network resulted significantly improved prediction accuracy            rate (Huang et al., 2007). Second, PPI networks typically <br /> protein complexes. Furthermore, method applicable PPI net-             sparse, partially high false-negative rate, which <br /> works obtained different experimental systems, affinity             places hurdle algorithms rely neighbour informa- <br /> purification, yeast two-hybrid (Y2H) protein-fragment complemen-             tion, example, gene function prediction (Chua et al., 2006; <br /> tation assay (PCA), evidence shows predicted edges              Sharan et al., 2007). Third, PPI networks known have <br /> likely bona fide physical interactions. Finally, application human       skewed degree distribution, meaning than <br /> PPI network increased coverage network 100%.              expected quantity hub genes. Such hub nodes reduce <br /> Availability: www.cs.utsa.edu/ jruan/RWS/.                                       performance existing graph theoretic algorithms (e.g. for <br /> Contact: Jianhua.Ruan@utsa.edu                                                   predicting protein complexes) designed net- <br /> Supplementary information: Supplementary data available                   works relatively uniform degree distributions. <br /> Bioinformatics online.                                                              In article, present novel method improve qual- <br />                                                                                  ity given PPI network computationally predicting some <br /> Received June 8, 2012; revised November 2, 2012; accepted               new interactions removing spurious edges. It worth noting <br /> November 26, 2012                                                                idea purely based topology network, <br />                                                                                  additional biological information involved. This ensures <br />                                                                                  algorithm easily combined algorithms <br /> 1 INTRODUCTION                                                                   developed predicting protein com- <br /> Recent advances high-throughput techniques, yeast                     plexes performing PPI-based studies. There several <br /> two-hybrid (Y2H) tandem affinity purification, enabled                  studies attempt combine additional biological informa- <br /> the production large protein&#226;&#8364;&#8220;protein interaction                  tion, gene ontology gene expression, PPI net- <br /> (PPI) data (Gavin et al., 2006; Krogan et al., 2006; Tarassov et                 work protein complex prediction (Asthana et al., 2004; <br /> al., 2008; Yu et al., 2008). These PPI data modelled                   Ulitsky Shamir, 2009; Wang et al., 2010). It be <br /> networks, nodes networks represent proteins edges                   straightforward use method approaches, repla- <br /> between nodes represent physical interactions pro-                   cing PPI network reconstructed PPI network. <br /> teins. These networks, high-throughput                          <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> assess performance algorithm, tested on <br />                                                                                  yeast PPI networks obtained different experimental <br /> *To correspondence addressed.                                     systems, examined biological relevance results <br />  <br />  <br /> &#195;&#376; The Author 2012. Published Oxford University Press. All rights reserved. For Permissions, e-mail: journals.permissions@oup.com         355 <br />  C.Lei J.Ruan <br />  <br />  <br />                                                                                                             Topological profiles node 6 node 8 <br /> using multiple information sources, including gene ontology an- <br /> notations, gene expression data, protein complexes, list essen- <br /> tial genes, conservation species large collection                                                     Compute <br />                                                                                                                        topological <br /> known physical interactions BioGRID database. Results                                                           profiles for <br /> show predicted PPIs higher functional rele-                                                         node <br /> vance removed ones, likely bona fide phys-                              Original network <br /> ical interactions. Comparison existing methods <br /> shows network reconstructed method                                                                      Compute topological <br />                                                                                                                                 similarities nodes <br /> highest overall quality. Furthermore, applying independent <br /> graph clustering algorithms, reconstructed                                                          Reconstruct <br />                                                                                                                       network by <br /> network resulted significantly improved prediction accur-                                                      linking nodes <br /> acy protein complexes. Finally, application human PPI                                                      high <br /> network shows encouraging results.                                                                                    similarities <br />                                                                                             Reconstructed network <br />                                                                                                                        Topological similarity <br /> 2     METHODS                                                                                                          node 6 node 8 <br />  <br /> 2.1    Rationale related works                                                Fig. 1. Basic idea method <br />  <br /> In recent years, computational methods developed to <br /> predict missing links networks (Fouss et al., 2007; Li Horvath,          created connect nodes topologically similar. The frame- <br /> 2007; Ruan Zhang, 2006; Radicchi et al., 2004; Tong et al., 2006),            work general, step multiple designing choices. Here, <br /> and reviewed (Lu&#194;&#168; Zhou, 2011), success-            propose novel random walk procedure computing topological <br /> fully applied PPI networks. These methods basically fall              profiles, handle hub nodes better existing random <br /> categories: common neighbour based distance based. The type             walk-based methods. We use simple ideas steps, leaving <br /> of methods based simple effective idea&#226;&#8364;&#8221;two nodes sharing              room future improvements. <br /> many common neighbours likely module (Li and <br /> Horvath, 2007; Ruan Zhang, 2006; Radicchi et al., 2004; Wang <br /> et al., 2007a). These methods limited value PPI networks              2.2    Random walk resistance <br /> that usually sparse. To circumvent problem, Fang et al. (2011)           Let G&#195;&#176;V, E&#195;&#382; undirected graph representing PPI network, V <br /> generalized idea consider neighbours greater path lengths           set nodes E set edges (for convenience, self-loop also <br /> showed so-called &#226;&#8364;&#732;global geometric affinity&#226;&#8364;&#8482; (GGA) measurement           added node). For v 2 V,        let N&#195;&#176;v&#195;&#382; &#194;&#188; fu 2 V jfv, ug 2 Eg set <br /> can help predict new PPI.                                                         neighbours v d&#195;&#176;v&#195;&#382; &#194;&#188;  N&#195;&#176;v&#195;&#382;  degree v. <br />    The second type methods measures &#226;&#8364;&#732;distance&#226;&#8364;&#8482; pairs                The simple random walk node graph G walk G <br /> nodes network taking consideration alternative paths;             node chosen uniformly random set of <br /> popular examples include algorithms based random walks,                    neighbours current node. Formally, random walker is <br /> namely, Euclidean commute time (ECT) (Fouss et al., 2007)                     node i, probability step neighbour <br /> random walk restart (RWR) (Tong et al., 2006). ECT measures                  j Pij &#194;&#188; 1=d&#195;&#176;i&#195;&#382; fi, jg 2 E 0 otherwise. Assume random walk <br /> the expected number steps needed random walker travel be-             initiated node v. Let q&#195;&#176;k&#195;&#382;v, probability random walker <br /> tween nodes distance them, RWR computes                sitting node discrete time point k. Then, time point k &#195;&#190; 1, the <br /> the probability random walker starting node reach         probability random walker taking path node node j <br /> node j. Performance type methods significantly affected         calculated as <br /> by hub nodes connected nodes network. Another <br />                                                                                                                  f&#195;&#176;k&#195;&#190;1&#195;&#382; <br />                                                                                                                   v, ij &#194;&#188; q&#195;&#176;k&#195;&#382; <br />                                                                                                                            v, Pij ,                          &#195;&#176;1&#195;&#382; <br /> method falling category Kuchaiev et al. (2009), <br /> where attempted embed PPI network low dimensional                probability random walker reach node j time point <br /> geometric space using multiple dimensional scaling (MDS), assign              k &#195;&#190; 1 calculated as <br /> edges pairs nodes short distances embedded space.                                                X &#195;&#176;k&#195;&#190;1&#195;&#382; <br /> While good performance observed, method work                                           qv,&#195;&#176;k&#195;&#190;1&#195;&#382; <br />                                                                                                                   j   &#194;&#188;  f v, ij :                    &#195;&#176;2&#195;&#382; <br />                                                                                                                             i <br /> well general easy accurate geometric <br /> embedding given PPI network.                                                   It important note that, mild conditions, this <br />    In work, propose novel method considered            simple random walk, probability reach node j, qv, j , converges to <br /> generalization simple common neighbour-based method, combin-               value regardless starting point v (Lova&#194;&#180;sz, 1993). <br /> ing ideas distance-based method. We hypothesize                 Therefore, stationary probability distribution obtained the <br /> nodes having similar &#226;&#8364;&#732;distances&#226;&#8364;&#8482; (in view random walker)          simple random walk used directly measure distance between <br /> other nodes network potentially interact other. This         nodes. To provide measurement distance nodes a <br /> idea general aforementioned types methods be-             network, modifications proposed. For example, ECT <br /> cause predict links covered aforementioned        measures expected number hops needed travel two <br /> two types methods predict links nodes            nodes simple random walk (Fouss et al., 2007), RWR <br /> themselves far away (for random walker)              attempts bias random walk stay close starting node by <br /> share common neighbours. The basic idea method consists             teleporting random walker starting node certain prob- <br /> three steps illustrated Figure 1. First, topological profile,   ability (Tong et al., 2006). <br /> measures &#226;&#8364;&#732;distances&#226;&#8364;&#8482; target node nodes            Here, random walk algorithm, named random walk with <br /> network, calculated node. Second, similarities topological         resistance (RWS), based key ideas: (i) bias random walker <br /> profiles calculated pair nodes. In final step, edges     staying close starting point adding small of <br />  <br />  <br />  <br /> 356 <br />                                                                                                                Algorithm reconstructing PPI networks <br />  <br />  <br />  <br /> resistance edge network, (ii) introduce condition           using columns offset matrix topological profiles, is, letting <br /> to discourage random walker roaming new territories               Cij &#194;&#188;pcc( 1 jVj, ,  1 jVj, j ), works slightly better rows. Informally <br /> hub nodes, adding additional resistance random walker over-             speaking, row vector represents information passed node <br /> come meeting new node visited before. The                 nodes network, column vector represents in- <br /> algorithm best described modification simple random              formation node receives network; therefore, a <br /> walk algorithm. Basically, replace Equation (1)                               accurate way describing position node network. <br />                 8 <br />                 &gt;         &#195;&#176;k&#195;&#382;                &#195;&#176;k&#195;&#382; <br />                 &lt; max&#195;&#176;0, qv, Pij    &#195;&#382;, qv, j 40; <br />        f&#195;&#176;k&#195;&#190;1&#195;&#382; &#194;&#188;           &#195;&#176;k&#195;&#382;                &#195;&#176;k&#195;&#382;             &#195;&#176;k&#195;&#382;             &#195;&#176;3&#195;&#382;    2.4    Reconstructing PPI network <br />         v, ij <br />                 &gt; max&#195;&#176;0, qv, Pij    &#195;&#382;, qv, j &#194;&#188; 0 maxt &#195;&#176;qv, t Ptj &#195;&#382;    ; <br />                 : <br />                   0, otherwise:                                                     Finally, network reconstructed topological similarity matrix <br />                                                                                     connecting pairs nodes similarity certain thresh- <br />    The probability reaching node j time point k &#195;&#190; 1 calcu-            old. Although sophisticated methods possible (e.g. Ruan, 2009), <br /> lated adding probabilities enter j paths,                 article, choose implement simple strategy easy evalu- <br /> re-normalized probability vector sums 1:                             ation fair comparison methods: simply pick cut-off <br />                         &#195;&#176;k&#195;&#190;1&#195;&#382; <br />                                 X &#195;&#176;k&#195;&#190;1&#195;&#382;   X &#195;&#176;k&#195;&#190;1&#195;&#382;                                   value number edges kept the original <br />                         v, j  &#194;&#188;   fv, ij      fv,                 &#195;&#176;4&#195;&#382; <br />                                                                                network. We simple strategy worked well. In Section 4, <br />                                                                                     discuss future plans improving cut-off selection, should <br />    As Equation (3), algorithm checks destination             improve quality reconstructed network, particu- <br /> node, j, new random walker. If not, is, qv,&#195;&#176;k&#195;&#382;j 40, random       larly, reduce false-negative rate PPI networks. <br /> walker visit node probability qv,&#195;&#176;k&#195;&#382;i Pij    . The probability will <br /> be set 0 50. Combined normalization step, purpose <br /> of introducing   parameter similar teleporting idea RWR <br />                                                                                     3     RESULTS AND DISCUSSION <br /> that biases random walker stay close starting node, that <br /> the stationary probability vectors different starting nodes             For evaluation, applied algorithm yeast PPI <br /> always converged values.                                                networks obtained different technologies: tandem-affinity <br />    If node j new random walker, additional param-             purification (Krogan et al., 2006), Y2H (Yu et al., 2008) and <br /> eter,  , introduced discourage random walker visiting            protein-fragment complementation assay (Tarassov et al., <br /> new node, unless path v enter j sufficiently      2008). In Sections 3.1&#226;&#8364;&#8220;3.3, discuss results Krogan <br /> large probability. The main motivation step reduce               dataset, largest, Sections 3.4 3.5, we <br /> impact hub nodes random walk process. Imagining              present comparative analysis datasets. In <br /> the   parameter, random walker reaching hub node able to <br />                                                                                     Section 3.6, present application human PPI network. <br /> travel, probability, nodes connected <br /> with hub nodes, functionally related the <br /> starting node. In contrast, algorithm, given scenario,            3.1    Reconstructed PPI network better functional <br /> the random walker selectively prefer nodes                     relevance <br /> connected hub nodes connected starting node <br /> via alternative paths involving hub nodes. (See <br />                                                                                     We performed random walk Krogan PPI network, <br /> Supplementary Fig. S1 toy example).                                           covers 2708 genes 7123 edges, derived modified <br />    In experiment,   set jVj=jEj2   set 1=jEj. This choice       PPI network choosing 7123 potential connections the <br /> is based analysis minimum average value fv, ij          highest similarities (see Section 2). Within modified network, <br /> edge. Empirically, values perform              2870 (40%) edges new (and number edges the <br /> multiple, biological non-biological, networks. Variations               original network removed). To evaluate functional <br /> these values constant multiple significantly change             relevance newly predicted edges, resort types <br /> the results.                                                                        sources, including gene ontology, gene expression, essentiality, <br />    The aforementioned procedure applied iteratively starting            known protein complexes conservation interactions in <br /> node. A random walker considered reached stationary dis- <br />                                                                                     species. To facilitate discussion, group of <br /> tribution change probability arrive node less <br />                                                                                     edges present original network &#226;&#8364;&#732;before&#226;&#8364;&#8482; group, that <br /> than small cut-off value. In experiment, nodes converged 5&#226;&#8364;&#8220;20 <br /> iterations, took minutes personal computer.                   modified network &#226;&#8364;&#732;after&#226;&#8364;&#8482; group. Furthermore, &#226;&#8364;&#732;new&#226;&#8364;&#8482; edges <br />                                                                                     designate edges &#226;&#8364;&#732;after&#226;&#8364;&#8482; &#226;&#8364;&#732;before&#226;&#8364;&#8482; group, <br />                                                                                     &#226;&#8364;&#732;removed&#226;&#8364;&#8482; edges &#226;&#8364;&#732;before&#226;&#8364;&#8482; &#226;&#8364;&#732;after&#226;&#8364;&#8482;. Finally, present <br /> 2.3    Calculating topological similarity                                           &#226;&#8364;&#732;before&#226;&#8364;&#8482; &#226;&#8364;&#732;after&#226;&#8364;&#8482; called &#226;&#8364;&#732;confirmed&#226;&#8364;&#8482;. We gen- <br /> After applying aforementioned random walk procedure                      erated random networks randomly rewiring procedure <br /> network, jVj   jVj probability matrix, denoted                         preserves degree node (Ruan Zhang, 2008). <br />   &#194;&#188; h i, j ijVj jVj , i, j represents probability random               As interacting proteins likely involved similar biological <br /> walker started node reach node j convergence.                            processes, expected similar functional annota- <br />    To magnify difference probability vectors different             tions gene ontology similar gene expression patterns <br /> nodes, obtain median vector H vectors, the <br />                                                                                     diverse conditions. Therefore, measure functional <br /> j-th element H defined Hj &#194;&#188;median ( i&#194;&#188;1 jVj, j ). This median prob- <br />                                                                                     relevance pair genes connected an <br /> ability vector similar stationary probability vector simple <br /> random walk. We calculate jVj   jVj offset matrix  ,                 edge using semantic similarity GO terms anno- <br />  ij &#194;&#188; ij  Hj .                                                                      tated proteins, using popular method (Wang et al., <br />    Finally, calculate Pearson correlation coefficient           2007b; Yu et al., 2010). Results shown based &#226;&#8364;&#732;molecu- <br /> pair rows columns offset matrix measurement topo-              lar function&#226;&#8364;&#8482; branch Gene Ontology. Using &#226;&#8364;&#732;biological pro- <br /> logical similarity, Cij , nodes. Empirically,            cess&#226;&#8364;&#8482; yielded similar values, &#226;&#8364;&#732;cellular localization&#226;&#8364;&#8482; resulted in <br />  <br />  <br />                                                                                                                                                             357 <br />  C.Lei J.Ruan <br />  <br />  <br />  <br /> slightly lower consistent values (Supplementary Fig. S2).                                           (also Section 3.2). In contrast, measurement the <br /> We measured Pearson correlation coefficient                                            new edges close confirmed PPIs. <br /> the gene expression profiles pair genes, using                                             We looked conservation edges spe- <br /> yeast stress response microarray data (Gasch et al., 2000). We                                          cies. We downloaded conserved PPIs yeast four <br /> used average similarity pairs nodes connected                                       species including Caenorhabditis elegans, fly, mouse human <br /> edge certain group represent functional relevance                                        InteroLogFinder (http://www.interologfinder.org/) (Wiles <br /> that edge group. As shown Figure 2a, group                                           et al., 2010). As shown Figure 2c, considerable fraction of <br /> higher functional relevance group based                                         confirmed edges conserved species. <br /> GO gene expression. Moreover, confirmed group                                           Although small fraction removed edges conserved <br /> highest functional similarity compared groups,                                           species, rarely conserved two <br /> and removed group far lower new group. The                                              species. In comparison, new edges tend conserved <br /> standard error average measurements 510 5 ;                                            removed edges, confirmed <br /> therefore, differences highly significant. Further inves-                                     ones. The conservation analysis suggests predicted <br /> tigation showed GO-based similarity 40:95 36%                                           edges bona fide physical interactions functional <br /> and 32% edges confirmed new groups, respect-                                          links (see Section 3.5). <br /> ively. In contrast, 7% removed edges GO-based                                               Finally, shown genes high connectivity in <br /> similarity 40:95 (Fig. 2b).                                                                             PPI network tend essential, known <br />    Next, used essential genes compare different edge                                              connectivity essentiality (percentage genes are <br /> groups. The list essential genes yeast retrieved                                      essential) weakly correlated (Jeong et al., 2001). For <br /> Saccharomyces Genome Database (Dwight et al., 2004). As                                             example, Figure 1d shows essentiality of <br /> interacting proteins belong protein complex,                                            genes generally increasing genes low intermediate <br /> they tend essentiality. In words,                                         degrees, essentiality genes highest degrees rela- <br /> is (not) essential, expected (not) essential.                                   tively lower expected degrees. Besides pos- <br /> As shown Figure 2a, percentage removed edges                                         sible explanations, proteins the <br /> share essentiality actually lower                                          highest degrees &#226;&#8364;&#732;sticky&#226;&#8364;&#8482; sense to <br /> randomly generated edges, suggests removed                                               interact proteins experimental protocol, but <br /> edges probably connecting genes different complexes                                              interactions exist reality protein is <br />  <br />  <br />                 (a)                             0.7                                                     (b)                                 0.4 <br />                                                                           All (before)                                                             New <br />                                                 0.6                       All (after)                                                      0.35    Removed <br />                                                                           New                                                                      Confirmed <br />                                                                           Removed                                                           0.3 <br />                                                                                                          Fraction edges <br />  <br />  <br />  <br />  <br />                                                 0.5 <br />                              Similarity score <br />  <br />  <br />  <br />  <br />                                                                           Confirmed <br />                                                                           Random                                                           0.25 <br />                                                 0.4 <br />                                                                                                                                             0.2 <br />                                                 0.3 <br />                                                                                                                                            0.15 <br />                                                 0.2 <br />                                                                                                                                             0.1 <br />  <br />                                                 0.1                                                                                        0.05 <br />  <br />                                                  0                                                                                           0 <br />                                                       Gene Ontology   Expression         Essentiality                                         0     0.2            0.4      0.6        0.8            1 <br />                                                                                                                                                       GO&#226;&#710;&#8217;based similarity score <br />  <br />                 (c) 0.09                                                                                (d)                                  1 <br />                                                                                            New                                                                                               After <br />                                        0.08                                                Removed                                         0.9                                               Before <br />                                                                                                              Fraction essential genes <br />  <br />  <br />  <br />  <br />                                                                                            Confirmed <br />                                        0.07                                                                                                0.8 <br />                   Fraction edges <br />  <br />  <br />  <br />  <br />                                        0.06 <br />                                                                                                                                            0.7 <br />                                        0.05 <br />                                                                                                                                            0.6 <br />                                        0.04 <br />                                                                                                                                            0.5 <br />                                        0.03 <br />                                                                                                                                            0.4 <br />                                        0.02 <br />                                                                                                                                            0.3 <br />                                        0.01 <br />  <br />                                                  0                                                                                         0.2 0               1                   2                  3 <br />                                                              1        2          3         4                                                 10           10                      10              10 <br />                                                              Number species conserved                                                                           Connectivity <br />  <br /> Fig. 2. Results method Krogan PPI network. (a) Different edge groups evaluated GO-based similarity, gene expression correlation <br /> and co-essentiality. (b) Distribution GO-based similarity scores edge groups. (c) Fraction conserved edges edge groups. <br /> (d) Relationship connectivity essentiality <br />  <br />  <br />  <br /> 358 <br />                                                                                                                               Algorithm reconstructing PPI networks <br />  <br />  <br />  <br /> poorly expressed co-expressed proteins                (a) 0.7 <br />                                                                                                                                                            Before&#226;&#710;&#8217;MCL <br /> potentially interact with. It seen reconstructed <br />  <br />  <br />  <br />  <br />                                                                                 Quality predicted complexes <br />                                                                                                                  0.6                                       After&#226;&#710;&#8217;MCL <br /> network, essentiality degree better correlation                                                                                            Before&#226;&#710;&#8217;HQcut <br />                                                                                                                                                            After&#226;&#710;&#8217;HQcut <br /> (Fig. 2d).                                                                                                       0.5 <br />    In summary, using multiple independent sources evidence, <br />                                                                                                                  0.4 <br /> we shown new edges clearly higher functional <br /> relevance removed ones. These results suggest                                                  0.3 <br /> algorithm reduce noise PPI network im- <br />                                                                                                                  0.2 <br /> prove network quality. <br />                                                                                                                  0.1 <br /> 3.2   Reconstructed PPI network improves accuracy of <br />                                                                                                                   0 <br />       protein complex prediction                                                                                       Known complex     Gene Ontology    Co&#226;&#710;&#8217;expression <br />  <br /> We investigated improved PPI network also <br />                                                                                                                  30 <br /> improve prediction accuracy protein complexes. We                       (b)                                                              All (before) <br /> applied network clustering algorithms original                                                                                   All (after) <br />                                                                                                                  25                             New <br /> modified PPI networks, compared predicted complexes <br />  <br />  <br />  <br />  <br />                                                                                   Perentage edges <br />                                                                                                                                                 Removed <br /> with MIPS known protein complexes (Mewes et al., 2006),                                                      20                             Confirmed <br />                                                                                                                                                 Random <br /> which included 767 proteins 170 known complexes inter- <br /> secting PPI network. Markov Clustering Algorithm                                                        15 <br /> (MCL) well-known graph clustering algorithm has <br /> been shown outperform protein complex prediction al-                                                    10 <br />  <br /> gorithms independent evaluation studies (Brohee van <br />                                                                                                                   5 <br /> Helden, 2006; Vlasblom Wodak, 2009). HQcut commu- <br /> nity discovery algorithm developed co-authors,                                                      0 <br /> based optimization so-called modularity function                                                                In&#226;&#710;&#8217;complex             Between&#226;&#710;&#8217;complex <br />  <br /> (Ruan Zhang, 2008). For MCL, set inflation param-          Fig. 3. Evaluation method based protein complexes. In (a), the <br /> eter 1.8 suggested (Brohee van Helden, 2006).     original PPI network (before) reconstructed counterpart (after) is <br /> HQcut does require user-tuned parameters. To measure          partitioned using algorithms (MCL HQcut) protein complex <br /> the accuracy prediction, used Fowlkes&#226;&#8364;&#8220;Mallows           predictions. Each prediction evaluated similarity (F&#226;&#8364;&#8220;M <br /> index comparing clustering (Fowlkes Mallows, 1983;            index, main text) MIPS known complexes, average gene <br /> Meila, 2005). Formally, let A list gene pairs fall     ontology similarity pairs nodes complex, by <br />                                                                       average gene co-expression correlation pairs nodes same <br /> into complex set predicted complexes B <br />                                                                       complex. In (b), different edge groups evaluated fraction of <br /> that set known     complexes, prediction accuracy is <br />                         p&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;                           edges connecting edges different MIPS known protein <br /> measured jA \ Bj= jAj   jBj, jAj denotes cardin-         complexes <br /> ality set A. As shown Figure 3a, prediction accuracy <br /> is significantly improved MCL HQcut, demonstrat- <br /> ing improvement general. Moreover, MIPS            complex improved network, connectivity is <br /> database known protein complexes covers 530%               increased complex, between-complex edges <br /> the proteins PPI network, measured average pair-        removed. Interestingly, anaphase-promoting complex <br /> wise functional similarity using gene ontology semantic similarity    (APC) complex, algorithm removed external <br /> and co-expression (see Section 3.1) pair nodes       edges added in-complex edges predicted inter- <br /> that predicted complex. Again, shown      actions non-member protein, MND2 complex <br /> that results improved significantly modified net-      members. It turns MND2 member the <br /> work MCL HQcut (Fig. 3a).                                APC complex (Hall et al., 2003). <br />    To investigate reconstructed network can <br /> result better prediction accuracy protein complexes, dir- <br /> ectly compared different edge groups fraction edges        3.3   Comparisons previous methods <br /> that connecting genes known complex                   We compared algorithm existing methods, namely, <br /> (in-complex) versus different known complexes       ECT (Fouss et al., 2007), RWR (Tong et al., 2006), MDS <br /> (between-complex). Indeed, shown Figure 3b, new edges       (Kuchaiev et al., 2009) GGA (Fang et al., 2011). The ECT <br /> have higher in-complex probability lower between-            RWR methods known data mining network <br /> complex probability compared removed edges,          analysis communities, MDS GGA methods <br /> the confirmed edges highest in-complex probability       recently proposed improve quality PPI networks <br /> lowest between-complex probability. Therefore, likely      (see Section 2.1). As algorithms calculate some <br /> the reconstructed PPI network combined exist-         topology-based similarity scores pairs nodes, then <br /> ing protein complex prediction algorithm improves ac-         use different (ad hoc) approaches determine cut-offs, a <br /> curacy. Figure 4 shows changes PPI network relevant        fair comparison, here, simply took 7123 pairs of <br /> for known protein complexes. The prediction Arp2/3        genes having highest scores predicted PPIs. Table 1 <br />  <br />  <br />                                                                                                                                                                           359 <br />  C.Lei J.Ruan <br />  <br />  <br />  <br /> shows change network introduced method. We                                                                                                      Although affinity purification (AP) technique used in <br /> then compared functional relevance reconstructed PPI                                                                                                    Krogan et al. (2006) designed capture co-complex member- <br /> networks. As shown Figure 5, PPI network reconstructed                                                                                                      ships, Y2H PCA directly detect binary interactions were <br /> by method highest GO similarity, highest fraction                                                                                                   shown higher false-negative rate lower false-positive <br /> in-complex edges lowest fraction between-complex edges.                                                                                                     rate AP (Yu et al., 2008; Tarassov et al., 2008). <br /> In fact, algorithm method shows consistent                                                                                                       The original Yu Tarassov networks cover 1278 1124 <br /> improvement original network using criteria.                                                                                                     genes 1641 2513 edges, respectively, excluding <br /> For example, RWR improved GO similarity scores frac-                                                                                                       self-interacting edges. With method, able replace <br /> tion in-complex edges, time reduced                                                                                                         (predict remove) 706 (43.0%) 1203 (47.9%) edges <br /> co-expression scores increased between-complex edges.                                                                                                          networks, respectively. As shown Figure 6, the <br /> GGA resulted lower co-expression scores. The network recon-                                                                                                     evaluation results based Gene Ontology datasets <br /> structed ECT slightly higher co-expression net-                                                                                                    evaluation results based co-expression Yu data <br /> work reconstructed method, GO similarity score                                                                                                   set consistent Krogan dataset, confirming <br /> much lower original network, decreased                                                                                                       general applicability method PPI networks regard- <br /> fraction in-complex edges increased fraction                                                                                                          experimental systems used infer them. On other <br /> between-complex edges. MDS resulted degraded functional                                                                                                         hand, differences predicted removed edges <br /> relevance scores according measurements                                                                                                              smaller Yu/Tarassov data Krogan data. In fact, for <br /> between-complex edges.                                                                                                                                             Tarassov data, removed edges slightly better <br />                                                                                                                                                                    co-expression predicted ones; nevertheless, are <br />                                                                                                                                                                    lower confirmed edges significantly higher than <br /> 3.4          Applicability types PPI networks                                                                                                          randomly predicted ones. These deviations the <br /> We applied method datasets, obtained                                                                                                      Krogan dataset likely reflect lower false-positive rate the <br /> Yu et al. (2008) Tarassov et al. (2008), using Y2H                                                                                                         Y2H PCA data compared AP data (Tarassov et al., <br /> protein-fragment complementation assay (PCA), respectively.                                                                                                        2008, Yu et al., 2008). As results consistently showed the <br />                                                                                                                                                                    removed edges higher functional relevance than <br />   (a)                          YER064C                                        ARC40 <br />                                                                                           PAC1 <br />  <br />                                                                                                                                              ARC40 <br />                                                                                                                                                                    random predictions, chances removed <br />                                                                                                                                                                    edges really false positives&#226;&#8364;&#8221;they just tend contain <br />                                                                                                   ARC18              ARC35 <br />                                           ARC18          ARC35 <br />                                                                                                                                                                    false-positive edges confirmed group edges. <br />                                                                 ARP2 <br />                                                                               IFA38                                                                                To reiterate, chose original number edges in <br />                                                                                                ARC19                            ARP2 <br />                                      ARC19 <br />                                                                                                                                                                    predicted networks facilitate unbiased comparison of <br />                         ASR1 <br />                                           ARC15           ARP3 <br />                                                                               RAD23 <br />                                                                                                        ARC15             ARP3 <br />                                                                                                                                                                    different approaches, changing parameters sig- <br />                                    MSK1                                       SMC1                                                                                 nificantly alter bias evaluation outcomes. In practice, <br />                                              CRN1 MSS51 RAD1           TAP42 <br />                                                                                                    MSK1 <br />                                                                                                                                              TAP42                 high false-negative rate PPI networks, especially <br />                                                                                                                                                                    Y2H PCA based networks, prefer choose a <br />   (b)                                                                                                                                                              lower similarity threshold make predictions remove <br />       FMP52 <br />                         Mlc1 <br />                                                                        BSP1 <br />                                                                                        FMP52 <br />                                                                                                                                                                    fewer edges here. As mentioned Section 2 <br />       OAF3                                MND2 <br />                                                         APC1 <br />                                                                                 SML1                              MND2 <br />                                                                                                                                  APC1 <br />                                                                                                                                                             SML1 <br />                                                                                                                                                                    later Section 4, aware developing better <br />                                                                                                                                                                    ways select cut-offs improve coverage PPI networks, <br />                                                                                        OAF3 <br />  <br />                  mrp7            APC5                                                                  APC5 <br />                                                                 apc4 <br />  <br />                                                                                                                                                                    presented elsewhere. <br />                                                                                                                                           apc4 <br />                                                                               APC9                                                                   APC9 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       Similarly, Krogan data, predicted edges Yu/ <br />                                                  Doc1 <br />                           APC2                                                                                           Doc1 <br />                                                                                                    APC2 <br />  <br />  <br />  <br />  <br />                                                                 CDC26 <br />                                                                                                                                           CDC26                    Tarassov data higher co-essentiality removed <br />                                  cdc27                                                                    cdc27 <br />  <br />  <br />    YRO2 <br />                                              CDC16      CDC23 <br />                                                                                                                    CDC16          CDC23 <br />                                                                                                                                                                    edges, suggesting predicted ones likely be <br />                                                                                         ssp2 <br />           ssp2                                                                                                                                                     in-complex removed ones. On hand, although <br /> Fig. 4. Subnetworks (a) Arp2/3 protein complex (b) APC protein <br />                                                                                                                                                                    confirmed edges Krogan data high co-essentiality, <br /> complex. The subnetworks contain interactions known members                                                                                                  Yu/Tarassov data lower co-essentiality, <br /> (dark) MIPS protein complexes direct neighbours (light)                                                                                            indicating large portion between-complex edges in <br /> the original Krogan PPI network (left) reconstructed network                                                                                           datasets preserved algorithm. As shown <br /> our method (right)                                                                                                                                                 Yu et al. (2008), Y2H PCA usually detect more <br />  <br />  <br />  <br /> Table 1. Changes network statistics different algorithms <br />  <br />  <br /> Network property                                                                         RWS                                                      RWR                        ECT                   MDS                   GGA <br />  <br /> Number nodes/edges                                                             2708/7123                                                2708/7123                  2708/7123             2708/7123             2708/7123 <br /> Number nodes/edges                                                              2549/7123                                                2708/7123                  2016/7123             2241/7123             2173/7123 <br /> Number replaced edges                                                                 2870 (40.3%)                                             2795 (39.2%)               5671 (79.6%)          5468 (76.8%)          4712 (66.2%) <br />  <br />  <br />  <br />  <br /> 360 <br />                                                                                                                              Algorithm reconstructing PPI networks <br />  <br />  <br />                0.7 <br />                                                                                            Original   functional interactions. To answer question, validated <br />                0.6 <br />                                                                                            RWS        edges predicted algorithm yeast datasets <br />                                                                                            RWR <br />                                                                                            ECT <br />                                                                                                       using physical interactions present BioGRID database <br />                0.5                                                                         MDS        (Stark et al., 2011). It known different PPI assay tech- <br />                                                                                            GGA        niques different characteristics produce complemen- <br />                0.4                                                                                    tary, largely disjoint, results (Tarassov et al., 2008; Yu et al., <br />        Score <br />  <br />  <br />  <br />  <br />                                                                                                       2008). We, therefore, classified PPIs BioGRID four <br />                0.3 <br />                                                                                                       categories according experimental systems: AP, Y2H), <br />                0.2 <br />                                                                                                       PCA other. Note PPI BioGRID have <br />                                                                                                       captured multiple experiments and, therefore, appear <br />                0.1                                                                                    multiple categories. The datasets used, Krogan, Yu <br />                                                                                                       Tarassov, generated systems, respect- <br />                     0 <br />                                                 GO          Expr         In&#226;&#710;&#8217;C     Between&#226;&#710;&#8217;C <br />                                                                                                       ively, present BioGRID database. To ensure <br />                                                                                                       fair comparison, evaluating results particular dataset <br /> Fig. 5. Comparison algorithms. GO: gene ontology-based                                     (e.g. Krogan), BioGRID entries corresponding publi- <br /> similarities; Expr: gene co-expression (Pearson correlation coefficient); <br />                                                                                                       cation removed (as validation rate the <br /> In-C: fraction edges in-complex; Between-C: fraction of <br /> edges between-complex <br />                                                                                                       confirmed removed group Krogan dataset be <br />                                                                                                       100%). <br />                                                                                                          Table 2 shows number fraction edges different <br />                (a) 0.8 <br />                                                                    All (before) <br />                                                                                                       groups validated BioGRID, using physical interactions <br />                                       0.7                          All (after)                        (column &#226;&#8364;&#732;All&#226;&#8364;&#8482;) using edges detected specific experimental <br />                                                                    New <br />                                                                    Removed <br />                                                                                                       systems (columns &#226;&#8364;&#732;AP&#226;&#8364;&#8482;, &#226;&#8364;&#732;Y2H&#226;&#8364;&#8482;, &#226;&#8364;&#732;PCA&#226;&#8364;&#8482; &#226;&#8364;&#732;other&#226;&#8364;&#8482;). When all <br />                                       0.6 <br />                                                                    Confirmed                          physical interactions considered, datasets, the <br />                                                                    Random <br />                    Similarity score <br />  <br />  <br />  <br />  <br />                                       0.5                                                             confirmed edges highest validation rate com- <br />                                                                                                       pared edge groups. The validation rates pre- <br />                                       0.4 <br />                                                                                                       dicted/removed edges lower confirmed ones <br />                                       0.3                                                             significantly higher random ones (51% all <br />                                       0.2 <br />                                                                                                       cases). For Krogan data, predicted edges much <br />                                                                                                       higher validation rate removed ones, for <br />                                       0.1                                                             Tarassov Yu data, validation rates predicted <br />                                         0                                                             edges similar lower removed ones. <br />                                             Gene Ontology      Expression         Essentiality <br />                                                                                                       These results suggest predicted edges data- <br />                                                                                                       sets likely bona fide physical interactions, reconfirm <br />                (b) 0.8                                                                                removed ones necessarily false-positive results, <br />                                                                    All (before)                       especially Y2H PCA-based data, known to <br />                                       0.7                          All (after) <br />                                                                    New                                lower false-positive rate AP (Tarassov et al., 2008; <br />                                       0.6                          Removed                            Yu et al., 2008). Therefore, preferred lower <br />                                                                    Confirmed <br />                                                                    Random                             cut-off increase coverage PPI networks, discussed in <br />                 Similarity score <br />  <br />  <br />  <br />  <br />                                       0.5 <br />                                                                                                       Section 3.4. Overall, Krogan data highest percentage of <br />                                       0.4                                                             validated predictions (36.0% 1033/2870), compared Yu <br />                                       0.3 <br />                                                                                                       Tarassov data, 9.2% (65/706) 11.2% (135/ <br />                                                                                                       1203) predicted edges validated BioGRID, respectively (but <br />                                       0.2                                                             later text). As BioGRID PPI data a <br />                                       0.1                                                             high false-negative rate, real validation rate likely <br />                                                                                                       underestimated. <br />                                        0 <br />                                             Gene Ontology     Expression          Essentiality           We validated results using BioGRID PPIs within <br />                                                                                                       specific category. Interestingly, different <br /> Fig. 6. Results algorithm additional yeast PPI datasets <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> (a) Yu et al., 2008 (b) Tarassov et al. 2008, evaluated functional                             characteristics different experimental systems carried over <br /> relevance scores based gene ontology, gene expression gene                                     predicted edges. For example, predicted edges for <br /> essentiality                                                                                          Krogan data validated AP-based interactions <br />                                                                                                       BioGRID. Moreover, Y2H covers 12.6% of <br /> between-complex edges AP. Our algorithm nicely preserved                                         edges BioGRID, accounts 53.9% (35/65) validated <br /> this property.                                                                                        predictions Yu data. Similarly, PCA contributes 7.1% of <br />                                                                                                       edges BioGRID, accounts 17.0% (23/135) the <br />                                                                                                       validated predictions Tarassov data. Therefore, relatively <br /> 3.5   Predicted edges bona fide physical interactions                                             low validation rate Yu Tarassov data partially <br /> Finally, interesting ask edges predicted                                  explained insufficient presence Y2H PCA data in <br /> algorithm bona fide physical interactions simply                                               BioGRID database. <br />  <br />  <br />                                                                                                                                                                     361 <br />  C.Lei J.Ruan <br />  <br />  <br />  <br /> Table 2. Validation BioGRID, breaking according experimental systemsa <br />  <br />  <br />                                                           BioGRID physical interactions <br />  <br /> Edge group               Number edges                  All                 AP                                     Y2H               PCA                     Other <br />  <br /> BioGRID                  73 929                           73 929 (1.00)       52 842 (0.71)                          9303 (0.12)       5237 (0.07)             13 080 (0.18) <br /> Krogan <br />   Predicted                2870                            1033   (0.36)         983   (0.34)                         134   (0.05)       35   (0.01)             180   (0.06) <br />   Removed                  2870                             412   (0.14)         393   (0.14)                          48   (0.02)       16   (0.01)              67   (0.02) <br />   Confirmed                4253                            2732   (0.64)        2682   (0.63)                         595   (0.14)      165   (0.04)             770   (0.18) <br />   Random                   7123                              54   (0.01)          33   (0.00)                          13   (0.00)        1   (0.00)              13   (0.00) <br /> Yu <br />   Predicted                 706                              65   (0.09)          52   (0.07)                          35   (0.05)        4   (0.01)              20   (0.03) <br />   Removed                   706                             114   (0.16)          53   (0.07)                          85   (0.12)        3   (0.00)              19   (0.03) <br />   Confirmed                 935                             405   (0.43)         304   (0.33)                         290   (0.31)       35   (0.04)             169   (0.18) <br />   Random                   1641                               7   (0.00)           3   (0.00)                           1   (0.00)        0   (0.00)               3   (0.00) <br /> Tarassov <br />   Predicted                1203                             135   (0.11)          96   (0.08)                          39   (0.03)       23   (0.02)              42   (0.03) <br />   Removed                  1203                             109   (0.09)          81   (0.07)                          22   (0.02)       15   (0.01)              26   (0.02) <br />   Confirmed                1333                             352   (0.26)         301   (0.23)                         143   (0.11)       34   (0.03)             121   (0.09) <br />   Random                   2536                              24   (0.01)          18   (0.01)                           2   (0.00)        1   (0.00)               6   (0.00) <br />  <br /> a <br /> Values parentheses fractions edges validated. <br />  <br />  <br />  <br />  <br />                                                                                                               0.45 <br /> 3.6   Application human PPI network                                                                                                                 All (before) <br /> Finally, apply method predict novel interactions                                              0.4                                     All (after) <br />                                                                                                                                                        New <br /> human PPI network downloaded Human Protein                                                           0.35                                     Removed <br /> Reference Database (HPRD, version 9) (Keshava Prasad et al.,                                                                                           Confirmed <br />                                                                                            Similarity score <br />  <br />  <br />  <br />  <br />                                                                                                                0.3                                     Random <br /> 2009). The largest connected component network contains                                                                                        Top New <br /> 9205 nodes 36 720 edges. We tried use idea                                                0.25                                     Top Removed <br /> retain number edges, causes 24 187 (65.9%)                                                   0.2 <br /> edges replaced. To evaluate result, calculated gene <br />                                                                                                               0.15 <br /> ontology-based similarity gene co-expression different <br /> edge groups, following logic Section 3.1. The                                                   0.1 <br /> gene expression data used purpose downloaded from <br />                                                                                                               0.05 <br /> M2 DB contains 878 normal (non-diseased) tissue samples <br /> from Affymetrix Human U133A platform, following                                                         0 <br />                                                                                                                        Gene Ontology              Expression <br /> parameters suggested developers (Cheng et al., 2010). <br /> Figure 7 shows validation results. Similar yeast                    Fig. 7. Results algorithm human PPI network, evaluated by <br />                                                                                    functional relevance scores based gene ontology, gene expression. See <br /> data set, reconstructed network improved scores using <br />                                                                                    main text definitions new removed <br /> both criteria. Although confirmed edges highest <br /> similarities scores, predicted ones better scores than <br /> the removed ones, significantly better                           low GO-based similarity. In contrast, average <br /> random predictions.                                                                GO-based similarity score high-confidence false-negative <br />    On hand, high replacement rate                         edges high confirmed edges, have <br /> (65.9%), non-trivial functional gene co-expression                     high co-expression. These results indicate topological <br /> similarity scores, suspect removed edges                   similarity scores used prioritize edges valid- <br /> probably true interactions. Therefore, investigated                 ation. In fact, shown Figure 8, 35 000 <br /> distribution topological similarity scores PPIs              (50 000) predicted edges, average gene co-expression (GO simi- <br /> original network, clearly follow bimodal distribution                      larity) interacting genes good original PPI <br /> (Supplementary Fig. S3). Therefore, used topological similar-                   network, indicating increase coverage the <br /> ity score 50.2 select 4309 edges treated high-confidence               HPRD human PPI network 100%. <br /> false-positive edges (top removed), topological similarity <br /> score 40.9 select 3802 edges high-confidence false-negative <br /> edges (top new) evaluated using gene ontology gene                    4      CONCLUSIONS <br /> co-expression. As shown Figure 7, high-confidence false-                    In article, presented novel network topology-based <br /> positive edges close-to-random gene co-expression (0.01),                     algorithm improve quality PPI networks, turn <br />  <br />  <br /> 362 <br />                                                                                                                                  Algorithm reconstructing PPI networks <br />  <br />  <br />  <br />         (a)                             0.52                                                        reconstruction/clustering algorithms. In fact, found <br />                                                                      org PPI edges <br />                                          0.5                         predicted edges            edges similarity slightly cut-off also <br />                 Average GO similarity   0.48                                                        higher biological relevance removed, shown in <br />                                         0.46 <br />                                                                                                     human data. This biologically understandable&#226;&#8364;&#8221;the <br />                                                                                                     original PPI network higher false-negative rate than <br />                                         0.44 <br />                                                                                                     false-positive rate (Huang et al., 2007; Kuchaiev et al., 2009). <br />                                         0.42 <br />                                                                                                     In future work, worthwhile develop methods that <br />                                          0.4                                                        guide selection appropriate cut-off would <br />                                         0.38                                                        allow functionally relevant edges included without <br />                                         0.36                                                        introducing false-positive edges. One possible way to <br />                                         0.34 <br />                                                                                                     examine distribution similarity scores original <br />                                              2            3              4                      5 <br />                                            10        10             10                     10       edges non-edges determine edge weights might <br />                                                       Number edges                               represent good separation. <br />                                                    Average GO similarity <br />         (b) <br />                                         0.24 <br />                                                                      org PPI edges                  ACKNOWLEDGEMENT <br />          Average gene co&#226;&#710;&#8217;expression <br />  <br />  <br />  <br />  <br />                                         0.22                         predicted edges <br />                                                                                                     The authors like thank Saleh Tamim assistance <br />                                          0.2 <br />                                                                                                     obtaining human microarray data. We thank an- <br />                                         0.18                                                        onymous reviewers insightful comments sig- <br />                                         0.16                                                        nificantly improved manuscript. <br />                                         0.14 <br />  <br />                                         0.12 <br />                                                                                                     FUNDING <br />                                          0.1 <br />                                                                                                     NIH      (SC3GM086305,      R01CA152063, U54CA113001, <br />                                         0.08 <br />                                              2         3                4                   5 <br />                                                                                                     G12MD007591 (Computational Systems Biology Core)), NSF <br />                                            10        10             10                     10 <br />                                                       Number edges                               (IIS-1218201, IOS-0848135). <br />                                                  Average gene co-expression                         Conflict Interest: declared. <br /> Fig. 8. Quality predicted human PPIs compared of <br /> the original PPIs. Predicted/original PPIs ranked topological <br /> similarity scores. All edges particular rank used                               REFERENCES <br /> calculate average GO similarity score (a) average gene co- <br />                                                                                                     Asthana,S. et al. (2004) Predicting protein complex membership using probabilistic <br /> expression (b)                                                                                          network reliability. Genome Res., 14, 1170&#226;&#8364;&#8220;1175. <br />                                                                                                     Bader,G. Hogue,C. (2002) Analyzing yeast protein-protein interaction data <br />                                                                                                         obtained different sources. Nat. Biotechnol., 20, 991&#226;&#8364;&#8220;997. <br /> can improve prediction accuracy protein complexes. The                                       Brohee,S. van Helden,J. (2006) Evaluation clustering algorithms for <br /> key idea algorithm proteins sharing                                             protein-protein interaction networks. BMC Bioinformatics, 7, 488. <br /> high-order topological similarities, measured                                        Cheng,W.-C. et al. (2010) Microarray meta-analysis database (m2db): uniformly <br />                                                                                                         pre-processed, quality controlled, manually curated human clinical micro- <br /> novel random walk-based procedure, likely interacting                                          array database. BMC Bioinformatics, 11, 421. <br /> each belong protein complex. Overall,                                     Chua,H.N. et al. (2006) Exploiting indirect neighbours topological weight to <br /> the reconstructed yeast PPI network higher biological                                         predict protein function protein-protein interactions. Bioinformatics, 22, <br /> relevance original network, better quality                                            1623&#226;&#8364;&#8220;1630. <br />                                                                                                     Chuang,H.Y. et al. (2007) Network-based classification breast cancer metastasis. <br /> those reconstructed existing algorithms, assessed by <br />                                                                                                         Mol. Syst. Biol., 3, 140&#226;&#8364;&#8220;140. <br /> multiple types information, including gene ontology, gene ex-                                    Dwight,S. et al. (2004) Saccharomyces genome database: underlying principles and <br /> pression, essentiality conservation species. The re-                                        organisation. Brief. Bioinform., 5, 9&#226;&#8364;&#8220;22. <br /> constructed network resulted significantly improved                                     Fang,Y. et al. (2011) Global geometric affinity revealing high fidelity protein <br /> protein complex prediction accuracy using different algo-                                           interaction network. PLoS ONE, 6, e19349. <br />                                                                                                     Fouss,F. et al. (2007) Random-walk computation similarities nodes a <br /> rithms. Furthermore, method applicable PPI networks                                           graph application collaborative recommendation. IEEE Trans. Knowl. <br /> obtained different experimental systems, Y2H, af-                                          Data Eng., 19, 355&#226;&#8364;&#8220;369. <br /> finity purification based protein-fragment complementation                                      Fowlkes,E. Mallows,C. (1983) A method comparing hierarchical clus- <br /> assay, evidence shows predicted edges likely                                           terings. J. Am. Stat. Assoc., 78, 553&#226;&#8364;&#8220;569. <br />                                                                                                     Friedel,C. et al. (2009) Bootstrapping interactome: unsupervised identification <br /> bona fide physical interactions. Finally, application a <br />                                                                                                         protein complexes yeast. J. Comput. Biol., 16, 1&#226;&#8364;&#8220;17. <br /> human PPI network increased coverage network                                          Gasch,A. et al. (2000) Genomic expression programs response yeast cells to <br /> at 100%.                                                                                          environmental changes. Mol. Biol. Cell, 11, 4241&#226;&#8364;&#8220;4257. <br />    Our method improved directions. For ex-                                        Gavin,A. et al. (2006) Proteome survey reveals modularity yeast cell machin- <br /> ample, derive network topology-based similarity                                           ery. Nature, 440, 631&#226;&#8364;&#8220;636. <br />                                                                                                     Hall,M.C. et al. (2003) Mnd2 swm1 core subunits saccharomyces <br /> matrix, used simple cut-off-based strategy maintain                                        cerevisiae anaphase-promoting complex. J. Biol. Chem., 278, 16698&#226;&#8364;&#8220;16705. <br /> the number edges original network. We                                           Hannum,G. et al. (2009) Genome-wide association data reveal global map of <br /> choice facilitate fair evaluation different network                                             genetic interactions protein complexes. PLoS Genet., 5, e1000782. <br />  <br />  <br />  <br />                                                                                                                                                                                   363 <br />  C.Lei J.Ruan <br />  <br />  <br />  <br /> Han,J.-D.J. et al. (2004) Evidence dynamically organized modularity yeast   Ruan,J. Zhang,W. (2006) Identification evaluation weak community <br />     protein-protein interaction network. Nature, 430, 88&#226;&#8364;&#8220;93.                               structures networks. In Proceedings National Conference Artificial <br /> Hidalgo,C. et al. (2009) A dynamic network approach study human                 Intelligence (AAAI-06). Boston, MA, AAAI press, pp. 470&#226;&#8364;&#8220;475. <br />     phenotypes. PLoS Comput. Biol., 5, e1000353.                                       Ruan,J. Zhang,W. (2008) Identifying network community structures a <br /> Huang,H. et al. (2007) Where interactions gone? Estimating cover-          <br /> </body></html>