<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Rpp1, an essential protein subunit of <br /> nuclear RNase P required for processing <br /> of precursor tRNA and 35S precursor <br /> rRNA in Saccharomyces cerevisiae <br />  <br />  <br />  <br /> The gene for an essential protein subunit of nuclear RNase P <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> Saccharomyces cerevisiae has been cloned. <br /> The gene for this protein, RPP1, was identified by virtue of its homology with a human scleroderma <br /> autoimmune antigen, Rpp30, which copurifies with human RNase P. Epitope-tagged Rpp1 can be found in <br /> association with both RNase P RNA and a related endoribonuclease, RNase MRP RNA, in immunoprecipitates <br /> from crude extracts of cells. Depletion of Rpp1 in vivo leads <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> the accumulation of precursor tRNAs with <br /> unprocessed 5* and 3* termini and reveals rRNA processing defects that have not been described previously for <br /> proteins associated with RNase P or RNase MRP. Immunoprecipitated complexes cleave both yeast precursor <br /> tRNAs and precursor rRNAs. <br /> [Key Words: Rpp1; essential protein subunit; nuclear RNase P; S. cerevisiae; precursor tRNA; precursor rRNA] <br /> Received June 5, 1997; revised version accepted July 25, 1997. <br />  <br />  <br />  <br />  <br /> Ribonuclease P (RNase P) is a ubiquitous endoribonucle-              limited. RNAse P isolated from human cells copurifies <br /> ase that consists of protein and RNA subunits. It cleaves            with an RNA (H1; 340), and at least six proteins&#226;&#8364;&#8221;Rpp14, <br /> 58-terminal leader sequences of precursor tRNAs (Darr et             Rpp20, Rpp25, Rpp30, Rpp38, and Rpp40 (Eder et al. <br /> al. 1992; Altman et al. 1993). Escherichia coli RNase P is           1997). Genetic approaches in Saccharomyces cerevisiae <br /> also known to process precursors of other small, meta-               have identified three essential proteins, Pop1, Pop3, and <br /> biologically stable RNAs in vivo, such as 4.5S RNA                   Pop4, which associate with RNase P RNA (RPR1); these <br /> (Bothwell et al. 1976), 10Sa RNA (Komine et al. 1994),               proteins also associate with the RNA (NME1) of a related <br /> the polycistronic mRNA from the histidine operon (Ali-               endoribonuclease, RNase mitochondrial RNA process- <br /> fano et al. 1994), and some small RNAs encoded by bac-               ing (MRP) (Schmitt and Clayton 1992; Lygerou et al. <br /> teriophage (Bothwell et al. 1974; Hartmann et al. 1995).             1994; Dichtl and Tollervey 1997; Chu et al. 1997). Analy- <br /> In eubacteria, the RNA component alone of RNase P is                 sis of temperature-sensitive alleles of these proteins or <br /> catalytic in vitro (Guerrier-Takada et al. 1983). The eu-            depletion of these proteins in yeast cells has shown that <br /> bacterial protein subunit is a basic protein of &#226;&#710;&#188;14 kD and           all three have a role in tRNA processing as well as rRNA <br /> serves as an essential cofactor in vivo by enhancing the             processing. It has been suggested that tRNA and rRNA <br /> catalytic efficiency and substrate range of the holoen-              processing are coordinated (Pace and Burgin 1990; Clay- <br /> zyme (Liu and Altman 1994; Gopalan et al. 1997). In                  ton 1994; Morrissey and Tollervey 1995; Lee et al. 1996). <br /> contrast, the RNA components of archaeal and eukary-                    In eukaryotes, coordination of tRNA and rRNA pro- <br /> otic RNase P are not catalytically active in vitro in the            cessing may be mediated by the activity of two related <br /> absence of their respective protein subunits, despite                enzymes, RNase P and RNase MRP (Morrissey and Tol- <br /> their structural homology to the eubacterial RNAs (Alt-              lervey 1995; Chamberlain et al. 1996b). RNase P is es- <br /> man et al. 1995; Haas et al. 1996).                                  sential for biosynthesis of tRNAs (Lee et al. 1991) and <br />    Although the RNA subunit of nuclear RNase P has                   also appears to have a role in rRNA processing in yeast <br /> been characterized from a variety of eukaryotic organ-               (Chamberlain et al. 1996b). RNase MRP is related to <br /> isms (Altman et al. 1993; Tranguch and Engelke 1993;                 RNase P by structural similarities found in its RNA <br /> Chamberlain et al. 1996a; Eder et al. 1996), information             component (Forster and Altman 1990; Schmitt et al. <br /> regarding the protein subunits of eukaryotic RNase P is              1993). It has been suggested that RNase P is an ancestor <br />                                                                      of RNase MRP (Morrissey and Tollervey 1995). RNase <br /> 3 <br />  Corresponding author.                                               MRP was described originally as an endonuclease that <br /> E-MAIL sidney.altman@qm.yale.edu; FAX (203) 432-5713.                cleaves RNA primers for mitochondrial DNA replication <br />  <br />  <br /> 2926      GENES &amp; DEVELOPMENT 11:2926&#226;&#8364;&#8220;2937 &#194;&#169; 1997 by Cold Spring Harbor Laboratory Press ISSN 0890-9369/97 $5.00 <br />                                                                               An essential protein subunit of RNase P from yeast <br />  <br />  <br />  <br /> (Chang and Clayton 1987; Stohl and Clayton 1992). Re-        amino acid sequence identity with human Rpp 30 (Fig. <br /> cently, its role in nuclear processing of precursor rRNA     1A). This yeast gene is now named RPP1 for RNase P <br /> (prRNA) has been established (Lygerou et al. 1996a).         Protein 1. <br /> RNase MRP does not cleave precursor tRNAs (ptRNAs)             To address whether the putative protein encoded by <br /> in vitro, and depletion of NME1 RNA does not affect          RPP1 is an essential gene, we disrupted this gene by re- <br /> tRNA processing in vivo (Schmitt and Clayton 1993; Ly-       placing it with the LEU2 gene (see Materials and Meth- <br /> gerou et al. 1996a). However, when proteins associated       ods). The heterozygous RPP1/rpp1::LEU2 strain (VS161) <br /> with RNase MRP are depleted or inactivated by a muta-        was sporulated and subsequent tetrad analysis showed a <br /> tion in yeast, cleavage of ptRNAs is blocked (Lygerou et     2:2 segregation for cell viability (Fig. 1B). All viable <br /> al. 1994; Chu et al. 1997; Dichtl and Tollervey 1997).       spores were Leu&#226;&#710;&#8217;, indicating that they had the wild-type <br />    To learn more about the functions of nuclear RNase P      RPP1 allele. Therefore, RPP1 is an essential gene in S. <br /> in vivo, we report here the cloning and functional char-     cerevisiae. <br /> acterization of an essential protein (Rpp1, 32.2 kD) com- <br /> ponent of S. cerevisiae RNase P. Rpp1 is homologous to <br />                                                              Construction of an epitope-tagged allele of RPP1 <br /> the human scleroderma autoimmune antigen, Rpp30, <br /> which was identified recently as a protein that copurifies   Epitope-tagged proteins are useful in the study of subunit <br /> with human RNase P and that is recognized by sera from       function and interactions in large holoenzyme com- <br /> patients with autoimmune disease that is referred to as      plexes. To determine whether or not Rpp1 associates <br /> Th/To antisera (Eder et al. 1997). To identify novel func-   with RPR1 RNA and RNase P activity, an epitope-tagged <br /> tions of RNase P in yeast, a strain of S. cerevisiae that    RPP1 strain of S. cerevisiae (VS162) was constructed <br /> conditionally expresses Rpp1 protein was constructed.        (Table 1). A DNA fragment that encodes three copies of <br /> Using this strain and a strain that contains an epitope-     a c-myc epitope (3 &#195;&#8212; myc; TerBush and Novick 1996) was <br /> tagged RPP1 gene, we demonstrate a role for Rpp1 in          fused in-frame 38 to the initiator ATG codon of RPP1 in <br /> processing tRNA and ribosomal RNA precursors. Deple-         a low-copy-number plasmid (pRS316), pRS316::3 &#195;&#8212; myc&#226;&#8364;&#8220; <br /> tion of Rpp1 protein revealed global defects in rRNA         RPP1. The resulting strain grew at identical rates to the <br /> processing. Depletion or inactivation of proteins associ-    wild-type cells suggesting that the 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 allele <br /> ated with RNase P or RNase MRP affect only a subset of       is fully functional (Fig. 2A, lanes 2,4). Immunoblots of <br /> the same rRNA processing events (Lygerou et al. 1994;        protein extracts from 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 cells using an anti- <br /> Chu et al. 1997; Dichtl and Tollervey 1997). Based on the    myc antibody (9E10) detected a polypeptide of 36 kD&#226;&#8364;&#8221; <br /> observed defects in rRNA processing, we suggest a pos-       the size is consistent with that predicted for the <br /> sible functional interaction of RNase P with RNase           3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fusion protein (Fig. 2B, lanes 2,4). Wild- <br /> MRP, and other small nucleolar ribonucleoprotein (RNP)       type haploids (VS162A and VS162C) lacking the c-myc <br /> complexes (snoRNP; for review, see Filipowicz and Kiss       tag do not contain the 36-kD protein. These results show <br /> 1993; Fournier and Maxwell 1993; Maxwell and Fournier        that the myc epitope-tagged Rpp1 protein migrates as a <br /> 1995), which is required for processing of prRNA.            36-kD polypeptide and is fully functional. <br />  <br />  <br />                                                              RNase P and RNase MRP RNAs coprecipitate with the <br /> Results <br />                                                              3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fusion protein <br /> An essential yeast gene encodes a homolog of the <br />                                                              Whether or not Rpp1 is associated with RNase P RNA <br /> human scleroderma autoimmune antigen, Rpp30 <br />                                                              was determined by immunoprecipitation experiments <br /> We used a combination of biochemical and genetic stud-       (Fig. 3). Extracts from 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 and RPP1 strains <br /> ies in human and S. cerevisiae cells to characterize a       were incubated with anti-myc monoclonal antibody <br /> protein subunit of eukaryotic RNase P. We employed a         (9E10) and RNA was extracted from the immunoprecipi- <br /> computational sequence search of yeast genes that have       tates (see Materials and Methods). The RNA was 38 end- <br /> amino acid sequence similarity to biochemically identi-      labeled with [32P]pCp and analyzed by denaturing gel <br /> fied human RNase P protein subunits (Eder et al. 1997).      electrophoresis (Fig. 3A,B). Immunoprecipitatd RNA was <br /> To determine which of the proteins associated with hu-       also analyzed by Northern hybridization to confirm the <br /> man RNase P might be essential components required           identity of the labeled RNAs (Fig. 3C,D). Mature RNase <br /> for catalytic function (Eder et al. 1997), we searched for   P RNA is the major RNA species found in the 3 &#195;&#8212; myc&#226;&#8364;&#8220; <br /> homologs of Rpp14, Rpp20, Rpp25, Rpp30, Rpp38, and           Rpp1 precipitate from 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 cells but not from <br /> Rpp40 in the genome of S. cerevisiae (Goffeau et al. 1996)   the control cell lysates. Two putative precursors of <br /> by performing a BLAST search (blastp and tblastn algo-       RNase P RNA (Lee et al. 1991) and RNase MRP RNA are <br /> rithms; Altschul et al. 1990) of the S. cerevisiae genome    also found in immunoprecipitates from 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 <br /> database (Cherry et al. 1996). The human scleroderma         but not in control immunoprecipitates. Approximately <br /> autoimmune antigen, Rpp30, has the highest amino acid        equal levels of RNase P and RNase MRP RNAs (RPR1 <br /> sequence similarity to a predicted sequence. A previ-        and NME1, respectively) were found in end-labeled RNA <br /> ously uncharacterized open reading frame (ORF),              derived from immunoprecipitates that were washed <br /> YHR062c, on the right arm of chromosome VIII has the         with buffer that contained 150 mM KCl (Fig. 3A). How- <br /> potential to encode a protein of 32.2 kD and shares 23%      ever, RNase P RNA is the major RNA species detected <br />  <br />  <br />                                                                                          GENES &amp; DEVELOPMENT              2927 <br />  Stolc and Altman <br />  <br />  <br />  <br />  <br /> Figure 1. RPP1, an essential yeast gene <br /> encodes a 32.2-kD protein ortholog of the <br /> human scleroderma autoimmune antigen, <br /> Rpp30. (A) Predicted amino acid sequence <br /> of S. cerevisiae Rpp1. The protein is en- <br /> coded by ORF YHR062c on chromosome <br /> VIII and its alignment with the human <br /> scleroderma autoimune antigen Rpp30 is <br /> shown. The amino acid sequences are <br /> numbered from the first methionine resi- <br /> due of each protein. Identical amino acids <br /> are shaded and similar amino acids are <br /> boxed. (B) The heterozygous diploid strain <br /> VS161, RPP1/rpp1::LEU2, was sporulated <br /> and dissections were performed on 20 tet- <br /> rads. The four spores (A&#226;&#8364;&#8220;D) derived from <br /> each of five tetrads are shown in vertical <br /> rows. <br />  <br />  <br />  <br /> by 38 end-labeling of RNA in 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 immunopre-               The association of Rpp1 with RNase P activity was <br /> cipitates that were washed with buffer that contained            also demonstrated by immunoprecipitation. Immuno- <br /> 600 mM KCl (Fig. 3B). Therefore, it is possible to achieve       precipiated (and resuspended) 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 pellets ac- <br /> a significant separation of the two enzymes, both physi-         curately cleaved radiolabeled ptRNASer (Fig. 4) and <br /> cally and functionally (Lygerou et al. 1996a; and see            ptRNATyr (data not shown) in vitro. We conclude that <br /> below).                                                          Rpp1 protein is a component of, or is tightly associated <br />  <br />  <br />  <br /> Table 1.   Strains of S. cerevisiae used in this study <br />  <br /> Strain                                                         Genotype <br />  <br /> JN161       MATa/MATa ade2-1/ade2-1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 his4-260/his4-260 thr1-4/thr1-4 lys2nNheI/lys2nNheI <br /> VS161       MATa/MATa ade2-1/ade2-1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 his4-260/his4-260 thr1-4/thr1-4 lys2nNheI/lys2nNheI <br />              RPP1/rpp1::LEU2/RPP1 <br /> VS162       MATa/MATa ade2-1/ade2-1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 his4-260/his4-260 thr1-4/thr1-4 lys2nNheI/lys2nNheI <br />              RPP1/rpp1::LEU2/RPP1 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162A      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162B      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI rpp1::LEU2 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162C      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162D      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI rpp1::LEU2 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162A8     MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 <br /> VS162C8     MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 <br /> NY1060      MATa/MATa GAL1/GAL1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 <br /> VS163       MATa/MATa GAL1/GAL1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 RPP1/rpp1::LEU2 <br /> VS164       MATa GAL1 leu2-3,112 ura3-1 + pYCpGAL::rpp1 (URA3) <br /> VS165       MATa GAL1 leu2-3,112 ura3-1 + pYCpGAL (URA3) <br />  <br />  <br />  <br /> 2928       GENES &amp; DEVELOPMENT <br />                                                                                       An essential protein subunit of RNase P from yeast <br />  <br />  <br />  <br /> with, catalytically active RNase P holoenzyme. Indirect <br /> supporting evidence for this conclusion comes from ex- <br /> periments in which the human homolog of Rpp1, Rpp30, <br /> was shown not to be separable from the active holoen- <br /> zyme after extensive biochemical purification (Eder et al. <br /> 1997). <br />  <br />  <br /> Construction of a conditional lethal allele of RPP1 <br /> RPP1 was placed under the control of the GAL10 pro- <br /> moter, which allows expression of the gene in culture <br /> medium that contains galactose but suppresses expres- <br /> sion in culture medium that contains glucose. The re- <br /> sulting strain, rpp1::LEU2&#226;&#8364;&#8220;pGAL::rpp1 (VS164), was <br /> compared in <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> to the control strain RPP1&#226;&#8364;&#8220; <br /> pGAL (VS165). In liquid culture that contained galac- <br /> tose, there was no difference in growth rate between the <br /> GAL::rpp1 strain and the wild-type RPP1 strain. After <br /> the cultures were transferred to medium that contained <br /> glucose, cell growth continued initially with a doubling <br /> time of 2 hr. After 12&#226;&#8364;&#8220;16 hr, the growth rate of the <br />  <br />  <br />  <br />  <br />                                                                     Figure 3. The RNA subunits of RNase P (RPR1) and RNase <br />                                                                     MRP (NME1) coprecipitate with 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1. (A) Immuno- <br />                                                                     precipitated RNAs extracted from the 9E10 Ab&#226;&#8364;&#8220;IgG&#226;&#8364;&#8220;agarose <br />                                                                     beads that were incubated with protein extracts from the four <br />                                                                     spores (VS162A8, VS162B, VS162C8, and VS162D; lanes 1&#226;&#8364;&#8220;4, re- <br />                                                                     spectively). RNA was extracted from the immunoprecipitated <br />                                                                     beads that were washed with 150 mM KCl (see Materials and <br />                                                                     Methods). The RNA was 38 end-labeled with [58-32P]pCp, and <br />                                                                     fractionated on a 8% polyacrylamide/7 M urea gel. (B) Immu- <br />                                                                     noprecipitated RNAs derived from the same immunoprecipi- <br />                                                                     tated beads as in A (lanes 1&#226;&#8364;&#8220;4, respectively) except that the <br />                                                                     immunoprecipitated beads were washed with 600 mM KCl prior <br />                                                                     to 38 end-labeling of the RNA. (C) Immunoprecipitated RNAs <br />                                                                     derived from the same immunoprecipitated beads as in A were <br />                                                                     transferred to a positively charged nylon membrane (Boehringer <br />                                                                     Mannheim) by electroblotting and hybridized with a uniformly <br />                                                                     labeled DNA probe complementary to the RPR1 gene (see Ma- <br /> Figure 2. 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 is functional and recognized by 9E10         terials and Methods). (Lanes 1&#226;&#8364;&#8220;4) Immunoprecipitated RNA <br /> antibody. (A) The four spores (VS162 A&#226;&#8364;&#8220;D; lanes 1&#226;&#8364;&#8220;4, respec-        from spores VS162A8, VS162B, VS162C8, and VS162D, respec- <br /> tively) derived from the diploid strain VS162, were dissected on    tively; (lane 5) RNA from supernatant of the immunoprecipi- <br /> rich medium plates (YPAD), and then replica-plated onto plates      tated extract derived from spore VS162A8 after centrifugation of <br /> that contained synthetic complete medium that lacked either         beads; (lane 6) RNA from supernatant of the immunoprecipi- <br /> leucine (leu) or uracil (ura), or that contained 5-FOA. (B) Immu-   tated extract derived from spore VS162B after centrifugation of <br /> noblot of protein extracts prepared from the four spores            beads; (lane 7) RNase P (RPR1) and RNase MRP (NME1) RNAs <br /> VS162A8, VS162B, VS162C8, and VS162D, derived from the dip-         (0.001 pmoles each) transcribed in vitro. Two small arrows <br /> loid strain VS162. All four spores are isogenic except the          point to larger species, which may be precursors to mature <br /> VS162A8 and VS162C8 spores do not have the pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;          RNase P RNA, RPR1 (Lee et al. 1991). The larger arrows indi- <br /> Rpp1 plasmid. (Lanes 1&#226;&#8364;&#8220;4) VS162A8, VS162B, VS162C8, and             cate RNase P (RPR1) and RNase MRP (NME1) RNAs. (D) Same <br /> VS162D, respectively. The arrowhead points to the 36-kD             as in C, except the membrane was hybridized with a uniformly <br /> 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fusion protein. The upper band is a nonspecifi-        labeled DNA probe complementary to the NME1 gene (Schmitt <br /> cally reacting protein.                                             and Clayton 1992). <br />  <br />  <br />                                                                                                  GENES &amp; DEVELOPMENT              2929 <br />  Stolc and Altman <br />  <br />  <br />  <br />                                                                   GAL::rpp1 strain declined rapidly and there was little <br />                                                                   growth after 16 hr (Fig. 5A). <br />  <br />  <br />                                                                   Rpp1 is required for processing of ptRNA <br />                                                                   To determine the effects of Rpp1 depletion on the bio- <br />                                                                   synthesis of metabolically stable RNAs, total RNA was <br />                                                                   isolated from the GAL::rpp1 strain (VS164) and the RPP1 <br />                                                                   strain (VS165) at various times during growth in glucose- <br />                                                                   containing medium. RNA samples from VS164 were <br />                                                                   compared with those from VS165 on a denaturing gel <br />                                                                   stained with ethidium bromide and by Northern hybrid- <br /> Figure 4. RNase P activity coprecipitates with 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1.      ization. <br /> IgG&#226;&#8364;&#8220;agarose pellets, to which is bound immunoprecipitated           The effect of Rpp1 depletion on the accumulation of <br /> RNase P RNA, derived from spores VS162A8, VS162B, VS162C8,        precursor tRNAs is shown in Figure 5B. Several RNAs <br /> and VS162D (as in Fig. 3B), were incubated with a uniformly       began to accumulate at 7 hr after transfer to glucose- <br /> labeled precursor tRNASer for 30 min at 37&#194;&#176;C, and then frac-      containing medium. At this time, the culture growth <br /> tionated on a 8% polyacrylamide/7 M urea gel (see Materials <br />                                                                   slowed. The abundance of these RNAs increased with <br /> and Methods). (Lane 1) Precursor tRNASer; (lanes 2&#226;&#8364;&#8220;5) immuno- <br />                                                                   time and their sizes were appropriate for ptRNAs. The <br /> precipitated RNase P from the four spores VS162A8, VS162B, <br /> VS162C8, and V6S162D, respectively; (lane 6) glycerol gradient-   abundance of mature tRNAs decreased accordingly. <br /> purified human RNase P, fraction F29 (Eder et al. 1997). Arrows   Analysis by Northern hybridization using probes <br /> indicate ptRNA, accurately processed mature tRNA, and 58          complementary to tRNALeu3 (Fig. 5C), the intervening <br /> leader sequence.                                                  sequence (IVS) (data not shown) and 58 leader sequence <br />  <br />  <br />  <br />  <br /> Figure 5. Rpp1 is required for processing of <br /> ptRNA and 5.8S rRNA in vivo. (A) Growth of the <br /> strain VS164 (j), and the wild-type, isogenic strain <br /> VS165 (h), after transfer from galactose-containing <br /> to glucose-containing medium at T = 0. Cell den- <br /> sity was measured at the times indicated and the <br /> cultures were diluted with glucose-containing me- <br /> dium at each time point to prevent nutritional de- <br /> privation and to maintain exponetial growth. (B) <br /> RNA was extracted from VS164 following growth <br /> in glucose-containing medium at the indicated <br /> times, and was fractionated in an 8% polyacryl- <br /> amide/7 M urea gel, and stained with ethidium <br /> bromide. The positions of 5.8S (L) RNA, 5.8S (S) <br /> RNA, 5S RNA, ptRNA, and mature tRNA are in- <br /> dicated. (C) Total RNA extracted from VS164 and <br /> VS165 was transferred to a positively charged ny- <br /> lon membrane (Boehringer Mannheim) by electro- <br /> blotting and hybridized with a g-32P-labeled oligo- <br /> nucleotide complementary to mature tRNALeu <br /> (see Materials and Methods). The position of the <br /> three processing intermediates of the ptRNA are <br /> indicated. PT is the primary transcript, which has <br /> extra sequences at both of its termini and contains <br /> an intron; IVS is the ptRNA that contains the in- <br /> tron but has been processed at both the 58 and 38 <br /> ends; 58 38 is the spliced ptRNA that is unpro- <br /> cessed at both termini. (D) Steady-state levels of <br /> RNase P RNA (RPR1) and RNase MRP RNA <br /> (NME1) in VS164 after transfer from galactose- <br /> containing to glucose-containing medium. The up- <br /> per band in the RPR1 panel is the putative precur- <br /> sor RPR1 RNA that has extra sequences at both <br /> termini (Lee et al. 1991). U6 snRNA levels were <br /> detected with oligo U6 (Table 2), as an internal <br /> control for RNA levels (Brow and Guthrie 1990). <br />  <br />  <br /> 2930      GENES &amp; DEVELOPMENT <br />                                                                               An essential protein subunit of RNase P from yeast <br />  <br />  <br />  <br /> of the pre-tRNALeu3 (data not shown), showed an accu-        both mature and precursor RPR1 RNAs and NME1 RNA <br /> mulation of tRNA that was unprocessed at both termini.       are reduced (Lygerou et al. 1996a; Chu et al. 1997; Dichtl <br /> After growth of the GAL::rpp1 strain in glucose for 12 hr,   and Tollervey 1997). These data suggest that the amount <br /> 58 and 38 unprocessed ptRNALeu3 accumulated to ap-           of Rpp1 is correlated with the maturation and stability of <br /> proximately the same level as that of mature tRNA. The       RPR1 and stability of NME1 in vivo. Both RNAs may be <br /> level of pre-tRNALeu3 that was unspliced but processed       found within a large RNP complex, or alternatively, <br /> at both ends was reduced correspondingly. These results      Rpp1 may be shared between the two RNP particles in <br /> showed that the sequentially ordered removal of the 58       vivo. <br /> leader sequence, the 38 trailing sequence, and finally the <br /> intron of ptRNALeu3, is impaired in Rpp1-depleted cells. <br /> As this phenotype is observed in the RNase P RNA mu-         Rpp1 is required for processing of the 35S prRNA <br /> tants, rpr1 and rpr1 (T315DT307) (Lee et al. 1991; Cham-     In S. cerevisiae and other eukaryotes, rRNA is tran- <br /> berlain et al. 1996b), we conclude that Rpp1 is an essen-    scribed as a 35S precursor RNA that contains within it <br /> tial protein subunit of the catalytically active RNase P     the sequences for three of the four rRNA molecules (18S, <br /> complex in vivo.                                             5.8S, and 25&#226;&#8364;&#8220;28S). Subsequent processing and nucleotide <br />                                                              modifications involving endonucleolytic and exonucleo- <br />                                                              lytic cleavages and methylation generate mature rRNA <br /> Rpp1 is required for accumulation of RNase P and <br />                                                              (for review, see Eichler and Craig 1994; Venema and Tol- <br /> RNase MRP RNAs <br />                                                              lervey 1995; Tollervey 1996). We examined the fidelity <br /> We investigated further the effect of Rpp1 depletion on      of rRNA processing in Rpp1-depleted cells to determine <br /> the steady-state levels of RNase P and RNase MRP             if defects in this pathway were similar to those described <br /> RNAs to ascertain if cells lacking Rpp1 shared pheno-        previously for mutants that affect RNase P and RNase <br /> typic traits with previously described conditional lethal    MRP (Shuai and Warner 1991; Lindahl et al. 1992; <br /> mutants of Pop1, Pop3 and Pop4, proteins that associate      Schmitt and Clayton 1993; Chamberlain et al. 1996b; <br /> with both RNPs. Depletion of Rpp1 results in a decrease      Lygerou et al. 1996a; Chu et al. 1997; Dichtl and Toller- <br /> of the steady-state levels of RNase P and RNase MRP          vey 1997). rRNAs were analyzed by gels stained with <br /> RNAs (Fig. 5D). However, the RPR1 RNA precursor does         ethidium bromide and by Northern analysis with oligo- <br /> not appear to decrease to the same extent as mature          nucleotide probes (Table 2) to detect various prRNA spe- <br /> RPR1 RNA, even after 30 hr of Rpp1 depletion. As with        cies (see Figs. 5B and 6). <br /> Pop4 depletion (Chu et al. 1997), mature RPR1 RNA is            On depletion of the Rpp1 protein, we observed mul- <br /> not detectable after 21 hr of Rpp1 depletion. In contrast,   tiple defects in rRNA processing at the A0 and A1 sites <br /> depletion of Pop3 does not affect steady-state levels of     in the 58 external transcribed sequence (58 ETS), at the <br /> these RNAs, whereas in pop1-1 steady-state levels of         A2 and A3 sites within the internal transcribed sequence <br />  <br />  <br />  <br /> Table 2.   Oligonucleotides used in this study <br />  <br /> Oligo   1: 58-CAGCAGAGAGACCCGA-38 <br /> Oligo   2: 58-ACTATCTTAAAAGAAGAAGC-38 <br /> Oligo   3: 58-GAATTACCACGGTTATACC-38 <br /> Oligo   4: 58-GCACAGAAATCTCTCACC-38 <br /> Oligo   5: 58-ATGAAAACTCCACAGTG-38 <br /> Oligo   6: 58-CCAGTTACGAAAATTCTTG-38 <br /> Oligo   7: 58-CGCATTTCGCTGCGTTCTTCATCG-38 <br /> Oligo   8: 58-ACAGAATGTTTGAGAAGGAAATG-38 <br /> Oligo   U6: 58-TCATCCTTATGCAGGG-38 <br /> Oligo   mtRNA: 58-GCATCTTACGATACCTG-38 <br /> Oligo   ltRNA: 58-CCAAACAACCACTTATTTGTTGA-38 <br /> Oligo   itRNA: 58-CACAGTTAACTGCGGTC-38 <br /> Oligo   T7MRP: 58-GGGAATTCGAAATTAATACGACTCACTATAGAATCCATGACCAAAGAATCGTCA-38 <br /> Oligo   38 MRP: 58-TCCCCCGGGTGAATCCATGGACCAAGA-38 <br /> Oligo   T7ITS14: 58-GGGAATTCGAAATTAATACGACTCACTATAGACACTGTGGAGTTTTCATATC-38 <br /> Oligo   38 ITS14: 58-GGGGATCCTTAAAATTTCCAGTTACGAAAATTC-38 <br /> Oligo   T7ITS16: 58-GGGAATTCGAAATTAATACGACTCACTATAGGCCAAACGGTGAGAGATTTCTG-38 <br /> Oligo   38 ITS16: 58-GGGGATCCACTTTAAGAACATTGTTCGCCT-38 <br /> Oligo   MYC5: 58-GCGGAATGCTGGAACAGAAACTTATT-38 <br /> Oligo   T7ITS16: 58-GGGAATTCGAAATTAATACGACTCACTATAGGCCAAACGGTGAGAGATTTCTG-38 <br /> Oligo   38 ITS16: 58-GGGGATCCACTTTAAGAACATTGTTCGCCT-38 <br /> Oligo   MYC3: 58-ACCAGCATTCCCAAATCTTCTTCAGA-38 <br /> Oligo   5 GAL30: 58-GGGGATCCGCATTATAGAACCGAGAATGC-38 <br /> Oligo   3 GAL30: 58-GGTCTAGAGATCCCCAAATTTTTTTGTCT-38 <br />  <br />  <br />  <br />                                                                                          GENES &amp; DEVELOPMENT              2931 <br />  Stolc and Altman <br />  <br />  <br />  <br /> (ITS1), and at the E and/or C2 sites within the internal       5.8S(L) rRNA increased (Fig. 5B). The altered ratio be- <br /> transcribed sequence 2 (ITS2) (Fig. 6). Analysis of low-       tween 5.8S(L) and 5.8S(S) suggests a defect in cleavage at <br /> molecular-weight RNA showed that synthesis of 5.8S(S)          the A3 site in the ITS1 and/or other processing sites <br /> rRNA was reduced in an Rpp1-depleted strain, whereas           required for maturation of 5.8S rRNA in vivo (Schmitt <br />  <br />  <br />  <br />  <br />                                          Figure 6. (See facing page for legend.) <br />  <br />  <br /> 2932      GENES &amp; DEVELOPMENT <br />                                                                                          An essential protein subunit of RNase P from yeast <br />  <br />  <br />  <br /> and Clayton 1993; Henry et al. 1994). Figure 6D shows <br /> an accumulation of a 7S precursor to 5.8S rRNA after <br /> 7 hr of Rpp1 depletion. This precursor is predicted to <br /> have its 58 end at the A2 site of ITS1 and its 38 end at <br /> the 38 end of 5.8S rRNA. If this prediction is proven <br /> correct, then we will be able to conclude that there is a <br /> defect in cleavage at the A3 site under the conditions we <br /> used. <br />    In addition, two large precursors of the 18S rRNA ac- <br /> cumulated at later times in Rpp1-depleted cells (Fig. <br /> 6C,D). The 24S rRNA precursor that contains both 58 <br /> ETS and ITS1 sequences shows defects in cleavage at the <br /> A0, A1, A2, and A3 processing sites, and the 21S rRNA <br /> precursor shows defects in cleavage and processing at the <br /> A2 and A3 processing sites in ITS1 and at the E site in <br /> ITS2 (Fig. 6C,D). The 17S and 12S rRNA degradation <br /> intermediates represent sequences with fragmented 58 <br /> ends within the 18S rRNA (Fig. 6C,D; Allmang et al. <br /> 1996a). Figure 6E shows depletion of the 7S(S) and 7S(L) <br /> precursors to 5.8S rRNA, depletion of the 27SA and 27SB <br /> precursors to 25S rRNA, and accumulation of an 8S <br /> rRNA precursor of the 5.8S rRNA, which contains 58                    Figure 7. Processing of prRNA by Rpp1 immunoprecipitates. <br /> extended sequences from ITS1 and 38 extended se-                      (A) Position of the ITS141 rRNA substrate relative to the 35S <br /> quences from ITS2. These intermediates indicate defects               rRNA precursor [58 end is at A2 site and 38 end is at B1(L) site]. <br /> in cleavage at the A3, and E and/or C2 processing sites of            (B) Uniformly labeled rRNA transcript, ITS141 (see Materials <br /> the ITS1 and ITS2, respectively. All probes also detected             and Methods), was incubated with 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 immunopre- <br /> a moderate accumulation of the 35S rRNA primary tran-                 cipitates derived from the same immunoprecipitates as in Fig. <br /> script and probes 2&#226;&#8364;&#8220;7 also detect the 32S precursor rRNA              3A for 2 hr at 37&#194;&#176;C, and then fractionated on a 8% polyacryl- <br />                                                                       amide/7 M urea gel. (Lanes 1&#226;&#8364;&#8220;4) ITS141 rRNA, plus IgG&#226;&#8364;&#8220;agarose <br /> (Fig. 6C&#226;&#8364;&#8220;E; data not shown). Despite these defects the <br />                                                                       pellets to which are bound immunoprecipitated RNase P and <br /> steady-state levels of the 18S rRNA and 25S rRNA re-                  RNase MRP RNAs as in Fig. 3A. (Lane 5) ITS141 rRNA. The <br /> mained unchanged (Fig. 6B), suggesting delayed process-               arrow indicates the position of two cleavage products of almost <br /> ing of 35S rRNA in the absence of Rpp1.                               identical size from the ITS1 rRNA transcripts. The site of cleav- <br />    Our results show that Rpp1 is required for processing              age corresponds to the region of the A3 site in the internal <br /> of 35S rRNA in the 58 ETS, ITS1, and ITS2. Interestingly,             transcribed sequence 1 (ITS1) (Lygerou et al. 1996). <br /> some of the same processing reactions have been shown <br /> to be dependent on snoRNPs, RNase MRP, and RNase P, <br /> and the same stable rRNA degradation intermediates ac- <br /> cumulate in cells deficient in snoRNP components                      RNP particles exhibit the same global defects in prRNA <br /> (Shuai and Warner 1991; Lindahl et al. 1992; Chamber-                 and ptRNA processing found in Rpp1-depleted cells. In- <br /> lain et al. 1996b; Venema and Tollervey 1996). However,               terestingly, depletion of the snoRNP protein, Rrp5, re- <br /> none of the known proteins that associate with these                  sults in striking similarities to Rpp1-depleted cells with <br />  <br />  <br />  <br /> Figure 6. Effects of Rpp1 depletion on the processing of 35S prRNA. (A) Schematic plan of the rRNA processing pathway (Venema <br /> and Tollervey 1995). Steady-state levels of rRNA, prRNA intermediates, and aberrant rRNA species were detected by ethidium <br /> bromide staining and by Northern hybridization with oligonucleotide probes 1&#226;&#8364;&#8220;7. (B) Total RNA extracted from VS164 and VS165 was <br /> obtained after transfer from galactose-containing to glucose-containing medium at the times indicated, and separated in a l.2% agarose <br /> gel stained with ethidium bromide. Arrows indicate 25S rRNA, 18S rRNA, 5.8S rRNA, and tRNAs. (C&#226;&#8364;&#8220;E) The gel shown in B was <br /> transferred to a positively charged nylon membrane (Boehringer Mannheim) by capillary diffusion and hybridized with g-32-labeled <br /> oligonucleotide probes 4&#226;&#8364;&#8220;6 (see Materials and Methods). Oligonucleotide 4 (C) is complementary to the ITS1, between 38 end of 18S <br /> rRNA and the A2 site (position 157&#226;&#8364;&#8220;180), oligonucleotide 5 (D) is complementary to the ITS1, between A2 and A3 sites (position <br /> 119&#226;&#8364;&#8220;236), and oligonucleotide 8 (E) is complementary to the ITS2, between the 38 end of 5.8S rRNA and C2 site (position &#226;&#710;&#8217;3 to +21). <br /> The 24S precursor rRNA is predicted to extend from the 58 end of the 58 ETS to the 38 end of 5.8S rRNA, and represents a product of <br /> the 35S rRNA precursor that is cleaved in ITS2 in the absence of cleavage at the A0, A1, A2, and A3 sites. The 21S precursor rRNA <br /> appears to be comprised of rRNA species that are predicted to extend from the 58 end of 18S rRNA to the 38 end of 5.8S rRNA, and <br /> also have 38 ends that extend 38 to the 38 end of 5.8S rRNA, and may be heterogeneous. The 208S rRNA species is predicted to extend <br /> from the 58 end of 18S rRNA to the region of the A3 site in the ITS1. The 17S8 and 12S8 aberrant rRNA species represent stable <br /> degradation intermediates, which may have fragmented 58 ends that correspond to sequences within 18S rRNA and 38 ends close to <br /> the 38 end of 5.8S rRNA. The 8S rRNA species is predicted to extend from the 58 end of ITS1 to the 38 extended ends of 5.8S rRNA. <br /> The 7S RNA is the precursor of the 5.8S rRNA, which is predicted to extend from the region of the A2 site in the ITS1 to the 38 end <br /> of 5.8S rRNA. <br />  <br />  <br />                                                                                                     GENES &amp; DEVELOPMENT              2933 <br />  Stolc and Altman <br />  <br />  <br />  <br /> respect to defects of prRNA processing (Venema and Tol-      site), ascribed previously to RNase MRP (Lygerou et al. <br /> lervey 1996; see Discussion). Therefore, we suggest that     1996a). However, in contrast to RNase P, RNase MRP <br /> RNase P interacts functionally with RNase MRP and            RNA and the rRNA processing activity can be separated <br /> perhaps other snoRNPs in the processing of rRNA in           from Rpp1 after high-salt washes. Furthermore, the hu- <br /> yeast.                                                       man Rpp30 does not associate with RNase MRP (N. Jar- <br />                                                              rous and S. Altman, unpubl.) Whether yeast RNase P <br />                                                              cleaves the prRNA substrates directly in vivo is un- <br /> Processing of precursor rRNA by Rpp1 <br />                                                              known. Proposals for the secondary structure of ITS1 <br /> immunoprecipitates <br />                                                              that are based on phylogenetic analysis and chemical <br /> We tested whether Rpp1 is associated directly with           mapping of ITS1 (Van Nues et al. 1994; Allmang et al. <br /> prRNA processing activity by in vitro cleavage assay of      1996b) and computational folding of the ITS1.141 <br /> two fragments of the 35S rRNA. We assayed cleavage in        prRNA substrate, do not show a common structural fea- <br /> vitro by resuspending 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1-containing immu-          ture of an RNase P substrate (a helical segment at the <br /> noprecipitates with two fragments of the 35S precursor       junction of a single-stranded region; Altman et al. 1995). <br /> rRNA, ITS1.603 (Chamberlain et al. 1996b) and ITS1.141       Moreover, prRNA substrates used in this study are not <br /> (similar to Lygerou et al. 1996a). Both prRNA substrates     cleaved at the A3 site in vitro by reconstituted E. coli <br /> overlap the ITS1 (see Fig. 7A, and Materials and Meth-       RNase P (V. Stolc and S. Altman, unpubl.), which has a <br /> ods). Rpp1 immunoprecipitates, which contained both          broader substrate specificity than eukaryotic RNase P <br /> RNase P and RNase MRP RNAs, cleaved ITS1.141 (Fig.           (Yuan and Altman 1994). Therefore, in vivo, yeast Rpp1 <br /> 7B) and ITS1.603 (data not shown) in the region of the A3    may be a shared subunit of a large RNP complex con- <br /> processing site. This result is consistent with, but does    taining both RNase P and RNase MRP, and perhaps <br /> not rigorously prove, the observed defects in processing     other snoRNPs. <br /> the 35S rRNA on depletion of the Rpp1 protein in vivo.          On depletion of Rpp1 in vivo, we found a defect in <br /> However, 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 immunoprecipitates that were           ptRNA processing, an indication that Rpp1 is associated <br /> washed extensively with high ionic buffer (see Material      with RNase P activity. Surprisingly, we also found an <br /> and Methods) failed to cleave the prRNA substrates but       rRNA processing defect characterized by the absence of <br /> cleaved ptRNAs in vitro (data not shown). These results      cleavage at the A0 and A1 sites in the 58 ETS, and at the <br /> suggest that in addition to RNase MRP, there is a direct     A2 and A3 sites in ITS1, and at the E and/or C2 sites in <br /> role for an RNase P-containing complex in rRNA pro-          ITS2 of the primary 35S rRNA transcript. To our knowl- <br /> cessing in vivo.                                             edge, Rpp1 is the only RNase P or RNase MRP protein, <br />                                                              that on depletion simultaneously inhibits cleavages at <br />                                                              all of these major processing sites of 35S rRNA and is <br /> Discussion <br />                                                              required for ptRNA processing. Moreover, processing of <br /> We have cloned a gene encoding a protein subunit of          prRNA at sites A0, A1, A2, A3, E, and/or C2 may be <br /> nuclear RNase P from S. cerevisiae based on its homol-       coordinated by RNase P, as depletion of RNase MRP <br /> ogy to human Rpp30. Yeast Rpp1 is homologous to the          RNA alone results only in defects at the A3 site (Schmitt <br /> human scleroderma autoimmune antigen, Rpp30, which           and Clayton 1993). <br /> was described previously as an autoantigen that copuri-         Apart from RNase MRP, other essential snoRNPs may <br /> fies with at least six other Rpp protein subunits of hu-     be involved directly in the mechanism of rRNA process- <br /> man RNase P&#226;&#8364;&#8221;Rpp14, Rpp20, Rpp25, Rpp30, Rpp38,               ing (for recent reviews, see Eichler and Craig 1994; Ven- <br /> and Rpp40 (Eder et al. 1997). Using computer database        ema and Tollervey 1995; Tollervey 1996). In contrast to <br /> searches (blastp and tblastn algorithms), we compared        the Rpp1 phenotype, depletion- or temperature-sensitive <br /> the predicted amino acid sequences of human Rpp pro-         alleles of these snoRNAs or their associated proteins <br /> teins with the S. cerevisiae genome database. This analy-    show no defect in ptRNA processing and, with the ex- <br /> sis revealed that Rpp1 is one of three yeast proteins with   ception of snoRNP protein Rrp5, no single component is <br /> amino acid sequence similarity to the human Rpp pro-         defective for processing at all of the processing sites in <br /> teins. A second such yeast protein is Pop4, a subunit of     the 58 ETS, ITS1, and ITS2. For example, Gar1p, Nop1p, <br /> yeast RNase P and RNase MRP (Chu et al. 1997), which         Sof1p, as well as snoRNA U3, U14, and snR30-depleted <br /> is related in amino acid sequence to a previously unchar-    cells, and strains lacking snR10 are defective at A0, A1, <br /> acterized human Rpp protein, Rpp29 (P. Eder, N. Jarrous,     and A2 sites without detectable inhibition of the A3 site <br /> and S. Altman, unpubl.). The third yeast, protein, Rpp2,     (Tollervey 1987; Li et al. 1990; Hughes and Ares 1991; <br /> shares amino acid similarity with Rpp20 (V. Stolc and S.     Tollervey et al. 1991; Girard et al. 1992; Jansen et al. <br /> Altman, unpubl.)                                             1993; Morrissey and Tollervey 1993). The prRNA pro- <br />    Rpp1 is a small basic protein with a predicted molecu-    cessing phenotype of Rrp5-depleted strains differs from <br /> lar mass of 32.2 kD and pI 9.76. It does not have any        that of Rpp1 in that it is defective in maintaining accu- <br /> previously identified RNA-binding domains. In vitro,         mulation of 25S rRNA, 18S rRNA, and relative levels of <br /> Rpp1 remains associated with the mature RNase P RNA          rRNA intermediates (Venema and Tollervey 1996). In- <br /> and RNase P activity, even in buffers of high ionic          terestingly, as in Rpp1-depleted cells, cleavage at the A3 <br /> strength. Rpp1 also associates with RNase MRP RNA            site is also defective in Pop1, Pop3, Pop4, and RNase <br /> and an rRNA processing activity (cleavage at the A3          MRP (NME1) conditional mutants (Shaui and Warner <br />  <br />  <br /> 2934      GENES &amp; DEVELOPMENT <br />                                                                                         An essential protein subunit of RNase P from yeast <br />  <br />  <br />  <br /> 1981; Lindahl et al. 1992; Schmitt and Clayton 1993;                 rRNA ITS1 substrate, ITS1.603 (Chamberlain et al. 1996b), oli- <br /> Lygerou et al. 1996a; Chu et al. 1997; Dichtl and Toller-            gonucleotides T7ITS16 and 38 ITS16 were used to PCR amplify <br /> vey 1997). Therefore, although Rpp1 shares rRNA pro-                 yeast genomic DNA and the amplified fragment was subse- <br /> cessing defects with known RNase P, RNase MRP, and                   quently subcloned into pUC19 the same way as for the ITS1.141 <br />                                                                      fragment. The NME-coding sequence was amplified by PCR <br /> snoRNPs, it is the only described protein that has both <br />                                                                      from yeast genomic DNA using T7MRP and 38 MRP oligo- <br /> ptRNA processing defect as well as rRNA processing de- <br />                                                                      nucleotides, and subcloned into EcoRI and SmaI sites of pUC19 <br /> fects at the A0, A1, A2, A3, and E, and/or C2 processing             to generate pUCT7MRP. <br /> sites. <br />                                                                      Strain construction <br /> Materials and methods                                                The diploid strain heterozygous for the RPP1 deletion (VS161) <br /> Strains, media, and general procedures                               was transformed with pRS316::3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 plasmid and <br />                                                                      sporulated. After sporulation and dissection, spores disrupted <br /> S. cerevisiae strains used in this work are listed in Table 1. The   for RPP1, but harboring the plasmid encoded 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fu- <br /> composition of the media with appropriate nutrients for plas-        sion protein, were viable. Only two haploid cells (VS162A8 and <br /> mid maintenance and S. cerevisiae growth and handling tech-          VS162C8), derived from sporulation of a single tetrad, were vi- <br /> niques were used as described (Guthrie and Fink 1991). Unless        able after the loss of the pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 plasmid during <br /> stated otherwise, all techniques for manipulating DNA, RNA,          selection on plates that contain 5-fluoro-orotic acid (5-FOA): <br /> and oligonucleotides were performed according to standard pro-       These haploids were Leu&#226;&#710;&#8217;, showing that they had lost the <br /> cedures (Sambrook et al. 1989). The identities of all constructs     pRS316&#226;&#8364;&#8220;3 &#195;&#8212; &#226;&#8364;&#8220;RPP1 plasmid and had the wild-type RPP1 allele. <br /> were verified by sequence analysis. Oligonucleotides used in         The other two haploids (VS162B and VS162D) were not viable <br /> this study are listed in Table 2.                                    without the pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 plasmid during selection on <br />                                                                      5-FOA plates and were Leu+, showing that they had the <br /> Gene disruption                                                      3 &#195;&#8212; myc&#226;&#8364;&#8220;tagged RPP1 allele. The same results were obtained <br />                                                                      after dissection of five additional tetrads. The growth rate of two <br /> A genomic clone spanning the RPP1 locus was identified using         haploid rpp1:LEU2 cells (VS162B and VS162D), which depend <br /> the S. cerevisiae Genome Database (SGD) and obtained from the        on a low-copy-number episomal plasmid encoding the 3 &#195;&#8212; myc&#226;&#8364;&#8220; <br /> American Tissue Culture Collection (ATCC) as cosmid 8025.            RPP1 (pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1) for viability, was identical to <br /> Clone 8025 was sequenced previously (Johnston et al. 1994). A        that of the wild-type haploid cells (VS162A and VS162C). <br /> 1.4-kb BamHI&#226;&#8364;&#8220;XbaI fragment encoding the RPP1 gene was sub-             VS163 was transformed with pYCp&#226;&#8364;&#8220;GAL::rpp1 plasmid <br /> cloned from cosmid 8025 into pBluescript (SK) vector (Strata-        (VS164) or a control plasmid, pYCp&#226;&#8364;&#8220;GAL (VS165), sporulated <br /> gene) cut with BamHI&#226;&#8364;&#8220;XbaI to generate pRPP1SK. The RPP1              and germinated on medium that contained galactose. Four vi- <br /> gene was disrupted by replacing a 0.6-kb NcoI&#226;&#8364;&#8220;NdeI fragment          able spores were obtained from several independent tetrads and <br /> with the LEU2 gene, thereby deleting 65% of the coding se-           the presence of the plasmid-encoded GAL::rpp1 fusion gene was <br /> quence. The rpp1::LEU2 construct was integrated at the RPP1          verified on ura&#226;&#710;&#8217; and leu&#226;&#710;&#8217; plates containing galactose. <br /> genomic locus in diploid strain JN161 (VS161) and correct re- <br /> placement of one allele was verified by PCR digest analysis: <br /> Oligonucleotides 5GAL30 and 3GAL30 were used to amplify              Immunoprecipitation <br /> genomic DNA isolated from VS161, the resulting PCR-ampli-            Extracts were prepared by lysis of yeast cells (25 ml, <br /> fied fragment (2.7 kb) encoding LEU2 and flanking sequences of       OD600 = 0.5) with glass beads (Guthrie and Fink 1991). Extracts <br /> Rpp1 was mapped with restriction enzymes. The heterozygous           were clarified by centrifugation three times at 15,000g for 10 <br /> RPP1/rpp1::LEU2 strain (VS161) was sporulated and tetrad             min. The final supernatants were used in immunoprecipitations <br /> analysis showed a 2:2 segregation for cell viability. Dissections    by adding 4 &#194;&#181;g of 9E10 antibody to the protein extract derived <br /> were performed on 20 tetrads. The same disruption and analysis       from spores VS162A8, VS162B, VS162C8, and VS162D, and in- <br /> was performed for strain VS163, which was disrupted with the         cubated for 2.5 hr, followed by addition of 50 &#194;&#181;l of 100 mg/ml of <br /> rpp1::LEU2 construct.                                                anti-mouse IgG (whole molecule) agarose (Sigma) in IP150 <br />                                                                      buffer (150 mM KCl, 10 mM Tris-Cl (pH 7.9), 0.1% NP-40, and <br />                                                                      0.1% NaN3). Pellets were washed five times with IP150 or <br /> Construction of plasmids <br />                                                                      IP600 (same composition as IP150, except 600 mM KCl). RNA <br /> To generate myc-epitope-tagged RPP1, three myc epitope do-           was extracted from the immunoprecipitated beads by adding <br /> mains (3 &#195;&#8212; myc) were PCR-amplified from the 3 &#195;&#8212; myc&#226;&#8364;&#8220;SEC8             100 &#194;&#181;l of IPR buffer (100 mM Tris-Cl at pH 7.5, 100 mM EDTA <br /> construct (TerBush and Novick 1996) using oligonucleotides           at pH 8.0, 150 mM NaCl, 1% SDS), followed by two extractions <br /> MYC5 and MYC3, and then cut with BsmI. The PCR fragment              with phenol and one extraction with phenol/chloroform/iso- <br /> was subcloned into the BsmI site of pRPP1SK plasmid to gen-          amylalcohol solution (25:24:1). RNA was precipitated with <br /> erate p3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1SK. A 1.5-kb BamHI&#226;&#8364;&#8220;XbaI fragment from             ethanol. <br /> p3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1SK was subcloned into pRS316 plasmid to gen- <br /> erate pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc::RPP1. The coding sequence of RPP1 was <br />                                                                      Assays for RNase P activity and rRNA processing activity <br /> PCR-amplified from pRPP1SK using oligonucleotides 5GAL30 <br /> and 3GAL30, cut with BamHI and XhoI, and subcloned into              IgG&#226;&#8364;&#8220;agarose pellets, to which are bound immunoprecipitated <br /> modified pYCp33 vector that contained the GAL1&#226;&#8364;&#8220;10 promoter           RNase P and RNase MRP RNAs, were washed with IP150 and <br /> to generate pYCp&#226;&#8364;&#8220;GAL::rpp1. To generate rRNA ITS1 substrate,         incubated with labeled ptRNASer (Drainas et al. 1989) for 30 <br /> ITS1.141 (similar to Lygerou et al. 1996a), oligonucleotides         min at 37&#194;&#176;C in 1&#195;&#8212; BB (10 mM HEPES at pH 8.0, 400 mM <br /> T7ITS14 and 38 ITS14 were used to PCR-amplify yeast genomic          NH4OAc, 10 mM Mg2OAc, 5% glycerol). rRNA transcripts <br /> DNA and then the 141-bp fragment was cloned into EcoRI and           ITS1.141 and ITS603 (Chamberlain et al. 1996b) were incubated <br /> BamHI sites in pUC19 (New England Biolabs). To generate              with IgG pellets to which are bound immunoprecipitated <br />  <br />  <br />  <br />  <br /> RNase P and RNase MRP RNAs (washed with IP150) for 2 hr at          <br /> 37&#194;&#176;C in 1&#195;&#8212; PMB (20 mM Tris-HCl at pH 8.0, 10 mM MgCl2, 1 mM          <br /> EDTA, 50 mM KCl, 2 units of RNasin, and 50 mg/ml of BSA).             <br /> IgG pellets, to which is bound RNase P RNA, were washed with         <br /> IP600 and incubated with either ptRNASer, ITS141, or ITS603 as           <br /> above. The tRNA substrate, ITS141, and ITS603 rRNA sub-                <br /> strates were labeled uniformly with [32P]GTP (3000 mCi/              <br /> mmole, Amersham) and purified on an 8% polyacrylamide/7 M               <br /> urea gel. The assays were performed with 0.5 nM ptRNA, 3.2 nM            <br /> ITS141 RNA, or 0.5 nM ITS603 RNA. The RNA products were               <br /> fractionated on an 8% polyacrylamide/7 M urea gel.                <br /> RNA analysis <br />                                                                             <br /> Total RNA was isolated by disruption of cell pellets (25 ml,                <br /> OD600 = 0.5), resuspended in AE buffer [50 mM NaAc at pH 5.3,        <br /> 10 mM EDTA, 10 mM RNase inhibitor, vanadyl ribonucleoside                  <br /> complex (GIBCO BRL)] and equal volume of phenol and chlo-                   <br /> roform, with glass beads at 65&#194;&#176;C for 15 min. Phenol/chloroform             <br /> extractions were repeated four times at 65&#194;&#176;C for 15 min, fol-          <br /> lowed by an extraction with chloroform at 25&#194;&#176;C and ethanol                 <br /> precipitation. Northern hybridization was performed as de-               <br /> scribed previously (Guerrier-Takada et al. 1995). All oligo-              <br /> nucleotides were end-labeled with T4 polynucleotide kinase             <br /> (New England Biolabs) and [g-32P]ATP (Amersham). NME1                    <br /> EcoRI&#226;&#8364;&#8220;SmaI DNA fragment from pUCT7MRP and EcoRI&#226;&#8364;&#8220;SmaI                      <br /> DNA fragment from pScRNAP (Lee et al. 1991) were labeled                  <br /> uniformly with [32P]dCTP (Amersham).                                  <br /> </body></html>