<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                ARTICLE <br /> Haploinsufficiency TAB2 Causes <br /> Congenital Heart Defects Humans <br /> Bernard Thienpont,1,14 Litu Zhang,2,15 Alex V. Postma,3 Jeroen Breckpot,1 Le&#194;&#180;on-Charles Tranchevent,4 <br /> Peter Van Loo,5,6 Kjeld M&#195;&#184;llga&#203;&#353;rd,7 Niels Tommerup,2 Iben Bache,2 Zeynep Tu  &#194;&#168; mer,2,8 <br /> Klaartje van Engelen, Bjo&#194;&#168;rn Menten, Geert Mortier, <br />                      9                10             10,11 Darrel Waggoner, Marc Gewillig,13 <br />                                                                             12 <br />              4                  1 <br /> Yves Moreau, Koen Devriendt, Lars Allan Larsen    2,* <br />  <br /> Congenital heart defects (CHDs) common major developmental anomalies frequent cause perinatal <br /> mortality, etiology remains obscure. We identi&#239;&#172;?ed locus CHDs 6q24-q25. Genotype-<span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>phenotype</span> correlations <br /> in 12 patients carrying chromosomal deletion 6q delineated critical 850 kb region 6q25.1 harboring &#239;&#172;?ve genes. Bioinformatics <br /> prioritization candidate genes locus role CHDs identi&#239;&#172;?ed TGF-b-activated kinase 1/MAP3K7 binding protein 2 gene <br /> (TAB2) top-ranking candidate gene. A role candidate gene cardiac development supported conserved <br /> expression developing human zebra&#239;&#172;?sh heart. Moreover, critical, dosage-sensitive role development demon- <br /> strated cardiac defects observed titrated knockdown tab2 expression zebra&#239;&#172;?sh embryos. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> de&#239;&#172;?nitively con&#239;&#172;?rm the <br /> role candidate gene CHDs, performed <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation analysis</span> TAB2 402 patients CHD, revealed evolu- <br /> tionarily conserved missense mutations. Finally, balanced translocation identi&#239;&#172;?ed, cosegregating familial CHD. Mapping of <br /> the breakpoints demonstrated translocation disrupts TAB2. Taken together, data clearly demonstrate role TAB2 in <br /> human cardiac development. <br />  <br />  <br /> Introduction                                                                   To accommodate limitations, attempted an <br />                                                                             alternative approach identifying loci associated with <br /> Congenital heart defects (CHDs) single impor-                  CHDs chromosomal rearrangements. Such strat- <br /> tant congenital cause perinatal mortality mor-                      egy enables identi&#239;&#172;?cation regions harboring genes <br /> bidity,1,2 despite manifest importance, etiology             involved heart development dosage-sensitive man- <br /> remains largely obscure. Although epidemiological studies                   ner construction human morbidity map for <br /> demonstrated certain environmental factors con-                    CHDs. We previously reported identi&#239;&#172;?cation several <br /> tributory,3 family twin studies suggest major genetic                 candidate loci CHD screening patients <br /> component.4,5 Indeed, mutations genes asso-                 CHD means array comparative genome <br /> ciated monogenic CHDs, mainly linkage anal-                    hybridization (aCGH).10,11 We delinea- <br /> ysis large families CHD segregates auto-                tion characterization loci, located on <br /> somal-dominant trait.6 Given mortality associated                  chromosome 6q24-q25 containing MAP3K7IP2 <br /> CHDs, large families rare. Although family studies                 (TAB2 [MIM 605101]) gene. TAB2 maps critical <br /> suggest partially penetrant causes CHDs                    region deleted patients CHD, dosage-sensitive <br /> more common purely monogenic causes,7,8 loci                      zebra&#239;&#172;?sh development, speci&#239;&#172;?cally expressed <br /> have remained unidenti&#239;&#172;?ed linkage studies. As result,                  human zebra&#239;&#172;?sh cardiovascular system. <br /> only causative genes identi&#239;&#172;?ed, <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span>                  We disrupted balanced trans- <br /> analyses shown account small                 location family members CHD is <br /> fraction (&lt; 1%) CHD cases,9 representing limita-               mutated 402 patients CHDs, providing <br /> tion genetic counseling CHD patients                    strong evidence TAB2 major role cardiac <br /> families elucidation pathogenesis CHD.                 development. <br />  <br />  <br />  <br /> 1 <br />   Laboratory Genetics Human Development, Department Human Genetics, University Leuven, B3000 Leuven, Belgium; 2Wilhelm Johann- <br /> sen Centre Functional Genome Research, Department Cellular Molecular Medicine, University Copenhagen, DK-2200 Copenhagen, <br /> Denmark; 3Department Anatomy &amp; Embryology, Heart Failure Research Center, L2-108-2, Academic Medical Center Meibergdreef 15, 1105 AZ Amster- <br /> dam, The Netherlands; 4Department Electrical Engineering ESAT-SCD, University Leuven, B3000 Leuven, Belgium; 5Department Human Genetics, <br /> University Leuven, B3000 Leuven, Belgium; 6Department Molecular Developmental Genetics, VIB, B3000 Leuven, Belgium; 7Developmental <br /> Biology <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, Department Cellular Molecular Medicine, University Copenhagen, DK-2200 Copenhagen, Denmark; 8Applied Human Molecular <br /> Genetics, Kennedy Center, DK-2600 Glostrup, Denmark; 9Department Cardiology Clinical Genetics, Academic Medical Centre, 1105 AZ Amster- <br /> dam, The Netherlands; 10Center Medical Genetics, Ghent University Hospital, B9000 Ghent, Belgium; 11Department Medical Genetics, Antwerp <br /> University Antwerp University Hospital, B2160 Antwerpen, Belgium; 12Department Pediatrics Department Genetics University Chi- <br /> cago, Chicago, IL 60637, USA; 13Paediatric Cardiology <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>, University Hospitals Leuven, B3000 Leuven, Belgium <br /> 14 <br />    Present address: Laboratory Molecular Signalling &amp; Laboratory Developmental Genetics Imprinting, Babraham Institute, Cambridge <br /> CB22 3AT, UK <br /> 15 <br />    Present address: Guangxi Cancer Institute, Af&#239;&#172;?liated Cancer Hospital, Guangxi Medical University, 530000 Nanning, Guangxi, China <br /> *Correspondence: larsal@sund.ku.dk <br /> DOI 10.1016/j.ajhg.2010.04.011. &#194;&#170;2010 The American Society Human Genetics. All rights reserved. <br />  <br />  <br />  <br />  <br />                                                                    The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 839 <br />  Subjects Methods                                                   Sanger Institute) chosen chromosomes 6 2 tiling <br />                                                                        resolution (set obtained BACPAC Chori, Oakland, CA, USA). <br /> Patients                                                               244K arrays obtained Agilent hybridized according <br /> Informed consent obtained patients legal         manufacturer&#226;&#8364;&#8482;s protocol. Genomic DNA labeled by <br /> guardians investigations patient material anony-        BioPrime Array CGH Genomic Labeling System (Invitrogen, <br /> mous publication. Syndromic CHD patients (Table S1, available          Carlsbad, CA, USA) use Cy3- Cy5-labeled dCTPs <br /> online) followed Pediatric Cardiology <span id='am-323' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>         (20 -deoxycytidine 50 -triphosphate) (Amersham Biosciences, <br /> Clinical Genetics <span id='am-324' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> University Hospitals Leuven (patient     Boston, MA, USA) recommended manufacturer with <br /> A), Clinical Genetics Unit Ghent University Hospital     minor modi&#239;&#172;?cations.15 <br /> (patient F), described (patients B, C, I).12 Patients      Translocation breakpoints mapped &#239;&#172;&#8218;uorescence situ <br /> isolated CHDs (Table S2) followed Pediatric Cardiology     hybridization (FISH) analysis use 200 ng BAC DNA <br /> Unit University Hospitals Leuven collected          according standard procedures. The FISH signals visualized <br /> CONCOR national registry database DNA bank.13 Patients             avidin-FITC detection system. Chromosomes coun- <br /> with CHD translocation identi&#239;&#172;?ed systematic reex-        terstained DAPI (4,6-diamino-2-phenylindole), the <br /> amination carriers balanced translocations.14 Patients      signals investigated Leica DMRB epi&#239;&#172;&#8218;uorescence <br /> isolated CHDs investigated aCGH. These studies             microscope equipped Sensys 1400 CCD camera (Photomet- <br /> were approved local ethics committees.                          rics, Tucson, AZ, USA) IPLab Spectrum imaging software <br />                                                                        (Abbott Laboratories, Abbott Park, IL, USA). <br />  <br /> Tissue Samples Immunohistochemistry Analysis <br /> Human embryonic tissues collected legal abortions.           Candidate-Gene Prioritization <br /> Informed consent obtained according Helsinki Declara-       Candidate-gene prioritization performed use an <br /> tion II. Embryonic age based crown-rump <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> measure-         adapted version Endeavour.16 In brief, data sources for <br /> ment. Tissue samples dissected appropriate tissue blocks     prioritization added regular set data sources: an <br /> and &#239;&#172;?xed 12&#226;&#8364;&#8220;24 hr 4  C 10% neutral buffered formalin, 4%     expression microarray data set murine heart development <br /> Formol-Calcium, Lillie&#226;&#8364;&#8482;s Bouin&#226;&#8364;&#8482;s &#239;&#172;?xatives. The specimens    (GEO accession number GSE1479&#226;&#8364;&#8221;transposed human gene <br /> dehydrated graded alcohols, cleared xylene, embedded       identi&#239;&#172;?ers use BioMart) set representing gene <br /> in paraf&#239;&#172;?n. Serial sections, 3&#226;&#8364;&#8220;5 mm thick, cut transverse,     homology, extracted HomoloGene, BioMart, Inpara- <br /> sagittal, horizontal planes placed silanized slides.         noid. These data summarized vector representations, <br />    Sections deparaf&#239;&#172;?nized rehydrated xylene followed       scoring use Pearson correlation. Addi- <br /> by series graded alcohols accordance established          tionally, prioritizations based data sources displaying <br /> procedures. The sections treated fresh 0.5% solution       strong Spearman rank correlation (&gt; 0.3) fused prior fusing <br /> of hydrogen peroxide methanol 15 min quenching           results prioritizations based independent data <br /> endogenous peroxidase rinsed TRIS buffered            sources (Figure S1). Data sources validated leave-one-out <br /> saline (TBS, 5 mM Tris-HCl, 146 mM NaCl, pH 7.6). Nonspeci&#239;&#172;?c           cross-validation (LOOCV) described previously,16 sources <br /> binding inhibited incubation 30 min blocking           area curve 0.6 omitted (Figure S2). <br /> buffer (ChemMate antibody diluent S2022, DakoCytomation,               Genes prioritized basis seven training sets <br /> Glostrup, Denmark) room temperature. The sections              (Table S5), representing discrete aspects cardiac development <br /> then incubated overnight 4  C polyclonal rabbit antibody     genetics, results seven sets fused the <br /> which speci&#239;&#172;?cally recognizes human TAB2 (MAP3K7IP2),                use rank-order statistics.16 In silico testing LOOCV demon- <br /> immunoblotting immunohistochemistry (ARP32402, Aviva               strated adapted algorithm readily ranks genes an <br /> Systems Biology, 1:1000) blocking buffer (ChemMate antibody         established involvement heart development, average in <br /> diluent S2022, DakoCytomation).                                        5%. All 105 annotated protein-coding genes 6q24- <br />    The sections washed TBS incubated            q25 prioritized. Gene identi&#239;&#172;?cation prioritization was <br /> 30 min peroxidase-labeled anti-rabbit polymer (DAKO             based Ensembl Release 49 Endeavour databases May <br /> EnVision &#195;&#190; System/HRP K4011, DakoCytomation). The sections             2008. <br /> were washed TBS, followed incubation 10 min with <br /> 3,30 -diamino-benzidine chromogen solution. Positive staining <br />                                                                        Zebra&#239;&#172;?sh Assays <br /> was recognized brown color. The sections dehydrated in <br />                                                                        Wild-type (AB) &#239;&#172;&#8218;k-green &#239;&#172;&#8218;uorescent protein (&#239;&#172;&#8218;k-GFP)17 zebra- <br /> graded alcohols followed xylene coverslipped DPX <br />                                                                        &#239;&#172;?sh (Danio rerio) stocks maintained accordance stan- <br /> mounting media. Nonimmune rabbit IgG1 (X0936) staining <br />                                                                        dard aquaculture guidelines. Eggs collected natural <br /> without primary antibody used negative controls. Sections <br />                                                                        mating 30 min laid timed development <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> human embryos containing osteoclasts served positive <br />                                                                        assured. RNA extracted TRIzol, cDNA was <br /> controls. Control sections stained antibody with <br />                                                                        synthesized random primers Superscript III kit (all <br /> a nonimmune rabbit IgG1 blank, stained osteoclasts <br />                                                                        Invitrogen, Merelbeke, Belgium). <br /> were positive. <br />                                                                           Whole-mount situ hybridizations (WISH) wild-type and <br />                                                                        &#239;&#172;&#8218;k-GFP zebra&#239;&#172;?sh embryos performed described previ- <br /> Molecular Cytogenetics                                                 ously18 use sequence-veri&#239;&#172;?ed probe ampli&#239;&#172;?ed by <br /> aCGH performed described15 in-house-created microar-         PCR synthesized cDNA. The resulting embryos were <br /> ray slides, constructed bacterial arti&#239;&#172;?cial chromosome (BAC)      imaged embedded acrylamide-bisacrylamide gel, cryosec- <br /> or P1-derived arti&#239;&#172;?cial chromosome (PAC) probes chosen              tioned 20 mm, mounted vectabond-pretreated micro- <br /> a genome-wide manner 1 Mb spacing (set donated             scope slides. Sections rehydrated phosphate-buffered <br />  <br />  <br />  <br /> 840 The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 <br />                                                                                                                          Figure 1. Fine Mapping Chromosome <br /> A        q24.1             q24.2     q24.3         q25.1             q25.2                   q25.3                      Aberrations 6q24-q25, Candidate- <br />  140Mb                       145Mb            150Mb                       155Mb                        160Mb            Gene Prioritization <br />                                                                                                                         (A) Position deletions patients <br />                                      A                                       J                                          CHD (red bars) CHD <br />                                          B                                             K                                (black bars). Deletion reference letters are <br />                                                           C                                                             shown bars. Regions containing <br />                        I                        D                    G                                                  deletion breakpoints shown gray. <br />                                                                  E                                                      Phenotypes shown Table S1. Deletion <br />                                                F                                                                        F male mother, <br />                                                                              H                                          affected CHDs. The critically <br />                                                                                                                         deleted region CHDs demarcated by <br /> B           149.25                   149.50               149.75                    150.00                              light red box. <br />                                                                                                                         (B) Position deletions the <br />                  UST                                      TAB2             PPIL4      KATNA1   LATS1    PCMT1   LRP11 <br />                                                                                                                         region deleted CHD patients (com- <br />                                                           SUMO1 ZC3H12D            C6orf72      NUP43                   monly deleted region) respect the <br />     A                                                                                                                   11 annotated genes encoded this <br />     B                                                                                                                   region. The deletion patient H <br />     C                                                                                                                   (black, CHD) does affect TAB2 <br />     D                                                                                                                   gene, demarcates critically <br />     E                                                                                                                   deleted region, contains &#239;&#172;?ve genes. <br />     F                                                                                                                   The arrow indicates position the <br />                                                                                                                 H       translocation breakpoint chromosome <br />                                                       N                                                                 6 family N. <br />  <br />  <br />  <br /> saline 0.1% Tween 20 (PBT) incubated 4 hr PBT-                      tional studies, present results mutation <br /> 1% preimmune donkey serum (PDS) room temperature                               analysis. <br /> then overnight 4  C Alexa-647-labeled GFP antibody <br /> (sc-9996 AF647 Santa Cruz Biotechnology, Santa Cruz, CA, <br /> USA; 1/200 PBT-1% PDS). Slides washed times                           Molecular Characterization Deletions 6q24-q25 <br /> PBT, mounted vectashield-DAPI, imaged Olympus                            We observed deletions 6q24-q25 multiple syndromic <br /> FV1000 confocal imaging Olympus AX51 bright-                           CHD patients. In framework aCGH screening <br /> &#239;&#172;?eld imaging. <br />                                                                                       study,4 aCGH 1Mb arrays revealed presence of <br />    Morpholinos (MOs) designed intron 2-exon 3 <br />                                                                                       6q24.2-q25.1 deletion patient A, shown have <br /> splice site (50 -ATC ACT CTT GTT CTG AGG AAA GAA G-3, <br />                                                                                       occurred novo rtqPCR analysis parental DNA. <br /> splice-site-blocking MO [sbMO]) translation initiation <br /> site (50 -ATC TGC TGG TTT CCC TGT GCC ATT C-30 , translation-                         Patient A aortic coarctation, hypoplastic aortic <br /> blocking MO [tbMO]) tab2. Both MOs control MO                                arch, ventricular septal defect, additional <br /> (cMO) (50 -CCT CTT ACC TCA GTT ACA ATT TAT A-30 )                                problems. During routine analysis patients an <br /> ordered Gene Tools (Philomath, OR, USA). Fidelity anno-                       unexplained syndromic disorder, second deletion was <br /> tated nucleotide sequences MO binding sites con&#239;&#172;?rmed                        detected delineated patient Fp use Agi- <br /> direct sequencing. MOs diluted 1/50 Rhodamine solu-                         lent 44K arrays. It shown rtqPCR inherited <br /> tion, estimated 0.5 nL injected one- two-                        mother (Fm). This mother (Fm) diagnosed in <br /> cell stage speci&#239;&#172;?ed doses. tab2 mRNA concentrations                           childhood aortic mitral valve stenosis and <br /> measured real-time quantitative (rtq) PCR described previ- <br />                                                                                       episodes sinus tachycardia. Our studies prompted <br /> ously15 normalized b-actin. <br />                                                                                       perform echocardiographic evaluation patient <br />                                                                                       Fp, revealed mild centrovalvular insuf&#239;&#172;?ciency of <br /> Mutation Analysis                                                                     aortic pulmonary valves. Previously reported inter- <br /> DNA collected patients out&#239;&#172;&#8218;ow tract defect                           stitial 6q deletions (patients B, C, I)12 further <br /> (tetralogy Fallot [ToF], pulmonic stenosis [PS], aortic stenosis                   delineated aCGH 244K arrays. Genotype data on <br /> [AS]). Exons exon-intron boundaries TAB2 ampli&#239;&#172;?ed <br />                                                                                       patients (D, E, G, H, J, K) collected from <br /> by PCR use oligonucleotide primers described Table <br />                                                                                       literature reports12,19&#226;&#8364;&#8220;23 online database car- <br /> S3. PCR products sized DNA gel electrophoresis and <br />                                                                                       diogenetics, CHDWiki.24 The extent deletions and <br /> sequenced BigDye DNA sequencing kit (Applied Biosys- <br /> tems, Foster City, CA, USA).                                                          presence absence CHDs deletion carriers de- <br />                                                                                       picted Figure 1A, detailed phenotype and <br />                                                                                       genotype data provided Table S1. <br /> Results                                                                                  Cardiac defects identi&#239;&#172;?ed deletion patients mostly <br />                                                                                       affected out&#239;&#172;&#8218;ow tract. They included stenosis one <br /> We present identi&#239;&#172;?cation delineation locus                           heart valves, hypoplasia aortic arch, and <br /> CHDs, candidate gene selected                               atrial ventricular septal defects. Patients larger dele- <br /> from locus. We investigated role gene                             tions typically mild mental retardation addi- <br /> heart development expression analyses func-                               tional congenital problems, smaller <br />  <br />  <br />                                                                           The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 841 <br />  deletions (patients Fp, Fm, J) (low) normal mental     A                                 B <br /> development. Genotype comparison CHD patients <br /> revealed commonly deleted region 1.2 Mb, affecting                      At                                             EC <br /> 11 genes (Figure 1B). Deletion H, patient <br /> without CHD, demarcates 850 kb region <br /> deleted CHD patients unaffected patients                                   B <br />  <br /> without CHD. This critically deleted region extends <br /> from 149.09 149.96 Mb (Figure 1B) contains &#239;&#172;?ve                                                                       EC <br />                                                                              Tr <br /> genes unaffected deletion H. <br />  <br />  <br /> Prioritization Genes 6q24-q25 CHD                      C                                  D <br /> The clustering CHD-associated deletions 6q24-q25 <br /> suggested haploinsuf&#239;&#172;?ciency genes                           Ao <br />  <br /> in locus causes CHDs. Although putative CHD                       D           At <br /> gene likely resides commonly deleted region, <br /> we exclude alternative hypothesis                   Ve <br /> or genes located 6q24-q25 cause <br /> CHDs incomplete penetrance. The silico analysis <br /> was extended genes 6q24-q25. In addi- <br /> tion, extension test set enabled better assess- <br /> ment statistical power &#239;&#172;?ndings. Gene prioriti- <br /> zation yielded ranked list candidate genes CHDs,        Figure 2. TAB2 Expression Human Embryonic Hearts <br />                                                                  (A) Section 5.5 wk embryo. TAB2 expression (brown color) is <br /> with MAP3K7IP2 (also known TAB2) ranking &#239;&#172;?rst          prominent ventricular trabeculae endocardial <br /> 105 genes 6q24-q25 (Table S4 Figure S3). The cor-       cushions out&#239;&#172;&#8218;ow tract. <br /> responding p value (4.17 3 10 7) highly signi&#239;&#172;?cant;       (B) Magni&#239;&#172;?cation endocardial cushions. <br /> is located critically deleted region, signi&#239;&#172;?cant   (C) Frontal section 7.5 wk embryo. <br />                                                                  (D) Magni&#239;&#172;?cation showing cytoplasmic expression TAB2 the <br /> prioritization (p &lt; 0.01) obtained gene     endothelial cells lining developing aortic valves. <br /> the commonly deleted region. Upon genome-wide prioriti-          Abbreviations follows: At, atrium; Ao, aorta; EC, endocardial <br /> zation, TAB2 ranks 44th genes          cushions; Tr, trabeculae; Ve, ventricle. <br /> human genome (Table S6). It like likely <br /> candidate gene CHDs.                                            To investigate function tab2 zebra&#239;&#172;?sh <br />                                                                  development, knocked expression using <br />                                                                  independent MOs interfering either <br /> TAB2 Expression Human Embryonic Hearts                        normal splicing translation tab2 RNA. Injection <br /> To explore potential role TAB2 candidate gene      resulted similar developmental abnormalities, <br /> human heart development, analyzed expression           injection cMO did not <br /> human embryos. Immunohistochemistry analysis tissue           result abnormal development. Knockdown thus <br /> sections 5.5 wk 7.5 wk human embryos showed             demonstrated speci&#239;&#172;?c. The &#239;&#172;?rst embryological defects <br /> cytoplasmic expression TAB2 cells ventricular       apparent gastrulation (Figures 4F 4G) <br /> trabeculae (Figure 2A), endothelial cells conotrun-    included delayed epiboly progression convergent <br /> cal cushions out&#239;&#172;&#8218;ow tract (Figures 2A 2B),        extension defects. Later development (36&#226;&#8364;&#8220;48 hpf), <br /> in endothelial cells lining developing aortic valves     severe heart failure apparent (Figures 4D 4E): <br /> (Figures 2C 2D). These results indicate function        heart tube appeared elongated, blood <br /> TAB2 developing human heart.                              pooling common cardinal vein entry into <br />                                                                  heart. <br /> tab2 Zebra&#239;&#172;?sh Assays                                                The molecular effect sbMO injection splicing of <br /> We used zebra&#239;&#172;?sh model organism investigate          nascent tab2 RNA investigated PCR cDNA <br /> role dosage sensitivity tab2 vertebrate devel-     injected embryos (Figure 4B). Sequencing RT-PCR <br /> opment. WISH staged wild-type zebra&#239;&#172;?sh embryos                products revealed transcript lacking wild-type exon <br /> revealed ubiquitous tab2 expression early devel-          3 formed treatment, resulting frameshift <br /> opment (12&#226;&#8364;&#8220;18 hr postfertilization [hpf])             premature stop codon starting seventh nucle- <br /> restricted expression <span id='am-322' about='xsp:pattern' typeof='owl:Thing'>pattern</span> later development,          otide wild-type exon 4 (Figure 4A). This sbMO-generated <br /> with expression developing cardiac out&#239;&#172;&#8218;ow tract,          transcript encodes 95% truncated variant the <br /> the dorsal aorta, posterior cardinal vein (Figure 3).    normal protein. As con&#239;&#172;?rmed similar phenotype <br /> This expression pattern reminiscent expression         observed tbMO injection, represents loss- <br /> during human heart development suggests                 of-function situation. The relative correctly <br /> a conserved role tab2 cardiac out&#239;&#172;&#8218;ow tract.           spliced mRNA assessed rtqPCR cDNA reverse <br />  <br />  <br /> 842 The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 <br />                                                                                         A                          C <br />  <br />  <br />  <br />  <br />                                                                                                                   C&#226;&#8364;&#8482; <br />  <br />      12hpf                        18hpf                     24hpf <br />                                                                                        B <br />  <br />  <br />                                                                                                                   C&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />                                                                                                         C&#226;&#8364;&#8482;&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />  <br />  <br />  <br />      30hpf                        55hpf                     72hpf <br />  <br /> Figure 3. Gene Expression tab2 Developing Zebrafish <br /> mRNA expression tab2 developing zebra&#239;&#172;?sh distinct developmental stages (indicated lower left corner). Boxed parts of <br /> embryos 30 hpf 55 hpf repeated (A) (C000 ), respectively. <br /> (A): Magni&#239;&#172;?cation showing expression tab2 dorsal aorta (arrow) posterior cardinal vein (arrowhead). <br /> (B) Transverse section hindbody zebra&#239;&#172;?sh 30 hpf depicted (A), showing restricted expression dorsal aorta <br /> (arrowhead) posterior cardinal vein (arrow). <br /> (C): Bright-&#239;&#172;?eld image expression tab2 cardiac out&#239;&#172;&#8218;ow tract 55 hpf (arrow). <br /> (C0 ) DAPI staining section depicted (C) (pseudocolored blue). <br /> (C00 ) The section depicted (C), immunostained Alexa-647-labeled GFP antibody (pseudocolored red), displaying position of <br /> the cardiac out&#239;&#172;&#8218;ow tract (arrow). <br /> (C000 ) Image 55 hpf, whole-mount-stained embryo, presenting orientation section shown (C), (C0 ), (C00 ) (white line). <br />  <br />  <br /> transcribed RNA extracted sbMO-                      protein function (Figure 5C). Neither DNA samples nor <br /> injected embryos 20 hpf. This revealed sbMO injec-              phenotypic data available family members of <br /> tion resulted dose-dependent reduction normal                   patients L M. These mutations were, however, absent <br /> splicing (Figure 4C). Phenotypes apparent dose              658 ethnically matched control chromosomes. <br /> of 2 3 ng, corresponding 41% 58% reduction of <br /> normal expression, showing haploinsuf&#239;&#172;?ciency <br />                                                                         Investigation Family t(2;6) Translocation <br /> of tab2 causes developmental defects (Figures 4C&#226;&#8364;&#8220;4I). <br />                                                                         CHD <br />                                                                         A family CHD segregating t(2;6)(q21;q25) <br /> Sequence Analysis TAB2 Human Patients                             translocation previously identi&#239;&#172;?ed.14 In family <br /> with Out&#239;&#172;&#8218;ow Tract Defects                                               (family N), translocation carriers history <br /> Because TAB2 relevant spatiotemporal expression                   CHD (Figure 5D). Patient N-II.2 diagnosed with <br /> pattern dosage-sensitive role development,                  aortic stenosis atrial &#239;&#172;?brillation age 67 required <br /> was good candidate gene explaining CHDs                  aortic valve replacement. Patient N-III.1 diagnosed <br /> evident deletion 6q24-q25. To con&#239;&#172;?rm                    aortic stenosis age 2 yrs. Her aortic valve was <br /> the role TAB2 CHDs, analyzed DNA sequence                  surgically replaced age 25. At age 34 hospitalized <br /> of TAB2 402 patients out&#239;&#172;&#8218;ow tract defects                       paroxysmal supraventricular tachycardia. She died at <br /> (Table S2). These patients analyzed aCGH.                   age 49 myocardial infarction. Patient N-III.2 was <br /> Sequencing analysis revealed heterozygous missense                  diagnosed&#226;&#8364;&#8221; like mother sister&#226;&#8364;&#8221;with cardiac <br /> mutations TAB2 gene. One female (patient L) carried              rhythm problems (sick sinus syndrome, tachycardia). <br /> a c.622C&gt;T mutation, causing p.Pro208Ser mutation                     However, refused participate clinical exam- <br /> at protein level (Figures 5A 5C). She left-               inations investigated echocardiography. <br /> ventricular out&#239;&#172;&#8218;ow tract obstruction, subaortic stenosis                 The translocation segregating family bal- <br /> (subAS) &#239;&#172;?bromuscular shelf, residual aortic regur-             anced, demonstrated aCGH 1Mb arrays on <br /> gitation, atrial &#239;&#172;?brillation died 61 years age            chromosome 6 2 tiling microarrays (data shown). <br /> because heart failure. Patient M carried c.688C&gt;A                  FISH studies positioned translocation breakpoint <br /> mutation, causing p.Gln230Lys mutation protein                 2q21 14.2 kb region chromosome position <br /> level (Figure 5B). He bicuspid aortic valve                131,691,033-131,705,253, disrupting POTEE gene <br /> aortic dilation. Both mutations alter highly conserved resi-            (encoding prostate, ovary, testis-expressed protein E). <br /> dues predicted detrimental effects                   The position translocation breakpoint 6q25 <br />  <br />  <br />                                                                 The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 843 <br />  A                                   1105kb                                   1110kb                   1115kb                      1120kb <br />      chr20 <br />                                                                      tbMO sbMO <br />       tab2 <br /> isoform 1 <br /> isoform 2 <br />     sbMO <br />  injec on <br />  <br />  <br />                                                            100% <br />                                     wt-spliced tab2 mRNA <br />                  marker <br />           sbMO <br />     wt <br />  <br />  <br />  <br />  <br /> B                               C                                                                 D                          D&#226;&#8364;&#8482; <br />                                      rela ve quan ty of <br />  <br />                                                            80% <br />                           2 <br />                           1.5                              60% <br />                           1 <br />                           0.8                              40% <br />                           0.6                                                                     E                          E&#226;&#8364;&#8482; <br />                                                            20% <br />                           0.4 <br />                                                             0% <br />                                                                    0 0.5 1 2 3 <br />                           0.2                                          ng MO injected <br />  <br />  <br />  <br /> F                                                                                                       G <br />                                                                                                                             12hpf <br />                                                                                                         100% <br />                                                                                                             80% <br />                                                                                                             60% <br />                                                                                                             40% <br />                                                                                                             20% <br />                                                                                                              0% <br />                                                                                                                      4ng 2ng 1ng 0ng <br />          Class I                    Class II                      Class II            Class III                          C-I C-II C-III <br />  <br /> H                                                                                                       I                   20hpf <br />                                                                                                             100% <br />                                                                                                             80% <br />                                                                                                             60% <br />                                                                                                             40% <br />                                                                                                             20% <br />                                                                                                              0% <br />      Class I                    Class II                           Class III          Class IV                           4ng 2ng 1ng 0ng <br />                                                                                                                    C-I      C-II C-III C-IV <br /> Figure 4. tab2 Knockdown Zebrafish <br /> (A) Schematic organization tab2 gene zebra&#239;&#172;?sh genome. Two alternative splice isoforms detected PCR reverse- <br /> transcribed mRNA shown. Introns dotted lines, exons boxes (coding sequences black, noncoding sequences orange). The <br />  <br />  <br />  <br /> 844 The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 <br />  A                    C                                                                   D                                            I <br />  <br />  <br />  <br />  <br />                                             208 <br />  <br />  <br />  <br />  <br />                                                                         230 <br />                     Human IHGVPPPVLNSPQGNSIYIRP YITTPGGTTRQTQQHSGWVSQFN P <br /> TAACAGTCCACAGGG <br />          c.622C&gt;T       Cat IHGVPPPVLNSPQGNSIYIRP YITTPGGTARQTQQHSGWVSQFN P                       1                 2   3        4 <br />                      Mouse IHGVPPPVLNSPQGNSIYIRP YITTPSGTARQTQQHSGWVSQFN P <br />                                                                                                                                       II <br />                   Armadillo IHGVPPPVLNSPQGNSIYIRP YITTPSGTARQTQQHSGWVSQFN P <br />                    Platypus IHGVPPPALNSPQGNSIYIRP YITAPSGTSRQAQQPPGWVSQLSP                                              * <br />                     Chicken IHGVPPPVLNSPQGNSIYIRP YITAPSGTARQTQQQAGWASQFNP <br />                                              5 <br />  <br /> B               Zebra Finch IHGVPPPVLNSPQGNSIYIRP YITAPSGTARQTQQQPGWASQFNP                                          1       2    3 <br />                    Xenopus IHGVPPPILNSPHGNSIYIRP YVTSQSGTARQAQQSPSWVSHN-P                                                             III <br />                  Tetraodon IHGGPQSGLSSPLGNSIYIRP FVS-QSGSSRLSQQ-QGGRAQYSP                                       *       * <br />                                                                                                Aor c stenosis <br /> ACAGACACAACAGCA        Fugu IHGGPQSGLSSPQGNSIYIRP YVS-QSGTSRLNQQ-QGGRAQYSP                     Aor c valve <br />                   Zebra&#239;&#172;?sh IHGGPQSGLNSP--NSIYIRP YVT-QPGSTRQVQC----RAQYSP                      replacement <br />         c.688C&gt;A Muta ons              S                      K                                Arrhythmia                             IV <br />                                                                                                Transloca on <br />                                                                                            *   t(2:6) carrier <br />  <br />  <br /> Figure 5. TAB2 Is Mutated Disrupted CHD Patients <br /> (A B) Partial TAB2 reference-sequence-read traces corresponding traces missense mutations identi&#239;&#172;?ed patients L M <br /> (phenotypes detailed main text). <br /> (C) Conservation mutated residues tetrapod &#239;&#172;?sh lineages. TAB2 lower lineages; mutated residues are <br /> not conserved paralogous TAB3. <br /> (D) Pedigree family N. Translocation carriers marked, breakpoint chromosome 6 disrupts TAB2. Phenotypes as <br /> annotated insert detailed main text. <br />  <br />  <br /> lies 17 kb region chromosome position                    preferences. TAB2 ranked &#239;&#172;?rst (p &#194;&#188; 4.17 3 10 7) <br /> 149,678,240-149,695,219, &#239;&#172;?rst intron, &#239;&#172;?rst                candidate genes (Figure 1C). Importantly, <br /> exon, promoter region TAB2 (Figure S4).                        genes commonly deleted <br />                                                                      region generated signi&#239;&#172;?cant signal involvement <br />                                                                      heart development use method (p &gt; <br /> Discussion                                                           0.03 all). <br />                                                                         Support involvement TAB2 human heart <br /> We identi&#239;&#172;?ed locus CHDs 6q24-q25,                  development provided expression of <br /> deleted seven individuals CHD (Figure 1A). In              gene developing human zebra&#239;&#172;?sh embryos. <br /> search genes causing CHDs region, prior-           TAB2 expressed endothelial lining devel- <br /> itized 105 genes chromosome 6q24 6q25               oping human heart: detected expression endothelial <br /> their involvement heart development, using tool                 cells lining trabeculae developing aortic valves <br /> based established genomic data fusion algorithm.16             (Figure 2). Endocardial cushions involved endothe- <br /> This strategy ensures gene selection in&#239;&#172;&#8218;uenced               lial-mesenchymal transformation, important event <br /> to minor extent researcher&#226;&#8364;&#8482;s prior knowledge             development cardiac valves out&#239;&#172;&#8218;ow <br />  <br />  <br /> target sites sbMO) tbMO indicated double arrowheads. The lower schema indicates effect sbMO injection the <br /> mature tab2 mRNA (isoforms 1 2). <br /> (B) Gel electrophoresis product PCR cDNA extracted nonmorphant (WT) 2 ng sbMO-injected zebra&#239;&#172;?sh embryos <br /> 24 hpf DNA marker size comparison, showing band 2000 bp wild-type situation, additional frag- <br /> ments morphant situation. <br /> (C) Effect TAB2 sbMO injection correctly spliced mRNA. <br /> (D) Wild-type AB embryo 48 hpf. <br /> (D0 ) Noninjected embryo 48 hpf expressing GFP vascular endothelial cells in&#239;&#172;&#8218;uence &#239;&#172;&#8218;k promoter (&#239;&#172;&#8218;k-GFP). <br /> (E) Morphant embryo 48 hpf. Note enlarged pericardial sac (arrow). <br /> (E0 ) Morphant &#239;&#172;&#8218;k-GFP embryo 48 hpf. Note elongated heart. <br /> (F) Phenotypical classi&#239;&#172;?cation sbMO-injected zebra&#239;&#172;?sh embryos 12 hpf. Defects epiboly progression evident, progression <br /> of yolk-syncytial layer schematically displayed below. Phenotypes ordered severe normal. Class I: anterior-posterior <br /> gradient evident. Class II: yolk-syncytial layer progresses vegetal pole; different embryos shown <br /> to illustrate continuum phenotypes class II. Class III: wild-type. Left, posterior; up, dorsal. <br /> (G) Distribution phenotypes 12 hpf dependent sbMO dose injected. Y axis: percentage embryos class indi- <br /> cated color <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span>. X axis: TAB2 sbMO injected one-cell stage. At 42 embryos successfully injected for <br /> each dose. In total, 230 embryos successfully injected. <br /> (H) Phenotypical classi&#239;&#172;?cation sbMO-injected zebra&#239;&#172;?sh embryos 24 hpf. Phenotypes ordered severe normal. Class I: <br /> death. Other classes described main text. Lateral images: up, anterior. <br /> (I) Distribution phenotypes 24 hpf dependent sbMO dose injected. Y axis: percentage embryos class. X axis: <br /> amount TAB2 sbMO injected one-cell stage. At 42 embryos successfully injected dose. In total, over <br /> 230 embryos successfully injected. Injection 4 ng cMO did increase frequency abnormal phenotypes comparison <br /> to noninjected 0 ng-injected control embryos (not shown). <br />  <br />  <br />  <br />                                                              The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 845 <br />  tract.25 The conservation Tab2 expression devel-        abnormal somites, small head comparison to <br /> oping cardiovascular mouse26 zebra-         control larvae. However, early defects observed these <br /> &#239;&#172;?sh (Figure 3) supports functional relevance       embryos preclude conclusive analysis involve- <br /> TAB2 cardiovascular development.                               ment tab2 heart development: hearts <br />    TAB2 (TAK1 binding protein 2) encodes protein        morphant embryos abnormally structured and <br /> studied mainly role in&#239;&#172;&#8218;ammatory response.          heart failure evident enlarged pericardial sac <br /> TAB2 causes autophosphorylation activation                 (Figure 4E), exclude effect sec- <br /> mitogen-activated protein kinase kinase kinase 7 (MAP3K7,         ondary earlier defects gastrulation. Other reports <br /> also known TAK1 [MIM 602614]),27 (through media-          similarly shown <span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>knockout</span> genes involved in <br /> tors TNF-receptor-associated factors [TRAFs]          isolated CHDs humans cause pleio- <br /> receptor-interacting proteins [RIPs]) relaying signals       tropic phenotype zebra&#239;&#172;?sh.33 This probably attribut- <br /> receptors chemokines (TNF, LPS, IL1) extracel-      able species-speci&#239;&#172;?c differences; example, the <br /> lular signaling molecules (TGF-b Wnt) downstream           epibolic movements direct correlate in <br /> proteins IKK, p38, JNK, RCAN, NLK. TAK1 conse-        mammalian development. Nevertheless, clear dosage <br /> quently modulates activity molecules NF-kB,        sensitivity tab2 zebra&#239;&#172;?sh demonstrates that <br /> b-catenin, NFAT, HDAC3 downstream target            proper tab2 dosage critical normal development <br /> genes. Tak1 /  mice die embryonic day 10.5                 vertebrates. <br /> (E10.5) E12.5 neural tube cardiovascular             Because &#239;&#172;?ndings con&#239;&#172;?rmation of <br /> defects.28,29 Xie colleagues (2006)30 generated mice          role TAB2 CHD, investigated presence of <br /> with cardiac-speci&#239;&#172;?c expression dominant-negative          germline mutations CHD patients. This revealed two <br /> Tak1, showing inhibiting Tak1 function heart          missense mutations affecting evolutionarily highly con- <br /> leads altered electrical conduction (shortened PR interval),   served residues (Figures 5A&#226;&#8364;&#8220;5C). These conserved residues <br /> impaired ventricular &#239;&#172;?lling, cardiac hyperthropy. Inter-      recognized protein domain, affect <br /> estingly, authors discussed unpublished results     hitherto described protein binding sites; future <br /> showed cardiac-speci&#239;&#172;?c deletion Tak1 causes midges-       studies need establish function conserved <br /> tation death.30 Zhang colleagues (2000)31 reported         domain. The observed paucity mutations unex- <br /> mice cardiac-speci&#239;&#172;?c expression activated Tak1,           pected given known etiological genetic heteroge- <br /> showing develop cardiac hypertrophy. Combined,          neity CHDs: mutations genes established <br /> these results demonstrate altered Tak1 signaling      role CHD development similarly small <br /> heart leads cardiac disorders. Of interest, aCGH        (&lt; 1%) fraction CHD patients.9 Furthermore, not <br /> screening study led identi&#239;&#172;?cation presented    CHD phenotypes investigated, exclude <br /> TAB2 deletion, identi&#239;&#172;?ed 650 kb TAK1 duplication          TAB2 mutations associated additional CHD <br /> a girl pulmonary stenosis (data shown). This           phenotypes. Moreover, identi&#239;&#172;?ed family which <br /> duplication inherited mother,             CHD cardiomyopathy cosegregate balanced <br /> cardiac defect. Her brother died early childhood           translocation 2q21 6q25. Family members <br /> from severe pulmonary valve stenosis well,            carry translocation CHD. <br /> DNA available study.                                      The breakpoint 2q21 disrupts POTEE gene (Figure S4), <br />    Tab2 /  mice die E12.5 display severe liver         seven poorly conserved paralogues recently <br /> degeneration increased hepatocyte apoptosis. No              expanded gene family.34 Given genes not <br /> immunological hematological problems described            expressed mammalian heart POTEE vari- <br /> in patients TAB2 disruption. Interestingly, however,         able copy number normal human population, it <br /> 70% Tab2&#195;&#190;/  mice die 1 wk birth. The              unlikely disruption POTEE causes observed <br /> reason increased mortality established.32        CHDs family. The breakpoint 6q25 located <br /> Although obviously precludes direct phenotypic             TAB2, leads disruption, support- <br /> comparison human hemizygous patients, does                ing role TAB2 pathogenesis CHD. Taken <br /> demonstrate Tab2 dosage sensitive mouse            together, results demonstrate perturbations and <br /> and excellent candidate explaining CHDs             mutations TAB2 causing CHDs. <br /> observed patients.                                            Although described CHDs out&#239;&#172;&#8218;ow tract <br />    To investigate dosage sensitivity tab2,      defects, exclude phenotypic spectrum <br /> titrated knockdown tab2 gene expression              extends further. Indeed, patient C presented atrial <br /> MO injection. We observed phenotypic changes                 septum defect (ASD) patient E ventricular septal <br /> the fraction normal tab2 mRNA approximately halved,            defect (VSD) (Table S1). This surprising, given <br /> demonstrating tab2 gene dosage sensitive         large variation CHD phenotypes been <br /> in zebra&#239;&#172;?sh. The morphants displayed early embryo-            described genetic syndromes. A classic example <br /> logical defects (delayed epibolic movements gastru-        22q11 deletion syndrome, CHD phenotypes <br /> lation; Figure 4F) developed larvae         include Tetralogy Fallot (ToF), pulmonary stenosis (PS), <br /> were shorter dysfunctional heart, curved tail,        interrupted aortic arch, VSD, ASD, cardiac <br />  <br />  <br /> 846 The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 <br />  malformations.35 Other examples include Wolf-Hirchhorn                  CHDWiki, homes.esat.kuleuven.be/~bioiuser/chdwiki <br /> syndrome, patients ASD, PS, VSD,                   Endeavour, www.esat.kuleuven.be/endeavour <br /> patent ductus arteriosus,36 Smith-Magenis syndrome,                 Gene expression omnibus (GEO), www.ncbi.nlm.nih.gov/geo <br /> in valvular abnormalities, ASD, VSD, ToF                      Homologene, www.ncbi.nlm.nih.gov/homologene <br />                                                                         Inparanoid, inparanoid.sbc.su.se <br /> have reported.37 Variable CHD phenotypes also <br />                                                                         Online Mendelian Inheritance Man (OMIM), www.ncbi.nlm. <br /> observed patients gene mutations. Examples are <br />                                                                           nih.gov/Omim <br /> plenty include patients JAG1 mutations, in <br /> which PS, aortic stenosis, VSD, ASD re- <br /> ported,38 patients CITED2 mutations,                   Accession Numbers <br /> present ASD, VSD, ToF, transposition great <br />                                                                         The GenBank accession numbers human TAB2 cDNA and <br /> arteries.39 Patients NOTCH1 mutations present <br />                                                                         protein sequences reported paper NM_015093.3 and <br /> with AS, VSD, ToF, double outlet right ventricle, but <br />                                                                         NP_055908.1. Human zebra&#239;&#172;?sh genome coordinates are <br /> they risk late-onset complications as <br />                                                                         numbered according March 2006 human reference <br /> aortic valve calci&#239;&#172;?cation AS, requiring valve replace-              sequence (NCBI build 36.1) July 2007 zebra&#239;&#172;?sh (Danio re- <br /> ment,40 observed patient N-II.3.                              rio) Zv7 assembly. <br />    In conclusion, demonstrated ability posi- <br /> tional cloning strategy identi&#239;&#172;?cation genes <br /> involved human heart defects. The strategy presented                 References <br /> here enables rapid, generic, powerful positional <br />                                                                          1. Hoffman, J.I., Kaplan, S. (2002). The incidence congen- <br /> cloning led identi&#239;&#172;?cation gene involved                     ital heart disease. J. Am. Coll. Cardiol. 39, 1890&#226;&#8364;&#8220;1900. <br /> in CHD, TAB2. This gene expressed developing                   2. Thom, T., Haase, N., Rosamond, W., Howard, V.J., Rumsfeld, J., <br /> heart, dosage sensitive zebra&#239;&#172;?sh development,                     Manolio, T., Zheng, Z.J., Flegal, K., O&#226;&#8364;&#8482;Donnell, C., Kittner, S., <br /> is mutated, deleted, disrupted translocation                     et al., American Heart Association Statistics Committee and <br /> CHD patients.                                                               Stroke Statistics Subcommittee. (2006). Heart disease and <br />                                                                             stroke statistics&#226;&#8364;&#8221;2006 update: report American <br /> Supplemental Data                                                           Heart Association Statistics Committee Stroke Statistics <br /> Supplemental Data include &#239;&#172;?gures tables             Subcommittee. Circulation 113, e85&#226;&#8364;&#8220;e151. <br /> found article online http://www.ajhg.org.                   3. Jenkins, K.J., Correa, A., Feinstein, J.A., Botto, L., Britt, A.E., <br />                                                                             Daniels, S.R., Elixson, M., Warnes, C.A., Webb, C.L., <br />                                                                             American Heart Association Council Cardiovascular <br /> Acknowledgments                                                             Disease Young. (2007). Noninherited risk factors and <br />                                                                             congenital cardiovascular defects: current knowledge: scien- <br /> B.T. supported Ph.D. fellowship Agentschap voor <br />                                                                             ti&#239;&#172;?c statement American Heart Association Council <br /> Innovatie door Wetenschap en Technologie (IWT). P.V.L. post- <br />                                                                             Cardiovascular Disease Young: endorsed the <br /> doctoral researcher, K.D. senior clinical investigator, J.B. an <br />                                                                             American Academy Pediatrics. Circulation 115, 2995&#226;&#8364;&#8220;3014. <br /> aspirant investigator Research Foundation-Flanders (FWO). <br />                                                                          4. Calcagni, G., Digilio, M.C., Sarkozy, A., Dallapiccola, B., and <br /> L.A.L. supported The Danish Heart Foundation The <br />                                                                             Marino, B. (2007). Familial recurrence congenital heart <br /> Novo Nordisk Foundation. This work supported OT/O2/ <br />                                                                             disease: overview review literature. Eur. J. Pediatr. <br /> 40, GOA/2006/12, Centre Excellence SymBioSys (EF/05/007) <br />                                                                             166, 111&#226;&#8364;&#8220;116. <br /> from University Leuven Belgian program of <br />                                                                          5. Manning, N., Archer, N. (2006). A study determine the <br /> Interuniversity Poles Attraction (IUAP), ProMeta, GOA Ambior- <br />                                                                             incidence structural congenital heart disease monochor- <br /> ics, GOA MaNet, START 1, FWO (G.0318.05, G.0254.05, <br />                                                                             ionic twins. Prenat. Diagn. 26, 1062&#226;&#8364;&#8220;1064. <br /> G.0553.06, G.0302.07, ICCoS, ANMMM, MLDM, G.0733.09, <br />                                                                          6. Pierpont, M.E., Basson, C.T., Benson, D.W. Jr., Gelb, B.D., <br /> G.082409), IWT (Silicos, SBO-BioFrame, SBO-MoKa, TBM-IOTA3), <br />                                                                             Giglia, T.M., Goldmuntz, E., McGee, G., Sable, C.A., Srivas- <br /> IUAP P6/25 BioMaGNet, ERNSI (FP7-HEALTH CHeartED). The <br />                                                                             tava, D., Webb, C.L., American Heart Association Congen- <br /> Wilhelm Johannsen Centre Functional Genome Research is <br />                                                                             ital Cardiac Defects Committee, Council Cardiovascular <br /> established Danish National Research Foundation. We thank <br />                                                                             Disease Young. (2007). Genetic basis congenital <br /> D. Stainier sharing transgenic &#239;&#172;&#8218;k-GFP zebra&#239;&#172;?sh, S. For- <br />                                                                             heart defects: current knowledge: scienti&#239;&#172;?c statement from <br /> schhammer A. Ilgun technical assistance, Leuven <br />                                                                             American Heart Association Congenital Cardiac Defects <br /> Aquatic Facility excellent &#239;&#172;?sh care. <br />                                                                             Committee, Council Cardiovascular Disease Young: <br />                                                                             endorsed American Academy Pediatrics. Circulation <br /> Received: November 15, 2009 <br />                                                                             115, 3015&#226;&#8364;&#8220;3038. <br /> Revised: April 12, 2010 <br />                                                                          7. Burn, J., Brennan, P., Little, J., Holloway, S., Coffey, R., Somer- <br /> Accepted: April 20, 2010 <br />                                                                             ville, J., Dennis, N.R., Allan, L., Arnold, R., Dean&#239;&#172;?eld, J.E., et al. <br /> Published online: May 20, 2010 <br />                                                                             (1998). Recurrence risks offspring adults major heart <br />                                                                             defects: results &#239;&#172;?rst cohort British collaborative study. <br /> Web Resources                                                               Lancet 351, 311&#226;&#8364;&#8220;316. <br />                                                                          8. Whittemore, R., Wells, J.A., Castellsague, X. (1994). A second- <br /> The URLs data presented follows: <br />                                                                             generation study 427 probands congenital heart defects <br /> BioMart, www.ensembl.org/biomart                                            837 children. J. Am. Coll. Cardiol. 23, 1459&#226;&#8364;&#8220;1467. <br />  <br />  <br />  <br />                                                                The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 847 <br />   9. Posch, M.G., Perrot, A., Schmitt, K., Mittelhaus, S., Esenwein,                 abnormal karyotype, mental retardation, dysmorphic <br />     E.M., Stiller, B., Geier, C., Dietz, R., Gessner, R., Ozcelik, C.,              features. Am. J. Med. Genet. A. 140, 2180&#226;&#8364;&#8220;2187. <br />     Berger, F. (2008). Mutations GATA4, NKX2.5, CRELD1,                21.   Osoegawa, K., Vessere, G.M., Utami, K.H., Mansilla, M.A., <br />     BMP4 infrequently patients congenital                     Johnson, M.K., Riley, B.M., L&#226;&#8364;&#8482;Heureux, J., Pfundt, R., Staaf, J., <br />     cardiac septal defects. Am. J. Med. Genet. A. 146A, 251&#226;&#8364;&#8220;253.                    van der Vliet, W.A., et al. (2008). Identi&#239;&#172;?cation novel candi- <br /> 10. Erdogan, F., Larsen, L.A., Zhang, L., Tu  &#194;&#168; mer, Z., Tommerup, N.,              date genes associated cleft lip palate using array <br />     Chen, W., Jacobsen, J.R., Schubert, M., Jurkatis, J., Tzschach,                 comparative genomic hybridisation. J. Med. Genet. 45, 81&#226;&#8364;&#8220;86. <br />     A., et al. (2008). High frequency submicroscopic genomic               22.   Nagamani, S.C., Erez, A., Eng, C., Ou, Z., Chinault, C., <br />     aberrations detected tiling path array comparative genome                    Workman, L., Coldwell, J., Stankiewicz, P., Patel, A., Lupski, <br />     hybridisation patients isolated congenital heart                        J.R., Cheung, S.W. (2009). Interstitial deletion of <br />     disease. J. Med. Genet. 45, 704&#226;&#8364;&#8220;709.                                            6q25.2-q25.3: novel microdeletion syndrome associated <br /> 11. Thienpont, B., Mertens, L., Ravel, T., Eyskens, B., Boshoff,                 microcephaly, developmental delay, dysmorphic features <br />     D., Maas, N., Fryns, J.P., Gewillig, M., Vermeesch, J.R.,                   hearing loss. Eur. J. Hum. Genet. 17, 573&#226;&#8364;&#8220;581. <br />     Devriendt, K. (2007). Submicroscopic chromosomal imbal-                   23.   Pirola, B., Bortotto, L., Giglio, S., Piovan, E., Janes, A., Guer- <br />     ances detected array-CGH frequent cause con-                        rini, R., Zuffardi, O. (1998). Agenesis corpus cal- <br />     genital heart defects selected patients. Eur. Heart J. 28,                   losum Probst bundles owing haploinsuf&#239;&#172;?ciency a <br />     2778&#226;&#8364;&#8220;2784.                                                                      gene 8 cM region 6q25. J. Med. Genet. 35, 1031&#226;&#8364;&#8220;1033. <br /> 12. Nowaczyk, M.J., Carter, M.T., Xu, J., Huggins, M., Raca, G.,              24.   Barriot, R., Breckpot, J., Thienpont, B., Brohe&#194;&#180;e, S., Van Vooren, <br />     Das, S., Martin, C.L., Schwartz, S., Rosen&#239;&#172;?eld, R., Wagg-                   S., Coessens, B., Tranchevent, L.C., Van Loo, P., Gewillig, M., <br />     oner, D.J. (2008). Paternal deletion 6q24.3: new congenital                   Devriendt, K., Moreau, Y. (2010). Collaboratively charting <br />     anomaly syndrome associated intrauterine growth                            gene-to-phenotype network human congenital heart <br />     failure, early developmental delay characteristic facial                    defects. Genome Med 2, 16. <br />     appearance. Am. J. Med. Genet. A. 146, 354&#226;&#8364;&#8220;360.                           25.   Kirby, M.L. (2007). Cardiac Development (Oxford, New York: <br /> 13. van der Velde, E.T., Vander, V.E., Vriend, J.W., Mannens,                       Oxford University Press). <br />     M.M., Uiterwaal, C.S., Brand, R., Mulder, B.J. (2005).                26.   Orelio, C., Dzierzak, E. (2003). Identi&#239;&#172;?cation 2 novel <br />     CONCOR, initiative national registry                           genes developmentally regulated mouse aorta-gonad- <br />     DNA-bank patients congenital heart disease                       mesonephros region. Blood 101, 2246&#226;&#8364;&#8220;2249. <br />     Netherlands: rationale, design, &#239;&#172;?rst results. Eur. J. Epide-          27.   Xia, Z.P., Sun, L., Chen, X., Pineda, G., Jiang, X., Adhikari, A., <br />     miol. 20, 549&#226;&#8364;&#8220;557.                                                              Zeng, W., Chen, Z.J. (2009). Direct activation protein <br /> 14. Bache, I., Hjorth, M., Bugge, M., Holstebroe, S., Hilden, J.,                   kinases unanchored polyubiquitin chains. Nature 461, <br />     Schmidt, L., Brondum-Nielsen, K., Bruun-Petersen, G., Jensen,                   114&#226;&#8364;&#8220;119. <br />     P.K., Lundsteen, C., et al. (2006). Systematic re-examination          28.   Shim, J.H., Xiao, C., Paschal, A.E., Bailey, S.T., Rao, P., Hayden, <br />     carriers balanced reciprocal translocations: strategy                   M.S., Lee, K.Y., Bussey, C., Steckel, M., Tanaka, N., et al. (2005). <br />     search candidate regions common complex                             TAK1, TAB1 TAB2, plays essential role multiple <br />     diseases. Eur. J. Hum. Genet. 14, 410&#226;&#8364;&#8220;417.                                      signaling pathways vivo. Genes Dev. 19, 2668&#226;&#8364;&#8220;2681. <br /> 15. Menten, B., Maas, N., Thienpont, B., Buysse, K., Vandesom-                29.   Jadrich, J.L., O&#226;&#8364;&#8482;Connor, M.B., Coucouvanis, E. (2006). The <br />     pele, J., Melotte, C., Ravel, T., Van Vooren, S., Balikova, I.,              TGF beta activated kinase TAK1 regulates vascular develop- <br />     Backx, L., et al. (2006). Emerging patterns cryptic chromo-                  ment vivo. Development 133, 1529&#226;&#8364;&#8220;1541. <br />     somal imbalance patients idiopathic mental retarda-               30.   Xie, M., Zhang, D., Dyck, J.R., Li, Y., Zhang, H., Morishima, <br />     tion multiple congenital anomalies: new series 140                     M., Mann, D.L., Taffet, G.E., Baldini, A., Khoury, D.S., and <br />     patients review published reports. J. Med. Genet. 43,                    Schneider, M.D. (2006). A pivotal role endogenous TGF- <br />     625&#226;&#8364;&#8220;633.                                                                        beta-activated kinase-1 LKB1/AMP-activated protein <br /> 16. Aerts, S., Lambrechts, D., Maity, S., Van Loo, P., Coessens, B.,                kinase energy-sensor pathway. Proc. Natl. Acad. Sci. USA <br />     De Smet, F., Tranchevent, L.C., De Moor, B., Marynen, P., Has-                  103, 17378&#226;&#8364;&#8220;17383. <br />     san, B., et al. (2006). Gene prioritization genomic               31.   Zhang, D., Gaussin, V., Taffet, G.E., Belaguli, N.S., Yamada, M., <br />     data fusion. Nat. Biotechnol. 24, 537&#226;&#8364;&#8220;544.                                      Schwartz, R.J., Michael, L.H., Overbeek, P.A., Schneider, <br /> 17. Motoike, T., Loughna, S., Perens, E., Roman, B.L., Liao, W.,                    M.D. (2000). TAK1 activated myocardium pres- <br />     Chau, T.C., Richardson, C.D., Kawate, T., Kuno, J., Weinstein,                  sure overload suf&#239;&#172;?cient provoke heart failure trans- <br />     B.M., et al. (2000). Universal GFP reporter study                    genic mice. Nat. Med. 6, 556&#226;&#8364;&#8220;563. <br />     vascular development. Genesis 28, 75&#226;&#8364;&#8220;81.                                  32.   Sanjo, H., Takeda, K., Tsujimura, T., Ninomiya-Tsuji, J., Matsu- <br /> 18. Thisse, C., Thisse, B. (2008). High-resolution situ                      moto, K., Akira, S. (2003). TAB2 essential prevention <br />     hybridization whole-mount zebra&#239;&#172;?sh embryos. Nat. Pro-                        apoptosis fetal liver interleukin-1 signaling. <br />     toc. 3, 59&#226;&#8364;&#8220;69.                                                                  Mol. Cell. Biol. 23, 1231&#226;&#8364;&#8220;1238. <br /> 19. Caselli, R., Mencarelli, M.A., Papa, F.T., Uliana, V., Schiavone,         33.   Roessler, E., Ouspenskaia, M.V., Karkera, J.D., Ve&#194;&#180;lez, J.I., Kant- <br />     S., Strambi, M., Pescucci, C., Ariani, F., Rossi, V., Longo, I., et al.         ipong, A., Lacbawan, F., Bowers, P., Belmont, J.W., Towbin, <br />     (2007). A 2.6 Mb deletion 6q24.3-25.1 patient                      J.A., Goldmuntz, E., et al. (2008). Reduced NODAL signaling <br />     growth failure, cardiac septal defect, upperlip asym-                  strength mutation pathway members including <br />     metric dysmorphic ears. Eur. J. Med. Genet. 50, 315&#226;&#8364;&#8220;321.                        FOXH1 linked human heart defects holoprosence- <br /> 20. Bisgaard, A.M., Kirchhoff, M., Tu &#194;&#168; mer, Z., Jepsen, B., Br&#195;&#184;ndum-               phaly. Am. J. Hum. Genet. 83, 18&#226;&#8364;&#8220;29. <br />     Nielsen, K., Cohen, M., Hamborg-Petersen, B., Bryndorf, T.,               34.   Bera, T.K., Zimonjic, D.B., Popescu, N.C., Sathyanarayana, <br />     Tommerup, N., Skovby, F. (2006). Additional chromo-                         B.K., Kumar, V., Lee, B., Pastan, I. (2002). POTE, a <br />     somal abnormalities patients previously detected                      highly <span id='am-319' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> family located numerous <br />  <br />  <br />  <br /> 848 The American Journal Human Genetics 86, 839&#226;&#8364;&#8220;849, June 11, 2010 <br />      chromosomes expressed prostate, ovary, testis,                    deletion Smith-Magenis syndrome [del(17)(p11.2p11.2)]. <br />     placenta, prostate cancer. Proc. Natl. Acad. Sci. USA                Genet. Med. 5, 430&#226;&#8364;&#8220;434. <br />     99, 16975&#226;&#8364;&#8220;16980.                                                     38. McElhinney, D.B., Krantz, I.D., Bason, L., Piccoli, D.A., Emer- <br /> 35. Ryan, A.K., Goodship, J.A., Wilson, D.I., Philip, N., Levy, A.,          ick, K.M., Spinner, N.B., Goldmuntz, E. (2002). Analysis of <br />     Seidel, H., Schuffenhauer, S., Oechsler, H., Belohradsky, B.,            cardiovascular phenotype genotype-phenotype correla- <br />     Prieur, M., et  <br /> </body></html>