<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, July 2011, p. 2960&#226;&#8364;&#8220;2972                                                                         Vol. 31, No. 14 <br /> 0270-7306/11/$12.00 doi:10.1128/MCB.01397-10 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         Spn1 Regulates GNBP3-Dependent Toll Signaling Pathway <br />                          Drosophila melanogaster&#228;&#338;&#164; <br />                 Ane Fullaondo, Susana Garc&#195;&#173;a-Sa&#194;&#180;nchez, Arantza Sanz-Parra, Emma Recio,&#226;&#8364;&#160; <br />                                    So Young Lee,* David Gubb* <br />                              CIC bioGUNE, Biscay Technology Park, Building 801A, 48160 Derio, Spain <br />                         Received 6 December 2010/Returned modification 11 January 2011/Accepted 4 May 2011 <br />  <br />             The Drosophila genome encodes 29 serpins, unknown function. We Spn1 active <br />           protease inhibitor serpin superfamily. Spn1 inhibits trypsin vitro regulates Toll-mediated <br />           immune response vivo. Expression Toll-dependent transcripts Drosomycin IM1 increased in <br />           Spn1 <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>. Overexpression Spn1 reduces induction Drosomycin immune challenge with <br />           fungi Gram-positive bacteria. Similar reductions Drosomycin levels observed psh, spz, and <br />           grass mutants Toll signaling pathway. These results support role Spn1 repressor Toll <br />           activation fungal infection. Epistatic analysis places Spn1 upstream Spa &#194;&#168;tzle processing enzyme and <br />           Grass, fungal cell wall-activated branch pathway. Overexpression <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span> recognition <br />           receptor GNBP3 activates &#226;?&#164;-1,3-glucan-sensitive branch Toll pathway. The resultant increased <br />           Drosomycin level reduced concomitant overexpression Spn1, confirming Spn1 regulates fungal <br />           cell wall branch. Spn1 null mutants altered susceptibility fungal infection compared wild <br />           type, demonstrating requirement Spn1 fine regulation immune response. <br />  <br />  <br />    Many physiological responses regulated serpins (ser-                 teolytic cascade activated upstream protease. This <br /> ine protease inhibitors) mammals, including blood clot-                 mechanism gives rapid, amplified response. With continued <br /> ting, inflammatory, complement activation, angiogenesis                    activation, serpin downstream proteases tend be <br /> pathways (15, 48). Disorders serpin metabolism respon-                  upregulated transcriptional level. <br /> sible wide range human diseases, emphysema,                      The number serpins highly variable mamma- <br /> cirrhosis, blood coagulation disorders, dementia (36, 38).                 lian species, humans having 35 mice having 64. This <br /> For reason, inhibitory mechanism serpins                  compares 29 serpin genes Drosophila, includes 12 <br /> extensively studied, mainly humans mammalian model                      noninhibitory serpin fold proteins (20, 43) lack char- <br /> organisms.                                                                     acteristic conserved &#226;&#8364;&#339;hinge region&#226;&#8364;? motif active protease <br />    Serpins large-molecular-mass protease inhibitors                 inhibitors (26). However, 17 putative inhibitory <br /> core &#226;?&#164;-sheets connected short, &#226;?&#163;-helical linkers. In               serpin genes, Spn42Da (Sp4; CG9453), contains alternatively <br /> the native state, reactive center loop (RCL) extends                spliced exons. Spn42Da gives rise 11 transcripts encoding <br /> the serpin core presents ideal bait target protease              protein isoforms, different RCL sequences tar- <br /> (48). The native serpin metastable (stressed) configura-               geted different cellular compartments (6). On basis, the <br /> tion. Following cleavage, structure adopts stable (re-                   Drosophila genome encodes 25 inhibitory serpin activities. <br /> laxed) configuration. The rapid insertion RCL                     In comparison serpins mammals, relatively little is <br /> serpin &#226;?&#164;-sheet A distorts protease&#226;&#8364;&#8482;s catalytic site            known insect serpins, studied <br /> esterification step hydrolysis reaction com-                  mainly affect immune response. The mammals two- <br /> pleted. As result, protease serpin form covalently                 stage response immune challenge: immediate innate re- <br /> linked complex. The protease translocated 70 &#195;&#8230;                  sponse delayed adaptive response, mediated antibod- <br /> crushed serpin core (25). This process denatures                   ies. The insects lack antibody response mount effective <br /> the serpin/protease complex, targeted proteolytic                 innate responses, including synthesis antimicrobial pep- <br /> destruction. This &#226;&#8364;&#339;suicide-inhibition&#226;&#8364;? mechanism destroys                  tides (AMPs), phagocytosis, encapsulation, melanization <br /> activity serpin target protease,                 pathogens. In Drosophila, humoral response involves <br /> have high turnover. Reduction serpin levels results                    localized melanization (8, 41) AMP expression the <br /> explosive activation signaling pathways inhibition                fat body (14, 27, 52) cellular response mediated by <br /> target protease lost. Each subsequent zymogen pro-                   hemocytes, phagocytose encapsulate invading patho- <br />                                                                                gens (29, 39). Expression AMPs, including Diptericin (Dpt), <br />    * Corresponding author. Mailing address: Department Func-                Drosomycin (Drs), Drosocin, several <br /> tional Genomics, CIC bioGUNE, Parque Tecnolo       &#194;&#180;gico Bizkaia,           cecropins attacins (21, 52), mainly regulated two <br /> Edificio 801A, 48160 Derio, Spain. Phone: 34 944 061 309. Fax: 34 944          signaling pathways: Imd (immune deficiency) Toll. Drs <br /> 061 301. E-mail So Young Lee: sylee@cicbiogune.es. E-mail              expression regulated Toll pathway Dpt activity <br /> David Gubb: dgubb@cicbiogune.es. <br />    &#226;&#8364;&#160; Present address: CIBIR, Infectious diseases, c/ Piqueras, 98-3a, <br />                                                                                controlled Imd pathway. An additional protein regu- <br /> 26006 Logron &#203;&#339;o, Spain.                                                        lated Toll pathway IM1 (immune-induced molecule 1) <br />    &#228;&#338;&#164; <br />      Published ahead print 16 May 2011.                                  (24) molecular function protein un- <br />  <br />                                                                         2960 <br />  VOL. 31, 2011                                                       Spn1 REGULATES THE Toll SIGNALING PATHWAY                                    2961 <br />  <br />  <br /> known. The intracellular components Imd Toll             24 48 h hatching pupal case (19). Spn27A <br /> pathways similarity mammalian innate immune             (CG11331) controls pathogen melanization, mediated the <br /> response pathways, tumor necrosis factor alpha Toll-like        phenoloxidase (PO) pathway (34, 41). Lack Spn27A results <br /> receptor/interleukin-1, respectively (23). The Imd signaling        melanization internal tissue associated high levels of <br /> pathway involves kinase cascade activated pep-        PO hemolymph. (In addition immune-related <br /> tidoglycan recognition protein LC (PGRP-LC) PGRP-LE             function, Spn27A regulates formation dorso-ventral <br /> receptors, recognize meso-diaminopimelic acid-type        embryonic axis [22, 35].) Spn28D (Spn28Dc; CG7219) regu- <br /> peptidoglycans Gram-negative bacterial cell walls (10, 18,     lates PO pathway response wounding (47) while <br /> 50), leading expression Dpt major AMP.                   Spn77Ba (CG6680) regulates tracheal melanization, can <br />    The Toll receptor originally identified role     trigger systemic expression Drs Toll pathway (51). <br /> embryonic dorso-ventral patterning (2) later recog-         Spn5 (Spn88Ea; CG18525) functions PO Toll cas- <br /> nized component innate immune response fungal        cades (1) wing inflation (9). A possible immune <br /> and Gram-positive bacterial infections adults (31, 33, 40). In   function Spn1 (CG9456; Spn42Dd) suggested upregu- <br /> the embryo, Toll ligand, Spa&#194;&#168;tzle (Spz), activated        lation transcript fungal Gram-positive infec- <br /> cleavage serine protease, Easter (Ea) (12),     tions (27). A number serpins present high levels in <br /> adult Spz activated Spa&#194;&#168;tzle processing enzyme (SPE) (28).    seminal fluid (55), differentially spliced transcripts of <br /> The activated Spz ligand binds Toll receptor,     Spn4 target separate groups proteases: furin (44), sub- <br /> initiates intracellular immune signaling pathway (31, 33,       tilase, chymotrypsin, papain-like cysteine protease (6). <br /> 54). The binding Spz Toll cell membrane induces           In paper characterize putative inhibitory serpin, <br /> the subsequent nuclear translocation NF-&#226;?&#172;B family            Spn1. As predicted presence conserved inhib- <br /> members, Dorsal (Dl) Dorsal-related immunity factor             itory hinge region motif (43), transgenically expressed Spn1 <br /> (Dif) (16, 56). Once nucleus, Dl/Dif upregulates         forms native serpin stressed conformation. Spn1 shows <br /> expression Drs AMPs.                                   narrow-range target specificity trypsin vitro, which <br />    The extracellular proteolytic signaling cascade acti-      forms putative RCL-cleaved serpin/protease complex. In <br /> vates Spz represents major difference Drosophila      fly, Spn1 modulates activity fungal cell wall <br /> and mammalian innate immune responses. In contrast           pattern recognition branch extracellular Toll sig- <br /> fly, mammalian Toll-like receptors recognize microbial deter-       naling pathway. <br /> minants directly. In Drosophila, lysine-type peptidoglycans <br /> of Gram-positive bacterial cell walls bind pattern rec-                              MATERIALS AND METHODS <br /> ognition receptors (PRRs) PGRP-SA, PGRP-SD, GNBP1 <br />                                                                         Construction Spn1 protein expression construct. The Spn1 cDNA was <br /> (4, 17, 40) &#226;?&#164;-1,3-glucans fungal cell walls bind    amplified Berkeley Drosophila Genome Project (BDGP) Drosophila <br /> the GNBP3 receptor (18). These branches Toll        Gene Collection (DGC) Gold (GH4125) collection subcloned SpeI/ <br /> pathway converge on, above, serine protease Grass (13,       EcoRI-cut pTYB12 vector using forward primer 5&#226;&#172;&#732;-GCG CCA TAT GCA <br /> 30). It recently proposed Modular serine pro-         GAC CTC CAA AGA GAT CTA C-3&#226;&#172;&#732; reverse primer 5&#226;&#172;&#732;-CGG CGA <br />                                                                     ATT CTC AAG GGC TTA CAA CAC GCC C-3&#226;&#172;&#732;. <br /> tease (ModSP) apical PRR pathway protease and <br />                                                                         Protein purification. Spn1 protein expressed Escherichia coli <br /> is common branches (7). In addition, danger-         BL21(DE3) cells purified using Impact-CN (New England Bio- <br /> signaling branch Toll pathway activated patho-    Labs), following manufacturer&#226;&#8364;&#8482;s recommendations. Spn1 fused an <br /> gen-secreted virulence factors, particular proteases,       N-terminal intein/chitin-binding domain (pTYB12-Spn1). E. coli BL21 cells con- <br /> fungi Gram-positive bacteria (13, 18) signals       taining pTYB12-Spn1 construct induced 1 mM IPTG (isopropyl- <br />                                                                     beta-D-thiogalactopyranoside) grown overnight 15&#194;&#176;C. Bacteria son- <br /> the Persephone (Psh) serine protease. All       icated 20 mM Tris-HCl, pH 7.5, 500 mM NaCl, 0.5 M EDTA buffer <br /> branches converge SPE activation Spz signals        containing 0.1 mM phenylmethylsulfonyl fluoride (PMSF) complete inhib- <br /> through Toll receptor intracellular pathway. This        itors (protease inhibitor cocktail tablets; Roche); samples centrifuged at <br /> complex set branches extracellular pathway al-       16,400 rpm 30 min loaded chitin column. Protein concen- <br />                                                                     trated 30,000-nominal-molecular-weight-limit (NMWL) membrane Ami- <br /> lows response pathogen challenges triggered re- <br />                                                                     centrifugal filter (Millipore) 20 mM Tris-HCl, pH 7.5, buffer, using an <br /> dundantly different inputs. This particularly important    &#194;&#168; KTA-FLPC fast protein liquid chromatography system. Protein aggregates and <br />                                                                     A <br /> insects, entomopathogenic fungal infections major       low-molecular-mass proteins removed using Superdex 200 10/300 gel <br /> factor limiting population growth antibody-           filtration column (Amersham Biosciences). <br /> mediated adaptive response lacking. It clear         TUG-PAGE gels. Transverse urea gradient-polyacrylamide gel electrophoresis <br />                                                                     (TUG-PAGE) gels poured glass plates following method of <br /> proteases immune-related function Drosoph-         Lomas et al. (37). The urea gradient, increasing left right gel, <br /> ila, proteome includes 211 chymotrypsin fold serine         causes native, stressed serpins unfold stable, relaxed conformation. <br /> proteases serine protease homologues (46),           The stressed-<span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>-relaxed transition active serpins gives characteristic S-curve <br /> humans 176. Thirty-seven Drosophila chymo-         profile. <br />                                                                         CD. Circular dichroism (CD) spectroscopy measurements 100 <br /> trypsin fold genes include CLIP domain, is <br />                                                                     mM Tris-HCl, pH 7.4, 25&#194;&#176;C, using JASCO-815 spectropolarimeter. Thermal <br /> associated insect immune-related proteases (46).               stability assessed monitoring CD signal 216 222 nm 25 <br />    In Drosophila, serpins known immune re-         95&#194;&#176;C using heating rate 1&#194;&#176;C/min concentration 0.2 mg/ml a <br /> sponse-related functions. Necrotic (nec, Spn43Ac, CG1857)        0.05-cm-path-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> cuvette. The thermal denaturation point (Tm) calcu- <br /> inhibits danger-signaling branch Toll-mediated re-       lated regression analysis described Dafforn et al. (11). <br />                                                                         Serpin inhibitory activity Serpin/protease complex formation. Spn1 was <br /> sponse fungal Gram-positive bacterial infections. Null       incubated different molar ratios serine proteases (kallikrein, trypsin, <br /> mutants nec constitutive expression Drs adults,       chymotrypsin, thrombin, elastase, cathepsin B, cathepsin D, cathepsin G) for <br /> even absence immune challenge (19, 32), die           30 min room temperature 50 mM HEPES containing 150 mM NaCl. For <br />  2962        FULLAONDO ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br /> inhibitory assays, 1 mM appropriate chromogenic substrate (see above)                 dish lawn sporulating B. bassiana M. anisopliae. Fly cultures were <br /> added, samples incubated 30 min. The absorbance (405               incubated 29&#194;&#176;C 25&#194;&#176;C transferred fresh vials 1 3 days. <br /> nm) determined Ultrospec 3100 Pro spectrophotometer (Amersham                    <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>Survival</span> immune challenge. The B. bassiana strain used exper- <br /> Biosciences). Spn1/protease complex formation analyzed running samples             iments recent isolate (CECT 20548; F. Granero, 1999, dead white fly). <br /> on 10% SDS-PAGE gel immunoblotting using mouse anti-Spn1 anti-                    The B. bassiana CECT lyophilate subcultured liquid yeast extract- <br /> body.                                                                                     peptone-dextrose (YPD) medium (at 25&#194;&#176;C 4 days). A total 100 ml the <br />    Proteases chromogenic substrates. The following purchased                resulting saturated culture containing hyphae conidia sonicated, <br /> Sigma: bovine pancreatic &#226;?&#163;-chymotrypsin, plasma thrombin, spleen cathepsin D;         plated potato dextrose agar (PDA), incubated 4 days 25&#194;&#176;C to <br /> porcine pancreatic trypsin, elastase, kallikrein; human placental cathepsin B     produce lawn aerial conidia. Plates stored 4&#194;&#176;C maximum 3 <br /> leukocyte cathepsin G. Most chromogenic protease substrates purchased      weeks incubated 1 day 25&#194;&#176;C just infection. Cultures were <br /> Sigma: kallikrein (B2133; N-benzoyl-Pro-Phe-Arg-p-nitroanilide hydrochloride), trypsin    replicated times detection significant reduction the <br /> and thrombin (B7632; N-benzoyl-Phe-Val-Arg-p-nitroanilide hydrochloride), chymo-          virulence fungus wild-type flies. Anesthetized flies rolled the <br /> trypsin (S7388; N-succinyl-Ala-Ala-Pro-Phe-p-nitroanilide), elastase (S8511; N-Suc-   surface fungal culture plate 30 s transferred vial <br /> cinyl-Ala-Ala-Pro-Leu-p-nitroanilide). The cathepsin B substrate (Z-Arg-Arg-pNA; 2        containing fly food incubated 29&#194;&#176;C. Flies counted, survivors <br /> HCl) purchased Calbiochem.                                                       transferred fresh vials daily. Infections M. anisopliae (DMS1490) <br />    Protein extraction flies. Frozen flies suspended 200 &#226;?&#174;l 50 mM          performed similarly. The fungus grown directly PDA plates 10 <br /> Tris buffer containing 20 mM EDTA, pH 6.5, centrifuged 5,000 &#226;&#171;&#187; g 20            13 days black conidiophores appeared. Flies incubated 25&#194;&#176;C and <br /> min 4&#194;&#176;C. An equal volume Laemmli sample loading buffer added             transferred vials fresh food prepared preservatives. The survival <br /> supernatant heated 95&#194;&#176;C 4 min.                                                 functions mutant strains following immune challenge fungal pathogens, <br />    Microbial strains. The following strains used: Gram-positive bacte-           M. luteus, sterile injury (see Fig. 8) analyzed plotted graphically <br /> rium Micrococcus luteus (CIP A270), Gram-negative bacterium E. coli (K-               using MedCalc software (www.medcalc.org/manual/kaplan-meier.php). The <br /> 12), yeast Candida albicans (ATCC 36232),and filamentous fungi Beau-              analysis includes data independent trials 20 30 infected <br /> veria bassiana (strains 80.2 CECT 20548) Metarhizium anisopliae                   flies. <br /> (DMS1490).                                                                                   Antibody production, purification, Western blot analysis. Recombinant <br />                                                                                           Spn1 protein purified 10% SDS-PAGE reducing conditions. A <br />    Drosophila strains. Stocks maintained cornmeal-agar medium 25&#194;&#176;C. <br />                                                                                           mouse polyclonal antibody raised Animal Service Centre CIC <br /> The transgenic GNBP3UAS, GNBP1UAS, PGRP-SAUAS, Drs-GFPUAS strains <br />                                                                                           bioGUNE affinity purified membrane-bound Spn1. For Western <br /> have previously described (see references 14, 17, 40, respectively). <br />                                                                                           blotting, proteins transferred SDS-PAGE gels polyvinylidene di- <br /> Spn1UAS Spn1oka strains constructed study (see below). The <br />                                                                                           fluoride (PVDF) membranes (Millipore), blocked 3% bovine serum albumin <br /> Spn1RNAi.UAS strain provided National Institute Genetics (Japan). <br />                                                                                           (BSA)&#226;&#8364;&#8220;TBS-T (0.1% Tris-buffered saline Tween) 1 h, incubated <br /> The Dif1, grassHerrade, relE20, spzrm7, psh4, SPEPasteur/Df(3R)mbc-R1 strains <br />                                                                                           1:100 dilution anti-Spn1 antibody overnight 4&#194;&#176;C. The secondary <br /> were described previously (31, 33). [Note SPEPasteur complete null <br />                                                                                           antibody anti-mouse antibody&#226;&#8364;&#8220;horseradish peroxidase (HRP) (1:5,000 dilu- <br /> <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> Df(3R)mbc-R1 deletes SPE gene. Hemizygous SPEPasteur/ <br />                                                                                           tion 1 h). Blots developed using enhanced chemiluminescence (ECL) <br /> Df(3R)mbc-R1 flies retain SPE activity homozygous <br />                                                                                           system. <br /> SPEPasteur flies.] The UAS/Gal4 expression (5) used transgenic fly <br />                                                                                              RT-PCR quantitative real-time PCR (qPCR). Samples 20 flies were <br /> strains, Gal4 driver Gal4-YP, Gal4-c564 (P{GawB}c564), Gal4-ap <br />                                                                                           frozen ground &#226;&#171;&#186;80&#194;&#176;C. Total RNA extracted using RNeasy Kit <br /> (P{GawB}ap) strains. Gal4-Yolk Protein (Gal4-YP) expressed adult <br />                                                                                           (Qiagen) following manufacturer&#226;&#8364;&#8482;s protocol. A ThermoScript reverse tran- <br /> female fat body, Gal4-c564 expressed larval adult fat body, Gal4-ap <br />                                                                                           scription-PCR (RT-PCR) kit (Invitrogen) oligo(dT)20 primers was <br /> is expressed dorsal surfaces mesonotum wing. Equal numbers males <br />                                                                                           used reverse transcription reaction. <br /> and females used assays using Gal4-YP driver,               For qPCR analysis, SYBR green One-Step Quantitative RT-PCR kit (Bio- <br /> which female flies used. The Oregon-R wild-type strain used             Rad) used Bio-Rad i-Cycler IQ. Preincubation 95&#194;&#176;C 5 min, <br /> experimental controls.                                                                    followed 35 cycles 94&#194;&#176;C 30 s, 60&#194;&#176;C 30 s, 72&#194;&#176;C 1 min. The <br />    Quantization Drs-GFP fluorescence. To make comparisons                  specificity amplified products confirmed DNA melting curve analysis. <br /> Toll pathway readout using Drs-GFP (where GFP green fluorescent pro-               Amplified transcript levels normalized internal control, ribo- <br /> tein) reporter construct, adults ages grown identical               somal protein 49 (rp49), expressed percentage normalized <br /> conditions used. For set comparisons, adult flies taken 24           transcript levels wild-type, noninfected, controls. Error bars indicate standard <br /> to 48 h posthatching photographed together, UV illumina-               deviations independent experiments, calculated using Stu- <br /> tion conditions, Leica M216F fluorescence microscope using GFP2 filter.          dent&#226;&#8364;&#8482;s two-tailed t test data shown Fig. 5E, represent five <br />    Recovery Spn1 null Spn1UAS overexpression strains. The Spn1 null                experimental replicas. For assays following immune challenge, flies col- <br /> mutation, Spn1oka, recovered intragenic deletion (of 398 bp),             lected 6 h postinfection E. coli, 24 h postinfection M. luteus, at <br /> includes exons Spn1 transcript deletes              48 h 72 h postinfection B. bassiana C. albicans. <br /> critical RCL sequence. This chromosomal segment lies                             Primers used following: Spn1, 5&#226;&#172;&#732;-GGC ATC CGA GAG CTA <br /> PBac(WH)Spn1f02145 PBac(WH)f04856 insertion sites, carry              TTC AC-3&#226;&#172;&#732; (forward) 5&#226;&#172;&#732;-GGC CAT GAG GAA CGT AGA AA-3&#226;&#172;&#732; (re- <br /> yeast FRT recombination site w&#226;&#171;&#185;mC marker. The deletion generated              verse); Drs, 5&#226;&#172;&#732;-CGT GAG AAC CTT TTC CAA TAT GAT G-3&#226;&#172;&#732; (forward) <br /> by FLP-mediated recombination PBac(WH)Spn1f02145 dpov1 w&#226;&#171;&#185;mC sp                    5&#226;&#172;&#732;-TCC CAG GAC CAC CAG CAT-3&#226;&#172;&#732; (reverse); Dpt, 5&#226;&#172;&#732;-GGC TTA <br /> and PBac(WH)f04856 w&#226;&#171;&#185;mC male germ line. The required chromosome,                   TCC GAT GCC CGA CG-3&#226;&#172;&#732; (forward) 5&#226;&#172;&#732;-TCT GTA GGT GTA GGT GCT <br /> Df(2R)PBac(WH)f02145L f04856R dpov1 Spn1oka w&#226;&#171;&#185;mC, identified dpov1               TCC-3&#226;&#172;&#732; (reverse); rp49, 5&#226;&#172;&#732;-AGA TCG TGA AGA AGC GCA CCA <br /> sp&#226;&#171;&#185; recombinant, deficiency confirmed PCR sequencing using                 AG-3&#226;&#172;&#732; (foward) 5&#226;&#172;&#732;-CAC CAG GAA CTT CTT GAA TCC GG-3&#226;&#172;&#732; (reverse). <br /> PBac-specific genomic primers (data shown). As transcripts are <br /> included deletion, refer chromosome simply Spn1oka in <br /> the text. <br />    The Spn1 overexpression strain, Spn1UAS, recovered P-element-medi- <br />                                                                                                                             RESULTS <br /> ated germ line transformation using standard techniques. The transformation                  Characterization purified Spn1. Recombinant Spn1 pro- <br /> construct, pUAST-Spn1, PCR amplification Spn1 cDNA <br /> (GH4125) using forward primer 5&#226;&#172;&#732;-CCG CCG CGG CCG CGA CGG TCA <br />                                                                                           tein E. coli purified described Materials <br /> A-3&#226;&#172;&#732; reverse primer 5&#226;&#172;&#732;-CGC GCG GTA CCA AAT ATT GTT TTT ATT                            Methods. The recombinant protein runs single band <br /> TAT-3&#226;&#172;&#732;. The amplified fragment cloned NotI/EcoRI-cut pUAST vector,               SDS-PAGE, close predicted size 41.2 kDa for <br /> and cDNA insert confirmed sequencing. The recovered Spn1UAS                    Spn1 (Fig. 1A). Transverse urea gradient (TUG)-PAGE shows <br /> strain carries homozygous viable, P{w&#226;&#171;&#185;mC&#226;&#171;&#189;Spn1Scer&#226;&#182;&#191;UAS} insertion the <br />                                                                                           classic S-curve profile, typical native serpin; between <br /> second chromosome. <br />    Microbial infections. For septic injury, 2- 3-day-old adult flies pricked <br />                                                                                           5 6 M urea serpin fold undergoes stressed-to-relaxed <br /> with tungsten needle dipped concentrated microbial culture (C. albicans,           transition (Fig. 1B). Circular dichroism measurements indicate <br /> M. luteus, E. coli). For natural infection, flies shaken 30 s petri      mixture &#226;?&#163;-helix &#226;?&#164;-sheet secondary structure is <br />  VOL. 31, 2011                                                                Spn1 REGULATES THE Toll SIGNALING PATHWAY                          2963 <br />  <br />  <br />  <br />  <br />   FIG. 1. Biochemical characterization inhibitory activity Spn1. (A) Coomassie-stained 10% SDS-PAGE gel loaded purified Spn1 <br /> protein (lane 1, protein size marker; lane 2, Spn1). (B) TUG-PAGE gel showing unfolding Spn1 0 8 M urea gradient. The curve <br /> reveals typical S profile serpin native metastable, state, transition stressed relaxed conformations occurring <br /> between 5 M 6 M urea. (C) CD spectrum showing thermal denaturation Spn1. The CD signal 220 nm reflects changes protein <br /> secondary structure respect temperature. Spn1 shows single transition 48.6&#194;&#176;C, typical inhibitory serpins. mDeg, millidegrees. <br />  <br />  <br />  <br /> characteristic serpins. The thermal denaturation point                    real-time PCR (qPCR) measure Spn1 transcript levels fol- <br /> (48.6&#194;&#176;C) typical inhibitory serpins native conforma-            lowing immune challenge. Wild-type flies infected the <br /> tion (Fig. 1C). Spn1 forms complex trypsin (Fig. 2A)              Gram-positive bacterium M. luteus, fungus B. bassiana, or <br /> not elastase, chymotrypsin, kallikrein, thrombin, cathe-             yeast C. albicans increased Spn1 transcript levels (Fig. <br /> psins B, D, G (data shown). In case trypsin,                  3A); increases Spn1 transcript levels confirmed at <br /> increasing ratio serpin protease gives increase              protein level Western blotting (Fig. 3B). No increase in <br /> the 60- 70-kDa protease/serpin complex band 35-                  Spn1 transcript levels occurs sterile injury E. coli in- <br /> 40-kDa putative, RCL-cleaved serpin band (Fig. 2A). Chro-                    fection (Fig. 3A). To confirm Spn1 expression regulated <br /> mogenic substrate assays confirm Spn1 inhibits trypsin                  Toll-mediated immune response, Toll pathway grass, <br /> strongly (96% serpin/protease ratio 10:1) (Fig. 2B)              spz, SPE, Dif mutants infected B. bassiana M. <br /> not proteases. Thus, Spn1 inhibitor trypsin-like              luteus. In mutant backgrounds, upregulation of <br /> proteases vitro, narrow substrate specificity.                     Spn1 transcript blocked, Imd pathway mutant, <br />    These data confirm recombinant Spn1 protein sam-                 Relish (Rel), does decrease Spn1 transcript levels (Fig. 3C <br /> ple consists native, monomeric serpin active, inhib-              D). <br /> itory conformation.                                                             Spn1 regulates Drs expression. <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-342' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-345' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) <br />    Activating Toll pathway upregulates Spn1. To charac-                  knockdown Spn1 transcript activates expression a <br /> terize function Spn1 vivo, used quantitative              Drs-GFP reporter construct noninfected, Spn1RNAi flies, <br />  2964      FULLAONDO ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />                                                                                concomitant overexpression Spn1 reduces Drs- <br />                                                                                GFP signal wild-type values Fig. 4A. These results suggest <br />                                                                                Spn1 inhibiting Drs expression absence immune <br />                                                                                challenge. However, knockdown <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> in- <br />                                                                                complete Spn1RNAi flies strongly reduced levels of <br />                                                                                Spn1, protein remains (Fig. 4B). To confirm and <br />                                                                                extend results, generated null mutant Spn1, <br />                                                                                Spn1oka (Materials Methods). The expression Drs <br />                                                                                IM1 genes, targets Toll signaling pathway, was <br />                                                                                measured qPCR, expression Dpt, induced <br />                                                                                mainly Imd pathway. Spn1oka mutants in- <br />    FIG. 2. (A) Immunoblot showing Spn1/trypsin complex formation               creased (more 2-fold) levels Drs IM1 transcripts, <br /> (stained anti-Spn1 antibodies). The left-hand lane contains pure          Dpt expression altered respect the <br /> Spn1, successive lanes containing serpin-protease mixtures           wild-type fly (Fig. 4C). Overexpression Spn1UAS does not <br /> successively protease molar ratios 1:1, 2:1, 5:1, 10:1. In <br /> the right-hand lane, SDS-stable serpin-protease complex indi-           reduce Drs transcript levels noninfected flies. This result <br /> cated arrowhead, native serpin indicated arrow,         implies target protease Spn1 fully inhibited the <br /> and putative RCL-cleaved, uncomplexed serpin indicated           endogenous serpin levels additional Spn1 activity has <br /> dashed arrow. (Lower-molecular-mass digested serpin fragments              effect. Thus, lack Spn1 inhibitor leads to <br /> also present.) (B) The trypsin-inhibitory activity Spn1, determined <br /> by competition assay using chromogenic substrate trypsin. <br />                                                                                constitutive activation Toll pathway-specific gene expression <br /> White columns represent Spn1, black columns represent incubated                absence immune challenge. <br /> mixtures Spn1-trypsin, gray columns represent trypsin. Columns             Spn1 represses Drs transcript levels response fungi but <br /> show average different independent assays sam-           bacteria. After fungal challenge B. bassiana, Spn1UAS- <br /> ple. OD, optical density. <br />  <br />  <br />  <br />  <br />    FIG. 3. Spn1 levels wild-type Toll pathway mutants microbial infection. (A) Spn1 transcript level, measured qPCR the <br /> following groups: C, noninfected wild-type flies; CI, clean injury; E.c, 6 h postinfection E. coli; M.l, 24 h postinfection M. luteus; B.b, 48 h <br /> postinfection B. bassiana; C.a, 48 h postinfection C. albicans. (B) Spn1 protein levels microbial infection, detected by <br /> immunoblotting anti-Spn1 6 h postinfection E. coli (E.c), 24 h postinfection M. luteus (M.l), 48 h 72 h postinfection with <br /> B. bassiana (B.b). (C) The grass, spz, Dif mutants reduced levels Spn1 transcript compared wild-type (&#226;&#171;&#185;) flies fungal infection <br /> (B. bassiana), Imd pathway mutant, Rel, does affect Spn1 transcript levels. (D) Similarly, grass, SPE [SPEPasteur/Df(3R)mbc-R1 <br /> hemizygotes], Dif mutants decrease Spn1 transcript levels compared wild-type (&#226;&#171;&#185;) Rel mutant, Gram-positive bacterial <br /> infection (M. luteus). &#226;&#180;&#177;, significant difference (P &#226;&#172;? 0.01) respect control fly strain. <br />  VOL. 31, 2011                                                               Spn1 REGULATES THE Toll SIGNALING PATHWAY                          2965 <br />  <br />  <br />  <br />  <br />   FIG. 4. Lack Spn1 results increased Drs IM1 levels unchallenged flies. Complete genotypes given parentheses the <br /> abbreviated forms. (A) Fluorescent microscope images Drs-GFP expression control (Drs-GFP; Gal4-YP/TM6C), Spn1RNAi (Spn1RNAi.UAS/ <br /> Drs-GFP; Gal4-YP/TM6C), Spn1RNAi Spn1UAS (Spn1RNAi.UAS/Drs-GFP; Gal4-YP/Spn1UAS) flies. Spn1RNAi flies constitutive activation of <br /> Drs-GFP expression. (B) The corresponding Spn1 protein levels increased Spn1UAS (Spn1UAS; Gal4-YP) flies strongly reduced in <br /> Spn1RNAi (Spn1RNAi.UAS; Gal4-YP) flies, shown immunoblotting anti-Spn1 antibodies. (C) Drs, Dpt, IM1, Spn1 transcript levels in <br /> wild-type control, Spn1UAS (Spn1UAS; Gal4-YP), Spn1oka flies. The lack Spn1 transcript Spn1oka null mutant associated parallel <br /> increases Drs IM1 transcripts, Dpt transcript remains wild-type levels. C, control. &#226;&#180;&#177;, P &#226;&#172;? 0.01. <br />  <br />  <br />  <br /> overexpressing flies reduced levels Drs transcript com-             SPE, common branches Toll signaling <br /> pared wild type (Fig. 5A). This reduction similar              pathway. The Grass protease acts downstream circulating <br /> that observed mutants Toll pathway proteases grass             pattern recognition receptors upstream SPE. To local- <br /> psh. Since B. bassiana releases virulence factors                ize Spn1 function upstream Toll, determined epi- <br /> &#226;?&#164;-1,3-glucans, infections fungus activate Toll             static relationships Spn1 SPE Grass <br /> both danger signaling PRR branches               proteases. <br /> pathway. C. albicans, however, activates mainly fungus-                    Overexpression activated form SPE (SPE*) up- <br /> sensitive PRR branch. Upon injection C. albicans, in-                regulates Drs expression (13). The upregulated level Drs- <br /> duction Drs strongly reduced Spn1UAS-overexpressing                GFP protein SPE*UAS flies blocked expression of <br /> flies (Fig. 5B), suggesting Spn1 represses PRR                Spn1UAS (Fig. 6A). On hand, upregulated expres- <br /> branch Toll pathway. In contrast, Spn1UAS-overexpress-               sion Drs transcript Spn1oka mutants suppressed SPE <br /> ing flies change Drs transcript levels Gram-               mutant flies (Fig. 6C). Taken together, data indicate that <br /> positive challenge M. luteus (Fig. 5C). The Dipt transcript            Spn1 acts upstream SPE. <br /> levels (regulated Imd pathway) decreased                    Similarly, upregulated level Drs-GFP protein in <br /> Spn1UAS flies infected Gram-negative bacterium E.                  grassUAS flies blocked expression Spn1UAS (Fig. <br /> coli (Fig. 5D). Spn1oka flies elevated levels Drs tran-             6B), showing Spn1 downstream Grass. As with <br /> script absence immune challenge, Drs transcript                  SPE, Drs activation Spn1oka mutants reduced grass <br /> levels increased response infection B.               mutant background (Fig. 6D). Therefore, Spn1 functions up- <br /> bassiana (Fig. 5E). These results indicate Spn1 represses              stream Grass. <br /> Drs expression specifically response fungal challenge.                   Spn1 acts downstream GNBP3 receptor the <br />    Spn1 acts upstream Grass. Following fungal Gram-                   PGRP-SA GNBP1 receptors. The increased level Drs <br /> positive bacterial infection, Toll ligand Spz activated           expression GNBP3UAS flies reduced Spn1UAS overex- <br />  2966      FULLAONDO ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. AMP transcript levels Spn1UAS flies following immune challenge fungi Gram-positive Gram-negative bacteria. Drs <br /> transcript levels wild-type (&#226;&#171;&#185;), Spn1UAS (Gal4-c564/&#226;&#171;&#185;; Spn1UAS/&#226;&#171;&#185;), psh, spz, grass flies 72 h postinfection B. bassiana (A), 48 h <br /> postinfection C. albicans (B), 24 h postinfection M. luteus (C). (D) Dpt levels Spn1UAS Rel flies 6 h postinfection with <br /> the Gram-negative bacterium E. coli. Spn1UAS overexpression reduces elevated Drs transcript levels flies challenged B. bassiana C. <br /> albicans (&#226;&#180;&#177;, P &#226;&#172;&#381; 0.01) flies challenged M. luteus. The Dpt transcript levels following E. coli infection unaffected in <br /> Spn1UAS-overexpressing flies, Dpt expression blocked Imd pathway mutant Rel. (E) Time course Drs transcript levels following <br /> immune challenge B. bassiana wild-type, psh, Spn1oka flies. qPCR measurements taken 0 72 h postinfection. <br />  <br />  <br />  <br /> pression, monitored Drs-GFP fluorescence qPCR                     reduced modSP mutation (in Spn1oka; modSP double <br /> (Fig. 7A B). These results indicate Spn1 functions                 mutants) (Fig. 7E). Thus, ModSP activity required for <br /> downstream fungal recognition receptor GNBP3. In                     upregulation Drs transcript Spn1oka background. <br /> contrast, activation Drs expression Gram-positive                 On hand, Drs-GFP fluorescence ModSPUAS <br /> bacterial recognition receptors PGRP-SAUAS GNBP1UAS                     overexpressing fly attenuated overexpression of <br /> is unaffected Spn1UAS overexpression (Fig. 7C D).                    Spn1 (Fig. 7F). It follows Spn1 does repress ModSP <br />    Spn1 independent modSP. We confirm Drs                    activity. <br /> transcript level strongly reduced modSP background                   The Spn1oka mutant shows altered susceptibility fungal <br /> (7), elevated Drs level Spn1oka flies partially          infections M. luteus. The increased Drs levels ob- <br />  VOL. 31, 2011                                                             Spn1 REGULATES THE Toll SIGNALING PATHWAY                       2967 <br />  <br />  <br />  <br />  <br />   FIG. 6. Spn1 acts upstream SPE grass. Complete genotypes given parentheses abbreviated forms. (A) Drs-GFP <br /> expression upregulated activated SPE&#226;&#180;&#177; (SPE&#226;&#180;&#177;.UAS/Drs-GFP; Gal4-YP/&#226;&#171;&#185;). Spn1UAS overexpression does block upregulation SPE&#226;&#180;&#177; <br /> Spn1UAS (SPE&#226;&#180;&#177;.UAS/Drs-GFP; Gal4-YP/Spn1UAS) flies, indicating Spn1UAS acting upstream SPE, common branch extracellular <br /> Toll pathway. (B) Similarly, overexpression Spn1UAS does block Drs-GFP expression grassUAS (grassUAS/Drs-GFP; Gal4-YP) grassUAS <br /> Spn1UAS (grassUAS/Drs-GFP; Gal4-YP/Spn1UAS) flies. The increased Drs transcript levels Spn1oka flies reduced SPE (C) grass <br /> (D) mutant backgrounds. Genotypes abbreviated follows: Spn1oka; SPE denotes Spn1oka; SPEPasteur/Df(3R)mbc-R1 flies SPE denotes <br /> SPEPasteur/Df(3R)mbc-R1 flies. C, uninfected wild-type controls. For panels C D, asterisk indicates significant difference (P &#226;&#172;? 0.01) <br /> between double mutant strain Spn1oka single mutant. <br />  <br />  <br />  <br /> served Spn1oka flies expected correlate               analysis). The survival Spn1oka flies reduced compared to <br /> increased resistance fungal infections. However, Drs             psh mutants 72 h postinfection, but <br /> expression acts useful marker Toll pathway activation,            mortality psh flies higher, 15% <br /> Drs AMP, secreted AMPs form               surviving 168 h 264 h. The survival rates psh <br /> defense mechanism following immune challenge. As direct                 Dif mutants reduced similar degrees respect to <br /> test effect lack Spn1 activity immune re-             wild-type flies, previously reported (18). Surprisingly, psh; <br /> sponse, viability Spn1oka mutants determined             Spn1oka double mutants die rapidly psh mutants. <br /> infection species entomopathogenic fungi, Gram-             Given Spn1oka psh mutants additive effect <br /> positive bacterium, clean injury.                                     susceptibility infection, Spn1 involved dif- <br />    The survival probability Spn1oka mutant strain              ferent branch Toll signaling pathway Psh. In contrast, <br /> compared psh danger-signaling DIF              susceptibility Spn1oka mutant similar mu- <br /> the intercellular Toll pathway (Fig. 8A, panels)             tants fungal PRR branch, somewhat lower than <br /> GNBP3, modSP, grass mutants PPR branch                 GNBP3, similar modSP, a <br /> the Toll pathway (Fig. 8A, panels) following natural               little higher grass mutants. Unlike psh; <br /> infection B. bassiana. The survival rate noninfected              Spn1oka double mutant, combination Spn1oka and <br /> controls indicates nonspecific effects viability        modSP mutations does additive effect suscep- <br /> different mutant strains (Fig. 8A, right panel). Spn1oka mutants          tibility. Indeed Spn1oka; modSP double mutant slightly <br /> show rapid decrease survival postinfection, 85%            resistant infection single mutant alone. <br /> surviving 24 h, 85% wild-type flies survive          (We unable test GNBP3; Spn1oka grass; <br /> for 120 h. After 264 h, survival probability Spn1oka flies         Spn1oka double mutant combinations flies have <br /> is 25%, compared 75% wild-type flies. These differences            strongly reduced viability.) <br /> in survival statistically significant (P &#226;&#172;? 0.0001, log rank              Following immune challenge M. anisopliae, survival <br />  2968      FULLAONDO ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. Spn1 acts GNBP3-dependent Toll pathway PGRP-SA/GNBP1-dependent pathway. Complete genotypes given <br /> in parentheses abbreviated forms. (A) The Drs-GFP fluorescence GNBP3UAS (GNBP3UAS/Drs-GFP; Gal4-YP/&#226;&#171;&#185;) flies reduced in <br /> GNBP3UAS Spn1UAS (GNBP3UAS/Drs-GFP; Gal4-YP/Spn1UAS) flies. (B) The elevated Drs transcript levels GNBP3UAS (GNBP3UAS/&#226;&#171;&#185;; Gal4- <br /> YP/&#226;&#171;&#185;) flies reduced GNBP3UAS Spn1UAS (GNBP3UAS/&#226;&#171;&#185;; Gal4-YP/Spn1UAS) flies. C, uninfected wild-type control. &#226;&#180;&#177;, P &#226;&#172;? 0.01, between <br /> GNBP3UAS GNBP3UAS Spn1UAS flies. (C) Drs-GFP fluorescence PGRP-SAUAS GNBP1UAS (PGRP-SAUAS GNBP1UAS/Drs-GFP; Gal4-YP) <br /> flies remains activated PGRP-SAUAS GNBP1UAS Spn1UAS (PGRP-SAUAS GNBP1UAS/Drs-GFP; Spn1UAS/Gal4-YP) flies. (D) The elevated Drs <br /> transcript levels PGRP-SAUAS GNBP1UAS (PGRP-SAUAS GNBP1UAS/Drs-GFP; Gal4-YP) flies remains unchanged PGRP-SAUAS GNBP1UAS <br /> Spn1UAS (PGRP-SAUAS GNBP1UAS/Drs-GFP; Spn1UAS/Gal4-YP) flies. C, uninfected wild-type control. (E) The elevated levels Drs transcript in <br /> the Spn1oka mutant contrasts reduced levels modSP mutants. The double mutant Spn1oka; modSP shows intermediate levels Drs <br /> transcript, implying ModSP activates Drs transcription, required Spn1 function. C, wild-type control. &#226;&#180;&#177;, P &#226;&#172;? 0.01, between <br /> the Spn1oka; modSP Spn1oka single mutant. (F) The Drs-GFP fluorescence modSPUAS (modSPUAS/Drs-GFP; Gal4-YP/&#226;&#171;&#185;) flies not <br /> reduced modSPUAS Spn1UAS (modSPUAS/Drs-GFP; Gal4-YP/Spn1UAS) flies. This result confirms Spn1 inhibitor downstream the <br /> ModSP protease. <br />  <br />  <br />  <br /> of Spn1oka mutants higher wild-type flies (Fig.             rank analysis). The increased resistance Spn1oka mutants to <br /> 8B). At 40 h postinfection 72% Spn1oka mutants survive,                  M. anisopliae infection consistent role Spn1 a <br /> compared 52% wild-type flies. At 94 h postinfection, 33%              specific repressor immune response fungus. The <br /> of Spn1oka flies survive, 10% wild-type flies survive.          differential response B. bassiana M. anisopliae appar- <br /> These differences statistically significant (P &#226;&#171;&#189; 0.0012, log            ently reflects different pathogenic mechanisms fun- <br />  VOL. 31, 2011                                                                 Spn1 REGULATES THE Toll SIGNALING PATHWAY                     2969 <br />  <br />  <br />                                                                               genetic analysis role innate immune response. The <br />                                                                               recombinant Spn1 protein exhibits stressed-to-relaxed tran- <br />                                                                               sition 5 6 M urea thermal denaturation <br />                                                                               point 48.6&#194;&#176;C. Both values typical inhibitory <br />                                                                               serpins native conformation. Recombinant Spn1 protein <br />                                                                               potent inhibitor trypsin inactive chymo- <br />                                                                               trypsin, elastase, thrombin, kallikrein, cathepsin proteases. <br />                                                                               The putative P1/P1&#226;&#172;&#732; site protease cleavage RCL <br />                                                                               falls 17 residues C-terminal start hinge region of <br />                                                                               RCL (26). On basis, P1/P1&#226;&#172;&#732; site Spn1 R/A, <br />                                                                               ideal target trypsin (3) consistent our <br />                                                                               inhibitory assays. Taken together, data establish that <br />                                                                               Spn1 inhibitor trypsin-like proteases. In contrast, the <br />                                                                               Nec serpin (which regulates immune response) does <br />                                                                               inhibit trypsin broad-range inhibitor elastase, <br />                                                                               thrombin, chymotrypsin-like proteases (45). In case of <br />                                                                               Nec, serpin core carries N-terminal peptide 73 amino <br />                                                                               acids (after cleavage putative export signal peptide) (42) <br />                                                                               cleaved immune challenge (32). Both full-length <br />                                                                               serpin (Nec-fl) truncated core serpin (Nec-&#226;&#338;&#172;N) are <br />                                                                               active protease inhibitors, Nec-fl shows 13-fold increase <br />                                                                               specificity porcine pancreatic elastase compared to <br />                                                                               Nec-&#226;&#338;&#172;N (42). These inhibitors Toll pathway show <br />                                                                               different ranges protease target specificities vitro. <br />                                                                                  In absence infection, Spn1oka nec mutants <br />                                                                               constitutive expression Drs antifungal peptide <br />                                                                               (19). If nec Spn1 inhibit single, linear pathway, both <br />                                                                               need inactivated allow Drs expression re- <br />                                                                               sponse infection. If, however, serpins inhibit sepa- <br />                                                                               rate pathways, lack function of activate <br />    FIG. 8. Viability Spn1oka mutants immune challenge with <br /> fungi, Gram-positive bacterium, clean injury. Kaplan-Meier plot          Drs expression. Recently, clear extra- <br /> of survival functions. (A) Natural infection fungus B.           cellular proteolytic cascade activates Toll receptor is <br /> bassiana CECT 20548. The left graph shows data survival           simple linear zymogen cascade but, instead, split into <br /> of Spn1oka flies compared mutants danger-signaling pathway          separate branches (13, 18), allowing serpins <br /> (psh psh; Spn1oka) intracellular Toll pathway (Dif). The <br /> survival wild-type Spn1oka mutant flies significantly different <br />                                                                               act independently. An additional phenotype shown nec <br /> according log rank test (inverse density function &#226;?&#185;2 [&#226;?&#185;2IDF] &#226;&#171;&#189;         mutants develop melanotic spots, associ- <br /> 32.658; P &#226;&#172;? 0.0001). The right graph shows data nonin-            ated cellular necrosis phenotype, shortly eclosion <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> fected controls. In left graft, survival Spn1oka flies       pupa die 24 48 h (19). This phenotype is <br /> compared mutants PRR signaling pathway (GNBP3,              completely suppressed psh; nec double mutant flies, together <br /> grass, modSP, Spn1oka; modSP). The right graph shows <br /> survival data noninfected, control flies. (B) Natural infection       activation Drs expression (33). In contrast, the <br /> the fungus M. anisopliae wild-type Spn1oka mutant flies. Control       Spn1oka mutant shows cellular necrosis phenotype. <br /> noninfected flies (wt C Spn1oka C) represented               The separate branches Toll signaling path- <br /> graph. The survival wild-type Spn1oka mutant flies significantly    way activated Gram-positive fungal cell wall <br /> different according log rank test (&#226;?&#185;2IDF &#226;&#171;&#189; 10.552; P &#226;&#171;&#189; 0.0012). <br /> (C) Survival injection saturated culture M. luteus sterile <br />                                                                               pattern recognition receptors microbial proteases (which <br /> injection (S.I) wild-type Spn1oka flies. The differences survival   correspond pathogen-encoded virulence factors, secreted <br /> between flies injected M. luteus sterile injection controls      contact host) (4, 13, 17, 18, 40). The side <br /> are significant.                                                          branches converge Spa&#194;&#168;tzle processing enzyme (28), <br />                                                                               giving multiply triggered sensing mechanism. It mi- <br />                                                                               crobial protease-sensitive, danger-signaling branch of <br /> gus (see Discussion). In contrast fungal challenge, injection              Toll pathway regulated Nec (32) includes <br /> of M. luteus does cause significant changes viability              serine protease Psh (33). As here, Spn1 acts <br /> of Spn1oka flies compared clean, mock injection (Fig. 8C)                downstream fungal cell wall pattern recognition re- <br /> although treatments cause transient, rapid lethality,                  ceptor, GNBP3. The inhibition branches is <br /> with 20% flies dying 10 h.                            independent lack serpin function acti- <br />                                                                               vate Drs expression (Fig. 9). <br />                             DISCUSSION                                           Spn1 transcript levels upregulated following immune <br />                                                                               challenge fungi B. bassiana C. albicans unless the <br />    We report biochemical functional character-                   Toll pathway blocked (in SPE, grass, spz, Dif mutant flies). <br /> ization Drosophila Spn1 serpin. We initially determined                In unchallenged flies, Drs transcript levels Spn1oka mu- <br /> the biochemical properties Spn1 vitro, followed                    tants increased 2.5 3.5 times wild-type <br />  2970      FULLAONDO ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />                                                                            activation Drs expression Spn1oka mutant is <br />                                                                            blocked grass SPE mutant flies (Fig. 6). In this <br />                                                                            context, Grass protease originally assigned the <br />                                                                            Gram-positive pattern recognition pathway (30), activated by <br />                                                                            GNBP1 PGRP-SA receptors (17). Subsequent study, <br />                                                                            however, shown Grass downstream the <br />                                                                            Gram-positive bacterial fungal PRRs (13). Our analysis <br />                                                                            confirms Grass common PRR-regulated <br />                                                                            branches; Spn1 acts GNBP3 receptor Grass <br />                                                                            independent GNBP1 PGRP-SA receptors. <br />                                                                            We show, however, Drs upregulation Spn1oka back- <br />                                                                            ground does require ModSP activity, un- <br />                                                                            expected ModSP apical protease PRR- <br />                                                                            regulated pathways. The simplest model consistent our <br />                                                                            data Spn1 regulates branch fungal cell wall <br />                                                                            pattern-sensing pathway (Fig. 9). GNBP3 activate the <br />                                                                            Grass zymogen target protease inhibited Spn1 <br />                                                                            ModSP. Even complete removal Spn1 inhibitor, <br />                                                                            however, allows partial activation Grass moder- <br />                                                                            ate increase Drs transcription. These results consistent <br />                                                                            results Buchon et al. (7): modSP mutant <br />                                                                            completely blocks Drs activation PGRP-SAUAS GNBP1UAS- <br />                                                                            overexpressing flies, GNBP3UAS-overexpressing flies Drs <br />                                                                            transcript levels partially blocked modSP mu- <br />                                                                            tant remain 3-fold higher levels wild <br />    FIG. 9. Model Spn1 regulation Toll-mediated innate im-        type. Apparently, Grass partially activated the <br /> mune response. Three separate branches Toll signaling          GNBP3 pathway absence ModSP activity. In addition, <br /> pathway activated microbial cell wall PRRs (GNBP3,       activation Drosomycin ModSP-overex- <br /> fungi yeasts; GNBP1, PGRP-SA, PGRP-SD Gram-posi- <br /> tive bacteria) danger-signaling (virulence factors). These   pressing flies blocked Spn1 overexpression (Fig. 7E); <br /> branches pathway converge SPE, activates Spz (the          criterion, Spn1 inhibitory activity does act down- <br /> ligand Toll). The intracellular Toll pathway signals trough Dif      stream ModSP protease. This result strongly supports <br /> activate expression Drs antimicrobial peptides     model presented Fig. 9. <br /> fat body. The GNBP3 danger-signaling branches Toll sig- <br /> naling pathway regulated independently Spn1 nec. Both of <br />                                                                               The increased susceptibility Spnoka flies immune chal- <br /> these serpins represent nonredundant genetic functions, lack       lenge B. bassiana opposite ex- <br /> activates Drs expression.                                                  pected increasing expression Drs AMP. It un- <br />                                                                            likely, however, moderate increase Drs activity in <br />                                                                            unchallenged Spnoka flies sufficient itself confer <br /> flies. The levels Toll-responsive transcript IM1           significant resistance fungal infections. In contrast infec- <br /> increased, parallel Drs transcript, Spn1oka mu-             tion B. bassiana, resistance infection M. anisopliae is <br /> tants Imd pathway-responsive transcript, Dpt, re-                increased Spnoka flies, agreement role Spn1 a <br /> mains unaffected (Fig. 4C). Overexpression Spn1UAS does                 repressor immune response. Although response of <br /> not reduce Drs IM1 transcript levels unchallenged flies              Drosophila M. anisopliae characterized <br /> (Fig. 4C). Similarly, Drs transcription activated               B. bassiana, mutants block Toll pathway <br /> overexpressing Gram-positive PRRs (in GNBP1UAS                         increased susceptibility M. anisopliae infections <br /> PGRP-SAUAS flies), increasing endogenous levels Spn1                (21). Significantly, PR1 protease M. anisopliae been <br /> does reduce elevated Drs transcript levels (Fig. 7D). In           shown major virulence factor (49, 53). Following im- <br /> contrast, Drs transcription levels reduced Spn1 overex-             mune challenge M. anisopliae, Drs expression not <br /> pression flies overexpressing GNBP3UAS (the receptor                blocked psh GNBP3 single mutant flies, is <br /> fungal cell walls) (Fig. 7B) infected B. bassiana C.            abolished psh; GNBP3 double mutant background. In <br /> albicans (Fig. 5A B). The levels Spn1 rate                   contrast, B. bassiana infection Drs expression very <br /> limiting Drs transcript expression                   strongly reduced psh mutants unaffected GNBP3 <br /> GNBP3-regulated branch Toll signaling pathway               mutants (18). These results suggest B. bassiana has <br /> activated. This striking result; general, regulatory serpin        evolved mechanism allows escape GNBP3 sur- <br /> activity excess target proteases              veillance (18). Thus, opposite effects viability of <br /> overexpression physiological consequences. In addition              Spn1oka mutants fungal pathogens reflect <br /> to facilitating epistatic analysis, GNBP3 pathway-depen-               differential contributions danger signaling GNBP3- <br /> dent requirement levels Spn1 activity rate                dependent mechanisms detecting immune challenge. As <br /> limiting consistent Spn1 having specific role             suggested Gottar et al. (18), likely original <br /> Toll-mediated response fungal infections (see below).                   mechanism sensing fungal pathogens PPRs that <br />    Further analysis lack-of-function mutations shows             surveillance virulence factors represents host counter- <br />  VOL. 31, 2011                                                                       Spn1 REGULATES THE Toll SIGNALING PATHWAY                                      2971 <br />  <br />  <br /> strategy detect pathogens adapted avoid                         15. Gettins, P. G. 2002. Serpin structure, mechanism, function. Chem. Rev. <br />                                                                                         102:4751&#226;&#8364;&#8220;4804. <br /> GNBP detection mechanism. It unlikely adap-                         16. Gillespie, S. K., S. A. Wasserman. 1994. Dorsal, Drosophila Rel-like <br /> tive pressure respond pathogens avoid PPR                            protein, phosphorylated activation transmembrane protein <br /> sensor lead additional cross-regulatory                             Toll. Mol. Cell. Biol. 14:3559&#226;&#8364;&#8220;3568. <br />                                                                                     17. Gobert, V., et al. 2003. Dual activation Drosophila toll pathway two <br /> interactions GNBP3-dependent activation Toll                              pattern recognition receptors. Science 302:2126&#226;&#8364;&#8220;2130. <br /> pathway.                                                                            18. Gottar, M., et al. 2006. Dual detection fungal infections Drosophila via <br />    In summary, characterized serpin family inhib-                           recognition glucans sensing virulence factors. Cell 127:1425&#226;&#8364;&#8220;1437. <br />                                                                                     19. Green, C., et al. 2000. The necrotic gene Drosophila corresponds of <br /> itor Spn1. This protein forms covalently linked complexes                          cluster serpin transcripts mapping 43A1.2. Genetics 156:1117&#226;&#8364;&#8220; <br /> trypsin vitro, indicating natural target protease                        1127. <br /> Spn1 trypsin-like protease. Spn1 regulates innate im-                      20. Gubb, D., A. Robertson, T. Dafforn, L. Troxler, J. M. Reichhart. 2007. <br />                                                                                         Drosophila Serpins: regulatory cascades innate immunity morphogen- <br /> mune response fungal pathogens Drosophila. In par-                           esis, p. 207&#226;&#8364;&#8220;227. In G. A. Silverman D. A. Lomas (ed.), Molecular and <br /> ticular, Spn1 acts downstream GNBP3 receptor                                 cellular aspects serpinopathies disorders serpin activity, vol. 1. <br /> upstream Grass protease fungal cell wall recog-                           World Scientific Publishing Co., Singapore, Republic Singapore. <br />                                                                                     21. Han, Z. S., Y. T. Ip. 1999. Interaction specificity Rel-related <br /> nition branch Toll pathway.                                                      proteins regulating Drosophila immunity gene expression. J. Biol. Chem. <br />                                                                                         274:21355&#226;&#8364;&#8220;21361. <br />                                                                                     22. Hashimoto, C., D. R. Kim, L. A. Weiss, J. W. Miller, D. Morisato. 2003. <br />                          ACKNOWLEDGMENTS                                                Spatial regulation developmental signaling serpin. Dev. Cell 5:945&#226;&#8364;&#8220; <br />                                                                                         950. <br />   We thank Bloomington Stock Center, National Institute Ge- <br />                                                                                     23. Hoffmann, J. A., J. M. Reichhart. 2002. Drosophila innate immunity: an <br /> netics (Japan), Vienna Drosophila RNAi Centre fly strains.                  evolutionary perspective. Nat. Immunol. 3:121&#226;&#8364;&#8220;126. <br /> We thank Jean-Marc Reichhart Laure El Chamy critical dis-                   24. Huang, H. R., Z. J. Chen, S. Kunes, G. D. Chang, T. Maniatis. 2010. <br /> cussion encouragement reviewers valuable comments.                      Endocytic pathway required Drosophila Toll innate immune signaling. <br />   This work supported Spanish Ministry Education                        Proc. Natl. Acad. Sci. U. S. A. 107:8322&#226;&#8364;&#8220;8327. <br /> Science Grant D.G. Additional support provided                      25. Huntington, J. A., R. J. Read, R. W. Carrell. 2000. Structure a <br /> Department Industry, Tourism Trade Government                      serpin-protease complex shows inhibition deformation. Nature 407:923&#226;&#8364;&#8220; <br /> Autonomous Community Basque Country (Etortek Research                            926. <br /> Programs), Department Education, Universities Re-                   26. Irving, J. A., R. N. Pike, A. M. Lesk, J. C. Whisstock. 2000. Phylogeny <br />                                                                                         serpin superfamily: implications patterns amino acid conserva- <br /> search Basque Government (Basic Applied Research Pro- <br />                                                                                         tion structure function. Genome Res. 10:1845&#226;&#8364;&#8220;1864. <br /> gram), Innovation Technology Department Bizkaia                     27. Irving, P., et al. 2001. A genome-wide analysis immune responses in <br /> County. A.F supported Ph.D. studentship Basque                        Drosophila. Proc. Natl. Acad. Sci. U. S. A. 98:15119&#226;&#8364;&#8220;15124. <br /> Government Department Education, Universities Research.                      28. Jang, I. H., et al. 2006. A Spatzle-processing enzyme required toll sig- <br />   We declare conflicts interest.                                     naling activation Drosophila innate immunity. Dev. Cell 10:45&#226;&#8364;&#8220;55. <br />                                                                                     29. Jiravanichpaisal, P., S. Sricharoen, I. Soderhall, K. Soderhall. 2006. <br />                                 REFERENCES                                              White spot syndrome virus (WSSV) interaction crayfish haemocytes. <br />                                                                                         Fish Shellfish Immunol. 20:718&#226;&#8364;&#8220;727. <br />  1. Ahmad, S. T., S. T. Sweeney, J. A. Lee, N. T. Sweeney, F. B. Gao. 2009.     30. Kambris, Z., et al. 2006. Drosophila immunity: large-scale vivo RNAi <br />     Genetic screen identifies serpin5 regulator toll pathway            screen identifies serine proteases required Toll activation. Curr. Biol. <br />     CHMP2B toxicity associated frontotemporal dementia. Proc. Natl.                16:808&#226;&#8364;&#8220;813. <br />     Acad. Sci. U. S. A. 106:12168&#226;&#8364;&#8220;12173.                                            31. Lemaitre, B., E. Nicolas, L. Michaut, J. M. Reichhart, J. A. Hoffmann. <br />  2. Anderson, K. V., G. Jurgens, C. Nusslein-Volhard. 1985. Establishment           1996. The dorsoventral regulatory gene cassette Spatzle/Toll/cactus controls <br />     dorsal-ventral polarity Drosophila embryo: genetic studies         potent <br /> </body></html>