<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Riemer et al. BMC Systems Biology 2013, 7:33 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                               Open Access <br />  <br /> A metabolite-centric view flux distributions in <br /> genome-scale metabolic models <br /> S Alexander Riemer&#226;&#8364;&#160;, Ren&#195;&#169; Rex&#226;&#8364;&#160; Dietmar Schomburg* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Genome-scale metabolic models important tools systems biology. They permit in-silico <br />   prediction cellular phenotypes mathematical optimisation procedures, importantly flux balance analysis. <br />   Current studies metabolic models consider reaction fluxes isolation. Based recently proposed <br />   metabolite-centric approach, set methods enable analysis interpretation flux <br />   distributions integrated metabolite-centric view. We demonstrate framework used the <br />   refinement genome-scale metabolic models. <br />   Results: We applied metabolite-centric view developed recent metabolic reconstruction of <br />   Escherichia coli. By compiling balance sheets small number currency metabolites, able fully <br />   characterise energy metabolism predicted model identify possibility model refinement in <br />   NADPH metabolism. Selected branch points examined in order demonstrate metabolite-centric <br />   view allows identifying functional roles metabolites. Fructose 6-phosphate aldolase sedoheptulose <br />   bisphosphate bypass identified enzymatic reactions carry high fluxes model unlikely <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />   exhibit significant activity vivo. Performing metabolite essentiality analysis, unconstrained import export iron <br />   ions identified potentially problematic quality model predictions. <br />   Conclusions: The system-wide analysis split ratios branch points allows deeper insight the <br />   metabolic network reaction-centric analyses. Extending earlier metabolite-centric approach, methods <br />   introduced establish integrated metabolite-centric framework interpretation flux distributions in <br />   genome-scale metabolic networks complement classical reaction-centric framework. Analysing fluxes and <br />   metabolic context simultaneously opens door systems biological interpretations apparent <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />   isolated reaction fluxes. Particularly powerful demonstrations analyses complete metabolic <br />   contexts energy metabolism folate-dependent one-carbon pool presented work. Finally, a <br />   metabolite-centric view flux distributions guide refinement metabolic reconstructions specific growth <br />   scenarios. <br />   Keywords: Split ratios, Branch points, Metabolic modelling, Flux balance analysis, Metabolic reconstruction, Constraint- <br />   based modelling, Stoichiometric matrix, Linear programming, iJO1366, iAF1260 <br />  <br />  <br /> Background                                                                           number genome-scale metabolic reconstructions has <br /> The advent rapid sequencing technologies, collec-                             growing steadily, recent review lists metabolic <br /> tion sharing omics data electronic databases,                          reconstructions organisms [2]. <br /> the development wealth computational tools sys-                             Whole-genome metabolic models play dual role sys- <br /> tems biology allowed reconstruction metabolic                            tems biology: Firstly, structured knowledge bases <br /> networks genome-wide scale. Since whole-                              integrate data metabolism modelled <br /> genome metabolic model published 1999 [1],                                organism wide variety sources, including data- <br />                                                                                      bases, primary literature, expert knowledge [3]. Sec- <br /> * Correspondence: d.schomburg@tu-bs.de                                               ondly, used predict cellular phenotypes, <br /> &#226;&#8364;&#160; <br />  Equal contributors                                                                  allows, inter alia, in-silico prediction ef- <br /> Department Bioinformatics Biochemistry, Technische Universit&#195;&#164;t <br /> Braunschweig, Braunschweig, Germany                                                  fects genetic regulatory modifications growth in <br />  <br />                                        &#194;&#169; 2013 Riemer et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                        reproduction medium, provided original work properly cited. <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                     Page 2 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> different environments <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span> [4]. Thus,        regulatory suppression gene product mod- <br /> are useful tools generating hypotheses guide wet-lab    elled constraining flux reactions catalysed <br /> experiments providing context interpret-       gene product zero [18]. Of particular importance <br /> ation experimental data. In addition, metabolic models      gene expression data, usually incorpo- <br /> are needed metabolic engineering, directed modifi-     rated form constraints individual reaction <br /> cation organism respect metabolic goal        fluxes [19-22]. Furthermore, rapid phenotyping data (such <br /> such overproduction export industrially         BIOLOG phenotype microarrays) employed in <br /> relevant chemical compound [5].                                form binary variables (growth/ growth par- <br />   Escherichia coli far best-studied prokaryote       ticular substrate) [15,23]. <br /> and important model organism biology. Its metab-            While performance models initially be <br /> olism particularly understood, enzymes        analysed terms growth rate efficiency, most <br /> and biochemical pathways described E. coli       specific tool validation metabolic models the <br /> (e.g. [6-9]). The latest whole-genome metabolic recon-         comparison predicted flux distributions experimen- <br /> struction E. coli K-12 MG1655, designated iJO1366           tally determined fluxes, usually obtained 13C-labeling <br /> [10], comprehensive genome-scale            experiments [11,24]. Due lower concentrations of <br /> metabolic models available. It extension earl-    metabolites peripheral pathways, method limited <br /> ier models iAF1260 [11] iJR904 [12].                       fluxes close central carbon metabolism [25-27]. <br />   Flux balance analysis related methods ana-          Model-based metabolic engineering employs a <br /> lysis whole-genome metabolic models produce vector        reaction-centric view. The commonly used ap- <br /> of reaction fluxes. These usually given abso-    proaches try modify individual reaction fluxes re- <br /> lute values mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 percentage          direct metabolic flux combination gene <br /> substrate uptake rate. Such values are, however, hard     knockouts leading overproduction desired me- <br /> to interpret. For instance, does mean flux      tabolite [28-31]. <br /> from 2-oxoglutarate L-glutamate glutamate de-              A metabolite-centric approach analysis of <br /> hydrogenase 6.3 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1? Or rate         genome-scale metabolic models introduced 2007 <br /> aspartate formation aspartate transaminase 22%       [32] expanded 2009 [33]. This approach, termed <br /> the glucose uptake rate?                                       flux-sum analysis, employed successfully the <br />   All genome-scale metabolic models published date          prediction metabolite essentiality studying the <br /> are stoichiometric models, incorporate reaction          robustness metabolic networks perturbations me- <br /> stoichiometry require knowledge kin-         tabolite turnover rates. To knowledge, application <br /> etic parameters. While allow analysis       far limited study network robustness, <br /> non-equilibrium dynamics, stoichiometric models           fundamental network property [33] the <br /> been useful predicting metabolic cap-       specific goal identifying novel drug targets [34]. <br /> abilities steady state [12-15].                                The concept branch points flux split ratios has <br />   Flux balance analysis (FBA) introduced 1986           used wet-lab metabolic flux analysis using small- <br /> [16], decade genome-scale         scale metabolic models [35-37], particularly context <br /> metabolic reconstruction, com-         metabolic engineering projects [38-40]. There exist <br /> monly used method predicting steady-state fluxes        implementations FBA explicitly support linear con- <br /> metabolic networks. Its core idea assumption       straints defined terms split ratios [41,42]. However, <br /> wild-type organisms optimised            knowledge, concept applied sys- <br /> course evolution [17], e.g. rapid efficient    tematically analysis (predicted) flux distribu- <br /> growth.                                                        tion whole-genome metabolic reconstruction. <br />   Oberhardt et al. [4] list following major goals com-        Here system-wide analysis split ra- <br /> monly pursued analyses whole-genome metabolic re-        tios branch points, leading metabolite-centric <br /> constructions: (1) Contextualisation high-throughput        view flux distributions, allows deeper insight <br /> data, (2) guidance metabolic engineering, (3) directing     metabolic network based results flux <br /> hypothesis-driven discovery, (4) interrogation multi-       balance analysis. For purpose, introduce set of <br /> species relationships, (5) network property discovery.     methods generate metabolite-centric description <br /> Analyses pertaining goals currently        based given flux distribution. Moreover, demon- <br /> performed chiefly reaction-centric framework.             strate system-wide analysis metabolite essenti- <br />   Contextualisation high-throughput data achieved        ality, possible metabolite-centric <br /> by adding constraints model modifying existing       framework, identify potential problems network <br /> constraints [4]. Usually, changes pertain iso-   reconstruction. We how, e.g., energy me- <br /> lated reaction fluxes. For instance, gene <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>knockout</span>      tabolism characterised examining flux split <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                               Page 3 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> ratios small number metabolite nodes. We                    molecules ATP glucose molecule. The largest part <br /> also demonstrate metabolite-centric view                     (61.4% total ATP flux) produced oxidative <br /> used refinement models experimental                    phosphorylation. 25.5% produced glycolysis, while <br /> data, particular identifying enzymatic reactions                8.5% produced conversion acetyl-CoA to <br /> known occur vivo known specific sce-                    acetate, excreted. While 4.0% total pro- <br /> narios only, i.e. reactions enzymes presence                  ducing ATP flux stems reaction ADP with <br /> the model lower quality model&#226;&#8364;&#8482;s predic-                    diphosphate, 0.7% produced TCA cycle. <br /> tions some, possibly most, scenarios.                                     The ratios ATP-consuming fluxes shown in <br />                                                                           Table 2. Most ATP consumed biomass re- <br /> Results                                                                   action growth-associated maintenance requirement <br /> FBA performed chosen scenario, carbon-                        (GAM; Methods) (66.2%), fuelling growth-related pro- <br /> limited aerobic growth glucose minimal medium                     cesses. The second largest fraction glycolysis, which <br /> forced acetate secretion (see Methods section),                  consumes 8.8% overall positive ATP bal- <br /> following constraints: Glucose uptake limited rate               ance. 6.2% used biosynthesis L-amino acids, <br /> of 11.0 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, lower bound                      one-third accounted L-glutamine alone <br /> 6.4 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 imposed acetate export                      (2.1% ATP flux), followed L-threonine (1.3%) <br /> flux. For scenario, FBA predicted growth rate                   L-arginine (1.1%). The non-growth associated main- <br /> 0.85 h&#226;&#710;&#8217;1, oxygen uptake rate 16.5 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1,                   tenance requirement (NGAM; Methods) accounts for <br /> and CO2 emission rate 18.2 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. In                    4.5% total ATP consumption, purine metabolism <br /> FBA solution, 552 2583 reactions (21%) involv-                consumes 3.4%, 2.7% used (re)phosphoryl- <br /> ing 557 1805 metabolites (31%) active (at cutoff             ation AMP ADP. Another 2.7% consumed fatty <br /> of 10&#226;&#710;&#8217;12 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1). Of 557 active metabolites,                  acid biosynthesis, 1.6% converted nu- <br /> 401 unbranched paths, i.e. pro-                 cleoside triphosphates GTP (0.9%), UTP (0.5%), CTP <br /> duced single reaction consumed sin-                 (0.2%). 1.2% ATP spent pyrimidine metabolism, <br /> gle reaction. These metabolites purely transitory              1.1% used pentose phosphate pathway the <br /> scenario consideration. Consequently,                      production phosphoribosyl pyrophosphate, an- <br /> remaining 156 metabolites (8.6% metabolite nodes                0.6% consumed sulfate assimilation. 0.2% the <br /> 28% active metabolites) branch points,                   total ATP enters biomass directly. The rest used in <br /> fluxes split join.                                                     biosynthesis biomass components, including <br />                                                                           cell wall components (peptidoglycans), deoxyribonucleoside <br /> Balance sheets currency metabolites                                    triphosphates, glycogen, lipopolysaccharides, phospholipids, <br /> Split-ratio analysis used explore fate currency             various cofactors. <br /> metabolites, importantly energy-rich phosphates                         The fluxes nucleoside triphosphates <br /> (such ATP, GTP, phosphoenolpyruvate (PEP))                     low (GTP: 0.64, UTP: 0.34, CTP: 0.25 mmol gDW&#226;&#710;&#8217;1 <br /> reducing equivalents (chiefly NADH NADPH).                            h&#226;&#710;&#8217;1), contributions energy metabolism are <br />                                                                           substantial compared ATP PEP. <br /> ATP <br /> Table 1 lists relative contributions reactions pro-                Phosphoenolpyruvate (PEP) <br /> ducing ATP. The total flux ATP 69.7 mmol                       The split ratios PEP shown Table 3. The total <br /> gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, corrected salvage reac-                    flux PEP 16.3 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. As expected, <br /> tions, diphosphate, regenerate                     PEP produced entirely glycolysis enolase reac- <br /> &#226;&#8364;&#339;spent&#226;&#8364;? ATP. This correction yields ATP production                     tion. 67.5% PEP used phosphotransferase <br /> rate 66.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, i.e. net yield 6.1                    glucose import phosphorylation, yielding <br />                                                                           pyruvate second product. 15.4% consumed the <br /> Table 1 Split ratios ATP-producing fluxes                              anaplerotic reaction catalysed PEP carboxylase, while <br />                             Producing fluxes                              13.0% used phosphorylating dihydroxyacetone di- <br /> Reaction/Enzyme                  Pathway/Product                  Ratio   hydroxyacetone phosphate. Another 3.9% consumed in <br /> ATP synthase                     oxidative phosphorylation       61.35%   reactions shikimate pathway, produces <br /> phosphoglycerate kinase          glycolysis                      25.47% <br />                                                                           precursors aromatic amino acids. Small quan- <br />                                                                           tities PEP consumed pathways producing <br /> acetate kinase                   acetate export                   8.46% <br />                                                                           peptidoglycans (0.14%) lipopolysaccharides (0.13%). <br /> polyphosphate kinase             ADP + diphosphate                4.02%      The large flux dihydroxyacetone (DHA) kinase <br /> succinyl-CoA synthetase          TCA cycle                        0.71%   unexpected, DHA known intracellular <br /> Total flux: 69.7 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown.            metabolite E. coli usable carbon source [43] <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                               Page 4 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> Table 2 Split ratios ATP-consuming fluxes                                  Table 3 Balance sheet phosphoenolpyruvate <br />                              Consuming fluxes                                                             Producing fluxes <br /> Reaction/Enzyme                    Pathway/Product                    Ratio   Reaction/Enzyme                             Pathway/Product         Ratio <br /> Biomass (incl. GAM)                                                 66.39%    enolase                                     glycolysis              100% <br /> phosphofructokinase                glycolysis                         8.77%                               Consuming fluxes <br /> ATP maintenance requirement        NGAM                               4.52%   Reaction/Enzyme                             Pathway/Product         Ratio <br /> adenylate kinase                   AMP&#226;&#8224;&#8217;ADP                            2.73%   glucose import                              glycolysis            67.49% <br />                                                                               (phosphotransferase system) <br /> acetyl-CoA carboxylase             fatty acids                        2.68% <br />                                                                               PEP carboxylase                             anaplerotic reactions 15.35% <br /> AICAR biosynthesis PRPP       purines                            2.67% <br /> (5 reactions)                                                                 dihydroxyacetone kinase                     glycolysis            12.99% <br /> glutamine synthetase               Gln                                2.13%   3-phosphoshikimate                          shikimate pathway      1.95% <br />                                                                               1-carboxyvinyltransferase <br /> aspartate kinase                   Thr, Met, Lys, peptidoglycans      1.28% <br />                                                                               3-deoxy-7-phosphoheptulonate synthase shikimate pathway            1.95% <br /> PRPP synthetase                    pentose phosphate pathway          1.11% <br />                                                                               UDP-N-acetylglucosamine                     peptidoglycans         0.14% <br /> nucleoside-diphosphate kinase      GTP                                0.92%   1-carboxyvinyltransferase <br /> carbamate kinase                   pyrimidines, Arg, polyamines       0.75%   3-deoxy-8-phosphooctulonate synthase        lipopolysaccharides    0.13% <br /> homoserine kinase                  Thr                                0.65%   Total flux: 16.3 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown. <br /> nucleoside-diphosphate kinase      UTP                                0.49% <br /> shikimate kinase                   shikimate pathway                  0.46%   optimum growth. If flux F6P aldolase is <br /> UMP kinase                         pyrimidines                        0.44% <br />                                                                               constrained condensation direction (i.e. producing <br />                                                                               F6P glyceraldehyde 3-phosphate DHA), of <br /> acetylglutamate kinase             Arg, polyamines                    0.40% <br />                                                                               glycolytic flux fructose <br /> argininosuccinate synthase         Arg                                0.35%   bisphosphate aldolase reaction sedoheptulose <br /> adenylyl-sulfate kinase            sulfur metabolism                  0.30%   bisphosphate bypass (first described [46]) order to <br /> sulfate adenyltransferase          sulfur metabolism                  0.30%   achieve optimum growth. <br /> guanylate kinase                   purines                            0.29%     By combining balance sheets ATP PEP, it <br /> GMP synthase                       purines                            0.29% <br />                                                                               possible compute net balance glycolysis: It <br />                                                                               generates 1.6 molecules ATP 1.5 PEP glucose <br /> asparagine synthetase              Asn                                0.29% <br />                                                                               molecule, consuming 0.6 ATP 1.2 PEP, i.e. the <br /> glutamate 5-kinase                 Pro                                0.26%   balance positive net production 1.1 ATP and <br /> cytidylate kinase                  pyrimidines                        0.20%   0.3 PEP molecule glucose. This corresponds to <br /> CTP synthase (glutamine)           pyrimidines                        0.19%   67.4% theoretical maximum 2 molecules of <br /> glucose-1-phosphate                glycogen                           0.19%   ATP PEP [47], i.e. one-third flux branches off <br /> adenylyltransferase                                                           glycolysis pentose phosphate pathway to <br /> nucleoside-diphosphate kinase      CTP                                0.17%   anabolic pathways producing biomass components. <br /> 6 reactions cell wall           peptidoglycans                     0.17% <br /> biosynthesis                                                                  NADH <br /> 7 reactions lipopolysaccharide lipopolysaccharides                 0.13%   Equally important energy balance cell are <br /> biosynthesis                                                                  reducing equivalents NADH NADPH. The cor- <br /> phospholipid export                phospholipids                      0.11%   responding balance sheets shown Tables 4 5. <br /> (2&#195;&#8212;3 reactions)                                                               The flux NADH 33.7 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. <br /> ATP phosphoribosyltransferase      His, purines                       0.11%   52.7% produced glycolysis, 32.1% gen- <br /> Total flux: 69.7 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown.                erated formation acetyl-CoA pyruvate, <br /> Abbreviations: AICAR &#226;&#8364;&#8220; 5-aminoimidazole-4-carboxamide ribotide, Arg, Asn, &#226;&#8364;&#166;   8.1% produced TCA cycle. 6.2% of <br /> &#226;&#8364;&#8220; biosynthesis L-amino acids, PRPP &#226;&#8364;&#8220; phosphoribosyl pyrophosphate. <br />                                                                               NADH generated byproduct amino acid <br />                                                                               biosynthesis pathways, rest produced a <br /> or degradation glycerol [44,45], occurs                  byproduct purine folate metabolism, respectively. <br /> very low rate particular scenario (5.5 &#206;&#188;mol gDW&#226;&#710;&#8217;1                     Almost NADH used energy metabolism: <br /> h&#226;&#710;&#8217;1, FVA: 4&#226;&#8364;&#8220;19 &#206;&#188;mol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1). In FBA solution,                          94.5% enters electron transport chain NADH de- <br /> DHA produced large rate (2.1 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1)                          hydrogenase. Most remaining NADH (5.1%) is <br /> via fructose-6-phosphate (F6P) aldolase, cleaves F6P                    consumed fatty acid biosynthesis, 0.4% used <br /> into glyceraldehyde 3-phosphate DHA. FVA used                         biosynthesis L-methionine. This consistent <br /> to verify activity reaction required                  general observation bacteria grown <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                 Page 5 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> Table 4 NADH balance                                                       transhydrogenase, 21% ICD, 37% PPP. At <br />                                Producing fluxes                            time, FBA solution slightly better at <br /> Reaction/Enzyme                         Pathway/Product           Ratio    predicting oxygen uptake rate (16.7 vs. 16.5 mmol <br /> glyceraldehyde-3-phosphate              glycolysis               52.74% <br />                                                                            gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1; experimental value: 18.2 &#194;&#177; 0.8 mmol gDW&#226;&#710;&#8217;1 <br /> dehydrogenase                                                              h&#226;&#710;&#8217;1 [49]) obtained original parameters, <br /> pyruvate dehydrogenase                  glycolysis/TCA cycle     32.14%    predicting roughly values growth rate <br />                                                                            CO2 emission rate. <br /> malate dehydrogenase                    TCA cycle                 5.35% <br />                                                                               Apart biomass component itself, NADPH <br /> phosphoglycerate dehydrogenase          Ser                       4.31% <br />                                                                            consumed 33 reactions flux 0.1 &#206;&#188;mol <br /> 2-oxoglutarate dehydrogenase            TCA cycle                 2.78%    gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. 77.8% used biosynthesis various <br /> 3-isopropylmalate dehydrogenase         Leu                       1.11%    amino acids, two-thirds (51.7% total <br /> IMP dehydrogenase                       purines                   0.60%    NADPH) accounted L-glutamate alone, followed <br /> histidinol dehydrogenase                His                       0.47%    L-threonine (6.5%) L-lysine (4.1%). The biosyn- <br />                                                                            theses branched-chain amino acids together <br /> prephenate dehydrogenase                Tyr                       0.34% <br />                                                                            consume 7.0% NADPH. Other major processes <br /> glycine cleavage                 folate metabolism         0.14% <br />                                                                            consuming NADPH fatty acid biosynthesis (13.4%) <br />                                Consuming fluxes                            sulfur assimilation (4.6%). 2.1% used restoring <br /> Reaction/Enzyme                         Pathway/Product           Ratio    thioredoxin reduced state, turn used for <br /> NADH dehydrogenase                      oxidative phosphorylation 94.48%   sulfur assimilation (71%) biosynthesis deoxyribonu- <br /> enoyl-[acyl-carrier-protein]            fatty acids               5.10%    cleotides (29%). 1.0% NADPH used phospho- <br /> reductase (11 reactions)                                                   lipid biosynthesis, remainder consumed the <br /> methylenetetrahydrofolate               Met                       0.40%    biosynthesis biomass components, including <br /> reductase                                                                  polyamines, peptidoglycans, lipopolysaccharides, vari- <br /> Total flux: 33.7 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown.             ous cofactors. 0.16% consumed folate metabolism. <br /> Abbreviations: His, Leu, &#226;&#8364;&#166; &#226;&#8364;&#8220; biosynthesis L-amino acids. <br />                                                                            Both lower flux NADPH compared NADH <br />                                                                            distribution NADPH-consuming flux over <br /> aerobically sugars, NADH produced primarily                       large number reactions various biosynthetic path- <br /> the catabolic pathways glycolysis TCA cycle                     ways good agreement paradigm while <br /> used exclusively ATP generation [47].                           NADH used energy metabolism, NADPH acts, <br />                                                                            functions, reducing agent anabolic <br /> NADPH                                                                      pathways [47]. <br /> At 13.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, flux NADPH is <br /> 59% lower NADH. The largest                         Selected branch points <br /> (80.7%) stems oxidative pentose phosphate path-                   A detailed analysis split ratios selected metabolite <br /> way (PPP). Another 13.4% produced enzyme                         nodes performed goals elucidating the <br /> isocitrate dehydrogenase (ICD) TCA cycle,                     metabolic roles metabolite species identifying the <br /> 5.9% generated byproduct purine metabolism. It                  relative contributions biochemical pathways biomass <br /> has shown 13C tracer experiments E. coli                   formation energy generation, respectively, sce- <br /> grown aerobically glucose, 35&#226;&#8364;&#8220;45% NADPH                       nario consideration. <br /> produced NADH NADP+ membrane-                                Table 6 shows fluxes producing consuming 2- <br /> bound proton-translocating transhydrogenase PntAB,                         oxoglutarate. The total flux metabolite 8.1 <br /> while 35&#226;&#8364;&#8220;45% produced PPP, 20&#226;&#8364;&#8220;                       mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, 77.0% generated a <br /> 25% produced ICD [48]. In contrast, FBA predicted                    byproduct various transaminases, transfer the <br /> no transhydrogenase activity model, FVA con-                   amino group L-glutamate different acceptors, produ- <br /> firmed constraint maximum biomass                        cing L-amino acids. In contrast, 23.0% produced de <br /> production, significant transhydrogenase flux pos-                   novo isocitrate TCA cycle. The consuming side <br /> sible. The prediction malic enzyme does play                    reflects presence transamination cycle: 88.5% <br /> significant role NADPH generation scenario                      2-oxoglutarate converted L-glutamate, while <br /> under consideration (confirmed FVA) agreement                     11.5% metabolised TCA cycle. <br /> with experimental observations [48].                                     Subtracting flux 6.3 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 going <br />    An exploratory FBA alternative scenario                    transamination cycle yields net balance, in <br /> forced transhydrogenase flux 5 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 yielded                   100% 2-oxoglutarate (1.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1) is <br /> a flux distribution agreement results [48],                 produced TCA cycle. Of non-cyclic flux, 50.1% <br /> predicting 36% NADPH produced                              continues TCA cycle, 49.8% converted L- <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                Page 6 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> Table 5 NADPH balance <br />                                                                  Producing fluxes <br /> Reaction/Enzyme                                                               Pathway/Product                                      Ratio <br /> glucose 6-phosphate dehydrogenase                                             pentose phosphate pathway                           40.35% <br /> phosphogluconate dehydrogenase                                                pentose phosphate pathway                           40.35% <br /> isocitrate dehydrogenase                                                      TCA cycle                                           13.40% <br /> methylenetetrahydrofolate dehydrogenase                                       purines                                              5.89% <br />                                                                  Consuming fluxes <br /> Reaction/Enzyme                                                               Pathway/Product                                      Ratio <br /> glutamate dehydrogenase                                                       Glu                                                 51.65% <br /> 3-oxoacyl-[acyl-carrier-protein] reductase (11 reactions)                     fatty acids                                         13.41% <br /> ketol-acid reductoisomerase (2 reactions)                                     Val, Leu, Ile, coenzyme A                            6.95% <br /> aspartate semialdehyde dehydrogenase                                          Thr, Met, Lys, peptidoglycans                        6.43% <br /> sulfite reductase                                                             sulfur metabolism                                    4.57% <br /> homoserine dehydrogenase                                                      Thr, Met                                             4.21% <br /> shikimate dehydrogenase                                                       shikimate pathway                                    2.28% <br /> dihydrodipicolinate reductase                                                 Lys, peptidoglycans                                  2.22% <br /> thioredoxin reductase                                                         conversions redox carriers                  2.14% <br /> N-acetyl-&#206;&#179;-glutamyl-phosphate reductase                                       Arg, polyamines                                      2.01% <br /> glutamate 5-semialdehyde dehydrogenase                                        Pro                                                  1.32% <br /> pyrroline 5-carboxylate reductase                                             Pro                                                  1.32% <br /> glycerol 3-phosphate dehydrogenase                                            phospholipids                                        1.03% <br /> UDP-N-acetylenolpyruvoylglucosamine reductase                                 peptidoglycans                                       0.17% <br /> dihydrofolate reductase                                                       folate metabolism                                    0.16% <br /> dTDP-4-dehydrorhamnose reductase                                              lipopolysaccharides                                  0.05% <br /> Total flux: 13.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown. <br /> Abbreviations: Arg, Glu, &#226;&#8364;&#166; &#226;&#8364;&#8220; biosynthesis L-amino acids. <br />  <br />  <br />  <br /> glutamate starting molecule various biosynthetic                    depth) split ratios Figure 1. All G6P gener- <br /> pathways.                                                                 ated glucose phosphotransferase system, <br />   A second transamination cycle exists L-                         main consumer PEP main <br /> glutamine amino-group donor L-glutamate. The                       producer pyruvate. This illustrates how, scenario <br /> balance sheet L-glutamate (Table 7) shows trans-                 consideration, phosphotransferase ef- <br /> amination cycles: Of total flux 8.5 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1                 fectively couples step glycolysis (glucose uptake <br /> going L-glutamate, 85.0% produced 2-                      phosphorylation) step (dephosphoryla- <br /> oxoglutarate, 15.0% result transamination                 tion PEP pyruvate) [50]. Only 47.4% G6P con- <br /> with L-glutamine amino-group donor. 74.0% used                      tinues glycolytic path, 51.1% enters the <br /> for transferring amino groups, 17.6% converted                pentose phosphate pathway (PPP). The remaining 1.5% is <br /> L-glutamine.                                                              converted glucose 1-phosphate, 79.1% used <br />   The net balance L-glutamate fluxes                   build glycogen, rest consumed re- <br /> two transamination cycles removed shown Table 8.                    actions lipopolysaccharide biosynthesis. It be <br /> The remaining 0.93 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, pro-                         noted carbon initially enters PPP <br /> duced exclusively 2-oxoglutarate, chiefly used               returns glycolytic route transaldolase trans- <br /> the biosynthesis amino acids L-arginine (26.4%), L-                ketolase, respectively. Of 66.0 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 flux <br /> glutamine (23.5%), L-proline (19.8%),                   carbon atoms G6P, 48.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 <br /> 23.5% enters biomass directly. The rest consumed                (74.1%) reach PEP glycolysis. The fluxes the <br /> the biosynthesis polyamines (3.7%), peptidoglycans                     PPP connections glycolysis shown <br /> (2.5%), various cofactors.                                            Additional file 1: Figure S1. <br />   Glucose 6-phosphate (G6P) branch point                       Figure 2 shows split ratios acetate meta- <br /> in glycolysis. It shown metabolic context (two levels               bolic context. Acetate produced secreted the <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                            Page 7 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> Table 6 Split ratios 2-oxoglutarate                                    Table 7 Split ratios L-glutamate <br />                             Producing fluxes                                                          Producing fluxes <br /> Reaction/Enzyme                            Pathway/Product        Ratio   Reaction/Enzyme                              Pathway/Product         Ratio <br /> aspartate transaminase                     Asp                   29.99%   glutamate dehydrogenase                      TCA cycle             85.02% <br /> isocitrate dehydrogenase                   TCA cycle             22.96%   amidophosphoribosyltransferase               purines                4.40% <br /> phosphoserine transaminase                 Ser                   17.85%   phosphoribosylformylglycinamidine            purines                4.40% <br />                                                                           synthase <br /> L-alanine transaminase                     Ala                    5.78% <br />                                                                           GMP synthase                                 purines                2.39% <br /> leucine transaminase                       Leu                    4.61% <br />                                                                           CTP synthase                                 CTP                    1.57% <br /> valine transaminase                        Val                    4.33% <br />                                                                           imidazole-glycerol-3-phosphate synthase      His, purines           0.93% <br /> succinyldiaminopimelate transaminase       Lys, peptidoglycans    3.80% <br />                                                                           glutamine-fructose-6-phosphate               peptidoglycans,        0.72% <br /> acetylornithine transaminase               Arg, polyamines        3.45%   transaminase                                 lipopolysaccharides <br /> isoleucine transaminase                    Ile                    2.97%   anthranilate synthase                        Trp                    0.56% <br /> phenylalanine transaminase                 Phe                    1.89%                              Consuming fluxes <br /> tyrosine transaminase                      Tyr                    1.41%   Reaction/Enzyme                              Pathway/Product         Ratio <br /> histidinol-phosphate transaminase          His                    0.97%   aspartate transaminase                       Asp                   28.82% <br />                            Consuming fluxes                               glutamine synthetase                         Gln                   17.57% <br /> Reaction/Enzyme                            Pathway/Product        Ratio <br />                                                                           phosphoserine transaminase                   Ser                   17.15% <br /> glutamate dehydrogenase                    Glu                   88.48% <br />                                                                           L-alanine transaminase                       Ala                    5.55% <br /> 2-oxoglutarate dehydrogenase               TCA cycle             11.51% <br />                                                                           leucine transaminase                         Leu                    4.43% <br /> Total flux: 8.12 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown. <br /> Abbreviations: Ala, Arg, &#226;&#8364;&#166; &#226;&#8364;&#8220; biosynthesis L-amino acids. <br />                                                                           valine transaminase                          Val                    4.16% <br />                                                                           succinyldiaminopimelate transaminase         Lys, peptidoglycans    3.65% <br />                                                                           N-acetylglutamate synthase                   Arg, polyamines        3.31% <br /> forced rate 6.4 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. While acetate is <br /> produced byproduct arginine metabolism (4.4%),                    acetylornithine transaminase                 Arg, polyamines        3.31% <br /> L-cysteine biosynthesis (3.3%), biosynthesis li-               isoleucine transaminase                      Ile                    2.85% <br /> popolysaccharides (0.2%), largest (92.1%) pro-                Biomass                                                             2.59% <br /> duced acetyl-CoA phosphotransacetylase                       glutamate 5-kinase                           Pro                    2.17% <br /> acetate kinase. At rate acetate secretion,              phenylalanine transaminase                   Phe                    1.82% <br /> 50% acetyl-CoA takes acetate-exporting route in- <br />                                                                           tyrosine transaminase                        Tyr                    1.36% <br /> stead TCA cycle. FVA used verify indeed <br /> the forced acetate secretion sole reason activity              histidinol-phosphate transaminase            His                    0.93% <br /> in route: If constraint disabled, acetate kinase             glutamate racemase                           peptidoglycans         0.28% <br /> and phosphotransacetylase work opposite direction                  glutamyl-tRNA synthetase                     cobalamin, heme        0.05% <br /> to salvage acetate produced byproduct                     Total flux: 8.46 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown. <br /> other pathways. FVA confirmed route                 Abbreviations: Ala, Arg, &#226;&#8364;&#166; &#226;&#8364;&#8220; biosynthesis L-amino acids. <br /> cheapest way regard biomass yield produce <br /> acetate required rate.                                             described above, 50.1% 2-oxoglutarate con- <br />                                                                           tinues TCA cycle, forming succinyl-CoA. All the <br /> Metabolic cycles metabolite pools                                     succinyl-CoA eventually converted succinate, 52.7% <br /> Branch point analysis permits visualisation                  TCA cycle, 33.0% L-lysine biosynthesis, and <br /> pathways actual fluxes. This particularly useful                  14.3% biosynthesis L-methionine. All suc- <br /> for metabolic cycles metabolite pools                        cinate oxidized fumarate, succinate dehydro- <br /> depleted replenished different reactions.                    genase sole source fumarate. Virtually of <br /> Figure 3 shows fluxes TCA cycle sce-                    fumarate converted oxaloacetate fumarase <br /> nario consideration.                                                malate dehydrogenase, largest oxa- <br />   As mentioned above, one-sixth acetyl-                 loacetate (58.1%) results anaplerotic reaction <br /> CoA enters TCA cycle, majority converted                 catalysed PEP carboxylase. <br /> to acetate secretion. All flux entering citrate syn-                 Oxaloacetate second major branch point the <br /> thase goes isocitrate dehydrogenase, i.e.                TCA cycle. 56.6% flux node is <br /> the citrate converted 2-oxoglutarate,            redirected anabolism, forming L-aspartate the <br /> of major branch points anabolic pathways. As                  step. However, branch point analysis revealed the <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                  Page 8 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> Table 8 Split ratios L-glutamate discounting                           predicted ones &#226;&#8364;&#8220; instance, flux through <br /> transamination cycles                                                     isocitrate dehydrogenase measured 27&#226;&#8364;&#8220;33% of <br />                             Producing fluxes                              glucose uptake rate, predicted value 17%. <br /> Reaction                               Pathway                    Ratio      In alternative scenario forced transhydrogenase <br /> glutamate dehydrogenase                TCA cycle                  100%    flux 5 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, flux TCA cycle <br />                            Consuming fluxes <br />                                                                           considerably larger (isocitrate dehydrogenase: 2.9 vs. 1.9 <br />                                                                           mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1), closer experimental <br /> Reaction                               Pathway                    Ratio <br />                                                                           observations &#226;&#8364;&#8220; 2.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 amounts 26% of <br /> N-acetylglutamate synthase             Arg, polyamines           30.10% <br />                                                                           glucose uptake rate 11 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. As result, <br /> glutamine synthetase                   Gln                       23.51%   fraction NADH produced TCA cycle in- <br /> Biomass                                                          23.51%   creases 8.1% 12.9%. <br /> glutamate 5-kinase                     Pro                       19.75%      5,6,7,8-Tetrahydrofolate (THFA), activated <br /> glutamate racemase                     peptidoglycans             2.54%   forms 5,10-methylene-THFA, 5-methyl-THFA, 10- <br />                                                                           formyl-THFA, cell&#226;&#8364;&#8482;s important donors <br /> glutamyl-tRNA synthetase               cobalamin, heme            0.49% <br />                                                                           one-carbon groups. In addition, act redu- <br /> dihydrofolate synthase                 folate metabolism          0.08% <br />                                                                           cing agent. Figure 5 shows fluxes folate- <br /> Total flux: 0.93 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Ratios 0.05% shown. <br />                                                                           dependent one-carbon pool. <br /> Abbreviations: Arg, Gln, Pro &#226;&#8364;&#8220; biosynthesis L-amino acids. <br />                                                                              The total flux THFA 0.98 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, <br />                                                                           0.08% stems de-novo synthesis, while <br /> presence cycle oxaloacetate fumarate                     THFA circulates. Almost THFA is <br /> (shown Figure 4). In cycle, carries flux               converted 5,10-methylene-THFA, transferred <br /> 0.87 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, L-aspartate formed oxaloace-                    methylene group stems cleavage L-serine <br /> tate transfers amino group different acceptors                  (94.8%) glycine (4.9%). 0.23% converted 10-formyl <br /> purine arginine metabolism, respectively, yielding fu-                -THFA instead, reassimilating formate cost ATP <br /> marate, converted oxaloacetate                    hydrolysis. The largest 5,10-methylene-THFA <br /> TCA cycle. Correcting cycle yields ratio 54.3%              (84.0%) subsequently converted 10-formyl-THFA, <br /> of oxaloacetate remaining cycle 45.7% entering                 transfers formyl group reactions bio- <br /> anabolic pathways. The fraction anaplerotic oxaloace-                  synthesis purine nucleotides. 13.8% 5,10-methy- <br /> tate (produced PEP carboxylase) increases 72.8%                     lene-THFA converted 5-methyl-THFA, donates <br /> with correction.                                                     methyl group biosynthesis L-methionine. An- <br />   While predicted fraction acetyl-CoA converted                 2.2% consumed biosynthesis dTMP, where <br /> acetate (54%) close value determined Fischer                 donates methylene group electrons, <br /> et al. [49] (55&#226;&#8364;&#8220;59%), fraction predicted enter                 resulting dihydrofolate depleted form, is <br /> TCA cycle considerably lower (17% vs. 25&#226;&#8364;&#8220;30%). The                     subsequently reduced THFA. Finally, 5,10-methy- <br /> measured fluxes TCA cycle twice high                 lene-THFA transfers methylene group step of <br />  <br />  <br />  <br />  <br />  Figure 1 Split ratios glucose 6-phosphate adjacent nodes. Green: Metabolite nodes, yellow: enzyme nodes. Numbers metabolite <br />  nodes total flux mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Edge labels split ratios fractions flux adjacent metabolite node. <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                     Page 9 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />  Figure 2 Split ratios acetate adjacent nodes. Green: Metabolite nodes, yellow: enzyme nodes. Numbers metabolite nodes total <br />  flux mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Edge labels split ratios fractions flux adjacent metabolite node. <br />  <br />  <br /> coenzyme A biosynthesis, accounts 0.05%                intracellular metabolites agree findings Kim <br /> total flux.                                                             et al. based earlier model iAF1260 [32]. However, <br />                                                                         study did consider periplasmic extracellular <br /> Essentiality metabolite nodes                                        metabolites. Menaquinone-8, involved twenty- <br /> 548 557 metabolite nodes carrying flux (98%)              reactions, identified promiscuous <br /> identified essential optimal growth metabolite                nonessential metabolite. While menaquinone-8 does not <br /> flux minimisation. The remaining metabolites                   carry flux particular FBA solution used this <br /> shown Table 9, number reactions                 work, FVA revealed reactions involving <br /> which occur. As predicted model, in-               metabolite carry flux constraint <br /> termediates nonessential pathways. The results               optimal biomass production. Of reactions <br />  <br />  <br />  <br />  <br />  Figure 3 Metabolic fluxes split ratios TCA cycle. Green: Metabolite nodes, yellow: enzyme nodes. Numbers metabolite nodes <br />  total flux mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Edge labels split ratios fractions flux adjacent metabolite node. <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                Page 10 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />  Figure 4 Cycle oxaloacetate fumarate. Green: Metabolite nodes, yellow: enzyme nodes. Numbers metabolite nodes total <br />  flux mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Edge labels split ratios fractions flux adjacent metabolite node. <br />  <br />  <br /> involving thioredoxin, seven actually carry flux,            Hydroxypyruvate occurs intermediate reaction <br /> determined FVA.                                                    pair hydroxypyruvate isomerase + hydroxypyruvate reduc- <br />   Adenosine &#206;&#177;-D-ribose 1-phosphate occur salvage             tase, stoichiometrically equivalent single re- <br /> pathway adenine, alternatively converted             action catalysed tartronate semialdehyde reductase. <br /> to AMP step adenine phosphoribosyltransferase.             Thioredoxin replaced reducing agents, as <br /> Fructose 1,6-bisphosphate completely bypassed               model contains alternative reactions catalysed <br /> the model sedoheptulose bisphosphate bypass,             thioredoxin-dependent enzymes. <br /> phosphofructokinase fructose bisphosphate aldolase act               The result active metabolite nodes essen- <br /> on sedoheptulose 7-phosphate pentose phosphate               tial complemented observation according to <br /> pathway sedoheptulose 1,7-bisphosphate instead                    FVA, 76 552 active reactions (14%) nones- <br /> of fructose 6-phosphate fructose 1,6-bisphosphate, re-            sential, i.e. active reactions need carry flux <br /> spectively. Since pathways stoichiometrically               optimal biomass production. <br /> equivalent, use pathway allows optimal bio-                Table 10 shows thirty-two intracellular metabolites <br /> mass production. Periplasmic glycine intermediate               largest minimum turnover rates. Many entries in <br /> in transport cascade net effect importing           list ubiquitous metabolites, H+, H2O, and <br /> Ca2+ ions. As Ca2+ importer available model           phosphate, circulating currency metabolites cofac- <br /> is Ca2+/Na+ antiporter, second transporter                tors, ATP/ADP, NADH/NAD+, coenzyme A. <br /> active pump Na+ ions cytosol. A large number             The list contains large number glycolytic metabo- <br /> Na+ symporters (including glycine, glycolate, acet-           lites, high flux glycolysis, as <br /> ate) present model,             carbon flux initially passes glycolysis and <br /> same energy cost, used organic compounds               branches pathways gradually. Metabolite <br /> can exported periplasm 1:1 H+ symport.                  flux minimisation predicts large required fluxes <br /> Excess intracellular protons exported extracellu-          O2 CO2, expected aerobic <br /> lar space FBA solution. There exists stoichiometric          condition. Due transamination cycle described <br /> alternative periplasmic ferroxidase, produces             above, L-glutamate 2-oxoglutarate high mini- <br /> water protons, oxygen, electrons donated Fe2+,            mum fluxes. Acetate acetyl phosphate appear the <br /> so protons exported medium. The                    list high forced acetate export flux. In addition, <br /> additional import flux Fe2+ required conversion            metabolites pentose phosphate pathway to <br /> of excess protons way 14.0 mmol gDW&#226;&#710;&#8217;1                  carry high flux, constitutes cheapest way for <br /> h&#226;&#710;&#8217;1, equal required Fe3+ export flux.                 model produce NADPH. <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                         Page 11 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />  Figure 5 Metabolite fluxes split ratios tetrahydrofolate pool. Green: Metabolite nodes, yellow: enzyme nodes. Numbers in <br />  metabolite nodes total flux mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Edge labels split ratios fractions flux adjacent metabolite node. <br />  <br />  <br />  <br />   For carbon-containing metabolites, useful                         shown Table 11. Half occur glycolysis, and <br /> ranking total flux, carbon flux, i.e.                 intermediates pentose phosphate <br /> product metabolite flux number carbon atoms                      pathway. The remaining metabolites list are <br /> in compound. The intracellular non-cofactor                     L-glutamate, 2-oxoglutarate, oxaloacetate (all involved in <br /> metabolites highest required carbon fluxes                    transamination cycles), CO2, acetate. The example <br />                                                                            glucose 6-phosphate illustrates usefulness this <br /> Table 9 Metabolites carrying flux predicted                    alternative flux measure: It ranked eighth among <br /> nonessential                                                               carbon metabolites Table 10 active <br /> Metabolite                                          Number reactions    metabolite carbon flux. <br /> adenosine                                                      7 <br /> &#206;&#177;-D-ribose 1-phosphate                                         7           Discussion <br />                                                                            We demonstrated energy metabolism of <br /> fructose 1,6-bisphosphate                                      4 <br />                                                                            cell, predicted model, characterised <br /> glycine[periplasm]                                             4 <br />                                                                            compiling balance sheets handful currency <br /> hydroxypyruvate                                                3           metabolites. The split ratios currency metabolites can <br /> H+[extracellular]                                              6           used check metabolic model biological plausi- <br /> thioredoxin (reduced)                                         10           bility absence specific experimental data for <br /> thioredoxin (oxidized)                                        10           studied organism, energy metabolism tends fol- <br />                                                                            low certain well-studied paradigms solely based ecol- <br /> [cell wall precursor linked disaccharide                3 <br /> pentapeptide tetrapeptide murein units]                                ogy (phototrophic vs. chemotrophic, aerobic vs. anaerobic, <br /> Shown number reactions nonessential metabolite <br />                                                                            etc.). Thus, split-ratio analysis powerful tool plausi- <br /> appears network.                                                    bility checking refinement metabolic models. <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                Page 12 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> Table 10 Intracellular metabolites largest minimum fluxes <br /> Metabolite                                          Minimum flux                     Metabolite                          Minimum flux <br /> H+                                                  239.2                            NADP+                               13.9 <br /> H2O                                                 108.1                            NADPH                               13.9 <br /> ADP                                                 69.7                             pyruvate                            13.3 <br /> ATP                                                 69.7                             coenzyme A                          11.8 <br /> phosphate                                           66.9                             glucose 6-phosphate                 11.0 <br /> NAD+                                                33.7                             acetyl-CoA                          10.8 <br /> NADH                                                33.7                             ammonium                            9.4 <br /> ubiquinol-8                                         33.0                             L-glutamate                         8.5 <br /> ubiquinone-8                                        33.0                             dihydroxyacetone phosphate          8.2 <br /> CO2                                                 23.5                             2-oxoglutarate                      8.1 <br /> glyceraldehyde 3-phosphate                          17.8                             fructose 6-phosphate                6.6 <br /> 3-phospho-D-glycerate                               17.7                             acetate                             6.4 <br /> 1,3-bisphospho-D-glycerate                          17.7                             acetyl phosphate                    5.9 <br /> O2                                                  16.5                             6-phospho-D-gluconate               5.6 <br /> 2-phospho-D-glycerate                               16.3                             6-phospho-D-glucono-1,5-lactone     5.6 <br /> PEP                                                 16.3                             D-ribulose 5-phosphate              5.6 <br /> Shown minimum flux (in mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1) metabolite carry growth rate maximal. <br />  <br />  <br />  <br />  <br /> Table 11 Metabolites largest essential carbon fluxes <br /> Metabolite                                                   C atoms                            Minimum flux           Minimum C flux <br /> glucose 6-phosphate                                          6                                  11                     66 <br /> glyceraldehyde 3-phosphate                                   3                                  17.75                  53.26 <br /> 3-phospho-D-glycerate                                        3                                  17.75                  53.24 <br /> 1,3-bisphospho-D-glycerate                                   3                                  17.75                  53.24 <br /> 2-phospho-D-glycerate                                        3                                  16.3                   48.89 <br /> PEP                                                          3                                  16.3                   48.89 <br /> L-glutamate                                                  5                                  8.46                   42.28 <br /> 2-oxoglutarate                                               5                                  8.12                   40.62 <br /> pyruvate                                                     3                                  13.3                   39.9 <br /> fructose 6-phosphate                                         6                                  6.59                   39.55 <br /> 6-phospho-D-gluconate                                        6                                  5.62                   33.7 <br /> 6-phospho-D-glucono-1,5-lactone                              6                                  5.62                   33.7 <br /> D-ribulose 5-phosphate                                       5                                  5.62                   28.08 <br /> dihydroxyacetone phosphate                                   3                                  8.23                   24.68 <br /> CO2                                                          1                                  23.5                   23.5 <br /> acetyl-CoA                                                   2*                                 10.82                  21.64 <br /> oxaloacetate                                                 4                                  4.3                    17.21 <br /> D-xylulose 5-phosphate                                       5                                  3.09                   15.47 <br /> acetate                                                      2                                  6.4                    12.8 <br /> D-ribose 5-phosphate                                         5                                  2.5                    12.5 <br />                          &#226;&#710;&#8217;1    &#226;&#710;&#8217;1 * <br /> Fluxes mmol gDW        h . discounting CoA moiety. <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                    Page 13 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />   By inspecting selected metabolites detail,         subsequently analysed bio- <br /> demonstrated branch point analysis used           logical significance. <br /> elucidate metabolic roles compounds occurring            The system-wide analysis branch points split ra- <br /> the network scenario consideration. For in-        tios complemented metabolite essentiality analysis, <br /> stance, branch point analysis used determine          implemented using flux-sum analysis metabolite <br /> whether metabolite carrying flux acts mere path-       flux minimisation. It noted type ana- <br /> way intermediate, produced solely biomass com-          lysis possible metabolite-centric view. For <br /> ponent, acts energy redox carrier, acts group        instance, merely examining fluxes flux variabilities <br /> donor, occurs mere byproduct, recycled         twenty-three reactions E. coli model involving <br /> to central metabolism.                                          menaquinone-8 reactions involving thioredoxin <br />   Split ratio analysis used exploratory ap-   reveal metabolites nonessential op- <br /> proach, relevant context interpreting flux       timal growth. In case thioredoxin, seven ten <br /> through reaction provided split ratios re-   fluxes actually carry flux constraint op- <br /> spect flux sums substrates prod-      timal biomass production. <br /> ucts involved. Branch point analysis presents just            In metabolite flux minimisation, nonessential metabol- <br /> context glance. To revisit questions posed      ite identified metabolite minimum flux sum <br /> Background section: The flux glutamate dehydro-         zero constraint optimal biomass flux. In <br /> genase easily interpretable expressed fraction     flux-sum analysis, contrast, nonessential metabolite is <br /> of total producing glutamate flux, total consuming      removed network (modelled by <br /> 2-oxoglutarate flux, total consuming NADPH flux,            restricting fluxes consuming metabolite zero) <br /> and possibly total ammonia-consuming flux. Likewise,        decrease biomass flux [32]. The defini- <br /> the flux aspartate transaminase best expressed       tions obviously equivalent. However, metabolite flux <br /> as fraction total fluxes oxaloacetate, as-     minimisation alters objective function not <br /> partate, glutamate, possibly 2-oxoglutarate.                constraints defining solution space, amenable <br />   Key split ratios, 2-oxoglutarate oxa-    strategy used speeding FVA [53] <br /> loacetate, used identify relative contributions   and, thus, computationally efficient. <br /> of pathway energy generation biomass produc-              Applying metabolite-centric view introduced here <br /> tion, respectively. Hence, metabolite-centric view flux    published model iJO1366 E. coli, discovered <br /> distributions guide metabolic engineering,        number hitherto discussed peculiarities net- <br /> goal optimise split ratios pathways         work behaviour: <br /> leading product formation, biomass production,              A systematic analysis balance sheets en- <br /> energy generation order achieve maximum product           ergy metabolites revealed unexpectedly strong activity of <br /> yield [51,52]. In fact, enzymes pathway producing     dihydroxyacetone kinase, led dis- <br /> desired product overexpressed, effect       covery high activity fructose-6-phosphate (F6P) <br /> an increased flux pathway, shift     aldolase FBA solution. Usually, F6P phosphory- <br /> split ratios, matter entering product-forming        lated fructose 1,6-bisphosphate, only <br /> route diverted energy growth metabolism.            cleaved (into glyceraldehyde 3-phosphate di- <br />   It noted small fraction          hydroxyacetone phosphate). F6P aldolase discovered <br /> metabolites network branch points             E. coli 2000 [54], function vivo remains <br /> given scenario, global network behaviour        obscure. Moreover, known in-vivo ex- <br /> be characterised small number detailed analyses.       pression enzyme. Sch&#195;&#188;rmann Sprenger [54] <br />   By reducing metabolic network               note F6P cleavage reaction 10 kJ mol&#226;&#710;&#8217;1 more <br /> nodes branch points, branch point analysis al-         endergonic fructose-1,6-bisphosphate cleavage <br /> lows visualisation fluxes metabolic         reaction, making unlikely reaction proceed <br /> cycle, metabolite pool, larger subsystems,         cleavage direction reverse direction. <br /> we demonstrated using examples TCA              The central role, tight regulation, ubiquitous prese- <br /> cycle, folate-dependent one-carbon pool,            nce fructose-bisphosphate aldolase glycolytic <br /> superpathway consisting glycolysis pentose           pathways organisms capable classical (Embden- <br /> phosphate pathway. Using advanced graph layouting al-           Meyerhof-Parnas) glycolysis sheds doubt a <br /> gorithms (possibly guided manually drawn pathway           notable alternative active E. coli. While the <br /> map), possible produce clear            reaction pairs F6P kinase + fructose-bisphosphate al- <br /> informative visualisation steady-state fluxes         dolase F6P aldolase + dihydroxyacetone kinase are <br /> the network. In addition, branch point analysis           stoichiometrically equivalent phosphorylat- <br /> allows graphical identification metabolic cycles,        ing agent involved (ATP vs. PEP), model likely <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                     Page 14 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> reflect metabolic behaviour vivo accurately       producing reactions. In absence new experimental <br /> the flux F6P aldolase constrained          data, black-box approach taken, where <br /> disallow cleavage direction. Possibly, reaction          transhydrogenase flux set fixed frac- <br /> should omitted altogether known            tion carbon flux entering system. Of course, <br /> about in-vivo expression regulation enzyme.           ratio NADPH-producing reactions may <br /> The inclusion F6P aldolase model (which         change dramatically depending carbon source and <br /> discussed authors iJO1366 prede-      availability oxygen, constraints are <br /> cessors iAF1260 iJR904) represent case            valid particular scenario carbon-limited <br /> spurious enzyme function, possibly resulting blanket        aerobic growth sugars. <br /> inclusion enzymes automated database             In original scenario, predicted flux going through <br /> search. If added unconstrained flux, cryptic en-       TCA cycle (~ 1.9 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1) low in <br /> zymes dramatically lower quality predic-          comparison acetyl-CoA (10.8 mmol <br /> tions model.                                         gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1) PEP (16.3 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1). In addition, <br />    A systematic survey metabolite essentiality resulted    anabolic fluxes branching TCA cycle very <br /> the equally surprising observation model, fruc-      large (50.1% 2-oxoglutarate 45.7% oxaloacetate). <br /> tose 1,6-bisphosphate nonessential optimal growth.        Taken together, results indicate original <br /> The reason sedoheptulose bisphosphate           scenario, primary purpose TCA cycle, pre- <br /> bypass, equivalent standard reactions         dicted model, energy generation produc- <br /> phosphofructokinase fructose bisphosphate aldolase           tion C5 C4 compounds biosynthesis of <br /> with regard biomass flux. Notable activity path-      biomass components. This seen low <br /> way has, however, observed transaldolase-           contribution TCA cycle NADH NADPH <br /> deficient mutants, wild-type strains E. coli     production. When adding flux proton- <br /> [46]. It likely affinities enzymes   translocating transhydrogenase 5 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, the <br /> respective sedoheptulose phosphates lower          role TCA cycle shifts somewhat produc- <br /> to corresponding fructose phosphates, vivo,       tion biomass precursors energy metabolism, as <br /> the bypass reactions expected display            fraction NADH produced TCA cycle in- <br /> significant activity unless accumulation          creases 8.1% 12.9%. This reflected the <br /> sedoheptulose 7-phosphate reason. Therefore,         lowered fractions 2-oxoglutarate (31% vs. 50%) oxa- <br /> would advisable constrain fluxes by-       loacetate (35% vs. 46%) branching TCA cycle to- <br /> pass reactions zero specifically studying wild-      wards anabolic pathways. <br /> type network.                                                       Metabolite flux minimisation revealed potential prob- <br />    We strong discrepancy predicted           lem model predicting flux extracel- <br /> NADPH balance measured [48]. This dis-               lular protons essential optimal growth: Excess <br /> crepancy explained fact vivo,           intracellular protons, transported peri- <br /> costly reaction (proton-translocating transhydrogenase)          plasm complexes electron transfer chain, <br /> accounts substantial fraction NADPH pro-            converted water ferroxidase periplasm <br /> duced. Flux reaction reduces number          instead exported outer membrane. <br /> protons periplasm available ATP synthase,              The electrons required involved reduction of <br /> which means carbon respired               molecular oxygen donated Fe2+, be <br /> CO2, lowering biomass yield. Therefore, reaction        imported extreme rate 14 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, <br /> is used generation NADPH FBA flux           equivalent 0.8 grams iron gram dry <br /> distribution optimised biomass production. To im-            weight hour. This obviously unrealistic, especially as <br /> prove capability model predict fluxes           produced Fe3+ low solubility neutral pH [55]. <br /> &#226;&#8364;&#732;aerobic growth sugars&#226;&#8364;&#8482; scenarios, flux        The reason free import electrons resulting <br /> reaction needs enforced,               free import Fe2+ does pro- <br /> preliminary tests altered simulation parameters.            nounced effects model lim- <br /> Ideally, postulating stoichio-          ited periplasm, i.e. electrons used to <br /> metric coupling reactions instead           perform work cytosol. To limit flux the <br /> of fixing flux absolute value. For instance,         ferroxidase reaction, secretion rate Fe3+ be <br /> constant ratio periplasmic protons forced         restricted zero (or value close zero). <br /> take route, resulting stoichiometric coupling          We demonstrated ability split-ratio analysis and <br /> ATP synthase. Or, reproduce experimental          branch point analysis assign metabolic roles. These <br /> results faithfully, flux reaction          roles agreement long-established know- <br /> could coupled fluxes NADPH-              ledge metabolic pathways, cases it <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                      Page 15 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> becomes obvious metabolic roles strongly             Methods <br /> modified different contexts metabolism            Metabolic model iJO1366 <br /> is highly flexible making use different capabilities        The genome-scale metabolic model iJO1366 covers 1366 <br /> many metabolites provide.                                         open reading frames Escherichia coli K-12 MG1655. <br />   In small number discussed examples, role          The reconstructed network comprises total 2583 reac- <br /> L-glutamine, L-glutamate, L-aspartate amino-               tion nodes (including transport exchange reactions and <br /> group donors obvious. Furthermore, forced              artificial reactions modelling formation biomass) <br /> acetate secretion flux shown responsible            1805 metabolite nodes (1039 cytoplasmic, 442 periplas- <br /> more 90% total acetate-producing flux.                mic, 324 extracellular). The model downloaded from <br /> Thioredoxin predicted redundant steady-              supplementary material corresponding publica- <br /> state metabolism despite involved reactions          tion [10] SBML format [58]. <br /> in network. This prediction agreement             iJO1366 stoichiometric model, mathem- <br /> observation trxA trxC double mutants E. coli          atically represented stoichiometric matrix S a <br /> phenotypically inconspicuous [56]. However, thioredoxins          set lower upper flux bounds lb &#226;&#8240;&#164; v &#226;&#8240;&#164; ub <br /> may essential oxidative stress response [57],          constraining flux variables v [59]. S m &#195;&#8212; n <br /> which inherently non-steady-state process          matrix, m number metabolites (1805), n <br /> cannot studied using FBA.                                      number reactions network (2583), v <br />   It noted transformation reaction          n &#195;&#8212; 1 vector. The entry sij S stoichiometric <br /> fluxes metabolite fluxes split ratios com-         coefficient metabolite reaction j. It positive if <br /> putational methods introduced applicable              metabolite occurs right-hand reaction j, <br /> any flux distribution, computationally pre-               negative occurs left-hand side, zero it <br /> dicted experimentally determined, e.g. 13C tracer          involved reaction. The flux bounds {lb, ub} <br /> analysis. The requirement flux distribution          represent, inter alia, thermodynamic constraints (e.g. ir- <br /> the resolution individual reactions                reversibility reactions) medium properties (e.g. <br /> summed &#226;&#8364;&#339;pathway fluxes&#226;&#8364;?.                                          constraining uptake compound present in <br />                                                                   medium zero). <br />                                                                      Important model parameters growth-associated <br /> Conclusions                                                       non-growth-associated maintenance requirements (ab- <br /> This paper introduces new, metabolite-centric view           breviated GAM NGAM, respectively). The GAM is <br /> flux distributions genome-scale metabolic networks.            biomass reaction models ATP cost of <br /> We shown metabolite-centric view flux dis-          growth-associated processes, importantly DNA, RNA, <br /> tributions opens door detailed analysis        protein polymerisation, gram dry weight, the <br /> is possible traditional reaction-centric frame-        NGAM modelled ATP hydrolysis reaction a <br /> work. Split-ratio analysis extension branch point         fixed flux represents maintenance processes that <br /> analysis facilitate interpretation flux distributions   consume ATP associated growth [60]. <br /> providing metabolic context.                                      These processes include, others, DNA repair and <br />    Within metabolite-centric view, easy           maintaining turgor pressure membrane potential. <br /> simultaneously analyse fluxes metabolic context, <br /> as demonstrated metabolic contexts           Flux balance analysis (FBA) <br /> energy metabolism, TCA cycle, interconnected              The fluxes network steady state com- <br /> transamination cycles, superpathway glycolysis             puted using flux balance analysis [16,61,62]. FBA math- <br /> and pentose phosphate pathway, folate-                ematical method predicting flux distribution a <br /> dependent one-carbon pool. This contextualisation                 stoichiometric network requiring knowledge in- <br /> leads new biological interpretations apparent              vivo kinetic parameters. Its underlying assumptions are <br /> from isolated reaction fluxes, used val-         steady state optimality. The assumption, mathem- <br /> idating refining model&#226;&#8364;&#8482;s predictive capabilities          atically expressed Sv = 0, flux <br /> for specific growth scenarios.                                    equilibrium metabolite concentrations constant <br />    In summary, switching reaction-centric             [63]. The second assumption organism been <br /> metabolite-centric view flux distributions allows            optimised evolutionary history bio- <br /> wealth inferences drawn apparent       logical goal [17], usually biomass production (growth). <br /> the reaction fluxes alone. The computational methods             We used implementation FBA metano, the <br /> have introduced allow analysis reaction fluxes,       open-source software toolkit developed group <br /> whether predicted determined experimentally, context        (Riemer et al., manuscript preparation). The software <br /> rather isolation.                                         written Python programming language is <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                         Page 16 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br /> available online (http://metano.tu-bs.de). We verified         1. Compute partial fluxes &#207;?ij = sij &#194;&#183; vj reactions j <br /> implementation comparing solutions earlier             group positive (producing) negative <br /> model iAF1260 corresponding published FBA solu-                (consuming): <br /> tions [11], using identical simulation parameters dif-                 n         o        n             o <br />                                                                                                       <br /> ferent scenarios (growth glucose minimal medium;                      P &#194;&#188; j &#207;?ij &gt; 0 ; C &#194;&#188; j &#207;?ij &lt; 0 <br /> scenario 1: glucose uptake rate 8 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, <br /> carbon-limited; scenario 2: glucose uptake rate 11 mmol            2. Compute: <br /> gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, oxygen uptake rate 18.2 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1,                                 X                  X <br />                                                                           &#206;&#166;i &#194;&#188;              &#207;?ij &#194;&#188; &#226;&#710;&#8217;           &#207;?ij <br /> carbon- oxygen-limited). Apart alternate (but also <br />                                                                                     j&#226;&#710;&#710;P             j&#226;&#710;&#710;C i <br /> optimal) pathway usage, FBA solutions be <br /> reproduced (data shown).                                          divide partial fluxes flux sum: <br />                                                                                   &#207;?ij <br /> Simulation parameters                                                     &#207;? i &#194;&#188; <br />                                                                                   &#206;&#166;i <br /> For analyses model iJO1366, glucose uptake <br /> rate set 11.0 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1, acetate secretion <br /> was added rate 6.4 mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. These values          &#206;&#166;i total flux metabolite (or flux sum, de- <br /> were reported [49] exponential growth bioreac-       fined [32]), &#207;? ij split ratios metabolite i, <br /> tor, scenario corresponds best infinite envir-   grouped sign (producing consuming) <br /> onment assumed implicitly FBA. Moreover, specific         expressed percentages &#206;&#166;i. More precisely, &#207;? ij is <br /> constraints described [11] aerobic growth glucose      fraction metabolite produced (if positive) con- <br /> were employed. These include blocking reactions         sumed (if negative) reaction j. <br /> not active scenario fixing effective proton <br /> translocation rate NADH dehydrogenase complex <br /> at 1.5 protons molecule NADH. The equiva-       Branch point analysis <br /> lent setting ratio proton-translocating       Branch point analysis developed generalise ap- <br /> non-proton-translocating NADH dehydrogenase 1:1. The          proach split-ratio analysis metabolic net- <br /> metano input files scenario resulting output    work visualise extended metabolic context a <br /> files provided Additional files 2 3, respectively.    selected metabolite reaction. The analysis starts with <br />    A second scenario, derived             complete bipartite reaction graph, consists all <br /> described adding forced flux 5 mmol                reactions metabolites active given FBA <br /> gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1 proton-translocating transhydrogenase,         solution (i.e. carry flux threshold). As the <br /> was employed attempt resolve discrepancies        complete graph complex meaningful in- <br /> initially model predictions experimental data.       terpretation, graph reduced according user- <br />                                                                  supplied parameters. <br />                                                                     The user selects node (i.e. reaction metabolite) as <br /> Flux variability analysis (FVA)                                  hub analysis. Optionally, user provide a <br /> The solutions returned FBA necessarily unique,        list nodes excluded list nodes dis- <br /> as whole-genome models, objective function usu-          connected. Excluded nodes completely removed from <br /> ally assumes optimum point           graph, disconnected nodes split separate <br /> solution space [60]. Flux variability analysis successively      instances occurrence metabolite reaction <br /> minimises maximises flux variable             network. Typically, list used to <br /> additional constraint optimal suboptimal objective      exclude metabolites ubiquitous, con- <br /> function value [64], exploring shape opti-       texts uninteresting user (e.g. H2O H+). The sec- <br /> mal solution space. In present study, FVA used        ond list used prevent crowding graph by <br /> test reported fluxes split ratios necessar-      promiscuous compounds currency metabolites <br /> ily assume values predicted FBA al-       (e.g. ATP NADH). Finally, user specify max- <br /> ternate optimal solutions. FVA implemented           imum distance selected hub node. Nodes beyond <br /> metano &#226;&#8364;&#732;fast FVA&#226;&#8364;&#8482; variety algorithm [53].          limit shown final graph. <br />                                                                     It possible select hub analysis, <br /> Split-ratio analysis                                             allows fine-grained selection nodes inclusion <br /> We developed following algorithm compute me-              exclusion. However, resulting graph discon- <br /> tabolite fluxes split ratios reaction fluxes:           nected, largest connected component displayed. <br />   Given stoichiometric matrix S flux vector v,              The algorithm illustrated schematically Figure 6. <br />   For metabolite i:                                         Branch point analysis starts calculation the <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                         Page 17 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />  Figure 6 Sequence example graphs demonstrating algorithm branch point analysis. Node R1 selected hub, node X <br />  disconnected, Y removed. The configured maximum distance selected node two. A) Initial bipartite graph. <br />  B) Disconnection (X) removal (Y), respectively, selected nodes. C) Bridging transitory nodes (B, R2, C). D) Removal nodes a <br />  distance greater hub, R1. E) Condensed metabolic context reaction R1 produced branch point analysis. <br />  <br />  <br />  <br /> split ratios metabolites based given flux                   Analysis), distributed GNU General Public Li- <br /> distribution. Subsequently, bipartite reaction graph                cense version 3 downloaded http:// <br /> built, edges labelled corresponding                 metano.tu-bs.de/ameba.       Our    implementation   uses <br /> split ratios (as percentage flux adjacent               NetworkX [65] graph operations Graphviz [66] for <br /> metabolite node). Figure 6A shows initial state                  graph layouting. An interactive mode been <br /> example graph representing metabolic context reac-                  implemented using xdot module (http://<span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span>.google. <br /> tion R1, edge labels omitted clarity. Next, re-               com/p/jrfonseca/wiki/XDot). <br /> spective nodes specified user removed and <br /> disconnected (Figure 6B). At point, complexity                 Metabolite flux minimisation <br /> the graph reduced bridging transitory nodes                  The following algorithm used determine mini- <br /> (Figure 6C). Specifically, node exactly                  mum flux metabolite node carry under <br /> neighbours replaced dashed edge. This step ap-                  constraint (sub)optimal biomass production: <br /> plied iteratively nodes remain. Disconnected <br /> nodes taken account number                           1. Split flux variables positive negative <br /> neighbours determined. Finally, nodes                          components: <br /> specified maximum distance removed (Figure 6D). Be- <br /> sides selected node, remaining graph contains                        vj &#194;&#188; v&#195;&#190;   &#226;&#710;&#8217;          &#195;&#190;         &#226;&#710;&#8217; <br />                                                                                         j &#226;&#710;&#8217;vj ; vj &#226;&#8240;&#165;0 vj &#226;&#8240;&#165;0; <br /> nodes metabolic flux branches &#226;&#8364;&#8220; metabolic <br /> branch points (Figure 6E).                                                       duplicate negate corresponding columns <br />   We implemented branch point analysis Python                             stoichiometric matrix S. The result vector <br /> programming language based metano. The resulting                              non-negative flux variables. Lower upper <br /> program, AMEBA (Advanced MEtabolic Branchpoint                                   bounds adjusted accordingly. In practice, <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                                      Page 18 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />      flux variables constrained zero removed                             Acknowledgements <br />      corresponding columns S.                                    RR funded Deutsche Forschungsgemeinschaft SFB <br />                                                                                    Transregio 51 &#226;&#8364;&#732;Roseobacter&#226;&#8364;&#8482;. The authors like thank Dr. Thomas Ulas <br />   2. Using modified flux vector stoichiometric                             testing new methods helpful discussions feedback. We <br />      matrix step 1:                                                           warmly thank Dr. Antje Chang proofreading early version the <br />      For metabolite i, solve following convex                             manuscript. We grateful Dr. Ida Schomburg Melanie Bende <br />                                                                                    stylistic suggestions preparation manuscript, to <br />      linear optimisation problem:                                                  anonymous reviewers valuable feedback suggestions. <br />                 X                          <br />                      &#195;&#190;:             &#195;&#190;      sij ; sij &#226;&#8240;&#165;0 <br />      Minimise       sij vj ; sij &#194;&#188;                                           Received: 14 August 2012 Accepted: 3 April 2013 <br />                  j <br />                                              0;                               Published: 12 April 2013 <br />  <br />  <br />                                                                                    References <br />       steady state: Sv = 0                                                      1. Edwards JS: Systems properties haemophilus influenzae Rd <br />       inequality constraints lb &#226;&#8240;&#164; v &#226;&#8240;&#164; ub                                          metabolic genotype. J Biol Chem 1999, 274:17410&#226;&#8364;&#8220;17416. <br />       additional constraint:                                               2. Kim TY, Sohn SB, Kim YB, Kim WJ, Lee SY: Recent advances in <br />                                                                                        reconstruction applications genome-scale metabolic models. <br />             vBiomass &#226;&#8240;&#165; &#206;&#182; : vBiomass;opt 0 &lt; &#206;&#182;&#226;&#8240;&#164;1:                                  Curr Opin Biotechnol 2012, 23:617&#226;&#8364;&#8220;623. <br />                                                                                    3. Durot M, Bourguignon P-Y, Schachter V: Genome-scale models bacterial <br />                                                                                        metabolism: reconstruction applications. FEMS Microbiol Rev 2009, <br />                                                                                        33:164&#226;&#8364;&#8220;190. <br />   In effect, algorithm successively minimises                         4. Oberhardt MA, Palsson B&#195;&#732;, Papin JA: Applications genome-scale <br /> the producing metabolite flux sums &#206;&#166;i. The optimisation                                metabolic reconstructions. Mol Syst Biol 2009, 5:320. <br /> problems solved using GNU Linear Program-                                 5. Kjeldsen KR, Nielsen J: In silico genome-scale reconstruction and <br />                                                                                        validation Corynebacterium glutamicum metabolic network. <br /> ming Kit (GLPK; www.gnu.org/software/glpk/glpk.html).                                  Biotechnol Bioeng 2009, 102:583&#226;&#8364;&#8220;597. <br /> As metabolite flux minimisation similar FVA,                            6. Racker E: Enzymatic formation breakdown pentose phosphate. <br /> the &#226;&#8364;&#732;fast FVA&#226;&#8364;&#8482; strategy applicable algorithm                             Fed Proc 1948, 7:180. <br />                                                                                    7. Rohmer M, Knani M, Simonin P, Sutter B, Sahm H: Isoprenoid biosynthesis <br /> well, implemented metano                                  bacteria: novel pathway early steps leading isopentenyl <br /> performance optimisation. In study, metabolite flux                               diphosphate. Biochem J 1993, 295(Pt 2):517&#226;&#8364;&#8220;524. <br /> minimisation used identifying nonessential me-                             8. Bausch C, Peekhaus N, Utz C, Blais T, Murray E, Lowary T, Conway T: <br />                                                                                        Sequence analysis GntII (subsidiary) gluconate <br /> tabolites determining minimal metabolite turn-                                 metabolism reveals novel pathway L-idonic acid catabolism in <br /> over rates &#206;&#166;i.                                                                         Escherichia coli. J Bacteriol 1998, 180:3704&#226;&#8364;&#8220;3710. <br />                                                                                    9. Jin RZ, Tang JC-T, Lin ECC: Experimental evolution novel pathway for <br />                                                                                        glycerol dissimilation Escherichia coli. J Mol Evol 1983, 19:429&#226;&#8364;&#8220;436. <br /> Additional files                                                                   10. Orth JD, Conrad TM, Na J, Lerman JA, Nam H, Feist AM, Palsson B&#195;&#732;: A <br />                                                                                        comprehensive genome-scale reconstruction Escherichia coli <br />  Additional file 1: Figure S1. Metabolite fluxes split ratios               metabolism-2011. Mol Syst Biol 2011, 7:535. <br />  superpathway glycolysis pentose phosphate pathway.                 11. Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br />  Green: Metabolite nodes, yellow: enzyme nodes. Numbers metabolite                  Broadbelt LJ, Hatzimanikatis V, Palsson B&#195;&#732;: A genome-scale metabolic <br />  nodes total flux mmol gDW&#226;&#710;&#8217;1 h&#226;&#710;&#8217;1. Edge labels split ratios               reconstruction Escherichia coli K-12 MG1655 accounts 1260 <br />  fractions flux adjacent metabolite node.                           ORFs thermodynamic information. Mol Syst Biol 2007, 3:121. <br />                                                                                    12. Reed JL, Vo TD, Schilling CH, Palsson BO: An expanded genome-scale <br />  Additional file 2: Metano input files generated model <br />                                                                                        model Escherichia coli K-12 (iJR904 GSM/GPR). Genome Biol 2003, 4:R54. <br />  iJO1366. Model scenario file &#226;&#8364;&#8220; http://metano.tu-bs.de/quickref. <br />                                                                                    13. Feist AM, Palsson B&#195;&#732;: The growing scope applications genome-scale <br />  html description file formats. <br />                                                                                        metabolic reconstructions using Escherichia coli. Nat Biotechnol 2008, <br />  Additional file 3: Metano output files (FBA, FVA, split-ratio                         26:659&#226;&#8364;&#8220;667. <br />  analysis). See http://metano.tu-bs.de/quickref.html description file       14. Shinfuku Y, Sorpitiporn N, Sono M, Furusawa C, Hirasawa T, Shimizu H: <br />  formats.                                                                              Development experimental verification genome-scale metabolic <br />                                                                                        model Corynebacterium glutamicum. Microb Cell Fact 2009, 8:43. <br />                                                                                    15. Oberhardt MA, Pucha&#197;&#8218;ka J, Fryer KE, Martins dos Santos VAP, Papin JA: <br /> Abbreviations                                                                          Genome-scale metabolic network analysis opportunistic pathogen <br /> CoA: Coenzyme A; DHA: Dihydroxyacetone; F6P: Fructose 6-phosphate;                     Pseudomonas aeruginosa PAO1. J Bacteriol 2008, 190:2790&#226;&#8364;&#8220;2803. <br /> FBA: Flux balance analysis; FVA: Flux variability analysis; G6P: Glucose 6-        16. Fell DA, Small JR: Fat synthesis adipose tissue. An examination of <br /> phosphate; GAM: Growth-associated maintenance; ICD: Isocitrate                         stoichiometric constraints. Biochem J 1986, 238:781&#226;&#8364;&#8220;786. <br /> dehydrogenase; NGAM: Non-growth-associated maintenance;                            17. Schuetz R, Kuepfer L, Sauer U: Systematic evaluation objective <br /> PEP: Phosphoenolpyruvate; TCA cycle: Tricarboxylic acid cycle;                         functions predicting intracellular fluxes Escherichia coli. Mol Syst <br /> THFA: Tetrahydrofolate.                                                                Biol 2007, 3:119. <br />                                                                                    18. Yizhak K, Benyamini T, Liebermeister W, Ruppin E, Shlomi T: Integrating <br /> Competing interests                                                                    quantitative proteomics metabolomics genome-scale <br /> The authors declare competing interests.                             metabolic network model. Bioinformatics 2010, 26:i255&#226;&#8364;&#8220;i260. <br />                                                                                    19. Shlomi T, Cabili MN, Herrg&#195;&#165;rd MJ, Palsson B&#195;&#732;, Ruppin E: Network-based <br /> Authors&#226;&#8364;&#8482; contributions                                                                 prediction human tissue-specific metabolism. Nat Biotechnol 2008, <br /> SAR RR developed implemented new methods described                    26:1003&#226;&#8364;&#8220;1010. <br /> (SAR: split-ratio analysis metabolite flux minimisation, RR: branch point      20. Chechik G, Oh E, Rando O, Weissman J, Regev A, Koller D: Activity motifs <br /> analysis), carried in-silico study E. coli model, drafted       reveal principles timing transcriptional control yeast <br /> manuscript. SAR DS conceived study. DS participated                  metabolic network. Nat Biotechnol 2008, 26:1251&#226;&#8364;&#8220;1259. <br /> design coordination study helped draft manuscript. All       21. Colijn C, Brandes A, Zucker J, Lun DS, Weiner B, Farhat MR, Cheng T-Y, <br /> authors read approved final manuscript.                                        Moody DB, Murray M, Galagan JE: Interpreting expression data with <br />  Riemer et al. BMC Systems Biology 2013, 7:33                                                                                                             Page 19 19 <br /> http://www.biomedcentral.com/1752-0509/7/33 <br />  <br />  <br />  <br />  <br />       metabolic flux models: predicting mycobacterium tuberculosis mycolic              46. Nakahigashi K, Toya Y, Ishii N, Soga T, Hasegawa M, Watanabe H, Takai Y, <br />       acid production. PLoS Comput Biol 2009, 5:e1000489.                                   Honma M, Mori H, Tomita M: Systematic phenome analysis Escherichia <br /> 22.   Lewis NE, Nagarajan H, Palsson B&#195;&#732;: Constraining metabolic genotype&#226;&#8364;&#8220;               coli multiple-knockout mutants reveals hidden reactions central <br />       phenotype relationship using phylogeny silico methods. Nat Rev                carbon metabolism. Mol Syst Biol 2009, 5:306. <br />       Microbiol 2012, 10:291&#226;&#8364;&#8220;305.                                                       47. Madigan MT, Martinko JM, Stahl DA, Clark DP: Brock Biology of <br /> 23.   Liao Y-C, Huang T-W, Chen F-C, Charusanti P, Hong JSJ, Chang H-Y, Tsai S-F,           Microorganisms. 13th edition. San Francisco, Calif: Pearson; 2011. <br />       Palsson BO, Hsiung CA: An experimentally validated genome-scale                   48. Sauer U, Canonaco F, Heri S, Perrenoud A, Fischer E: The soluble and <br />       metabolic reconstruction klebsiella pneumoniae MGH 78578, iYL1228.                 membrane-bound transhydrogenases UdhA PntAB divergent <br />       J Bacteriol 2011, 193:1710&#226;&#8364;&#8220;1717.                                                      functions NADPH metabolism Escherichia coli. J Biol Chem 2003, <br /> 24.   Williams TCR, Poolman MG, Howden AJM, Schwarzlander M, Fell DA,                       279:6613&#226;&#8364;&#8220;6619. <br />       Ratcliffe RG, Sweetlove LJ: A genome-scale metabolic model accurately             49. Fischer E, Zamboni N, Sauer U: High-throughput metabolic flux analysis <br />       predicts fluxes central carbon metabolism stress conditions.                 based gas chromatography&#226;&#8364;&#8220;mass spectrometry derived 13C <br />       Plant Physiol 2010, 154:311&#226;&#8364;&#8220;323.                                                      constraints. Anal Biochem 2004, 325:308&#226;&#8364;&#8220;316. <br /> 25.   Sauer U: Metabolic networks motion: 13C-based flux analysis. Mol Syst          50. Sig&#195;&#188;enza R, Flores N, Hern&#195;&#161;ndez G, Mart&#195;&#173;nez A, Bolivar F, Valle F: Kinetic <br />       Biol 2006, 2:62.                                                                      characterization batch continuous culture Escherichia coli <br /> 26.   Maier K, Hofmann U, Reuss M, Mauch K: Identification metabolic fluxes              mutants affected phosphoenolpyruvate metabolism: differences in <br />       hepatic cells transient 13C-labeling experiments: Part II. Flux               acetic acid production. World J Microbiol Biotechnol 1999, 15:587&#226;&#8364;&#8220;592. <br />       estimation. Biotechnol Bioeng 2008, 100:355&#226;&#8364;&#8220;370.                                  51. Fischer CR, Klein-Marcuschamer D, Stephanopoulos G: Selection and <br /> 27.   Metallo CM, Walther JL, Stephanopoulos G: Evaluation 13C isotopic tracers          optimization microbial hosts biofuels production. Metab Eng 2008, <br />       metabolic flux analysis mammalian cells. J Biotechnol 2009, 144:167&#226;&#8364;&#8220;174.       10:295&#226;&#8364;&#8220;304. <br /> 28.   Lee JW, Na D, Park JM, Lee J, Choi S, Lee SY: Systems metabolic                   52. Chotani G, Dodge T, Hsu A, Kumar M, LaDuca R, Trimbur D, Weyler W, <br />       engineering microorganisms natural non-natural chemicals.                  Sanford K: The commercial production chemicals using pathway <br />       Nat Chem Biol 2012, 8:536&#226;&#8364;&#8220;546.                                                        engineering. BBA-Protein Struct M 2000, 1543:434&#226;&#8364;&#8220;455. <br /> 29.   Kim HU, Kim TY, Lee SY: Metabolic flux analysis metabolic                     53. Gudmundsson S, Thiele I: Computationally efficient flux variability <br />       engineering microorganisms. Mol Biosyst 2008, 4:113.                               analysis. BMC Bioinforma 2010, 11:489. <br /> 30.   Ranganathan S, Suthers PF, Maranas CD: OptForce: optimization                  54. Sch&#195;&#188;rmann M, Sprenger GA: Fructose-6-phosphate aldolase novel <br />       procedure identifying genetic manipulations leading targeted               class I aldolase Escherichia coli related novel group of <br />       overproductions. PLoS Comput Biol 2010, 6:e1000744.                                   bacterial transaldolases. J Biol Chem 2001, 276:11055&#226;&#8364;&#8220;11061. <br /> 31.   Vongsangnak W, Figueiredo LF, F&#195;&#182;rster J, Weber T, Thykaer J, Stegmann E,          55. Guerinot ML: Microbial Iron Transport. Annu Rev Microbiol 1994, 48:743&#226;&#8364;&#8220;772. <br />       Wohlleben W, Nielsen J: Genome-scale metabolic representation                  56. Meyer Y, Buchanan BB, Vignols F, Reichheld J-P: Thioredoxins and <br />       Amycolatopsis balhimycina. Biotechnol Bioeng 2012, 109:1798&#226;&#8364;&#8220;1807.                     glutaredoxins: unifying elements redox biology. Annu Rev Genet 2009, <br /> 32.   Kim P-J, Lee D-Y, Kim TY, Lee KH, Jeong H, Lee SY, Park S: Metabolite                 43:335&#226;&#8364;&#8220;367. <br />       essentiality elucidates robustness Escherichia coli metabolism.                57. Carmel-Harel O, Storz G: Roles glutathione- thioredoxin- <br />       Proc Natl Acad Sci U S A 2007, 104:13638&#226;&#8364;&#8220;13642.                                       dependent reduction systems Escherichia coli Saccharomyces <br /> 33.   Chung B, Lee D-Y: Flux-sum analysis: metabolite-centric approach                cerevisiae responses oxidative stress. Annu Rev Microbiol 2000, 54:439&#226;&#8364;&#8220;461. <br />       understanding metabolic network. BMC Syst Biol 2009, 3:117.                   58. Hucka M, Finney A, Sauro HM, Bolouri H, Doyle JC, Kitano H, Arkin AP, <br /> 34.   Kim HU, Kim SY, Jeong H, Kim TY, Kim JJ, Choy HE, Yi KY, Rhee JH, Lee SY:             Bornstein BJ, Bray D, Cornish-Bowden A, Cuellar AA, Dronov S, Gilles ED, <br />       Integrative genome-scale metabolic analysis Vibrio vulnificus drug             Ginkel M, Gor V, Goryanin II, Hedley WJ, Hodgman TC, Hofmeyr J-H, Hunter <br />       targeting discovery. Mol Syst Biol 2011, 7:460.                                   PJ, Juty NS, Kasberger JL, Kremling A, Kummer U, Le Nov&#195;&#168;re N, Loew LM, <br /> 35.   Hochachka PW: Action temperature branch points glucose                   Lucio D, Mendes P, Minch E, Mjolsness ED, Nakayama Y, Nelson MR, Nielsen <br />       acetate metabolism. Comp Biochem Physiol 1968, 25:107&#226;&#8364;&#8220;118.                            PF, Sakurada T, Schaff JC, Shapiro BE, Shimizu TS, Spence HD, Stelling J, <br /> 36.   Fell DA, Sauro HM: Metabolic control analysis. Additional                     Takahashi K, Tomita M, Wagner J, Wang J: The systems biology markup <br />       relationships elasticities control coefficients. Eur J Biochem            language (SBML): medium representation exchange of <br />       1985, 148:555&#226;&#8364;&#8220;561.                                                                    biochemical network models. Bioinformatics 2003, 19:524&#226;&#8364;&#8220;531. <br />                                                                                         59. Llaneras F, Pic&#195;&#179; J: Stoichiometric modelling cell metabolism. J Biosci <br /> 37.   Vallino JJ, Stephanopoulos G: Metabolic flux distributions in <br />                                                                                             Bioeng 2008, 105:1&#226;&#8364;&#8220;11. <br />       Corynebacterium glutamicum growth lysine overproduction. <br />                                                                                         60. Warren PB, Queiros SMD, Jones JL: Flux networks metabolic graphs. <br />       Biotechnol Bioeng 1993, 41:633&#226;&#8364;&#8220;646. <br />                                                                                             Phys Biol 2009, 6:046006. <br /> 38.   Voit EO: Design principles operating principles: yin yang of <br />                                                                                         61. Kauffman KJ, Prakash P, Edwards JS: Advances flux balance analysis. <br />       optimal functioning. Math Biosci 2003, 182:81&#226;&#8364;&#8220;92. <br />                                                                                             Curr Opin Biotechnol 2003, 14:491&#226;&#8364;&#8220;496. <br /> 39.   Koffas M, Stephanopoulos G: Strain improvement metabolic <br />                                                                                         62. Orth JD, Thiele I, Palsson B&#195;&#732;: What flux balance analysis? Nat Biotechnol <br />       engineering: lysine production case study systems biology. <br />                                                                                             2010, 28:245&#226;&#8364;&#8220;248. <br />       Curr Opin Biotechnol 2005, 16:361&#226;&#8364;&#8220;366. <br />                                                                                         63. Burton AC: The properties steady state compared of <br /> 40.   Huang D, Jia X, Wen J, Wang G, Yu G, Caiyin Q, Chen Y: Metabolic flux <br />                                                                                             equilibrium shown characteristic biological behavior. J Cell Comp <br />       analysis principal nodes identification daptomycin production <br />                                                                                             Physiol 1939, 14:327&#226;&#8364;&#8220;349. <br />       improvement streptomyces roseosporus. Appl Biochem Biotech 2011, <br />                                                                                         64. Mahadevan R, Schilling CH: The effects alternate optimal solutions in <br />       165:1725&#226;&#8364;&#8220;1739. <br />                                                                                             constraint-based genome-scale metabolic models. Metab Eng 2003, <br /> 41.   Park JM, Kim TY, Lee SY: Prediction metabolic fluxes incorporating <br />                                                                                             5:264&#226;&#8364;&#8220;276. <br />       genomic context flux-converging <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> analyses. Proc Natl Acad Sci <br />                                                                                         65. Hagberg AA, Schult DA, Swart PJ: Exploring network structure, dynamics, <br />       U S A 2010, 107:14931&#226;&#8364;&#8220;14936. <br />                                                                                             function using NetworkX. In Proceedings 7th Python Science <br /> 42.   McAnulty MJ, Yen JY, Freedman BG, Senger RS: Genome-scale modeling <br />                                                                                             Conference (SciPy2008). Pasadena, CA USA; 2008:11&#226;&#8364;&#8220;15. <br />       using flux ratio constraints enable metabolic engineering clostridial       66. Ellson J, Gansner ER, Koutsofios E, North SC, Woodhull G: Graphviz and <br />       metabolism silico. BMC Syst Biol 2012, 6:42.                                       Dynagraph &#226;&#8364;&#8221; Static Dynamic Graph Drawing Tools. In Graph <br /> 43.   Paulsen IT, Reizer J, Jin RZ, Lin EC, Saier MH Jr: Functional genomic studies         Drawing Software. Edited J&#195;&#188;nger M, Mutzel P. Berlin Heidelberg: Springer; <br />       dihydroxyacetone utilization Escherichia coli. Microbiology (Reading,           2004:127&#226;&#8364;&#8220;148. <br />       Engl) 2000, 146(Pt 10):2343&#226;&#8364;&#8220;2344. <br /> 44.   Erni B,  <br /> </body></html>