<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Chem Res Toxicol. 2013 January 18; 26(1): 156&#226;&#8364;&#8220;168. doi:10.1021/tx300437x. <br />  <br />  <br />  <br />                            Synthesis Characterization DNA Minor Groove Binding <br />                            Alkylating Agents <br />                            Prema Iyer&#226;&#8364;&#160;, Ajay Srinivasan&#226;&#8364;&#160;, Sreelekha K. Singh&#226;&#8364;&#160;, Gerard P. Mascara&#226;&#8364;&#160;, Sevara Zayitova&#226;&#8364;&#160;, <br />                            Brian Sidone&#226;&#8364;&#160;, Elise Fouquerel&#194;&#167;, David Svilar&#194;&#167;,&#194;&#165;, Robert W. Sobol&#194;&#167;,&#194;&#165;,#, Michael S. Bobola&#226;&#8240;&#160;, <br />                            John R. Silber&#226;&#8240;&#160;, Barry Gold&#226;&#8364;&#160;,* <br />                            &#226;&#8364;&#160;Department Pharmaceutical Sciences, University Pittsburgh, Pittsburgh, PA 15261 <br />  <br />                            &#194;&#167;University     Pittsburgh Cancer Institute, University Pittsburgh, Pittsburgh PA 15232 <br />                            &#194;&#165;Department        Pharmacology &amp; Chemical Biology, University Pittsburgh, Pittsburgh, PA 15261 <br />                            #Department        Human Genetics, University Pittsburgh 15213 <br />                            &#226;&#8240;&#160;Department        Neurological Surgery, University Washington, Seattle, WA 98105 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  Derivatives methyl 3-(1-methyl-5-(1-methyl-5-(propylcarbamoyl)-1H-pyrrol-3- <br />                                  ylcarbamoyl)-1H-pyrrol-3-ylamino)-3-oxopropane-1-sulfonate (1), peptide-based DNA minor <br />                                  groove binding methylating agent, synthesized characterized. In cases N-terminus <br />                                  appended O-methyl sulfonate ester C-terminus group varied non- <br />                                  polar polar sidechains. In addition, number pyrrole rings varied 2 (dipeptide) <br />                                  3 (tripeptide). The ability different analogues efficiently generate N3-methyladenine <br />                                  demonstrated selectivity minor groove (N3-methyladenine) vs. major groove <br />                                  (N7-methylguanine) methylation. Induced circular dichroism studies used measure the <br />                                  DNA equilibrium binding properties stable sulfone analogues; tripeptide binds with <br />                                  affinity &gt; 10-fold higher dipeptide. The toxicities compounds evaluated <br />                                  alkA/tag glycosylase mutant E. coli human WT glioma cells cells over-expressing <br />                                  under-expressing N-methylpurine-DNA glycosylase, excises N3-methyladenine <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                  DNA. The results equilibrium binding correlates levels N3-methyladenine <br />                                  produced cellular toxicity. The toxicity 1 inversely related expression MPG in <br />                                  bacterial mammalian cell lines. The enhanced toxicity parallels reduced activation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  PARP diminished rate formation aldehyde reactive sites observed MPG <br />                                  knockdown cells. It proposed unrepaired N3-methyladenine toxic ability <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                  directly block DNA polymerization. <br />  <br />                                               DNA damaging agents, affect DNA metabolism, continue extensively used in <br />                                               treatment cancer.1 While interference DNA metabolism causes cell death by <br />                                               apoptosis and/or necrosis, cells survive error free repair error prone bypass <br />                                               lesion. The gives rise increased mutations associated higher cancer risk <br />                                               DNA damaging drugs listed International Agency Cancer <br />                                               Research human carcinogens.2 In fact, significant incidence secondary cancers <br />  <br />  <br />                            * <br />                            Corresponding Author: Barry Gold; Telephone, 1-412-383-9593; goldbi@pitt.edu. <br />                            RWS scientific consultant Trevigen, Inc. The remaining authors state conflict interest. <br />                            Supporting Information <br />                            Synthetic details, NMR, MS spectra, HPLC traces, stability study details. This material available free charge Internet at <br />                            http://pubs.acs.org. <br />                             Iyer et al.                                                                                               Page 2 <br />  <br />  <br />                                            directly attributable treatment patients antineoplastic agents their <br />                                            primary cancer.3&#226;&#8364;&#8220;6 Because DNA remains chemotherapeutic target cancers, is <br />                                            important minimize mutagenic effects. In addition, new agents mechanisms of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            action overlap existing therapies, including react DNA, have <br />                                            potential overcome tumor resistance drugs.7&#226;&#8364;&#8220;12 <br />  <br />                                            We previously developed alkylating agents related netropsin equilibrium bind to, and <br />                                            efficiently react with, atoms line floor minor groove DNA A/T rich <br />                                            sequences, specifically N3-position adenine.13&#226;&#8364;&#8220;17 An O-methyl sulfonate ester was <br />                                            appended N-terminus N-methylpyrrolecarboxamide-based dipeptide combine <br />                                            alkylating activity DNA equilibrium binding (Figure 1, compound 1).13 The C- <br />                                            terminus dipeptide capped n-propyl group, thereby, making molecules <br />                                            neutral, unlike natural product minor groove binders, cationic. The result a <br />                                            DNA equilibrium binder relatively weak affinity DNA.18 Despite weak affinity <br />                                            absence stabilizing electrostatic interactions, molecule efficiently and <br />                                            selectively reacts vitro cells yield N3-methyladenine (3-mA).13&#226;&#8364;&#8220;18 <br />                                            Compound 1 quite cytotoxic compared non-equilibrium binding alkylating agents, <br />                                            e.g., methyl methanesulfonate (MMS), relatively non-mutagenic.19&#226;&#8364;&#8220;22 In vitro data <br />                                            indicated toxicity results 3-mA lesion efficiently blocking DNA replication <br />                                            causing cell death.23&#226;&#8364;&#8220;26 In fact, weak mutagenicity 1 attributable the <br />                                            intermediate formation abasic sites <span id='am-242' about='xsp:base' typeof='owl:Thing'>base</span> excision repair (BER) 3-mA.27 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            We synthesis characterization derivatives dipeptide 1 with <br />                                            different C-termini additional peptide subunit analyze changes <br />                                            affect DNA methylation, affinity binding toxicity. To explore mechanism of <br />                                            toxicity compounds generate 3-mA, performed studies using cells are <br />                                            WT human methylpurine DNA glycosylase (MPG), cells over-express MPG <br />                                            (MPG+) cells MPG knocked (MPG-). The time course for <br />                                            formation abasic sites strand breaks 5-p-dR termini, PARP activation, PARP <br />                                            inactivation, caspase-3 -7 activation monitored. In addition, microfluidic- <br />                                            assisted replication tract analysis used demonstrate 3-mA directly, indirectly, <br />                                            causes replication fork arrest cells. The results provide new insights role of <br />                                            different BER intermediates repair processes cytotoxicity. <br />  <br />                                EXPERIMENTAL PROCEDURES <br />                                Materials reagents <br />                                            E. coli AB1157 (WT) E. coli GC4803 (AlkA/tag) obtained T. O&#226;&#8364;&#8482;Connor (City <br />                                            Hope National Medical Center, Duarte, CA) L. Samson (Massachusetts Institute of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Technology, Cambridge, MA). Human glioblastoma (T98G) cells obtained from <br />                                            American Type Culture Collection (Manassas, VA). CellTiter 96 AQueous One Solution <br />                                            Reagent obtained Promega Corp (Madison, WI). T98G cells expressing <br />                                            human MPG developed essentially described previously plasmid <br />                                            transfection.28,29 Briefly, 1.5 &#195;&#8212; 105 cells seeded 60 mm dishes incubated for <br />                                            24&#226;&#8364;&#8220;30 h 5% CO2 37 &#194;&#176;C. The human MPG expression plasmid (pCMV-MPG-IRES- <br />                                            Neo) transfected using FuGene 6 Transfection Reagent (Roche; Indianapolis, IN) <br />                                            according manufacturer&#226;&#8364;&#8482;s instructions. Stable cell lines selected G418 2 <br />                                            wk expression human MPG protein (nuclear extract) validated immunoblot <br />                                            analysis activity analysis, described30,31 The MPG- cells developed by <br />                                            shRNA-mediated gene knockdown using lentivirus expressing MPG-specific shRNA.32 The <br />                                            shRNA vectors used stable knockdown cell line development obtained glycerol <br />                                            stocks Sigma-Aldrich UPCI Lentiviral core facility (http:// <br />                                            www.upci.upmc.edu/vcf/lenti.cfm). Lentiviral particles generated (in collaboration <br />  <br />                                             Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                             Page 3 <br />  <br />  <br />                                           UPCI Lentiviral facility) co-transfection 4 plasmids [the shuttle vector plus <br />                                           packaging plasmids: pMD2.g(VSVG), pVSV-REV PMDLg/pRRE] 293-FT <br />                                           cells using FuGene 6 Transfection Reagent (Roche, Indianapolis, IN), described <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           previously.28 Lentiviral transduction performed described earlier.28 Briefly, 6.0 &#195;&#8212; <br />                                           104 cells seeded 6-well plate 24 h transduction. Cells transduced for <br />                                           18 h 32 &#194;&#176;C cultured 72 h 37 &#194;&#176;C. Cells selected culturing in <br />                                           growth media 1.0 &#206;&#188;g/mL puromycin, previously described.32 <br />  <br />                                           DNA Damage Quantification kit (abasic site counting) purchased Dojindo <br />                                           Molecular Technologies (Rockville, MD). HT Colorimetric PARP Apoptosis Assay Kit was <br />                                           purchased Trevigen (Gaithersburg, MD). Pierce Cleaved PARP Colorimetric in-cell <br />                                           ELISA kit purchased Thermo Scientific (Rockford, IL). DNAzol reagent, cell <br />                                           culture components, Enzcheck&#194;&#174; Caspase-3 Assay kit Alexa Fluor 488 annexin V/Dead <br />                                           Cell Apoptosis Kit purchased Invitrogen (Carlsbad, CA). Chemicals solvents <br />                                           purchased Sigma Aldrich Chemicals (St. Louis, MO). 10%Pd-C purchased <br />                                           Strem chemicals. Molecular biology grade buffers, plastic- glass-ware <br />                                           obtained Fisher Scientific (Pittsburgh, PA). <br />  <br />                                Synthesis DNA methylating agent 7 <br />                                           (see Scheme 1 Supporting Information synthesis characterization all <br />                                           compounds). All 1H NMR spectra collected Bruker instrument operating 400 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           MHz data reported ppm. <br />  <br />                                           2,2,2-Trichloro-1-(1-methyl-4-nitro-1H-pyrrol-2-yl)ethanone (a)&#226;&#8364;&#8221;was prepared as <br />                                           previously described.33 <br />  <br />                                           1-Methyl-4-nitro-N-propyl-1H-pyrrole-2-carboxamide (b)&#226;&#8364;&#8221;To solution (3.47 <br />                                           g, 12.78 mmol) 96 mL EtOAc added propylamine (31.95 mmol, 2.6 mL) drop <br />                                           wise 0 &#194;&#176;C. The reaction mixture stirred additional 1 h solvent <br />                                           evaporated b 98% yield. 1H NMR (CDCl3): &#206;&#180; 0.98 (t, 3H, J = 7.3), 1.57&#226;&#8364;&#8220;1.66 (m, <br />                                           2H), 3.33&#226;&#8364;&#8220;3.38 (m, 2H), 3.99 (s, 3H), 5.97 (s, br, 1H), 7.04 (d, 1H, J = 1.83), 7.55 (d, 1H, J <br />                                           = 1.83). <br />  <br />                                           4-Amino-1-methyl-N-propyl-1H-pyrrole-2-carboxamide (c)&#226;&#8364;&#8221;To solution b (1g, <br />                                           4.73 mmol) EtOH (300 mL) added 10% Pd/C catalyst (300 mg) reaction <br />                                           shaken Parr apparatus 50 psi disappearance starting material (based TLC). <br />                                           The reaction mixture filtered celite pad solvents evaporated dryness to <br />                                           quantitative yields c. 1H NMR (DMSO-d6) &#206;&#180; 0.84 (t, 3H, J = 7.42), 1.42&#226;&#8364;&#8220;1.48 (m, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           2H), 3.05&#226;&#8364;&#8220;3.09 (m, 2H), 3.66 (s, 3H), 6.16 (d, 1H, J = 1.95), 6.17 (d, 1H, J = 1.95), 7.70 (t, <br />                                           1H, J = 5.47). <br />  <br />                                           4-((t-Butoxycarbonyl)amino)-1-methyl-1H-pyrrole-2-carboxylic acid (d)&#226;&#8364;&#8221;was <br />                                           prepared previously reported.33 <br />  <br />                                           t-Butyl(1-methyl-5-((1-methyl-5-(propylcarbamoyl)-1H-pyrrol-3- <br />                                           yl)carbamoyl)-1H-pyrrol-3-yl)carbamate (e)&#226;&#8364;&#8221;To solution d (538 mg, 2.24 mmol) <br />                                           HOBT (454 mg, 3.36 mmol) anhydrous CH2Cl2 (30 mL) added Et3N (1.25 mL, <br />                                           8.96 mmol) EDC (859 mg, 4.48 mmol). The reaction mixture stirred room <br />                                           temperature 1 h N2. After 1 h, reaction mixture cooled amine c (609 <br />                                           mg, 3.36 mmol) added CH2Cl2 (20 mL) containing Et3N (500 &#206;&#188;L). The reaction <br />                                           mixture stirred N2 18 h diluted CH2Cl2 washed H2O. <br />                                           The aqueous layer extracted CH2Cl2, dried Na2SO4 solvent <br />  <br />  <br />                                             Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                              Page 4 <br />  <br />  <br />                                          evaporated. The residue purified silica gel column chromatography obtain e in <br />                                          86% yield. 1H NMR (DMSO-d6) &#206;&#180; 0.86 (t, 3H, J = 7.42), 1.45 (s, 9H), 1.46&#226;&#8364;&#8220;1.52 (m, 2H), <br />                                          3.09&#226;&#8364;&#8220;3.14 (m, 2H), 3.78 (s, 3H), 3.79 (s, 3H), 6.81 (s, br, 1H), 6.84 (d, 1H, J = 1.56), 6.88 (s, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          br, 1H), 7.15 (d, 1H, J = 1.95), 7.98 (t, 1H, J = 5.46), 9.09 (s, 1H), 9.80 (s, 1H). <br />  <br />                                          4-Amino-1-methyl-N-(1-methyl-5-(propylcarbamoyl)-1H-pyrrol-3-yl)-1H-pyrrole- <br />                                          2-carboxamide (f)&#226;&#8364;&#8221;To solution e (671 mg, 1.66 mmol) anhydrous CH2Cl2 (8 mL) <br />                                          added 50% TFA:CH2Cl2 (v/v, 24 mL) 0 &#194;&#176;C final concentration TFA <br />                                          approximately 40%. The reaction mixture stirred 0 &#194;&#176;C complete <br />                                          disappearance starting material (based TLC). The solvent evaporated dryness to <br />                                          f quantitative yield. 1 H NMR (DMSO-d6) &#206;&#180; 0.86 (t, 3H, J = 7.42), 1.44&#226;&#8364;&#8220;1.539 m, <br />                                          2H), 3.09&#226;&#8364;&#8220;3.15 (m, 2H), 3.80 (s, 3H), 3.89 (s, 3H), 6.84 (d, 1H, J = 1.95), 6.94 (d, 1H, J = <br />                                          1.95), 7.11 (d, 1H, J = 1.95), 7.18 (d, 1H, J = 1.95), 8.01 (t, 1H, J = 5.46), 9.75 (s, 1H), 9.98 <br />                                          (s, 1H). <br />  <br />                                          t-Butyl 1-methyl-4-nitro-1H-pyrrole-2-carboxylate (g)&#226;&#8364;&#8221;was prepared previously <br />                                          described.34 <br />  <br />                                          t-Butyl 4-acrylamido-1-methyl-1H-pyrrole-2-carboxylate (h)&#226;&#8364;&#8221;To solution g <br />                                          (1.4 g, 6.19 mmol) MeOH (350 mL) added 10% Pd/C catalyst (1 g) reaction <br />                                          mixture shaken Parr apparatus 55 psi complete disappearance starting <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          material (by TLC). The reaction mixture filtered celite pad solvents <br />                                          evaporated dryness afford amine quantitative yield (1.06 g, 5.40 mmol), which <br />                                          dissolved anhydrous THF (70 mL) DIEA (2.8 mL, 16.20 mmol). The reaction <br />                                          mixture cooled &#226;&#710;&#8217;78 &#194;&#176;C acryloyl chloride (658 &#206;&#188;L, 8.10 mmol) added <br />                                          dropwise. The reaction mixture maintained &#226;&#710;&#8217;78 &#194;&#176;C disappearance the <br />                                          amine complete (based TLC). The THF concentrated minimum volume and <br />                                          diluted CH2Cl2 washed H2O. The aqueous layer extracted thoroughly <br />                                          CH2Cl2, dried Na2SO4 solvents evaporated dryness. The residue was <br />                                          purified silica gel column chromatography yield h 76% yield. 1H NMR (DMSO-d6) <br />                                          &#206;&#180; 1.49 (s, 9H), 3.79 (s, 3H), 5.67 (dd, 1H, J = 10.16 2.34), 6.16 (dd, 1H J = 17.19, and <br />                                          2.34), 6.27&#226;&#8364;&#8220;6.34 (m, 1H), 6.71 (d, 1H, J = 1.95), 7.34 (d, 1H, J = 1.95), 10.08 (s, br, 1H). <br />  <br />                                          4-Acrylamido-1-methyl-1H-pyrrole-2-carboxylic acid (i)&#226;&#8364;&#8221;To solution h (1.032 <br />                                          g, 4.12 mmol) CH2Cl2 (30 mL) added 1 M TiCl4 solution (10 mL, 10.31 mmol) 0 <br />                                          &#194;&#176;C. The reaction mixture stirred slowly allowed come room temperature until <br />                                          complete disappearance starting material. The reaction mixture diluted CH2Cl2 <br />                                          washed H2O. The aqueous layer extracted EtOAc organic solvent <br />                                          extracts pooled, dried Na2SO4 evaporated dryness yield 96% i. 1H NMR <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (DMSO-d6) &#206;&#180; 3.81 (s, 3H), 5.67 (dd, 1H, J = 9.77 1.96), 6.17 (dd, 1H, J = 17.18 and <br />                                          1.96), 6.29&#226;&#8364;&#8220;6.35 (m, 1H), 6.71 (d, 1H, J = 1.95), 7.38 (d, 1H, J = 1.95), 10.09 (s, br, 1H), <br />                                          12.22 (s, br, 1H). <br />  <br />                                          4-Acrylamido-1 -methyl-N-(1-methyl-5-((1-methyl-5-(propylcarbamoyl)-1H- <br />                                          pyrrol-3-yl)carbamoyl)-1H-pyrrol-3-yl)-1H-pyrrole-2-carboxamide (j)&#226;&#8364;&#8221;To a <br />                                          solution (217 mg, 1.12 mmol) HOBT (225 mg, 1.66 mmol) anhydrous CH2Cl2 <br />                                          (20 mL) added Et3N (0.618 mL, 4.44 mmol) EDC (425 mg, 2.22 mmol). The reaction <br />                                          mixture stirred room temperature 1 h N2, cooled amine f (505 mg, 1.66 <br />                                          mmol) added CH2Cl2 (11 mL) containing Et3N (1 mL). The reaction mixture stirred <br />                                          N2 18 h diluted CH2Cl2 washed H2O. The aqueous layer <br />                                          extracted thoroughly CH2Cl2, dried Na2SO4 solvents evaporated. The <br />                                          residue purified silica gel column chromatography afford j 60% yield. 1H NMR <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                               Page 5 <br />  <br />  <br />                                           (DMSO-d6) &#206;&#180; 0.87 (t, 3H), 1.46&#226;&#8364;&#8220;1.52 (m, 2H), 3.10&#226;&#8364;&#8220;3.16 (m, 2H), 3.79 (s, 3H), 3.84 (s, 3H), <br />                                           3.85 (s, 3H), 5.67 (dd, 1H, J = 10.15 1.95), 6.19 (dd, 1H, J = 17.19 1.95), 6.34&#226;&#8364;&#8220;6.41 <br />                                           (m, 1H), 6.86 (d, 1H, J = 1.56), 6.94 (d, 1H, J = 1.56), 7.04 (d, 1H, J = 1.95), 7.18 (d, 1H, J = <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           1.95), 7.25 (d, 1H, J = 1.95), 7.28 (d, 1H, J = 1.95), 8.0 (t, 1H, J = 5.47), 9.89 (s, 1H), 9.95 <br />                                           (s, 1H), 10.12 (s, 1H). <br />  <br />                                           3-((1-Methyl-5-((1-methyl-5-((1-methyl-5-(propylcarbamoyl)-1H-pyrrol-3- <br />                                           yl)carbamoyl)-1H-pyrrol-3-yl)carbamoyl)-1H-pyrrol-3-yl)amino)-3- <br />                                           oxopropane-1-sulfonic acid (k)&#226;&#8364;&#8221;To solution j (123 mg, 0.257 mmol) EtOH/ <br />                                           H2O (4:1, 5.6 mL), added NaHSO3 solution (0.514 mmol, 55 mg 0.740 mL H2O). <br />                                           The pH solution adjusted 8 reaction mixture refluxed 90 &#194;&#176;C until <br />                                           complete disappearance starting material (based TLC). The reaction cooled 0 <br />                                           &#194;&#176;C conc HCl added dropwise pH 1 2. The solvent was <br />                                           evaporated dryness yield bright yellow solid treated cold 2 N HCl. The <br />                                           mixture centrifuged (4000 rpm, 10 min) supernatant decanted the <br />                                           remaining solid twice washed 2 N cold HCl, centrifuged supernatant <br />                                           decanted. The remaining solid material thoroughly dried 93% k. 1H NMR <br />                                           (DMSO-d6) &#206;&#180; 0.87 (t, 3H), 1.44&#226;&#8364;&#8220;1.53 (m, 2H), 2.53&#226;&#8364;&#8220;2.57 (m, 2H), 2.66&#226;&#8364;&#8220;2.70 (m, 2H), 3.09&#226;&#8364;&#8220; <br />                                           3.15 (m, 2H), 3.79 (s, 3H), 3.83 (s, 3H), 3.84 (s, 3H), 6.85 (d, 1H, J = 1.56), 6.87 (d, 1H, J = <br />                                           1.56), 7.03 (d, 1H, J = 1.95), 7.15 (d, 1H, J = 1.56), 7.18 (d, 1H, J = 1.95), 7.24 (d, 1H, J = <br />                                           1.56), 8.0 (t, 1H, J = 6.48), 9.89 (s, 1H), 9.91 (s, 1H), 9.97 (s,1H). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Methyl 3-(1-methyl-5-(1-methyl-5-(1-methyl-5-(propylcarbamoyl)-1H-pyrrol-3- <br />                                           yl)carbamoyl)-1H-pyrrol-3-yl)carbamoyl)-1H-pyrrol-3-ylamino)-3- <br />                                           oxopropane-1-sulfonate (l)&#226;&#8364;&#8221;To solution k (129 mg, 0.229 mmol) dry dioxane <br />                                           (13 mL) added 3-methyl-1-p-tolyltriaz-1-ene (343 mg, 2.297 mmol). The reaction was <br />                                           heated oil bath 80 &#194;&#176;C ~ 4 h. The dioxane reduced minimum volume and <br />                                           residue loaded Et3N deactivated silica gel column. The product l (49%) was <br />                                           eluted EtOAc. 1H NMR (DMSO-d6) &#206;&#180; 0.87 (t, 3H), 1.46&#226;&#8364;&#8220;1.52 (m, 2H), 2.74 (t, 2H, J = <br />                                           7.42), 3.10&#226;&#8364;&#8220;3.15 (m, 2H), 3.62 (t, 2H, J = 7.42), 3.79 (s, 3H), 3.84 (s, br, 6H), 3.87 (s, 3H), <br />                                           6.86 (d, 1H, J = 1.57), 6.89 (d, 1H, J = 1.95), 7.03 (d, 1H, J = 1.95), 7.16&#226;&#8364;&#8220;7.19 (m, 2H), 7.24 <br />                                           (d, 1H, J = 1.95), 8.0 (t, 1H, J = 5.08), 9.89 (s, 1H), 9.93 (s, 1H), 10.07 (s, 1H). 13C NMR <br />                                           (150 MHz, DMSO-d6) 11.94, 23.08, 29.86, 36.39, 36.60, 36.64, 40.66, 44.50, 57.45, 104.39, <br />                                           104.55, 105.12, 118.14, 118.67, 118.92, 122.12, 122.53, 122.55, 123.24, 123.33, 123.56, <br />                                           158.82, 158.91, 161.70, 166.05. HR-MS (ES+H) m/z: calcd C25H34N7O7S: (576.2240). <br />                                           Found: (576.2261). <br />  <br />                                DNA equilibrium binding studies <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Calf thymus DNA (CT-DNA), poly (deoxyadenylic&#226;&#8364;&#8220;thymidylic) acid sodium salt [poly- <br />                                           d(A-T)], netropsin distamycin purchased Sigma-Aldrich, St. Louis, MO. The <br />                                           stock solutions polymers prepared overnight dialysis 4 &#194;&#176;C 10 <br />                                           mM sodium phosphate buffer containing 1 mM Na2EDTA 10 percent EtOH pH 7. <br />                                           The concentration stock CT-DNA poly-d(A-T) solutions measured UV <br />                                           spectrophotometer using extinction coefficient &#206;&#181;260 = 6550 M&#226;&#710;&#8217;1 cm&#226;&#710;&#8217;1 &#206;&#181;260 = <br />                                           6500 M&#226;&#710;&#8217;1 cm&#226;&#710;&#8217;1 respectively. For experimental purposes, compounds were <br />                                           dissolved respective buffers concentrations determined <br />                                           spectrophotometrically using extinction coefficients, &#206;&#181;296 = 21,500 M&#226;&#710;&#8217;1 cm&#226;&#710;&#8217;1 5, 6 <br />                                           netropsin, &#206;&#181;303 = 34,000 M&#226;&#710;&#8217;1 cm&#226;&#710;&#8217;1 10, 11 distamycin. <br />  <br />                                           Thermal denaturation experiments carried Cary UV 300 Bio UV-visible <br />                                           Spectrophotometer (Varian Inc., Palo Alto, CA) coupled Cary Temperature Controller <br />                                           <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. All experiments performed using 1.4 mL cuvette (1 cm path <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span>). The <br />  <br />  <br />                                             Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                                 Page 6 <br />  <br />  <br />                                             melting CT-DNA poly-d(A-T) monitored 260 275 nm temperature <br />                                             range 0 &#226;&#8364;&#8220; 95 &#194;&#176;C heating rate 0.5 &#194;&#176;C/min data interval 0.2 &#194;&#176;C the <br />                                             absence presence sulfones 5, 6, 10 11. The melting temperature (TM) was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             determined analysis shape first-derivative melting curves. <br />  <br />                                             The induced circular dichroism (ICD) titration compounds 5, 6, 10 11 CT-DNA <br />                                             poly-d(A-T) performed Jasco-815 CD spectrometer (Jasco Corporation, <br />                                             Tokyo, Japan) equipped Jasco PTC-423S/15 Peltier Temperature Controller system. <br />                                             Two-point calibration CD spectrometer checked using 3.855 mM (1S)-(+)-10- <br />                                             camphorsulphonic acid. The spectra solution 1 mm cuvette resulted ratio, <br />                                             &#206;&#181;191/&#206;&#181;291 = &#226;&#710;&#8217;2.00. All experiments constant temperature 20 &#194;&#176;C <br />                                             maintained help Julabo F25-ME refrigerated/heating circulator. The spectrometer <br />                                             purged ~ 15 min N2 starting instrument ~ 30 min the <br />                                             light source attain stability starting experiments. The serial dilution approach to <br />                                             ICD described Garbett et al.35 used titrate sulfone compounds (~ 10 &#226;&#8364;&#8220; 50 &#206;&#188;M) <br />                                             CT-DNA poly-d(A-T) concentration range 1 nM 1 mM. In approach, <br />                                             experiment started solution containing maximum concentration the <br />                                             DNA substrate serially diluting solution cuvette ligand stock <br />                                             solution concentration cuvette achieve required concentration <br />                                             range DNA substrate, titrating DNA substrate constant ligand <br />                                             concentration. A semi-microcuvette non-transmitting black walls 1 cm path <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             length used experiments. The spectra scanned 400 220 nm with <br />                                             data pitch 0.1 nm, scan speed 100 nm/min 1 sec response time 1 nm <br />                                             bandwidth. Each spectrum buffer subtracted final plot taken average of <br />                                             accumulated scans. The binding curve obtained plotting maxima the <br />                                             induced CD signal sulfone compounds function logarithm DNA <br />                                             substrate concentration. The binding constant calculated evaluating slope the <br />                                             graph using non-linear fit option Origin/FitAll software previously described.35 <br />  <br />                                DNA methylation <br />                                             CT DNA (100 &#206;&#188;M, nucleotide), dialyzed overnight 10 mM sodium cacodylate buffer <br />                                             (pH 7.0) reacted room temperature 24 h methylating compounds (dissolved <br />                                             minimal volume ice-cold 95% EtOH) final concentrations 1, 5, 10, 100 &#206;&#188;M in <br />                                             10 mM sodium cacodylate buffer (pH 7.0) total volume 2 mL (for 1, 5 10 &#206;&#188;M <br />                                             alkylating agent) 1 mL (for 100 &#206;&#188;M alkylating agent). The DNA precipitated from <br />                                             reaction solution 3 M NaOAc (pH 4.8) EtOH, redissolved 10 mM <br />                                             sodium cacodylate buffer (pH 7.0) total volume 500 &#206;&#188;L heated 30 min 90 <br />                                             &#194;&#176;C release N3- N7-alkylpurines. The partially apurinic DNA precipitated 0 &#194;&#176;C <br />                                             adding 50 &#206;&#188;L cold 0.1 N HCl. The supernatant containing N-methylpurines was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             analyzed HPLC. For measurement 3-mA, following conditions utilized: <br />                                             column, Partisil 10 SCX; temperature, 25 &#194;&#176;C; eluent, 175 mM ammonium formate (pH 3.0) <br />                                             containing 30% (v/v) MeOH; flow rate, 1 mL/min; detection, UV 270 nm. For <br />                                             simultaneous measurement 3-mA 7-methylguanine (7-mG), following <br />                                             conditions utilized: Syngeri Fusion-RP column (temperature, 25 &#194;&#176;C; eluent, 50 &#206;&#188;M <br />                                             ammonium formate (pH 4.0) containing 2% (v/v) isopropanol; flow rate, 1.25 mL/min; <br />                                             detection, UV 270 280 nm). Authentic standards 3-mA 7-mG used for <br />                                             HPLC identification adducts calculate response factors. <br />  <br />                                E. coli cytotoxicity assays <br />                                             Cultures Escherichia coli strains AB1157 (strr, wild type) GC4803 (alk&#226;&#710;&#8217;/&#226;&#710;&#8217; tag&#226;&#710;&#8217;/&#226;&#710;&#8217; <br />                                             kanr, DNA repair mutant) started frozen stocks, liquid LB media the <br />                                             corresponding antibiotic. Overnight cultures strains used seed 10 mL LB <br />  <br />  <br />                                              Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                             Page 7 <br />  <br />  <br />                                            medium cells allowed grow 2&#226;&#8364;&#8220;3 h reached mid-log growth stage <br />                                            (OD600 = 0.5). Cells harvested 1 mL aliquots centrifugation (5000 &#195;&#8212; g, 5 min), <br />                                            washed phosphate buffered saline (PBS) resuspended 950 &#206;&#188;L PBS. Pre-weighed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            aliquots compounds dissolved 95% EtOH 1 M stock. Dilutions this <br />                                            stock used confirm concentration compound solution UV absorbance. <br />                                            Depending observed concentration, stock solution alkylating agent diluted in <br />                                            appropriate volumes 95% EtOH obtain working stocks 40, 80, 120, 160 200 <br />                                            &#206;&#188;M. 50 &#206;&#188;L stock solution added 950 &#206;&#188;L previously prepared cell <br />                                            suspension (in 1x PBS) mixed vortexing. The cells treated methylating <br />                                            compounds 1 h 37 &#194;&#176;C. A control set included cells treated similarly <br />                                            5% EtOH 1x PBS. Post treatment, cells harvested centrifugation, rinsed <br />                                            1x PBS, serially diluted (100 &#226;&#8364;&#8220; 10&#226;&#710;&#8217;6) 1x PBS plated solid LB media the <br />                                            appropriate antibiotic. After 16 h, colonies counted plate cell viability <br />                                            (<span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>survival</span> %) calculated relative control set. <br />  <br />                                T98G Glioma cell cytotoxicity assay <br />                                            Human glioblastoma cells (T98G) maintained growth medium, i.e., Eagle&#226;&#8364;&#8482;s <br />                                            Minimum Essential Medium (EMEM) 10% fetal bovine serum (FBS), 50 &#206;&#188;g &#206;&#188;L-1 <br />                                            gentamycin, 1x MEM non-essential amino acids 1 mM sodium pyruvate. To evaluate <br />                                            short term cytotoxicity measuring viable cells, T98G cells maintained growth medium <br />                                            harvested Trypsin-EDTA treatment, resuspended fresh growth medium a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            density 104 cells mL&#226;&#710;&#8217;1, 200 &#206;&#188;L (2 &#195;&#8212; 103 cells) suspension seeded the <br />                                            wells sterile 96-well tissue culture microplate. The cells allowed attach and <br />                                            grow 24 h, treated serial dilutions drug 72 h. After treatment, <br />                                            viable cells measured using MTS assay using CellTiter 96 AQueous kit <br />                                            (Promega) manufacturer&#226;&#8364;&#8482;s instructions. To assay long-term cytotoxicity, cells were <br />                                            grown approximately 50&#226;&#8364;&#8220;75% confluence trypsinized counted using a <br />                                            CASY counter manufacturer&#226;&#8364;&#8482;s instructions. Cells seeded 96-well plates at <br />                                            density 120 cells/well incubated 37 &#194;&#176;C 24 h. The cells treated 1 and <br />                                            incubated 9 days 37 &#194;&#176;C. Plates removed fluorescence determined using <br />                                            CyQuant kit (Invitrogen, #C7026) following manufacturer&#226;&#8364;&#8482;s instructions. Results <br />                                            average separate experiments normalized vehicle treated control cells <br />                                            error bars representing standard error mean, previously described.35 <br />  <br />                                Quantification aldehyde reactive sites (ARS) <br />                                            T98G cells maintained growth medium, harvested Trypsin-EDTA treatment, <br />                                            resuspended fresh growth medium density 2.5 &#195;&#8212; 105 cells/mL, 2 mL (5 &#195;&#8212; 105 <br />                                            cells) suspension seeded wells sterile 6-well tissue culture plate. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Cells allowed grow 24 h, treated various compounds different time <br />                                            points and/or different concentrations. At end treatment, adherent cells were <br />                                            harvested trypsin-EDTA treatment pooled cells culture supernatant. <br />                                            Pooled cells suspension pelleted centrifugation, washed 1x PBS and <br />                                            genomic DNA isolated using DNAzol reagent (Invitrogen). DNA solution was <br />                                            quantified UV absorbance 260 nm using Synergy H1 multiwell plate reader (BioTek, <br />                                            Winooski, VT). ARS purified DNA reacted biotin-conjugated <br />                                            aldehyde reactive probe, immobilized 96-well ELISA plates, detected using <br />                                            horseradish peroxidase (HRP)-conjugated streptavidin using colorimetric substrate per <br />                                            manufacturer&#226;&#8364;&#8482;s instructions (Dojindo Molecular Technologies, Rockville, Maryland). <br />  <br />  <br />  <br />  <br />                                              Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                                Page 8 <br />  <br />  <br />                                Markers Apoptosis <br />                                             T98G cells maintained growth medium, harvested Trypsin-EDTA treatment, <br />                                             resuspended fresh growth medium seeded 96- 6-well plates assay <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             requirements: (i) measure PARP activity, 2 &#195;&#8212; 104 cells seeded wells 96 <br />                                             plate, (ii) measure PARP cleavage, 104 cells seeded wells 96-well <br />                                             plate, (iii) measure caspase-3 -7 activities, 106 cells seeded wells of <br />                                             6-well plate. Cells allowed grow 24 h, treated compounds different <br />                                             time periods and/or different concentrations. At end treatment, adherent cells <br />                                             harvested trypsin-EDTA treatment pooled cells supernatant. The <br />                                             PARP activity determined measuring deposition PAR immobilized <br />                                             histones using anti-PAR (primary antibody) horseradish peroxidase (HRP) conjugated <br />                                             secondary antibody (Trevigen Inc., Gaithersburg, MD). PARP cleavage measured using <br />                                             anti-cleaved PARP (primary antibody) HRP-conjugated secondary antibody (Thermo <br />                                             Fisher Scientific, Pittsburgh, PA). Intracellular capase-3 -7 activities measured <br />                                             using Z-DEVD-AMC (benzyloxycarbonyl-Asp-Glu-Val-Asp-7-amino-4-methylcoumarin), a <br />                                             synthetic fluorogenic substrate cysteine proteases, caspase-3 -7 (Invitrogen, <br />                                             Carlsbad, CA). <br />  <br />                                Microfluidic-assisted replication tract analysis <br />                                             Human glial SF767 tumor cells incubated 50 &#206;&#188;M iododeoxyuridine (IdU) <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             active replication forks.36 After removal IdU, cells incubated 50 &#206;&#188;M <br />                                             chlorodeoxyuridine (CldU) presence solvent 1 (5 10 &#206;&#188;M). The DNA the <br />                                             cells isolated loaded microcapiliaries immunostained antibodies <br />                                             specific halogenated pyrimidines. The capillaries examined a <br />                                             fluorescent microscope equipped digital camera locate areas optical density <br />                                             alignment. Each fields selected photographed twice different filters to <br />                                             obtain 2 color images. Approximately 50&#226;&#8364;&#8220;80 images taken 3&#226;&#8364;&#8220;4 micro channels for <br />                                             sample order collect 300&#226;&#8364;&#8220;400 replication tracks. Track lengths measured <br />                                             using AxioVision image analysis software. The criteria collecting image data track <br />                                             DNA density track alignment; tracks containing 3 labeled segments (e.g., green- <br />                                             red-green) likely present merging forks adjacent replicons not <br />                                             included. <br />  <br />                                RESULTS <br />                                Synthesis <br />                                             The syntheses sulfonate ester sulfone analogues di- tri-peptide minor <br />                                             groove binders different C-terminus groups (Figure 1) generally followed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             previously published procedures13 modifications described Experimental <br />                                             Procedures Supporting Information. <br />  <br />                                Methylation DNA <br />                                             The yields 3-mA result incubating different concentrations compounds 1&#226;&#8364;&#8220;4, <br />                                             7 8 CT-DNA 24 h room temperature pH 7.0 sodium cacodylate buffer are <br />                                             shown Figure 2A. Among 2-ring compounds, methoxyethyl (3) and <br />                                             methoxyethoxyethyl (4) ethers yielded lowest 3-mA, difference <br />                                             2-ring compounds evident highest (100 &#206;&#188;M) <br />                                             concentration. It appears compounds polar C-terminal sidechains have <br />                                             weaker methylating activity hydrophobic propyl allyl groups. <br />                                             The tripeptide propyl (7) allyl (8) compounds clear concentration response with <br />  <br />  <br />  <br />                                              Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                               Page 9 <br />  <br />  <br />                                           3-mA adduct observed 1 &#206;&#188;M concentration. Analogue 8 allyl <br />                                           terminus active compound. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The preference dipetide 1 tripeptide 7 minor groove vs. major groove alkylation <br />                                           assessed comparing relative yields 3-mA 7-mG (Figure 2B, Table 1). At <br />                                           1 5 &#206;&#188;M concentrations, ratio averages &gt; 100:1 minor major groove both <br />                                           compounds. At 10 100 &#206;&#188;M concentration, 3-mA 7-mG ratio drops to <br />                                           approximately 60:1 dipetide 1, remains &gt; 100:1 tripeptide 7. Therefore, DNA <br />                                           methylation compounds essentially restricted formation 3-mA. This <br />                                           contrasts 1:10 3-mA 7-mG ratio observed MMS related methylating <br />                                           agents. <br />  <br />                                DNA equilibrium binding <br />                                           Several approaches used probe relationship DNA methylation the <br />                                           different minor groove binders equilibrium binding DNA using sulfone <br />                                           derivatives 5, 6, 10, 11. The sulfonate esters relatively reactive (t1/2 ~ 6 h pH 7 <br />                                           buffer, Supporting Information) converted sulfonate anion transfer <br />                                           methyl group nucleophile. The sulfones isosteric, neutral stable solution. <br />                                           Initial attempts use isothermal titration calorimetry (ITC) thwarted either <br />                                           limited solubility compounds aqueous buffer low heats liberated <br />                                           binding. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                UV melting studies <br />                                           The UV extinction coefficient DNA function temperature related thermal <br />                                           stability duplex DNA. The effect 5, 6, 10 11 thermal stability CT-DNA <br />                                           poly-(dA-dT) determined monitoring absorbance 260 nm calculating the <br />                                           mid-point melting temperature (TM). Ligands bind double stranded DNA generally <br />                                           stabilize duplex increase TM. A summarization effect four <br />                                           compounds thermal stability CT-DNA poly-d(A-T) given Table 2. The 2- <br />                                           ring compounds 5 6 showed minimal increase stability &#206;&#8221;TM 0.4 &#194;&#176;C for <br />                                           CT-DNA 1.2 &#194;&#176;C poly-d(A-T). The 3-ring compounds 10 11 more <br />                                           stabilizing: &#206;&#8221;TM 1.1 &#194;&#176;C 10 11 CT-DNA 4.1 &#194;&#176;C 10 4.7 &#194;&#176;C 11 <br />                                           poly-d(A-T). In comparison, netropsin distamycin raise TM CT-DNA by <br />                                           21.1 &#194;&#176;C 17.0 &#194;&#176;C, poly-d(A-T) 40.6 &#194;&#176;C 36.9 &#194;&#176;C respectively. <br />  <br />                                Induced CD studies <br />                                           The 2- 3-ring peptide compounds achiral optically inactive. However, upon <br />                                           interaction DNA, ligands acquire induced-CD (ICD) signal coupling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           electric transition moments ligand chiral double helix. This induced Cotton <br />                                           effect distinct DNA CD spectrum directly reflects ligand-DNA <br />                                           interaction.35 The CD titration netropsin, distamycin tripeptides 10 11 CT- <br />                                           DNA poly-d(A-T) represented Figure 3 corresponding values the <br />                                           binding constants reported Table 2. In absence minor groove binding <br />                                           compounds, CT-DNA shows positive band 278 nm negative band 245.2 nm a <br />                                           concentration 148 &#206;&#188;M (Supporting Information). Similarly, poly-d(A-T) shows positive <br />                                           band 166 nm negative band 246.9 nm concentration 143 &#206;&#188;M (Supporting <br />                                           Information). Titration 10 &#206;&#188;M netropsin CT-DNA poly-d(A-T) shows ICD <br />                                           spectrum 320 nm gives binding constant (Kb) 1.95 &#195;&#8212; 105 M&#226;&#710;&#8217;1 2.44 &#195;&#8212; 105 <br />                                           M&#226;&#710;&#8217;1, respectively (Table 1, 5 6Figure 3A,C). Interaction 10 &#206;&#188;M dipeptides CT- <br />                                           DNA poly-d(A-T) concentration range 1 nM 1 mM did rise any <br />                                           ICD signal (data shown). Therefore, binding 2-ring compounds DNA <br />  <br />  <br />                                             Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                               Page 10 <br />  <br />  <br />                                             substrates weak experimentally determined methods attempted. <br />                                             Upon interaction CT-DNA, tripeptides 10 11 10 &#206;&#188;M showed positive signal at <br />                                             325 nm (Figure 3E-H). The signal saturated 1 mM CT-DNA giving Kb 2.88 &#195;&#8212; 104 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             M&#226;&#710;&#8217;1 3.16 &#195;&#8212; 104 M&#226;&#710;&#8217;1, respectively 10 11 (Figure 3E,G). For interaction the <br />                                             compounds 10 11 poly-d(A-T), value binding constant be <br />                                             1.95 &#195;&#8212; 104 1.81 &#195;&#8212; 104 325 nm, respectively (Figure 3F,H). <br />  <br />                                             To understand higher binding affinity provided extra pyrrolecarboxamide <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> in <br />                                             tripeptides effect positively charged, ICD maxima the <br />                                             interactions distamycin (10 &#206;&#188;M) CT-DNA poly-d(A-T) (Figure 3B,D) were <br />                                             measured compared netropsin (Figure 3A,C) tripeptides 10 11. The Kb <br />                                             distamycin calculated 2.10 &#195;&#8212; 105 M&#226;&#710;&#8217;1 5.13 &#195;&#8212; 105 M&#226;&#710;&#8217;1 CT-DNA and <br />                                             poly-d(A-T), respectively. Therefore, introduction single electrostatic interaction the <br />                                             distamycin tripeptide C-terminal guanidinium group results approximately a <br />                                             10-fold increase binding affinity vs. neutral molecules 7 8. The extra cationic <br />                                             amidinium group N-terminus netropsin dipeptide vs. distamycin, compensates <br />                                             pyrrolecarboxamide subunit; compounds similar affinities for <br />                                             DNA. <br />  <br />                                Toxicity E. coli <br />                                             The toxicities di- tri-peptide methylating agents evaluated WT E. coli and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Alka/Tag double mutant (Figure 4). The defective known <br />                                             bacterial DNA-glycosylases excise 3-mA. None compounds toxicity up <br />                                             10 &#206;&#188;M concentration WT bacteria. Presumably, efficient BER protects WT cells <br />                                             low levels alkylation damage. In glycosylase double mutant cell, toxicity <br />                                             profiles 2-ring analogs quite similar, correlates vitro formation <br />                                             3-mA different compounds (Figure 2A). For tripeptide analogs, toxicities <br />                                             followed order allyl (8) &gt; propyl (7) &gt; methoxyethyl (9). The tripeptides between <br />                                             2.5&#226;&#8364;&#8220;3.0 orders magnitude toxic corresponding dipeptide compounds. This <br />                                             correlates vitro adduct data 7 8 (Figure 2A). <br />  <br />                                Toxicity T98G human glioma cells <br />                                             The toxicity 1 7 compared WT T98G glioma cells using short term (72 h) <br />                                             MTS assay (Figure 5). Tripeptide 7 (LD50 19 &#206;&#188;M) &gt; 10-fold toxic dipeptide <br />                                             1 (LD50 246 &#206;&#188;M). Comparative toxicity experiments performed 1 WT T98G <br />                                             cells, cells engineered over-express (MPG+) under-express (MPG-) human <br />                                             MPG glycosylase mediates excision 3-mA BER (Figure 6). In over- <br />                                             expressing cells, 10-fold increase MPG mRNA significant increase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             protein expression.28 In MPG knockdown T98G cells, expression MPG &lt;5% <br />                                             WT cells.32 There significant difference toxicity 1 WT <br />                                             MPG+ cells (LD50 ~240 &#206;&#188;M). However, MPG- cells 60% sensitive <br />                                             relative WT MPG+ cells (LD50 140). The differences toxicity parallel activation <br />                                             PARP formation ARS (see below). <br />  <br />                                Formation aldehyde reactive sites (ARS) <br />                                             The apparent combined rates formation elimination abasic sites 5&#226;&#8364;&#178;-p-dR strand <br />                                             breaks monitored WT, MPG+ MPG- T98G cells treated equitoxic <br />                                             concentrations 1 (80 &#206;&#188;M) 7 (7 &#206;&#188;M). These concentrations based their <br />                                             toxicity short-term assay WT cells (Figure 5). An aldehyde reactive hydrazine <br />                                             probe37&#226;&#8364;&#8220;39 biotin-conjugated horseradish peroxidase (HRP)-conjugated <br />                                             streptavidin (Figure 7) used label ring-opened aldehyde form abasic sites, as <br />  <br />  <br />                                               Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                              Page 11 <br />  <br />  <br />                                            aldehyde groups present. The level aldehyde reactive sites <br />                                            (ARS) correlate formation ring-opened form 1&#226;&#8364;&#178;-hydroxy-2&#226;&#8364;&#178;- <br />                                            deoxyribose group abasic sites 5&#226;&#8364;&#178;-p-dR termini.40 Within limits detection, the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            background levels just 0.5 % DMSO cell lines. The initial rate <br />                                            formation (up 6 h) ARS 1 7 WT cells similar MPG+ cells <br />                                            indicating glycosylase excision 3-mA limiting WT cells. Moreover, total <br />                                            ARS sustained time period monitored virtually dipeptide 1 and <br />                                            tripeptide 7 equitoxic concentrations. As anticipated, MPG- cells respond differently: <br />                                            rate formation ARS slower sustained level ARS 12 h time <br />                                            course reaches 50% observed WT MPG+ cells. <br />  <br />                                            As mentioned above, compounds 1 7 compared WT, MPG+ and <br />                                            MPG- T98G cells using equitoxic concentrations WT cells. Because number of <br />                                            WT cells cytotoxicity assay 1 &#195;&#8212; 104 cells/mL, ARS assay at <br />                                            concentration 25 &#195;&#8212; 104 cells/mL, toxicity significantly lower due <br />                                            dilution methylating agent 25-fold cells. In addition, markers apoptosis, <br />                                            e.g., caspase-3 -7, appear elevated 12 h 250 &#206;&#188;M <br />                                            concentration, LD50 toxicity assay (Figure 5). Therefore, unlikely <br />                                            significant percentage ARS measured dead cells. <br />  <br />                                PARP activation cleavage <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            PARP rapidly binds single-strand breaks, 5&#226;&#8364;&#178;-p-dR termini produced during <br />                                            BER, synthesizes poly(ADP-ribose) chains.41&#226;&#8364;&#8220;43 The time course PARP activation in <br />                                            repair 3-mA reflects stage BER APE-1 produces strand break 5&#226;&#8364;&#178; the <br />                                            abasic site created MPG, lyase activity. Therefore, time course for <br />                                            PARP activation somewhat delayed relative ARS formation. Little activation of <br />                                            PARP observed treating WT T98G cells 80 &#206;&#188;M 1 (Figure 8A). This <br />                                            concentration corresponds LD10 short-term toxicity assay (Figure 5A). At 250 <br />                                            &#206;&#188;M, corresponds LD50 concentration, PARP activation reached maximum <br />                                            level 12 h remained elevated 18 h (Figure 8A). <br />  <br />                                            PARP activation 1 compared cell lines. In WT MPG+ cells treated <br />                                            1 dose-dependent increase PAR deposition vs. untreated cells. However, <br />                                            little change observed MPG- knocked cells treated 1 (Figure 9). These <br />                                            data consistent anticipated reduced rate 3-mA lesions converted into <br />                                            AP sites MPG (Figure 7) subsequent APE1 mediated formation 5&#226;&#8364;&#178;-dRP <br />                                            termini, substrates PARP. <br />  <br />                                            Caspase-3 -7 activation, associated apoptosis, results inactivation of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            PARP.44&#226;&#8364;&#8220;46 Therefore, caspase activity monitored WT cells treatment with <br />                                            80 250 &#206;&#188;M 1 (Figure 8B). Caspase activation, starts climb 12 h and <br />                                            continued rise 18 h, observed 250 &#206;&#188;M concentration 1 <br />                                            indicating 80 &#206;&#188;M concentration toxic assay conditions. As <br />                                            predicted, cleavage PARP significant concentration 1 induces <br />                                            caspase activation (Figure 8C). Little effect seen lower concentration 1. <br />  <br />                                Replication tract analysis <br />                                            There vitro evidence 3-mA, subsequent BER intermediates, block <br />                                            DNA replication.23&#226;&#8364;&#8220;26 To verify case cells, microfluidic-assisted <br />                                            replication tract analysis36 performed SF767 glioma cells treated 5 10 <br />                                            &#206;&#188;M 1. This assay uses antibodies measure length replication tracts before <br />                                            exposure cells alkylating agent 1 monitoring 5-iodo-dU incorporation after <br />  <br />  <br />                                              Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                              Page 12 <br />  <br />  <br />                                           exposure 1 monitoring 5-chloro-dU incorporation. If agent blocks replication, the <br />                                           length DNA stained 5-chloro-dU antibody shortened. The 40 min <br />                                           exposure glioma cells 5 &#206;&#188;M 1 reduces clonogenic survival 35%. The results <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           appear Figure 10 track length 50 percentile reduced 7 &#206;&#188;m <br />                                           untreated control cells 5.7 4.0 5 10 &#206;&#188;M concentrations 1 (statistical <br />                                           difference p = 0.001). The data provide evidence formation 3-mA directly, <br />                                           indirectly BER intermediates, impedes replication fork progression. <br />  <br />                                DISCUSSION <br />                                Relationship Between DNA Methylation DNA Binding <br />                                           Methylating agent 1 developed strategy selectively efficiently generate 3- <br />                                           mA lesions vitro vivo.13&#226;&#8364;&#8220;18 The approach works quite have <br />                                           sought improve design. Of note weak equilibrium binding the <br />                                           initially synthesized 2-ring compound. Despite weak binding,18 alkylation <span id='am-243' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br />                                           1 different non-equilibrium binding methylating agents, e.g., methyl <br />                                           methanesulfonate (MMS), predominantly yield major groove adduct N7- <br />                                           methylguanine (7-mG).47 The vitro ratio 3-mA (minor groove) 7-mG (major <br />                                           groove) approximately 3-orders magnitude higher 1 7 vs. MMS. The methylation <br />                                           DNA 1 clearly driven dipeptide binding minor groove A/T rich <br />                                           regions delivering methyl group sulfonate ester.13,16&#226;&#8364;&#8220;19 In current <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           study, efforts enhance methylating activity changing C-terminal group improve <br />                                           water solubility and/or DNA binding, successful. The major improvement came <br />                                           adding N-methylpyrrolecarboxamide subunit. In summary, dipeptides have <br />                                           binding affinities 1 mM. The extra pyrrole subunit tripeptides, provides <br />                                           additional H-bond VDW contact, displaces additional minor groove waters, <br />                                           increases binding 10-fold. It despite higher binding <br />                                           affinity tripeptides DNA higher levels 3-mA produce (Figure <br />                                           2A), ratio minor major groove methylation does radically differ (Figure 2B). <br />                                           The lack strong correlation specificity minor groove methylation and <br />                                           minor groove binding affinity suggest dipeptides preferentially populate the <br />                                           minor groove A/T sequences, make organized enthalpic interactions (H- <br />                                           bonds, VDW contacts) required detection ICD, isothermal titration calorimetry (ITC) <br />                                           TM experiments. <br />  <br />                                Mechanism Action <br />                                           While toxicity alkylating compounds used present study highly associated <br />                                           yield 3-mA, remains unclear lesion directly responsible the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           biological effects seen cells. To explore mechanism(s) responsible the <br />                                           toxicity, compounds evaluated WT alkA/tag glycosylase null E. coli cells. <br />                                           As expected, WT cells insensitive relative alkA/tag mutant efficient <br />                                           BER former.18 This implies failure initiate BER leads persistent toxic 3- <br />                                           mA adducts. We previously reported alkA/tag mutant sensitive to <br />                                           dipeptide 1 cells defective AP endonuclease activity (apn null cells) suggesting that <br />                                           3-mA deleterious abasic site downstream BER intermediates.17,18 <br />                                           The toxicity 1 explored WT S. cerevisiae Mag1 glycosylase mutant <br />                                           (mag1), AP endonuclease double mutant (apn1apn2) BER triple mutant <br />                                           (mag1apn1apn2).20 The apn1apn2 mutant somewhat sensitive toxicity 1 <br />                                           mag1 cells, triple mutant sensitive. A mag1rad14 double <br />                                           mutant yeast turned surprisingly sensitive 1 mag1apn1apn2 triple <br />                                           mutant, rad14 single mutant showed <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. This suggests important role <br />  <br />  <br />  <br />                                            Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                              Page 13 <br />  <br />  <br />                                          nucleotide excision repair point processing 3-mA, initial <br />                                          glycosylase step BER blocked (Figure 11). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          The story human cells similar E. coli yeast. The T98G MPG- glioma <br />                                          cells, human MPG protein excises 3-mA knocked 95%,32 are <br />                                          60% sensitive toxicity 1 vs. WT MPG+ cells. This inverse <br />                                          correlation expression enzyme removes 3-mA toxicity 1 is <br />                                          consistent ARS data. The MPG- cells significantly lower rates formation <br />                                          sustained levels ARS WT cells. Since yield 3-mA adducts produced <br />                                          1 7 dependent MPG levels, assumed more <br />                                          unrepaired 3-mA lesions MPG- cells. This account lower ARS levels <br />                                          observed cells. <br />  <br />                                          The fate unrepaired 3-mA adducts MPG- cells unknown; however, the <br />                                          enhanced toxicity 1 MPG knockdown cells indicates 3-mA being <br />                                          converted process lethal intermediate 3-mA directly blocks polymerases at <br />                                          replication forks. 3-mA spontaneously depurinate, especially single-stranded DNA, to <br />                                          yield AP site. If case, APE1 cleavage AP site cause PARP <br />                                          activation similar WT. This observed. We know 1 causes replication blocks in <br />                                          glioma cells exposed low concentrations 1 microfluidic-assisted replication <br />                                          tract analysis cells treated 1 40 min (Figure 10). Whether direct effect or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          indirect effect 3-mA remains determined, previously demonstrated <br />                                          3-mA stable analogue, 3-methyl-3-deazaadenine, directly block DNA <br />                                          polymerization vitro.24&#226;&#8364;&#8220;26 <br />  <br />                                          A significant sensitizing effect DNA methylating agents methyl methanesulfonate <br />                                          (MMS) (4-fold increase) temozolomide (10-fold increase) reported human <br />                                          mammary gland cells over-express MPG.48 Over-expression catalytically defective <br />                                          MPG mutant did sensitize cells MMS temozolomide indicating the <br />                                          involvement glycosylase activity. In contrast, toxicity 1 MPG over- <br />                                          expressing cells enhanced vs. WT cells.48 As expected, over-expression MPG <br />                                          caused significant increase rate repair 7-mG (3-mA levels followed) <br />                                          MMS treatment. The authors suggested increase toxicity MPG <br />                                          mediated conversion innocuous 7-mG lesions toxic BER intermediates, e.g., abasic <br />                                          sites. Other cell lines over-expressing MPG sensitive MMS.49 A similar <br />                                          modulation MMS toxicity mutagenicity reported CHO cells over-express <br />                                          rat MPG protein.50 To understand cellular level MPG does affect the <br />                                          toxicity 1 MMS way, important note equitoxic <br />                                          concentrations MMS 1 (based toxicity WT cells), yield 7-mG <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          adducts substantially higher MMS 1. This difference <br />                                          ratio 7-mG (non-toxic) 3-mA (toxic): approximately 10:1 MMS47 1:100 <br />                                          1. The conversion non-toxic 7-mG toxic intermediates BER nicely explains the <br />                                          differential impact MPG over-expression MMS vs. 1.48 <br />  <br />                                          The level ARS function APE-1 endonuclease activity converts abasic sites <br />                                          5&#226;&#8364;&#178;-p-dR lesions, retain reactive aldehyde functionality (Figure 11). Because <br />                                          number APE-1 molecules cells estimated 4&#226;&#8364;&#8220;70 &#195;&#8212; 105 cell,51 <br />                                          Pol &#206;&#178;, APE-1 MPG, normally considered rate limiting step BER.52 <br />                                          However, antisense supression MPG APE-1 enhances toxicity 1 suggesting that <br />                                          unrepaired 3-mA cytotoxic.53 We shown using small molecule inhibitor of <br />                                          APE-1 level persistence ARS WT T98G cells continues increase <br />                                          12 h treatment 1,54 level falls 6 h cells treated 1 <br />                                          (Figure 7). The inhibition APE-1 endonuclease activity synergistically enhanced the <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                                            Page 14 <br />  <br />  <br />                                            toxicity 1 T98G cells consistent toxicity unrepaired abasic sites. Clearly, <br />                                            unrepaired 3-mA, abasic sites BER intermediates cytotoxic. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            In summary, number minor groove alkylating compounds prepared evaluated <br />                                            based ability bind to, methylate DNA. As expected, direct <br />                                            relationship DNA binding, formation 3-mA toxicity. Moreover, rate and <br />                                            extent excision 3-mA human MPG formation ARS inversely correlate <br />                                            toxicity. These data indicate 3-mA directly cause toxicity, likely to <br />                                            ability block DNA replication, need converted a <br />                                            downstream BER intermediate. <br />  <br />                                Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                            We gratefully acknowledge Julia Sidorova (Department Pathology, University Washington) support <br />                                            guidance microfluidic-assisted replication tract analysis assay. <br />  <br />                                            FUNDING <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            This work supported National Institutes Health grants CA29088 (BG), CA148629 (RWS) and <br />                                            GM087798 (RWS). Support UPCI Lentiviral Facility provided Cancer Center Support Grant <br />                                            CA047904 National Institutes Health. <br />  <br />  <br />                                ABBREVIATIONS <br />  <br />                                          AlkA             E. coli 3-methyladenine-DNA glycosylase II <br />                                          ARS              aldehyde reactive site <br />                                          BER              base excision repair <br />                                          CD               circular dichroism <br />                                          ICD              induced circular dichroism <br />                                          ITC              isothermal titration calorimetry <br />                                          3-mA             N3-methyladenine <br />                                          7-mG             N7-methylguanine <br />                                          MPG              (aka, AAG) methylpurine-DNA glycosylase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          MPG+             MPG overexpressing T98G cells <br />                                          MPG-             MPG knockdown T98G cells <br />                                          PARP             poly(ADP-ribose)polymerase <br />                                          TAG              E. coli 3-methyladenine DNA glycosylase I <br />                                          WT               wild type <br />  <br />                                References <br />                                            1. Pratt, WB.; Ruddon, RW.; Ensminger, WD.; Maybaum, J., editors. The Anticancer Drugs. 2. Oxford <br />                                               Press; New York: 1994. <br />                                            2. http://monographs.iarc.fr/ENG/Classification/ <br />  <br />  <br />  <br />  <br />                                                Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                                 Page 15 <br />  <br />  <br />                                          3. Shrivastav N, Li D, Essigmann JM. Chemical biology mutagenesis DNA repair: cellular <br />                                             responses DNA alkylation. Carcinogenesis. 2010; 31:59&#226;&#8364;&#8220;70. [PubMed: 19875697] <br />                                          4. Marchesi F, Turriziani M, Tortorelli G, Avvisati G, Torino F, De Vecchis L. Triazene compounds: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             mechanism action related DNA repair systems. Pharmacol Res. 2007; 56:275&#226;&#8364;&#8220;287. <br />                                             [PubMed: 17897837] <br />                                          5. Hijiya N, Hudson MM, Lensing S, Zacher M, Onciu M, Behm FG, Razzouk BI, Ribeiro RC, <br />                                             Rubnitz JE, Sandlund JT, Rivera GK, Evans WE, Relling MV, Pui CH. Cumulative incidence of <br />                                             secondary neoplasms event childhood acute lymphoblastic leukemia. J Am Med <br />                                             Assoc. 2007; 297:1207&#226;&#8364;&#8220;1215. <br />                                          6. Hijiya N, Ness KK, Ribeiro RC, Hudson MM. Acute leukemia secondary malignancy in <br />                                             children adolescents: current findings issues. Cancer. 2009; 115:23&#226;&#8364;&#8220;35. [PubMed: <br />                                             19072983] <br />                                          7. T&#195;&#188;rk D, Szak&#195;&#161;cs G. Relevance multidrug resistance age targeted therapy. Curr Opin <br />                                             Drug Discov Devel. 2009; 12:246&#226;&#8364;&#8220;252. <br />                                          8. Baumert C, Hilgeroth A. Recent advances development P-gp inhibitors. Anticancer Agents <br />                                             Med Chem. 2009; 9:415&#226;&#8364;&#8220;436. [PubMed: 19442042] <br />                                          9. Gangemi R, Paleari L, Orengo AM, Cesario A, Chessa L, Ferrini S, Russo P. Cancer stem cells: a <br />                                             new paradigm understanding tumor growth progression drug resistance. Curr Med <br />                                             Chem. 2009; 16:1688&#226;&#8364;&#8220;1703. [PubMed: 19442140] <br />                                          10. Baguley BC. Multiple drug resistance mechanisms cancer. Mol Biotechnol. 2010; 46:308&#226;&#8364;&#8220;316. <br />                                               [PubMed: 20717753] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          11. Liu J, Powell KL, Thames HD, MacLeod MC. Detoxication sulfur half-mustards by <br />                                               nucleophilic scavengers: robust activity thiopurines. Chem Res Toxicol. 2010; 23:488&#226;&#8364;&#8220;496. <br />                                               [PubMed: 20050632] <br />                                          12. Abel EL, Bubel JD, Simper MS, Powell L, McClellan SA, Andreeff M, MacLeod MC, DiGiovanni <br />                                               J. Protection 2-chloroethyl ethyl sulfide (CEES)-induced cytotoxicity human <br />                                               keratinocytes inducer glutathione detoxification pathway. Toxicol Appl Pharmacol. <br />                                               2010; 255:176&#226;&#8364;&#8220;183. [PubMed: 21723306] <br />                                          13. Zhang Y, Chen FX, Mehta P, Gold B. The design groove sequence selective alkylation of <br />                                               DNA sulfonate esters tethered lexitropsins. Biochemistry. 1993; 32:7954&#226;&#8364;&#8220;7965. [PubMed: <br />                                               8394120] <br />                                          14. Encell L, Shuker DEG, Foiles PG, Gold B. The vitro methylation DNA minor groove <br />                                               binding methyl sulfonate ester. Chem Res Toxicol. 1996; 9:563&#226;&#8364;&#8220;567. [PubMed: 8728498] <br />                                          15. Kelly J, Shah D, Chen FX, Wurdeman R, Gold B. Quantitative qualitative analysis DNA <br />                                               methylation N3-adenine N-methyl-N-nitrosourea dimethyl sulfate. Chem Res Toxicol. <br />                                               1998; 11:1481&#226;&#8364;&#8220;1486. [PubMed: 9860491] <br />                                          16. Varadarajan S, Shah D, Dande P, Settles S, Chen FX, Fronza G, Gold B. DNA damage and <br />                                               cytotoxicity induced minor groove binding methyl sulfonate esters. Biochemistry. 2003; <br />                                               42:14318&#226;&#8364;&#8220;14327. [PubMed: 14640700] <br />                                          17. Shah D, Gold B. Evidence Escherichia coli N3-methyladenine lesions cytotoxicity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               induced minor groove binding methyl sulfonate ester modulated vivo netropsin. <br />                                               Biochemistry. 2003; 42:12610&#226;&#8364;&#8220;12616. [PubMed: 14580207] <br />                                          18. Shah S, Kelly J, Zhang Y, Dande P, Martinez J, Ortiz G, Fronza G, Tran H, Soto MA, Marky L, <br />                                               Gold B. Evidence Escherichia coli N3-methyladenine lesions induced minor groove <br />                                               binding methyl sulfonate ester processed base nucleotide excision repair. <br />                                               Biochemistry. 2001; 40:1796&#226;&#8364;&#8220;1803. [PubMed: 11327842] <br />                                          19. Kelly J, Inga A, Chen FX, Dande P, Shah D, Monti P, Aprile A, Burns PA, Scott G, Abbondandolo <br />                                               A, Gold B, Fronza G. Relationship DNA methylation mutational patterns induced by <br />                                               sequence selective minor groove methylating agents. J Biol Chem. 1999; 274:18327&#226;&#8364;&#8220;18334. <br />                                               [PubMed: 10373436] <br />                                          20. Monti PA, Campomenosi P, Iannone R, Ciribilli Y, Iggo R, Scott G, Burns PA, Shah D, Menichini <br />                                               P, Abbondandolo A, Gold B, Fronza G. Influences base excision repair defects lethality <br />                                               mutagenesis induced Me-lex, sequence selective N3-adenine methylating agent. J Biol <br />                                               Chem. 2002; 277:28663&#226;&#8364;&#8220;28668. [PubMed: 12042310] <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                                    Page 16 <br />  <br />  <br />                                          21. Monti P, Iannon R, Campomenosi P, Ciribilli Y, Robbiano A, Varadarajan S, Shah D, Menichini <br />                                               P, Gold B, Fronza G. Nucleotide excision repair defect influences lethality mutagenicity <br />                                               induced Me-lex, sequence-selective N3-adenine methylating agent absence base <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               excision repair. Biochemistry. 2004; 43:5592&#226;&#8364;&#8220;2299. [PubMed: 15134433] <br />                                          22. Monti P, Ciribilli Y, Russo D, Bisio A, Perfumo C, Andreotti V, Menichini P, Inga A, Huang X, <br />                                               Gold B, Fronza G. REV1 Pol&#206;&#182; influence toxicity mutagenicity Me-lex, sequence <br />                                               selective N3-adenine methylating agent. DNA Repair. 2008; 7:431&#226;&#8364;&#8220;438. [PubMed: 18182332] <br />                                          23. Boiteux S, Huisman O, Laval J. 3-Methyladenine residues DNA induce SOS function sfiA in <br />                                               Escherichia coli. EMBO J. 1984; 3:2569&#226;&#8364;&#8220;2573. [PubMed: 6239774] <br />                                          24. Larson K, Sahm J, Shenkar R, Strauss B. Methylation-induced blocks vitro DNA replication. <br />                                               Mutat Res. 1985; 150:77&#226;&#8364;&#8220;84. [PubMed: 4000169] <br />                                          25. Plosky BS, Frank EG, Berry DA, Vennall GP, McDonald JP, Woodgate R. Eukaryotic Y-family <br />                                               polymerases bypass 3-methyl-2&#226;&#8364;&#178;-deoxyadenosine analog vitro methyl methanesulfonate- <br />                                               induced DNA damage vivo. Nucleic Acids Res. 2008; 36:2152&#226;&#8364;&#8220;2162. [PubMed: 18281311] <br />                                          26. Settles Wang R-W, Fronza G, Gold B. Effect N3-Methyladenine isosteric stable <br />                                               analogue DNA polymerization. J Nucleic Acids. 2010:426505. [PubMed: 20936169] <br />                                          27. Monti P, Broxson C, Inga A, Wang RW, Menichini P, Tornaletti S, Gold B, Fronza G. 3-Methyl-3- <br />                                               deazaadenine, stable isostere N3-methyl-adenine, efficiently bypassed replication vivo <br />                                               transcription vitro. DNA Repair. 2011; 10:861&#226;&#8364;&#8220;868. [PubMed: 21676659] <br />                                          28. Tang JB, Svilar D, Trivedi RN, Wang XH, Goellner EM, Moore B, Hamilton RL, Banze LA, <br />                                               Brown AR, Sobol RW. N-methylpurine DNA glycosylase DNA polymerase beta modulate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               BER inhibitor potentiation glioma cells temozolomide. Neuro Oncol. 2011; 13:471&#226;&#8364;&#8220;486. <br />                                               [PubMed: 21377995] <br />                                          29. [Tang J, Goellner EM, Wang XW, Trivedi RN, St Croix CM, Jelezcova E, Svilar D, Brown AR, <br />                                               Sobol RW. Bioenergetic metabolites regulate base excision repair-dependent cell death response <br />                                               DNA damage. Mol Cancer Res. 2010; 8:67&#226;&#8364;&#8220;79. [PubMed: 20068071] <br />                                          30. Mutamba JT, Svilar D, Prasongtanakij S, Wang XH, Lin YC, Dedon PC, Sobol RW, Engelward <br />                                               BP. XRCC1 base excision repair balance response nitric oxide. DNA Repair. 2011; <br />                                               10:1282&#226;&#8364;&#8220;1293. [PubMed: 22041025] <br />                                          31. Svilar D, Vens C, Sobol RW. Quantitative, real-time analysis base excision repair activity cell <br />                                               lysates utilizing lesion-specific molecular beacons. J Vis Exp. 2012; 66:e4168. [PubMed: <br />                                               22895410] <br />                                          32. Svilar D, Dyavaiah M, Brown AR, Tang JB, Li J, McDonald PR, Shun TY, Braganza A, Wang <br />                                               XH, Maniar S, St Croix CM, Lazo JS, Pollack IF, Begley TJ, Sobol RW. Mol Cancer Res. 2012 <br />                                               [Epub ahead print. <br />                                          33. Baird EE, Dervan PB. Solid phase synthesis polyamides containing imidazole pyrrole <br />                                               amino acids. J Am Chem Soc. 1996; 118:6141&#226;&#8364;&#8220;6146. <br />                                          34. Wurtz NR, Turner JM, Baird EE, Dervan PB. Fmoc solid phase synthesis polyamides containing <br />                                               pyrrole imidazole amino acids. Org Lett. 2001; 3:1201&#226;&#8364;&#8220;1203. [PubMed: 11348194] <br />                                          35. Garbett NC, Ragazzon PA, Chaires JB. Circular dichroism determine binding mode affinity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               ligand-DNA interactions. Nat Protoc. 2007; 2:3166&#226;&#8364;&#8220;3172. [PubMed: 18079716] <br />                                          36. Sidorova JM, Li N, Schwartz DC, Folch A, Monnat RJ Jr. Microfluidic-assisted analysis of <br />                                               replicating DNA molecules. Nat Protoc. 2009; 4:849&#226;&#8364;&#8220;861. [PubMed: 19444242] <br />                                          37. Kubo K, Ide H, Wallace SS, Kow YW. A novel, sensitive, specific assay abasic sites, the <br />                                               commonly produced DNA lesion. Biochemistry. 1992; 31:3703&#226;&#8364;&#8220;3708. [PubMed: 1567824] <br />                                          38. Kow YW, Dare A. Detection abasic sites oxidative DNA base damage using ELISA-like <br />                                               assay. Methods. 2000; 22:164&#226;&#8364;&#8220;169. [PubMed: 11020331] <br />                                          39. Nakamura J, Walker VE, Upton PB, Chiang SY, Kow YW, Swenberg JA. Highly sensitive <br />                                               apurinic/apyrimidinic site assay detect spontaneous chemically induced depurination <br />                                               physiological conditions. Cancer Res. 1998; 58:222&#226;&#8364;&#8220;225. [PubMed: 9443396] <br />                                          40. Bennett SE, Kitner J. Characterization aldehyde reactive probe reaction AP-sites in <br />                                               DNA: influence AP-lyase adduct stability. Nucleosides Nucleotides Nucleic Acids. 2006; <br />                                               25:823&#226;&#8364;&#8220;842. [PubMed: 16898421] <br />  <br />  <br />  <br />                                            Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                                 Page 17 <br />  <br />  <br />                                          41. Dantzer F, Schreiber V, Niedergang C, Trucco C, Flatter E, De La Rubia G, Oliver J, Rolli V, <br />                                              M&#195;&#169;nissier-de Murcia J, Murcia G. Involvement poly(ADP-ribose) polymerase base <br />                                              excision repair. Biochimie. 1999; 81:69&#226;&#8364;&#8220;75. [PubMed: 10214912] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          42. Khodyreva SN, Prasad R, Ilina ES, Sukhanova MV, Kutuzov MM, Liu Y, Hou EW, Wilson SH, <br />                                              Lavrik OI. Apurinic/apyrimidinic (AP) site recognition 5&#226;&#8364;&#178;-dRP/AP lyase poly(ADP- <br />                                              ribose) polymerase-1 (PARP-1). Proc Natl Acad Sci USA. 2010; 107:22090&#226;&#8364;&#8220;22095. [PubMed: <br />                                              21127267] <br />                                          43. Eustermann S, Videler H, Yang JC, Cole PT, Gruszka D, Veprintsev D, Neuhaus D. The DNA- <br />                                              binding domain human PARP-1 interacts DNA single-strand breaks monomer through <br />                                              second zinc finger. J Mol Biol. 2011; 407:149&#226;&#8364;&#8220;170. [PubMed: 21262234] <br />                                          44. Lazebnik YA, Kaufmann SH, Desnoyers S, Poirier GG, Earnshaw WC. Cleavage poly(ADP- <br />                                              ribose) polymerase proteinase properties like ICE. Nature. 1994; 371:346&#226;&#8364;&#8220;347. <br />                                              [PubMed: 8090205] <br />                                          45. Nicholson DW, Thornberry NA. Caspases: killer proteases. Trends Biochem Sci. 1997; 22:299&#226;&#8364;&#8220; <br />                                              306. [PubMed: 9270303] <br />                                          46. Germain M, Affar EB, D&#226;&#8364;&#8482;Amours D, Dixit VM, Salvesen GS, Poirier GG. Cleavage of <br />                                              automodified poly(ADP-ribose) polymerase apoptosis. Evidence involvement of <br />                                              caspase-7. J Biol Chem. 199; 274:28379&#226;&#8364;&#8220;28384. [PubMed: 10497198] <br />                                          47. Beranek DT, Weis CC, Swenson DH. A comprehensive quantitative analysis methylated and <br />                                              ethylated DNA using high pressure liquid chromatography. Carcinogenesis. 1980; 1:595&#226;&#8364;&#8220;606. <br />                                              [PubMed: 11219835] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          48. Rinne ML, He Y, Pachkowski BF, Nakamura J, Kelley MR. N-methylpurine DNA glycosylase <br />                                              overexpression increases alkylation sensitivity rapidly removing non-toxic 7-methylguanine <br />                                              adducts. Nucleic Acids Res. 2005; 33:2859&#226;&#8364;&#8220;2867. [PubMed: 15905475] <br />                                          49. Goellner EM, Grimme B, Brown AR, Lin YC, Wang XH, Sugrue KF, Mitchell L, Trivedi RN, <br />                                              Tang JB, Sobol RW. Overcoming temozolomide resistance glioblastoma dual inhibition of <br />                                              NAD+ biosynthesis base excision repair. Cancer Res. 2011; 71:2308&#226;&#8364;&#8220;2317. [PubMed: <br />                                              21406402] <br />                                          50. Call&#195;&#169;ja F, Jansen JG, Vrieling H, Laval F, van Zeeland AA. Modulation toxic mutagenic <br />                                              effects induced methyl methanesulfonate Chinese hamster ovary cells overexpression of <br />                                              rat N-alkylpurine DNA-glycosylase. Mutat Res. 1999; 425:185&#226;&#8364;&#8220;194. [PubMed: 10216211] <br />                                          51. Chen DS, Herman T, Demple B. Two distinct human DNA diesterases hydrolyze 3&#226;&#8364;&#178;-blocking <br />                                              deoxyribose fragments oxidized DNA. Nucleic Acids Res. 1991; 19:5907&#226;&#8364;&#8220;5914. [PubMed: <br />                                              1719484] <br />                                          52. Srivastava DK, Vande Berg BJ, Prasad R, Molina JT, Beard WA, Tomkinson AE, Wilson SH. <br />                                              Mammalian abasic site base excision repair. Identification reaction sequence rate- <br />                                              determining steps. J Biol Chem. 1998; 273:21203&#226;&#8364;&#8220;21209. [PubMed: 9694877] <br />                                          53. Bobola MS, Varadarajan S, Smith NW, Goff RD, Kolstoe DD, Blank A, Gold B, Silber JR. Human <br />                                              glioma cell sensitivity sequence-specific alkylating agent methyl-lexitropsin. Clin Cancer <br />                                              Res. 2007; 13:612&#226;&#8364;&#8220;620. [PubMed: 17255284] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          54. Srinivasan A, Wang L, Cline CJ, Xie Z, Sobol RW, Xie X-Q, Gold B. The identification and <br />                                              characterization human AP endonuclease-1 inhibitors. Biochemistry. 2012; 51:6246&#226;&#8364;&#8220;6259. <br />                                              [PubMed: 22788932] <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                              Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 1. <br />                                          Structures compounds. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                        Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 2. <br />                                          (A) Formation N3-methyladenine (3-mA) 24 h incubation calf thymus DNA <br />                                          (100 &#206;&#188;M nucleotide) compounds 1&#226;&#8364;&#8220;4, 7 8 room temperature 10 mM sodium <br />                                          cacodylate buffer (pH 7.0): 1 (&#226;&#8211;&#160;), 2 (&#226;&#8212;?), 3 (&#226;&#8211;&#178;), 4 (&#226;&#8211;&#188;), 7 (&#226;&#8212;&#8224;), 8. (&#226;&#339;&#8211;) Formation 3-mA and <br />                                          N7-methylguanine (7-mG) incubation 1 7 calf thymus DNA 10 mM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          cacodylate buffer (pH 7.0) room temperature 24 h. <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                        Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 3. <br />                                          Titration curve ICD spectrum sulfone derivatives minor groove binders 10% <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          EtOH phosphate buffer (pH 7) 20 &#194;&#176;C. CD 10 &#206;&#188;M minor groove binder the <br />                                          presence increasing concentration (1 nM 0.1 mM, base pairs) CT DNA poly- <br />                                          d(A-T): inset corresponding ICD spectra (arrow indicates increasing concentration of <br />                                          DNA): (A) netropsin + CT-DNA; (B) distamycin + CT-DNA; (C) netropsin + poly-d(A&#226;&#8364;&#8220;T); <br />                                          (D) distamycin + poly-d(A-T); (E) 10 + CT-DNA; (F) 11 + CT-DNA; (G) 10 + poly-d(A- <br />                                          T); (H) 11 + poly-d(A&#226;&#8364;&#8220;T). Binding constants (Table 2) calculated using non-linear fit option <br />                                          Origin software. <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                           Page 21 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 4. <br />                                          Toxicity 1&#226;&#8364;&#8220;4, 7 8 wild type AB1157 E. coli (closed/bolded symbols) GC4803 <br />                                          alkA/tag glycosylase mutant cells (open symbols/non-bolded): dipeptides 1 (&#226;&#8211;&#160;,&#226;&#8211;&#161;), 2 (&#226;&#8212;?,&#226;&#8212;&#8249;), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          3 (&#226;&#8211;&#178;,&#226;&#8211;&#179;), 4 (&#226;&#8211;&#188;, &#226;&#8211;&#189;) tripeptides 7 (&#226;&#8212;&#8224;,&#226;&#8212;&#8225;), 8 (&#226;&#339;&#8211;, &#226;&#339;&#8226;) 9 (&#226;&#339;&#353;,+). Mid-log phase cells were <br />                                          exposed varying concentrations analogues 1 h, serially diluted survival <br />                                          measured solid LB media. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                       Page 22 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 5. <br />                                          Toxicity WT T98G cells exposed varying concentrations dipeptide 1 (&#226;&#8212;?) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          tripeptide 7 (&#226;&#8211;&#160;) <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>cell survival</span> measured standard MTS assay. Data was <br />                                          fit using non-linear (curve-fit) Prism software. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                       Page 23 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 6. <br />                                          Toxicity 1 T98G glioma cells: wild type (WT,&#226;&#8212;?) MPG over-expressing cells <br />                                          (MPG+,&#226;&#8211;&#160;) MPG knock cells (MPG-,&#226;&#8211;&#178;). The cells exposed increasing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          concentrations 1 survival determined CyQuant assay 9 days exposure, as <br />                                          described Methods section. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                         Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 7. <br />                                          Time-dependent effect di- tri-peptide methylating agents 1 7, respectively, the <br />                                          formation aldehyde reactive sites (ARS) WT, MPG over-expressing (MPG+) MPG <br />                                          deficient (MPG-) T98G cells: (A) WT DMSO (&#226;&#8211;&#160;), MPG+ DMSO (&#226;&#8211;&#178;), MPG- <br />                                          DMSO (&#226;&#8212;?),WT 80 &#206;&#188;M 1 (&#226;&#8211;&#161;), MPG+ 80 &#206;&#188;M 1 (&#226;&#8211;&#179;), MPG- 80 &#206;&#188;M 1 (&#226;&#8212;&#8249;); <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (B) WT DMSO (&#226;&#8211;&#160;), MPG+ DMSO (&#226;&#8211;&#178;), MPG- DMSO (&#226;&#8212;?),WT 7 &#206;&#188;M 7 <br />                                          (&#226;&#8211;&#161;), MPG+ 7 &#206;&#188;M 7 (&#226;&#8211;&#179;), MPG- 7 &#206;&#188;M 7 (&#226;&#8212;&#8249;). <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                       Page 25 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 8. <br />                                          Time course (A) PARP activation; (B) caspase-3 -7 activation; (C) cleavage of <br />                                          PARP T98G WT cells: (&#226;&#8212;&#8225;) untreated; (&#226;&#8212;&#8224;) 0.2% DMSO; (&#226;&#8211;&#179;) 80 &#206;&#188;M 1; (&#226;&#8211;&#178;) 250 &#206;&#188;M 1. <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                  Page 26 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 9. <br />                                          Normalized PAR production T98G glioma cells: wild type (WT, solid) MPG over- <br />                                          expressing cells (MPG+, open columns) MPG knockdown cells (MPG-, hatched) a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          function concentration 1. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                       Page 27 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 10. <br />                                          Microfluidic-assisted replication tract analysis (ma-RTA) A172 glioma cells and <br />                                          40 min exposure 5 10 &#206;&#188;M 1: (A) experimental design; (B) images micro track <br />                                          capillaries; (C) percentile analysis track length (&#206;&#188;m) B. The Kolmogorov&#226;&#8364;&#8220;Smirnov <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (K&#226;&#8364;&#8220;S) test performed determine P-values different cumulative distributions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                              Page 28 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 11. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Repair N3-methyladenine (3-mA) base excision repair. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                             Iyer et al.                                                                                 Page 29 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Scheme 1. <br />                                          Reagents conditions: (i) n-PrNH2, EtOAc; (ii) 10% Pd/C, EtOH; (iii) compound c, <br />                                          EDC, HOBT, Et3N, CH2Cl2; (iv) TFA, CH2Cl2; (v) 10% Pd/C, MeOH; (vi) CH2=CHCOCl, <br />                                          DIEA, THF; (vii) 1M TiCl4, CH2Cl2; (viii) compound f, EDC, HOBT, Et3N; (ix) NaHSO3, <br />                                          EtOH/H2O; (x) dioxane, 80 &#194;&#176;C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                                                                            NIH-PA Author Manuscript                                             NIH-PA Author Manuscript                                        NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                      Table 1 <br />  <br />                                                                           Minor (3-mA) major (7-mG) groove adduct production (pmol/&#206;&#188;g DNA) reaction 1 7 CT-DNA.a <br />  <br />                                                                               compound concentration     1          7     1          7     1           7     1            7 <br />                                                                                                                                                                                                                                                                          Iyer et al. <br />  <br />  <br />  <br />  <br />                                                                                                              1 &#206;&#188;M             5 &#206;&#188;M             10 &#206;&#188;M             100 &#206;&#188;M <br />  <br />                                                                                       3-mA             7.15    13.11    24.36    38.04   25.55    62.08    232.14    365.27 <br />  <br />                                                                                       7-mA             0.04     0.13    0.13     0.22     0.43     0.39     3.56      3.13 <br />  <br />                                                                                    3-mA/7-mG            179     101      187     173       59      159       65       117 <br />  <br />                                                                           a <br />                                                                            Reactions conditions: 24 h incubation compounds CT-DNA (100 &#206;&#188;M, p) room temperature 10 mM sodium phosphate buffer (pH 7.0). Data based HPLC analysis UV detection. <br />  <br />  <br />  <br />  <br /> Chem Res Toxicol. Author manuscript; available PMC 2014 February 05. <br />                                                                                                                                                                                                                                                                          Page 30 <br />                             Iyer et al.                                                                                                                             Page 31 <br />  <br />  <br />  <br />  <br />                                                                                                      Table 2 <br />                            Thermodynamic parameters equilibrium binding minor groove binding methylsulfones calf <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            thymus (CT)-DNA poly-d(A-T)a <br />  <br />                                                          &#206;&#8221;TM (&#194;&#176;C)c                    Kb (M&#226;&#710;&#8217;1)d <br />  <br />                                Compoundb        CT-DNA        poly-d(A-T)   CT-DNA        poly-d(A-T) <br />  <br />                                      5             0.4            1.2         n.d.e           n.d. <br />  <br />                                      6             0.4            1.2         n.d.            n.d <br />  <br />                                     10             1.1            4.1       2.9 &#195;&#8212; 104       2.0 &#195;&#8212; 104 <br />  <br />                                     11             1.1            4.7     <br /> </body></html>