<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Unbalanced Charge Distribution as a Determinant for Dependence of <br /> a Subset of Escherichia coli Membrane Proteins on the Membrane <br /> Insertase YidC <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span> Membrane proteins are involved in numerous essential cell processes, including transport, gene regulation, motility, <br /> and metabolism. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> function properly, they must be inserted into the membrane and folded correctly. YidC, an essential protein <br /> in Escherichia coli with homologues in other bacteria, Archaea, mitochondria, and chloroplasts, functions by incompletely un- <br /> derstood mechanisms in the insertion and folding of certain membrane proteins. Using a genome-scale approach, we identified <br /> 69 E. coli membrane proteins that, in the absence of YidC, exhibited aberrant localization by microscopy. Further examination <br /> of a subset revealed biochemical defects in membrane insertion in the absence of YidC, indicating their dependence on YidC for <br /> proper membrane insertion or folding. Membrane proteins possessing an unfavorable distribution of positively charged resi- <br /> dues were significantly more likely to depend on YidC for membrane insertion. Correcting the charge distribution of a charge- <br /> unbalanced YidC-dependent membrane protein abrogated its requirement for YidC, while perturbing the charge distribution of <br /> a charge-balanced YidC-independent membrane protein rendered it YidC dependent, demonstrating that charge distribution <br /> can be a necessary and sufficient determinant of YidC dependence. These findings provide insights into a mechanism by which <br /> YidC promotes proper membrane protein biogenesis and suggest a critical function of YidC in all organisms and organelles that <br /> express it. <br /> IMPORTANCE Biological membranes are fundamental components of cells, providing barriers that enclose the cell and separate <br /> compartments. Proteins inserted into biological membranes serve critical functions in molecular transport, molecular partition- <br /> ing, and other essential cell processes. The mechanisms involved in the insertion of proteins into membranes, however, are in- <br /> completely understood. The YidC protein is critical for the insertion of a subset of proteins into membranes across an evolution- <br /> arily wide group of organisms. Here we identify a large group of proteins that depend on YidC for membrane insertion in <br /> Escherichia coli, and we identify unfavorable distribution of charge as an important determinant of YidC dependence for proper <br /> membrane insertion. <br />  <br />   <br /> How proteins are properly inserted into and folded within <br />      membranes is a fundamental and incompletely understood <br /> question of cell biology. Escherichia coli genes encode approxi- <br />                                                                                               Certain cytoplasmic membrane proteins require the mem- <br />                                                                                           brane protein YidC for proper insertion (2&#226;&#8364;&#8220;4). Some YidC sub- <br />                                                                                           strates are inserted into the membrane via the Sec translocon, <br /> mately 900 cytoplasmic membrane proteins, constituting ~20%                               while others utilize a YidC-dependent but Sec-independent path- <br /> of the proteins produced by the cell (1). Most are inserted into the                      way (5&#226;&#8364;&#8220;8). YidC is highly conserved, with homologues in mito- <br /> membrane via a canonical signal recognition particle (SRP)-                               chondria and chloroplasts, most bacterial species, and some mem- <br /> dependent, Sec translocon-dependent insertion pathway (re-                                bers of the domain Archaea (9, 10), and is essential for cell viability <br /> viewed in reference 2). Nascent membrane proteins are recog-                              (7). On the basis of cross-linking to model membrane proteins, <br /> nized and bound by SRP, targeted to the membrane via the                                  YidC has been proposed to act as a chaperone (11), mediating the <br /> membrane receptor FtsY, and transferred to the SecYEG translo-                            partitioning of nascent transmembrane segments <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> the Sec <br /> con for cotranslational insertion, whereby exterior hydrophilic <br />                                                                                           translocon, with which it is physically associated (12), into the <br /> domains are translocated to the periplasm, hydrophobic trans- <br />                                                                                           lipid bilayer, as well as the proper bundling of transmembrane <br /> membrane segments partitioned into the lipid bilayer, and inte- <br />                                                                                           segments (13). Some substrates strictly require YidC for mem- <br /> rior hydrophilic domains synthesized in the cytoplasm. <br />  <br />  <br /> November/December 2011 Volume 2 Issue 6 e00238-11                                                                                                            &#194;&#174; <br />  <br />                                                                                                                                                                  mbio.asm.org 1 <br />  Gray et al. <br />  <br />  <br />  <br /> brane insertion (14, 15), while others require it for efficient inser-   duced with IPTG after 3 h of growth either in the absence of <br /> tion (16, 17) or proper folding (18).                                    arabinose, at which time no detectable YidC was present, or in the <br />     Nearly one-third of E. coli membrane proteins that have been         presence of arabinose (Fig. S1B and S1C). Most of the 415 proteins <br /> quantified in proteomic studies display significantly altered abun-      screened exhibited similar fluorescence localization patterns un- <br /> dance in the absence of YidC (19, 20), suggesting its involvement        der the two conditions (e.g., YaiZ in Fig. 1B), suggesting that in- <br /> in the insertion of a substantial subset of E. coli membrane pro-        sertion of most membrane proteins was not significantly per- <br /> teins; however, dependence on YidC for membrane insertion per            turbed in the absence of YidC. <br /> se has been demonstrated for only 14 proteins in E. coli. While              Sixty-nine membrane proteins (16.6%) showed distinct fluo- <br /> studies employing model membrane proteins have begun to de-              rescence localization patterns in the presence versus absence of <br /> fine the role of YidC in membrane protein biogenesis, how it func-       YidC (see Table S1 in the supplemental material). Most (40 of 69) <br /> tions and which traits of substrate proteins contribute to YidC          showed a &#226;&#172;&#381;50% decrease in circumferential membrane fluores- <br /> dependence remain poorly understood. To address these issues,            cence in the absence of YidC with no increase in diffuse cytoplas- <br /> we identified and characterized a large group of YidC-dependent          mic fluorescence (e.g., AtpB, XylE, and YbhS in Fig. 1A), most <br /> membrane proteins in E. coli. We show that membrane proteins             likely reflecting an overall decrease in protein abundance. A pleio- <br /> possessing an unfavorable distribution of positively charged resi-       tropic effect of YidC depletion on gene expression was unlikely, <br /> dues are significantly more likely to require YidC for proper inser-     since most membrane proteins showed no appreciable decrease in <br /> tion and that charge distribution per se can be a necessary and          fluorescence, and a panel of cytoplasmic membrane and soluble <br /> sufficient determinant of YidC dependence.                               cytoplasmic proteins showed no change in abundance (see Fig. S2 <br />                                                                          in the supplemental material). Since membrane proteins that are <br /> RESULTS                                                                  not inserted into the membrane or that fold improperly are often <br /> Screen to identify YidC-dependent cytoplasmic membrane pro-              subject to degradation, the observed reduction in circumferential <br /> teins in E. coli. Bacterial cells synthesizing green fluorescent pro-    fluorescence was most consistent with protein degradation conse- <br /> tein (GFP)-tagged membrane proteins typically exhibit a fluores-         quent to altered membrane insertion or folding. In certain cases, <br /> cent signal that is circumferential around the cell periphery,           mislocalization may have been due to the effects of YidC depletion <br /> reflecting uniform distribution of the protein within the mem-           on other proteins required for the stability or proper localization <br /> brane. The signal intensity is proportional to the abundance of the      of the protein, and reduced fluorescence may have been due to <br /> protein in the membrane. Based on the principle that a perturba-         aggregation, since this can lead to fluorescence quenching (24). <br /> tion of proper membrane insertion can alter the abundance, sub-              Two other patterns that were observed in the absence of YidC <br /> cellular localization, or distribution of this signal, we designed a     were bright punctate fluorescent foci around the periphery of the <br /> screen to identify E. coli cytoplasmic membrane proteins that are        cell (17 of 69 hits, e.g., YccF and YhaH) or at the cell poles (12 of 69 <br /> dependent on YidC for proper membrane insertion.                         hits, e.g., MsbB and YecS; Fig. 1A). The presence of punctate foci <br />     We screened a subset of a genome-scale plasmid library con-          indicated that the GFP-tagged proteins were unevenly distributed <br /> sisting of each open reading frame of E. coli strain W3110 cloned        in the membrane and were accumulating at discrete locations, <br /> with an N-terminal His6 tag and a C-terminal GFP tag and ex-             consistent with protein aggregation and strongly suggestive of de- <br /> pressed under the control of the isopropyl-&#226;&#65533;&#164;-D-                          fects in membrane insertion. Their position at the cell periphery <br /> thiogalactopyranoside (IPTG)-inducible PT5-lac promoter (the             suggested that the proteins either had been partially inserted into <br /> ASKA Library [21]). The subcellular localization of each protein         the membrane or remained otherwise membrane associated. Mis- <br /> in this library had been categorized, with 492 proteins showing a        folded proteins have an increased propensity to form aggregates, <br /> circumferential distribution consistent with localization to the         particularly when hydrophobic regions of a protein become sol- <br /> membrane. Using the hidden Markov model topology algorithm               vent exposed, as would occur if a hydrophobic transmembrane <br /> Phobius (22), E. coli W3110 is predicted to encode 936 integral          segment failed to insert into the lipid bilayer. <br /> cytoplasmic membrane proteins, consistent with published esti-               Among the membrane proteins that were mislocalized in the <br /> mates for other E. coli strains (1). Of these 936 proteins, 428 (46%)    absence of YidC were two proteins previously shown to be YidC <br /> were included in the membrane-localized subset of the ASKA Li-           dependent for membrane insertion, AtpB (F0c) (Fig. 1A) and <br /> brary (see Data Set S1 in the supplemental material). Sixty-four         ProW (7, 25). Two additional previously described YidC sub- <br /> additional proteins observed to be membrane-localized proteins           strates, MalF and MtlA, were included in the input to the screen <br /> but not predicted to be integral membrane proteins are likely to be      but not identified as hits; MalF requires YidC for stability and <br /> peripheral membrane proteins and were excluded from our                  MalFGK2 complex assembly, but not for insertion or topogenesis <br /> screen. Since GFP folds rapidly in the cytoplasm and cannot be           (16), and MtlA interacts with YidC during membrane insertion in <br /> secreted by Sec in its folded state, we restricted our analysis to       vitro (13), but whether it depends on YidC for insertion has not <br /> membrane proteins that were either predicted (22) or have been           been examined; these observations likely account for MalF and <br /> shown experimentally (23) to have C termini in the cytoplasm. Of         MtlA not being identified as hits in our screen. The identification <br /> the 428 membrane-localized integral cytoplasmic membrane pro-            of previously described YidC-dependent membrane proteins as <br /> teins, the 415 that met these criteria constituted the input pool for    hits in our screen indicated the sensitivity of our assay. Membrane <br /> our screen.                                                              proteins that, in the absence of YidC, fail to insert properly yet <br />     We examined the subcellular localization of each GFP-tagged          remain membrane associated and evenly distributed might be <br /> membrane protein in the absence versus presence of YidC, using a         missed by our screen. <br /> strain in which yidC expression was controlled by the arabinose-             Membrane proteins mislocalized in the absence of YidC <br /> inducible ParaBAD promoter (see Fig. S1A in the supplemental ma-         show biochemical defects in membrane insertion. To test <br /> terial). Synthesis of each GFP-tagged membrane protein was in-           whether membrane proteins that displayed altered fluorescence <br />  <br />  <br /> 2        &#194;&#174; <br />  <br />               mbio.asm.org                                                                  November/December 2011 Volume 2 Issue 6 e00238-11 <br />                                                                                                          Unbalanced Charge Distribution in YidC Substrates <br />  <br />  <br />  <br />  <br /> FIG 1 YidC-dependent localization of a subset of GFP-tagged membrane proteins. (a) Seven examples of membrane proteins that showed altered GFP signals <br /> in the absence of YidC (&#226;&#171;&#186; YidC). The AtpB protein previously shown to be dependent on YidC for membrane insertion (&#227;&#8364;&#171;) is shown. (b) Membrane protein <br /> that showed comparable circumferential GFP signals in the presence (&#226;&#171;&#185;) and absence (&#226;&#171;&#186;) of YidC (YaiZ). Images are representative. Bar, 5 &#226;&#65533;&#174;m. <br />  <br />  <br />  <br /> localization in the absence of YidC were indeed dependent on                   In spheroplasted cells, which lack an outer membrane and cell <br /> YidC for proper membrane insertion, we assayed membrane in-                 wall, periplasmic segments of integral membrane proteins are ex- <br /> sertion biochemically, focusing on those screen hits that main-             posed and can be cleaved by an exogenously added protease to <br /> tained sufficient abundance in the absence of YidC for this analysis        which they are sensitive (Fig. 2A). Defects in membrane insertion <br /> (see Fig. S3A to S3C in the supplemental material). For certain of          can result in either protection from proteolysis or altered cleavage. <br /> these hits, as well as for a representative nonhit, we generated            We examined whether, in the absence of YidC, the periplasmic <br /> strains in which the GFP-tagged protein was expressed from a                segments of hit proteins had altered accessibility to the protease <br /> single copy on the chromosome (Fig. S3C and S1D), allowing us to            proteinase K. Because the GFP tag on each fusion was retained in <br /> examine membrane insertion at lower levels of expression. To                the cytoplasm, it was protected from proteolysis, and C-terminal <br /> enable induction of protein synthesis after depletion of YidC, the          cleavage products were detected as truncated GFP fusions. <br /> IPTG-inducible promoter from the parent plasmid was retained                   Of the 19 GFP-tagged membrane proteins that we examined, <br /> in the chromosomally integrated constructs. In all examined                 four&#226;&#8364;&#8221;AtpB, CrcB, ExbB, and YdhU&#226;&#8364;&#8221; demonstrated cleavage by <br /> cases, the GFP fluorescence localization phenotypes for the single-         proteinase K when synthesized in the presence of YidC that was <br /> copy integrants were the same as those observed in the initial              distinct from that observed when synthesized in the absence of <br /> microscopy screen (Fig. S1E and S1F). Thus, episomal expression             YidC or following cell lysis. In the absence of YidC, proteolysis of <br /> did not contribute significantly to the YidC dependence of these            each of these four proteins was dramatically reduced [Fig. 2B, <br /> proteins.                                                                   increase in the abundance of the full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> protein (asterisk) rel- <br />  <br />  <br /> November/December 2011 Volume 2 Issue 6 e00238-11                                                                                     &#194;&#174; <br />  <br />                                                                                                                                           mbio.asm.org 3 <br />  Gray et al. <br />  <br />  <br />  <br />  <br /> FIG 2 Membrane proteins mislocalized in the absence of YidC are YidC dependent for membrane insertion. (A and B) Proteinase K accessibility assay of <br /> spheroplasts depleted of YidC or not depleted of YidC (see Materials and Methods). (A) Experimental approach. (B) Screen hits that displayed reduced protease <br /> accessibility following YidC depletion. The spheroplasts were treated with proteinase K (PK) (&#226;&#171;&#185;) in the protease accessibility assay. Cells pretreated with Triton <br /> X-100 (&#226;&#171;&#185;) to promote lysis served as a control for proteolysis. In the absence of YidC (&#226;&#171;&#186;), increased abundance of full-length protein (*) and decreased <br /> abundance of major cleavage product(s) (**) was observed. The positions of molecular mass standards (in kilodaltons) are shown to the left of the gels. (C to E) <br /> Differential fractionation of screen hits into Triton X-100-soluble and -insoluble membrane fractions following synthesis in cells depleted of YidC or not depleted <br /> of YidC. (C) Experimental approach. (D) Screen hits that displayed reduced Triton X-100 solubility when synthesized in the absence of YidC. (E) A screen nonhit <br /> that displayed comparable Triton X-100 solubilities when synthesized in the presence and absence of YidC. Western blots using antibody to GFP are shown. The <br /> cell fractions are as follows: crude, whole-cell proteins; soluble, soluble cytoplasmic and periplasmic proteins; insoluble, membranes and other insoluble proteins; <br /> Triton-soluble, Triton X-100-soluble fraction of insoluble proteins; Triton-insoluble, Triton X-100-insoluble fraction of insoluble proteins. The loads were <br /> proportional and normalized to the optical density at 600 nm (OD600) of culture. Percent solubilization, the ratio of Triton-soluble band to crude fraction band <br /> as determined by band densitometry. Images are representative. <br />  <br />  <br /> ative to the major cleavage product(s) (two asterisks)], indicating                gregation. As an indicator of protein aggregation, we examined <br /> that they were protected from proteolysis and suggesting that they                 solubility in the detergent Triton X-100, in which cytoplasmic <br /> were either not inserted into the membrane or were inserted with                   membrane proteins can generally be solubilized but outer mem- <br /> an altered, protease-resistant conformation. Even in the presence                  brane proteins, protein aggregates, and inclusion bodies cannot <br /> of YidC, some CrcB and some ExbB were inaccessible to protei-                      (26). After separating membranes and other insoluble material <br /> nase K cleavage, suggesting either that the proteins were incom-                   from soluble cytoplasmic and periplasmic contents by crude frac- <br /> pletely inserted under our synthesis conditions or, in the case of                 tionation, we selectively solubilized nonaggregated cytoplasmic <br /> CrcB, that two conformations exist (see below). Nevertheless, al-                  membrane proteins from the total insoluble fraction with Triton <br /> tered protease accessibility in the absence of YidC indicated that                 X-100 (Fig. 2C). Each of the proteins examined was solubilized in <br /> these proteins are dependent on YidC for proper membrane in-                       the presence of YidC but showed greatly reduced solubilization in <br /> sertion.                                                                           the absence of YidC (Fig. 2D). Where both assays could be per- <br />     Membrane proteins that exhibit punctate localization in the                    formed, these results consistently correlated with those of protease <br /> absence of YidC form insoluble protein aggregates. Many GFP-                       accessibility experiments (Fig. 2B). The nonhit YaiZ showed sim- <br /> tagged membrane proteins that were mislocalized in the absence                     ilar solubility under the two conditions (Fig. 2E). <br /> of YidC exhibited punctate fluorescence, suggestive of protein ag-                     Together, the results of our biochemical analyses demonstrate <br />  <br />  <br /> 4        &#194;&#174; <br />  <br />               mbio.asm.org                                                                               November/December 2011 Volume 2 Issue 6 e00238-11 <br />                                                                                                  Unbalanced Charge Distribution in YidC Substrates <br />  <br />  <br />  <br /> that mislocalization of GFP-tagged membrane proteins as ob-            charge-neutral first transmembrane segment, a balanced second <br /> served by microscopy was strongly indicative of membrane inser-        transmembrane segment, and an unbalanced third transmem- <br /> tion defects. Our screen hits thus represent a newly identified        brane segment (Fig. 3A). Other algorithms suggest that the CrcB <br /> group of 67 E. coli cytoplasmic membrane proteins (in addition to      signal peptide is not cleaved, resulting in an N-in four-span mem- <br /> AtpB and ProW) that are dependent on YidC for proper mem-              brane protein. Importantly, whether the signal peptide is cleaved <br /> brane insertion.                                                       has no effect on the charge balance of wild-type CrcB (CrcBWT) or <br />     Multiple proteins essential for cell growth are YidC depen-        the charge-altered variants described below. Substituting alanines <br /> dent. YidC is essential for cell growth and viability (7). The Sec-    for positively charged periplasmic residues, we created variants of <br /> independent membrane insertion pathway, substrates of which            CrcB that were successively more balanced (Fig. 3A); each con- <br /> include subunit Foc of the F1Fo ATPase (8), has been implicated in     struct was integrated onto the chromosome. <br /> mediating this essentiality (27). In addition, several membrane            The localization of balanced variants of CrcB was less affected <br /> proteins that we identified as YidC dependent are known or pre-        by YidC depletion than that of CrcBWT. CrcB with the R95A sub- <br /> dicted to be essential&#226;&#8364;&#8221;CorA, CydD, FtsX, Lnt, RodA, YbhN, and          stitution (CrcBR95A) and CrcB with the R25A and R95A substitu- <br /> YgaP (28&#226;&#8364;&#8220;33). Thus, the essentiality of YidC may result from the       tions (CrcBR25A/R95A), in which the third transmembrane segment <br /> loss of function of multiple essential cytoplasmic membrane pro-       is no longer unbalanced, showed substantial membrane localiza- <br /> teins that require it for proper membrane insertion.                   tion in the absence of YidC, whereas CrcBWT and CrcBR25A, which <br />     Membrane proteins containing unbalanced transmembrane              each retain an unbalanced third transmembrane segment, were <br /> segments are more likely to require YidC for proper membrane           highly mislocalized (Fig. 3B). In addition, while CrcBWT exhibited <br /> insertion. We searched for determinants of YidC dependence by          loss of proteinase K sensitivity in the absence of YidC (Fig. 3C, <br /> examining commonalities among the 69 proteins identified as hits       proteolytic product of CrcBWT detected in the presence, but not <br /> in our screen. There was no significant enrichment for gene onto-      absence, of YidC [asterisk]), indicating a membrane insertion de- <br /> logical function and process categories, or for protein families,      fect, the balanced CrcBR25A/R95A variant exhibited equivalent pro- <br /> groups, or domains, as determined using the Swiss-Prot database,       teolysis in the presence and absence of YidC (Fig. 3C). Partially <br /> nor was there significant enrichment in the number of transmem-        balanced variants CrcBR25A and CrcBR95A exhibited an intermedi- <br /> brane segments, the sizes of predicted periplasmic and cytoplas-       ate <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>phenotype</span> (Fig. 3C). Similarly, while CrcBWT showed no de- <br /> mic protein segments, or overall topology (N in versus N out), as      tectable solubilization in Triton X-100 in the absence of YidC, the <br /> determined using Phobius (22), in the hydrophobicity of pre-           balanced CrcBR25A/R95A variant showed near-equivalent solubili- <br /> dicted transmembrane segments, as determined using Kyte-               zation in the presence and absence of YidC, with the partially <br /> Doolittle and JTT2 hydropathy scales (1, 34), or in the distribution   balanced CrcBR95A variant exhibiting an intermediate phenotype <br /> of negatively charged residues.                                        (Fig. 3D). As shown above (Fig. 2B), even in the presence of YidC, <br />     A characteristic that was significantly enriched among YidC-       some protein is inaccessible to proteinase K cleavage, suggesting <br /> dependent membrane proteins was an atypical distribution of            that either two conformations exist or that the protein is incom- <br /> positively charged residues. Positively charged residues serve as      pletely inserted under our synthesis conditions. Nevertheless, <br /> determinants of membrane protein topology (35, 36), with a             whereas the unbalanced variants of CrcB were YidC dependent, <br /> strong topological preference for lysine and arginine residues to      the balanced variants no longer required YidC for proper mem- <br /> reside in the cytoplasm (the &#226;&#8364;&#339;positive inside rule&#226;&#8364;&#65533;). A transmem-      brane insertion, indicating that the distribution of positively <br /> brane segment for which the number of positively charged resi-         charged residues in CrcB, and likely in other membrane proteins, <br /> dues in the adjacent periplasmic segment is greater than the num-      serves as a strong determinant of YidC dependence. <br /> ber of positively charged residues in the adjacent cytoplasmic             The CrcBWT and CrcB variants used in these experiments all <br /> segment is referred to as &#226;&#8364;&#339;unbalanced.&#226;&#8364;&#65533; Such a topology is unfa-       contained a C-terminal GFP fusion. When the GFP tag was absent, <br /> vorable and is predicted to occur at low frequency.                    CrcBWT exhibited dual topology, consistent with recent observa- <br />     Within our screen input pool, 34 of 415 proteins (8.2%) were       tions (38), as a C-terminal LacZ fusion formed blue colonies on <br /> predicted by Phobius to contain unbalanced transmembrane seg-          agar containing 5-bromo-4-chloro-3-indolyl-galactopyranoside <br /> ments (see Data Set S1 and Fig. S4 in the supplemental material).      (BCIG or X-Gal), and a C-terminal PhoA fusion formed blue <br /> Two-thirds of these (67.6% [23 of 34]) were YidC dependent             colonies on agar that contained 5-bromo-4-chloro-3-indolyl <br /> (Fig. S5). Unbalanced membrane proteins constituted 33% of our         phosphate (XP) (see Fig. S6 in the supplemental material). It is <br /> screen hits (23 of 69) (Table S1) and were &#226;&#172;&#381;5-fold more likely to      likely that under our experimental conditions the GFP-tagged <br /> be YidC dependent than balanced membrane proteins (12.1% [46           CrcB constructs exhibited a C-in topology, as indicated by fluo- <br /> of 381]) (P &#226;&#171;&#189; 0.0001). Of these 23 proteins, all but 2 were also       rescence signal from the GFP; however, the GFP tag does not <br /> predicted to have an unbalanced topology when examined by a            prevent the remainder of the proteins from assuming dual topol- <br /> second algorithm, OCTOPUS (data not shown) (37). These find-           ogies. In any case, the dual topology of CrcB appeared to be inde- <br /> ings strongly suggest that unbalanced charge distribution can be       pendent of YidC dependence and the presence of unbalanced <br /> an important determinant of YidC dependence.                           transmembrane segments, as the partially balanced and balanced <br />     Rendering an unbalanced YidC-dependent membrane pro-               variants all displayed dual topology (Fig. S6). <br /> tein balanced promotes YidC-independent membrane inser-                    Rendering a balanced cytoplasmic membrane protein unbal- <br /> tion. To test whether YidC dependence can be determined by             anced is sufficient to promote dependence on YidC. To deter- <br /> unbalanced charge distribution per se, we altered the charge dis-      mine whether an unbalanced distribution of positively charged <br /> tribution of the YidC-dependent unbalance membrane protein             residues can also be sufficient to render a membrane protein YidC <br /> CrcB. CrcB is predicted by Phobius to be an N-out, C-terminus-in       dependent, we altered the charge distribution of YaiZ, a charge- <br /> (C-in) three-span polytopic membrane protein possessing a              balanced cytoplasmic membrane protein that did not require <br />  <br />  <br /> November/December 2011 Volume 2 Issue 6 e00238-11                                                                             &#194;&#174; <br />  <br />                                                                                                                                   mbio.asm.org 5 <br />  Gray et al. <br />  <br />  <br />  <br />  <br /> FIG 3 The distribution of positively charged residues determines the dependence of CrcB on YidC for membrane insertion. (A) Topological illustrations of the <br /> cytoplasmic membrane protein CrcB (22) and variants with altered charge balances. Balanced transmembrane segments (green), charge-neutral transmembrane <br /> segments (grey), and charge-unbalanced transmembrane segments (red) are shown. Charge balance altered by mutagenesis is indicated by an asterisk. (B) Subcellular <br /> localization of GFP-tagged CrcB (CrcB-GFP) variants in the presence or absence of YidC. Bar, 5 &#226;&#65533;&#174;m. (C and D) Protease susceptibility (C) or differential fractionation <br /> into Triton X-100-soluble and -insoluble membrane fractions (D) of CrcB-GFP variants following synthesis in the presence or absence of YidC (see Fig. 2 legend and <br /> Materials and Methods). The position of a proteolytic product detected only in the absence of YidC for CrcBWT but in increasing amounts in the absence of YidC for the <br /> balanced variants is indicated by an asterisk. Images are representative; all images in each panel are from the same Western blot or same microscopy experiment. <br />  <br />  <br />  <br /> 6        &#194;&#174; <br />  <br />               mbio.asm.org                                                                                 November/December 2011 Volume 2 Issue 6 e00238-11 <br />                                                                                                              Unbalanced Charge Distribution in YidC Substrates <br />  <br />  <br />  <br />  <br /> FIG 4 The distribution of positively charged residues is sufficient to determine the dependence of YaiZ on YidC for membrane insertion. (A) Topological <br /> illustrations of the cytoplasmic membrane protein YaiZ (22) and variants with altered charge balances (see Fig. 3 legend). (B) Subcellular localization of <br /> YaiZ-GFP variants in the presence or absence of YidC. (C) Differential fractionation of YaiZ-GFP variants into Triton X-100-soluble and -insoluble membrane <br /> fractions following synthesis in the presence or absence of YidC (see Fig. 2 legend and Materials and Methods). Images are representative. <br />  <br />  <br />  <br /> YidC for membrane insertion. Substituting alanines for positively              YidC, consistent with YidC-dependent defects in membrane in- <br /> charged residues in the cytoplasm, we created variants of YaiZ that            sertion. Thus, the introduction of unbalancing mutations ap- <br /> were unbalanced in one or both transmembrane segments                          peared to be sufficient to promote a dependence on YidC for <br /> (Fig. 4A). Each construct was integrated into the chromosome. In               membrane insertion. The YaiZ variant possessing two unbalanced <br /> contrast to the balanced wild-type protein, all unbalanced variants            transmembrane segments, and to a lesser extent the N-terminal <br /> of YaiZ showed a dramatic reduction in membrane localization                   unbalanced variant, also showed solubility defects in the presence <br /> (Fig. 4B) and Triton X-100 solubility (Fig. 4C) in the absence of              of YidC (Fig. 4C), suggesting that YidC was limited in its capacity <br />  <br />  <br /> November/December 2011 Volume 2 Issue 6 e00238-11                                                                                         &#194;&#174; <br />  <br />                                                                                                                                               mbio.asm.org 7 <br />  Gray et al. <br />  <br />  <br />  <br /> to promote stable membrane insertion of these more severely un-         were enriched in either our screen input pool or hits, suggesting <br /> balanced variants.                                                      that the His tag was not a significant factor in YidC dependence. <br />     For YaiZ and its variants, protease accessibility was uninforma-        Membrane proteins that were predicted to contain charge- <br /> tive due to the absence of cleavage products distinct from those        unbalanced transmembrane segments were significantly more <br /> observed following cell lysis. As determined by C-terminal LacZ         likely to depend on YidC for proper membrane insertion (67.6% <br /> and PhoA fusions, YaiZWT and the partially unbalanced variant           of unbalanced versus 12.1% of balanced membrane proteins) and <br /> YaiZKIRR6-8AIAA maintained C-in topology, whereas the unbal-            constituted 33% (23 of 69) of the YidC-dependent membrane <br /> anced variant YaiZKIRR6-8AIAA/RRR60-62AAA and to a lesser extent        proteins that we identified overall. Correcting the charge distribu- <br /> the partially unbalanced variant YaiZRRR60-62AAA displayed dual         tion across the unbalanced transmembrane segment of CrcB ab- <br /> topology (see Fig. S6 in the supplemental material), suggesting         rogated dependency on YidC for proper membrane insertion, in- <br /> that the RRR60-62AAA <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>mutation</span> contributed to both YidC de-              dicating that an unbalanced distribution of positively charged <br /> pendence and the disruption of normal YaiZ topology.                    residues was sufficient to explain its YidC dependence. Moreover, <br />                                                                         introduction of unbalancing mutations into the balanced mem- <br /> DISCUSSION                                                              brane protein YaiZ appeared to render the protein YidC depen- <br />                                                                         dent. Thus, an unbalanced distribution of positively charged res- <br /> YidC is required for the membrane insertion of certain cytoplas-        idues can act as a necessary and sufficient determinant of YidC <br /> mic membrane proteins in E. coli and other bacteria. Our results        dependence. <br /> indicate that while YidC is dispensable for a majority of cytoplas-         Positively charged residues act as strong determinants of mem- <br /> mic membrane proteins in E. coli, it is required to facilitate proper   brane protein topology (the &#226;&#8364;&#339;positive inside rule&#226;&#8364;&#65533;). Membrane <br /> insertion of a sizeable subset. We identified 69 cytoplasmic mem-       proteins that cannot be inserted into the membrane in an orien- <br /> brane proteins (16.6% of 415 examined) whose membrane inser-            tation such that all predicted transmembrane segments remain <br /> tion and/or folding is severely impaired in the absence of YidC.        both in the membrane and charge-balanced have been referred to <br />     Consistent with a substantial percentage of membrane pro-           as &#226;&#8364;&#339;topologically frustrated&#226;&#8364;&#65533; (40, 41). Most of the unbalanced <br /> teins inserting in a YidC-dependent manner, in recent studies           transmembrane proteins we identified as YidC dependent fall into <br /> utilizing isotope labeling (19) or two-dimensional (2D) blue            this category. While not directly examined here, in the absence of <br /> native/SDS-PAGE (20) and mass spectroscopy, sizeable subsets of         YidC, some unbalanced membrane proteins may adopt an in- <br /> examinable cytoplasmic membrane proteins (38/120 [19] and               verted topology, where it is more favorable. While the positive <br /> 20/44 [20], respectively) have exhibited significantly reduced          inside rule has been recognized for over two decades, there have <br /> abundance in the absence versus presence of YidC (19). In addi-         been few insights into the mechanism that underlies the rule or <br /> tion, analysis of a subset of membrane proteins of less than            into the mechanism by which certain proteins are able to possess a <br /> 50 amino acids indicates that many are dependent on YidC for            topology that defies it. <br /> membrane insertion (39). Our approach permitted systematic                  Our data strongly suggest that the mechanism by which many <br /> analysis of a substantially larger fraction of the E. coli membrane     unbalanced transmembrane proteins are properly inserted into <br /> proteome and, importantly, more direct examination of YidC de-          and folded in the membrane depends on YidC. We propose a <br /> pendence for membrane insertion per se, through the combina-            model in which YidC protects the charge-unbalanced segments of <br /> tion and cross-validation of microscopic visualization of protein       these proteins from the influence of electrostatic and/or other <br /> distribution, detergent solubilization and protease accessibility       topogenic forces that would normally compel more positively <br /> analyses. There was partial but incomplete overlap in the proteins      charged extramembrane segments to reside in the cytoplasm <br /> shown by Price et al. (19) and Wickstr&#195;&#182;m et al. (20) to have altered    (Fig. 5), thereby allowing folding events that could not occur if <br /> abundance in the absence of YidC, as well as between these sets         sequential transmembrane segments were inserted directly from <br /> and the proteins identified as YidC dependent in our study (see         the Sec translocon into the lipid bilayer. We postulate that YidC <br /> Data Set S1 in the supplemental material). These differences likely     thus serves as a stabilizing environment that lies intermediate be- <br /> result from differences in approach and highlight the importance        tween the translocon channel and the lipid bilayer. Prior work <br /> and difficulty of defining YidC substrates.                             lends additional support to this model. YidC is physically associ- <br />     Unlike some YidC substrates that have been characterized,           ated with the Sec translocon (12, 42), the Sec translocon possesses <br /> most of the membrane proteins we identified as being YidC de-           a lateral gate that has been proposed to serve as an exit site for <br /> pendent are likely inserted via the canonical SRP/Sec-dependent         transmembrane segments (43), and a projection structure of <br /> pathway. Most are polytopic membrane proteins possessing three          YidC, when docked with that of the Sec translocon, shows the <br /> or more transmembrane segments (59 of 69 hits; see Data Set S1 in       potential positioning of YidC adjacent to this gate (44), which <br /> the supplemental material). Among the 10 that possess one or two        would ideally position YidC to carry out the functions we propose; <br /> transmembrane segments, half are large (&#226;&#172;&#381;300 residues in                nascent transmembrane segments of YidC substrates interact first <br /> length), and one possesses a long periplasmic segment (&#226;&#172;&#381;50 resi-        with SecY and then with YidC (45); and YidC can interact with <br /> dues). These traits are inconsistent with insertion via the Sec-        multiple transmembrane segments simultaneously (13) and pro- <br /> independent pathway, whose substrates are small (&#226;&#172;&#65533;150 residues)         mote proper folding (18). Although not examined in this paper, it <br /> and either single-span or bitopic membrane proteins with short          is possible that YidC plays a similar role in the insertion of bal- <br /> periplasmic loops (&#226;&#172;&#65533;20 residues) (5, 7, 15). All proteins we ana-       anced YidC-dependent membrane proteins, promoting proper <br /> lyzed carried an N-terminal His tag; although we did not directly       topology, folding, and membrane insertion under circumstances <br /> examine whether this affected N-terminal translocation across the       where insertion of sequential transmembrane segments directly <br /> cytoplasmic membrane, neither proteins with predicted                   into the lipid bilayer would not permit proper biogenesis. <br /> N-terminus-in (N-in) nor N-terminus-out (N-out) topologies                  Since most YidC-dependent membrane proteins that we iden- <br />  <br />  <br /> 8        &#194;&#174; <br />  <br />               mbio.asm.org                                                                November/December 2011 Volume 2 Issue 6 e00238-11 <br />                                                                                                                    Unbalanced Charge Distribution in YidC Substrates <br />  <br />  <br />  <br />                                                                                       potential determinant topological features that we examined were <br />                                                                                       enriched among our screen hits. Another feature of cytoplasmic <br />                                                                                       membrane proteins that has been shown to correlate with reduced <br />                                                                                       abundance in the absence of YidC is soluble domains shorter than <br />                                                                                       100 residues (20); this parameter was not included in our analyses. <br />                                                                                       It is likely that in most cases, YidC dependence is a complex trait or <br />                                                                                       is determined by features that we did not examine or that are less <br />                                                                                       well defined. It should also be noted that some of the screen hits <br />                                                                                       that were classified as balanced membrane proteins may actually <br />                                                                                       be unbalanced, since the empirical bias against unbalanced trans- <br />                                                                                       membrane segments is likely to influence the topology predictions <br />                                                                                       of Phobius and other hidden Markov model algorithms (22). <br />                                                                                            Our identification of a large set of YidC-dependent membrane <br />                                                                                       proteins in E. coli greatly improves our understanding of which <br />                                                                                       and what proportion of cytoplasmic membrane proteins in E. coli <br />                                                                                       require YidC for proper membrane insertion. Given their high <br />                                                                                       degree of conservation and, at least in certain cases, functional <br />                                                                                       complementarity (9, 10), it is likely that YidC homologues in <br />                                                                                       other organisms, mitochondria, and chloroplasts play similar <br />                                                                                       roles and function by similar mechanisms in promoting proper <br />                                                                                       membrane protein biogenesis. <br />  <br />                                                                                       MATERIALS AND METHODS <br />                                                                                       Bacterial strains and plasmids. Bacterial strains and plasmids used in this <br />                                                                                       study are listed in Table S2 in the supplemental material. <br />                                                                                           Genetic methods. Site-directed mutagenesis was performed by two- <br />                                                                                       step overlap extension PCR. Integration of ASKA library clones into the <br />                                                                                       chromosome was performed using a strain generated to contain homol- <br />                                                                                       ogy to the plasmids at the attphi80 phage attachment site. Additional <br />                                                                                       genetic methods are described in the supplemental material. <br />                                                                                           Fluorescence microscopy screen. In 96-well plate format, cells were <br />                                                                                       grown to exponential phase, washed, and resuspended in medium con- <br />                                                                                       taining arabinose (0.2%) or not containing arabinose; growth was contin- <br />                                                                                       ued for 3 h to allow depletion of YidC. Synthesis of GFP-tagged mem- <br />                                                                                       brane proteins was induced by the addition of IPTG to 100 &#226;&#65533;&#174;M, and <br />                                                                                       growth was continued for an additional 30 min. The cultures were trans- <br />                                                                                       ferred to glass-bottom 96-well plates, and bacteria were brought to the <br />                                                                                       bottom of the wells by gentle centrifugation. Live-cell microscopy and <br />                                                                                       imaging were performed using a Nikon TE300 microscope. <br />                                                                                           Protease accessibility assay. Exponential-phase bacteria were grown <br />                                                                                       in the presence or absence of arabinose for 90 to 180 min, so as to maintain <br />                                                                                       or deplete YidC. Synthesis of GFP-tagged membrane proteins was in- <br />                                                                                       duced by the addition of IPTG to 100 &#226;&#65533;&#174;M, and growth was continued for <br />                                                                                       an additional 30 min. Spheroplasts were generated by cold osmotic shock <br />                                                                                       and treatment with 3 mM EDTA and 40 &#226;&#65533;&#174;g/ml lysozyme, and then were <br /> FIG 5 Model for YidC-dependent insertion of membrane proteins contain-                treated with 100 to 200 &#226;&#65533;&#174;g/ml proteinase K for 1 h at 4&#194;&#176;C, followed by <br /> ing unbalanced transmembrane segments. Transmembrane segments (TMs) <br />                                                                                       protease quenching with 2 mM phenylmethylsulfonyl fluoride (PMSF) <br /> of nascent cytoplasmic membrane proteins targeted to the Sec translocon are <br /> sequentially inserted into the translocon and then partitioned into the lipid         and trichloroacetic acid (TCA) precipitation of proteins. Additional de- <br /> bilayer via a lateral gate. YidC, docked at the lateral gate, interacts with trans-   tails are provided in the supplemental material. <br /> membrane segments as they exit the translocon, allowing the stabilization of              Membrane fractionation. Membranes and insoluble material were <br /> unbalanced transmembrane segments and folding of the protein with correct             isolated from crude lysates by centrifugation. Cytoplasmic membrane <br /> topology into an intrinsically stable conformation, which is then released into       proteins were recovered by solubilization in 1% Triton X-100. Additional <br /> the lipid bilayer. Balanced transmembrane segments (green) and charge-                details are provided in the supplemental material. <br /> unbalanced transmembrane segments (red) are shown. Positively charged res- <br /> idues within the flanking extramembrane domain are indicated by plus signs. <br />                                                                               <br /> tified do not or are not predicted to contain unbalanced trans-               <br /> membrane segments, other features must also contribute to YidC                          <br /> dependence. A recent study demonstrated that negatively charged                <br /> residues within transmembrane segments can act as such a deter-                     <br /> minant (46). However, neither this feature nor any of the other                    <br /> </body></html>