<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>           The REB1 Site Is an Essential Component of a Terminator <br />               for RNA Polymerase I in Saccharomyces cerevisiae <br />  <br />               We have identified a terminator for transcription by RNA polymerase I in the genes coding for rRNA of the <br />            yeast Saccharomyces cerevisiae. The terminator is located 108 bp downstream of the 3&apos; end of the mature 25S <br />            rRNA and shares several characteristics with previously studied polymerase I terminators in the vertebrates. <br />            For example, the yeast terminator is orientation dependent, is inhibited by its own sequence, and forms RNA <br />            3&apos; ends 17 2 bp upstream of an essential protein binding site. The recognition sequence for binding of the <br />            previously cloned REB1 protein (Q. Ju, B. E. Morrow, and J. R. Warner, Mol. Cell. Biol. 10:5226-5234, 1990) <br />            is an essential component of the terminator. In addition, the efficiency of termination depends upon sequence <br />            context extending at least 12 bp upstream of the REB1 site. <br />  <br />    Studies with both mouse and frog systems have shown                  includes the binding site for REB1, an essential yeast protein <br /> that termination of transcription by RNA polymerase I                   that has previously been cloned (11, 23, 24). <br /> requires the binding of a termination protein <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> a specific <br /> sequence in the DNA template. In mice, the protein has been <br /> called TTF1, and its specific binding site is an 18-bp-long                            MATERIALS AND METHODS <br /> element called the Sal box (10) (the element fortuitously                  Plasmids. The yeast rRNA genes (rDNA) used in this <br /> contains an SalI restriction site). In frogs, the binding               study were derived originally <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> an SmaI repeat <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> that <br /> protein has been christened Rib2 (21) and it binds to the site          was cloned as pBD4 (1). <br /> GACTTGCNC (19). In both systems, the terminator se-                        A termination test vector was constructed by arranging in <br /> quence will only cause termination when situated in the                 serial order a ribosomal gene promoter, a polylinker (for <br /> normal orientation and ceases to function when reversed.                insertion of sequences to be tested for termination activity), <br /> Partially purified TllF1 stimulates termination when added              and a series of unique restriction sites for linearizing the <br /> to a crude extract, and termination is abolished when an                template. This construct is diagrammed in Fig. 1A and was <br /> excess of an oligonucleotide containing the Sal box sequence            made as follows. A 1,078-bp fragment of rDNA (extending <br /> is added to the reaction. Likewise, partially purified Rib2             from an SmaI site 220 bp upstream of transcription initiation <br /> causes specific DNase I footprints over the terminator                  to an XbaI site 858 bp downstream of initiation) was inserted <br /> element, and both footprinting and termination are abolished            into the polylinker of pBluescript SK+ (Stratagene). The <br /> by competition with oligonucleotides containing the termi-              XbaI site, as well as an NdeI site in the rDNA portion, was <br /> nator sequence. Further progress in understanding the                   then destroyed by cutting, filling in, and religation. The <br /> mechanism of termination has been hampered by the inabil-               rDNA fragment was then excised by cutting with SmaI and <br /> ity so far to completely purify either Tl F1 or Rib2 or to              cutting at an SstI site in the pBluescript polylinker, and the <br /> clone the relevant cDNAs coding for these proteins.                     fragment was inserted into pUC18. (The pUC18 recipient <br />    In the hope that we could circumvent the problems of the             was cut with EcoRI, the end was filled in to blunt it, and then <br /> vertebrate systems, we have begun to study termination by               it was recut with SstI; insertion of the rDNA fragment <br /> RNA polymerase I in the yeast Saccharomyces cerevisiae.                 destroyed the SmaI site upstream of the rDNA promoter.) <br /> The yeasts offer at least two potential advantages over the             As shown in Fig. 1, the final construct has a polylinker <br /> vertebrates: first, it is easier to obtain the large amounts of         located about 900 bp downstream of the transcription initi- <br /> extract needed for biochemical purification, and second, the            ation site. <br /> possibility of using the well-developed genetics of yeasts to              Fragments of the intergenic spacer were tested for 3&apos;-end- <br /> supplement or replace biochemical approaches exists. The                forming ability by insertion into various restriction sites <br /> recent development of yeast extracts capable of supporting              within the polylinker. The StuI-HpaI (positions -245 to <br /> accurate initiation by RNA polymerase I in vitro (20, 27, 28)           +309), AccI-HpaI (positions -36 to +309), EcoRI-HpaI <br /> also offers the opportunity to develop in vitro assays for both         (positions +2 to +309), EcoRI-PvuII (positions +2 to +871), <br /> DNA and protein elements involved in termination.                       and EcoRV-SmaI (positions +1955 to +2162) fragments <br />    In this paper, we show that the yeast whole-cell extract             were all blunt ended and were inserted in either orientation <br /> developed by Schultz et al. (28) is capable of supporting               into the SmaI site of the polylinker (Fig. 1A, position +897 <br /> RNA 3&apos;-end formation of RNA polymerase I transcripts.                   of the termination test vector). Nucleotides are numbered <br /> One of these sites has features similar to the previously               according to the system used in Fig. 1B and 2A, which <br /> characterized vertebrate polymerase I terminators. Further-             reflects the actual sequences inserted into the test vector. <br /> more, fine-scale mutagenesis indicates that the terminator <br />                                                                         Position +1 designates the G residue in the EcoRI site <br />                                                                         adjacent to the REB1 protein binding site. <br />                                                                            Transcription assays. Transcription extracts were pre- <br />                                                                         pared (either from the multiply protease-deficient strain <br />   * <br />       Corresponding author.                                             BJ2168 or from a strain containing the rpo4l <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>) as <br />                                                                   649 <br />  650        LANG AND REEDER                                                                                                                                                                                                                 MOL. CELL. BIOL. <br />  <br />                                   Test Site                                             B. <br />  A.                               (polylirnker) <br />                          Noat Asp718 SmiaI Barnl-11 XbZa                                      .::d <br />                                                                                         -&apos;e n:l                                             LO -&apos;                                                                             :A <br />                          8E9 893      897 S02       S08 <br />                                                                                              :GT T   T -T A    r7 <br />                                                                                                                                               C ._ <br />                                                                                                                     T C -T T C TAAG T 2;GG T Ar.;:   &apos; <br />                                                                                                                                                          ` : -G.r- :-&quot;:.R::&apos;&apos;-:A:.: -.&apos;:   A ::;&apos;.    ............... <br />                                                                                                                                                                                                          &quot;I ,i:  .^ <br />                                                                                                                                                                                                                                 &apos;-4 &apos;:&apos; -&apos;:&apos; <br />                                                                                                                                                                                                                         ,................ <br />  <br />       (Sm&amp;nI)                                   NdeI              S-pI    ScaI <br />        -220     +I                               11503            1I06    1890          EC    RT <br />  <br />  <br />                                                                                                                        REB: <br />                                        _   -1 <br />                                             -   -1.      _.- <br />               o -n3t                                                             -.-1 <br />                                                                                                                                                                                                                               :A:A~~~~~~~~~~~c2c:A:-(x;L~~~ <br />                                                                                                                                                                                                                                                          .. ^   :A(5A <br />          p.rorr,. oe-r     5&apos;end 35S <br />  <br />  <br />  <br />  <br />                                                                                         TOC CC        zA      TT:    :TG CT     T C.- <br />                                                                                                                               TCA  A <br />                                                                                                                                   G-    -t <br />                                                                                                                                       ..CCAA <br />  <br />  <br />                                                                                                       Hpa T <br />                                                                                         c:&quot;TA TAG :LTAA&apos;; <br />  <br />  <br />  <br />                                                 wt           rpo 4                                                                FIG. 1. Specific initiation by RNA polymerase I on a linear <br />                                                                                                                                template. (A) Structure of the termination assay template. (B) <br />                                            12        3445 6                                                                    Sequence of the intergenic spacer from the 3&apos; end of 25S rRNA to <br />                                                                                                                                the HpaI site. We designate the first G residue of the EcoRI site as <br />                                                                                                                                + 1. The binding site for REB2 and the consensus site for REB1 are <br />                                                                                                                                underlined. (C) Transcription of linear templates showing specific <br />                                                                                                                                initiation. Templates were linearized with either Ndel (lanes 1 and <br />                                                                           .: <br />                                                                            r <br />                                                                                                                                4), SspI (lanes 2 and 5), or ScaI (lanes 3 and 6). Extracts were made <br />                                                                                                                                from a wild-type strain (BJ2168) or from a strain deficient in <br />                                                                                                                                mitochondrial polymerase (rpo4l). Sizes of the runoff transcripts <br />                                                                                                                                were estimated by using coelectrophoresed DNA markers. <br />  <br />  <br />                                                 -)       r <br />                                                          - <br />  <br />  <br />  <br />                                                                                                                                Tris-HCl (pH 8.0), 50 mM KCI, 1 mM EDTA, 0.5 pug of <br />                                                                                                                                poly(dI-dC), 5% glycerol, 10 ng (8,000 cpm) of oligonucle- <br />                                                                                                                                otide probe, and 1 ,ul of recombinant REB1 protein in a final <br />                                                                                                                                volume of 10 ,ul. Samples were incubated on ice for 20 min, <br />                                                      U) <br />                                                                                                                                loaded, and electrophoresed on 5% polyacrylamide gels <br />                                                                           l,&gt;lfjr <br />                                                                                                                                (80:1, acrylamide to bis) in 20 mM Tris-HCl (pH 8.0)-i mM <br />                                                                                                                                EDTA. The buffer was changed prior to loading and once <br />                                                                                                                                after 1 h. <br />                                                      HH        Hi    Hi <br />  <br />  <br />                                             a:} tn           D)      tJ <br />  <br />  <br />                                                                                                                                                                                                      RESULTS <br />                                                                                                                                  Establishment of an in vitro termination assay for RNA <br />                                                                                                                                polymerase I. A template containing a yeast ribosomal gene <br /> described previously (28), except that cells were ground the                                                                   promoter with convenient unique restriction sites located <br /> minimum amount to achieve 90% breakage. More extensive                                                                         downstream was constructed (Fig. 1A). Aliquots of this <br /> grinding resulted in higher background in the runoff assay.                                                                    template were cut at either the NdeI, SspI, or ScaI site and <br /> Protein concentrations in the extracts were in the range of 15                                                                 transcribed in vitro, and labeled runoff RNA was subjected <br /> to 25 mg/ml as assayed by the Bradford reaction (2).                                                                           to gel electrophoresis. As shown in Fig. 1C, RNA was <br />   Templates for runoff transcription were linearized at the                                                                    generated in the various reactions to the correct <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> to <br /> NdeI site unless stated otherwise. Typical reactions were                                                                      extend from the initiation site at the gene promoter to each <br /> run at room temperature in a volume of 40 ,ul containing 20                                                                    unique restriction site. We have previously shown that this <br /> mM HEPES (N-2-hydroxyethylpiperazine-N&apos;-2-ethanesul-                                                                           extract supports specific initiation by RNA polymerase I on <br /> fonic acid)-KOH (pH 7.9), 50 mM KCl, 5 mM EGTA                                                                                 circular templates (5, 28). The data in Fig. 1C support the <br /> [ethylene glycol-bis(,B-aminoethyl ether)-N,N,N&apos;,N&apos;-tetra-                                                                     conclusion that specific initiation at the ribosomal gene <br /> acetic acid], 0.05 mM EDTA, 2.5 mM dithiothreitol, 10 ,ug of                                                                   promoter is also occurring on linear templates. <br /> a-amanitin per ml, 10% glycerol, 0.5 mM (each) ATP, GTP,                                                                         All of the transcription reactions described in this paper <br /> and UTP, 0.1 mM CTP, 10 ,uCi of a-32P-CTP (800 Ci/mmol),                                                                       contained enough oa-amanitin to eliminate transcription by <br /> 10 pl of extract, and 25 ng of template. Reactions were                                                                        RNA polymerase II. Previous analysis of the DNA fragment <br /> stopped by adding 250 ptl 7 M urea-100 mM LiCl-0.5%                                                                            used as a promoter cassette (5, 28) showed that it contains no <br /> sodium dodecyl sulfate-10 mM EDTA-10 mM Tris-HCl (pH                                                                           promoters for RNA polymerase III. Furthermore, the sig- <br /> 8.0) and were extracted twice with phenol-chloroform.                                                                          nals are insensitive to Tagetin (Epicentre Technologies, <br /> tRNA carrier (10 p,g) was added to each reaction, and                                                                          Madison, Wis.), a specific inhibitor of RNA polymerase III <br /> nucleic acids were precipitated with 2.5 volumes of ethanol.                                                                   (data not shown). The only other possible interfering RNA <br /> Pellets were resuspended in a mixture of 5 pul of 98%                                                                          polymerase activity is mitochondrial polymerase. A cryptic <br /> formamide, 20 mM EDTA, and 0.025% (wt/vol) (each)                                                                              promoter for mitochondrial RNA polymerase is located at <br /> xylene cyanole and bromphenol blue, boiled for 3 min, and                                                                      about position -75 relative to the initiation site for RNA <br /> loaded onto 3% polyacrylamide gels containing 7 M urea.                                                                        polymerase I (5, 27), and at the level of resolution of the gel <br />    Gel retardation assays. Each reaction contained 20 mM                                                                       shown in Fig. 1C, RNA initiated at position -75 might <br />  <br />   FIG. 2. Survey of the intergenic spacer for RNA 3&apos;-end-forming sites. (A) Map of the intergenic spacer showing location of fragments that <br /> were  tested. (B) Survey of 3&apos;-end-forming sites between the StuI site (position -248) and the HpaI site (position +307). For lane 1, the <br /> StuI-HpaI fragment was inserted into the SmaI site of the test vector and transcribed after linearization further downstream at the NdeI site. <br /> For lane 2, the AccI-HpaI fragment was inserted into the test site. For lane 3, the EcoRI-HpaI fragment was tested. The locations of runoff <br /> RNA bands attributable to the 3&apos; end of the 25S rRNA, 3&apos;-end formation at sites I, II, and III, and full-length runoff are indicated. (C) <br /> Orientation dependence of 3&apos;-end-forming sites. The EcoRI-HpaI fragmnent was inserted into the test site in either the forward (lane 1) or <br /> reverse (lane 2) orientation and tested in runoff transcription. Likewise, the EcoRV-SmaI fragment was tested in either the forward (lane 3) <br /> or reverse (lane 4) orientation, and the EcoRI-PvuII fragment was tested in either the forward (lane 5) or reverse (lane 6) orientation. The <br /> locations of runoff RNA bands attributable to 3&apos;-end formation at sites I, II, III, and IV are indicated. <br />  <br />  <br /> possibly be confused with RNA initiated at position + 1.                                            tional, shorter runoff products indicative of RNA 3&apos;-end <br /> However, mitochondrial polymerase makes little, if any,                                             formation. <br /> contribution to the runoff signal seen in this assay system.                                           Survey of the ribosomal gene spacer for RNA 3&apos;-end- <br /> Figure 1C, lanes 1 through 3, shows the runoff signal                                               forming sites. We began by inserting into the test site a <br /> generated in a whole-cell extract made from wild-type yeast                                         550-bp StuI-HpaI fragment that extends from within the 25S <br /> cells. Lanes 4 through 6 show transcripts generated from the                                        rRNA coding region to a point downstream of the ribosomal <br /> same truncated templates but utilizing an extract from an                                           gene enhancer. (The sequence of the relevant portion of this <br /> rpo4l strain which lacks mitochondrial RNA polymerase                                               fragment is shown in Fig. 1B; the location of this fragment <br /> activity (9). The relative size and intensity of the runoff                                         within the spacer is shown in Fig. 2A.) As shown in Fig. 2B, <br /> RNAs are essentially identical in both cases. We conclude,                                          lane 1, the StuI-HpaI fragment caused the appearance of <br /> therefore, that the RNA signals generated in this assay                                             four shorter RNAs in the runoff assay. The shortest RNA <br /> system are attributable to specific initiation by RNA poly-                                         has a length suggesting that its 3&apos; end corresponds to the 3&apos; <br /> merase   I.                                                                                         end of the 25S rRNA itself. Other workers have presented <br />    To assay for RNA 3&apos;-end-forming elements, a polylinker                                           evidence that this 3&apos; end is formed by processing (26, 32), <br /> was inserted into the template, thus locating a unique SmaI                                         and our own experiments agree with that conclusion (Fig. 3). <br /> site 897 bp downstream from the position + 1 initiation site.                                       The 3&apos; ends of the other three RNAs are formed at sites I, II, <br /> Various DNA fragments were then inserted into this SmaI                                             and III, which we have numbered in order of increasing <br /> site to see whether they were capable of generating addi-                                           distance from the ribosomal gene promoter. <br />  652     LANG AND REEDER                                                                                                  MOL. CELL. BIOL. <br />  <br />                         1 2 3 4 5 6                                      this survey are shown in Fig. 2C. Comparing lanes 1 and 2, <br />                                                                          we observed that sites II and III function in either orienta- <br />                                                                          tion (although site II function decreases somewhat when <br />            run off--                                                     orientation is reversed). However, the 3&apos;-end-forming ability <br />                                                                          of site I is eliminated by reversing the sequence. Lanes 5 and <br />                              V           ^Z--. run off                   6 show essentially the same result when a longer test <br />                  IT-                                                     fragment is used. (However, in lane 6, site II is not clearly <br />                   I                                                      resolved from the band attributable to site III.) Comparison <br />             3&apos;25S-               3;     4 ,--i                           of lanes 3 and 4 shows that site IV also functions in either <br />                                                                          orientation. Since only site I shows complete orientation <br />                                                                          dependence, this site becomes our prime candidate for being <br />                                                                          a terminator of RNA polymerase I transcription. <br />                                                                             RNA processing activity is not detected at site I, II, or III. <br />                                                                          Figure 3, lane 2, shows a typical experiment in which the <br />                                                                          StuI-HpaI fragment was inserted at the test site and tran- <br />    FIG. 3. Lack of behavior by sites I, II, and III as RNA process-      scription was allowed to proceed for 20 min. 3&apos; ends attrib- <br /> ing sites. In lanes 1 to 3, the test vector contained the StuI-HpaI      utable to the end of the 25S rRNA as well as sites I, II, and <br /> fragment. In lanes 4 to 6, the EcoRI-HpaI fragment was tested. For       III are clearly visible. Figure 3, lane 3, shows a parallel <br /> lanes 2 and 5, transcription ran for 20 min before isolation of RNA.     reaction in which transcription was also allowed to proceed <br /> For lanes 3 and 6, transcription ran for 20 min, actinomycin D was <br /> added to 20 ,ug/ml to stop transcription, and incubation was contin-     for 20 min, at which time actinomycin D was added to stop <br /> ued for an additional 20 min before RNA was isolated. For lanes 1        transcription but incubation was continued for an additional <br /> and 4, actinomycin D was added to 20 pLg/ml before transcription         20-min chase. During the chase, formation of the 25S 3&apos; end <br /> was initiated with triphosphates. Note that accumulation of the site     continued to occur (band intensity increased 60%, as mea- <br /> I, II, and III bands and runoff bands require continued transcription,   sured by densitometry). In contrast, the intensity of all of the <br /> while accumulation of the 3&apos; end of the 25S continues after tran-        other bands decreased by about 50%. Yip and Holland (32) <br /> scription is stopped.                                                    have previously shown that an RNA transcribed by SP6 <br />                                                                          polymerase can be processed in vitro to yield the 3&apos; end of <br />                                                                          the 25S rRNA, thus providing strong evidence that this <br />    The processing site at the 3&apos; end of the 25S rRNA is so               location harbors a processing site and is not a terminator of <br /> active (both in vivo and in vitro) that it is easier to study            transcription. The chase experiment shown in Fig. 3, lanes 2 <br /> other 3&apos;-end-forming sites in the spacer if the 3&apos; end of the            and 3, is in good agreement with that conclusion. <br /> 25S is first eliminated. This was accomplished by inserting                 To examine the behavior of sites I, II, and III more <br /> the AccI-HpaI fragment into the test site. As shown in Fig.              closely, we inserted the EcoRI-HpaI fragment into the test <br /> 2B, lane 2, once the 3&apos; end of the 25S is eliminated, site I             site and performed the same actinomycin D chase experi- <br /> becomes the predominant 3&apos;-end-forming site, but sites II                ment. After 20 min of transcription, 3&apos; ends attributable to <br /> and III are still clearly detected. Under typical in vitro assay         sites I, II, and III are clearly visible (Fig. 3, lane 5), and after <br /> conditions, about 70% of the total transcripts end at site I             20 min of chase in the absence of transcription, their <br /> when the AccI-HpaI fragment is assayed. Figure 2B, lane 3,               intensities have decreased by about 20% (compare lanes 5 <br /> shows the result obtained when the EcoRI-HpaI fragment is                and 6). We conclude that sites I, II, and III require continued <br /> inserted into the test site. Shorter RNAs attributable to sites          RNA transcription in order to form 3&apos; ends. In this respect, <br /> I, II, and III appear as seen previously with the AccI-HpaI              they all have the behavior expected of termination sites, in <br /> fragment (compare lane 2 with lane 3), but in the case of the            contrast to the 3&apos; end of the 25S rRNA, which has the <br /> shorter fragment, only about 25% of the total transcripts are            behavior expected of a processing site. <br /> attributable to site I.                                                     Site I is sensitive to competition with its own DNA sequence. <br />    In Fig. 2C, the survey is extended to other parts of the              Another distinguishing characteristic of the vertebrate poly- <br /> intergenic spacer. Lane 1 shows a repeated experiment in                 merase I terminators is that they require the binding of a <br /> which the EcoRI-HpaI fragment was inserted into the test                 sequence-specific protein in order to cause termination (10, <br /> site. Lane 5 shows the results of inserting a larger EcoRI-              21). Because of this, termination in both mouse and frog <br /> PvuII fragment. Again, only RNAs attributable to sites I, II,            systems is highly sensitive to competition caused by adding <br /> and III appear. Finally, lane 3 shows the result of inserting            an excess of a DNA fragment containing the terminator <br /> an EcoRV-SmaI fragment into the test site. With this frag-               sequence. We have tested yeast sites I, II, and III to see <br /> ment, a new truncated RNA appears, indicating another                    whether they have a similar sensitivity to competition. As <br /> RNA 3&apos;-end-forming location, which we have numbered site                 shown in Fig. 4A, addition of increasing amounts of the <br /> IV.                                                                      EcoRI-HpaI fragment (which contains sites I, II, and III) <br />    The relative locations of all of the 3&apos;-end-forming sites we          causes a rapid loss of 3&apos;-end formation at site I. Site II is also <br /> have identified in this survey are summarized at the bottom              sensitive to competition, but site III is completely resistant <br /> of Fig. 2A. This survey omits 1,100 bp of the intergenic                 within this range of competitor. <br /> spacer (from the PvuII site to the EcoRV site). Apparently                  The sensitivity of site I to competition is examined more <br /> some element in this region is unstable in the vectors we are            closely in Fig. 4B. In this experiment, the competitor was a <br /> using, because we only recovered rearranged inserts (if any)             polymerized 30-bp double-stranded oligonucleotide (from <br /> in repeated attempts to subclone this region.                            positions 7 to 30 within the EcoRI-HpaI fragment shown in <br />    Only site I is orientation dependent. RNA polymerase I                Fig. 1B). 3&apos;-end formation at site I is specifically and rapidly <br /> terminators in both frogs and mice function only when                    inhibited by this oligonucleotide, while sites II and III are <br /> positioned in the forward orientation (10, 18). We therefore             unaffected. <br /> surveyed the yeast sites I, II, III, and IV to see whether any              We conclude that, among the RNA 3&apos;-end-forming sites <br /> of them had similar orientation dependence. The results of               we have identified within the intergenic spacer, site I is the <br />  <br />  <br />  <br />   FIG. 4. Sensitivity of sites I, II, and III to competition with their       FIG. 5. 5&apos; and 3&apos; deletion mapping of sites I, II, and III. (A) <br /> own sequence. Test vector (25 ng) containing the EcoRI-HpaI                Assay of deletions of the EcoRI-HpaI fragment. For lane 1, NotI <br /> fragment was added to each transcription reaction. (A) Competition         adaptors were added to the EcoRI-HpaI fragment, and it was <br /> with the   EcoRI-HpaI fragment. The following amounts of gel-              inserted into the NotI site at position 869 within the test vector <br /> purified restriction fragment were added to each reaction before           polylinker. The procedure for lanes 2, 3, and 4 was the same as that <br /> transcription began: none (lane 1), 50 ng (lane 2), 100 ng (lane 3), 200   for lane 1 but with a novel XbaI site introduced at positions 87, 144, <br /> ng (lane 4), and 400 ng (lane 5). (B) Competition with an oligonu-         and 185, respectively. For lanes 5 to 10, the XbaI sites were used to <br /> cleotide containing the REBi site. Oligonucleotide 7/30 (sequence          create and test various deletions of the EcoRI-HpaI fragment. <br /> shown in Fig. 6B) was polymerized, and an aliquot was added to             Coordinates of the sequences tested are as follows: 92 to 309 (lane <br /> each reaction before transcription began, as follows: no competitor        5), 147 to 309 (lane 6), 192 to 309 (lane 7), 1 to 87 (lane 8), 1 to 143 <br /> (lane 1), 0.5 ng (lane 2), 2 ng (lane 3), and 10 ng (lane 4).              (lane 9), and 1 to 185 (lane 10). (B) Assay of finer deletions made by <br />                                                                            ExoIII digestion. Lanes 1 to 4 show tests of 5&apos; deletions of the <br />                                                                            EcoRI-HpaI fragment. Coordinates of the sequences tested are as <br />                                                                            follows: 1 to 87 (lane 1), 1 to 137 (lane 2), 1 to 23 (lane 3), 1 to 16 (lane <br /> only one which passes all the tests we have applied to be a                4), 2 to 309 (lane 5), 2 to 288 (lane 6), 2 to 266 (lane 7), 2 to 194 (lane <br /> terminator. It requires continued transcription for 3&apos;-end                 8), and 15 to 309 (lane 9). <br /> formation, it is orientation dependent, and it is sensitive to <br /> competition. Although the precise molecular events occur- <br /> ring at site I remain to be determined, we will refer to it as a <br /> terminator in the rest of this paper.                                      assayed a finer set of deletions within the EcoRI-HpaI <br />    5&apos; and 3&apos; deletion mapping of 3&apos;-end-forming sites within the           fragment, as shown in Fig. SB. Deleting positions 24 to 309 <br /> EcoRI-HpaI fragment. The RNA 3&apos; ends formed by sites I,                    still allows 3&apos;-end formation at site I (lane 3), whereas <br /> II, and III map to the approximate positions shown at the                  removal of positions 17 to 309 (lane 4) and positions 1 to 14 <br /> bottom of Fig. 2A. We also wished to know the location of                  (lane 9) abolishes it. Deleting positions 289 to 309 still allows <br /> the DNA sequences responsible for forming these 3&apos; ends.                   3&apos;-end formation at site III (lane 6), but further removal of <br /> Accordingly, a series of mutants were made in which three                  sequence to position 267 abolishes it (lane 7). Site II is not <br /> novel XbaI sites were introduced at intervals across the                   affected by deletion of positions 195 to 309 (lane 8). <br /> EcoRI-HpaI fragment (the novel XbaI sites are designated                      The conclusions from these experiments are that se- <br /> ELS-1, -2, and -3, and their sequence locations are shown in               quences essential for site I map between positions 1 and 23, <br /> Fig. iB). As shown in Fig. 5A, lanes 2 to 4, introduction of               sequences for site II map between positions 106 and 194, and <br /> these novel Xbal sites did not cause any alteration in RNA                 sequences for site III map between positions 192 and 288. <br /> 3&apos;-end formation. The XbaI sites were then used as end                        At this level of resolution, each of the RNA 3&apos; ends formed <br /> points to produce a small series of 5&apos;- and 3&apos;-terminal                    by sites I, II, and III maps to the same general location, as do <br /> deletions of the EcoRI-Hpal fragment.                                      the DNA sequences that cause the 3&apos; ends. Site II and its 3&apos; <br />    Results from assay of the 5&apos; deletion mutants are shown in              end map near a region of sequence that is almost pure AT. <br /> Fig. 5A, lanes 5 through 7. Removing sequences from                        Site III maps between the HindIlI site and the HpaI site. Site <br /> positions 1 to 91 eliminates 3&apos;-end formation by site 1 (lane              I, the putative terminator, maps close to the EcoRI site at the <br /> 5). Further deletion to position 146 eliminates 3&apos;-end forma-              5&apos; end of the EcoRI-HpaI fragment. We will now examine <br /> tion at site II (lane 6). Site III remains even after 5&apos; deletion          this latter region in more detail. <br /> to position 191 (lane 7).                                                     The terminator (site I) contains the binding site for REB1. <br />    T&apos;he corresponding set of 3&apos; deletions is shown in Fig. 5,              DNase I footprinting and gel shift analyses have identified <br /> lanes 8 through 10. The deletion from positions 186 to 309                 binding sites for two proteins, the so-called REB1 and REB2 <br /> eliminates site III and greatly reduces site IIL Site I is                 sites, just downstream of the EcoRI restriction site (23, 24) <br /> unaffected by any of the 3&apos; deletions of sequences between                 (Fig. 1B). The REB1 site consists of 8 bp from positions 13 <br /> positions 88 and 309 (lanes 8 and 9).                                      to 20, and the REB2 site is located at positions 42 to 56. The <br />    To map the boundaries of sites I, II, and III further, we               REB1 protein has been cloned (11) and is likely to be <br />  654     LANG AND REEDER                                                                                                                            MOL. CELL. BIOL. <br />  <br />        A.           2 3 4 5 6 7 8 9 10          B. <br />                                                 Wi ldtypef:               TATG A rSC G G: A;AA A A A.-I TAP <br />                                                                 G A.ATT(rlC                                     Ar.i <br />                                                                                                                    A; A&apos;J &quot;:A&apos;1 T ATAe^ <br />                                                                                                                                       k A TAC G AT A GA &apos;.A- A &apos;&apos;-r|r <br />                                                                                                                                                      i&quot; <br />  <br />  <br />                                                                                            REBI                                        REE2 <br />                                                 07 /30 Oligo:   t t a aaTATGATCCGGG-&apos;AAAAA.&lt;: <br />                                                                                           -A A&apos;AG?iA <br />                                                                                               A      c AgA <br />              .1--   illw         !              35/62 Oligo: <br />                                                 10/38 Alice:               gGAfrCCGGGTAAA -A.A.  .A&apos;1 <br />                                                                                                        7:TA <br />                                                                                                         TA A&apos; TAT A:&apos;:AAATACO <br />                                                                                                     A TT&apos;TATATAq ..at-cCC- <br />                                                                                                                            T .CA:ATiA <br />                                                                                                                               A     a <br />                                                                                                                                   A c* <br />  <br />                                                 10/23 Olig.:               gGATCCGGGTAAAA..^^&gt;Aggatce <br />  <br />    FIG. 6. Termination activity of oligonucleotides containing either the REBM or REB2 binding site. Oligonucleotides 7/30 and 35/62 were <br /> inserted into the XbaI site of the test vector polylinker; oligonucleotides 10/38 and 10/23 were inserted into the BamHI site of the polylinker. <br /> The sequence of these oligonucleotides is shown in Fig. 7C. (A) Termination activity of the oligonucleotides. Lanes show oligonucleotides <br /> (and orientations) as follows: 1 and 10, assay vector without any insert; 2, oligonucleotide 7/30 (forward); 3, oligonucleotide 7/30 (reverse); <br /> 4, oligonucleotide 35/62 (forward); 5, oligonucleotide 35/62 (reverse); 6, oligonucleotide 10/38 (forward); 7, oligonucleotide 10/38 (reverse); 8, <br /> oligonucleotide 10/23 (forward); and 9, oligonucleotide 10/23 (reverse). (B) Sequences of the oligonucleotides used in panel A. Uppercase <br /> letters represent natural rDNA sequence; lowercase letters indicate linker sequence added for cloning purposes. REBM and REB2 binding <br /> sites are underlined in the wild-type sequence. <br />  <br />  <br />  <br /> identical to a protein that was independently purified and                    -216 to -225 relative to transcription initiation [24]). This <br /> called GRF2 (4). The REB2 site may be a subset of a                           site does not match the consensus binding site exactly; the <br /> sequence that binds the protein ABF1 (3). To test their                       purine residue at position 23 is mutated to a C. <br /> possible involvement in termination, we prepared oligonu-                        Each of the REBM mutants that was tested for termination <br /> cleotides which contained either the REB1 or REB2 site and                    activity in Fig. 7A was also tested for its ability to bind the <br /> cloned them into the test site. Results of the termination                    REB1 protein in a gel shift assay (Fig. 7B). The correlation <br /> assay are shown in Fig. 6A.                                                   between termination activity and the ability to bind REB1 <br />    The oligonucleotide containing the REB2 site showed no                     protein is complete. The behavior of mutant 21/22/23 is <br /> detectable termination (oligonucleotide 35/62) (Fig. 6A, lane <br /> 4). However, an oligonucleotide containing the REB1 site <br /> did exhibit termination activity (oligonucleotide 7/30, lane 2). <br /> This oligonucleotide contains 6 bp of sequence upstream of                                                 A         234 5678910 <br /> the CCGGGTAA core sequence which was identified by <br /> DNase I footprinting (23, 24). Other oligonucleotides con- <br /> taining the core sequence but which had only 3 bp of <br /> upstream sequence failed to cause termination (oligonucle- <br /> otides 10/38 and 10/23, lanes 6 and 8). Lane 3 shows that the <br /> REB1 oligonucleotide which did cause termination still <br /> shows the same orientation dependence that is characteristic <br /> of RNA polymerase I terminators. <br />    The experiments shown in Fig. 5 and 6 suggest that                                                 B. <br /> nucleotides between positions 7 and 23 are important for <br /> termination activity, a region which includes the REB1                                                         00               46*    bouno DNA <br /> consensus binding site. To examine this region in more <br /> detail, we made a series of mutants in which just two <br /> nucleotides at a time were mutated in stepwise fashion <br /> starting at one side of the consensus REB1 binding site and <br /> proceeding across to the other side. The sequences of these                                                    ILA                    - free DNA <br /> mutants are shown in Fig. 7C, and their ability to cause <br /> termination when inserted into the test site is shown in Fig. <br /> 7A. Beginning upstream of the REB1 site, the pairwise                          C. <br /> mutants at positions 7 to 8, 9 to 10, and 11 to 12 (designated <br /> 7/8, 9/10, and 11/12-lanes 2, 3, and 4, respectively) have no                  .4.: . !.!.y e <br />  <br /> effect on termination. All of these mutations are also outside <br /> of the REB1 binding site. Mutation 13/14, which changes the <br /> first nucleotide of the binding site, eliminates termination                          I: <br /> (lane 5). The same is true for other mutations within the site                        .. <br />  <br />  <br />  <br />  <br /> (mutations 15/16, 17/18, and 19/20-lanes 6, 7, and 8, respec- <br /> tively). So far, the correlation between mutations which                        FIG. 7. REBM site as an essential part of the polymerase I <br /> affect the REB1 site and mutations which eliminate termina-                   terminator. (A) Assay of REBM binding site mutants for termination <br /> tion is perfect. The apparent anomaly is the next cluster of                  activity. Lanes: 1 and 10, termination by wild-type sequences 1 to <br /> mutations (21/22/23), in which three A residues were all                      23; 2, mutant 7/8; 3, mutant 9/10; 4, mutant 11/12; 5, mutant 13/14; <br /> changed to C&apos;s. These changes are within the REB1 consen-                     6, mutant 15/16; 7, mutant 17/18; 8, mutant 19/20; and 9, mutant <br /> sus binding site (4), but they do not eliminate termination                   21/22/23. (B) Assay of REB1 binding site mutants for binding to the <br />                                                                               REBM protein. Lanes: 1, mutant 7/8; 2, mutant 9/10; 3, mutant 11/12; <br /> (lane 9). After making this cluster of mutations, however, we                 4, mutant 13/14; 5, mutant 15/16; 6, mutant 17/18; 7, mutant 19/20; <br /> realized that the three changes inadvertently altered the                     and 8, mutant 21/22/23. (C) Sequences of REBM site mutants. The <br /> sequence to that of a second REB1 binding site that is known                  sequence of the consensus REB1 site is underlined. Term, termina- <br /> to exist upstream of the ribosomal gene promoter (positions                   tion activity. -, absent; +, present; + +, strongly present. <br />  <br />  <br />    FIG. 8. Map of the 3&apos; end formed at site I 17 &#194;&#177; 2 bp upstream of the REB1 binding site. The length of the transcript specified by site I <br /> was  compared with the length of runoff RNA products generated by transcription of vectors truncated at known sites. Lane 1 shows <br /> 5&apos;-end-labeled DNA size standards (HpaI digest of pBR322). DNA size standards do not run precisely the same as RNA standards of the same <br /> sequence. These DNA standards are included only to illustrate the degree of resolution obtained in this region of the gel. Lane 2 shows RNA <br /> size standards generated when the assay vector (with no insert) is transcribed after linearization at either the BglIl or HindIII site. These two <br /> RNA standards have lengths of 123 and 256 nucleotides, corresponding to map positions -69 and +64, respectively. Lane 3, a third RNA <br /> size standard, was generated by deleting part of the assay vector (to shorten it) and linearizing at the Asp-718 site. This produces an RNA <br /> 191 nucleotides in length whose 3&apos; end corresponds to map position -2. Lane 4, an assay template containing the EcoRI-HpaI fragment at <br /> the test site, was modified by removing 706 bp of vector sequence between the polymerase I promoter and the test site. Upon transcription <br /> of this template, RNA generated by site I migrates slightly faster than the position -2 marker RNA. This places the 3&apos; end formed by site <br /> 1 17 &#194;&#177; 2 bp upstream of the REB1 consensus binding site. <br />  <br />  <br /> especially interesting. It has been shown that this particular                     dependent upon sequences upstream of the REBM site. This <br /> REB1 binding sequence has a higher level of affinity for the                       is illustrated in Fig. 9. Figure 9, lane 1, shows the result of <br /> REB1 protein (4) than wild-type sequence does. Our data                            inserting the AccI-HpaI fragment into the test site. This <br /> agree with this observation in that the 21/22/23 mutant                            fragment contains 49 bp of normal sequence upstream of the <br /> reproducibly shifts a larger fraction of the probe than does                       REBM consensus binding site and is the most efficient <br /> the wild-type sequence in gel shift assays (Fig. 7B, compare                       terminator fragment we have assayed (about 70% of the <br /> lane 3 with lane 8). In many assays, mutant 21/22/23 also is                       transcripts terminate at site I). In lane 2, a shorter EcoRI- <br /> a more efficient terminator than the wild-type sequence is,                        HpaI fragment was blunt ended, NotI linkers were attached, <br /> but this effect is not clearly seen in Fig. 7B. Thus, within this                  and the result was inserted into the test site. These manip- <br /> set of mutants, the correlation between sequences required                         ulations allowed retention of the first G of the EcoRI <br /> for REB1 binding and sequences required for termination by                         recognition site (13 bp upstream of the REB1 site). Termi- <br /> RNA polymerase I is perfect.                                                       nation occurs with this fragment, but it is clearly less <br />    We conclude that the REBM site is an essential component                        efficient than with the AccI-HpaI fragment shown in lane 1. <br /> of the yeast polymerase I terminator. This implies that either                     In lane 3, the EcoRI-HpaI fragment is assayed again, but this <br /> REBM itself or some unknown protein with identical DNA                             time it was blunt ended and was inserted directly into the <br /> binding specificity is essential for termination.                                  SmaI site of the test site. These manipulations result in the <br />    3&apos; ends formed at site I map about 17 bp upstream of the                        loss of the first G of the EcoRI recognition sequence, and <br /> REB1 site. An additional characteristic of both the mouse <br /> and frog polymerase I terminators is that the 3&apos; ends map <br /> about 11 to 15 bp upstream of the essential terminator <br /> protein binding site (15, 17). Finer mapping of the 3&apos; end                                                            1 2 345 <br /> specified by site I shows that the same is also true for the <br /> yeast terminator (Fig. 8). In this experiment, marker RNA <br /> molecules of precisely known length were generated by                                                    run off- <br /> RNA polymerase I runoff transcription from the assay <br /> template truncated at appropriate sites. The three marker                                                             U <br /> RNAs we used had lengths that would correspond to 3&apos;-end <br /> formation at positions -69 and +64 (lane 2) and -2 (lane 3). <br /> In lane 4, we show that RNA produced by 3&apos;-end formation <br /> at site I migrates slightly faster than the position -2 marker. <br /> This places the 3&apos; end specified by site I about 17 + 2 bp <br /> upstream of the boundary of the REBM site that is essential <br /> for its formation. Once again, the yeast terminator has <br /> characteristics similar to its vertebrate counterparts. Similar                      FIG. 9. Dependence of the efficiency of termination at site I upon <br /> experiments with less resolution located the 3&apos; ends formed                        sequences upstream of the REBM site. DNA fragments with various <br /> by site II and site III around positions 155 and 250, respec-                      amounts of natural sequence upstream of the REBM consensus <br /> tively (data not shown).                                                           binding site were assayed for the relative efficiency of termination. <br />                                                                                    Lanes are as follows: 1, assay of the AccI-HpaI fragment assayed <br />    Efficiency of termination at site I is influenced by sequences                  (49 bp of upstream sequence); 2, EcoRI-HpaI fragment with NotI <br /> upstream of the REB1 site. The mutagenesis shown in Fig. 7                         adaptors (13 bp upstream); 3, EcoRI-HpaI fragment blunt ended into <br /> demonstrates that the REB1 site is an essential component                          the SmaI site (12 bp upstream); 4, oligonucleotide 7/30 (7 bp <br /> of the terminator. However, the efficiency of termination is                       upstream); and 5, oligonucleotide 10/23 (4 bp upstream). <br />  656    LANG AND REEDER                                                                                          MOL. CELL. BIOL. <br />  <br /> there is a significant further reduction in termination effi-     chinery for RNA polymerase I, is it possible that some of the <br /> ciency. In lane 4, an oligonucleotide with only 7 bp upstream     other REB1 sites in the genome are part of terminators for <br /> of the REB1 site was tested and a further drop in termination     RNA polymerase II? Many of the REB1 sites that have been <br /> efficiency was observed. Finally, lane 5 shows that when an       identified are located between known genes, often just <br /> oligonucleotide is tested with only 4 bp upstream of the          upstream of the activating sequences for a nearby promoter <br /> REBM site, termination is not detectable. It will require         (4). Thus, they could conceivably function to prevent tran- <br /> further work to determine whether this region upstream of         scription from one gene running through and dislodging <br /> the REB1 site functions as the binding site for an additional     activator proteins from the control region of the downstream <br /> protein or influences the efficiency of termination via some      gene. REB1 sites located upstream of polymerase II activa- <br /> other mechanism.                                                  tor regions were all found to be in the reverse orientation <br />                                                                   compared with the REB1 site we have studied at site I. This <br />                         DISCUSSION                                indicates that if REB1 actually is involved in polymerase II <br />                                                                   termination, the orientation of the element is polymerase <br />    What is the function of the other REB1 binding sites in the    specific. <br /> yeast genome? The experiments reported in this paper show            In vivo searches for the terminator of RNA polymerase I <br /> that a protein with DNA binding characteristics indistin-         transcription. The 3&apos; end of the yeast rRNA coding region <br /> guishable from those of REBM is involved in termination of        has previously been studied in vivo in attempts to locate <br /> RNA polymerase I transcription in yeast. It will require          possible terminators of polymerase I transcription. For <br /> further experiments to definitively determine whether or not      example, Planta&apos;s group has studied transcription of riboso- <br /> REBM is the terminator protein. But, on the reasonable            mal minigenes introduced on multicopy plasmids and reports <br /> assumption that the terminator protein actually is REB1,          identification of five sites of 3&apos;-end formation (12, 30). The <br /> what is already known about this protein is worth reviewing.      first site, TO, corresponds to the 3&apos; end of the 25S rRNA and <br />    REB1 is probably identical to GRF2, a protein that was         was concluded to be an RNA processing site on the basis <br /> independently identified and purified by the Kornberg group <br /> (4, 8). A computer search has revealed REB1 consensus             that this 3&apos; end largely disappears in yeast cells carrying the <br /> binding sites in a number of locations throughout the yeast       ma 82.1 mutation, a mutation that inactivates an endonucle- <br /> genome, including near UAS elements for many polymerase           ase involved in 3&apos;-end processing of 5S RNA (26). Yip and <br /> II promoters, at telomeres, and near a centromere (4).            Holland (32) have further shown that an artificial transcript <br /> Within the rDNA, there are two REB1 sites, one which we           can be processed in vitro at this site to yield the 3&apos; end of the <br /> have studied for this paper at site I, downstream of the 3&apos; end   25S. In our in vitro transcription assays, we also observed a <br /> of the 25S rRNA, and a second located about 220 bp                3&apos; end that maps to the terminus of the 25S rRNA (Fig. 2B, <br /> upstream of the gene promoter but in the opposite orienta-        lane 1; this corresponds to position -96 of the sequence <br /> tion (23). Considering the orientation dependence of site I, it   shown in Fig. 1B). The results of actinomycin D chase <br /> is unlikely that the REBM site near the gene promoter is          experiment shown in Fig. 3 lend further support to the <br /> involved in polymerase I termination.                             conclusion that this is an RNA processing site and not a <br />    REB1 is an essential gene in yeasts (11, 24). Since it also    termination site. <br /> has numerous binding sites throughout the genome (4, 31),            Other in vivo sites of 3&apos;-end formation identified by <br /> the possibility arises that it is involved in more than just      Planta&apos;s group include Ti, T2, T3, and Tp. Ti 3&apos; ends are <br /> termination of transcription for RNA polymerase I. One            located downstream of the 3&apos; end of the 25S rRNA (Fig. 1B, <br /> possibility is that REB1 sites function as part of complex        between positions -81 and -50) and also appear to be <br /> activation regions to stimulate transcription from certain        attributable to processing (30). We have not observed any <br /> promoters. For example, Fedor et al. (8) have shown that          termini that would correspond to Ti in our in vitro assays. <br /> REB1 binding to a site between the promoters for the GALl            T2 maps to the beginning of a T-rich stretch between the <br /> and GALIO genes excludes nucleosomes from about 230 bp            EcoRI and HindIII sites (about position 100 in Fig. 1B) and <br /> of DNA and phases nucleosome positioning to either side.          was concluded to be the most likely candidate for an <br /> When the REB1 site was combined with a T-rich activator           authentic terminator (30). We do not observe a 3&apos; end at <br /> from the control region of the DEDI gene, a striking syner-       precisely this location in vitro. However, it is possible that <br /> gism in activating transcription from a nearby TATA box           T2 is the same as what we are calling site II. Site II maps to <br /> was observed. The REBi site by itself was essentially             the other side of the T-rich stretch as compared with T2 <br /> inactive in this assay. The ability of the REBi site to           (about position 155 in Fig. 1B). In Si nuclease protection <br /> synergize with the DEDI element was strongly distance             assays (as employed by the Planta group), the T-rich hybrid <br /> dependent and was completely eliminated by spacing the            could be lost, yielding an apparent map position coinciding <br /> two elements 121 bp apart. As Chasman et al. (4) point out,       with T2. As we have shown in Fig. 2C, 3&apos;-end formation at <br /> these results suggest a mechanism whereby REBi binding            site II occurs when the sequence is in either orientation. <br /> creates a nucleosome free space around its site, thereby          Therefore, we have concluded that site II (and presumably <br /> making nearby activator elements more accessible to the           T2 as well) is likely to be a simple roadblock caused by the <br /> binding of activator proteins. An alternate suggestion is that    tight binding of one or more proteins. It is unlikely, however, <br /> REB1 is a repressor of RNA polymerase II transcription.           to be an authentic terminator of polymerase I transcription. <br /> Wang et al. (31) placed an REB1 site between the CYCi                T3 refers to two locations, T3A and T3B, which would be <br /> upstream activator and its cognate TATA element and noted         at about map positions 550 and 890 in Fig. 2A. This would <br /> that this caused repression of the CYCi promoter. However,        put them within the EcoRI-PvuII fragment that was sur- <br /> it seems possible that both the activating and repressive         veyed in Fig. 2C, lane 5. We found nothing in vitro that <br /> effects of REBi binding could be incidental to the fact that      corresponds to these 3&apos; ends. Tp was reported to be about <br />  REB1 is a strong DNA binder and is likely to displace other       -300 bp upstream of the site of transcription initiation for <br />  nearby DNA binding proteins.                                     the ribosomal gene promoter. Our approximate mapping <br />    If the REB1 site actually is part of the termination ma-        suggests that site IV is near the SmaI site upstream of the <br />  VOL. 13, 1993                                                                     YEAST POLYMERASE I TERMINATOR                       657 <br />  <br /> promoter at position -216; however, in the absence of finer          3&apos; end of the 25S is formed by processing (the only known <br /> mapping, it is possible that Tp and site IV are the same.            exception to this rule is Ascaris lumbricoides, in which the <br />    Previous in vivo searches have not found 3&apos; ends that             terminator and the 3&apos; end of the largest mature rRNA <br /> would correspond to the site I we observed in vitro (12, 30).        coincide [25]). The yeast terminator requires a specific <br /> We can think of two possible reasons why this discrepancy            sequence in the DNA and is the binding site for a specific <br /> exists. First, multicopy rDNA plasmids in yeast may be               protein (REB1). Termination is not caused by a simple <br /> subject to transcription by more than one type of poly-              roadblock, however, as evidenced by the fact that the <br /> merase. Such transcription could have the dual effect of             terminator only functions in one orientation. Finally, the 3&apos; <br /> obscuring termini attributable to RNA polymerase I as well           ends formed at site I map about 17 bp upstream of the <br /> as creating additional termini having nothing to do with             protein binding site. <br /> polymerase I transcription. A second problem is that the 3&apos;             An interesting characteristic of the yeast polymerase I <br /> end formed at site I is located within a few nucleotides of the      terminator is the influence of sequences upstream of the <br /> EcoRI site. Many of the Si protection probes used in the in          REB1 protein binding site on the efficiency of 3&apos;-end forma- <br /> vivo studies (12, 30) were made from the EcoRI-HpaI                  tion (Fig. 9). The related effects of context sequences have <br /> fragment and could not have detected that particular 3&apos; end.         also been noted in the vertebrate systems. For example, in <br />    Direct evidence that the REB1 site (and site I) is involved       mice, sequences both upstream and downstream of the SalI <br /> in terminator function in vivo as well as in vitro comes from        box influence termination efficiency and the precise location <br /> the work of Holland&apos;s group (22). They assayed RNA 3&apos;-end            of the 3&apos; ends (16). In the frog system, we saw no context <br /> formation on ribosomal minigenes utilizing a low-copy-               effect in vitro (21), but the efficiency of termination appeared <br /> number CEN plasmid. When the EcoRI-HpaI fragment was                 to be context dependent in oocyte injection experiments <br /> placed downstream of a ribosomal gene promoter, it gener-            (17). <br /> ated a single 3&apos; terminus. They further showed that upon                Is REB1 the terminator protein? We believe we have <br /> deletion of sequences corresponding to positions 8 to 46             shown in this paper that the REB1 binding site is an essential <br /> (Fig. 1B), including the REB1 site, 3&apos;-end formation was             element of an RNA polymerase I terminator in yeast. The <br /> eliminated and longer read-through products were observed.           obvious question is whether or not it is the REB1 protein <br /> It is interesting that in vivo on a low-copy-number plasmid,         whose binding to that site causes termination. To summarize <br /> only the 3&apos; terminus specified by the REB1 binding element           the results from a number of frustrating experiments, we still <br /> (site I) is observed and there are no 3&apos; termini corresponding       do not know the answer to that question. J. Warner and <br /> to sites II and III. This leads us to think that termination at      coworkers have generously supplied us with recombinant <br /> site I is very efficient in vivo and that transcription ordinarily   REB1 protein and with strains of yeast carrying tempera- <br /> never reaches sites II and III.                                      ture-sensitive mutations in the REB1 protein. To date, we <br />    What causes 3&apos;-end formation at sites II and III? At both of      have been unsuccessful in obtaining any significant stimula- <br /> these locations, 3&apos;-end formation is independent of orienta-         tion of termination by adding back excess REB1 protein. We <br /> tion. This suggests that at both locations, tightly bound            have added back recombinant protein made in bacteria as <br /> proteins may act as roadblocks for RNA polymerase I                  well as native protein partially purified from yeast extracts. <br /> approaching from either direction. Lac repressor protein has         The adding back has been made to extracts that were <br /> been reported to block elongating RNA polymerase I and II            partially depleted in REB1 binding activity either by compe- <br /> (6, 14), and it is likely that other tightly bound proteins would    tition with specific oligonucleotides or by heat shock of <br /> have similar effects. The question then becomes, what                temperature-sensitive extracts. Among the several possibil- <br /> proteins are likely to be bound at sites II and III? The 3&apos; ends     ities, it may be that either (i) REB1 really is the relevant <br /> formed at site II map closely to what appears to the the             protein and we have not found the correct protocol for <br /> upstream boundary of the rDNA enhancer element. In vivo              reconstitution; (ii) the real terminator protein is something <br /> work (7, 22, 29) indicates that the enhancer element extends         other than REB1, which happens to bind to essentially the <br /> from about position 150 to slightly beyond the HindIII site.         identical REB1 binding site; or (iii) an additional factor is <br /> We have shown elsewhere that the enhancer element is                 involved in termination, and it is limiting in our extracts. It <br /> highly active in the yeast whole-cell extract (27a), and thus        will require further work, including some fractionation of the <br /> we would expect proteins to be bound to the enhancer                 extract, to decide between these possibilities. <br /> element in our termination assays as well. We conclude it is <br /> possible that the 3&apos; ends observed at site II are attributable to           <br /> a roadblock imposed by one or more enhancer binding <br /> proteins.                                                   <br />    The explanation for site III is more speculative. Some          <br /> reports fail to detect any transcription enhancer activity in      <br /> this region (7, 22). However, the work of Stewart and <br /> Roeder (29) clearly showed that there was something in this                        <br /> region that strongly affected recombination activity. We also        <br /> note a recent paper showing that a terminator of replication          <br /> maps in this same general region (13). We might be detecting             <br /> a transcription roadblock caused by proteins binding to one           <br /> or more of these elements in vitro.                                      <br />    Comparison of site I with other known terminators of                <br /> polymerase I transcription. The yeast polymerase I termina-               <br />                                                                          <br /> tor has characteristics very similar to those of terminators          <br /> that have been previously studied in both mice and frogs. To             <br /> begin with, the yeast terminator is located some distance               <br /> (108 bp) downstream of the 3&apos; end of the mature 25S, and the              <br /> </body></html>