<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Unfolded Protein Response Selectively Targets Active Smoothened <br /> Mutants <br /> Suresh Marada,a Daniel P. Stewart,a William J. Bodeen,a,c Young-Goo Han,b Stacey K. Ogdena <br /> Departments Biochemistrya Developmental Neurobiology,b St. Jude Children&#226;&#8364;&#8482;s Research Hospital, Memphis, Tennessee, USA; Integrated Program Biomedical <br /> Sciences, University Tennessee Health Science Center, Memphis, Tennessee, USAc <br />  <br /> The Hedgehog signaling pathway, essential regulator developmental patterning, implicated playing causative <br /> and <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span> roles range human cancers. The signal-transducing component pathway, Smoothened, revealed <br /> itself efficacious therapeutic target combating oncogenic signaling. However, therapeutic challenges remain cases <br /> where tumors acquire resistance Smoothened antagonists, cases signaling driven active Smoothened <br /> mutants exhibit reduced sensitivity compounds. We previously demonstrated active Smoothened mutants are <br /> subjected prolonged endoplasmic reticulum (ER) retention, likely mutations triggering conformation shifts that <br /> are detected ER quality control. We attempted exploit biology demonstrate deregulated Hedgehog signaling <br /> driven active Smoothened mutants specifically attenuated ER stressors induce unfolded protein response (UPR). <br /> Upon UPR induction, active Smoothened mutants targeted ER-associated degradation, resulting attenuation inap- <br /> propriate pathway activity. Accordingly, UPR agonist thapsigargin attenuated mutant Smoothened-induced <br /> phenotypes vivo Drosophila melanogaster. Wild-type Smoothened physiological Hedgehog patterning af- <br /> fected, suggesting UPR modulation provide novel therapeutic window evaluated targeting active Smooth- <br /> ened mutants disease. <br />  <br />  <br />  <br /> T   Hedgehog (Hh) signaling pathway provides essential pat- <br />     terning information development frequently ac- <br /> tivated cancer (1, 2). Inappropriate Hh signaling causative in <br />                                                                                   attenuate translation nascent proteins bound ER (22). <br />                                                                                   The ATF6 IRE1&#226;?&#163; branches activate transcription factors that <br />                                                                                   drive expression UPR target genes involved protein quality <br /> medulloblastoma basal cell carcinoma impli-                      control ER-associated degradation (ERAD). ERAD targets <br /> cated number additional cancers, including                   misfolded proteins retro-translocation ER cy- <br /> lung, breast, prostate, digestive tract (2&#226;&#8364;&#8220;9). Smoothened                     toplasm, undergo proteasome-mediated degradation <br /> (Smo), member G-protein-coupled receptor superfamily,                    (20, 23&#226;&#8364;&#8220;25). Persistent ER stress corrected the <br /> functions requisite signal-transducing molecule Hh                  UPR eventually result apoptosis (20). However, exact <br /> pathway (10, 11). Accordingly, oncogenic <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>mutation</span> Smo                   mechanisms UPR signals induction apoptosis un- <br /> mechanism Hh pathway inappropriately                      der conditions clear. <br /> activated cancer (3; http://www.sanger.ac.uk).                                     Given ability influence cellular homeostasis apopto- <br />     We recently described set active Smo mutants that, like                  sis, surprise UPR attractive target <br /> oncogenic Smo, induce ligand-independent Hh pathway activity                      therapeutic intervention cancer. Because tumor cells typi- <br /> (12). These mutants, C320A C339A Drosophila mela-                      cally exist <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-poor, hypoxic conditions readily <br /> nogaster protein C299A C318A murine protein,                   induce ER stress, widely acknowledged therapeutic <br /> predicted break disulfide bonds stabilize regulated con-                manipulation UPR conditions serve an <br /> formation Smo extracellular loop domain (12, 13). Consis-                  Achilles&#226;&#8364;&#8482; heel targeting tumor cells (26, 27). Accordingly, a <br /> tent prediction alteration bonds results               number small-molecule ER stress modulators, UPR ago- <br /> misfolded protein, mutants largely retained               nists antagonists, currently en route clinic (27). <br /> endoplasmic reticulum (ER) (12). Similarly, oncogenic Smo                         The increased localization active Smo mutants ER <br /> mutant SmoM2 reported largely ER localized (14,                    prompted test sensitive alteration <br /> 15). However, small pool M2 escapes ER traffics               ER homeostasis induction UPR. Here, describe <br /> primary cilium atypical Rab8 dependent secretory                       findings, demonstrate active Smo mutants, includ- <br /> route (16, 17). This transport ER primary cilium               ing extracellular loop C-<span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span>-A mutants oncogenic mutant <br /> important M2 oncogenic activity, genetic ablation                   SmoM2, specifically destabilized UPR conditions <br /> primary cilium attenuates M2-induced tumor formation mice <br /> (16, 18). <br />     Accumulation misfolded protein ER adversely affects                    Received 24 October 2012 Returned modification 13 December 2012 <br /> ER homeostasis (19, 20). This result high ER stress, leading                  Accepted 1 April 2013 <br /> to induction unfolded protein response (UPR), compen-                       Published ahead print 9 April 2013 <br /> satory process aimed ameliorating ER stress preventing                        Address correspondence Stacey K. Ogden, Stacey.Ogden@stjude.org. <br /> stress-induced cell death (20, 21). The UPR organized                  Copyright &#194;&#169; 2013, American Society Microbiology. All Rights Reserved. <br /> branches, controlled unique upstream activator. The                        doi:10.1128/MCB.01445-12 <br /> PERK branch triggers phosphorylation elongation factor 2&#226;?&#163; to <br />  <br />  <br /> June 2013 Volume 33 Number 12                                     Molecular Cellular Biology   p. 2375&#226;&#8364;&#8220;2387                                    mcb.asm.org   2375 <br />  Marada et al. <br />  <br />  <br />  <br /> of thermally chemically induced ER stress. Under                      3T3 cells, described above. Cl8 lysates prepared 1% NP-40, 150 <br /> conditions, signaling active Smo mutants attenuated                    mM NaCl, 50 mM Tris, 50 mM NaF, 0.5 mM DTT, 1&#226;&#171;&#187; PIC (Roche), <br /> their selective degradation ERAD. Consistent                     centrifuged 10 min 2,000 &#226;&#171;&#187; g. NIH 3T3 cells lysed as <br /> results, ER stress UPR-inducing compound thapsi-                        described above. Supernatants treated peptide-N-glycosidase F <br /> gargin blocks Smo-mediated Hh gain-of-function phenotypes                       (PNGase), endoglycosidase H (endo H), O-glycosidase, &#226;?&#173;-phosphatase <br />                                                                                 (NEB) 2 h room temperature analyzed SDS-PAGE and <br /> in vivo Drosophila. These findings suggest ER stress <br />                                                                                 Western blotting. <br /> modulators trigger UPR represent novel thera- <br />                                                                                     Control Hrd1 small interfering RNAs (siRNAs) obtained <br /> peutic window evaluated treatment Hh-dependent                     Invitrogen transfected manufacturer&#226;&#8364;&#8482;s instructions. <br /> cancers. Such compounds particularly efficacious                      Hrd1 siRNA consisted pool individual siRNAs. For knock- <br /> cancers initiated oncogenic Smo and/or tumors harbor-                     experiments, approximately &#226;&#172;&#402;7 &#226;&#171;&#187; 105 cells transfected with <br /> ing Smo mutations demonstrate reduced sensitivities                     indicated mSmo expression vector 60 nM siRNA using Lipo- <br /> the current cache Smo inhibitors (28&#226;&#8364;&#8220;31).                                    fectamine 2000 (Invitrogen). Cells grown 37&#194;&#176;C &#226;&#172;&#402;44 h, shifted <br />                                                                                 40&#194;&#176;C 4 h prior lysis, processed described above. <br /> MATERIALS AND METHODS                                                               Reverse transcription-PCR (RT-PCR) analyses. To assess endoge- <br />                                                                                 nous Xbp1 splicing, nontransgenic w1118 larvae grown 22&#194;&#176;C or <br /> Functional assays biochemical analyses. Reporter assays per-           29&#194;&#176;C, collected third-instar stage, homogenized extracted <br /> formed described previously (12), following modifications.          using TRIzol reagent (Sigma). cDNA synthesized 5 &#226;?&#174;g RNA <br /> For rescue experiments, &#226;&#172;&#402;1.5e6 Cl8 cells transfected 100 ng           using SuperScript III (Invitrogen) following prim- <br /> ptc&#226;&#338;&#172;136-luciferase, 10 ng pAc-Renilla, 20 ng smo 5= untranslated region         ers: Xbp1-RT-f, 5=-AGATGCATCAGCCAATCCAAC; Xbp1-RT-r, 5=-C <br /> (UTR) double-stranded RNA (dsRNA), 100 ng pAc-hh vector <br />                                                                                 AGGTGGACACACAGTCGT. <br /> control, 20 ng indicated wild-type mutant pAc-smo con- <br />                                                                                     To assess gli1 smo expression quantitative PCR (qPCR), NIH <br /> struct (12, 32, 33). For dominant activity assays, 20 ng indicated <br />                                                                                 3T3 cells plated described transfected using Fugene 6 <br /> myc-smo expression vector expressed absence Hh, re- <br />                                                                                 (Promega). Approximately 30 h posttransfection, culture medium was <br /> porter activity assessed described previously (12). Cells trans- <br />                                                                                 replaced complete medium containing 250 nM thapsigargin the <br /> fected 25&#194;&#176;C allowed recover 24 h prior shifting 22&#194;&#176;C or <br />                                                                                 vehicle (ethanol) control. RNA extracted RNeasy (Qiagen) &#226;&#172;&#402;16 <br /> 29&#194;&#176;C 24 h prior analysis. For Hsp70 inhibition, cells treated with <br />                                                                                 h medium exchange. cDNA synthesized 5 &#226;?&#174;g RNA by <br /> VER155008 (VER; Tocris Bioscience) vehicle control (dimethyl <br />                                                                                 using SuperScript III first-strand synthesis (Invitrogen). <br /> sulfoxide [DMSO]) &#226;&#172;&#402;16 h prior cell lysis. Reporter assays were <br />                                                                                 qPCRs performed cDNA diluted 1:10 using SYBR green PCR <br /> performed times duplicate, data pooled. Re- <br />                                                                                 master mix (Applied Biosystems). GAPDH used reference gene, <br /> porter activity shown percent activity relative control Hh <br />                                                                                 results analyzed using standard 2&#226;&#171;&#186;&#226;&#338;&#172;&#226;&#338;&#172;CT method (35) the <br /> response temperature, set 100%. Error bars represent standard <br />                                                                                 following gene-specific primers: Gli1-qPCR-f, 5=-GGTCTCGGGGTCTCA <br /> errors means. <br />     For protein stability analysis Drosophila cells, &#226;&#172;&#402;5 &#226;&#171;&#187; 106 Cl8 cells      AACTGC; Gli1-qPCR-r, 5=-CGGCTGACTGTGTAAGCAGAG; mSmo- <br /> were transfected 5 &#226;?&#174;g wild-type mutant pAc-smo using              qPCR-f, 5=-CGCCAAGGCCTTCTCTAAGCG; mSmo-qPCR-r, 5=-CCT <br /> Lipofectamine 2000 (Invitrogen). Cells transfected 25&#194;&#176;C al-         CTGCCTGGGCTCAGCAT; mGAPDH-qPCR-f, 5=-GTGGTGAAGCAG <br /> lowed recover 24 h shifting 22&#194;&#176;C 29&#194;&#176;C. Lysates         GCATCTGA; mGAPDH-qPCR-r, 5=-GCCATGTAGGCCATGAGGTC. <br /> prepared 24 h temperature shift 2% SDS, 4% glycerol, 40 mM             qPCR analysis performed times triplicate, data <br /> Tris-HCl (pH 6.8), 0.5 mM dithiothreitol (DTT), 1&#226;&#171;&#187; protease inhib-          pooled. Error bars indicate standard errors means. <br /> itor cocktail (PIC; Roche). Extracts sheared passing 5 times                Fly crosses. Fly stocks maintained 18&#194;&#176;C Jazz agarose <br /> through 26-gauge syringe. Equal amounts protein analyzed           (Fisher). Crosses performed 22&#194;&#176;C 29&#194;&#176;C indicated. Upstream <br /> SDS-PAGE Western blotting anti-Myc (Roche) antikinesin             activation sequence (UAS)-myc-smo, UAS-myc-smoC320A, UAS- <br /> (cytoskeleton) antibodies.                                                      myc-smoC339A (12) expressed control MS1096-Gal4, <br />     For protein stability analysis mammalian cells, &#226;&#172;&#402;1 &#226;&#171;&#187; 106 NIH 3T3         C765-Gal4, Ap-Gal4 indicated. The Xbp1k13803 allele used for <br /> cells transfected 2 &#226;?&#174;g wild-type mutant pcDNA3.1-msmo           synthetic interaction studies. Synthetic interaction crosses per- <br /> (12). Transfected cells maintained 37&#194;&#176;C &#226;&#172;&#402;44 h shifted      formed 25&#194;&#176;C. For drug treatment, 1- 24-h MS1096-GAL4, UAS-myc- <br /> to 40&#194;&#176;C 4 h prior lysis. Lysates prepared 50 mM Tris-HCl         smoC320A C765-GAL4, UAS-myc-smoC320A embryos collected <br /> (pH 7.4), 150 mM NaCl, 0.25% deoxycholic acid, 1% NP-40, 1 mM                   transferred vials containing 2 ml Jazz agarose containing vehicle <br /> EDTA, 0.1% SDS, 0.5 mM DTT, 1&#226;&#171;&#187; PIC (Roche) incubated                (ethanol) thapsigargin (Sigma) final concentration 1 &#226;?&#174;M. Drug <br /> 30 min 4&#194;&#176;C (34). Extracts cleared centrifuging 16,000 &#226;&#171;&#187; g      feeding performed separate crosses, multiple progeny <br /> 4&#194;&#176;C 45 min. Supernatants analyzed SDS-PAGE Western              analyzed. <br /> blotting using anti-GADD153/CHOP (B-3; Santa Cruz Biotechnology),                   For wing analyses, crosses performed twice, multiple <br /> anti-Smo (E5; Santa Cruz Biotechnology), antitubulin (Cell Signal-          progeny analyzed. Representative wings adult flies were <br /> ing). For Gli1 protein analysis, cells transferred Dulbecco&#226;&#8364;&#8482;s mod-      mounted glass slides using DPX mounting medium imaged on <br /> ified Eagle&#226;&#8364;&#8482;s medium (DMEM) plus 0.5% serum &#226;&#172;&#402;40 h prior lysis.           Zeiss Stemi 2000-C11 microscope Zeiss AxioCam ICc3 camera. In <br /> Gli1 analyzed Western blotting L42B10 (Cell Signaling).             cases wings severely blistered, fly imaged. Im- <br />     For drug sensitivity analysis, NIH 3T3 cells expressing murine Smo          ages prepared using Photoshop CS4. <br /> (here referred mSmo) proteins 37&#194;&#176;C treated vehicle 1          For salivary gland analysis, UAS-Xbp1-GFP (36) and/or UAS-myc- <br /> &#226;?&#174;M thapsigargin (Sigma) indicated concentration 17-AAG,               smoC320A expressed control sgs3-Gal4. Green fluores- <br /> SNX2112, Bortezomib (Selleck Chemicals) DMEM containing 0.5%              cent protein (GFP) expression examined salivary glands dissected <br /> fetal calf serum 4 h prior lysis. Whole-cell lysates prepared   third-instar larvae. Multiple salivary glands examined across <br /> analyzed described above. For assays requiring Sonic Hedgehog (SHH)          independent crosses, representative samples shown. <br /> stimulation, cells treated recombinant SHH (R&amp;D Systems)               Immunofluorescence. pAc-myc-smo constructs expressed S2 <br /> a concentration 100 ng/ml &#226;&#172;&#402;12 h prior indicated drug treat-       cells described previously (12). For temperature sensitivity analyses, <br /> ments.                                                                          cells transfected 25&#194;&#176;C allowed recover 6 h prior to <br />     For glycosylation analysis, cell lysates prepared Cl8 NIH      temperature shift. Fixation, immunostaining, image analysis were <br />  <br />  <br /> 2376   mcb.asm.org                                                                                                            Molecular Cellular Biology <br />                                                                                                         Active Smoothened Mutants Are UPR Substrates <br />  <br />  <br />  <br /> performed 48 h temperature shift, described previously (12).   robust GFP signal salivary glands dissected thapsigargin- <br /> Primary cilium analysis NIH 3T3 cells performed exactly de-       fed larvae (Fig. 1G=). We tested genetic interaction be- <br /> scribed previously (12). GRP94 antibody provided L. Hendershot       tween smo IRE1&#226;?&#163; stress response pathway. The Myc- <br /> (37). Anti-Ci 2A1 hybridoma obtained DHSB. Imaginal disc       SmoC320A <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> modified introduction the <br /> and salivary gland staining analyses performed described        Xbp1 loss-of-function allele Xbp1k13803 (Fig. 1H H=), further <br /> previously (12). <br />                                                                             supporting observed wing phenotypes did result <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br />     For indirect immunofluorescence assays, indicated primary an- <br /> tibodies detected using Alexa Fluor (Invitrogen) secondary antibod-    chronic ER stress triggered expression active Smo mutants. <br /> ies conjugated 488 555 fluorophores. Data obtained using           To confirm observed vivo temperature sensitivity <br /> Zeiss LSM 510 confocal microscope processed using LSM Image             specific effect mutant Smo proteins, artifact of <br /> Browser Photoshop CS4 software.                                         vivo transgene expression, switched vitro Clone 8 <br />                                                                             (Cl8) cell culture allowed perform functional <br /> RESULTS                                                                     assays permissive (22&#194;&#176;C) restrictive (29&#194;&#176;C) temperatures. <br /> To determine activities ER-retained active Dro-              Cl8 cells derived wing imaginal disc tissue possess an <br /> sophila Smo mutants C320A C339A affected induction               intact Hh pathway, making ideal cell line perform <br /> of cellular stress response, induced thermal stress perform-        biochemical functional analyses Hh pathway activity (33, <br /> ing crosses high temperature (29&#194;&#176;C). To determine              40). We assessed ability wild-type mutant Smo proteins <br /> the UPR induced temperature, monitored ER stress           rescue reporter gene expression Cl8 cells dsRNA-medi- <br /> examining (i) IRE1&#226;?&#163;-mediated splicing endogenous Xbp1 tran-              ated smo knockdown background 22&#194;&#176;C 29&#194;&#176;C (Fig. 2A). <br /> script (ii) activity Xbp1-GFP stress sensor expresses        Knockdown endogenous smo using 5=-UTR dsRNA attenu- <br /> GFP stress-induced Xbp1 splicing (36, 38). Only unspliced              ated Hh-induced reporter gene expression permissive and <br /> Xbp1 transcript detected RNA harvested nontrans-                restrictive temperatures (Fig. 2A, UTR dsRNA). Reexpression of <br /> genic w1118 larvae grown 22C&#194;&#176; (Fig. 1A). Conversely, robust            wild-type Myc-Smo using cDNA lacking UTR sequence did <br /> enrichment spliced Xbp1 transcript evident RNA har-               alter baseline signaling activity, rescued Hh-dependent <br /> vested larvae grown 29&#194;&#176;C (Fig. 1A), indicating pres-            reporter gene induction similar levels temperatures, <br /> ence ER stress. Similar results obtained Xbp1-             suggesting that, observed flies, wild-type Myc-Smo func- <br /> GFP stress sensor. Whereas minimal expression GFP                    tion significantly altered temperature. Consistent with <br /> observed salivary glands dissected ER stress reporter lar-          previous studies performed 25&#194;&#176;C (12), C320A and <br /> vae grown 22&#194;&#176;C, strong GFP signal observed 29&#194;&#176;C (Fig.           C339A mutant Myc-Smo proteins significantly elevated baseline <br /> 1B B=). Taken together, results support ER stress         signaling partially (C320A) fully (C339A) rescued Hh-de- <br /> response induced performing crosses 29&#194;&#176;C.                      pendent reporter gene activity smo knockdown background <br />     We previously demonstrated expressed                22&#194;&#176;C (Fig. 2A, white light gray bars). However, mu- <br /> control UAS/GAL4 25&#194;&#176;C, wild-type Smo                   tants compromised ability elevate baseline activity <br /> transgene induces modest Hh gain-of-function phenotype,               rescue Hh-induced reporter gene activity restrictive <br /> SmoC320A SmoC339A transgenes induce strong phenotypes                   29&#194;&#176;C temperature (Fig. 2A, dark gray black bars). <br /> (12). When expressed transgenes 22&#194;&#176; 29&#194;&#176;C,                   We examined abilities wild-type mutant Myc- <br /> wild-type Myc-Smo did trigger robust Hh gain-of-function              Smo proteins induce ectopic reporter gene activity permis- <br /> phenotype 22&#194;&#176;C did induce mild ectopic vein formation                sive restrictive temperatures wild-type smo background. <br /> when expressed 29&#194;&#176;C, likely higher-level UAS/GAL4                 Consistent Smo posttranslationally regulated (41, 42), <br /> transgene expression 29&#194;&#176;C (39) (Fig. 1D compared D=               provision exogenous wild-type Myc-Smo did increase <span id='am-242' about='xsp:base' typeof='owl:Thing'>base</span>- <br /> C). Conversely, expression activating C320A C339A                line signaling temperature (Fig. 2B, wtSmo). Conversely, <br /> Myc-Smo mutants induced mild phenotypes 29&#194;&#176;C dra-                    22&#194;&#176;C, mutants increased baseline signaling level near <br /> matic phenotypes, including dorsal wing overgrowth, wing blis-              (C320A) equal (C339A) control Hh response (Fig. 2B, <br /> tering, delayed eclosion 22&#194;&#176;C (Fig. 1E F, compared E=         white bars). This activity attenuated 29&#194;&#176;C, suggesting that <br /> and F=). Additionally, C320A C339A Myc-Smo                 presence endogenous Smo does correct temperature <br /> transgenic flies eclosed normal ratios 29&#194;&#176;C, expression        sensitivity C320A C339A Myc-Smo mutants (Fig. 2B, <br /> more active Myc-SmoC339A mutant induced high degree                    gray bars). Taken smo rescue reporter assay re- <br /> pupal lethality 22&#194;&#176;C. Taken together, results suggest         sults, results suggest observed vivo temperature <br /> active Smo mutants induce robust signaling low tempera-              sensitivity activation Smo mutants triggered molec- <br /> ture activity significantly attenuated temper-       ular mechanism affects mutant Smo proteins, by <br /> ature induces stress response pathways, including UPR              altered transgene expression. <br /> (Fig. 1A B).                                                                To determine attenuation mutant Smo signaling <br />     To confirm strong wing phenotypes observed              result heat shock response, tested the <br /> low temperature chronic ER stress resulting            Hsp70 inhibitor VER restore Myc-SmoC339A activity the <br /> high-level expression mutant Smo protein, expressed myc-              restrictive temperature (Fig. 2C). We did observe significant <br /> smoC320A Xbp1-GFP stress reporter monitored                effect Myc-SmoC339A-induced reporter gene activity re- <br /> GFP induction Smo-expressing salivary glands. Despite robust             sponse VER treatment, suggesting heat shock re- <br /> accumulation Myc-SmoC320A protein salivary gland cells,               sponsible attenuation signaling active Smo mutants at <br /> activity GFP stress sensor detected (Fig. 1G). As          29&#194;&#176;C. <br /> control, treated Xbp1-GFP transgenic larvae potent                  Because high-level Hh pathway activity Drosophila corre- <br /> ER stress-inducing compound thapsigargin, observed                 lates accumulation Smo plasma membrane (PM) <br />  <br />  <br /> June 2013 Volume 33 Number 12                                                                                                   mcb.asm.org 2377 <br />  Marada et al. <br />  <br />  <br />  <br />  <br /> FIG 1 Active Smo mutants temperature sensitive vivo. (A B) Growth 29&#194;&#176;C induces UPR. RNA harvested w1118 larvae assessed for <br /> endogenous Xbp1 splicing RT-PCR. The stress-induced Xbp1 splice variant observed RNA larvae grown 22&#194;&#176;C, evident RNA <br /> extracted larvae grown 29&#194;&#176;C (A). The ER stress sensor UAS-Xbp1-GFP expressed salivary glands control sgs3-Gal4 (sgs&#226;&#172;&#381;Xbp1-GFP). <br /> Crosses performed 22&#194;&#176;C 29&#194;&#176;C, indicated. Individual salivary glands outlined white. Under conditions low ER stress, Xbp1 frame <br /> with GFP, minimal GFP expression observed (B=). Upon ER stress induction, Xbp1 alternately spliced place frame GFP, resulting a <br /> robust GFP signal (B). (C F) Active Smo mutants temperature sensitive. Wild-type, C320A, C339A Myc-Smo proteins expressed 22&#194;&#176;C (D to <br /> F) 29&#194;&#176;C (D= F=) control MS1096-Gal4 driver. Wild-type Myc-Smo did induce significant phenotype 22&#194;&#176;C induced mild <br /> phenotype 29&#194;&#176;C (D=, compared panel D). Conversely, C320A C339A Smo mutants induced mild phenotypes 29&#194;&#176;C strong phenotypes 22&#194;&#176;C (E <br /> and F), suggesting activity reduced conditions thermal stress. The MS1096 driver wing served control (C). (G) Expression active <br /> Smo mutant does induce ER stress. Salivary glands larvae expressing Xpb1-GFP stress sensor (G=) combination Myc-SmoC320A <br /> (G) control sgs3-Gal4 grown 22&#194;&#176;C vehicle- thapsigargin-treated food. Myc-SmoC320A (blue) did induce GFP stress sensor (G), <br /> but potent ER stress-inducing compound thapsigargin did (G=). The filamentous actin stain phalloidin marks plasma membrane (F-actin [magenta]). (H <br /> and H=). The Xbp1 mutation does modify Myc-SmoC320A phenotype. Wings flies expressing Myc-SmoC320A control epithelial <br /> driver C765-Gal4 absence (H) presence (H=) Xpb1 loss-of-function allele Xbp1k13803 shown. Introduction Xbp1k13803 allele did not <br /> modify SmoC320A-induced wing phenotype. For conditions, representative salivary glands wings shown. Bars, 50 &#226;?&#174;m. <br />  <br />  <br />  <br /> (41), wanted determine robust vitro signaling                overlap ER marker calreticulin (Cal)-GFP-KDEL (43) either <br /> and strong vivo phenotypes induced 22&#194;&#176;C result                   22&#194;&#176;C 29&#194;&#176;C (Fig. 3A A=). Myc-SmoC320A C339A dem- <br /> active Smo mutants escaping ER localizing PM                    onstrated different localization <span id='am-243' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, colocalizing almost <br /> this permissive temperature. To so, examined subcellular                    completely Cal-GFP ER marker 22&#194;&#176;C 29&#194;&#176;C <br /> localization wild-type mutant Myc-Smo proteins                          (Fig. 3B B= C C=). We did detect obvious colocal- <br /> Schneider 2 (S2) cells, embryonic Drosophila cell line com-                    ization Myc-SmoC320A C339A sub-PM fila- <br /> monly used imaging studies, 22&#194;&#176;C 29&#194;&#176;C (Fig. 3A C).                  mentous actin stain phalloidin temperature (Fig. 3B and <br /> In absence Hh, Smo localizes intracellular vesicles                 C), suggesting increased signaling activity mu- <br /> recycling endosomes (14). Accordingly, wild-type Myc-Smo lo-                      tants 22&#194;&#176;C result bulk relocalization ER <br /> calized primarily punctate structures did significantly               PM. <br />  <br />  <br /> 2378   mcb.asm.org                                                                                                                 Molecular Cellular Biology <br />                                                                                                                    Active Smoothened Mutants Are UPR Substrates <br />  <br />  <br />  <br />  <br /> FIG 2 Active Smo mutants temperature sensitive vitro. (A) Cl8 cells transfected vector wild-type, C320A, C339A Myc-Smo <br /> expression vectors (pAc-myc-smo) presence pAc-hh vector control smo 5=UTR dsRNA, indicated. Reporter gene activity was <br /> determined cells cultured 22&#194;&#176;C 29&#194;&#176;C, indicated. Whereas wild-type Myc-Smo rescued ptc&#226;&#338;&#172;136-luciferase activity temperatures, C320A and <br /> C339A compromised abilities modulate reporter gene activity restrictive 29&#194;&#176;C temperature. The control Hh response temperature <br /> was set 100%. Reporter gene activity shown percent activity relative control Hh response. Hh reporter gene activity normalized a <br /> pAc-Renilla control. Error bars indicate standard errors means (SEM). (B) Cl8 cells transfected hh wild-type mutant myc-smo expression <br /> vectors, indicated. The Hh response temperature set 100%. Reporter activity induced indicated Myc-Smo protein wild-type smo <br /> background shown relative Hh response. Hh reporter gene activity normalized pAc-Renilla control. Error bars indicate SEM. (C) The heat <br /> shock response does alter signaling active Smo mutant. Cl8 cells grown 29&#194;&#176;C transfected hh myc-smoC339A expression vectors and <br /> treated DMSO (vehicle) Hsp70 inhibitor VER, indicated. Reporter activity induced Myc-SmoC339A shown relative Hh response, set <br /> to 100%. VER did rescue C339A signaling restrictive temperature, indicating attenuated mutant Smo signaling 29&#194;&#176;C heat <br /> shock response. Hh reporter gene activity normalized pAc-Renilla control. Error bars indicate SEM. <br />  <br />  <br />    To biochemically confirm activating mutants un-                 subjected lysates Smo-expressing cells endo H, PNGase, <br /> able escape ER, conditions favorable                 &#226;?&#173;-phosphatase treatments. Endo H cleaves high-mannose <br /> for high-level Hh signaling, expressed wild-type, C320A,                  oligosaccharides added ER, does cleave the <br /> C339A Myc-Smo proteins Cl8 cells 22&#194;&#176;C presence                   complex oligosaccharides processed post-ER <br /> Hh examined processing N-linked glycans. To so,                  compartments. PNGase cleaves N-linked oligosaccharide resi- <br />  <br />  <br /> June 2013 Volume 33 Number 12                                                                                                                mcb.asm.org 2379 <br />  Marada et al. <br />  <br />  <br />  <br />  <br />                                                                                 FIG 4 Active Smo mutants destabilized restrictive temperature. <br />                                                                                 pAc-myc-smo vectors encoding wild-type (A D), C320A (B E), and <br />                                                                                 C339A (C F) Myc-Smo proteins cotransfected S2 cells with <br /> FIG 3 Active Smo mutants retained ER. (A C) Wild-type (A),        pAc-GFP permissive (22&#194;&#176;C) restrictive (29&#194;&#176;C) temperatures. Cells were <br /> C320A (B), C339A (C) Myc-Smo proteins expressed S2 cells         stained Smo indirect immunofluorescence using anti-Myc (magenta) <br /> permissive (A C) restrictive (A= C=) temperatures. Myc-Smo visu-   imaged confocal microscopy. Whereas wild-type Myc-Smo (A A=) <br /> alized indirect immunofluorescence. Smo (Myc) red, phalloidin (PM         GFP (D F D= F=) stability expression significantly <br /> marker) blue, Cal-GFP-KDEL (ER marker) green. Wild-type Smo       affected temperature, active mutants destabilized restric- <br /> vesicular temperatures (A A=). Mutants overlapped ER       tive temperature (B C compared B= C=). Multiple fields cells were <br /> marker temperatures (B C). Bar, 5 &#226;?&#174;m. (D) Activated Smo mutants     examined independent experiments. Representative fields shown. <br /> are post-ER glycosylated. Whole-cell lysates Cl8 cells expressing Hh   Bar, 50 &#226;?&#174;m. (G) The Smo protein destabilized restrictive temperature. <br /> and wild-type, C320A, C339A Myc-Smo proteins 22&#194;&#176;C treated       Western blot analysis whole-cell lysates Cl8 cells expressing wild-type <br /> &#226;?&#173;-phosphatase indicated deglycosylating enzymes. Post-ER-phospho-       C339A Myc-Smo proteins revealed C339A protein levels decreased at <br /> rylated glycosylated forms wild-type Smo detected (lanes 1 3     29&#194;&#176;C. Wild-type Smo destabilized 29&#194;&#176;C. Kinesin (Kin) used as <br /> compared lane 4, fully collapsed). C320A C339A Smo proteins fully        loading control. <br /> collapsed endo H PNGase treatments, indicative being <br /> retained ER (lanes 5 12). Kinesin served loading control. <br />                                                                                     While examining Smo subcellular localization (Fig. 3A C), <br /> dues. EndoH, PNGase, &#226;?&#173;-phosphatase treatments revealed                      noticed wild-type Smo protein stability did not <br /> that, wild-type Smo detectable ER-glycosylated,                  appear affected temperature, C320A C339A Smo <br /> post-ER-glycosylated, Hh-induced phosphorylated forms,                      proteins consistently appeared stable permissive 22&#194;&#176;C <br /> C320A C339A proteins present ER resident                   temperature (Fig. 3A C). To determine spe- <br /> fraction; C320A C339A fully collapsed endo H                 cific stability effect mutant Smo proteins, effects <br /> PNGase treatments (Fig. 3D, compare lanes 1 4 7 12).                 transfection efficiency vitro transgene expression the <br /> We did observe partial endo H sensitivity wild-type Myc-Smo                  different temperatures, cotransfected pAc-GFP of <br /> protein (Fig. 3D, lane 3). This likely result N-linked               pAc-myc-smo expression vectors S2 cells permissive <br /> glycans post-ER-processed insect proteins having hybrid                   restrictive temperatures examined expression by <br /> paucimannosidic characteristics (44, 45). As such, lower band               gain-equalized immunofluorescence (Fig. 4A F). Wild-type <br /> of doublet observed endo H treatment wild-type Myc-                 Myc-Smo GFP tracer significantly affected by <br /> Smo likely represents ER resident fraction upper band               temperature (Fig. 4A compared A= D F compared D= <br /> likely represents post-ER processed protein. Taken together,                F=). Conversely, Myc-SmoC320A Myc-SmoC339A were <br /> these results support conclusion increased activity                consistently detectable higher temperature (Fig. 4B and <br /> observed C320A C339A mutants 22&#194;&#176;C                    C compared B= C=). To confirm results biochemically, <br /> bulk relocalization PM permissive temperature.                    prepared whole-cell lysates Cl8 cells expressing wild-type <br />  <br />  <br /> 2380   mcb.asm.org                                                                                                               Molecular Cellular Biology <br />                                                                                                      Active Smoothened Mutants Are UPR Substrates <br />  <br />  <br />  <br /> or highly active C339A mutant 22&#194;&#176;C 29&#194;&#176;C examined          temperature shift, C318A M2 mSmo proteins were <br /> Myc-Smo protein Western blotting. Whereas wild-type Myc-             significantly destabilized 40&#194;&#176;C (Fig. 5C=, lanes 3 6 compared <br /> Smo destabilized restrictive temperature, Myc-       1 2). This destabilization likely result ERAD, as <br /> SmoC339A protein level significantly reduced (Fig. 4G, lanes        ER-resident forms Smo variants stabilized at <br /> 3 4 compared lanes 5 6). Taken             40&#194;&#176;C proteasome inhibitor MG132 (Fig. 5C=, white arrow- <br /> immunofluorescence data, results suggest activating      head, lanes 8, 10, 12 compared 7, 9, 11). The post-ER <br /> Smo mutants specifically destabilized 29&#194;&#176;C.                      form wild-type Smo altered MG132 treatment (Fig. <br />     We previously demonstrated that, like fly proteins, mSmo        5C=, black arrowhead, lanes 7 8). <br /> mutants corresponding Drosophila Smo C320A C339A                     To confirm ER-retained mSmo proteins cleared by <br /> are largely ER retained (12). To biochemically validate ER         ERAD following UPR induction, targeted ERAD-specific <br /> retention mSmo mutants, expressed wild-type,                ubiquitin ligase Hrd1 siRNA tested stabilization of <br /> mSmoC299A (C320A equivalent), C318A (C339A equiva-                  mSmo mutants restrictive temperature (Fig. 5D). We ob- <br /> lent) NIH 3T3 cells examined ER post-ER glycosyla-           served C318A M2 mSmo proteins effec- <br /> tion patterns (Fig. 5A). Whereas wild-type mSmo existed         tively destabilized 40&#194;&#176;C cells treated control siRNA, their <br /> an endo H-sensitive ER form endo H-resistant/PNGase-             destabilization attenuated following Hrd1 knockdown (Fig. <br /> sensitive post-ER form, ER form C299A C318A         5D, lanes 3 4 compared lanes 9 12). Consistent these <br /> active mutants detected (Fig. 5A). We noted slow-            results, mSmoM2-mediated induction endogenous Gli1 pro- <br /> migrating form wild-type mSmo, presence               tein attenuated high temperature rescued following <br /> PNGase, cleaves ER post-ER N-linked glycosylation         Hrd1 knockdown 40&#194;&#176;C (Fig. 5D=, lanes 1 2 compared to <br /> moieties (Fig. 5A, lane 2). &#226;?&#173; phosphatase treatment confirmed      lane 4). Curiously, ER-retained form wild-type mSmo was <br /> this shift phosphorylation (Fig. 5A=, lanes 4 9   reduced 40&#194;&#176;C following Hrd1 knockdown (Fig. 5D, lanes 7 and <br /> 11). To determine mobility shift observed wild-         8). Additionally, mSmoM2-induced Gli1 protein level was <br /> type Smo result post-ER O-linked glycosylation event,      consistently reduced Hrd1 knockdown cells cultured 37&#194;&#176;C <br /> we treated lysates cells expressing wild-type mSmo O-         (Fig. 5D=, lane 3). We know cause reductions, <br /> glycosidase combination PNGase and/or endo            speculate Hrd1 knockdown induces compensatory ER <br /> H (Fig. 5A=). We observed modest collapse O-glycosidase          stress responses impact mSmoWT stability and/or down- <br /> treatment (Fig. 5A=, lane 2) complete collapse treatment       stream signaling events. Consistent prediction ERAD <br /> with enzymes (Fig. 5A=, lane 8), confirming re-      attenuation induces higher-level ER stress, observed enhanced <br /> sidual shift observed wild-type protein likely    CHOP induction cells treated Hrd1 siRNA (Fig. 5D, lanes <br /> O-linked glycosylation event.                                           7 12). <br />     The murine equivalent oncogenic W535L SmoM2 mu-                  We wanted determine effects observed <br /> tant (SmoA1, W539L, referred mSmoM2) induces ro-             following thermal ER stress induction recapitulated by <br /> bust Sonic Hedgehog-independent signaling activity (3, 28). Like        ER stress modulator specific UPR. To end, we <br /> C299A C318A mSmo mutants, significant fraction                 treated Smo-expressing cells potent ER stress-inducing <br /> mSmoM2 retained ER (14, 15). In order examine          compound thapsigargin. We expressed wild-type, C318A, M2 <br /> ratio ER-localized post-ER-localized mSmoM2, ex-               mSmo proteins NIH 3T3 cells treated cells vehicle or <br /> pressed mSmoM2 NIH 3T3 cells examined ER post-ER             thapsigargin 4 h prior lysis (Fig. 6A). Whereas wild-type <br /> glycosylation levels (Fig. 5B). Wild-type mSmo demonstrated           mSmo stability affected thapsigargin treatment, both <br /> near equal distribution ER post-ER glycosylated             C318A M2 mSmo proteins destabilized (Fig. 6A, lanes 3 <br /> pools (Fig. 5B, lanes 1 3). Conversely, bulk mSmoM2           6 compared lanes 1 2), suggesting addition to <br /> protein existed endo H-sensitive, ER-localized fraction (Fig.    affected thermal stress, active mutants sensi- <br /> 5B, lanes 4 6, post-ER versus deglycosylated). Although mod-         tive chemically induced ER stress. To confirm thapsi- <br /> est, endo H-resistant post-ER form SmoM2 detected             gargin-mediated mSmoM2 destabilization sufficient at- <br /> (Fig. 5B, lane 6) likely represented mSmoM2 protein            tenuate downstream signaling, assessed endogenous gli1 <br /> was localized en route primary cilium. Accord-         induction using qPCR Western blotting (Fig. 6B C). <br /> ingly, detected mSmoM2 primary cilium using in-            qPCR analysis revealed approximate 40-fold increase gli1 <br /> direct immunofluorescence; however, bulk mSmoM2 pro-             expression mSmoM2-expressing cells observed in <br /> tein colocalized ER resident protein GRP94 (Fig. 5B=,          mSmoWT-expressing cells (Fig. 6B, white bars). Consistent with <br /> ciliary slice [arrowhead] versus ER slice).                             result, observed robust increase Gli1 protein lysates <br />     Given bulk mSmoC318A mSmoM2 ER                  prepared mSmoM2-expressing cells (Fig. 6C, lanes 1 2). <br /> localized, reasoned that, like active Drosophila mutants,        Thapsigargin treatment specifically reduced mSmoM2-mediated <br /> they affected induction thermal ER stress re-          gli1 transcript induction approximately 50% level &#226;&#172;&#402;18-fold <br /> sponse. Thermal ER stress observed mammalian cells          mSmoWT baseline (Fig. 6B). Accordingly, Gli1 protein <br /> cultured 40&#194;&#176;C (46). Accordingly, observed induction        levels decreased lysates thapsigargin-treated cells <br /> ER stress response transcription factor CHOP NIH 3T3 cells           (Fig. 6C, lanes 3 4 compared lane 2). Consistent our <br /> incubated high temperature 2 4 h (Fig. 5C). To assess    biochemical analyses, thapsigargin did attenuate gli1 expres- <br /> the effect thermal ER stress mSmo proteins, expressed      sion cells expressing wild-type mSmo instead modestly <br /> wild type, C318A, mSmoM2 NIH 3T3 cells cultured 37&#194;&#176;C,          increased gli1 expression &#226;&#172;&#402;4-fold baseline (Fig. 6B, gray <br /> then shifted 40&#194;&#176;C 4 h prior whole-cell lysis (Fig. 5C=). We   bars). We know exact mechanism gli1 expres- <br /> found wild-type mSmo significantly affected        sion increased wild-type mSmo background response <br />  <br />  <br /> June 2013 Volume 33 Number 12                                                                                                mcb.asm.org 2381 <br />  Marada et al. <br />  <br />  <br />  <br />  <br /> FIG 5 Murine Smo mutants ER localized temperature sensitive. (A A=) Mammalian mSmo mutants post-ER glycosylated. Lysates prepared <br /> from NIH 3T3 cells expressing wild-type, C299A, C318A mSmo proteins treated vehicle (&#226;&#171;&#186;), endo H, PNGase, O-glycosidase, and/or &#226;?&#173;-phos- <br /> phatase indicated (&#226;&#171;&#185;) analyzed Western blotting. The post-ER glycosylated form, present wild-type mSmo, affected endo H <br /> (post-ER; A A=) affected PNGase O-glycosidase (deglycosylated; A A=). The ER-localized forms wild type active <br /> mutants sensitive endo H, demonstrating faster mobility treatment (deglycosylated; A A=). Endo H- PNGase-treated C299A C318A <br />  <br />  <br />  <br /> 2382   mcb.asm.org                                                                                                            Molecular Cellular Biology <br />                                                                                                                         Active Smoothened Mutants Are UPR Substrates <br />  <br />  <br />  <br />  <br /> FIG 6 Small-molecule UPR modulators target oncogenic Smo. (A) Active mSmo mutants destabilized thapsigargin. NIH 3T3 cells expressing wild-type, <br /> C318A, M2 mSmo proteins treated 1 &#226;?&#174;M thapsigargin (Thaps, &#226;&#171;&#185;) vehicle control (&#226;&#171;&#186;), indicated 4 h prior lysis. Western blotting of <br /> whole-cell lysates revealed C318A M2 Smo proteins destabilized response drug treatment. Wild-type mSmo significantly affected. Tubulin <br /> was loading control. (B C). Thapsigargin attenuates mSmoM2-induced pathway activity. RNA harvested NIH 3T3 cells expressing either <br /> wild-type M2 mSmo proteins. qPCR analysis revealed thapsigargin treatment resulted specific attenuation mSmoM2-induced gli1 transcript (B) and <br /> protein levels (C). smo expression increased modestly response drug treatment (B=). For qPCR analysis, expression shown fold induction the <br /> wild-type mSmo vehicle control. Expression normalized GAPDH reference gene. Error bars indicate standard errors means. For protein analysis <br /> (C), tubulin served loading control, results CHOP indicate ER stress induction. (D) UPR modulators destabilize mSmoM2. NIH 3T3 cells <br /> expressing mSmoM2 protein treated HSP90 inhibitors 17-AAG (50 &#226;?&#174;M 100 &#226;?&#174;M) SNX-2112 (25 &#226;?&#174;M 50 &#226;?&#174;M) proteasome <br /> inhibitor bortezomib (25 &#226;?&#174;M 50 &#226;?&#174;M) 4 h prior lysis. DMSO vehicle control. Western blotting whole-cell lysates revealed mSmoM2 protein <br /> to destabilized response HSP-90 inhibitors response bortezomib. Tubulin loading control. CHOP results indicate ER stress. <br />  <br />  <br />  <br /> to thapsigargin. However, speculate result                       ezomib, induces ER stress blocking ERAD by <br /> increased transcriptional activity mSmo expression vectors                     engaging UPR, did destabilize mSmoM2 protein expres- <br /> the presence drug; wild-type M2 transcripts increased                    sion (Fig. 6D, lanes 6 7 compared lane 1). Conversely, <br /> modestly response drug treatment (Fig. 6B=).                                   17-AAG SNX-2112, induce UPR target- <br />    To determine small-molecule ER stress/UPR                           ing chaperone HSP90, decreased mSmoM2 protein levels <br /> modulators similar effects, treated mSmoM2-ex-                         (lanes 2 5 compared lane 1), supporting sensitiv- <br /> pressing NIH 3T3 cells compounds currently                       ity active Smo mutants UPR induction. <br /> being evaluated clinical use: HSP90 inhibitors 17-AAG                        Our vivo vitro results far suggest active Smo <br /> SNX-2112 proteasome inhibitor bortezomib (27). Bort-                         mutants retained ER sensitive ER stressors <br />  <br />  <br />  <br /> had identical mobilities, suggesting lack post-ER glycosylation (deglycosylated; A). &#226;?&#173;-Phosphatase did affect mobility, indicating wild-type <br /> mSmo phosphorylated absence Shh (A=, lane 10). Tubulin loading control. (B) Oncogenic mSmoM2 largely ER retained. Lysates from <br /> NIH 3T3 cells expressing wild-type M2 mSmo proteins treated deglycosylating agents described panel A. A significant pool endo H-resistant <br /> post-ER protein evident wild-type mSmo. The bulk mSmoM2 endo H sensitive. The post-ER pool mSmoM2 modest detectable (lane <br /> 6, post-ER <span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span>). Tubulin used loading control. (B=) Indirect immunofluorescence mSmoM2 NIH 3T3 cells demonstrated that, pool <br /> of mSmoM2 (green) detected primary cilium (ciliary slice [arrow]), bulk protein colocalized ER-resident protein GRP94 (ER slice <br /> [red]). 4=,6-diamidino-2-phenylindole DAPI (blue) marks nucleus. Bar, 20 &#226;?&#174;m. (C C=) Active mSmo mutants temperature sensitive. (C) NIH 3T3 <br /> cells grown 37&#194;&#176;C &#226;&#172;&#402;44 h maintained 37&#194;&#176;C shifted 40&#194;&#176;C additional 4 h prior lysis, indicated. Induction ER stress sensor <br /> CHOP assessed Western blotting whole-cell lysates. Tubulin served loading control. (C=) NIH 3T3 cells expressing wild-type, C318A, M2 mSmo <br /> proteins cultured panel C. Whereas ER (white arrowhead) post-ER (black arrowhead) forms wild type Smo significantly affected by <br /> temperature shift, mutants destabilized high temperature. Destabilization ER-resident forms wild type mutants 40&#194;&#176;C <br /> was attenuated treating cells proteasome inhibitor MG132, suggesting cleared ERAD. The post-ER form wild type Smo was <br /> unaffected MG132 treatment. Tubulin loading control (con). (D D=). ERAD attenuation stabilizes mutant mSmo proteins. NIH 3T3 cells <br /> expressing wild-type, C318A, M2 mSmo proteins transfected control Hrd1 siRNA indicated. Cells shifted 40&#194;&#176;C 4 h prior lysis. <br /> Hrd1 knockdown stabilized ER-retained mutant mSmo proteins (D, lanes 1 6, compared lanes 7 12) rescued mSmoM2-mediated induction of <br /> endogenous Gli1 (D=, lanes 1 2 compared lane 4). Vec (D=, lane 5) vector control lysate. Tubulin loading control. <br />  <br />  <br />  <br /> June 2013 Volume 33 Number 12                                                                                                                      mcb.asm.org 2383 <br />  Marada et al. <br />  <br />  <br />  <br />  <br /> FIG 7 Thapsigargin attenuates ectopic signaling active Smo mutant vivo. (A) Transgene expression unaffected thapsigargin. MS1096-GAL4, <br /> UAS-EGFP (MS1096&#226;&#172;&#381;EGFP) larvae grown media containing vehicle thapsigargin, indicated. Wing imaginal discs third-instar larvae under <br /> both conditions demonstrated comparable GFP expression (green). 4=,6-Diamidino-2-phenylindole (DAPI; magenta) marks nuclei. (B E) Thapsigargin <br /> prevents Myc-SmoC320A-induced Hh gain-of-function wing phenotypes. Larvae expressing wild-type C320A UAS-myc-smo control MS1096- <br /> Gal4 C765-Gal4 grown 22&#194;&#176;C food containing vehicle (C E) thapsigargin (C= E=). Representative wings adult flies shown. The <br /> MS1096-Gal4 driver wing served control (B). Thapsigargin did affect wings expressing wild-type Smo (C C=). The C320A-induced phenotype was <br /> significantly attenuated drug, allowing development near-normal adult wings (D=and E=, compared D E). (F G) SmoC320A-induced <br /> downstream pathway activity attenuated thapsigargin. Wing imaginal discs dissected myc-smoC320A-expressing third-instar larvae grown on <br /> vehicle-containing (F G) thapsigargin-containing (G=) food 22&#194;&#176;C. UAS-myc-smoC320A expressed using dorsal compartment wing driver <br /> Apterous (Ap)-Gal4. Wing discs immunostained Myc-SmoC320A (anti-Smo; blue), full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> Ci (green), &#226;?&#164;-galactosidase (dpp-lacZ; magenta). Note <br /> the significantly reduced SmoC320A protein level, reduced Ci stabilization, decreased &#226;?&#164;-galactosidase signals thapsigargin-treated discs (G compared to <br /> G=; wing pouch shown). The Ap-Gal4 driver wing disc (F) served control. For wing disc images, discs shown dorsal anterior to <br /> the left. Bar, 100 &#226;?&#174;m. (H H=) Thapsigargin does attenuate signaling induced aberrant Hh ligand production. Hhmrt larvae grown 22&#194;&#176;C food <br /> containing vehicle (H) thapsigargin (H=). Thapsigargin did alter gain-of-function phenotype induced Hhmrt allele. Representative wings from <br /> adult flies shown. <br />  <br />  <br />  <br /> that enhance protein misfolding induce UPR, e.g., high                    gene control MS1096-Gal4 wing pouch driver <br /> temperature, thapsigargin, HSP90 inhibitors (46, 47). Wild-                   (Fig. 7A). GFP expression unaffected thapsigargin treat- <br /> type Smo significantly targeted UPR following                ment 22&#194;&#176;C, confirming drug does nonspecifically <br /> thermal chemical ER stress induction, suggesting small-                   alter transgene expression vivo. We expressed wild- <br /> molecule-mediated UPR modulation represent selective                        type C320A Myc-Smo proteins 22&#194;&#176;C control of <br /> process target active Smo mutants disease. To test                 MS1096-Gal4 C765-Gal4 drivers grew larvae food con- <br /> this hypothesis, examined thapsigargin alter                     taining vehicle thapsigargin (Fig. 7C E). Consistent <br /> mutant Smo-induced Drosophila wing phenotypes. Due                         vitro results, wings wild-type Myc-Smo-express- <br /> high degree pupal lethality induced Myc-SmoC339A                        ing flies unaffected thapsigargin treatment (Fig. 7C and <br /> mutant 22&#194;&#176;C, chose perform assay using Myc-                     C=, compared B [control]). Conversely, Myc-SmoC320A-ex- <br /> SmoC320A mutant.                                                                  pressing thapsigargin-fed larvae demonstrated significantly re- <br />    To confirm thapsigargin treatment transgenic Drosophila                duced Hh gain-of-function phenotype compared the <br /> would nonspecifically alter UAS/GAL4 transgene expression                     vehicle-fed control (Fig. 7D E compared D= E=). Con- <br /> in wing imaginal discs, tested effect feeding thap-               sistent observed effects wing phenotypes, thapsigargin <br /> sigargin vehicle control larvae expressing GFP trans-                     treatment resulted significant reduction Myc-SmoC320A <br />  <br />  <br /> 2384   mcb.asm.org                                                                                                                 Molecular Cellular Biology <br />                                                                                                         Active Smoothened Mutants Are UPR Substrates <br />  <br />  <br />  <br /> protein wing imaginal discs dissected drug-fed larvae (Fig.       response. However, argue observed effects not <br /> 7G compared G= [blue]). This sufficient attenuate down-         solely high-level Smo expression, detect <br /> stream signaling, C320A-mediated ectopic stabilization          activation ER stress responses transgenic Drosophila in <br /> Hh pathway transcriptional effector Ci (green) induction           cultured cells overexpressing active Smo mutants. Furthermore, <br /> the dpp reporter gene (magenta) reduced response          overexpressed wild-type Smo protein localizing ER fraction <br /> drug (Fig. 7G compared G= F [control]).                            eliminated UPR response ER stress. As such, <br />     To determine thapsigargin effective Hh             speculate inducing ER stress Smo-expressing cells, <br /> pathway activity induced aberrant ligand production,            wild-type Smo protein continues fold properly, does engage <br /> than mutant Smo signaling, tested attenuation phe-           UPR, exits ER normal secretory route. <br /> notypes induced Hhmrt allele. Hhmrt causes ectopic expres-         Conversely, moderately misfolded active Smo mutants readily <br /> sion Hh ligand dorsal/ventral border devel-       detected active UPR, leading elimination through <br /> oping wing imaginal disc, triggering robust overgrowth             ERAD. As such, observed destabilization signaling atten- <br /> distal portion anterior wing blade (Fig. 7H) (48, 49). In          uation active Smo mutants following ER stress induction and <br /> contrast observed flies expressing active Smo        UPR engagement results molecular mechanism spe- <br /> mutant, thapsigargin feeding did attenuate Hhmrt pheno-           cific mutant Smo protein, ideal scenario clinical efficacy. <br /> type (Fig. 7H=). Taken together, results suggest altera-       Although know relative abundance oncogenic <br /> tion ER homeostasis induction UPR block              SmoM2 protein human tumor cell, hypothesize its <br /> ligand-induced pathway activity represent selective op-         altered conformation atypical ER exit (17, 28) result in <br /> tion attenuating aberrant Smo signaling disease. <br />                                                                           SmoM2 having extended ER retention time, sensitizing <br /> DISCUSSION                                                                active ER stress response. <br />                                                                               Work rhodopsin suggests recurring <br /> We demonstrated signaling active Smo mutants localiz- <br />                                                                           theme G-protein-coupled receptor biology. Studies performed <br /> ing ER attenuated conditions hyperthermal- or <br />                                                                           using Drosophila model autosomal dominant retinitis pig- <br /> small molecule-mediated ER stress. Our studies focused Smo pro- <br /> teins harboring activating C-to-A mutations extracellular loop 1       mentosa, closely mimics human disease, revealed that <br /> and transmembrane domain 3 oncogenic M2 mutation,               sustained ER stress protects retinal degeneration triggered <br /> W-to-L alteration transmembrane domain 7 (3, 12). Each        ER-retained disease-causing rhodopsin mutants (53, 54). Al- <br /> mutations likely induces Smo conformational shift mimics       mechanism chronic, low-level ER stress fa- <br /> Hh-induced active conformation, triggering ligand-inde-           cilitates protection clear, recent study suggested mild <br /> pendent signaling (12, 28). We speculate that, despite active con-   ER stress activate autophagy resist apoptosis (55). Inter- <br /> formation, mutant protein recognized ER mis-       estingly, flies heterozygous rhodopsinG69D disease-caus- <br /> folded, resulting prolonged ER retention. We attempted exploit      ing mutation, ER stress-induced ERAD specifically degraded mu- <br /> this phenomenon specifically target active Smo mutants,         tant rhodopsin did target wild-type protein (53). <br /> were successful attenuating mutant Smo protein stability           Importantly, studies, mutant allele expressed at <br /> downstream signaling, vitro vivo, inducing       endogenous levels endogenous promoter (36, 53). <br /> thermal chemical ER stress.                                                Based results presented here, speculate the <br />     Given significant identifying novel methods        UPR holds promise rapidly expand cache small molecules <br /> targeting aberrant Hh signaling disease (50&#226;&#8364;&#8220;52), propose            available treatment Hh-dependent cancer. Notably, num- <br /> that findings clinical relevance, particularly          ber ER stress- UPR-modulating compounds, including a <br /> cases disease results Smo mutation. Previous vitro          thapsigargin-derived prodrug, approved currently <br /> studies demonstrated reduced sensitivity oncogenic            evaluated clinical use treating cancers leuke- <br /> M2 mutant small-molecule Smo inhibitors (28). More signifi-            mia, lymphoma, multiple myeloma, prostate cancer (27, 56). <br /> cantly, acquired resistance FDA-approved Smo inhib-           As such, future studies reveal UPR-modulating compounds be <br /> itor, GDC-0449 (vismodegib), observed clinic              efficacious targeting Smo-driven malignancies, ability trans- <br /> (29). In case, resistance conferred novel Smo mutation       late findings clinic expedited. This prove to <br /> that attenuates ability compound bind Smo            significant advantage novel Hh pathway-specific small mol- <br /> inhibit signaling activity (30). As such, remains clinical    ecules approved clinical use. <br /> need additional methods targeting Hh pathway, in <br /> frontline combination therapy salvage therapy relapsed          ACKNOWLEDGMENTS <br /> patients develop resistance current Smo antagonists. We <br /> propose compounds affecting normal folding environ-              This work supported research grants MOD 5-FY10-6 and <br />                                                                           1R01GM101087 (S.K.O.), NIH/NCI Cancer Center Core Support <br /> ment ER niche specifically targeting con- <br />                                                                           5P30CA021765 (St. Jude), ALSAC SJCRH. <br /> formationally unstable Smo mutants. Such compounds of- <br />                                                                               We thank Y. Ahmed, T. Gruber, L. Hendershot thoughtful <br /> fer significant advantage Smo-specific small molecules,            discussion comments manuscript. L. Hendershot provided <br /> because UPR modulators exploit cellular process distinct        GRP94 antibody. D. Casso provided Cal-GFP-KDEL Hhmrt flies. H. <br /> from Hh signaling pathway. As such, efficacy          Steller provided Xbp1-GFP flies. Xbp1k13803, Gal4 driver, UAS-GFP <br /> unaltered acquired Smo mutation.                                    stocks obtained Bloomington Stock Center. We grateful <br />     Admittedly, current studies performed overex-           SJCRH Hartwell Center Cell Tissue Imaging Center for <br /> pressed Smo, membrane protein that, highly expressed,           expert assistance. <br /> likely accumulate ER induce baseline ER stress                We declare conflicts interest. <br />  <br />  <br /> June 2013 Volume 33 Number 12                                                                                                   mcb.asm.org 2385 <br />  Marada et al. <br />  <br />  <br />  <br /> REFERENCES                                                                               Involvement basic leucine zipper transcription factors. J. Biol. Chem. <br />  1. Jiang J, Hui CC. 2008. Hedgehog signaling development cancer.                 273:33741&#226;&#8364;&#8220;33749. <br />     Dev. Cell 15:801&#226;&#8364;&#8220; 812.                                                         24.   Tirasophon W, Lee K, Callaghan B, Welihinda A, Kaufman RJ. 2000. <br />                                                                                          The endoribonuclease activity mammalian IRE1 autoregulates its <br />  2. Barakat MT, Humke EW, Scott MP. 2010. Learning Jekyll to <br />                                                                                          mRNA required unfolded protein response. Genes Dev. <br />     control Hyde: Hedgehog signaling development cancer. Trends <br />                                                                                          14:2725&#226;&#8364;&#8220;2736. <br />     Mol. Med. 16:337&#226;&#8364;&#8220;348. <br />                                                                                    25.   McCracken AA, Brodsky JL. 1996. Assembly ER-associated protein <br />  3. Xie J, Murone M, Luoh SM, Ryan A, Gu Q, Zhang C, Bonifas JM, Lam <br />                                                                                          degradation vitro: dependence cytosol, calnexin, ATP. J. Cell <br />     CW, Hynes M, Goddard A, Rosenthal A, Epstein EH, Jr, Sauvage FJ. <br />                                                                                          Biol. 132:291&#226;&#8364;&#8220;298. <br />     1998. Activating Smoothened mutations sporadic basal-cell carcinoma.        26.   Liu Y, Ye Y. 2011. Proteostasis regulation endoplasmic reticulum: <br />     Nature 391:90 &#226;&#8364;&#8220;92.                                                                   new perturbation site targeted cancer therapy. Cell Res. 21:867&#226;&#8364;&#8220; 883. <br />  4. Fan H, Oro AE, Scott MP, Khavari PA. 1997. Induction basal cell             27.   Li X, Zhang K, Li Z. 2011. Unfolded protein response cancer: the <br />     carcinoma features transgenic human skin expressing Sonic Hedgehog.               physician&#226;&#8364;&#8482;s perspective. J. Hematol. Oncol. 4:8. <br />     Nat. Med. 3:788 &#226;&#8364;&#8220;792.                                                          28.   Taipale J, Chen JK, Cooper MK, Wang B, Mann RK, Milenkovic L, <br />  5. Goodrich LV, Milenkovic L, Higgins KM, Scott MP. 1997. Altered                       Scott MP, Beachy PA. 2000. Effects oncogenic mutations in <br />     neural cell fates medulloblastoma mouse patched mutants. Science              Smoothened Patched reversed cyclopamine. Nature 406: <br />     277:1109 &#226;&#8364;&#8220;1113.                                                                      1005&#226;&#8364;&#8220;1009. <br />  6. Karhadkar SS, Bova GS, Abdallah N, Dhara S, Gardner D, Maitra A,               29.   Rudin CM, Hann CL, Laterra J, Yauch RL, Callahan CA, Fu L, <br />     Isaacs JT, Berman DM, Beachy PA. 2004. Hedgehog signalling pros-                  Holcomb T, Stinson J, Gould SE, Coleman B, LoRusso PM, Von <br />     tate regeneration, neoplasia metastasis. Nature 431:707&#226;&#8364;&#8220;712.                     Hoff DD, Sauvage FJ, Low JA. 2009. Treatment medulloblas- <br />  7. Watkins DN, Berman DM, Burkholder SG, Wang B, Beachy PA, Baylin                      toma hedgehog pathway inhibitor GDC-0449. N. Engl. J. Med. <br />     SB. 2003. Hedgehog signalling airway epithelial progenitors            361:1173&#226;&#8364;&#8220;1178. <br />     small-cell lung cancer. Nature 422:313&#226;&#8364;&#8220;317.                                    30.   Yauch RL, Dijkgraaf GJ, Alicke B, Januario T, Ahn CP, Holcomb T, <br />  8. Yuan Z, Goetz JA, Singh S, Ogden SK, Petty WJ, Black CC, Memoli VA,                  Pujara K, Stinson J, Callahan CA, Tang T, Bazan JF, Kan Z, Seshagiri <br />     Dmitrovsky E, Robbins DJ. 2007. Frequent requirement hedgehog                     S, Hann CL, Gould SE, Low JA, Rudin CM, Sauvage FJ. 2009. <br />     signaling non-small cell lung carcinoma. Oncogene 26:1046 &#226;&#8364;&#8220;1055.                  Smoothened mutation confers resistance Hedgehog pathway inhibi- <br />  9. Berman DM, Karhadkar SS, Maitra A, Montes De Oca R, Gerstenblith                     tor medulloblastoma. Science 326:572&#226;&#8364;&#8220;574. <br />     MR, Briggs K, Parker AR, Shimada Y, Eshleman JR, Watkins DN,                   31.   Wang Y, Davidow L, Arvanites AC, Blanchard J, Lam K, Xu K, Oza V, <br />     Beachy PA. 2003. Widespread requirement Hedgehog ligand stimu-                   Yoo JW, Ng JM, Curran T, Rubin LL, McMahon AP. 2012. Glucocor- <br />     lation growth digestive tract tumours. Nature 425:846 &#226;&#8364;&#8220; 851.                   ticoid compounds modify smoothened localization hedgehog path- <br /> 10. van den Heuvel M, Ingham PW. 1996. smoothened encodes receptor-                    way activity. Chem. Biol. 19:972&#226;&#8364;&#8220;982. <br />     like serpentine protein required Hedgehog signalling. Nature 382:547&#226;&#8364;&#8220;      32.   Ogden SK, Casso DJ, Ascano M, Jr, Yore MM, Kornberg TB, Robbins <br />     551.                                                                                 DJ. 2006. Smoothened regulates activator repressor functions of <br /> 11. Alcedo J, Ayzenzon M, Von Ohlen T, Noll M, Hooper JE. 1996. The                      Hedgehog signaling distinct mechanisms. J. Biol. Chem. 281: <br />     Drosophila smoothened gene encodes seven-pass membrane protein,                  7237&#226;&#8364;&#8220;7243. <br />     putative receptor hedgehog signal. Cell 86:221&#226;&#8364;&#8220;232.                    33.   Chen CH, von Kessler DP, Park W, Wang B, Ma Y, Beachy PA. 1999. <br /> 12. Carroll CE, Marada S, Stewart DP, Ouyang JX, Ogden SK. 2012. The                     Nuclear trafficking Cubitus interruptus transcriptional regula- <br />     extracellular loops Smoothened play regulatory role control               tion Hedgehog target gene expression. Cell 98:305&#226;&#8364;&#8220;316. <br />     Hedgehog pathway activation. Development 139:612&#226;&#8364;&#8220; 621.                         34.   Nachtergaele S, Mydock LK, Krishnan K, Rammohan J, Schlesinger PH, <br /> 13. Karnik SS, Sakmar TP, Chen HB, Khorana HG. 1988. Cysteine residues                   Covey DF, Rohatgi R. 2012. Oxysterols allosteric activators the <br />     110 187 essential formation correct structure bovine           oncoprotein Smoothened. Nat. Chem. Biol. 8:211&#226;&#8364;&#8220;220. <br />     rhodopsin. Proc. Natl. Acad. Sci. U. S. A. 85:8459 &#226;&#8364;&#8220; 8463.                     35.   Livak KJ, Schmittgen TD. 2001. Analysis relative gene expression data <br /> 14. Incardona JP, Gruenberg J, Roelink H. 2002. Sonic hedgehog induces               using real-time quantitative PCR 2&#226;&#171;&#186;&#226;&#338;&#172;&#226;&#338;&#172;CT method. Methods 25: <br />     segregation patched smoothened endosomes. Curr. Biol. 12:                  402&#226;&#8364;&#8220; 408. <br />     983&#226;&#8364;&#8220;995.                                                                       36.   Ryoo HD, Domingos PM, Kang MJ, Steller H. 2007. Unfolded protein <br /> 15. Chen JK, Taipale J, Cooper MK, Beachy PA. 2002. Inhibition Hedge-                 response Drosophila model retinal degeneration. EMBO J. 26:242&#226;&#8364;&#8220; <br />     hog signaling direct binding cyclopamine Smoothened. Genes                  252. <br />     Dev. 16:2743&#226;&#8364;&#8220;2748.                                                             37.   Shen Y, Meunier L, Hendershot LM. 2002. Identification character- <br />                                                                                          ization novel endoplasmic reticulum (ER) DnaJ homologue, which <br /> 16. Wong SY, Seol AD, So PL, Ermilov AN, Bichakjian CK, Epstein EH, Jr, <br />                                                                                          stimulates ATPase activity BiP vitro induced ER stress. J. <br />     Dlugosz AA, Reiter JF. 2009. Primary cilia mediate suppress <br />                                                                                          Biol. Chem. 277:15947&#226;&#8364;&#8220;15956. <br />     Hedgehog pathway-dependent tumorigenesis. Nat. Med. 15:1055&#226;&#8364;&#8220;1061. <br />                                                                                    38.   Iwawaki T, Akai R, Kohno K, Miura M. 2004. A transgenic mouse model <br /> 17. Hoffmeister H, Babinger K, Gurster S, Cedzich A, Meese C, Schaden- <br />                                                                                          monitoring endoplasmic reticulum stress. Nat. Med. 10:98 &#226;&#8364;&#8220;102. <br />     dorf K, Osten L, Vries U, Rascle A, Witzgall R. 2011. Polycystin-2 <br />                                                                                    39.   Duffy JB. 2002. GAL4 Drosophila: fly geneticist&#226;&#8364;&#8482;s Swiss army <br />     takes different routes somatic ciliary plasma membrane. J. Cell           knife. Genesis 34:1&#226;&#8364;&#8220;15. <br />     Biol. 192:631&#226;&#8364;&#8220; 645.                                                            40.   Aza-Blanc P, Ramirez-Weber FA, Laget MP, Schwartz C, Kornberg TB. <br /> 18. Han Y <br /> </body></html>