<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>1372&#226;&#8364;&#8220;1380 Nucleic Acids Research, 2011, Vol. 39, No. 4                                                         Published online 21 October 2010 <br /> doi:10.1093/nar/gkq981 <br />  <br />  <br /> Microhomology-mediated deletion gene <br /> conversion African trypanosomes <br /> Lucy Glover, Junho Jun David Horn* <br /> London School Hygiene Tropical Medicine, Keppel Street, London, WC1E 7HT, UK <br />  <br /> Received July 26, 2010; Revised September 28, 2010; Accepted October 4, 2010 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                        fungi mammals (1). Important roles for <br /> Antigenic variation African trypanosomes                                  microhomology-mediated end joining (MMEJ) other <br /> induced DNA double-strand breaks (DSBs). In                                  forms &#226;&#8364;&#732;alternative end-joining&#226;&#8364;&#8482; recently emerged <br /> these protozoan parasites, DSB repair (DSBR)                                 class switch recombination B cells (2) cancer <br />                                                                                 development (3,4). However, MMEJ revealed <br /> dominated homologous recombination (HR) and <br />                                                                                 NHEJ disrupted cells, pathway <br /> microhomology-mediated end joining (MMEJ),                                appears serve &#226;&#8364;&#732;back-up&#226;&#8364;&#8482; function (5). In <br /> non-homologous end joining (NHEJ)                                  contrast, MMEJ dominates end-joining reactions tryp- <br /> reported. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> facilitate analysis chromosomal                             anosomes (6), divergent protozoan parasites humans <br /> end-joining, established                                    livestock rely DSBR effective antigenic <br /> inter-allelic repair HR lethal loss                          variation immune evasion (7). This suggests that <br /> essential gene. Analysis intrachromosomal end                                MMEJ universally conserved pathway is <br /> joining individual DSBR survivors exclusively                                obscured suppressed organisms the <br /> revealed MMEJ-based deletions NHEJ.                                      capacity NHEJ. The prominence MMEJ tryp- <br /> A survey microhomologies typically revealed se-                              anosomes presents unique opportunity study the <br /> quences 5 20 bp <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>                                   features pathway. <br />                                                                                    End-joining mechanisms non-conservative, typically <br /> several mismatches tolerated longer stretches. <br />                                                                                 resulting sequence loss. NHEJ exploit <br /> Mean deletions 54 bp closest                                just paired nucleotides commonly results <br /> to break 284 bp total. Break proximity,                              loss &lt;10 bp break (1). MMEJ is <br /> microhomology length GC-content favored                                 distinct allows imperfect, directly repeated <br /> repair <span id='am-84' about='xsp:pattern' typeof='owl:Thing'>pattern</span> MMEJ described                                  sequence 5&#226;&#8364;&#8220;20 nt &#239;&#172;&#8218;anking break recombine fol- <br /> was similar different loci                                lowing annealing complementary strands each <br /> genome. We identified interchromosomal                                     repeated sequence (8). Single-strand annealing (SSA) is <br /> gene conversion involving HR MMEJ different                              non-conservative DSBR mechanism allows <br /> ends duplicated sequence. While MMEJ-based                                 extensive directly repeated sequences recombine <br /> deletions RAD51-independent, one-sided                                     (9). The products MMEJ SSA contain one <br /> MMEJ RAD51 dependent. Thus,                                     copy repeated sequence, deletion se- <br />                                                                                 quences originally present repeats. <br /> the features MMEJ Trypanosoma brucei, <br />                                                                                 Thus, analogy SSA, MMEJ known as <br /> which analogous micro single-strand anneal-                               micro-SSA idea supported genetic analysis <br /> ing; RAD51 dependent, one-sided MMEJ. We                                    (4,8,10,11). However, alternative end-joining, typically <br /> discuss contribution MMEJ pathways                                    de&#239;&#172;?ned genetically KU70/80 ligase 4 independent, <br /> genome evolution, subtelomere recombination                                 differ substantially different organisms (12); in <br /> antigenic variation.                                                            cases, little microhomology required, the <br />                                                                                 size deletions vary substantially insertions <br />                                                                                 observed. <br /> INTRODUCTION                                                                       DSBR DNA rearrangement central the <br /> Homologous recombination (HR) non-homologous                                process antigenic variation African trypanosomes <br /> end joining (NHEJ) make major contribution                               (7), mechanisms underlying process remain <br /> mitotic double-strand break-repair (DSBR) gen-                          partially characterized. Switching variant <br /> eration genetic diversity organisms ranging                          surface glycoprotein (VSG) coat depends upon <br />  <br />  <br />  <br /> *To correspondence addressed. Tel: +44 20 7927 2352; Fax: +44 20 7636 8739; Email: david.horn@lshtm.ac.uk <br />  <br /> &#195;&#376; The Author(s) 2010. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.5), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />                                                                                                           Nucleic Acids Research, 2011, Vol. 39, No. 4 1373 <br />  <br />  <br /> monoallelic expression VSG gene telomere (13)                                    To delete 2110b allele RsrII/XcmI fragment in <br /> and large reservoir subtelomeric VSG templates,                                  pARD-NEO (16) replaced RsrII/EcoRI <br /> can used copy new VSGs active telomeric                                     fragment pbRn5 (18) encompassing portion of <br /> expression site (ES). While key factors required NHEJ                                  NPT gene aldolase polyA signal. This <br /> are absent diverged trypanosomatids (14),                                            pAN HR construct digested SmaI/ApaI prior <br /> RAD51-independent MMEJ exogenous templates                                            transfection correct integration con&#239;&#172;?rmed by <br /> has described Trypanosoma brucei (15)                                    PCR. <br /> reactions shown KU-independent in <br /> in vitro assays (14). However, little known                                       DNA analysis <br /> chromosomal MMEJ contribute to <br />                                                                                            For Southern blot analysis DSBR, genomic DNA was <br /> VSG rearrangement. Here, characterize features <br />                                                                                            digested HindIII Bsp120I processed accord- <br /> of chromosomal MMEJ T. brucei. We robust <br />                                                                                            ing standard protocols. The 2110 probe 699 bp, <br /> RAD51-independent, MMEJ-based deletions also <br />                                                                                            SacI fragment pARD (16). A series chromosome <br /> RAD51 dependent, one-sided, MMEJ-based gene conver- <br />                                                                                            11, RFP, PAC TUB-speci&#239;&#172;?c primers used to <br /> sion. Our &#239;&#172;?ndings suggest pathways have <br />                                                                                            amplify sequence repair junctions, typically using <br /> contributed evolution compact genome and <br />                                                                                            Taq polymerase presence 1.5% DMSO. Direct <br /> to switching VSG gene expression, typically <br />                                                                                            sequencing PCR products carried according to <br /> involves recombination short, repetitive &#239;&#172;&#8218;anking <br />                                                                                            standard protocols. <br /> sequences. <br />  <br />                                                                                            RESULTS <br /> MATERIALS AND METHODS <br />                                                                                            MMEJ dominates repair chromosomal <br /> Trypanosoma brucei growth manipulation <br />                                                                                            end-joining assay <br /> Lister 427, MITat1.2 (clone 221a), bloodstream form cells <br />                                                                                            We previously used I-SceI meganuclease introduce <br /> were grown HMI-11 transformed described <br />                                                                                            single DSB T. brucei chromosome 11a (6); RsPa <br /> earlier (16). For limiting dilution cloning, cells were <br />                                                                                            strain used contains tetracycline-inducible I-SceI gene <br /> distributed 96-well plates analyzed if <br />                                                                                            single I-SceI cleavage site adjacent the <br /> &lt;40% wells displayed cell growth 5 days. <br />                                                                                            Tb11.02.2110 gene (Figure 1A). Using strain, &gt;50% <br /> Tetracycline Sigma used 1 mg ml 1. <br />                                                                                            cells survived I-SceI induction repaired break <br />                                                                                             85% survivors underwent allelic HR using <br /> Plasmid construction <br />                                                                                            2110b-allele repair template. In addition, al- <br /> Plasmid constructs expression tetracycline re-                                  ternatives allelic HR revealed; among <br /> pressor TUB locus (TetR-BLE), tet-on ex-                                      26-independent repair events, clones displayed <br /> pression I-SceI N-terminal SV40 nuclear                                         ectopic HR displayed MMEJ (6), insuf&#239;&#172;?cient <br /> ocalization signal rRNA spacer locus (I-SceI-                                       number detailed analysis. To facilitate isola- <br /> HYG) (17) integration RSP cassette                                       tion survivors display end joining charac- <br /> the Tb11.02.2110 locus (6) described previously.                                      terize MMEJ chromosomal context, devised an <br />  <br />  <br />                                                                                            B                          RSPa         R&#206;&#8221; <br />                        A            H                                              B                           WT <br />                                a:                              2110                                     Tet:         -     +   -         + <br />                                                       6 kbp <br />                                                                                                        kbp <br />                        WT                                                          B <br />                                                                                                          8 <br />                                                  H                                                       6                                       a <br />                                                                2110 <br />                                b: <br />                                                                                                          4                                       b <br />                                                                3.6 kbp <br />                                                                                                          3                         2110 <br />  <br />                                                                          DSB <br />                        RSPa   H <br />                                                                     XH                 X <br />                                                                                            C   DSB                         H                         B <br />                                                           2110           RFP:PAC                             ectopic HR <br />                                             5.2 kbp                                                                                  7.9 kbp <br />  <br />                                                                                                                                              H <br />                                                                                                TUB (1)       end joining   H <br />                                              H                                     B           RFP:PAC <br />                        RSPa / &#206;&#8221;2110b: (R&#206;&#8221;)                    NPT                                                              5.2 kbp <br />                                                                                                2110a <br />  <br /> Figure 1. An experimental study end-joining. (A) The schematic maps illustrate Tb11.02.2110 alleles wild-type (WT), RsPa R  <br /> strains. The meganuclease cleavage site embedded dsRed Fluorescent Protein (RFP)&#226;&#8364;&#8220;Puromycin ACetyltransferase (PAC) fusion gene. B, <br /> Bsp120I; H, HindIII; X, XcmI. (B). The R  strain validated Southern blotting WT RsPa controls. The R  RsPa strains were <br /> grown absence presence tetracycline (1 mg ml 1) 1 week. Genomic DNA digested Bsp120I HindIII. Bands representing <br /> the 2110 alleles indicated right. In RsPa strain, allelic HR regenerates 6 kb allele while, R  strain, ectopic HR end <br /> joining generate allele fragments 7.9 5.2 kbp, respectively; fragment sizes (A) (C). (C) The schematic maps illustrate result of <br /> ectopic HR end joining expected predominate RsPa/ 2110b survivors. The TUB sequences &#239;&#172;&#8218;anking RsP cassette promote RsP <br /> pre-mRNA trans-splicing polyadenylation allow ectopic HR replaces RsP aTUB gene copied chromosome 1. <br />  1374 Nucleic Acids Research, 2011, Vol. 39, No. 4 <br />  <br />  <br />                          A                                                         B <br />                                               100 <br />                                                                                                       60 <br />  <br />  <br />                                                80                                                     50 <br />  <br />  <br />  <br />  <br />                                                                                    Number clones <br />                          Cloning efficiency                                                           40 <br />                                                60 <br />  <br />                                                                                                       30 <br />                                                40 <br />                                                                                                       20 <br />  <br />                                                20 <br />                                                                                                       10 <br />  <br />                                                 0                                                      0 <br />                                               Tet:   -            +   -        +                           allelic ectopic 1-sided <br />                                                                                                             HR       HR    MMEJ MMEJ NHEJ <br />                                                          R SP           R&#206;&#8221; <br />                                                                                                                  HR         end-joining <br /> Figure 2. MMEJ common R  survivors. (A) As expected, R  strain displays reduced cloning ef&#239;&#172;?ciency DSBR cell death <br /> after allelic HR. Data derived dilution cloning 96-well plates: Tet, n = 4; +Tet, n = 6. (B) R  survivors display ectopic HR, one-sided <br /> MMEJ MMEJ-based deletions determined DNA sequencing; n = 110. <br />  <br />  <br />  <br /> experimental eliminate survivors use                                               <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>survival</span> (Figure 2A). To distinguish different <br /> major repair pathway allelic HR. This achieved                                                 repair pathways quantify relative contribution <br /> by replacing 2110b-allele break-adjacent                                                pathway R  cells, derived panel of <br /> homology NEO selectable marker (Figure 1A). In                                                 survivor clones. This approach favored analysis <br /> the resulting RsPa/ 2110b (R ) strains, allelic HR                                                mixed populations DNA ampli&#239;&#172;?cation is <br /> expected cause loss heterozygocity, loss                                                 required access sequence end-joining junctions <br /> remaining 2110a allele and, encoded                                                       (see below) ampli&#239;&#172;?cation multiple, related DNA <br /> N-terminal protein acetyltransferase essential                                                 fragments prone &#226;&#8364;&#732;template-switching&#226;&#8364;&#8482; artifacts and <br /> growth (16), cell death. Disruption single 2110 allele                                           ampli&#239;&#172;?cation bias. In addition, cloned-survivor <br /> had detectable impact growth rate R                                                  approach improves chance revealing repair junc- <br /> strains (data shown).                                                                             tions unanticipated locations clones initially <br />    To validate strains, devised Southern blot                                              fail reveal junction speci&#239;&#172;?cally targeted for <br /> analysis distinguish 2110 alleles                                              analysis. <br /> three modes repair seen previously. Genomic DNA                                                       We used limiting dilution I-SceI-inducing condi- <br /> was extracted Southern analysis prior DSBR                                                 tions generate panel 107 survivor clones. In order <br /> 1 week meganuclease induction DSBR.                                                         survey repair mechanisms, prepared genomic DNA <br /> Consistent previous &#239;&#172;?ndings, control RsPa                                                    complete set survivors. A series primer <br /> strain displayed modi&#239;&#172;?ed 2110a allele prior DSBR                                                 pairs, &#239;&#172;&#8218;anking RsP cassette, was <br /> and, DSBR, survivors displayed dominant allelic                                                 used amplify repair junctions PCR then <br /> HR reconstitution &#226;&#8364;&#732;wild-type&#226;&#8364;&#8482; allele                                                     sequenced directly ampli&#239;&#172;?ed products. The process <br /> (Figure 1A B). In contrast expected, R                                                 iterative, starting primers closer break and <br /> strain displayed modi&#239;&#172;?ed 2110a allele 2110b allele                                           progressing distal sites ampli&#239;&#172;?ed <br /> prior DSBR and, DSBR, survivors displayed                                                    products survivor; yielded pair of <br /> DNA fragments consistent ectopic HR end                                                      &#226;&#8364;&#732;repair fragments&#226;&#8364;&#8482; repair cells placed <br /> joining (Figure 1B C), allelic HR. Thus,                                                   replicated genomes one <br /> allelic HR leads cell death R  strain sub-                                            cell. Sequencing set 110 DNA fragments revealed 65 <br /> stantial proportion survivors display repair                                                   MMEJ-based deletions (Table 1), 41 cases ectopic HR, <br /> end-joining. A second independent R  strain displayed                                                 4 cases MMEJ-based gene conversion (see below), and <br /> similar results (data shown) &#239;&#172;?rst strain                                               NHEJ (Figure 2B). Thus, end joining exclusively <br /> selected detailed analysis.                                                                  microhomology mediated assay. <br />    We employed clonogenic assays determine pro- <br /> portion R  cells survive I-SceI-induced lesions                                               Microhomology pairing junction formation <br /> (Figure 2A). The RsPa strain served control                                             Our survey yielded 65 MMEJ junction sequences, data <br /> analysis and, consistent previous &#239;&#172;?ndings, indicated                                             set provides suf&#239;&#172;?cient information vari- <br /> &gt;50% survival. In contrast, consistent cell death                                            ous characteristics chromosomal MMEJ T. brucei. <br /> following allelic HR, R  strain displayed &lt;10%                                                    Three sequences revealed MMEJ using perfect 18-bp <br />                                                                                          Nucleic Acids Research, 2011, Vol. 39, No. 4 1375 <br />  <br />  <br /> Table 1. MMEJ junctions R  survivors <br />  <br /> Microhomology        Junction       Survivor(s)                                               Sum           RFP           PAC       Total  b <br /> (MH) class           typea                                                                    Survivors <br />  <br /> 1                    X              1, 13, 30, 33, 49, 54, 60, 66, 76, 83, 86, 87, 88, 91,    20                  52          20         81 <br />                                        93, 96, 98, 103, 104, 106 <br /> 2                    X              72                                                         1                165          19         192 <br /> 3                    X/x            68                                                         1                  6         224         240 <br /> 4                    X              3, 14c, 25, 28, 42, 81, 100, 107                           8                231          47         291 <br />                      xa             10, 17, 29, 31, 63, 64, 92                                 7                227          57         291 <br />                      xb             43, 94, 97, 102                                            4                217          64         290 <br />                      (xa)           58                                                         1                215          71         290 <br />                      (xb)           90                                                         1                225          61         291 <br />                      Mixed          8d (4xa/xb), 56 (4X/xa), 41 [4X/(xb)]                      3            217&#226;&#8364;&#8220;231       47&#226;&#8364;&#8220;64     290&#226;&#8364;&#8220;291 <br /> 5                    (x)            34                                                         1                 15         288         312 <br /> 6                    (x)            5                                                          1                  6         330         341 <br /> 7                    X              27                                                         1                129         327         465 <br /> 8                    X              50                                                         1                430          59         498 <br /> 9                    X              12, 53                                                     2                  8         504         523 <br />                      (x)            45                                                         1                  7         511         522 <br /> 10                   x              20, 74, 78                                                 3                 58         454         522 <br /> 11                   X              7, 79                                                      2                454          59         522 <br /> 12                   X              14c                                                        1                235         355         598 <br /> 13                   X              48, 82                                                     2                387         217         615 <br /> 14                   X              23                                                         1                379         561         948 <br /> XcmI                 X              40, 44, 85                                                 3                953         961        1914 <br /> a <br />   Junction types de&#239;&#172;?ned legend Figure 4. <br /> b <br />   The net loss bp ( ) includes 4 bp representing single-stranded overhangs left I-SceI cleavage. <br /> c <br />  Survivor 14 revealed independent MMEJ events. <br /> d <br />   The sequence survivor 8 shown Figure 4 &#226;&#8364;&#732;mixed&#226;&#8364;&#8482; junction generates frame-shifted, overlapping traces. <br />  <br />  <br />  <br /> microhomology created strain assembly. These se-                      similarly multiple genomic loci, sequences of <br /> quences (gCCAgtccttgtgTGGgt) contain XcmI sites                          MMEJ junctions PES survivors consist- <br /> (upper case characters) used assembly RsP                      ent interpretation (Supplementary Table S1). The <br /> construct 953- 961 bp                    contribution MMEJ survival, known the <br /> the break (Figure 1A). This longest                          RSPa, R  PES strains,  5 (6), 59 (Figure 2B) and <br /> break-distal microhomology observed (Table 1                          23%, respectively, indicating PCR assay quan- <br /> see below), indicating factors                             titative. The fainter banding pattern obtained the <br /> co-operatively promote MMEJ-associated annealing,                            rDNA TUB samples indicate the <br /> length microhomology proximity break.                          relative contribution MMEJ repair reduced at <br />    For remaining 62 sequences, plotted fre-                       tandem arrayed loci multiple adjacent and <br /> quency joining events use microhomology blocks                       allelic copies likely facilitate HR-based mechanisms. <br /> within 50 bp intervals physical map RsP                          Analysis microhomology-pairing survey-set <br /> cassette (Figure 3A). We illustrate frequency                    revealed 14 distinct classes (numbered MH1-14) with <br /> each microhomology pairing spanning DSB                                  MH1 MH4 accounting 70% events <br /> (Figure 3B). These data strong bias selection                  (Table 1). Microhomologies typically 5&#226;&#8364;&#8220;20 bp in <br /> microhomology closer DSB revealing mean dele-                         length mismatches tolerated longer <br /> tions 54 bp closest break                          stretches (Figure 4A). Inspection junction sequences <br /> 284 bp total. Deletions ranged 6- 561 bp                       revealed multiple possible outcomes single <br /> one 81- 948 bp total (see Table 1                    microhomology pairing exempli&#239;&#172;?ed MH4 which <br /> details).                                                                    yields &#239;&#172;?ve different junctions (Figure 4A). Four other <br />    Since template availability, chromatin structure                      classes revealed mixture possible junctions <br /> sequence context impact pattern repair;                      emerging single microhomology pairing (3X/x, <br /> next asked MMEJ-based repair similar dif-                      4X/xa, 4X/[xb] Figure 4A 4xa/xb Table 1) pre- <br /> ferent loci T. brucei genome. A PCR assay using                   sumably differential processing strand. <br /> primers &#239;&#172;&#8218;anking RsP cassette used amplify                            We asked <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> composition in&#239;&#172;&#8218;uences <br /> repaired fragments large population survivors                      MMEJ. The RFP PAC sequences contain 63 and <br /> (&gt;50 000). Using assay, examined pattern                      73% GC base pairs, respectively sum paired <br /> repaired fragments current RsPa R  strains                    bases sequences depicted Figure 4 724 (gray <br /> at loci: tubulin gene array (TUB),                    boxes) expected 492 GC-pairs (68%) no <br /> within rDNA spacer (19) subtelomeric                         bias. We observed 543 (75%) GC-pairs indicates <br /> VSG expression site (PES). A similar banding pattern                      GC base pairs signi&#239;&#172;?cantly (P &lt; 0.0001) favor pro- <br /> all cases (Figure 3C) suggested MMEJ operates                           ductive annealing (Figure 4B); presumably to <br />  1376 Nucleic Acids Research, 2011, Vol. 39, No. 4 <br />  <br />  <br />                          A                                                       RFP                                                   PAC <br />  <br />                                                     25 <br />  <br />  <br />  <br />                          Number joining events   20 <br />  <br />  <br />  <br />                                                     15 <br />  <br />  <br />  <br />                                                     10 <br />  <br />  <br />  <br />                                                      5 <br />  <br />  <br />  <br />                                                      0 <br />                          B 25 <br />  <br />  <br />                                                     20                                 MH4                     MH1 <br />                          Number joining events <br />  <br />  <br />  <br />  <br />                                                     15 <br />  <br />  <br />  <br />                                                     10 <br />  <br />  <br />  <br />                                                      5                                                            MH10 <br />                                                                                               MH13 <br />                                                                           MH11 <br />                                                                                                                                                    MH9 <br />  <br />                                                      0 <br />                                                                     -500            -250         DSB        +250                             +500 <br />                                                                                   Location microhomology (bp) <br />  <br />                                                                       RS Pa          R&#206;&#8221;        TUB         rDNA          PES <br />                                                          C <br />                                                                Tet:   -     +    -        +   -      +     -   +     -         + <br />                                                                                                                                    Deletion (bp) <br />                                                                bp <br />                                                              1500 <br />                                                                                                                                    81 (MH1) <br />                                                              1000                                                                  290-291 (MH4) <br />                                                               750                                                                  522-523 (MH9-11) <br />  <br /> Figure 3. The distribution microhomologies used repair. (A) Frequency microhomologies mapped 50-bp intervals of <br /> the DSB. (B) Frequency paired microhomologies mapped A. All microhomology (MH) classes represented &gt;1 junction indicated; <br /> see Table 1. (C) A PCR assay indicates similar pattern MMEJ different loci genome. Products corresponding frequent MH <br /> pairings size deletion indicated; Table 1. <br />  <br />  <br />  <br /> increased stability. Furthermore, observed 423/504                                                    One-sided MMEJ-based gene conversion <br /> (84%) GC-pairs MH1 repair, favored break                                                       The junction sequences survivor clones dis- <br /> proximity, excluded analysis (Figure 4B).                                                   played gene conversion-based interchromosomal repair <br /> These results indicate GC-rich microhomologies                                                  mechanism. This resulted allelic HR of <br /> long break-proximal microhomologies act coopera-                                                      RsP cassette MMEJ other, replacing RsP <br /> tively promote annealing MMEJ.                                                                    segment NPT cassette chromosome <br />                                                                                        Nucleic Acids Research, 2011, Vol. 39, No. 4 1377 <br />  <br />  <br />   A    1X                                               4X                                         5[x] <br />                    R GGAGCAGTACGAGCGCACCGAGGG <br />                      GGAGCAGTACAAGCCCACGGTGCG      20                                                     R CCTGTGCGGC-CCCGCGGATAGGG* <br />                    P CACCGAGTACAAGCCCACGGTGCG                                                               CCTGTGCGACCCCCGCGCATGGCC     1 <br />                                                                                                           P CGAGATCGG--CCCGCGCATGGCC <br />                                                         R TGTACCCCCGCGACGGCGTGCTGA <br />        2X                                                 TGTACCCCCGCGACGACGTCCCCA         8      6[x] <br />                    R CTGGTGGAGTTCAAGTCCATCTAC           P TCGCCACCCGCGACGACGTCCCCA <br />                      CTGGTGGAGTACAAGCCCACGGTG      1 <br />                    P CGCACCGAGTACAAGCCCACGGTG           4xa                                               R TCCTGTGCGGCCCCGCGGATAGGG* <br />                                                                                                             TCCTGTGCGGCCCCGCGGCAACAG     1 <br />                                                                                                           P TCCCGGCTGGCCGCGCAGCAACAG <br />       3X/x                        G/C                                                             7X      R CAAGAAGCCCGTGCAGCTGCCCGG <br />                                                         R TGTACCCCCGCGACGGCGTGCTGA                          CAAGAAGCCCGTGCAGCAACAGAT     1 <br />                                                           TGTACCCCCGCGACGGCGTCCCCA         7              P CCGGCTGGCCGCGCAGCAACAGAT <br />                                                         P TCGCCACCCGCGACGACGTCCCCA <br />                    R TCCTGTGCGGCCCCGCGGATAGGG*                                                     8X <br />                      TCCTGTGCGGCXCCGCGGTGGCGG      1    4xb <br />                    P CGGACGACGGCGCCGCGGTGGCGG <br />                                                                                                                        G <br />                                                                                                           R CTGTCCTTCCCCAGGGCTTCAAGT <br />                                                                                                             CTGTCCTTCCCCAGGGCCGTACGC     1 <br />                                                                                                           P GACGACGTCCCCAGGGCCGTACGC <br />                                                         R CCCGCGACGGCGTGCTGAAGGGCG <br />                                                           CCCGCGACGGCGTGCTGAAGGGCC         4              R TCCTGTGCGGCCCCGCGGATAGGG* <br />                                                         P CCCGCGACGACGTCC-CCAGGGCC                 9X       TCCTGTGCGGCCCCGCAACCTCCC     2 <br />                                                                                                           P GACCTCCGCGCCCCGCAACCTCCC <br />                                                         4[xa] <br />   B <br />                                                                                                           R TCCTGTGCGGCCCCGCGGATAGGG* <br />                       100                                                                         9[x]      TCCTGTGCGGCCCCGCGACCTCCC     1 <br />                                                         R CCCGCGACGGCGTGCTGAAGGGCGAGATC                   P GACCTCCGCGCCCCGCAACCTCCC <br />          GC content (%) <br />  <br />  <br />  <br />  <br />                                         *                 CCCGCGACGGCGTGCTGAAGGGCGATACG 1 <br />                                                         P CCCGCGACGACGTCC-CCAGGGCCGTACG                   R CGTGGAGCAGTACGAGCGCACCGA <br />                                    *                                                                    CGTGGAGGCGGCCGAGCGCGCCGG         3 <br />                                                         4[xb]                                     10x P AGTGGAGGCGGCCGAGCGCGCCGG <br />                           75 <br />  <br />                                                                                                           R AGCACCCCGCCGACATCCCCGACT <br />                                                                                                   11X       AGCACCCCGCCGACATCCCCAGGG     2 <br />                                                         R CCCGCGACGGCGTGCTGAAGGGCG                        P CCACCCGCGACGACGTCCCCAGGG <br />                                                           CCCGCGACGGCGTGCCCAGGGCCG         1 <br />                           50                            P CCCGCGACGACGTCCCCAGGGCCG <br />                                Ex d 1-14    2-14 <br />                                                                                                    12X <br />                                                                                                           R TGTACCCCCGCGACGGCGTGCTGA <br />                                                                                                             TGTACCCCCGCACCGGCCCAAGGA     1 <br />                                                         4X/xa     G/A <br />                                                                                                           P CCTGGCGCCGCACCGGCCCAAGGA <br />                                        Observed <br />                                                                                            1              R TTCGAGGACGGCGGCGTGGTGACC <br />                                                                                                    13X      TTCGAGGACGGCGCCGCGGTGGCG     2 <br />                                                                                                           P GCGGACGACGGCGCCGCGGTGGCG <br />                                                         4X/[xb]   G/A G/C <br />                                                                                                           R GGACGGCGGCGTGGTGACCGTGAC <br />                                                                                            1       14X      GGACGGCGGCGTGGTGCCCGAAGG     1 <br />                                                                                                           P CGCCGACGTCGAGGTGCCCGAAGG <br />  <br /> Figure 4. Microhomology classes; Table 1 details. (A) MMEJ junctions (sense strand only) shown 14 MH classes identi&#239;&#172;?ed. <br /> The parental RFP (R) PAC (P) sequences shown below, respectively. The template switch site indicated white lettering. <br /> The junction types follows: X, processed largest homology patch; x, processed &#226;&#8364;&#732;minor&#226;&#8364;&#8482; homology patch; [x] processed <br /> outside homology patches. Microhomologies highlighted ( 3 bp patches plus 2 bp patches 1 bp larger patch). Sequence traces <br /> are shown [x] junctions, 3X/x, 8X MH4 sub-classes (see text), numbers sub-class recorded indicated right. <br /> Asterisk MH3, 5, 6 9 indicates I-SceI cleaved terminus. (B) GC-content over-represented microhomology patches. *P &lt; 0.0001 as <br /> determined using  2 test. <br />  <br />  <br />  <br /> 11b (Figure 5A). A second PCR ampli&#239;&#172;?cation frag-                         one-sided MMEJ-based gene conversion speci&#239;&#172;?cally <br /> ments spanning gene conversion tract con&#239;&#172;?rmed                          ablated rad51 null strain (Figure 6A, lower panel). <br /> the expected size case (Figure 5B) sequencing                   Sequencing con&#239;&#172;?rmed major products detected <br /> indicated clones 4 71 used related                               using assay wild-type RAD51 cells represented <br /> microhomology, clones 24 73 used                              one-sided MMEJ (Figure 6B C). Thus, one-sided <br /> same microhomology generated different junctions                        MMEJ-based gene conversion use allelic ectopic <br /> (Figure 5C).                                                                homology chromosome 1. The RAD51-requirement <br />    MMEJ typically RAD51-independent. To assess                       suggests gene conversion initiated HR within <br /> role RAD51 chromosomal MMEJ-based deletion                            tubulin sequence resolved MMEJ. We also <br /> and gene conversion, applied PCR assay popula-                      analyzed RsPa survivor previously failed <br /> tions survivors RsPa strains wild-type RAD51                   identify repair mechanism [see Figure 5B, lane 4 in <br /> expression rad51 disrupted. An assay                            (6)] survivor arisen <br /> MMEJ-based deletions indicated robust activity                       one-sided ectopic MMEJ (Figure 6B C, iii). <br /> absence RAD51 (Figure 6A, upper panel),                              Segments copied chromosome 1 these <br /> sequencing rad51 survivors revealed exclusively                     RAD51-dependent, one-sided MMEJ reactions ranged <br /> MMEJ-based deletions (data shown). In contrast,                         28 1084 bp. <br />  1378 Nucleic Acids Research, 2011, Vol. 39, No. 4 <br />  <br />  <br />        A                                    DSB <br />                                                                              single-stranded tails maintenance mis- <br />                    11a:         2110       RFP:PAC                           matched base pairs DNA replication cycle. <br />                             24 / 73        4 / 71 <br />                                                                                 A DSB initiates homology search which, successful, <br />                                                                              results HR. This search homology typically suc- <br />                                                                              cessful diploid loci T. brucei non-conservative <br />                    11b:              NPT <br />                                                                              MMEJ generates deletion. Using experimental <br />                                             clones                           system,  5% cells survive DSBR MMEJ a <br />        B <br />                           Mwt <br />                                 wt <br />                                                                              single copy diploid locus increased  60% <br />                                        4    24 71    73 <br />                                                                              R  strains allelic HR lethal. A bias <br />                    kbp <br />                                                                              deletions associated DSBR end joining <br />                      2                                                       trypanosomatids major contribution <br />                                                                              genome evolution. Speci&#239;&#172;?cally, MMEJ-mediated dele- <br />                    1.5                                                       tions, mean size 284 bp assays, be <br />                                                                              responsible remarkably compact organization of <br />                                                                              trypanosomatid genomes, comprised  50% <br />        C                                                                     protein-coding sequence. MMEJ responsible <br />                                                                              synteny gaps trypanosomatid <br />          4   TUB     GCTGTGTGCGTGTGTGTGTGTGTGTGTGAG <br />                      GCTGTGTGCGTGTGTGTGTGTTGTTGTTGT                          chromosomes re&#239;&#172;&#8218;ect loss redundant genes <br />              ALD     AGTGTGTGTGTGTGTGTGTGTTGTTGTTGT                          lineage (20). <br />                                                                                 Interestingly, subtelomeric VSG expression site, <br />          71 TUB      GTGTGCGTGTGTGTGTGTGTGTGTGAGCCA                           25% cells survive DSBR using MMEJ-based repair. <br />                      GTGTGCGTGTGTGTGTGTGTGTGTGTTGTT <br />              ALD     AAATAAGTGTGTGTGTGTGTGTGTGTTGTT                          This increased contribution DSBR compared to <br />                                                                              loci re&#239;&#172;&#8218;ect reduced competition allelic <br />              24 2110 AGCTCCGGAAAGGGGCGGCGACAGC                               HR equally explained feature of <br />                      AGCTCCGGAAAGGGGCGGCGAACCA <br />                 ALD CACCCCGGAAATGGTCAGCGAACCA                                subtelomeric chromatin. To explore contribu- <br />                                                                              tion microhomology DNA rearrangement at <br />              73 2110 AGCTCCGGAAAGGGGCGGCGACAGC                               subtelomeric loci, examined available VSG expres- <br />                      AGCTCCGGAAATGGTCAGCGAACCA                               sion site sequences thought re&#239;&#172;&#8218;ect host&#226;&#8364;&#8220; <br />                 ALD CACCCCGGAAATGGTCAGCGAACCA <br />                                                                              parasite interactions (21). Remarkably, identi&#239;&#172;?ed <br /> Figure 5. One-sided MMEJ-based gene conversion. (A) The schematic            examples putative microhomology-based <br /> map illustrates allelic one-sided MMEJ events; gray box indi-       deletion (Supplementary Figure S1). In cases, <br /> cates HR lines indicate locations microhomologies,         putative parental sequences multiple <br /> the aldolase (ALD) processing sequence (cross-hatched box) linked to <br /> the NPT gene case. In clones 24 73, terminal 21 codons       subtelomeres and, consistent frequent recombination, <br /> of 2110 gene replaced 11 new codons stop codon;           putative derived sequences multiple <br /> presumably allowing expression functional protein. (B) PCR          subtelomeres. Retention junction sequences <br /> ampli&#239;&#172;?cation fragments spanning gene conversion tract reveals       suggests deletions occurred relatively recently. <br /> product expected size case; wild-type (wt) cells serve a <br /> control. The locations primers indicated A (small arrows). <br />                                                                              At 12&#226;&#8364;&#8220;18 bp though, deleted segments short <br /> (C) The one-sided allelic MMEJ junction sequences shown;            relative described above. We noted that <br /> other details Figure 4A.                                               larger deletions involving loss ESAGs have <br />                                                                              resulted SSA mediated &#239;&#172;&#8218;anking homology; <br />                                                                              ESAG3 (pseudo)genes case ESAG4/8 deletion <br /> DISCUSSION                                                                   BES12 BES14 example. <br />                                                                                 Approximately 30 copies tubulin sequence are <br /> In SSA, repeated sequences &#239;&#172;&#8218;anking DSB able to <br />                                                                              present tandem chromosome 1 presence of <br /> anneal single-stranded regions exposed.                             tubulin gene segments &#239;&#172;&#8218;anking reporter cassette <br /> Resolution requires processing digestion                           allowed ectopic HR chromosomes 1 11. <br /> single-stranded tails gap &#239;&#172;?lling (9). MMEJ                       This revealed 41 (37%) repair events allelic HR <br /> known micro-SSA results provide insight                           eliminated clearly demonstrates capacity to <br /> into equivalent processing steps T. brucei MMEJ.                      search genome wide suitable homologous templates <br /> For example, MH4-type events report indicate                          DSBR T. brucei. In addition, subset inter- <br /> removal single-stranded tails &#239;&#172;?ve alternative loca-                    actions initiated one-sided MMEJ. These robust ectopic <br /> tions. In case, cleavage digestion                            recombination pathways re&#239;&#172;&#8218;ect mechanisms are <br /> single-stranded tail common 9 3 bp                       important subtelomeric interactions underpin <br /> microhomology patch. Comparison MH1-type                                VSG recombination antigenic variation. <br /> events, processing exclusively longer                       So contribution does MMEJ make antigenic <br /> microhomology patch, suggests nearby,                                 variation? A switch VSG expression involves replace- <br /> out-of-register 5 bp patch microhomology (Figure 4A,                      ment previously active VSG single active <br /> 4xb) distorts paired sequence promotes processing                    telomeric ES (13) proceeds RAD51 dependent <br /> at different locations. In addition, junction sequences                 independent mechanisms (22) fully under- <br /> revealed mixture possible outcomes (3X/x, 4X/xa,                    stood. This typically interchromosomal gene conver- <br /> 4X/[xb] 4xa/xb), suggests staggered processing                     sion process relies imperfect homology within <br />                                                                                           Nucleic Acids Research, 2011, Vol. 39, No. 4 1379 <br />  <br />  <br />                           A        RAD51              rad51                 B                             DSB <br />                                                                                                       x <br />                           kbp      -      +       -      +      Tet               11a:     2110        RFP:PAC <br />                                                                                                                 y <br />                            1.5 <br />                                                                                                               ii <br />                                                                                                           iii <br />                             1                                                       1:                &#206;&#177;TUB <br />                                                                                                                  z <br />  <br />                                                                             C <br />                            0.5                    PCR: x + y <br />                                                                                   RFP AAGCACCCCGCCG <br />                                                                                         AAGCACCCCGAGC <br />                                                                                     TUB TTCCACCCCGAGC <br />                           1.5                                              ii     RFP GCCCCGCGGATAGGG* <br />                                                                                         GCCCCGCGGATATGG <br />                               1                                                     TUB AGTCCGCGGATATGG <br />                                                                 ii           iii RFP TGTACCCCCGCGACGGCGTG <br />                                                                                      TGTACCCCCGCGACGGCAAG <br />                                                                                  TUB GCGACCCCCGCCACGGCAAG <br />                           0.5                    PCR: x + z <br />  <br /> Figure 6. One-sided MMEJ-based gene conversion RAD51 dependent. (A) PCR assays indicate similar pattern MMEJ-based deletion <br /> in RAD51 rad51 null strains (upper panel) one-sided MMEJ-based gene conversion ablated rad51 null strains (lower panel). <br /> The locations primers indicated B. (B) The schematic map illustrates ectopic one-sided MMEJ events. (i) (ii) <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Figure 6A (iii) (6); details Figure 5A. (C) The one-sided ectopic MMEJ junction sequences shown; details in <br /> Figure 4A. <br />  <br />  <br />  <br /> repetitive sequences upstream VSG genes (7); &#226;&#8364;&#732;70 bp&#226;&#8364;&#8482;                      repair mechanism trypanosomes. We de- <br /> repeats widely distributed subtelomeres.                      veloped strains facilitate studies chromosomal <br /> Break-induced replication (BIR), typically RAD51 de-                       MMEJ T. brucei proximity the <br /> pendent process Saccharomyces cerevisiae (23),                   break, number proportion matched bases and <br /> proposed mechanism telomere conversion based                         GC-content promote pairing MMEJ-based dele- <br /> VSG switching (7). Recent work T. brucei indicates                        tions. In addition, one-sided MMEJ can <br /> both RAD51-dependent -independent BIR sup-                          mediate interchromosomal gene conversion. <br /> pression RAD51-dependent pathway TOPO3a <br /> (24). Since demonstrate RAD51 dependent inde- <br />                                                                              SUPPLEMENTARY DATA <br /> pendent pathways, MMEJ excellent candidate for <br /> mediating 70 bp repeat recombination. Gene conversion                        Supplementary Data available NAR Online. <br /> tracts involving BIR two-sided recombination could <br /> be initiated, terminated MMEJ this <br />                                                                              ACKNOWLEDGEMENT <br /> could explain antigenic variation relatively insensi- <br /> tive regulation mismatch repair (25,26). It                    We thank LSHTM colleagues, Sam Alsford John <br /> interesting note respect translocations                      Kelly comments draft article. <br /> involved class switch recombination B cells (2), as <br /> well replication fork breakage-induced rearrangements <br /> in human cells (27), use micro-BIR pathway.                            FUNDING <br /> MMEJ-based equivalents major HR mechan-                         The Wellcome Trust (083648). Funding open access <br /> isms, gene conversion, BIR SSA, one-sided gene                       charge: The Wellcome Trust. <br /> conversion possible two, <br /> micro-SSA one-sided gene conversion, chromo-                        Con&#239;&#172;&#8218;ict statement. None declared. <br /> somal context T. brucei. Thus, gene conversions we <br /> describe re&#239;&#172;&#8218;ect important pathways VSG re-                          REFERENCES <br /> arrangement antigenic variation. However, will <br /> be challenging distinguish use                                1. Lieber,M.R. (2010) The mechanism double-strand DNA break <br />                                                                                    repair nonhomologous DNA end-joining pathway. Annu. <br /> microhomology longer tracts homology following                               Rev. Biochem., 79, 181&#226;&#8364;&#8220;211. <br /> recombination highly repetitive T. brucei                                 2. Yan,C.T., Boboila,C., Souza,E.K., Franco,S., Hickernell,T.R., <br /> sequences.                                                                         Murphy,M., Gumaste,S., Geyer,M., Zarrin,A.A., Manis,J.P. et al. <br />    MMEJ considered backup end-joining mechanism                               (2007) IgH class switching translocations use robust <br />                                                                                    non-classical end-joining pathway. Nature, 449, 478&#226;&#8364;&#8220;482. <br /> in cells NHEJ operates. Unusually T. brucei,                           3. Zhu,C., Mills,K.D., Ferguson,D.O., Lee,C., Manis,J., Fleming,J., <br /> MMEJ dominates end joining (6), little known                                Gao,Y., Morton,C.C. Alt,F.W. (2002) Unrepaired DNA <br />  1380 Nucleic Acids Research, 2011, Vol. 39, No. 4 <br />  <br />  <br />     breaks p53-de&#239;&#172;?cient cells lead oncogenic gene ampli&#239;&#172;?cation      16. Ingram,A.K., Cross,G.A. Horn,D. (2000) Genetic <br />     subsequent translocations. Cell, 109, 811&#226;&#8364;&#8220;821.                         manipulation indicates ARD1 essential <br />  4. McVey,M. Lee,S.E. (2008) MMEJ repair double-strand                 Na-acetyltransferase Trypanosoma brucei. <br />     breaks (director&#226;&#8364;&#8482;s cut): deleted sequences alternative endings.       Mol. Biochem. Parasitol., 111, 309&#226;&#8364;&#8220;317. <br />     Trends Genet., 24, 529&#226;&#8364;&#8220;538.                                           17. Glover,L., Alsford,S., Beattie,C. Horn,D. (2007) Deletion of <br />  5. Nussenzweig,A. Nussenzweig,M.C. (2007) A backup DNA                   trypanosome telomere leads loss silencing progressive <br />     repair pathway moves forefront. Cell, 131, 223&#226;&#8364;&#8220;225.                loss terminal DNA absence cell cycle arrest. Nucleic <br />  6. Glover,L., McCulloch,R. Horn,D. (2008) Sequence homology              Acids Res., 35, 872&#226;&#8364;&#8220;880. <br />     microhomology dominate chromosomal double-strand break            18. Horn,D. Cross,G.A. (1997) Position-dependent and <br />     repair African trypanosomes. Nucleic Acids Res., 36,                   promoter-speci&#239;&#172;?c regulation gene expression Trypanosoma <br />     2608&#226;&#8364;&#8220;2618.                                                                brucei. EMBO J., 16, 7422&#226;&#8364;&#8220;7431. <br />  7. Boothroyd,C.E., Dreesen,O., Leonova,T., Ly,K.I.,                      19. Glover,L. Horn,D. (2009) Site-speci&#239;&#172;?c DNA double-strand <br />     Figueiredo,L.M., Cross,G.A. Papavasiliou,F.N. (2009) A                breaks greatly increase stable transformation ef&#239;&#172;?ciency in <br />     yeast-endonuclease-generated DNA break induces antigenic                  Trypanosoma brucei. Mol. Biochem. Parasitol., 166, 194&#226;&#8364;&#8220;197. <br />     switching Trypanosoma brucei. Nature, 459, 278&#226;&#8364;&#8220;281.                20. El-Sayed,N.M., Myler,P.J., Blandin,G., Berriman,M., Crabtree,J., <br />  8. Decottignies,A. (2007) Microhomology-mediated end joining              Aggarwal,G., Caler,E., Rena <br /> </body></html>