<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>B RIEFINGS IN FUNC TIONAL GENOMICS . VOL 12. NO 4. 316 ^327                                                                            doi:10.1093/bfgp/elt017 <br />  <br />  <br />  <br />  <br /> Functional genomics Plasmodium <br /> falciparum using metabolic <br /> modelling analysis <br /> StepanTymoshenko, Rebecca D. Oppenheim, Dominique Soldati-Favre Vassily Hatzimanikatis <br /> Advance Access publication date 22 June 2013 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> Plasmodium falciparum obligate intracellular parasite leading cause severe malaria responsible tre- <br /> mendous morbidity mortality particularly sub-Saharan Africa. Successful completion P. falciparum <br /> genome sequencing project 2002 provided comprehensive foundation functional genomic studies this <br /> pathogen following decade. Over period, large spectrum experimental approaches deployed <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> improve expand scope functionally annotated genes. Meanwhile, rapidly evolving methods systems <br /> biology begun contribute global understanding various aspects biology patho- <br /> genesis malaria. Herein provide overview metabolic modelling, capability integrate <br /> information functional genomics studies P. falciparum guide future malaria research efforts the <br /> identification novel candidate drug targets. <br /> Keywords: Plasmodium falciparum; central carbon metabolism; systems biology; flux-balance analysis; constraint-based <br /> modelling; silico gene essentiality <br />  <br />  <br /> INTRODUCTION                                                                    &#226;&#8364;&#732;hypothetical proteins&#226;&#8364;&#8482; &#226;&#8364;&#732;conserved hypothetical <br /> According recent &#226;&#8364;&#732;World Malaria Report&#226;&#8364;&#8482;                               proteins&#226;&#8364;&#8482; [3]. These genes, especially re- <br /> the World Health Organization, malaria remains                                stricted Plasmodium apicomplexan species, are <br /> major healthcare issue, responsible                              particular unique nucleotide se- <br /> than 200 million cases hundreds thousands                             quences provide higher selectivity new anti- <br /> deaths 2010 [1]. An efficient cost-effect-                         parasitic drugs. However, absence tools for <br /> ive artemisinin-based treatment available,                           high-throughput gene knockdown RNAi [4], <br /> the emergence resistance malaria parasites,                          identification validation essentiality of <br /> result drug treatments, urges development                             genes parasite remains major bottleneck <br /> medicines new targets novel mechanisms                              [3]. Available experimental approaches establish- <br /> action. Despite tremendous research efforts                             ing gene essentiality P. falciparum cumbersome <br /> passing decade publication                                unique properties pathogen. <br /> Plasmodium falciparum genome sequence [2],                              First, possesses extremely AT-rich genome <br /> half genes remain annotated coding                          unusually low frequency of <br />  <br />  <br /> Corresponding author. Stepan Tymoshenko, Institute Chemical Engineering, Laboratory Computational Systems Biotechnology, <br /> E&#194;&#180;cole Polytechnique Fe&#194;&#180;de&#194;&#180;rale Lausanne, CH-1015, Switzerland. Tel: &#195;&#190;41 (0) 21 693 98 69; Fax: &#195;&#190;41 (0) 21 693 98 75; <br /> E-mail:stepan.tymoshenko@epfl.ch <br /> StepanTymoshenko PhD student E&#194;&#180;cole Polytechnique Fe&#194;&#180;de&#194;&#180;rale Lausanne. He involved interdisciplinary model- <br /> driven research project aims elucidation poorly understood aspects metabolism Plasmodium falciparum. <br /> Rebecca Oppenheim PhD student University Geneva working central carbon metabolism apicomplexan <br /> parasites Toxoplasma gondii Plasmodium berghei. <br /> Dominique Soldati-Favre professor faculty Medicine University Geneva head laboratory focusing <br /> on motility, host cell attachment, invasion strategies allow intracellular proliferation apicomplexan parasites. <br /> Vassily Hatzimanikatis associate professor E&#194;&#180;cole Polytechnique Fe&#194;&#180;de&#194;&#180;rale Lausanne head Laboratory of <br /> Computational Systems Biotechnology. His research interests areas systems biology, bioinformatics, complexity of <br /> biological systems. <br />  <br /> &#195;&#376; The Author 2013. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which <br /> permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />                    Functional genomics P. falciparum using metabolic modelling analysis                       317 <br />  <br /> homologous recombination, makes refrac-              studies, silico metabolic models shown <br /> tory genetic manipulation gene replace-            provide context, enabling researchers <br /> ment [3]. Second, vitro cultivation delicate          use available datasets (i.e. content) improve <br /> process [5] mainly restricted intraer-   challenge models derive additional, <br /> ythrocytic stages. Third, limitation using pri-        non-intuitive insights. In particular, need an <br /> mates animal model makes invivo assessment          explicit, up-to-date context driving force for <br /> of gene essentiality limited expensive.              progress scope generic biochemical <br /> Nevertheless, emerging experimental breakthroughs             knowledge organism-specific (or life stage specific) <br /> hold promises cost-effective gene knockdown               metabolic networks, discuss further. <br /> strategies P. falciparum essential genes high-        In past decades, databases bio- <br /> throughput scale [6, 7]. In context, need     chemical reactions metabolic pathways been <br /> to list genes immediate interest,          developed provide systematic comprehensive <br /> validated antimalarial targets affordable ex-         overview metabolism. A pertinent example the <br /> perimental means available.                               Kyoto Encyclopedia Genes Genomes (KEGG) <br />     Metabolic modelling modern approach sys-          Pathway Database [16], established itself as <br /> tems biology that, applications,          encyclopedia biochemical knowledge the <br /> has extensively exerted predict gene essen-           point reference numerous academic and <br /> tiality various bacteria including number             industrial researches. However, KEGG similar <br /> pathogenic species [8]. Computational (i.e. silico)        large databases universal organism- <br /> metabolic models offer cost-effective pipeline           specific, cover crucial aspects <br /> identify putatively indispensable metabolic functions         P. falciparum metabolism, cofactor util- <br /> that, case pathogens, represent potential           ization, compartmentalization proteins involved <br /> targets medical intervention [9]. Eight years             transport metabolites. The web-resource <br /> after publication computational              Malaria Parasite Metabolic Pathways (MPMP) es- <br /> model tentative metabolic network P. falcip-          sentially product extensive manual revision <br /> arum, evident reconstruction           KEGG maps according known recently es- <br /> analysis silico models valuable tool study-    tablished metabolic features P. falciparum intraer- <br /> ing various aspects pathogen [10&#226;&#8364;&#8220;14]. In          ythrocytic life stage. Furthermore, integration with <br /> study, aim review provide outlook             web-resources facilitates quick access add- <br /> the current state contribution silico meta-         itional information related primary literature <br /> bolic modelling efforts functional genomics                [17]. MPMP probably comprehensive <br /> studies deadly malaria parasite. <br />                                                               up-to-date knowledge database P. falciparum&#226;&#8364;&#8482;s <br />                                                               metabolism, unfortunately lack appli- <br />                                                               cation-programming interface limits access a <br /> THE PATH <span id='am-2' about='protege:FROM' typeof='owl:Thing'>FROM</span> METABOLIC                                       broad research community high-quality data. <br /> MAPS TO &#226;&#8364;&#732;CONTEXT FOR                                          Utility MPMP research purposes been <br /> CONTENT&#226;&#8364;&#8482; MODELS                                               demonstrated interpretation expression pat- <br /> With constantly decreasing cost high-                  terns genes involved pentose-phosphate <br /> throughput measurements, tendency generate             pathway using corresponding maps [18]. This led <br /> very large -omics datasets emerging holds         authors postulate oxidative the <br /> also true P. falciparum [15]. There distinct      pathway predominantly active early stage <br /> approaches high-throughput measurements: hy-               intraerythrocytic replication cycle, the <br /> pothesis-driven studies generate large datasets            non-oxidative activated later <br /> prove falsify hitherto existing hypothesis,        stages [18]. In contrast approach, at- <br /> hypothesis-free studies primarily rely thorough          tempts comprehend metabolic fluxes malaria <br /> analysis datasets presumptions aiming           parasite aimed going study particu- <br /> the formulation conclusions testable hypoth-           lar pathway ultimately led reconstruction <br /> eses. Importantly, hypothesis-free approaches require         large-scale models hundreds metabolites <br /> an appropriate context, i.e. framework related           interconnected reactions involved [10&#226;&#8364;&#8220;14]. <br /> prior knowledge, obtained dataset                 Graph-based models (GBMs) built com- <br /> can interpreted. In case functional genomic         prehend analyse metabolic capabilities of <br />  318                                              Tymoshenko et al. <br />  <br /> P. falciparum systematic manner. GBMs represent        describes uninterrupted pathway that <br /> metabolism graphs nodes (to denote metab-           leads transformation externally supplied <br /> olites) links (respective metabolic        substrates biomass precursor the <br /> reactions) [19]. The links represent possibility   precursors. To satisfy requirement, model- <br /> of interconversion particular metabolite an-     building process involves inclusion &#226;&#8364;&#732;or- <br /> other (justified correspondingly annotated       phan reactions&#226;&#8364;&#8482;, enzyme-coding gene <br /> genes), regardless properties reaction    annotated [25]. <br /> including stoichiometry. A natural advantage             A conventional reconstruction workflow [25] <br /> GBMs require minimal input information         clearly defines list enzymes be <br /> and assess well-annotated parts given meta-        genome complete pathways <br /> bolic network skipping unclear ones. However,         meet experimentally observed metabolic beha- <br /> such representation metabolic network          viour. With approach, putative metabolic func- <br /> capturing mass-balance property suitable       tions proposed 17 genes Leishmania <br /> framework incorporation experimental infor-          major, previously functional annota- <br /> mation, making scope GBM-based meth-            tions [26]. Although assignments, based <br /> ods metabolic modelling limited. Nowadays,        moderate sequence identity, sufficient <br /> graph-based metabolic reconstructions topo-             proofs suggested functions, represent <br /> logical analysis largely obsolete         set testable hypotheses experimental valid- <br /> replaced comprehensive constraint-based             ation. Such list orphan reactions provides an <br /> approaches.                                                 invaluable guidance functional genomics studies <br />     Over past decade flux-balance analysis (FBA)        P.falciparum 2000 genes with- <br /> has established leading approach study-       putative functional annotation. <br /> ing constraint-based models (CBMs) cellular me-              One important applications CBMs <br /> tabolism [20]. Constraint-based metabolic models        pathogenic species capability predict po- <br /> FBA based two-dimensional arrays,              tential vulnerabilities metabolism [9]. Often <br /> each row represents metabolite column            metabolic networks redundant contain more <br /> corresponds reaction, which, rule, linked      chain reactions produce certain bio- <br /> to certain enzyme gene organism [21].          mass precursors. To explore redundancy, FBA <br /> Each intersection row column contains nu-          enables attempt simulation growth <br /> merical value standing stoichiometric coeffi-       reactions model removed <br /> cient given metabolite given reaction. Such       one-at-a-time manner. Whenever production of <br /> notation enables explicit quantitative de-          biomass blocked certain reaction, the <br /> scription metabolic network allows             classified essential. In similar manner, <br /> imposing basic constraint&#226;&#8364;&#8221;mass balance.        FBA simulate outcomes withdrawal of <br /> A model accounts mass balance lar-         enzymes genes attempting simulate growth <br /> gely undefined constraints imposed       utilization reactions associated an <br /> enrich range feasible solutions biologic-       enzyme gene disabled. We refer reader a <br /> ally realistic ones. The scope additional constraints    comprehensive publication insilico essentiality stu- <br /> is constantly growing currently includes thermo-        dies CBM Saccharomyces cerevisiae [27] for <br /> dynamic (tFBA), regulatory (rFBA) con-            details examples. <br /> straints inherent cellular metabolism (reviewed in <br /> [22]) constraints inferred experimental data. <br />     CBMs, unlike GBMs, require pre-defined             OUTLOOK OF IN SILICO <br /> objective utilization available sub-       METABOLIC MODELS FOR <br /> strates optimized [23]. A common object-          P. FALCIPARUM <br /> ive function fast-growing cells biomass          Several studies date focused compre- <br /> reaction (other plausible objectives reviewed        hensive reconstruction metabolic network <br /> [24]). It represents cellular replication reaction     P. falciparum (Table 1). Early studies produced <br /> that consumes pre-defined metabolites             GBMs [11, 14, 28] considered the <br /> referred precursors biomass. Existence        set metabolic activities reported time with- <br /> solution implies stoichiometric array              taking account information the <br />                        Functional genomics P. falciparum using metabolic modelling analysis                                                     319 <br />  <br /> Table 1: Comparison silico metabolic reconstructions P. falciparum <br />  <br /> Authors (year publication)         Information modela <br />  <br />                                       Metabolites           Reactions             Genes                 Compartmentsb <br />  <br /> Yeh et al. (2004) [14]                525                   696                   &#195;&#167;                     &#195;&#167; <br /> Fatumo et al. (2009) [11]             554                   575                   &#195;&#167;                     &#195;&#167; <br /> Huthmacher et al. (2010) [12]         P: 1622 E: 566        P: 1375 E: 437        P: 579                P: c, m, a, n, r, v, g; E: e, c <br /> Plata et al. (2010) [13]              915                   1001                  366                   P: e, c, m, a <br /> Bazzani et al. (2012) [10]            P: 1622 H: 1149       P: 1394 H: 2539       P: 579    H: 704      P: c, m, a, n, r, v, g; H: c, r, g, l, m, n, p, b, s <br />  <br />  <br /> The models [11,14] built using graph-based approach following constraint-based models [10,12,13]. <br /> a <br />  &#226;&#8364;&#732;P&#226;&#8364;&#8482; denotes model parasite,&#226;&#8364;&#732;E&#226;&#8364;&#8482; human erythrocyte,&#226;&#8364;&#732;H&#226;&#8364;&#8482; human hepatocyte. <br /> b <br />   Abbreviated names compartments: e, extracellular space; c, cytosol; m, mitochondrion, a, apicoplast; n, nucleus; v, digestive vacuole; r, endoplas- <br /> mic reticulum; g, Golgi complex; l, lysosome; p, peroxisome; b, bile canaliculus; s, sinusoidal space. <br />  <br />  <br />  <br /> compartments occur differences                                metabolite exchanges host cell the <br /> in life-stage-specific metabolism pathogen.                            parasite, identified inconsistencies <br /> As mentioned before, consistency mass-                             experimental data, need investi- <br /> balance relationships, models suitable                         gation [12]. <br /> for integration experimental data allowed                                  The second CBM developed independently by <br /> only qualitative predictions gene essentiality.                            Plata et al. [13] took account compartmental- <br /> Nevertheless, reconstruction process                               ization intracellular space mass-balance <br /> putative functions suggested hundreds                             constraints. The results gene deletions performed <br /> genes previously annotated coding &#226;&#8364;&#732;hypo-                             silico correspondence with <br /> thetical protein&#226;&#8364;&#8482; [14]. These modelling efforts                         reports primary literature: 100% agreement <br /> gave broad overview metabolic capabilities                           compared gene deletion studies and <br /> of P. falciparum annotated genome                                 70% case enzymatic inhibition experiments. <br /> compared expected ones provided                                Comparison number essential genes in <br /> solid foundation building modern,                                models P. falciparum S. cerevisiae confirmed <br /> comprehensive reconstructions discussed hereafter.                            notion parasite likely possesses signifi- <br />     Using constraint-based approach, Huthmacher                             cantly lower metabolic flexibility bypass single- <br /> et al. [12] assembled compartmentalized,                            gene deletions compared free-living <br /> mass-balanced life-stage-specific model                                organisms similar genome sizes [13]. Forty <br /> P. falciparum metabolism. Through insilico simulations,                       genes suggested potential drug targets due <br /> the authors identified enzymatic activities                          silico essentiality extremely low or <br /> essential proliferation parasite. Thirty                           absent sequence identity human proteins [13]. <br /> of silico essential reactions catalyzed                        The essentiality genes encoding the <br /> enzymes homologues human prote-                                nicotinate mononucleotide adenylyltransferase was <br /> ome (E-value &gt; 0.075). These ranked targets                           verified using experimental inhibitor, which <br /> of particular based evidence activity                          caused arrest P. falciparum proliferation an <br /> during multiple stages P. falciparum life cycle                        IC50 50 mM in vitro culture [13]. In addition, <br /> studied and/or presence SuperTarget database                           Plata et al. [13] report silico double- <br /> [29] candidates treatment infections                          gene deletion simulations P. falciparum leading to <br /> [12]. Furthermore, natural environment                             identification 16 pairs genes pre- <br /> parasite simulated embedding CBM                              dicted non-essential single-gene <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>knockout</span> <br /> the metabolic reconstruction human erythro-                              simulation resulted dramatic impairment of <br /> cyte. This limited substrate accessibility                        metabolism targeted simultaneously. <br /> those available host cell milieu. Such con-                                Holzhutter co-workers developed the <br /> straints cellular interface constraints                         recent CBM P. falciparum metabolic net- <br /> reaction fluxes, deduced gene expression pro-                            work [10] updating previous PlasmoNet1 <br /> files (obtained different life stages),                           model [12]. In PlasmoNet2 [10] added <br /> allowed model retrieve known directions                             new transport reactions based metabolomics <br />  320                                               Tymoshenko et al. <br />  <br /> data [30] removed reaction according            modelling yeast metabolism, expect the <br /> updated version KEGG database. Through inte-               emergence reconciliation efforts aim at <br /> gration PlasmoNet2 CBM human                   obtaining consensus, up-to-date CBM P. falcip- <br /> hepatocyte [31], evaluated silico essential-      arum. Recently developed workflows manual <br /> ity P. falciparum genes liver stage assessed,       [35] semi-automated [36] reconciliations of <br /> using &#226;&#8364;&#732;reduced fitness&#226;&#8364;&#8482; approach, effects targeting      existing metabolic models significantly facilitate <br /> enzymes homologous predicted               efforts. There areas which <br /> essential host pathogen [10].             foresee room upcoming improvements: the <br />     Comparison essentiality predictions           first, mentioned before, systematic implemen- <br /> by aforementioned models trivial task.           tation thermodynamic constraints models of <br /> First, study Plata et al. aimed pre-        P. falciparum; second deliberate revision of <br /> dicting essentiality genes,          objective function (i.e. biomass reaction) to <br /> studies assessed essentiality enzymes, pre-        make consistent actual biomass compos- <br /> dictions overlap gene-to-enzyme              ition parasite life stage interest; the <br /> relations one-to-one. Second, gene               experimental verification quantifica- <br /> enzyme absent list &#226;&#8364;&#732;predicted           tion uptake fluxes present models. <br /> essential&#226;&#8364;&#8482; predicted          These improvements likely make silico pre- <br /> non-essential, possibly, fact        dictions gene essentiality reliable also <br /> the corresponding metabolic process simply             expand number metabolic functions currently <br /> included model interest. Third, essentiality        known essential P. falciparum. <br /> predictions CBMs directly dependent                 Metabolic reconstruction efforts date have <br /> set metabolites included biomass reac-          summarized results decades experimental <br /> tion, differences assumed biomass compos-          research metabolism P. falciparum the <br /> ition directly affect results silico simulations.   form silico models, reproduce <br /> For overview P. falciparum genes enzymes            prior knowledge, provide novel insights. <br /> predicted essential existing models, refer       Nevertheless, fact cultivation parasite <br /> reader Supplementary Table 1. The table           fully defined medium impossible clearly <br /> provides reader literature references         highlights important metabolic peculiari- <br /> experimental assessment genes/enzymes predicted            ties remain discovered. The promising direc- <br /> to essential different models.                      tion utilization CBMs frameworks of <br />     An important simplification common             current knowledge, challenged, <br /> aforementioned models ad hoc assignment             refined constrained various high-throughput <br /> directionality reactions, pre-set            datasets discussed following section. <br /> either reversible [12] assumed the <br /> same reversibility metabolic models non- <br /> related organisms [13]. In principle, directionality      HIGH-THROUGHPUT DATA AND <br /> of reaction subject thermodynamic proper-         METABOLIC MODELS: &#226;&#8364;&#732;CONTENT <br /> ties spurious ad hoc assignments violate       FOR CONTEXT&#226;&#8364;&#8482; <br /> fundamental constraint. This issue addressed         As mentioned above, CBMs hold great potential to <br /> rigorously genome-scale metabolic networks              incorporate various types experimental data as <br /> several organisms [32&#226;&#8364;&#8220;34] implementation                content context computational metabolic <br /> thermodynamic constraints reaction directionality          networks. In section review provide an <br /> as extension conventional FBA methodology.              outlook currently published studies have <br />     Overall, metabolic modelling P. falciparum          integrated computational modelling experimen- <br /> date lags years similar efforts        tal research efforts type high-throughput <br /> the model eukaryote S. cerevisiae; exist          data introduced Table 2. <br /> independently reconstructed models lack <br /> consistency differences         Genomics <br /> the reconstruction workflows, sources primary          Genomics studies P. falciparum yielded far <br /> information, level complexity varying          complete genome sequences (for 3D7 IT <br /> degree comprehensiveness. Similarly trends          strains [55]); however, functional annotation <br />                      Functional genomics P. falciparum using metabolic modelling analysis                                321 <br />  <br />  <br /> Table 2: Overview high-throughput methods applied functional genomics P. falciparum available <br /> options integration data CBMs <br />  <br /> Approach          Examples methods                   Maximal coverage   Integration CBMs <br />  <br /> Genomics          BLASTP [2], metaSHARK [37], DETECT [38]   ca. 50%            Used input data building CBMs [25]; validation <br />                                                                                  silico gene essentiality [39] <br /> Transcriptomics   DNA microarray [40 ^ 42], RNA-seq [43]    ca. 99%            Used constrain CBMs [12, 13]; reviewed [44] <br /> Proteomics        2D LC-MS/MS [45], nano-LC-MS/MS [46]      ca. 45%            Used constrain CBMs [47] <br /> Metabolomics      HPLC-MS/MS [30, 48], 1H-NMR [49]          ca. 15-20%         Used constrain CBMs [32, 34], reviewed [33] <br />                   13 <br /> Fluxomics           C-NMR [50, 51], 14C-NMR [52]            Less 1%       Used constrain CBMs, reviewed [53, 54] <br />  <br />  <br />  <br />  <br /> remains far complete [3]. Nonetheless,                 applied comparison high-throughput <br /> these partially annotated genomes gave rise nu-                datasets viable/non-viable single- double- <br /> merous genome-wide transcriptomics large-scale                <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>gene knockout</span> mutants S. cerevisiae silico <br /> proteomics studies possible                   gene essentiality predictions [27]. This approach has <br /> reconstruction parasite&#226;&#8364;&#8482;s metabolic network                led corrections model (e.g. <br /> in silico.                                                        inclusion additional reactions, compounds and <br />     High-throughput functional genomics P. falcip-             genes changes biomass reaction), each <br /> arum nascent field malaria research            supported literature evidence largely im- <br /> a limited number studies succeeded gen-                proved consistency model existing <br /> erating single-gene mutant parasite clones large             experimental data [27]. Once successful high- <br /> scale. The largest coverage reported date                  throughput forward reverse genetic technique is <br /> achieved forward genetics approach based                   established P. falciparum, GrowMatch similar <br /> on transposon mutagenesis using transposable                  algorithm validate computational model, <br /> element piggyBac [56]. The collection regroups                    improve understanding obtained results and <br /> about 200 mutant parasite lines cover                 produce hypotheses experimental investigation, <br /> modest P. falciparum genome. Among                    applicable malaria drug research. <br /> those, 24 single-gene disruptions caused severe <br /> growth defects vitro, lead lethal               Transcriptomics <br /> phenotypes vivo [56]. A major drawback                 Transcriptomic profiling appears of <br /> method inability reveal essential genes             common high-throughput methods applied in <br /> to haploid state malaria parasite through-             malaria research; numerous gene expression datasets <br /> out life stages. Although specific design             available PlasmoDB database (http://www. <br /> of transposon integrated inducible pro-               plasmodb.org) different lineages parasite <br /> moter proposed assess essential genes [6],            various conditions. Although scope of <br /> it remains proved practicable. In contrast,           gene expression data growing, new increasingly <br /> in vitro essentiality data genetically tractable           advanced algorithms integrate datasets into <br /> parasite Trypanosoma brucei obtained                    CBMs developed [44]. To date, two <br /> for nearly coding sequences genome                 models P. falciparum metabolism incorporated <br /> using RNA-interference target sequencing                      available transcriptomics data constraints [12, 13] <br /> approach [57].                                                    represent life-stage-specific metabolic features of <br />     The value gene essentiality datasets                parasite. <br /> reinforced possibility silico simulation              Huthmacher et al. integrated model with <br /> similar genetic perturbations. This offers reciprocal             life-stage-specific gene expression profiles <br /> benefits experimental computational                       [40&#226;&#8364;&#8220;42, 58, 59] avoid, possible, utiliza- <br /> research: CBMs suggest underlying                   tion reactions likely inactive based <br /> reason viable/non-viable phenotypes knock-                 abundance corresponding messenger <br /> out strains and, otherwise, reveal weak points,               ribonucleic acids (mRNAs) [12]. This approach has <br /> where model improved.                               allowed authors infer plausible metabolite <br /> Accordingly, algorithm GrowMatch [39]                     exchanges human erythrocyte the <br />  322                                             Tymoshenko et al. <br />  <br /> parasite, predict directionality      identification proteins involved parasite&#226;&#8364;&#8482;s <br /> pathways easily inferred gene          response treatment conventional anti- <br /> expression data alone. The predictions host&#226;&#8364;&#8220;        malarial drugs. A recent alternative method quan- <br /> parasite metabolite exchanges significantly        titative proteomics relies externally supplied, <br /> better agreement physiology reported           known amounts proteins interest, obtained <br /> literature fluxes allowed re-        using QconCAT technique [65]. QconCAT-derived <br /> action mRNA absent life-          proteins labelled heavy isotopes serve <br /> stage-specific transcriptome present          markers identification similar un- <br /> earlier (up 12 preceding hours) [12]. This obser-       labeled proteins scale quantification <br /> vation consistent earlier study   [65]. This approach offers new level sensitivity <br /> significant time delay maximal gene expres-        holds promise comprehensive high- <br /> sion peak accumulation corresponding         resolution quantitative proteomics P. falciparum. <br /> proteins [60].                                                 Taking account complexity inter- <br />     Plata et al. [13] attempted integrate gene     play mRNA protein abundances (thor- <br /> expression data [30, 61] model predict       oughly examined [66]), studies combine <br /> the shifts extracellular (intraerythrocytic)        transcriptomic proteomic measurements <br /> abundance metabolites ring              particular interest. As result inte- <br /> trophozoite trophozoite schizont stages. The        grated approach applied P. falciparum, time-delayed <br /> authors assumed higher influx metabolite       correlation peaks transcription max- <br /> into parasite lower abundance          imal abundance proteins observed <br /> substrate molecule cytosol infected          glycolytic enzymes exception <br /> erythrocyte, higher efflux rate metab-      enolase [60]. However, enzymes, <br /> olite parasite increase intraerythro-    accumulation mRNA did correlate with <br /> cytic abundance. The computational predictions             changes abundance corresponding <br /> were verified existing metabolomics            enzymes complete intraerythrocytic replica- <br /> data [30] average accuracy 70%.              tion cycle [60]. Mair et al. [67] demonstrated in <br />     With major improvements accuracy sensi-         Plasmodium species genes transcribed <br /> tivity thresholds achieved modern           expressed translational repressions. <br /> RNA-seq approaches [43], utility transcriptomic             Despite availability data necessary <br /> studies increase significantly experimental       computational methods, proteomics been <br /> and silico functional genomics.                         used systematically development analysis <br />                                                            CBMs P. falciparum. Meanwhile however, has <br /> Proteomics                                                 intercellular pathogen Trypanosoma <br /> The large-scale proteomics datasets pub-         cruzi: Roberts et al. integrated CBM a <br /> lished 2002 [45, 46] provided relatively high         proteomics dataset constrains fluxes re- <br /> coverage (ca. 45% 23% respectively) ex-         actions corresponding enzyme not <br /> pected proteome P. falciparum [62]. While deliver-      detected life-stage-specific proteome. Using this <br /> ing semi-quantitative results, studies          constraint authors aimed making model <br /> provided unique possibility verify correct      representative possible metabolic <br /> the genome annotation terms assignments           state pathogen particular stage life <br /> open reading frames, splicing patterns confirm      cycle. Initially resulted over-constrained <br /> the presence particular enzymes different stages     model unable simulate growth, suggesting <br /> of infection.                                              enzymes reactions were <br />     Several approaches used generate          detected proteome likely be <br /> quantitative proteomics datasets. Nirmalan et al. [63]     present. On hand, constraints inferred from <br /> have established method fully quantitative           proteomics data corrected reaction essenti- <br /> proteomics using [13C6 15N1]-labelled isoleucine        ality predictions agreement ex- <br /> recognize novo synthesized proteins. Following     perimental data constraints [47]. <br /> method, Prieto et al. [64] quantified 1253 proteins         Although complex interplay concen- <br /> P. falciparum trophozoites exposure    trations mRNAs enzymes fluxes <br /> the parasite chloroquine artemisinin, allowing      corresponding reactions remains be <br />                   Functional genomics P. falciparum using metabolic modelling analysis                323 <br />  <br /> elucidated, utilization transcriptomics prote-     potential (e.g. glycerol reported [74]). Using <br />                                                           1 <br /> omics information ensure higher con-           H-NMR, concentrations 50 me- <br /> fidence enzyme (or not)         tabolites measured cell extracts of <br /> present life stage interest. For instance,   P. falciparum trophozoites [49]. The unbiased nature <br /> is reasonable assume reaction    method enabled identification some <br /> enzyme transcripts detected         unexpected metabolites (e.g. aminobutyric acid and <br /> likely occur tightly constrained     buffering agent HEPES) present relatively high <br /> in stage-specific CBMs.                               concentrations lysates parasite cells [49]. <br />                                                               An important drawback current metabolo- <br /> Metabolomics                                              mics studies P. falciparum that, metabolites <br /> Several recent reviews discuss methodol-    present compartment (e.g. cyto- <br /> ogies techniques currently used          solic mitochondrial adenosine diphosphate), the <br /> metabolomics various organisms [68&#226;&#8364;&#8220;70] mal-        measured concentration reflects average <br /> aria parasites particular [71]. The largest metabo-    value, differ significantly the <br /> lomics profile date obtained liquid             actual concentration compartments. <br /> chromatography coupled tandem mass-spec-             Metabolomics separate organelles emerging <br /> trometry (LC-MS/MS) analysis uninfected           field functional genomics research, exemplified <br /> and P. falciparum-infected human erythrocytes [30].       algea [75], enrich verify current <br /> Relative changes concentrations 90        knowledge subcellular localization various <br /> metabolites monitored medium             metabolic processes. The issue compartmentaliza- <br /> and cell lysates replication cycle,        tion especially complex case malaria para- <br /> with measurements 8 h [30]. A attempt       sites. Indeed, obligate intracellular parasites <br /> to correlate results gene expression data      develop hepatocytes erythrocytes the <br /> obtained time points infection         intermediate host. Moreover, parasite harbours <br /> revealed despite common periodic <span id='am-394' about='xsp:pattern' typeof='owl:Thing'>pattern</span>        symbiotic organelles: mitochondrion and <br /> gene expression, mea-       apicoplast (relic plastid organelle acquired <br /> sured metabolite abundances fluctuated periodically       engulfment algae), host <br /> [30]. The coverage P. falciparum putative meta-    metabolic pathways crucial central <br /> bolome achieved study 15%          carbon metabolism P. falciparum. <br /> 20% (estimated similarly [72]) compares with <br /> the metabolomics Leishmania donovani promasti-         Fluxomics <br /> gotes [72]. Currently, algorithm available       Fluxomics largely unexplored area malaria <br /> directly incorporate relative concentration           research, experimentally amenable species <br /> values CBMs. Even so, Plata et al. used data      represents relatively well-established rapidly <br /> as reference verify silico predictions   developing field [53, 54]. Incorporation mea- <br /> discussed above. Furthermore, dataset [30] gives      sured values flux reactions present in <br /> a valuable insight actively consumed se-         CBMs significantly improve accuracy the <br /> creted metabolites, aspect metabolic fate       models reducing uncertainty ranges and <br /> that used constraint CBMs.                 distributions metabolic fluxes. To best our <br />     Quantification absolute concentration values       knowledge, fluxomics studies P. falcip- <br /> using MS technically possible, ham-     arum, date, assessments influx rates for <br /> pered need standard solution        single substrates: glucose (in infected non-in- <br /> metabolite [48, 73]. On contrary, nuclear mag-        fected human erythrocytes [50, 51]), isoleucine <br /> netic resonance (NMR) techniques require           [76], pantothenate [52] inorganic phosphate <br /> external standards identification     [77]. Due indispensability substrates <br /> quantification metabolites integral      parasite, incorporation flux values <br /> output signal proportional concentration        represent overriding constraints CBMs. <br /> of studied nuclei. Despite lower sensitivity          In case model organisms, e.g. S. cerevisiae, <br /> compared modern MS methods [70],                 measured metabolic fluxes included the <br /> 13 <br />    C-NMR identify metabolites other-         CBMs allowed silico resolution experimen- <br /> wise undetectable MS low ionization       tally observed metabolic features be <br />  324                                              Tymoshenko et al. <br />  <br /> inferred otherwise&#226;&#8364;&#8221;neither transcriptomics             reaction gradually reduced extent in <br /> nor proteomics data [54]. A relevant example           models. Although shows purely theoretical <br /> large increase glycolytic flux, main-       sensitivity host parasite inhibition of <br /> tained yeast exposed low levels oxygen         certain enzymatic activity, approach be <br /> without changing expression levels involved         considered caution neglects import- <br /> genes [78]. This led important conclusion        ant aspects, captured CBMs. <br /> fluxes primary metabolism likely              FBA suitable simulation out- <br /> controlled regulation enzymatic activities       comes gene knockout experiments, tar- <br /> not changes gene expression secondary           geted enzymatic activity abolished completely. <br /> metabolism [54].                                            However, realistic simulation effects of <br />     Overall, numerous methods developed          reduced enzymatic activity (i.e. action drug) <br /> the integration experimental data CBMs,             requires careful consideration kinetic thermo- <br /> only modest proportion hitherto applied         dynamics properties. Ultimately, assumes <br /> to pathogenic organisms P. falciparum particu-       identical kinetic properties inhibited enzymes in <br /> lar. We argue com-         infected host cell parasite, sensitivity the <br /> plexity experimental study pathogens,         flux catalyzed reaction same <br /> also fact majority silico       inhibitor different simply the <br /> methods discussed initially designed         expression level enzyme in <br /> free-living organisms. Although methods are,         organisms [79]. <br /> principle, applicable intracellular pathogens,          Metabolic control analysis (MCA) considers all <br /> aspects unique parasites crucial          relevant properties provide realistic <br /> obtaining relevant computational results. Examples          estimation extent flux through <br /> of aspects common absence            reaction sensitive action an <br /> clearly defined set substrates by-products       inhibitor [80&#226;&#8364;&#8220;82]. Using methodologies from <br /> the metabolism intracellular parasites, changes      MCA, predict inhibition <br /> in composition biomass multiple stages         target enzyme selective effect the <br /> their life cycles sub-optimal utilization available   parasite relative host. Using kinetic modelling <br /> substrates, etc. These issues aforementioned        MCA, Bakker et al. [83] demonstrated <br /> experimental challenges represent area            glyceraldehyde-3-phosphate dehydrogenase <br /> we expect forthcoming improvements, leading            phosphoglycerate kinase represent promis- <br /> better understanding metabolic peculiarities             ing targets treatment sleeping sickness due <br /> encoded genome P. falciparum.                     drastically different susceptibility glycolytic <br />                                                             fluxes inhibition enzymes human and <br /> FUTURE DIRECTIONS                                           African trypanosome. <br /> Obligate intracellular parasitism P. falciparum              A major limiting factor application MCA to <br /> represents significant challenge medical treat-       metabolic models P. falciparum need inte- <br /> ment, virtually impossible avoid impact      grate kinetic parameters enzymes model <br /> antimalarial drugs host. Thus, high selectivity      concentration profile key metabolites [84]. <br /> is essential prerequisite potential drug can-    To date, experimental information exists only <br /> didates. However, genes enzymes             modest CBMs relatively small <br /> predicted essential P. falciparum possess    number organisms [85], limited <br /> a variable level sequence identity human        applicability [86, 87]. The recently developed <br /> counterparts [10, 12, 13], suggesting interfer-     ORACLE framework used perform <br /> ence affect uninfected host              MCA significant uncertainty provide a <br /> cells. To assess phenomenon insilico, &#226;&#8364;&#732;reduced     guidance quantitative ranking drug target <br /> fitness&#226;&#8364;&#8482; approach applied Bazzani et al. [10]        identification [84]. Kinetic modelling MCA <br /> to CBM P. falciparum embedded               represent promising field research a <br /> model human hepatocyte. This method                fine-grained evaluation pathways contain <br /> estimate quantitatively production biomass           silico predicted drug targets. These efforts also <br /> would affected host parasite&#226;&#8364;&#8482;s meta-      provide valuable guidance development <br /> bolic networks flux targeted             functional genomics P. falciparum, whereas <br />                       Functional genomics P. falciparum using metabolic modelling analysis                                         325 <br />  <br /> analytical technologies provide information                    7.    Augagneur Y, Wesolowski D, Tae HS, et al. Gene selective <br />                                                                               mRNA cleavage inhibits development Plasmodium <br /> necessary building kinetic model meta-                           falciparum. Proc Natl Acad Sci USA 2012;109:6235&#226;&#8364;&#8220;40. <br /> bolic network.                                                          8.    Kim HU, Sohn SB, Lee SY. Metabolic network modeling <br />                                                                               simulation drug targeting discovery. BiotechnolJ <br />                                                                               2012;7:330&#226;&#8364;&#8220;42. <br /> SUPPLEMENTARY DATA                                                      9.    Pinney JW, Papp B, Hyland C, et al. Metabolic reconstruc- <br />                                                                               tion analysis parasite genomes. Trends Parasitol 2007; <br /> Supplementary data available online http://                            23:548&#226;&#8364;&#8220;54. <br /> bib.oxfordjournals.org/                                                 10.   Bazzani S, Hoppe A, Holzhutter HG. Network-based <br />                                                                               assessment selectivity metabolic drug targets in <br />                                                                               Plasmodium falciparum respect human liver metabol- <br />                                                                               ism. BMC Syst Biol 2012;6:118. <br />  Key Points                                                             11.   Fatumo S, Plaimas K, Mallm JP, et al. Estimating novel <br />                                                                               potential drug targets Plasmodium falciparum analysing <br />    Modelling efforts P. falciparum focused revealing indispens-         metabolic network <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>knock-out</span> strains silico. Infect <br />    able metabolic functions expanded list putatively              Genet Evol 2009;9:351&#226;&#8364;&#8220;8. <br />    essential genes enzymes. <br />    Interpretation high-throughput data context CBMs        12.   Huthmacher C, Hoppe A, Bulik S, et al. Antimalarial drug <br />    yields new, non-intuitive insights functions metabolic            targets Plasmodium falciparum predicted stage-specific <br />    genes offers testable hypotheses experimental research.            metabolic network analysis. BMC Syst Biol 2010;4:120. <br />    Kinetic modelling approaches enable integration        13.   Plata G, Hsiao TL, Olszewski KL, et al. Reconstruction and <br />    broad range data functional genomics, metabolo-            flux-balance analysis Plasmodium falciparum metabolic <br />    mics, address limitations CBMs.                             network. Mol Syst Biol 2010;6:408. <br />    Reliable kinetic modes metabolic networks accelerate         14.   Yeh I, Hanekamp T, Tsoka S, et al. Computational analysis <br />    drug target identification rational drug development.                  Plasmodium falciparum metabolism: organizing genomic <br />                                                                               information facilitate drug discovery. Genome Res 2004; <br />                                                                               14:917&#226;&#8364;&#8220;24. <br />                                                                         15.   Hayes CN, Wheelock AM, Normark J, et al. Enlistment of <br /> Acknowledgements                                                              omics technologies fight malaria: Panacea or <br /> The authors like acknowledge Keng Cher Soh                       Pandora&#226;&#8364;&#8482;s Box? J Pestic Sci 2006;31:263&#226;&#8364;&#8220;72. <br /> Ljubisha Miskovic helpful discussions methods            16.   Ogata H, Goto S, Sato K, etal. KEGG: Kyoto Encyclopedia <br /> reviewed manuscript Alexandros Kiparissides,                      Genes Genomes. Nucleic Acids Res 1999;27:29&#226;&#8364;&#8220;34. <br /> Aikaterini Zisaki, Stefano Andreozzi, Anirikh Chakrabarti           17.   Ginsburg H. Progress in silico functional genomics: the <br /> Noushin Hadadi.                                                               malaria Metabolic Pathways database. Trends Parasitol 2006; <br />                                                                               22:238&#226;&#8364;&#8220;40. <br />                                                                         18.   Bozdech Z, Ginsburg H. Data mining transcriptome <br /> FUNDING                                                                       Plasmodium falciparum: pentose phosphate pathway <br /> S.T. supported iPhD grant Swiss National                    ancillary processes. MalarJ 2005;4:17. <br /> Science Foundation initiative SystemsX.ch; R.O.              19.   Stelling J. Mathematical models microbial systems biol- <br /> was supported SystemsX.ch initiative Swiss                      ogy. Curr Opin Microbiol 2004;7:513&#226;&#8364;&#8220;8. <br /> National Science Foundation (441 grant LipidX-2008/011).                20.   Orth JD, Thiele I, Palsson BO. What flux balance ana- <br />                                                                               lysis? Nat Biotechnol 2010;28:245&#226;&#8364;&#8220;8. <br />                                                                         21.   Palsson B. Systems Biology: Properties Reconstructed Networks. <br />                                                                               New York: Cambridge University Press, 2006. <br /> References <br />                                                                         22.   Lewis NE, Nagarajan H, Palsson BO. Constraining meta- <br /> 1.   Aregawi M. World Health Organization. Global Malaria                     bolic genotype-<span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> relationship using phylogeny of <br />      Programme. World Malaria Report 2010. Geneva: World                      silico methods. Nat Rev Microbiol 2012;10:291&#226;&#8364;&#8220;305. <br />      Health Organization, 2010.                                         23.   Feist AM, Palsson BO. The biomass objective function. <br /> 2.   Gardner MJ, Hall N, Fung E, et al. Genome sequence                Curr Opin Microbiol 2010;13:344&#226;&#8364;&#8220;349. <br />      human malaria parasite Plasmodium falciparum. Nature 2002;         24.   Schuetz R, Kuepfer L, Sauer U. Systematic evaluation of <br />      419:498&#226;&#8364;&#8220;511.                                                             objective functions predicting intracellular fluxes in <br /> 3.   Limenitakis J, Soldati-Favre D. Functional genetics                   Escherichia coli. Mol Syst Biol 2007;3:119. <br />      Apicomplexa: potentials limits. FEBS Lett 2011;585:            25.   Thiele I, Palsson BO. A protocol generating high- <br />      1579&#226;&#8364;&#8220;88.                                                                 quality genome-scale metabolic reconstruction. Nature <br /> 4.   Baum J, Papenfuss AT, Mair GR, et al. Molecular genetics                 Protoc 2010;5:93&#226;&#8364;&#8220;121. <br />      comparative genomics reveal RNAi functional          26.   Chavali AK, Whittemore JD, Eddy JA, etal. Systems analysis <br />      malaria parasites. Nucleic Acids Res 2009;37:3788&#226;&#8364;&#8220;98.                    metabolism pathogenic trypanosomatid Leishmania <br /> 5.   Visvesvara GS, Garcia LS. Culture protozoan parasites.                major. Mol Syst Biol 2008;4:177. <br />      Clin Microbiol Rev 2002;15:327&#226;&#8364;&#8220;28.                                 27.   Zomorrodi AR, Maranas CD. Improving iMM904 S. <br /> 6.   Balu B. Moving &#226;&#8364;&#732;&#226;&#8364;&#732;forward&#226;&#8364;&#8482;&#226;&#8364;&#8482; Plasmodium genetics                cerevisiae metabolic model using essentiality synthetic <br />      transposon-based approach. J Trop Med 2012;2012:829210.                lethality data. BMC Syst Biol 2010;4:178. <br />  326                                                     Tymoshenko et al. <br />  <br /> 28. Fatumo S, Plaimas K, Adebiyi E, etal. Comparing metabolic        46. Lasonder E, Ishihama Y, Andersen JS, et al. Analysis of <br />     network models based genomic automatically                    Plasmodium falciparum proteome high-accuracy mass <br />     inferred enzyme information Plasmodium                  spectrometry. Nature 2002;419:537&#226;&#8364;&#8220;42. <br />     human host define drug targets silico. Infect Genet Evol   47. Roberts SB, Robichaux JL, Chavali AK, et al. Proteomic <br />     2011;11:201&#226;&#8364;&#8220;8.                                                       network analysis characterize stage-specific metabolism <br /> 29. Hecker N, Ahmed J, Eichborn J, et al. SuperTarget goes               Trypanosoma cruzi. BMC Syst Biol 2009;3:52. <br />     quantitative: update drug-target interactions. Nucleic        48. Duy SV, Besteiro S, Berry L, et al. A quantitative <br />     Acids Res 2012;40:D1113&#226;&#8364;&#8220;7.                                           liquid chromatography tandem mass spectrometry <br /> 30. Olszewski KL, Morrisey JM, Wilinski D, et al. Host-parasite          method metabolomic analysis Plasmodium <br />     interactions revealed Plasmodiumfalciparum metabolomics.          falciparum lipid related metabolites. Analy Chim Acta 2012; <br />     Cell Host Microbe 2009;5:191&#226;&#8364;&#8220;9.                                      739:47&#226;&#8364;&#8220;55. <br /> 31. Gille C, Bolling C, Hoppe A, et al. HepatoNet1:                49. Teng RW, Junankar PR, Bubb WA, et al. Metabolite pro- <br />     comprehensive metabolic reconstruction human                  filing intraerythrocytic malaria parasite Plasmodium <br />     hepatocyte analysis liver physiology. Mol Syst            falciparum H-1 NMR spectroscopy. NMR Biomed 2009; <br />     Biol 2010;6:411.                                                     22:292&#226;&#8364;&#8220;302. <br /> 32. Henry       CS,    Broadbelt     LJ,    Hatzimanikatis     V.    50. Mehta M, Sonawat HM, Sharma S. Malaria parasite- <br />     Thermodynamics-based metabolic flux analysis. Biophys J              infected erythrocytes inhibit glucose utilization unin- <br />     2007;92:1792&#226;&#8364;&#8220;805.                                                    fected red cells. FEBS Lett 2005;579:6151&#226;&#8364;&#8220;8. <br /> 33. Soh KC, Hatzimanikatis V. Network thermodynamics              51. Mehta M, Sonawat HM, Sharma S. Glycolysis in <br />     post-genomic era. Curr Opin Microbiol 2010;13:350&#226;&#8364;&#8220;7.             Plasmodium falciparum results modulation host enzyme <br /> 34. Soh KC, Miskovic L, Hatzimanikatis V. From network                   activities. J Vector Borne Dis 2006;43:95&#226;&#8364;&#8220;103. <br />     models network responses: integration thermodynamic        52. Saliba KJ, Kirk K. H&#195;&#190;-coupled pantothenate transport in <br />     kinetic properties yeast genome-scale metabolic               intracellular malaria parasite. J Biol Chem 2001;276: <br />     networks. FemsYeast Res 2012;12:129&#226;&#8364;&#8220;43.                              18115&#226;&#8364;&#8220;21. <br /> 35. Oberhardt MA, Puchalka J, dos Santos VAPM, et al.                53. Tang YJ, Martin HG, Myers S, et al. Advances analysis of <br />     Reconciliation genome-scale metabolic reconstructions             microbial metabolic fluxes C-13 isotopic labeling. Mass <br />     comparative systems analysis. PLoS Computat Biol 2011;           Spectrom Rev 2009;28:362&#226;&#8364;&#8220;75. <br />     7:e1001116.                                                      54. Winter G, Kromer JO. Fluxomics&#226;&#8364;&#8221;connecting &#226;&#8364;&#732;omics ana- <br /> 36. Chindelevitch L, Stanley S, Hung D, et al. MetaMerge:                lysis phenotypes. Environ Microbiol 2012; doi: 10.1111/ <br />     scaling genome-scale metabolic reconstructions               1462-2920.12064 (Advance Access publication 6 December <br />     application Mycobacterium tuberculosis. Genome Biol 2012;         2012). <br />     13:r6.                                                           55. Aurrecoechea C, Heiges M, Wang H, et al. ApiDB: inte- <br /> 37. Pinney JW, Shirley MW, McConkey GA, et al.                           grated resources apicomplexan bioinformatics <br />     metaSHARK: software automated metabolic network                  resource center. Nucleic Acids Res 2007;35:D427&#226;&#8364;&#8220;30. <br />     prediction DNA sequence application          56. Balu B, Singh N, Maher SP, et al. A genetic screen for <br />     genomes Plasmodium falciparum Eimeria tenella. Nucleic        attenuated growth identifies genes crucial intraerythro- <br />     Acids Res 2005;33:1399&#226;&#8364;&#8220;409.                                          cytic development Plasmodium falciparum. PLoS One 2010; <br /> 38. Hung SS, Wasmuth J, Sanford C, et al. DETECT&#226;&#8364;&#8221;a                       5:e13282. <br />     density estimation tool enzyme classification        57. Alsford S, Turner DJ, Obado SO, et al. High-throughput <br />     application Plasmodium falciparum. Bioinformatics 2010;26:        phenotyping using parallel sequencing <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-325' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-351' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-360' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br />     1690&#226;&#8364;&#8220;8.                                                              targets African trypanosome. Genome Res 2011;21: <br /> 39. Kumar VS, Maranas CD. GrowMatch: automated                        915&#226;&#8364;&#8220;24. <br />     method reconciling silico/in vivo growth predictions.     58. Sacci JB, Ribeiro JMC, Huang FY, et al. Transcriptional <br />     PLoS Comput Biol 2009;5:e1000308.                                    analysis invivoPlasmodiumyoelii liver stage gene expression. <br /> 40. Bozdech Z, Llinas M, Pulliam BL, et al. The transcriptome            Mol Biochem Parasitol 2005;142:177&#226;&#8364;&#8220;83. <br />     intraerythrocytic developmental cycle Plasmodium       59. Tarun AS, Peng X, Dumpit RF, et al. A combined tran- <br />     falciparum. PLoS Biol 2003;1:85&#226;&#8364;&#8220;100.                                 scriptome proteome survey malaria parasite liver <br /> 41. Le Roch KG, Zhou YY, Blair PL, et al. Discovery gene              stages. Proc Natl Acad Sci USA 2008;105:305&#226;&#8364;&#8220;10. <br />     function expression profiling malaria parasite life    60. Le Roch KG, Johnson JR, Florens L, et al. Global analysis of <br />     cycle. Science 2003;301:1503&#226;&#8364;&#8220;8.                                      transcript protein levels Plasmodium falciparum <br /> 42. Daily JP, Scanfeld D, Pochet N, et al. Distinct physiological        life cycle. Genome Res 2004;14:2308&#226;&#8364;&#8220;18. <br />     states Plasmodium falciparum malaria-infected patients.    61. Llinas M, Bozdech Z, Wong ED, et al. Comp- <br />     Nature 2007;450:1091&#226;&#8364;&#8220;5.                                              arative genome transcriptome analysis three <br /> 43. Otto TD, Wilinski D, Assefa S, et al. New insights          Plasmodium falciparum strains. Nucleic Acids Res 2006;34: <br />     blood-stage transcriptome Plasmodium falciparum using             1166&#226;&#8364;&#8220;73. <br />     RNA-Seq. Mol Microbiol 2010;76:12&#226;&#8364;&#8220;24.                            62. Wastling JM, Xia D, Sohal A, etal. Proteomes transcrip- <br /> 44. Blazier AS, Papin JA. Integration expression data              tomes Apicomplexa&#226;&#8364;&#8221;where&#226;&#8364;&#8482;s message? Int J <br />     genome-scale metabolic network reconstructions. Front                Parasitol 2009;39:135&#226;&#8364;&#8220;43. <br />     Physiol 2012;3:299.                                              63. Nirmalan N, Sims PFG, Hyde JE. Quantitative proteomics <br /> 45. Florens L, Washburn MP, Raine JD, etal. A proteomic view             human malaria parasite Plasmodium falciparum and <br />     Plasmodium falciparum life cycle. Nature 2002;419:            application studies development inhibition. <br />     520&#226;&#8364;&#8220;6.                                                               Mol Microbiol 2004;52:1187&#226;&#8364;&#8220;99. <br />                      Functional genomics P. falciparum using metabolic modelling analysis                                 327 <br />  <br /> 64. Prieto JH, Koncarevic S, Park SK, et al. Large-scale differ-    77. Saliba KJ, Martin RE, Broer A, et al. Sodium-dependent <br />     ential proteome analysis Plasmodium falciparum drug        uptake inorganic phosphate  <br /> </body></html>