<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Integrin-a5 Coordinates Assembly Posterior Cranial <br /> Placodes Zebrafish Enhances Fgf-Dependent <br /> Regulation Otic/Epibranchial Cells <br /> Neha Bhat, Bruce B. Riley* <br /> Department Biology, Texas A&amp;M University, College Station, Texas, United States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Vertebrate sensory organs develop cranial placodes, series ectodermal thickenings coalesce a <br />      common domain preplacodal ectoderm. Mechanisms coordinating morphogenesis differentiation discrete <br />      placodes poorly understood. We investigated placodal assembly zebrafish requires Integrin- a5 <br />      (itga5), extracellular matrix receptor initially expressed preplacodal ectoderm. Morpholino knockdown of <br />      itga5 detectable effect anterior placodes (pituitary, nasal lens), posterior placodes developed abnormally, <br />      resulting disorganization trigeminal epibranchial ganglia reduction otic vesicle. Cell motion analysis in <br />      GFP-transgenic embryos showed cell migration itga5 morphants highly erratic unfocused, impairing <br />      convergence blocking successive recruitment new cells placodes. Further studies revealed genetic <br />      interactions itga5 Fgf signaling. First, itga5 morphants showed changes gene expression mimicking modest <br />      reduction Fgf signaling. Second, itga5 morphants showed elevated apoptosis otic/epibranchial domain, was <br />      rescued misexpression Fgf8. Third, knockdown Fgf effector erm effect itself strongly enhanced <br />      defects itga5 morphants. Finally, proper regulation itga5 requires dlx3b/4b pax8, regulated <br />      Fgf. These findings support model itga5 coordinates cell migration posterior placodes augments Fgf <br />      signaling required patterning tissues cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>survival</span> otic/epibranchial placodes. <br />  <br />   Citation: Bhat N, Riley BB (2011) Integrin-a5 Coordinates Assembly Posterior Cranial Placodes Zebrafish Enhances Fgf-Dependent Regulation Otic/ <br />   Epibranchial Cells. PLoS ONE 6(12): e27778. doi:10.1371/journal.pone.0027778 <br />   Editor: Christoph Winkler, National University Singapore, Singapore <br />   Received July 5, 2011; Accepted October 25, 2011; Published December 2, 2011 <br />   Copyright: &#195;&#376; 2011 Bhat, Riley. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: National Institutes Health-National Institute Deafness Communication Disorders (NIDCD), grant R01-DC03806. The funders role <br />   study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: briley@mail.bio.tamu.edu <br />  <br />  <br />  <br />  <br /> Introduction                                                                        migration rearrangement. For example, precursors the <br />                                                                                     anterior pituitary, olfactory lens placodes initially <br />    Development cranial sensory organs vertebrates requires                    intermixed subsequently sort form respective <br /> essential contributions transient embryonic structures termed                  placodes [9&#226;&#8364;&#8220;12]. In case olfactory placode, precursors <br /> cranial placodes. Cranial placodes form early segmentation                   converge compact placode chemotaxis mediated the <br /> stages series epithelial thickenings adjacent developing                 Sdf1-Cxcr4 chemokine signaling pathway [13]. Similarly, trigem- <br /> brain tissue [1,2]. The anterior-most placodes produce                          inal precursors initially widely scattered undergo <br /> anterior pituitary, olfactory epithelium, lens eye.                  Sdf1/Cxcr4-dependent chemotaxis converge coherent <br /> Amongst posterior placodes, otic placode produces                      placode [14]. Less known otic epibranchial <br /> entire inner ear, including complex epithelial labyrinth,                       placodes, zebrafish form rapid succession broad <br /> internal sensory epithelia, innervating neurons;                 field contiguous gene expression includes pax8, pax2a and <br /> trigeminal epibranchial placodes produce segmental array                   sox3 [15,16]. The otic domain forms induces epibranchial <br /> sensory ganglia innervate craniofacial                         development lateral cells [17]. The otic/epibranchial gene <br /> pharyngeal apparatus. Despite morphological functional                    expression domain undergoes marked contraction the <br /> diversity, cranial placodes arise common domain                       respective placodes coalesce, suggesting active cell migration and <br /> preplacodal ectoderm forms earlier anterior                         convergence. However, systematic studies of <br /> neural plate [2,3]. Specification preplacodal ectoderm involves                  cell migration associated formation otic epibranchial <br /> a sequence signaling interactions occur blastula                 placodes. It possible directed cell migration general <br /> gastrula stages, culminating expression characteristic set               feature common placodes, case important <br /> transcription factor genes near end gastrulation [4&#226;&#8364;&#8220;8]. This                 identify factors coordinate morphogenetic move- <br /> contiguous domain gene expression subsequently breaks                       ments. <br /> discrete clusters cells generate various diverse placodes.                 Directed cell migration involves navigation specific <br />    Lineage studies zebrafish chick indicate resolution               ECM domains, attachment requires cellular Integrins. <br /> preplacodal ectoderm discrete placodes requires active cell                    Integrins comprise a/b transmembrane heterodimers bind <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               1                          December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                               itga5 Fgf Regulate Placode Assembly <br />  <br />  <br /> Fibronectin Laminin ECM coordinate cell attachment,               morphants showed changes gene expression mimic the <br /> migration, differentiation survival [18&#226;&#8364;&#8220;21]. Integrin-ECM                  effects reducing Fgf signaling; knockdown Fgf- <br /> binding triggers signal transduction pathways, including               mediator erm, normally causes overt defects itself, <br /> Ras-MAPK PI3K signaling, regulate rapid reorganization                  significantly enhanced defects itga5 morphants. Finally, we <br /> of actin cytoskeleton changes gene expression. In            showed proper expression itga5 requires dlx3b, dlx4b and <br /> zebrafish, integrin- a5 (itga5) shown regulate number            pax8, regulated Fgf. These data support model in <br /> of early developmental processes, including formation regular               itga5 coordinates directed cell migration posterior <br /> somite boundaries proper differentiation cranial neural                 placodes augments Fgf signaling promote <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>cell survival</span> and <br /> crest [22&#226;&#8364;&#8220;24]. Expression initially widespread, near end            tissue patterning otic/epibranchial domain. <br /> of gastrulation itga5 restricted primarily preplacodal ectoderm <br /> [22]. However, studies role itga5              Results <br /> development preplacodal ectoderm derivatives. <br />    Here investigate role itga5 morphogenesis cranial           itga5 required proper development posterior <br /> placodes zebrafish. Impairment itga5 function caused                  placodes <br /> discernable change development anterior placodes,                       itga5 upregulates preplacodal ectoderm 10 hpf [22]. We <br /> posterior placodes showed number developmental defects                    hypothesized itga5 regulates morphogenetic movements <br /> resulting disorganization trigeminal epibranchial ganglia            associated formation discrete cranial placodes. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> test this <br /> and significant reduction size otic placode/otic                 idea, knocked itga5 using morpholinos monitored <br /> vesicle. To examine cell migration patterns, time lapse movies                 subsequent placodal development. We observed changes in <br /> were taken transgenic embryos expressing pax2a:GFP (otic/                   development anterior placodes, including anterior pituitary, <br /> epibranchial precursors) neuroD:EGFP (trigeminal precursors).              olfactory, lens placodes, judging morphology early <br /> Analysis control (non-morphant) embryos showed otic/               expression pitx3, cxcr4b foxe3, respectively (data shown). <br /> epibranchial trigeminal domains normally coalesce highly                In sharp contrast, posterior placodes, including trigeminal, otic <br /> focused convergence cells respective fields.              epibranchial placodes, showed variety developmental <br /> Furthermore, new cells continued enter pax2a:GFP                        defects. In control embryos, pax2a strongly expressed 12 hpf in <br /> expression domain lateral regions process                    otic cells, weaker expression epibranchial precursors just <br /> ongoing recruitment. In itga5 morphants, cell migration erratic            lateral otic domain (Fig. 1A). The otic cells appear to <br /> and unfocused, causing inefficient convergence, redistribution              undergo marked convergence 14 hpf form morphologically <br /> distal preotic cells epibranchial regions, failure                 visible otic placode (Figs. 1C). Convergence otic cells also <br /> recruitment new cells. Additionally, cells otic/epibran-             revealed expression otic marker fgf24 (Fig. 1E). In itga5 <br /> chial domain showed significantly elevated rate apoptosis,                morphants, initial otic/epibranchial domain pax2a was <br /> limiting increase epibranchial cells exacerbating               normal 12 hpf. However, otic placode appeared smaller <br /> deficiency otic cells. Further studies revealed strong genetic              normal 14 hpf (Fig. 1D, F), otic vesicle was <br /> interactions itga5 Fgf. For example, cell death                similarly reduced size 24 hpf (compare Fig. 1G, H). <br /> defect rescued misexpressing Fgf8. Furthermore, itga5                   Epibranchial ganglia, marked expression phox2a [25], were <br />  <br />  <br />  <br />  <br /> Figure 1. Knockdown itga5 impairs morphogenesis posterior cranial placodes. (A&#226;&#8364;&#8220;D) pax2a expression 12 14 hpf otic/ <br /> epibranchial domain control embryos (A, C) itga5 morphants (B, D). Expression normal 12 hpf itga5 morphants otic placode (o, <br /> brackets) smaller normal 14 hpf. (E, F) fgf24 expression 14 hpf control embryo (E) itga5 morphant (F). (G, H) Otic vesicles at <br /> 24hpf control embryo (G) itga5 morphant (H). (I, J) ngn1 expression 11 hpf control embryo (I) itga5 morphant (J). (K, L) neuroD <br /> expression 14 hpf control embryo (K) itga5 morphant (L). Precursors trigeminal ganglion (tg) anterior lateral line (al) are <br /> indicated. (M, N) Anti-Isl-1 immunostaining 24 hpf control embryo (M) itga5 morphant (N). (O, P) phox2a expression epibranchial <br /> ganglia 30 hpf control embryo (O) itga5 morphant (P). Facial (f), glossopharyngeal (g), vagal ganglia (v1+v2) indicated. A&#226;&#8364;&#8220;E, I&#226;&#8364;&#8220;K are <br /> dorsal views anterior top; G, H, M&#226;&#8364;&#8220;P lateral views anterior left. Scale bar, 50 mm. <br /> doi:10.1371/journal.pone.0027778.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                        December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                            itga5 Fgf Regulate Placode Assembly <br />  <br />  <br /> highly disorganized itga5 morphants, unclear                  overlap pax2a:GFP neuroD:EGFP, anterior <br /> whether epibranchial tissue altered (compare                non-otic domain contains precursors anterior lateral <br /> Fig. 1O, P). Early development trigeminal placodes                     line (data shown). Fourth, observed numerous cases in <br /> monitored expression ngn1, initially seen scattered         GFP-negative cells neighboring pax2a:GFP domain <br /> cells 11 hpf control embryos (Fig. 1I). Subsequently,                   subsequently entered domain GFP-positive (Fig. <br /> trigeminal cells converge form compact ganglia 14 hpf                   S2). Indeed, 29% tracked cells (12/42 cells embryos) <br /> [14,26], shown neuroD expression (Fig. 1K). In itga5                    exhibited <span id='am-395' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. Presumably cells induced to <br /> morphants, ngn1 expression normal 11 hpf (Fig. 1J)                 express pax2a migrated range inductive Fgf <br /> neuroD staining 14 hpf revealed trigeminal cells           signaling, reflecting process ongoing recruitment. <br /> scattered disorganized (Fig. 1L). Anti-Islet1/2 staining 24               Recordings itga5 morphants revealed number striking <br /> hpf showed trigeminal ganglia persisted disorganized                  differences control embryos. First, cells itga5 morphants <br /> clusters itga5 morphants (Fig. 1N). All defects          showed erratic migration patterns, frequent <br /> also observed itga5b926 mutants [22] (Fig. S1), confirming            changes direction (Fig. 2H&#226;&#8364;&#8220;J, Movie S2). Although total <br /> the itga5 morpholino specifically effectively phenocopies             distance traveled comparable seen control embryos, <br /> mutant. In summary, disruption itga5 does alter initial                meandering course cell migration itga5 morphants <br /> placodal development impairs assembly trigeminal, otic             resulted marked reduction net displacement (the straight- <br /> epibranchial placodes induction. These defects               line distance start finish). Accordingly, mean efficiency <br /> relatively specific gross morphology head appeared                  migration (net displacement/total distance) .496.15 in <br /> normal morphants mutants.                                              itga5 morphants compared .716.08 control embryos (Fig. 2I; <br />                                                                               Table 1). Moreover, range angle net displacement was <br /> Morphogenesis otic/epibranchial domain                                 greater quadrants itga5 morphants (Fig. 2J). Second, <br />     We hypothesized abnormal development posterior                inefficient cell migration led aberrant partitioning the <br /> placodes seen itga5 morphants arose defective              pax2a:GFP domain itga5 morphants. Specifically, cells <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> cell migration. To test this, tracked cell movements time-              quadrant (0/6), fewer half cells the <br /> lapse movies using transgenic backgrounds precursor                  second quadrant (6/13), migrated otic placode (Fig. 2I,J). <br /> cells labeled GFP. To monitor otic/epibranchial                     This contrast control embryos half (3/6) cells in <br /> morphogenesis, used pax2a:GFP transgenic line                       quadrant 82% (9/11) cells second quadrant <br /> recapitulates expression endogenous pax2a otic/                 converged otic placode (Fig. 2F,G). Consequently, itga5 <br /> epibranchial domain midbrain-hindbrain border [27].                       morphants formed smaller otic placode correspondingly <br /> Additionally, pax2a:GFP line shows ectopic expression                  enlarged anterior domain non-otic cells. Third, observed no <br /> rhombomeres 3 5 hindbrain, providing convenient spatial              examples recruitment neighboring cells pax2a:GFP <br /> marker help gauge positions individual cells. In pilot studies          domain itga5 morphants. That is, cells tracked itga5 <br /> we difficult track individual cells         morphants (43/43 cells 4 embryos) RFP-positive and <br /> hours multi-layered otic placode. To enhance cell-                 GFP-positive recording period, no <br /> discrimination, injected transgenic embryos plasmid DNA               neighboring RFP-positive cells observed enter the <br /> to express RFP mosaically, permitting independent tracking                 pax2a:GFP domain GFP-positive. Finally, unlike <br /> RFP GFP individual cells (see Materials Methods).                  control embryos, itga5 morphants showed striking incidence of <br /> Embryos recorded time-lapse 11.5 hpf to14.5 hpf. An              cell-lysis pax2a:GFP domain. For example, 12/43 (27%) <br /> example early late frames movie control embryo               RFP-positive cells tracked itga5 morphants lysed the <br /> (Movie S1) shown Fig. 2A,B. Tracks RFP-labeled cells              course recording (Fig. 2C,D,H,I), 42 cells <br /> the specimen summarized Fig. 2E. Most cells migrated              tracked control embryos observed lyse (Fig. 2F). In <br /> in relatively straight line contribute otic placode,         contrast, detected consistent changes rate cell <br /> little deviation angle migration. Similar patterns             division itga5 morphants compared controls based the <br /> documented total control embryos. The net                        pattern BrdU incorporation (Fig. S3). Additionally, tracking of <br /> displacement RFP-positive cells (n = 42) tracked              lateral RFP-positive cells, entered otic- <br /> five embryos represented Fig. 2F. To assess typical cell                epibranchial domain, revealed cell migration patterns are <br /> behaviors different regions, divided pax2a:GFP domain               relatively normal outside domain elevated itga5 expression <br /> into quadrants anteroposterior axis calculated              (Fig. S4, Table 1). Together data itga5 required <br /> mean net displacement range angle displacement              specifically normal migration survival cells otic/ <br /> for cells quadrant (Fig. 2G). Several conclusions regarding           epibranchial domain. In absence itga5, otic placode is <br /> patterns migration emerged analysis. First, cells                reduced size combination inefficient convergence, <br /> beginning near medial edge pax2a:GFP domain usually                increased cell death, faulty recruitment new cells the <br /> migrated short distances build otic               pax2a domain. <br /> placode, cells lateral positions migrated much <br /> longer distances predominant medial vector. Second,                 Retrospective tracking epibranchial precursors <br /> addition general medial migration, marked                  Although data provided detailed information about <br /> centripetal convergence otic placode broader                otic placode, random RFP-labeling marked cells the <br /> field GFP-positive cells. That is, cells anterior end          non-otic domains (5/42 control embryos, Fig. 2F). However, we <br /> of field showed posterior trajectory cells posterior       able track additional cells non-otic domain to <br /> showed anterior trajectory. Third, labeled cells        origins based solely GFP expression. Cells tracked this <br /> and second quadrants did contribute otic placode               way showed migration patterns consistent previously <br /> instead migrated position just anterior otic placode.           tracked RFP-labeling (Fig. 3A). The majority cells <br /> The anterior posterior non-otic domains pax2a:GFP                      contributing anterior non-otic domain converged the <br /> contribute predominantly epibranchial ganglia and, based                regions adjacent quadrants cells the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                       December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                              itga5 Fgf Regulate Placode Assembly <br />  <br />  <br />  <br />  <br /> Figure 2. Otic/epibranchial precursors aberrant migration itga5 morphants. (A&#226;&#8364;&#8220;D) Images time-lapse movies showing <br /> transgenic expression pax2a:GFP (green) mosaic expression cmv:RFP (red). The frame (11.5 hpf) final frame (14.5 hpf) control <br /> movie (A, B) itga5 morphant movie (C, D) shown. Arrows indicate cells expressed GFP RFP. Blue arrows indicate cells that <br /> contributed otic domain, white arrows indicate cells contributed non-otic domains. Red arrows indicate cells lysed the <br /> recording period (C, D). Positions rhombomeres 3 5 (r3, r5) indicated. (E, H) Maps showing trajectories marked cells the <br /> embryos recorded A&#226;&#8364;&#8220;D. Trajectories red denote cells lysed recording (H). The origins cell trajectories marked dots. The <br /> initial final positions pax2a:GFP domain indicated purple green boundaries, respectively. Final positions otic placode and <br /> non-otic domains indicated. (F, I) Vector maps showing net displacement cells tracked 5 control movies (F) 4 itga5 morphant movies <br /> (I). Red arrows indicate cells died recording (I). (G, J) Summaries average migration patterns cells different quadrants the <br /> pax2a:GFP domain control embryos (G) itga5 morphants (J). Arrow <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> indicates mean net displacement cells indicated <br /> region, colored cones represent range angle net displacement. Quadrants 1 2 contained cells contributing otic non-otic <br /> domains, grouped separately. All images depict right half embryo lateral right anterior top. Scale bar, <br /> 50 mm. <br /> doi:10.1371/journal.pone.0027778.g002 <br />  <br />  <br /> posterior domain converged fourth quadrant (Fig. 3B).                keeping lateral origins, cells recruited <br /> Surprisingly, majority (15/19) cells originated             relatively late recording period. Whereas recruitment of <br /> lateral regions outside domain contiguous pax2a:GFP                    otic cells usually occurred 12 hpf (range, 11.7&#226;&#8364;&#8220;12.5 hpf), <br /> expression, appearing sparsely scattered GFP-positive cells          recruitment non-otic cells occurred 12.7 hpf (range, <br /> before migrating non-otic domains (Fig. 3A,B). Moreover,             12.5&#226;&#8364;&#8220;13.3 hpf) (Fig. S2). This consistent previous <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         4                        December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                                    itga5 Fgf Regulate Placode Assembly <br />  <br />  <br />  <br />  Table 1. Knockdown itga5 impairs efficiency directed cell migration. <br />  <br />  <br />                                                                              Net displacement            Total distance         Efficiency migration (net/ <br />  Transgenic marker                    Experimental condition                 (mm)&#194;&#177;SD                     (mm)&#194;&#177;SD                total mm)&#194;&#177;SD <br />  <br />  pax2a:gfp (otic/epibranchial)        control itga5mo itga5mo+hs:fgf8        49.4617.0 32.1612.5{        68.7622.5 64.9624.9{   0.7160.08 0.4960.15{ <br />                                                                              28.8611.3{                  61.068.9{              0.4660.17{ <br />  pax2a:gfp (otic/epibranchial)        control-mosaic* itga5mo-mosaic*        54.9611.1 28.3613.3{        68.8613.4 71.0619.3    0.7960.04 0.4060.17{ <br />  neuroD:gfp (trigeminal)              control itga5mo itga5mo+hs:fgf8        56.669.1 44.4617.0{         76.8610.5 97.7627.3{   0.7460.06 0.4560.13{ <br />                                                                              35.9621.2{                  61.4619.4{             0.5460.20{ <br />  non-placodal cells                   control itga5mo                        54.7612.3 40.466.2          70.7614.6 61.3611.4    0.7660.02 0.6660.10 <br />  <br />  *Indicates genotype RFP-labeled cells transplanted pax2a:gfp host embryo. <br />  { <br />   Significantly different control value based t-test. <br />  doi:10.1371/journal.pone.0027778.t001 <br />  <br />  <br /> findings otic cells specified earlier and,                             In itga5 morphants, cell migration highly erratic the <br /> upregulation fgf24, subsequently induce epibranchial precursors                   majority (24/27) non-otic cells originated the <br /> from adjacent tissue [17].                                                           contiguous domain pax2a:GFP expression (Fig. 3C,D). Addi- <br />                                                                                      tionally, anterior posterior non-otic domains were <br />                                                                                      enlarged expense otic placode. Despite these <br />                                                                                      abnormalities, general mechanism sequential induction of <br />                                                                                      epibranchial precursors operate relatively normally <br />                                                                                      itga5 morphants based expression fgf24 (Fig. 1F) sox3 <br />                                                                                      (see below). Note did observe cell lysis amongst <br />                                                                                      retrospectively tracked cells technique focused solely <br />                                                                                      cells survived end recording period. <br />                                                                                      Together, data confirmed itga5 required normal <br />                                                                                      migration survival cells contributing non-otic domains. <br />  <br />                                                                                      Cell autonomous requirement itga5 function <br />                                                                                         To examine defect morphogenesis otic/ <br />                                                                                      epibranchial precursors direct effect itga5 knockdown, we <br />                                                                                      transplanted itga5 morphant cells wild-type embryos and <br />                                                                                      tracked migration using time-lapse imaging 11.5&#226;&#8364;&#8220;14.5 <br />                                                                                      hpf. As non-mosaic itga5 morphants, migration isolated itga5- <br />                                                                                      MO cells wild-type host embryos highly erratic (Fig. 4), with <br />                                                                                      migration efficiency 0.4460.03 (n = 16, Table 1). <br />                                                                                      Additionally, 19% tracked cells (n = 3/16) lysed during <br />                                                                                      recording. In contrast, wild-type cells transplanted into <br />                                                                                      wild-type hosts, migration efficiency 0.7160.08 (n = 7) no <br />                                                                                      cell lysis detected. These data itga5 required cell <br />                                                                                      autonomously proper migration survival cells the <br />                                                                                      otic/epibranchial domain. <br />  <br />                                                                                      Morphogenesis trigeminal placode <br />                                                                                          We conducted time-lapse analysis morphogenesis the <br />                                                                                      trigeminal placode neuroD:EGFP transgenic embryos, which <br />                                                                                      begin express GFP trigeminal precursors 11.5 hpf [28]. <br /> Figure 3. Trajectories non-otic cells tracked reverse. (A, C)                  Because small number broad distribution of <br /> Maps showing trajectories GFP positive cells pooled                  neuroD:EGFP cells, readily track trigeminal <br /> time-lapse movies Fig. 2 control embryos (A) itga5 morphants               precursors duration recording did not <br /> (C). Trajectories orange indicate cells originally tracked coexpres-           require additional RFP plasmid injection. Consistent with <br /> sion GFP RFP. All trajectories represent cells tracked                  endogenous gene expression data (Fig. 1), trigeminal cells in <br /> retrospectively GFP alone. Dots indicate origins tracked cells. <br /> Trajectories red denote cells died recording (C). (B, D) <br />                                                                                      control embryos initially scattered 11.5hpf rapidly <br /> Summaries average migration patterns non-otic cells different               converged compact placode 14 hpf (Fig. 5A&#226;&#8364;&#8220;C, Movie <br /> quadrants control embryos (B) itga5 morphants (D). The mean                   S3). Tracks individual cells relative straight showed <br /> length net displacement (arrows) range angle net                        little deviation (Fig. 5C). In itga5 morphants, initial distribution <br /> displacement (colored cones) indicated. The dashed orange lines                  neuroD:EGFP cells normal 11.5 hpf subsequent <br /> indicate regions non-otic cells originated. Initial final             convergence severely impaired (Fig. 5D&#226;&#8364;&#8220;F, Movie S4). <br /> positions pax2a:GFP domain represented purple and <br /> green boundaries, respectively. The position rhombomere 5 (r5) is <br />                                                                                      Migration trigeminal cells highly erratic frequent <br /> indicated. Lateral right anterior top. Scale bar,            changes direction (Fig. 5F). Hence mean efficiency of <br /> 50 mm.                                                                               migration smaller normal (Table 1). Moreover, <br /> doi:10.1371/journal.pone.0027778.g003                                                cells failed converge 14 hpf (Figs. 1J, 5E). Trigeminal <br />  <br />  <br />        PLoS ONE | www.plosone.org                                               5                        December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                              itga5 Fgf Regulate Placode Assembly <br />  <br />  <br />  <br />  <br /> Figure 4. Cell-autonomous requirement Itga5 otic/ <br /> epibranchial cells. (A) Trajectories cells transplanted wild <br /> type donor wild type host tracked time-lapse 11.5&#226;&#8364;&#8220; <br /> 14.5 hpf. Data tracks 7 cells single embryo. (B) <br /> Trajectories cells transplanted itga5 morphant donors wild <br /> type host embryos tracked time-lapse 11.5&#226;&#8364;&#8220;14.5 hpf. A <br /> total 16 cells 3 embryos tracked, not <br /> included map avoid confusion. Red tracks represent cells that <br /> underwent lysis time-lapse. The purple green <br /> boundaries represent initial final pax2a domain time-          Figure 5. Convergence trigeminal precursors impaired in <br /> lapse recording. Dots represent initial positions cells. Images        itga5 morphants. (A, B, D, E) Images time-lapse movies showing <br /> show dorsal views anterior top. Scale bar, 50 mm.                 transgenic neuroD:EGFP expression (11.5 hpf) final (14 <br /> doi:10.1371/journal.pone.0027778.g004                                         hpf) frames control movie (A, B) itga5 morphant movie (D, E). <br />                                                                               Positions precursors trigeminal ganglion (tg) anterior <br /> cells did eventually converge form poorly organized clusters            lateral line (al) indicated. (C, F) Maps showing trajectories of <br />                                                                               individual trigeminal precursors control embryo (C) itga5 <br /> 15.5 hpf (not shown). Thus, itga5 required efficient migration <br />                                                                               morphant (F). Black red boundaries mark initial final <br /> and convergence trigeminal cells otic/epibranchial            distribution, respectively, neuroD:EGFP-positive trigeminal precursors. <br /> cells. However, did observe cell death neuroD:EGFP              Black dots represent initial final positions, respectively, of <br /> expressing trigeminal cells itga5 morphants. Thus require-             individual cells. Images dorsal views anterior top, and <br /> ment survival appears restricted otic/epibranchial               summary figures right trigeminal field embryo, with <br /> precursors.                                                                   lateral right. Scale bar, 50 mm. <br />                                                                               doi:10.1371/journal.pone.0027778.g005 <br />  <br /> itga5 regulates cell survival otic/epibranchial <br />                                                                               Fgf signaling promote cell survival. To test idea, used a <br /> domain                                                                        heat shock-inducible transgene, hs:fgf8 [29], elevate Fgf signaling <br />    To confirm cell-lysis observed time-lapse               itga5 morphants. Activation hs:fgf8 expression 11.5 hpf had <br /> imaging otic/epibranchial precursors (Figs. 2,3,4)               effect apoptosis non-morphants (Fig. 6C) itga5 <br /> apoptosis, visualized apoptotic cells staining                  morphants reduced number Caspase-3 positive apoptotic <br /> antibody directed activated Caspase-3. This analysis              cells normal 13.5 hpf (Fig. 6D,E). These data support the <br /> performed pax2a:GFP transgenic embryos help visualize               hypothesis itga5 promotes cell survival through <br /> otic/epibranchial region. We observed fold higher               augmenting Fgf signaling. <br /> levels apoptosis itga5 morphants compared control                     We examined effects Fgf misexpression cell <br /> embryos (Fig. 6A,B,E). Elevated apoptosis limited otic/            migration itga5 morphants. Activation hs:fgf8 11.5 hpf in <br /> epibranchial domain did extend anterior                       itga5 morphants did rescue cell migration defects seen in <br /> ectoderm near developing trigeminal placodes (data                    otic/epibranchial trigeminal domains (Fig. S5, Table 1). <br /> shown). Similar results obtained using Acridine Orange (data             Thus, elevating Fgf signaling bypass requirement for <br /> not shown). We observed significant increase number             itga5 regulating coordinated cell migration. <br /> of apoptotic cells itga5b926 mutants (Fig. S1). These data indicate            We tested itga5 influences Fgf-dependent gene <br /> that itga5 specifically required survival cells otic/        expression. Although genes Fgf-feedback pathway, including <br /> epibranchial domain.                                                          sprouty4, pea3 erm [30&#226;&#8364;&#8220;32], used indicators the <br />                                                                               presence Fgf, sensitive detect modest <br /> itga5 cooperates Fgf regulate otic/epibranchial                       reduction level Fgf signaling [33]. However, recently <br /> survival development                                                      reported sox3 expression provides sensitive readout of <br />    Previous studies shown Integrin-ECM binding                  changing levels Fgf signaling otic/epibranchial region, <br /> promote cell survival activation MAPK PI3K                     distinct threshold responses Fgf [17]. Specifically, sox3 <br /> signal transduction. Because otic/epibranchial induction                  initially induced high level otic domain response to <br /> maintenance requires Fgf signaling, operates                   moderate Fgf signaling otic induction [15,16]. Subsequent- <br /> MAPK PI3K activity, hypothesized itga5 augments                   ly, response otic expression fgf24, sox3 expression <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                        December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                             itga5 Fgf Regulate Placode Assembly <br />  <br />  <br />                                                                                high level activation dnfgfr1 ablated sox3 entirely (Fig. 7E, <br />                                                                                F). In contrast, activation hs:fgf8 resulted low level sox3 <br />                                                                                expression otic-epibranchial domain both <br />                                                                                control embryos itga5 morphants (Fig. 7C, D). Thus, the <br />                                                                                effects itga5 knockdown sox3 expression mimic effects of <br />                                                                                modest reduction Fgf signaling reversed by <br />                                                                                overexpression fgf8, supporting notion itga5 normally <br />                                                                                augments Fgf signaling. <br />                                                                                   Note changes sox3 expression not <br />                                                                                necessarily reflect changes cell fate. For example, retention of <br />                                                                                high levels sox3 otic domain itga5 morphants, in <br />                                                                                embryos weakly expressing dnfgfr1, does indicate wholesale <br />                                                                                switching otic epibranchial fate embryos still <br />                                                                                produce substantial otic vesicles. Likewise, retention elevated <br />                                                                                sox3 otic domain fgf24 morphants does impair otic <br />                                                                                development, does forced over-expression sox3 [17]. Thus, <br />                                                                                knockdown itga5 appears cause modest reduction in <br />                                                                                cell signaling sufficient alter sox3 expression, cell fate, <br />                                                                                otic domain. <br />                                                                                   To explore relationship itga5 MAPK <br />                                                                                signaling, examined genetic interaction itga5 and <br />                                                                                erm, direct MAPK-target helps mediate effects Fgf <br />                                                                                signaling [31,32,34]. Because erm partially redundant pea3, <br />                                                                                knockdown erm negligible effects gross morphology <br />                                                                                [34, unpublished observations], exception a <br />                                                                                variable incidence otolith deficiencies otic vesicle <br /> Figure 6. Elevated cell death itga5 morphants rescued by <br />                                                                                (Fig. 8G). In addition, early placodal development appears normal <br /> hs:fgf8. (A&#226;&#8364;&#8220;D) Transgenic pax2a:GFP embryos immunostained GFP <br /> (green) Caspase3 (red). Images dorsal views (anterior up)          erm morphants (Fig. 8C), otic expression sox3 shows <br /> the right otic/epibranchial domain control embryo (A), itga5              slightly pronounced downregulation compared control <br /> morphant (B), hs:fgf8/+ embryo (C) hs:fgf8/+ embryo injected          embryos (Fig. 7G). Thus erm morphants provide sensitized <br /> itga5-MO (D). White arrows mark apoptotic cells. All embryos heat         background detecting reduction MAPK signaling. As <br /> shocked 39uC 30 minutes beginning 11.5hpf fixed 13.5          observed earlier, itga5 morphants showed reduced otic <br /> hpf. Scale bar, 50 mm. (E) Mean number Caspase3-positive cells the <br /> otic/epibranchial domain groups embryos. Error <br />                                                                                tissue (Fig. 8B, F), simultaneously knocking erm <br /> bars indicate S.E.M.                                                           enhanced deficiency (Fig. 8D, H). While itga5 morphants <br /> doi:10.1371/journal.pone.0027778.g006                                          produced abundant epibranchial neurons, albeit disorganized <br />                                                                                pattern (Fig. 8J), erm morphants showed marked deficiency of <br />                                                                                epibranchial ganglia (Fig. 8K) erm-itga5 double morphants <br /> downregulates discrete lower level otic domain              produced (Fig. 8L). Thus, itga5 erm work <br /> induced higher level abutting epibranchial tissue [17]. We             control otic epibranchial development, possibly by <br /> therefore reasoned itga5 morphants experience decreased                acting pathway. These data, the <br /> MAPK PI3K signaling, expression sox3 remain                       effects itga5-MO sox3 expression ability hs:fgf8 to <br /> elevated otic domain. In support, itga5 morphants showed                rescue cell death <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> itga5 morphants, support the <br /> equally heavy expression sox3 otic epibranchial                  idea itga5 normally augments Fgf signaling promote cell <br /> domains (Fig. 7B). Similar changes observed following low                 survival regulate differential gene expression otic/ <br /> level misexpression dominant-negative Fgf receptor (hs:dnfgfr1),            epibranchial precursors. In contrast, itga5 provides unique <br />  <br />  <br />  <br />  <br /> Figure 7. Similar effects Itga5 Fgf sox3 expression. (A&#226;&#8364;&#8220;H) sox3 expression 12.5 hpf control embryo (A), itga5 morphant (B), <br /> Tg(hs:fgf8/+) heat shocked 37uC (C) itga5 morpholino (D), Tg(hs:dnfgfr1/+) heat shocked 39uC (E) 35uC (F), erm morphant (G) and <br /> itga5-erm double morphant (H). The otic region sox3 normally downregulates indicated. Scale bar, 50 mm. <br /> doi:10.1371/journal.pone.0027778.g007 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          7                       December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                              itga5 Fgf Regulate Placode Assembly <br />  <br />  <br />  <br />  <br /> Figure 8. itga5 erm interact otic epibranchial development. (A&#226;&#8364;&#8220;D) Otic/epibranchial expression pax2a 13 hpf. (E&#226;&#8364;&#8220;H) <br /> Otic vesicle morphology 27 hpf. (I&#226;&#8364;&#8220;L) phox2a expression epibranchial ganglia 30 hpf. Positions facial (f), glossopharyngeal (g) vagal <br /> (v1 v2) ganglia indicated. All images lateral views dorsal anterior left. Scale bar, 50 mm. <br /> doi:10.1371/journal.pone.0027778.g008 <br />  <br /> function required directed cell migration                   expression expands otic/epibranchial domain further <br /> replaced elevating Fgf, likely                 intensifies 13 hpf (Fig. 9B). The spatial pattern itga5 <br /> influence cells experience Fgf signaling (see Discussion).            expression strongly resembles dlx3b dlx4b, the <br />                                                                                earliest preplacodal markers zebrafish required <br /> Regulation itga5 expression                                                 proper development placodal derivatives [7,35,36]. <br />    To elucidate complete picture pathway                We examined dlx3b/4b genes required for <br /> itga5 acts, sought identify upstream regulators itga5             proper expression itga5. Knockdown dlx3b/4b did prevent <br /> expression. Expression itga5 shows elevated expression                 initiation preplacodal expression itga5, subsequent <br /> horseshoe-shaped pattern marking preplacodal ectoderm 10                upregulation maintenance itga5 anterior preplacodal <br /> hpf [22]. By 11 hpf expression strongly upregulates anterior-           domain severely impaired (Fig. 9C, D). Surprisingly, <br /> most portion preplacodal domain (Fig. 9A), gives rise             however, expression itga5 otic/epibranchial ectoderm <br /> to anterior pituitary, nasal lens placodes [11]. By 12 hpf             domain relatively normal dlx3b/4b double morphants <br />                                                                                (Fig. 9D). Because pax8 regulates early aspects otic and <br />                                                                                epibranchial development [17,37,38], examined expression of <br />                                                                                itga5 pax8 morphants. Expression itga5 normal pax8 <br />                                                                                morphants 11 hpf (Fig. 9E) expression failed to <br />                                                                                upregulate properly otic domain 13 hpf (Fig. 9F, and <br />                                                                                data shown). In contrast, itga5 expression anterior <br />                                                                                preplacodal domain relatively normal pax8 morphants. In <br />                                                                                summary, dlx3b/4b required upregulation maintenance <br />                                                                                itga5 anterior preplacodal ectoderm, pax8 required <br />                                                                                upregulation otic/epibranchial domain. <br />  <br />                                                                                Discussion <br />                                                                                   We shown essential role itga5 proper formation of <br />                                                                                trigeminal, otic epibranchial placodes. Cell motion analysis in <br />                                                                                GFP-transgenic backgrounds shows cells normally converge <br />                                                                                broad fields compact placodes highly directed <br />                                                                                migration. Overall, cells contributing diverse placodes <br />                                                                                originate distinct regions limited overlap (Fig. 10A). <br />                                                                                The data otic epibranchial domains <br />                                                                                continue expand gastrulation ongoing recruitment <br />                                                                                cells lateral nonneural ectoderm (Fig. 10B). In itga5 <br />                                                                                morphants, cell migration directional: Though the <br />                                                                                total distance traveled comparable normal, cells frequently <br /> Figure 9. Differential spatial regulation itga5 dlx3b/4b             change direction backtrack, causing significant impairment of <br /> pax8. (A&#226;&#8364;&#8220;F) itga5 expression 11 hpf 13 hpf control embryos           convergence complete abrogation recruitment. Loss itga5 <br /> (A, B), dlx3b-dlx4b double morphants (C, D), pax8 morphants (E, <br />                                                                                impairs patterning cell survival otic/epibranchial <br /> F). Regions expression normally upregulates precursors of <br /> anterior placodes (arrows) otic/epibranchial precursors (arrow-            domain. Several lines evidence indicate itga5 interacts with <br /> heads) indicated. Images dorsal views anterior top.       Fgf signaling regulate region. First, misexpression Fgf8 is <br /> doi:10.1371/journal.pone.0027778.g009                                          sufficient rescue cell death phenotype itga5 morphants <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          8                        December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                              itga5 Fgf Regulate Placode Assembly <br />  <br />  <br />  <br />  <br /> Figure 10. Model regulation posterior placode development itga5. (A) Summary cell migration morphogenesis of <br /> trigeminal (t), epibranchial (e) otic (o) placodes. Arrows indicate migration routes cells tracked neuroD:EGFP pax2a:GFP expression <br /> domains (green). Dashed circles indicate general areas trigeminal epibranchial precursors tracked. Most epibranchial <br /> precursors detected relatively late (12.5&#226;&#8364;&#8220;13.2 hpf) activated pax2:GFP scattered pattern lateral the <br /> contiguous domain expression. We infer cells originated lateral positions zone recruitment (dashed tracks). RFP- <br /> positive otic cells observed migrate pax2a:GFP domain nearby zone recruitment (red tracks) 11.7&#226;&#8364;&#8220;12.5 hpf. <br /> Positions midbrain-hindbrain border (MHB) rhombomere 4 (r4) indicated. (B) A model recruitment otic/epibranchial cells. An <br /> initial otic domain (green) induced dorsally expressed Fgfs. Subsequently, itga5-dependent medial migration drives convergence otic field <br /> and draws new cells range inductive signaling. (C) Model itga5 reinforcing Fgf signaling. Erm helps mediate Fgf signaling, begets <br /> more erm expression. Fgf activates pax8, stimulates upregulation itga5 otic/epibranchial domain, reinforcing Fgf signaling. <br /> Fgf acts primarily MAPK PI3K pathways, Itga5 facilitates pathways variety mechanisms (see text details). <br /> Together functional interactions control directed cell migration, cell survival, gene expression otic/epibranchial field. <br /> doi:10.1371/journal.pone.0027778.g010 <br />  <br /> (Fig. 6). Second, itga5 morphants changes sox3 expression              Instead, Itga5 help provide directionality sensing <br /> that mimic effects reduced Fgf signaling, effect           adhesive gradients ECM (haptotaxis). Fibronectin-1 (Fn1) is <br /> be reversed misexpressing fgf8 (Fig. 7). Third, knockdown erm            ECM component strongly binds Itga5 expressed <br /> enhances patterning defects itga5 morphants (Fig. 8). Finally,          maximally mesoderm just beneath neural-nonneural border <br /> proper upregulation itga5 requires Fgf-target gene pax8                 [39,40], possibly providing cue medial migration. Addition- <br /> (Fig. 9), suggesting itga5 acts feed-forward loop                 ally, Integrin-ECM binding appears necessary restricting <br /> reinforce Fgf signaling otic/epibranchial domain. Together,             pseudopod production leading edge migratory cells <br /> our data support model itga5 coordinates cell migration,            [21]. Otherwise, cells tend extend multiple pseudopods in <br /> differentiation survival posterior cranial placodes,         random directions, resulting frequent changes direction <br /> enhancing Fgf signaling (Fig. 10C).                                            similar observed itga5 morphants. Dynamic <br />                                                                                regulation Integrins helps stabilize pseudopods leading <br /> Role Integrin directed cell migration                                    edge chemotaxis cell types [18] plays vital <br />    Preplacodal ectoderm forms end gastrulation              role modulating response T-lymphocytes chemokines <br /> quickly breaks discrete placodes directed cell                    [41]. The chemokine Sdf1 required focal migration of <br /> migration [9&#226;&#8364;&#8220;14]. Integrins good candidates regulating                 trigeminal precursors [14], knockdown itga5 causes a <br /> such morphogenetic processes, previous                  phenotype similar loss Sdf1 receptor Cxcr4. It not <br /> studies role Integrins placodal development                known specific chemokines regulate convergence otic <br /> species. We chose study itga5 zebrafish initially          epibranchial cells. Otic placode cells express chemokine <br /> expressed low level preplacodal ectoderm               Cxcl14 (Scyba) [42], knockdown gene causes no <br /> later upregulates various placodal derivatives form [22]            discernable defects (our unpublished observations). Epibranchial- <br /> (Fig. 9). Surprisingly, placodes derived anterior portion          specific chemokines identified. <br /> the preplacodal ectoderm (anterior pituitary, olfactory lens <br /> placodes) appear develop normally itga5 morphants. In                    Role Integrin cell signaling <br /> contrast, itga5 knockdown impairs directed migration precursors                Itga5 facilitates signaling associated specification and <br /> in posterior placodes, resulting disorganization trigeminal              survival otic/epibranchial precursors, probably multi- <br /> and epibranchial ganglia significant deficiency otic tissue.            ple distinct mechanisms. First, itga5-dependent migration draws <br /> Interestingly, cells observed migrate long distances itga5          lateral cells range inductive Fgf signaling, thereby <br /> morphants albeit non-directed manner, indicating Itga5               promoting recruitment additional cells placodes <br /> does simply provide traction migratory substrate.                   (Fig. 10B). We showed previously number pax2a- <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          9                        December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                            itga5 Fgf Regulate Placode Assembly <br />  <br />  <br /> positive cells roughly doubles 11 hpf 14 hpf,               domain critical period. The transition possibly related <br /> similar fold-increase seen mutants blocked cell division,            long cells exposed varying Fgf concentrations, which <br /> leaving recruitment means available increase cell              co-regulate fate-specification activation genes that <br /> number [43]. Second, Integrin-ECM binding required                 mediate contact-dependent repulsion. Further studies needed <br /> cell types proper nuclear import phosphorylated MAPK/                  test ideas. <br /> ERK [44&#226;&#8364;&#8220;46]. Otherwise, ERK continue phosphory- <br /> lated accumulate ineffectually cytosol. Third, itga5         Materials Methods <br /> specifically amplify Fgf signaling direct physical interac- <br /> tions. In mammalian vascular endothelial cells, example,                  Strains <br /> Integrin-avb3 binds Fgf1 Fgf2 directly form ternary                     Control embryos derived AB line (Eugene, OR). <br /> complex FGFR1 promotes receptor clustering [47&#226;&#8364;&#8220;49].                  In various experiments used transgenic lines Tg(pax2a:GFP)e1 <br /> Such binding interactions required sustained MAPK/ERK                 [27], Tg(neuroD:EGFP)nl1 [28] Tg(hsp70:fgf8)x17 [29], the <br /> signaling. Itga5 stimulate PI3K [50], signal                 mutant line itga5b926 [22]. <br /> transducer shared Fgf pathway regulates cell migration <br /> and fate specification variety cell types [51&#226;&#8364;&#8220;53].                    In-situ hybrization immunostaining <br />    In case otic/epibranchial development, itga5                   In-situ hybridization performed described [54]. <br /> required sustained Fgf signaling appears necessary                Primary antibodies used Isl-1 (Hybridoma Bank, 1:100), <br /> achieving level Fgf signaling. Although knockdown itga5          Caspase3 (R&amp;D Systems, 1:100) GFP (Santa Cruz Biotech- <br /> does detectably alter expression genes Fgf                      nology, 1:250). Secondary antibodies HRP-conjugated anti- <br /> synexpression group, erm, pea3 sprouty4, genes            mouse (Vector Labs, 1:200) Alexa-conjugated anti-mouse and <br /> not suitable detecting modest reduction Fgf signaling                  anti-rabbit (Invitrogen, 1:50). <br /> [17,33]. Fortunately, sox3 expression shows distinct threshold- <br /> responses Fgf signaling: Moderate Fgf signaling induces high             Morpholino injections <br /> level sox3 expression epibranchial domain                     Morpholino sequences itga5, dlx3b, dlx4b pax8 been <br /> elevated Fgf signaling downregulates sox3 expression discrete            previously described tested [23,36,38]. For erm knockdown we <br /> lower level otic domain [17]. The finding sox3                    used erm-MO 59-CTTGCTGGTCATAAAACCCATCCAT-39, <br /> expression inappropriately maintained high level otic          nearly identical etv5/erm-MO described previously <br /> placode itga5 morphants (Fig. 7B) provides evidence Fgf               [34]. Embryos injected one-cell stage 5 ng of <br /> signaling diminished domain. Furthermore,                   indicated MOs. <br /> reduced signaling itga5 morphants explains elevated cell <br /> death seen otic/epibranchial domain,                     RFP misexpression <br /> activating hs:fgf8 rescues phenotype (Fig. 5). Nevertheless,            CMV:RFP plasmid injected one-cell stage embryos at <br /> is interesting sox3 shows normal upregulation                     concentration 75 ng/ul mosaic expression RFP. <br /> peripheral/epibranchial domain itga5 morphants (Fig. 7B). This <br /> process Fgf-dependent indicates moderate Fgf signaling            Time lapse imaging <br /> remains sufficient edge signaling range properly               Embryos mounted glass slides drops 0.8% low-gel <br /> regulate gene expression absence itga5 function. Thus,          agarose surrounded plastic frame. After mounting, embryos <br /> effect signaling patterning separated extent             hydrated coverslips affixed petroleum jelly. <br /> from regulation cell survival, sensitive        Embryos imaged time-lapse using Zeiss Axioimager2- <br /> to slight changes signaling activity.                                      ApoTome. Fluorescent images taken 3 minutes using <br />    The link cell fate specification cell migration         Axiovision 4.6.3 software. <br /> otic/epibranchial field complex. We showed previously otic <br /> and epibranchial fates specified sequentially: The otic placode           Cell tracking vector maps <br /> forms expresses fgf24, induces epibranchial                 To correct inadvertent movement embryo during <br /> development upregulation sox3 expression                   recording, GFP expression rhombomeres 3 5 used to <br /> lateral ectoderm [17]. Based cell-tracking pax2a:GFP                provide fixed reference frame. ImageJ software used <br /> domain, recruitment otic epibranchial cells occurs                  manually track individual cells. Tracks projected onto <br /> correspondingly different times. Recruitment otic cells occurs             representational maps using Photoshop. Calculations displace- <br /> mostly 11.5&#226;&#8364;&#8220;12 hpf, majority epibranchial              ment, total distance angle vector calculated <br /> cells recruited 12.5&#226;&#8364;&#8220;13.3 hpf slightly                using ImageJ software. <br /> lateral domain. Given 40-minute lag onset of <br /> transcription accumulation GFP, times agree             Generating chimeric embryos <br /> dynamic changes sox3 expression otic vs. epibranchial                      Tg(CMV:RFP) transgenic embryos coinjected 100 ng/ <br /> precursors [15&#226;&#8364;&#8220;17]. Changes processes itga5 morphants             nl CMV:RFP plasmid DNA 5 ng/nl itga5 morpholino just <br /> provide insight morphogenesis fate                         plasmid DNA used donors. At 1000-cell stage donor <br /> specification coordinated. Because recruitment new cells               cells transplanted glass needle pax2a:GFP <br /> fails entirely itga5 morphants, fates specified        transgenic host embryos. Chimeras containing cells otic/ <br /> fixed population pax2a:GFP-positive cells. Inefficient cell                epibranchial domain selected time-lapse imaging, as <br /> migration appears contribute marked deficiency otic                described above. <br /> cells corresponding increase non-otic domains. It is <br /> understandable slow migration impedes peripheral cells from <br />                                                                               Supporting Information <br /> entering otic domain timely manner, don&#226;&#8364;&#8482;t <br /> peripheral cells intercalate otic cells later times? There         Figure S1 Abnormal development posterior placodes <br /> possibly timing mechanism excludes new cells otic             itga5b926/b926 mutants. (A, B) pax2a expression 14 hpf in <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        10                       December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                                                   itga5 Fgf Regulate Placode Assembly <br />  <br />  <br /> the otic/epibranchial domain wild-type embryo (A) itga5                             migration efficiency significantly different itga5 <br /> mutant (B). Otic placodes (o, brackets) indicated. (C, D) neuroD                         morphants relative controls (p = 0.10) (see Table 1). <br /> expression 14 hpf control embryo (C) itga5 morphant                              (TIF) <br /> (D). Precursors trigeminal ganglion (tg) anterior lateral <br />                                                                                              Figure S5 Elevating Fgf does rescue cell <br /> line (al) indicated. (E, F) Immunolocalization Pax2 (green) <br />                                                                                              migration defect itga5 morphants. Maps cell <br /> and Caspase 3 (red) wild-type embryo (E) itga5 mutant (F). <br />                                                                                              trajectories pax2a:GFP/+; hs:fgf8/+ neuroD:GFP/+; hs:fgf8/ <br /> (G, H) Otic vesicles 24hpf wild-type embryo (G) itga5 <br />                                                                                              + embryos injected itga5-MO. To assist cell-tracking, <br /> mutant (H). (I, J) phox2a expression epibranchial ganglia 30 <br />                                                                                              embryos injected one-cell stage cmv:GFP. <br /> hpf wild-type embryo (I) itga5 mutant (J). Facial (f), <br />                                                                                              Embryos heat shocked 11 hpf filmed 11.5&#226;&#8364;&#8220;14.5 <br /> glossopharyngeal (g), vagal ganglia (v1+v2) indicated. A&#226;&#8364;&#8220;E <br />                                                                                              hpf. Cells tracked RFP expression pax2a:GFP <br /> are dorsal views anterior top; G&#226;&#8364;&#8220;J lateral views with <br />                                                                                              background, GFP expression neuroD:GFP background. <br /> anterior left. Scale bar, 50 mm. <br />                                                                                              Initial boundaries transgenic GFP expression shown in <br /> (TIF) <br />                                                                                              purple black, final boundaries shown green and <br /> Figure S2 Recruitment lateral cells pax2-                                        red. <br /> a:GFP domain. Representative frames movie                                        (TIF) <br /> pax2a:GFP transgenic embryo injected cmv:RFP plasmid DNA.                               Movie S1 Migration cells pax2a:GFP control <br /> RFP-positive cells originating position lateral otic/                          embryo. In addition GFP expression, embryo injected <br /> epibranchial domain tracked entered pax2a:GFP                               one-cell stage cmv:RFP plasmid DNA, providing mosaic <br /> domain activated expression GFP. White arrows indicate                                expression facilitate cell-discrimination otic placode. <br /> the positions cells respect domains GFP                           The movie runs 11.5 hpf 14.5 hpf. <br /> both GFP RFP. A map migration patterns                                 (MOV) <br /> tracked cells indicated, purple green borders <br /> marking initial final positions, respectively,                                Movie S2 Migration cells pax2a:GFP embryo <br /> pax2a:GFP domain.                                                                            injected itga5-MO. In addition GFP expression, the <br /> (TIF)                                                                                        embryo injected one-cell stage cmv:RFP plasmid <br />                                                                                              DNA, providing mosaic expression facilitate cell-discrimination <br /> Figure S3 Knockdown itga5 does alter prolifera- <br />                                                                                              otic placode. The movie runs 11.5 hpf 14.5 hpf. <br /> tion. Embryos incubated BrdU beginning 11.5 hpf,                                  (MOV) <br /> fixed 13.5 hpf immunostained BrdU (green) Pax2a <br /> (red).                                                                                       Movie S3 Migration cells neuroD;EGFP control <br /> (TIF)                                                                                        embryo. The movie runs 11.5 14.0 hpf. <br />                                                                                              (MOV) <br /> Figure S4 Knockdown itga5 does perturb cell <br /> migration lateral regions. Maps migration                                         Movie S4 Migration cells neuroD;EGFP embryo <br /> patterns RFP-positive cells observed movies pax2a:GFP/+                           injected itga5-MO. The movie runs 11.5 14.0 hpf. <br /> control embryo pax2a:GFP/+ embryo injected itga5-                                 (MOV) <br /> MO. Embryos injected cmv:RFP plasmid one-cell <br /> stage imaged 11.5 hpf 14.5 hpf. In itga5 morphant,                           Author Contributions <br /> lateral cells migrated normally filming period. Two                          Conceived designed experiments: NB BBR. Performed the <br /> cells showed deviations nearing otic/epibranchial                             experiments: NB. Analyzed data: NB BBR. Contributed reagents/ <br /> domain itga5 expression normally upregulates. Nevertheless,                            materials/analysis tools: BBR. Wrote paper: NB BBR. <br />  <br /> References <br />  1. McCabe KL, Bronner-Fraser M (2009) Molecular tissue interactions                     11. Toro S, Varga ZM (2007) Equivalent progenitor cells zebrafish anterior <br />     governing induction cranial ectodermal placodes. Dev Biol 332(2): 189&#226;&#8364;&#8220;195.                preplacodal field rise adenohypophysis, lens, olfactory placodes. Sem <br />  2. Schlosser G (2010) Making senses development vertebrate cranial placodes.                 Cell Dev Biol 18(4): 534&#226;&#8364;&#8220;542. <br />     Int Rev Cell Mol Biol 283: 129&#226;&#8364;&#8220;234.                                                      12. Whitlock KE, Westerfield M (2000) The olfactory placodes zebrafish form <br />  3. Streit A (2007) The preplacodal region: ectodermal domain                            convergence cellular fields edge neural plate. Development <br />     multipotential progenitors contribute sense organs cranial sensory               127(17): 3645&#226;&#8364;&#8220;3653. <br />     ganglia. Int J Dev Biol 51(6&#226;&#8364;&#8220;7): 447&#226;&#8364;&#8220;461.                                                13. Miyasaka N, Miyasaka N, Knaut H, Yoshihara Y (2007) Cxcl12/Cxcr4 <br />  4. Ahrens K, Schlosser G (2005) Tissues signals involved induction                chemokine signaling required placode assembly sensory axon <br />     placodal Six1 expression Xenopus laevis. Dev Biol 288(1): 40&#226;&#8364;&#8220;59.                          pathfinding zebrafish olfactory system. Development 134(13): 2459&#226;&#8364;&#8220;2468. <br />  5. Brugmann SA, Pandur PD, Kenyon KL, Pignoni F, Moody SA (2004) Six1                       14. Knaut H, Blader P, Strahle U, Schier AF (2005) Assembly trigeminal sensory <br />     promotes placodal fate lateral neurogenic ectoderm functioning               ganglia chemokine signaling. Neuron 47(5): 653&#226;&#8364;&#8220;666. <br />     transcriptional activator repressor. Development 131(23):                  15. Nikaido M, Doi K, Shimizu T, Hibi M, Kikuchi Y, et al. (2007) Initial <br />     5871&#226;&#8364;&#8220;5881.                                                                                   specification epibranchial placode zebrafish embryos depends the <br />  6. Glavic A, Maris HS, Gloria FC, Bastidas F, Allende ML, et al. (2004) Role                 fibroblast growth factor signal. Dev Dyn 236(2): 564&#226;&#8364;&#8220;571. <br />     BMP signaling homeoprotein Iroquois specification cranial          16. Sun SK, Dee CT, Tripathi VB, Rengifo A, Hirst CS, et al. (2007) Epibranchial <br />     placodal field. Dev Biol 272(1): 89&#226;&#8364;&#8220;103.                                                     otic placodes induced common Fgf signal, subsequent <br />  7. Kwon HJ, Bhat N, Sweet EM, Cornell RA, Riley BB (2010) Identification                     development independent. Dev Biol 303(2): 675&#226;&#8364;&#8220;686. <br />     early requirements preplacodal ectoderm sensory organ development.               17. Padanad MS, Riley BB (2011) Pax2/8 proteins coordinate sequential induction <br />     PLoS Gen 6(9): e1001133.                                                                     otic epibranchial placodes differential regulation foxi1, sox3 <br />  8. Litsiou A, Hanson S, Streit A (2005) A balance FGF, BMP WNT                           fgf24. Dev Biol 351(1): 90&#226;&#8364;&#8220;98. <br />     signalling positions future placode territory head. Development               18. Chodniewicz D, Klemke RL (2004) Guiding cell migration directed <br />     132(18): 4051&#226;&#8364;&#8220;4062.                                                                          extension stabilization pseudopodia. Exp Cell Res 301(1): 31&#226;&#8364;&#8220;37. <br />  9. Bhattacharyya S, Bailey AP, Bronner-Fraser M, Streit A (2004) Segregation             19. Gilcrease MZ (2007) Integrin signaling epithelial cells. Cancer Lett 247(1): 1&#226;&#8364;&#8220;25. <br />     lens olfactory precursors common territory: cell sorting                  20. Hehlgans S, Haase M, Cordes N (2007) Signalling integrins: Implications for <br />     reciprocity Dlx5 Pax6 expression. Dev Biol 271(2): 403&#226;&#8364;&#8220;414.                           cell survival anticancer strategies. Biochimica Et Biophysica Acta-Reviews <br /> 10. Dutta S, Dietrich JE, Aspock G, Burdine RD, Schier A, et al. (2005) pitx3 defines            Cancer 1775(1): 163&#226;&#8364;&#8220;180. <br />     equivalence domain lens anterior pituitary placode. Development               21. Petrie RJ, Doyle AD, Yamada KM (2009) Random versus directionally <br />     132(7): 1579&#226;&#8364;&#8220;1590.                                                                           persistent cell migration. Nat Rev Mol Cell Biol 10(8): 538&#226;&#8364;&#8220;549. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      11                            December 2011 | Volume 6 | Issue 12 | e27778 <br />                                                                                                                                      itga5 Fgf Regulate Placode Assembly <br />  <br />  <br /> 22. Crump JG, Swartz ME, Kimmel CB (2004) An integrin-dependent role pouch                   39. Mould AP, Koper EJ, Byron A, Zahn G, Humphries MJ (2009) Mapping the <br />     endoderm hyoid cartilage development. Plos Biology 2(9): 1432&#226;&#8364;&#8220;1445.                          ligand-binding pocket integrin alpha5beta1 using gain-of-function approach. <br /> 23. Ju&#194;&#168;lich D, Geisler R, Holley SA (2005) Integrinalpha5 delta/notch signaling                 Biochem J 424(2): 179&#226;&#8364;&#8220;89. <br />     complementary spatiotemporal requirements zebrafish somitogen-                  40. Trinh LA, Stainier DYR (2004) Fibronectin regulates epithelial organization <br />     esis. Dev Cell 8(4): 575&#226;&#8364;&#8220;586.                                                                   myocardial migration zebrafish. Dev Cell 6(3): 371&#226;&#8364;&#8220;382. <br /> 24. Koshida S, Kishimoto Y, Ustumi H, Shimizu T, Furutani-Seiki M, et al. (2005)                41. Ward SG, Marelli-Berg FG (2009) Mechanisms chemokine antigen- <br />     Integrinalpha5-dependent fibronectin accumulation maintenance somite                     dependent T-lymphocyte navigation. Biochem J 418: 13&#226;&#8364;&#8220;27. <br />     boundaries zebrafish embryos. Dev Cell 8(4): 587&#226;&#8364;&#8220;598.                                    42. Long QM, Quint E, Lin S, Ekker M (2000) The zebrafish scyba gene encodes a <br /> 25. Lee SA, Shen EL, Fiser A, Sali A, Guo S (2003) The zebrafish forkhead                           novel CXC-type chemokine distinctive expression patterns vestibulo- <br />     transcription factor Foxi1 specifies epibranchial placode-derived sensory                       acoustic embryogenesis. Mech Dev 97(1&#226;&#8364;&#8220;2): 183&#226;&#8364;&#8220;186. <br />     neurons. Development 130(12): 2669&#226;&#8364;&#8220;2679.                                                    43. Riley BB, Sweet EM, Heck R, Evans A, McFarland KN, et al. (2010) <br /> 26. Andermann P, Ungos J, Raible D (2002) Neurogenin1 defines zebrafish cranial                     Characterization harpy/Rca1/emi1 Mutants: Patterning Absence Cell <br />     sensory ganglia precursors. Dev Biol 251(1): 45&#226;&#8364;&#8220;58.                                             Division. Dev Dyn 239(3): 828&#226;&#8364;&#8220;843. <br /> 27. Picker A, Scholpp S, Bohli H, Takeda H, Brand M (2002) A novel positive                     44. Aplin AE, Stewart SA, Assoian RK, Juliano RL (2001) Integrin-mediated <br />     transcriptional feedback loop midbrain-hindbrain boundary development                     adhesion regulates ERK nuclear translocation phosphorylation Elk-1. <br />     revealed analysis zebrafish pax2.1 promoter transgenic lines.                 J Cell Biol 153(2): 273&#226;&#8364;&#8220;82. <br />     Development 129(13): 3227&#226;&#8364;&#8220;3239.                                                             45. Hirsch E, Barberis L, Brancaccio M, Azzolino O, Xu DZ, et al. (2002) Defective <br /> 28. Obholzer N, Wolfson S, Trapani JG, Mo W, Nechiporuk A, et al. (2008)                            Rac-mediated proliferation survival targeted <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span> beta(1) <br />     Vesicular glutamate transporter 3 required synaptic transmission                      integrin cytodomain. J Cell Biol 157(3): 481&#226;&#8364;&#8220;492. <br />     zebrafish hair cells. J Neourosc 28(9): 2110&#226;&#8364;&#8220;2118.                                          46. James BP, Bunch TA, Krishnamoorthy S, Perkins LA, Brower DL (2007) <br /> 29. Millimaki BB, Sweet EM, Riley BB (2010) Sox2 required maintenance                    Nuclear localization ERK MAP kinase mediated Drosophila <br />     regeneration, initial development, hair cells zebrafish inner ear.            alphaPS2betaPS integrin importin-7. Mol Biol Cell 18(10): 4190&#226;&#8364;&#8220;4199. <br />                                                                                                 47. Mori S, Wu CY, Yamaji S, Saegusa J, Shi B, et al. Direct binding integrin <br />     Dev Biol 338(2): 262&#226;&#8364;&#8220;267. <br />                                                                                                     alphavbeta3 FGF1 plays role FGF1 signaling. J Biol Chem 283(26): <br /> 30. Fu&#194;&#168;rthauer M, Reifers F, Brand M, Thisse B, Thisse C (2001) sprouty4 acts vivo <br />                          <br /> </body></html>