<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Handling Uncertainty Dynamic Models: The Pentose <br /> Phosphate Pathway Trypanosoma brucei <br /> Eduard J. Kerkhoven1,2., Fiona Achcar3,4., Vincent P. Alibu1&#194;&#164;a, Richard J. Burchmore1, Ian H. Gilbert5, <br /> Maciej Trybi&#197;&#8218;o6, Nicole N. Driessen7&#194;&#164;b, David Gilbert6, Rainer Breitling8*, Barbara M. Bakker7,9*, <br /> Michael P. Barrett1* <br /> 1 Wellcome Trust Centre Molecular Parasitology, Institute Infection, Immunity Inflammation Glasgow Polyomics, College Medical, Veterinary Life <br /> Sciences, University Glasgow, Glasgow, United Kingdom, 2 Systems Synthetic Biology Group, Department Chemical Biological Engineering, Chalmers <br /> University Technology, Gothenburg, Sweden, 3 Institute Molecular, Cell Systems Biology, College Medical, Veterinary Life Sciences, University Glasgow, <br /> Glasgow, United Kingdom, 4 Groningen Bioinformatics Centre, Groningen Biomolecular Sciences Biotechnology Institute, University Groningen, Groningen, the <br /> Netherlands, 5 Biological Chemistry Drug Discovery, College Life Sciences, University Dundee, Dundee, United Kingdom, 6 School Information Systems, <br /> Computing Mathematics, Brunel University, Uxbridge, United Kingdom, 7 Department Molecular Cell Physiology, Faculty Earth Life Sciences, VU University <br /> Amsterdam, Amsterdam, Netherlands, 8 Faculty Life Sciences, Manchester Institute Biotechnology, University Manchester, Manchester, United Kingdom, <br /> 9 Department Pediatrics Systems Biology Centre Energy Metabolism Ageing, University Groningen, University Medical Center Groningen, Groningen, the <br /> Netherlands <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Dynamic models metabolism useful identifying potential drug targets, especially unicellular organisms. A <br />      model glycolysis causative agent human African trypanosomiasis, Trypanosoma brucei, shown the <br />      utility approach. Here add pentose phosphate pathway (PPP) T. brucei glycolytic model. The PPP is <br />      localized cytosol glycosome adding glycolytic model adjustments leads <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />      draining essential bound-phosphate moiety glycosome. This phosphate &#226;&#8364;&#732;&#226;&#8364;&#732;leak&#226;&#8364;&#8482;&#226;&#8364;&#8482; resolved the <br />      model reasonable representation parasite physiology. Two main types theoretical solution problem <br />      identified: (i) including additional enzymatic reactions glycosome, (ii) adding mechanism transfer <br />      bound phosphates cytosol glycosome. One example type solution presence a <br />      glycosomal ribokinase regenerate ATP ribose 5-phosphate ADP. Experimental characterization ribokinase in <br />      T. brucei showed low enzyme levels sufficient parasite <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>survival</span>, indicating mechanisms are <br />      required controlling phosphate leak. Examples second type involve presence ATP:ADP <br />      exchanger recently described permeability pores glycosomal membrane, current absence of <br />      identified genes encoding molecules impedes experimental testing genetic manipulation. Confronted this <br />      uncertainty, present modeling strategy identifies robust predictions context incomplete system <br />      characterization. We illustrate strategy exploring mechanism underlying essential function PPP <br />      enzymes, validate confirming model predictions experimentally. <br />  <br />   Citation: Kerkhoven EJ, Achcar F, Alibu VP, Burchmore RJ, Gilbert IH, et al. (2013) Handling Uncertainty Dynamic Models: The Pentose Phosphate Pathway in <br />   Trypanosoma brucei. PLoS Comput Biol 9(12): e1003371. doi:10.1371/journal.pcbi.1003371 <br />   Editor: Martin Meier-Schellersheim, National Institutes Health, United States America <br />   Received July 3, 2013; Accepted October 16, 2013; Published December 5, 2013 <br />   Copyright: &#195;&#376; 2013 Kerkhoven et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported SULSA PhD studentship EJK (http://www.sulsa.ac.uk/), NWO Vidi grant RB (http://www.nwo.nl/en/funding/our- <br />   funding-instruments/nwo/innovational-research-incentives-scheme/vidi/index.html) SysMO2 grant (SilicoTryp, http://www.sysmo.net/index. <br />   php?index = 164) RB, MPB BMB. BMB recipient Rosalind Franklin Fellowship (http://www.rug.nl/umcg/research/rosalind-franklin-fellowships/). <br />   The Wellcome Trust Centre Molecular Parasitology supported core funding Wellcome Trust (085349, http://www.wellcome.ac.uk/Funding/ <br />   Biomedical-science/Funded-projects/Major-initiatives/Wellcome-Trust-Centres/index.htm). The funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: rainer.breitling@manchester.ac.uk (RB); b.m.bakker@umcg.nl (BMB); michael.barrett@glasgow.ac.uk (MPB) <br />   . These authors contributed equally work. <br />   &#194;&#164;a Current address: Department Veterinary Parasitology Microbiology, Faculty Science, Makerere University, Kampala, Uganda. <br />   &#194;&#164;b Current address: Department Parasitology, Leiden University Medical Center, Leiden, Netherlands. <br />  <br />  <br />  <br />  <br /> Introduction                                                                            computational models, focusing energy production the <br />                                                                                         glycolytic pathway [4&#226;&#8364;&#8220;8]. This pathway considered be <br />    Human African trypanosomiasis, sleeping sickness, lethal                     particularly promising target new drugs, 50% reduction <br /> disease caused protozoan parasite Trypanosoma brucei [1,2].                      ATP production T. brucei inhibitors glycolysis was <br /> Current drugs trypanosomiasis difficult administer,                      demonstrated sufficient kill parasites [9]. <br /> unacceptably toxic, relatively expensive considering                          In paper, extended model glycolysis a <br /> impoverished state patients [3]. The search new                             description pentose phosphate pathway (PPP), another <br /> promising drug targets supported series                                essential [10,11] maintenance pathway generates reducing <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                   1                        December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                        Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />  <br />   Author Summary                                                             based predictions [20]. Figure 1 gives overview models <br />                                                                              considered paper, stoichiometry kinetic <br />   Mathematical models valuable tools                           parameters described Table 1 S1, respectively. <br />   investigating complex behaviors metabolism. Due                         To provide glucose 6-phosphate (Glc-6-P) cytosolic <br />   incomplete knowledge biological systems,                       glucose 6-phosphate dehydrogenase (G6PDH), trace of <br />   models contain inevitable uncertainty. This uncertainty                 cytosolic hexokinase added models B&#226;&#8364;&#8220;D. An alternative <br />   present measured estimated parameter values,                     solution, transport Glc-6-P glycosomal <br />   structure metabolic network. In                membrane allowed, gave similar results. Fractionation studies <br />   paper increase coverage particularly                      suggest exclusively glycosomal localization hexokinase <br />   studied model glucose metabolism Trypanosoma                         activity [21] insufficiently sensitive discriminate trace <br />   brucei, tropical parasite causes African sleeping                   cytosolic hexokinase. Moreover, enzymes targeted to <br />   sickness, extending additional pathway                    glycosome fully folded cytosol, leaves room <br />   compartments. During modeling process we <br />                                                                              small, residual activity compartment. We therefore <br />   highlighted uncertainties parameter values network <br />                                                                              decided explore model behavior presence cytosolic <br />   structure used formulate new hypotheses <br />   subsequently tested experimentally. The                         hexokinase following analyses, acknowledge this <br />   models improved experimentally derived                       choice somewhat arbitrary. A reaction utilizing cytosolic Glc-6-P <br />   data, uncertainty remained concerning exact                        added models B&#226;&#8364;&#8220;D, absence reaction resulted in <br />   topology system. These models allowed                         accumulation Glc-6-P cytosol (25% models a <br />   investigate effects loss enzyme, 6-                      cytosolic Glc-6-P concentration 20 mM, contrast a <br />   phosphogluconate dehydrogenase. By taking uncertainty                      measured steady-state value 2.6 mM). This reaction primarily <br />   account, models demonstrated loss                     represents use Glc-6-P cytosolic pathways, as <br />   enzyme lethal parasite mechanism                       glycosylation proteins [22]. <br />   different organisms. Our methodology                        The model glycolysis (Figure 1B, model A) conserved <br />   shows formally introducing uncertainty model                      sum phosphorylated metabolites glycosome, consisting of <br />   building provides robust model behavior indepen-                   ATP, ADP glycolytic intermediates (Table 2). Extension the <br />   dent network structure parameter values.                         glycolysis model PPP (Figure 1B, model B) introduces a <br />                                                                              drain moiety phosphorylated glycosomal compounds. <br /> equivalents form NADPH used cells&#226;&#8364;&#8482;                 While extension PPP adds metabolites the <br /> protection oxidative stress provides metabolic            glycosome derived Glc-6-P, phosphorylated <br /> link important drug target trypanosomes [12].                  PPP intermediates prevented re-entering glycolysis two <br /> Bloodstream form T. brucei relies PPP primary source           key enzymes non-oxidative PPP&#226;&#8364;&#8221;ribulose-5-phosphate <br /> of NADPH, contrast procyclic T. brucei                  epimerase [23] transketolase [17,23]&#226;&#8364;&#8221;are expressed in <br /> produce NADPH malic enzyme [13].                                         bloodstream form parasite. This renders ribose 5- <br />    Kinetic models metabolism, constructed T.             phosphate (Rib-5-P) end product PPP according to <br /> brucei glycolysis, include kinetic parameters uncertain       classical scheme, leads lethal drainage phosphates <br /> variety reasons, measurement errors lack data. We           previously conserved moiety (Figure 2). This phosphate <br /> have previously shown importance taking uncertainty                &#226;&#8364;&#732;&#226;&#8364;&#732;leak&#226;&#8364;&#8482;&#226;&#8364;&#8482; entails bound phosphate, inorganic phos- <br /> these parameter values account [8]. This approach                   phate, independent conserved moiety. This leak of <br /> demonstrate model behaviors robust uncertainty,             phosphates conserved moiety bound phosphates only <br /> helping indicate fragilities model highlighting areas       observed glycosome, bound phosphates not <br /> need detailed experimental characterization. When                    conserved cytosol, net production ATP <br /> introducing PPP model glycolysis face                   pyruvate kinase. Model B obviously non- <br /> additional challenge uncertainty model topology itself;         physiological model investigated further. The <br /> do know sure reactions active context            phosphate leak effect stoichiometry model and <br /> this pathway. Again explicit consideration uncertainty         enzyme kinetics: formal proof exists <br /> the model construction reveals robust behavior indicates gaps            elementary mode (Table S2) model B flux through <br /> in knowledge T. brucei metabolism. We illustrate                  PPP. <br /> consequences uncertainty, robust predictions that <br /> are possible, exploring model behavior various                Exploring solutions phosphate leak <br /> physiological conditions (different levels oxidative stress            There uncertainty glycosomal phosphate leak is <br /> external glucose concentration) characterizing effects            prevented parasite. Two types solution considered: <br /> the inhibition 6-phosphogluconate dehydrogenase (6PGDH),               (i) maintaining conserved moiety phosphorylated metabo- <br /> essential enzyme pentose phosphate pathway, proposed               lites including additional reactions glycosome, (ii) <br /> another potential drug target [14].                                          transferring phosphorylated metabolites glycosomal <br />                                                                              membrane. No priori knowledge exists balancing bound <br /> Results                                                                      phosphates glycosome convincingly support <br />                                                                              solution other. <br /> Simple modeling PPP creates glycosomal <br /> phosphate &#226;&#8364;&#732;&#226;&#8364;&#732;leak&#226;&#8364;&#8482;&#226;&#8364;&#8482;                                                           First type solution: Maintaining conserved moiety <br />   The existing mathematical model glycolysis T. brucei [8]                For type solution, stoichiometric analysis all <br /> was extended include reactions pentose phosphate             known predicted glycosomal enzymes [18&#226;&#8364;&#8220;20] performed <br /> pathway (PPP). In T. brucei, PPP localized cytosol        indicate additional reactions glycosome recover <br /> and glycosomes, based subcellular localization studies [15&#226;&#8364;&#8220;17],           bound-phosphate lost Rib-5-P (Tables S3, S4, Figure S1). <br /> proteomic analysis [18,19] subcellular targeting-sequence-               Only ribokinase, C-terminal PTS-1 sequence that <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                        2                    December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                        Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 1. Schematic overview models used simulation. (A) Detailed scheme modeled metabolic pathways. The numbered <br /> arrows correspond reactions Table 1. Extensions original model glycolysis indicated colored shapes. Boundary metabolites are <br /> in bold, glycosomal Rib-5-P boundary metabolite model C D. (B) Schematic overview different models, consisting unique <br /> combination colored modules described (A) Table 1. Model C D alternatively utilize fructose (model Cfru Dfru), this <br /> branch switched unless specifically mentioned. <br /> doi:10.1371/journal.pcbi.1003371.g001 <br />  <br /> localizes glycosome, able support steady-state flux        provide alternative solutions phosphate leak, however, the <br /> through PPP requirement additional reactions              ribokinase solution representative class solutions where <br /> that known predicted localized glycosome.            bound phosphate balanced glycosome the <br /> The addition hypothetical reactions glycosome                  moiety bound phosphates conserved. This solution requires <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                        3                     December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                                     Modelling Metabolism Trypanosoma brucei <br />  <br />  Table 1. Model stoichiometry modules subsequent reactions. <br />  <br />  <br />  Module                              No.       Reaction                              Reaction stoichiometry <br />  <br />  Glycolysis                          1         GlcTplasma   membrane                      Glcout&#194;&#171;Glcc <br />                                      2         GlcTglycosomal   membrane                  Glcc&#194;&#171;Glcg <br />                                      3         HXKg                                       Glcg+ATPg&#194;&#171;Glc-6-Pg+ADPg <br />                                      4         PGI                                        Glc-6-Pg&#194;&#171;Fru-6-Pg <br />                                      5         PFK                                        Fru-6-Pg+ATPg&#194;&#171;Fru-1,6-BPg+ADPg <br />                                      6         ALD                                        Fru-1,6-BPg&#194;&#171;DHAPg+GA-3-Pg <br />                                      7         TPI                                        DHAPg&#194;&#171;GA-3-Pg <br />                                      8         GAPDH                                      GA-3-Pg+NAD+g+Pig&#194;&#171;1,3-BPGAg+NADHg <br />                                      9         PGK                                        1,3-BPGAg+ADPg&#194;&#171;3-PGAg+ATPg <br />                                      10        PGAT                                       3-PGAg&#194;&#171;3-PGAc <br />                                      11        PGAM                                       3-PGAc&#194;&#171;2-PGAc <br />                                      12        ENO                                        2-PGAc&#194;&#171;PEPc <br />                                      13        PYK                                        PEPc+ATPc&#194;&#171;Pyrc+ADPc <br />                                      14        PyrT                                       PyrcRPyrout <br />                                      15        GDH                                        DHAPg+NADHg&#194;&#171;NAD+g+Gly-3-Pg <br />                                      16        DHAP:Gly-3-P antiporter                    DHAPc+Gly-3-Pg&#194;&#171;DHAPg+Gly-3-Pc <br />                                      17        GPO                                        Gly-3-Pc+0.5 O2RDHAPc <br />                                      18        GK                                         Gly-3-Pg+ADPg&#194;&#171;Glyout+ATPg <br />                                      19        ATP utilization                            ATPcRADPc <br />                                      20        AKg                                        2 ADPg&#194;&#171;ATPg+AMPg <br />                                      21        AKc                                        2 ADPc&#194;&#171;ATPc+AMPc <br />  <br />  Cytosolic extension                 22        G6P utilization                            Glc-6-Pc&#194;&#171;Glcc+Pic <br />                                      23        HXKc                                       Glcc+ATPc&#194;&#171;Glc-6-Pc+ADPc <br />                                      24        G6PDH                                      Glc-6-Pc+NADP+c&#194;&#171;6-PGLc+NADPHc <br />                                      25        PGL                                        6-PGLc&#194;&#171;6-PGc <br />                                      26        6PGDH                                      6-PGc+NADP+c&#194;&#171;Rul-5-Pc+NADPHc+CO2 <br />                                      27        PPI                                        Rul-5-Pc&#194;&#171;Rib-5-Pc <br />                                      28        NADPH utilization                          NADPHc&#194;&#171;NADP+c <br />                                      29        TR                                         TS2,c+NADPHc&#194;&#171;T(SH)2,c+NADP+c <br />                                      30        TOX                                        T(SH)2,cRTS2,c <br />  <br />  Glycosomal extension                31        G6PDH                                      Glc-6-Pg+NADP+g&#194;&#171;6-PGLg+NADPHg <br />                                      32        PGL                                        6-PGLg&#194;&#171;6-PGg <br />                                      33        6PGDH                                      6-PGg+NADP+g&#194;&#171;Rul-5-Pg+NADPHg+CO2 <br />                                      34        PPI                                        Rul-5-Pg&#194;&#171;Rib-5-Pg <br />                                      35        NADPH utilization                          NADPHg&#194;&#171;NADP+g <br />  <br />  Model C extension                   36        RK                                         Rib-5-Pg+ADPg&#194;&#171;Ribg+ATPg <br />  <br />  Model D extension                   37        ATP:ADP antiporter                         ATPc+ADPg&#194;&#171;ATPg+ADPc <br />  <br />  Fructose growth                     38        FruTplasma   membrane                      Fruout&#194;&#171;Fruc <br />                                      39        FruTglycosomal   membrane                  Fruc&#194;&#171;Frug <br />                                      40        HXK(fru)g                                  Frug+ATPg&#194;&#171;Fru-6-Pg+ADPg <br />  <br />  Bold characters indicate external metabolites kept fixed. Other metabolites internal variables system. <br />  doi:10.1371/journal.pcbi.1003371.t001 <br />  <br />  <br /> ribokinase operate direction ribose ATP                              improved low ATP/ADP ratio. A similar solution, however, <br /> production, favored thermodynamically standard                        exploited regard glycerol kinase the <br /> conditions. With equilibrium constant 0.0036 (in                            onset anaerobiosis. In response loss mitochondrial <br /> direction ribose formation, [24]) ribokinase require                      alternative oxidase reaction conditions, trypanosomal <br /> almost 300-fold accumulation Rib-5-P ribose                            glycerol kinase runs direction glycerol ATP <br /> glycosome, indicating ribose glycosome                    production, glycerol 3-phosphate accumulates low <br /> mM range Rib-5-P mM range. This ratio                       ATP/ADP ratio maintained glycosome [25]. <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                 4                        December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                                   Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />  <br />  Table 2. Conserved moieties models Figure 1. <br />  <br />  <br />  Model             Moieties                          Metabolites <br />  <br />  A                 1                                 ATPg+ADPg+AMPg <br />                    2                                 ATPc+ADPc+AMPc <br />                    3                                 NADHg+NAD+g <br />                    4                                 Gly3-Pc+DHAPc <br />                    5                                 Glc-6-Pg+Fru-6-Pg+2 Fru-1,6-BPg+DHAPg+GA-3-Pg+1,3-BPGAg+Gly-3-Pg+2 ATPg+ADPg <br />  B                 1,2,3,4 <br />                    6                                 NADPHg+NADP+g <br />                    7                                 NADPHc+NADP+c <br />                    8                                 TS2,c+T(SH)2,c <br />  C                 1,2,3,4,6,7,8 <br />                    9                                 Glc-6-Pg+Fru-6-Pg+2 Fru-1,6-BPg+DHAPg+GA-3-Pg+1,3-BPGAg+Gly-3-Pg+2 ATPg+ADPg+6-PGg+6-PGLg+Rul-5- <br />                                                      Pg+Rib-5-Pg <br />  D                 1,2,3,4,6,7,8 <br />  <br />  Where moieties present multiple models, number indicated. Moieties 1&#226;&#8364;&#8220;4 conserved model versions. Moieties 6&#226;&#8364;&#8220;8 result the <br />  extension pentose phosphate pathway. Moiety 9 modified version moiety 5, including phosphorylated metabolites pentose-phosphate <br />  pathway. The appears glycosomal PPP completed ribokinase. When glycosomal PPP linked ATP transport instead, moiety 5 disappears <br />  altogether. <br />  doi:10.1371/journal.pcbi.1003371.t002 <br />  <br />  <br />    Model B extended glycosomal ribokinase,                      Table S1 determined kinetic parameters). Dynamic simulations <br /> generating model C (Figure 1), represents solution                     extended model subsequently indicated feasibility of <br /> the glycosomal conserved moiety bound phosphates restored                      ribokinase solution, allows metabolic fluxes all <br /> (Table 2, moiety 9). In order provide kinetic parameters                      branches (Figure S2 Table S2). <br /> model C, enzyme kinetics heterologously expressed T. brucei                      When ribokinase responsible maintaining the <br /> ribokinase characterized used update model (see                      phosphate balance glycosomes, presence expected be <br />                                                                                      essential parasite survival. To investigate hypothesis, a <br />                                                                                      genetic mutant (RKRNAi) generated knockdown levels of <br />                                                                                      ribokinase transcript <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-334' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-353' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-373' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-385' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>. However, growth <br />                                                                                      <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> observed (Figure S3). Attempts generate gene <br />                                                                                      <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>knockout</span> ribokinase, surprisingly, failed spite numerous <br />                                                                                      rounds transfection. A conditional knockout generated with <br />                                                                                      inducible expression exogenous copy T. brucei <br />                                                                                      ribokinase gene. While non-induced growth resulted depletion <br />                                                                                      ribokinase, growth phenotype observed, even <br />                                                                                      ectopic copy switched off. It concluded that <br />                                                                                      ribokinase activity essential residual levels RKRNAi <br />                                                                                      leaky expression conditional knockout sufficient to <br />                                                                                      fulfill essential role. <br />                                                                                         Additionally, requirement low ribose/Rib-5-P ratio to <br />                                                                                      catalyze formation ribose ribokinase predicts ribose <br />                                                                                      toxic parasites (Figure S4), similar way to <br />                                                                                      glycerol prevent glycerol kinase working the <br />                                                                                      reverse direction. However, incubation parasites ribose up <br />                                                                                      30 mM 72 hours non-lethal. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> results <br />                                                                                      genetic mutants, concluded ribokinase is <br /> Figure 2. Phosphate leak model PPP. Time course                              unlikely sole glycosomal enzyme responsible maintaining <br /> simulation model B, reactions glycosomal PPP              phosphate balance. Other kinases, arginine kinase [26] <br /> switched t = 0 increasing Vmax value zero                 phosphoenolpyruvate carboxykinase (PEPCK), also <br /> value given Table S1. Glce 5 mM kTOX = 2 ml?min21?mg <br /> protein21. Solid lines indicate medians, shaded areas interquartile <br />                                                                                      present glycosome bound phosphate homeostasis <br /> ranges. Fluxes (J) plotted left y-axis indicative              involve large redundant network contributing <br /> glucose uptake (GlcTplasma membrane), glycerol (GK) pyruvate                     metabolic pathways. While uncertainty remains enzymes <br /> production (PyrT) branches pentose phosphate                          involved preventing phosphate leak restoring <br /> pathways (G6PDHc/g). The sum bound phosphates glycosome                    conserved moiety phosphorylated metabolites, model C <br /> (SPg), exists model glycolysis (Table 2, moiety 5), plotted <br />                                                                                      represents generic representation solution. Continued <br /> on right y-axis. Within 25 minutes, bound phosphates the <br /> glycosome depleted metabolic fluxes subsequently drop                 analysis model can, therefore, provide insight the <br /> zero.                                                                                effects solution absence specific <br /> doi:10.1371/journal.pcbi.1003371.g002                                                individual enzyme. <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                5                       December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                          Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />  <br /> Second type solution: Transferring phosphates                          bloodstream forms [32], procyclic forms, grow <br /> the glycosomal membrane                                                        glucose proline, rescued removal of <br />    An alternative solution phosphate leak requires transfer-            sugars medium [32], ablation hexokinase <br /> ring bound-phosphates glycosomal membrane. This                   [33]. Simulations model D indicate presence an <br /> be achieved presence single ATP:ADP antiporter                     ATP:ADP antiporter mimics absence glycosome, a <br /> operative glycosomal membrane, analogous yeast               high risk sugar-phosphate accumulation (Figure 3). While the <br /> peroxisomal adenine nucleotide transporter [27]. Model D                   presence PPP partly rescue model ATP:ADP <br /> generated addition ATP:ADP antiporter model B                  antiporter accumulation sugar-phosphates (Figure 3B&#226;&#8364;&#8220;C, <br /> (Figure 1), stoichiometric analysis model D showed                  compare model A+AAT D), high risk accumulation <br /> absence conserved moiety phosphorylated glycosomal                     remains antiporter active (Figure 3D&#226;&#8364;&#8220;E). It has <br /> metabolites (Table 2).                                                         shown experimentally procyclic trypanosomes such <br />    Dynamic analysis model D showed absence                  accumulation occurs enzymes mislocalized, this <br /> conservation constraint phosphorylated metabolites                   detrimental cells [31]. Other types transporters, <br /> glycosome introduced risk high accumulation                      including recently described pores glycosomal membrane <br /> metabolites. The autocatalytic design glycolysis means                 [34], facilitating net transport phosphorylated compounds into <br /> surplus ATP produced later steps glycolysis fuel             glycosome gave similar results, absence known <br /> first reactions positive feedback loop. This &#226;&#8364;&#732;&#226;&#8364;&#732;turbo design&#226;&#8364;&#8482;&#226;&#8364;&#8482; [28]        genes encoding transporters impedes testing this <br /> is controlled organisms additional negative feedback                hypothesis genetic manipulation. <br /> regulation, e.g. inhibition PFK PEP [29]. Trypanosomes lack                  The hypothesized solutions phosphate leak both <br /> such regulation, dynamic modeling supported direct                      theoretically able restore glycolytic PPP flux. Although <br /> experimentation revealed glycosomal localization              current available knowledge insufficient explicitly exclude one <br /> of glycolysis prevents turbo effect trypanosomes [30,31].               types solutions, analysis models can <br /> Glucose predicted lethal glycosome deficient                      valuable insights instance 6PGDH inhibition, <br />  <br />  <br />  <br />  <br /> Figure 3. ATP:ADP antiporter mimics turbo-state. (A) Overview models used figure. Model A D Figure 1, model A&#226;&#8364;&#8220; <br /> glyc model A glycosomal localization, described [31], model A+AAT model A ATP:ADP antiporter. (B&#226;&#8364;&#8220;C) Steady-state <br /> concentrations glycosomal Glc-6-P Fru-1,6-BP depicted various models. (D) Increasing activity ATP:ADP antiporter <br /> (Vmax,ATP:ADP antiporter) model D leads high risk accumulation hexose phosphates. The green line indicates concentration Fru-1,6-BP <br /> in original model glycolysis (17.2 mM, panel C, model A). Glce simulation 25 mM. (E) Time course simulation model D 25 mM Glce <br /> and various values Vmax,ATP:ADP antiporter parameter. Plotted concentration glycosomal phosphates (SP similar Figure 2, moiety 5 <br /> in Table 2). ATP:ADP antiporter activity values 1 nmol?min21?mg protein21 result depletion glycosomal phosphates (cf. Figure 2). <br /> kTOX = 2 ml?min21?mg protein21 models. Solid lines indicate medians, shaded areas error bars interquartile ranges, derived the <br /> uncertainty modeling. <br /> doi:10.1371/journal.pcbi.1003371.g003 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          6                     December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                        Modelling Metabolism Trypanosoma brucei <br />  <br />  <br /> while ignoring uncertainty involved. We demonstrate             inhibited trypanosomes potentially renders parasites more <br /> below.                                                                       susceptible oxidative stress. To analyze hypotheses, <br />                                                                              activity 6PGDH investigated models C D. <br /> Predictive behavior using extended models                                6PGDH inhibition models resulted accumulation <br /> trypanosome glucose metabolism                                               6-PG anticipated (Figure 5A&#226;&#8364;&#8220;B). Both models gave very <br />    Dynamic simulations using models, containing               similar results, major difference glycosomal 6- <br /> the theoretical solutions phosphate-leak problem,                 PG model C restricted 45 mM constraint the <br /> showed oxidative stress regulates flux PPP              conserved moiety phosphorylated metabolites. Highly oxidative <br /> (Figure 4A), line measured increases flux         conditions resulted cytosolic 6-PG accumulation started <br /> PPP related species T. cruzi induced oxidative stress            lower 6PGDH inhibitions, cytosolic PPP flux activated <br /> [35]. Low oxidative stress (kTOX = 2 mLNmin21Nmg protein21)                  provide NADPH required trypanothione reductase. <br /> represents healthy proliferating trypanosome results              Only 6PGDH activity decreased 95% <br /> low PPP flux, supported experimental observation                 glycolysis reduced negligible flux. To observe effects of <br /> almost consumed glucose excreted pyruvate                accumulated 6-PG PGI isolation effects, model A <br /> glycerol [36,37]. A sudden burst oxidative stress simulated           6-PG sensitive PGI simulated different 6-PG levels. A <br /> by completely (99%) oxidizing pools trypanothione              50% reduction glycolytic flux lethal parasite [9], but <br /> and NADPH subsequently allowing model reach steady                level inhibition reached 6-PG levels around <br /> state. The parasite able restore redox balance           500 mM (Figure 5C). In model C, conserved moiety bound <br /> one minute (Figure 4B). This experiment simulates extreme                 phosphates restrict glycosomal 6-PG level 45 mM <br /> level oxidative stress, reality pools          (Figure 5A), model D high accumulation of <br /> 99% oxidized, PPP respond faster.                               glycosomal 6-PG occurs combination far extreme <br />    It previously proposed inhibition              accumulation cytosolic 6-PG (Figure 5B). Neither model <br /> enzyme PPP, 6-phosphogluconate dehydrogenase                          supports PGI inhibition hypothesis explana- <br /> (6PGDH), lethal trypanosomes [38]. In Drosophila                tion lethality 6PGDH inhibition T. brucei. <br /> [39,40], yeast [41] mammalian cancer cells [42], lethality               The second hypothesis initially proposed posits 6PGDH <br /> 6PGDH inhibition based inhibition glycolysis             inhibition lethal reduced flux PPP the <br /> accumulation 6-phosphogluconate (6-PG). Inhibition                     associated increased sensitivity oxidative stress. In models, <br /> 6PGDH results accumulation substrate 6-PG,                simulation 6PGDH inhibition demonstrates cell&#226;&#8364;&#8482;s <br /> potent inhibitor phosphoglucose isomerase (PGI) [43], 6-PG                ability cope oxidative stress strongly affected, flux <br /> subsequently inhibits conversion glucose 6-phosphate                trypanothione reductase inhibited activity of <br /> (Glc-6-P) fructose 6-phosphate. With Glc-6-P instead                 6PGDH reduced 50% (Figure 5A&#226;&#8364;&#8220;B). A third <br /> directed PPP, 6-PG accumulates higher levels,            hypothesis introduced model C reveals accumulation of <br /> ultimately inhibiting ATP producing glycolytic pathway. In               glycosomal 6-PG depletes bound phosphates the <br /> Drosophila yeast, <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> rescued                glycosome, inhibiting glycolytic flux. Sensitivity oxidative <br /> additional loss G6PDH, completely shutting PPP               stress, depletion bound phosphates conserved <br /> as preventing accumulation 6-PG [40,41]. Growth                   moiety, explain loss viability ablation <br /> alternative hexoses fructose, bypassing PGI reaction             6PGDH. While accumulation toxic lactone precursor 6- <br /> and preventing inhibition glycolysis, shown                PG product inhibition lactonase enzyme another <br /> rescue 6PGDH deficient cancer cells [42]. A similar route                 possibility, high equilibrium constant reaction renders it <br /> lethality 6PGDH inhibition PGI inhibition hypothesized            insensitive product [44]. Finally, reduced production the <br /> for T. brucei [10], experimental proof exists corroborate          nucleotide precursor Rib-5-P PPP explain 6PGDH <br /> this hypothesis. Alternatively, reduced PPP flux 6PGDH                inhibition lethality. <br />  <br />  <br />  <br />  <br /> Figure 4. Simulations oxidative stress model C. (A) The steady state flux cytosolic pentose phosphate pathway model C as <br /> a function oxidative stress varying kinetic constant kTOX. (B) Fluxes cytosolic PPP enzymes function time upon <br /> sudden oxidative stress. During time-course, kTOX = 2 ml?min21 ? mg protein21. The removed steady state t = 0, setting <br /> 99% NAD(P)H trypanothione pools oxidized form. Shown relaxation cytosolic PPP fluxes. Solid lines indicate medians, <br /> shaded areas interquartile ranges. Near identical results obtained model D (Figure S5). <br /> doi:10.1371/journal.pcbi.1003371.g004 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                        7                     December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                          Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />  <br />  <br /> Figure 5. Simulations 6PGDH inhibition 6-PG accumulation. (A&#226;&#8364;&#8220;B) The effects inhibition 6PGDH 6-PG concentrations and <br /> metabolic fluxes simulated reducing Vmax,6PGDH model C D high oxidative stress (kTOX = 200 ml?min21?mg protein21). Simulations at <br /> low oxidative stress (kTOX = 2 ml?min21?mg protein21) shown Figure S6. ATP production flux steady-state flux PFK indicated red, <br /> while trypanothione reductase steady-state flux indicated yellow, plotted left y-axis. Steady-state concentration cytosolic (blue) <br /> and glycosomal (green) 6-phosphogluconate plotted right y-axis. Shaded areas indicate interquartile ranges. (C) Steady-state flux through <br /> glycolysis function glycosomal 6-PG concentration model A. A glycosomal 6-PG concentration 500 mM reduces glycolytic <br /> flux 50%. <br /> doi:10.1371/journal.pcbi.1003371.g005 <br />  <br />  <br /> RNA interference confirms 6PGDH essential, but <br /> lethality inhibition glycolysis <br />    The predictions modeling tested experimentally. <br /> A genetic mutant 6PGDH transcript knocked down <br /> by RNA interference (6PGDHRNAi) yielded reduced levels and <br /> activities 6PGDH (Figure 6A) subsequent cell death, <br /> confirming severe effects 6PGDH inhibition observed <br /> during model simulations. Antibodies 6PGDH also <br /> revealed enzyme distributed cytosol a <br /> lesser degree glycosome (Figure 6B). <br />    The 6PGDH activity measurements indicate growth of <br /> the parasites affected 6PGDH activity reduced by <br /> less 50% 24 hours. Even 72 hours, 15% of <br /> the 6PGDH activity remains. This suggests third <br /> hypotheses, i.e. PGI inhibition depletion bound <br /> phosphate moiety, unlikely. 6PGDHRNAi parasites not <br /> be rescued addition 0.5 mM ribose, alternative <br /> pathway Rib-5-P, suggesting availability Rib-5-P for <br /> nucleotide biosynthesis responsible lethal <br /> phenotype. <br />    To test proposed feedback loop related inhibition <br /> of PGI accumulating 6-PG tested 6PGDHRNAi                       Figure 6. Ablation 6PGDH. (A) Effect 6PGDH ablation the <br /> trypanosomes behaved grown fructose, glycolytic                     growth rate. A non-induced 6PGDHRNAi culture, grown glucose- <br /> substrate [11,37,45] enter pathway PGI step,           containing HMI-9 split 0 h, +tet induced tetracycline, <br /> which previously shown rescue growth 6PGDH deficient                2tet non-induced control. (B) Specific activities 6PGDH in <br /> cancer cells [42].                                                            induced control 6PGDHRNAi parasites. (C) Western blot showing <br />                                                                               predominant co-localization 6PGDH glycosomal marker <br />    Model C D extended using fructose                        aldolase (fraction S), faint band observed the <br /> transporter subsequent utilizing reactions (Figure 1), using          cytosolic fraction P marker enolase. (D) Cell densities during <br /> parameter values literature (Table S1). The rate equation            growth different substrates. At t = 0 h, 6PGDHRNAi culture grown <br /> of hexokinase adapted substrate competition            glucose split 1?105 ml21 HMI-9 glucose or <br /> glucose fructose. Steady-state calculation fructose                fructose, absence presence tetracycline. Plotted cell <br />                                                                               densities cumulative, 2tet cultures split 48 h to <br /> versions models (model Cfru Dfru) predict growth of <br />                                                                               1?105 ml21. A higher starting cell density used allow the <br /> parasites supported fructose, lethal effect             parasites adapt change carbon source. Growth fructose <br /> 6PGDH inhibition averted using substrate                 slower glucose, unable rescue induced cultures. <br /> (Figure S7). This indicates lethal effect 6PGDH             doi:10.1371/journal.pcbi.1003371.g006 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                         8                      December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                            Modelling Metabolism Trypanosoma brucei <br />  <br />  <br /> loss trypanosomes catastrophic positive feedback              tandem low levels residual glycolytic enzyme activity the <br /> loop induced PGI inhibition.                                             cytosol [46], knowing genes encode the <br />    In vitro, 6PGDHRNAi slightly slower growth glucose              glycosomal membrane translocases remain uncertain any <br /> the medium substituted fructose (Figure 6C). However,                roles play. <br /> common model predictions, growth fructose unable                     Since topology active metabolic network the <br /> rescue lethal phenotype 6PGDH ablation 6PGDHRNAi                   glycosome remains uncertain, types solutions <br /> cell line (Figure 6C). The fact G6PDH essential                 models discarded, models used to <br /> bloodstream form T. brucei [11], loss enzyme                 probe aspects behavior glycolytic pentose <br /> revert lethality associated loss 6PGDH Drosophila [40]            phosphate pathways, expanding modeling strategy intro- <br /> yeast [41], indicates loss oxidative branch         duced [8]. Simulations model types provided very <br /> the PPP se T. brucei responsible death,       similar results; behavior oxidative stress nearly identical, <br /> glycolysis inhibitory loop proposed eukaryotes.                         inhibition 6PGDH showed minor differences <br />                                                                                  models, taking account additional <br /> Discussion                                                                       uncertainty exact kinetic parameters enzymes <br />                                                                                  involved. <br />    The mathematical model glucose metabolism                              While types model showed different behavior very <br /> bloodstream form trypanosome extensively                       low 6PGDH activities, able exclude initial <br /> curated dynamic models cellular metabolism. The model                         hypothesis inhibition PGI accumulated 6-PG the <br /> accurately simulates range features glucose metabolism                explanation lethal effect 6PGDH inhibition [10,38]. This <br /> the parasitic protist. In paper, extended curated              verified experimentally RNA interference confirmed <br /> model trypanosomal glycolysis PPP. However, doing                 loss 6PGDH lethal cells lethality is <br /> so soon obvious knowledge central               retained fructose used carbon energy source. <br /> part metabolism incomplete. Using previously established               Since fructose enters pathway PGI, feedback loop via <br /> strategy explicitly handling uncertainty dynamic models,              PGI, proposed operate eukaryotes [39&#226;&#8364;&#8220;42], cannot <br /> additionally taking uncertainty metabolic network topology                explain lethality. A remaining possibility inhibition of <br /> into account, able characterize important gaps                  6PGDH renders parasite sensitive oxidative stress, but <br /> knowledge trypanosomal metabolism,                     test given knockdown gene lethal. <br /> which behaviors robust uncertainty.                                      In article demonstrate building models of <br />    The type solution observed phosphate &#226;&#8364;&#732;&#226;&#8364;&#732;leak&#226;&#8364;&#8482;&#226;&#8364;&#8482;,                metabolism provide powerful means indicate our <br /> introduced extension glycolytic model PPP,                  knowledge biological lacking. We then <br /> required presence additional enzymatic reactions               hypothesize possible solutions gaps knowledge <br /> glycosome recover bound phosphate. As example type                 test possible solutions, silico experimentally. <br /> of solution, glycosomal ribokinase added kinetic model                Interestingly, uncertainties actual structure <br /> able restore glycosomal phosphate balance generating                   resolved technical reasons and <br /> ATP. Kinetic characterization recombinant T. brucei ribokinase                experimental testing hypotheses limited, explicit <br /> showed enzyme work ribose production                consideration uncertainties model construction and <br /> direction (albeit far effectively opposite direction).          analysis allows make meaningful predictions face of <br /> Various genetic mutants ribokinase, however, subsequently                     limited information. This considerably extends usefulness of <br /> indicated unlikely ribokinase involved                dynamic models, uncertainty levered strength. <br /> restoring phosphate balance. The gene appears essential, <br /> but ribokinase enzyme apparently present vast excess             Materials Methods <br /> no phenotype observed cells transcript had <br /> been knocked RNA interference. Also, cells                   Stoichiometry <br /> inducible copy gene present place endogenous                   The computational model constructed modular way. <br /> alleles, growth occurs gene switched &#226;&#8364;&#732;&#226;&#8364;&#732;off&#226;&#8364;&#8482;&#226;&#8364;&#8482;,                   The reactions included module listed Table 1 and <br /> presumably leaky expression. The essential role small                 graphically represented Figure 1. The starting point module <br /> amounts ribokinase unknown, unlikely                describing glycolysis bloodstream form T. brucei [7,8]. A few <br /> sole enzyme capable sustaining bound-phosphate balance                 modifications kinetic equations listed below. There are <br /> these cells. Given glycosome contains multiple pathways                 PPP modules, consisting identical set of <br /> beyond glycolysis PPP, purine salvage, pyrimidine                reactions; localized entirely glycosomal matrix and <br /> biosynthesis, parts ether lipid, fatty acid isoprenoid            the cytosol. To allow flux glycosomal PPP <br /> metabolic pathways (Table S3), highly redundant                   model extended ribokinase <br /> provide robust network maintain phosphate balance.                          module ATP:ADP antiporter glycosomal <br />    A second type solution bound-phosphate leak                   membrane. The rationale extensions discussed in <br /> involve flow bound phosphates glycosome.                Results section. <br /> As example type solution, addition theoretical                   Ribose 5-phosphate ribose modeled end-products of <br /> glycosomal ADP:ATP antiporter analogous adenine                           PPP, depending model version compartment. As no <br /> nucleotide antiporter yeast peroxisomes [27] allowed                    published intracellular concentrations available ribose 5- <br /> glycolysis oxidative branch PPP operate                     phosphate ribose, cytosolic glycosomal concentra- <br /> concert, shown modeling results. However,                     tions fixed 0.01 mM unless stated otherwise. <br /> genes encoding necessary transporters identified, <br /> experimental support model predictions possible               Kinetic equations <br /> this point. Another variant type solution involve                 Generic equations.       Most rate equations follow the <br /> non-specific pores glycosomal membrane, working                        generic form: <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                            9                    December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                                                                     Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />  <br />  <br />                                                    <br />                                       &#194;&#189;S1 c=g &#194;&#189;S2 c=g <br />                                                                                                                                                                                     <br />                                      : 1{ C x,c=g <br />                                              :                                                                                                           &#194;&#189;Glc-6-P g <br />                                                                                                                                                                       : 1{ CPGI,c <br />                                       Km,S1 Km,S2 <br />                                            Keq,x                                                                                                         Km,Glc-6-P       K <br /> vx,c=g ~Vmax,c=g :                                         , &#195;&#176;1&#195;&#382;                                                                vPGI,g ~Vmax,g : <br />                                                                                                                                                                                eq,PGI <br />                                                                                                                                                                                                      &#195;&#176;8&#195;&#382; <br />                        &#194;&#189;S1     &#194;&#189;P1        &#194;&#189;S2       &#194;&#189;P2                                                                                               &#194;&#189;Glc-6-P g     &#194;&#189;Fru-6-P g         &#194;&#189;6-PG g <br />                     1z K c=g z K c=g : 1z K c=g z K c=g                                                                                            1z K            zK              zK <br />                                    m,S1          m,P1                      m,S2               m,P2                                                     m,Glc-6-P      m,Fru-6-P            i,6-PG <br />  <br /> for reactions substrates products. Reactions                                                       CPGI,g follows equation 4. <br /> only substrate product involved follow:                                                                           ATP:ADP antiporter, G6PDH, 6PGDH, PPI, RK, <br />                                                                                                                          TR. Most reversible reactions pentose phosphate <br />                                                &#194;&#189;S1 c=g                                                                  pathway modeled according Equations 1&#226;&#8364;&#8220;4, dependent on <br />                                                : 1{ C x,c=g                                                              number substrates products. For ATP:ADP <br />                                          Km,S1        Keq,x <br />                       vx,c=g ~Vmax,c=g :     &#194;&#189;S1 c=g   &#194;&#189;P1  <br />                                                                                     ,                     &#195;&#176;2&#195;&#382;            antiporter substrates S1 S2 ATPc ADPg the <br />                                          1z K        z K c=g                                                             products P1 P2 ATPg ADPc. <br />                                                m,S1      m,P1 <br />                                                                                                                            Phosphogluconolactonase. For 6-phosphogluconolacto- <br /> where x identifies reaction, S1 S2 substrates                                                      nase (PGL) reaction, spontaneous hydrolysis the <br /> P1 P2 products reaction. Subscripts c/g indicates                                                     enzyme-catalyzed rate included [48]: <br /> that separate equations used cytosolic glycosomal <br /> hexokinase. Keq standard equilibrium constant, Cx,c/g                                                                                                                          <br />                                                                                                                                                          &#194;&#189;6-PGL c=g          CPGL,c <br /> specifies ratio substrates products defined                                                                                                      : 1{ <br />                                                                                                                                                          Km,6-PGL            Keq,PGL <br />                                                                                                                                   vPGL,c=g ~Vmax,c=g :        &#194;&#189;6-PGL c=g       &#194;&#189;6-PG c=g <br />                                         &#194;&#189;P1 c=g :&#194;&#189;P2 c=g                                                                                                  1z K <br />                                                                                                                                                                  m,6-PGL <br />                                                                                                                                                                            zK <br />                                                                                                                                                                                 m,6-PG               &#195;&#176;9&#195;&#382; <br />                                Cx,c=g ~                  ,                                                &#195;&#176;3&#195;&#382; <br />                                         &#194;&#189;S1  :&#194;&#189;S2   c=g           c=g <br />                                                                                                                                                         <br />                                                                                                                                                                                &#194;&#189;6-PG c=g <br />                                                                                                                                                                                                <br />                                                                                                                                              zk:Vc=g : &#194;&#189;6-PGL          c=g { <br /> in Equation 1,                                                                                                                                                             Keq,PGL <br />  <br />                                                                                                                          k rate spontaneous hydrolysis Vc/g volume <br />                                                     &#194;&#189;P c=g                                                               relevant compartment, 5.4549 0.2451 mL per <br />                                       Cx,c=g ~                ,                                           &#195;&#176;4&#195;&#382; <br />                                                     &#194;&#189;S c=g                                                               mg protein cytosol glycosome, respectively. <br />                                                                                                                             Glc-6-P utilization. The Glc-6-P utilization reaction was <br /> in Equation 2.                                                                                                           modeled loss Glc-6-P glucose hypothetical <br />    Modified glycolysis model. The glycolysis module                                                             phosphatase reaction, using Equations 2 4. Other fates Glc-6- <br /> modified compared latest version [7,8] using reversible                                                     P, entry inositol metabolism mannose glycocon- <br /> rate equations hexokinase (HXK), phosphofructokinase (PFK)                                                           jugate biosynthesis, kinetic parameters known, are <br /> and pyruvate kinase (PYK), modeled previously                                                              covered generic Glc-6-P consuming reaction. <br /> irreversible reactions. Hexokinase follows Equation 1 3.                                                           NADPH utilization trypanothione oxidation reactions <br />    PFK modeled reversible ordered bi-bi kinetics                                                            (TOX). Glycosomal cytosolic oxidation NADPH and <br /> mixed inhibition fructose 1,6-bisphosphate [47]:                                                                      trypanothione modeled mass action kinetics: <br />  <br />  <br /> vPFK,g ~Vmax,g : <br />                            Kii        :                                                                                                       vx,c=g ~kx :&#194;&#189;NADPH c=g                                &#195;&#176;10&#195;&#382; <br />                    &#194;&#189;Fru-1,6-BP g zKii <br />                                                                   <br />                               &#194;&#189;Fru-6-P g : &#194;&#189;ATP g   : 1{ CPFK,g                                            &#195;&#176;5&#195;&#382;           kx mass-action constant reaction x. <br />                              Km,Fru-6-P Km,ATP          K     eq,PFK                                      (5) <br /> Ks,ATP     &#194;&#189;ATP g        &#194;&#189;Fru-6-P g       &#194;&#189;Fru-6-P g :&#194;&#189;ATP g        &#194;&#189;ADP g              &#194;&#189;Fru-1,6-BP g :&#194;&#189;ADP g <br />                                                                                                                 , <br /> Km,ATP   zK <br />               m,ATP <br />                       zK <br />                           m,Fru-6-P <br />                                       zK             : <br />                                           m,Fru-6-P Km,ATP <br />                                                                   zK <br />                                                                         m,ADP <br />                                                                                 z           Kis :Km,ADP                  Kinetic parameters <br />                                                                                                                            The kinetic parameters rate equations mentioned above <br />                                                                                                                          predominantly derived literature. If parameters of <br /> where CPFK,g follows Equation 3, Kii Kis mixed                                                          T. brucei enzyme available, parameters related <br /> inhibition fructose 1,6-bisphosphate taken [47].                                                    species used. If necessary, Keq values corrected pH 7. <br />   Pyruvate kinase modeled reversibly cooperativity PEP:                                                      Kinetic parameters used fixed parameter model listed in <br />                                                                                                                          Table S1. Distributions parameters used uncertainty <br /> vPK,c ~ <br />                                                                                                                          modeling given Text S1. <br />                                    <br />                             &#194;&#189;PEP c n : &#194;&#189;ADP c       : 1{ CPK,c <br />                              KPEP     Km,ADP            K          eq,PK                                  &#195;&#176;6&#195;&#382;            Metabolite concentrations <br /> Vmax,c :                                                 <br />                  &#194;&#189;PEP c n   &#194;&#189;Pyr c  : 1z &#194;&#189;ADP c z &#194;&#189;ATP c                                                                    The extension model glycosomal pentose <br />              1z K         zK            K        K                                                                       phosphate pathway led extension conserved moiety of <br />                        PEP             m,Pyr                   m,ADP                m,ATP <br />                                                                                                                          phosphorylated glycosomal metabolites compared earlier <br /> where CPK,c meaning Equation 3,                                                             model versions [4]. For precise overview conserved moieties in <br />                                                                                                                          model version Table 2. The new conserved sum was <br />                                                     <br />                                   &#194;&#189;ATP c &#194;&#189;ADP c                                                                          chosen extension glycolysis model the <br />                 KPEP ~Km,PEP : 1z        z         :                                                      &#195;&#176;7&#195;&#382;            glycosomal PPP ribokinase results similar steady state <br />                                   Ki,ADP   Ki,ADP <br />                                                                                                                          concentrations glycolytic metabolites models. The <br />   Phosphoglucose isomerase. The known inhibition                                                                      conserved sum NADPH cytosol glycosome was <br /> phosphoglucose isomerase (PGI) PPP intermediate 6-                                                                arbitrarily chosen 2 mM, conserved sum of <br /> phosphogluconate (6-PG) included described [43]:                                                               trypanothione chosen 0.38 mM [12]. In models <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                                                   10                    December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                      Modelling Metabolism Trypanosoma brucei <br />  <br />  <br /> including fructose transport, cytosolic Fru-6-P fixed           hygromycin phosphotransferase puromycin N-acetyl-transferase <br /> measured steady-state value 0.9 mM, additional reactions          gene antibiotic resistance markers. The tetracycline-inducible <br /> the cytosol included model consume Fru-          expression ribokinase conditional knockout DRK::HYG/ <br /> 6-P. Inorganic phosphate (Pi) explicitly included            PAC::RKtet generated cloning ribokinase coding <br /> model, assumed present saturating levels             sequence pHD1336 using cell line 449 (kind gifts Prof <br /> regulated independently glycolysis.                                     Christine Clayton, Heidelberg). Parasites transfected by <br />                                                                            Amaxa nucleofection (Lonza) [57] using Tb-BSF buffer [58], and <br /> Calculations simulations                                               plated limiting serial dilution. <br />    The model initially constructed simulations were <br /> performed PySCeS 0.7.8 [49] COPASI 4.8 [50]. SBML                    Western blotting <br /> versions models supplied Dataset S1. The models              Polyclonal antibodies T. brucei ribokinase raised in <br /> this paper additionally modeled taking uncer-              rabbits injection purified recombinant T. brucei ribokinase, <br /> tainty account described [8]. Each simulation exists         purified protein A-chromatography, antibody titer of <br /> 1000 individual runs, run parameter values           .1:192,000 measured ELISA (GenicBio, Shanghai, <br /> sampled simultaneously. The 6PGDH Vmax scans Figures 5              China). The anti-serum purified immunoaffinity <br /> S5, S6, S7, S8 250 simulations, longer time-limit          chromatography. Recombinant T. brucei ribokinase coupled to <br /> accumulating metabolites greatly increased time                mixture Affi-Gel 10 15 (Bio-Rad). Antibodies eluted <br /> simulation finish (a week desktop PC model D). The             column 3.5 M MgCl2. Polyclonal antibodies against <br /> parameter distributions described Text S1,           T. brucei 6PGDH raised rats infection purified <br /> while parameter sets provided Dataset S2, S3.                   recombinant T. brucei 6PGDH. <br />                                                                               For protein samples, 5?106 cells harvested centrifugation <br /> Expression recombinant ribokinase                                          washed PBS. Samples separated NuPAGE 4&#226;&#8364;&#8220; <br />    The T. brucei ribokinase gene (GenBank ID:3664382)                  12% Bis-Tris precast gels (Life Technologies) 45 min 200 V <br /> amplified Lister 427 strain T. brucei genomic DNA                 transferred nitrocellulose membrane (Hybond-ECL, <br /> cloned plasmid pET28a(+) (Novagen) add N-terminal               Amersham) 100 V 2 h. Membranes blocked overnight <br /> His-tag. The protein overexpressed E. coli BL21(DE3),               4uC 5% milk PBS-T (PBS 0.05% Tween-20), washed <br /> purified Ni2+ chelate chromatography dialyzed 50 mM              times PBS-T, probed purified polyclonal <br /> Tris-HCl pH 7.0, 100 mM NaCl, 20% glycerol. Purity                     rabbit anti-T. brucei ribokinase antibody, diluted 1:2000 1% milk <br /> assessed SDS-PAGE Coomassie blue staining.                          PBS-T, 4 h room temperature. After additional <br />                                                                            washes PBS-T, membrane incubated the <br /> Enzyme activity assays                                                     secondary antibody (goat anti-rabbit IgG peroxidase conjugate, <br />                                                                            Calbiochem) 1:2000 1% milk PBS-T 2 h room <br />    Activity ribokinase assayed 25uC buffer containing <br />                                                                            temperature. After washes PBS-T, horseradish peroxidase <br /> 10 mM KH2PO4, 15 mM NaCl, 85 mM KCl 10 mM MgCl2 <br />                                                                            activity measured SuperSignal HRP substrate (Novagen). <br /> at pH 7. Ribokinase activity direction ribose 5-phosphate <br />                                                                               Subcellular localization 6PGDH trypanosomes was <br /> production assayed spectrophotometrically according [51], <br />                                                                            performed modification digitonin fractionation method <br /> with 5 mM ribose, 5 mM phosphoenolpyruvate, 5 mM ATP, <br />                                                                            described [59]. Specifically, 5?107 procyclic cells washed <br /> 0.2 mM NADH, 5 u PK 5 u LDH. Ribokinase activity the <br />                                                                            STEN buffer (250 mM sucrose; 25 mM Tris-HCl, pH 7.5; <br /> direction ribose production assayed spectrophotometrically <br />                                                                            1 mM EDTA 150 mM NaCl) resuspended 300 mL <br /> with 5 mM ribose 5-phosphate, 5 mM glucose, 5 mM ADP, <br />                                                                            STEN supplemented protease inhibitors (Roche). The cells <br /> 0.2 mM NADP, 5 u HXK 5 u G6PDH. <br />                                                                            differentially permeabilized 40 mg digitonin 100 mg <br />                                                                            cellular protein (2.5?107 cells) 2 hours ice centrifuged at <br /> Parasites genetic mutants                                              16,000 g minutes. After TCA precipitation, equal amounts <br />    Bloodstream form Lister 427 strain T. brucei parasites             (equivalent 4?106 cells) supernatant (S) pellet (P) were <br /> routinely cultured HMI-9 [52] 37uC 5% CO2. For growth            analyzed SDS-PAGE blotted rat anti-Tb6PGDH. A <br /> of 6PGDHRNAi alternative hexose source, glucose HMI-9             goat anti-rabbit IgG peroxidase used visualization as <br /> was substituted fructose concentration. Antibiotic        described ribokinase. Enolase aldolase used as <br /> concentrations genetic mutants were: 0.5 mg mL21 phleomycin            cytosolic glycosomal markers, respectively, antibodies <br /> and 2.5 mg mL21 hygromycin RKRNAi; 0.5 mg mL21 phleo-                  proteins kindly provided Prof. Paul Michels <br /> mycin, 5 mg mL21 G418 10 mg mL21 hygromycin                        (University Louvain, Belgium). <br /> 6PGDHRNAi; 0.5 mg mL21 puromycin DRK::PAC/RK; <br /> 2.5 mg mL21 hygromycin DRK::HYG/RK 0.5 mg mL21 <br />                                                                            Supporting Information <br /> puromycin, 2.5 mg mL21 hygromycin, 0.5 mg mL21 phleomycin <br /> and 2 mg mL21 blasticidin DRK::HYG/PAC::RKtet. Sensitivity             Dataset S1 SBML versions models. SBML versions of <br /> of T. brucei ribose assayed alamarBlue described            models presented Figure 1. <br /> previously [53].                                                           (ZIP) <br />    The ribokinase knockdown cell line RKRNAi generated <br />                                                                            Dataset S2 Parameter sets used simulations of <br /> using pRPaiSL vector/2T1 cell line [54], 6- <br />                                                                            models A&#226;&#8364;&#8220;D. Sets parameter values sampled the <br /> phosphogluconate dehydrogenase (Tb427tmp.211.3180) knock- <br />                                                                            distributions described Text S1, used simulations of <br /> down cell line 6PGDHRNAi generated using p2T7 vector/ <br />                                                                            models A&#226;&#8364;&#8220;D growth glucose. <br /> 1313 cell line [55]. The generation ribokinase <br />                                                                            (CSV) <br /> knockout cell line DRK::HYG/PAC attempted using <br /> constructs containing 600 bp 59 39 UTRs adjacent                Dataset S3 Parameter sets used simulations of <br /> ribokinase gene, cloned vector pTBT [56], flanking           models Cfru Dfru. Sets parameter values sampled <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                     11                    December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                         Modelling Metabolism Trypanosoma brucei <br />  <br />  <br /> from distributions described Text S1, used simulations          steady state flux cytosolic pentose phosphate pathway <br /> of models Cfru Dfru.                                                      model D function oxidative stress varying the <br /> (CSV)                                                                         kinetic constant kTOX. (B) Fluxes cytosolic PPP <br />                                                                               enzymes function time sudden oxidative stress. <br /> Figure S1 Enzymes purine salvage pathway localized <br />                                                                               During time-course, kTOX = 2 ml?min21?mg protein21. <br /> in glycosome. The glycosomal PPP shown enzymes <br />                                                                               The removed steady state t = 0, setting 99% of <br /> from purine salvage pathway predicted glycosomal <br />                                                                               NAD(P)H trypanothione pools oxidized form. <br /> localization (Table S3). Reactions PPP model indicated <br />                                                                               Shown relaxation cytosolic PPP fluxes. Solid lines <br /> by number Table 1, additional reactions are <br />                                                                               indicate medians, shaded areas interquartile ranges. <br /> indicated as: PRPP: phosphoribosyl pyrophosphate synthetase; <br />                                                                               (TIF) <br /> AdK: adenosine kinase; ANase: adenosine nucleosidase; APRT: <br /> adenine phosphoribosyltransferase. Metabolites                   Figure S6 Simulations 6PGDH inhibition 6-PG <br /> balanced pathway indicated bold. The scheme                accumulation. (A&#226;&#8364;&#8220;B) The effects inhibition 6PGDH 6- <br /> demonstrates presence purine salvage pathway                PG concentrations metabolic fluxes simulated by <br /> unable rescue phosphate leak, resulting overall reaction        reducing Vmax,6PGDH model C D low oxidative stress <br /> is glucose+ATPRribose+AMP+PPi (+CO2, implied balanced                   (kTOX = 2 ml?min21?mg protein21). Results simulations low <br /> with gaseous CO2). Additional ADK ANase reactions does                oxidative stress (kTOX = 200 ml?min21?mg protein21) shown in <br /> improve situation, resulting overall reaction                  Figure 5. ATP production flux steady-state flux PFK is <br /> glucose+ADPR2 ribose+adenine+2 PPi (+CO2). In contrast,                       indicated red, trypanothione reductase steady-state flux is <br /> ribokinase capable resolving phosphate leak                  indicated yellow, plotted left y-axis. Steady-state <br /> resulting overall reaction glucoseRribose (+CO2).                          concentration cytosolic (blue) glycosomal (green) 6- <br /> (TIF)                                                                         phosphogluconate plotted right y-axis. Shaded areas <br />                                                                               indicate interquartile ranges. <br /> Figure S2 Steady-state fluxes various models. <br />                                                                               (TIF) <br /> Steady state fluxes standard conditions (green, kTOX = 2 ml?- <br /> min21?mg protein21) cytosolic PPP maximized (red,                   Figure S7 Simulations 6PGDH inhibition during <br /> kTOX = 200 ml?min21?mg protein21). Error bars indicate inter-                 growth fructose. (A&#226;&#8364;&#8220;B) The effects inhibition 6PGDH <br /> quartile ranges. NA denotes &#226;&#8364;&#732;Not Applicable&#226;&#8364;&#8482; branches            6-PG concentrations metabolic fluxes simulated by <br /> absent certain model version, Glce 5 mM models.                reducing Vmax,6PGDH model C D high oxidative stress <br /> The glucose consumption flux distributed production            (kTOX = 200 ml?min21?mg protein21), similar Figure 5. ATP <br /> glycerol pyruvate branches PPP. Note              production flux steady-state flux PFK indicated in <br /> ALD generates trioses hexose, fluxes             red, trypanothione reductase steady-state flux indicated in <br /> through trioses glycerol pyruvate double hexose               yellow, plotted left y-axis. Steady-state concentration <br /> flux. The errors bars indicated interquartile ranges result           cytosolic (blue) glycosomal (green) 6-phosphogluconate are <br /> the uncertainty modelling. The large error bars glycerol                  plotted right y-axis. Shaded areas indicate interquartile <br /> production result low flux smaller uncertainties         ranges. <br /> assigned fluxes pathways. No information             (TIF) <br /> is given model B, model incomplete reach <br />                                                                               Figure S8 Percentage models reaching steady-state 10 <br /> a steady state (outlined main text). <br />                                                                               million simulation minutes 6PGDH inhibitions. As <br /> (TIF) <br />                                                                               explained in Text S1, 250 random parameter sets were <br /> Figure S3 Genetic investigations T. brucei riboki-                         used calculating steady-states 6PGDH inhibition <br /> nase. (A) Northern blot RKRNAi induced non-induced T.                  (Figures 5 S5). Outcome model C shown high (A) <br /> brucei, samples taken 24, 48 72 hours post induction                   low (B) oxidative stress; model D high (C) low (D) <br /> tetracycline. Tubulin used loading control. (B) Western blot           oxidative stress. Panel A C correspond Figures 5A&#226;&#8364;&#8220;B, while <br /> of RKRNAi Drk::HYG/PAC::RKtet, samples taken t = 0, 1,                 panel B D correspond Figures S6A&#226;&#8364;&#8220;B. <br /> 2 3 days post induction tetracycline. (C) Growth curve              (TIF) <br /> ribokinase knockdown mutant RNA interference. Induction of <br />                                                                               Figure S9 Percentage models reaching steady-state 10 <br /> RNAi started t = 0 addition 1 mg/mL tetracycline. <br />                                                                               million simulation minutes 6PGDH inhibitions with <br /> Cell densities induced (dotted line) control (solid line) <br />                                                                               growth fructose. As explained in Text S1, 250 <br /> cultures determined cell counts, cultures diluted <br />                                                                               random parameter sets used calculating steady-states <br /> down 2?104 cells ml21 t = 48 96 h. No difference in <br />                                                                               6PGDH inhibition (Figures 6 S6). Outcome high <br /> growth effect observed. (D) Successful transfection T. <br />                                                                               oxidative stress shown model C (A) D (B). Results from <br /> brucei knockout constructs confirmed PCR. <br />                                                                               simulations shown Figure S7. <br /> (TIF) <br />                                                                               (TIF) <br /> Figure S4 Ribose sensitivity. Steady state concentrations (A) <br />                                                                               Table S1 Kinetic parameters enzymes related PPP. <br /> and fluxes (B) model C various concentrations ribose. Solid <br />                                                                               Activities, depend expression level enzymes, are <br /> lines indicate medians, shaded areas interquartile ranges. <br />                                                                               specified separately fraction enzyme localized in <br /> Increasing ribose concentration results depletion ATP and <br />                                                                               glycosome and, applicable, fraction located in <br /> accumulation ribose 5-phosphate glycosome. While the <br />                                                                               cytosol. c/g indicates cytosolic glycosomal activities are <br /> glycosomal PPP flux (JG6PDHg) remains unaffected, the <br />                                                                               identical. Other parameters assumed identical for <br /> glycolytic flux (JPGI) strongly reduced. <br />                                                                               glycosomal cytosolic enzyme fractions. Parameter values given <br /> (TIF) <br />                                                                               used fixed parameter models. Distributions of <br /> Figure S5 Simulations oxidative stress model D.                         parameter values used uncertainty modelling given Text S1. <br /> Near identical Figure 4, results model D instead. (A) The          (DOCX) <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                        12                     December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                                                Modelling Metabolism Trypanosoma brucei <br />  <br />  <br /> Table S2 Elementary flux modes models PPP. The                                           alternative oxidase (TAO), phosphofructokinase (PFK), fructose <br /> elementary modes different model versions including                                     bisphosphatase (FBPase), phosphogluconolactonase (PGL). <br /> glycosomal PPP listed overall reactions plus brackets                            Protons, Pi, PPi, H2O, CO2 O2 included the <br /> the individual enzyme-catalyzed reactions relative flux                             reactions. Glucose, 3-phosphoglyceric acid, ribose glycerol <br /> weight. For simplicity cytosolic PPP cytosolic NADPH                               set external metabolites. The resulting model 9 <br /> utilization left analysis. The modules included                            elementary modes, modes occurring in <br /> each model version refer Table 1 color-coded                                        models described paper. Elementary mode 5 futile <br /> extensions Figure 1 main text. A negative number                                     cycle external metabolites involved. Elementary modes 6&#226;&#8364;&#8220; <br /> indicates reaction occurs reverse direction                                 9 unlikely occur dividing bloodstream trypanosomes, a <br /> compared Table 1 main text. The order reactions                                   high glucose pyruvate glycerol flux maintained. <br /> corresponds Table 1 main text. The glycolytic                                Additionally, activity fructose bisphosphatase be <br /> modes 1&#226;&#8364;&#8220;3 possible model versions, listed                               measured [23,61]. <br /> again model versions extended glycosomal PPP.                                 (DOCX) <br /> Elementary mode analysis model B results flux <br /> modes 1&#226;&#8364;&#8220;3.                                                                                     Text S1 Uncertainty modeling distribution of <br /> (DOCX)                                                                                         parameter values. Detailed description methodology <br />                                                                                                uncertainty modeling distributions defined sampling of <br /> Table S3 Predicted glycosomal proteome. All reactions                                          random parameter values. <br /> present bloodstream form T. brucei glycosome, according                              (DOCX) <br /> comprehensible glycosomal proteomics [18,19]. Only reactions <br /> present bloodstream form parasite included. <br /> Abbreviations EC numbers given reaction.                                      Acknowledgments <br /> Models indicated reactions model                                We like thank Jurgen Haanstra careful look the <br /> main text. Indicated reactions elementary                              manuscript modelling. <br /> model (see Table S4). <br /> (DOCX)                                                                                         Author Contributions <br /> Table S4 Elementary modes glycosomal proteome. A                                            Conceived designed experiments: EJK FA RJB IHG DG RB BMB <br /> model glycosomal metabolism constructed use                                      MPB. Performed experiments: EJK FA VPA MT NND. Analyzed the <br /> METATOOL version 4.9.3 [60]. The reactions Table S3                                       data: EJK FA VPA MT NND. Contributed reagents/materials/analysis <br /> were used. All reactions set reversible,                                    tools: RJB. Wrote paper: EJK FA RB BMB MPB. <br />  <br />  <br /> References <br />  1. Barrett MP, Burchmore RJ, Stich A, Lazzari JO, Frasch AC, et al. (2003) The                      malic enzyme pentose phosphate pathway fed gluconeogenic flux. <br />     trypanosomiases. Lancet 362: 1469&#226;&#8364;&#8220;1480. doi:10.1016/S0140-6736(03)14694-6.                       J Biol Chem 288: 18494&#226;&#8364;&#8220;18505. doi:10.1074/jbc.M113.462978. <br />  2. Brun R, Blum J, Chappuis F, Burri C (2010) Human African trypanosomiasis.                  14.   Barrett MP, Le Page RWF (1993) A 6-phosphogluconate dehydrogenase gene <br />     Lancet 375: 148&#226;&#8364;&#8220;159. doi:10.1016/S0140-6736(09)60829-1.                                          Trypanosoma brucei. Mol Biochem Parasitol 57: 89&#226;&#8364;&#8220;99. doi:10.1016/ <br />  3. Barrett MP, Boykin DW, Brun R, Tidwell RR (2007) Human African                                   0166-6851(93)90247-U. <br />     trypanosomiasis: pharmacological re-engagement neglected disease.                   15.   Heise N, Opperdoes FR (1999) Purification, localisation characterisation of <br />     Br J Pharmacol 152: 1155&#226;&#8364;&#8220;1171. doi:10.1038/sj.bjp.0707354.                                       glucose-6-phosphate dehydrogenase Trypanosoma brucei. Mol Biochem <br />  4. Bakker BM, Michels PAM, Opperdoes FR, Westerhoff HV (1997) Glycolysis                         Parasitol 99: 21&#226;&#8364;&#8220;32. <br />     bloodstream form Trypanosoma brucei understood terms                      16.   Duffieux F, Van Roy J, Michels PA, Opperdoes FR (2000) Molecular <br />     kinetics glycolytic enzymes. J Biol Chem 272: 3207&#226;&#8364;&#8220;3215. doi:10.1074/                     characterization enzymes pentose-phosphate pathway of <br />     jbc.272.6.3207.                                                                                  Trypanosoma brucei. Glucose-6-phosphate dehydrogenase 6-phosphoglu- <br />  5. Bakker BM, Michels PAM, Opperdoes FR, Westerhoff HV (1999) What                                  conolactonase. J Biol Chem 275: 27559&#226;&#8364;&#8220;27565. doi:10.1074/jbc.M004266200. <br />     controls glycolysis bloodstream form Trypanosoma brucei? J Biol Chem 274:               17.   Stoffel SA, Alibu VP, Hubert J, Ebikeme C, Portais J-C, et al. (2011) <br />     14551&#226;&#8364;&#8220;14559. doi:10.1074/jbc.274.21.14551.                                                       Transketolase Trypanosoma brucei. Mol Biochem Parasitol 179: 1&#226;&#8364;&#8220;7. <br />  6. Helfert S, Este&#194;&#180;vez AM, Bakker B, Michels P, Clayton C (2001) Roles                           doi:10.1016/j.molbiopara.2011.04.006. <br />     triosephosphate isomerase aerobic metabolism Trypanosoma brucei.                    18.   Colasante C, Ellis M, Ruppert T, Voncken F (2006) Comparative proteomics of <br />     Biochem J 357: 117&#226;&#8364;&#8220;125.                                                                          glycosomes bloodstream form procyclic culture form Trypanosoma <br />  7. Albert M-A, Haanstra JR, Hannaert V, Roy JV, Opperdoes FR, et al. (2005)                         brucei brucei. Proteomics 6: 3275&#226;&#8364;&#8220;3293. doi:10.1002/pmic.200500668. <br />     Experimental silico analyses glycolytic flux control bloodstream form         19.   Vertommen D, Van Roy J, Szikora J-P, Rider MH, Michels PAM, et al. (2008) <br />     Trypanosoma brucei. J Biol Chem 280: 28306&#226;&#8364;&#8220;28315. doi:10.1074/                                   Differential expression glycosomal mitochondrial proteins two <br />     jbc.M502403200.                                                                                  major life-cycle stages Trypanosoma brucei. Mol Biochem Parasitol 158: 189&#226;&#8364;&#8220; <br />  8. Achcar F, Kerkhoven EJ, The SilicoTryp Consortium, Bakker BM, Barrett MP,                        201. doi:10.1016/j.molbiopara.2007.12.008. <br />     et al. (2012) Dynamic modelling uncertainty: case Trypanosoma                 20.   Opperdoes FR, Szikora J-P (2006) In silico prediction glycosomal enzymes <br />     brucei energy metabolism. PLoS Comput Biol 8: e1002352. doi:10.1371/                             Leishmania major trypanosomes. Mol Biochem Parasitol 147: 193&#226;&#8364;&#8220;206. <br />     journal.pcbi.1002352.                                                                            doi:10.1016/j.molbiopara.2006.02.010. <br />  9. Haanstra JR, Kerkhoven EJ, van Tuijl A, Blits M, Wurst M, et al. (2011) A domino           21.   Visser N, Opperdoes FR (1980) Glycolysis Trypanosoma brucei. <br />     effect drug action: metabolic assault parasite differentiation. Mol              Eur J Biochem 103: 623&#226;&#8364;&#8220;632. <br />     Microbiol 79: 94&#226;&#8364;&#8220;108. doi:10.1111/j.1365-2958.2010.07435.x.                                22.   Turnock DC, Ferguson MAJ (2007) Sugar Nucleotide Pools Trypanosoma <br /> 10. Hanau S, Rippa M, Bertelli M, Dallocchio F, Barrett MP (1996) 6-                                 brucei, Trypanosoma cruzi, Leishmania major. Eukaryotic Cell 6: 1450&#226;&#8364;&#8220; <br />     Phosphogluconate dehydrogenase Trypanosoma brucei. Eur J Biochem                            1463. doi:10.1128/EC.00175-07. <br />     240: 592&#226;&#8364;&#8220;599. doi:10.1111/j.1432-1033.1996.0592h.x.                                        23.   Cronin CN, Nolan DP, Paul Voorheis H (1989) The enzymes classical <br /> 11. Cordeiro AT, Thiemann OH, Michels PAM (2009) Inhibition Trypanosoma                           pentose phosphate pathway display differential activities procyclic and <br />     brucei glucose-6-phosphate dehydrogenase human steroids effects                     bloodstream forms Trypanosoma brucei. FEBS Letters 244: 26&#226;&#8364;&#8220;30. <br />     viability cultured parasites. Bioorg Med Chem 17: 2483&#226;&#8364;&#8220;2489. doi:doi:                  doi:10.1016/0014-5793(89)81154-8. <br />     DOI: 10.1016/j.bmc.2009.01.068.                                                            24.   Tewari YB, Steckler DK, Goldberg RN, Gitomer WL (1988) Thermodynamics <br /> 12. Fairlamb AH, Henderson GB, Bacchi CJ, Cerami A (1987) In vivo effects                         hydrolysis sugar phosphates. J Biol Chem 263: 3670&#226;&#8364;&#8220;3675. <br />     difluoromethylornithine trypanothione polyamine levels bloodstream               25.   Krakow JL, Wang CC (1990) Purification characterization glycerol kinase <br />     forms Trypanosoma brucei. Mol Biochem Parasitol 24: 185&#226;&#8364;&#8220;191.                                  Trypanosoma brucei. Mol Biochem Parasitol 43: 17&#226;&#8364;&#8220;25. <br /> 13. Allmann S, Morand P, Ebikeme C, Gales L, Biran M, et al. (2013) Cytosolic                  26.   Voncken F, Gao F, Wadforth C, Harley M, Colasante C (2013) The <br />     NADPH homeostasis glucose-starved procyclic Trypanosoma brucei relies                      Phosphoarginine Energy-Buffering System Trypanosoma brucei Involves <br />  <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                         13                         December 2013 | Volume 9 | Issue 12 | e1003371 <br />                                                                                                                                   Modelling Metabolism Trypanosoma brucei <br />  <br />  <br />       Multiple Arginine Kinase Isoforms Different Subcellular Locations. PLoS                      characterization gene analysis enzyme. Eur J Biochem 184: <br />       ONE 8: e65908. doi:10.1371/journal.pone.0065908.                                                  455&#226;&#8364;&#8220;464. <br /> 27.   Van Roermund CW, Drissen R, van Den Berg M, Ijlst L, Hettema EH, et al.                     44.   Goldberg RN, Tewari YB (1995) Thermodynamics Enzyme-Catalyzed <br />       (2001) Identification peroxisomal ATP carrier required medium-chain                      Reactions: Part 5. Isomerases Ligases. J Phys Chem Ref Data 24: 1765&#226;&#8364;&#8220; <br />       fatty acid beta-oxidation normal peroxisome proliferation Saccharomyces                    1801. <br />       cerevisiae. Mol Cell Biol 21: 4321&#226;&#8364;&#8220;4329. doi:10.1128/MCB.21.13.4321-                        45.   Ryley J (1962) Studies metabolism protozoa. 9. Comparative <br />       4329.2001.                                                                                        metabolism blood-stream culture forms Trypanosoma rhodesiense. <br /> 28.   Teusink B, Walsh MC, van Dam K, Westerhoff HV (1998) The danger                                Biochem J 85: 211&#226;&#8364;&#8220;223. <br />       metabolic pathways turbo design. Trends Biochem Sci 23: 162&#226;&#8364;&#8220;169.                       46.   Achcar F, Barrett MP, Breitling R (2013) Explicit consideration topological <br />       doi:10.1016/S0968-0004(98)01205-5.                                                                parameter uncertainty gives new insights well-established model of <br /> 29.   Blangy D, Buc H, Monod J (1968) Kinetics allosteric interactions                        glycolysis. FEBS Journal doi: 10.1111/febs.12436. <br />       phosphofructokinase Escherichia coli. J Mol Biol 31: 13&#226;&#8364;&#8220;35. doi:10.1016/               47.   Cronin CN, Tipton KF (1987) Kinetic studies reaction catalysed by <br />       0022-2836(68)90051-X.                                                                             phosphofructokinase Trypanosoma brucei. Biochem J 245: 13&#226;&#8364;&#8220;18. <br /> 30.   Bakker BM, Mensonides FI, Teusink B, van Hoek P, Michels PA, et al. (2000)                  48.   Miclet E, Stoven V, Michels PA, Opperdoes FR, Lallemand JY, et al. (2001) <br />       Compartmentation protects trypanosomes dangerous design                               NMR spectroscopic analysis steps pentose-phosphate <br />       glycolysis. Proc Natl Acad Sci USA 97: 2087&#226;&#8364;&#8220;2092. doi:10.1073/                                    pathway elucidates role 6-phosphogluconolactonase. J Biol Chem 276: <br />       pnas.030539197.                                                                                   34840&#226;&#8364;&#8220;34846. doi:10.1074/jbc.M105174200. <br /> 31.   Haanstra JR, Tuijl A van, Kessler P, Reijnders W, Michels PAM, et al. (2008)                49.   Olivier BG, Rohwer JM, Hofmeyr J-HS (2005) Modelling cellular systems with <br />       Compartmentation prevents lethal turbo-explosion glycolysis trypanosomes.                 PySCeS. Bioinformatics 21: 560&#226;&#8364;&#8220;561. doi:10.1093/bioinformatics/bti046. <br />       Proc Natl Acad Sci USA 105: 17718&#226;&#8364;&#8220;17723. doi:10.1073/pnas.0806664105.                       50.   Hoops S, Sahle S, Gauges R, Lee C, Pahle J, et al. (2006) COPASI&#226;&#8364;&#8220;a COmplex <br /> 32.   Furuya T, Kessler P, Jardim A, Schnaufer A, Crudder C, et al. (2002) Glucose                   PAthway SImulator. Bioinformatics 22: 3067&#226;&#8364;&#8220;3074. doi:10.1093/bioinfor- <br />       toxic glycosome-deficient trypanosomes. Proc Natl Acad Sci USA 99: 14177&#226;&#8364;&#8220;                      matics/btl485. <br />       14182. doi:10.1073/pnas.222454899.                                                          51.   Anderson A, Cooper RA (1969) The significance ribokinase ribose <br /> 33.   Kessler PS, Parsons M (2005) Probing role compartmentation glycolysis                   utilization Escherichia coli. Biochim Biophys Acta 177: 163&#226;&#8364;&#8220;165. <br />       procyclic form Trypanosoma brucei RNA interference studies PEX14,                     52.   Hirumi H, Hirumi K (1989) Continuous cultivation Trypanosoma brucei <br />       hexokinase phosphofructokinase. J Biol Chem 280: 9030&#226;&#8364;&#8220;9036.                                   blood stream forms medium containing low concentration serum <br />       doi:10.1074/jbc.M412033200.                                                                       protein feeder cell layers. J Parasitol 75: 985&#226;&#8364;&#8220;989. <br />                                                                                                   53.   Ward CP, Wong PE, Burchmore RJ, Koning HP, Barrett MP (2011) <br /> 34.   Gualdron-Lo&#194;&#180;pez M, Vapola MH, Miinalainen IJ, Hiltunen JK, Michels PAM, et <br />                                                                                                         Trypanocidal furamidine analogues: influence pyridine nitrogens on <br />       al. (2012) Channel-forming activities glycosomal fraction the <br />                                                                                                         trypanocidal activity, transport kinetics, resistance patterns. Antimicrob <br />       bloodstream form Trypanosoma brucei. PLoS ONE 7: e34530. doi:10.1371/ <br />                                                                                                         Agents Chemother 55: 2352&#226;&#8364;&#8220;2361. doi:10.1128/AAC.01551-10. <br />       journal.pone.0034530. <br />                                                                                                   54.   Alsford S, Horn D (2008) Single-locus targeting constructs reliable regulated <br /> 35.   Maugeri DA, Cazzulo JJ (2004) The pentose phosphate pathway in <br />                                                                                                         RNAi transgene expression Trypanosoma brucei. Mol Biochem Parasitol <br />       Trypanosoma cruzi. FEMS Microbiol Lett 234: 117&#226;&#8364;&#8220;123. doi:10.1016/ <br />                                                                                                         161: 76&#226;&#8364;&#8220;79. doi:10.1016/j.molbiopara.2008.05.006. <br />       j.femsle.2004.03.018. <br />                                                                                                   55.   Alibu VP, Storm L, Haile S, Clayton C, Horn D (2005) A doubly inducible <br /> 36.   Grant PT, Fulton JD (1957) The catabolism glucose strains                                RNA interference rapid RNAi plasmid construction in <br />       Trypanosoma rhodesiense. Biochem J 66: 242&#226;&#8364;&#8220;250.                                                   Trypanosoma brucei. Mol Biochem Parasitol 139: 75&#226;&#8364;&#8220;82. doi:10.1016/ <br /> 37.   Haanstra JR, van Tuijl A, van Dam J, van Winden W, Tielens AGM, et al.                            j.molbiopara.2004.10.002. <br />       (2012) Proliferating bloodstream-form Trypanosoma brucei use negligible              56.   Cross M, Kieft R, Sabatini R, Dirks-Mulder A, Chaves I, et al. (2002) J-binding <br />       consumed glucose anabolic processes. Int J Parasitol 42: 667&#226;&#8364;&#8220;673.                          protein increases level retention unusual <span id='am-394' about='xsp:base' typeof='owl:Thing'>base</span> J trypanosome <br />       doi:10.1016/j.ijpara.2012.04.009.                                                                 DNA. Mol Microbiol 46: 37&#226;&#8364;&#8220;47. doi:10.1046/j.1365-2958.2002.03144.x. <br /> 38.   LePage RW, Barrett MP (1990) Cloning 6-phosphogluconate dehydro-                     57.   Burkard G, Fragoso CM, Roditi I (2007) Highly efficient stable transformation <br />       genase gene Trypanosoma brucei complementation Escherichia coli.                       bloodstream forms Trypanosoma brucei. Mol Biochem Parasitol 153: 220&#226;&#8364;&#8220; <br />       Biochem Soc Trans 18: 724&#226;&#8364;&#8220;727.                                                                    223. doi:10.1016/j.molbiopara.2007.02.008. <br /> 39.   Bewley GC, Lucchesi JC (1975) Lethal effects low &#226;&#8364;&#732;&#226;&#8364;&#732;null&#226;&#8364;&#8482;&#226;&#8364;&#8482; activity alleles        58.   Schumann Burkard G, Jutzi P, Roditi I (2011) Genome-wide RNAi screens in <br />       6-phosphogluconate dehydrogenase Drosophila melanogaster. Genetics 79:                         bloodstream form trypanosomes identify drug transporters. Mol Biochem <br />       451&#226;&#8364;&#8220;457.                                                                                          Parasitol 175: 91&#226;&#8364;&#8220;94. doi:10.1016/j.molbiopara.2010.09.002. <br /> 40.   Hughes MB, Lucchesi JC (1977) Genetic rescue lethal &#226;&#8364;&#732;&#226;&#8364;&#732;null&#226;&#8364;&#8482;&#226;&#8364;&#8482; activity allele           59.   Hauser R, Pypaert M, Hausler T, Horn EK, Schneider A (1996) In vitro import <br />       6-phosphogluconate dehydrogenase Drosophila melanogaster. Science 196:                      pr <br /> </body></html>