<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Int. J. Biol. Sci. 2013, Vol. 9                                                                                                                       267 <br />  <br />            Ivyspring <br />           International Publisher <br />                                                                International Journal Biological Sciences <br />                                                                                                          2013; 9(3):267-278. doi: 10.7150/ijbs.4887 <br /> Research Paper <br />  <br /> Yes-Associated Protein (Yap) Is Required Early <br /> Embryonic Development Zebrafish (Danio Rerio) <br /> Jingying Hu1*, Shuna Sun2*, Qiu Jiang1, Shaoyang Sun1, Wei Wang2, Yonghao Gui2, Houyan Song1&#239;&#8364;&#170; <br /> 1.   Department Biochemistry Molecular Biology, Shanghai Medical School Key Laboratory Molecular Medicine, Ministry of <br />      Education, Fudan University, Shanghai 200032, PR China. <br /> 2.   Children&#226;&#8364;&#8482;s Hospital, Fudan University, Shanghai 200032, PR China. <br /> * These authors contributed equally work. <br />  <br />  <br /> &#239;&#8364;&#170; Corresponding author: Houyan Song, Department Biochemistry Molecular Biology, Shanghai Medical School Key Laboratory <br /> of Molecular Medicine, Ministry Education, Fudan University, Shanghai 200032, PR China. <br />  <br /> &#194;&#169; Ivyspring International Publisher. This open-access article distributed terms Creative Commons License (http://creativecommons.org/ <br /> licenses/by-nc-nd/3.0/). Reproduction permitted personal, noncommercial use, provided article whole, unmodified, properly cited. <br />  <br /> Received: 2012.07.17; Accepted: 2013.02.26; Published: 2013.03.06 <br />  <br />  <br />                 <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                 The hippo (Hpo) signaling pathway plays critical role regulation organ size. The kinase <br />                 cascade ultimately antagonizes transcriptional co-activator Yki/YAP, key regulator of <br />                 cell proliferation apoptosis. In study, performed knocking study using antisense <br />                 morpholino (MO) reagents zebrafish YAP, key transcriptional co-activator Hpo <br />                 pathway, plays critical role early embryonic development. At cellular level, yap inhibition <br />                 increases apoptosis decreases cell proliferation. Reduction yap function severely delays <br />                 developmental events, including gastrulation, cardiogenesis hematopoiesis. Knock- <br />                 yap showed evidence ventralization, including reduction dorsally expressed <br />                 marker goosecoid (gsc), expansion ventral marker gata2, disruption somites, reduction <br />                 head size. Finally, performed preliminary analysis real-time polymerase chain reaction <br />                 (qPCR) candidate targets zebrafish Hpo pathway. In conclusion, results revealed that <br />                 zebrafish yap coordinately regulates cell proliferation apoptosis required dorso- <br />                 ventral axis formation, gastrulation, cardiogenesis, hematopoiesis, somitogenesis. <br />                 Key words: Yes-Associated Protein, Yap, Zebrafish <br />  <br />  <br /> Introduction <br />       The Hpo pathway, novel signaling pathway                                (Sd) possible transcription factors regu- <br /> defined drosophila, extremely powerful                             late transcription cell- cycle cell-death reg- <br /> regulation growth suppression tumors                              ulators cyclin E diap 1 [5, 6]. Like dro- <br /> because ability simultaneously inhibit cell                           sophila studies, genetic cell culture studies in <br /> proliferation promote apoptosis [1-4]. Activation                           mammalian cells linked Hpo kinase <br /> of pathway induces kinase cascade                            cascade phosphorylation YAP/TAZ, two <br /> Ste20-type kinase Hpo forms complex                                  mammalian homologs Yki, (transcriptional <br /> WW-domain-containing adaptor protein Salvador                                   co-activator PDZ-binding motif; known as <br /> (Sav). This complex phosphorylates activates                           WWTR1), regulation growth [7-13]. <br /> the NDR family kinase Warts (Wts). After Wts binds                                    The ease creating genetic mosaics drosoph- <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> Mob family protein Mats, phosphorylates                           ila exactly circumvented hindrance early <br /> inactivates transcriptional co-activator Yorkie                             lethality following gene depletion Hpo signal <br /> (Yki), binds YEAD/TEF factor Scalloped                             transduction pathway [4]. However, efforts <br />  <br />  <br />  <br />                                                                                                                                    http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                   268 <br />  <br /> cannot lead complete understanding vivo        knockdown endogenous Yap protein West- <br /> function early embryonic development. Because         ern blotting (Fig. 1C). <br /> its semitransparent body, genetics ease ma-                Almost yap-MO-injected embryos showed a <br /> nipulation, zebrafish considered ideal verte-        <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> delayed development, defined by <br /> brate model study early development          comparison age-matched control embryos (Fig. <br /> cancer-related genes [14, 15]. Increasing         2A-D A&#226;&#8364;&#8482;-D&#226;&#8364;&#8482;). At 24 hpf, 51% abnormal <br /> evidence suggest Hpo pathway              yap-MO morphants (2-5 ng yap-MO embryo) <br /> evolutionarily conserved zebrafish [16-20]. Bioin-       showed moderate deviation morphological <br /> formatics analyses indicated considerable            norm (Fig. 1D-c, c&#226;&#8364;&#8482;, E), 26% mild morpho- <br /> amount evolutionary conservation                 logical abnormality (Fig. 1D-b, b&#226;&#8364;&#8482;, E), rest <br /> zebrafish species sequences, domains       23% showed severe morphological deviations (Fig. <br /> and motifs Hpo pathway components, in-           1D-d, d&#226;&#8364;&#8482;, E). Mild morphants indistinct heads <br /> cluding YAP/TAZ, LATS, Mats, Fat [16-22].               tails bent abdomen, moderate <br /> Mats1, regulates cell proliferation, apoptosis,       morphants short bodies curved trunks, and <br /> and growth, evolutionary conserved               severe morphants small trunks completely <br /> zebrafish drosophila [17]. Depletion zebrafish       indistinct bodies. Mortality calculated 48 hpf <br /> yap ameliorates changes pronephros develop-          (Fig. 1E). No death detected mild group <br /> ment caused Fat1 knockdown [18]. To elu-         (0/24). There 52% death rate moderate <br /> cidate effects components hippo sig-      group (25/48) 64% death rate severe <br /> naling pathway early embryonic development,           group (14/22). This effect be <br /> investigated role yap zebrafish embryo-       MO-concentration-dependent (Table 1). Previous <br /> genesis. We zebrafish yap regulates cell         work shown embryos reduced Yap ex- <br /> proliferation <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>survival</span> required      hibit marked increase cell death brain 24 hpf <br /> normal formation dorsoventral axis.                  [19]. In study, examined yap-MO-injected <br />                                                             embryos multiple time points 24 h <br /> Results                                                     development compared their <br /> Embryos depleted Yap exhibit phenotype                 wild-type siblings (Fig. 2). At 6 hpf, when <br />                                                             con-MO-injected wild-type siblings reached the <br /> of delayed general development <br />                                                             60%-epiboly stage, yap morphants de- <br />      To determine role yap zebrafish          veloped 50%-epiboly stage. The germ <br /> embryogenesis, translation-blocking morpholino            ring appeared thicker wild-type (Fig. 2A <br /> (yap-MO) splicing-blocking morpholino                 A&#226;&#8364;&#8482;). At 10 hpf. The epiboly siblings injected <br /> (yap-MO(S)) designed knock yap ex-             con-MO complete yap mor- <br /> pression (Fig. 1A). The efficacy specificity         phants showed 60-80% epiboly (Fig. 2B B&#226;&#8364;&#8482;). <br /> yap-MO recently confirmed [19]. We performed            By 18 hpf, yolk extension con-MO-injected sib- <br /> RT-PCR demonstrate activity yap-MO(S). Our        lings clearly delimited yolk ball the <br /> result showed yap mRNA splicing com-               tail straightened out, yap morphants were <br /> pletely disrupted embryos injected 20 ng         developmentally equivalent wild-type control <br /> the yap-MO(S). They lacked exon2           embryos 14 hpf (Fig. 2C C&#226;&#8364;&#8482;). At 24 hpf, <br /> additional 71 bp fragment intron2 (Fig. 1B). To          yap-MO-injected embryos displayed shortened body <br /> produce phenotype, used higher           axis notable opacity head (Fig. 2D D&#226;&#8364;&#8482;). <br /> dose yap-MO(S) yap-MO. We confirmed <br />  <br />  <br /> Table 1. Phenotypes observed subjects exposed different concentrations yap-MO. <br />                            n      Phenotype <br />                                   Normal       Mild           Moderate       Severe          Number deaths 48 hpf <br /> con-MO                     100    96 (96.0%)   4 (4.0%)       0 (0.0%)       0 (0.0%)        2 (2.00%) <br /> 0.25&#226;&#8364;&#8220;1 ng yap-MO           102    26 (25.5%)   55 (53.9%)     20 (19.6%)     1 (1%)          11 (10.78%) <br /> 2&#226;&#8364;&#8220;5 ng yap-MO              99     0 (0.0%)     23 (23.2%)     51 (51.5%)     25 (25.3%)      43 (43.43%) <br /> 10&#226;&#8364;&#8220;15 ng yap-MO            101    0 (0.0%)     2 (2.0%)       29 (28.7%)     70 (69.3%)      61 (60.39%) <br />  <br />  <br />  <br />  <br />                                                                                                   http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                    269 <br />  <br />  <br />  <br />  <br /> Fig 1. Analysis yap-MOs yap morphant phenotype. A) Structure zebrafish yap pre-mRNA (up) mature mRNA <br /> (down). The black lines represent introns, black rectangles represent exons, white rectangles represent untranslated regions <br /> (UTR). yap-MO(S) binds exon2-intron2 boundary yap-MO binds 5&#226;&#8364;&#178;UTR near ATG site. The blue rectangular dashed <br /> lines represent WW-domain, omitted. B) RT-PCR analysis yap-MO(S). RT-PCR performed pair of <br /> primers (green arrows (A), Table 1). C) Western blot analysis yap-MO yap-MO(S). Total proteins extracted 24 hpf <br /> zebrafish embryos. D) Bright-field images 24 hpf embryos injected con-MO (2-5 ng/embryo) (a a&#226;&#8364;&#178;) yap-MO <br /> (2&#226;&#8364;&#8220;5 ng/embryo) (b-d, b&#226;&#8364;&#178;-d&#226;&#8364;&#178;). At 24 hpf, yap morphants divided groups according phenotypes: wild (b b&#226;&#8364;&#178;), <br /> moderate (c c&#226;&#8364;&#178;) severe (d d&#226;&#8364;&#178;). D) The horizontal bars distribution phenotypes observed (C), vertical <br /> bars represent death rate group 48 hpf. <br />  <br />  <br />  <br />  <br />                                                                                                                   http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                            270 <br />  <br />  <br />  <br />  <br /> Fig 2. Phenotypes yap morphant rescued phenotype. A&#226;&#8364;&#8220;D) A&#226;&#8364;&#178;&#226;&#8364;&#8220;D&#226;&#8364;&#178;) Bright-field images yap morphants, of <br /> which exhibited developmental delay. A&#226;&#8364;&#178;&#226;&#8364;&#178;&#226;&#8364;&#8220;D&#226;&#8364;&#178;&#226;&#8364;&#178;) This phenotype partially rescued injection synthesized yap-mRNA. <br />  <br />  <br />                                                                   vertebrate embryos, TUNEL assay performed <br /> Both zyap1 zyap2 partially rescued the <br />                                                                   24 hpf (Fig. 4A-D A&#226;&#8364;&#8482;-D&#226;&#8364;&#8482;). Both the <br /> phenotype yap morphants <br />                                                                   control embryos yap morphant embryos showed <br />      During sequencing expression vector                TUNEL-positive signaling 6 10 hpf <br /> synthetic yap-mRNA, splicing variant              (Fig. 4A, A&#226;&#8364;&#8482;, B, B&#226;&#8364;&#8482;). This apoptotic <br /> encodes zebradfish Yap WW-domain.                  cell death rarely occurs 10 hpf. The <br /> Yap WW-domains, known preciously                      yap-MO-injected embryos clearly exhibited increased <br /> zebrafish Yap, designated Yap2 new             apoptosis 18 24 hpf, mainly head and <br /> Yap, WW-domain, desig-                 caudal parts (Fig. 4C, C&#226;&#8364;&#8482;, D, D&#226;&#8364;&#8482;). In way, <br /> nated Yap1. Yap1 differs Yap 2 lacks           knockdown yap result increased <br /> the WW domain encoded fourth exon. Yap2                    cell death embryonic development. <br /> be splicing variant (Supplementary Material: Fig.                     Then determined cell proliferation <br /> S1). The fourth exon downstream far away                yap morphant embryos defective. We used <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> target region yap-MO(S). This                  phosphohistone H3 (PH3) antibody staining mark <br /> yap-MO(S) disrupt normal splicing (Fig. 1A, B).               M-phase cells (Fig. 4E-H E&#226;&#8364;&#8482;-H&#226;&#8364;&#8482;). At earlier stages <br /> We injected zebrafish yap1 (zyap1) yap2 (zyap2)               (6, 10, 18 hpf), yap morphant embryos showed <br /> mRNA, lacking targeted 5&#226;&#8364;&#8482;-UTR, differ-              fewer mitotic cells control wild-type siblings <br /> ent individuals. The phenotypes resulting yap              (Fig. 4E,E&#226;&#8364;&#8482;, F, F&#226;&#8364;&#8482;, G, G&#226;&#8364;&#8482;). However, 24 hpf, no <br /> MO morphants partially rescued injec-                 significant difference PH3 staining found <br /> tion yap mRNA (Fig. 2A&#226;&#8364;&#8482;&#226;&#8364;&#8482;- D&#226;&#8364;?). We counted                  yap morphants control embryos (Fig. 4H <br /> number moderate severe embryo                 H&#226;&#8364;&#8482;). This consistent results flu- <br /> group 24 hpf obvious differences                  orescence-activated cell sorting (FACS) analysis (data <br /> between yap1 yap2 respect rescue (Fig. 3).            shown). <br /> Both <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>cell survival</span> proliferation defec-                   Disrupted dorsoventral patterning in <br /> tive yap morphant embryos                                      yap-depleted embryos <br />      Yki/YAP potent promoter cell survival                     To determine yap inhibition affects <br /> and proliferation drosophila variety cul-             markers dorsoventral patterning early <br /> tured mammalian cells [1-4, 7-12], inhibited            stages development, examined expression <br /> Hpo signaling. To determine knocking                 various ventral dorsal markers early <br /> of yap increases rate apoptosis developing              late gastrula stages. There obvious dif- <br />  <br />  <br />                                                                                                             http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                       271 <br />  <br /> ference spatial expression ventral marker                    E, E&#226;&#8364;&#8482;) presence foxA2 (Fig. 5F F&#226;&#8364;&#8482;) <br /> bmp2b yap morphant control embryos                         [24-26]. However, control expression profiles of <br /> (Fig. 5A A&#226;&#8364;&#8482;) [23], appeared                 gsc gata2 gastrulation showed reduced gsc <br /> clear overall delay development age-matched                  expression prechordal plate increased gata2 <br /> morphants relative control wild-type siblings.                  expression ventral mesoderm a <br /> Defective epiboly movement gastrulation                     simple delayed <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> (Fig. 5B, B&#226;&#8364;&#8482;, C, C&#226;&#8364;&#8482;) [27-29]. <br /> be direct cause delayed development                     In total, data suggested reduction yap <br /> similar spatial expression. The true                    disrupted dorsoventral marker expression during <br /> the absence dorsal markers tail (ntl) (Fig. 5D, D&#226;&#8364;&#8482;,              zebrafish gastrulation. <br />  <br />  <br />  <br />  <br /> Fig 3. No obvious differences zyap1 zyap2 respect rescue. The percentage abnormally developed <br /> embryos control embryos, yap morphants, yap-mRNA rescued morphants. The synthetic yap-mRNAs partially <br /> rescue yap-MO phenotype. <br />  <br />  <br />  <br />  <br /> Fig 4. Role yap regulation apoptosis proliferation 24 hpf. A&#226;&#8364;&#8220;D) A&#226;&#8364;&#178;&#226;&#8364;&#8220;D&#226;&#8364;&#178;) TUNEL staining indi- <br /> cates knockdown yap leads increased cell death head caudal parts. E&#226;&#8364;&#8220;H) E&#226;&#8364;&#178;&#226;&#8364;&#8220;H&#226;&#8364;&#178;) PH3 antibody staining suggests that <br /> yap inhibition results reduced cell proliferation. <br />  <br />  <br />                                                                                                                      http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                        272 <br />  <br />  <br />  <br />  <br /> Fig 5. Analysis dorsoventral markers yap morphants gastrulation. A&#226;&#8364;&#8220;F) A&#226;&#8364;&#178;&#226;&#8364;&#8220;F&#226;&#8364;&#178;) Whole-mount situ hybrid- <br /> ization 6&#226;&#8364;&#8220;9.5 hpf embryos injected 2&#226;&#8364;&#8220;5 ng con-MO (A-F) yap-MO (A&#226;&#8364;&#8482;-F&#226;&#8364;&#8482;) bmp2b (A A&#226;&#8364;&#178;), gata2 (B B&#226;&#8364;&#178;), gsc (C C&#226;&#8364;&#178;), <br /> ntl (D, D&#226;&#8364;&#178;, E E&#226;&#8364;&#178;), foxA2 (F F&#226;&#8364;&#178;). Embryos shown laterally ventral left (A&#226;&#8364;&#8220;C A&#226;&#8364;&#178;&#226;&#8364;&#8220;C&#226;&#8364;&#178;), dorsally anterior to <br /> the (D, D&#226;&#8364;&#178;, F F&#226;&#8364;&#178;), vegetally ventral (E E&#226;&#8364;&#178;). <br />  <br />  <br />  <br /> Effects zebrafish yap somitogenesis                               primordia control embryos trans- <br />                                                                         formed linear tube (Fig. 7A, A&#226;&#8364;&#8482;, B, B&#226;&#8364;&#8482;) [30, <br />       Both expression pattern yap somi- <br />                                                                         33]. However, observable difference in <br /> togenesis shortened body axis yap-depleted <br />                                                                         timing blood vessel development yap <br /> embryos indicate yap function critical so- <br />                                                                         morphants control embryos, assessed using the <br /> matic development [19]. For reason, examined <br />                                                                         vascular marker flk-1 (Fig.7C C&#226;&#8364;&#8482;) [34]. <br /> the expression somatic marker myoD [30] control <br />                                                                               Erythrocytes 30 hpf 52 hpf embyos were <br /> and yap morphants. In situ hybridization showed that <br />                                                                         identified o-dianisdine staining (Fig. 8I), which <br /> the level myoD transcription up-regulated <br />                                                                         presence hemoglobin erythrocytes was <br /> in yap-depleted embryos relative control embryos <br />                                                                         shown brown color staining [35]. yap-MO <br /> (Fig. 6C D). To confirm result, performed <br />                                                                         embryos showed red blood cells (RBC) 30 hpf, <br /> real-time PCR quantify myoD expression level at <br />                                                                         significantly recovered 52 hpf. Further <br /> 18hpf. We 2-fold increase myoD mRNA <br />                                                                         analysis yap morphant phenotype in <br /> level yap-morphants compared control (Fig. <br />                                                                         Tg(gata1:EGFP) fish showed clear delay hemato- <br /> 6I). Despite delayed onset somatic develop- <br />                                                                         poiesis 28 hpf (Fig.8II). Overall, data indicate <br /> ment, somitogenesis proceeded faster yap mor- <br />                                                                         yap-MO-injected embryos experience delayed <br /> phant embryos control, and <br />                                                                         cardiogenesis hematopoiesis. <br /> yap-depleted embryos control siblings reached <br /> the 18-somite stage time (Fig. 6A-D). titin                 RT-PCR real-time PCR analysis can- <br /> marks somatic border zebrafish embryos [31]. In                    didate target genes zebrafish Hpo sig- <br /> control embryos, dorsal ventral portions                     naling pathway <br /> each myotome converge point hori-                           In drosophila murine livers, genes <br /> zontal myoseptum forms, somites                       shown targets Hpo pathway. <br /> v-shape (Fig. 6E G). In contrast, somites                    These include cell-cycle regulator cyclin E (ccne) <br /> yap-MO-injected embryos appeared U-shaped (Fig. 6F                      cell-death regulator Diap1 [11.36]. To in- <br /> and H) [32]. These results indicate yap morphants                  vestigate mechanism zebrafish Hpo <br /> have higher expression level myoD.                                 pathway coordinately regulates proliferation and <br /> Delayed cardiogenesis hematopoiesis                              apoptosis, searched updated zebrafish pro- <br /> yap-MO-injected embryos                                                 teome     database     (http://blast.ncbi.nlm.nih.gov/ <br />                                                                         Blast.cgi?PROGRAM=blastp&amp;BLAST_PROGRAMS= <br />      Knockdown yap causes developmental delay <br />                                                                         blastp&amp;PAGE_TYPE=BlastSearch&amp;SHOW_DEFAUL <br /> of cardiogenesis hematopoiesis. In situ hybridi- <br />                                                                         TS=on&amp;LINK_LOC=blasthome) candidate target <br /> zation 24 hpf cardiac-specific markers nkx2.5 <br />                                                                         genes Hpo pathway. Based sequence similarity, <br /> and titin demonstrated heart primordia in <br />                                                                         selected candidate downstream targets Hpo <br /> yap-MO-injected embryo forms shallow cone with <br />                                                                         pathway validate using RT-PCR analysis, including <br /> its apex raised dorsally lumen, the <br />                                                                         ccne1, ccne2, ccnd1, birc2, xiap, ciapin. We found <br />  <br />  <br />                                                                                                                       http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                        273 <br />  <br /> that transcriptional level ccne2                             ity TAZ, shares sequence similarity YAP <br /> down-regulated yap-depleted embryos (Fig. 9I).                       modulates mesenchymal differentiation [13]. In <br /> Real-time PCR used confirm results                   addition, TAZ-deficient zebrafish embryos were <br /> of RT-PCR (Fig. 9II). In addition, Hpo shown                   defective bone formation, our <br /> to phosphorylate destabilized Diap1                         previous report phenotype yap-MO-injected <br /> non-transcriptional processes [37, 38]. This                     embryos [39]. For reason, determined expres- <br /> possible explanation lack marked changes                  sion level TAZ yap-morphants. We that <br /> transcriptional level possible regulators                level TAZ transcripts present in <br /> of cell death. In addition, undetected inhibitors                    yap-depleted embryos controls (Fig. 9I), ruling <br /> apoptosis involved pro-                      possibility yap-MO produced these <br /> cess.                                                                   RT-PCR results delayed phenotypes reducing <br />       In mammalian cells, Lats regulate activ-                  level TAZ. <br />  <br />  <br />  <br />  <br /> Fig 6. Effect zebrafish yap somitogenesis. A&#226;&#8364;&#8220;H) Whole-mount situ hybridization (A&#226;&#8364;&#8220;D) myoD (E&#226;&#8364;&#8220;H) titin 12, 18, <br /> and 24 hpf. A&#226;&#8364;&#8220;D) Dorsal view head left. E&#226;&#8364;&#8220;H) Lateral view head left. C, D, G, H) The insets (bottom left corner) <br /> show corresponding embryos. I) Real-time PCR result myoD expression. <br />  <br />  <br />  <br />  <br /> Fig 7. Delayed cardiogenesis normal vasculogenesis yap morphants. A&#226;&#8364;&#8220;C A&#226;&#8364;&#178;&#226;&#8364;&#8220;C&#226;&#8364;&#178;) Whole-mount situ hybridization <br /> with nkx2.5, titin, flk-1. A, A&#226;&#8364;&#178;, B, B&#226;&#8364;&#178;) Anterior view dorsal top. C C&#226;&#8364;&#178;) Lateral view head left. <br />  <br />  <br />                                                                                                                       http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                                 274 <br />  <br />  <br />  <br />  <br /> Fig 8. Delayed hematopoiesis yap morphants. I) O-dianisdine staining shows yap-MO embryos (B) red blood cells <br /> (RBC) 30 hpf, (D) significant recovery 52 hpf. II) (A&#226;&#8364;&#8220;J) Bright-field image (A&#226;&#8364;&#178;&#226;&#8364;&#8220;J&#226;&#8364;&#178;) fluorescent image live Tg(gata1:EGFP) <br /> embryos (A, A&#226;&#8364;&#178;, B, B&#226;&#8364;&#178;, E, E&#226;&#8364;&#178;, G, G&#226;&#8364;&#178;, I, I&#226;&#8364;&#178;) con-MO (C, C&#226;&#8364;&#178;, D, D&#226;&#8364;&#178;, F, F&#226;&#8364;&#178;, H, H&#226;&#8364;&#178;, J, J&#226;&#8364;&#178;) yap-MO. A&#226;&#8364;&#8220;J A&#226;&#8364;&#178;&#226;&#8364;&#8220;J&#226;&#8364;&#178;) Lateral view head <br /> to left, B, B&#226;&#8364;&#178;, D, D&#226;&#8364;&#178; (posterior view dorsal top). <br />  <br />  <br />  <br />  <br />                                                                                                                              http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                       275 <br />  <br />  <br />  <br />  <br /> Fig 9. RT-PCR realtime-PCR analysis candidate target genes zebrafish Hpo signaling pathway. I) For gene <br /> (ccne, ccne2, ccnd1, birc2, xiap, ciapin, taz), relative expression based RT-PCR shown. Total mRNA extracted 18 hpf <br /> yap-MO con-MO embryos, respectively. ef-1a used loading control. II) Real-time-PCR result ccne2, ccnd1, birc2, ciapin. <br />  <br />  <br />                                                                        ting. It likely RT-PCR sensitive <br /> Discussion                                                             enough, maternal Yap proteins <br />       Previous analysis sequence conserva-                  present stage. <br /> tion domains core components Hpo                            Yap gain function causes shortened body axis <br /> pathway facilitated in-depth investigations                    perturbed somatic head morphology both <br /> the role signaling pathway zebrafish                    zebrafish xenopus [20]. Because late mor- <br /> early embryogenesis [16-20]. Our previous works                        phologies similar yap knocking phe- <br /> have shown yap knockdown caused morpholog-                        notype, difficult determine the <br /> ical defects, including small heads, small eyes,                   rescue caused overexpression yap knocking <br /> less cartilage branchial arches [19]. In study,            yap. The ratio yap MO yap mRNA needs <br /> the difination increased cell apoptosis de-                     experimentation determined. <br /> creased cell proliferation direct explana-                        As previously reported, zebrafish Yap (Yap2), <br /> tion morphological defects. Otherwise,                     like mouse human YAPs, contains WW do- <br /> demonstrated yap-mediated signaling establish                     mains. We identified WW domain <br /> specific dorsoventral patterning zebrafish. Reduc-                  splicing variant zebrafish (Yap1). Zebrafish Yaps <br /> tion yap expression severely de-                  share significant sequence similarity hom- <br /> lay developmental events, gastrula-                    ologues chickens, mice, humans (Supple- <br /> tion, cardiogenesis, hematopoiesis.                                mentary Material: Fig. S2). This especially true of <br />       In study, tested efficacy                YAP2s humans, mice, zebrafish. Human <br /> translation-blocking      yap-MO      [19]                     YAP, WW-domains, potent <br /> splice-blocking yap-MO(S) (Fig.1B) RT-PCR                       transcriptional co-activator compared YAP <br /> Western blotting. Our results indicate MOs                   WW-domain [40]. In mice, YAPs <br /> are effective knock endogenous Yap                         WW-domains. The WW-domain has <br /> protein, albeit need use higher dose yap                pronounced affinity PPxY sequence, to <br /> MO(S) (20 ng). Although showed disrupted yap                      binds strongly second <br /> mRNA-splicing 20 ng yap MO(S) group                          WW-domain does [41]. Although functions of <br /> RT-PCR, small Yap protein                  zebrafish Yap1 WW domain Yap2 still <br /> present morphants shown western blot-                     unknown, determination precise differences <br />  <br />  <br />  <br />                                                                                                                      http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                               276 <br />  <br /> in signaling yap splicing variants         AATTTATA-3&#226;&#8364;&#178;) obtained Gene Tools, LLC <br /> important.                                                (U.S.). One- two-cell stage wildtype and <br />      During gastrulation, overall developmental de-       Tg(gata1:EGFP) zebrafish embryos injected with <br /> lay increases difficulty analyzing defective       MOs (2&#226;&#8364;&#8220;5 ng/embryo). <br /> dorsoventral patterns yap-MO-injected embryos, <br />                                                           Whole-mount TUNEL (terminal deoxynucle- <br /> but yap knockdown does reduce gsc expression and <br />                                                           otide transferase mediated dUTP nick-end <br /> increase gata2 expression. Confirmation gsc <br /> down-regulation showed yap-depleted embryos <br />                                                           labeling) staining <br /> had up-regulated levels myoD 18 hpf small           Embryos fixed 4% PFA overnight 4&#194;&#176;C, <br /> heads. This consistent previously described     rinsed PBS proceeding with <br /> pattern gsc-depleted xenopus embryos [42].             whole-mount TUNEL staining using situ Cell <br /> yap-depleted embryos develop ventralized        Death Detection Kit (Roche) according manu- <br /> morphology. During cell fate specification phase      facturer&#226;&#8364;&#8482;s instructions. <br /> of mouse trophectoderm (TE) development, Hpo <br />                                                           Whole-mount PH3 antibody staining <br /> signaling pathway components Lats Yap increase <br /> Tead4 activity, induces Cdx2 expression                Mitotic cells stained rabbit polyclo- <br /> overcoming Oct3/4-mediated suppression [43]. Ma-          nal anti phospho-histone H3 (Ser10) antibody (Cell <br /> ternal zebrafish Spg/Pou2/Oct4 controls for-          Signaling Technology, Inc.). This performed al- <br /> mation vertebrate dorsoventral axis        exactly described The Zebrafish Book a <br /> spg/pou2/oct4 mutant embryos develop dorsalized         dilution 1:1000 [46]. The secondary antibody rho- <br /> morphology [44]. This contrast ventralized   damine (TRITC)-conjugated goat anti-rabbit IgG <br /> yap morphants, raising possibility Yap re-    (Southern Biotech, U.S.) diluted 1:200. <br /> quired establish dorsoventral patterning   Whole-mount situ hybridization <br /> does overcoming Spg/Pou2/Oct4-mediated <br />                                                                Digoxigenin-labeled antisense RNA probes for <br /> suppression. Further elucidation mechanisms <br />                                                           zebrafish bmp2b, gsc, ntl, foxA2, gata2, myoD, titin, <br /> by Yap controls dorsoventral axis formation <br />                                                           nkx2.5, flk-1 synthesized described in <br /> are required complete understanding of <br />                                                           previously published studies. Whole-mount situ <br /> maternal control embryonic development. <br />                                                           hybridization assay performed described [48]. <br /> Conclusion                                                O-dianisidine staining globin <br />      In conclusion, zebrafish yap required           Embryos dechorionated fixed 4% <br /> early embryonic development. Reduction yap leads       PFA overnight 4&#194;&#176;C, rinsed PBS pro- <br /> to increased apoptosis, decreased proliferation, dis-     ceeding o-dianisdine staining. Fixed embryos <br /> rupted dorsoventral patterns somitogenesis,        incubated staining buffer 15 min the <br /> well delayed cardiogenesis hematopoiesis. Full     dark. The staining buffer consisted 0.6 mg/ml <br /> identification zYAP2 analysis yap       o-dianisidine (Sigma), 10 mM sodium acetate, 0.65% <br /> function essential understanding function   hydrogen peroxide, 40% (v/v) ethanol. <br /> the Hippo signaling pathway zebrafish embryonic <br /> development.                                              Reverse transcription-polymerase chain reac- <br />                                                           tion (RT-PCR) quantitative RT-PCR <br /> Materials methods                                           Total RNA extracted zebrafish em- <br /> Fish care                                                 bryos using Trizol reagent (Invitrogen) according <br />                                                           manufacturer&#226;&#8364;&#8482;s protocol. Complimentary DNA <br />      The maintenance, breeding, staging of <br />                                                           synthesis performed using standard protocol <br /> zebrafish lines (AB Shanghai) performed as <br />                                                           oligo-dT primers Moloney murine leukemia <br /> described previously [45, 46]. The Tg(gata1:EGFP) <br />                                                           virus reverse transcriptase (TaKaRa). <br /> zebrafish line used study described <br />                                                                 Primers (Table 2) designed using PRIMER <br /> elsewhere [47]. <br />                                                           3 software. PCR conditions follows: 95&#194;&#176;C 6 <br /> Morpholinos microinjection                            min; 30 cycles 94&#194;&#176;C 30 s; 53&#194;&#176;C 30 s; 72&#194;&#176;C <br />       Morpholino (MO) antisense oligonucleotides          45 s; 72&#194;&#176;C 10 min. The products were <br /> yap-MO (5&#226;&#8364;&#178;-CTCTTCTTTCTATCCAACAGAAACC-                     stored 4&#194;&#176;C. PCR products separated 1.5% <br /> 3&#226;&#8364;&#178;), yap-MO(S) (5&#226;&#8364;&#178;-GCAACATTAACAACTCACTTTA                 agarose gel. <br /> GGA-3&#226;&#8364;&#178;), con-MO (5&#226;&#8364;&#178;-CCTCTTACCTCAGTTAC                       Quantitative RT-PCR performed 7500 <br />                                                           (ABI) using SYBR Premix Ex Taq (TaKaRa). <br />  <br />  <br />                                                                                                http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                           277 <br />  <br /> Measurements performed quadruplicate                   col. Protein concentrations determined using the <br /> normalized expression levels Elf1-&#206;&#177;. Fold                Bradford assay (BioRad). The proteins re- <br /> changes values (X) calculated using follow-               solved SDS/polyacrylamide gels transferred <br /> ing formula: X=2-&#206;&#8221;&#206;&#8221;Ct. Bars indicate standard devia-               polyvinylidene fluoride membranes (Millipore). <br /> tions. The PCR cycling conditions follows: 40              The membranes probed following <br /> cycles pre-denaturation 95&#194;&#176;C 30 s, denatura-             primary antibodies: YAP1 (Proteintech) &#206;&#178;-actin <br /> tion 95&#194;&#176;C 5 s, annealing extension               (Cell Signaling Technology, Inc.). After wash- <br /> 60&#194;&#176;C 34 s.                                                     es, membranes incubated appro- <br />                                                                    priate horseradish peroxidase-conjugated secondary <br /> Protein extraction Western blot analysis <br />                                                                    antibodies (Proteintech). Blots developed using <br />      Total proteins extracted zebrafish                  ECL (Millipore). <br /> embryos using RIPA solutions (Cell Signaling Tech- <br /> nology, Inc.) according manufacturer&#226;&#8364;&#8482;s proto- <br />  <br />  <br /> Table 2. Primer sequences. <br />  <br /> Gene product          Gene bank ID          Upstream primers (5&#226;&#8364;&#178;&gt;3&#226;&#8364;&#178;)               Downstream primers (5&#226;&#8364;&#178;&gt;3&#226;&#8364;&#178;)             Product size (bp) <br /> ccne                  NM_130995.1           cgcagtgctgactgatgaat                   actcgagtccccatgacaac                   422 <br /> ccne2                 NM_001002075.1        cacgacaagagcttcatcca                   aggtttcctgcgagaactga                   439 <br /> ccnd1                 NM_131025.2           cccaccatctgaccaatacc                   gaaaaagcagggagcacttg                   410 <br /> birc2                 NM_194395.2           catcataggggaaggaagca                   ccttacaacacgcctggatt                   438 <br /> xiap                  NM_194396.2           atggcgaattggaatctgac                   atggcgaattggaatctgac                   408 <br /> ciapin                NM_001003738.1        tgcctccagctatgactgtg                   tgcatccaaaagagcatctg                   490 <br /> taz                   NM_001037696.1        aacagtgggccgtatcactc                   tgagagcacaaatccgacag                   415 <br /> ef-1a                 NM_131263.1           cttcgtcccaatttcaggat                   cagagactcgtggtgcatct                   300 <br /> myoD                  NM_131262.2           cgaggtcctgcgaattttag                   ctggggtccaagtcttcaaa                   273 <br /> yap-sp                NM_001139480.1        ggttgagaaagctgccagac                   tgggaaccttgctttactgg                   583 <br />  <br />  <br />  <br />                                                                    oversaw experiments helped write the <br /> Supplementary Material                                             manuscript. All authors read approved the <br /> Fig.S1 - Fig.S2. http://www.ijbs.com/v09p0267s1.pdf                final manuscript. <br />  <br /> Acknowledgements                                                   Competing Interests <br />       This work supported grants                        The authors declared competing <br /> National Natural Science Foundation China                    exists. <br /> Shuna Sun (30901472/H0204), National <br /> Natural Science Foundation China Yuexiang                    References <br /> Wang (30600489), Ph.D. Programs                       1.   Edgar BA. From cell structure transcription: Hippo forges new path. <br />                                                                         Cell. 2006; 124: 267- 273. <br /> Foundation Ministry Education China to <br />                                                                    2.   Harvey K, Tapon N. The Salvador-Warts-Hippo pathway - emerging <br /> Shuna Sun (200802461111). We like thank                        tumour-suppressor network. Nat Rev Cancer. 2007; 7: 182- 191. <br /> Prof. Tingxi Liu generous gift Tg (gata1:          3.   Pan D. Hippo signaling organ size control. Genes Dev. 2007; 21: 886- <br />                                                                         897. <br /> EGFP) zebrafish. We like thank Prof. Lei             4.   Saucedo LJ, Edgar BA. Filling Hippo pathway. Nat Rev Mol Cell <br /> Zhang, Drs. Jian Yang Dong Liu helpful                    Biol. 2007; 8: 613- 621. <br />                                                                    5.   Wu S, Liu Y, Zheng Y, et al. The TEAD/TEF family protein Scalloped <br /> advice, thank Dr. Hui Xu advice written                   mediates transcriptional output Hippo growth-regulatory <br /> English members Song Tang&#226;&#8364;&#8482;s                        pathway. Dev Cell. 2008; 14: 388- 398. <br />                                                                    6.   Zhang L, Ren F, Zhang Q, et al. The TEAD/TEF family transcription <br /> lab providing technical assistance.                                 factor Scalloped mediates Hippo signaling organ size control. Dev <br />                                                                         Cell. 2008; 14: 377- 387. <br /> Authers&#226;&#8364;&#8482; contributions                                             7.   Hao Y, Chun A, Cheung K, et al. Tumor suppressor LATS1 negative <br />                                                                         regulator oncogene YAP. J Biol Chem. 2008; 283: 5496- 5509. <br />     JyH, QJ, SnS conceived designed                    8.   Lee JH, Kim TS, Yang TH, et al. A crucial role WW45 developing <br /> experiments. JyH, QJ, SyS performed experi-                     epithelial tissues mouse. EMBO J. 2008; 27: 1231- 1242. <br /> ments. SnS, WW, YhG contributed reagents                   9.   Zhang J, Smolen GA, Haber DA. Negative regulation YAP LATS1 <br />                                                                         underscores evolutionary conservation Drosophila Hippo <br /> materials. JyH SnS wrote manuscript. HyS                        pathway. Cancer Res. 2008; 68: 2789- 2794. <br />  <br />  <br />  <br />  <br />                                                                                                                        http://www.ijbs.com <br />  Int. J. Biol. Sci. 2013, Vol. 9                                                                                                                        278 <br />  <br /> 10. Zhao B, Wei X, Li W, et al. Inactivation YAP oncoprotein Hippo    37. Harvey KF, Pfleger CM, Hariharan IK. The Drosophila Mst ortholog, <br />     pathway involved cell contact inhibition tissue growth control.       hippo, restricts growth cell proliferation promotes apoptosis. <br />     Genes Dev. 2007; 21: 2747- 2761.                                                Cell. 2003; 114: 457-467. <br /> 11. Dong J, Feldmann G, Huang J, et al. Elucidation universal              38. Pantalacci S, Tapon N, Leopold P. The Salvador partner Hippo promotes <br />     size-control mechanism Drosophila mammals. Cell. 2007; 130:              apoptosis cell-cycle exit Drosophila. Nat Cell Biol. 2003; 5: 921- <br />     1120- 1133.                                                                     927. <br /> 12. Camargo FD, Gokhale S, Johnnidis JB, et al. YAP1 increases organ size       39. Hong JH, Hwang ES, McManus MT, et al. TAZ, transcriptional <br />     expands undifferentiated progenitor cells. Curr Biol. 2007; 17: 2054-       modulator mesenchymal stem cell differentiation. Science. 2005; 309: <br />     2060.                                                                           1074- 1078. <br /> 13. Lei QY, Zhang H, Zhao B, et al. TAZ promotes cell proliferation         40. Komuro A, Nagai M, Navin NE, et al. WW domain-containing protein <br />     epithelial-mesenchymal transition inhibited hippo                 YAP associates ErbB-4 acts co-transcriptional activator for <br />     pathway. Mol Cell Biol. 2008; 28: 2426- 2436.                                   carboxyl-terminal fragment ErbB-4 translocates the <br /> 14. Amatruda JF, Shepard JL, Stern HM, et al. Zebrafish cancer model           nucleus. J Biol Chem. 2003; 278: 33334- 33341. <br />     system. Cancer Cell. 2002; 1: 229- 231.                        <br /> </body></html>