<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Schizosaccharomyces pombe cdc5+ gene encodes <br /> an essential protein with homology <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> c-Myb <br />  <br />  <br />  <br />  <br />                         and biochemical analyses of the cell cycle in a wide variety <br />                                      of eukaryotic organisms have shown that many of the genes <br />                                  important for the successful traverse of the cell cycle are <br />                                                 functionally conserved <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> yeast to humans. <br />                                                                       In particular, the product of the S.pombe cdc2+ gene, the <br />         Cdc2 protein kinase, has been the subject of intense <br />                              investigation during the last few years. The cdc2+ gene was <br />                                             identified through the first screen for cdc mutants in S.pombe <br />                                           (Nurse et al., 1976). The cdc2&apos;s strains are unique among <br /> The Schizosaccharomyces pombe cdc5+ gene was                       the other cdc mutants in that cdc2+ gene function is <br /> identified in the first screen for cell division cycle mutants     required both in GI prior to the beginning of DNA <br /> in this yeast. The cdc5+ gene was reported to be                   replication and in G2 for entry into mitosis (Nurse and <br /> required for nuclear division but because of its modest            Bissett, 1981). Mutants allelic to cdc2+ were also identified <br /> elongation and leaky nature at the non-permissive                  which underwent cell division at a reduced size (reviewed <br /> temperature, it was not investigated further. Here, we             in MacNeill and Nurse, 1989). These findings demonstrated <br /> report the characterization of the single allele of this gene,     that cdc2+ plays a central role in the regulation of the <br /> cdc5-120, in more detail. The mutant arrests with a 2N             S.pombe cell cycle. It has since been shown that cdc2+ <br /> DNA content and a single interphase nucleus. Further               homologs from diverse organisms are able to complement <br /> genetic analyses suggest that cdc5+ gene function is               S.ponmbe cdc2P mutants, and that cdc2+ function is essential <br /> essential in the G2 phase of the cell cycle. We have               for entry into mitosis in all eukaryotic cells (reviewed in <br /> cloned and sequenced the cdc5+ gene. The deduced                   Nurse, 1990). <br /> protein sequence predicts that Cdc5 is an 87 kDa protein              Two other cdc mutants identified in these original screens, <br /> and contains a region sharing significant homology with            cdcl3ts and cdc25ts, have also been studied in great detail. <br /> the DNA binding domain of the Myb family of                        Both are required for the G2/M transition. The cdcJ3+ gene <br /> transcription factors. Deletion mapping of the cdc5+               encodes a cyclin B homolog which is required for Cdc2 <br /> gene has shown that the N-terminal 232 amino acids of              function (reviewed in Hunt, 1991), and the cdc25+ gene <br /> the protein, which contain the Myb-related region, are             encodes a protein phosphatase which activates Cdc2 <br /> sufficient to complement the cdc5/ strain. A cdc5 null             (reviewed in Millar and Russell, 1991). Homologs of these <br /> mutant was generated by homologous recombination.                  genes have also been found in higher eukaryotes and detailed <br /> Haploid cells lacking cdc5+ are inviable, indicating that          biochemical studies have demonstrated that the interactions <br /> cdc5+ is an essential gene. A fusion protein consisting of         between Cdc2, cyclin B and Cdc25 are fundamentally similar <br /> bacterial glutathione S-transferase joined in-frame to the         in all eukaryotic organisms (reviewed in Solomon, 1993). <br /> N-terminal 127 amino acids of the Cdc5 protein is able                At least five other cdc genes (cdcl+, cdc5+, cdc6+, <br /> to bind to DNA cellulose at low salt concentrations. This          cdc27+ and cdc28+) are thought to be required for the <br /> evidence suggests that cdc5+ might encode a tran-                  G2/M transition. Mutants defective in these genes arrest <br /> scription factor whose activity is required for cell cycle         with a 2N content of DNA and have been proposed to arrest <br /> progression and growth during G2.                                  in G2 or late S phase since they do not exhibit the <br /> Key words: cdc5lcell cyclelc-MyblSchizosaccharomyces               characteristics of mitotic cells (Nurse et al., 1976; Nasmyth <br /> pombe                                                              and Nurse, 1981). The cdc27+ gene has been cloned and <br />                                                                    sequenced. The predicted Cdc27 protein shares no homology <br />                                                                    with any of the protein sequences found within the databases <br />                                                                    and its function is unknown (Hughes et al., 1992). The <br /> Introduction                                                        sequences of the cdcl +, cdc6+ and cdc28+ genes have not <br /> The fission yeast Schizosaccharomyces pombe has proved             been reported and the functions of their protein products are <br /> to be a useful organism for elucidating the mechanisms which        also unknown. In this report, we present a detailed <br /> govern  cell cycle progression in eukaryotic cells. A number        characterization of the cdc5ts mutant. In addition, we show <br /> of conditionally lethal cell division cycle (cdc) mutants were      that the predicted Cdc5 protein sequence contains a region <br /> isolated which arrest at a specific point in the cell cycle upon    which bears significant sequence identity to the DNA binding <br /> incubation at a restrictive temperature (Nurse et al., 1976;        domain found within Myb-related proteins. We demonstrate <br /> Nasmyth and Nurse, 1981). By studying these mutants, it             that cdc5+ is an essential gene whose function is required <br /> has been possible to identify and characterize at the molecular     during the G2 phase of the cell cycle and that the Myb-like <br /> level some of the genes and their respective protein products       DNA binding domain of Cdc5 exhibits affinity for DNA <br /> which constitute the cell cycle regulatory machinery. Genetic       when subjected to DNA-cellulose chromatography. <br />  <br />  <br />    Oxford University Press                                                                                                  471 <br />  R.Ohi et al. <br />  <br />  Results                                                                    The morphology of cdc5&apos;5 cells at the non-permissive <br />                                                                             temperature was also dissimilar to other well-characterized <br /> <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>Phenotype</span> of the cdc5- 120 mutant                                           cdc mutants in that cdc5ts cells incubated at the restricitve <br /> The S.pombe mutant strain, cdc5-120, was originally                         temperature for 5 h appeared wider than wild type cells <br /> described as being defective for nuclear division at a                      grown under the same conditions (Figure 1A). The observed <br /> restrictive temperature of 35&#194;&#176;C (Nurse et al., 1976). The                   increase in diameter was quantified and found to be small <br /> cdc5-120 mutant was reported not to arrest cell division                    but reproducible. Whereas wild type cells were 4.50 ,um in <br /> completely at 35 or 36&#194;&#176;C (Nurse et al., 1976; Nasmyth and                   diameter, cdcS&apos;s cells were 4.85 ltm in diameter after 5 h <br /> Nurse, 1981). We have also found that cdc5-120 is leaky                     at 37&#194;&#176;C. The increased diameter of the cdc5ts mutant <br /> at 35&#194;&#176;C (data not shown). However, when an exponentially                    coupled with the increased <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> predicts a 1.45-fold <br /> growing culture of cdc5-120 cells was shifted from 25 to                    increase in cell volume. <br /> 37&#194;&#176;C, cell division ceased 4.5 h following the shift to the                    When stained with the DNA binding dye DAPI, cdc5-120 <br /> non-permissive temperature in either rich medium or                         cells incubated at the restrictive temperature were observed <br /> minimal medium (Figure 2A and B). On solid medium, it                       to arrest with a single nucleus with decondensed chromo- <br /> should be noted that cdc5-120 is stably arrested at 36&#194;&#176;C as                 somes (Figure lA), suggesting that these cells do not arrest <br /> well as at 37&#194;&#176;C, although revertant colonies begin to form                  in mitosis. To define further the stage of the cell cycle in <br /> 3-4 days following incubation at 36&#194;&#176;C (data not shown).                     which the cdc5-120 mutant strain arrests, cdc5-120 cells were <br />     Observation of cdc5-120 cells incubated at 370C revealed                subjected to flow cytometric analysis. As a control, flow <br /> that they elongated only moderately when compared with                      cytometric analysis was also conducted on cdclO-129 cells <br /> cells grown at the permissive temperature (Figure lA). After                which arrest in GI with a IN content of DNA (Nurse et al., <br /> 5 h at 37&#194;&#176;C in rich medium, cdc5-120 cells elongated to                     1976). At the permissive temperature, cdclO-129 cells <br /> an average length of 18 ,um. Wild type cells grown under                    exhibit one peak of DNA corresponding to a 2N DNA <br /> the same conditions reached an average length of 14.5 /tm                   content (Figure iB). Exponentially growing S.pombe cells <br /> at the time of septation. Thus, cdc5-120 cells do not elongate              are predominantly in the G2 and M phases of the cell cycle <br /> to nearly the same extent as other well-characterized cdc                   and a DNA peak corresponding to a 1N DNA content is <br /> mutants such as cdc2ts (Nurse et al., 1976). Indeed, the lack               not observed (Sazer and Sherwood, 1990). After 2 and 4 <br /> of continued growth as cdc5-120 cells are held at the non-                  h at the non-permissive temperature, a culture of cdclO-129 <br /> permissive temperature is indicative of a cell growth defect.               cells begins to arrest in GI and a IN DNA peak was <br />  <br />  <br /> Fig. 1. Phenotypic characterization of the cdc5-120 mutant. (A) The cdc5-120 strain was grown to mid-exponential phase and shifted to 37&#194;&#176;C for <br /> 4 h. Samples of cells were taken at the permissive (25&#194;&#176;C) and restrictive (370C) temperatures, fixed and stained with DAPI. (B) Cells of the <br /> genotypes cdclO-129 and cdc5-120 were grown to mid-exponential phase and shifted to 37&#194;&#176;C. Samples of cells were taken at 0, 2 and 4 h following <br /> the temperature shift and fixed in ethanol. Cells were stained with propidium iodide and processed for flow cytometric analysis described in <br />                                                                                                                                  as <br />  <br /> Materials and methods. Linear fluorescence histograms show relative DNA content in arbitrary units on the horizontal axis and the cell number <br />                                                                                                                                           on <br />  <br /> the vertical axis. Arrows indicate the positions of 1N and 2N DNA peaks. <br />  <br /> 472 <br />                                                                                            S.pombe Cdc5 has homology to c-Myb <br />  <br /> observed (Figure iB). After 0, 2 and 4 h of incubation at         (Figure 4C) and cdc25-22 radl::ura4+ combinations <br /> the restrictive temperature, cdc5-120 cells exhibited only a      arrested at the restrictive temperature as elongated cells with <br /> single DNA peak consistent with a 2N content of DNA               a single nucleus, whereas cdc2l-M68 radl::ura4+ cells did <br /> (Figure 1B).                                                      not elongate substantially at 36&#194;&#176;C and appeared to enter <br />                                                                   mitosis (not shown). <br /> cdc5+ function is required after the hydroxyurea <br /> block point                                                       cdc5+ is not essential for growth in G1 <br /> To elucidate in which phase of the cell cycle cdc5+ gene          Wild type S.pombe cells in rich medium grow primarily <br /> activity is required, the execution point of cdc5-120 was         during G2 and undergo little growth in other phases of the <br /> sequenced with respect to the point in the cell cycle blocked     cell cycle (reviewed in Forsburg and Nurse, 1991). Because <br /> by hydroxyurea (HU), an inhibitor of DNA replication. Wild        cdc5-120 cells elongate only modestly at the restrictive <br /> type and cdc5-120 cells were arrested for 4 h with HU and         temperature (see Figure 1A) and block in G2 (see above), <br /> released into rich medium at the non-permissive temperature.      the possibility existed that cdc5+ gene function was <br /> To ensure that the function of the cdcSts allele was impaired     required primarily for growth during the cell cycle. The cell <br /> in cdc5-120 cells, both cultures were incubated at 37&apos;C           cycle arrest of cdcSts cells might then be a consequence of <br /> during the last hour of treatment with HU. Following              a general growth defect. To examine this possibility, we <br /> removal of the inhibitor, wild type cells began to grow           asked whether cdc5+ function was required for growth of <br /> exponentially at 37&#194;&#176;C. In contrast, cdc5-120 cells showed         cdcSts cells at a different stage of the cell cycle. Growth <br /> only a slight increase in cell number at the non-permissive       during the GI phase of the cell cycle becomes necessary <br /> temperature (Figure 2C). When released from a HU block            when cells re-enter the cell cycle from a Go state induced <br /> at 25&#194;&#176;C, cdc5ts cells continued exponential growth in             by nitrogen deprivation in order to achieve a critical cell size <br /> parallel with wild type cells (Figure 2D). These data suggest     for the GI/S transition (reviewed in Forsburg and Nurse, <br /> that cdc5-120 cells block with or after the HU block and          1991). <br /> before cell division.                                                Exponentially growing wild type, cdc5-120, cdcl1-123 <br />                                                                   and cdcIO-129 cells were incubated in medium lacking <br /> cdc5ts cells are not defective in DNA replication                 nitrogen for 20 h at 25&#194;&#176;C. The four cultures were shifted <br /> The previous experiments demonstrated that cdc5+ function         to 37&#194;&#176;C during the final hour of nitrogen starvation to <br /> is required after the HU block and that cdc5-120 cells            inactivate the temperature-sensitive cdc5-120, cdcll-123 and <br /> arrested at the restrictive temperature do not contain con-       cdclO-129 gene products. The cells were reinoculated into <br /> densed chromatin. These observations suggest that cdcS+           nitrogen-rich medium at 37&#194;&#176;C to allow the cells to grow and <br /> is involved in either late S phase or G2. It has been             re-enter the cell cycle. The cdcll-123 strain was chosen as <br /> previously demonstrated that a cdc17 (DNA ligase) radl            a control because it is defective only for cytokinesis and not <br /> double mutant displays a greatly increased lethality at the       for cell growth (Nurse et al., 1976). The cdclO- 129 mutant <br /> restrictive temperature compared with the cdc17 mutant            was chosen as a control because it becomes stably arrested <br /> alone (Al-Khodairy and Carr, 1992; Rowley et al., 1992).          in GI at the non-permissive temperature (Nurse et al., <br /> This occurs because the radl <span id='am-317' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-318' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-324' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-329' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-337' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-344' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-346' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-348' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-354' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-391' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> allows cells with           1976). As expected, flow cytometric analyses revealed that <br /> improperly replicated DNA to enter mitosis and die. In            all four strains arrested primarily in GI with a 1N content <br /> contrast, a double mutant between cdc25-22, which blocks          of DNA when starved for nitrogen (Figure 2E-H). <br /> after DNA replication in G2, and radl displays no increased       Following the readdition of nitrogen to the media, cdcl0-129 <br /> lethality at the restrictive temperature (Al-Khodairy and Carr,   cells remained with a IN DNA content, since cdcJO+ is <br /> 1992; Rowley et al., 1992). In order to determine whether         required for the GI/S transition (Figure 2F). Wild type, <br /> cdc5+ is essential for DNA replication, we constructed a          cdcll-123 and cdc5-120 cells progressed through G1 and <br /> cdc5-120 radl double mutant using a radl deletion allele          accumulated a 2N content of DNA with variable kinetics <br /> (Sunnerhagen et al., 1990). The viability of the cdc5-120         (Figure 2E, G and H). During the 6 h of refeeding, the <br /> radl::ura4+ strain was compared with cdc5-120 alone after         overall mass of the cdc5-120 culture increased in parallel <br /> incubation on plates at 36&#194;&#176;C for 0-9 h. As a positive             with wild type and cdcllts cells as measured by light scatter <br /> control, we assayed the viability of the cdc2l-M68 mutant         (data not shown). The average length of cdc5-120 cells also <br /> at the restrictive temperature when combined with the             increased in parallel with the other two strains until 6 h <br /> radl::ura4+ mutation. The cdc2l-M68 mutant was chosen             following nitrogen readdition (Figure 21). At this point, wild <br /> because it is thought to be defective in DNA replication          type cells had resumed exponential growth. These data <br /> although, like cdc5-120, it arrests with a 2N DNA content         indicate that Cdc5 is not critical for growth during the early <br /> (Coxon et al., 1992). Finally, a cdc25-22 radl::ura4+             stages of the cell cycle. Despite the parallel increase in cell <br /> strain was used as a negative control, because cdc25+ does        length and mass during nitrogen refeeding (Figure 21 and <br /> not play a role in S phase. As in previous experiments both       data not shown), cdc5-120 cells transited the cell cycle more <br /> the cdc25-22 and cdc25-22 radl::ura4+ mutant cells                slowly than wild type or cdcllts strains (Figure 2E, G and <br /> displayed little decrease in viability with time at 36&#194;&#176;C          H). Thus, while the cdc5-120 strain is not essential for <br /> (Figure 3A). However, the cdc2l-M68 radl::ura4+ cells             progression through GI, it does appear to be defective for <br /> lost viability much more rapidly than cdc2l-M68 cells alone       some GI-specific process(es) other than mass accumulation. <br /> (Figure 3B), which is consistent with cdc2l + being <br /> involved in S phase. Both cdc5-120 radl::ura4+ and                cdc5 cells arrest with aberrant morphology in the <br /> cdc5-120 cells showed little decrease in viability with time      presence of the cdc2-3w mutation <br /> at the restrictive temperature (Figure 3C), suggesting that       Cells carrying the cdc2-3w mutation are defective for size <br /> cdc5+ is not involved in DNA replication. This is                 control in the G2 period of the cell cycle and enter mitosis <br /> supported by the fact that both cdc5-120 radl::ura4+              at a reduced size as short cylindrical cells (Fantes, 1981). <br />  <br />  <br />  <br /> Fig. 2. Physiological analyses of the cdc5-120 mutant. (A) and (B) Experimental data corresponding to cells maintained at 25&#194;&#176;C (0) were plotted on <br /> an  arbitrary log scale. The real values per ml culture at 0 h are given within brackets next to the symbol key. (A) Exponentially growing cdc5-120 <br /> cells were incubated in yeast extract medium for 2 h at the permissive (25&apos;C) temperature and then shifted to the restrictive (37&apos;C) temperature for <br /> 9 h or kept at 25&apos;C for an additional 8 h. 0, culture maintained at 25&apos;C (1.6 x 106); 0, culture shifted to 37&#194;&#176;C. (B) Exponentially growing <br /> cdc5-120 cells were incubated in minimal medium for 2 h at 25&apos;C and then shifted to 37&apos;C for 9 h or kept at 25&#194;&#176;C for an additional 8 h. 0, <br /> culture maintained at 25&#194;&#176;C (6.5 x 105); 0, culture shifted to 37&apos;C. (C) Exponentially growing wild type and cdc5-120 cells were grown in yeast <br /> extract medium and treated with 12 mM HU for 4 h. During the last hour of incubation, the cells were placed at 37&#194;&#176;C. The cells were resuspended <br /> in inhibitor-free medium at time 0 and placed at 37&apos;C. 0, wild type; 0, cdc5-120. (D) Exponentially growing wild type and cdc5-120 cells were <br /> grown in yeast extract medium and treated with 12 mM HU for 4 h. The cells were resuspended in inhibitor-free medium and placed at 25&apos;C. An <br /> arrow indicates the time of HU removal. 0, wild type; 0, cdc5-120. (E-H) Cells were starved for nitrogen for 20 h and then reinoculated into <br /> yeast extract medium prewarmed to 37&#194;&#176;C. Samples were collected and fixed in ethanol at 0, 2, 4 and 6 h. Cells were stained with propidium iodide <br /> and subjected to flow cytometric analysis as described in Materials and methods. Linear fluorescence histograms show relative DNA content in <br /> arbitrary units on the horizontal axis and the cell number on the vertical axis. Arrows indicate the positions of IN and 2N DNA peaks. (E) wild <br /> type; (F) cdclO-129; (G) cdcll-123; (H) cdc5-120. (O) The lengths of wild type, cdcll-123 and cdc5-120 cells during the nitrogen starvation and <br /> refeeding experiment were determined hourly for 6 h. The vertical axis represents cell length in micrometers and the horizontal axis represents time <br /> in hours. 0, wild type; *, cdcll-123; V, cdc5-120. The asterisk indicates that only septated cells at this point were measured. <br />  <br /> 474 <br />                                                                                                  S.pombe Cdc5 has homology to c-Myb <br />  <br />        A                                                                 To address the effect of altering the normal size control over <br />                103                                                       entry into mitosis on the cdcSts phenotype, a double mutant <br />                                                                          between cdc5-120 and cdc2-3w was constructed. Cultures <br />                                                                          of cdc5-120 cdc2-3w cells contained cells of widely varying <br />                                                                          size and shape after 5 h at 37&#194;&#176;C, and were inviable <br />                                                                          (Figure 4D). Elongated cells as well as round or pear-shaped <br />                102 1                                                     cells were seen. This is in contrast to the phenotypes <br />            I                                                             of cdc5-120 (Figure 4B) and cdc5-120 radl::ura4+ <br />                                                                          (Figure 4C) cells which remained cylindrical and somewhat <br />                                                                          elongated at 37&#194;&#176;C. DAPI staining of cdc5-120 cdc2-3w cells <br />                                                                          arrested at 37&#194;&#176;C revealed that -30% of these cells had <br />                                                                          entered mitosis and failed to segregate correctly their <br />                                                                          chromatin. Many of these cells displayed a cut phenotype <br />                                                                          (Hirano et al., 1986; Samejima et al., 1993) with a single <br />                10 <br />                100                                                       nucleus bisected by a septum, or had a septum and only a <br />                            2             4         6         8           single displaced nucleus (Figure 4E). We believe this <br />                                 hInubation tkm at 36.0 (h)               phenotype to be numerically significant since similar <br />                                                                          abnormalities are found in only a small percentage of <br />                                                                          cdc5-120 or cdc5-120 radl::ura4+ cells at 37&#194;&#176;C. Cells <br />                                                                          carrying the cdc5-120 mutation and a disrupted weel gene <br />        B                                                                 displayed similar abnormalities at 37&#194;&#176;C to those seen in the <br />                                                                          cdc5-120 cdc2-3w strain (data not shown). <br />  <br />                                                                          Cloning of cdc5+ <br />                                                                          To gain insight into the possible function of the cdcS+ gene <br />                102                                                       product, the cdc5+ gene was cloned by complementation of <br />            I                                                             the cdc5-120 mutant and sequenced. A S.pombe genomic <br />                                                                          library contained within the pURl9 vector was transformed <br />                                                                          into cdc5-120 ura4-D06 h+ spheroplasts and uracil proto- <br />                10&apos;                                                       trophic transformants were selected at the restrictive <br />                                                                          temperature. From 55 000 Ura+ transformants, 110 Cdc+ <br />                                                                          Ura+ colonies were obtained. Plasmids were recovered <br />                                                                          from 60 of these transformants and restriction mapping and <br />                10      4                                                 Southern blot analyses indicated that they all contained the <br />                                                                          same gene. Five of these genomic clones (designated 1-5) <br />                             Incubaion thm at 36.0 (h)                    were analyzed in greater detail (see Figure 6) after <br />                                                                          confirming that they enabled cdc5-120 cells to form colonies <br />                                                                          at 36&#194;&#176;C upon retransformation. Experimental proof <br />                                                                          suggesting that these plasmids carried the cdc5+ gene rather <br />        C                                                                 than a high dosage suppressor of the cdcS!s mutant has been <br />                103                                                       detailed in a later section. DNA sequencing of the smallest <br />                                                                          genomic clone (#2) revealed discontinuous open reading <br />                                                                          frames (ORFs) and suggested the presence of introns within <br />                                                                          the protein coding region of the cdc5+ gene. To identify the <br />                10 2 <br />                                                                          protein coding region, a S.pombe cDNA expression library <br />                                                                          was screened using the ApaI-BstXI restriction fragment of <br />            I                                                             clone 2 as probe. Three overlapping cDNA clones of lengths <br />                                                                          2412, -1300 and -600 bp were isolated. The DNA <br />                                                                          sequences of the inserts contained witiin genomic clone 2 <br />                10 I                                                      and the three cDNA clones were determined on both strands. <br />                                                                            The nucleotide sequence of the 2412 bp cdc5+ cDNA <br />                                                                          clone and the predicted amino acid sequence are shown in <br />                                                                          Figure 5A. A comparison between the sequences of clone <br />                100                                                       2 and the longest cDNA clone revealed that the ORF of the <br />                             2            4         6         8           cdcS+ gene contained five introns (Figure 5A) of lengths <br />                                 ncubdBon tImskat 36. o(h)                40, 66, 63, 69 and 64 bp (data not shown). Each of these <br /> Fig. 3. Percent <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span> of cdc mutants in the presence of the radl      intron sequences contained consensus splice donor, splice <br /> null  mutation at 36&#194;&#176;C. cdc- radl::ura4+ double mutants are shown in     acceptor and branch point sequences known to be present <br /> filled circles. The cdc mutant alone is shown in open circles. (A) The   in S.pombe introns (data not shown; Russell, 1989). An in- <br /> percent survival of cdc25-22 radl::ura4+ and cdc25-22 cells plotted      frame stop codon was located 6 bp upstream of the putative <br /> against time of incubation at 360C. (B) The percent survival of          translation start site (Figure 5A). Additionally, no ORF was <br /> cdc2l-M68 radl::ura4+ and cdc2l-M68 cells plotted against time of <br /> incubation at 360C. (C) The percent survival of cdc5-120 radl::ura4+     identified within 600 bp of genomic sequence located <br /> and cdc5-120 cells plotted against time of incubation at 360C.           upstream of the putative translation start site (data not <br />                                                                                                                                  475 <br />   R.Ohi at al. <br />  <br />                                                                          with regions R2 and R3 of human c-myb. It shares 25, 33 <br />                                                                          and 30% identity with the c-Myb-related regions found in <br />                                                                          Saccharomyces cerevisiae transcription factors REB] and <br />                                                                          BAS], and Cl of Zea mays, respectively. The remaining 651 <br />                                                                          amino acids of the polypeptide share no significant homology <br />                                                                          with other proteins contained within the SwissProt database. <br />                                                                             To ascertain the genomic location of the cdc5+ gene, the <br />                                                                          full-length cDNA clone was hybridized to bacteriophage and <br />                                                                          cosmid clones covering the entire S.pombe genome (a gift <br />                                                                          from Dr E.Maier; Maier et al., 1992, 1993; Hoheisel et al., <br />                                                                          1993). The cdc5+ gene was found to be located within <br />                                                                          cosmid 14d8 on the NotI (H) fragment of chromosome 1 <br />                                                                          &lt; 30 kb away from the rhpSl gene (see Hoheisel et al., <br />                                                                          1993) (data not shown). <br />                                                                          Minimal complementing region of cdc5+ <br />                                                                          To define the minimal region of cdc5+ required to <br />                                                                          complement the cdc5ts strain, a series of fragments with 5&apos;- <br />                                                                          and 3&apos;-terminal deletions of the cdc5+ gene were tested for <br />                                                                          their ability to promote colony formation of the cdc5-120 <br />                                                                          strain at 36&#194;&#176;C (Figure 6). Deletion mutants at the 5&apos; end <br />                                                                          of the genomic clone were generated to determine whether <br />                                                                          or not specific sequences located upstream of the proposed <br />                                                                          coding region were required for complementation. The <br />                                                                          ability of these constructs to complement cdc5-120 suggests <br />                                                                          that sequences &gt; 186 bp upstream of the putative initiation <br />                                                                          ATG codon do not contain promoter or enhancer elements <br />                                                                          that are essential for the gene&apos;s expression. Deletion mutants <br />                                                                          at the 3&apos; end of the cdc5+ genomic clone showed that only <br />                                                                          the N-terminal 232 amino acids of the proposed protein are <br />                                                                          required for complementation of the cdc5-120 mutant strain. <br />                                                                         As expected, deletion constructs which removed both the <br /> Fig. 4. Phase contrast micrographs were taken of wild type (A),         putative promoter region and the c-myb-like DNA binding <br /> cdc5-120 (B), cdc5-120 radl::ura4+ (C) or cdc5-120 cdc2-3w (D)          domain abolished complementing activity (Figure 6). The <br /> cells incubated at 37&#194;&#176;C for 5 h. (E) cdc5-120 cdc2-3w cells were        minimal genomic cdc5+ fragment required to rescue <br /> incubated at 37&#194;&#176;C for 5 h and then stained with DAPI. Stars indicate <br /> cells with a single displaced nucleus and a septum. A cell with a cut   growth of cdc5-120 cells at the restrictive temperature on <br /> phenotype is indicated by a triangle.                                   solid medium was found to lie between the XmnI and PJlMI <br />                                                                         restriction sites (Figure 6). <br />  shown). These observations suggest that the proposed                      Despite the ability of most cdc5+ genomic fragments to <br />  translation start site for the cdc5+ gene is most likely               rescue the cdc5ts mutant on agar plates, there were <br />  coinincident with the initiating methionine depicted in                noticeable differences in the time and efficiency of colony <br />  Figure 5. Northern blot analysis of total yeast RNA                    formation following transformation. To quantitate some of <br />  hybridized with a probe synthesized from the longest cDNA              these differences, the doubling times of some of the <br />  clone revealed a single message of -2.5 kb (data not                   transformant strains in liquid medium at 37&#194;&#176;C were <br />  shown). These data suggest that the 2412 <span id='am-395' about='xsp:base' typeof='owl:Thing'>base</span> pair cDNA                measured. Transformant strains carrying genomic deletion <br /> clone is a nearly complete copy of the cdc5+ mRNA.                      mutants lacking both sequences upstream of the translation <br />    Translation of the putative coding region of cdc5+                   start site and in the C-terminal third of the predicted Cdc5 <br /> predicts a 757 amino acid polypeptide of molecular weight               polypeptide displayed longer generation times than strains <br /> 87 kDa. Comparison of the predicted protein sequence with               carrying full-length genomic clones (Figure 6). However, <br /> those contained within the SwissProt database revealed that             truncation of either 5&apos; untranslated sequences or up to one- <br /> the protein product of the isolated gene is most related to             third of the C-terminal protein coding sequences individually <br /> human c-Myb and the family of c-Myb-related proteins. The               did not significantly affect strain generation times (Figure 6). <br /> sequence similarity is restricted to the DNA binding N-ter-             Strains dependent on severly truncated forms of the gene <br /> minal region of c-Myb (Figure SB) which is characterized                displayed long generation times. Indeed, the cdc5-120 <br /> by three imperfect repeats (RI, R2, R3) of 51-52 amino                  transformant strain carrying the minimal rescuing fragment <br /> acids (reviewed in Liischer and Eisenmen, 1990). It is                  (XinnI-PflMI) showed extremely poor growth in liquid <br /> significant to note that cdc5+ encodes only two of the three            medium at the restrictive temperature (Figure 6). <br /> Myb repeats. They are located in the N-terminal domain of <br /> Cdc5 between amino acid residues 4 and 105. Figure SB                   Disruption of cdc5+ <br /> shows the similarities of the Myb-related region of Cdc5 with           To understand further the role of cdc5+ gene function, a <br /> the human c-myb protein and c-Myb-related proteins from                 <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>null allele</span> of the gene was constructed by the one-step gene <br /> a variety of organisms. The derived amino acid sequence <br /> of the c-Myb-related region of Cdc5 shares 33% identity <br />                                                                         disruption method. The entire 2.0 kb ApaI-BglII coding <br />                                                                         region fragment of cdc5+ was excised and replaced with <br />  <br /> Fig. 5. Nucleotide sequence of the cdcS+ gene and deduced primary structure of the encoded protein. (A) The nucleotide sequence is numbered <br /> starting with the first base of the ATG initiation codon. The ORF is interrupted by five introns in the gene whose positions are indicated with <br /> arrowheads. The amino acid sequence is in single letter <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> underneath the DNA sequence. An asterisk denotes the position of the stop codon. <br /> Consensus polyadenylation signals are underlined. (B) Amino acid sequence alignment of Myb-related repeats. The predicted amino acid sequence of <br /> the N-terminus of Cdc5 is compared with human c-Myb (Majello et al., 1986; Slamon et al., 1986), S.cerevisiae BASI (Tice-Baldwin et al., 1989), <br /> Zea mays Cl (Paz-Ares et al., 1987), Arabidopsis thaliana GLI (Oppenheimer et al., 1991) and S.cerevisiae REBI (Ju et al., 1990). The two <br /> repeating motifs of -50 amino acids are grouped. The characteristic tryptophans, critical cysteine and other amino acids which are identical in four <br /> or more proteins are in bold type. The numbers in parentheses represent residue numbers of Cdc5 which begin the line. <br />  <br />   <br />  <br /> Fig. 6. Restriction map and deletion analysis of the cdc5+ gene. The restriction map of the cdc5+ gene locus was compiled from five genomic <br /> clones. Relevant restricition enzyme cleavage sites are indicated. The protein coding region is boxed. The direction of transcription is indicated with <br /> an arrow. A &apos;+&apos; indicates the ability of the plasmids to complement the mutants on solid media. A         indicates that the clone did not support <br /> growth of the mutants. N.D., not done. Doubling times of cdc5SS transformants grown in liquid medium at 370C are listed in hours. <br />  <br /> the ura4+ gene (Figure 7A). The resulting cdc5::ura4+                               varied phenotypes. Flow cytometric analysis of germinated <br /> fragment was subcloned into the yeast expression vector                             spores indicated that the null mutant, like the cdcSls mutant, <br /> pIRT2 and was shown to be incapable of complementing                                arrested with a 2N DNA content (data not shown). <br /> the cdc5ts mutant (data not shown).                                                    The phenotype of a cdcS null mutant was also examined <br />    A uracil auxotrophic diploid strain was transformed with                         in a different manner. Cdc5 was expressed from a multicopy <br /> the cdc5::ura4+ fragment. Southern blot analysis of                                 plasmid in a haploid strain containing a deletion of the <br /> genomic DNA from 12 stable Ura+ integrants digested with                            genomic copy of cdcS+. These cells were grown in rich <br /> HindIll revealed that a single transformant appeared to be                          medium to allow for plasmid loss and the phenotypes were <br /> of the heterozygous genotype cdc5+/cdc5::ura4+. Genomic                             assessed by DAPI staining (Figure 7D). The majority of cells <br /> DNA from this single positive disruptant was subjected to                           which had lost the plasmid were elongated and contained <br /> further analysis to show unequivocally that one copy of the                         a single nucleus with no septum, although a small portion <br /> cdcS+ gene had been successfully replaced by the                                    of arrested cells did contain a single septum. These <br /> cdc5::ura4+ fragment. The cdcS+ gene is released on a                               phenotypes were very similar to those observed in <br /> single restriction fragment when S.pombe genomic DNA is                             germinating cdc5::ura4+ spores. <br /> digested with either BamHl or PstI. Since the restriction                              As described above, the full-length Cdc5 protein was able <br /> enzyme EcoRV cuts within the ura4+ coding region, but                               to rescue growth of the cdc5 null mutant (Figure 6) when <br /> not within BamHl or PstI fragments containing the cdc5+                             expressed from a multicopy plasmid. In contrast, C-tenrminal <br /> gene, it was used to show that the diploid strain was indeed                        truncated forms of the protein, which were able to rescue <br /> heterozygous for the cdc5 null allele (Figure 7B). The probe                        growth of the cdc5ts strain, were unable to rescue growth <br /> that was used in the Southern analyses corresponded to the                          of the cdc5 null mutant (Figure 6). <br /> Ndel-Xmnl restriction fragment located upstream of the                                 As proof that the cloned gene was in fact the cdc5+ gene <br /> cdcS+ coding region (Figure 7B).                                                    and not a suppressor, the diploid strain cdc5::ura4+I <br />    In order to visualize the phenotyrpe of the cdcS null mutant,                    cdcS-120 h+Ih- was constructed. This strain is viable at the <br /> the cdc5::ura4+/cdc5+ diploid was sporulated. When                                  permissive temperature on minimal medium lacking uracil, <br /> plated onto minimal medium lacking uracil, these spores                             but is Cdc- at 360C. A diploid strain of the genotype <br /> failed to form colonies, indicating that cdcS+ is an essential                      cdc5::ura4+Icdc5+, in contrast, is able to form colonies at <br /> gene. Haploid spores carrying the null mutation germinated,                         both 25 and 360C (Figure 8). This result is possible only <br /> but died as elongated cells. To obtain a more precise                               if the cloned gene is the wild type allele of cdc5+. <br /> characterization of the terminal phenotype displayed by the                         Therefore, we conclude that we have cloned the cdc5 + <br /> cdcS null mutant, spores were germinated in liquid medium                           gene. <br /> lacking uracil and observed microscopically. DAPI staining <br /> revealed that three phenotypes predominanted during the                             DNA binding activity of the cdc5+ gene product <br /> 20 h period in which the germination and growth of spores                           Since cdc5+ encodes a protein that shares significant <br /> was observed: (i) elongated cells with a single circular or                         homology to known DNA binding transcription factors, it <br /> half-moon shaped nucleus, (ii) elongated cells with a single                        was of interest to test whether the cdc5+ gene product had <br /> nucleus stretched along the length of the cell and (iii)                            DNA binding activity. The N-terminal domaln of Cdc5 was <br /> elongated cells with two nuclei and a single septum                                 fused in-frame with bacterial glutathione S-transferase (GST), <br /> (Figure 7C). A small percentage of cells displayed other                            and was subsequently purified from bacterial cells using <br /> 478 <br />  <br /> Fig. 7. Deletion of the cdc5+ gene. (A) Restriction map of the cdcS+ gene. Relevant restriction sites are indicated; Sau3A (S), PstI (P), HindI <br /> (H), NdeI (N), XmnI (X), ApaI (A), BamHI (B) and Bglll (Bg). The construction of pTZ19RcdcS::ura4 is shown below the map. The 2.0 kb <br /> ApaI-Bgml fragment was removed and replaced with the ura4 gene. The NdeI-XmnI fragment used to probe Southern blots in panel B is indicated <br /> above the map. (B) Southern blot analysis of chromosomal cdc5 deletion. Genomic DNAs from a wild type diploid and a heterozygous cdcS::ura4 <br /> diploid were digested with HindU, EcoRV and BamHI, or EcoRV and PstI, and then resolved on a 0.7% agarose gel. DNAs were blotted to <br /> GeneScreen plus and hybridized with the probe fragment indicated in panel A. Bands were visualized by autoradiography. Exposure time was 3 days <br /> at -70&#194;&#176;C. (C) Phenotype of the cdcS null mutant. Spores from the cdcS::ura4+/cdcS+ diploid were germinated in minimal medium lacking uracil. <br /> Samples were fixed in formaldehyde at 8-12 h (early germination period) and 15-20 h (late germination period), and stained with DAPI. The <br /> relative percentages of cells with each phenotype are given. (D) Phenotype of Cdc5 loss. A haploid cdcS deletion strain carrying a Cdc5-expressing <br /> multipcopy plasmid was grown to mid-exponential phase in non-selective medium to allow plasmid loss. Cells were fixed in formaldehyde and <br /> stained with DAPI. <br />                                                                                                                                                              479 <br />  <br />                                                                           Fig. 9. DNA binding activity of Cdc5. The first 127 amino acids of <br />                                                                           Cdc5 were fused in-frame to the bacterial GST enzyme. The fusion <br />                                                                           protein was expressed in bacteria and purified as described in <br />                                                                           Materials and methods. Ten jig of the fusion protein were loaded onto <br />                                                                           a DNA-cellulose column. The column was washed and proteins were <br />                                                                           eluted with a salt step gradient as described in Materials and methods. <br /> Fig. 8. Complementation of the cdc5 null allele by the cdc5-120 allele.   One-thirtieth of each flow through, wash and eluate sample was <br /> The heterozygous diploid strains, cdc5::ura4+1cdcS+ and                   resolved on an SDS-polyacrylamide gel along with 10 jig of the <br /> cdc5::ura4+1cdcS-120 were grown at 25&#194;&#176;C and then replica plated and       fusion protein, and then transferred to immobilon-P. The fusion protein <br /> incubated at the permissive (25&#194;&#176;C) and restrictive (36&apos;C) temperatures.   was detected with antibodies to GST followed by [125I]protein A, and <br />                                                                           the bands were visualized by autoradiography. The exposure time was <br />                                                                           16 h. Ten yg of the fusion protein were in the lane designated &apos;load&apos;. <br /> immobilized glutathione. The purified fusion protein was                  The asterisk depicts the lane in which the majority of the fusion <br /> subjected to chromatography on DNA-cellulose (Figure 9).                  protein was detected. <br /> At 40 mM NaCl, traces of the fusion protein were detectable <br /> in the column flow through and in the wash fractions.                     phase of the cell cycle cdc5+ gene function is required, the <br /> Subsequent washing of the column with 100 mM NaCl did                     viability of a cdc5-120 radl::ura4+ double mutant was <br /> not elute a detectable amount of Cdc5 fusion protein.                     determined. It has been shown previously that a mutant <br /> However, upon addition of 200 mM NaCl, the majority of                    defective in cdc17+, a DNA ligase whose function is <br /> the fusion protein was released from the column. Affinity-                involved in the completion of S phase, displays an increased <br /> purified GST alone showed no specific DNA binding                         lethality at the restrictive temperature when the radl null <br /> activity. It was detected primarily in the column flow through            mutation is present. Therefore, if cdc5+ gene function is <br /> and traces were also observed in the 40 mM wash fractions                 required during late S phase for completion of DNA <br /> (data not shown). From these data, we conclude that the N-                replication, the double mutant may have shown a decreased <br /> terminal 127 amino acids of Cdc5 are capable of binding                   viablity when grown at 25&#194;&#176;C following incubation at the <br /> DNA at low salt concentrations.                                           restrictive temperature when compared with the cdc5-120 <br />                                                                           single mutant alone. The results presented here suggest that <br /> Discussion                                                                cdc5+ executes its function following the completion of S <br />                                                                           phase as the cdc5-120 radl::ura4+ double mutant displays <br /> In this study, we have characterized the cdc5+ gene of                    no increase in lethality. <br /> S.pombe. Our results confirm the initial observation that the                It is of further interest to note that although both cdc5-120 <br /> cdc5-120 mutant is not completely arrested in the cell                    and cdc5-120 radl::ura4+ cells arrested at 37&#194;&#176;C as <br /> division cycle at 35 or 360C in liquid culture (Nurse et al.,             elongated cylindrical cells with a single nucleus, the presence <br /> 1976; Nasmyth and Nurse, 1981). However, we have found                    of the cdc2-3w mutation caused cdc5-120 cells to become <br /> that cdc5-120 cells block tightly when incubated in either                aberrantly shaped. Many of these cells displayed a round <br /> yeast extract or minimal medium at 37 &#194;&#176;C (Figure 1A and B).               or pear-shaped morphology. A significant percentage of <br /> This has allowed a further analysis of the cdc5(s phenotype               cdc5-120 cdc2-3w cells appeared to enter mitosis and were <br /> and the cdc5+ gene function.                                              unable to segregate their chromatin properly. These <br />    An asynchronous culture of cdc5-120 cells arrests with                 abnormalities are significant in light of the fact that cdc2-3w <br /> a 2N DNA content as measured by flow cytometry (this                      mutant cells are defective in both a size control checkpoint <br /> study), in agreement with previous results obtained by direct             as well as the checkpoint control for unreplicated DNA <br /> DNA measurements (Nurse et al., 1976). DAPI staining                      (Enoch and Nurse, 1990). Both of these checkpoints act in <br /> revealed that arrested cdc5ts cells do not contain condensed              the G2 phase of the cell cycle. Since cdc5-120 radl::ura4+ <br /> chromosomes, a feature characteristic of S.pombe cells as                 cells arrested with a similar phenotype to cdc5-120 cells, <br /> they enter metaphase. When subjected to treatment with HU,                it seems unlikely that the abnormalities seen in the cdc5-120 <br /> a well-known inhibitor of DNA replication, cdc5-120 cells,                cdc2-3w strain were due to the lack of a replication <br /> like wild type cells, accumulated in S phase with little                  checkpoint in these cells. This conclusion is supported by <br /> increase in cell number (data not shown). Upon concurrent                 the fact that wee] cells are defective for the size control but <br /> removal of the inhibitor and placement of the mutant cells                not the DNA replication checkpoint in G2 (Enoch and <br /> at the restrictive temperature, cell number measurements                  Nurse, 1990), and we found that cdc5-120 weel::ura4+ <br /> indicated that the cells did not proceed to divide. These data            cells displayed similar abnormalities to cdc5-120 cdc2-3w <br /> indicated that cdc5+ gene function is required either in late             cells at the restrictive temperature. A possible explanation <br /> S phase or in G2. To understand more accurately in which                  for these results is that inactivation of Cdc5 triggers a <br /> 480 <br />                                                                                                 S.pombe Cdc5 has homology to c-Myb <br />  <br /> checkpoint control mechanism that causes cells to arrest in        comprises up to three imperfect repeats of 50 amino acids <br />                                                                                                                          - <br />  <br />  <br />  <br /> G2. The cdc2-3w mutant may be defective for this                   each (reviewed in Luscher and Eisenman, 1990) which have <br /> checkpoint. Therefore, cdc5-120 cdc2-3w cells at 37&#194;&#176;C             been proposed to form a helix-turn-helix motif (Ogata <br /> attempt to progress through the cell cycle in the absence of       et al., 1992). Three tryptophan residues 18 or 19 amino acids <br /> cdc5+ function, and this leads to a variety of phenotypic          apart are characteristic of each repeat and are required for <br /> abnormalities.                                                     Myb DNA binding activity (Saikumar et al., 1990; <br />     A haploid cell containing a null allele of the cdc5+ gene      Frampton et al., 1991). The cdcS+ gene product contains <br /> was found to be inviable. Analysis of spores generated from       two such repeats, although a tyrosine is found in the position <br /> a diploid strain heterozygous for the cdcS+ null allele           where the sixth tryptophan residue is expected (see <br /> revealed that they germinated, but elongated and arrested         Figure SB). Other conserved residues in the Myb-related <br /> in the first cell cycle with three predominant phenotypes.        repeats are important for high affinity DNA binding activity <br /> Two-thirds of a population of cdcS <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>null mutants</span> elongated          (Saikumar et al., 1990; Frampton et al., 1991). In <br /> and contained a single nucleus, although a portion of these       particular, a cysteine residue has been shown to be critical <br /> cells contained chromatin that was dispersed and stretched        for both c-Myb and v-Myb DNA binding activity (Grasser <br /> along the length of the cell. The other third contained two       et al., 1992; Guehmann et al., 1992). The cdc5+ gene <br /> nuclei and a single septum. This result indicates that cdc5+      product contains all of these critical residues (see Figure 5). <br /> is an essential gene whose function is required for completion       The deletion mapping of cdc5+ presented in this study <br /> of the cell division cycle.                                       indicates that the C-terminal two-thirds of Cdc5 is not <br />    Results described previously (Nurse et al., 1976) and in       required for complementation of the cdcSts mutant. In <br /> this report raised the possibility that the cdc5-120 mutant       contrast, the entire cdc5+ protein coding region is required <br /> is defective for general growth processes at the restrictive      to complement the cdcS null mutant. A possible explanation <br /> temperature. Indeed, at 37&#194;&#176;C cdc5-120 cells elongate only         for this result is that the active form of Cdc5 is a homodimer. <br /> modestly and cell mass increases as well as cell cycle            This assumes that the protein product of the cdc5-120 allele <br /> progression stops (Nasmyth and Nurse, 1981; this study).          is impaired in its ability to bind DNA at 370C and that the <br /> Although a defect in cell mass accumulation is indicated,         C-terminal region of Cdc5 is involved in regulation. Thus, <br /> results presented here suggest that the primary defect of the     a dimer formed between the temperature-sensitive protein <br /> cdc5-120 strain lies in cell cycle progression. First, cdc5-120   product of the cdc5-120 allele and a Cdc5 molecule truncated <br /> cells arrest with 1.4 times the volume of wild type cells on      at the C-terminus would have both functional DNA binding <br /> the verge of septation. These cells would be predicted to         and regulatory domains. A more refined analysis of the Cdc5 <br /> contain sufficient material to form two daughter cells.           polypeptide will be needed to identify accurately functional <br /> Second, cdc5-120 cells blocked in S phase with HU do not          domains within the protein. <br /> divide when reinoculated into inhibitor-free medium at the           By analogy to the Myb family of proteins, it is possible <br /> restrictive temperature. Cells arrested at the HU block point     that cdc5+ encodes a transcription factor. To test this <br /> also become elongated. If cdc5+ function was required             hypothesis, we have shown that Cdc5 has affinity for DNA. <br /> primarily for growth, cdc5&apos;s cells would be expected to           A GST -Cdc5 fusion protein containing the putative DNA <br /> arrest in the second, not the first, cell cycle after release     binding domain of Cdc5 is retained by DNA-cellulose at <br /> from the HU block. Third, cdc5-120 cells subjected to             low salt concentrations. This evidence supports the notion <br /> nitrogen starvation followed by reinoculation into rich           that Cdc5 could be functioning as a sequence-specific DNA <br /> medium at the restrictive temperature increased in length         binding protein. Other transcription factors that have roles <br /> with the same kinetics as wild type cells. However, flow          in cell cycle progression have been isolated in the yeasts <br /> cytometric analysis of these cells showed that cell cycle         S.pombe and S. cerevisiae. The products of the fission yeast <br /> progression was slowed. This indicates that cdcSts cells          genes cdcJO+ (reviewed in Johnston and Lowndes, 1992) <br /> were not defective for growth during the early stages of the      and sctl +Iresl + (Tanaka et al., 1992; Caligiuri and Beach, <br /> cell cycle and that Cdc5 might be important, although not         1993) are transcription factors which act in concert during <br /> essential, for cell cycle progression through GI as well as       the G1 phase of the cell cycle. The genes SW74 and SW76 <br /> G2. Finally, spores containing the cdcS null mutation             of S.cerevisiae encode transcription factors that resemble <br /> germinated and elongated. If Cdc5 was crucial for cell            setl+lresl+ and cdclO+ and also act in GI (reviewed in <br /> growth, spores would not be expected to germinate or to           Johnston and Lowndes, 1992; Caligiuri and Beach, 1993). <br /> grow. Therefore, we propose that cdc5+ gene function is           To date, the product of the cdc5+ gene is the first <br /> required primarily for S.pombe cell cycle progression during      transcription factor reported to have a role in the G2 phase <br /> G2. A role for Cdc5 in additional cell processes is certainly     of the cell cycle and it will therefore be very interesting to <br /> possible, especially since Cdc5 is likely to function as a        determine the nature of target gene(s) for Cdc5. <br /> transcription factor (see below) which might have several <br /> target genes.                                                     Materials and methods <br />    To gain a better understanding of the possible function <br /> of the cdc5+ gene product, the cdc5+ gene and corres-             Strains, growth media and genetic methods <br />                                                                   The yeast strains used in this study are listed in Table I. Media used to <br /> ponding cDNAs were cloned and sequenced. The protein              grow Spombe cells and general genetic manipulations of Spombe have <br /> product predicted from the nucleotide sequence of cdcS+           been described (Moreno et al., 1991). Transformations were performed <br /> is 87 kDa in molecular mass and comprises 757 amino acids.        by the lithium acetate and spheroplast procedures (Moreno et al., 1991) <br /> Although the C-terminal 650 amino acids do not bear               or by electroporation (Prentice, 1991). <br /> significant sequence similarity to any proteins within the        Physiological analysis <br /> databases, the first 100 amino acids share striking identity      Samples for cell number determinations were collected at 30 min intervals, <br /> with the DNA binding domain of the Myb family of                  fixed in 0.12 M NaCl, 3% formaldehyde and cell number determined using <br /> transcription factors. This domain in Myb-related proteins        a Coulter Counter. The phase of the cell cycle in which the cdc5-120 mutant <br />                                                                                                                                        481 <br />  R.Ohi et al. <br />  <br />                                                                                  fragments of genomic clone 2 and the three cDNA clones were subcloned <br /> Table I. A list of yeast strains and their genotypes                             into pTZ19R and pTZ18R and single strands were generated. The single <br />                                                                                  strands were sequenced using Sequenase 2.0 (United States Biochemical, <br /> Genotype                                                       Source            Cleveland, OH) with either custom synthesized oligonucleotides (Operon, <br />                                                                                  Alameda, CA) or the M13 reverse primer. The predicted amino acid <br /> cdc5-120 h+                                                    P.Nurse           sequence was used to search the SwissProt protein database for homologous <br />                                                                                  sequences by utilizing the FASTDB program (Brutlag et al, 1990). Plasmids <br /> cdc5-120 ura4-D06 h+                                           P.Nurse           used in delineating the minimal rescuing fragment of cdcS+ were contructed <br /> cdc5-120 ura4-D06 leul-32 h+                                   P.Nurse           by digesting either clone 1 or 2 with the desired restriction enzymes (see <br /> cdc5-120 radl::ura4+ ura4-D06 h+                               This study        Figure 6) and ligating the resulting fragments into the yeast expression vector <br /> cdc5::ura4+1cdcS+ ade6-7041ade6-704                                              pIRT2 (Moreno et al., 1991). <br />   ura4-294/ura4-294 h+lh-                                      This study <br /> cdc25-22 ura4-D18 h-                                           P.Nurse           Gene disruption <br /> cdc25-22 ura4-D18 h+                                           P.Nurse           To create a cdcS+ genomic clone with sufficient flanking sequences at both <br /> cdc25-22 radl::ura4+ ura4-D18 h+                               This study        the 5&apos; and 3&apos; ends of the protein coding region to allow efficient homologous <br /> cdclO-129 h+                                                   P.Nurse           recombination to occur between the clone and the genomic copy of cdc5+, <br /> cdc2l-M68 h+                                                   P.Nurse           the inserts of two cdcS+ genomic clones (clones 1 and 4) were joined (see <br /> cdc2l-M68 radl::ura4+ h+                                       This study        Figures 6 and 7). The insert of clone 1 was excised as a 4.5 kb PstI-SacI <br />                                                                S.Subramani       fragment and subcloned into pTZ19R to form the plasmid pTZ19R53. This <br /> radl::ura4+ his3-237 leul-32 ura4-D18                                            plasmid was linearized with PstI, blunt-ended, digested with ApaI and the <br /> cdc2-3w leul-32                                                P.Nurse           vector fragment from this digest was purified. Clone 4 (see Figure 6) was <br /> weel::ura4+ ura4-D18 leul-32 h-                                P.Nurse           linearized with EcoRI, blunt-ended and cut with ApaL. The resulting 1.8 kb <br /> 972 h-                                                         P.Nurse           insert fragment was subsequently cloned into the prepared pTZ19R53 vector <br /> ade6-M210 leul-32 ura4-D18 h+                                  P.Nurse           to form the plasmid pTZ19R553. <br /> ade6-704/ade6-704 ura4-2941ura4-294 h+lh-                      P.Nurse              The DNA construct used for the cdcS+ gene deletion was produced by <br />                                                                                  replacing the 2.0 kb ApaI-Bgll cdc5+ protein coding region contained <br />                                                                                  within pTZ19R553 with a 1.8 kb HindEII fragment containing the <br /> arrests was sequenced with respect to the HU block point essentially as          S.pombe selectable marker, ura4+. The resulting plasmid, designated <br /> described by Nasmyth and Nurse (1981). The mutant strain cdc5-120 h+             pTZ19Rcdc5::ura4+, was cut with Sacl and the 6.1 kb cdc5::ura4+ insert <br /> was incubated in yeast extract medium containing 12 mM HU for 3 h at             was used to transform the diploid Spombe strain ade6-704/ade6-704 <br /> 25&#194;&#176;C. Under these conditions, cell division is no longer observed as HU          ura4-294/ura4-294 h+Ih+. Stable Ura+ integrants were isolated by replica <br /> inhibits DNA replication (Nasmyth and Nurse, 1981). The culture was then         plating to yeast extract agar five times and then back to selective conditions. <br /> shifted to 37&#194;&#176;C for 1 h to inactivate cdc5Ys gene function. The HU was           A heterozygous deletion mutant was identified by Southern blot analysis <br /> then removed from the culture by pelleting the cells and resuspending them       (described below). A sporulation competent diploid strain (cdcS::ura4+1 <br /> in fresh yeast extract medium that had been prewarmed to 37&#194;&#176;C. For nitrogen      cdc5+ ade6-704/ade6-704 ura4-294/ura4-294 h901h+) arose spontaneously <br /> starvation, cells were grown in minimal medium to a density of 4 x 106           from the parental h+/h+ disruptant strain, and was detected by replica <br /> cells/ml, washed three times in minimal medium lacking a nitrogen source,        plating colonies to sporulation medium. Colonies which contained spores <br /> and then inoculated into the nitrogen-deprived medium for 19 h at 25&#194;&#176;C            were identified by exposure to iodine vapors as previously described (Moreno <br /> and 1 h at 37&#194;&#176;C to inactivate temperature-sensitive mutant proteins. Cells       et al., 1991). <br /> were then pelleted and reinoculated into yeast extract medium that had been         To obtain viable haploid cells containing the cdc5 null mutant, the genomic <br /> prewarmed to 37&#194;&#176;C. Samples were collected and subjected to flow cytometric        cdc5+ clone in the plasmid, pIRT2, was transformed into the heterozygous <br /> analysis as described below.                                                     diploid deletion strain (cdcS::ura4+1cdc5+ ade6-704/ade6-704 leul-32/ <br />                                                                                   leul-32 ura4-294/ura4-294 h-7h+) and Leu+ transformants were selected. <br /> Flow cytometry and microscopy                                                     These diploids were sporulated and the desired Leu+ Ura+ haploid cells <br /> Cells for flow cytometric analysis were fixed in ice-cold 70% ethanol, treated    were isolated. <br /> with RNase A for 2 h at 37&#194;&#176;C in 50 mM sodium citrate and stained with <br /> 2 pig/ml propidium iodide for at least 1 h at 4&#194;&#176;C in the dark. These cells       Southern blot analysis <br /> were sonicated and subjected to flow cytometry as previously detailed (Sazer     Genomic DNA was prepared from diploid Spombe strains by procedures <br /> and Sherwood, 1990). To visualize nuclei, cells and spores were fixed with       detailed previously (Moreno et al., 1991). For Southern hybridization <br /> formaldehyde and stained with the fluorescent DNA binding dye DAPI as            analyses, 0.5 1tg of genomic DNA was digested overnight at 37&#194;&#176;C, size <br /> previously described (Moreno et al., 1991), and photographed using a Zeiss       fractionated on a 0.8% agarose gel and transferred to a Gene Screen Plus <br /> Axioscope photomicroscope and Kodak Ektachrome 400 film. To determine            membrane. The membrane was prehybridized for 1 h in hybridization buffer <br /> cell length and width, cells were measured microscopically in liquid medium      (5 x Denhardt&apos;s solution, 0.5% SDS, 5 x SSPE and 100 itg/ml hydrolyzed <br /> using phase optics and a x 100 objective with an eyepiece drum micrometer.       yeast RNA), and then incubated with a random primed [a-32P]dCTP- <br /> For cell length determinations, 16, 20 or 100 cells were measured for each       labelled probe for 16 h at 65&#194;&#176;C. Following hybridization, the filters were <br /> sample. For cell width determinations, 20 cells were measured for each           washed twice at 65&#194;&#176;C for 30 min in 0.1 x SSPE, 0.1% SDS at 650C. <br /> sample. The standard error of each mean was &lt;5%. <br /> Viability counts                                                                 Bacterial expression and purification of cdc5 protein <br /> The viability of cdc5-120 radl::ura4+; cdc2l-M68 radl::ura4+ and                 Oligonucleotide primers were used to amplify the 2.4 kb cdc5+ cDNA <br /> cdc25-22 radl::ura4+ strains was determined by plating -500 cells per            described above by PCR. Primer 1 (5&apos;-GTATGGATCCATGGTTGTTT- <br /> YE plate. These plates were placed at 36&#194;&#176;C for 0-9 h and then shifted            TA-3&apos;) was complementary to the 5&apos; end of the cDNA and primer 2 <br /> back to 25&#194;&#176;C for 72 h at which time, colonies were counted.                      (5&apos;-ACAAAGGA4TCCTGAAGTTTA-3&apos;) was complementary to the 3&apos; end <br />                                                                                  of the cDNA. Both primers were designed to add BamHI restriction sites <br /> Cloning and DNA sequence                                                         to the ends of the coding region during amplification. The amplified cDNAs <br /> Spheroplasts prepared from cdc5-120 ura4-D06 h+ cells were transformed           were blunt-ended and subcloned into pTZ19R linearized with SniaI to form <br /> with a S.pomnbe genomic DNA library in which fragments from a partial            the plasmid pTZ19R5cDNA. The insert of pTZ19R5cDNA was then <br /> Sau3A digest were inserted into pUR19 (a gift from Dr A.M.Carr; Barbet           sequenced on one strand using a single-stranded template as described above <br /> et al., 1992). Transformants were plated at the restrictive temperature on       to ensure that no mutations had been introduced by PCR. The BamHI-HpaI <br /> minimal plates lacking uracil to select for cells that could form colonies       fragment corresponding to the N-terminal third of Cdc5 was excised from <br /> under these conditions. Plasmid DNAs were recovered from Ura+ Cdc+               pTZ19R5cDNA and ligated downstream of the IPTG-inducible promoter <br /> transformants and were introduced into cdc5-120 ura4-D06 h+ spheroplasts         at the BamHI site in the bacterial expression vector pGEX2T (Smith and <br /> to confrmn rescue.                                                               Johnson, 1988). This contruct produced an inframe fusion protein with GST. <br />   A pDB20 based cDNA library (Fikes et al., 1990) was screened with              The Cdc5 fusion protein was expressed from the plasmid, designated <br /> the ApaI-BstXI fragment of genomic clone 2 (see Figure 6) to obtain              pGEX2T5AHpaI, in the Escherichia coli strain BL21DE3 and purified by <br /> cdcS+ cDNA clones. Two clones corresponded to the 5&apos; and 3&apos; halves of            affinity chromatography on immobilized glutathione as described (Smith <br /> the gene and overlapped with a tiird clone which was full-length. Restriction    and Johnson, 1988). <br />  <br />  <br />  <br /> DNA - cellulose chromatography and Western blotting                            <br /> The procedure for DNA -cellulose chromatography was modified from              <br /> Klempnauer and Sippel (1986). A total of 10 yg of purified               <br /> GST-CdcSAHpaI fusion protein in 0.5 ml of 10 mM HEPES (pH 8.0)                 <br /> and 1 mM EDTA were loaded onto a native calf thymus DNA-cellulose              <br /> column (bed volume was equal to - 1 ml; Sigma Chemical Co., St Louis,      <br /> MO). Equilibration of the column and adjustment of the flow rate were        <br /> as described (Klempnauer and Sippel, 1986). The column was then washed   <br /> with 4 ml of ice-cold equilibration buffer containing 10 mM HEPES (pH      <br /> 8.0) and 1 mM EDTA. Elution of protein bound to the column was               <br /> accomplished by sequentially adding two column volumes each of 100, 200,  <br /> 300, 400 and 500 mM NaCl in 10 mM HEPES (pH 8.0), 1 mM EDTA.              <br />    Samples of the flowthrough, wash and eluates were analyzed by             <br /> immunoblotting. A 15 Al aliquot of each fraction was boiled in 1 x SDS         <br /> gel sample buffer and resolved on a 1O0% SDS-polyacrylamide gel. Proteins    <br /> were then transferred to immobilon-P (Millipore, Bedford, MA) and             <br /> incubated with a 500-fold dilution of antibodies which recognize the GST        <br /> component of the Cdc5 fusion protein. These antibodies were produced       <br /> against a GST fusion protein in which bacterial GST was linked to a             <br /> vertebrate homeodomain-containing protein (a gift from Dr C.V.E.Wright).     <br /> Detection of immunocomplexes was accomplished with [1251]protein A and         <br /> results were visualized by autoradiography.                                  <br /> </body></html>