<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Single-cell time-lapse analysis of depletion of the <br /> universally conserved essential protein YgjD <br /> Tobias Bergmiller1*, Rafael Pe&#195;&#177;a-Miller2, Alexander Boehm3 and Martin Ackermann1 <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: The essential Escherichia coli gene ygjD belongs <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> a universally conserved group of genes whose <br />   function has been the focus of a number of recent studies. Here, we put ygjD under control of an inducible <br />   promoter, and used time-lapse microscopy and single cell analysis to investigate the phenotypic consequences of <br />   the depletion of YgjD protein <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> growing cells. <br />   Results: We show that loss of YgjD leads to a marked decrease in cell size and termination of cell division. The <br />   transition towards smaller size occurs in a controlled manner: cell elongation and cell division remain coupled, but <br />   cell size at division decreases. We also find evidence that depletion of YgjD leads to the synthesis of the <br />   intracellular signaling molecule (p)ppGpp, inducing a cellular reaction resembling the stringent response. <br />   Concomitant deletion of the relA and spoT genes - leading to a strain that is uncapable of synthesizing (p)ppGpp - <br />   abrogates the decrease in cell size, but does not prevent termination of cell division upon YgjD depletion. <br />   Conclusions: Depletion of YgjD protein from growing cells leads to a decrease in cell size that is contingent on <br />   (p)ppGpp, and to a termination of cell division. The combination of single-cell timelapse microscopy and statistical <br />   analysis can give detailed insights into the phenotypic consequences of the loss of essential genes, and can thus <br />   serve as a new tool to study the function of essential genes. <br />  <br />  <br /> Background                                                                            Saccharomyces cerevisiae and Escherichia coli, respec- <br /> Genes that are highly conserved between different types                               tively) is required to add a threonyl carbamoyl adeno- <br /> of organisms <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> for important biological functions                                  sine (t6A) modification to a subset of tranfer-RNAs that <br /> and are therefore usually well studied and described.                                 recognize codons with an adenin at the first position. <br /> One group of conserved genes whose function has                                       Transfer-RNAs undergo complex modifications and <br /> remained enigmatic until recently is the Kae1(OSGEP)/                                 maturation steps [9] required for translational fidelity <br /> YgjD family. Genes from this family occur in almost all                               [10-12]. Mutations in these modification pathways can <br /> bacterial, archaeal and eukaryotic genomes. The gene                                  be lethal or cause severe defects [13-15], and the <br /> family consists of two groups: one group, GCP1/                                       involved genes are highly conserved in different organ- <br /> OSGEPL/Qri7, is of bacterial origin, the other, GCP2/                                 isms [14-16]. <br /> OSGEP/Kae, is supposed to originate from archaea [1].                                   Because ygjD is essential, it is not possible to delete <br /> In Escherichia coli, Kae1/YgjD is essential for viability                             the gene and study the phenotypic consequences. As an <br /> [2,3]; in Arabidopsis thaliana and Saccharomyces cerevi-                              alternative, one can put the gene under control of an <br /> sia, deletion mutants exhibit deleterious phenotypes                                  inducible promoter, and investigate the consequence of <br /> [4-6].                                                                                turning off its expression, and thereby depleting the <br />   A biochemical activity for YgjD has recently been                                   YgjD protein. Our aim here is to get insights into the <br /> described: as already suggested by [7], Srinivasan and                                morphological changes that come about when the YgjD <br /> colleagues [8] showed that Kae1/YgjD protein (of                                      protein is depleted from growing Escherichia coli cells. <br />                                                                                       In two studies ([3] and [17]), the authors have noticed <br />                                an effect on cell size in YgjD depletion strains, suggest- <br />  <br />                  ing a role of YgjD for cell division and/or cellular elon- <br />                       gation. However, while Katz et al. observed shorter cells <br />  <br />  <br />  <br />  <br />  <br /> Table 1 List of strains and plasmids                                       We find that depletion of YgjD changes the balance <br /> Strain name    Relevant genotype                           Source       between cell growth and cell division, indicating a dis- <br /> DY330          W3110&#226;&#8211;&#161;lacU169 gal490 cI857 (cro-bioA)       [42]         turbance in cell size homeostasis. Experiments with <br /> MG1655         F- lambda- ilvG- rfb-50 rph-1               [43]         Escherichia coli and Salmonella thyphimurium have <br /> TB55           MG1655 araC-kan-yabI                        This study   shown a high degree of cell size homeostasis, or <br /> TB79           kan-araC-Para-ygjD                          This study   balanced growth [19]: under steady state conditions, <br /> TB80           frt::araC-Para-ygjD                         This study   cells have a constant cell size, indicating that the rate by <br /> TB82           frt::araC-Para-ygjD &#206;&#8221;relA::kan              This study   which cells elongate and the interdivision intervals are <br /> TB83           frt::araC-Para-ygjD &#206;&#8221;relA::frt              This study   coupled - cells that grow slower will initiate cell division <br /> TB84           frt::araC-Para-ygjD &#206;&#8221;relA::frt &#206;&#8221;spoT::kan   This study   later, and thus reach a goal cell size despite their slower <br /> FfH            kan-araC-Para-ffh                           This study   growth. Under conditions of YgjD depletion, cell elonga- <br /> DnaT           kan-araC-Para-dnaT                          This study   tion slowed down while the interval between cell divi- <br /> FldA           kan-araC-Para-fldA                          This study   sions remained constant. As a consequence, cell size <br /> AB1058         &#206;&#8221;spoT::kan &#206;&#8221;relA::frt                       This study   steadily decreased over consecutive divisions, until a <br /> pCP20          FLP+ l cI857+ l PR Repts AmpR CamR          [39]         minimal size was reached and cell division stopped. <br />                                                                         These cellular changes are specific: they differ from the <br />                                                                         consequences of the depletion of three other essential <br /> under YgjD depletion conditions, Handford et al.                        genes we analyzed, and of the exposure to two antibio- <br /> observed a mixed population of elongated and short                      tics that inhibit translation. <br /> cells. The reason for this discrepancy remained unclear                    Using a statistical approach, we show that this growth <br /> but could be based on the different genetic background                  transition occurs in a controlled manner: cell elongation <br /> of the E. coli strains (MC4100 versus MG1655). Altered                  rates and interdivision times were negatively correlated <br /> cell size upon YgjD depletion could be based on changes                 on the level of single cells, suggesting that these two <br /> in cell division timing or the cellular elongation rate, or             physiological parameters remain coupled during the <br /> on a combination of these two effects. To distinguish                   growth transition. Genetic experiments indicated that <br /> between these possibilities and to clarify the role of                  this change in cell size homeostasis involves production <br /> YgjD for cell size we used single cell resolution time-                 of the alarmone (p)ppGpp (guanosine-penta/tetra-phos- <br /> lapse microscopy of growing microcolonies.                              phate), a signaling compound that is a key player of a <br />   We constructed a conditional lethal ygjD mutant, and                  cellular response to amino acid starvation known as <br /> investigated the consequences of depletion of the YgjD                  stringent response. <br /> protein with high temporal resolution at the single-cell <br /> level. Similarly to ([3,6,17]) we put the expression of                 Results and Discussion <br /> ygjD under control of a promoter that is inducible by                   Our rationale here is that we can get insights into the <br /> the sugar L-arabinose. The resulting strain can be                      biological role of YgjD by following the cellular response <br /> grown normally in presence of L-arabinose, but ceases                   of its depletion on the single cell level and with high <br /> to grow in absence of L-arabinose and presence of glu-                  temporal resolution. We diluted cultures of the condi- <br /> cose. Then, single bacterial cells are placed on a nutri-               tional lethal Para-ygjD mutant TB80 onto pads of solid <br /> tious agar surface lacking the inducer and are observed                 LB medium that either contained L-arabinose (inducing <br /> with time lapse microscopy.                                             ygjD expression) or D-glucose (repressing ygjD expres- <br />   We used the cell tracking software &#226;&#8364;&#339;Schnitzcell&#226;&#8364;&#65533;[18] to               sion) and used time-lapse microscopy to follow single <br /> analyze images from the time-lapse microscopy experi-                   cells growing into microcolonies, taking an image every <br /> ments. This software identifies cells and tracks them                   2 or 4 minutes. The images were analyzed with the soft- <br /> across images from consecutive time points. It keeps                    ware &#226;&#8364;&#339;Schnitzcell&#226;&#8364;&#65533; [18]. The growth rate and cellular <br /> track of cell division events and of relatedness of cells               morphology of the P ara -ygjD strain grown in the pre- <br /> (e.g., it can relate each cell to the other cell that                   sence of L-arabinose was similar to the wild type grown <br /> emerged from the same division). The software also                      under the same conditions (Figure 1a and 1c, and Addi- <br /> extracts information about cell size and fluorescence                   tional file 1 - movie 1 and Additional file 2 - movie 2). <br /> intensity. The resulting dataset can be used to recon-                    A shift of the Para-ygjD strain to glucose lead to the <br /> struct the lineage of the clonal microcolony, and to plot               depletion of YgjD. This depletion is based on two <br /> phenotypic information like cell size and fluorescence                  effects. First, transcription of ygjD stops after the shift to <br /> intensity on this lineage. We used derivatives of these                 glucose. Residual L-arabinose that remains in the cells <br /> parameters (cell elongation rate and interval between                   from growth under permissive conditions is rapidly <br /> divisions) to describe and analyze the effects of YgjD                  metabolized. Lack of L-arabinose turns the transcrip- <br /> depletion.                                                              tional activator (AraC) of the P ara promoter into a <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                                                        Page 3 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br />                             A        TB80 arabinose                         B        TB80 glucose                      C              MG1655 <br />  <br />  <br />            100                                                    100                                            100 <br />  Number of cells <br />  <br />  <br />  <br />  <br />                    10                                             10                                             10 <br />  <br />  <br />  <br />  <br />                     1                                               1                                             1 <br />                         0       20   40   60   80   100 120 140         0       50    100    150    200    250         0   20    40    60   80   100     120 140 <br />  <br />                                      Time (minutes)                                  Time (minutes)                             Time (minutes) <br />  Figure 1 ygjD-expression determines patterns of growth. Each panel depicts data of cell numbers versus time from three independent <br />  experiments; each experiment is based on a microcolony that was initiated with a single cell, and followed over about six to seven divisions. A) <br />  TB80 (Para-ygjD) grown in presence of 0.1% L-arabinose. B). TB80 (Para-ygjD) grown in presence of 0.4% glucose. Note that the growth rate <br />  decreased after about 150 minutes. C) MG1655 (E. coli wild type) grown in LB medium with additional 0.4% glucose. Growth rates are similar to <br />  panel A, indicating that the induction of ygjD-expression in TB80 (panel A) lead to growth rates that are similar to wild type E. coli. <br />  <br />  <br /> transcription repressor. In addition, glucose metabolism                                    depletion, we separated data from different cell genera- <br /> causes depletion of the cellular co-inducer cyclic AMP.                                     tions of the depletion process. The first cell that is <br /> Together these effects lead to effective repression of                                      founding a microcolony is generation 0; this cell divides <br /> ygjD transcription in TB80. After termination of de novo                                    into two cells of generation 1, which divide into four <br /> ygjD mRNA synthesis the amount of YgjD in each cell                                         cells of generation 2, and so on (also see Additional File <br /> declines, because the mRNA and the protein are diluted                                      5 - Figure S2). To avoid comparisons between cells that <br /> through cell division, and degraded by cellular nucleases                                   are in different phases of their cell cycle, we only used <br /> and proteases, respectively [20]. The rapid cessation of                                    cell size measurements (and later fluorescence intensi- <br /> transcription from Para after the shift to glucose was evi-                                 ties) of cells immediately before division. Also, to avoid <br /> dent in control experiments with a strain that expressed                                    incomplete and biased sampling, we removed data from <br /> the green fluorescent protein (GFP) from the arabinose                                      above generation 6. <br /> promoter (Additional file 3 - Figure S1)                                                       This analysis revealed that the small size of cells <br />                                                                                             depleted for YgjD was a consequence of two effects: <br /> YgjD depletion leads to a change in cell size homeostasis                                   first, the rate of elongation (cell <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> increase over <br /> Time-lapse microscopy revealed that YgjD depletion                                          time) decreased (Figure 3a). Second, cells did not <br /> lead to a gradual phenotypic transition in division and                                     respond to the decrease in elongation rate by adjusting <br /> growth. Single cells that were transferred from permis-                                     the frequency at which they divided; the interval <br /> sive conditions to solid pads of LB medium with glucose                                     between two cell divisions remained initially constant. <br /> first continued to divide regularly, forming microcolo-                                     As a direct consequence, cell length at division <br /> nies in which the number of cells initially increased                                       decreased continuously (Figure 2a). <br /> exponentially. Then, after about four divisions, cell divi-                                    The <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span> induced by YgjD depletion was speci- <br /> sion slowed down and stopped (Figure 1b and Addi-                                           fic, and depletions of other essential genes lead to differ- <br /> tional file 4 - movie 3). Analysis of the time-lapse                                        ent cellular morphologies. We analyzed time-lapse <br /> images (Additional file 4 - movie 3) showed that, during                                    images of the depletion of three other essential genes <br /> this transition, cells size decreased (Figure 2a). This                                     (dnaT, fldA and ffh). Depletion of each protein resulted <br /> indicates a disturbance in of cell size homeostasis [19] -                                  in cellular phenotypes that were different from each <br /> that cells divide before their cell size doubled.                                           other and from YgjD when depleted (Additional file 6 - <br /> We used elongation rates of single cells and the time                                       Figure S3; also see Additional Files 7, 8 and 9 - movies <br /> interval between two divisions to analyze the change in                                     4, 5 and 6). Also, the effects of YgjD depletion were dif- <br /> cell size homeostasis during YgjD depletion. Since we                                       ferent from the consequences of exposure to two anti- <br /> were interested in how these parameters changed during                                      biotics that we tested: we followed wildtype E. coli cells <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                                                                        Page 4 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br />  A                                                                                        B <br />                                        15       Generations                                                                   15       Generations <br />   <br />  Figure 2 Depletion of YgjD lead to a change in cell size homeostasis. The figure is based on data from one microcolony of TB80 (Para-ygjD) <br />  depleted for YgjD (A; also see Additional File 4 - movie 3) and E. coli wildtype MG1655 (B; also see Additional File 2 - movie 2). Each point <br />  represents information about one cell, and the color of the point indicates which generation this cell belongs to (for a <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of &#226;&#8364;&#732;generation&#226;&#8364;&#8482; <br />  see main text). A) Changes in cell size during YgjD depletion. Cell size at division decreases continuously during the depletion experiment; B) <br />  Growth characteristics of MG1655 on single cell level. MG1655 exhibits a nearly constant cell size at division, and a slight increase of growth rate <br />  over consecutive divisions. <br />  <br />  <br /> exposed to the translational inhibitors kanamycin and                                  during the transition, the cell elongation rate and the <br /> chloramphenicol at minimum inhibitory concentration                                    timing of cell division would still be linked, but that this <br /> (2.5 &#206;&#188;g/ml for chloramphenicol, 5 &#206;&#188;g/ml for kanamycin),                                link would change quantitatively over time. In fact this <br /> and observed no decrease in cell size (Additional file 10                              is what we observed when we analyzed each generation <br /> - Figure S4, and Additional Files 11 and 12 - movies 7                                 of cells during the depletion process separately (inserts <br /> and 8).                                                                                Figure 3a and 3b). Within a given generation the time <br />   For reference, we also analyzed images of growing                                    interval between divisions and the rate by which a cell <br /> microcolonies of wildtype E. coli MG1655 cells on LB                                   elongated was negatively correlated: cells that grew fas- <br /> medium supplemented with glucose. This experiments                                     ter than the average of their generation tended to initi- <br /> confirmed cell size homeostasis, as expected from nor-                                 ate division more quickly; cells that grew more slowly <br /> mally growing cell: cells divided close to the moment                                  initiated division later. This suggests that cell growth <br /> when they had doubled their size, and small fluctuations                               and the timing of cell division are still linked within <br /> in cell elongation rates were compensated for by adjust-                               each generation in the depletion process, but that this <br /> ments in the time of cell division (Figure 2b and 3b).                                 link changes quantitatively over successive generations. <br />                                                                                           This analysis has, however, an important limitation: <br /> The transition towards smaller cell size is controlled                                 cells within a given generation are not independent <br /> What kind of disturbance of cell size homeostasis is                                   from each other. Some of these cells are more closely <br /> induced by depletion of YgjD? We considered two pos-                                   related, because they derive from the same mother or <br /> sibilities. First, it is possible that the control that couples                        grandmother. This can lead to spurious correlations <br /> cell division to cell size is lost, so that cells divide in an                         between traits; in our case, this effect could lead to arti- <br /> uncontrolled way, irrespective of their size. Second, it is                            ficial correlations between cell elongation rates and <br /> conceivable that cell division remains coupled to cell                                 interdivision intervals. This problem of relatedness in <br /> size, but the target size that a cell needs to reach before                            lineage trees is known from phylogenetic studies, where <br /> initiating division decreases over time.                                               it is referred to as phylogenetic dependence [21]. In the <br />   If the decrease in cell size is the result of a controlled                           context of phylogenetic studies, these dependencies can <br /> transition towards smaller cells, one would expect that,                               be resolved by analyzing differences between <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                                                                                                                               Page 5 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br />      A                                                                                                                    B <br />                                                                                      Generations        rho    p                                                                                              Generations      rho      p <br />                                                        70                                                                                                                     70 <br />                                                          Difference in cell elongation rate <br />   Figure 3 Cell elongation rate and the interval between two divisions are coupled during YgjD depletion. The contour line depicts all <br />   combinations of cell elongation rate and interval between divisions that correspond to a cell size doubling before division. Points below the <br />   contour line corresponds to cells that divide before they double in size, and whose size thus steadily declines. A) The relationship between the <br />   cell elongation rate and the interval between two divisions during YgjD depletion (Movie 2, additional files), and B) for MG1655 (Movie 3, <br />   additional files). For YgjD depletion, cell elongation rate starts to decrease from generation 3 on. However, this decrease in cell elongation rate <br />   is initially not compensated for by an increase in the interval between two divisions. Points below the contour line correspond to cells that <br />   divide before they double in size, and whose size thus steadily declines. The inset lists the result of a non-parametric correlation analysis <br />   between &#226;&#8364;&#732;cell elongation rate&#226;&#8364;&#8482; and &#226;&#8364;&#732;time to division&#226;&#8364;&#8482;, performed separately for every generation. A negative correlation indicates coupling of the <br />   interval between division and the cell elongation rate. For MG1655, the majority of cells cluster around the contour line. C) and D) show the <br />   result of the independent contrast correlation analysis for YgjD depletion in TB80, and MG1655 growth. Each point depicts the difference <br />   (residual) between two sister cells in the cell elongation rate (horizontal axis) and in the interval between cell divisions (vertical axis). Cells that <br />   have a higher elongation rate than their sister tend to have a shorter interval between divisions. The inset lists the result of a non-parametric <br />   correlation analysis between &#226;&#8364;&#732;difference in cell elongation rate&#226;&#8364;&#8482; and &#226;&#8364;&#732;difference in interval between two divisions&#226;&#8364;&#8482;, performed separately for every <br />   generation. Again, negative correlation indicates coupling of the interval between division and the cell elongation rate. <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                                                               Page 6 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br /> independent pairs of species, rather than calculating <br />                                                                          A <br /> correlations on the basis of the whole phylogenetic line-                                                                                Strain              Generations <br />                                                                                                           60                                   TB80 depleted      1 <br /> age [21].                                                                                                                                      TB84 depleted <br />                                                                                                                                                                   2 <br />                                                                                                                                                                   3 <br />    We used a variation of this approach to get an                                                                                                                 4 <br />  <br />  <br />  <br />  <br />                                                                     Interval between two divisions <br />                                                                                                           50                                                      5 <br /> unbiased view on the relationship between cell growth                                                                                                             6 <br /> and the timing of cell division: for each generation, we <br />                                                                                                           40 <br /> analyzed pairs of cells emerging from the same cell divi- <br /> sion, and calculated the difference in growth rates and <br /> in the time to division for each pair. We refer to two                                                    30 <br />  <br /> cells emerging from the same division as &#226;&#8364;&#732;sisters&#226;&#8364;&#8482; <br /> (thereby ignoring that these two cells have cell poles of                                                 20 <br />  <br /> different ages, [22,23]). The differences for all sister <br /> pairs represent independent data points, and we can use                                                   10 <br />  <br /> them to calculate the correlation between cell growth <br /> and time to division in an unbiased way.                                                                  0 <br />                                                                                                            0.00   0.01       0.02          0.03        0.04        0.05 <br />    The independent contrast analysis confirmed our ear- <br />                                                                   B                                                      Cell elongation rate <br /> lier conclusions: comparing each cell to its sister cell, <br /> we found that cells that grew slower than their sisters                                                                                   Strain              Generations <br />  <br />                                                                Mean cell size at division (micrometers) <br />                                                                                                           12 <br /> also displayed a longer interval between cell divisions,                                                                                        TB80 depleted      1 <br />                                                                                                                                                                    2 <br />                                                                                                                                                 TB84 depleted      3 <br /> while faster growing cells divided earlier. This manifests                                                10                                                          4 <br /> as a negative correlation between the difference in cell                                                                                                              5 <br />                                                                                                                                                                       6 <br /> elongation rate and the difference in interdivision inter-                                                 8 <br /> vals between two sisters (inserts Figure 3c and 3d; see <br /> also Additional File 13 - Figure S5). This is consistent                                                   6 <br /> with the interpretation that, during YgjD depletion, the <br /> timing of cell division remained coupled to a given cell                                                   4 <br /> size - and that the target cell size declined. <br />    The transition to decreased cell size is reminiscent of <br />                                                                                                            2 <br /> morphological changes that occur during the &#226;&#8364;&#732;stringent <br /> response&#226;&#8364;&#8482; [24,25], a stress adaptation program that is eli- <br />                                                                                                            0 <br /> cited when cells encounter amino-acid or carbon-starva-                                                           1      2           3            4     5         6 <br /> tion [26]. The stringent response is induced by                                                                                     Generations <br /> accumulation of the &#226;&#8364;&#732;alarmone&#226;&#8364;&#8482; guanosine tetra/penta            Figure 4 The change in cell size homeostasis in response to <br /> phosphate ((p)ppGpp), e.g. in response to low concentra-        YgjD depletion depends on (p)ppGpp. A) Changes in cell <br /> tions of amino-acylated tRNAs [26]. We thus wanted to           elongation rate and the interval between two divisions during YgjD <br />                                                                 depletion, for TB80 (ppGpp+) and TB84 (ppGpp0). For each strain, <br /> investigate this possible link to (p)ppGpp signaling more       means and standard errors of three independent experiments are <br /> closely, and asked whether the changes in cell homeostas-       shown. In TB80, cell elongation rate starts to decrease after <br /> tis upon YgjD depletion are mediated through (p)ppGpp.          generation 3, and cells divide before they double in size. In TB84, <br />                                                                 cell division occurs close to the moment of cell size doubling (the <br /> Changes in cell size homeostastis are mediated through          means are close to the contour line of constant cell size). B) <br />                                                                 Change of mean cell size during YgjD depletion, for and TB80 <br /> ppGpp                                                           (ppGpp+) and TB84 (ppGpp-). In TB80, cell size starts to decrease <br /> We constructed a strain, TB84, that is deficient in (p)         after generation 3, as a consequence of cell division that occurs <br /> ppGpp synthesis ((p)pGpp0), due to deletions of relA and        before cells double in size (see panel A). In TB84, cell size decreases <br /> spoT [26,27], and in which expression of ygjD was again         only slightly, and then increases after generation 5, when cells <br /> under control of Para. We followed growing microcolo-           divide after the moment of cell doubling (see panel A). <br /> nies of TB84 as described above and found that the con- <br /> sequences of YgjD depletion were profoundly different: <br /> cell elongation rate decreased during the YgjD depletion      final cell length of depleted (p)ppGpp0 cells (TB84) was <br /> process as for the relA+ spoT+ strain TB80 (Figure 4a). In    on average twice that of depleted (p)ppGpp+ cells (TB80) <br /> contrast to what we observed with this (p)ppGpp+ strain,      (Figure 4b). This is reminiscent of the elongated cells <br /> the decrease in elongation rate was compensated for by        found in populations of cells depleted for YgjD by Hand- <br /> an increase in the time interval between two divisions        ford and colleagues [3]. <br /> (Additional file 14 - movie 9, and Figure 4a). As a conse-      This suggests that the changes in cell size in response <br /> quence, cell size at division was not reduced, and the        to YgjD depletion are mediated through the alarmone <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                 Page 7 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br /> (p)ppGpp; an alternative explanation is that the absence        This mechanism prevents premature cell division before <br /> of (p)ppGpp leads to cell elongation (as has been pre-          chromosomes have been distributed to opposite cell <br /> viously reported [27]), and that this elongation compen-        halves. However, termination of cell division also mani- <br /> sates indirectly for reductive fission upon YgjD                fests in a (p)ppGpp0 strain (Additional File 17 - Figure <br /> depletion. Importantly, TB84 cells still ceased cell divi-      S8 c): depleted cells were elongated, and only a small <br /> sion (Additional file 15 - Figure S6). Thus, ygjD is still      fraction of the cell volume was filled with DNA. Thus, <br /> essential even in the absence of (p)ppGpp, and termina-         in the (p)ppGpp0 background, nucleoid occlusion alone <br /> tion of cell division is not solely a consequence of a          cannot be responsible for termination of cell division. <br /> diminished cellular growth rate.                                The elongated phenotype of YgjD depleted (p)ppGpp0 <br />    To further test the idea that ygjD depletion triggers (p)    cells resembles filamentous cells blocked in cell division. <br /> ppGpp synthesis we measured, on a single cell level dur-        However, since abrogating cell division is not inhibiting <br /> ing YgjD depletion, the activity of two promoters known         DNA replication or DNA segregation [32] it appears <br /> to respond to the intracellular level of (p)ppGpp: Papt is      unlikely that YgjD directly affects cell division. <br /> repressed by (p)ppGpp, while P rsd is induced by (p) <br /> ppGpp [28]. We transformed TB84 with plasmids carry-            Conclusions <br /> ing transcriptional promoter-gfp fusions [29] encoding          Our results show that single cell experiments coupled <br /> P apt -gfp and P rsd -gfp, and measured gene expression         with statistical analysis can uncover phenotypic transi- <br /> from these promoters as fluorescence intensity over             tions that come about when an essential gene is <br /> consecutive cell divisions. The level of GFP expression         depleted. We captured phenotypic changes with high <br /> steadily decreased in the strains where gfp was con-            temporal resolution across several cell generations. Cell <br /> trolled by Papt (Figure 5a), and steadily increased when        tracking techniques allowed us to build lineages of cells, <br /> controlled by Prsd (Figure 5c). Furthermore, this change        and to analyze correlations between phenotypic traits at <br /> in fluorescence was tightly linked to the rate by which         the level of sister cells emerging from the same division. <br /> cells elongated (Figure 5b and 5d). When the same               This information can be used to describe growth transi- <br /> strains were grown on L-arabinose containing medium             tions on the cellular level. <br /> no consistent changes of fluorescence could be observed           We found that YgjD depletion has two, possibly <br /> (Additional file 16 - Figure S7). These observations are        linked, effects: a decrease in cell size that is accompa- <br /> consistent with the scenario that YgjD depletion induces        nied by accumulation of (p)ppGpp, and the arrest of cell <br /> (p)ppGpp synthesis, and thus influences promoters               division. The involvement of (p)ppGpp in the alteration <br /> whose expression depends on the levels of (p)ppGpp.             of cell size homeostasis under YgjD depletion conditions <br />    Single cell analysis indicated that, in the cells depleted   might explain the discrepancies between two studies ([3] <br /> for YgjD, there is a link between decreased cell elonga-        and [17]) that observed opposite effects on cell size <br /> tion rate and (p)ppGpp levels. Using independent com-           upon YgjD depletion. Katz et al. [17] used a relA+ spoT+ <br /> parisons between sister cells in the microcolonies              strain that is very similar to the ppGpp + strain TB80 <br /> undergoing YjgD depletion, we found that if a cell had a        used here, and - consistent with our findings - observed <br /> lower elongation rate than its sister, it also tended to        shorter cells upon YgjD depletion. In contrast the <br /> have lower levels of GFP expressed from P apt (details          MC4100 derivative that was used by Handford and col- <br /> not shown; for Prsd-gfp, this <span id='am-167' about='xsp:pattern' typeof='owl:Thing'>pattern</span> was not observed).        leagues [3] carries a relA1 allele. This allele is known to <br /> These data support the idea that the link between (p)           cause reduced cellular (p)ppGpp levels under certain <br /> ppGpp levels and the cell elongation rate is direct; for        growth conditions [26,33]. Thus, their finding of elon- <br /> example, it is possible that high levels of (p)ppGpp            gated cells upon YgjD depletion might be similar to <br /> cause low elongation rates [30].                                what we observed with the ppGpp0 strain TB84. <br />    Our results suggest further that YgjD depletion has            Our observations seem consistent with the finding <br /> two (possibly linked) effects: first, depletion triggers (p)    that ygjD is involved in forming transfer-RNA <span id='am-164' about='xsp:base' typeof='owl:Thing'>base</span> <br /> ppGpp synthesis. Second, it leads to termination of cell        modifications [8]. Thus, depletion of YgjD protein leads <br /> division. To gain insights in which phase of the cell           to a pool of un- or undermodified transfer-RNAs (as <br /> cycle YgjD-depleted cells are arrested we visualized the        described by [8]), possibly resulting in non-optimal <br /> DNA-content of individual cells with DNA-staining and           interactions between transfer-RNAs and mRNA inside <br /> subsequent fluorescence microscopy (Additional File 17          the ribosome. This could potentially elicit a stringent- <br /> - Figure S8). After YgjD depletion in (p)ppGpp + cells          response like program (governed by (p)ppGpp release) <br /> (TB80), DNA was localized at midcell and filled large           and explain the phenotypic consequences of YgjD deple- <br /> areas of the cell (Additional File 17 - Figure S8 b), possi-    tion that we observed. Non-optimal interactions <br /> bly indicating that cells were unable to carry out addi-        between non-modified tRNAs and mRNA could be <br /> tional cell divisions due to &#226;&#8364;&#339;nucleoid occlusion&#226;&#8364;&#65533; [31].         similar to the effects caused by ribosomes that are <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                                                                                                                              Page 8 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br />       A                                                                                                                                  B <br />                                                                                                                    Generations                                                       Generations <br />                                          40                                                                                  1                                              40             1 <br />                                                                                                                              2                                                             2 <br />                                                                                                                              3                                                             3 <br />                                                                                                                              4                                                             4 <br />  Mean GFP fluorescence intensity <br />  <br />  <br />  <br />  <br />                                                                                                                                          Mean GFP fluorescence intensity <br />                                                                                                                              5                                                             5 <br />                                                                                                                              6                                                             6 <br />  <br />  <br />  <br />                                          30                                                                                                                                 30 <br />  <br />  <br />  <br />  <br />                                          20                                                                                                                                 20 <br />  <br />  <br />  <br />  <br />                                          10                                                                                                                                 10 <br />                                                                      0            50          100          150         200         250                                        0.00        0.01         0.02       0.03             0.04 <br />                                                                                            Time (minutes)                                                                                      Cell elongation rate <br />                                      C                                                                                                     D <br />                                                                                        Generations                                                                                                                       Generations <br />                                    Mean GFP fluorescence intensity <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                1 <br />                                                                                                                                           Mean GFP fluorescence intensity <br />  <br />  <br />  <br />  <br />                                                                                               1                                                                                                                                2 <br />                                                                      100                      2                                                                             100                                                3 <br />                                                                                               3                                                                                                                                4 <br />                                                                                               4                                                                                                                                5 <br />                                                                                               5                                                                                                                                6 <br />                                                                                               6 <br />  <br />  <br />                                                                          80                                                                                                 80 <br />  <br />  <br />  <br />  <br />                                                                          60                                                                                                 60 <br />  <br />  <br />  <br />  <br />                                                                          40                                                                                                 40 <br />                                                                               0          50          100         150             200                                          0.00         0.01         0.02          0.03           0.04 <br />                                                                                                                                                                                                    Cell elongation rate <br />                                                                                                      Time (minutes) <br />  Figure 5 Expression of Papt and Prsd during YgjD depletion. Single cell measurements of cell elongation rate and GFP fluorescence of two <br />  strains with transcriptional reporters for Papt (A and B) and Prsd (C and D). Each point represents a measurement for a single cell. In both strains, <br />  cell elongation rate decreased with increasing generations during YgjD depletion as shown in Figures 1B and 2A. A) and B) Papt is repressed by <br />  (p)ppGpp; its expression decreases during YgjD depletion, and decreases steadily with decreasing cell elongation rate. C) and D) Prsd is induced <br />  by (p)ppGpp; its expression increases during YgjD depletion, and steadily increases with decreasing cell elongation rate. <br />  <br />  <br /> stalled on &#226;&#8364;&#339;hungry&#226;&#8364;&#65533; codons: these codons are unsuccess-                                                                                        over time and reach a level where several cellular pro- <br /> fully trying to pair with either rare transfer-RNAs or                                                                                         cesses might be affected simultaneously. <br /> transfer-RNAs that are non-aminoacylated due to                                                                                                  Although the biochemical activity of YgjD has been <br /> amino-acid limitation. Hungry codons can provoke the                                                                                           described [8], the cellular functions of YgjD are not <br /> production of aberrant proteins by frame shifts, slides of                                                                                     completely resolved. It will be interesting to ask how the <br /> the translational machinery or incorporation of noncog-                                                                                        proteins in the YgjD/YeaZ/YjeE complex [3] of Escheri- <br /> nate transfer-RNAs [34,35]. This might also explain the                                                                                        chia coli are interacting to fulfill their functions, and to <br /> slow onset of the consequences of YgjD depletion: accu-                                                                                        ask whether YgjD is involved in other cellular processes <br /> mulation of aberrant proteins would slowly increase                                                                                            or responding to environmental cues. Single-cell <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                               Page 9 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br /> observations of YgjD depletion experiments might be              ygjD_insert3&#226;&#8364;&#8482; GCCGGTTTCATCGCAGGAAGTTT- <br /> helpful to generate and test hypotheses about the essen-      CAATACCCAGTACACGCATCGTTTCACTCCAT- <br /> tial role of this protein, and to help explain why it is so   CCAAAAAA <br /> widely conserved.                                                dnaT_insert5&#226;&#8364;&#8482; TCCGTGTGTTACTATAAAAGTTA- <br />                                                               TCTCCCTTCTCGTTCATCGAATGTGCC <br /> Methods                                                          TGTCAAATGGACG <br /> Bacterial strains and growth medium                              dnaTC_insert3&#226;&#8364;&#8482; GTCAATACCAACGACGTCCGGG- <br /> P1 transduction and TSS transformation were per-              GTCAAAACTCTGGAAGACATCGTTTCAC <br /> formed as described elsewhere [36,37]. Strain DY330 as           TCCATCCAAAAAA <br /> well as strains harboring the plasmid pCP20 [38] were            ffh_insert5&#226;&#8364;&#8482; <br /> grown at 32&#194;&#176;. All other strains were grown at 37&#194;&#176;. To            GACGCCTTCATGTTATACTGCGGCAAAATAC- <br /> grow TB80 and TB84 under permissive conditions, we            TGATGATGTGTAATGTGCC <br /> used LB medium (Sigma) supplemented with 0.1%                    TGTCAAATGGACG <br /> (batch culture) or 0.01% (before time-lapse microscopy)          ffh_insert3&#226;&#8364;&#8482; GCGCAGCGTGCGCGACAAACGATC- <br /> L-arabinose (Sigma). LB agar (1.5% agar) was from             GGTTAAATTATCAAACATCGTTTCAC <br /> Sigma, and used for preparing agar plates and agar pads          TCCATCCAAAAAA <br /> for time-lapse microscopy.                                       fldA_insert5&#226;&#8364;&#8482; TGCCTTTATCCGTGGGCAATTTT- <br />                                                               CCACCCCCATTTCAATAAGAATGTGCC <br /> Strain construction                                              TGTCAAATGGACG <br /> Strains containing more than one <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>knockout</span> or marker              fldA_insert3&#226;&#8364;&#8482; <br /> were generated by sequential P1-transductions. Resis-            ATTACCGGTGTCGCTGCCGAAAAAGATGCCA- <br /> tance markers were removed by Flp recombinase                 GTGATAGCCATCGTTTCAC <br /> mediated site-specific recombination [39]. To control            TCCATCCAAAAAA <br /> expression of ygjD, we constructed a conditional mutant          DY330 cells were grown in LB medium supplemented <br /> with a second copy of the promoter of the araBAD              with 0.2% L-arabinose and made electro- and recombina- <br /> operon in front of the native chromosomal locus of ygjD       tion competent [42], and electroporated with the above <br /> by directly inserting a Para-construct in front of ygjD, as   described PCR product. After electroporation, cells were <br /> described previously [40]. Removal of L-arabinose and         transferred to LB medium containing 0.1% L-arabinose <br /> addition of glucose allows tight repression of target         and incubated at 32&#194;&#176; for 1.5 hours prior to plating on LB <br /> genes under control of Para [40,41].                          plates agar containing L-arabinose (0.1%) and kanamycin <br />   We closely followed the description of [40] and first       (50 &#206;&#188;g/ml, Sigma). Clones were checked on LB agar <br /> constructed a strain in which a kanamycin resistance          plates supplemented with 0.4% glucose to confirm that <br /> cassette is linked to the promoter of araBAD. We              they were unable to grow in the presence of glucose. <br /> inserted such a kanamycin marker downstream of araC              The promoter fusions and the adjacent araC gene <br /> with the following primers:                                   were verified by sequencing with the following primers: <br />   5&#226;&#8364;&#8482;_araC_yabI_insert                                            araC_FW GCTACTCCGTCAAGCCGTCA; <br />   AATCAGACAATTGACGGCTTGACGGAGTAGCA-                              ygjD_RW GGCAATTGGTCTGGGGAGCA. <br /> TAGGGTTTTGTGTAGGCTGGAGCTGCTTC;                                   dnaTC_RW AGAGTTGATCGTCCAGAGCG <br />   3&#226;&#8364;&#8482;_araC_yabI_insert                                            ffh_RW ATTTTGACGAACTCCTGCCC <br />   GCATAATGTGCCTGTCAAATGGACGAAGCAGG-                              fldA_RW CGAGAGTCGGGAAGAAGTCA <br /> GATTCTGCCATATGAATA                                               The constructs were then moved by P1 transduction <br />   TCCTCCTTAGTTCCTAT.                                          into MG1655. To construct TB80 the kanamycin cas- <br />   The insertion was done in DY330 following the proto-        sette was removed with pCP20. <br /> col described by [42], verified by PCR and moved to              The knockout &#206;&#8221;relA::kan was derived from the KEIO <br /> MG1655 by P1 transduction, thus generating TB55.              library clone JW2755 [2] and P1-transduced into TB82. <br />   TB55 was subsequently used to generate a PCR product        &#206;&#8221;spoT::kan was introduced using AB1058) as donor <br /> that spanned the kanamycin cassette adjacent to araC,         strain for P1 transduction. <br /> araC, the full intergenic region between araC and araB,          To measure activity of the promoters Para, Prsd and Papt, <br /> and 42 basepairs at the 5&#226;&#8364;&#8482; and 3&#226;&#8364;&#8482; -prime ends that were       MG1655 and TB80 were transformed [37] with plasmids <br /> homologous to the upstream and 5&#226;&#8364;&#8482;-coding region of ygjD,      that contain transcriptional promoter-gfp fusions [29]. <br /> dnaT ,fldA or ffh. The sequence of the primers was <br />   ygjD_insert5&#226;&#8364;&#8482; AGTTTTACATCAACCCGCATTGG-                      Microscopy <br /> TCCTACACTGCGCGGTAATAATGTGCCTGTCA-                             LB agar pads were prepared by filling a cavity of a sterile <br /> AATGGACG                                                      microscope cavity slide with a drop of freshly melted LB <br />  Bergmiller et al. BMC Microbiology 2011, 11:118                                                                                            Page 10 of 12 <br /> http://www.biomedcentral.com/1471-2180/11/118 <br />  <br />  <br />  <br />  <br /> agar, and covering it with a cover slip to attain a flat sur-                compressed into 15 seconds. This movie was used to extract the growth <br /> face. The cavity slide was transferred to a fridge for a                     dynamics shown in Figure 2 and 3. <br /> short time to allow the agar to solidify. Upon removal of                    Additional File 5: Figure S2: Lineage trees of microcolonies of A) <br /> the cover slip and removal of excess LB agar, the pads                       MG1655 growth and B) YgjD depletion. After tracking of individual <br />                                                                              cells across recorded images with &#226;&#8364;&#339;Schnitzcell&#226;&#8364;&#65533;, the lineage structure of a <br /> can be inoculated with precultured cells as described in                     microcolony can be derived. In such a lineage tree, the branch length <br /> the next paragraph.                                                          corresponds to the time interval between divisions, and division events <br />   TB80 and TB84 were cultured over night at 37&#194;&#176; in LB                        occur at branching points. The different colors depict the color <span id='am-166' about='Thesaurus:code' typeof='owl:Thing'>code</span> <br />                                                                              used for cells from different generations throughout all figures. The dots <br /> medium with 0.1% L-arabinose and diluted 1:100 into                          at the end of individual branches represent the time points where <br /> fresh LB medium containing 0.01% L-arabinose. In early                       individual physiological measurements (cell size and fluorescent intensity) <br /> exponential phase, cultures were washed at least twice                       were derived from. <br /> in LB supplemented with 0.4% glucose to remove resi-                         Additional File 6: Figure S3: Depletion of essential genes induces <br />                                                                              unique phenotypes. Time-lapse experiments of cells depleting for fldA, <br /> dual L-arabinose. Wildtype E. coli MG1655 was treated                        ffh and dnaT (see Additional Files 7, 8 and 9 - movies 4, 5 and 6) were <br /> similar for control experiments. 1.5 &#206;&#188;l of a washed and                      tracked, and the cell size at division over consecutive divisions was <br /> diluted culture were transferred to the surface of a pad                     plotted. <br /> of LB agar (supplemented with D-glucose, L-arabinose,                        Additional File 7: movie 4: Depletion of FldA from growing cells. A <br />                                                                              Para-fldA conditional lethal mutant was shifted from 0.1% arabinose to an <br /> chloramphenicol or kanamycin as indicated for indivi-                        agar pad with 0.4% glucose. FldA is essential for isoprenoid biosynthesis <br /> dual experiments) in a microscope cavity slide. The agar                     [44], and as the movie shows, depletion of FldA leads to lysis of cells. 80 <br /> pad was closed with a cover slip and sealed with                             frames (one frame per four minutes) were compressed into 8 seconds. <br /> vacuum grease. Under these conditions, cells can grow                        Additional File 8: movie 5: Depletion of Ffh from growing cells. A <br />                                                                              Para-ffh conditional lethal mutant was shifted from 0.1% arabinose to an <br /> exponentially in a two-dimensional plane for many gen-                       agar pad with 0.4% glucose. Ffh protein is part of the signal recognition <br /> erations without restrictions [23]. The slide was                            particle translocation system, that cotranslationaly sequesters proteins <br /> mounted onto an automated microscope (Olympus                                into or across the cytoplasmic membrane [45]. Depletion resulted in <br />                                                                              visible intracellular aggregates, followed by elongation and cell lysis. 120 <br /> BX81) and incubated at 37&#194;&#176;C (Cube and Box incubation                         frames (one frame per two minutes) were compressed into 12 seconds. <br /> system, Life Imaging Services, Reinach, Switzerland).                        Additional File 9: movie 6: Depletion of DnaT from growing cells. A <br /> Images were recorded every 2 or 4 minutes. Intensity                         Para-dnaT conditional lethal mutant was shifted from 0.01% arabinose to <br /> and exposure times to fluorescent light were minimized                       a 0.4% glucose containing agar pad. Depletion resulted in filament <br />                                                                              formation, which is in agreement with &#226;&#8364;&#339;unbalanced&#226;&#8364;&#65533; growth upon <br /> to avoid cellular damage. The resulting image sequences                      abrogation of DNA replication. dnaT (and the following gene dnaC) is <br /> were analyzed with the Matlab based script package                           part of the &#226;&#8364;&#339;primosome&#226;&#8364;&#65533; and is crucial for initiation of DNA replication. <br /> &#226;&#8364;&#339;Schnitzcell&#226;&#8364;&#65533; (kindly provided by Michael Elowitz, Cal-                      100 frames (one frame per four minutes) were compressed into 10 <br />                                                                              seconds. <br /> Tech, USA [18]), and data was extracted with custom- <br />                                                                              Additional File 10: Figure S4: Effects of minimum inhibitory <br /> made Matlab scripts (Table 1).                                               concentrations (MIC) of chloramphenicol and kanamycin on growth <br />                                                                              of E. coli MG1655. Recorded image series of E.coli MG1655 growing on <br /> Statistical analysis                                                         MIC concentrations of chloramphenicol (2.5 &#206;&#188;g/ml) and kanamycin (5 <br />                                                                              &#206;&#188;g/ml) (see Additional Files 11 and 12 - movies 7 and 8) were tracked, <br /> To quantify associations between phenotypic traits, we                       and the cell size over consecutive division was plotted. <br /> used non-parametric correlation analysis (Spearman&#226;&#8364;&#8482;s                         Additional File 11: movie 7: Growth of E. coli MG1655 on 2.5 &#206;&#188;g/ml <br /> rank correlation in PASW Statistics 18.0).                                   chloramphenicol. E. coli MG1655 was precultured in LB medium and <br />                                                                              transferred to an agar pad containing 2.5 &#206;&#188;g/ml chloramphenicol. 100 <br />                                                                              frames (one frame per four minutes) were compressed into 10 seconds,. <br /> Additional material <br />                                                                              Additional File 12: movie 8: Growth of E. coli MG1655 on 5 &#206;&#188;g/ml <br />                                                                              kanamycin. E. coli MG1655 was precultured in LB medium and <br />                                             + <br />  Additional File 1: Movie 1. TB80 (ppGpp ) growing on LB agar with           transferred to an agar pad containing 5 &#206;&#188;g/ml kanamycin. 60 frames <br />  0.1% L-arabinose. 100 frames (one frame per two minutes) were               (one frame per four minutes) were compressed into 6 seconds. <br />  compressed into 10 seconds. The scale bar is 5 &#206;&#188;m in size (same in all      Additional File 13: Figure S5: Coupling of cell elongation rate and <br />  movies hereafter).                                                          interval between division across multiple experiments. The pattern <br />  Additional File 2: Movie 2: MG1655 growing on LB agar with 0.4%             observed in Figure 3 is repeatable and consistent across independent <br />  glucose. 100 frames (one frame per two minutes) were compressed into        experiments. Non-parametric correlation analysis for the differences <br />  10 seconds.                                                                 between sisters in these two traits was performed for seven independent <br />  Additional File 3: Figure S1: MG1655 expressing GFP from Para               microcolonies (YgjD depletion in TB80), and the median and the range <br />  shifted from LB arabinose 0.01% to LB glucose 0.4%. This                    of the correlation coefficients is reported; the median correlation <br />  experiment was performed with the wild type strain MG1655 carrying a        coefficients are negative from generation 3 on, indicating a coupling <br />  plasmid encoding a transcriptional fusion of gfp to Para [29]. The strain   between cell elongation rate and the interval between two divisions. <br />  was grown in 0.01% arabinose, analogously to the depletion experiments      Additional File 14: Movie 9. TB84 (ppGpp0) growing on LB agar <br />  with TB80 and TB84, washed in LB supplemented with glucose and              with 0.4% glucose. 200 frames (one frame per two minutes) were <br />  transferred onto an agar pad consisting of LB agar with 0.4% glucose.       compressed into 20 seconds. <br />  The level of GFP fluorescence decreased rapidly and approached the          Additional File 15: Figure S6: YgjD is also essential in absence of (p) <br />  level of background fluorescence when cells reached generation 4.           ppGpp. Data of cell numbers versus time from three independent <br />  Additional File 4: Movie 3. TB80 (ppGpp+) growing on LB agar with           experiments; each experiment is based on a microcolony that was <br />  0.4% glucose. 150 frames (one frame per two minutes) were                   initiated with a single cell of strain TB84 (ppGpp0), and grown in the <br />  <br />  presence of glucose, leading to YgjD depletion. Cell division terminates           <br />  after about five to six divisions.                                              <br />  Additional File 16: Figure S7: Control movies of Papt and Prsd                    <br />  expression of TB80 grown with 0.1% L-arabinose. Single cell                          <br />  measurements of cell elongation rate and GFP fluorescence of two                <br />  strains with transcriptional reporters for Papt (A and B) and Prsd (B and             <br />  C), analogous to Figure 5 in the main manuscript.                                 <br />  Additional File 17: Figure S8: DNA staining of cells with and                  <br />  without YgjD in TB80 (ppGpp+) and TB84 (ppGpp0). Cells were                          <br />  grown for two hours in liquid culture, and stained with 1 &#206;&#188;g/ml DAPI                   <br />  (4&#226;&#8364;&#8482;,6-diamidino-2-phenylindole) to visualize DNA. Scale bars are 5 &#206;&#188;m. A)              <br />  TB80 grown with 0.1% arabinose to induce YgjD expression. B) TB80                  <br />  grown with 0.4% glucose, leading to YgjD depletion. Cells are small, and               <br />  the DNA stain occupies a large fraction of the cell area. C) TB84 grown               <br />  with 0.4% glucose, leading to YgjD depletion. Cells are elongated, and           <br />  the DNA stain only occupies a small fraction of the cell area.                        <br /> </body></html>