<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Orth Palsson BMC Systems Biology 2012, 6:30 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                                 Open Access <br />  <br /> Gap-filling analysis iJO1366 Escherichia coli <br /> metabolic network reconstruction discovery of <br /> metabolic functions <br /> Jeffrey D Orth Bernhard   Palsson* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: The iJO1366 reconstruction metabolic network Escherichia coli complete <br />   accurate metabolic reconstructions available organism. Still, knowledge well- <br />   studied model organisms incomplete, network reconstruction contains gaps possible <br />   errors. There total 208 blocked metabolites iJO1366, representing gaps network. <br />   Results: A new model improvement workflow developed compare model based phenotypic predictions <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />   experimental data gaps correct errors. A Keio Collection based dataset E. coli gene essentiality was <br />   obtained literature data compared model predictions. The SMILEY algorithm used predict <br />   likely missing reactions reconstructed network, adding reactions KEGG based universal set of <br />   metabolic reactions. The feasibility putative reactions determined comparing updated versions of <br />   model experimental dataset, genes predicted feasible reactions. <br />   Conclusions: Numerous improvements iJO1366 metabolic reconstruction suggested analyses. <br />   Experiments performed verify computational predictions, including new mechanism growth <br />   myo-inositol. The predictions study experimentally verifiable similar means. <br />   Validating predictions represents substantial important undertaking. <br />   Keywords: Constraint-based modeling, Metabolic network reconstruction, Escherichia coli, Gap-filling, Gene <br />   annotation <br />  <br />  <br /> Background                                                                              model missing essential reaction [5]. This pre- <br /> Constraint-based modeling widely used systems biology                              diciton called false negative. No current metabolic net- <br /> method particularly suited predicting                               work reconstruction entirely complete realistic <br /> phenotypes microbial organisms gene knockouts                               knowledge metabolism organism <br /> when grown different substrates [1-3]. These variable                                complete. Even well-studied model organisms <br /> conditions simply represented additional constraints                             Escherichia coli genes un- <br /> on model, growth predicted flux balance                                 known functions [6,7]. The result are <br /> analysis (FBA) [4]. Because realistic constraint                           gaps metabolic network reconstructions. These gaps take <br /> represented typical metabolic model, quite possible                          form dead-end metabolites, no <br /> for model predict growth conditions                               producing consuming reactions [8]. <br /> growth does really occur. The actual organism                                 Several different types gaps exist reconstructed <br /> express required gene growth, fluxes limited                            metabolic networks [8,9]. These gaps result blocked <br /> by kinetic thermodynamic constraints, example. This                              reactions, unable carry flux steady state, <br /> case called false positive prediction. On hand,                          blocked metabolites, exist blocked reac- <br /> false predictions growth taken indications                              tions produced consumed. Root no- <br />                                                                                         production gaps metabolites consuming <br /> * Correspondence: palsson@ucsd.edu                                                      reactions blocked producing <br /> Department Bioengineering, University California, San Diego, La Jolla, <br /> CA, USA, 9500 Gilman Drive, Mail <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>Code</span> 0412, La Jolla CA 92093-0412, USA                 reactions. Metabolites produced <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />  <br />                                           &#194;&#169; 2012 Orth Palsson; licensee BioMed Central Ltd. This Open Access article distributed terms the <br />                                           Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, <br />                                           distribution, reproduction medium, provided original work properly cited. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                    Page 2 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> root no-production metabolites blocked,       organism specific databases BioCyc [17]. This program <br /> referred downstream gaps. Likewise, root no-con-         fills gaps complete metabolic pathways includes <br /> sumption gaps metabolites producing reactions         hole-filling algorithm assigns genes gap-filling reac- <br /> but consuming reactions, metabolites          tions [18,19]. Another recent procedure uses network ex- <br /> blocked gaps called upstream gaps. The gaps       pansion determine minimum number reactions <br /> in metabolic network classified        need added network make compliant <br /> scope gaps knowledge gaps. Scope gaps        experimental data [20]. The production metabolites <br /> exist scope metabolic network models       macromolecule degradation products considered, <br /> does include features like macromolecular degradation      genes predicted using hidden Markov models. <br /> or use charged tRNAs protein synthesis. Know-        This strategy applied improve metabolic models of <br /> ledge gaps, hand, actually result      E. coli [21] Chlamydomonas reinhardtii [22]. <br /> our incomplete knowledge metabolism or-             The present study builds methods new <br /> ganism [10].                                                   workflow includes use SMILEY algorithm. <br />    The comparison model predictions experimental         SMILEY used instead GapFill GrowMatch be- <br /> data useful way network gaps dis-         cause modified make predictions a <br /> cover new genes reactions. There possible         wider range experimental data originally <br /> outcomes comparing computationally predicted           applied to. Specifically, used make predictions <br /> experimentally measured growth phenotypes: true posi-          using gene essentiality data network gaps addition <br /> tives, model correctly predicts growth; true          data growth different substrates. The iJO1366 <br /> negatives, model correctly predicts           metabolic network reconstruction E. coli K-12, lat- <br /> growth possible; false positives, model pre-       est complete genome-scale reconstruction of <br /> dicts growth condition growth            organism [10], used analysis. To begin, a <br /> observed; false negatives, model fails         large dataset E. coli gene essentiality Keio <br /> predict growth growth experimentally                 Collection [23], combined published datasets <br /> observed. Both false positive false negative results       [10,23-25], assembled. Next, model growth predic- <br /> can useful refining model content,        tions using iJO1366 model compared to <br /> false negative cases help gaps. Several meth-    dataset, false positive false negative <br /> ods developed predict correct gap-fill-       comparisons analyzed identify potential errors in <br /> ing reactions based comparisons experimental             model experimental datasets. The <br /> data.                                                          SMILEY algorithm used predict gap-filling <br />    The method published called            reactions reactions correct false negative model <br /> SMILEY [5]. This mixed-integer linear programming         predictions. The feasibility reactions then <br /> algorithm identifies minimum number reac-          assessed comparing augmented model predictions to <br /> tions need added metabolic model        experimental dataset. Finally, genes predicted <br /> universal database reactions order allow mini-      feasible putative reactions. Several sets of <br /> mum defined growth rate achieved. The SMILEY             gene function predictions presented, provide <br /> algorithm developed used predict reac-        plausible hypotheses experimental validation. These <br /> tions missing iJR904 E. coli reconstruction [11]      predictions potential improve metabolic <br /> that caused false negative model growth predictions            reconstruction lead new metabolic gene discover- <br /> when compared Biolog growth data [12]. Several              ies [8]. <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>Knockout</span> strain growth phenotyping experiments <br /> results experimentally verified new genes        performed identify gene involved myo-inosi- <br /> characterized [5]. SMILEY recently used pre-       tol metabolism, demonstrating types experimental <br /> dict gap-filling reactions Recon 1 human metabolic      analyses validate biological predictions. <br /> reconstruction [13,14]. The algorithms GapFind/GapFill [9] <br /> and GrowMatch [15] later developed, pre-        Results <br /> dict missing reactions connecting model gaps         Comparison model predictions experimental data <br /> comparing model predictions gene essentiality data, re-     By applying developed workflow analyze iJO1366 <br /> spectively. To date, methods used make      model gaps compare model predicted phenotypes to <br /> predictions E. coli yeast metabolic networks       experimental data, new biological hypotheses gener- <br /> [15,16], predictions experimen-    ated (Figure 1). First, experimental datasets were <br /> tally verified. Non-constraint-based methods recon-        assembled combined. Each dataset consisted large <br /> structing metabolic networks filling gaps   set E. coli <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span> strains grown different types <br /> developed. One example PathoLogic, component       media. All gene knockout strains the <br /> Pathway Tools software used assemble      Keio Collection E. coli BW25113 single gene knockouts, <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                Page 3 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />     Baba et al.        Joyce et al.          Orth et al.           Ito et al. <br />       Entire Keio        Entire Keio         1075 genes       1440 genes on <br />      Collection      Collection       glucose, lactate,     Biolog GN2 <br />        glucose            glycerol           succinate           plates <br />  <br />  <br />  <br />  <br />      JO1366 E.                  Combined                KEGG universal <br />     coli metabolic                phenotypic                 reaction <br />                                    dataset                  database <br />          model                                              8447 reactions <br />                                 13,470 phenotypes <br />  <br />  <br />  <br />  <br />                                         711 false <br />             Compared model              positives <br />               predictions to <br />             experimental data                                                                           Computed <br />                                         265 false                                                                             Feasible <br />                                                                                                        feasibility of <br />                                         negatives                                                                             solutions <br />                                                                                                      SMILEY solutions <br />                                                           Used SMILEY               1176 <br />                                                           predict gap-filling        SMILEY <br />                                                               reactions              solutions <br />              Used GapFind              208 <br />                                                                                                      Predicted vivo <br />              identify network           network                                                                                In vivo <br />                                                                                                       experiments         experiments <br />                    gaps                  gaps <br />                                                                                                      confirm reactions <br />  <br />  <br />  <br />  Figure 1 Workflow predicting FN-correcting gap-filling reactions using SMILEY. Input datasets shown blue, computational <br />  prediction steps orange, analyzed outputs green. <br />  <br />  <br />  <br /> allowing analyzed together. The dataset,                    259 1440 genes iJO1366, only <br /> from Baba et al. [23], contained phenotypes entire                  genes included. Five 38 substrates were <br /> Keio Collection grown glucose MOPS minimal media.                         included iJO1366 model KEGG com- <br /> This defined media contains buffer MOPS (3-(N-mor-                       pound database, included they <br /> pholino)propanesulfonic acid), potential sulfur source.                    connected model content using the <br /> The second dataset growth screen entire                   methods presented here. The phenotypes combined <br /> Keio Collection, glycerol M9 minimal media [25].                      dataset adjusted slightly original publica- <br /> The dataset screen 1075 Keio Collection                       tions based recent analysis Keio Collection <br /> strains, genes included iAF1260 E. coli meta-                 genotypes [26]. Several new genes classified essen- <br /> bolic reconstruction [21], grown different media                     tial, added essential genes glucose <br /> conditions [10]. The strains grown glucose M9                        glycerol. One gene, b0103, removed Bio- <br /> media aerobic anaerobic conditions, lac-                   log screen data based analysis. The screen 1075 <br /> tate M9 aerobically, succinate M9 aerobically. The                    strains conditions performed Keio <br /> fourth dataset consisted phenotypes 1440 Keio Col-                   Collection update, accounted these <br /> lection strains grown Biolog GN2 plates [24]. It                      changes. Some datasets contained phenotypes the <br /> found wild-type E. coli grow 38 different car-                 substrates. For example, Biolog data contained <br /> bon sources Biolog plate, dataset                        strains grown glycerol, succinate, lactate. In these <br /> included 38 substrates.                                                cases, data point included gene <br />    The datasets combined large                   knockout strain grown substrate. If the <br /> phenotypic dataset. From screens entire Keio                      datasets included &#226;&#8364;&#339;growth&#226;&#8364;? <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>, phenotype <br /> Collection glucose glycerol, growth phenotype                   set &#226;&#8364;&#339;growth&#226;&#8364;? combined dataset. Only strain <br /> included 1366 genes iJO1366. For                      &#226;&#8364;&#339;no growth&#226;&#8364;? phenotype datasets classified <br /> screen conditions, phenotypes available                     essential combined dataset. After making these <br /> 1075 1366 genes. For screen Biolog plates,                     adjustments, final combined dataset contained 13,470 <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                       Page 4 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> experimental phenotypes. There 12,120 &#226;&#8364;&#339;growth&#226;&#8364;? phe-           &#226;&#8364;&#339;Cofactor Prosthetic Group Metabolism,&#226;&#8364;? 7.4 % and <br /> notypes 1350 &#226;&#8364;&#339;no growth&#226;&#8364;? phenotypes.                           7.7 % respectively. These false negative cases indicate the <br />    The iJO1366 E. coli metabolic network model            likely presence currently unknown isozymes alterna- <br /> used predict growth phenotypes 13,470 condi-          tive pathways. <br /> tions. This model E. coli K-12 MG1655 metabolism was <br /> first modified slightly match genotype E. coli           False positive model predictions <br /> BW25113, parent strain Keio Collection. FBA [4]         The set false positive predictions investigated in <br /> was used predict growth rates using iJO1366            determine occurred. Every gene <br /> core biomass objective gene knockout              false positive prediction sub- <br /> every substrate experimental dataset. Any growth rate       strate experimental growth substrate <br /> above zero classified computational &#226;&#8364;&#339;growth&#226;&#8364;?              tested. It possible <br /> phenotype, growth rate zero classified &#226;&#8364;&#339;no       reasons false positive prediction the <br /> growth&#226;&#8364;?. An silico dataset 13,470 phenotypes        iJO1366 model. First, possible model may <br /> generated, compared vivo dataset. Each           contain error unrealistic reaction (Table 1). <br /> model prediction classified true positive, true    In model, reaction CBPS (carbamoyl phosphate syn- <br /> negative, false positive, false negative. See Additional file   thase (glutamine-hydrolyzing)) converts L-glutamine car- <br /> 1 complete sets computational experimental          bamoyl phosphate, essential precursor L-arginine. <br /> phenotypes. A total 11,855 true positives, 639 true nega-       This reaction catalyzed complex carA (b0032) and <br /> tives, 711 false positives, 265 false negatives identi-   carB (b0033), experimentally essen- <br /> fied (Figure 2 a). The Matthews Correlation Coefficient            tial glucose, glycerol, succinate, lactate minimal <br /> (MCC) predictions, measure accuracy           media. In model, genes non-essential to <br /> binary classifications, 0.5418. Overall, silico         alternate reaction produces carbamoyl phosphate, <br /> screen predicted growth phenotypes            CBMKr (carbamate kinase), catalyzed products of <br /> in experimental data (93.3 % 90.0 %, respectively).        yahI (b0323), arcC (b0521), yqeA (b2874). This putative <br /> This result largely explained nature con-         reaction included iJO1366 based physiological <br /> straint-based modeling FBA. Because iJO1366                data [28], functions genes well <br /> model does contain regulation, FBA use reac-           characterized. It likely CBMKr reaction <br /> tion network produce biomass. In E. coli cell,        unrealistic. False positives caused errors in <br /> different levels regulation make certain enzymes un-        iJO1366 core biomass reaction. The gene pdxH (b1638) <br /> available certain conditions,          catalyzes reactions PDX5POi (pyridoxine 5&#226;&#8364;&#8482;-phosphate <br /> allowed growth. Other real constraints, kinetic        oxidase) PYAM5PO (pyridoxamine 5&#226;&#8364;&#8482;-phosphate oxi- <br /> or thermodynamic constraints [27], accounted            dase), required synthesis pyridoxal 5&#226;&#8364;&#8482;-phosphate <br /> for model cause false positive       (vitamin B6). This vitamin included core bio- <br /> predictions.                                                       mass, reactions essential model. <br />    The genes iJO1366 model classified        However, essentiality gene glucose gly- <br /> 11 functional categories, according metabolic func-         cerol minimal media indicates vitamin B6 fact es- <br /> tions serve [10]. The different categories      sential E. coli, included model <br /> contain genes varying levels predictive accuracy           biomass reaction. <br /> (Figure 2 b). Genes &#226;&#8364;&#339;Others&#226;&#8364;? category, including               Some false positive predictions likely occurred because <br /> mainly tRNA charging genes genes             gene incorrectly identified essential of <br /> placed categories, lead false        experimental screens (Table 2). This case with <br /> positives 23.7 % cases. This known tRNA        genes involved energy production. The cyto- <br /> charging gaps iJO1366 model [10]. These tRNA char-          chrome oxidase gene cydA (b0733) knockout strain does <br /> ging genes essential vivo. There false       exist Keio Collection, presumed be <br /> positives &#226;&#8364;&#339;Energy Production Conversion&#226;&#8364;?             essential. A viable knockout strain gene been <br /> genes (12.4 %). This outcome partly caused miss-         produced, however, knockouts other <br /> ing thermodynamic constraints, partly fact         cytochrome oxidases [29]. The ATP synthase genes <br /> disruptions cellular energy generation cause E. coli     atpCDGAHFEB (b3731-8) classified essential on <br /> grow slowly, strains            minimal media, inspection actual growth mea- <br /> found essential experimental screens             surements experiments [10,23,25] reveals that <br /> though actually slowly growing. The computa-             knockout strains did actually grow, albeit slowly. <br /> tional screen classified growing strains non-essential,        Many false positive cases occurred gene knockout <br /> even grew slowly. False negatives com-           strains known isozymes alternative pathways <br /> mon genes &#226;&#8364;&#339;Amino Acid Metabolism&#226;&#8364;?                 (Table 3). In iJO1366 model, knockouts are <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                                                                                  Page 5 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />                                                                a <br />                                                                                                                       Experimental <br />                                                                                                                  Growth        No growth <br />                                                                                                                   12,120          1350 <br />                                                                                                                  (90.0%)        (10.0%) <br />  <br />  <br />                                                                                                12,566             11,855          711 <br />                                                            Computational <br />                                                                                Growth <br />                                                                                               (93.3%)            (88.0%)        (5.3%) <br />  <br />  <br />  <br />  <br />                                                                                No growth        904                265                639 <br />                                                                                               (6.7%)             (2.0%)             (4.7%) <br />  <br />  <br />                                                                                                                        13,470 Total <br />  <br />  <br />  <br />                      b <br />                                                                                 True Positives                                                         False Positives <br />                                                  100                                                                           25 <br />                        Percent wtihin category <br />  <br />  <br />  <br />  <br />                                                  80                                                                            20 <br />  <br />                                                  60                                                                            15 <br />  <br />                                                  40                                                                            10 <br />  <br />                                                  20                                                                             5 <br />  <br />                                                    0                                                                            0 <br />                                                        A   C               W    P   E    I     L    N        O   TI   TO              A      C   W      P     E    I     L    N    O   TI   TO <br />  <br />  <br />  <br />  <br />                                                                                False Negatives                                                         True Negatives <br />                                                   10                                                                           20 <br />                      Percent wtihin category <br />  <br />  <br />  <br />  <br />                                                    8                                                                           16 <br />  <br />                                                    6                                                                           12 <br />  <br />                                                    4                                                                            8 <br />  <br />                                                    2                                                                            4 <br />  <br />                                                    0                                                                            0 <br />                                                        A   C               W    P   E     I     L   N        O   TI   TO              A   C      W      P     E    I    L     N    O   TI   TO <br />  <br />                                                                            Metabolic functional categories                                           Metabolic functional categories <br />  <br />  <br />  Figure 2 Comparison model predicted growth phenotypes experimental data. (a) The overall comparison, indicating numbers true <br />  positives, true negatives, false positives, false negatives. (b) The numbers type prediction 11 functional categories of <br />  metabolic reactions. The categories are: amino acid metabolism (A), carbohydrate metabolism (C), cell wall/membrane/envelope metabolism (W), <br />  cofactor prosthetic group metabolism (P), energy production conversion (E), inorganic ion transport metabolism (I), lipid metabolism <br />  (L), nucleotide metabolism (N), (O), inner membrane transport (TI), outer membrane transport (TO). <br />  <br />  <br /> overcome using isozyme alternative pathway                                                                    reactions blocked scope gaps, important reac- <br /> synthesize biomass components. In vivo, genes                                                                    tions used model. Finally, false <br /> be essential isozyme genes expressed                                                                 positives explained model alone. For ex- <br /> the experimental conditions, capable                                                                 ample, gene spoT (b3650) required synthesize the <br /> catalyzing reaction sufficient rate growth                                                               signaling molecule guanosine tetraphosphate (ppGpp). <br /> to occur. These types false positive model predictions                                                                  Since metabolic model does require signaling, this <br /> cannot overcome standard FBA using meta-                                                                      gene non-essential. Experimentally, spoT is <br /> bolic model. A model including regulation add-                                                                    essential rich media, likely non- <br /> itional constraints required. Many false positives                                                                 metabolic function. The false negatives be <br /> occur tRNA charging genes knocked                                                                      explained model ftsI (b0084), adk (b0474), mrdA <br /> model (Table 4). Since iJO1366 tRNA charging                                                                           (b0635), cydC (b0886), gapA (b1779), ligA (b2411), suhB <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                  Page 6 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 1 False positive model predictions indicate                   indication missing metabolic reactions pathways in <br /> model errors                                                             model. In section, use SMILEY algorithm <br /> Gene           Error                                                     predict likely missing reactions presented. There are <br /> carA (b0032)   alternate pathway (CBMKr) gene functions              possible explanations false negative predic- <br />                confirmed                                                 tions. First, possible model biomass reaction <br /> carB (b0033)   alternate pathway (CBMKr) gene functions              used objective incorrect (Table 5). Several <br />                confirmed                                                 false negative cases occurred knockouts genes <br /> proB (b0242)   alternate pathway (NACODA) gene function              involved molybdenum cofactor synthesis, including mog <br />                confirmed                                                 (b0009), moaA (b0781), moaC (b0783), moaD (b0784), <br /> proA (b0243)   alternate pathway (NACODA) gene function              moaE (b0785), moeA (b0826), moeB (b0827), mobA <br />                confirmed <br />                                                                          (b3857). In iJO1366 model, genes essential <br /> folD (b0529)   5fthf[c] methf[c] essential                    required produce bmocogdp[c] (bis- <br /> entD (b0583)   enter[c] essential                                 molybdopterin guanine dinucleotide), component the <br /> pyrD (b0945)   alternate pathway (DHORDfum) orphan reaction        core biomass formulation. Because gene knockout <br /> pdxH (b1638)   pydx5p[c] essential                                strains experimentally viable conditions, is <br />                                                                          likely cofactor essential growth, and <br /> pgsA (b1912)   pgp120[p] - pgp181[p] essential <br /> nrdA (b2234)   alternate pathway (RNDR1b &#226;&#8364;&#8220; RNDR4b) gene functions <br />                confirmed <br /> nrdB (b2235)   alternate pathway (RNDR1b &#226;&#8364;&#8220; RNDR4b) gene functions <br />                                                                          Table 3 False positive model predictions caused by <br />                confirmed                                             isozymes alternate pathways <br /> ptsI (b2416)   alternate pathway (GLCt2pp) glucose transport         Gene           Isozyme alternate pathway reactions <br />                confirmed                                                 thrA (b0002)   metL (b3940) lysC (b4024) <br /> waaK (b3623)   colipa[e] essential                                carA (b0032)   alternate reaction: CBMKr <br /> wzyE (b3793)   eca4colipa[e] essential                            carB (b0033)   alternate reaction: CBMKr <br /> ubiE (b3833)   reactions AMMQLT8 OMBZLM blocked gaps          folA (b0048)   folM (b1606) <br /> ubiB (b3835)   alternate pathway (OPXHH3) orphan reaction          (b0126)    cynT (b0339) <br /> ppa (b4226)    isozymes, ppx (b2502) surE (b2744),            pyrH (b0171)   cmk (b0910) <br />                incorrect <br />                                                                          int (b0657)    lpp (b1677) <br />                                                                          fldA (b0684)   fldB (b2895) <br /> (b2533), eno (b2779), fbaA (b2925), pgk (b2926), dut                     fabA (b0954)   fabZ (b0180) <br /> (b3640), pslB (b4041), alsK (b4084). <br />                                                                          nrdA (b2234)   alternate reactions: RNDR1b, RNDR2b, RNDR3b, RNDR4b <br />  <br /> False negative model predictions                                         nrdB (b2235)   alternate reactions: RNDR1b, RNDR2b, RNDR3b, RNDR4b <br /> All genes false negative predictions               cysK (b2414)   cysM (b2421) <br /> substrate computationally predicted growth                 ptsI (b2416)   alternate reaction: GLCt2pp <br /> substrate investigated detail. If constraint-             cysA (b2422)   modA (b0763) + modB (b0764) + modC (b0765) <br /> based metabolic model fails predict growth con-               cysP (b2425)   modA (b0763) + modB (b0764) + modC (b0765) <br /> dition growth observed experimentally, an <br />                                                                          guaB (b2508)   alternate reaction: XPPT <br />                                                                          glyA (b2551)   alternate reaction: GLYCL <br /> Table 2 False positive model predictions indicate <br /> incorrectly identified essential genes                                   acpS (b2563)   acpT (b3475) <br /> Gene                   Reason incorrect phenotype                    serA (b2913)   alternate reaction: GHMT2r <br /> cydA (b0733)           knocked successfully Portnoy et al. [29]   metC (b3008)   tnaA (b3708) malY (b1622) <br /> atpC (b3731)           ATP synthase knockout causes low growth rate      aroE (b3281)   ydiB (b1692) <br /> atpD (b3732)           ATP synthase knockout causes low growth rate      ilvA (b3772)   tdcB (b3117) <br /> atpG (b3733)           ATP synthase knockout causes low growth rate      metE (b3829)   metH (b4019) <br /> atpA (b3734)           ATP synthase knockout causes low growth rate      ubiB (b3835)   alternate reaction: OPHHX3 <br /> atpH (b3735)           ATP synthase knockout causes low growth rate      glnA (b3870)   ycjK (b1297) <br /> atpF (b3736)           ATP synthase knockout causes low growth rate      metL (b3940)   thrL (b0002) malY (b1622) <br /> atpE (b3737)           ATP synthase knockout causes low growth rate      ppa (b4226)    ppx (b2502) surE (b2744) <br /> atpB (b3738)           ATP synthase knockout causes low growth rate      serB (b4388)   alternate reaction: GHMT2r <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                              Page 7 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 4 False positive model predictions caused tRNA             reaction essential histidine synthesis <br /> charging reactions                                                  pathway, essential minimal media for <br /> Gene                                              Amino Acid        model. In vivo datasets, however, hisH not <br /> ileS (b0026)                                      L-isoleucine      essential aerobic conditions. It essential <br /> proS (b0194)                                      L-proline         HisH, HisF able catalyze this <br />                                                                     reaction, using NH3 instead glutamine N donor <br /> cysS (b0526)                                      L-cysteine <br />                                                                     [30]. hisH essential compo- <br /> leuS (b0642)                                      L-leucine <br />                                                                     nent IG3PS GPR. The GPR change sug- <br /> glnS (b0680)                                      L-glutamine       gested cyaY (b3807), gene involved in <br /> serS (b0893)                                      L-serine          transferring iron [Fe-S] cluster synthesis. In the <br /> asnS (b0930)                                      L-asparagine      model, gene essential component reac- <br /> tyrS (b1637)                                      L-tyrosine        tions ISC SUF [Fe-S] cluster synthesis <br />                                                                     pathways, essential conditions. This gene <br /> pheT (b1713)                                      L-phenylalanine <br />                                                                     characterized, experimen- <br /> pheS (b1714)                                      L-phenylalanine <br />                                                                     tally non-essential, likely strictly required the <br /> thrS (b1719)                                      L-threonine       reactions I2FE2SS, I2FE2SS2, S2FE2SS, 2FE2SS2. <br /> aspS (b1866)                                      L-aspartate       Other false positive cases likely experimental <br /> argS (b1876)                                      L-arginine        errors (Table 7). Several genes involved synthesis <br /> metG (b2114)                                      L-methionine      cofactors biotin thiamin experimentally <br />                                                                     classified non-essential. These cofactors known to <br /> hisS (b2514)                                      L-histidine <br />                                                                     required small quantities [31-33], likely that <br /> alaS (b2697)                                      L-alanine <br />                                                                     residual biotin thiamin media dur- <br /> fmt (b3288)                                       N-formyl-L-       ing growth experiments. In experimental screen on <br />                                                   methionine <br />                                                                     different conditions, thorough washing proce- <br /> trpS (b3384)                                      L-tryptophan      dures used prevent carryover preculture <br /> glyS (b3559)                                      glycine           media, genes classified essential. Fi- <br /> glyQ (b3560)                                      glycine           nally, false negatives caused currently unidenti- <br /> valS (b4258)                                      L-valine          fied isozymes (Table 8). For cases false <br />                                                                     negatives explained means, <br /> thus included iJO1366 core biomass             BLASTp used identify possible isozymes E. <br /> reaction.                                                           coli genome. One predicted isozyme ex- <br />   Two false positive cases explained incor-             perimentally verified. prpC (b0333), currently in <br /> rect gene-protein-reaction associations (GPRs)                   model associated MCITS (2-methylcitrate <br /> iJO1366 (Table 6). In one, gene hisH (b2023)                 synthase), confirmed isozyme of <br /> required reaction IG3PS (Imidazole-glycerol-3-              gltA (b0720), catalyzing CS (citrate synthase) [34,35]. <br /> phosphate synthase), hisF (b2025). This <br />                                                                     Computational prediction gap-filling reactions <br /> Table 5 False negative model predictions caused                  One cause model gaps false negative phenotypic <br /> incorrect core biomass composition                                  predictions realistic reactions miss- <br /> Gene                                          Biomass component     ing iJO1366 model. The current version of <br /> mog (b0009)                                   bmocogdp[c]           iJO1366 contains 48 root no-production gaps, 63 root <br /> moaA (b0781)                                  bmocogdp[c]           no-consumption gaps, 52 downstream gaps, 69 up- <br /> moaC (b0783)                                  bmocogdp[c]           stream gaps. Many scope gaps, caused by <br />                                                                     limited scope metabolic network, these <br /> moaD (b0784)                                  bmocogdp[c] <br />                                                                     previously identified [8,10]. The SMILEY algo- <br /> moaE (b0785)                                  bmocogdp[c] <br />                                                                     rithm used predict likely sets reac- <br /> moeA (b0826)                                  bmocogdp[c]           tions missing model. To predict false negative <br /> moeB (b0827)                                  bmocogdp[c] <br /> ubiX (b2311)                                  2ohph[c] <br />                                                                     Table 6 False negative model predictions suggest <br /> iscS (b2530)                                  bmocogdp[c]           changes iJO1366 model GPRs <br /> cysG (b3368)                                  sheme[c]              Gene             GPR correction <br /> mobA (b3857)                                  bmocogdp[c]           hisH (b2023)     essential IG3PS [30] <br /> ubiC (b4039)                                  2ohph[c]              cyaY (b3807)     essential I2FE2SS, I2FE2SS2, S2FE2SS, S2FE2SS2 <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                       Page 8 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 7 False negative model predictions                      Table 8 False negative model predictions caused by <br /> misidentified experiment phenotypes media                         missing isozymes alternate pathways <br /> compositions                                                         Gene                    Putative Isozyme             E-value <br /> Gene           Explanation                                           purK (b0522)            purT (b1849)                 2.00E-12 <br /> mtn (b0159)    essential according Choi-Rhee et al. [36]          gltA (b0720)            prpC (b0333)                 1.00E-41 <br /> thiI (b0423)   possibly thiamin media incomplete washing   aspC (b0928)            tyrB (b4054)                 4.00E-94 <br /> bioA (b0774)   possibly biotin media incomplete washing    fabH (b1091)            identified <br /> bioB (b0775)   possibly biotin media incomplete washing    pabC (b1096)            ilvE (b3770)                 7.00E-8 <br /> bioF (b0776)   possibly biotin media incomplete washing    icd (b1136)             dmlA (b1800)                 3.00E-19 <br /> bioC (b0777)   possibly biotin media incomplete washing    aldA (b1415)            gabD (b2661)                 1.00E-90 <br /> bioD (b0778)   possibly biotin media incomplete washing                            prr (b1444)                  3.00E-80 <br /> aroD (b1693)   experimental growth condition,                                 feaB (b1385)                 1.00E-69 <br />                possible error <br />                                                                                              aldB (b3588)                 2.00E-66 <br /> thiD (b2103)   essential according Orth et al. [10] <br />                                                                                              betB (b0312)                 4.00E-65 <br /> cysD (b2752)   experimental growth condition, <br />                possible error                                        ubiX (b2311)            identified <br /> argG (b3172)   experimental growth condition,         luxS (b2687)            identified <br />                possible error                                        thyA (b2827)            identified <br /> cysG (b3368)   experimental growth condition,         zupT (b3040)            identified <br />                possible error <br />                                                                      folB (b3058)            folX (b2303)                 1.00E-4 <br /> bioH (b3412)   possibly biotin media incomplete washing <br />                                                                      argG (b3172)            identified <br /> ilvE (b3770)   experimental growth condition, <br />                possible error                                        folP (b3177)            identified <br /> thiH (b3990)   essential according Orth et al. [10]               yrbG (b3196)            identified <br /> thiG (b3991)   essential according Orth et al. [10]               kdsC (b3198)            identified <br /> thiF (b3992)   essential according Orth et al. [10]               argD (b3359)            astC (b1748)                 1.00E-146 <br /> thiE (b3993)   essential according Orth et al. [10]                                       gabT (b2662)                 3.00E-64 <br /> thiC (b3994)   essential according Orth et al. [10]                                       puuE (b1302)                 3.00E-54 <br /> cysQ (b4214)   MOPS possible alternate S source                                         patA (b3073)                 4.00E-52 <br />                                                                                              hemL (b0154)                 3.00E-32 <br /> resolving reactions, model constrained match              cysG (b3368)            identified <br /> each false negative condition, time, SMILEY             ilvE (b3770)            pabC (b1096)                 8.00E-8 <br /> was run. For gene knockout strains lead false               dapF (b3809)            identified <br /> negative predictions 34 tested substrates (or all <br />                                                                      argC (b3958)            identified <br /> but two), likely set missing <br /> reactions cause incorrect predictions         argB (b3959)            identified <br /> strain. In cases, SMILEY run model                  hemE (b3997)            identified <br /> with glucose (both aerobic anaerobic), glycerol,            ubiC (b4039)            identified <br /> lactate, succinate substrates. To predict gap-filling         purA (b4177)            identified <br /> reactions, small lower bound placed known <br /> producing consuming reaction knowledge <br /> gap metabolite, SMILEY run. In order actu-                incomplete reactions removed set (Add- <br /> ally carry small flux reactions satisfy          itional file 3). <br /> all model constraints, gap-filling reaction set reac-           A total 1176 optimal suboptimal solutions were <br /> tions need added. SMILEY run 166                  identified SMILEY. Solutions identified 106 of <br /> false negative cases 49 gap reactions (Additional file           false negative cases 32 gaps. Multiple optimal <br /> 2). Only model knowledge gaps [8] targeted,                 solutions cases, total <br /> scope gaps. The algorithm set 25 alter-            198 different optimal solutions 983 different subopti- <br /> nate solutions condition, time limit 2 h           mal solutions. Five solutions optimal <br /> was placed solution. The reactions added                  suboptimal solutions different cases, 385 solu- <br /> SMILEY universal set reactions based               tions multiple times. Most for <br /> all reactions KEGG Release 58.0 [37]. Unrealistic             gene knockout strains grown multiple substrates for <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                                       Page 9 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> genes required GPRs reaction                                         model used predict growth phenotypes by <br /> for reactions pathway. For false negative                                          FBA 13,470 conditions experimental <br /> cases gaps, small number optimal solutions                                            dataset. The false positive false negative predictions <br /> were (Figure 3 a). No solution 77 cases,                                        identified comparison experimental data- <br /> and solutions 91 cases. In                                           set, number false negatives eliminated and <br /> four cases, 25 solutions optimal. These cases                                         new false positives created SMILEY solution <br /> for iscS (b2530) knockout strain grown different                                        counted. The feasible solutions be <br /> conditions. This gene ISC [Fe-S] cluster gen-                                      fixed false negatives introdu- <br /> eration model essential role                                        cing false positives. On average, solution cor- <br /> in molybdenum cofactor synthesis. Each alternate                                           rected 7.48 false negatives created 7.07 new false <br /> solutions involves import cofactor. The average                                         positives. A total 144 solutions (11 optimal 133 <br /> number optimal solutions SMILEY run                                                suboptimal) eliminate false negatives <br /> 2.56. Most optimal solutions included reaction,                                            producing new false positives. GapFind also <br /> and included (Figure 3 b). The average                                          run model solution added, determine <br /> number reactions optimal solution 1.41.                                                  model gaps eliminated. 74 solutions fill <br />   As molybdenum cofactor uptake reactions demon-                                                gap found. <br /> strate, SMILEY solutions realistic. A computa- <br /> tional feasibility check performed identify                                              Predictions genes hypothesized reactions <br /> most realistic solutions. Each solutions added                                           The feasible SMILEY solutions complete <br /> to iJO1366 model time, augmented                                               set 1176 solutions investigated detail. <br />                                                                                                     For feasible solutions, BLASTp used try <br />                                                                                                     identify candidate genes E. coli genome <br />                                                                                                     (Table 9). These solutions divided categor- <br />         a <br />                                                                                                     ies. Category I solutions optimal solutions that <br />                           80 77                                                                     eliminated false negative condition cre- <br />      Number FNs/gaps <br />  <br />  <br />  <br />  <br />                                                                                                     ating new false positives. These solutions gave an <br />                                     61 <br />                           60                                                                        average MCC 0.5436, slightly better original <br />                                                                                                     model. Of 11 category I solutions, fixed false <br />                           40                                                                        negatives adding deleted model reaction in <br />                                      30                                                             universal reaction list. This indicates that <br />                           20                         17                                             uncharacterized isozymes possible aspC (b0928), <br />                                                           7                          8              pabC (b1096), aldA (b1415), argD (b3359), hemE <br />                                              5                4                                 4 <br />                                                  2                                                  (b3397). Another solution suggested false negatives <br />                            0 <br />                                 0        2       4        6       8   10 12 14 16 18 20 22 24       &#206;&#8221;aspC strains corrected adding exist- <br />                                                   Number optimal solutions                       ing model reaction ASP1DC (aspartate 1-decarboxylase) <br />                                                                                                     reverse. No literature evidence support <br />     b                                                                                               refute reversibility reaction. The five <br />                           120 <br />                                                                                                     solutions involve addition new reactions the <br />                                      109 <br />                                                                                                     model. Four provide potential production routes <br />    Number solutions <br />  <br />  <br />  <br />  <br />                           100                                                                       aspartate compliment aspC deletion. The other <br />                           80                                                                        provides new reaction consume glycoaldehyde for <br />                                                                                                     &#206;&#8221;aldA strains. None reactions asso- <br />                           60 <br />                                                                                                     ciated genes KEGG database, indicating they <br />                           40                                                                        global orphan reactions. Candidate genes these <br />                                                                                                     reactions identified reference <br />                           20                                             16 <br />                                                               9 <br />                                                                                          4 <br />                                                                                                     sequences available. <br />                            0                                                                          The second category SMILEY solutions inves- <br />                                          1                    2           3      4       5 <br />                                                                                                     tigated optimal solutions fixed more <br />                                                  Number reactions solution <br />                                                                                                     false negatives number new false positives <br />  Figure 3 Properties 198 optimal SMILEY solutions. (a) <br />                                                                                                     created. There 70 category II solutions (not in- <br />  Number optimal solutions SMILEY run. For cases,                                        cluding category I solutions, fall within <br />  zero optimal solutions found. (b) Number                                   <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>). The average MCC solutions <br />  reactions optimal solution. Most optimal solutions consisted                                0.5531. Most solutions involve uptake of <br />  reaction.                                                                                 molybdenum cofactors precursors. As explained <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                            Page 10 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 9 Predicted genes feasible FN-                        solutions added feasible new reactions. In one, slightly <br /> correcting SMILEY solutions                                              different reaction producing dTMP added to <br /> Hypothesized changes directionality                                   compliment thyA (b2827) deletion. A currently unchar- <br /> Reaction     Category   Support                                          acterized E. coli gene, ybiU (b0821), identified by <br /> ASP1DC       I                                                           BLASTp candidate gene reaction, providing <br /> ASPT         III        reversible (Karsten Viola [38]) <br />                                                                          testable hypothesis function gene. The <br />                                                                          category II feasible solution added new reaction <br /> ICL          IV         reversible (MacKintosh Nimmo [39]) <br />                                                                          convert L-glutamate &#206;&#177;-ketoglutarate. Two candi- <br /> AKGDH        IV         reversible (EcoCyc)                          date genes high sequence homology known genes <br /> CITL         IV                                                          organisms, ydbL (b0600) ydcR (b1439), <br /> Hypothesized gap-filling reactions                                       found. <br /> Reaction     Category   Putative gene                         E-value       The category investigated consisted of <br /> R00352 (R)   IV         sucD (b0729)                          2.00E-20   suboptimal solutions fixed false <br /> R00373 (F)   I          global orphan                                    negative producing new false positives. A <br />                                                                          total 133 category III solutions found, having <br /> R00400 (F)   I          global orphan <br />                                                                          average MCC 0.5433. Some solutions <br /> R00507 (R)   IV         yhfW (b3380)                          0.47 <br />                                                                          included unrealistic reactions, oxygen <br /> R00529 (F)   IV         cysN (b2751) cysD (b2752) *                  consuming KEGG reaction R00357 reverse, <br /> R00530 (F)   IV         global orphan                                    oxygen producing direction. Others attempt com- <br /> R00531 (R)   IV         global orphan                                    pensate loss cofactor producing pathways <br /> R00695 (R)   I          global orphan                                    simply adding new uptake reactions those <br />                                                                          cofactors. Still, realistic reactions suggested <br /> R00709 (F)   IV         dmlA (b1800)                          6.00E-26 <br />                                                                          BLASTp identified candidate genes. One solution <br />                         icd (b1136)                           1.00E-26 <br />                                                                          consisted addition current model reac- <br />                         leuB (b0073)                          2.00E-15   tion ASPT (L-aspartase) reverse. Experimental evi- <br /> R00732 (R)   III        aroA (b0908)                          5.00E-32   dence supports reversibility reaction [38], <br />                         murA (b3189)                          7.00E-8    currently listed irreversible iJO1366. <br /> R00733 (R)   III        tyrA (b2600)                          2.80E-2    The fourth final category SMILEY solutions to <br />                                                                          examined optimal solutions were <br /> R01393 (R)   I          global orphan <br />                                                                          categories I II. There 62 solutions in <br /> R01618 (R)   IV         glgP (b3428)                          2.10 <br />                                                                          category average MCC of <br /> R01713 (F)   I          global orphan                                    0.5304, slightly worse unmodified iJO1366 <br /> R01731 (F)   IV         tyrB (b4054) *                                   model. Most solutions simply new up- <br /> R01785 (R)   III        rhaD (b3902) *                                   reactions blocked essential biomass compo- <br /> R01902 (R)   III        rhaB (b3904) *                                   nents, 14 new realistic reactions suggested, <br />                                                                          current model reactions running in <br /> R02200 (F)   IV         global orphan <br />                                                                          opposite directions. One new reversible <br /> R04209 (R)   IV         purC (b2476)                          7.00E-16 <br />                                                                          reactions, ICL (isocitrate lyase), confirmed a <br /> R05717 (R)   IV         cysH (b2762)                          3.00E-12   published study [39], another, AKGDH (2-Oxo- <br /> R06613 (F)   II         ybiU (b0821)                          1.6        glutarate dehydrogenase), actually reversible <br /> R07164 (R)   III        ydiJ (b1687)                          0.9        according EcoCyc [40]. See Additional file 4 all <br /> R07165 (R)   III        ydiJ (b1687)                          0.9        category I-IV solutions investigated. <br />                                                                             All 72 gap-filling SMILEY solutions investigated <br /> R07176 (R)   IV         global orphan <br />                                                                          (Additional file 5), BLASTp used predict genes <br /> R07463 (F)   IV         dadA (b1189)                          2.00E-18 <br />                                                                          realistic reactions (Table 10). A total 20 new real- <br /> R07613 (R)   II         ydbL (b0600)                          7.00E-26   istic reactions found, candidate genes be <br />                         ydcR (b1439)                          6.00E-15   predicted half them. The global or- <br /> R08553 (R)   IV         ysaA (b3573)                          4.00E-5    phan reactions. SMILEY suggested 15 existing <br />                                                                          model reactions reversible gaps. <br />                                                                          According EcoCyc, reactions not <br /> above, likely explanation false nega-                 reversible. However, evidence supporting the <br /> tives molybdenum cofactor strictly                    reversibility model reactions, DKGLCNR1 (2,5- <br /> required growth E. coli. Several solutions                  diketo-D-gluconate reductase) [41] DKGLCNR2y (2,5- <br /> added deleted reactions network,                   diketo-D-gluconate reductase (NADPH)) [42]. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                         Page 11 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Experimental validation predicted genes                    Table 10 Predicted genes gap-filling SMILEY solutions <br /> SMILEY gap-filling algorithms useful be-        Hypothesized changes directionality <br /> cause use model existing experimental          Reaction               Support <br /> data generate predictions. Without performing ex-       DOGULNR                reversible (EcoCyc) <br /> periment verify predictions, hy- <br />                                                               DKGLCNR1               reversible (Habrych et al. [41]) <br /> potheses. The iJO1366 model used design simple <br />                                                               DKGLCNR2y              reversible (Yum et al. [42]) <br /> growth phenotype experiments confirm these <br /> predictions. Each 1176 solutions added      PGLYCP                 reversible (EcoCyc) <br /> model time, growth simulated          CYSSADS <br /> combinations set 115 carbon sources 62 ni-        HMPK1                  reversible (EcoCyc) <br /> trogen sources aerobic anaerobic condi-        4HTHRS <br /> tions. These substrates selected readily <br />                                                               HETZK                  reversible (EcoCyc) <br /> available chemicals use laboratory. For every <br />                                                               NNDMBRT                reversible (EcoCyc) <br /> substrate combination growth predicted for <br /> the model SMILEY solution added,           ACONMT                 reversible (EcoCyc) <br /> the unmodified iJO1366 model, vivo experiment           CINNDO                 reversible (EcoCyc) <br /> can performed determine E. coli actually         MCPST <br /> grow substrates, giving supporting experi-         GPDDAS                 reversible (EcoCyc) <br /> mental evidence predicted reactions. For most <br />                                                               APCS                   reversible (EcoCyc) <br /> solutions, new growth conditions identified. All <br />                                                               SARCOX <br /> realistic solutions testable experimental conditions <br /> are listed Additional file 6.                              Hypothesized gap-filling reactions <br />   One reaction growth experiment pre-         Reaction               Putative gene                          E-value <br /> dicted possible R01184, myo-inositol:oxygen         R01742 (F)             ydiS (b1699)                           0.003 <br /> oxidoreductase. This reaction combines myo-inositol           R00893 (F)             ygfM (b0419)                           0.58 <br /> with oxygen form D-glucuronate water. Myo-in-          R02133 (F)             yhbO (b3153)                           0.069 <br /> ositol root-no consumption gap iJO1366            R02721 (F)             global orphan <br /> model, reaction fills gap. With reaction <br />                                                               R03472 (R)             global orphan <br /> included, E. coli predicted grow myo-inositol <br /> as substrate. An vivo experiment performed,          R01297 (R)             global orphan <br /> and wild-type E. coli inoculated 2 g/L myo-in-       R01299 (R)             global orphan <br /> ositol minimal media carbon sources.            R02252 (F)             fadH (b3081)                           7.00E-71 <br /> Three replicates performed, 72 h, cul-                            nemA (b1650)                           1.00E-22 <br /> tures reached OD600 0.017 &#194;&#177; 0.006. This result      R00895 (R)             aspC (b0928)* <br /> indicates E. coli grow slowly myo-in- <br />                                                               R03530 (F)             ndk (bb2518)* <br /> ositol carbon source. Next, candidate <br /> genes predicted reaction using BLASTp.          R00012 (F)             global orphan <br /> These genes ydeN (b1498), yfdE (b2371), yphC             R01232 (R)             yjhG (b4297)                           0.028 <br /> (b2545), yhiJ (b3488). The functions genes                              yagF (b0269)                           0.057 <br /> are currently unknown, non-essential. The        R00838 (F)             chbF (b1734)                           9.00E-42 <br /> Keio Collection knockout strains genes                                melA (b4119)                           2.00E-21 <br /> were obtained grown glucose myo- <br />                                                               R00655 (R)             global orphan <br /> inositol minimal media. All strains grew similar <br /> OD600 wild-type E. coli glucose, myo-inosi-      R07300 (F)             global orphan <br /> tol, knockout strains did grow (Figure 4).    R00683 (F)             global orphan <br /> The yhiJ knockout strain did grow all, indicating      R00367 (F)             global orphan <br /> this gene likely codes myo-inositol:oxygen oxidore-     R02559 (F)             global orphan <br /> ductase E. coli.                                           R02560 (F)             global orphan <br />                                                               R05623 (F)             yjiN (b4336)                           0.16 <br /> Discussion <br /> In study, iJO1366 metabolic network model E. <br /> coli used discovery tool, leading predictions     coli metabolism structured format, compar- <br /> for new metabolic gene functions. The date         ing model predictions experimental data, errors and <br /> model represents current state knowledge E.         gaps knowledge identified. A large dataset <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                     Page 12 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> was assembled Keio Collection gene knockout phe-                  Through careful analysis false positive results, <br /> notypes grown 34 different substrates. These pheno-              possible model errors identified. In some <br /> types compared model predicted phenotypes,                  cases, model predicted growth pathways with <br /> and model false positive false negative predic-             poorly characterized genes required. These genes <br /> tions identified. When analyzed, false positive            need investigated detail. If future experi- <br /> predictions indicated possible errors                mental evidence shows encode the <br /> current model, including pathways thought cata-               enzymes currently believed encode, the <br /> lyzed poorly studied enzymes, uncertain                  model updated removing reactions. False <br /> requirements E. coli biomass formation. The false                negative results, hand, indicate errors <br /> negative cases indicated potential errors           experimental design, model. Several <br /> the biomass likely experimental errors.          cases identified E. coli grow min- <br /> Importantly, false negative cases indicated          imal media despite lacking genes synthesis es- <br /> the model currently incomplete. The SMILEY algo-                 sential cofactors. Biotin thiamin known be <br /> rithm used predict likely missing reac-             essential, substrates growth observed <br /> tions, novel procedure, feasibility           knockout strains able produce <br /> predictions assessed comparisons ex-             cofactors. One possible explanation there <br /> perimental data model gap analysis. Gene                alternate synthesis routes compounds, but <br /> predictions feasible predicted               cases, likely trace amounts bio- <br /> reactions, experimental evidence generated               tin thiamin present media the <br /> support predicted function gene yhiJ.                    growth experiments [23,25]. Both false positive false <br />                                                                     negative results indicate potential errors iJO1366 <br />                                                                     core biomass reaction. False positives indicate gene <br />                                                                     helps produce essential compound missing <br />                                                                   biomass reaction, false negatives indicate <br />                   2                                                 gene produces non-essential compound is <br />                                                                     included. These biomass components essen- <br />                 1.6 <br />                                                                     tial non-essential certain conditions, however, <br />                                                                     necessitating use condition-specific biomass reac- <br />        OD 600 <br />  <br />  <br />  <br />  <br />                 1.2 <br />                                                                     tions specific model applications. <br />                 0.8                                                    SMILEY used predict gap-filling false <br />                                                                     negative correcting reactions added the <br />                 0.4 <br />                                                                     metabolic network. Some reactions same <br />                   0 <br />                                                                     existing model reactions opposite direction. <br />                        WT   &#206;&#8221;ydeN   &#206;&#8221;yfdE    &#206;&#8221;yphC     &#206;&#8221;yhiJ        Literature data searched confirm refute pre- <br />                                     Strain                          dictions, supporting evidence several <br />                                                                     reactions. Other SMILEY solutions predicted addition <br />                                                                     completely new reactions network. All gap-filing <br />    b <br />                                                                     solutions feasible false negative correcting <br />                 0.03 <br />                                                                     solutions inspected manually, potentially real- <br />             0.025                                                   istic reactions, genes predicted based protein se- <br />              0.02 <br />                                                                     quence homology. The feasible predicted reactions <br />    OD 600 <br />  <br />  <br />  <br />  <br />                                                                     cover wide range metabolic functions. Many them <br />             0.015 <br />                                                                     corrected false negative predictions aspC knockout <br />              0.01                                                   strains, model unable produce L-aspar- <br />                                                                     tate. Several predicted new reactions involving TCA <br />             0.005 <br />                                                                     cycle intermediates. New reactions predicted <br />                  0                                                  metabolism adenosine 5&#226;&#8364;&#8482;-phosphosulfate, <br />                        WT   &#206;&#8221;ydeN   &#206;&#8221;yfdE    &#206;&#8221;yphC     &#206;&#8221;yhiJ <br />                                                                     dehydroglycine, dTMP, glycoaldehyde, L-isoleucine, <br />                                     Strain <br />                                                                     5-phosphoribosyl-5-carboxyaminoimidazole. <br />  Figure 4 Growth Keio Collection gene knockout stains <br />                                                                        The new workflow presented theoretically <br />  identify possible myo-inositol:oxygen oxidoreductase. (a)     applied organism metabolic net- <br />  Growth strains WT glucose M9 minimal media.     work reconstruction available. The requirement <br />  (b) Growth strains WT myo-inositol M9 minimal   fairly large set experimental phenotypes, <br />  media. Three replicate measurements performed.                growth different substrates growth <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                         Page 13 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> with gene knockouts, available. It possible   Methods <br /> that workflow applied using meta-     Comparison model predictions experimental data <br /> bolic network gaps, case organisms     The experimental gene essentiality data obtained <br /> extensive experimental data. A similar study utilizing   publications [10,23-25], &#226;&#8364;&#339;essential&#226;&#8364;? or <br /> only gaps phenotypes performed       &#226;&#8364;&#339;non-essential&#226;&#8364;? designations assigned original <br /> human metabolic network [13]. Despite number         studies used. Several corrections essentiality <br /> of potentially useful predictions made, SMILEY did       assignments based updated analysis of <br /> not solutions nearly half cases       Keio Collection [26]. The newly identified essential <br /> which run. Part reason    genes added lists essential genes all <br /> the universal set reactions used based         conditions, genes essentiality identi- <br /> KEGG [37]. This database contains reactions    fied uncertain changed original <br /> are known exist organism,     designations. <br /> so completely undiscovered reactions              The iJO1366 E. coli K-12 MG1655 metabolic network <br /> added. Also, metabolite iJO1366      reconstruction loaded COBRA Toolbox <br /> connected KEGG reactions. Of 1133 compart-        [44], adjusted match phenotype E. coli <br /> ment-independent metabolites iJO1366, 203      BW25113, missing metabolic genes <br /> have KEGG compound IDs. A larger set reactions        (&#206;&#8221;araBAD, &#206;&#8221;rhaBAD, &#206;&#8221;lacZ). The associated reactions <br /> including model metabolites allow         isozymes (ARAI, RBK_L1, RMPA, LYXI, RMI, <br /> additional valid SMILEY solutions found. Many      RMK, LACZ) constrained carry zero flux. <br /> gene predictions based sequence hom-        All model reactions retained default bounds <br /> ology, reactions, gene pre-     [10]. Minimal media simulated setting lower <br /> dicted reference sequence           bound &#226;&#710;&#8217;1000 (allowing unlimited uptake) ex- <br /> available. These global orphan reactions [43],       change reactions Ca2+, Cl-, CO2, Co2+, Cu2+, Fe2+, <br /> which known gene organism. The pro-       Fe3+, H+, H2O, K+, Mg2+, Mn2+, MoO2-         +     2+ <br />                                                                                                 4 , Na , Ni , NH4, <br />                                                                                                                   - <br />                                                                    2-    2-     2-    2-     2-          2+ <br /> liferation global orphans (estimated 30-40 %    O2, HPO4 , SeO4 , SeO3 , SO4 , WO4 , Zn . A lower <br /> of known enzymatic functions [6]) makes gene         bound &#226;&#710;&#8217;0.01 placed cob(I)alamin exchange <br /> function prediction difficult, account   reaction. Each knockout strain modeled using the <br /> fact well-studied organism E.     deleteModelGenes function constrain correct <br /> coli, uncharacterized genes.        reactions zero. Model growth phenotypes deter- <br />                                                          mined using FBA core biomass reaction the <br /> Conclusions                                              objective, time condition. Strains with <br /> This study utilized genome-scale metabolic network     growth rates zero classified non-essential, <br /> reconstruction tool analysis high-       strains growth rates zero classified as <br /> throughput experimental data. The ultimate result     essential. The Tomlab (Tomlab Optimization Inc., Se- <br /> this study number valuable predictions      attle, WA) linear programming solver used per- <br /> have made. Some predictions        form FBA. <br /> adjustments iJO1366 model, pre- <br /> dicted changes core biomass reaction          Computational prediction gap-filling reactions <br /> changes GPRs reactions IG3PS, I2FE2SS,     The COBRA Toolbox 2.0 implementation the <br /> I2FE2SS2, S2FE2SS, S2FE2SS2. Corroborating lit-      SMILEY algorithm (growthExpMatch) used pre- <br /> erature evidence        dict sets gap-filling reactions false negative <br /> predictions, incorporated fu-     model comparison. The universal database reactions <br /> ture updates reconstruction. The predic-    obtained KEGG Release 58.0 [37]. All reactions <br /> tions study gene functions,    set listed &#226;&#8364;&#339;incomplete reaction&#226;&#8364;? black- <br /> both isozymes reactions currently        listed, excluded possible SMILEY solutions. Any <br /> known occur E. coli. These predictions provide     reaction compound appearing a <br /> hypotheses experimentally tested. As      substrate product blacklisted, with <br /> example, prediction missing myo-inositol:oxy-   reactions identified initial tests (R00090, <br /> gen oxidoreductase reaction led design       R00113, R00274) forming unrealistic energy gen- <br /> simple experiment previously uncharac-      erating reaction loops existing iJO1366 model reac- <br /> terized gene yhiJ essential growth   tions. The minimum growth threshold required the <br /> on myo-inositol. We expect        SMILEY algorithm 0.05 h-1. Up 25 alternate solu- <br /> predictions study likewise serve     tions allowed, single solution time limit of <br /> hypotheses experimental analysis.                    2 h. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                 Page 14 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />    When SMILEY run gaps instead false nega-           sodium phosphate dibasic, 3.0 g/L potassium phosphate <br /> tive cases, producing consuming reaction            monobasic, 0.5 g/L sodium chloride, 0.24 g/L magnesium <br /> each gap metabolite identified iJO1366             sulfate, 0.011 g/L calcium chloride), trace elements (0.1 g/L <br /> model. A lower bound 0.01 mmol/gDW/h applied             iron (III) chloride, 0.02 g/L zinc sulfate, 0.004 g/L copper <br /> to reaction, time, SMILEY used         chloride, 0.01 g/L manganese sulfate, 0.006 g/L cobalt <br /> predict gap-filling reactions. For gaps demand        chloride, 0.006 g/L disodium EDTA), Wolfe&#226;&#8364;&#8482;s Vitamin <br /> reactions model, demand reactions con-          Solution. Wild-type E. coli strains the <br /> strained zero running SMILEY. The Tomlab              Keio Collection yphC (JW5842), yfdE (JW2368), ydeN <br /> mixed-integer linear programming solver used.               (JW5243), yhiJ (JW3455) gene knockouts (supplied by <br />                                                                 Open Biosystems) grown overnight LB. The next <br /> Computational feasibility analysis predictions           day, 15 mL culture centrifuged 4000 rpm for <br /> After predicting sets false negative correcting          8 min, supernatant discarded, culture was <br /> gap-filling reactions, sets solution reac-     resuspended M9 salt solution carbon nitro- <br /> tions added iJO1366 model time.             gen sources. The culture centrifuged resuspended <br /> The growth phenotype of strains            new M9 times completely wash LB. <br /> 13,470 experimental data conditions predicted          Next, 1 &#206;&#188;L washed E. coli cultures used in- <br /> using FBA, threshold zero determining             oculate 10 mL aerobic myo-inositol M9 cultures, which <br /> growth growth. The number new false positives          grown 37&#194;&#176;C. The optical density 600 nm was <br /> for solution determined number             measured points growth. <br /> conditions true negatives wild-type <br /> model grow new reactions                Additional files <br /> added. The number corrected false negatives each <br /> solution number false negatives           Additional file 1: Experimental computational growth phenotypes <br />                                                                  13,470 conditions. <br /> true positives new reactions added. Gap- <br />                                                                  Additional file 2: All false negative conditions gaps which <br /> Find run iJO1366 model set          SMILEY run. <br /> solution reactions added it, time. The set        Additional file 3: The KEGG based universal database reactions and <br /> network gaps compared set gaps ori-         list excluded reactions. <br /> ginal model determine gaps eliminated.            Additional file 4: All Category I-IV SMILEY solutions. <br />   In order determine SMILEY solutions             Additional file 5: All gap-filling SMILEY solutions. <br /> be tested simple vivo experiments, FBA used          Additional file 6: Media conditions indicate presence of <br /> to test growth iJO1366 model solution           predicted reactions. <br /> added set 115 carbon sources 62 nitrogen <br /> sources aerobic anaerobic conditions.            Competing interests <br />                                                                 The authors declare competing interests. <br /> The growth unmodified iJO1366 model first <br /> tested condition using FBA. Next, model             Acknowledgements <br /> with solution added, time, tested      We like thank Pep Charusanti, Tom Conrad, Adam Feist for <br />                                                                 helpful comments insights. This work funded US <br /> 3896 conditions unmodified model pre-              National Institutes Health grant GM057089. <br /> dicted growth. Conditions modified ver- <br /> sions model grow used design               Authors&#226;&#8364;&#8482; contributions <br />                                                                 JDO BOP designed study. JDO performed computational <br /> experiments.                                                    predictions, analyzed results, performed experimental analysis, and <br />                                                                 wrote manuscript. JDO BOP revised manuscript. Both authors <br />                                                                 approved content final manuscript. <br /> Experimental validation predicted genes <br /> Candidate genes SMILEY predicted reaction sets <br /> were predicted using bi-directional protein BLAST               Received: 16 January 2012 Accepted: 1 May 2012 <br />                                                                 Published: 1 May 2012 <br /> (BLASTp) gene organism in <br /> KEGG E. coli K-12 MG1655 genome. Protein <br />                                                                 References <br /> sequences organisms phylogenically close          1. Edwards JS, Palsson, B. .: Metabolic flux balance analysis in <br /> to E. coli used possible. The gene               silico analysis Escherichia coli K-12 gene deletions. BMC bioinformatics <br /> highest BLAST expectation value (E-value)                 2000, 1(1). <br />                                                                 2. Edwards JS, Covert M, Palsson B : Metabolic modeling microbes: the <br /> reported. When multiple genes E-values              flux-balance approach. Environmental Microbiology 2002, 4(3):133&#226;&#8364;&#8220;140. <br /> below 10-13, reported candidate genes.              3. Price ND, Papin JA, Schilling CH, Palsson B : Genome-scale microbial in <br />   To test growth E. coli myo-inositol carbon       silico models: constraints-based approach. Trends biotechnology <br />                                                                     2003, 21(4):162&#226;&#8364;&#8220;169. <br /> and energy source, 2 g/L myo-inositol M9 media         4. Orth JD, Thiele I, Palsson B : What flux balance analysis? Nature <br /> and filter sterilized. This media contained M9 salts (6.8 g/L       biotechnology 2010, 28(3):245&#226;&#8364;&#8220;248. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                                      Page 15 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> 5.    Reed JL, Patel TR, Chen KH, Joyce AR, Applebee MK, Herring CD, Bui OT,              allantoin nitrogen linked glyoxylate metabolism Escherichia coli. <br />       Knight EM, Fong SS, Palsson B : Systems approach refining genome                 Journal bacteriology 1999, 181(24):7479&#226;&#8364;&#8220;7484. <br />       annotation. Proceedings National Academy Sciences United     29.   Portnoy VA, Herrgard MJ, Palsson B : Aerobic fermentation D-glucose <br />       States America 2006, 103(46):17480&#226;&#8364;&#8220;17484.                                        evolved cytochrome oxidase-deficient Escherichia coli strain. <br /> 6.    Karp PD: Call enzyme genomics initiative. Genome biology 2004,               Appl Environ Microbiol 2008, 74(24):7561&#226;&#8364;&#8220;7569. <br />       5(8):401.                                                                     30.   Klem TJ, Davisson VJ: Imidazole glycerol phosphate synthase: the <br /> 7.    Pouliot Y, Karp PD: A survey orphan enzyme activities.                           glutamine amidotransferase histidine biosynthesis. Biochemistry 1993, <br />       BMC bioinformatics 2007, 8:244.                                                     32(19):5177&#226;&#8364;&#8220;5186. <br /> 8.    Orth JD, Palsson B : Systematizing generation missing metabolic        31.   Neidhardt FC (ed.): Escherichia coli Salmonella: cellular molecular <br />       knowledge. Biotechnology bioengineering 2010, 107(3):403&#226;&#8364;&#8220;412.                   biology, 2nd edn. Washington, D.C.: ASM Press; 1996. <br /> 9.    Satish Kumar V, Dasika MS, Maranas CD: Optimization based automated           32.   Lin S, Hanson RE, Cronan JE: Biotin synthesis begins hijacking fatty <br />       curation metabolic reconstructions. BMC bioinformatics 2007, 8:212.              acid synthetic pathway. Nature chemical biology 2010, 6(9):682&#226;&#8364;&#8220;688. <br /> 10.   Orth JD, Conrad TM, Na J, Lerman JA, Nam H, Feist AM, Palsson B : A           33.   Bettendorff L, Wins P: Thiamin diphosphate biological chemistry: new <br />       comprehensive genome-scale reconstruction Escherichia coli                       aspects thiamin metabolism, especially triphosphate derivatives acting <br />       metabolism-2011. Molecular systems biology 2011, 7:535.                             cofactors. FEBS J 2009, 276(11):2917&#226;&#8364;&#8220;2925. <br /> 11.   Reed JL, Vo TD, Schilling CH, Palsson B : An expanded genome-scale            34.   Patton AJ, Hough DW, Towner P, Danson MJ: Does Escherichia coli possess <br />       model Escherichia coli K-12 (iJR904 GSM/GPR). Genome biology 2003,               second citrate synthase gene? European journal biochemistry/FEBS <br />       4(9):R54.51&#226;&#8364;&#8220;R54.12.                                                                 1993, 214(1):75&#226;&#8364;&#8220;81. <br /> 12.   Bochner BR, Gadzinski P, Panomitros E: Phenotype microarrays              35.   Gerike U, Hough DW, Russell NJ, Dyall-Smith ML, Danson MJ: Citrate <br />       high-throughput phenotypic testing assay gene function.                      synthase 2-methylcitrate synthase: structural, functional and <br />       Genome research 2001, 11(7):1246&#226;&#8364;&#8220;1255.                                              evolutionary relationships. Microbiology (Reading, England) <br /> 13.   Rolfsson O, Palsson B , Thiele I: The human metabolic reconstruction                1998, 144 (Pt 4):929&#226;&#8364;&#8220;935. <br />       Recon 1 directs hypotheses novel human metabolic functions. BMC            36.   Choi-Rhee E, Cronan JE: A nucleosidase required vivo function of <br />       systems biology 2011, 5:155.                                                        S-adenosyl-L-methionine radical enzyme, biotin synthase. Chem Biol <br /> 14.   Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, Palsson         2005, 12(5):589&#226;&#8364;&#8220;593. <br />       B : Global reconstruction human metabolic network based             37.   Kanehisa M, Goto S, Furumichi M, Tanabe M, Hirakawa M: KEGG for <br />       genomic bibliomic data. Proceedings National Academy                  representation analysis molecular networks involving diseases <br />       Sciences United States America 2007, 104(6):1777&#226;&#8364;&#8220;1782.                    drugs. Nucleic acids research 2010, 38(Database issue):D355-360. <br /> 15.   Kumar VS, Maranas CD: GrowMatch: automated method reconciling          38.   Karsten WE, Viola RE: Kinetic studies L-aspartase Escherichia coli: <br />       silico/in vivo growth predictions. PLoS computational biology 2009,              pH-dependent activity changes. Archives biochemistry biophysics <br />       5(3):e1000308.                                                                      1991, 287(1):60&#226;&#8364;&#8220;67. <br /> 16.   Zomorrodi AR: Maranas CD. Improving iMM904 S. cerevisiae metabolic        39.   MacKintosh C, Nimmo HG: Purification regulatory properties of <br />       model using essentiality synthetic lethality data. BMC systems biology          isocitrate lyase Escherichia coli ML308. The Biochemical journal 1988, <br />       2010, 4:178.                                                                        250(1):25&#226;&#8364;&#8220;31. <br /> 17.   Paley SM, Karp PD: Evaluation computational metabolic-pathway              40.   Keseler IM, Bonavides-Martinez C, Collado-Vides J, Gama-Castro S, Gunsalus <br />       predictions Helicobacter pylori. Bioinformatics (Oxford, England) 2002,         RP, Johnson DA, Krummenacker M, Nolan LM, Paley S, Paulsen IT et al: <br />       18(5):715&#226;&#8364;&#8220;724.                                                                      EcoCyc: comprehensive view Escherichia coli biology. Nucleic acids <br /> 18.   Green ML, Karp PD: A Bayesian method identifying missing enzymes                research 2009, 37(Database issue):D464-470. <br />       predicted metabolic pathway databases. BMC bioinformatics 2004, 5:76.      41.   Habrych M, Rodriguez S, Stewart JD: Purification identification an <br /> 19.   Green ML, Karp PD: Using genome-context data identify specific types             Escherichia coli beta-keto ester reductase 2,5-diketo-D-gluconate <br />       functional associations pathway/genome databases. Bioinformatics              reductase YqhE. Biotechnol Prog 2002, 18(2):257&#226;&#8364;&#8220;261. <br />       (Oxford, England) 2007, 23(13):i205-211.                                      42.   Yum DY, Lee BY, Pan JG: Identification yqhE yafB genes <br /> 20.   Christian N, May P, Kempa S, Handorf T, Ebenhoh O: An integrative                   encoding 2, 5-diketo-D-gluconate reductases Escherichia coli. <br />       approach completing genome-scale metabolic networks.                        Appl Environ Microbiol 1999, 65(8):3341&#226;&#8364;&#8220;3346. <br />       Molecular bioSystems 2009, 5(12):1889&#226;&#8364;&#8220;1903.                                   43.   Osterman A, Overbeek R: Missing genes metabolic pathways: a <br /> 21.   Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, Broadbelt           comparative genomics approach. Curr Opin Chem Biol 2003, 7(2):238&#226;&#8364;&#8220;251. <br />       LJ, Hatzimanikatis V, Palsson B&#195;&#732;: A genome-scale metabolic reconstruction     44.   Schellenberger J, Que R, Fleming RM, Thiele I, Orth JD, Feist AM, Zielinski <br />       Escherichia coli K-12 MG1655 accounts 1260 ORFs                    DC, Bordbar A, Lewis NE, Rahmanian S, et al: Quantitative prediction of <br />       thermodynamic information. Molecular systems biology 2007, 3(121).                  cellular metabolism constraint-based models: COBRA Toolbox <br /> 22.   Boyle NR, Morgan JA: Flux balance analysis primary metabolism                 v2.0. Nature protocols  <br /> </body></html>