<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Grassi Tramontano BMC Evolutionary Biology 2011, 11:301 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Horizontal vertical growth S. cerevisiae <br /> metabolic network <br /> Luigi Grassi1 Anna Tramontano1,2* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: The growth development biological organism reflected metabolic network, the <br />   evolution relies essential gene duplication mechanism. There current views the <br />   evolution metabolic networks. The retrograde model hypothesizes pathway evolves recruiting novel <br />   enzymes direction opposite metabolic flow. The patchwork model instead based assumption <br />   evolution based exploitation broad-specificity enzymes capable catalysing variety of <br />   metabolic reactions. <br />   Results: We analysed well-studied unicellular eukaryotic organism, S. cerevisiae, studied effect the <br />   removal paralogous gene products metabolic network. Our results, obtained using different paralog and <br />   network definitions, that, initial period gene duplication instrumental expanding <br />   metabolic space, reached equilibrium subsequent gene duplications used source of <br />   specialized enzymes source novel reactions. We switch the <br />   evolutionary strategies S. cerevisiae dated 350 million years ago. <br />   Conclusions: Our data, obtained novel analysis methodology, strongly supports hypothesis the <br />   patchwork model better explains recent evolution S. cerevisiae metabolic network. Interestingly, <br />   effects patchwork strategy acting Euascomycete-Hemiascomycete divergence still <br />   detectable today. <br />  <br />  <br /> Background                                                                           networks assign metabolites nodes pro- <br /> Metabolism defines reactions provide energy                                 cessing enzymes edges. <br /> and constituents cells organisms. It includes                              Although metabolic networks are, doubt, the <br /> the features related growth development                               studied biological networks, models are <br /> of living organism. The modular units metabo-                               appropriate explaining evolutionary history is <br /> lism metabolic pathways, sets chemical reac-                              matter debate. Two main models been <br /> tions metabolite transformed                              presented literature evolution: retro- <br /> through series steps catalysed enzymes. The evo-                             grade model patchwork model (for recent <br /> lutionary history enzymes necessarily reflects evo-                           review [1]). In retrograde model, pathways <br /> lution organism belong                                  evolve backwards key metabolite [2]. According <br /> constitutes interesting subject study.                                         model, enzymes recruited direction <br />    The metabolic networks provide global view                               opposite metabolic flow environment <br /> metabolic pathways organism repre-                                  assumed rich metabolites initial <br /> sented alternative ways. In enzyme-centric net-                               key metabolites intermediates. This abundance of <br /> works, proteins defined nodes, connected                               initial intermediate metabolites difficult recon- <br /> metabolites process, whilst metabolite-centric                                  cile network evolution occur when <br />                                                                                      organic molecules depleted environment. <br /> * Correspondence: anna.tramontano@uniroma1.it <br />                                                                                      The patchwork model formulated aim of <br /> 1 <br />  Physics Department, Sapienza University Rome, Piazzale Aldo Moro, 5 I-           overcoming conceptual limitation [3,4]. This alter- <br /> 00185 Roma, Italy                                                                    native model hypothesizes important role broad- <br /> Full list author information available end article <br />  <br />                                        &#194;&#169; 2011 Grassi Tramontano; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                        reproduction medium, provided original work properly cited. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                              Page 2 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> specificity enzymes (capable catalysing variety       Results <br /> metabolic reactions) forging evolution meta-        Evolutionary perspective <br /> bolic pathways. These enzymes, thanks broad          In study propose alternative perspective for <br /> substrate specificities, involved meta-      study evolutionary history metabolic net- <br /> bolic pathways synthesis similar       works. Our primary focused novelty of <br /> metabolites. In context, gene duplications         enzymes progressively added network. The <br /> bring selective advantage pathways: increased     metabolic space defined set the <br /> level enzyme cause increase      metabolites present metabolic network. The addi- <br /> key metabolites. Finally, specialization differ-   tion new enzyme metabolic network can <br /> ent pathways occur result specialization     essentially consequences: i) creation a <br /> paralogous enzymes.                                           new functionality (i.e. new metabolic reaction); ii) the <br />   Previous analyses showed paralogous enzymes            duplication functionality present the <br /> tend closer metabolic network expected         network. <br /> by chance [5,6]. Although result interpreted        The innovation brought enzymes progressively <br /> as confirmation retrograde model, studies      added metabolic network related evolu- <br /> [7,8] showed homologous enzymes dif-        tionary debate neutralism selectionism, ele- <br /> ferent pathways pathway,          gantly reconciled analysis Wagner [15,16]. <br /> in accordance patchwork model. The current             The addition new enzyme processes new <br /> way reconcile different findings postulate     metabolites lead expansion metabolic <br /> complementarity evolutionary models,         space, reflected horizontal growth meta- <br /> are commonly accepted non mutually exclusive            bolic network (Figure 1A). Alternatively vertical <br /> [9].                                                          growth metabolic network realised time <br />   In study propose new approach study       enzyme able process metabolites present <br /> of evolution metabolic networks based           metabolic space added network (Figure <br /> evaluation effects gene duplication a <br /> given network. As test case, selected S. cerevisiae <br /> metabolic network. <br />   S. cerevisiae studied eukaryotic mono-cellular <br /> organism information metabolic <br /> reactions accumulated reported thousands <br /> of papers included publicly available data- <br /> bases. Several studies confirmed S. cerevisiae <br /> has experienced genome duplication (WGD) <br /> around 100 million years ago [10-12]. Moreover, the <br /> results careful study [13] traced evolutionary <br /> history protein-coding genes seventeen genomes <br /> of Ascomycota fungi identified orthologs para- <br /> logs different periods duplication. <br />   The results analysis reported the <br /> S. cerevisiae metabolic network evolved discon- <br /> tinuous manner: growing moves increased the <br /> number metabolites. Subsequently, newly added <br /> enzymes did increase number metabolites, <br /> but specialized specific cellular compartments <br /> and/or cell cycle phase or, alternatively, included <br /> in existing complexes. The datation duplications        Figure 1 Network growth models. The duplication proteins <br /> also indicates shift network evolution            involved metabolic reactions results growth metabolic <br /> occurred Euascomycete-Hemiascomycete                network. Paralogous pairs process different metabolites increase <br />                                                                number nodes lead horizontal expansion the <br /> divergence, estimated happened 350 mil-         network, depicted A. Paralogs catalysing same <br /> lion years ago [14]. This discontinuity growing         reactions, example different conditions, result a <br /> modality, results discussed here,          specialized expression proteins lead vertical <br /> strongly supports hypothesis patchwork            growth metabolic network number nodes <br /> model played main role forging evolution       remaining constant (part B). The colour scale goes red (less <br />                                                                recent duplications) blue (more recent duplications). <br /> S. cerevisiae metabolic network. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                              Page 3 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> 1B). We studied effects progressive addition       <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> paralogs led removal 152 178 <br /> of paralogous enzymes metabolic network             edges KEGG YBP networks, respectively. <br /> evaluating effects growth metabolic          We consistently obtained result repli- <br /> space.                                                        cates: collapsed networks includes large majority <br />                                                               metabolites empirical network. As be <br /> Network analysis                                              seen inset Figure 2, reports percen- <br /> In order evaluate contribution paralogous           tage metabolites YBP network collapsed <br /> enzymes metabolic network S. cerevisiae          according alignment paralog <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> with <br /> collapsed network maintaining member          respect empirical one, 99% 100% of <br /> of paralogous family (collapsed network). Next,       metabolites preserved (see Additional file 1, Fig- <br /> compared properties metabolic space         ure S1 data obtained using different metabolic net- <br /> original network 1000 collapsed networks        work paralogy definitions). <br /> (i.e. networks paralogous enzymes                The effect removal node network is <br /> removed randomly selected, Methods). The             strictly dependent connectivity degree, i.e. on <br /> analysis performed replicates starting        number links nodes. <br /> two independent metabolic network databases            Figure 2 shows metabolites empirical <br /> paralogous family definitions. The metabolite-centric         1000 collapsed networks equivalent connectiv- <br /> networks reconstructed linking products           ity distributions. This result independent choice <br /> reagents processing enzymes (see Methods).        paralogous families reference meta- <br /> The initial networks Yeast Biochemical           bolic network (see Additional file 1, Figure S2) indi- <br /> Pathways (YBP), manually curated corrected ver-         cates paralogous proteins similar connectivity. <br /> sion Metacyc database [17,18] available the <br /> Saccharomyces Genome Database (SGD) [19,20]             Functional overlap paralogous proteins <br /> network reconstructed according reactions pre-         In metabolic networks examination, enzymes <br /> sent KEGG database [21,22]. The paralogous             represented edges. In order define func- <br /> families defined according strategies. The      tional properties enzyme listed pairs of <br /> first based comparison protein pairs.       metabolites (nodes) connects. Each reaction con- <br /> All pairs proteins aligned using EMBOSS           sidered once, regardless reversibility, e.g. the <br /> needle program [23] 30%              reaction A &#226;&#8225;&#338; B listed AB, BA (see <br /> sequence identity connected paralogy net-         Metabolic Network Reconstruction section Methods). <br /> work (see Methods). The connected components          We compared paralogous pairs present the <br /> network defined protein families and, consequently, <br /> members family identified paralo- <br />                                                                            0 <br /> gous proteins. The analysis repeated using               10 <br /> paralogous families retrieved ENSEMBL compara                                                          Empirical network <br />                                                                                                             Collapsed networks <br /> [24] study Wapinski et al. [13].                           -1 <br />                                                                       10 <br />    As test, compared metabolic space the <br /> empirical network collapsed networks <br />                                                                P(k) <br />  <br />  <br />  <br />  <br /> obtained removal paralogous proteins. The aim                  -2 <br />                                                                       10 <br /> of comparison understand gene                                       100 <br />  <br />  <br />  <br /> duplication contributes vertical horizontal <br />                                                                                 Metabolites (%) <br />  <br />  <br />  <br />  <br />                                                                            -3 <br /> growth metabolic network. Clearly, metabolic               10 <br /> space collapsed network smaller than <br /> that empirical paralogous enzymes                   -4 <br />                                                                                               99 <br />  <br />  <br />                                                                       10        0                       1                          2 <br /> would add new metabolites network. This                       10                      10                            10 <br /> not case, example, paralogous enzymes                                                 k <br /> would responsible catalysing            Figure 2 Effects paralog removal metabolic network. <br /> reaction different compartments different             The main figure reports connectivity distributions P(k) for <br />                                                                metabolites empirical network (red squares) 1000 <br /> conditions.                                                    collapsed networks (blue dots). The distributions very <br />    Collapsing paralogs according global sequence        similar. In inset boxplot reporting fraction metabolites <br /> alignment led removal 311 198 edges          empirical network present 1000 collapsed networks is <br /> the KEGG YBP networks, respectively. When              shown. Dashed lines indicate mean, upper lower box <br /> ENSEMBL <span id='am-85' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> paralogs used, corre-            margins correspond standard deviation (SD), whiskers <br />                                                                indicate SD. <br /> sponding figures 265 113 edges. The Wapinski <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                             Page 4 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> metabolic network order quantify functional                                 similar amino acid sequences (72% identity 84% <br /> overlap calculating relative Dice coefficient [25].                              similarity). They catalyze conversion orotate <br /> Given paralogous enzymes E1 E2, Dice M                                      orotidine-5&#226;&#8364;&#8482;-monophosphate. URA5 accounts for <br /> coefficient defined twice number overlapping                               80% orotate phosphoribosyltransferase <br /> metabolite pairs (intersection) divided total num-                               activity yeast cells (major isozyme) URA10 <br /> ber metabolite pairs (union): Dice M = Dice = 2                                      responsible remaining 20% (minor isozyme) <br /> (X1&#226;&#710;&#169;X2)/(X1&#226;&#710;&#170;X2) X1 X2 metabolite                                      [26,27]. Interestingly, expression patterns <br /> pairs E1 E2. This coefficient equal zero                               mildly anti-correlated accordance the <br /> enzymes functional overlap, whilst equal                               results Zhang et al., according URA5 <br /> one totally functionally overlapping enzymes.                                       transcription down-regulated dimethyl sulfoxide <br />   Figure 3A shows normalized distributions                                   URA10 expression up-regulated the <br /> Dice M coefficient paralogous pairs (defined                                    conditions [28]. The WGD paralogs YDL131W <br /> through paralogy alignment network) enzymes                                   (LYS21) YDL182W (LYS20) constitute case <br /> the YBP network million randomly                                         isozymes totally overlapping function. They have <br /> assorted pairs. Most paralogous pairs (~80%)                                 identical amino acid sequences (87% identity and <br /> made enzymes performing exactly reaction                                    90% similarity) catalyze condensation acetyl- <br /> and remaining fraction formed equal                                  CoA alpha-ketoglutarate form homocitrate the <br /> number paralogous enzymes functional over-                                   lysine biosynthesis pathway. A recent study [29] showed <br /> lap (~10%) paralogous enzymes partial                                     condition growth ethanol, homocitrate is <br /> functional overlap (~10%). This result, confirmed                                mainly synthesized LYS21, fermen- <br /> replicates (see Additional file 1, Figure S3), indicates                                tative metabolism, LYS21 LYS20, play redundant <br /> that enzymes identified paralogs                                    roles. <br /> have contributed vertical growth S. cerevi- <br /> siae metabolic network, whilst small fraction con-                               Paralogous proteins catalyzing different reactions <br /> tributed horizontal growth.                                                      Although vast majority paralogous pairs made <br />   Additional file 2 reports paralogous pairs ana-                               isozymes totally overlapping functions a <br /> lyzed study. For pair calculated                               small fraction (~10%) paralogs catalyzing different <br /> Pearson correlation coefficient expression                                reactions. This interesting minority constitutes few <br /> levels using data SGD&#226;&#8364;&#8482;s Expression Con-                                    horizontal growing moves S. cerevisiae metabolic <br /> nection tool (http://www.yeastgenome.org/cgi-bin/                                       network, here. <br /> expression/expressionConnection.pl).                                                       Table 1 reports pairs enzymes identified as <br />   An interesting well-studied example paralogous                                 paralogs paralogy assignments no <br /> enzymes total functional overlap reported fig-                               functional overlap metabolic network. A <br /> ure 3B. The isozymes YMR271C (URA 10)                                               examination pairs immediately highlights <br /> YML106W (URA5) WGD paralogs [11]                                          enzymatic similarity proteins. Indeed one <br />  <br />  <br />                          10 <br />                                                       Randomly selected pairs <br />                                                       Paralogous pairs <br />  Normalized occurrence <br />  <br />  <br />  <br />  <br />                           8 <br />  <br />  <br />                           6 <br />  <br />  <br />                          4 <br />  <br />  <br />                           2 <br />  <br />  <br />                           0 <br />                               0   0.2   0.4           0.6        0.8            1 <br />                                               DiceM <br />   A                                                                                 B <br />  Figure 3 Functional overlap paralogous pairs. (A) DiceM coefficient paralogous pairs (red bars). Grey bars refer 106 randomly <br />  selected pairs. (B) Scheme pathway section including isozymes YMR271C (URA10) e YML106W (URA5) sharing 75% amino acid <br />  similarity responsible reaction, conversion orotate orotidine-5&#226;&#8364;&#8482;-phosphate. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                                       Page 5 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> Table 1 Paralogous pairs divergent functions <br /> Paralogous pairs                    Paralogy <span id='am-86' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>                          DiceM coefficient               Correlation expression levels <br /> Enzyme 1          Enzyme 2          Need          Ens            Wap            YBP            KEGG              Pearson coeff.                 P value <br /> YBR149W           YHR104W           Y             Y                N             0              N/A                    0.68                      2.2-16 <br /> YDL080C           YDR380W           Y             Y                Y            0.67             0                     0.08                      0.033 <br /> YDL246C           YLR070C           Y             Y                Y            N/A              0                      N/A                       N/A <br /> YDR147W           YLR133W           Y             Y                N             0              0.5                    0.09                       0.02 <br /> YDR380W           YGR087C           Y             Y                Y            0.54             0                     -0.07                     0.065 <br /> YDR380W           YLR044C           Y             Y                Y            0.54             0                     0.08                      0.047 <br /> YDR380W           YLR134W           Y             Y                Y            0.54             0                     0.01                       0.76 <br /> YDR399W           YJR133W           Y             Y                Y             0              N/A                    0.25                      7.6-11 <br /> YHR123W           YNL130C           Y             Y                Y             0               0                     0.50                      2.2-16 <br /> YJR159W           YLR070C           Y             Y                Y            N/A              0                     0.38                      2.2-16 <br /> Only pairs identified paralogs paralogy definitions (Need: sequence identity 30%; Ens: derived Ensembl database; Wap: <br /> listed Wapinski et al. [13] ) DiceM coefficient equal 1 metabolic network shown. The complete set data shown in <br /> Additional file 2. <br />  <br />  <br /> pair enzymes having exactly EC                               PDC1, PDC5 PDC6 exclusively mapped <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> number, pairs partial overlap (one                        &#226;&#8364;&#339;valine degradation&#226;&#8364;?, &#226;&#8364;&#339;acetoin biosynthesis II&#226;&#8364;?, &#226;&#8364;&#339;buta- <br /> the enzymes EC number                               nediol biosynthesis&#226;&#8364;? &#226;&#8364;&#339;glucose fermentation&#226;&#8364;? <br /> digit) pairs EC numbers differing                         pathways. <br /> in digit. This indicates paralogous                          Some studies showed ARO10 the <br /> pairs, defined divergent network ana-                         PDC1, PDC5 PDC6 isozymes share substrates <br /> lysis, similar enzymatic point view,                         process specific substrates [31,32]. <br /> their difference level substrates                         We investigated growth metabolic <br /> they process. YBR149W (ARA1) arabinose dehy-                               network occurs continuous fashion. In words, <br /> drogenase catalyzes oxidation D-arabinose,                           asked different modalities meta- <br /> L-xylose, L-fucose L-galactose presence                            bolic network growth happened different evolutionary <br /> NADP+; YHR104W (GRE3) paralog aldose                                   periods. To answer question investigated the <br /> reductase reduces cytotoxic compound methyl-                            duplication age paralogous enzymes. <br /> glyoxal (MG) (R)-lactaldehyde, purified                            Following reported Wapinski et. al [13], <br /> GRE3 catalyze reduction xylose xyli-                        divided paralogous pairs groups: i) pre- <br /> tol. In YBP proteins mapped different                           WGD Euascomycetes paralogous pairs, ancient <br /> pathways: ARA1 dehydro-D-arabinono-1,4-lac-                               duplications shared species Euascomycetes <br /> tone biosynthesis GRE3 xylose metabolism.                             class; ii) pre-WGD Hemiascomycetes paralogous pairs, <br /> YBR149W present KEGG reaction database                             including duplications WGD excluding <br /> and DiceM coefficient relative KEGG                         Hemiascomycetes class; iii) WGD paralogous pairs; <br /> network computed. The YLR044C (PDC1),                                  iv) post-WGD paralogous pairs. <br /> YGR087C (PDC6) YLR134W (PDC5) isozymes                                     Six pairs DiceM coefficient equal to <br /> have pyruvate decarboxylase activity PDC1 acting                            zero (obtained using YBP network definition) belong <br /> as major decarboxylase [30]. They paralogs                        pre-WGD Euascomycetes group (hypergeometric <br /> YDR380W (ARO10), phenylpyruvate decarboxylase                                  test P value = 0.0007). A confirmation given <br /> that catalyzes decarboxylation phenylpyruvate                          analysis network derived KEGG, where <br /> phenylacetaldehyde [31]. In KEGG, PDC1, PDC5                                 pairs Dice M coefficient equal to <br /> PDC6 mapped &#226;&#8364;&#339;Glycolysis/Gluconeogen-                              zero belong pre-WGD Euascomycetes group <br /> esis&#226;&#8364;? pathway, whilst paralog functional                           (hypergeometric test P value = 0.0005). <br /> overlap ARO10 &#226;&#8364;&#339;Phenylalanine metabolism&#226;&#8364;?                                 This result indicates paralogous pairs <br /> pathway. In YBP enzymes partial functional                            functional overlap derive ancient <br /> overlap, DiceM coefficient equal 0.53. All                        duplications, dated Wapinski et al. [13] the <br /> proteins perform reaction &#226;&#8364;&#339;isoleucine                            Euascomycete-Hemiascomycete divergence, estimated to <br /> degradation&#226;&#8364;?, &#226;&#8364;&#339;phenylalanine degradation&#226;&#8364;? &#226;&#8364;&#339;trypto-                           occurred 350 million years ago [14]. <br /> phan degradation&#226;&#8364;? pathways, ARO10                              Paralogous enzymes functional overlap gener- <br /> mapped &#226;&#8364;&#339;leucine degradation&#226;&#8364;? pathway                                  ally belong different pathways opposite is <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                    Page 6 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> true enzymes catalysing reaction. For        expected chance and, time display a <br /> enzyme pairs shown Table 1, calculated            strong chemical similarity (essentially catalyse simi- <br /> DiceP coefficient (see Methods). Most pairs       lar reactions). However, results used to <br /> a DiceP coefficient different 1 indicating     discriminate retrograde patchwork <br /> tend exclusively belong single pathway (further       model. In scenario, neighbourhood dupli- <br /> information available Additional file 3).              cated enzymes interpreted proof step- <br />    Another confirmation finding derives      wise growth metabolic network consequent a <br /> calculation mean shortest paths        stepwise addition enzymes. On hand the <br /> metabolites (reagents products) connected          enzymatic similarity paralogous enzymes be <br /> pairs enzymes shown Table 1 (see Additional        interpreted supporting patchwork model, accord- <br /> file 3). According measure, hypothetical pair    ing broad specific enzymes tend catalyse <br /> enzymes connecting contiguous metabolites (i.e.           chemically similar reactions acting differ- <br /> products reaction catalysed enzyme          ent types substrates [43]. These studies mapped the <br /> the reagents reaction catalysed other)      paralogous enzymes enzyme-centric metabolic net- <br /> a mean shortest path equal close 1. The mean            work, problem short distance <br /> shortest path network derived YBP 7.7 &#194;&#177;       totally overlapping enzymes dis- <br /> 3.3 6.8 &#194;&#177; 1.8 KEGG network. This indicates        tinguished signal resulting contiguous <br /> that horizontal growing moves tend add             enzymes. For reason, selected approach <br /> enzymes distant points S. cerevisiae metabolic      problem using metabolite-centric reconstruction <br /> network.                                                      metabolic network, informative. <br />                                                               For example, let consider totally overlapping <br /> Discussion                                                    paralogs E1 E2, processing reaction <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Gene duplication generally accepted impor-       metabolite A metabolite B let assume that <br /> tant source raw material functional innovation         A B processed different <br /> [33-36]. In S. cerevisiae, studies [37-40] pointed       metabolite C. In enzyme-centric network two <br /> out key role gene duplication evolution      enzymes distance equal 2, our <br /> the regulatory networks, particularly loss,       approach, enzymes E1 E2 DiceM coef- <br /> gain rewiring regulatory interactions occurring        ficient equal 1 considered totally <br /> after duplication. Our study aims clarifying       overlapping paralogs. <br /> effective contribution gene duplication evolu-        Our results strongly support conclusion the <br /> tion metabolic networks. This theme treated         vast majority paralogous pairs nowadays detectable in <br /> many authors attempt identifying          S. cerevisiae similar E1 E2 hypothetical <br /> best model able explain evolution metabolic         enzymes contribute horizontal expan- <br /> networks. A study Tsoka Ouzounis [41] reports          sion metabolic network. <br /> that enzymes family distributed               This conclusion robust independent the <br /> among different pathways. Similarly, Teichmann et al.         definition metabolic network and/or method <br /> [8] homologous enzymes twice likely         used paralogy assignment. <br /> to different pathways path-        In fact, metabolic space remains unchanged <br /> way. These pioneering studies interestingly converge       removal paralogous enzymes con- <br /> assigning high likelihood patchwork model,           nected metabolites, consistently previous studies <br /> nevertheless limitations: analyse E.      demonstrating strong conservation biochemical <br /> coli, prokaryotic organism widespread          function S. cerevisiae duplicated genes [13,44,45]. <br /> phenomenon lateral gene transfer, particularly affect-       Furthermore, Gash et al. demonstrated S. cerevi- <br /> ing proteins involved metabolism [42], compro-         siae isozymes different transcriptional regulation <br /> mise reliability homology inference.               specific members isozyme families responding to <br /> Furthermore studies analyse metabolism        environmental stress [46] study Ihmels et al. <br /> from network viewpoint misleading           [47] elegantly showed isozyme multiplicity a <br /> since metabolic reactions cells, mapped     strategy adopted S. cerevisiae permit differential <br /> different pathways, close network. Subse-       regulation reactions shared different processes. <br /> quent studies analysed metabolism network          Such strategy provides possibility indepen- <br /> perspective. Alves et al. [5] D&#195;&#173;az-Mej&#195;&#173;a et al. [6]       dent control associated reactions response to <br /> analysed metabolic networks 12 species E.       pathway-specific requirements. <br /> coli, respectively, reaching similar conclusions: homolo-       Although paralogous enzymes isozymes <br /> gous enzymes tend closer network            completely overlapping functions, small <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                    Page 7 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> fraction (~10% pairs) paralogs catalysing totally       space large gene duplications dated <br /> non-overlapping reactions. The characterization      Euascomycete-Hemiascomycete divergence. <br /> divergent paralogs useful uncover evolutionary      Almost subsequent duplications gave raise to <br /> process definition metabolic network        enzymes performing reaction, specialized <br /> of S. cerevisiae. Interestingly,       different cellular localizations differently regu- <br /> derived ancient duplications, dated              lated. This result suggests patchwork model is <br /> Wapinski et al. [13] Euascomycete-Hemiasco-        compatible retrograde model the <br /> mycete divergence (about 350 million years ago). This         growth modality metabolic network S. cerevi- <br /> result indicates discontinuity growth modalities     siae. A confirmation given the <br /> of S. cerevisiae metabolic network, meta-        finding paralogs partial func- <br /> bolic space remaining unaltered unaltered       tional overlap far away metabolic network. <br /> 350 million years.                                            In conclusion, described method be <br />   Interestingly, growth strategy accordance        useful studying contribution paralogous <br /> with theoretical prediction appearance        enzymes metabolism S. cerevisiae be <br /> enzymes occurs bursts phyletic gradu-       applied evolutionary study metabolism of <br /> alism proposed Sch&#195;&#188;tte et al. [48].                        organisms. <br />   This result reaching network equili- <br /> brium, growing moves add          Methods <br /> existing enzymatic reactions creating novel       Metabolic Networks Reconstruction <br /> functionalities. The differentiation broad-specificity     All analyses performed study relied two <br /> enzymes hypothesized patchwork model             metabolic networks: YBP network KEGG <br /> have event contributed reach            network. Both metabolite-centric networks, recon- <br /> equilibrium, enzymes added net-        structed connecting substrate products through <br /> work specialized performing specific      processing enzymes. The networks represented <br /> reaction.                                                     directed graphs nodes denote compounds and <br />   Although expect that, retrograde      edges denote proteins. The directionality edges <br /> model, equilibrium state reached,         reflects direction reaction. Reactions catalyzed <br /> for example, possible biochemical reactions      protein complexes represented including the <br /> had explored, second scenario,         reaction member complex. Highly <br /> expect signs retrograde model. The         connected hubs source noise [49], therefore <br /> horizontal growing moves result dupli-          restricted analysis main reactants both <br /> cated enzymes close metabolic network,       networks, excluding currency metabolites the <br /> this case: paralogs partial    analysis. <br /> or functional overlap mapped different path-           The YBP network derived SGD [19,20] (http:// <br /> ways mean distance metabolites pro-       downloads.yeastgenome.org/curation/literature/archive/ <br /> cess short expect according     yeastcyc14.0.tar.201009.gz). <br /> retrograde model.                                                We referred SGD Pathway Tools version 14.0. The <br />   Our results suggest methodology used          file pathways.dat contains details reac- <br /> here powerful straightforward way studying       tion present database (enzymes, substrates, pro- <br /> the effect paralogous enzymes metabolism         ducts, directionality). We used files <br /> an organism unravel evolutionary strategy.            protcplxs.col enzrxns.dat. The detailed list <br />                                                               protein complexes components, <br /> Conclusions                                                   whilst describes enzyme features (gene <br /> The purpose analysis evaluate contri-       name, modality action dimer/monomer the <br /> bution paralogous enzymes growth         catalyzed reaction). Only reactants classified &#226;&#8364;&#339;pri- <br /> metabolic network. In order used sim-         maries&#226;&#8364;? taken account. <br /> ple strategy: comparison empirical metabolic          The KEGG network derived KEGG <br /> network collapsed networks, obtained         PATHWAY Database [21,22]. The reaction file (version <br /> random removal paralogous enzymes. This                    04/03/2011) contains information reac- <br /> approach, applied specific case S. cerevisiae,      tions (substrate/product reversibility). Only the <br /> gave interesting results. We obtain results          RPAIRS (reactant pairs) classified KEGG &#226;&#8364;&#339;main&#226;&#8364;? <br /> regardless paralog network definitions used:       pairs taken account. The enzyme-reaction <br /> the metabolic network S. cerevisiae grew discontinu-       associations retrieved file sce01100.xml (ver- <br /> ously. The moves able expand metabolic             sion 04/05/2011). <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                     Page 8 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br />   The names compounds glycans                Comparison expression paralogous enzymes <br /> retrieved compound file glycan file          The similarity transcriptional regulation paralo- <br /> derived KEGG LIGAND database.                        gous enzymes assessed Pearson&#226;&#8364;&#8482;s cor- <br />                                                               relation coefficient, calculated using data available at <br /> Paralogy detection                                            SGD expression connection tool (http://www.yeast- <br /> All analyses performed using paralogy          genome.org/cgi-bin/expression/expressionConnection. <br /> definitions. A paralogy network defined consider-      pl). This includes 29 datasets obtained different <br /> ing proteins S. cerevisiae nodes. All pre-      experiments. We joined data present pre- <br /> dicted ORFs sequences S. cerevisiae genome        clustered files (in log2 space) global expression <br /> downloaded Ensembl release 59 (http://aug2010.           table. From table removed columns (experi- <br /> archive.ensembl.org/) [24]. The amino acid sequence           ments) 5% missing genes lines <br /> identity computed using needle, global alignment        (genes) 5% missing experiments. The fil- <br /> algorithm included EMBOSS suite programs            tered table, containing 5294 genes 658 experiments, <br /> [23]. Pairs 30% sequence identity         used calculate Pearson&#226;&#8364;&#8482;s correlation coeffi- <br /> considered paralogs. The paralogy assign-           cient paralogous pairs. <br /> ments derived ENSEMBL compara, release <br /> 59 (http://aug2010.archive.ensembl.org/) [24]        Additional material <br /> the results analysis reported Wapinski et al. <br /> [13].                                                          Additional file 1: Supplementary Figures. Supplementary figures <br />                                                                reporting data obtained using alternative definitions metabolic <br />                                                                network and/or paralogous pairs. <br /> Network analysis <br />                                                                Additional file 2: Supplementary Table 1. Detailed list the <br /> Networks analysed using Cytoscape freeware            studied paralogous pairs. For pair paralogy assignment all <br /> (version 2.8.0) [50]. The calculation connectivity      methods, DiceM coefficient relative networks and <br /> was using in-house developed Perl script.              Pearson correlation coefficient expression levels the <br />                                                                relative P values reported. <br />   All properties empirical networks com- <br />                                                                Additional file 3: Supplementary Table 2. Paralogous pairs or <br /> pared randomly collapsed networks. For           partial functional overlap. The table reports DiceP coefficient the <br /> each paralogous family formed enzymes          shortest path <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> pairs listed Table 1. <br /> present metabolic network, enzyme, ran- <br /> domly chosen, kept (together <br /> with connected nodes) removed. The Short-          Abbreviations <br /> estPath finder plugin Cytoscape (http://csresources.      SGD: Saccharomyces Genome Database; YBP: Yeast Biochemical Pathways; <br /> sourceforge.net/ShortestPath/) used calculate      WGD: genome duplication. <br /> shortest path length metabolites.                   Acknowledgements <br />                                                               The authors like thank Domenico Raimondo useful <br /> Dice coefficient                                              discussions. <br />                                                               This work partially supported Award No. KUK-I1-012-43 by <br /> The functional overlap paralogs measured          King Abdullah University Science Technology (KAUST), Fondazione <br /> by Dice coefficient [25], using enzyme                Roma Italian Ministry Health, contract no.onc_ord 25/07, FIRB <br /> properties.                                                   ITALBIONET PROTEOMICA. <br />    For E1 E2 enzymes, coefficient             Author details <br /> defined (DiceM) twice number shared metabo-         1 <br />                                                                Physics Department, Sapienza University Rome, Piazzale Aldo Moro, 5 I- <br /> lite pairs sum metabolite pairs processed      00185 Roma, Italy. 2Istituto Pasteur - FondazioneCenci Bolognetti, Sapienza <br />                                                               University Rome, Piazzale Aldo Moro, 5 I-00185 Roma, Italy. <br /> the enzymes (DiceP) twice number com- <br /> mon pathways divided number pathways             Authors&#226;&#8364;&#8482; contributions <br /> which enzymes involved:                               AT LG designed project wrote paper. LG performed the <br />                                                               analyses. All authors read approved final manuscript. <br />    Dice = 2(X1 &#226;&#710;&#169; X2)(X1 &#226;&#710;&#170; X2) <br />                                                               Received: 12 May 2011 Accepted: 14 October 2011 <br />                                                               Published: 14 October 2011 <br />   Where X1 X2 number metabolite pairs <br /> or pathways, respectively.                                    References <br />   The reversibility reactions taken       1. Yamada T, Bork P: Evolution biomolecular networks: lessons from <br /> account. The order metabolites substrate           metabolic protein interactions. Nat Rev Mol Cell Biol 2009, 10:791-803. <br />                                                               2. Horowitz NH: On Evolution Biochemical Syntheses. Proc Natl Acad <br /> &#194;&#174; product (KEGG) Left &#194;&#174; Right (YBP). Pathway                   Sci USA 1945, 31:153-157. <br /> definitions obtained KEGG YBP                   3. Jensen RA: Enzyme recruitment evolution new function. Annu Rev <br /> Metacyc.                                                          Microbiol 1976, 30:409-425. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                                             Page 9 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> 4.    Ycas M: On earlier states biochemical system. J Theor Biol 1974,        29. Quezada H, Aranda C, DeLuna A, Hern&#195;&#161;ndez H, Calcagno ML, Mar&#195;&#173;n- <br />       44:145-160.                                                                        Hern&#195;&#161;ndez A, Gonz&#195;&#161;lez A: Specialization paralogue LYS21 <br /> 5.    Alves R, Chaleil RA, Sternberg MJ: Evolution enzymes metabolism:           determines lysine biosynthesis respiratory metabolism in <br />       network perspective. J Mol Biol 2002, 320:751-770.                                 Saccharomyces cerevisiae. Microbiology 2008, 154:1656-1667. <br /> 6.    D&#195;&#173;az-Mej&#195;&#173;a JJ, P&#195;&#169;rez-Rueda E, Segovia L: A network perspective          30. Konig S: Subunit structure, function organisation pyruvate <br />       evolution metabolism gene duplication. Genome Biol 2007, 8:R26.              decarboxylases various organisms. Biochim Biophys Acta 1998, <br /> 7.    Copley RR, Bork P: Homology (betaalpha)(8) barrels: implications             1385:271-286. <br />       evolution metabolic pathways. J Mol Biol 2000, 303:627-641.         31. Vuralhan Z, Morais MA, Tai SL, Piper MD, Pronk JT: Identification and <br /> 8.    Teichmann SA, Rison SC, Thornton JM, Riley M, Gough J, Chothia C: The              characterization phenylpyruvate decarboxylase genes in <br />       evolution structural anatomy small molecule metabolic                   Saccharomyces cerevisiae. Appl Environ Microbiol 2003, 69:4534-4541. <br />       pathways Escherichia coli. J Mol Biol 2001, 311:693-708.                    32. Dickinson JR, Salgado LE, Hewlins MJ: The catabolism amino acids to <br /> 9.    Schmidt S, Sunyaev S, Bork P, Dandekar T: Metabolites: helping hand          long chain complex alcohols Saccharomyces cerevisiae. J Biol <br />       pathway evolution? Trends Biochem Sci 2003, 28:336-341.                            Chem 2003, 278:8028-8034. <br /> 10.   Dietrich FS, Voegeli S, Brachat S, Lerch A, Gates K, Steiner S, Mohr C,        33. Ohno S: Evolution gene duplication London: George Alien &amp; Unwin Ltd. <br />       P&#195;&#182;hlmann R, Luedi P, Choi S, et al: The Ashbya gossypii genome tool           Berlin, Heidelberg New York: Springer-Verlag; 1970. <br />       mapping ancient Saccharomyces cerevisiae genome. Science               34. Conant GC, Wolfe KH: Turning hobby job: duplicated genes <br />       2004, 304:304-307.                                                                 new functions. Nat Rev Genet 2008, 9:938-950. <br /> 11.   Kellis M, Birren BW, Lander ES: Proof evolutionary analysis ancient     35. Hughes AL: The evolution functionally novel proteins gene <br />       genome duplication yeast Saccharomyces cerevisiae. Nature 2004,             duplication. Proc Biol Sci 1994, 256:119-124. <br />       428:617-624.                                                                   36. Prince VE, Pickett FB: Splitting pairs: diverging fates duplicated <br /> 12.   Wolfe KH, Shields DC: Molecular evidence ancient duplication             genes. Nat Rev Genet 2002, 3:827-837. <br />       entire yeast genome. Nature 1997, 387:708-713.                             37. Conant GC: Rapid reorganization transcriptional regulatory <br /> 13.   Wapinski I, Pfeffer A, Friedman N, Regev A: Natural history                    network genome duplication yeast. Proc Biol Sci 2010, <br />       evolutionary principles gene duplication fungi. Nature 2007,                 277:869-876. <br />       449:54-61.                                                                     38. Conant GC, Wolfe KH: Functional partitioning yeast co-expression <br /> 14.   Galagan JE, Henn MR, Ma LJ, Cuomo CA, Birren B: Genomics fungal             networks genome duplication. PLoS Biol 2006, 4:e109. <br />       kingdom: insights eukaryotic biology. Genome Res 2005,                    39. Fusco D, Grassi L, Bassetti B, Caselle M, Cosentino Lagomarsino M: Ordered <br />       15:1620-1631.                                                                      structure transcription network inherited yeast whole- <br /> 15.   Wagner A: Robustness evolvability: paradox resolved. P R Soc B               genome duplication. BMC Syst Biol 2010, 4:77. <br />       2008, 275:91-100.                                                              40. Ward JJ, Thornton JM: Evolutionary models formation network <br /> 16.   Wagner A: Neutralism selectionism: network-based reconciliation.             motifs modularity Saccharomyces transcription factor <br />       Nat Rev Genet 2008, 9:965-974.                                                     network. PLoS Comput Biol 2007, 3:1993-2002. <br /> 17.   Caspi R, Altman T, Dale JM, Dreher K, Fulcher CA, Gilham F, Kaipa P,           41. Tsoka S, Ouzounis CA: Functional versatility molecular diversity of <br />       Karthikeyan AS, Kothari A, Krummenacker M, et al: The MetaCyc database             metabolic map Escherichia coli. Genome Res 2001, 11:1503-1510. <br />       metabolic pathways enzymes BioCyc collection                 42. Jain R, Rivera MC, Lake JA: Horizontal gene transfer genomes: the <br />       pathway/genome databases. Nucleic Acids Res 2010, 38:D473-479.                     complexity hypothesis. Proc Natl Acad Sci USA 1999, 96:3801-3806. <br /> 18.   Karp PD, Ouzounis CA, Moore-Kochlacs C, Goldovsky L, Kaipa P, Ahr&#195;&#169;n D,         43. Light S, Kraulis P: Network analysis metabolic enzyme evolution in <br />       Tsoka S, Darzentas N, Kunin V, L&#195;&#179;pez-Bigas N: Expansion BioCyc              Escherichia coli. BMC Bioinformatics 2004, 5:15. <br />       collection pathway/genome databases 160 genomes. Nucleic Acids           44. Grassi L, Fusco D, Sellerio A, Cor&#195;&#160; D, Bassetti B, Caselle M, Lagomarsino MC: <br />       Res 2005, 33:6083-6089.                                                            Identity divergence protein domain architectures yeast <br /> 19.   Dwight SS, Balakrishnan R, Christie KR, Costanzo MC, Dolinski K, Engel SR,         whole-genome duplication event. Mol Biosyst 2010, 6:2305-2315. <br />       Feierbach B, Fisk DG, Hirschman J, Hong EL, et al: Saccharomyces genome        45. Guan Y, Dunham MJ, Troyanskaya OG: Functional analysis gene <br />       database: underlying principles organisation. Brief Bioinform 2004,            duplications Saccharomyces cerevisiae. Genetics 2007, 175:933-943. <br />       5:9-22.                                                                        46. Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz G, <br /> 20.   Engel SR, Balakrishnan R, Binkley G, Christie KR, Costanzo MC, Dwight SS,          Botstein D, Brown PO: Genomic expression programs response of <br />       Fisk DG, Hirschman JE, Hitz BC, Hong EL, et al: Saccharomyces Genome               yeast cells environmental changes. Mol Biol Cell 2000, 11:4241-4257. <br />       Database provides mutant <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> data. Nucleic Acids Res 2010, 38:           47. Ihmels J, Levy R, Barkai N: Principles transcriptional control the <br />       D433-436.                                                                          metabolic network Saccharomyces cerevisiae. Nat Biotechnol 2004, <br /> 21.   Kanehisa M, Goto S: KEGG: kyoto encyclopedia genes genomes.                 22:86-92. <br />       Nucleic Acids Res 2000, 28:27-30.                                               <br /> </body></html>