<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Toxicon. Author manuscript; available PMC 2012 August 30. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Toxicon. 2010 November ; 56(6): 1025&#226;&#8364;&#8220;1034. doi:10.1016/j.toxicon.2010.07.007. <br />  <br />  <br />  <br />                            Pseudomonas Exotoxin kills Drosophila S2 cells Apoptosis <br />                            Ashima K. Sharma David Fitzgerald1 <br />                            Biotherapy Section, Laboratory Molecular Biology, Center Cancer Research, National <br />                            Cancer Institute. NIH, HHS. 37 Convent Dr, Room 5124, Bethesda, MD 20892 <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                  Pseudomonas exotoxin A (PE) cytotoxic eukaryotic cells enters cells receptor- <br />                                  mediated endocytosis, translocates cell cytosol ADP-ribosylates elongation factor 2 <br />                                  (EF2). However, interaction toxin eukaryotic cells mechanism PE <br />                                  mediated cell death extensively characterized. The feasibility carrying a <br />                                  genome-wide RNAi screen, makes Drosophila melanogaster S2 cells good model <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                  identify essential genes PE-mediated cytotoxicity, provided suitable multi-well assay is <br />                                  developed. Here, using alamarBlue&#194;&#174; viability assay, Drosophila S2 cells are <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  sensitive PE picomolar concentrations toxin treatments provoke increase in <br />                                  caspase activity. This prompted use RNAi characterize mechanism cell death. <br />                                  Results indicated PE-mediated death S2 cells dependent presence of <br />                                  diphthamide, post translational modification EF2, presence Drice, terminal <br />                                  caspase insect cells. RNAi drice chemical inhibition caspase action z-VAD-fmk <br />                                  protected cells PE-mediated death. Protection death RNAi z-VAD-fmk did not <br />                                  interfere toxin delivery cytosol leading inhibition protein synthesis. Using a <br />                                  convenient alamarBlue&#194;&#174; assay, data confirms cytotoxicity PE S2 cells and <br />                                  establishes apoptosis mode PE-mediated death. This confirms suitability of <br />                                  Drosophila cells convenient simple model elucidate role specific genes and <br />                                  proteins required PE action. <br />  <br />  <br />                            Keywords <br />                                  Pseudomonas exotoxin A; apoptosis; diphthamide; RNAi; Drice <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            1. Introduction <br />                                              The pathogenesis P. aeruginosa, Gram-negative soil organism leading cause of <br />                                              infections cystic fibrosis patients, burn victims immunocompromised individuals, has <br />                                              studied variety phylogenetically diverse model hosts including: vertebrates, <br />                                              insects, nematodes, mycetozoans plants. (Mahajan-Miklos et al., 2000). Pseudomonas <br />                                              exotoxin A (PE) soluble exoprotein toxic virulence factors amongst <br />                                              secreted proteins important pathogenesis Pseudomonas aeruginosa <br />                                              (Pollack, 1983). Mutations toxA gene, encoding PE, resulted attenuated virulence <br />                                              plants (Rahme et al., 1995), nematodes (Tan et al., 1999) mice (Nicas Iglewski, <br />  <br />  <br />  <br />  <br />                            1 <br />                             To correspondence directed. djpf@helix.nih.gov. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Sharma Fitzgerald                                                                                           Page 2 <br />  <br />  <br />                                                1985) suggesting extensive conservation mechanism PE mediates injury, in <br />                                                evolutionarily diverse hosts. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Bacterial toxins studied gain insights pathogenic mechanisms. They also <br />                                                studied probes cellular function components novel therapies. In case of <br />                                                latter, usual modify toxin allow targeting specific cell types. Because <br />                                                toxins targeted kill cancer cells, investigating pathways mechanisms cell <br />                                                death important. Antibody-toxin fusion proteins, called immunotoxins, have <br />                                                valuable therapy targeted treatment cancer (Pastan et al., 2006). <br />                                                Immunotoxins exploit precision antibodies lethality protein toxins target <br />                                                kill cancer cells expressing specific cell surface proteins. PE-derived immunotoxins are <br />                                                evaluated anti-cancer agents human trials (Hassan et al., 2007, Kreitman et al., <br />                                                2009, Kreitman et al., 2001, Kreitman et al., 1999, Rand et al., 2000). Results trials <br />                                                reflect divergent <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> hematological malignancies respond positively, <br />                                                including complete responses, epithelial tumors exhibit sensitivity. Whether <br />                                                differences reflect problems gaining access individual cells tumor mass or <br />                                                differences susceptibility death established. This prompted detailed <br />                                                investigation mechanism cell death caused PE PE immunotoxins. <br />                                                Moreover, mammalian cells, investigating mechanism cell death highlighted <br />                                                possible involvement distinct pathways including apoptosis, autophagy and <br />                                                necrosis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                While, evidence involvement apoptosis PE mediated killing in <br />                                                mammalian cells (Hafkemeyer et al., 1999, Jenkins et al., 2004, Keppler-Hafkemeyer et al., <br />                                                2000, Komatsu et al., 1998), mechanisms PE-mediated cell death hosts not <br />                                                widely studied. The genes involved control execution apoptosis are <br />                                                conserved evolution (Cashio et al., 2005, Twomey McCarthy, 2005). <br />                                                However, actual molecular mechanisms used genes vary species to <br />                                                species. There high degree genetic redundancy mammalian systems made <br />                                                difficult identify specific gene functions trace exact apoptotic pathway. The <br />                                                fruit fly, D. melanogaster, provides intermediate complexity, sharing many <br />                                                mammalian components pathways, redundancy. Thus genetic <br />                                                conservation, low redundancy, feasibility high-throughput genetic screens have <br />                                                Drosophila cells valuable convenient model studying biochemical pathways. <br />                                                Here chosen Drosophila S2 cells model tissue culture the <br />                                                investigation interaction PE cells invertebrates eventually facilitating a <br />                                                better understanding PE PE immunotoxin-mediated cell death. <br />  <br />                                                PE 66KDa domain bacterial toxin. Domain I N-terminus binds LRP1 and/or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                LRP1b plasma membrane receptor cell entry (Kounnas et al., 1992, Pastrana et al., <br />                                                2005). Domain II harbors site furin processing poorly defined activity for <br />                                                translocation cell cytosol (Chiron et al., 1994, Jinno et al., 1989, Moehring et al., <br />                                                1993). Domain III ADP-ribosylating activity C-terminal KDEL-like sequence aids <br />                                                retrotranslocation ER cytosol (Chaudhary et al., 1990, Jackson et al., <br />                                                1999). PE mediates eukaryotic cell death blocking protein synthesis ADP- <br />                                                ribosylation cytosolic elongation factor 2 (EF2) (Morimoto Bonavida, 1992). <br />                                                However, protein synthesis inhibition toxins sufficient mediate target cell lysis <br />                                                (Morimoto Bonavida, 1992). Also, decreased ADP ribosylation activity does always <br />                                                result decreased PE-induced cytotoxicity, suggesting possible dissociation between <br />                                                protein synthesis inhibition cell death (Gallant et al., 2000). Thus, cell death mediated <br />                                                PE involve additional mechanisms. <br />  <br />  <br />  <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                          Page 3 <br />  <br />  <br />                                                The biochemical target PE diphthamide residue mammalian EF2. H715 is <br />                                                modified step enzymatic process resulting posttranslational modification that <br />                                                renders EF2 susceptible ADP-ribosylation bacterial toxins PE, diphtheria toxin <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                cholix toxin (Liu et al., 2004, Zhang et al., 2008). There endogenous enzyme <br />                                                modifies diphthamide poorly characterized. Toxin-mediated killing depends <br />                                                presence diphthamide mutants diphthamide pathway exhibit <br />                                                resistant phenotypes (Foley et al., 1992). While universal agreement ADP- <br />                                                ribosylation EF2 PE inhibits protein synthesis lead cell death, the <br />                                                mechanism universality death established. <br />  <br />                                                Cell death, mediated caspase-dependent apoptosis, driven set core proteins that <br />                                                evolutionarily conserved mammals insect cells (Hay Guo, 2006, <br />                                                Twomey McCarthy, 2005). Caspases cysteine proteases function both <br />                                                initiators executioners apoptotic cell death. In Drosophila, Dronc, caspase-9-like <br />                                                enzyme functions initiator caspase DCP-1 Drice, caspase-3-like effector <br />                                                caspases, critical executionery response. In absence apoptotic signals, the <br />                                                inhibitor apoptosis proteins (IAPs) Drosophila IAP1 (Diap1) bind caspases and <br />                                                inhibit activity. In presence apoptotic stimuli, pro-apoptotic proteins, Reaper, <br />                                                Hid Grim (RHG proteins) bind Diap1 release caspases. Released Dronc binds <br />                                                adaptor protein Ark form apoptosome subsequently cleaves activates <br />                                                effector caspases DCP-1 Drice. These caspases trigger series downstream <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                cleavage reactions resulting physiological changes cell characterized cell <br />                                                shrinkage, nuclear fragmentation, chromatin condensation DNA fragmentation. <br />  <br />                                                Here investigate PE-mediated death S2 Drosophila cells conclude apoptosis is <br />                                                mechanism death. <br />  <br />                               2. Materials Methods <br />                               2.1. Cell culture induction apoptosis <br />                                                S2 cells (S2-DRSC, stock #181) obtained Drosophila Genomics Resource <br />                                                Center, Indiana University, Bloomington, Indiana propagated M3 (Shields Sang <br />                                                medium) + BPYE media (Sigma-Aldrich, St. Louis, MO) supplemented heat <br />                                                inactivated 10% Fetal bovine serum (HyClone) 100 U/ml penicillin 100 &#206;&#188;g/ml <br />                                                streptomycin (Sigma-Aldrich 25&#194;&#176;C. To induce apoptosis, S2 cells treated 150 <br />                                                ng/ml actinomycin D (Sigma-Aldrich), 200 &#206;&#188;g/ml cycloheximide (Sigma-Aldrich) 30, 50 <br />                                                100 ng/ml PE, indicated. Caspase inhibition achieved including 100 &#206;&#188;M z- <br />                                                VAD-fmk (Calbiochem) medium PE cycloheximide, indicated. <br />                                                To probe specificity PE action, enzymatically inactive version PE, termed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                ntPE (for non-toxic PE), used competitor receptor uptake (FitzGerald et al., <br />                                                1998). <br />  <br />                               2.2. Cytotoxicity assays <br />                                                S2 cells seeded concentration 2 &#195;&#8212; 106 cells/ml 96 plates or <br />                                                indicated concentrations PE, ntPE, cycloheximide actinomycin D. 100 &#206;&#188;M <br />                                                z-VAD-fmk added caspase inhibition, indicated. S2 cells incubated for <br />                                                48 h 25&#194;&#176;C cell viability measured adding 10% alamarBlue&#194;&#174; (Invitrogen, <br />                                                Carlsbad, CA) followed incubation 1-5 h 37&#194;&#176;C measurement fluorescence at <br />                                                excitation wavelength 540nm emission wavelength 590nm using Cary <br />                                                eclipse fluorescence spectrophotometer (Varian). Fluorescence obtained untreated S2 <br />                                                cells set value 100, indicating 100% viability, variously treated S2 <br />                                                cells calculated accordingly. <br />  <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                       Page 4 <br />  <br />  <br />                               2.3. dsRNA synthesis <br />                                                gfp dsRNA synthesized using plasmid, pBC-S65T-GFP (kindly provided Dr. <br />                                                Bruce Patterson). Linear fragments pBC-S65T-GFP obtained digestion with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                restriction enzymes, EagI KpnI used templates dsRNA synthesis using the <br />                                                MEGASCRIPT T7 transcription kit (Applied Biosystems, Foster City, CA) or <br />                                                MEGASCRIPT T3 transcription kit (Applied Biosystems), respectively. The T7 RNA <br />                                                polymerase T3 RNA polymerase transcribed RNAs mixed heated at <br />                                                75&#194;&#176;C 5min followed cooling room temperature. The dsRNA subjected to <br />                                                DNase RNase (specific single stranded RNA) treatments 1 h 37&#194;&#176;C followed by <br />                                                purification sample using RNeasy MinElute columns (Ambion). Drosophila dph1 and <br />                                                drice amplified PCR using Drosophila S2 cDNA library template. To <br />                                                generate Drosophila S2 cDNA library, total RNA isolated using Trizol reagent <br />                                                (Invitrogen) followed reverse transcription using High Capacity cDNA Reverse <br />                                                Transcription Kit (Applied Biosystems). Drosophila dph1 amplified using primers, <br />                                                Dph1-For1 CTGTGATAATGGGCGATGTG Dph1-Rev1 <br />                                                GCAACGGGTTTGCTATCATT. In order T7 promoter site 5&#226;&#8364;&#8482; 3&#226;&#8364;&#8482; end of <br />                                                dph1, dph1 PCR product cloned Topo-pcr2.1 vector orientations <br />                                                resulting formation Topo-Dph1-For Topo-Dph1-Rev constructs. Similarly drice <br />                                                amplified using Drice-For primer, TAAACACGGACTTTCCCCAG Drice-Rev <br />                                                primer, CTTACGGTAGCTGGACGAGG cloned Topo-pcr2.1 orientations <br />                                                resulting Topo-Drice-For Topo-Drice-Rev constructs. PCR products obtained by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                amplifying using M13-For Dph1-For1 primers M13-For Dph1-Rev1 <br />                                                primers using M13-For Drice-For primers M13-For Drice-Rev <br />                                                primers used templates synthesizing dph1 drice dsRNA using the <br />                                                MEGASCRIPT RNAi kit (Applied Biosystems), respectively. 5ng dsRNA analyzed <br />                                                2% agarose gel electrophoresis ensure majority dsRNA migrated a <br />                                                single band. 350bp 444bp fragment dsRNAs produced corresponding the <br />                                                coding transcript drice dph1, respectively. The dsRNA stored -20&#194;&#176;C. <br />  <br />                               2.4. RNAi treatment Drosophila S2 cells <br />                                                For cytotoxicity protein synthesis inhibition assays, S2 cells plated 96 well <br />                                                plates, concentration 2&#195;&#8212;106 cells/ml serum free M3+BPYE media. For caspase 3 <br />                                                activity assays, S2 cells plated 24 plates, indicated concentration of <br />                                                6&#195;&#8212;106 cells/ml 3&#195;&#8212;106cells/ml serum free M3+BPYE media. 20&#206;&#188;g/ml dsRNA was <br />                                                mixed 2&#206;&#188;l cellfectin&#194;&#174; transfection reagent (Invitrogen) 50 &#206;&#188;l serum-free <br />                                                M3+BPYE media added S2 cells. The S2 cells mixed vigorously the <br />                                                dsRNA+cellfectin mixture. After incubating cells 4 hours 25&#194;&#176;C, equal volume of <br />                                                M3+BPYE media supplemented 20% FBS added final concentration of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                10% FBS supplemented M3+BPYE media. The cells incubated additional 48 h at <br />                                                25&#194;&#176;C allow turnover protein. For cytotoxicity assays, indicated concentration PE, <br />                                                cycloheximide, actinomycin D z-VAD-fmk added incubated 48 h 25&#194;&#176;C <br />                                                followed addition 10% alamarBlue&#194;&#174; (Invitrogen). <br />  <br />                               2.5. Protein Synthesis inhibition assay <br />                                                S2 cells (2&#195;&#8212;106 cells/ml) transfected dsRNA 96 plates followed by <br />                                                incubation presence indicated PE z-VAD-fmk 24 h. 1&#206;&#188;Ci {3H}- <br />                                                Leucine added incubated 4 h 25&#194;&#176;C. The plate incubated <br />                                                overnight -80&#194;&#176;C aid cell lysis. The effect protein synthesis assessed by <br />                                                measuring {3H}-Leucine incorporated. <br />  <br />  <br />  <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                          Page 5 <br />  <br />  <br />                               2.6. Caspase activity Assay <br />                                                S2 cells transfected dsRNA followed treatment indicated concentrations PE <br />                                                actinomycin D 24 h washed thoroughly PBS. Caspase activity was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                determined using caspase-3 fluorometric-kit (Invitrogen). Specific caspase activity was <br />                                                determined dividing total fluorescence units protein present the <br />                                                cell lysate sample. Protein concentration estimated using BCA&#226;&#8222;&#162; <br />                                                Protein Assay Kit (Pierce). The specific caspase activity untreated sample set to <br />                                                1.0 toxin treated samples calculated accordingly order fold <br />                                                increase caspase activity. <br />  <br />                               2.7. Western Blot analysis <br />                                                Equal number S2 cells transfected gfp dsRNA, drice dsRNA untransfected cells <br />                                                dislodged media washed thoroughly PBS. The cells incubated on <br />                                                ice 20min RIPA lysis buffer (5mM Tri pH 7.5, 150mM NaCl, 5mM EDTA, 1% NP40 <br />                                                1%, 0.1% SDS, 1% Sodium Deoxycholate, complete Protease inhibitor). The homogenates <br />                                                passed 28 G half inch needle times centrifuged 14000g for <br />                                                10min supernatants stored -20&#194;&#176;C western blots. Lysates <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                untransfected transfected samples boiled 5min Laemmli buffer and <br />                                                electrophoresed 8-16% Tris Glycine SDS-PAGE gels (Invitrogen). The proteins were <br />                                                transferred Immobilon-P nitrocellulose membrane (Millipore) probed using rabbit <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                anti-Drice (kindly provided Dr. Andreas Bergman), diluted 1:1,000 rabbit anti-&#206;&#179;- <br />                                                tubulin (Sigma-Aldrich) used protein loading control, diluted 1:3,000. The <br />                                                immobilized proteins incubated HRP labeled secondary antibodies (Jackson <br />                                                immunoresearch laboratories, inc.) signal detected LumiGLO Peroxidase <br />                                                Chemiluminescent Substrate Kit (KPL) <br />  <br />                               2.8. Reverse transcription real time PCR <br />                                                S2 cells seeded 24 wells concentration 2&#195;&#8212;106 cells/ml transfected with <br />                                                10ug/ml gfp dph1 dsRNA supplemented 5&#206;&#188;l cellfectin transfection reagent. RNA <br />                                                isolated S2 cells TRIzol (Invitrogen) extraction according manufacturer&#226;&#8364;&#8482;s <br />                                                instructions. Each RNA sample treated 16 units RNase free DNase I <br />                                                (Qiagen) elute RNA using RNeasy Minelute kit (Qiagen). The RNA pellet was <br />                                                resuspended DEPC-treated water, nucleic acid quantified using ND-1000 <br />                                                spectrophotometer (NanoDrop Technologies, Wilmington, DE). First strand cDNA was <br />                                                generated 2&#206;&#188;g total RNA using high capacity cDNA reverse transcription kit (Applied <br />                                                biosystems). Each cDNA sample diluted 1:5 water, 2.5 &#206;&#188;l used the <br />                                                template reaction. All primers minor groove binding (MGB), FAM <br />                                                labelled probes, purchased Applied Biosystems, Foster City, CA. All quantitative <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                measurements performed triplicate rpl32 used internal control for <br />                                                sample. dph1 mRNA values obtained gfp dsRNA treated samples set a <br />                                                value 100% dph1 dsRNA treated samples calculated accordingly. rpl32 <br />                                                expression levels altered gfp dph1 dsRNA transfected S2 cells. All <br />                                                reactions run 7300 Real-Time PCR (Applied Biosystems, Foster City, CA) <br />                                                standard reaction conditions. <br />  <br />                               3. Results <br />                               3.1. PE cytotoxic S2 Drosophila cells <br />                                                To characterize cytotoxic activity PE Drosophila cells tissue culture setting, <br />                                                S2 cells incubated various concentrations toxin 48 hr death <br />                                                measured using alamarBlue&#194;&#174; cell viability assay. AlamarBlue&#194;&#174; selected because, <br />                                                unlike viability reagents, interference growth media. Results <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                         Page 6 <br />  <br />  <br />                                                indicated PE potently toxic IC50 1 ng/ml (Fig. 1) complete <br />                                                cell death achieved 10 ng/ml greater. In order establish specificity PE <br />                                                cytotoxicity, competition assay performed S2 cells preincubated an <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                enzymatically inactive version <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> PE (ntPE). The competitor added 20 min <br />                                                prior addition varied concentrations active PE. Viability results showed that <br />                                                10&#206;&#188;g/ml ntPE able block completely cytotoxic effect highest <br />                                                concentration active PE used, 10ng/ml (Fig. 1). However, lower concentration of <br />                                                competitor, 1&#206;&#188;g/ml ntPE, completely inhibited cytotoxic effect 1ng/ml active PE <br />                                                failed higher concentrations PE. This result confirms receptor-mediated <br />                                                uptake PE leading death. A confirmation specific uptake established <br />                                                human-specific immunotoxin targeted human transferrin receptor failed kill <br />                                                cells concentrations 100 ng/ml (data shown). <br />  <br />                               3.2. PE-mediated ADP-ribosylation EF2 requires &#226;&#8364;&#732;diphthamide&#226;&#8364;&#8482; <br />                                                In mammalian cells, modification newly translated EF2 mature EF2 involves five <br />                                                enzyme (Dph1-5) post translational modification convert His residue 715 to <br />                                                diphthamide (Liu et al., 2004). To confirm toxin-mediated cell death dependent the <br />                                                presence diphthamide Drosophila cells, RNAi used knockdown first <br />                                                enzyme, Dph1, pathway. In absence antibody Drosophila Dph1, extent <br />                                                knockdown monitored RT-PCR. An approximate 70% reduction the <br />                                                endogenous dph1 mRNA expression observed cells treated dsRNA to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                dph1 (Fig. 2A). To assess PE activity following dph1 knockdown, protein synthesis <br />                                                cell viability measured. The Dph1 knockdown protected S2 cells PE- <br />                                                mediated inhibition protein synthesis (Fig. 2B); indicating diphthamide biosynthesis <br />                                                required toxin-mediated inhibition protein synthesis. For controls determined <br />                                                Dph1 knockdown did protect cells agents inhibit protein <br />                                                synthesis different mechanisms including actinomycin D (Fig. 2C) cycloheximide <br />                                                (Fig. 2D). When cell death measured, addition dsRNA dph1, gfp, <br />                                                protected cells PE-mediated toxicity, confirming Drosophila S2 cells PE- <br />                                                mediated death dependent diphthamide pathway (Fig 2 E). <br />  <br />                               3.3. Increase caspase activity addition PE <br />                                                The data indicate PE treatment causes inhibition protein synthesis dph1 <br />                                                dependent manner results cell death. Toxic agents kill cells distinct <br />                                                pathways. To investigate possible role apoptosis, post-treatment S2 cell extracts were <br />                                                assayed caspase activity. The role dph1 altering caspase activity presence of <br />                                                PE monitored. Toxin treated cell extracts tested activated caspase by <br />                                                monitoring activity using DEVD-AFC substrate. When S2 cells treated a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                control dsRNA (targeted gfp), substantial increase cell-associated caspase <br />                                                activity presence PE actinomycin D (Fig. 3). This suggested that <br />                                                toxin-mediated killing S2 cells occurred result caspase mediated apoptosis. <br />                                                However, S2 cells treated dsRNA dph1, increase caspase activity <br />                                                seen actinomycin D PE (Fig. 3). This confirmed PE induced <br />                                                caspase activation requires prior protein synthesis inhibition dph1-dependent manner. <br />                                                The actinomycin D induced caspase activation independent dph1. <br />  <br />                               3.4. PE mediates S2 cell death apoptosis <br />                                                To confirm participation caspase-like enzyme toxin-mediated apoptosis, a <br />                                                chemical inhibitor caspase activity, z-VAD-fmk used. The addition z-VAD-fmk <br />                                                (100 &#206;&#188;M) protected cells PE-mediated death (Fig. 4A) indicating apoptosis was <br />                                                probable mechanism cell death. Inhibitors caspase reported narrow <br />                                                substrate specificity interfere endocytic entry, toxin translocation the <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                         Page 7 <br />  <br />  <br />                                                cytosol ADP-ribosylation EF2. To confirm this, effect z-VAD-fmk PE- <br />                                                mediated inhibition protein synthesis determined (Fig. 4B). The addition z-VAD- <br />                                                fmk did interfere toxin uptake ADP-ribosylation EF2 leading inhibition <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                protein synthesis. Thus conclude death occurs downstream ADP- <br />                                                ribosylation EF2. <br />  <br />                               3.5. Drice required PE-mediated cell death <br />                                                The terminal effector caspases mammalian cells caspase 3 7. The Drosophila <br />                                                homolog caspases Drice. RNAi used downregulate drice gene expression <br />                                                evaluate role toxin-mediated death. The addition dsRNA drice, but <br />                                                gfp, resulted lower levels Drice protein detected western blot analysis <br />                                                (Fig. 5A). In order test efficiency drice knockdown preventing apoptotic cell <br />                                                death, actinomycin D cycloheximide, compounds previously reported kill Drosophila <br />                                                cells apoptosis (Zimmermann et al., 2002) used test cytotoxicity control <br />                                                drice knockdown conditions. As noted previously, dsRNA drice prevented S2 cell <br />                                                death compounds (Fig. 5B 5C). <br />  <br />                                                To determine role Drice PE-mediated death, cells pretreated dsRNA to <br />                                                drice gfp challenged PE. Cells treated gfp dsRNA killed by <br />                                                greater 90% cells treated dsRNA drice showed 20% decrease in <br />                                                viability, compared cells exposed toxin, supporting role <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                apoptosis PE-mediated cell death (Fig. 5D). <br />  <br />                                                Above shown downregulation inhibition essential genes ADP- <br />                                                ribosylation apoptosis pathway protects cells PE-mediated cell death. To support <br />                                                &#226;&#8364;&#732;inhibitory approaches&#226;&#8364;&#8482; positive evidence PE-mediated apoptosis, assayed <br />                                                treated cells activated Drice using DEVD-AFC substrate. When S2 cells were <br />                                                treated dsRNA drice, increase Drice enzymatic activity when <br />                                                challenged actinomycin D PE (Fig. 6). This confirmed toxin-mediated <br />                                                activation drosophila caspase pathway S2 cells occurred result activation of <br />                                                Drice. <br />  <br />                               3.6. Drice required PE-mediated protein synthesis inhibition <br />                                                Since PE treatment causes protein synthesis inhibition ADP-ribosylation of <br />                                                diphthamide residue EF2 cell death Drice dependent apoptotic pathway, the <br />                                                effect downregulation drice protein synthesis PE treatment examined. <br />                                                Downregulation drice did protect cells PE-mediated inhibition protein <br />                                                synthesis (Fig. 7) confirming PE-mediated modification EF2 occurred Drice- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                independent manner PE-mediated cell death occurred Drice-dependent manner, <br />                                                subsequent inhibition protein synthesis. <br />  <br />                               4. Discussion <br />                                                Here PE-mediated cell death apoptosis Drosophila S2 cells. Treatment <br />                                                pan caspase inhibitor, z-VAD-fmk preserved cell viability presence PE. <br />                                                However, z-VAD-fmk did prevent toxin-mediated inhibition protein synthesis, <br />                                                confirming inhibition protein synthesis cell death distinct events that <br />                                                apoptosis follows inhibition protein synthesis. In Drosophila, Drice effector <br />                                                caspase that, cleavage activation Dronc, mediates apoptosis. Depletion Drice <br />                                                S2 cells shown inhibit apoptotic events response variety stimuli <br />                                                (Muro et al., 2006). Likewise, study shows PE-mediated cell death prevented <br />                                                using dsRNA targeted drice, indicating pivotal role Drice PE-mediated cell death. <br />                                                And like inhibitor studies z-VAD-fmk, knockdown drice did prevent PE- <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                          Page 8 <br />  <br />  <br />                                                mediated inhibition protein synthesis confirming Drice functions downstream the <br />                                                ADP-ribosylation EF2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                PE-mediated cell death dependent presence diphthamide-modified EF2 (Liu and <br />                                                Leppla, 2003, Pavlovskis et al., 1978). Diphthamide conserved all <br />                                                eukaryotes archaebacteria. The absence appropriate &#226;&#8364;&#732;diphthamide modification&#226;&#8364;&#8482; in <br />                                                mammalian cells renders EF2 refractory modification PE resistant toxin- <br />                                                mediated cell death (Foley et al., 1995). Our study S2 cells shows Dph1 <br />                                                knockdown prevents toxin-mediated inhibition protein synthesis apoptosis, <br />                                                indicating ADP-ribosylation EF2 necessary sufficient PE- <br />                                                mediated cell death. However, Dph1 knockdown did prevent cycloheximide- <br />                                                mediated inhibition protein synthesis (data shown). Cycloheximide acts inhibition <br />                                                80S ribosome function (Schneider-Poetsch et al., 2010) does use diphthamide <br />                                                pathway. These kinds observations confirm diphthamide residue EF2 the <br />                                                single terminal target exotoxin A. <br />  <br />                                                The enzyme specificity Drice closely resembles mammalian caspase 3, were <br />                                                able use caspase 3 substrate monitor cell death. In order provide positive evidence <br />                                                Drice-dependent cell death, measured caspase-3 activity S2 cells treated either <br />                                                PE actinomycin D. While non-specific dsRNA-treated cells showed significant increase <br />                                                caspase-3 activity PE actinomycin D treatment, drice knockdown prevented this <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                increase. However, Dph1 levels knocked-down increase caspase 3 activity <br />                                                seen actinomycin D, confirming requirement Dph1 specifically the <br />                                                PE pathway. Additionally, Dph1 mediated protein synthesis inhibition PE necessary to <br />                                                induce apoptosis results Drosophila cell death. <br />  <br />                                                The potency PE (IC50 100 pM) lack cytotoxicity presence of <br />                                                inactive version PE, suggests receptor mediated binding nonspecific <br />                                                uptake responsible internalization toxin cells. High potency suggests <br />                                                components intoxication pathway conserved Drosophila cells <br />                                                sensitive mammalian cells. Having potent agent kills S2 cells defined <br />                                                pathway useful probe future studies apoptosis. In mammalian cells <br />                                                apoptosis blocked chemical inhibition caspase activity by <br />                                                overexpression prosurvival Bcl2 proteins. The identification equivalent factors <br />                                                currently investigation Drosophila toxin provide a <br />                                                convenient model test overexpression candidate prosurvival proteins. In fact, <br />                                                toxin mediated death S2 cells lends itself global searches genes essential for <br />                                                binding, endocytic uptake, retrograde transport ER retrotranslocation the <br />                                                cytosol. Gene knockdowns toxin pathway, using RNAi, produce survivors, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                monitored easily using alamarBlue&#194;&#174; assay. Additionally, the <br />                                                toxin interacts highly conserved eukaryotic proteins, resulting survivors help <br />                                                identify genes role discovered using gene <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>knockout</span> strategies (because of <br />                                                lethality). For example, knockdown Dph1 results EF2 that <br />                                                ADP-ribosylated. However, Dph1 -/- Dph3-/- mice embryonic lethal <br />                                                (Liang et al., 2008, Liu et al., 2006) effects related absence Dph1 Dph3 <br />                                                difficult discover using <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>gene knockout</span> strategy. <br />  <br />                                                Bacterial toxins serve valuable probes elucidating eukaryotic cell functions. Our <br />                                                findings facilitate use potent protein toxin combination RNAi way to <br />                                                monitor complex pathway beginning binding endocytosis ending the <br />                                                modification EF2 leading apoptotic cell death. <br />  <br />  <br />  <br />  <br />                                                      Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                                         Page 9 <br />  <br />  <br />                               Supplementary Material <br />                                                Refer Web version PubMed Central supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                                We like thank Dr. Bruce Patterson Dr. Concetta Lipardi pBC-S65T-GFP plasmid their <br />                                                valuable suggestions RNAi technique S2 cells. We thank Robert Sarnovsky providing ntPE Dr. <br />                                                Antonella Antignani reviewing manuscript. <br />  <br />                                                This research supported Intramural Research Program NIH, National Cancer Institute, Center for <br />                                                Cancer Research. The content publication does necessarily reflect views policies the <br />                                                Department Health Human Services, does mention trade names, commercial products, or <br />                                                organizations imply endorsement U.S. Government. <br />  <br />  <br />                               References <br />                                                Cashio P, Lee TV, Bergmann A. Genetic control programmed cell death Drosophila <br />                                                   melanogaster. Semin Cell Dev Biol. 2005; 16:225&#226;&#8364;&#8220;235. [PubMed: 15797833] <br />                                                FitzGerald DJ, Fryling CM, McKee ML, Vennari JC, Wrin T, Cromwell ME, Daugherty AL, Mrsny <br />                                                   RJ. Characterization V3 loop-Pseudomonas exotoxin chimeras. Candidate vaccines human <br />                                                   immunodeficiency virus-1. J Biol Chem. 1998; 273:9951&#226;&#8364;&#8220;9958. [PubMed: 9545339] <br />                                                Hafkemeyer P, Brinkmann U, Gottesman MM, Pastan I. Apoptosis induced Pseudomonas exotoxin: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   sensitive rapid marker gene delivery vivo. Hum Gene Ther. 1999; 10:923&#226;&#8364;&#8220;934. <br />                                                   [PubMed: 10223726] <br />                                                Hassan R, Bullock S, Premkumar A, Kreitman RJ, Kindler H, Willingham MC, Pastan I. Phase I study <br />                                                   SS1P, recombinant anti-mesothelin immunotoxin given bolus I.V. infusion patients with <br />                                                   mesothelin-expressing mesothelioma, ovarian, pancreatic cancers. Clin Cancer Res. 2007; <br />                                                   13:5144&#226;&#8364;&#8220;5149. [PubMed: 17785569] <br />                                                Jenkins CE, Swiatoniowski A, Issekutz AC, Lin TJ. Pseudomonas aeruginosa exotoxin A induces <br />                                                   human mast cell apoptosis caspase-8 -3-dependent mechanism. J Biol Chem. 2004; <br />                                                   279:37201&#226;&#8364;&#8220;37207. [PubMed: 15205454] <br />                                                Keppler-Hafkemeyer A, Kreitman RJ, Pastan I. Apoptosis induced immunotoxins used the <br />                                                   treatment hematologic malignancies. Int J Cancer. 2000; 87:86&#226;&#8364;&#8220;94. [PubMed: 10861457] <br />                                                Komatsu N, Oda T, Muramatsu T. Involvement caspase-like proteases serine proteases in <br />                                                   apoptotic cell death induced ricin, modeccin, diphtheria toxin, pseudomonas toxin. J <br />                                                   Biochem. 1998; 124:1038&#226;&#8364;&#8220;1044. [PubMed: 9792931] <br />                                                Kreitman RJ, Wilson WH, Robbins D, Margulies I, Stetler-Stevenson M, Waldmann TA, Pastan I. <br />                                                   Responses refractory hairy cell leukemia recombinant immunotoxin. Blood. 1999; 94:3340&#226;&#8364;&#8220; <br />                                                   3348. [PubMed: 10552943] <br />                                                Kreitman RJ, Wilson WH, Bergeron K, Raggio M, Stetler-Stevenson M, FitzGerald DJ, Pastan I. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Efficacy anti-CD22 recombinant immunotoxin BL22 chemotherapy-resistant hairy-cell <br />                                                   leukemia. N Engl J Med. 2001; 345:241&#226;&#8364;&#8220;247. [PubMed: 11474661] <br />                                                Kreitman RJ, Stetler-Stevenson M, Margulies I, Noel P, Fitzgerald DJ, Wilson WH, Pastan I. Phase II <br />                                                     trial recombinant immunotoxin RFB4(dsFv)-PE38 (BL22) patients hairy cell leukemia. <br />                                                     J Clin Oncol. 2009; 27:2983&#226;&#8364;&#8220;2990. [PubMed: 19414673] <br />                                                Mahajan-Miklos S, Rahme LG, Ausubel FM. Elucidating molecular mechanisms bacterial <br />                                                     virulence using non-mammalian hosts. Mol Microbiol. 2000; 37:981&#226;&#8364;&#8220;988. [PubMed: 10972817] <br />                                                Nicas TI, Iglewski BH. The contribution exoproducts virulence Pseudomonas aeruginosa. Can <br />                                                     J Microbiol. 1985; 31:387&#226;&#8364;&#8220;392. [PubMed: 2988728] <br />                                                Pastan I, Hassan R, Fitzgerald DJ, Kreitman RJ. Immunotoxin therapy cancer. Nat Rev Cancer. <br />                                                     2006; 6:559&#226;&#8364;&#8220;565. [PubMed: 16794638] <br />                                                Pollack M. The role exotoxin A pseudomonas disease immunity. Rev Infect Dis. 1983; <br />                                                     5(Suppl 5):S979&#226;&#8364;&#8220;984. [PubMed: 6419320] <br />  <br />  <br />  <br />  <br />                                                       Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                             Page 10 <br />  <br />  <br />                                                Rahme LG, Stevens EJ, Wolfort SF, Shao J, Tompkins RG, Ausubel FM. Common virulence factors <br />                                                   bacterial pathogenicity plants animals. Science. 1995; 268:1899&#226;&#8364;&#8220;1902. [PubMed: <br />                                                   7604262] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Rand RW, Kreitman RJ, Patronas N, Varricchio F, Pastan I, Puri RK. Intratumoral administration of <br />                                                   recombinant circularly permuted interleukin-4-Pseudomonas exotoxin patients high-grade <br />                                                   glioma. Clin Cancer Res. 2000; 6:2157&#226;&#8364;&#8220;2165. [PubMed: 10873064] <br />                                                Schneider-Poetsch T, Ju J, Eyler DE, Dang Y, Bhat S, Merrick WC, Green R, Shen B, Liu JO. <br />                                                   Inhibition eukaryotic translation elongation cycloheximide lactimidomycin. Nat Chem <br />                                                   Biol. 2010; 6:209&#226;&#8364;&#8220;217. [PubMed: 20118940] <br />                                                Tan MW, Rahme LG, Sternberg JA, Tompkins RG, Ausubel FM. Pseudomonas aeruginosa killing of <br />                                                   Caenorhabditis elegans used identify P. aeruginosa virulence factors. Proc Natl Acad Sci U S A. <br />                                                   1999; 96:2408&#226;&#8364;&#8220;2413. [PubMed: 10051655] <br />                                                Twomey C, McCarthy JV. Pathways apoptosis importance development. J Cell Mol Med. <br />                                                   2005; 9:345&#226;&#8364;&#8220;359. [PubMed: 15963254] <br />  <br />                               Abbreviations <br />  <br />                                             LRP1                  low density lipoprotein-related protein 1 <br />                                             ADP                   Adenosine diphosphate <br />                                             PE                    Pseudomonas exotoxin A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             EF2                   elongation factor-2 <br />                                             Act D                 actinomycin D <br />                                             CHX                   cycloheximide <br />                                             RNAi                  <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> <br />                                             dsRNA                 double stranded RNA <br />                                             GFP                   green fluorescence protein <br />                                             Dph1                  Diphthamide biosynthesis-like 1 <br />                                             Drice                 Drosophila Interlukin-1B-converting enzyme <br />                                             DCP-1                 death caspase-1 <br />                                             Dronc                 Drosophila Nedd2-like caspase <br />                                             Hid                   head involution defective <br />                                             Bcl-2                 B cell lymphoma-2 <br />                                             z-VAD-fmk             carbobenzoxy-valyl-alanyl-aspartyl-[O-methyl]- fluoromethylketone <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DEVD-AFC              Asp-Glu-Val-Asp-7-amino-4-trifluoromethyl coumarin <br />  <br />  <br />  <br />  <br />                                                       Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                      Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 1. <br />                                                PE shows specific cytotoxicity S2 cells dose-dependent manner. S2 cells were <br />                                                incubated indicated concentrations PE 48 h, prior incubation <br />                                                indicated concentrations ntPE 20min, cell viability determined a <br />                                                percentage control, using alamarBlue&#194;&#174;. Each point mean three <br />                                                determinations. Error bars represent standard deviation mean. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                          Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                  Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Toxicon. Author manuscript; available PMC 2012 August 30. <br />                             Sharma Fitzgerald                                                                                     Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 2. <br />                                                dph1 knockdown prevents PE-mediated inhibition protein synthesis cell death. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                (A) S2 cells incubated dsRNA dph1 non-target gene, gfp, the <br />                                                expression dph1 mRNA measured using real time PCR. Rpl32 expression used an <br />                                                internal control. The data expressed means independent experiments. (B) S2 <br />                                                cells subjected RNAi dph1 control gene, followed incubation with <br />                                                30ng/ml PE 24 h. Protein synthesis inhibition assessed measuring of <br />                                                {3H}-leucine incorporated cellular material presented percentage value of <br />                                                untreated sample. S2 cells subjected RNAi using dsRNA dph1 followed an <br />                                                incubation 150ng/ml actinomycin D (C), 200&#206;&#188;g/ml cycloheximide (D) 100ng/ml PE <br />                                                (E) 48 h. Cell viability measured using alamarBlue&#194;&#174; data presented as <br />                                                percentage value untreated sample. Each bar mean three <br />                                                determinations. Error bars represent standard deviation mean. <br />  <br />  <br />  <br />  <br />        <br /> </body></html>