<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A genetic strategy identify targets the <br /> development drugs prevent <br /> bacterial persistence <br /> Jee-Hyun Kima,1, Kathryn M. O&#226;&#8364;&#8482;Briena,1, Ritu Sharmaa, Helena I. M. Boshoffb, German Rehrena, Sumit Chakrabortyc, <br /> Joshua B. Wallacha, Mercedes Monteleonea, Daniel J. Wilsond, Courtney C. Aldrichd, Clifton E. Barry IIIb, Kyu Y. Rheec, <br /> Sabine Ehrta, Dirk Schnappingera,2 <br /> a <br />  Department Microbiology Immunology, Weill Cornell Medical College, New York, NY 10065; bTuberculosis Research Section, National Institute <br /> of Allergy Infectious Diseases, National Institutes Health, Bethesda, MD 20892; cDepartment Medicine, Weill Cornell Medical College, New York, NY <br /> 10065; dCenter Drug Design, University Minnesota, Minneapolis, MN 55455 <br />  <br /> Edited William R. Jacobs, Jr., Albert Einstein College Medicine Yeshiva University, Bronx, NY, approved October 14, 2013 (received review <br /> August 21, 2013) <br />  <br /> Antibacterial drug development suffers paucity targets               cystic &#239;&#172;?brosis-associated lung infections, salmonellosis, tu- <br /> whose inhibition kills replicating nonreplicating bacteria. The            berculosis (TB) (8&#226;&#8364;&#8220;11). We sought address needs by <br /> latter include phenotypically dormant cells, known persisters,              improving ability current genetic technologies more <br /> which tolerant antibiotics contribute                 closely mimic kinetics magnitude chemical inhibition <br /> failure treatment chronic infections. This           facilitate accurate identi&#239;&#172;?cation new po- <br /> apparent tuberculosis caused Mycobacterium tubercu-                 tential drug targets. <span id='am-7' about='protege:TO' typeof='owl:Thing'>To</span> end, engineered dual-control <br /> losis, pathogen tolerates antibiotics ceases               (DUC) switch single inducer&#226;&#8364;&#8221;anhydrotetracycline <br /> to replicate. We developed strategy identify proteins                (atc) doxycycline (doxy)&#226;&#8364;&#8221;simultaneously triggers transcrip- <br /> Mycobacterium tuberculosis requires grow persist <br />                                                                                tional repression target gene degradation encoded <br /> and inhibition potential prevent drug tolerance <br />                                                                                protein demonstrated switch identify proteins <br /> and persister formation. This strategy based tunable dual- <br />                                                                                required replicating nonreplicating Mycobacterium <br /> control genetic switch provides regulatory range spanning <br /> three orders magnitude, quickly depletes proteins rep- <br />                                                                                tuberculosis (Mtb). <br /> licating nonreplicating mycobacteria, exhibits increased <br />                                                                                   TB remains leading cause death single bacterial <br /> robustness phenotypic reversion. Using switch, demon-               infection worldwide, drug-resistant Mtb strains continue to <br /> strated depletion nicotinamide adenine dinucleotide                evolve spread (12). Drug-sensitive TB cured un- <br /> synthetase (NadE) rapidly killed Mycobacterium tuberculosis              interrupted treatment multiple drugs 6 mo. The <br /> conditions standard growth nonreplicative persistence in-               requirement long therapy is, part, bacilli that <br /> duced oxygen <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> limitation                  persist slow-growing nongrowing state recalcitrant <br /> acute chronic phases infection mice. These &#239;&#172;?ndings es-               killing TB drugs. However, speci&#239;&#172;?c activities <br /> tablish dual-control switch robust tool                 Mtb requires persist drug-tolerant state remain largely <br /> probe essentiality Mycobacterium tuberculosis proteins un- <br /> der different conditions, including induce antibiotic tol-              Signi&#239;&#172;?cance <br /> erance, NadE target potential shorten current <br /> tuberculosis chemotherapies.                                                       Chronic bacterial infections, caused Myco- <br />                                                                                    bacterium tuberculosis (Mtb), continue claim lives of <br />  <br /> T   arget-based approaches emerged major paradigm <br />     modern drug development, largely failed dis- <br /> cover new antibacterials (1, 2). The reasons multi- <br />                                                                                    millions people. New antibiotics needed treat these <br />                                                                                    infections, development hindered lack tar- <br />                                                                                    gets inhibition quickly eradicates bacterial pathogens <br /> factorial. Nonetheless, fundamental requirement target-                       prevents <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> drug-tolerant persisters. We de- <br /> based approaches need identify targets inhibition                  scribe unique dual-control (DUC) switch combines re- <br /> can selectively quickly improve pathophysiologic phe-                      pression transcription controlled proteolysis silence <br /> notype interest. For development antibacterials,                    gene activities Mtb. By conditionally inactivating Mtb&#226;&#8364;&#8482;s nic- <br /> need initially thought addressable recombinant                      otinamide adenine dinucleotide synthetase, demonstrate <br /> genetic technologies enabled untargeted genome-wide                      DUC switch identify proteins pathogen <br /> mutations targeted gene deletions (2). Subsequent advances                     requires growth nonreplicating persistence vitro <br /> achieved conditional transcriptional silencing genes,                     infections. Targeting proteins holds prom- <br />                                                                                                                                                                                    MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> allowed identi&#239;&#172;?cation targets functions es-                   ise yielding drugs shorten duration antibacterial <br /> sential normal physiology cell metabolism                    chemotherapies. <br /> in context pathophysiology disease interest <br />                                                                                Author contributions: J.-H.K., K.M.O., R.S., H.I.M.B., G.R., S.C., M.M., C.E.B., K.Y.R., S.E., <br /> (3&#226;&#8364;&#8220;6). Notwithstanding, advances failed address <br />                                                                                D.S. designed research; J.-H.K., K.M.O., R.S., H.I.M.B., G.R., S.C., J.B.W., M.M. <br /> that drugs act time scale faster associated               performed research; D.J.W. C.C.A. contributed new reagents/analytic tools; J.-H.K., <br /> with changes novo transcription genetic                    K.M.O., R.S., H.I.M.B., G.R., S.C., J.B.W., M.M., C.C.A., C.E.B., K.Y.R., S.E., D.S. analyzed <br /> approaches achieve narrow ranges regulation (6, 7).                    data; J.-H.K., K.M.O., C.E.B., K.Y.R., S.E., D.S. wrote paper. <br />  <br /> Furthermore, existing genetic approaches limited                   The authors declare con&#239;&#172;&#8218;ict interest. <br /> their inability identify proteins required growth              This article PNAS Direct Submission. <br /> but survival persistence nongrowing bacteria.              1 <br />                                                                                 J.-H.K. K.M.O. contributed equally work. <br /> Bacteria actively dividing include persisters,              2 <br />                                                                                 To correspondence addressed. E-mail: dis2003@med.cornell.edu. <br /> phenotypic variants withstand bactericidal antibiotics                This article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> complicate treatment chronic infections including                  1073/pnas.1315860110/-/DCSupplemental. <br />  <br />  <br />  <br /> www.pnas.org/cgi/doi/10.1073/pnas.1315860110                                                 PNAS | November 19, 2013 | vol. 110 | no. 47 | 19095&#226;&#8364;&#8220;19100 <br />  unknown. Using DUC switch, demonstrated Mtb <br /> depends nicotinamide adenine dinucleotide (NAD) synthesis <br /> for growth survival nonreplicating persistence and <br /> identi&#239;&#172;?ed NAD synthetase (NadE) essential persis- <br /> tence target inactivation bactericidal. Moreover, de- <br /> pletion NadE acute chronic infections rapidly <br /> eliminated Mtb mice. These &#239;&#172;?ndings validate po- <br /> tential genetic strategy identify targets inhibition <br /> is capable killing Mtb persister population potentially <br /> shortening duration TB chemotherapies. <br />  <br /> Results <br /> Construction Genetic Switch That Combines Transcriptional and <br /> Proteolytic Silencing. To develop switch atc simulta- <br /> neously represses transcription induces proteolysis, &#239;&#172;?rst <br /> designed tetracycline repressors (TetRs), T38 TSC10, to <br /> recognize different tet operators (tetO tetO-4C5G), form <br /> heterodimers, respectively activated inactivated atc <br /> (Fig. S1A). T38 derived TetR28 (13), requires atc <br /> to repress transcription, introducing mutations&#226;&#8364;&#8221;E37A <br /> and P39K&#226;&#8364;&#8221;that prevent binding tetO allow binding to <br /> tetO-4C5G (14). TSC10 constructed converting TetR10, <br /> which inactivated atc (13), single-chain repressor to <br /> prevent heterodimerization T38 (15). We con&#239;&#172;?rmed that <br /> TSC10 T38 repress promoters contain tetO (PtetO) <br /> and tetO-4C5G (PtetO-4C5G), respectively, using Mycobacte- <br /> rium smegmatis (Msm) dual-reporter strain (Fig. S1 B C). <br />    SspB Escherichia coli-derived protein deliver <br /> proteins containing DAS+4-tag degradation machin-             Fig. 1. The DUC switch. (A) Genetic organization. (B) Atc dose&#226;&#8364;&#8220;response. <br /> ery mycobacteria (16, 17). In Msm strain tran-            Relative luminescence units (RLU) strains luciferase repressed <br /> scription sspB luciferase encoding luxAB-DAS operon           SspB (red), T38 (blue), SspB T38 (purple) grown dif- <br /> were controlled TSC10 T38, respectively (Fig. 1A),             ferent atc concentrations. Black squares represent Msm strains expressing <br /> expected atc (i) represses transcription luxAB-DAS (via          luxAB-DAS constitutively regulatory components. Data are <br />                                                                          means &#194;&#177; SEM replicates. The gray area indicates autoluminescence of <br /> inactivation PtetO-4C5G T38&#226;&#8364;&#8220;atc complex), (ii) causes <br />                                                                          Msm does express luciferase. <br /> proteolysis DAS+4-tagged luciferase subunit LuxB-DAS (via <br /> the atc-induced release TSC10 activation PtetO- <br /> sspB), (iii) reduces luciferase activity ef&#239;&#172;?ciently   reporter genes replicating and, importantly, nonrep- <br /> when luciferase activity controlled repression           licating mycobacteria. <br /> transcription controlled proteolysis alone. We con&#239;&#172;?rmed these <br /> expectations &#239;&#172;?rst using episomal PtetO-4C5G-luxAB-DAS                 Biological Utility DUC Switch Conditional Inactivation of <br /> construct (Fig. 1B). When luciferase inactivated           Mtb Genes. The 7,8-diaminopelargonic acid synthase (BioA) is <br /> inducible proteolysis transcriptional repression alone, atc re-       required Mtb synthesize biotin, grow biotin-free media, <br /> duced luminescence 270-fold 420-fold, respectively,         multiply persist mice, Mtb remarkably <br /> whereas combination reduced luminescence 5,800-fold.         resistant partial inactivation enzyme (7). To determine <br /> All regulatory systems showed similar atc dose&#226;&#8364;&#8220;responses.          DUC switch ef&#239;&#172;?ciently inactivate BioA, gener- <br /> Similar results obtained Msm expressing chromosomally          ated Mtb BioA-T38, atc turns bioA transcrip- <br /> integrated luxAB-DAS Msm Mtb expressing red &#239;&#172;&#8218;uo-               tional repression; Mtb BioA-SspB, atc inactivates BioA <br /> rescent protein (Fig. S2). <br />                                                                          controlled proteolysis; Mtb BioA-DUC, BioA is <br />    Next, analyzed kinetics luciferase inactivation and <br />                                                                          regulated DUC switch. For mutants used strong <br /> reactivation replicating Msm. Half-maximal repression lu- <br />                                                                          translation initiation site causes approximately 10-fold <br /> ciferase activity reached 1 h controlled pro- <br />                                                                          overexpression BioA (7). Addition atc Mtb BioA-SspB <br /> teolysis proteolysis combined with <br /> transcriptional repression (Fig. S3A). Transcriptional silencing,        Mtb BioA-DUC depleted BioA protein level of <br /> in comparison, slow did reach half-maximum               detection 48 h stopped growth (Fig. 2). In <br /> until 10 h addition atc. When Msm &#239;&#172;?rst grown            Mtb BioA-T38, BioA protein undetectable 72 h the <br /> with atc transferred medium atc, luciferase          addition atc, growth remained unaffected. When growth <br /> activity reappeared similar kinetics strains (Fig. S3B).     monitored longer period, apparent atc <br /> To analyze kinetics gene silencing nonreplicating Msm,         reduced growth Mtb BioA-T38, abolished <br /> we adopted PBS starvation model developed Mtb (18).              growth Mtb BioA-SspB Mtb BioA-DUC 15 d (Fig. S4 <br /> Luciferase activity decreased slowly inactivation solely            A B). During period colony-forming units (cfu) of <br /> depended transcriptional repression (Fig. S3C). In contrast,          Mtb BioA-SspB Mtb BioA-DUC decreased &gt;90% (Fig. <br /> half-maximal repression reached approximately 3 h              S4C). After 15 d, OD Mtb BioA-SspB cultures increased <br /> when controlled proteolysis used combination             (Fig. S4B) atc-resistant revertants. Such revertants also <br /> with transcriptional repression. Only DUC switch achieved            occurred Mtb BioA-DUC lower frequency. When we <br /> a reduction luciferase activity background levels. In sum-         analyzed appearance revertants 96 cultures both <br /> mary, experiments established DUC switch mediates         mutants, revertants 8% BioA-SspB cultures <br /> ef&#239;&#172;?cient, reversible, tunable, rapid inactivation different       BioA-DUC cultures 27 d (Fig. S4D). After <br />  <br /> 19096 | www.pnas.org/cgi/doi/10.1073/pnas.1315860110                                                                                         Kim et al. <br />                                                                                  BioA-SspB decreased 0.2% day 135, Mtb <br />                                                                                 BioA-DUC decreased 0.02% (P &lt; 0.01). Taken together, <br />                                                                                 experiments demonstrated DUC switch ach- <br />                                                                                 ieve near-complete inactivation Mtb protein vitro and <br />                                                                                 infections increases resistance conditional <br />                                                                                 knockdown mutants phenotypic reversion. <br />  <br />                                                                                 Consequences Inactivating NadE Growth Survival Mtb in <br />                                                                                 Vitro During Infections. NAD essential cofactor nu- <br />                                                                                 merous biochemical reactions (20) substrate DNA <br />                                                                                 ligases (21), ADP ribosylases (22), protein deacetylases (23). <br />                                                                                 Mtb synthesize NAD novo pathway recycle via <br />                                                                                 salvage pathway (Fig. 3A). Both pathways share reactions <br />                                                                                 catalyzed NadE nicotinic acid mononucleotide adenylyl- <br /> Fig. 2. Kinetics BioA inactivation. (Left) Growth biotin-free Sauton&#226;&#8364;&#8482;s <br />                                                                                 transferase (encoded nadD) (24, 25). Transposon mutagen- <br /> medium atc added day 5. (Right) Immunoblot proteins ex-              esis identi&#239;&#172;?ed nadD nadE required optimal vitro <br /> tracted cultures shown Left using antisera BioA         growth (26, 27). This essentiality replication far precluded <br /> PrcB (loading control).                                                         analysis requirement viability nonreplicating or <br />                                                                                 slowly replicating Mtb associated chronic <br />                                                                                 phase infection. The importance NAD synthesis survival <br /> 35 d, 60% BioA-SspB 40% BioA-DUC cultures                     nonreplicating persistence questioned experi- <br /> contained revertants.                                                           ments showed Mtb mutants incapable synthesizing <br />    The bioA mutants constructed study contain unique                   NAD novo (but capable NAD recycling) survived <br /> DNA tags, used analyze growth persistence                 unimpaired absence exogenous nicotinamide their <br /> in multistrain mouse infections (19). In addition                  replication arrested weeks in- <br /> knockdown mutants, infections included WT Mtb,                      cubation hypoxic conditions <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> starvation (24). <br /> which abundance strains normalized; Mtb                    Mtb remained virulent mice salvage the <br /> &#206;&#8221;bioA; Mtb DUC-control. In Mtb DUC-control,                           novo pathway inactivated (24, 25). To determine the <br /> contains WT bioA addition bioA-DAS, atc depleted DAS-                     consequences inactivating novo salvage <br /> tagged BioA protein did interfere biotin synthesis                 pathways survival nonreplicating Mtb vitro for <br /> due continued expression WT BioA protein. The relative                    growth persistence mice, constructed Mtb NadE-DUC, <br /> abundance Mtb BioA DUC-control similar WT                     DUC switch allowed inducible depletion NadE. <br /> Mtb mice doxy (Fig. S5A). Inactivation                       In <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span>-rich media, atc prevented growth Mtb NadE- <br /> BioA-DAS DUC switch did exert phenotypic off-                   DUC, rapidly depleted NadE, caused cfu decrease less <br /> target dominant-negative effects. &#206;&#8221;bioA strongly attenu-                 0.1% 24 h (Fig. S6). Inhibition growth atc <br /> ated mice, abundance Mtb BioA-T38                     dose-dependent accompanied dose-dependent <br /> similar WT (Fig. S5 B C). Mtb BioA-SspB Mtb                  decrease NAD increase NadE substrate NaAD <br /> BioA-DUC strongly attenuated doxy ad-                        (Fig. 3). Depletion NadE furthermore caused NaMN ac- <br /> ministered infection, doxy given                    cumulate, possibly product inhibition NadD dead- <br /> later, Mtb BioA-DUC showed stronger <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> Mtb                        enylation NaAD (28, 29). Next, reliance nonreplicating Mtb <br /> BioA-SspB (Fig. S5 D E). For example, mice started                  NadE survival analyzed starvation hypoxia. <br /> receiving doxy 10 d postinfection, relative abundance Mtb                During starvation PBS, inactivation NadE decreased cfu to <br />  <br />  <br />  <br />  <br />                                                                                                                                                                MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> Fig. 3. Impact depleting NadE NAD synthesis. (A) NAD biosynthesis pathways quanti&#239;&#172;?cation NaMN, NaAD, NAD extracts NadE-DUC <br /> after exposure different atc concentrations. (B) Growth different atc concentrations. OD, optical density; data means &#194;&#177; SEM replicates. <br />  <br />  <br /> Kim et al.                                                                                        PNAS | November 19, 2013 | vol. 110 | no. 47 | 19097 <br />  &#226;&#710;&#188;2% &#239;&#172;?rst 24 h (Fig. 4A). Inactivation NadE                  Discussion <br /> hypoxia sterilized culture 108 cfu/mL (Fig. 4B).                            Effective rapid genetic inactivation potential to <br />   To assess effect silencing acute chronic in-                 identify targets development drugs prevent bac- <br /> fection mice, NadE inactivated administering doxy                      terial persistence better treat chronic bacterial infections. <br /> beginning day 1, 10, 28, 56. In mice received doxy                    However, prevented transcriptional leakiness, slow <br /> throughout infection, Mtb NadE-DUC re-                         incomplete depletion transcriptionally regulated gene prod- <br /> covered lungs immediately infection,                   ucts, mutations interfere regulation, combination <br /> pathological signs infection observed (Fig. 5 Fig. S7).              thereof (6, 7, 30). Here demonstrated problems can <br /> When doxy administration started later, cfu declined rapidly                 drastically reduced overcome entirely switch that <br />                                                                                  combines transcriptional repression regulated proteolysis. The <br /> in lungs spleens acutely chronically infected <br />                                                                                  DUC switch reduced reporter gene activities 5,800-fold <br /> mice. Decreases cfu accompanied decreases lung                     accelerated gene inactivation growing nonreplicating <br /> pathology (Fig. 5 Fig. S7). Inactivation NadE-DUC                      bacteria, did compromise reactivation gene expression or <br /> doxy impact growth survival Mtb vitro                      atc dose-responsiveness (Fig. 1 Figs. S2 S3). We &#239;&#172;?rst <br /> during infections presence additional WT copy                    demonstrated features facilitate analyses essential <br /> nadE controlled DUC switch (Fig. S8). These                   gene functions clinically relevant pathogens targeting BioA in <br /> results revealed inactivation NAD recycling                Mtb. We previously demonstrated Mtb phenotypically re- <br /> de novo synthesis bactericidal Mtb growth                 sistant partial depletion BioA, greatly complicated the <br /> and nonreplicating persistence vitro rapidly eradicates              isolation knockdown mutants capable revealing con- <br /> culturable Mtb mouse lungs spleens acute                     sequences ef&#239;&#172;?cient BioA depletion vitro infections <br /> chronic infections.                                                              (7). In contrast, BioA-DUC allowed rapid bactericidal de- <br />                                                                                  pletion ef&#239;&#172;?cient translation caused &#226;&#710;&#188;10-fold in- <br />                                                                                  crease BioA, indicated DUC switch regulated <br />                                                                                  BioA &#226;&#710;&#188;1,000-fold, &#226;&#710;&#188;10-fold WT expression <br />                                                                                  level &#226;&#710;&#188;99-fold WT expression level required stop <br />                                                                                  growth (7). The DUC switch furthermore generated BioA <br />                                                                                  knockdown mutant susceptible phenotypic reversion <br />                                                                                  (Fig. S4D). This signi&#239;&#172;?cant accumulation rever- <br />                                                                                  tants complicates construction, maintenance, anal- <br />                                                                                  ysis knockdown mutants vitro essential genes. <br />                                                                                     Degradation proteins ClpXP, protease complex that <br />                                                                                  mediates controlled proteolysis DUC switch, consumes ATP <br />                                                                                  (31). This ATP consumption itself did impair growth survival <br />                                                                                  Mtb evident control strains. In &#239;&#172;?rst, Mtb DUC- <br />                                                                                  control, atc caused degradation DAS-tagged BioA did not <br />                                                                                  interfere biotin synthesis continued expression of <br />                                                                                  WT BioA. Even doxy, Mtb DUC-control grew persisted <br />                                                                                  like WT mice (Fig. S5A). In second strain, WT nadE was <br />                                                                                  used complement Mtb NadE-DUC, led normal <br />                                                                                  growth vitro infections (Figs. S6 S8). These <br />                                                                                  controls demonstrated depletion BioA NadE by <br />                                                                                  DUC switch caused dominant-negative effects. <br />                                                                                     NAD biosynthesis enzymes proposed targets for <br />                                                                                  development drugs pathogens (24, 25, 32). <br />                                                                                  However, Mtb mutants unable recycle synthesize <br />                                                                                  NAD novo remained virulent mice, Mtb lacking de <br />                                                                                  novo synthesis pathway remained viable nonreplicating <br />                                                                                  persistence external NAD (24, 25). These results <br />                                                                                  left essentiality NAD biosynthesis nonreplicating <br />                                                                                  slowly replicating Mtb populations vivo unresolved. Our <br />                                                                                  studies established depletion NadE Mtb NadE-DUC <br />                                                                                  occurred rapidly, prevented growth, caused atc dose-dependent <br />                                                                                  reduction NAD atc dose-dependent increases the <br />                                                                                  substrates NadE (NaAD) NadD (NaMN) (Fig. 3). Strong <br />                                                                                  depletion NadE bactericidal Mtb growth in <br />                                                                                  broth (Fig. S6B), nonreplicating persistence (Fig. 4), infec- <br />                                                                                  tions (Fig. 5). For validation potential drug target, is <br />                                                                                  signi&#239;&#172;?cant ef&#239;&#172;?ciently depletion NadE killed Mtb during <br />                                                                                  hypoxia-induced nonreplicating persistence tuberculous <br />                                                                                  granulomas hypoxic (33), current TB drugs not <br />                                                                                  effective hypoxia-adapted Mtb (34). This the <br />                                                                                  impact NadE depletion starvation suggested inhib- <br /> Fig. 4. Impact depleting NadE nonreplicating Mtb. (A) Survival during <br />                                                                                  iting enzyme kill Mtb metabolic <br /> starvation PBS atc (white symbols) atc (black symbols). <br /> Data means &#194;&#177; SEM &#239;&#172;?ve replicates. (B) Survival hypoxia        states infections. In accordance expectation, de- <br /> adding atc day 9 (black), day 16 (orange), day 24 (purple). Dotted lines   pletion NadE strongly attenuated Mtb vivo acute <br /> end start data points titer limit      chronic infections. The rates cfu declined mouse <br /> detection 1 cfu/mL. Data means &#194;&#177; SEM replicates.                 lungs spleens fast similar irrespective when <br />  <br /> 19098 | www.pnas.org/cgi/doi/10.1073/pnas.1315860110                                                                                           Kim et al. <br />  Fig. 5. Impact depleting NadE Mtb acute chronic infections. (A) Growth survival mouse lungs (Left) spleens (Right). Mice <br /> received doxy-containing food day 1 (black), day 10 (orange), day 28 (purple), day 56 (green), (gray). Dotted lines end data points <br /> for titer limit detection 1 cfu/organ. Data means &#194;&#177; SEM mice. (B) Gross pathology lungs 168 d <br /> postinfection. <br />  <br />  <br />  <br /> depletion initiated (Fig. 5). This remarkable                   Survival Mtb During Nonreplicating Persistence. To obtain nonreplicating <br /> existing TB drugs generally effective acute                 Mtb starvation, bacteria grown 7H9 medium, washed twice with <br /> phase (e.g., isoniazid) chronic phase infection (e.g., ri-               PBST, suspended PBST OD 0.6. After 10 d, atc added to <br />                                                                                    half cultures, survival determined periodically plating <br /> fampicin pyrazinamide) (35, 36). Targeting <br />                                                                                    serial dilutions agar plates. For hypoxia experiments, bacteria were <br /> NadE provides rational approach development                    grown glycerol-free Dubos medium OD 0.5. Cells &#239;&#172;?ltered <br /> shorter anti-TB chemotherapies.                                                    5-&#206;&#188;m &#239;&#172;?lter remove clumps diluted OD &#226;&#710;&#188;0.005 in <br />    The development new classes antibacterial drugs repre-                    glycerol-free Dubos medium. Cells transferred septum sealed tubes <br /> sents urgent need major challenge, met                    headspace ratio 0.5. A control tube strain contained meth- <br /> with validation new targets alone. The identi&#239;&#172;?cation                     ylene blue ensure cells hypoxic expected time. Tubes <br /> novel small molecules penetrate bacterial cell envelope                   placed multitube stir plate 37 &#194;&#176;C covered foil. The meth- <br /> and ef&#239;&#172;?ciently selectively inactivate bacterial target remains               ylene blue tubes decolorized day 9. When indicated, tubes transferred <br /> a major bottleneck. Target-based whole-cell screens, use                     anaerobic chamber, atc solvent atc injected through <br />                                                                                    rubber septum. For time point, tubes strain were <br /> bacteria underexpressing selected protein bias hit com- <br />                                                                                    opened, serial dilutions prepared plated agar plates. <br /> pounds inhibitors protein (37&#226;&#8364;&#8220;41), promise to <br /> widen bottleneck best applied proteins rep-                   Analysis atc Resistance Conditional BioA Knockdown Mutants. BioA mu- <br /> resent well-validated targets. The strategy used identify                    tant cultures 7H9 medium washed used inoculate 100 &#206;&#188;L <br /> NadE validated target scalable applied,                 biotin-free Sauton&#226;&#8364;&#8482;s medium containing atc &#226;&#710;&#188;5,000 cfu. OD mea- <br /> principle, gene Mtb genome. We expect                       sured SpectraMax M5 plate reader. Aliquots BioA mutants had <br /> DUC switch facilitate large-scale studies im-                     grown OD &gt; 0.1 streaked biotin-free 7H9 agar plates or <br /> proved regulatory ranges increase frequency                    atc con&#239;&#172;?rm loss regulation. <br /> phenotypically well-regulated conditional knockdown mutants <br /> can constructed. Such mutants help de&#239;&#172;?ne                           Metabolite Analyses. Bacteria grown 7H9 medium transferred onto <br /> broadly genes activities Mtb requires grow                  0.22-&#206;&#188;m nitrocellulose &#239;&#172;?lters vacuum &#239;&#172;?ltration. After growing 7H11 <br />                                                                                    plates 6 d, Mtb-laden &#239;&#172;?lters transferred 7H11 plates containing <br /> vitro acute infections persist despite the <br />                                                                                    varying concentrations atc incubated additional 2 d they <br /> chemical insults faces activated immune                       plunged buffer containing 10% (vol/vol) acetonitrile, 0.15 mM am- <br /> chemotherapy. The regulatory components DUC switch                      monium acetate, 0.5 mM ascorbic acid (pH 6.0), precooled &#226;&#710;&#8217;40 &#194;&#176;C. The <br /> control gene expression bacteria eukaryotes (42),                   metabolite extraction achieved mechanically lysing bacteria con- <br /> which broadens utility studying organisms diseases.               taining solution 0.1-mm Zirconia beads Precellys tissue homogenizer <br />                                                                                    8 min continuous cooling. Lysates clari&#239;&#172;?ed centrifugation, <br /> Methods                                                                            &#239;&#172;?ltered 0.22-&#206;&#188;m &#239;&#172;?lter, passed centrifugal column <br />                                                                                    (Nanosep 10K Omega). Residual protein content metabolite extracts was <br />                                                                                                                                                                      MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> Plasmids, Bacterial Strains, General Procedures. Bacteria plasmids are <br /> listed Table S1. Mycobacteria generally grown 37 &#194;&#176;C Mid-             measured normalize samples cell biomass. Metabolites separated in <br /> dlebrook 7H11 agar 0.5% glycerol Middlebrook 7H9 medium            Agilent Accurate Mass 6520 quadrupole time-of-&#239;&#172;&#8218;ight coupled Agilent <br /> 0.5% glycerol 0.02% tyloxapol (Sigma). For Mtb, 7H11 supplemented          1200 liquid chromatography Cogent Diamond Hydride Type C <br /> with 10% (vol/vol) Middlebrook oleic acid-albumin-dextrose-catalase (Becton        column using following mobile phase: solvent A [50% (vol/vol) methanol, <br /> Dickinson); 7H9 supplemented 0.2% glycerol, 0.05% tyloxapol, 0.5%         0.05% formic acetic acid] solvent B [90% (vol/vol) acetonitrile, 1.5 mM <br /> BSA, 0.2% dextrose, 0.085% NaCl. Preparation cell lysates; immunoblot       ammonium acetate, pH6.8]. Dynamic mass axis calibration achieved by <br /> analyses; measurements &#239;&#172;&#8218;uorescence, luminescence, &#206;&#178;-galactosidase       continuous infusion reference mass solution using isocratic pump with <br /> activity performed described previously (7, 13, 16, 43). To measure    100:1 splitter. This con&#239;&#172;?guration achieved mass errors 5 ppm, mass reso- <br /> kinetics luciferase inactivation starvation, Msm transferred    lution ranging 10,000 25,000 (over m/z 121&#226;&#8364;&#8220;955 atomic mass <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>), 5 <br /> PBS 0.05% tyloxapol (PBST). After incubating 37 &#194;&#176;C 32 h, atc (Sigma)   log10 dynamic range. Detected ions deemed metabolites basis of <br /> was added half cultures, luciferase activity measured peri-      unique accurate mass retention time identi&#239;&#172;?ers masses exhibiting the <br /> odically. Single-strain multistrain infections performed described     expected distribution accompanying isotopomers. Metabolite identities were <br /> (19). Procedures involving mice reviewed approved Institutional    searched using mass tolerance &lt;0.005 Da. Metabolites quanti&#239;&#172;?ed <br /> Animal Care Use Committee Weill Cornell Medical College.                    using calibration curve generated chemical standard spiked into <br />  <br />  <br /> Kim et al.                                                                                           PNAS | November 19, 2013 | vol. 110 | no. 47 | 19099 <br />  homologous mycobacterial extract correct matrix-associated ion                           ACKNOWLEDGMENTS. We thank C. Nathan PrcB-speci&#239;&#172;?c antiserum. <br /> suppression effects. This approach allowed quanti&#239;&#172;?cation nicotinamide,                       This work supported Grants 42848 OPP1024065 (to D.S.) and <br /> NaMN, NaAD, NAD. Nicotinamide levels did change response                          OPP10204050 (to K.Y.R.); Bill Melinda Gates Foundation the <br />                                                                                                 Wellcome Trust Grand Challenges Global Health Initiative <br /> inactivation NadE, changes occurred NaMN, NaAD,                         (to D.S., S.E., C.C.A., C.E.B.); National Institutes Health (NIH) <br /> NAD shown Fig. 3. Concentrations metabolites NAD                     Grant AI089911 (to D.S.); Intramural Research Program the <br /> biosynthesis pathway level detection.                                         National Institute Allergy Infectious Diseases, NIH (to C.E.B.). <br />  <br />  <br />  1. Cole ST, Riccardi G (2011) New tuberculosis drugs horizon. Curr Opin Microbiol       23. Denu JM (2005) The Sir 2 family protein deacetylases. Curr Opin Chem Biol 9(5): <br />     14(5):570&#226;&#8364;&#8220;576.                                                                                  431&#226;&#8364;&#8220;440. <br />  2. Payne DJ, Gwynn MN, Holmes DJ, Pompliano DL (2007) Drugs bad bugs: Con-                 24. Boshoff HI, et al. (2008) Biosynthesis recycling nicotinamide cofactors my- <br />     fronting challenges antibacterial discovery. Nat Rev Drug Discov 6(1):29&#226;&#8364;&#8220;40.             cobacterium tuberculosis. An essential role NAD nonreplicating bacilli. J Biol <br />  3. Gandotra S, Schnappinger D, Monteleone M, Hillen W, Ehrt S (2007) In vivo gene                  Chem 283(28):19329&#226;&#8364;&#8220;19341. <br />     silencing identi&#239;&#172;?es Mycobacterium tuberculosis proteasome essential          25. Vilch&#195;&#168;ze C, Weinrick B, Wong KW, Chen B, Jacobs WR, Jr. (2010) NAD+ auxotrophy is <br />     bacteria persist mice. Nat Med 13(12):1515&#226;&#8364;&#8220;1520.                                          bacteriocidal tubercle bacilli. Mol Microbiol 76(2):365&#226;&#8364;&#8220;377. <br />  4. Leblanc C, et al. (2012) 4&#226;&#8364;&#178;-Phosphopantetheinyl transferase PptT, new drug target         26. Grif&#239;&#172;?n JE, et al. (2011) High-resolution phenotypic pro&#239;&#172;?ling de&#239;&#172;?nes genes essential <br />     required Mycobacterium tuberculosis growth persistence vivo. PLoS Pathog             mycobacterial growth cholesterol catabolism. PLoS Pathog 7(9):e1002251. <br />     8(12):e1003097.                                                                             27. Zhang YJ, et al. (2012) Global assessment genomic regions required growth in <br />  5. Marrero J, Rhee KY, Schnappinger D, Pethe K, Ehrt S (2010) Gluconeogenic carbon                 Mycobacterium tuberculosis. PLoS Pathog 8(9):e1002946. <br />     &#239;&#172;&#8218;ow tricarboxylic acid cycle intermediates critical Mycobacterium tuberculosis    28. Mehl RA, Kinsland C, Begley TP (2000) Identi&#239;&#172;?cation Escherichia coli nicotinic <br />     establish maintain infection. Proc Natl Acad Sci USA 107(21):9819&#226;&#8364;&#8220;9824.                  acid mononucleotide adenylyltransferase gene. J Bacteriol 182(15):4372&#226;&#8364;&#8220;4374. <br />  6. Wei JR, et al. (2011) Depletion antibiotic targets widely varying effects         29. Olland AM, et al. (2002) Identi&#239;&#172;?cation, characterization, crystal structure Ba- <br />     growth. Proc Natl Acad Sci USA 108(10):4176&#226;&#8364;&#8220;4181.                                               cillus subtilis nicotinic acid mononucleotide adenylyltransferase. J Biol Chem 277(5): <br />  7. Woong Park S, et al. (2011) Evaluating sensitivity Mycobacterium tuberculosis            3698&#226;&#8364;&#8220;3707. <br />     biotin deprivation using regulated gene expression. PLoS Pathog 7(9):e1002264.           30. Ollinger J, O&#226;&#8364;&#8482;Malley T, Ahn J, Odingo J, Parish T (2012) Inhibition sole type I <br />  8. Lewis K (2010) Persister cells. Annu Rev Microbiol 64:357&#226;&#8364;&#8220;372.                                  signal peptidase Mycobacterium tuberculosis bactericidal replicating and <br />  9. Fauvart M, De Groote VN, Michiels J (2011) Role persister cells chronic infections: <br />                                                                                                     nonreplicating conditions. J Bacteriol 194(10):2614&#226;&#8364;&#8220;2619. <br />     Clinical relevance perspectives anti-persister therapies. J Med Microbiol 60(Pt <br />                                                                                                 31. Baker TA, Sauer RT (2012) ClpXP, ATP-powered unfolding protein-degrada- <br />     6):699&#226;&#8364;&#8220;709. <br />                                                                                                     tion machine. Biochim Biophys Acta 1823(1):15&#226;&#8364;&#8220;28. <br /> 10. Monack DM (2012) Salmonella persistence transmission strategies. Curr Opin <br />                                                                                                 32. Osterman AL, Begley TP (2007) A subsystems-based approach identi&#239;&#172;?cation of <br />     Microbiol 15(1):100&#226;&#8364;&#8220;107. <br />                                                                                                     drug targets bacterial pathogens. Prog Drug Res 64:131, 133&#226;&#8364;&#8220;170. <br /> 11. Dhar N, McKinney JD (2007) Microbial phenotypic heterogeneity antibiotic tol- <br />                                                                                                 33. Via LE, et al. (2008) Tuberculous granulomas hypoxic guinea pigs, rabbits, and <br />     eran <br /> </body></html>